
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Roles of the lipopolysaccharide biosynthesis-related gene HP0858 in the fitness of Helicobacter pylori and its virulence in Galleria mellonella - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="1525492C8AF36AA305492C0003DC673F.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="viru">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377144/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Virulence">
<meta name="citation_title" content="Roles of the lipopolysaccharide biosynthesis-related gene HP0858 in the fitness of Helicobacter pylori and its virulence in Galleria mellonella">
<meta name="citation_author" content="Tram Thi Hong Nguyen">
<meta name="citation_author_institution" content="Institute of Molecular Medicine, National Tsing Hua University, Hsinchu, Taiwan">
<meta name="citation_author" content="Pei-Chun Chen">
<meta name="citation_author_institution" content="Institute of Molecular Medicine, National Tsing Hua University, Hsinchu, Taiwan">
<meta name="citation_author" content="Po-Chuan Wang">
<meta name="citation_author_institution" content="Department of Gastroenterology, Hsinchu MacKay Memorial Hospital, Hsinchu, Taiwan">
<meta name="citation_author" content="Wen-Han Wang">
<meta name="citation_author_institution" content="Institute of Molecular and Cellular Biology, National Tsing Hua University, Hsinchu, Taiwan">
<meta name="citation_author" content="Chung-Yu Lan">
<meta name="citation_author_institution" content="Institute of Molecular and Cellular Biology, National Tsing Hua University, Hsinchu, Taiwan">
<meta name="citation_author_institution" content="Department of Life Science, National Tsing Hua University, Hsinchu, Taiwan">
<meta name="citation_author_institution" content="School of Medicine, National Tsing Hua University, Hsinchu, Taiwan">
<meta name="citation_author" content="Steffen Backert">
<meta name="citation_author_institution" content="Division of Microbiology, Department of Biology, Friedrich-Alexander-Universität Erlangen-Nürnberg, Erlangen, Germany">
<meta name="citation_author" content="Mou-Chieh Kao">
<meta name="citation_author_institution" content="Institute of Molecular Medicine, National Tsing Hua University, Hsinchu, Taiwan">
<meta name="citation_author_institution" content="Department of Life Science, National Tsing Hua University, Hsinchu, Taiwan">
<meta name="citation_author_institution" content="School of Medicine, National Tsing Hua University, Hsinchu, Taiwan">
<meta name="citation_publication_date" content="2025 Aug 24">
<meta name="citation_volume" content="16">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="2548620">
<meta name="citation_doi" content="10.1080/21505594.2025.2548620">
<meta name="citation_pmid" content="40849897">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377144/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377144/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377144/pdf/KVIR_16_2548620.pdf">
<meta name="description" content="Helicobacter pylori is a pathogenic bacterium associated with the development of gastric cancer and other gastric disorders. One of its major virulence factors, lipopolysaccharide (LPS), plays a crucial role in maintaining bacterial integrity, ...">
<meta name="og:title" content="Roles of the lipopolysaccharide biosynthesis-related gene HP0858 in the fitness of Helicobacter pylori and its virulence in Galleria mellonella">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Helicobacter pylori is a pathogenic bacterium associated with the development of gastric cancer and other gastric disorders. One of its major virulence factors, lipopolysaccharide (LPS), plays a crucial role in maintaining bacterial integrity, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377144/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12377144">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1080/21505594.2025.2548620"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/KVIR_16_2548620.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377144%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12377144/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12377144/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377144/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-viru.png" alt="Virulence logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Virulence" title="Link to Virulence" shape="default" href="https://doi.org/10.1080/21505594.2025.2548620" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Virulence</button></div>. 2025 Aug 24;16(1):2548620. doi: <a href="https://doi.org/10.1080/21505594.2025.2548620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1080/21505594.2025.2548620</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Virulence%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Virulence%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Virulence%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Virulence%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Roles of the lipopolysaccharide biosynthesis-related gene <em>HP0858</em> in the fitness of <em>Helicobacter pylori</em> and its virulence in <em>Galleria mellonella</em>
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20TTH%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Tram Thi Hong Nguyen</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Tram Thi Hong Nguyen</span></h3>
<div class="p">
<sup>a</sup>Institute of Molecular Medicine, National Tsing Hua University, Hsinchu, Taiwan</div>
<div>Data curation, Formal analysis, Investigation, Methodology, Validation, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20TTH%22%5BAuthor%5D" class="usa-link"><span class="name western">Tram Thi Hong Nguyen</span></a>
</div>
</div>
<sup>a,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20PC%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Pei-Chun Chen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Pei-Chun Chen</span></h3>
<div class="p">
<sup>a</sup>Institute of Molecular Medicine, National Tsing Hua University, Hsinchu, Taiwan</div>
<div>Data curation, Formal analysis, Investigation, Methodology, Validation, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20PC%22%5BAuthor%5D" class="usa-link"><span class="name western">Pei-Chun Chen</span></a>
</div>
</div>
<sup>a,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20PC%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Po-Chuan Wang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Po-Chuan Wang</span></h3>
<div class="p">
<sup>b</sup>Department of Gastroenterology, Hsinchu MacKay Memorial Hospital, Hsinchu, Taiwan</div>
<div>Conceptualization, Funding acquisition, Resources, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20PC%22%5BAuthor%5D" class="usa-link"><span class="name western">Po-Chuan Wang</span></a>
</div>
</div>
<sup>b</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20WH%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Wen-Han Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Wen-Han Wang</span></h3>
<div class="p">
<sup>c</sup>Institute of Molecular and Cellular Biology, National Tsing Hua University, Hsinchu, Taiwan</div>
<div>Investigation, Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20WH%22%5BAuthor%5D" class="usa-link"><span class="name western">Wen-Han Wang</span></a>
</div>
</div>
<sup>c</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lan%20CY%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Chung-Yu Lan</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Chung-Yu Lan</span></h3>
<div class="p">
<sup>c</sup>Institute of Molecular and Cellular Biology, National Tsing Hua University, Hsinchu, Taiwan</div>
<div class="p">
<sup>d</sup>Department of Life Science, National Tsing Hua University, Hsinchu, Taiwan</div>
<div class="p">
<sup>e</sup>School of Medicine, National Tsing Hua University, Hsinchu, Taiwan</div>
<div>Methodology, Resources</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lan%20CY%22%5BAuthor%5D" class="usa-link"><span class="name western">Chung-Yu Lan</span></a>
</div>
</div>
<sup>c,</sup><sup>d,</sup><sup>e</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Backert%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Steffen Backert</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Steffen Backert</span></h3>
<div class="p">
<sup>f</sup>Division of Microbiology, Department of Biology, Friedrich-Alexander-Universität Erlangen-Nürnberg, Erlangen, Germany</div>
<div>Data curation, Funding acquisition, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Backert%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Steffen Backert</span></a>
</div>
</div>
<sup>f</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kao%20MC%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Mou-Chieh Kao</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Mou-Chieh Kao</span></h3>
<div class="p">
<sup>a</sup>Institute of Molecular Medicine, National Tsing Hua University, Hsinchu, Taiwan</div>
<div class="p">
<sup>d</sup>Department of Life Science, National Tsing Hua University, Hsinchu, Taiwan</div>
<div class="p">
<sup>e</sup>School of Medicine, National Tsing Hua University, Hsinchu, Taiwan</div>
<div>Conceptualization, Data curation, Funding acquisition, Resources, Supervision, Validation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kao%20MC%22%5BAuthor%5D" class="usa-link"><span class="name western">Mou-Chieh Kao</span></a>
</div>
</div>
<sup>a,</sup><sup>d,</sup><sup>e,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff0001">
<sup>a</sup>Institute of Molecular Medicine, National Tsing Hua University, Hsinchu, Taiwan</div>
<div id="aff0002">
<sup>b</sup>Department of Gastroenterology, Hsinchu MacKay Memorial Hospital, Hsinchu, Taiwan</div>
<div id="aff0003">
<sup>c</sup>Institute of Molecular and Cellular Biology, National Tsing Hua University, Hsinchu, Taiwan</div>
<div id="aff0004">
<sup>d</sup>Department of Life Science, National Tsing Hua University, Hsinchu, Taiwan</div>
<div id="aff0005">
<sup>e</sup>School of Medicine, National Tsing Hua University, Hsinchu, Taiwan</div>
<div id="aff0006">
<sup>f</sup>Division of Microbiology, Department of Biology, Friedrich-Alexander-Universität Erlangen-Nürnberg, Erlangen, Germany</div>
<div class="author-notes p">
<div class="fn" id="an0001">
<sup>✉</sup><p class="display-inline">CONTACT Mou-Chieh Kao <span>mckao@life.nthu.edu.tw</span></p>
</div>
<div class="fn" id="an0002">
<sup>*</sup><p class="display-inline">These authors contributed equally to this work.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Tram Thi Hong Nguyen</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Methodology, Validation, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Pei-Chun Chen</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Methodology, Validation, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Po-Chuan Wang</span></strong>: <span class="role">Conceptualization, Funding acquisition, Resources, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Wen-Han Wang</span></strong>: <span class="role">Investigation, Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Chung-Yu Lan</span></strong>: <span class="role">Methodology, Resources</span>
</div>
<div>
<strong class="contrib"><span class="name western">Steffen Backert</span></strong>: <span class="role">Data curation, Funding acquisition, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mou-Chieh Kao</span></strong>: <span class="role">Conceptualization, Data curation, Funding acquisition, Resources, Supervision, Validation, Writing – review &amp; editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jul 3; Accepted 2025 Aug 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). Published by Informa UK Limited, trading as Taylor &amp; Francis Group.</div>
<p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. The terms on which this article has been published allow the posting of the Accepted Manuscript in a repository by the author(s) or with their consent.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12377144  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40849897/" class="usa-link">40849897</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p><em>Helicobacter pylori</em> is a pathogenic bacterium associated with the development of gastric cancer and other gastric disorders. One of its major virulence factors, lipopolysaccharide (LPS), plays a crucial role in maintaining bacterial integrity, mediating host adhesion, and modulating the immune response. Recent studies have indicated that ADP-heptose, an intermediate in the heptose biosynthetic pathway involved in the LPS synthesis cascade, is a novel pathogen-associated molecular pattern for <em>H. pylori</em>. This study focuses on the <em>HP0858</em> gene, which is predicted to encode RfaE/HldE, an enzyme with kinase and ADP-transferase activities essential for heptose production. An <em>HP0858</em> gene-disrupted mutant was first generated, and the resulting mutant exhibited a truncated LPS structure, confirming its role in LPS biosynthesis. The <em>HP0858</em>-deficient mutant displayed increased sensitivity to the detergent SDS and the antibiotic novobiocin, heightened surface hydrophobicity, and a propensity for autoaggregation. Additionally, the mutant exhibited reduced adhesion and internalization capabilities, a diminished elongation phenotype, and failed to induce IL-8 secretion in infected gastric AGS cells. In an <em>in vivo Galleria mellonella</em> infection model, the <em>HP0858</em> knockout mutant showed significantly attenuated virulence, as no bacterial load was detectable in the larvae’s hemolymph 48 h post-infection, unlike the wild-type strain. Finally, we provided evidence that the enzyme encoded by <em>HP0858</em> is involved in a general protein glycosylation system linked to LPS biosynthesis, specifically glycosylating the adhesin AlpA. These findings highlight the essential role of RfaE/HldE/HP0858 in LPS biosynthesis and bacterial virulence, making it a promising target for future therapeutic interventions against <em>H. pylori</em> infections.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>KEYWORDS:</strong> <em>Helicobacter pylori</em>, lipopolysaccharide, ADP-heptose, bi-functional kinase/adenylyltransferase (RfaE/hlde/hp0858), protein glycosylation, <em>Galleria mellonella</em></p></section></section><section id="s0001"><h2 class="pmc_sec_title">Introduction</h2>
<p><em>Helicobacter pylori</em> (<em>H. pylori</em>) infects almost half of the world’s population and is identified as a common cause of gastritis and gastric and duodenal ulcers. Although most infected people have no clinical symptoms, <em>H. pylori</em> indeed causes inflammation in host cells [<a href="#cit0001" class="usa-link" aria-describedby="cit0001">1</a>], and the infected individuals may further develop gastric cancers or even mucosa-associated lymphoid tissue (MALT) lymphoma [<a href="#cit0002" class="usa-link" aria-describedby="cit0002">2</a>,<a href="#cit0003" class="usa-link" aria-describedby="cit0003">3</a>].</p>
<p>The adherence of bacteria to host cells is an essential step for <em>H. pylori</em> colonization, which helps bacteria fight against peristalsis, acquire nutrients, and deliver bacterial effector molecules to host cells. One group of bacterial virulence factors contains the bacterial outer membrane proteins (OMPs) recognized as adhesins, which contribute to the colonization of human gastric mucosa and the pathogenesis of <em>H. pylori</em> infection. Among these adhesins that are reported for binding <em>H. pylori</em> to the gastric epithelial cells, blood-group-antigen-binding adhesin A (BabA), sialic-acid binding adhesin A (SabA), outer inflammatory protein A (OipA), adherence-associated lipoproteins A and B (AlpA/AlpB), and <em>Helicobacter</em> outer membrane protein Q (HopQ) are the most studied OMPs in establishing a bacterial role in colonization [<a href="#cit0004" class="usa-link" aria-describedby="cit0004">4</a>].</p>
<p>Lipopolysaccharide (LPS) is a crucial surface determinant for <em>H. pylori</em> colonization and persistence in the gastric niche. LPS is a large and complex glycolipid localized in the outer membrane of Gram-negative bacteria. It has many functions, such as mediating the interaction between bacteria and their surrounding environment and maintaining the barrier feature of bacterial outer membranes. A recent analysis has redefined the <em>H. pylori</em> LPS structure, which suggested that LPS in <em>H. pylori</em> is composed of lipid A, a short and conserved core oligosaccharide, and a longer O-antigen embracing a domain previously assigned as the outer core region (<a href="#f0001" class="usa-link">Figure 1(a)</a>) [<a href="#cit0005" class="usa-link" aria-describedby="cit0005">5</a>]. Compared to other <em>Enterobacteriaceae</em> such as <em>Escherichia coli</em>, <em>Neisseria meningitides</em>, and <em>Pseudomonas aeruginosa</em> [<a href="#cit0006" class="usa-link" aria-describedby="cit0006">6</a>,<a href="#cit0007" class="usa-link" aria-describedby="cit0007">7</a>], the constitutive modifications of <em>H. pylori</em> lipid A result in weak activation of toll-like receptor 4 (TLR4), thus evading host innate immune response and promoting chronic infection [<a href="#cit0008" class="usa-link" aria-describedby="cit0008">8</a>]. The latest redefined core oligosaccharide of <em>H. pylori</em> LPS is known as a conserved region in most Gram-negative bacteria. It contains one 3-<em>deoxy</em>-D-<em>manno</em>-oct-2-ulsonic acid (Kdo), two LD-heptose residues (heptose I and heptose II), and one DD-heptose (heptose III) with a branched disaccharide composed of galactose and glucose. This part of LPS can contribute to <em>H. pylori</em> colonization and pathogenesis. The outermost domain of <em>H. pylori</em> LPS is O-antigen, which has a unique structure consisting of a conserved trisaccharide (-GlcNAc-Fuc-DD-Hep-), homopolymers named DD-heptane and α-1, 6-glucan, and variable Lewis antigens which can mimic the Lewis antigens in the human stomach and evade the host’s immune recognition [<a href="#cit0009" class="usa-link" aria-describedby="cit0009">9–12</a>].
</p>
<figure class="fig xbox font-sm" id="f0001"><h3 class="obj_head">Figure 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3251/12377144/d8905a5ed64e/KVIR_A_2548620_F0001_OC.jpg" loading="lazy" height="535" width="800" alt="Figure 1."></p>
<div class="p text-right font-secondary"><a href="figure/f0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Diagram of the LPS structure and heptose biosynthetic pathway in <em>H. pylori</em>. (a) The redefined LPS structure of <em>H. pylori</em> strains 26,695 and G27. (b) The heptose biosynthetic pathway in <em>H. pylori</em> contains five steps. It is catalyzed sequentially by GmhA (sedoheptulose 7-phosphate isomerase), RfaE/HldE (bifunctional D-β-D-heptose 7-phosphate kinase/D-β-D-heptose 1-phosphate adenyltransferase), GmhB (D-α, β-D-heptose 1,7-bisphosphate phosphatase), RfaE/HldE and RfaD/HldD (ADP-D-β-D-heptose epimerase). The resulting ADP-LD-Hep is then transferred to the core region of LPS by WaaC (heptosyltransferase I). The enzyme tested in this study is marked in red color.</p></figcaption></figure><p>Heptose is one of the main components of the LPS core region. As early as 2001, Fischer <em>et al</em>. proposed that an unknown factor could be injected into the host cell via the type IV secretion system (T4SS) during <em>H. pylori</em> infection, and thus trigger cytokine interleukin-8 (IL-8) activation in host cells [<a href="#cit0013" class="usa-link" aria-describedby="cit0013">13</a>]. CagA was later identified as one of the injected T4SS factors and a potent contributor to IL8 activation in many strains [<a href="#cit0014" class="usa-link" aria-describedby="cit0014">14</a>]. In 2015, D-<em>glycero</em>-β-D-<em>manno</em>-heptose 1, 7-bisphosphate (HBP), an intermediate metabolite of the heptose biosynthetic pathway, was identified as another potent effector of IL-8 activation by <em>Neisseria</em> species [<a href="#cit0015" class="usa-link" aria-describedby="cit0015">15</a>]. More recently, it was reported that proteins α-kinase 1 (ALPK1) and TRAF-interacting protein with forkhead-associated domain (TIFA) are critical components in the response to HBP during <em>H. pylori</em> infection, leading to NF-κB activation and IL-8 transcription [<a href="#cit0016" class="usa-link" aria-describedby="cit0016">16–18</a>]. However, the latest research indicated that this immune-regulatory pathway is mainly elicited by ADP-heptose, another intermediate metabolite of the heptose biosynthetic pathway, rather than HBP [<a href="#cit0019" class="usa-link" aria-describedby="cit0019">19</a>]. The complete ADP-LD-heptose biosynthetic pathway in <em>H. pylori</em> contains five steps that are similar to the corresponding biosynthetic pathway present in most Gram-negative bacteria (<a href="#f0001" class="usa-link">Figure 1(b)</a>) [<a href="#cit0020" class="usa-link" aria-describedby="cit0020">20</a>,<a href="#cit0021" class="usa-link" aria-describedby="cit0021">21</a>]: (1) the starting D-sedoheptulose 7-phosphate is first converted into D-<em>glycero</em>-α, β-D-<em>manno</em>-heptose 7-phosphate by an isomerase GmhA (encoded by <em>HP0857</em> in <em>H. pylori</em>); (2) the resulting product is phosphorylated to form D-<em>glycero</em>-β-D-<em>manno</em>-heptose 1, 7-bisphosphate (HBP) by the kinase activity of a bi-functional enzyme, RfaE/HldE (encoded by <em>HP0858</em> in <em>H. pylori</em>); (3) HBP is then dephosphorylated to form D-<em>glycero</em>-β-D-<em>manno</em>-heptose 1-phosphate (HMP-1) by a phosphatase GmhB (encoded by <em>HP0860</em> in <em>H. pylori</em>); (4) through nucleotide activation by the adenylyltransferase activity of RfaE/HldE, ADP-D-<em>glycero</em>-β-D-<em>manno</em>-heptose (ADP-DD-Hep) is formed; (5) ADP-DD-Hep is epimerized to ADP-L-<em>glycero</em>-β-D-manno-heptose (ADP-LD-Hep) by the epimerase RfaD/HldD (encoded by <em>HP0859</em> in <em>H. pylori</em>). The resulting ADP-LD-Hep is then transferred to Kdo in the LPS core region by the heptosyltransferase I WaaC (encoded by <em>HP0279</em> in <em>H. pylori</em>).</p>
<p><em>Galleria mellonella</em> (greater wax moth) is an efficient invertebrate model for studying bacterial virulence [<a href="#cit0022" class="usa-link" aria-describedby="cit0022">22</a>]. Compared to the mammalian models of bacterial infection, <em>G. mellonella</em> larvae offer several advantages, including no requirement for time-consuming ethical certification, being convenient to use and handle, generating results quickly, and having a low experimental cost. In particular, <em>G. mellonella</em> larvae can be injected with precise doses of bacterial concentration and incubated at 37°C, corresponding to the host body temperature. More importantly, <em>G. mellonella</em> larvae have an innate immune system with phagocytic cells known as hemocytes, which can engulf and kill invading pathogens. <em>G. mellonella</em> has been used as an <em>in vivo</em> model over recent years to study the virulence of various <em>Helicobacter</em> species by determining the survival rate of larvae and viable bacteria in the hemolymph of larvae during a time course of infection [<a href="#cit0023" class="usa-link" aria-describedby="cit0023">23</a>,<a href="#cit0024" class="usa-link" aria-describedby="cit0024">24</a>].</p>
<p>In previous studies, we investigated the functions/structures of HP0857 (GmhA), HP0860 (GmhB), and HP0859 (RfaD/HldD) proteins involved in the heptose biosynthetic pathway. We demonstrated that those proteins play crucial roles in LPS core biosynthesis and significantly influence bacterial fitness and virulence [<a href="#cit0025" class="usa-link" aria-describedby="cit0025">25–27</a>]. To clarify the remaining putative gene involved in heptose biosynthesis, we aimed to explore the involvement of HP0858 (RfaE/HldE) protein in LPS biosynthesis, bacterial physiology, as well as its virulence assessment in an <em>in vivo</em> model newly launched in this study. A knockout mutant and its corresponding complemented strain were constructed, and the effects of the <em>HP0858</em> mutations on LPS profile, pathogenesis, and bacterial virulence assessment in <em>G. mellonella</em> larvae were examined. Together, the experimental results from this study shed new light on the function of <em>H. pylori</em> LPS genes, which in turn can serve as promising novel targets for anti-microbial therapy in the future.</p></section><section id="s0002"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="s0002-s2001"><h3 class="pmc_sec_title">Bacterial strains, plasmids, and growth conditions</h3>
<p>Bacterial strains and plasmids used in this study are listed in <a href="#t0001" class="usa-link">Table 1</a>. Unless stated otherwise, stationary-phase <em>H. pylori</em> cultures were used throughout the assays. This growth phase represents a state of nutrient limitation and environmental stress, conditions that more accurately simulate the physiological challenges <em>H. pylori</em> faces within the gastric mucosa. Therefore, employing stationary-phase bacteria offers a more representative model for investigating bacterial adherence and other host-pathogen interaction phenotypes examined in this study. <em>H. pylori</em> strain 26,695 [KE26695] (ATCC 700,392; CagA<sup>+</sup>, VacA<sup>+</sup>) and its derived mutants were inoculated on sheep blood (5%, w/v) agar plates and incubated for 48–72 h at 37°C under a microaerophilic environment (5% O<sub>2</sub>, 10% CO<sub>2</sub>, and 85% N<sub>2</sub>). Bacteria were scraped from the agar plates and resuspended in flasks containing Brucella broth media (BD Biosciences, Franklin Lakes, NJ, USA) with 10% fetal bovine serum (FBS, Biological Industries, Kibbutz Beit Haemek, Israel) and 1% Iso Vitalex (Taiwan Prepared Media, Taipei, Taiwan). Flasks were then placed into an anaerobic chamber and cultivated for 48 h under microaerophilic conditions at 37°C with shaking at 140 r.p.m.</p>
<section class="tw xbox font-sm" id="t0001"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Bacterial strains and plasmids used in this study.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="left" span="1">
</colgroup>
<thead><tr>
<th colspan="2" align="left" rowspan="1">Strain/Plasmid</th>
<th align="center" colspan="1" rowspan="1">Characteristic and marker<sup>1</sup>
</th>
<th align="center" colspan="1" rowspan="1">Source</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">
<em>E. coli</em> DH5α</td>
<td align="left" colspan="1" rowspan="1">The host for the construction of pGEM-T-HP0858:Cm<sup>r</sup> clone</td>
<td align="left" colspan="1" rowspan="1">Invitrogen, Carlsbad, CA, USA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Strain</td>
<td align="left" colspan="1" rowspan="1">
<em>H. pylori</em> 26,695</td>
<td align="left" colspan="1" rowspan="1">
<em>H. pylori</em> whole-genome sequencing strain, isolated from the stomach of a patient with gastritis</td>
<td align="left" colspan="1" rowspan="1">ATCC<sup>2</sup> 700392</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">KO 0858</td>
<td align="left" colspan="1" rowspan="1">
<em>H. pylori</em> 26,695 strain with a chloramphenicol resistance cassette in <em>HP0858</em>; Cm<sup>r</sup>
</td>
<td align="left" colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">Com 0858</td>
<td align="left" colspan="1" rowspan="1">KO 0858 with <em>HP0858</em> insertion in <em>HP0954</em> (RdxA); Met<sup>r</sup>, Cm<sup>r</sup>
</td>
<td align="left" colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">pGEM-T</td>
<td align="left" colspan="1" rowspan="1">T-A cloning vector; Amp<sup>r</sup>
</td>
<td align="left" colspan="1" rowspan="1">Promega, Madison, WI, USA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">pJET1.2/blunt</td>
<td align="left" colspan="1" rowspan="1">Blunt PCR cloning vector</td>
<td align="left" colspan="1" rowspan="1">Thermo Fisher Scientific</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">pGEM-T-HP0858</td>
<td align="left" colspan="1" rowspan="1">pGEM-T with <em>HP0858</em> DNA fragment; Amp<sup>r</sup>
</td>
<td align="left" colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Plasmid</td>
<td align="left" colspan="1" rowspan="1">pGEM-T- HP0858:Cm<sup>r</sup>
</td>
<td align="left" colspan="1" rowspan="1">pGEM-T with <em>HP0858</em> interrupted with a chloramphenicol resistance cassette; Cm<sup>r</sup>
</td>
<td align="left" colspan="1" rowspan="1">This<sup>2</sup> study</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">pGEM-T-RdxA<sub>L</sub>- P<sub>HP1563</sub>-HP0858-T7<sub>ter</sub> -RdxA<sub>R</sub>
</td>
<td align="left" colspan="1" rowspan="1">pGEM-T with <em>HP0858</em> insertion in <em>HP0954</em> (RdxA); Met<sup>r</sup>, Cm<sup>r</sup>
</td>
<td align="left" colspan="1" rowspan="1">This study</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/t0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tfn0001"><p><sup>1</sup>Amp<sup>r</sup>, ampicillin-resistant; Cm<sup>r</sup>, chloramphenicol-resistant; and Met<sup>r</sup>, metronidazole-resistant.</p></div>
<div class="fn" id="tfn0002"><p><sup>2</sup>ATCC: American Type Culture Collection.</p></div>
</div></section></section><section id="s0002-s2002"><h3 class="pmc_sec_title">Construction of the <em>HP0858</em> knockout mutant and the corresponding complemented strain</h3>
<p>The <em>HP0858</em> knockout mutant was constructed using a strategy based on gene splicing by overlapping extension (SOEing) [<a href="#cit0028" class="usa-link" aria-describedby="cit0028">28–30</a>]. For the construction of the gene disruption mutant, the upstream (457 base pairs (bps)) and downstream (463 bps) regions of the <em>HP0858</em> gene were amplified by polymerase chain reaction (PCR) using <em>Taq</em> DNA polymerase (Mdbio, Taipei, Taiwan) with the genomic DNA extracted from <em>H. pylori</em> 26,695 wild-type (WT) strain using the primer pairs KO0858F1-KO0858R1/KO0858F2-KO0858R2 (<a href="#t0002" class="usa-link">Table 2</a>). The above two sets of generated PCR products were then joined via SOEing PCR with the primer pair KO0858F1-KO0858R2 carrying a <em>BamHI</em> restriction site in the PCR products for subsequent insertion of an antibiotic resistance cassette to disrupt the <em>HP0858</em> gene. The amplified 920-bp fragment of <em>HP0858</em> was purified and ligated into the pGEM-T Easy Vector (Promega, Madison, WI, USA) with T4 DNA ligase (New England Biolabs, Ipswich, MA, USA) to generate the pGEM-T-HP0858 plasmid. After ligation, the plasmid containing the target fragment was inserted with a chloramphenicol resistance gene at the <em>BamHI</em> restriction site and later transformed into <em>E. coli</em> strain DH5α (Thermo Fisher Scientific, Waltham, MA, USA). The resultant plasmid DNA was isolated and used for natural transformation into <em>H. pylori</em> 26,695 WT strain. The gene disruption mutant was selected using chloramphenicol-containing agar plates.</p>
<section class="tw xbox font-sm" id="t0002"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Sequences and restriction sites of the primers used in this study.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="left" span="1">
</colgroup>
<thead><tr>
<th align="left" colspan="1" rowspan="1">Primer</th>
<th align="center" colspan="1" rowspan="1">Restriction enzyme site</th>
<th align="center" colspan="1" rowspan="1">Sequence (5’→ 3’)<sup>3</sup>
</th>
</tr></thead>
<tbody><tr>
<td align="left" colspan="1" rowspan="1">KO0858F1<sup>1</sup><br>KO0858R1<sup>1</sup><br>KO0858F2<sup>1</sup><br>KO0858R2<sup>1</sup><br>Com0858F1<sup>2</sup><br>Com0858R1<sup>2</sup><br>Com0858F2<sup>2</sup><br>Com0858R2<sup>2</sup>
</td>
<td align="left" colspan="1" rowspan="1">-<br><em>BamHI</em><br><em>BamHI<br>-<br>NcoI<br>-<br>-<br>KpnI</em>
</td>
<td align="left" colspan="1" rowspan="1">TCTTAGTCATAGGCGATCTG<br>GAGTTCAAAGGATCCAACAC<br>GTGTTGGATCCTTTGAACTC<br>TCTTCTAAACTCGCTAACGC<sup>2</sup><br>AGTACCATGGTTGACTTGGATTTC<sup>3</sup><br>CGCCTATGACTAAGATTTTTTTCATATCGTAACTCC<br>GGAGTTACGATATGAAAAAAATCTTAGTCATAGGCG<br>AGTAGGTACCTCAATCATTGC</td>
</tr></tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/t0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tfn0003"><p><sup>1</sup>Primers used in the construction of the <em>HP0858</em> knockout mutant.</p></div>
<div class="fn" id="tfn0004"><p><sup>2</sup>Primers used in the construction of the HP0858 complemented strain.</p></div>
<div class="fn" id="tfn0005"><p><sup>3</sup>The sequences of restriction enzyme sites are underlined.</p></div>
</div></section><p>Gene complementation was conducted to complement the HP0858 knockout mutation by inserting the WT gene into the locus of the <em>rdxA</em> (<em>HP0954</em>) gene. A 1736-bp fragment containing the promoter region of the <em>HP1563</em> gene (350 bps) and the open reading frame (ORF) of the <em>HP0858</em> gene (1386 bps) was amplified by PCR using <em>Pfu</em> DNA polymerase (Promega) from <em>H. pylori</em> 26,695 genomic DNA as the template using the primer pairs Com0858F1-Com0858R1 and Com0858F2-Com0858R2, respectively (<a href="#t0002" class="usa-link">Table 2</a>). The obtained fragments were fused together after overlapping extension PCR with the primer pair Com0858F1-Com0858R2. This extension PCR product was next ligated into the pJET1.2/blunt Cloning Vector (Thermo Fisher Scientific) and transformed into <em>E. coli</em> strain DH5α. After purification, the isolated plasmid was digested with restriction enzymes <em>NcoI</em> and <em>KpnI</em>, resulting in an insert containing the promoter region of the <em>HP1563</em> gene and the <em>HP0858</em> gene. The resultant fragment was then ligated into the pGEM-T-RdxA<sub>L</sub>-T7<sub>ter</sub>-RdxA<sub>R</sub> plasmid, which had been <em>NcoI</em>/<em>KpnI</em>-digested to create the pGEM-T-RdxA<sub>L</sub>-P<sub><em>HP1563</em></sub>-<em>HP0858</em>-T7<sub>ter</sub>-RdxA<sub>R</sub> plasmid. After isolation, the consequent plasmid DNA was used for natural transformation into the <em>HP0858</em> disruption mutant, and the corresponding complemented strain was selected using chloramphenicol- and metronidazole-containing agar plates. Finally, the <em>HP0858</em> knockout mutation and its corresponding complemented mutation were confirmed by PCR amplification using primers spanning DNA upstream and downstream of the target sites, which was then confirmed through sequencing by Mission Biotech (Taipei, Taiwan). The plasmid DNA extraction and gel extraction kits were bought from Biokit (Miaoli, Taiwan). The restriction enzymes were purchased from New England Biolabs. The antibiotics used to select mutants were bought from Sigma-Aldrich (St. Louis, MO, USA).</p></section><section id="s0002-s2003"><h3 class="pmc_sec_title">Bioinformatics analysis</h3>
<p>The sequences of the <em>HP0858</em> gene and its corresponding protein were searched on the Kyoto Encyclopedia of Genes and Genomes (KEGG) website (<a href="https://www.genome.jp/kegg/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genome.jp/kegg/</a>) and UniProt (<a href="https://www.uniprot.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.uniprot.org/</a>), respectively. The conserved domain searching of the target protein was conducted by using the Conserved Domains Database (CDD) on the National Center for Biotechnology Information (NCBI) website (<a href="https://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml</a>). The protein sequence alignment with various bacterial species was generated by Clustal Omega (<a href="https://www.ebi.ac.uk/Tools/msa/clustalo/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/Tools/msa/clustalo/</a>) and colored by Jalview (<a href="https://www.jalview.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.jalview.org/</a>).</p></section><section id="s0002-s2004"><h3 class="pmc_sec_title">LPS profile analysis</h3>
<p>Bacteria were first scraped from agar plates and cultivated under liquid culture conditions for 48 h. After harvesting by centrifugation (4,000<em> × g</em>, 10 min, 4°C) to collect the bacteria, the obtained pellets were resuspended in 600 μL of the lysis buffer (1 M Tris-HCl, 1 M NaCl, 0.5 M EDTA, 10% SDS). The BCA protein assay kit was used to evaluate the amount of protein in the sample. The whole-cell lysate from the sample was added with 15 μL of proteinase K (20 mg/mL, Sigma-Aldrich), followed by the addition of deionized water to a total volume of 150 μL, and the resulting suspension was incubated at 60°C for 120 min. Then, 150 μL of 90% hot phenol solution (70°C) was added to the suspension, and the resulting mixture was incubated in a 70°C water bath for 15 min with gentle inversion every 5 min. The mixture was then cooled down on ice for 10 min. Finally, the upper aqueous phase containing LPS was collected by centrifugation (10,000<em>×</em>g, 10 min, 4°C). LPS profiles were analyzed by silver staining. The LPS samples were separated by 15% Tricine gel and fixed in 150 mL fixing solution (25% isopropanol and 7% acetic acid) overnight. The obtained gel was then soaked in the oxidizing solution (27% ethanol, 0.7% periodic acid, 0.3% acetic acid) for 10 min and washed for 30 min with distilled water three times. After oxidation, the gel was transferred into the staining solution (18.7% 1N NaOH, 1.3% ammonium hydroxide, 0.6% silver nitrate), stained for 25 min, and washed for 10 min with distilled water three times. Finally, the gel was developed by a reduction reaction in the developing solution (0.074% formaldehyde and 0.05% citric acid). The purified LPS samples were also separated by 15% sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE), transferred to 0.45 µm nitrocellulose membranes (NC membrane, Millipore, Billerica, MA, USA) for immunoblotting analysis. The transferred membranes were blocked with 5% skim milk in phosphate-buffered saline (PBS) for 1 h at room temperature, followed by incubation overnight at 4°C with the primary monoclonal mouse-anti-Lewis X antibody and mouse-anti-Lewis Y antibody (1:1,000, both from Abcam, Cambridge, UK), respectively. Afterward, the membranes were washed and incubated with the secondary HRP goat anti-mouse IgM antibody (1:10,000, Thermo Fisher Scientific) for 1 h at room temperature. After washing, the membranes were visualized by the Bio-Rad ChemiDoc MP Imaging System (Hercules, California, USA).</p></section><section id="s0002-s2005"><h3 class="pmc_sec_title">Subcellular fractionation and immunoblotting analysis</h3>
<p><em>H. pylori</em> was cultured to the stationary phase for 48 h and then harvested by centrifugation (4,000×g, 10 min, 4°C) to collect the bacterial pellet. After washing once with 1×PBS and centrifugation at 4,000×g for 10 min, the obtained bacteria were resuspended in 20 mM Tris-HCl (pH 7.4) and broken by sonication (30% amplitude for 10 min with a 5 s interval in each cycle). The debris was first removed by centrifugation (10,000×g, 10 min, 4°C), and the whole cell lysate was further fractionated by ultracentrifugation. After the first ultracentrifugation (50,000<em>×</em>g, 45 min, 4°C), the supernatant was considered the cytoplasm/periplasm (C/P) fraction, and the proteins inside were collected by acetone precipitation. The obtained pellet, considered the total membrane fraction, was resuspended in 20 mM Tris-HCl (pH 7.4) supplemented with 2% N-lauroylsarcosine sodium (Sigma-Aldrich), followed by a second ultracentrifugation at 50,000<em> × g</em> (45 min, 4°C). Finally, the inner membrane (IM) fraction contained in the obtained supernatant was collected by acetone precipitation, and the resulting pellet, as the outer membrane (OM) fraction, was resuspended in 20 mM Tris-HCl (pH 7.4). Protein concentration in each subcellular fraction was evaluated using the BCA protein assay kit. Protein samples from different subcellular fractions were separated by 12% SDS-PAGE gel and then stained with 0.25% Coomassie brilliant blue R250 (Sigma-Aldrich, Burlington, MA, USA) or transferred to 0.45 µm NC membranes and followed by visualization by immunoblotting analysis using monoclonal rabbit anti-AlpA antibody (1:3,000).</p></section><section id="s0002-s2006"><h3 class="pmc_sec_title">Bacterial growth curve analysis</h3>
<p>Bacteria were scraped from agar plates after 48 h of growth and then switched to Brucella broth media with 10% FBS and 1% Iso Vitalex under microaerophilic conditions at 37°C for 16 h. The bacterial suspensions were then diluted to an optical density of 0.05 at 600 nm (OD<sub>600</sub>) and cultivated under microaerophilic conditions at 37°C with shaking at 140 r.p.m. for up to 4 days. The growth of bacteria in the Brucella broth medium was evaluated by measuring the absorbance at OD<sub>600</sub> with a spectrophotometer during an 84-h cell growth period. Bacterial growth was also assessed using the plate counting method under the above-specified conditions to determine the colony-forming unit (CFU) counts. The experiment was repeated in triplicate to obtain the average values and standard deviation.</p></section><section id="s0002-s2007"><h3 class="pmc_sec_title">SDS and novobiocin sensitivity assay</h3>
<p>Bacteria were scraped from agar plates, cultivated under liquid culture conditions for 36 h, and then diluted to an OD<sub>600</sub> of 0.6 in Brucella broth media. The bacterial suspensions were added to 6-well plates (2 mL/well) supplemented with the indicated agent (detergent SDS or antibiotic novobiocin) in different concentrations and cultivated under a microaerophilic condition at 37°C with shaking at 140 r.p.m. After growth for 72 h, the absorbance at OD<sub>600</sub> was measured, and the survival rate was defined as follows: survival rate (%) = (OD<sub>600</sub> of treatment/OD<sub>600</sub> of control) ×100. Each experiment was executed in three independent tests to obtain the average results and standard deviation.</p></section><section id="s0002-s2008"><h3 class="pmc_sec_title">Surface hydrophobicity and autoaggregation assays</h3>
<p>Bacteria were scraped from agar plates and grown in Brucella broth media under liquid culture conditions for 48 h. For the surface hydrophobicity assay, bacteria were harvested by centrifugation (4,000<em> × g</em>, 10 min, 4°C), resuspended and diluted to OD<sub>600</sub> of 1 in 5 mL phosphate-urea-magnesium sulfate buffer (PUM: 22.2 g/L K<sub>2</sub>HPO<sub>4</sub> ×3 H<sub>2</sub>O, 7.26 g/L KH<sub>2</sub>PO<sub>4</sub>, 1.8 g/L urea, 0.02 g/L MgSO<sub>4</sub> ×7 H<sub>2</sub>O; pH 7.1) supplemented with 400 μL n-Hexadecane. The bacterial suspensions were thoroughly mixed using a vortex for 2 min. The tubes containing the suspensions were then left undisturbed on the bench up for 24 h. After phase separation, the aqueous phase was carefully removed, and the absorbance at OD<sub>600</sub> was measured at 0 (A<sub>0</sub>) and other tested time points (A<sub>t</sub>). The hydrophobicity index was calculated as follows: the hydrophobicity index = (A<sub>0</sub>–A<sub>t</sub>)/A<sub>0</sub>. For the autoaggregation assay, bacteria were harvested by centrifugation (4,000<em> × g</em>, 10 min, 4°C), resuspended, and diluted to an OD<sub>600</sub> of 1 in 6 mL 1× PBS. The resulting tubes were left standing on the bench up for 24 h. The collection of the aqueous phase samples and the OD<sub>600</sub> recording were executed as described for the surface hydrophobicity assay. The ratio of autoaggregation was defined as follows: the autoaggregation rate (%) = (A<sub>0</sub>–A<sub>t</sub>)/A<sub>0</sub> ×100. Each experiment was repeated in triplicate to obtain the average values and standard deviation.</p></section><section id="s0002-s2009"><h3 class="pmc_sec_title">Cell culture and bacterial infection assay</h3>
<p>Gastric AGS cells (ATCC® CRL-1739™, a human gastric adenocarcinoma cell line) were cultivated at 37°C in Ham’s F-12 media (Sigma-Aldrich) supplemented with 10% FBS and 1% penicillin/streptomycin (Gibco). For <em>H. pylori</em> infection experiments, the bacteria used were in the stationary growth phase. AGS cells, seeded on 6-well plates (4 × 10<sup>5</sup> cells/well), were infected at a multiplicity of infection (MOI) of 100 in Ham’s F-12 media containing 10% FBS for 6 h. After co-culture with bacteria, the morphological changes of infected cells were recorded by Olympus I×71research inverted system microscope (Olympus, Tokyo, Japan) with 200× magnification.</p></section><section id="s0002-s2010"><h3 class="pmc_sec_title">Adhesion assay</h3>
<p>Following the bacterial infection process described above, the dish with the bacteria-infected AGS cells was washed three times with PBS to remove non-adherent bacteria. The obtained AGS cells were lysed with 0.1% saponin to release the cell-associated bacteria. The obtained lysates were then serially diluted and plated on sheep blood agar plates. The adhered bacteria were quantified by counting the viable colony forming units (CFU) on agar plates after 72 h of incubation under microaerophilic conditions. Each experiment was repeated in triplicate.</p></section><section id="s0002-s2011"><h3 class="pmc_sec_title">Internalization assay</h3>
<p>Following the bacterial infection process described above, the dish with the bacteria-infected AGS cells was treated with Ham’s F-12 media containing 10% FBS and 100 μg/mL gentamicin, incubated for 1 h to eliminate extracellular bacteria, and washed with PBS. The infected cells were further incubated in Ham’s F-12 media containing 10% FBS with 10 μg/mL gentamicin for 24 h under standard culture conditions. Cell lysate preparation, plating, and incubation were carried out as described in the adhesion assay. The number of internalized bacteria was quantified by counting the CFU on agar plates. Each experiment was repeated in triplicate.</p></section><section id="s0002-s2012"><h3 class="pmc_sec_title">IL-8 measurement</h3>
<p>AGS cells were infected with bacteria (MOI of 100) for 6 h, and the supernatants were collected for IL-8 measurements. IL-8 produced by the infected AGS cells was quantitatively analyzed using a human IL-8 uncoated ELISA kit (Invitrogen) according to the manufacturer’s instructions.</p></section><section id="s0002-s2013"><h3 class="pmc_sec_title">
<em>Galleria mellonella</em> virulence assay</h3>
<p><em>G. mellonella</em> larvae were applied to study the effect of <em>HP0858</em> gene disruption on <em>H. pylori</em> virulence. No ethical approval was required to use <em>G. mellonella</em> larvae for bacterial infection. <em>G. mellonella</em> larvae were maintained in a plastic container in the dark at 10–15 °C while reaching the last larval stage, approximately at a 40-day-old caterpillar stage, with 0.1 and 0.3 g in weight and 2.5 to 3 cm in length. <em>G. mellonella</em> larvae were ready for injection with <em>H. pylori</em> [<a href="#cit0031" class="usa-link" aria-describedby="cit0031">31</a>,<a href="#cit0032" class="usa-link" aria-describedby="cit0032">32</a>]. Bacteria were collected from blood agar plates, placed into 1 ml of PBS, adjusted to an OD<sub>600</sub> of 1, and then serially diluted in PBS, followed by plating on blood agar plates under microaerophilic conditions to estimate the bacterial concentration accurately. Before the bacterial infection procedure, larvae were acclimated to room temperature without food for 1 day. Ten selected larvae in a group were placed in sterile Petri dishes, decontaminated with 70% ethanol using a sterile cotton swab, followed by injecting 20 µL of bacterial suspensions appropriately diluted in PBS (from 1 × 10<sup>7</sup> to 1 × 10<sup>4</sup> CFUs/larva) using a Hamilton syringe into the hemocoel at the last left proleg of the larva. A cohort of ten larvae injected with 20 µL PBS was used as the control group to measure any potential lethal effects of the PBS solution or the injection procedure. After injection, larvae in the Petri dishes were incubated at 37°C in standard aerobic conditions without food, and the survival rate of larvae was monitored and recorded at 24-h intervals for 4 days post-infection. Larvae were considered dead when they changed into dark brown or black color and had no movement or were unresponsive to touch [<a href="#cit0023" class="usa-link" aria-describedby="cit0023">23</a>,<a href="#cit0033" class="usa-link" aria-describedby="cit0033">33</a>]. To determine the number of viable bacteria in larvae at 0, 12, 24, 48, and 72 h post-infection, an inoculum of bacterial suspensions, approximately 10<sup>7</sup> CFUs/larva, suspended in PBS, was injected into <em>G. mellonella</em> larvae. The exact inoculum was determined by CFU quantification. After the specified post-infection time, the injection site – specifically the last left proleg – was disinfected with 70% ethanol. A sterile needle was then gently inserted at this site to extract hemolymph. On average, approximately 4 µL of hemolymph was collected from each larva, resulting in a pooled volume of around 40 µL from 10 larvae. The pooled hemolymph was maintained on ice throughout the procedure to ensure sample stability. Subsequently, a 20 µL aliquot was subjected to a 10-fold serial dilution in 1× PBS to lyse hemocytes. The diluted samples were plated onto blood agar and incubated at 37°C for 3 to 4 days, after which the bacterial load per larva was quantified by counting CFUs.</p></section><section id="s0002-s2014"><h3 class="pmc_sec_title">Statistics analysis</h3>
<p>All statistical analyses were performed using GraphPad Prism version 8.3.0.538 (GraphPad Software, San Diego, CA, USA). Each experiment was conducted with at least three independent repetitions. Survival curves were plotted using the Kaplan-Meier method, and the differences in survival were calculated using the log-rank (Mantel-Cox) test for multiple comparisons. Error bars represent the standard deviations (SD), and the probability values (<em>p</em> values, p) are calculated using an unpaired, two-tailed Student’s <em>t</em>-test. Asterisks indicate the data with statistical significance (* p &lt; 0.05, ** p &lt; 0.01, *** p &lt; 0.001, **** p &lt; 0.0001).</p></section></section><section id="s0003"><h2 class="pmc_sec_title">Results</h2>
<section id="s0003-s2001"><h3 class="pmc_sec_title">Sequence analysis of HP0858 protein in <em>H.</em>
<em>pylori</em> strain 26,695</h3>
<p>The <em>HP0858</em> coding sequence is located at the minus strand of the <em>H. pylori</em> genome from 910,923 to 912,308, flanked by the <em>HP0857</em> gene and the <em>HP0859</em> gene in strain 26,695 (<a href="#f0002" class="usa-link">Figure 2(a)</a>). The entire length of the <em>HP0858</em> gene is 1,386 bps and can be translated into a protein with 461 amino acid residues. The conserved domain of the HP0858 protein is found in the members of the RfaE superfamily (<a href="#f0002" class="usa-link">Figure 2(b)</a>). The amino acid sequences of HP0858 homologous proteins from different species of Gram-negative bacteria were aligned using Clustal Omega, and the obtained results were colored using Jalview. According to the results, HP0858 protein shows high sequence homology with RfaE, also called HldE, from other Gram-negative bacteria (<a href="#t0003" class="usa-link">Table 3</a> and <a href="#f0002" class="usa-link">Figure 2(c)</a>). This result indicates that HP0858 protein could be an RfaE that contains both kinase and adenylyltransferase activity and plays a crucial role in the biosynthesis of heptose and, thus, is necessary for the structural integrity of the LPS core region in <em>H. pylori</em> (<a href="#f0001" class="usa-link">Figure 1</a>).
</p>
<figure class="fig xbox font-sm" id="f0002"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3251/12377144/3f1f72b06106/KVIR_A_2548620_F0002_OC.jpg" loading="lazy" height="553" width="800" alt="Figure 2."></p>
<div class="p text-right font-secondary"><a href="figure/f0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The bioinformatic analysis of <em>HP0858</em> gene and its encoding protein in <em>H. pylori</em> 26,695 strain. (a) The gene locus of <em>HP0858</em>. (b) The prediction of conserved domains in the HP0858 protein. The data were obtained from the NCBI (<a href="https://www.ncbi.nlm.nih.gov/pmc/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/pmc/</a>) accessed on 12 November 2022. (c) Multiple sequence alignment of HP0858 protein and its homologues with D-<em>glycero</em>-α, β-D-<em>manno</em>-heptose 7-phosphate kinase/D-<em>glycero</em>-β-D-<em>manno</em>-heptose 1-phosphate adenosyltransferase activity (RfaE/hlde): <em>Helicobacter pylori</em> 26,695 [<a href="https://www.ncbi.nlm.nih.gov/protein/O25529" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">O25529</a>], <em>Pseudomonas aeruginosa</em> PAO1 [<a href="https://www.ncbi.nlm.nih.gov/protein/Q9HUG9" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9HUG9</a>], <em>Escherichia coli</em> K-12 [<a href="https://www.ncbi.nlm.nih.gov/protein/P76658" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P76658</a>], <em>Salmonella typhimurium</em> LT2 [<a href="https://www.ncbi.nlm.nih.gov/protein/Q7CPR9" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q7CPR9</a>] and <em>Yersinia enterocolitica</em> 8081 [<a href="https://www.ncbi.nlm.nih.gov/protein/A1JQV6" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">A1JQV6</a>]. The numbers on the right side of the figure indicate the amino acid positions. The gaps are represented by dashes. The identical and similar residues were marked with dark and light blue colors, respectively, indicating different degrees of similarity. Multiple sequence alignment was performed using the ClustalW2 program (<a href="https://www.ebi.acuk/Tools/msa/clustalw2/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.acuk/Tools/msa/clustalw2/</a>) accessed on 12 November 2022. Shading was obtained using the jalview server (<a href="https://www.jalview.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.jalview.org/</a>) accessed on 12 November 2022.</p></figcaption></figure><section class="tw xbox font-sm" id="t0003"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Sequence comparison of <em>H. pylori</em> HP0858 protein from <em>H. pylori</em> wild-type strain 26,695 with the corresponding homologues from other gram-negative bacteria.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="center" span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="char" char="." span="1">
<col width="1*" align="center" span="1">
</colgroup>
<thead><tr>
<th align="left" colspan="1" rowspan="1"> </th>
<th align="center" colspan="1" rowspan="1">Species</th>
<th align="center" colspan="1" rowspan="1">UniProt AC<sup>1</sup>
</th>
<th align="center" colspan="1" rowspan="1">Identities (%)</th>
<th align="center" colspan="1" rowspan="1">Similarity (%)</th>
<th align="center" colspan="1" rowspan="1">Length</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="5" align="left" colspan="1">HP0858 (RfaE/HldE)</td>
<td align="left" colspan="1" rowspan="1">
<em>Helicobacter pylori</em> 26,695</td>
<td align="center" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/O25529" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">O25529</a></td>
<td align="center" colspan="1" rowspan="1">100%</td>
<td align="center" colspan="1" rowspan="1">100%</td>
<td align="center" colspan="1" rowspan="1">461 aa</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Pseudomonas aeruginosa</em> PAO1</td>
<td align="center" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q9HUG9" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9HUG9</a></td>
<td align="center" colspan="1" rowspan="1">42%</td>
<td align="center" colspan="1" rowspan="1">60%</td>
<td align="center" colspan="1" rowspan="1">474 aa</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Escherichia coli</em> K-12</td>
<td align="center" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P76658" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P76658</a></td>
<td align="center" colspan="1" rowspan="1">41%</td>
<td align="center" colspan="1" rowspan="1">59%</td>
<td align="center" colspan="1" rowspan="1">477 aa</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Salmonella typhimurium</em> LT2</td>
<td align="center" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q7CPR9" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q7CPR9</a></td>
<td align="center" colspan="1" rowspan="1">41%</td>
<td align="center" colspan="1" rowspan="1">58%</td>
<td align="center" colspan="1" rowspan="1">477 aa</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Yersinia enterocolitica</em> 8081</td>
<td align="center" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/A1JQV6" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">A1JQV6</a></td>
<td align="center" colspan="1" rowspan="1">40%</td>
<td align="center" colspan="1" rowspan="1">59%</td>
<td align="center" colspan="1" rowspan="1">476 aa</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/t0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tfn0006"><p><sup>1</sup>An accession number (AC) that is assigned to each sequence upon inclusion into UniProt (<a href="https://www.uniprot.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.uniprot.org/</a>) accessed on 12 November 2022; aa, amino acid.</p></div></div></section></section><section id="s0003-s2002"><h3 class="pmc_sec_title">Gene disruption in <em>HP0858</em> results in a truncated LPS structure</h3>
<p><em>H. pylori</em> strain 26,695 [KE26695] (ATCC 700,392; CagA<sup>+</sup>, VacA<sup>+</sup>) was used in this study for all analyses. This isolate is non-motile and non-flagellated [<a href="#cit0034" class="usa-link" aria-describedby="cit0034">34</a>]. Supplementary Figure S1 demonstrates that the <em>H. pylori</em> strain 26,695 and its derived mutants tested in this study consistently remained near the inoculation zone over the 6-day culture period, providing evidence of their lack of motility. To investigate the role of <em>HP0858</em> in LPS biosynthesis, the <em>HP0858</em> knockout mutant and its corresponding complemented control strain were constructed. The <em>HP0858</em> gene in genomic <em>H. pylori</em> DNA was disrupted by inserting a chloramphenicol resistance cassette (Cm<sup>r</sup>) to generate the <em>HP0858</em> knockout mutant. The <em>HP0858</em> complemented strain was constructed by inserting a DNA fragment carrying the promoter region of the <em>HP1563</em> gene in front of the full-length <em>HP0858</em> gene into the <em>rdxA</em> locus of the <em>HP0858</em> knockout mutant through natural transformation. The successful generation of these mutants was first confirmed by PCR amplification of the target genes and then further verified through gene sequencing (data not shown). To examine the effects of <em>HP0858</em> gene disruption on LPS expression, LPS from different <em>H. pylori</em> strains were isolated and analyzed using silver staining and immunoblotting with anti-Lewis X and Y antibodies. As shown in <a href="#f0003" class="usa-link">Figure 3(a)</a>, the <em>HP0858</em> knockout mutant had a truncated LPS structure, which contained only a part of the Lipid A-core oligosaccharide and lost all of the O-antigen domain while separating on a tricine gel with silver staining. In addition, while testing with immunoblotting analysis to examine the presence of Lewis antigens, the signals of Lewis X and Lewis Y antigens were utterly missing (<a href="#f0003" class="usa-link">Figure 3(b)</a>). In contrast, the above LPS defects were fully restored in the <em>HP0858</em> complemented strain (<a href="#f0003" class="usa-link">Figures 3(a,b)</a>). Our results suggest that the protein encoded by the <em>HP0858</em> gene plays a crucial role in the formation of LPS core oligosaccharide, and the disruption of <em>HP0858</em> involved in the heptose biosynthetic pathway not only affects the length of <em>H. pylori</em> LPS but also causes a complete loss of Lewis antigen moieties.
</p>
<figure class="fig xbox font-sm" id="f0003"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3251/12377144/07af53fad0d8/KVIR_A_2548620_F0003_OC.jpg" loading="lazy" height="311" width="800" alt="Figure 3."></p>
<div class="p text-right font-secondary"><a href="figure/f0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effects of the <em>HP0858</em> gene-disrupted mutation on LPS structure and Lewis antigen appearance. The LPS profiles from 26,695 strain WT, the <em>HP0858</em> knockout mutant, and its corresponding complemented strain were examined by (a) 15% tricine-PAGE with silver staining or (b) immunoblotting analysis with anti-Lewis X and anti-Lewis Y antibodies. The relative positions of the O-antigen and lipid A-core region were marked with braces. WT, wild-type strain; KO, knockout mutant; Com, complemented strain.; M, LPS from <em>E. coli</em> O111:B4; m’, the protein ladder used was the TriColor prestained protein ladder (10–180 kDa; catalog number AGEL3523), supplied by Assay Genie (Dublin, Ireland); m, the protein marker used was the AccuRuler RGB prestained protein ladder (10–180 kDa; catalog number 02101–250), supplied by MaestroGen Inc. (Hsinchu city, Taiwan).</p></figcaption></figure></section><section id="s0003-s2003"><h3 class="pmc_sec_title">
<em>HP0858</em> gene-disrupted mutation slightly affects bacterial growth and causes hypersensitivity to detergent SDS and antibiotic novobiocin</h3>
<p>To investigate if a deficiency in the heptose biosynthetic pathway of LPS could have an impact on the bacterial growth rate of <em>H. pylori</em>, the bacterial growth of the <em>H. pylori</em> 26,695 WT strain WT, the <em>HP0858</em> knockout mutant, and the <em>HP0858</em> complemented strain was measured at OD<sub>600</sub> by a spectrophotometer at different time points up to 84 h. The growth curves of all tested bacteria entered the log phase after 12 h of cultivation, reached the stationary phase after 36 h of growth, and then went into the death phase after 48 h of growth, even though a slight reduction in the growth rate was observed for the <em>HP0858</em> knockout mutant (<a href="#f0004" class="usa-link">Figure 4(a)</a>). These results indicate that the disruption of the <em>HP0858</em> gene involved in the heptose biosynthetic pathway of LPS formation does not cause a dramatic effect on the growth of <em>H. pylori</em> at the time points tested.
</p>
<figure class="fig xbox font-sm" id="f0004"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3251/12377144/b9c39a94114b/KVIR_A_2548620_F0004_OC.jpg" loading="lazy" height="526" width="800" alt="Figure 4."></p>
<div class="p text-right font-secondary"><a href="figure/f0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effects of the <em>HP0858</em> gene-disrupted mutation on <em>H. pylori</em> growth, sensitivity to the detergent SDS, and resistance to the antibiotic novobiocin. (a) The growth curves of 26,695 strain WT, the <em>HP0858</em> knockout mutant, and its corresponding complemented strain. The growth data were measured with a spectrophotometer by recording the OD<sub>600</sub> values at different time points. The data are displayed as the average OD<sub>600</sub> values obtained from triplicate tests with statistical analysis (unpaired, two-tailed Student’s t-test; *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001). (b) SDS sensitivity and (c) novobiocin resistance of the tested <em>H. pylori</em> strains. The effects of the <em>HP0858</em> mutation on SDS sensitivity and novobiocin resistance of <em>H. pylori</em> were evaluated by the survival rate, which was calculated from the ratio of the OD<sub>600</sub> of bacteria treated with/without various concentrations of SDS and novobiocin. The data displayed were obtained from triplicate tests with statistical analysis (unpaired, two-tailed Student’s t-test; *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001, ****<em>p</em> &lt; 0.0001). WT, wild-type strain; KO, knockout mutant; Com, complemented strain.</p></figcaption></figure><p>LPS in Gram-negative bacteria is thought to maintain the structural integrity of bacterial outer membranes and provide a barrier against the entry of toxic hydrophobic compounds. Previous research from our laboratory and other laboratories has already found that Gram-negative mutants defective in heptose biosynthesis for the LPS construction display increasing sensitivity to low concentrations of antibiotics and detergents [<a href="#cit0025" class="usa-link" aria-describedby="cit0025">25–27</a>,<a href="#cit0035" class="usa-link" aria-describedby="cit0035">35</a>]. To investigate whether the disruption of the <em>HP0858</em> gene would affect the resistance of <em>H. pylori</em> to detergents and hydrophobic antibiotics, SDS and novobiocin, respectively, were used as agents to induce bacterial growth stress. As shown in <a href="#f0004" class="usa-link">Figures 4(b,c)</a>, the <em>HP0858</em> gene-disrupted mutant exhibited hypersensitivity to SDS and novobiocin, and the <em>HP0858</em> complemented strain restored most of the WT resistance level to these agents. These results indicate that the HP0858 protein involved in the heptose biosynthetic pathway is essential in constructing the <em>H. pylori</em> LPS core structure, which is critical in bacterial resistance to the entry of toxic hydrophobic molecules.</p></section><section id="s0003-s2004"><h3 class="pmc_sec_title">The <em>HP0858</em> gene-disrupted mutation significantly changes the surface properties of <em>H.</em>
<em>pylori</em>
</h3>
<p>The disappearance of core oligosaccharides in LPS has been demonstrated to increase the cell surface hydrophobicity of <em>E. coli</em> [<a href="#cit0036" class="usa-link" aria-describedby="cit0036">36</a>]. The surface hydrophobicity and autoaggregation assays were applied to test the effects of <em>HP0858</em> gene disruption on the surface properties of <em>H. pylori</em>. The surface hydrophobicity assay applied here measured the hydrophobic surface properties of bacteria by testing their ability to adhere to a hydrocarbon, n-Hexadecane. As shown in <a href="#f0005" class="usa-link">Figure 5(a)</a>, a “creamy and white” layer was formed at the top of the tubes after phase separation. This layer is an oil-in-water emulsion consisting of n-Hexadecane droplets and can be laden with adherent bacteria if bacteria’s adhesion to n-Hexadecane occurs. A reduction in the OD<sub>600</sub> measurement of the lower aqueous phase was used to evaluate the cell surface hydrophobicity. The result showed that the <em>HP0858</em> knockout mutant significantly increased its surface hydrophobicity compared to the WT strain at all the tested time points (6 h, 8 h, and 24 h). In addition, this mutant’s autoaggregation tendency also displayed a similar trend to that found in the surface hydrophobicity assay (<a href="#f0005" class="usa-link">Figure 5(b)</a>). The <em>HP0858</em> knockout mutant strain started to autoaggregate after 30 min of incubation. As for the <em>HP0858</em> complemented strain, it displayed a WT-like level of surface hydrophobicity and autoaggregation.
</p>
<figure class="fig xbox font-sm" id="f0005"><h4 class="obj_head">Figure 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3251/12377144/b06d0cb14ce9/KVIR_A_2548620_F0005_OC.jpg" loading="lazy" height="417" width="800" alt="Figure 5."></p>
<div class="p text-right font-secondary"><a href="figure/f0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effects of the <em>HP0858</em> gene-disrupted mutation on the surface properties of <em>H. pylori</em>. The bacterial suspensions were incubated with (a) n-hexadecane for the surface hydrophobicity analysis and (b) phosphate-buffered saline (PBS) for the autoaggregation assay. The hydrophobicity index and autoaggregation percentage were calculated according to the procedures described in the materials and methods section. They represent the cell surface hydrophobicity and bacterial autoaggregation, respectively. The OD<sub>600</sub> value obtained from the zero-time point was employed as a control for background subtraction. Data displayed are derived from triplicate tests with statistical analysis (unpaired, two-tailed Student’s t-test; *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001). WT, wild-type strain; KO, knockout mutant; Com, complemented strain.</p></figcaption></figure></section><section id="s0003-s2005"><h3 class="pmc_sec_title">The <em>HP0858</em> gene-disrupted mutation has significant impacts on H. pylori virulence</h3>
<p><em>H.</em>
<em>pylori</em> has been reported to infect AGS cells and cause a morphological change called the “hummingbird or elongation phenotype” [<a href="#cit0037" class="usa-link" aria-describedby="cit0037">37–40</a>]. To study the role of HP0858 upon infection, AGS cells were co-incubated with the various produced strains for 6 h. <a href="#f0006" class="usa-link">Figures 6(a,b)</a> show that approximately 37% and 39% of AGS cells displayed an elongated phenotype after being infected with WT and the <em>HP0858</em> complemented strain, respectively. Conversely, less than 10% of elongated AGS cells were present in those infected by the <em>HP0858</em> knockout mutant. These results suggest that the <em>HP0858</em> gene-disrupted mutation could reduce the infection abilities of <em>H. pylori</em>.</p>
<figure class="fig xbox font-sm" id="f0006"><h4 class="obj_head">Figure 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3251/12377144/6fc69c22a72a/KVIR_A_2548620_F0006_OC.jpg" loading="lazy" height="507" width="800" alt="Figure 6."></p>
<div class="p text-right font-secondary"><a href="figure/f0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effects of the <em>HP0858</em> gene-disrupted mutation on bacterial virulence. (a) Morphological changes of the AGS cells infected by different <em>H. pylori</em> strains. The morphological changes of the AGS cells infected by various <em>H. pylori</em> strains were observed by phase-contrast microscopy with 200× magnification. The AGS cells displaying an elongated character (hummingbird phenotype) were indicated by arrows (scale bar = 200 µm). (b) Quantification of the AGS cells with the elongated phenotype after <em>H. pylori</em> infection. The percentage of elongated cells was quantified from 10 different 0.25 mm<sup>2</sup> fields. The data shown were obtained from triplicate tests with statistical analysis (unpaired, two-tailed Student’s t-test; **<em>p</em> &lt; 0.01). (c) Adhesion of various <em>H. pylori</em> strains to AGS cells during bacterial infection. Various <em>H. pylori</em> strains were co-incubated with AGS cells at an MOI (multiplicity of infection) of 100. The adhesion of different mutants assayed by the viable CFU counting method was compared to that of 26,695 strain WT to acquire the relative percentage of bacterial adhesion. The data shown were obtained from triplicate tests with statistical analysis (unpaired, two-tailed Student’s t-test; ***<em>p</em> &lt; 0.001). (d) Internalization of various <em>H. pylori</em> strains into AGS cells during bacterial infection. Various <em>H. pylori</em> strains in the logarithmic phase (cultured for 27 hours) and stationary phase (cultured for 48 hours) were co-incubated with AGS cells at an MOI of 100. Using the viable CFU counting method, the internalization efficiency of each strain was assessed and compared to that of the 26,695 strain WT in the logarithmic phase to determine the relative percentage of bacterial internalization. The data presented were derived from triplicate trials with statistical analysis (unpaired, two-tailed Student’s t-test; ns (non-significant) <em>p</em> &gt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001, ****<em>p</em> &lt; 0.0001). (e) The amount of IL-8 production from AGS cells after being infected by various <em>H. pylori</em> strains. The data presented were derived from triplicate trials with statistical analysis (unpaired, two-tailed Student’s t-test; ***<em>p</em> &lt; 0.001). WT, wild-type strain; KO, knockout mutant; Com, complemented strain.</p></figcaption></figure><p><em>H. pylori’</em>s adhesion to gastric epithelial cells is crucial in successfully colonizing the gastric mucosa. The difference in the adhesion capabilities of <em>H. pylori</em> WT and the LPS core-disrupted <em>HP0858</em> knockout mutant toward AGS cells was investigated. The findings revealed that approximately 50.67% of the WT <em>H. pylori</em> inoculum adhered to AGS cells following a 6-h infection period using stationary-phase bacteria cultured for 48 h (data not shown). Under the same experimental conditions, the <em>HP0858</em> knockout mutant exhibited a significantly reduced adhesion rate compared to the WT strain (<a href="#f0006" class="usa-link">Figure 6(c)</a>). In contrast, the adhesion level of its corresponding complemented strain to AGS cells was restored to the WT level. This finding indicates that the heptose biosynthesis-related HP0858 plays a role in the adhesion capabilities of this pathogenic bacterium. On the other hand, <em>H. pylori</em> is considered an extracellular pathogen and does not invade host cells during bacterial infection. However, some evidence suggests that a limited number of these bacteria can still enter host cells under certain circumstances [<a href="#cit0041" class="usa-link" aria-describedby="cit0041">41</a>]. Therefore, the effect of the LPS core-related <em>HP0858</em> mutation on the ability of <em>H. pylori</em> to invade gastric cells was also tested here. When stationary-phase bacteria (cultured for 48 h) were tested, approximately 0.009% of the WT <em>H. pylori</em> inoculum was internalized into AGS cells after 6 h of infection (data not shown). Under the same conditions, the <em>HP0858</em> knockout mutant exhibited a significantly lower percentage of internalization (<a href="#f0006" class="usa-link">Figure 6(d)</a>). However, the invasion rate of the <em>HP0858</em> complemented strain was restored to the WT level. Interestingly, when logarithmic-phase bacteria (cultured for 27 h) were used in the internalization assay, the <em>HP0858</em> knockout mutant again exhibited a pronounced decrease in internalization efficiency compared to the WT strain, mirroring the trend observed with stationary-phase bacteria. Two notable observations emerged from this experiment: (1) the WT strain in the logarithmic phase demonstrated approximately a 6-fold increase in internalization (about 0.053% of the initial inoculum) relative to the stationary phase (approximately 0.009%); and (2) the complemented strain surpassed the WT by 64% in internalization efficiency (0.087% vs. 0.053% of the original inoculum) under logarithmic-phase conditions. While both findings reached statistical significance, their biological relevance may be limited given the overall low internalization levels observed under these experimental conditions.</p>
<p>Given the significant impact of <em>HP0858</em> disruption on <em>H. pylori</em> adhesion, it was pertinent to investigate its effect on interleukin-8 (IL-8) secretion – a response primarily dependent on direct contact between <em>H. pylori</em> and gastric epithelial cells. Moreover, prior studies have identified ADP-heptose as a novel pathogen-associated molecular pattern (PAMP) in Gram-negative bacteria, capable of stimulating IL-8 production in human epithelial cells [<a href="#cit0019" class="usa-link" aria-describedby="cit0019">19</a>,<a href="#cit0042" class="usa-link" aria-describedby="cit0042">42</a>]. Accordingly, we compared IL-8 secretion levels in AGS cells infected with either the <em>H. pylori</em> WT strain or the <em>HP0858</em> knockout mutant. As illustrated in <a href="#f0006" class="usa-link">Figure 6(e)</a>, the <em>HP0858</em> mutant elicited significantly lower IL-8 secretion than the WT strain. This finding strongly suggests that the reduced IL-8 response is primarily attributable to the adhesion deficiency caused by <em>HP0858</em> disruption. Additionally, it is plausible that intermediates in the HP0858-associated heptose biosynthesis pathway also contribute to IL-8 induction in gastric epithelial cells. Collectively, these results indicate that impairment of the HP0858-involved heptose biosynthetic pathway markedly compromises key virulence traits of <em>H. pylori</em>.</p></section><section id="s0003-s2006"><h3 class="pmc_sec_title">
<em>HP0858</em> gene-disrupted mutation attenuates <em>H.</em>
<em>pylori</em> virulence in the <em>G.</em>
<em>mellonella</em> infection model</h3>
<p>The <em>G. mellonella</em> (greater wax moth larvae) infection model has emerged as a valuable tool in microbiology research for assessing microbial virulence, including <em>H. pylori</em>. These larvae possess an immune system that shares key features with the human innate immune system, such as cellular and humoral responses [<a href="#cit0043" class="usa-link" aria-describedby="cit0043">43</a>]. In this study, <em>G. mellonella</em> larvae were utilized as an <em>in vivo</em> infection model to explore a distinct aspect of how HP0858 disruption influences the host interactions compared to previously studied epithelial cell models. Initially, a range of bacterial dosages (10<sup>4</sup>, 10<sup>5</sup>, 10<sup>6</sup>, and 10<sup>7</sup> CFUs/larva) of the 26,695 WT strain was injected into <em>G. mellonella</em> larvae. The infected larvae were then incubated at 37 °C for up to 4 days to determine the optimal dosage that could cause more than 50% larval mortality [<a href="#cit0023" class="usa-link" aria-describedby="cit0023">23</a>]. The larvae were considered dead if they did not show a noticeable response to external stimuli. As displayed in <a href="#f0007" class="usa-link">Figure 7(a)</a>, the <em>H. pylori</em> 26,695 WT strain caused larval death in a time- and dose-dependent manner. No mortality was found in the control group’s larvae, which were only injected with PBS buffer alone. At the dosage of 1 × 10<sup>7</sup> CFUs/larva, approximately 30% of the larvae still survived after the first day of post-infection, and no larvae were alive after the third day of post-infection, which clearly showed the difference in virulence potential compared to that of other concentrations used in the test (<em>p</em> &lt; 0.0001). Therefore, the bacterial dosage of 1 × 10<sup>7</sup> CFUs/larva was chosen as the optimal dosage of injected bacteria for the following experiments to evaluate the effect of the <em>HP0858</em>-disrupted mutation on the virulence of <em>H. pylori</em> infection. As shown in <a href="#f0007" class="usa-link">Figure 7(b)</a>, larvae infected with the <em>HP0858</em> gene-disrupted mutant showed a 70% survival rate after the first day of post-infection. The survival rate was slightly reduced to 60% and 50% after the second and the fourth days of post-infection, respectively, significantly lower than those observed in larvae infected with the 26,695 WT strain and the <em>HP0858</em> complemented strain. As control, no mortality was observed in <em>G. mellonella</em> larvae injected with PBS during the experiment.
</p>
<figure class="fig xbox font-sm" id="f0007"><h4 class="obj_head">Figure 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3251/12377144/76bb51d7103a/KVIR_A_2548620_F0007_OC.jpg" loading="lazy" height="227" width="800" alt="Figure 7."></p>
<div class="p text-right font-secondary"><a href="figure/f0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Survival curves of <em>G. mellonella</em> larvae after infection with various <em>H. pylori</em> strains. (a) Kaplan-meier survival curves of <em>G. mellonella</em> larvae following infection with increasing concentrations of <em>H. pylori</em> 26,695 strain WT. <em>G. mellonella</em> larvae were injected with 1 × 10<sup>4</sup>, 1 × 10<sup>5</sup>, 1 × 10<sup>6</sup>, and 1 × 10<sup>7</sup> CFUs/larva and incubated at 37°C for up to 4 days to obtain Kaplan-meier survival curves. (b) Kaplan-meier survival curves of <em>G. mellonella</em> larvae following infection with various <em>H. pylori</em> strains. <em>G. mellonella</em> larvae were injected with 1 × 10<sup>7</sup> CFUs/larva of the 26,695 strain WT, the <em>HP0858</em> gene-disrupted mutant and the <em>HP0858</em> complemented strain and incubated at 37°C up to 4 days to obtain Kaplan-Meier survival curves. The data shown represent the percentage of survival rate from three independent experiments. Differences in survival curves were calculated using the log-rank (mantel-cox) test for multiple comparisons. Differences were considered statistically significant at ****<em>p</em> &lt; 0.0001, **<em>p</em> &lt; 0.01, *<em>p</em> &lt; 0.05, ns: no significant difference. PBS, phosphate-buffered saline; WT, wild-type strain; KO, knockout mutant; Com, complemented strain.</p></figcaption></figure><p>Next, the bacterial load in the hemolymph of <em>Galleria mellonella</em> larvae was evaluated at 0, 12, 24, 48, and 72 h following a 30-min post-injection period with the various <em>H. pylori</em> strains, using the viable CFU quantification. As detailed in <a href="#t0004" class="usa-link">Table 4</a>, viable bacteria were successfully recovered from the hemolymph of larvae infected with all tested <em>H. pylori</em> strains at the initial time (30 min post-injection). Across all strains, a gradual decline in viable bacterial load was observed over time compared to the original inoculum. Notably, viable bacterial loads for larvae injected with the 26,695 strain WT and the <em>HP0858</em> complemented strain remained detectable at 72 h post-injection. In stark contrast, no viable bacteria were recovered from larvae injected with the <em>HP0858</em> gene-disrupted mutant as early as 48 h post-infection. Furthermore, no bacterial colonies were detected in larvae injected with PBS. These results suggest that the <em>HP0858</em> gene-disrupted mutation caused attenuation of bacterial virulence in the <em>G. mellonella</em> infection model.</p>
<section class="tw xbox font-sm" id="t0004"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Viable counts (CFUs; means ±SD) of various <em>H. pylori</em> strains in the hemolymph of <em>G. mellonella</em> at 0, 12, 24, 48, and 72 h post-infection.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="center" span="1">
<col width="1*" align="center" span="1">
<col width="1*" align="center" span="1">
<col width="1*" align="center" span="1">
<col width="1*" align="center" span="1">
<col width="1*" align="center" span="1">
</colgroup>
<thead><tr>
<th align="left" colspan="1" rowspan="1">Strains</th>
<th align="center" colspan="1" rowspan="1">Ti<sup>2</sup>
</th>
<th align="center" colspan="1" rowspan="1">T0<sup>1</sup>
</th>
<th align="center" colspan="1" rowspan="1">T12</th>
<th align="center" colspan="1" rowspan="1">T24</th>
<th align="center" colspan="1" rowspan="1">T48</th>
<th align="center" colspan="1" rowspan="1">T72</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">PBS (1Χ)</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="center" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WT</td>
<td align="center" colspan="1" rowspan="1">1.6 (±0.6) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0001"><math id="MATH0001"><mrow><msup><mn>10</mn><mn>7</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">1.4 (±0.6) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0002"><math id="MATH0002"><mrow><msup><mn>10</mn><mn>7</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">1.8 (±0.7) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0003"><math id="MATH0003"><mrow><msup><mn>10</mn><mn>6</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">2.9 (±1.4) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0004"><math id="MATH0004"><mrow><msup><mn>10</mn><mn>4</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">1.4 (±0.6) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0005"><math id="MATH0005"><mrow><msup><mn>10</mn><mn>4</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">1.7 (±0.6) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0006"><math id="MATH0006"><mrow><msup><mn>10</mn><mn>1</mn></msup></mrow></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KO 0858</td>
<td align="center" colspan="1" rowspan="1">1.5 (±0.3) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0007"><math id="MATH0007"><mrow><msup><mn>10</mn><mn>7</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">1.1 (±0.1) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0008"><math id="MATH0008"><mrow><msup><mn>10</mn><mn>7</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">2.3 (±0.6) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0009"><math id="MATH0009"><mrow><msup><mn>10</mn><mn>3</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">1.7 (±0.6) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0010"><math id="MATH0010"><mrow><msup><mn>10</mn><mn>1</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">0</td>
<td align="center" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Com 0858</td>
<td align="center" colspan="1" rowspan="1">1.5 (±0.6) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0011"><math id="MATH0011"><mrow><msup><mn>10</mn><mn>7</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">1.3 (±0.6) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0012"><math id="MATH0012"><mrow><msup><mn>10</mn><mn>7</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">1.7 (±0.4) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0013"><math id="MATH0013"><mrow><msup><mn>10</mn><mn>4</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">1.9 (±0.7) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0014"><math id="MATH0014"><mrow><msup><mn>10</mn><mn>3</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">1.7 (±0.6) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0015"><math id="MATH0015"><mrow><msup><mn>10</mn><mn>3</mn></msup></mrow></math></span>
</td>
<td align="center" colspan="1" rowspan="1">0.7 (±0.6) × <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ilm0016"><math id="MATH0016"><mrow><msup><mn>10</mn><mn>1</mn></msup></mrow></math></span>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/t0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tfn0007">
<p>Ti is the original time for larvae infection</p>
<p><sup>1</sup>Time 0 corresponds to 30 min after larvae infection.</p>
</div></div></section></section><section id="s0003-s2007"><h3 class="pmc_sec_title">The <em>HP0858</em> gene-encoded enzyme is involved in the glycosylation process of adhesin AlpA in the <em>H.</em>
<em>pylori</em> 26,695 strain</h3>
<p>We recently demonstrated that several major adhesins, including AlpA in the <em>H. pylori</em> 26,695 strain, are modified with glycans, and the glycosylation process of these adhesins is associated with a general protein glycosylation system, which partially shares the enzymes from the LPS biosynthetic pathway [<a href="#cit0044" class="usa-link" aria-describedby="cit0044">44</a>]. We therefore tested if the gene-disrupted mutation in <em>HP0858</em> would affect the glycosylation of AlpA, a well-known adhesin of <em>H. pylori</em>. As illustrated in <a href="#f0008" class="usa-link">Figure 8(a)</a>, numerous proteins are present in the outer membrane (OM) fractions, displaying unique profiles that differ noticeably from those observed in the cytoplasm/periplasm (C/P) and the inner membrane (IM) fractions. This distinction indicates that the subcellular fractionation process was carried out effectively. Interestingly, the electrophoretic mobility of AlpA indeed displayed an observable two-step molecular weight upshift pattern first from the cytoplasm to the inner membrane and then from the inner membrane to the outer membrane, a typical phenomenon for glycosylated adhesins in <em>H. pylori</em>, which was observed both in WT and the <em>HP0858</em> complemented strains (<a href="#f0008" class="usa-link">Figure 8(b)</a>). In contrast, this molecular weight upshift phenomenon disappeared in the <em>HP0858</em> knockout mutant. This observation supports our recent finding and suggests that the <em>HP0858</em> gene-encoded enzyme is also a component of the general protein glycosylation system just discovered in <em>H. pylori</em>.
</p>
<figure class="fig xbox font-sm" id="f0008"><h4 class="obj_head">Figure 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3251/12377144/e211e64d78b3/KVIR_A_2548620_F0008_OC.jpg" loading="lazy" height="1175" width="800" alt="Figure 8."></p>
<div class="p text-right font-secondary"><a href="figure/f0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effects of the <em>HP0858</em> gene-disrupted mutation on the glycosylation of adhesin AlpA. (a) The protein profiles of the subcellular fraction samples (10 μg/lane) from the tested bacterial strains were analyzed by staining with 0.25% coomassie brilliant blue R250. (b) The electrophoretic mobility of adhesin AlpA in different subcellular fractions from the tested bacterial strains was analyzed by immunoblotting analysis with anti-AlpA antibody. Protein samples were collected from the cytoplasm/periplasm (C/P, 80 μg), the inner membrane (IM, 60 μg), and the outer membrane (OM, 10 μg). The result of ponceau S staining was used as the loading control for comparison. WT, wild-type strain; KO, knockout mutant; Com, complemented strain; m, the protein marker used was the AccuRuler RGB prestained protein ladder (10–180 kDa; catalog number 02101–250), supplied by MaestroGen Inc. (Hsinchu city, Taiwan).</p></figcaption></figure></section></section><section id="s0004"><h2 class="pmc_sec_title">Discussion</h2>
<p>Although most of the <em>H. pylori</em> enzymes associated with the biosynthesis of lipid A and O-antigen have been identified and characterized, some of the enzymes involved in the biosynthesis of the core region of LPS remain to be explored [<a href="#cit0020" class="usa-link" aria-describedby="cit0020">20</a>]. Our group previously conducted studies on the <em>HP0857</em>, <em>HP0859</em>, and <em>HP0860</em> genes and demonstrated that they all play essential roles in the heptose biosynthetic pathway for constructing the core region of <em>H. pylori</em> LPS [<a href="#cit0025" class="usa-link" aria-describedby="cit0025">25–27</a>]. In this study, the remaining <em>HP0858</em> gene in the gene cluster for heptose biosynthesis was characterized by several molecular techniques.</p>
<p>The sequence analysis of the HP0858 protein compared to several Gram-negative bacterial homologs indicates that HP0858 is highly similar to the members of the RfaE superfamily that are known to possess both kinase and adenylyltransferase activities required for heptose biosynthesis. Here we could show that the <em>HP0858</em> mutant not only exhibits a severe truncated LPS structure in the core region but also displays a complete loss of Lewis antigens, proving that the encoded enzyme is indeed involved in heptose biosynthesis and thus is critical for the formation of <em>H. pylori</em> LPS. In addition, this mutant increases the sensitivity to detergent SDS and antibiotic novobiocin, supporting the well-recognized concept that the LPS core plays an essential role in providing a protective barrier to restrict the entrance of toxic hydrophobic compounds like bile salts, detergents, lipophilic antibiotics, and other poisonous substances that might harm the bacterium [<a href="#cit0045" class="usa-link" aria-describedby="cit0045">45</a>,<a href="#cit0046" class="usa-link" aria-describedby="cit0046">46</a>].</p>
<p>Specific LPS metabolites derived from heptose biosynthesis are acknowledged as novel pathogen-associated molecular patterns (PAMPs) identified in Gram-negative bacteria [<a href="#cit0047" class="usa-link" aria-describedby="cit0047">47</a>]. Instead of the initially recognized HBP (the product of the second enzymatic step in the ADP-heptose biosynthetic pathway) [<a href="#cit0016" class="usa-link" aria-describedby="cit0016">16–18</a>], ADP-heptose, the fourth-step product of the pathway as mentioned above, is currently recognized as the predominant PAMP of <em>H. pylori</em> in the activation of the NF-κB pathway in human epithelial and phagocytic cells through the ALPK1-TIFA axis and thus induce the secretion of the proinflammatory cytokine IL-8 [<a href="#cit0019" class="usa-link" aria-describedby="cit0019">19</a>,<a href="#cit0048" class="usa-link" aria-describedby="cit0048">48</a>]. In addition to triggering proinflammatory signal transduction in host cells, the induction of NF-κB signaling by ADP-heptose sensing during <em>H. pylori</em> infection has also been reported to trigger RNA-loop-mediated DNA replication stress and promote DNA double-strand breaks (DSBs) [<a href="#cit0049" class="usa-link" aria-describedby="cit0049">49</a>]. The above genotoxic stress, joined with erroneous DNA repair in infected cells, can increase genome instability and thus promote gastric cancer development. The current study, consistent with earlier findings [<a href="#cit0019" class="usa-link" aria-describedby="cit0019">19</a>], reveals that disrupting <em>HP0858</em> expression, which impairs ADP-heptose production, significantly decreases IL-8 secretion in AGS cells during infection. However, our team also observed that AGS cells infected with an <em>H. pylori</em> mutant lacking <em>HP0860</em> expression still produced notable amounts of IL-8 [<a href="#cit0027" class="usa-link" aria-describedby="cit0027">27</a>]. We explained this unexpected outcome by proposing that other phosphatase activities within <em>H. pylori</em> may partially compensate for the loss of <em>HP0860</em> expression in the mutant strain.</p>
<p>Hauke et al. recently demonstrated that, compared to other genes involved in the biosynthesis of heptose metabolites, the <em>RfaE/HldE/HP0858</em> gene and its corresponding enzyme exhibit notable sequence variability and expression differences among <em>H. pylori</em> strains [<a href="#cit0050" class="usa-link" aria-describedby="cit0050">50</a>]. They further revealed that the heptose gene cluster’s expression varies within and between strains, influenced by factors such as host cell contact, the presence of the <em>cag</em> pathogenicity island (<em>cag</em>PAI), and the carbon starvation regulator A (CsrA). Interestingly, using an approach that reconstituted the heptose biosynthetic pathway and analyzed the resulting metabolites both <em>in vitro</em> and in <em>H. pylori</em> lysates, the researchers identified a third heptose metabolite – likely heptose-1-monophosphate (HMP-1) – in addition to ADP-heptose and HBP. This metabolite was characterized as a novel proinflammatory compound produced by <em>H. pylori</em> [<a href="#cit0050" class="usa-link" aria-describedby="cit0050">50</a>]. RfaE/HldE/HP0858 is a bifunctional enzyme comprising two domains, d1 and d2, responsible for kinase activity and adenylyltransferase activity, respectively, which are essential for the second and fourth steps in the heptose biosynthetic pathway. Unexpectedly, the same group found that the d1 domain also exhibits phosphatase activity <em>in vitro</em>, enabling it to produce HMP-1 during the first two steps of heptose biosynthesis without requiring the third-step enzyme GmhB/HP0860. This discovery provides a possible explanation for the observation that AGS cells infected with the HP0860 knockout mutant can still secrete significant levels of IL-8.</p>
<p>Our study observed a notable increase in surface hydrophobicity in the <em>HP0858</em> knockout mutant, accompanied by a tendency for the bacteria to autoaggregate. Autoaggregation can introduce variability in OD<sub>600</sub> readings, potentially distorting their reflection of bacterial growth. To address this, we reevaluated the growth curve using the CFU counting method along with OD<sub>600</sub> measurements. The results were then compared using semi-logarithmic graphs. As shown in Supplementary Figure S2, the absence of data collection during the first 12 h of culture prevented the identification of a clear lag phase in both growth curves. Interestingly, the two methods yielded similar growth trends during the stationary phase (36–48 h) and death phase (48–84 h) but diverged significantly in the logarithmic phase. The OD<sub>600</sub>-based growth curve displayed a steep, linear increase, whereas the CFU-derived curve was more level. Moreover, during this stage, the OD<sub>600</sub> measurements revealed more pronounced differences between the WT strain and the <em>HP0858</em> knockout mutant than the CFU method. It is worth noting that CFU counts may underestimate viable cell numbers when cells experience stress, while OD<sub>600</sub> readings might overestimate growth by including non-viable cells. However, the observed discrepancies suggest that the autoaggregation in the <em>HP0858</em> knockout mutant could lead to reduced OD<sub>600</sub> readings, resulting in an underestimation of its growth. Although statistical analysis shows that the mutant grows slightly slower than the WT strain at specific time points, these differences are unlikely to carry significant biological implications.</p>
<p>Beyond the direct effects outlined in the main text, the autoaggregation phenotype resulting from <em>HP0858</em> disruption could also indirectly influence specific findings examined in this study. Aggregating populations of the <em>HP0858</em> knockout mutant are expected to be less efficient at dispersing, which could diminish their ability to colonize and adhere to host epithelial cells effectively. This reduced dispersion may lead to localized bacterial clustering, limiting the bacteria’s capacity to spread across the gastric mucosa. Consequently, this can decrease the efficiency of bacterial adherence and invasion, which is crucial for establishing infection and initiating host-pathogen interactions. Without efficient adherence, key virulence factors such as adhesins (e.g. BabA and SabA) may not optimally engage with host receptors, thereby reducing the infection’s overall pathogenic impact. Furthermore, a reduced direct interaction due to bacterial aggregation may impair CagA translocation through the T4SS, dampening the morphological changes typically induced by this effector protein. The T4SS activity is also crucial for the induction of pro-inflammatory cytokine IL-8 via NF-κB activation. Aggregation, by reducing T4SS functionality, may lead to lower levels of IL-8 production, potentially moderating the inflammatory response.</p>
<p>Consistent with the findings for the <em>HP0858</em> knockout mutant in this study, our previous research demonstrated that disrupting other heptose biosynthesis-related genes (<em>HP0857</em> (coding for an isomerase), <em>HP0859</em> (coding for an epimerase), and <em>HP0860</em> (coding for a phosphatase)) not only truncates the LPS structure, thereby eliminating the presence of Lewis X and/or Lewis Y antigens, but also reduces specific phenotypes such as adhesion, antibiotic and detergent resistance, and virulence [<a href="#cit0025" class="usa-link" aria-describedby="cit0025">25–27</a>]. This alignment suggests that the phenotypic changes observed in this study are not exclusive to HP0858 but are shared among other enzymes involved in heptose biosynthesis. Interestingly, we recently identified similar phenotypic defects, including increased surface hydrophobicity and autoaggregation, in mutants with disruptions in <em>HP0044</em> and <em>HP1275</em> [<a href="#cit0051" class="usa-link" aria-describedby="cit0051">51</a>]. These genes encode enzymes critical for fucose biosynthesis. Fucose is a key component of the Lewis X and Lewis Y antigens in the LPS O-antigen of <em>H. pylori</em>. Furthermore, fucose is part of a conserved trisaccharide (-GlcNAc-Fuc-DD-Hep-) known as “Trio,” which connects the core oligosaccharide to the most recently redefined O-antigen structure. Based on these findings, it is plausible to hypothesize that the absence of Lewis X/Lewis Y antigens, or more broadly, the disruption of the O-antigen structure, underlies the phenotypic defects observed in these <em>H. pylori</em> mutants.</p>
<p>Glycosylation is a type of post-translational modification that plays diverse roles in protein function and contributes to the complexity of a cell’s proteome. In Gram-negative bacteria, an increasing number of glycoproteins have been identified, many of which act as virulence factors. For instance, AIDA-I, an adhesin involved in the diffuse adherence of diarrhea-causing <em>E. coli</em> strain 2787 (O126:H27), has been recognized as a glycoprotein [<a href="#cit0052" class="usa-link" aria-describedby="cit0052">52</a>]. Benz and Schmidt revealed that AIDA-I is modified with heptose residues, and its glycosylated form significantly enhances bacterial adherence to human cell lines. In contrast, the non-glycosylated version lacks this ability. Recently, we demonstrated that <em>H. pylori</em> possesses a general protein glycosylation system responsible for modifying several key adhesins. This modification system is crucial for ensuring adhesins’ stability and protease resistance and enhancing their binding capabilities [<a href="#cit0044" class="usa-link" aria-describedby="cit0044">44</a>]. The enzymes linked to this newly discovered system partially overlap with those involved in the biosynthesis of LPS. In this research, we identified a connection between LPS biosynthesis and the potential glycosylation of the adhesin AlpA, mediated by a sugar motif containing heptose residues or their derivatives. Based on this relationship, it is proposed that AlpA undergoes glycosylation through a general protein glycosylation pathway associated with LPS biosynthesis, with HP0858 being one of the enzymes involved. The <em>HP0858</em>–disrupted mutant reduces bacterial adhesion and internalization abilities when infecting AGS cells, and the infected AGS cells show a lower degree of the “elongation phenotype” compared to those infected with the <em>H. pylori</em> 26,695 WT strain. These altered characteristics in the HP0858-deficient mutant can be attributed to two potential mechanisms: (1) Direct impact: LPS functions as an adhesion molecule in <em>H. pylori</em>, and disruption of heptose biosynthesis compromises the structural integrity of LPS, significantly impairing the bacterium’s adhesion ability and virulence; (2) Indirect impact: Loss of HP0858 expression may prevent the glycosylation of key adhesins like AlpA, hindering <em>H. pylori</em> from effectively binding to host cells and delivering virulence factors such as CagA. These two mechanisms are not mutually exclusive and may collectively contribute to the observed phenotypic changes.</p>
<p>Notably, the molecular size of AlpA exhibited a distinct two-step upshift as it transitioned from the cytoplasm to the inner membrane and eventually to the outer membrane of <em>H. pylori</em>. This progressive size increase suggests that the glycosylation of AlpA likely begins during or shortly after its transport across the inner membrane into the periplasm and is completed before its integration into the outer membrane. Based on this observation, it is reasonable to infer that AlpA in the outer membrane is fully glycosylated. In contrast, the form detected in the inner membrane represents a partially glycosylated intermediate state. However, an alternative explanation cannot be ruled out entirely: a secondary glycosylation system located within the periplasm might finalize the glycosylation process for AlpA. Further research is required to confirm this possibility.</p>
<p>In this study, we utilized <em>Galleria mellonella</em> (greater wax moth) larvae as an <em>in vivo</em> infection model to assess the impact of <em>HP0858</em> disruption on <em>H. pylori</em> virulence by evaluating bacterial viability within the larval hemocoel. Unlike mammalian hosts, <em>G. mellonella</em> larvae have a protective cuticle analogous to mammalian skin, serving as a physical barrier. When the cuticle is breached, the larvae activate their humoral innate immune response, producing soluble defense compounds such as antimicrobial peptides, lytic enzymes, and opsonins. Simultaneously, the cellular innate immune system is stimulated, enhancing the larvae’s ability to combat invading pathogens [<a href="#cit0053" class="usa-link" aria-describedby="cit0053">53</a>]. The proliferation of hemocytes triggers cellular immune responses, including phagocytosis, nodulation, and melanogenesis, providing a robust defense mechanism [<a href="#cit0023" class="usa-link" aria-describedby="cit0023">23</a>,<a href="#cit0054" class="usa-link" aria-describedby="cit0054">54</a>]. In general, hemocytes in the hemocoel engulf invading microorganisms, similar to the role of mammalian neutrophils, and release respiratory burst products, such as free radicals and peroxides. These compounds contribute to tissue damage, mirroring the immune response seen in humans [<a href="#cit0055" class="usa-link" aria-describedby="cit0055">55</a>]. Additionally, melanogenesis, evidenced by observable color changes in infected larvae, reflects the immune system’s response to bacterial presence in the gut or body cavity. This process is thought to protect endogenous tissues from damage caused by pathogen elimination. Our findings demonstrated that <em>H. pylori</em> virulence in larvae is both time- and dose-dependent. Notably, the <em>HP0858</em>-disrupted mutant exhibited significantly reduced virulence compared to the WT strain 26,695 and the <em>HP0858</em> complemented strain (<a href="#f0007" class="usa-link">Figure 7(b)</a>). Overall, these results suggest that <em>G. mellonella</em> larvae provide a valuable <em>in vivo</em> model for investigating the role of LPS biosynthesis in <em>H. pylori</em> virulence.</p></section><section id="s0005"><h2 class="pmc_sec_title">Conclusion</h2>
<p>We demonstrated that the enzyme encoded by the <em>HP0858</em> gene plays a critical role in heptose biosynthesis. Disruption of this gene in <em>H. pylori</em> results in a truncated LPS structure, lacking the complete O-antigen and much of the core region. The resulting mutant exhibited hypersensitivity to the detergent SDS and the antibiotic novobiocin, increased surface hydrophobicity, and a tendency to autoaggregate. Additionally, the <em>HP0858</em> knockout mutant showed a significant reduction in its ability to adhere to and internalize gastric AGS cells. Infected AGS cells also exhibited a diminished hummingbird phenotype and a marked decrease in IL-8 secretion. Furthermore, the mutant’s reduced virulence was confirmed using the <em>Galleria mellonella</em> wax moth as an in vivo <em>H. pylori</em> infection model. The pivotal role of RfaE/HldE/HP0858 in bacterial fitness and virulence makes it a promising target for the future development of novel preventive and therapeutic strategies against <em>H. pylori</em> infections.</p></section><section id="sec27"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="SM6948"><div class="caption p"><span>Supplementary Fig 2_2.jpg</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377144/bin/KVIR_A_2548620_SM6948.jpg" data-ga-action="click_feat_suppl" class="usa-link">KVIR_A_2548620_SM6948.jpg</a><sup> (267.2KB, jpg) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM6947"><div class="caption p"><span>HP0858_paper_Supplementary_material_20241230 (1).docx</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377144/bin/KVIR_A_2548620_SM6947.docx" data-ga-action="click_feat_suppl" class="usa-link">KVIR_A_2548620_SM6947.docx</a><sup> (58.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM6946"><div class="caption p"><span>Supplementary Fig 1_2.jpeg</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377144/bin/KVIR_A_2548620_SM6946.jpeg" data-ga-action="click_feat_suppl" class="usa-link">KVIR_A_2548620_SM6946.jpeg</a><sup> (117.7KB, jpeg) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Prof. Rainer Haas from Medical Microbiology and Hospital Epidemiology, Max von Pettenkofer Institute, Faculty of Medicine, Ludwig-Maximilians-University Munich, Munich, Germany, for kindly providing the AlpA-specific antibody used in the immunoblotting analyses. We acknowledge the use of ChatGPT (version GPT-4o, OpenAI, <a href="https://chatgpt.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://chatgpt.com/</a>) for language improvement during the editing process of preparing the revised manuscript.</p>
<p>Conceptualization, M.-C.K. and P.-C.W.; methodology, T.T.H.N., P.-C.C., W.-H.W, and C.-Y.L.; validation, T.T.H.N., P.-C.C. and M.-C.K.; formal analysis, T.T.H.N. and P.-C.C.; investigation, T.T.H.N., P.-C.C. and W.-H.W.; resources, M.-C.K., C.-Y.L. and P.-C.W.; data curation, T.T.H.N., P.-C.C., S.B. and M.-C.K.; writing – original draft preparation, T.T.H.N. and P.-C.C.; writing – review and editing, M.-C.K., S.B. and P.-C.W.; supervision, M.-C.K.; funding acquisition, M.-C.K., P.-C.W. and S.B. All authors have read and agreed to the published version of the manuscript.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>The work of M.-C.K. was supported by the Ministry of Science and Technology, Taiwan (MOST) under Grant number [MOST 111–2311-B-007–006], National Science and Technology Council (NSTC) under Grant number [NSTC 113–2320-B-007–002], and National Tsing Hua University under Grant number [111Q2505E1], Grant [number 112Q2506E1] and Grant number [114QF033E1]. The work of P.-C.W.was supported by Hsinchu MacKay Memorial Hospital, Taiwan under Grant number [MMH-TH-114–04]. The work of S.B.is funded by the German Science Foundation (DFG) grants [BA 1671/15–1] and [BA 1671/16–1].</p></section><section id="s0006"><h2 class="pmc_sec_title">Disclosure statement</h2>
<p>No potential conflict of interest was reported by the author(s).</p></section><section id="s0007"><h2 class="pmc_sec_title">Data availability statement</h2>
<p>The data supporting this study’s findings are openly available in Figshare at <a href="https://doi.org/10.6084/m9.figshare.28081937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.6084/m9.figshare.28081937</a></p></section><section id="s0008"><h2 class="pmc_sec_title">Supplemental data</h2>
<p>Supplemental data for this article can be accessed online at <a href="https://doi.org/10.1080/21505594.2025.2548620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1080/21505594.2025.2548620</a></p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="cit0001">
<span class="label">[1].</span><cite>Morris AJ, Ali MR, Nicholson GI, et al. Long-term follow-up of voluntary ingestion of <em>Helicobacter pylori</em>. Ann Intern Med. 1991;114(8):662–20. doi: 10.7326/0003-4819-114-8-662
</cite> [<a href="https://doi.org/10.7326/0003-4819-114-8-662" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2003713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&amp;title=Long-term%20follow-up%20of%20voluntary%20ingestion%20of%20Helicobacter%20pylori&amp;volume=114&amp;issue=8&amp;publication_year=1991&amp;pages=662-20&amp;pmid=2003713&amp;doi=10.7326/0003-4819-114-8-662&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0002">
<span class="label">[2].</span><cite>Blaser MJ, Perez-Perez GI, Kleanthous H, et al. Infection with <em>Helicobacter pylori</em> strains possessing cagA is associated with an increased risk of developing adenocarcinoma of the stomach. Cancer Res. 1995;55(10):2111–2115.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/7743510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Infection%20with%20Helicobacter%20pylori%20strains%20possessing%20cagA%20is%20associated%20with%20an%20increased%20risk%20of%20developing%20adenocarcinoma%20of%20the%20stomach&amp;volume=55&amp;issue=10&amp;publication_year=1995&amp;pages=2111-2115&amp;pmid=7743510&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0003">
<span class="label">[3].</span><cite>Wotherspoon AC, Diss TC, Pan L, et al. Regression of primary low-grade B-cell gastric lymphoma of mucosa-associated lymphoid tissue type after eradication of <em>Helicobacter pylori</em>. Lancet. 1993;342(8871):575–577. doi: 10.1016/0140-6736(93)91409-F
</cite> [<a href="https://doi.org/10.1016/0140-6736(93)91409-F" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8102719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Regression%20of%20primary%20low-grade%20B-cell%20gastric%20lymphoma%20of%20mucosa-associated%20lymphoid%20tissue%20type%20after%20eradication%20of%20Helicobacter%20pylori&amp;volume=342&amp;issue=8871&amp;publication_year=1993&amp;pages=575-577&amp;pmid=8102719&amp;doi=10.1016/0140-6736(93)91409-F&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0004">
<span class="label">[4].</span><cite>Matsuo Y, Kido Y, Yamaoka Y.. 
<em>Helicobacter pylori</em> outer membrane protein-related pathogenesis. Toxins (Basel). 2017;9(3):101. doi: 10.3390/toxins9030101
</cite> [<a href="https://doi.org/10.3390/toxins9030101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5371856/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28287480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Toxins%20(Basel)&amp;title=Helicobacter%20pylori%20outer%20membrane%20protein-related%20pathogenesis&amp;volume=9&amp;issue=3&amp;publication_year=2017&amp;pages=101&amp;pmid=28287480&amp;doi=10.3390/toxins9030101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0005">
<span class="label">[5].</span><cite>Li H, Yang T, Liao T, et al. The redefinition of <em>Helicobacter pylori</em> lipopolysaccharide O-antigen and core-oligosaccharide domains. PLOS Pathog. 2017;13(3):e1006280. doi: 10.1371/journal.ppat.1006280
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1006280" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5371381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28306723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Pathog&amp;title=The%20redefinition%20of%20Helicobacter%20pylori%20lipopolysaccharide%20O-antigen%20and%20core-oligosaccharide%20domains&amp;volume=13&amp;issue=3&amp;publication_year=2017&amp;pages=e1006280&amp;pmid=28306723&amp;doi=10.1371/journal.ppat.1006280&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0006">
<span class="label">[6].</span><cite>Moran A.P.
<em>et al</em>. <em>Helicobacter pylori</em> lipopolysaccharide and Lewis antigens
Yamaoka, Y.
<em>Helicobacter pylori</em>: molecular genetics and cellular biology. Poole, UK: Caister Academic Press; 2008.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Helicobacter%20pylori:%20molecular%20genetics%20and%20cellular%20biology&amp;publication_year=2008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0007">
<span class="label">[7].</span><cite>King JD, Kocíncová D, Westman EL, et al. Review: lipopolysaccharide biosynthesis in <em>Pseudomonas aeruginosa</em>. Innate Immun. 2009;15(5):261–312. doi: 10.1177/1753425909106436
</cite> [<a href="https://doi.org/10.1177/1753425909106436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19710102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Innate%20Immun&amp;title=Review:%20lipopolysaccharide%20biosynthesis%20in%20Pseudomonas%20aeruginosa&amp;volume=15&amp;issue=5&amp;publication_year=2009&amp;pages=261-312&amp;pmid=19710102&amp;doi=10.1177/1753425909106436&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0008">
<span class="label">[8].</span><cite>Pachathundikandi SK, Tegtmeyer N, Backert S. Masking of typical TLR4 and TLR5 ligands modulates inflammation and resolution by <em>Helicobacter pylori</em>. Trends Microbiol. 2023;31(9):903–915. doi: 10.1016/j.tim.2023.03.009
</cite> [<a href="https://doi.org/10.1016/j.tim.2023.03.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37012092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Microbiol&amp;title=Masking%20of%20typical%20TLR4%20and%20TLR5%20ligands%20modulates%20inflammation%20and%20resolution%20by%20Helicobacter%20pylori&amp;volume=31&amp;issue=9&amp;publication_year=2023&amp;pages=903-915&amp;pmid=37012092&amp;doi=10.1016/j.tim.2023.03.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0009">
<span class="label">[9].</span><cite>Altman E, Chandan V, Li J, et al. Lipopolysaccharide structures of <em>Helicobacter pylori</em> wild-type strain 26695 and 26695 HP0826: kan mutant devoid of the O-chain polysaccharide component. Carbohydr Res. 2011;346(15):2437–2444. doi: 10.1016/j.carres.2011.06.036
</cite> [<a href="https://doi.org/10.1016/j.carres.2011.06.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21903201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Carbohydr%20Res&amp;title=Lipopolysaccharide%20structures%20of%20Helicobacter%20pylori%20wild-type%20strain%2026695%20and%2026695%20HP0826:%20kan%20mutant%20devoid%20of%20the%20O-chain%20polysaccharide%20component&amp;volume=346&amp;issue=15&amp;publication_year=2011&amp;pages=2437-2444&amp;pmid=21903201&amp;doi=10.1016/j.carres.2011.06.036&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0010">
<span class="label">[10].</span><cite>Valkonen KH, Wadström T, Moran AP. Interaction of lipopolysaccharides of <em>Helicobacter pylori</em> with basement membrane protein laminin. Infect Immun. 1994;62(9):3640–3648. doi: 10.1128/iai.62.9.3640-3648.1994
</cite> [<a href="https://doi.org/10.1128/iai.62.9.3640-3648.1994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC303013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8063380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&amp;title=Interaction%20of%20lipopolysaccharides%20of%20Helicobacter%20pylori%20with%20basement%20membrane%20protein%20laminin&amp;volume=62&amp;issue=9&amp;publication_year=1994&amp;pages=3640-3648&amp;pmid=8063380&amp;doi=10.1128/iai.62.9.3640-3648.1994&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0011">
<span class="label">[11].</span><cite>Moran AP. <em>Helicobacter pylori</em> lipopolysaccharide-mediated gastric and extragastric pathology. J Physiol Pharmacol. 1999;50(5):787–805.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/10695559/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Physiol%20Pharmacol&amp;title=Helicobacter%20pylori%20lipopolysaccharide-mediated%20gastric%20and%20extragastric%20pathology&amp;volume=50&amp;issue=5&amp;publication_year=1999&amp;pages=787-805&amp;pmid=10695559&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0012">
<span class="label">[12].</span><cite>Altman E, Smirnova N, Li J, et al. Occurrence of a nontypable <em>Helicobacter pylori</em> strain lacking Lewis blood group O antigens and DD-heptoglycan: evidence for the role of the core 1,6-glucan chain in colonization. Glycobiology. 2003;13(11):777–783. doi: 10.1093/glycob/cwg106
</cite> [<a href="https://doi.org/10.1093/glycob/cwg106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12907691/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Glycobiology&amp;title=Occurrence%20of%20a%20nontypable%20Helicobacter%20pylori%20strain%20lacking%20Lewis%20blood%20group%20O%20antigens%20and%20DD-heptoglycan:%20evidence%20for%20the%20role%20of%20the%20core%201,6-glucan%20chain%20in%20colonization&amp;volume=13&amp;issue=11&amp;publication_year=2003&amp;pages=777-783&amp;pmid=12907691&amp;doi=10.1093/glycob/cwg106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0013">
<span class="label">[13].</span><cite>Fischer W, Püls J, Buhrdorf R, et al. Systematic mutagenesis of the <em>Helicobacter pylori</em> cag pathogenicity island: essential genes for CagA translocation in host cells and induction of interleukin-8. Mol Microbiol. 2001;42(5):1337–1348. doi: 10.1046/j.1365-2958.2001.02714.x
</cite> [<a href="https://doi.org/10.1046/j.1365-2958.2001.02714.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11886563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&amp;title=Systematic%20mutagenesis%20of%20the%20Helicobacter%20pylori%20cag%20pathogenicity%20island:%20essential%20genes%20for%20CagA%20translocation%20in%20host%20cells%20and%20induction%20of%20interleukin-8&amp;volume=42&amp;issue=5&amp;publication_year=2001&amp;pages=1337-1348&amp;pmid=11886563&amp;doi=10.1046/j.1365-2958.2001.02714.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0014">
<span class="label">[14].</span><cite>Brandt S, Kwok T, Hartig R, et al. Nf-κb activation and potentiation of proinflammatory responses by the <em>Helicobacter pylori</em> CagA protein. Proc Natl Acad Sci USA. 2005;102(26):9300–9305. doi: 10.1073/pnas.0409873102
</cite> [<a href="https://doi.org/10.1073/pnas.0409873102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1166591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15972330/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Nf-%CE%BAb%20activation%20and%20potentiation%20of%20proinflammatory%20responses%20by%20the%20Helicobacter%20pylori%20CagA%20protein&amp;volume=102&amp;issue=26&amp;publication_year=2005&amp;pages=9300-9305&amp;pmid=15972330&amp;doi=10.1073/pnas.0409873102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0015">
<span class="label">[15].</span><cite>Gaudet RG, Sintsova A, Buckwalter CM, et al. Cytosolic detection of the bacterial metabolite HBP activates TIFA-dependent innate immunity. Science. 2015;348(6240):1251–1255. doi: 10.1126/science.aaa4921
</cite> [<a href="https://doi.org/10.1126/science.aaa4921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26068852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Cytosolic%20detection%20of%20the%20bacterial%20metabolite%20HBP%20activates%20TIFA-dependent%20innate%20immunity&amp;volume=348&amp;issue=6240&amp;publication_year=2015&amp;pages=1251-1255&amp;pmid=26068852&amp;doi=10.1126/science.aaa4921&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0016">
<span class="label">[16].</span><cite>Zimmermann S, Pfannkuch L, Al-Zeer MA, et al. Alpk1- and Tifa-dependent innate immune response triggered by the <em>Helicobacter pylori</em> type IV secretion system. Cell Rep. 2017;20(10):2384–2395. doi: 10.1016/j.celrep.2017.08.039
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.08.039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28877472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Alpk1-%20and%20Tifa-dependent%20innate%20immune%20response%20triggered%20by%20the%20Helicobacter%20pylori%20type%20IV%20secretion%20system&amp;volume=20&amp;issue=10&amp;publication_year=2017&amp;pages=2384-2395&amp;pmid=28877472&amp;doi=10.1016/j.celrep.2017.08.039&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0017">
<span class="label">[17].</span><cite>Stein SC, Faber E, Bats SH, et al. <em>Helicobacter pylori</em> modulates host cell responses by CagT4SS-dependent translocation of an intermediate metabolite of LPS inner core heptose biosynthesis. PLOS Pathog. 2017;13(7):e1006514. doi: 10.1371/journal.ppat.1006514
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1006514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5531669/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28715499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Pathog&amp;title=Helicobacter%20pylori%20modulates%20host%20cell%20responses%20by%20CagT4SS-dependent%20translocation%20of%20an%20intermediate%20metabolite%20of%20LPS%20inner%20core%20heptose%20biosynthesis&amp;volume=13&amp;issue=7&amp;publication_year=2017&amp;pages=e1006514&amp;pmid=28715499&amp;doi=10.1371/journal.ppat.1006514&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0018">
<span class="label">[18].</span><cite>Gall A, Gaudet RG, Gray-Owen SD, et al. TIFA signaling in gastric epithelial cells initiates the Cag type 4 secretion system-dependent innate immune response to <em>Helicobacter pylori</em> infection. MBio. 2017;8(4). doi: 10.1128/mBio.01168-17</cite> [<a href="https://doi.org/10.1128/mBio.01168-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5559637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28811347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MBio&amp;title=TIFA%20signaling%20in%20gastric%20epithelial%20cells%20initiates%20the%20Cag%20type%204%C2%A0secretion%20system-dependent%20innate%20immune%20response%20to%20Helicobacter%20pylori%20infection&amp;volume=8&amp;issue=4&amp;publication_year=2017&amp;pmid=28811347&amp;doi=10.1128/mBio.01168-17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0019">
<span class="label">[19].</span><cite>Pfannkuch L, Hurwitz R, Trauisen J, et al. Adp heptose, a novel pathogen-associated molecular pattern identified in <em>Helicobacter pylori</em>. Faseb J. 2019;33(8):9087–9099. doi: 10.1096/fj.201802555R
</cite> [<a href="https://doi.org/10.1096/fj.201802555R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6662969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31075211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Faseb%20J&amp;title=Adp%20heptose,%20a%20novel%20pathogen-associated%20molecular%20pattern%20identified%20in%20Helicobacter%20pylori&amp;volume=33&amp;issue=8&amp;publication_year=2019&amp;pages=9087-9099&amp;pmid=31075211&amp;doi=10.1096/fj.201802555R&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0020">
<span class="label">[20].</span><cite>Li H, Liao T, Debowski AW, et al. Lipopolysaccharide structure and biosynthesis in <em>Helicobacter pylori</em>. Helicobacter. 2016;21(6):445–461. doi: 10.1111/hel.12301
</cite> [<a href="https://doi.org/10.1111/hel.12301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26934862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Helicobacter&amp;title=Lipopolysaccharide%20structure%20and%20biosynthesis%20in%20Helicobacter%20pylori&amp;volume=21&amp;issue=6&amp;publication_year=2016&amp;pages=445-461&amp;pmid=26934862&amp;doi=10.1111/hel.12301&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0021">
<span class="label">[21].</span><cite>Valvano MA, Messner P, Kosma P. Novel pathways for biosynthesis of nucleotide-activated glycero-manno-heptose precursors of bacterial glycoproteins and cell surface polysaccharides. Microbiology. 2002;148(Pt 7):1979–1989. doi: 10.1099/00221287-148-7-1979
</cite> [<a href="https://doi.org/10.1099/00221287-148-7-1979" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12101286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Novel%20pathways%20for%20biosynthesis%20of%20nucleotide-activated%20glycero-manno-heptose%20precursors%20of%20bacterial%20glycoproteins%20and%20cell%20surface%20polysaccharides&amp;volume=148&amp;issue=Pt%207&amp;publication_year=2002&amp;pages=1979-1989&amp;pmid=12101286&amp;doi=10.1099/00221287-148-7-1979&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0022">
<span class="label">[22].</span><cite>Senior NJ, Titball RW. Isolation and primary culture of <em>galleria mellonella</em> hemocytes for infection studies. F1000Res. 2020;9:1392. doi: 10.12688/f1000research.27504.1
</cite> [<a href="https://doi.org/10.12688/f1000research.27504.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7818094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33520196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=F1000Res&amp;title=Isolation%20and%20primary%20culture%20of%20galleria%20mellonella%20hemocytes%20for%20infection%20studies&amp;volume=9&amp;publication_year=2020&amp;pages=1392&amp;pmid=33520196&amp;doi=10.12688/f1000research.27504.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0023">
<span class="label">[23].</span><cite>Ochoa S, Fernández F, Devotto L, et al. Virulence assessment of enterohepatic <em>Helicobacter</em> species carried by dogs using the wax moth larvae <em>Galleria mellonella</em> as infection model. Helicobacter. 2021;26(4):e12808. doi: 10.1111/hel.12808
</cite> [<a href="https://doi.org/10.1111/hel.12808" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33884706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Helicobacter&amp;title=Virulence%20assessment%20of%20enterohepatic%20Helicobacter%20species%20carried%20by%20dogs%20using%20the%20wax%20moth%20larvae%20Galleria%20mellonella%20as%20infection%20model&amp;volume=26&amp;issue=4&amp;publication_year=2021&amp;pages=e12808&amp;pmid=33884706&amp;doi=10.1111/hel.12808&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0024">
<span class="label">[24].</span><cite>Killiny N. Generous hosts: why the larvae of greater wax moth, <em>Galleria mellonella</em> is a perfect infectious host model?
Virulence. 2018;9(1):860–865. doi: 10.1080/21505594.2018.1454172
</cite> [<a href="https://doi.org/10.1080/21505594.2018.1454172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5955462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29726300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virulence&amp;title=Generous%20hosts:%20why%20the%20larvae%20of%20greater%20wax%20moth,%20Galleria%20mellonella%20is%20a%20perfect%20infectious%20host%20model?&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=860-865&amp;pmid=29726300&amp;doi=10.1080/21505594.2018.1454172&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0025">
<span class="label">[25].</span><cite>Yu CK, Wang C-J, Chew Y, et al. Functional characterization of <em>Helicobacter pylori</em> 26695 sedoheptulose 7-phosphate isomerase encoded by hp0857 and its association with lipopolysaccharide biosynthesis and adhesion. Biochem Biophys Res Commun. 2016;477(4):794–800. doi: 10.1016/j.bbrc.2016.06.137
</cite> [<a href="https://doi.org/10.1016/j.bbrc.2016.06.137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27369071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&amp;title=Functional%20characterization%20of%20Helicobacter%20pylori%2026695%20sedoheptulose%207-phosphate%20isomerase%20encoded%20by%20hp0857%20and%20its%20association%20with%20lipopolysaccharide%20biosynthesis%20and%20adhesion&amp;volume=477&amp;issue=4&amp;publication_year=2016&amp;pages=794-800&amp;pmid=27369071&amp;doi=10.1016/j.bbrc.2016.06.137&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0026">
<span class="label">[26].</span><cite>Chang PC, Wang C-J, You C-K, et al. Effects of a HP0859 (rfaD) knockout mutation on lipopolysaccharide structure of <em>Helicobacter pylori</em> 26695 and the bacterial adhesion on AGS cells. Biochem Biophys Res Commun. 2011;405(3):497–502. doi: 10.1016/j.bbrc.2011.01.060
</cite> [<a href="https://doi.org/10.1016/j.bbrc.2011.01.060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21256827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&amp;title=Effects%20of%20a%20HP0859%20(rfaD)%20knockout%20mutation%20on%20lipopolysaccharide%20structure%20of%20Helicobacter%20pylori%2026695%20and%20the%20bacterial%20adhesion%20on%20AGS%20cells&amp;volume=405&amp;issue=3&amp;publication_year=2011&amp;pages=497-502&amp;pmid=21256827&amp;doi=10.1016/j.bbrc.2011.01.060&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0027">
<span class="label">[27].</span><cite>Chiu SF, Teng K-W, Wang P-C, et al. <em>Helicobacter pylori</em> GmhB enzyme involved in ADP-heptose biosynthesis pathway is essential for lipopolysaccharide biosynthesis and bacterial virulence. Virulence. 2021;12(1):1610–1628. doi: 10.1080/21505594.2021.1938449
</cite> [<a href="https://doi.org/10.1080/21505594.2021.1938449" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8204981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34125649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virulence&amp;title=Helicobacter%20pylori%20GmhB%20enzyme%20involved%20in%20ADP-heptose%20biosynthesis%20pathway%20is%20essential%20for%20lipopolysaccharide%20biosynthesis%20and%20bacterial%20virulence&amp;volume=12&amp;issue=1&amp;publication_year=2021&amp;pages=1610-1628&amp;pmid=34125649&amp;doi=10.1080/21505594.2021.1938449&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0028">
<span class="label">[28].</span><cite>Horton RM. Gene splicing by overlap extension. Methods Enzymol. 1993;217:270–279.
</cite> [<a href="https://doi.org/10.1016/0076-6879(93)17067-f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8474334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Enzymol&amp;title=Gene%20splicing%20by%20overlap%20extension&amp;volume=217&amp;publication_year=1993&amp;pages=270-279&amp;pmid=8474334&amp;doi=10.1016/0076-6879(93)17067-f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0029">
<span class="label">[29].</span><cite>Lefebvre B, Formstecher P, Lefebvre P. Improvement of the gene splicing overlap (SOE) method. Biotechniques. 1995;19(2):186–188.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/8527132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechniques&amp;title=Improvement%20of%20the%20gene%20splicing%20overlap%20(SOE)%20method&amp;volume=19&amp;issue=2&amp;publication_year=1995&amp;pages=186-188&amp;pmid=8527132&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0030">
<span class="label">[30].</span><cite>Horton RM. In vitro recombination and mutagenesis of DNA. SOEing together tailor-made genes. Methods Mol Biol. 1997;67:141–149.
</cite> [<a href="https://doi.org/10.1385/0-89603-483-6:141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9031138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&amp;title=In%20vitro%20recombination%20and%20mutagenesis%20of%20DNA.%20SOEing%20together%20tailor-made%20genes&amp;volume=67&amp;publication_year=1997&amp;pages=141-149&amp;pmid=9031138&amp;doi=10.1385/0-89603-483-6:141&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0031">
<span class="label">[31].</span><cite>Jorjão AL, Oliveira LD, Scorzoni L, et al. From moths to caterpillars: ideal conditions for <em>Galleria mellonella</em> rearing for in vivo microbiological studies. Virulence. 2018;9(1):383–389. doi: 10.1080/21505594.2017.1397871
</cite> [<a href="https://doi.org/10.1080/21505594.2017.1397871" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5955185/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29130369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virulence&amp;title=From%20moths%20to%20caterpillars:%20ideal%20conditions%20for%20Galleria%20mellonella%20rearing%20for%20in%20vivo%20microbiological%20studies&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=383-389&amp;pmid=29130369&amp;doi=10.1080/21505594.2017.1397871&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0032">
<span class="label">[32].</span><cite>Askoura M, Stintzi A. Using <em>Galleria mellonella</em> as an infection model for <em>Campylobacter jejuni</em> pathogenesis. Methods Mol Biol. 2017;1512:163–169.
</cite> [<a href="https://doi.org/10.1007/978-1-4939-6536-6_14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27885606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&amp;title=Using%20Galleria%20mellonella%20as%20an%20infection%20model%20for%20Campylobacter%20jejuni%20pathogenesis&amp;volume=1512&amp;publication_year=2017&amp;pages=163-169&amp;pmid=27885606&amp;doi=10.1007/978-1-4939-6536-6_14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0033">
<span class="label">[33].</span><cite>Giannouli M, Palatucci AT, Rubino V, et al. Use of larvae of the wax moth <em>Galleria mellonella</em> as an in vivo model to study the virulence of <em>Helicobacter pylori</em>. BMC Microbiol. 2014;14(1):228. doi: 10.1186/s12866-014-0228-0
</cite> [<a href="https://doi.org/10.1186/s12866-014-0228-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4148543/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25170542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol&amp;title=Use%20of%20larvae%20of%20the%20wax%20moth%20Galleria%20mellonella%20as%20an%20in%20vivo%20model%20to%20study%20the%20virulence%20of%20Helicobacter%20pylori&amp;volume=14&amp;issue=1&amp;publication_year=2014&amp;pages=228&amp;pmid=25170542&amp;doi=10.1186/s12866-014-0228-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0034">
<span class="label">[34].</span><cite>Hoffman PS, Vats N, Hutchison D, et al. Development of an interleukin-12-deficient mouse model that is permissive for colonization by a motile KE26695 strain of <em>Helicobacter pylori</em>. Infect Immun. 2003;71(5):2534–2541. doi: 10.1128/IAI.71.5.2534-2541.2003
</cite> [<a href="https://doi.org/10.1128/IAI.71.5.2534-2541.2003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC153236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12704125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&amp;title=Development%20of%20an%20interleukin-12-deficient%20mouse%20model%20that%20is%20permissive%20for%20colonization%20by%20a%20motile%20KE26695%20strain%20of%20Helicobacter%20pylori&amp;volume=71&amp;issue=5&amp;publication_year=2003&amp;pages=2534-2541&amp;pmid=12704125&amp;doi=10.1128/IAI.71.5.2534-2541.2003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0035">
<span class="label">[35].</span><cite>Tamaki S, Sato T, Matsuhashi M. Role of lipopolysaccharides in antibiotic resistance and bacteriophage adsorption of <em>Escherichia coli</em> K-12. J Bacteriol. 1971;105(3):968–975. doi: 10.1128/jb.105.3.968-975.1971
</cite> [<a href="https://doi.org/10.1128/jb.105.3.968-975.1971" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC248525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4926688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Role%20of%20lipopolysaccharides%20in%20antibiotic%20resistance%20and%20bacteriophage%20adsorption%20of%20Escherichia%20coli%20K-12&amp;volume=105&amp;issue=3&amp;publication_year=1971&amp;pages=968-975&amp;pmid=4926688&amp;doi=10.1128/jb.105.3.968-975.1971&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0036">
<span class="label">[36].</span><cite>Wang Z, Wang J, Ren G, et al. Influence of core oligosaccharide of lipopolysaccharide to outer membrane behavior of <em>Escherichia coli</em>. Mar Drugs. 2015;13(6):3325–3339. doi: 10.3390/md13063325
</cite> [<a href="https://doi.org/10.3390/md13063325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4483631/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26023839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mar%20Drugs&amp;title=Influence%20of%20core%20oligosaccharide%20of%20lipopolysaccharide%20to%20outer%20membrane%20behavior%20of%20Escherichia%20coli&amp;volume=13&amp;issue=6&amp;publication_year=2015&amp;pages=3325-3339&amp;pmid=26023839&amp;doi=10.3390/md13063325&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0037">
<span class="label">[37].</span><cite>Tohidpour AJMP. Caga-mediated pathogenesis of <em>Helicobacter pylori</em>. Microb Pathog. 2016;93:44–55. doi: 10.1016/j.micpath.2016.01.005
</cite> [<a href="https://doi.org/10.1016/j.micpath.2016.01.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26796299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microb%20Pathog&amp;title=Caga-mediated%20pathogenesis%20of%20Helicobacter%20pylori&amp;volume=93&amp;publication_year=2016&amp;pages=44-55&amp;pmid=26796299&amp;doi=10.1016/j.micpath.2016.01.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0038">
<span class="label">[38].</span><cite>Hatakeyama M. Structure and function of <em>Helicobacter pylori</em> CagA, the first-identified bacterial protein involved in human cancer. Proc Jpn Acad Ser B Phys Biol Sci. 2017;93(4):196–219. doi: 10.2183/pjab.93.013</cite> [<a href="https://doi.org/10.2183/pjab.93.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5489429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28413197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Jpn%20Acad%20Ser%20B%20Phys%20Biol%20Sci&amp;title=Structure%20and%20function%20of%20Helicobacter%20pylori%20CagA,%20the%20first-identified%20bacterial%20protein%20involved%20in%20human%20cancer&amp;volume=93&amp;issue=4&amp;publication_year=2017&amp;pages=196-219&amp;pmid=28413197&amp;doi=10.2183/pjab.93.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0039">
<span class="label">[39].</span><cite>Handa O, Naito Y, Yoshikawa T. Caga protein of <em>Helicobacter pylori</em>: a hijacker of gastric epithelial cell signaling. Biochem Pharmacol. 2007;73(11):1697–1702. doi: 10.1016/j.bcp.2006.10.022
</cite> [<a href="https://doi.org/10.1016/j.bcp.2006.10.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17134680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&amp;title=Caga%20protein%20of%20Helicobacter%20pylori:%20a%20hijacker%20of%20gastric%20epithelial%20cell%20signaling&amp;volume=73&amp;issue=11&amp;publication_year=2007&amp;pages=1697-1702&amp;pmid=17134680&amp;doi=10.1016/j.bcp.2006.10.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0040">
<span class="label">[40].</span><cite>Foegeding NJ, Caston R, McClain M, et al. An overview of <em>Helicobacter pylori</em> VacA toxin biology. Toxins (Basel). 2016;8(6):173. doi: 10.3390/toxins8060173
</cite> [<a href="https://doi.org/10.3390/toxins8060173" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4926140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27271669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Toxins%20(Basel)&amp;title=An%20overview%20of%20Helicobacter%20pylori%20VacA%20toxin%20biology&amp;volume=8&amp;issue=6&amp;publication_year=2016&amp;pages=173&amp;pmid=27271669&amp;doi=10.3390/toxins8060173&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0041">
<span class="label">[41].</span><cite>Kwok T, Backert S, Schwarz H, et al. Specific entry of <em>Helicobacter pylori</em> into cultured gastric epithelial cells via a zipper-like mechanism. Infect Immun. 2002;70(4):2108–2120. doi: 10.1128/IAI.70.4.2108-2120.2002
</cite> [<a href="https://doi.org/10.1128/IAI.70.4.2108-2120.2002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC127843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11895977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&amp;title=Specific%20entry%20of%20Helicobacter%20pylori%20into%20cultured%20gastric%20epithelial%20cells%20via%20a%20zipper-like%20mechanism&amp;volume=70&amp;issue=4&amp;publication_year=2002&amp;pages=2108-2120&amp;pmid=11895977&amp;doi=10.1128/IAI.70.4.2108-2120.2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0042">
<span class="label">[42].</span><cite>Garcia-Weber D, Dangeard A-S, Cornil J, et al. Adp-heptose is a newly identified pathogen-associated molecular pattern of <em>Shigella flexneri</em>. EMBO Rep. 2018;19(12). doi: 10.15252/embr.201846943</cite> [<a href="https://doi.org/10.15252/embr.201846943" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6280651/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30455202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Rep&amp;title=Adp-heptose%20is%20a%20newly%20identified%20pathogen-associated%20molecular%20pattern%20of%20Shigella%20flexneri&amp;volume=19&amp;issue=12&amp;publication_year=2018&amp;pmid=30455202&amp;doi=10.15252/embr.201846943&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0043">
<span class="label">[43].</span><cite>Ménard G, Rouillon A, Cattoir V, et al. <em>Galleria mellonella</em> as a suitable model of bacterial infection: past, present and future. Front Cell Infect Microbiol. 2021;11:782733. doi: 10.3389/fcimb.2021.782733
</cite> [<a href="https://doi.org/10.3389/fcimb.2021.782733" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8727906/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35004350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Infect%20Microbiol&amp;title=Galleria%20mellonella%20as%20a%20suitable%20model%20of%20bacterial%20infection:%20past,%20present%20and%20future&amp;volume=11&amp;publication_year=2021&amp;pages=782733&amp;pmid=35004350&amp;doi=10.3389/fcimb.2021.782733&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0044">
<span class="label">[44].</span><cite>Teng KW, Hsieh K-S, Hung J-S, et al. <em>Helicobacter pylori</em> employs a general protein glycosylation system for the modification of outer membrane adhesins. Gut Microbes. 2022;14(1):2130650. doi: 10.1080/19490976.2022.2130650
</cite> [<a href="https://doi.org/10.1080/19490976.2022.2130650" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9553153/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36206406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gut%20Microbes&amp;title=Helicobacter%20pylori%20employs%20a%20general%20protein%20glycosylation%20system%20for%20the%20modification%20of%20outer%20membrane%20adhesins&amp;volume=14&amp;issue=1&amp;publication_year=2022&amp;pages=2130650&amp;pmid=36206406&amp;doi=10.1080/19490976.2022.2130650&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0045">
<span class="label">[45].</span><cite>Moran AP. The role of lipopolysaccharide in <em>Helicobacter pylori</em> pathogenesis. Aliment Pharmacol Ther. 1996;10 Suppl 1(Sup1):39–50. doi: 10.1046/j.1365-2036.1996.22164004.x</cite> [<a href="https://doi.org/10.1046/j.1365-2036.1996.22164004.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8730258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aliment%20Pharmacol%20Ther&amp;title=The%20role%20of%20lipopolysaccharide%20in%20Helicobacter%20pylori%20pathogenesis&amp;volume=10%20Suppl%201&amp;issue=Sup1&amp;publication_year=1996&amp;pages=39-50&amp;pmid=8730258&amp;doi=10.1046/j.1365-2036.1996.22164004.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0046">
<span class="label">[46].</span><cite>Nikaido H. Molecular basis of bacterial outer membrane permeability revisited. Microbiol Mol Biol Rev. 2003;67(4):593–656. doi: 10.1128/MMBR.67.4.593-656.2003
</cite> [<a href="https://doi.org/10.1128/MMBR.67.4.593-656.2003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC309051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14665678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol%20Mol%20Biol%20Rev&amp;title=Molecular%20basis%20of%20bacterial%20outer%20membrane%20permeability%20revisited&amp;volume=67&amp;issue=4&amp;publication_year=2003&amp;pages=593-656&amp;pmid=14665678&amp;doi=10.1128/MMBR.67.4.593-656.2003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0047">
<span class="label">[47].</span><cite>Sidor K, Skirecki T. A bittersweet kiss of gram-negative bacteria: the role of ADP-heptose in the pathogenesis of infection. Microorganisms. 2023;11(5):11. doi: 10.3390/microorganisms11051316</cite> [<a href="https://doi.org/10.3390/microorganisms11051316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10221265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37317291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microorganisms&amp;title=A%20bittersweet%20kiss%20of%20gram-negative%20bacteria:%20the%20role%20of%20ADP-heptose%20in%20the%20pathogenesis%20of%20infection&amp;volume=11&amp;issue=5&amp;publication_year=2023&amp;pages=11&amp;pmid=37317291&amp;doi=10.3390/microorganisms11051316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0048">
<span class="label">[48].</span><cite>Faass L, Stein SC, Hauke M, et al. Contribution of heptose metabolites and the cag pathogenicity island to the activation of monocytes/macrophages by <em>Helicobacter pylori</em>. Front Immunol. 2021;12:632154. doi: 10.3389/fimmu.2021.632154
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.632154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8174060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34093525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=Contribution%20of%20heptose%20metabolites%20and%20the%20cag%20pathogenicity%20island%20to%20the%20activation%20of%20monocytes/macrophages%20by%20Helicobacter%20pylori&amp;volume=12&amp;publication_year=2021&amp;pages=632154&amp;pmid=34093525&amp;doi=10.3389/fimmu.2021.632154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0049">
<span class="label">[49].</span><cite>Bauer M, Nascakova Z, Mihai A-I, et al. The ALPK1/TIFA/NF-κB axis links a bacterial carcinogen to R-loop-induced replication stress. Nat Commun. 2020;11(1):5117. doi: 10.1038/s41467-020-18857-z
</cite> [<a href="https://doi.org/10.1038/s41467-020-18857-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7547021/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33037203/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=The%20ALPK1/TIFA/NF-%CE%BAB%20axis%20links%20a%20bacterial%20carcinogen%20to%20R-loop-induced%20replication%20stress&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=5117&amp;pmid=33037203&amp;doi=10.1038/s41467-020-18857-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0050">
<span class="label">[50].</span><cite>Hauke M, Metz F, Rapp J, et al. <em>Helicobacter pylori</em> modulates heptose metabolite biosynthesis and heptose-dependent innate immune host cell activation by multiple mechanisms. Microbiol Spectr. 2023;11(3):e0313222. doi: 10.1128/spectrum.03132-22
</cite> [<a href="https://doi.org/10.1128/spectrum.03132-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10269868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37129481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol%20Spectr&amp;title=Helicobacter%20pylori%20modulates%20heptose%20metabolite%20biosynthesis%20and%20heptose-dependent%20innate%20immune%20host%20cell%20activation%20by%20multiple%20mechanisms&amp;volume=11&amp;issue=3&amp;publication_year=2023&amp;pages=e0313222&amp;pmid=37129481&amp;doi=10.1128/spectrum.03132-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0051">
<span class="label">[51].</span><cite>Liu AN, Teng K-W, Chew Y, et al. The effects of HP0044 and HP1275 knockout mutations on the structure and function of lipopolysaccharide in <em>Helicobacter pylori</em> strain 26695. Biomedicines. 2022;10(1):145. doi: 10.3390/biomedicines10010145
</cite> [<a href="https://doi.org/10.3390/biomedicines10010145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8773439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35052824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomedicines&amp;title=The%20effects%20of%20HP0044%20and%20HP1275%20knockout%20mutations%20on%20the%20structure%20and%20function%20of%20lipopolysaccharide%20in%20Helicobacter%20pylori%20strain%2026695&amp;volume=10&amp;issue=1&amp;publication_year=2022&amp;pages=145&amp;pmid=35052824&amp;doi=10.3390/biomedicines10010145&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0052">
<span class="label">[52].</span><cite>Benz I, Schmidt MA. Glycosylation with heptose residues mediated by the aah gene product is essential for adherence of the AIDA-I adhesin. Mol Microbiol. 2001;40(6):1403–1413. doi: 10.1046/j.1365-2958.2001.02487.x
</cite> [<a href="https://doi.org/10.1046/j.1365-2958.2001.02487.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11442838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&amp;title=Glycosylation%20with%20heptose%20residues%20mediated%20by%20the%20aah%20gene%20product%20is%20essential%20for%20adherence%20of%20the%20AIDA-I%20adhesin&amp;volume=40&amp;issue=6&amp;publication_year=2001&amp;pages=1403-1413&amp;pmid=11442838&amp;doi=10.1046/j.1365-2958.2001.02487.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0053">
<span class="label">[53].</span><cite>Senior NJ, Bagnall MC, Champion OL, et al. <em>Galleria mellonella</em> as an infection model for <em>Campylobacter jejuni</em> virulence. J Med Microbiol. 2011;60(Pt 5):661–669. doi: 10.1099/jmm.0.026658-0
</cite> [<a href="https://doi.org/10.1099/jmm.0.026658-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21233296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Med%20Microbiol&amp;title=Galleria%20mellonella%20as%20an%20infection%20model%20for%20Campylobacter%20jejuni%20virulence&amp;volume=60&amp;issue=Pt%205&amp;publication_year=2011&amp;pages=661-669&amp;pmid=21233296&amp;doi=10.1099/jmm.0.026658-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0054">
<span class="label">[54].</span><cite>Trevijano-Contador N, Zaragoza O. Immune response of <em>Galleria mellonella</em> against human fungal pathogens. J Fungi (Basel). 2018;5(1):3. doi: 10.3390/jof5010003
</cite> [<a href="https://doi.org/10.3390/jof5010003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6463112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30587801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Fungi%20(Basel)&amp;title=Immune%20response%20of%20Galleria%20mellonella%20against%20human%20fungal%20pathogens&amp;volume=5&amp;issue=1&amp;publication_year=2018&amp;pages=3&amp;pmid=30587801&amp;doi=10.3390/jof5010003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0055">
<span class="label">[55].</span><cite>Hillyer JF. Insect immunology and hematopoiesis. Dev Comp Immunol. 2016;58:102–118. doi: 10.1016/j.dci.2015.12.006
</cite> [<a href="https://doi.org/10.1016/j.dci.2015.12.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4775421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26695127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev%20Comp%20Immunol&amp;title=Insect%20immunology%20and%20hematopoiesis&amp;volume=58&amp;publication_year=2016&amp;pages=102-118&amp;pmid=26695127&amp;doi=10.1016/j.dci.2015.12.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Supplementary Fig 2_2.jpg</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377144/bin/KVIR_A_2548620_SM6948.jpg" data-ga-action="click_feat_suppl" class="usa-link">KVIR_A_2548620_SM6948.jpg</a><sup> (267.2KB, jpg) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><span>HP0858_paper_Supplementary_material_20241230 (1).docx</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377144/bin/KVIR_A_2548620_SM6947.docx" data-ga-action="click_feat_suppl" class="usa-link">KVIR_A_2548620_SM6947.docx</a><sup> (58.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><span>Supplementary Fig 1_2.jpeg</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377144/bin/KVIR_A_2548620_SM6946.jpeg" data-ga-action="click_feat_suppl" class="usa-link">KVIR_A_2548620_SM6946.jpeg</a><sup> (117.7KB, jpeg) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data supporting this study’s findings are openly available in Figshare at <a href="https://doi.org/10.6084/m9.figshare.28081937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.6084/m9.figshare.28081937</a></p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Virulence are provided here courtesy of <strong>Taylor & Francis</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1080/21505594.2025.2548620"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/KVIR_16_2548620.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12377144/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12377144/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377144%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377144/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12377144/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12377144/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40849897/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12377144/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40849897/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12377144/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12377144/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="tTMJlG6yFJZlxvIYSXP8xCHYkpNTVgbo76Dr9x9Dh5p86zZQ6hcIqFBAoYtp3ZOR">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
