
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Noncanonical terpene cyclases for the biosynthesis of diterpenoids regulating chlamydospore formation in plant-associated Trichoderma - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15253A0F8AF204A3053A0F0012CB78AE.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370890/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Noncanonical terpene cyclases for the biosynthesis of diterpenoids regulating chlamydospore formation in plant-associated Trichoderma">
<meta name="citation_author" content="Min-Jie Yang">
<meta name="citation_author_institution" content="State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China">
<meta name="citation_author" content="De-Sen Li">
<meta name="citation_author_institution" content="State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China">
<meta name="citation_author" content="Hua-Qin Deng">
<meta name="citation_author_institution" content="State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China">
<meta name="citation_author" content="Wen-Yuan Li">
<meta name="citation_author_institution" content="State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China">
<meta name="citation_author" content="Xin-Yu Zheng">
<meta name="citation_author_institution" content="State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China">
<meta name="citation_author" content="Jonathan Gershenzon">
<meta name="citation_author_institution" content="Department of Biochemistry, Max Planck Institute for Chemical Ecology, Hans-Knoell-Strasse 8, Jena, 07745 Germany">
<meta name="citation_author" content="Xue-Mei Niu">
<meta name="citation_author_institution" content="School of Life Sciences, State Key Laboratory for Conservation and Utilization of Bio-Resources &amp; Key Laboratory for Microbial Resources of the Ministry of Education, Yunnan University, Kunming, 650091 P. R. China">
<meta name="citation_author" content="Yan Liu">
<meta name="citation_author_institution" content="State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China">
<meta name="citation_author_institution" content="State Key Laboratory of Southwestern Chinese Medicine Resources, and Innovative Institute of Chinese Medicine and Pharmacy, Chengdu University of Traditional Chinese Medicine, Chengdu, 611137 P. R. China">
<meta name="citation_author" content="Sheng-Hong Li">
<meta name="citation_author_institution" content="State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China">
<meta name="citation_author_institution" content="State Key Laboratory of Southwestern Chinese Medicine Resources, and Innovative Institute of Chinese Medicine and Pharmacy, Chengdu University of Traditional Chinese Medicine, Chengdu, 611137 P. R. China">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7823">
<meta name="citation_doi" content="10.1038/s41467-025-63055-4">
<meta name="citation_pmid" content="40841380">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370890/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370890/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370890/pdf/41467_2025_Article_63055.pdf">
<meta name="description" content="Terpene cyclases catalyze exquisite and complicated cyclization reactions to generate diverse terpenoid skeletons. Trichoderma fungi are important biocontrol agents, characteristic of producing complex bioactive tetracyclic diterpenoids named ...">
<meta name="og:title" content="Noncanonical terpene cyclases for the biosynthesis of diterpenoids regulating chlamydospore formation in plant-associated Trichoderma">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Terpene cyclases catalyze exquisite and complicated cyclization reactions to generate diverse terpenoid skeletons. Trichoderma fungi are important biocontrol agents, characteristic of producing complex bioactive tetracyclic diterpenoids named ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370890/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370890">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63055-4"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63055.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370890%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370890/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370890/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370890/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 22;16:7823. doi: <a href="https://doi.org/10.1038/s41467-025-63055-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63055-4</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Noncanonical terpene cyclases for the biosynthesis of diterpenoids regulating chlamydospore formation in plant-associated <em>Trichoderma</em>
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20MJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Min-Jie Yang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Min-Jie Yang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20MJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Min-Jie Yang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20DS%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">De-Sen Li</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">De-Sen Li</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20DS%22%5BAuthor%5D" class="usa-link"><span class="name western">De-Sen Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20HQ%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Hua-Qin Deng</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Hua-Qin Deng</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20HQ%22%5BAuthor%5D" class="usa-link"><span class="name western">Hua-Qin Deng</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20WY%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Wen-Yuan Li</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Wen-Yuan Li</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20WY%22%5BAuthor%5D" class="usa-link"><span class="name western">Wen-Yuan Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20XY%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Xin-Yu Zheng</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Xin-Yu Zheng</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20XY%22%5BAuthor%5D" class="usa-link"><span class="name western">Xin-Yu Zheng</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gershenzon%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jonathan Gershenzon</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jonathan Gershenzon</span></h3>
<div class="p">
<sup>2</sup>Department of Biochemistry, Max Planck Institute for Chemical Ecology, Hans-Knoell-Strasse 8, Jena, 07745 Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gershenzon%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jonathan Gershenzon</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Niu%20XM%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Xue-Mei Niu</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Xue-Mei Niu</span></h3>
<div class="p">
<sup>3</sup>School of Life Sciences, State Key Laboratory for Conservation and Utilization of Bio-Resources &amp; Key Laboratory for Microbial Resources of the Ministry of Education, Yunnan University, Kunming, 650091 P. R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Niu%20XM%22%5BAuthor%5D" class="usa-link"><span class="name western">Xue-Mei Niu</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Yan Liu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Yan Liu</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China </div>
<div class="p">
<sup>4</sup>State Key Laboratory of Southwestern Chinese Medicine Resources, and Innovative Institute of Chinese Medicine and Pharmacy, Chengdu University of Traditional Chinese Medicine, Chengdu, 611137 P. R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yan Liu</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20SH%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Sheng-Hong Li</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Sheng-Hong Li</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China </div>
<div class="p">
<sup>4</sup>State Key Laboratory of Southwestern Chinese Medicine Resources, and Innovative Institute of Chinese Medicine and Pharmacy, Chengdu University of Traditional Chinese Medicine, Chengdu, 611137 P. R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20SH%22%5BAuthor%5D" class="usa-link"><span class="name western">Sheng-Hong Li</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>State Key Laboratory of Phytochemistry and Natural Medicines, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 650201 P. R. China </div>
<div id="Aff2">
<sup>2</sup>Department of Biochemistry, Max Planck Institute for Chemical Ecology, Hans-Knoell-Strasse 8, Jena, 07745 Germany </div>
<div id="Aff3">
<sup>3</sup>School of Life Sciences, State Key Laboratory for Conservation and Utilization of Bio-Resources &amp; Key Laboratory for Microbial Resources of the Ministry of Education, Yunnan University, Kunming, 650091 P. R. China </div>
<div id="Aff4">
<sup>4</sup>State Key Laboratory of Southwestern Chinese Medicine Resources, and Innovative Institute of Chinese Medicine and Pharmacy, Chengdu University of Traditional Chinese Medicine, Chengdu, 611137 P. R. China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 20; Accepted 2025 Aug 8; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370890  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841380/" class="usa-link">40841380</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Terpene cyclases catalyze exquisite and complicated cyclization reactions to generate diverse terpenoid skeletons. <em>Trichoderma</em> fungi are important biocontrol agents, characteristic of producing complex bioactive tetracyclic diterpenoids named harzianes and trichodermanins, but their biosynthesis and biological functions have long been enigmatic. Here we identify TriDTCs, an unprecedented family of terpene cyclases in <em>Trichoderma</em>, responsible for cyclizing geranylgeranyl diphosphate (GGPP) into major diterpenes harzianol I and wickerol A, via heterologous expressions, gene deletion, and in vitro assays. TriDTCs represent a brand new class of terpene cyclases, lacking known motifs and diverging from all known enzymes. Mechanistically, TriDTCs likely employ a unique DxxDxxxD aspartate triad for cyclization initiation, a critical valine residue modulating product specificity, and “gatekeeper” residues for activity. Phylogenetic analysis shows TriDTCs have a narrow distribution in three fungal genera and are highly functionally specific within <em>Trichoderma</em>, suggesting a genus-specific acquisition and independent evolution. Functional studies implicated TriDTCs in fungal survival strategies by regulating formation of resistant propagules (chlamydospore in <em>Trichoderma</em>, sclerotia in <em>Aspergillus oryzae</em>). These findings expand the knowledge of terpene cyclase diversity and biological significance, herald a strategy to enhance <em>Trichoderma</em>’s biocontrol efficacy, and open avenues for pharmacological investigation of these diterpenoids.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Natural product synthesis, Chemical ecology</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">This study identifies TriDTCs as a family of terpene cyclases responsible for harzianol I and wickerol A biosynthesis in <em>Trichoderma</em> fungi and are found to regulate <em>Trichoderma</em>’s chlamydospore and <em>Aspergillus oryzae</em>’s sclerotia formation through producing harzianol I.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Terpenoids constitute the largest family of natural products, with over 100,000 members identified from various organisms (dnp.chemnetbase.com)<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. They possess important biological functions and participate in the growth and survival process of the organisms that produce them, such as serving as membrane components (e.g., hopanoids, sterols), hormones (e.g., gibberellins, sobralene<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>), defensive chemicals (e.g., pyrethrins, trichothecenes<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>, sesterterpenoids<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>), and pigments (e.g., carotenoids). Their structural complexity also underpins the pharmaceutical significance, exemplified by clinical medicines such as Taxol and artemisinin. Terpenoid biosynthesis begins with C<sub>5</sub> isoprenoid units, which are assembled into acyclic pyrophosphate precursors (C<sub>5n</sub>) and subsequently cyclized by terpene cyclases (TCs) to form intricate carbocyclic scaffolds. The catalysis of TCs starts with the formation of carbocation, which triggers a cascade of electron flow and carbon–carbon bond formation to generate fused rings and stereocenters. Based on different carbocation-initiating mechanisms, canonical TCs are categorized into type I and II, featuring specific conserved catalytic motifs: type I enzymes typically contain Mg<sup>2+</sup>-binding DDxxD and NSE/DTE motifs, while type II TCs utilize a DxDD motif. Recently, noncanonical TCs lacking these motifs, such as YtpB and AsR6<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> classified as type I TCs from uncharacterized protein families, have expanded our understanding of TC diversity<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. However, the cryptic nature and unpredictable functions of these noncanonical TCs present major challenges for their discovery and characterization.</p>
<p id="Par4">Fungi possess specialized machinery to produce diverse terpenoids with unique chemical structures and significant biological functions<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. For instance, farnesol acts as a communication molecule in <em>Candida albicans</em><sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, while austin, ophiobolins, and trichothecenes serve as defensive chemicals in their producing fungi<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. The genus <em>Trichoderma</em> (Hypocreales, Ascomycota) stands out among fungi with its prominent environmental adaptability, which thrives across diverse habitats ranging from terrestrial soils to marine ecosystems and even acts as an opportunistic human pathogens<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. <em>Trichoderma</em> species are widely used as biocontrol agents (BCAs) due to their rapid growth, prolific propagation, and versatile biocontrol traits, making them one of the most promising microorganisms for eco-sustainable agriculture<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. <em>Trichoderma</em> produces three types of asexual propagules, including chlamydospores, conidiospores, and mycelia, which confer a competitive advantage in occupying ecological niches<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Among these, chlamydospores exhibit strong resistance to adverse environmental conditions, which greatly enhances <em>Trichoderma</em>’s biocontrol efficacy<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p>
<p id="Par5"><em>Trichoderma</em> is a rich source of terpenoids<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, particularly the characteristic harziane and trichodermanin diterpenoids, which are exclusively reported from <em>Trichoderma</em> species. Harzianes feature a characteristic 4/7/5/6-fused tetracyclic scaffold, with more than 40 members having been identified from various <em>Trichoderma</em> species (Supplementary Fig. <a href="#MOESM1" class="usa-link">S1</a>). In contrast, trichodermanins have a 6/5/6/6 ring system, with 8 compounds identified (Supplementary Fig. <a href="#MOESM1" class="usa-link">S2</a>). These diterpenoids display extensive significant biological activities, including antifungal, anti-HIV, cytotoxic, and phytotoxic effects<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. The biosynthesis of <em>Trichoderma</em> diterpenoids has attracted considerable interest. Previous studies on the cyclization mechanism employing <sup>13</sup>C- and <sup>2</sup>H-labeled substrates suggest that they are biosynthesized from geranylgeranyl diphosphate (GGPP) via type I carbocation-initiated reactions (Fig. <a href="#Fig1" class="usa-link">1a</a>)<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. However, further elucidation of the enzymes responsible for their biosynthesis has confronted significant challenges since their first discovery in 1992<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, despite the availability of 151 genome sequences covering 34 <em>Trichoderma</em> species and the identification of several TCs involved in sesquiterpenoid biosynthesis<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. In addition, although the pharmacological activities of these unique diterpenoids have been extensively investigated, their biological functions in <em>Trichoderma</em> fungi remain unknown.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. Chemical structures of harziane and trichodermanin diterpenoids from <em>Trichoderma</em> and their biosynthetic mechanisms.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370890_41467_2025_63055_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6d37/12370890/969ec5921961/41467_2025_63055_Fig1_HTML.jpg" loading="lazy" id="d33e473" height="224" width="799" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The cyclization mechanism of harzianes (<strong>1</strong>) and trichodermanins (<strong>2</strong>) from GGPP catalyzed by TriDTCs is indicated by plain arrows. WMR: Wagner-Meerwein rearrangement. The proposed biosynthetic pathway of harzianone (<strong>3</strong>) and harziandione (<strong>4</strong>) is indicated by dotted arrows. <strong>a</strong>–<strong>f</strong> are intermediates in the cyclization process.</p></figcaption></figure><p id="Par6">In this study, employing an enzymatic activity-guided protein fractionation strategy, we identified an unprecedented family of enzymes from <em>Trichoderma</em> fungi responsible for the biosynthesis of <em>Trichoderma</em>-characteristic diterpenoids, which we named TriDTCs (<em><span class="text-underline">Tri</span></em><em>choderma</em>
<span class="text-underline">d</span>i<span class="text-underline">t</span>erpene <span class="text-underline">c</span>yclase<span class="text-underline">s</span>). Mutational experiments guided by structural simulations revealed a distinct yet convergent enzymatic mechanism. Phylogenetic analysis and homolog characterization uncovered the specific distribution and specialized function of TriDTCs in nature, suggesting their unique evolution in <em>Trichoderma</em>. Furthermore, we discovered that <em>Trichoderma</em> fungi employ these specific enzymes and the diterpenoid products to regulate chlamydospore formation. Moreover, this function, mediated by TriDTCs and harzianol I, could also be applied to promote sclerotia formation in <em>Aspergillus oryzae</em>.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Mining and characterization of TriDTC for <em>Trichoderma</em>-characteristic diterpenoid biosynthesis</h3>
<p id="Par7">To explore the biosynthetic pathways of harzianes and trichodermanins, we performed genome sequencing of <em>T. atroviride</em> B7, an endophytic fungus isolated from the medicinal plant <em>Colquhounia coccinea</em> var. <em>mollis</em>, which produces these diterpenoids<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. AntiSMASH analysis predicted eight biosynthetic gene clusters (BGCs) related to terpenoid biosynthesis (Supplementary Fig. <a href="#MOESM1" class="usa-link">S3</a>, Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>). Among these, the candidate gene <em>triTPS8</em> showed 89% sequence identity to the characterized trichobrasilenol synthase (TaTC6)<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Five additional genes encoding isoprenoid synthase domain-containing proteins were also recognized from the genome (Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>). All 12 candidates (excluding <em>triTPS8</em>) were cloned and heterologously expressed in both engineered <em>E. coli</em> BL21 and filamentous fungal host <em>A. oryzae</em> NSAR1. Thereinto, the engineered <em>E. coli</em> strain harboring pBbA5c-MevT-MBIS and pCDF-Duet1-<em>crtE</em><sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> plasmids can provide sufficient acyclic terpene precursors, including farnesyl pyrophosphate (FPP) and GGPP. However, none of these constructs yielded harziane or trichodermanin diterpenes. In contrast, when <em>triTPS4</em>, a gene predicted to encode a GGPP synthase, was co-expressed with pBbA5c-MevT-MBIS in <em>E. coli</em>, geranylgeraniol (GGOH) was detected, confirming that <em>triTPS4</em> encodes a functional GGPP synthase (Supplementary Fig. <a href="#MOESM1" class="usa-link">S4</a>).</p>
<p id="Par8">Given that the scaffolds of harzianes and trichodermanins may be synthesized by noncanonical TCs, we performed enzymatic activity-guided protein fractionation combined with comparative transcriptomic analysis to identify the target enzyme. Firstly, an enzymatic assay approach was established using crude protein extracts from <em>T. atroviride</em> B7 with GGPP as the substrate. Two enzymatic products were detected, which were then traced and purified from the fermentation broth of <em>T. atroviride</em> B7 and identified as harzianol I (<strong>1</strong>) and wickerol A (<strong>2</strong>) through comparing their <sup>1</sup>H and <sup>13</sup>C NMR spectra with spectroscopic data in the literature (Fig. <a href="#Fig2" class="usa-link">2a</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">S5</a>, <a href="#MOESM1" class="usa-link">37</a>‒<a href="#MOESM1" class="usa-link">40</a>)<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Notably, this enzymatic activity was Mg<sup>2+</sup>-dependent (Supplementary Fig. <a href="#MOESM1" class="usa-link">S6</a>). Next, the crude protein was subjected to sequential fractionation using ammonium sulfate precipitation (ASP) and ion-exchange chromatography (IEC), which afforded three highly active fractions and one weakly active control fraction (Supplementary Fig. <a href="#MOESM1" class="usa-link">S7</a>). LC–MS/MS analysis of the four fractions, referenced against the <em>T. atroviride</em> B7 genome, identified 731, 579, 702, and 647 proteins, respectively (Supplementary Table <a href="#MOESM1" class="usa-link">S2</a>). Meanwhile, we attempted to culture <em>T. atroviride</em> B7 in different broth media and found that TB3 broth could produce diterpenoids <strong>1</strong>‒<strong>4</strong>, while CD broth did not (Supplementary Fig. <a href="#MOESM1" class="usa-link">S8</a>). Compound <strong>3</strong> was determined as harzianone through purification from <em>T. atroviride</em> B7 fermentation broth and comparison of its NMR data with those in literature (Supplementary Figs. <a href="#MOESM1" class="usa-link">S41</a>, <a href="#MOESM1" class="usa-link">42</a>)<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, while compound <strong>4</strong> was identified as harziandione through comparing retention time and MS spectrum with an authentic sample previously isolated from this strain<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Transcriptome sequencing of the mycelia cultured in TB3 and CD broth, integrated with the protein LC-MS/MS results, yielded 49 differentially expressed candidate genes for further study (Supplementary Data File <a href="#MOESM3" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Functional characterization of TriDTCs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370890_41467_2025_63055_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6d37/12370890/53e5f14b1819/41467_2025_63055_Fig2_HTML.jpg" loading="lazy" id="d33e752" height="725" width="675" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> GC–MS extracted ion chromatogram (EIC, <em>m/z</em> 272) of enzymatic assays using <em>T. atroviride</em> B7 protein extracts: (i) Crude protein incubated with GGPP; (ii) Purified compound <strong>1</strong> standard; (iii) Purified compound <strong>2</strong> standard; (iv) Heat-inactivated control (boiled crude protein with GGPP); (v) Negative control (crude protein without GGPP). <strong>b</strong> EIC (<em>m/z</em> 272) of heterologous expression products in <em>E. coli</em> BL21 strains containing: (i) pBbA5c-MevT-MBIS, pET-28a-<em>triTPS4</em>, and pCold-TF-<em>tri4155</em>; (ii) pBbA5c-MevT-MBIS, pCDF-Duet1-<em>crtE</em>, and pCold-TF-<em>tri4155</em>; (iii) pBbA5c-MevT-MBIS, pCDF-Duet1-<em>crtE</em>, and pET-28a-<em>tri4155</em>; and (iv) pBbA5c-MevT-MBIS and pET-28a-<em>triTPS4</em>. Peak in the yellow column is GGOH. <strong>c</strong> Functional analysis in <em>A. oryzae</em> NSAR1 systems containing: (i) PTAex3-P<em>tri4155</em>-<em>tri4155</em>, (ii) PTAex3-P<em>amyB</em>-<em>tri4155</em>, and (iii) PTAex3 (negative control). <strong>d</strong> In vitro enzymatic tests using TriDTCs purified from <em>E. coli</em> system: (i) Tri4155 incubated with GGPP; (ii) Tri4155<sup>V409A/L335A</sup> with GGPP; (iii) Tcie612 with GGPP; (iv) Boiled Tri4155 with GGPP.</p></figcaption></figure><p id="Par9">These candidates were cloned from <em>T. atroviride</em> B7 cDNA and expressed in the engineered <em>E. coli</em> BL21 system. Among them, only <em>tri4155</em>, annotated as a hypothetical gene with FPKM values of 341.2 in TB3 and 7.8 in CD broth, produced trace amounts of compound <strong>1</strong> when expressed using the pET-28a vector (Fig. <a href="#Fig2" class="usa-link">2b</a>, Supplementary Data File <a href="#MOESM3" class="usa-link">1</a>). Switching the expression vector of <em>tri4155</em> from pET-28a to pCold-TF and co-expressing with <em>triTPS4</em> for GGPP biosynthesis boosted <strong>1</strong> production, with the yield of 3.55 ± 0.85 mg L<sup>−1</sup> in the shake flasks, and compound <strong>2</strong> was produced as a minor product with a yield of 0.20 ± 0.05 mg L<sup>−1</sup> (Fig. <a href="#Fig2" class="usa-link">2b</a> and Fig. <a href="#Fig3" class="usa-link">3c</a>). Although both CrtE and TriTPS4 can provide the GGPP precursor, the higher yield observed with co-expression of <em>tri4155</em> and <em>triTPS4</em> suggests potential synergistic catalysis between these <em>Trichoderma</em>-derived enzymes. Furthermore, <em>tri4155</em> was introduced into <em>A. oryzae</em> NSAR1 under the constitutive promoter P<em>amyB</em>, resulting in the production of trace amounts of <strong>1</strong> (Fig. <a href="#Fig2" class="usa-link">2c</a>). Replacing P<em>amyB</em> promoter with the native promoter P<em>tri4155</em> (700 bp sequence upstream of <em>tri4155</em>) markedly increased the production of <strong>1</strong> and <strong>2</strong>, yielding 10.09 ± 2.08 mg L<sup>−1</sup> and 1.55 ± 0.39 mg L<sup>−1</sup>, respectively (Figs. <a href="#Fig2" class="usa-link">2</a>c and <a href="#Fig4" class="usa-link">4d</a>). In addition, in situ knockout of <em>tri4155</em> in <em>T. atroviride</em> B7 using a split-marker homologous recombination strategy abolished the production of <strong>1</strong> and <strong>2</strong> (Supplementary Figs. <a href="#MOESM1" class="usa-link">S9</a>, <a href="#MOESM1" class="usa-link">S10</a>). Notably, the activity of Tri4155 depends on the presence of Mg<sup>2+</sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">S11</a>), consistent with the Mg<sup>2+</sup>-dependence observed in crude protein extracts from <em>T. atroviride</em> B7. However, the recombinant Tri4155 protein purified from <em>E. coli</em> exhibited relatively low enzymatic activity (Fig. <a href="#Fig2" class="usa-link">2d</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">S11</a>), and the expression level of C-terminal 8×His-tagged Tri4155 in <em>A. oryzae</em> NSAR1 was low (Supplementary Fig. <a href="#MOESM1" class="usa-link">S12</a>), limiting its use for detailed biochemical analysis</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Mechanistic analysis of TriDTC catalysis.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370890_41467_2025_63055_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6d37/12370890/60b7c3a8b690/41467_2025_63055_Fig3_HTML.jpg" loading="lazy" id="d33e923" height="664" width="674" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Structure model of Tri4155 predicted by AlphaFold2 and docked with GGPP substrate (pyrophosphate group shown in red). <strong>b</strong> Predicted catalytic cavity residues with a docked GGPP. Residue carbons in the highly charged and hydrophobic regions are shown in yellow and green, respectively. <strong>c</strong> Production of compounds <strong>1</strong> and <strong>2</strong> by Tri4155 variants expressed in <em>E. coli</em> BL21, showing differential effects of residues in highly charged and hydrophobic regions. Productions are represented as mean values ± SD (<em>n</em> = 3 biological replicates). Source data are provided as a Source Data file.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Phylogenetic and functional analysis of TriDTC homologs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370890_41467_2025_63055_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6d37/12370890/c80218f16dcf/41467_2025_63055_Fig4_HTML.jpg" loading="lazy" id="d33e966" height="800" width="776" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Neighbor-joining tree of TCs including: TriDTCs (red), canonical type I TCs from bacteria (purple), fungi (brown), plants (green), and octocorals (yellow), as well as noncanonical TCs (pink). <strong>b</strong> Expanded view of the TriDTC clade with characterized enzymes labeled. The functional members are highlighted in red. <strong>c</strong> In vivo enzymatic production of compounds <strong>1</strong> and <strong>2</strong> by TriDTCs expressed in <em>E. coli</em> BL21 system. <strong>d</strong> Production of compounds <strong>1</strong> and <strong>2</strong> by TriDTCs expressed in <em>A. oryzae</em> NSAR1 system. Productions are represented as mean values ± SD (<em>n</em> = 3 biological replicates). Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par10">To enable more robust in vitro enzymatic characterization of TriDTC, the <em>T. citrinoviride</em> homolog Tcie612 was selected for heterologous expression. Tcie612 shares 60.41% sequence identity with Tri4155 and exhibited significantly enhanced yield of <strong>1</strong> when expressed in the <em>E. coli</em> system (33.34 ± 4.71 mg L<sup>−1</sup>), despite no previous reports of these compounds in <em>T. citrinoviride</em>. When incubated with GGPP and Mg<sup>2+</sup>, the recombinant Tcie612 protein produced compound <strong>1</strong> with improved production compared to Tri4155 (Fig. <a href="#Fig2" class="usa-link">2d</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">S11</a>), but failed to produce compound <strong>2</strong> either in vitro or heterologously expressed in <em>E. coli</em> or <em>A. oryzae</em>, suggesting the functional divergence of Tcie612 and Tri4155 that warrants further investigation. To determine the substrate specificity, Tcie612 was incubated with FPP or GFPP, but no cyclized product was detected, indicating that Tcie612 is GGPP specific (Supplementary Fig. <a href="#MOESM1" class="usa-link">S11</a>). Additionally, incubation of Tcie612 either with DMAPP and three equivalents of IPP, or with equal equivalents of FPP and IPP, did not yield any specific product, revealing that Tcie612 is a monofunctional diterpene cyclase (DTC), unlike fungal chimeric DTCs that possess both prenyltransferase and terpene synthase domains. In addition to Mg<sup>2+</sup>, some TCs can utilize other divalent metal cofactors<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, such as Mn<sup>2+</sup> and Co<sup>2+</sup>. We then tested the enzyme activity of Tcie612 with GGPP as a substrate in the presence of various metal ions, which revealed the following activity preference: Mg<sup>2+</sup> &gt; Co<sup>2+</sup> &gt; Mn<sup>2+</sup> = Fe<sup>2+</sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">S11</a>). To determine the number of metal ions per enzyme, isothermal titration calorimetry (ITC) experiments were applied. When titrating Tcie612 pre-incubated with GGPP using MgCl<sub>2</sub> solution, exothermic signals were observed, corresponding to Mg<sup>2+</sup> binding. The data analysis revealed two Mg<sup>2+</sup> and one GGPP binding per enzyme, consistent with docking results (Supplementary Fig. <a href="#MOESM1" class="usa-link">S13</a>). Interestingly, no binding effect was observed when titrating Tcie612 without GGPP incubation, revealing that Mg<sup>2+</sup> binding depends on both the enzyme and substrate presence (Supplementary Fig. <a href="#MOESM1" class="usa-link">S13</a>). The kinetic parameter of Tcie612 for GGPP was also measured, with a <em>K</em><sub>M</sub> value of 86.1 ± 8.19 μM (Supplementary Fig. <a href="#MOESM1" class="usa-link">S14</a>). Collectively, the results of heterologous expressions in <em>E. coli</em> and <em>A. oryzae</em> NSAR1, gene deletion in <em>T. atroviride</em> B7, and in vitro enzyme assays indicated that Tri4155 and Tcie612 are responsible for the biosynthesis of <em>Trichoderma</em>-characteristic diterpenoids.</p></section><section id="Sec4"><h3 class="pmc_sec_title">TriDTCs possess a distinct yet convergent catalytic mechanism</h3>
<p id="Par11">Tri4155 was initially annotated as a hypothetical protein, and it lacks the conserved motifs of classical type I and II TCs, such as DDxxD, NSE/DTE, RY pairs, and DxDD. Moreover, InterProScan analysis revealed that Tri4155 does not contain conserved domains characteristic of any known protein families<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Phylogenetic analysis of Tri4155, alongside classical type I DTCs from fungi, bacteria, plants, and marine organisms, as well as previously reported noncanonical TCs, showed that Tri4155 is phylogenetically distinct from classical type I DTCs and does not cluster with any known noncanonical TCs (Fig. <a href="#Fig4" class="usa-link">4a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">S15</a>). Therefore, Tri4155 represents a novel family of TCs, which we named TriDTC. Interestingly, phomactatriene synthase (PhmA)<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> and taxadiene synthase (TXS)<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, which likely produce a common intermediate, verticillen-12-yl cation, with TriDTC during their cyclization process (Supplementary Fig. <a href="#MOESM1" class="usa-link">S16</a>), are found in separate clades in the phylogenetic tree (Fig. <a href="#Fig4" class="usa-link">4a</a>). TXS clusters with the classical TCs characterized by conserved motifs, while PhmA represents a divergent TC subfamily lacking RY pairs.</p>
<p id="Par12">To elucidate the catalytic mechanism of TriDTC, we initially attempted to determine the crystal structure of Tri4155. However, the recombinant protein expressed in <em>E. coli</em> system was unstable and tended to aggregate. As an alternative, we turned to conduct a protein structure prediction on Tri4155 using AlfhaFold2, generating a simulated model (Fig. <a href="#Fig3" class="usa-link">3a</a>). The substrate binding pockets of the model were analyzed, and GGPP was energy-minimized and docked into these predicted active cavities to generate an optimum pocket (Fig. <a href="#Fig3" class="usa-link">3b</a>). Notably, the key residues in the pocket are within the highly conserved region across TriDTCs (Supplementary Fig. <a href="#MOESM1" class="usa-link">S21</a>), indicating the reliability of the predicted catalytic cavity. To experimentally validate this model, we constructed 23 alanine-substitution variants targeting residues within the cavity. Among these, 17 variants exhibited either dramatically reduced or completely abolished diterpene production (Fig. <a href="#Fig3" class="usa-link">3c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">S17</a>), which further confirmed the reliability of the generated pocket. Compared to TXS, which has a larger active-site volume relative to its substrate GGPP, allowing for high intermediate flexibility and product promiscuity<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, the active site contour of Tri4155 is markedly more compact and constrained (Supplementary Fig. <a href="#MOESM1" class="usa-link">S18</a>), likely contributing to the strict substrate selectivity and product fidelity of Tri4155.</p>
<p id="Par13">Like classical TCs, the predicted cavity of Tri4155 is bifacial, consisting of a highly charged region surrounding the pyrophosphate moiety at the entrance of the pocket and a hydrophobic region at the base (Fig. <a href="#Fig3" class="usa-link">3b</a>). The highly charged region primarily consists of polar residues, and mutagenesis of these residues resulted in dramatic reduction or complete loss of diterpene production (Fig. <a href="#Fig3" class="usa-link">3c</a>). Tri4155 lacks the classical metal-binding motifs typically found in TCs; instead, an aspartate triad (D<sup>445</sup>VGD<sup>448</sup>LCAD<sup>452</sup>) is positioned surrounding the pyrophosphate moiety, likely interacting with divalent metal ions to initiate allyl cation formation. Supporting this hypothesis, alanine substitutions of the aspartate triad (D445A, D448A, and D452A) completely abolished diterpene production. In contrast to TXS and other classical type I TCs that contain a conserved H-bonding cluster (YRRRY) in the cavity, appearing to bind bulk water to prevent premature carbocation quenching<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, Tri4155 lacks this cluster, which may facilitate the formation of hydroxylated products. In the hydrophobic region, over half of the residues could be mutated with minimal impact on enzyme activity, while L335A and V409A variants actually enhanced diterpene production (Fig. <a href="#Fig3" class="usa-link">3c</a>). In contrast, substitutions of aromatic residues with Ala (H318A, Y410A, and F321A) led to a substantial reduction in diterpene production, probably due to their essential roles in stabilizing intermediate cations via π–π and π–cation interactions. Compared to TXS containing four aromatic residues in the hydrophobic region proposed to stabilize the Δ<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup> position of verticillen-12-yl cation via π–π interaction and promote the cascade to cation <strong>g</strong> (Supplementary Fig. <a href="#MOESM1" class="usa-link">S16</a>)<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>, Tri4155 contains only three aromatic residues that likely facilitate the conversion of verticillen-12-yl cation to intermediate <strong>b</strong>, which is subsequently converted into harzianes and trichodermanins. Two residues located at the entrance of the cavity, V412 and L317, appear to function as “gatekeepers” controlling substrate ingress. Alanine substitutions at these positions caused a drastic decrease in diterpene production (Fig. <a href="#Fig3" class="usa-link">3c</a>), and structural modeling revealed the compromised cavity integrity in these variants (Supplementary Fig. <a href="#MOESM1" class="usa-link">S19</a>). Conversely, the V412F and L317F variants partly restored compound <strong>1</strong> production and displayed constrictive cavity entrances in modeling studies. Notably, both V409A and L335A variants increased the production of minor product <strong>2</strong> (Fig. <a href="#Fig3" class="usa-link">3c</a>), with V409A showing a more pronounced effect. The double variant L335A/V409A led to a further increase of compound <strong>2</strong> in both in vitro and in vivo tests (Figs. <a href="#Fig2" class="usa-link">2</a>d and <a href="#Fig3" class="usa-link">3c</a>). Structural modeling suggested that V409 was adjacent to the C-13 cation at intermediate <strong>c</strong> (Supplementary Fig. <a href="#MOESM1" class="usa-link">S20</a>), likely engaging in a hydrophobic interaction that may influence substrate orientation. The V409A substitution likely weakened this interaction, thereby shifting the enzymatic product profile in favor of compound <strong>2</strong>.</p></section><section id="Sec5"><h3 class="pmc_sec_title">TriDTCs exhibit genus-specific function in <em>Trichoderma</em>
</h3>
<p id="Par14">To investigate the evolutionary origin of Tri4155, we performed a BLASTP search using its amino acid sequence against the public nonredundant protein sequence database. A total of 38 homologous proteins were found, with sequence identities ranging from 47 to 97% (Fig. <a href="#Fig4" class="usa-link">4b</a>). Notably, 25 homologous proteins were from the <em>Trichoderma</em> genus (Hypocreaceae), while 7 were from the <em>Metarhizium</em> genus (Clavicipitaceae) and 6 were from the <em>Ophiocordyceps</em> genus (Ophiocordycipitaceae). To further characterize their functions, we selected 13 representative homologs from <em>Trichoderma</em>, <em>Metarhizium</em>, and <em>Ophiocordyceps</em>, representing each differentiated branch in the phylogenetic tree (Fig. <a href="#Fig4" class="usa-link">4b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">S21</a>). These genes were codon-optimized and expressed in engineered <em>E. coli</em> BL21 harboring both pBbA5c-MevT-MBIS and pET-28a-<em>triTPS4</em> plasmids. As a result, seven <em>Trichoderma</em>-derived homologs produced compounds <strong>1</strong> and <strong>2</strong>, whereas the remaining homologs, including those from <em>Ophiocordyceps</em> and <em>Metarhizium</em>, showed no detectable enzyme activity (Fig. <a href="#Fig4" class="usa-link">4c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">S22</a>). In addition, all 13 homologous genes were expressed in <em>A. oryzae</em> NSAR1 under the control of the P<em>tri4155</em> promoter. The results were found to be similar to those observed in the <em>E. coli</em> system, that seven homologs from <em>Trichoderma</em> produce diterpenes, while the homologs from <em>Metarhizium</em> and <em>Ophiocordyceps</em> remained nonfunctional (Fig. <a href="#Fig4" class="usa-link">4d</a>). Notably, the relative yields of the two products differed between the two heterologous systems, suggesting host-specific influences on enzyme performance.</p>
<p id="Par15">Intriguingly, <em>Metarhizium</em> and <em>Ophiocordyceps</em> are genera known to colonize insects, while <em>Trichoderma</em> is considered to share a last common ancestor with entomoparasitic hypocrealean fungi<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Based on the distribution and function of TriDTC homologs across fungal genera, we proposed that TriDTCs originated from the <em>Trichoderma</em> genus, probably through de novo evolution from non-coding genomic regions. The acquisition of the TriDTC function likely occurred after the diversification into infrageneric groups, as some <em>Trichoderma</em> species harbor inactive TriDTC homologs. Horizontal gene transfer (HGT) may have facilitated the introduction of TriDTC homologs into <em>Metarhizium</em> and <em>Ophiocordyceps</em> lineages; however, catalytic function appears to have been lost in these recipient genera. This proposed evolutionary trajectory is consistent with the highly specific distribution of harziane and trichodermanin diterpenoids. The gain of the TriDTC function during <em>Trichoderma</em> evolution may play an important role in environmental adaptation, potentially offering ecological advantages that contributed to the success of this genus in diverse habitats.</p></section><section id="Sec6"><h3 class="pmc_sec_title">TriDTCs and harzianol I regulate <em>Trichoderma</em> chlamydospore and <em>A. oryzae</em> sclerotia formations</h3>
<p id="Par16">To explore the biological role of TriDTCs, we evaluated the growth rates of the wild-type (WT) and Δ<em>tri4155 T. atroviride</em> B7 strains. The mutant strain exhibited an approximately 20% increase in growth radius compared to the WT strain after culturing on the PDA plates for two days (Fig. <a href="#Fig5" class="usa-link">5a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">S23</a>). Considering that harzianes have been reported to possess anti-phytopathogenic activity in vitro, we conducted antagonism assays against the fungus <em>Fusarium graminearum</em>, a major causative pathogen of wheat scab. Consequently, both the WT and Δ<em>tri4155</em> strains inhibited the growth of <em>F. graminearum</em>, likely by rapidly occupying the growth niches (Fig. <a href="#Fig5" class="usa-link">5b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">S24</a>), indicating that Tri4155 and its diterpenoid products are not essential for <em>Trichoderma</em>’s antifungal properties. Remarkably, after seven days of growth on PDA, WT strains developed dense white aerial hyphae containing large, spherical, and thick-walled chlamydospores at the tips of the hyphae, while the Δ<em>tri4155</em> strain failed to form chlamydospores but produced thinner aerial hyphae compared to the WT strain (Fig. <a href="#Fig5" class="usa-link">5c</a> i, ii, Supplementary Figs. <a href="#MOESM1" class="usa-link">S25</a>‒<a href="#MOESM1" class="usa-link">27</a>). In addition, the hyphae of the mutant strain were more hydrophobic and displayed a greater number of short branches under the microscope (Supplementary Fig. <a href="#MOESM1" class="usa-link">S28</a>). These results indicate that Tri4155 plays an important role in chlamydospore formation and hyphal morphogenesis.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. <em>triDTCs</em> and harzianol I (1) regulate <em>Trichoderma</em> chlamydospore and <em>A. oryzae</em> sclerotia formation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370890_41467_2025_63055_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6d37/12370890/86ea5c9347b4/41467_2025_63055_Fig5_HTML.jpg" loading="lazy" id="d33e1507" height="374" width="667" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Colony growth comparison between wild-type (WT) and Δ<em>tri4155 T. atroviride</em> B7 strains after 2 days of culture. The growth radius is represented as mean radius ± SD (<em>n</em> = 3 biological replicates. Statistical analysis was performed using <em>t</em>-test). <strong>b</strong> Antagonism assays of <em>T. atroviride</em> B7 against <em>F. graminearum</em> after 6 days: (i) WT <em>T. atroviride</em> (bottom) <em>vs. F. graminearum</em> (top); (ii) Δ<em>tri4155</em> mutant (bottom) <em>vs. F. graminearum</em> (top); (iii) self-pairing control of <em>F. graminearum</em>. <strong>c</strong> Fungal phenotypes after 7 days of culture: (i) WT showing normal hyphae and chlamydospores; (ii) Δ<em>tri4155</em> with aberrant hyphae and lacking chlamydospores; (iii) Δ<em>tri4155</em> supplemented with 10 μM compound <strong>1</strong> showing conidiation; (iv) Δ<em>tri4155</em> supplemented with 400 μM compound <strong>1</strong> restoring chlamydospore formation. scale bar = 50 μm. Each experiment contains more than 3 independent repeats. <strong>d</strong> Chlamydospore formation in <em>T. asperellum</em> and <em>T. asperellum-</em>Δ<em>triDTC</em> (1-day culture), as well as in <em>T. gamsii</em> and <em>T. gamsii-</em>Δ<em>triDTC</em> (3-day cultures). scale bar = 50 μm. Each experiment contains 3 independent repeats. <strong>e</strong> Sclerotia formation in <em>A. oryzae</em> NSAR1 (control), <em>A. oryzae</em>-<em>tri4155</em> (heterologous expression of <em>tri4155</em> gene), and <em>A. oryzae</em> treated with compound <strong>1</strong> (<em>A. oryzae</em>-S<strong>1</strong>) or <strong>2</strong> (<em>A. oryzae</em>-S<strong>2</strong>). Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par17">Furthermore, we conducted complementary experiments using compounds <strong>1,</strong>
<strong>2</strong>, and <strong>3</strong> to test their rescue effects on chlamydospore formation. The results showed that all three compounds stimulated the <em>Δtri4155</em> strain to form conidiospores in a concentration-dependent manner, with compound <strong>1</strong> being the most effective (Supplementary Figs. <a href="#MOESM1" class="usa-link">S26</a>, <a href="#MOESM1" class="usa-link">S27</a>). However, none of the compounds could rescue chlamydospores formation of the Δ<em>tri4155</em> strain at concentrations up to 40 μM. Given that the water solubility of compound <strong>1</strong> is poor, and higher concentrations should be needed to saturate the medium to attain an effective intracellular concentration, we increased the concentration of compound <strong>1</strong> to 400 μM in the medium and then inoculated the Δ<em>tri4155</em> strain. As anticipated, chlamydospore formation was rescued after seven days of culture (Fig. <a href="#Fig5" class="usa-link">5c</a> iv), and the hyphae reverted to a less-branched morphology, resembling that of the WT strain (Supplementary Fig. <a href="#MOESM1" class="usa-link">S28</a>). These findings suggest that <em>tri4155</em> is involved in chlamydospore formation through producing harzianol I (<strong>1</strong>).</p>
<p id="Par18">To verify the function of TriDTCs in <em>Trichoderma</em> species, four additional <em>Trichoderma</em> species were obtained and identified as <em>T. asperellum</em>, <em>T. gamsii</em>, <em>T. reesei</em>, and <em>T. harzianum</em>, respectively. When cultured in liquid media for three days, <em>T. asperellum</em>, <em>T. gamsii</em>, and <em>T. atroviride</em> B7 produced substantial amounts of harziane and trichodermanin diterpenoids <strong>1</strong>‒<strong>4</strong>, while only trace amounts of these diterpenoids were detected in <em>T. reesei</em> and <em>T. harzianum</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">S29a</a>). Microscopic observations revealed that <em>T. asperellum</em>, <em>T. gamsii</em>, and <em>T. atroviride</em> B7 formed chlamydospores, while no chlamydospore was observed in <em>T. reesei</em> or <em>T. harzianum</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">S29c</a>). Solid media cultures showed similar trends, with <em>T. asperellum</em> and <em>T. harzianum</em> additionally producing abundant conidiospores (Supplementary Fig. <a href="#MOESM1" class="usa-link">S29b, c</a>). These findings suggest that the chlamydospore formation depends on the diterpenoid production in <em>Trichoderma</em> species. Supplementation with 400 μM compound <strong>1</strong> significantly enhanced chlamydospore formation in <em>T. harzianum</em>. In <em>T. reesei</em>, compound <strong>1</strong> induced sclerotia-like hyphal aggregates in solid media, whereas in liquid culture, it promoted chlamydospore formation (Supplementary Fig. <a href="#MOESM1" class="usa-link">S30</a>). To genetically validate these findings, the homologous genes of <em>tri4155</em> in <em>T. asperellum</em> and <em>T. gamsii</em> were knocked out using a split maker strategy (Supplementary Fig. <a href="#MOESM1" class="usa-link">S31</a>). The resulting mutants showed abolished diterpenoid production and impaired chlamydospore formation, accompanied by increased hyphae aggregation in liquid culture (Fig. <a href="#Fig5" class="usa-link">5d</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">S31</a>), consistent with enhanced surface hydrophobicity.</p>
<p id="Par19">We further tested the function of <em>tri4155</em> and its products in the model filamentous fungus <em>A. oryzae</em>, which has an asexual reproduction cycle involving mycelia, conidiospores, and sclerotia. The sclerotia of <em>A. oryzae</em> are considered a degenerate sexual structure that no longer produces spores<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> but forms a compact mass of hardened mycelia, allowing the fungus to survive in extreme environments<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. In our observation, <em>A. oryzae</em> NSAR1 began to produce conidiospores after 1–2 days of culture on PDA plates. On day five, white sclerotia with highly branched hyphae formed peripherally to the conidiospore circle. These structures progressively turned into mature sclerotia with thick brown outer layers (Supplementary Fig. <a href="#MOESM1" class="usa-link">S32</a>). In contrast, the strains heterologously expressing <em>tri4155</em> produced significantly denser white sclerotia that preferentially overlying conidiospore circle by day five, with significantly smaller conidiospore circle (Fig. <a href="#Fig5" class="usa-link">5e</a>). Similar to the effect of <em>tri4155</em> heterologous expression, compound <strong>1</strong> also promoted sclerotia formation in <em>A. oryzae</em>, while compound <strong>2</strong> had no effect (Fig. <a href="#Fig5" class="usa-link">5e</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">S33</a>). These results suggested that <em>tri4155</em> promotes sclerotia formation through producing harzianol I, mirroring its role in <em>Trichoderma</em>.</p></section></section><section id="Sec7"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par20">Although the rapid development of genome sequencing and bioinformatic technology has greatly promoted the identification of new biosynthetic enzymes of natural products, discovering noncanonical enzymes lacking typical conserved motifs remains a formidable challenge. Harzianes and trichodermanins have been reported from <em>Trichoderma</em> fungi for decades and exhibit various bioactivities, including antifungal and antiviral properties. Despite considerable efforts to elucidate their biosynthesis, the key enzymes have long been a mystery. In this work, we demonstrated an effective strategy for mining noncanonical enzymes. Using a combination of enzymatic activity-guided protein fractionation and comparative transcriptome analysis, verified by heterologous expression, gene deletion, and in vitro enzyme assays, we discovered a class of noncanonical enzymes, named TriDTCs, responsible for the formation of harziane and trichodermanin scaffolds from <em>Trichoderma</em>. This approach offers an effective complement to sequence mining in the study of natural product biosynthesis. Importantly, this study represents just one successful case, while fungal genomes contain numerous uncharacterized proteins and harbor vast, unexplored biosynthetic potential awaiting discovery.</p>
<p id="Par21">TriDTCs represent an unprecedented subfamily of TCs that defy conventional classification. They represent an independent and unique branch of TCs, lacking all the conserved motifs of typical terpene synthases and highly differentiating from other noncanonical TCs, including YtpB and AsR6. Despite their highly divergent primary sequences, TriDTCs share fundamental mechanistic and structural features with TCs: (1) their enzymatic activity depends on divalent metals, (2) they possess bifacial catalytic cavities, (3) they employ an aspartate triad for metal binding, and (4) they adopt characteristic α-helical folds forming barrel-shaped cavities, according to the AlphaFold2 structural predictions and site-directed mutagenesis (Supplementary Fig. <a href="#MOESM1" class="usa-link">S34</a>). However, TriDTCs contain significantly more loop regions compared to canonical TCs, and the spatial arrangement and orientations of the helices constituting the catalytic cavity differ considerably from those of known TC families. The structural divergences suggest that TriDTCs may have followed a distinct evolutionary trajectory from other type I TCs. Interestingly, despite pronounced structural differentiation, the shared structural features partially explain how TriDTCs, TXS, and PhmA utilize an identical substrate and initiate similar cyclization cascades, while variations in cavity architecture likely underlie their distinct product profiles, suggesting an intriguing mechanistic dimension that warrants future investigation. However, our mutagenesis studies failed to redirect product outcomes beyond harziane and trichodermanin scaffolds, indicating a high product fidelity of TriDTCs and a multifactor-controlled mechanism of the product specificity. More importantly, the common structural features shared between TriDTCs and other type I TCs (e.g., the α-helical folds and a barrel-shaped cavity) indicate the universal properties of all type I TCs and provide valuable insights for identifying additional noncanonical TCs based on protein structure simulation databases. However, in the absence of an experimentally resolved crystal structure and given that TriDTCs represent a novel protein family, the results of our structural predictions are still preliminary. Further experimental studies, such as X-ray crystallography or cryo-EM, would be essential to determine the structure of this unique enzyme family.</p>
<p id="Par22">TriDTCs exhibit an unusually narrow taxonomic distribution, with functional activity largely restricted to <em>Trichoderma</em> species. We propose that TriDTCs might have originated de novo from non-coding genomic regions within the <em>Trichoderma</em> genus and gained function during evolution, as evidenced by their absence in closely related taxa and predominantly intron-less gene structures. The presence of catalytically inactive homologs in <em>Metarhizium</em> and <em>Ophiocordyceps</em>, two phylogenetically distant genera, probably resulted from HGT events. The non-<em>Trichoderma</em> TriDTCs lack three regions corresponding to the N-terminal loop of <em>Trichoderma</em>-derived TriDTCs (Supplementary Fig. <a href="#MOESM1" class="usa-link">S21</a>), which we proved to be essential for enzymatic activity, as the N-terminal truncation Tri4155 variant (Tri4155<sup>Δ1–250</sup>) completely abolished compound production (Supplementary Fig. <a href="#MOESM1" class="usa-link">S35</a>). Notably, <em>Trichoderma</em> homologues, such as Tvis324, possess the intact N-terminal loop but still exhibit no detectable activity, suggesting that other residues are also essential for enzymatic activity. Interestingly, although <em>triDTC</em> is not clustered with the GGPP synthase gene, the surrounding genomic regions in both <em>T. atroviride B7</em> and <em>T. asperellum</em> harbor two putative cytochrome P450 genes, two FAD-binding genes, and one SAM-dependent methyltransferase gene that may participate in downstream metabolite biosynthesis, warranting further investigation (Supplementary Fig. <a href="#MOESM1" class="usa-link">S36</a>).</p>
<p id="Par23">While fungal natural products are often studied for their antimicrobial potential as defensive chemicals, relatively less attention has been devoted to understanding their roles in regulating fungal physiology. Although several metabolites have been implicated in fungal development, such as <em>A. nidulans</em> psi factor<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, <em>F. graminearum</em> zearalenone<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, <em>A. terreus</em> butyrolactone I<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, and <em>Penicillium cyclopium</em> conidiogenone<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, none have been reported to influence resistant propagule formation. Our work demonstrates that <em>Trichoderma</em> has evolved TriDTCs specifically to produce unique diterpenoids regulating chlamydospore development. While previous studies have reported the antifungal activity of harzianes, our findings indicated that these chemicals function primarily in physiological regulation rather than defense. Although TriDTCs naturally function only in the <em>Trichoderma</em> genus, their heterologous expression in <em>A. oryzae</em> significantly improved sclerotia formation. Chlamydospores and sclerotia are two kinds of resistant propagules widely existing in filamentous fungi, enabling them to adapt to extreme environmental conditions. Our results indicate a conserved regulation pathway underlying the formation of chlamydospores and sclerotia. While previous research has reported the involvement of velvet family proteins in the regulation of sporulation and morphogenesis in <em>T. reesei</em><sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, and G-protein signaling pathway in <em>A. oryzae</em> sclerotia formation<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, the molecular mechanisms governing chlamydospore formation remain completely unexplored. Our work provides the first molecular handle for investigating this important fungal survival strategy, and TriDTCs and harzianol I hold promise for improving fungal viability during long-term storage.</p>
<p id="Par24">In summary, by integrating enzymatic activity-guided protein fractionation, comparative transcriptome analysis, in vivo and in vitro enzymatic assays, and in situ gene knockout, we demonstrated that <em>Trichoderma</em> fungi employ an unprecedented family of enzymes, named TriDTCs, to catalyze the formation of the characteristic, harziane and trichodermanin diterpenoids. Structural modeling and site-directed mutagenesis demonstrated the catalytic mechanism, identifying the DxxDxxxD Asp-triad, V409, and the “gatekeeper” residues crucial for enzyme activity and product profile. Phylogenetic analysis and homolog characterization revealed the specific distribution and highly functional specificity of TriDTCs in <em>Trichoderma</em>. Biological function research through gene deletions, heterologous expression, and compound supplementation revealed that TriDTCs and their products regulate the formation of fungal-resistant propagules. In addition, we identified a GGPPS and a promoter from <em>Trichoderma</em> fungi for heterologous production of harziane and trichodermanin diterpenoids. Our study exemplifies how fungi have evolved specialized biosynthetic enzymes to produce physiologically important chemicals as survival strategies. These findings shed new light on the biosynthetic machinery and biological functions of fungal terpenoids, herald a promising strategy to enhance the biocontrol efficacy of <em>Trichoderma</em> fungi in agricultural practices, and open avenues for producing these unique natural products for further pharmaceutical investigation and applications.</p></section><section id="Sec8"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec9"><h3 class="pmc_sec_title">General methods</h3>
<p id="Par25">GC-MS analysis was performed using an Agilent 7890 A GC system equipped with an Agilent 5795 C Inert MSD Triple-Axis detector and a 30 m × 250 μm × 0.25 μm column, with high-purity helium as the carrier gas. The temperature program was as follows: 80–220 °C at 15 °C min<sup>−1</sup>, then 220–270 °C at 4 °C min<sup>−1</sup>, followed by a 2 min hold. The flow rate was maintained at 1.2 mL min<sup>−1</sup>, with 5 μL injections in splitless mode. The inlet and detector temperatures were set at 250 °C and 300 °C, respectively. Column chromatography was performed using 200–300 mesh silica gel (Qingdao Marine Chemical Factory), and thin-layer chromatography (TLC) was conducted on silica gel plates (GF254, 10–40 μm). TLC spots were visualized under UV light or by heating after spraying with 10% H<sub>2</sub>SO<sub>4</sub> in EtOH (v/v). Analytical grade solvents (n-hexane, petroleum ether, EtOAc) were used for extraction and analysis. NMR spectra were acquired on a Bruker AVANCE III-600 spectrometer using TMS as an internal standard.</p>
<p id="Par26"><em>E. coli</em> cultures used TB broth (12 g L<sup>−1</sup> tryptone, 24 g L<sup>−1</sup> yeast extract, 25 mL L<sup>−1</sup> glycerol, 2.31 g L<sup>−1</sup> KH<sub>2</sub>PO<sub>4</sub>, 12.54 g L<sup>−1</sup> K<sub>2</sub>HPO<sub>4</sub>), while protein purification employed LB broth (10 g L<sup>−1</sup> tryptone, 5 g L<sup>−1</sup> yeast extract, 10 g L<sup>−1</sup> NaCl). <em>A. oryzae</em> was cultured in DPY broth (20 g L<sup>−1</sup> dextrin, 10 g L<sup>−1</sup> polypeptone, 5 g L<sup>−1</sup> yeast extract, 2.31 g L<sup>−1</sup> KH<sub>2</sub>PO<sub>4</sub>, 0.5 g L<sup>−1</sup> MgSO<sub>4</sub>), and <em>Trichoderma</em> in PDB broth (200 g L<sup>−1</sup> potato, 20 g L<sup>−1</sup> glucose).</p></section><section id="Sec10"><h3 class="pmc_sec_title">Genome sequencing of <em>T. atroviride</em> B7</h3>
<p id="Par27">Genomic DNA was extracted from <em>T. atroviride</em> B7 mycelia using a Fungi Genomic DNA Purification Kit (Sangon Biotech) and sequenced via Illumina technology, yielding 17 contigs.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Comparative transcriptomic sequencing of <em>T. atroviride</em> B7</h3>
<p id="Par28"><em>T. atroviride</em> B7 was inoculated into 100 mL TB3 broth (200 g L<sup>−1</sup> sucrose, 6 g L<sup>−1</sup> yeast extract, 6 g L<sup>−1</sup> tryptone) or CD broth (200 g L<sup>−1</sup> sucrose, 3 g L<sup>−1</sup> NaNO<sub>3</sub>, 1 g L<sup>−1</sup> K<sub>2</sub>HPO<sub>4</sub>, 0.5 g L<sup>−1</sup> MgSO<sub>4</sub>, 0.5 g L<sup>−1</sup> KCl, 0.01 g L<sup>−1</sup> FeSO<sub>4</sub>) and cultured at 30 °C, 180 rpm for 4 days. Mycelia were harvested for RNA extraction (Fungal RNA Kit, Omega Biotek) and transcriptome sequencing (Illumina). Parallel mycelial samples were extracted with EtOAc and then subjected to GC–MS analysis.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Enzymatic activity-guided protein fractionation and proteomics analyses</h3>
<p id="Par29">Freshly grown hyphae of <em>T. atroviride</em> B7 were inoculated into 100 mL PDB medium and cultured at 30 °C with shaking at 180 rpm for 48 h. The culture was then scaled up to 3 L PDB medium and incubated under the same conditions for an additional 6 days. Mycelia were harvested using a cell strainer and washed with distilled water. A total of 53.8 g of mycelia were obtained, flash-frozen in liquid nitrogen, and ground to a fine powder. The powdered mycelia were suspended in protein extraction buffer (25 mM HEPES, 100 mM KCl, 5 mM DTT, 10% glycerol, pH 7.5) and stirred gently on ice for 30 min using a glass rod. After centrifugation at 13,523 g (4 °C for 10 min), the supernatant was subjected to ammonium sulfate precipitation (ASP) by sequentially adding (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> to achieve 20%, 40%, 60%, and 80% saturation, with each step incubated on ice until complete precipitation occurred. Each fraction was centrifuged at 13,523<em>g</em> for 10 min at 4 °C. The resulting supernatants were transferred for subsequent precipitation steps, while the pellets were resuspended in protein extraction buffer for enzyme assays. The ASP procedure was repeated twice with optimized gradients to remove inactive proteins. Active fractions were further purified by ion-exchange chromatography (HiTrap Capto Q, 1 mL, Cytiva Sweden AB) using a linear gradient elution of 0-100% buffer B (20 mM Tris-HCl, 1 M NaCl, pH 7.5) in buffer A (20 mM Tris-HCl, pH 7.5) at a 1 mL/min flow rate (15 mL per gradient). Three fractions showing high enzymatic activity were collected as positive samples, along with one neighboring, weakly active fraction used as a negative control.</p>
<p id="Par30">Enzyme assays were performed in 200 μL reaction volumes containing 100 μL reaction buffer (50 mM HEPES, 100 mM KCl, 5 mM DTT, 50 mM MgCl<sub>2</sub>, 10% glycerol, pH 7.5), 10 μg GGPP, and 100 μg protein. Reactions were incubated at 30 °C for 1 h, then extracted twice with 400 μL hexane. The combined organic phase was evaporated under nitrogen gas and the residue was resuspended in 50 μL hexane for GC–MS analysis.</p>
<p id="Par31">For proteomic analysis, protein samples were determined using a NanoDrop 2000C spectrophotometer (Thermo Scientific). In total, 40 μg of each sample was separated by SDS-PAGE, and the entire gel lanes were excised for processing. Gel slices were destained, alkylated with iodoacetamide, and digested with trypsin. The resulting peptides were extracted, desalted, and subject to LC–MS/MS analysis using an EASY-nLC 1000 system (Thermo Scientific) coupled to an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific) with Proteome Discoverer 2.4 software.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Heterologous expression in <em>E. coli</em>
</h3>
<p id="Par32">The <em>crtE</em> gene encoding GGPPS from <em>Erwinia</em> was cloned into pCDF-Duet1 to create pCDF-Duet1-<em>crtE</em>, which was co-transformed with pBbA5c-MevT-MBIS into <em>E. coli</em> BL21 to generate a strain producing FPP and GGPP precursors. Similarly, <em>TriTPS4</em> from <em>T. atroviride</em> B7 was cloned into pET-28a and co-transformed with pBbA5c-MevT-MBIS to create an alternative GGPP-producing strain. Candidate genes were PCR-amplified from <em>T. atroviride</em> B7 cDNA and cloned into pET-28a in XhoI/SalI restriction sites using a seamless cloning kit (BBI). <em>tri4155</em> and homologs were cloned into pCold-TF (SalI/KpnI sites), generating pCold-TF-<em>tri4155</em> and other pCold-TF-<em>triDTCs</em>. Site-directed mutagenesis was performed using pCold-TF-<em>tri4155</em> as template with overlapping primers (with 15 bp overlaps), followed by DpnI digestion and transformation into <em>E. coli</em> DH5α. Verified plasmids were isolated from 6 mL overnight cultures.</p>
<p id="Par33">Positive <em>E. coli</em> strains were grown in LB medium (6 mL) supplemented with appropriate antibiotics (100 mg L<sup>−1</sup> ampicillin or 50 mg L<sup>−1</sup> streptomycin, plus 50 mg L<sup>−1</sup> chloramphenicol and 50 mg L<sup>−1</sup> kanamycin) at 37 °C, 180 rpm overnight. 1 mL culture was inoculated into 100 mL TB broth with antibiotics and grown to OD<sub>600</sub> value of 0.6–0.8. Protein expression was induced with 0.3 mM IPTG, followed by incubation at 16 °C for 18 h and then at 30 °C for 72 h. Cultures were extracted with 150 mL ethyl acetate assisted by ultrasonication for 20 min. Organic phases were concentrated by rotary evaporation, dried under nitrogen gas, and resuspended in 800 μL hexane for GC–MS analysis. Every experiment has three biological replicates.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Heterologous expression in <em>A. oryzae</em> NSAR1</h3>
<p id="Par34">The <em>tri4155</em> cDNA was cloned into pTAex3 (NdeI/SmaI sites) under the P<em>amyB</em> promoter. For C-terminal His-tagging fusion, <em>tri4155</em> or homologous gene was amplified with His-tag-encoding primers and fused with 700 bp upstream (P<em>tri4155</em>) and 500 bp downstream (T<em>tri4155</em>) sequences by overlap extension PCR, which was then ligated into pTAex3 to create pTAex3-P<em>tri4155</em>-<em>triDTC</em>-<em>His</em>-T<em>tri4155</em>.</p>
<p id="Par35">Protoplast preparation was performed following established methods<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> with modifications. One-day-old <em>A. oryzae</em> NSAR1 mycelia were digested in 10 mL Yatalase solution (0.1 g Yatalase, 0.79 g (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> in 50 mM maleic acid, pH 5.5) at 30 °C, 100 rpm for 1–3 h. Protoplasts were filtered, centrifuged (1258<em>g</em>, 10 min), washed, and resuspended in a stabilizing buffer (1.2 M sorbitol, 50 mM CaCl<sub>2</sub>, 10 mM Tris-HCl, pH 7.5) to a final density of 1–5 × 10<sup>7</sup> mL<sup>−1</sup>. For transformation, 200 μL protoplasts were gently mixed with 10 μg of plasmid DNA and incubated on ice for 30 min. Subsequently, 1.35 mL PEG solution (150 mM PEG4000, 50 mM CaCl<sub>2</sub>, 10 mM Tris-HCl, pH 7.5) was added, and the mixture was incubated at room temperature for 20 min. After adding 5 mL stabilizing buffer and centrifugation, protoplasts were resuspended in 10 mL top agar (M medium with 0.15% methionine, 0.01% adenine, 0.8% agar) and plated on selection plates (1.0% agar). Transformants were verified by PCR and cultured in DPY broth for 7 days before extraction and GC–MS analysis. Every experiment contains three biological replicates.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Gene knockout in <em>Trichoderma</em> fungi</h3>
<p id="Par36">A split-marker strategy was applied with <em>HygR</em> cassette fragments splitting into 969 bp and 1,336 bp with 174 bp overlapping; the split fragments were flanked by 1 kb up- and down-stream target gene homology arms, respectively. Protoplasts were prepared following the protocol described for <em>A. oryzae</em>, and transformations were conducted as previously described<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> with modifications. Briefly, 6 μg each of upstream and downstream deletion fragments were mixed with 200 μL protoplasts and 50 μL PEG buffer (25% PEG6000, 10 mM Tris-HCl, 50 mM CaCl<sub>2</sub>, pH 7.5), followed by incubation on ice for 30 min. Subsequently, 2 mL PEG buffer was added, and the mixture was incubated at room temperature for 20 min. After the addition of 5 mL stabilizing buffer, the cells were centrifuged and plated in top agar (M medium containing 100 mg/L hygromycin B and 0.8% agar), then spread onto selection plates (1.0% agar). Positive transformants were verified by PCR using primers flanking the deletion region (Supplementary Data File <a href="#MOESM6" class="usa-link">4</a>) and then cultured in PDB for 4 days before metabolite analysis.</p></section><section id="Sec16"><h3 class="pmc_sec_title">In vitro enzyme assays</h3>
<p id="Par37">The <em>Tcie612</em> gene was cloned into pCold-II (SalI/KpnI) with a C-terminal His-tag. Site-specific mutagenesis was performed as described above. <em>E. coli</em> Rosetta strains transformed with the constructs were grown overnight in LB medium (6 mL) supplemented with 100 mg L<sup>−1</sup> ampicillin and 50 mg L<sup>−1</sup> chloramphenicol at 37 °C, 180 rpm. 1 mL culture was inoculated into 100 mL LB broth containing the same antibiotics. After growing to OD<sub>600</sub> 0.6–0.8, the cultures were induced with 0.3 mM IPTG at 16 °C for 18 h. Cells were harvested and lysed by sonication in 50 mM Tris-HCl (pH 7.5), and the cleared lysate was incubated with Ni-NTA resin in a binding buffer containing 20 mM imidazole at 4 °C for 1 h. After washing with buffer containing 20 mM and 40 mM imidazole, the target proteins were eluted with 300 mM imidazole and concentrated to 500 μL using 10 kDa centrifugal filters.</p>
<p id="Par38">Standard enzyme assays were performed in 200 μL volumes containing 50 mM Tris-HCl (pH 7.5), 25 mM MgCl<sub>2</sub>, 100 mM KCl, 5 mM DTT, 10% glycerol, and 5 mM hydroxypropyl-β-cyclodextrin. For kinetic measurements, 100 μL reaction mixtures were prepared with varying concentrations of GGPP (4, 8, 12, 16, 20, 30, 60, 80, and 200 μM). Reactions were incubated at 30 °C for 30 min and then stopped by flash-freezing in liquid nitrogen. The reaction mixtures were extracted with hexane and then analyzed by GC–MS. Each experiment contains three biological replicates. Kinetic parameters were determined using nonlinear regression based on the Michaelis-Menten equation (Origin 2021) from triplicate experiments.</p></section><section id="Sec17"><h3 class="pmc_sec_title">ITC experiments</h3>
<p id="Par39">Isothermal titration calorimetry (ITC) was performed using a MicroCal PEAQ-ITC instrument to determine the number of Mg<sup>2</sup><sup>+</sup> binding per enzyme. The Tcie612 enzyme was prepared at a concentration of 20 μM in Tris-HCl buffer (50 mM Tris-HCl, 50 mM NaCl, pH 8.0). Before titration, the enzyme was either pre-incubated with or without two equivalents of GGPP (40 μM). Subsequently, 280 μL of the protein solution was loaded into the sample cell, and 60 μL of MgCl<sub>2</sub> solution (1.2 mM) was loaded into the injector. The titration was carried out under the following conditions: stir speed 750 rpm, cell temperature 25 °C, first injection of 0.4 μL over 3.0 s, and then 2 μL over 3.0 s for the rest 18 injections, with an interval of 150 s between each injection. The data were analyzed using MicroCal PEAQ-ITC analysis software (Malvern Panalytical). The “Two Sets of Sites” binding model was applied for curve fitting.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Compound quantification</h3>
<p id="Par40">Compounds <strong>1</strong> and <strong>2</strong> were dissolved in hexane and diluted to concentrations of 0.0625, 0.125, 0.25, 0.50, 1.00, and 2.00 mg/mL using a double dilution method for GC‒MS analysis. The analytical parameters were the same as those described above. Standard curves correlating concentrations to peak areas were obtained using Origin 2021. The calibration equation of <strong>1</strong> (EIC: <em>m/z</em> 244) was: <em>y</em> = 5972018<em>x</em> (<em>R</em><sup>2</sup> = 0.985), while the equation of <strong>2</strong> (EIC: <em>m/z</em> 272) was: <em>y</em> = 17872250<em>x</em> (<em>R</em><sup>2</sup> = 0.973). The standard curves were used to quantify compounds <strong>1</strong> and <strong>2</strong> in the in vivo and in vitro experiments. The production bar charts were generated using GraphPad Prism 10.8.1. Each experiment was performed with at least three replicates.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Phylogeny analysis</h3>
<p id="Par41">Multiple sequence alignment was performed using MAFFT v7.520 (auto strategy). Neighbor-joining trees were constructed using MEGA 7.</p></section><section id="Sec20"><h3 class="pmc_sec_title">In silico experiments</h3>
<p id="Par42">Protein structures were predicted using AlphaFold2.3.2 (<a href="https://wemol.wecomput.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://wemol.wecomput.com/</a>). Binding pockets were analyzed using CB-dock2 (<a href="https://cadd.labshare.cn/cb-dock2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cadd.labshare.cn/cb-dock2</a>). The substrate (GGPP) and intermediates (c, e) were energy-minimized and docked using AutoDock-GPU (WeMol server).</p></section><section id="Sec21"><h3 class="pmc_sec_title">Biological function experiments</h3>
<p id="Par43">Test compounds were dissolved in DMSO and incorporated into PDA plates, while plates containing DMSO alone served as negative controls. Hyphae were center-inoculated and cultured at 25 °C under 12 h light/dark cycles. Plates were photographed, and hyphae were examined microscopically (Evos XL Core). For liquid cultures, hyphae were inoculated into PDB and grown at 30 °C, 180 rpm for 3 days. All experiments were performed in triplicate.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par44">Further information on research design is available in the <a href="#MOESM9" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (5.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63055_MOESM2_ESM.pdf</a><sup> (95.5KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 1</a><sup> (12.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 2</a><sup> (16.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 3</a><sup> (15.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 4</a><sup> (11.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 5</a><sup> (11.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 6</a><sup> (25KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (76.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM10_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (663.7KB, pdf) </sup>
</div></div></section></section><section id="Sec24"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM11_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (183.2KB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We gratefully acknowledge Prof. Tao Liu (Tianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences) for providing plasmid pBbA5c-MevT-MBIS. We sincerely thank Prof. Katsuya Gomi (Tohoku University) and Profs. Katsuhiko Kitamoto and Jun-ichi Maruyama (The University of Tokyo) for generously providing the <em>A. oryzae</em> NSAR1 strain and expression vectors. We appreciate Prof. Ze-Fen Yu (Yunnan University) for her expert guidance on the physiological architecture of fungi. We appreciate Prof. Shi-Hong Luo (Shenyang Agricultural University) for providing the <em>Fusarium graminearum</em> strain. This research was financially supported by National Natural Science Foundation of China (Nos. 82222072, 22107103 and U23A20510), Strategic Priority Research Program of the Chinese Academy of Sciences (No. XDB1230000), the China Postdoctoral Science Foundation (No. 2021M703285), the Natural Science Foundation of Yunnan (No. 202401BC070016), and the Yunnan Revitalization Talent Support Program Young Talent Project (XDYC―QNRC―2024―560).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Sheng-Hong Li and Yan Liu conceived and supervised the project. Jonathan Gershenzon provided valuable scientific guidance. Xuemei Niu identified the <em>Trichoderma</em> strains. Min-Jie Yang designed and performed the majority of experiments. De-Sen Li constructed the <em>E. coli</em> strains. Hua-Qin Deng conducted fermentation experiments. Wen-Yuan Li sequenced the <em>T. atroviride</em> B7 genome. Xin-Yu Zheng performed gene cloning. All authors participated in writing and revising the paper.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par45"><em>Nature Communications</em> thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data are available in the main text, Supplementary Information, Supplementary Data files, Source data, and public repositories. Protein sequences for constructing the phylogeny tree were retrieved from the UniProt database or NCBI GenBank; the accession numbers were listed in Supplementary Data File <a href="#MOESM4" class="usa-link">2</a>. Plasmids and strains were listed in Supplementary Data File <a href="#MOESM7" class="usa-link">5</a>, which are available on request. Sequences of this study were listed in Supplementary Data File <a href="#MOESM8" class="usa-link">6</a>. The Whole Genome Shotgun project of <em>T. atroviride</em> B7 has been deposited at GenBank under accession code <a href="https://www.ncbi.nlm.nih.gov/nuccore/JBPAFJ000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JBPAFJ000000000</a>. Raw comparative transcriptome sequencing data have been deposited at NCBI SRA under accession code PRJNA1235459. The <em>tri4155</em> gene cluster sequence has been deposited at NCBI SRA under accession code PRJNA1234899. Proteomics data have been deposited at iProX under accession code IPX0011342000. <a href="#Sec24" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par46">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Yan Liu, Email: liuyan@cdutcm.edu.cn.</p>
<p>Sheng-Hong Li, Email: shli@mail.kib.ac.cn.</p></section><section id="sec25"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63055-4.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Christianson, D. W. Structural and chemical biology of terpenoid cyclases. <em>Chem. Rev.</em><strong>117</strong>, 11570–11648 (2017).
</cite> [<a href="https://doi.org/10.1021/acs.chemrev.7b00287" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5599884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28841019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Christianson,%20D.%20W.%20Structural%20and%20chemical%20biology%20of%20terpenoid%20cyclases.%20Chem.%20Rev.117,%2011570%E2%80%9311648%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Ducker, C. et al. A diterpene synthase from the sandfly <em>Lutzomyia longipalpis</em> produces the pheromone sobralene. <em>Proc. Natl Acad. Sci. USA</em><strong>121</strong>, e2322453121 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2322453121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10962984/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38470919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ducker,%20C.%20et%20al.%20A%20diterpene%20synthase%20from%20the%20sandfly%20Lutzomyia%20longipalpis%20produces%20the%20pheromone%20sobralene.%20Proc.%20Natl%20Acad.%20Sci.%20USA121,%20e2322453121%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Palframan, M. J., Bandi, K. K., Hamilton, J. G. C. &amp; Pattenden, G. Sobralene, a new sex-aggregation pheromone and likely shunt metabolite of the taxadiene synthase cascade, produced by a member of the sand fly <em>Lutzomyia longipalpis</em> species complex. <em>Tetrahedron Lett.</em><strong>59</strong>, 1921–1923 (2018).
</cite> [<a href="https://doi.org/10.1016/j.tetlet.2018.03.088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5937913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29780183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palframan,%20M.%20J.,%20Bandi,%20K.%20K.,%20Hamilton,%20J.%20G.%20C.%20&amp;%20Pattenden,%20G.%20Sobralene,%20a%20new%20sex-aggregation%20pheromone%20and%20likely%20shunt%20metabolite%20of%20the%20taxadiene%20synthase%20cascade,%20produced%20by%20a%20member%20of%20the%20sand%20fly%20Lutzomyia%20longipalpis%20species%20complex.%20Tetrahedron%20Lett.59,%201921%E2%80%931923%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Avalos, M. et al. Biosynthesis, evolution and ecology of microbial terpenoids. <em>Nat. Prod. Rep.</em><strong>39</strong>, 249–272 (2022).
</cite> [<a href="https://doi.org/10.1039/d1np00047k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34612321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Avalos,%20M.%20et%20al.%20Biosynthesis,%20evolution%20and%20ecology%20of%20microbial%20terpenoids.%20Nat.%20Prod.%20Rep.39,%20249%E2%80%93272%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Luo, S. H. et al. Glandular trichomes of <em>Leucosceptrum canum</em> harbor defensive sesterterpenoids. <em>Angew. Chem. Int. Ed.</em><strong>49</strong>, 4471–4475 (2010).</cite> [<a href="https://doi.org/10.1002/anie.201000449" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20458721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luo,%20S.%20H.%20et%20al.%20Glandular%20trichomes%20of%20Leucosceptrum%20canum%20harbor%20defensive%20sesterterpenoids.%20Angew.%20Chem.%20Int.%20Ed.49,%204471%E2%80%934475%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Sato, T. et al. Sesquarterpenes (C<sub>35</sub> Terpenes) biosynthesized via the cyclization of a linear C<sub>35</sub> isoprenoid by a tetraprenyl-β-curcumene synthase and a tetraprenyl-β-curcumene cyclase: identification of a new terpene cyclase. <em>J. Am. Chem. Soc.</em><strong>133</strong>, 9734–9737 (2011).
</cite> [<a href="https://doi.org/10.1021/ja203779h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21627333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sato,%20T.%20et%20al.%20Sesquarterpenes%20(C35%20Terpenes)%20biosynthesized%20via%20the%20cyclization%20of%20a%20linear%20C35%20isoprenoid%20by%20a%20tetraprenyl-%CE%B2-curcumene%20synthase%20and%20a%20tetraprenyl-%CE%B2-curcumene%20cyclase:%20identification%20of%20a%20new%20terpene%20cyclase.%20J.%20Am.%20Chem.%20Soc.133,%209734%E2%80%939737%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Schor, R. et al. Three previously unrecognised classes of biosynthetic enzymes revealed during the production of xenovulene A. <em>Nat. Commun.</em><strong>9</strong>, 1963 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-018-04364-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5958101/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29773797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schor,%20R.%20et%20al.%20Three%20previously%20unrecognised%20classes%20of%20biosynthetic%20enzymes%20revealed%20during%20the%20production%20of%20xenovulene%20A.%20Nat.%20Commun.9,%201963%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Rudolf, J. D. &amp; Chang, C. Y. Terpene synthases in disguise: enzymology, structure, and opportunities of non-canonical terpene synthases. <em>Nat. Prod. Rep.</em><strong>37</strong>, 425–463 (2020).
</cite> [<a href="https://doi.org/10.1039/c9np00051h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7101268/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31650156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rudolf,%20J.%20D.%20&amp;%20Chang,%20C.%20Y.%20Terpene%20synthases%20in%20disguise:%20enzymology,%20structure,%20and%20opportunities%20of%20non-canonical%20terpene%20synthases.%20Nat.%20Prod.%20Rep.37,%20425%E2%80%93463%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Derengowski, L. S. et al. Antimicrobial effect of farnesol, a <em>Candida albicans</em> quorum sensing molecule, on <em>Paracoccidioides brasiliensis</em> growth and morphogenesis. <em>Ann. Clin. Microbiol. Antimicrob.</em><strong>8</strong>, 13 (2009).
</cite> [<a href="https://doi.org/10.1186/1476-0711-8-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2681445/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19402910/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Derengowski,%20L.%20S.%20et%20al.%20Antimicrobial%20effect%20of%20farnesol,%20a%20Candida%20albicans%20quorum%20sensing%20molecule,%20on%20Paracoccidioides%20brasiliensis%20growth%20and%20morphogenesis.%20Ann.%20Clin.%20Microbiol.%20Antimicrob.8,%2013%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>do Nascimento, J. S. et al. Natural trypanocidal product produced by endophytic fungi through co-culturing. <em>Folia Microbiol.</em><strong>65</strong>, 323–328 (2020).
</cite> [<a href="https://doi.org/10.1007/s12223-019-00727-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31250361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?do%20Nascimento,%20J.%20S.%20et%20al.%20Natural%20trypanocidal%20product%20produced%20by%20endophytic%20fungi%20through%20co-culturing.%20Folia%20Microbiol.65,%20323%E2%80%93328%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Arai, M., Niikawa, H. &amp; Kobayashi, M. Marine-derived fungal sesterterpenes, ophiobolins, inhibit biofilm formation of Mycobacterium species. <em>J. Nat. Med.</em><strong>67</strong>, 271–275 (2013).
</cite> [<a href="https://doi.org/10.1007/s11418-012-0676-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22684914/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arai,%20M.,%20Niikawa,%20H.%20&amp;%20Kobayashi,%20M.%20Marine-derived%20fungal%20sesterterpenes,%20ophiobolins,%20inhibit%20biofilm%20formation%20of%20Mycobacterium%20species.%20J.%20Nat.%20Med.67,%20271%E2%80%93275%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Duke, S. O. &amp; Dayan, F. E. Modes of action of microbially-produced phytotoxins. <em>Toxins</em><strong>3</strong>, 1038–1064 (2011).
</cite> [<a href="https://doi.org/10.3390/toxins3081038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3202864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22069756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duke,%20S.%20O.%20&amp;%20Dayan,%20F.%20E.%20Modes%20of%20action%20of%20microbially-produced%20phytotoxins.%20Toxins3,%201038%E2%80%931064%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Chaverri, P., Castlebury, L. A., Overton, B. E. &amp; Samuels, G. J. <em>Hypocrea/Trichoderma</em>: species with conidiophore elongations and green conidia. <em>Mycologia</em><strong>95</strong>, 1100–1140 (2003).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/21149016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chaverri,%20P.,%20Castlebury,%20L.%20A.,%20Overton,%20B.%20E.%20&amp;%20Samuels,%20G.%20J.%20Hypocrea/Trichoderma:%20species%20with%20conidiophore%20elongations%20and%20green%20conidia.%20Mycologia95,%201100%E2%80%931140%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Woo, S. L., Hermosa, R., Lorito, M. &amp; Monte, E. <em>Trichoderma</em>: a multipurpose, plant-beneficial microorganism for eco-sustainable agriculture. <em>Nat. Rev. Microbiol.</em><strong>21</strong>, 312–326 (2023).
</cite> [<a href="https://doi.org/10.1038/s41579-022-00819-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36414835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Woo,%20S.%20L.,%20Hermosa,%20R.,%20Lorito,%20M.%20&amp;%20Monte,%20E.%20Trichoderma:%20a%20multipurpose,%20plant-beneficial%20microorganism%20for%20eco-sustainable%20agriculture.%20Nat.%20Rev.%20Microbiol.21,%20312%E2%80%93326%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Lorito, M., Woo, S. L., Harman, G. E. &amp; Monte, E. in <em>Annual Review of Phytopathology</em> Vol. 48 (eds N. K. VanAlfen, G. Bruening &amp; J. E. Leach) pp 395–417 (2010).</cite> [<a href="https://doi.org/10.1146/annurev-phyto-073009-114314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20455700/" class="usa-link">PubMed</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Yuan, M. et al. Whole RNA-sequencing and gene expression analysis of <em>Trichoderma harzianum</em> Tr-92 under chlamydospore-producing condition. <em>Genes Genomics</em><strong>41</strong>, 689–699 (2019).
</cite> [<a href="https://doi.org/10.1007/s13258-019-00812-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30968334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan,%20M.%20et%20al.%20Whole%20RNA-sequencing%20and%20gene%20expression%20analysis%20of%20Trichoderma%20harzianum%20Tr-92%20under%20chlamydospore-producing%20condition.%20Genes%20Genomics41,%20689%E2%80%93699%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Zhu, X. C., Wang, Y. P., Wang, X. B. &amp; Wang, W. Exogenous regulators enhance the yield and stress resistance of chlamydospores of the biocontrol agent <em>Trichoderma harzianum</em> T4. <em>J. Fungi</em><strong>8</strong>, 1017 (2022).</cite> [<a href="https://doi.org/10.3390/jof8101017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9604748/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36294583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20X.%20C.,%20Wang,%20Y.%20P.,%20Wang,%20X.%20B.%20&amp;%20Wang,%20W.%20Exogenous%20regulators%20enhance%20the%20yield%20and%20stress%20resistance%20of%20chlamydospores%20of%20the%20biocontrol%20agent%20Trichoderma%20harzianum%20T4.%20J.%20Fungi8,%201017%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Bai, B. et al. <em>Trichoderma</em> species from plant and soil: an excellent resource for biosynthesis of terpenoids with versatile bioactivities. <em>J. Adv. Res.</em><strong>49</strong>, 81–102 (2023).
</cite> [<a href="https://doi.org/10.1016/j.jare.2022.09.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10334134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36195283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bai,%20B.%20et%20al.%20Trichoderma%20species%20from%20plant%20and%20soil:%20an%20excellent%20resource%20for%20biosynthesis%20of%20terpenoids%20with%20versatile%20bioactivities.%20J.%20Adv.%20Res.49,%2081%E2%80%93102%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Yamamoto, T. et al. Wickerols A and B: novel anti-influenza virus diterpenes produced by Trichoderma atroviride FKI-3849. <em>Tetrahedron</em><strong>68</strong>, 9267–9271 (2012).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yamamoto,%20T.%20et%20al.%20Wickerols%20A%20and%20B:%20novel%20anti-influenza%20virus%20diterpenes%20produced%20by%20Trichoderma%20atroviride%20FKI-3849.%20Tetrahedron68,%209267%E2%80%939271%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Barra, L. &amp; Dickschat, J. S. Harzianone biosynthesis by the biocontrol fungus <em>Trichoderma</em>. <em>Chembiochem</em><strong>18</strong>, 2358–2365 (2017).
</cite> [<a href="https://doi.org/10.1002/cbic.201700462" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28944564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barra,%20L.%20&amp;%20Dickschat,%20J.%20S.%20Harzianone%20biosynthesis%20by%20the%20biocontrol%20fungus%20Trichoderma.%20Chembiochem18,%202358%E2%80%932365%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Ghisalberti, E. L., Hockless, D. C. R., Rowland, C. &amp; White, A. H. Harziandione, a new class of diterpene from <em>Trichoderma harzianum</em>. <em>J. Nat. Prod.</em><strong>55</strong>, 1690–1694 (1992).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ghisalberti,%20E.%20L.,%20Hockless,%20D.%20C.%20R.,%20Rowland,%20C.%20&amp;%20White,%20A.%20H.%20Harziandione,%20a%20new%20class%20of%20diterpene%20from%20Trichoderma%20harzianum.%20J.%20Nat.%20Prod.55,%201690%E2%80%931694%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Cardoza, R. E. et al. Identification of loci and functional characterization of trichothecene biosynthesis genes in filamentous fungi of the genus <em>Trichoderma</em>. <em>Appl. Environ. Microbiol.</em><strong>77</strong>, 4867–4877 (2011).
</cite> [<a href="https://doi.org/10.1128/AEM.00595-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3147405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21642405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cardoza,%20R.%20E.%20et%20al.%20Identification%20of%20loci%20and%20functional%20characterization%20of%20trichothecene%20biosynthesis%20genes%20in%20filamentous%20fungi%20of%20the%20genus%20Trichoderma.%20Appl.%20Environ.%20Microbiol.77,%204867%E2%80%934877%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Murai, K. et al. An unusual skeletal rearrangement in the biosynthesis of the sesquiterpene trichobrasilenol from <em>Trichoderma</em>. <em>Angew. Chem. Int. Ed.</em><strong>58</strong>, 15046–15050 (2019).</cite> [<a href="https://doi.org/10.1002/anie.201907964" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7687074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31418991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murai,%20K.%20et%20al.%20An%20unusual%20skeletal%20rearrangement%20in%20the%20biosynthesis%20of%20the%20sesquiterpene%20trichobrasilenol%20from%20Trichoderma.%20Angew.%20Chem.%20Int.%20Ed.58,%2015046%E2%80%9315050%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Yan, Y. et al. Biosynthesis of the fungal glyceraldehyde-3-phosphate dehydrogenase inhibitor heptelidic acid and mechanism of self-resistance. <em>Chem. Sci.</em><strong>11</strong>, 9554–9562 (2020).
</cite> [<a href="https://doi.org/10.1039/d0sc03805a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8162069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34094220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan,%20Y.%20et%20al.%20Biosynthesis%20of%20the%20fungal%20glyceraldehyde-3-phosphate%20dehydrogenase%20inhibitor%20heptelidic%20acid%20and%20mechanism%20of%20self-resistance.%20Chem.%20Sci.11,%209554%E2%80%939562%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Li, W. Y. et al. Antibacterial harziane diterpenoids from a fungal symbiont <em>Trichoderma atroviride</em> isolated from <em>Colquhounia coccinea</em> var. mollis. <em>Phytochemistry</em><strong>170</strong>, 112198 (2020).
</cite> [<a href="https://doi.org/10.1016/j.phytochem.2019.112198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31765875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20W.%20Y.%20et%20al.%20Antibacterial%20harziane%20diterpenoids%20from%20a%20fungal%20symbiont%20Trichoderma%20atroviride%20isolated%20from%20Colquhounia%20coccinea%20var.%20mollis.%20Phytochemistry170,%20112198%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Misawa, N. et al. Canthaxanthin biosynthesis by the conversion of methylene to keto groups in a single-gene. <em>Biochem. Biophys. Res. Commun</em>. <strong>209</strong>, 867–876 (1995).</cite> [<a href="https://doi.org/10.1006/bbrc.1995.1579" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7733978/" class="usa-link">PubMed</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Miao, F. P. et al. Absolute configurations of unique harziane diterpenes from <em>Trichoderma</em> Species. <em>Org. Lett.</em><strong>14</strong>, 3815–3817 (2012).
</cite> [<a href="https://doi.org/10.1021/ol3014717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22823323/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miao,%20F.%20P.%20et%20al.%20Absolute%20configurations%20of%20unique%20harziane%20diterpenes%20from%20Trichoderma%20Species.%20Org.%20Lett.14,%203815%E2%80%933817%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Landmann, C. et al. Cloning and functional characterization of three terpene synthases from lavender (<em>Lavandula angustifolia</em>). <em>Arch. Biochem. Biophys.</em><strong>465</strong>, 417–429 (2007).
</cite> [<a href="https://doi.org/10.1016/j.abb.2007.06.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17662687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Landmann,%20C.%20et%20al.%20Cloning%20and%20functional%20characterization%20of%20three%20terpene%20synthases%20from%20lavender%20(Lavandula%20angustifolia).%20Arch.%20Biochem.%20Biophys.465,%20417%E2%80%93429%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Frick, S. et al. Metal ions control product specificity of isoprenyl diphosphate synthases in the insect terpenoid pathway. <em>Proc. Natl Acad. Sci. USA</em><strong>110</strong>, 4194–4199 (2013).
</cite> [<a href="https://doi.org/10.1073/pnas.1221489110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3600467/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23440195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Frick,%20S.%20et%20al.%20Metal%20ions%20control%20product%20specificity%20of%20isoprenyl%20diphosphate%20synthases%20in%20the%20insect%20terpenoid%20pathway.%20Proc.%20Natl%20Acad.%20Sci.%20USA110,%204194%E2%80%934199%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Paysan-Lafosse, T. et al. InterPro in 2022. <em>Nucleic Acids Res.</em><strong>51</strong>, D418–D427 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkac993" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36350672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paysan-Lafosse,%20T.%20et%20al.%20InterPro%20in%202022.%20Nucleic%20Acids%20Res.51,%20D418%E2%80%93D427%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Zhang, L. et al. Biosynthesis of phomactin platelet activating factor antagonist requires a two-enzyme cascade. <em>Angew. Chem. Int. Ed</em>. <strong>62</strong>, e202312996 (2023).</cite> [<a href="https://doi.org/10.1002/anie.202312996" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37804495/" class="usa-link">PubMed</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Koeksal, M. et al. Taxadiene synthase structure and evolution of modular architecture in terpene biosynthesis. <em>Nature</em><strong>469</strong>, 116–120 (2011).
</cite> [<a href="https://doi.org/10.1038/nature09628" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3059769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21160477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koeksal,%20M.%20et%20al.%20Taxadiene%20synthase%20structure%20and%20evolution%20of%20modular%20architecture%20in%20terpene%20biosynthesis.%20Nature469,%20116%E2%80%93120%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Schrepfer, P. et al. Identification of amino acid networks governing catalysis in the closed complex of class I terpene synthases. <em>Proc. Natl Acad. Sci. USA</em><strong>113</strong>, E958–E967 (2016).
</cite> [<a href="https://doi.org/10.1073/pnas.1519680113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4776449/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26842837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schrepfer,%20P.%20et%20al.%20Identification%20of%20amino%20acid%20networks%20governing%20catalysis%20in%20the%20closed%20complex%20of%20class%20I%20terpene%20synthases.%20Proc.%20Natl%20Acad.%20Sci.%20USA113,%20E958%E2%80%93E967%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Monte, E. The sophisticated evolution of Trichoderma to control insect pests. <em>Proc. Natl Acad. Sci. USA</em><strong>120</strong>, e2301971120 (2023).
</cite> [<a href="https://doi.org/10.1073/pnas.2301971120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10041122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36913591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Monte,%20E.%20The%20sophisticated%20evolution%20of%20Trichoderma%20to%20control%20insect%20pests.%20Proc.%20Natl%20Acad.%20Sci.%20USA120,%20e2301971120%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Geiser, D. M., Timberlake, W. E. &amp; Arnold, M. L. Loss of meiosis in <em>Aspergillus</em>. <em>Mol. Biol. Evol.</em><strong>13</strong>, 809–817 (1996).
</cite> [<a href="https://doi.org/10.1093/oxfordjournals.molbev.a025641" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8754217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Geiser,%20D.%20M.,%20Timberlake,%20W.%20E.%20&amp;%20Arnold,%20M.%20L.%20Loss%20of%20meiosis%20in%20Aspergillus.%20Mol.%20Biol.%20Evol.13,%20809%E2%80%93817%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Willetts, H. J. &amp; Bullock, S. Developmental biology of sclerotia. <em>Mycol. Res.</em><strong>96</strong>, 801–816 (1992).</cite> [<a href="https://scholar.google.com/scholar_lookup?Willetts,%20H.%20J.%20&amp;%20Bullock,%20S.%20Developmental%20biology%20of%20sclerotia.%20Mycol.%20Res.96,%20801%E2%80%93816%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Calvo, A. M., Gardner, H. W. &amp; Keller, N. P. Genetic connection between fatty acid metabolism and sporulation in <em>Aspergillus nidulans</em>. <em>J. Biol. Chem.</em><strong>276</strong>, 25766–25774 (2001).
</cite> [<a href="https://doi.org/10.1074/jbc.M100732200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11352908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Calvo,%20A.%20M.,%20Gardner,%20H.%20W.%20&amp;%20Keller,%20N.%20P.%20Genetic%20connection%20between%20fatty%20acid%20metabolism%20and%20sporulation%20in%20Aspergillus%20nidulans.%20J.%20Biol.%20Chem.276,%2025766%E2%80%9325774%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Wolf, J. C. &amp; Mirocha, C. J. Regulation of sexual reproduction in gibberella-zeae (<em>Fusariumroseum graminearum</em>) by F-2 (zearalenone). <em>Can. J. Microbiol.</em><strong>19</strong>, 725–734 (1973).
</cite> [<a href="https://doi.org/10.1139/m73-117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4712507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolf,%20J.%20C.%20&amp;%20Mirocha,%20C.%20J.%20Regulation%20of%20sexual%20reproduction%20in%20gibberella-zeae%20(Fusariumroseum%20graminearum)%20by%20F-2%20(zearalenone).%20Can.%20J.%20Microbiol.19,%20725%E2%80%93734%20(1973)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Schimmel, T. G., Coffman, A. D. &amp; Parsons, S. J. Effect of butyrolactone I on the producing fungus, <em>Aspergillus terreus</em>. <em>Appl. Environ. Microbiol.</em><strong>64</strong>, 3707–3712 (1998).
</cite> [<a href="https://doi.org/10.1128/aem.64.10.3707-3712.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC106526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9758788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schimmel,%20T.%20G.,%20Coffman,%20A.%20D.%20&amp;%20Parsons,%20S.%20J.%20Effect%20of%20butyrolactone%20I%20on%20the%20producing%20fungus,%20Aspergillus%20terreus.%20Appl.%20Environ.%20Microbiol.64,%203707%E2%80%933712%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Roncal, T. et al. Novel diterpenes with potent conidiation inducing activity. <em>Tetrahedron Lett.</em><strong>43</strong>, 6799–6802 (2002).</cite> [<a href="https://scholar.google.com/scholar_lookup?Roncal,%20T.%20et%20al.%20Novel%20diterpenes%20with%20potent%20conidiation%20inducing%20activity.%20Tetrahedron%20Lett.43,%206799%E2%80%936802%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Liu, K. et al. Regulation of cellulase expression, sporulation, and morphogenesis by velvet family proteins in <em>Trichoderma reesei</em>. <em>Appl. Microbiol. Biotechnol.</em><strong>100</strong>, 769–779 (2016).
</cite> [<a href="https://doi.org/10.1007/s00253-015-7059-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26481618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20K.%20et%20al.%20Regulation%20of%20cellulase%20expression,%20sporulation,%20and%20morphogenesis%20by%20velvet%20family%20proteins%20in%20Trichoderma%20reesei.%20Appl.%20Microbiol.%20Biotechnol.100,%20769%E2%80%93779%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Kim, D. M., Sakamoto, I. &amp; Arioka, M. Class VI G protein‑coupled receptors in <em>Aspergillus oryzae</em> regulate sclerotia formation through GTPase‑activating activity. <em>Appl. Microbiol. Biotechnol.</em><strong>108</strong>, 141 (2024).
</cite> [<a href="https://doi.org/10.1007/s00253-023-12862-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10794492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38231240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20D.%20M.,%20Sakamoto,%20I.%20&amp;%20Arioka,%20M.%20Class%20VI%20G%20protein%E2%80%91coupled%20receptors%20in%20Aspergillus%20oryzae%20regulate%20sclerotia%20formation%20through%20GTPase%E2%80%91activating%20activity.%20Appl.%20Microbiol.%20Biotechnol.108,%20141%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Brown, N. A., Schrevens, S., Dijck, P. &amp; Goldman, G. H. Fungal G-protein-coupled receptors: mediators of pathogenesis and targets for disease control. <em>Nat. Microbiol.</em><strong>3</strong>, 402–414 (2018).
</cite> [<a href="https://doi.org/10.1038/s41564-018-0127-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29588541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown,%20N.%20A.,%20Schrevens,%20S.,%20Dijck,%20P.%20&amp;%20Goldman,%20G.%20H.%20Fungal%20G-protein-coupled%20receptors:%20mediators%20of%20pathogenesis%20and%20targets%20for%20disease%20control.%20Nat.%20Microbiol.3,%20402%E2%80%93414%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Punt, P. J. &amp; Vandenhondel, C. Transformation of filamentous fungi based on hygromycin-B and phleomycin resistance markers. <em>Methods Enzymol.</em><strong>216</strong>, 447–457 (1992).
</cite> [<a href="https://doi.org/10.1016/0076-6879(92)16041-h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1479914/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Punt,%20P.%20J.%20&amp;%20Vandenhondel,%20C.%20Transformation%20of%20filamentous%20fungi%20based%20on%20hygromycin-B%20and%20phleomycin%20resistance%20markers.%20Methods%20Enzymol.216,%20447%E2%80%93457%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Gruber, F., Visser, J., Kubicek, C. P. &amp; Degraaff, L. H. The development of a heterologous transformation system for the cellulolytic fungus <em>Trichoderma reesei</em> based on a <em>pyrg</em>-negative mutant strain. <em>Curr. Genet.</em><strong>18</strong>, 71–76 (1990).
</cite> [<a href="https://doi.org/10.1007/BF00321118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2245476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gruber,%20F.,%20Visser,%20J.,%20Kubicek,%20C.%20P.%20&amp;%20Degraaff,%20L.%20H.%20The%20development%20of%20a%20heterologous%20transformation%20system%20for%20the%20cellulolytic%20fungus%20Trichoderma%20reesei%20based%20on%20a%20pyrg-negative%20mutant%20strain.%20Curr.%20Genet.18,%2071%E2%80%9376%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (5.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63055_MOESM2_ESM.pdf</a><sup> (95.5KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 1</a><sup> (12.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 2</a><sup> (16.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 3</a><sup> (15.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 4</a><sup> (11.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 5</a><sup> (11.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 6</a><sup> (25KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (76.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM10_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (663.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370890/bin/41467_2025_63055_MOESM11_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (183.2KB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data are available in the main text, Supplementary Information, Supplementary Data files, Source data, and public repositories. Protein sequences for constructing the phylogeny tree were retrieved from the UniProt database or NCBI GenBank; the accession numbers were listed in Supplementary Data File <a href="#MOESM4" class="usa-link">2</a>. Plasmids and strains were listed in Supplementary Data File <a href="#MOESM7" class="usa-link">5</a>, which are available on request. Sequences of this study were listed in Supplementary Data File <a href="#MOESM8" class="usa-link">6</a>. The Whole Genome Shotgun project of <em>T. atroviride</em> B7 has been deposited at GenBank under accession code <a href="https://www.ncbi.nlm.nih.gov/nuccore/JBPAFJ000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JBPAFJ000000000</a>. Raw comparative transcriptome sequencing data have been deposited at NCBI SRA under accession code PRJNA1235459. The <em>tri4155</em> gene cluster sequence has been deposited at NCBI SRA under accession code PRJNA1234899. Proteomics data have been deposited at iProX under accession code IPX0011342000. <a href="#Sec24" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63055-4"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63055.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370890/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370890/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370890%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370890/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370890/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370890/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841380/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370890/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841380/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370890/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370890/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="0qjzms7uifmmJYzx5FvAXzNveDg3ZDlEfppimQEfIvHDZQGUGvsbXTD5D47ehBVq">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
