
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Patient-derived tumoroids from CIC::DUX4 rearranged sarcoma identify MCL1 as a therapeutic target - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532BBD8AF1D5D3052BBD00655E46E9.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370961/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Patient-derived tumoroids from CIC::DUX4 rearranged sarcoma identify MCL1 as a therapeutic target">
<meta name="citation_author" content="Willemijn Breunis">
<meta name="citation_author_institution" content="Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Eva Brack">
<meta name="citation_author_institution" content="Division of Pediatric Hematology/Oncology, Department of Pediatrics, Inselspital, Bern University Hospital, Bern, Switzerland">
<meta name="citation_author" content="Anna C Ehlers">
<meta name="citation_author_institution" content="Hopp-Children’s Cancer Center (KiTZ), Heidelberg, Germany">
<meta name="citation_author_institution" content="Division of Translational Pediatric Sarcoma Research, German Cancer Research Center (DKFZ), German Cancer Consortium (DKTK), Heidelberg, Germany">
<meta name="citation_author_institution" content="National Center for Tumor Diseases (NCT), NCT Heidelberg, a partnership between DKFZ and Heidelberg University Hospital, Heidelberg, Germany">
<meta name="citation_author_institution" content="Medical Faculty, Ruprecht-Karls-University, Heidelberg, Germany">
<meta name="citation_author" content="Ingrid Bechtold">
<meta name="citation_author_institution" content="Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Samanta Kisele">
<meta name="citation_author_institution" content="Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Jakob Wurth">
<meta name="citation_author_institution" content="Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Lieke Mous">
<meta name="citation_author_institution" content="Balgrist University Hospital, Faculty of Medicine, University of Zurich (UZH), Zurich, Switzerland">
<meta name="citation_author" content="Dorita Zabele">
<meta name="citation_author_institution" content="Balgrist University Hospital, Faculty of Medicine, University of Zurich (UZH), Zurich, Switzerland">
<meta name="citation_author" content="Fabio Steffen">
<meta name="citation_author_institution" content="Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Felina Zahnow">
<meta name="citation_author_institution" content="Hopp-Children’s Cancer Center (KiTZ), Heidelberg, Germany">
<meta name="citation_author" content="Christian Britschgi">
<meta name="citation_author_institution" content="Department of Medical Oncology and Hematology, University Hospital Zurich, Comprehensive Cancer Center Zurich, Zurich, Switzerland">
<meta name="citation_author_institution" content="Present Address: Medical Oncology and Hematology, Cantonal Hospital Winterthur, Winterthur, Switzerland">
<meta name="citation_author" content="Lorenz Bankel">
<meta name="citation_author_institution" content="Department of Medical Oncology and Hematology, University Hospital Zurich, Comprehensive Cancer Center Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Christian Rothermundt">
<meta name="citation_author_institution" content="Department of Medical Oncology and Hematology, Cantonal Hospital St. Gallen, St. Gallen, Switzerland">
<meta name="citation_author" content="Cornelia Vetter">
<meta name="citation_author_institution" content="Division of Pediatric Hematology/Oncology, Children’s Hospital of Eastern Switzerland, St. Gallen, Switzerland">
<meta name="citation_author" content="Daniel Müller">
<meta name="citation_author_institution" content="University Sarcoma Center Zürich (CCCZ), Balgrist University Hospital, University of Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Sander Botter">
<meta name="citation_author_institution" content="Swiss Center for Musculoskeletal Biobanking, Balgrist Campus AG, Zurich, Switzerland">
<meta name="citation_author" content="Chantal Pauli">
<meta name="citation_author_institution" content="Department of Pathology and Molecular Pathology, University Hospital Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Peter Bode">
<meta name="citation_author_institution" content="Department of Pathology and Molecular Pathology, University Hospital Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Beate Rinner">
<meta name="citation_author_institution" content="Division of Biomedical Research, Medical University of Graz, Graz, Austria">
<meta name="citation_author" content="Jean-Pierre Bourquin">
<meta name="citation_author_institution" content="Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Jochen Roessler">
<meta name="citation_author_institution" content="Division of Pediatric Hematology/Oncology, Department of Pediatrics, Inselspital, Bern University Hospital, Bern, Switzerland">
<meta name="citation_author" content="Thomas G P Grünewald">
<meta name="citation_author_institution" content="Hopp-Children’s Cancer Center (KiTZ), Heidelberg, Germany">
<meta name="citation_author_institution" content="Division of Translational Pediatric Sarcoma Research, German Cancer Research Center (DKFZ), German Cancer Consortium (DKTK), Heidelberg, Germany">
<meta name="citation_author_institution" content="National Center for Tumor Diseases (NCT), NCT Heidelberg, a partnership between DKFZ and Heidelberg University Hospital, Heidelberg, Germany">
<meta name="citation_author_institution" content="Institute of Pathology, Heidelberg University Hospital, Heidelberg, Germany">
<meta name="citation_author" content="Beat W Schäfer">
<meta name="citation_author_institution" content="Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland">
<meta name="citation_author" content="Didier Surdez">
<meta name="citation_author_institution" content="Balgrist University Hospital, Faculty of Medicine, University of Zurich (UZH), Zurich, Switzerland">
<meta name="citation_author" content="Marco Wachtel">
<meta name="citation_author_institution" content="Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7688">
<meta name="citation_doi" content="10.1038/s41467-025-62629-6">
<meta name="citation_pmid" content="40841513">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370961/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370961/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370961/pdf/41467_2025_Article_62629.pdf">
<meta name="description" content="High-risk sarcomas, such as metastatic and relapsed Ewing and CIC-rearranged sarcoma, still have a poor prognosis despite intensive therapeutic regimens. Precision medicine approaches offer hope, and ex vivo drug response profiling of ...">
<meta name="og:title" content="Patient-derived tumoroids from CIC::DUX4 rearranged sarcoma identify MCL1 as a therapeutic target">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="High-risk sarcomas, such as metastatic and relapsed Ewing and CIC-rearranged sarcoma, still have a poor prognosis despite intensive therapeutic regimens. Precision medicine approaches offer hope, and ex vivo drug response profiling of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370961/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370961">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62629-6"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62629.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370961%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370961/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370961/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370961/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7688. doi: <a href="https://doi.org/10.1038/s41467-025-62629-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62629-6</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Patient-derived tumoroids from CIC::DUX4 rearranged sarcoma identify MCL1 as a therapeutic target</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Breunis%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Willemijn Breunis</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Willemijn Breunis</span></h3>
<div class="p">
<sup>1</sup>Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Breunis%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Willemijn Breunis</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brack%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Eva Brack</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Eva Brack</span></h3>
<div class="p">
<sup>2</sup>Division of Pediatric Hematology/Oncology, Department of Pediatrics, Inselspital, Bern University Hospital, Bern, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brack%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eva Brack</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ehlers%20AC%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Anna C Ehlers</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Anna C Ehlers</span></h3>
<div class="p">
<sup>3</sup>Hopp-Children’s Cancer Center (KiTZ), Heidelberg, Germany </div>
<div class="p">
<sup>4</sup>Division of Translational Pediatric Sarcoma Research, German Cancer Research Center (DKFZ), German Cancer Consortium (DKTK), Heidelberg, Germany </div>
<div class="p">
<sup>5</sup>National Center for Tumor Diseases (NCT), NCT Heidelberg, a partnership between DKFZ and Heidelberg University Hospital, Heidelberg, Germany </div>
<div class="p">
<sup>6</sup>Medical Faculty, Ruprecht-Karls-University, Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ehlers%20AC%22%5BAuthor%5D" class="usa-link"><span class="name western">Anna C Ehlers</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bechtold%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ingrid Bechtold</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ingrid Bechtold</span></h3>
<div class="p">
<sup>1</sup>Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bechtold%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ingrid Bechtold</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kisele%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Samanta Kisele</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Samanta Kisele</span></h3>
<div class="p">
<sup>1</sup>Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kisele%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Samanta Kisele</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wurth%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jakob Wurth</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jakob Wurth</span></h3>
<div class="p">
<sup>1</sup>Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wurth%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jakob Wurth</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mous%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Lieke Mous</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Lieke Mous</span></h3>
<div class="p">
<sup>7</sup>Balgrist University Hospital, Faculty of Medicine, University of Zurich (UZH), Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mous%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lieke Mous</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zabele%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Dorita Zabele</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Dorita Zabele</span></h3>
<div class="p">
<sup>7</sup>Balgrist University Hospital, Faculty of Medicine, University of Zurich (UZH), Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zabele%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dorita Zabele</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Steffen%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Fabio Steffen</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Fabio Steffen</span></h3>
<div class="p">
<sup>1</sup>Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Steffen%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fabio Steffen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zahnow%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Felina Zahnow</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Felina Zahnow</span></h3>
<div class="p">
<sup>3</sup>Hopp-Children’s Cancer Center (KiTZ), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zahnow%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Felina Zahnow</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Britschgi%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Christian Britschgi</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Christian Britschgi</span></h3>
<div class="p">
<sup>8</sup>Department of Medical Oncology and Hematology, University Hospital Zurich, Comprehensive Cancer Center Zurich, Zurich, Switzerland </div>
<div class="p">
<sup>16</sup>Present Address: Medical Oncology and Hematology, Cantonal Hospital Winterthur, Winterthur, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Britschgi%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christian Britschgi</span></a>
</div>
</div>
<sup>8,</sup><sup>16</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bankel%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Lorenz Bankel</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Lorenz Bankel</span></h3>
<div class="p">
<sup>8</sup>Department of Medical Oncology and Hematology, University Hospital Zurich, Comprehensive Cancer Center Zurich, Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bankel%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lorenz Bankel</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rothermundt%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Christian Rothermundt</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Christian Rothermundt</span></h3>
<div class="p">
<sup>9</sup>Department of Medical Oncology and Hematology, Cantonal Hospital St. Gallen, St. Gallen, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rothermundt%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christian Rothermundt</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vetter%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Cornelia Vetter</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Cornelia Vetter</span></h3>
<div class="p">
<sup>10</sup>Division of Pediatric Hematology/Oncology, Children’s Hospital of Eastern Switzerland, St. Gallen, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vetter%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Cornelia Vetter</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22M%C3%BCller%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Daniel Müller</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Daniel Müller</span></h3>
<div class="p">
<sup>11</sup>University Sarcoma Center Zürich (CCCZ), Balgrist University Hospital, University of Zurich, Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22M%C3%BCller%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel Müller</span></a>
</div>
</div>
<sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Botter%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Sander Botter</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Sander Botter</span></h3>
<div class="p">
<sup>12</sup>Swiss Center for Musculoskeletal Biobanking, Balgrist Campus AG, Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Botter%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sander Botter</span></a>
</div>
</div>
<sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pauli%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Chantal Pauli</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Chantal Pauli</span></h3>
<div class="p">
<sup>13</sup>Department of Pathology and Molecular Pathology, University Hospital Zurich, Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pauli%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chantal Pauli</span></a>
</div>
</div>
<sup>13</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bode%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Peter Bode</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Peter Bode</span></h3>
<div class="p">
<sup>13</sup>Department of Pathology and Molecular Pathology, University Hospital Zurich, Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bode%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter Bode</span></a>
</div>
</div>
<sup>13</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rinner%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Beate Rinner</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Beate Rinner</span></h3>
<div class="p">
<sup>14</sup>Division of Biomedical Research, Medical University of Graz, Graz, Austria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rinner%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Beate Rinner</span></a>
</div>
</div>
<sup>14</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bourquin%20JP%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Jean-Pierre Bourquin</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Jean-Pierre Bourquin</span></h3>
<div class="p">
<sup>1</sup>Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bourquin%20JP%22%5BAuthor%5D" class="usa-link"><span class="name western">Jean-Pierre Bourquin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roessler%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Jochen Roessler</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Jochen Roessler</span></h3>
<div class="p">
<sup>2</sup>Division of Pediatric Hematology/Oncology, Department of Pediatrics, Inselspital, Bern University Hospital, Bern, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roessler%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jochen Roessler</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gr%C3%BCnewald%20TGP%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Thomas G P Grünewald</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Thomas G P Grünewald</span></h3>
<div class="p">
<sup>3</sup>Hopp-Children’s Cancer Center (KiTZ), Heidelberg, Germany </div>
<div class="p">
<sup>4</sup>Division of Translational Pediatric Sarcoma Research, German Cancer Research Center (DKFZ), German Cancer Consortium (DKTK), Heidelberg, Germany </div>
<div class="p">
<sup>5</sup>National Center for Tumor Diseases (NCT), NCT Heidelberg, a partnership between DKFZ and Heidelberg University Hospital, Heidelberg, Germany </div>
<div class="p">
<sup>15</sup>Institute of Pathology, Heidelberg University Hospital, Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gr%C3%BCnewald%20TGP%22%5BAuthor%5D" class="usa-link"><span class="name western">Thomas G P Grünewald</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sch%C3%A4fer%20BW%22%5BAuthor%5D" class="usa-link" aria-describedby="id23"><span class="name western">Beat W Schäfer</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Beat W Schäfer</span></h3>
<div class="p">
<sup>1</sup>Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sch%C3%A4fer%20BW%22%5BAuthor%5D" class="usa-link"><span class="name western">Beat W Schäfer</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Surdez%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id24"><span class="name western">Didier Surdez</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Didier Surdez</span></h3>
<div class="p">
<sup>7</sup>Balgrist University Hospital, Faculty of Medicine, University of Zurich (UZH), Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Surdez%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Didier Surdez</span></a>
</div>
</div>
<sup>7,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wachtel%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id25"><span class="name western">Marco Wachtel</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Marco Wachtel</span></h3>
<div class="p">
<sup>1</sup>Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wachtel%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marco Wachtel</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Oncology and Children’s Research Center, University Children’s Hospital, University of Zurich, Zurich, Switzerland </div>
<div id="Aff2">
<sup>2</sup>Division of Pediatric Hematology/Oncology, Department of Pediatrics, Inselspital, Bern University Hospital, Bern, Switzerland </div>
<div id="Aff3">
<sup>3</sup>Hopp-Children’s Cancer Center (KiTZ), Heidelberg, Germany </div>
<div id="Aff4">
<sup>4</sup>Division of Translational Pediatric Sarcoma Research, German Cancer Research Center (DKFZ), German Cancer Consortium (DKTK), Heidelberg, Germany </div>
<div id="Aff5">
<sup>5</sup>National Center for Tumor Diseases (NCT), NCT Heidelberg, a partnership between DKFZ and Heidelberg University Hospital, Heidelberg, Germany </div>
<div id="Aff6">
<sup>6</sup>Medical Faculty, Ruprecht-Karls-University, Heidelberg, Germany </div>
<div id="Aff7">
<sup>7</sup>Balgrist University Hospital, Faculty of Medicine, University of Zurich (UZH), Zurich, Switzerland </div>
<div id="Aff8">
<sup>8</sup>Department of Medical Oncology and Hematology, University Hospital Zurich, Comprehensive Cancer Center Zurich, Zurich, Switzerland </div>
<div id="Aff9">
<sup>9</sup>Department of Medical Oncology and Hematology, Cantonal Hospital St. Gallen, St. Gallen, Switzerland </div>
<div id="Aff10">
<sup>10</sup>Division of Pediatric Hematology/Oncology, Children’s Hospital of Eastern Switzerland, St. Gallen, Switzerland </div>
<div id="Aff11">
<sup>11</sup>University Sarcoma Center Zürich (CCCZ), Balgrist University Hospital, University of Zurich, Zurich, Switzerland </div>
<div id="Aff12">
<sup>12</sup>Swiss Center for Musculoskeletal Biobanking, Balgrist Campus AG, Zurich, Switzerland </div>
<div id="Aff13">
<sup>13</sup>Department of Pathology and Molecular Pathology, University Hospital Zurich, Zurich, Switzerland </div>
<div id="Aff14">
<sup>14</sup>Division of Biomedical Research, Medical University of Graz, Graz, Austria </div>
<div id="Aff15">
<sup>15</sup>Institute of Pathology, Heidelberg University Hospital, Heidelberg, Germany </div>
<div id="Aff16">
<sup>16</sup>Present Address: Medical Oncology and Hematology, Cantonal Hospital Winterthur, Winterthur, Switzerland </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Sep 5; Accepted 2025 Jul 25; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370961  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841513/" class="usa-link">40841513</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">High-risk sarcomas, such as metastatic and relapsed Ewing and <em>CIC</em>-rearranged sarcoma, still have a poor prognosis despite intensive therapeutic regimens. Precision medicine approaches offer hope, and ex vivo drug response profiling of patient-derived tumor cells emerges as a promising tool to identify effective therapies for individual patients. Here, we establish ex vivo culture conditions to propagate Ewing sarcoma and CIC::DUX4 sarcoma as tumoroids. These models retain their original molecular and functional characteristics, including recurrent <em>ARID1A</em> mutations in CIC::DUX4 sarcoma, and serve as tumor avatars for large-scale drug testing. Screening a large drug library on a small living biobank of such tumors not only reveals distinct differences in drug response between the two entities, but also identifies a dependency of CIC::DUX4 sarcoma cells on MCL1. Mechanistically, <em>MCL1</em> is identified as a direct transcriptional target of the CIC::DUX4 fusion oncogene. Genetic and pharmacological inhibition of MCL1 induces rapid apoptosis in CIC::DUX4 sarcoma cells and inhibits tumor growth in a xenograft model. Thus, MCL1 represents a potential therapeutic target for CIC::DUX4 sarcoma. Overall, our study highlights the feasibility of drug response profiling for individual sarcoma cases and suggests that further clinical assessments of its benefit are warranted.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Sarcoma, Targeted therapies</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Despite their differences, the rarer sarcoma CIC::DUX4 sarcoma (CDS) is typically treated with therapies developed for Ewing Sarcoma (EwS) with limited success. Here, the authors develop a co-clinical drug response profiling platform to establish patient-derived CDS and EwS tumoroids, identifying MCL1 inhibition as a promising therapeutic approach in CDS.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Sarcomas are a heterogeneous group of tumors of mesenchymal origin that are broadly subclassified into soft tissue and bone sarcomas. Ewing sarcoma (EwS) is the second most common malignant bone tumor in children and young adolescents, with an incidence of approximately one case per 1.5 million people per year. EwS can occur in any bone but is most commonly found in the diaphysis and diaphyseal-metaphyseal portions of long bones, the pelvis and ribs. Up to 20% of these tumors have an extraosseous localization<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. EwS is primarily driven by FET::ETS fusion transcription factors (TFs), most commonly EWSR1::FLI1<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. In addition to classical EwS, there are rarer tumors with histological similarities, which were originally classified as Ewing-like sarcomas. Based on the identification of pathognomonic translocations, these sarcoma entities were reclassified in the 5th edition of the World Health Organization (WHO) classification of soft tissue and bone tumors as undifferentiated small round cell sarcomas of bone and soft tissue (URCS). These are further subdivided into three entities: <em>CIC</em>-rearranged sarcoma, round cell sarcoma with <em>EWSR1</em>::non-<em>ETS</em> fusions, and sarcoma with <em>BCOR</em> genetic alterations<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. <em>CIC</em>-rearranged sarcomas are the most frequent URCS entity. They are defined by <em>CIC</em>-related gene fusions, most commonly <em>CIC::DUX4</em> (<em>CIC::DUX</em> sarcoma; CDS)<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. In the latter case, the capicua transcriptional repressor (<em>CIC</em>) gene, located on chromosome 19q13, is fused with one of two double homeobox 4 (<em>DUX4</em>) genes located on chromosome 4q35 or 10q26, respectively. The fusion event functionally alters the CIC repressor to become a strong transcriptional activator that induces an oncogenic gene expression program including target genes such as <em>ETV1</em> and <em>ETV4</em><sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Patients with <em>CIC</em>-rearranged sarcoma have a wide age range at presentation, with a peak incidence in young adults. The tumors are often localized in the soft tissue of the limbs or the trunk. In rare cases, osseous involvement is identified and up to 40% of patients present with metastases at diagnosis<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Currently, both EwS and URCS are treated with a similar multidisciplinary approach, including neo-adjuvant/induction chemotherapy, local treatment via surgery and/or radiotherapy, and consolidation chemotherapy. EwS tumors respond quite well to this treatment, resulting in a 3-year overall survival of 75% for patients with localized disease. In stark contrast, overall survival in patients with CDS is significantly worse, with survival rates described to be between 0% and 56%<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. This underscores the clinical importance of the re-classification and strongly suggests the need for identifying alternative, more effective treatment approaches for CDS. While in the long term, the fusion oncogenes involved may represent the most promising therapeutic targets, the challenge of developing efficient methods to interfere with their activity as TFs has hindered the development of effective, clinically applicable inhibitors so far. Furthermore, fusion-driven tumors normally have a flat mutational landscape and the fusion protein is often the only apparent oncogenic driver present<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Genetic analyses aimed at identifying other driver oncogenes as therapeutic targets therefore have only limited therapeutic potential<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>.</p>
<p id="Par4">As an alternative, cells isolated from patient tumors can be directly tested with drugs to identify the most effective therapy in a personalized manner, independent of underlying molecular characteristics. Particularly for leukemia, co-clinical drug response profiling (DRP) procedures have been successfully established by different groups in recent years. Therapy guidance based on DRP data has been shown to improve outcomes in heavily pretreated patients with hematologic malignancies<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Hence, for blood cancer, this approach is now close to translation into clinical practice<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>–<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. In contrast to leukemia, where blood sampling represents a straightforward method for obtaining tumor cells, material available from solid tumors is often limited to small needle biopsies. This limitation complicates the DRP approach, making cell expansion crucial for testing a broader range of drugs. Advances in culture conditions over the last decade have considerably improved the success rates of establishing patient-derived ex vivo models. A prominent example is the organoid model approach amenable to different types of epithelial cancers, as well as some non-epithelial tumors<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. However, standardized culture conditions for rarer tumors such as EwS and CDS have yet to be established.</p>
<p id="Par5">Here, we describe a DRP platform for EwS and CDS tumors. We developed culture conditions that enable the establishment of patient-derived tumoroid models from EwS and CDS tumors with high efficiency. Importantly, these models faithfully recapitulate their original tumors at both the molecular and functional drug-response levels. Differential drug sensitivity analyses reveal a strong dependency of CDS cells on the anti-apoptotic protein MCL1. This finding could pave the way for more effective therapeutic strategies for CDS.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Tumoroid models of EwS and CDS</h3>
<p id="Par6">To expand cells derived from small biopsies of EwS and CDS tumors for patient-specific drug profiling, we used an organoid culture approach. The basis was conditions that were originally optimized for epithelial and carcinoma organoids, which are now widely used to culture organoids derived from various tissues, including sarcomas<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Tumor tissue was first dissociated mechanically (small biopsies) or enzymatically (larger biopsies), and single cells were suspended in Matrigel and plated as Matrigel domes (if &lt;500,000 cells were available) or the cell suspension was directly plated on a thick layer of Matrigel (if &gt; 500,000 cells were available) overlaid with culture medium (Fig. <a href="#Fig1" class="usa-link">1a</a>). In total, 14 EwS tumors originating from 13 patients (one diagnostic-relapse pair) and four CDS tumors originating from three patients (one diagnostic-relapse pair) (see Supplementary Table <a href="#MOESM1" class="usa-link">1</a> for clinical data) were available for this procedure. When enough dissociated cells from the patient tumor were available, a direct drug profiling was performed in parallel and used for comparison with the tumoroid approach (Fig. <a href="#Fig1" class="usa-link">1a</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Generation of tumoroid models from EwS and CDS tumors.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370961_41467_2025_62629_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0ea1/12370961/bbaedf6827ec/41467_2025_62629_Fig1_HTML.jpg" loading="lazy" id="d33e669" height="641" width="800" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Scheme depicting the drug profiling pipeline with separate arms for direct drug profiling (lower arm) and profiling after generation of ex vivo tumoroid models (upper arm). Created in BioRender. Wachtel, M. (2025) <a href="https://BioRender.com/t5q1jju" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/t5q1jju</a>. <strong>b</strong> Phase-contrast microscopy images illustrating the development of three EwS models. Images were taken at an early phase of culture in Matrigel domes (left panels) and at later time points after cell expansion in Matrigel domes or on a thick layer of Matrigel (middle and right panels, respectively). Each tumoroid was established once from human tumor material. Scale bar, 400 μm. <strong>c</strong> Time required to generate individual tumoroid models (left panels) and overall success rate (right panels) for EwS and CDS tumors. <strong>d</strong> Viability of indicated EwS and CDS cells after treatment with bFGF, EGF and IGF1 for seven days relative to cells grown in absence of growth factors. Cell viability was assessed using CTG assay. Plotted are means ± SD. <em>n</em> = 4 independent experiments, two-way ANOVA, Tukey’s multiple comparisons test. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par7">In seven EwS cases (50%) and all four CDS cases (100%), we could successfully establish a tumoroid model (Fig. <a href="#Fig1" class="usa-link">1b and c</a>). After a few days to several weeks in culture, tumor cells typically formed small clusters that further enlarged over time. After one to two passages in Matrigel domes, cells were propagated on a thick layer of Matrigel, on which they continued to grow as 3D structures (Fig. <a href="#Fig1" class="usa-link">1b</a>). Most of the models also grew as a monolayer when plated on a thin layer of Matrigel, in some cases as 3D clusters of cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">S1</a>). On average, after 113 days (42-210 days) of culture, a sufficient number of cells for large-scale drug testing was available (Fig. <a href="#Fig1" class="usa-link">1c</a>). To further optimize culture conditions, we tested whether different growth factors would accelerate tumoroid growth. We chose bFGF and EGF, both widely used in organoid protocols, and IGF1, as the IGF1R pathway plays a crucial role in EwS and other sarcoma entities. We supplemented the culture medium of three EwS and three CDS models with each growth factor and assessed cell viability after seven days of treatment. Interestingly, bFGF reduced viability of several models, in case of ES-ZH001 through rapid and strong induction of cell death as detected already after four days of treatment (Fig. <a href="#Fig1" class="usa-link">1d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">S2a-d</a>), a finding consistent with published data on EwS cell lines<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Furthermore, in the EwS models, expression of known EWSR1::FLI1 target genes was strongly deregulated upon bFGF treatment (Supplementary Fig. <a href="#MOESM1" class="usa-link">S2e</a>). In contrast, EGF and IGF1 had only minor effects (Fig. <a href="#Fig1" class="usa-link">1d</a>). Based on these data, all tumoroid models were subsequently cultured in absence of any supplemental growth factors.</p>
<p id="Par8">To evaluate whether the models maintained representative characteristics for the respective tumor entities, we characterized them at the molecular level. We performed DNA methylation profiling, RNA sequencing (RNA-seq) and whole exome sequencing (WES) of all the models, as well as of the original patient tumors, if material was available. We first integrated the DNA methylation profiles from the tumoroid models with a large reference cohort from the sarcoma classifier (<a href="http://www.molecularneuropathology.org/mnp" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.molecularneuropathology.org/mnp</a>, <em>n</em> = 1077)<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> using t-SNE dimensionality reduction. This analysis revealed close association of the tumoroids with the corresponding tumor class (Fig. <a href="#Fig2" class="usa-link">2a</a>). Accordingly, the DNA methylation-based classification score from the sarcoma classifier revealed that with the exception of two EwS cases, samples matched the DNA methylation class of the respective tumor type (calibrated classifier scores ≥0.9 used as threshold for sarcoma subtype prediction) (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). Interestingly, the histological appearance of one of the two outlier tumors (classifier score 0.63) was non-classical for EwS, despite the presence of <em>EWSR1::FLI1</em> (see below). In contrast, classifier scores of classical EwS cell lines (data from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE176339" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE176339</a><sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>) are mostly low (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). This suggests that DNA methylation patterns are stable during the establishment of the tumoroid models under the applied conditions, whereas in some of the available cell lines major changes in the methylation profiles have occurred.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Molecular characterization of EwS and CDS tumoroid models.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370961_41467_2025_62629_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0ea1/12370961/f2fb26af4efe/41467_2025_62629_Fig2_HTML.jpg" loading="lazy" id="d33e773" height="1000" width="666" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Visualization of DNA methylation profiles of CDS and EwS models (<em>n</em> = 9) integrated with methylation profiles of the sarcoma classifier reference cohort<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> (<em>n</em> = 1077) using t-SNE dimensionality reduction. Samples are color-coded according to 62 tumor and three control DNA methylation classes. The inset shows a magnification of the dashed-line area and indicates the location of the individual models. For a description of abbreviations, see extended legend in the supplementary section. <strong>b</strong> Fusion genes and mutational landscape of EwS and CDS tumoroid models. Data from corresponding primary tumors is shown for comparison when available. <strong>c</strong> Schematic representation of the <em>CIC</em> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_001379485.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_001379485.1</a>) and <em>DUX4</em> gene architectures, with translocation breakpoints in CDS models indicated by red lines. <strong>d</strong> Unsupervised hierarchical clustering based on RNA-seq gene expression data from EwS and CDS tumoroid models. The 2000 most variably expressed genes were used. <strong>e</strong> Volcano plot depicting genes differentially expressed between EwS and CDS models. Analysis was performed with DSeq2 using fold change&gt; 2 and FDR &lt; 0.1. The ten most differentially expressed genes, as well as CDS markers ETV4, ETV5 and WT1, are labeled.</p></figcaption></figure><p id="Par9">Next, we used RNA-seq and WES data for identification of translocations and somatic mutations, respectively. A search for translocations revealed different types of <em>EWSR1::FLI1</em> fusions in the EwS samples (Fig. <a href="#Fig2" class="usa-link">2b</a>). In addition, some of the EwS tumors also contained <em>STAG2</em> and <em>TP53</em> mutations (Fig. <a href="#Fig2" class="usa-link">2b</a>), corroborating known mutations in EwS tumors. Additional intracellular flow cytometry revealed a loss of STAG2 expression in two out of four tested models (Supplementary Fig. <a href="#MOESM1" class="usa-link">S3</a>). In case of CDS models, some commonly used fusion callers did not detect <em>CIC</em>-translocations based on RNA-seq data, similar to another case described in the literature<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, possibly due to the complex structure of the <em>DUX4</em> loci, which contain numerous pseudogenes. Archer Fusionplex analysis however confirmed the <em>CIC::DUX4</em> fusion in all CDS samples (Fig. <a href="#Fig2" class="usa-link">2c</a>). Interestingly, two of the CDS models contained frameshift mutations in <em>ARID1A</em>, as already described for individual cases<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>, suggesting that this is indeed a common mutation in CDS (Fig. <a href="#Fig2" class="usa-link">2b</a>). Overall, our data reflects the known mutational landscape of these two types of sarcoma.</p>
<p id="Par10">Finally, we performed a gene expression analysis with the RNA-seq dataset. We applied a comparative approach between EwS and CDS datasets to identify subtype specific gene signatures. Unsupervised hierarchical clustering with the 2,000 most differentially expressed genes revealed large differences between the two entities (Fig. <a href="#Fig2" class="usa-link">2d</a> and Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). Differential gene expression analysis with fc&gt; 2 and fdr&lt; 0.1 identified 2,139 and 2,544 genes up- and downregulated in CDS compared to EwS tumors, respectively (Fig. <a href="#Fig2" class="usa-link">2e</a> and Supplementary Data <a href="#MOESM4" class="usa-link">2</a>). Among the genes upregulated in CDS were ETV4, ETV5, and WT1, all well-known markers of CDS. Gene set enrichment analysis (GSEA) using the curated Molecular Signatures Database (MSigDB) revealed the expected association of EwS and EWSR1::FLI1 target gene signatures with the EwS models (Supplementary Table <a href="#MOESM1" class="usa-link">3</a>). Interestingly, GSEA with the KEGG pathway database revealed an upregulation of different signaling pathways in CDS compared to EwS cells, including the PI3K and the Notch pathway, the former being in agreement with recently published data from CDS PDX models (Supplementary Table <a href="#MOESM1" class="usa-link">3</a>)<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Overall, the molecular analyses demonstrate that the specific molecular characteristics of the corresponding tumor types are maintained in our tumor models.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Tumoroid models as tool for ex vivo drug profiling</h3>
<p id="Par11">To investigate whether tumoroids can serve as models for therapy guidance, we performed drug response profiling (DRP) using a clinically oriented library of 245 drugs (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>) applied at four different concentrations ranging from 10 nM to 10 μM. Cell viability was assessed by CellTiter-Glo assay after 72 h of treatment. The drug library included most of the chemotherapeutics commonly used for sarcoma treatment, as well as a range of experimental drugs targeting various signaling pathways to identify druggable dependencies. Importantly, in one CDS and one EwS case, a sufficient number of cells from the patient’s tumor was available to perform upfront direct DRP. Comparison of the response patterns from these direct screens with those from the corresponding tumoroid models revealed a high concordance (Pearson correlation coefficients r = 0.8524 and r = 0.8352, respectively) (Fig. <a href="#Fig3" class="usa-link">3a</a>), suggesting that drug response characteristics are also maintained in our organoid models. Comparison of drug responses across different models revealed heterogeneity among samples (Fig. <a href="#Fig3" class="usa-link">3b and c</a>). Strikingly, EwS and CDS models were clearly separated by unsupervised hierarchical clustering and PCA, with EwS cells showing greater sensitivity to many of the tested drugs than CDS cells (Fig. <a href="#Fig3" class="usa-link">3b and c</a>). To better quantify these differences in response, we calculated a differential drug sensitivity score (dDSS)<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> for each drug in every sample. The drug sensitivity score (DSS) is computed as the normalized area under the curve (AUC) over the measured concentration range, scaling between 0 (minimal drug activity) and 100 (maximal drug activity). The dDSS score quantifies the difference between the drug response of a given sample of interest and the average drug response of a reference cohort<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. It is calculated by subtracting the average DSS of the reference cohort from the DSS of the sample of interest. Thus, the larger the difference to the reference cohort, the more exceptional the drug response of a given sample. As a reference dataset, we used DRP data generated in our lab from 34 sarcoma tumors from different entities (23 rhabdomyosarcomas, two synovial sarcomas, and nine other sarcomas) (Fig. <a href="#Fig3" class="usa-link">3d</a>). We first calculated a mean dDSS value for several classes of chemotherapeutics (chemoscores). These scores highlight the poorer response of CDS cells to most chemotherapeutics compared to other sarcomas, with dDSS values mostly below 0, indicating lower-than-average sensitivity (Fig. <a href="#Fig3" class="usa-link">3e</a>). In contrast, EwS cells exhibited above-average sensitivity to most chemotherapeutic classes. Overall, these data align with the poor clinical prognosis of CDS and the relatively favorable response of EwS tumors to cytotoxic chemotherapy<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Comparative drug response profiling identifies MCL1 as a therapeutic target in CDS.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370961_41467_2025_62629_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0ea1/12370961/315b954e6179/41467_2025_62629_Fig3_HTML.jpg" loading="lazy" id="d33e958" height="1000" width="724" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Correlation of drug response of cells tested immediately after isolation from patient tumors with the corresponding tumoroid models. For correlations, area under the curve (AUC) values were calculated from dose-response curves of 245 compounds. Direct tests: <em>n</em> = 1 experiment, tumoroid models: <em>n</em> = 2 independent experiments; Pearson correlation. <strong>b</strong> Principal component analysis using IC50 values generated from dose-response data from 245 drugs. Arrows link diagnostic and relapse samples from the same patient. <em>n</em> = 2 independent experiments. <strong>c</strong> Unsupervised hierarchical clustering of indicated EwS and CDS based on IC50 values of 245 drugs, as described in <strong>b</strong>. <strong>d</strong> Median differential drug sensitivity scores (dDSS) calculated from dose-response data generated with 245 drugs. Data points indicate individual tested models (<em>n</em> = 5 CDS and <em>n </em>= 8 EwS models). Plotted are means ± SD. <strong>e</strong> Mean differential dDSS calculated for indicated classes of chemotherapeutics (chemotype score). Chemotype scores represent the mean dDSS of all chemotherapeutics within the indicated drug class. Data points indicate individual models (<em>n</em> = 5 CDS and <em>n</em> = 8 EwS models). Plotted are means ± SD. <strong>f</strong> Individual dDSS calculated from dose-response data of 245 drugs and indicated CDS and EwS models. Data points for the <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a> and Talazoparib are labeled in red and green, respectively. <strong>g</strong> Dose-response curves for <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a> generated with a sarcoma reference cohort. In addition to EwS and CDS models, 23 rhabdomyosarcomas, two synovial sarcomas, and nine other sarcomas are included. Data are presented as means ± SD. CDS are highlighted in red. <strong>h</strong> Competition assay for evaluation of MCL1-dependency in CDS cells. <em>MCL1</em> was knocked out in CDS-ZH003 cells using CRISPR-Cas9 with sgRNAs cloned into mCherry-expressing Cas9-expression vectors. Knockout efficiency was measured by Western blot (upper right panel, representative blot from <em>n</em> = 2 independent experiments). Effects of <em>MCL1</em> knockouts on cell survival were determined by monitoring depletion of mCherry-labeled cells between day two and nine after transduction using flow cytometry (left panel). Plotted are means ± SD. <em>n</em> = 3 independent experiments, one-way ANOVA, Tukey’s multiple comparisons test. Cell morphology three days after sgRNA transduction is shown in the lower right panels. Scale bar, 200 μm. Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Exceptional sensitivity of CDS to MCL1 inhibition</h3>
<p id="Par12">Given the urgent clinical need for better treatments for patients diagnosed with CDS, we investigated whether certain drugs might be particularly effective against them. We hypothesized that drugs eliciting exceptional ex vivo responses in a specific test case compared to a larger reference cohort, are the most promising candidates for clinical benefit in these patients. Strikingly, in four out of five CDS datasets, the drug with the highest dDSS, indicative of the highest selective effect in CDS tumors relative to the reference cohort, was the MCL1 inhibitor <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a> (Fig. <a href="#Fig3" class="usa-link">3f</a> and Supplementary Data <a href="#MOESM5" class="usa-link">3</a>). Other drugs with high dDSS in some CDS cases included the XIAP/cIAP1 antagonist birinapant and the p300/CBP inhibitor A-485 (Supplementary Data <a href="#MOESM5" class="usa-link">3</a>), the latter aligning with the mechanism of action of CIC::DUX4<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Dose-response curves confirmed the exceptional sensitivity of CDS cells to <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a> relative to other sarcoma models, with IC50 values ranging from 1-10 nM in the most sensitive cases (Fig. <a href="#Fig3" class="usa-link">3g</a>, and Supplementary Fig. <a href="#MOESM1" class="usa-link">S4</a> and Supplementary Table <a href="#MOESM1" class="usa-link">5</a>). Tests with BRD-810, a structurally unrelated MCL1 inhibitor, for which PRISM data from 696 cell lines are available<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, further confirmed these findings. For CDS-ZH001, CDS-ZH003, and an additional CDS model (MUG CIDUS; established at a different center), IC50 values of BRD-810 ranked 7th, 13th, and 32nd when added to the list of 696 tumor cell lines (Supplementary Fig. <a href="#MOESM1" class="usa-link">S5a–c</a>). In agreement with the known high sensitivity of EwS cells to PARP inhibitors<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, talazoparib was detected among the most selective drugs for the EwS models, while for <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a> a wide range of sensitivities, including some good responses, was detected (Fig. <a href="#Fig3" class="usa-link">3f</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">S4</a>).</p>
<p id="Par13">Based on these data, we focused on MCL1 and validated the dependency of CDS cells on MCL1 at the genetic level using a CRISPR-based knockout approach in CDS-ZH003 cells. We used two sgRNAs directed against <em>MCL1</em>, both of which efficiently disrupted MCL1 expression (Fig. <a href="#Fig3" class="usa-link">3h</a>), while expression levels of BCL2 and BCL-XL were not significantly affected under these conditions (Supplementary Fig. <a href="#MOESM1" class="usa-link">S6</a>). MCL1 knockout was accompanied by very efficient induction of cell death, as well as depletion of the cells in a competition assay with cells transduced with a control sgRNA, measured by flow cytometry (Fig. <a href="#Fig3" class="usa-link">3h</a>). Overall, these data demonstrate the strong dependency of CDS cells on MCL1 and suggest that this protein may serve as a therapeutic target for CDS tumors.</p>
<p id="Par14">We next aimed to characterize these findings at the molecular level. For this, we used the MCL1 inhibitor <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a>, a more clinically advanced derivative of <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a>, which was included in the original library. Dose-response analysis confirmed the high sensitivity of the CDS also to this drug (Fig. <a href="#Fig4" class="usa-link">4a and b</a>). Flow cytometric analysis of CDS-ZH003 cells stained with Annexin-V and propidium iodide after treatment with 10 and 50 nM <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> for 24 h revealed effective and rapid induction of cell death (Fig. <a href="#Fig4" class="usa-link">4c</a>). PARP cleavage could be detected at this early time point (Fig. <a href="#Fig4" class="usa-link">4d</a>), as expected from an inhibitor of an anti-apoptotic protein. This was further confirmed by co-treatment of the cells with different concentrations of the pan-caspase inhibitor Z-VAD-FMK. 50 μM Z-VAD-FMK efficiently rescued cell viability after 24 h of treatment with different <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> concentrations (Fig. <a href="#Fig4" class="usa-link">4e</a>). Taken together, these findings demonstrate that MCL1 inhibitors quickly and efficiently induce caspase-dependent apoptosis in CDS cells.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. MCL1 inhibition efficiently induces apoptosis in CDS tumor cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370961_41467_2025_62629_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0ea1/12370961/9b07896e14ac/41467_2025_62629_Fig4_HTML.jpg" loading="lazy" id="d33e1070" height="917" width="800" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Dose-response curves for CDS (left panel) and EwS (right panel) tumoroid models treated with the MCL1 inhibitor <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a>. Cell viability was measured at the three indicated time points by CTG assay. Data are presented as means. <em>n</em> = 2 independent experiments. <strong>b</strong> IC50 values calculated from the data shown in a after 72 h of treatment with <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a>. <strong>c</strong> Detection of cell death after treatment of CDS-ZH003 with the indicated doses of <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> for 24 h using Annexin V and 7-AAD staining followed by flow cytometry. Representative pseudocolor density plots illustrate the number of live and dead cells (upper and lower-left panels). The percentage of dead cells, defined as Annexin V and/or 7-AAD positive, is shown in the lower right panel. Plotted are means ± SD. <em>n</em> = 3 independent experiments, one-way ANOVA, Tukey’s multiple comparisons test. <strong>d</strong> Western blot analysis of cell lysates from three indicated CDS tumoroid models after treatment with <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> for 24 h. <em>n</em> = 1 independent experiment. <strong>e</strong> Viability of CDS-ZH003 cells after treatment with the indicated dose range of <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> in presence or absence of different doses of Z-VAD-FMK. Plotted are means ± SD. <em>n</em> = 3 independent experiments, two-way ANOVA, Tukey’s multiple comparisons test. Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">MCL1 is a target gene of CIC::DUX4</h3>
<p id="Par15">Next, we investigated whether the exceptional sensitivity of CDS cells to MCL1 inhibitors correlated with the expression levels of MCL1 and other anti-apoptotic BCL2 family members. We measured MCL1, BCL2 and BCL-XL levels by Western blot in three CDS and three EwS models. Both MCL1 and BCL2 were found to be highly overexpressed in CDS compared to EwS cells, whereas BCL-XL expressed levels were similar in both (Fig. <a href="#Fig5" class="usa-link">5a</a>). Similar differences were seen at the mRNA level (Fig. <a href="#Fig5" class="usa-link">5b</a>). To confirm these findings in a larger cohort of samples, we leveraged gene expression data from a large study of various sarcomas and normal tissues<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. This analysis revealed that <em>BCL2</em> mRNA levels are overexpressed in CDS tumors compared to other sarcoma subtypes and to a range of normal tissues (Supplementary Fig. <a href="#MOESM1" class="usa-link">S7a-b</a>), in agreement with a case study showing high BCL2 protein expression in a CDS tumor<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. <em>MCL1</em> overexpression was less evident, but a statistically significant difference between CDS and EwS could also be seen (Supplementary Fig. <a href="#MOESM1" class="usa-link">S7a-b</a>). We therefore wondered whether the high MCL1 expression levels in CDS cells result from transcriptional upregulation by the CIC::DUX4 fusion transcription factor.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. <em>MCL1</em> is a CIC::DUX4 target gene in CDS cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370961_41467_2025_62629_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0ea1/12370961/21c802370905/41467_2025_62629_Fig5_HTML.jpg" loading="lazy" id="d33e1198" height="997" width="800" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Western blot analysis of lysates from indicated CDS and EwS tumoroid models. <em>n</em> = 1 independent experiment. <strong>b</strong> mRNA levels of <em>MCL1</em> and <em>BCL2</em>, determined by normalized read count analysis of RNA-seq data. Data points indicate individual samples (<em>n</em> = 5 CDS and <em>n</em> = 6 EwS samples). Red dots indicate primary tumors, and black dots indicate tumoroid models. Data are presented as means ± SD. Unpaired two-sided t-test. <strong>c</strong> ChIP-seq tracks for CIC::DUX4 and H3K27Ac at the <em>MCL1</em> locus in CDS1 and CDS2 cells. CIC::DUX4 peaks further studied are highlighted in yellow. ChIP-seq data was previously published<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. <strong>d</strong> Log2 fold change of <em>MCL1</em> mRNA levels in CDS-ZH003 cells after CRISPR interference with the indicated sgRNAs. Data are presented as means ± SD. One-way ANOVA, Tukey’s multiple comparison test, <em>n</em> = 3 independent experiments. <strong>e</strong> Effect of the indicated shRNA on viability of CDS-ZH003 cells. Cells transduced with indicated doxycycline-inducible shRNA constructs were either treated with doxycycline or left untreated for five days. Live and dead cells were quantified using high content analysis. Plotted are means ± SD. Two-way ANOVA, Tukey’s multiple comparison test, <em>n</em> = 3 independent experiments. p-values were calculated for the live cell fractions. <strong>f</strong> Western blot analysis of lysates from CDS-ZH003 cells transduced with the indicated shRNA construct and either treated with doxycycline for four days or left untreated. The blot is representative for <em>n</em> = 3 independent experiments. <strong>g</strong> Quantification of CIC::DUX4 silencing. Western blot bands of the short forms of CIC::DUX4 and CIC were quantified by densitometry, and the CIC::DUX4-to-CIC ratio was calculated. Plotted are means ± SD. Two-way ANOVA, Tukeyʹs multiple comparison test, <em>n</em> = 3 independent experiments. <strong>h</strong> Quantification of MCL1 protein downregulation. MCL1 and β-Actin Western blot bands were quantified by densitometry and the ratio was calculated. Plotted are means ± SD. Two-way ANOVA, Tukey’s multiple comparison test, <em>n</em> = 3 independent experiments. <strong>i</strong> Log2 fold change of mRNA levels of <em>MCL1, BCL2</em> and <em>ETV4</em> in CDS-ZH003 cells after silencing of <em>CIC::DUX4</em> with the indicated shRNA. <em>B2M</em> was used as reference gene. Plotted are means ± SD. Two-way ANOVA, Tukey’s multiple comparison test, <em>n</em> = 3 independent experiments. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par16">To explore a potential link between CIC::DUX4 and the regulation of MCL1 and/or BCL2 expression, we analyzed published CIC::DUX4 ChIP-seq data from two different CDS cell lines for evidence of CIC::DUX4 binding in the vicinity of <em>MCL1</em><sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. The analysis identified two CIC::DUX4 peaks in or near <em>MCL1</em>, one in exon 2 (transcript ID <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_021960.5" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_021960.5</a>) and another one ~6.4 kbp downstream of the gene (Fig. <a href="#Fig5" class="usa-link">5c</a>). High levels of H3K27Ac were detected at the same sites, suggesting that these regions may function as enhancer elements (Fig. <a href="#Fig5" class="usa-link">5c</a>). In contrast, in case of <em>BCL2</em>, in only one of the two cell lines a small CIC::DUX4 peak was found near the last intron at the 3‘-end (Supplementary Fig. <a href="#MOESM1" class="usa-link">S8a</a>). We used a CRISPR interference approach to assess the relevance of these potential regulatory elements for MCL1 expression in CDS-ZH003 cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">S8b</a>). Targeting the downstream peak with dCas9-KRAB and specific sgRNAs significantly reduced <em>MCL1</em> mRNA levels, whereas interference with the exonic peak had no effect (Fig. <a href="#Fig5" class="usa-link">5d</a>). This indicates that the downstream genetic element bound by CIC::DUX4 is indeed involved in the transcriptional regulation of <em>MCL1</em>.</p>
<p id="Par17">To further validate this regulatory relationship, we silenced CIC::DUX4 expression in CDS-ZH003 cells using three doxycycline (dox)-inducible shRNAs targeting the <em>DUX4</em> portion of the fusion transcript. Dox treatment of cells stably transduced with shDUX4-1 and -3 halted cell proliferation and induced cell death within three to four days, whereas shDUX4-2 had no physiological effect (Fig. <a href="#Fig5" class="usa-link">5e</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">S9a–c</a>). To quantify silencing at the protein level, we compared the levels of CIC::DUX4 with those of wildtype CIC by Western Blot. Both proteins are expressed at nearly stoichiometric levels in a long and a short form, corresponding to two transcripts with different transcriptional start sites (TSSs). These forms are distinguished by a large exon near the N-terminus, as determined by CRISPR interference using sgRNAs targeting the two TSSs (Supplementary Fig. <a href="#MOESM1" class="usa-link">S10a-b</a>). shDUX4-1 and -3 both reduced CIC::DUX4 levels by more than 50 percent compared to CIC, whereas shDUX4-2 had only minor effects (Fig. <a href="#Fig5" class="usa-link">5f–g</a>). CIC::DUX4 silencing led to a marked reduction in MCL1 protein levels, most notably with shDUX4-3, as well as to a slight reduction in BCL2 levels (Fig. <a href="#Fig5" class="usa-link">5f and h</a>). qRT-PCR detection of <em>MCL1</em> transcripts demonstrated that this effect corresponds to reduced mRNA levels (Fig. <a href="#Fig5" class="usa-link">5i</a>). Notably, we detected a similar degree of downregulation for the well-known CIC::DUX4 target gene <em>ETV4</em> (Fig. <a href="#Fig5" class="usa-link">5i</a>). Importantly, <em>GAPDH</em> mRNA levels were also strongly affected by CIC::DUX4 silencing and could not be used as a reference for mRNA quantification (Supplementary Fig. <a href="#MOESM1" class="usa-link">S11a</a>). In contrast, <em>B2M</em> and <em>ANXA5</em> mRNA levels were both found to be CIC::DUX4 independent (Supplementary Fig. <a href="#MOESM1" class="usa-link">S11b</a>), and therefore <em>B2M</em> was used as a reference gene in this experiment. Taken together, these data establish <em>MCL1</em> as a direct transcriptional target of the CIC::DUX4 fusion protein and provide a mechanistic basis for the exceptional sensitivity of CDS to MCL1 inhibition.</p></section><section id="Sec7"><h3 class="pmc_sec_title">MCL1 inhibition slows CDS tumor growth in vivo</h3>
<p id="Par18">Since <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> is used in combination treatment approaches in ongoing clinical trials, we sought to identify potential drug synergies in CDS cells when combined with it. We performed a synergy screen by treating the cells with our drug library in presence or absence of 1 nM <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> as a backbone and calculated the IC50 for each drug under both conditions. This approach revealed that <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> exhibited the strongest synergy with inhibitors of other BCL2 family proteins, including A1331852, venetoclax, and navitoclax (Fig. <a href="#Fig6" class="usa-link">6a</a>). Additionally, we identified several BET bromodomain inhibitors and the PI3K inhibitor PQR514 among the top hits. We further characterized the combinatorial effect between <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> and venetoclax or PQR514 in additional CDS models and compared these effects to combinations of <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> with chemotherapeutic agents used in the first-line treatment of CDS and EwS tumors. These chemotherapeutics are part of the so-called VIDE regimen, which includes vincristine, ifosfamide, doxorubicin, and etoposide. Bliss, ZIP, and Loewe synergy score analyses showed no synergy between <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> and chemotherapeutic agents. In contrast, the combination of <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> with venetoclax demonstrated clear synergism in two of the four CDS models, while its combination with PQR514 showed synergism in one model (Fig. <a href="#Fig6" class="usa-link">6b–c</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">S12a–b</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Inhibition of MCL1 slows growth of CDS xenograft tumors in vivo.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370961_41467_2025_62629_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0ea1/12370961/d223d4886d09/41467_2025_62629_Fig6_HTML.jpg" loading="lazy" id="d33e1332" height="714" width="800" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Ratio of IC50 values for 244 drugs determined with CDS-ZH003 cells in presence or absence of 1 nM <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a>. Drug classes showing high cooperativity with <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> are highlighted in red (BCL2-family inhibitors), blue (bromodomain inhibitors), and pink (PI3K inhibitors). <em>n</em> = 2 independent experiments. <strong>b</strong> Combination matrix used to test the synergy between the MCL1 inhibitor <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> and the BCL2 inhibitor venetoclax. The relative number of viable CDS-ZH003 cells as determined by image-based analysis after 72 h of treatment is depicted. <em>n </em>= 2 independent experiments. <strong>c</strong> Bliss synergy scores for the indicated CDS models after treatment with <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> in combination with different chemotherapeutics, venetoclax or PQR514 for 72 h. Calculation was based on the number of viable cells, determined by image-based analysis. <em>n</em> = 2 independent experiments. <strong>d</strong> Growth curves of xenograft tumors in individual female mice transplanted with CDS-ZH001 cells and treated twice a week with either vehicle (black) or 20 mg/kg <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> (red) (upper panel). Black triangles indicate the time points of drug application. Survival curves calculated with the data from the upper panel (lower panel). Log-rank (Mantel-Cox) test. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par19">Finally, we investigated the impact of MCL1 inhibition on tumor growth in vivo. We selected the CDS-ZH001 model for this purpose, which exhibited the lowest IC50 for <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> in ex vivo experiments. Tumor-bearing mice were treated with 20 mg/kg <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> twice a week for three weeks. Several tumors in the treated group exhibited strong regression, and mouse survival was significantly prolonged compared to the control group (Fig. <a href="#Fig6" class="usa-link">6d</a>). Overall, these data confirm the exceptional MCL1 dependency of CDS tumors and highlight MCL1 as a therapeutic target in these tumors.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par20">Although most patients with EwS respond to standard treatment modalities, approximately one quarter remain uncured. In patients with metastatic or recurrent disease, the prognosis is markedly worse. The prognosis is even worse for CDS patients, with the majority of patients dying from the disease<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Therefore, novel therapeutic approaches to improve clinical outcomes are needed for all these patients. Personalized therapeutic strategies may help to achieve a better clinical response. Importantly, the rarity of clinically actionable driver mutations in childhood cancer in general is a major limitation in the genetic search for therapeutic targets. As an alternative, phenotypic profiling of short- to medium-term ex vivo cultures of patient-derived tumor cells has emerged as a promising tool for identifying the most effective drugs among large drug libraries<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. The short-term approach, originally developed for blood cancers, is now increasingly used for different types of solid tumors, including sarcoma<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. This approach requires sufficient cells from the patient tumor and is mainly suitable for tumor resections<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. In the case of biopsies, the number of cells is a limiting factor and ex vivo expansion is often necessary, which requires appropriate culture conditions<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>–<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. To this end, we developed a tumoroid culture system that enables efficient expansion of cells from EwS and CDS patient tumors. In a relevant number of cases, we were able to expand single or small clusters of cells isolated from biopsies into large tumoroids within a few weeks. At this stage, the number of available cells is sufficient to test a limited set of drugs, such as the chemotherapeutics used as first-line therapy. This timeframe may also be compatible with a co-clinical approach. Here, however, we expanded the cells for an additional one to two passages to obtain at least 10 million cells, a number required to test our drug library of 245 drugs. The average duration for this approach of 130 days is too long for a co-clinical application for therapy guidance in individual patients. However, this approach not only allowed us to test clinically approved compounds, but also to characterize the models in great detail and gain insights into unidentified relevant pathways. A beneficial byproduct of this procedure is the generation of novel models, which serve as valuable research tools for understudied tumors like CDS, facilitating future biological characterization. Still, in about half of the EwS tumors model generation failed. While the exact reason for this is unclear in individual cases, the number and viability of cells isolated from biopsy samples as well as cell-intrinsic (tumor aggressiveness) or extrinsic (tumor microenvironment) factors may play an important role. Therefore, obtaining additional biopsies could further optimize the success rate and accelerate model generation.</p>
<p id="Par21">A key finding of our study was the preservation of drug response characteristics throughout model establishment. There was no major difference between cells tested immediately after isolation and after model establishment. This is in line with published DRP data from a rhabdomyosarcoma case<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. The models also reproduced the difference in drug response between EwS and CDS tumors as observed in clinical practice<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. However, the ex vivo DRP technology used here also has its limitations. It does not consider the actual tumor environment or the patient’s immune system, both of which can have a major impact on treatment response. Furthermore, pharmacological differences between ex vivo cell cultures and in vivo patient physiology (pharmacokinetics, prodrug formulations and liver metabolism) introduce additional uncertainties that need to be considered. Prospective co-clinical studies are needed to test whether ex vivo response is predictive of in vivo response in patients. Encouragingly, in a recent prospective observational study of the DRP approach with a small set of different pediatric solid tumors, DRP-guided therapy led to significantly improved treatment responses compared to non-guided therapy<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>, mirroring findings previously reported in leukemia<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p>
<p id="Par22">Based on our differential drug sensitivity analysis, we identified an exceptional dependency of CDS cells on MCL1, suggesting that this protein could serve as a therapeutic target. While this dependency has not been described yet, other therapeutic vulnerabilities of CDS cells have been reported recently. This includes the sensitivity of CDS cells to AKT/mTOR-inhibitors and trabectedin<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. The combination of the PI3K/mTOR inhibitor dactolisib (NVP-BEZ235) with trabectedin has been shown to be effective not only against local tumor growth, but also against metastatic spread<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. We did not observe exceptional sensitivity to dactolisib either as single agent or in combination with <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> in our CDS models; however, we detected a synergy between the PI3K inhibitor PQR514 and <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> in one of the models. Furthermore, our comparative gene expression analysis also hinted towards an activation of the PI3K pathway in CDS cells. Other studies demonstrated that p300 may be a promising candidate target for CDS treatment, which is in line with the mechanism of action of CIC::DUX4 as p300 recruiter<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. In line with these studies, we observed a high sensitivity of one of the CDS models against A-485.</p>
<p id="Par23">The IC50 values of the MCL1 inhibitor <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a> in CDS were comparable to those found in hematological malignancies, which are currently the focus of MCL1-targeted therapy, and significantly lower than those observed in almost all tested solid tumor cell lines<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Several clinical studies testing the effect of different MCL1 inhibitors have been started a few years ago with some of them still ongoing<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Importantly, cardiac problems have been revealed as a major dose-limiting side effect in both a study with <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> in AML/MDS (ClinicalTrials.gov ID <a href="https://clinicaltrials.gov/ct2/show/NCT02979366" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT02979366</a>), as well as in a study with AMG397 in different hematological malignancies<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. The exact mechanism of MCL1 inhibitor mediated cardiotoxicity is not clear and reasons are probably multifactorial. However, conditional knockout studies of MCL1 in the mouse detected an important role for MCL1 in cardiomyocyte survival<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Therefore, the observed clinical side effects are likely to reflect this function and potentially limit the therapeutic window of MCL1 inhibitors. The MCL1 dependency of other cell types, including haematopoietic stem cells and lymphocytes, might lead to additional limitations for clinical use<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Combining MCL1 inhibitors with agents targeting other BCL2 family members may enable dose reduction and mitigate these issues. Indeed, in our drug combination screen we detected a strong synergy between MCL1 and BCL2 inhibitors. Further clinical development of such a combination appears feasible, since in a recent phase 1 study combining <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> and venetoclax (<a href="https://clinicaltrials.gov/ct2/show/NCT03672695" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT03672695</a>) no new toxicity signals were detected, even though the trial was stopped due to lack of clinical efficacy. New MCL1 inhibitors, like BRD-810, which has a short half-life and thereby undergoes rapid systemic clearance, are currently in development and may also mitigate cardiac side effects<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Alternatively, the development of a degrader variant of one of the available MCL1 inhibitors, analogous to the PROTAC variant of the BCL2 and BCL-XL dual inhibitor navitoclax, may allow for a further reduction in drug dose and help avoid on-target toxicities<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>.</p>
<p id="Par24">Overall, our study of patient-derived tumor models identified MCL1 as therapeutic target in CDS. Moreover, as a proof of concept, this study highlights the feasibility of DRP for individual sarcoma patients. Implementation of the pipeline into clinical studies is highly warranted to evaluate the correlation with clinical response in prospective manner.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Ethical considerations</h3>
<p id="Par25">Patients and/or their legal guardians provided written informed consent prior to trial entry, allowing the tumor material or blood to be bio-banked for further research and for the publication of their images. Use of the patient material for the experiments was approved by the ethics committee of the Kanton Zurich (BASEC number 2020-01609).</p></section><section id="Sec11"><h3 class="pmc_sec_title">Patient samples</h3>
<p id="Par26">Patients included both male and female individuals. All Patients, irrespective of age, that were diagnosed in one of the participating centers with a pathology-confirmed EwS or CDS and from which enough fresh biopsy material was available, were eligible. This included the primary diagnosis as well as progression or relapse.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Sample collection process</h3>
<p id="Par27">Fresh biopsy material was either used directly or frozen in either CryoStor10 freezing medium (STEMCELL Technologies, 7930) or DMEM (Sigma-Aldrich, D5671) supplemented with 10% FBS and 10% DMSO at −80 °C in a cell freezing container at the participating sites. The frozen tumors were then shipped to the laboratory in Zurich where the tumoroid cultures were established.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Tumoroid culture</h3>
<p id="Par28">Fresh or frozen biopsies from patient tumors were first washed with HBSS buffer (Sigma-Aldrich, H6648) and then cut into smaller fragments using two scalpels. For very small biopsies, the fragments were directly mixed with Matrigel and cultured as described below. Otherwise, the tumor material was transferred to a microcentrifuge tube and digested with 1 ml of digestion buffer containing 0.125 mg/ml Liberase DH (Roche, 5401054001) and 1 mM MgCl<sub>2</sub> in HBSS buffer for 2–4 h, until the tissue was completely dissolved. The digestion progress was monitored by counting cells in the supernatant at several time points. Cells were then pelleted by centrifugation for 2 min at 350 g. In case of the presence of red blood cells, these were lysed by incubation in 1 ml red blood cell lysis buffer for 10 min in the dark. Cells were then pelleted as before and washed once with HBSS buffer. The resulting cell pellet was suspended in 100–150 μl Matrigel (Corning, 354234) and the cell suspension was used to form 3D Matrigel domes in the center of a 6-well. After solidifying of the Matrigel, the domes were overlaid with 3 ml Adv.DMEM/F12 medium (Thermo Fisher Scientific, 12634028) supplemented with 100U/ml Penicillin/Streptomycin (Thermo Fisher Scientific, 15140122), 2 mM Glutamax (Thermo Fisher Scientific, 35050-061), 1xB27 (Thermo Fisher Scientific, 17504044), 1.25 mM N-acetylcysteine (Sigma-Aldrich, A9165), 5 μM A83-01 (Tocris Bioscience, 2939) and Matrigel (1:1000). The cells were then cultured at 37 °C and 5% CO<sub>2</sub> and the medium was exchanged three times a week. To prevent anoikis after tissue dissociation, 10 μM ROCK inhibitor Y-27632 (Focus Biomolecules, 10-2301) was added to the medium for the first two to three days of culture. The cells were passaged when larger tumoroids became visible. For this, the Matrigel domes were washed from the plate in 1 ml Cultrex organoid harvesting solution (R&amp;D Systems, 3700-100-01) and incubated on ice in a microcentrifuge tube until the Matrigel was completely dissolved. After pelleting, the organoids were dissociated with Accutase (Sigma-Aldrich, A6964) diluted 1:1 with PBS for 3-5 min at 37 °C. After washing once with PBS, cells were either filtered through a 70 μm cell strainer and used for the drug profiling or directly plated again either in Matrigel domes in 6 well plates (early passages) or on a thick layer of Matrigel in cell culture flasks (later passages). For passaging of tumoroids growing on a thick layer of Matrigel, the Matrigel was dissolved by incubation with Cultrex organoid harvesting solution at 4 °C under constant shaking for 30-60 min.</p>
<p id="Par29">For testing the effect of growth factors on tumoroid growth, the culture medium was supplemented with 20 ng/ml bFGF (Peprotech, #100-18B), EGF (Peprotech, AF100-15) or IGF-I LR3 (Peprotech, #100-11R3).</p></section><section id="Sec14"><h3 class="pmc_sec_title">Genomic analyses</h3>
<p id="Par30">The genomic data evaluated in this study was produced and kindly provided by the INFORM program<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>–<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">RNA-seq</h3>
<p id="Par31">The libraries of the extracted RNA samples were prepared using the TruSeq Stranded RNA Library Prep kit (Illumina). The libraries were sequenced on one NovaSeq 6 K PE 100 S1 flow cell as 2x650M reads per lane by the NGS Core Facility of DKFZ (Heidelberg, Germany).</p>
<p id="Par32">The analysis of the sequencing reads was then performed in Galaxy (<a href="https://usegalaxy.eu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://usegalaxy.eu/</a>) with the following tools. Illumina adapters were trimmed off using Cutadapt and quality control was performed with FastQC. RNA-seq reads were then aligned to the GRCh38 reference genome using RNA STAR. Read counts per gene were determined using featureCounts and normalization between samples was performed with DESeq2. Fusion genes were detected using Arriba. Relative gene expression analysis was performed with iDEP1.1<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Whole exome sequencing</h3>
<p id="Par33">The libraries of the extracted DNA samples were prepared using the Sure Select XT HS Agilent + Human All Exon V7 protocol by the NGS Core Facility of DKFZ. The libraries were sequenced on one NovaSeq 6k PE 100 S2 flow cell as 1 × 1650 M reads per lane by the NGS Core Facility of DKFZ (Heidelberg, Germany). Analysis of the sequencing reads was then performed in Galaxy (<a href="https://usegalaxy.eu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://usegalaxy.eu/</a>) with the following tools. Trimming and filtering of reads was done with Trimmomatic. The sequencing reads were then aligned to the hg19 reference genome using BWA-MEM. Reads were filtered with the Filter BAM tool using the following parameters: mapquality ≥1, ismapped =yes, ismatemapped =yes. Duplicate reads were removed with RmDup. Positional distribution of insertions and deletions in the input was homogenized by using left realignment with BamLeftAlign with maximum number of iterations=5. Read mapping quality was recalibrated with CalMD with a coefficient to cap mapping quality of poorly mapped read=50. Reads were refiltered based on mapping quality using Filter BAM datasets with mapquality ≤254. Somatic variants were identified using VarScan somatic with a minimum base quality=28 and a minimum mapping quality=1. Variants were annotated with SnpEf eff.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Methylation profiling</h3>
<p id="Par34">The extracted DNA samples were prepared and analyzed by Infinium MethylationEPIC BeadChip Array by the Microarray Core Facility of the DKFZ.</p>
<p id="Par35">Raw methylation array data from 1077 sarcoma samples spanning 65 methylation classes was obtained from the public repository Gene Expression Omnibus (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140686" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE140686</a>) and processed alongside methylation data from 9 of our patient-derived tumoroid models as previously described<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. For dimensionality reduction analysis, the 10,000 most variable CpG probes based on their standard deviation across all samples were selected. These batch corrected and normalized methylation beta values were then used to embed the high-dimensional methylation patterns using t-distributed Stochastic Neighbor Embedding (t-SNE), as implemented in the Rtsne R package (v0.17), with 3000 iterations and perplexity set at 30.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Identification of <em>CIC::DUX4</em> translocation breakpoints</h3>
<p id="Par36"><em>CIC::DUX4</em> fusions were detected with the Archer FusionPlex assay using the sarcoma v2 panel.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Preparation of the drug library for the drug response profiling</h3>
<p id="Par37">The 245 drugs of the library were purchased either pre-dissolved in DMSO as a 10 mM solution or as powder (Selleckchem). 5 mM and 0.5 mM stock solutions were then prepared in DMSO (238 drugs) or water (7 drugs) and transferred to 384-well source plates (Labcyte; PPL-0200) qualified for an Echo dispenser (Labcyte). The Echo dispenser was used to dispense the drugs into 384-well cell culture plates (Greiner, 7.781 098). Four wells were used per drug and filled with either 5 nl of drug solution from the 0.5 mM source plate or 5 nl, 50 nl, or 500 nl of drug solution from the 5 mM source plate, corresponding to final drug concentrations of 10 nM, 100 nM, 1,000 nM, and 10,000 nM, respectively. DMSO volumes were normalized using the Echo dispenser. The pre-drugged plates were sealed with an adhesive aluminum cover (Sigma-Aldrich, Z721530) and stored at -80 °C for up to several months until use. On the day of use, the plates were equilibrated to room temperature, centrifuged, and filled with 25 μl prewarmed culture medium per well using a Mantis liquid handler (Formulatrix). The drug plates were then incubated at room temperature overnight under constant shaking on an orbital shaker protected from light. The next day, 2 μl of each drug solution was transferred to the plate containing the cells in 18 μl medium using a 24-channel pipette. Screening parameters are summarized in Supplementary Table <a href="#MOESM1" class="usa-link">6</a>.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Drug response profiling</h3>
<p id="Par38">For drug response profiling, 10,000–20,000 cells in a volume of 20 μl were plated per 384-well coated with Matrigel. For the complete library, a total of 1100 wells were used. The next day, the medium was changed and the drugs were added to the cells and incubated for a total of 72 h. Cell viability was then determined with the CellTiter Glo 3D assay (Promega, G9682) according to the instructions of the manufacturer. To assess cell death, cells were stained with 10 μM Hoechst 33342 and 1 μg/ml propidium iodide for 15 min, followed by image-based analysis with an Operetta CLS high-content analysis system. Live and dead cells were quantified using the Harmony software.</p>
<p id="Par39">AUC and IC50 values were calculated by nonlinear regression using GraphPad Prism 8.0.0. dDSS values were calculated using published procedures<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Flow cytometry</h3>
<p id="Par40">For analysis of cell death, cells were grown and treated in 6-well plates. After treatment, remaining adherent cells were detached with Accutase (diluted 1:2 with PBS) (Sigma-Aldrich, A6964), and combined with floating cells in a centrifuge tube. After washing twice with PBS, single cells were resuspended in 100 μl Annexin V binding buffer (BD Biosciences, #556454) and stained with 5 μl Annexin V-APC (BD Biosciences, #550474) and 5 μl 7-AAD (BD Biosciences #559925) for 15 min at room temperature in the dark. After dilution with an additional 400 μl binding buffer, cells were filtered through a cell strainer into a FACS tube and analyzed on a Fortessa flow cytometer (BD Biosciences).</p>
<p id="Par41">For STAG2-intracellular staining, cells were grown in T25 flasks and detached using Accutase (Sigma-Aldrich, A6964), 1:2 diluted with PBS. After washing twice with PBS, cells were fixed in PBS/1% PFA (Thermo Fischer Scientific, 28906)/0.5% BSA (Sigma-Aldrich, A2153) for 15 minutes on ice. Cells were washed twice with PBS/0.5% BSA and stored in PBS/0.5% BSA at 4 °C for up to one week before staining. Cells were permeabilized with PBS/0.5% Tween-20 (Sigma-Aldrich, P9416)/0.5% BSA (permeabilization buffer), and blocked in PBS/0.5%Tween-20/3% BSA for 1 h at RT. Cells were stained with a primary antibody against STAG2 (Santa Cruz, sc-81852) diluted 1:500 in permeabilization buffer, for 2 h at RT. Cells were washed twice with permeabilization buffer and incubated with Alexa Fluor™ 647-conjugated secondary antibody (Thermo Fischer Scientific, A-21235) diluted 1:400 in permeabilization buffer, for 1 h at RT. After two washes with permeabilization buffer, cells were resuspended in PBS/0.5% BSA and analyzed on a Fortessa flow cytometer (BD Biosciences).</p></section><section id="Sec22"><h3 class="pmc_sec_title">Western Blot</h3>
<p id="Par42">Cell lysates for Western blots were generated using RIPA buffer (Thermo Fisher Scientific, #89900), supplemented with Complete Mini Protease Inhibitor cocktail (Sigma-Aldrich, #11697498001). Proteins were separated using NuPAGE™ Novex™ 4–12% Bis-Tris gels (Thermo Fisher Scientific) and transferred to nitrocellulose membranes (GE Healthcare Life Sciences) by wet blotting. Membranes were then blocked with 5% milk in TBS/0.05% Tween-20 for 20 min, followed by incubation with the primary antibody diluted at 1:1000 in blocking buffer overnight at 4 °C. After three washing steps with TBS/0.05% Tween-20 for 5 min, membranes were incubated with a horseradish peroxidase-linked secondary antibody (Cell Signaling, #7074) diluted 1:2000 in blocking buffer for 1 h at RT. After three additional washing steps with TBS/0.05% Tween-20 for 5 min and one final wash step with TBS for 1 min, proteins were detected by chemiluminescence using either the Pierce™ ECL or the SuperSignal Western blotting reagent (both from Thermo Fisher Scientific) and a ChemiDoc<sup>TM</sup> MP imaging system (BioRad). Antibodies used included the following ones from Cell Signaling: anti-MCL1 (#4572S), anti-BCL-XL (#2764S), anti-PARP (#9542S), anti-β-Actin (#4970) and anti-GAPDH (#2118). Antibody against BCL2 was purchased from Abcam (ab32124) and against CIC from Novus Biologicals (#NB110-59905SS).</p></section><section id="Sec23"><h3 class="pmc_sec_title">CRISPR-mediated gene knockout</h3>
<p id="Par43">Gene-specific sgRNAs were cloned into LentiCRISPR-mCherry vectors (Addgene #75161). The following sgRNAs were used: sgAAVS1 (negative control; GGGGCCACTAGGGACAGGAT), sgMCL1-2 (CGAGTTGTACCGGCAGTCGC) and sgMCL1-3 (CTGGAGACCTTACGACGGGT). sgRNA constructs were transfected into Lenti-X<sup>TM</sup> cells (Takara Bio) together with pMD2.G (Addgene #12259) and psPAX2 (Addgene #12260) packaging vectors. The next day, the medium was changed. Sixty hours after transfection, supernatants containing lentiviral particles were harvested and concentrated using Amicon Ultra columns (Merck #UFC910008) with a cut-off of 100 kDa. For transduction of CDS target cells, viral concentrates were diluted 1:10–1:20 in medium and added to the target cells grown in 24- or 6-well plates. Infection rate was enhanced by spinoculation with 1500 g for 90 min at 32 °C. After overnight incubation, the medium was changed.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Competition assay</h3>
<p id="Par44">CDS cells were transduced with mCherry-expressing LentiCRISPR sgRNA constructs. Three days after transduction, transduced cells were mixed with untransduced cells as a reference and the fraction of mCherry-positive to mCherry-negative cells was determined by flow cytometry on a Fortessa flow cytometer (BD Biosciences). Seven days later, the fraction was again determined, and the ratio of mCherry-positive fractions between day 2 and day 7 was calculated.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Quantitative PCR</h3>
<p id="Par45">Total RNA was isolated from cultured cells with the RNeasy Plus Mini Kit (Qiagen). cDNA from 1 μg total RNA was generated using the High Capacity cDNA Reverse Transcription Kit (Thermo Fisher Scientific). mRNA levels of genes of interest were then determined with specific TaqMan<sup>TM</sup> assays including assays for <em>MCL1</em> (Hs06626047_g1), <em>BCL2</em> (Hs04986394_s1), <em>ETV4</em> (Hs00383361_g1), <em>GAPDH</em> (Hs02758991_g1) (all from Thermo Fisher Scientific), <em>B2M</em> (Hs.PT.58 v.18759587) and <em>ANXA5</em> (Hs.PT.56a.38876508) (both from Integrated DNA Technologies) and the TaqMan<sup>TM</sup> Gene Expression Master Mix (Thermo Fisher Scientific). Relative gene expression levels were calculated using the ΔΔCT method with <em>GAPDH</em>, <em>B2M</em> or <em>ANXA5</em> as reference gene.</p></section><section id="Sec26"><h3 class="pmc_sec_title">CRISPR interference</h3>
<p id="Par46">For CRISPR interference, dCas9-KRAB was cloned into a lentiviral expression plasmid (Addgene #134966), replacing wildtype Cas9. The plasmid also contains GFP as a selection marker. CDS-ZH003 cells were transduced with the construct and sorted for uniform GFP expression. The selected cells were then transduced with different sgRNA expression constructs (sg_shuttle_RFP657, Addgene #134968). For interference with the CIC::DUX4 binding sites near <em>MCL1</em>, the following sgRNAs were used: sgPeak1-1 (GAGTCATAACCAGCCCAGTA), sgPeak1-2 (CACTGGCAGTAGACTTATAC) (both directed against the MCL1 downstream enhancer) and sgPeak2-1 (TTCAGTTGAATACTCTTCAG), sgPeak2-2 (CCATGAAAAAATAAGTCACC), and sgPeak3-1 (GAACAACAGTCTTAGATGAT) (all directed against the genic CIC::DUX4 binding site). Two to three days after transduction, RNA was isolated and <em>MCL1</em> mRNA levels were measured by qRT-PCR.</p>
<p id="Par47">To evaluate <em>CIC</em> transcriptional start sites active in CDS-ZH003 cells, the following sgRNAs were used: sgCIC-1 (TGGCAGCGGTAGCGGCACGA), sgCIC-2 (AATCGAGAGGGAGAGCCGGA), and sgCIC-3 (CCTGCCTCCCGCCGCCCGGG). Two to three days after transduction, protein lysates were generated and CIC::DUX4 and CIC protein levels were evaluated by Western blot.</p></section><section id="Sec27"><h3 class="pmc_sec_title">shRNA-mediated CIC::DUX4 silencing</h3>
<p id="Par48">Three different shRNAs targeting sequences in the C-terminal region of <em>DUX4</em> were tested for silencing of CIC::DUX4. The following shRNAs were used (21-nt targeting sequence): shDUX4-1 (AGGCGCAACCTCTCCTAGAAA), shDUX4-2 (GGCTCTGCTGGAGGAGCTTTA) and shDUX4-3 (CAACCTCTCCTAGAAACGGAG). A non-targeting sequence was used as negative control (CAACAAGATGAAGAGCACCAA). The shRNA sequences were cloned into the pRSIT12 vector under a doxycycline-inducible promoter (Cellecta). CDS-ZH003 cells were transduced with the constructs and shRNA expression was induced by treating the cells with 1 μg/ml doxycycline. The culture medium was changed every other day.</p></section><section id="Sec28"><h3 class="pmc_sec_title">ChIP-seq analysis</h3>
<p id="Par49">CIC::DUX4 binding sites in the vicinity of <em>MCL1</em> were analyzed in the Integrative Genomics Viewer (IGV) using published ChIP-seq data downloaded from the Gene Expression Omnibus (GEO) database under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE248117" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE248117</a><sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>.</p></section><section id="Sec29"><h3 class="pmc_sec_title">In vivo tumor treatment</h3>
<p id="Par50">NSG mice (NOD.Cg-Prkdc&lt; scid&gt; Il2rg&lt;tm1Wjl &gt; /SzJ; Charles River F, L’Arbresle, France) were bred in-house. Health screens were conducted in accordance with FELASA guidelines to confirm their pathogen-free status once per quartal. Animals were housed in groups of 4-6 mice per individually ventilated cage in 12 h light/dark cycle, with controlled room temperature (21 °C) and relative humidity (40-60%). The in vivo study exclusively examined female. To establish xenografts, 5 million CDS-ZH001 cells were injected s.c. into the flank of 8-10 weeks old mice. Tumor-bearing mice were randomized into treatment and control cohorts of 6 animals when the average tumor size reached ~100 mm<sup>3</sup>. <a href="https://www.ncbi.nlm.nih.gov/nuccore/S64315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S64315</a> (MIK665; Medchemexpress) was dissolved in 20% 2-Hydroxypropyl-beta-cyclodextrin (Sigma-Aldrich), 25 mM HCl<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Drug and vehicle were administered by i.v. injection twice a week. Tumor size was measured three times a week with a caliper, and mouse weight was measured twice a week. No mice had to be euthanized due to severe weight loss ( &gt; 15% of baseline). All mice were euthanized when tumor volumes exceeded 1000 mm³. All animal experiments were approved by the veterinary service of the canton of Zurich and were performed according to the animal license ZH013/2021.</p></section><section id="Sec30"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par51">Further information on research design is available in the <a href="#MOESM6" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec31"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62629_MOESM2_ESM.pdf</a><sup> (27.5KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (297KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (763.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (28.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (105.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (559.5KB, pdf) </sup>
</div></div></section></section><section id="Sec32"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (2.6MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the NGS Core Facility at the German Cancer Research Center (DKFZ) for providing excellent RNA-seq, WES and methylation profiling services. We would also like to express our sincere thanks to Carsten Maus and Erjia Wang (Genomics and Proteomics Core Facility, DKFZ), Lena Weiser and Gregor Warsow (Omics IT and Data Management Core Facility, DKFZ) for their highly dedicated support in data management and processing and Robert Autry, Gnanaprakash Balasubramanian, Christopher Previti, and Rolf Kabbe (Division of Pediatric Neurooncology, DKFZ) for their sincere and dedicated contribution to the bioinformatics analyses. Finally, we thank Rishi Puram and Todd Golub (Broad Institute of MIT and Harvard) for providing BRD-810. The work was supported by grants from the Swiss National Science Foundation (3100-175558 to BWS and 10000473 to DS), the Cancer League Switzerland (KLS-5143-08-2020 to BWS and KFS-5422-08-2021-R to DS), the Childhood Research Foundation Switzerland (to BWS), the Berner Stiftung für krebskranke Kinder und Jugendliche (to JR), the European Union’s Horizon 2020 research and innovation program under the Marie Skłodowska–Curie grant agreement No. 956285 (VAGABOND) (to BWS), and a clinical research focus grant from the University of Zurich (to BWS, DS, and JPB). The laboratory of TGPG is supported by the Barbara and Wilfried Mohr Foundation and co-funded by the European Union (ERC, CANCER-HARAKIRI, 101122595). Views and opinions expressed are, however, those of the authors only and do not necessarily reflect those of the European Union or the European Research Council. Neither the European Union nor the granting authority can be held responsible for them. ACE was supported by a scholarship from the German Cancer Aid and the German Academic Scholarship Foundation. The INFORM program is financially supported by the German Cancer Research Center (DKFZ), several German health insurance companies, the German Cancer Consortium (DKTK), the German Federal Ministry of Education and Research (BMBF), the German Federal Ministry of Health (BMG), the Ministry of Science, Research and the Arts of the State of Baden-Württemberg (MWK BW); the German Cancer Aid (DKH), the German Childhood Cancer Foundation (DKS), RTL television, the aid organization BILD hilft e.V. (Ein Herz für Kinder) and the generous private donation of the Scheu family.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: W.B., E.B., B.W.S., T.G.P.G., D.S., M.W. Methodology: M.W. Formal analysis: A.E., M.W. Investigation: F.Z., I.B., S.K., J.W., L.M., D.Z., M.W. Resources: W.B., E.B., J.R., C.B., L.B., C.R., D.M., J.P.B., F.S., S.B., C.P., P.B., B.R., C.V. Writing-original draft: W.B., E.B., I.B., T.G., M.W. Writing-review and editing: W.B., E.B., D.S., T.G.P.G., B.W.S., M.W. Visualization: A.C.E., M.W. Supervision: T.G.P.G., D.S., B.W.S., M.W. Project administration: T.G.P.G., E.B., J.R., D.S., B.W.S. Funding acquisition: W.B., E.B., J.R., T.G.P.G., D.S., B.W.S. The order of the first two authors’ names was determined by a joint decision of both authors and M.W.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par52"><em>Nature Communications</em> thanks Joseph Ludwig, and the other, anonymous, reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>WES, RNA-seq and DNA methylation data from tumoroid models is deposited in the European Genome-Phenome Archive under the study ID <a href="https://ega-archive.org/studies/EGAS00001008039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EGAS00001008039</a>. Due to patient data protection, the data are available upon request to the data access committee (DAC). The links to the DAC can be found under the dataset IDs EGAD00001015608 (WES and RNA-seq; <a href="https://ega-archive.org/datasets/EGAD00001015608" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ega-archive.org/datasets/EGAD00001015608</a>) and EGAD00010002760 (DNA methylation data; <a href="https://ega-archive.org/datasets/EGAD00010002760" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ega-archive.org/datasets/EGAD00010002760</a>). The data will be made accessible to all interested researchers, with no limitations regarding the time or purpose of data use. Previously published ChIP-seq data<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> was downloaded from the Gene Expression Omnibus (GEO) database under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE248117" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE248117</a>. The remaining data are available within the article, supplementary information or source data file. <a href="#Sec32" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par53">The authors declare no conflict of interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Willemijn Breunis, Eva Brack.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Beat W. Schäfer, Email: beat.schafer@kispi.uzh.ch</p>
<p>Didier Surdez, Email: didier.surdez@balgrist.ch.</p>
<p>Marco Wachtel, Email: marco.wachtel@kispi.uzh.ch.</p></section><section id="sec33"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62629-6.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Riggi, N., Suva, M. L. &amp; Stamenkovic, I. Ewing’s Sarcoma. <em>N. Engl. J. Med</em>. <strong>384</strong>, 154–164 (2021).
</cite> [<a href="https://doi.org/10.1056/NEJMra2028910" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33497548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Riggi,%20N.,%20Suva,%20M.%20L.%20&amp;%20Stamenkovic,%20I.%20Ewing%E2%80%99s%20Sarcoma.%20N.%20Engl.%20J.%20Med.%20384,%20154%E2%80%93164%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Zollner S. K. et al. Ewing Sarcoma-Diagnosis, Treatment, Clinical Challenges and Future Perspectives. <em>J. Clin. Med</em>. <strong>10</strong>, 1685 (2021).</cite> [<a href="https://doi.org/10.3390/jcm10081685" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8071040/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33919988/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Grunewald, T. G. P. et al. Ewing sarcoma. <em>Nat. Rev. Dis. Prim.</em><strong>4</strong>, 5 (2018).
</cite> [<a href="https://doi.org/10.1038/s41572-018-0003-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29977059/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grunewald,%20T.%20G.%20P.%20et%20al.%20Ewing%20sarcoma.%20Nat.%20Rev.%20Dis.%20Prim.4,%205%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Delattre, O. et al. The Ewing family of tumors-a subgroup of small-round-cell tumors defined by specific chimeric transcripts. <em>N. Engl. J. Med</em>. <strong>331</strong>, 294–299 (1994).
</cite> [<a href="https://doi.org/10.1056/NEJM199408043310503" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8022439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Delattre,%20O.%20et%20al.%20The%20Ewing%20family%20of%20tumors-a%20subgroup%20of%20small-round-cell%20tumors%20defined%20by%20specific%20chimeric%20transcripts.%20N.%20Engl.%20J.%20Med.%20331,%20294%E2%80%93299%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Desmaze, C. et al. Multiple chromosomal mechanisms generate an EWS/FLI1 or an EWS/ERG fusion gene in Ewing tumors. <em>Cancer Genet Cytogenet</em><strong>97</strong>, 12–19 (1997).
</cite> [<a href="https://doi.org/10.1016/s0165-4608(96)00326-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9242212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Desmaze,%20C.%20et%20al.%20Multiple%20chromosomal%20mechanisms%20generate%20an%20EWS/FLI1%20or%20an%20EWS/ERG%20fusion%20gene%20in%20Ewing%20tumors.%20Cancer%20Genet%20Cytogenet97,%2012%E2%80%9319%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Mitelman, F., Johansson, B. &amp; Mertens, F. The impact of translocations and gene fusions on cancer causation. <em>Nat. Rev. Cancer</em><strong>7</strong>, 233–245 (2007).
</cite> [<a href="https://doi.org/10.1038/nrc2091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17361217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mitelman,%20F.,%20Johansson,%20B.%20&amp;%20Mertens,%20F.%20The%20impact%20of%20translocations%20and%20gene%20fusions%20on%20cancer%20causation.%20Nat.%20Rev.%20Cancer7,%20233%E2%80%93245%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Choi, J. H. &amp; Ro, J. Y. The 2020 WHO classification of tumors of bone: an updated review. <em>Adv. Anat. Pathol.</em><strong>28</strong>, 119–138 (2021).
</cite> [<a href="https://doi.org/10.1097/PAP.0000000000000293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33480599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choi,%20J.%20H.%20&amp;%20Ro,%20J.%20Y.%20The%202020%20WHO%20classification%20of%20tumors%20of%20bone:%20an%20updated%20review.%20Adv.%20Anat.%20Pathol.28,%20119%E2%80%93138%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Cidre-Aranaz, F. et al. Small round cell sarcomas. <em>Nat. Rev. Dis. Prim.</em><strong>8</strong>, 66 (2022).
</cite> [<a href="https://doi.org/10.1038/s41572-022-00393-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36202860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cidre-Aranaz,%20F.%20et%20al.%20Small%20round%20cell%20sarcomas.%20Nat.%20Rev.%20Dis.%20Prim.8,%2066%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Brahmi, M., Vanacker, H., Macagno, N., Tirode, F. &amp; Dufresne, A. CIC-DUX4 sarcomas. <em>Curr. Opin. Oncol.</em><strong>34</strong>, 342–347 (2022).
</cite> [<a href="https://doi.org/10.1097/CCO.0000000000000855" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35730520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brahmi,%20M.,%20Vanacker,%20H.,%20Macagno,%20N.,%20Tirode,%20F.%20&amp;%20Dufresne,%20A.%20CIC-DUX4%20sarcomas.%20Curr.%20Opin.%20Oncol.34,%20342%E2%80%93347%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Antonescu, C. R. et al. Sarcomas With CIC-rearrangements are a distinct pathologic entity with aggressive outcome: a clinicopathologic and molecular study of 115 cases. <em>Am. J. Surg. Pathol.</em><strong>41</strong>, 941–949 (2017).
</cite> [<a href="https://doi.org/10.1097/PAS.0000000000000846" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5468475/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28346326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Antonescu,%20C.%20R.%20et%20al.%20Sarcomas%20With%20CIC-rearrangements%20are%20a%20distinct%20pathologic%20entity%20with%20aggressive%20outcome:%20a%20clinicopathologic%20and%20molecular%20study%20of%20115%20cases.%20Am.%20J.%20Surg.%20Pathol.41,%20941%E2%80%93949%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Brahmi, M. et al. Patterns of care and outcome of CIC-rearranged sarcoma patients: a nationwide study of the French sarcoma group. <em>Cancer Med</em>. <strong>12</strong>, 7801–7807 (2023).
</cite> [<a href="https://doi.org/10.1002/cam4.5539" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10134374/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36537582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brahmi,%20M.%20et%20al.%20Patterns%20of%20care%20and%20outcome%20of%20CIC-rearranged%20sarcoma%20patients:%20a%20nationwide%20study%20of%20the%20French%20sarcoma%20group.%20Cancer%20Med.%2012,%207801%E2%80%937807%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Brennan, B. et al. Comparison of two chemotherapy regimens in patients with newly diagnosed Ewing sarcoma (EE2012): an open-label, randomised, phase 3 trial. <em>Lancet</em><strong>400</strong>, 1513–1521 (2022).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(22)01790-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36522207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brennan,%20B.%20et%20al.%20Comparison%20of%20two%20chemotherapy%20regimens%20in%20patients%20with%20newly%20diagnosed%20Ewing%20sarcoma%20(EE2012):%20an%20open-label,%20randomised,%20phase%203%20trial.%20Lancet400,%201513%E2%80%931521%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Connolly, E. A. et al. Systemic treatments and outcomes in CIC-rearranged Sarcoma: a national multi-centre clinicopathological series and literature review. <em>Cancer Med</em>. <strong>11</strong>, 1805–1816 (2022).
</cite> [<a href="https://doi.org/10.1002/cam4.4580" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9041083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35178869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Connolly,%20E.%20A.%20et%20al.%20Systemic%20treatments%20and%20outcomes%20in%20CIC-rearranged%20Sarcoma:%20a%20national%20multi-centre%20clinicopathological%20series%20and%20literature%20review.%20Cancer%20Med.%2011,%201805%E2%80%931816%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Palmerini, E. et al. A global collaboRAtive study of CIC-rearranged, BCOR::CCNB3-rearranged and other ultra-rare unclassified undifferentiated small round cell sarcomas (GRACefUl). <em>Eur. J. Cancer</em><strong>183</strong>, 11–23 (2023).
</cite> [<a href="https://doi.org/10.1016/j.ejca.2023.01.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36791667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palmerini,%20E.%20et%20al.%20A%20global%20collaboRAtive%20study%20of%20CIC-rearranged,%20BCOR::CCNB3-rearranged%20and%20other%20ultra-rare%20unclassified%20undifferentiated%20small%20round%20cell%20sarcomas%20(GRACefUl).%20Eur.%20J.%20Cancer183,%2011%E2%80%9323%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Sparber-Sauer, M. et al. Clinical characteristics and outcomes for children, adolescents and young adults with “CIC-fused” or “BCOR-rearranged” soft tissue sarcomas: A multi-institutional European retrospective analysis. <em>Cancer Med</em>. <strong>12</strong>, 14346–14359 (2023).
</cite> [<a href="https://doi.org/10.1002/cam4.6113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10358194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37212486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sparber-Sauer,%20M.%20et%20al.%20Clinical%20characteristics%20and%20outcomes%20for%20children,%20adolescents%20and%20young%20adults%20with%20%E2%80%9CCIC-fused%E2%80%9D%20or%20%E2%80%9CBCOR-rearranged%E2%80%9D%20soft%20tissue%20sarcomas:%20A%20multi-institutional%20European%20retrospective%20analysis.%20Cancer%20Med.%2012,%2014346%E2%80%9314359%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>van Tilburg, C. M. et al. The pediatric precision oncology INFORM registry: clinical outcome and benefit for patients with very high-evidence targets. <em>Cancer Discov.</em><strong>11</strong>, 2764–2779 (2021).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-21-0094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9414287/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34373263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20Tilburg,%20C.%20M.%20et%20al.%20The%20pediatric%20precision%20oncology%20INFORM%20registry:%20clinical%20outcome%20and%20benefit%20for%20patients%20with%20very%20high-evidence%20targets.%20Cancer%20Discov.11,%202764%E2%80%932779%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Kornauth, C. et al. Functional precision medicine provides clinical benefit in advanced aggressive hematological cancers and identifies exceptional responders. <em>Cancer Discov</em>. <strong>12</strong>, 372–387 (2022).</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-21-0538" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9762339/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34635570/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Dietrich, S. et al. Drug-perturbation-based stratification of blood cancer. <em>J. Clin. Invest</em>. <strong>128</strong>, 427–445 (2018).
</cite> [<a href="https://doi.org/10.1172/JCI93801" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5749541/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29227286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dietrich,%20S.%20et%20al.%20Drug-perturbation-based%20stratification%20of%20blood%20cancer.%20J.%20Clin.%20Invest.%20128,%20427%E2%80%93445%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Pemovska, T. et al. Individualized systems medicine strategy to tailor treatments for patients with chemorefractory acute myeloid leukemia. <em>Cancer Discov.</em><strong>3</strong>, 1416–1429 (2013).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-13-0350" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24056683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pemovska,%20T.%20et%20al.%20Individualized%20systems%20medicine%20strategy%20to%20tailor%20treatments%20for%20patients%20with%20chemorefractory%20acute%20myeloid%20leukemia.%20Cancer%20Discov.3,%201416%E2%80%931429%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Snijder, B. et al. Image-based ex-vivo drug screening for patients with aggressive haematological malignancies: interim results from a single-arm, open-label, pilot study. <em>Lancet Haematol.</em><strong>4</strong>, e595–e606 (2017).
</cite> [<a href="https://doi.org/10.1016/S2352-3026(17)30208-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5719985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29153976/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Snijder,%20B.%20et%20al.%20Image-based%20ex-vivo%20drug%20screening%20for%20patients%20with%20aggressive%20haematological%20malignancies:%20interim%20results%20from%20a%20single-arm,%20open-label,%20pilot%20study.%20Lancet%20Haematol.4,%20e595%E2%80%93e606%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Tuveson, D. &amp; Clevers, H. Cancer modeling meets human organoid technology. <em>Science</em><strong>364</strong>, 952–955 (2019).
</cite> [<a href="https://doi.org/10.1126/science.aaw6985" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31171691/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tuveson,%20D.%20&amp;%20Clevers,%20H.%20Cancer%20modeling%20meets%20human%20organoid%20technology.%20Science364,%20952%E2%80%93955%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Sato, T. et al. Long-term expansion of epithelial organoids from human colon, adenoma, adenocarcinoma, and Barrett’s epithelium. <em>Gastroenterology</em><strong>141</strong>, 1762–1772 (2011).
</cite> [<a href="https://doi.org/10.1053/j.gastro.2011.07.050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21889923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sato,%20T.%20et%20al.%20Long-term%20expansion%20of%20epithelial%20organoids%20from%20human%20colon,%20adenoma,%20adenocarcinoma,%20and%20Barrett%E2%80%99s%20epithelium.%20Gastroenterology141,%201762%E2%80%931772%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Al Shihabi, A. et al. The landscape of drug sensitivity and resistance in sarcoma. <em>Cell Stem Cell</em><strong>31</strong>, 1524–1542.e1524 (2024).
</cite> [<a href="https://doi.org/10.1016/j.stem.2024.08.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12318355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39305899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Al%20Shihabi,%20A.%20et%20al.%20The%20landscape%20of%20drug%20sensitivity%20and%20resistance%20in%20sarcoma.%20Cell%20Stem%20Cell31,%201524%E2%80%931542.e1524%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Calandrini, C. et al. Organoid-based drug screening reveals neddylation as therapeutic target for malignant rhabdoid tumors. <em>Cell Rep.</em><strong>36</strong>, 109568 (2021).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2021.109568" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34433038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Calandrini,%20C.%20et%20al.%20Organoid-based%20drug%20screening%20reveals%20neddylation%20as%20therapeutic%20target%20for%20malignant%20rhabdoid%20tumors.%20Cell%20Rep.36,%20109568%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Manzella, G. et al. Phenotypic profiling with a living biobank of primary rhabdomyosarcoma unravels disease heterogeneity and AKT sensitivity. <em>Nat. Commun.</em><strong>11</strong>, 4629 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-18388-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7492191/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32934208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manzella,%20G.%20et%20al.%20Phenotypic%20profiling%20with%20a%20living%20biobank%20of%20primary%20rhabdomyosarcoma%20unravels%20disease%20heterogeneity%20and%20AKT%20sensitivity.%20Nat.%20Commun.11,%204629%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Meister, M. T. et al. Mesenchymal tumor organoid models recapitulate rhabdomyosarcoma subtypes. <em>EMBO Mol. Med</em>. <strong>14</strong>, e16001 (2022).
</cite> [<a href="https://doi.org/10.15252/emmm.202216001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9549731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35916583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meister,%20M.%20T.%20et%20al.%20Mesenchymal%20tumor%20organoid%20models%20recapitulate%20rhabdomyosarcoma%20subtypes.%20EMBO%20Mol.%20Med.%2014,%20e16001%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Sato, T. et al. Single Lgr5 stem cells build crypt-villus structures in vitro without a mesenchymal niche. <em>Nature</em><strong>459</strong>, 262–265 (2009).
</cite> [<a href="https://doi.org/10.1038/nature07935" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19329995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sato,%20T.%20et%20al.%20Single%20Lgr5%20stem%20cells%20build%20crypt-villus%20structures%20in%20vitro%20without%20a%20mesenchymal%20niche.%20Nature459,%20262%E2%80%93265%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Schweigerer, L., Neufeld, G. &amp; Gospodarowicz, D. Basic fibroblast growth factor as a growth inhibitor for cultured human tumor cells. <em>J. Clin. Invest</em><strong>80</strong>, 1516–1520 (1987).
</cite> [<a href="https://doi.org/10.1172/JCI113236" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC442414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2824563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schweigerer,%20L.,%20Neufeld,%20G.%20&amp;%20Gospodarowicz,%20D.%20Basic%20fibroblast%20growth%20factor%20as%20a%20growth%20inhibitor%20for%20cultured%20human%20tumor%20cells.%20J.%20Clin.%20Invest80,%201516%E2%80%931520%20(1987)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Sturla, L. M., Westwood, G., Selby, P. J., Lewis, I. J. &amp; Burchill, S. A. Induction of cell death by basic fibroblast growth factor in Ewing’s sarcoma. <em>Cancer Res</em>. <strong>60</strong>, 6160–6170 (2000).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/11085540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sturla,%20L.%20M.,%20Westwood,%20G.,%20Selby,%20P.%20J.,%20Lewis,%20I.%20J.%20&amp;%20Burchill,%20S.%20A.%20Induction%20of%20cell%20death%20by%20basic%20fibroblast%20growth%20factor%20in%20Ewing%E2%80%99s%20sarcoma.%20Cancer%20Res.%2060,%206160%E2%80%936170%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Koelsche, C. et al. Sarcoma classification by DNA methylation profiling. <em>Nat. Commun.</em><strong>12</strong>, 498 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-020-20603-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7819999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33479225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koelsche,%20C.%20et%20al.%20Sarcoma%20classification%20by%20DNA%20methylation%20profiling.%20Nat.%20Commun.12,%20498%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Orth, M. F. et al. Systematic multi-omics cell line profiling uncovers principles of Ewing sarcoma fusion oncogene-mediated gene regulation. <em>Cell Rep.</em><strong>41</strong>, 111761 (2022).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2022.111761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10333306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36476851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Orth,%20M.%20F.%20et%20al.%20Systematic%20multi-omics%20cell%20line%20profiling%20uncovers%20principles%20of%20Ewing%20sarcoma%20fusion%20oncogene-mediated%20gene%20regulation.%20Cell%20Rep.41,%20111761%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Panagopoulos, I., Gorunova, L., Bjerkehagen, B. &amp; Heim, S. The “grep” command but not FusionMap, FusionFinder or ChimeraScan captures the CIC-DUX4 fusion gene from whole transcriptome sequencing data on a small round cell tumor with t(4;19)(q35;q13). <em>PLoS One</em><strong>9</strong>, e99439 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0099439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4064965/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24950227/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Panagopoulos,%20I.,%20Gorunova,%20L.,%20Bjerkehagen,%20B.%20&amp;%20Heim,%20S.%20The%20%E2%80%9Cgrep%E2%80%9D%20command%20but%20not%20FusionMap,%20FusionFinder%20or%20ChimeraScan%20captures%20the%20CIC-DUX4%20fusion%20gene%20from%20whole%20transcriptome%20sequencing%20data%20on%20a%20small%20round%20cell%20tumor%20with%20t(4;19)(q35;q13).%20PLoS%20One9,%20e99439%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Lazo de la Vega, L. et al. Targeted next-generation sequencing of CIC-DUX4 soft tissue sarcomas demonstrates low mutational burden and recurrent chromosome 1p loss. <em>Hum. Pathol.</em><strong>58</strong>, 161–170 (2016).
</cite> [<a href="https://doi.org/10.1016/j.humpath.2016.09.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5125884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27664537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lazo%20de%20la%20Vega,%20L.%20et%20al.%20Targeted%20next-generation%20sequencing%20of%20CIC-DUX4%20soft%20tissue%20sarcomas%20demonstrates%20low%20mutational%20burden%20and%20recurrent%20chromosome%201p%20loss.%20Hum.%20Pathol.58,%20161%E2%80%93170%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Carrabotta, M. et al. Integrated molecular characterization of patient-derived models reveals therapeutic strategies for treating CIC-DUX4 sarcoma. <em>Cancer Res</em>. <strong>82</strong>, 708–720 (2022).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-21-1222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9359717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34903601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carrabotta,%20M.%20et%20al.%20Integrated%20molecular%20characterization%20of%20patient-derived%20models%20reveals%20therapeutic%20strategies%20for%20treating%20CIC-DUX4%20sarcoma.%20Cancer%20Res.%2082,%20708%E2%80%93720%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Yadav, B. et al. Quantitative scoring of differential drug sensitivity for individually optimized anticancer therapies. <em>Sci. Rep.</em><strong>4</strong>, 5193 (2014).
</cite> [<a href="https://doi.org/10.1038/srep05193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4046135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24898935/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yadav,%20B.%20et%20al.%20Quantitative%20scoring%20of%20differential%20drug%20sensitivity%20for%20individually%20optimized%20anticancer%20therapies.%20Sci.%20Rep.4,%205193%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Bakaric, A. et al. CIC-DUX4 chromatin profiling reveals new epigenetic dependencies and actionable therapeutic targets in CIC-rearranged sarcomas. <em>Cancers (Basel)</em><strong>16</strong>, 457 (2024).</cite> [<a href="https://doi.org/10.3390/cancers16020457" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10814785/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38275898/" class="usa-link">PubMed</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Bosnakovski, D. et al. Inactivation of the CIC-DUX4 oncogene through P300/CBP inhibition, a therapeutic approach for CIC-DUX4 sarcoma. <em>Oncogenesis</em><strong>10</strong>, 68 (2021).
</cite> [<a href="https://doi.org/10.1038/s41389-021-00357-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8511258/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34642317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bosnakovski,%20D.%20et%20al.%20Inactivation%20of%20the%20CIC-DUX4%20oncogene%20through%20P300/CBP%20inhibition,%20a%20therapeutic%20approach%20for%20CIC-DUX4%20sarcoma.%20Oncogenesis10,%2068%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Rauh, U. et al. BRD-810 is a highly selective MCL1 inhibitor with optimized in vivo clearance and robust efficacy in solid and hematological tumor models. <em>Nat. Cancer</em><strong>5</strong>, 1479–1493 (2024).
</cite> [<a href="https://doi.org/10.1038/s43018-024-00814-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11502502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39179926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rauh,%20U.%20et%20al.%20BRD-810%20is%20a%20highly%20selective%20MCL1%20inhibitor%20with%20optimized%20in%20vivo%20clearance%20and%20robust%20efficacy%20in%20solid%20and%20hematological%20tumor%20models.%20Nat.%20Cancer5,%201479%E2%80%931493%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Stewart, E. et al. Targeting the DNA repair pathway in Ewing sarcoma. <em>Cell Rep.</em><strong>9</strong>, 829–841 (2014).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2014.09.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4386669/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25437539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stewart,%20E.%20et%20al.%20Targeting%20the%20DNA%20repair%20pathway%20in%20Ewing%20sarcoma.%20Cell%20Rep.9,%20829%E2%80%93841%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Baldauf, M. C. et al. Systematic identification of cancer-specific MHC-binding peptides with RAVEN. <em>Oncoimmunology</em><strong>7</strong>, e1481558 (2018).
</cite> [<a href="https://doi.org/10.1080/2162402X.2018.1481558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6140548/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30228952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baldauf,%20M.%20C.%20et%20al.%20Systematic%20identification%20of%20cancer-specific%20MHC-binding%20peptides%20with%20RAVEN.%20Oncoimmunology7,%20e1481558%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Tang, S. &amp; Dodd, L. G. CIC-DUX4 sarcoma diagnosed by fine-needle aspiration cytology: a case report. <em>Diagn. Cytopathol.</em><strong>46</strong>, 958–963 (2018).
</cite> [<a href="https://doi.org/10.1002/dc.24027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30353686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang,%20S.%20&amp;%20Dodd,%20L.%20G.%20CIC-DUX4%20sarcoma%20diagnosed%20by%20fine-needle%20aspiration%20cytology:%20a%20case%20report.%20Diagn.%20Cytopathol.46,%20958%E2%80%93963%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Mayoh, C. et al. High-throughput drug screening of primary tumor cells identifies therapeutic strategies for treating children with high-risk cancer. <em>Cancer Res</em>. <strong>83</strong>, 2716–2732 (2023).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-22-3702" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10425737/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37523146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mayoh,%20C.%20et%20al.%20High-throughput%20drug%20screening%20of%20primary%20tumor%20cells%20identifies%20therapeutic%20strategies%20for%20treating%20children%20with%20high-risk%20cancer.%20Cancer%20Res.%2083,%202716%E2%80%932732%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Renzi, S., Anderson, N. D., Light, N. &amp; Gupta, A. Ewing-like sarcoma: an emerging family of round cell sarcomas. <em>J. Cell Physiol.</em><strong>234</strong>, 7999–8007 (2019).
</cite> [<a href="https://doi.org/10.1002/jcp.27558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30257034/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Renzi,%20S.,%20Anderson,%20N.%20D.,%20Light,%20N.%20&amp;%20Gupta,%20A.%20Ewing-like%20sarcoma:%20an%20emerging%20family%20of%20round%20cell%20sarcomas.%20J.%20Cell%20Physiol.234,%207999%E2%80%938007%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Acanda De La Rocha, A. M. et al. Feasibility of functional precision medicine for guiding treatment of relapsed or refractory pediatric cancers. <em>Nat. Med</em>. <strong>30</strong>, 990–1000 (2024).
</cite> [<a href="https://doi.org/10.1038/s41591-024-02848-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11031400/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38605166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Acanda%20De%20La%20Rocha,%20A.%20M.%20et%20al.%20Feasibility%20of%20functional%20precision%20medicine%20for%20guiding%20treatment%20of%20relapsed%20or%20refractory%20pediatric%20cancers.%20Nat.%20Med.%2030,%20990%E2%80%931000%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Kotschy, A. et al. The MCL1 inhibitor S63845 is tolerable and effective in diverse cancer models. <em>Nature</em><strong>538</strong>, 477–482 (2016).
</cite> [<a href="https://doi.org/10.1038/nature19830" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27760111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kotschy,%20A.%20et%20al.%20The%20MCL1%20inhibitor%20S63845%20is%20tolerable%20and%20effective%20in%20diverse%20cancer%20models.%20Nature538,%20477%E2%80%93482%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Tantawy, S. I., Timofeeva, N., Sarkar, A. &amp; Gandhi, V. Targeting MCL-1 protein to treat cancer: opportunities and challenges. <em>Front Oncol.</em><strong>13</strong>, 1226289 (2023).
</cite> [<a href="https://doi.org/10.3389/fonc.2023.1226289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10436212/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37601693/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tantawy,%20S.%20I.,%20Timofeeva,%20N.,%20Sarkar,%20A.%20&amp;%20Gandhi,%20V.%20Targeting%20MCL-1%20protein%20to%20treat%20cancer:%20opportunities%20and%20challenges.%20Front%20Oncol.13,%201226289%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Bolomsky, A. et al. MCL-1 inhibitors, fast-lane development of a new class of anti-cancer agents. <em>J. Hematol. Oncol.</em><strong>13</strong>, 173 (2020).
</cite> [<a href="https://doi.org/10.1186/s13045-020-01007-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7731749/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33308268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bolomsky,%20A.%20et%20al.%20MCL-1%20inhibitors,%20fast-lane%20development%20of%20a%20new%20class%20of%20anti-cancer%20agents.%20J.%20Hematol.%20Oncol.13,%20173%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Thomas, R. L. et al. Loss of MCL-1 leads to impaired autophagy and rapid development of heart failure. <em>Genes Dev.</em><strong>27</strong>, 1365–1377 (2013).
</cite> [<a href="https://doi.org/10.1101/gad.215871.113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3701192/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23788623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thomas,%20R.%20L.%20et%20al.%20Loss%20of%20MCL-1%20leads%20to%20impaired%20autophagy%20and%20rapid%20development%20of%20heart%20failure.%20Genes%20Dev.27,%201365%E2%80%931377%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Opferman, J. T. et al. Obligate role of anti-apoptotic MCL-1 in the survival of hematopoietic stem cells. <em>Science</em><strong>307</strong>, 1101–1104 (2005).
</cite> [<a href="https://doi.org/10.1126/science.1106114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15718471/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Opferman,%20J.%20T.%20et%20al.%20Obligate%20role%20of%20anti-apoptotic%20MCL-1%20in%20the%20survival%20of%20hematopoietic%20stem%20cells.%20Science307,%201101%E2%80%931104%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Opferman, J. T. et al. Development and maintenance of B and T lymphocytes requires antiapoptotic MCL-1. <em>Nature</em><strong>426</strong>, 671–676 (2003).
</cite> [<a href="https://doi.org/10.1038/nature02067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14668867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Opferman,%20J.%20T.%20et%20al.%20Development%20and%20maintenance%20of%20B%20and%20T%20lymphocytes%20requires%20antiapoptotic%20MCL-1.%20Nature426,%20671%E2%80%93676%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Khan, S. et al. A selective BCL-X(L) PROTAC degrader achieves safe and potent antitumor activity. <em>Nat. Med</em>. <strong>25</strong>, 1938–1947 (2019).
</cite> [<a href="https://doi.org/10.1038/s41591-019-0668-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6898785/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31792461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khan,%20S.%20et%20al.%20A%20selective%20BCL-X(L)%20PROTAC%20degrader%20achieves%20safe%20and%20potent%20antitumor%20activity.%20Nat.%20Med.%2025,%201938%E2%80%931947%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Heipertz, A. E. et al. Outcome of children and adolescents with relapsed/refractory/progressive malignancies treated with molecularly informed targeted drugs in the pediatric precision oncology registry INFORM. <em>JCO Precis Oncol.</em><strong>7</strong>, e2300015 (2023).
</cite> [<a href="https://doi.org/10.1200/PO.23.00015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37364231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heipertz,%20A.%20E.%20et%20al.%20Outcome%20of%20children%20and%20adolescents%20with%20relapsed/refractory/progressive%20malignancies%20treated%20with%20molecularly%20informed%20targeted%20drugs%20in%20the%20pediatric%20precision%20oncology%20registry%20INFORM.%20JCO%20Precis%20Oncol.7,%20e2300015%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Peterziel, H. et al. Drug sensitivity profiling of 3D tumor tissue cultures in the pediatric precision oncology program INFORM. <em>NPJ Precis Oncol.</em><strong>6</strong>, 94 (2022).
</cite> [<a href="https://doi.org/10.1038/s41698-022-00335-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9794727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36575299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peterziel,%20H.%20et%20al.%20Drug%20sensitivity%20profiling%20of%203D%20tumor%20tissue%20cultures%20in%20the%20pediatric%20precision%20oncology%20program%20INFORM.%20NPJ%20Precis%20Oncol.6,%2094%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Worst, B. C. et al. Next-generation personalised medicine for high-risk paediatric cancer patients - the INFORM pilot study. <em>Eur. J. Cancer</em><strong>65</strong>, 91–101 (2016).
</cite> [<a href="https://doi.org/10.1016/j.ejca.2016.06.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27479119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Worst,%20B.%20C.%20et%20al.%20Next-generation%20personalised%20medicine%20for%20high-risk%20paediatric%20cancer%20patients%20-%20the%20INFORM%20pilot%20study.%20Eur.%20J.%20Cancer65,%2091%E2%80%93101%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Ge, S. X., Son, E. W. &amp; Yao, R. iDEP: an integrated web application for differential expression and pathway analysis of RNA-Seq data. <em>BMC Bioinforma.</em><strong>19</strong>, 534 (2018).</cite> [<a href="https://doi.org/10.1186/s12859-018-2486-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6299935/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30567491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ge,%20S.%20X.,%20Son,%20E.%20W.%20&amp;%20Yao,%20R.%20iDEP:%20an%20integrated%20web%20application%20for%20differential%20expression%20and%20pathway%20analysis%20of%20RNA-Seq%20data.%20BMC%20Bioinforma.19,%20534%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Szlavik, Z. et al. Discovery of S64315, a potent and selective Mcl-1 inhibitor. <em>J. Med Chem.</em><strong>63</strong>, 13762–13795 (2020).
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.0c01234" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33146521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Szlavik,%20Z.%20et%20al.%20Discovery%20of%20S64315,%20a%20potent%20and%20selective%20Mcl-1%20inhibitor.%20J.%20Med%20Chem.63,%2013762%E2%80%9313795%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62629_MOESM2_ESM.pdf</a><sup> (27.5KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (297KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (763.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (28.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (105.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (559.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370961/bin/41467_2025_62629_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (2.6MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>WES, RNA-seq and DNA methylation data from tumoroid models is deposited in the European Genome-Phenome Archive under the study ID <a href="https://ega-archive.org/studies/EGAS00001008039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EGAS00001008039</a>. Due to patient data protection, the data are available upon request to the data access committee (DAC). The links to the DAC can be found under the dataset IDs EGAD00001015608 (WES and RNA-seq; <a href="https://ega-archive.org/datasets/EGAD00001015608" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ega-archive.org/datasets/EGAD00001015608</a>) and EGAD00010002760 (DNA methylation data; <a href="https://ega-archive.org/datasets/EGAD00010002760" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ega-archive.org/datasets/EGAD00010002760</a>). The data will be made accessible to all interested researchers, with no limitations regarding the time or purpose of data use. Previously published ChIP-seq data<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> was downloaded from the Gene Expression Omnibus (GEO) database under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE248117" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE248117</a>. The remaining data are available within the article, supplementary information or source data file. <a href="#Sec32" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62629-6"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62629.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370961/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370961/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370961%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370961/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370961/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370961/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841513/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370961/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841513/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370961/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370961/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="UrcBIgr1KepjkbdxnJ1KC4z6GT33AsHH91e85EoA1lUKyn609wiKV8NSI5BP110y">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
