
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Protein-DNA interaction in tight-binding paradigm - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5338128AF30593053812002BF71333.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368152/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Protein-DNA interaction in tight-binding paradigm">
<meta name="citation_author" content="Hamze Mousavi">
<meta name="citation_author_institution" content="Department of Physics, Razi University, Kermanshah, Iran">
<meta name="citation_author" content="Ronak Emami">
<meta name="citation_author_institution" content="Department of Physics, Razi University, Kermanshah, Iran">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30565">
<meta name="citation_doi" content="10.1038/s41598-025-15972-z">
<meta name="citation_pmid" content="40835689">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368152/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368152/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368152/pdf/41598_2025_Article_15972.pdf">
<meta name="description" content="The interaction between protein and DNA across three distinct conformations of protein chains is examined in the framework of band structures and density of states analysis, utilizing a tight-binding Hamiltonian model in conjunction with Green’s ...">
<meta name="og:title" content="Protein-DNA interaction in tight-binding paradigm">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The interaction between protein and DNA across three distinct conformations of protein chains is examined in the framework of band structures and density of states analysis, utilizing a tight-binding Hamiltonian model in conjunction with Green’s ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368152/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368152">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-15972-z"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_15972.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368152%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368152/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368152/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368152/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 20;15:30565. doi: <a href="https://doi.org/10.1038/s41598-025-15972-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-15972-z</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Protein-DNA interaction in tight-binding paradigm</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mousavi%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Hamze Mousavi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Hamze Mousavi</span></h3>
<div class="p">
<sup>1</sup>Department of Physics, Razi University, Kermanshah, Iran </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mousavi%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hamze Mousavi</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Emami%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ronak Emami</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ronak Emami</span></h3>
<div class="p">
<sup>1</sup>Department of Physics, Razi University, Kermanshah, Iran </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Emami%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ronak Emami</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Physics, Razi University, Kermanshah, Iran </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jun 2; Accepted 2025 Aug 12; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368152  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835689/" class="usa-link">40835689</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The interaction between protein and DNA across three distinct conformations of protein chains is examined in the framework of band structures and density of states analysis, utilizing a tight-binding Hamiltonian model in conjunction with Green’s function technique. At a constant temperature and with a predetermined number of sub-sites on the DNA strand, the spectral diagrams reveal a flat energy dispersion curve for both the protein molecules and the DNA strand independently, demonstrating characteristics akin to those of semiconductors. An increase in the contact points between protein molecules and the DNA strand results in a transition from semiconducting to metallic properties, a change that is also affected by the hydrogen bonds contributed by the mutant protein at these contact points. The electronic characteristics of the protein-DNA system are modulated by the size of the DNA, leading to a conversion of localized states within the structures into less defined energy levels as the length of the DNA strand increases. The influence of temperature on the density of states causes variations in both the peak heights and their positions. The stretching effects of DNA influenced by the presence of protein molecules, result in modifications by redistributing spectral characteristics within the density of states. The interaction between protein and DNA is anticipated to have a direct impact on the electronic properties of DNA, which differ across various protein conformations, thus paving the way for new research opportunities with considerable biological significance.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Protein, DNA, Tight-binding, Density of states, Band structures</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Biophysics, Computational biophysics, Nanoscale biophysics</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Deoxyribonucleic acid (DNA) is a complex macromolecule<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup> composed of repetitive sequences of base pairs (bps), specifically adenine (A) paired with thymine (T) and guanine (G) paired with cytosine (C). These bps are interconnected through hydrogen bonds and are organized in a double-helix structure, which is reinforced by a sugar-phosphate backbone. The double-helix structure of the DNA molecule, distinguished by its <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f21e7a500209/d33e155.gif" loading="lazy" id="d33e155" alt="Inline graphic"></span>-electron cores due to base stacking, facilitates a one-dimensional (1D) pathway for electronic charge transport. The electronic characteristics of DNA are predominantly influenced by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f21e7a500209/d33e161.gif" loading="lazy" id="d33e161" alt="Inline graphic"></span>-electrons, with each site possessing a single orbital, thereby rendering it appropriate for modeling as a tight-binding (TB) system characterized by defined nearest neighbor (nn) interactions<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Essential features such as self-assembly and self-recognition are crucial for the role of DNA molecules as carriers of genetic information, and these attributes may be leveraged in the design and construction of electronic circuits<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Additionally, DNA can be integrated with various materials and methodologies to produce an extensive range of biosensor designs<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. In essence, nucleic acids serve a dual purpose; they not only transmit genetic information but also function as adaptable and potent biomolecules in the creation of electrochemical biosensors capable of detecting and identifying proteins<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>.</p>
<p id="Par3">Proteins are intricate biological macromolecules composed of a specific sequence of amino acid residues<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. The synthesis of proteins involves the linear linkage of amino acids via peptide bonds, which create a structural backbone for the protein, with diverse side chains branching off. In contrast to other biological macromolecules, proteins exhibit complex and irregular configurations. There are twenty distinct amino acids that serve as the essential building blocks of all proteins. The linear arrangement of these amino acids is known as the primary structure of the protein. Furthermore, under physiological conditions, each protein adopts a unique 3D conformation<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, which is referred to as its native tertiary structure. This native configuration is crucial, as it dictates the functional characteristics of the protein. The relationship between a protein’s amino acid sequence and its native tertiary structure represents a significant challenge in biological research. While various computational techniques have been developed to forecast protein structures, only a few of these methods provide robust mathematical assurances. Lattice models have been particularly effective in exploring the intricacies of protein structures<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. These models aid in deriving fundamental principles, making predictions, and improving our understanding of different protein properties<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p>
<p id="Par4">Interactions between proteins and DNA are fundamental, as they play a vital role in numerous cellular processes such as the regulation of gene expression, DNA replication, repair, recombination, and the organization of chromosomal DNA, thereby enabling cells to interpret and utilize the genetic information encoded in DNA for protein synthesis and essential functions<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>–<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. A key concern regarding these interactions is the approach proteins utilize to search for specific targets on DNA. The ways in which proteins can detect relatively small cognate sequences, typically consisting of 15 to 20 bps for repressors and 4 to 6 bps for restriction enzymes within the millions of bps of non-specific chromosomal DNA, have largely gone unexplored by researchers for decades<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Studies indicate that protein molecules exhibit an exceptional proficiency in quickly finding their target within the DNA sequence<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>–<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. To identify a specific DNA sequence, a protein requires multiple interactions with the bps found in the major groove of the DNA, achieved through the formation of hydrogen bonds<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> or van der Waals interactions by the protein’s amino acid side chains with the bases. It has been established that the recognition of consensus sequences on DNA by proteins is contingent upon the entanglement of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f21e7a500209/d33e235.gif" loading="lazy" id="d33e235" alt="Inline graphic"></span>-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f21e7a500209/d33e242.gif" loading="lazy" id="d33e242" alt="Inline graphic"></span> electrons between DNA nucleotides and protein amino acid<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>.</p>
<p id="Par5">DNA exhibits a helical configuration, and proteins have the capacity to modify this twist by either unwinding or overwinding the DNA<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. The natural flexibility of DNA enables it to bend and twist in reaction to protein binding, with these deformations being pivotal in numerous cellular processes. Proteins can further induce twisting and stretching of DNA, thereby influencing its overall structure and functionality. The interactions between proteins and DNA represent a dynamic interplay of forces and conformations. These interactions are profoundly affected by the mechanical properties of DNA, particularly its capacity to twist and stretch. Indeed, proteins attach to DNA and can instigate alterations in its twist and stretch, while the mechanical state of DNA can, in turn, influence its interactions with proteins. Grasping this interplay is vital for comprehending many fundamental biological processes. This relationship between protein binding and DNA mechanics is crucial for gene regulation, DNA replication, repair, and other essential cellular functions. For instance, in the context of twist-stretch profiles of DNA chains, the relationship between twist and stretch for a collection of short DNA molecules, as modeled by a mesoscopic Hamiltonian, has been investigated<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. This model also incorporates the bending and twisting fluctuations that occur between neighboring base pairs within the molecular stack. The results indicate that the chains typically exhibit over-twisting when subjected to an external stretching force, and this over-twisting is physically linked to a reduction in the average diameter of the helix, which corresponds to a damping effect on the fluctuations of the bps.</p>
<p id="Par6">Artificial intelligence (AI) and deep learning (DL) techniques are transforming the investigation of DNA-protein interactions by offering robust tools for predicting binding sites, elucidating interaction mechanisms, and even designing innovative proteins<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. These techniques utilize extensive datasets of established protein structures, DNA sequences, and interaction data to discern intricate patterns and relationships, resulting in predictions that are more precise and efficient compared to traditional methods. Recent developments in DNA-protein interaction research have been greatly accelerated by the implementation of AI and DL techniques<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>–<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. These advancements are enabling not only more accurate forecasts of protein-DNA binding locations<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, improving the understanding of protein-DNA interactions<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, but also accelerating the process of drug discovery<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. For example, AI and DL are advancing structure-based drug discovery by refining ligand binding site identification, estimating protein-ligand binding poses, and developing scoring functions<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Furthermore, AI-driven methods enable a comprehensive analysis of protein structures, which can be associated with the prediction of drug-target interactions. Additionally, AI and DL are being incorporated into computational models to scrutinize protein structures and forecast drug-target interactions.</p>
<p id="Par7">It has been demonstrated that certain proteins may lose their capacity to bind to particular DNA sequences when exposed to low temperatures, whereas others may exhibit modified specificity<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. At low temperatures, the interactions between proteins and DNA can undergo significant alterations. While some interactions may weaken or become disrupted due to diminished molecular motion, others may be enhanced or exhibit increased specificity. It is important to observe that while it is theoretically possible for protein-DNA interactions to occur at very low temperatures, the significant decrease in molecular motion greatly influences the dynamics necessary for stable and specific binding. However, some interactions may still emerge due to quantum mechanical effects or residual thermal energy. The specific outcomes are contingent upon the individual protein and DNA sequence involved, as well as the characteristics of their interaction<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Consequently, protein-DNA interactions can be profoundly affected by extremely low temperatures, potentially resulting in alterations in binding affinity, stability, and even the capacity of proteins to engage with DNA. In fact, lower temperatures typically enhance the stability of protein-DNA complexes. This phenomenon occurs because the reduction in thermal energy makes it more challenging for the complex to dissociate<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Furthermore, the changes in enthalpy and entropy associated with binding can differ markedly at low temperatures compared to physiological conditions. For example, certain protein-DNA interactions that are entropy-driven at room temperature may transition to being enthalpy-driven at low temperature<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Additionally, low temperatures can encourage the emergence of novel protein-DNA interactions that are not present at elevated temperatures<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Comprehending the interactions between proteins and DNA at low temperatures is crucial for grasping cellular responses. Numerous organisms face stress due to low temperatures, and their survival hinges on the effective operation of protein-DNA interactions<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. By elucidating how proteins engage with DNA in cold environments, researchers can develop innovative materials and pharmaceuticals that operate under extreme conditions<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Investigations at low temperatures can yield valuable insights into the basic principles governing protein folding and the dynamics of protein-DNA interactions<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p>
<p id="Par8">Proteins, in contrast to other biological macromolecules, exhibit intricate and irregular structures. The geometric properties of proteins are related to the impact of their 3D conformation and the spatial arrangement of amino acids on the architecture, stability, and functionality of the protein. These geometric features include hydrogen bonding, van der Waals forces, and the overall folding configuration. Indeed, the models have a specific amino acid sequence, with differences in the distribution of hydrogen bonds, where thirteen hydrogen bonds are designed to simulate a mutation (disease) observed in a natural protein, as noted by<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. The electronic properties of proteins arise from their amino acid composition, notably the presence of aromatic residues and their potential to form <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f21e7a500209/d33e345.gif" loading="lazy" id="d33e345" alt="Inline graphic"></span>-interactions, which play a crucial role in determining their folding, stability, and molecular interactions. Protein-DNA interaction research can leverage a TB model that portrays DNA as a system with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f21e7a500209/d33e351.gif" loading="lazy" id="d33e351" alt="Inline graphic"></span>-electrons and specific nn interactions, allowing for the modeling of proteins as engaging in these interactions. A TB model for DNA-protein interactions employs the principles of TB theory to depict the relationship between a protein and DNA, conceptualizing the DNA molecule as a series of distinct sites for protein binding. The binding strength is influenced by the specific interactions between the amino acids of the protein and the bases of the DNA at each site, facilitating the computation of binding affinities and sequence specificity derived from the interaction energies at these locations<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. In the TB framework concerning the interaction between 3D proteins and 1D DNA, proteins first move through 3D space. Upon meeting DNA, they transition to a 1D sliding motion along the DNA to locate their specific target site<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. This research seeks to explore the interaction between protein and DNA by examining the band structure (BS) and density of states (DOS) within a system that comprises a double-stranded DNA configuration<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup> alongside three unique types of protein chains (Fig. <a href="#Fig1" class="usa-link">1</a>a) designated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8ecb3cf3d805/d33e376.gif" loading="lazy" id="d33e376" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d126173e404e/d33e382.gif" loading="lazy" id="d33e382" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/4936d1b20030/d33e388.gif" loading="lazy" id="d33e388" alt="Inline graphic"></span><sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. In the context of the DNA model, each site corresponds to a nitrogenous base (A, T, C, and G), with electronic pathways illustrated as solid lines, representing the form of a ladder (Fig. <a href="#Fig1" class="usa-link">1</a>b). The arrangement of protein molecules is dictated by a specific sequence of amino acids, with differences in the orientation of hydrogen bonds; a total of thirteen hydrogen bonds are utilized to model a mutation associated with a certain disease in a natural protein<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. This examination is focused on protein chains that comprise thirty-six amino acids. The <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8ecb3cf3d805/d33e411.gif" loading="lazy" id="d33e411" alt="Inline graphic"></span> conformation shows a simple structural framework where amino acids are connected exclusively by covalent (peptide) bonds. In contrast, the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d126173e404e/d33e417.gif" loading="lazy" id="d33e417" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/4936d1b20030/d33e423.gif" loading="lazy" id="d33e423" alt="Inline graphic"></span> conformations exhibit a structural configuration in which amino acids are linked through both peptide bonds and non-covalent (hydrogen bond) interactions. This study builds on the protein models as presented in<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. They utilized computer simulations within the lattice model framework to assess and validate the primary hypotheses of the protein engineering analysis regarding the transition state between the folded and unfolded states. In their model, a protein chain is represented as a self-avoiding walk on a cubic lattice, with covalently bonded residues occupying neighboring lattice sites. Their approach is based on the principle of conducting sequence design and folding simulations. This sequence was developed to achieve a low relative energetic stabilization, which makes the hydrogen bonds vital for their models. Besides, the double-stranded model offers a simplified yet effective representation of DNA’s double helix structure in computational studies, enabling researchers to simulate the behavior of DNA under various conditions. This model is significant for comprehending DNA’s role in biological processes and its potential applications in nanotechnology. It is utilized to investigate DNA’s properties, such as its structure, flexibility, and electronic transport. The model frequently incorporates elements like base pairing interactions, the flexibility of the sugar-phosphate backbone, and the helical twist to accurately simulate DNA’s behavior. Through this computational model, researchers can explore how DNA interacts with proteins, other molecules, or surfaces, which is essential for understanding gene regulation and the development of DNA-based technologies. This study utilized the TB Hamiltonian model alongside Green’s function methodology to examine the interactions between DNA and three conformations of protein models, considering a range of temperatures and varying hopping parameter values within the protein chains, as well as the dimerization effects of parallel hopping in the DNA and the effects of stretching of the DNA strand on the protein-DNA interaction. We have employed existence models for both protein chains and DNA strands to investigate the interaction between proteins and DNA. Due to the complex nature of these systems, we opted for a simplified model to make the problem more tractable. Such simplifications have been incorporated into the models utilized in this research. For example, the quantum transport of protein chains has been examined in<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, while the electronic characteristics of DNA using a double-stranded model have been studied in<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Despite the simplifications, these models successfully capture the qualitative behavior of protein chains and DNA strands. Concerning the length of the DNA sequence, it is fundamentally important for the efficiency and stability of protein-DNA interactions. This length affects the search mechanism, binding affinity, and the kinetics of the interaction as a whole, thereby influencing numerous cellular processes that depend on these interactions, including gene regulation and DNA replication. The findings of this study indicate that the interaction between protein and DNA is influenced not only by the number of contact points, temperature, and the conformation of the protein, but also by the total length of the DNA segment under consideration.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368152_41598_2025_15972_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/febd68706ea1/41598_2025_15972_Fig1_HTML.jpg" loading="lazy" id="MO1" height="367" width="752" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Panel (<strong>a</strong>) presents a schematic illustration of the conformations of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8ecb3cf3d805/d33e457.gif" loading="lazy" id="d33e457" alt="Inline graphic"></span> (left), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d126173e404e/d33e463.gif" loading="lazy" id="d33e463" alt="Inline graphic"></span> (center), and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/4936d1b20030/d33e469.gif" loading="lazy" id="d33e469" alt="Inline graphic"></span> (right), with hydrogen bonds indicated by dashed lines. Panel (<strong>b</strong>) illustrates the double-stranded DNA, highlighting the connections between four sub-sites of a protein molecule and the DNA strand, along with the representation of the hopping terms.</p></figcaption></figure></section><section id="Sec2"><h2 class="pmc_sec_title">Hamiltonian model</h2>
<p id="Par9">The TB Hamiltonian framework functions as a theoretical model for elucidating the dynamics of quantum systems. This model enables the examination of electronic characteristics across diverse materials by utilizing a Hamiltonian that integrates various factors. The TB Hamiltonian encompasses the complete system, which includes the DNA strand denoted by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/470c5716420d/d33e484.gif" loading="lazy" id="d33e484" alt="Inline graphic"></span>, the protein chain represented as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/394e1b168569/d33e490.gif" loading="lazy" id="d33e490" alt="Inline graphic"></span>, and their interactions or couplings symbolized by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/0555ae750595/d33e496.gif" loading="lazy" id="d33e496" alt="Inline graphic"></span>. The model is characterized by the following Hamiltonian:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/3ee96c0716b9/d33e502.gif" loading="lazy" id="d33e502" alt="graphic file with name d33e502.gif"></td>
<td class="label">1</td>
</tr></table>
<p>each term, when articulated within the framework of second quantization, can be represented through its local and hopping components as follows:</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f6ee7285d5ab/d33e509.gif" loading="lazy" id="d33e509" alt="graphic file with name d33e509.gif"></td>
<td class="label">2</td>
</tr></table>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/4896eaf15aa5/d33e515.gif" loading="lazy" id="d33e515" alt="graphic file with name d33e515.gif"></td>
<td class="label">3</td>
</tr></table>
<p>and</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/3b241263f044/d33e523.gif" loading="lazy" id="d33e523" alt="graphic file with name d33e523.gif"></td>
<td class="label">4</td>
</tr></table>
<p>in this context, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/73975112fc4a/d33e530.gif" loading="lazy" id="d33e530" alt="Inline graphic"></span> refers to <em>a</em> and <em>b</em> strand indices of the DNA model, <em>i</em> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/942eccd67fa5/d33e545.gif" loading="lazy" id="d33e545" alt="Inline graphic"></span>) and <em>j</em> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/69a2166b115e/d33e555.gif" loading="lazy" id="d33e555" alt="Inline graphic"></span>) denote specific sub-sites within the DNA strand (protein chain). The notation <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/7d9098917f31/d33e561.gif" loading="lazy" id="d33e561" alt="Inline graphic"></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/eae538c89231/d33e567.gif" loading="lazy" id="d33e567" alt="Inline graphic"></span>) signifies the on-site energy associated with an electron positioned at the sub-site or nucleic acid (amino acid) <em>i</em> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/942eccd67fa5/d33e576.gif" loading="lazy" id="d33e576" alt="Inline graphic"></span>). The parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/4eb94c926e6e/d33e583.gif" loading="lazy" id="d33e583" alt="Inline graphic"></span> represents the hopping amplitudes for a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f21e7a500209/d33e589.gif" loading="lazy" id="d33e589" alt="Inline graphic"></span>-electron moving from sub-site <em>i</em> to its nn <em>j</em>. It is important to observe that the parallel and perpendicular notations on <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/4eb94c926e6e/d33e601.gif" loading="lazy" id="d33e601" alt="Inline graphic"></span> indicate hopping between nn sites within the same strand and between two distinct strands, respectively. It is worth mentioning that the perpendicular notation on <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/61e312139f58/d33e607.gif" loading="lazy" id="d33e607" alt="Inline graphic"></span> signifies hopping between two separate strands. Consequently, the electron may hop, for instance, from the <em>i</em> site on strand <em>a</em> to the corresponding site <em>i</em> on strand <em>b</em>. Thus, the perpendicular hopping pertains solely to the two bp mates on the complementary DNA strands, meaning <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/21a90cc4db6f/d33e626.gif" loading="lazy" id="d33e626" alt="Inline graphic"></span>. Also, the parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/c7763a9c43cd/d33e632.gif" loading="lazy" id="d33e632" alt="Inline graphic"></span> (along with its counterpart <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/414341abd6c8/d33e639.gif" loading="lazy" id="d33e639" alt="Inline graphic"></span>) represents the nn hopping amplitude for a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f21e7a500209/d33e645.gif" loading="lazy" id="d33e645" alt="Inline graphic"></span>-electron moving from sub-site <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/942eccd67fa5/d33e651.gif" loading="lazy" id="d33e651" alt="Inline graphic"></span> to sub-site <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/69a2166b115e/d33e657.gif" loading="lazy" id="d33e657" alt="Inline graphic"></span>, which is associated with the energy of the peptide bond (hydrogen bond energy). Moreover, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/70490c4e2454/d33e663.gif" loading="lazy" id="d33e663" alt="Inline graphic"></span> indicates the overlap term between the DNA strand and the protein molecule and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/c4e10f229f5b/d33e669.gif" loading="lazy" id="d33e669" alt="Inline graphic"></span> denotes the Hermitian conjugate. Furthermore, the operators <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/2323f46cbb52/d33e676.gif" loading="lazy" id="d33e676" alt="Inline graphic"></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/15fb46c7221a/d33e682.gif" loading="lazy" id="d33e682" alt="Inline graphic"></span>) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/b209696e842a/d33e688.gif" loading="lazy" id="d33e688" alt="Inline graphic"></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/3088c0cc42a8/d33e694.gif" loading="lazy" id="d33e694" alt="Inline graphic"></span>) are defined as the creation and annihilation operators for a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f21e7a500209/d33e700.gif" loading="lazy" id="d33e700" alt="Inline graphic"></span>-electron at sub-site <em>i</em> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/942eccd67fa5/d33e710.gif" loading="lazy" id="d33e710" alt="Inline graphic"></span>). It is essential to acknowledge that variations in temperature can induce structural disorder and fluctuations within the protein<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Therefore, we investigate the effects of temperature changes on the hopping terms, particularly emphasizing the fluctuations in twist angles. We denote <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/3fea3126f7ac/d33e720.gif" loading="lazy" id="d33e720" alt="Inline graphic"></span> to represent the fluctuations in twist angles between adjacent sub-sites, which are comparable to the torsional angles found along a chemical bond. By assuming a Gaussian distribution for the variable <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/edd700580a15/d33e726.gif" loading="lazy" id="d33e726" alt="Inline graphic"></span>, we find that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/e55eca764ee2/d33e732.gif" loading="lazy" id="d33e732" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/72a12c785b3d/d33e738.gif" loading="lazy" id="d33e738" alt="Inline graphic"></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/b2ee031d8360/d33e745.gif" loading="lazy" id="d33e745" alt="Inline graphic"></span>. Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/0fd53e864e7d/d33e751.gif" loading="lazy" id="d33e751" alt="Inline graphic"></span> denotes the reduced moment of inertia related to the relative rotation of nn amino acids, while <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/0fa42dae2c49/d33e757.gif" loading="lazy" id="d33e757" alt="Inline graphic"></span> represents the oscillator frequency associated with the vibrational mode<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. These formulations enable the derivation of temperature-dependent hopping integrals, which can be approximated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/38929f142c3d/d33e767.gif" loading="lazy" id="d33e767" alt="Inline graphic"></span>. We operate within a framework where the physical constants are normalized to unity.</p>
<p id="Par10">If we consider a DNA strand (protein chain) comprising <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/a1a6946ad7d3/d33e775.gif" loading="lazy" id="d33e775" alt="Inline graphic"></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/ad664d8646e4/d33e781.gif" loading="lazy" id="d33e781" alt="Inline graphic"></span>) sub-sites, the total Hamiltonian, denoted as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/361c063ed35e/d33e787.gif" loading="lazy" id="d33e787" alt="Inline graphic"></span>, can be represented in a matrix form as a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/a3e113ba67ef/d33e793.gif" loading="lazy" id="d33e793" alt="Inline graphic"></span> matrix where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8d7f27c0a01e/d33e799.gif" loading="lazy" id="d33e799" alt="Inline graphic"></span>. This matrix includes sub-matrices: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/3592c04852ee/d33e806.gif" loading="lazy" id="d33e806" alt="Inline graphic"></span> for the DNA strand, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/21c35f54b765/d33e812.gif" loading="lazy" id="d33e812" alt="Inline graphic"></span> for the protein molecule, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/a9e54a326c01/d33e818.gif" loading="lazy" id="d33e818" alt="Inline graphic"></span> for the interaction or coupling component of the Hamiltonian:</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/29b235d4237e/d33e824.gif" loading="lazy" id="d33e824" alt="graphic file with name d33e824.gif"></td>
<td class="label">5</td>
</tr></table>
<p>In Eq. (<a href="#Equ5" class="usa-link">5</a>), the Hamiltonian sub-matrix of the DNA strand can be represented as:</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/66d0d1240dec/d33e834.gif" loading="lazy" id="d33e834" alt="graphic file with name d33e834.gif"></td>
<td class="label">6</td>
</tr></table>
<p>in this equation, the parallel hopping amplitudes (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/fd4ea870f839/d33e842.gif" loading="lazy" id="d33e842" alt="Inline graphic"></span>) between nn sub-sites are represented by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/6bc7a10d7e6c/d33e848.gif" loading="lazy" id="d33e848" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/580d28d5ac85/d33e854.gif" loading="lazy" id="d33e854" alt="Inline graphic"></span>, whereas <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/b0794c074fe4/d33e860.gif" loading="lazy" id="d33e860" alt="Inline graphic"></span> denotes the term for perpendicular hopping (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/c65a376e425b/d33e866.gif" loading="lazy" id="d33e866" alt="Inline graphic"></span>). One would acknowledge that the distinct parallel hoppings lead to the introduction of dimerization parameters. According to the Hamiltonian in Eq. (<a href="#Equ6" class="usa-link">6</a>), we have selected a sequence of AT and GC within the DNA strand. In fact, a DNA strand characterized by repetitive AT and GC bps is a prevalent characteristic in genomes, exhibiting diverse distributions across various regions. These repeating motifs, particularly as simple sequence repeats, may be associated with numerous genomic functions and even diseases<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. The protein Hamiltonian sub-matrix of Eq. (<a href="#Equ5" class="usa-link">5</a>) for the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8ecb3cf3d805/d33e883.gif" loading="lazy" id="d33e883" alt="Inline graphic"></span> conformation can be presented by:</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/b8025cf9ca76/d33e889.gif" loading="lazy" id="d33e889" alt="graphic file with name d33e889.gif"></td>
<td class="label">7</td>
</tr></table>
<p>wherein <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/2f78ed79ccc1/d33e896.gif" loading="lazy" id="d33e896" alt="Inline graphic"></span> is defined as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/0d50e4c69510/d33e902.gif" loading="lazy" id="d33e902" alt="Inline graphic"></span>. The Hamiltonian matrices for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d126173e404e/d33e909.gif" loading="lazy" id="d33e909" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/4936d1b20030/d33e915.gif" loading="lazy" id="d33e915" alt="Inline graphic"></span> can be calculated in a comparable fashion. The nonzero elements of the Hamiltonian sub-matrix presented in Eq. (<a href="#Equ5" class="usa-link">5</a>) related to the interaction component are contingent upon the number of coupling points between the protein molecule and the DNA strand. For instance, if the interaction between the protein and DNA occurs at a single point located at sub-site number 3 on one side of the DNA strand, the nonzero elements of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/a9e54a326c01/d33e924.gif" loading="lazy" id="d33e924" alt="Inline graphic"></span> would be represented as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/dc6d233236d5/d33e930.gif" loading="lazy" id="d33e930" alt="Inline graphic"></span> (Fig. <a href="#Fig1" class="usa-link">1</a>). As we wrap up this section, we emphasize that the characteristic equation presented below can serve to numerically identify the spectrum of the system through the total Hamiltonian matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/361c063ed35e/d33e940.gif" loading="lazy" id="d33e940" alt="Inline graphic"></span>:</p>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/71ee31fbff8b/d33e946.gif" loading="lazy" id="d33e946" alt="graphic file with name d33e946.gif"></td>
<td class="label">8</td>
</tr></table>
<p>where the symbol <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/604675c64bec/d33e953.gif" loading="lazy" id="d33e953" alt="Inline graphic"></span> denotes the energy levels of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f21e7a500209/d33e959.gif" loading="lazy" id="d33e959" alt="Inline graphic"></span>-electrons, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/4f3e3b441977/d33e965.gif" loading="lazy" id="d33e965" alt="Inline graphic"></span> represents a unit matrix of size <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/a3e113ba67ef/d33e971.gif" loading="lazy" id="d33e971" alt="Inline graphic"></span>. As a result, the system is defined by <em>N</em> eigenvalues.</p></section><section id="Sec3"><h2 class="pmc_sec_title">Green’s function and DOS</h2>
<p id="Par11">Green’s functions are essential in various applications of quantum mechanics, especially in the study of transport phenomena, as they enable the investigation of particle dynamics within complex systems. The Green’s function is fundamentally connected to the Hamiltonian operator of the system, and its properties are closely related to the system’s energy spectrum. The matrix representation of the Green’s function<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> associated with the Hamiltonian can be formulated as:</p>
<table class="disp-formula p" id="Equ9"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/039cdbce5b9c/d33e989.gif" loading="lazy" id="d33e989" alt="graphic file with name d33e989.gif"></td>
<td class="label">9</td>
</tr></table>
<p>wherein <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/41b743166fc5/d33e996.gif" loading="lazy" id="d33e996" alt="Inline graphic"></span>. The imaginary component of the Green’s function at a specific energy level reflects the probability of finding a state at that energy. In the context of quantum mechanics, the DOS at a given energy can be directly obtained from the imaginary part of the trace of the Green’s function associated with that energy. To determine the DOS for the system, it is necessary to calculate the trace over all quantum numbers that characterize the Hamiltonian:</p>
<table class="disp-formula p" id="Equ10"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/b770f9ed0843/d33e1002.gif" loading="lazy" id="d33e1002" alt="graphic file with name d33e1002.gif"></td>
<td class="label">10</td>
</tr></table>
<p>The trace presented in Eq. (<a href="#Equ10" class="usa-link">10</a>) can be calculated in the following manner:</p>
<table class="disp-formula p" id="Equ11"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/cfcb70c4418d/d33e1013.gif" loading="lazy" id="d33e1013" alt="graphic file with name d33e1013.gif"></td>
<td class="label">11</td>
</tr></table>
<p>where both site indices <em>i</em> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq77"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/942eccd67fa5/d33e1023.gif" loading="lazy" id="d33e1023" alt="Inline graphic"></span> are shown by <em>m</em>. To obtain the DOS for either DNA or protein independently, one may incorporate the relevant sub-matrices from Eqs. (<a href="#Equ6" class="usa-link">6</a>) and (<a href="#Equ7" class="usa-link">7</a>) into the Green’s function as defined in Eq. (<a href="#Equ9" class="usa-link">9</a>).</p></section><section id="Sec4"><h2 class="pmc_sec_title">Results and discussion</h2>
<p id="Par12">The formalism established in the preceding sections is employed to examine the coupling of various protein conformations, designated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq78"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8ecb3cf3d805/d33e1046.gif" loading="lazy" id="d33e1046" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq79"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d126173e404e/d33e1052.gif" loading="lazy" id="d33e1052" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq80"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/4936d1b20030/d33e1058.gif" loading="lazy" id="d33e1058" alt="Inline graphic"></span>, with the double-strand DNA, utilizing the energy spectrum of the system. This study investigates the effects of different numbers of contact points (CPs) between protein and DNA, the consequences of altering the length of the DNA strand, and the impact of temperature on the BS and DOS of the system. Initially, we outline the key parameters necessary for obtaining numerical results. The DNA model features three distinct hopping parameters, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq81"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/6753d2744180/d33e1064.gif" loading="lazy" id="d33e1064" alt="Inline graphic"></span> represent the parallel hopping terms between bases, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq82"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/27735a4db69e/d33e1070.gif" loading="lazy" id="d33e1070" alt="Inline graphic"></span> denotes the perpendicular hopping amplitudes (Fig. <a href="#Fig1" class="usa-link">1</a>), with values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq83"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f53a5b3eb5f6/d33e1080.gif" loading="lazy" id="d33e1080" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq84"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/2900ad504c63/d33e1086.gif" loading="lazy" id="d33e1086" alt="Inline graphic"></span><sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, and the dimerization parameter is defined as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq85"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/c8e03b3af4b2/d33e1095.gif" loading="lazy" id="d33e1095" alt="Inline graphic"></span>. Furthermore, the hopping term within the protein is established based on the values referenced in<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, specifically <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq86"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/274d5262a87b/d33e1106.gif" loading="lazy" id="d33e1106" alt="Inline graphic"></span>. For the assessment, a configuration of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq87"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/e766a79b8d55/d33e1112.gif" loading="lazy" id="d33e1112" alt="Inline graphic"></span> is generated across thirty-six sub-sites, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq88"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/e766a79b8d55/d33e1118.gif" loading="lazy" id="d33e1118" alt="Inline graphic"></span> values varying within the range of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq89"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/4c2eb9c6bf26/d33e1124.gif" loading="lazy" id="d33e1124" alt="Inline graphic"></span>. Consequently, as outlined following Eq. (<a href="#Equ7" class="usa-link">7</a>), we derive <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq90"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/9b2012f06896/d33e1133.gif" loading="lazy" id="d33e1133" alt="Inline graphic"></span> for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq91"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/46479aaf9859/d33e1140.gif" loading="lazy" id="d33e1140" alt="Inline graphic"></span>, utilizing thirty-five distinct values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq92"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/e766a79b8d55/d33e1146.gif" loading="lazy" id="d33e1146" alt="Inline graphic"></span>. Additionally, the energy associated with hydrogen bonds indicates relatively weak intermolecular interactions compared to peptide bonds, as illustrated in Fig. <a href="#Fig1" class="usa-link">1</a>a by dashed lines, thus we set <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq93"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/28c9436e79d0/d33e1155.gif" loading="lazy" id="d33e1155" alt="Inline graphic"></span>. Despite the relative weakness of hydrogen bonds, their abundance and the specific arrangement of these bonds among amino acids in the chain can considerably stabilize and influence the spatial configuration of proteins. It is crucial to recognize that protein sequences offer a more intricate alignment than DNA, as DNA is made up of merely four nucleotides, while proteins are formed from twenty distinct amino acids. In this regard, we explore thirteen connections that play a vital role in stabilizing these conformational structures<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. We posit that all on-site energies within the system are normalized to zero, thereby creating a baseline for the energy levels. Although this may seem like an oversimplification, this basic model has successfully captured the semiconducting characteristics of DNA double-strands by utilizing diverse values for the hopping parameters or dimerization effects<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. In general, when the on-site energy is set to zero, the Hamiltonian simplifies, facilitating an analysis of the correlation between the hopping parameters and the lattice geometry. To design a semiconducting material by manipulating the on-site energy and hopping parameters, one can employ a tight-binding model to depict the electronic structure of the material. By fixing the on-site energy at zero and adjusting the parallel hopping parameters between adjacent sites, it becomes feasible to create an energy band gap, which is a characteristic feature of semiconductors<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. As a result, rather than maintaining a uniform overlap among sub-sites in each segment of the system, we suggest that the hopping parameters fluctuate across protein chains from one site to another, while still being close to the value indicated in<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> and taking into account the dimerization within the DNA component of the system. In essence, to engineer a semiconducting material through the manipulation of on-site energy and hopping parameters, one can utilize a TB model to represent the electronic structure of the material. By establishing the on-site energy at zero and altering the hopping parameters among adjacent sites, it is possible to induce the emergence of an energy band gap, a defining feature of semiconductors. Furthermore, with the on-site energy fixed at zero, the Hamiltonian simplifies, enabling a clearer analysis of the interplay between hopping parameters and lattice geometry. It is significant to highlight that while a protein may interact with DNA at a singular site, effective recognition and robust binding typically necessitate interactions at multiple sites along a short sequence. This is achieved through various non-covalent interactions, including hydrogen bonds, ionic interactions, and van der Waals forces, which collectively enhance the stability of the protein-DNA complex<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. The primary atomic interactions occur between the nitrogen and oxygen atoms of the nucleotide bases, which are accessible in the major and minor grooves of the DNA structure, facilitating the formation of hydrogen bonds with specific amino acid side chains of the protein<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Furthermore, it is noteworthy that the distance between adjacent nucleotides (or bps) in a DNA strand is approximately <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq94"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/c516534cc533/d33e1198.gif" loading="lazy" id="d33e1198" alt="Inline graphic"></span>, while the inter-atomic distance between amino acids in a protein chain is generally around <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq95"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/83f1c7217f4e/d33e1204.gif" loading="lazy" id="d33e1204" alt="Inline graphic"></span>, although this can fluctuate based on the specific amino acids involved and the overall conformation of the protein. The typical range for interaction distances between side chains is estimated to be between 4 and 6 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq96"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/9ed145161524/d33e1211.gif" loading="lazy" id="d33e1211" alt="Inline graphic"></span><sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. In the context of DNA bases such as G, C, T, and A, the term ’hopping electrons’ refers to the transfer of electrons from one base to another along the DNA strand. G is particularly notable for its propensity to accept electrons due to its relatively low ionization potential, which facilitates its ability to gain electrons. This phenomenon is termed electron hopping, as electrons transition between adjacent bases in the DNA sequence, predominantly occurring between stacked GC pairs. Among the DNA bases, G exhibits the lowest ionization energy, followed by A, C, and T, which has the highest ionization energy. This indicates that G is the most readily ionizable base, while T is the least. The energy required for electron transfer, which is the process of hopping between an amino acid and a nucleic acid, typically ranges from approximately <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq97"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d4b0975cc9de/d33e1220.gif" loading="lazy" id="d33e1220" alt="Inline graphic"></span> to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq98"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5ed07b94504d/d33e1226.gif" loading="lazy" id="d33e1226" alt="Inline graphic"></span>, contingent upon the specific amino acid and nucleic acid base involved, as well as the distance separating them. The most probable electron hopping events occur between aromatic amino acids, such as tyrosine and tryptophan, and the nucleobases G or A<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. We analyze the interactions of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq99"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f21e7a500209/d33e1239.gif" loading="lazy" id="d33e1239" alt="Inline graphic"></span>-electrons among protein chains and DNA strand, considering one to four CPs between amino acids and nucleic acids. Four unique interactions between protein molecules and the DNA strand are analyzed separately. These interactions are classified into four distinct modes of interaction: (i) sub-site 1 of the protein molecules, (ii) sub-sites 19 and 20 of the protein molecules, (iii) sub-sites 29, 30, and 31 of the protein molecules, and (iv) sub-sites 1, 2, 3, and 4 of the protein molecules (see Fig. <a href="#Fig1" class="usa-link">1</a>). Each mode delineates a unique interaction pattern between the protein molecules and the DNA strand. The coupling terms (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq100"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/70490c4e2454/d33e1249.gif" loading="lazy" id="d33e1249" alt="Inline graphic"></span>) are quantified as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq101"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/f8355c61b957/d33e1255.gif" loading="lazy" id="d33e1255" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq102"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/674b6102e4dd/d33e1261.gif" loading="lazy" id="d33e1261" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq103"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/df124acbc516/d33e1267.gif" loading="lazy" id="d33e1267" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq104"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/e75bd0fec5f3/d33e1273.gif" loading="lazy" id="d33e1273" alt="Inline graphic"></span>, which arise from the interaction of various sub-sites of DNA with protein molecules in each mode of interaction. In order to investigate the interaction between the proteins and the DNA strand, we analyze a DNA model that incorporates <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq105"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/a1a6946ad7d3/d33e1280.gif" loading="lazy" id="d33e1280" alt="Inline graphic"></span> values of 40 and 160 sub-sites.</p>
<p id="Par13">Figure <a href="#Fig2" class="usa-link">2</a> illustrates the DOS of DNA in comparison to three distinct conformations of protein molecules at a constant temperature of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq109"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8712523c2f3c/d33e1317.gif" loading="lazy" id="d33e1317" alt="Inline graphic"></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq110"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5bfa661456cf/d33e1323.gif" loading="lazy" id="d33e1323" alt="Inline graphic"></span> and for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq111"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d99f27f5e9e9/d33e1329.gif" loading="lazy" id="d33e1329" alt="Inline graphic"></span> sub-sites of DNA. The results indicate that all examined molecules display semiconducting properties, with DNA demonstrating a more substantial energy gap. The DOS for these finite systems is characterized by localization due to the quantization of energy levels inherent to the system. In such finite systems, the energy states are discrete, possessing a limited number of quantum states, in contrast to an infinite system where energy levels are continuous. This discreteness results in a localized DOS, wherein the likelihood of locating an electron at a specific energy level is concentrated in defined spatial regions rather than being uniformly distributed. The boundaries of these molecules confine particle movement, leading to quantized energy levels, which means that only certain discrete energy values are permissible for the particles within the system. Consequently, the DOS, which quantifies the number of available energy states per unit energy, becomes concentrated at these discrete energy levels. This phenomenon engenders a localized DOS, where the probability of finding an electron at a given energy level is focused in particular spatial locations. The implications of a localized DOS in finite systems are profound, influencing the behavior of particles within the system and potentially altering the physical properties of the system.</p>
<figure class="fig xbox font-sm" id="Fig2"><h3 class="obj_head">Fig. 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368152_41598_2025_15972_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/10c344f743b9/41598_2025_15972_Fig2_HTML.jpg" loading="lazy" id="MO2" height="499" width="762" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The DOS associated with an individual DNA strand and three distinct conformations of the protein chains at a constant temperature of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq106"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8712523c2f3c/d33e1292.gif" loading="lazy" id="d33e1292" alt="Inline graphic"></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq107"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/c4c1367e7230/d33e1298.gif" loading="lazy" id="d33e1298" alt="Inline graphic"></span>), along with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq108"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d99f27f5e9e9/d33e1304.gif" loading="lazy" id="d33e1304" alt="Inline graphic"></span> sub-sites of DNA, demonstrates semiconducting properties and localization, with the DNA showing a more pronounced energy gap.</p></figcaption></figure><p id="Par14">Figure <a href="#Fig3" class="usa-link">3</a> depicts the DOS and BS of the protein-DNA system, focusing on the interactions between three unique conformations of protein molecules and a DNA strand. Each panel within this figure presents distinct CP numbers for DNA-protein interactions, with parameters set at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq116"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d99f27f5e9e9/d33e1372.gif" loading="lazy" id="d33e1372" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq117"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8712523c2f3c/d33e1378.gif" loading="lazy" id="d33e1378" alt="Inline graphic"></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq118"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5bfa661456cf/d33e1384.gif" loading="lazy" id="d33e1384" alt="Inline graphic"></span>. The BS is represented solely for the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq119"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8ecb3cf3d805/d33e1390.gif" loading="lazy" id="d33e1390" alt="Inline graphic"></span> conformation of the protein to enhance the clarity of the diagram. The BS is depicted in terms of site index, which encompasses 76 sub-sites within the protein-DNA system. The BS diagrams are positioned on the left, while the DOS for all three conformations is shown on the right, with the DOS diagrams corresponding to the BS represented in matching color. It is crucial to acknowledge that while a traditional BS plot, predicated on the assumption of a periodic lattice, is not directly applicable to finite systems, we can still explore the energy levels of protein-DNA complexes in a manner that emulates the BS. This exploration can be conducted by plotting the energy levels as a function of a relevant parameter, such as the site index, which provides a representation of the band-like features inherent in this system. Figure <a href="#Fig3" class="usa-link">3</a> illustrates discrete energy levels that can be graphed against the selected parameter to depict the energy distribution. Essentially, within this finite system, the BS, when expressed in terms of the site index, represents the energy levels plotted against the site number or the position of each block within the system. This effectively showcases how an electron’s energy varies as it moves from one sub-site to another within the protein-DNA framework, with each distinct energy level corresponding to a band in the system, and the site index acting as a discrete variable along the <em>x</em>-axis of the graph. The confinement of electrons to limited dimensions leads to the quantization of energy levels. As a result, instead of forming a continuous band, the allowed energy states are converted into discrete values, resembling distinct steps or individual levels. This phenomenon, referred to as quantum confinement, arises when a system of constrained dimensions leads to the quantization of its energy levels. Additionally, the flat band in the BS represents a region where the energy of electrons remains constant regardless of their momentum. This results in a flat energy dispersion curve on a graph, indicating that the charge carriers within this band possess nearly zero group velocity and are highly localized. Furthermore, this figure illustrates that three conformations exhibit distinct energy levels or sub-band characteristics, implying the absence of states between these levels, particularly near the zero energy point. When a system, such as a molecule, engages with its environment, like a DNA strand, energy transfer may occur between the two. This engagement can result in the emergence of novel energy states or modifications to the density of pre-existing states. If the interaction between the system and its environment generates new energy levels or substantially modifies the density of existing ones, it may be observed as peaks in the DOS. In Fig. <a href="#Fig3" class="usa-link">3</a>, the influence of protein-DNA interactions is evident, as indicated by the presence of peaks in the BS and DOS, with the quantity of additional peaks. The observed peaks in the DOS, or the additional bands in the BS, are a consequence of the interactions between protein molecules and DNA strands. Such interactions may give rise to the emergence of novel energy levels or modifications in the density of pre-existing levels, thereby producing peaks in the DOS. The precise positioning and morphology of these peaks are contingent upon the characteristics of the interaction between the protein and the DNA. The peaks resulting from protein-DNA interactions resemble those caused by impurities within a system. In fact, the presence of impurities in a material can generate localized states that profoundly influence the electronic structure and characteristics of the material. These localized states may occur around the impurity atom itself. The characteristics of these peaks, including their position, height, and width, are contingent upon various factors such as the nature of the impurity, its concentration, and its interactions with the surrounding host material. The presence of impurities within a material can result in the emergence of peaks in the DOS that are situated away from zero energy. These peaks, commonly known as impurity bands, originate from the localized energy levels established by the impurity atoms, which contrast with the band structure of the unadulterated material.</p>
<figure class="fig xbox font-sm" id="Fig3"><h3 class="obj_head">Fig. 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368152_41598_2025_15972_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/b41e0639d5c7/41598_2025_15972_Fig3_HTML.jpg" loading="lazy" id="MO3" height="679" width="779" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>On the left side, the BS is displayed, whereas the right side demonstrates the DOS for DNA interacting with each of the three conformations of the protein molecules, with each panel corresponding to a particular number of CPs. The BS is shown solely for the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq112"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8ecb3cf3d805/d33e1341.gif" loading="lazy" id="d33e1341" alt="Inline graphic"></span> conformation of the protein, with the temperature held constant at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq113"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8712523c2f3c/d33e1347.gif" loading="lazy" id="d33e1347" alt="Inline graphic"></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq114"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5bfa661456cf/d33e1353.gif" loading="lazy" id="d33e1353" alt="Inline graphic"></span>) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq115"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d99f27f5e9e9/d33e1359.gif" loading="lazy" id="d33e1359" alt="Inline graphic"></span>.</p></figcaption></figure><p id="Par15">As previously mentioned in relation to Fig. <a href="#Fig2" class="usa-link">2</a>, the individual components of the system, namely protein and DNA, exhibit semiconducting properties. However, as illustrated in all panels of Fig. <a href="#Fig3" class="usa-link">3</a>, the interaction between these two components results in a narrower band gap in panel (a) when compared to the DOS of the separate protein or DNA shown in Fig. <a href="#Fig2" class="usa-link">2</a>. Furthermore, panels (b) to (d) of Fig. <a href="#Fig3" class="usa-link">3</a> reveal a metallic behavior, characterized by the emergence of a peak at the zero point energies and diminish of the gap around the Fermi level. Thus, it can be concluded that the interaction between protein and DNA significantly modifies the electrical characteristics of the system, effectively enhancing its metallic properties. This transition from semiconducting to metallic behavior is attributed to the increased availability of electron channels in both components of the system. In order to concentrate more on this phenomenon, the band energies of the system are illustrated in Fig. <a href="#Fig4" class="usa-link">4</a>, depicting only the one lower and one upper bands relative to zero energy for various CP numbers. These correspond to a specific conformation of the protein chain at a constant temperature of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq120"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8712523c2f3c/d33e1424.gif" loading="lazy" id="d33e1424" alt="Inline graphic"></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq121"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5bfa661456cf/d33e1430.gif" loading="lazy" id="d33e1430" alt="Inline graphic"></span> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq122"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d99f27f5e9e9/d33e1436.gif" loading="lazy" id="d33e1436" alt="Inline graphic"></span> in all panels. It is observed that as the number of CPs increases, the gap around the Fermi level vanishes, resulting in metallic behavior. This phenomenon can be ascribed to the rising quantity of electron channels that enhance mobility within the system, which is further elucidated by the heightened interaction of impurities with DNA resulting from the increasing number of charge carriers. Furthermore, the presence of non-covalent (hydrogen) bonds in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq123"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d126173e404e/d33e1442.gif" loading="lazy" id="d33e1442" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq124"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/4936d1b20030/d33e1448.gif" loading="lazy" id="d33e1448" alt="Inline graphic"></span> conformations creates additional channels when compared to the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq125"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8ecb3cf3d805/d33e1454.gif" loading="lazy" id="d33e1454" alt="Inline graphic"></span> conformation. This additional bonding becomes more evident when the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq126"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/4936d1b20030/d33e1461.gif" loading="lazy" id="d33e1461" alt="Inline graphic"></span> conformation of the protein molecule interacts with DNA in three CPs, which incorporate hydrogen bonds in the CPs, resulting in a greater reduction of energy level intervals approaching the zero point energy. Enhancing the quantity of bonding interactions between two components of the system typically results in a reduction of the gap energy. This phenomenon occurs because intensified bonding interactions facilitate greater overlap of atomic orbitals, which in turn leads to a broader distribution of energy levels within the energy bands, consequently resulting in a diminished gap between them.</p>
<figure class="fig xbox font-sm" id="Fig4"><h3 class="obj_head">Fig. 4.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368152_41598_2025_15972_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/2061ff31fe89/41598_2025_15972_Fig4_HTML.jpg" loading="lazy" id="MO4" height="1106" width="643" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The BS of the protein-DNA system tends towards zero energy across multiple conformations of the protein molecule and various CPs. The figure illustrates a single lower and a single upper band in relation to zero energy for different quantities of CPs. These bands are associated with a specific conformation of the protein chain maintained at a constant temperature of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq130"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8712523c2f3c/d33e1512.gif" loading="lazy" id="d33e1512" alt="Inline graphic"></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq131"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5bfa661456cf/d33e1518.gif" loading="lazy" id="d33e1518" alt="Inline graphic"></span> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq132"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d99f27f5e9e9/d33e1524.gif" loading="lazy" id="d33e1524" alt="Inline graphic"></span>.</p></figcaption></figure><p id="Par16">The length of a DNA molecule can greatly influence the manner in which proteins interact with it. The sensitivity to the length of the DNA strand is also observable in the mechanical response of DNA to external forces<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Extended DNA molecules frequently demonstrate a greater affinity for protein binding, attributed to the heightened likelihood of protein-DNA interactions, a phenomenon referred to as the antenna effect. This effect occurs because longer DNA offers more chances for a protein to locate its specific binding site through diffusion along the DNA strand<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. In fact, proteins are capable of traversing the DNA molecule (1D diffusion) to identify their specific binding sites. Lengthier DNA facilitates more effective diffusion, thereby increasing the probability of locating the target<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Nevertheless, our focus here is on examining the effects of DNA length concerning four distinct CPs between protein and DNA in their interaction. Notably, the positions of these CPs remain constant, while the length of the DNA strand is variable. To examine the impact of DNA length on protein-DNA interactions, the DOS for the system is illustrated for three distinct values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq127"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/a39388894ef3/d33e1481.gif" loading="lazy" id="d33e1481" alt="Inline graphic"></span> at a fixed temperature of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq128"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8712523c2f3c/d33e1487.gif" loading="lazy" id="d33e1487" alt="Inline graphic"></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq129"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5bfa661456cf/d33e1494.gif" loading="lazy" id="d33e1494" alt="Inline graphic"></span> on Fig. <a href="#Fig5" class="usa-link">5</a>. This analysis incorporates four CPs between the protein and DNA, along with a defined conformation of the protein chain represented in each panel of Fig. <a href="#Fig5" class="usa-link">5</a>. This illustration demonstrates that the electronic properties of the protein-DNA system are affected by the size of the DNA, leading to a transformation of localized states within the structures into less distinct energy levels or sub-bands. Additionally, regardless of the length of DNA, the illustration reveals that the curves display symmetry about the zero-energy point, which is a result of the particle-hole symmetry present in the Hamiltonian framework. The figure indicates that the amplitude of peaks in the DOS diminishes with the increasing length of the DNA strand. This phenomenon occurs because the discrete energy levels characteristic of smaller systems, which are represented as sharp peaks in the DOS, tend to broaden and become less distinct as the system expands, with the spacing between these levels decreasing. Specifically, smaller systems demonstrate quantization, where energy is confined to particular levels, resulting in clearly defined peaks in the DOS. Conversely, as the system size increases, the energy levels converge, leading to a reduction in the visibility of quantization effects. Consequently, the total number of accessible energy states rises, contributing to a denser DOS. This implies that while the individual peaks may exhibit reduced height, the cumulative area beneath the DOS curve, indicative of the total number of states, increases. Ultimately, as the system enlarges, these discrete energy levels broaden and merge, causing the peaks to become less pronounced and resulting in a more continuous and smooth DOS. In all scenarios depicted across the three panels, it is evident that proteins generate identical peaks and alter the electronic properties of DNA, transitioning its behavior from semiconducting to metallic, irrespective of the DNA strand’s length. As the DNA size increases, the peak heights related to protein states in the DOS typically diminish. This occurs because, in larger DNA, the protein states tend to broaden and blend with the conduction band or other energy bands, resulting in a reduction of the localized DOS at particular energy levels. Conversely, in shorter DNA, the protein molecule establishes localized states at a specific energy level. Consequently, this leads to a pronounced peak in the DOS at that energy.</p>
<figure class="fig xbox font-sm" id="Fig5"><h3 class="obj_head">Fig. 5.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368152_41598_2025_15972_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d74c8aa278d7/41598_2025_15972_Fig5_HTML.jpg" loading="lazy" id="MO5" height="1095" width="786" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The influence of DNA length on protein-DNA interactions is demonstrated by the DOS for the system at two specific values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq133"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/1132e114bd61/d33e1538.gif" loading="lazy" id="d33e1538" alt="Inline graphic"></span> at a constant temperature of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq134"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8712523c2f3c/d33e1544.gif" loading="lazy" id="d33e1544" alt="Inline graphic"></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq135"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5bfa661456cf/d33e1550.gif" loading="lazy" id="d33e1550" alt="Inline graphic"></span>, where DNA strands engage with the various conformations of the protein molecule depicted in each panel.</p></figcaption></figure><p id="Par17">Figure <a href="#Fig6" class="usa-link">6</a> illustrates the influence of temperature on the DOS curves associated with protein-DNA interactions, as shown in panels (a), (b), and (c) for three distinct conformations of protein chains at two different temperatures, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq140"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8712523c2f3c/d33e1605.gif" loading="lazy" id="d33e1605" alt="Inline graphic"></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq141"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5bfa661456cf/d33e1611.gif" loading="lazy" id="d33e1611" alt="Inline graphic"></span>) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq142"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5e78592485a3/d33e1617.gif" loading="lazy" id="d33e1617" alt="Inline graphic"></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq143"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/1b8036f92f1d/d33e1623.gif" loading="lazy" id="d33e1623" alt="Inline graphic"></span>). An increase in temperature results in modifications to both the peak heights and positions across the three protein conformations interacting with DNA. This phenomenon arises from the alterations in the hopping terms within the protein molecules, which are temperature-dependent and reflected in the Hamiltonian of the protein chain. Consequently, changes in the hopping terms of the protein molecules modify the system’s spectrum, leading to variations in both the position and height of the peaks. It is evident that the DOS, which quantifies the number of accessible electronic states at a specific energy level, is intrinsically linked to the energy spectrum of the material. Any shifts or alterations in the energy spectrum will correspondingly affect the DOS. For instance, the emergence of a new energy level in the spectrum will manifest as an additional peak in the DOS, while a shift in an existing energy level will result in a corresponding displacement of the peak in the DOS. Furthermore, the peak height, indicative of the number of states, may also be influenced by changes in the energy spectrum. Furthermore, this phenomenon can be understood as a standard broadening of the fermionic distribution at elevated temperatures; the energy levels of electrons also increase, facilitating their movement throughout the system. As temperature rises, atomic vibrations intensify, leading to an increased average separation between atoms, a process referred to as thermal expansion. This increased interatomic distance reduces the potential barriers faced by electrons, consequently decreasing the energy required to transition electrons across the energy gap between the peaks. This leads to changes in both the heights and positions of the peaks.</p>
<figure class="fig xbox font-sm" id="Fig6"><h3 class="obj_head">Fig. 6.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368152_41598_2025_15972_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/c03d6f61e255/41598_2025_15972_Fig6_HTML.jpg" loading="lazy" id="MO6" height="1087" width="755" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The influence of temperature on the DOS curves related to protein-DNA interactions is illustrated in panels (<strong>a</strong>), (<strong>b</strong>), and (<strong>c</strong>) for three unique conformations of protein chains at two varying temperatures, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq136"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8712523c2f3c/d33e1573.gif" loading="lazy" id="d33e1573" alt="Inline graphic"></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq137"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5bfa661456cf/d33e1579.gif" loading="lazy" id="d33e1579" alt="Inline graphic"></span>) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq138"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5e78592485a3/d33e1586.gif" loading="lazy" id="d33e1586" alt="Inline graphic"></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq139"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/1b8036f92f1d/d33e1592.gif" loading="lazy" id="d33e1592" alt="Inline graphic"></span>).</p></figcaption></figure><p id="Par18">The stretching of the DNA strand refers to the modification of its structure, where the atoms are either separated or compressed. This phenomenon can arise through various mechanisms, including the introduction of impurities or defects, as well as the interaction of proteins with the DNA strand, which can lead to distortions or stretching of its configuration. These alterations may be reflected in changes to the electronic DOS, potentially resulting in a shift of spectral features within the DOS. Taking into account the effect of stretching on the DOS, we can manipulate it within the TB model by varying the hopping parameters in DNA and the coupling strength between protein molecules and the DNA strand. For this purpose, we selected two additional sets of hopping parameters: one set (set 2) consists of 0.8 of the values of the hopping parameters used to create previous figures (set 1), while the other set has been increased by a factor of 1.2 (set 3) compared to set 1. Figure <a href="#Fig7" class="usa-link">7</a> demonstrates the effect of reducing the hopping parameters in DNA and the coupling strength between protein molecules and the DNA strand (set 2) relative to set 1 by a factor of 0.8, as well as the effect of increasing the same hopping parameters (set 3) by a factor of 1.2 on the DOS curves related to the protein-DNA system for three conformations of protein molecules across three panels at fixed temperature <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq144"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8712523c2f3c/d33e1634.gif" loading="lazy" id="d33e1634" alt="Inline graphic"></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq145"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5bfa661456cf/d33e1640.gif" loading="lazy" id="d33e1640" alt="Inline graphic"></span> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq146"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d99f27f5e9e9/d33e1646.gif" loading="lazy" id="d33e1646" alt="Inline graphic"></span>. The variations in the hopping terms, whether they are increasing or decreasing, result in alterations in the DOS by shifting and redistributing spectral features within the DOS. These changes can stem from local structural distortions, such as stretching, which introduce localized energy levels into the energy band. Consequently, this leads to an increase or decrease in the DOS at specific energy levels, resulting in a shift of the peaks in the DOS. This shift alters the overall shape of the DOS when comparing the unstretched DNA sequence to the stretched DNA strand. Therefore, the interaction between protein molecules and the unstretched DNA sequence, as opposed to the stretched DNA strand, can be distinctly influenced by the DOS within the TB model. This has biological implications, including gene regulation, DNA repair, and the efficiency of protein-DNA interactions<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig7"><h3 class="obj_head">Fig. 7.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368152_41598_2025_15972_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/2fc5e20bffb1/41598_2025_15972_Fig7_HTML.jpg" loading="lazy" id="MO7" height="1096" width="771" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The impact of various hopping parameters resulting from the elongation of DNA sequences on the DOS of the protein-DNA system is analyzed for three unique conformations of protein chains, with each conformation illustrated in an individual panel. The temperature is maintained at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq147"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8712523c2f3c/d33e1663.gif" loading="lazy" id="d33e1663" alt="Inline graphic"></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq148"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/5bfa661456cf/d33e1669.gif" loading="lazy" id="d33e1669" alt="Inline graphic"></span> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq149"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d99f27f5e9e9/d33e1675.gif" loading="lazy" id="d33e1675" alt="Inline graphic"></span>.</p></figcaption></figure></section><section id="Sec5"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par19">In summary, the TB Hamiltonian model was employed alongside the Green’s function methodology to explore the protein-DNA interaction across three unique protein chain conformations, referred to as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq150"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/8ecb3cf3d805/d33e1687.gif" loading="lazy" id="d33e1687" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq151"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/d126173e404e/d33e1693.gif" loading="lazy" id="d33e1693" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq152"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/46fa/12368152/4936d1b20030/d33e1699.gif" loading="lazy" id="d33e1699" alt="Inline graphic"></span>, within the context of BS and DOS analysis. Under constant temperature and a fixed number of sub-sites within the DNA strand, the BS and DOS exhibited the flat energy dispersion curves for both the protein molecules and the DNA strand independently, which can be attributed to the existence of localized states within these structures. Indeed, the confines of these molecules restrict particle mobility, resulting in quantized energy levels. This indicates that only specific discrete energy values are allowed for the particles within the system. Furthermore, these finite structures exhibited characteristics akin to those of semiconductors, which are associated with the existence of a forbidden zone near zero energy, with DNA showing a more pronounced energy gap. The interaction between the two components of the system introduces novel effects in the protein-DNA system that are contingent upon the number of CPs between the protein and DNA. An increase in CPs leads to a transition from semiconducting to metallic characteristics, which is also influenced by the contribution of hydrogen bonds from the mutant protein at these CPs. Such interactions may lead to the formation of new energy levels or alterations in the density of existing levels, consequently generating peaks in the DOS. The electronic properties of the protein-DNA system are influenced by the size of the DNA, resulting in a transformation of localized states within the structures into less defined energy levels as the length of the DNA strand increases. This phenomenon arises because the discrete energy levels typical of smaller systems, depicted as sharp peaks in the DOS, tend to broaden and become less defined as the system enlarges, with the spacing between these levels diminishing. The effect of temperature on the BS and DOS, irrespective of the type of configuration, results in changes in both the peak heights and positions due to modifications in the hopping terms within the protein molecules and alterations in the system’s spectrum. This occurrence can be interpreted as a conventional broadening of the fermionic distribution at higher temperatures; the energy levels of electrons also rise, promoting their movement throughout the system. The fluctuations in the hopping parameters, whether they are on the rise or decline, caused by the stretching effects of DNA due to the presence of protein molecules, lead to changes by shifting and redistributing spectral characteristics within the DOS. The interaction between protein and DNA is expected to directly affect the electronic properties of DNA, which vary across different protein conformations, thereby opening new research avenues with significant biological implications.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>The authors have contributed, in equal measure, to the study’s conception and findings. Material preparation, data collection and analysis were performed by Hamze Mousavi and Ronak Emami. The first draft of the manuscript was written by Hamze Mouavi and other authors commented on previous versions of the manuscript. All authors read and approved the final manuscript. Formulation and formal analysis; coding; results interpretation and writing the original draft were performed by Hamze Mousavi. Coding; results interpretation; and editing were performed by Ronak Emami.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study is in no way funded by any organization and is an independent investigation carried out exclusively by the author of the study.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets generated during and/or analysed during the current study are available from the corresponding author on reasonable request.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar2"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par25">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Watson, J. D. &amp; Crick, F. H. C. Molecular structure of nucleic acids: A structure for deoxyribose nucleic acid. <em>Nature</em><strong>171</strong>, 737 (1953).
</cite> [<a href="https://doi.org/10.1038/171737a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/13054692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Watson,%20J.%20D.%20&amp;%20Crick,%20F.%20H.%20C.%20Molecular%20structure%20of%20nucleic%20acids:%20A%20structure%20for%20deoxyribose%20nucleic%20acid.%20Nature171,%20737%20(1953)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Mousavi, H., Khodadadi, J. &amp; Grabowski, M. Electronic properties of long DNA nanowires in dry and wet conditions. <em>Solid State Commun.</em><strong>222</strong>, 42 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mousavi,%20H.,%20Khodadadi,%20J.%20&amp;%20Grabowski,%20M.%20Electronic%20properties%20of%20long%20DNA%20nanowires%20in%20dry%20and%20wet%20conditions.%20Solid%20State%20Commun.222,%2042%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Mousavi, H. &amp; Grabowski, M. Nonlinear electron transport across short DNA segment between graphene leads. <em>Solid State Commun.</em><strong>279</strong>, 30 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mousavi,%20H.%20&amp;%20Grabowski,%20M.%20Nonlinear%20electron%20transport%20across%20short%20DNA%20segment%20between%20graphene%20leads.%20Solid%20State%20Commun.279,%2030%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Mousavi, H., Jalilvand, S., Sani, S. S., Hartman, J. A. L. &amp; Grabowski, M. Electronic properties of different configurations of double-strand DNA-like nanowires. <em>Solid State Commun.</em><strong>319</strong>, 113974 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mousavi,%20H.,%20Jalilvand,%20S.,%20Sani,%20S.%20S.,%20Hartman,%20J.%20A.%20L.%20&amp;%20Grabowski,%20M.%20Electronic%20properties%20of%20different%20configurations%20of%20double-strand%20DNA-like%20nanowires.%20Solid%20State%20Commun.319,%20113974%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Jalilvand, S., Sepahvand, R. &amp; Mousavi, H. Electronic behavior of randomly dislocated RNA and DNA nanowires: A multi-model approach. <em>Euro. Phys. J. Plus</em><strong>137</strong>, 928 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Jalilvand,%20S.,%20Sepahvand,%20R.%20&amp;%20Mousavi,%20H.%20Electronic%20behavior%20of%20randomly%20dislocated%20RNA%20and%20DNA%20nanowires:%20A%20multi-model%20approach.%20Euro.%20Phys.%20J.%20Plus137,%20928%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Mousavi, H. Electron transport through protein chains interfacing nanoribbon and nanotube leads. <em>Solid State Commun.</em><strong>404</strong>, 116057 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mousavi,%20H.%20Electron%20transport%20through%20protein%20chains%20interfacing%20nanoribbon%20and%20nanotube%20leads.%20Solid%20State%20Commun.404,%20116057%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Mousavi, H. Quantum transport in protein chains. <em>Sci. Rep.</em><strong>15</strong>, 26703 (2025).
</cite> [<a href="https://doi.org/10.1038/s41598-025-11140-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12284114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40695897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mousavi,%20H.%20Quantum%20transport%20in%20protein%20chains.%20Sci.%20Rep.15,%2026703%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Mousavi, H. &amp; Emami, R. Protein chains in tight-binding framework. <em>J. Mol. Model.</em><strong>31</strong>, 223 (2025).
</cite> [<a href="https://doi.org/10.1007/s00894-025-06450-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40728749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mousavi,%20H.%20&amp;%20Emami,%20R.%20Protein%20chains%20in%20tight-binding%20framework.%20J.%20Mol.%20Model.31,%20223%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Keren, K., Berman, R. S., Buchstab, E., Sivan, U. &amp; Braun, E. DNA-templated carbon nanotube field-effect transistor. <em>Science</em><strong>302</strong>, 1380 (2003).
</cite> [<a href="https://doi.org/10.1126/science.1091022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14631035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keren,%20K.,%20Berman,%20R.%20S.,%20Buchstab,%20E.,%20Sivan,%20U.%20&amp;%20Braun,%20E.%20DNA-templated%20carbon%20nanotube%20field-effect%20transistor.%20Science302,%201380%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Cao, X., Chen, C. &amp; Zhu, Q. Biosensors based on functional nucleic acids and isothermal amplification techniques. <em>Talanta</em><strong>253</strong>, 123977 (2023).
</cite> [<a href="https://doi.org/10.1016/j.talanta.2022.123977" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36201957/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cao,%20X.,%20Chen,%20C.%20&amp;%20Zhu,%20Q.%20Biosensors%20based%20on%20functional%20nucleic%20acids%20and%20isothermal%20amplification%20techniques.%20Talanta253,%20123977%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Fu, Z., Lu, Y.-C. &amp; Lai, J. J. Recent advances in biosensors for nucleic acid and exosome detection. <em>Chonnam. Med. J.</em><strong>55</strong>, 86 (2019).
</cite> [<a href="https://doi.org/10.4068/cmj.2019.55.2.86" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6536430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31161120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fu,%20Z.,%20Lu,%20Y.-C.%20&amp;%20Lai,%20J.%20J.%20Recent%20advances%20in%20biosensors%20for%20nucleic%20acid%20and%20exosome%20detection.%20Chonnam.%20Med.%20J.55,%2086%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Gunasekaran, B. M., Srinivasan, S., Ezhilan, M. &amp; Nesakumar, N. Nucleic acid-based electrochemical biosensors. <em>Clin. Chim. Acta</em><strong>559</strong>, 119715 (2024).
</cite> [<a href="https://doi.org/10.1016/j.cca.2024.119715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38735514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gunasekaran,%20B.%20M.,%20Srinivasan,%20S.,%20Ezhilan,%20M.%20&amp;%20Nesakumar,%20N.%20Nucleic%20acid-based%20electrochemical%20biosensors.%20Clin.%20Chim.%20Acta559,%20119715%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Bhopatkar, A. A., Uversky, V. N. &amp; Rangachari, V. Disorder and cysteines in proteins: A design for orchestration of conformational see-saw and modulatory functions. <em>Prog. Mol. Biol. Transl. Sci.</em><strong>174</strong>, 331 (2020).
</cite> [<a href="https://doi.org/10.1016/bs.pmbts.2020.06.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8048112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32828470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bhopatkar,%20A.%20A.,%20Uversky,%20V.%20N.%20&amp;%20Rangachari,%20V.%20Disorder%20and%20cysteines%20in%20proteins:%20A%20design%20for%20orchestration%20of%20conformational%20see-saw%20and%20modulatory%20functions.%20Prog.%20Mol.%20Biol.%20Transl.%20Sci.174,%20331%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Douglas, S. M. et al. Self-assembly of DNA into nanoscale three-dimensional shapes. <em>Nature</em><strong>459</strong>, 414 (2009).
</cite> [<a href="https://doi.org/10.1038/nature08016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2688462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19458720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Douglas,%20S.%20M.%20et%20al.%20Self-assembly%20of%20DNA%20into%20nanoscale%20three-dimensional%20shapes.%20Nature459,%20414%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Rajendran, A., Endo, M. &amp; Sugiyama, H. Single-molecule analysis using DNA origami. <em>Angew. Chem. Int. Ed. Engl.</em><strong>51</strong>, 874 (2012).
</cite> [<a href="https://doi.org/10.1002/anie.201102113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22121063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rajendran,%20A.,%20Endo,%20M.%20&amp;%20Sugiyama,%20H.%20Single-molecule%20analysis%20using%20DNA%20origami.%20Angew.%20Chem.%20Int.%20Ed.%20Engl.51,%20874%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Hart, W.E. &amp; Newman, A. Handbook of Computational Molecular Biology, Ch. 30, Edited by S. Aluru, 1st Ed. (2005).</cite>
</li>
<li id="CR17">
<span class="label">17.</span><cite>Dill, K. A. et al. Principles of protein folding: A perspective from simple exact models. <em>Prot. Sci.</em><strong>4</strong>, 561 (1995).</cite> [<a href="https://doi.org/10.1002/pro.5560040401" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2143098/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7613459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dill,%20K.%20A.%20et%20al.%20Principles%20of%20protein%20folding:%20A%20perspective%20from%20simple%20exact%20models.%20Prot.%20Sci.4,%20561%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Redding, S. &amp; Greene, E. C. How do proteins locate specific targets in DNA?. <em>Chem. Phys. Lett.</em><strong>570</strong>, 1 (2013).</cite> [<a href="https://doi.org/10.1016/j.cplett.2013.03.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3810971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24187380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Redding,%20S.%20&amp;%20Greene,%20E.%20C.%20How%20do%20proteins%20locate%20specific%20targets%20in%20DNA?.%20Chem.%20Phys.%20Lett.570,%201%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Rice, P. A. &amp; Correll, C. C. <em>Protein-Nucleic Acid Interactions: Structural Biology</em> (The Royal Society of Chemistry, Cambridge, 2008).</cite> [<a href="https://scholar.google.com/scholar_lookup?Rice,%20P.%20A.%20&amp;%20Correll,%20C.%20C.%20Protein-Nucleic%20Acid%20Interactions:%20Structural%20Biology%20(The%20Royal%20Society%20of%20Chemistry,%20Cambridge,%202008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Murugan, R. DNA-protein interactions under random jump conditions. <em>Phys. Rev. E</em><strong>69</strong>, 011911 (2004).</cite> [<a href="https://doi.org/10.1103/PhysRevE.69.011911" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14995651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murugan,%20R.%20DNA-protein%20interactions%20under%20random%20jump%20conditions.%20Phys.%20Rev.%20E69,%20011911%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Murugan, R. Generalized theory of site-specific DNA-protein interactions. <em>Phys. Rev. E</em><strong>76</strong>, 011901 (2007).</cite> [<a href="https://doi.org/10.1103/PhysRevE.76.011901" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17677488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murugan,%20R.%20Generalized%20theory%20of%20site-specific%20DNA-protein%20interactions.%20Phys.%20Rev.%20E76,%20011901%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Murugan, R. Packaging effects on site-specific DNA-protein interactions. <em>Phys. Rev. E</em><strong>79</strong>, 061920 (2009).</cite> [<a href="https://doi.org/10.1103/PhysRevE.79.061920" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19658537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murugan,%20R.%20Packaging%20effects%20on%20site-specific%20DNA-protein%20interactions.%20Phys.%20Rev.%20E79,%20061920%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Vasumathi, V. &amp; Daniel, M. Base-pair opening and bubble transport in a DNA double helix induced by a protein molecule in a viscous medium. <em>Phys. Rev. E</em><strong>80</strong>, 061904 (2009).</cite> [<a href="https://doi.org/10.1103/PhysRevE.80.061904" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20365187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vasumathi,%20V.%20&amp;%20Daniel,%20M.%20Base-pair%20opening%20and%20bubble%20transport%20in%20a%20DNA%20double%20helix%20induced%20by%20a%20protein%20molecule%20in%20a%20viscous%20medium.%20Phys.%20Rev.%20E80,%20061904%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>D’Acunto, M. Protein-DNA target search relies on quantum walk. <em>Biosyst.</em><strong>201</strong>, 104340 (2021).</cite> [<a href="https://doi.org/10.1016/j.biosystems.2020.104340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33387562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?D%E2%80%99Acunto,%20M.%20Protein-DNA%20target%20search%20relies%20on%20quantum%20walk.%20Biosyst.201,%20104340%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Kamagata, K., Itoh, Y. &amp; Subekti, D. G. R. How p53 molecules solve the target DNA search problem: A review. <em>Int. J. Mol. Sci.</em><strong>21</strong>, 1031 (2020).
</cite> [<a href="https://doi.org/10.3390/ijms21031031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7037437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32033163/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kamagata,%20K.,%20Itoh,%20Y.%20&amp;%20Subekti,%20D.%20G.%20R.%20How%20p53%20molecules%20solve%20the%20target%20DNA%20search%20problem:%20A%20review.%20Int.%20J.%20Mol.%20Sci.21,%201031%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Monico, C., Capitanio, M., Belcastro, G., Vanzi, F. &amp; Pavone, F. S. Optical methods to study protein-DNA interactions in vitro and in living cells at the single-molecule level. <em>Int. J. Mol. Sci.</em><strong>14</strong>, 3961 (2013).
</cite> [<a href="https://doi.org/10.3390/ijms14023961" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3588080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23429188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Monico,%20C.,%20Capitanio,%20M.,%20Belcastro,%20G.,%20Vanzi,%20F.%20&amp;%20Pavone,%20F.%20S.%20Optical%20methods%20to%20study%20protein-DNA%20interactions%20in%20vitro%20and%20in%20living%20cells%20at%20the%20single-molecule%20level.%20Int.%20J.%20Mol.%20Sci.14,%203961%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Bonnet, I. et al. Sliding and jumping of single EcoRV restriction enzymes on noncognate DNA. <em>Nucleic Acids Res.</em><strong>36</strong>, 4118 (2008).
</cite> [<a href="https://doi.org/10.1093/nar/gkn376" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2475641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18544605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bonnet,%20I.%20et%20al.%20Sliding%20and%20jumping%20of%20single%20EcoRV%20restriction%20enzymes%20on%20noncognate%20DNA.%20Nucleic%20Acids%20Res.36,%204118%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Sarmentoa, R. G., Frazaob, N. F. &amp; Macedo-Filhoc, A. Electronic transport on the spatial structure of the protein: Three-dimensional lattice model. <em>Phys. Lett. A</em><strong>381</strong>, 276 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sarmentoa,%20R.%20G.,%20Frazaob,%20N.%20F.%20&amp;%20Macedo-Filhoc,%20A.%20Electronic%20transport%20on%20the%20spatial%20structure%20of%20the%20protein:%20Three-dimensional%20lattice%20model.%20Phys.%20Lett.%20A381,%20276%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Zoli, M. Twist-stretch profiles of DNA chains. <em>J. Phys. Condens. Matter</em><strong>29</strong>, 225101 (2017).
</cite> [<a href="https://doi.org/10.1088/1361-648X/aa6c50" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28394255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zoli,%20M.%20Twist-stretch%20profiles%20of%20DNA%20chains.%20J.%20Phys.%20Condens.%20Matter29,%20225101%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Xu, W., Dunlap, D. &amp; Finzi, L. Energetics of twisted DNA topologies. <em>Biophys. J.</em><strong>120</strong>, 3242 (2021).
</cite> [<a href="https://doi.org/10.1016/j.bpj.2021.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8391063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33974883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20W.,%20Dunlap,%20D.%20&amp;%20Finzi,%20L.%20Energetics%20of%20twisted%20DNA%20topologies.%20Biophys.%20J.120,%203242%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Strick, T., Allemand, J.-F., Croquette, V. &amp; Bensimon, D. Twisting and stretching single DNA molecules. <em>Prog. Biophys. Mol. Biol.</em><strong>74</strong>, 115 (2000).
</cite> [<a href="https://doi.org/10.1016/s0079-6107(00)00018-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11106809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Strick,%20T.,%20Allemand,%20J.-F.,%20Croquette,%20V.%20&amp;%20Bensimon,%20D.%20Twisting%20and%20stretching%20single%20DNA%20molecules.%20Prog.%20Biophys.%20Mol.%20Biol.74,%20115%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Liu, X., Yu, E., Zhao, Q., Han, H. &amp; Li, Q. Enzymes as green and sustainable tools for DNA data storage. <em>Chem. Commun.</em><strong>61</strong>, 2891 (2025).</cite> [<a href="https://doi.org/10.1039/d4cc06351a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39834292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20X.,%20Yu,%20E.,%20Zhao,%20Q.,%20Han,%20H.%20&amp;%20Li,%20Q.%20Enzymes%20as%20green%20and%20sustainable%20tools%20for%20DNA%20data%20storage.%20Chem.%20Commun.61,%202891%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Mann, M., Kumar, C., Zeng, W.-F. &amp; Strauss, M. T. Artificial intelligence for proteomics and biomarker discovery. <em>Cell Syst.</em><strong>12</strong>, 759 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cels.2021.06.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34411543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mann,%20M.,%20Kumar,%20C.,%20Zeng,%20W.-F.%20&amp;%20Strauss,%20M.%20T.%20Artificial%20intelligence%20for%20proteomics%20and%20biomarker%20discovery.%20Cell%20Syst.12,%20759%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Sim, J., Kim, D., Kim, B., Choi, J. &amp; Lee, J. Recent advances in AI-driven protein-ligand interaction predictions. <em>Curr. Opin. Struct. Biol.</em><strong>92</strong>, 103020 (2025).
</cite> [<a href="https://doi.org/10.1016/j.sbi.2025.103020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39999605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sim,%20J.,%20Kim,%20D.,%20Kim,%20B.,%20Choi,%20J.%20&amp;%20Lee,%20J.%20Recent%20advances%20in%20AI-driven%20protein-ligand%20interaction%20predictions.%20Curr.%20Opin.%20Struct.%20Biol.92,%20103020%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Suh, D., Lee, J. W., Choi, S. &amp; Lee, Y. Recent applications of deep learning methods on evolution- and contact-based protein structure prediction. <em>Int. J. Mol. Sci.</em><strong>22</strong>, 6032 (2021).
</cite> [<a href="https://doi.org/10.3390/ijms22116032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8199773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34199677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suh,%20D.,%20Lee,%20J.%20W.,%20Choi,%20S.%20&amp;%20Lee,%20Y.%20Recent%20applications%20of%20deep%20learning%20methods%20on%20evolution-%20and%20contact-based%20protein%20structure%20prediction.%20Int.%20J.%20Mol.%20Sci.22,%206032%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Qu, C. &amp; Du, H. Advance technologies for DNA-protein interactions and future research prospect. <em>Crop Des.</em><strong>4</strong>, 100082 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?Qu,%20C.%20&amp;%20Du,%20H.%20Advance%20technologies%20for%20DNA-protein%20interactions%20and%20future%20research%20prospect.%20Crop%20Des.4,%20100082%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Kimijima, J., Inagawa, A., Miyagawa, A., Nasuno, E. &amp; Uehara, N. Probing the interaction between biomolecules under sub-zero temperature conditions by electrophoresis in ice grain boundaries. <em>Anal. Chim. Acta.</em><strong>1311</strong>, 342713 (2024).
</cite> [<a href="https://doi.org/10.1016/j.aca.2024.342713" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38816152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kimijima,%20J.,%20Inagawa,%20A.,%20Miyagawa,%20A.,%20Nasuno,%20E.%20&amp;%20Uehara,%20N.%20Probing%20the%20interaction%20between%20biomolecules%20under%20sub-zero%20temperature%20conditions%20by%20electrophoresis%20in%20ice%20grain%20boundaries.%20Anal.%20Chim.%20Acta.1311,%20342713%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Foguel, D. &amp; Silva, J. L. Cold denaturation of a repressor-operator complex: the role of entropy in protein-DNA recognition. <em>PNAS</em><strong>91</strong>, 8244 (1994).
</cite> [<a href="https://doi.org/10.1073/pnas.91.17.8244" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC44582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8058788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Foguel,%20D.%20&amp;%20Silva,%20J.%20L.%20Cold%20denaturation%20of%20a%20repressor-operator%20complex:%20the%20role%20of%20entropy%20in%20protein-DNA%20recognition.%20PNAS91,%208244%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Zheng, J., Guo, N., Huang, Y., Guo, X. &amp; Wagner, A. High temperature delays and low temperature accelerates evolution of a new protein phenotype. <em>Nat. Commun.</em><strong>15</strong>, 2495 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-46332-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10980763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38553445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20J.,%20Guo,%20N.,%20Huang,%20Y.,%20Guo,%20X.%20&amp;%20Wagner,%20A.%20High%20temperature%20delays%20and%20low%20temperature%20accelerates%20evolution%20of%20a%20new%20protein%20phenotype.%20Nat.%20Commun.15,%202495%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Privalov, P. L. et al. What drives proteins into the major or minor grooves of DNA?. <em>J. Mol. Biol.</em><strong>365</strong>, 1 (2007).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2006.09.059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1934558/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17055530/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Privalov,%20P.%20L.%20et%20al.%20What%20drives%20proteins%20into%20the%20major%20or%20minor%20grooves%20of%20DNA?.%20J.%20Mol.%20Biol.365,%201%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Hauschild, K. E. et al. Temperature-sensitive protein-DNA dimerizers. <em>PNAS</em><strong>102</strong>, 5008 (2005).
</cite> [<a href="https://doi.org/10.1073/pnas.0501289102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC555971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15781856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hauschild,%20K.%20E.%20et%20al.%20Temperature-sensitive%20protein-DNA%20dimerizers.%20PNAS102,%205008%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Muthusamy, M., Kim, J.-H., Kim, J. A. &amp; Lee, S.-I. Plant RNA binding proteins as critical modulators in drought, high salinity, heat, and cold stress responses: An updated overview. <em>Int. J. Mol. Sci.</em><strong>22</strong>, 6731 (2021).
</cite> [<a href="https://doi.org/10.3390/ijms22136731" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8269355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34201749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muthusamy,%20M.,%20Kim,%20J.-H.,%20Kim,%20J.%20A.%20&amp;%20Lee,%20S.-I.%20Plant%20RNA%20binding%20proteins%20as%20critical%20modulators%20in%20drought,%20high%20salinity,%20heat,%20and%20cold%20stress%20responses:%20An%20updated%20overview.%20Int.%20J.%20Mol.%20Sci.22,%206731%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Gutin, A. M., Abkevich, V. &amp; Shakhnovich, E. A protein engineering analysis of the transition state for protein folding: Simulation in the lattice model. <em>Fold. Des.</em><strong>3</strong>, 183 (1998).
</cite> [<a href="https://doi.org/10.1016/S1359-0278(98)00026-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9562547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gutin,%20A.%20M.,%20Abkevich,%20V.%20&amp;%20Shakhnovich,%20E.%20A%20protein%20engineering%20analysis%20of%20the%20transition%20state%20for%20protein%20folding:%20Simulation%20in%20the%20lattice%20model.%20Fold.%20Des.3,%20183%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Liao, X. et al. Repetitive DNA sequence detection and its role in the human genome. <em>Commun. Biol.</em><strong>6</strong>, 954 (2023).
</cite> [<a href="https://doi.org/10.1038/s42003-023-05322-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10509279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37726397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liao,%20X.%20et%20al.%20Repetitive%20DNA%20sequence%20detection%20and%20its%20role%20in%20the%20human%20genome.%20Commun.%20Biol.6,%20954%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Economou, E. N. <em>Green’s Functions in Quantum Physics</em> 3rd edn. (Springer-Verlag, 2006).</cite> [<a href="https://scholar.google.com/scholar_lookup?Economou,%20E.%20N.%20Green%E2%80%99s%20Functions%20in%20Quantum%20Physics%203rd%20edn.%20(Springer-Verlag,%202006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Paez, C. J., Schulz, P. A., Wilson, N. R. &amp; Römer, R. A. Robust signatures in the current-voltage characteristics of DNA molecules oriented between two graphene nanoribbon electrodes. <em>N. J. Phys.</em><strong>14</strong>, 093049 (2012).</cite> [<a href="https://scholar.google.com/scholar_lookup?Paez,%20C.%20J.,%20Schulz,%20P.%20A.,%20Wilson,%20N.%20R.%20&amp;%20R%C3%B6mer,%20R.%20A.%20Robust%20signatures%20in%20the%20current-voltage%20characteristics%20of%20DNA%20molecules%20oriented%20between%20two%20graphene%20nanoribbon%20electrodes.%20N.%20J.%20Phys.14,%20093049%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Pandey, P., Hasnain, S. &amp; Ahmad, S. Protein-DNA interactions, encyclopedia of bioinformatics and computational. <em>Biology</em><strong>2</strong>, 142 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Pandey,%20P.,%20Hasnain,%20S.%20&amp;%20Ahmad,%20S.%20Protein-DNA%20interactions,%20encyclopedia%20of%20bioinformatics%20and%20computational.%20Biology2,%20142%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Luscombe, N. M., Laskowski, R. A. &amp; Thornton, J. M. Amino acid-base interactions: A three-dimensional analysis of protein-DNA interactions at an atomic level. <em>Nucleic Acids Res.</em><strong>29</strong>, 2860 (2001).
</cite> [<a href="https://doi.org/10.1093/nar/29.13.2860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC55782/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11433033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luscombe,%20N.%20M.,%20Laskowski,%20R.%20A.%20&amp;%20Thornton,%20J.%20M.%20Amino%20acid-base%20interactions:%20A%20three-dimensional%20analysis%20of%20protein-DNA%20interactions%20at%20an%20atomic%20level.%20Nucleic%20Acids%20Res.29,%202860%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Rico, F., Rigato, A., Picas, L. &amp; Scheuring, S. Mechanics of proteins with a focus on atomic force microscopy. <em>J. Nanobiotechnol.</em><strong>11</strong>, S3 (2013).</cite> [<a href="https://doi.org/10.1186/1477-3155-11-S1-S3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4029730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24565326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rico,%20F.,%20Rigato,%20A.,%20Picas,%20L.%20&amp;%20Scheuring,%20S.%20Mechanics%20of%20proteins%20with%20a%20focus%20on%20atomic%20force%20microscopy.%20J.%20Nanobiotechnol.11,%20S3%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Beratan, D. N. Why are DNA and protein electron transfer so different?. <em>Annu. Rev. Phys. Chem.</em><strong>70</strong>, 71 (2019).
</cite> [<a href="https://doi.org/10.1146/annurev-physchem-042018-052353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6591729/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30726186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beratan,%20D.%20N.%20Why%20are%20DNA%20and%20protein%20electron%20transfer%20so%20different?.%20Annu.%20Rev.%20Phys.%20Chem.70,%2071%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Singh, A. &amp; Singh, N. Pulling DNA: The effect of chain length on the mechanical stability of DNA chain. <em>Macromol. Symp.</em><strong>357</strong>, 64 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Singh,%20A.%20&amp;%20Singh,%20N.%20Pulling%20DNA:%20The%20effect%20of%20chain%20length%20on%20the%20mechanical%20stability%20of%20DNA%20chain.%20Macromol.%20Symp.357,%2064%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Shimamoto, N. et al. Dependence of DNA length on binding affinity between TrpR and trpO of DNA. <em>Sci. Rep.</em><strong>10</strong>, 15624 (2020).
</cite> [<a href="https://doi.org/10.1038/s41598-020-71598-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7518442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32973254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shimamoto,%20N.%20et%20al.%20Dependence%20of%20DNA%20length%20on%20binding%20affinity%20between%20TrpR%20and%20trpO%20of%20DNA.%20Sci.%20Rep.10,%2015624%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>King, G. A., Peterman, E. J. G. &amp; Wuite, G. J. L. Unravelling the structural plasticity of stretched DNA under torsional constraint. <em>Nat. Commun.</em><strong>7</strong>, 11810 (2016).
</cite> [<a href="https://doi.org/10.1038/ncomms11810" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4897764/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27263853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?King,%20G.%20A.,%20Peterman,%20E.%20J.%20G.%20&amp;%20Wuite,%20G.%20J.%20L.%20Unravelling%20the%20structural%20plasticity%20of%20stretched%20DNA%20under%20torsional%20constraint.%20Nat.%20Commun.7,%2011810%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated during and/or analysed during the current study are available from the corresponding author on reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-15972-z"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_15972.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368152/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368152/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368152%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368152/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368152/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368152/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835689/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368152/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835689/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368152/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368152/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="XX5qZfbSi9XQBwvQfJeZRFKxbZNUrLmCp4G8yQ1fjdmnkkczDjSKwilMVUUnS6lW">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
