
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Complete mitochondrial genomes of the Prototheca genus: comparative genomics and evolutionary insights - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15253C1E8AF24583053C1E001552E743.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcgeno">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369231/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Genomics">
<meta name="citation_title" content="Complete mitochondrial genomes of the Prototheca genus: comparative genomics and evolutionary insights">
<meta name="citation_author" content="Daniel Wibberg">
<meta name="citation_author_institution" content="Center for Biotechnology (CeBiTec), Genome Research of Industrial Microorganisms, Bielefeld University, Bielefeld, Germany">
<meta name="citation_author_institution" content="Institute of Bio- and Geosciences IBG-5 - Computational Metagenomics, Forschungszentrum Jülich GmbH, Bielefeld, Germany">
<meta name="citation_author" content="Zofia Bakuła">
<meta name="citation_author_institution" content="Department of Medical Microbiology, Institute of Microbiology, Faculty of Biology, University of Warsaw, Warsaw, Poland">
<meta name="citation_author" content="Iván García-Cunchillos">
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, Faculty of Biology, Biological and Chemical Research Centre, University of Warsaw, Warsaw, Poland">
<meta name="citation_author" content="Jan Gawor">
<meta name="citation_author_institution" content="DNA Sequencing and Synthesis Facility, Institute of Biochemistry and Biophysics, Polish Academy of Sciences, Warsaw, Poland">
<meta name="citation_author" content="Robert Gromadka">
<meta name="citation_author_institution" content="DNA Sequencing and Synthesis Facility, Institute of Biochemistry and Biophysics, Polish Academy of Sciences, Warsaw, Poland">
<meta name="citation_author" content="Anna Karnkowska">
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, Faculty of Biology, Biological and Chemical Research Centre, University of Warsaw, Warsaw, Poland">
<meta name="citation_author" content="Jacek Bielecki">
<meta name="citation_author_institution" content="Department of Medical Microbiology, Institute of Microbiology, Faculty of Biology, University of Warsaw, Warsaw, Poland">
<meta name="citation_author" content="Tomasz Jagielski">
<meta name="citation_author_institution" content="Department of Medical Microbiology, Institute of Microbiology, Faculty of Biology, University of Warsaw, Warsaw, Poland">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="766">
<meta name="citation_doi" content="10.1186/s12864-025-11952-9">
<meta name="citation_pmid" content="40836223">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369231/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369231/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369231/pdf/12864_2025_Article_11952.pdf">
<meta name="description" content="Prototheca (Chlorophyta: Trebouxiophyceae) is a genus of non-photosynthetic microalgae that causes increasingly frequent infections in both humans and animals, collectively referred to as protothecosis The genetic landscape of the Prototheca algae ...">
<meta name="og:title" content="Complete mitochondrial genomes of the Prototheca genus: comparative genomics and evolutionary insights">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Prototheca (Chlorophyta: Trebouxiophyceae) is a genus of non-photosynthetic microalgae that causes increasingly frequent infections in both humans and animals, collectively referred to as protothecosis The genetic landscape of the Prototheca algae ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369231/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12369231">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12864-025-11952-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12864_2025_Article_11952.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369231%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12369231/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12369231/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369231/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcgeno.png" alt="BMC Genomics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Genomics" title="Link to BMC Genomics" shape="default" href="https://bmcgenomics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Genomics</button></div>. 2025 Aug 21;26:766. doi: <a href="https://doi.org/10.1186/s12864-025-11952-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12864-025-11952-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Genomics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Genomics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Complete mitochondrial genomes of the <em>Prototheca</em> genus: comparative genomics and evolutionary insights</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wibberg%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Daniel Wibberg</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Daniel Wibberg</span></h3>
<div class="p">
<sup>1</sup>Center for Biotechnology (CeBiTec), Genome Research of Industrial Microorganisms, Bielefeld University, Bielefeld, Germany </div>
<div class="p">
<sup>2</sup>Institute of Bio- and Geosciences IBG-5 - Computational Metagenomics, Forschungszentrum Jülich GmbH, Bielefeld, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wibberg%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel Wibberg</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baku%C5%82a%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Zofia Bakuła</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Zofia Bakuła</span></h3>
<div class="p">
<sup>3</sup>Department of Medical Microbiology, Institute of Microbiology, Faculty of Biology, University of Warsaw, Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baku%C5%82a%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zofia Bakuła</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garc%C3%ADa-Cunchillos%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Iván García-Cunchillos</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Iván García-Cunchillos</span></h3>
<div class="p">
<sup>4</sup>Institute of Evolutionary Biology, Faculty of Biology, Biological and Chemical Research Centre, University of Warsaw, Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garc%C3%ADa-Cunchillos%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Iván García-Cunchillos</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gawor%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jan Gawor</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jan Gawor</span></h3>
<div class="p">
<sup>5</sup>DNA Sequencing and Synthesis Facility, Institute of Biochemistry and Biophysics, Polish Academy of Sciences, Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gawor%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jan Gawor</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gromadka%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Robert Gromadka</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Robert Gromadka</span></h3>
<div class="p">
<sup>5</sup>DNA Sequencing and Synthesis Facility, Institute of Biochemistry and Biophysics, Polish Academy of Sciences, Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gromadka%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Robert Gromadka</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Karnkowska%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Anna Karnkowska</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Anna Karnkowska</span></h3>
<div class="p">
<sup>4</sup>Institute of Evolutionary Biology, Faculty of Biology, Biological and Chemical Research Centre, University of Warsaw, Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Karnkowska%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anna Karnkowska</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bielecki%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jacek Bielecki</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jacek Bielecki</span></h3>
<div class="p">
<sup>3</sup>Department of Medical Microbiology, Institute of Microbiology, Faculty of Biology, University of Warsaw, Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bielecki%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jacek Bielecki</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jagielski%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Tomasz Jagielski</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Tomasz Jagielski</span></h3>
<div class="p">
<sup>3</sup>Department of Medical Microbiology, Institute of Microbiology, Faculty of Biology, University of Warsaw, Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jagielski%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tomasz Jagielski</span></a>
</div>
</div>
<sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Center for Biotechnology (CeBiTec), Genome Research of Industrial Microorganisms, Bielefeld University, Bielefeld, Germany </div>
<div id="Aff2">
<sup>2</sup>Institute of Bio- and Geosciences IBG-5 - Computational Metagenomics, Forschungszentrum Jülich GmbH, Bielefeld, Germany </div>
<div id="Aff3">
<sup>3</sup>Department of Medical Microbiology, Institute of Microbiology, Faculty of Biology, University of Warsaw, Warsaw, Poland </div>
<div id="Aff4">
<sup>4</sup>Institute of Evolutionary Biology, Faculty of Biology, Biological and Chemical Research Centre, University of Warsaw, Warsaw, Poland </div>
<div id="Aff5">
<sup>5</sup>DNA Sequencing and Synthesis Facility, Institute of Biochemistry and Biophysics, Polish Academy of Sciences, Warsaw, Poland </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 23; Accepted 2025 Jul 28; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12369231  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40836223/" class="usa-link">40836223</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background </h3>
<p id="Par1"><em>Prototheca</em> (Chlorophyta: Trebouxiophyceae) is a genus of non-photosynthetic microalgae that causes increasingly frequent infections in both humans and animals, collectively referred to as protothecosis The genetic landscape of the <em>Prototheca</em> algae has remained largely uncharted until recent advances in sequencing and genomics. In this study, a combination of Illumina and Oxford Nanopore technologies was employed for sequencing of 18 mitochondrial genomes, representing all currently recognized <em>Prototheca</em> species.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">The genomes differed in terms of size and GC content, ranging from 38 kbp to 68 kbp and from 25 to 30%, respectively. The gene content and gene order within the mitochondrial DNA exhibited specific characteristics. The gene content was conserved but showed variable number of hypothetical proteins and a clustering tendency for <em>nad</em> genes. Noteworthy, most genes were located on the clockwise strand, with type I introns, containing long open reading frames encoding homing endonucleases, suggesting a mechanism for intron mobility and genome plasticity. Comparative genomic analyses and phylogenetic classification across the 21 core genes showed a close relationship between the mitochondrial genomes, as evidenced by average nucleotide identity (ANI) and average amino acid identity (AAI), supportive for the current <em>cytb</em> gene-based taxonomy. The phylogenetic tree constructed from concatenated alignments of the core genes confirmed the presence of three distinct <em>Prototheca</em> clades, indicating the polyphyletic nature of the genus.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">In conclusion, this work provides another important step toward elucidating the genetics of <em>Prototheca</em> algae, serving as a framework for future studies on the phylogeny and evolution of these peculiar microorganisms.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12864-025-11952-9.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> <em>Prototheca</em>, Algae, Genome, Mitochondrion, Sequencing, Phylogenetics, Taxonomy</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par4">In the entire green plant lineage (Viridiplantae), standing at about 400,000 species [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>], unicellular, achlorophyllous microalgae of the <em>Prototheca</em> (Trebouxiophyceae, Chlorophyta) genus are unique in having been repeatedly implicated in the causation of diseases in both humans and animals, collectively referred to as protothecosis. Human protothecosis, whose rising incidence has now been an emerging trend, affects mainly patients with underlying immunosuppression and may have various clinical manifestations, the most severe of which is disseminated infection, with limited treatment options, poor prognosis, and a high mortality rate [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. In animals, the most prevalent form of <em>Prototheca</em> disease is bovine mastitis, a globally recognized problem that leads to a significant drop in milk production and heavy economic losses to the dairy industry [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Of 18 thus far described <em>Prototheca</em> species, six (<em>P. bovis</em>, <em>P. blaschkeae</em>, <em>P. ciferrii</em>, <em>P. cutis</em>, <em>P. miyajii</em>, and <em>P. wickerhamii</em>) are pathogenic to either animals or humans [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. The <em>Prototheca</em> algae and protothecosis have been much neglected areas of research. However, recent interest in this genus has increased due to their growing significance in clinical and epidemiological contexts. Therefore, over the past two decades, significant progress has been made in understanding the biology and genetic composition of <em>Prototheca</em> algae. The early molecular studies on <em>Prototheca</em> spp. focused on their rRNA clusters and substantially contributed towards resolving the phylogenetic and taxonomic position of the <em>Prototheca</em> genus [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. Compelling evidence for a close phylogenetic relationship between <em>Prototheca</em> and green algae was provided by studies investigating mitochondrial DNA. Early phylogenies targeting mitochondrially encoded rRNA (mt-rRNA) genes including those for the small and large subunit of the ribosomal RNA (SSU and LSU rRNA), as well as the gene for the subunit I of the cytochrome oxidase (<em>coxI</em>) showed a conspicuous affiliation of the <em>Prototheca</em> mitochondrial genome with that of higher plants [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. These observations were further confirmed by the phylogenetic analysis inferred from the mitochondrial <em>cytb</em> gene, coding for cytochrome b, a component of the respiratory chain bc1 complex [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. The <em>cytb</em> gene has demonstrated a high discriminatory power for the <em>Prototheca</em> genus, making it an efficient phylogenetic and diagnostic marker for these algae [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. It is the <em>cytb</em> gene that the present classification system of the <em>Prototheca</em> algae has been built upon [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>].</p>
<p id="Par5">An important step forward in disclosing phylogeny and evolution of the <em>Prototheca</em> clade was made with the introduction of new-generation sequencing (NGS) technology, which allowed to assess the genetic landscape of the algae at the whole-genome level. Genome-wide sequencing of <em>Prototheca</em> is still in its infancy, with merely a few attempts made so far, for five <em>Prototheca</em> species [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. The organellar genomes have most often been studied along with nuclear genomes [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Although the first protothecal genome to be sequenced, as early as in the mid-1990s, was mtDNA of <em>P. wickerhamii</em> [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>] (or in fact <em>P. xanthoriae</em> – [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]), only six more mitochondrial genomes representing four <em>Prototheca</em> species (single strains of <em>P. bovis</em>, <em>P. ciferrii</em>, <em>P. stagnora</em>, and three strains of <em>P. wickerhamii</em>) have been reported since then [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. In contrast, plastid genomes have been studied more frequently, mainly to gain a better understanding of their evolution in non-photosynthetic lineages and to determine whether photosynthesis was lost once or several times within the genus [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Although the plastid genomes are eroded, a core set of genes present in all plastid genomes enabled the reconstruction of the phylogeny of the genus and pointed to its polyphyly [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The purpose of this study was to perform an extensive parallel sequencing of mitochondrial genomes of all <em>Prototheca</em> species currently accepted, using the same sequencing and analytical pipelines. Such an approach minimized technical biases, facilitated interspecies comparisons, and enhanced the reliability of data collected. In undertaking this work, we sought to provide more insights into the genetic diversity of <em>Prototheca</em> algae to better resolve their interspecies phylogenetic relationships, and thereby to either support or revise the <em>cytb</em> gene-based <em>Prototheca</em> taxonomy.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials &amp; methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">
<em>Prototheca</em> strains and culture conditions</h3>
<p id="Par6">The 18 selected <em>Prototheca</em> sp. strains used in this study (Table <a href="#Tab1" class="usa-link">1</a>) were stored in Viabank™ cryogenic vials (Medical Wire &amp; Equipment Co Ltd., UK), at − 70 °C and were revived by streaking a loopful (10 µL) of the frozen culture on Yeast Peptone Dextrose (YPD) (Difco, USA) agar plates. Cultures were maintained aerobically at 37 °C for 72 h. Cells of each strain were picked from a single colony and grown in a 100 mL volume of YPD broth under the same conditions, yet with shaking (200 rpm) until the optical density at A600 reached ca. 5.0 (ca. 6.5 × 10<sup>5</sup> CFU).</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>List of <em>Prototheca</em> species type strains used in this study</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">Species</th>
<th align="left" colspan="1" rowspan="1">Origin</th>
<th align="left" colspan="1" rowspan="1">Strain name</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. blaschkeae</em>
</td>
<td align="left" colspan="1" rowspan="1">Human onychomycosis, Germany</td>
<td align="left" colspan="1" rowspan="1">SAG 2064 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. bovis</em>
</td>
<td align="left" colspan="1" rowspan="1">Milk from a mastitis cow, Germany</td>
<td align="left" colspan="1" rowspan="1">SAG 2021 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. cerasi</em>
</td>
<td align="left" colspan="1" rowspan="1">Bark of a cherry tree, Japan</td>
<td align="left" colspan="1" rowspan="1">JCM 9400 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. ciferrii</em>
</td>
<td align="left" colspan="1" rowspan="1">Swine manure, Germany</td>
<td align="left" colspan="1" rowspan="1">SAG 2063 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. cookei</em>
</td>
<td align="left" colspan="1" rowspan="1">Digested sludge, USA</td>
<td align="left" colspan="1" rowspan="1">ATCC 16,527 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. cutis</em>
</td>
<td align="left" colspan="1" rowspan="1">Human dermatitis, Japan</td>
<td align="left" colspan="1" rowspan="1">ATCC PRA-338 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. fontanea</em>
</td>
<td align="left" colspan="1" rowspan="1">Water from a municipal fountain, Poland</td>
<td align="left" colspan="1" rowspan="1">PK2 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. lentecrescens</em>
</td>
<td align="left" colspan="1" rowspan="1">Water from the Vistula riverbank, Poland</td>
<td align="left" colspan="1" rowspan="1">PK1 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. miyajii</em>
</td>
<td align="left" colspan="1" rowspan="1">Human dermatitis, Japan</td>
<td align="left" colspan="1" rowspan="1">IFM 53,848 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">10.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. moriformis</em>
</td>
<td align="left" colspan="1" rowspan="1">Slime flux of lime (<em>Tilia</em> sp.), Germany</td>
<td align="left" colspan="1" rowspan="1">SAG 263-2 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">11.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. paracutis</em>
</td>
<td align="left" colspan="1" rowspan="1">Soil in a mangrove forest, Thailand</td>
<td align="left" colspan="1" rowspan="1">JCM 32,112 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">12.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. pringsheimii</em>
</td>
<td align="left" colspan="1" rowspan="1">
<em>Xanthoria parietina</em>, The Netherlands</td>
<td align="left" colspan="1" rowspan="1">SAG 263-3 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">13.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. stagnora</em>
</td>
<td align="left" colspan="1" rowspan="1">Digested sludge, USA</td>
<td align="left" colspan="1" rowspan="1">ATCC 16,528 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">14.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. tumulicola</em>
</td>
<td align="left" colspan="1" rowspan="1">Soil from a stone chamber, Japan</td>
<td align="left" colspan="1" rowspan="1">JCM 31,123 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">15.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. vistulensis</em>
</td>
<td align="left" colspan="1" rowspan="1">Water from the Vistula River, Poland</td>
<td align="left" colspan="1" rowspan="1">W3 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">16.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. wickerhamii</em>
</td>
<td align="left" colspan="1" rowspan="1">Household plumbing, USA</td>
<td align="left" colspan="1" rowspan="1">ATCC 16,529 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">17.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. xanthoriae</em>
</td>
<td align="left" colspan="1" rowspan="1">
<em>Xanthoria parietina</em>
</td>
<td align="left" colspan="1" rowspan="1">SAG 263 − 11 (T)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">18.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. zopfii</em>
</td>
<td align="left" colspan="1" rowspan="1">Unknown</td>
<td align="left" colspan="1" rowspan="1">ATCC 16,533 (T)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Genomic DNA extraction</h3>
<p id="Par8">Genomic DNA (gDNA) was isolated as described before [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>] with a method based on three-pronged approach for whole cell lysis, i.e. mechanical (glass bead pulverization and grinding in liquid nitrogen using mortar), surfactant-based (Triton-X100, SDS, and CTAB treatment), and enzymatic (Proteinase K treatment) disruption methods. DNA integrity was analyzed by 0.8% agarose gel electrophoresis and by capillary PFGE using gDNA 165 kb kit and FemtoPulse instrument (Agilent, USA).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Genome sequencing</h3>
<p id="Par9">In total, 16 of 18 <em>Prototheca</em> genomes were sequenced using a combination of Illumina (Illumina Inc., USA) and Oxford Nanopore (Oxford Nanopore, UK) technologies. The genome of <em>P. moriformis</em> was sequenced by means of Illumina sequencing. The <em>P. wickerhamii</em> genome was sequenced using a combination of Illumina (Illumina, USA) and PacBio (Pacific Biosciences, USA) technologies, essentially as described elsewhere [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. The Illumina pair-end sequencing libraries construction was performed with 500 ng of input DNA extract and the NEB Ultra II FS reagents (New England Biolabs, USA), according to manufacturer’s instructions. The libraries were pooled and sequenced on a NextSeq 550 instrument (Illumina, USA) using 300-cycle chemistry kit. The long-read libraries were constructed using approximately 10 µg of gDNA that was first size-selected using the BluePippin apparatus and High Pass Plus cassettes (Sage Science, USA) with a minimum cutoff of 10 kb. During ONT library construction, samples were indexed with the native barcoding expansion kit (EXP-NBD103). Long read sequencing was conducted with R 9.4.1 flowcells using MinION and GridION instruments (Oxford Nanopore Technologies, UK).</p>
<p id="Par10">Illumina sequence quality metrics [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>] were initially assessed using FASTQC and quality trimmed using fastp [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Raw nanopore data were basecalled using Guppy v6.1.3 in super accuracy mode (Oxford Nanopore Technologies, UK) and quality checked using NanoPlot v1.41.6 [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. The assemblies were performed using Canu v2.1.1 [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>] and Flye v2.9.2 [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>].</p>
<p id="Par11">Nucleotide sequence data including annotation for all 18 <em>Prototheca</em> mt genomes were submitted to the GenBank/EBI/DDBJ database and are retrievable under the Bioproject No. PRJEB82245. In addition, the data have been deposited in Zenodo and are available at 10.5281/zenodo.16410224.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Mitochondrial genome identification and annotation</h3>
<p id="Par12">Based on the ‘Contig length versus Read count’ analysis [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] the mitochondrial (mt) genomes of all <em>Prototheca</em> strains were identified as one contig in the corresponding assembled data set. Gene prediction and functional annotation was performed by MFannot [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] using gene code 16 for chlorophyte algae. Gene prediction and annotation results were manually curated, as described elsewhere [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>].</p></section><section id="Sec7"><h3 class="pmc_sec_title">Comparative analysis &amp; phylogenetic classification</h3>
<p id="Par13">The mt genomes of the <em>Prototheca</em> strains and their closest available relatives (Table <a href="#Tab2" class="usa-link">2</a>) were used for comparative genomic and phylogenetic analysis. Comparative genomics was accomplished using a modified version of the program EDGAR 3.2 designed to handle multi-exon genes [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. The analyses included identification of orthologous genes, classification of genes as core genes or singletons and visualization of Venn diagrams. In addition, average nucleotide identity (ANI) [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] and average amino acid identity (AAI) [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] analyses were performed to determine the relationship between different species. Synteny analysis was performed by applying EasyFig [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>] and BLASTn [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>].</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>List of the closest relative species to <em>Prototheca</em> sp. used in this study</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">Species</th>
<th align="left" colspan="1" rowspan="1">Strain name</th>
<th align="left" colspan="1" rowspan="1">Size [bp]</th>
<th align="left" colspan="1" rowspan="1">GenBank accession no.</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1.</td>
<td align="left" colspan="1" rowspan="1">
<em>Auxenchlorella protothecoides</em>
</td>
<td align="left" colspan="1" rowspan="1">UTEX 2341</td>
<td align="center" colspan="1" rowspan="1">57,063</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/KY681419" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY681419</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2.</td>
<td align="left" colspan="1" rowspan="1">
<em>Chlorella heliozoae</em>
</td>
<td align="left" colspan="1" rowspan="1">SAG 3.83</td>
<td align="center" colspan="1" rowspan="1">65,049</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/KY629615" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY629615</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3.</td>
<td align="left" colspan="1" rowspan="1">
<em>Chlorella ohadii</em>
</td>
<td align="left" colspan="1" rowspan="1">No data</td>
<td align="center" colspan="1" rowspan="1">52,560</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/MT995630" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MT995630</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4.</td>
<td align="left" colspan="1" rowspan="1">
<em>Chlorella sorakiniana</em>
</td>
<td align="left" colspan="1" rowspan="1">1230</td>
<td align="center" colspan="1" rowspan="1">52,528</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/KJ742377" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KJ742377</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5.</td>
<td align="left" colspan="1" rowspan="1">
<em>Chlorella</em> sp.</td>
<td align="left" colspan="1" rowspan="1">ATCC 30,562</td>
<td align="center" colspan="1" rowspan="1">79,601</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/KY629618" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY629618</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6.</td>
<td align="left" colspan="1" rowspan="1">
<em>Chlorella</em> sp.</td>
<td align="left" colspan="1" rowspan="1">ArM0029B</td>
<td align="center" colspan="1" rowspan="1">65,049</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/KF554428" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KF554428</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7.</td>
<td align="left" colspan="1" rowspan="1">
<em>Chlorella variabilis</em>
</td>
<td align="left" colspan="1" rowspan="1">NC64A</td>
<td align="center" colspan="1" rowspan="1">78,500</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/KM252919" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KM252919</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8.</td>
<td align="left" colspan="1" rowspan="1">
<em>Chlorella vulgaris</em>
</td>
<td align="left" colspan="1" rowspan="1">UTEX 259</td>
<td align="center" colspan="1" rowspan="1">98,062</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/MK948103" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MK948103</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9.</td>
<td align="left" colspan="1" rowspan="1">
<em>Chlorella vulgaris*</em>
</td>
<td align="left" colspan="1" rowspan="1">ITBBA3-12</td>
<td align="center" colspan="1" rowspan="1">88,754</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/MT419367" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MT419367</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">10.</td>
<td align="left" colspan="1" rowspan="1">
<em>Chlorella vulgaris</em>
</td>
<td align="left" colspan="1" rowspan="1">NJ-7</td>
<td align="center" colspan="1" rowspan="1">87,477</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/MK948101" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MK948101</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">11.</td>
<td align="left" colspan="1" rowspan="1">
<em>Chlorella vulgaris</em>
</td>
<td align="left" colspan="1" rowspan="1">KNUA007</td>
<td align="center" colspan="1" rowspan="1">84,368</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/MW900258" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MW900258</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">12.</td>
<td align="left" colspan="1" rowspan="1">
<em>Helicosporidium</em> sp.</td>
<td align="left" colspan="1" rowspan="1">ATCC 50,920</td>
<td align="center" colspan="1" rowspan="1">49,343</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/GQ339576" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GQ339576</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">13.</td>
<td align="left" colspan="1" rowspan="1">
<em>Marvania geminata</em>
</td>
<td align="left" colspan="1" rowspan="1">SAG 12–88</td>
<td align="center" colspan="1" rowspan="1">51,936</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP746561" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP746561</a></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p18"><p>*<em>Chlorella vulgaris</em> ITBBA3-12 appears to have an incomplete mitochondrial genome, as the rRNA genes (<em>rns</em> and <em>rnl</em>) are missing, indicating that the mtDNA is either misannotated or incomplete. However, since rRNA genes were not included in the alignment used for the phylogenetic trees, their absence does not affect those analyses</p></div></div></section><p id="Par15">We conducted phylogenetic analyses on the 21 core annotated mitochondrial genes (Table S2, genes marked in green) from the 18 <em>Prototheca</em> strains (Table <a href="#Tab1" class="usa-link">1</a>). The core genes were identified by EDGAR, relying on the Blast Score Ratio (BSR) values. BSR helps to determine sequence similarity and is used to identify orthologous genes, ensuring that only highly conserved genes are included in the core genome. By using specific BSR thresholds, EDGAR can accurately determine which genes are present across all genomes studied.</p>
<p id="Par16">We further complemented the dataset with information from <em>Auxenochlorella protothecoides</em> and <em>Helicosporidium</em> sp. Different <em>Chlorella</em> species and <em>Marvania geminata</em> were selected as the outgroup for the phylogenies (Table <a href="#Tab2" class="usa-link">2</a>). Individual alignments with amino acid sequences from all the species were constructed using MUSCLE [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>] with default settings for each of the 21 genes. The original alignments are accessible through doi: 10.6084/m9.figshare.29328425. Alignments were automatically trimmed with trimAl v1.5.0 [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>] using a heuristic selection of the positions with the option -automated1, which considers the average identity score (i) among all the sequences, (ii) for each most similar pair of sequences, and (iii) the number of sequences in the alignment. Maximum likelihood phylogenetic trees were inferred for every single-gene alignment with IQ-TREE 2.0 [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>], selecting best-fitting substitution models with ModelFinder [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] under the Bayesian information criterion, and 1000 ultrafast bootstrap replicates [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. Since no incongruences among the single-gene phylogenies were detected, we constructed a concatenated alignment comprising 5,277 positions. The concatenated dataset did not show differences when considering potential partition schemes among the genes, so we considered mitochondrial genomes as single-evolving units and constructed the final maximum likelihood phylogeny as previously stated, under the model LG + F + R4, with 500 non-parametric bootstrap replicates (Fig. 2). Because the mitochondrial cytochrome b gene (<em>cytb</em>) is often employed to differentiate <em>Prototheca</em> species as barcode region [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>] we additionally inferred the single-gene phylogeny of this gene following the same procedure under the model mtZOA + I + R3, and considering 1000 non-parametric bootstrap replicates (Supplementary Fig. 1).</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Results &amp; discussion</h2>
<section id="Sec9"><h3 class="pmc_sec_title">The complete Mt genome sequences of 18 <em>Prototheca</em> species</h3>
<p id="Par17">The genome sequencing of <em>Prototheca</em> species was part of an extensive project focused on 18 different strains, including <em>P. moriformis</em>, which was sequenced exclusively via Illumina technology. For the remaining 17 species, third-generation sequencing technologies, such as Oxford Nanopore and PacBio were employed, complemented by second-generation Illumina sequencing. For 18 <em>Prototheca</em> species (type strains) under the study, the number of reads obtained by Nanopore sequencing ranged from 72,339 to 438,974 with an average read count of 159,546, while the N50 read length was 14,445 bp (5,547–21,755 bp). For more details, see Table S1. Assemblies polished by Illumina short reads were screened by “Contig length vs. Read Count Analyses” [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] revealing that all <em>Prototheca</em> mt genomes were identified as single circular contigs in the assembled data set.</p>
<p id="Par18">The complete mt genome sequences ranged in their size from 38 kbp (<em>P. ciferrii</em>) to 68 kbp (<em>P. miyajii</em>). Interestingly, the main reason for the larger sizes of the <em>Prototheca</em> mt genomes related to the presence of non-conserved hypothetical ORFs and numerous introns in areas without core genes, explained in detail in the next chapter (Fig. <a href="#Fig1" class="usa-link">1</a>). Furthermore, when compared to related species, the mt genome sizes were generally similar, except for <em>Chlorella vulgaris</em>, whose mt genomes reached a size of up to 100 kbp (Table <a href="#Tab2" class="usa-link">2</a>). The GC content of the <em>Prototheca</em> mt genomes ranged from 25 to 30%, which is comparable to mt genomes of <em>Chlorella</em>, <em>Auxenochlorella</em>, <em>Marvania</em> and <em>Helicosporidium</em>. The highest GC ratio in mitogenomes can be found in green algae. For example, the mtDNA of <em>Chlamydomonas reinhardtii</em> has a GC content of 57% [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>].</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369231_12864_2025_11952_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2972/12369231/ec1d89eaa8cd/12864_2025_11952_Fig2_HTML.jpg" loading="lazy" id="d33e1252" height="1269" width="778" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative analysis of <em>Prototheca</em> mt genomes. Alignments computed by EasyFig and BLASTn of the sequenced <em>Prototheca</em> mt genomes (see Table 1 for characteristics) are presented. Annotated coding sequences are displayed as arrows. Gray highlighted regions mark homologous regions</p></figcaption></figure><p id="Par20">The features of the protothecal mt genomes, including genome size, GC content, and number of annotated genes are detailed in Table <a href="#Tab3" class="usa-link">3</a>. On average, <em>Prototheca</em> mt genomes contained 38 protein-coding sequences. In general, all mt genomes contained genes encoding 26 transfer RNA (tRNA) and three ribosomal RNA (rRNA). To identify replication origins in bacterial genomes GC-skew analysis is commonly applied [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. A similar approach has been proposed for predicting the origin of replication in fungal mt genomes [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. However, for the <em>Prototheca</em> mt genomes, it was impossible to predict the origin of replication based on GC-content or GC-skew analyses due to their strong mosaic structure. Mosaic refers to the structure when mt genomes are compared against each other. This reveals modules that are in different mt genomes in different positions. In comparison, chromosomes typically have a conserved structure with few, if any, rearrangements. Nevertheless, here the structure is fragmented for many mt genomes.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>General features of <em>Prototheca</em> sp. mt genomes</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">
<em>Prototheca</em> species</th>
<th align="left" colspan="1" rowspan="1">Size [bp]</th>
<th align="left" colspan="1" rowspan="1">GC%</th>
<th align="left" colspan="1" rowspan="1">rRNAs</th>
<th align="left" colspan="1" rowspan="1">tRNAs</th>
<th align="left" colspan="1" rowspan="1">CDS</th>
<th align="left" colspan="1" rowspan="1">Introns</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. blaschkeae</em>
</td>
<td align="center" colspan="1" rowspan="1">41,428</td>
<td align="center" colspan="1" rowspan="1">28.6%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">25</td>
<td align="center" colspan="1" rowspan="1">35</td>
<td align="center" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. bovis</em>
</td>
<td align="center" colspan="1" rowspan="1">39,222</td>
<td align="center" colspan="1" rowspan="1">28.7%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">34</td>
<td align="center" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. cerasi</em>
</td>
<td align="center" colspan="1" rowspan="1">38,642</td>
<td align="center" colspan="1" rowspan="1">28.2%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">34</td>
<td align="center" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. ciferrii</em>
</td>
<td align="center" colspan="1" rowspan="1">38,164</td>
<td align="center" colspan="1" rowspan="1">28.7%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">33</td>
<td align="center" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. cookei</em>
</td>
<td align="center" colspan="1" rowspan="1">44,721</td>
<td align="center" colspan="1" rowspan="1">29.7%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">38</td>
<td align="center" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. cutis</em>
</td>
<td align="center" colspan="1" rowspan="1">61,116</td>
<td align="center" colspan="1" rowspan="1">27.9%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">37</td>
<td align="center" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. fontanea</em>
</td>
<td align="center" colspan="1" rowspan="1">55,995</td>
<td align="center" colspan="1" rowspan="1">26.0%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">37</td>
<td align="center" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. lentecrescens</em>
</td>
<td align="center" colspan="1" rowspan="1">61,276</td>
<td align="center" colspan="1" rowspan="1">28.2%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">38</td>
<td align="center" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. miyajii</em>
</td>
<td align="center" colspan="1" rowspan="1">68,318</td>
<td align="center" colspan="1" rowspan="1">30.6%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">44</td>
<td align="center" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">10.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. moriformis</em>
</td>
<td align="center" colspan="1" rowspan="1">45,176</td>
<td align="center" colspan="1" rowspan="1">28.5%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">40</td>
<td align="center" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">11.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. paracutis</em>
</td>
<td align="center" colspan="1" rowspan="1">55,057</td>
<td align="center" colspan="1" rowspan="1">30.1%</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">24</td>
<td align="center" colspan="1" rowspan="1">32</td>
<td align="center" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">12.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. pringsheimii</em>
</td>
<td align="center" colspan="1" rowspan="1">38,972</td>
<td align="center" colspan="1" rowspan="1">28.4%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">35</td>
<td align="center" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">13.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. stagnora</em>
</td>
<td align="center" colspan="1" rowspan="1">55,223</td>
<td align="center" colspan="1" rowspan="1">27.7%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">41</td>
<td align="center" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">14.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. tumulicola</em>
</td>
<td align="center" colspan="1" rowspan="1">49,847</td>
<td align="center" colspan="1" rowspan="1">28.0%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">40</td>
<td align="center" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">15.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. vistulensis</em>
</td>
<td align="center" colspan="1" rowspan="1">38,671</td>
<td align="center" colspan="1" rowspan="1">28.3%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">25</td>
<td align="center" colspan="1" rowspan="1">34</td>
<td align="center" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">16.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. wickerhamii</em>
</td>
<td align="center" colspan="1" rowspan="1">53,822</td>
<td align="center" colspan="1" rowspan="1">25.8%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">39</td>
<td align="center" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">17.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. xanthoriae</em>
</td>
<td align="center" colspan="1" rowspan="1">55,435</td>
<td align="center" colspan="1" rowspan="1">25.8%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">37</td>
<td align="center" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">18.</td>
<td align="left" colspan="1" rowspan="1">
<em>P. zopfii</em>
</td>
<td align="center" colspan="1" rowspan="1">38,948</td>
<td align="center" colspan="1" rowspan="1">28.7%</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">34</td>
<td align="center" colspan="1" rowspan="1">0</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec10"><h3 class="pmc_sec_title">Gene structure and gene order of the <em>Prototheca</em> mt genomes</h3>
<p id="Par22">The number of protein-coding genes is highly variable among eukaryotic mt genomes [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. Universally, two mt genes, namely <em>cox1</em> and <em>cytb</em> are always present in green algae mt genomes, coding for subunit I of cytochrome c oxidase and cytochrome b, respectively. In addition, organisms with complete respiratory complexes always retain the <em>nad</em> genes (<em>nad1</em>, <em>nad2</em>, <em>nad4</em>, and <em>nad5</em>) in their mitogenomes. Additionally, most algae possess genes, such as <em>cox2</em>, <em>cox3</em>, <em>nad3</em>, <em>nad4L</em>, <em>atp6</em>, and <em>atp8</em>. The proteins encoded by these genes are highly hydrophobic, featuring numerous transmembrane stretches [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>].</p>
<p id="Par23">The core mitochondrial genome of <em>Prototheca</em> contained 21 genes, while its pan-genome comprised 78 genes, demonstrating the remarkable plasticity of the mt genome content in these algae. A few essential genes are surprisingly absent from some mt genomes that reduces the amount of potential core genes. Technical problems for the <em>Prototheca</em> mt genomes were not found. Only one mt genome showed a quite high methylation rate, which could lead to more sequencing errors, resulting in SNPs that might prevent a proper gene prediction. However, these genes were in detail checked and found that sometimes they are completely missing, sometimes only fragments can be found, and sometimes point mutations seem to “destroy” these genes. This variability is primarily reflected in the diverse array of potential endonuclease-like genes present. As reference, the pangenome analysis of eight available <em>Chlorella</em> mt genomes produced a total of 41 genes. In a pangenome vs. pangenome comparison, the pangenomes of <em>Prototheca</em> mt genomes with those of reference mt genomes were compared. Only a few unique genes, so called singletons, were identified in each pangenome, mainly coding for hypothetical proteins. The average number of genes coding for hypothetical proteins is at about 6 per mt genome overall, whereas <em>P. stagnora</em> has 12. A few of these genes are present in several mt genomes (Table S2). Here, groups can be indentified: 1) <em>P. blaschkeae</em>,<em> P. bovis</em>,<em> P. cerasi</em>,<em> P. ciferrii</em>,<em> P. cookei</em>,<em> P. moriformis</em>,<em> P. pringsheimii</em>,<em> P. zopfii</em>,<em> P. stagnora</em>,<em> P. tumulicola</em>, and <em>P. vistulensis</em> shared homologous hypothecial proteins as well as <em>P. lentecrescens</em>,<em> P. fontanea</em>, <em>P. wickerhamii</em>,<em> P. cutis</em>,<em> P. paracutis and P. miyajii.</em> Notably, there was no significant evidence of typical gene loss or gene transfer to the nuclear genome. The mt gene content of the 18 <em>Prototheca</em> species seems to be typical for algal genomes, since all the genes mentioned above were found in all <em>Prototheca</em> and <em>Prototheca</em>-related mt genomes (Table S2).</p>
<p id="Par24">In <em>Prototheca</em> mt genomes, the predicted genes were typically mono-exonic without introns, with the notable exception of the <em>cox1</em> gene. In 12 out of the 18 examined mt genomes, the <em>cox1</em> gene contained introns, with each genome having one to three introns on average. These introns, featuring T and G bases at their splice sites, are classified as type I introns. On the other hand, in the remaining six protothecal mt genomes, the <em>cox1</em> gene consisted of a single exon without introns. The introns in the <em>cox1</em> gene showed variation across different <em>Prototheca</em> species in terms of their position, size, and number, as well as in the overall structure of the exon-intron configuration.</p>
<p id="Par25">Type I introns in mt genomes are characterized by ORFs encoding the so-called homing endonucleases. These enzymes facilitate intron mobility by recognizing rare DNA sites and cleaving them, triggering the cell’s DNA repair mechanisms to insert the intron sequence precisely into the target gene [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. In the <em>Prototheca</em> mt genomes, nearly all type I introns included sequences similar to endonuclease genes.</p>
<p id="Par26">In addition to the GC skew analysis, the analysis of <em>Prototheca</em> mt genomes highlights the variability in the gene order of core genes across different species. Nonetheless, the NADH dehydrogenase (<em>nad</em>) genes showed a tendency for clustering in specific groups, such as <em>nad2</em>-<em>nad3</em>, <em>nad</em>4L-<em>nad5</em>, and <em>nad4</em>-<em>nad6</em>, demonstrating a conserved organizational pattern within <em>Prototheca</em> mt genomes. Similar clusters have also been identified in other algal [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>] and also fungal mt genomes [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Further comparative analysis revealed a significant synteny only in the protein-coding genes, even amidst numerous rearrangements (Fig. <a href="#Fig1" class="usa-link">1</a>). Overall, a cross structure could be found, which is also typical for bacterial chromosomes or plasmids [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. This pattern suggests that the genomes had been actively shaped by the evolutionary forces, imparting a high degree of plasticity. Despite these variations, certain elements, including the core gene set, tRNAs, and rRNAs, remained conserved across species, likely due to their crucial functional roles. However, manual inspection revealed that a few essential genes are missing in some mitochondrial genomes. It is well established that multiple variants of mitochondrial genomes can coexist within the same cell, potentially representing different versions or heteroplasmic states. Typically, the consensus sequence generated from short or long read data reflects the dominant variant present in the sample; however, minority variants may be underrepresented or even absent from the consensus. In this case, while the consensus sequence likely reflects the main mtDNA version, we cannot entirely exclude the possibility that some genes may be missing due to sequencing or assembly artifacts—or, alternatively, due to biological heterogeneity among mtDNA copies (heteroplasmy or structural variation) or other biological reasons [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>].</p>
<p id="Par27">Overall, the mitogenomes of <em>Prototheca</em> species exhibited a typical gene composition for algal genomes, including protein-coding genes, tRNAs, and rRNAs, showcasing their functional stability. However, the variability in gene order, the presence of different type I introns as well as other genes coding for hypothetical proteins, and numerous rearrangements emphasize the plasticity of their mt DNA.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Phylogenetic classification of <em>Prototheca</em> mt genomes based on shared core genes</h3>
<p id="Par28">We inferred the phylogeny of the <em>Prototheca</em> strains from the reconstruction of the maximum-likelihood phylogenomic tree based on the 21 core genes encoded in the mt genomes (Fig. <a href="#Fig2" class="usa-link">2</a>). The resulting tree topology revealed three main <em>Prototheca</em> clades (A − C), as previously reported in the phylogeny based on plastid genomes [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. While clades A (including <em>Auxenochlorella protothecoides</em>) and C showed full phylogenetic support (BS = 100), clade B received low support (BS = 51) and consisted of two well-supported subclades (Fig. <a href="#Fig2" class="usa-link">2</a>). Remarkably, clade C included all strains without introns in the <em>cox1</em> gene and mt genome sizes of about 39 kb, whereas clades A and B were more heterogeneous (Table <a href="#Tab3" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369231_12864_2025_11952_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2972/12369231/9e7b324a3237/12864_2025_11952_Fig1_HTML.jpg" loading="lazy" id="d33e1953" height="591" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phylogenomic maximum-likelihood tree based on the 21 core concatenated proteins of <em>Prototheca</em> mitochondrial genomes, comprising 5277 unambiguously aligned sites under the model LG + F + R4. Related Chlorellaceae species were selected as the outgroup. Phylogenetic support assessment is based on 500 non-parametric bootstrap replicates; only support values &gt; 50% are shown, and black dots indicate maximum support</p></figcaption></figure><p id="Par30">The phylogeny inferred from the <em>cytb</em> gene (Supplementary Fig. 1) recovered the same clades B and C, although the former did not show phylogenetic support (BS = 38). The phylogenetic reconstruction based solely on the <em>cytb</em> gene did not recover clade A comprising species of <em>P</em>. <em>xanthoriae</em> and <em>A</em>. <em>protothecoides</em>. Similarly, the position of <em>Helicosporidum</em> sp. differed between both phylogenies, branching as the sister species to all <em>Prototheca</em> clades according to <em>cytb</em> phylogeny (BS = 99). The genome-based phylogeny recovered this species as sister to clade C (BS = 100), and the corresponding clade in the plastid phylogeny [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. To validate the accurate classification of all species and to assess the genetic similarities within the <em>Protothec</em>a genus, a comprehensive analysis using pairwise ANI and AAI analysis was performed. The results of these analyses are detailed in Table S3 and Table S4, respectively. These metrics provide a quantitative measure of genomic and protein-level similarities, further supporting the taxonomic distinctions and relationships among the <em>Prototheca</em> species. Typically, genomes of prokaryotic isolates belonging to the same species exhibit higher ANI and AAI values (&gt; 95%) than those representing different species [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>, <a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. For fungal genomes, these values are even higher (&gt; 97%) [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>, <a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Pairwise comparisons of the <em>Prototheca</em> mt genomes revealed that the average ANI values were less than 97%, indicating still a considerably high degree of sequence homology between strains representing different species. Likewise, the AAI values were below 97% except for <em>P. bovis</em> and <em>P. zopfii</em> (97.8%) and for <em>P. pringsheimii</em> and <em>P. ciferii</em> (98.5%), suggesting a closer relationship between the two species. For species demarcation, the ANI threshold appears to be similar to that observed in fungi (&gt; 97%), whereas the AAI threshold may need to be somewhat elevated. Although ANI analysis may offer potential insights for species delineation, it has to be done with caution. While ANI is a well-established tool for prokaryotes, its application to eukaryotic genomes requires careful consideration due to the unique evolutionary dynamics involved. The distinction between the slower evolution of mtDNA genes and the faster evolution of intergenic regions highlights the complexity of using whole-genome comparisons. Despite the lack of standardized thresholds for classifying plastid or mtDNA, ANI can still provide valuable preliminary insights, which, when combined with other molecular and morphological data (e.g. AAI), can contribute to a more comprehensive understanding of species relationships.</p>
<p id="Par31">In summary, two genome-based indices (ANI, and AAI) supported the delineation of three clades among <em>Prototheca</em> species, which may translate into taxonomic shifts within the genus. Interestingly, no clear ecological link can be established between species belonging to a particular clade. Both species typically saprophytic and pathogenic to either humans or animals shared the same clade. Whether species distribution within the clades reflects any specific ecological adaptations and niche preferences can be resolved by the sequencing analysis of protothecal nuclear genomes.</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par32">In this study, we have explored the mt genome diversity of 18 <em>Prototheca</em> species, providing further insights into their genetic makeup and evolutionary trajectory. A relatively high genetic variation of the <em>Prototheca</em> mt genomes was demonstrated, evidenced by differences in the size of the genomes, their GC content, gene content, and gene order.</p>
<p id="Par33">The clustering of <em>nad</em> genes, which encode key subunits of the NADH dehydrogenase complex involved in mitochondrial energy production, along with the predominance of genes on the clockwise strand, were found to be hallmarks of the protothecal mt genomes. Additionally, the presence of type I introns harboring homing endonucleases hints at mechanisms of intron mobility and mitogenome evolution. Similar genomic arrangements and features have been observed in other related algae, including <em>Helicosporidium</em> and <em>Chlorella</em> [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. Here also, the mechanisms involving intron mobility and mt genome evolution are often characterized by the presence of type I introns and clustering of essential genes. Therefore, while these features are notable in <em>Prototheca</em>, they may also reflect broader evolutionary pattern shared among other related algal species, indicating common strategies in mitogenome structuring and adaptation. More comparative studies across these and other related taxa can provide greater insights into the uniqueness or conservation of these genomic characteristics.</p>
<p id="Par34">The phylogenetic tree constructed from concatenated 21 core gene alignments delineated three distinct clades within the <em>Prototheca</em> species pointing to the polyphyly of the genus. The core-gene tree aligned with the <em>cytb</em> gene tree, which showed two distinct <em>Prototheca</em> clades with varying mitogenome sizes and intron presence, suggesting different evolutionary paths. In addition, this phylogenetic framework, coupled with the high intra-species sequence homology observed in ANI and AAI analyses, provided a new lens through which to view the evolutionary relationships within this genus. Our comparative genomic analyses underlined the phylogenetic relatedness within the genus <em>Prototheca</em>, corroborating the current <em>cytb</em> gene-based taxonomy while also opening up avenues for taxonomic refinements.</p>
<p id="Par35">Our findings help address a knowledge gap in <em>Prototheca</em> genetics, and provide a foundation for future research into the taxonomy, phylogeny, and evolution of this genus. As we continue to unravel the genetic complexities of the <em>Prototheca</em> algae, we have moved closer towards reconstruction of the evolutionary history of Chlorophyta algae and their mt genomes. Looking ahead, the next step involves the analysis of whole genomes of 18 <em>Prototheca</em> species, which is expected to significantly advance our understanding of the genomic diversity and evolutionary patterns of <em>Prototheca</em> algae.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12369231/bin/12864_2025_11952_MOESM1_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">12864_2025_11952_MOESM1_ESM.png</a><sup> (646.3KB, png) </sup><p>Supplementary Material 1: Phylogenetic maximum-likelihood tree of <em>Prototheca</em> based on the cytochrome b protein (<em>cytb</em>), comprising 384 unambiguously aligned sites under the model mtZOA + I + R3. Related Chlorellaceae species were selected as the outgroup. Phylogenetic support assessment is based on 1000 non-parametric bootstrap replicates; only support values &gt; 50% are shown, and black dots indicate maximum support.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12369231/bin/12864_2025_11952_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (24.5KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Not applicable.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>DW carried out bioinformatic analyses, wrote the first draft of the manuscript, and prepared all figures and tables; ZB cultured <em>Prototheca </em>strains and performed DNA isolation; IGC carried out bioinformatic analyses; JG and RG performed sequencing experiments including initial data curation; AK and JB reviewed the manuscript; TJ conceived and planned the project, obtained the funding, and wrote the final version of the manuscript with input of other co-authors. All authors read, critically commented, and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This research received no specific grant from any funding agency in the public, commercial, or not-for-profit sectors.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Nucleotide sequence data for all 18 <em>Prototheca</em> mt genomes are deposited in the GenBank/EBI/DDBJ database under the Bioproject No. PRJEB82245. The original alignments are accessible through doi: 10.6084/m9.figshare.29328425. Furthermore, data generated or analysed during this study are included in this published article and its supplementary information files.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par36">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par37">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par38">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Lughadha EN, et al. Counting counts: revised estimates of numbers of accepted species of flowering plants, seed plants, vascular plants and land plants with a review of other recent estimates. Phytotaxa. 2016;272:82.</cite> [<a href="https://scholar.google.com/scholar_lookup?Lughadha%20EN,%20et%20al.%20Counting%20counts:%20revised%20estimates%20of%20numbers%20of%20accepted%20species%20of%20flowering%20plants,%20seed%20plants,%20vascular%20plants%20and%20land%20plants%20with%20a%20review%20of%20other%20recent%20estimates.%20Phytotaxa.%202016;272:82." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Jagielski T, Lagneau P-E. Protothecosis. A pseudofungal infection. J Mycol Médicale. 2007;17:261–70.</cite> [<a href="https://scholar.google.com/scholar_lookup?Jagielski%20T,%20Lagneau%20P-E.%20Protothecosis.%20A%20pseudofungal%20infection.%20J%20Mycol%20M%C3%A9dicale.%202007;17:261%E2%80%9370." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Todd JR, et al. Protothecosis: report of a case with 20-year follow-up, and review of previously published cases. Med Mycol. 2012;50:673–89.
</cite> [<a href="https://doi.org/10.3109/13693786.2012.677862" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22571772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Todd%20JR,%20et%20al.%20Protothecosis:%20report%20of%20a%20case%20with%2020-year%20follow-up,%20and%20review%20of%20previously%20published%20cases.%20Med%20Mycol.%202012;50:673%E2%80%9389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Jagielski T, et al. A survey on the incidence of prototheca mastitis in dairy herds in Lublin province, Poland. J Dairy Sci. 2019;102:619–28.
</cite> [<a href="https://doi.org/10.3168/jds.2018-15495" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30447976/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jagielski%20T,%20et%20al.%20A%20survey%20on%20the%20incidence%20of%20prototheca%20mastitis%20in%20dairy%20herds%20in%20Lublin%20province,%20Poland.%20J%20Dairy%20Sci.%202019;102:619%E2%80%9328." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Jagielski T, et al. Prevalence of <em>Prototheca</em> spp. On dairy farms in Poland – a cross-country study. Microb Biotechnol. 2019;12:556–66.
</cite> [<a href="https://doi.org/10.1111/1751-7915.13394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6465227/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30891936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jagielski%20T,%20et%20al.%20Prevalence%20of%20Prototheca%20spp.%20On%20dairy%20farms%20in%20Poland%20%E2%80%93%20a%20cross-country%20study.%20Microb%20Biotechnol.%202019;12:556%E2%80%9366." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Jagielski T, et al. Occurrence of <em>Prototheca</em> microalgae in aquatic ecosystems with a description of three new species, prototheca fontanea, prototheca lentecrescens, and prototheca vistulensis. Appl Environ Microbiol. 2022;88:e01092–22.
</cite> [<a href="https://doi.org/10.1128/aem.01092-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9680628/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36300932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jagielski%20T,%20et%20al.%20Occurrence%20of%20Prototheca%20microalgae%20in%20aquatic%20ecosystems%20with%20a%20description%20of%20three%20new%20species,%20prototheca%20fontanea,%20prototheca%20lentecrescens,%20and%20prototheca%20vistulensis.%20Appl%20Environ%20Microbiol.%202022;88:e01092%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Jagielski T, et al. The genus <em>Prototheca</em> (Trebouxiophyceae, Chlorophyta) revisited: implications from molecular taxonomic studies. Algal Res. 2019;43:101639.</cite> [<a href="https://scholar.google.com/scholar_lookup?Jagielski%20T,%20et%20al.%20The%20genus%20Prototheca%20(Trebouxiophyceae,%20Chlorophyta)%20revisited:%20implications%20from%20molecular%20taxonomic%20studies.%20Algal%20Res.%202019;43:101639." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Huss VAR, et al. Biochemical taxonomy and molecular phylogeny of the genus <em>Chlorella</em> sensu lato (chlorophyta). J Phycol. 1999;35:587–98.</cite> [<a href="https://scholar.google.com/scholar_lookup?Huss%20VAR,%20et%20al.%20Biochemical%20taxonomy%20and%20molecular%20phylogeny%20of%20the%20genus%20Chlorella%20sensu%20lato%20(chlorophyta).%20J%20Phycol.%201999;35:587%E2%80%9398." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Huss VAR, Sogin ML. Phylogenetic position of some <em>Chlorella</em> species within the Chlorococcales based upon complete small-subunit ribosomal RNA sequences. J Mol Evol. 1990;31:432–42.
</cite> [<a href="https://doi.org/10.1007/BF02106057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2124631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huss%20VAR,%20Sogin%20ML.%20Phylogenetic%20position%20of%20some%20Chlorella%20species%20within%20the%20Chlorococcales%20based%20upon%20complete%20small-subunit%20ribosomal%20RNA%20sequences.%20J%20Mol%20Evol.%201990;31:432%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Ryohei U, Naoto U, Shun W, Shigeru K. Optimization of heterotrophic culture conditions for n-alkane utilization and phylogenetic position based on the 18S rDNA sequence of a thermotolerant prototheca zopfii strain. J Biosci Bioeng. 2002;94:160–5.
</cite> [<a href="https://doi.org/10.1263/jbb.94.160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16233286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ryohei%20U,%20Naoto%20U,%20Shun%20W,%20Shigeru%20K.%20Optimization%20of%20heterotrophic%20culture%20conditions%20for%20n-alkane%20utilization%20and%20phylogenetic%20position%20based%20on%20the%2018S%20rDNA%20sequence%20of%20a%20thermotolerant%20prototheca%20zopfii%20strain.%20J%20Biosci%20Bioeng.%202002;94:160%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Ueno R, Urano N, Suzuki M. Phylogeny of the non-photosynthetic green micro-algal genus prototheca (Trebouxiophyceae, Chlorophyta) and related taxa inferred from SSU and LSU ribosomal DNA partial sequence data. FEMS Microbiol Lett. 2003;223:275–80.
</cite> [<a href="https://doi.org/10.1016/S0378-1097(03)00394-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12829298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ueno%20R,%20Urano%20N,%20Suzuki%20M.%20Phylogeny%20of%20the%20non-photosynthetic%20green%20micro-algal%20genus%20prototheca%20(Trebouxiophyceae,%20Chlorophyta)%20and%20related%20taxa%20inferred%20from%20SSU%20and%20LSU%20ribosomal%20DNA%20partial%20sequence%20data.%20FEMS%20Microbiol%20Lett.%202003;223:275%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Jagielski T, et al. Cytb as a new genetic marker for differentiation of prototheca species. J Clin Microbiol. 2018;56:e00584–18.
</cite> [<a href="https://doi.org/10.1128/JCM.00584-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6156311/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30068534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jagielski%20T,%20et%20al.%20Cytb%20as%20a%20new%20genetic%20marker%20for%20differentiation%20of%20prototheca%20species.%20J%20Clin%20Microbiol.%202018;56:e00584%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Bakuła Z, et al. A first insight into the genome of <em>Prototheca wickerhamii</em>, a major causative agent of human protothecosis. BMC Genomics. 2021;22:168.
</cite> [<a href="https://doi.org/10.1186/s12864-021-07491-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7941945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33750287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baku%C5%82a%20Z,%20et%20al.%20A%20first%20insight%20into%20the%20genome%20of%20Prototheca%20wickerhamii,%20a%20major%20causative%20agent%20of%20human%20protothecosis.%20BMC%20Genomics.%202021;22:168." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Bakuła Z, et al. Sequencing and analysis of the complete organellar genomes of <em>Prototheca wickerhamii</em>. Front Plant Sci. 2020;11:1296.
</cite> [<a href="https://doi.org/10.3389/fpls.2020.01296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7492744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32983192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baku%C5%82a%20Z,%20et%20al.%20Sequencing%20and%20analysis%20of%20the%20complete%20organellar%20genomes%20of%20Prototheca%20wickerhamii.%20Front%20Plant%20Sci.%202020;11:1296." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Guo J, et al. Genome sequences of two strains of <em>Prototheca wickerhamii</em> provide insight into the protothecosis evolution. Front Cell Infect Microbiol. 2022;12:797017.
</cite> [<a href="https://doi.org/10.3389/fcimb.2022.797017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8847788/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35186789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20J,%20et%20al.%20Genome%20sequences%20of%20two%20strains%20of%20Prototheca%20wickerhamii%20provide%20insight%20into%20the%20protothecosis%20evolution.%20Front%20Cell%20Infect%20Microbiol.%202022;12:797017." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Severgnini M, et al. Genome sequencing of <em>Prototheca zopfii</em> genotypes 1 and 2 provides evidence of a severe reduction in organellar genomes. Sci Rep. 2018;8:14637.
</cite> [<a href="https://doi.org/10.1038/s41598-018-32992-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6168571/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30279542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Severgnini%20M,%20et%20al.%20Genome%20sequencing%20of%20Prototheca%20zopfii%20genotypes%201%20and%202%20provides%20evidence%20of%20a%20severe%20reduction%20in%20organellar%20genomes.%20Sci%20Rep.%202018;8:14637." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Suzuki S, Endoh R, Manabe R, Ohkuma M, Hirakawa Y. Multiple losses of photosynthesis and convergent reductive genome evolution in the colourless green algae <em>Prototheca</em>. Sci Rep. 2018;8:940.
</cite> [<a href="https://doi.org/10.1038/s41598-017-18378-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5772498/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29343788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suzuki%20S,%20Endoh%20R,%20Manabe%20R,%20Ohkuma%20M,%20Hirakawa%20Y.%20Multiple%20losses%20of%20photosynthesis%20and%20convergent%20reductive%20genome%20evolution%20in%20the%20colourless%20green%20algae%20Prototheca.%20Sci%20Rep.%202018;8:940." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Wu Q, Lan Y, Xu H, Cao Y. Characterization of the complete mitochondrial genome and phylogenetic analysis of <em>Prototheca stagnorum</em> (chlorellales: Chlorellaceae). Mitochondrial DNA Part B. 2019;4:4000–1.
</cite> [<a href="https://doi.org/10.1080/23802359.2019.1688101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7707720/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33366289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20Q,%20Lan%20Y,%20Xu%20H,%20Cao%20Y.%20Characterization%20of%20the%20complete%20mitochondrial%20genome%20and%20phylogenetic%20analysis%20of%20Prototheca%20stagnorum%20(chlorellales:%20Chlorellaceae).%20Mitochondrial%20DNA%20Part%20B.%202019;4:4000%E2%80%931." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Yan D, et al. <em>Auxenochlorella protothecoides</em> and prototheca wickerhamii plastid genome sequences give insight into the origins of non-photosynthetic algae. Sci Rep. 2015;5:14465.
</cite> [<a href="https://doi.org/10.1038/srep14465" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4585924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26403826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan%20D,%20et%20al.%20Auxenochlorella%20protothecoides%20and%20prototheca%20wickerhamii%20plastid%20genome%20sequences%20give%20insight%20into%20the%20origins%20of%20non-photosynthetic%20algae.%20Sci%20Rep.%202015;5:14465." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Zeng X, Kudinha T, Kong F, Zhang Q. Comparative genome and transcriptome study of the gene expression difference between pathogenic and environmental strains of <em>Prototheca zopfii</em>. Front Microbiol. 2019;10:443.
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.00443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6416184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30899253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng%20X,%20Kudinha%20T,%20Kong%20F,%20Zhang%20Q.%20Comparative%20genome%20and%20transcriptome%20study%20of%20the%20gene%20expression%20difference%20between%20pathogenic%20and%20environmental%20strains%20of%20Prototheca%20zopfii.%20Front%20Microbiol.%202019;10:443." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Wolff G, Plante I, Lang BF, Kück U, Burger G. Complete sequence of the mitochondrial DNA of the chlorophyte Alga <em>Prototheca wickerhamii</em>. J Mol Biol. 1994;237:75–86.
</cite> [<a href="https://doi.org/10.1006/jmbi.1994.1210" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8133522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolff%20G,%20Plante%20I,%20Lang%20BF,%20K%C3%BCck%20U,%20Burger%20G.%20Complete%20sequence%20of%20the%20mitochondrial%20DNA%20of%20the%20chlorophyte%20Alga%20Prototheca%20wickerhamii.%20J%20Mol%20Biol.%201994;237:75%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Maciszewski K, et al. Reduced plastid genomes of colorless facultative pathogens <em>Prototheca</em> (Chlorophyta) are retained for membrane transport genes . 2024;BMC Biol 22, 294.
</cite> [<a href="https://doi.org/10.1186/s12915-024-02089-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11657870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39696433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maciszewski%20K,%20et%20al.%20Reduced%20plastid%20genomes%20of%20colorless%20facultative%20pathogens%20Prototheca%20(Chlorophyta)%20are%20retained%20for%20membrane%20transport%20genes%20.%202024;BMC%20Biol%2022,%20294." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Jagielski T, et al. An optimized method for high quality DNA extraction from microalga <em>Prototheca wickerhamii</em> for genome sequencing. Plant Methods. 2017;13:77.
</cite> [<a href="https://doi.org/10.1186/s13007-017-0228-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5627410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29026433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jagielski%20T,%20et%20al.%20An%20optimized%20method%20for%20high%20quality%20DNA%20extraction%20from%20microalga%20Prototheca%20wickerhamii%20for%20genome%20sequencing.%20Plant%20Methods.%202017;13:77." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Chen S, Zhou Y, Chen Y, Gu J. Fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics. 2018;34:i884–90.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20S,%20Zhou%20Y,%20Chen%20Y,%20Gu%20J.%20Fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor.%20Bioinformatics.%202018;34:i884%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>De Coster W, D’Hert S, Schultz DT, Cruts M, Van Broeckhoven C. NanoPack: visualizing and processing long-read sequencing data. Bioinformatics. 2018;34:2666–9.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6061794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29547981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Coster%20W,%20D%E2%80%99Hert%20S,%20Schultz%20DT,%20Cruts%20M,%20Van%20Broeckhoven%20C.%20NanoPack:%20visualizing%20and%20processing%20long-read%20sequencing%20data.%20Bioinformatics.%202018;34:2666%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Koren S, et al. Canu: scalable and accurate long-read assembly via adaptive <em>k</em> -mer weighting and repeat separation. Genome Res. 2017;27:722–36.
</cite> [<a href="https://doi.org/10.1101/gr.215087.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5411767/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28298431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koren%20S,%20et%20al.%20Canu:%20scalable%20and%20accurate%20long-read%20assembly%20via%20adaptive%20k%20-mer%20weighting%20and%20repeat%20separation.%20Genome%20Res.%202017;27:722%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Kolmogorov M, Yuan J, Lin Y, Pevzner PA. Assembly of long, error-prone reads using repeat graphs. Nat Biotechnol. 2019;37:540–6.
</cite> [<a href="https://doi.org/10.1038/s41587-019-0072-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30936562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kolmogorov%20M,%20Yuan%20J,%20Lin%20Y,%20Pevzner%20PA.%20Assembly%20of%20long,%20error-prone%20reads%20using%20repeat%20graphs.%20Nat%20Biotechnol.%202019;37:540%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Heinl S, et al. Insights into the completely annotated genome of Lactobacillus buchneri CD034, a strain isolated from stable grass silage. J Biotechnol. 2012;161:153–66.
</cite> [<a href="https://doi.org/10.1016/j.jbiotec.2012.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22465289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heinl%20S,%20et%20al.%20Insights%20into%20the%20completely%20annotated%20genome%20of%20Lactobacillus%20buchneri%20CD034,%20a%20strain%20isolated%20from%20stable%20grass%20silage.%20J%20Biotechnol.%202012;161:153%E2%80%9366." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Schwientek P, et al. The complete genome sequence of the acarbose producer actinoplanes sp. SE50/110. BMC Genomics. 2012;13:112.
</cite> [<a href="https://doi.org/10.1186/1471-2164-13-112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3364876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22443545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schwientek%20P,%20et%20al.%20The%20complete%20genome%20sequence%20of%20the%20acarbose%20producer%20actinoplanes%20sp.%20SE50/110.%20BMC%20Genomics.%202012;13:112." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Wibberg D, et al. Establishment and interpretation of the genome sequence of the phytopathogenic fungus rhizoctonia Solani AG1-IB isolate 7/3/14. J Biotechnol. 2013;167:142–55.
</cite> [<a href="https://doi.org/10.1016/j.jbiotec.2012.12.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23280342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wibberg%20D,%20et%20al.%20Establishment%20and%20interpretation%20of%20the%20genome%20sequence%20of%20the%20phytopathogenic%20fungus%20rhizoctonia%20Solani%20AG1-IB%20isolate%207/3/14.%20J%20Biotechnol.%202013;167:142%E2%80%9355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Lang BF, et al. Mitochondrial genome annotation with mfannot: a critical analysis of gene identification and gene model prediction. Front Plant Sci. 2023;14:1222186.
</cite> [<a href="https://doi.org/10.3389/fpls.2023.1222186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10352661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37469769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lang%20BF,%20et%20al.%20Mitochondrial%20genome%20annotation%20with%20mfannot:%20a%20critical%20analysis%20of%20gene%20identification%20and%20gene%20model%20prediction.%20Front%20Plant%20Sci.%202023;14:1222186." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Eikmeyer F, et al. The complete genome sequences of four new IncN plasmids from wastewater treatment plant effluent provide new insights into IncN plasmid diversity and evolution. Plasmid. 2012;68:13–24.
</cite> [<a href="https://doi.org/10.1016/j.plasmid.2012.01.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22326849/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eikmeyer%20F,%20et%20al.%20The%20complete%20genome%20sequences%20of%20four%20new%20IncN%20plasmids%20from%20wastewater%20treatment%20plant%20effluent%20provide%20new%20insights%20into%20IncN%20plasmid%20diversity%20and%20evolution.%20Plasmid.%202012;68:13%E2%80%9324." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Blom J, et al. EDGAR 2.0: an enhanced software platform for comparative gene content analyses. Nucleic Acids Res. 2016;44:W22–8.
</cite> [<a href="https://doi.org/10.1093/nar/gkw255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4987874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27098043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blom%20J,%20et%20al.%20EDGAR%202.0:%20an%20enhanced%20software%20platform%20for%20comparative%20gene%20content%20analyses.%20Nucleic%20Acids%20Res.%202016;44:W22%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Blom J, et al. EDGAR: A software framework for the comparative analysis of prokaryotic genomes. BMC Bioinformatics. 2009;10:154.
</cite> [<a href="https://doi.org/10.1186/1471-2105-10-154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2696450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19457249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blom%20J,%20et%20al.%20EDGAR:%20A%20software%20framework%20for%20the%20comparative%20analysis%20of%20prokaryotic%20genomes.%20BMC%20Bioinformatics.%202009;10:154." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Dieckmann MA, et al. EDGAR3.0: comparative genomics and phylogenomics on a scalable infrastructure. Nucleic Acids Res. 2021;49:W185–92.
</cite> [<a href="https://doi.org/10.1093/nar/gkab341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8262741/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33988716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dieckmann%20MA,%20et%20al.%20EDGAR3.0:%20comparative%20genomics%20and%20phylogenomics%20on%20a%20scalable%20infrastructure.%20Nucleic%20Acids%20Res.%202021;49:W185%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Konstantinidis KT, Tiedje JM. Genomic insights that advance the species definition for prokaryotes. Proc Natl Acad Sci U S A. 2005;102:2567–72.
</cite> [<a href="https://doi.org/10.1073/pnas.0409727102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC549018/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15701695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Konstantinidis%20KT,%20Tiedje%20JM.%20Genomic%20insights%20that%20advance%20the%20species%20definition%20for%20prokaryotes.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202005;102:2567%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Goris J, et al. DNA-DNA hybridization values and their relationship to whole-genome sequence similarities. Int J Syst Evol Microbiol. 2007;57:81–91.
</cite> [<a href="https://doi.org/10.1099/ijs.0.64483-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17220447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goris%20J,%20et%20al.%20DNA-DNA%20hybridization%20values%20and%20their%20relationship%20to%20whole-genome%20sequence%20similarities.%20Int%20J%20Syst%20Evol%20Microbiol.%202007;57:81%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Sullivan MJ, Petty NK, Beatson SA. Easyfig: a genome comparison visualizer. Bioinformatics. 2011;27:1009–10.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3065679/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21278367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sullivan%20MJ,%20Petty%20NK,%20Beatson%20SA.%20Easyfig:%20a%20genome%20comparison%20visualizer.%20Bioinformatics.%202011;27:1009%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J Mol Biol. 1990;215:403–10.
</cite> [<a href="https://doi.org/10.1016/S0022-2836(05)80360-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2231712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Altschul%20SF,%20Gish%20W,%20Miller%20W,%20Myers%20EW,%20Lipman%20DJ.%20Basic%20local%20alignment%20search%20tool.%20J%20Mol%20Biol.%201990;215:403%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Edgar RC. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 2004;32:1792–7.
</cite> [<a href="https://doi.org/10.1093/nar/gkh340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC390337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15034147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edgar%20RC.%20MUSCLE:%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput.%20Nucleic%20Acids%20Res.%202004;32:1792%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Capella-Gutiérrez S, Silla-Martínez JM, Gabaldón T. TrimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. Bioinformatics. 2009;25:1972–3.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2712344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Capella-Guti%C3%A9rrez%20S,%20Silla-Mart%C3%ADnez%20JM,%20Gabald%C3%B3n%20T.%20TrimAl:%20a%20tool%20for%20automated%20alignment%20trimming%20in%20large-scale%20phylogenetic%20analyses.%20Bioinformatics.%202009;25:1972%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Minh BQ, et al. IQ-TREE 2: new models and efficient methods for phylogenetic inference in the genomic era. Mol Biol Evol. 2020;37:1530–4.
</cite> [<a href="https://doi.org/10.1093/molbev/msaa015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7182206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32011700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Minh%20BQ,%20et%20al.%20IQ-TREE%202:%20new%20models%20and%20efficient%20methods%20for%20phylogenetic%20inference%20in%20the%20genomic%20era.%20Mol%20Biol%20Evol.%202020;37:1530%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Kalyaanamoorthy S, Minh BQ, Wong TKF, Von Haeseler A, Jermiin LS. ModelFinder: fast model selection for accurate phylogenetic estimates. Nat Methods. 2017;14:587–9.
</cite> [<a href="https://doi.org/10.1038/nmeth.4285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5453245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28481363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kalyaanamoorthy%20S,%20Minh%20BQ,%20Wong%20TKF,%20Von%20Haeseler%20A,%20Jermiin%20LS.%20ModelFinder:%20fast%20model%20selection%20for%20accurate%20phylogenetic%20estimates.%20Nat%20Methods.%202017;14:587%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Hoang DT, Chernomor O, Von Haeseler A, Minh BQ, Vinh LS. UFBoot2: improving the ultrafast bootstrap approximation. Mol Biol Evol. 2018;35:518–22.
</cite> [<a href="https://doi.org/10.1093/molbev/msx281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5850222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29077904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoang%20DT,%20Chernomor%20O,%20Von%20Haeseler%20A,%20Minh%20BQ,%20Vinh%20LS.%20UFBoot2:%20improving%20the%20ultrafast%20bootstrap%20approximation.%20Mol%20Biol%20Evol.%202018;35:518%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Smith DR, Lee RW. Nucleotide diversity in the mitochondrial and nuclear compartments of <em>Chlamydomonas reinhardtii:</em> investigating the origins of genome architecture. BMC Evol Biol. 2008;8:156.
</cite> [<a href="https://doi.org/10.1186/1471-2148-8-156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2412866/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18495022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith%20DR,%20Lee%20RW.%20Nucleotide%20diversity%20in%20the%20mitochondrial%20and%20nuclear%20compartments%20of%20Chlamydomonas%20reinhardtii:%20investigating%20the%20origins%20of%20genome%20architecture.%20BMC%20Evol%20Biol.%202008;8:156." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Grigoriev A. Analyzing genomes with cumulative skew diagrams. Nucleic Acids Res. 1998;26:2286–90.
</cite> [<a href="https://doi.org/10.1093/nar/26.10.2286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC147580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9580676/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grigoriev%20A.%20Analyzing%20genomes%20with%20cumulative%20skew%20diagrams.%20Nucleic%20Acids%20Res.%201998;26:2286%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Formighieri EF, et al. The mitochondrial genome of the phytopathogenic basidiomycete moniliophthora perniciosa is 109 kb in size and contains a stable integrated plasmid. Mycol Res. 2008;112:1136–52.
</cite> [<a href="https://doi.org/10.1016/j.mycres.2008.04.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18786820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Formighieri%20EF,%20et%20al.%20The%20mitochondrial%20genome%20of%20the%20phytopathogenic%20basidiomycete%20moniliophthora%20perniciosa%20is%20109%C2%A0kb%20in%20size%20and%20contains%20a%20stable%20integrated%20plasmid.%20Mycol%20Res.%202008;112:1136%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Adams K. Evolution of mitochondrial gene content: gene loss and transfer to the nucleus. Mol Phylogenet Evol. 2003;29:380–95.
</cite> [<a href="https://doi.org/10.1016/s1055-7903(03)00194-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14615181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adams%20K.%20Evolution%20of%20mitochondrial%20gene%20content:%20gene%20loss%20and%20transfer%20to%20the%20nucleus.%20Mol%20Phylogenet%20Evol.%202003;29:380%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Mitochondrial. <em>genome evolution</em>. (Elsevier, 2012).</cite>
</li>
<li id="CR50">
<span class="label">50.</span><cite>Lang BF, Laforest M-J, Burger G. Mitochondrial introns: a critical view. Trends Genet. 2007;23:119–25.
</cite> [<a href="https://doi.org/10.1016/j.tig.2007.01.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17280737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lang%20BF,%20Laforest%20M-J,%20Burger%20G.%20Mitochondrial%20introns:%20a%20critical%20view.%20Trends%20Genet.%202007;23:119%E2%80%9325." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Pombert J-F, Keeling PJ. The mitochondrial genome of the entomoparasitic green Alga <em>Helicosporidium</em>. PLoS ONE. 2010;5:e8954.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0008954" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2813288/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20126458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pombert%20J-F,%20Keeling%20PJ.%20The%20mitochondrial%20genome%20of%20the%20entomoparasitic%20green%20Alga%20Helicosporidium.%20PLoS%20ONE.%202010;5:e8954." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Schneiker-Bekel S, et al. The complete genome sequence of the dominant <em>Sinorhizobium meliloti</em> field isolate SM11 extends the S. meliloti pan-genome. J Biotechnol. 2011;155:20–33.
</cite> [<a href="https://doi.org/10.1016/j.jbiotec.2010.12.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21396969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schneiker-Bekel%20S,%20et%20al.%20The%20complete%20genome%20sequence%20of%20the%20dominant%20Sinorhizobium%20meliloti%20field%20isolate%20SM11%20extends%20the%20S.%20meliloti%20pan-genome.%20J%20Biotechnol.%202011;155:20%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Smith DR, Keeling PJ. Mitochondrial and plastid genome architecture: Reoccurring themes, but significant differences at the extremes. Proc Natl Acad Sci 2015;112:10177–10184.
</cite> [<a href="https://doi.org/10.1073/pnas.1422049112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4547224/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25814499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith%20DR,%20Keeling%20PJ.%20Mitochondrial%20and%20plastid%20genome%20architecture:%20Reoccurring%20themes,%20but%20significant%20differences%20at%20the%20extremes.%20Proc%20Natl%20Acad%20Sci%202015;112:10177%E2%80%9310184." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Wibberg D, et al. High quality genome sequences of thirteen hypoxylaceae (Ascomycota) strengthen the phylogenetic family backbone and enable the discovery of new taxa. Fungal Divers. 2021;106:7–28.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wibberg%20D,%20et%20al.%20High%20quality%20genome%20sequences%20of%20thirteen%20hypoxylaceae%20(Ascomycota)%20strengthen%20the%20phylogenetic%20family%20backbone%20and%20enable%20the%20discovery%20of%20new%20taxa.%20Fungal%20Divers.%202021;106:7%E2%80%9328." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Wibberg D, et al. Draft genome sequence of the potato pathogen rhizoctonia Solani AG3-PT isolate Ben3. Arch Microbiol. 2017;199:1065–8.
</cite> [<a href="https://doi.org/10.1007/s00203-017-1394-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28597196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wibberg%20D,%20et%20al.%20Draft%20genome%20sequence%20of%20the%20potato%20pathogen%20rhizoctonia%20Solani%20AG3-PT%20isolate%20Ben3.%20Arch%20Microbiol.%202017;199:1065%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369231/bin/12864_2025_11952_MOESM1_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">12864_2025_11952_MOESM1_ESM.png</a><sup> (646.3KB, png) </sup><p>Supplementary Material 1: Phylogenetic maximum-likelihood tree of <em>Prototheca</em> based on the cytochrome b protein (<em>cytb</em>), comprising 384 unambiguously aligned sites under the model mtZOA + I + R3. Related Chlorellaceae species were selected as the outgroup. Phylogenetic support assessment is based on 1000 non-parametric bootstrap replicates; only support values &gt; 50% are shown, and black dots indicate maximum support.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369231/bin/12864_2025_11952_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (24.5KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Nucleotide sequence data for all 18 <em>Prototheca</em> mt genomes are deposited in the GenBank/EBI/DDBJ database under the Bioproject No. PRJEB82245. The original alignments are accessible through doi: 10.6084/m9.figshare.29328425. Furthermore, data generated or analysed during this study are included in this published article and its supplementary information files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Genomics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12864-025-11952-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12864_2025_Article_11952.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12369231/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12369231/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369231%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369231/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12369231/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12369231/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40836223/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12369231/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40836223/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12369231/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12369231/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="YxrE7ZJ1c6S6Pd2sn4JJZYWkZUWN7ZxhIWEWruuw5S2INxm2BnlNk3LktEIhyIva">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
