
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Identification of models describing gene expression data leveraging machine learning methods - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532D3A8AF202F3052D3A000319CA08.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="interfacefoc">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371343/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Interface Focus">
<meta name="citation_title" content="Identification of models describing gene expression data leveraging machine learning methods">
<meta name="citation_author" content="Lucas F Jansen Klomp">
<meta name="citation_author_institution" content="Mathematics of Imaging &amp; AI, Department of Applied Mathematics, University of Twente, Enschede, The Netherlands">
<meta name="citation_author_institution" content="Developmental BioEngineering, University of Twente, Enschede, The Netherlands">
<meta name="citation_author" content="Elena Queirolo">
<meta name="citation_author_institution" content="IRMAR, University of Rennes, Rennes, France">
<meta name="citation_author" content="Janine N Post">
<meta name="citation_author_institution" content="Developmental BioEngineering, University of Twente, Enschede, The Netherlands">
<meta name="citation_author" content="Hil G E Meijer">
<meta name="citation_author_institution" content="Mathematics of Imaging &amp; AI, Department of Applied Mathematics, University of Twente, Enschede, The Netherlands">
<meta name="citation_author" content="Christoph Brune">
<meta name="citation_author_institution" content="Mathematics of Imaging &amp; AI, Department of Applied Mathematics, University of Twente, Enschede, The Netherlands">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="15">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="20250014">
<meta name="citation_doi" content="10.1098/rsfs.2025.0014">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371343/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371343/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371343/pdf/rsfs.2025.0014.pdf">
<meta name="description" content="Mechanistic ordinary differential equation models of gene regulatory networks are a valuable tool for understanding biological processes that occur inside a cell, and they allow for the formulation of novel hypotheses on the mechanisms underlying ...">
<meta name="og:title" content="Identification of models describing gene expression data leveraging machine learning methods">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Mechanistic ordinary differential equation models of gene regulatory networks are a valuable tool for understanding biological processes that occur inside a cell, and they allow for the formulation of novel hypotheses on the mechanisms underlying ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371343/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371343">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1098/rsfs.2025.0014"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/rsfs.2025.0014.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371343%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371343/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371343/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371343/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-interfacefoc.gif" alt="Interface Focus logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Interface Focus" title="Link to Interface Focus" shape="default" href="http://rsfs.royalsocietypublishing.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Interface Focus</button></div>. 2025 Aug 22;15(3):20250014. doi: <a href="https://doi.org/10.1098/rsfs.2025.0014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1098/rsfs.2025.0014</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Interface%20Focus%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Interface%20Focus%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Interface%20Focus%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Interface%20Focus%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>
Identification of models describing gene expression data leveraging machine learning methods</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jansen%20Klomp%20LF%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Lucas F Jansen Klomp</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Lucas F Jansen Klomp</span></h3>
<div class="p">
<sup><sup>1</sup></sup>
Mathematics of Imaging &amp; AI, Department of Applied Mathematics, University of Twente, Enschede, The Netherlands
</div>
<div class="p">
<sup><sup>2</sup></sup>
Developmental BioEngineering, University of Twente, Enschede, The Netherlands
</div>
<div>Conceptualization, Formal analysis, Investigation, Methodology, Software, Visualization, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jansen%20Klomp%20LF%22%5BAuthor%5D" class="usa-link"><span class="name western">Lucas F Jansen Klomp</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Queirolo%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Elena Queirolo</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Elena Queirolo</span></h3>
<div class="p">
<sup><sup>3</sup></sup>
IRMAR, University of Rennes, Rennes, France
</div>
<div>Conceptualization, Formal analysis, Investigation, Methodology, Software, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Queirolo%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Elena Queirolo</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Post%20JN%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Janine N Post</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Janine N Post</span></h3>
<div class="p">
<sup><sup>2</sup></sup>
Developmental BioEngineering, University of Twente, Enschede, The Netherlands
</div>
<div>Conceptualization, Supervision, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Post%20JN%22%5BAuthor%5D" class="usa-link"><span class="name western">Janine N Post</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meijer%20HGE%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Hil G E Meijer</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Hil G E Meijer</span></h3>
<div class="p">
<sup><sup>1</sup></sup>
Mathematics of Imaging &amp; AI, Department of Applied Mathematics, University of Twente, Enschede, The Netherlands
</div>
<div>Conceptualization, Supervision, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meijer%20HGE%22%5BAuthor%5D" class="usa-link"><span class="name western">Hil G E Meijer</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brune%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Christoph Brune</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Christoph Brune</span></h3>
<div class="p">
<sup><sup>1</sup></sup>
Mathematics of Imaging &amp; AI, Department of Applied Mathematics, University of Twente, Enschede, The Netherlands
</div>
<div>Conceptualization, Methodology, Supervision, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brune%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christoph Brune</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>
Mathematics of Imaging &amp; AI, Department of Applied Mathematics, University of Twente, Enschede, The Netherlands
</div>
<div id="aff2">
<sup><sup>2</sup></sup>
Developmental BioEngineering, University of Twente, Enschede, The Netherlands
</div>
<div id="aff3">
<sup><sup>3</sup></sup>
IRMAR, University of Rennes, Rennes, France
</div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>One contribution of 6 to a theme issue ‘Combinatorial models for evolving representation of dynamical behaviours in biological networks’.</p></div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Lucas F Jansen Klomp</span></strong>: <span class="role">Conceptualization, Formal analysis, Investigation, Methodology, Software, Visualization, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Elena Queirolo</span></strong>: <span class="role">Conceptualization, Formal analysis, Investigation, Methodology, Software, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Janine N Post</span></strong>: <span class="role">Conceptualization, Supervision, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Hil G E Meijer</span></strong>: <span class="role">Conceptualization, Supervision, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Christoph Brune</span></strong>: <span class="role">Conceptualization, Methodology, Supervision, Writing – review and editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 28; Revised 2025 Jun 13; Accepted 2025 Jun 27; Collection date 2025 Aug 22.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors.</div>
<p>
Published by the Royal Society under the terms of the Creative Commons Attribution License <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>, which permits unrestricted use, provided the original author and source are credited.
</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371343</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Mechanistic 
ordinary differential equation models of gene regulatory networks
are a valuable tool for understanding biological processes that occur inside a cell, and they allow for the formulation of novel hypotheses on the mechanisms underlying these processes. Although data-driven methods for inferring these mechanistic models are becoming more prevalent, it is often unclear how recent advances in machine learning can be used effectively without jeopardi
zing the interpretability of the resulting models. In this work, we present a framework to leverage neural networks for the identification of data-driven models for time-dependent intracellular processes, such as cell differentiation. In particular, we use a graph autoencoder model to suggest novel connections in a gene regulatory network. We show how the improvement of the graph suggested using this neural network leads to the generation of hypotheses on the dynamics of the resulting identified dynamical system.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> 
ODE modelling, graph neural network, gene regulatory network, scRNA-seq</p></section></section><section id="s1"><h2 class="pmc_sec_title">1.  
Introduction</h2>
<p>With the advent of single-cell sequencing technologies that quantify gene expression, computational methods have become paramount in understanding intracellular biological processes [
<a href="#B1" class="usa-link" aria-describedby="B1">1</a>,<a href="#B2" class="usa-link" aria-describedby="B2">2</a>]. Gene regulatory networks (GRNs), for example, describe the connections between downstream transcription factors and target genes [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>
]. Such networks are represented as directed graphs, indicating the relations between different genes in the network. Many methods exist to infer GRNs from gene expression data, and while these methods are often well suited to tasks such as clustering and identifying key nodes, individual functional relations between nodes in the network are typically not well represented [<a href="#B4" class="usa-link" aria-describedby="B4">4</a>–<a href="#B7" class="usa-link" aria-describedby="B7">7</a>
]. This lack of reliable representation limits the use of data-driven inferred GRNs for dynamical modelling of intracellular processes, where such functional relations are needed to obtain accurate predictions. For example, when the model parameters are perturbed, different GRNs can result in remarkably different behaviour. In this work, we ask how we can leverage successes in the machine learning field to identify more informative mechanistic models describing intracellular processes based on single-cell RNA-sequencing data (scRNA-seq).</p>
<p>
Nowadays, scRNA-seq data are commonly used in computational methods, and consist
of the gene expression of many (approx. 500−10 000) cells harvested from the
same population 
[
<a href="#B1" class="usa-link" aria-describedby="B1">1</a>,<a href="#B2" class="usa-link" aria-describedby="B2">2</a>
]. Hence, the data consist of an <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui1"><math id="u1" display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>×</mo></mrow><mrow><mi>m</mi></mrow></math></span> matrix, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui2"><math id="u2" display="inline" overflow="linebreak"><mi>n</mi></math></span> is the number of measured genes and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui3"><math id="u3" display="inline" overflow="linebreak"><mi>m</mi></math></span> is the number of cells. Sometimes, time points can also be associated with individual cells, either because cells were harvested at specific time points during the experiment or through computational methods inferring pseudotime trajectories [<a href="#B8" class="usa-link" aria-describedby="B8">8</a>,<a href="#B9" class="usa-link" aria-describedby="B9">9</a>
]. The high resolution of scRNA-seq allows for the application of more advanced techniques that cannot be used on bulk data. In particular, the large number of samples makes machine learning a promising avenue to explore and obtain meaningful insights from single-cell data.</p>
<p>One class of machine learning methods 
used to study scRNA-seq data is tasked with recovering the underlying 
GRN based on (in principle) data collected from one population of cells. These methods often rely on an encoding
–decoding scheme to predict links between nodes in the 
GRN. Examples include methods like DeepSEM [<a href="#B10" class="usa-link" aria-describedby="B10">10</a>
], which uses structural equation 
modelling in its encoder and decoder schemes, and GNNLink [<a href="#B11" class="usa-link" aria-describedby="B11">11</a>
], DeepRIG [<a href="#B12" class="usa-link" aria-describedby="B12">12</a>
] and DGCGRN [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>
], which leverage the inherent graph structure underlying the data by using a graph autoencoder (GAE) to learn node embeddings and subsequently predict links between genes. Another recent method, GENELink, uses the graph attention mechanism to recover links between nodes [<a href="#B14" class="usa-link" aria-describedby="B14">14</a>
]. In general, deep learning-based GRN inference methods report much higher accuracy than classical statistics-based methods, although good interpretability of the resulting networks is rarely achieved. To remedy this, we explore how we can fit a biologically interpretable model using the output of a graph autoencoder.</p>
<p>Specifically, we focus on methods that retrieve a dynamical model based on data. In the literature, the focus is sometimes on recovering the dynamics underlying a temporal process captured in the single-cell data as its own goal, whereas in other cases, the dynamics 
are instrumental 
in obtaining a GRN. Some notable references in the field include the method SCODE, which aims to reconstruct the GRN based on inferred parameters of a linear ordinary differential equation
(ODE) model [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>
]. In RNAForecaster, a neural ODE (nODE), an ODE where a neural network represents the right-hand side, is used to recover gene expression dynamics [<a href="#B16" class="usa-link" aria-describedby="B16">16</a>
]. Furthermore, PHOENIX [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>
] aims to build a more biologically informed model and fits a model based on Hill functions to the data while using a linear ODE to regularize model training.</p>
<p>From a practical perspective, the inferred computational models resulting from these machine learning methods can be used to identify key genes influencing the biological process. Perhaps more importantly, the models can be used to obtain viable hypotheses on what would happen if particular genes are knocked down or if connections are modified. In this second use case, having a precise view of which functional relations exist between genes is crucial.</p>
<p>It is natural to encode a GRN as a directed graph, in line with a variety of other works such as [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>,<a href="#B18" class="usa-link" aria-describedby="B18">18</a>
], where each vertex <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui4"><math id="u4" display="inline" overflow="linebreak"><msub><mi>x</mi><mi>i</mi></msub></math></span> represents the expression of a given gene and each edge <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui5"><math id="u5" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mi>i</mi></msub><mo stretchy="false">→</mo></mrow><mrow><msub><mi>x</mi><mi>j</mi></msub></mrow></math></span> indicates that the expression of a target gene <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui6"><math id="u6" display="inline" overflow="linebreak"><msub><mi>x</mi><mi>j</mi></msub></math></span> depends on the expression of the source gene <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui7"><math id="u7" display="inline" overflow="linebreak"><msub><mi>x</mi><mi>i</mi></msub></math></span>. We then denote with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui8"><math id="u8" display="inline" overflow="linebreak"><mi>V</mi></math></span> the set of vertices of the graph, that is, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui9"><math id="u9" display="inline" overflow="linebreak"><mrow><mi>V</mi><mo>=</mo></mrow><mrow><mo form="prefix" stretchy="false">{</mo><msub><mi>x</mi><mn>1</mn></msub><mo separator="true">,</mo><msub><mi>x</mi><mn>2</mn></msub><mo separator="true">,</mo><mo>…</mo><mo separator="true">,</mo><msub><mi>x</mi><mi>K</mi></msub><mo form="postfix" stretchy="false">}</mo></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui10"><math id="u10" display="inline" overflow="linebreak"><mi>E</mi></math></span> the set of edges. A directed graph can also be represented as an adjacency matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui11"><math id="u11" display="inline" overflow="linebreak"><mrow><mi>A</mi><mo>∈</mo></mrow><mrow><mo form="prefix" stretchy="false">[</mo><mn>0,1</mn><msup><mo form="postfix" stretchy="false">]</mo><mrow><mi>K</mi><mo>×</mo><mi>K</mi></mrow></msup></mrow></math></span>, where we interpret <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui12"><math id="u12" display="inline" overflow="linebreak"><msub><mi>A</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> to be the probability of the existence of the edge <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui13"><math id="u13" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mi>i</mi></msub><mo stretchy="false">→</mo></mrow><mrow><msub><mi>x</mi><mi>j</mi></msub></mrow></math></span>. Based on the given GRN, we can construct a model of the form
</p>
<table class="disp-formula p" id="uFD1"><tr>
<td class="formula"><math id="um1" display="block" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mi>τ</mi><mrow><mover><mi>x</mi><mo>˙</mo></mover></mrow><mo>=</mo><mo>−</mo><mi>x</mi><mo>+</mo><mi>H</mi><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><msub><mi>W</mi><mrow><mtext>in</mtext></mrow></msub><mo>,</mo><mi>u</mi><mo stretchy="false">)</mo><mo>.</mo></mrow></mstyle></math></td>
<td class="label">(1.1)</td>
</tr></table>
<p>Our challenge is to find the optimal parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui14"><math id="u14" display="inline" overflow="linebreak"><mrow><mi>p</mi><mo>=</mo></mrow><mrow><mo form="prefix" stretchy="false">(</mo><msub><mi>W</mi><mtext>in</mtext></msub><mo separator="true">,</mo><mi>u</mi><mo separator="true">,</mo><mi>τ</mi><mo form="postfix" stretchy="false">)</mo></mrow></math></span>, where the sense of this optimality will be specified later in
§
2.2
. Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui15"><math id="u15" display="inline" overflow="linebreak"><mi>σ</mi></math></span> is an activation function. This formulation resembles commonly used models in biochemical applications, where Hill functions are often used to describe the nonlinearity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui16"><math id="u16" display="inline" overflow="linebreak"><mi>σ</mi></math></span>. We say that the parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui17"><math id="u17" display="inline" overflow="linebreak"><mi>p</mi></math></span> are compatible with a graph given by the adjacency matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui18"><math id="u18" display="inline" overflow="linebreak"><mi>A</mi></math></span> if the matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui19"><math id="u19" display="inline" overflow="linebreak"><msub><mi>W</mi><mrow><mtext>in</mtext><mo separator="true">,</mo><mi>i</mi><mi>j</mi></mrow></msub></math></span> is non-zero if and only if <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui20"><math id="u20" display="inline" overflow="linebreak"><msub><mi>A</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> is non-zero for all <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui21"><math id="u21" display="inline" overflow="linebreak"><mrow><mi>i</mi><mo separator="true">,</mo></mrow><mrow><mi>j</mi></mrow></math></span>. An advantage of this parameter formulation is that the parameter size is completely determined by the phase space dimension, that is, the number of 
modelled genes, rather than the specific structure of the GRN. This also permits us to compare all parameters associated with a given time series, even when they are not compatible with the same graph.</p>
<p>In this 
article, we present a framework that combines machine learning methods to infer a model of the form (1.1) (an overview of the framework is shown in <a href="#F1" class="usa-link">
figure 1
</a>). In particular, we use a graph autoencoder to update and improve a GRN obtained from prior knowledge or other GRN inference methods. Based on the improved GRN, we fit the parameters of a mechanistic nonlinear ODE model (1.1) to the available data.</p>
<figure class="fig xbox font-sm" id="F1" title="Overview of our proposed framework."><h3 class="obj_head">Figure 1. </h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371343_rsfs.2025.0014.f001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97e1/12371343/e3d6fde1bc03/rsfs.2025.0014.f001.jpg" loading="lazy" height="349" width="735" alt="Overview of our proposed framework."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overview of our proposed framework. We use scRNA-seq data to adjust an available 
GRN using a graph autoencoder model. Subsequently, we use the adjusted graph topology as a basis for an ODE model describing the scRNA-seq. By fitting the parameters in this ODE model, we obtain an interpretable ODE model that can be used to describe the data and to predict what happens when chosen parameters are adjusted.</p></figcaption></figure><p>Our framework highlights how recent successes in the deep learning field can be leveraged in the construction of mechanistic ODE models. We evaluate the predictive properties of the resulting ODE models, including their suitability for extensive parameter sensitivity analysis through a bifurcation analysis or even more rigorous identification of parameter regimes around the inferred model parameters [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>
].</p></section><section id="s2"><h2 class="pmc_sec_title">2.  
Methods</h2>
<p>In this 
section, we first introduce a GAE to improve
GRNs based on scRNA-seq data. Subsequently, we describe how the output of this GAE is used to infer parameters of a nonlinear ODE model. Finally, we introduce benchmarking data to test the performance of our framework.</p>
<p>When considering a GRN, the dynamics 
are strictly related to the graph that supports it. In many cases, such as models with ramp functions as nonlinearities, it is possible to construct a 
characterization of the supported dynamical behaviour across the entire parameter space
(
see, for 
example, the DSGRN framework
[<a href="#B18" class="usa-link" aria-describedby="B18">18</a>
]) or explore parameter regions using a Boolean network view [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>
]. Recent results indicate that such results also apply to a much broader class of nonlinearities [<a href="#B20" class="usa-link" aria-describedby="B20">20</a>,<a href="#B21" class="usa-link" aria-describedby="B21">21</a>
]. While the graph structure is known to be critical in the definition of the dynamics, it is also
well known
that the effects of the graph structure on dynamics are far from being fully understood. For this reason, we first concentrate on the task of improving known graphs using a GAE before fitting parameters of an ODE model.</p>
<section id="s2-1"><h3 class="pmc_sec_title">2.1.  
Graph autoencoder to predict missing edges in a
gene regulatory network
</h3>
<p>Assuming that an initial (imperfect) graph is known, we want to improve its accuracy based on available scRNA-seq data. We assume that a graph is given where one or a few edges that are in the true graph are obscured. Specifically, a prior knowledge graph <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui22"><math id="u22" display="inline" overflow="linebreak"><mrow><msub><mi>G</mi><mtext>PK</mtext></msub><mo>=</mo></mrow><mrow><mo form="prefix" stretchy="false">(</mo><mi>V</mi><mo separator="true">,</mo><mi>E</mi><mo form="postfix" stretchy="false">)</mo></mrow></math></span> and the associated adjacency matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui23"><math id="u23" display="inline" overflow="linebreak"><mi>A</mi></math></span> are given as input imbued with node features <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui24"><math id="u24" display="inline" overflow="linebreak"><mrow><mi>X</mi><mo>∈</mo></mrow><mrow><msup><mi>ℝ</mi><mrow><mo>|</mo><mi>V</mi><mo>|</mo><mo>×</mo><mi>N</mi></mrow></msup></mrow></math></span>
, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui25"><math id="u25" display="inline" overflow="linebreak"><mi>N</mi></math></span> is the number of cells included in the scRNA-seq dataset and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui26"><math id="u26" display="inline" overflow="linebreak"><mrow><mo>|</mo><mi>V</mi><mo>|</mo></mrow></math></span> is the number of nodes in the GRN. In our case, we assume <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui27"><math id="u27" display="inline" overflow="linebreak"><mi>X</mi></math></span> includes gene expression data measured in many cells, and is stored in pairs that contain cells along with associated time points per cell, that is, we also have access to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui28"><math id="u28" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>∈</mo></mrow><mrow><msup><mi>ℝ</mi><mi>N</mi></msup></mrow></math></span> that includes these time points.</p>
<p>In this 
section, we use <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui29"><math id="u29" display="inline" overflow="linebreak"><msub><mi>G</mi><mtext>PK</mtext></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui30"><math id="u30" display="inline" overflow="linebreak"><mi>X</mi></math></span> as inputs for a
GAE
, with the explicit goal of learning node embeddings. These node embeddings are vectors 
that, in a well-trained network, store representative features of a particular node. Based on these node embeddings, we can decode the network by computing a similarity measure between the node embeddings of different nodes in the network. Such a scheme has previously been used successfully to infer
GRNs [<a href="#B12" class="usa-link" aria-describedby="B12">12</a>
].</p>
<p>We then encode the node features to a latent embedding using a 2-layer neural network, where each layer is given by a GraphSAGE layer [<a href="#B22" class="usa-link" aria-describedby="B22">22</a>
]. Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui31"><math id="u31" display="inline" overflow="linebreak"><msub><mi>X</mi><mi>i</mi></msub></math></span> contains the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui32"><math id="u32" display="inline" overflow="linebreak"><mi>i</mi></math></span>th row of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui33"><math id="u33" display="inline" overflow="linebreak"><mi>X</mi></math></span>, that is, all measurements for a single node <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui34"><math id="u34" display="inline" overflow="linebreak"><mi>i</mi></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui35"><math id="u35" display="inline" overflow="linebreak"><mtext>ReLU</mtext></math></span> is the rectified linear unit activation function:</p>
<table class="disp-formula p" id="uFDquWL1"><tr><td class="formula"><math id="um2" display="block" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mrow><mo>{</mo><mtable columnalign="left left" rowspacing=".2em" columnspacing="1em" displaystyle="false"><mtr><mtd><msubsup><mi>X</mi><mrow><mi>i</mi></mrow><mrow><mn>0</mn></mrow></msubsup><mo>=</mo><msub><mi>X</mi><mrow><mi>i</mi></mrow></msub><mo>,</mo></mtd></mtr><mtr><mtd><msubsup><mi>X</mi><mrow><mi>i</mi></mrow><mrow><mi>n</mi><mo>+</mo><mn>1</mn></mrow></msubsup><mo>=</mo><mtext>ReLU</mtext><mo stretchy="false">(</mo><msub><mi>W</mi><mrow><mn>1</mn></mrow></msub><msubsup><mi>X</mi><mrow><mi>i</mi></mrow><mrow><mi>n</mi></mrow></msubsup><mo>+</mo><msub><mi>W</mi><mrow><mn>2</mn></mrow></msub><mo>⋅</mo><msub><mtext>mean</mtext><mrow><mi>j</mi><mo>∈</mo><mrow><mrow><mi class="mathcal" mathvariant="script">N</mi></mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></mrow></msub><msubsup><mi>X</mi><mrow><mi>j</mi></mrow><mrow><mi>n</mi></mrow></msubsup><mo stretchy="false">)</mo><mo>.</mo></mtd></mtr></mtable><mo fence="true" stretchy="true" symmetric="true"></mo></mrow></mrow></mstyle></math></td></tr></table>
<p>Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui36"><math id="u36" display="inline" overflow="linebreak"><mrow><mi mathvariant="script">N</mi><mo form="prefix" stretchy="false">(</mo><mi>i</mi><mo form="postfix" stretchy="false">)</mo></mrow></math></span> denotes the neighbourhood of node <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui37"><math id="u37" display="inline" overflow="linebreak"><mi>i</mi></math></span>, defined as all nodes that are at most one edge away in the graph (each edge is assumed to be undirected for the purpose of propagating signals through the GraphSAGE layer). The weights <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui38"><math id="u38" display="inline" overflow="linebreak"><msub><mi>W</mi><mn>1</mn></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui39"><math id="u39" display="inline" overflow="linebreak"><msub><mi>W</mi><mn>2</mn></msub></math></span> are learnable matrices. Effectively, this neural network layer aims to aggregate information from neighbouring nodes to obtain a representative encoding of the features of nodes in the network. After a latent representation <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui40"><math id="u40" display="inline" overflow="linebreak"><mi>Z</mi></math></span> of the node features is obtained using the encoder, an adjusted adjacency matrix for the graph is obtained by decoding the representations:</p>
<table class="disp-formula p" id="uFDquWL2"><tr><td class="formula"><math id="um3" display="block" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msub><mrow><mover><mi>A</mi><mo stretchy="false">^</mo></mover></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mo stretchy="false">(</mo><mi>σ</mi><mo stretchy="false">(</mo><msup><mi>Z</mi><mrow><mi>T</mi></mrow></msup><msub><mi>W</mi><mrow><mn>3</mn></mrow></msub><mi>Z</mi><mo stretchy="false">)</mo><msub><mo stretchy="false">)</mo><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mstyle></math></td></tr></table>
<p>with</p>
<table class="disp-formula p" id="uFDquWL3"><tr><td class="formula"><math id="um4" display="block" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mi>σ</mi><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>=</mo><mfrac><mn>1</mn><mrow><mn>1</mn><mo>+</mo><msup><mrow><mrow><mtext>e</mtext></mrow></mrow><mrow><mo>−</mo><mi>x</mi></mrow></msup></mrow></mfrac><mo>.</mo></mrow></mstyle></math></td></tr></table>
<p>Here, the goal is to assign to each predicted edge a score in the interval [0, 1]
, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui41"><math id="u41" display="inline" overflow="linebreak"><mn>0</mn></math></span> indicates the certitude of no connection being present and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui42"><math id="u42" display="inline" overflow="linebreak"><mn>1</mn></math></span> indicates the certitude of a connection being present. The sigmoid function <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui43"><math id="u43" display="inline" overflow="linebreak"><mi>σ</mi></math></span> maps the output from the GAE to this interval <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui44"><math id="u44" display="inline" overflow="linebreak"><mrow><mo form="prefix" stretchy="false">[</mo><mn>0,1</mn><mo form="postfix" stretchy="false">]</mo></mrow></math></span> and is often used for this purpose in machine learning applications. Again, the matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui45"><math id="u45" display="inline" overflow="linebreak"><msub><mi>W</mi><mn>3</mn></msub></math></span> contains learnable parameters, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui46"><math id="u46" display="inline" overflow="linebreak"><msub><mover><mi>A</mi><mo stretchy="false" class="tml-capshift" mathvariant="normal">^</mo></mover><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> denotes the estimated entry in the output adjacency matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui47"><math id="u47" display="inline" overflow="linebreak"><mover><mi>A</mi><mo stretchy="false" class="tml-capshift" mathvariant="normal">^</mo></mover></math></span>. This decoding scheme is also used in DeepRIG [<a href="#B12" class="usa-link" aria-describedby="B12">12</a>
].</p>
<p>For training the network, we use Binary Cross Entropy (BCE) loss to 
optimize the weights <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui48"><math id="u48" display="inline" overflow="linebreak"><mrow><msub><mi>W</mi><mn>1</mn></msub><mo separator="true">,</mo></mrow><mrow><msub><mi>W</mi><mn>2</mn></msub><mo separator="true">,</mo></mrow><mrow><msub><mi>W</mi><mn>3</mn></msub></mrow></math></span> of the network, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui49"><math id="u49" display="inline" overflow="linebreak"><msub><mi>E</mi><mi>T</mi></msub></math></span> denotes the set of true edges and non-existent edges the network is trained over, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui50"><math id="u50" display="inline" overflow="linebreak"><mrow><msub><mi>A</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>∈</mo></mrow><mrow><mo form="prefix" stretchy="false">{</mo><mn>0,1</mn><mo form="postfix" stretchy="false">}</mo></mrow></math></span> denotes the label of edge <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui51"><math id="u51" display="inline" overflow="linebreak"><mrow><mo form="prefix" stretchy="false">(</mo><mi>i</mi><mo separator="true">,</mo><mi>j</mi><mo form="postfix" stretchy="false">)</mo><mo>∈</mo></mrow><mrow><msub><mi>E</mi><mi>T</mi></msub></mrow></math></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui52"><math id="u52" display="inline" overflow="linebreak"><mrow><msub><mi>A</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo></mrow><mrow><mn>1</mn></mrow></math></span> if <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui53"><math id="u53" display="inline" overflow="linebreak"><mrow><mo form="prefix" stretchy="false">(</mo><mi>i</mi><mo separator="true">,</mo><mi>j</mi><mo form="postfix" stretchy="false">)</mo><mo>∈</mo></mrow><mrow><mi>E</mi></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui54"><math id="u54" display="inline" overflow="linebreak"><mrow><msub><mi>A</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo></mrow><mrow><mn>0</mn></mrow></math></span> if <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui55"><math id="u55" display="inline" overflow="linebreak"><mrow><mo form="prefix" stretchy="false">(</mo><mi>i</mi><mo separator="true">,</mo><mi>j</mi><mo form="postfix" stretchy="false">)</mo><mo>∉</mo></mrow><mrow><mi>E</mi></mrow></math></span>):</p>
<table class="disp-formula p" id="uFDquWL4"><tr><td class="formula"><math id="um5" display="block" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mstyle displaystyle="true" scriptlevel="0"><msub><mi>L</mi><mrow><mtext>GAE</mtext></mrow></msub><mo>=</mo><mfrac><mn>1</mn><mrow><mrow><mo stretchy="false">|</mo></mrow><msub><mi>E</mi><mrow><mi>T</mi></mrow></msub><mrow><mo stretchy="false">|</mo></mrow></mrow></mfrac><munder><mo>∑</mo><mrow><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo><mo>∈</mo><msub><mi>E</mi><mrow><mi>T</mi></mrow></msub></mrow></munder><msub><mi>A</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mi>log</mi><mo>⁡</mo><mrow><mo stretchy="false">(</mo><msub><mrow><mover><mi>A</mi><mo stretchy="false">^</mo></mover></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow><mo stretchy="false">)</mo><mo>+</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msub><mi>A</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo stretchy="false">)</mo><mi>log</mi><mo>⁡</mo><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msub><mrow><mover><mi>A</mi><mo stretchy="false">^</mo></mover></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo stretchy="false">)</mo></mrow><mo>.</mo></mstyle></mrow></mstyle></math></td></tr></table>
<p>In the practical use case for the GAE model, the GAE is trained on an available prior knowledge 
GRN along with available scRNA-seq data. Through an iterative approach training the encoder
–decoder scheme for each possible edge that could be added to the graph, we obtain the connections in the graph that are most likely to exist given the input data (<a href="#F2" class="usa-link">
figure 2B
</a>). We can then use the inferred corrected network as an improved graph structure for downstream applications. This leads to an improved network <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui56"><math id="u56" display="inline" overflow="linebreak"><mrow><msub><mi>G</mi><mi>I</mi></msub><mo>=</mo></mrow><mrow><mo form="prefix" stretchy="false">(</mo><mi>V</mi><mo separator="true">,</mo><mi>E</mi><mo form="postfix" stretchy="false">)</mo></mrow></math></span> 
characterized by the adjacency matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui57"><math id="u57" display="inline" overflow="linebreak"><mover><mi>A</mi><mo stretchy="false" class="tml-capshift" mathvariant="normal">^</mo></mover></math></span>. We note that the GAE is time-invariant, and does not require pseudotime trajectories to adjust the 
GRN. The GAE presented here differs from the graph autoencoder-based GRN inference method DeepRIG [<a href="#B12" class="usa-link" aria-describedby="B12">12</a>
] by using a different GNN layer in the encoding step, by relying on a general prior knowledge network rather than a GRN inferred using WGCNA, and by introducing a leave-one-out policy to infer connections.</p>
<figure class="fig xbox font-sm" id="F2" title="Graph autoencoder for predicting additional connections in GRNs."><h4 class="obj_head">Figure 2. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371343_rsfs.2025.0014.f002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97e1/12371343/e24f174e1ce4/rsfs.2025.0014.f002.jpg" loading="lazy" height="344" width="788" alt="Graph autoencoder for predicting additional connections in GRNs."></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Graph autoencoder for predicting additional connections in
GRNs. (
A)
GAE network architecture. The encoder consists of a 
two-layer graphSAGE network. The decoder consists of an inner-product decoder with learnable weights. (
B)
Procedure for improving the 
GRN. The GAE is trained for each edge for which a prediction is desired, and the results are subsequently combined to obtain an improved graph structure.</p></figcaption></figure><p>Our GAE architecture consists of 
two GraphSAGE layers, each with a hidden dimension 200. Hence, the dimension of the learned node embeddings is also 200. During training, we dropout edges with a probability <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui58"><math id="u58" display="inline" overflow="linebreak"><mrow><msub><mi>p</mi><mi>e</mi></msub><mo>=</mo></mrow><mrow><mn>0.2</mn></mrow></math></span> and apply a dropout layer to the output of the first GraphSAGE layer with a probability <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui59"><math id="u59" display="inline" overflow="linebreak"><mrow><msub><mi>p</mi><mi>d</mi></msub><mo>=</mo></mrow><mrow><mn>0.3</mn></mrow></math></span> to prevent the model from overfitting the known edges. Such dropout layers are particularly important since we often consider small networks, where the risk of overfitting is large.</p></section><section id="s2-2"><h3 class="pmc_sec_title">2.2.  
Model parameter inference</h3>
<p>Based on the 
optimized GRN structure, we construct mechanistic models that describe temporal intracellular processes. In our case, we fit the already introduced nonlinear ODE model to the experimental data:</p>
<table class="disp-formula p" id="uFD2"><tr>
<td class="formula"><math id="um6" display="block" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mi>τ</mi><mrow><mover><mi>x</mi><mo>˙</mo></mover></mrow><mo>=</mo><mo>−</mo><mi>x</mi><mo>+</mo><mi>H</mi><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><msub><mi>W</mi><mrow><mtext>in</mtext></mrow></msub><mo>,</mo><mi>u</mi><mo stretchy="false">)</mo><mo>,</mo></mrow></mstyle></math></td>
<td class="label">(2.1)</td>
</tr></table>
<p>where we choose the nonlinearity to be</p>
<table class="disp-formula p" id="uFDquWL5"><tr><td class="formula"><math id="um7" display="block" overflow="linebreak"><mtable displaystyle="true"><mtr><mtd class="tml-right" style="padding-left:0em;padding-right:0em;"><mrow><msub><mi>H</mi><mi>i</mi></msub><mo form="prefix" stretchy="false">(</mo><mi>x</mi><mo separator="true">,</mo><msub><mi>W</mi><mtext>in</mtext></msub><mo separator="true">,</mo><mi>u</mi><mo form="postfix" stretchy="false">)</mo><mo>=</mo><mfrac><mrow><msub><mi>u</mi><mi>i</mi></msub><mo>+</mo><msub><mo movablelimits="false">∑</mo><mrow><msub><mi>W</mi><mrow><mtext>in</mtext><mo separator="true">,</mo><mi>j</mi><mi>i</mi></mrow></msub><mo>&gt;</mo><mn>0</mn></mrow></msub><msub><mi>W</mi><mrow><mtext>in</mtext><mo separator="true">,</mo><mi>j</mi><mi>i</mi></mrow></msub><msubsup><mi>x</mi><mi>j</mi><mn>3</mn></msubsup></mrow><mrow><mn>1</mn><mo>+</mo><msub><mi>u</mi><mi>i</mi></msub><mo>+</mo><msub><mo movablelimits="false">∑</mo><mrow><msub><mi>W</mi><mrow><mtext>in</mtext><mo separator="true">,</mo><mi>j</mi><mi>i</mi></mrow></msub><mo>&gt;</mo><mn>0</mn></mrow></msub><msub><mi>W</mi><mrow><mtext>in</mtext><mo separator="true">,</mo><mi>j</mi><mi>i</mi></mrow></msub><msubsup><mi>x</mi><mi>j</mi><mn>3</mn></msubsup><mo>+</mo><msub><mo movablelimits="false">∑</mo><mrow><msub><mi>W</mi><mrow><mtext>in</mtext><mo separator="true">,</mo><mi>j</mi><mi>i</mi></mrow></msub><mo>&lt;</mo><mn>0</mn></mrow></msub><mo>−</mo><msub><mi>W</mi><mrow><mtext>in</mtext><mo separator="true">,</mo><mi>j</mi><mi>i</mi></mrow></msub><msubsup><mi>x</mi><mi>j</mi><mn>3</mn></msubsup></mrow></mfrac><mi>.</mi></mrow></mtd></mtr></mtable></math></td></tr></table>
<p>Such a Hill function-based model is commonly used for modelling intracellular signalling [<a href="#B23" class="usa-link" aria-describedby="B23">23</a>,<a href="#B24" class="usa-link" aria-describedby="B24">24</a>
]. Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui60"><math id="u60" display="inline" overflow="linebreak"><mrow><mi>x</mi><mo>∈</mo></mrow><mrow><msup><mi>ℝ</mi><mi>N</mi></msup></mrow></math></span>, 
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui61"><math id="u61" display="inline" overflow="linebreak"><mrow><mi>τ</mi><mo>∈</mo></mrow><mrow><mi>ℝ</mi></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui62"><math id="u62" display="inline" overflow="linebreak"><mrow><mi>u</mi><mo>∈</mo></mrow><mrow><msup><mi>ℝ</mi><mi>N</mi></msup></mrow></math></span>
and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui63"><math id="u63" display="inline" overflow="linebreak"><mrow><msub><mi>W</mi><mtext>in</mtext></msub><mo>∈</mo></mrow><mrow><msup><mi>ℝ</mi><mrow><mi>N</mi><mo>×</mo><mi>N</mi></mrow></msup></mrow></math></span>. We note that fitting parameters for this network can be seen as training a one-layer graph neural ODE, where this single neural network layer is given by the right-hand side of the ODE. We remark that the weights in this model are not related to the ones of the
GAE
introduced in
§
2
.1.</p>
<p>We assume that data 
are provided in pairs of time point and gene expression of genes at that time point. With <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui64"><math id="u64" display="inline" overflow="linebreak"><mrow><mo form="prefix" stretchy="false">(</mo><msub><mi>t</mi><mi>i</mi></msub><mo separator="true">,</mo><msub><mi>X</mi><mi>i</mi></msub><mo form="postfix" stretchy="false">)</mo><mo>=</mo></mrow><mrow><msub><mi>D</mi><mrow><mi>i</mi><mo separator="true">,</mo><mi>t</mi></mrow></msub><mo>∈</mo></mrow><mrow><mi>D</mi></mrow></math></span> we denote the measured expression of gene <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui65"><math id="u65" display="inline" overflow="linebreak"><mi>i</mi></math></span> at time point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui66"><math id="u66" display="inline" overflow="linebreak"><msub><mi>t</mi><mi>i</mi></msub></math></span>. We can use the comparison between the model simulation at time <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui67"><math id="u67" display="inline" overflow="linebreak"><msub><mi>t</mi><mi>i</mi></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui68"><math id="u68" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mi>i</mi></msub><mo form="prefix" stretchy="false">(</mo><msub><mi>t</mi><mi>i</mi></msub><mo form="postfix" stretchy="false">)</mo></mrow></math></span>, and the data <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui69"><math id="u69" display="inline" overflow="linebreak"><msub><mi>D</mi><mrow><mi>i</mi><mo separator="true">,</mo><mi>t</mi></mrow></msub></math></span> to evaluate the performance of the fitted dynamical model.</p>
<p>Specifically, we improve the fit of the model to the given experimental data by 
minimizing the mean squared error between data and model simulations:</p>
<table class="disp-formula p" id="uFDquWL6"><tr><td class="formula"><math id="um8" display="block" overflow="linebreak"><mtable displaystyle="true"><mtr><mtd class="tml-right" style="padding-left:0em;padding-right:0em;"><mrow><msub><mi>L</mi><mtext>model</mtext></msub><mo>=</mo><mrow><munder><mo movablelimits="false">∑</mo><mrow><msub><mi>D</mi><mrow><mi>i</mi><mo separator="true">,</mo><mi>t</mi></mrow></msub><mo>∈</mo><mi>D</mi></mrow></munder></mrow><mo>|</mo><msub><mi>D</mi><mrow><mi>i</mi><mo separator="true">,</mo><mi>t</mi></mrow></msub><mo>−</mo><msub><mi>x</mi><mi>i</mi></msub><mo form="prefix" stretchy="false">(</mo><mi>t</mi><mo form="postfix" stretchy="false">)</mo><msup><mo>|</mo><mn>2</mn></msup><mi>.</mi></mrow></mtd></mtr></mtable></math></td></tr></table>
<p>Moreover, we aim to have a minimal model describing the data, and therefore we add an <em>L</em>1 loss term applied to the parameters. This <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui70"><math id="u70" display="inline" overflow="linebreak"><mrow><mi>L</mi><mn>1</mn></mrow></math></span> regularization term strikes a balance between pushing parameters towards zero against their importance in recovering the correct time-series data, and is commonly used as a way to promote sparse solutions in 
optimization problems [<a href="#B25" class="usa-link" aria-describedby="B25">25</a>
]. This is represented as
</p>
<table class="disp-formula p" id="uFDquWL7"><tr><td class="formula"><math id="um9" display="block" overflow="linebreak"><mtable displaystyle="true"><mtr><mtd class="tml-right" style="padding-left:0em;padding-right:0em;"><mrow><msub><mi>L</mi><mtext>p</mtext></msub><mo>=</mo><mrow><munder><mo movablelimits="false">∑</mo><mrow><msub><mi>p</mi><mi>i</mi></msub><mo>∈</mo><mi>p</mi></mrow></munder></mrow><mo>|</mo><msub><mi>p</mi><mi>i</mi></msub><mo>|</mo><mo separator="true">,</mo></mrow></mtd></mtr></mtable></math></td></tr></table>
<p>with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui71"><math id="u71" display="inline" overflow="linebreak"><mrow><mi>p</mi><mo>=</mo></mrow><mrow><mo form="prefix" stretchy="false">(</mo><msub><mi>W</mi><mtext>in</mtext></msub><mo separator="true">,</mo><mi>u</mi><mo separator="true">,</mo><mi>τ</mi><mo form="postfix" stretchy="false">)</mo></mrow></math></span>.</p>
<p>Finally, in some cases, the trajectories in the data split up from a common starting point (see, for example, the pseudotime analysis shown in <a href="#F3" class="usa-link">
figure 3C
</a>). In these cases, we add the initial conditions for each splitting trajectory as tunable parameters, and add a term to the loss function ensuring that these initial conditions are as close as possible to the starting point in the reference data. In this way, we 
recognize that measured initial conditions can be noisy:</p>
<figure class="fig xbox font-sm" id="F3" title="A Results from hyperparameter tuning for our graph autoencoder model introduced in Section 2.1 ."><h4 class="obj_head">Figure 3. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371343_rsfs.2025.0014.f003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97e1/12371343/8db1c8cefca5/rsfs.2025.0014.f003.jpg" loading="lazy" height="618" width="658" alt="A Results from hyperparameter tuning for our graph autoencoder model introduced in Section 2.1 ."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>
(A)
Results from hyperparameter tuning for our graph autoencoder model introduced in
§
2.1
. Each parallel axis denotes a hyperparameter choice, and the colour of the line indicates the AUROC for the mCAD network with 2000 simulated cells as provided in BEELINE [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>
]. The configuration of learning rate 0.0001, hidden dimension 200 and 1000 epochs performed best. (
B)
Application of the GAE on the bifurcating network.
<em>
L
</em>
<em>
eft
</em>. The damaged bifurcating graph given as input, where the dashed line indicates the edge removed from the original true graph.
<em>
R
</em>
<em>
ight
</em>. The improved graph structure based on the output of the GAE. Edges that were added are indicated in green (threshold score for adding the edge was set to 0.9).
<em>
C
</em>
<em>
ent
</em>
<em>
re
</em>
. A representation of the GAE. The training procedure is repeated <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui72"><math id="u72" display="inline" overflow="linebreak"><mrow><msup><mi>m</mi><mn>2</mn></msup><mo>−</mo></mrow><mrow><mo>|</mo><mi>E</mi><mo>|</mo></mrow></math></span> times to build the adjusted output graph, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui73"><math id="u73" display="inline" overflow="linebreak"><mi>m</mi></math></span> is the number of nodes and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui74"><math id="u74" display="inline" overflow="linebreak"><mrow><mo>|</mo><mi>E</mi><mo>|</mo></mrow></math></span> is the number of edges. (
C)
Results for six representative networks based on our graph autoencoder network. For each network, we show a dimension reduction using t-SNE for the data simulated using BoolODE, where each dot indicates one sample in the dataset. Here, blue indicates <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui75"><math id="u75" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo></mrow><mrow><mn>0</mn></mrow></math></span> and yellow indicates <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui76"><math id="u76" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo></mrow><mrow><mn>1</mn></mrow></math></span> (the end point of the simulated data). Next to each data 
visualization, we show statistics for networks improved using our GAE network. The grey edges indicate edges that are in the true network, and the thickness of the edge indicates how well this edge is recovered using the GAE (rank among all inferred connections/amount of possible connections in the graph). The red lines indicate edges that are often inferred using the GAE network (mean score <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui77"><math id="u77" display="inline" overflow="linebreak"><mo lspace="0em" rspace="0em">&gt;</mo></math></span>0.5) that are <em>not</em>
<strong>
</strong>
in the true underlying network.</p></figcaption></figure><table class="disp-formula p" id="uFDquWL8"><tr><td class="formula"><math id="um10" display="block" overflow="linebreak"><mtable displaystyle="true"><mtr><mtd class="tml-right" style="padding-left:0em;padding-right:0em;"><mrow><msub><mi>L</mi><mtext>IC</mtext></msub><mo>=</mo><munder><mi>max</mi><mi>i</mi></munder><mo>⁡</mo><mo>|</mo><msub><mi>D</mi><mrow><mi>i</mi><mo separator="true">,</mo><mn>0</mn></mrow></msub><mo>−</mo><msub><mi>x</mi><mi>i</mi></msub><mo form="prefix" stretchy="false">(</mo><mn>0</mn><mo form="postfix" stretchy="false">)</mo><mo>|</mo><mi>.</mi></mrow></mtd></mtr></mtable></math></td></tr></table>
<p>A similar 
regularization on the initial conditions has been used in the context of physics-informed neural networks [<a href="#B26" class="usa-link" aria-describedby="B26">26</a>
]. Combining terms, the complete loss function becomes
</p>
<table class="disp-formula p" id="uFDquWL9"><tr><td class="formula"><math id="um11" display="block" overflow="linebreak"><mtable displaystyle="true"><mtr><mtd class="tml-right" style="padding-left:0em;padding-right:0em;"><mrow><mi>L</mi><mo>=</mo><msub><mi>L</mi><mtext>model</mtext></msub><mo>+</mo><msub><mi>λ</mi><mtext>p</mtext></msub><msub><mi>L</mi><mtext>p</mtext></msub><mo>+</mo><msub><mi>λ</mi><mtext>IC</mtext></msub><msub><mi>L</mi><mtext>IC</mtext></msub><mi>.</mi></mrow></mtd></mtr></mtable></math></td></tr></table>
<p>If multiple trajectories are available,
the losses for each individual trajectory are summed. Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui78"><math id="u78" display="inline" overflow="linebreak"><msub><mi>λ</mi><mtext>IC</mtext></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui79"><math id="u79" display="inline" overflow="linebreak"><msub><mi>λ</mi><mtext>p</mtext></msub></math></span> are hyperparameters used to scale the importance of the different terms in the loss function. In our experiments, we set <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui80"><math id="u80" display="inline" overflow="linebreak"><mrow><msub><mi>λ</mi><mtext>p</mtext></msub><mo>=</mo></mrow><mrow><mn>0.001</mn></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui81"><math id="u81" display="inline" overflow="linebreak"><mrow><msub><mi>λ</mi><mtext>IC</mtext></msub><mo>=</mo></mrow><mrow><mn>10</mn></mrow></math></span>. The purpose of these hyperparameter settings is to introduce a hierarchy of importance for different terms in the loss function. The fit to the time-series data along with initial conditions close to the starting point in the data are 
prioritized. Only after a reasonable fit for these terms is achieved, excess connections and parameters are slowly adjusted to zero if they do not contribute meaningfully to the fit.</p>
<p>There are many methods for optimizing parameters for a given system of ODEs. Although many approaches achieve similar results, we use a global optimizer of differential evolution type, adaptive DE/rand/1/bin [<a href="#B27" class="usa-link" aria-describedby="B27">27</a>
]. Let us remark here that determining the correct graph in the previous step is fundamental to decrease the size of the parameter space. If a complete graph was considered, we would need to fit a full matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui82"><math id="u82" display="inline" overflow="linebreak"><msub><mi>W</mi><mtext>in</mtext></msub></math></span>, consisting of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui83"><math id="u83" display="inline" overflow="linebreak"><msup><mi>n</mi><mn>2</mn></msup></math></span> parameters. Here, instead, we fit only the necessary non-zero parameters, so that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui84"><math id="u84" display="inline" overflow="linebreak"><msub><mi>W</mi><mtext>in</mtext></msub></math></span> has only <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui85"><math id="u85" display="inline" overflow="linebreak"><mrow><mo>|</mo><mi>E</mi><mo>|</mo></mrow></math></span> non-zero elements, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui86"><math id="u86" display="inline" overflow="linebreak"><mi>E</mi></math></span> denotes the set of edges associated with the improved graph <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui87"><math id="u87" display="inline" overflow="linebreak"><msub><mi>G</mi><mi>I</mi></msub></math></span>. We 
recognize that our model formulation is less flexible than other possible formulations. However, we 
emphasize that the smaller parameter space in our formulation pushes the model to use the available network structure, rather than relying on per-node flexibility to model time-
series data.</p></section><section id="s2-3"><h3 class="pmc_sec_title">2.3.  
Training data</h3>
<p>To test our methods, we apply the GAE and model fitting to synthetic datasets generated using BoolODE [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>
]. Based on this method, six representative small
GRNs are used as input to our method. We use the bifurcating, bifurcating-converging, trifurcating, cycle, linear and long linear graphs provided in this dataset
; see <a href="#F3" class="usa-link">
figure 3
</a>. These networks are well documented and have been used to test and benchmark GRN inference methods [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>,<a href="#B12" class="usa-link" aria-describedby="B12">12</a>
].</p>
<p>Next to the graph structures provided for these six graphs, synthetic scRNA-seq data 
are generated. For each of the graphs, we have simulated 3000
cells using BoolODE [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>
] with maximum time 8. One time point is randomly selected for each of these 3000 simulations, and the gene expression at this time point is added to the synthetic scRNA-seq dataset. Hence, our synthetic dataset
consists
of the gene expression of 3000
cells along with an associated time point for each cell. While a noisy dynamical system is used to simulate the gene expression over time, we note that in our current setup, we do not randomly drop out gene expression for specific genes. To fit ODE models to th
ese data, we collect all cells within bins of
pseudotime with 
width 0.1
to get time-series data to fit the ODE model. Afterwards, we linearly scale the time-series data to the interval [0.1, 0.9] to obtain data that can be represented using <a href="#uFD2" class="usa-link">equation (2.1)</a>. In case multiple clusters or trajectories are available, we construct time series in this way individually for each cluster or trajectory. In these cases, we scale the time series based on all available time series, in order to preserve differences in expression between the available data.</p>
<p>In addition to synthetic data, we evaluate our GAE on one large real-world GRN describing human embryonic stem cell (hESC) differentiation using scRNA-seq data as described by Chu 
<em>et al.
</em>
[<a href="#B28" class="usa-link" aria-describedby="B28">28</a>
]. We use the pre-processed 
GRN provided by Pratapa 
<em>et al.
</em>
[<a href="#B3" class="usa-link" aria-describedby="B3">3</a>
]
, which consists of putative connections between 130 transcription factors (TFs) and 18 104 target genes
based on ChIP-seq data
. While the full network can be used, the degree distribution of this network is heavily skewed with the 130 TFs being the only nodes with outgoing connections. For this reason, we have chosen to consider only a subnetwork of this large network, which entails the 130
TFs
and connections between these TFs. This real-world GRN is still much bigger than the small synthetic networks we consider. Single-cell RNA-seq data 
are available for 758
cells, and the considered TF network has 130 nodes and 3866 edges. In
§
3.1
, we use this real-world network to assess the scalability of our graph autoencoder approach to improving
GRNs.</p>
<p>For each of the six representative networks, we first consider the original graph (
<em>true</em>
<em>graph</em>
), and then remove one connection from the true graph topology to obtain the 
<em>damaged graph</em>
. We then use the damaged graph as input for our GAE, which leads to an 
<em>improved graph</em>
. We compare the features of the original BoolODE model 
with two dynamic models obtained based on the damaged and improved graphs. First, we fit the parameters of the same model based on the damaged graph, and then we fit the parameters of the model based on the improved graph. The purpose of this comparison is to evaluate how the additional GAE step influences key properties of the resulting dynamic model.</p></section></section><section id="s3"><h2 class="pmc_sec_title">3.  
Results</h2>
<p>We now focus on the results of the two key parts of our proposed framework. First, we show the results of the
GAE
when used to recover edges in a 
GRN. Next, we discuss the results of fitting an ODE model based on the graphs.</p>
<section id="s3-1"><h3 class="pmc_sec_title">3.1.  
Graph autoencoder
network recovers unseen connections in
gene regulatory networks
</h3>
<p>In this section, we focus on the performance of our GAE described in
§
2.1 
with the purpose of improving the graph structure of
GRNs.</p>
<p>Before testing our GAE on various
GRNs, we have tuned the hyperparameters for the
GAE
model. We have used the mCAD network with synthetic scRNA-seq data provided through BEELINE [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>
] as a validation network to test the effect of key hyperparameter choices. The results of this parameter tuning are shown in <a href="#F3" class="usa-link">
figure 3A
</a>. Here, we see that a learning rate of 0.0001, a hidden dimension of the encoder of 200 or 500 and 1000 or 2000 training epochs perform well for the mCAD network. Since a learning rate of 0.0001, a hidden dimension of 200 and 1000 training epochs performed best for the validation network, we use these parameter choices for the remainder of this section.</p>
<p>First, we show the application of our GAE network in a typical use case (<a href="#F3" class="usa-link">
figure 3B
</a>). For this example, we take the bifurcating graph as provided in the BoolODE dataset and remove one edge from this graph [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>
]. We use the damaged graph along with available scRNA-seq data as input for our GAE, which we train once for each connection that could be added to the graph, predicting whether those connections exist. Each of these training runs leads to a score associated with the predicted edge between 0 and 1. Combining the results from the different training runs results in a new graph structure, where new edges are added to the graph. In this example case, we recover the removed edge using a score threshold of 0.9, but we also predict some additional edges to be in the graph that were not in the true graph.</p>
<p>To gain insight into the performance of our GAE on multiple graphs, we apply our GAE for each edge in each of the six representative graphs described in
§
2.3 
(<a href="#F3" class="usa-link">
figure 3
</a>). We have subsequently computed ROC curves for each of these networks to evaluate the performance of our GAE as a binary classifier that predicts additional edges in the 
GRN (<a href="#F4" class="usa-link">
figure 4A
</a>). Here, we compare the performance of our proposed graph autoencoder 
with three alternative network inference methods, GENIE3, GRNBOOST2 and PIDC [<a href="#B6" class="usa-link" aria-describedby="B6">6</a>,<a href="#B29" class="usa-link" aria-describedby="B29">29</a>,<a href="#B30" class="usa-link" aria-describedby="B30">30</a>
], as implemented in BEELINE [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>
]. We see that our GAE outperforms these three GRN inference methods on the bifurcating network and the trifurcating network. Performance in the bifurcating-converging network is similar to that in the compared methods. However, we also see that the performance of our GAE is similar to
that of a
random predictor for the cycle, linear and long linear synthetic networks.</p>
<figure class="fig xbox font-sm" id="F4" title="Statistics for the performance of the GAE."><h4 class="obj_head">Figure 4. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371343_rsfs.2025.0014.f004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97e1/12371343/0e15d7f9dc78/rsfs.2025.0014.f004.jpg" loading="lazy" height="784" width="650" alt="Statistics for the performance of the GAE."></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Statistics for the performance of the GAE. (
A)
ROC curves for each of the tested synthetic GRNs (<a href="#F3" class="usa-link">
figure 3C
</a>),
comparing the performance of our graph autoencoders against the methods GENIE3, PIDC and GRNBOOST2 [<a href="#B6" class="usa-link" aria-describedby="B6">6</a>,<a href="#B29" class="usa-link" aria-describedby="B29">29</a>,<a href="#B30" class="usa-link" aria-describedby="B30">30</a>
] as implemented in BEELINE [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>
]. A higher area under the ROC curve (AUROC) corresponds to a better binary classifier. (
B)
The total amount of edges added to the graph when using the GAE to recover a removed edge (threshold for inclusion is set to a score of 0.9) divided by the number of potential edges that can be added. This is an indication of how many spurious connections are expected to be added to the graph if the GAE is used to improve a 
GRN. (
C)
Rank of the edge removed from the network compared 
with all possible edges in the graph. (
D)
Effects of changing the decision threshold for adding connections on the AUROC.</p></figcaption></figure><p>We also present some summary statistics for the performance of the GAE on the six representative graphs in <a href="#F4" class="usa-link">
figure 4
</a>. We see that the edges removed are typically recovered earlier than other spurious edges in the graph (<a href="#F4" class="usa-link">
figure 4C
</a>). Moreover, we look into how many edges we add to the graph if we include all edges given a score higher than 0.9. We find that
a
pproximately 15
% of all possible edges are added (<a href="#F4" class="usa-link">
figure 4B
</a>). Hence, our autoencoder significantly increases the amount of connections for the tested
GRNs, even if only one true edge is removed. We also consider how the score threshold used to determine if an edge is added affects the performance of our method (<a href="#F4" class="usa-link">
figure 4D
</a>). We see that, in general, the AUROC does not change much after tuning this decision threshold, indicating that most edges get a score that is very close to 0 or very close to 1.</p>
<p>To gain more insight into where our GAE performs well and where it underperforms, we show two key metrics for each of the synthetic graphs (<a href="#F3" class="usa-link">
figure 3C
</a>). First, we show how well each of the removed edges is recovered if it is removed from the graph. This is indicated by the thickness of the grey edges in the graphs shown (all grey edges are edges in the true graph). Specifically, we calculate the rank of the removed edge among all possible edges in the graph <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui88"><math id="u88" display="inline" overflow="linebreak"><mrow><mi>R</mi><mo form="prefix" stretchy="false">(</mo><mi>i</mi><mo separator="true">,</mo><mi>j</mi><mo form="postfix" stretchy="false">)</mo></mrow></math></span>
<em>,</em> based on the output score for each edge from the GAE. For a graph with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui89"><math id="u89" display="inline" overflow="linebreak"><mi>m</mi></math></span> nodes, the edge weight based on this rank is linearly related to</p>
<table class="disp-formula p" id="uFDquWL10"><tr><td class="formula"><math id="um12" display="block" overflow="linebreak"><mrow><mfrac><mrow><msup><mi>m</mi><mn>2</mn></msup><mo>−</mo><mo>|</mo><msub><mi>E</mi><mi>I</mi></msub><mo>|</mo><mo>−</mo><mi>R</mi><mo form="prefix" stretchy="false">(</mo><mi>i</mi><mo separator="true">,</mo><mi>j</mi><mo form="postfix" stretchy="false" lspace="0em" rspace="0em">)</mo></mrow><mrow><msup><mi>m</mi><mn>2</mn></msup><mo>−</mo><mo>|</mo><msub><mi>E</mi><mi>I</mi></msub><mo>|</mo></mrow></mfrac><mi>.</mi></mrow></math></td></tr></table>
<p>Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui90"><math id="u90" display="inline" overflow="linebreak"><msub><mi>E</mi><mi>I</mi></msub></math></span> denotes the set of edges of the damaged graph used as input for the GAE. If the removed edge is easily recovered, this defines a larger edge weight.</p>
<p>Second, we indicate in red edges that are often recovered based on the output of our GAE that are not in the true graph. To identify which erroneous edges are often recovered, we take the mean of the output score of the GAE trained for each of the edges in our network, and say that an edge is often recovered if this mean score is larger than 0.5.</p>
<p>We see that the graph structure has a large influence on how well the GAE recovers removed edges. For example, many edges in the bifurcating graph are recovered correctly, whereas the linear network is difficult to encode and decode using the GAE. This difficulty may be caused by the fact that our 
two-layer GraphSAGE-based encoder can only use information from up to two edges away in the damaged graph to obtain a node embedding for all nodes in the system. Moreover, we typically get better results when recovering self-loops or edges that are part of a direct feedback loop (i.e. even if the edge is removed, there is an edge going the other direction). This makes sense based on our GAE architecture, since information is still shared between the two nodes involved in the edge in these cases.</p>
<p>There are also some patterns visible when considering the edges that are often erroneously added to the graphs. First, there is a bias towards self-loops. This is again due to the way that the GraphSAGE layer is set up, in which the nodes always pass information to themselves. Next, there are some cases where 
‘indirect
’ connections (edges between nodes that are at a distance of 2 in the true graph) are erroneously found. Likely, the scRNA-seq data for these connected nodes are similar, which, along with similar information being passed to these nodes through the GAE (since the source nodes are connected), leads to similar node embeddings for both source nodes. In turn, this makes it difficult to discern which node truly has a connection to a downstream node. Similarly, we see a clear bias in the linear networks towards adding connections that skip one node in the chain. Potential improvements could be made to the performance of the GAE by adapting the architecture of the GAE so that global features can be taken into account, e.g. 
through positional encoding.</p>
<p>To see how our approach 
generalizes to larger real-world networks where data 
are more noisy, we have tested our
GAE
using the scRNA-seq data available for
hESCs
along with a cell-type specific network based on ChIP-seq data as provided in BEELINE [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>,<a href="#B28" class="usa-link" aria-describedby="B28">28</a>
]. We consider a transcription factor network of 130 nodes and 3866 edges. Since we now have many edges available, we can use the standard training procedure for graph autoencoders, where we split off test and validation edges from the network before training and use these edges to assess the performance of the network. When using the full transcription factor network, we obtain an average AUROC on the set of test edges of approximately 0.84 over 20 runs. This shows that our GAE is capable of predicting unseen connections based on noisy experimental data if many good examples of edges are available.</p>
<p>We subsequently ask how many nodes and edges are needed in the TF network to obtain a good prediction of unseen edges. To investigate this, we have created subnetworks of the top <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui91"><math id="u91" display="inline" overflow="linebreak"><mi>N</mi></math></span> nodes with the highest degree based on the TF network, and we have trained our GAE using these subnetworks. We show that our GAE is better on average than a random predictor even for a small networks of 10 nodes and 72 edges. Importantly, performance and stability increase as more examples of edges are available (<a href="#F5" class="usa-link">
figure 5
</a>). Whereas overfitting must be explicitly avoided in the small synthetic networks, this shows the possibility of avoiding the leave-one-out strategy used for the small synthetic graphs if more data 
are available.</p>
<figure class="fig xbox font-sm" id="F5" title="Performance of our graph autoencoder on a real-world GRN and experimental data describing hESC differentiation as obtained from BEELINE [2,27]."><h4 class="obj_head">Figure 5. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97e1/12371343/1e66f36117d1/rsfs.2025.0014.f005.jpg" loading="lazy" height="483" width="788" alt="Performance of our graph autoencoder on a real-world GRN and experimental data describing hESC differentiation as obtained from BEELINE [2,27]."></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance of our
GAE
on a real-world GRN and experimental data describing hESC differentiation as obtained from BEELINE [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>,<a href="#B28" class="usa-link" aria-describedby="B28">28</a>
]. We show the AUROC over 10 training runs for subnetworks of different sizes built based on the ChIP-seq network, constrained to only transcription factors. Each subnetwork is constructed by taking the top <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui92"><math id="u92" display="inline" overflow="linebreak"><mi>N</mi></math></span> nodes based on degree in the ChIP-seq network and all connections between these top <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui93"><math id="u93" display="inline" overflow="linebreak"><mi>N</mi></math></span> nodes.</p></figcaption></figure></section><section id="s3-2"><h3 class="pmc_sec_title">3.2.  
Dynamical models based on improved networks</h3>
<p>Based on the improved network topology, we fit an ODE model to trajectories in the synthetic scRNA-seq data generated using BoolODE for the true graph structure (see
§
2.3 
for details). In this section, we focus only on the bifurcating network. Specifically, we fit an ODE model to the two trajectories in the bifurcating network (as seen in <a href="#F3" class="usa-link">
figure 3
</a>). We follow the example shown in
§
3.1,
where the edge from node 3 to 5 is removed from the graph. Hence, we consider the two distinct cases for 
optimized ODE models introduced in
§
2.3
that are based on the following graphs:</p>
<ul id="list1" class="list" style="list-style-type:none">
<li>
<span class="label">—</span><p class="display-inline">The damaged bifurcating graph with the edge from node 3 to 5 removed.</p>
</li>
<li>
<span class="label">—</span><p class="display-inline">The improved bifurcating graph using our GAE after the edge from node 3 to 5 was removed.</p>
</li>
</ul>
<p>Considering these two networks and their associated ODE model, we find parameters for each model to fit the synthetic data for the true bifurcating network simulated using BoolODE (<a href="#F6" class="usa-link">
figure 6A
</a>). After testing the descriptive performance of the fitted models by comparing model simulations to the synthetic data, we test the performance of the fitted models in predictive tasks. To do this, we perform an 
<em>in silico</em>
knockdown experiment. In particular, to knock down a chosen gene <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui94"><math id="u94" display="inline" overflow="linebreak"><msub><mi>x</mi><mi>i</mi></msub></math></span> in the network, we modify <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui95"><math id="u95" display="inline" overflow="linebreak"><msub><mover><mi>x</mi><mo stretchy="false" class="tml-xshift" mathvariant="normal">˙</mo></mover><mi>i</mi></msub></math></span> to be
</p>
<figure class="fig xbox font-sm" id="F6" title="Results from simulated knockdown experiments of gene 3 in BoolODE for the true bifurcating graph."><h4 class="obj_head">Figure 6. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371343_rsfs.2025.0014.f006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97e1/12371343/9f4676bec8c3/rsfs.2025.0014.f006.jpg" loading="lazy" height="824" width="661" alt="Results from simulated knockdown experiments of gene 3 in BoolODE for the true bifurcating graph."></a></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>
(A)
Results from simulated knockdown experiments of gene 3 in BoolODE for the true bifurcating graph. 
We show that knockdown of gene 3 should result in lower expression of gene 5 (red) compared 
with a model without knockdown (blue). (
B)
Results from fitting the ODE system to the damaged bifurcating graph (removing the edge from node 3 to node 5) and the improved bifurcating graph based on the graph autoencoder. For each graph, we show three results. First, we show all commonly found parameter settings after the fit (
<em>
left
</em>
). Then, we show a phase portrait for genes 4 and 5 with simulations of the fitted model (<em>
middle
</em>). These simulations should follow the splitting trajectories found in the data (black, dashed line in this figure, synthetic input data 
are shown in <a href="#F3" class="usa-link">
figure 3C
</a>). Finally, we show the result of knocking down gene 3 on a simulation of the expression of gene 5 (<em>
right
</em>). This simulated knockdown experiment should correspond to the simulated knockdown experiment in
(A)
.</p></figcaption></figure><table class="disp-formula p" id="uFDquWL11"><tr><td class="formula"><math id="um13" display="block" overflow="linebreak"><mtable displaystyle="true"><mtr><mtd class="tml-right" style="padding-left:0em;padding-right:0em;"><mrow><mi>τ</mi><msub><mover><mi>x</mi><mo stretchy="false" class="tml-xshift" mathvariant="normal">˙</mo></mover><mi>i</mi></msub><mo>=</mo><mo form="prefix" stretchy="false">−</mo><mo form="prefix" stretchy="false">(</mo><mi>k</mi><mo>+</mo><mn>1</mn><mo form="postfix" stretchy="false">)</mo><msub><mi>x</mi><mi>i</mi></msub><mo>+</mo><mi>H</mi><msub><mrow><mo form="prefix" stretchy="false" lspace="0em" rspace="0em">(</mo><mi>x</mi><mo separator="true">,</mo><msub><mi>W</mi><mtext>in</mtext></msub><mo separator="true">,</mo><mi>u</mi><mo form="postfix" stretchy="false" lspace="0em" rspace="0em">)</mo></mrow><mi>i</mi></msub><mo separator="true">,</mo></mrow></mtd></mtr></mtable></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui96"><math id="u96" display="inline" overflow="linebreak"><mi>k</mi></math></span> denotes the strength of the knockdown, which we set to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui97"><math id="u97" display="inline" overflow="linebreak"><mrow><mi>k</mi><mo>=</mo></mrow><mrow><mn>2</mn></mrow></math></span>. In this way, we evaluate whether the parameters fitted for the alternative graph structures lead to meaningfully different results if the model is used to predict the outcome of an experiment. This knockdown can also be represented in the BoolODE model used to generate synthetic data, leading to synthetic knockdown data to which the simulations of the fitted model can be compared (<a href="#F6" class="usa-link">
figure 6A
</a>). In BoolODE, we have modified the degradation rate to be 2 times, 5 times and 8 times the original degradation rate to simulate knockdown experiments, and compare our model simulations 
with the
five times knockdown experiment.</p>
<p>After running our parameter fit 25 individual times for both graphs, we find that the desired bistable behaviour is always fitted correctly for the damaged GRN (<a href="#F6" class="usa-link">
figure 6B
</a>). However, once knockdown is simulated for gene 3 in this network, we observe an effect opposite to the desired effect as gene 5 expression increases. Fitting parameters for the GAE-improved network, we instead find that there are three alternative parameter choices resulting from our parameter 
optimization (<a href="#F6" class="usa-link">
figure 6B
</a>). In some fits, we find a network configuration that is very similar to the parameters found for the damaged GRN, in which case the 
<em>in silico</em>
knockdown experiment gives a similar undesired result. However, in 7 of the 25 fits run, the added edge from gene 3 to gene 5 is given a larger connection strength, and we find knockdown simulations that are in line with the simulated true model. Finally, there are some cases where the system gets stuck in a local minimum where no bistability is present, in which case
the characteristic negative feedback loop between nodes 4 and 5 is not recovered through the parameter fit. The final class of parameter fits is not shown since it only appeared once in the 25 fits run. This final parameter configuration does have the negative feedback loop between genes 4 and 5, but both nodes do not receive input from node 3. Instead, the model relies on the self-activation <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui98"><math id="u98" display="inline" overflow="linebreak"><mi>u</mi></math></span> to reconstruct the time-series data.</p>
<p>A key observation is that the models fitted for both configurations are capable of fitting the single-cell data for the two trajectories well, as exemplified by the simulations for gene 5,
where a bump in expression is recovered for all the graphs considered (<a href="#F6" class="usa-link">
figure 6
</a>). Hence, even if key edges are removed from the graph, the model definition has enough flexibility to capture key dynamical features of the data. An exception to this is when the only incoming edge of a node is removed. In this case, the dynamics of the node must follow a linear system, typically leading to suboptimal solutions of the fitted ODE model. This result strongly suggests that fitting data alone is not an indication that a fitted model is correct and more rigorous analysis is necessary to evaluate the performance of a data-driven model.</p>
<p>To look at the graph structure resulting from the fit, we 
visualize <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui99"><math id="u99" display="inline" overflow="linebreak"><msub><mi>W</mi><mtext>in</mtext></msub></math></span> to get a view of which connections are stronger and which connections are weaker in each of the fitted ODE models (<a href="#F6" class="usa-link">
figure 6B
</a>). By looking at these graphs, it is apparent that a choice is made for parameter fits for the model based on the GAE-improved graph: gene 3 can influence the negative feedback loop between genes 4 and
5 either
through activating node 4 or through activating node 5. How these two effects are balanced subsequently determines the outcome of knockdown experiments. As this choice is not available for the damaged network, the models fitted for the damaged networks are consistent, but always incorrect.</p>
<p>Our results suggest that it is necessary to look at an ensemble of fits, rather than an individual fit, in order to obtain well-founded hypotheses on the underlying dynamical system. Given a reasonable formulation of the ODE model, there is too much freedom in selecting viable parameters when fitting to a single time-series data, so that many configurations of the graph lead to reasonable results of the dynamic model. Within this context, the
GAE resented in this work is capable of suggesting new connections to add to the GRN. This helps to bridge the gap between the underdefined damaged network, where there is no possibility of recovering the correct mechanism, and the updated network, where finding the correct mechanism is not certain but is possible.</p></section></section><section id="s4"><h2 class="pmc_sec_title">4.  
Discussion</h2>
<p>In this work, we have explored how recent developments in machine learning can be used to aid in constructing interpretable mechanistic models for intracellular processes. In particular, we have constructed a
GAE
to suggest new connections in
GRNs. Subsequently, we have fitted nonlinear ODE models to three graphs: the original true graph structure, a damaged graph structure where a key edge is removed, and an improved graph structure, the output of our GAE using this damaged graph as input. We have tested the results according to their ability to fit the dynamics of not only the true system, but also the one expressed by 
<em>in silico</em>
knockdown experiments. Our work shows an example of how methods from deep learning have a place within mechanistic modelling, providing the interpretability of models that is often desired.</p>
<p>Starting from a given graph structure of the GRN and scRNA-seq data, our GAE can suggest additional edges to add to the small input graph. We have shown that this method works best on densely connected graphs, and performs worse on weakly connected graphs, such as simple rings. We present a training procedure that provides predicted edges even if only a very small GRN is available, and we show the scalability of our approach to real-world cases if a larger baseline GRN is available. In the architecture presented, we also show that there are areas of improvement for the constructed GAE, particularly a bias towards self-loops and edges to nodes that are two edges away in the true graph. Both of these fallbacks can be attributed to the structure of the 
two-layer GraphSAGE neural network used for encoding node features. They could be overcome by explicitly learning on global features of the graph rather than local features.</p>
<p>The fitting of an ODE system to the synthetic scRNA-seq data has demonstrated that even with a correct graph structure, there are a variety of parameters possible that lead to a simulation matching the data. This result, while not ideal, is compatible with the results from DSGRN [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>
] indicating that many parameter regions can support identical dynamical signatures. This shows that having the right graph structure is necessary but not sufficient for finding a correct model. This conclusion challenges the standards used in modelling applications, where a good fit to the experimental data is often considered as a proof of having found the correct graph structure. On the flip side, we also show that correctly identifying an edge to be activating or inhibiting based on the fitted parameters does not seem to depend excessively on the graph structure used, thus allowing us to deduce with reasonable certainty the relationship between two nodes, if such edge exists in the true graph.</p>
<p>The contributions presented in this work have a place in a broader framework for data-driven modelling. Specifically, our methods find viable configurations of parameters for models of temporal intracellular processes. Based on these parameters, an interesting next step is to identify regions of parameters close to the identified parameters with qualitatively different behaviour. Such an analysis could be conducted through a numerical bifurcation analysis [<a href="#B31" class="usa-link" aria-describedby="B31">31</a>
], a rigorous identification of these parameter regions [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>
] or a combination of both.</p>
<p>We note that our results on parameter fits of an ODE model are dependent on the model equations used. When using another formulation, the model could rely on model parameters specific to that formulation to recover the desired time-series data, which would lead to a different set of alternative hypotheses on the underlying mechanism. Moreover, the loss function used to 
optimize the model may need to be adjusted if a different model is used. For example, it is often desired that simulations of the model end in an equilibrium state corresponding to the final data point. For some model formulations, this behaviour may not be easy to find, in which case a term could be added to the loss function promoting this behaviour.</p>
<p>Future research could be directed towards different methods for retrieving a dynamic model from data while exploiting the
GAE
model presented in this 
article. In particular, SINDy has become a popular method for retrieving nonlinear models based on a library of functions that can be used in the definition of the right-hand side of the ODE [<a href="#B32" class="usa-link" aria-describedby="B32">32</a>–<a href="#B34" class="usa-link" aria-describedby="B34">34</a>
]. Thus, a SINDy integration in our code would be our first step in this direction.</p>
<p>We could also consider a more drastic overhaul of our architecture. Indeed, in recent years, neural ODEs have become more prominent in the machine learning community [<a href="#B35" class="usa-link" aria-describedby="B35">35</a>
]. When graph topologies sustain the dynamics, graph neural ODEs (
GDEs
) have also been proposed [<a href="#B36" class="usa-link" aria-describedby="B36">36</a>
]. We foresee two key ways in which neural ODEs can be used to extend the methods presented in this work. First, a GDE could be used in place of the GraphSAGE layers currently in our encoder model. This would allow us to obtain a better view of the global properties of the graph compared 
with our current architecture, where only information up to a distance of two edges away can be seen. Second, GDEs could be used to directly infer a surrogate model of the dynamical system. If such an approach is successful and provides an accurate (though not easily interpretable) representation of the underlying dynamics, identification of an interpretable model could be done based on fitting the vector field rather than fitting the time series directly. This is a faster method and could lead to improved mechanistic models.</p>
<p>As a direction for future research, it is interesting to consider if adopting a distributional view of the data, using techniques from optimal transport, leads to better results, similar to techniques such as TrajectoryNet and MIOFlow [<a href="#B37" class="usa-link" aria-describedby="B37">37</a>,<a href="#B38" class="usa-link" aria-describedby="B38">38</a>
]. Related to these approaches, recent work also shows success in using optimal transport-based approaches to construct stochastic dynamics describing splitting trajectories in single-cell data [<a href="#B39" class="usa-link" aria-describedby="B39">39</a>,<a href="#B40" class="usa-link" aria-describedby="B40">40</a>
]. A key question for future work could be whether these approaches can be adjusted to explicitly use an available GRN structure. For example, a graph autoencoder similar to the one used in this work could be used to encode the high-dimensional scRNA-seq data, or a graph neural ODE could be 
used in place of the multi-layer perceptrons often used in the neural ODE central to these approaches. Such adaptations could improve the interpretability of the models resulting from these methods, and would open the way to the discovery of novel biological hypotheses.</p>
<p>In conclusion, this work presents an application of graph neural networks for constructing mechanistic models for time-dependent gene expression. Our result highlights not only the flexibility of GAEs, but also their limitations, appearing in the form of spurious edges. Furthermore, we demonstrate how a good fit of the data does not relate to a good representation of the underlying model, thus highlighting the importance of additional testing on fitted models, such as the presented knockdown experiments. The novelty of integrating machine learning techniques with additional insights from network dynamics and biology makes strides in showing how GAEs and other methods can be successfully used in practical scenarios to create interpretable models on a case-by-case basis.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We have used BioRender to make <a href="#F1" class="usa-link">figures 1</a>, <a href="#F2" class="usa-link">
2</a> and <a href="#F3" class="usa-link">
3B</a>. We have also used BioRender to make the GRN representations in <a href="#F6" class="usa-link">figure 6</a>. The licences for these figures can be found at <a href="https://BioRender.com/x17b250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/x17b250</a>, <a href="https://BioRender.com/o11x571" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/o11x571</a>, <a href="https://BioRender.com/h7oci6d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/h7oci6d</a>, <a href="https://BioRender.com/2cmrqfo" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/2cmrqfo</a>, <a href="https://BioRender.com/93cnjgv" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/93cnjgv</a>, <a href="https://BioRender.com/15el1ti" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/15el1ti</a> and <a href="https://BioRender.com/7x9jrjy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/7x9jrjy</a>.</p></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Lucas F. Jansen Klomp, Email: l.f.jansenklomp@utwente.nl.</p>
<p>Elena Queirolo, Email: elena.queirolo@univ-rennes.fr.</p>
<p>Janine N. Post, Email: j.n.post@utwente.nl.</p>
<p>Hil G. E. Meijer, Email: H.G.E.Meijer@utwente.nl.</p>
<p>Christoph Brune, Email: c.brune@utwente.nl.</p></section><section id="s5"><h2 class="pmc_sec_title">Ethics</h2>
<p>This work did not require ethical approval from a human subject or animal welfare committee.</p></section><section id="s6"><h2 class="pmc_sec_title">Data accessibility</h2>
<p>All code used to generate the results and figures shown in this paper can be accessed at GitHub [<a href="#B41" class="usa-link" aria-describedby="B41">41</a>] and has been archived with Zenodo [<a href="#B42" class="usa-link" aria-describedby="B42">42</a>]. For the generation of synthetic single-cell data, BoolODE was used [<a href="#B43" class="usa-link" aria-describedby="B43">43</a>]. We have used six representative graphs provided in this repository for testing our framework: dyn-bifurcating, dyn-bifurcating-converging, dyn-linear, dyn-linear-long, dyn-cycle and dyn-trifurcating. We have used the implementations of GENIE3, PIDC and GRNBOOST2 for benchmarking as included in the BEELINE package [<a href="#B44" class="usa-link" aria-describedby="B44">44</a>].</p></section><section id="s7"><h2 class="pmc_sec_title">Declaration of AI use</h2>
<p>The authors have used AI tools to review specific paragraphs of the text, leading to insights into how these paragraphs could be made more clear. No AI-generated text is included in the manuscript. Furthermore, the authors used AI tools to generate specific snippets of code. The resulting code was thereafter thoroughly checked and the authors take full responsibility for any parts of this code used in the final codebase.</p></section><section id="s8"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>L.J.K.: conceptualization, formal analysis, investigation, methodology, software, visualization, writing—original draft; E.Q.: conceptualization, formal analysis, investigation, methodology, software, writing—original draft, writing—review and editing; J.P.: conceptual‑ ization, supervision, writing—review and editing; H.M.: conceptualization, supervision, writing—review and editing; C.B.: conceptualization, methodology, supervision, writing—review and editing.</p>
<p>All authors gave final approval for publication and agreed to be held accountable for the work performed therein.</p></section><section id="s9"><h2 class="pmc_sec_title">Conflict of interest declaration</h2>
<p>We declare we have no competing interests.</p></section><section id="s10"><h2 class="pmc_sec_title">Funding</h2>
<p>L.J.K., J.P., H.M. and C.B. acknowledge support by the NWO-XL project SCI-MAP: OCENW.GROOT.2019.079. E.Q. was partially supported by the German Science Foundation (Deutsche Forschungsgemeinschaft, DFG) through Walter-Benjamin Grant QU 579/1-1.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>
Tang F, et al. 
. 
2009. 
mRNA-Seq whole-transcriptome analysis of a single cell
. 

Nat. Methods



6


, 
377
–
382
. (
10.1038/nmeth.1315
)

</cite> [<a href="https://doi.org/10.1038/nmeth.1315" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19349980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=mRNA-Seq%20whole-transcriptome%20analysis%20of%20a%20single%20cell&amp;volume=6&amp;publication_year=2009&amp;pages=377-382&amp;pmid=19349980&amp;doi=10.1038/nmeth.1315&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>
Haque A
, 
Engel J
, 
Teichmann SA
, 
Lönnberg T
. 
2017. 
A practical guide to single-cell RNA-sequencing for biomedical research and clinical applications
. 

Genome Med.



9


, 
1
–
12
. (
10.1186/s13073-017-0467-4
)

</cite> [<a href="https://doi.org/10.1186/s13073-017-0467-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5561556/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28821273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Med.&amp;title=A%20practical%20guide%20to%20single-cell%20RNA-sequencing%20for%20biomedical%20research%20and%20clinical%20applications&amp;volume=9&amp;publication_year=2017&amp;pages=1-12&amp;pmid=28821273&amp;doi=10.1186/s13073-017-0467-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>
Pratapa A
, 
Jalihal AP
, 
Law JN
, 
Bharadwaj A
, 
Murali TM
. 
2020. 
Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data
. 

Nat. Methods


17

, 
147
–
154
. (
10.1038/s41592-019-0690-6
)

</cite> [<a href="https://doi.org/10.1038/s41592-019-0690-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7098173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31907445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Benchmarking%20algorithms%20for%20gene%20regulatory%20network%20inference%20from%20single-cell%20transcriptomic%20data&amp;volume=17&amp;publication_year=2020&amp;pages=147-154&amp;pmid=31907445&amp;doi=10.1038/s41592-019-0690-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>
Langfelder P
, 
Horvath S
. 
2008. 
WGCNA: an R package for weighted correlation network analysis
. 

BMC Bioinform.



9


, 
1
–
13
. (
10.1186/1471-2105-9-559
)
</cite> [<a href="https://doi.org/10.1186/1471-2105-9-559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2631488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19114008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinform.&amp;title=WGCNA:%20an%20R%20package%20for%20weighted%20correlation%20network%20analysis&amp;volume=9&amp;publication_year=2008&amp;pages=1-13&amp;pmid=19114008&amp;doi=10.1186/1471-2105-9-559&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>
Margolin AA
, 
Nemenman I
, 
Basso K
, 
Wiggins C
, 
Stolovitzky G
, 
Dalla Favera R
, 
Califano A
. 
2006. 
ARACNE: an algorithm for the reconstruction of gene regulatory networks in a mammalian cellular context
. 

BMC Bioinform.



7


, 
1
–
15
. (
10.1186/1471-2105-7-S1-S7
)
</cite> [<a href="https://doi.org/10.1186/1471-2105-7-S1-S7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1810318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16723010/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinform.&amp;title=ARACNE:%20an%20algorithm%20for%20the%20reconstruction%20of%20gene%20regulatory%20networks%20in%20a%20mammalian%20cellular%20context&amp;volume=7&amp;publication_year=2006&amp;pages=1-15&amp;pmid=16723010&amp;doi=10.1186/1471-2105-7-S1-S7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>
Huynh-Thu VA
, 
Irrthum A
, 
Wehenkel L
, 
Geurts P
. 
2010. 
Inferring regulatory networks from expression data using tree-based methods
. 

PLoS One


5

, 
e12776
. (
10.1371/journal.pone.0012776
)

</cite> [<a href="https://doi.org/10.1371/journal.pone.0012776" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2946910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20927193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Inferring%20regulatory%20networks%20from%20expression%20data%20using%20tree-based%20methods&amp;volume=5&amp;publication_year=2010&amp;pages=e12776&amp;pmid=20927193&amp;doi=10.1371/journal.pone.0012776&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>
Aibar S, et al. 
. 
2017. 
SCENIC: single-cell regulatory network inference and clustering
. 

Nat. Methods


14

, 
1083
–
1086
. (
10.1038/nmeth.4463
)

</cite> [<a href="https://doi.org/10.1038/nmeth.4463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5937676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28991892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=SCENIC:%20single-cell%20regulatory%20network%20inference%20and%20clustering&amp;volume=14&amp;publication_year=2017&amp;pages=1083-1086&amp;pmid=28991892&amp;doi=10.1038/nmeth.4463&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>
Street K
, 
Risso D
, 
Fletcher RB
, 
Das D
, 
Ngai J
, 
Yosef N
, 
Purdom E
, 
Dudoit S
. 
2018. 
Slingshot: cell lineage and pseudotime inference for single-cell transcriptomics
. 

BMC Genom.



19


, 
1
–
16
. (
10.1186/s12864-018-4772-0
)
</cite> [<a href="https://doi.org/10.1186/s12864-018-4772-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6007078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29914354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&amp;title=Slingshot:%20cell%20lineage%20and%20pseudotime%20inference%20for%20single-cell%20transcriptomics&amp;volume=19&amp;publication_year=2018&amp;pages=1-16&amp;pmid=29914354&amp;doi=10.1186/s12864-018-4772-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>
Trapnell C, et al. 
. 
2014. 
The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells
. 

Nat. Biotechnol.

32

, 
381
–
386
. (
10.1038/nbt.2859
)

</cite> [<a href="https://doi.org/10.1038/nbt.2859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4122333/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24658644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=The%20dynamics%20and%20regulators%20of%20cell%20fate%20decisions%20are%20revealed%20by%20pseudotemporal%20ordering%20of%20single%20cells&amp;volume=32&amp;publication_year=2014&amp;pages=381-386&amp;pmid=24658644&amp;doi=10.1038/nbt.2859&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>
Mao G
, 
Pang Z
, 
Zuo K
, 
Wang Q
, 
Pei X
, 
Chen X
, 
Liu J
. 
2023. 
Predicting gene regulatory links from single-cell RNA-seq data using graph neural networks
. 

Briefings Bioinform.

24

, 
d414
. (
10.1093/bib/bbad414
)
</cite> [<a href="https://doi.org/10.1093/bib/bbad414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10661972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37985457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Briefings%20Bioinform.&amp;title=Predicting%20gene%20regulatory%20links%20from%20single-cell%20RNA-seq%20data%20using%20graph%20neural%20networks&amp;volume=24&amp;publication_year=2023&amp;pages=d414&amp;pmid=37985457&amp;doi=10.1093/bib/bbad414&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>
Shu H
, 
Zhou J
, 
Lian Q
, 
Li H
, 
Zhao D
, 
Zeng J
, 
Ma J
. 
2021. 
Modeling gene regulatory networks using neural network architectures
. 

Nat. Comput. Sci.

1

, 
491
–
501
. (
10.1038/s43588-021-00099-8
)

</cite> [<a href="https://doi.org/10.1038/s43588-021-00099-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38217125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Comput.%20Sci.&amp;title=Modeling%20gene%20regulatory%20networks%20using%20neural%20network%20architectures&amp;volume=1&amp;publication_year=2021&amp;pages=491-501&amp;pmid=38217125&amp;doi=10.1038/s43588-021-00099-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>
Wang J
, 
Chen Y
, 
Zou Q
. 
2023. 
Inferring gene regulatory network from single-cell transcriptomes with graph autoencoder model
. 

PLoS Genet.

19

, 
e1010942
. (
10.1371/journal.pgen.1010942
)

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1010942" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10519590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37703293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=Inferring%20gene%20regulatory%20network%20from%20single-cell%20transcriptomes%20with%20graph%20autoencoder%20model&amp;volume=19&amp;publication_year=2023&amp;pages=e1010942&amp;pmid=37703293&amp;doi=10.1371/journal.pgen.1010942&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>
Wei PJ
, 
Guo Z
, 
Gao Z
, 
Ding Z
, 
Cao RF
, 
Su Y
, 
Zheng CH
. 
2024. 
Inference of gene regulatory networks based on directed graph convolutional networks
. 

Brief. Bioinform.



25


, 
bbae309
.

</cite> [<a href="https://doi.org/10.1093/bib/bbae309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11209731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38935070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brief.%20Bioinform.&amp;title=Inference%20of%20gene%20regulatory%20networks%20based%20on%20directed%20graph%20convolutional%20networks&amp;volume=25&amp;publication_year=2024&amp;pages=bbae309&amp;pmid=38935070&amp;doi=10.1093/bib/bbae309&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B14">
<span class="label">14.</span><cite>
Chen G
, 
Liu ZP
. 
2022. 
Graph attention network for link prediction of gene regulations from single-cell RNA-sequencing data
. 

Bioinformatics


38

, 
4522
–
4529
. (
10.1093/bioinformatics/btac559
)

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btac559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35961023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Graph%20attention%20network%20for%20link%20prediction%20of%20gene%20regulations%20from%20single-cell%20RNA-sequencing%20data&amp;volume=38&amp;publication_year=2022&amp;pages=4522-4529&amp;pmid=35961023&amp;doi=10.1093/bioinformatics/btac559&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B15">
<span class="label">15.</span><cite>
Matsumoto H
, 
Kiryu H
, 
Furusawa C
, 
Ko MSH
, 
Ko SBH
, 
Gouda N
, 
Hayashi T
, 
Nikaido I
. 
2017. 
SCODE: an efficient regulatory network inference algorithm from single-cell RNA-Seq during differentiation
. 

Bioinformatics


33

, 
2314
–
2321
. (
10.1093/bioinformatics/btx194
)

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5860123/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28379368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SCODE:%20an%20efficient%20regulatory%20network%20inference%20algorithm%20from%20single-cell%20RNA-Seq%20during%20differentiation&amp;volume=33&amp;publication_year=2017&amp;pages=2314-2321&amp;pmid=28379368&amp;doi=10.1093/bioinformatics/btx194&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B16">
<span class="label">16.</span><cite>
Erbe R
, 
Stein-O’Brien G
, 
Fertig EJ
. 
2023. 
Transcriptomic forecasting with neural ordinary differential equations
. 

Patterns


4

, 
100793
. (
10.1016/j.patter.2023.100793
)

</cite> [<a href="https://doi.org/10.1016/j.patter.2023.100793" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10435954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37602211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Patterns&amp;title=Transcriptomic%20forecasting%20with%20neural%20ordinary%20differential%20equations&amp;volume=4&amp;publication_year=2023&amp;pages=100793&amp;pmid=37602211&amp;doi=10.1016/j.patter.2023.100793&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B17">
<span class="label">17.</span><cite>
Hossain I
, 
Fanfani V
, 
Fischer J
, 
Quackenbush J
, 
Burkholz R
. 
2024. 
Biologically informed NeuralODEs for genome-wide regulatory dynamics
. 

Genome Biol.

25

, 
127
. (
10.1186/s13059-024-03264-0
)

</cite> [<a href="https://doi.org/10.1186/s13059-024-03264-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11106922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38773638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Biologically%20informed%20NeuralODEs%20for%20genome-wide%20regulatory%20dynamics&amp;volume=25&amp;publication_year=2024&amp;pages=127&amp;pmid=38773638&amp;doi=10.1186/s13059-024-03264-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B18">
<span class="label">18.</span><cite>
Cummins B
, 
Gedeon T
, 
Harker S
, 
Mischaikow K
. 
2018. 
DSGRN: examining the dynamics of families of logical models
. 

Front. Physiol.

9

, 
549
. (
10.3389/fphys.2018.00549
)

</cite> [<a href="https://doi.org/10.3389/fphys.2018.00549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5975363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29875674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Physiol.&amp;title=DSGRN:%20examining%20the%20dynamics%20of%20families%20of%20logical%20models&amp;volume=9&amp;publication_year=2018&amp;pages=549&amp;pmid=29875674&amp;doi=10.3389/fphys.2018.00549&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B19">
<span class="label">19.</span><cite>
Remy É
, 
Ruet P
, 
Thieffry D
. 
2008. 
Graphic requirements for multistability and attractive cycles in a Boolean dynamical framework
. 

Adv. Appl. Math.

41

, 
335
–
350
. (
10.1016/j.aam.2007.11.003
)
</cite> [<a href="https://doi.org/10.1016/j.aam.2007.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Appl.%20Math.&amp;title=Graphic%20requirements%20for%20multistability%20and%20attractive%20cycles%20in%20a%20Boolean%20dynamical%20framework&amp;volume=41&amp;publication_year=2008&amp;pages=335-350&amp;doi=10.1016/j.aam.2007.11.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B20">
<span class="label">20.</span><cite>
Gameiro M
, 
Gedeon T
, 
Harker S
, 
Mischaikow K
. 
2024. 
Global dynamics of ordinary differential equations: wall labelings, Conley complexes, and ramp systems
<a href="https://arxiv.org/abs/2412.11078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/2412.11078</a>
</cite>
</li>
<li id="B21">
<span class="label">21.</span><cite>
Kepley S
, 
Mischaikow K
, 
Queirolo E
. 
2022. 
Global analysis of regulatory network dynamics: equilibria and saddle-node bifurcations
<a href="https://arxiv.org/abs/2204.13739" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/2204.13739</a>
</cite>
</li>
<li id="B22">
<span class="label">22.</span><cite>
Hamilton W
, 
Ying Z
, 
Leskovec J
. 
2017. 
Inductive representation learning on large graphs
. 
In

Proc. 31st Int. Conf. on Neural Information Processing Systems, Long Beach, CA, USA, 4–9 December 2017

, 
pp
. 
1025
–
1035
.
</cite> [<a href="https://scholar.google.com/scholar_lookup?Hamilton%20W%20,%20Ying%20Z%20,%20Leskovec%20J%20.%202017.%20Inductive%20representation%20learning%20on%20large%20graphs%20.%20In%20Proc.%2031st%20Int.%20Conf.%20on%20Neural%20Information%20Processing%20Systems,%20Long%20Beach,%20CA,%20USA,%204%E2%80%939%20December%202017%20,%20pp%20.%201025%20%E2%80%93%201035%20." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B23">
<span class="label">23.</span><cite>
Kang X
, 
Hajek B
, 
Hanzawa Y
. 
2020. 
From graph topology to ODE models for gene regulatory networks
. 

PLoS One


15

, 
e0235070
. (
10.1371/journal.pone.0235070
)

</cite> [<a href="https://doi.org/10.1371/journal.pone.0235070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7326199/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32603340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=From%20graph%20topology%20to%20ODE%20models%20for%20gene%20regulatory%20networks&amp;volume=15&amp;publication_year=2020&amp;pages=e0235070&amp;pmid=32603340&amp;doi=10.1371/journal.pone.0235070&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B24">
<span class="label">24.</span><cite>
Zhou JX
, 
Brusch L
, 
Huang S
. 
2011. 
Predicting pancreas cell fate decisions and reprogramming with a hierarchical multi-attractor model
. 

PLoS One


6

, 
e14752
. (
10.1371/journal.pone.0014752
)

</cite> [<a href="https://doi.org/10.1371/journal.pone.0014752" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3056652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21423725/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Predicting%20pancreas%20cell%20fate%20decisions%20and%20reprogramming%20with%20a%20hierarchical%20multi-attractor%20model&amp;volume=6&amp;publication_year=2011&amp;pages=e14752&amp;pmid=21423725&amp;doi=10.1371/journal.pone.0014752&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B25">
<span class="label">25.</span><cite>
Tibshirani R
. 
1996. 
Regression shrinkage and selection via the lasso
. 

J. R. Stat. Soc. B



58


, 
267
–
288
.
</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Stat.%20Soc.%20B&amp;title=Regression%20shrinkage%20and%20selection%20via%20the%20lasso&amp;volume=58&amp;publication_year=1996&amp;pages=267-288&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B26">
<span class="label">26.</span><cite>
Cai S
, 
Mao Z
, 
Wang Z
, 
Yin M
, 
Karniadakis GE
. 
2021. 
Physics-informed neural networks (PINNs) for fluid mechanics: a review
. 

Acta Mech. Sin.

37

, 
1727
–
1738
. (
10.1007/s10409-021-01148-1
)
</cite> [<a href="https://doi.org/10.1007/s10409-021-01148-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Mech.%20Sin.&amp;title=Physics-informed%20neural%20networks%20(PINNs)%20for%20fluid%20mechanics:%20a%20review&amp;volume=37&amp;publication_year=2021&amp;pages=1727-1738&amp;doi=10.1007/s10409-021-01148-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B27">
<span class="label">27.</span><cite>
Storn R
, 
Price K
. 
1997. 
Differential evolution—a simple and efficient heuristic for global optimization over continuous spaces
. 

J. Global Opti



11


, 
341
–
359
. (
10.1023/A:1008202821328
)
</cite> [<a href="https://doi.org/10.1023/A:1008202821328" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Global%20Opti&amp;title=Differential%20evolution%E2%80%94a%20simple%20and%20efficient%20heuristic%20for%20global%20optimization%20over%20continuous%20spaces&amp;volume=11&amp;publication_year=1997&amp;pages=341-359&amp;doi=10.1023/A:1008202821328&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B28">
<span class="label">28.</span><cite>
Chu LF, et al. 
. 
2016. 
Single-cell RNA-seq reveals novel regulators of human embryonic stem cell differentiation to definitive endoderm
. 

Genome Biol.

17

, 
20
. (
10.1186/s13059-016-1033-x
)

</cite> [<a href="https://doi.org/10.1186/s13059-016-1033-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4989499/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27534536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Single-cell%20RNA-seq%20reveals%20novel%20regulators%20of%20human%20embryonic%20stem%20cell%20differentiation%20to%20definitive%20endoderm&amp;volume=17&amp;publication_year=2016&amp;pages=20&amp;pmid=27534536&amp;doi=10.1186/s13059-016-1033-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B29">
<span class="label">29.</span><cite>
Moerman T
, 
Aibar Santos S
, 
Bravo González-Blas C
, 
Simm J
, 
Moreau Y
, 
Aerts J
, 
Aerts S
. 
2019. 
GRNBoost2 and Arboreto: efficient and scalable inference of gene regulatory networks
. 

Bioinformatics


35

, 
2159
–
2161
. (
10.1093/bioinformatics/bty916
)

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty916" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30445495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GRNBoost2%20and%20Arboreto:%20efficient%20and%20scalable%20inference%20of%20gene%20regulatory%20networks&amp;volume=35&amp;publication_year=2019&amp;pages=2159-2161&amp;pmid=30445495&amp;doi=10.1093/bioinformatics/bty916&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B30">
<span class="label">30.</span><cite>
Chan TE
, 
Stumpf MPH
, 
Babtie AC
. 
2017. 
Gene regulatory network inference from single-cell data using multivariate information measures
. 

Cell Syst.

5

, 
251
–
267
.(
10.1016/j.cels.2017.08.014
)

</cite> [<a href="https://doi.org/10.1016/j.cels.2017.08.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5624513/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28957658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Syst.&amp;title=Gene%20regulatory%20network%20inference%20from%20single-cell%20data%20using%20multivariate%20information%20measures&amp;volume=5&amp;publication_year=2017&amp;pages=251-267&amp;pmid=28957658&amp;doi=10.1016/j.cels.2017.08.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B31">
<span class="label">31.</span><cite>
Dhooge A
, 
Govaerts W
, 
Kuznetsov YuA
, 
Meijer HGE
, 
Sautois B
. 
2008. 
New features of the software MatCont for bifurcation analysis of dynamical systems
. 

Math. Comput. Model. Dyn. Syst.

14

, 
147
–
175
. (
10.1080/13873950701742754
)
</cite> [<a href="https://doi.org/10.1080/13873950701742754" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Math.%20Comput.%20Model.%20Dyn.%20Syst.&amp;title=New%20features%20of%20the%20software%20MatCont%20for%20bifurcation%20analysis%20of%20dynamical%20systems&amp;volume=14&amp;publication_year=2008&amp;pages=147-175&amp;doi=10.1080/13873950701742754&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B32">
<span class="label">32.</span><cite>
Brunton SL
, 
Proctor JL
, 
Kutz JN
. 
2016. 
Discovering governing equations from data by sparse identification of nonlinear dynamical systems
. 

Proc. Natl Acad. Sci. USA



113


, 
3932
–
3937
. (
10.1073/pnas.1517384113
)

</cite> [<a href="https://doi.org/10.1073/pnas.1517384113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4839439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27035946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=Discovering%20governing%20equations%20from%20data%20by%20sparse%20identification%20of%20nonlinear%20dynamical%20systems&amp;volume=113&amp;publication_year=2016&amp;pages=3932-3937&amp;pmid=27035946&amp;doi=10.1073/pnas.1517384113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B33">
<span class="label">33.</span><cite>
de Silva B
, 
Champion K
, 
Quade M
, 
Loiseau JC
, 
Kutz J
, 
Brunton S
. 
2020. 
PySINDy: a Python package for the sparse identification of nonlinear dynamical systems from data
. 

J. Open Source Softw.

5

, 
2104
. (
10.21105/joss.02104
)
</cite> [<a href="https://doi.org/10.21105/joss.02104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Open%20Source%20Softw.&amp;title=PySINDy:%20a%20Python%20package%20for%20the%20sparse%20identification%20of%20nonlinear%20dynamical%20systems%20from%20data&amp;volume=5&amp;publication_year=2020&amp;pages=2104&amp;doi=10.21105/joss.02104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B34">
<span class="label">34.</span><cite>
Kaptanoglu A, et al. 
. 
2022. 
PySINDy: a comprehensive Python package for robust sparse system identification
. 

J. Open Source Softw.

7

, 
3994
. (
10.21105/joss.03994
)
</cite> [<a href="https://doi.org/10.21105/joss.03994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Open%20Source%20Softw.&amp;title=PySINDy:%20a%20comprehensive%20Python%20package%20for%20robust%20sparse%20system%20identification&amp;volume=7&amp;publication_year=2022&amp;pages=3994&amp;doi=10.21105/joss.03994&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B35">
<span class="label">35.</span><cite>
Chen RT
, 
Rubanova Y
, 
Bettencourt J
, 
Duvenaud DK
. 
2018. 
Neural ordinary differential equations
. 
In

Proc. 32nd Int. Conf. on Neural Information Processing Systems, Montreal, Canada, 3–8 December 2018

, 
pp
. 
6572
–
6583
. 
<a href="https://dl.acm.org/doi/pdf/10.5555/3327757.3327764" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dl.acm.org/doi/pdf/10.5555/3327757.3327764</a>
.
</cite> [<a href="https://scholar.google.com/scholar_lookup?Chen%20RT%20,%20Rubanova%20Y%20,%20Bettencourt%20J%20,%20Duvenaud%20DK%20.%202018.%20Neural%20ordinary%20differential%20equations%20.%20In%20Proc.%2032nd%20Int.%20Conf.%20on%20Neural%20Information%20Processing%20Systems,%20Montreal,%20Canada,%203%E2%80%938%20December%202018%20,%20pp%20.%206572%20%E2%80%93%206583%20.%20https://dl.acm.org/doi/pdf/10.5555/3327757.3327764%20." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B36">
<span class="label">36.</span><cite>
Poli M
, 
Massaroli S
, 
Park J
, 
Yamashita A
, 
Asama H
, 
Park J
. 
2019. 
Graph neural ordinary differential equations
. 
See
<a href="https://arxiv.org/abs/1911.07532" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/1911.07532</a>
.
</cite>
</li>
<li id="B37">
<span class="label">37.</span><cite>
Tong A
, 
Huang J
, 
Wolf G
, 
van Dijk D
, 
Krishnaswamy S
. 
2020. 
TrajectoryNet: a dynamic optimal transport network for modeling cellular dynamics
. 

PMLR



119


, 
9526
–
9536
. 
<a href="https://proceedings.mlr.press/v119/tong20a.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://proceedings.mlr.press/v119/tong20a.html</a>

</cite> [<a href="/articles/PMC8320749/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34337419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PMLR&amp;title=TrajectoryNet:%20a%20dynamic%20optimal%20transport%20network%20for%20modeling%20cellular%20dynamics&amp;volume=119&amp;publication_year=2020&amp;pages=9526-9536&amp;pmid=34337419&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B38">
<span class="label">38.</span><cite>
Huguet G
, 
Magruder DS
, 
Tong A
, 
Fasina O
, 
Kuchroo M
, 
Wolf G
, 
Krishnaswamy S
. 
2022. 
Manifold interpolating optimal-transport flows for trajectory inference
. 

Adv. Neural Inf. Process. Syst.

35

, 
29705
–
29718
.

</cite> [<a href="/articles/PMC10312391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37397786/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Neural%20Inf.%20Process.%20Syst.&amp;title=Manifold%20interpolating%20optimal-transport%20flows%20for%20trajectory%20inference&amp;volume=35&amp;publication_year=2022&amp;pages=29705-29718&amp;pmid=37397786&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B39">
<span class="label">39.</span><cite>
Jiang Q
, 
Wan L
. 
2024. 
A physics-informed neural SDE network for learning cellular dynamics from time-series scRNA-seq data
. 

Bioinformatics



40


, 
ii120
–
ii127
. (
10.1093/bioinformatics/btae400
)

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btae400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11373338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39230705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20physics-informed%20neural%20SDE%20network%20for%20learning%20cellular%20dynamics%20from%20time-series%20scRNA-seq%20data&amp;volume=40&amp;publication_year=2024&amp;pages=ii120-ii127&amp;pmid=39230705&amp;doi=10.1093/bioinformatics/btae400&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B40">
<span class="label">40.</span><cite>
Zhang Z
, 
Li T
, 
Zhou P
. 
2024. 
Learning stochastic dynamics from snapshots through regularized unbalanced optimal transport
. 
See
<a href="https://arxiv.org/abs/2410.00844" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/2410.00844</a>
.
</cite>
</li>
<li id="B41">
<span class="label">41.</span><cite>
Queirolo E
. 
2025. 

BioML. GitHub

. 
See
<a href="https://github.com/elenaquei/BioML/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/elenaquei/BioML/</a>
.
</cite>
</li>
<li id="B42">
<span class="label">42.</span><cite>
Queirolo E
, 
Jansen Klomp L
. 
2025. 
elenaquei/BioML: v1.0 (release). Zenodo.
(
10.5281/zenodo.15829387
)
</cite> [<a href="https://doi.org/10.5281/zenodo.15829387" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="B43">
<span class="label">43.</span><cite>
Murali-group 
. 
2023. 

BoolODE. GitHub

. 
See
<a href="https://github.com/Murali-group/BoolODE" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Murali-group/BoolODE</a>
.
</cite>
</li>
<li id="B44">
<span class="label">44.</span><cite>
Murali-group 
. 
2025. 

Beeline. GitHub

. 
See
<a href="https://github.com/Murali-group/Beeline" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Murali-group/Beeline</a>
.
</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All code used to generate the results and figures shown in this paper can be accessed at GitHub [<a href="#B41" class="usa-link" aria-describedby="B41">41</a>] and has been archived with Zenodo [<a href="#B42" class="usa-link" aria-describedby="B42">42</a>]. For the generation of synthetic single-cell data, BoolODE was used [<a href="#B43" class="usa-link" aria-describedby="B43">43</a>]. We have used six representative graphs provided in this repository for testing our framework: dyn-bifurcating, dyn-bifurcating-converging, dyn-linear, dyn-linear-long, dyn-cycle and dyn-trifurcating. We have used the implementations of GENIE3, PIDC and GRNBOOST2 for benchmarking as included in the BEELINE package [<a href="#B44" class="usa-link" aria-describedby="B44">44</a>].</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Interface Focus are provided here courtesy of <strong>The Royal Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1098/rsfs.2025.0014"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/rsfs.2025.0014.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371343/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371343/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371343%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371343/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371343/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371343/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371343/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371343/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371343/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="GpWMx7HJy2FyT6qUlfNVPF18Lb2u2LjAaczIuvA4ydvC5WUPymP7uEOFjxOl1guM">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
