
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            SV-MeCa: an XGBoost-based meta-caller approach for structural variant calling from short-read data - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5337928AF2FAF3053792003CDAC7D4.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcbioi">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366149/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Bioinformatics">
<meta name="citation_title" content="SV-MeCa: an XGBoost-based meta-caller approach for structural variant calling from short-read data">
<meta name="citation_author" content="Rudel Christian Nkouamedjo Fankep">
<meta name="citation_author_institution" content="Center for Familial Breast and Ovarian Cancer, Center for Integrated Oncology (CIO), University of Cologne, Faculty of Medicine and University Hospital Cologne, Cologne, Germany">
<meta name="citation_author" content="Arda Söylev">
<meta name="citation_author_institution" content="Institute for Medical Biometry and Bioinformatics, Heinrich Heine University Düsseldorf, Medical Faculty, Düsseldorf, Germany">
<meta name="citation_author_institution" content="Center for Digital Medicine, Heinrich Heine University Düsseldorf, Düsseldorf, Germany">
<meta name="citation_author" content="Anna-Lena Kobiela">
<meta name="citation_author_institution" content="Center for Familial Breast and Ovarian Cancer, Center for Integrated Oncology (CIO), University of Cologne, Faculty of Medicine and University Hospital Cologne, Cologne, Germany">
<meta name="citation_author" content="Jochen Blom">
<meta name="citation_author_institution" content="Bioinformatics and Systems Biology, Justus Liebig University Gießen, Gießen, Germany">
<meta name="citation_author" content="Corinna Ernst">
<meta name="citation_author_institution" content="Center for Familial Breast and Ovarian Cancer, Center for Integrated Oncology (CIO), University of Cologne, Faculty of Medicine and University Hospital Cologne, Cologne, Germany">
<meta name="citation_author" content="Susanne Motameny">
<meta name="citation_author_institution" content="Cologne Center for Genomics, University of Cologne, Medical Faculty, Cologne, Germany">
<meta name="citation_author_institution" content="West German Genome Center - Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="218">
<meta name="citation_doi" content="10.1186/s12859-025-06246-6">
<meta name="citation_pmid" content="40836322">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366149/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366149/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366149/pdf/12859_2025_Article_6246.pdf">
<meta name="description" content="Calling structural variants (SVs), i.e., genomic alterations of 50bp, from whole genome short-read data remains challenging, as existing callers are known to lack accuracy and robustness. Therefore, meta-caller approaches combining the results of ...">
<meta name="og:title" content="SV-MeCa: an XGBoost-based meta-caller approach for structural variant calling from short-read data">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Calling structural variants (SVs), i.e., genomic alterations of 50bp, from whole genome short-read data remains challenging, as existing callers are known to lack accuracy and robustness. Therefore, meta-caller approaches combining the results of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366149/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366149">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12859-025-06246-6"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12859_2025_Article_6246.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366149%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366149/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366149/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366149/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcbioi.png" alt="BMC Bioinformatics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Bioinformatics" title="Link to BMC Bioinformatics" shape="default" href="https://bmcbioinformatics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Bioinformatics</button></div>. 2025 Aug 20;26:218. doi: <a href="https://doi.org/10.1186/s12859-025-06246-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12859-025-06246-6</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Bioinformatics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Bioinformatics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Bioinformatics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Bioinformatics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>SV-MeCa: an XGBoost-based meta-caller approach for structural variant calling from short-read data</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nkouamedjo%20Fankep%20RC%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Rudel Christian Nkouamedjo Fankep</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Rudel Christian Nkouamedjo Fankep</span></h3>
<div class="p">
<sup>1</sup>Center for Familial Breast and Ovarian Cancer, Center for Integrated Oncology (CIO), University of Cologne, Faculty of Medicine and University Hospital Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nkouamedjo%20Fankep%20RC%22%5BAuthor%5D" class="usa-link"><span class="name western">Rudel Christian Nkouamedjo Fankep</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22S%C3%B6ylev%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Arda Söylev</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Arda Söylev</span></h3>
<div class="p">
<sup>2</sup>Institute for Medical Biometry and Bioinformatics, Heinrich Heine University Düsseldorf, Medical Faculty, Düsseldorf, Germany </div>
<div class="p">
<sup>3</sup>Center for Digital Medicine, Heinrich Heine University Düsseldorf, Düsseldorf, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22S%C3%B6ylev%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Arda Söylev</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kobiela%20AL%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Anna-Lena Kobiela</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Anna-Lena Kobiela</span></h3>
<div class="p">
<sup>1</sup>Center for Familial Breast and Ovarian Cancer, Center for Integrated Oncology (CIO), University of Cologne, Faculty of Medicine and University Hospital Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kobiela%20AL%22%5BAuthor%5D" class="usa-link"><span class="name western">Anna-Lena Kobiela</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blom%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jochen Blom</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jochen Blom</span></h3>
<div class="p">
<sup>4</sup>Bioinformatics and Systems Biology, Justus Liebig University Gießen, Gießen, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blom%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jochen Blom</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ernst%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Corinna Ernst</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Corinna Ernst</span></h3>
<div class="p">
<sup>1</sup>Center for Familial Breast and Ovarian Cancer, Center for Integrated Oncology (CIO), University of Cologne, Faculty of Medicine and University Hospital Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ernst%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Corinna Ernst</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Motameny%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Susanne Motameny</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Susanne Motameny</span></h3>
<div class="p">
<sup>5</sup>Cologne Center for Genomics, University of Cologne, Medical Faculty, Cologne, Germany </div>
<div class="p">
<sup>6</sup>West German Genome Center - Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Motameny%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Susanne Motameny</span></a>
</div>
</div>
<sup>5,</sup><sup>6,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Center for Familial Breast and Ovarian Cancer, Center for Integrated Oncology (CIO), University of Cologne, Faculty of Medicine and University Hospital Cologne, Cologne, Germany </div>
<div id="Aff2">
<sup>2</sup>Institute for Medical Biometry and Bioinformatics, Heinrich Heine University Düsseldorf, Medical Faculty, Düsseldorf, Germany </div>
<div id="Aff3">
<sup>3</sup>Center for Digital Medicine, Heinrich Heine University Düsseldorf, Düsseldorf, Germany </div>
<div id="Aff4">
<sup>4</sup>Bioinformatics and Systems Biology, Justus Liebig University Gießen, Gießen, Germany </div>
<div id="Aff5">
<sup>5</sup>Cologne Center for Genomics, University of Cologne, Medical Faculty, Cologne, Germany </div>
<div id="Aff6">
<sup>6</sup>West German Genome Center - Cologne, University of Cologne, Cologne, Germany </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 20; Accepted 2025 Jul 31; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366149  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40836322/" class="usa-link">40836322</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Calling structural variants (SVs), i.e., genomic alterations of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/24c50dfb2a52/d33e229.gif" loading="lazy" id="d33e229" alt="Inline graphic"></span>50bp, from whole genome short-read data remains challenging, as existing callers are known to lack accuracy and robustness. Therefore, meta-caller approaches combining the results of multiple standalone tools in a consensus set of reported SV calls, are widely used. Here, SV-MeCa (Structural Variant Meta-Caller) is presented, the first SV meta-caller incorporating variant-specific quality metrics from individual VCF outputs, rather than relying solely on number and combination of tools supporting consensus SV calls. In addition, SV-MeCa offers a suitable score to rank obtained consensus SV calls according to evidence of representing true positive calls, i.e., real-world variants.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">SV-MeCa applies seven standalone SV callers and merges resulting deletion and insertion calls into a union VCF file using SURVIVOR. For each entry in the SURVIVOR-generated consensus, caller-specific quality measures are extracted from corresponding standalone VCF files, and serve as input for an either deletion- or insertion-specific XGBoost decision tree classifier, which was previously trained on the HG002 SV benchmark data provided by the Genome in a Bottle consortium. The SV-MeCa XGBoost models assign a probability to (consensus) SV calls to represent true positive calls, which can be used for ranking the final output according to evidence. Performance of SV-MeCa and four previously published meta-caller approaches were evaluated based on autosomal SV calls in samples curated by the Human Genome Structural Variation Consortium, Phase 2. With regard to F<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dc63c1904b90/d33e239.gif" loading="lazy" id="d33e239" alt="Inline graphic"></span> scores, which were 0.58 on average for deletions and 0.42 on average for insertions, SV-MeCa outperformed the other meta-callers. With regard to precision, only ConsensuSV achieved higher values (0.97 versus 0.64 on average for deletions, 0.75 versus 0.53 on average for insertions), and with regard to recall, SV-MeCa was outperformed exclusively by Meta-SV for deletions (0.55 versus 0.53).</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">SV-MeCa, publicly available at <a href="https://github.com/ccfboc-bioinformatics/SV-MeCa" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ccfboc-bioinformatics/SV-MeCa</a>, outperforms existing SV meta-caller approaches by taking variant-specific quality measures into account. Moreover, due to the XGBoost prediction probabilities serving as scores, the output of SV-MeCa can be continuously adjusted to user needs in terms of sensitivity and precision.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12859-025-06246-6.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Structural variants, Variant calling, Whole-genome sequencing, Meta-caller</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par4">Structural variants (SVs) are usually defined as genomic alterations of the types deletions, duplications, insertions, inversions, or translocations affecting <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/24c50dfb2a52/d33e272.gif" loading="lazy" id="d33e272" alt="Inline graphic"></span>50bp [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Detection of SVs based on short-read data requires more sophisticated algorithms compared to single nucleotide variants and short indels discovery [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Current approaches typically pursue one or a combination of four main strategies. Read depth- or read count-based approaches are commonly used for the identification of copy number variants (CNVs), i.e., large deletions or duplications having significant effects on observed read depth in the affected genomic region. On the other hand, read-pair methods consider insert-size and orientation of paired-end reads, whereas the split read-based approaches identify the start and stop positions of putative SVs by scanning for partially mapped (i.e., split) reads. Furthermore, SVs can also be detected based on <em>de novo</em> assembly.</p>
<p id="Par5">Apart from the inherent limitations of short-read sequencing, the accuracy and robustness of current SV callers limit our ability to discover SVs effectively. Thus, we are not able to detect the full range of existing SVs [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Meta-caller approaches, combining the results of multiple standalone tools have been developed and shown to achieve improved performance [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>] in this regard. However, all the currently available approaches applicable to short-read whole-genome sequencing (WGS) data have their specific drawbacks and limitations. MetaSV expects already generated VCF files as input, leaving the execution of standalone SV callers to the responsibility of the user [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Furthermore, it is unable to extract insertion calls properly from VCF input (except for Pindel [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]) and therefore includes its own split read- and assembly-based method for insertion detection, which in turn makes the intended meta-approach obsolete. In contrast, Parliament2 [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>] and ConsensuSV [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>] provide ready-to-use SV meta-caller pipelines that start from raw FASTQ files and allow an SV meta-calling with minimal user effort. Both tools, however, predominantly assess the evidence of an SV call being true positive (TP) based on the consistency with which it is detected by multiple standalone callers or their combination. Parliament2 reports quality values of consensus SV calls based on pre-computed precisions per combination of involved standalone SV callers, SV type and SV length range [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. ConsensuSV considers and reports only overlapping SV calls of the same type originating from at least a pre-defined number of standalone callers [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Similarly, FusorSV uses pre-computed cut-offs per SV caller combination and SV length range to determine the final output of consensus SV calls [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. Also, the recently published VISTA tool creates the final output from the union of all SV calls of a pre-defined set of standalone callers per SV type and length range [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. All these strategies share the common fundamental limitation that individual SV calls enter the process of building a final consensus list of putative SVs as virtually binary encodings of their presence. SV calls from only one or very few standalone tools are generally considered to have a low probability to represent TP events, regardless of the qualities assigned to them. Inversely, consistent calls from many approaches are always associated with a high probability of representing TPs, even if consistently reported as low confidence calls.</p>
<p id="Par6">Here we present SV-MeCa (Structural Variant Meta-Caller), which overcomes these limitations of consensus SV call quality assessment by incorporating variant-specific quality metrics from individual SV caller’s VCF outputs. In addition, SV-MeCa provides a suitable and continuous score to rank consensus SV calls according to evidence of representing TP events.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">General approach of SV-MeCa</h3>
<p id="Par7">Starting from a paired-end short-read WGS BAM file including split read information, SV-MeCa incorporates seven SV callers, namely BreakDancer [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>], Delly [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>], LUMPY [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>], Manta [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>], Pindel [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>], TARDIS [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>], and INSurVeyor [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>] (Fig. <a href="#Fig1" class="usa-link">1</a>). Resulting VCF files are filtered for insertions (including duplications) and deletions with length <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/24c50dfb2a52/d33e366.gif" loading="lazy" id="d33e366" alt="Inline graphic"></span>50bp. Subsequently, deletions reported by LUMPY, Manta, Delly, Pindel, and TARDIS, and insertions reported by LUMPY, Manta, Delly, TARDIS, and INSurVeyor are merged into a single VCF file using SURVIVOR [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. For each entry in the SURVIVOR-generated consensus, caller-specific quality measures are extracted from the corresponding VCF files and serve as input for an either deletion- or insertion-specific XGBoost (Extreme Gradient Boost) decision tree classifier [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. These pre-trained XGBoost models assign probabilities to (consensus) SV calls to represent TPs, which are then used for the ranking of the final output.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366149_12859_2025_6246_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/493f9d7dc2b8/12859_2025_6246_Fig1_HTML.jpg" loading="lazy" id="MO1" height="466" width="709" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Summary flowchart of the SV-MeCa approach. VCF output files from seven standalone structural variant (SV) callers (BreakDancer, LUMPY, Delly, TARDIS, Manta, and Pindel for deletions; Delly, TARDIS, Manta, Pindel, and INSurVeyor for insertions including duplications) are processed. Steps specific to deletions are shown in blue, whereas steps specific to insertions including duplications are shown in red</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Benchmarks</h3>
<p id="Par8">We employed the HG002 (NA24385) benchmark data provided by the Genome in a Bottle (GIAB) consortium [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>] and SVs identified in the children of three parent–child trios (HG00514, HG00733, NA19240) by the Human Genome Structural Variation Consortium, Phase 2 (HGSVC2) [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>], Freeze 3, as ground truths for model development and evaluation. Details on data preparation are given in Supplementary Methods.</p>
<p id="Par9">For the creation of the training data, the GIAB HG002 Tier1 v0.6 SV benchmark [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], including curated SVs in challenging medically relevant genes [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>], was restricted to autosomal variants with length <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/24c50dfb2a52/d33e407.gif" loading="lazy" id="d33e407" alt="Inline graphic"></span>50bp in heterozygous or homozygous genotype. We aligned 250bp Illumina paired reads of HG002 against the GRCh37 reference genome, resulting in a mean coverage of 52x. In order to make it more consistent with scientific and clinical practice, we downsampled the mean coverage to 35x.</p>
<p id="Par10">For evaluation, we employed data mapped to the GRCh38 reference genome exclusively. To assess performance of standalone SV callers and SV-MeCa for SVs in challenging medically relevant genes [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>], we used 150bp paired-end PCR-free WGS data, which was already downsampled to approximately 30x and provided by the Human Pangenome Reference Consortium<a href="#Fn1" class="usa-link">1</a> (HG002-CMRG).</p>
<p id="Par12">In addition, we retrieved paired-end PCR-free WGS data from HG00514, HG00733, and NA19240 from the 1000 Genomes International Genome Sample Resource (IGSR) data portal<a href="#Fn2" class="usa-link">2</a> to compare the performance of SV-MeCa against standalone SV callers and existing meta-caller approaches on autosomal chromosomes. Mean coverages of HG00514, HG00733, and NA1940 ranged between 30x and 31x, and read length was 150bp.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Standalone structural variant callers</h3>
<p id="Par14">Various tools for SV calling based on short-read WGS data in SAM/BAM format were eligible for inclusion in the SV-MeCa workflow. We chose the established tools BreakDancer [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>], Delly [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>], LUMPY [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>], Manta [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>], Pindel [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>], and TARDIS [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>], as well as INSurveyor, which was specifically developed for calling insertions [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. We merged individual SV calls of identical type into consensus calls using the merge utility of SURVIVOR v1.0.7 [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. A short description of each tool, settings for the SV calling procedure and running SURVIVOR, are given in Supplementary Methods.</p>
<p id="Par15">Finally, for XGBoost classifier training, we compared SURVIVOR’s output against the GIAB HG002 benchmark VCF using the Truvari v3.5.0 bench utility [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>] with default parameters, but restricted to autosomes, specifically autosomal regions specified in GIAB Tier v0.6 BED, and setting –pctseq to 0 to avoid sequence comparisons.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Model features</h3>
<p id="Par16">For each SV type, we only considered features originating from indidividual VCF output files, if the corresponding SV caller achieved values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/5b5cb945226f/d33e478.gif" loading="lazy" id="d33e478" alt="Inline graphic"></span> based on the 35x GIAB HG002 reference (Table <a href="#Tab1" class="usa-link">1</a>), with</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/79dd1048405d/d33e487.gif" loading="lazy" id="d33e487" alt="graphic file with name d33e487.gif"></td>
<td class="label">1</td>
</tr></table>
<p>and #TP denoting the number of TP, #FP the number of false positive, and #FN of false negative predictions. Therefore, we used quality metrics retrieved from the VCF output of BreakDancer, Delly, LUMPY, Manta, Pindel, and TARDIS only for the deletion-specific model, and of Delly, Manta, Pindel, TARDIS, and INSurVeyor only for the insertion-specific model.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Performance of standalone structural variant callers on 35x HG002</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">#TP</th>
<th align="left" colspan="1" rowspan="1">#FP</th>
<th align="left" colspan="1" rowspan="1">#FN</th>
<th align="left" colspan="1" rowspan="1">Precision</th>
<th align="left" colspan="1" rowspan="1">Recall</th>
<th align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e657.gif" loading="lazy" id="d33e657" alt="Inline graphic"></span></th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="2" colspan="1">BreakDancer</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="left" colspan="1" rowspan="1">902</td>
<td align="left" colspan="1" rowspan="1">581</td>
<td align="left" colspan="1" rowspan="1">3149</td>
<td align="left" colspan="1" rowspan="1">0.61</td>
<td align="left" colspan="1" rowspan="1">0.22</td>
<td align="left" colspan="1" rowspan="1">0.33</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INS</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">5211</td>
<td align="left" colspan="1" rowspan="1">0.22</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/d526c0574251/d33e693.gif" loading="lazy" id="d33e693" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/d526c0574251/d33e700.gif" loading="lazy" id="d33e700" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">Delly</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="left" colspan="1" rowspan="1">2251</td>
<td align="left" colspan="1" rowspan="1">402</td>
<td align="left" colspan="1" rowspan="1">1800</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
<td align="left" colspan="1" rowspan="1">0.56</td>
<td align="left" colspan="1" rowspan="1">0.67</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INS</td>
<td align="left" colspan="1" rowspan="1">352</td>
<td align="left" colspan="1" rowspan="1">45</td>
<td align="left" colspan="1" rowspan="1">4861</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.07</td>
<td align="left" colspan="1" rowspan="1">0.13</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INSurVeyor</td>
<td align="left" colspan="1" rowspan="1">INS</td>
<td align="left" colspan="1" rowspan="1">1800</td>
<td align="left" colspan="1" rowspan="1">131</td>
<td align="left" colspan="1" rowspan="1">3413</td>
<td align="left" colspan="1" rowspan="1">0.93</td>
<td align="left" colspan="1" rowspan="1">0.35</td>
<td align="left" colspan="1" rowspan="1">0.50</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">LUMPY</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="left" colspan="1" rowspan="1">2055</td>
<td align="left" colspan="1" rowspan="1">132</td>
<td align="left" colspan="1" rowspan="1">1996</td>
<td align="left" colspan="1" rowspan="1">0.94</td>
<td align="left" colspan="1" rowspan="1">0.51</td>
<td align="left" colspan="1" rowspan="1">0.66</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INS</td>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">5191</td>
<td align="left" colspan="1" rowspan="1">0.59</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/d526c0574251/d33e784.gif" loading="lazy" id="d33e784" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/d526c0574251/d33e791.gif" loading="lazy" id="d33e791" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">Manta</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="left" colspan="1" rowspan="1">2649</td>
<td align="left" colspan="1" rowspan="1">97</td>
<td align="left" colspan="1" rowspan="1">1402</td>
<td align="left" colspan="1" rowspan="1">0.96</td>
<td align="left" colspan="1" rowspan="1">0.65</td>
<td align="left" colspan="1" rowspan="1">0.78</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INS</td>
<td align="left" colspan="1" rowspan="1">571</td>
<td align="left" colspan="1" rowspan="1">55</td>
<td align="left" colspan="1" rowspan="1">4642</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">0.11</td>
<td align="left" colspan="1" rowspan="1">0.20</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">Pindel</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="left" colspan="1" rowspan="1">2617</td>
<td align="left" colspan="1" rowspan="1">504</td>
<td align="left" colspan="1" rowspan="1">1434</td>
<td align="left" colspan="1" rowspan="1">0.84</td>
<td align="left" colspan="1" rowspan="1">0.65</td>
<td align="left" colspan="1" rowspan="1">0.73</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INS</td>
<td align="left" colspan="1" rowspan="1">1061</td>
<td align="left" colspan="1" rowspan="1">668</td>
<td align="left" colspan="1" rowspan="1">4152</td>
<td align="left" colspan="1" rowspan="1">0.61</td>
<td align="left" colspan="1" rowspan="1">0.20</td>
<td align="left" colspan="1" rowspan="1">0.31</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">TARDIS</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="left" colspan="1" rowspan="1">1278</td>
<td align="left" colspan="1" rowspan="1">28</td>
<td align="left" colspan="1" rowspan="1">2773</td>
<td align="left" colspan="1" rowspan="1">0.98</td>
<td align="left" colspan="1" rowspan="1">0.32</td>
<td align="left" colspan="1" rowspan="1">0.48</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INS</td>
<td align="left" colspan="1" rowspan="1">414</td>
<td align="left" colspan="1" rowspan="1">471</td>
<td align="left" colspan="1" rowspan="1">4799</td>
<td align="left" colspan="1" rowspan="1">0.47</td>
<td align="left" colspan="1" rowspan="1">0.08</td>
<td align="left" colspan="1" rowspan="1">0.14</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">Union SV calls</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="left" colspan="1" rowspan="1">3207</td>
<td align="left" colspan="1" rowspan="1">967</td>
<td align="left" colspan="1" rowspan="1">844</td>
<td align="left" colspan="1" rowspan="1">0.77</td>
<td align="left" colspan="1" rowspan="1">0.79</td>
<td align="left" colspan="1" rowspan="1">0.78</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INS</td>
<td align="left" colspan="1" rowspan="1">2518</td>
<td align="left" colspan="1" rowspan="1">1254</td>
<td align="left" colspan="1" rowspan="1">2695</td>
<td align="left" colspan="1" rowspan="1">0.67</td>
<td align="left" colspan="1" rowspan="1">0.48</td>
<td align="left" colspan="1" rowspan="1">0.56</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p36"><p>Summary of numbers of true positive (#TP), false positive (#FP), and false negative (#FN) structural variant (SV) calls, and resulting precision, recall and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e928.gif" loading="lazy" id="d33e928" alt="Inline graphic"></span> value (Eq. <a href="#Equ1" class="usa-link">1</a>), given the Genome in a Bottle (GIAB) 35x HG002 reference data set per standalone caller and for the (from SURVIVOR consensus derived) union of all calls, subdivided by deletions (DEL) and insertions including duplications (INS). Note that INSurVeyor can only call insertions, and for the evaluation of Delly and INSurVeyor as standalone approaches, only high confidence calls were considered (see Supplementary Methods)</p></div></div></section><p id="Par17">We provide details on extraction and transformation of features that are retrieved from SURVIVOR and individual VCF output files in Supplementary Methods, and a summary of all covariates considered in model development in Supplementary Table S1.</p>
<p id="Par18">The selection of features to be incorporated into the XGBoost classifiers aimed to provide a model applicable to input data within a broad range in terms of read lenghts and observed sequencing coverages.</p>
<p id="Par19">To achieve this, we normalized the quality metrics that refer to (local) coverage or counts of supporting reads by observed mean coverage (Supplementary Table S1). Further, we also normalized the average edit distances of reads supporting insertion calls (so-called stable reads), as reported by INSurVeyor, by read length.</p>
<p id="Par20">To assess the dependence of reported quality scores on mean coverage, we analyzed corresponding values from concordant calls based on the 52x and the 35x 250bp HG002 sample (see Supplementary Methods, Supplementary Figures S1 and S2 for details).</p>
<p id="Par21">Finally, we applied a correlation-based feature selection to determine the initial set of covariates to be considered for the SV-MeCa XGBoost classifier training (Supplementary Methods).</p>
<p id="Par22">This filtering strategy resulted in a set of 35 covariates as initial input for XGBoost training of the deletion-specific SV-MeCa model, namely 3 derived from BreakDancer’s VCF output, 10 from Delly, 7 from LUMPY, 7 from Manta, 3 from PINDEL, and 4 from TARDIS. To train the insertion-specific SV-MeCa model, 36 features were initially included, 6 of which were specific to Delly, 18 to INSurVeyor, 6 to Manta, 3 to Pindel, and 2 to TARDIS (Supplementary Table S1).</p></section><section id="Sec7"><h3 class="pmc_sec_title">XGBoost classifier training</h3>
<p id="Par23">For each SV type, two classifier models were trained on autosomal 35x GIAB HG002 data and evaluated, namely the final SV-MeCa XGBoost model, hereafter referred to as full model (FM), including the final subset of covariates to consider, and a basic model (BM) incorporating SV length and binary encoding of the presence of corresponding SV calls per standalone caller exclusively, to demonstrate the benefit of incorporating VCF-derived, SV call-specific quality metrics.</p>
<p id="Par24">Python implementation of XGBoost Classifier,<a href="#Fn3" class="usa-link">3</a> an ensemble learner of decision trees, was chosen as supervised learning approach and subsequent predictor due to its scalability, robustness and handling of sparse data [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. The algorithm works by iteratively fitting decision trees on the residuals of the previous tree, and adjusting the weights of the misclassified samples to prioritize their correct classification in subsequent trees. For the final prediction, the results of individual trees in the entire ensemble are combined.</p>
<p id="Par26">We employed the exact greedy method with default gbtree booster for individual tree construction, and according to the aim of generating a binary classifier, we used binary logistic regression as objective. Details on hyperparameter tuning are given in Supplementary Methods.</p>
<p id="Par27">We evaluated model performances based on precision <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/02c77db4f130/d33e526.gif" loading="lazy" id="d33e526" alt="Inline graphic"></span>, recall <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/0efb957d44db/d33e532.gif" loading="lazy" id="d33e532" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e538.gif" loading="lazy" id="d33e538" alt="Inline graphic"></span> (Eq. <a href="#Equ1" class="usa-link">1</a>). SV calls were predicted as positive, if the classifier’s prediction probability was <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/24c50dfb2a52/d33e547.gif" loading="lazy" id="d33e547" alt="Inline graphic"></span>0.5. For calculating #FN, also those SVs were included that were not called by any of the standalone callers applied.</p>
<p id="Par28">If covariates in trained FMs showed zero feature importance (due to the feature_importances_ attribute), which is defined as the relative contribution of the corresponding feature to the model (gain) per default, we repeated hyperparameter tuning and model training with the corresponding covariates discarded.</p></section><section id="Sec8"><h3 class="pmc_sec_title">SV-MeCa Implementation</h3>
<p id="Par29">SV-MeCa is implemented as a Nextflow v23.10 [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>] pipeline, and uses a conda v23.1.0 environment to ensure consistent package management. It is available as a Docker container, ensuring streamlined deployment, portability, and reproducibility.</p>
<p id="Par30">SV-MeCa expects BAM or VCF files as input. Given BAM files, SV-MeCa employs the CollectWGSMetrics utility of GATK v4.3 to assess read coverages. Read lengths are retrieved from BreakDancer’s CFG output. If VCF files are given as input, mean coverage and read length are required as additional arguments.</p>
<p id="Par31">SV-MeCa provides one VCF per executed SV caller and a final consensus VCF as output. In the consensus VCF, prediction probabilities, i.e., final ranking scores, are encoded as QUAL values. Furthermore, to enable straightforward lookup of originating calls in standalone VCFs, all identifiers contributing to a joined call are listed in the final consensus VCF.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Evaluation</h3>
<p id="Par32">We evaluated the performance of SV-MeCa and four other meta-caller approaches for SV calling, namely ConsensuSV [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>], MetaSV [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>], Parliament2 [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>], and recently published VISTA [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>], using the samples HG00514, HG00733, and NA19240 from HGSVC2, considering precision, recall and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e591.gif" loading="lazy" id="d33e591" alt="Inline graphic"></span> score. Details on execution of the meta-caller approaches can be found in Supplementary Methods. We were unable to resolve the dependencies of the requirements for running FusorSV [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. Thus, FusorSV could not be included in our evaluation. Further, the VISTA framework provides only pre-trained models and corresponding scripts for deletions, and hence, was only included in our evaluation for deletions.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Standalone structural variant caller results</h3>
<p id="Par33">From the GIAB HG002 reference, 9264 SVs (4051 deletions and 5213 insertions) passed all preprocessing filters and were further considered in SV-MeCa model development and evaluation. Considering the 35x HG002 sample, 20.83% (844/4051) of the deletions and 51.69% (2695/5213) of the insertions were not included in the SURVIVOR-generated union of the results of all standalone SV callers under investigation (Table <a href="#Tab1" class="usa-link">1</a>). All SV callers performed noticeably better for deletions than for insertions, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e610.gif" loading="lazy" id="d33e610" alt="Inline graphic"></span> values for deletions ranging from 0.33 (BreakDancer) to 0.78 (Manta) and for insertions from &lt;0.01 (BreakDancer) to 0.31 (Pindel), respectively 0.50 for the insertion-only caller INSurVeyor.</p>
<p id="Par34">Considering the 52x HG002 sample, 18.04% (731/4051) of the deletions and 44.91% (2341/5213) of the insertions were not included in the SURVIVOR-generated union of the results of all standalone SV callers under investigation (Supplementary Table S2). However, besides #TP, also the amounts of false positive calls in the SURVIVOR union increased with the increase in sequencing coverage from 35x to 52x, namely from 23.16% (967/4174) to 50.11% (3335/6655) for deletions and from 33.24% (1254/3772) to 56.64% (3751/6623) for insertions (Table <a href="#Tab1" class="usa-link">1</a> and Supplementary Table S2), resulting in decreased <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e621.gif" loading="lazy" id="d33e621" alt="Inline graphic"></span> values (for deletions: 0.78 versus 0.62, for insertions: 0.56 versus 0.49).</p></section><section id="Sec12"><h3 class="pmc_sec_title">Model parameters</h3>
<p id="Par35">We show hyperparameters retrieved by grid search in Supplementary Table S3. For the deletion-specific FM, a feature importance of zero was reported for 4 of the 35 covariates initially included in XGBoost training, as well as for 2 of the 36 covariates initially included in the insertion-specific FM training (Supplementary Table S1). Omitting these features did not affect the results of hyperparameter tuning (Supplementary Table S3), nor training accuracies (Supplementary Table S4).</p>
<p id="Par36">Tables <a href="#Tab2" class="usa-link">2</a> and <a href="#Tab3" class="usa-link">3</a> present comprehensive summaries of the features considered for each standalone SV caller in the final SV-MeCa approach per SV type.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Densed summary of quality metrics considered by the deletion-specific SV-MeCa XGBoost classifier</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">BreakDancer</th>
<th align="left" colspan="1" rowspan="1">Delly</th>
<th align="left" colspan="1" rowspan="1">LUMPY</th>
<th align="left" colspan="1" rowspan="1">Manta</th>
<th align="left" colspan="1" rowspan="1">Pindel</th>
<th align="left" colspan="1" rowspan="1">TARDIS</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">No. of supporting read pairs</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e983.gif" loading="lazy" id="d33e983" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e992.gif" loading="lazy" id="d33e992" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1000.gif" loading="lazy" id="d33e1000" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Microhomology length</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1011.gif" loading="lazy" id="d33e1011" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1019.gif" loading="lazy" id="d33e1019" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1026.gif" loading="lazy" id="d33e1026" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Quality assigned (QUAL, CNVL)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1045.gif" loading="lazy" id="d33e1045" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1053.gif" loading="lazy" id="d33e1053" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1061.gif" loading="lazy" id="d33e1061" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Strand bias</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1071.gif" loading="lazy" id="d33e1071" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1079.gif" loading="lazy" id="d33e1079" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Genotype quality</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1093.gif" loading="lazy" id="d33e1093" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1100.gif" loading="lazy" id="d33e1100" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Local read depth</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1114.gif" loading="lazy" id="d33e1114" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1123.gif" loading="lazy" id="d33e1123" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">No. of supporting SRs</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1135.gif" loading="lazy" id="d33e1135" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1145.gif" loading="lazy" id="d33e1145" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Variant fraction / Genotype</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1157.gif" loading="lazy" id="d33e1157" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1165.gif" loading="lazy" id="d33e1165" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SR consensus alignment quality</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1177.gif" loading="lazy" id="d33e1177" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Consensus sequence entropy</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1192.gif" loading="lazy" id="d33e1192" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PRECISE vs IMPRECISE</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1213.gif" loading="lazy" id="d33e1213" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p40"><p>Considered metrics per standalone structural variant caller for deletions. A detailed listing of the overall 31 features of the deletion-specific model is given in Supplementary Table S1. SR: Split read</p></div></div></section><section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Densed summary of quality metrics considered by the insertion-specific SV-MeCa XGBoost classifier</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">Delly</th>
<th align="left" colspan="1" rowspan="1">INSurVeyor</th>
<th align="left" colspan="1" rowspan="1">Manta</th>
<th align="left" colspan="1" rowspan="1">Pindel</th>
<th align="left" colspan="1" rowspan="1">TARDIS</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Microhomology length</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1255.gif" loading="lazy" id="d33e1255" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1262.gif" loading="lazy" id="d33e1262" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1269.gif" loading="lazy" id="d33e1269" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">No. of supporting SRs</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1281.gif" loading="lazy" id="d33e1281" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1288.gif" loading="lazy" id="d33e1288" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1296.gif" loading="lazy" id="d33e1296" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Quality assigned (QUAL, CNVL)</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1313.gif" loading="lazy" id="d33e1313" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1323.gif" loading="lazy" id="d33e1323" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">No. of supporting read pairs</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1334.gif" loading="lazy" id="d33e1334" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1341.gif" loading="lazy" id="d33e1341" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PRECISE vs IMPRECISE</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1359.gif" loading="lazy" id="d33e1359" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1366.gif" loading="lazy" id="d33e1366" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Local read depth</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1379.gif" loading="lazy" id="d33e1379" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1387.gif" loading="lazy" id="d33e1387" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SR consensus alignment quality</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1398.gif" loading="lazy" id="d33e1398" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Consensus sequence entropy</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1412.gif" loading="lazy" id="d33e1412" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Genotype quality</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1428.gif" loading="lazy" id="d33e1428" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Variant fraction / Genotype</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/dcd0eadbe23f/d33e1444.gif" loading="lazy" id="d33e1444" alt="Inline graphic"></span></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p42"><p>Considered metrics per standalone structural variant caller for insertions. A detailed listing of the overall 34 features of the insertion-specific model is given in Supplementary Table S1. Reported strand biases were not considered by the model. SR: Split read</p></div></div></section><p id="Par37">Within the 31 covariates of the deletion-specific FM, the (normalized) quality values QUAL from Manta showed by far the highest feature importance, i.e., relative contribution to the model, directly followed by (normalized) counts of paired reads provided by Delly, variant fraction estimates from Pindel, and allele balance reported by LUMPY (Supplementary Figure S2). The covariate with the highest contribution provided by TARDIS, namely reported quality CNVL, was ranked seventh. The SV length and all features retrieved from BreakDancer were listed within the eight features with lowest feature importances.</p>
<p id="Par38">Of the 34 covariates of the insertion-specific FM, INSurVeyor provided the two features with highest feature importances, directly followed by Manta’s (normalized) count of supporting paired reads, TARDIS’ (normalized) count of supporting split reads, and Pindel’s local coverage estimate (Supplementary Figure S3). The feature importances of the four covariates provided by Delly were consistently below the one of SV length, which was ranked 12th.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Model evaluation</h3>
<p id="Par40">The number of reference deletions and insertions (including duplications) per HGSVC2 benchmark sample (Table <a href="#Tab4" class="usa-link">4</a>) was more than double compared to the HG002 GIAB reference (Table <a href="#Tab1" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Performance of SURVIVOR union, according to curated, GRCh38-based structural variant references</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1"></th>
<th align="left" rowspan="2" colspan="1"></th>
<th align="left" rowspan="2" colspan="1">Reference</th>
<th align="left" colspan="4" rowspan="1">SURVIVOR union</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">#TP</th>
<th align="left" colspan="1" rowspan="1">#FP</th>
<th align="left" colspan="1" rowspan="1">#FN</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e1496.gif" loading="lazy" id="d33e1496" alt="Inline graphic"></span>
</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="2" colspan="1">HG002-CMRG</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="left" colspan="1" rowspan="1">98</td>
<td align="left" colspan="1" rowspan="1">45</td>
<td align="left" colspan="1" rowspan="1">68</td>
<td align="left" colspan="1" rowspan="1">53</td>
<td align="left" colspan="1" rowspan="1">0.43</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INS</td>
<td align="left" colspan="1" rowspan="1">118</td>
<td align="left" colspan="1" rowspan="1">29</td>
<td align="left" colspan="1" rowspan="1">46</td>
<td align="left" colspan="1" rowspan="1">89</td>
<td align="left" colspan="1" rowspan="1">0.30</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">HG00514</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="left" colspan="1" rowspan="1">8583</td>
<td align="left" colspan="1" rowspan="1">4938</td>
<td align="left" colspan="1" rowspan="1">11460</td>
<td align="left" colspan="1" rowspan="1">3645</td>
<td align="left" colspan="1" rowspan="1">0.40</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INS</td>
<td align="left" colspan="1" rowspan="1">13441</td>
<td align="left" colspan="1" rowspan="1">5509</td>
<td align="left" colspan="1" rowspan="1">18565</td>
<td align="left" colspan="1" rowspan="1">7932</td>
<td align="left" colspan="1" rowspan="1">0.29</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">HG00733</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="left" colspan="1" rowspan="1">8771</td>
<td align="left" colspan="1" rowspan="1">5037</td>
<td align="left" colspan="1" rowspan="1">11958</td>
<td align="left" colspan="1" rowspan="1">3734</td>
<td align="left" colspan="1" rowspan="1">0.39</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INS</td>
<td align="left" colspan="1" rowspan="1">13603</td>
<td align="left" colspan="1" rowspan="1">5633</td>
<td align="left" colspan="1" rowspan="1">22277</td>
<td align="left" colspan="1" rowspan="1">7970</td>
<td align="left" colspan="1" rowspan="1">0.27</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">NA19240</td>
<td align="left" colspan="1" rowspan="1">DEL</td>
<td align="left" colspan="1" rowspan="1">10274</td>
<td align="left" colspan="1" rowspan="1">5832</td>
<td align="left" colspan="1" rowspan="1">12434</td>
<td align="left" colspan="1" rowspan="1">4442</td>
<td align="left" colspan="1" rowspan="1">0.41</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">INS</td>
<td align="left" colspan="1" rowspan="1">15451</td>
<td align="left" colspan="1" rowspan="1">6448</td>
<td align="left" colspan="1" rowspan="1">21430</td>
<td align="left" colspan="1" rowspan="1">9003</td>
<td align="left" colspan="1" rowspan="1">0.30</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p47"><p>Summary of counts of true positive (#TP), false positive (#FP), and false negative (#FN) structural variant calls, as well as resulting <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e1618.gif" loading="lazy" id="d33e1618" alt="Inline graphic"></span> values, according to GRCh38-based structural variant reference in challenging medically relevant genes of sample HG002 (HG002-CMRG) and HGSVC2 reference in 1000 Genomes samples HG00514, HG00733 and NA19240. Values of #TP, #FP and #FN for deletions (DEL) refer to the SURVIVOR-derived union of SV calls retrieved from BreakDancer, Delly, LUMPY, Manta, Pindel, and TARDIS. Corresponding values for insertions (INS) refer to the SURVIVOR-derived union of SV calls retrieved from Delly, INSurVeyor, Manta, Pindel, and TARDIS</p></div></div></section><p id="Par41">Values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e1635.gif" loading="lazy" id="d33e1635" alt="Inline graphic"></span> achieved by the SURVIVOR union (Table <a href="#Tab4" class="usa-link">4</a>) of deletion calls retrieved from BreakDancer, Delly, LUMPY, Manta, Pindel, and TARDIS ranged from 0.39 to 0.43. Values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e1644.gif" loading="lazy" id="d33e1644" alt="Inline graphic"></span> achieved by the SURVIVOR union of insertion (including duplication) calls retrieved from Delly, INSurVeyor, Manta, Pindel, and TARDIS ranged from 0.27 to 0.30. Numbers of exclusive calls, i.e., TP calls that were uniquely identified by a single tool, are shown for HGSVC2 reference samples in Supplementary Figure S5.</p>
<p id="Par42">The BMs and the final FMs included in SV-MeCa were applied to the deletion and insertion calls in the SURVIVOR union per HGSVC2 sample in order to evaluate XGBoost model performances. The BMs achieved model accuracies between 0.71 and 0.72 (AUC = 0.90 – 0.91) for deletions and between 0.63 and 0.65 (AUC = 0.85 – 0.87) for insertions (Supplementary Table S5). In contrast, the FMs incorporated into SV-MeCa achieved accuracies ranging from 0.79 to 0.81 (AUC = 0.93 each) for deletions and from 0.76 to 0.80 (AUC = 0.88 – 0.89) for insertions. Corresponding ROC curves are shown in Fig. <a href="#Fig2" class="usa-link">2</a>.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366149_12859_2025_6246_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/e91c76f0c509/12859_2025_6246_Fig2_HTML.jpg" loading="lazy" id="MO2" height="228" width="706" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Receiver operating characteristic (ROC) curves of the basic and full (i.e. SV-MeCa) models (BM/FM). Curves for deletions (DEL) are shown in blue, curves for insertions including duplications (INS) are shown in red</p></figcaption></figure><p id="Par43">Overall performance metrics of the BMs and the FMs under inclusion of the entire set of false negative calls, i.e., reference SVs that were not detected by any of the incorporated standalone callers, are visualized in Fig. <a href="#Fig3" class="usa-link">3</a> and listed in Supplementary Table S6. SV-MeCa’s FMs showed slightly decreased recall, but outperformed the BMs with regard to precision and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e1667.gif" loading="lazy" id="d33e1667" alt="Inline graphic"></span>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366149_12859_2025_6246_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/59b47b037238/12859_2025_6246_Fig3_HTML.jpg" loading="lazy" id="MO3" height="530" width="706" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance comparison for SURVIVOR union, basic and full (SV-MeCa) model. Precision, recall and F1 value (Eq. <a href="#Equ1" class="usa-link">1</a>) for reference samples HG00514, HG00733 and NA19240 under consideration of the SURVIVOR union (SU), the basic model (BM) and the final full model (FM) included in the SV-MeCa approach for deletions (shown in blue) and insertions including duplications (shown in red)</p></figcaption></figure></section><section id="Sec14"><h3 class="pmc_sec_title">Comparison to standalone structural variant callers</h3>
<p id="Par44">We compared the performance of SV-MeCa with its incorporated standalone callers using two datasets, namely SVs in challenging medically relevant genes in HG002, HG002-CMRG, and SVs in autosomes of samples HG00514, HG00733, and NA19240 (Fig. <a href="#Fig4" class="usa-link">4</a>, Supplementary Table S7). For deletions, Manta, LUMPY, and TARDIS achieved consistently higher precision values than SV-MeCa, as well as Delly for HGSVC2 reference samples. For insertions, SV-MeCa was outperformed by INSurVeyor und Manta, and for HG002-CMRG also by Delly. With respect to recall, SV-MeCa consistently outperformed all standalone caller approaches under consideration. Regarding <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e1691.gif" loading="lazy" id="d33e1691" alt="Inline graphic"></span> scores, Manta achieved slightly higher values than SV-MeCa for deletions in the HGSVC2 reference samples (0.60 on average versus 0.58 on average), but was outperformed by SV-MeCa for deletions in the HG002-CMRG reference (0.55 versus 0.58). For insertions, SV-MeCa consistently achieved higher <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e1697.gif" loading="lazy" id="d33e1697" alt="Inline graphic"></span> values than all standalone caller approaches under consideration.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366149_12859_2025_6246_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/442db0d10543/12859_2025_6246_Fig4_HTML.jpg" loading="lazy" id="MO4" height="473" width="707" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance comparison for standalone callers and SV-MeCa. Precision, recall and F1 value (Eq. <a href="#Equ1" class="usa-link">1</a>) for GRCh38-based references, namely structural variants in challenging medically relevant genes (HG002-CMRG) and HGSVC2 samples HG00514, HG00733 and NA19240, stratified by deletions (shown in blue) and insertions including duplications (shown in red)</p></figcaption></figure></section><section id="Sec15"><h3 class="pmc_sec_title">Ranking of consensus calls</h3>
<p id="Par45">A key feature of SV-MeCa is the provision of quality values, which are retrieved from the XGBoost predicition probabilites and can be used to sort SV calls according to evidence to represent TPs. To evaluate the performance of resulting rankings, we considered SV-MeCa calls for samples HG00514, HG00733, and NA19240 per decile of assigned XGBoost prediction probabilities. Between 5550 and 5939 deletion calls per sample were assigned a probability &lt;0.1, corresponding to 32.51<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/99456ba7ce5f/d33e1718.gif" loading="lazy" id="d33e1718" alt="Inline graphic"></span>34.32% of all calls (Fig. <a href="#Fig5" class="usa-link">5</a>, Supplementary Table S8). The proportion of TP calls was 0.02 for each sample. The second most frequently assigned probability decile was the highest one with between 4038 and 4817 calls, corresponding to 23.95<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/99456ba7ce5f/d33e1727.gif" loading="lazy" id="d33e1727" alt="Inline graphic"></span>26.37% of all deletion calls. In this subgroup, the proportion of TP calls was 0.86 for each sample.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366149_12859_2025_6246_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/51be7f8efbee/12859_2025_6246_Fig5_HTML.jpg" loading="lazy" id="MO5" height="391" width="706" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Numbers of calls per decile of SV-MeCa XGBoost prediction probability. Upper: Number of deletion calls. Lower: Number of insertion (including duplication) calls. Proportion of true positive (TP) calls is indicated by filled areas respectively</p></figcaption></figure><p id="Par46">The lowest probability decile was also the most frequently assigned regarding insertions, with 5000 to 7185 calls per sample, corresponding to 20.77<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/99456ba7ce5f/d33e1735.gif" loading="lazy" id="d33e1735" alt="Inline graphic"></span>25.74% of all calls (Fig. <a href="#Fig5" class="usa-link">5</a>, Supplementary Table S9). The proportion of TP calls in this subgroup ranged from 0.02 to 0.03. The highest probability decile comprised between 3557 and 4236 calls (13.01<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/99456ba7ce5f/d33e1744.gif" loading="lazy" id="d33e1744" alt="Inline graphic"></span>15.19%). In this subgroup, the proportion of TP calls ranged from 0.77 to 0.79.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Comparison to other meta-caller approaches</h3>
<p id="Par47">We compared the performance of SV-MeCa with respect to recall, precision, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e1761.gif" loading="lazy" id="d33e1761" alt="Inline graphic"></span> against ConsensuSV [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>], MetaSV [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>], Parliament2 [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>], and VISTA [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>] (deletions only) on samples HG00514, HG00733, and NA19240 (Fig. <a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366149_12859_2025_6246_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/58fa3e269f17/12859_2025_6246_Fig6_HTML.jpg" loading="lazy" id="MO6" height="551" width="708" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of performances of five meta-caller approaches, including SV-MeCa. Resulting values of precision, recall and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e1809.gif" loading="lazy" id="d33e1809" alt="Inline graphic"></span> value (Eq. <a href="#Equ1" class="usa-link">1</a>) for reference samples HG00514, HG00733 and NA19240 using five different meta-caller approaches for structural variant calling: ConsensuSV, MetaSV, Parliament2, SV-MeCa, and VISTA. Results for deletions (DEL) are shown in blue, results for insertions including duplications (INS) are shown in red. For VISTA, only results for deletions were considered</p></figcaption></figure><p id="Par48">ConsensuSV consistently achieved the highest precision, with 0.96<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/99456ba7ce5f/d33e1785.gif" loading="lazy" id="d33e1785" alt="Inline graphic"></span>0.97 for deletions and 0.73<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/99456ba7ce5f/d33e1791.gif" loading="lazy" id="d33e1791" alt="Inline graphic"></span>0.77 for insertions (Supplementary Table S10). On the other hand, MetaSV achieved the highest recall for deletions (0.55 for each sample), whereas for insertions, SV-MeCa outperformed the other meta-callers with recall 0.35 on average. Finally, based on the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e1797.gif" loading="lazy" id="d33e1797" alt="Inline graphic"></span> score, SV-MeCa was superior to all the other meta-callers for both deletions (0.58 for each sample) and insertions (0.42 on average).</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par49">We presented SV-MeCa, which is to our knowledge the first meta-caller approach for SVs that incorporates individual, VCF-retrieved quality metrics into a comprehensive model for generation of the final list of consensus calls. We demonstrated the benefit of the introduced approach by comparing the results of SV-MeCa to the results of a basic XGBoost model, considering caller agreement and SV length exclusively. Further, SV-MeCa outperformed existing SV meta-caller approaches consistently with respect to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/513f/12366149/be3792b2e763/d33e1824.gif" loading="lazy" id="d33e1824" alt="Inline graphic"></span>.</p>
<p id="Par50">Based on the prediction probabilities assigned by deletion- and insertion-specific XGBoost classifiers, SV-MeCa provides a score for each SV call that indicates the evidence of representing a TP, real-world finding. The models may also allow conclusions to be drawn about which metrics must be prioritized from VCF output of established SV callers in order to distinguish TP and artifact calls as reliably as possible. For example, information about microhomologies was consistently included for both SV types (Tables <a href="#Tab2" class="usa-link">2</a> and <a href="#Tab3" class="usa-link">3</a>), again emphasizing the role of microhomologies as prominent mediators of SV generation [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Further, Delly’s QUAL (at least indirectly as binary encoding of the PASS flag) and TARDIS’ CNVL values were consistently considered for both deletion and insertion calls, indicating the general ability of these metrics to appropriately aggregate the overall evidence for a TP SV call. A comprehensive investigation of VCF-retrieved quality metrics and their contribution to the assement of SV calls could be the subject of a future study.</p>
<p id="Par51">It is undisputed that SV calling on short-read data is principally limited. A comprehensive evaluation by the HGSVC2 consortium revealed that approximately 30% of SVs detectable in long-read sequencing data could not be identified in Illumina short-read data [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Due to the known limitations, it is often argued that meaningful SV calling can and should only be applied to so-called 3rd generation, i.e., long-read sequencing data. However, in large-scale sequencing projects, and particularly in routine clinical diagnostics, short-read sequencing still represents the standard for reasons of cost, efficiency, and expertise. Nevertheless, it should be noted that consistently more than 40% of the known SVs in our benchmark data were not identified by any of the integrated standalone SV callers, and thus, contribute the vast majority of observed #FNs, rather than those misclassified by the SV-MeCa model. However, the introduced approach allows for the easy inclusion of additional (future) SV callers for recall improvement, while controlling the rate of false positives.</p>
<p id="Par52">The development of SV callers is principally hampered by limited availability of appropriate benchmark data. All standalone SV callers included in SV-MeCa, as well as the meta-callers against which we compared its performance, were also developed and evaluated, at least in part, on the basis of data from 1000 Genomes samples. Therefore, overfitting effects can not be ruled out, besides a bias towards recurrent SVs in high confidence regions. With the rise of WGS in routine diagnostics, we will be able to broaden the data basis for SV-MeCa model training as well as its evaluation by real-world data from clinical practice. The robustness of future SV-MeCa models will benefit in particular from more variable mean coverages and read lengths in training data. The current training data from GIAB HG002 is characterized by its high quality and quite long reads (2x250), which might be uncommon in practice. The following further improvements of the SV-MeCa approach are aimed for by the developers: In addition to its extension to further SV types, such as inversions and genomic rearrangements, the objective is to assess whether the development and application of specific models for insertions and duplications will further improve SV-MeCa’s performance. Finally, we are aware that the accuracy of SV meta-callers crucially relies on the accuracy of the tools employed for SV comparison and merging. Limitations of existing tools have been reported consistently [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Therefore, we plan to evaluate if and how SV-MeCa’s performance may be further improved by employing alternative approaches to SV merging.</p></section><section id="Sec18"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par53">SV-MeCa, which is publicly available at <a href="https://github.com/ccfboc-bioinformatics/SV-MeCa" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ccfboc-bioinformatics/SV-MeCa</a> as a ready-to-use Docker container, outperforms existing SV meta-caller approaches by taking variant-specific quality metrics into account. These metrics are retrieved from individual VCF outputs of seven incorporated SV callers. The XGBoost prediction probabilities serving as scores in SV-MeCa’s VCF output, provide a suitable filter criterion to adjust the final consensus of SV calls straightforwardly and continuously to user needs in terms of sensitivity and precision.</p></section><section id="Sec20"><h2 class="pmc_sec_title">Additional file</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366149/bin/12859_2025_6246_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 1.</a><sup> (686.4KB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the ITCC (IT Center University of Cologne) for providing compute resources on the DFG-funded HPC (High Performance Computing) system RAMSES (Research Accelerator for Modeling and Simulation with Enhanced Security) as well as support (DFG funding number: INST 216/512-1 FUGG).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: RF, JB, CE, and SM; Data curation: RF, AS, and CE; Formal analysis: RF and CE; Investigation: RF and CE; Methodology: RF, AS, JB, SM, and CE; Software: RF and CE; Supervision: JB, CE, and SM; Validation: RF and CE; Visualization: AK and CE; Writing – original draft: CE; Writing – review &amp; editing: All authors. All authors have read and agreed to the final version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open Access funding enabled and organized by Projekt DEAL. This work was supported by the German Federal Ministry of Health within the genomDE Initiative, by the Ministry of Culture and Science of the State of North Rhine-Westphalia (MODS project funded from the programme ”Profilbildung 2020” [grant no. PROFILNRW-2020-107-A], West German Genome Center [76.06.04-000126]), by the German Federal Ministry of Education and Research [031L0184A], and by the German Research Foundation (DFG), Project ”West German Genome Center” [project 407493903].</p></section><section id="notes3"><h2 class="pmc_sec_title"> Data availability</h2>
<p>All data analyzed during this study are publicly available. Raw sequencing data of HG002 (NA24385) in FASTQ format was obtained from <a href="https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/data/AshkenazimTrio/HG002_NA24385_son/NIST_Illumina_2x250bps/reads/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/data/AshkenazimTrio/HG002_NA24385_son/NIST_Illumina_2x250bps/reads/</a>. The GIAB SV ground truth of HG002, HG002_SVs_Tier1_v0.6.vcf.gz, was retrieved from <a href="https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/NIST_SV_v0.6/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/NIST_SV_v0.6/</a>, and additionally, the substituted SVs in challenging medically relevant genes, HG002_GRCh37_CMRG_SV_v1.00.vcf.gz, from <a href="https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/CMRG_v1.00/GRCh37/StructuralVariant/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/CMRG_v1.00/GRCh37/StructuralVariant/</a>. The corresponding specification of high confidence regions for SV benchmarking, HG002_SVs_Tier2_v0.6.bed, was downloaded from <a href="https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/NIST_SV_v0.6/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/NIST_SV_v0.6/</a>. Raw 150bp paired-end WGS data of HG002, downsampled to 30x, was obtained from <a href="https://github.com/human-pangenomics/HG002_Data_Freeze_v1.0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/human-pangenomics/HG002_Data_Freeze_v1.0</a>, and hg38-based SV benchmark in challenging medically relevant genes from <a href="https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/CMRG_v1.00/GRCh38/StructuralVariant/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/CMRG_v1.00/GRCh38/StructuralVariant/</a>. Raw sequencing data from samples HG00514, HG00733, and NA19240 in CRAM format was retrieved from <a href="https://ftp.sra.ebi.ac.uk/vol1/run/ERR398/ERR3988781/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp.sra.ebi.ac.uk/vol1/run/ERR398/ERR3988781/</a>, <a href="https://ftp.sra.ebi.ac.uk/vol1/run/ERR398/ERR3988823/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp.sra.ebi.ac.uk/vol1/run/ERR398/ERR3988823/</a>, and <a href="https://ftp.sra.ebi.ac.uk/vol1/run/ERR398/ERR3989410/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp.sra.ebi.ac.uk/vol1/run/ERR398/ERR3989410/</a>. Ground truth SV data for samples HG00514, HG00733, and NA19240, was downloaded from <a href="https://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/release/v2.0/integrated_callset/variants_freeze4_sv_insdel_alt.vcf.gz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/release/v2.0/integrated_callset/variants_freeze4_sv_insdel_alt.vcf.gz</a>. Data used for XGBoost model training and evaluation is available at <a href="https://github.com/ccfboc-bioinformatics/SV-MeCa_data" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ccfboc-bioinformatics/SV-MeCa_data</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par54">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par55">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par56">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="Fn1">
<sup>1</sup><p class="display-inline" id="Par11">https://github.com/human-pangenomics/HG002_Data_Freeze_v1.0.</p>
</div>
<div class="fn p" id="Fn2">
<sup>2</sup><p class="display-inline" id="Par13">www.internationalgenome.org.</p>
</div>
<div class="fn p" id="Fn3">
<sup>3</sup><p class="display-inline" id="Par25">https://pypi.org/project/xgboost/</p>
</div>
<div class="fn p" id="fn4">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Corinna Ernst, Email: corinna.ernst@uk-koeln.de.</p>
<p>Susanne Motameny, Email: susanne.motameny@uni-koeln.de.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Alkan C, Coe BP, Eichler EE. Genome structural variation discovery and genotyping. Nat Rev Genet. 2011;12(5):363–76.
</cite> [<a href="https://doi.org/10.1038/nrg2958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4108431/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21358748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alkan%20C,%20Coe%20BP,%20Eichler%20EE.%20Genome%20structural%20variation%20discovery%20and%20genotyping.%20Nat%20Rev%20Genet.%202011;12(5):363%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Mahmoud M, Gobet N, Cruz-Dávalos DI, Mounier N, Dessimoz C, Sedlazeck FJ. Structural variant calling: the long and the short of it. Genome Biol. 2019;20:1–14.
</cite> [<a href="https://doi.org/10.1186/s13059-019-1828-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6868818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31747936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mahmoud%20M,%20Gobet%20N,%20Cruz-D%C3%A1valos%20DI,%20Mounier%20N,%20Dessimoz%20C,%20Sedlazeck%20FJ.%20Structural%20variant%20calling:%20the%20long%20and%20the%20short%20of%20it.%20Genome%20Biol.%202019;20:1%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Tattini L, D’Aurizio R, Magi A. Detection of genomic structural variants from next-generation sequencing data. Front Bioeng Biotechnol. 2015;3:92.
</cite> [<a href="https://doi.org/10.3389/fbioe.2015.00092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4479793/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26161383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tattini%20L,%20D%E2%80%99Aurizio%20R,%20Magi%20A.%20Detection%20of%20genomic%20structural%20variants%20from%20next-generation%20sequencing%20data.%20Front%20Bioeng%20Biotechnol.%202015;3:92." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Chander V, Gibbs RA, Sedlazeck FJ. Evaluation of computational genotyping of structural variation for clinical diagnoses. GigaScience. 2019;8(9):giz110.
</cite> [<a href="https://doi.org/10.1093/gigascience/giz110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6732172/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31494671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chander%20V,%20Gibbs%20RA,%20Sedlazeck%20FJ.%20Evaluation%20of%20computational%20genotyping%20of%20structural%20variation%20for%20clinical%20diagnoses.%20GigaScience.%202019;8(9):giz110." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Gabrielaite M, Torp MH, Rasmussen MS, Andreu-Sánchez S, Vieira FG, Pedersen CB, et al. A comparison of tools for copy-number variation detection in germline whole exome and whole genome sequencing data. Cancers. 2021;13(24):6283.
</cite> [<a href="https://doi.org/10.3390/cancers13246283" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8699073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34944901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gabrielaite%20M,%20Torp%20MH,%20Rasmussen%20MS,%20Andreu-S%C3%A1nchez%20S,%20Vieira%20FG,%20Pedersen%20CB,%20et%20al.%20A%20comparison%20of%20tools%20for%20copy-number%20variation%20detection%20in%20germline%20whole%20exome%20and%20whole%20genome%20sequencing%20data.%20Cancers.%202021;13(24):6283." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Kosugi S, Momozawa Y, Liu X, Terao C, Kubo M, Kamatani Y. Comprehensive evaluation of structural variation detection algorithms for whole genome sequencing. Genome Biol. 2019;20:1–18.
</cite> [<a href="https://doi.org/10.1186/s13059-019-1720-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6547561/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31159850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kosugi%20S,%20Momozawa%20Y,%20Liu%20X,%20Terao%20C,%20Kubo%20M,%20Kamatani%20Y.%20Comprehensive%20evaluation%20of%20structural%20variation%20detection%20algorithms%20for%20whole%20genome%20sequencing.%20Genome%20Biol.%202019;20:1%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Mohiyuddin M, Mu JC, Li J, Bani Asadi N, Gerstein MB, Abyzov A, et al. MetaSV: an accurate and integrative structural-variant caller for next generation sequencing. Bioinformatics. 2015;31(16):2741–4.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv204" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4528635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25861968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mohiyuddin%20M,%20Mu%20JC,%20Li%20J,%20Bani%20Asadi%20N,%20Gerstein%20MB,%20Abyzov%20A,%20et%20al.%20MetaSV:%20an%20accurate%20and%20integrative%20structural-variant%20caller%20for%20next%20generation%20sequencing.%20Bioinformatics.%202015;31(16):2741%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Ye K, Schulz MH, Long Q, Apweiler R, Ning Z. Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads. Bioinformatics. 2009;25(21):2865–71.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2781750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19561018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye%20K,%20Schulz%20MH,%20Long%20Q,%20Apweiler%20R,%20Ning%20Z.%20Pindel:%20a%20pattern%20growth%20approach%20to%20detect%20break%20points%20of%20large%20deletions%20and%20medium%20sized%20insertions%20from%20paired-end%20short%20reads.%20Bioinformatics.%202009;25(21):2865%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Zarate S, Carroll A, Mahmoud M, Krasheninina O, Jun G, Salerno WJ, et al. Parliament2: accurate structural variant calling at scale. GigaScience. 2020;9(12):giaa145.
</cite> [<a href="https://doi.org/10.1093/gigascience/giaa145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7751401/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33347570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zarate%20S,%20Carroll%20A,%20Mahmoud%20M,%20Krasheninina%20O,%20Jun%20G,%20Salerno%20WJ,%20et%20al.%20Parliament2:%20accurate%20structural%20variant%20calling%20at%20scale.%20GigaScience.%202020;9(12):giaa145." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Chiliński M, Plewczynski D. ConsensuSV-from the whole-genome sequencing data to the complete variant list. Bioinformatics. 2022;38(24):5440–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btac709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9750118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36315072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chili%C5%84ski%20M,%20Plewczynski%20D.%20ConsensuSV-from%20the%20whole-genome%20sequencing%20data%20to%20the%20complete%20variant%20list.%20Bioinformatics.%202022;38(24):5440%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Becker T, Lee WP, Leone J, Zhu Q, Zhang C, Liu S, et al. FusorSV: an algorithm for optimally combining data from multiple structural variation detection methods. Genome Biol. 2018;19:1–14.
</cite> [<a href="https://doi.org/10.1186/s13059-018-1404-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5859555/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29559002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Becker%20T,%20Lee%20WP,%20Leone%20J,%20Zhu%20Q,%20Zhang%20C,%20Liu%20S,%20et%20al.%20FusorSV:%20an%20algorithm%20for%20optimally%20combining%20data%20from%20multiple%20structural%20variation%20detection%20methods.%20Genome%20Biol.%202018;19:1%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Sarwal V, Lee S, Yang J, Sankararaman S, Chaisson M, Eskin E, et al. VISTA: an integrated framework for structural variant discovery. Brief Bioinform. 2024;25(5):bbae462.
</cite> [<a href="https://doi.org/10.1093/bib/bbae462" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11411772/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39297879/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sarwal%20V,%20Lee%20S,%20Yang%20J,%20Sankararaman%20S,%20Chaisson%20M,%20Eskin%20E,%20et%20al.%20VISTA:%20an%20integrated%20framework%20for%20structural%20variant%20discovery.%20Brief%20Bioinform.%202024;25(5):bbae462." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Chen K, Wallis JW, McLellan MD, Larson DE, Kalicki JM, Pohl CS, et al. BreakDancer: an algorithm for high-resolution mapping of genomic structural variation. Nat Methods. 2009;6(9):677–81.
</cite> [<a href="https://doi.org/10.1038/nmeth.1363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3661775/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19668202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20K,%20Wallis%20JW,%20McLellan%20MD,%20Larson%20DE,%20Kalicki%20JM,%20Pohl%20CS,%20et%20al.%20BreakDancer:%20an%20algorithm%20for%20high-resolution%20mapping%20of%20genomic%20structural%20variation.%20Nat%20Methods.%202009;6(9):677%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Rausch T, Zichner T, Schlattl A, Stütz AM, Benes V, Korbel JO. DELLY: structural variant discovery by integrated paired-end and split-read analysis. Bioinformatics. 2012;28(18):i333–9.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts378" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3436805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22962449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rausch%20T,%20Zichner%20T,%20Schlattl%20A,%20St%C3%BCtz%20AM,%20Benes%20V,%20Korbel%20JO.%20DELLY:%20structural%20variant%20discovery%20by%20integrated%20paired-end%20and%20split-read%20analysis.%20Bioinformatics.%202012;28(18):i333%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Layer RM, Chiang C, Quinlan AR, Hall IM. LUMPY: a probabilistic framework for structural variant discovery. Genome Biol. 2014;15:1–19.</cite> [<a href="https://doi.org/10.1186/gb-2014-15-6-r84" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4197822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24970577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Layer%20RM,%20Chiang%20C,%20Quinlan%20AR,%20Hall%20IM.%20LUMPY:%20a%20probabilistic%20framework%20for%20structural%20variant%20discovery.%20Genome%20Biol.%202014;15:1%E2%80%9319." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Chen X, Schulz-Trieglaff O, Shaw R, Barnes B, Schlesinger F, Källberg M, et al. Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications. Bioinformatics. 2016;32(8):1220–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv710" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26647377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20X,%20Schulz-Trieglaff%20O,%20Shaw%20R,%20Barnes%20B,%20Schlesinger%20F,%20K%C3%A4llberg%20M,%20et%20al.%20Manta:%20rapid%20detection%20of%20structural%20variants%20and%20indels%20for%20germline%20and%20cancer%20sequencing%20applications.%20Bioinformatics.%202016;32(8):1220%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Soylev A, Kockan C, Hormozdiari F, Alkan C. Toolkit for automated and rapid discovery of structural variants. Methods. 2017;129:3–7.
</cite> [<a href="https://doi.org/10.1016/j.ymeth.2017.05.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28583483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Soylev%20A,%20Kockan%20C,%20Hormozdiari%20F,%20Alkan%20C.%20Toolkit%20for%20automated%20and%20rapid%20discovery%20of%20structural%20variants.%20Methods.%202017;129:3%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Soylev A, Le TM, Amini H, Alkan C, Hormozdiari F. Discovery of tandem and interspersed segmental duplications using high-throughput sequencing. Bioinformatics. 2019;35(20):3923–30.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz237" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6792081/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30937433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Soylev%20A,%20Le%20TM,%20Amini%20H,%20Alkan%20C,%20Hormozdiari%20F.%20Discovery%20of%20tandem%20and%20interspersed%20segmental%20duplications%20using%20high-throughput%20sequencing.%20Bioinformatics.%202019;35(20):3923%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Rajaby R, Liu DX, Au CH, Cheung YT, Lau AYT, Yang QY, et al. INSurVeyor: improving insertion calling from short read sequencing data. Nat Commun. 2023;14(1):3243.
</cite> [<a href="https://doi.org/10.1038/s41467-023-38870-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10241795/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37277343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rajaby%20R,%20Liu%20DX,%20Au%20CH,%20Cheung%20YT,%20Lau%20AYT,%20Yang%20QY,%20et%20al.%20INSurVeyor:%20improving%20insertion%20calling%20from%20short%20read%20sequencing%20data.%20Nat%20Commun.%202023;14(1):3243." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Jeffares DC, Jolly C, Hoti M, Speed D, Shaw L, Rallis C, et al. Transient structural variations have strong effects on quantitative traits and reproductive isolation in fission yeast. Nat Commun. 2017;8(1):14061.
</cite> [<a href="https://doi.org/10.1038/ncomms14061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5286201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28117401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jeffares%20DC,%20Jolly%20C,%20Hoti%20M,%20Speed%20D,%20Shaw%20L,%20Rallis%20C,%20et%20al.%20Transient%20structural%20variations%20have%20strong%20effects%20on%20quantitative%20traits%20and%20reproductive%20isolation%20in%20fission%20yeast.%20Nat%20Commun.%202017;8(1):14061." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Chen T, Guestrin C. Xgboost: A scalable tree boosting system. In: Proceedings of the 22nd acm sigkdd international conference on knowledge discovery and data mining; 2016. p. 785–794.</cite>
</li>
<li id="CR22">
<span class="label">22.</span><cite>Zook JM, Hansen NF, Olson ND, Chapman L, Mullikin JC, Xiao C, et al. A robust benchmark for detection of germline large deletions and insertions. Nat Biotechnol. 2020;38(11):1347–55.
</cite> [<a href="https://doi.org/10.1038/s41587-020-0538-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8454654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32541955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zook%20JM,%20Hansen%20NF,%20Olson%20ND,%20Chapman%20L,%20Mullikin%20JC,%20Xiao%20C,%20et%20al.%20A%20robust%20benchmark%20for%20detection%20of%20germline%20large%20deletions%20and%20insertions.%20Nat%20Biotechnol.%202020;38(11):1347%E2%80%9355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Chaisson MJ, Sanders AD, Zhao X, Malhotra A, Porubsky D, Rausch T, et al. Multi-platform discovery of haplotype-resolved structural variation in human genomes. Nat Commun. 2019;10(1):1784.
</cite> [<a href="https://doi.org/10.1038/s41467-018-08148-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6467913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30992455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chaisson%20MJ,%20Sanders%20AD,%20Zhao%20X,%20Malhotra%20A,%20Porubsky%20D,%20Rausch%20T,%20et%20al.%20Multi-platform%20discovery%20of%20haplotype-resolved%20structural%20variation%20in%20human%20genomes.%20Nat%20Commun.%202019;10(1):1784." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Ebert P, Audano PA, Zhu Q, Rodriguez-Martin B, Porubsky D, Bonder MJ, et al. Haplotype-resolved diverse human genomes and integrated analysis of structural variation. Science. 2021;372(6537):eabf7117.
</cite> [<a href="https://doi.org/10.1126/science.abf7117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8026704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33632895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ebert%20P,%20Audano%20PA,%20Zhu%20Q,%20Rodriguez-Martin%20B,%20Porubsky%20D,%20Bonder%20MJ,%20et%20al.%20Haplotype-resolved%20diverse%20human%20genomes%20and%20integrated%20analysis%20of%20structural%20variation.%20Science.%202021;372(6537):eabf7117." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Wagner J, Olson ND, Harris L, McDaniel J, Cheng H, Fungtammasan A, et al. Curated variation benchmarks for challenging medically relevant autosomal genes. Nat Biotechnol. 2022;40(5):672–80.
</cite> [<a href="https://doi.org/10.1038/s41587-021-01158-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9117392/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35132260/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wagner%20J,%20Olson%20ND,%20Harris%20L,%20McDaniel%20J,%20Cheng%20H,%20Fungtammasan%20A,%20et%20al.%20Curated%20variation%20benchmarks%20for%20challenging%20medically%20relevant%20autosomal%20genes.%20Nat%20Biotechnol.%202022;40(5):672%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>English AC, Menon VK, Gibbs RA, Metcalf GA, Sedlazeck FJ. Truvari: refined structural variant comparison preserves allelic diversity. Genome Biol. 2022;23(1):271.
</cite> [<a href="https://doi.org/10.1186/s13059-022-02840-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9793516/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36575487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?English%20AC,%20Menon%20VK,%20Gibbs%20RA,%20Metcalf%20GA,%20Sedlazeck%20FJ.%20Truvari:%20refined%20structural%20variant%20comparison%20preserves%20allelic%20diversity.%20Genome%20Biol.%202022;23(1):271." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Di Tommaso P, Chatzou M, Floden EW, Barja PP, Palumbo E, Notredame C. Nextflow enables reproducible computational workflows. Nat Biotechnol. 2017;35(4):316–9.
</cite> [<a href="https://doi.org/10.1038/nbt.3820" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28398311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Di%20Tommaso%20P,%20Chatzou%20M,%20Floden%20EW,%20Barja%20PP,%20Palumbo%20E,%20Notredame%20C.%20Nextflow%20enables%20reproducible%20computational%20workflows.%20Nat%20Biotechnol.%202017;35(4):316%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Carvalho CM, Lupski JR. Mechanisms underlying structural variant formation in genomic disorders. Nat Rev Genet. 2016;17(4):224–38.
</cite> [<a href="https://doi.org/10.1038/nrg.2015.25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4827625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26924765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carvalho%20CM,%20Lupski%20JR.%20Mechanisms%20underlying%20structural%20variant%20formation%20in%20genomic%20disorders.%20Nat%20Rev%20Genet.%202016;17(4):224%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Zheng Z, Zhu M, Zhang J, Liu X, Hou L, Liu W, et al. A sequence-aware merger of genomic structural variations at population scale. Nat Commun. 2024;15(1):960.
</cite> [<a href="https://doi.org/10.1038/s41467-024-45244-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10837428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38307885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng%20Z,%20Zhu%20M,%20Zhang%20J,%20Liu%20X,%20Hou%20L,%20Liu%20W,%20et%20al.%20A%20sequence-aware%20merger%20of%20genomic%20structural%20variations%20at%20population%20scale.%20Nat%20Commun.%202024;15(1):960." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366149/bin/12859_2025_6246_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 1.</a><sup> (686.4KB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data analyzed during this study are publicly available. Raw sequencing data of HG002 (NA24385) in FASTQ format was obtained from <a href="https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/data/AshkenazimTrio/HG002_NA24385_son/NIST_Illumina_2x250bps/reads/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/data/AshkenazimTrio/HG002_NA24385_son/NIST_Illumina_2x250bps/reads/</a>. The GIAB SV ground truth of HG002, HG002_SVs_Tier1_v0.6.vcf.gz, was retrieved from <a href="https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/NIST_SV_v0.6/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/NIST_SV_v0.6/</a>, and additionally, the substituted SVs in challenging medically relevant genes, HG002_GRCh37_CMRG_SV_v1.00.vcf.gz, from <a href="https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/CMRG_v1.00/GRCh37/StructuralVariant/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/CMRG_v1.00/GRCh37/StructuralVariant/</a>. The corresponding specification of high confidence regions for SV benchmarking, HG002_SVs_Tier2_v0.6.bed, was downloaded from <a href="https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/NIST_SV_v0.6/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/NIST_SV_v0.6/</a>. Raw 150bp paired-end WGS data of HG002, downsampled to 30x, was obtained from <a href="https://github.com/human-pangenomics/HG002_Data_Freeze_v1.0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/human-pangenomics/HG002_Data_Freeze_v1.0</a>, and hg38-based SV benchmark in challenging medically relevant genes from <a href="https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/CMRG_v1.00/GRCh38/StructuralVariant/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/CMRG_v1.00/GRCh38/StructuralVariant/</a>. Raw sequencing data from samples HG00514, HG00733, and NA19240 in CRAM format was retrieved from <a href="https://ftp.sra.ebi.ac.uk/vol1/run/ERR398/ERR3988781/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp.sra.ebi.ac.uk/vol1/run/ERR398/ERR3988781/</a>, <a href="https://ftp.sra.ebi.ac.uk/vol1/run/ERR398/ERR3988823/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp.sra.ebi.ac.uk/vol1/run/ERR398/ERR3988823/</a>, and <a href="https://ftp.sra.ebi.ac.uk/vol1/run/ERR398/ERR3989410/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp.sra.ebi.ac.uk/vol1/run/ERR398/ERR3989410/</a>. Ground truth SV data for samples HG00514, HG00733, and NA19240, was downloaded from <a href="https://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/release/v2.0/integrated_callset/variants_freeze4_sv_insdel_alt.vcf.gz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC2/release/v2.0/integrated_callset/variants_freeze4_sv_insdel_alt.vcf.gz</a>. Data used for XGBoost model training and evaluation is available at <a href="https://github.com/ccfboc-bioinformatics/SV-MeCa_data" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ccfboc-bioinformatics/SV-MeCa_data</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Bioinformatics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12859-025-06246-6"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12859_2025_Article_6246.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366149/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366149/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366149%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366149/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366149/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366149/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40836322/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366149/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40836322/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366149/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366149/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="t55KVKX7LRqYsbIejuKV35i3d4M8WpwR92BfJgUkr45gazLeXadS9HL8W3yWUiJ2">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
