
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            QTL detection and genomic prediction for resistance to anthracnose in alfalfa (Medicago sativa) - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15251B3F8AEEE2C3051B3F0001329E29.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326290/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="QTL detection and genomic prediction for resistance to anthracnose in alfalfa (Medicago sativa)">
<meta name="citation_author" content="Marie Pégard">
<meta name="citation_author_institution" content="INRAE, P3F, Lusignan, France">
<meta name="citation_author" content="Camille Gréard">
<meta name="citation_author_institution" content="GIE GRASS, Saint‐Sauvant, France">
<meta name="citation_author" content="Marius Grelier">
<meta name="citation_author_institution" content="Cérience, Saint‐Sauvant, France">
<meta name="citation_author" content="Marie‐Christine Gras">
<meta name="citation_author_institution" content="RAGT 2n, Druelle, France">
<meta name="citation_author" content="Laure Saint‐Pierre">
<meta name="citation_author_institution" content="RAGT 2n, Druelle, France">
<meta name="citation_author" content="Bernard Tharel">
<meta name="citation_author_institution" content="Barenbrug, Connantre, France">
<meta name="citation_author" content="Philippe Barre">
<meta name="citation_author_institution" content="INRAE, P3F, Lusignan, France">
<meta name="citation_author" content="Bernadette Julier">
<meta name="citation_author_institution" content="INRAE, P3F, Lusignan, France">
<meta name="citation_publication_date" content="2025 Aug 6">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70085">
<meta name="citation_doi" content="10.1002/tpg2.70085">
<meta name="citation_pmid" content="40765523">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326290/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326290/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326290/pdf/TPG2-18-e70085.pdf">
<meta name="description" content="Forage production, persistence, and associated ecosystem services provided by the major forage legume, alfalfa (Medicago sativa), may be affected by disease susceptibility. Resistance to anthracnose, caused by Colletotrichum trifolii, has been ...">
<meta name="og:title" content="QTL detection and genomic prediction for resistance to anthracnose in alfalfa (Medicago sativa)">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Forage production, persistence, and associated ecosystem services provided by the major forage legume, alfalfa (Medicago sativa), may be affected by disease susceptibility. Resistance to anthracnose, caused by Colletotrichum trifolii, has been ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326290/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12326290">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70085"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70085.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12326290%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12326290/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12326290/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326290/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70085" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Aug 6;18(3):e70085. doi: <a href="https://doi.org/10.1002/tpg2.70085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70085</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>QTL detection and genomic prediction for resistance to anthracnose in alfalfa (<em>Medicago sativa</em>)</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22P%C3%A9gard%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Marie Pégard</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Marie Pégard</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>INRAE, P3F, Lusignan, France</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22P%C3%A9gard%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marie Pégard</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gr%C3%A9ard%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Camille Gréard</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Camille Gréard</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>GIE GRASS, Saint‐Sauvant, France</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gr%C3%A9ard%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Camille Gréard</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grelier%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Marius Grelier</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Marius Grelier</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Cérience, Saint‐Sauvant, France</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grelier%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marius Grelier</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gras%20MC%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Marie‐Christine Gras</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Marie‐Christine Gras</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>RAGT 2n, Druelle, France</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gras%20MC%22%5BAuthor%5D" class="usa-link"><span class="name western">Marie‐Christine Gras</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saint%E2%80%90Pierre%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Laure Saint‐Pierre</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Laure Saint‐Pierre</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>RAGT 2n, Druelle, France</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saint%E2%80%90Pierre%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Laure Saint‐Pierre</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tharel%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Bernard Tharel</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Bernard Tharel</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Barenbrug, Connantre, France</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tharel%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bernard Tharel</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Barre%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Philippe Barre</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Philippe Barre</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>INRAE, P3F, Lusignan, France</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Barre%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Philippe Barre</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Julier%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Bernadette Julier</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Bernadette Julier</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>INRAE, P3F, Lusignan, France</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Julier%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bernadette Julier</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270085-aff-0001">
<sup>
<sup>1</sup>
</sup>INRAE, P3F, Lusignan, France</div>
<div id="tpg270085-aff-0002">
<sup>
<sup>2</sup>
</sup>GIE GRASS, Saint‐Sauvant, France</div>
<div id="tpg270085-aff-0003">
<sup>
<sup>3</sup>
</sup>Cérience, Saint‐Sauvant, France</div>
<div id="tpg270085-aff-0004">
<sup>
<sup>4</sup>
</sup>RAGT 2n, Druelle, France</div>
<div id="tpg270085-aff-0005">
<sup>
<sup>5</sup>
</sup>Barenbrug, Connantre, France</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Bernadette Julier, INRAE, P3F, Lusignan, France. Email: <span>bernadette.julier@inrae.fr</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 26; Received 2025 Apr 11; Accepted 2025 Jun 30; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12326290  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40765523/" class="usa-link">40765523</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Forage production, persistence, and associated ecosystem services provided by the major forage legume, alfalfa (<em>Medicago sativa</em>), may be affected by disease susceptibility. Resistance to anthracnose, caused by <em>Colletotrichum trifolii</em>, has been described as an oligogenic trait, but the precise location of resistance genes on the alfalfa genome is not known. Therefore, we phenotyped a set of 417 alfalfa accessions for anthracnose resistance as the frequency of resistant plants. With available genotyping by sequencing data for 380 accessions from this collection, we performed quantitative trait locus (QTL) detection by genome‐wide association study (GWAS) and genomic prediction using a validation set of 97 accessions randomly selected. A wide range of variation for anthracnose resistance was observed, with newer varieties and breeding materials exhibiting greater resistance than old varieties and landraces. Accessions from America showed the highest resistance, although some European accessions also displayed notable resistance. Six QTLs, controlling 58% of the variation, were identified by GWAS. Two major QTLs were found on chromosome 8, within a region already identified in an alfalfa mapping population. Four other QTLs, each controlling less than 5% of the variation, were also found, including one near a major QTL on chromosome 4 in the model species <em>M. truncatula</em>. The predictive ability of our set of accessions was surprisingly high: 85%. These results are promising and highlight the potential of molecular markers and genomic prediction to improve anthracnose resistance in alfalfa breeding programs.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270085-list-10001" class="list" style="list-style-type:disc">
<li><p>We report a large diversity for anthracnose resistance among alfalfa accessions.</p></li>
<li><p>Two large quantitative trait loci were identified by genome‐wide association study on chromosome 8 within a short distance.</p></li>
<li><p>The genomic prediction of anthracnose resistance reached 0.85.</p></li>
<li><p>Marker‐assisted breeding could improve anthracnose resistance.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270085-dl-0001">
<dt>GBLUP</dt>
<dd><p>genomic best linear unbiased prediction</p></dd>
<dt>GBS</dt>
<dd><p>genotyping by sequencing</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>MLMM</dt>
<dd><p>multi‐locus mixed model</p></dd>
<dt>PDA</dt>
<dd><p>potato dextrose agar</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
</dl>
<section id="tpg270085-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Alfalfa or lucerne (<em>Medicago sativa</em> L.) is a forage crop of great interest, especially in the context of the increasing global demand for locally produced protein (Dubeux et al., <a href="#tpg270085-bib-0015" class="usa-link" aria-describedby="tpg270085-bib-0015">2024</a>; Julier et al., <a href="#tpg270085-bib-0032" class="usa-link" aria-describedby="tpg270085-bib-0032">2017</a>; Singer et al., <a href="#tpg270085-bib-0052" class="usa-link" aria-describedby="tpg270085-bib-0052">2018</a>). This species is an autotetraploid with a genome size of 2738 Mb (Chen et al., <a href="#tpg270085-bib-0011" class="usa-link" aria-describedby="tpg270085-bib-0011">2020</a>). This complexity makes breeding and marker‐assisted selection more difficult and slower. Moreover, it also provides numerous other ecosystem services such as breaking pests and diseases’ cycles, improving soil structure, and saving fossil energy. However, Southern anthracnose, caused by <em>Colletotrichum trifolii</em>, is one of the most important diseases of perennial alfalfa worldwide, responsible for significant yield losses (Nutter et al., <a href="#tpg270085-bib-0041" class="usa-link" aria-describedby="tpg270085-bib-0041">2002</a>; Yang et al., <a href="#tpg270085-bib-0057" class="usa-link" aria-describedby="tpg270085-bib-0057">2007</a>). This pathogen causes dry necrosis on the stems and the leaves, resulting in defoliation and yield losses of up to 25%–30%, as well as a vigor loss in the spring following the infection (Barnes et al., <a href="#tpg270085-bib-0007" class="usa-link" aria-describedby="tpg270085-bib-0007">1969</a>; Irwin et al., <a href="#tpg270085-bib-0027" class="usa-link" aria-describedby="tpg270085-bib-0027">1980</a>).</p>
<p>Breeding for resistance to anthracnose mainly uses two pathotypes (described as “races” in the literature) of <em>C. trifolii</em>, defined by the resistance level of two American reference varieties: Arc, resistant to pathotype 1 and susceptible to pathotype 2, and Saranac AR, resistant to both pathotypes (Ostazeski &amp; Elgin, <a href="#tpg270085-bib-0043" class="usa-link" aria-describedby="tpg270085-bib-0043">1982</a>). An inheritance study showed that the resistance of Arc to pathotype 1 was controlled by the dominant allele of the <em>An<sub>1</sub>
</em> gene, whereas the resistance of Saranac AR to pathotypes 1 and 2 was controlled by another independent gene <em>An<sub>2</sub>
</em> (Elgin and Ostazeski, <a href="#tpg270085-bib-0017" class="usa-link" aria-describedby="tpg270085-bib-0017">1985</a>). In Europe and North Africa, different isolates were collected in alfalfa fields showing anthracnose symptoms, and the varieties showing resistance to pathotype 1 were also resistant to these isolates (Gondran, <a href="#tpg270085-bib-0021" class="usa-link" aria-describedby="tpg270085-bib-0021">1982</a>; Gosset et al., <a href="#tpg270085-bib-0023" class="usa-link" aria-describedby="tpg270085-bib-0023">1989</a>; Raynal, <a href="#tpg270085-bib-0047" class="usa-link" aria-describedby="tpg270085-bib-0047">1977</a>; Raynal et al., <a href="#tpg270085-bib-0048" class="usa-link" aria-describedby="tpg270085-bib-0048">1989</a>). Resistance to the pathotype 1 and the French isolates was also described to be only partially dominant (Guy, <a href="#tpg270085-bib-0024" class="usa-link" aria-describedby="tpg270085-bib-0024">1976</a>; Mackie et al., <a href="#tpg270085-bib-0038" class="usa-link" aria-describedby="tpg270085-bib-0038">2007</a>). Studies of the genetics of resistance to anthracnose pathotype 1 in the model species <em>Medicago truncatula</em> reported a major locus of resistance in the upper part of chromosome 4, named <em>RCT1</em> (Yang et al., <a href="#tpg270085-bib-0057" class="usa-link" aria-describedby="tpg270085-bib-0057">2007</a>) or <em>Ct1</em> (Ameline‐Torregrosa, Cazaux, et al., <a href="#tpg270085-bib-0001" class="usa-link" aria-describedby="tpg270085-bib-0001">2008</a>). The resistant allele of the <em>RCT1</em> gene, identified as a member of the Toll‐interleukin‐1 receptor/nucleotide‐binding site/leucine‐rich repeat (TIR‐NBS‐LRR) class of plant R genes, conferred anthracnose resistance when transferred into susceptible alfalfa plants (Yang et al., <a href="#tpg270085-bib-0058" class="usa-link" aria-describedby="tpg270085-bib-0058">2008</a>). In an alfalfa mapping population that was a backcross between a resistant clone and a susceptible one, six quantitative trait loci (QTLs) for resistance to pathotype 1 were identified, including a major QTL at the top of chromosome 8 and a minor QTL at the top of chromosome 4, on a homeologous region of <em>M. truncatula RCT1</em> (Mackie et al., <a href="#tpg270085-bib-0038" class="usa-link" aria-describedby="tpg270085-bib-0038">2007</a>). The positions of the QTL identified so far on alfalfa and <em>M. truncatula</em> genomes have been synthetized and aligned with the first alfalfa reference genome (Chen et al., <a href="#tpg270085-bib-0011" class="usa-link" aria-describedby="tpg270085-bib-0011">2020</a>) (Figure <a href="#tpg270085-fig-0001" class="usa-link">1</a>). Before the release of this alfalfa genome sequence, as <em>M. truncatula</em> was the model species for legume crops, most of the <em>M. sativa</em> genetic maps and genomes were aligned on the successive versions of the <em>M. truncatula</em> reference genome of Jemalong‐A17 (Pecrix et al., <a href="#tpg270085-bib-0044" class="usa-link" aria-describedby="tpg270085-bib-0044">2018</a>; Young et al., <a href="#tpg270085-bib-0059" class="usa-link" aria-describedby="tpg270085-bib-0059">2011</a>). This first alfalfa reference genome was compared to an <em>M. sativa</em> genetic map (Chen et al., <a href="#tpg270085-bib-0011" class="usa-link" aria-describedby="tpg270085-bib-0011">2020</a>) that was itself aligned to the original reference genome of <em>M. truncatula</em> (X. H. Li et al., <a href="#tpg270085-bib-0036" class="usa-link" aria-describedby="tpg270085-bib-0036">2014</a>). This evidenced that the reference genome was not orientated as the <em>M. truncatula</em> genome, and namely, chromosomes 1, 3, 5, and 6 were in the same orientation, while chromosomes 2, 4, 7, and 8 were inversed. In addition, there was a translocation between the bottom of chromosome 4 and the bottom of chromosome 8 when comparing between <em>M. truncatula</em> Jemalong‐A17 genome and other <em>M. truncatula</em> genotypes (A. Li et al., <a href="#tpg270085-bib-0034" class="usa-link" aria-describedby="tpg270085-bib-0034">2022</a>), and this translocation was conserved in <em>M. sativa</em> (Chen et al., <a href="#tpg270085-bib-0011" class="usa-link" aria-describedby="tpg270085-bib-0011">2020</a>; X. H. Li et al., <a href="#tpg270085-bib-0036" class="usa-link" aria-describedby="tpg270085-bib-0036">2014</a>). The schematic view of the genome alignment is summarized in Figure <a href="#tpg270085-fig-0001" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="tpg270085-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac50/12326290/e9fd02b61ced/TPG2-18-e70085-g005.jpg" loading="lazy" id="jats-graphic-1" height="262" width="600" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270085-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Synthetic view of alignment among <em>M. truncatula</em> genomes of Jemalong‐A17 v5 (Pécrix et al., <a href="#tpg270085-bib-0044" class="usa-link" aria-describedby="tpg270085-bib-0044">2018</a>) and R108 (A. Li et al., <a href="#tpg270085-bib-0034" class="usa-link" aria-describedby="tpg270085-bib-0034">2022</a>), <em>M. sativa</em> genetic map (X. H. Li et al., <a href="#tpg270085-bib-0036" class="usa-link" aria-describedby="tpg270085-bib-0036">2014</a>), and genome XinJiangDaYe (Chen et al., <a href="#tpg270085-bib-0011" class="usa-link" aria-describedby="tpg270085-bib-0011">2020</a>) for chromosomes 4 and 8. The translocation between the bottom of chromosomes 4 and 8 of Jemalong‐A17 and R108, conserved in <em>M. sativa</em> is represented. The arrows represent the genes or quantitative trait loci (QTLs) found in previous studies: <em>RCT1</em> (Yang et al., <a href="#tpg270085-bib-0057" class="usa-link" aria-describedby="tpg270085-bib-0057">2007</a>), <em>Ct1</em> (Ameline‐Torregrossa et al., <a href="#tpg270085-bib-0001" class="usa-link" aria-describedby="tpg270085-bib-0001">2008a</a>), two QTL in a mapping population (Mackie et al., <a href="#tpg270085-bib-0038" class="usa-link" aria-describedby="tpg270085-bib-0038">2007</a>). The boxes Gwas indicate the single‐nucleotide polymorphism (SNP) found with genome‐wide association study (GWAS) in this study.</p></figcaption></figure><p>Selection for anthracnose resistance in breeding programs is a key lever for controlling anthracnose in alfalfa (Elgin et al., <a href="#tpg270085-bib-0016" class="usa-link" aria-describedby="tpg270085-bib-0016">1981</a>; Mackie et al., <a href="#tpg270085-bib-0038" class="usa-link" aria-describedby="tpg270085-bib-0038">2007</a>). Recurrent selection is a method to obtain varieties with a certain level of resistance to <em>C. trifolii</em> (Clements et al., <a href="#tpg270085-bib-0012" class="usa-link" aria-describedby="tpg270085-bib-0012">1984</a>; Devine et al., <a href="#tpg270085-bib-0014" class="usa-link" aria-describedby="tpg270085-bib-0014">1971</a>). At the individual level, each plant is scored as susceptible or resistant. At the variety level, because of their synthetic structure, the resistance is expressed as the proportion of resistant plants. Due to the variety heterogeneity, the high level of heterozygosity (Flajoulot et al., <a href="#tpg270085-bib-0018" class="usa-link" aria-describedby="tpg270085-bib-0018">2005</a>; Valizadeh et al., <a href="#tpg270085-bib-0053" class="usa-link" aria-describedby="tpg270085-bib-0053">2011</a>), the severe inbreeding depression (Busbice &amp; Wilsie, <a href="#tpg270085-bib-0008" class="usa-link" aria-describedby="tpg270085-bib-0008">1966</a>; Wilsie, <a href="#tpg270085-bib-0056" class="usa-link" aria-describedby="tpg270085-bib-0056">1958</a>) of this autotetraploid species (2<em>n</em> = 4<em>x</em> = 32), and the dominance of the resistance, a certain proportion of susceptible plants is maintained in the varieties considered as resistant (Mackie et al., <a href="#tpg270085-bib-0038" class="usa-link" aria-describedby="tpg270085-bib-0038">2007</a>). So far, markers found to be associated with anthracnose resistance have not been used in breeding programs. Indeed, the loci identified in <em>M. truncatula</em> have not been proven to be responsible for resistance in alfalfa, and because of the short linkage disequilibrium in alfalfa (Herrmann et al., <a href="#tpg270085-bib-0025" class="usa-link" aria-describedby="tpg270085-bib-0025">2010</a>), the QTL found in an alfalfa mapping population (Mackie et al., <a href="#tpg270085-bib-0038" class="usa-link" aria-describedby="tpg270085-bib-0038">2007</a>) cannot be used in other populations.</p>
<p>The recent release of alfalfa reference genome sequences (Chen et al., <a href="#tpg270085-bib-0011" class="usa-link" aria-describedby="tpg270085-bib-0011">2020</a>; W. Y. Li et al., <a href="#tpg270085-bib-0035" class="usa-link" aria-describedby="tpg270085-bib-0035">2024</a>; Long et al., <a href="#tpg270085-bib-0037" class="usa-link" aria-describedby="tpg270085-bib-0037">2022</a>; Shen et al., <a href="#tpg270085-bib-0050" class="usa-link" aria-describedby="tpg270085-bib-0050">2020</a>; Shi et al., <a href="#tpg270085-bib-0051" class="usa-link" aria-describedby="tpg270085-bib-0051">2024</a>) is key information to detect QTL with genome‐wide association study (GWAS) and to build genomic prediction equations (Flint‐Garcia et al., <a href="#tpg270085-bib-0019" class="usa-link" aria-describedby="tpg270085-bib-0019">2003</a>; Meuwissen et al., <a href="#tpg270085-bib-0039" class="usa-link" aria-describedby="tpg270085-bib-0039">2001</a>). These methods have already been applied to several agronomic traits (Annicchiarico et al., <a href="#tpg270085-bib-0004" class="usa-link" aria-describedby="tpg270085-bib-0004">2015</a>, <a href="#tpg270085-bib-0005" class="usa-link" aria-describedby="tpg270085-bib-0005">2021</a>, <a href="#tpg270085-bib-0003" class="usa-link" aria-describedby="tpg270085-bib-0003">2022</a>; X. Jiang et al., <a href="#tpg270085-bib-0029" class="usa-link" aria-describedby="tpg270085-bib-0029">2024</a>; X.‐Q. Jiang et al., <a href="#tpg270085-bib-0030" class="usa-link" aria-describedby="tpg270085-bib-0030">2022</a>; Murad Leite Andrade et al., <a href="#tpg270085-bib-0040" class="usa-link" aria-describedby="tpg270085-bib-0040">2022</a>; Pégard et al., <a href="#tpg270085-bib-0045" class="usa-link" aria-describedby="tpg270085-bib-0045">2023</a>), with a prediction ability ranging from 0.2 to 0.8 depending on the population or the method. Previous studies have explored various statistical models for genomic prediction in alfalfa, focusing on improving prediction accuracy for agronomic traits. Traditional methods such as genomic best linear unbiased prediction (GBLUP) and Bayesian approaches (e.g., BayesA, BayesB, BayesC) are commonly used as baseline models and often perform similarly. Some studies have shown that machine learning methods, particularly support vector machine (SVM), often outperform traditional statistical models in terms of prediction accuracy. For instance, Zhang et al. (<a href="#tpg270085-bib-0060" class="usa-link" aria-describedby="tpg270085-bib-0060">2023</a>) reported that SVM regression achieved the highest prediction accuracy (64.1%) for autumn dormancy, surpassing other models such as Lasso and ElasticNet. However, with the same population, we found that Lasso and GBLUP performed similarly for autumn dormancy, even when major QTLs were detected (Pégard et al., <a href="#tpg270085-bib-0045" class="usa-link" aria-describedby="tpg270085-bib-0045">2023</a>). To our knowledge, genomic prediction has not been used in relation to anthracnose resistance. GWAS and genomic prediction could help to decipher the genetic control of anthracnose resistance in alfalfa and to detect trait‐associated markers that could be used in breeding programs. The selection of anthracnose‐resistant alfalfa could be accelerated by using such markers.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270085-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270085-list-0001" class="list" style="list-style-type:disc">
<li><p>We report a large diversity for anthracnose resistance among alfalfa accessions.</p></li>
<li><p>Two large quantitative trait loci were identified by genome‐wide association study on chromosome 8 within a short distance.</p></li>
<li><p>The genomic prediction of anthracnose resistance reached 0.85.</p></li>
<li><p>Marker‐assisted breeding could improve anthracnose resistance.</p></li>
</ul></section></section><p>This study reports on the testing of resistance to anthracnose in a large set of alfalfa accessions that have been genotyped with genotyping‐by‐sequencing (GBS) markers in a previous work (Pégard et al., <a href="#tpg270085-bib-0045" class="usa-link" aria-describedby="tpg270085-bib-0045">2023</a>). Resistance variation was analyzed in relation to geographical origin and year of registration. GWAS was performed, and the ability to predict anthracnose resistance with genomic prediction was assessed. Finally, the potential use of the results for marker‐assisted selection and genomic selection was discussed.</p></section><section id="tpg270085-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270085-sec-0040"><h3 class="pmc_sec_title">2.1. Materials</h3>
<p>A total of 400 alfalfa accessions were collected worldwide to cover the diversity of the cultivated pool within the dormancy range of 3–7, as described by Pégard et al. (<a href="#tpg270085-bib-0045" class="usa-link" aria-describedby="tpg270085-bib-0045">2023</a>). In addition, 17 breeding materials and three varieties were added. In this population of 417 accessions (Table <a href="#tpg270085-supinfo-0001" class="usa-link">S1</a>), there is an over‐representation of European varieties with both recent materials, including breeding materials, and old varieties or landraces. Accessions from the United States were mostly recent varieties or breeding materials. For all varieties, the country of origin or registration was noted. The status of the accessions (code 300 for landraces, 400 for breeding materials, and 500 for registered varieties) was recorded (<a href="https://www.genesys-pgr.org/documentation/basics#mcpd-sampstat" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genesys‐pgr.org/documentation/basics#mcpd‐sampstat</a>). The set included 23 landraces, 86 breeding materials, and 308 registered varieties. For 267 varieties, their registration year was available. For the landraces, the year 1950 was attributed as the registration year.</p></section><section id="tpg270085-sec-0050"><h3 class="pmc_sec_title">2.2. Genotyping</h3>
<p>The procedures for DNA extraction, optimization of the GBS methodology, and GBS have been previously documented in Julier et al. (<a href="#tpg270085-bib-0031" class="usa-link" aria-describedby="tpg270085-bib-0031">2018</a>). In summary, each accession consisted of 100 plants, and DNA was extracted from a pooled sample of 100 leaflets, with each leaflet taken from an individual plant. This method has been demonstrated to effectively estimate the allele frequency within an accession (Julier et al., <a href="#tpg270085-bib-0031" class="usa-link" aria-describedby="tpg270085-bib-0031">2018</a>). For the double‐digest GBS on alfalfa, the enzymes PstI–MseI were employed to achieve a sufficient number of loci while reducing missing data and accounting for the read count per accession. The reads were mapped to the reference genome of the Chinese variety XinJiangDaYe (Chen et al., <a href="#tpg270085-bib-0011" class="usa-link" aria-describedby="tpg270085-bib-0011">2020</a>), using haplotype 2 of each chromosome. After a series of trimming steps (Pégard et al., <a href="#tpg270085-bib-0045" class="usa-link" aria-describedby="tpg270085-bib-0045">2023</a>), 248,060 markers with less than 5% missing data per position were obtained. The genomic relationship matrix (<strong>G</strong>) was calculated with the subset of 118,418 single‐nucleotide polymorphisms (SNPs) without missing values (Pégard et al., <a href="#tpg270085-bib-0045" class="usa-link" aria-describedby="tpg270085-bib-0045">2023</a>). Because of poor sequencing quality for some accessions combined with seed shortage, genomic information was available for 380 populations in this study.</p></section><section id="tpg270085-sec-0060"><h3 class="pmc_sec_title">2.3. Inoculum production and inoculation</h3>
<p>The <em>C. trifolii</em> strain used in this study was the C86‐2. This strain was isolated in 1972 in Lusignan (France) and has been shown to be similar to pathotype 1 (Gondran, <a href="#tpg270085-bib-0021" class="usa-link" aria-describedby="tpg270085-bib-0021">1982</a>; Gondran &amp; Mainer‐Casado, <a href="#tpg270085-bib-0022" class="usa-link" aria-describedby="tpg270085-bib-0022">1973</a>). <em>C. trifolii</em> was cultivated on a potato dextrose agar (PDA) medium and conserved at −80°C. For spore multiplication, a drop of defrost spore solution was first spread on a sterile PDA medium (10 mL of inclined medium in a 2.2‐cm diameter tube). The tube was then exposed to natural light or natural‐sunlight‐imitating light at 20°C. <em>C. trifolii</em> sporulated after 1 week, when a pink jelly‐like substance could be observed at the surface of the black mycelium. The jelly was scratched with a sterile inoculating loop and introduced into 2 mL of sterile water. After homogenization, 0.5 mL of spore suspension was spread in Roux flasks containing PDA medium horizontally jellified. The flasks were exposed to light at 20°C for 1 week. Once the sporulation occurred, 50 mL of sterile water was poured into each flask, which was then closed and gently shaken to collect the jelly substance containing spores. This spore solution was filtered. Spore concentration was evaluated by counting spores on a Malassez cell and diluted to a concentration of 1–2 × 10<sup>6</sup> spores/mL for the inoculation.</p></section><section id="tpg270085-sec-0070"><h3 class="pmc_sec_title">2.4. Phenotyping</h3>
<p>Anthracnose resistance is studied as part of variety evaluation tests since 2010. All varieties registered in France from 2010 along with varieties from the reference collection were thus characterized for anthracnose resistance by GEVES. GEVES is the French registration office responsible for variety evaluation and seed testing. Of the 417 accessions, GEVES made the results of 150 accessions available to us. In addition, three private breeders active in France (Barenbrug, Cérience, and RAGT 2n) phenotyped 295 accessions, of which 28 accessions were tested both by the private partners and GEVES (Table <a href="#tpg270085-tbl-0001" class="usa-link">1</a>). Three to four control varieties (Ambra, Marshal, Alexis, and Kali), marketed in Europe and showing a resistance level ranging from highly resistant to highly susceptible, were included in each test as control varieties.</p>
<section class="tw xbox font-sm" id="tpg270085-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Series of anthracnose tests at GEVES and at three breeders’ places, number of repetitions, and number of accessions, including three to four control varieties, in each test.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Test identifier</th>
<th align="left" rowspan="1" colspan="1">Number of repetitions</th>
<th align="left" rowspan="1" colspan="1">Number of accessions</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">GEVES 12_3</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">32</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GEVES 13_1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">38</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GEVES 14_1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">44</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GEVES 15_1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">35</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GEVES 15_2</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">29</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GEVES 16_1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">19</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GEVES 17_1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">15</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GEVES 18_1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">20</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GEVES 18_2</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">8</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GEVES 19_1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">24</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GEVES 20_1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">18</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GEVES 21_1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">26</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Barenbrug 1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">37</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Barenbrug 2</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">14</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Barenbrug 3</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">48</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Barenbrug 4</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">50</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cérience 1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">49</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cérience 2</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">50</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cérience 3</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">50</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RAGT 2n 1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">56</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RAGT 2n 2</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">41</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270085-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The protocol used by GEVES and in this study (Perrot et al., <a href="#tpg270085-bib-0046" class="usa-link" aria-describedby="tpg270085-bib-0046">2009</a>) is similar to the protocol used in the United States (O'Neill, <a href="#tpg270085-bib-0042" class="usa-link" aria-describedby="tpg270085-bib-0042">2022</a>). For each accession, the seeds were placed on blotting paper at 25°C under continuous light for 24–72 h for germination. Then, 50–90 germinated seeds per repetition were planted in seedling trays placed in a growth chamber at 25°C/23°C day/night, with a 12‐h photoperiod. The number of repetitions and accessions for each test are indicated in Table <a href="#tpg270085-tbl-0001" class="usa-link">1</a>. Plants were inoculated with <em>C. trifolii</em> C86‐2 at the first trifoliate leaf stage. Before inoculation, the number of living plants in each repetition of each accession was counted. Inoculation was performed by evenly spraying 130–160 mL spore suspension with a concentration of 1–2 × 10<sup>6</sup> spores/mL per tray, as described in Gondran and Mainer‐Casado (<a href="#tpg270085-bib-0022" class="usa-link" aria-describedby="tpg270085-bib-0022">1973</a>). Plant trays were then placed back in the growth chamber under the same conditions (25°C/23°C day/night, with a 12‐h photoperiod), with a saturated hygrometry for the first 48 h. Resistance was scored 15 days after inoculation by evaluating the symptoms on each inoculated plant to classify them into two categories—dead plants (susceptible) and non‐dead plants (resistant)—without visible symptoms or with wilting signs.</p></section><section id="tpg270085-sec-0080"><h3 class="pmc_sec_title">2.5. Data analysis</h3>
<p>The percentage of resistance (<em>y</em>) was adjusted for trial and repetition effects using the following mixed model:
</p>
<table class="disp-formula p" id="tpg270085-disp-0001"><tr><td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><mi>y</mi><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><mi>TR</mi><mo linebreak="goodbreak">+</mo><mi>X</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">ε</mi></mrow></mrow></math></td></tr></table>
<p>where <em>μ</em> is the overall mean, TR is the combination of the trial and repetition effects, treated as a random effect, <em>X</em> is the genetic effect also considered as a random effect, and <em>Ɛ</em> is the residual effect. Notably, the accession identification was used as the genetic effect rather than the kinship between accessions based on markers. This approach was necessary because some of the control varieties were not genotyped. The percentage of resistance was adjusted by removing the component explained by the trial–repetition combination from the raw percentage. Subsequently, the adjusted percentage was used as the phenotype for further analysis. The percentage of resistance was represented as a histogram to illustrate the range of variation. It was plotted as a function of the registration year, when available, and the accession status (landrace, breeding material, registered variety). The mean resistance score standard deviation was calculated for each country of origin. The broad‐sense heritability was estimated using the variances calculated with the above model as follows:
</p>
<table class="disp-formula p" id="tpg270085-disp-0002"><tr><td class="formula"><math id="jats-math-2" display="block"><mrow><mrow><mi>Var</mi><mspace width="0.16em"></mspace><mi>X</mi><mo>/</mo><mo stretchy="false">(</mo><mi>Var</mi><mspace width="0.16em"></mspace><mi>X</mi><mo linebreak="goodbreak">+</mo><mi>Var</mi><mspace width="0.16em"></mspace><mi>TR</mi><mo linebreak="goodbreak">+</mo><mi>Var</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">ε</mi><mo stretchy="false">)</mo><mo>.</mo></mrow></mrow></math></td></tr></table>
<p>The genomic relationship matrix <strong>G</strong> was based on VanRaden (<a href="#tpg270085-bib-0054" class="usa-link" aria-describedby="tpg270085-bib-0054">2008</a>), adapted to use allele frequencies (continuous values from 0 to 1) instead of allele dose (Ashraf et al., <a href="#tpg270085-bib-0006" class="usa-link" aria-describedby="tpg270085-bib-0006">2014</a>). The genotyping matrix <strong>M</strong> was normalized by the minimum allele frequency (<em>p</em>) to obtain the normalized genotyping matrix <strong>Z</strong> used to compute <strong>G</strong> as follows:
</p>
<table class="disp-formula p" id="tpg270085-disp-0003"><tr><td class="formula"><math id="jats-math-3" display="block"><mrow><mrow><mi mathvariant="bold">G</mi><mo linebreak="badbreak">=</mo><mfrac><msup><mi mathvariant="bold">ZZ</mi><mo>′</mo></msup><mrow><mfrac><mn>1</mn><mi>n</mi></mfrac><msubsup><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>m</mi></msubsup><msub><mi>p</mi><mi>j</mi></msub><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msub><mi>p</mi><mi>j</mi></msub><mo stretchy="false">)</mo></mrow></mrow></mfrac></mrow></mrow></math></td></tr></table>
<p>The denominator is a scaling parameter, corresponding to the sum of the expected SNP variance across genotypes (Ashraf et al., <a href="#tpg270085-bib-0006" class="usa-link" aria-describedby="tpg270085-bib-0006">2014</a>), where <em>m</em> represents the number of markers, <em>p<sub>j</sub>
</em> equals the frequency of the <em>j</em>th marker, and <em>n</em> represents a scaling number to obtain a diagonal mean close to 1. This has been recommended in previous studies on polyploid species (Ashraf et al., <a href="#tpg270085-bib-0006" class="usa-link" aria-describedby="tpg270085-bib-0006">2014</a>; Cericola et al., <a href="#tpg270085-bib-0010" class="usa-link" aria-describedby="tpg270085-bib-0010">2018</a>), and with <em>n </em>= 16, the diagonal mean was close to 1.</p>
<p>GWAS was conducted using the multi‐locus mixed model (MLMM) method (Segura et al., <a href="#tpg270085-bib-0049" class="usa-link" aria-describedby="tpg270085-bib-0049">2012</a>), which accounted for the genetic structure of the cultivated material through a genomic relationship matrix. This method employed a stepwise mixed‐model regression approach, incorporating SNPs as cofactors through forward inclusion and backward elimination. At each step, the variance components of the model were re‐estimated, enhancing detection power while reducing the false detection rate. The analysis was limited to a maximum of 10 steps, with the optimal step selected using an adjusted multiple Bonferroni criterion (10<sup>−5</sup>). The percentage of phenotypic variation explained by each QTL was determined by comparing the <em>R</em>
<sup>2</sup> values of a linear model that included all QTLs as fixed effects and the genomic relationship matrix (<strong>G</strong>) as a random effect, with and without the focused QTL.</p>
<p>The genes located under the QTL were determined using the Genome Browser developed on the XinJiangDaYe genotype of <em>M. sativa</em>
<a href="https://bbric-pipelines.toulouse.inra.fr/myGenomeBrowser?search=1&amp;portalname=MSAT_XinJiangDaYe&amp;owner=sebastien.carrere@inrae.fr&amp;key=PyG9k9tK" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bbric‐pipelines.toulouse.inra.fr/myGenomeBrowser?search=1&amp;portalname=MSAT_XinJiangDaYe&amp;owner=sebastien.carrere@inrae.fr&amp;key=PyG9k9tK</a>. The positions of the QTL were compared to other QTLs found on <em>M. truncatula</em> when the available reference genome sequence was version 1 or a genetic map of <em>M. sativa</em>. On <em>M. truncatula</em>, the QTL pointed to <em>RCT1</em> gene and its protein sequence <a href="https://www.ncbi.nlm.nih.gov/protein/ACF19650.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ACF19650.1</a> (Yang et al., <a href="#tpg270085-bib-0057" class="usa-link" aria-describedby="tpg270085-bib-0057">2007</a>) or <em>Ct1</em> gene included in the BAC <a href="https://www.ncbi.nlm.nih.gov/nuccore/BF519144" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">BF519144</a> (Ameline‐Torregrosa, Cazaux, et al., <a href="#tpg270085-bib-0001" class="usa-link" aria-describedby="tpg270085-bib-0001">2008</a>). On <em>M. sativa</em>, a major QTL for resistance to anthracnose pathotype 1 was found at the top of chromosome 8 (Mackie et al., <a href="#tpg270085-bib-0038" class="usa-link" aria-describedby="tpg270085-bib-0038">2007</a>), and a minor QTL was found at the top of chromosome 4, but, as the surrounding markers are Amplified Fragment‐Length Polymorphism (AFLP), it was not possible to find the exact homologous region on the current <em>M. sativa</em> reference genome.</p>
<p>As several QTLs were found in a small region of the genome, an estimation of linkage disequilibrium was calculated as in Pégard et al. (<a href="#tpg270085-bib-0045" class="usa-link" aria-describedby="tpg270085-bib-0045">2023</a>). For this region, the genomic sequences of the four homologs of chromosome 8 (Chen et al., <a href="#tpg270085-bib-0011" class="usa-link" aria-describedby="tpg270085-bib-0011">2020</a>) were extracted and aligned with the D‐Genies program available at <a href="https://dgenies.toulouse.inra.fr/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dgenies.toulouse.inra.fr/</a> with the default options (Cabanettes &amp; Klopp, <a href="#tpg270085-bib-0009" class="usa-link" aria-describedby="tpg270085-bib-0009">2018</a>). The Pearson correlation between allele frequencies of the SNP involved in QTL was also calculated.</p>
<p>For genomic prediction, a validation population consisting of 97 randomly selected accessions was formed, ensuring that each genetic group identified by Pégard et al. (<a href="#tpg270085-bib-0045" class="usa-link" aria-describedby="tpg270085-bib-0045">2023</a>) was represented proportionally to its size. The remaining accessions were randomly sampled to evaluate the effect of the training population size. Eight different sample sizes were tested, ranging from 20% of the remaining accessions (59 accessions) to 95% (270 accessions). For each sample size, 10 iterations were performed. The quality of the predictions was assessed by calculating the predictive ability, defined as the Pearson correlation between the adjusted phenotypes and the predicted values. The best linear unbiased prediction based on genomic information (GBLUP) (Meuwissen et al., <a href="#tpg270085-bib-0039" class="usa-link" aria-describedby="tpg270085-bib-0039">2001</a>; Whittaker et al., <a href="#tpg270085-bib-0055" class="usa-link" aria-describedby="tpg270085-bib-0055">2000</a>) was employed to predict the genomic estimated breeding values using all SNPs to compute the genomic relationship matrix (<strong>G</strong>). The analysis was conducted using the R package breedR.</p></section></section><section id="tpg270085-sec-0090"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270085-sec-0100"><h3 class="pmc_sec_title">3.1. Range of variation for anthracnose resistance in alfalfa</h3>
<p>The four control varieties showed a wide range of variation for the adjusted resistance percentage, with Ambra being highly resistant (80.4%) and Marshal being resistant (70.0%), while Alexis was susceptible (26.2%) and Kali highly susceptible (17.5%). The resistance of these control varieties thus adequately covered the range of variation.</p>
<p>In the set of accessions, the adjusted resistance percentages ranged from −3.9% to 83.0%, with a mean value of 32.9% (Figure <a href="#tpg270085-fig-0002" class="usa-link">2A</a>). If we take an arbitrary threshold of 60% resistant plants to define an accession as resistant, then only 59 accessions can be considered resistant. Landraces and old varieties (registered before 1990) were susceptible, with resistances percentages ranging from −3.9% to 49.3% (Figure <a href="#tpg270085-fig-0002" class="usa-link">2B</a>). After 1990, the varieties had at least resistances above 10%, and most of them had resistances below 40%, but 21 varieties exhibited resistances above 60%. Over the whole set of 417 accessions, 42 accessions (varieties or breeding materials) had resistance percentages above 60%, 26 from the United States, 13 from France, and one from each of China, Italy, and Romania. The landraces were on average susceptible but showed a range of resistance percentages between 6% and 35%, while the breeding materials and the registered varieties covered the full range of variation in resistance percentage (Figure <a href="#tpg270085-fig-0002" class="usa-link">2C</a>).</p>
<figure class="fig xbox font-sm" id="tpg270085-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac50/12326290/cadd1bd2cd69/TPG2-18-e70085-g004.jpg" loading="lazy" id="jats-graphic-3" height="277" width="600" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270085-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Variation for anthracnose resistance (%) in alfalfa. (A) Range of variation among 417 accessions, (B) resistance of 287 accessions as a function of their registration year, and (C) resistance of 394 accessions as a function of their status (codes 300: landraces, 400: breeding materials, 500: registered varieties).</p></figcaption></figure><p>The mean resistances were the highest for the accessions from the United States, Argentina, and Uruguay. Accessions from France, Romania, Serbia, Hungary, and Italy generally had intermediate mean resistance. Most of the accessions of the other countries had a resistance below 30% (Figure <a href="#tpg270085-fig-0003" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="tpg270085-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac50/12326290/a975c131090d/TPG2-18-e70085-g003.jpg" loading="lazy" id="jats-graphic-5" height="327" width="760" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270085-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mean anthracnose resistance (%) in the countries represented in our collection of accessions. When there was more than one accession per country, vertical bars indicate the standard deviation, and dots represent accessions that deviated substantially from the country mean.</p></figcaption></figure></section><section id="tpg270085-sec-0110"><h3 class="pmc_sec_title">3.2. QTL detection</h3>
<p>The heritability of the resistance percentage was high, estimated at 0.53. Each of the first six steps of MLMM procedures detected a significant QTL (Table <a href="#tpg270085-tbl-0002" class="usa-link">2</a>, Figure <a href="#tpg270085-supinfo-0001" class="usa-link">S1</a>). Two major QTLs were detected, corresponding to the SNP chr8_80198129 and chr8_83798444, both located at the bottom of chromosome 8 that comprises about 85 Mb (Chen et al., <a href="#tpg270085-bib-0011" class="usa-link" aria-describedby="tpg270085-bib-0011">2020</a>). Among the other four QTLs, two were also located at the bottom of chromosome 8 (chr8_77736763, chr8_80205951), one was at the top of chromosome 2 (chr2_1220775), and the fourth was at the bottom of chromosome 4 (chr4_76542639). The QTL detected in the fifth step, SNP chr8_80205951, was located in the gene just next to the first QTL, SNP chr8_80198129. As the chromosomes of this reference genome were not orientated as in the reference genome of <em>M. truncatula</em> (Figure <a href="#tpg270085-fig-0001" class="usa-link">1</a>), the four QTLs at the bottom of chromosome 8 correspond to the top of chromosome 8 of <em>M. truncatula</em> and the QTL at the bottom of chromosome 4 correspond to the top of chromosome 4 of <em>M. truncatula</em>. The homeologous region to the <em>M. truncatula RCT1</em> gene (Yang et al., <a href="#tpg270085-bib-0057" class="usa-link" aria-describedby="tpg270085-bib-0057">2007</a>) in the <em>M. sativa</em> genome spanned from 86665076 to 86671634 on chromosome 4, about 10 million bp from the QTL chr4_76542639. The QTL <em>Ct1</em> (Ameline‐Torregrosa, Cazaux, et al., <a href="#tpg270085-bib-0001" class="usa-link" aria-describedby="tpg270085-bib-0001">2008</a>) that corresponded to the BAC <a href="https://www.ncbi.nlm.nih.gov/nuccore/BF519144" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">BF519144</a> on <em>M. truncatula</em> was homeologous to MS.gene052809, located between 86940546 and 86948333 on chromosome 4, close to <em>RCT1</em> homeologous but far from our QTL.</p>
<section class="tw xbox font-sm" id="tpg270085-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Positions of the quantitative trait locus (QTL) for anthracnose resistance detected in the first six steps of the multi‐locus mixed model (MLMM) procedure, percentage of variation explained, and description of the candidate gene around the position: gene length, protein length, number of exons, location of the mutation and effect on gene and protein sequences, gene annotation, and homeology with <em>M. truncatula</em>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Position</th>
<th align="left" rowspan="1" colspan="1">% of explained variation</th>
<th align="left" rowspan="1" colspan="1">Candidate gene in <em>M. sativa</em>
</th>
<th align="left" rowspan="1" colspan="1">Gene length (bp)</th>
<th align="left" rowspan="1" colspan="1">Protein length (aa)</th>
<th align="left" rowspan="1" colspan="1">#Exons</th>
<th align="left" rowspan="1" colspan="1">Mutation</th>
<th align="left" rowspan="1" colspan="1">Candidate gene annotation</th>
<th align="left" rowspan="1" colspan="1">
<em>M. truncatula</em> homeolog and annotation</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">chr8_80198129</td>
<td align="center" rowspan="1" colspan="1">39.0</td>
<td align="left" rowspan="1" colspan="1">MS.gene37046 (strand +)</td>
<td align="center" rowspan="1" colspan="1">3663</td>
<td align="left" rowspan="1" colspan="1">491</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">Intron</td>
<td align="left" rowspan="1" colspan="1">Leucine‐rich repeat domain superfamily</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr8_83798444</td>
<td align="center" rowspan="1" colspan="1">11.3</td>
<td align="left" rowspan="1" colspan="1">MS.gene055334 (strand −)</td>
<td align="center" rowspan="1" colspan="1">6294</td>
<td align="left" rowspan="1" colspan="1">1 095</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">Exon 6, A → G, synonymous</td>
<td align="left" rowspan="1" colspan="1">EamA‐like transporter family</td>
<td align="left" rowspan="1" colspan="1">
<p>MtrunA17_Chr8g0334751</p>
<p>Putative EamA domain‐containing protein</p>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr8_77736763</td>
<td align="center" rowspan="1" colspan="1">3.6</td>
<td align="left" rowspan="1" colspan="1">MS.gene36672 (strand +)</td>
<td align="center" rowspan="1" colspan="1">36,672</td>
<td align="left" rowspan="1" colspan="1">637</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">Intron</td>
<td align="left" rowspan="1" colspan="1">Glutamate/leucine/phenylalanine/valine dehydrogenase signature</td>
<td align="left" rowspan="1" colspan="1">
<p>MtrunA17_Chr8g0341541</p>
<p>Putative glutamate dehydrogenase (NADP(+))</p>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr2_1220775</td>
<td align="center" rowspan="1" colspan="1">1.4</td>
<td align="left" rowspan="1" colspan="1">MS.gene83580 (strand +)</td>
<td align="center" rowspan="1" colspan="1">1407</td>
<td align="left" rowspan="1" colspan="1">468</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">Exon1, T → A, L → M</td>
<td align="left" rowspan="1" colspan="1">Leucine‐rich repeat domain superfamily</td>
<td align="left" rowspan="1" colspan="1">
<p>MtrunA17_Chr2g0333661</p>
<p>Putative leucine‐rich repeat domain superfamily</p>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr8_80205951</td>
<td align="center" rowspan="1" colspan="1">3.1</td>
<td align="left" rowspan="1" colspan="1">MS.gene37047 (strand −)</td>
<td align="center" rowspan="1" colspan="1">3136</td>
<td align="left" rowspan="1" colspan="1">590</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">Exon 1, T → C, synonymous</td>
<td align="left" rowspan="1" colspan="1">Leucine‐rich repeat domain superfamily</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr4_76542639</td>
<td align="center" rowspan="1" colspan="1">3.9</td>
<td align="left" rowspan="1" colspan="1">At 159 b from MS.gene82171 (strand +)</td>
<td align="center" rowspan="1" colspan="1">4641</td>
<td align="left" rowspan="1" colspan="1">339</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">3′ region</td>
<td align="left" rowspan="1" colspan="1">Unknown</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total</td>
<td align="center" rowspan="1" colspan="1">58.5</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270085-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The two major QTLs explained 39.0% and 11.3% of the variation, respectively, while the other QTLs explained only 1.4%–3.9% of the variation (Table <a href="#tpg270085-tbl-0002" class="usa-link">2</a>). Altogether, the six QTLs explained 58.5% of the variation. The frequency of the SNP chr8_80198129 varied from 0 to 0.63 and that of the SNP chr8_83798444 varied from 0 to 0.52 in the set of accessions. For both SNPs, the higher their frequency, the higher the resistance level of the accessions, but no accession had these SNPs at a very high frequency (Figure <a href="#tpg270085-fig-0004" class="usa-link">4</a>). For the other four QTLs, the correlation between SNP frequency and anthracnose resistance was convincing for the SNPs chr8_80205951 and chr4_76542639, but not for the SNPs chr8_77736763 and chr2_1220775 (Figure <a href="#tpg270085-fig-0004" class="usa-link">4</a>). For these SNPs as well, the ranges of variation of their allele frequencies did not span from 0 to 1. The two major QTLs were positively correlated (Figure <a href="#tpg270085-fig-0004" class="usa-link">4</a>, Table <a href="#tpg270085-tbl-0003" class="usa-link">3</a>), the most resistant accessions combining a high frequency for chr8_80198129 and chr8_83798444. Surprisingly, some accessions, such as EUC_MS_254, had high resistance levels but SNP frequencies lower than 0.10 at the two major QTLs, highlighting the role of the second QTL in the resistance. Among all six QTLs, the correlation was significant at <em>p</em> &lt; 0.05 between the SNPs chr8_80198129, chr8_83798444, and chr8_80205951, all on chromosome 8, but no other correlation was high, either with the fourth SNP on chromosome 8 (chr8_77736763) or the SNPs on chromosomes 2 and 4 (Table <a href="#tpg270085-tbl-0003" class="usa-link">3</a>, Figure <a href="#tpg270085-supinfo-0001" class="usa-link">S2</a>). Within the 4 million bp on chromosome 8 where three of the four QTLs were localized, the linkage disequilibrium, calculated after kinship correction, was short, with a sharp drop of <em>R</em>
<sup>2</sup> after 250 bp (Figure <a href="#tpg270085-supinfo-0001" class="usa-link">S3</a>).</p>
<figure class="fig xbox font-sm" id="tpg270085-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac50/12326290/78f966158851/TPG2-18-e70085-g002.jpg" loading="lazy" id="jats-graphic-7" height="535" width="800" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270085-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Anthracnose percentage of resistance depending on the allele frequencies of the six quantitative trait loci (QTLs). The two graphs on the top left represent the percentage of resistance (<em>y</em>‐axis) depending on the allele frequencies of chr8_80198129 (top) and chr8_83798444 (bottom). The blue line represents the regression between the percentage of resistance of these two QTLs, and the gray part represents the confidence interval of this regression; the equation and <em>R</em>
<sup>2</sup> of that regression are represented on each graph. The graph on the top right represents the allele frequencies of chr8_83798444 (<em>y</em>‐axis) depending on the allelic frequencies of chr8_80198129. The points, representing the accessions, are colored depending on their percentage of resistance, from red indicating a low percentage of resistance to green for a high percentage of resistance. The four graphs at the bottom visualize the relationship between anthracnose resistance and the allele frequency of the other four QTLs.</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270085-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Pearson correlation between allele frequencies of the six quantitative trait loci (QTLs) for anthracnose resistance.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">chr8_83798444</th>
<th align="left" rowspan="1" colspan="1">chr8_77736763</th>
<th align="left" rowspan="1" colspan="1">chr2_1220775</th>
<th align="left" rowspan="1" colspan="1">chr8_80205951</th>
<th align="left" rowspan="1" colspan="1">chr4_76542639</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">chr8_80198129</td>
<td align="center" rowspan="1" colspan="1">0.724</td>
<td align="center" rowspan="1" colspan="1">−0.191</td>
<td align="center" rowspan="1" colspan="1">−0.081ns</td>
<td align="center" rowspan="1" colspan="1">−0.554</td>
<td align="center" rowspan="1" colspan="1">0.166</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr8_83798444</td>
<td align="center" rowspan="1" colspan="1">1.000</td>
<td align="center" rowspan="1" colspan="1">−0.158</td>
<td align="center" rowspan="1" colspan="1">−0.077ns</td>
<td align="center" rowspan="1" colspan="1">−0.503</td>
<td align="center" rowspan="1" colspan="1">0.142</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr8_77736763</td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1">1.000</td>
<td align="center" rowspan="1" colspan="1">−0.037ns</td>
<td align="center" rowspan="1" colspan="1">0.080ns</td>
<td align="center" rowspan="1" colspan="1">0.043ns</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr2_1220775</td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1">1.000</td>
<td align="center" rowspan="1" colspan="1">−0.009ns</td>
<td align="center" rowspan="1" colspan="1">−0.096ns</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr8_80205951</td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1">1.000</td>
<td align="center" rowspan="1" colspan="1">−0.146</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr4_76542639</td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1">1.000</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270085-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270085-tbl3-note-0001"><p>
<em>Note</em>: “ns” indicates that the <em>p</em>‐value of the correlation is &gt;0.05.</p></div></div></section><p>Among the five QTLs mapped into genes annotated on the Chen et al. (<a href="#tpg270085-bib-0011" class="usa-link" aria-describedby="tpg270085-bib-0011">2020</a>) <em>M. sativa</em> genome, three were in exons (chr8_83798444, chr2_1220775, chr8_80205951) and two in introns (chr8_80198129, chr8_77736763). The sixth QTL was located very closely to a gene at 159 bp downstream only (Table <a href="#tpg270085-tbl-0002" class="usa-link">2</a>). The three SNPs in coding sequences generated an amino acid shift, with unknown effect on protein activity. As four independent QTLs were located at the bottom of chromosome 8 within 6 million bases, this region was investigated in more details. The gene MS.gene36672 (SNP chr8_77736763) had no homolog on either homologous chromosomes, and the genes MS.gene37046 and MS.gene37047 (SNPs chr8_80198129 and chr8_80205951, respectively) had similar homologs on chromosomes 8.1 and 8.3 but no homolog on chromosome 8.4. The gene MS.gene05533 (SNP chr8_83798444) was absent from chromosome 8.1 but present on the other two homologous chromosomes. The alignment of the last 7‐million‐base genomic sequences of the four homologs of chromosome 8 revealed large insertions/deletions on this part of the genome (Figure <a href="#tpg270085-supinfo-0001" class="usa-link">S3</a>).</p></section><section id="tpg270085-sec-0120"><h3 class="pmc_sec_title">3.3. Genomic prediction</h3>
<p>With a training population composed of 270 accessions, the predictive ability averaged 0.85 over the 10 iterations, ranging from 0.84 to 0.87. The predictive ability was over 0.75 when the training population comprised more than 89 accessions (Figure <a href="#tpg270085-fig-0005" class="usa-link">5</a>). The increase in the size of the training population increased the predictive ability and decreased its range of variation. The predictive ability tended to stabilize after a training set of 180 accessions.</p>
<figure class="fig xbox font-sm" id="tpg270085-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac50/12326290/1711c3a588d1/TPG2-18-e70085-g001.jpg" loading="lazy" id="jats-graphic-9" height="304" width="600" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270085-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Boxplot for the predictive ability for anthracnose resistance computed on 10 repetitions as a function of the size of the training population. A table showing the average predictive ability for each training population size, along with its standard deviation, is presented in the lower part of the graph.</p></figcaption></figure></section></section><section id="tpg270085-sec-0130"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>The experimental design was based on a set of 417 phenotyped populations (percentage of resistant plants), and among them, 380 were genotyped (allele frequencies). Given that alfalfa is a predominantly cross‐pollinating crop with high levels of heterozygosity, the advantage of working directly on populations is noteworthy. Instead of focusing on individual plants, populations can be multiplied and conserved as seeds. They are immediately available without the need to produce progeny. Additionally, they allow to assess anthracnose resistance as the mean of resistant plants within an accession, which is usually done. In addition, this material can be used to evaluate various traits of agronomic interest, including those measured at the plot level, as shown in previous studies (Julier et al., <a href="#tpg270085-bib-0033" class="usa-link" aria-describedby="tpg270085-bib-0033">2022</a>; Pégard et al., <a href="#tpg270085-bib-0045" class="usa-link" aria-describedby="tpg270085-bib-0045">2023</a>). This approach can be enriched by more accessions, provided that the same markers are obtained and the phenotypic tests are connected with common control varieties.</p>
<section id="tpg270085-sec-0140"><h3 class="pmc_sec_title">4.1. Diversity for anthracnose resistance</h3>
<p>The resistance to anthracnose ranged from 0% to 83% in this set of accessions, covering the variation already described among alfalfa varieties. Old varieties and the landraces were generally susceptible, in accordance with results obtained in official French variety tests when anthracnose emerged (Gondran, <a href="#tpg270085-bib-0021" class="usa-link" aria-describedby="tpg270085-bib-0021">1982</a>). The most resistant accessions were either recent varieties or breeding materials, thus highlighting the selection efforts in breeding. Similarly, in red clover (<em>Trifolium pratense</em>), landraces and ecotypes were mostly susceptible to anthracnose, and resistant accessions were either registered varieties or breeding materials (Frey et al., <a href="#tpg270085-bib-0020" class="usa-link" aria-describedby="tpg270085-bib-0020">2022</a>). Nonetheless, in alfalfa as well as in red clover, a certain percentage of resistant plants was often present, even in susceptible accessions (Frey et al., <a href="#tpg270085-bib-0020" class="usa-link" aria-describedby="tpg270085-bib-0020">2022</a>; Gondran &amp; Mainer‐Casado, <a href="#tpg270085-bib-0022" class="usa-link" aria-describedby="tpg270085-bib-0022">1973</a>; Irwin et al., <a href="#tpg270085-bib-0027" class="usa-link" aria-describedby="tpg270085-bib-0027">1980</a>). These resistant plants facilitated the breeding of resistant varieties, without the need to introduce exotic germplasm, which would have had a negative effect on agronomically important traits such as yield and persistence.</p></section><section id="tpg270085-sec-0150"><h3 class="pmc_sec_title">4.2. Genetic control of anthracnose resistance</h3>
<p>Two major QTLs on chromosome 8 explained 39.0% and 11.3% of the phenotypic variation, and with four additional QTLs with minor effects, 58.5% of the total variation for anthracnose resistance was explained. All the QTLs were located within or close to genes, and three of them included an LRR domain, a typical pattern observed in resistance genes. Three SNPs were located in exons but did not induce a change in amino acid sequence, and the three other SNPs were in noncoding sequences. However, in this study, as in other GWAS, the chance to reveal the causative mutations is low.</p>
<p>Interestingly, the allele frequencies did not cover the full range of variation from 0 to 1 in the set of accessions. For each SNP, it would be interesting to score anthracnose resistance in breeding materials that would have allele frequency close to 0 or 1. However, for the two major QTLs, and more strikingly for chr8_80198129, the anthracnose resistance reached 75% with an allele frequency around 0.50.</p>
<p>Four QTLs were located on chromosome 8, within a region of 6 million bp on the reference genome where the linkage disequilibrium was short (&lt;250 bp), as usually observed in alfalfa (Herrmann et al., <a href="#tpg270085-bib-0025" class="usa-link" aria-describedby="tpg270085-bib-0025">2010</a>; Pégard et al., <a href="#tpg270085-bib-0045" class="usa-link" aria-describedby="tpg270085-bib-0045">2023</a>). However, the allele frequencies of three of the four QTLs were correlated to each other. Two hypotheses can be proposed: (1) a common selection pressure on these four genes conferred anthracnose resistance, thus generating a correlation among QTLs in a region with a low linkage disequilibrium, and (2) the genome assembly (Chen et al., <a href="#tpg270085-bib-0011" class="usa-link" aria-describedby="tpg270085-bib-0011">2020</a>) is poor in this region, probably due to suboptimal assembly strategies available for autotetraploid organisms a few years ago. This exacerbates the structural variations observed when comparing the sequences of the four homologous chromosomes 8.</p>
<p>Surprisingly, some accessions inhibited a high level of resistance, in spite of having non‐favorable alleles at the two major QTLs for anthracnose resistance, which suggested that, in addition to the QTL identified in this study, other QTLs not detected in this study were probably involved in anthracnose resistance. Several reasons may explain this situation: (1) as the marker density was not extremely high, especially when considering the short LD, some genomic regions were not detected; (2) these cases of unexpected resistance were related to rare alleles that were excluded during the SNP trimming process; and (3) errors in phenotyping and/or genotyping.</p>
<p>The genetic control of resistance to anthracnose pathotype 1 was described to be mainly controlled by a single gene (Elgin &amp; Ostazeski, <a href="#tpg270085-bib-0017" class="usa-link" aria-describedby="tpg270085-bib-0017">1985</a>) in alfalfa. With molecular markers, the role of a major gene was shown in both <em>M. truncatula</em> and <em>M. sativa</em> (Ameline‐Torregrosa, Cazaux, et al., <a href="#tpg270085-bib-0001" class="usa-link" aria-describedby="tpg270085-bib-0001">2008</a>; Irwin et al., <a href="#tpg270085-bib-0026" class="usa-link" aria-describedby="tpg270085-bib-0026">2006</a>; Mackie et al., <a href="#tpg270085-bib-0038" class="usa-link" aria-describedby="tpg270085-bib-0038">2007</a>; Yang et al., <a href="#tpg270085-bib-0057" class="usa-link" aria-describedby="tpg270085-bib-0057">2007</a>). The region mainly controlling anthracnose resistance was at the bottom of chromosome 8 in our study, with two major QTLs located 3.5 million bases apart and two minor QTLs. Mackie et al. (<a href="#tpg270085-bib-0038" class="usa-link" aria-describedby="tpg270085-bib-0038">2007</a>) observed a major QTL at the top of chromosome 8 in an alfalfa mapping population. This region seems to be homologous to the positions of the two major QTLs found in our GWAS study, after considering the inversion of the reference genome and the genetic map (Figure <a href="#tpg270085-fig-0001" class="usa-link">1</a>). Even if the genome portions apart from the two major QTLs contained some genes with LRR and TIR domains, this chromosomal arm was not recognized as a region that was rich in NBS‐LRR genes (Ameline‐Torregrosa, Wang, et al., <a href="#tpg270085-bib-0002" class="usa-link" aria-describedby="tpg270085-bib-0002">2008</a>). We also found a minor QTL at the bottom of chromosome 4, corresponding to the top of chromosome 4 of <em>M. truncatula</em> (Figure <a href="#tpg270085-fig-0001" class="usa-link">1</a>). In <em>M. truncatula</em>, the main QTL region explaining 33%–43% of the variation was located at the top of chromosome 4 (Ameline‐Torregrosa, Cazaux, et al., <a href="#tpg270085-bib-0001" class="usa-link" aria-describedby="tpg270085-bib-0001">2008</a>; Yang et al., <a href="#tpg270085-bib-0057" class="usa-link" aria-describedby="tpg270085-bib-0057">2007</a>) and controlled resistance against pathotypes 1, 2, and C86‐2, the latter being also used in the present study. At the top of chromosome 4, a QTL explaining 7%–8% of the variation was found in alfalfa (Irwin et al., <a href="#tpg270085-bib-0026" class="usa-link" aria-describedby="tpg270085-bib-0026">2006</a>; Mackie et al., <a href="#tpg270085-bib-0038" class="usa-link" aria-describedby="tpg270085-bib-0038">2007</a>), possibly in the same region. Our QTL was located about 10 million bp from the two major QTLs <em>RCT1</em> and <em>Ct1</em> in <em>M. truncatula</em>. In addition, we found a minor QTL on chromosome 2, not described in the literature. A simple genetic inheritance, conferred by a major genome region and modified by minor QTL, was thus evidenced, in accordance with previous studies. However, the genetic control seemed to be different between alfalfa and <em>M. truncatula</em>. A simple genetic control was hypothesized for anthracnose resistance in red clover too (Jacob et al., <a href="#tpg270085-bib-0028" class="usa-link" aria-describedby="tpg270085-bib-0028">2015</a>). A GWAS conducted in red clover revealed a major QTL at the top of chromosome 1 on the reference red clover genome of De Vega et al. (<a href="#tpg270085-bib-0013" class="usa-link" aria-describedby="tpg270085-bib-0013">2015</a>), explaining 16.8% of the variation, in addition to QTL of less importance (Frey et al., <a href="#tpg270085-bib-0020" class="usa-link" aria-describedby="tpg270085-bib-0020">2022</a>). This QTL was located in a gene that was annotated as a 3‐oxoacyl‐(acyl carrier) synthase II (De Vega et al., <a href="#tpg270085-bib-0013" class="usa-link" aria-describedby="tpg270085-bib-0013">2015</a>). Given these results, it seems that the control of anthracnose resistance in red clover is possibly different than that in alfalfa.</p></section><section id="tpg270085-sec-0160"><h3 class="pmc_sec_title">4.3. Genomic prediction</h3>
<p>The predictive ability of our genomic selection reached 0.85, which is likely the highest achieved in alfalfa to date, even higher than obtained for autumn dormancy on the same material (Pégard et al., <a href="#tpg270085-bib-0045" class="usa-link" aria-describedby="tpg270085-bib-0045">2023</a>). Similar to this earlier publication, with 89 accessions and beyond, the mean predictive ability plateaued, but the range of variation was reduced by including a larger number of accessions in the training populations. These exceptional results might be explained by the presence of six QTLs, which account for nearly 60% of the phenotypic variation. This study is the first to demonstrate genomic prediction for anthracnose resistance in alfalfa, complementing numerous studies that predict the feasibility of genomic prediction in this species.</p>
<p>In this study, we chose to test the performance of GBLUP exclusively because with the same population, we found that Lasso and GBLUP performed similarly for autumn dormancy, even when major QTLs were detected (Pégard et al., <a href="#tpg270085-bib-0045" class="usa-link" aria-describedby="tpg270085-bib-0045">2023</a>). This might be explained by an infinitesimal genetic architecture, or the presence of QTL can significantly influence predictive ability, as seen in the study where six QTLs explained 60% of the phenotypic variation, leading to high predictive ability even with a small number of accessions in the training set.</p>
<p>In addition to the effect of the method, previous research indicated that predictive accuracies can vary significantly depending on the populations and the traits, with forage quality showing the narrowest range (0.30–0.40) and biomass yield showing the widest range (0.20–0.80) (Annicchiarico et al., <a href="#tpg270085-bib-0004" class="usa-link" aria-describedby="tpg270085-bib-0004">2015</a>, <a href="#tpg270085-bib-0005" class="usa-link" aria-describedby="tpg270085-bib-0005">2021</a>, <a href="#tpg270085-bib-0003" class="usa-link" aria-describedby="tpg270085-bib-0003">2022</a>; X.‐Q. Jiang et al., <a href="#tpg270085-bib-0030" class="usa-link" aria-describedby="tpg270085-bib-0030">2022</a>; Murad Leite Andrade et al., <a href="#tpg270085-bib-0040" class="usa-link" aria-describedby="tpg270085-bib-0040">2022</a>; Pégard et al., <a href="#tpg270085-bib-0045" class="usa-link" aria-describedby="tpg270085-bib-0045">2023</a>). Overall, these results contribute to the growing body of research demonstrating the feasibility and potential of genomic prediction in alfalfa but highlight the need for continued exploration and optimization of genomic prediction models.</p></section><section id="tpg270085-sec-0170"><h3 class="pmc_sec_title">4.4. Consequences for breeding</h3>
<p>For future breeding efforts in alfalfa, the use of molecular markers could help to increase anthracnose resistance. We propose two strategies, which can be combined. First, from the two major QTLs on chromosome 8, an analysis of the candidate gene polymorphism could be conducted, and the correlation between such polymorphism and phenotypic variation could help to identify causal mutation in the genes. With the development of high‐throughput markers, such as KASP markers, polycrosses composed of plants carrying three to four doses of the allele conferring the resistance would produce highly resistant progeny. Second, a genomic prediction could be conducted on relevant breeding pools, and the plants predicted to be the most resistant ones could be intercrossed. Both strategies could be combined to select plants carrying the anthracnose resistance alleles at the two major QTLs and plants that have the highest genomic prediction values. In the case of anthracnose resistance that is relatively easy to score, marker‐assisted selection may not be economically feasible. However, given the complex inheritance due to tetraploidy in alfalfa, molecular markers could be the only way to fix resistance alleles in breeding populations. Moreover, the number of studies showing promising QTL or genomic prediction results increases, and genomic selection allows to improve multiple traits simultaneously, thus speeding up breeding.</p>
<p>Results obtained with GWAS and genomic prediction are always associated with the set of accessions. Considering that susceptible accessions were over‐represented in our set, and the most resistant accessions mostly originated from the United States, it would be valuable to introduce more resistant accessions from diverse origins, if available. If the new accessions are phenotyped with the same protocol as the one used here and genotyped with the same marker set, it would be possible to check if the composition of the set affected the QTL detection and the genomic prediction in our study.</p>
<p>Breeding alfalfa for disease resistance is of major importance to increase protein‐rich forage production and stand persistency, thus contributing to lower agricultural inputs. So far, the varieties that were scored as anthracnose resistant in the past have maintained their resistance to the same pathotypes, as if the resistance genes had no significant effect on the pathogen evolution. In the future, if by using molecular markers alfalfa varieties containing 100% resistant plants would be created, we could expect the evolution of the <em>C. trifolii</em> pathogen toward pathotypes that overcome this resistance, as often described in crops grown as homogeneous varieties. This risk was mentioned in the past (Guy, <a href="#tpg270085-bib-0024" class="usa-link" aria-describedby="tpg270085-bib-0024">1976</a>; Raynal, <a href="#tpg270085-bib-0047" class="usa-link" aria-describedby="tpg270085-bib-0047">1977</a>). A solution could be to create alfalfa varieties with a small percentage of susceptible plants while maintaining the low selection pressure on the pathogen.</p></section></section><section id="tpg270085-sec-0180"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Marie Pégard</strong>: Data curation; formal analysis; investigation; methodology; validation; writing—original draft; writing—review and editing. <strong>Camille Gréard</strong>: Data curation; formal analysis; investigation; methodology; writing—original draft; writing—review and editing. <strong>Marius Grelier</strong>: Data curation; formal analysis; visualization; writing—review and editing. <strong>Marie‐Christine Gras</strong>: Conceptualization; investigation; methodology; writing—review and editing. <strong>Laure Saint‐Pierre</strong>: Investigation; methodology; validation; writing—review and editing. <strong>Bernard Tharel</strong>: Conceptualization; investigation; methodology; writing—review and editing. <strong>Philippe Barre</strong>: Conceptualization; methodology; writing—review and editing. <strong>Bernadette Julier</strong>: Conceptualization; funding acquisition; methodology; project administration; supervision; writing—original draft; writing—review and editing.</p></section><section id="tpg270085-sec-0200"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec20"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270085-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270085-supitem-0001"><div class="caption p">
<p>Supplemental Figure S1. QQplot for the QTL detection, showing six significant SNP.</p>
<p>Supplemental Figure S2. Correlation between allele frequencies of chr8_83798444 (y‐axis) and chr8_80205951 and between chr8_80198129 and chr8_80205951. The points, representing the accessions, are colored depending on their percentage of resistance, from red for a low percentage of resistance to green for a high percentage of resistance.</p>
<p>Supplemental Figure S3. Linkage disequilibrium on chromosome 8 within a distance of 4 million base pairs, in the region around SNP chr8_80198129.</p>
<p>Supplemental Figure S4. Alignment of the homologous chromosomes 8 in the region of the QTL, with chr8.2, used as the reference for GBS mapping, compared to the other three chromosomes. Identity legend: 0‐0.25: yellow, 0.25‐0.5: orange, 0.5‐0.75 light green, 0.75‐1: dark green.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326290/bin/TPG2-18-e70085-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70085-s001.docx</a><sup> (385KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270085-supitem-0002"><div class="caption p"><p>Supplemental Table S1: list of alfalfa accessions with their country of origin, status, registration year when available, and adjusted resistance to anthracnose, pathotype C86‐2.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326290/bin/TPG2-18-e70085-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70085-s002.xlsx</a><sup> (35.1KB, xlsx) </sup>
</div></div></section></section><section id="tpg270085-sec-0190" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank Geves for providing us with data of anthracnose resistance for varieties of the alfalfa reference collection. We also thank the technicians who phenotyped the accessions at Barenbrug, Cérience, and RAGT 2n. The genotyping work has been done by the INRAE staff in Lusignan (URP3F) and Toulouse (GetPlage).</p></section><section id="notes1"><p>


Pégard, M.
, 
Gréard, C.
, 
Grelier, M.
, 
Gras, M.‐C.
, 
Saint‐Pierre, L.
, 
Tharel, B.
, 
Barre, P.
, &amp; 
Julier, B.
 (2025). QTL detection and genomic prediction for resistance to anthracnose in alfalfa (<em>Medicago sativa</em>). The Plant Genome, 18, e70085. 10.1002/tpg2.70085


</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270085-note-0001"><p>Assigned to Associate Editor Steve Larson.</p></div></div></section></section><section id="tpg270085-sec-0220"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The genotyping data are available at <a href="https://doi.org/10.57745/L0FLJD" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.57745/L0FLJD</a>. The phenotyping data are available in the Table <a href="#tpg270085-supinfo-0001" class="usa-link">S1</a>.</p></section><section id="tpg270085-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270085-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270085-bib-0001">
<cite>

Ameline‐Torregrosa, C.
, 
Cazaux, M.
, 
Danesh, D.
, 
Chardon, F.
, 
Cannon, S. B.
, 
Esquerré‐Tugayé, M.‐T.
, 
Dumas, B.
, 
Young, N. D.
, 
Samac, D. A.
, &amp; 
Huguet, T.
, 
Jacquet, C.
 (2008). Genetic dissection of resistance to anthracnose and powdery mildew in <em>Medicago truncatula</em>

Molecular Plant‐Microbe Interactions, 21(1), 61–69. 10.1094/MPMI-21-1-0061

</cite> [<a href="https://doi.org/10.1094/MPMI-21-1-0061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18052883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant%E2%80%90Microbe%20Interactions&amp;title=Genetic%20dissection%20of%20resistance%20to%20anthracnose%20and%20powdery%20mildew%20in%20Medicago%20truncatula&amp;author=C.%20Ameline%E2%80%90Torregrosa&amp;author=M.%20Cazaux&amp;author=D.%20Danesh&amp;author=F.%20Chardon&amp;author=S.%20B.%20Cannon&amp;volume=21&amp;issue=1&amp;publication_year=2008&amp;pages=61-69&amp;pmid=18052883&amp;doi=10.1094/MPMI-21-1-0061&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0002">
<cite>

Ameline‐Torregrosa, C.
, 
Wang, B.‐B.
, 
O'Bleness, M. S.
, 
Deshpande, S.
, 
Zhu, H.
, 
Roe, B.
, 
Young, N. D.
, &amp; 
Cannon, S. B.
 (2008). Identification and characterization of nucleotide‐binding site‐Leucine‐rich repeat genes in the model plant <em>Medicago truncatula</em>
. Plant Physiology, 146(1), 5–21. 10.1104/pp.107.104588

</cite> [<a href="https://doi.org/10.1104/pp.107.104588" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2230567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17981990/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Identification%20and%20characterization%20of%20nucleotide%E2%80%90binding%20site%E2%80%90Leucine%E2%80%90rich%20repeat%20genes%20in%20the%20model%20plant%20Medicago%20truncatula&amp;author=C.%20Ameline%E2%80%90Torregrosa&amp;author=B.%E2%80%90B.%20Wang&amp;author=M.%20S.%20O'Bleness&amp;author=S.%20Deshpande&amp;author=H.%20Zhu&amp;volume=146&amp;issue=1&amp;publication_year=2008&amp;pages=5-21&amp;pmid=17981990&amp;doi=10.1104/pp.107.104588&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0003">
<cite>

Annicchiarico, P.
, 
Nazzicari, N.
, 
Bouizgaren, A.
, 
Hayek, T.
, 
Laouar, M.
, 
Cornacchione, M.
, 
Basigalup, D.
, 
Monterrubio Martin, C.
, 
Brummer, E. C.
, &amp; 
Pecetti, L.
 (2022). Alfalfa genomic selection for different stress‐prone growing regions. The Plant Genome, 15(4), Article e20264. 10.1002/tpg2.20264

</cite> [<a href="https://doi.org/10.1002/tpg2.20264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36222346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Alfalfa%20genomic%20selection%20for%20different%20stress%E2%80%90prone%20growing%20regions&amp;author=P.%20Annicchiarico&amp;author=N.%20Nazzicari&amp;author=A.%20Bouizgaren&amp;author=T.%20Hayek&amp;author=M.%20Laouar&amp;volume=15&amp;issue=4&amp;publication_year=2022&amp;pages=e20264&amp;pmid=36222346&amp;doi=10.1002/tpg2.20264&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0004">
<cite>

Annicchiarico, P.
, 
Nazzicari, N.
, 
Li, X. H.
, 
Wei, Y. L.
, 
Pecetti, L.
, &amp; 
Brummer, E. C.
 (2015). Accuracy of genomic selection for alfalfa biomass yield in different reference populations. BMC Genomics, 16, Article 1020. 10.1186/s12864-015-2212-y

</cite> [<a href="https://doi.org/10.1186/s12864-015-2212-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4667460/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26626170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Accuracy%20of%20genomic%20selection%20for%20alfalfa%20biomass%20yield%20in%20different%20reference%20populations&amp;author=P.%20Annicchiarico&amp;author=N.%20Nazzicari&amp;author=X.%20H.%20Li&amp;author=Y.%20L.%20Wei&amp;author=L.%20Pecetti&amp;volume=16&amp;publication_year=2015&amp;pages=1020&amp;pmid=26626170&amp;doi=10.1186/s12864-015-2212-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0005">
<cite>

Annicchiarico, P.
, 
Nazzicari, N.
, &amp; 
Pecetti, L.
 (2021). Genomic selection for higher yield and quality in alfalfa. In Yu L. X. &amp; Kole C. (Eds.), The alfalfa genome (pp. 191–210). Springer. 10.1007/978-3-030-74466-3_12
</cite> [<a href="https://doi.org/10.1007/978-3-030-74466-3_12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=The%20alfalfa%20genome&amp;author=P.%20Annicchiarico&amp;author=N.%20Nazzicari&amp;author=L.%20Pecetti&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0006">
<cite>

Ashraf, B. H.
, 
Jensen, J.
, 
Asp, T.
, &amp; 
Janss, L. L.
 (2014). Association studies using family pools of outcrossing crops based on allele‐frequency estimates from DNA sequencing. Theoretical and Applied Genetics, 127(6), 1331–1341. 10.1007/s00122-014-2300-4

</cite> [<a href="https://doi.org/10.1007/s00122-014-2300-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4035547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24668443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Association%20studies%20using%20family%20pools%20of%20outcrossing%20crops%20based%20on%20allele%E2%80%90frequency%20estimates%20from%20DNA%20sequencing&amp;author=B.%20H.%20Ashraf&amp;author=J.%20Jensen&amp;author=T.%20Asp&amp;author=L.%20L.%20Janss&amp;volume=127&amp;issue=6&amp;publication_year=2014&amp;pages=1331-1341&amp;pmid=24668443&amp;doi=10.1007/s00122-014-2300-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0007">
<cite>

Barnes, D. K.
, 
Ostazeski, S. A.
, 
Schillinger, J. A.
, &amp; 
Hanson, C. H.
 (1969). Effect of anthracnose (<em>Colletotrichum trifolii</em>) infection on yield, stand, and vigor of alfalfa. Crop Science, 9(3), 344–346. 10.2135/cropsci1969.0011183x000900030029x
</cite> [<a href="https://doi.org/10.2135/cropsci1969.0011183x000900030029x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Effect%20of%20anthracnose%20(Colletotrichum%20trifolii)%20infection%20on%20yield,%20stand,%20and%20vigor%20of%20alfalfa&amp;author=D.%20K.%20Barnes&amp;author=S.%20A.%20Ostazeski&amp;author=J.%20A.%20Schillinger&amp;author=C.%20H.%20Hanson&amp;volume=9&amp;issue=3&amp;publication_year=1969&amp;pages=344-346&amp;doi=10.2135/cropsci1969.0011183x000900030029x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0008">
<cite>

Busbice, T. H.
, &amp; 
Wilsie, C. P.
 (1966). Inbreeding depression and heterosis in autotetraploids with application to <em>Medicago sativa</em> L. Euphytica, 15(1), 52–67. 10.1007/BF00024079
</cite> [<a href="https://doi.org/10.1007/BF00024079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Inbreeding%20depression%20and%20heterosis%20in%20autotetraploids%20with%20application%20to%20Medicago%20sativa%20L&amp;author=T.%20H.%20Busbice&amp;author=C.%20P.%20Wilsie&amp;volume=15&amp;issue=1&amp;publication_year=1966&amp;pages=52-67&amp;doi=10.1007/BF00024079&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0009">
<cite>

Cabanettes, F.
, &amp; 
Klopp, C.
 (2018). D‐GENIES: dot plot large genomes in an interactive, efficient and simple way. PeerJ, 6, Article e4958. 10.7717/peerj.4958

</cite> [<a href="https://doi.org/10.7717/peerj.4958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5991294/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29888139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=D%E2%80%90GENIES:%20dot%20plot%20large%20genomes%20in%20an%20interactive,%20efficient%20and%20simple%20way&amp;author=F.%20Cabanettes&amp;author=C.%20Klopp&amp;volume=6&amp;publication_year=2018&amp;pages=e4958&amp;pmid=29888139&amp;doi=10.7717/peerj.4958&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0010">
<cite>

Cericola, F.
, 
Lenk, I.
, 
Fè, D.
, 
Byrne, S.
, 
Jensen, C. S.
, 
Pedersen, M. G.
, 
Asp, T.
, 
Jensen, J.
, &amp; 
Janss, L.
 (2018). Optimized use of low‐depth genotyping‐by‐sequencing for genomic prediction among multi‐parental family pools and single plants in perennial ryegrass (<em>Lolium perenne</em> L.). Frontiers in Plant Science, 9, Article 369. 10.3389/fpls.2018.00369

</cite> [<a href="https://doi.org/10.3389/fpls.2018.00369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5871745/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29619038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Optimized%20use%20of%20low%E2%80%90depth%20genotyping%E2%80%90by%E2%80%90sequencing%20for%20genomic%20prediction%20among%20multi%E2%80%90parental%20family%20pools%20and%20single%20plants%20in%20perennial%20ryegrass%20(Lolium%20perenne%20L.)&amp;author=F.%20Cericola&amp;author=I.%20Lenk&amp;author=D.%20F%C3%A8&amp;author=S.%20Byrne&amp;author=C.%20S.%20Jensen&amp;volume=9&amp;publication_year=2018&amp;pages=369&amp;pmid=29619038&amp;doi=10.3389/fpls.2018.00369&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0011">
<cite>

Chen, H.
, 
Zeng, Y.
, 
Yang, Y.
, 
Huang, L.
, 
Tang, B.
, 
Zhang, H.
, 
Hao, F.
, 
Liu, W.
, 
Li, Y.
, 
Liu, Y.
, 
Zhang, X.
, 
Zhang, R.
, 
Zhang, Y.
, 
Li, Y.
, 
Wang, K.
, 
He, H.
, 
Wang, Z.
, 
Fan, G.
, 
Yang, H.
, … 
Qiu, Q.
 (2020). Allele‐aware chromosome‐level genome assembly and efficient transgene‐free genome editing for the autotetraploid cultivated alfalfa. Nature Communications, 11(1), Article 2494. 10.1038/s41467-020-16338-x
</cite> [<a href="https://doi.org/10.1038/s41467-020-16338-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7237683/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32427850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Allele%E2%80%90aware%20chromosome%E2%80%90level%20genome%20assembly%20and%20efficient%20transgene%E2%80%90free%20genome%20editing%20for%20the%20autotetraploid%20cultivated%20alfalfa&amp;author=H.%20Chen&amp;author=Y.%20Zeng&amp;author=Y.%20Yang&amp;author=L.%20Huang&amp;author=B.%20Tang&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=2494&amp;pmid=32427850&amp;doi=10.1038/s41467-020-16338-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0012">
<cite>

Clements, R. J.
, 
Turner, J. W.
, 
Irwin, J. A. G.
, 
Langdon, P. W.
, &amp; 
Bray, R. A.
 (1984). Breeding disease resistant, aphid resistant lucerne for sub‐tropical Queensland. Australian Journal of Experimental Agriculture, 24(125), 178–188. 10.1071/EA9840178
</cite> [<a href="https://doi.org/10.1071/EA9840178" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Australian%20Journal%20of%20Experimental%20Agriculture&amp;title=Breeding%20disease%20resistant,%20aphid%20resistant%20lucerne%20for%20sub%E2%80%90tropical%20Queensland&amp;author=R.%20J.%20Clements&amp;author=J.%20W.%20Turner&amp;author=J.%20A.%20G.%20Irwin&amp;author=P.%20W.%20Langdon&amp;author=R.%20A.%20Bray&amp;volume=24&amp;issue=125&amp;publication_year=1984&amp;pages=178-188&amp;doi=10.1071/EA9840178&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0013">
<cite>

De Vega, J. J.
, 
Ayling, S.
, 
Hegarty, M.
, 
Kudrna, D.
, 
Goicoechea, J. L.
, 
Ergon, Å.
, 
Rognli, O. A.
, 
Jones, C.
, 
Swain, M.
, 
Geurts, R.
, 
Lang, C.
, 
Mayer, K. F. X.
, 
Rössner, S.
, 
Yates, S.
, 
Webb, K. J.
, 
Donnison, I. S.
, 
Oldroyd, G. E. D.
, 
Wing, R. A.
, 
Caccamo, M.
, … 
Skøt, L.
 (2015). Red clover (<em>Trifolium pratense</em> L.) draft genome provides a platform for trait improvement. Scientific Reports, 5, Article 17394. 10.1038/srep17394

</cite> [<a href="https://doi.org/10.1038/srep17394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4663792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26617401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Red%20clover%20(Trifolium%20pratense%20L.)%20draft%20genome%20provides%20a%20platform%20for%20trait%20improvement&amp;author=J.%20J.%20De%20Vega&amp;author=S.%20Ayling&amp;author=M.%20Hegarty&amp;author=D.%20Kudrna&amp;author=J.%20L.%20Goicoechea&amp;volume=5&amp;publication_year=2015&amp;pages=17394&amp;pmid=26617401&amp;doi=10.1038/srep17394&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0014">
<cite>

Devine, T. E.
, 
Hanson, C. H.
, 
Ostazeski, S. A.
, &amp; 
Campbell, T. A.
 (1971). Selection for resistance to anthracnose (<em>Colletotrichum trifolii</em>) in 4 alfalfa populations. Crop Science, 11(6), 854–855. 10.2135/cropsci1971.0011183x001100060026x
</cite> [<a href="https://doi.org/10.2135/cropsci1971.0011183x001100060026x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Selection%20for%20resistance%20to%20anthracnose%20(Colletotrichum%20trifolii)%20in%204%20alfalfa%20populations&amp;author=T.%20E.%20Devine&amp;author=C.%20H.%20Hanson&amp;author=S.%20A.%20Ostazeski&amp;author=T.%20A.%20Campbell&amp;volume=11&amp;issue=6&amp;publication_year=1971&amp;pages=854-855&amp;doi=10.2135/cropsci1971.0011183x001100060026x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0015">
<cite>

Dubeux, J. C. B.
Jr.
, 
Jaramillo, D. M.
, 
Santos, E. R. S.
, 
Garcia, L.
, 
Queiroz, L. M. D.
, 
Bretas, I. L.
, &amp; 
Trumpp, K. R.
 (2024). Sustainable intensification of livestock systems using forage legumes in the Anthropocene. Grass and Forage Science, 79(3), 481–498. 10.1111/gfs.12696
</cite> [<a href="https://doi.org/10.1111/gfs.12696" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Grass%20and%20Forage%20Science&amp;title=Sustainable%20intensification%20of%20livestock%20systems%20using%20forage%20legumes%20in%20the%20Anthropocene&amp;author=J.%20C.%20B.%20Dubeux&amp;author=D.%20M.%20Jaramillo&amp;author=E.%20R.%20S.%20Santos&amp;author=L.%20Garcia&amp;author=L.%20M.%20D.%20Queiroz&amp;volume=79&amp;issue=3&amp;publication_year=2024&amp;pages=481-498&amp;doi=10.1111/gfs.12696&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0016">
<cite>

Elgin, J. H.
, 
Barnes, D. K.
, 
Busbice, T. H.
, 
Buss, G. R.
, 
Clark, N. A.
, 
Cleveland, R. W.
, 
Ditterline, R. L.
, 
Evans, D. W.
, 
Fransen, S. C.
, 
Horrocks, R. D.
, 
Hunt, O. J.
, 
Kehr, W. R.
, 
Lowe, C. C.
, 
Miller, D. A.
, 
Offutt, M. S.
, 
Pickett, R. C.
, 
Sorensen, E. L.
, 
Taliaferro, C. M.
, 
Tesar, M. B.
, &amp; 
Van Keuren, R. W.
 (1981). Anthracnose resistance increases alfalfa yields. Crop Science, 21(3), 457–460. 10.2135/cropsci1981.0011183x002100030026x
</cite> [<a href="https://doi.org/10.2135/cropsci1981.0011183x002100030026x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Anthracnose%20resistance%20increases%20alfalfa%20yields&amp;author=J.%20H.%20Elgin&amp;author=D.%20K.%20Barnes&amp;author=T.%20H.%20Busbice&amp;author=G.%20R.%20Buss&amp;author=N.%20A.%20Clark&amp;volume=21&amp;issue=3&amp;publication_year=1981&amp;pages=457-460&amp;doi=10.2135/cropsci1981.0011183x002100030026x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0017">
<cite>

Elgin, J. H.
Jr.
, &amp; 
Ostazeski, S. A.
 (1985). Inheritance of resistance to race 1 and race 2 anthracnose in Arc and Saranac AR alfalfa. Crop Science, 25(5), 861–865. 10.2135/cropsci1985.0011183x002500050032x
</cite> [<a href="https://doi.org/10.2135/cropsci1985.0011183x002500050032x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Inheritance%20of%20resistance%20to%20race%201%20and%20race%202%20anthracnose%20in%20Arc%20and%20Saranac%20AR%20alfalfa&amp;author=J.%20H.%20Elgin&amp;author=S.%20A.%20Ostazeski&amp;volume=25&amp;issue=5&amp;publication_year=1985&amp;pages=861-865&amp;doi=10.2135/cropsci1985.0011183x002500050032x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0018">
<cite>

Flajoulot, S.
, 
Ronfort, J.
, 
Baudouin, P.
, 
Barre, P.
, 
Huguet, T.
, 
Huyghe, C.
, &amp; 
Julier, B.
 (2005). Genetic diversity among alfalfa (<em>Medicago sativa</em>) cultivars coming from a single breeding program, using SSR markers. Theoretical and Applied Genetics, 111, 1420–1429. 10.1007/s00122-005-0074-4

</cite> [<a href="https://doi.org/10.1007/s00122-005-0074-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16151797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genetic%20diversity%20among%20alfalfa%20(Medicago%20sativa)%20cultivars%20coming%20from%20a%20single%20breeding%20program,%20using%20SSR%20markers&amp;author=S.%20Flajoulot&amp;author=J.%20Ronfort&amp;author=P.%20Baudouin&amp;author=P.%20Barre&amp;author=T.%20Huguet&amp;volume=111&amp;publication_year=2005&amp;pages=1420-1429&amp;pmid=16151797&amp;doi=10.1007/s00122-005-0074-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0019">
<cite>

Flint‐Garcia, S. A.
, 
Thornsberry, J. M.
, &amp; 
Buckler, E. S.
IV.
 (2003). Structure of linkage disequilibrium in plants. Annual Review of Plant Biology, 54, 357–374. 10.1146/annurev.arplant.54.031902.134907
</cite> [<a href="https://doi.org/10.1146/annurev.arplant.54.031902.134907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14502995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Plant%20Biology&amp;title=Structure%20of%20linkage%20disequilibrium%20in%20plants&amp;author=S.%20A.%20Flint%E2%80%90Garcia&amp;author=J.%20M.%20Thornsberry&amp;author=E.%20S.%20Buckler&amp;volume=54&amp;publication_year=2003&amp;pages=357-374&amp;pmid=14502995&amp;doi=10.1146/annurev.arplant.54.031902.134907&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0020">
<cite>

Frey, L. A.
, 
Vleugels, T.
, 
Ruttink, T.
, 
Schubiger, F. X.
, 
Pégard, M.
, 
Skøt, L.
, 
Grieder, C.
, 
Studer, B.
, 
Roldán‐Ruiz, I.
, &amp; 
Kölliker, R.
 (2022). Phenotypic variation and quantitative trait loci for resistance to southern anthracnose and clover rot in red clover. Theoretical and Applied Genetics, 135, 4337–4349. 10.1007/s00122-022-04223-8

</cite> [<a href="https://doi.org/10.1007/s00122-022-04223-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9734235/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36153770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Phenotypic%20variation%20and%20quantitative%20trait%20loci%20for%20resistance%20to%20southern%20anthracnose%20and%20clover%20rot%20in%20red%20clover&amp;author=L.%20A.%20Frey&amp;author=T.%20Vleugels&amp;author=T.%20Ruttink&amp;author=F.%20X.%20Schubiger&amp;author=M.%20P%C3%A9gard&amp;volume=135&amp;publication_year=2022&amp;pages=4337-4349&amp;pmid=36153770&amp;doi=10.1007/s00122-022-04223-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0021">
<cite>

Gondran, J.
 (1982). Résistance à l'anthracnose des cultivars de luzerne inscrits au catalogue français. Fourrages, 91, 99–103.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Fourrages&amp;title=R%C3%A9sistance%20%C3%A0%20l'anthracnose%20des%20cultivars%20de%20luzerne%20inscrits%20au%20catalogue%20fran%C3%A7ais&amp;author=J.%20Gondran&amp;volume=91&amp;publication_year=1982&amp;pages=99-103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0022">
<cite>

Gondran, J.
, &amp; 
Mainer‐Casado, A.
 (1973). Inoculation artificielle de la luzerne avec <em>Colletotrichum trifolii</em> Bain et Essary. Classement de 19 cultivars de lucerne suivant leur résistance au parasite. Annales de l'Amélioration des Plantes, 23(4), 367–379.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Annales%20de%20l'Am%C3%A9lioration%20des%20Plantes&amp;title=Inoculation%20artificielle%20de%20la%20luzerne%20avec%20Colletotrichum%20trifolii%20Bain%20et%20Essary.%20Classement%20de%2019%20cultivars%20de%20lucerne%20suivant%20leur%20r%C3%A9sistance%20au%20parasite&amp;author=J.%20Gondran&amp;author=A.%20Mainer%E2%80%90Casado&amp;volume=23&amp;issue=4&amp;publication_year=1973&amp;pages=367-379&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0023">
<cite>

Gosset, H.
, 
Trœung, B. M.
, 
Raynal, G.
, 
Huet, P.
, 
Boukhari, N.
, 
Mesbah, F.‐Z.
, &amp; 
Ferrari, F.
 (1989). Resistance of alfalfa (<em>Medicago sativa</em> L.) to isolates of <em>Colletotrichum trifolii</em> (bain et essary) from the Mediterranean basin, especially from Morocco. Agronomie, 9(5), 495–501. 10.1051/agro:19890508
</cite> [<a href="https://doi.org/10.1051/agro:19890508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomie&amp;title=Resistance%20of%20alfalfa%20(Medicago%20sativa%20L.)%20to%20isolates%20of%20Colletotrichum%20trifolii%20(bain%20et%20essary)%20from%20the%20Mediterranean%20basin,%20especially%20from%20Morocco&amp;author=H.%20Gosset&amp;author=B.%20M.%20Tr%C5%93ung&amp;author=G.%20Raynal&amp;author=P.%20Huet&amp;author=N.%20Boukhari&amp;volume=9&amp;issue=5&amp;publication_year=1989&amp;pages=495-501&amp;doi=10.1051/agro:19890508&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0024">
<cite>

Guy, P.
 (1976). Déterminisme génétique de la résistance à <em>Colletotrichum trifolii</em> chez <em>Medicago sativa</em> L. Annales de l'Amélioration des Plantes, 26(2), 215–234.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Annales%20de%20l'Am%C3%A9lioration%20des%20Plantes&amp;title=D%C3%A9terminisme%20g%C3%A9n%C3%A9tique%20de%20la%20r%C3%A9sistance%20%C3%A0%20Colletotrichum%20trifolii%20chez%20Medicago%20sativa%20L&amp;author=P.%20Guy&amp;volume=26&amp;issue=2&amp;publication_year=1976&amp;pages=215-234&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0025">
<cite>

Herrmann, D.
, 
Barre, P.
, 
Santoni, S.
, &amp; 
Julier, B.
 (2010). Association of a <em>CONSTANS‐LIKE</em> gene to flowering and height in autotetraploid alfalfa. Theoretical and Applied Genetics, 121(5), 865–876. 10.1007/s00122-010-1356-z

</cite> [<a href="https://doi.org/10.1007/s00122-010-1356-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20473652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Association%20of%20a%20CONSTANS%E2%80%90LIKE%20gene%20to%20flowering%20and%20height%20in%20autotetraploid%20alfalfa&amp;author=D.%20Herrmann&amp;author=P.%20Barre&amp;author=S.%20Santoni&amp;author=B.%20Julier&amp;volume=121&amp;issue=5&amp;publication_year=2010&amp;pages=865-876&amp;pmid=20473652&amp;doi=10.1007/s00122-010-1356-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0026">
<cite>

Irwin, J. A. G.
, 
Aitken, K. S.
, 
Mackie, J. M.
, &amp; 
Musial, J. M.
 (2006). Genetic improvement of lucerne for anthracnose (<em>Colletotrichum trifolii</em>) resistance. Australasian Plant Pathology, 35(6), 573–579. 10.1071/AP06059
</cite> [<a href="https://doi.org/10.1071/AP06059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Australasian%20Plant%20Pathology&amp;title=Genetic%20improvement%20of%20lucerne%20for%20anthracnose%20(Colletotrichum%20trifolii)%20resistance&amp;author=J.%20A.%20G.%20Irwin&amp;author=K.%20S.%20Aitken&amp;author=J.%20M.%20Mackie&amp;author=J.%20M.%20Musial&amp;volume=35&amp;issue=6&amp;publication_year=2006&amp;pages=573-579&amp;doi=10.1071/AP06059&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0027">
<cite>

Irwin, J. A. G.
, 
Lloyd, D. L.
, 
Bray, R. A.
, &amp; 
Langdon, P. W.
 (1980). Selection for resistance to <em>Colletotrichum trifolii</em> in the Lucerne Cultivars Hunter River and Siro Peruvian. Australian Journal of Experimental Agriculture, 20(105), 447–451. 10.1071/EA9800447
</cite> [<a href="https://doi.org/10.1071/EA9800447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Australian%20Journal%20of%20Experimental%20Agriculture&amp;title=Selection%20for%20resistance%20to%20Colletotrichum%20trifolii%20in%20the%20Lucerne%20Cultivars%20Hunter%20River%20and%20Siro%20Peruvian&amp;author=J.%20A.%20G.%20Irwin&amp;author=D.%20L.%20Lloyd&amp;author=R.%20A.%20Bray&amp;author=P.%20W.%20Langdon&amp;volume=20&amp;issue=105&amp;publication_year=1980&amp;pages=447-451&amp;doi=10.1071/EA9800447&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0028">
<cite>

Jacob, I.
, 
Hartmann, S.
, 
Schubiger, F. X.
, &amp; 
Struck, C.
 (2015). Resistance screening of red clover cultivars to <em>Colletotrichum trifolii</em> and improving the resistance level through recurrent selection. Euphytica, 204(2), 303–310. 10.1007/s10681-014-1323-x
</cite> [<a href="https://doi.org/10.1007/s10681-014-1323-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Resistance%20screening%20of%20red%20clover%20cultivars%20to%20Colletotrichum%20trifolii%20and%20improving%20the%20resistance%20level%20through%20recurrent%20selection&amp;author=I.%20Jacob&amp;author=S.%20Hartmann&amp;author=F.%20X.%20Schubiger&amp;author=C.%20Struck&amp;volume=204&amp;issue=2&amp;publication_year=2015&amp;pages=303-310&amp;doi=10.1007/s10681-014-1323-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0029">
<cite>

Jiang, X.
, 
Yang, T.
, 
He, F.
, 
Zhang, F.
, 
Jiang, X.
, 
Wang, C.
, 
Gao, T.
, 
Long, R.
, 
Li, M.
, 
Yang, Q.
, 
Wang, Y.
, 
Zhang, T.
, &amp; 
Kang, J.
 (2024). A genome‐wide association study reveals novel loci and candidate genes associated with plant height variation in <em>Medicago sativa</em>
. BMC Plant Biology, 24(1), Article 544. 10.1186/s12870-024-05151-z

</cite> [<a href="https://doi.org/10.1186/s12870-024-05151-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11177520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38872112/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=A%20genome%E2%80%90wide%20association%20study%20reveals%20novel%20loci%20and%20candidate%20genes%20associated%20with%20plant%20height%20variation%20in%20Medicago%20sativa&amp;author=X.%20Jiang&amp;author=T.%20Yang&amp;author=F.%20He&amp;author=F.%20Zhang&amp;author=X.%20Jiang&amp;volume=24&amp;issue=1&amp;publication_year=2024&amp;pages=544&amp;pmid=38872112&amp;doi=10.1186/s12870-024-05151-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0030">
<cite>

Jiang, X.‐Q.
, 
Zhang, F.
, 
Wang, Z.
, 
Long, R.‐C.
, 
Li, M.‐N.
, 
He, F.
, 
Yang, X.‐J.
, 
Yang, C.‐F.
, 
Jiang, X.
, 
Yang, Q.‐C.
, 
Wang, Q.‐Z.
, &amp; 
Kang, J.‐M.
 (2022). Detection of quantitative trait loci (QTL) associated with spring regrowth in alfalfa (<em>Medicago sativa</em> L.). Journal of Integrative Agriculture, 21(3), 812–818. 10.1016/s2095-3119(21)63671-7
</cite> [<a href="https://doi.org/10.1016/s2095-3119(21)63671-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Integrative%20Agriculture&amp;title=Detection%20of%20quantitative%20trait%20loci%20(QTL)%20associated%20with%20spring%20regrowth%20in%20alfalfa%20(Medicago%20sativa%20L.)&amp;author=X.%E2%80%90Q.%20Jiang&amp;author=F.%20Zhang&amp;author=Z.%20Wang&amp;author=R.%E2%80%90C.%20Long&amp;author=M.%E2%80%90N.%20Li&amp;volume=21&amp;issue=3&amp;publication_year=2022&amp;pages=812-818&amp;doi=10.1016/s2095-3119(21)63671-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0031">
<cite>

Julier, B.
, 
Barre, P.
, 
Lambroni, P.
, 
Delaunay, S.
, 
Thomasset, M.
, 
Lafaillette, F.
, &amp; 
Gensollen, V.
 (2018). Use of GBS markers to distinguish among lucerne varieties, with comparison to morphological traits. Molecular Breeding, 38, Article 133. 10.1007/s11032-018-0891-1
</cite> [<a href="https://doi.org/10.1007/s11032-018-0891-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Use%20of%20GBS%20markers%20to%20distinguish%20among%20lucerne%20varieties,%20with%20comparison%20to%20morphological%20traits&amp;author=B.%20Julier&amp;author=P.%20Barre&amp;author=P.%20Lambroni&amp;author=S.%20Delaunay&amp;author=M.%20Thomasset&amp;volume=38&amp;publication_year=2018&amp;pages=133&amp;doi=10.1007/s11032-018-0891-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0032">
<cite>

Julier, B.
, 
Gastal, F.
, 
Louarn, G.
, 
Badenhausser, I.
, 
Annicchiarico, P.
, 
Crocq, G.
, 
Chatelier, D. L.
, 
Guillemot, E.
, &amp; 
Emile, J. C.
 (2017). Alfalfa (lucerne) in European cropping systems. In Murphy‐Bokern D., Stoddard F. L., &amp; Watson C. A. (Eds.), Legume futures (pp. 168–192). CABI. <a href="https://www.legumehub.eu/wp-content/uploads/2021/05/Chapter-11-Legumes-in-Croping-Systems.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.legumehub.eu/wp-content/uploads/2021/05/Chapter-11-Legumes-in-Croping-Systems.pdf</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Legume%20futures&amp;author=B.%20Julier&amp;author=F.%20Gastal&amp;author=G.%20Louarn&amp;author=I.%20Badenhausser&amp;author=P.%20Annicchiarico&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0033">
<cite>

Julier, B.
, 
Gréard, C.
, 
Pégard, M.
, &amp; 
Barre, P.
 (2022, November 14–17). 
Alfalfa breeding with implementation of molecular tools
. World Alfalfa Congress, San Diego, CA, United States. <a href="https://alfalfasymposium.ucdavis.edu/%2Bsymposium/proceedings/2022/Articles/Bernadette-Julier-2022-Alfalfa-Breeding-With-Implementation-Of-Molecular-Tools.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://alfalfasymposium.ucdavis.edu/+symposium/proceedings/2022/Articles/Bernadette-Julier-2022-Alfalfa-Breeding-With-Implementation-Of-Molecular-Tools.pdf</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Julier,%20B.%20,%20Gr%C3%A9ard,%20C.%20,%20P%C3%A9gard,%20M.%20,%20&amp;%20Barre,%20P.%20(2022,%20November%2014%E2%80%9317).%20Alfalfa%20breeding%20with%20implementation%20of%20molecular%20tools%20.%20World%20Alfalfa%20Congress,%20San%20Diego,%20CA,%20United%20States.%20https://alfalfasymposium.ucdavis.edu/+symposium/proceedings/2022/Articles/Bernadette-Julier-2022-Alfalfa-Breeding-With-Implementation-Of-Molecular-Tools.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0034">
<cite>

Li, A.
, 
Liu, A.
, 
Wu, S.
, 
Qu, K.
, 
Hu, H.
, 
Yang, J.
, 
Shrestha, N.
, 
Liu, J.
, &amp; 
Ren, G.
 (2022). Comparison of structural variants in the whole genome sequences of two <em>Medicago truncatula</em> ecotypes: Jemalong A17 and R108. BMC Plant Biology, 22(1), Article 77. 10.1186/s12870-022-03469-0

</cite> [<a href="https://doi.org/10.1186/s12870-022-03469-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8862580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35193491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Comparison%20of%20structural%20variants%20in%20the%20whole%20genome%20sequences%20of%20two%20Medicago%20truncatula%20ecotypes:%20Jemalong%20A17%20and%20R108&amp;author=A.%20Li&amp;author=A.%20Liu&amp;author=S.%20Wu&amp;author=K.%20Qu&amp;author=H.%20Hu&amp;volume=22&amp;issue=1&amp;publication_year=2022&amp;pages=77&amp;pmid=35193491&amp;doi=10.1186/s12870-022-03469-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0035">
<cite>

Li, W. Y.
, 
Li, X. Z.
, 
Li, W.
, 
Yang, H. B.
, 
Guo, D. D.
, 
Guo, T.
, 
Meng, Y.
, 
He, Q.
, 
Lin, H.
, 
Du, H.
, &amp; 
Niu, L. F.
 (2024). A haplotype‐resolved genome assembly of tetraploid <em>Medicago sativa</em> ssp. <em>falcata</em>
. Science China Life Sciences, 68, 1186–1189. 10.1007/s11427-024-2753-2

</cite> [<a href="https://doi.org/10.1007/s11427-024-2753-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39724396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20China%20Life%20Sciences&amp;title=A%20haplotype%E2%80%90resolved%20genome%20assembly%20of%20tetraploid%20Medicago%20sativa%20ssp.%20falcata&amp;author=W.%20Y.%20Li&amp;author=X.%20Z.%20Li&amp;author=W.%20Li&amp;author=H.%20B.%20Yang&amp;author=D.%20D.%20Guo&amp;volume=68&amp;publication_year=2024&amp;pages=1186-1189&amp;pmid=39724396&amp;doi=10.1007/s11427-024-2753-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0036">
<cite>

Li, X. H.
, 
Wei, Y. L.
, 
Acharya, A.
, 
Jiang, Q. Z.
, 
Kang, J. M.
, &amp; 
Brummer, E. C.
 (2014). A saturated genetic linkage map of autotetraploid alfalfa (<em>Medicago sativa</em> L.) developed using genotyping‐by‐sequencing is highly syntenous with the <em>Medicago truncatula</em> genome. G3‐Genes Genomes Genetique, 4(10), 1971–1979. 10.1534/g3.114.012245
</cite> [<a href="https://doi.org/10.1534/g3.114.012245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4199703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25147192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%E2%80%90Genes%20Genomes%20Genetique&amp;title=A%20saturated%20genetic%20linkage%20map%20of%20autotetraploid%20alfalfa%20(Medicago%20sativa%20L.)%20developed%20using%20genotyping%E2%80%90by%E2%80%90sequencing%20is%20highly%20syntenous%20with%20the%20Medicago%20truncatula%20genome&amp;author=X.%20H.%20Li&amp;author=Y.%20L.%20Wei&amp;author=A.%20Acharya&amp;author=Q.%20Z.%20Jiang&amp;author=J.%20M.%20Kang&amp;volume=4&amp;issue=10&amp;publication_year=2014&amp;pages=1971-1979&amp;pmid=25147192&amp;doi=10.1534/g3.114.012245&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0037">
<cite>

Long, R. C.
, 
Zhang, F.
, 
Zhang, Z. W.
, 
Li, M. N.
, 
Chen, L.
, 
Wang, X.
, 
Liu, W.
, 
Zhang, T.
, 
Yu, L. X.
, 
He, F.
, 
Jiang, X.
, 
Yang, X.
, 
Yang, C.
, 
Wang, Z.
, 
Kang, J.
, &amp; 
Yang, Q. C.
 (2022). Genome assembly of alfalfa cultivar Zhongmu‐4 and identification of SNPs associated with agronomic traits. Genomics Proteomics &amp; Bioinformatics, 20(1), 14–28. 10.1016/j.gpb.2022.01.0021672-0229
</cite> [<a href="https://doi.org/10.1016/j.gpb.2022.01.0021672-0229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9510860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35033678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics%20Proteomics%20&amp;%20Bioinformatics&amp;title=Genome%20assembly%20of%20alfalfa%20cultivar%20Zhongmu%E2%80%904%20and%20identification%20of%20SNPs%20associated%20with%20agronomic%20traits&amp;author=R.%20C.%20Long&amp;author=F.%20Zhang&amp;author=Z.%20W.%20Zhang&amp;author=M.%20N.%20Li&amp;author=L.%20Chen&amp;volume=20&amp;issue=1&amp;publication_year=2022&amp;pages=14-28&amp;pmid=35033678&amp;doi=10.1016/j.gpb.2022.01.0021672-0229&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0038">
<cite>

Mackie, J. M.
, 
Musial, J. M.
, 
Armour, D. J.
, 
Phan, H. T. T.
, 
Ellwood, S. E.
, 
Aitken, K. S.
, &amp; 
Irwin, J. A. G.
 (2007). Identification of QTL for reaction to three races of <em>Colletotrichum trifolii</em> and further analysis of inheritance of resistance in autotetraploid lucerne. Theoretical and Applied Genetics, 114(8), 1417–1426. 10.1071/AR03079

</cite> [<a href="https://doi.org/10.1071/AR03079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17356866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Identification%20of%20QTL%20for%20reaction%20to%20three%20races%20of%20Colletotrichum%20trifolii%20and%20further%20analysis%20of%20inheritance%20of%20resistance%20in%20autotetraploid%20lucerne&amp;author=J.%20M.%20Mackie&amp;author=J.%20M.%20Musial&amp;author=D.%20J.%20Armour&amp;author=H.%20T.%20T.%20Phan&amp;author=S.%20E.%20Ellwood&amp;volume=114&amp;issue=8&amp;publication_year=2007&amp;pages=1417-1426&amp;pmid=17356866&amp;doi=10.1071/AR03079&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0039">
<cite>

Meuwissen, T. H. E.
, 
Hayes, B. J.
, &amp; 
Goddard, M. E.
 (2001). Prediction of total genetic value using genome‐wide dense marker maps. Genetics, 157(4), 1819–1829. 10.1093/genetics/157.4.1819

</cite> [<a href="https://doi.org/10.1093/genetics/157.4.1819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11290733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Prediction%20of%20total%20genetic%20value%20using%20genome%E2%80%90wide%20dense%20marker%20maps&amp;author=T.%20H.%20E.%20Meuwissen&amp;author=B.%20J.%20Hayes&amp;author=M.%20E.%20Goddard&amp;volume=157&amp;issue=4&amp;publication_year=2001&amp;pages=1819-1829&amp;pmid=11290733&amp;doi=10.1093/genetics/157.4.1819&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0040">
<cite>

Murad Leite Andrade, M. H.
, 
Acharya, J. P.
, 
Benevenuto, J.
, 
De Bem Oliveira, I.
, 
Lopez, Y.
, 
Munoz, P.
, 
Resende, M. F. R.
, &amp; 
Rios, E. F.
 (2022). Genomic prediction for canopy height and dry matter yield in alfalfa using family bulks. The Plant Genome, 15(3), Article e20235. 10.1002/tpg2.20235

</cite> [<a href="https://doi.org/10.1002/tpg2.20235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35818699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20prediction%20for%20canopy%20height%20and%20dry%20matter%20yield%20in%20alfalfa%20using%20family%20bulks&amp;author=M.%20H.%20Murad%20Leite%20Andrade&amp;author=J.%20P.%20Acharya&amp;author=J.%20Benevenuto&amp;author=I.%20De%20Bem%20Oliveira&amp;author=Y.%20Lopez&amp;volume=15&amp;issue=3&amp;publication_year=2022&amp;pages=e20235&amp;pmid=35818699&amp;doi=10.1002/tpg2.20235&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0041">
<cite>

Nutter, F. W.
, 
Guan, J.
, 
Gotlieb, A. R.
, 
Rhodes, L. H.
, 
Grau, C. R.
, &amp; 
Sulc, R. M.
 (2002). Quantifying alfalfa yield losses caused by foliar diseases in Iowa, Ohio, Wisconsin, and Vermont. Plant Disease, 86(3), 269–277. 10.1094/PDIS.2002.86.3.269

</cite> [<a href="https://doi.org/10.1094/PDIS.2002.86.3.269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30818606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Disease&amp;title=Quantifying%20alfalfa%20yield%20losses%20caused%20by%20foliar%20diseases%20in%20Iowa,%20Ohio,%20Wisconsin,%20and%20Vermont&amp;author=F.%20W.%20Nutter&amp;author=J.%20Guan&amp;author=A.%20R.%20Gotlieb&amp;author=L.%20H.%20Rhodes&amp;author=C.%20R.%20Grau&amp;volume=86&amp;issue=3&amp;publication_year=2002&amp;pages=269-277&amp;pmid=30818606&amp;doi=10.1094/PDIS.2002.86.3.269&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0042">
<cite>

O'Neill, N.
 (2022). Anthracnose resistance. NAAIC. <a href="https://www.naaic.org/stdtests/updated/pdfs/Anthracnose-revised.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.naaic.org/stdtests/updated/pdfs/Anthracnose‐revised.pdf</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Anthracnose%20resistance&amp;author=N.%20O'Neill&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0043">
<cite>

Ostazeski, S. A.
, &amp; 
Elgin, J. H.
 (1982). Use of hypodermic inoculations of alfalfa for identifying host reactions and races of <em>Colletotrichum trifolii</em>
. Crop Science, 22(3), 545–546. 10.2135/cropsci1982.0011183x002200030026x
</cite> [<a href="https://doi.org/10.2135/cropsci1982.0011183x002200030026x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Use%20of%20hypodermic%20inoculations%20of%20alfalfa%20for%20identifying%20host%20reactions%20and%20races%20of%20Colletotrichum%20trifolii&amp;author=S.%20A.%20Ostazeski&amp;author=J.%20H.%20Elgin&amp;volume=22&amp;issue=3&amp;publication_year=1982&amp;pages=545-546&amp;doi=10.2135/cropsci1982.0011183x002200030026x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0044">
<cite>

Pecrix, Y.
, 
Staton, S. E
, 
Sallet, E.
, 
Lelandais‐Brière, C.
, 
Moreau, S.
, 
Carrère, S.
, 
Blein, T.
, 
Jardinaud, M.‐F.
, 
Latrasse, D.
, 
Zouine, M.
, 
Zahm, M.
, 
Kreplak, J.
, 
Mayjonade, B.
, 
Satgé, C.
, 
Perez, M.
, 
Cauet, S.
, 
Marande, W.
, 
Chantry‐Darmon, C.
, 
Lopez‐Roques, C.
, … 
Gamas, P.
 (2018). Whole‐genome landscape of <em>Medicago truncatula</em> symbiotic genes. Nature Plants, 4(12), 1017–1025. 10.1038/s41477-018-0286-7

</cite> [<a href="https://doi.org/10.1038/s41477-018-0286-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30397259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Plants&amp;title=Whole%E2%80%90genome%20landscape%20of%20Medicago%20truncatula%20symbiotic%20genes&amp;author=Y.%20Pecrix&amp;author=S.%20E%20Staton&amp;author=E.%20Sallet&amp;author=C.%20Lelandais%E2%80%90Bri%C3%A8re&amp;author=S.%20Moreau&amp;volume=4&amp;issue=12&amp;publication_year=2018&amp;pages=1017-1025&amp;pmid=30397259&amp;doi=10.1038/s41477-018-0286-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0045">
<cite>

Pégard, M.
, 
Barre, P.
, 
Delaunay, S.
, 
Surault, F.
, 
Karagic, D.
, 
Milic, D.
, 
Zorić, M.
, 
Ruttink, T.
, &amp; 
Julier, B.
 (2023). Genome‐wide genotyping data renew knowledge on genetic diversity of a worldwide alfalfa collection and give insights on genetic control of phenology traits. Frontiers in Plant Science, 14, Article 1196134. 10.3389/fpls.2023.1196134

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1196134" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10354441/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37476178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%E2%80%90wide%20genotyping%20data%20renew%20knowledge%20on%20genetic%20diversity%20of%20a%20worldwide%20alfalfa%20collection%20and%20give%20insights%20on%20genetic%20control%20of%20phenology%20traits&amp;author=M.%20P%C3%A9gard&amp;author=P.%20Barre&amp;author=S.%20Delaunay&amp;author=F.%20Surault&amp;author=D.%20Karagic&amp;volume=14&amp;publication_year=2023&amp;pages=1196134&amp;pmid=37476178&amp;doi=10.3389/fpls.2023.1196134&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0046">
<cite>

Perrot, S.
, 
Brochard, C.
, &amp; 
Grimault, V.
 (2009). 
Vers un nouveau critère de Valeur Agronomique et Technologique (VAT) pour l'inscription des nouvelles variétés de luzerne au catalogue officiel—Evaluation de la résistance de la luzerne à Colletotrichum trifolii
. GEVES. <a href="https://www.geves.fr/wp-content/uploads/Vers-un-nouveau-crit%C3%A8re-de-Valeur-Agronomique-et-Technologique.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.geves.fr/wp-content/uploads/Vers-un-nouveau-crit%C3%A8re-de-Valeur-Agronomique-et-Technologique.pdf</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Perrot,%20S.%20,%20Brochard,%20C.%20,%20&amp;%20Grimault,%20V.%20(2009).%20Vers%20un%20nouveau%20crit%C3%A8re%20de%20Valeur%20Agronomique%20et%20Technologique%20(VAT)%20pour%20l'inscription%20des%20nouvelles%20vari%C3%A9t%C3%A9s%20de%20luzerne%20au%20catalogue%20officiel%E2%80%94Evaluation%20de%20la%20r%C3%A9sistance%20de%20la%20luzerne%20%C3%A0%20Colletotrichum%20trifolii%20.%20GEVES.%20https://www.geves.fr/wp-content/uploads/Vers-un-nouveau-crit%C3%A8re-de-Valeur-Agronomique-et-Technologique.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0047">
<cite>

Raynal, G.
 (1977). Comparison in artificial contaminations of pathogenicity of <em>Colletotrichum</em> spp. isolated in France on lucerne. Annales De Phytopathologie, 9(2), 193–203.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Annales%20De%20Phytopathologie&amp;title=Comparison%20in%20artificial%20contaminations%20of%20pathogenicity%20of%20Colletotrichum%20spp.%20isolated%20in%20France%20on%20lucerne&amp;author=G.%20Raynal&amp;volume=9&amp;issue=2&amp;publication_year=1977&amp;pages=193-203&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0048">
<cite>

Raynal, G.
, 
Gondran, J.
, 
Bournoville, R.
, &amp; 
Courtillot, M.
 (1989). Ennemis et maladies des prairies. INRA.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Ennemis%20et%20maladies%20des%20prairies&amp;author=G.%20Raynal&amp;author=J.%20Gondran&amp;author=R.%20Bournoville&amp;author=M.%20Courtillot&amp;publication_year=1989&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0049">
<cite>

Segura, V.
, 
Vilhjálmsson, B. J.
, 
Platt, A.
, 
Korte, A.
, 
Seren, Ü.
, 
Long, Q.
, &amp; 
Nordborg, M.
 (2012). An efficient multi‐locus mixed‐model approach for genome‐wide association studies in structured populations. Nature Genetics, 44(7), 825–830. 10.1038/ng.2314

</cite> [<a href="https://doi.org/10.1038/ng.2314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3386481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22706313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=An%20efficient%20multi%E2%80%90locus%20mixed%E2%80%90model%20approach%20for%20genome%E2%80%90wide%20association%20studies%20in%20structured%20populations&amp;author=V.%20Segura&amp;author=B.%20J.%20Vilhj%C3%A1lmsson&amp;author=A.%20Platt&amp;author=A.%20Korte&amp;author=%C3%9C.%20Seren&amp;volume=44&amp;issue=7&amp;publication_year=2012&amp;pages=825-830&amp;pmid=22706313&amp;doi=10.1038/ng.2314&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0050">
<cite>

Shen, C.
, 
Du, H.
, 
Chen, Z.
, 
Lu, H.
, 
Zhu, F.
, 
Chen, H.
, 
Meng, X.
, 
Liu, Q.
, 
Liu, P.
, 
Zheng, L.
, 
Li, X.
, 
Dong, J.
, 
Liang, C.
, &amp; 
Wang, T.
 (2020). The chromosome‐level genome sequence of the autotetraploid alfalfa and resequencing of core germplasms provide genomic resources for alfalfa research. Molecular Plant, 13(9), 1250–1261. 10.1016/j.molp.2020.07.003

</cite> [<a href="https://doi.org/10.1016/j.molp.2020.07.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32673760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=The%20chromosome%E2%80%90level%20genome%20sequence%20of%20the%20autotetraploid%20alfalfa%20and%20resequencing%20of%20core%20germplasms%20provide%20genomic%20resources%20for%20alfalfa%20research&amp;author=C.%20Shen&amp;author=H.%20Du&amp;author=Z.%20Chen&amp;author=H.%20Lu&amp;author=F.%20Zhu&amp;volume=13&amp;issue=9&amp;publication_year=2020&amp;pages=1250-1261&amp;pmid=32673760&amp;doi=10.1016/j.molp.2020.07.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0051">
<cite>

Shi, K.
, 
Dong, H.
, 
Du, H.
, 
Li, Y.
, 
Zhou, L.
, 
Liang, C.
, 
Şakiroğlu, M.
, &amp; 
Wang, Z.
 (2024). The chromosome‐level assembly of the wild diploid alfalfa genome provides insights into the full landscape of genomic variations between cultivated and wild alfalfa. Plant Biotechnology Journal, 22(6), 1757–1772. 10.1111/pbi.14300

</cite> [<a href="https://doi.org/10.1111/pbi.14300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11123407/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38288521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=The%20chromosome%E2%80%90level%20assembly%20of%20the%20wild%20diploid%20alfalfa%20genome%20provides%20insights%20into%20the%20full%20landscape%20of%20genomic%20variations%20between%20cultivated%20and%20wild%20alfalfa&amp;author=K.%20Shi&amp;author=H.%20Dong&amp;author=H.%20Du&amp;author=Y.%20Li&amp;author=L.%20Zhou&amp;volume=22&amp;issue=6&amp;publication_year=2024&amp;pages=1757-1772&amp;pmid=38288521&amp;doi=10.1111/pbi.14300&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0052">
<cite>

Singer, S. D.
, 
Hannoufa, A.
, &amp; 
Acharya, S.
 (2018). Molecular improvement of alfalfa for enhanced productivity and adaptability in a changing environment. Plant Cell and Environment, 41(9), 1955–1971. 10.1111/pce.13090
</cite> [<a href="https://doi.org/10.1111/pce.13090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29044610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20and%20Environment&amp;title=Molecular%20improvement%20of%20alfalfa%20for%20enhanced%20productivity%20and%20adaptability%20in%20a%20changing%20environment&amp;author=S.%20D.%20Singer&amp;author=A.%20Hannoufa&amp;author=S.%20Acharya&amp;volume=41&amp;issue=9&amp;publication_year=2018&amp;pages=1955-1971&amp;pmid=29044610&amp;doi=10.1111/pce.13090&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0053">
<cite>

Valizadeh, M.
, 
Mohayeji, M.
, 
Yasinzadeh, N.
, 
Nasrullazadeh, S.
, &amp; 
Moghaddam, M.
 (2011). Genetic diversity of synthetic alfalfa generations and cultivars using tetrasomic inherited allozyme markers. Journal of Agricultural Science and Technology, 13(3), 425–430.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agricultural%20Science%20and%20Technology&amp;title=Genetic%20diversity%20of%20synthetic%20alfalfa%20generations%20and%20cultivars%20using%20tetrasomic%20inherited%20allozyme%20markers&amp;author=M.%20Valizadeh&amp;author=M.%20Mohayeji&amp;author=N.%20Yasinzadeh&amp;author=S.%20Nasrullazadeh&amp;author=M.%20Moghaddam&amp;volume=13&amp;issue=3&amp;publication_year=2011&amp;pages=425-430&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0054">
<cite>

VanRaden, P. M.
 (2008). Efficient methods to compute genomic predictions. Journal of Dairy Science, 91(11), 4414–4423. 10.3168/jds.2007-0980

</cite> [<a href="https://doi.org/10.3168/jds.2007-0980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18946147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Dairy%20Science&amp;title=Efficient%20methods%20to%20compute%20genomic%20predictions&amp;author=P.%20M.%20VanRaden&amp;volume=91&amp;issue=11&amp;publication_year=2008&amp;pages=4414-4423&amp;pmid=18946147&amp;doi=10.3168/jds.2007-0980&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0055">
<cite>

Whittaker, J. C.
, 
Thompson, R.
, &amp; 
Denham, M. C.
 (2000). Marker‐assisted selection using ridge regression. Genetical Research, 75(2), 249–252. 10.1017/S0016672399004462

</cite> [<a href="https://doi.org/10.1017/S0016672399004462" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10816982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetical%20Research&amp;title=Marker%E2%80%90assisted%20selection%20using%20ridge%20regression&amp;author=J.%20C.%20Whittaker&amp;author=R.%20Thompson&amp;author=M.%20C.%20Denham&amp;volume=75&amp;issue=2&amp;publication_year=2000&amp;pages=249-252&amp;pmid=10816982&amp;doi=10.1017/S0016672399004462&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0056">
<cite>

Wilsie, C. P.
 (1958). Effect of inbreeding on fertility and vigor of alfalfa. Agronomy Journal, 50, 182–185. 10.2134/agronj1958.00021962005000040004x
</cite> [<a href="https://doi.org/10.2134/agronj1958.00021962005000040004x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy%20Journal&amp;title=Effect%20of%20inbreeding%20on%20fertility%20and%20vigor%20of%20alfalfa&amp;author=C.%20P.%20Wilsie&amp;volume=50&amp;publication_year=1958&amp;pages=182-185&amp;doi=10.2134/agronj1958.00021962005000040004x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0057">
<cite>

Yang, S.
, 
Gao, M.
, 
Deshpande, S.
, 
Lin, S.
, 
Roe, B. A.
, &amp; 
Zhu, H.
 (2007). Genetic and physical localization of an anthracnose resistance gene in <em>Medicago truncatula</em>
. Theoretical and Applied Genetics, 116, 45–52. 10.1007/s00122-007-0645-7

</cite> [<a href="https://doi.org/10.1007/s00122-007-0645-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17891371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genetic%20and%20physical%20localization%20of%20an%20anthracnose%20resistance%20gene%20in%20Medicago%20truncatula&amp;author=S.%20Yang&amp;author=M.%20Gao&amp;author=S.%20Deshpande&amp;author=S.%20Lin&amp;author=B.%20A.%20Roe&amp;volume=116&amp;publication_year=2007&amp;pages=45-52&amp;pmid=17891371&amp;doi=10.1007/s00122-007-0645-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0058">
<cite>

Yang, S.
, 
Gao, M.
, 
Xu, C.
, 
Gao, J.
, 
Deshpande, S.
, 
Lin, S.
, 
Roe, B. A.
, &amp; 
Zhu, H.
 (2008). Alfalfa benefits from <em>Medicago truncatula</em>: The RCT1 gene from <em>M. truncatula</em> confers broad‐spectrum resistance to anthracnose in alfalfa. Proceedings of the National Academy of Sciences of the United States of America, 105, 12164–12169. 10.1073/pnas.0802518105

</cite> [<a href="https://doi.org/10.1073/pnas.0802518105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2527883/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18719113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Alfalfa%20benefits%20from%20Medicago%20truncatula:%20The%20RCT1%20gene%20from%20M.%20truncatula%20confers%20broad%E2%80%90spectrum%20resistance%20to%20anthracnose%20in%20alfalfa&amp;author=S.%20Yang&amp;author=M.%20Gao&amp;author=C.%20Xu&amp;author=J.%20Gao&amp;author=S.%20Deshpande&amp;volume=105&amp;publication_year=2008&amp;pages=12164-12169&amp;pmid=18719113&amp;doi=10.1073/pnas.0802518105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0059">
<cite>

Young, N. D.
, 
Debellé, F.
, 
Oldroyd, G. E. D.
, 
Geurts, R.
, 
Cannon, S. B.
, 
Udvardi, M. K.
, 
Benedito, V. A.
, 
Mayer, K. F. X.
, 
Gouzy, J.
, 
Schoof, H.
, 
Van De Peer, Y.
, 
Proost, S.
, 
Cook, D. R.
, 
Meyers, B. C.
, 
Spannagl, M.
, 
Cheung, F.
, 
De Mita, S.
, 
Krishnakumar, V.
, 
Gundlach, H.
, … 
Roe, B. A.
 (2011). The <em>Medicago</em> genome provides insight into the evolution of rhizobial symbioses. Nature, 480, 520–524. 10.1038/nature10625

</cite> [<a href="https://doi.org/10.1038/nature10625" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3272368/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22089132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20Medicago%20genome%20provides%20insight%20into%20the%20evolution%20of%20rhizobial%20symbioses&amp;author=N.%20D.%20Young&amp;author=F.%20Debell%C3%A9&amp;author=G.%20E.%20D.%20Oldroyd&amp;author=R.%20Geurts&amp;author=S.%20B.%20Cannon&amp;volume=480&amp;publication_year=2011&amp;pages=520-524&amp;pmid=22089132&amp;doi=10.1038/nature10625&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270085-bib-0060">
<cite>

Zhang, F.
, 
Kang, J.
, 
Long, R.
, 
Li, M.
, 
Sun, Y.
, 
He, F.
, 
Jiang, X.
, 
Yang, C.
, 
Yang, X.
, 
Kong, J.
, 
Wang, Y.
, 
Wang, Z.
, 
Zhang, Z.
, &amp; 
Yang, Q.
 (2023). Application of machine learning to explore the genomic prediction accuracy of fall dormancy in autotetraploid alfalfa. Horticulture Research, 10(1), Article uhac225. 10.1093/hr/uhac225

</cite> [<a href="https://doi.org/10.1093/hr/uhac225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9832841/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36643744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=Application%20of%20machine%20learning%20to%20explore%20the%20genomic%20prediction%20accuracy%20of%20fall%20dormancy%20in%20autotetraploid%20alfalfa&amp;author=F.%20Zhang&amp;author=J.%20Kang&amp;author=R.%20Long&amp;author=M.%20Li&amp;author=Y.%20Sun&amp;volume=10&amp;issue=1&amp;publication_year=2023&amp;pages=uhac225&amp;pmid=36643744&amp;doi=10.1093/hr/uhac225&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>Supplemental Figure S1. QQplot for the QTL detection, showing six significant SNP.</p>
<p>Supplemental Figure S2. Correlation between allele frequencies of chr8_83798444 (y‐axis) and chr8_80205951 and between chr8_80198129 and chr8_80205951. The points, representing the accessions, are colored depending on their percentage of resistance, from red for a low percentage of resistance to green for a high percentage of resistance.</p>
<p>Supplemental Figure S3. Linkage disequilibrium on chromosome 8 within a distance of 4 million base pairs, in the region around SNP chr8_80198129.</p>
<p>Supplemental Figure S4. Alignment of the homologous chromosomes 8 in the region of the QTL, with chr8.2, used as the reference for GBS mapping, compared to the other three chromosomes. Identity legend: 0‐0.25: yellow, 0.25‐0.5: orange, 0.5‐0.75 light green, 0.75‐1: dark green.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326290/bin/TPG2-18-e70085-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70085-s001.docx</a><sup> (385KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>Supplemental Table S1: list of alfalfa accessions with their country of origin, status, registration year when available, and adjusted resistance to anthracnose, pathotype C86‐2.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326290/bin/TPG2-18-e70085-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70085-s002.xlsx</a><sup> (35.1KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The genotyping data are available at <a href="https://doi.org/10.57745/L0FLJD" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.57745/L0FLJD</a>. The phenotyping data are available in the Table <a href="#tpg270085-supinfo-0001" class="usa-link">S1</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70085"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70085.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12326290/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12326290/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12326290%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326290/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12326290/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12326290/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40765523/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12326290/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40765523/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12326290/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12326290/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="G4fgj87hg0wsBtzzDi8GVvceIpyG6MypG1JLCJFSEcb7imY2To7BG2bACI2HR6HX">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
