
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Bioinformatic insights into five Chinese population substructures inferred from the East Asian-specific AISNP panel - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152551258AF429C3055125002FD6BDB0.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcgeno">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359956/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Genomics">
<meta name="citation_title" content="Bioinformatic insights into five Chinese population substructures inferred from the East Asian-specific AISNP panel">
<meta name="citation_author" content="Jing Chen">
<meta name="citation_author_institution" content="School of Forensic Medicine, Shanxi Medical University, Jinzhong, 030600 China">
<meta name="citation_author_institution" content="Institute of Rare Diseases, West China Hospital of Sichuan University, Sichuan University, Chengdu, 610000 China">
<meta name="citation_author_institution" content="Shanxi Province Engineering Research Center of Forensic Identification, Taiyuan, 030001 China">
<meta name="citation_author" content="Yuguo Huang">
<meta name="citation_author_institution" content="Institute of Rare Diseases, West China Hospital of Sichuan University, Sichuan University, Chengdu, 610000 China">
<meta name="citation_author" content="Jie Zhong">
<meta name="citation_author_institution" content="Institute of Rare Diseases, West China Hospital of Sichuan University, Sichuan University, Chengdu, 610000 China">
<meta name="citation_author" content="Mengge Wang">
<meta name="citation_author_institution" content="Institute of Rare Diseases, West China Hospital of Sichuan University, Sichuan University, Chengdu, 610000 China">
<meta name="citation_author_institution" content="Center for Archaeological Science, Sichuan University, Chengdu, 610000 China">
<meta name="citation_author" content="Guanglin He">
<meta name="citation_author_institution" content="Institute of Rare Diseases, West China Hospital of Sichuan University, Sichuan University, Chengdu, 610000 China">
<meta name="citation_author_institution" content="Center for Archaeological Science, Sichuan University, Chengdu, 610000 China">
<meta name="citation_author" content="Jiangwei Yan">
<meta name="citation_author_institution" content="School of Forensic Medicine, Shanxi Medical University, Jinzhong, 030600 China">
<meta name="citation_author_institution" content="Shanxi Province Engineering Research Center of Forensic Identification, Taiyuan, 030001 China">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="748">
<meta name="citation_doi" content="10.1186/s12864-025-11947-6">
<meta name="citation_pmid" content="40826445">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359956/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359956/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359956/pdf/12864_2025_Article_11947.pdf">
<meta name="description" content="Recent advances in population-specific high-quality reference databases have significantly improved the performance of forensic panel development for personal identification, parentage testing, and biogeographical ancestry inference. However, the ...">
<meta name="og:title" content="Bioinformatic insights into five Chinese population substructures inferred from the East Asian-specific AISNP panel">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Recent advances in population-specific high-quality reference databases have significantly improved the performance of forensic panel development for personal identification, parentage testing, and biogeographical ancestry inference. However, the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359956/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12359956">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12864-025-11947-6"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12864_2025_Article_11947.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359956%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12359956/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12359956/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359956/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcgeno.png" alt="BMC Genomics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Genomics" title="Link to BMC Genomics" shape="default" href="https://bmcgenomics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Genomics</button></div>. 2025 Aug 18;26:748. doi: <a href="https://doi.org/10.1186/s12864-025-11947-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12864-025-11947-6</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Genomics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Genomics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Bioinformatic insights into five Chinese population substructures inferred from the East Asian-specific AISNP panel</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Jing Chen</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Jing Chen</span></h3>
<div class="p">
<sup>1</sup>School of Forensic Medicine, Shanxi Medical University, Jinzhong, 030600 China </div>
<div class="p">
<sup>2</sup>Institute of Rare Diseases, West China Hospital of Sichuan University, Sichuan University, Chengdu, 610000 China </div>
<div class="p">
<sup>3</sup>Shanxi Province Engineering Research Center of Forensic Identification, Taiyuan, 030001 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jing Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yuguo Huang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yuguo Huang</span></h3>
<div class="p">
<sup>2</sup>Institute of Rare Diseases, West China Hospital of Sichuan University, Sichuan University, Chengdu, 610000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuguo Huang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhong%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jie Zhong</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jie Zhong</span></h3>
<div class="p">
<sup>2</sup>Institute of Rare Diseases, West China Hospital of Sichuan University, Sichuan University, Chengdu, 610000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhong%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jie Zhong</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mengge Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mengge Wang</span></h3>
<div class="p">
<sup>2</sup>Institute of Rare Diseases, West China Hospital of Sichuan University, Sichuan University, Chengdu, 610000 China </div>
<div class="p">
<sup>4</sup>Center for Archaeological Science, Sichuan University, Chengdu, 610000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mengge Wang</span></a>
</div>
</div>
<sup>2,</sup><sup>4,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Guanglin He</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Guanglin He</span></h3>
<div class="p">
<sup>2</sup>Institute of Rare Diseases, West China Hospital of Sichuan University, Sichuan University, Chengdu, 610000 China </div>
<div class="p">
<sup>4</sup>Center for Archaeological Science, Sichuan University, Chengdu, 610000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guanglin He</span></a>
</div>
</div>
<sup>2,</sup><sup>4,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yan%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jiangwei Yan</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jiangwei Yan</span></h3>
<div class="p">
<sup>1</sup>School of Forensic Medicine, Shanxi Medical University, Jinzhong, 030600 China </div>
<div class="p">
<sup>3</sup>Shanxi Province Engineering Research Center of Forensic Identification, Taiyuan, 030001 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yan%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiangwei Yan</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Forensic Medicine, Shanxi Medical University, Jinzhong, 030600 China </div>
<div id="Aff2">
<sup>2</sup>Institute of Rare Diseases, West China Hospital of Sichuan University, Sichuan University, Chengdu, 610000 China </div>
<div id="Aff3">
<sup>3</sup>Shanxi Province Engineering Research Center of Forensic Identification, Taiyuan, 030001 China </div>
<div id="Aff4">
<sup>4</sup>Center for Archaeological Science, Sichuan University, Chengdu, 610000 China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Sep 11; Accepted 2025 Jul 25; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12359956  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40826445/" class="usa-link">40826445</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Recent advances in population-specific high-quality reference databases have significantly improved the performance of forensic panel development for personal identification, parentage testing, and biogeographical ancestry inference. However, the discriminative power of previously developed AISNP panels remains limited in applications involving regional Chinese population substructures.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">We used the high-quality Chinese population-specific genetic resource to develop six nested C5ClusterTag-50/100/250/500/1000/2000 ancestry-informative SNP panels focused on inferring population stratification among geographically and genetically distinct Chinese populations. We used comprehensive bioinformatics approaches and machine learning to validate the effectiveness of these panels in both the testing and training datasets. A total of 2,772 individuals were screened across different AISNP panels based on the <em>I</em><sub><em>n</em></sub> value. Ancestry inference power was assessed via principal component analysis (PCA), ADMIXTURE, and uniform manifold approximation and projection (UMAP) across the 2000-AISNP, 1000-AISNP, 500-AISNP, 250-AISNP, 100-AISNP, and 50-AISNP panels. The 1000-AISNP panel demonstrated the optimal balance between the SNP count and discriminative power for forensic applications. The accuracy of the random forest model was confirmed through a confusion matrix based on machine learning.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">These panels can differentiate ethnolinguistic Chinese populations into five subgroups based on geographical divisions or linguistic affiliations, achieving a high average accuracy rate in machine learning models. This work not only developed a robust ancestry inference panel and new tools for predicting the ancestry of ethnolinguistic Chinese populations but also created a comprehensive reference dataset and machine learning model applicable to population and forensic uses globally.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12864-025-11947-6.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Biogeographic ancestry, Ancestry-informative SNPs, Forensic genetics, Chinese population substructures, Machine learning</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par4">China, the largest country in East Asia by area, exhibits significant ethnic, genetic, and linguistic diversity [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Previous genetic and archaeological studies have revealed complex migration patterns and the admixture history of East Asian populations [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Large-scale genetic studies, such as the Pan-Asian SNP Project, have demonstrated a strong correlation between the genetic structure of Asian populations and their linguistic classifications [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Recent anthropologically informed genomic resources from autosomal and uniparental perspectives further confirmed the geography-related genetic substructure, which was also associated with multiple social-cultural factors [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. The northern region of China is home to a variety of language families, including Mongolic, Tungusic, Turkic, Japanese, and Korean, which are collectively referred to as “Transeurasian” families [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. In Central China, the Sino-Tibetan language family, comprising both Sinitic and Tibeto-Burman branches, is the most widespread and populous [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>–<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. The Hmong-Mien and Tai-Kadai language families are prevalent in southern East Asia [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Additionally, the Austronesian language family dominates the southeastern coastal regions of China, whereas the Austroasiatic language family is found in inland areas. China’s rich ethnic diversity is also reflected in its 55 officially recognized ethnic minority groups and Han Chinese, as well as many unrecognized groups, such as the Chuanqing, Xijia, and Gejia peoples.</p>
<p id="Par5">In investigating various criminal and civil cases within multiethnic communities, determining the population origin of biological materials can provide valuable leads, especially when STR profiling of samples collected from crime scenes does not yield matches in DNA databases or when doubts persist regarding eyewitness testimonies. Such information is crucial for narrowing the scope of investigations and increasing the likelihood of case resolution. Single nucleotide polymorphisms (SNPs), which are located in various genomic regions, including coding sequences of genes, noncoding regions, and intergenic regions, are highly polymorphic and stable [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>–<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. These markers can be effectively analyzed even in highly degraded samples and possess significant potential for discerning biogeographical ancestry [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. In forensic science, SNPs used as genetic markers are categorized into identity-informative SNPs, lineage-informative SNPs, ancestry-informative SNPs (AISNPs), and phenotype-informative SNPs. These markers are vital for complex parentage testing, reconstructing family lineages, inferring geographic ancestry, and predicting individual phenotypes, which are prominent research areas in forensic genetics [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>–<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Widely used genetic ancestry-informative markers (AIMs) include SNPs [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>], triallelic insertion-deletions [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>], diallelic insertion-deletions [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>–<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>], multiple insertion-deletions [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>, <a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>], and microhaplotypes [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. Different detection systems select genetic markers according to the target populations, but there is often little overlap in loci shared between different systems, and reference population data are frequently lacking for each system [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. Over the past several decades, AIMs have been developed to infer an individual’s continental or biogeographic origins. For example, Kidd et al. developed a panel of 55 AISNPs for distinguishing global populations [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>], whereas Seldin et al. developed a panel of 128 AISNPs specific to Native American populations [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Chen et al. developed four nested AISNP panels capable of discriminating between phenotypically and genetically similar neighboring East Asian populations, such as Han Chinese, Japanese, and Korean populations [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>].</p>
<p id="Par6">The identification and distribution patterns of fine-scale genetic substructures within Chinese populations provide a robust theoretical foundation for precise biogeographic tracing in forensic studies [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. Xu et al. used whole-genome sequencing techniques to investigate the population history of Han Chinese individuals and revealed distinct genetic substructures, categorized as Northern Han and Southern Han, which closely correspond to geographical regions [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. These findings were further validated through large-scale Chinese genomic resources [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>–<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>] and anthropologically informed studies [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Li et al. developed a system of 74 AISNPs for global populations, enabling the classification of these populations into ten distinct genetic subgroups [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. He et al. assessed the effectiveness of biogeographic ancestry inference via 165 AIM genetic markers among major ethnic groups in China [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>–<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. These studies suggest that while current ancestry inference systems effectively distinguish populations at the intercontinental level, their ability to differentiate populations within the same continent remains limited. Consequently, there is a critical need to develop more refined ancestry inference panels for diverse populations in China.</p>
<p id="Par7">An optimal SNP panel of ancestry-informative SNPs (AISNPs, a subset of AIMs) should consist of a minimal set of SNPs that exhibit significant allele frequency differences across diverse populations. The selected SNPs should collectively ensure the highest discriminatory accuracy for ancestral inference [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. However, a notable limitation of AIMs is their ability to distinguish only among previously studied populations [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. Current whole-genome chip designs are primarily based on population-specific variations, and the presence of ascertainment bias can diminish the efficiency and accuracy of ancestry analysis using whole-genome loci. Thus, identifying a set of AIMs that effectively capture interpopulation differences for specific populations is crucial for facilitating rapid and accurate individual ancestry analysis and localization. In this study, high-quality Chinese population-specific genetic resource was used to develop six nested AISNP panels tailored for ancestry inference [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. Comprehensive bioinformatic approaches and machine learning techniques were employed to validate the effectiveness of the panels in both the training and testing datasets.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title"><strong>Sample collection and quality control</strong></h3>
<p id="Par8">Saliva samples of 2,772 Chinese individuals were obtained from the anthropologically informed 10 K Chinese People Genomic Diversity Project. The extracted DNA was genotyped via the Infinium Global Screening Array (Illumina, CA, USA). All participants provided written informed consent, and the study was conducted in accordance with the Declaration of Helsinki and approved by the Medical Ethics Committees of West China Hospital of Sichuan University (2023 − 306). Family relationships were assessed via KING2 [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>] by estimating kinship coefficients. All the participants reported having parents and grandparents from the same ethnic group, with no consanguineous marriages for at least three generations. Quality control filtering was conducted via PLINK 1.9 [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>] with parameters set to --mind 0.1, --geno 0.1, --hwe 0.001, and --maf 0.01. Individuals with more than 10% missing genotypes were excluded, as were those with SNPs missing data in more than 10% of the samples or those in Hardy-Weinberg disequilibrium (<em>P</em> &lt; 0.001). SNPs on sex chromosomes were not included. A genome-wide SNP dataset consisting of 434,143 clean SNPs was compiled for 2,772 unrelated individuals, including 1,097 Han_North, 898 Han_South, 365 Hmong-Mien (HM), 154 Tibeto-Burman (TB), and 258 Tai-Kadai (TK) individuals. This dataset generated an ancestry-informative panel (Fig. <a href="#Fig1" class="usa-link">1</a>, Table <a href="#MOESM1" class="usa-link">S1</a>). After quality control filtering, a total of 434,143 Illumina SNPs remained, which were selected on the basis of the presence of a dbSNP reference ID number for the Illumina platform. The data was submitted to the public databases in the Genome Variation Map (GVM) [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>] at the National Genomics Data Center [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>].</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359956_12864_2025_11947_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dee/12359956/c40a94f41791/12864_2025_11947_Fig1_HTML.jpg" loading="lazy" id="MO1" height="526" width="750" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Geographical locations of the collected samples. These five distinct populations are represented through color-coding that corresponds to their respective language family categories. The base map was downloaded from the website <a href="http://bzdt.ch.mnr.gov.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bzdt.ch.mnr.gov.cn/</a> and officially approved with the number GS(2019)1694</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Ancestry informative SNP selection</h3>
<p id="Par10">The AIM generator (Python script, AIMs_generator.py) [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>] was used to identify the top 50, 100, 250, 500, 1000, and 2000 AIMs. The program accepts genotype files and allele frequency files from PLINK, performs LD pruning, and selects AIMs based on Rosenberg’s <em>I</em><sub><em>n</em></sub> Statistic [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>], which defines the informativeness of SNPs. AIMs were selected according to the following criteria: (1) exclusion of duplicate SNPs and SNPs located on sex chromosomes; (2) maintenance of a minimum distance of 1 Mb between SNPs on the same chromosome to reduce linkage disequilibrium effects; and (3) computation of the <em>I</em><sub><em>n</em></sub> value for each SNP across the five populations. The AIMs were then ranked in descending order according to their <em>I</em><sub><em>n</em></sub> values. On the basis of these criteria, a maximum of 2,020 SNPs were filtered. AIM panels with the top 50, 100, 250, 500, 1000, and 2000 AIMs were subsequently generated (Figs. <a href="#Fig2" class="usa-link">2</a>, S1, and Table <a href="#MOESM2" class="usa-link">S2</a>−3). The performance of these AIM sets in differentiating the five populations was then examined and evaluated.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359956_12864_2025_11947_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dee/12359956/b562efb5c7c8/12864_2025_11947_Fig2_HTML.jpg" loading="lazy" id="MO2" height="756" width="750" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The density of AISNPs across six panels on 22 chromosomes. For more details on the markers, refer to Table <a href="#MOESM1" class="usa-link">S1</a></p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title"><strong>Evaluation of the AISNP panel’s performance</strong></h3>
<p id="Par12">The efficacy of these AIMs in delineating population stratification among various regional populations was evaluated via principal component analysis (PCA), uniform manifold approximation and projection (UMAP) and ADMIXTURE analysis. PCA was performed with the smartpca package [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>] from EIGENSOFT, which employs default parameters for both the screened AIMs and the raw data. Non-linear dimensionality reduction and visualization of complex population structures were performed using UMAP analysis via the umap and ggplot2 packages in R v4.3.2. ADMIXTURE analysis [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>] was then conducted to evaluate and visualize population relationships and individual ancestral proportions. We performed the model-based ADMIXTURE analysis with K values ranging from 2 to 20, using the default 10-fold cross-validation (CV = 10) and 100 bootstrap replicates. Cross-validation was used to identify the optimal number of clusters, revealing that the admixture model with five ancestral sources (K = 5) provided the best fit (Figure <a href="#MOESM2" class="usa-link">S2</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title"><strong>Construction of the random forest model</strong></h3>
<p id="Par13">The outcomes of the PCA and ADMIXTURE analyses were used as input features to construct random forest (RF) models, which were implemented via the randomForest package in R version 4.3.2. Models 1 through 5 were constructed based on the top 50, 100, 250, 500, 1000, and 2000 SNPs with the highest <em>I</em><sub><em>n</em></sub> values, respectively. A stratified random sampling approach was applied, with 70% of the samples from each designated population forming the training dataset and the remaining 30% serving as the testing dataset for predicting individuals’ biogeographical origins. To enhance model robustness, 5-fold cross-validation was employed during training. The predictive accuracy of each model was evaluated to determine ancestry within the five Chinese subpopulations. Finally, a parameter tuning process was conducted to identify the optimal hyperparameters for each model.</p></section><section id="Sec7"><h3 class="pmc_sec_title"><strong>Panel performance assessment</strong></h3>
<p id="Par14">The performance of the AIM panel in differentiating among five population groups was assessed via five metrics: sensitivity (= <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dee/12359956/60a483fa35b1/d33e554.gif" loading="lazy" id="d33e554" alt="Inline graphic"></span>), specificity (=<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dee/12359956/9527e2e93f12/d33e560.gif" loading="lazy" id="d33e560" alt="Inline graphic"></span>), positive predictive value (PPV= <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dee/12359956/ad099e8dc137/d33e566.gif" loading="lazy" id="d33e566" alt="Inline graphic"></span>), negative predictive value (NPV= <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dee/12359956/eb373720b1ce/d33e572.gif" loading="lazy" id="d33e572" alt="Inline graphic"></span>), and accuracy (= <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dee/12359956/ad303572b39f/d33e578.gif" loading="lazy" id="d33e578" alt="Inline graphic"></span>). Here, TP represents true positives, TN represents true negatives, FN represents false negatives, and FP represents false positives. Sensitivity and specificity describe the distribution of classifications based on true outcomes, whereas PPV and NPV provide the probabilities of true outcomes given the classifications. The random forest model was validated through a confusion matrix to determine the sensitivity, specificity, PPV, and NPV. Additionally, the performance of each model was evaluated via Receiver Operating Characteristic (ROC) analysis and average accuracy (AAC). These metrics provide a comprehensive evaluation of the panel’s effectiveness in distinguishing between different population groups.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Results</h2>
<p id="Par15">From an extensive dataset comprising 434,143 SNPs from 2,772 individuals across five distinct geographical and linguistic subgroups, we identified AIMs. We then developed six nested panels containing 50, 100, 250, 500, 1000, and 2000 AISNPs to effectively differentiate among the five Chinese subpopulations: Han_North, Han_South, Hmong-Mien, Tibeto-Burman, and Tai-Kadai. Our goal was to design a nested AIM panel, named C5ClusterTag-50/100/250/500/1000/2000, based on a series of bioinformatics screening and validation strategies, as outlined in the following sections.</p>
<section id="Sec9"><h3 class="pmc_sec_title"><strong>Genetic clustering and population structure of the five subpopulations in China based on six AISNP panels</strong></h3>
<p id="Par16">PCA and ADMIXTURE analyses were conducted to explore genetic admixture and relationships among the five Chinese subpopulations. Using genome-wide data, ADMIXTURE (K = 2) classified these populations into two major groups: southern (red) and northern (blue) (Figure <a href="#MOESM3" class="usa-link">S3</a>). PC1 highlighted genetic differentiation between northern populations, represented by the Han_North and TB-speaking groups, and southern populations, which included the Han_South, TK-speaking, and HM-speaking groups (Figure <a href="#MOESM5" class="usa-link">S5</a>G). The TK-speaking population was positioned at the southernmost end of the southern cluster. PCA on six AIM panels revealed partial overlap in the genetic profiles of Han_North and Han_South, suggesting genetic integration among Sinitic-speaking populations, with increased discriminative power as the number of SNPs increased. Notably, relatively close genetic affinity was observed between parts of the HM- and TB-speaking populations. We used cross-validation to identify an “optimal” number of clusters (K = 5). At this optimal K value, five distinct ancestry components were observed among the studied populations. Each component corresponds to a regionally and ethnolinguistically relevant ancestral source: The teal component was predominantly observed in TK-speaking populations, primarily distributed in southern China and parts of Southeast Asia. The indigo component was maximized in northern Han and the turquoise blue component was enriched in southern Han, reflecting differentiated gene flow events from northern and southern East Asian ancestries. The royal blue component was predominant in TB-speaking groups, such as Tibetan and Qiang populations, consistent with their genetic distinctiveness and adaptation to high-altitude environments. The vermilion component was maximized in HM-speaking groups, typically located in southwestern China. We found that subpopulations belonging to the same linguistic group generally exhibited similar ancestry profiles, suggesting shared genetic backgrounds within groups. ADMIXTURE analysis of 2,772 individuals confirmed the effectiveness of panels containing 50, 100, 250, 500, 1000, and 2000 SNPs in differentiating the genetic structure of the five subpopulations, with the 1000 and 2000 AISNP panels showing high discriminative power (Figs. <a href="#Fig3" class="usa-link">3</a> and <a href="#Fig4" class="usa-link">4</a>). To evaluate the consistency of ancestry inference across different SNP subsets, we calculated pairwise cosine similarity between the ADMIXTURE outputs at K = 5. For each pairwise comparison, all possible permutations of ancestry components were considered, and the highest average diagonal similarity was retained. The results revealed high concordance among SNP subsets, with average cosine similarity values ranging from 0.853 to 0.991, indicating stable and reliable ancestry estimation across different marker sets (Figure <a href="#MOESM4" class="usa-link">S4</a> and Table <a href="#MOESM4" class="usa-link">S4</a>). Genome-wide data (434,143 SNPs) were also compared with the six panels in both the PCA and ADMIXTURE analyses (K = 2 to 5) (Figures <a href="#MOESM5" class="usa-link">S5</a>−6). UMAP was used to provide a detailed visualization of genotypic information at fine scales. UMAP revealed clusters that were challenging to identify via PCA plots or ADMIXTURE analyses, such as a linguistically restricted cluster within Han_North and Han_South (Fig. <a href="#Fig5" class="usa-link">5</a>). While PCA is effective in capturing the global structure of the data, UMAP excels in illustrating both the global structure (overall shape and distances) and the local structure (clusters and nearest neighbors) of the data [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. The combined use of these methods offers a comprehensive understanding of the data. </p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359956_12864_2025_11947_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dee/12359956/d8afdbc596fe/12864_2025_11947_Fig3_HTML.jpg" loading="lazy" id="MO3" height="466" width="677" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PCA and ADMIXTURE analyses of the 50-AISNP (A-B), 100-AISNP (C-D), and 250-AISNP (E-F) panels. <strong>(A</strong>,<strong> C</strong>,<strong> E)</strong> PCA plots for five subpopulation distributions on the 50-AISNP, 100-AISNP, and 250-AISNP panels. These subpopulations were categorized into two groups: southern (red) and northern (blue). <strong>(B</strong>,<strong> D</strong>,<strong> F)</strong> Results from the model-based ADMIXTURE clustering analysis at K = 5 are shown</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359956_12864_2025_11947_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dee/12359956/54bde17fd452/12864_2025_11947_Fig4_HTML.jpg" loading="lazy" id="MO4" height="466" width="677" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PCA and ADMIXTURE analyses of the 500-AISNP (A-B), 1000-AISNP (<strong>C-D</strong>), and 2000-AISNP (E-F) panels. <strong>(A</strong>,<strong> C</strong>,<strong> E)</strong> PCA plots for five subpopulation distributions on the 500-AISNP, 1000-AISNP, and 2000-AISNP panels. These subpopulations were categorized into two groups: southern (red) and northern (blue). <strong>(B</strong>,<strong> D</strong>,<strong> F)</strong> Results from the model-based ADMIXTURE clustering analysis at K = 5 are shown</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359956_12864_2025_11947_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dee/12359956/bf1b772105b3/12864_2025_11947_Fig5_HTML.jpg" loading="lazy" id="MO5" height="594" width="750" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Uniform manifold approximation and projection (UMAP) analysis is based on 50, 100, 250, 500, 1000, and 2000 AIMs. These five populations were color-coded according to their language family categories. HM: Hmong-Mien; TB: Tibeto-Burman; TK: Tai-Kadai</p></figcaption></figure></section><section id="Sec10"><h3 class="pmc_sec_title">Construction and efficacy assessment of ancestry-predictive modeling with six AISNP panels across five populations </h3>
<p id="Par19">To develop the most effective ancestry-predictive models, we utilized PCA and ADMIXTURE results as input features to construct a series of RF models labeled Models 1 through 5. These models were built using the top 50, 100, 250, 500, 1000, and 2000 high-<em>I</em><sub><em>n</em></sub> SNPs, respectively. We assessed the discriminative power of the selected SNPs on the basis of their cumulative accuracy in assigning individuals to their regions of origin [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. Our findings revealed that the accuracy of ancestry inference increased with the inclusion of more SNPs in the model. Notably, the model that included the top 100 high-<em>I</em><sub><em>n</em></sub> SNPs exceeded the 90% accuracy threshold, demonstrating a substantial improvement in predictive performance. Further addition of AISNPs improved the assignment accuracy from 89.18% for 50 AISNPs to 98.44% for 2000 AISNPs. The RF model with 2000 SNPs (Model 5) demonstrated the highest predictive accuracy for the five subpopulations, reaching approximately 98.44%. Using these AISNPs, we developed a nested ancestry-informative panel to differentiate among the Han_North, Han_South, HM, TB, and TK populations, with overall accuracies of 89.18%, 90.99%, 94.59%, 95.43%, 97.36%, and 98.44%, respectively (Fig. <a href="#Fig6" class="usa-link">6</a>). Details of these panels are presented in Table <a href="#Tab1" class="usa-link">1</a>. The discrimination between the Han_North and Han_South, TB and HM, and TK and Han_South populations was more prone to error. For example, with the 50 AISNP panel, 6.37% of Han_South individuals were misclassified as Han_North, 15.79% as TK, and 10.00% of HM individuals were misclassified as TB. However, incorporating more AISNPs into the panels consistently reduced the error rates. Notably, the maximum error rate in distinguishing Han_South from TK decreased from 15.79% with the 50 AISNP panel to 2.63% with the 2000 AISNP panel.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dee/12359956/8586e7f5ea61/12864_2025_11947_Fig6_HTML.jpg" loading="lazy" id="MO6" height="511" width="646" alt="Fig. 6"></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance of the C5ClusterTag-50/100/250/500/1000/2000 ancestry informative panels. The accuracy achieved was over 89.18% for the first 50 SNPs, 90.99% for 100 SNPs, 94.59% for 250 SNPs, 95.43% for 500 SNPs, 97.36% for 1000 SNPs, 98.44% for 2000 SNPs, and 97.12% for 434,143 SNPs</p></figcaption></figure><section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Ancestry inference accuracies (%) of the 50, 100, 250, 500, 1000, and 2000 AISNP panels</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="6" rowspan="1">Predicted value</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">Han_North</th>
<th align="left" colspan="1" rowspan="1">Han_South</th>
<th align="left" colspan="1" rowspan="1">Hmong-Mien</th>
<th align="left" colspan="1" rowspan="1">Tibeto-Burman</th>
<th align="left" colspan="1" rowspan="1">Tai-Kadai</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="5" colspan="1">
<strong>True value</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>Han_North</strong>
</td>
<td align="left" colspan="1" rowspan="1">96.56/95.00/97.50/98.44/99.06/98.75</td>
<td align="center" colspan="1" rowspan="1">6.37/4.87/2.25/1.87/0.75/0.75</td>
<td align="center" colspan="1" rowspan="1">1.55/0.78/0/0/0/0</td>
<td align="left" colspan="1" rowspan="1">0/0/0/0/0/0</td>
<td align="left" colspan="1" rowspan="1">0/0/0/0/0/0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Han_South</strong>
</td>
<td align="left" colspan="1" rowspan="1">3.44/5.00/2.50/1.25/0.94/1.25</td>
<td align="center" colspan="1" rowspan="1">86.52/89.51/95.88/96.25/97.75/98.50</td>
<td align="center" colspan="1" rowspan="1">12.40/6.20/5.43/1.55/0/0.78</td>
<td align="left" colspan="1" rowspan="1">0/0/0/0/0/0</td>
<td align="left" colspan="1" rowspan="1">15.79/13.16/10.53/9.21/5.26/2.63</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Hmong-Mien</strong>
</td>
<td align="left" colspan="1" rowspan="1">0/0/0/0.31/0/0</td>
<td align="center" colspan="1" rowspan="1">3.00/1.12/0.37/0.37/0.75/0.37</td>
<td align="center" colspan="1" rowspan="1">79.07/86.05/89.92/91.47/96.12/98.45</td>
<td align="left" colspan="1" rowspan="1">10.00/7.50/12.50/12.50/10.00/2.50</td>
<td align="left" colspan="1" rowspan="1">0/0/0/0/0/0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Tibeto-Burman</strong>
</td>
<td align="left" colspan="1" rowspan="1">0/0/0/0/0/0</td>
<td align="center" colspan="1" rowspan="1">0/0.37/0.37/0.75/0.37/0</td>
<td align="center" colspan="1" rowspan="1">6.20/6.98/4.65/6.98/3.88/0.78</td>
<td align="left" colspan="1" rowspan="1">90.00/92.50/87.50/87.50/90.00/97.50</td>
<td align="left" colspan="1" rowspan="1">0/0/0/0/0/0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Tai-Kadai</strong>
</td>
<td align="left" colspan="1" rowspan="1">0/0/0/0/0/0</td>
<td align="center" colspan="1" rowspan="1">4.12/4.12/1.12/0.75/0.37/0.37</td>
<td align="center" colspan="1" rowspan="1">0.78/0/0/0/0/0</td>
<td align="left" colspan="1" rowspan="1">0/0/0/0/0/0</td>
<td align="left" colspan="1" rowspan="1">84.21/86.84/89.47/90.79/94.74/97.37</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par21">The panels showed strong performance in cross-validation assessments, exhibiting high sensitivity, specificity, PPV, and NPV (Table <a href="#Tab2" class="usa-link">2</a>). The average values for these metrics were as follows: 0.9211 for the 50 AISNP panel, 0.9339 for the 100 AISNP panel, 0.9550 for the 250 AISNP panel, 0.9570 for the 500 AISNP panel, 0.9737 for the 1000 AISNP panel, and 0.9888 for the 2000 AISNP panel.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Performance of the 50, 100, 250, 500, 1000, and 2000 AISNP panels on each subpopulation</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">Sensitivity</th>
<th align="left" colspan="1" rowspan="1">Specificity</th>
<th align="left" colspan="1" rowspan="1">PPV</th>
<th align="left" colspan="1" rowspan="1">NPV</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Han_North</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.9656/0.9500/0.9750/0.9844/0.9906/0.9875</td>
<td align="center" colspan="1" rowspan="1">0.9629/0.9727/0.9883/0.9902/0.9961/0.9961</td>
<td align="center" colspan="1" rowspan="1">0.9421/0.9560/0.9811/0.9844/0.9937/0.9937</td>
<td align="center" colspan="1" rowspan="1">0.9782/0.9689/0.9844/0.9902/0.9942/0.9922</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Han_South</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.8652/0.8951/0.9588/0.9625/0.9775/0.9850</td>
<td align="center" colspan="1" rowspan="1">0.9310/0.9398/0.9593/0.9770/0.9876/0.9876</td>
<td align="center" colspan="1" rowspan="1">0.8556/0.8755/0.9176/0.9519/0.9739/0.9741</td>
<td align="center" colspan="1" rowspan="1">0.9359/0.9499/0.9801/0.9822/0.9894/0.9929</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Hmong-Mien</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.7907/0.8605/0.8992/0.9147/0.9612/0.9845</td>
<td align="center" colspan="1" rowspan="1">0.9829/0.9915/0.9915/0.9900/0.9915/0.9972</td>
<td align="center" colspan="1" rowspan="1">0.8947/0.9487/0.9508/0.9440/0.9538/0.9845</td>
<td align="center" colspan="1" rowspan="1">0.9624/0.9748/0.9817/0.9844/0.9929/0.9972</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Tibeto-Burman</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.9000/0.9250/0.8750/0.8750/0.9000/0.9750</td>
<td align="center" colspan="1" rowspan="1">0.9899/0.9874/0.9912/0.9861/0.9924/0.9987</td>
<td align="center" colspan="1" rowspan="1">0.8182/0.7872/0.8333/0.7609/0.8571/0.9750</td>
<td align="center" colspan="1" rowspan="1">0.9949/0.9962/0.9937/0.9936/0.9949/0.9987</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Tai-Kadai</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.8421/0.8684/0.8947/0.9079/0.9474/0.9737</td>
<td align="center" colspan="1" rowspan="1">0.9841/0.9855/0.9960/0.9974/0.9987/0.9987</td>
<td align="center" colspan="1" rowspan="1">0.8421/0.8571/0.9578/0.9718/0.9863/0.9867</td>
<td align="center" colspan="1" rowspan="1">0.9841/0.9868/0.9895/0.9908/0.9947/0.9974</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec11"><h3 class="pmc_sec_title">Evaluation of model classification via ROC curves</h3>
<p id="Par23">The ROC curve illustrates the relationship between the true-positive rate and the false-positive rate of the test. The area under the curve (AUC) serves as an indicator of classification power, with values closer to one reflecting superior classification performance. We conducted ROC analysis on six sets of SNPs and plotted the corresponding ROC curves. The results indicate that the performance of the RF model improved as the number of AISNPs increased (Fig. <a href="#Fig7" class="usa-link">7</a>). Specifically, the AUC values for the panel with 2000 AISNPs ranged from 0.991 to 1.000, demonstrating excellent classification accuracy. Even with the smallest panel size of 50 markers, the AUC for each population was no less than 0.961, highlighting the robustness of our classification model.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359956_12864_2025_11947_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dee/12359956/5cafcbcb987d/12864_2025_11947_Fig7_HTML.jpg" loading="lazy" id="MO7" height="1100" width="750" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>RF model classification assessment via ROC analysis for 50</strong>,<strong> 100</strong>,<strong> 250</strong>,<strong> 500</strong>,<strong> 1000</strong>,<strong> and 2000 AISNPs.</strong> AUC values indicate the classification performance for each subpopulation</p></figcaption></figure></section></section><section id="Sec12"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par25">Genetic studies utilizing whole-genome sequencing and high-density SNP chip detection have revealed significant genetic substructures within East Asian populations, which closely correspond to linguistic classifications and geographic distributions [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>, <a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>, <a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. Despite extensive research on ancestry inference in forensic and medical genetics, challenges persist owing to variations in target populations across studies and the lack of high-coverage reference datasets [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. As a result, developing population-specific ancestry inference systems based on finer-scale genetic structures, along with the creation of comprehensive reference databases, could enhance the effective integration of ancestry inference into judicial practice.</p>
<p id="Par26">In this study, we identified an optimal subset of AIMs for ancestry inference among five major Chinese populations by analyzing whole-genome SNP data from 2,772 individuals. We then constructed six AISNP panels and applied a random forest algorithm to model the data, aiming to identify the optimal ancestry inference model for Chinese subpopulations based on metrics such as accuracy, sensitivity, and specificity. While the greatest degree of differentiation among the five populations was achieved via genome-wide data, which included over 434,143 SNPs, this approach is not feasible for forensic investigations. In contrast, the AISNP panels demonstrated strong discriminative power (Table <a href="#Tab1" class="usa-link">1</a>) and exhibited high sensitivity, specificity, PPV, and NPV in analyzing the five populations, as detailed in Table <a href="#Tab2" class="usa-link">2</a>. These results suggest that increasing the number of AISNPs enhances the accuracy of ancestry inference. Notably, the model using 1000 AISNPs showed greater accuracy in distinguishing the five subpopulations than the genome-wide approach with 434,143 SNPs. Thus, inferring ancestry through specific genetic markers is crucial for narrowing the search scope and improving investigative efficiency.</p>
<p id="Par27">The increasing number of genome-wide studies on these populations has revealed a north-south admixture pattern, as well as distinct genetic substructures across various geographical regions [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>, <a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. Our PCA revealed partial genetic overlap between the Han_North subgroup, comprising individuals from Henan and Shandong Provinces, and the Han_South subgroup, including individuals from Fujian and Guangdong Provinces. Our findings support the hypothesis that the primary ancestry of modern southeastern coastal Han Chinese individuals is derived from northern China, which is consistent with previous genome-wide analyses of Han Chinese individuals [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. The five ancestry components inferred at K = 5 not only reflect genetic differentiation but also correspond closely to known linguistic and geographic structures in East Asia. Furthermore, our UMAP analysis revealed significant local genetic structures, particularly within HM-speaking populations. These findings suggest the existence of genetic substructures that are influenced by linguistic affiliations, even within the context of larger geographical and linguistic populations, highlighting the complex genetic diversity and historical population dynamics within the Chinese population. Previous research has demonstrated significant genetic interactions between Hainan TK-speaking populations and adjacent Han Chinese regions [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]. Such genetic interactions could lead to inaccuracies in ancestry predictions. Notably, when ancestry inference panels are used to identify individuals from populations not represented in the training dataset, there is an increased risk of bias or error.</p>
<p id="Par28">Currently, systematic genetic studies on the Chinese population are limited, and no publicly available platforms, such as GEDMatch or consumer genomic databases, such as FamilyTree, provide ancestry-related data for forensic applications [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. Additionally, forensic ancestry inference systems specifically tailored to Chinese ethnic groups are still underdeveloped. Addressing these gaps requires large-scale and representative population sampling to establish a high-quality reference database that can support both forensic and population genetic studies [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. In this study, we developed an ancestry inference panel and introduced innovative tools for predicting the ancestry of ethnolinguistic Chinese populations. Moreover, we compiled a comprehensive reference dataset and constructed a machine-learning model applicable to both population genetics and forensic investigations. However, challenges remain in optimizing ancestry inference efficiency, particularly concerning the applicability of the 1000 and 2000 AISNP panels in forensic contexts. Our findings also highlight the need for further investigation into additional subpopulations and the identification of genetic markers with greater discriminatory power to enhance future ancestry inference models.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par29">We successfully developed six nested AISNP panels (C5ClusterTag-50/100/250/500/1000/2000) that demonstrate robust discriminative power in distinguishing five major Chinese populations: Han_North, Han_South, TB, HM, and TK-speaking populations. Notably, the 1000-AISNP panel offers an optimal balance between SNP density and discriminatory power, making it the most suitable for forensic applications. These six AISNP panels have proven valuable for population stratification within China, significantly improving the ability to differentiate closely related subpopulations. The RF ancestry-predictive models demonstrated high specificity and sensitivity, enabling precise differentiation and inference of the five ethnolinguistic subpopulations in China. These panels have successfully classified the populations into five distinct subgroups based on geographical regions or linguistic affiliations, achieving a high average accuracy rate in machine learning models. The developed AISNP panels are valuable tools for providing investigative leads and narrowing the scope of forensic investigations. Our findings contribute a new, highly effective method for ancestry inference in Chinese populations, with potential applications extending to both population genetics and forensic analysis.</p></section><section id="Sec14"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM1_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (1.3MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM2_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (1.5MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM3_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (1.1MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM4_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4.</a><sup> (713.3KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM5_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5.</a><sup> (2.7MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM6_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6.</a><sup> (2.2MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7.</a><sup> (17.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM8_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8.</a><sup> (206.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 9.</a><sup> (4MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 10.</a><sup> (9.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM11_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 11.</a><sup> (15.1KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM12"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM12_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 12.</a><sup> (11.4KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Not applicable.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>J.Y. and G.H. conceived and supervised the project. G.H. and M.W. collected the samples. G.H. and M.W. extracted the genomic DNA and coordinated the genome sequencing. J.C., Y.H. and J.Z. analyzed the data and wrote the draft of the manuscript. J.Y. and G.H. revised the manuscript. All authors reviewed the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the National Natural Science Foundation of China (82402203, 82030058, and 82202078), the Major Project of the National Social Science Foundation of China (23&amp;ZD203), the Open Project of the Key Laboratory of Forensic Genetics of the Ministry of Public Security (2022FGKFKT05), the Center for Archaeological Science of Sichuan University (23SASA01), the Sichuan Science and Technology Program (2024NSFSC1518) and the Shanxi Province Graduate Education Innovation Program Project (2023KY369).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data necessary to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials. The genomic variation data reported in this study have been deposited in the Genome Variation Map of the National Genomics Data Center, China National Center for Bioinformation/Beijing Institute of Genomics, Chinese Academy of Sciences (https://ngdc.cncb.ac.cn/gvm/) under the accession GVM000852. The data collection and usage adhered to the guidelines stipulated by the People’s Republic of China on the administration of human genetic resources. To comply with the informed-consent agreements, the data are available upon written request for population-history research only; they may not be used for medical, disease-related, or natural-selection studies, shared with third parties, exploited commercially, deposited in other public databases without prior written approval, or employed to identify any individual donor. Access requests require scientific review and a completed Material Transfer Agreement (MTA) through Sichuan University. Requests should be submitted to: Guanglin He (guanglinhescu@163.com). The use of genetic data in this work is approved by the National Health Commission of the People’s Republic of China (No. 2025BAT00384).</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par30">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par31">The authors declare no competing interests.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Ethics approval</h3>
<p id="Par32">The Medical Ethics Committees of West China Hospital of Sichuan University approved this study (2023 − 306).</p></section><section id="FPar4"><h3 class="pmc_sec_title">Human ethics and consent to participate in declarations</h3>
<p id="Par33">All participants provided informed consent, and the study was conducted in accordance with the principles of the Helsinki Declaration.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Mengge Wang, Email: Menggewang2021@163.com.</p>
<p>Guanglin He, Email: guanglinhescu@163.com.</p>
<p>Jiangwei Yan, Email: yanjw@sxmu.edu.cn.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Wang M, Huang Y, Liu K, Wang Z, Zhang M, Yuan H, Duan S, Wei L, Yao H, Sun QJMB et al. Multiple human population movements and cultural dispersal events shaped the landscape of Chinese paternal heritage. Mol Biol Evol. 2024; 41(7):msae122.</cite> [<a href="https://doi.org/10.1093/molbev/msae122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11232699/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38885310/" class="usa-link">PubMed</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Sun Y, Wang M, Sun Q, Liu Y, Duan S, Wang Z, Zhou Y, Zhong J, Huang Y, Huang XJJG et al. Distinguished biological adaptation architecture aggravated population differentiation of Tibeto-Burman-speaking people. J Genet Genomics. 2024;51(5):517–30.</cite> [<a href="https://doi.org/10.1016/j.jgg.2023.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37827489/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Ning C, Li T, Wang K, Zhang F, Li T, Wu X, Gao S, Zhang Q, Zhang H, Hudson MJ, et al. Ancient genomes from northern China suggest links between subsistence changes and human migration. Nat Commun. 2020;11(1):2700.
</cite> [<a href="https://doi.org/10.1038/s41467-020-16557-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7264253/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32483115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ning%20C,%20Li%20T,%20Wang%20K,%20Zhang%20F,%20Li%20T,%20Wu%20X,%20Gao%20S,%20Zhang%20Q,%20Zhang%20H,%20Hudson%20MJ,%20et%20al.%20Ancient%20genomes%20from%20northern%20China%20suggest%20links%20between%20subsistence%20changes%20and%20human%20migration.%20Nat%20Commun.%202020;11(1):2700." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Yang MA, Fan X, Sun B, Chen C, Lang J, Ko YC, Tsang CH, Chiu H, Wang T, Bao Q, et al. Ancient DNA indicates human population shifts and admixture in Northern and Southern China. Science. 2020;369(6501):282–8.
</cite> [<a href="https://doi.org/10.1126/science.aba0909" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32409524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20MA,%20Fan%20X,%20Sun%20B,%20Chen%20C,%20Lang%20J,%20Ko%20YC,%20Tsang%20CH,%20Chiu%20H,%20Wang%20T,%20Bao%20Q,%20et%20al.%20Ancient%20DNA%20indicates%20human%20population%20shifts%20and%20admixture%20in%20Northern%20and%20Southern%20China.%20Science.%202020;369(6501):282%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Wang M, Sun Q, Feng Y, Wei LH, Liu K, Luo L, Huang Y, Zhou K, Yuan H, Lv H, et al. Paleolithic divergence and multiple neolithic expansions of ancestral nomadic emperor-related paternal lineages. J Genet Genomics. 2025;52(4):502–12.
</cite> [<a href="https://doi.org/10.1016/j.jgg.2024.11.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39608672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20M,%20Sun%20Q,%20Feng%20Y,%20Wei%20LH,%20Liu%20K,%20Luo%20L,%20Huang%20Y,%20Zhou%20K,%20Yuan%20H,%20Lv%20H,%20et%20al.%20Paleolithic%20divergence%20and%20multiple%20neolithic%20expansions%20of%20ancestral%20nomadic%20emperor-related%20paternal%20lineages.%20J%20Genet%20Genomics.%202025;52(4):502%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Sun Y, Wang M, Sun Q, Liu Y, Duan S, Wang Z, Zhou Y, Zhong J, Huang Y, Huang X, et al. Distinguished biological adaptation architecture aggravated population differentiation of Tibeto-Burman-speaking people. J Genet Genomics. 2024;51(5):517–30.
</cite> [<a href="https://doi.org/10.1016/j.jgg.2023.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37827489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun%20Y,%20Wang%20M,%20Sun%20Q,%20Liu%20Y,%20Duan%20S,%20Wang%20Z,%20Zhou%20Y,%20Zhong%20J,%20Huang%20Y,%20Huang%20X,%20et%20al.%20Distinguished%20biological%20adaptation%20architecture%20aggravated%20population%20differentiation%20of%20Tibeto-Burman-speaking%20people.%20J%20Genet%20Genomics.%202024;51(5):517%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Sun Q, Wang M, Lu T, Duan S, Liu Y, Chen J, Wang Z, Sun Y, Li X, Wang S, et al. Differentiated adaptative genetic architecture and language-related demographical history in South China inferred from 619 genomes from 56 populations. BMC Biol. 2024;22(1):55.
</cite> [<a href="https://doi.org/10.1186/s12915-024-01854-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10918984/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38448908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun%20Q,%20Wang%20M,%20Lu%20T,%20Duan%20S,%20Liu%20Y,%20Chen%20J,%20Wang%20Z,%20Sun%20Y,%20Li%20X,%20Wang%20S,%20et%20al.%20Differentiated%20adaptative%20genetic%20architecture%20and%20language-related%20demographical%20history%20in%20South%20China%20inferred%20from%20619%20genomes%20from%2056%20populations.%20BMC%20Biol.%202024;22(1):55." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Li X, Wang M, Su H, Duan S, Sun Y, Chen H, Wang Z, Sun Q, Yang Q, Chen J, et al. Evolutionary history and biological adaptation of Han Chinese people on the Mongolian Plateau. hLife. 2024;2(6):296–313.</cite> [<a href="https://scholar.google.com/scholar_lookup?Li%20X,%20Wang%20M,%20Su%20H,%20Duan%20S,%20Sun%20Y,%20Chen%20H,%20Wang%20Z,%20Sun%20Q,%20Yang%20Q,%20Chen%20J,%20et%20al.%20Evolutionary%20history%20and%20biological%20adaptation%20of%20Han%20Chinese%20people%20on%20the%20Mongolian%20Plateau.%20hLife.%202024;2(6):296%E2%80%93313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>He G, Wang P, Chen J, Liu Y, Sun Y, Hu R, Duan S, Sun Q, Tang R, Yang J, et al. Differentiated genomic footprints suggest isolation and long-distance migration of Hmong-Mien populations. BMC Biol. 2024;22(1):18.
</cite> [<a href="https://doi.org/10.1186/s12915-024-01828-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10809681/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38273256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20G,%20Wang%20P,%20Chen%20J,%20Liu%20Y,%20Sun%20Y,%20Hu%20R,%20Duan%20S,%20Sun%20Q,%20Tang%20R,%20Yang%20J,%20et%20al.%20Differentiated%20genomic%20footprints%20suggest%20isolation%20and%20long-distance%20migration%20of%20Hmong-Mien%20populations.%20BMC%20Biol.%202024;22(1):18." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Abdulla MA, Ahmed I, Assawamakin A, Bhak J, Brahmachari SK, Calacal GC, Chaurasia A, Chen CH, Chen J, Chen YT, et al. Mapping human genetic diversity in Asia. Science. 2009;326(5959):1541–5.
</cite> [<a href="https://doi.org/10.1126/science.1177074" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20007900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abdulla%20MA,%20Ahmed%20I,%20Assawamakin%20A,%20Bhak%20J,%20Brahmachari%20SK,%20Calacal%20GC,%20Chaurasia%20A,%20Chen%20CH,%20Chen%20J,%20Chen%20YT,%20et%20al.%20Mapping%20human%20genetic%20diversity%20in%20Asia.%20Science.%202009;326(5959):1541%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>He G, Yao H, Duan S, Luo L, Sun Q, Tang R, Chen J, Wang Z, Sun Y, Li XJSCLS. Pilot work of the 10K Chinese people genomic diversity project along the silk road suggests a complex east-west admixture landscape and biological adaptations. Sci China Life Sci. 2025;68(4):914-33.</cite> [<a href="https://doi.org/10.1007/s11427-024-2748-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39862346/" class="usa-link">PubMed</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Luo L, Wang M, Liu Y, Li J, Bu F, Yuan H, Tang R, Liu C, He GJSCLS. Sequencing and characterizing human mitochondrial genomes in the biobank-based genomic research paradigm. Sci China Life Sci. 2025;68(6):1610-25.</cite> [<a href="https://doi.org/10.1007/s11427-024-2736-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39843848/" class="usa-link">PubMed</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>He G, Wang M, Luo L, Sun Q, Yuan H, Lv H, Feng Y, Liu X, Cheng J, Bu F, et al. Population genomics of central Asian peoples unveil ancient trans-Eurasian genetic admixture and cultural exchanges. hLife. 2024;2(11):554–62.</cite> [<a href="https://scholar.google.com/scholar_lookup?He%20G,%20Wang%20M,%20Luo%20L,%20Sun%20Q,%20Yuan%20H,%20Lv%20H,%20Feng%20Y,%20Liu%20X,%20Cheng%20J,%20Bu%20F,%20et%20al.%20Population%20genomics%20of%20central%20Asian%20peoples%20unveil%20ancient%20trans-Eurasian%20genetic%20admixture%20and%20cultural%20exchanges.%20hLife.%202024;2(11):554%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Yang Q, Sun Y, Duan S, Nie S, Liu C, Deng H, Wang M, He G. High-quality Population-specific Haplotype-resolved Reference Panel in the Genomic and Pangenomic Eras. Genomics, Proteomics &amp; Bioinformatics. 2025;Mar 10:qzaf022. </cite> [<a href="https://doi.org/10.1093/gpbjnl/qzaf022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40059317/" class="usa-link">PubMed</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Wang M, Duan S, Li X, Yang J, Yuan H, Liu C, He G. Genome-driven Chinese precision medicine: Biobank-scale genomic research as a new paradigm. The Innovation Life; 2025.</cite>
</li>
<li id="CR16">
<span class="label">16.</span><cite>Wang Z, Liu K, Yuan H, Duan S, Liu Y, Luo L, Jiang X, Chen S, Wei L-H, Tang R, et al. YanHuang paternal genomic resource suggested a Weakly-Differentiated Multi-Source Admixture model for the formation of Han’s founding ancestral lineages. Genomics, Proteomics &amp; Bioinformatics. 2025;Jun 4:qzaf049.</cite> [<a href="https://doi.org/10.1093/gpbjnl/qzaf049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40468594/" class="usa-link">PubMed</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>He G, Yao H, Duan S, Luo L, Sun Q, Tang R, Chen J, Wang Z, Sun Y, Li X, et al. Pilot work of the 10K Chinese people genomic diversity project along the silk road suggests a complex east-west admixture landscape and biological adaptations. Sci China Life Sci. 2025;68(4):914–33.
</cite> [<a href="https://doi.org/10.1007/s11427-024-2748-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39862346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20G,%20Yao%20H,%20Duan%20S,%20Luo%20L,%20Sun%20Q,%20Tang%20R,%20Chen%20J,%20Wang%20Z,%20Sun%20Y,%20Li%20X,%20et%20al.%20Pilot%20work%20of%20the%2010K%20Chinese%20people%20genomic%20diversity%20project%20along%20the%20silk%20road%20suggests%20a%20complex%20east-west%20admixture%20landscape%20and%20biological%20adaptations.%20Sci%20China%20Life%20Sci.%202025;68(4):914%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Wang M, Huang Y, Liu K, Wang Z, Zhang M, Yuan H, Duan S, Wei L, Yao H, Sun Q et al. Multiple human population movements and cultural dispersal events shaped the landscape of Chinese paternal heritage. Mol Biol Evol.2024;41(7):msae122.</cite> [<a href="https://doi.org/10.1093/molbev/msae122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11232699/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38885310/" class="usa-link">PubMed</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>He GL, Wang MG, Zou X, Yeh HY, Liu CH, Liu C, Chen G, Wang CC. Extensive ethnolinguistic diversity at the crossroads of North China and South Siberia reflects multiple sources of genetic diversity. J Syst Evol. 2022;61(1):230–50.</cite> [<a href="https://scholar.google.com/scholar_lookup?He%20GL,%20Wang%20MG,%20Zou%20X,%20Yeh%20HY,%20Liu%20CH,%20Liu%20C,%20Chen%20G,%20Wang%20CC.%20Extensive%20ethnolinguistic%20diversity%20at%20the%20crossroads%20of%20North%20China%20and%20South%20Siberia%20reflects%20multiple%20sources%20of%20genetic%20diversity.%20J%20Syst%20Evol.%202022;61(1):230%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>He G, Wang M, Luo L, Sun Q, Yuan H, Lv H, Feng Y, Liu X, Cheng J. Bu fjh: population genomics of central Asian peoples unveil ancient Trans-Eurasian genetic admixture and cultural exchanges. hLife. 2024;2(11):554–62.</cite>
</li>
<li id="CR21">
<span class="label">21.</span><cite>Zheng W, He Y, Guo Y, Yue T, Zhang H, Li J, Zhou B, Zeng X, Li L, Wang B, et al. Large-scale genome sequencing redefines the genetic footprints of high-altitude adaptation in Tibetans. Genome Biol. 2023;24(1):73.
</cite> [<a href="https://doi.org/10.1186/s13059-023-02912-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10099689/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37055782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng%20W,%20He%20Y,%20Guo%20Y,%20Yue%20T,%20Zhang%20H,%20Li%20J,%20Zhou%20B,%20Zeng%20X,%20Li%20L,%20Wang%20B,%20et%20al.%20Large-scale%20genome%20sequencing%20redefines%20the%20genetic%20footprints%20of%20high-altitude%20adaptation%20in%20Tibetans.%20Genome%20Biol.%202023;24(1):73." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Wang H, Yang MA, Wangdue S, Lu H, Chen H, Li L, Dong G, Tsring T, Yuan H, He W, et al. Human genetic history on the Tibetan plateau in the past 5100 years. Sci Adv. 2023;9(11):eadd5582.
</cite> [<a href="https://doi.org/10.1126/sciadv.add5582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10022901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36930720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20H,%20Yang%20MA,%20Wangdue%20S,%20Lu%20H,%20Chen%20H,%20Li%20L,%20Dong%20G,%20Tsring%20T,%20Yuan%20H,%20He%20W,%20et%20al.%20Human%20genetic%20history%20on%20the%20Tibetan%20plateau%20in%20the%20past%205100%20years.%20Sci%20Adv.%202023;9(11):eadd5582." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>He G, Li YX, Wang MG, Zou X, Yeh HY, Yang XM, Wang Z, Tang RK, Zhu SM, Guo JX, et al. Fine-scale genetic structure of Tujia and central Han Chinese revealing massive genetic admixture under Language borrowing. J Syst Evol. 2021;59(1):1–20.</cite> [<a href="https://scholar.google.com/scholar_lookup?He%20G,%20Li%20YX,%20Wang%20MG,%20Zou%20X,%20Yeh%20HY,%20Yang%20XM,%20Wang%20Z,%20Tang%20RK,%20Zhu%20SM,%20Guo%20JX,%20et%20al.%20Fine-scale%20genetic%20structure%20of%20Tujia%20and%20central%20Han%20Chinese%20revealing%20massive%20genetic%20admixture%20under%20Language%20borrowing.%20J%20Syst%20Evol.%202021;59(1):1%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>He GL, Li YX, Zou X, Yeh HY, Tang RK, Wang PX, Bai JY, Yang XM, Wang Z, Guo JX, et al. Northern gene flow into southeastern East Asians inferred from genome-wide array genotyping. J Syst Evol. 2023;61(1):179–97.</cite> [<a href="https://scholar.google.com/scholar_lookup?He%20GL,%20Li%20YX,%20Zou%20X,%20Yeh%20HY,%20Tang%20RK,%20Wang%20PX,%20Bai%20JY,%20Yang%20XM,%20Wang%20Z,%20Guo%20JX,%20et%20al.%20Northern%20gene%20flow%20into%20southeastern%20East%20Asians%20inferred%20from%20genome-wide%20array%20genotyping.%20J%20Syst%20Evol.%202023;61(1):179%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>He GL, Wang MG, Li YX, Zou X, Yeh HY, Tang RK, Yang XM, Wang Z, Guo JX, Luo T, et al. Fine-scale north-to-south genetic admixture profile in Shaanxi Han Chinese revealed by genome-wide demographic history reconstruction. J Syst Evol. 2022;60(4):955–72.</cite> [<a href="https://scholar.google.com/scholar_lookup?He%20GL,%20Wang%20MG,%20Li%20YX,%20Zou%20X,%20Yeh%20HY,%20Tang%20RK,%20Yang%20XM,%20Wang%20Z,%20Guo%20JX,%20Luo%20T,%20et%20al.%20Fine-scale%20north-to-south%20genetic%20admixture%20profile%20in%20Shaanxi%20Han%20Chinese%20revealed%20by%20genome-wide%20demographic%20history%20reconstruction.%20J%20Syst%20Evol.%202022;60(4):955%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Bai F, Liu Y, Wangdue S, Wang T, He W, Xi L, Tsho Y, Tsering T, Cao P, Dai QJCB. Ancient genomes revealed the complex human interactions of the ancient Western Tibetans. Curr Biol. 2024;34(12):2594–605. e7.</cite> [<a href="https://doi.org/10.1016/j.cub.2024.04.068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38781957/" class="usa-link">PubMed</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Wang MG, He GL, Zou X, Chen PY, Wang Z, Tang RK, Yang XM, Chen J, Yang MQ, Li YX, et al. Reconstructing the genetic admixture history of Tai-Kadai and sinitic people: insights from genome-wide SNP data from South China. J Syst Evol. 2023;61(1):157–78.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang%20MG,%20He%20GL,%20Zou%20X,%20Chen%20PY,%20Wang%20Z,%20Tang%20RK,%20Yang%20XM,%20Chen%20J,%20Yang%20MQ,%20Li%20YX,%20et%20al.%20Reconstructing%20the%20genetic%20admixture%20history%20of%20Tai-Kadai%20and%20sinitic%20people:%20insights%20from%20genome-wide%20SNP%20data%20from%20South%20China.%20J%20Syst%20Evol.%202023;61(1):157%E2%80%9378." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Tao Y, Wei Y, Ge J, Pan Y, Wang W, Bi Q, Sheng P, Fu C, Pan W, Jin L, et al. Phylogenetic evidence reveals early Kra-Dai divergence and dispersal in the late holocene. Nat Commun. 2023;14(1):6924.
</cite> [<a href="https://doi.org/10.1038/s41467-023-42761-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10616200/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37903755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tao%20Y,%20Wei%20Y,%20Ge%20J,%20Pan%20Y,%20Wang%20W,%20Bi%20Q,%20Sheng%20P,%20Fu%20C,%20Pan%20W,%20Jin%20L,%20et%20al.%20Phylogenetic%20evidence%20reveals%20early%20Kra-Dai%20divergence%20and%20dispersal%20in%20the%20late%20holocene.%20Nat%20Commun.%202023;14(1):6924." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>All of Us Research Program Genomics I. Genomic data in the all of Us research program. Nature. 2024;627(8003):340–6.
</cite> [<a href="https://doi.org/10.1038/s41586-023-06957-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10937371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38374255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?All%20of%20Us%20Research%20Program%20Genomics%20I.%20Genomic%20data%20in%20the%20all%20of%20Us%20research%20program.%20Nature.%202024;627(8003):340%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Byrska-Bishop M, Evani US, Zhao X, Basile AO, Abel HJ, Regier AA, Corvelo A, Clarke WE, Musunuri R, Nagulapalli K, et al. High-coverage whole-genome sequencing of the expanded 1000 genomes project cohort including 602 trios. Cell. 2022;185(18):3426–40. e3419.
</cite> [<a href="https://doi.org/10.1016/j.cell.2022.08.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9439720/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36055201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Byrska-Bishop%20M,%20Evani%20US,%20Zhao%20X,%20Basile%20AO,%20Abel%20HJ,%20Regier%20AA,%20Corvelo%20A,%20Clarke%20WE,%20Musunuri%20R,%20Nagulapalli%20K,%20et%20al.%20High-coverage%20whole-genome%20sequencing%20of%20the%20expanded%201000%20genomes%20project%20cohort%20including%20602%20trios.%20Cell.%202022;185(18):3426%E2%80%9340.%20e3419." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Taliun D, Harris DN, Kessler MD, Carlson J, Szpiech ZA, Torres R, Taliun SAG, Corvelo A, Gogarten SM, Kang HM, et al. Sequencing of 53,831 diverse genomes from the NHLBI topmed program. Nature. 2021;590(7845):290–9.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03205-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7875770/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33568819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Taliun%20D,%20Harris%20DN,%20Kessler%20MD,%20Carlson%20J,%20Szpiech%20ZA,%20Torres%20R,%20Taliun%20SAG,%20Corvelo%20A,%20Gogarten%20SM,%20Kang%20HM,%20et%20al.%20Sequencing%20of%2053,831%20diverse%20genomes%20from%20the%20NHLBI%20topmed%20program.%20Nature.%202021;590(7845):290%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>GenomeAsia KC. The genomeasia 100K project enables genetic discoveries across Asia. Nature. 2019;576(7785):106–11.
</cite> [<a href="https://doi.org/10.1038/s41586-019-1793-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7054211/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31802016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?GenomeAsia%20KC.%20The%20genomeasia%20100K%20project%20enables%20genetic%20discoveries%20across%20Asia.%20Nature.%202019;576(7785):106%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Genomes Project C, Auton A, Brooks LD, Durbin RM, Garrison EP, Kang HM, Korbel JO, Marchini JL, McCarthy S, McVean GA, et al. A global reference for human genetic variation. Nature. 2015;526(7571):68–74.
</cite> [<a href="https://doi.org/10.1038/nature15393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4750478/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Genomes%20Project%20C,%20Auton%20A,%20Brooks%20LD,%20Durbin%20RM,%20Garrison%20EP,%20Kang%20HM,%20Korbel%20JO,%20Marchini%20JL,%20McCarthy%20S,%20McVean%20GA,%20et%20al.%20A%20global%20reference%20for%20human%20genetic%20variation.%20Nature.%202015;526(7571):68%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Li JZ, Absher DM, Tang H, Southwick AM, Casto AM, Ramachandran S, Cann HM, Barsh GS, Feldman M, Cavalli-Sforza LL, et al. Worldwide human relationships inferred from genome-wide patterns of variation. Science. 2008;319(5866):1100–4.
</cite> [<a href="https://doi.org/10.1126/science.1153717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18292342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20JZ,%20Absher%20DM,%20Tang%20H,%20Southwick%20AM,%20Casto%20AM,%20Ramachandran%20S,%20Cann%20HM,%20Barsh%20GS,%20Feldman%20M,%20Cavalli-Sforza%20LL,%20et%20al.%20Worldwide%20human%20relationships%20inferred%20from%20genome-wide%20patterns%20of%20variation.%20Science.%202008;319(5866):1100%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Wang M, Chen H, Luo L, Huang Y, Duan S, Yuan H, Tang R, Liu C, He G. Forensic investigative genetic genealogy: expanding pedigree tracing and genetic inquiry in the genomic era. <em>Journal of genetics and genomics = Yi chuan xue bao</em> 2024.</cite> [<a href="https://doi.org/10.1016/j.jgg.2024.06.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38969261/" class="usa-link">PubMed</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Wang M, Chen H, Luo L, Huang Y, Duan S, Yuan H, Tang R, Liu C, He G. Forensic investigative genetic genealogy: expanding pedigree tracing and genetic inquiry in the genomic era. J Genet Genomics. 2025;52(4):460–72.
</cite> [<a href="https://doi.org/10.1016/j.jgg.2024.06.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38969261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20M,%20Chen%20H,%20Luo%20L,%20Huang%20Y,%20Duan%20S,%20Yuan%20H,%20Tang%20R,%20Liu%20C,%20He%20G.%20Forensic%20investigative%20genetic%20genealogy:%20expanding%20pedigree%20tracing%20and%20genetic%20inquiry%20in%20the%20genomic%20era.%20J%20Genet%20Genomics.%202025;52(4):460%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Luo L, Wang M, Liu Y, Li J, Bu F, Yuan H, Tang R, Liu C, He G. Sequencing and characterizing human mitochondrial genomes in the biobank-based genomic research paradigm. Sci China Life Sci.2025;68(6):1610-25.</cite> [<a href="https://doi.org/10.1007/s11427-024-2736-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39843848/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>He G, Liu C, Wang M. Perspectives and opportunities in forensic human, animal, and plant integrative genomics in the pangenome era. Forensic Sci Int. 2025;367:112370.
</cite> [<a href="https://doi.org/10.1016/j.forsciint.2025.112370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39813779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20G,%20Liu%20C,%20Wang%20M.%20Perspectives%20and%20opportunities%20in%20forensic%20human,%20animal,%20and%20plant%20integrative%20genomics%20in%20the%20pangenome%20era.%20Forensic%20Sci%20Int.%202025;367:112370." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Wang Z, Wang M, Hu L, He G, Nie S. Evolutionary profiles and complex admixture landscape in East Asia: new insights from modern and ancient Y chromosome variation perspectives. Heliyon. 2024;10(9):e30067.
</cite> [<a href="https://doi.org/10.1016/j.heliyon.2024.e30067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11096704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38756579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Z,%20Wang%20M,%20Hu%20L,%20He%20G,%20Nie%20S.%20Evolutionary%20profiles%20and%20complex%20admixture%20landscape%20in%20East%20Asia:%20new%20insights%20from%20modern%20and%20ancient%20Y%20chromosome%20variation%20perspectives.%20Heliyon.%202024;10(9):e30067." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Pakstis AJ, Kang L, Liu L, Zhang Z, Jin T, Grigorenko EL, Wendt FR, Budowle B, Hadi S, Al Qahtani MS, et al. Increasing the reference populations for the 55 AISNP panel: the need and benefits. Int J Legal Med. 2017;131(4):913–7.
</cite> [<a href="https://doi.org/10.1007/s00414-016-1524-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5491587/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28070634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pakstis%20AJ,%20Kang%20L,%20Liu%20L,%20Zhang%20Z,%20Jin%20T,%20Grigorenko%20EL,%20Wendt%20FR,%20Budowle%20B,%20Hadi%20S,%20Al%20Qahtani%20MS,%20et%20al.%20Increasing%20the%20reference%20populations%20for%20the%2055%20AISNP%20panel:%20the%20need%20and%20benefits.%20Int%20J%20Legal%20Med.%202017;131(4):913%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Pakstis AJ, Gurkan C, Dogan M, Balkaya HE, Dogan S, Neophytou PI, Cherni L, Boussetta S, Khodjet-El-Khil H et al. Ben Ammar ElGaaied A: Genetic relationships of European, Mediterranean, and SW Asian populations using a panel of 55 AISNPs. Eur J Hum Genet. 2019;27(12):1885– 93.</cite> [<a href="https://doi.org/10.1038/s41431-019-0466-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6871633/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31285530/" class="usa-link">PubMed</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Pakstis AJ, Haigh E, Cherni L, ElGaaied ABA, Barton A, Evsanaa B, Togtokh A, Brissenden J, Roscoe J, Bulbul O, et al. 52 additional reference population samples for the 55 AISNP panel. Forensic Sci Int Genet. 2015;19:269–71.
</cite> [<a href="https://doi.org/10.1016/j.fsigen.2015.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26355664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pakstis%20AJ,%20Haigh%20E,%20Cherni%20L,%20ElGaaied%20ABA,%20Barton%20A,%20Evsanaa%20B,%20Togtokh%20A,%20Brissenden%20J,%20Roscoe%20J,%20Bulbul%20O,%20et%20al.%2052%20additional%20reference%20population%20samples%20for%20the%2055%20AISNP%20panel.%20Forensic%20Sci%20Int%20Genet.%202015;19:269%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Kosoy R, Nassir R, Tian C, White PA, Butler LM, Silva G, Kittles R, Alarcon-Riquelme ME, Gregersen PK, Belmont JW, et al. Ancestry informative marker sets for determining continental origin and admixture proportions in common populations in America. Hum Mutat. 2009;30(1):69–78.
</cite> [<a href="https://doi.org/10.1002/humu.20822" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3073397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18683858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kosoy%20R,%20Nassir%20R,%20Tian%20C,%20White%20PA,%20Butler%20LM,%20Silva%20G,%20Kittles%20R,%20Alarcon-Riquelme%20ME,%20Gregersen%20PK,%20Belmont%20JW,%20et%20al.%20Ancestry%20informative%20marker%20sets%20for%20determining%20continental%20origin%20and%20admixture%20proportions%20in%20common%20populations%20in%20America.%20Hum%20Mutat.%202009;30(1):69%E2%80%9378." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Zhang S, Zhu Q, Chen X, Zhao Y, Zhao X, Yang Y, Gao Z, Fang T, Wang Y, Zhang J. Forensic applicability of multi-allelic indels with mononucleotide homopolymer structures. Electrophoresis. 2018;39(16):2136–43.
</cite> [<a href="https://doi.org/10.1002/elps.201700468" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29704251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20S,%20Zhu%20Q,%20Chen%20X,%20Zhao%20Y,%20Zhao%20X,%20Yang%20Y,%20Gao%20Z,%20Fang%20T,%20Wang%20Y,%20Zhang%20J.%20Forensic%20applicability%20of%20multi-allelic%20indels%20with%20mononucleotide%20homopolymer%20structures.%20Electrophoresis.%202018;39(16):2136%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Fondevila M, Phillips C, Santos C, Freire Aradas A, Vallone PM, Butler JM, Lareu MV, Carracedo A. Revision of the SNPforID 34-plex forensic ancestry test: assay enhancements, standard reference sample genotypes and extended population studies. Forensic Sci Int Genet. 2013;7(1):63–74.
</cite> [<a href="https://doi.org/10.1016/j.fsigen.2012.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22749789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fondevila%20M,%20Phillips%20C,%20Santos%20C,%20Freire%20Aradas%20A,%20Vallone%20PM,%20Butler%20JM,%20Lareu%20MV,%20Carracedo%20A.%20Revision%20of%20the%20SNPforID%2034-plex%20forensic%20ancestry%20test:%20assay%20enhancements,%20standard%20reference%20sample%20genotypes%20and%20extended%20population%20studies.%20Forensic%20Sci%20Int%20Genet.%202013;7(1):63%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Lan Q, Shen C, Jin X, Guo Y, Xie T, Chen C, Cui W, Fang Y, Yang G, Zhu B. Distinguishing three distinct biogeographic regions with an in-house developed 39-AIM-InDel panel and further admixture proportion Estimation for Uyghurs. Electrophoresis. 2019;40(11):1525–34.
</cite> [<a href="https://doi.org/10.1002/elps.201800448" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30758063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lan%20Q,%20Shen%20C,%20Jin%20X,%20Guo%20Y,%20Xie%20T,%20Chen%20C,%20Cui%20W,%20Fang%20Y,%20Yang%20G,%20Zhu%20B.%20Distinguishing%20three%20distinct%20biogeographic%20regions%20with%20an%20in-house%20developed%2039-AIM-InDel%20panel%20and%20further%20admixture%20proportion%20Estimation%20for%20Uyghurs.%20Electrophoresis.%202019;40(11):1525%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Jin XY, Cui W, Chen C, Guo YX, Tao YW, Lan Q, Kong TT, Zhu BF. Biogeographic origin prediction of three continental populations through 42 ancestry informative SNPs. Electrophoresis. 2020;41(3–4):235–45.
</cite> [<a href="https://doi.org/10.1002/elps.201900241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31743462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin%20XY,%20Cui%20W,%20Chen%20C,%20Guo%20YX,%20Tao%20YW,%20Lan%20Q,%20Kong%20TT,%20Zhu%20BF.%20Biogeographic%20origin%20prediction%20of%20three%20continental%20populations%20through%2042%20ancestry%20informative%20SNPs.%20Electrophoresis.%202020;41(3%E2%80%934):235%E2%80%9345." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Sun K, Yun L, Zhang C, Shao C, Gao T, Zhao Z, Hou Y, Xie J, Tang Q. Evaluation of 12 Multi-InDel markers for forensic ancestry prediction in Asian populations. Forensic Sci Int Genet. 2019;43:102155.
</cite> [<a href="https://doi.org/10.1016/j.fsigen.2019.102155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31479932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun%20K,%20Yun%20L,%20Zhang%20C,%20Shao%20C,%20Gao%20T,%20Zhao%20Z,%20Hou%20Y,%20Xie%20J,%20Tang%20Q.%20Evaluation%20of%2012%20Multi-InDel%20markers%20for%20forensic%20ancestry%20prediction%20in%20Asian%20populations.%20Forensic%20Sci%20Int%20Genet.%202019;43:102155." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Fan G, Ye Y, Luo H, Hou Y. Use of multi-InDels as novel markers to analyze 13 X-chromosome haplotype loci for forensic purposes. Electrophoresis. 2015;36(23):2931–8.
</cite> [<a href="https://doi.org/10.1002/elps.201500159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26265037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fan%20G,%20Ye%20Y,%20Luo%20H,%20Hou%20Y.%20Use%20of%20multi-InDels%20as%20novel%20markers%20to%20analyze%2013%20X-chromosome%20haplotype%20loci%20for%20forensic%20purposes.%20Electrophoresis.%202015;36(23):2931%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Oldoni F, Kidd KK, Podini D. Microhaplotypes in forensic genetics. Forensic Sci Int Genet. 2019;38:54–69.
</cite> [<a href="https://doi.org/10.1016/j.fsigen.2018.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30347322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oldoni%20F,%20Kidd%20KK,%20Podini%20D.%20Microhaplotypes%20in%20forensic%20genetics.%20Forensic%20Sci%20Int%20Genet.%202019;38:54%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Soundararajan U, Yun L, Shi M, Kidd KK. Minimal SNP overlap among multiple panels of ancestry informative markers argues for more international collaboration. Forensic Sci Int Genet. 2016;23:25–32.
</cite> [<a href="https://doi.org/10.1016/j.fsigen.2016.01.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26977931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Soundararajan%20U,%20Yun%20L,%20Shi%20M,%20Kidd%20KK.%20Minimal%20SNP%20overlap%20among%20multiple%20panels%20of%20ancestry%20informative%20markers%20argues%20for%20more%20international%20collaboration.%20Forensic%20Sci%20Int%20Genet.%202016;23:25%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Kidd KK, Speed WC, Pakstis AJ, Furtado MR, Fang R, Madbouly A, Maiers M, Middha M, Friedlaender FR, Kidd JR. Progress toward an efficient panel of SNPs for ancestry inference. Forensic Sci Int Genet. 2014;10:23–32.
</cite> [<a href="https://doi.org/10.1016/j.fsigen.2014.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24508742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kidd%20KK,%20Speed%20WC,%20Pakstis%20AJ,%20Furtado%20MR,%20Fang%20R,%20Madbouly%20A,%20Maiers%20M,%20Middha%20M,%20Friedlaender%20FR,%20Kidd%20JR.%20Progress%20toward%20an%20efficient%20panel%20of%20SNPs%20for%20ancestry%20inference.%20Forensic%20Sci%20Int%20Genet.%202014;10:23%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Shi CM, Liu Q, Zhao S, Chen H. Ancestry informative SNP panels for discriminating the major East Asian populations: Han chinese, Japanese and Korean. Ann Hum Genet. 2019;83(5):348–54.
</cite> [<a href="https://doi.org/10.1111/ahg.12320" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31025319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi%20CM,%20Liu%20Q,%20Zhao%20S,%20Chen%20H.%20Ancestry%20informative%20SNP%20panels%20for%20discriminating%20the%20major%20East%20Asian%20populations:%20Han%20chinese,%20Japanese%20and%20Korean.%20Ann%20Hum%20Genet.%202019;83(5):348%E2%80%9354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Qin P, Li Z, Jin W, Lu D, Lou H, Shen J, Jin L, Shi Y, Xu S. A panel of ancestry informative markers to estimate and correct potential effects of population stratification in Han Chinese. Eur J Hum Genet. 2014;22(2):248–53.
</cite> [<a href="https://doi.org/10.1038/ejhg.2013.111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3895631/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23714748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qin%20P,%20Li%20Z,%20Jin%20W,%20Lu%20D,%20Lou%20H,%20Shen%20J,%20Jin%20L,%20Shi%20Y,%20Xu%20S.%20A%20panel%20of%20ancestry%20informative%20markers%20to%20estimate%20and%20correct%20potential%20effects%20of%20population%20stratification%20in%20Han%20Chinese.%20Eur%20J%20Hum%20Genet.%202014;22(2):248%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Walters RG, Millwood IY, Lin K, Schmidt Valle D, McDonnell P, Hacker A, Avery D, Edris A, Fry H, Cai N, et al. Genotyping and population characteristics of the China kadoorie biobank. Cell Genom. 2023;3(8):100361.
</cite> [<a href="https://doi.org/10.1016/j.xgen.2023.100361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10435379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37601966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Walters%20RG,%20Millwood%20IY,%20Lin%20K,%20Schmidt%20Valle%20D,%20McDonnell%20P,%20Hacker%20A,%20Avery%20D,%20Edris%20A,%20Fry%20H,%20Cai%20N,%20et%20al.%20Genotyping%20and%20population%20characteristics%20of%20the%20China%20kadoorie%20biobank.%20Cell%20Genom.%202023;3(8):100361." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Cheng S, Xu Z, Bian S, Chen X, Shi Y, Li Y, Duan Y, Liu Y, Lin J, Jiang Y, et al. The STROMICS genome study: deep whole-genome sequencing and analysis of 10K Chinese patients with ischemic stroke reveal complex genetic and phenotypic interplay. Cell Discov. 2023;9(1):75.
</cite> [<a href="https://doi.org/10.1038/s41421-023-00582-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10362040/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37479695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng%20S,%20Xu%20Z,%20Bian%20S,%20Chen%20X,%20Shi%20Y,%20Li%20Y,%20Duan%20Y,%20Liu%20Y,%20Lin%20J,%20Jiang%20Y,%20et%20al.%20The%20STROMICS%20genome%20study:%20deep%20whole-genome%20sequencing%20and%20analysis%20of%2010K%20Chinese%20patients%20with%20ischemic%20stroke%20reveal%20complex%20genetic%20and%20phenotypic%20interplay.%20Cell%20Discov.%202023;9(1):75." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Zhang P, Luo H, Li Y, Wang Y, Wang J, Zheng Y, Niu Y, Shi Y, Zhou H, Song T, et al. NyuWa genome resource: A deep whole-genome sequencing-based variation profile and reference panel for the Chinese population. Cell Rep. 2021;37(7):110017.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2021.110017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34788621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20P,%20Luo%20H,%20Li%20Y,%20Wang%20Y,%20Wang%20J,%20Zheng%20Y,%20Niu%20Y,%20Shi%20Y,%20Zhou%20H,%20Song%20T,%20et%20al.%20NyuWa%20genome%20resource:%20A%20deep%20whole-genome%20sequencing-based%20variation%20profile%20and%20reference%20panel%20for%20the%20Chinese%20population.%20Cell%20Rep.%202021;37(7):110017." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Li L, Huang P, Sun X, Wang S, Xu M, Liu S, Feng Z, Zhang Q, Wang X, Zheng X, et al. The ChinaMAP reference panel for the accurate genotype imputation in Chinese populations. Cell Res. 2021;31(12):1308–10.
</cite> [<a href="https://doi.org/10.1038/s41422-021-00564-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8648815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34489580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20L,%20Huang%20P,%20Sun%20X,%20Wang%20S,%20Xu%20M,%20Liu%20S,%20Feng%20Z,%20Zhang%20Q,%20Wang%20X,%20Zheng%20X,%20et%20al.%20The%20ChinaMAP%20reference%20panel%20for%20the%20accurate%20genotype%20imputation%20in%20Chinese%20populations.%20Cell%20Res.%202021;31(12):1308%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Yang C, Zhang X, Yan S, Yang S, Wu B, You F, Cui Y, Xie N, Wang Z, Jin LJNHB. Large-scale lexical and genetic alignment supports a hybrid model of Han Chinese Demic and cultural diffusions. Nat Hum Behav. 2024;8(6):1163-76.</cite> [<a href="https://doi.org/10.1038/s41562-024-01886-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38740988/" class="usa-link">PubMed</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Li CX, Pakstis AJ, Jiang L, Wei YL, Sun QF, Wu H, Bulbul O, Wang P, Kang LL, Kidd JR, et al. A panel of 74 aisnps: improved ancestry inference within Eastern Asia. Forensic Sci Int Genet. 2016;23:101–10.
</cite> [<a href="https://doi.org/10.1016/j.fsigen.2016.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27077960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20CX,%20Pakstis%20AJ,%20Jiang%20L,%20Wei%20YL,%20Sun%20QF,%20Wu%20H,%20Bulbul%20O,%20Wang%20P,%20Kang%20LL,%20Kidd%20JR,%20et%20al.%20A%20panel%20of%2074%20aisnps:%20improved%20ancestry%20inference%20within%20Eastern%20Asia.%20Forensic%20Sci%20Int%20Genet.%202016;23:101%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>He G, Wang Z, Wang M, Luo T, Liu J, Zhou Y, Gao B, Hou Y. Forensic ancestry analysis in two Chinese minority populations using massively parallel sequencing of 165 ancestry-informative SNPs. Electrophoresis. 2018;39(21):2732–42.
</cite> [<a href="https://doi.org/10.1002/elps.201800019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29869338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20G,%20Wang%20Z,%20Wang%20M,%20Luo%20T,%20Liu%20J,%20Zhou%20Y,%20Gao%20B,%20Hou%20Y.%20Forensic%20ancestry%20analysis%20in%20two%20Chinese%20minority%20populations%20using%20massively%20parallel%20sequencing%20of%20165%20ancestry-informative%20SNPs.%20Electrophoresis.%202018;39(21):2732%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Wang Z, He G, Luo T, Zhao X, Liu J, Wang M, Zhou D, Chen X, Li C, Hou Y. Massively parallel sequencing of 165 ancestry informative SNPs in two Chinese Tibetan-Burmese minority ethnicities. Forensic Sci Int Genet. 2018;34:141–7.
</cite> [<a href="https://doi.org/10.1016/j.fsigen.2018.02.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29477877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Z,%20He%20G,%20Luo%20T,%20Zhao%20X,%20Liu%20J,%20Wang%20M,%20Zhou%20D,%20Chen%20X,%20Li%20C,%20Hou%20Y.%20Massively%20parallel%20sequencing%20of%20165%20ancestry%20informative%20SNPs%20in%20two%20Chinese%20Tibetan-Burmese%20minority%20ethnicities.%20Forensic%20Sci%20Int%20Genet.%202018;34:141%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>He G, Liu J, Wang M, Zou X, Ming T, Zhu S, Yeh HY, Wang C, Wang Z, Hou Y. Massively parallel sequencing of 165 ancestry-informative SNPs and forensic biogeographical ancestry inference in three Southern Chinese Sinitic/Tai-Kadai populations. Forensic Sci Int Genet. 2021;52:102475.
</cite> [<a href="https://doi.org/10.1016/j.fsigen.2021.102475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33561661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20G,%20Liu%20J,%20Wang%20M,%20Zou%20X,%20Ming%20T,%20Zhu%20S,%20Yeh%20HY,%20Wang%20C,%20Wang%20Z,%20Hou%20Y.%20Massively%20parallel%20sequencing%20of%20165%20ancestry-informative%20SNPs%20and%20forensic%20biogeographical%20ancestry%20inference%20in%20three%20Southern%20Chinese%20Sinitic/Tai-Kadai%20populations.%20Forensic%20Sci%20Int%20Genet.%202021;52:102475." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Manichaikul A, Mychaleckyj JC, Rich SS, Daly K, Sale M, Chen WM. Robust relationship inference in genome-wide association studies. Bioinformatics. 2010;26(22):2867–73.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3025716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20926424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manichaikul%20A,%20Mychaleckyj%20JC,%20Rich%20SS,%20Daly%20K,%20Sale%20M,%20Chen%20WM.%20Robust%20relationship%20inference%20in%20genome-wide%20association%20studies.%20Bioinformatics.%202010;26(22):2867%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience. 2015;4:7.
</cite> [<a href="https://doi.org/10.1186/s13742-015-0047-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4342193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25722852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chang%20CC,%20Chow%20CC,%20Tellier%20LC,%20Vattikuti%20S,%20Purcell%20SM,%20Lee%20JJ.%20Second-generation%20PLINK:%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets.%20Gigascience.%202015;4:7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Li C, Tian D, Tang B, Liu X, Teng X, Zhao W, Zhang Z, Song S. Genome variation map: a worldwide collection of genome variations across multiple species. Nucleic Acids Res. 2021;49(D1):D1186–91.
</cite> [<a href="https://doi.org/10.1093/nar/gkaa1005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33170268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20C,%20Tian%20D,%20Tang%20B,%20Liu%20X,%20Teng%20X,%20Zhao%20W,%20Zhang%20Z,%20Song%20S.%20Genome%20variation%20map:%20a%20worldwide%20collection%20of%20genome%20variations%20across%20multiple%20species.%20Nucleic%20Acids%20Res.%202021;49(D1):D1186%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Database Resources of the National Genomics Data Center. China National center for bioinformation in 2022. Nucleic Acids Res. 2022;50(D1):D27–38.
</cite> [<a href="https://doi.org/10.1093/nar/gkab951" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728233/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34718731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Database%20Resources%20of%20the%20National%20Genomics%20Data%20Center.%20China%20National%20center%20for%20bioinformation%20in%202022.%20Nucleic%20Acids%20Res.%202022;50(D1):D27%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Daya M, van der Merwe L, Galal U, Möller M, Salie M, Chimusa ER, Galanter JM, van Helden PD, Henn BM, Gignoux CR, et al. A panel of ancestry informative markers for the complex five-way admixed South African coloured population. PLoS One. 2013;8(12):e82224.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0082224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3869660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24376522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Daya%20M,%20van%20der%20Merwe%20L,%20Galal%20U,%20M%C3%B6ller%20M,%20Salie%20M,%20Chimusa%20ER,%20Galanter%20JM,%20van%20Helden%20PD,%20Henn%20BM,%20Gignoux%20CR,%20et%20al.%20A%20panel%20of%20ancestry%20informative%20markers%20for%20the%20complex%20five-way%20admixed%20South%20African%20coloured%20population.%20PLoS%20One.%202013;8(12):e82224." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Rosenberg NA, Li LM, Ward R, Pritchard JK. Informativeness of genetic markers for inference of ancestry. Am J Hum Genet. 2003;73(6):1402–22.
</cite> [<a href="https://doi.org/10.1086/380416" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1180403/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14631557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosenberg%20NA,%20Li%20LM,%20Ward%20R,%20Pritchard%20JK.%20Informativeness%20of%20genetic%20markers%20for%20inference%20of%20ancestry.%20Am%20J%20Hum%20Genet.%202003;73(6):1402%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Patterson N, Price AL, Reich D. Population structure and eigenanalysis. PLoS Genet. 2006;2(12):e190.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.0020190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1713260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17194218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patterson%20N,%20Price%20AL,%20Reich%20D.%20Population%20structure%20and%20eigenanalysis.%20PLoS%20Genet.%202006;2(12):e190." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Patterson N, Moorjani P, Luo Y, Mallick S, Rohland N, Zhan Y, Genschoreck T, Webster T, Reich D. Ancient admixture in human history. Genetics. 2012;192(3):1065–93.
</cite> [<a href="https://doi.org/10.1534/genetics.112.145037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3522152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patterson%20N,%20Moorjani%20P,%20Luo%20Y,%20Mallick%20S,%20Rohland%20N,%20Zhan%20Y,%20Genschoreck%20T,%20Webster%20T,%20Reich%20D.%20Ancient%20admixture%20in%20human%20history.%20Genetics.%202012;192(3):1065%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Diaz-Papkovich A, Anderson-Trocmé L, Gravel S. A review of UMAP in population genetics. J Hum Genet. 2021;66(1):85–91.
</cite> [<a href="https://doi.org/10.1038/s10038-020-00851-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7728596/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33057159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diaz-Papkovich%20A,%20Anderson-Trocm%C3%A9%20L,%20Gravel%20S.%20A%20review%20of%20UMAP%20in%20population%20genetics.%20J%20Hum%20Genet.%202021;66(1):85%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Huang S, Liu S, Huang M, He JR, Wang C, Wang T, Feng X, Kuang Y, Lu J, Gu Y, et al. The born in Guangzhou cohort study enables generational genetic discoveries. Nature. 2024;626(7999):565–73.
</cite> [<a href="https://doi.org/10.1038/s41586-023-06988-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38297123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20S,%20Liu%20S,%20Huang%20M,%20He%20JR,%20Wang%20C,%20Wang%20T,%20Feng%20X,%20Kuang%20Y,%20Lu%20J,%20Gu%20Y,%20et%20al.%20The%20born%20in%20Guangzhou%20cohort%20study%20enables%20generational%20genetic%20discoveries.%20Nature.%202024;626(7999):565%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Luo H, Zhang P, Zhang W, Zheng Y, Hao D, Shi Y, Niu Y, Song T, Li Y, Zhao S et al. Recent positive selection signatures reveal phenotypic evolution in the Han Chinese population. Sci Bull (Beijing). 2023;68(20):2391-404.</cite> [<a href="https://doi.org/10.1016/j.scib.2023.08.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37661541/" class="usa-link">PubMed</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Wen Y, Liu J, Su Y, Chen X, Hou Y, Liao L, Wang Z. Forensic biogeographical ancestry inference: recent insights and current trends. Genes Genomics. 2023;45(10):1229–38.
</cite> [<a href="https://doi.org/10.1007/s13258-023-01387-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37081293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wen%20Y,%20Liu%20J,%20Su%20Y,%20Chen%20X,%20Hou%20Y,%20Liao%20L,%20Wang%20Z.%20Forensic%20biogeographical%20ancestry%20inference:%20recent%20insights%20and%20current%20trends.%20Genes%20Genomics.%202023;45(10):1229%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Luo T, Wang R, Wang CC. Inferring the population structure and admixture history of three Hmong-Mien-speaking Miao tribes from Southwest China based on genome-wide SNP genotyping. Ann Hum Biol. 2021;48(5):418–29.
</cite> [<a href="https://doi.org/10.1080/03014460.2021.2005825" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34763584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luo%20T,%20Wang%20R,%20Wang%20CC.%20Inferring%20the%20population%20structure%20and%20admixture%20history%20of%20three%20Hmong-Mien-speaking%20Miao%20tribes%20from%20Southwest%20China%20based%20on%20genome-wide%20SNP%20genotyping.%20Ann%20Hum%20Biol.%202021;48(5):418%E2%80%9329." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Wang Y, Zou X, Wang M, Yuan D, Yang L, Zeng Y, Cheng F, Tang R, He G. The genomic history of Southwestern Chinese populations demonstrated massive population migration and admixture among proto-Hmong-Mien speakers and incoming migrants. Mol Genet Genomics: MGG. 2022;297(1):241–62.
</cite> [<a href="https://doi.org/10.1007/s00438-021-01837-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35031862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Y,%20Zou%20X,%20Wang%20M,%20Yuan%20D,%20Yang%20L,%20Zeng%20Y,%20Cheng%20F,%20Tang%20R,%20He%20G.%20The%20genomic%20history%20of%20Southwestern%20Chinese%20populations%20demonstrated%20massive%20population%20migration%20and%20admixture%20among%20proto-Hmong-Mien%20speakers%20and%20incoming%20migrants.%20Mol%20Genet%20Genomics:%20MGG.%202022;297(1):241%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>He G, Wang Z, Guo J, Wang M, Zou X, Tang R, Liu J, Zhang H, Li Y, Hu R, et al. Inferring the population history of Tai-Kadai-speaking people and southernmost Han Chinese on Hainan Island by genome-wide array genotyping. Eur J Hum Genet. 2020;28(8):1111–23.
</cite> [<a href="https://doi.org/10.1038/s41431-020-0599-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7381617/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32123326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20G,%20Wang%20Z,%20Guo%20J,%20Wang%20M,%20Zou%20X,%20Tang%20R,%20Liu%20J,%20Zhang%20H,%20Li%20Y,%20Hu%20R,%20et%20al.%20Inferring%20the%20population%20history%20of%20Tai-Kadai-speaking%20people%20and%20southernmost%20Han%20Chinese%20on%20Hainan%20Island%20by%20genome-wide%20array%20genotyping.%20Eur%20J%20Hum%20Genet.%202020;28(8):1111%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM1_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (1.3MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM2_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (1.5MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM3_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (1.1MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM4_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4.</a><sup> (713.3KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM5_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5.</a><sup> (2.7MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM6_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6.</a><sup> (2.2MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7.</a><sup> (17.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM8_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8.</a><sup> (206.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 9.</a><sup> (4MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 10.</a><sup> (9.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM11_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 11.</a><sup> (15.1KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359956/bin/12864_2025_11947_MOESM12_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 12.</a><sup> (11.4KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data necessary to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials. The genomic variation data reported in this study have been deposited in the Genome Variation Map of the National Genomics Data Center, China National Center for Bioinformation/Beijing Institute of Genomics, Chinese Academy of Sciences (https://ngdc.cncb.ac.cn/gvm/) under the accession GVM000852. The data collection and usage adhered to the guidelines stipulated by the People’s Republic of China on the administration of human genetic resources. To comply with the informed-consent agreements, the data are available upon written request for population-history research only; they may not be used for medical, disease-related, or natural-selection studies, shared with third parties, exploited commercially, deposited in other public databases without prior written approval, or employed to identify any individual donor. Access requests require scientific review and a completed Material Transfer Agreement (MTA) through Sichuan University. Requests should be submitted to: Guanglin He (guanglinhescu@163.com). The use of genetic data in this work is approved by the National Health Commission of the People’s Republic of China (No. 2025BAT00384).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Genomics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12864-025-11947-6"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12864_2025_Article_11947.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12359956/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12359956/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359956%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359956/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12359956/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12359956/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40826445/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12359956/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40826445/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12359956/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12359956/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="q6ItZ9XxUmif2UROwqweOLnS05cZworJ75tJDu70Gxcjj5FsNwS2AlloDUUNwCZS">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
