
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Culture-free detection of bacteria from blood for rapid sepsis diagnosis - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E484E18AEF1C030584E100336EE4FA.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="npjdigitmed">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375708/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="NPJ Digital Medicine">
<meta name="citation_title" content="Culture-free detection of bacteria from blood for rapid sepsis diagnosis">
<meta name="citation_author" content="M Henar Marino Miguélez">
<meta name="citation_author_institution" content="Micro and Nanosystems, KTH Royal Institute of Technology, Stockholm, Sweden">
<meta name="citation_author_institution" content="Molecular Systems Biology, Uppsala University, Uppsala, Sweden">
<meta name="citation_author" content="Mohammad Osaid">
<meta name="citation_author_institution" content="Micro and Nanosystems, KTH Royal Institute of Technology, Stockholm, Sweden">
<meta name="citation_author" content="Erik Hallström">
<meta name="citation_author_institution" content="Department of Information Technology, Uppsala University, Uppsala, Sweden">
<meta name="citation_author" content="Kerem Kaya">
<meta name="citation_author_institution" content="Micro and Nanosystems, KTH Royal Institute of Technology, Stockholm, Sweden">
<meta name="citation_author" content="Jimmy Larsson">
<meta name="citation_author_institution" content="Molecular Systems Biology, Uppsala University, Uppsala, Sweden">
<meta name="citation_author" content="Vinodh Kandavalli">
<meta name="citation_author_institution" content="Molecular Systems Biology, Uppsala University, Uppsala, Sweden">
<meta name="citation_author" content="Carolina Wählby">
<meta name="citation_author_institution" content="Department of Information Technology, Uppsala University, Uppsala, Sweden">
<meta name="citation_author_institution" content="SciLifeLab, Uppsala University, Uppsala, Sweden">
<meta name="citation_author" content="Johan Elf">
<meta name="citation_author_institution" content="Molecular Systems Biology, Uppsala University, Uppsala, Sweden">
<meta name="citation_author_institution" content="SciLifeLab, Uppsala University, Uppsala, Sweden">
<meta name="citation_author" content="Wouter van der Wijngaart">
<meta name="citation_author_institution" content="Micro and Nanosystems, KTH Royal Institute of Technology, Stockholm, Sweden">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="8">
<meta name="citation_firstpage" content="544">
<meta name="citation_doi" content="10.1038/s41746-025-01948-w">
<meta name="citation_pmid" content="40851034">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375708/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375708/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375708/pdf/41746_2025_Article_1948.pdf">
<meta name="description" content="Approximately 50 million people suffer from sepsis yearly, and 13 million die from it. For every hour a patient with septic shock is untreated, their survival rate decreases by 8%. Therefore, rapid detection and antibiotic susceptibility profiling ...">
<meta name="og:title" content="Culture-free detection of bacteria from blood for rapid sepsis diagnosis">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Approximately 50 million people suffer from sepsis yearly, and 13 million die from it. For every hour a patient with septic shock is untreated, their survival rate decreases by 8%. Therefore, rapid detection and antibiotic susceptibility profiling ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375708/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375708">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41746-025-01948-w"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41746_2025_Article_1948.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375708%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375708/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375708/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375708/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-npjdigitmed.jpg" alt="NPJ Digital Medicine logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NPJ Digital Medicine" title="Link to NPJ Digital Medicine" shape="default" href="https://www.nature.com/npjdigitalmed/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">NPJ Digit Med</button></div>. 2025 Aug 25;8:544. doi: <a href="https://doi.org/10.1038/s41746-025-01948-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41746-025-01948-w</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22NPJ%20Digit%20Med%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22NPJ%20Digit%20Med%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22NPJ%20Digit%20Med%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22NPJ%20Digit%20Med%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Culture-free detection of bacteria from blood for rapid sepsis diagnosis</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marino%20Migu%C3%A9lez%20MH%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">M Henar Marino Miguélez</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">M Henar Marino Miguélez</span></h3>
<div class="p">
<sup>1</sup>Micro and Nanosystems, KTH Royal Institute of Technology, Stockholm, Sweden </div>
<div class="p">
<sup>2</sup>Molecular Systems Biology, Uppsala University, Uppsala, Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marino%20Migu%C3%A9lez%20MH%22%5BAuthor%5D" class="usa-link"><span class="name western">M Henar Marino Miguélez</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Osaid%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Mohammad Osaid</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Mohammad Osaid</span></h3>
<div class="p">
<sup>1</sup>Micro and Nanosystems, KTH Royal Institute of Technology, Stockholm, Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Osaid%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohammad Osaid</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hallstr%C3%B6m%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Erik Hallström</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Erik Hallström</span></h3>
<div class="p">
<sup>3</sup>Department of Information Technology, Uppsala University, Uppsala, Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hallstr%C3%B6m%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Erik Hallström</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaya%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Kerem Kaya</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kerem Kaya</span></h3>
<div class="p">
<sup>1</sup>Micro and Nanosystems, KTH Royal Institute of Technology, Stockholm, Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaya%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kerem Kaya</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Larsson%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jimmy Larsson</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jimmy Larsson</span></h3>
<div class="p">
<sup>2</sup>Molecular Systems Biology, Uppsala University, Uppsala, Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Larsson%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jimmy Larsson</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kandavalli%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Vinodh Kandavalli</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Vinodh Kandavalli</span></h3>
<div class="p">
<sup>2</sup>Molecular Systems Biology, Uppsala University, Uppsala, Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kandavalli%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Vinodh Kandavalli</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22W%C3%A4hlby%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Carolina Wählby</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Carolina Wählby</span></h3>
<div class="p">
<sup>3</sup>Department of Information Technology, Uppsala University, Uppsala, Sweden </div>
<div class="p">
<sup>4</sup>SciLifeLab, Uppsala University, Uppsala, Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22W%C3%A4hlby%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Carolina Wählby</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elf%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Johan Elf</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Johan Elf</span></h3>
<div class="p">
<sup>2</sup>Molecular Systems Biology, Uppsala University, Uppsala, Sweden </div>
<div class="p">
<sup>4</sup>SciLifeLab, Uppsala University, Uppsala, Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elf%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Johan Elf</span></a>
</div>
</div>
<sup>2,</sup><sup>4,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20der%20Wijngaart%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Wouter van der Wijngaart</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Wouter van der Wijngaart</span></h3>
<div class="p">
<sup>1</sup>Micro and Nanosystems, KTH Royal Institute of Technology, Stockholm, Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20der%20Wijngaart%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wouter van der Wijngaart</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Micro and Nanosystems, KTH Royal Institute of Technology, Stockholm, Sweden </div>
<div id="Aff2">
<sup>2</sup>Molecular Systems Biology, Uppsala University, Uppsala, Sweden </div>
<div id="Aff3">
<sup>3</sup>Department of Information Technology, Uppsala University, Uppsala, Sweden </div>
<div id="Aff4">
<sup>4</sup>SciLifeLab, Uppsala University, Uppsala, Sweden </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jun 27; Accepted 2025 Aug 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375708  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40851034/" class="usa-link">40851034</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Approximately 50 million people suffer from sepsis yearly, and 13 million die from it. For every hour a patient with septic shock is untreated, their survival rate decreases by 8%. Therefore, rapid detection and antibiotic susceptibility profiling of bacterial agents in the blood of sepsis patients are crucial for determining appropriate treatment. Here, we introduce a method to isolate bacteria from whole blood with high separation efficiency through <em>Smart centrifugation</em>, followed by microfluidic trapping and subsequent detection using deep learning applied to microscopy images. We detected, within 2 h, <em>E. coli</em>, <em>K. pneumoniae</em>, or <em>E. faecalis</em> from spiked samples of healthy human donor blood at clinically relevant concentrations as low as 9, 7 and 32 colony-forming units per ml of blood, respectively. However, the detection of <em>S. aureus</em> remains a challenge. This rapid isolation and detection represents a significant advancement towards culture-free detection of bloodstream infections.</p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c7c1/12375708/a739a8f2cae7/41746_2025_1948_Figa_HTML.jpg" loading="lazy" id="d33e245" height="235" width="633" alt="graphic file with name 41746_2025_1948_Figa_HTML.jpg"></p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Biological techniques, Diseases, Medical research, Microbiology</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Sepsis is a severe medical condition characterized by a systemic host inflammatory response to an infection<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. With an incidence of ~50 million cases annually,<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> sepsis represents a significant healthcare burden. The mortality of diagnosed patients is around 26%<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>, and chances of survival drop dramatically if treatment is delayed<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. An estimated 25–30% of sepsis cases involve bloodstream infections (BSI)<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Rapid detection and identification of pathogens in the patient’s blood is therefore important in order to initiate effective antibiotic treatment quickly.</p>
<p id="Par4">The low microbial loads in the bloodstream of patients, as low as one to ten colony-forming units (CFUs) per ml of blood<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>, require large blood sample volumes, which is particularly challenging for pediatric patients<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Furthermore, to confirm the presence of bacteria in the blood and identify the species, several hours to days of culture are usually needed<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. This delay is problematic considering the 8%-drop in the survival rate per hour of delayed treatment for patients suffering from septic shock<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. The urgency of the condition, coupled with the latency of the available diagnosis methods, means that combination therapy with broad-range antibiotics is often prescribed as first-line therapy immediately, before the result of the diagnostic test has been obtained<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. This praxis results in suboptimal treatment<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, contributes to increased antibiotic resistance<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, and has been shown to increase liver toxicity compared to targeted antibiotic monotherapy<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>.</p>
<p id="Par5">The state-of-the-art methods for identifying bacteria from the blood of sepsis patients can be divided into<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> genotypic (e.g., polymerase chain reaction), phenotypic (e.g., subcultures), or mass spectrometry methods (e.g., matrix-assisted laser desorption ionisation time-of-flight mass spectrometry (MALDI-TOF))<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. While genotypic methods can identify bacteria at low concentrations without culturing the blood<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, they do not provide susceptibility information<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, limiting clinical impact. Phenotypic and proteomic approaches typically require a higher bacterial load, necessitating blood cultures and thus long processing times.</p>
<p id="Par6">Traditional antibiotic susceptibility testing (AST) methods in the clinical setting rely on disk diffusion or broth microdilution, which can take several hours up to a day to yield results<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. These methods rely on isolated cultures which require two previous culture steps, thus significantly increasing the total turnaround times to several days. New rapid EUCAST methodologies have significantly shortened the process, enabling testing directly from blood cultures, and providing results in as little as 4 to 8 h<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>–<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Moreover, in recent years, substantial progress has been made in developing novel AST methods that can be applied directly to positive blood cultures. For instance, systems like the PA-100 from SYSMEX has been shown to predict phenotypic resistance from positive blood cultures<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. There are also 3D microwell-based chips that can perform AST using resazurin as a sensing probe<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Specific metabolic enzyme-based AST, such as fluorescence lifetime imaging microscopy (FILM), which measures metabolic perturbations after antibiotic exposure, can differentiate susceptible and resistant bacteria within 10–60 min<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. In addition, there are pathogen-specific metabolic enzymes that serve as reliable AST biomarkers and can enable AST within 30 min<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>–<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>.</p>
<p id="Par7">Further, the emergence of artificial intelligence (AI) systems has given rise to AI-powered technologies that use machine learning to interpret complex growth patterns, image-based phenotypes, or spectral data. AI algorithms embedded in platforms like Accelerate Pheno and dRAST analyze real-time bacterial responses, enhancing accuracy and reducing time to result to 4–7 h<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. From a commercial perspective, companies like Q-linea, Gradientech, and QuantaMatrix offer integrated solutions capable of delivering AST results directly from positive blood culture samples within 4 to 6 h<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Molecular AST platforms, such as FilmArray (bioMérieux, Inc., Hazelwood, MO) and Verigene (Nanosphere, Northbrook, IL), can provide identification of several species and a narrow panel of antimicrobial resistance markers<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Despite these significant advances, the main bottleneck remains in the culture time required to obtain a positive blood culture, which can take several days for certain fastidious organisms<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Our approach focuses on solving this bottleneck.</p>
<p id="Par8">Single-cell phenotypic methods, which bypass the requirement for blood culture, offer a quicker alternative but require that the bacteria be isolated from the excess of host blood cells<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>–<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p>
<p id="Par9">Isolation of bacteria from whole blood has been accomplished with inertial<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>–<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> and elastoinertial microfluidics<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>, sedimentation velocity-based separation<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>–<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, filtration<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, chemical capture<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>, magnetic bead-based separation<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>, dielectrophoresis<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>, or acoustic separation<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. However, most of these isolation methods suffer limited throughput and low blood cell rejection rates, or have only been demonstrated for bacteria concentrations of 1000 CFU/ml or above, thus necessitating time-consuming bacterial preculture.</p>
<p id="Par10">This study aims to develop a rapid, high-throughput assay for isolating bacteria from whole blood with high efficiency and minimal dilution by trapping them in a microfluidic chip. In the chip, the bacteria are optically identified with a deep learning-based detection algorithm. The whole assay takes less than a few hours.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Overall work-flow</h3>
<p id="Par11">Our assay concatenates five steps to isolate and detect bacteria from blood: smart centrifugation, selective blood cell lysis, volume reduction, microfluidic trapping combined with miscroscopy imaging, and deep-learning based detection of bacterial cells (Fig. <a href="#Fig1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Workflow for bacterial detection from blood samples in five assay steps.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375708_41746_2025_1948_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c7c1/12375708/c0c63a49d164/41746_2025_1948_Fig1_HTML.jpg" loading="lazy" id="d33e495" height="191" width="691" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>1</strong> Isolation using smart centrifugation; <strong>2</strong> selective blood cell lysis; <strong>3</strong> volume reduction; <strong>4</strong> microfluidic trapping and microscopy imaging; and <strong>5</strong> deep-learning based bacterial detection. BCM is blood culture medium. Created in BioRender. Lab, E. (2025) <a href="https://BioRender.com/gyvhthf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/gyvhthf</a>and in open-source software Inkscape.</p></figcaption></figure><p id="Par12">To ensure reproducibility during the technical development, this study was conducted with healthy human EDTA donor blood, with all the blood parameters within the normal count. The blood was spiked with bacteria at concentrations, C, in the range 4–4000 CFU/ml. To facilitate development and quantification, we used a fluorescent lab <em>Escherichia coli</em> strain. The fluorescence facilitated the easy distinction between the <em>E. coli</em> test strain and potential contaminating bacteria, none of which were observed. Method evaluation was performed with clinical isolates of <em>Klebsiella pneumoniae</em>, <em>Enterococcus faecalis</em>, and <em>Staphylococcus aureus</em>, in their exponential growth phase, to resemble the clinical scenario<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>.</p>
<p id="Par13">Details about materials and methods are provided under section “Methods and all measurement values” are tabled in SI section “Tabled experimental measurements and their results”.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Smart centrifugation</h3>
<p id="Par14">The first assay step, which we call smart centrifugation, removes most of the blood cells while recovering most of the bacteria in the supernatant. This step is essential to avoid downstream clogging of the microfluidic device. During blood centrifugation, bacteria are enriched into the supernatant (see SI section “Cell and fluid movement during smart centrifugation” for a detailed description of bacterial cell trajectories during blood sedimentation). However, a large fraction of the bacteria are trapped in the plasma remaining in the blood cell sediment. Our strategy to avoid such bacterial loss is to layer the sample on top of a high-density medium with a volume sufficient to replace all plasma in the sediment. We adjusted the volumes and densities of both the sample and the density medium to achieve optimal bacterial isolation efficiency with minimal dilution. To tune the densities of the blood sample and density medium, we used blood culture medium (BCM) to support bacterial growth throughout the assay.</p>
<p id="Par15">The densities of RBCs, WBCs, and platelets are in the ranges 1.086–1.122 g/ml, 1.057–1.092 g/ml and 1.072–1.077 g/ml<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, respectively. As density medium, we chose a 2:1 volumetric mixture of Lymphoprep (STEMCELL Technologies, Canada) and BCM, which has a density of 1.051 g/ml. This mixing ratio increased the medium’s density, improving particle separation during sedimentation, while remaining low enough to allow even the least dense blood cells to settle into the medium, thereby ensuring effective separation of all blood cells from the plasma. The volume of the density medium was experimentally tuned to be small but sufficient to replace all plasma remaining between the sedimented cells. Diluting the blood sample with 25% BCM ensured that the sample’s density was below the density of the medium used for separation while only slightly reducing the bacterial concentration. Centrifuging time and force were experimentally tuned for optimal enrichment of <em>E. coli</em> in the supernatant while removing at least 99.8% of the RBCs. Details of our experimental deduction of these parameters are provided in SI.</p>
<p id="Par16">The optimized procedure involved layering 3 ml of BCM-diluted spiked blood on top of 1 ml density medium and centrifuging for 5 min at 600 × <em>g</em> in a hanging bucket centrifuge. The relative movements of the RBCs, bacteria, and liquid in such a system are described with a linear model in SI Section “Cell and fluid movement during smart centrifugation” and illustrated in Fig. <a href="#Fig2" class="usa-link">2</a>a. After centrifugation, ~2.5 ml of clear supernatant containing most bacteria (SI Fig. <a href="#MOESM1" class="usa-link">4</a>) could be removed for further processing.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Bacterial isolation from blood by smart centrifugation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375708_41746_2025_1948_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c7c1/12375708/fb5741730f78/41746_2025_1948_Fig2_HTML.jpg" loading="lazy" id="d33e633" height="649" width="700" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>
<em>Illustration of liquid and cell movement</em>. The left and right tubes illustrate the positions of sample liquid, density medium, red blood cells and bacteria before and after smart centrifugation. The middle graph qualitatively illustrates the trajectories (solid lines) of bacteria (green) and red blood cells (red) during centrifugation, from a mixed state (left brackets) to a separated state (right brackets). The slopes of the lines are the particle sedimentation speeds, <em>v</em><sub><em>R</em><em>B</em><em>C</em></sub> and <em>v</em><sub><em>b</em><em>a</em><em>c</em></sub>, and liquid interface, <em>v</em><sub><em>i</em><em>n</em><em>t</em></sub>, and sedimentation interface velocity, <em>v</em><sub><em>s</em><em>e</em><em>d</em></sub>, derived in SI. <strong>b</strong>
<em>Blood cell removal efficiency</em>, meaning the fraction of blood cells removed from the supernatant after centrifugation relative to the initial number of blood cells in the sample (n=3). <strong>c</strong>
<em>Bacterial isolation efficiency</em>, meaning the number of colony-forming units in the supernatant after centrifugation relative to the initial number of colony-forming units in the spiked sample. Bar heights are mean; error bars are sd; n.s. and *** indicate significance levels <em>p</em> &gt; 0.05 and <em>p</em> ≤ 0.001, respectively; the bacterial concentration <em>C</em> refers to the CFU /ml in the blood sample. Each data point corresponds to an individual experiment. A total of <em>n</em> = 70 samples were tested. The figures were created and edited using the open-source software Inkscape (<strong>a</strong>) and MATLAB R2021 (<strong>b</strong>) and (<strong>c</strong>).</p></figcaption></figure><p id="Par17">By counting the blood cells in the supernatant, we showed the we could remove 99.82 ± 0.04% of the RBCs, 95 ± 4% of the WBCs, and 63 ± 2% of the platelets (mean ± sd, <em>n</em> = 3; Fig. <a href="#Fig2" class="usa-link">2</a>b). By plating the supernatant on agar plates and counting the resulting colonies, we demonstrated that, out of the spiked bacteria, we could recover 65 ± 16% of <em>E. coli</em>, 95 ± 17% of <em>K. pneumoniae</em>, 64 ± 24% of <em>E. faecalis</em>, or 8 ± 7% of <em>S. aureus</em> (mean ± sd, <em>n</em> = 10–26; Fig. <a href="#Fig2" class="usa-link">2</a>c). Recovery exceeding 100% can be attributed to low colony counts, leading to considerable variations<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Selective blood cell lysis</h3>
<p id="Par18">In the second step of the assay, the remaining blood cells in the sample are removed by selective lysis using a mixture of sodium cholate hydrate and saponin<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Approximately 2.5 ml supernatant from the smart centrifugation step was mixed with 1 ml of the selective lysing solution and kept in a shaking incubator at 37 °C for 10 min, completely lysing remaining RBCs, WBCs, and platelets. Previous studies indicate a limited effect of the lysing solution on bacterial viability<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Volume reduction</h3>
<p id="Par19">The third assay step enriches the sample and removes the excess lysing buffer in a second centrifugation step. The sample-lysate mixture was layered on top of 0.3 ml high-density liquid, consisting of a 1:2 volumetric mixture of percoll and BCM, and centrifuged for 13 min at 1000 × <em>g</em> to sediment the bacterial cells on the liquid interface. The supernatant was removed, leaving ~0.5 ml liquid containing sedimented bacterial cells, as well as some blood lysate. Sedimenting the sample into a higher-density liquid resulted in a smoother liquid flow in the microfluidic chip during downstream processing (SI Fig. <a href="#MOESM1" class="usa-link">5</a>), while not significantly affecting the bacterial isolation efficiency (<em>p</em>-value &gt;0.05) (SI Fig. <a href="#MOESM1" class="usa-link">6</a>).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Bacterial microfluidic trapping</h3>
<p id="Par20">In the last step, we loaded the 0.5 ml bacterial sample on a microfluidic analysis chip at a pressure of 1 bar, keeping all outlets at atmospheric pressure. The design of the chip is the one introduced by Baltekin et al.<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> for cross-flow filtration of bacteria into individual filter traps. We ran the resuspended sample through the microchip for 70 min to allow the bacteria to enter the traps. The chips contained in total 8000 microtraps of length 50 μm, height 1.25 μm and width 1.25 μ. A 300 nm restriction at the end of the trap prevents the bacteria from escaping into the back channel while still allowing liquid to flow over the cells. This design makes each microtrap function as a miniature culture chamber. The microtraps were monitored with fluorescence-microscopy to detect <em>E. coli</em>; alternatively phase-contrast microscopy to detect the clinical isolates of <em>K. pneumoniae</em>, <em>E. faecalis</em>, and <em>S. aureus</em>.</p>
<p id="Par21">For <em>E. coli</em>-spiked samples, ~30% of the sample flowed through the traps with a typical filtrate flow rate of 1–2 μl/min ((SI Fig. <a href="#MOESM1" class="usa-link">5b</a>) and (SI Fig. <a href="#MOESM1" class="usa-link">7a</a>)). The trapping efficiency, meaning the number of positive traps, <em>N</em>, relative to the number of CFUs after smart centrifugation, was 29 ± 6% (mean ± sd, <em>n</em> = 5; SI Fig. <a href="#MOESM1" class="usa-link">7b</a>). The overall detection rates (%), meaning the number of cells observed on chip relative to the number of CFUs in the initial sample, ranged from 5 to 44%, depending on the bacterial species, as shown in SI Fig. <a href="#MOESM1" class="usa-link">8</a>.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Microfluidic trapping efficiency</h3>
<p id="Par22">We evaluated the entire assay with blood spiked with <em>E. coli</em>, <em>K. pneumoniae</em>, and <em>E. faecalis</em>, respectively. All experiments resulted in successful trapping and subsequent detection of bacteria.</p>
<p id="Par23">The lowest bacterial concentrations, <em>C</em>, tested and detected were 9, 7 and 32 CFU/ml, respectively (Fig. <a href="#Fig3" class="usa-link">3</a>a). Fitting a least mean square linear curve, <em>N</em> = <em>η</em> ⋅ <em>C</em>, to the data allowed us to estimate the overall assay sensitivity, <em>η</em>. We used the data collected from the first manual labeling of the image data set, to avoid data bias. We could infer that the detection limits, which are the bacterial concentrations for which we expect <em>N</em> = 1 positive trap, <em>C</em><sub><em>N</em>=1</sub> = 1/<em>η</em>, were in the range 1–10 CFU/ml. After trapping, the bacterial cells divide inside the microchannels, while the sample is still being loaded (Fig. <a href="#Fig3" class="usa-link">3</a>b).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Bacterial microtrapping.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375708_41746_2025_1948_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c7c1/12375708/a0a37c3b33e9/41746_2025_1948_Fig3_HTML.jpg" loading="lazy" id="d33e829" height="318" width="700" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>
<em>Overall assay performance</em>, where dots represent the number of positive microtraps detected, <em>N</em>, for various bacterial concentrations in the 2.25 ml blood sample, <em>C</em>, and lines depict the least square fitting linear calibration curves <em>N</em> = <em>η</em> ⋅ <em>C</em>. Each point corresponds to an individual experiment. A total of <em>n</em> = 12 samples were tested. <strong>b</strong>
<em>Timelapse images of bacterial capture and growth in a single microtrap</em>, showing the capture of one bacterium of <em>K. pneumoniae</em> 40 min after sample addition, followed by bacterial cell division. The figure was created with Microsoft Excel and edited and assembled using the open-source software Inkscape.</p></figcaption></figure><p id="Par24">Due to poor isolation efficiency of the smart centrifugation step, we could not detect <em>S. aureus</em> (<em>n</em> = 2).</p></section><section id="Sec9"><h3 class="pmc_sec_title">Deep-learning based detection of bacteria</h3>
<p id="Par25">We trained deep learning models to automatically detect the presence of bacteria in 8-frame (70 min) 1300 × 40 pixel (height, width) time-lapse image series of microfluidic traps, posing the detection as a video classification problem. The dataset was collected through phase-contrast microscopy, capturing bacterial growth in the microfluidic device and then using a pre-processing pipeline to find and crop out individual traps. Each image of every time-lapse was manually labeled, tagging the presence of bacteria in each frame. The training set totaled 57,608 time-lapses (24,220 positives and 33,388 empty at 70 min; SI Table <a href="#MOESM1" class="usa-link">4</a>), and the test set 66,869 time-lapses (721 positives and 66,148 empty at 70 min; SI Table <a href="#MOESM1" class="usa-link">5</a>), of data from eight different experiments. A trap has a positive label if it contains or previously contained any frames with bacteria until the current evaluation time point. The number of positive instances changes over time as cells get trapped in previously empty traps, shown in SI Table <a href="#MOESM1" class="usa-link">6</a>. Additionally, we performed negative control experiments (<em>n</em> = 3), using non-spiked sterile blood, each containing a total of 8000 traps, to further evaluate the specificity of the detection.</p>
<p id="Par26">First, time-lapse frames were concatenated horizontally to a frame montage, allowing us to use image classification networks for the video detection task. We trained and compared three similarly sized models: ResNet 18<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> from 2016 (11.2M parameters, 8.4 GFLOPs), EfficientNet B2<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> from 2021 (7.7M parameters 3.2 GFLOPs), and the more modern foundation model DinoV2 Small Patch 14<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> from 2023 (21.8M parameters 24.8 GFLOPs). DinoV2 is a standard vision transformer, ViT<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>, trained using self-supervision on a large dataset with the DinoV2 methodology; for simplicity, we hereafter refer to it as DinoV2. Then, we employed the Video ResNet R(2+1)D<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> from 2018 (31.3M parameters 47.5 GFLOPs), processing the unaltered time-lapse video directly. The evaluation was conducted by progressively increasing the number of frames in the test time-lapse movies, measuring the performance over time (0–70 min) (SI Fig. <a href="#MOESM1" class="usa-link">11a</a>). Also, we trained and tested the models using spatially downsampled images to assess the complexity of the task and evaluate how much performance remains as image detail is progressively lost (SI Fig. <a href="#MOESM1" class="usa-link">11b</a>). Each network and downsampling step was retrained 30 times with a specific random seed for reproducibility and better statistics. The evaluation metrics used were precision, defined as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e888"><mfrac><mrow><mspace width="0.25em"></mspace><mi mathvariant="normal">True</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">Positives</mi></mrow><mrow><mi mathvariant="normal">True</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">Positives</mi><mo>+</mo><mi mathvariant="normal">False</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">Positives</mi><mspace width="0.25em"></mspace></mrow></mfrac></math></span>, recall (sensitivity), defined as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e917"><mfrac><mrow><mspace width="0.25em"></mspace><mi mathvariant="normal">True</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">Positives</mi></mrow><mrow><mi mathvariant="normal">True</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">Positives</mi><mo>+</mo><mi mathvariant="normal">False</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">Negatives</mi><mspace width="0.25em"></mspace></mrow></mfrac></math></span>, and the F1-score, which is defined as the harmonic mean of precision and recall: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e946"><mrow><msub><mrow><mi>F</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>=</mo><mn>2</mn><mo>⋅</mo><mfrac><mrow><mspace width="0.25em"></mspace><mi mathvariant="normal">precision</mi><mo>⋅</mo><mi mathvariant="normal">recall</mi></mrow><mrow><mi mathvariant="normal">precision</mi><mo>+</mo><mi mathvariant="normal">recall</mi></mrow></mfrac></mrow></math></span>. Given the large test set imbalance, we chose the F1-Score as an evaluation metric because this method takes into account both the precision and recall. Additionally, ROC (Receiver Operating Characteristic) curves and AUC (Area Under the Curve) scores were calculated. At 70 min, the mean and standard deviation F1 scores were 85.5 ± 2.8% for Video ResNet R(2+1)D, 87.3 ± 2.2% for ResNet 18, 90.1 ± 2.3% for EfficientNet B2 and 93.1 ± 1.6% for DinoV2, respectively (full resolution). Figure <a href="#Fig4" class="usa-link">4</a> shows the confusion matrix, downsampling, and time evaluation heatmap for DinoV2 (the best-performing model), along with the time evaluation classification metrics for all models at full resolution. For the negative control tests with sterile blood, the results showed that all video detection models reported zero bacteria detections at time 70 min for all tests (<em>n</em> = 3) (SI Fig. <a href="#MOESM1" class="usa-link">17</a>, see <a href="#MOESM1" class="usa-link">SI</a> section “Negative control tests”). The inference latency for each model is outlined in SI section “Inference times and computational complexity”. Some cases of obviously incorrect manual labeling of the test data were corrected. For details, see SI section “Data cleaning and label adjustments”.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Deep learning model performance.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375708_41746_2025_1948_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c7c1/12375708/763faf80129d/41746_2025_1948_Fig4_HTML.jpg" loading="lazy" id="d33e1004" height="715" width="691" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Evaluating networks using downsampled data, testing on an increasing number of frames (each corresponding to 10 min). The heatmap shows the F1-Score of DinoV2. <strong>b</strong> Precision, recall, AUC (Area under the curve), and F1-Score of all models over time at full resolution, corresponding to the red dashed rectangle in part <strong>a</strong> Lines show mean, and shaded areas show standard deviations among the 30 retrainings. <strong>c</strong> Confusion matrix, performance metrics, and ROC (Receiver Operating Characteristics) curve of the DinoV2 model with the median AUC score among the 30 retrainings, at full resolution, evaluated at the final time (70 min). The figure was created using the open-source software Inkscape and the Python plotting library Matplotlib.</p></figcaption></figure><p id="Par27">We also analyzed the image series by performing inference on each frame separately, disregarding any temporal or sequential dependencies that might exist between consecutive frames. This approach, which is outlined in SI section “Posing the problem as single-frame image classification", is feasible and performs well, as shown in SI Fig. <a href="#MOESM1" class="usa-link">15</a>. However, in the negative control experiments, false positives occurred throughout the whole time-series, as shown in SI Fig. <a href="#MOESM1" class="usa-link">18</a>.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par28">We developed and tested a five-step assay to isolate and detect bacteria in blood samples without pre-culture. We show that we can successfully use the assay to trap and detect three sepsis-causing bacterial species, <em>K. pneumoniae</em>, <em>E. faecalis</em>, and <em>E. coli</em>, which cover ~20% of the bacterial species causing BSIs in the European Union<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> and 35% of those in Japan<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. The total assay time is below 2 h, which includes 5 min for the first centrifugation, 10 min for lysing, 13 min for the second centrifugation and 70 min for the microfluidic trapping. Automatic image analysis, based on deep-learning, occurs in less than 30 s for the whole data set. The current hands-on time is ~10 min. The estimated detection limit in the range 1–10 CFU/ml is clinically relevant. Compared to previous work, our approach does not require blood culture and allows the detection of bacteria phenotypically at clinically relevant concentrations. For <em>S. aureus</em>, the poor isolation efficiency lead to detection failure.</p>
<p id="Par29">Smart centrifugation is a simple, inexpensive, high-throughput, and easily parallelizable method to rapidly isolate bacteria from whole blood with a high bacterial isolation efficiency and blood cell rejection. The method’s compatibility with standard lab equipment supports translation to the clinical setting. The bacteria are isolated in a small volume of liquid, similar to the original plasma volume, which facilitates downstream processing.</p>
<p id="Par30">Two distinct mechanisms contribute to the enrichment of bacteria in the supernatant during centrifugation. First, the Stokes terminal velocity of bacteria is a factor <em>α</em> ≃ 1/30 times that of blood cells, allowing efficient terminal velocity-based differentiation.<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup> Second, the high hematocrit results in downward sedimenting RBCs causing a backflow of the plasma, thereby dragging bacteria upward and further increasing differentiation between the two. Smart centrifugation enhances the separation by extending the sedimentation path of particles through the density medium. For comparison, we experimentally optimised isolation of <em>E. coli</em> from spiked whole blood in conventional centrifugation, i.e., without sample dilution or density medium, and achieved 34% ± 7% isolation efficiency in the supernatant (SI Fig. <a href="#MOESM1" class="usa-link">2</a>). Smart centrifugation thus provides a 65%/34% = 1.9 times improved isolation efficiency for <em>E. coli</em> for a comparable blood cell rejection efficiency (SI Table <a href="#MOESM1" class="usa-link">3</a>).</p>
<p id="Par31">Even though our method was optimised for <em>E. coli</em>, isolation efficiency of <em>K. pneumoniae</em> was significantly higher. Isolation efficiency of <em>E. coli</em> and <em>E. faecalis</em> were not significantly different, but isolation efficiency of <em>S. aureus</em>, however, was much lower. We hypothesize that several factors can cause the recovery differences between the species, as well as the large variation in recovery between different measurements with the same strain. These factors include differences in Stokes radius between the bacterial species<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, differences in interaction between the blood cells and the bacteria, and differences in viscosity. We ascribe the variability in the isolation efficiency data to biological variability in patients and bacteria, and to technical factors. The blood samples used in the study were obtained from different donors, introducing biological variability, particularly in hematocrit levels, which can range from 40 to 54% in males and 36 to 48% in females<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. Since smart centrifugation relies on density-driven sedimentation, such hematocrit differences can directly affect the separation efficiency and bacterial recovery. Another important source of variability arises from the use of low bacterial concentrations, especially close to the detection limit. At these low CFU levels, differences in individual bacteria behavior are amplified. Furthermore, plate counting becomes more susceptible to statistical fluctuations, further increasing measurement uncertainty and contributing to variability in recovery<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Additional sources of variability may include variations in the immune response of the patient blood to the bacteria (reducing the bacterial count), or variations in bacterial growth phase (compared to <em>K. pneumoniae</em>, <em>E. coli</em> were spiked in stationary phase) or growth rate (<em>E. faecalis</em> have a slower growth rate than <em>K. pneumoniae</em>). We found no significant differences in bacterial isolation efficiencies between samples spiked with the same strain at different concentrations (SI Fig. <a href="#MOESM1" class="usa-link">1</a>), with the exception of <em>E. coli</em>, where we found a significantly lower isolation efficiency for concentrations from 21 to 100 CFU/ml compared to &gt;100 CFU/ml (<em>p</em> ≤ 0.05). However, the statistical power of this finding is low considering the very low number of measurements.</p>
<p id="Par32">The behavior of <em>S. aureus</em> in blood has been extensively studied previously<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. <em>S. aureus</em> secretes two coagulases, Coagulase (Coa) and von Willebrand factor binding protein (vWbp), which hijack the host coagulation cascade and trigger the formation of fibrin clots and/or blood clots, in which the bacteria hide from the effects of the immune system and thereby increase their survival<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>–<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. We speculate that the poor isolation efficiency of <em>S. aureus</em> in our approach is caused by this phenomenon, in which clots containing bacteria rapidly sediment together with the blood cells. Indeed, experiments in which <em>S. aureus</em> were spiked in a sample liquid with the same density as diluted blood but without blood cells resulted in 54 ± 6% (<em>n</em> = 6) isolation efficiency by smart centrifugation, compared to 8 ± 7 % (<em>n</em> = 10) isolation efficiency from spiked blood (SI Fig. <a href="#MOESM1" class="usa-link">3</a>). To approach this challenge, some literature suggests adding the anti-coagulant and direct thrombin inhibitor Argatroban Monohydrate to the blood<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Although this drug has been clinically proven to improve outcomes in patients treated with it and that had previous positive <em>S. aureus</em> blood cultures<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>, we argue that its use post-sampling would not be successful, as clot formation occurs inside the patient, pre-sampling, and the drug cannot reverse clotting. Poor isolation of <em>S. aureus</em> remains an ongoing challenge and we suggest future research to focus on alternative fibrinolytic agents. Based on epidemiological data, we estimate that this limitation could lead to false negatives in ~10% of sepsis cases, where <em>S. aureus</em> or <em>MRSA</em> are the causative agents<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Other common sepsis-causing pathogens, including <em>E. coli</em>, <em>K. pneumoniae</em>, <em>E. faecalis</em>, <em>S. pneumoniae</em>, <em>S. pyogenes</em>, <em>P. aeruginosa</em>, Candida species, and other Enterococcus species, do not exhibit the same coagulase activity and are therefore not expected to pose similar isolation challenges.</p>
<p id="Par33">Based on our reading of the literature, <em>S</em><em>mart centrifugation</em> has superior recovery and throughput for <em>E. coli</em>, data for which are presented for most tests (SI Table <a href="#MOESM1" class="usa-link">3</a>).</p>
<p id="Par34">The concentrated bacterial sample was loaded onto the microfluidic chip at 1000 mbar. At higher pressures, we observed bacterial loss through the 300 nm constrictions of the microtraps. Sedimenting into a higher-density liquid during the prior volume reduction step reduced the compression of bacterial cells and lysate and allowed more homogeneous resuspension, leading to lower on-chip clogging.</p>
<p id="Par35">The trapping of bacterial cells and blood cell debris in the traps increases the fluidic resistance of the filter and decreases the through-filter flow over time. The chips enabled processing 0.5 ml sample within 70 to 90 min, during which 30% of the sample flows through the microtraps (filtrate flow) and the rest flows past the filter directly to the waste outlet (retentate flow). The low filtrate flow rate is thus a major contributor to bacterial loss between isolation and detection.</p>
<p id="Par36">One limitation of the current assay is the risk of false negatives at very low bacterial concentrations due to an overall detection yield of 5–44%. To mitigate this, two strategies could be employed: (1) increasing the processed blood volume from 2.25 mL to 10 mL to enhance bacterial capture probability, and (2) introducing a brief pre-incubation step (2–3 h) to allow bacterial enrichment before processing. As antibiotics are typically administered in 6-h intervals, such enrichment would still support timely clinical decision-making. A potential limitation of this would be the detection of organisms that do not exhibit growth, which would have to be investigated in the future.</p>
<p id="Par37">The deep learning experiments demonstrate the feasibility of using deep neural networks to accurately detect the presence of bacteria in phase-contrast time-lapses. Both transformer-based models and convolutional neural networks achieve similar performance. The task can also be framed as single-frame image classification with good results. However, in a real-world scenario, it would be more advantageous to use the entire time-lapse sequence to detect bacterial growth, especially if using low-resolution microscopy. This is also demonstrated by our negative control experiments, where single-frame models produce more false positives in later time steps (see SI Fig. <a href="#MOESM1" class="usa-link">17</a>) due to their inability to detect actual growth.</p>
<p id="Par38">This proof of concept demonstrates that it is possible to train a classifier from a few experiments with good performance. Future work could stratify classification metrics by species, although it remains unclear whether significant differences in classification difficulty exist between species. Inference times for a single time-lapse (corresponding to one micro-trap) ranged from 1 to 3 ms on our system, indicating that latency should not pose a significant issue, even on automated microscopy systems with less powerful hardware.</p>
<p id="Par39">In summary, our approach offers a rapid, culture-free approach for detecting bacteria directly from blood at clinically relevant concentrations within 2 h. The overall hands-on time is ~10 min and could be reduced through future robotic automation for further clinical implementation. The experimental workflow, encompassing smart centrifugation, selective lysis, and microfluidic trapping, is supported by automatic image analysis that yields results regarding bacterial presence in under a second. We suggest future work to focus on clinical validation of the assay, as well as incorporation of species identification<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>,<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>, and phenotypic-AST at the single-cell level<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>, for which the microfluidic platform is intrinsically suited. Finally, for seamless clinical incorporation, robotic automation will be required to further reduce operator time, making this assay a powerful candidate for routine, high-throughput sepsis diagnostics.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Microfluidic platform fabrication</h3>
<p id="Par40">The microfluidic chip design and fabrication were previously reported<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. A silicon wafer mold was fabricated by the company ConScience AB, Sweden. The mold was fabricated on a 6-inch (100) silicon wafer. A thermal oxide layer was first grown to a thickness matching the height of the nanochannels. An etch mask for the nanochannels was then patterned using electron beam lithography, and the nanochannels were etched into the oxide using reactive ion etching. The microchannel structures were subsequently patterned in photoresist by laser lithography, and the photoresist defining the microchannels was reflowed. The wafer was once silanized for 30 min prior to polydimethylsiloxane (PDMS; Sylgard 184, DOW, USA) replication. Subsequently, a 10:1 w/w PDMS:curing agent mixture was poured on the wafer and baked at 80 °C overnight. Holes on the PDMS ports 2.0, 2.1, 2.2, 5.1, and 5.2 (see chip design in refs. <sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>) were made using a 0.5 mm puncher. Then, the PDMS stamps were cleaned in IPA before covalent bonding to a glass coverslip (No. 1.5, Menzel-Gläser, Germany) after plasma treatment, followed by heat curing for 1 h at 80 °C. Approximately 10% of the devices fabricated were discarded due to the presence of dust in the channels of the PDMS stamp, derived from the microfabrication process.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Microfluidic flow control</h3>
<p id="Par41">The microfluidic device was mounted on the microscope and connected to the sample reservoirs with flexible plastic tubing (TYGON, Saint-Gobain, North America). OB1 CONTROLLER (Elveflow, France) flow control units were used to pressurize reservoirs.</p>
<p id="Par42">The reservoirs connected to ports 2.1, 2.2, 5.1 and 5.2 were first pressurized at 500 mbar for chip priming with water infused with 0.085 g/l of Pluronic F108 (Sigma-Aldrich, USA) and was decreased to 0 mbar. Then the reservoir connected to inlet port 2.0 was pressurized at 500 mbar for priming and at 1000 mbar for sample loading.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Blood sample preparation</h3>
<p id="Par43">Blood in EDTA tubes from healthy donors was purchased from the blood bank (Blodcentralen, Stockholm, Sweden) or Uppsala University Hospital (Akademiska Sjukhuset, Uppsala, Sweden). The blood samples were stored at 4 °C and used for experiments no later than two days after collection. Samples presenting a milky supernatant (&lt;10%), i.e., indicating abnormally high concentrations of triglycerides in the blood, were discarded. Before processing the samples, they were put at room temperature, and blood samples were spiked with respective bacteria concentrations of approximately 10<sup>3</sup>, 10<sup>2</sup>, and 10 CFU/ml. The concentration of the spiking solution was quantified by plate counting (<em>n</em> = 3). The spiking volume was always less than 5% of the total blood solution volume.</p>
<p id="Par44">We used anonymized blood obtained from healthy donors through a certified blood bank, solely for technical development purposes, and therefore, no specific ethical approval was required under Swedish regulations.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Bacterial strains</h3>
<p id="Par45">We used three clinical isolates that were randomly collected from a clinical microbiology laboratory in Sweden, covering both gram-negative and gram-positive species. As gram-positive representatives, <em>S. aureus</em> (DA70300) and <em>E. feacalis</em> (DA70208) were used, and as gram-negative, <em>K. pneumoniae</em> (DA72206). We also used <em>E. coli</em> (EL3041) cells harboring a plasmid expressing mVenusNB fluorescence proteins. For the machine learning detection experiments, a non-fluorescent strain of <em>E. coli</em> (EL330) was used.</p>
<p id="Par46">Bacteria were stored for long term at –80 °C in standard glycerol solution and were incubated overnight at 37 °C in BD BACTEC Plus Aerobic medium (BD, USA), referred to as Blood Culture Medium (BCM), prior to use, and later diluted to approximately 10<sup>4</sup>, 10<sup>3</sup>, and 10<sup>2</sup> CFU/ml and used to spike blood with different concentrations and quantified for isolation. For detection experiments with <em>E. coli</em> cells harboring a plasmid expressing mVenusNB fluorescence proteins, the above-mentioned protocol was followed. For detection experiments with clinical isolates and non-fluorescent <em>E. coli</em>, 2 μL of bacteria from an overnight culture were spiked in 2 ml of BD BACTEC Plus Aerobic Medium, and cultured for 2–4 h depending on the species. They were then diluted to approximately 10<sup>4</sup>, 10<sup>3</sup>, and 10<sup>2</sup> CFU/ml and used to spike blood with different concentrations.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Smart centrifugation</h3>
<p id="Par47">Lymphophrep (STEMCELL Technologies, Canada), a density medium with 1.077 g/ml density, was mixed with BD BACTEC Plus Aerobic medium (BD, USA) in a volumetric ratio of 2:1, and 1 ml of the solution, with 1.051 g/ml density, was poured into a 15 ml Falcon centrifuge tube. 3 ml of spiked blood mixed with BD BACTEC Plus Aerobic medium (BD, USA) (3:1 Blood:BD BACTEC Plus Aerobic medium) was gently placed over the density media. The tube was centrifuged at 600 × <em>g</em> for 5 min in a hanging bucket centrifuge. The supernatant was removed and mixed with the lysing solution.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Conventional centrifugation for bacteria isolation</h3>
<p id="Par48">To establish a control measurement, we centrifuged 4 ml of whole blood without any dilution and density media. After being spiked, the sample underwent centrifugation at 500 × <em>g</em> for 4 min, resulting in the separation of clean plasma at the top. This process effectively settled most of the blood cells.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Selective lysing solution</h3>
<p id="Par49">The selective lysing solution was prepared by mixing sodium chocolate hydrate (Sigma-Aldrich, USA) and saponin (Sigma-Aldrich, USA). Concentrations of 2% (W/V) sodium cholate hydrate and 1% (W/V) saponin were prepared by dissolving the chemicals in BD BACTEC Standard Aerobic medium.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Volume reduction</h3>
<p id="Par50">Percoll (Sigma-Aldrich, USA), a density medium having a density of 1.125–1.135 g/ml was mixed with BD BACTEC Plus Aerobic medium (BD, USA) in a volumetric ratio of 1:2, and 0.3 ml of the solution, with ~1.04 g/ml density, was poured into a 15 ml Falcon centrifuge tube. The supernatant infused with the lysing solution was gently placed over the density medium. The tube was centrifuged at 1000 × <em>g</em> for 13 min in a fixed rotor centrifuge. All excess liquid above 0.5 ml was removed, without removing the pellet.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Optical setup</h3>
<p id="Par51">Images of all the microtraps contained in the microfluidic device were acquired using a Nikon Ti, inverted microscope. For imaging experiments containing clinical strains, a CFI plan Apo lambda 100x (1.45 NA, oil) objective was used. Phase contrast images were acquired by using DMK 38UX304 (the imaging source) camera, with an exposure time of 20 ms with an interval of 10 min, for a maximum of 90 min. Cells expressing mVenus fluorescence proteins were captured using a filter cube consisting of a Di02-R488 (Semrock, USA) dichroic mirror, a FF02-482/18 (Semrock, USA) excitation filter, and a FF01-524/24 (Semrock, USA) emission filter and CFI Plan Apo lambda DM Ph2 20x objective. Each fluorescence image was acquired with an exposure time of 500 ms with an interval of 5 min, for a maximum of 90 min. A constant temperature of 37 °C was maintained during the measurements using a temperature-controllable unit (Okolab, Italy). The imaging setup was operated by micromanager 1.4 version software.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Bacteria quantitation</h3>
<p id="Par52">All bacterial sample quantitation was performed by plate counting after plating a sample aliquot on agar plates and overnight culture at 37 °C in the incubator. The agar plates were prepared by dissolving LB broth with agar (Miller) (Sigma-Aldrich, USA) in deionized water at 40 g/L concentration followed by autoclaving and placing in Petri dishes.</p>
<p id="Par53">For on-chip bacteria quantitation of the clinical isolates, we employed ImageJ software image analysis of phase-contrast microscopy images. Each microtrap was analyzed manually and determined positive if filled with at least one bacterium. The same analysis pipeline was performed for fluorescence microscopy images of the <em>E. coli</em> cells expressing mVenus fluorescence proteins.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Blood cell quantitation</h3>
<p id="Par54">The concentrations of the blood cells in whole blood before dilution and in the supernatant after smart centrifugation were measured using a hematology analyzer (Swelab Alfa Plus, Boule Diagnostics, Sweden).</p></section><section id="Sec23"><h3 class="pmc_sec_title">Flow characterization</h3>
<p id="Par55">The ports 2.1 and 2.2 were coupled together, and ports 5.1 and 5.2 were coupled together, using a PEEK union T-connector (Thermo Fisher Scientific, USA). Flow measurements were taken from both couplers using a SLI Liquid Flow Sensor (80 μL/min) (Sensirion, Switzerland).</p>
<p id="Par56">Two independent flow measurements were acquired with a sampling time of 1 s, for a maximum time of 90 min, using the Sensor Viewer Software (Sensirion, Switzerland). The data was analysed using MATLAB R2021.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Image cropping and stabilizing</h3>
<p id="Par57">An image processing pipeline was further improved from ref. <sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> to rotate, find, and crop out 1300 × 40 pixel trap images in the microfluidic chip, and then stabilizing the corresponding time-lapse described in SI section “Preprocessing and cropping”. During deep learning model training and inference, the frames were cropped vertically at 336 pixels from the physical stop at the top of each trap and padded to a width of 42 pixels. The reason for padding the traps to 42 pixels width is that we hypnotize DinoV2 would perform better, as it uses patches of 14 × 14 pixels for the token embeddings, meaning it would fit three tokens on the width of a trap. The rationale for the 336-pixel cropping of the height was to fit the time-lapses in the GPU memory and concerns of overfitting. Additionally, the difficult instances had one or a few cells situated at the top adjacent to the physical stop, thus, processing the whole trap was unnecessary.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Time-lapse images and downsampling</h3>
<p id="Par58">When employing image classification models (ResNet 18, DinoV2, and EfficientNet B2), the frames of each trap were concatenated horizontally to a 336 × 336 pixel 2D image showing the time-lapse. Zero-padding was applied on the right side when performing the time evaluation, shown in SI Fig. <a href="#MOESM1" class="usa-link">11</a>. Padding was also applied during training if the time-lapses from the experiment contained less than eight frames, seen in SI Table <a href="#MOESM1" class="usa-link">4</a>. For the Video ResNet R(2+1)D, we instead relied on the global averaging pooling<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup> to test and train on time-lapses with different numbers of frames. A time-lapse sample had a positive label if it contained any positive frames to the evaluated time-point (also if cells had been previously trapped and subsequently escaped). Lancoz<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup> downsampling was used to test under degraded image conditions and test how much high-frequency information can be removed without degrading performance. Before feeding the downsampled images into the model, they were upsampled to their original size using bicubic interpolation. This step was necessary because DinoV2 is a transformer with a fixed patch size, thus, it could not process images with smaller widths than 14 pixels. The single-channel grayscale phase-contrast data was transformed to RGB using the Vidiris color map<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup> for visibility and to aid human labeling. The transformed color images were also used when feeding into the networks (we kept the 3-channel input).</p></section><section id="Sec26"><h3 class="pmc_sec_title">Network training</h3>
<p id="Par59">All models were trained for 20 epochs with batch size 32 and learning rate 10<sup>−3</sup> except DinoV2 (batch 128, learning rate 10<sup>−5</sup>, trained for 15 epochs) and Video ResNet R(2+1)D (used learning rate 10<sup>−4</sup>). All networks used pretrained weights, employed the standard ADAM optimizer<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>, Cross Entropy loss, Cosine learning rate scheduler<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>, and 15% label smoothing<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>. The Pytorch deep learning framework was used with Pytorch Image Models (Timm)<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>. During training, images were sampled with replacement for each mini-batch from the two classes, with sampling probabilities inversely proportional to the class overrepresentation in the training set (using a Random Weighted Sampler in Pytorch). The augmentations ShiftScaleRotate, HorizontalFlip, VerticalFlip, RandomBrightnessContrast, and Blur were employed during training from the Albumetations library<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. Additionally, 1-7 frames (sampled uniformly) were randomly erased during training (random frame erase augmentation). All models were trained on the NvidiaA100 GPU.</p>
<p id="Par60">For further information, refer to the released software package containing the dataset and software to reproduce the deep learning experiments.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par61">Data analyses were performed with R and Microsoft Excel. The data samples were tested for normality using the Shapiro-Wilk test; variance between normal populations was studied using the F-test; and statistically significant differences between groups were studied using a <em>t</em>-test, for samples following a normal distribution, or a Mann–Whitney test for samples not following a normal distribution. A two-sided <em>p</em>-value of ≤0.05 was considered statistically significant.</p></section></section><section id="Sec28"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12375708/bin/41746_2025_1948_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (25.3MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We gratefully acknowledge financial support from the Swedish Foundation for Strategic Reasearch: Carolina Wählby, Johan Elf, Wouter van der Wijngaart ARC19-0016; and Swedish Research Council: Carolina Wählby 2022-06725. The computations were enabled by the Alvis cluster provided by the National Academic Infrastructure for Super computing in Sweden (NAISS) at Chalmers University of Technology, partially funded by the Swedish Research Council through grant agreement no. 2022-06725. Additionally, the computations were enabled by the Berzelius supercomputing resource provided by the National Supercomputer Centre at Linköping University and supported by the Knut and Alice Wallenberg Foundation. We would like to thank Dr. Anders Olof Larsson for the support during this project.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>W.W. and J.E. designed and directed the research project. M.O. and M.M. conducted and designed the experiments. M.O. performed the centrifugation experiments. M.M. fabricated the microfluidic devices and performed the centrifugation and microfluidic experiments. V.K. and J.L. provided the bacterial strains. K.K. built the annotation tool for the microscopy images, used for the training and testing of the ML algorithms. M.M., M.O., and K.K. annotated the ML data sets. E.H., M.O., and K.K. contributed to the Machine Learning methods. E.H. performed all computational ML operations. C.W. advised on the ML operations. M.M., M.O., E.H., and W.W. analyzed the data. M.M., M.O., E.H., V.K., J.E., and W.W. wrote the manuscript. All authors discussed the results and commented on the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open access funding provided by Royal Institute of Technology.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The raw data (microscopy image data, flow measurements, plate count data, analysis summary files, codes, etc.) used to generate the information compiled in this manuscript can be found in the following open repository: <a href="https://figshare.com/s/affea349261af5a2f637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/s/affea349261af5a2f637</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>All codes and replication packages for the ML models can be found in the following open repository: <a href="https://figshare.com/s/affea349261af5a2f637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/s/affea349261af5a2f637</a>.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par62">J.E. and W.W. have commercial interests in the diagnostic field. M.M., M.O., J.E., and W.W. have filed a patent application based on the method presented in this paper.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: M. Henar Marino Miguélez, Mohammad Osaid.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Johan Elf, Email: johan.elf@icm.uu.se.</p>
<p>Wouter van der Wijngaart, Email: wouter@kth.se.</p></section><section id="sec29"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41746-025-01948-w.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Singer, M. et al. The third international consensus definitions for sepsis and septic shock (sepsis-3). <em>J. Am. Med. Assoc.</em><strong>315</strong>, 801–810 (2016).</cite> [<a href="https://doi.org/10.1001/jama.2016.0287" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4968574/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26903338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singer,%20M.%20et%20al.%20The%20third%20international%20consensus%20definitions%20for%20sepsis%20and%20septic%20shock%20(sepsis-3).%20J.%20Am.%20Med.%20Assoc.315,%20801%E2%80%93810%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Rudd, K. E. et al. Global, regional, and national sepsis incidence and mortality, 1990-2017: analysis for the Global Burden of Disease Study. <em>Lancet</em><strong>395</strong>, 200–211 (2020).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(19)32989-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6970225/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31954465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rudd,%20K.%20E.%20et%20al.%20Global,%20regional,%20and%20national%20sepsis%20incidence%20and%20mortality,%201990-2017:%20analysis%20for%20the%20Global%20Burden%20of%20Disease%20Study.%20Lancet395,%20200%E2%80%93211%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Thompson, K., Venkatesh, B. &amp; Finfer, S. Sepsis and septic shock: current approaches to management. <em>Intern. Med. J.</em><strong>49</strong>, 160–170 (2019).
</cite> [<a href="https://doi.org/10.1111/imj.14199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30754087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thompson,%20K.,%20Venkatesh,%20B.%20&amp;%20Finfer,%20S.%20Sepsis%20and%20septic%20shock:%20current%20approaches%20to%20management.%20Intern.%20Med.%20J.49,%20160%E2%80%93170%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Kumar, A. et al. Duration of hypotension before initiation of effective antimicrobial therapy is the critical determinant of survival in human septic shock. <em>Crit. Care Med.</em><strong>34</strong>, 1589–1596 (2006).
</cite> [<a href="https://doi.org/10.1097/01.CCM.0000217961.75225.E9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16625125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20A.%20et%20al.%20Duration%20of%20hypotension%20before%20initiation%20of%20effective%20antimicrobial%20therapy%20is%20the%20critical%20determinant%20of%20survival%20in%20human%20septic%20shock.%20Crit.%20Care%20Med.34,%201589%E2%80%931596%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Huerta, L. E. &amp; Rice, T. W. Pathologic difference between sepsis and bloodstream infections. <em>J. Appl. Lab. Med.</em><strong>3</strong>, 654–663 (2019).
</cite> [<a href="https://doi.org/10.1373/jalm.2018.026245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31639733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huerta,%20L.%20E.%20&amp;%20Rice,%20T.%20W.%20Pathologic%20difference%20between%20sepsis%20and%20bloodstream%20infections.%20J.%20Appl.%20Lab.%20Med.3,%20654%E2%80%93663%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Bates, D. W. et al. Predicting bacteremia in patients with sepsis syndrome. <em>J. Infect. Dis.</em><strong>176</strong>, 1538–1551 (1997).
</cite> [<a href="https://doi.org/10.1086/514153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9395366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bates,%20D.%20W.%20et%20al.%20Predicting%20bacteremia%20in%20patients%20with%20sepsis%20syndrome.%20J.%20Infect.%20Dis.176,%201538%E2%80%931551%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Yagupsky, P. &amp; Nolte, F. S. Quantitative aspects of septicemia. <em>Clin. Microbiol. Rev.</em><strong>3</strong>, 269–279 (1990).
</cite> [<a href="https://doi.org/10.1128/cmr.3.3.269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC358159/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2200606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yagupsky,%20P.%20&amp;%20Nolte,%20F.%20S.%20Quantitative%20aspects%20of%20septicemia.%20Clin.%20Microbiol.%20Rev.3,%20269%E2%80%93279%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Cohen, J. et al. Sepsis: a roadmap for future research. <em>Lancet Infect. Dis.</em><strong>15</strong>, 581–614 (2015).
</cite> [<a href="https://doi.org/10.1016/S1473-3099(15)70112-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25932591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cohen,%20J.%20et%20al.%20Sepsis:%20a%20roadmap%20for%20future%20research.%20Lancet%20Infect.%20Dis.15,%20581%E2%80%93614%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Morgenthaler, N. G. &amp; Kostrzewa, M. Rapid identification of pathogens in positive blood culture of patients with sepsis: Review and meta-analysis of the performance of the Sepsityper kit. <em>Int. J. Microbiol.</em><strong>2015</strong>, 827416 (2015).
</cite> [<a href="https://doi.org/10.1155/2015/827416" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4426779/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26000017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morgenthaler,%20N.%20G.%20&amp;%20Kostrzewa,%20M.%20Rapid%20identification%20of%20pathogens%20in%20positive%20blood%20culture%20of%20patients%20with%20sepsis:%20Review%20and%20meta-analysis%20of%20the%20performance%20of%20the%20Sepsityper%20kit.%20Int.%20J.%20Microbiol.2015,%20827416%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Lambregts, M. M., Bernards, A. T., van der Beek, M. T., Visser, L. G. &amp; de Boer, M. G. Time to positivity of blood cultures supports early re-evaluation of empiric broad-spectrum antimicrobial therapy. <em>PLoS ONE</em><strong>14</strong>, 1–12 (2019).</cite> [<a href="https://doi.org/10.1371/journal.pone.0208819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6314566/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30601829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lambregts,%20M.%20M.,%20Bernards,%20A.%20T.,%20van%20der%20Beek,%20M.%20T.,%20Visser,%20L.%20G.%20&amp;%20de%20Boer,%20M.%20G.%20Time%20to%20positivity%20of%20blood%20cultures%20supports%20early%20re-evaluation%20of%20empiric%20broad-spectrum%20antimicrobial%20therapy.%20PLoS%20ONE14,%201%E2%80%9312%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>McGregor, C. Improving time to antibiotics and implementing the “Sepsis 6”. <em>BMJ Open Quality</em><strong>2</strong><a href="https://bmjopenquality.bmj.com/content/2/2/u202548.w1443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bmjopenquality.bmj.com/content/2/2/u202548.w1443</a> (2014).</cite> [<a href="https://doi.org/10.1136/bmjquality.u202548.w1443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4663828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26734240/" class="usa-link">PubMed</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Bochud, P. Y., Glauser, M. P., Calandra, T. &amp; International Sepsis Forum. Antibiotics in sepsis. <em>Intens. Care Med.</em><strong>27</strong>, S33–48 (2001).</cite> [<a href="https://doi.org/10.1007/pl00003796" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11307369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bochud,%20P.%20Y.,%20Glauser,%20M.%20P.,%20Calandra,%20T.%20&amp;%20International%20Sepsis%20Forum.%20Antibiotics%20in%20sepsis.%20Intens.%20Care%20Med.27,%20S33%E2%80%9348%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Rhee, C. et al. Prevalence, underlying causes, and preventability of sepsis-associated mortality in US acute care hospitals. <em>JAMA Netw. Open</em><strong>2</strong>, 1–14 (2019).</cite> [<a href="https://doi.org/10.1001/jamanetworkopen.2018.7571" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6484603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30768188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rhee,%20C.%20et%20al.%20Prevalence,%20underlying%20causes,%20and%20preventability%20of%20sepsis-associated%20mortality%20in%20US%20acute%20care%20hospitals.%20JAMA%20Netw.%20Open2,%201%E2%80%9314%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Llor, C. &amp; Bjerrum, L. Antimicrobial resistance: Risk associated with antibiotic overuse and initiatives to reduce the problem. <em>Ther. Adv. Drug Saf.</em><strong>5</strong>, 229–241 (2014).
</cite> [<a href="https://doi.org/10.1177/2042098614554919" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4232501/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25436105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Llor,%20C.%20&amp;%20Bjerrum,%20L.%20Antimicrobial%20resistance:%20Risk%20associated%20with%20antibiotic%20overuse%20and%20initiatives%20to%20reduce%20the%20problem.%20Ther.%20Adv.%20Drug%20Saf.5,%20229%E2%80%93241%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Korvick, J. A., Peacock, J. E., Muder, R. R., Wheeler, R. R. &amp; Yu, V. L. Addition of rifampin to combination antibiotic therapy for Pseudomonas aeruginosa bacteremia: prospective trial using the Zelen protocol. <em>Antimicrob. Agents Chemother.</em><strong>36</strong>, 620–625 (1992).
</cite> [<a href="https://doi.org/10.1128/aac.36.3.620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC190567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1622173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Korvick,%20J.%20A.,%20Peacock,%20J.%20E.,%20Muder,%20R.%20R.,%20Wheeler,%20R.%20R.%20&amp;%20Yu,%20V.%20L.%20Addition%20of%20rifampin%20to%20combination%20antibiotic%20therapy%20for%20Pseudomonas%20aeruginosa%20bacteremia:%20prospective%20trial%20using%20the%20Zelen%20protocol.%20Antimicrob.%20Agents%20Chemother.36,%20620%E2%80%93625%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Ombelet, S. et al. Best practices of blood cultures in low- and middle-income countries. <em>Front. Med.</em><strong>6</strong>, 131 (2019).</cite> [<a href="https://doi.org/10.3389/fmed.2019.00131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6591475/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31275940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ombelet,%20S.%20et%20al.%20Best%20practices%20of%20blood%20cultures%20in%20low-%20and%20middle-income%20countries.%20Front.%20Med.6,%20131%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Mazzulli, T. et al. Impact of implementation of bact/alert virtuo on blood culture time to positivity in sepsis patients. <em>Microbiol. Spectr.</em><strong>11</strong>, e05003–22 (2023).
</cite> [<a href="https://doi.org/10.1128/spectrum.05003-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10101113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36749050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mazzulli,%20T.%20et%20al.%20Impact%20of%20implementation%20of%20bact/alert%20virtuo%20on%20blood%20culture%20time%20to%20positivity%20in%20sepsis%20patients.%20Microbiol.%20Spectr.11,%20e05003%E2%80%9322%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Tjandra, K. C. et al. Diagnosis of bloodstream infections: an evolution of technologies towards accurate and rapid identification and antibiotic susceptibility testing. <em>Antibiotics</em><strong>11</strong>, 511 (2022).
</cite> [<a href="https://doi.org/10.3390/antibiotics11040511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9029869/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35453262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tjandra,%20K.%20C.%20et%20al.%20Diagnosis%20of%20bloodstream%20infections:%20an%20evolution%20of%20technologies%20towards%20accurate%20and%20rapid%20identification%20and%20antibiotic%20susceptibility%20testing.%20Antibiotics11,%20511%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Khan, Z. A., Siddiqui, M. F. &amp; Park, S. Current and emerging methods of antibiotic susceptibility testing. <em>Diagnostics</em><strong>9</strong>, 49 (2019).
</cite> [<a href="https://doi.org/10.3390/diagnostics9020049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6627445/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31058811/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khan,%20Z.%20A.,%20Siddiqui,%20M.%20F.%20&amp;%20Park,%20S.%20Current%20and%20emerging%20methods%20of%20antibiotic%20susceptibility%20testing.%20Diagnostics9,%2049%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Jorgensen, J. H. &amp; Ferraro, M. J. Antimicrobial susceptibility testing: a review of general principles and contemporary practices. <em>Clin. Infect. Dis.</em><strong>49</strong>, 1749–1755 (2009).
</cite> [<a href="https://doi.org/10.1086/647952" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19857164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jorgensen,%20J.%20H.%20&amp;%20Ferraro,%20M.%20J.%20Antimicrobial%20susceptibility%20testing:%20a%20review%20of%20general%20principles%20and%20contemporary%20practices.%20Clin.%20Infect.%20Dis.49,%201749%E2%80%931755%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Ekwall-Larson, A., Fröding,I., Mert, B., Åkerlund, A. &amp; Özenci, V. Analytical performance and potential clinical utility of EUCAST rapid antimicrobial susceptibility testing in blood cultures after four hours of incubation. <em>Microbiol. Spectrum</em><strong>11</strong>, e0500122 (2023).</cite> [<a href="https://doi.org/10.1128/spectrum.05001-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10100889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36809027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ekwall-Larson,%20A.,%20Fr%C3%B6ding,I.,%20Mert,%20B.,%20%E2%84%ABkerlund,%20A.%20&amp;%20%C3%96zenci,%20V.%20Analytical%20performance%20and%20potential%20clinical%20utility%20of%20EUCAST%20rapid%20antimicrobial%20susceptibility%20testing%20in%20blood%20cultures%20after%20four%20hours%20of%20incubation.%20Microbiol.%20Spectrum11,%20e0500122%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Banerjee, R. &amp; Humphries, R. Rapid antimicrobial susceptibility testing methods for blood cultures and their clinical impact. <em>Front. Med.</em><strong>8</strong>, 635831 (2021).</cite> [<a href="https://doi.org/10.3389/fmed.2021.635831" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7987685/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33777978/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banerjee,%20R.%20&amp;%20Humphries,%20R.%20Rapid%20antimicrobial%20susceptibility%20testing%20methods%20for%20blood%20cultures%20and%20their%20clinical%20impact.%20Front.%20Med.8,%20635831%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Jonasson, E., Matuschek, E. &amp; Kahlmeter, G. The EUCAST rapid disc diffusion method for antimicrobial susceptibility testing directly from positive blood culture bottles. <em>J. Antimicrob. Chemother.</em><strong>75</strong>, 968–978 (2020).
</cite> [<a href="https://doi.org/10.1093/jac/dkz548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7069491/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32016342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jonasson,%20E.,%20Matuschek,%20E.%20&amp;%20Kahlmeter,%20G.%20The%20EUCAST%20rapid%20disc%20diffusion%20method%20for%20antimicrobial%20susceptibility%20testing%20directly%20from%20positive%20blood%20culture%20bottles.%20J.%20Antimicrob.%20Chemother.75,%20968%E2%80%93978%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Baltekin, Ö. et al. Evaluation of an ultra-rapid antibiotic susceptibility testing method on positive blood cultures with Escherichia coli. Preprint at <a href="https://www.medrxiv.org/content/10.1101/2021.12.14.21267046v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.medrxiv.org/content/10.1101/2021.12.14.21267046v1</a> (2021).</cite>
</li>
<li id="CR25">
<span class="label">25.</span><cite>Wu, W. et al. Direct single-cell antimicrobial susceptibility testing of Escherichia coli in urine using a ready-to-use 3d microwell array chip. <em>Lab Chip</em><strong>23</strong>, 2399–2410 (2023).
</cite> [<a href="https://doi.org/10.1039/d2lc01095j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36806255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20W.%20et%20al.%20Direct%20single-cell%20antimicrobial%20susceptibility%20testing%20of%20Escherichia%20coli%20in%20urine%20using%20a%20ready-to-use%203d%20microwell%20array%20chip.%20Lab%20Chip23,%202399%E2%80%932410%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Rojas-Andrade, M. D. et al. Rapid antibiotic susceptibility determination by fluorescence lifetime tracking of bacterial metabolism. <em>ACS Infect. Dis.</em><strong>10</strong>, 4057–4065 (2024).
</cite> [<a href="https://doi.org/10.1021/acsinfecdis.4c00491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39572010/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rojas-Andrade,%20M.%20D.%20et%20al.%20Rapid%20antibiotic%20susceptibility%20determination%20by%20fluorescence%20lifetime%20tracking%20of%20bacterial%20metabolism.%20ACS%20Infect.%20Dis.10,%204057%E2%80%934065%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Wu, W. et al. Enzymatic antimicrobial susceptibility testing with bacteria identification in 30 min. <em>Anal. Chem.</em><strong>95</strong>, 16426–16432 (2023).
</cite> [<a href="https://doi.org/10.1021/acs.analchem.3c04316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37874622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20W.%20et%20al.%20Enzymatic%20antimicrobial%20susceptibility%20testing%20with%20bacteria%20identification%20in%2030%20min.%20Anal.%20Chem.95,%2016426%E2%80%9316432%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Wu, W. et al. All-in-one Escherichia coli viability assay for multi-dimensional detection of uncomplicated urinary tract infections. <em>Anal. Chem.</em><strong>94</strong>, 17853–17860 (2022).
</cite> [<a href="https://doi.org/10.1021/acs.analchem.2c03604" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36524619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20W.%20et%20al.%20All-in-one%20Escherichia%20coli%20viability%20assay%20for%20multi-dimensional%20detection%20of%20uncomplicated%20urinary%20tract%20infections.%20Anal.%20Chem.94,%2017853%E2%80%9317860%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Wu, W. et al. Inoculum size-insensitive susceptibility determination of urine sample based on in-situ measurement of inducible enzyme activity after 20 min of antibiotic exposure. <em>Anal. Chim. Acta</em><strong>1282</strong>, 341858 (2023).
</cite> [<a href="https://doi.org/10.1016/j.aca.2023.341858" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37923403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20W.%20et%20al.%20Inoculum%20size-insensitive%20susceptibility%20determination%20of%20urine%20sample%20based%20on%20in-situ%20measurement%20of%20inducible%20enzyme%20activity%20after%2020%20min%20of%20antibiotic%20exposure.%20Anal.%20Chim.%20Acta1282,%20341858%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Wong, A. Y. W., Johnsson, A. T. A. &amp; Özenci, V. Performance of dRAST on prospective clinical blood culture samples in a simulated clinical setting and on multidrug-resistant bacteria. <em>Microbiol. Spectrum</em><strong>10</strong>, e0210721 (2022).</cite> [<a href="https://doi.org/10.1128/spectrum.02107-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8941874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35234503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wong,%20A.%20Y.%20W.,%20Johnsson,%20A.%20T.%20A.%20&amp;%20%C3%96zenci,%20V.%20Performance%20of%20dRAST%20on%20prospective%20clinical%20blood%20culture%20samples%20in%20a%20simulated%20clinical%20setting%20and%20on%20multidrug-resistant%20bacteria.%20Microbiol.%20Spectrum10,%20e0210721%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Marschal, M. et al. Evaluation of the accelerate pheno system for fast identification and antimicrobial susceptibility testing from positive blood cultures in bloodstream infections caused by gram-negative pathogens. <em>J. Clin. Microbiol.</em><strong>55</strong>, 2116–2126 (2017).
</cite> [<a href="https://doi.org/10.1128/JCM.00181-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5483913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28446572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marschal,%20M.%20et%20al.%20Evaluation%20of%20the%20accelerate%20pheno%20system%20for%20fast%20identification%20and%20antimicrobial%20susceptibility%20testing%20from%20positive%20blood%20cultures%20in%20bloodstream%20infections%20caused%20by%20gram-negative%20pathogens.%20J.%20Clin.%20Microbiol.55,%202116%E2%80%932126%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Reszetnik, G. et al. Next-generation rapid phenotypic antimicrobial susceptibility testing. <em>Nat. Commun.</em><strong>15</strong>, 9719 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-53930-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11550857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39521792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reszetnik,%20G.%20et%20al.%20Next-generation%20rapid%20phenotypic%20antimicrobial%20susceptibility%20testing.%20Nat.%20Commun.15,%209719%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Timsit, J. F., Ruppé, E., Barbier, F., Tabah, A. &amp; Bassetti, M. Bloodstream infections in critically ill patients: an expert statement. <em>Intens. Care Med.</em><strong>46</strong>, 266–284 (2020).</cite> [<a href="https://doi.org/10.1007/s00134-020-05950-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7223992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32047941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Timsit,%20J.%20F.,%20Rupp%C3%A9,%20E.,%20Barbier,%20F.,%20Tabah,%20A.%20&amp;%20Bassetti,%20M.%20Bloodstream%20infections%20in%20critically%20ill%20patients:%20an%20expert%20statement.%20Intens.%20Care%20Med.46,%20266%E2%80%93284%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Marco, F. Molecular methods for septicemia diagnosis. <em>Enfermedades Infecciosas y. Microbiologia Clin.</em><strong>35</strong>, 586–592 (2017).</cite> [<a href="https://doi.org/10.1016/j.eimc.2017.03.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28427796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marco,%20F.%20Molecular%20methods%20for%20septicemia%20diagnosis.%20Enfermedades%20Infecciosas%20y.%20Microbiologia%20Clin.35,%20586%E2%80%93592%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Kirn, T. J. &amp; Weinstein, M. P. Update on blood cultures: How to obtain, process, report, and interpret. <em>Clin. Microbiol. Infect.</em><strong>19</strong>, 513–520 (2013).
</cite> [<a href="https://doi.org/10.1111/1469-0691.12180" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23490046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kirn,%20T.%20J.%20&amp;%20Weinstein,%20M.%20P.%20Update%20on%20blood%20cultures:%20How%20to%20obtain,%20process,%20report,%20and%20interpret.%20Clin.%20Microbiol.%20Infect.19,%20513%E2%80%93520%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Choi, J. et al. A rapid antimicrobial susceptibility test based on single-cell morphological analysis. <em>Sci. Transl. Med.</em><strong>6</strong>, 267ra174 (2014).
</cite> [<a href="https://doi.org/10.1126/scitranslmed.3009650" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25520395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choi,%20J.%20et%20al.%20A%20rapid%20antimicrobial%20susceptibility%20test%20based%20on%20single-cell%20morphological%20analysis.%20Sci.%20Transl.%20Med.6,%20267ra174%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Kinnunen, P. et al. Monitoring the growth and drug susceptibility of individual bacteria using asynchronous magnetic bead rotation sensors. <em>Biosens. Bioelectron.</em><strong>26</strong>, 2751–2755 (2011).
</cite> [<a href="https://doi.org/10.1016/j.bios.2010.10.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3059723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21095112/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kinnunen,%20P.%20et%20al.%20Monitoring%20the%20growth%20and%20drug%20susceptibility%20of%20individual%20bacteria%20using%20asynchronous%20magnetic%20bead%20rotation%20sensors.%20Biosens.%20Bioelectron.26,%202751%E2%80%932755%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Godin, M. et al. Using buoyant mass to measure the growth of single cells. <em>Nat. Methods</em><strong>7</strong>, 387–390 (2010).
</cite> [<a href="https://doi.org/10.1038/nmeth.1452" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2862099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20383132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Godin,%20M.%20et%20al.%20Using%20buoyant%20mass%20to%20measure%20the%20growth%20of%20single%20cells.%20Nat.%20Methods7,%20387%E2%80%93390%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Baltekin, Ö., Boucharin, A., Tano, E., Andersson, D. I. &amp; Elf, J. Antibiotic susceptibility testing in less than 30 min using direct single-cell imaging. <em>Proc. Natl Acad. Sci. USA</em><strong>114</strong>, 9170–9175 (2017).
</cite> [<a href="https://doi.org/10.1073/pnas.1708558114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5576829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28790187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baltekin,%20%C3%96.,%20Boucharin,%20A.,%20Tano,%20E.,%20Andersson,%20D.%20I.%20&amp;%20Elf,%20J.%20Antibiotic%20susceptibility%20testing%20in%20less%20than%2030%20min%20using%20direct%20single-cell%20imaging.%20Proc.%20Natl%20Acad.%20Sci.%20USA114,%209170%E2%80%939175%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Osaid, M. et al. A multiplexed nanoliter array-based microfluidic platform for quick, automatic antimicrobial susceptibility testing. <em>Lab Chip.</em><strong>21</strong>, 2223–2231 (2021).
</cite> [<a href="https://doi.org/10.1039/d1lc00216c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33890605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Osaid,%20M.%20et%20al.%20A%20multiplexed%20nanoliter%20array-based%20microfluidic%20platform%20for%20quick,%20automatic%20antimicrobial%20susceptibility%20testing.%20Lab%20Chip.21,%202223%E2%80%932231%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Mach, A. J. &amp; Di Carlo, D. Continuous scalable blood filtration device using inertial microfluidics. <em>Biotechnol. Bioeng.</em><strong>107</strong>, 302–311 (2010).
</cite> [<a href="https://doi.org/10.1002/bit.22833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20589838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mach,%20A.%20J.%20&amp;%20Di%20Carlo,%20D.%20Continuous%20scalable%20blood%20filtration%20device%20using%20inertial%20microfluidics.%20Biotechnol.%20Bioeng.107,%20302%E2%80%93311%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Hou, H. W., Bhattacharyya, R. P., Hung, D. T. &amp; Han, J. Direct detection and drug-resistance profiling of bacteremias using inertial microfluidics. <em>Lab Chip</em><strong>15</strong>, 2297–2307 (2015).
</cite> [<a href="https://doi.org/10.1039/c5lc00311c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4437799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25882432/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hou,%20H.%20W.,%20Bhattacharyya,%20R.%20P.,%20Hung,%20D.%20T.%20&amp;%20Han,%20J.%20Direct%20detection%20and%20drug-resistance%20profiling%20of%20bacteremias%20using%20inertial%20microfluidics.%20Lab%20Chip15,%202297%E2%80%932307%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Lu, X. et al. Sheathless and high-throughput elasto-inertial bacterial sorting for enhancing molecular diagnosis of bloodstream infection. <em>Lab Chip</em><strong>21</strong>, 2163–2177 (2021).
</cite> [<a href="https://doi.org/10.1039/d1lc00085c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33899072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20X.%20et%20al.%20Sheathless%20and%20high-throughput%20elasto-inertial%20bacterial%20sorting%20for%20enhancing%20molecular%20diagnosis%20of%20bloodstream%20infection.%20Lab%20Chip21,%202163%E2%80%932177%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Narayana Iyengar, S., Kumar, T., Mårtensson, G. &amp; Russom, A. High resolution and rapid separation of bacteria from blood using elasto-inertial microfluidics. <em>Electrophoresis</em><strong>42</strong>, 2538–2551 (2021).
</cite> [<a href="https://doi.org/10.1002/elps.202100140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34510466/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Narayana%20Iyengar,%20S.,%20Kumar,%20T.,%20M%C3%A5rtensson,%20G.%20&amp;%20Russom,%20A.%20High%20resolution%20and%20rapid%20separation%20of%20bacteria%20from%20blood%20using%20elasto-inertial%20microfluidics.%20Electrophoresis42,%202538%E2%80%932551%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Zeng, K., Osaid, M. &amp; van der Wijngaart, W. Efficient filter-in-centrifuge separation of low-concentration bacteria from blood. <em>Lab Chip.</em><strong>23</strong>, 4334–4342 (2023).
</cite> [<a href="https://doi.org/10.1039/d3lc00594a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37712252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng,%20K.,%20Osaid,%20M.%20&amp;%20van%20der%20Wijngaart,%20W.%20Efficient%20filter-in-centrifuge%20separation%20of%20low-concentration%20bacteria%20from%20blood.%20Lab%20Chip.23,%204334%E2%80%934342%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Marino Miguélez, M. H., Huguenin-Dumittan, A., Osaid, M. &amp; van der Wijngaart, W. Isolation and identification of bacteria from blood within 12 h using standard laboratory equipment. <em>Sci. Rep.</em><strong>15</strong>, 1–10 (2025).
</cite> [<a href="https://doi.org/10.1038/s41598-025-09024-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12241390/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40634447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marino%20Migu%C3%A9lez,%20M.%20H.,%20Huguenin-Dumittan,%20A.,%20Osaid,%20M.%20&amp;%20van%20der%20Wijngaart,%20W.%20Isolation%20and%20identification%20of%20bacteria%20from%20blood%20within%2012%20h%20using%20standard%20laboratory%20equipment.%20Sci.%20Rep.15,%201%E2%80%9310%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Pitt, W. G. et al. Rapid separation of bacteria from blood-review and outlook. <em>Biotechnol. Prog.</em><strong>32</strong>, 823–839 (2016).
</cite> [<a href="https://doi.org/10.1002/btpr.2299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5297886/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27160415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pitt,%20W.%20G.%20et%20al.%20Rapid%20separation%20of%20bacteria%20from%20blood-review%20and%20outlook.%20Biotechnol.%20Prog.32,%20823%E2%80%93839%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Forsyth, B. et al. A rapid single-cell antimicrobial susceptibility testing workflow for bloodstream infections. <em>Biosensors</em><strong>11</strong>, 288 (2021).
</cite> [<a href="https://doi.org/10.3390/bios11080288" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8391654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34436090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Forsyth,%20B.%20et%20al.%20A%20rapid%20single-cell%20antimicrobial%20susceptibility%20testing%20workflow%20for%20bloodstream%20infections.%20Biosensors11,%20288%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Fang, Y.-L. et al. An integrated microfluidic system for early detection of sepsis-inducing bacteria. <em>Lab Chip</em><strong>21</strong>, 113–121 (2021).
</cite> [<a href="https://doi.org/10.1039/d0lc00966k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33232424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fang,%20Y.-L.%20et%20al.%20An%20integrated%20microfluidic%20system%20for%20early%20detection%20of%20sepsis-inducing%20bacteria.%20Lab%20Chip21,%20113%E2%80%93121%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Kang, J. H. et al. An extracorporeal blood-cleansing device for sepsis therapy. <em>Nat. Med.</em><strong>20</strong>, 1211–1216 (2014).
</cite> [<a href="https://doi.org/10.1038/nm.3640" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25216635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kang,%20J.%20H.%20et%20al.%20An%20extracorporeal%20blood-cleansing%20device%20for%20sepsis%20therapy.%20Nat.%20Med.20,%201211%E2%80%931216%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Lee, J.-J. et al. Synthetic ligand-coated magnetic nanoparticles for microfluidic bacterial separation from blood. <em>Nano Lett.</em><strong>14</strong>, 1–5 (2014).
</cite> [<a href="https://doi.org/10.1021/nl3047305" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23367876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20J.-J.%20et%20al.%20Synthetic%20ligand-coated%20magnetic%20nanoparticles%20for%20microfluidic%20bacterial%20separation%20from%20blood.%20Nano%20Lett.14,%201%E2%80%935%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Kim, T. H. et al. Blood culture-free ultra-rapid antimicrobial susceptibility testing. <em>Nature</em><strong>632</strong>, 893–902 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-024-07725-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39048820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20T.%20H.%20et%20al.%20Blood%20culture-free%20ultra-rapid%20antimicrobial%20susceptibility%20testing.%20Nature632,%20893%E2%80%93902%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Cheng, I.-F., Chen, T.-Y., Lu, R.-J. &amp; Wu, H.-W. Rapid identification of bacteria utilizing amplified dielectrophoretic force-assisted nanoparticle-induced surface-enhanced Raman spectroscopy. <em>Nanoscale Res. Lett.</em><strong>9</strong>, 324 (2014).
</cite> [<a href="https://doi.org/10.1186/1556-276X-9-324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4085094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25024685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20I.-F.,%20Chen,%20T.-Y.,%20Lu,%20R.-J.%20&amp;%20Wu,%20H.-W.%20Rapid%20identification%20of%20bacteria%20utilizing%20amplified%20dielectrophoretic%20force-assisted%20nanoparticle-induced%20surface-enhanced%20Raman%20spectroscopy.%20Nanoscale%20Res.%20Lett.9,%20324%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Ohlsson, P. et al. Integrated acoustic separation, enrichment, and microchip polymerase chain reaction detection of bacteria from blood for rapid sepsis diagnostics. <em>Anal. Chem.</em><strong>88</strong>, 9403–9411 (2016).
</cite> [<a href="https://doi.org/10.1021/acs.analchem.6b00323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27264110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ohlsson,%20P.%20et%20al.%20Integrated%20acoustic%20separation,%20enrichment,%20and%20microchip%20polymerase%20chain%20reaction%20detection%20of%20bacteria%20from%20blood%20for%20rapid%20sepsis%20diagnostics.%20Anal.%20Chem.88,%209403%E2%80%939411%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Cooney, S., O’Brien, S., Iversen, C. &amp; Fanning, S. Bacteria: Other Pathogenic Enterobacteriaceae - Enterobacter and Other Genera. in <em>Encyclopedia of Food Safety</em> (ed Motarjemi, Y.) 433–441 (Academic Press, Waltham, 2014). <a href="https://www.sciencedirect.com/science/article/pii/B9780123786128001049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sciencedirect.com/science/article/pii/B9780123786128001049</a>.</cite>
</li>
<li id="CR56">
<span class="label">56.</span><cite>Sutton Scott. Accuracy of plate counts. <em>J. Valid. Technol.</em><strong>17</strong>, 42–46 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sutton%20Scott.%20Accuracy%20of%20plate%20counts.%20J.%20Valid.%20Technol.17,%2042%E2%80%9346%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Narayana Iyengar, S. et al. Toward rapid detection of viable bacteria in whole blood for early sepsis diagnostics and susceptibility testing. <em>ACS Sens.</em><strong>6</strong>, 3357–3366 (2021).
</cite> [<a href="https://doi.org/10.1021/acssensors.1c01219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8477386/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34410700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Narayana%20Iyengar,%20S.%20et%20al.%20Toward%20rapid%20detection%20of%20viable%20bacteria%20in%20whole%20blood%20for%20early%20sepsis%20diagnostics%20and%20susceptibility%20testing.%20ACS%20Sens.6,%203357%E2%80%933366%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>He, K., Zhang, X., Ren, S. &amp; Sun, J. Deep residual learning for image recognition. In <em>Proc. IEEE Conference on Computer Vision and Pattern Recognition</em>, 770–778. <a href="http://ieeexplore.ieee.org/document/7780459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://ieeexplore.ieee.org/document/7780459</a> (2016).</cite>
</li>
<li id="CR59">
<span class="label">59.</span><cite>Tan, M. &amp; Le, Q. Efficientnetv2: smaller models and faster training. In <em>Proc. 38th International Conference on Machine Learning</em>, 10096–10106 (PMLR, 2021).</cite>
</li>
<li id="CR60">
<span class="label">60.</span><cite>Oquab, M. et al. Dinov2: Learning robust visual features without supervision. Preprint at <a href="https://arxiv.org/abs/2304.07193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/2304.07193</a> (2023).</cite>
</li>
<li id="CR61">
<span class="label">61.</span><cite>Dosovitskiy, A. et al. An image is worth 16x16 words: Transformers for image recognition at scale. In <em>Proc. International Conference on Learning Representations</em> (2021). Preprint at <a href="https://arxiv.org/abs/2010.11929" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/2010.11929</a>.</cite>
</li>
<li id="CR62">
<span class="label">62.</span><cite>Tran, D. et al. A closer look at spatiotemporal convolutions for action recognition. In <em>Proc. IEEE Conference on Computer Vision and Pattern Recognition (CVPR)</em>, 6450–6459. 10.1109/CVPR.2018.00675 (2018).</cite>
</li>
<li id="CR63">
<span class="label">63.</span><cite>Tabah, A. et al. Characteristics and determinants of outcome of hospital-acquired bloodstream infections in intensive care units: the Eurobact International Cohort Study. <em>Intensive Care Med.</em><strong>38</strong>, 1930–1945 (2012).
</cite> [<a href="https://doi.org/10.1007/s00134-012-2695-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23011531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tabah,%20A.%20et%20al.%20Characteristics%20and%20determinants%20of%20outcome%20of%20hospital-acquired%20bloodstream%20infections%20in%20intensive%20care%20units:%20the%20Eurobact%20International%20Cohort%20Study.%20Intensive%20Care%20Med.38,%201930%E2%80%931945%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Umemura, Y. et al. Current spectrum of causative pathogens in sepsis: a prospective nationwide cohort study in Japan. <em>Int. J. Infect. Dis.</em><strong>103</strong>, 343–351 (2021).
</cite> [<a href="https://doi.org/10.1016/j.ijid.2020.11.168" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33221519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Umemura,%20Y.%20et%20al.%20Current%20spectrum%20of%20causative%20pathogens%20in%20sepsis:%20a%20prospective%20nationwide%20cohort%20study%20in%20Japan.%20Int.%20J.%20Infect.%20Dis.103,%20343%E2%80%93351%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Blumenreich, M., Walker, H., Hall, W. &amp; Hurst, J. <em>Clinical Methods: The History, Physical, and Laboratory Examinations</em>. 3rd edn, (1990).</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/21250045/" class="usa-link">PubMed</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Loeb, L. The influence of certain bacteria on the coagulation of the blood. <em>J. Med. Res.</em><strong>10</strong>, 407–419 (1903).
</cite> [<a href="/articles/PMC2105975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19971581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Loeb,%20L.%20The%20influence%20of%20certain%20bacteria%20on%20the%20coagulation%20of%20the%20blood.%20J.%20Med.%20Res.10,%20407%E2%80%93419%20(1903)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>McAdow, M., Missiakas, D. M. &amp; Schneewind, O. Staphylococcus aureus secretes coagulase and von Willebrand factor binding protein to modify the coagulation cascade and establish host infections. <em>J. Innate Immun.</em><strong>4</strong>, 141–148 (2012).
</cite> [<a href="https://doi.org/10.1159/000333447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3388267/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22222316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McAdow,%20M.,%20Missiakas,%20D.%20M.%20&amp;%20Schneewind,%20O.%20Staphylococcus%20aureus%20secretes%20coagulase%20and%20von%20Willebrand%20factor%20binding%20protein%20to%20modify%20the%20coagulation%20cascade%20and%20establish%20host%20infections.%20J.%20Innate%20Immun.4,%20141%E2%80%93148%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Evans, D. C. et al. Cooperation between coagulase and von Willebrand factor binding protein in Staphylococcus aureus fibrin pseudocapsule formation. <em>Biofilm</em><strong>8</strong>, 100233 (2024).
</cite> [<a href="https://doi.org/10.1016/j.bioflm.2024.100233" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11564979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39555140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Evans,%20D.%20C.%20et%20al.%20Cooperation%20between%20coagulase%20and%20von%20Willebrand%20factor%20binding%20protein%20in%20Staphylococcus%20aureus%20fibrin%20pseudocapsule%20formation.%20Biofilm8,%20100233%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Liesenborghs, L., Verhamme, P. &amp; Vanassche, T. Staphylococcus aureus, master manipulator of the human hemostatic system. <em>J. Thrombosis Haemost.</em><strong>16</strong>, 441–454 (2018).</cite> [<a href="https://doi.org/10.1111/jth.13928" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29251820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liesenborghs,%20L.,%20Verhamme,%20P.%20&amp;%20Vanassche,%20T.%20Staphylococcus%20aureus,%20master%20manipulator%20of%20the%20human%20hemostatic%20system.%20J.%20Thrombosis%20Haemost.16,%20441%E2%80%93454%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Peetermans, M. et al. Targeting coagulase activity in Staphylococcus aureus bacteraemia: a randomized controlled single-centre trial of Staphylothrombin inhibition. <em>Thrombosis Haemost.</em><strong>118</strong>, 818–829 (2018).</cite> [<a href="https://doi.org/10.1055/s-0038-1639586" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29614521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peetermans,%20M.%20et%20al.%20Targeting%20coagulase%20activity%20in%20Staphylococcus%20aureus%20bacteraemia:%20a%20randomized%20controlled%20single-centre%20trial%20of%20Staphylothrombin%20inhibition.%20Thrombosis%20Haemost.118,%20818%E2%80%93829%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Hallström, E., Kandavalli, V., Wählby, C. &amp; Hast, A. Rapid identification of seven bacterial species using microfluidics, time-lapse phase-contrast microscopy, and deep learning. Preprint at <em>bioRxiv</em><a href="https://www.biorxiv.org/content/10.1101/2024.10.15.618380v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.biorxiv.org/content/10.1101/2024.10.15.618380v1</a> (2024).</cite>
</li>
<li id="CR72">
<span class="label">72.</span><cite>Kandavalli, V., Karempudi, P., Larsson, J. &amp; Elf, J. Rapid antibiotic susceptibility testing and species identification for mixed samples. <em>Nat. Commun.</em><strong>13</strong>, 6215 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-33659-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9584937/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36266330/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kandavalli,%20V.,%20Karempudi,%20P.,%20Larsson,%20J.%20&amp;%20Elf,%20J.%20Rapid%20antibiotic%20susceptibility%20testing%20and%20species%20identification%20for%20mixed%20samples.%20Nat.%20Commun.13,%206215%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Ahmad, I. et al. Phenotypic antibiotic susceptibility testing at the limit of one bacterial cell. Preprint at <em>bioRxiv</em><a href="https://www.biorxiv.org/content/10.1101/2025.04.13.648565v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.biorxiv.org/content/10.1101/2025.04.13.648565v1</a> (2025).</cite>
</li>
<li id="CR74">
<span class="label">74.</span><cite>Camsund, D. et al. Time-resolved imaging-based CRISPRi screening. <em>Nat. Methods</em><strong>17</strong>, 86–92 (2020).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0629-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31740817/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Camsund,%20D.%20et%20al.%20Time-resolved%20imaging-based%20CRISPRi%20screening.%20Nat.%20Methods17,%2086%E2%80%9392%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Lin, M., Chen, Q. &amp; Yan, S. Network in network. Preprint at <a href="https://arxiv.org/abs/1312.4400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/1312.4400</a> (2013).</cite>
</li>
<li id="CR76">
<span class="label">76.</span><cite>Duchon, C. E. Lanczos filtering in one and two dimensions. <em>J. Appl. Meteorol. Climatol.</em><strong>18</strong>, 1016–1022 (1979).</cite> [<a href="https://scholar.google.com/scholar_lookup?Duchon,%20C.%20E.%20Lanczos%20filtering%20in%20one%20and%20two%20dimensions.%20J.%20Appl.%20Meteorol.%20Climatol.18,%201016%E2%80%931022%20(1979)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Garnier et al. <em>viridis(Lite) - Colorblind-Friendly Color Maps for R</em> (2024). <a href="https://sjmgarnier.github.io/viridis/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sjmgarnier.github.io/viridis/</a>. Viridis package version 0.6.5.</cite>
</li>
<li id="CR78">
<span class="label">78.</span><cite>Kingma, D. P. &amp; Ba, J. Adam: a method for stochastic optimization. Preprint at <a href="https://arxiv.org/abs/1412.6980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/1412.6980</a> (2014).</cite>
</li>
<li id="CR79">
<span class="label">79.</span><cite>Loshchilov, I. &amp; Hutter, F. Sgdr: stochastic gradient descent with warm restarts. Preprint at <a href="https://arxiv.org/abs/1608.03983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/1608.03983</a> (2016).</cite>
</li>
<li id="CR80">
<span class="label">80.</span><cite>Szegedy, C., Vanhoucke, V., Ioffe, S., Shlens, J. &amp; Wojna, Z. Rethinking the inception architecture for computer vision.In <em>Proc. IEEE Computer Society Conference on Computer Vision and Pattern Recognition</em>, 2818–2826. 10.1109/CVPR.2016.308 (2016).</cite>
</li>
<li id="CR81">
<span class="label">81.</span><cite>Wightman, R. Pytorch image models. <a href="https://github.com/rwightman/pytorch-image-models" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/rwightman/pytorch-image-models</a> (2019).</cite>
</li>
<li id="CR82">
<span class="label">82.</span><cite>Buslaev, A. et al. Albumentations: fast and flexible image augmentations. <em>Information</em><strong>11</strong><a href="https://www.mdpi.com/2078-2489/11/2/125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.mdpi.com/2078-2489/11/2/125</a> (2020).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375708/bin/41746_2025_1948_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (25.3MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The raw data (microscopy image data, flow measurements, plate count data, analysis summary files, codes, etc.) used to generate the information compiled in this manuscript can be found in the following open repository: <a href="https://figshare.com/s/affea349261af5a2f637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/s/affea349261af5a2f637</a>.</p>
<p>All codes and replication packages for the ML models can be found in the following open repository: <a href="https://figshare.com/s/affea349261af5a2f637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/s/affea349261af5a2f637</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from NPJ Digital Medicine are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41746-025-01948-w"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41746_2025_Article_1948.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375708/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375708/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375708%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375708/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375708/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375708/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40851034/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375708/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40851034/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375708/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375708/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ZiswGWEbKIapjfhJFSQKNosVLep7vSyeG2dnf2H2QnG1ijyR17mbvgbI2IfqYxKX">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
