
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Transferability of genomic prediction models across market segments in potato and the effect of selection - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152545448AF31493054544004E10E31C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="springeropen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367938/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik">
<meta name="citation_title" content="Transferability of genomic prediction models across market segments in potato and the effect of selection">
<meta name="citation_author" content="Kathrin Thelen">
<meta name="citation_author_institution" content="Julius Kühn-Institute (JKI) – Federal Research Centre for Cultivated Plants, Institute for Breeding Research on Agricultural Crops, 18190 Sanitz, Germany">
<meta name="citation_author_institution" content="Professorship for Utilization of Plant Genetic Resources for Breeding Purposes, Faculty of Agricultural- and Environmental Sciences, University of Rostock, 18051 Rostock, Germany">
<meta name="citation_author" content="Po-Ya Wu">
<meta name="citation_author_institution" content="Julius Kühn-Institute (JKI) – Federal Research Centre for Cultivated Plants, Institute for Breeding Research on Agricultural Crops, 18190 Sanitz, Germany">
<meta name="citation_author_institution" content="Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), 06466 Seeland, Germany">
<meta name="citation_author" content="Nadia Baig">
<meta name="citation_author_institution" content="Institute for Quantitative Genetics and Genomics of Plants, Heinrich Heine University, 40225 Düsseldorf, Germany">
<meta name="citation_author_institution" content="EUROPLANT Innovation GmbH &amp; Co. KG, 21337 Lüneburg, Germany">
<meta name="citation_author" content="Vanessa Prigge">
<meta name="citation_author_institution" content="SaKa Pflanzenzucht GmbH &amp; Co. KG, 24340 Windeby, Germany">
<meta name="citation_author" content="Julien Bruckmüller">
<meta name="citation_author_institution" content="SaKa Pflanzenzucht GmbH &amp; Co. KG, 24340 Windeby, Germany">
<meta name="citation_author" content="Katja Muders">
<meta name="citation_author_institution" content="Nordring-Kartoffelzucht- und Vermehrungs- GmbH, 18190 Sanitz, Germany">
<meta name="citation_author" content="Bernd Truberg">
<meta name="citation_author_institution" content="Nordring-Kartoffelzucht- und Vermehrungs- GmbH, 18190 Sanitz, Germany">
<meta name="citation_author" content="Stefanie Hartje">
<meta name="citation_author_institution" content="EUROPLANT Innovation GmbH &amp; Co. KG, 21337 Lüneburg, Germany">
<meta name="citation_author" content="Juliane Renner">
<meta name="citation_author_institution" content="EUROPLANT Innovation GmbH &amp; Co. KG, 21337 Lüneburg, Germany">
<meta name="citation_author" content="Delphine Van Inghelandt">
<meta name="citation_author_institution" content="Julius Kühn-Institute (JKI) – Federal Research Centre for Cultivated Plants, Institute for Breeding Research on Agricultural Crops, 18190 Sanitz, Germany">
<meta name="citation_author_institution" content="Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), 06466 Seeland, Germany">
<meta name="citation_author" content="Benjamin Stich">
<meta name="citation_author_institution" content="Julius Kühn-Institute (JKI) – Federal Research Centre for Cultivated Plants, Institute for Breeding Research on Agricultural Crops, 18190 Sanitz, Germany">
<meta name="citation_author_institution" content="Professorship for Utilization of Plant Genetic Resources for Breeding Purposes, Faculty of Agricultural- and Environmental Sciences, University of Rostock, 18051 Rostock, Germany">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="138">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="219">
<meta name="citation_doi" content="10.1007/s00122-025-05004-9">
<meta name="citation_pmid" content="40833434">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367938/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367938/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367938/pdf/122_2025_Article_5004.pdf">
<meta name="description" content="Genomic prediction (GP) can help increase the efficiency of breeding programs, as genotypes can be selected based on their predicted performance. However, to the best of our knowledge, this procedure is not yet routine in commercial breeding ...">
<meta name="og:title" content="Transferability of genomic prediction models across market segments in potato and the effect of selection">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Genomic prediction (GP) can help increase the efficiency of breeding programs, as genotypes can be selected based on their predicted performance. However, to the best of our knowledge, this procedure is not yet routine in commercial breeding ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367938/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12367938">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1007/s00122-025-05004-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/122_2025_Article_5004.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367938%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12367938/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12367938/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367938/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-springeropen.png" alt="Springer logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Springer" title="Link to Springer" shape="default" href="https://doi.org/10.1007/s00122-025-05004-9" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Theor Appl Genet</button></div>. 2025 Aug 20;138(9):219. doi: <a href="https://doi.org/10.1007/s00122-025-05004-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1007/s00122-025-05004-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Theor%20Appl%20Genet%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Theor%20Appl%20Genet%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Theor%20Appl%20Genet%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Theor%20Appl%20Genet%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Transferability of genomic prediction models across market segments in potato and the effect of selection</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thelen%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Kathrin Thelen</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Kathrin Thelen</span></h3>
<div class="p">
<sup>1</sup>Julius Kühn-Institute (JKI) – Federal Research Centre for Cultivated Plants, Institute for Breeding Research on Agricultural Crops, 18190 Sanitz, Germany </div>
<div class="p">
<sup>2</sup>Professorship for Utilization of Plant Genetic Resources for Breeding Purposes, Faculty of Agricultural- and Environmental Sciences, University of Rostock, 18051 Rostock, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thelen%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kathrin Thelen</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20PY%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Po-Ya Wu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Po-Ya Wu</span></h3>
<div class="p">
<sup>1</sup>Julius Kühn-Institute (JKI) – Federal Research Centre for Cultivated Plants, Institute for Breeding Research on Agricultural Crops, 18190 Sanitz, Germany </div>
<div class="p">
<sup>7</sup>Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), 06466 Seeland, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20PY%22%5BAuthor%5D" class="usa-link"><span class="name western">Po-Ya Wu</span></a>
</div>
</div>
<sup>1,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baig%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Nadia Baig</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Nadia Baig</span></h3>
<div class="p">
<sup>3</sup>Institute for Quantitative Genetics and Genomics of Plants, Heinrich Heine University, 40225 Düsseldorf, Germany </div>
<div class="p">
<sup>6</sup>EUROPLANT Innovation GmbH &amp; Co. KG, 21337 Lüneburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baig%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nadia Baig</span></a>
</div>
</div>
<sup>3,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prigge%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Vanessa Prigge</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Vanessa Prigge</span></h3>
<div class="p">
<sup>4</sup>SaKa Pflanzenzucht GmbH &amp; Co. KG, 24340 Windeby, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prigge%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Vanessa Prigge</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bruckm%C3%BCller%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Julien Bruckmüller</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Julien Bruckmüller</span></h3>
<div class="p">
<sup>4</sup>SaKa Pflanzenzucht GmbH &amp; Co. KG, 24340 Windeby, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bruckm%C3%BCller%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Julien Bruckmüller</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Muders%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Katja Muders</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Katja Muders</span></h3>
<div class="p">
<sup>5</sup>Nordring-Kartoffelzucht- und Vermehrungs- GmbH, 18190 Sanitz, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Muders%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Katja Muders</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Truberg%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Bernd Truberg</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Bernd Truberg</span></h3>
<div class="p">
<sup>5</sup>Nordring-Kartoffelzucht- und Vermehrungs- GmbH, 18190 Sanitz, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Truberg%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bernd Truberg</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hartje%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Stefanie Hartje</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Stefanie Hartje</span></h3>
<div class="p">
<sup>6</sup>EUROPLANT Innovation GmbH &amp; Co. KG, 21337 Lüneburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hartje%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Stefanie Hartje</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Renner%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Juliane Renner</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Juliane Renner</span></h3>
<div class="p">
<sup>6</sup>EUROPLANT Innovation GmbH &amp; Co. KG, 21337 Lüneburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Renner%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Juliane Renner</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20Inghelandt%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Delphine Van Inghelandt</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Delphine Van Inghelandt</span></h3>
<div class="p">
<sup>1</sup>Julius Kühn-Institute (JKI) – Federal Research Centre for Cultivated Plants, Institute for Breeding Research on Agricultural Crops, 18190 Sanitz, Germany </div>
<div class="p">
<sup>7</sup>Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), 06466 Seeland, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20Inghelandt%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Delphine Van Inghelandt</span></a>
</div>
</div>
<sup>1,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stich%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Benjamin Stich</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Benjamin Stich</span></h3>
<div class="p">
<sup>1</sup>Julius Kühn-Institute (JKI) – Federal Research Centre for Cultivated Plants, Institute for Breeding Research on Agricultural Crops, 18190 Sanitz, Germany </div>
<div class="p">
<sup>2</sup>Professorship for Utilization of Plant Genetic Resources for Breeding Purposes, Faculty of Agricultural- and Environmental Sciences, University of Rostock, 18051 Rostock, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stich%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Benjamin Stich</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Julius Kühn-Institute (JKI) – Federal Research Centre for Cultivated Plants, Institute for Breeding Research on Agricultural Crops, 18190 Sanitz, Germany </div>
<div id="Aff2">
<sup>2</sup>Professorship for Utilization of Plant Genetic Resources for Breeding Purposes, Faculty of Agricultural- and Environmental Sciences, University of Rostock, 18051 Rostock, Germany </div>
<div id="Aff3">
<sup>3</sup>Institute for Quantitative Genetics and Genomics of Plants, Heinrich Heine University, 40225 Düsseldorf, Germany </div>
<div id="Aff4">
<sup>4</sup>SaKa Pflanzenzucht GmbH &amp; Co. KG, 24340 Windeby, Germany </div>
<div id="Aff5">
<sup>5</sup>Nordring-Kartoffelzucht- und Vermehrungs- GmbH, 18190 Sanitz, Germany </div>
<div id="Aff6">
<sup>6</sup>EUROPLANT Innovation GmbH &amp; Co. KG, 21337 Lüneburg, Germany </div>
<div id="Aff7">
<sup>7</sup>Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), 06466 Seeland, Germany </div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Communicated by Mikko J. Sillanpää.</p></div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 8; Accepted 2025 Jul 26; Issue date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12367938  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40833434/" class="usa-link">40833434</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Genomic prediction (GP) can help increase the efficiency of breeding programs, as genotypes can be selected based on their predicted performance. However, to the best of our knowledge, this procedure is not yet routine in commercial breeding programs in tetraploid organisms like potato (<em>Solanum tuberosum</em> L.). The objectives of this study were to </p>
<ul class="list" style="list-style-type:none">
<li>
<span class="label">(i)</span><p class="display-inline" id="Par2">Estimate the prediction accuracy for 26 different potato traits in a panel of about 1000 genotypes based on 202,008 single nucleotide polymorphisms,</p>
</li>
<li>
<span class="label">(ii)</span><p class="display-inline" id="Par3">Evaluate the influence of the size and constitution of the training set on the prediction accuracy, and</p>
</li>
<li>
<span class="label">(iii)</span><p class="display-inline" id="Par4">Investigate how the effect of selection in the training set influences the outcome of GP.</p>
</li>
</ul>
<p> GP revealed high prediction accuracies using genomic best linear unbiased prediction. Our results indicated that a training set of 280–480 clones and 10,000 markers was sufficient. Prediction within a specific market segment led to a higher prediction accuracy compared to adding clones from other market segments to the training set or to predict between different market segments. Lastly, we found a higher prediction accuracy when in a training set of selected clones, i.e., a training set that consists of clones with high trait values, 20% of the clones were replaced by clones that were sampled from the clones that showed the lowest 10% trait values. This observation shows that clones from advanced breeding stages can be used as training set, if some clones specifically from the other side of the distribution range are added to the training set.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1007/s00122-025-05004-9.</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par5">Potato (<em>Solanum tuberosum</em> L.) is one of the most important food crops worldwide (Devaux et al. <a href="#CR18" class="usa-link" aria-describedby="CR18">2020</a>; FAOSTAT database <a href="#CR22" class="usa-link" aria-describedby="CR22">2024</a>). Its high nutrient content, together with its high amount of carbohydrates (Reddy et al. <a href="#CR44" class="usa-link" aria-describedby="CR44">2018</a>), makes it an essential component of the diets of one billion people worldwide (Devaux et al. <a href="#CR18" class="usa-link" aria-describedby="CR18">2020</a>). Potato can be grown in many different environments under temperate, subtropical, and tropical conditions (Devaux et al. <a href="#CR18" class="usa-link" aria-describedby="CR18">2020</a>; Bradshaw <a href="#CR9" class="usa-link" aria-describedby="CR9">2017</a>). Besides, it is a versatile short-duration crop whose demand is steadily increasing (Reddy et al. <a href="#CR44" class="usa-link" aria-describedby="CR44">2018</a>). Potato has various market segments, such as fresh consumption, processing to crisps or French fries, or starch production. Potato breeding programs aim for the development of specific cultivars suited for those different market segments (Bradshaw <a href="#CR9" class="usa-link" aria-describedby="CR9">2017</a>). The breeding progress made for relevant traits in the past is limited by the autotetraploidy of potato, as well as its high heterozygosity and inbreeding depression (Naeem et al. <a href="#CR36" class="usa-link" aria-describedby="CR36">2021</a>). Furthermore, more than 40 different traits have to be evaluated during the selection process (Slater et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2013</a>). Some of these traits are evaluated relatively late in the breeding process, especially because the number of available tubers is low in the early stages of the selection process and the evaluation of some traits includes destructive measurements (Gebhardt <a href="#CR24" class="usa-link" aria-describedby="CR24">2013</a>), and/or requires multi-environment testing (Jansky <a href="#CR31" class="usa-link" aria-describedby="CR31">2009</a>).</p>
<p id="Par6">Genetic gain can be improved by using molecular genetic markers like single nucleotide polymorphisms (SNPs) for selection (Crossa et al. <a href="#CR16" class="usa-link" aria-describedby="CR16">2017</a>). This is especially true as the genotyping costs were notably reduced over the past years and will likely continue to drop (Slater et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2013</a>; Crossa et al. <a href="#CR16" class="usa-link" aria-describedby="CR16">2017</a>). One promising approach of using molecular markers for selection of quantitative traits is genomic prediction (GP). GP, which was originally proposed by Meuwissen et al. (<a href="#CR34" class="usa-link" aria-describedby="CR34">2001</a>), uses genome-wide marker effects estimated in a phenotyped and genotyped training set (TS) to estimate so-called genomic estimated breeding values (GEBVs) of a validation set (VS) that is only genotyped. This reduces the time as well as the costs of one breeding cycle (Slater et al. <a href="#CR47" class="usa-link" aria-describedby="CR47">2016</a>), as selection can be performed in earlier stages of the breeding program based on the predicted performance of the clones and thereby lowers the phenotyping costs (Sverrisdóttir et al. <a href="#CR52" class="usa-link" aria-describedby="CR52">2018</a>). Thus, the implementation of GP can increase the gain of selection, which has also been illustrated for potato based on computer simulations (Wu et al. <a href="#CR64" class="usa-link" aria-describedby="CR64">2023</a>). GP was originally designed for animal breeding (Meuwissen et al. <a href="#CR34" class="usa-link" aria-describedby="CR34">2001</a>), but its implication in plant breeding has been studied intensively during the last decades (for review see Crossa et al., <a href="#CR16" class="usa-link" aria-describedby="CR16">2017</a>; Alemu et al., <a href="#CR4" class="usa-link" aria-describedby="CR4">2024</a>). However, studies focusing on the implementation of GP in tetraploid organisms like alfalfa (Li et al. <a href="#CR32" class="usa-link" aria-describedby="CR32">2015</a>), ryegrass (Guo et al. <a href="#CR26" class="usa-link" aria-describedby="CR26">2018</a>), blueberry (De Bem Oliveira et al. <a href="#CR17" class="usa-link" aria-describedby="CR17">2020</a>), and potato (e.g., Slater et al. <a href="#CR47" class="usa-link" aria-describedby="CR47">2016</a>; Sverrisdóttir et al. <a href="#CR52" class="usa-link" aria-describedby="CR52">2018</a>; Habyarimana et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2017</a>; Selga et al. <a href="#CR46" class="usa-link" aria-describedby="CR46">2022</a>, ) are less abundant. Particularly empirical analyses of prediction accuracy in large sets of tetraploid potato clones for a wide range of commercially relevant traits in breeding programs of potato are largely missing.</p>
<p id="Par7">One important step in the implementation of GP is the establishment of the TS. Several studies focusing on the optimization of the TS, mainly in diploids (e.g., Rio et al. <a href="#CR45" class="usa-link" aria-describedby="CR45">2021</a>; Akdemir et al. <a href="#CR2" class="usa-link" aria-describedby="CR2">2015</a>; Isidro et al. <a href="#CR30" class="usa-link" aria-describedby="CR30">2015</a>, ), and a few in tetraploids (e.g., Sverrisdóttir et al. <a href="#CR52" class="usa-link" aria-describedby="CR52">2018</a>; Wilson et al. <a href="#CR60" class="usa-link" aria-describedby="CR60">2021a</a>, ) have been published. These studies aimed at optimizing the composition of the training set based on the known genomic relationship among the individual entries. Wilson et al. (<a href="#CR60" class="usa-link" aria-describedby="CR60">2021a</a>) found that including genetically more similar clones in TS and VS led to better predictions in tetraploids. Brauner et al. (<a href="#CR11" class="usa-link" aria-describedby="CR11">2020</a>) found that using unrelated genotypes decreases the prediction accuracy in maize, while Norman et al. (<a href="#CR39" class="usa-link" aria-describedby="CR39">2018</a>) found that the accuracy can be improved by increasing the diversity of the TS in wheat. Sverrisdóttir et al. (<a href="#CR52" class="usa-link" aria-describedby="CR52">2018</a>) found that adding individuals from more distant populations to the training set will not increase the PA for the traits chipping quality and dry matter content.</p>
<p id="Par8">In potato breeding programs, clones from different market segments are available in each breeding stage. However, to the best of our knowledge, it is not clear if and how to combine these clones from different market segments in the context of TS establishment. In addition, only little is known about the effect of a preselection for target traits in potato, or how promising it is to include in the TS genotypes that would not have been advanced in a breeding program.</p>
<p id="Par9">The objectives of this study were to </p>
<ul class="list" style="list-style-type:none">
<li>
<span class="label">(i)</span><p class="display-inline" id="Par10">Estimate the prediction accuracy for 26 different potato traits in a panel of about 1,000 genotypes based on 202,008 SNPs,</p>
</li>
<li>
<span class="label">(ii)</span><p class="display-inline" id="Par11">Evaluate the influence of the size and constitution of the training set on the prediction accuracy, and</p>
</li>
<li>
<span class="label">(iii)</span><p class="display-inline" id="Par12">Investigate how selection in the TS can influence the outcome of GP.</p>
</li>
</ul></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Genetic material</h3>
<p id="Par13">The plant material of this study was a subset of the potato breeding populations from the breeding companies SaKa (SaKa Pflanzenzucht GmbH &amp; Co. KG), Norika (Nordring-Kartoffelzucht- und Vermehrungs- GmbH), and Europlant (EUROPLANT Innovation GmbH &amp; Co. KG). Overall, 1066 tetraploid potato clones corresponding to the A clone level (<em>cf.</em> Stich &amp; Van Inghelandt <a href="#CR51" class="usa-link" aria-describedby="CR51">2018</a>) were used in our study. These clones belong to 171 full-sib families, which were designated in the following as segregating populations. The number of clones within one population varied from one to 38 clones (Fig. S1).</p>
<p id="Par14">The above-mentioned 1066 clones were evaluated in the years 2019, 2020, and 2021 in field experiments in different locations in Germany. Each breeding company evaluated their proprietary clones (i.e., entries) together with eight common variety checks. This experimental design was chosen due to intellectual property and phytosanitary reasons. The evaluated entries represent four different market segments, which correspond to the main usage groups of potatoes in Europe: crisp production (CR), French fries production (FF), starch production (ST), and table potato (TA). The number of clones within one market segment varied among the segments, with ST being the smallest (150 clones), followed by TA (263 clones) and FF (266 clones), while CR was the largest group with 379 clones. The assignment of clones to the market segments was conducted based on the parental market segment as cross purpose as defined by the responsible breeder.</p>
<p id="Par15">In order to examine the effect of preselection on the prediction accuracy of GP, 330 out of the overall 1066 genotypes corresponded to clones that would have normally been discarded in commercial breeding programs in the single hills stage based on breeder’s selection on different trait combinations for the four different market segments (Table S1). This group of clones, in the following designated as clones with discard status, comprised clones from biparental families that would have normally been discarded completely (68 clones, discard status 2), as well as clones from biparental families, where other clones from the same family were retained as A clones (262 clones, discard status 1).</p>
<p id="Par16">Clones were evaluated at one location in 2019, which was Kaltenberg for Europlant, Groß Lüsewitz for Norika, and Windeby for SaKa. In 2020 and 2021, an additional location was added for each breeding company, which was Böhlendorf for Europlant, Mehringen for Norika, and Gransebieth for SaKa. This resulted in five different year-location combinations for each of the three breeding companies, which were designated in the following as environments (Table S2).</p>
<p id="Par17">In general, more clones were evaluated in 2019 than in 2020 and 2021, which was a result of virus infections in multiplication plots that lead to the unavailability of clones in the 2020 and 2021 experiments due to phytosanitary requirements. Within the environments, the clones were organized in a block system following an augmented experimental study design. SaKa organized their clones in eight different blocks, whereas Europlant and Norika each organized their clones in up to four different blocks. The blocks were further organized into rows and columns. The eight checks were replicated eight times in each environment, at least one time in each block, while the entries were cultivated only once within each environment of their donor breeding company. Thus, each entry was replicated five times in total across all environments. The number of plants per plot ranged from nine to 20, depending on the respective environment (Table S2). In 2021, the experiment from the breeding company SaKa was further organized in two different trials, which split the clones according to their foliage maturity group. One trial contained clones from the extra early to early maturity group, and the other trial contained clones from the middle early to middle late maturity group. The two trials were adjacent to one another in the field and each trial contained the eight check clones in each block.</p>
<p id="Par18">Data were recorded on an individual plot basis for 26 different traits (Table <a href="#Tab1" class="usa-link">1</a>). Trait values were either assessed as a rating from 1 to 9 or given in the form of a percentage value. The traits were assessed using methods that were commonly used by the three breeding companies (Table <a href="#Tab1" class="usa-link">1</a>) and are standardized techniques, so that the breeding companies minimized the level of subjectivity within and between the environments. In addition, the ratings were either performed by one person per environment or one person per block. Furthermore, the total tuber yield per plot (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e473"><msub><mtext>YLD</mtext><mtext>raw</mtext></msub></math></span>) was measured in kilograms and corrected to a tuber yield per plot of 16 plants (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e484"><mtext>YLD</mtext></math></span>) based on the following model, which is employed in the commercial breeding programs (Truberg, personal communication):</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e494" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mtext>YLD</mtext><mo>=</mo><mfrac><msub><mtext>YLD</mtext><mtext>raw</mtext></msub><mrow><mtext>PN</mtext><mo>-</mo><msub><mtext>MP</mtext><mrow><mo>&gt;</mo><mn>20</mn><mo>%</mo></mrow></msub></mrow></mfrac><mrow></mrow><mo>∗</mo><mn>16</mn><mspace width="0.166667em"></mspace><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">1</td>
</tr></table>
<p>where PN is the number of plants planted for the corresponding plot and MP is the amount of missing plants in the plot. Here, MP was only subtracted from PN if MP exceeded 20% of PN, as for a lower extent of MP a full compensation of the remaining plants is expected (Truberg, personal communication). The plot size was set to 16 plants, as this was across all environments the most frequently used plot size (Table S2).</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Abbreviations and units for the evaluated traits considered in our study</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Abbreviation</th>
<th align="left" colspan="1" rowspan="1">Trait</th>
<th align="left" colspan="1" rowspan="1">Unit</th>
<th align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq130"><math id="d33e3258"><msup><mi>h</mi><mn>2</mn></msup></math></span></th>
<th align="left" colspan="1" rowspan="1">Method</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">BRU *</td>
<td align="left" colspan="1" rowspan="1">Susceptibility to bruising</td>
<td align="left" colspan="1" rowspan="1">%</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">5 kg tuber sample [1]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CR4 *</td>
<td align="left" colspan="1" rowspan="1">Crisps color after storage at 4<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq131"><math id="d33e3288"><mmultiscripts><mrow></mrow><mrow></mrow><mo>∘</mo></mmultiscripts></math></span>C</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.79</td>
<td align="left" colspan="1" rowspan="1">1 = bad quality (very dark), 9 = good quality (no discoloration) [2]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CR8 *</td>
<td align="left" colspan="1" rowspan="1">Crisps color after storage at 8<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq132"><math id="d33e3312"><mmultiscripts><mrow></mrow><mrow></mrow><mo>∘</mo></mmultiscripts></math></span>C</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">1 = bad quality (very dark), 9 = good quality (no discoloration) [2]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DEV</td>
<td align="left" colspan="1" rowspan="1">Foliage development</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.67</td>
<td align="left" colspan="1" rowspan="1">1 = not grown to very weak development, 9 = superior / extraordinary growth [2]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DSC *</td>
<td align="left" colspan="1" rowspan="1">After cooking discoloration</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.59</td>
<td align="left" colspan="1" rowspan="1">1 = very dark, 9 = no discoloration [2]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EMR</td>
<td align="left" colspan="1" rowspan="1">Emergence</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.74</td>
<td align="left" colspan="1" rowspan="1">rate of emergence from the soil, 1 = very poor, 9 = very good [2]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EYE</td>
<td align="left" colspan="1" rowspan="1">Eye depth</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">1 = very deep, 9 = very flat [2]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FLE</td>
<td align="left" colspan="1" rowspan="1">Flesh color</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">1 = white, 9 = blue/purple [1]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FRI *</td>
<td align="left" colspan="1" rowspan="1">French fry color</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.80</td>
<td align="left" colspan="1" rowspan="1">1 = bad quality (very dark), 9 = good quality (no discoloration) [2]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IMP</td>
<td align="left" colspan="1" rowspan="1">General impression</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.68</td>
<td align="left" colspan="1" rowspan="1">1 = deficiencies, 9 = very good [2]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MAT</td>
<td align="left" colspan="1" rowspan="1">Foliage maturity</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.84</td>
<td align="left" colspan="1" rowspan="1">1 = still flowering, 9 = dead, relative to checks [2]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PPO</td>
<td align="left" colspan="1" rowspan="1">Polyphenol oxidase activity</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.90</td>
<td align="left" colspan="1" rowspan="1">tuber flesh after DL-DOPA incubation, 1 = no color change, 9 = very dark coloration</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RHI</td>
<td align="left" colspan="1" rowspan="1">Rhizoctonia symptoms</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.57</td>
<td align="left" colspan="1" rowspan="1">1 = &gt;90 % infested, 9 = no symptoms</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SCA</td>
<td align="left" colspan="1" rowspan="1">Common scab symptoms</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.45</td>
<td align="left" colspan="1" rowspan="1">
<p>1 = &gt;90 % infested, 9 = no symptoms ,</p>
<p>2-8 = up to 90, 60, 30, 20 10, 5, 2 % infected, respectively</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SHD</td>
<td align="left" colspan="1" rowspan="1">Tuber shape diagonally</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.48</td>
<td align="left" colspan="1" rowspan="1">1 = round, 9 = flat</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SHL</td>
<td align="left" colspan="1" rowspan="1">Tuber shape longitudinally</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.93</td>
<td align="left" colspan="1" rowspan="1">1 = round, 9 = long [2]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SIZ</td>
<td align="left" colspan="1" rowspan="1">Tuber size</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.82</td>
<td align="left" colspan="1" rowspan="1">1 = very small, 9 = very big [2]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SKC</td>
<td align="left" colspan="1" rowspan="1">Skin color</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">1 = cream, 9 = blue/purple [2]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SKT</td>
<td align="left" colspan="1" rowspan="1">Skin type</td>
<td align="left" colspan="1" rowspan="1">1-4</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">1 = smooth, 4 = cracked, 5 = russet [1]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">STA</td>
<td align="left" colspan="1" rowspan="1">Starch content</td>
<td align="left" colspan="1" rowspan="1">%</td>
<td align="left" colspan="1" rowspan="1">0.95</td>
<td align="left" colspan="1" rowspan="1">
<p>under water weight</p>
<p>measurement with automatic starch scale [1]</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TEX *</td>
<td align="left" colspan="1" rowspan="1">Texture after cooking</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.50</td>
<td align="left" colspan="1" rowspan="1">1 = tuber falls completely apart, 9 = tuber stays tightly together [1]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TST *</td>
<td align="left" colspan="1" rowspan="1">Taste</td>
<td align="left" colspan="1" rowspan="1">1–9</td>
<td align="left" colspan="1" rowspan="1">0.33</td>
<td align="left" colspan="1" rowspan="1">1 = very strong deficits (e.g., bitter), 9 = nice potato taste [1]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TUL *</td>
<td align="left" colspan="1" rowspan="1">Proportion of large tubers &gt;65 mm</td>
<td align="left" colspan="1" rowspan="1">%</td>
<td align="left" colspan="1" rowspan="1">0.88</td>
<td align="left" colspan="1" rowspan="1">weighing after grading</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TUN *</td>
<td align="left" colspan="1" rowspan="1">Proportion of normal tubers 35-65 mm</td>
<td align="left" colspan="1" rowspan="1">%</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
<td align="left" colspan="1" rowspan="1">weighing after grading</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TUS *</td>
<td align="left" colspan="1" rowspan="1">Proportion of small tubers &lt; 35 mm</td>
<td align="left" colspan="1" rowspan="1">%</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">weighing after grading</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YLD</td>
<td align="left" colspan="1" rowspan="1">Total tuber yield</td>
<td align="left" colspan="1" rowspan="1">kg</td>
<td align="left" colspan="1" rowspan="1">0.78</td>
<td align="left" colspan="1" rowspan="1">normalized to a 16 plant plot</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p112">
<p>The traits are sorted alphabetically</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq133"><math id="d33e3590"><mrow><mrow></mrow><mo>∗</mo></mrow></math></span> only assessed in years 2020 and 2021</p>
<p>[1] Bundessortenamt (<a href="#CR13" class="usa-link" aria-describedby="CR13">2019</a>)</p>
<p>[2] Tiemens-Hulscher et al. (<a href="#CR54" class="usa-link" aria-describedby="CR54">2013</a>)</p>
</div></div></section><p id="Par19">Due to limitations of the number of available tubers in 2019, the traits BRU, CR4, CR8, DSC, FRI, TEX, TST, TUL, TUN, and TUS were only assessed for the second and third year (2020 and 2021), while the other traits were assessed in all three years. The traits CR4 and FRI were not evaluated for all clones, but only for those clones that belonged to the specific market segment, which was CR for CR4 and FF for FRI.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Statistical analyses</h3>
<p id="Par20">In general, data preprocessing and phenotypic data analyses were performed as described in detail by Thelen et al. (<a href="#CR53" class="usa-link" aria-describedby="CR53">2025</a>). Briefly, the following analyses were carried out. In a first step, the data of the breeding company SaKa from 2021 were corrected for the trial effects of the two trials. This was done for each trait individually by first calculating the mean values of the eight checks for each trial. Then, the absolute value of the mean difference between the checks of both trials was subtracted from each observation of the trial with the higher mean value.</p>
<p id="Par21">In the following analyses, the breeding companies were denoted as B1, B2, and B3 and the different locations for each breeding company were given by L1 and L2. Potential outliers were then identified by fitting model (<a href="#Equ2" class="usa-link">2</a>) to the complete data set. As only checks were replicated in each environment, the genotype–environment interaction effect could only be estimated for the checks:</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e542" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><msub><mi>y</mi><mrow><mi mathvariant="italic">ijklm</mi></mrow></msub><mo>=</mo><mi>μ</mi><mo>+</mo><msub><mi>g</mi><mi>i</mi></msub><mo>+</mo><msub><mi>e</mi><mi>j</mi></msub><mo>+</mo><msub><mi>C</mi><mi>i</mi></msub><msub><mrow><mo stretchy="false">(</mo><mi>g</mi><mi>e</mi><mo stretchy="false">)</mo></mrow><mrow><mi mathvariant="italic">ij</mi></mrow></msub><mo>+</mo><msub><mi>b</mi><mrow><mi mathvariant="italic">kj</mi></mrow></msub><mo>+</mo><msub><mi>r</mi><mrow><mi mathvariant="italic">lkj</mi></mrow></msub><mo>+</mo><msub><mi>h</mi><mrow><mi mathvariant="italic">mkj</mi></mrow></msub><mo>+</mo><msub><mi>ϵ</mi><mrow><mi mathvariant="italic">ijklm</mi></mrow></msub><mspace width="0.166667em"></mspace><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">2</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e612"><msub><mi>y</mi><mrow><mi mathvariant="italic">ijklm</mi></mrow></msub></math></span> is the phenotypic observation of the <em>i</em>th potato clone in the <em>m</em>th column and the <em>l</em>th row of the <em>k</em>th block in the <em>j</em>th environment, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e641"><mi>μ</mi></math></span> an intercept term, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e650"><msub><mi>g</mi><mi>i</mi></msub></math></span> the effect of the <em>i</em>th clone, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e665"><msub><mi>e</mi><mi>j</mi></msub></math></span> the effect of the <em>j</em>th environment, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e679"><msub><mi>C</mi><mi>i</mi></msub></math></span> a dummy variable filtering for checks with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e690"><mrow><msub><mi>C</mi><mi>i</mi></msub><mo>=</mo><mn>1</mn></mrow></math></span> for checks and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e704"><mrow><msub><mi>C</mi><mi>i</mi></msub><mo>=</mo><mn>0</mn></mrow></math></span> for entries, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e718"><msub><mrow><mo stretchy="false">(</mo><mi>g</mi><mi>e</mi><mo stretchy="false">)</mo></mrow><mrow><mi mathvariant="italic">ij</mi></mrow></msub></math></span> the interaction effect of the <em>i</em>th clone and the <em>j</em>th environment, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e744"><msub><mi>b</mi><mrow><mi mathvariant="italic">kj</mi></mrow></msub></math></span> the effect of the <em>k</em>th block of the <em>j</em>th environment, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e763"><msub><mi>r</mi><mrow><mi mathvariant="italic">lkj</mi></mrow></msub></math></span> the effect of the <em>l</em>th row of the <em>k</em>th block of the <em>j</em>th environment, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e786"><msub><mi>h</mi><mrow><mi mathvariant="italic">mkj</mi></mrow></msub></math></span> the effect of the <em>m</em>th column of the <em>k</em>th block of the <em>j</em>th environment, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e809"><msub><mi>ϵ</mi><mrow><mi mathvariant="italic">ijklm</mi></mrow></msub></math></span> the residual error. Except for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e822"><msub><mi>g</mi><mi>i</mi></msub></math></span>, all effects were considered random. Based on this analysis for each trait, records with a standardized absolute residual value greater than 3.5 were considered as outliers and were removed from the data set.</p>
<p id="Par22">In the next step, a correction for the check-based block effect was realized as described for the trial effect in case of a significant (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e835"><mi>α</mi></math></span> = 0.05) likelihood ratio test (LRT) in model (<a href="#Equ2" class="usa-link">2</a>). The corrected trait values were used for all further analyses.</p>
<p id="Par23">Adjusted entry means (AEMs) for all clones (checks and entries) were calculated across all environments. When doing so, the residual error variance was assumed to be heterogeneous across the different environments (Thelen et al. <a href="#CR53" class="usa-link" aria-describedby="CR53">2025</a>).</p>
<p id="Par24">The genotypic variance of the entries was calculated by model (<a href="#Equ3" class="usa-link">3</a>), where the clone effect was split up between checks and entries:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e858" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><msub><mi>y</mi><mrow><mi mathvariant="italic">ijklm</mi></mrow></msub><mo>=</mo><mi>μ</mi><mo>+</mo><msub><mi>C</mi><mi>i</mi></msub><msub><mi>g</mi><mi>i</mi></msub><mo>+</mo><msub><mi>D</mi><mi>i</mi></msub><msub><mi>g</mi><mi>i</mi></msub><mo>+</mo><msub><mi>e</mi><mi>j</mi></msub><mo>+</mo><msub><mi>C</mi><mi>i</mi></msub><msub><mrow><mo stretchy="false">(</mo><mi>g</mi><mi>e</mi><mo stretchy="false">)</mo></mrow><mrow><mi mathvariant="italic">ij</mi></mrow></msub><mo>+</mo><msub><mi>b</mi><mrow><mi mathvariant="italic">kj</mi></mrow></msub><mo>+</mo><msub><mi>r</mi><mrow><mi mathvariant="italic">lkj</mi></mrow></msub><mo>+</mo><msub><mi>h</mi><mrow><mi mathvariant="italic">mkj</mi></mrow></msub><mo>+</mo><msub><mi>ϵ</mi><mrow><mi mathvariant="italic">ijklm</mi></mrow></msub><mspace width="0.166667em"></mspace><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">3</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e939"><msub><mi>D</mi><mi>i</mi></msub></math></span> is an indicator variable filtering for entries with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e950"><msub><mi>D</mi><mi>i</mi></msub></math></span>=0 for checks and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e961"><msub><mi>D</mi><mi>i</mi></msub></math></span>= 1 for entries. Thereby, the effect of the checks was regarded as fixed and the effect of the entries was regarded as random. Again, the residual error variance was assumed to be heterogeneous across the different environments.</p>
<p id="Par25">Heritability on an entry mean basis was calculated for each trait according to the following formula based on the entries (Piepho &amp; Möhring <a href="#CR42" class="usa-link" aria-describedby="CR42">2007</a>):</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e978" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><msup><mi>h</mi><mn>2</mn></msup><mo>=</mo><mfrac><msubsup><mi>σ</mi><mrow><mi>g</mi></mrow><mn>2</mn></msubsup><mrow><msubsup><mi>σ</mi><mrow><mi>g</mi></mrow><mn>2</mn></msubsup><mo>+</mo><mfrac><mover accent="true"><mi>ν</mi><mo stretchy="false">^</mo></mover><mn>2</mn></mfrac></mrow></mfrac><mspace width="0.166667em"></mspace><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">4</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e1016"><msubsup><mi>σ</mi><mrow><mi>g</mi></mrow><mn>2</mn></msubsup></math></span> is the genotypic variance of the entries estimated by model (<a href="#Equ3" class="usa-link">3</a>) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e1032"><mover accent="true"><mi>ν</mi><mo stretchy="false">^</mo></mover></math></span> the mean variance of a difference of two adjusted treatment means of the entries as they were estimated by model (<a href="#Equ2" class="usa-link">2</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Genomic prediction</h3>
<p id="Par26">Out of the 1066 different clones included in the field experiment described above, 988 have been genotyped using an SNP array (Baig et al., in preparation). The overall 224,009 high-quality SNP markers were filtered according to the following criteria: </p>
<ul class="list" style="list-style-type:none">
<li>
<span class="label">(i)</span><p class="display-inline" id="Par27">Loci had &lt; 20% missing data, and</p>
</li>
<li>
<span class="label">(ii)</span><p class="display-inline" id="Par28">Loci had a minor allele frequency <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e1063"><mo>≥</mo></math></span> 0.05.</p>
</li>
</ul>
<p>A total of 202,008 SNP markers remained after quality preprocessing of the complete data set. The 2.9% missing SNP data points were mean imputed.</p>
<p>GP was performed using the univariate genomic best linear unbiased prediction (GBLUP) model, which can be described as</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><math id="d33e1076" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mi mathvariant="bold">y</mi><mo>=</mo><msub><mn mathvariant="bold">1</mn><mi mathvariant="bold">n</mi></msub><mi>μ</mi><mo>+</mo><mrow><msub><mi mathvariant="bold">Z</mi><mi mathvariant="bold">a</mi></msub><mi mathvariant="bold">a</mi></mrow><mo>+</mo><mi>ϵ</mi><mspace width="0.166667em"></mspace><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">5</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e1111"><mi mathvariant="bold">y</mi></math></span> is a vector of the AEMs calculated from model (<a href="#Equ2" class="usa-link">2</a>) for each trait, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e1124"><msub><mn mathvariant="bold">1</mn><mi mathvariant="bold">n</mi></msub></math></span> a vector of ones, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e1137"><mi>μ</mi></math></span> the overall mean, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e1147"><msub><mi mathvariant="bold">Z</mi><mi mathvariant="bold">a</mi></msub></math></span> the design matrix mapping AEMs to clones, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e1160"><mi mathvariant="bold">a</mi></math></span> the vector of genetic values contributed by additive effects that are assumed to be normally distributed with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e1170"><mrow><mi>N</mi><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msub><mi mathvariant="bold">G</mi><mi mathvariant="bold">A</mi></msub><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></math></span>, in which <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e1195"><msub><mi mathvariant="bold">G</mi><mi mathvariant="bold">A</mi></msub></math></span> was the genomic relationship matrix calculated according to VanRaden (<a href="#CR57" class="usa-link" aria-describedby="CR57">2008</a>) and extended following Ashraf et al. (<a href="#CR7" class="usa-link" aria-describedby="CR7">2016</a>) using an additive autotetraploid model, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e1215"><mi>ϵ</mi></math></span> the vector of residual, assumed to be normally distributed <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e1224"><mrow><mi>N</mi><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><mi mathvariant="bold">I</mi><msubsup><mi>σ</mi><mi>ϵ</mi><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e1246"><mi mathvariant="bold">I</mi></math></span> is an identity matrix and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e1256"><msubsup><mi>σ</mi><mi>ϵ</mi><mn>2</mn></msubsup></math></span> the residual variance.</p>
<p id="Par29">In the next step, the model was extended by including dominance effects:</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><math id="d33e1271" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mi mathvariant="bold">y</mi><mo>=</mo><msub><mn mathvariant="bold">1</mn><mi mathvariant="bold">n</mi></msub><mi>μ</mi><mo>+</mo><mrow><msub><mi mathvariant="bold">Z</mi><mi mathvariant="bold">a</mi></msub><mi mathvariant="bold">a</mi></mrow><mo>+</mo><mrow><msub><mi mathvariant="bold">Z</mi><mi mathvariant="bold">d</mi></msub><mi mathvariant="bold">d</mi></mrow><mo>+</mo><mi>ϵ</mi><mspace width="0.166667em"></mspace><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">6</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e1316"><msub><mi mathvariant="bold">Z</mi><mi mathvariant="bold">d</mi></msub></math></span> is the design matrix mapping AEMs to clones, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e1329"><mi mathvariant="bold">d</mi></math></span> the vector of genetic values contributed by dominance effects, distributed as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e1339"><mrow><mi>N</mi><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msub><mi mathvariant="bold">G</mi><mi mathvariant="bold">D</mi></msub><msubsup><mi>σ</mi><mi>d</mi><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e1364"><msubsup><mi>σ</mi><mi>d</mi><mn>2</mn></msubsup></math></span> is the dominance variance and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e1377"><msub><mi mathvariant="bold">G</mi><mi mathvariant="bold">D</mi></msub></math></span> the dominance relationship matrix. In this study, the dominance effects, which are the deviation of the observed genetic value from the expected based on the pure additive model, were set differently for the three heterozygous genotypes (Aaaa, AAaa, and AAAa) and were designated by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e1390"><mrow><mi mathvariant="bold">d</mi><mn mathvariant="bold">1</mn></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e1403"><mrow><mi mathvariant="bold">d</mi><mn mathvariant="bold">2</mn></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e1416"><mrow><mi mathvariant="bold">d</mi><mn mathvariant="bold">3</mn></mrow></math></span>, respectively. This procedure resulted in the three separate dominance effect matrices <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><math id="d33e1429"><msub><mi mathvariant="bold">X</mi><mrow><mi mathvariant="bold">d</mi><mn mathvariant="bold">1</mn></mrow></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><math id="d33e1445"><msub><mi mathvariant="bold">X</mi><mrow><mi mathvariant="bold">d</mi><mn mathvariant="bold">2</mn></mrow></msub></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><math id="d33e1462"><msub><mi mathvariant="bold">X</mi><mrow><mi mathvariant="bold">d</mi><mn mathvariant="bold">3</mn></mrow></msub></math></span> with a dimension <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><math id="d33e1478"><mrow><mi>n</mi><mo>×</mo><mi>m</mi></mrow></math></span>, in which <em>n</em> was the number of clones and <em>m</em> the number of SNP markers. For <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><math id="d33e1496"><msub><mi mathvariant="bold">X</mi><mrow><mi mathvariant="bold">d</mi><mn mathvariant="bold">1</mn></mrow></msub></math></span>, clones carrying Aaaa were coded as 1 and the others 0; for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><math id="d33e1512"><msub><mi mathvariant="bold">X</mi><mrow><mi mathvariant="bold">d</mi><mn mathvariant="bold">2</mn></mrow></msub></math></span>, clones carrying AAaa were coded as 1 and the others 0, and for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><math id="d33e1529"><msub><mi mathvariant="bold">X</mi><mrow><mi mathvariant="bold">d</mi><mn mathvariant="bold">3</mn></mrow></msub></math></span>, clones carrying AAAa were coded as 1 and the others 0 (Gallais, <a href="#CR23" class="usa-link" aria-describedby="CR23">2003</a>; Table 2 in Wu et al., <a href="#CR63" class="usa-link" aria-describedby="CR63">2025</a>). Then, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><math id="d33e1551"><msub><mi mathvariant="bold">X</mi><mi mathvariant="bold">d</mi></msub></math></span> was defined as a combinatorial matrix of the three dominance effects matrices, expressed by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><math id="d33e1564"><mrow><msub><mi mathvariant="bold">X</mi><mi mathvariant="bold">d</mi></msub><mo>=</mo><mrow><mo stretchy="false">[</mo><msub><mi mathvariant="bold">X</mi><mrow><mi mathvariant="bold">d</mi><mn mathvariant="bold">1</mn></mrow></msub><mo>,</mo><msub><mi mathvariant="bold">X</mi><mrow><mi mathvariant="bold">d</mi><mn mathvariant="bold">2</mn></mrow></msub><mo>,</mo><msub><mi mathvariant="bold">X</mi><mrow><mi mathvariant="bold">d</mi><mn mathvariant="bold">3</mn></mrow></msub><mo stretchy="false">]</mo></mrow></mrow></math></span>, and was used to calculate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><math id="d33e1610"><mrow><msub><mi mathvariant="bold">G</mi><mi mathvariant="bold">D</mi></msub><mo>=</mo><mfrac><msup><mrow><msub><mi mathvariant="bold">X</mi><mi mathvariant="bold">d</mi></msub><msub><mi mathvariant="bold">X</mi><mi mathvariant="bold">d</mi></msub></mrow><mi>T</mi></msup><mrow><mn>3</mn><mspace width="0.166667em"></mspace><mi>m</mi></mrow></mfrac></mrow></math></span>.</p>
<p id="Par30">For the implementation of the dominance effect, we chose a median imputation in order to have a unique assignment of genotypes to dominance effects.</p>
<p id="Par31">The GBLUP models were implemented using the R packages AGHmatrix (Amadeu et al. <a href="#CR6" class="usa-link" aria-describedby="CR6">2023</a>) and sommer (Covarrubias-Pazaran <a href="#CR15" class="usa-link" aria-describedby="CR15">2016</a>).</p>
<p id="Par32">The effects of the model were estimated within a TS and then used to estimate the GEBVs of the genotypes of the VS. If not mentioned differently, a fivefold cross-validation (CV) approach was used in our study. For this procedure, the data were randomly split into five parts, and one part was used as the VS, while the other four parts were used as TS. Each of the five parts was independently used as VS once.</p>
<p id="Par33">Model performance for each individual trait was measured by the prediction accuracy (PA), which was the Pearson correlation coefficient between the AEMs from equation (<a href="#Equ2" class="usa-link">2</a>) and the GEBVs of the VS, divided by h (the square root of the heritability), to account for differences in the predictions which are due to differences in the heritability. Different modifications of the TS and VS were analyzed as described below.</p>
<p id="Par34">
<strong>Modifications of TS and VS</strong>
</p>
<p id="Par35"><em>Analysis of the complete data set:</em> In a first step, GP was applied to the complete data set with all available genotypes and markers. The CV process was replicated 50 times. This analysis was also used to assess the influence of dominance effects as described above in models (<a href="#Equ5" class="usa-link">5</a>) and (<a href="#Equ6" class="usa-link">6</a>). If the model resulted in a singularity, the calculation was canceled. In all further scenarios, only model (<a href="#Equ5" class="usa-link">5</a>) was used.</p>
<p id="Par36"><em>Effect of the size of the training set:</em> The influence of the size of the TS was examined using different random subsets of the complete data set. The subset sizes of the data set ranged from 50 to 950 in steps of 50 and, thus, the resulting TS sizes <em>N</em> ranged from 40 to 760 in steps of 40. The CV process was replicated 50 times, and for each replication, a new subset of the respective size was sampled from the complete data set.</p>
<p id="Par37"><em>Effect of the number of markers:</em> The influence of the number of SNPs on the PA was explored by using a stratified sampling approach. The genome was split into equally sized windows, and from each window one marker was randomly chosen. In different sub-scenarios, the window size was doubled and, thus, increased from 2 (set with 101,004 markers) to 1024 (197 markers). For each subset of markers, all available genotypes were used, and the fivefold CV was performed with 50 replications for each size scenario. For each replication, new SNPs were sampled.</p>
<p id="Par38"><em>Effect of the variability in the number of clones per segregating population: </em> In this study, the size of the different segregating populations differed considerably. Thus, to evaluate the effect of this unbalancedness, we compared the PA of the original unbalanced set, to a set with balanced population sizes. For the balanced set, four clones were taken randomly from each population to build the TS. Populations with less clones were (i) fully included or (ii) excluded. The respective TS sizes were defined by the number of populations with/without more than four clones and were for i) 457 clones and for ii) 268 clones. These sizes were then used to build an unbalanced TS with the contribution of the individual populations as in the complete set, by using a stratified sampling approach across populations. Thereby, for each TS, the clones of each population had the same proportion as in the complete data set, but TS sizes were comparable to scenario i) and ii). Again, clones from smaller populations were iii) included fully (TS size of 474) or iv) excluded (TS size of 257). For the random scenario, the v) 450 and vi) 270 clones were sampled randomly from the complete set of clones. The VS comprised 120 random clones not present in the TS and were the same across the three scenarios of similar sizes for the respective replication. Each procedure was repeated 50 times.</p>
<p id="Par39"><em>Transferability of GP models across market segments:</em> Within a breeding program that develops genetic material for different market segments, two different perspectives for the transferability of GP models across market segments exist (Fig. <a href="#Fig1" class="usa-link">1</a>). In the first perspective, breeders aim for the best PA of clones of one specific market segment. In this scenario, designated in the following as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><math id="d33e1701"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span>, the clones in the VS were explicitly taken from one market segment. Here, several sub-scenarios were examined. In the sub-scenario <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><math id="d33e1712"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span>, the clones in the TS belonged to the same market segment as the clones in the VS. However, we also examined in the sub-scenario <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><math id="d33e1723"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span> the influence of other clones in the TS. In this case, the clones in the TS were randomly chosen across all market segments but at the same time, the VS was kept constant. In the sub-scenario <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><math id="d33e1734"><msub><mtext>TS</mtext><mtext>between</mtext></msub></math></span>, the TS was a subset of clones from three market segments to equal shares, while the VS comprised the clones of the fourth market segment. The TS size was set to 200 clones; thus, the before-described analyses were only performed for VS comprising clones from TA, FF, or CR. The VS comprised 45 clones and the analyses were run with 150 replications. The traits CR4 and FRI were excluded from this analysis, as they were only evaluated in their own market segments.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367938_122_2025_5004_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/269e/12367938/bda852048887/122_2025_5004_Fig1_HTML.jpg" loading="lazy" id="MO1" height="390" width="676" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic diagram of the different prediction scenarios evaluated in our study to examine the transferability of prediction models among genetic groups, i.e., market segments, which were represented here by four different icons. The two main perspectives are depicted in the two rows and differ by the analyzed validation set (VS). In the first case of a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq140"><math id="d33e1752"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span>, the clones of the VS are from only one market segment, exemplified by the crisps potato. In the second perspective of a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq141"><math id="d33e1763"><mrow><mi>V</mi><msub><mi>S</mi><mtext>across</mtext></msub></mrow></math></span>, the clones in the VS come from all four market segments (French fry potato, crisps potato, starch potato, and table potato). Both perspectives can be trained with different training sets (TS). The first perspective was analyzed in the case of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq142"><math id="d33e1776"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span> when clones in the TS are from the same market segment as clones in the VS. In case of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq143"><math id="d33e1787"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span> clones in the TS are from all market segments. In case of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq144"><math id="d33e1798"><msub><mtext>TS</mtext><mtext>between</mtext></msub></math></span>, clones in the TS are from all market segments except the one in the VS. For the second perspective, clones in the TS are in the case of a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq145"><math id="d33e1810"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span> again from all market segments like in the VS, and in the case of a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq146"><math id="d33e1821"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span> from only one of the market segments. In the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq147"><math id="d33e1832"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span> scenarios, the market segments crisps potato, French fry potato, and table potato were considered separately</p></figcaption></figure><p id="Par40">In addition to a prediction where only clones from one market segment are part of the VS, in the context of a breeding program it is also relevant to realize GP with a VS consisting of different market segments. In this second perspective, the clones of the VS were sampled across different market segments instead of within one market segment. This scenario was called <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><math id="d33e1847"><msub><mtext>VS</mtext><mtext>across</mtext></msub></math></span>, which means that the clones in the VS were randomly sampled across all market segments. The performance of the clones from this VS was predicted from TS of clones of one market segment but also from TS that were assembled across market segments.</p>
<p id="Par41">In another scenario named <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><math id="d33e1860"><msub><mtext>TS</mtext><mrow><mi mathvariant="normal">between</mi><mo>-</mo><mi mathvariant="normal">pairwise</mi></mrow></msub></math></span>, the before-described <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><math id="d33e1876"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span>-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><math id="d33e1887"><msub><mtext>TS</mtext><mtext>between</mtext></msub></math></span> market segment analysis was also performed in a one-to-one comparison for various pairs of market segments: Clones from a single market segment were used as TS to predict the clones of another market segment. The number of replications for this variant was set to 15. Here again, the TS size was set to 200 clones.</p>
<p id="Par42"><em>Transferability of GP models across breeding companies:</em> The PA was also compared for predictions of the different breeding companies. Like for the market segments, two scenarios were examined, where the first scenario was the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><math id="d33e1902"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span> breeding company scenario. Here, the VS comprised only clones of one breeding company, leading again to the three sub-scenarios <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><math id="d33e1913"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span> (TS and VS from the same breeding company), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><math id="d33e1924"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span> (TS as a sample from all breeding companies), and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><math id="d33e1935"><msub><mtext>TS</mtext><mtext>between</mtext></msub></math></span> (TS from the other breeding companies, excluding clones from the breeding company in the VS). The second scenario was the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><math id="d33e1946"><msub><mtext>VS</mtext><mtext>across</mtext></msub></math></span> breeding company analysis. In this case, the VS was made up of clones from all breeding companies and the TS comprised clones of only one breeding company or clones from all breeding companies. All analyses were performed with identical sizes and proportions of the TS as for the market segment analyses.</p>
<p id="Par43"><em>Effect of truncation selection:</em> We analyzed the effect of trait-specific selection on the PA. To exemplify this selection, we selected for each sampling round clones of the TS belonging to the x% clones with the highest trait values. This does not necessarily correspond to the fraction selected by breeding companies but should not influence our conclusion. x ranged from 30 to 70 in steps of 10. Additionally, y% of each TS was exchanged by clones that did not belong to the clones with the x% highest trait values, where y ranged from 0 to 30 in steps of 10. For each combination of x and y values, 50 replications were performed. All clones that were not part of the TS were part of the VS. For this study, we used the four example traits STA, FLE, YLD, and EMR, which cover the whole range of high, medium, and low PA.</p>
<p id="Par44">To furthermore evaluate the influence of the type of exchanged clones, we compared for the scenario y=20% different sampling methods: </p>
<ul class="list" style="list-style-type:none">
<li>
<span class="label">(i)</span><p class="display-inline" id="Par45">Random sampling among all clones that did not belong to the set of clones with the x% highest trait values (as described above),</p>
</li>
<li>
<span class="label">(ii)</span><p class="display-inline" id="Par46">Stratified sampling among all clones that did not belong to the set of clones with the x% highest trait values. For stratification, the entire phenotypic range of a trait was divided into ten equally sized windows. The clones were then randomly selected to complement the TS from those windows that did not comprise clones of the TS. Each window contributed the equal amount of clones to realize a uniform distribution of the previously not represented windows. And</p>
</li>
<li>
<span class="label">(iii)</span><p class="display-inline" id="Par47">Sampling the clones from the set of clones with the lowest 10% trait values.</p>
</li>
</ul>
<p>Statistical differences between the sampling methods of the exchanged clones with respect to the PA were evaluated by using a t test with Bonferroni correction.</p>
<p>Breeders do not select their clones based on one trait alone, but on a combination of different traits. Thus, to evaluate this influence, we extracted the rank of each clone for each trait and then added up the ranks of the four above-mentioned traits STA, FLE, YLD, and EMR. This final rank value across the four traits was then used to define the top 50% and the bottom 10% of clones, which were used to build the TS and redo the above-described analyses. We examined the PA from a TS sampled from clones in the top 50% ranks and exchanged 0% or 20% in the TS with clones that belong to the bottom 10% ranks. Furthermore, we compared the PA of these analyses with the PA of the analyses with single trait values as described before that had the same values for x and y.</p>
<p id="Par48"><em>Effect of selection on correlated traits:</em> In a last step, we investigated if a previous selection on correlated traits of the examined clones influenced the PA. This was done by building a TS from subsets of the clones, including different amounts of clones with discard status. Clones with discard status 1 would normally have been discarded in the single hills stage based on their phenotype, but other clones of their population were retained as A clones. To remove the influence of the number and sizes of different populations, a stratified sampling approach was used. Thus, in the first scenario, clones without discard status (i.e., the clones that were in fact advanced in the single hills stage in the commercial breeding program) were sampled according to their population structure so that each population contributed proportionally as many clones to the reduced TS as they contributed to the entire set of clones. For the further scenarios, for each sampling round, <em>Z</em> clones without discard status were exchanged from the TS by clones with discard status 1, while keeping the contribution from each population to the TS constant. <em>Z</em> ranged from 10% to 50% of the TS in steps of 10% where the TS size was 200 clones. Each scenario was replicated 150 times. The VS only consisted of clones without discard status for this particular analysis.</p>
<p id="Par49">In order to check for the presence of market segment effects and the removal of these effects, an analysis including 0%, 20%, and 40% of clones with discard status 1 in the TS was also performed exclusively for clones from one market segment, namely CR. The market segment CR was used, as this was the biggest market segment with 235 clones with discard status 0, which led to a TS size of 188 clones in the fivefold CV. Here, the exchange was not performed within the families, but 0%, 20%, and 40% of clones without discard status were randomly replaced by clones with discard status 1. A total of 50 replications were used for each scenario.</p>
<p id="Par50">
<strong>Diversity of TS and VS</strong>
</p>
<p id="Par51">In order to evaluate to what extent the PA depends on the genetic distance between TS and VS, the genetic distance (Nei &amp; Roychoudhury <a href="#CR38" class="usa-link" aria-describedby="CR38">1974</a>) between each TS and VS was calculated for each prediction of the market segment analysis and the breeding company analysis using the R package StAMPP (Pembleton et al. <a href="#CR41" class="usa-link" aria-describedby="CR41">2013</a>). Furthermore, to evaluate to which extent the clones in the TS or the VS vary genetically, the gene diversity for each TS and VS, quantified by the expected heterozygosity (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><math id="d33e2007"><msub><mi mathvariant="normal">H</mi><mi mathvariant="normal">e</mi></msub></math></span>), was calculated as:</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><math id="d33e2021" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><msub><mi mathvariant="normal">H</mi><mi mathvariant="normal">e</mi></msub><mo>=</mo><mfrac><mn>1</mn><mi>m</mi></mfrac><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>m</mi></munderover><mfenced close=")" open="("><mn>1</mn><mo>-</mo><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>2</mn></munderover><msubsup><mi>p</mi><mrow><mi>i</mi><mo stretchy="false">(</mo><mi>j</mi><mo stretchy="false">)</mo></mrow><mn>4</mn></msubsup></mfenced><mspace width="0.166667em"></mspace><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">7</td>
</tr></table>
<p>where <em>m</em> is the number of markers, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><math id="d33e2076"><msub><mi>p</mi><mrow><mi>i</mi><mo stretchy="false">(</mo><mi>j</mi><mo stretchy="false">)</mo></mrow></msub></math></span> the allele frequency of the <em>i</em>th allele at the <em>j</em>th marker (Gallais <a href="#CR23" class="usa-link" aria-describedby="CR23">2003</a>; Wu et al. <a href="#CR63" class="usa-link" aria-describedby="CR63">2025</a>). <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><math id="d33e2105"><msub><mi mathvariant="normal">H</mi><mi mathvariant="normal">e</mi></msub></math></span> was also calculated for the set of parental clones. To compare this estimate with the gene diversity of the non-parental clones while accounting for differences in the set sizes, we randomly sampled 150 times 121 clones, which corresponds to the number of parental clones, from the set of non-parental clones and averaged the observed <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><math id="d33e2119"><msub><mi mathvariant="normal">H</mi><mi mathvariant="normal">e</mi></msub></math></span>.</p>
<p id="Par52">Phenotypic diversity was measured in our study by calculating the variance of the AEMs of the clones in the TS and the VS. The diversity measures were averaged across all replications from the same analysis, and statistical significance was evaluated by a t test with Bonferroni correction. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><math id="d33e2134"><msub><mi>Q</mi><mrow><mi mathvariant="italic">ST</mi></mrow></msub></math></span> (Spitze <a href="#CR50" class="usa-link" aria-describedby="CR50">1993</a>) is a statistic intended to measure the degree of genetic differentiation among populations with regard to a quantitative trait. It was calculated as the proportion of the genetic variance among the market segments to the complete genetic variance, which was the variance among the market segments plus the variance within the market segments. To check for the presence of structure in the data, a principal component analysis (PCA) was performed using the R package ade4 (Dray &amp; Dufour <a href="#CR19" class="usa-link" aria-describedby="CR19">2007</a>). Furthermore, the global fixation index (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><math id="d33e2153"><msub><mi>F</mi><mrow><mi mathvariant="italic">st</mi></mrow></msub></math></span>) was calculated according to Nei (<a href="#CR37" class="usa-link" aria-describedby="CR37">1986</a>) where for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><math id="d33e2170"><msub><mi>H</mi><mi>t</mi></msub></math></span> the complete data set was considered as one large meta population, and for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><math id="d33e2181"><msub><mi>H</mi><mi>s</mi></msub></math></span> the average expected <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><math id="d33e2192"><msub><mi>H</mi><mi>e</mi></msub></math></span> of the four market segments was calculated. Pairwise <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><math id="d33e2203"><msub><mi>F</mi><mrow><mi mathvariant="italic">st</mi></mrow></msub></math></span> values between the market segments were calculated using the R package StAMPP (Pembleton et al. <a href="#CR41" class="usa-link" aria-describedby="CR41">2013</a>).</p>
<p id="Par53">If not mentioned differently, all analyses were performed using the software R, version 4.2.1 (R Core Team <a href="#CR43" class="usa-link" aria-describedby="CR43">2022</a>).</p></section></section><section id="Sec6"><h2 class="pmc_sec_title">Results</h2>
<p id="Par54">Population structure based on SNP information was examined by a PCA, where the first two principal components explained 3.4% and 3.1%, respectively. Overall, four clusters were visible (Figs. S2 and S3). Two smaller clusters consisted exclusively of clones from one market segment, namely TA and ST. The clones from the market segment CR clustered together with the remaining clones of the market segment ST, while the clones from the market segment FF clustered together with clones from TA and CR (Figure S2). Furthermore, the two smaller clusters completely consisted of clones from breeding company 2, while no clear structure was observed in the other two clusters regarding the breeding company (Figure S3). The global <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><math id="d33e2228"><msub><mi>F</mi><mrow><mi mathvariant="italic">st</mi></mrow></msub></math></span> was 0.01, and the pairwise <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><math id="d33e2241"><msub><mi>F</mi><mrow><mi mathvariant="italic">st</mi></mrow></msub></math></span> values of the clones per market segment ranged between 0.01 and 0.02 (Table S3). The mean expected heterozygosity from 150 replications of 121 random tested clones was with 0.58 the same as that of the parental clones.</p>
<p id="Par55">In the next step, GP was carried out using the complete data set using a univariate additive and an additive-dominance model. Except for FLE, no significant improvement using the additive-dominance model was observed compared to the additive model (Fig. <a href="#Fig2" class="usa-link">2</a>). The highest PAs with values between 0.84 and 0.91 were found for the traits STA, CR8, SHL, and PPO. Overall, 22 out of 26 traits had a median PA above 0.5. Only EMR, MAT, TUS, and RHI had a median PA below 0.5. The median PA for tuber yield was 0.55. We observed that traits with a higher median PA also varied less around their median in the single CV runs compared to the traits with a lower median PA. A general high dispersion of the PA values was found for SCA, FRI, TST, CR4, TEX, and RHI.</p>
<figure class="fig xbox font-sm" id="Fig2"><h3 class="obj_head">Fig. 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367938_122_2025_5004_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/269e/12367938/9a5f5b8456f6/122_2025_5004_Fig2_HTML.jpg" loading="lazy" id="MO2" height="332" width="676" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Boxplot of the prediction accuracy (PA) of the complete data set for the 26 evaluated traits using the additive (blue) and additive-dominance (green) models. For each trait, a fivefold cross-validation with 50 replications was used. Significant differences of the PA between the models of one trait are indicated by ***. For abbreviations of the traits, see Table <a href="#Tab1" class="usa-link">1</a></p></figcaption></figure><p id="Par56">
<em>Effect of TS size and SNP number on PA:</em>
</p>
<p id="Par57">In a next step, the influences of the size of the TS (Fig. <a href="#Fig3" class="usa-link">3</a>A) and the number of SNPs (Fig. <a href="#Fig3" class="usa-link">3</a>B) on the PA were examined. The minimum size of the TS, which was needed to reach an optically defined plateau, depended on the trait (Fig. <a href="#Fig3" class="usa-link">3</a>A). Traits with an overall lower PA needed in general a bigger TS to reach the plateau (approx. 480 clones) compared to traits with a higher PA, where a TS size of roughly 280 clones was sufficient to reach the plateau. Nevertheless, the ranking of the traits regarding the PA remained the same irrespective of the TS sizes. Across all traits, a bigger TS size led to a reduced coefficient of variation (CoV) of the PA. However, we observed smaller CoVs even for small TS sizes when the median PA was high.</p>
<figure class="fig xbox font-sm" id="Fig3"><h3 class="obj_head">Fig. 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367938_122_2025_5004_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/269e/12367938/f75a810e2d82/122_2025_5004_Fig3_HTML.jpg" loading="lazy" id="MO3" height="811" width="676" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Median prediction accuracy (PA) of genomic predictions using different subsets of clones (A) and markers (B) of the complete data set for the 26 evaluated traits. The median PA was calculated across the fivefold cross-validation with 50 replications. Shaded backgrounds indicate the coefficient of variation (CoV) and are shown for the traits starch content (STA, pink), flesh color (FLE, orange), yield (YLD, purple), and emergence (EMR, green). For abbreviations of the traits, see Table <a href="#Tab1" class="usa-link">1</a></p></figcaption></figure><p id="Par58">When varying the number of SNPs used for the estimation of the genetic relationship matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq77"><math id="d33e2277"><msub><mi mathvariant="bold">G</mi><mi mathvariant="bold">a</mi></msub></math></span>, the same trend of trait-specific differences regarding the minimum number of markers to reach a plateau can be observed as for the TS size (Fig. <a href="#Fig3" class="usa-link">3</a>B). For traits with a high median PA, 2,000 SNPs were enough to reach the plateau, while traits with a lower median PA needed about 10,000 SNPs. Increasing the number of SNPs further only led to a marginal increase in PA. In addition to the PA, the number of SNPs also strongly impacted the CoV. The use of less than 2,000 SNPs to estimate the genetic relationship matrix yielded high CoVs for all traits. However, traits with high median PA showed again smaller CoVs than those with a low PA (Fig. <a href="#Fig3" class="usa-link">3</a>B).</p>
<p id="Par59">
<em>Effect of the variability in the number of clones per segregating population:</em>
</p>
<p id="Par60">We analyzed the effect of the unbalancedness of the number of clones per segregating population by comparing a TS which was build using a balanced number of clones from each segregating population, an unbalanced number of clones from each population which represented the population size of the complete set by stratified sampling, and a completely random set of clones (Figure S4). For the first two scenarios, clones from smaller populations were either included completely or in an alternative scenario excluded. The mean of the median PAs across all traits was in the case of including clones from smaller populations 0.61 for the balanced setting, 0.65 for the unbalanced setting, and 0.64 for the random sampling. For the scenario, in which the clones from the smaller populations were excluded, the mean of the median PAs across all traits was 0.56, 0.59, and 0.58 for the balanced, unbalanced, and random setting, respectively. The ranking of the methods was trait dependent. The balanced number of clones per segregating population yielded across all traits mostly the smallest PA.</p>
<p id="Par61">
<em>Transferability of GP models across market segments and breeding companies:</em>
</p>
<p id="Par62">We evaluated two different prediction scenarios in our study. On the one side, the prediction of differences among the clones of one market segment (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq78"><math id="d33e2310"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span>). In this situation, the highest PAs were observed after correcting for TS size in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq79"><math id="d33e2321"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span> sub-scenario, i.e., where clones from the same market segment were part of the TS and the VS (Fig. <a href="#Fig4" class="usa-link">4</a>). For some traits, the median PA within a specific market segment differed remarkably from the PA in the other market segments (e.g., IMP in FF or TST in CR). For these traits, also the CoV was much higher in the market segments with the deviating PA compared to the rest of the market segments. In general, the market segment which yielded the highest PA for the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq80"><math id="d33e2335"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span> market segment prediction varied noticeably among the traits. A wider TS, as it was the case in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq81"><math id="d33e2346"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span> sub-scenario, resulted in slightly lower PA than the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq82"><math id="d33e2358"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span> sub-scenario. However, the trends observed across the traits regarding the median PA in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq83"><math id="d33e2369"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span> market segment analysis were similar to those of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq84"><math id="d33e2380"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span> market segment analysis. Predicting the GEBVs of the clones of one market segment by only using clones from other market segments in the TS further reduced the median PA across all traits and market segments even more. This <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq85"><math id="d33e2391"><msub><mtext>TS</mtext><mtext>between</mtext></msub></math></span> market segment sub-scenario also showed the highest variation in the PA across the CV runs. The measure of genetic differentiation <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq86"><math id="d33e2402"><msub><mi>Q</mi><mrow><mi mathvariant="italic">ST</mi></mrow></msub></math></span> ranged across all traits from 0 to 0.74 (Table <a href="#Tab2" class="usa-link">2</a>). For some trait-market segment combinations, the predictions resulted in a negative PA, which were TEX in FF and RHI in FF and TA. Using the clones of one market segment to predict the clones of one other market segment in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq87"><math id="d33e2419"><msub><mtext>TS</mtext><mrow><mi mathvariant="normal">between</mi><mo>-</mo><mi mathvariant="normal">pairwise</mi></mrow></msub></math></span> market segment prediction yielded comparable PAs to the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq88"><math id="d33e2435"><msub><mtext>TS</mtext><mtext>between</mtext></msub></math></span> market segment sub-scenario (Figure S5).</p>
<figure class="fig xbox font-sm" id="Fig4"><h3 class="obj_head">Fig. 4.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367938_122_2025_5004_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/269e/12367938/ca9d67cae61f/122_2025_5004_Fig4_HTML.jpg" loading="lazy" id="MO4" height="611" width="676" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Heatmap of the median prediction accuracy (PA, numbers) and the respective coefficient of variation (CoV, colors) for prediction scenarios using clones from different market segments (TS size = 200 clones). The analyzed scenarios are described by their combination of clones from different market segments in the training set (TS) and validation set (VS). Analyzed market segments are crisps potato (CR), French fries potato (FF), and table potato (TA). In the sets where clones from more than one market segment are present, also clones from market segment starch production (ST) are present. Across designates a combination of clones from all market segments. CoVs higher than 1 or smaller than -1 are marked in brown. The boxplots above the heatmap represent the gene diversity (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq148"><math id="d33e3967"><msub><mi mathvariant="normal">H</mi><mi mathvariant="normal">e</mi></msub></math></span>) of each TS and VS and the measures of genetic distance for each TS-VS combination for each scenario. For abbreviations of the traits, see Table <a href="#Tab1" class="usa-link">1</a></p></figcaption></figure><section class="tw xbox font-sm" id="Tab2"><h3 class="obj_head">Table 2.</h3>
<div class="caption p"><p>Genetic variance between (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq134"><math id="d33e3617"><msubsup><mi>σ</mi><mrow><mi>m</mi></mrow><mn>2</mn></msubsup></math></span>) and within (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq135"><math id="d33e3630"><msubsup><mi>σ</mi><mrow><mi>m</mi><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo></mrow><mn>2</mn></msubsup></math></span>) market segments and the resulting degree of genetic differentiation <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq136"><math id="d33e3648"><msub><mi>Q</mi><mrow><mi mathvariant="italic">ST</mi></mrow></msub></math></span></p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq137"><math id="d33e3670"><msubsup><mi>σ</mi><mrow><mi>m</mi></mrow><mn>2</mn></msubsup></math></span></th>
<th align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq138"><math id="d33e3684"><msubsup><mi>σ</mi><mrow><mi>m</mi><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo></mrow><mn>2</mn></msubsup></math></span></th>
<th align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq139"><math id="d33e3703"><msub><mi>Q</mi><mrow><mi mathvariant="italic">ST</mi></mrow></msub></math></span></th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">STA</td>
<td align="left" colspan="1" rowspan="1">5.06</td>
<td align="left" colspan="1" rowspan="1">1.76</td>
<td align="left" colspan="1" rowspan="1">0.74</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CR8</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.49</td>
<td align="left" colspan="1" rowspan="1">0.65</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SHL</td>
<td align="left" colspan="1" rowspan="1">1.17</td>
<td align="left" colspan="1" rowspan="1">0.57</td>
<td align="left" colspan="1" rowspan="1">0.67</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PPO</td>
<td align="left" colspan="1" rowspan="1">1.81</td>
<td align="left" colspan="1" rowspan="1">2.02</td>
<td align="left" colspan="1" rowspan="1">0.47</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IMP</td>
<td align="left" colspan="1" rowspan="1">0.10</td>
<td align="left" colspan="1" rowspan="1">0.15</td>
<td align="left" colspan="1" rowspan="1">0.39</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BRU</td>
<td align="left" colspan="1" rowspan="1">438.38</td>
<td align="left" colspan="1" rowspan="1">241.19</td>
<td align="left" colspan="1" rowspan="1">0.65</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FLE</td>
<td align="left" colspan="1" rowspan="1">0.16</td>
<td align="left" colspan="1" rowspan="1">0.40</td>
<td align="left" colspan="1" rowspan="1">0.29</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EYE</td>
<td align="left" colspan="1" rowspan="1">0.41</td>
<td align="left" colspan="1" rowspan="1">0.39</td>
<td align="left" colspan="1" rowspan="1">0.51</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DSC</td>
<td align="left" colspan="1" rowspan="1">0.21</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">0.45</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SHD</td>
<td align="left" colspan="1" rowspan="1">0.02</td>
<td align="left" colspan="1" rowspan="1">0.29</td>
<td align="left" colspan="1" rowspan="1">0.07</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SCA</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">0.12</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SKT</td>
<td align="left" colspan="1" rowspan="1">0.08</td>
<td align="left" colspan="1" rowspan="1">0.23</td>
<td align="left" colspan="1" rowspan="1">0.26</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SKC</td>
<td align="left" colspan="1" rowspan="1">0.04</td>
<td align="left" colspan="1" rowspan="1">0.51</td>
<td align="left" colspan="1" rowspan="1">0.07</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TST</td>
<td align="left" colspan="1" rowspan="1">0.06</td>
<td align="left" colspan="1" rowspan="1">0.09</td>
<td align="left" colspan="1" rowspan="1">0.38</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SIZ</td>
<td align="left" colspan="1" rowspan="1">0.08</td>
<td align="left" colspan="1" rowspan="1">0.30</td>
<td align="left" colspan="1" rowspan="1">0.22</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TEX</td>
<td align="left" colspan="1" rowspan="1">0.08</td>
<td align="left" colspan="1" rowspan="1">0.22</td>
<td align="left" colspan="1" rowspan="1">0.27</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TUL</td>
<td align="left" colspan="1" rowspan="1">0.56</td>
<td align="left" colspan="1" rowspan="1">194.21</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DEV</td>
<td align="left" colspan="1" rowspan="1">0.02</td>
<td align="left" colspan="1" rowspan="1">0.27</td>
<td align="left" colspan="1" rowspan="1">0.07</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YLD</td>
<td align="left" colspan="1" rowspan="1">1.58</td>
<td align="left" colspan="1" rowspan="1">9.42</td>
<td align="left" colspan="1" rowspan="1">0.14</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TUN</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">138.90</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EMR</td>
<td align="left" colspan="1" rowspan="1">0.03</td>
<td align="left" colspan="1" rowspan="1">0.49</td>
<td align="left" colspan="1" rowspan="1">0.06</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MAT</td>
<td align="left" colspan="1" rowspan="1">0.04</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
<td align="left" colspan="1" rowspan="1">0.05</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TUS</td>
<td align="left" colspan="1" rowspan="1">0.37</td>
<td align="left" colspan="1" rowspan="1">5.62</td>
<td align="left" colspan="1" rowspan="1">0.06</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RHI</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">0.26</td>
<td align="left" colspan="1" rowspan="1">0.01</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p117"><p>For abbreviations of the traits, see Table <a href="#Tab1" class="usa-link">1</a></p></div></div></section><p id="Par63">Next to the before-described scenario, where the goal was to predict clones of a specific market segment, it might also be of interest how well clones of various market segments can be predicted. Therefore, we also analyzed the scenario <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq89"><math id="d33e2448"><msub><mtext>VS</mtext><mtext>across</mtext></msub></math></span>, where the VS was built from clones across all market segments and the TS was on the other hand restricted to one market segment or also built across market segments (Fig. <a href="#Fig4" class="usa-link">4</a>). Here, the highest PA was observed when using the broadest TS, i.e., a TS which was also built from clones across market segments (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq90"><math id="d33e2462"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span>). In comparison with a TS made up across market segments, the PA was only about 2/3 when using a TS made up from one market segment.</p>
<p id="Par64">In order to understand better the reasons for the observed pattern of PA, genetic distances between the TS and the VS and gene diversities within each TS and VS were calculated. In the case of a VS made up from one market segment (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq91"><math id="d33e2476"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span>), the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq92"><math id="d33e2487"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span> market segment sub-scenario revealed the smallest genetic distances between the TS and the VS, while in the case of a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq93"><math id="d33e2498"><msub><mtext>VS</mtext><mtext>across</mtext></msub></math></span> market segment prediction, the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq94"><math id="d33e2509"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span> showed the smallest genetic distance between TS and VS. Furthermore, we observed market segment specific differences in the genetic distance between the TS and the VS for the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq95"><math id="d33e2520"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span> - <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq96"><math id="d33e2532"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span> market segment and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq97"><math id="d33e2543"><msub><mtext>TS</mtext><mtext>between</mtext></msub></math></span> - <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq98"><math id="d33e2554"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span> market segment analyses, where the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq99"><math id="d33e2565"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span> - <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq100"><math id="d33e2576"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span> showed the smaller genetic distances between the TS and the VS compared to the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq101"><math id="d33e2587"><msub><mtext>TS</mtext><mtext>between</mtext></msub></math></span> - <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq102"><math id="d33e2599"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span> market segment sub-scenario. The highest genetic distance between the TS and the VS in the case of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq103"><math id="d33e2610"><msub><mtext>VS</mtext><mtext>across</mtext></msub></math></span> scenario was observed if the TS consisted of clones from the market segment TA.</p>
<p id="Par65">Across all traits and sub-scenarios, the gene diversity was higher in the TS than in the VS when considering the scenario of market segment specific VS (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq104"><math id="d33e2623"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span>). The gene diversity in the TS of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq105"><math id="d33e2634"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span> market segment prediction (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq106"><math id="d33e2645"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span> - <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq107"><math id="d33e2656"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span>) was remarkably lower compared to the gene diversity of the TS from the other sub-scenarios. Within each sub-scenario, the differences in the gene diversity between the different market segments were low. For the case of a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq108"><math id="d33e2667"><msub><mtext>VS</mtext><mtext>across</mtext></msub></math></span> market segment prediction, the gene diversity was higher in the VS than in the TS, except for the case where also the TS was built from clones across all market segments.</p>
<p id="Par66">Across all sub-scenarios in the case of market segment specific VS (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq109"><math id="d33e2680"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span>), we observed high market segment specific differences between the phenotypic variances. The phenotypic variance was across all traits mostly increased in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq110"><math id="d33e2691"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq111"><math id="d33e2702"><msub><mtext>TS</mtext><mtext>between</mtext></msub></math></span> market segment sub-scenario compared to the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq112"><math id="d33e2713"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span> market segment sub-scenario (Figure S6). Furthermore, for the second scenario of a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq113"><math id="d33e2724"><msub><mtext>VS</mtext><mtext>across</mtext></msub></math></span> market segments, the VS showed a higher phenotypic variance compared to the TS.</p>
<p id="Par67">In addition to the market segment subsets, we observed the effect of grouping clones within and across breeding companies. The general PA observed in these analyses was higher compared to the above-described market segment based analyses (Figure S7). However, the general trends of PA that were observed across the different subsets of breeding companies were similar to those of the different subsets of market segments (Figure S7 and Figure S8).</p>
<p id="Par68">
<em>Effect of truncation selection:</em>
</p>
<p id="Par69">We examined the effect of trait-specific selection across the entire data set by taking only a subset of the clones with the x% highest trait values in the TS. We observed an almost linear decrease in the PA when restricting the TS to the clones with the highest trait values (Fig. <a href="#Fig5" class="usa-link">5</a>A) despite a constant TS size. The CoV of the PA showed the inverse of this trend. The PA increased when randomly replacing 10, 20, or 30% of the clones in the TS by random clones that were not part of the clones with the x% highest trait values. For some cases, this inclusion led to a recovery of the original PA from the TS without selection. Including random clones that did not belong to the clones with the x% highest trait values also reduced the CoV and therefore yielded more stable results across all runs. While traits with a high PA predicted from a set of the 30% clones with the highest trait values yielded a medium PA and CoV (e.g., STA), this was not the case for traits with a lower median PA in the original scenario (e.g., YLD; EMR).</p>
<figure class="fig xbox font-sm" id="Fig5"><h3 class="obj_head">Fig. 5.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367938_122_2025_5004_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/269e/12367938/32d8312e76e5/122_2025_5004_Fig5_HTML.jpg" loading="lazy" id="MO5" height="1062" width="748" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Median prediction accuracy (PA) and the respective coefficient of variation (CoV) for four representative traits when sampling the clones for the training set (TS) of the clones with the x% highest trait values, and additionally exchanging y% of clones in the TS with clones that do not belong to the clones with the x% highest trait values (A) and different methods to exchange 20% of the clones in the training set (B). In A, different amounts (y%) of clones were randomly exchanged from the TS, which was sampled from clones with the x% highest trait values, by clones which do not belong the clones with the x% highest trait values. In B, three different methods to exchange clones in the TS were used: i) randomly from the set of clones whose trait values were not in the x% highest values, ii) stratified from equally sized windows of the trait values, from all windows that were not included in the sampling of the TS, and iii) clones were sampled from the clones with the 10% lowest trait values. Plus signs indicate the recovery of the median PA of the unselected TS. For abbreviations of the traits, see Table <a href="#Tab1" class="usa-link">1</a></p></figcaption></figure><p id="Par70">To evaluate the influence of the type of clone replacement, we compared different sampling methods for the exchange of 20% of clones in the TS with clones that did not belong to the set of clones with the x% highest trait values (Fig. <a href="#Fig5" class="usa-link">5</a> A and B). For all three cases in which the truncated TS was enriched by replacing clones of the TS, a higher PA was observed compared to the truncated TS alone (Fig. <a href="#Fig5" class="usa-link">5</a>B). However, the increase was more pronounced the less random the replacement was organized. We observed, e.g., a stronger increase in the PA for the scenario where clones of the truncated TS were replaced by clones with the 10% lowest trait values compared to replacing them randomly. This difference in the sampling methods was particularly pronounced when selection was stronger. On the other hand, this effect was less important the lower the effect of the truncated selection was.</p>
<p id="Par71">As breeders mostly do not select on one trait alone but on a combination of different traits, we also performed a selection on the best 50% of clones across all traits (Figure S9). Here, we found that the selection across multiple traits by the overall rank of a clone led to an increased PA compared to the selection on just the trait values of the trait of interest. However, also here an exchange of 20% of the clones in the TS with clones that belong to the clones with the 10% lowest ranks still improved the PA compared to using solely clones of the top 50% ranks as TS. However, this difference in the PA was less pronounced compared to the above-described individual trait analysis.</p>
<p id="Par72">
<em>Effect of selection on correlated traits in earlier generations:</em>
</p>
<p id="Par73">Lastly, the effect of selection on correlated traits was investigated in our study by examining the PA using different TS that included varying proportions of clones that would have been discarded in the single hills stage in a commercial breeding program. Across all examined traits, no difference in the median PA was observed if clones with discard status 1 were included with up to 50% of the TS (Fig. <a href="#Fig6" class="usa-link">6</a>). For some traits, e.g., IMP, TST, and TEX, an increase in the PA was observed, but for traits like, e.g., TUS, DSC, or SKT the reverse trend was observed. The CoV also showed no clear trend across all traits for the examined scenarios. Furthermore, the gene diversity decreased when adding clones with a discard status to the TS. This is presumably because the clones with discard status are more similar to each other than random clones.</p>
<figure class="fig xbox font-sm" id="Fig6"><h3 class="obj_head">Fig. 6.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367938_122_2025_5004_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/269e/12367938/b870e15c408f/122_2025_5004_Fig6_HTML.jpg" loading="lazy" id="MO6" height="599" width="676" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Heatmap of the median prediction accuracy (PA) and the respective coefficient of variation (CoV) for predictions where the training set of a size of 200 clones comprised an increasing amount of clones with discard status 1 (i.e., clones that would have been discarded in a commercial breeding program, but other clones of the same population would have been retained as A clones). Numbers indicate the median PA, while the colors show the CoV. CoV larger than 0.3 is marked in brown. The boxplot on the top represents the measures of gene diversity (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq149"><math id="d33e4000"><msub><mi mathvariant="normal">H</mi><mi mathvariant="normal">e</mi></msub></math></span>) within each training set averaged over all replications. For abbreviations of the traits, see Table <a href="#Tab1" class="usa-link">1</a></p></figcaption></figure><p id="Par74">To check whether the observed trends were caused by predicting across different market segments, which were selected by different criteria in the single hill stage, the effect of selection on correlated traits was also analyzed within the market segment CR alone (Figure S10). Here, we also found no noticeable difference in the median PA across the different traits when including clones with discard status or not.</p></section><section id="Sec7"><h2 class="pmc_sec_title">Discussion</h2>
<section id="Sec8"><h3 class="pmc_sec_title">Prediction accuracy of key traits for potato breeding</h3>
<p id="Par75">We observed across all traits high PAs in this study when using all genotypes and markers (Fig. <a href="#Fig2" class="usa-link">2</a>). This is presumably caused by the large number of clones in the TS, as well as the high number of SNPs compared to previous studies in potato (e.g., Stich &amp; Van Inghelandt <a href="#CR51" class="usa-link" aria-describedby="CR51">2018</a>; Endelman et al. <a href="#CR21" class="usa-link" aria-describedby="CR21">2018</a>, ). The PA reported in our study for common scab symptoms (SCA) was high compared to the one found by Enciso-Rodriguez et al. (<a href="#CR20" class="usa-link" aria-describedby="CR20">2018</a>). Nevertheless, the results of our study for common scab symptoms as well as rhizoctonia symptoms (RHI) need to be treated with care as the disease symptoms were assessed in environments without artificial disease inoculation. This can lead to a low phenotypic differentiation among clones.</p>
<p id="Par76">In our study, the genotype–environment interaction could be only estimated for the checks, as only these were repeated in the individual environments. However, as revealed in our companion study (Thelen et al. <a href="#CR53" class="usa-link" aria-describedby="CR53">2025</a>), the genotype–environment interaction was in general trait specific and especially high for the traits emergence, skin type, foliage development, and general impression. We observed a clear tendency that traits where phenotypic variation was largely driven by genotypic variance compared to environmental as well as genotype–environment interaction variance, as found in our companion paper (Thelen et al. <a href="#CR53" class="usa-link" aria-describedby="CR53">2025</a>), yielded the best PAs, e.g., STA, FLE, and SHL. In contrast, traits with a lower importance of genetic variance and, correspondingly, a higher importance of environmental variance led to lower PAs (e.g., MAT, YLD, and TUS), even though we considered these differences in the genetic complexity by comparing the PA, which includes <em>h</em>, and not prediction abilities, which would have been the correlation of the AEMs and the GEBVs. The high negative correlation of the PA of the different traits and the importance of genotype–environment interaction for the corresponding traits was in accordance with previous results (Wilson et al. <a href="#CR61" class="usa-link" aria-describedby="CR61">2021b</a>). Furthermore, we found that traits with a high PA showed less dispersion of the PAs across the individual runs of the CV procedure. Therefore, for traits with higher PA, not only the accuracy was higher but also the precision, which makes them particularly attractive targets for GP approaches. This is especially relevant for traits whose assessment on the phenotypic level is laborious or requires a destruction of tubers, such as CR8, BRU, FRI, or DSC, so that currently these traits can only be used for selection in advanced stages.</p>
<p id="Par77">One reason for the lower PA of some traits is presumably the way how the traits were assessed. We observed that traits with a stronger influence of environmental effects and lower heritability were rated on an ordinal scale. This classification might add some subjectivity and potential misclassification (Azevedo et al. <a href="#CR8" class="usa-link" aria-describedby="CR8">2024</a>), although it was performed based on methods commonly used in the breeding programs, which should minimize the given subjectivity. Furthermore, the rating was carried out by only one person per block or environment, so at least a part of the remaining subjectivity should be covered in the calculation of the AEMs (Model <a href="#Equ2" class="usa-link">2</a>) by the block- or environmental effect. However, this illustrates the potential of high-throughput phenotyping approaches, which allow a quantitative assessment of phenotypic properties and thereby might increase the PA.</p>
<p id="Par78">An additional factor that potentially explains the differences in PA among the traits is the chosen GP model. GBLUP is known to work best for traits that are influenced by many small genetic effects (Onogi et al. <a href="#CR40" class="usa-link" aria-describedby="CR40">2015</a>). Therefore, we have evaluated for some traits, which have been reported previously to be oligogenically inherited such as FLE, EYE (Van Eck <a href="#CR56" class="usa-link" aria-describedby="CR56">2007</a>), or PPO (Werij et al. <a href="#CR58" class="usa-link" aria-describedby="CR58">2007</a>; Urbany et al. <a href="#CR55" class="usa-link" aria-describedby="CR55">2011</a>), the PA of the BayesA GP model that is particularly suitable for traits with major effect loci (Meuwissen et al. <a href="#CR34" class="usa-link" aria-describedby="CR34">2001</a>). However, only a little improvement in the PAs was observed for these three traits using BayesA instead of GBLUP (increase of 0.02 to 0.06, data not shown), which is in accordance with the results of Stich &amp; Van Inghelandt (<a href="#CR51" class="usa-link" aria-describedby="CR51">2018</a>) and Wilson et al. (<a href="#CR61" class="usa-link" aria-describedby="CR61">2021b</a>). Therefore, we think that an inadequacy of the chosen GP model is not the main reason for PA differences among traits.</p>
<p id="Par79">The global as well as the pairwise <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq114"><math id="d33e2835"><msub><mi>F</mi><mrow><mi mathvariant="italic">st</mi></mrow></msub></math></span> values observed in our study (Table S3) suggested that the structure in our data set is considerably lower compared to that reported in other species (e.g., Akohoue et al. <a href="#CR3" class="usa-link" aria-describedby="CR3">2020</a>; Windhausen et al. <a href="#CR62" class="usa-link" aria-describedby="CR62">2012</a>). This was also supported by the PCA (Figures S2 and S3), where the PCs were made up from the SNPs that were also considered in the GBLUP analysis. Therefore, even if stronger population structure than presented here would be present, we do not expect from a methodological perspective that the conclusions made on the GBLUP model would change, if we used a prediction method that more directly accounted for population structure, e.g., by including the top PCs as covariates (Guo et al. <a href="#CR27" class="usa-link" aria-describedby="CR27">2014</a>). Thus, for all following analyses, only the GBLUP model was used for computational reasons.</p>
<p id="Par80">In our study, GP based on the GBLUP model with additive effects, as well as with additive and dominance effects, was examined. We found no improvement in the PA using the additive-dominance GBLUP compared to the additive GBLUP for all traits except FLE (Fig. <a href="#Fig2" class="usa-link">2</a>). This is in agreement with earlier studies. Amadeu et al. (<a href="#CR5" class="usa-link" aria-describedby="CR5">2020</a>) found little differences in the PA comparing different modeling strategies, including additive and additive-dominance BLUP. Wilson et al. (<a href="#CR61" class="usa-link" aria-describedby="CR61">2021b</a>) and Endelman et al. (<a href="#CR21" class="usa-link" aria-describedby="CR21">2018</a>) found trait-specific differences for the importance of dominance effects, where the PA of most traits was not improved by adding dominance effects to the model. Stich &amp; Van Inghelandt (<a href="#CR51" class="usa-link" aria-describedby="CR51">2018</a>) found for four out of seven traits an increase in the PA when modeling dominance effects. However, the traits examined in all these studies differ largely, making it difficult to conclude on the reason for the observed differences. Nevertheless, we can conclude that across all traits of our study, an additive model was sufficient to achieve high PAs and was therefore used for all further analyses in this study.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Factors influencing the PA</h3>
<p id="Par81">The proper setup of TS is an important task in breeding programs starting to implement GP. Increasing the training set size led to higher PAs (Fig. <a href="#Fig3" class="usa-link">3</a>A), which is in accordance with other studies in potato (Wilson et al. <a href="#CR60" class="usa-link" aria-describedby="CR60">2021a</a>; Endelman et al. <a href="#CR21" class="usa-link" aria-describedby="CR21">2018</a>). Nevertheless, TS sizes of around 400 clones led to almost the same PA and CoV as TS sizes of 760 clones. This is in contrast to, e.g., the results of Norman et al. (<a href="#CR39" class="usa-link" aria-describedby="CR39">2018</a>), who found in wheat a strong increase in the PA for up to 2,000 genotypes in the TS. This difference is presumably not due to differences in genetic complexity of the examined traits, as both studies examined traits of varying genetic complexity. Instead, the reason for smaller TS size requirements in our study compared to Norman et al. (<a href="#CR39" class="usa-link" aria-describedby="CR39">2018</a>) might be that in the latter a diversity panel was examined, while we studied clones from multiple segregating populations, which tends to reduce the genetic diversity. This means that in our study a smaller TS size was sufficient for an overall good PA, as more distant relationships might increase the noise and add bias to the prediction (Norman et al. <a href="#CR39" class="usa-link" aria-describedby="CR39">2018</a>; Hickey et al. <a href="#CR29" class="usa-link" aria-describedby="CR29">2014</a>).</p>
<p id="Par82">In our study, the clones of the respective TS were chosen randomly. Wilson et al. (<a href="#CR60" class="usa-link" aria-describedby="CR60">2021a</a>) found a significant interaction between sampling method and sampling size. Thus, changing the sampling method from random sampling to analytical sampling may further decrease the minimal TS size for maximal PA and therefore reduce phenotyping costs. However, this aspect was not further examined in our study.</p>
<p id="Par83">We found no increase in the median PA after passing a threshold of roughly 10,000 SNPs for prediction (Fig. <a href="#Fig3" class="usa-link">3</a>B). Also, Sverrisdóttir et al. (<a href="#CR52" class="usa-link" aria-describedby="CR52">2018</a>) found that about 10,000 SNPs were sufficient to obtain reliable PAs, where SNPs were derived from genotyping-by-sequencing. The relatively small number of required SNPs could be resulting from the fact that the analyses in our study but also Sverrisdóttir et al. (<a href="#CR52" class="usa-link" aria-describedby="CR52">2018</a>) were performed on segregating populations. That means that the linkage disequilibrium is higher compared to that of diversity panels (e.g., Stich &amp; Van Inghelandt <a href="#CR51" class="usa-link" aria-describedby="CR51">2018</a>, ) and, thus, fewer SNPs are required in the former case to obtain high PA. In addition, we observed that traits with a higher PA required fewer SNPs to reach a plateau of the PA compared to traits with a lower PA (Fig. <a href="#Fig3" class="usa-link">3</a>B). This is in agreement with the results of Aalborg et al. (<a href="#CR1" class="usa-link" aria-describedby="CR1">2024</a>), who also found a correlation between the PA and the required number of markers to reach a certain PA plateau.</p>
<p id="Par84">To deal with missing data, mean imputation was used in our study. A more complex imputation method like for example the one implemented in beagle (Browning et al. <a href="#CR12" class="usa-link" aria-describedby="CR12">2018</a>) is not possible. This is because linkage disequilibrium measures are mainly senseful to calculate only if the phase of the alleles is known (Slatkin <a href="#CR49" class="usa-link" aria-describedby="CR49">2008</a>). However, this is not the case for most polyploid species (Gerard <a href="#CR25" class="usa-link" aria-describedby="CR25">2021</a>). Due to the low rate of missing SNPs in our data set, we only expect marginal differences between scenarios of mean imputation and more sophisticated approaches and, thus, the simplest procedure was used in this manuscript.</p>
<p id="Par85">In our analyses, the SNPs were chosen purely based on trait independent aspects such as minor allele frequency and missing data rate (Baig et al., in preparation). However, a trait-specifically chosen subset of SNPs might further decrease the number of required SNPs. This was shown for fry color by Byrne et al. (<a href="#CR14" class="usa-link" aria-describedby="CR14">2020</a>), who found a higher prediction ability for SNPs that were selected by genome-wide association scans of the trait of interest compared to random SNPs. These findings demonstrated that an overall reduction of the number of markers can further decrease the genotyping costs and therefore make GP an even more efficient procedure. However, as this selection needs to be performed for each of the many traits of potato, this might not lead to a dramatic reduction of the total number of markers. Furthermore, with today’s genotyping technologies, the number of genotyped loci is typically not the main limitation. Therefore, we did not examine this aspect further in our study.</p>
<p id="Par86">The before described analyses were each build using a fivefold CV approach, i.e., the PA was calculated based on the same base population for the TS and VS, meaning they are not genetically independent of each other as the same alleles are shared and are not from a new breeding cycle. This has been shown before to inflate the PA (Werner et al. <a href="#CR59" class="usa-link" aria-describedby="CR59">2020</a>) and might not reflect the real in-field situation, where each year a new breeding cycle is started. In the frame of our study, we are not able to assess the size of this effect as genetic material from additional breeding cycles is required.</p>
<p id="Par87">In our study, the number of clones is largely unbalanced across segregating populations. This reflects the breeder’s reality. We tested in a sub-analysis the influence of the unbalanced number of clones per segregating population, comparing three sets of equal size: a balanced set across all segregating populations with the unbalanced set and a completely randomly sampled set (Figure S4). This analysis revealed a marginal effect of the unbalancedness on the PA.</p></section><section id="Sec02"><h3 class="pmc_sec_title">Transferability of prediction models among genetic groups</h3>
<p id="Par89">Potatoes from different market segments differ with respect to their profiles of trait values. Therefore, potato breeders often subdivide their genetic material into sub-groups that we designated in our study as market segments. Here, we compared the PA of predictions using combinations of clones from various market segments with those from only one market segment in the TS or VS. We considered two different perspectives. The first perspective was the case when breeders want to predict the performance of a set of clones from one market segment (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq115"><math id="d33e2953"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span>) by using clones from the same or other market segments in the TS. We found that an addition of clones from other market segments (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq116"><math id="d33e2964"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span>) to predict the clones of one market segment did not improve the PA (Fig. <a href="#Fig4" class="usa-link">4</a>). In addition, we observed that the genetic distance between TS and VS increased in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq117"><math id="d33e2978"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span> prediction sub-scenario compared to the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq118"><math id="d33e2989"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span> prediction sub-scenario. This is in accordance with findings from Lorenz &amp; Smith (<a href="#CR33" class="usa-link" aria-describedby="CR33">2015</a>) in barley, who also found a decrease in the PA when adding more genetically distant inbreds. These findings illustrate that the clones from the same market segment are diverse enough to allow high PA and, on the other side, do not add noise due to high genetic distance (Norman et al. <a href="#CR39" class="usa-link" aria-describedby="CR39">2018</a>; Hickey et al. <a href="#CR29" class="usa-link" aria-describedby="CR29">2014</a>). This first point was supported by our analysis of the genetic differentiation (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq119"><math id="d33e3010"><msub><mi>Q</mi><mrow><mi mathvariant="italic">ST</mi></mrow></msub></math></span>, Table <a href="#Tab2" class="usa-link">2</a>), which revealed that across all traits a high amount of genetic variance can be observed within the market segments compared to the complete genetic variance and, thus, the variance within the market segments seemed overall enough to reach a good PA.</p>
<p id="Par90">A consequence of the strategy to rely on within market segment prediction could be that some breeding programs who cannot afford a TS for each market segment may instead perform predictions for market segments that are of lower priority to them on the basis of between market segment predictions. We observed that the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq120"><math id="d33e3028"><msub><mtext>TS</mtext><mtext>between</mtext></msub></math></span> market segment prediction was only meaningful for traits that showed very high PA in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq121"><math id="d33e3039"><msub><mtext>TS</mtext><mtext>within</mtext></msub></math></span> market segment prediction, like STA, CR8, or SHL. However, especially for these traits, the variation among the different market segments was higher (Table <a href="#Tab2" class="usa-link">2</a>), indicated by a positive correlation of the PA of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq122"><math id="d33e3053"><msub><mtext>TS</mtext><mtext>between</mtext></msub></math></span> market segment prediction (Fig. <a href="#Fig4" class="usa-link">4</a>) and the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq123"><math id="d33e3068"><msub><mi>Q</mi><mrow><mi mathvariant="italic">ST</mi></mrow></msub></math></span> value (Table <a href="#Tab2" class="usa-link">2</a>). This is not intuitive, as this means that higher differences between the market segments did overall not decrease the PA but instead increased them. One explanation for this observation is that the genetic differences between the market segments might be due to the influence of some major genes on the phenotype. However, the polygenic background of these traits has to be caused by variation at the same loci in the different market segments. Otherwise it is not possible to explain why the prediction works well also in the presence of high <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq124"><math id="d33e3084"><msub><mi>Q</mi><mrow><mi mathvariant="italic">ST</mi></mrow></msub></math></span> values.</p>
<p id="Par91">The low PA values in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq125"><math id="d33e3099"><msub><mtext>TS</mtext><mtext>between</mtext></msub></math></span> market segment prediction scenario for most traits can most likely not be improved by increasing the marker density as found by Hickey et al. (<a href="#CR29" class="usa-link" aria-describedby="CR29">2014</a>) and Norman et al. (<a href="#CR39" class="usa-link" aria-describedby="CR39">2018</a>). This is because the overall marker number in our study exceeded the marker numbers used in those studies. The low PA values might be explained by genetic differences between TS and VS (Table <a href="#Tab2" class="usa-link">2</a> and Fig. <a href="#Fig4" class="usa-link">4</a>), meaning that different loci are responsible for the phenotypic variation in the different market segments. Also, different linkage disequilibrium blocks in the different market segments can explain the low PA values in the (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq126"><math id="d33e3123"><msub><mtext>TS</mtext><mtext>between</mtext></msub></math></span>) market segment prediction sub-scenario. In summary, this first scenario of the analysis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq127"><math id="d33e3134"><msub><mtext>VS</mtext><mtext>within</mtext></msub></math></span>) showed that for a prediction of clones from one market segment it is best to completely rely on clones from the same market segment, as an addition of more genetically distant clones reduced the median PA.</p>
<p id="Par92">However, one can also imagine that breeders might be interested in predicting the performance of clones of unknown market segments or even predict the assignment to a market segment. We found that this scenario, i.e., prediction of a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq128"><math id="d33e3147"><msub><mtext>VS</mtext><mtext>across</mtext></msub></math></span> market segments, worked best when also the TS was built across market segments (Fig. <a href="#Fig4" class="usa-link">4</a>). This was not surprising, as only in the case of a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq129"><math id="d33e3161"><msub><mtext>TS</mtext><mtext>across</mtext></msub></math></span> the complete genetic diversity of the VS was also covered in the TS and, therefore, the genetic distance was small between TS and VS. However, we also found acceptable PA when the TS was built from clones of individual and especially the CR market segment. Therefore, we recommend breeders to broaden the TS when the goal is to predict across market segments so that a bigger part of the genetic diversity of the VS is covered in the TS, which improves the PA.</p>
<p id="Par93">Collaborating breeding companies may decide to exploit a joint TS. Therefore, we evaluated the PA of breeding company specific TS and between breeding company predictions. Overall, we observed the same tendencies as described above for the market segment analysis (Figure S7). The higher PA in the breeding company analyses might be explained thereby that the clones split by market segments are a more strictly defined genetic group than the clones split by breeding companies. Thus, even though the clones from each breeding company vary genetically from each other, the clones of one breeding company reflect a broader genetic and phenotypic spectrum than clones of one market segment, leading to higher PAs across all sub-scenarios. This explanation is also underlined by the PCA (Figure S3), which showed that clusters per market segment were more clearly defined than clusters per breeding company.</p>
<p id="Par94">In summary, this analysis showed that it is not effective for breeders to combine their breeding material for a joint TS when keeping the TS size constant. However, the addition of clones from other breeding companies, e.g., through a collaboration or through joint databases, is advantageous when this leads to an increased size of the TS (Fig. <a href="#Fig3" class="usa-link">3</a>A).</p></section><section id="Sec10"><h3 class="pmc_sec_title">Effect of selection on the PA</h3>
<p id="Par95">The essence of breeding programs is that genotypes are discarded from the program based on assessed or predicted phenotypic values. This in turn influences which genotypes and/or data are available to serve as TS. Therefore, we assessed the effect of a simulated selection on the PA to provide a guideline for the setup of TS, especially if material from more advanced generations than A clones is used as TS.</p>
<p id="Par96">We observed that a more intense selection led to lower PA values (Fig. <a href="#Fig5" class="usa-link">5</a>A). This illustrates the importance of sufficient genotypic and phenotypic variation for obtaining high PAs. Similar findings have been reported before for maize (Zhao et al. <a href="#CR65" class="usa-link" aria-describedby="CR65">2012</a>; Brandariz &amp; Bernardo <a href="#CR10" class="usa-link" aria-describedby="CR10">2018</a>).</p>
<p id="Par97">In accordance with the results of Zhao et al. (<a href="#CR65" class="usa-link" aria-describedby="CR65">2012</a>), Brandariz &amp; Bernardo (<a href="#CR10" class="usa-link" aria-describedby="CR10">2018</a>), and Michel et al. (<a href="#CR35" class="usa-link" aria-describedby="CR35">2020</a>), we observed that replacing clones of the truncated TS leads to increases in the PA. The inclusion of clones with lower trait values to the TS keeps a minimum amount of phenotypic variance and the resulting higher PA is therefore in accordance with the other findings in our study. However, our observation also suggests that it is sufficient and even preferable to include only clones that were sampled from the fraction with the lowest trait values when the clones in the TS are strictly selected for their high trait values (Fig. <a href="#Fig5" class="usa-link">5</a>B). Our results suggest to include at least 20% of clones with low trait values to strongly selected TS, such as B or C clones, as this was the proportion of exchange that was needed to realize a PA that corresponds to the PA of the unselected TS (Fig. <a href="#Fig5" class="usa-link">5</a>A). Our observation is in accordance with Michel et al. (<a href="#CR35" class="usa-link" aria-describedby="CR35">2020</a>) and Zhao et al. (<a href="#CR65" class="usa-link" aria-describedby="CR65">2012</a>).</p>
<p id="Par98">These results based on individual trait selections are abstract, as potato breeders select for many traits simultaneously and, thus, the chance of only getting high-performing clones in the TS is low. This might lead to a TS, where also some clones with lower trait values were kept, as these clones performed well for the other traits of interest. We evaluated this aspect by a small example, using a TS comprising clones in the top 50%. But instead of sampling from the 50% highest trait values of one trait, we selected the top 50% based on the sum of the ranks across four example traits (STA, FLE, YLD, and EMR). Using the rank criterion instead of the single trait value as selection criterion increased the PA noticeably (Figure S9). This supports our explanation that the clones for each trait are more diverse when selecting for the overall rank. This difference between the PA of a selection based on the rank criterion <em>vs.</em> single trait values gets smaller when exchanging 20% of the TS with clones with lower trait values or ranks, as then both sets were more diverse. Nevertheless, also this result supports our above recommendation to include at least 20% of clones with low trait values to strongly selected TS, such as B or C clones.</p>
<p id="Par99">The above-presented behavior of the PA can also be seen when choosing a trait combination relevant for breeding (Figure S11). We tested in a sub-analysis the trait combination STA 17% to 19%, CR8 high values, SHL low values, EYE high values, and BRU low values. However, we observed for the combination of these traits the same tendencies as described before for the combination of traits selected by the highest trait values. Therefore, we can conclude that the criteria for selection are not the important factor, but rather the fact that selection occurred in general, i.e., it is not important if selection is based on the highest trait values, lowest trait values, or based on a specific range. As long as breeding companies want to build a TS based on selected clones, it is preferable to add clones with non-preferable trait values to the TS to increase variability and thereby increase the PA.</p>
<p id="Par100">Because of this considerable effect of selection on PA described above, we evaluated in the specific potato example if the performance of TS made up from A clones can be improved by including clones that would have been discarded in the single hill stage of commercial breeding programs. Across the examined traits, we found no increased PA when clones with discard status 1 were added to the TS (Fig. <a href="#Fig6" class="usa-link">6</a>). This is in accordance with the observation that the gene diversity in the TS decreased when clones that would have been discarded in a breeding program were part of the TS. This finding can be explained by the lack of correlation between the traits assessed in the step prior to the A clone, i.e., the single hill stage, and the traits assessed in the later stages. This explanation is in agreement with the results of Thelen et al. (<a href="#CR53" class="usa-link" aria-describedby="CR53">2025</a>), who observed no mean differences between the genotypic variances of the sets including and excluding clones with discard status. We furthermore checked if the combination of different market segments in this analysis influenced the non-changing PA, as the different market segments were selected based on different thresholds and relative weights of the different traits. However, the same trend was observed for the analyses across market segments and within a market segment. This observation suggests that random sets of A clones are appropriate as TS and show enough genotypic and phenotypic diversity, and it is not necessary to include clones with a discard status from the single hills stage into the TS to increase diversity and correspondingly PA.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Conclusions from a breeding point of view</h3>
<p id="Par101">Our analyses revealed overall high PAs, suggesting the relevance of GP for potato breeding. We found no improvement in the PA using the additive-dominance GBLUP compared to the additive GBLUP for all traits except one, suggesting the possibility to generate fast progress in many traits. Our results indicated that TS of about 400 diverse clones is sufficient to result in close to the maximal PA when considering segregating populations. Similarly, about 10,000 markers are sufficient to result in almost maximal PA for such material. We found that an addition of clones from other market segments to predict the clones of one market segment did not improve the PA compared to prediction within one market segment alone. A prediction between different market segments was in our study only meaningful for traits that showed a high PA in the within market segment prediction. Furthermore, we found that it is sufficient if breeding companies use their own genetic material to predict the breeding values of untested clones. Further, it is not necessary to include in the TS clones from the single hills stage that would normally have been discarded. Our analyses suggested that if genetic material that underwent strong phenotypic selection, such as B, C, or D clones, is used as TS, about 20% of clones with the opposite values than the selected clones should be included, to recover the original prediction accuracy of an unselected TS.</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Key message</h2>
<p id="Par103">Genomic prediction in potato works best within market segments. Adding 20% low-trait clones to a high-trait-selected training set restores accuracy to that of an phenotypically unselected set.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12367938/bin/122_2025_5004_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 1 (pdf 1126 KB)</a><sup> (1.1MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Computational infrastructure and support were provided by the Department of Digitization and Artificial Intelligence of Julius Kühn-Institute and by the German Network for Bioinformatic Infrastructure (de.NBI). We highly appreciate that Anja Kohlmorgen assisted in the assessment of the phenotypic data for SaKa Pflanzenzucht GmbH &amp; Co. KG. We also want to thank Solana Research GmbH for providing the icons that are used in Fig. <a href="#Fig1" class="usa-link">1</a>.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>Kathrin Thelen was involved in conceptualization, data analysis, and writing. Po-Ya Wu was involved in data analysis, writing, and review. Nadia Baig helped in genotyping and generation of the SNP array data and review. Vanessa Prigge, Julien Bruckmüller, Katja Muders, Bernd Truberg, Stefanie Hartje, and Juliane Renner contributed to funding acquisition, resources, collection of experimental data, and review. Delphine Van Inghelandt was involved in conceptualization, project design, project coordination, and writing. Benjamin Stich was involved in conceptualization, funding acquisition, project design, project coordination, and writing.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open Access funding enabled and organized by Projekt DEAL. This study was funded by the German Federal Ministry of Food and Agriculture/ Fachagentur Nachwachsende Rohstoffe e.V. (FNR) (grant ID 22011818, PotatoTools and 2222NR078A, PotatoPredict). The funders did not influence the study design, the collection, analysis and interpretation of data, the writing of the manuscript, and the decision to submit the manuscript for publication.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data Availability</h2>
<p>The original data sets generated and/or analyzed in the current study are not publicly available due to the material being part of the company secrets of SaKa Pflanzenzucht GmbH &amp; Co. KG, Nordring-Kartoffelzucht- und Vermehrungs- GmbH &amp; Co. KG, and EUROPLANT Innovation GmbH &amp; Co. KG. However, the data are available in encoded form from the corresponding author upon reasonable request.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Conflict of interest</h3>
<p id="Par108">Vanessa Prigge and Julien Bruckmüller are employees of SaKa Pflanzenzucht GmbH &amp; Co. KG. Katja Muders and Bernd Truberg are employees of Nordring-Kartoffelzucht- und Vermehrungs- GmbH &amp; Co. KG. Stefanie Hartje, Juliane Renner, and Nadia Baig are employees of EUROPLANT Innovation GmbH &amp; Co. KG. The authors declare no other conflict of interest.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn2">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ol class="ref-list font-sm">
<li id="CR1">
<cite>Aalborg T, Sverrisdóttir E, Kristensen HT, Nielsen KL (2024) The effect of marker types and density on genomic prediction and GWAS of key performance traits in tetraploid potato. Front Plant Sci 15:1340189
</cite> [<a href="https://doi.org/10.3389/fpls.2024.1340189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10957621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38525152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aalborg%20T,%20Sverrisd%C3%B3ttir%20E,%20Kristensen%20HT,%20Nielsen%20KL%20(2024)%20The%20effect%20of%20marker%20types%20and%20density%20on%20genomic%20prediction%20and%20GWAS%20of%20key%20performance%20traits%20in%20tetraploid%20potato.%20Front%20Plant%20Sci%2015:1340189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<cite>Akdemir D, Sanchez JI, Jannink J-L (2015) Optimization of genomic selection training populations with a genetic algorithm. Genet Sel Evol 47(1):38
</cite> [<a href="https://doi.org/10.1186/s12711-015-0116-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4422310/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25943105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akdemir%20D,%20Sanchez%20JI,%20Jannink%20J-L%20(2015)%20Optimization%20of%20genomic%20selection%20training%20populations%20with%20a%20genetic%20algorithm.%20Genet%20Sel%20Evol%2047(1):38" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<cite>Akohoue F, Achigan-Dako EG, Sneller C, Van Deynze A, Sibiya J (2020) Genetic diversity, SNP-trait associations and genomic selection accuracy in a west African collection of Kersting’s groundnut [Macrotyloma geocarpum(Harms) Maréchal &amp; Baudet]. PLoS ONE 15(6):e0234769
</cite> [<a href="https://doi.org/10.1371/journal.pone.0234769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7326195/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32603370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akohoue%20F,%20Achigan-Dako%20EG,%20Sneller%20C,%20Van%20Deynze%20A,%20Sibiya%20J%20(2020)%20Genetic%20diversity,%20SNP-trait%20associations%20and%20genomic%20selection%20accuracy%20in%20a%20west%20African%20collection%20of%20Kersting%E2%80%99s%20groundnut%20%5BMacrotyloma%20geocarpum(Harms)%20Mar%C3%A9chal%20&amp;%20Baudet%5D.%20PLoS%20ONE%2015(6):e0234769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<cite>Alemu A, Åstrand J, Montesinos-López OA, Sánchez IYJ, Fernández-Gónzalez J, Tadesse W, Vetukuri RR, Carlsson AS, Ceplitis A, Crossa J, Ortiz R, Chawade A (2024) Genomic selection in plant breeding: key factors shaping two decades of progress. Mol Plant 17(4):552–578
</cite> [<a href="https://doi.org/10.1016/j.molp.2024.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38475993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alemu%20A,%20%C3%85strand%20J,%20Montesinos-L%C3%B3pez%20OA,%20S%C3%A1nchez%20IYJ,%20Fern%C3%A1ndez-G%C3%B3nzalez%20J,%20Tadesse%20W,%20Vetukuri%20RR,%20Carlsson%20AS,%20Ceplitis%20A,%20Crossa%20J,%20Ortiz%20R,%20Chawade%20A%20(2024)%20Genomic%20selection%20in%20plant%20breeding:%20key%20factors%20shaping%20two%20decades%20of%20progress.%20Mol%20Plant%2017(4):552%E2%80%93578" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<cite>Amadeu RR, Ferrão LFV, Oliveira IDB, Benevenuto J, Endelman JB, Munoz PR (2020) Impact of dominance effects on autotetraploid genomic prediction. Crop Sci 60(2):656–665</cite> [<a href="https://scholar.google.com/scholar_lookup?Amadeu%20RR,%20Ferr%C3%A3o%20LFV,%20Oliveira%20IDB,%20Benevenuto%20J,%20Endelman%20JB,%20Munoz%20PR%20(2020)%20Impact%20of%20dominance%20effects%20on%20autotetraploid%20genomic%20prediction.%20Crop%20Sci%2060(2):656%E2%80%93665" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<cite>Amadeu RR, Garcia AAF, Munoz PR, Ferrão LFV (2023) AGHmatrix: genetic relationship matrices in R. Bioinformatics 39(7):btad445
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btad445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10371492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37471595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amadeu%20RR,%20Garcia%20AAF,%20Munoz%20PR,%20Ferr%C3%A3o%20LFV%20(2023)%20AGHmatrix:%20genetic%20relationship%20matrices%20in%20R.%20Bioinformatics%2039(7):btad445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<cite>Ashraf BH, Byrne S, Fé D, Czaban A, Asp T, Pedersen MG, Lenk I, Roulund N, Didion T, Jensen CS, Jensen J, Janss LL (2016) Estimating genomic heritabilities at the level of family-pool samples of perennial ryegrass using genotyping-by-sequencing. Theor Appl Genet 129(1):45–52
</cite> [<a href="https://doi.org/10.1007/s00122-015-2607-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4703624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26407618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ashraf%20BH,%20Byrne%20S,%20F%C3%A9%20D,%20Czaban%20A,%20Asp%20T,%20Pedersen%20MG,%20Lenk%20I,%20Roulund%20N,%20Didion%20T,%20Jensen%20CS,%20Jensen%20J,%20Janss%20LL%20(2016)%20Estimating%20genomic%20heritabilities%20at%20the%20level%20of%20family-pool%20samples%20of%20perennial%20ryegrass%20using%20genotyping-by-sequencing.%20Theor%20Appl%20Genet%20129(1):45%E2%80%9352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<cite>Azevedo CF, Ferrão LFV, Benevenuto J, De Resende MDV, Nascimento M, Nascimento ACC, Munoz PR (2024) Using visual scores for genomic prediction of complex traits in breeding programs. Theor Appl Genet 137(1):9</cite> [<a href="https://doi.org/10.1007/s00122-023-04512-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38102495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Azevedo%20CF,%20Ferr%C3%A3o%20LFV,%20Benevenuto%20J,%20De%20Resende%20MDV,%20Nascimento%20M,%20Nascimento%20ACC,%20Munoz%20PR%20(2024)%20Using%20visual%20scores%20for%20genomic%20prediction%20of%20complex%20traits%20in%20breeding%20programs.%20Theor%20Appl%20Genet%20137(1):9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<cite>Bradshaw JE (2017) Review and analysis of limitations in ways to improve conventional potato breeding. Potato Res 60(2):171–193</cite> [<a href="https://scholar.google.com/scholar_lookup?Bradshaw%20JE%20(2017)%20Review%20and%20analysis%20of%20limitations%20in%20ways%20to%20improve%20conventional%20potato%20breeding.%20Potato%20Res%2060(2):171%E2%80%93193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<cite>Brandariz SP, Bernardo R (2018) Maintaining the accuracy of genomewide predictions when selection has occurred in the training population. Crop Sci 58(3):1226–1231</cite> [<a href="https://scholar.google.com/scholar_lookup?Brandariz%20SP,%20Bernardo%20R%20(2018)%20Maintaining%20the%20accuracy%20of%20genomewide%20predictions%20when%20selection%20has%20occurred%20in%20the%20training%20population.%20Crop%20Sci%2058(3):1226%E2%80%931231" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<cite>Brauner PC, Müller D, Molenaar WS, Melchinger AE (2020) Genomic prediction with multiple biparental families. Theor Appl Genet 133(1):133–147
</cite> [<a href="https://doi.org/10.1007/s00122-019-03445-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31595337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brauner%20PC,%20M%C3%BCller%20D,%20Molenaar%20WS,%20Melchinger%20AE%20(2020)%20Genomic%20prediction%20with%20multiple%20biparental%20families.%20Theor%20Appl%20Genet%20133(1):133%E2%80%93147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<cite>Browning BL, Zhou Y, Browning SR (2018) A one-penny imputed genome from next-generation reference panels. Am J Hum Genet 103(3):338–348
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2018.07.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6128308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30100085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Browning%20BL,%20Zhou%20Y,%20Browning%20SR%20(2018)%20A%20one-penny%20imputed%20genome%20from%20next-generation%20reference%20panels.%20Am%20J%20Hum%20Genet%20103(3):338%E2%80%93348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13"><cite>Bundessortenamt (2019) Wertprüfungsrichtlinie Kartoffel, Überarbeitete Richtlinie Kartoffel</cite></li>
<li id="CR14">
<cite>Byrne S, Meade F, Mesiti F, Griffin D, Kennedy C, Milbourne D (2020) Genome-wide association and genomic prediction for fry color in potato. Agronomy 10(1):90</cite> [<a href="https://scholar.google.com/scholar_lookup?Byrne%20S,%20Meade%20F,%20Mesiti%20F,%20Griffin%20D,%20Kennedy%20C,%20Milbourne%20D%20(2020)%20Genome-wide%20association%20and%20genomic%20prediction%20for%20fry%20color%20in%20potato.%20Agronomy%2010(1):90" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<cite>Covarrubias-Pazaran G (2016) Genome-assisted prediction of quantitative traits using the R package sommer. PLoS ONE 11(6):e0156744
</cite> [<a href="https://doi.org/10.1371/journal.pone.0156744" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4894563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27271781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Covarrubias-Pazaran%20G%20(2016)%20Genome-assisted%20prediction%20of%20quantitative%20traits%20using%20the%20R%20package%20sommer.%20PLoS%20ONE%2011(6):e0156744" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<cite>Crossa J, Pérez-Rodríguez P, Cuevas J, Montesinos-López O, Jarquín D, De Los Campos G, Burgueño J, González-Camacho JM, Pérez-Elizalde S, Beyene Y, Dreisigacker S, Singh R, Zhang X, Gowda M, Roorkiwal M, Rutkoski J, Varshney RK (2017) Genomic selection in plant breeding: methods, models, and perspectives. Trends Plant Sci 22(11):961–975
</cite> [<a href="https://doi.org/10.1016/j.tplants.2017.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28965742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crossa%20J,%20P%C3%A9rez-Rodr%C3%ADguez%20P,%20Cuevas%20J,%20Montesinos-L%C3%B3pez%20O,%20Jarqu%C3%ADn%20D,%20De%20Los%20Campos%20G,%20Burgue%C3%B1o%20J,%20Gonz%C3%A1lez-Camacho%20JM,%20P%C3%A9rez-Elizalde%20S,%20Beyene%20Y,%20Dreisigacker%20S,%20Singh%20R,%20Zhang%20X,%20Gowda%20M,%20Roorkiwal%20M,%20Rutkoski%20J,%20Varshney%20RK%20(2017)%20Genomic%20selection%20in%20plant%20breeding:%20methods,%20models,%20and%20perspectives.%20Trends%20Plant%20Sci%2022(11):961%E2%80%93975" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<cite>De Bem Oliveira I, Amadeu RR, Ferrão LFV, Muñoz PR (2020) Optimizing whole-genomic prediction for autotetraploid blueberry breeding. Heredity 125(6):437–448
</cite> [<a href="https://doi.org/10.1038/s41437-020-00357-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7784927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33077896/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Bem%20Oliveira%20I,%20Amadeu%20RR,%20Ferr%C3%A3o%20LFV,%20Mu%C3%B1oz%20PR%20(2020)%20Optimizing%20whole-genomic%20prediction%20for%20autotetraploid%20blueberry%20breeding.%20Heredity%20125(6):437%E2%80%93448" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<cite>Devaux A, Goffart J-P, Petsakos A, Kromann P, Gatto M, Okello J, Suarez V, Hareau G (2020) Global food security, contributions from sustainable potato agri-food systems. In: Campos H, Ortiz O (eds) The potato crop. Springer, Cham, pp 3–35</cite> [<a href="https://scholar.google.com/scholar_lookup?Devaux%20A,%20Goffart%20J-P,%20Petsakos%20A,%20Kromann%20P,%20Gatto%20M,%20Okello%20J,%20Suarez%20V,%20Hareau%20G%20(2020)%20Global%20food%20security,%20contributions%20from%20sustainable%20potato%20agri-food%20systems.%20In:%20Campos%20H,%20Ortiz%20O%20(eds)%20The%20potato%20crop.%20Springer,%20Cham,%20pp%203%E2%80%9335" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19"><cite>Dray S, Dufour, A-B (2007) The ade4 package: implementing the duality diagram for ecologists. J Stat Softw 22(4)</cite></li>
<li id="CR20">
<cite>Enciso-Rodriguez F, Douches D, Lopez-Cruz M, Coombs J, De Los Campos G (2018) Genomic Selection for Late Blight and Common Scab Resistance in Tetraploid Potato (Solanum tuberosum ). G3 Genes Genomes Genet 8(7):2471–2481</cite> [<a href="https://doi.org/10.1534/g3.118.200273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6027896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29794167/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Enciso-Rodriguez%20F,%20Douches%20D,%20Lopez-Cruz%20M,%20Coombs%20J,%20De%20Los%20Campos%20G%20(2018)%20Genomic%20Selection%20for%20Late%20Blight%20and%20Common%20Scab%20Resistance%20in%20Tetraploid%20Potato%20(Solanum%20tuberosum%20).%20G3%20Genes%20Genomes%20Genet%208(7):2471%E2%80%932481" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<cite>Endelman JB, Carley CAS, Bethke PC, Coombs JJ, Clough ME, Da Silva WL, De Jong WS, Douches DS, Frederick CM, Haynes KG, Holm DG, Miller JC, Muñoz PR, Navarro FM, Novy RG, Palta JP, Porter GA, Rak KT, Sathuvalli VR, Thompson AL, Yencho GC (2018) Genetic variance partitioning and genome-wide prediction with allele dosage information in autotetraploid potato. Genetics 209(1):77–87
</cite> [<a href="https://doi.org/10.1534/genetics.118.300685" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5937173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29514860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Endelman%20JB,%20Carley%20CAS,%20Bethke%20PC,%20Coombs%20JJ,%20Clough%20ME,%20Da%20Silva%20WL,%20De%20Jong%20WS,%20Douches%20DS,%20Frederick%20CM,%20Haynes%20KG,%20Holm%20DG,%20Miller%20JC,%20Mu%C3%B1oz%20PR,%20Navarro%20FM,%20Novy%20RG,%20Palta%20JP,%20Porter%20GA,%20Rak%20KT,%20Sathuvalli%20VR,%20Thompson%20AL,%20Yencho%20GC%20(2018)%20Genetic%20variance%20partitioning%20and%20genome-wide%20prediction%20with%20allele%20dosage%20information%20in%20autotetraploid%20potato.%20Genetics%20209(1):77%E2%80%9387" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22"><cite>FAOSTAT database (2024)</cite></li>
<li id="CR23">
<cite>Gallais A (2003) Quantitative genetics and breeding methods in autopolyploid plants. Mieux comprendre, Institut national de la recherche agronomique, Paris</cite> [<a href="https://scholar.google.com/scholar_lookup?Gallais%20A%20(2003)%20Quantitative%20genetics%20and%20breeding%20methods%20in%20autopolyploid%20plants.%20Mieux%20comprendre,%20Institut%20national%20de%20la%20recherche%20agronomique,%20Paris" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<cite>Gebhardt C (2013) Bridging the gap between genome analysis and precision breeding in potato. Trends Genet 29(4):248–256
</cite> [<a href="https://doi.org/10.1016/j.tig.2012.11.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23261028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gebhardt%20C%20(2013)%20Bridging%20the%20gap%20between%20genome%20analysis%20and%20precision%20breeding%20in%20potato.%20Trends%20Genet%2029(4):248%E2%80%93256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<cite>Gerard D (2021) Pairwise linkage disequilibrium estimation for polyploids. Mol Ecol Resour 21(4):1230–1242
</cite> [<a href="https://doi.org/10.1111/1755-0998.13349" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33559321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gerard%20D%20(2021)%20Pairwise%20linkage%20disequilibrium%20estimation%20for%20polyploids.%20Mol%20Ecol%20Resour%2021(4):1230%E2%80%931242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<cite>Guo X, Cericola F, Fè D, Pedersen MG, Lenk I, Jensen CS, Jensen J, Janss LL (2018) Genomic prediction in tetraploid ryegrass using allele frequencies based on genotyping by sequencing. Front Plant Sci 9:1165
</cite> [<a href="https://doi.org/10.3389/fpls.2018.01165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6104567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30158944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20X,%20Cericola%20F,%20F%C3%A8%20D,%20Pedersen%20MG,%20Lenk%20I,%20Jensen%20CS,%20Jensen%20J,%20Janss%20LL%20(2018)%20Genomic%20prediction%20in%20tetraploid%20ryegrass%20using%20allele%20frequencies%20based%20on%20genotyping%20by%20sequencing.%20Front%20Plant%20Sci%209:1165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<cite>Guo Z, Tucker DM, Basten CJ, Gandhi H, Ersoz E, Guo B, Xu Z, Wang D, Gay G (2014) The impact of population structure on genomic prediction in stratified populations. Theor Appl Genet 127(3):749–762
</cite> [<a href="https://doi.org/10.1007/s00122-013-2255-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24452438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20Z,%20Tucker%20DM,%20Basten%20CJ,%20Gandhi%20H,%20Ersoz%20E,%20Guo%20B,%20Xu%20Z,%20Wang%20D,%20Gay%20G%20(2014)%20The%20impact%20of%20population%20structure%20on%20genomic%20prediction%20in%20stratified%20populations.%20Theor%20Appl%20Genet%20127(3):749%E2%80%93762" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<cite>Habyarimana E, Parisi B, Mandolino G (2017) Genomic prediction for yields, processing and nutritional quality traits in cultivated potato ( Solanum tuberosum L.). Plant Breed 136(2):245–252</cite> [<a href="https://scholar.google.com/scholar_lookup?Habyarimana%20E,%20Parisi%20B,%20Mandolino%20G%20(2017)%20Genomic%20prediction%20for%20yields,%20processing%20and%20nutritional%20quality%20traits%20in%20cultivated%20potato%20(%20Solanum%20tuberosum%20L.).%20Plant%20Breed%20136(2):245%E2%80%93252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<cite>Hickey JM, Dreisigacker S, Crossa J, Hearne S, Babu R, Prasanna BM, Grondona M, Zambelli A, Windhausen VS, Mathews K, Gorjanc G (2014) Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation. Crop Sci 54(4):1476–1488</cite> [<a href="https://scholar.google.com/scholar_lookup?Hickey%20JM,%20Dreisigacker%20S,%20Crossa%20J,%20Hearne%20S,%20Babu%20R,%20Prasanna%20BM,%20Grondona%20M,%20Zambelli%20A,%20Windhausen%20VS,%20Mathews%20K,%20Gorjanc%20G%20(2014)%20Evaluation%20of%20genomic%20selection%20training%20population%20designs%20and%20genotyping%20strategies%20in%20plant%20breeding%20programs%20using%20simulation.%20Crop%20Sci%2054(4):1476%E2%80%931488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<cite>Isidro J, Jannink J-L, Akdemir D, Poland J, Heslot N, Sorrells ME (2015) Training set optimization under population structure in genomic selection. Theor Appl Genet 128(1):145–158
</cite> [<a href="https://doi.org/10.1007/s00122-014-2418-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4282691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25367380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Isidro%20J,%20Jannink%20J-L,%20Akdemir%20D,%20Poland%20J,%20Heslot%20N,%20Sorrells%20ME%20(2015)%20Training%20set%20optimization%20under%20population%20structure%20in%20genomic%20selection.%20Theor%20Appl%20Genet%20128(1):145%E2%80%93158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31"><cite>Jansky S (2009) Breeding, genetics, and cultivar development. Advances in Potato Chemistry and Technology. Elsevier. pp 27–62</cite></li>
<li id="CR32">
<cite>Li X, Wei Y, Acharya A, Hansen, JL, Crawford JL, Viands DR, Michaud R, Claessens A, Brummer EC (2015) Genomic Prediction of Biomass Yield in Two Selection Cycles of a Tetraploid Alfalfa Breeding Population. Plant Genome 8(2), plantgenome2014.12.0090</cite> [<a href="https://doi.org/10.3835/plantgenome2014.12.0090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33228301/" class="usa-link">PubMed</a>]</li>
<li id="CR33">
<cite>Lorenz AJ, Smith KP (2015) Adding genetically distant individuals to training populations reduces genomic prediction accuracy in barley. Crop Sci 55(6):2657–2667</cite> [<a href="https://scholar.google.com/scholar_lookup?Lorenz%20AJ,%20Smith%20KP%20(2015)%20Adding%20genetically%20distant%20individuals%20to%20training%20populations%20reduces%20genomic%20prediction%20accuracy%20in%20barley.%20Crop%20Sci%2055(6):2657%E2%80%932667" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<cite>Meuwissen THE, Hayes BJ, Goddard ME (2001) Prediction of total genetic value using genome-wide dense marker maps. Genetics 157(4):1819–1829
</cite> [<a href="https://doi.org/10.1093/genetics/157.4.1819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11290733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meuwissen%20THE,%20Hayes%20BJ,%20Goddard%20ME%20(2001)%20Prediction%20of%20total%20genetic%20value%20using%20genome-wide%20dense%20marker%20maps.%20Genetics%20157(4):1819%E2%80%931829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<cite>Michel S, Löschenberger F, Sparry E, Ametz C, Bürstmayr H (2020) Mitigating the impact of selective phenotyping in training populations on the prediction ability by multi-trait pedigree and genomic selection models. Plant Breed 139(6):1067–1075</cite> [<a href="https://scholar.google.com/scholar_lookup?Michel%20S,%20L%C3%B6schenberger%20F,%20Sparry%20E,%20Ametz%20C,%20B%C3%BCrstmayr%20H%20(2020)%20Mitigating%20the%20impact%20of%20selective%20phenotyping%20in%20training%20populations%20on%20the%20prediction%20ability%20by%20multi-trait%20pedigree%20and%20genomic%20selection%20models.%20Plant%20Breed%20139(6):1067%E2%80%931075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<cite>Naeem M, Demirel U, Yousaf MF, Caliskan S, Caliskan ME (2021) Overview on domestication, breeding, genetic gain and improvement of tuber quality traits of potato using fast forwarding technique (GWAS): A review. Plant Breed 140(4):519–542</cite> [<a href="https://scholar.google.com/scholar_lookup?Naeem%20M,%20Demirel%20U,%20Yousaf%20MF,%20Caliskan%20S,%20Caliskan%20ME%20(2021)%20Overview%20on%20domestication,%20breeding,%20genetic%20gain%20and%20improvement%20of%20tuber%20quality%20traits%20of%20potato%20using%20fast%20forwarding%20technique%20(GWAS):%20A%20review.%20Plant%20Breed%20140(4):519%E2%80%93542" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<cite>Nei M (1986) Definition and estimation of fixation indices. Evolution 40(3):643–645
</cite> [<a href="https://doi.org/10.1111/j.1558-5646.1986.tb00516.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28556335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nei%20M%20(1986)%20Definition%20and%20estimation%20of%20fixation%20indices.%20Evolution%2040(3):643%E2%80%93645" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<cite>Nei M, Roychoudhury AK (1974) Sampling variances of heterozygosity and genetic distance. Genetics 76(2):379–390
</cite> [<a href="https://doi.org/10.1093/genetics/76.2.379" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1213072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4822472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nei%20M,%20Roychoudhury%20AK%20(1974)%20Sampling%20variances%20of%20heterozygosity%20and%20genetic%20distance.%20Genetics%2076(2):379%E2%80%93390" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<cite>Norman A, Taylor J, Edwards J, Kuchel H (2018) Optimising genomic selection in wheat: effect of marker density, population size and population structure on prediction accuracy. G3 Genes Genomes Genet 8(9):2889–2899</cite> [<a href="https://doi.org/10.1534/g3.118.200311" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6118301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29970398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Norman%20A,%20Taylor%20J,%20Edwards%20J,%20Kuchel%20H%20(2018)%20Optimising%20genomic%20selection%20in%20wheat:%20effect%20of%20marker%20density,%20population%20size%20and%20population%20structure%20on%20prediction%20accuracy.%20G3%20Genes%20Genomes%20Genet%208(9):2889%E2%80%932899" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<cite>Onogi A, Ideta O, Inoshita Y, Ebana K, Yoshioka T, Yamasaki M, Iwata H (2015) Exploring the areas of applicability of whole-genome prediction methods for Asian rice (Oryza sativa L.). Theor Appl Genet 128(1):41–53
</cite> [<a href="https://doi.org/10.1007/s00122-014-2411-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25341369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Onogi%20A,%20Ideta%20O,%20Inoshita%20Y,%20Ebana%20K,%20Yoshioka%20T,%20Yamasaki%20M,%20Iwata%20H%20(2015)%20Exploring%20the%20areas%20of%20applicability%20of%20whole-genome%20prediction%20methods%20for%20Asian%20rice%20(Oryza%20sativa%20L.).%20Theor%20Appl%20Genet%20128(1):41%E2%80%9353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<cite>Pembleton LW, Cogan NOI, Forster JW (2013) StAMPP: an R package for calculation of genetic differentiation and structure of mixed-ploidy level populations. Mol Ecol Resour 13(5):946–952
</cite> [<a href="https://doi.org/10.1111/1755-0998.12129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23738873/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pembleton%20LW,%20Cogan%20NOI,%20Forster%20JW%20(2013)%20StAMPP:%20an%20R%20package%20for%20calculation%20of%20genetic%20differentiation%20and%20structure%20of%20mixed-ploidy%20level%20populations.%20Mol%20Ecol%20Resour%2013(5):946%E2%80%93952" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<cite>Piepho H-P, Möhring J (2007) Computing heritability and selection response from unbalanced plant breeding trials. Genetics 177(3):1881–1888
</cite> [<a href="https://doi.org/10.1534/genetics.107.074229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2147938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18039886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Piepho%20H-P,%20M%C3%B6hring%20J%20(2007)%20Computing%20heritability%20and%20selection%20response%20from%20unbalanced%20plant%20breeding%20trials.%20Genetics%20177(3):1881%E2%80%931888" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<cite>R Core Team (2022) R: A language and environment for statistical computing. Austria, Vienna</cite> [<a href="https://scholar.google.com/scholar_lookup?R%20Core%20Team%20(2022)%20R:%20A%20language%20and%20environment%20for%20statistical%20computing.%20Austria,%20Vienna" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<cite>Reddy BJ, Mandal R, Chakroborty M, Hijam L, Dutta P (2018) A review on potato (Solanum Tuberosum L.) and its genetic diversity. Int J Genet 10(2):360</cite> [<a href="https://scholar.google.com/scholar_lookup?Reddy%20BJ,%20Mandal%20R,%20Chakroborty%20M,%20Hijam%20L,%20Dutta%20P%20(2018)%20A%20review%20on%20potato%20(Solanum%20Tuberosum%20L.)%20and%20its%20genetic%20diversity.%20Int%20J%20Genet%2010(2):360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<cite>Rio S, Gallego-Sánchez L, Montilla-Bascón G, Canales FJ, Sánchez IY (2021) Genomic prediction and training set optimization in a structured Mediterranean oat population. Theor Appl Genet 134(11):3595–3609
</cite> [<a href="https://doi.org/10.1007/s00122-021-03916-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34341832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rio%20S,%20Gallego-S%C3%A1nchez%20L,%20Montilla-Basc%C3%B3n%20G,%20Canales%20FJ,%20S%C3%A1nchez%20IY%20(2021)%20Genomic%20prediction%20and%20training%20set%20optimization%20in%20a%20structured%20Mediterranean%20oat%20population.%20Theor%20Appl%20Genet%20134(11):3595%E2%80%933609" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<cite>Selga C, Reslow F, Pérez-Rodríguez P, Ortiz R (2022) The power of genomic estimated breeding values for selection when using a finite population size in genetic improvement of tetraploid potato. G3 Genes Genomes Genet 12(1):jkab362</cite> [<a href="https://doi.org/10.1093/g3journal/jkab362" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728039/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34849763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Selga%20C,%20Reslow%20F,%20P%C3%A9rez-Rodr%C3%ADguez%20P,%20Ortiz%20R%20(2022)%20The%20power%20of%20genomic%20estimated%20breeding%20values%20for%20selection%20when%20using%20a%20finite%20population%20size%20in%20genetic%20improvement%20of%20tetraploid%20potato.%20G3%20Genes%20Genomes%20Genet%2012(1):jkab362" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<cite>Slater AT, Cogan NO, Forster JW, Hayes BJ, Daetwyler HD (2016) Improving genetic gain with genomic selection in autotetraploid potato. Plant Genome 9(3):plantgenome2016.02.0021</cite> [<a href="https://doi.org/10.3835/plantgenome2016.02.0021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27902807/" class="usa-link">PubMed</a>]</li>
<li id="CR48">
<cite>Slater AT, Cogan NOI, Forster JW (2013) Cost analysis of the application of marker-assisted selection in potato breeding. Mol Breed 32(2):299–310</cite> [<a href="https://scholar.google.com/scholar_lookup?Slater%20AT,%20Cogan%20NOI,%20Forster%20JW%20(2013)%20Cost%20analysis%20of%20the%20application%20of%20marker-assisted%20selection%20in%20potato%20breeding.%20Mol%20Breed%2032(2):299%E2%80%93310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<cite>Slatkin M (2008) Linkage disequilibrium–understanding the evolutionary past and mapping the medical future. Nat Rev Genet 9(6):477–485
</cite> [<a href="https://doi.org/10.1038/nrg2361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5124487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18427557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Slatkin%20M%20(2008)%20Linkage%20disequilibrium%E2%80%93understanding%20the%20evolutionary%20past%20and%20mapping%20the%20medical%20future.%20Nat%20Rev%20Genet%209(6):477%E2%80%93485" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<cite>Spitze K (1993) Population structure in Daphnia obtusa: quantitative genetic and allozymic variation. Genetics 135(2):367–374
</cite> [<a href="https://doi.org/10.1093/genetics/135.2.367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1205642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8244001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spitze%20K%20(1993)%20Population%20structure%20in%20Daphnia%20obtusa:%20quantitative%20genetic%20and%20allozymic%20variation.%20Genetics%20135(2):367%E2%80%93374" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<cite>Stich B, Van Inghelandt D (2018) Prospects and potential uses of genomic prediction of key performance traits in tetraploid potato. Front Plant Sci 9:159
</cite> [<a href="https://doi.org/10.3389/fpls.2018.00159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5845909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29563919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stich%20B,%20Van%20Inghelandt%20D%20(2018)%20Prospects%20and%20potential%20uses%20of%20genomic%20prediction%20of%20key%20performance%20traits%20in%20tetraploid%20potato.%20Front%20Plant%20Sci%209:159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<cite>Sverrisdóttir E, Sundmark EHR, Johnsen HØ, Kirk HG, Asp T, Janss L, Bryan G, Nielsen KL (2018) The value of expanding the training population to improve genomic selection models in tetraploid potato. Front Plant Sci 9:1118
</cite> [<a href="https://doi.org/10.3389/fpls.2018.01118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6090097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30131817/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sverrisd%C3%B3ttir%20E,%20Sundmark%20EHR,%20Johnsen%20H%C3%98,%20Kirk%20HG,%20Asp%20T,%20Janss%20L,%20Bryan%20G,%20Nielsen%20KL%20(2018)%20The%20value%20of%20expanding%20the%20training%20population%20to%20improve%20genomic%20selection%20models%20in%20tetraploid%20potato.%20Front%20Plant%20Sci%209:1118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<cite>Thelen K, Prigge V, Kohlmorgen A, Muders K, Truberg B, Hartje S, Renner J, Stich B (2025) Variance and covariance components of agronomic and quality traits assessed in tetraploid potato and their implications on practical breeding. Front Plant Sci 15:1505193
</cite> [<a href="https://doi.org/10.3389/fpls.2024.1505193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11780675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39886678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thelen%20K,%20Prigge%20V,%20Kohlmorgen%20A,%20Muders%20K,%20Truberg%20B,%20Hartje%20S,%20Renner%20J,%20Stich%20B%20(2025)%20Variance%20and%20covariance%20components%20of%20agronomic%20and%20quality%20traits%20assessed%20in%20tetraploid%20potato%20and%20their%20implications%20on%20practical%20breeding.%20Front%20Plant%20Sci%2015:1505193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54"><cite>Tiemens-Hulscher M, Delleman J, Eising J, Lammerts van Bueren ET (eds) (2013) Potato breeding: a practical manual for the potato chain. Aardappelwereld BV, Den Haag</cite></li>
<li id="CR55">
<cite>Urbany C, Stich B, Schmidt L, Simon L, Berding H, Junghans H, Niehoff K-H, Braun A, Tacke E, Hofferbert H-R, Lübeck J, Strahwald J, Gebhardt C (2011) Association genetics in Solanum tuberosum provides new insights into potato tuber bruising and enzymatic tissue discoloration. BMC Genomics 12(1):7
</cite> [<a href="https://doi.org/10.1186/1471-2164-12-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3023753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21208436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Urbany%20C,%20Stich%20B,%20Schmidt%20L,%20Simon%20L,%20Berding%20H,%20Junghans%20H,%20Niehoff%20K-H,%20Braun%20A,%20Tacke%20E,%20Hofferbert%20H-R,%20L%C3%BCbeck%20J,%20Strahwald%20J,%20Gebhardt%20C%20(2011)%20Association%20genetics%20in%20Solanum%20tuberosum%20provides%20new%20insights%20into%20potato%20tuber%20bruising%20and%20enzymatic%20tissue%20discoloration.%20BMC%20Genomics%2012(1):7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56"><cite>Van Eck HJ (2007) Genetics of morphological and tuber traits. Potato Biol Biotechnol. Elsevier, pp 91–115</cite></li>
<li id="CR57">
<cite>VanRaden P (2008) Efficient methods to compute genomic predictions. J Dairy Sci 91(11):4414–4423
</cite> [<a href="https://doi.org/10.3168/jds.2007-0980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18946147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?VanRaden%20P%20(2008)%20Efficient%20methods%20to%20compute%20genomic%20predictions.%20J%20Dairy%20Sci%2091(11):4414%E2%80%934423" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<cite>Werij JS, Kloosterman B, Celis-Gamboa C, De Vos CHR, America T, Visser RGF, Bachem CWB (2007) Unravelling enzymatic discoloration in potato through a combined approach of candidate genes, QTL, and expression analysis. Theor Appl Genet 115(2):245–252
</cite> [<a href="https://doi.org/10.1007/s00122-007-0560-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1913181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17492422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Werij%20JS,%20Kloosterman%20B,%20Celis-Gamboa%20C,%20De%20Vos%20CHR,%20America%20T,%20Visser%20RGF,%20Bachem%20CWB%20(2007)%20Unravelling%20enzymatic%20discoloration%20in%20potato%20through%20a%20combined%20approach%20of%20candidate%20genes,%20QTL,%20and%20expression%20analysis.%20Theor%20Appl%20Genet%20115(2):245%E2%80%93252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<cite>Werner CR, Gaynor RC, Gorjanc G, Hickey JM, Kox T, Abbadi A, Leckband G, Snowdon RJ, Stahl A (2020) How population structure impacts genomic selection accuracy in cross-validation: implications for practical breeding. Front Plant Sci 11:592977
</cite> [<a href="https://doi.org/10.3389/fpls.2020.592977" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7772221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33391305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Werner%20CR,%20Gaynor%20RC,%20Gorjanc%20G,%20Hickey%20JM,%20Kox%20T,%20Abbadi%20A,%20Leckband%20G,%20Snowdon%20RJ,%20Stahl%20A%20(2020)%20How%20population%20structure%20impacts%20genomic%20selection%20accuracy%20in%20cross-validation:%20implications%20for%20practical%20breeding.%20Front%20Plant%20Sci%2011:592977" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<cite>Wilson S, Malosetti M, Maliepaard C, Mulder HA, Visser RGF, Van Eeuwijk F (2021) Training set construction for genomic prediction in auto-tetraploids: an example in potato. Front Plant Sci 12:771075
</cite> [<a href="https://doi.org/10.3389/fpls.2021.771075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8651708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34899794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilson%20S,%20Malosetti%20M,%20Maliepaard%20C,%20Mulder%20HA,%20Visser%20RGF,%20Van%20Eeuwijk%20F%20(2021)%20Training%20set%20construction%20for%20genomic%20prediction%20in%20auto-tetraploids:%20an%20example%20in%20potato.%20Front%20Plant%20Sci%2012:771075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<cite>Wilson S, Zheng C, Maliepaard C, Mulder HA, Visser RGF, Van Der Burgt A, Van Eeuwijk F (2021) Understanding the effectiveness of genomic prediction in tetraploid potato. Front Plant Sci 12:672417
</cite> [<a href="https://doi.org/10.3389/fpls.2021.672417" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8381724/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34434201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilson%20S,%20Zheng%20C,%20Maliepaard%20C,%20Mulder%20HA,%20Visser%20RGF,%20Van%20Der%20Burgt%20A,%20Van%20Eeuwijk%20F%20(2021)%20Understanding%20the%20effectiveness%20of%20genomic%20prediction%20in%20tetraploid%20potato.%20Front%20Plant%20Sci%2012:672417" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<cite>Windhausen VS, Atlin GN, Hickey JM, Crossa J, Jannink J-L, Sorrells ME, Raman B, Cairns JE, Tarekegne A, Semagn K, Beyene Y, Grudloyma P, Technow F, Riedelsheimer C, Melchinger AE (2012) Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments. G3 Genes Genomes Genet 2(11):1427–1436</cite> [<a href="https://doi.org/10.1534/g3.112.003699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3484673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23173094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Windhausen%20VS,%20Atlin%20GN,%20Hickey%20JM,%20Crossa%20J,%20Jannink%20J-L,%20Sorrells%20ME,%20Raman%20B,%20Cairns%20JE,%20Tarekegne%20A,%20Semagn%20K,%20Beyene%20Y,%20Grudloyma%20P,%20Technow%20F,%20Riedelsheimer%20C,%20Melchinger%20AE%20(2012)%20Effectiveness%20of%20genomic%20prediction%20of%20maize%20hybrid%20performance%20in%20different%20breeding%20populations%20and%20environments.%20G3%20Genes%20Genomes%20Genet%202(11):1427%E2%80%931436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<cite>Wu P-Y, Stich B, Hartje S, Muders K, Prigge V, Van Inghelandt D (2025) Optimal implementation of genomic selection in clone breeding programs exemplified in potato: II. Effect of selection strategy and cross-selection method on long-term genetic gain. Plant Genome 18(1):e70000
</cite> [<a href="https://doi.org/10.1002/tpg2.70000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11835509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39965909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20P-Y,%20Stich%20B,%20Hartje%20S,%20Muders%20K,%20Prigge%20V,%20Van%20Inghelandt%20D%20(2025)%20Optimal%20implementation%20of%20genomic%20selection%20in%20clone%20breeding%20programs%20exemplified%20in%20potato:%20II.%20Effect%20of%20selection%20strategy%20and%20cross-selection%20method%20on%20long-term%20genetic%20gain.%20Plant%20Genome%2018(1):e70000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<cite>Wu P-Y, Stich B, Renner J, Muders K, Prigge V, Van Inghelandt D (2023) Optimal implementation of genomic selection in clone breeding programs–Exemplified in potato: I. Effect of selection strategy, implementation stage, and selection intensity on short-term genetic gain. Plant Genome 16(2):e20327
</cite> [<a href="https://doi.org/10.1002/tpg2.20327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37177848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20P-Y,%20Stich%20B,%20Renner%20J,%20Muders%20K,%20Prigge%20V,%20Van%20Inghelandt%20D%20(2023)%20Optimal%20implementation%20of%20genomic%20selection%20in%20clone%20breeding%20programs%E2%80%93Exemplified%20in%20potato:%20I.%20Effect%20of%20selection%20strategy,%20implementation%20stage,%20and%20selection%20intensity%20on%20short-term%20genetic%20gain.%20Plant%20Genome%2016(2):e20327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<cite>Zhao Y, Gowda M, Longin FH, Würschum T, Ranc N, Reif JC (2012) Impact of selective genotyping in the training population on accuracy and bias of genomic selection. Theor Appl Genet 125(4):707–713
</cite> [<a href="https://doi.org/10.1007/s00122-012-1862-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22481121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20Y,%20Gowda%20M,%20Longin%20FH,%20W%C3%BCrschum%20T,%20Ranc%20N,%20Reif%20JC%20(2012)%20Impact%20of%20selective%20genotyping%20in%20the%20training%20population%20on%20accuracy%20and%20bias%20of%20genomic%20selection.%20Theor%20Appl%20Genet%20125(4):707%E2%80%93713" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12367938/bin/122_2025_5004_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 1 (pdf 1126 KB)</a><sup> (1.1MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The original data sets generated and/or analyzed in the current study are not publicly available due to the material being part of the company secrets of SaKa Pflanzenzucht GmbH &amp; Co. KG, Nordring-Kartoffelzucht- und Vermehrungs- GmbH &amp; Co. KG, and EUROPLANT Innovation GmbH &amp; Co. KG. However, the data are available in encoded form from the corresponding author upon reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik are provided here courtesy of <strong>Springer</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1007/s00122-025-05004-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/122_2025_Article_5004.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12367938/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12367938/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367938%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367938/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12367938/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12367938/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40833434/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12367938/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40833434/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12367938/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12367938/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="38zrGMXHHMVlAShyoLovbLGyCBKuFGTdBVzPk9JfJB6shioVMMmmnZxZ0kLCG9Vq">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
