
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genome‐wide association studies reveal genetic control of nutritional quality, milling traits, and agronomic characteristics in oat (Avena sativa L.) - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15251B3F8AEEE2C3051B3F0007EAB905.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12234416/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Genome‐wide association studies reveal genetic control of nutritional quality, milling traits, and agronomic characteristics in oat (Avena sativa L.)">
<meta name="citation_author" content="Naa Korkoi Ardayfio">
<meta name="citation_author_institution" content="Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA">
<meta name="citation_author_institution" content="USDA‐ARS Cereals Crops Improvement Research Unit, Edward T. Schafer Agricultural Research Center, Fargo, North Dakota, USA">
<meta name="citation_author" content="Christy Kinney">
<meta name="citation_author_institution" content="USDA‐ARS Cereals Crops Improvement Research Unit, Edward T. Schafer Agricultural Research Center, Fargo, North Dakota, USA">
<meta name="citation_author" content="Dandan Li">
<meta name="citation_author_institution" content="USDA‐ARS Cereals Crops Improvement Research Unit, Edward T. Schafer Agricultural Research Center, Fargo, North Dakota, USA">
<meta name="citation_author" content="Sepehr Mohajeri Naraghi">
<meta name="citation_author_institution" content="Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA">
<meta name="citation_author" content="Michael S McMullen">
<meta name="citation_author_institution" content="Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA">
<meta name="citation_author" content="Jason D Fiedler">
<meta name="citation_author_institution" content="USDA‐ARS Cereals Crops Improvement Research Unit, Edward T. Schafer Agricultural Research Center, Fargo, North Dakota, USA">
<meta name="citation_publication_date" content="2025 Jul 7">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70060">
<meta name="citation_doi" content="10.1002/tpg2.70060">
<meta name="citation_pmid" content="40623918">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12234416/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12234416/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12234416/pdf/TPG2-18-e70060.pdf">
<meta name="description" content="Marker‐trait associations (MTAs) are invaluable to the understanding of biological processes and implementation of marker‐assisted selection (MAS) to increase genetic gain in modern breeding programs. In this study, a genome‐wide association study ...">
<meta name="og:title" content="Genome‐wide association studies reveal genetic control of nutritional quality, milling traits, and agronomic characteristics in oat (Avena sativa L.)">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Marker‐trait associations (MTAs) are invaluable to the understanding of biological processes and implementation of marker‐assisted selection (MAS) to increase genetic gain in modern breeding programs. In this study, a genome‐wide association study ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12234416/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12234416">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70060"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70060.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12234416%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12234416/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12234416/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12234416/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70060" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Jul 7;18(3):e70060. doi: <a href="https://doi.org/10.1002/tpg2.70060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70060</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genome‐wide association studies reveal genetic control of nutritional quality, milling traits, and agronomic characteristics in oat (<em>Avena sativa</em> L.)</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ardayfio%20NK%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Naa Korkoi Ardayfio</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Naa Korkoi Ardayfio</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>USDA‐ARS Cereals Crops Improvement Research Unit, Edward T. Schafer Agricultural Research Center, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ardayfio%20NK%22%5BAuthor%5D" class="usa-link"><span class="name western">Naa Korkoi Ardayfio</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kinney%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Christy Kinney</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Christy Kinney</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>USDA‐ARS Cereals Crops Improvement Research Unit, Edward T. Schafer Agricultural Research Center, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kinney%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christy Kinney</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Dandan Li</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Dandan Li</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>USDA‐ARS Cereals Crops Improvement Research Unit, Edward T. Schafer Agricultural Research Center, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dandan Li</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Naraghi%20SM%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sepehr Mohajeri Naraghi</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sepehr Mohajeri Naraghi</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Naraghi%20SM%22%5BAuthor%5D" class="usa-link"><span class="name western">Sepehr Mohajeri Naraghi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McMullen%20MS%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Michael S McMullen</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Michael S McMullen</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McMullen%20MS%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael S McMullen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fiedler%20JD%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jason D Fiedler</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jason D Fiedler</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>USDA‐ARS Cereals Crops Improvement Research Unit, Edward T. Schafer Agricultural Research Center, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fiedler%20JD%22%5BAuthor%5D" class="usa-link"><span class="name western">Jason D Fiedler</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270060-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA</div>
<div id="tpg270060-aff-0002">
<sup>
<sup>2</sup>
</sup>USDA‐ARS Cereals Crops Improvement Research Unit, Edward T. Schafer Agricultural Research Center, Fargo, North Dakota, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Jason D. Fiedler, USDA‐ARS Cereals Crops Improvement Research Unit, Edward T. Schafer Agricultural Research Center, Fargo, ND 58102, USA. Email: <span>Jason.fiedler@usda.gov</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Feb 10; Received 2024 Sep 13; Accepted 2025 May 16; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America. This article has been contributed to by U.S. Government employees and their work is in the public domain in the USA.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12234416  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40623918/" class="usa-link">40623918</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Marker‐trait associations (MTAs) are invaluable to the understanding of biological processes and implementation of marker‐assisted selection (MAS) to increase genetic gain in modern breeding programs. In this study, a genome‐wide association study (GWAS) was performed to identify MTAs that influence oat nutritional quality, as well as agronomic and milling traits, in advanced breeding spring oat (<em>Avena sativa</em> L.) germplasm from North Dakota State University. High‐density sequence‐based molecular markers (15,037) were identified in a population of 1,092 unique lines evaluated in 11 field conditions over three years. Linkage disequilibrium analysis confirmed that decay metrics of 5, 11.5, and 2.2 Mb for the A, C, and D sub‐genomes, respectively, matched the 1.25 Mb/single nucleotide polymorphism marker density used in this study. We identified 160 MTAs and 44 durable quantitative trait loci (QTL) for nine traits that individually explained 3.7%–43.9% of the variation in the data. Haplotypes assembled from the three most predictive QTL for each trait increased the variation explained to 16%–46%. The haplotype effects are substantial, with their presence resulting in predicted mean percentage increases—greater than 20% for many of the investigated traits. These results confirm that significant MTAs for oat can be identified directly within breeding germplasm using GWAS. These markers are excellent candidates for the implementation of MAS to increase the selection efficiency and genetic gain.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270060-list-10001" class="list" style="list-style-type:disc">
<li><p>A genome‐wide association study was conducted on advanced germplasm of an oat breeding program.</p></li>
<li><p>Several marker‐trait associations (MTAs) were found for breeding‐related traits.</p></li>
<li><p>Haplotype analysis of the most important MTAs highlights their potential for early screening.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270060-dl-0001">
<dt>AYT</dt>
<dd><p>advanced yield trial</p></dd>
<dt>BLUE</dt>
<dd><p>best linear unbiased estimator</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>LD</dt>
<dd><p>linkage disequilibrium</p></dd>
<dt>MTA</dt>
<dd><p>marker‐trait association</p></dd>
<dt>NDSU</dt>
<dd><p>North Dakota State University</p></dd>
<dt>PYT</dt>
<dd><p>preliminary yield trial</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
</dl>
<section id="tpg270060-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Oat (<em>Avena sativa</em> L.) is an important cereal crop worldwide. In the United States, oat, which requires minimal inputs, is commonly produced as a grain or forage in an annual rotation. In 2022, over 1 million ha of planted oat yielded 57 million bushels of oat grain (groat) and generated over $294 million in revenue (USDA‐NASS, <a href="#tpg270060-bib-0020" class="usa-link" aria-describedby="tpg270060-bib-0020"><sup>2020</sup></a>). Additionally, oat cultivation has been shown to improve soil health and can increase the yields of rotation crops grown afterward (Bowles et al., <a href="#tpg270060-bib-0007" class="usa-link" aria-describedby="tpg270060-bib-0007">2020</a>).</p>
<p>Oat possesses a unique nutritional profile of essential amino acids, fatty acids, dietary fibers, and phenolic compounds (Butt et al., <a href="#tpg270060-bib-0014" class="usa-link" aria-describedby="tpg270060-bib-0014">2008</a>; Strychar, <a href="#tpg270060-bib-0067" class="usa-link" aria-describedby="tpg270060-bib-0067"><sup>2011</sup></a>; Peterson et al., <a href="#tpg270060-bib-0062" class="usa-link" aria-describedby="tpg270060-bib-0062"><sup>1992</sup></a>), making it an exceptionally healthy grain for human consumption. Among these nutritional qualities, macronutrients β‐glucan, oil, and protein have been prominent foci for breeding programs. Cereal β‐<span class="font-variant-small-caps">d</span>‐(1→3), (1→4)‐glucan, most abundant in oat and barley, is a hemicellulose present in the cell walls of the endosperm (Fincher, <a href="#tpg270060-bib-0025" class="usa-link" aria-describedby="tpg270060-bib-0025">2009</a>). Consumption of this dietary fiber reduces blood pressure, lowers serum cholesterol levels, and improves immune response—which in turn decreases the risk of cardiovascular disease, diabetes, and cancer (Braaten et al., <a href="#tpg270060-bib-0008" class="usa-link" aria-describedby="tpg270060-bib-0008">1994</a>; Estrada et al., <a href="#tpg270060-bib-0022" class="usa-link" aria-describedby="tpg270060-bib-0022"><sup>1997</sup></a>; Grundy et al., <a href="#tpg270060-bib-0029" class="usa-link" aria-describedby="tpg270060-bib-0029"><sup>2018</sup></a>; Jenkins et al., <a href="#tpg270060-bib-0040" class="usa-link" aria-describedby="tpg270060-bib-0040"><sup>2002</sup></a>; Keenan et al., <a href="#tpg270060-bib-0042" class="usa-link" aria-describedby="tpg270060-bib-0042"><sup>2002</sup></a>; Meydani, <a href="#tpg270060-bib-0055" class="usa-link" aria-describedby="tpg270060-bib-0055"><sup>2009</sup></a>). Notably, in scientific summary [21 Code of Federal Regulations (CFR) 101.9(c)(6)(i) and 21 CFR 101.81], the US Food and Drug Administration has confirmed that consumption of oat β‐glucan can reduce coronary heart disease risk and has approved health benefit labeling on oat products containing a minimum of 4.5% β‐glucan content (FDA, <a href="#tpg270060-bib-0024" class="usa-link" aria-describedby="tpg270060-bib-0024"><sup>2018</sup></a>).</p>
<p>Oat oil content provides significant health benefits for consumers. Healthy unsaturated oleic and linoleic fatty acids are highly concentrated in oat oil, which can comprise up to 18% of the edible grain (Brown &amp; Craddock, <a href="#tpg270060-bib-0012" class="usa-link" aria-describedby="tpg270060-bib-0012">1972</a>; Brown et al., <a href="#tpg270060-bib-0011" class="usa-link" aria-describedby="tpg270060-bib-0011"><sup>1966</sup></a>; Halima et al., <a href="#tpg270060-bib-0030" class="usa-link" aria-describedby="tpg270060-bib-0030"><sup>2015</sup></a>). Unsaturated fatty acids have been shown to reduce low‐density lipoprotein cholesterol in the blood, consequently reducing coronary heart disease risk (Mensink &amp; Katan, <a href="#tpg270060-bib-0054" class="usa-link" aria-describedby="tpg270060-bib-0054">1992</a>). Oil content is a highly heritable trait influenced by multiple genes with additive effects (Thro &amp; Frey, <a href="#tpg270060-bib-0070" class="usa-link" aria-describedby="tpg270060-bib-0070">1985</a>). Breeding goals for oat oil content are bifurcated, with low‐oil varieties considered desirable for milled products and high‐oil varieties showing promise as ingredients in novel food ingredients and cosmetics (Banaś &amp; Harasym, <a href="#tpg270060-bib-0003" class="usa-link" aria-describedby="tpg270060-bib-0003">2021</a>).</p>
<p>The protein composition and abundance in oat has revealed its potential as a significant plant‐based dietary protein source. Protein levels can exceed 20% by weight (Sunilkumar et al., <a href="#tpg270060-bib-0068" class="usa-link" aria-describedby="tpg270060-bib-0068">2017</a>) and are composed of mostly non‐immunogenic globulin and avenin classes, accounting for 50%–80% and 10%–20% of the total grain protein, respectively (Frey, <a href="#tpg270060-bib-0027" class="usa-link" aria-describedby="tpg270060-bib-0027">1951</a>; Peterson &amp; Smith, <a href="#tpg270060-bib-0063" class="usa-link" aria-describedby="tpg270060-bib-0063"><sup>1976</sup></a>). Thus, oats can be consumed without the allergy and tolerance concerns posed by soy and wheat protein sources.</p>
<p>Given its numerous health benefits and food product potential, oat is well‐positioned to meet the nutritional needs of the ever‐growing world population. However, in the face of emerging diseases and variable growing conditions, new oat varieties must continually be released to reliably provide high amounts of quality grain. The development of new oat cultivars has traditionally been conducted through direct phenotypic selection. With this method, breeders aim to simultaneously improve many agronomic, disease resistance, quality, and yield component characteristics to make the crop appealing for producers, processors, and consumers (Murphy &amp; Hoffman, <a href="#tpg270060-bib-0057" class="usa-link" aria-describedby="tpg270060-bib-0057">1992</a>). Due to the inverse correlations of many traits, both parent and cultivar selections can be challenging when employing a single breeding selection strategy (Lewis, <a href="#tpg270060-bib-0048" class="usa-link" aria-describedby="tpg270060-bib-0048">2006</a>; W. Yan &amp; Wallace, <a href="#tpg270060-bib-0079" class="usa-link" aria-describedby="tpg270060-bib-0079"><sup>1995</sup></a>). Additionally, it takes several site‐years of evaluations after progeny have been sufficiently inbred to determine if a particular line is promising enough for parent selection or promotion to variety candidate trials. For disease and abiotic stress resistance, introgression from wild progenitors has been employed to transfer these qualitative traits to adapted germplasm (Huang &amp; Han, <a href="#tpg270060-bib-0038" class="usa-link" aria-describedby="tpg270060-bib-0038">2014</a>). However, this process is laborious due to the low efficiency of interspecific crosses in the <em>Avena</em> genus and the significant resources required to break linkage drag.</p>
<p>Molecular breeding tools use predictive genomic loci to improve the efficiency of trait selection and introgression, enabling more rapid development of new cultivars compared to phenotypic selection (Bevan &amp; Uauy, <a href="#tpg270060-bib-0006" class="usa-link" aria-describedby="tpg270060-bib-0006">2013</a>). Marker‐assisted selection (MAS) uses single nucleotide polymorphisms (SNPs) or small insertion/deletion (InDel) markers that are genetically linked to quantitative trait loci (QTL) to enable the prediction of traits without direct observation. With high‐throughput screening of these marker‐trait associations (MTAs) in early‐generation germplasm, desirable traits can be enriched in lines that are evaluated directly for selection (Tanksley et al., <a href="#tpg270060-bib-0069" class="usa-link" aria-describedby="tpg270060-bib-0069">1989</a>; Young, <a href="#tpg270060-bib-0081" class="usa-link" aria-describedby="tpg270060-bib-0081"><sup>1999</sup></a>).</p>
<p>Genome‐wide association studies (GWASs) are a set of widely used procedures that connect phenotypic data and high‐density genotypic data to identify QTL and linked MTAs across the genome. This approach utilizes historical recombination in nonstructured populations and can capture large amounts of variation within crops (Hamblin et al., <a href="#tpg270060-bib-0031" class="usa-link" aria-describedby="tpg270060-bib-0031">2011</a>). When genomes are saturated with molecular markers, gene‐level resolution mapping can be achieved (Brachi et al., <a href="#tpg270060-bib-0009" class="usa-link" aria-describedby="tpg270060-bib-0009">2011</a>). Through GWAS, MTAs have been identified for various important traits in many crops, including oat. One such study, using 446 North American oat breeding lines genotyped with 1,005 Diversity Array Technology markers and two years of phenotypic data, identified 21 molecular markers that influenced β‐glucan concentration with a single marker test. Application of the mixed‐model least absolute shrinkage and selection operator with the same data identified 51 significant markers, 15 of which had been previously reported (Asoro et al., <a href="#tpg270060-bib-0002" class="usa-link" aria-describedby="tpg270060-bib-0002">2013</a>). Newell and colleagues also conducted a GWAS with 431 oat accessions from the National Small Grains Collection. Three independent markers were identified as significantly associated with groat β‐glucan content. One of these significant markers showed homology to a genetic sequence on rice Chr7 adjacent to the CslF gene family that has been implicated with β‐glucan synthase function (Newell et al., <a href="#tpg270060-bib-0059" class="usa-link" aria-describedby="tpg270060-bib-0059">2012</a>). In another GWAS, field experiments were conducted with 174 varied oat accessions to evaluate groat protein content (GPC) in three locations. Of the 38,313 genotyping‐by‐sequencing markers identified in the study, 41 markers tagged 27 unique QTL that influence GPC. Two QTL on chromosomes 6C and 4D, were highly predictive and detected in multiple environments (Yan et al., <a href="#tpg270060-bib-0078" class="usa-link" aria-describedby="tpg270060-bib-0078">2023</a>). A GWAS study was conducted on a total of 2,993 lines from a nested association mapping population of 10 diverse spring oat lines where 60 unique variables for panicles, rachis, and spikelets were investigated. The authors discovered many QTL and concluded that taller, stiff stalked and open panicle structured oat plants were important for adult plant resistance for crown rust (Carlson et al., <a href="#tpg270060-bib-0086" class="usa-link" aria-describedby="tpg270060-bib-0086">2023</a>) As a final example, three different GWAS approaches (multivariate mixed model approach, principal component, and univariate analyses) were used to evaluate the composition of ten fatty acids in 500 oat lines grown in two field environments. These methods identified 148, 129, and 73 significant MTAs (Carlson et al., <a href="#tpg270060-bib-0086" class="usa-link" aria-describedby="tpg270060-bib-0086">2023</a>).</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270060-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270060-list-0001" class="list" style="list-style-type:disc">
<li><p>A genome‐wide association study was conducted on advanced germplasm of an oat breeding program.</p></li>
<li><p>Several marker‐trait associations (MTAs) were found for breeding‐related traits.</p></li>
<li><p>Haplotype analysis of the most important MTAs highlights their potential for early screening.</p></li>
</ul></section></section><p>The use of unbalanced data in GWAS to identify QTL has become increasingly popular due to the availability of many years of experimental trials from breeding programs that often use this field design to increase efficiency. The predominant drawbacks of an unbalanced data set for genetic studies are limited population sizes and the increased risk of false positives. A study was conducted on barley breeding germplasm by Wang et al. (<a href="#tpg270060-bib-0073" class="usa-link" aria-describedby="tpg270060-bib-0073">2012</a>) to evaluate differences between a balanced and unbalanced population in detecting QTL. Two panels were created to evaluate heading date, which comprised of a balanced trial of 766 spring barley breeding genotypes and a second panel of a balanced and unbalanced trial subset of 384 spring barley breeding genotypes. Three QTL that had been previously detected in a biparental mapping studies were identified in both panels. There was, however, an increase in false‐positive rate in the unbalanced dataset, which was reduced in a one‐step analysis. The researchers concluded that data from breeding programs are important to detect QTL, but experimental design and population size must be carefully considered (Wang et al., <a href="#tpg270060-bib-0073" class="usa-link" aria-describedby="tpg270060-bib-0073">2012</a>).</p>
<p>In this study, we conducted GWAS with late‐stage breeding germplasm from the North Dakota State University (NDSU) breeding program that was evaluated over three years in multiple field sites. The overall objectives of the study were to (i) identify predictive QTL for agronomic, milling, and nutritional quality characteristics in hexaploid oats, (ii) select the most informative markers for each QTL, and (iii) predict trait outcomes with haplotype analysis of unlinked QTL. With this knowledge in hand, we aim to accelerate breeding efforts with the conversion of QTL into high‐throughput informative markers and screening of early‐generation germplasm‐.</p>
<p>Molecular breeding methods such as MAS and genomic selection (GS) are both powerful tools to increase genetic gain in breeding programs (Goddard &amp; Hayes, <a href="#tpg270060-bib-0028" class="usa-link" aria-describedby="tpg270060-bib-0028">2007</a>; Jannink et al., <a href="#tpg270060-bib-0039" class="usa-link" aria-describedby="tpg270060-bib-0039"><sup>2010</sup></a>; Kou &amp; Wang, <a href="#tpg270060-bib-0046" class="usa-link" aria-describedby="tpg270060-bib-0046"><sup>2010</sup></a>; Weckwerth, <a href="#tpg270060-bib-0075" class="usa-link" aria-describedby="tpg270060-bib-0075"><sup>2011</sup></a>). They both use molecular markers to predict traits, but differ in the scope of the genotyping and processing of the decision support model for the breeder. Single or multiple large‐effect QTL that reliably predict traits can be quickly ascertained with low‐cost polymerase chain reaction‐based markers and directly converted to a trait score in an MAS scheme. On the other hand, GS can better predict highly quantitative traits that result from many low‐effect QTL but is more expensive to implement as it requires a more costly durable genome‐wide marker platform and a relatively complex statistical model. Here we focus on supporting MAS through the identification of a small set of high‐effect QTL because of its cost and ease of use in many classical breeding programs that have not utilized molecular breeding tools in the past.</p></section><section id="tpg270060-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270060-sec-0040"><h3 class="pmc_sec_title">2.1. Plant materials</h3>
<p>The 1092 unique NDSU breeding lines used in this study were evaluated in North Dakota in 2016, 2018, and 2019. There were several related parents within the breeding lines, as shown by pedigree in Tables <a href="#tpg270060-supinfo-0001" class="usa-link">S1</a> and <a href="#tpg270060-supinfo-0001" class="usa-link">S2</a>. Certain check genotypes were repeated across years and experimental trials (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S3</a>). Breeding lines in the seventh generation of self‐pollination were evaluated in preliminary yield trials (PYTs) in 2016. Evaluation in 2018 and 2019 consisted of PYT lines in the seventh‐generation and eighth‐generation lines from advanced yield trials (AYT). For each trial, lines were evaluated in 4 × 8 feet plots in a randomized complete block field design with each replicate representing a block. The middle two rows of each block were harvested to ensure genotype purity during pollination. The 2016 trials were conducted in two locations with two replications per location (Table <a href="#tpg270060-tbl-0001" class="usa-link">1</a>). Traits measured in 2016 included heading date (days to head), height (cm), thins (%), yield (kg/ha), and test weight (kg/hL). The 2018 trials consisted of five experiments: two AYTs with four locations and three PYTs with two locations. Experimental trials PYTs and AYTs had different experiments that focused on different breeding objectives. For instance, there were early, mid, and late maturing experiments. The number of experiments was consistent between locations of the same year. Entries within experiments were replicated three times in AYTs and twice in PYTs. Entries denotes numbers assigned to individual genotypes in a replication. A second replication of the same location will therefore consist of a randomization of the individual genotypes (entries) of the first replication in the same location. Traits measured in 2018 included β‐glucan (%), protein (%), oil (%), heading date, height, test weight, thins, yield, and groat (%). In 2019, there were four PYT experiments grown in two locations with two replicates each, as well as two AYT experiments grown in five locations with three replicates each. Traits measured in 2019 included β‐glucan, protein, oil, heading date, height, test weight, thins, yield and groat. The number of locations, traits measured, and replications in PYTs differed from that of AYTs. PYTs across the three years consisted of the Casselton and Fargo locations, each with two replications. The AYTs of 2018 and 2019 consisted of four and five locations respectively, with three replications each. The specific experimental trials across the three years and their unique genotype number, together with locations, traits measured within location and between years as well as specific replications within location and years is tabulated in Table <a href="#tpg270060-supinfo-0001" class="usa-link">S1</a>. Field evaluation locations were near the North Dakota cities of Fargo (DMS Latitude: 46° 52′ 37.0596″ N, DMS Longitude: 96° 47′ 4.6932″ W), Casselton (DMS Latitude: 46° 54′ 1.8036″ N, DMS Longitude: 97° 12′ 40.8852″ W), Carrington (DMS Latitude: 47° 26′ 58.99″ N, DMS Longitude: 99° 07′ 34.41″ W), Edgeley (DMS Latitude: 46° 21′ 37.29″ N, DMS Longitude: 98° 42′ 43.38″ W), Verona (DMS Latitude: 46° 21′ 49.89″ N, DMS Longitude: 98° 04′ 20.36″ W), and Minot (DMS Latitude: 48° 13′ 48.5904″ N, DMS Longitude: 101° 17′ 28.3164″ W).</p>
<section class="tw xbox font-sm" id="tpg270060-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Summary of spatially adjusted trait measurements for each location per year and averaged trait best linear unbiased estimator (BLUE) values.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Year/location</th>
<th align="left" rowspan="1" colspan="1">β ‐Glucan (%)</th>
<th align="left" rowspan="1" colspan="1">Protein (%)</th>
<th align="left" rowspan="1" colspan="1">Oil (%)</th>
<th align="left" rowspan="1" colspan="1">Test weight (kg/hL)</th>
<th align="left" rowspan="1" colspan="1">Groat (%)</th>
<th align="left" rowspan="1" colspan="1">
<p>Yield</p>
<p>(kg/ha)</p>
</th>
<th align="left" rowspan="1" colspan="1">Thins (%)</th>
<th align="left" rowspan="1" colspan="1">Heading date (days)</th>
<th align="left" rowspan="1" colspan="1">
<p>Height</p>
<p>(cm)</p>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>2016</strong>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">
<strong>50 ± 2.6</strong>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">
<strong>4878 ± 738.9</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>9 ± 4.2</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>23 ± 3.3</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>99 ± 12.1</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Casselton</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">50 ± 2.5</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">5309 ± 677.9</td>
<td align="left" rowspan="1" colspan="1">9 ± 4.2</td>
<td align="left" rowspan="1" colspan="1">25 ± 2.2</td>
<td align="left" rowspan="1" colspan="1">109 ± 7.4</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Fargo</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">50 ± 2.6</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">4448 ± 520.1</td>
<td align="left" rowspan="1" colspan="1">9 ± 4.3</td>
<td align="left" rowspan="1" colspan="1">20 ± 2.0</td>
<td align="left" rowspan="1" colspan="1">89 ± 6.1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>2018</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>5 ±</strong>
<strong>0.5</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>17 ± 1.2</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>6 ± 0.7</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>51 ± 2.2</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>74 ± 4.4</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>4878 ± 764.0</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>8 ± 3.6</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>23 ± 1.8</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>94 ± 7.7</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Carrington</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">55 ± 2.1</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">4663 ± 573.9</td>
<td align="left" rowspan="1" colspan="1">9 ± 4.2</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Casselton</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">51 ± 1.7</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">5309 ± 835.8</td>
<td align="left" rowspan="1" colspan="1">7 ± 3.5</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">100 ± 5.0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Edgeley</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">54 ± 1.6</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">4197 ± 491.4</td>
<td align="left" rowspan="1" colspan="1">8 ± 3.6</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Fargo</td>
<td align="left" rowspan="1" colspan="1">5 ± 0.5</td>
<td align="left" rowspan="1" colspan="1">17 ± 1.2</td>
<td align="left" rowspan="1" colspan="1">6 ± 0.7</td>
<td align="left" rowspan="1" colspan="1">50 ± 2.0</td>
<td align="left" rowspan="1" colspan="1">74 ± 4.4</td>
<td align="left" rowspan="1" colspan="1">4555 ± 444.8</td>
<td align="left" rowspan="1" colspan="1">7 ± 3.5</td>
<td align="left" rowspan="1" colspan="1">23 ± 1.8</td>
<td align="left" rowspan="1" colspan="1">89 ± 5.2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>2019</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>4 ± 0.5</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>14 ± 1.3</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>6 ± 0.8</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>50 ± 2.4</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>74 ± 4.8</strong>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">
<strong>7 ± 3.7</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>38 ± 2.7</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>111 ± 9.0</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Carrington</td>
<td align="left" rowspan="1" colspan="1">4 ± 0.5</td>
<td align="left" rowspan="1" colspan="1">14 ± 1.2</td>
<td align="left" rowspan="1" colspan="1">6 ± 0.7</td>
<td align="left" rowspan="1" colspan="1">51 ± 1.4</td>
<td align="left" rowspan="1" colspan="1">76 ± 2.0</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">6 ± 2.2</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Casselton</td>
<td align="left" rowspan="1" colspan="1">4 ± 0.5</td>
<td align="left" rowspan="1" colspan="1">14 ± 1.3</td>
<td align="left" rowspan="1" colspan="1">6 ± 0.8</td>
<td align="left" rowspan="1" colspan="1">‐</td>
<td align="left" rowspan="1" colspan="1">74 ± 5.3</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">6 ± 3.6</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">115 ± 8.5</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Fargo</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">‐</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">7 ± 3.9</td>
<td align="left" rowspan="1" colspan="1">38 ± 2.7</td>
<td align="left" rowspan="1" colspan="1">106 ± 7.1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Minot</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">51 ± 1.7</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">10 ± 3.8</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Verona</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">47 ± 1.7</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">7 ± 3.5</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270060-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270060-tbl1-note-0001"><p>
<em>Note</em>: Values are averages ± standard deviation, Bold values denote year BLUEs.</p></div></div></section></section><section id="tpg270060-sec-0050"><h3 class="pmc_sec_title">2.2. Phenotyping</h3>
<p>Experiments were conducted under field conditions at the various experiment stations. Soil amendments were added when necessary to maintain optimum plant growth according to standard local management practices. Agronomic and milling traits measured include test weight, groat percentage, proportions of both plump and thin grains per sample, heading date, and height. Individual traits were measured in various years and locations as specified in Table <a href="#tpg270060-tbl-0001" class="usa-link">1</a>. Heading date was evaluated when 50% or more panicle nodes had just emerged from the flag leaf sheath and recorded as the number of days after May 31. Height measurements were collected when the oat plants began senescence and were measured in centimeters from the ground level to the tip of the longest panicle, guided by the hand along a measuring pole. Quality and yield‐related traits were measured after harvest. Oat panicles were threshed in the field, and grains were weighed for yield estimate after 14% moisture was obtained. Test weight, which represents grain density, was measured by weighing grain contained in a 500‐mL chondrometer and reported in kilograms per hectoliter, with higher test weight indicating higher quality grain. Plumps and thins are subsets of yield in which thins represent the smallest seed sizes, typically from the tertiary florets, while plumps are the large seeds, typically from the primary and secondary florets, that meet a desirable milling threshold value. These subsets were determined after threshing by sifting grains with a 5/64 inch (12.7 cm) perforated screen. Grains ≤5/64 inch (12.7 cm) were considered thins, and presented as a percentage of the total grain sample.</p>
<p>Oat groats are the portion of grains (kernels) left after removal of the outer hull. Samples were de‐hulled in a Codema dehuller for 1 min 30 s duration, and groat percentage was calculated from groat weight as a fraction of the entire weight of the whole grain. β‐glucan, protein, and oil content were measured using near infrared (NIR) spectroscopy on 50 g of ground or whole de‐hulled seed in an NIR grain analyzer and reported as a percentage of total mass. These phenotypic data used in this study are available in Table <a href="#tpg270060-supinfo-0001" class="usa-link">S2</a>.</p></section><section id="tpg270060-sec-0060"><h3 class="pmc_sec_title">2.3. Genotyping‐by‐sequencing</h3>
<p>Individuals from F<sub>7</sub> generation of the NDSU breeding germplasm (<em>n</em> = 1,124) were genotyped with a previously described dual‐enzyme genotyping‐by‐sequencing (GBS) method (Carlson et al., <a href="#tpg270060-bib-0086" class="usa-link" aria-describedby="tpg270060-bib-0086">2023</a>) with some minor modifications in the bioinformatic processing. Approximately 2.5 million paired end 2 × 150 bp reads per line were acquired for this population and after quality filtering and trimming with bbduk (BBTools, <a href="https://jgi.doe.gov/data-and-tools/bbtools" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://jgi.doe.gov/data‐and‐tools/bbtools</a>), sequences were aligned to the <em>A. sativa</em> OT3098 reference genome assembly (v1; PepsiCo, <a href="#tpg270060-bib-0061" class="usa-link" aria-describedby="tpg270060-bib-0061"><sup>2021</sup></a>; <a href="https://wheat.pw.usda.gov/GG3/graingenes_downloads/oat-ot3098-pepsico" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://wheat.pw.usda.gov/GG3/graingenes_downloads/oat‐ot3098‐pepsico</a>) with Bowtie 2 (Langmead &amp; Salzberg, <a href="#tpg270060-bib-0047" class="usa-link" aria-describedby="tpg270060-bib-0047">2012</a>). Duplicate alignments were marked with <em>samblaster</em> (Faust &amp; Hall, <a href="#tpg270060-bib-0023" class="usa-link" aria-describedby="tpg270060-bib-0023">2014</a>) and reads with a mapQ score of at least 30 were processed with <em>samtools/bcftools</em> (Li, <a href="#tpg270060-bib-0049" class="usa-link" aria-describedby="tpg270060-bib-0049">2011</a>) to identify population‐level SNPs and small insertions/deletions. Genotype calls were filtered with VCFtools (minDP = 2, max‐missing = 0.5, maf = 0.01) and TASSEL v 5.2.50 (Bradbury et al., <a href="#tpg270060-bib-0010" class="usa-link" aria-describedby="tpg270060-bib-0010">2007</a>) to remove sites with greater than 10% heterozygous calls. Missing data were imputed with the LD‐KNNi method (Money et al., <a href="#tpg270060-bib-0056" class="usa-link" aria-describedby="tpg270060-bib-0056">2015</a>), and an additional filtering step removed taxa with greater than 75% missing data or greater than 20% heterozygous calls. Positions in this file were then lifted over to the v2 release of the OT3098 genome (PepsiCo, <a href="#tpg270060-bib-0061" class="usa-link" aria-describedby="tpg270060-bib-0061">2021</a>) using unambiguous BLAST alignments of the surrounding sequence. This final genotyping data contained 15,037 sites (13,348 SNPs; 1,689 InDels), 1,092 taxa, and contained an overall missingness rate of 2% and heterozygous call rate of 4.6%.</p></section><section id="tpg270060-sec-0070"><h3 class="pmc_sec_title">2.4. Statistical analysis</h3>
<p>Linkage disequilibrium (LD) was calculated on a 50‐site sliding window with TASSEL5 (v5.2.50). The LD decay half‐life metrics were estimated for each chromosome, sub‐genome, and the entire genome using nonlinear regression of the <em>r</em>
<sup>2</sup> value versus distance between sites (Abecasis et al., <a href="#tpg270060-bib-0001" class="usa-link" aria-describedby="tpg270060-bib-0001">2001</a>; Hill &amp; Weir, <a href="#tpg270060-bib-0034" class="usa-link" aria-describedby="tpg270060-bib-0034"><sup>1988</sup></a>) (Table <a href="#tpg270060-tbl-0002" class="usa-link">2</a>) with a custom script in R (v4.2.1). Visualization of LD data was conducted with the TASSEL5 heatmap chart tool. Visualization of the trait data was conducted with ggpairs function in the GGally R package (Schloerke et al., <a href="#tpg270060-bib-0065" class="usa-link" aria-describedby="tpg270060-bib-0065">2022</a>) and Adobe Illustrator. Multi‐environment trial data were analyzed using a two‐stage approach. In Stage 1, each trial was analyzed using a spatial model implemented in the SpATS R package (Rodríguez‐Álvarez et al., <a href="#tpg270060-bib-0089" class="usa-link" aria-describedby="tpg270060-bib-0089">2018</a>). The spatial model was as follows:</p>
<table class="disp-formula p" id="tpg270060-disp-0001"><tr><td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><mi>Y</mi><mi>i</mi><mi>j</mi><mi>k</mi><mo linebreak="badbreak">=</mo><mi>G</mi><mi>i</mi><mo linebreak="goodbreak">+</mo><mi>f</mi><mfenced separators="" open="(" close=")"><mrow><mi>r</mi><mi>k</mi><mo>,</mo><mi>c</mi><mi>l</mi></mrow></mfenced><mo linebreak="goodbreak">+</mo><mi>B</mi><mi>j</mi><mo linebreak="goodbreak">+</mo><mi>R</mi><mi>k</mi><mo linebreak="goodbreak">+</mo><mi>C</mi><mi>l</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">ε</mi><mi>i</mi><mi>j</mi><mi>k</mi><mo>,</mo></mrow></mrow></math></td></tr></table>
<p>where <em>Yijk</em> is the observed phenotype, <em>Gi</em> is the effect of genotype <em>i</em>, <em>f</em>(<em>rk</em>, <em>cl</em>) represents the smooth spatial surface modeled using P‐splines, <em>Bj</em> is the fixed effect of block <em>j</em>, <em>Rk</em> and <em>Cl</em> are the random effects of row <em>k</em> and column <em>l</em>, respectively, and <em>εijk</em> is the residual error. The spatial surface was modeled using a penalized spline analysis of variance approach with tensor product P‐splines. Means were summarized in Table <a href="#tpg270060-tbl-0001" class="usa-link">1</a>, listed in Table <a href="#tpg270060-supinfo-0001" class="usa-link">S4</a>, and visualized in Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S1</a>.</p>
<section class="tw xbox font-sm" id="tpg270060-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Linkage disequilibrium (LD) decay half‐life for individual chromosomes, sub‐genomes, and the entire genome.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th align="left" rowspan="1" colspan="1">LD decay (Mb)</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Sub‐genome</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Chromosome</th>
<th align="left" rowspan="1" colspan="1">A</th>
<th align="left" rowspan="1" colspan="1">C</th>
<th align="left" rowspan="1" colspan="1">D</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">4.89</td>
<td align="left" rowspan="1" colspan="1">358.10</td>
<td align="left" rowspan="1" colspan="1">2.08</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">4.28</td>
<td align="left" rowspan="1" colspan="1">254.51</td>
<td align="left" rowspan="1" colspan="1">2.40</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">3.45</td>
<td align="left" rowspan="1" colspan="1">20.13</td>
<td align="left" rowspan="1" colspan="1">0.76</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">5.32</td>
<td align="left" rowspan="1" colspan="1">31.98</td>
<td align="left" rowspan="1" colspan="1">2.28</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">8.84</td>
<td align="left" rowspan="1" colspan="1">3.07</td>
<td align="left" rowspan="1" colspan="1">1.88</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">4.11</td>
<td align="left" rowspan="1" colspan="1">4.71</td>
<td align="left" rowspan="1" colspan="1">3.79</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">4.08</td>
<td align="left" rowspan="1" colspan="1">2.91</td>
<td align="left" rowspan="1" colspan="1">6.16</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total</td>
<td align="left" rowspan="1" colspan="1">4.99</td>
<td align="left" rowspan="1" colspan="1">11.47</td>
<td align="left" rowspan="1" colspan="1">2.21</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Genome</td>
<td colspan="3" align="left" rowspan="1">6.11</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270060-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>In Stage 2, spatially adjusted means were combined across environments using a weighted mixed model approach. First, within each environment, adjusted means were normalized using check varieties:
</p>
<table class="disp-formula p" id="tpg270060-disp-0002"><tr><td class="formula"><math id="jats-math-2" display="block"><mrow><mrow><mrow><mi>Normalized</mi><mo>_</mo></mrow><mi>i</mi><mi>j</mi><mo linebreak="badbreak">=</mo><mover accent="true"><mi>Y</mi><mo>^</mo></mover><mi>i</mi><mi>j</mi><mo>/</mo><mi mathvariant="normal">μ</mi><mi>c</mi><mo>_</mo><mi>j</mi><mo>,</mo></mrow></mrow></math></td></tr></table>
<p>where <em>Ŷij</em> is the adjusted mean of genotype <em>i</em> in environment <em>j</em>, and <em>μc</em>_<em>j</em> is the mean of check varieties in environment <em>j</em>. Standard errors were propagated through normalization as SE_norm = SE/<em>μc</em>_<em>j</em>. The across‐environment model was as follows:
</p>
<table class="disp-formula p" id="tpg270060-disp-0003"><tr><td class="formula"><math id="jats-math-3" display="block"><mrow><mrow><mrow><mi>Normalized</mi><mo>_</mo></mrow><mi>i</mi><mi>j</mi><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><mi>G</mi><mi>i</mi><mo linebreak="goodbreak">+</mo><mi>E</mi><mi>j</mi><mo linebreak="goodbreak">+</mo><mi>G</mi><mi>E</mi><mi>i</mi><mi>j</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">ε</mi><mi>i</mi><mi>j</mi><mo>,</mo></mrow></mrow></math></td></tr></table>
<p>where <em>Gi</em> is the fixed effect of genotype <em>i</em>, <em>Ej</em> is the random effect of environment <em>j</em>, <em>GEij</em> is the random genotype‐by‐environment interaction, and <em>εij</em> is the residual error. Observations were weighted using <em>w</em> = 1/SE_norm<sup>2</sup>. When the mixed model indicated convergence issues, a reduced fixed‐effects model was fitted as follows:
</p>
<table class="disp-formula p" id="tpg270060-disp-0004"><tr><td class="formula"><math id="jats-math-4" display="block"><mrow><mrow><mrow><mi>Normalized</mi><mo>_</mo></mrow><mi>i</mi><mi>j</mi><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><mi>G</mi><mi>i</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">ε</mi><mi>i</mi><mi>j</mi><mo>.</mo></mrow></mrow></math></td></tr></table>
<p>Environments with fewer than three observations or zero variance were excluded from analysis. Genotype reliability was calculated as 1 − [SE<sup>2</sup>/var(BLUE)]., where SE is the standard error of the genotype estimate, var(BLUE) is the variance of all genotype estimates, and BLUE is the best linear unbiased estimator. Normalized BLUE values can be found in Table <a href="#tpg270060-supinfo-0001" class="usa-link">S5</a>.</p>
<p>GWAS was performed with the Genome Association and Prediction Integrated Tool (Lipka et al., <a href="#tpg270060-bib-0050" class="usa-link" aria-describedby="tpg270060-bib-0050">2012</a>; Zhang et al., <a href="#tpg270060-bib-0083" class="usa-link" aria-describedby="tpg270060-bib-0083"><sup>2010</sup></a>) within R using the Bayesian‐information and linkage‐disequilibrium iteratively nested keyway (BLINK) (Huang et al., <a href="#tpg270060-bib-0037" class="usa-link" aria-describedby="tpg270060-bib-0037">2019</a>) model for the normalized BLUE values with the check entries removed (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S5</a>). The first three principal components were included as covariates to control for population substructure (explains 16.5% of variation; Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S2</a>). For all traits, the BLINK model performed well with <em>QQ</em> and Manhattan plots provided for each trait in Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S3</a>. For each trait of interest, markers with a percent variation explained (PVE) greater than 3% and a Benjami–Hochberg adjusted <em>p</em>‐value &lt; 0.05 were retained for further analysis (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>).</p>
<p>To simplify visualization and description of the markers and QTL identified when using the normalized BLUE values, traits were grouped by allele, and genotype calls were converted to a presence (+) or absence (−) of the SNP/QTL to demark the greater or lesser numeric value, respectively (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>). Heterozygous genotype calls were present as a minor allele state for many of the markers selected above. A Wilcoxon signed rank‐test was used to determine if trait values of the heterozygous state were significantly different than one of the homozygous calls for the purposes of assigning a QTL designation. In most cases, heterozygous states could be unambiguously assigned (e.g., β‐glucan:S4C_27965926; Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S4</a>). In situations where it was ambiguous (e.g., β‐glucan:S6D_289646620 or oil:S4C_21549747; Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S4</a>), heterozygous calls were converted to missing. The PVE by each significant marker identified by GWAS was estimated by fitting a simple linear model (lm) in R relating the allele call to the trait scores (normalized BLUE values or individual environment means) and extracting the adjusted <em>R</em>
<sup>2</sup> value. The effect of each QTL was estimated as the difference between the means of the trait variables grouped by the QTL presence or absence. This difference was multiplied by the per‐environment averages of the check lines to estimate a tangible trait value increase for discussion. Markers that displayed PVE &gt; 2% in at least half of the environments tested (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S7</a>) were kept for functional assessment. The local LD environment of each marker was assessed in TASSEL to determine an approximate QTL linkage block size of the remaining durable markers, and stepwise regression was used to identify the three most important QTL for each trait with the R package olsrr v0.6.0.9 (ols_step_best_subset function). The selected QTL were named with the convention Q<em>trait</em>.fgl‐<em>Chr</em>#. Visualization of trait distributions by QTL and haplotype was generated with R package ggplot2 (v 3.5.1). Comparative analysis of the QTL identified in this study with previously published QTL was possible with alignment of markers underlying both on the OT3098 v2 reference genome and Sang reference genome (Kamal et al., <a href="#tpg270060-bib-0041" class="usa-link" aria-describedby="tpg270060-bib-0041">2022</a>). Genes located within the estimated LD were evaluated for functionality. These data can be found in Table <a href="#tpg270060-supinfo-0001" class="usa-link">S8</a>.</p></section></section><section id="tpg270060-sec-0080"><h2 class="pmc_sec_title">3. RESULTS AND DISCUSSION</h2>
<section id="tpg270060-sec-0090"><h3 class="pmc_sec_title">3.1. Trait location relationships</h3>
<p>In this study, trait data were collected over three years from up to six locations per year. Within the 11 year‐location combinations (environments) tested, there were differences among the trait distributions (Table <a href="#tpg270060-tbl-0001" class="usa-link">1</a>; Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S1</a>), which can be attributed to the environment‐specific responses of the traits and the differences in germplasm grown in each environment. Due to the unbalanced nature of the data, spatially adjusted values within each experiment were normalized with up to 16 checks and combined into BLUEs (Tables <a href="#tpg270060-supinfo-0001" class="usa-link">S4</a> and <a href="#tpg270060-supinfo-0001" class="usa-link">S5</a>). BLUEs were normally distributed for all traits except for thins, which were right‐skewed (Figure <a href="#tpg270060-fig-0001" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="tpg270060-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b5d4/12234416/7ad5f7e05111/TPG2-18-e70060-g003.jpg" loading="lazy" id="jats-graphic-1" height="466" width="703" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270060-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pairwise correlation among trait best linear unbiased estimator (BLUE) values of β‐glucan, oil, protein, test weight, groat, yield, thins, heading date, and height. The diagonal displays the frequency distribution of the traits. The upper triangle shows the Pearson's correlation coefficient (<em>R</em>) between traits and the significance of correlation for the comparison (*<em>p</em> ≤ 0.05, **<em>p</em> ≤ 0.01, ***<em>p</em> ≤ 0.001). The lower triangle shows the scatterplots of the data.</p></figcaption></figure><p>Pairwise comparisons among the trait BLUEs revealed relatively weak correlations, with the strongest inverse association found between thins and yield (<em>r</em> = −0.387) (Figure <a href="#tpg270060-fig-0001" class="usa-link">1</a>), which is expected as a lower thins value indicates a larger percentage of plump groat and greater mass per groat on average. Conversely, the strongest identified positive correlation was between test weight and groat (<em>r</em> = 0.391), which is indicative of a denser grain maintaining integrity during the milling process (Doehlert et al., <a href="#tpg270060-bib-0021" class="usa-link" aria-describedby="tpg270060-bib-0021">2006</a>). The correlation between β‐glucan and protein BLUEs is consistent with the positive association of protein and β‐glucan reported in previous studies (Kibite &amp; Edney, <a href="#tpg270060-bib-0045" class="usa-link" aria-describedby="tpg270060-bib-0045">1998</a>; Peterson et al., <a href="#tpg270060-bib-0064" class="usa-link" aria-describedby="tpg270060-bib-0064"><sup>1995</sup></a>). Overall, the lack of strong correlations among these traits indicates a potential for the independent improvement of each trait through MAS.</p></section><section id="tpg270060-sec-0100"><h3 class="pmc_sec_title">3.2. Overall genetic analysis</h3>
<p>The germplasm evaluated in this study originates from the NDSU oat breeding program. These lines were a combination of F<sub>7</sub> and F<sub>8</sub> lines derived from crosses made between 2007 and 2009. All the entries for the preliminary and advanced yield trials were genotyped with a sequence‐based protocol and after taxa‐based quality filtering for excessive data missingness and/or heterozygosity levels, 1,092 lines remained for association analysis.</p>
<p>After site‐based quality filtering, 13,349 SNPs and 1,689 InDel molecular markers were used for subsequent analysis. The average marker density of 1 per 1.25 Mb was less than the estimates of LD decay of 4.99, 11.47, and 2.21 Mb for the A, C, and D sub‐genomes, respectively (Table <a href="#tpg270060-tbl-0002" class="usa-link">2</a>). The first three principal components cumulatively represented 16.5% of the genetic variation in the population, suggesting relatively low substructure, as well as suitability for GWAS (Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S2</a>).</p></section><section id="tpg270060-sec-0110"><h3 class="pmc_sec_title">3.3. Identified marker‐trait associations</h3>
<p>Across the GWAS experiments conducted here, 158 MTAs were identified (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>) with 65 associations having an overall PVE greater than 3% and 44 of these displaying durable effects in multiple environments. Evaluation of the LD decay among 65 high‐effect markers enabled further binning into linkage blocks. The 44 durable markers evaluated here resided on 35 unique linkage blocks, with some blocks influencing multiple traits. This population displayed localized regions of high LD where large genomic blocks were inherited together with minimal recombination, especially near the centromeres. This was especially apparent for QProtein.fgl‐3A, which spanned ∼75% of the physical chromosome.</p>
<p>While 44 durable QTL were identified for the nine traits studied here, enrichment for all of them would be impractical in a breeding program. Therefore, stepwise regression was used to select the three most predictive QTL per trait (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>). These named QTL were evaluated alone and in concert as haplotypes to provide evidence for their combined usage to explain trait variation in this data set.</p>
<section class="tw xbox font-sm" id="tpg270060-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Information about the three most predictive QTL and combined haplotype for traits.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr>
<th align="left" rowspan="1" colspan="1">QTL name</th>
<th align="left" rowspan="1" colspan="1">Chr</th>
<th align="left" rowspan="1" colspan="1">Position(s)</th>
<th align="left" rowspan="1" colspan="1">Size (Mb)</th>
<th align="left" rowspan="1" colspan="1">Linkage block (Mb)</th>
<th align="left" rowspan="1" colspan="1">TagSNP</th>
<th align="left" rowspan="1" colspan="1">TagSNP Chr (Sang)</th>
<th align="left" rowspan="1" colspan="1">
<p>TagSNP</p>
<p>(Sang)</p>
</th>
<th align="left" rowspan="1" colspan="1">BinaryMin.Allele</th>
<th align="left" rowspan="1" colspan="1">BinaryMax.Allele</th>
<th align="left" rowspan="1" colspan="1">Min.Ave</th>
<th align="left" rowspan="1" colspan="1">Max.Ave</th>
<th align="left" rowspan="1" colspan="1">
<em>R</em>
<sup>2</sup> (%)</th>
<th align="left" rowspan="1" colspan="1">Haplotype <em>R</em>
<sup>2</sup> (%)</th>
<th align="left" rowspan="1" colspan="1">Haplotype diff</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Qβglucan.fgl‐6D</td>
<td align="left" rowspan="1" colspan="1">6D</td>
<td align="left" rowspan="1" colspan="1">289,646,620</td>
<td align="left" rowspan="1" colspan="1">12.6</td>
<td align="left" rowspan="1" colspan="1">279–291.6</td>
<td align="left" rowspan="1" colspan="1">S6D_289646620</td>
<td align="left" rowspan="1" colspan="1">chr6D</td>
<td align="left" rowspan="1" colspan="1">280274759</td>
<td align="left" rowspan="1" colspan="1">G</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="left" rowspan="1" colspan="1">0.95</td>
<td align="left" rowspan="1" colspan="1">1.09</td>
<td align="left" rowspan="1" colspan="1">4.2</td>
<td align="left" rowspan="1" colspan="1">10.9</td>
<td align="left" rowspan="1" colspan="1">0.27</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Qβglucan.fgl‐4C</td>
<td align="left" rowspan="1" colspan="1">4C</td>
<td align="left" rowspan="1" colspan="1">27,965,926</td>
<td align="left" rowspan="1" colspan="1">57.8</td>
<td align="left" rowspan="1" colspan="1">2–59.8</td>
<td align="left" rowspan="1" colspan="1">S4C_27965926</td>
<td align="left" rowspan="1" colspan="1">chr4C</td>
<td align="left" rowspan="1" colspan="1">36304705</td>
<td align="left" rowspan="1" colspan="1">A/R</td>
<td align="left" rowspan="1" colspan="1">G</td>
<td align="left" rowspan="1" colspan="1">0.94</td>
<td align="left" rowspan="1" colspan="1">0.98</td>
<td align="left" rowspan="1" colspan="1">3.9</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Qβglucan.fgl‐3A</td>
<td align="left" rowspan="1" colspan="1">3A</td>
<td align="left" rowspan="1" colspan="1">105,822,640</td>
<td align="left" rowspan="1" colspan="1">338.5</td>
<td align="left" rowspan="1" colspan="1">64.5–403</td>
<td align="left" rowspan="1" colspan="1">S3A_105822640</td>
<td align="left" rowspan="1" colspan="1">chr3A</td>
<td align="left" rowspan="1" colspan="1">99356317</td>
<td align="left" rowspan="1" colspan="1">G</td>
<td align="left" rowspan="1" colspan="1">T/K</td>
<td align="left" rowspan="1" colspan="1">0.95</td>
<td align="left" rowspan="1" colspan="1">1.01</td>
<td align="left" rowspan="1" colspan="1">3.7</td>
<td align="left" rowspan="1" colspan="1"> </td>
<td align="left" rowspan="1" colspan="1"> </td>
</tr>
<tr><td colspan="15" style="border-bottom:solid 1px #000000" align="left" rowspan="1"> </td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">QProtein.fgl‐2A</td>
<td align="left" rowspan="1" colspan="1">2A</td>
<td align="left" rowspan="1" colspan="1">398,107,713</td>
<td align="left" rowspan="1" colspan="1">1.3</td>
<td align="left" rowspan="1" colspan="1">397.5–398.8</td>
<td align="left" rowspan="1" colspan="1">S2A_398107713</td>
<td align="left" rowspan="1" colspan="1">chr2A</td>
<td align="left" rowspan="1" colspan="1">391838360</td>
<td align="left" rowspan="1" colspan="1">G</td>
<td align="left" rowspan="1" colspan="1">A/R</td>
<td align="left" rowspan="1" colspan="1">1.01</td>
<td align="left" rowspan="1" colspan="1">1.06</td>
<td align="left" rowspan="1" colspan="1">6.1</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QProtein.fgl‐5A</td>
<td align="left" rowspan="1" colspan="1">5A</td>
<td align="left" rowspan="1" colspan="1">339,889,539</td>
<td align="left" rowspan="1" colspan="1">96</td>
<td align="left" rowspan="1" colspan="1">326–422</td>
<td align="left" rowspan="1" colspan="1">S5A_339889539</td>
<td align="left" rowspan="1" colspan="1">chr5A</td>
<td align="left" rowspan="1" colspan="1">341585243</td>
<td align="left" rowspan="1" colspan="1">G/R</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="left" rowspan="1" colspan="1">0.98</td>
<td align="left" rowspan="1" colspan="1">1.04</td>
<td align="left" rowspan="1" colspan="1">5.8</td>
<td align="left" rowspan="1" colspan="1">13.9</td>
<td align="left" rowspan="1" colspan="1">0.16</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QProtein.fgl‐3C</td>
<td align="left" rowspan="1" colspan="1">3C</td>
<td align="left" rowspan="1" colspan="1">580,559,480</td>
<td align="left" rowspan="1" colspan="1">516</td>
<td align="left" rowspan="1" colspan="1">90–606</td>
<td align="left" rowspan="1" colspan="1">S3C_580559480</td>
<td align="left" rowspan="1" colspan="1">chr3C</td>
<td align="left" rowspan="1" colspan="1">516556095</td>
<td align="left" rowspan="1" colspan="1">T</td>
<td align="left" rowspan="1" colspan="1">K</td>
<td align="left" rowspan="1" colspan="1">1.02</td>
<td align="left" rowspan="1" colspan="1">1.12</td>
<td align="left" rowspan="1" colspan="1">4.2</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr><td colspan="15" style="border-bottom:solid 1px #000000" align="left" rowspan="1"> </td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">QOil.fgl‐4C</td>
<td align="left" rowspan="1" colspan="1">4C</td>
<td align="left" rowspan="1" colspan="1">21,549,747</td>
<td align="left" rowspan="1" colspan="1">57.8</td>
<td align="left" rowspan="1" colspan="1">2–59.8</td>
<td align="left" rowspan="1" colspan="1">S4C_21549747</td>
<td align="left" rowspan="1" colspan="1">chr4C</td>
<td align="left" rowspan="1" colspan="1">29613037</td>
<td align="left" rowspan="1" colspan="1">G</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="left" rowspan="1" colspan="1">0.91</td>
<td align="left" rowspan="1" colspan="1">1.01</td>
<td align="left" rowspan="1" colspan="1">16.9</td>
<td align="left" rowspan="1" colspan="1">28.3</td>
<td align="left" rowspan="1" colspan="1">0.24</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QOil.fgl‐6A</td>
<td align="left" rowspan="1" colspan="1">6A</td>
<td align="left" rowspan="1" colspan="1">423,631,188</td>
<td align="left" rowspan="1" colspan="1">0.0003</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">S6A_423631188</td>
<td align="left" rowspan="1" colspan="1">chr6A</td>
<td align="left" rowspan="1" colspan="1">419213421</td>
<td align="left" rowspan="1" colspan="1">T/Y</td>
<td align="left" rowspan="1" colspan="1">C</td>
<td align="left" rowspan="1" colspan="1">0.86</td>
<td align="left" rowspan="1" colspan="1">0.98</td>
<td align="left" rowspan="1" colspan="1">11.0</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QOil.fgl‐5A</td>
<td align="left" rowspan="1" colspan="1">5A</td>
<td align="left" rowspan="1" colspan="1">469,061,327</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">S5A_469061327</td>
<td align="left" rowspan="1" colspan="1">chr5A</td>
<td align="left" rowspan="1" colspan="1">462151006</td>
<td align="left" rowspan="1" colspan="1">C</td>
<td align="left" rowspan="1" colspan="1">del/0</td>
<td align="left" rowspan="1" colspan="1">0.94</td>
<td align="left" rowspan="1" colspan="1">1.01</td>
<td align="left" rowspan="1" colspan="1">7.3</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr><td colspan="15" style="border-bottom:solid 1px #000000" align="left" rowspan="1"> </td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">QTestweight.fgl‐4D</td>
<td align="left" rowspan="1" colspan="1">4D</td>
<td align="left" rowspan="1" colspan="1">452,015,611</td>
<td align="left" rowspan="1" colspan="1">3.3</td>
<td align="left" rowspan="1" colspan="1">452–455.3</td>
<td align="left" rowspan="1" colspan="1">S4D_452015611</td>
<td align="left" rowspan="1" colspan="1">chr4D</td>
<td align="left" rowspan="1" colspan="1">415009851</td>
<td align="left" rowspan="1" colspan="1">C</td>
<td align="left" rowspan="1" colspan="1">T/Y</td>
<td align="left" rowspan="1" colspan="1">1.00</td>
<td align="left" rowspan="1" colspan="1">1.07</td>
<td align="left" rowspan="1" colspan="1">14.1</td>
<td align="left" rowspan="1" colspan="1">20.8</td>
<td align="left" rowspan="1" colspan="1">0.17</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QTestweight.fgl‐6C</td>
<td align="left" rowspan="1" colspan="1">6C</td>
<td align="left" rowspan="1" colspan="1">312,911,424</td>
<td align="left" rowspan="1" colspan="1">242</td>
<td align="left" rowspan="1" colspan="1">246–488</td>
<td align="left" rowspan="1" colspan="1">S6C_312911424</td>
<td align="left" rowspan="1" colspan="1">chr6C</td>
<td align="left" rowspan="1" colspan="1">288842472</td>
<td align="left" rowspan="1" colspan="1">C/S</td>
<td align="left" rowspan="1" colspan="1">G</td>
<td align="left" rowspan="1" colspan="1">1.008</td>
<td align="left" rowspan="1" colspan="1">1.11</td>
<td align="left" rowspan="1" colspan="1">10.9</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QTestweight.fgl‐7A</td>
<td align="left" rowspan="1" colspan="1">7A</td>
<td align="left" rowspan="1" colspan="1">255,608,481</td>
<td align="left" rowspan="1" colspan="1">252.1</td>
<td align="left" rowspan="1" colspan="1">233.9–486</td>
<td align="left" rowspan="1" colspan="1">S7A_255608481</td>
<td align="left" rowspan="1" colspan="1">chr7A</td>
<td align="left" rowspan="1" colspan="1">253792761</td>
<td align="left" rowspan="1" colspan="1">T</td>
<td align="left" rowspan="1" colspan="1">C</td>
<td align="left" rowspan="1" colspan="1">0.95</td>
<td align="left" rowspan="1" colspan="1">1.00</td>
<td align="left" rowspan="1" colspan="1">4.0</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr><td colspan="15" style="border-bottom:solid 1px #000000" align="left" rowspan="1"> </td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">QYield.fgl‐4D.1</td>
<td align="left" rowspan="1" colspan="1">4D</td>
<td align="left" rowspan="1" colspan="1">455,312,110</td>
<td align="left" rowspan="1" colspan="1">3.3</td>
<td align="left" rowspan="1" colspan="1">452–455.3</td>
<td align="left" rowspan="1" colspan="1">S4D_455312110</td>
<td align="left" rowspan="1" colspan="1">chr4D</td>
<td align="left" rowspan="1" colspan="1">411462465</td>
<td align="left" rowspan="1" colspan="1">T</td>
<td align="left" rowspan="1" colspan="1">C/Y</td>
<td align="left" rowspan="1" colspan="1">0.78</td>
<td align="left" rowspan="1" colspan="1">1.04</td>
<td align="left" rowspan="1" colspan="1">16.7</td>
<td align="left" rowspan="1" colspan="1">20.9</td>
<td align="left" rowspan="1" colspan="1">0.34</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QYield.fgl‐6C</td>
<td align="left" rowspan="1" colspan="1">6C</td>
<td align="left" rowspan="1" colspan="1">259,051,881</td>
<td align="left" rowspan="1" colspan="1">242</td>
<td align="left" rowspan="1" colspan="1">246–488</td>
<td align="left" rowspan="1" colspan="1">S6C_259051881</td>
<td align="left" rowspan="1" colspan="1">chr6C</td>
<td align="left" rowspan="1" colspan="1">243191511</td>
<td align="left" rowspan="1" colspan="1">T</td>
<td align="left" rowspan="1" colspan="1">C</td>
<td align="left" rowspan="1" colspan="1">0.68</td>
<td align="left" rowspan="1" colspan="1">1.03</td>
<td align="left" rowspan="1" colspan="1">7.0</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QYield.fgl‐4D.2</td>
<td align="left" rowspan="1" colspan="1">4D</td>
<td align="left" rowspan="1" colspan="1">450,577,160</td>
<td align="left" rowspan="1" colspan="1">0.6</td>
<td align="left" rowspan="1" colspan="1">450.6–451.2</td>
<td align="left" rowspan="1" colspan="1">S4D_450577160</td>
<td align="left" rowspan="1" colspan="1">chr4D</td>
<td align="left" rowspan="1" colspan="1">407603698</td>
<td align="left" rowspan="1" colspan="1">G</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="left" rowspan="1" colspan="1">0.91</td>
<td align="left" rowspan="1" colspan="1">1.03</td>
<td align="left" rowspan="1" colspan="1">3.8</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr><td colspan="15" style="border-bottom:solid 1px #000000" align="left" rowspan="1"> </td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">QThins.fgl‐4D.1</td>
<td align="left" rowspan="1" colspan="1">4D</td>
<td align="left" rowspan="1" colspan="1">454,967,583</td>
<td align="left" rowspan="1" colspan="1">3.3</td>
<td align="left" rowspan="1" colspan="1">452–455.3</td>
<td align="left" rowspan="1" colspan="1">S4D_454967583</td>
<td align="left" rowspan="1" colspan="1">chr4D</td>
<td align="left" rowspan="1" colspan="1">411984450</td>
<td align="left" rowspan="1" colspan="1">G</td>
<td align="left" rowspan="1" colspan="1">A/R</td>
<td align="left" rowspan="1" colspan="1">0.6</td>
<td align="left" rowspan="1" colspan="1">2.02</td>
<td align="left" rowspan="1" colspan="1">43.9</td>
<td align="left" rowspan="1" colspan="1">45.9</td>
<td align="left" rowspan="1" colspan="1">1.99</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QThins.fgl‐4D.2</td>
<td align="left" rowspan="1" colspan="1">4D</td>
<td align="left" rowspan="1" colspan="1">426,900,796</td>
<td align="left" rowspan="1" colspan="1">0.2</td>
<td align="left" rowspan="1" colspan="1">426.8–427</td>
<td align="left" rowspan="1" colspan="1">S4D_426900796</td>
<td align="left" rowspan="1" colspan="1">chr4D</td>
<td align="left" rowspan="1" colspan="1">384063720</td>
<td align="left" rowspan="1" colspan="1">A/R</td>
<td align="left" rowspan="1" colspan="1">G</td>
<td align="left" rowspan="1" colspan="1">0.61</td>
<td align="left" rowspan="1" colspan="1">2.1</td>
<td align="left" rowspan="1" colspan="1">34.2</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QThins.fgl‐6C</td>
<td align="left" rowspan="1" colspan="1">6C</td>
<td align="left" rowspan="1" colspan="1">24,305,773</td>
<td align="left" rowspan="1" colspan="1">64.5</td>
<td align="left" rowspan="1" colspan="1">22.6–87.1</td>
<td align="left" rowspan="1" colspan="1">S6C_24305773</td>
<td align="left" rowspan="1" colspan="1">chr6C</td>
<td align="left" rowspan="1" colspan="1">22479044</td>
<td align="left" rowspan="1" colspan="1">G</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">1.61</td>
<td align="left" rowspan="1" colspan="1">7.2</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr><td colspan="15" style="border-bottom:solid 1px #000000" align="left" rowspan="1"> </td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">QHeadingdate.fgl‐6A</td>
<td align="left" rowspan="1" colspan="1">6A</td>
<td align="left" rowspan="1" colspan="1">410,925,381</td>
<td align="left" rowspan="1" colspan="1">51.7</td>
<td align="left" rowspan="1" colspan="1">363–414.7</td>
<td align="left" rowspan="1" colspan="1">S6A_410925381</td>
<td align="left" rowspan="1" colspan="1">chr6A</td>
<td align="left" rowspan="1" colspan="1">407174339</td>
<td align="left" rowspan="1" colspan="1">C</td>
<td align="left" rowspan="1" colspan="1">T/Y</td>
<td align="left" rowspan="1" colspan="1">0.98</td>
<td align="left" rowspan="1" colspan="1">1.04</td>
<td align="left" rowspan="1" colspan="1">8.2</td>
<td align="left" rowspan="1" colspan="1">16.0</td>
<td align="left" rowspan="1" colspan="1">0.15</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QHeadingdate.fgl‐6C</td>
<td align="left" rowspan="1" colspan="1">6C</td>
<td align="left" rowspan="1" colspan="1">103,114,557</td>
<td align="left" rowspan="1" colspan="1">13.1</td>
<td align="left" rowspan="1" colspan="1">102.9–116</td>
<td align="left" rowspan="1" colspan="1">S6C_103114557</td>
<td align="left" rowspan="1" colspan="1">chr6C</td>
<td align="left" rowspan="1" colspan="1">97233736</td>
<td align="left" rowspan="1" colspan="1">C</td>
<td align="left" rowspan="1" colspan="1">T</td>
<td align="left" rowspan="1" colspan="1">0.95</td>
<td align="left" rowspan="1" colspan="1">1.01</td>
<td align="left" rowspan="1" colspan="1">4.0</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QHeadingdate.fgl‐1D</td>
<td align="left" rowspan="1" colspan="1">1D</td>
<td align="left" rowspan="1" colspan="1">349,564,707</td>
<td align="left" rowspan="1" colspan="1">3.2</td>
<td align="left" rowspan="1" colspan="1">347.5–350.7</td>
<td align="left" rowspan="1" colspan="1">S1D_349564707</td>
<td align="left" rowspan="1" colspan="1">chr1D</td>
<td align="left" rowspan="1" colspan="1">339995569</td>
<td align="left" rowspan="1" colspan="1">C/Y</td>
<td align="left" rowspan="1" colspan="1">T</td>
<td align="left" rowspan="1" colspan="1">0.97</td>
<td align="left" rowspan="1" colspan="1">1.01</td>
<td align="left" rowspan="1" colspan="1">5.5</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr><td colspan="15" style="border-bottom:solid 1px #000000" align="left" rowspan="1"> </td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">QHeight.fgl‐6D</td>
<td align="left" rowspan="1" colspan="1">6D</td>
<td align="left" rowspan="1" colspan="1">279,296,541</td>
<td align="left" rowspan="1" colspan="1">13.5</td>
<td align="left" rowspan="1" colspan="1">278–291.5</td>
<td align="left" rowspan="1" colspan="1">S6D_279296541</td>
<td align="left" rowspan="1" colspan="1">chr6D</td>
<td align="left" rowspan="1" colspan="1">270768735</td>
<td align="left" rowspan="1" colspan="1">C</td>
<td align="left" rowspan="1" colspan="1">0/del</td>
<td align="left" rowspan="1" colspan="1">1.03</td>
<td align="left" rowspan="1" colspan="1">1.06</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">15.9</td>
<td align="left" rowspan="1" colspan="1">0.08</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QHeight.fgl‐6A</td>
<td align="left" rowspan="1" colspan="1">6A</td>
<td align="left" rowspan="1" colspan="1">54,027,913</td>
<td align="left" rowspan="1" colspan="1">191.4</td>
<td align="left" rowspan="1" colspan="1">52.4–243.8</td>
<td align="left" rowspan="1" colspan="1">S6A_54027913</td>
<td align="left" rowspan="1" colspan="1">chr6A</td>
<td align="left" rowspan="1" colspan="1">54620180</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="left" rowspan="1" colspan="1">G/R</td>
<td align="left" rowspan="1" colspan="1">1.03</td>
<td align="left" rowspan="1" colspan="1">1.06</td>
<td align="left" rowspan="1" colspan="1">6.0</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">QHeight.fgl‐4D</td>
<td align="left" rowspan="1" colspan="1">4D</td>
<td align="left" rowspan="1" colspan="1">367,447,171</td>
<td align="left" rowspan="1" colspan="1">37</td>
<td align="left" rowspan="1" colspan="1">333–370</td>
<td align="left" rowspan="1" colspan="1">S4D_367447171</td>
<td align="left" rowspan="1" colspan="1">chr4D</td>
<td align="left" rowspan="1" colspan="1">328541356</td>
<td align="left" rowspan="1" colspan="1">C</td>
<td align="left" rowspan="1" colspan="1">0/del</td>
<td align="left" rowspan="1" colspan="1">1.03</td>
<td align="left" rowspan="1" colspan="1">1.06</td>
<td align="left" rowspan="1" colspan="1">5.5</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270060-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270060-tbl3-note-0001"><p>Abbreviations: Chr, chromosome; QTL, quantitative trait loci; SNP, single nucleotide polymorphism.</p></div></div></section></section><section id="tpg270060-sec-0120"><h3 class="pmc_sec_title">3.4. QTL for β‐glucan of hexaploid oat</h3>
<p>Four significant molecular markers across four chromosomes were identified for β‐glucan content in this study. Chromosomes 3A, 4C, 6D, and 7D each contained one SNP (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>). Significance values ranged from 1.3 to 2.3 (−log<sub>10</sub> Hochberg &amp; Benjamini <em>P</em> value, HBP), with the percent variation explained (PVE) by these markers ranging from 1.7% to 28.7% when the allele call was used as the estimator. Out of the four significant markers that met the initial PVE &gt; 3 criteria, one additional marker was dropped due to a lack of durability across the three environments tested. Stepwise regression identified the three most predictive QTL from this dataset as Qβglucan.fgl‐6D, Qβglucan.fgl‐4C, and Qβglucan.fgl‐3A. Grouping the lines by the presence or absence of each QTL revealed significant differences in the normalized BLUE values (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270060-fig-0002" class="usa-link">2A</a>) that were also observed across multiple environments (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S7</a>; Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S5A–C</a>). Qβglucan.fgl‐6D, the most predictive QTL, explained 4.2% of the variation for β‐glucan with presence of the QTL increasing β‐glucan content by approximately 0.6%. Further combining the three individual QTL into one haplotype explained 10.9% of the variation, and the presence of all three favorable QTL increased the β‐glucan content by 1.1% (Figure <a href="#tpg270060-fig-0002" class="usa-link">2A,B</a>). To be labeled as heart‐healthy by the FDA, oat varieties must consistently display a β‐glucan content of at least 4.5%. Being able to predict a 1% increase with markers alone will assist in early triage of poorly performing varieties and accelerate genetic gain for the trait.</p>
<figure class="fig xbox font-sm" id="tpg270060-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b5d4/12234416/1f6486d0c46b/TPG2-18-e70060-g002.jpg" loading="lazy" id="jats-graphic-3" height="731" width="700" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270060-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distributions of three most useful quantitative trait loci (QTL) indicate significant differences using best linear unbiased estimator (BLUE) value with or without alleles of the specific markers of (A) β‐glucan content, (C) protein content, and (E) oil content. The significance of differences between the pairwise means was determined with the Wilcox test (***<em>p</em> ≤ 0.001). Haplotype calls of collective distribution differences with or without alleles among the three most predictive QTL of (B) β‐glucan content, (D) protein content, and (F) oil content. The significance of differences between the groups was determined with Tukey's test (<em>α</em> = 0.05).</p></figcaption></figure><p>The major QTL found here on 4C, tagged with S4C_27965926 (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>), was found on a large linkage block spanning approximately 2–60 Mb. Within this region, we identified previously reported CELLULOSE SYNTHASE‐LIKE 9 (CslF9_chr4C_24 Mb), QTL QBG.CORE‐BG_11.1, as well as Qbgl.UFRGS.L17N‐Mrg11.1 (Bazzer et al., <a href="#tpg270060-bib-0005" class="usa-link" aria-describedby="tpg270060-bib-0005">2025</a>; Fogarty et al., <a href="#tpg270060-bib-0026" class="usa-link" aria-describedby="tpg270060-bib-0026"><sup>2019</sup></a>; Tinker et al., <a href="#tpg270060-bib-0071" class="usa-link" aria-describedby="tpg270060-bib-0071"><sup>2022</sup></a>). We also identified a <em>CslA7</em> gene (AVESA.00001b.r3.4Cg0000625 @ 47 Mb) that may also be implicated in this trait (Hazen et al., <a href="#tpg270060-bib-0033" class="usa-link" aria-describedby="tpg270060-bib-0033">2002</a>). The cellulose synthase‐like gene family is known to be responsible for the hemicelluloses backbone synthesis of plant cell walls and β‐glucan synthesis in oat (Burton et al., <a href="#tpg270060-bib-0013" class="usa-link" aria-describedby="tpg270060-bib-0013">2006</a>; Yin et al., <a href="#tpg270060-bib-0080" class="usa-link" aria-describedby="tpg270060-bib-0080"><sup>2009</sup></a>). Another QTL, Qβglucan.fgl‐6D, aligned to 6D:280 Mb on the Sang reference genome and is likely within the linkage block of the CslF9/11_Chr6D_254 Mb, another QTL/gene previously implicated with this trait (Tinker et al., <a href="#tpg270060-bib-0071" class="usa-link" aria-describedby="tpg270060-bib-0071">2022</a>). Six other QTL on chromosome 7D were previously discovered and may be connected to the S7D_442623148 marker identified in this study that was removed upon filtering due to lack of durability across environments (Bazzer et al., <a href="#tpg270060-bib-0005" class="usa-link" aria-describedby="tpg270060-bib-0005">2025</a>; Fogarty et al., <a href="#tpg270060-bib-0026" class="usa-link" aria-describedby="tpg270060-bib-0026"><sup>2019</sup></a>; Tinker et al., <a href="#tpg270060-bib-0071" class="usa-link" aria-describedby="tpg270060-bib-0071"><sup>2022</sup></a>). The mapping populations from which these β‐glucan QTL were identified involve the high‐β‐glucan parent, HiFi, which was developed by the NDSU program and is in the pedigree of many of the lines used in this study (McMullen et al., <a href="#tpg270060-bib-0053" class="usa-link" aria-describedby="tpg270060-bib-0053">2005</a>).</p></section><section id="tpg270060-sec-0130"><h3 class="pmc_sec_title">3.5. QTL for protein of hexaploid oat</h3>
<p>Five molecular markers located across five chromosomes (2A, 3C, 5A, 6A, and 7D) were identified as associated with protein content (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>). Significance values ranged from 1.8 to 8.9 (−log<sub>10</sub> HBP) and PVE ranged from 4% to 10.6%. Two markers were removed from analysis due to lack of durable predictive ability in the three environments tested, and stepwise regression identified the three most predictive QTL as QProtein.fgl‐2A, QProtein.fgl‐5A, and QProtein.fgl‐3C. (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270060-fig-0002" class="usa-link">2C</a>; Table <a href="#tpg270060-supinfo-0001" class="usa-link">S7</a>; Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S5D–F</a>). Linkage analysis suggested that these QTL were present on co‐inherited blocks with physical sizes of 1.3, 96, and 516 Mb, respectively. QProtein.fgl‐2A was the most predictive QTL, explaining 6.1% of variation in protein, with presence of this QTL providing an approximate increase of between 0.6% and 0.9% in protein content over its absence (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>). Combining the three individual QTL into haplotypes explained 13.9% of variation (Figure <a href="#tpg270060-fig-0002" class="usa-link">2D</a>) with the presence of all three QTL in the haplotype predicting an approximate increase of 2%–3% over the absence of the haplotype (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270060-fig-0002" class="usa-link">2D</a>).</p>
<p>The major QTL identified here, QProtein.fgl‐2A (S2A_398107713), was positioned ∼30 Mb proximal to a QTL identified by De Koeyer et al. (<a href="#tpg270060-bib-0019" class="usa-link" aria-describedby="tpg270060-bib-0019">2004</a>) and flanked by markers ISU707A/UMN464. Additionally, QProtein.fgl‐3C overlapped another reported grain protein content QTL on the long arm of 3C reported by Yan et al. (<a href="#tpg270060-bib-0078" class="usa-link" aria-describedby="tpg270060-bib-0078">2023</a>); however, the LD in our population was very high on chromosome 3C, so any connection between the two QTL may be coincidental. The last QTL selected in this study, QProtein.fgl‐5A, appears to be novel in relation to the handful of previously published mapping papers for grain protein content (Yan et al., <a href="#tpg270060-bib-0078" class="usa-link" aria-describedby="tpg270060-bib-0078">2023</a>). With the high potential for elevated protein content in oat, relative to other small grains, further validation of these markers in more environments will be a priority for future studies.</p></section><section id="tpg270060-sec-0140"><h3 class="pmc_sec_title">3.6. QTL for oil of hexaploid oat</h3>
<p>Groat oil percentage was found to be associated with six significant molecular markers on four chromosomes that explained 4.3%–15.7% of the observed variation (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>). LD analysis of the genotyping data suggested no linkage between the three markers identified on chromosome 6A. Stepwise regression analysis indicated the three most predictive QTL to be QOil.fgl‐4C, QOil.fgl‐6A, and QOil.fgl‐5A, explaining 16.9%, 11%, and 7.3% of variation, respectively (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270060-fig-0002" class="usa-link">2E</a>; Table <a href="#tpg270060-supinfo-0001" class="usa-link">S7</a>; Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S5G–I</a>).</p>
<p>The QTL QOil.fgl‐4C displayed a slightly greater PVE (16.9% vs. 15.9%) when the marker was evaluated as a binary (±) versus the original genotype call with three allele states. We estimated an approximate increase of 0.6% in oil content when the QTL was present. Additionally, the presence of all three QTL in a haplotype predicted an increase of approximately 1% oil content over the absence of the haplotype in this population (Figure <a href="#tpg270060-fig-0002" class="usa-link">2A</a>).</p>
<p>Among the QTL identified here for oil content in the NDSU oat breeding program (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S2</a>), those on chromosomes 4C and 6A were nearby five markers/QTL identified in previous studies (Campbell et al., <a href="#tpg270060-bib-0015" class="usa-link" aria-describedby="tpg270060-bib-0015">2021</a>; Kianian et al., <a href="#tpg270060-bib-0043" class="usa-link" aria-describedby="tpg270060-bib-0043"><sup>1999</sup></a>; Orr &amp; Molnar, <a href="#tpg270060-bib-0060" class="usa-link" aria-describedby="tpg270060-bib-0060"><sup>2007</sup></a>; Tinker et al., <a href="#tpg270060-bib-0071" class="usa-link" aria-describedby="tpg270060-bib-0071"><sup>2022</sup></a>; Zhu et al., <a href="#tpg270060-bib-0084" class="usa-link" aria-describedby="tpg270060-bib-0084"><sup>2004</sup></a>). One of these markers, S4C_21549747 (QOil.fgl‐4C), was located within 100 kb of marker CDO665_WHE03A39.r1 and the linked ACCaseA locus, which has previously been found associated with groat oil content in several biparental mapping populations—Kanota × Ogle, Kanota × Marion, Dal × Exeter, and Goslin × Hifi (Kianian et al., <a href="#tpg270060-bib-0043" class="usa-link" aria-describedby="tpg270060-bib-0043">1999</a>; Tinker et al., <a href="#tpg270060-bib-0071" class="usa-link" aria-describedby="tpg270060-bib-0071"><sup>2022</sup></a>).</p>
<p>The genome regions underlying QOil.fgl‐4C, QOil.fgl‐6A, and QOil.fgl‐5A contained 821, 30, and 13 annotated genes, respectively (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S8</a>). The gene <em>AVESA.00001b.r3.4Cg0000200</em> located at 15.1 Mb was within the LD block of our identified QTL and shares protein homology (91% identity) to the previously reported ACCase gene (Kianian et al., <a href="#tpg270060-bib-0043" class="usa-link" aria-describedby="tpg270060-bib-0043">1999</a>; Tinker et al., <a href="#tpg270060-bib-0071" class="usa-link" aria-describedby="tpg270060-bib-0071"><sup>2022</sup></a>). A duplicated ACCase gene (<em>AVESA.00001b.r3.6Ag0003033</em> at 420.6 Mb) is positioned nearby (∼3 Mb) QOil.fgl‐6A. ACCase catalyzes the formation of ATP‐dependent malonyl‐CoA from acetyl‐CoA and bicarbonate (Harwood, <a href="#tpg270060-bib-0032" class="usa-link" aria-describedby="tpg270060-bib-0032">1988</a>) and plays an essential role in the initial stage of fatty acid biosynthesis in oat. These two genes are likely strong influencers of oil in the oat grain, and natural variation at these loci may uncover new ways to manipulate the quantity and quality of this important nutrient.</p></section><section id="tpg270060-sec-0150"><h3 class="pmc_sec_title">3.7. QTL for test weight of hexaploid oat</h3>
<p>Nine significant markers were identified for test weight, on chromosomes 1D, 2C, 2D, 6C, 7A, 7C, and 7D (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>). The significance values ranged from 1.8 to 12.2 (−log<sub>10</sub> HBP), and the variation explained by these markers ranged from 3.4% to 13.1%. Three of the nine markers were removed because they were not durably predictive across the 10 environments tested. Stepwise regression identified the three most useful markers, QTestweight.fgl‐4D, QTestweight.fgl‐6C, and QTestweight.fgl‐7A, that explained 14.1%, 10.9%, and 4.0% of variation, respectively (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270060-fig-0003" class="usa-link">3A</a>; Table <a href="#tpg270060-supinfo-0001" class="usa-link">S7</a>; Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S6A–C</a>). Significant differences in the trait BLUE values were observed when lines were grouped by the presence or absence of a given QTL. The presence of the most predictive QTL, QTestweight.fgl‐4D, provided an approximate 4.5 kg/hL increase in test weight over its absence. A combination of the three individual QTL into haplotypes explained 20.8% of variation (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270060-fig-0003" class="usa-link">3B</a>), and an average test weight increase of 7.5–9 kg/hL when all three favorable QTL are present vs. absent within a line (Figure <a href="#tpg270060-fig-0003" class="usa-link">3A</a>).</p>
<figure class="fig xbox font-sm" id="tpg270060-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b5d4/12234416/de9e1fb361de/TPG2-18-e70060-g001.jpg" loading="lazy" id="jats-graphic-5" height="731" width="700" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270060-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distributions of three most useful quantitative trait loci (QTL) indicate significant differences using best linear unbiased estimator (BLUE) value with or without alleles of the specific markers of (A) test weight, (C) yield, (E) thins. The significance of differences between the pairwise means was determined with the Wilcox test (***<em>p</em> ≤ 0.001). Haplotype calls of collective distribution differences with or without alleles among the three most predictive QTL of (B) test weight, (D) yield, (F) thins. The significance of differences between the groups was determined with Tukey's test (<em>α</em> = 0.05).</p></figcaption></figure><p>Several test weight QTL have been detected from recombinant inbred populations in the literature (De Koeyer et al., <a href="#tpg270060-bib-0019" class="usa-link" aria-describedby="tpg270060-bib-0019">2004</a>; Tinker et al., <a href="#tpg270060-bib-0071" class="usa-link" aria-describedby="tpg270060-bib-0071"><sup>2022</sup></a>) with the most important test weight QTL being the N1 locus at 411 Mb (Sang reference; Kamal et al., <a href="#tpg270060-bib-0041" class="usa-link" aria-describedby="tpg270060-bib-0041">2022</a>) that confers the covered phenotype to the oat grain (Wang et al., <a href="#tpg270060-bib-0074" class="usa-link" aria-describedby="tpg270060-bib-0074">2023</a> and references therein). In this study, QTestweight.fgl‐4D aligned to 415 Mb on the Sang reference sequence and is also likely the N1 locus. This is interesting as the naked trials only represented 5% of the total plots tested in this study, validating the large effect of the loci. The other two QTL identified as most useful in this study appear to be novel; however, the high environmental impact on this trait suggests that several more years of data will be necessary to make an inference about a causative mechanism.</p></section><section id="tpg270060-sec-0160"><h3 class="pmc_sec_title">3.8. QTL for groat of hexaploid oat</h3>
<p>Four significant markers were identified for groat percentage, with two markers each located on chromosomes 4D and 7D (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>). Evaluation of the LD heatmaps suggested linkage block sizes of ∼10 Mb, so the four markers represented unique QTL. The variation explained by each identified SNP ranged from 4.3% to 9.4%, and marker significance values ranged from 2 to 5.7 (−log<sub>10</sub> HBP). Evaluation of the PVE within each environment showed that only a single marker (S4D_361348095) was durably predictive in the three locations it was measured in (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S7</a>). This QTL, QGroat.fgl‐4D, explained 7.1% of variation in groat with its presence predicting a 3% increase in groat content overall and within individual environments (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S7</a>; Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S6D</a>). It does not appear to overlap the N1 locus (∼90 Mb distal) or any of the other previously reported milling QTL (Esvelt Klos et al., <a href="#tpg270060-bib-0087" class="usa-link" aria-describedby="tpg270060-bib-0087">2021</a>; Tinker et al., <a href="#tpg270060-bib-0071" class="usa-link" aria-describedby="tpg270060-bib-0071"><sup>2022</sup></a>).</p></section><section id="tpg270060-sec-0170"><h3 class="pmc_sec_title">3.9. QTL for yield of hexaploid oat</h3>
<p>Seven significant markers were identified for yield, with a single SNP located on chromosomes 2D, 3C, 4C, 6C, and 7D (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>). Two markers each were identified on chromosome 4D. Significance values ranged from 3.1 to 20.9 (−log<sub>10</sub> HBP), and the variation explained by these QTL ranged from 1.2% to 43.4% (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>). Before the stepwise regression analysis to determine the three most predictive QTL, three of the significant markers were excluded because they were not durably predictive in the 11 tested environments. QYield.fgl‐4D.1, QYield.fgl‐6C, and QYield.fgl‐4D.2 were the most predictive QTL, explaining 16.7%, 7.0%, and 3.8% of variation, respectively (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270060-fig-0003" class="usa-link">3C</a>; Table <a href="#tpg270060-supinfo-0001" class="usa-link">S7</a>; Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S6G–I</a>). The most predictive QTL, QYield.fgl‐4D.1, provides an average yield increase of between 600 and 1500 kg/ha, depending on the environment (Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S6I</a>). When all three were collectively grouped into haplotypes, 20.9% of the variation was explained by the markers with a predicted increase of 800–2000 kg/ha in yield when the three QTL are present versus absent (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270060-fig-0003" class="usa-link">3D</a>).</p>
<p>The most informative yield QTL identified here, QYield.fgl‐4D.1, resides on the same linkage block as the QTestweight.fgl‐4D and the presumed N1 locus. Along with the covered (hulless, naked) phenotype, the loci determines several additional traits such as thins (%), 1000‐kernel weight, groat content, and yield (Tinker et al., <a href="#tpg270060-bib-0071" class="usa-link" aria-describedby="tpg270060-bib-0071">2022</a>). With such dramatic effects, it is possible that the causative agent underlying the N1 locus is a transcription regulator, of which four genes were identified within the QYield.fgl‐4D.1 linkage block that contain this function (AVESA.00001b.r3.4Dg000—3913, 3921, 3930, and 3935; Table <a href="#tpg270060-supinfo-0001" class="usa-link">S8</a>).</p></section><section id="tpg270060-sec-0180"><h3 class="pmc_sec_title">3.10. QTL for thins of hexaploid oat</h3>
<p>The thins metric represents the portion of kernels that fall below a size threshold determined for market desirability. In this study, seven significant molecular markers for thins were identified on four chromosomes, with three and two markers located on chromosomes 4D and 6C, respectively. A single SNP each was identified on chromosomes 3A and 4C (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>). LD analysis suggested no linkage between the three markers on 4D and the two markers on 6C, indicating the presence of distinct QTL. Significance values ranged from 3.1 to 43.0 (−log<sub>10</sub> HBP, Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>). QThins.fgl‐1D.1, QThins.fgl‐4D.2, and QThins.fgl‐6C were the three most predictive QTL identified by stepwise regression, explaining 43.9%, 34.2%, and 7.2% of variation, respectively (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270060-fig-0003" class="usa-link">3E</a>; Table <a href="#tpg270060-supinfo-0001" class="usa-link">S7</a>; Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S6J–L</a>). Three significant markers located on 3A, 4D, and 6C were removed before the stepwise regression analysis since they were not durable across the 11 environments. The most predictive QTL, QThins.fgl‐4D.1, provides an approximate average increase of 28%–50% over the absence. When grouped as a haplotype, the QTL explained a similar amount of variation (45.9%) as QThins.fgl‐4D.1 (Figure <a href="#tpg270060-fig-0003" class="usa-link">3F</a>), but predicted a larger average increase of 40%–70%% over the absence of the haplotype, suggesting that breeding efforts aiming to increase marketable yield should select for lines that do not contain these QTL. Again, the major QTL found for this trait appears to overlap the N1 locus described above, and the second most useful QTL, QThins.fgl‐4D.2, was very close by. The influence of many traits by this single locus is well characterized (Wang et al., <a href="#tpg270060-bib-0074" class="usa-link" aria-describedby="tpg270060-bib-0074">2023</a>) and is expected by the significant correlation among thins, yield, and test weight traits in this study (Figure <a href="#tpg270060-fig-0001" class="usa-link">1</a>).</p></section><section id="tpg270060-sec-0190"><h3 class="pmc_sec_title">3.11. QTL for heading date of hexaploid oat</h3>
<p>Eighteen significant markers were identified for heading date across 11 chromosomes. Chromosomes 1D, 2A, 3C, 4D, 5A, 5C, and 6C each contained a single marker, while chromosomes 4A, 5A, and 6A each contained two markers, and five markers were identified on 7D (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>). Significant values ranged from 1.4 to 12 (−log<sub>10</sub> HBP), and the variation explained by these markers ranged from 3.0% to 11.6%. QHeadingdate.fgl‐6A, QHeadingdate.fgl‐6C, and QHeadingdate.fgl‐1D were identified by stepwise regression as the most predictive QTL, explaining 8.2%, 4.0%, and 5.5% of variation, respectively, after five markers were removed due to lack of durability across environments (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270060-fig-0004" class="usa-link">4A</a>; Table <a href="#tpg270060-supinfo-0001" class="usa-link">S7</a>; Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S7A–C</a>). Significant differences in the trait BLUE values were identified when lines were grouped by the presence or absence of a specified QTL. Presence of the most predictive QTL, QHeadingdate.fgl‐6A, increased the days until heading by approximately 5–6 days. Combining the three individual QTL into haplotypes explained 16% of the variation (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>), and presence of all three QTL in a haplotype predicted an average increase of 12–16 days to heading over the absence of the haplotype (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270060-fig-0004" class="usa-link">4B</a>), depending on the environment (Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S7A–C</a>).</p>
<figure class="fig xbox font-sm" id="tpg270060-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b5d4/12234416/9114e4eaf478/TPG2-18-e70060-g004.jpg" loading="lazy" id="jats-graphic-7" height="486" width="700" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270060-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distributions of three most useful quantitative trait loci (QTL) indicate significant differences using best linear unbiased estimator (BLUE) value with or without alleles of the specific markers of (A) heading date and (C) height. The significance of differences between the pairwise means was determined with the Wilcox test (***<em>p</em> ≤ 0.001). Haplotype calls of collective distribution differences with or without alleles among the three most predictive QTL of (B) heading date and (D) height. The significance of differences between the groups was determined with Tukey's test (<em>α</em> = 0.05).</p></figcaption></figure><p>Heading date has been the focus of several QTL mapping studies. Zimmer et al. (<a href="#tpg270060-bib-0085" class="usa-link" aria-describedby="tpg270060-bib-0085">2018</a>) identified genomic regions controlling heading date in two hulled by naked oat populations. One significant SNP marker, GMI_ES15_c15940_121, aligned to 6A:419 Mb, very close to the linkage block surrounding QHeadingdate.fgl‐6A. A recent report characterized several heading date–related QTL using two biparental mapping populations (de Ubert &amp; Nava, <a href="#tpg270060-bib-0020" class="usa-link" aria-describedby="tpg270060-bib-0020"><sup>2024</sup></a>) and identified a QTL on 6A (Mrg05) that overlaps with QHeadingdate.fgl‐6A. Underlying this QTL, two annotated genes contain flowering time functions–the PSEUDO RESPONSE REGULATOR‐like (PRR37, AVESA.00001b.r3.6Ag0002826 @ 6A:405.7 Mb) and a Putative F‐box/FBD/LRR‐repeat protein (FDL24, AVESA.00001b.r3.6Ag0002728 @ 6A:397 Mb)–and are likely playing influential roles in this population. Another heading date QTL discovered in this study, QHeadingdate.fgl‐1D, is also nearby several QTL found in the literature (Canales et al., <a href="#tpg270060-bib-0016" class="usa-link" aria-describedby="tpg270060-bib-0016">2021</a>; de Ubert &amp; Nava, <a href="#tpg270060-bib-0020" class="usa-link" aria-describedby="tpg270060-bib-0020"><sup>2024</sup></a>; Siripoonwiwat et al., <a href="#tpg270060-bib-0066" class="usa-link" aria-describedby="tpg270060-bib-0066"><sup>1996</sup></a>) with the FRIGIDA‐like 3 (FRL3, AVESA.00001b.r3.1Dg0001847 @ 1D:347.6 Mb) gene as a potential causative agent.</p></section><section id="tpg270060-sec-0200"><h3 class="pmc_sec_title">3.12. QTL for height of hexaploid oat</h3>
<p>Five markers were identified for height across chromosomes 4D, 6A, and 6C (Table <a href="#tpg270060-supinfo-0001" class="usa-link">S6</a>). Chromosome 6D had two markers identified on it, and LD analysis of the genotyping data suggested no linkage among those two molecular markers. Significance values of all identified markers ranged from 1.3 to 2.3 (−log<sub>10</sub> HBP), and the PVE ranged from 4.2% to 7.0%. All the significant markers were found to be durably predictive in the six environments tested. The three most predictive QTL identified by stepwise regression from this data set were named QHeight.fgl‐6D, QHeight.fgl‐6A, and QHeight.fgl‐4D, explaining 7%, 6%, and 5.5% of variation, respectively (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270060-fig-0004" class="usa-link">4C</a>; Table <a href="#tpg270060-supinfo-0001" class="usa-link">S7</a>; Figure <a href="#tpg270060-supinfo-0001" class="usa-link">S7D–F</a>). All three QTL exhibited the same effect, with the presence of QTL, increasing the height of the line approximately 2 cm over its absence (Table <a href="#tpg270060-tbl-0003" class="usa-link">3</a>). Combining the three most predictive QTL into haplotypes resulted in a PVE of 15.9% (Figure <a href="#tpg270060-fig-0004" class="usa-link">4D</a>). Shorter plants are usually desirable in high‐production regions to limit lodging at the end of the growing season; thus, breeders selecting for the lines without these three QTL should expect an average decrease in height of approximately 6 cm.</p>
<p>The QTL identified in this study (QHeight.fgl‐6D) located on 6D from 278 to 291 Mb are close to several other previously published QTL for height. Zimmer et al. (<a href="#tpg270060-bib-0085" class="usa-link" aria-describedby="tpg270060-bib-0085">2018</a>) identified a plant height QTL flanked by markers GMI_ES02_c1838_173 and GMI_GBS_13021 that align to 6D:222 and 6D:230 Mb, respectively. These QTL may be linked to the dwarfing gene, <em>Dw‐6</em>, mapped to chromosome 6D:237 Mb (Yan et al., <a href="#tpg270060-bib-0077" class="usa-link" aria-describedby="tpg270060-bib-0077">2021</a>). A recent publication also identified several plant height QTL from a Dal/Exeter mapping population that are associated with marker avgbs_13292.1 and contain the 244–270 Mb region of Sang chromosome 6D (Tinker et al., <a href="#tpg270060-bib-0071" class="usa-link" aria-describedby="tpg270060-bib-0071">2022</a>).</p></section></section><section id="tpg270060-sec-0210"><h2 class="pmc_sec_title">4. CONCLUSION</h2>
<p>The simultaneous improvement of many different crop characteristics is a laborious and time‐consuming task for small grains breeding programs. Due to these difficulties, many traits necessary for cultivar development are not directly evaluated until germplasm is well‐advanced and evaluated in multiple field locations. When direct phenotyping cannot take place, early genotyping with predictive molecular markers can enhance breeding efficiency by enabling the early triage of lines predicted to possess undesirable traits. Depending on the molecular breeding scheme employed, larger pools of early‐generation material can be assessed. More stringent selection criteria can also be used, or both methods can be employed to increase the speed and efficiency of bringing new cultivars to the market. The durable predictive markers we have identified in this study will be useful for facilitating these goals in oat breeding programs. Identification of genes located near the QTL for each studied trait also has provided a starting point for future functional molecular studies.</p></section><section id="tpg270060-sec-0220"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Naa Korkoi Ardayfio</strong>: Data curation; formal analysis; investigation; methodology; project administration; writing—original draft; writing—review and editing. <strong>Christy Kinney</strong>: Visualization; writing—original draft; writing—review and editing. <strong>Dandan Li</strong>: Investigation; writing—original draft; writing—review and editing. <strong>Sepehr Mohajeri Naraghi</strong>: Data curation; methodology; resources; writing—review and editing. <strong>Michael S. McMullen</strong>: Methodology; project administration; resources; writing—review and editing. <strong>Jason D. Fiedler</strong>: Conceptualization; data curation; funding acquisition; investigation; project administration; supervision; writing—review and editing.</p></section><section id="tpg270060-sec-0240"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec24"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270060-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270060-supitem-0001"><div class="caption p">
<p>
<strong>Supplementary Table 1</strong> contains PYT/AYT specific and unique genotype number with traits measured, locations and replications across years.</p>
<p>
<strong>Supplementary Table 2</strong> contains trait data used in this study.</p>
<p>
<strong>Supplementary Table 3</strong> contains repeated genotypes across experimental trials and years.</p>
<p>
<strong>Supplementary Table 4</strong> contains spatially adjusted means for each environment.</p>
<p>
<strong>Supplementary Table 5</strong> contains normalized BLUE values (check values excluded prior to association analysis).</p>
<p>
<strong>Supplementary Table 6</strong> contains relevant information for all markers identified with GWAS.</p>
<p>
<strong>Supplementary Table 7</strong> contains Significant markers per‐environment PVE markers before stepwise regression analysis.</p>
<p>
<strong>Supplementary Table 8</strong> Annotated genes within QTL linkage blocks.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s001.xlsx</a><sup> (3.3MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270060-supitem-0002"><div class="caption p"><p>
<strong>Supplementary Figure 1</strong> contains the distribution of spatially adjusted environment means for β‐Glucan (A), Protein (B), Oil (C), Test weight (D), Groat (E), Yield (F), Thins (G), Heading date (H) and Height (I).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s008.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s008.pdf</a><sup> (33.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270060-supitem-0003"><div class="caption p"><p>
<strong>Supplementary Figure 2</strong> contains the principal component analysis of the taxa used in this study.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s004.pdf</a><sup> (40.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270060-supitem-0004"><div class="caption p"><p>
<strong>Supplementary Figure 3</strong> contains the GWAS results visualized with QQ and Manhattan plots for β‐Glucan (A,B), Protein (C,D), Oil (E,F), Test weight (G,H), Groat (I,J), Yield (K,L), Thins (M,N), Heading date (O,P) and Height (Q,R).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s006.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s006.pdf</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270060-supitem-0005"><div class="caption p"><p>
<strong>Supplementary Figure 4</strong> contains the distribution of normalized BLUE values grouped by allele state.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s007.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s007.pdf</a><sup> (4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270060-supitem-0006"><div class="caption p"><p>
<strong>Supplementary Figure 5</strong> contains the distribution of spatially adjusted environment means grouped by the three most significant QTL for β‐Glucan (A‐C), Protein (D‐F), and Oil (G‐I).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s005.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s005.pdf</a><sup> (13.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270060-supitem-0007"><div class="caption p"><p>
<strong>Supplementary Figure 6</strong> contains the distribution of spatially adjusted environment means grouped by the three most significant QTL for Test weight (A‐C), Groat (D‐F), Yield (G‐I), and Thins (J‐L).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s003.pdf</a><sup> (52.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270060-supitem-0008"><div class="caption p"><p>
<strong>Supplementary Figure 7</strong> contains the distribution of spatially adjusted environment means grouped by the three most significant QTL for Heading Date (A‐C) and Height (D‐F).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s002.pdf</a><sup> (19.3KB, pdf) </sup>
</div></div></section></section><section id="tpg270060-sec-0230" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank Mary Osenga and Terrance Peterson at the USDA‐ARS North Central Small Grains Genotyping Laboratory for assistance with genotyping and the entire North Dakota State University Oat Breeding Program Team for field operations. We also would like to thank Massub Tehseen for assistance with spatial correction and normalization of the unbalanced field data. This research was supported by USDA Agricultural Research Service projects number 3060‐21000‐046‐000D, the ARS Research Associate Program class of 2022, and National Institute of Food and Agriculture, Grants.gov No. GRANT13057210. Mention of trade names or commercial products in this publication is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the U.S. Department of Agriculture. USDA is an equal opportunity provider and employer.</p></section><section id="notes1"><p>


Ardayfio, N. K.
, 
Kinney, C.
, 
Li, D.
, 
Naraghi, S. M.
, 
McMullen, M. S.
, &amp; 
Fiedler, J. D.
 (2025). Genome‐wide association studies reveal genetic control of nutritional quality, milling traits, and agronomic characteristics in oat (<em>Avena sativa</em> L.). The Plant Genome, 18, e70060. 10.1002/tpg2.70060


</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270060-note-0001"><p>Assigned to Associate Editor Kulvinder Gill.</p></div></div></section></section><section id="tpg270060-sec-0260"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>Genotyping data have been deposited in the T3/Oat database under the “NDSU GBS” genotyping protocol. Phenotyping data can be found in Table <a href="#tpg270060-supinfo-0001" class="usa-link">S2</a>.</p></section><section id="tpg270060-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270060-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270060-bib-0001">
<cite>

Abecasis, G. R.
, 
Noguchi, E.
, 
Heinzmann, A.
, 
Traherne, J. A.
, 
Bhattacharyya, S.
, 
Leaves, N. I.
, 
Anderson, G. G.
, 
Zhang, Y.
, 
Lench, N. J.
, 
Carey, A.
, 
Cardon, L. R.
, 
Moffatt, M. F.
, &amp; 
Cookson, W. O. C.
 (2001). Extent and distribution of linkage disequilibrium in three genomic regions. American Journal of Human Genetics, 68(1), 191–197. 10.1086/316944

</cite> [<a href="https://doi.org/10.1086/316944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1234912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11083947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Extent%20and%20distribution%20of%20linkage%20disequilibrium%20in%20three%20genomic%20regions&amp;author=G.%20R.%20Abecasis&amp;author=E.%20Noguchi&amp;author=A.%20Heinzmann&amp;author=J.%20A.%20Traherne&amp;author=S.%20Bhattacharyya&amp;volume=68&amp;issue=1&amp;publication_year=2001&amp;pages=191-197&amp;pmid=11083947&amp;doi=10.1086/316944&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0002">
<cite>

Asoro, F. G.
, 
Newell, M. A.
, 
Scott, M. P.
, 
Beavis, W. D.
, &amp; 
Jannink, J.
 (2013). Genome‐wide association study for beta‐glucan concentration in elite North American oat. Crop Science, 53, 542–553. 10.2135/cropsci2012.01.0039
</cite> [<a href="https://doi.org/10.2135/cropsci2012.01.0039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genome%E2%80%90wide%20association%20study%20for%20beta%E2%80%90glucan%20concentration%20in%20elite%20North%20American%20oat&amp;author=F.%20G.%20Asoro&amp;author=M.%20A.%20Newell&amp;author=M.%20P.%20Scott&amp;author=W.%20D.%20Beavis&amp;author=J.%20Jannink&amp;volume=53&amp;publication_year=2013&amp;pages=542-553&amp;doi=10.2135/cropsci2012.01.0039&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0003">
<cite>

Banaś, K.
, &amp; 
Harasym, J.
 (2021). Current knowledge of content and composition of oat oil—Future perspectives of oat as oil source. Food Bioprocess Technology, 14, 232–247. 10.1007/s11947-020-02535-5
</cite> [<a href="https://doi.org/10.1007/s11947-020-02535-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Bioprocess%20Technology&amp;title=Current%20knowledge%20of%20content%20and%20composition%20of%20oat%20oil%E2%80%94Future%20perspectives%20of%20oat%20as%20oil%20source&amp;author=K.%20Bana%C5%9B&amp;author=J.%20Harasym&amp;volume=14&amp;publication_year=2021&amp;pages=232-247&amp;doi=10.1007/s11947-020-02535-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0005">
<cite>

Bazzer, S. K.
, 
Oliveira, G.
, 
Fiedler, J. D.
, 
Nandety, R. S.
, 
Jannink, J.‐L.
, &amp; 
Caffe, M.
 (2025). Genomic strategies to facilitate breeding for increased β‐glucan content in oat (<em>Avena sativa</em> L.). BMC Genomics, 26, Article 35. 10.1186/s12864-024-11174-5

</cite> [<a href="https://doi.org/10.1186/s12864-024-11174-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11731539/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39810135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Genomic%20strategies%20to%20facilitate%20breeding%20for%20increased%20%CE%B2%E2%80%90glucan%20content%20in%20oat%20(Avena%20sativa%20L.)&amp;author=S.%20K.%20Bazzer&amp;author=G.%20Oliveira&amp;author=J.%20D.%20Fiedler&amp;author=R.%20S.%20Nandety&amp;author=J.%E2%80%90L.%20Jannink&amp;volume=26&amp;publication_year=2025&amp;pages=35&amp;pmid=39810135&amp;doi=10.1186/s12864-024-11174-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0006">
<cite>

Bevan, M. W.
, &amp; 
Uauy, C.
 (2013). Genomics reveals new landscapes for crop improvement. Genome Biology, 14(6), 206–217. 10.1186/gb-2013-14-6-206

</cite> [<a href="https://doi.org/10.1186/gb-2013-14-6-206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3706852/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23796126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Genomics%20reveals%20new%20landscapes%20for%20crop%20improvement&amp;author=M.%20W.%20Bevan&amp;author=C.%20Uauy&amp;volume=14&amp;issue=6&amp;publication_year=2013&amp;pages=206-217&amp;pmid=23796126&amp;doi=10.1186/gb-2013-14-6-206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0007">
<cite>

Bowles, T. M.
, 
Mooshammer, M.
, 
Socolar, Y.
, 
Calderón, F.
, 
Cavigelli, M. A.
, 
Culman, S. W.
, 
Deen, W.
, 
Drury, C. F.
, 
Garcia, A. G.
, 
Gaudin, A. C. M.
, 
Harkcom, W. S.
, 
Lehman, R. M.
, 
Osborne, S. L.
, 
Robertson, G. P.
, 
Salerno, J.
, 
Schmer, M. R.
, 
Strock, J.
, &amp; 
Grandy, A. S.
 (2020). Long‐term evidence shows that crop‐rotation diversification increases agricultural resilience to adverse growing conditions in North America. One Earth, 2(3), 284–293. 10.1016/j.oneear.2020.02.007
</cite> [<a href="https://doi.org/10.1016/j.oneear.2020.02.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=One%20Earth&amp;title=Long%E2%80%90term%20evidence%20shows%20that%20crop%E2%80%90rotation%20diversification%20increases%20agricultural%20resilience%20to%20adverse%20growing%20conditions%20in%20North%20America&amp;author=T.%20M.%20Bowles&amp;author=M.%20Mooshammer&amp;author=Y.%20Socolar&amp;author=F.%20Calder%C3%B3n&amp;author=M.%20A.%20Cavigelli&amp;volume=2&amp;issue=3&amp;publication_year=2020&amp;pages=284-293&amp;doi=10.1016/j.oneear.2020.02.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0008">
<cite>

Braaten, T. J.
, 
Wood, P. J.
, 
Scott, F. W.
, 
Wolynetz, M. S.
, 
Lowe, M. K.
, 
Bradley‐Whyte, P.
, &amp; 
Collins, M. W.
 (1994). Oat β‐glucan reduces blood cholesterol concentration in hypercholesterolemic subjects. European Journal of Clinical Nutrition, 48(7), 465–474.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/7956987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Clinical%20Nutrition&amp;title=Oat%20%CE%B2%E2%80%90glucan%20reduces%20blood%20cholesterol%20concentration%20in%20hypercholesterolemic%20subjects&amp;author=T.%20J.%20Braaten&amp;author=P.%20J.%20Wood&amp;author=F.%20W.%20Scott&amp;author=M.%20S.%20Wolynetz&amp;author=M.%20K.%20Lowe&amp;volume=48&amp;issue=7&amp;publication_year=1994&amp;pages=465-474&amp;pmid=7956987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0009">
<cite>

Brachi, B.
, 
Morris, G. P.
, &amp; 
Borevitz, J. O.
 (2011). Genome‐wide association studies in plants: The missing heritability is in the field. Genome Biology, 12, 232–240. 10.1186/gb-2011-12-10-232

</cite> [<a href="https://doi.org/10.1186/gb-2011-12-10-232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3333769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22035733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Genome%E2%80%90wide%20association%20studies%20in%20plants:%20The%20missing%20heritability%20is%20in%20the%20field&amp;author=B.%20Brachi&amp;author=G.%20P.%20Morris&amp;author=J.%20O.%20Borevitz&amp;volume=12&amp;publication_year=2011&amp;pages=232-240&amp;pmid=22035733&amp;doi=10.1186/gb-2011-12-10-232&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0010">
<cite>

Bradbury, P. J.
, 
Zhang, Z.
, 
Kroon, D. E.
, 
Casstevens, T. M.
, 
Ramdoss, Y.
, &amp; 
Buckler, E. S.
 (2007). TASSEL: Software for association mapping of complex traits in diverse samples. Bioinformatics, 23(19), 2633–2635. 10.1093/bioinformatics/btm308

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btm308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17586829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=TASSEL:%20Software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&amp;author=P.%20J.%20Bradbury&amp;author=Z.%20Zhang&amp;author=D.%20E.%20Kroon&amp;author=T.%20M.%20Casstevens&amp;author=Y.%20Ramdoss&amp;volume=23&amp;issue=19&amp;publication_year=2007&amp;pages=2633-2635&amp;pmid=17586829&amp;doi=10.1093/bioinformatics/btm308&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0011">
<cite>

Brown, C. M.
, 
Alexander, D. E.
, &amp; 
Carmer, S. G.
 (1966). Variation in oil content and its relation to other characters in oats (<em>Avena sativa</em> L.). Crop Science, 6, 190–191. 10.2135/cropsci1966.0011183X000600020025x
</cite> [<a href="https://doi.org/10.2135/cropsci1966.0011183X000600020025x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Variation%20in%20oil%20content%20and%20its%20relation%20to%20other%20characters%20in%20oats%20(Avena%20sativa%20L.)&amp;author=C.%20M.%20Brown&amp;author=D.%20E.%20Alexander&amp;author=S.%20G.%20Carmer&amp;volume=6&amp;publication_year=1966&amp;pages=190-191&amp;doi=10.2135/cropsci1966.0011183X000600020025x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0012">
<cite>

Brown, C. M.
, &amp; 
Craddock, J. C.
 (1972). Oil content and groat weight of entries in the world oat collection. Crop Science, 12, 514–515. 10.2135/cropsci1972.0011183X001200040038x
</cite> [<a href="https://doi.org/10.2135/cropsci1972.0011183X001200040038x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Oil%20content%20and%20groat%20weight%20of%20entries%20in%20the%20world%20oat%20collection&amp;author=C.%20M.%20Brown&amp;author=J.%20C.%20Craddock&amp;volume=12&amp;publication_year=1972&amp;pages=514-515&amp;doi=10.2135/cropsci1972.0011183X001200040038x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0013">
<cite>

Burton, R. A.
, 
Wilson, S. M.
, 
Hrmova, M.
, 
Harvey, A. J.
, 
Shirley, N. J.
, 
Medhurst, A.
, 
Stone, B. A.
, 
Newbigin, E. J.
, 
Bacic, A.
, &amp; 
Fincher, G. B.
 (2006). Cellulose synthase‐like <em>CslF</em> genes mediate the synthesis of cell wall (1,3;1,4)‐ß‐d‐glucans. Science, 311(5769), 1940–1942. 10.1126/science.1122975

</cite> [<a href="https://doi.org/10.1126/science.1122975" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16574868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Cellulose%20synthase%E2%80%90like%20CslF%20genes%20mediate%20the%20synthesis%20of%20cell%20wall%20(1,3;1,4)%E2%80%90%C3%9F%E2%80%90d%E2%80%90glucans&amp;author=R.%20A.%20Burton&amp;author=S.%20M.%20Wilson&amp;author=M.%20Hrmova&amp;author=A.%20J.%20Harvey&amp;author=N.%20J.%20Shirley&amp;volume=311&amp;issue=5769&amp;publication_year=2006&amp;pages=1940-1942&amp;pmid=16574868&amp;doi=10.1126/science.1122975&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0014">
<cite>

Butt, M. S.
, 
Tahir‐Nadeem, M.
, 
Khan, M. K. I.
, 
Shabir, R.
, &amp; 
Butt, M. S.
 (2008). Oat: Unique among the cereals. European Journal of Nutrition, 47(2), 68–79. 10.1007/s00394-008-0698-7

</cite> [<a href="https://doi.org/10.1007/s00394-008-0698-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18301937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Nutrition&amp;title=Oat:%20Unique%20among%20the%20cereals&amp;author=M.%20S.%20Butt&amp;author=M.%20Tahir%E2%80%90Nadeem&amp;author=M.%20K.%20I.%20Khan&amp;author=R.%20Shabir&amp;author=M.%20S.%20Butt&amp;volume=47&amp;issue=2&amp;publication_year=2008&amp;pages=68-79&amp;pmid=18301937&amp;doi=10.1007/s00394-008-0698-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0015">
<cite>

Campbell, M. T.
, 
Hu, H.
, 
Yeats, T. H.
, 
Caffe‐Treml, M.
, 
Gutiérrez, L.
, 
Smith, K. P.
, 
Sorrells, M. E.
, 
Gore, M. A.
, &amp; 
Jannink, J.‐L.
 (2021). Translating insights from the seed metabolome into improved prediction for lipid‐composition traits in oat (<em>Avena sativa</em> L.). Genetics, 217(3), iyaa043. 10.1093/genetics/iyaa043
</cite> [<a href="https://doi.org/10.1093/genetics/iyaa043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8045723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33789350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Translating%20insights%20from%20the%20seed%20metabolome%20into%20improved%20prediction%20for%20lipid%E2%80%90composition%20traits%20in%20oat%20(Avena%20sativa%20L.)&amp;author=M.%20T.%20Campbell&amp;author=H.%20Hu&amp;author=T.%20H.%20Yeats&amp;author=M.%20Caffe%E2%80%90Treml&amp;author=L.%20Guti%C3%A9rrez&amp;volume=217&amp;issue=3&amp;publication_year=2021&amp;pmid=33789350&amp;doi=10.1093/genetics/iyaa043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0016">
<cite>

Canales, F. J.
, 
Montilla‐Bascón, G.
, 
Bekele, W. A.
, 
Howarth, C. J.
, 
Langdon, T.
, 
Rispail, N.
, 
Tinker, N. E.
, &amp; 
Prats, E.
 (2021). Population genomics of Mediterranean oat (<em>A. sativa</em>) reveals high genetic diversity and three loci for heading date. Theoretical and Applied Genetics, 134, 2063–2077. 10.1007/s00122-021-03805-2

</cite> [<a href="https://doi.org/10.1007/s00122-021-03805-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8263550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33770189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Population%20genomics%20of%20Mediterranean%20oat%20(A.%20sativa)%20reveals%20high%20genetic%20diversity%20and%20three%20loci%20for%20heading%20date&amp;author=F.%20J.%20Canales&amp;author=G.%20Montilla%E2%80%90Basc%C3%B3n&amp;author=W.%20A.%20Bekele&amp;author=C.%20J.%20Howarth&amp;author=T.%20Langdon&amp;volume=134&amp;publication_year=2021&amp;pages=2063-2077&amp;pmid=33770189&amp;doi=10.1007/s00122-021-03805-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0086">
<cite>

Carlson, C. H.
, 
Fiedler, J. D.
, 
Naraghi, S. M.
, 
Nazareno, E. S.
, 
Ardayfio, N. K.
, 
McMullen, M. S.
, &amp; 
Kianian, S. F.
 (2023). Archetypes of inflorescence: genome‐wide association networks of panicle morphometric, growth, and disease variables in a multiparent oat population. Genetics, 223(2). 10.1093/genetics/iyac128
</cite> [<a href="https://doi.org/10.1093/genetics/iyac128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9910404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36106985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Archetypes%20of%20inflorescence:%20genome%E2%80%90wide%20association%20networks%20of%20panicle%20morphometric,%20growth,%20and%20disease%20variables%20in%20a%20multiparent%20oat%20population&amp;author=C.%20H.%20Carlson&amp;author=J.%20D.%20Fiedler&amp;author=S.%20M.%20Naraghi&amp;author=E.%20S.%20Nazareno&amp;author=N.%20K.%20Ardayfio&amp;volume=223&amp;issue=2&amp;publication_year=2023&amp;pmid=36106985&amp;doi=10.1093/genetics/iyac128&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0019">
<cite>

De Koeyer, D. L.
, 
Tinker, N. A.
, 
Wight, C. P.
, 
Deyl, J.
, 
Burrows, V. D.
, 
O'Donoughue, L. S.
, 
Lybaert, A.
, 
Molnar, S. J.
, 
Armstrong, K. C.
, 
Fedak, G.
, 
Wesenberg, D. M.
, 
Rossnagel, B. G.
, &amp; 
McElroy, A. R.
 (2004). A molecular linkage map with associated QTLs from a hulless × covered spring oat population. Theoretical and Applied Genetics, 108, 1285–1298. 10.1007/s00122-003-1556-x

</cite> [<a href="https://doi.org/10.1007/s00122-003-1556-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14767596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=A%20molecular%20linkage%20map%20with%20associated%20QTLs%20from%20a%20hulless%20%C3%97%20covered%20spring%20oat%20population&amp;author=D.%20L.%20De%20Koeyer&amp;author=N.%20A.%20Tinker&amp;author=C.%20P.%20Wight&amp;author=J.%20Deyl&amp;author=V.%20D.%20Burrows&amp;volume=108&amp;publication_year=2004&amp;pages=1285-1298&amp;pmid=14767596&amp;doi=10.1007/s00122-003-1556-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0020">
<cite>

de Ubert, I. P.
, &amp; 
Nava, I. C.
 (2024). Correction: Genetic mapping and comparative analysis of heading date in hexaploid oat. Euphytica, 220, Article 189. 10.1007/s10681-024-03442-y
</cite> [<a href="https://doi.org/10.1007/s10681-024-03442-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Correction:%20Genetic%20mapping%20and%20comparative%20analysis%20of%20heading%20date%20in%20hexaploid%20oat&amp;author=I.%20P.%20de%20Ubert&amp;author=I.%20C.%20Nava&amp;volume=220&amp;publication_year=2024&amp;pages=189&amp;doi=10.1007/s10681-024-03442-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0021">
<cite>

Doehlert, D. C.
, 
McMullen, M. S.
, &amp; 
Jannink, J.‐L.
 (2006). Oat grain/groat size ratios: A physical basis for test weight. Cereal Chemistry, 83(1), 114–118. 10.1094/CC-83-0114
</cite> [<a href="https://doi.org/10.1094/CC-83-0114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cereal%20Chemistry&amp;title=Oat%20grain/groat%20size%20ratios:%20A%20physical%20basis%20for%20test%20weight&amp;author=D.%20C.%20Doehlert&amp;author=M.%20S.%20McMullen&amp;author=J.%E2%80%90L.%20Jannink&amp;volume=83&amp;issue=1&amp;publication_year=2006&amp;pages=114-118&amp;doi=10.1094/CC-83-0114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0022">
<cite>

Estrada, A.
, 
Yun, C.
, 
Kessel, A. V.
, 
Li, B.
, 
Hauta, S.
, &amp; 
Laarveld, B.
 (1997). Immunomodulatory activities of oat β‐glucan in vitro and in vivo. Microbiology Immunology, 41, 991–998. 10.1111/j.1348-0421.1997.tb01959.x

</cite> [<a href="https://doi.org/10.1111/j.1348-0421.1997.tb01959.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9492185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology%20Immunology&amp;title=Immunomodulatory%20activities%20of%20oat%20%CE%B2%E2%80%90glucan%20in%20vitro%20and%20in%20vivo&amp;author=A.%20Estrada&amp;author=C.%20Yun&amp;author=A.%20V.%20Kessel&amp;author=B.%20Li&amp;author=S.%20Hauta&amp;volume=41&amp;publication_year=1997&amp;pages=991-998&amp;pmid=9492185&amp;doi=10.1111/j.1348-0421.1997.tb01959.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0087">
<cite>

Esvelt Klos, K.
, 
Yimer, B. A.
, 
Howarth, C. J.
, 
McMullen, M. S.
, 
Sorrells, M. E.
, 
Tinker, N. A.
, 
Yan, W.
, &amp; 
Beattie, A. D.
 (2021). The genetic architecture of milling quality in spring oat lines of the collaborative oat research enterprise. Foods, 10(10), 2479. 10.3390/foods10102479

</cite> [<a href="https://doi.org/10.3390/foods10102479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8535619/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34681528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Foods&amp;title=The%20genetic%20architecture%20of%20milling%20quality%20in%20spring%20oat%20lines%20of%20the%20collaborative%20oat%20research%20enterprise&amp;author=K.%20Esvelt%20Klos&amp;author=B.%20A.%20Yimer&amp;author=C.%20J.%20Howarth&amp;author=M.%20S.%20McMullen&amp;author=M.%20E.%20Sorrells&amp;volume=10&amp;issue=10&amp;publication_year=2021&amp;pages=2479&amp;pmid=34681528&amp;doi=10.3390/foods10102479&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0023">
<cite>

Faust, G. G.
, &amp; 
Hall, I. M.
 (2014). SAMBLASTER: Fast duplicate marking and structural variant read extraction. Bioinformatics, 30(17), 2503–2505. 10.1093/bioinformatics/btu314

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4147885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24812344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SAMBLASTER:%20Fast%20duplicate%20marking%20and%20structural%20variant%20read%20extraction&amp;author=G.%20G.%20Faust&amp;author=I.%20M.%20Hall&amp;volume=30&amp;issue=17&amp;publication_year=2014&amp;pages=2503-2505&amp;pmid=24812344&amp;doi=10.1093/bioinformatics/btu314&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0024">
<cite>
FDA
. (2018). Review of the scientific evidence on the physiological effects of certain non‐digestible carbohydrates. Office of Nutrition and Food Labeling Center for Food Safety and Applied Nutrition Food and Drug Administration U.S. Department of Health and Human Services. <a href="https://www.fda.gov/media/113659/download" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/media/113659/download</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Review%20of%20the%20scientific%20evidence%20on%20the%20physiological%20effects%20of%20certain%20non%E2%80%90digestible%20carbohydrates&amp;publication_year=2018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0025">
<cite>

Fincher, G. B.
 (2009). Exploring the evolution of (1,3;1,4)‐ß‐d‐glucans in plant cell walls: Comparative genomics can help!
Current Opinion in Plant Biology, 12(2), 140–147. 10.1016/j.pbi.2009.01.002

</cite> [<a href="https://doi.org/10.1016/j.pbi.2009.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19168383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Plant%20Biology&amp;title=Exploring%20the%20evolution%20of%20(1,3;1,4)%E2%80%90%C3%9F%E2%80%90d%E2%80%90glucans%20in%20plant%20cell%20walls:%20Comparative%20genomics%20can%20help!&amp;author=G.%20B.%20Fincher&amp;volume=12&amp;issue=2&amp;publication_year=2009&amp;pages=140-147&amp;pmid=19168383&amp;doi=10.1016/j.pbi.2009.01.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0026">
<cite>

Fogarty, M. C.
, 
Smith, S. M.
, 
Sheridan, J. L.
, 
Hu, G.
, 
Islamovic, E.
, 
Reid, R.
, 
Jackson, E. W.
, 
Maughan, P. J.
, 
Ames, N. P.
, 
Jellen, E. N.
, &amp; 
Hsieh, T.
 (2019). Identification of mixed linkage β‐glucan quantitative trait loci and evaluation of AsCslF6 homoeologs in hexaploid oat. Crop Science, 60(2), 914–933. 10.1002/csc2.20015
</cite> [<a href="https://doi.org/10.1002/csc2.20015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Identification%20of%20mixed%20linkage%20%CE%B2%E2%80%90glucan%20quantitative%20trait%20loci%20and%20evaluation%20of%20AsCslF6%20homoeologs%20in%20hexaploid%20oat&amp;author=M.%20C.%20Fogarty&amp;author=S.%20M.%20Smith&amp;author=J.%20L.%20Sheridan&amp;author=G.%20Hu&amp;author=E.%20Islamovic&amp;volume=60&amp;issue=2&amp;publication_year=2019&amp;pages=914-933&amp;doi=10.1002/csc2.20015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0027">
<cite>

Frey, K. J.
 (1951). The relation between alcohol‐soluble and total nitrogen contents of oats. Cereal Chemistry, 28, 506–509.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Cereal%20Chemistry&amp;title=The%20relation%20between%20alcohol%E2%80%90soluble%20and%20total%20nitrogen%20contents%20of%20oats&amp;author=K.%20J.%20Frey&amp;volume=28&amp;publication_year=1951&amp;pages=506-509&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0028">
<cite>

Goddard, M. E.
, &amp; 
Hayes, B. J.
 (2007). Genomic selection. Journal of Animal Breeding and Genetics, 124, 323–330. 10.1111/j.1439-0388.2007.00702.x

</cite> [<a href="https://doi.org/10.1111/j.1439-0388.2007.00702.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18076469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Animal%20Breeding%20and%20Genetics&amp;title=Genomic%20selection&amp;author=M.%20E.%20Goddard&amp;author=B.%20J.%20Hayes&amp;volume=124&amp;publication_year=2007&amp;pages=323-330&amp;pmid=18076469&amp;doi=10.1111/j.1439-0388.2007.00702.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0029">
<cite>

Grundy, M. M.‐L.
, 
Fardet, A.
, 
Tosh, S. M.
, 
Rich, G. T.
, &amp; 
Wilde, P. J.
 (2018). Processing of oat: The impact on oat's cholesterol lowering effect. Food &amp; Function, 9, 1328–1343. 10.1039/C7FO02006F

</cite> [<a href="https://doi.org/10.1039/C7FO02006F" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5885279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29431835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20&amp;%20Function&amp;title=Processing%20of%20oat:%20The%20impact%20on%20oat's%20cholesterol%20lowering%20effect&amp;author=M.%20M.%E2%80%90L.%20Grundy&amp;author=A.%20Fardet&amp;author=S.%20M.%20Tosh&amp;author=G.%20T.%20Rich&amp;author=P.%20J.%20Wilde&amp;volume=9&amp;publication_year=2018&amp;pages=1328-1343&amp;pmid=29431835&amp;doi=10.1039/C7FO02006F&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0030">
<cite>

Halima, N. B.
, 
Saad, R. B.
, 
Khemakhem, B.
, 
Fendri, I.
, &amp; 
Abdelkafi, S.
 (2015). Oat (<em>Avena sativa</em> L.): Oil and nutriment compounds valorization for potential use in industrial applications. Journal of Oleo Science, 64(9), 915–932. 10.5650/jos.ess15074

</cite> [<a href="https://doi.org/10.5650/jos.ess15074" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26250424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Oleo%20Science&amp;title=Oat%20(Avena%20sativa%20L.):%20Oil%20and%20nutriment%20compounds%20valorization%20for%20potential%20use%20in%20industrial%20applications&amp;author=N.%20B.%20Halima&amp;author=R.%20B.%20Saad&amp;author=B.%20Khemakhem&amp;author=I.%20Fendri&amp;author=S.%20Abdelkafi&amp;volume=64&amp;issue=9&amp;publication_year=2015&amp;pages=915-932&amp;pmid=26250424&amp;doi=10.5650/jos.ess15074&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0031">
<cite>

Hamblin, M. T.
, 
Buckler, E. S.
, &amp; 
Jannink, J.‐L.
 (2011). Population genetics of genomics‐based crop improvement methods. Trends in Genetics, 27(3), 98–106. 10.1016/j.tig.2010.12.003

</cite> [<a href="https://doi.org/10.1016/j.tig.2010.12.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21227531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Genetics&amp;title=Population%20genetics%20of%20genomics%E2%80%90based%20crop%20improvement%20methods&amp;author=M.%20T.%20Hamblin&amp;author=E.%20S.%20Buckler&amp;author=J.%E2%80%90L.%20Jannink&amp;volume=27&amp;issue=3&amp;publication_year=2011&amp;pages=98-106&amp;pmid=21227531&amp;doi=10.1016/j.tig.2010.12.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0032">
<cite>

Harwood, J. L.
 (1988). Fatty acid metabolism. Annual Review of Plant Physiology, 39, 101–138. 10.1146/annurev.pp.39.060188.000533
</cite> [<a href="https://doi.org/10.1146/annurev.pp.39.060188.000533" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Plant%20Physiology&amp;title=Fatty%20acid%20metabolism&amp;author=J.%20L.%20Harwood&amp;volume=39&amp;publication_year=1988&amp;pages=101-138&amp;doi=10.1146/annurev.pp.39.060188.000533&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0033">
<cite>

Hazen, S. P.
, 
Scott‐Craig, J. S.
, &amp; 
Walton, J. D.
 (2002). Cellulose synthase‐like genes of rice. Plant Physiology, 128(2), 336–340. 10.1104/pp.010875

</cite> [<a href="https://doi.org/10.1104/pp.010875" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1540205/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11842136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Cellulose%20synthase%E2%80%90like%20genes%20of%20rice&amp;author=S.%20P.%20Hazen&amp;author=J.%20S.%20Scott%E2%80%90Craig&amp;author=J.%20D.%20Walton&amp;volume=128&amp;issue=2&amp;publication_year=2002&amp;pages=336-340&amp;pmid=11842136&amp;doi=10.1104/pp.010875&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0034">
<cite>

Hill, W. G.
, &amp; 
Weir, B. S.
 (1988). Variances and covariances of squared linkage disequilibria in finite populations. Theoretical Population Biology, 33(1), 54–78. 10.1016/0040-5809(88)90004-4

</cite> [<a href="https://doi.org/10.1016/0040-5809(88)90004-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3376052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20Population%20Biology&amp;title=Variances%20and%20covariances%20of%20squared%20linkage%20disequilibria%20in%20finite%20populations&amp;author=W.%20G.%20Hill&amp;author=B.%20S.%20Weir&amp;volume=33&amp;issue=1&amp;publication_year=1988&amp;pages=54-78&amp;pmid=3376052&amp;doi=10.1016/0040-5809(88)90004-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0037">
<cite>

Huang, M.
, 
Liu, X.
, 
Zhou, Y.
, 
Summers, R. M.
, &amp; 
Zhang, Z.
 (2019). BLINK: A package for the next level of genome‐wide association studies with both individuals and markers in the millions. GigaScience, 8(2), giy154. 10.1093/gigascience/giy154

</cite> [<a href="https://doi.org/10.1093/gigascience/giy154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6365300/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30535326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=BLINK:%20A%20package%20for%20the%20next%20level%20of%20genome%E2%80%90wide%20association%20studies%20with%20both%20individuals%20and%20markers%20in%20the%20millions&amp;author=M.%20Huang&amp;author=X.%20Liu&amp;author=Y.%20Zhou&amp;author=R.%20M.%20Summers&amp;author=Z.%20Zhang&amp;volume=8&amp;issue=2&amp;publication_year=2019&amp;pages=giy154&amp;pmid=30535326&amp;doi=10.1093/gigascience/giy154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0038">
<cite>

Huang, X.
, &amp; 
Han, B.
 (2014). Natural variations and genome‐wide association studies in crop plants. Annual Review of Plant Biology, 65, 531–551. 10.1146/annurev-arplant-050213-035715
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-050213-035715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24274033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Plant%20Biology&amp;title=Natural%20variations%20and%20genome%E2%80%90wide%20association%20studies%20in%20crop%20plants&amp;author=X.%20Huang&amp;author=B.%20Han&amp;volume=65&amp;publication_year=2014&amp;pages=531-551&amp;pmid=24274033&amp;doi=10.1146/annurev-arplant-050213-035715&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0039">
<cite>

Jannink, J.‐L.
, 
Lorenz, A. J.
, &amp; 
Iwata, H.
 (2010). Genomic selection in plant breeding: From theory to practice. Briefings in Functional Genomics, 9(2), 166–177. 10.1093/bfgp/elq001

</cite> [<a href="https://doi.org/10.1093/bfgp/elq001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20156985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Briefings%20in%20Functional%20Genomics&amp;title=Genomic%20selection%20in%20plant%20breeding:%20From%20theory%20to%20practice&amp;author=J.%E2%80%90L.%20Jannink&amp;author=A.%20J.%20Lorenz&amp;author=H.%20Iwata&amp;volume=9&amp;issue=2&amp;publication_year=2010&amp;pages=166-177&amp;pmid=20156985&amp;doi=10.1093/bfgp/elq001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0040">
<cite>

Jenkins, A. L.
, 
Jenkins, D. J. A.
, 
Zdravkovic, U.
, 
Würsch, P.
, &amp; 
Vuksan, V.
 (2002). Depression of the glycemic index by high levels of beta‐glucan fiber in two functional foods tested in type 2 diabetes. European Journal of Clinical Nutrition, 56(7), 622–628. 10.1038/sj.ejcn.1601367

</cite> [<a href="https://doi.org/10.1038/sj.ejcn.1601367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12080401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Clinical%20Nutrition&amp;title=Depression%20of%20the%20glycemic%20index%20by%20high%20levels%20of%20beta%E2%80%90glucan%20fiber%20in%20two%20functional%20foods%20tested%20in%20type%202%20diabetes&amp;author=A.%20L.%20Jenkins&amp;author=D.%20J.%20A.%20Jenkins&amp;author=U.%20Zdravkovic&amp;author=P.%20W%C3%BCrsch&amp;author=V.%20Vuksan&amp;volume=56&amp;issue=7&amp;publication_year=2002&amp;pages=622-628&amp;pmid=12080401&amp;doi=10.1038/sj.ejcn.1601367&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0041">
<cite>

Kamal, N.
, 
Renhuldt, N. T.
, 
Bentzer, J.
, 
Gundlach, H.
, 
Haberer, G.
, 
Juhász, A.
, 
Lux, T.
, 
Bose, U.
, 
Tye‐Din, J. A.
, 
Lang, D.
, 
Gessel, N. V.
, 
Reski, R.
, 
Fu, Y.
, 
Spégel, P.
, 
Ceplitis, A.
, 
Himmelbach, A.
, 
Waters, A. J.
, 
Bekele, W. A.
, 
Colgrave, M. L.
, … 
Sirijovski, N.
 (2022). The mosaic oat genome gives insights into a uniquely healthy cereal crop. Nature, 606, 113–119. 10.1038/s41586-022-04732-y

</cite> [<a href="https://doi.org/10.1038/s41586-022-04732-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9159951/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35585233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20mosaic%20oat%20genome%20gives%20insights%20into%20a%20uniquely%20healthy%20cereal%20crop&amp;author=N.%20Kamal&amp;author=N.%20T.%20Renhuldt&amp;author=J.%20Bentzer&amp;author=H.%20Gundlach&amp;author=G.%20Haberer&amp;volume=606&amp;publication_year=2022&amp;pages=113-119&amp;pmid=35585233&amp;doi=10.1038/s41586-022-04732-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0042">
<cite>

Keenan, J. M.
, 
Pins, J. J.
, 
Frazel, C.
, 
Moran, A.
, &amp; 
Turnquist, L.
 (2002). Oat ingestion reduces systolic and diastolic blood pressure in patients with mild or borderline hypertension: A pilot study. The Journal of Family Practice, 51(4), 369.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/11978262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Family%20Practice&amp;title=Oat%20ingestion%20reduces%20systolic%20and%20diastolic%20blood%20pressure%20in%20patients%20with%20mild%20or%20borderline%20hypertension:%20A%20pilot%20study&amp;author=J.%20M.%20Keenan&amp;author=J.%20J.%20Pins&amp;author=C.%20Frazel&amp;author=A.%20Moran&amp;author=L.%20Turnquist&amp;volume=51&amp;issue=4&amp;publication_year=2002&amp;pages=369&amp;pmid=11978262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0043">
<cite>

Kianian, S. F.
, 
Egli, M. A.
, 
Phillips, R. L.
, 
Rines, H. W.
, 
Somers, D. A.
, 
Gengenbach, B. G.
, 
Webster, F. H.
, 
Livingston, S. M.
, 
Groh, S.
, 
O'Donoughue, L. S.
, 
Sorrells, M. E.
, 
Wesenberg, D. M.
, 
Stuthman, D. D.
, &amp; 
Fulcher, R. G.
 (1999). Association of a major groat oil content QTL and an acetyl‐CoA carboxylase gene in oat. Theoretical and Applied Genetics, 98, 884–894. 10.1007/s001220051147
</cite> [<a href="https://doi.org/10.1007/s001220051147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Association%20of%20a%20major%20groat%20oil%20content%20QTL%20and%20an%20acetyl%E2%80%90CoA%20carboxylase%20gene%20in%20oat&amp;author=S.%20F.%20Kianian&amp;author=M.%20A.%20Egli&amp;author=R.%20L.%20Phillips&amp;author=H.%20W.%20Rines&amp;author=D.%20A.%20Somers&amp;volume=98&amp;publication_year=1999&amp;pages=884-894&amp;doi=10.1007/s001220051147&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0045">
<cite>

Kibite, S.
, &amp; 
Edney, M. J.
 (1998). The inheritance of β‐glucan concentration in three oat (<em>Avena sativa</em> L.) crosses. Canadian Journal of Plant Science, 78(2), 245–250. 10.4141/P96-148
</cite> [<a href="https://doi.org/10.4141/P96-148" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Canadian%20Journal%20of%20Plant%20Science&amp;title=The%20inheritance%20of%20%CE%B2%E2%80%90glucan%20concentration%20in%20three%20oat%20(Avena%20sativa%20L.)%20crosses&amp;author=S.%20Kibite&amp;author=M.%20J.%20Edney&amp;volume=78&amp;issue=2&amp;publication_year=1998&amp;pages=245-250&amp;doi=10.4141/P96-148&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0046">
<cite>

Kou, Y.
, &amp; 
Wang, S.
 (2010). Broad‐spectrum and durability: Understanding of quantitative disease resistance. Current Opinion in Plant Biology, 13(2), 181–185. 10.1016/j.pbi.2009.12.010

</cite> [<a href="https://doi.org/10.1016/j.pbi.2009.12.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20097118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Plant%20Biology&amp;title=Broad%E2%80%90spectrum%20and%20durability:%20Understanding%20of%20quantitative%20disease%20resistance&amp;author=Y.%20Kou&amp;author=S.%20Wang&amp;volume=13&amp;issue=2&amp;publication_year=2010&amp;pages=181-185&amp;pmid=20097118&amp;doi=10.1016/j.pbi.2009.12.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0047">
<cite>

Langmead, B.
, &amp; 
Salzberg, S.
 (2012). Fast gapped‐read alignment with Bowtie 2. Nature Methods, 9, 357–359. 10.1038/nmeth.1923

</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Fast%20gapped%E2%80%90read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.%20Salzberg&amp;volume=9&amp;publication_year=2012&amp;pages=357-359&amp;pmid=22388286&amp;doi=10.1038/nmeth.1923&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0048">
<cite>

Lewis, R. S.
 (2006). Identification of germplasm of possible value for confronting an unfavorable inverse genetic correlation in tobacco. Crop Science, 46, 1764–1771. 10.2135/cropsci2005.12-0519
</cite> [<a href="https://doi.org/10.2135/cropsci2005.12-0519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Identification%20of%20germplasm%20of%20possible%20value%20for%20confronting%20an%20unfavorable%20inverse%20genetic%20correlation%20in%20tobacco&amp;author=R.%20S.%20Lewis&amp;volume=46&amp;publication_year=2006&amp;pages=1764-1771&amp;doi=10.2135/cropsci2005.12-0519&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0049">
<cite>

Li, H.
 (2011). A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Bioinformatics, 27(21), 2987–2993. 10.1093/bioinformatics/btr509

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr509" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3198575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21903627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20statistical%20framework%20for%20SNP%20calling,%20mutation%20discovery,%20association%20mapping%20and%20population%20genetical%20parameter%20estimation%20from%20sequencing%20data&amp;author=H.%20Li&amp;volume=27&amp;issue=21&amp;publication_year=2011&amp;pages=2987-2993&amp;pmid=21903627&amp;doi=10.1093/bioinformatics/btr509&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0050">
<cite>

Lipka, A. E.
, 
Tian, F.
, 
Wang, Q.
, 
Peiffer, J.
, 
Li, M.
, 
Bradbury, P. J.
, 
Gore, M. A.
, 
Buckler, E. S.
, &amp; 
Zhang, Z.
 (2012). GAPIT: Genome association and prediction integrated tool. Bioinformatics, 28(18), 2397–2399. 10.1093/bioinformatics/bts444

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22796960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GAPIT:%20Genome%20association%20and%20prediction%20integrated%20tool&amp;author=A.%20E.%20Lipka&amp;author=F.%20Tian&amp;author=Q.%20Wang&amp;author=J.%20Peiffer&amp;author=M.%20Li&amp;volume=28&amp;issue=18&amp;publication_year=2012&amp;pages=2397-2399&amp;pmid=22796960&amp;doi=10.1093/bioinformatics/bts444&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0053">
<cite>

McMullen, M. S.
, 
Doehlert, D. C.
, &amp; 
Miller, J. D.
 (2005). Registration of ‘HiFi’ oat. Crop Science, 45, 1664. 10.2135/cropsci2005.003
</cite> [<a href="https://doi.org/10.2135/cropsci2005.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Registration%20of%20%E2%80%98HiFi%E2%80%99%20oat&amp;author=M.%20S.%20McMullen&amp;author=D.%20C.%20Doehlert&amp;author=J.%20D.%20Miller&amp;volume=45&amp;publication_year=2005&amp;pages=1664&amp;doi=10.2135/cropsci2005.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0054">
<cite>

Mensink, R. P.
, &amp; 
Katan, M. B.
 (1992). Effect of dietary fatty acids on serum lipids and lipoproteins. A meta‐analysis of 27 trials. Arteriosclerosis, Thrombosis, and Vascular Biology, 12(8), 911–919. 10.1161/01.ATV.12.8.911
</cite> [<a href="https://doi.org/10.1161/01.ATV.12.8.911" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1386252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arteriosclerosis,%20Thrombosis,%20and%20Vascular%20Biology&amp;title=Effect%20of%20dietary%20fatty%20acids%20on%20serum%20lipids%20and%20lipoproteins.%20A%20meta%E2%80%90analysis%20of%2027%20trials&amp;author=R.%20P.%20Mensink&amp;author=M.%20B.%20Katan&amp;volume=12&amp;issue=8&amp;publication_year=1992&amp;pages=911-919&amp;pmid=1386252&amp;doi=10.1161/01.ATV.12.8.911&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0055">
<cite>

Meydani, M.
 (2009). Potential health benefits of avenanthramides of oats. Nutrition Reviews, 67(12), 731–735. 10.1111/j.1753-4887.2009.00256.x

</cite> [<a href="https://doi.org/10.1111/j.1753-4887.2009.00256.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19941618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nutrition%20Reviews&amp;title=Potential%20health%20benefits%20of%20avenanthramides%20of%20oats&amp;author=M.%20Meydani&amp;volume=67&amp;issue=12&amp;publication_year=2009&amp;pages=731-735&amp;pmid=19941618&amp;doi=10.1111/j.1753-4887.2009.00256.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0056">
<cite>

Money, D.
, 
Gardner, K.
, 
Migicovsky, Z.
, 
Schwaninger, H.
, 
Zhong, G.‐Y.
, &amp; 
Myles, S.
 (2015). LinkImpute: Fast and accurate genotype imputation for nonmodel organisms. G3 Genes|Genomes|Genetics, 5(11), 2383–2390. 10.1534/g3.115.021667

</cite> [<a href="https://doi.org/10.1534/g3.115.021667" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4632058/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26377960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=LinkImpute:%20Fast%20and%20accurate%20genotype%20imputation%20for%20nonmodel%20organisms&amp;author=D.%20Money&amp;author=K.%20Gardner&amp;author=Z.%20Migicovsky&amp;author=H.%20Schwaninger&amp;author=G.%E2%80%90Y.%20Zhong&amp;volume=5&amp;issue=11&amp;publication_year=2015&amp;pages=2383-2390&amp;pmid=26377960&amp;doi=10.1534/g3.115.021667&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0057">
<cite>

Murphy, J. P.
, &amp; 
Hoffman, L. A.
 (1992). The origin, history, and production of oat. In Marshall H. G. &amp; Sorrells M. E. (Eds.), Oat science and technology (pp. 1–28). ASA and CSSA. 10.2134/agronmonogr33
</cite> [<a href="https://doi.org/10.2134/agronmonogr33" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Oat%20science%20and%20technology&amp;author=J.%20P.%20Murphy&amp;author=L.%20A.%20Hoffman&amp;publication_year=1992&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0059">
<cite>

Newell, M. A.
, 
Asoro, F. G.
, 
Scott, M. P.
, 
White, P. J.
, 
Beavis, W. D.
, &amp; 
Jannink, J.‐L.
 (2012). Genome‐wide association study for oat (<em>Avena sativa</em> L.) beta‐glucan concentration using germplasm of worldwide origin. Theoretical and Applied Genetics, 125(8), 1687–1696. 10.1007/s00122-012-1945-0

</cite> [<a href="https://doi.org/10.1007/s00122-012-1945-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22865125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genome%E2%80%90wide%20association%20study%20for%20oat%20(Avena%20sativa%20L.)%20beta%E2%80%90glucan%20concentration%20using%20germplasm%20of%20worldwide%20origin&amp;author=M.%20A.%20Newell&amp;author=F.%20G.%20Asoro&amp;author=M.%20P.%20Scott&amp;author=P.%20J.%20White&amp;author=W.%20D.%20Beavis&amp;volume=125&amp;issue=8&amp;publication_year=2012&amp;pages=1687-1696&amp;pmid=22865125&amp;doi=10.1007/s00122-012-1945-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0060">
<cite>

Orr, W.
, &amp; 
Molnar, S. J.
 (2007). Development and mapping of PCR‐based SCAR and CAPS markers linked to oil QTLs in oat. Crop Science, 47, 848–852. 10.2135/cropsci2006.01.0053
</cite> [<a href="https://doi.org/10.2135/cropsci2006.01.0053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Development%20and%20mapping%20of%20PCR%E2%80%90based%20SCAR%20and%20CAPS%20markers%20linked%20to%20oil%20QTLs%20in%20oat&amp;author=W.%20Orr&amp;author=S.%20J.%20Molnar&amp;volume=47&amp;publication_year=2007&amp;pages=848-852&amp;doi=10.2135/cropsci2006.01.0053&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0061">
<cite>
PepsiCo
. (2021). 
<em>Avena sativa</em>—<em>OT3098 v2</em>
. PepsiCo. <a href="https://wheat.pw.usda.gov/jb?data=/ggds/oat-ot3098v2-pepsico" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://wheat.pw.usda.gov/jb?data=/ggds/oat‐ot3098v2‐pepsico</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?PepsiCo%20.%20(2021).%20Avena%20sativa%E2%80%94OT3098%20v2%20.%20PepsiCo.%20https://wheat.pw.usda.gov/jb?data=/ggds/oat%E2%80%90ot3098v2%E2%80%90pepsico" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0062">
<cite>

Peterson, D. M.
 (1992). Composition and nutritional characteristics of oat grain and products. In Marshall H. G., &amp; Sorrells M. E. (Eds.), Oat science and technology (pp. 265–292). ASA and CSSA. 10.2134/agronmonogr33
</cite> [<a href="https://doi.org/10.2134/agronmonogr33" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Oat%20science%20and%20technology&amp;author=D.%20M.%20Peterson&amp;publication_year=1992&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0063">
<cite>

Peterson, D. M.
, &amp; 
Smith, D.
 (1976). Changes in nitrogen and carbohydrate fractions in developing oat groats. Crop Science, 16(1), 67–71. 10.2135/cropsci1976.0011183X001600010017x
</cite> [<a href="https://doi.org/10.2135/cropsci1976.0011183X001600010017x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Changes%20in%20nitrogen%20and%20carbohydrate%20fractions%20in%20developing%20oat%20groats&amp;author=D.%20M.%20Peterson&amp;author=D.%20Smith&amp;volume=16&amp;issue=1&amp;publication_year=1976&amp;pages=67-71&amp;doi=10.2135/cropsci1976.0011183X001600010017x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0064">
<cite>

Peterson, D. M.
, 
Wesenberg, D. M.
, &amp; 
Burrup, D. E.
 (1995). β‐Glucan content and its relationship to agronomic characteristics in elite oat germplasm. Crop Science, 35(4), 965–970. 10.2135/cropsci1995.0011183X003500040005x
</cite> [<a href="https://doi.org/10.2135/cropsci1995.0011183X003500040005x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=%CE%B2%E2%80%90Glucan%20content%20and%20its%20relationship%20to%20agronomic%20characteristics%20in%20elite%20oat%20germplasm&amp;author=D.%20M.%20Peterson&amp;author=D.%20M.%20Wesenberg&amp;author=D.%20E.%20Burrup&amp;volume=35&amp;issue=4&amp;publication_year=1995&amp;pages=965-970&amp;doi=10.2135/cropsci1995.0011183X003500040005x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0089">
<cite>

Rodríguez‐Álvarez, M. X.
, 
Boer, M. P.
, 
van Eeuwijk, F. A.
, &amp; 
Eilers, P. H. C.
 (2018). Correcting for spatial heterogeneity in plant breeding experiments with P‐splines. Spatial Statistics, 23, 52–71. 10.1016/j.spasta.2017.10.003
</cite> [<a href="https://doi.org/10.1016/j.spasta.2017.10.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Spatial%20Statistics&amp;title=Correcting%20for%20spatial%20heterogeneity%20in%20plant%20breeding%20experiments%20with%20P%E2%80%90splines&amp;author=M.%20X.%20Rodr%C3%ADguez%E2%80%90%C3%81lvarez&amp;author=M.%20P.%20Boer&amp;author=F.%20A.%20van%20Eeuwijk&amp;author=P.%20H.%20C.%20Eilers&amp;volume=23&amp;publication_year=2018&amp;pages=52-71&amp;doi=10.1016/j.spasta.2017.10.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0065">
<cite>

Schloerke, B.
, 
Cook, D.
, 
Larmarange, J.
, 
Briatte, F.
, 
Marbach, M.
, 
Thoen, E.
, 
Elberg, A.
, &amp; 
Crowley, J.
 (2022). 
GGally: Extension to ‘ggplot2’
. GitHub. <a href="https://github.com/ggobi/ggally" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ggobi/ggally</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Schloerke,%20B.%20,%20Cook,%20D.%20,%20Larmarange,%20J.%20,%20Briatte,%20F.%20,%20Marbach,%20M.%20,%20Thoen,%20E.%20,%20Elberg,%20A.%20,%20&amp;%20Crowley,%20J.%20(2022).%20GGally:%20Extension%20to%20%E2%80%98ggplot2%E2%80%99%20.%20GitHub.%20https://github.com/ggobi/ggally" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0066">
<cite>

Siripoonwiwat, W.
, 
O'Donoughue, L. S.
, 
Wesenberg, D.
, 
Hoffman, D. L.
, 
Barbosa‐Neto, J. F.
, &amp; 
Sorrells, M. E.
 (1996). Chromosomal regions associated with quantitative traits in oat. Journal of Agricultural Genomics, 2, 1–13.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agricultural%20Genomics&amp;title=Chromosomal%20regions%20associated%20with%20quantitative%20traits%20in%20oat&amp;author=W.%20Siripoonwiwat&amp;author=L.%20S.%20O'Donoughue&amp;author=D.%20Wesenberg&amp;author=D.%20L.%20Hoffman&amp;author=J.%20F.%20Barbosa%E2%80%90Neto&amp;volume=2&amp;publication_year=1996&amp;pages=1-13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0067">
<cite>

Strychar, R.
 (2011). World oat production, trade, and usage. In Webster F. &amp; Wood P. (Eds.), Oats: Chemistry and technology (pp. 1–10). AACC International Inc. 10.1094/9781891127649.fm
</cite> [<a href="https://doi.org/10.1094/9781891127649.fm" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Oats:%20Chemistry%20and%20technology&amp;author=R.%20Strychar&amp;publication_year=2011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0068">
<cite>

Sunilkumar, B. A.
, 
Leonova, S.
, 
Öste, R.
, &amp; 
Olsson, O.
 (2017). Identification and characterization of high protein oat lines from a mutagenized oat population. Journal of Cereal Science, 75, 100–107. 10.1016/j.jcs.2017.03.003
</cite> [<a href="https://doi.org/10.1016/j.jcs.2017.03.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cereal%20Science&amp;title=Identification%20and%20characterization%20of%20high%20protein%20oat%20lines%20from%20a%20mutagenized%20oat%20population&amp;author=B.%20A.%20Sunilkumar&amp;author=S.%20Leonova&amp;author=R.%20%C3%96ste&amp;author=O.%20Olsson&amp;volume=75&amp;publication_year=2017&amp;pages=100-107&amp;doi=10.1016/j.jcs.2017.03.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0069">
<cite>

Tanksley, S. D.
, 
Young, N. D.
, 
Paterson, A. H.
, &amp; 
Bonierbale, M. W.
 (1989). RFLP mapping in plant breeding: New tools for an old science. Nature Biotechnology, 7(3), 257–264. 10.1038/nbt0389-257
</cite> [<a href="https://doi.org/10.1038/nbt0389-257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=RFLP%20mapping%20in%20plant%20breeding:%20New%20tools%20for%20an%20old%20science&amp;author=S.%20D.%20Tanksley&amp;author=N.%20D.%20Young&amp;author=A.%20H.%20Paterson&amp;author=M.%20W.%20Bonierbale&amp;volume=7&amp;issue=3&amp;publication_year=1989&amp;pages=257-264&amp;doi=10.1038/nbt0389-257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0070">
<cite>

Thro, A. M.
, &amp; 
Frey, K. J.
 (1985). Inheritance of groat‐oil content and high‐oil selection in oats (<em>Avena Sativa</em> L.). Euphytica, 34, 251–263. 10.1007/BF00022917
</cite> [<a href="https://doi.org/10.1007/BF00022917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Inheritance%20of%20groat%E2%80%90oil%20content%20and%20high%E2%80%90oil%20selection%20in%20oats%20(Avena%20Sativa%20L.)&amp;author=A.%20M.%20Thro&amp;author=K.%20J.%20Frey&amp;volume=34&amp;publication_year=1985&amp;pages=251-263&amp;doi=10.1007/BF00022917&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0071">
<cite>

Tinker, N. A.
, 
Wight, C. P.
, 
Bekele, W. A.
, 
Yan, W.
, 
Jellen, E. N.
, 
Renhuldt, N. T.
, 
Sirijovsky, N.
, 
Lux, T.
, 
Spannagl, M.
, &amp; 
Mascher, M.
 (2022). Genome analysis in <em>Avena sativa</em> reveals hidden breeding barriers and opportunities for oat improvement. Communications Biology, 5, Article 474. 10.1038/s42003-022-03256-5

</cite> [<a href="https://doi.org/10.1038/s42003-022-03256-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9117302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35585176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Communications%20Biology&amp;title=Genome%20analysis%20in%20Avena%20sativa%20reveals%20hidden%20breeding%20barriers%20and%20opportunities%20for%20oat%20improvement&amp;author=N.%20A.%20Tinker&amp;author=C.%20P.%20Wight&amp;author=W.%20A.%20Bekele&amp;author=W.%20Yan&amp;author=E.%20N.%20Jellen&amp;volume=5&amp;publication_year=2022&amp;pages=474&amp;pmid=35585176&amp;doi=10.1038/s42003-022-03256-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0072">
<cite>
USDA‐NASS
. (2020). Quick stats. United States Department of Agriculture‐National Agricultural Statistics Service. <a href="https://quickstats.nass.usda.gov/results/0BD33158-B668-307F-8C75-8249653B6122?pivot=short_desc" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://quickstats.nass.usda.gov/results/0BD33158‐B668‐307F‐8C75‐8249653B6122?pivot=short_desc</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Quick%20stats&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0073">
<cite>

Wang, H.
, 
Smith, K. P.
, 
Combs, E.
, 
Blake, T.
, 
Horsley, R. D.
, &amp; 
Muehlbauer, G. J.
 (2012). Effect of population size and unbalanced data sets on QTL detection using genome‐wide association mapping in barley breeding germplasm. Theoretical and Applied Genetics, 124, 111–124. 10.1007/s00122-011-1691-8

</cite> [<a href="https://doi.org/10.1007/s00122-011-1691-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21898052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Effect%20of%20population%20size%20and%20unbalanced%20data%20sets%20on%20QTL%20detection%20using%20genome%E2%80%90wide%20association%20mapping%20in%20barley%20breeding%20germplasm&amp;author=H.%20Wang&amp;author=K.%20P.%20Smith&amp;author=E.%20Combs&amp;author=T.%20Blake&amp;author=R.%20D.%20Horsley&amp;volume=124&amp;publication_year=2012&amp;pages=111-124&amp;pmid=21898052&amp;doi=10.1007/s00122-011-1691-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0074">
<cite>

Wang, L.
, 
Xu, J.
, 
Wang, H.
, 
Chen, T.
, 
You, E.
, 
Bian, H.
, 
Chen, W.
, 
Zhang, B.
, &amp; 
Shen, Y.
 (2023). Population structure analysis and genome‐wide association study of a hexaploid oat landrace and cultivar collection. Frontiers in Plant Science, 14, 1131751. 10.3389/fpls.2023.1131751

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1131751" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10070682/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37025134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Population%20structure%20analysis%20and%20genome%E2%80%90wide%20association%20study%20of%20a%20hexaploid%20oat%20landrace%20and%20cultivar%20collection&amp;author=L.%20Wang&amp;author=J.%20Xu&amp;author=H.%20Wang&amp;author=T.%20Chen&amp;author=E.%20You&amp;volume=14&amp;publication_year=2023&amp;pages=1131751&amp;pmid=37025134&amp;doi=10.3389/fpls.2023.1131751&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0075">
<cite>

Weckwerth, W.
 (2011). Green systems biology—From single genomes, proteomes and metabolomes to ecosystems research and biotechnology. Journal of Proteomics, 75(1), 284–305. 10.1016/j.jprot.2011.07.010

</cite> [<a href="https://doi.org/10.1016/j.jprot.2011.07.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21802534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteomics&amp;title=Green%20systems%20biology%E2%80%94From%20single%20genomes,%20proteomes%20and%20metabolomes%20to%20ecosystems%20research%20and%20biotechnology&amp;author=W.%20Weckwerth&amp;volume=75&amp;issue=1&amp;publication_year=2011&amp;pages=284-305&amp;pmid=21802534&amp;doi=10.1016/j.jprot.2011.07.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0077">
<cite>

Yan, H.
, 
Yu, K.
, 
Xu, Y.
, 
Zhou, P.
, 
Zhao, J.
, 
Li, Y.
, 
Liu, X.
, 
Ren, C.
, &amp; 
Peng, Y.
 (2021). Position validation of the dwarfing gene <em>Dw6</em> in oat (<em>Avena sativa</em> L.) and its correlated effects on agronomic traits. Frontiers in Plant Science, 12, Article 668847. 10.3389/fpls.2021.668847

</cite> [<a href="https://doi.org/10.3389/fpls.2021.668847" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8172587/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34093626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Position%20validation%20of%20the%20dwarfing%20gene%20Dw6%20in%20oat%20(Avena%20sativa%20L.)%20and%20its%20correlated%20effects%20on%20agronomic%20traits&amp;author=H.%20Yan&amp;author=K.%20Yu&amp;author=Y.%20Xu&amp;author=P.%20Zhou&amp;author=J.%20Zhao&amp;volume=12&amp;publication_year=2021&amp;pages=668847&amp;pmid=34093626&amp;doi=10.3389/fpls.2021.668847&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0078">
<cite>

Yan, H.
, 
Zhang, H.
, 
Zhou, P.
, 
Ren, C.
, &amp; 
Peng, Y.
 (2023). Genome‐wide association mapping of QTL underlying groat protein content of a diverse panel of oat accessions. International Journal of Molecular Sciences, 24(6), 5581. 10.3390/ijms24065581

</cite> [<a href="https://doi.org/10.3390/ijms24065581" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10053717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36982656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Genome%E2%80%90wide%20association%20mapping%20of%20QTL%20underlying%20groat%20protein%20content%20of%20a%20diverse%20panel%20of%20oat%20accessions&amp;author=H.%20Yan&amp;author=H.%20Zhang&amp;author=P.%20Zhou&amp;author=C.%20Ren&amp;author=Y.%20Peng&amp;volume=24&amp;issue=6&amp;publication_year=2023&amp;pages=5581&amp;pmid=36982656&amp;doi=10.3390/ijms24065581&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0079">
<cite>

Yan, W.
, &amp; 
Wallace, D. H.
 (1995). Breeding for negatively associated traits. In Janick J. (Ed.), Plant breeding reviews (pp. 141–177). Wiley. 10.1002/9780470650059.ch4
</cite> [<a href="https://doi.org/10.1002/9780470650059.ch4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Plant%20breeding%20reviews&amp;author=W.%20Yan&amp;author=D.%20H.%20Wallace&amp;publication_year=1995&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0080">
<cite>

Yin, Y.
, 
Huang, J.
, &amp; 
Xu, Y.
 (2009). The cellulose synthase superfamily in fully sequenced plants and algae. BMC Plant Biology, 9, Article 99. 10.1186/1471-2229-9-99

</cite> [<a href="https://doi.org/10.1186/1471-2229-9-99" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3091534/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19646250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=The%20cellulose%20synthase%20superfamily%20in%20fully%20sequenced%20plants%20and%20algae&amp;author=Y.%20Yin&amp;author=J.%20Huang&amp;author=Y.%20Xu&amp;volume=9&amp;publication_year=2009&amp;pages=99&amp;pmid=19646250&amp;doi=10.1186/1471-2229-9-99&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0081">
<cite>

Young, N. V.
 (1999). A cautiously optimistic vision for marker‐assisted breeding. Molecular Breeding, 5, 505–510. 10.1023/A:1009684409326
</cite> [<a href="https://doi.org/10.1023/A:1009684409326" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=A%20cautiously%20optimistic%20vision%20for%20marker%E2%80%90assisted%20breeding&amp;author=N.%20V.%20Young&amp;volume=5&amp;publication_year=1999&amp;pages=505-510&amp;doi=10.1023/A:1009684409326&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0083">
<cite>

Zhang, Z.
, 
Ersoz, E.
, 
Lai, C. Q.
, 
Todhunter, R. J.
, 
Tiwari, H. K.
, 
Gore, M. A.
, 
Bradbury, P. J.
, 
Yu, J.
, 
Arnett, D. K.
, 
Ordovas, J. M.
, &amp; 
Buckler, E. S.
 (2010). Mixed linear model approach adapted for genome‐wide association studies. Nature Genetics, 42, 355–360. 10.1038/ng.546

</cite> [<a href="https://doi.org/10.1038/ng.546" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2931336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20208535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Mixed%20linear%20model%20approach%20adapted%20for%20genome%E2%80%90wide%20association%20studies&amp;author=Z.%20Zhang&amp;author=E.%20Ersoz&amp;author=C.%20Q.%20Lai&amp;author=R.%20J.%20Todhunter&amp;author=H.%20K.%20Tiwari&amp;volume=42&amp;publication_year=2010&amp;pages=355-360&amp;pmid=20208535&amp;doi=10.1038/ng.546&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0084">
<cite>

Zhu, S.
, 
Rossnagel, B. G.
, &amp; 
Kaeppler, H. F.
 (2004). Genetic analysis of quantitative trait loci for groat protein and oil content in oat. Crop Science, 44, 254–260. 10.2135/cropsci2004.2540
</cite> [<a href="https://doi.org/10.2135/cropsci2004.2540" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genetic%20analysis%20of%20quantitative%20trait%20loci%20for%20groat%20protein%20and%20oil%20content%20in%20oat&amp;author=S.%20Zhu&amp;author=B.%20G.%20Rossnagel&amp;author=H.%20F.%20Kaeppler&amp;volume=44&amp;publication_year=2004&amp;pages=254-260&amp;doi=10.2135/cropsci2004.2540&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270060-bib-0085">
<cite>

Zimmer, C. M.
, 
Ubert, I. P.
, 
Pacheco, M. T.
, &amp; 
Federizzi, L. C.
 (2018). Molecular and comparative mapping for heading date and plant height in oat. Euphytica, 214, 101. 10.1007/s10681-018-2182-7
</cite> [<a href="https://doi.org/10.1007/s10681-018-2182-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Molecular%20and%20comparative%20mapping%20for%20heading%20date%20and%20plant%20height%20in%20oat&amp;author=C.%20M.%20Zimmer&amp;author=I.%20P.%20Ubert&amp;author=M.%20T.%20Pacheco&amp;author=L.%20C.%20Federizzi&amp;volume=214&amp;publication_year=2018&amp;pages=101&amp;doi=10.1007/s10681-018-2182-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>
<strong>Supplementary Table 1</strong> contains PYT/AYT specific and unique genotype number with traits measured, locations and replications across years.</p>
<p>
<strong>Supplementary Table 2</strong> contains trait data used in this study.</p>
<p>
<strong>Supplementary Table 3</strong> contains repeated genotypes across experimental trials and years.</p>
<p>
<strong>Supplementary Table 4</strong> contains spatially adjusted means for each environment.</p>
<p>
<strong>Supplementary Table 5</strong> contains normalized BLUE values (check values excluded prior to association analysis).</p>
<p>
<strong>Supplementary Table 6</strong> contains relevant information for all markers identified with GWAS.</p>
<p>
<strong>Supplementary Table 7</strong> contains Significant markers per‐environment PVE markers before stepwise regression analysis.</p>
<p>
<strong>Supplementary Table 8</strong> Annotated genes within QTL linkage blocks.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s001.xlsx</a><sup> (3.3MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>
<strong>Supplementary Figure 1</strong> contains the distribution of spatially adjusted environment means for β‐Glucan (A), Protein (B), Oil (C), Test weight (D), Groat (E), Yield (F), Thins (G), Heading date (H) and Height (I).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s008.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s008.pdf</a><sup> (33.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>
<strong>Supplementary Figure 2</strong> contains the principal component analysis of the taxa used in this study.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s004.pdf</a><sup> (40.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p"><p>
<strong>Supplementary Figure 3</strong> contains the GWAS results visualized with QQ and Manhattan plots for β‐Glucan (A,B), Protein (C,D), Oil (E,F), Test weight (G,H), Groat (I,J), Yield (K,L), Thins (M,N), Heading date (O,P) and Height (Q,R).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s006.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s006.pdf</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p"><p>
<strong>Supplementary Figure 4</strong> contains the distribution of normalized BLUE values grouped by allele state.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s007.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s007.pdf</a><sup> (4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p"><p>
<strong>Supplementary Figure 5</strong> contains the distribution of spatially adjusted environment means grouped by the three most significant QTL for β‐Glucan (A‐C), Protein (D‐F), and Oil (G‐I).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s005.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s005.pdf</a><sup> (13.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p"><p>
<strong>Supplementary Figure 6</strong> contains the distribution of spatially adjusted environment means grouped by the three most significant QTL for Test weight (A‐C), Groat (D‐F), Yield (G‐I), and Thins (J‐L).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s003.pdf</a><sup> (52.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="caption p"><p>
<strong>Supplementary Figure 7</strong> contains the distribution of spatially adjusted environment means grouped by the three most significant QTL for Heading Date (A‐C) and Height (D‐F).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12234416/bin/TPG2-18-e70060-s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70060-s002.pdf</a><sup> (19.3KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Genotyping data have been deposited in the T3/Oat database under the “NDSU GBS” genotyping protocol. Phenotyping data can be found in Table <a href="#tpg270060-supinfo-0001" class="usa-link">S2</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70060"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70060.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12234416/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12234416/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12234416%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12234416/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12234416/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12234416/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40623918/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12234416/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40623918/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12234416/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12234416/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="TMHvuImTZmXURDzOfjQV8EsVGUB7etsrJ8oUFIXEq8nEBizCR7JYFu0Pv93NmCH5">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
