
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            3D structure and stability prediction of DNA with multi-way junctions in ionic solutions - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4BDC78AF455E305BDC70000E3B1DF.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ploscomp">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373291/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS Computational Biology">
<meta name="citation_title" content="3D structure and stability prediction of DNA with multi-way junctions in ionic solutions">
<meta name="citation_author" content="Xunxun Wang">
<meta name="citation_author_institution" content="Guizhou Key Laboratory of Microbio and Infectious Disease Prevention &amp; Control, School of Biology and Engineering, Guizhou Medical University, Guiyang, China">
<meta name="citation_author" content="Ya-Zhou Shi">
<meta name="citation_author_institution" content="Research Center of Nonlinear Science, School of Mathematics &amp; Statistics, Wuhan Textile University, Wuhan, China">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="21">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e1013346">
<meta name="citation_doi" content="10.1371/journal.pcbi.1013346">
<meta name="citation_pmid" content="40825053">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373291/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373291/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373291/pdf/pcbi.1013346.pdf">
<meta name="description" content="Understanding the three-dimensional (3D) structure and stability of DNA is essential for elucidating its biological functions and advancing structure-based drug design. Here, we present an improved coarse-grained (CG) model for ab initio prediction ...">
<meta name="og:title" content="3D structure and stability prediction of DNA with multi-way junctions in ionic solutions">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Understanding the three-dimensional (3D) structure and stability of DNA is essential for elucidating its biological functions and advancing structure-based drug design. Here, we present an improved coarse-grained (CG) model for ab initio prediction ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373291/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373291">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pcbi.1013346"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pcbi.1013346.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373291%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373291/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373291/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373291/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ploscomp.png" alt="PLOS Computational Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS Computational Biology" title="Link to PLOS Computational Biology" shape="default" href="https://doi.org/10.1371/journal.pcbi.1013346" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS Comput Biol</button></div>. 2025 Aug 18;21(8):e1013346. doi: <a href="https://doi.org/10.1371/journal.pcbi.1013346" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pcbi.1013346</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20Comput%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20Comput%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20Comput%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20Comput%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>3D structure and stability prediction of DNA with multi-way junctions in ionic solutions</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Xunxun Wang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xunxun Wang</span></h3>
<div class="p">
<sup>1</sup>Guizhou Key Laboratory of Microbio and Infectious Disease Prevention &amp; Control, School of Biology and Engineering, Guizhou Medical University, Guiyang, China</div>
<div>Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xunxun Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20YZ%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ya-Zhou Shi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ya-Zhou Shi</span></h3>
<div class="p">
<sup>2</sup>Research Center of Nonlinear Science, School of Mathematics &amp; Statistics, Wuhan Textile University, Wuhan, China</div>
<div>Conceptualization, Formal analysis, Funding acquisition, Methodology, Project administration, Resources, Software, Supervision, Validation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20YZ%22%5BAuthor%5D" class="usa-link"><span class="name western">Ya-Zhou Shi</span></a>
</div>
</div>
<sup>2,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Amar Singh</span><sup>3</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Guizhou Key Laboratory of Microbio and Infectious Disease Prevention &amp; Control, School of Biology and Engineering, Guizhou Medical University, Guiyang, China</div>
<div id="aff002">
<sup>2</sup>Research Center of Nonlinear Science, School of Mathematics &amp; Statistics, Wuhan Textile University, Wuhan, China</div>
<div id="edit1">
<sup>3</sup>KU: The University of Kansas, UNITED STATES OF AMERICA</div>
<div class="author-notes p">
<div class="fn" id="coi001"><p>The authors have declared that no competing interests exist.</p></div>
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>yzshi@wtu.edu.cn</span></p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Xunxun Wang</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ya-Zhou Shi</span></strong>: <span class="role">Conceptualization, Formal analysis, Funding acquisition, Methodology, Project administration, Resources, Software, Supervision, Validation, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Amar Singh</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 14; Accepted 2025 Jul 21; Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Wang, Shi</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373291  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40825053/" class="usa-link">40825053</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Understanding the three-dimensional (3D) structure and stability of DNA is essential for elucidating its biological functions and advancing structure-based drug design. Here, we present an improved coarse-grained (CG) model for <em>ab initio</em> prediction of DNA folding, integrating a refined electrostatic potential, replica-exchange Monte Carlo simulations, and weighted histogram analysis. The model accurately predicts the 3D structures of DNA with multi-way junctions (e.g., achieving a mean RMSD of ~8.8 Å for top-ranked structures across four DNAs with three- or four-way junctions) from sequence, outperforming existing fragment-assembly and AI-based approaches. The model also reproduces the thermal stability of junctions across diverse sequences and lengths, with predicted melting temperatures deviating by less than 5 °C from experimental values, under both monovalent (Na⁺) and divalent (Mg<sup>2</sup>⁺) ionic conditions. Furthermore, analysis of the thermal unfolding pathways reveals that the overall stability of multi-way junctions is primarily determined by the relative free energies of key intermediate states. These results provide a robust framework for predicting complex DNA architectures and offer mechanistic insights into DNA folding and function.</p></section><section class="abstract" id="abstract2"><h2>Author summary</h2>
<p>Beyond the familiar double helix, DNA can fold into complex three-dimensional (3D) structures such as multi-way junctions that are essential in biological processes and widely used in nanotechnology and drug design. Predicting how these structures form and how stable they are, especially under salt conditions, remains a major challenge. In this study, we introduce a coarse-grained model that can accurately predict both the 3D structures and thermal stability of three- and four-way DNA junctions directly from their sequence. By integrating a refined treatment of electrostatic interactions and advanced sampling techniques (e.g., REMC and WHAM), the model reproduces experimental 3D structures and melting temperatures across a wide range of sequences and salt conditions including monovalent (Na⁺) and divalent (Mg<sup>2</sup>⁺) ions. We also find that the stability of these DNA junctions is governed by the relative energies of intermediate folding states. This work provides a practical tool for researchers to model complex DNA architectures and deepens our understanding of how DNA sequence and ions together control DNA structure and function.</p></section><section id="sec001"><h2 class="pmc_sec_title">Introduction</h2>
<p>DNA is a fundamental macromolecule in living organisms, essential for the storage and transmission of genetic information [<a href="#pcbi.1013346.ref001" class="usa-link" aria-describedby="pcbi.1013346.ref001">1</a>], regulation of protein synthesis [<a href="#pcbi.1013346.ref002" class="usa-link" aria-describedby="pcbi.1013346.ref002">2</a>], and control of gene expression [<a href="#pcbi.1013346.ref003" class="usa-link" aria-describedby="pcbi.1013346.ref003">3</a>]. Beyond its classical B-form double helix, DNA can adopt a variety of complex 3D structures, such as hairpins, junctions, quadruplexes, and other non-canonical forms, which are increasingly recognized as functional elements involved in gene expression regulation, genome stability, and epigenetic control [<a href="#pcbi.1013346.ref004" class="usa-link" aria-describedby="pcbi.1013346.ref004">4</a>–<a href="#pcbi.1013346.ref007" class="usa-link" aria-describedby="pcbi.1013346.ref007">7</a>]. These dynamic conformations are not only critical for biological processes, e.g., facilitating enhancer-promoter communication or transcription factor binding through structure folding, but also form the structural basis for DNA-based nanotechnology [<a href="#pcbi.1013346.ref008" class="usa-link" aria-describedby="pcbi.1013346.ref008">8</a>]. Among them, DNA with multi-way junctions represents a higher level of structural complexity compared to linear or duplex DNA [<a href="#pcbi.1013346.ref009" class="usa-link" aria-describedby="pcbi.1013346.ref009">9</a>], and is thought to contribute to molecular recognition, signaling, and the assembly of nucleic acid-based nanodevices [<a href="#pcbi.1013346.ref010" class="usa-link" aria-describedby="pcbi.1013346.ref010">10</a>]. Therefore, gaining deeper insights into the 3D structure and stability of such junction-containing DNAs is essential for both understanding their biological relevance and advancing DNA-based material design.</p>
<p>The DNA 3D structures can be experimentally resolved through experimental techniques such as X-ray crystallography, nuclear magnetic resonance spectroscopy, and cryo-electron microscopy [<a href="#pcbi.1013346.ref011" class="usa-link" aria-describedby="pcbi.1013346.ref011">11</a>,<a href="#pcbi.1013346.ref012" class="usa-link" aria-describedby="pcbi.1013346.ref012">12</a>]. However, due to the high cost and technical difficulty of these methods, the number of determined DNA 3D structures deposited in the Protein Data Bank (PDB) remains limited [<a href="#pcbi.1013346.ref011" class="usa-link" aria-describedby="pcbi.1013346.ref011">11</a>–<a href="#pcbi.1013346.ref014" class="usa-link" aria-describedby="pcbi.1013346.ref014">14</a>], especially in contrast to the vast number of DNA sequences available in GenBank [<a href="#pcbi.1013346.ref015" class="usa-link" aria-describedby="pcbi.1013346.ref015">15</a>,<a href="#pcbi.1013346.ref016" class="usa-link" aria-describedby="pcbi.1013346.ref016">16</a>]. To complement experimental approaches, several theoretical and computational methods have been developed to predict/model DNA 3D structures [<a href="#pcbi.1013346.ref014" class="usa-link" aria-describedby="pcbi.1013346.ref014">14</a>,<a href="#pcbi.1013346.ref017" class="usa-link" aria-describedby="pcbi.1013346.ref017">17</a>]. These methods can be classified into three categories: deep learning-based, template-based, and physics-based approaches.</p>
<p>Deep learning-based approaches have achieved remarkable success in protein structure prediction and are increasingly being applied to nucleic acids [<a href="#pcbi.1013346.ref018" class="usa-link" aria-describedby="pcbi.1013346.ref018">18</a>–<a href="#pcbi.1013346.ref022" class="usa-link" aria-describedby="pcbi.1013346.ref022">22</a>]. For example, AlphaFold3 [<a href="#pcbi.1013346.ref018" class="usa-link" aria-describedby="pcbi.1013346.ref018">18</a>,<a href="#pcbi.1013346.ref019" class="usa-link" aria-describedby="pcbi.1013346.ref019">19</a>] has recently demonstrated the ability to model nucleic acid structures by learning from large datasets encompassing proteins, RNA, and DNA. These models use neural network architectures to directly infer structural patterns from sequence data, enabling rapid and scalable predictions. However, their performance on diverse DNA/RNA topologies remains limited due to the relatively sparse and biased training data for nucleic acids (e.g., dominated by canonical double-helical structures), compared to the extensive and diverse datasets available for proteins [<a href="#pcbi.1013346.ref023" class="usa-link" aria-describedby="pcbi.1013346.ref023">23</a>,<a href="#pcbi.1013346.ref024" class="usa-link" aria-describedby="pcbi.1013346.ref024">24</a>]. On the contrary, template-based fragment assembly methods offer a flexible framework for constructing 3D structures of DNA with arbitrary topologies including multi-way junctions by assembling known structural fragments based on the secondary structure. A representative example is the 3dDNA, which extends the original 3dRNA platform to support DNA modeling [<a href="#pcbi.1013346.ref025" class="usa-link" aria-describedby="pcbi.1013346.ref025">25</a>,<a href="#pcbi.1013346.ref026" class="usa-link" aria-describedby="pcbi.1013346.ref026">26</a>]. It assembles DNA 3D structures with high accuracy by aligning input sequence and secondary structure information with a curated template library of experimentally determined structure segments [<a href="#pcbi.1013346.ref026" class="usa-link" aria-describedby="pcbi.1013346.ref026">26</a>]. However, these methods rely heavily on accurate secondary structure input, which remains challenging for DNAs with noncanonical or complex folds [<a href="#pcbi.1013346.ref027" class="usa-link" aria-describedby="pcbi.1013346.ref027">27</a>]. Additionally, the limited diversity of template libraries could restrict their effectiveness in de novo predictions for novel sequences.</p>
<p>Physics-based approaches can model DNA folding and structure formation by simulating fundamental physical interactions, without requiring structure templates or secondary structure information. All-atom molecular dynamics (MD) simulations using well-established force fields such as CHARMM [<a href="#pcbi.1013346.ref028" class="usa-link" aria-describedby="pcbi.1013346.ref028">28</a>,<a href="#pcbi.1013346.ref029" class="usa-link" aria-describedby="pcbi.1013346.ref029">29</a>] and AMBER [<a href="#pcbi.1013346.ref030" class="usa-link" aria-describedby="pcbi.1013346.ref030">30</a>–<a href="#pcbi.1013346.ref032" class="usa-link" aria-describedby="pcbi.1013346.ref032">32</a>] have been widely applied to study the microscopic behavior of dsDNA, including its dynamics, flexibility, and structure transitions. However, their high computational cost restricts these simulations to small DNA fragments and short time scales (falling short of capturing folding processes). To address these limitations, a variety of coarse-grained (CG) models have been developed by reducing the degrees of freedom while retaining essential physical and thermodynamic characteristics [<a href="#pcbi.1013346.ref033" class="usa-link" aria-describedby="pcbi.1013346.ref033">33</a>–<a href="#pcbi.1013346.ref043" class="usa-link" aria-describedby="pcbi.1013346.ref043">43</a>]. For example, the oxDNA model represents nucleotides as rigid bodies with backbone, stacking, and hydrogen bonding interactions, and has been widely used to simulate both single- and double-stranded DNA behavior (e.g., mechanics and thermodynamics), as well as large-scale DNA nanostructures such as origami and nano-tweezers [<a href="#pcbi.1013346.ref039" class="usa-link" aria-describedby="pcbi.1013346.ref039">39</a>]. The 3SPN model adopts a three-site representation (phosphate, sugar, base), incorporating base-pairing and stacking to capture DNA denaturation, persistence length, and local curvature [<a href="#pcbi.1013346.ref040" class="usa-link" aria-describedby="pcbi.1013346.ref040">40</a>]. Similarly, another three-interaction-site model of TIS-DNA uses sequence-specific stacking energies derived from dimer thermodynamics, enabling accurate modeling of force-extension behavior, elasticity, and melting temperatures of simple DNAs [<a href="#pcbi.1013346.ref041" class="usa-link" aria-describedby="pcbi.1013346.ref041">41</a>]. Despite their successes, these models typically rely on Gö-like potentials that encode native structural information, limiting their capacity for structure prediction directly from sequence. To overcome this, several physics-based CG models have been proposed to fold DNAs without using the predefined structure restraints. For instance, the NARES-2P, with two CG beads for one nucleotide, can reproduce duplex formation, melting temperatures, and mechanical stability of dsDNAs just from two separate strands [<a href="#pcbi.1013346.ref042" class="usa-link" aria-describedby="pcbi.1013346.ref042">42</a>]. The HiRE-RNA, using six to seven beads per nucleotide, enables the study of both structural dynamics and sequence-dependent melting profiles [<a href="#pcbi.1013346.ref043" class="usa-link" aria-describedby="pcbi.1013346.ref043">43</a>]. However, the parameters of these models may need further validation for quantifying thermodynamic and 3D structure to accord with experiments, especially for ssDNA.</p>
<p>In addition, DNA folding and dynamics are further influenced by its polyanionic nature, making ionic conditions (e.g., Na<sup>+</sup> and Mg<sup>2+</sup>) crucial for accurate structural modeling [<a href="#pcbi.1013346.ref044" class="usa-link" aria-describedby="pcbi.1013346.ref044">44</a>–<a href="#pcbi.1013346.ref048" class="usa-link" aria-describedby="pcbi.1013346.ref048">48</a>]. Several existing CG models, including 3SPN, oxDNA, TIS-DNA, NARES-2P, and HiRE-RNA, incorporate electrostatic interactions using the Debye-Hückel approximation or mean-field multipole–multipole potentials to reproduce monovalent salt-dependent structural properties of DNA, such as persistence length, torsional stiffness or melting temperatures [<a href="#pcbi.1013346.ref049" class="usa-link" aria-describedby="pcbi.1013346.ref049">49</a>–<a href="#pcbi.1013346.ref053" class="usa-link" aria-describedby="pcbi.1013346.ref053">53</a>]. However, these models often neglect the significant influence of divalent ions on DNA/RNA folding. Recent studies have shown that explicitly including divalent cations, like Mg<sup>2+</sup>, enhances the accuracy of ion-nucleic acid and ion-ion interaction predictions, providing a more precise representation of the complex ionic environment around nucleic acids and better insights into the divalent ion effects on folding thermodynamics [<a href="#pcbi.1013346.ref054" class="usa-link" aria-describedby="pcbi.1013346.ref054">54</a>,<a href="#pcbi.1013346.ref055" class="usa-link" aria-describedby="pcbi.1013346.ref055">55</a>]. Very recently, we have also proposed a three-bead CG model to predict 3D structure folding for DNAs in monovalent/divalent ion solutions from sequence [<a href="#pcbi.1013346.ref013" class="usa-link" aria-describedby="pcbi.1013346.ref013">13</a>]. By integrating sequence-dependent base-pairing, base-stacking, and coaxial stacking interactions, along with an implicit electrostatic potential, the model accurately folded 20 dsDNAs (≤52 nt) and 20 ssDNAs (≤74 nt) into native-like structures (mean RMSD &lt; 4Å), and quantitatively predicted melting temperatures (mean deviation &lt; 3.0 °C) for 27 dsDNAs (including those with bulge loops and internal loops) and 24 ssDNAs (including hairpins and a pseudoknot). Despite these advances, accurately predicting the 3D folding of complex DNAs with multi-way junctions directly from sequence, especially under ionic conditions, remains a significant challenge for current models, including ours.</p>
<p>In this work, we have further refined our previously developed CG model by incorporating a structure-based electrostatic potential, an improved sampling algorithm (Replica Exchange Monte Carlo, REMC), and with the weighted histogram analysis method (WHAM). Specifically, the updated model includes: (i) a refined energy term to account for electrostatic energy term to capture electrostatic interactions between DNA and monovalent/divalent ions; (ii) the use of a more efficient REMC to enhance conformational sampling efficiency compared to conventional simulated annealing [<a href="#pcbi.1013346.ref013" class="usa-link" aria-describedby="pcbi.1013346.ref013">13</a>,<a href="#pcbi.1013346.ref056" class="usa-link" aria-describedby="pcbi.1013346.ref056">56</a>,<a href="#pcbi.1013346.ref057" class="usa-link" aria-describedby="pcbi.1013346.ref057">57</a>]; (iii) WHAM to analyze the thermal stability of DNA; and (iv) an all-atom reconstruction algorithm to recover atomistic structures from CG predictions. We applied the model to predict the 3D structures and thermodynamic stability of DNA with multi-way junctions in both monovalent and divalent ion solutions, and further analyzed their thermal unfolding pathways.</p></section><section id="sec002"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="sec003"><h3 class="pmc_sec_title">Coarse-grained representation for DNA structures</h3>
<p>In the present model, each DNA nucleotide is represented by a set of three CG beads [<a href="#pcbi.1013346.ref013" class="usa-link" aria-describedby="pcbi.1013346.ref013">13</a>]. Specifically, one bead is assigned to the phosphate group (P), another to the sugar moiety centered at the C4ʹ atom (C), and the third to the nucleobase (N), represented by the N1 atom in pyrimidines or the N9 atom in purines (<a href="#pcbi.1013346.g001" class="usa-link">Fig 1A</a>). This streamlined model retains the essential structural and chemical properties of DNA, such as backbone connectivity and base-pairing interactions [<a href="#pcbi.1013346.ref055" class="usa-link" aria-describedby="pcbi.1013346.ref055">55</a>,<a href="#pcbi.1013346.ref057" class="usa-link" aria-describedby="pcbi.1013346.ref057">57</a>]. The three types of beads are modeled as spheres with van der Waals radii of 1.9 Å (P), 1.7 Å (C), and 2.2 Å (N), respectively, and the P bead carries a unit negative charge to account for the polyanionic nature of DNA.</p>
<figure class="fig xbox font-sm" id="pcbi.1013346.g001"><h4 class="obj_head">Fig 1. Schematic overview of the coarse-grained (CG) modeling framework for DNA folding.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373291_pcbi.1013346.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9592/12373291/6703e4a828e4/pcbi.1013346.g001.jpg" loading="lazy" height="470" width="742" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013346.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> The folding progress of DNA is based solely on sequence information, incorporating the coarse-grained representation of DNA, the initial conformation, REMC simulations, and a structure ensemble derived from ten different temperature replicas to explore conformational diversity. <strong>(B)</strong> Refinement of the DNA structure, beginning with the identification of the most probable conformation through clustering of low-energy conformations, followed by the reconstruction of an all-atom model.</p></figcaption></figure></section><section id="sec004"><h3 class="pmc_sec_title">Coarse-grained force field</h3>
<p>In the present model, the total energy of a DNA chain is given by:</p>
<table class="disp-formula p" id="pcbi.1013346.e002"><tr>
<td class="formula"><math id="M2" display="block" overflow="linebreak"><mrow><mi>U</mi><mo>=</mo><msub><mrow><mi>U</mi></mrow><mrow><mi>b</mi></mrow></msub><mo>+</mo><msub><mrow><mi>U</mi></mrow><mrow><mi>a</mi></mrow></msub><mo>+</mo><msub><mrow><mi>U</mi></mrow><mrow><mi>d</mi></mrow></msub><mo>+</mo><msub><mrow><mi>U</mi></mrow><mrow><mi>e</mi><mi>x</mi><mi>c</mi></mrow></msub><mo>+</mo><msub><mrow><mi>U</mi></mrow><mrow><mi>b</mi><mi>p</mi></mrow></msub><mo>+</mo><msub><mrow><mi>U</mi></mrow><mrow><mi>b</mi><mi>s</mi></mrow></msub><mo>+</mo><msub><mrow><mi>U</mi></mrow><mrow><mi>c</mi><mi>s</mi></mrow></msub><mo>+</mo><msub><mrow><mi>U</mi></mrow><mrow><mi>e</mi><mi>l</mi></mrow></msub><mo>,</mo><mtext>\ \ \ \ \ \ \ \ \ \ \ </mtext></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e003"><math id="M3" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>U</mi></mrow><mrow><mi>b</mi></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e004"><math id="M4" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>U</mi></mrow><mrow><mi>a</mi></mrow></msub></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e005"><math id="M5" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>U</mi></mrow><mrow><mi>d</mi></mrow></msub></mrow></math></span> represent the bonded interactions corresponding to bond lengths, bond angles, and dihedral angles, respectively (see Eqs. S2–S4 in <a href="#pcbi.1013346.s013" class="usa-link">S1 Text</a>). These terms were parameterized by statistically analyzing the distributions of corresponding geometric features from experimentally resolved DNA structures (<a href="#pcbi.1013346.s007" class="usa-link">S1 Table</a>). Notably, most resolved DNA structures are dominated by well-formed helical regions, whose geometric characteristics are not ideal for capturing the folding behavior of flexible, unstructured DNA chains. To overcome this limitation, we separately analyzed base-paired helical regions and unpaired loop regions, deriving two distinct sets of bonded parameters: Para<sub>helix</sub> and Para<sub>loop</sub> (<a href="#pcbi.1013346.s008" class="usa-link">S2 Table</a>), respectively. During the folding simulations, only Para<sub>loop</sub> is used to better model the conformational dynamics of single-stranded DNA. In contrast, Para<sub>helix</sub> is employed exclusively in the final refinement stage to base-paired regions, ensuring the formation of canonical helical geometries with continuous base pairing.</p>
<p>The remaining terms in Eq. 1 describe non-bonded interactions; detailed formulations and parameterizations are provided in <a href="#pcbi.1013346.s013" class="usa-link">S1 Text</a>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e006"><math id="M6" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>U</mi></mrow><mrow><mi>e</mi><mi>x</mi><mi>c</mi></mrow></msub></mrow></math></span> accounts for excluded volume repulsion between CG beads (Eq. S10). <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e007"><math id="M7" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>U</mi></mrow><mrow><mi>b</mi><mi>p</mi></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e008"><math id="M8" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>U</mi></mrow><mrow><mi>b</mi><mi>s</mi></mrow></msub></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e009"><math id="M9" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>U</mi></mrow><mrow><mi>c</mi><mi>s</mi></mrow></msub></mrow></math></span> describe base-pairing (Eq. S9), base-stacking (Eq. S6), and coaxial-stacking (Eq. S11) interactions, respectively. The final term, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e010"><math id="M10" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>U</mi></mrow><mrow><mi>e</mi><mi>l</mi></mrow></msub></mrow></math></span> accounts for electrostatic interactions between phosphate groups (i.e., P beads), and is modeled using a Debye-Hückel approximation that incorporates both the counterion condensation theory and the tightly bound ion (TBI) model [<a href="#pcbi.1013346.ref013" class="usa-link" aria-describedby="pcbi.1013346.ref013">13</a>,<a href="#pcbi.1013346.ref058" class="usa-link" aria-describedby="pcbi.1013346.ref058">58</a>,<a href="#pcbi.1013346.ref059" class="usa-link" aria-describedby="pcbi.1013346.ref059">59</a>]:</p>
<table class="disp-formula p" id="pcbi.1013346.e011"><tr>
<td class="formula"><math id="M11" display="block" overflow="linebreak"><mrow><mrow><msub><mi>U</mi><mrow><mi>e</mi><mi>l</mi></mrow></msub></mrow><mo>=</mo><munderover><mo>∑</mo><mrow><mi>i</mi><mo>&lt;</mo><mi>j</mi></mrow><mi>N</mi></munderover><mfrac><mrow><mrow><mrow><msub><mi>Q</mi><mi>i</mi></msub></mrow><mrow><msub><mi>Q</mi><mi>j</mi></msub></mrow><mrow><msup><mi>e</mi><mn>2</mn></msup></mrow></mrow></mrow><mrow><mrow><mn>4</mn><mi>π</mi><mrow><msub><mi>ε</mi><mn>0</mn></msub></mrow><mi>ε</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mi>T</mi><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mrow><msub><mi>r</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></mrow></mfrac><mrow><msup><mi>e</mi><mrow><mo>−</mo><mfrac><mrow><mrow><mrow><msub><mi>r</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></mrow><mrow><mrow><mrow><msub><mi>l</mi><mi>D</mi></msub></mrow></mrow></mrow></mfrac></mrow></msup></mrow><mo>.</mo><mspace width="0.278em"></mspace></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e012"><math id="M12" display="inline" overflow="linebreak"><mrow><mi>e</mi></mrow></math></span> denotes the elementary charge, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e013"><math id="M13" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>r</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></math></span> is the distance between <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e014"><math id="M14" display="inline" overflow="linebreak"><mrow><mi>i</mi></mrow></math></span>- and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e015"><math id="M15" display="inline" overflow="linebreak"><mrow><mi>j</mi></mrow></math></span>-th P beads, and <em>N</em> is the total number of P beads in a DNA chain. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e016"><math id="M16" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>l</mi></mrow><mrow><mi>D</mi></mrow></msub></mrow></math></span> is the Debye length, which characterizes the ionic screening effect and depends on the ionic strength of solution. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e017"><math id="M17" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>ε</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e018"><math id="M18" display="inline" overflow="linebreak"><mrow><mi>ε</mi><mo stretchy="false">(</mo><mi>T</mi><mo stretchy="false">)</mo></mrow></math></span> denote the vacuum permittivity and the effective temperature-dependent dielectric constant, respectively [<a href="#pcbi.1013346.ref041" class="usa-link" aria-describedby="pcbi.1013346.ref041">41</a>,<a href="#pcbi.1013346.ref060" class="usa-link" aria-describedby="pcbi.1013346.ref060">60</a>]. In the present model, each P group is initially assigned a unit negative charge (−1 e). When cations are present in the environment, the effective charge of the phosphate groups is reduced. The reduced charge fraction <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e019"><math id="M19" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>Q</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span> is defined as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e020"><math id="M20" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>Q</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><mn>1</mn><mo>−</mo><msub><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e021"><math id="M21" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span> denotes the ion neutralization fraction for the <em>i</em>-th P bead. Given that complex DNA structures, particularly junctions or loop regions, exhibit distinct charge distributions compared to helical regions, the present model applies a structure-based correction to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e022"><math id="M22" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span> to more accurately reflect local electrostatic effects. For a solution containing pure <em>ν</em>-valent ions, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e023"><math id="M23" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span> is calculated as:</p>
<table class="disp-formula p" id="pcbi.1013346.e024"><tr>
<td class="formula"><math id="M24" display="block" overflow="linebreak"><mrow><mrow><msub><mi>f</mi><mi>i</mi></msub></mrow><mo>=</mo><mfrac><mrow><mrow><mi>N</mi><mspace width="negativethinmathspace"></mspace><mrow><msub><mrow><mover><mi>f</mi><mo stretchy="true">¯</mo></mover></mrow><mi>i</mi></msub></mrow></mrow></mrow><mrow><mrow><mo>∑</mo><msubsup><mi>\nolimits</mi><mi>j</mi><mi>N</mi></msubsup><mrow><msup><mi>e</mi><mrow><mo>−</mo><mi>β</mi><mi>v</mi><mrow><msub><mi>ϕ</mi><mi>i</mi></msub></mrow></mrow></msup></mrow></mrow></mrow></mfrac><mrow><msup><mi>e</mi><mrow><mo>−</mo><mi>β</mi><mi>v</mi><mrow><msub><mi>ϕ</mi><mi>i</mi></msub></mrow></mrow></msup></mrow><mo>,</mo></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e025"><math id="M25" display="inline" overflow="linebreak"><mrow><msub><mrow><mover><mrow><mi>f</mi></mrow><mo accent="true">―</mo></mover></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><mn>1</mn><mo>−</mo><mfrac><mrow><mi>b</mi></mrow><mrow><mo stretchy="false">(</mo><mi>v</mi><msub><mrow><mi>l</mi></mrow><mrow><mi>B</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mfrac></mrow></math></span> is the average neutralization fraction derived from counterion condensation theory, <em>b</em> is the average charge spacing along the DNA backbone, and <em>l</em><sub><em>B</em></sub> is the Bjerrum length. The electrostatic potential <em>ϕ</em><sub><em>i</em></sub> at the <em>i</em>-th P bead is approximated as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e026"><math id="M26" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>ϕ</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>≠</mo><mi>j</mi></mrow><mrow><mi>N</mi></mrow></msubsup><mrow><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mfrac><mrow><msub><mrow><mi>l</mi></mrow><mrow><mi>B</mi></mrow></msub><msub><mrow><mi>Q</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow><mrow><msub><mrow><mi>r</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mfrac><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><msup><mrow><mi>e</mi></mrow><mrow><mo>−</mo><mfrac><mrow><msub><mrow><mi>r</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow><mrow><msub><mrow><mi>l</mi></mrow><mrow><mi>D</mi></mrow></msub></mrow></mfrac></mrow></msup></mrow></mrow></math></span>. The reduced charge fractions <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e027"><math id="M27" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>Q</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span> determined through an iterative procedure: (1) set <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e028"><math id="M28" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><msub><mrow><mover><mrow><mi>f</mi></mrow><mo accent="true">―</mo></mover></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span>, and compute initial <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e029"><math id="M29" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>Q</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span>; (2) calculate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e030"><math id="M30" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>ϕ</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span> using the current <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e031"><math id="M31" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>Q</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span>; (3) update <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e032"><math id="M32" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e033"><math id="M33" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>Q</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span>; (4) repeat steps (2) and (3) until convergence. To reduce computational cost, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e034"><math id="M34" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span> is updated every 100 MC steps unless the secondary structure changes or replica exchange occurs, in which case it is updated immediately to reflect the new conformational environment. For mixed monovalent and divalent ions, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e035"><math id="M35" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><mi>x</mi><msubsup><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow><mrow><mn>1</mn></mrow></msubsup><mo>+</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mi>x</mi><mo stretchy="false">)</mo><msubsup><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e036"><math id="M36" display="inline" overflow="linebreak"><mrow><msubsup><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi>v</mi></mrow></msubsup></mrow></math></span> is the binding fraction of <em>v</em>-valent ions, and <em>x</em> denotes the contribution fractions of monovalent ions, which is derived from the TBI model <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e037"><math id="M37" display="inline" overflow="linebreak"><mrow><mi>x</mi><mo>=</mo><mo stretchy="false">[</mo><mn>1</mn><mo>+</mo><mo stretchy="false">]</mo><mo>/</mo><mo stretchy="false">(</mo><mo stretchy="false">[</mo><mn>1</mn><mo>+</mo><mo stretchy="false">]</mo><mo>+</mo><mi>α</mi><mo stretchy="false">[</mo><mn>2</mn><mo>+</mo><mo stretchy="false">]</mo><mo stretchy="false">)</mo></mrow></math></span>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e038"><math id="M38" display="inline" overflow="linebreak"><mrow><mi>α</mi><mo>=</mo><mo stretchy="false">(</mo><mn>8.1</mn><mo>−</mo><mn>64.8</mn><mo>/</mo><mi>N</mi><mo stretchy="false">)</mo><mo stretchy="false">(</mo><mn>5.2</mn><mo>−</mo><mi>l</mi><mi>n</mi><mo stretchy="false">[</mo><mn>1</mn><mo>+</mo><mo stretchy="false">]</mo><mo stretchy="false">)</mo></mrow></math></span> and [<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e039"><math id="M39" display="inline" overflow="linebreak"><mrow><mi>ν</mi><mo>+</mo><mo stretchy="false">]</mo></mrow></math></span> represents the bulk concentration of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e040"><math id="M40" display="inline" overflow="linebreak"><mrow><mi>υ</mi></mrow></math></span>-valent ion [<a href="#pcbi.1013346.ref048" class="usa-link" aria-describedby="pcbi.1013346.ref048">48</a>]; see more details in <a href="#pcbi.1013346.s013" class="usa-link">S1 Text</a>.</p></section><section id="sec005"><h3 class="pmc_sec_title">Replica-exchange Monte Carlo simulations</h3>
<p>The previous version of our CG model, which used MC simulated annealing algorithm for conformational sampling, successfully predicted the 3D structures of simple DNA topologies such as hairpins, duplexes, and minimal H-type pseudoknots [<a href="#pcbi.1013346.ref013" class="usa-link" aria-describedby="pcbi.1013346.ref013">13</a>]. However, it often became trapped in intermediate states when applied to more complex structures, particularly those with multi-way junctions (≥ 3-way), due to the rugged energy landscape and limited sampling efficiency [<a href="#pcbi.1013346.ref057" class="usa-link" aria-describedby="pcbi.1013346.ref057">57</a>]. To address this limitation, the present model employs a more efficient replica exchange Monte Carlo (REMC) algorithm [<a href="#pcbi.1013346.ref057" class="usa-link" aria-describedby="pcbi.1013346.ref057">57</a>,<a href="#pcbi.1013346.ref061" class="usa-link" aria-describedby="pcbi.1013346.ref061">61</a>], which enhances conformational sampling by enabling exchanges between parallel simulations (replicas) at different temperatures. Specifically, 10 replicas are simulated in parallel at temperatures ranging from 25°C to 110°C (e.g., 25°C, 31°C, 37°C, 45°C, 54°C, 64°C, 74°C, 86°C, 98°C, and 110°C). Each replica undergoes pivot-based conformational updates according to the Metropolis criterion, and adjacent replicas periodically attempt exchanges with probability <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e041"><math id="M41" display="inline" overflow="linebreak"><mrow><mi>p</mi><mo>=</mo><mi>m</mi><mi>i</mi><mi>n</mi><mo stretchy="false">(</mo><mn>1</mn><mo>,</mo><mi>e</mi><mi>x</mi><mi>p</mi><mo stretchy="false">(</mo><mo>−</mo><mrow><mi>Δ</mi></mrow><mo stretchy="false">)</mo><mo stretchy="false">)</mo></mrow></math></span><em>,</em> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e042"><math id="M42" display="inline" overflow="linebreak"><mrow><mrow><mi>Δ</mi></mrow><mo>=</mo><msub><mrow><mi>β</mi></mrow><mrow><mi>j</mi></mrow></msub><mi>E</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi>T</mi></mrow><mrow><mi>j</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>+</mo><msub><mrow><mi>β</mi></mrow><mrow><mi>i</mi></mrow></msub><mi>E</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>,</mo><msub><mrow><mi>T</mi></mrow><mrow><mi>i</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>−</mo><msub><mrow><mi>β</mi></mrow><mrow><mi>i</mi></mrow></msub><mi>E</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi>T</mi></mrow><mrow><mi>i</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>−</mo><msub><mrow><mi>β</mi></mrow><mrow><mi>i</mi></mrow></msub><mi>E</mi><mo stretchy="false">(</mo><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>,</mo><msub><mrow><mi>T</mi></mrow><mrow><mi>j</mi></mrow></msub><mo stretchy="false">)</mo><mo>,</mo></mrow></math></span> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e043"><math id="M43" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>β</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><mn>1</mn><mo>/</mo><mo stretchy="false">(</mo><msub><mrow><mi>k</mi></mrow><mrow><mi>B</mi></mrow></msub><msub><mrow><mi>T</mi></mrow><mrow><mi>i</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e044"><math id="M44" display="inline" overflow="linebreak"><mrow><mi>E</mi><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>T</mi><mo stretchy="false">)</mo></mrow></math></span>denoting the temperature-dependent potential energy of conformation <em>x</em> at temperature T. This generalized Metropolis acceptance criterion ensures that the detailed balance condition is satisfied even when the potential energy function explicitly depends on temperature [<a href="#pcbi.1013346.ref062" class="usa-link" aria-describedby="pcbi.1013346.ref062">62</a>,<a href="#pcbi.1013346.ref063" class="usa-link" aria-describedby="pcbi.1013346.ref063">63</a>].</p></section><section id="sec006"><h3 class="pmc_sec_title">Identifying top-scored coarse-grained 3D structures</h3>
<p>To identify the top-scoring 3D structures from the predicted CG structure ensemble, we first selected 1000 conformations with the lowest CG energies from the predicted ensemble at the lowest temperature (i.e., 25°C). We then applied clustering algorithms to group similar structures by calculating the RMSD values between all pairs of structures within the selected ensemble. The cluster with the largest number of structures, within a predefined RMSD threshold was identified, and its members were removed from the initial set. This process was iteratively repeated until all structures were assigned to a cluster [<a href="#pcbi.1013346.ref064" class="usa-link" aria-describedby="pcbi.1013346.ref064">64</a>], Here, a clustering threshold of 0.1 Å times the sequence length (e.g., 5 Å for a 50-residue sequence) was used here [<a href="#pcbi.1013346.ref064" class="usa-link" aria-describedby="pcbi.1013346.ref064">64</a>]. Finally, the medoids of the three largest clusters, along with the decoy with the lowest energy, were selected as the initial predictions.</p></section><section id="sec007"><h3 class="pmc_sec_title">Rebuilding all-atom structure</h3>
<p>For the top-scoring CG structures (top-1 or top-<em>n</em>), a simple structure refinement process was performed using a MC sampling at room temperature. In this process, the parameters of Para<sub>helix</sub> and Para<sub>loop</sub> were used to calculate the bonded interactions for base-pairing and single-stranded regions, respectively, and other nonbonded interactions were calculated as described earlier. Afterwards, the refined structure was subjected to all-atom reconstruction for practical application.</p>
<p>First, we constructed a small library of five non-redundant all-atom structures for each type of base pairing (G-C, A-T, C-G, T-A) as well as single nucleotides (A, G, C, T) using clustering according to their mutual RMSD values. Then, for each target CG structure, the corresponding nucleotide or base pair was aligned with same sequence fragments from the pre-built library based on CG atom positions. The fragment with the smallest RMSD to the CG atoms was selected as the all-atom replacement for that nucleotide or base pair. This process was repeated for all CG nucleotides until the complete all-atom 3D structure is constructed [<a href="#pcbi.1013346.ref014" class="usa-link" aria-describedby="pcbi.1013346.ref014">14</a>]; see <a href="#pcbi.1013346.s002" class="usa-link">S2 Fig</a>. Finally, to eliminate potential steric clashes and chain breaks in the rebuilt all-atom structures, another refinement step was performed using the QRNAS method [<a href="#pcbi.1013346.ref065" class="usa-link" aria-describedby="pcbi.1013346.ref065">65</a>].</p></section><section id="sec008"><h3 class="pmc_sec_title">Thermal stability analysis using WHAM</h3>
<p>In addition to predicting DNA 3D structures, we applied the WHAM [<a href="#pcbi.1013346.ref066" class="usa-link" aria-describedby="pcbi.1013346.ref066">66</a>] to analyze REMC trajectories and assess DNA thermal stability by calculating the population fractions of structural states across temperatures. This approach enabled us to characterize key thermal properties such as melting temperatures and unfolding pathways. For simplicity, for the simulated DNA, we first categorized the conformations in the DNA ensemble from REMC into different structural states <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e045"><math id="M45" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>S</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></math></span> based on the predicted secondary structures, specifically focusing on the retention of the stems. For instance, a DNA structure with a 3-way junction can be divided into 8 states: folded (<em>F</em>), unfolded (<em>U</em>), and 6 intermediate (<em>I</em>) states corresponding to different stem configurations; see <a href="#pcbi.1013346.s003" class="usa-link">S3 Fig</a> for details. Then, the conformational space was then discretized into bins along two reaction coordinates: structural states <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e046"><math id="M46" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>S</mi></mrow><mrow><mi>j</mi></mrow></msub><mo stretchy="false">(</mo><mi>j</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mi>…</mi><mo>.</mo><mo>,</mo><mi>F</mi><mo stretchy="false">)</mo></mrow></math></span> and energy levels <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e047"><math id="M47" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>E</mi></mrow><mrow><mi>k</mi></mrow></msub><mo stretchy="false">(</mo><mi>k</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mi>…</mi><mo>,</mo><mn>100</mn><mo stretchy="false">)</mo></mrow></math></span>. Each bin <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e048"><math id="M48" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo stretchy="false">)</mo></mrow></math></span> defines a micro-state for WHAM analysis.</p>
<p>The probability <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e049"><math id="M49" display="inline" overflow="linebreak"><mrow><msubsup><mrow><mi>p</mi></mrow><mrow><mo stretchy="false">(</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo stretchy="false">)</mo></mrow><mrow><mrow><mo>∘</mo></mrow></mrow></msubsup></mrow></math></span> of each microstate at the target temperature <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e050"><math id="M50" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span> was computed iteratively using [<a href="#pcbi.1013346.ref066" class="usa-link" aria-describedby="pcbi.1013346.ref066">66</a>]:</p>
<table class="disp-formula p" id="pcbi.1013346.e051"><tr>
<td class="formula"><math id="M51" display="block" overflow="linebreak"><mrow><msubsup><mrow><mi>p</mi></mrow><mrow><mo stretchy="false">(</mo><mi>j</mi><mtext>,</mtext><mi>k</mi><mo stretchy="false">)</mo></mrow><mrow><mrow><mo>∘</mo></mrow></mrow></msubsup><mo>=</mo><mfrac><mrow><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mtext>1</mtext></mrow><mrow><mi>M</mi></mrow></msubsup><msub><mrow><mi>n</mi></mrow><mrow><mi>i</mi><mtext>,</mtext><mo stretchy="false">(</mo><mi>j</mi><mtext>,</mtext><mi>k</mi><mo stretchy="false">)</mo></mrow></msub></mrow><mrow><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mtext>1</mtext></mrow><mrow><mi>M</mi></mrow></msubsup><mrow><msub><mrow><mi>N</mi></mrow><mrow><mi>i</mi></mrow></msub><msub><mrow><mi>Z</mi></mrow><mrow><mi>i</mi></mrow></msub><msub><mrow><mi>c</mi></mrow><mrow><mi>i</mi><mtext>,</mtext><mo stretchy="false">(</mo><mi>j</mi><mtext>,</mtext><mi>k</mi><mo stretchy="false">)</mo></mrow></msub></mrow></mrow></mfrac><mi>;</mi></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<table class="disp-formula p" id="pcbi.1013346.e052"><tr>
<td class="formula"><math id="M52" display="block" overflow="linebreak"><mrow><mtable columnspacing="1" displaystyle="true" rowspacing="3pt"><mtr><mtd columnalign="right"><msubsup><mrow><mi>Z</mi></mrow><mrow><mi>i</mi></mrow><mrow><mo>−</mo><mtext>1</mtext></mrow></msubsup><mo>=</mo><mo>∑</mo><msub><mi>\nolimits</mi><mrow><mi>j</mi><mtext>,</mtext><mi>k</mi></mrow></msub><mrow><msub><mrow><mi>c</mi></mrow><mrow><mi>i</mi><mtext>,</mtext><mo stretchy="false">(</mo><mi>j</mi><mtext>,</mtext><mi>k</mi><mo stretchy="false">)</mo></mrow></msub><msubsup><mrow><mi>p</mi></mrow><mrow><mo stretchy="false">(</mo><mi>j</mi><mtext>,</mtext><mi>k</mi><mo stretchy="false">)</mo></mrow><mrow><mrow><mo>∘</mo></mrow></mrow></msubsup></mrow><mtext>,</mtext></mtd></mtr></mtable></mrow></math></td>
<td class="label">(5)</td>
</tr></table>
<p>where <em>M</em> is the number of replicas, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e053"><math id="M53" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>n</mi></mrow><mrow><mi>i</mi><mo>,</mo><mo stretchy="false">(</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo stretchy="false">)</mo></mrow></msub></mrow></math></span> is the count of microstate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e054"><math id="M54" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo stretchy="false">)</mo></mrow></math></span> at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e055"><math id="M55" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e056"><math id="M56" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>N</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span> is the total number of conformations at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e057"><math id="M57" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e058"><math id="M58" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mi>i</mi><mo>,</mo><mo stretchy="false">(</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo stretchy="false">)</mo></mrow></msub></mrow></math></span> is the temperature bias factor. The relative fraction of each structural state <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e059"><math id="M59" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>S</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></math></span> at any temperature <em>T</em>, denoted <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e060"><math id="M60" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><msub><mrow><mi>S</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></msub><mo stretchy="false">(</mo><mi>T</mi><mo stretchy="false">)</mo></mrow></math></span>, was obtained by summing over energy bins:</p>
<table class="disp-formula p" id="pcbi.1013346.e061"><tr>
<td class="formula"><math id="M61" display="block" overflow="linebreak"><mrow><mrow><msub><mi>f</mi><mrow><mrow><msub><mi>S</mi><mi>j</mi></msub></mrow></mrow></msub></mrow><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mi>T</mi><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>=</mo><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mn>100</mn></mrow></munderover><msubsup><mi>p</mi><mrow><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mrow><mi>j</mi><mo>,</mo><mi>k</mi></mrow><mo fence="true" form="postfix" stretchy="true">)</mo></mrow></mrow><mo>∘</mo></msubsup><mo>.</mo></mrow></math></td>
<td class="label">(6)</td>
</tr></table>
<p>For simplicity, the fractions of the folded state (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e062"><math id="M62" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>F</mi></mrow></msub><mo stretchy="false">(</mo><mi>T</mi><mo stretchy="false">)</mo></mrow></math></span>) and unfolded state (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e063"><math id="M63" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>U</mi></mrow></msub><mo stretchy="false">(</mo><mi>T</mi><mo stretchy="false">)</mo></mrow></math></span>) were fitted to a two-state transition model [<a href="#pcbi.1013346.ref013" class="usa-link" aria-describedby="pcbi.1013346.ref013">13</a>,<a href="#pcbi.1013346.ref067" class="usa-link" aria-describedby="pcbi.1013346.ref067">67</a>]:</p>
<table class="disp-formula p" id="pcbi.1013346.e064"><tr>
<td class="formula"><math id="M64" display="block" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>F</mi></mrow></msub><mo stretchy="false">(</mo><mi>T</mi><mo stretchy="false">)</mo><mo>=</mo><mfrac><mrow><mtext>1</mtext></mrow><mrow><mtext>1</mtext><mo>+</mo><msup><mrow><mi>e</mi></mrow><mrow><mo stretchy="false">(</mo><mi>T</mi><mo>−</mo><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mtext>1</mtext></mrow></msub><mo stretchy="false">)</mo><mo>/</mo><mi>d</mi><msub><mrow><mi>T</mi></mrow><mrow><mtext>1</mtext></mrow></msub></mrow></msup></mrow></mfrac><mi>;</mi></mrow></math></td>
<td class="label">(7)</td>
</tr></table>
<table class="disp-formula p" id="pcbi.1013346.e065"><tr>
<td class="formula"><math id="M65" display="block" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>U</mi></mrow></msub><mo stretchy="false">(</mo><mi>T</mi><mo stretchy="false">)</mo><mo>=</mo><mtext>1</mtext><mo>−</mo><mfrac><mrow><mtext>1</mtext></mrow><mrow><mtext>1</mtext><mo>+</mo><msup><mrow><mi>e</mi></mrow><mrow><mo stretchy="false">(</mo><mi>T</mi><mo>−</mo><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mtext>2</mtext></mrow></msub><mo stretchy="false">)</mo><mo>/</mo><mi>d</mi><msub><mrow><mi>T</mi></mrow><mrow><mtext>2</mtext></mrow></msub></mrow></msup></mrow></mfrac><mo>.</mo></mrow></math></td>
<td class="label">(8)</td>
</tr></table>
<p>Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e066"><math id="M66" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e067"><math id="M67" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span> denote the melting temperatures for the folded-to-intermediate (F → I) and intermediate-to-unfolded (I → U) transitions, respectively. Detailed definitions of the structural states and further information on the WHAM procedure are provided in <a href="#pcbi.1013346.s013" class="usa-link">S1 Text</a>. It should be point out that the temperature dependence of the potential function in the present CG model may impact the WHAM results [<a href="#pcbi.1013346.ref066" class="usa-link" aria-describedby="pcbi.1013346.ref066">66</a>]. However, the simplifications in the structural states and the two-state transition fitting likely mitigate its effects.</p></section></section><section id="sec009"><h2 class="pmc_sec_title">Results and discussion</h2>
<p>In this work, we applied our newly developed CG model to predict the 3D structures of complex DNA molecules, including those with three-way and four-way junctions, extending beyond the simpler ssDNA and dsDNA structures. We then evaluated the thermal stability of these complex DNA structures in both monovalent and divalent ion solutions, surpassing the simple ssDNA, dsDNA, and pseudoknot structures explored in our previous work [<a href="#pcbi.1013346.ref013" class="usa-link" aria-describedby="pcbi.1013346.ref013">13</a>]. Finally, we performed a comprehensive analysis of the thermally unfolding pathways for DNA molecules containing three-way and four-way junctions.</p>
<section id="sec010"><h3 class="pmc_sec_title">Predicting 3D structures of DNA with multi-way junctions</h3>
<section id="sec011"><h4 class="pmc_sec_title">Overview of 3D structure prediction framework.</h4>
<p>As shown in <a href="#pcbi.1013346.g001" class="usa-link">Fig 1</a>, the present model predicts DNA 3D structures from its sequence through five key steps: (1) An initial random conformation of the DNA chain is generated based on the given sequence, utilizing bonded potential <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e068"><math id="M68" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>U</mi></mrow><mrow><mi>b</mi><mi>o</mi><mi>n</mi><mi>d</mi></mrow></msub></mrow></math></span> and the excluded-volume potential <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e069"><math id="M69" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>U</mi></mrow><mrow><mi>e</mi><mi>x</mi><mi>c</mi></mrow></msub></mrow></math></span> (Eq. 1); (2) This conformation undergoes a REMC simulation for conformational sampling, during which only the parameters of Para<sub>loop</sub> for bonded potential are applied to simulate the flexible nature of the DNA chain; (3) Top structures from the REMC ensemble at the lowest temperature (e.g., 25°C) are selected using the CG energy and a clustering algorithm [<a href="#pcbi.1013346.ref068" class="usa-link" aria-describedby="pcbi.1013346.ref068">68</a>]; (4) A further structure refinement (i.e., MC simulation at 25°C) is performed on the top-scoring conformations by applying two distinct sets of bonded potential parameters: Para<sub>loop</sub> for the loop regions and Para<sub>helix</sub> for the base-pairing regions, enabling a more accurate representation of the helical geometry of the stems [<a href="#pcbi.1013346.ref069" class="usa-link" aria-describedby="pcbi.1013346.ref069">69</a>]; (5) Finally, the refined CG structures are reconstructed into all-atom structures.</p>
<p>To evaluate the performance of the present model, two primary metrics (RMSD and F1-score) are calculated for the predicted structures. The RMSD metric, including both the RMSD of the top-ranked structure (top-1 RMSD) and the minimum RMSD observed in the ensemble (RMSDmin), quantifies the global deviation between the predicted and native structures [<a href="#pcbi.1013346.ref070" class="usa-link" aria-describedby="pcbi.1013346.ref070">70</a>,<a href="#pcbi.1013346.ref071" class="usa-link" aria-describedby="pcbi.1013346.ref071">71</a>]. Both top-1 RMSD and RMSDmin are used to assess the structural prediction accuracy of the model. In addition to the RMSD of the top-1 structure (i.e., top-1 RMSD), the structure with minimum RMSD (i.e., RMSD<sub>min</sub>) from the predicted structure ensemble is also used to assess the performance of model. To assess the accuracy of predicted DNA secondary structures, the F1-score, which measures the consistency of base-pairing interactions between the predicted and native structures [<a href="#pcbi.1013346.ref072" class="usa-link" aria-describedby="pcbi.1013346.ref072">72</a>,<a href="#pcbi.1013346.ref073" class="usa-link" aria-describedby="pcbi.1013346.ref073">73</a>], is also calculated by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e070"><math id="M70" display="inline" overflow="linebreak"><mrow><mi>F</mi><mn>1</mn><mo>=</mo><mn>2</mn><mi>×</mi><mi>P</mi><mi>R</mi><mi>×</mi><mi>S</mi><mi>N</mi><mo>/</mo><mo stretchy="false">(</mo><mi>P</mi><mi>R</mi><mo>+</mo><mi>S</mi><mi>N</mi><mo stretchy="false">)</mo></mrow></math></span>, where precision (PR) is defined as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e071"><math id="M71" display="inline" overflow="linebreak"><mrow><mi>P</mi><mi>R</mi><mo>=</mo><mi>T</mi><mi>P</mi><mo>/</mo><mo stretchy="false">(</mo><mi>T</mi><mi>P</mi><mo>+</mo><mi>F</mi><mi>P</mi><mo stretchy="false">)</mo></mrow></math></span> and sensitivity (SN) as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e072"><math id="M72" display="inline" overflow="linebreak"><mrow><mi>S</mi><mi>N</mi><mo>=</mo><mi>T</mi><mi>P</mi><mo>/</mo><mo stretchy="false">(</mo><mi>T</mi><mi>P</mi><mo>+</mo><mi>F</mi><mi>N</mi><mo stretchy="false">)</mo></mrow></math></span>, with TP, FP, and FN denoting true positives, false positives, and false negatives, respectively.</p></section></section><section id="sec012"><h3 class="pmc_sec_title">Validation using experimental DNA junction structures</h3>
<p>In this work, we predicted 3D structures of four DNAs containing three-way (PDB IDs: 1snj, 3hxq, 7qb3) and four-way (PDB ID: 2f1q) junctions under standard salt conditions (1M NaCl) [<a href="#pcbi.1013346.ref074" class="usa-link" aria-describedby="pcbi.1013346.ref074">74</a>–<a href="#pcbi.1013346.ref076" class="usa-link" aria-describedby="pcbi.1013346.ref076">76</a>]. As shown in <a href="#pcbi.1013346.g002" class="usa-link">Fig 2</a>, both secondary and tertiary structures were compared with experimentally determined structures. The predicted secondary structures (parsed by DSSR from 3D models with Top-1 RMSD and RMSD<sub>min</sub>) largely recapitulate the native base-pairing patterns across all four DNAs, as evidenced by high F1-scores ranging from 0.84 to 0.95. This highlights the ability of model to recover key structural motifs from sequence alone. However, discrepancies remain in certain local and non-canonical base pairs (e.g., C11-G14/C21-G24 in 1snj, G23-T35 in 3hxq, G17-T23 in 7qb3, and C11-G14/C19-G22/C28-G31 in 2f1q). These deviations mainly arise from model assumptions that restrict base pairing to nucleotides separated by at least three positions and exclude non-canonical interactions such as G-T pairs, which should be further improved in future.</p>
<figure class="fig xbox font-sm" id="pcbi.1013346.g002"><h4 class="obj_head">Fig 2. Comparison of predicted secondary and 3D structures of DNA junctions with RMSD<sub>min</sub> and top-1 RMSD using our CG model.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373291_pcbi.1013346.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9592/12373291/dfa2dff81b4b/pcbi.1013346.g002.jpg" loading="lazy" height="820" width="750" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013346.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A-D)</strong> F1 scores and RMSD values for the minimum RMSD and top-1 RMSD predicted structures of DNAs with three-way junctions (A-C) and four-way junctions (D). The predicted 3D structures with minimum RMSD (brown) and top-1 RMSD (pink) are superimposed on the corresponding native structures (blue). Secondary structures were visualized using VARNA [<a href="#pcbi.1013346.ref092" class="usa-link" aria-describedby="pcbi.1013346.ref092">92</a>], and 3D structures were rendered in PyMOL [<a href="#pcbi.1013346.ref093" class="usa-link" aria-describedby="pcbi.1013346.ref093">93</a>].</p></figcaption></figure><p>All four DNA molecules lack long-range tertiary base pairs (<a href="#pcbi.1013346.g002" class="usa-link">Fig 2</a>), which poses a significant challenge for accurately predicting the relative orientations between stems connected by junctions. Nevertheless, the best predicted 3D structures (RMSD<sub>min</sub>) achieve RMSD values ranging from 5.6 to 8.3 Å, while the top-ranked models (Top-1) fall within 8.1-9.2 Å. These results likely benefit from the coaxial stacking interactions in the present model, as well as efficient sampling enabled by the pivot move-based algorithm [<a href="#pcbi.1013346.ref056" class="usa-link" aria-describedby="pcbi.1013346.ref056">56</a>,<a href="#pcbi.1013346.ref057" class="usa-link" aria-describedby="pcbi.1013346.ref057">57</a>].</p>
<p>Collectively, these results demonstrate that the present model can accurately capture both secondary and tertiary structures of DNAs with multi-branched architectures, offering a powerful tool for modeling DNA structures beyond simple duplexes/hairpins.</p></section><section id="sec013"><h3 class="pmc_sec_title">Comparison with existing methods</h3>
<p>To benchmark the present model, we compared it with two representative methods for DNA 3D structure prediction: 3dRNA/DNA [<a href="#pcbi.1013346.ref077" class="usa-link" aria-describedby="pcbi.1013346.ref077">77</a>] and AlphaFold3 [<a href="#pcbi.1013346.ref018" class="usa-link" aria-describedby="pcbi.1013346.ref018">18</a>], using only sequence information as input for all approaches. For 3dRNA/DNA, we used its webserver (<a href="http://biophy.hust.edu.cn/new/3dRNA/create" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://biophy.hust.edu.cn/new/3dRNA/create</a>) with the default ‘RNAfold’ settings for secondary structure prediction and the ‘Optimization’ option for 3D structure refinement. For AlphaFold3, end-to-end structure predictions were obtained via its webserver (<a href="https://alphafoldserver.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://alphafoldserver.com/</a>).</p>
<p><a href="#pcbi.1013346.g003" class="usa-link">Fig 3A</a> and <a href="#pcbi.1013346.g003" class="usa-link">3C</a> display the RMSDs and F1-scores of the top-ranked (Top-1) structures predicted by three methods for each DNA molecule. For the four DNAs, the RMSDs predicted by the present model are consistently lower than those obtained from 3dRNA/DNA, with two of the predictions outperforming those from AlphaFold3 (<a href="#pcbi.1013346.g003" class="usa-link">Fig 3A</a>). As shown in <a href="#pcbi.1013346.g003" class="usa-link">Fig 3B</a>, for the four DNAs, the average RMSD for the top-ranked (Top-1) predictions from our model is approximately 8.9 Å, representing an improvement of about 43.6% compared to 3dRNA/DNA (~15.6 Å). This improvement is largely attributed to the significantly higher accuracy of our predicted secondary structures (<a href="#pcbi.1013346.g003" class="usa-link">Fig 3C</a> and <a href="#pcbi.1013346.g003" class="usa-link">3D</a>), highlighting the limitations of fragment assembly approaches that heavily depend on secondary structure inputs [<a href="#pcbi.1013346.ref026" class="usa-link" aria-describedby="pcbi.1013346.ref026">26</a>]. AlphaFold3 achieves an average RMSD of ~9.2 Å, which is comparable to our Top-1 results, but still higher than the best structures predicted by our model (RMSD<sub>min</sub>). It is important to note that the experimental structures of these four DNAs were released between 2004 and 2021, while AlphaFold3 was trained on all macromolecular structures available in the PDB prior to January 2023 [<a href="#pcbi.1013346.ref018" class="usa-link" aria-describedby="pcbi.1013346.ref018">18</a>]. This suggests that these DNAs were likely included in its training set, which may explain the near-perfect agreement between its predicted and native base-pairing patterns (<a href="#pcbi.1013346.g003" class="usa-link">Fig 3C</a> and <a href="#pcbi.1013346.s010" class="usa-link">S4 Table</a>). Nevertheless, the performance of AlphaFold3 on these DNAs is notably lower than its typical accuracy (&lt; 3 Å) for protein structure prediction [<a href="#pcbi.1013346.ref018" class="usa-link" aria-describedby="pcbi.1013346.ref018">18</a>,<a href="#pcbi.1013346.ref019" class="usa-link" aria-describedby="pcbi.1013346.ref019">19</a>]. This discrepancy underscores the limited availability of high-quality DNA structural data and highlights the continued need for physics-based modeling approaches.</p>
<figure class="fig xbox font-sm" id="pcbi.1013346.g003"><h4 class="obj_head">Fig 3. Performance comparison of our CG model with 3dRNA/DNA and AlphaFold3 in predicting the 3D structures of DNA molecules with multi-way junctions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373291_pcbi.1013346.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9592/12373291/150d0bb7221a/pcbi.1013346.g003.jpg" loading="lazy" height="543" width="756" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013346.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A, C)</strong> RMSD values (A) and F1 scores (C) for four DNAs with multi-way junctions predicted by our CG model, 3dRNA/DNA, and AlphaFold3. <strong>(B, D)</strong> Average RMSD values (B) and F1 scores (D) across the four DNAs for each method. Error bars represent the standard deviation of the RMSD and F1 score values.</p></figcaption></figure></section><section id="sec014"><h3 class="pmc_sec_title">Predicting thermal stabilities of DNA junctions under ion conditions</h3>
<p>DNA functionality depends not only on its 3D structure but also on its thermal stability [<a href="#pcbi.1013346.ref013" class="usa-link" aria-describedby="pcbi.1013346.ref013">13</a>]. Beyond 3D structure prediction, the present model enables quantitative prediction of the thermal unfolding behavior of DNAs containing complex motifs such as three-way (3WJ) and four-way junctions (4WJ) in monovalent and divalent ionic environments.</p>
<section id="sec015"><h4 class="pmc_sec_title">Predicting thermal stabilities of DNA with multi-way junctions.</h4>
<p>We tested the present model on six DNAs, three with 3-way junctions and three with 4-way junctions, whose sequences and structural properties are summarized in <a href="#pcbi.1013346.t001" class="usa-link">Table 1</a> and <a href="#pcbi.1013346.s004" class="usa-link">S4 Fig</a>. As described in the <em>Materials and Methods</em> section, for each DNA, we combined REMC simulations with the WHAM to compute two distinct melting temperatures: one corresponding to the transition from the folded state to an intermediate state (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e073"><math id="M73" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span>), and the other from an intermediate state to the unfolded state (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e074"><math id="M74" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span>).</p>
<section class="tw xbox font-sm" id="pcbi.1013346.t001"><h5 class="obj_head">Table 1. The predicted melting temperature (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e184"><math id="M184" display="inline" overflow="linebreak"><mrow><mrow><msub><mrow><mi mathvariant="bold-italic">T</mi></mrow><mrow><mi mathvariant="bold-italic">m</mi><mn mathvariant="bold">1</mn></mrow></msub></mrow></mrow></math></span><sup><em>a</em></sup> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e185"><math id="M185" display="inline" overflow="linebreak"><mrow><mrow><msub><mrow><mi mathvariant="bold-italic">T</mi></mrow><mrow><mi mathvariant="bold-italic">m</mi><mn mathvariant="bold">2</mn></mrow></msub></mrow></mrow></math></span><sup><em>b</em></sup>) of 6 DNAs by our present model.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">DNAs</th>
<th align="left" rowspan="1" colspan="1">Type</th>
<th align="left" rowspan="1" colspan="1">Refs</th>
<th align="left" rowspan="1" colspan="1">Length (nt)</th>
<th align="left" rowspan="1" colspan="1">Predicted <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e186"><math id="M186" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">T</mi></mrow></mrow><mrow><mrow><mrow><mi mathvariant="bold-italic">m</mi></mrow></mrow><mrow><mrow><mn mathvariant="bold">1</mn></mrow></mrow></mrow></msub></mrow></math></span>/<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e187"><math id="M187" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">T</mi></mrow></mrow><mrow><mrow><mrow><mi mathvariant="bold-italic">m</mi></mrow></mrow><mrow><mrow><mn mathvariant="bold">2</mn></mrow></mrow></mrow></msub></mrow></math></span> (°C)</th>
<th align="left" rowspan="1" colspan="1">Expt. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e188"><math id="M188" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">T</mi></mrow></mrow><mrow><mrow><mrow><mi mathvariant="bold-italic">m</mi></mrow></mrow><mrow><mrow><mn mathvariant="bold">1</mn></mrow></mrow></mrow></msub></mrow></math></span>/<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e189"><math id="M189" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">T</mi></mrow></mrow><mrow><mrow><mrow><mi mathvariant="bold-italic">m</mi></mrow></mrow><mrow><mrow><mn mathvariant="bold">2</mn></mrow></mrow></mrow></msub></mrow></math></span> (°C)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">3WJ</td>
<td align="left" rowspan="1" colspan="1">Three-way</td>
<td align="left" rowspan="1" colspan="1">[<a href="#pcbi.1013346.ref078" class="usa-link" aria-describedby="pcbi.1013346.ref078">78</a>]</td>
<td align="left" rowspan="1" colspan="1">37</td>
<td align="left" rowspan="1" colspan="1">34.4<sup><em>c</em></sup>/64.6<sup><em>c</em></sup>
</td>
<td align="left" rowspan="1" colspan="1">33.6<sup><em>c</em></sup>/70.3<sup><em>c</em></sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">L-3WJ</td>
<td align="left" rowspan="1" colspan="1">Three-way</td>
<td align="left" rowspan="1" colspan="1">[<a href="#pcbi.1013346.ref078" class="usa-link" aria-describedby="pcbi.1013346.ref078">78</a>]</td>
<td align="left" rowspan="1" colspan="1">37</td>
<td align="left" rowspan="1" colspan="1">36.2<sup><em>c</em></sup>/82.5<sup><em>c</em></sup>
</td>
<td align="left" rowspan="1" colspan="1">35.8<sup><em>c</em></sup>/87.6<sup><em>c</em></sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">R-3WJ</td>
<td align="left" rowspan="1" colspan="1">Three-way</td>
<td align="left" rowspan="1" colspan="1">[<a href="#pcbi.1013346.ref078" class="usa-link" aria-describedby="pcbi.1013346.ref078">78</a>]</td>
<td align="left" rowspan="1" colspan="1">37</td>
<td align="left" rowspan="1" colspan="1">50.4<sup><em>c</em></sup> (54.2<sup><em>d</em></sup>)/73.5<sup><em>c</em></sup>
</td>
<td align="left" rowspan="1" colspan="1">54.6<sup><em>d</em></sup>/77.9<sup><em>c</em></sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">4WJ</td>
<td align="left" rowspan="1" colspan="1">Four-way</td>
<td align="left" rowspan="1" colspan="1">[<a href="#pcbi.1013346.ref078" class="usa-link" aria-describedby="pcbi.1013346.ref078">78</a>]</td>
<td align="left" rowspan="1" colspan="1">53</td>
<td align="left" rowspan="1" colspan="1">28.6<sup><em>c</em></sup>/72.1<sup><em>c</em></sup>
</td>
<td align="left" rowspan="1" colspan="1">51.2<sup><em>e</em></sup>/65.0<sup><em>e</em></sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">L-4WJ</td>
<td align="left" rowspan="1" colspan="1">Four-way</td>
<td align="left" rowspan="1" colspan="1">[<a href="#pcbi.1013346.ref078" class="usa-link" aria-describedby="pcbi.1013346.ref078">78</a>]</td>
<td align="left" rowspan="1" colspan="1">52</td>
<td align="left" rowspan="1" colspan="1">32.6<sup><em>c</em></sup>/83.1<sup><em>c</em></sup>
</td>
<td align="left" rowspan="1" colspan="1">52.4<sup><em>f</em></sup>/87.9<sup><em>f</em></sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">R-4WJ</td>
<td align="left" rowspan="1" colspan="1">Four-way</td>
<td align="left" rowspan="1" colspan="1">[<a href="#pcbi.1013346.ref078" class="usa-link" aria-describedby="pcbi.1013346.ref078">78</a>]</td>
<td align="left" rowspan="1" colspan="1">52</td>
<td align="left" rowspan="1" colspan="1">31.7<sup><em>c</em></sup>/81.3<sup><em>c</em></sup>
</td>
<td align="left" rowspan="1" colspan="1">52.8<sup><em>g</em></sup>/72.9<sup><em>g</em></sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pcbi.1013346.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t001fn001"><p><sup>a</sup><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e190"><math id="M190" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> and <sup><em>b</em></sup><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e191"><math id="M191" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span> are the melting temperatures for the transitions from folded state to intermediate state and from intermediate state to unfolded state, respectively. <sup><em>c</em></sup>The melting temperature <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e192"><math id="M192" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> (3 stems transformed into 2 stems) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e193"><math id="M193" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span> (1 stem transformed into 0 stem). <sup><em>d</em></sup>The melting temperature <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e194"><math id="M194" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> (3 stems transformed into 1 stem). <sup><em>e</em></sup>The melting temperature <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e195"><math id="M195" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> (4 stems transformed into 1.5 stem) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e196"><math id="M196" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span> (1.5 stem transformed into 0 stem). <sup><em>f</em></sup> The melting temperature <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e197"><math id="M197" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> (4 stems transformed into 1.5 stem) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e198"><math id="M198" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span> (1 stem transformed into 0 stem). <sup><em>g</em></sup> The melting temperature <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e199"><math id="M199" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> (4 stems transformed into ~1.5 stem) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e200"><math id="M200" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span> (~1.5 stem transformed into 0 stem).</p></div></div></section><p>As illustrated in <a href="#pcbi.1013346.g004" class="usa-link">Fig 4A</a>–<a href="#pcbi.1013346.g004" class="usa-link">4C</a>, taking a 37-nt DNA (the 3WJ with sequence of 5’-GAAATTGCGCTTTTTGCGCGTGCTTTTTGCACAATTTC-3’) as an example [<a href="#pcbi.1013346.ref078" class="usa-link" aria-describedby="pcbi.1013346.ref078">78</a>], the REMC simulations at 0.1M NaCl yield predictions of its secondary structure (<a href="#pcbi.1013346.g004" class="usa-link">Fig 4A</a>). Based on the base-pairing configurations observed in REMC trajectories at different temperatures (<a href="#pcbi.1013346.g004" class="usa-link">Fig 4B</a>), the conformational ensemble can be grouped into eight microstates, including the folded state, the unfolded state, and six intermediate states (<a href="#pcbi.1013346.s003" class="usa-link">S3 Fig</a>). WHAM was then used to compute the fractional populations of each state across temperatures. Finally, using the temperature-dependent fractions of the folded and unfolded states (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e075"><math id="M75" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>F</mi></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e076"><math id="M76" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>U</mi></mrow></msub></mrow></math></span>), we fit two-state transition models to extract the two melting temperatures (<a href="#pcbi.1013346.g004" class="usa-link">Fig 4C</a>), representing the folded-to-intermediate and intermediate-to-unfolded transitions, respectively. The predicted two melting temperature: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e077"><math id="M77" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> = 34.4°C and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e078"><math id="M78" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span> = 64.6°C, are close to experimental data (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e079"><math id="M79" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> = 33.6°C and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e080"><math id="M80" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span> = 70.3°C), with deviations within ~2–6 °C (<a href="#pcbi.1013346.t001" class="usa-link">Table 1</a>). In addition, the <a href="#pcbi.1013346.g004" class="usa-link">Fig 4D</a>–<a href="#pcbi.1013346.g004" class="usa-link">4F</a> also present the predicted results for another DNA structure featuring a 4-way junction (4WJ, 53nt; sequence: 5’-GAAATTGCGCTTTTTGCGCATATCTTTTTGATAGGTGCTTTTTGCACAATTTC-3’) [<a href="#pcbi.1013346.ref078" class="usa-link" aria-describedby="pcbi.1013346.ref078">78</a>]. Compared to the 3-way junction case, this system exhibits more complex structural fluctuations (<a href="#pcbi.1013346.g004" class="usa-link">Fig 4E</a>), but follow similar thermal behavior. The predicted melting temperatures are <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e081"><math id="M81" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> = 28.6°C and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e082"><math id="M82" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span> = 72.1°C, showing notable deviation from the experimental values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e083"><math id="M83" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> = 51.2°C and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e084"><math id="M84" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span> = 65.0°C, especially in the first transition temperature. This discrepancy is likely due to the current simplification of model, which considers only folded and unfolded states while neglecting the stability differences among intermediate states.</p>
<figure class="fig xbox font-sm" id="pcbi.1013346.g004"><h5 class="obj_head">Fig 4. Predicted melting behavior of DNAs with 3-way (3WJ) and 4-way (4WJ) junctions using our CG model.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373291_pcbi.1013346.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9592/12373291/d408bc1b8594/pcbi.1013346.g004.jpg" loading="lazy" height="513" width="750" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013346.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A, D)</strong> Predicted secondary and 3D structures of the 3WJ (A) and 4WJ (D) by our CG model. <strong>(B, E)</strong> Time evolution of the base-pair fractions for 3WJ (B) and 4WJ (E) at different temperatures (110°C, 74°C, 45°C, 25°C, from bottom to top). <strong>(C, F)</strong> Temperature-dependent fractions of the folded (F, green) and unfolded (U, red) states for 3WJ (C) and 4WJ (F) in 1M NaCl.</p></figcaption></figure><p><a href="#pcbi.1013346.t001" class="usa-link">Table 1</a> further summarizes the predictions for four additional DNAs (L-3WJ, R-3WJ, L-4WJ, and R-4WJ) [<a href="#pcbi.1013346.ref078" class="usa-link" aria-describedby="pcbi.1013346.ref078">78</a>]. For the three DNAs with three-way junctions (3WJ, L-3WJ, and R-3WJ), the predicted melting temperatures (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e085"><math id="M85" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e086"><math id="M86" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span>) show good agreement with experimental data. The mean deviations are <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e087"><math id="M87" display="inline" overflow="linebreak"><mrow><mi>~</mi></mrow></math></span>1.8°C for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e088"><math id="M88" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e089"><math id="M89" display="inline" overflow="linebreak"><mrow><mi>~</mi></mrow></math></span>4.9°C for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e090"><math id="M90" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span>, suggesting that the model reliably captures the thermal stability of DNA with three-way junctions [<a href="#pcbi.1013346.ref078" class="usa-link" aria-describedby="pcbi.1013346.ref078">78</a>]. Additionally, for R-3WJ, the predicted melting temperature <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e091"><math id="M91" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> of 50.4°C is lower than experimental value (54.6°C). This could be because the experimental <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e092"><math id="M92" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> corresponds to the melting of two stems, while the model defined <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e093"><math id="M93" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> refers to the melting of a single stem. To address this discrepancy, we redefined the intermediate state containing two stems as the folded state, yielding a revised model prediction of 54.2 °C, closely matching the experimental result.</p>
<p>For DNAs with four-way junctions (4WJ, L-4WJ, and R-4WJ), a similar challenge arises. In experiments, the melting transitions often involve the cooperative melting of multiple stems [<a href="#pcbi.1013346.ref078" class="usa-link" aria-describedby="pcbi.1013346.ref078">78</a>], whereas our model treats transitions at the single-stem level. As a result, the predicted <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e094"><math id="M94" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> values are consistently lower than the experimental ones, while <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e095"><math id="M95" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span> values tend to be higher (<a href="#pcbi.1013346.t001" class="usa-link">Table 1</a>). This is because the experimentally determined melting temperatures likely correspond to more stable intermediate states involving more than one stem, whereas our model simplifies these to a single-stem melting event. Nevertheless, the overall thermal transition behavior is well captured, supporting the predictive reliability of the model.</p></section><section id="sec016"><h4 class="pmc_sec_title">Effect of monovalent ion on the stability of DNAs with multi-way.</h4>
<p>DNA 3D structures and their thermal stabilities are highly sensitive to the ionic conditions due to the polyanionic nature of the DNA backbone [<a href="#pcbi.1013346.ref044" class="usa-link" aria-describedby="pcbi.1013346.ref044">44</a>,<a href="#pcbi.1013346.ref045" class="usa-link" aria-describedby="pcbi.1013346.ref045">45</a>,<a href="#pcbi.1013346.ref079" class="usa-link" aria-describedby="pcbi.1013346.ref079">79</a>]. In this work, we examined the effects of the monovalent salt (Na<sup>+</sup>) on DNA stability using two representative systems: a 3-way junctions (3WJ) and a 4-way junctions (4WJ) [<a href="#pcbi.1013346.ref078" class="usa-link" aria-describedby="pcbi.1013346.ref078">78</a>].</p>
<p>We further extended our predictions to a wider range of Na<sup>+</sup> concentrations (see <a href="#pcbi.1013346.s011" class="usa-link">S5</a> and <a href="#pcbi.1013346.s012" class="usa-link">S6 Tables</a>). As shown in <a href="#pcbi.1013346.g005" class="usa-link">Fig 5A</a>, the stabilities of both the folded and intermediate hairpin structures increase with rising [Na<sup>+</sup>], which can be attributed to the stronger electrostatic screening at higher [Na<sup>+</sup>], particularly beneficial for compactly folded or intermediate conformations. Interestingly, we observed that the increase in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e096"><math id="M96" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span> (unfolding of intermediate hairpin to single strand) with Na<sup>+</sup> concentration is slightly pronounced than that of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e097"><math id="M97" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> (transition from fully folded to intermediate state). This is likely due to the fact that the difference in charge density between the intermediate hairpin and fully unfolded state is typically larger than that between the fully folded state and the intermediate state with a single stem. As shown in <a href="#pcbi.1013346.s005" class="usa-link">S5A Fig</a>, for the 3WJ, the average radius of gyration (Rg) of the folded state is 16.9Å, which is only slightly smaller than that (17.6Å) of the intermediate state with one melted stem. In contrast, the Rg of the fully unfolded conformations (25.2Å) is significantly larger than that of the intermediate hairpin state (22.9Å), indicating a more extended and less compact structure. In addition, as shown in <a href="#pcbi.1013346.s005" class="usa-link">S5B Fig</a>, the average ion neutralization fraction (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e098"><math id="M98" display="inline" overflow="linebreak"><mrow><mi>f</mi></mrow></math></span>) of the folded state (0.54) is only marginally higher than that of the one-stem-melted intermediate (0.48), whereas the fully unfolded conformations have a notably lower <em>f</em> value (0.37) than the hairpin intermediate (0.46). Similarly, for the 4WJ (<a href="#pcbi.1013346.s005" class="usa-link">S5C</a> and <a href="#pcbi.1013346.s005" class="usa-link">S5D Fig</a>), the folded state exhibits an average Rg of 19.2 Å, also slightly smaller than that of the one-melted-stem intermediate state (19.8 Å). In contrast, the fully unfolded conformations (31.4Å) exhibit a much larger Rg compared to the intermediate hairpin state (27.6Å). Regarding ion neutralization, the folded state shows a slightly higher <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e099"><math id="M99" display="inline" overflow="linebreak"><mrow><mi>f</mi></mrow></math></span> (0.62) than the single-melted-stem intermediate (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e100"><math id="M100" display="inline" overflow="linebreak"><mrow><mi>f</mi></mrow></math></span> = 0.60), whereas the fully unfolded state displays a significantly lower value <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e101"><math id="M101" display="inline" overflow="linebreak"><mrow><mi>f</mi></mrow></math></span> (0.47) relative to the three-stem-melted intermediate (0.52). These suggests that the electrostatic stabilization effect of Na<sup>+</sup> is more prominent during the transition from intermediate to fully unfolded states, thereby explaining the sharper increase in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e102"><math id="M102" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span> with ion concentration.</p>
<figure class="fig xbox font-sm" id="pcbi.1013346.g005"><h5 class="obj_head">Fig 5. Comparison of predicted (lines) and experimental (symbols) melting temperatures for 3WJ and 4WJ.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373291_pcbi.1013346.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9592/12373291/d1b5852b999c/pcbi.1013346.g005.jpg" loading="lazy" height="480" width="733" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013346.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Melting temperatures of the folded-to-intermediate (F → I, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e159"><math id="M159" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span>) and intermediate-to-unfolded (I → U, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e160"><math id="M160" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span>) transitions for 3WJ as a function of [Na<sup>+</sup>]. <strong>(B, C)</strong> Melting temperature (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e161"><math id="M161" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e162"><math id="M162" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span>) for 3WJ as a function of [Mg<sup>2+</sup>] at fixed [Na<sup>+</sup>] of 10 mM (B) and 100 mM (C). <strong>(D)</strong> Melting temperature for 4WJ of the F → I (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e163"><math id="M163" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span>) and I → U (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e164"><math id="M164" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span>) transitions as a function of [Na<sup>+</sup>]. <strong>(E, F)</strong> Melting temperature of 4WJ as a function of [Mg<sup>2+</sup>], with [Na<sup>+</sup>] fixed at 10 mM (E) and 100 mM (F).</p></figcaption></figure></section><section id="sec017"><h4 class="pmc_sec_title">Effect of divalent ion on the stability of DNAs with multi-way junction.</h4>
<p>To further investigate the ionic effects on DNA folding, we analyzed the predicted melting temperatures (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e103"><math id="M103" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e104"><math id="M104" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span>) of DNA with multi-way junctions across a wide range of [Mg<sup>2+</sup>], while fixing [Na<sup>+</sup>] at 10 mM (<a href="#pcbi.1013346.g005" class="usa-link">Fig 5B</a>) and 100 mM (<a href="#pcbi.1013346.g005" class="usa-link">Fig 5C</a>), respectively. The results reveal that the present model successfully captures the competitive and cooperative interactions between monovalent (Na<sup>+</sup>) and divalent (Mg<sup>2+</sup>) ions in modulating the stability of DNA with three-way junctions (3WJ). At low [Mg<sup>2+</sup>] (e.g., 1 mM), the stability of 3WJ is dominated by the background Na<sup>+</sup> concentration, and the predicted temperatures (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e105"><math id="M105" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>1</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e106"><math id="M106" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>m</mi><mn>2</mn></mrow></msub></mrow></math></span>) closely resemble those observed in pure Na<sup>+</sup> solutions. As [Mg<sup>2+</sup>] increases (~&gt;1 mM), the stability of the 3WJ structure is significantly enhanced due to the stronger charge neutralization of Mg<sup>2+</sup>. This stabilizing effect saturates at higher [Mg<sup>2+</sup>] (~&gt;100 mM). This phenomenon is due to the anti-cooperative binding between Na<sup>+</sup> and Mg<sup>2+</sup>, coupled with the more efficient stabilization provided by Mg<sup>2+</sup>. A similar trend is observed for four-way junctions (4WJ), as shown in <a href="#pcbi.1013346.g005" class="usa-link">Fig 5D</a>–<a href="#pcbi.1013346.g005" class="usa-link">5F</a>, further supporting the robustness of the model in describing the thermodynamic effects of divalent cations.</p>
<p>These findings indicate that the present model can quantitatively capture the influence of both monovalent and divalent ions on the thermal stability of complex DNA junction structures. However, recent studies [<a href="#pcbi.1013346.ref047" class="usa-link" aria-describedby="pcbi.1013346.ref047">47</a>,<a href="#pcbi.1013346.ref080" class="usa-link" aria-describedby="pcbi.1013346.ref080">80</a>,<a href="#pcbi.1013346.ref081" class="usa-link" aria-describedby="pcbi.1013346.ref081">81</a>] have shown that at very high ionic concentrations, excessive ion binding can lead to overcharging effects, which in turn destabilize DNA structures. Since the implicit ion model used in our model assumes a maximal screening effect, effectively neutralizing the DNA backbone charges, it is inherently unable to capture such overcharging phenomena. This limitation suggests that incorporating explicit ion representations, particularly for multivalent ions, may be essential for accurately modeling DNA stability under extreme ionic conditions [<a href="#pcbi.1013346.ref041" class="usa-link" aria-describedby="pcbi.1013346.ref041">41</a>,<a href="#pcbi.1013346.ref054" class="usa-link" aria-describedby="pcbi.1013346.ref054">54</a>,<a href="#pcbi.1013346.ref055" class="usa-link" aria-describedby="pcbi.1013346.ref055">55</a>].</p></section></section><section id="sec018"><h3 class="pmc_sec_title">Thermally unfolding pathway of DNAs with multi-way junctions</h3>
<p>Understanding the unfolding pathways of DNA multi-way junctions is essential for elucidating their structure-function relationships [<a href="#pcbi.1013346.ref078" class="usa-link" aria-describedby="pcbi.1013346.ref078">78</a>]. The present model not only accurately predicts the melting temperatures of DNA three- and four-way junctions but also reveals the detailed thermally induced transitions among folded, intermediate, and unfolded states based on their temperature-dependent fractions calculated from REMC trajectories using WHAM. In this work, we focus on two typical DNA with multi-way junctions structures 3WJ (DNA with three-way junction) and 4WJ (DNA with four-way junction) to analyze their unfolding pathways beyond the minimal ssDNA and dsDNA [<a href="#pcbi.1013346.ref013" class="usa-link" aria-describedby="pcbi.1013346.ref013">13</a>]. Here, the intermediate states are denoted as I1, I2,..., corresponding to the number of melting stems, while states without labels or those labeled with ‘<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e107"><math id="M107" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>’, ‘<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e108"><math id="M108" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>’, ‘<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e109"><math id="M109" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>’, etc., represent states with the highest, second-highest, third-highest fraction, and so on.</p>
<section id="sec019"><h4 class="pmc_sec_title">For a DNA with three-way junctions (3WJ).</h4>
<p>At 1 M NaCl, the 3WJ structure exhibits a stepwise unfolding mechanism involving three key intermediates (I1, I2, and I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e110"><math id="M110" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>) (<a href="#pcbi.1013346.g006" class="usa-link">Fig 6</a>). As shown in <a href="#pcbi.1013346.g006" class="usa-link">Fig 6A</a>, at low temperatures (&lt;~30°C), the 3WJ remains predominantly in the fully folded state (F). As the temperature increases from ~30°C to ~50°C, the fraction of F decreases from ~99% to ~33%, while the fractions of intermediate states I1, I2, and I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e111"><math id="M111" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span> gradually increase to ~31%, ~ 30%, and ~5%, respectively. Between ~50°C and ~70°C, the fractions of the F and I1 further decline to ~2% and ~9%, whereas I2 and I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e112"><math id="M112" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span> reach their peak values of ~45% and ~10%, respectively. At ~110°C, the system transitions into the unfolded state (U). These observations suggest that the dominant unfolding pathway of the 3WJ follows F → I1 → I2 → U, with I2 being the most populated intermediate at ~70°C. Additionally, two minor pathways are identified: F → I2 → U and F → I1 → I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e113"><math id="M113" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>→U, with the former having a slightly higher flux than the latter.</p>
<figure class="fig xbox font-sm" id="pcbi.1013346.g006"><h5 class="obj_head">Fig 6. Predicted thermally unfolding pathway of 3WJ at 1 M [Na<sup>+</sup>] using our CG model.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373291_pcbi.1013346.g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9592/12373291/48856343412c/pcbi.1013346.g006.jpg" loading="lazy" height="646" width="753" alt="Fig 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013346.g006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Temperature-dependent population fractions of different structural states: fully folded (F), intermediate (I1, I2, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e165"><math id="M165" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>), and fully unfolded (U). F represents the fully folded conformation, I1 corresponds to the intermediate state with Stems 2 and 3 melted, I2/I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e166"><math id="M166" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span> represent the hairpin intermediates with Stem 2/Stem 3, and U denotes the fully unfolded structure. <strong>(B-D)</strong> Temperature-dependent free energies of various intermediate states (I1, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e167"><math id="M167" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e168"><math id="M168" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I2, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e169"><math id="M169" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>, and I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e170"><math id="M170" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>) calculated using Mfold. <strong>(E)</strong> Schematic representation of the structure transitions along the unfolding pathway inferred from the state fractions shown in panel (A).</p></figcaption></figure><p>Given that the inferred unfolding pathways of the 3WJ have not been experimentally validated, we employed the well-established nearest-neighbor thermodynamic model (i.e., Mfold [<a href="#pcbi.1013346.ref067" class="usa-link" aria-describedby="pcbi.1013346.ref067">67</a>,<a href="#pcbi.1013346.ref082" class="usa-link" aria-describedby="pcbi.1013346.ref082">82</a>]) to independently estimate the free energies of eight structural states (see <a href="#pcbi.1013346.s003" class="usa-link">S3 Fig</a>) to further support our proposed mechanism. At ~50°C, the relative free energy differences between the I1 and I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e114"><math id="M114" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span> states <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e115"><math id="M115" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mrow><mi>Δ</mi></mrow><mrow><mi>Δ</mi></mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>I</mi><mn>1</mn><mo>,</mo><mi>I</mi><mn>1</mn><mrow><mi>′</mi></mrow></mrow></msub><mo>=</mo><mrow><mi>Δ</mi></mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>I</mi><mn>1</mn></mrow></msub><mo>−</mo><mrow><mi>Δ</mi></mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>I</mi><mn>1</mn><mrow><mi>′</mi></mrow></mrow></msub><mo stretchy="false">)</mo></mrow></math></span> was ~ -1.2 kcal/mol, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e116"><math id="M116" display="inline" overflow="linebreak"><mrow><mrow><mi>Δ</mi></mrow><mrow><mi>Δ</mi></mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>I</mi><mn>1</mn><mo>,</mo><mi>I</mi><mn>1</mn><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></msub></mrow></math></span> = ~ -2.8 kcal/mol, indicating that I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e117"><math id="M117" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span> and I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e118"><math id="M118" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>are less stable than I1. This agrees with the higher observed population of I1 compared to I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e119"><math id="M119" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span> and I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e120"><math id="M120" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span> at this temperature. Next, we examined the I2 state (stems 1 and 3 melted). Its free energy is higher than that of I1 below ~50°C but becomes lower at higher temperatures, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e121"><math id="M121" display="inline" overflow="linebreak"><mrow><mrow><mi>Δ</mi></mrow><mrow><mi>Δ</mi></mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>I</mi><mn>2</mn><mo>,</mo><mi>I</mi><mn>1</mn></mrow></msub></mrow></math></span> = ~ 0.5 kcal/mol at 50°C and ~-1.3 kcal/mol at 70°C. This suggests that I2 becomes more populated as temperature increases, supporting the major unfolding pathway F → I1 → I2 → U. We also evaluated states with two stems melted, including I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e122"><math id="M122" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span> (stems 1 and 2 melted) and I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e123"><math id="M123" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span> (stems 2 and 3 melted), which have negligible populations. Notably, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e124"><math id="M124" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span> exhibited significantly higher free energy than I2 and I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e125"><math id="M125" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>, consistent with its negligible fraction.</p></section><section id="sec020"><h4 class="pmc_sec_title">For a DNA with four-way junction (4WJ).</h4>
<p>The 4WJ displays a more complex landscape, with up to sixteen intermediate states (<a href="#pcbi.1013346.s006" class="usa-link">S6 Fig</a>). Based on our model predictions, we calculated the population fractions of key structural states across temperatures (<a href="#pcbi.1013346.g007" class="usa-link">Fig 7A</a>). At low temperatures (&lt;~20°C), the 4WJ predominantly adopts the fully folded state (F). As temperature increases to ~40°C, the fraction of F decreases to ~30%, accompanied by a rise in intermediate states I1 (Stem 3 melted), I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e126"><math id="M126" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span> (Stem 1 melted), and I2 (Stems 1 and 3 melted), reaching ~14%, ~ 13%, and ~38%, respectively. At ~50°C, I2 becomes the dominant intermediate (~48%), while F, I1, and I1′ nearly vanish. With further heating, I2 transitions into I3 (three stems melted), which peaks (~47%) around 70°C. Eventually, the system reaches the fully unfolded state (U) at ~120°C. Based on these results, the primary unfolding pathway is proposed as F → I2 → I3 → U, with additional minor routes: F → I1 → I2 → I3 → U, and F → I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e127"><math id="M127" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>→I2 → I3 → U.</p>
<figure class="fig xbox font-sm" id="pcbi.1013346.g007"><h5 class="obj_head">Fig 7. Predicted thermal unfolding pathway of 4WJ at 1 M [Na<sup>+</sup>] using our CG model.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373291_pcbi.1013346.g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9592/12373291/9045d1eaa922/pcbi.1013346.g007.jpg" loading="lazy" height="863" width="796" alt="Fig 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013346.g007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Temperature-dependent fractions of the folded state (F), intermediate states (I1, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e171"><math id="M171" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I2, I3), and unfolded state (U) during the thermally unfolding. I1: with Stems 1, 2, and 4, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e172"><math id="M172" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>: with Stems 2, 3, and 4, I2: with Stems 2 and 4, I3: with Stem 2. <strong>(C-F)</strong> Free energies of various intermediate states (I1, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e173"><math id="M173" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e174"><math id="M174" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e175"><math id="M175" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I2, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e176"><math id="M176" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e177"><math id="M177" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e178"><math id="M178" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e179"><math id="M179" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e180"><math id="M180" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I3, I3<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e181"><math id="M181" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I3<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e182"><math id="M182" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>, and I3<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e183"><math id="M183" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>) as functions of temperature calculated using Mfold. <strong>(G)</strong> Schematic representation of DNA structure transitions along the unfolding pathway.</p></figcaption></figure><p>To validate the model-predicted pathway, we computed the free energies of key intermediate states using Mfold [<a href="#pcbi.1013346.ref082" class="usa-link" aria-describedby="pcbi.1013346.ref082">82</a>]. For accurate secondary structure prediction for Mfold, loop sequences were replaced with ‘X’. We first analyzed single-stem-melted states at 1 M NaCl and ~30°C. The relative free energy differences were: (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e128"><math id="M128" display="inline" overflow="linebreak"><mrow><mrow><mi>Δ</mi></mrow><mrow><mi>Δ</mi></mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>I</mi><mn>1</mn><mo>,</mo><mi>I</mi><mn>1</mn><mrow><mi>′</mi></mrow></mrow></msub></mrow></math></span> = ~ -0.4 kcal/mol, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e129"><math id="M129" display="inline" overflow="linebreak"><mrow><mrow><mi>Δ</mi></mrow><mrow><mi>Δ</mi></mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>I</mi><mn>1</mn><mo>,</mo><mi>I</mi><mn>1</mn><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></msub></mrow></math></span> = - ~ 1.0 kcal/mol, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e130"><math id="M130" display="inline" overflow="linebreak"><mrow><mrow><mi>Δ</mi></mrow><mrow><mi>Δ</mi></mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>I</mi><mn>1</mn><mo>,</mo><mi>I</mi><mn>1</mn><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></msub></mrow></math></span> = - ~ 2.5 kcal/mol, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e131"><math id="M131" display="inline" overflow="linebreak"><mrow><mrow><mi>Δ</mi></mrow><mrow><mi>Δ</mi></mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>I</mi><mn>1</mn><mrow><mi>′</mi></mrow><mo>,</mo><mi>I</mi><mn>1</mn><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></msub></mrow></math></span> = - ~ 0.6 kcal/mol, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e132"><math id="M132" display="inline" overflow="linebreak"><mrow><mrow><mi>Δ</mi></mrow><mrow><mi>Δ</mi></mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>I</mi><mn>1</mn><mrow><mi>′</mi></mrow><mo>,</mo><mi>I</mi><mn>1</mn><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></msub></mrow></math></span> = ~ -2.2 kcal/mol, indicating that I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e133"><math id="M133" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span> and I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e134"><math id="M134" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span> are much less stable than I1 and I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e135"><math id="M135" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>, which is consistent with their low populations. We then examined the I2 state (Stems 1 and 3 melted). At ~30°C, it is less stable than I1/I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e136"><math id="M136" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e137"><math id="M137" display="inline" overflow="linebreak"><mrow><mrow><mi>Δ</mi></mrow><mrow><mi>Δ</mi></mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>I</mi><mn>2</mn><mo>,</mo><mi>I</mi><mn>1</mn></mrow></msub><mo>=</mo><mi>~</mi><mn>1.8</mn></mrow></math></span> kcal/mol and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e138"><math id="M138" display="inline" overflow="linebreak"><mrow><mrow><mi>Δ</mi></mrow><mrow><mi>Δ</mi></mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>I</mi><mn>2</mn><mo>,</mo><mi>I</mi><mn>1</mn><mrow><mi>′</mi></mrow></mrow></msub><mo>=</mo><mi>~</mi><mn>1.4</mn></mrow></math></span> kcal/mol), but becomes more favorable at ~50°C (both were ~ -0.6 kcal/mol), supporting its dominance in the major pathway. States with two (e.g., I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e139"><math id="M139" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e140"><math id="M140" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>) or three (e.g., I3<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e141"><math id="M141" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I3<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e142"><math id="M142" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>, I3<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e143"><math id="M143" display="inline" overflow="linebreak"><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></math></span>) stems melted exhibit significantly higher free energies and negligible populations (<a href="#pcbi.1013346.g007" class="usa-link">Fig 7</a>), consistent with their minor roles. Overall, the free energy landscape confirms that unfolding proceeds primarily via F → I2 → I3 → U, with I2 serving as a key thermodynamic intermediate.</p></section></section></section><section id="sec021"><h2 class="pmc_sec_title">Conclusion</h2>
<p>In conclusion, this study presents a significant advancement in modeling the 3D structure and thermal stability of complex DNAs, with a particular focus on multi-way junctions. By refining the electrostatic energy terms of our CG model, incorporating the REMC algorithm, and using the WHAM, we have extended the applicability of model to three-way and four-way DNA junctions under both monovalent and divalent ion conditions. The main contributions of this work are as follows:</p>
<ol class="list" style="list-style-type:decimal">
<li><p>Accurate structure predictions: The refined CG model can predict near-native structures of complex DNAs, including three-way and four-way junctions, from sequence. The predictions align well with results from state-of-the-art models such as 3dRNA/DNA and AlphaFold3, demonstrating the capacity of model to capture the intricate geometry of DNAs with multi-way junctions.</p></li>
<li><p>Reliable thermal stability profiling: The model can reproduce the thermal stability of DNA junctions with different sequences and topologies. Notably, it successfully captures the effects of both monovalent and divalent ions on DNA junction stability, showing strong agreement with experimental data. This demonstrates the capacity of the model to simulate biologically relevant conditions.</p></li>
<li><p>Insight into thermally unfolding pathways: Our analysis reveals that the thermal unfolding of multi-way junctions proceeds via discrete intermediate states, whose relative stabilities govern the unfolding trajectories. These findings elucidate the thermodynamic landscape of DNA junctions and offer a detailed view of their transition pathways.</p></li>
</ol>
<p>Despite these advances, several limitations remain. First, the model does not account for non-canonical base pairings (e.g., G-G, G-T, A-A, A-G) [<a href="#pcbi.1013346.ref083" class="usa-link" aria-describedby="pcbi.1013346.ref083">83</a>,<a href="#pcbi.1013346.ref084" class="usa-link" aria-describedby="pcbi.1013346.ref084">84</a>], which are crucial for the stability and structural features of functional DNA structures, such as DNA triplexes, and G-quadruplexes [<a href="#pcbi.1013346.ref085" class="usa-link" aria-describedby="pcbi.1013346.ref085">85</a>,<a href="#pcbi.1013346.ref086" class="usa-link" aria-describedby="pcbi.1013346.ref086">86</a>], meaning that the current version of the model is unable to predict these complex structures. Second, while the model incorporates the effects of mono-/divalent ions through the CC theory and TBI model, the implicit ion model could be insufficient to capture the interactions between multi-valent ions (e.g., Mg<sup>2+</sup> and Go<sup>3+</sup>) and DNA, particularly the specific binding of these ions. Such interactions are critical for stabilizing complex structures like DNA G-quadruplexes [<a href="#pcbi.1013346.ref087" class="usa-link" aria-describedby="pcbi.1013346.ref087">87</a>,<a href="#pcbi.1013346.ref088" class="usa-link" aria-describedby="pcbi.1013346.ref088">88</a>], making it necessary to incorporate explicit treatment of divalent ions in future improvements to the model [<a href="#pcbi.1013346.ref054" class="usa-link" aria-describedby="pcbi.1013346.ref054">54</a>,<a href="#pcbi.1013346.ref055" class="usa-link" aria-describedby="pcbi.1013346.ref055">55</a>]. Third, the top-ranked structure predicted by the present model typically does not correspond to the most native-like structure within the predicted ensemble, highlighting the need for further development of DNA scoring function to identify the near-native structure models [<a href="#pcbi.1013346.ref089" class="usa-link" aria-describedby="pcbi.1013346.ref089">89</a>]. Finally, the cellular environment of DNA involves not only ions but also other macromolecules (e.g., proteins and RNAs) and small molecules (e.g., ligands). This crowded environment and its interactions with DNA can influence DNA structural folding. Effectively accounting for these interactions to study DNA folding in a cell-like environment remains a significant challenge [<a href="#pcbi.1013346.ref053" class="usa-link" aria-describedby="pcbi.1013346.ref053">53</a>,<a href="#pcbi.1013346.ref090" class="usa-link" aria-describedby="pcbi.1013346.ref090">90</a>,<a href="#pcbi.1013346.ref091" class="usa-link" aria-describedby="pcbi.1013346.ref091">91</a>].</p>
<p>Although there are limitations, our refined CG model provides a powerful and reliable framework for studying the 3D structures and stability of complex DNA configurations in the presence of physiologically relevant ions. The insights gained from analyzing the thermally unfolding pathways of DNA junctions contribute to a deeper understanding of DNA stability and the mechanisms underlying their biological functions, providing a strong foundation for future studies in the field of DNA biophysics.</p></section><section id="sec022"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pcbi.1013346.s001"><div class="caption p">
<span>S1 Fig</span><p><strong>(A) The schematic diagram for the formation of one base stacking between base pairs (</strong><strong><em>i, j</em></strong><strong>) and (</strong><strong><em>i</em></strong><strong> + 1,</strong>
<strong><em>j</em></strong><strong>-1)</strong><strong>; (B) The conformational entropy changes ΔS</strong><sub><strong>c</strong></sub>
<strong>for the formation of base-pairs stacking at different location <em>i</em> (symbols), and the average value of ΔS</strong><sub><strong>c</strong></sub>
<strong>(line).</strong></p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s001.tif" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s001.tif</a><sup> (508.6KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013346.s002"><div class="caption p">
<span>S2 Fig. The schematic diagram of the rebuilding of coarse-grained DNA structures into all-atom ones in our present model.</span><p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s002.tif" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s002.tif</a><sup> (785.8KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013346.s003"><div class="caption p">
<span>S3 Fig. The eight structural states of 3WJ, including a folded state, six intermediate states, and a unfolded state.</span><p>Here, F state: Stems 1, 2, and 3 retained, I1: Stem 1 resolved, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e144"><math id="M144" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stem 3 resolved, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e145"><math id="M145" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stem 2 resolved, I2: Stems 1 and 3 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e146"><math id="M146" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 1 and 2 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e147"><math id="M147" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 2 and 3 resolved, U: All stems resolved.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s003.tif" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s003.tif</a><sup> (450KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013346.s004"><div class="caption p">
<span>S4 Fig. The sequence and secondary structure of 6 DNAs with multi-way junctions employed in this paper.</span><p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s004.tif" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s004.tif</a><sup> (480.6KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013346.s005"><div class="caption p">
<span>S5 Fig. The average radius of gyration (Rg) and ion neutralization fraction of different structural states for 3WJ and 4WJ.</span><p>(A,B) The average Rg (A) and ion neutralization fraction (B) of folded state (F), intermediate states I1*/I2* (two stems retained/one stem retained), unfolded state (zero stem retained) for 3WJ. (C,D) The average Rg (A) and ion neutralization fraction (B) of folded state (F), intermediate states I1*/I3* (three stems retained/one stem retained), unfolded state (zero stem retained) for 4WJ.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s005.tif" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s005.tif</a><sup> (1MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013346.s006"><div class="caption p">
<span>S6 Fig. The sixteen structural states of 4Wj, including a folded state, fourteen intermediate states, a unfolded state.</span><p>Here F: All stem retained, I1: Stem 3 resolved, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e148"><math id="M148" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stem 1 resolved, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e149"><math id="M149" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stem 4 resolved, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e150"><math id="M150" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stem 2 resolved, I2: Stems 1 and 3 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e151"><math id="M151" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 1 and 4 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e152"><math id="M152" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 2 and 3 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e153"><math id="M153" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 1 and 2 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e154"><math id="M154" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 2 and 3 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e155"><math id="M155" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 2 and 4 resolved, I3: Stems 1,3, and 4 resolved, I3<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e156"><math id="M156" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 1, 2, and 3 resolved, I3<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e157"><math id="M157" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 1, 2, and 4 resolved, I3<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013346.e158"><math id="M158" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 2, 3, and 4 resolved, U: All stems resolved.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s006.tif" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s006.tif</a><sup> (496.4KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013346.s007"><div class="caption p">
<span>S1 Table. The PDB codes of 138 DNAs used in our statistical analysis for CG force field.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s007.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s007.xlsx</a><sup> (10.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013346.s008"><div class="caption p">
<span>S2 Table. The parameters of bonded potentials of CG force field.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s008.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s008.xlsx</a><sup> (31.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013346.s009"><div class="caption p">
<span>S3 Table. The parameters for the energy functions of base pairing and base stacking.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s009.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s009.xlsx</a><sup> (21KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013346.s010"><div class="caption p">
<span>S4 Table. The predicted RMSD and F1-socre by three prediction models for 4 DNAs.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s010.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s010.xlsx</a><sup> (10.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013346.s011"><div class="caption p">
<span>S5 Table. The predicted temperature of 3WJ at extensive ion concentration by our present model.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s011.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s011.xlsx</a><sup> (14.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013346.s012"><div class="caption p">
<span>S6 Table. The predicted temperature of 4WJ at extensive ion concentration by our present model.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s012.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s012.xlsx</a><sup> (14.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013346.s013"><div class="caption p">
<span>S1 Text. Detailed description of the coarse-grained force field of the present model and the Weighted Histogram Analysis Method.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s013.docx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s013.docx</a><sup> (44.6KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We are grateful to Profs. Yang Yu (Guizhou Medical University), Qiude Li (Guizhou Medical University), Zhi-Jie Tan (Wuhan University), and Bengong Zhang (Wuhan Textile University) for valuable discussions, and we would like to acknowledge computing resources from the Super Computing Center of Guizhou Medical University.</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>All relevant computational data are within the paper and its Supporting information files. Experimental data are publicly available from published papers (DOI: <a href="https://doi.org/10.1039/c7cp08329g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1039/c7cp08329g</a>) cited in the work. The package of the present model is freely available at <a href="https://github.com/RNA-folding-lab/DNAfold2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/RNA-folding-lab/DNAfold2</a>.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This work was supported by grants from the National Science Foundation of China (No. 11605125 to YZS), the China Scholarship Council (No. 202208420104 to YZS), and the Guizhou Medical University High-Level Talent Scientific Research Startup Fund (No. 26242020163 to XW). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pcbi.1013346.ref001">
<span class="label">1.</span><cite>Cevallos Y, Nakano T, Tello-Oquendo L, Rushdi A, Inca D, Santillán I, et al. A brief review on DNA storage, compression, and digitalization. Nano Commun Netw. 2022;31:100391.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Nano%20Commun%20Netw&amp;title=A%20brief%20review%20on%20DNA%20storage,%20compression,%20and%20digitalization&amp;author=Y%20Cevallos&amp;author=T%20Nakano&amp;author=L%20Tello-Oquendo&amp;author=A%20Rushdi&amp;author=D%20Inca&amp;volume=31&amp;publication_year=2022&amp;pages=100391&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref002">
<span class="label">2.</span><cite>Kornberg A. Biologic synthesis of deoxyribonucleic acid: An isolated enzyme catalyzes synthesis of this nucleic acid in response to directions from pre-existing DNA. Science. 1960;131:1503–8.
</cite> [<a href="https://doi.org/10.1126/science.131.3412.1503" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14411056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Biologic%20synthesis%20of%20deoxyribonucleic%20acid:%20An%20isolated%20enzyme%20catalyzes%20synthesis%20of%20this%20nucleic%20acid%20in%20response%20to%20directions%20from%20pre-existing%20DNA&amp;author=A%20Kornberg&amp;volume=131&amp;publication_year=1960&amp;pages=1503-8&amp;pmid=14411056&amp;doi=10.1126/science.131.3412.1503&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref003">
<span class="label">3.</span><cite>Smith GR. DNA supercoiling: another level for regulating gene expression. Cell. 1981;24(3):599–600. doi: 10.1016/0092-8674(81)90085-4

</cite> [<a href="https://doi.org/10.1016/0092-8674(81)90085-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6265097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=DNA%20supercoiling:%20another%20level%20for%20regulating%20gene%20expression&amp;author=GR%20Smith&amp;volume=24&amp;issue=3&amp;publication_year=1981&amp;pages=599-600&amp;pmid=6265097&amp;doi=10.1016/0092-8674(81)90085-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref004">
<span class="label">4.</span><cite>Guiblet WM, Cremona MA, Harris RS, Chen D, Eckert KA, Chiaromonte F, et al. Non-B DNA: a major contributor to small- and large-scale variation in nucleotide substitution frequencies across the genome. Nucleic Acids Res. 2021;49(3):1497–516. doi: 10.1093/nar/gkaa1269

</cite> [<a href="https://doi.org/10.1093/nar/gkaa1269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7897504/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33450015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Non-B%20DNA:%20a%20major%20contributor%20to%20small-%20and%20large-scale%20variation%20in%20nucleotide%20substitution%20frequencies%20across%20the%20genome&amp;author=WM%20Guiblet&amp;author=MA%20Cremona&amp;author=RS%20Harris&amp;author=D%20Chen&amp;author=KA%20Eckert&amp;volume=49&amp;issue=3&amp;publication_year=2021&amp;pages=1497-516&amp;pmid=33450015&amp;doi=10.1093/nar/gkaa1269&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref005">
<span class="label">5.</span><cite>Xavier PL, Chandrasekaran AR. DNA-based construction at the nanoscale: emerging trends and applications. Nanotechnology. 2018;29(6):062001. doi: 10.1088/1361-6528/aaa120

</cite> [<a href="https://doi.org/10.1088/1361-6528/aaa120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29232197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nanotechnology&amp;title=DNA-based%20construction%20at%20the%20nanoscale:%20emerging%20trends%20and%20applications&amp;author=PL%20Xavier&amp;author=AR%20Chandrasekaran&amp;volume=29&amp;issue=6&amp;publication_year=2018&amp;pages=062001&amp;pmid=29232197&amp;doi=10.1088/1361-6528/aaa120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref006">
<span class="label">6.</span><cite>Zhang D, Lam J, Blobel GA. Engineering three-dimensional genome folding. Nat Genet. 2021;53(5):602–11. doi: 10.1038/s41588-021-00860-9

</cite> [<a href="https://doi.org/10.1038/s41588-021-00860-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33958782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Engineering%20three-dimensional%20genome%20folding&amp;author=D%20Zhang&amp;author=J%20Lam&amp;author=GA%20Blobel&amp;volume=53&amp;issue=5&amp;publication_year=2021&amp;pages=602-11&amp;pmid=33958782&amp;doi=10.1038/s41588-021-00860-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref007">
<span class="label">7.</span><cite>Bonev B, Cavalli G. Organization and function of the 3D genome. Nat Rev Genet. 2016;17(11):661–78. doi: 10.1038/nrg.2016.112

</cite> [<a href="https://doi.org/10.1038/nrg.2016.112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27739532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&amp;title=Organization%20and%20function%20of%20the%203D%20genome&amp;author=B%20Bonev&amp;author=G%20Cavalli&amp;volume=17&amp;issue=11&amp;publication_year=2016&amp;pages=661-78&amp;pmid=27739532&amp;doi=10.1038/nrg.2016.112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref008">
<span class="label">8.</span><cite>Andersson R, Sandelin A. Determinants of enhancer and promoter activities of regulatory elements. Nat Rev Genet. 2020;21(2):71–87. doi: 10.1038/s41576-019-0173-8

</cite> [<a href="https://doi.org/10.1038/s41576-019-0173-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31605096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&amp;title=Determinants%20of%20enhancer%20and%20promoter%20activities%20of%20regulatory%20elements&amp;author=R%20Andersson&amp;author=A%20Sandelin&amp;volume=21&amp;issue=2&amp;publication_year=2020&amp;pages=71-87&amp;pmid=31605096&amp;doi=10.1038/s41576-019-0173-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref009">
<span class="label">9.</span><cite>Liu D, Chen G, Akhter U, Cronin TM, Weizmann Y. Creating complex molecular topologies by configuring DNA four-way junctions. Nat Chem. 2016;8(10):907–14. doi: 10.1038/nchem.2564

</cite> [<a href="https://doi.org/10.1038/nchem.2564" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27657865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Chem&amp;title=Creating%20complex%20molecular%20topologies%20by%20configuring%20DNA%20four-way%20junctions&amp;author=D%20Liu&amp;author=G%20Chen&amp;author=U%20Akhter&amp;author=TM%20Cronin&amp;author=Y%20Weizmann&amp;volume=8&amp;issue=10&amp;publication_year=2016&amp;pages=907-14&amp;pmid=27657865&amp;doi=10.1038/nchem.2564&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref010">
<span class="label">10.</span><cite>Xia Y, Zheng K-w, He Y-d, Liu H-h, Wen C-j, Hao Y-h, et al. Transmission of dynamic supercoiling in linear and multi-way branched DNAs and its regulation revealed by a fluorescent G-quadruplex torsion sensor. Nucleic Acids Res. 2018;46:7418–24.
</cite> [<a href="https://doi.org/10.1093/nar/gky534" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6101514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29982790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Transmission%20of%20dynamic%20supercoiling%20in%20linear%20and%20multi-way%20branched%20DNAs%20and%20its%20regulation%20revealed%20by%20a%20fluorescent%20G-quadruplex%20torsion%20sensor&amp;author=Y%20Xia&amp;author=K-w%20Zheng&amp;author=Y-d%20He&amp;author=H-h%20Liu&amp;author=C-j%20Wen&amp;volume=46&amp;publication_year=2018&amp;pages=7418-24&amp;pmid=29982790&amp;doi=10.1093/nar/gky534&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref011">
<span class="label">11.</span><cite>Rose PW, Prlić A, Altunkaya A, Bi C, Bradley AR, Christie CH, et al. The RCSB protein data bank: integrative view of protein, gene and 3D structural information. Nucleic Acids Res. 2016;gkw1000.</cite> [<a href="https://doi.org/10.1093/nar/gkw1000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5210513/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27794042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20RCSB%20protein%20data%20bank:%20integrative%20view%20of%20protein,%20gene%20and%203D%20structural%20information&amp;author=PW%20Rose&amp;author=A%20Prli%C4%87&amp;author=A%20Altunkaya&amp;author=C%20Bi&amp;author=AR%20Bradley&amp;publication_year=2016&amp;pmid=27794042&amp;doi=10.1093/nar/gkw1000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref012">
<span class="label">12.</span><cite>Burley SK, Berman HM, Kleywegt GJ, Markley JL, Nakamura H, Velankar S. Protein Data Bank (PDB): the single global macromolecular structure archive. Protein Crystallogr Methods Protoc. 2017:627–41.</cite> [<a href="https://doi.org/10.1007/978-1-4939-7000-1_26" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5823500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28573592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Crystallogr%20Methods%20Protoc&amp;title=Protein%20Data%20Bank%20(PDB):%20the%20single%20global%20macromolecular%20structure%20archive&amp;author=SK%20Burley&amp;author=HM%20Berman&amp;author=GJ%20Kleywegt&amp;author=JL%20Markley&amp;author=H%20Nakamura&amp;publication_year=2017&amp;pages=627-41&amp;pmid=28573592&amp;doi=10.1007/978-1-4939-7000-1_26&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref013">
<span class="label">13.</span><cite>Mu Z-C, Tan Y-L, Zhang B-G, Liu J, Shi Y-Z. Ab initio predictions for 3D structure and stability of single- and double-stranded DNAs in ion solutions. PLoS Comput Biol. 2022;18(10):e1010501. doi: 10.1371/journal.pcbi.1010501

</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1010501" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9621594/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36260618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=Ab%20initio%20predictions%20for%203D%20structure%20and%20stability%20of%20single-%20and%20double-stranded%20DNAs%20in%20ion%20solutions&amp;author=Z-C%20Mu&amp;author=Y-L%20Tan&amp;author=B-G%20Zhang&amp;author=J%20Liu&amp;author=Y-Z%20Shi&amp;volume=18&amp;issue=10&amp;publication_year=2022&amp;pmid=36260618&amp;doi=10.1371/journal.pcbi.1010501&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref014">
<span class="label">14.</span><cite>Mu Z-C, Tan Y-L, Liu J, Zhang B-G, Shi Y-Z. Computational Modeling of DNA 3D Structures: From Dynamics and Mechanics to Folding. Molecules. 2023;28(12):4833. doi: 10.3390/molecules28124833

</cite> [<a href="https://doi.org/10.3390/molecules28124833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10303651/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37375388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Computational%20Modeling%20of%20DNA%203D%20Structures:%20From%20Dynamics%20and%20Mechanics%20to%20Folding&amp;author=Z-C%20Mu&amp;author=Y-L%20Tan&amp;author=J%20Liu&amp;author=B-G%20Zhang&amp;author=Y-Z%20Shi&amp;volume=28&amp;issue=12&amp;publication_year=2023&amp;pages=4833&amp;pmid=37375388&amp;doi=10.3390/molecules28124833&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref015">
<span class="label">15.</span><cite>Benson DA, Cavanaugh M, Clark K, Karsch-Mizrachi I, Lipman DJ, Ostell J, et al. GenBank. Nucleic Acids Res. 2012;41:D36–42.</cite> [<a href="https://doi.org/10.1093/nar/gks1195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23193287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=GenBank&amp;author=DA%20Benson&amp;author=M%20Cavanaugh&amp;author=K%20Clark&amp;author=I%20Karsch-Mizrachi&amp;author=DJ%20Lipman&amp;volume=41&amp;publication_year=2012&amp;pmid=23193287&amp;doi=10.1093/nar/gks1195&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref016">
<span class="label">16.</span><cite>Benson DA, Cavanaugh M, Clark K, Karsch-Mizrachi I, Ostell J, Pruitt KD, et al. GenBank. Nucleic Acids Res. 2018;46:D41–7.</cite> [<a href="https://doi.org/10.1093/nar/gkx1094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5753231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29140468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=GenBank&amp;author=DA%20Benson&amp;author=M%20Cavanaugh&amp;author=K%20Clark&amp;author=I%20Karsch-Mizrachi&amp;author=J%20Ostell&amp;volume=46&amp;publication_year=2018&amp;pmid=29140468&amp;doi=10.1093/nar/gkx1094&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref017">
<span class="label">17.</span><cite>Singh A, Maity A, Singh N. Structure and Dynamics of dsDNA in Cell-like Environments. Entropy (Basel). 2022;24(11):1587. doi: 10.3390/e24111587

</cite> [<a href="https://doi.org/10.3390/e24111587" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9689892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36359677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Entropy%20(Basel)&amp;title=Structure%20and%20Dynamics%20of%20dsDNA%20in%20Cell-like%20Environments&amp;author=A%20Singh&amp;author=A%20Maity&amp;author=N%20Singh&amp;volume=24&amp;issue=11&amp;publication_year=2022&amp;pages=1587&amp;pmid=36359677&amp;doi=10.3390/e24111587&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref018">
<span class="label">18.</span><cite>Abramson J, Adler J, Dunger J, Evans R, Green T, Pritzel A, et al. Accurate structure prediction of biomolecular interactions with AlphaFold 3. Nature. 2024;630(8016):493–500. doi: 10.1038/s41586-024-07487-w

</cite> [<a href="https://doi.org/10.1038/s41586-024-07487-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11168924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Accurate%20structure%20prediction%20of%20biomolecular%20interactions%20with%20AlphaFold%203&amp;author=J%20Abramson&amp;author=J%20Adler&amp;author=J%20Dunger&amp;author=R%20Evans&amp;author=T%20Green&amp;volume=630&amp;issue=8016&amp;publication_year=2024&amp;pages=493-500&amp;pmid=38718835&amp;doi=10.1038/s41586-024-07487-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref019">
<span class="label">19.</span><cite>Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, et al. Highly accurate protein structure prediction with AlphaFold. Nature. 2021;596(7873):583–9. doi: 10.1038/s41586-021-03819-2

</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;author=J%20Jumper&amp;author=R%20Evans&amp;author=A%20Pritzel&amp;author=T%20Green&amp;author=M%20Figurnov&amp;volume=596&amp;issue=7873&amp;publication_year=2021&amp;pages=583-9&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref020">
<span class="label">20.</span><cite>Wang W, Feng C, Han R, Wang Z, Ye L, Du Z, et al. trRosettaRNA: automated prediction of RNA 3D structure with transformer network. Nat Commun. 2023;14(1):7266. doi: 10.1038/s41467-023-42528-4

</cite> [<a href="https://doi.org/10.1038/s41467-023-42528-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10636060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37945552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=trRosettaRNA:%20automated%20prediction%20of%20RNA%203D%20structure%20with%20transformer%20network&amp;author=W%20Wang&amp;author=C%20Feng&amp;author=R%20Han&amp;author=Z%20Wang&amp;author=L%20Ye&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=7266&amp;pmid=37945552&amp;doi=10.1038/s41467-023-42528-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref021">
<span class="label">21.</span><cite>Li Y, Zhang C, Feng C, Pearce R, Lydia Freddolino P, Zhang Y. Integrating end-to-end learning with deep geometrical potentials for ab initio RNA structure prediction. Nat Commun. 2023;14(1):5745. doi: 10.1038/s41467-023-41303-9

</cite> [<a href="https://doi.org/10.1038/s41467-023-41303-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10505173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37717036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Integrating%20end-to-end%20learning%20with%20deep%20geometrical%20potentials%20for%20ab%20initio%20RNA%20structure%20prediction&amp;author=Y%20Li&amp;author=C%20Zhang&amp;author=C%20Feng&amp;author=R%20Pearce&amp;author=P%20Lydia%20Freddolino&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=5745&amp;pmid=37717036&amp;doi=10.1038/s41467-023-41303-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref022">
<span class="label">22.</span><cite>Shen T, Hu Z, Sun S, Liu D, Wong F, Wang J, et al. Accurate RNA 3D structure prediction using a language model-based deep learning approach. Nature Methods. 2024:1–12.
</cite> [<a href="https://doi.org/10.1038/s41592-024-02487-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11621015/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39572716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Accurate%20RNA%203D%20structure%20prediction%20using%20a%20language%20model-based%20deep%20learning%20approach&amp;author=T%20Shen&amp;author=Z%20Hu&amp;author=S%20Sun&amp;author=D%20Liu&amp;author=F%20Wong&amp;publication_year=2024&amp;pages=1-12&amp;pmid=39572716&amp;doi=10.1038/s41592-024-02487-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref023">
<span class="label">23.</span><cite>Li J, Chiu T-P, Rohs R. Predicting DNA structure using a deep learning method. Nat Commun. 2024;15(1):1243. doi: 10.1038/s41467-024-45191-5

</cite> [<a href="https://doi.org/10.1038/s41467-024-45191-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10858265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38336958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Predicting%20DNA%20structure%20using%20a%20deep%20learning%20method&amp;author=J%20Li&amp;author=T-P%20Chiu&amp;author=R%20Rohs&amp;volume=15&amp;issue=1&amp;publication_year=2024&amp;pages=1243&amp;pmid=38336958&amp;doi=10.1038/s41467-024-45191-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref024">
<span class="label">24.</span><cite>McDonnell RT, Henderson AN, Elcock AH. Structure Prediction of Large RNAs with AlphaFold3 Highlights its Capabilities and Limitations. J Mol Biol. 2024;436(22):168816. doi: 10.1016/j.jmb.2024.168816

</cite> [<a href="https://doi.org/10.1016/j.jmb.2024.168816" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39384035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Structure%20Prediction%20of%20Large%20RNAs%20with%20AlphaFold3%20Highlights%20its%20Capabilities%20and%20Limitations&amp;author=RT%20McDonnell&amp;author=AN%20Henderson&amp;author=AH%20Elcock&amp;volume=436&amp;issue=22&amp;publication_year=2024&amp;pages=168816&amp;pmid=39384035&amp;doi=10.1016/j.jmb.2024.168816&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref025">
<span class="label">25.</span><cite>Zhang Y, Wang J, Xiao Y. 3dRNA: 3D Structure Prediction from Linear to Circular RNAs. J Mol Biol. 2022;434(11):167452. doi: 10.1016/j.jmb.2022.167452

</cite> [<a href="https://doi.org/10.1016/j.jmb.2022.167452" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35662453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=3dRNA:%203D%20Structure%20Prediction%20from%20Linear%20to%20Circular%20RNAs&amp;author=Y%20Zhang&amp;author=J%20Wang&amp;author=Y%20Xiao&amp;volume=434&amp;issue=11&amp;publication_year=2022&amp;pages=167452&amp;pmid=35662453&amp;doi=10.1016/j.jmb.2022.167452&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref026">
<span class="label">26.</span><cite>Xiong Y, Zhang Y, Wang J, Xiao Y. Using 3dRNA/DNA for RNA and DNA 3D Structure Prediction and Evaluation. Curr Protoc. 2023;3:e770.</cite> [<a href="https://doi.org/10.1002/cpz1.770" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37166266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Protoc&amp;title=Using%203dRNA/DNA%20for%20RNA%20and%20DNA%203D%20Structure%20Prediction%20and%20Evaluation&amp;author=Y%20Xiong&amp;author=Y%20Zhang&amp;author=J%20Wang&amp;author=Y%20Xiao&amp;volume=3&amp;publication_year=2023&amp;pmid=37166266&amp;doi=10.1002/cpz1.770&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref027">
<span class="label">27.</span><cite>Mittal A, Turner DH, Mathews DH. NNDB: An Expanded Database of Nearest Neighbor Parameters for Predicting Stability of Nucleic Acid Secondary Structures. J Mol Biol. 2024;436(17):168549. doi: 10.1016/j.jmb.2024.168549

</cite> [<a href="https://doi.org/10.1016/j.jmb.2024.168549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11377154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38522645/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=NNDB:%20An%20Expanded%20Database%20of%20Nearest%20Neighbor%20Parameters%20for%20Predicting%20Stability%20of%20Nucleic%20Acid%20Secondary%20Structures&amp;author=A%20Mittal&amp;author=DH%20Turner&amp;author=DH%20Mathews&amp;volume=436&amp;issue=17&amp;publication_year=2024&amp;pages=168549&amp;pmid=38522645&amp;doi=10.1016/j.jmb.2024.168549&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref028">
<span class="label">28.</span><cite>Croitoru A, Kumar A, Lambry J-C, Lee J, Sharif S, Yu W, et al. Increasing the Accuracy and Robustness of the CHARMM General Force Field with an Expanded Training Set. J Chem Theory Comput. 2025;21(6):3044–65. doi: 10.1021/acs.jctc.5c00046

</cite> [<a href="https://doi.org/10.1021/acs.jctc.5c00046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11938330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40033678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Theory%20Comput&amp;title=Increasing%20the%20Accuracy%20and%20Robustness%20of%20the%20CHARMM%20General%20Force%20Field%20with%20an%20Expanded%20Training%20Set&amp;author=A%20Croitoru&amp;author=A%20Kumar&amp;author=J-C%20Lambry&amp;author=J%20Lee&amp;author=S%20Sharif&amp;volume=21&amp;issue=6&amp;publication_year=2025&amp;pages=3044-65&amp;pmid=40033678&amp;doi=10.1021/acs.jctc.5c00046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref029">
<span class="label">29.</span><cite>Vanommeslaeghe K, Hatcher E, Acharya C, Kundu S, Zhong S, Shim J, et al. CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields. J Comput Chem. 2010;31(4):671–90. doi: 10.1002/jcc.21367

</cite> [<a href="https://doi.org/10.1002/jcc.21367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2888302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19575467/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Chem&amp;title=CHARMM%20general%20force%20field:%20A%20force%20field%20for%20drug-like%20molecules%20compatible%20with%20the%20CHARMM%20all-atom%20additive%20biological%20force%20fields&amp;author=K%20Vanommeslaeghe&amp;author=E%20Hatcher&amp;author=C%20Acharya&amp;author=S%20Kundu&amp;author=S%20Zhong&amp;volume=31&amp;issue=4&amp;publication_year=2010&amp;pages=671-90&amp;pmid=19575467&amp;doi=10.1002/jcc.21367&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref030">
<span class="label">30.</span><cite>Qiang X-W, Zhang C, Dong H-L, Tian F-J, Fu H, Yang Y-J, et al. Multivalent Cations Reverse the Twist-Stretch Coupling of RNA. Phys Rev Lett. 2022;128(10):108103. doi: 10.1103/PhysRevLett.128.108103

</cite> [<a href="https://doi.org/10.1103/PhysRevLett.128.108103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35333091/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phys%20Rev%20Lett&amp;title=Multivalent%20Cations%20Reverse%20the%20Twist-Stretch%20Coupling%20of%20RNA&amp;author=X-W%20Qiang&amp;author=C%20Zhang&amp;author=H-L%20Dong&amp;author=F-J%20Tian&amp;author=H%20Fu&amp;volume=128&amp;issue=10&amp;publication_year=2022&amp;pages=108103&amp;pmid=35333091&amp;doi=10.1103/PhysRevLett.128.108103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref031">
<span class="label">31.</span><cite>Dong H-L, Zhang C, Dai L, Zhang Y, Zhang X-H, Tan Z-J. The origin of different bending stiffness between double-stranded RNA and DNA revealed by magnetic tweezers and simulations. Nucleic Acids Res. 2024;52(5):2519–29. doi: 10.1093/nar/gkae063

</cite> [<a href="https://doi.org/10.1093/nar/gkae063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10954459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38321947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20origin%20of%20different%20bending%20stiffness%20between%20double-stranded%20RNA%20and%20DNA%20revealed%20by%20magnetic%20tweezers%20and%20simulations&amp;author=H-L%20Dong&amp;author=C%20Zhang&amp;author=L%20Dai&amp;author=Y%20Zhang&amp;author=X-H%20Zhang&amp;volume=52&amp;issue=5&amp;publication_year=2024&amp;pages=2519-29&amp;pmid=38321947&amp;doi=10.1093/nar/gkae063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref032">
<span class="label">32.</span><cite>Dickson CJ, Walker RC, Gould IR. Lipid21: Complex Lipid Membrane Simulations with AMBER. J Chem Theory Comput. 2022;18(3):1726–36. doi: 10.1021/acs.jctc.1c01217

</cite> [<a href="https://doi.org/10.1021/acs.jctc.1c01217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9007451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35113553/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Theory%20Comput&amp;title=Lipid21:%20Complex%20Lipid%20Membrane%20Simulations%20with%20AMBER&amp;author=CJ%20Dickson&amp;author=RC%20Walker&amp;author=IR%20Gould&amp;volume=18&amp;issue=3&amp;publication_year=2022&amp;pages=1726-36&amp;pmid=35113553&amp;doi=10.1021/acs.jctc.1c01217&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref033">
<span class="label">33.</span><cite>Dans PD, Walther J, Gómez H, Orozco M. Multiscale simulation of DNA. Curr Opin Struct Biol. 2016;37:29–45.
</cite> [<a href="https://doi.org/10.1016/j.sbi.2015.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26708341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Struct%20Biol&amp;title=Multiscale%20simulation%20of%20DNA&amp;author=PD%20Dans&amp;author=J%20Walther&amp;author=H%20G%C3%B3mez&amp;author=M%20Orozco&amp;volume=37&amp;publication_year=2016&amp;pages=29-45&amp;pmid=26708341&amp;doi=10.1016/j.sbi.2015.11.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref034">
<span class="label">34.</span><cite>Ratajczyk EJ, Šulc P, Turberfield AJ, Doye JP, Louis AA. Coarse-grained modeling of DNA–RNA hybrids. J Chem Phys. 2024;160.</cite> [<a href="https://doi.org/10.1063/5.0199558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38497475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Phys&amp;title=Coarse-grained%20modeling%20of%20DNA%E2%80%93RNA%20hybrids&amp;author=EJ%20Ratajczyk&amp;author=P%20%C5%A0ulc&amp;author=AJ%20Turberfield&amp;author=JP%20Doye&amp;author=AA%20Louis&amp;volume=160&amp;publication_year=2024&amp;pmid=38497475&amp;doi=10.1063/5.0199558&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref035">
<span class="label">35.</span><cite>Reshetnikov RV, Stolyarova AV, Zalevsky AO, Panteleev DY, Pavlova GV, Klinov DV, et al. A coarse-grained model for DNA origami. Nucleic Acids Res. 2018;46(3):1102–12. doi: 10.1093/nar/gkx1262

</cite> [<a href="https://doi.org/10.1093/nar/gkx1262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5814798/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29267876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=A%20coarse-grained%20model%20for%20DNA%20origami&amp;author=RV%20Reshetnikov&amp;author=AV%20Stolyarova&amp;author=AO%20Zalevsky&amp;author=DY%20Panteleev&amp;author=GV%20Pavlova&amp;volume=46&amp;issue=3&amp;publication_year=2018&amp;pages=1102-12&amp;pmid=29267876&amp;doi=10.1093/nar/gkx1262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref036">
<span class="label">36.</span><cite>Souza PCT, Alessandri R, Barnoud J, Thallmair S, Faustino I, Grünewald F, et al. Martini 3: a general purpose force field for coarse-grained molecular dynamics. Nat Methods. 2021;18(4):382–8. doi: 10.1038/s41592-021-01098-3

</cite> [<a href="https://doi.org/10.1038/s41592-021-01098-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33782607/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Martini%203:%20a%20general%20purpose%20force%20field%20for%20coarse-grained%20molecular%20dynamics&amp;author=PCT%20Souza&amp;author=R%20Alessandri&amp;author=J%20Barnoud&amp;author=S%20Thallmair&amp;author=I%20Faustino&amp;volume=18&amp;issue=4&amp;publication_year=2021&amp;pages=382-8&amp;pmid=33782607&amp;doi=10.1038/s41592-021-01098-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref037">
<span class="label">37.</span><cite>Assenza S, Pérez R. Accurate Sequence-Dependent Coarse-Grained Model for Conformational and Elastic Properties of Double-Stranded DNA. J Chem Theory Comput. 2022;18(5):3239–56. doi: 10.1021/acs.jctc.2c00138

</cite> [<a href="https://doi.org/10.1021/acs.jctc.2c00138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9097290/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35394775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Theory%20Comput&amp;title=Accurate%20Sequence-Dependent%20Coarse-Grained%20Model%20for%20Conformational%20and%20Elastic%20Properties%20of%20Double-Stranded%20DNA&amp;author=S%20Assenza&amp;author=R%20P%C3%A9rez&amp;volume=18&amp;issue=5&amp;publication_year=2022&amp;pages=3239-56&amp;pmid=35394775&amp;doi=10.1021/acs.jctc.2c00138&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref038">
<span class="label">38.</span><cite>Starr FW, Wang W, Nocka LM, Wiemann BZ, Hinckley DM, Mukerji I. Holliday junction thermodynamics and structure: comparisons of coarse-grained simulations and experiments. Biophys J. 2016;110:178a.</cite> [<a href="https://doi.org/10.1038/srep22863" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4789735/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26971574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Holliday%20junction%20thermodynamics%20and%20structure:%20comparisons%20of%20coarse-grained%20simulations%20and%20experiments&amp;author=FW%20Starr&amp;author=W%20Wang&amp;author=LM%20Nocka&amp;author=BZ%20Wiemann&amp;author=DM%20Hinckley&amp;volume=110&amp;publication_year=2016&amp;pmid=26971574&amp;doi=10.1038/srep22863&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref039">
<span class="label">39.</span><cite>Poppleton E, Romero R, Mallya A, Rovigatti L, Šulc P. OxDNA.org: a public webserver for coarse-grained simulations of DNA and RNA nanostructures. Nucleic Acids Res. 2021;49(W1):W491–8. doi: 10.1093/nar/gkab324

</cite> [<a href="https://doi.org/10.1093/nar/gkab324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8265093/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34009383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=OxDNA.org:%20a%20public%20webserver%20for%20coarse-grained%20simulations%20of%20DNA%20and%20RNA%20nanostructures&amp;author=E%20Poppleton&amp;author=R%20Romero&amp;author=A%20Mallya&amp;author=L%20Rovigatti&amp;author=P%20%C5%A0ulc&amp;volume=49&amp;publication_year=2021&amp;pmid=34009383&amp;doi=10.1093/nar/gkab324&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref040">
<span class="label">40.</span><cite>Freeman GS, Hinckley DM, de Pablo JJ. A coarse-grain three-site-per-nucleotide model for DNA with explicit ions. J Chem Phys. 2011;135(16):165104. doi: 10.1063/1.3652956

</cite> [<a href="https://doi.org/10.1063/1.3652956" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3221706/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22047269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Phys&amp;title=A%20coarse-grain%20three-site-per-nucleotide%20model%20for%20DNA%20with%20explicit%20ions&amp;author=GS%20Freeman&amp;author=DM%20Hinckley&amp;author=JJ%20de%20Pablo&amp;volume=135&amp;issue=16&amp;publication_year=2011&amp;pages=165104&amp;pmid=22047269&amp;doi=10.1063/1.3652956&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref041">
<span class="label">41.</span><cite>Chakraborty D, Hori N, Thirumalai D. Sequence-Dependent Three Interaction Site Model for Single- and Double-Stranded DNA. J Chem Theory Comput. 2018;14(7):3763–79. doi: 10.1021/acs.jctc.8b00091

</cite> [<a href="https://doi.org/10.1021/acs.jctc.8b00091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6423546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29870236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Theory%20Comput&amp;title=Sequence-Dependent%20Three%20Interaction%20Site%20Model%20for%20Single-%20and%20Double-Stranded%20DNA&amp;author=D%20Chakraborty&amp;author=N%20Hori&amp;author=D%20Thirumalai&amp;volume=14&amp;issue=7&amp;publication_year=2018&amp;pages=3763-79&amp;pmid=29870236&amp;doi=10.1021/acs.jctc.8b00091&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref042">
<span class="label">42.</span><cite>He Y, Liwo A, Scheraga HA. Optimization of a Nucleic Acids united-RESidue 2-Point model (NARES-2P) with a maximum-likelihood approach. J Chem Phys. 2015;143(24):243111. doi: 10.1063/1.4932082

</cite> [<a href="https://doi.org/10.1063/1.4932082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4608961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26723596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Phys&amp;title=Optimization%20of%20a%20Nucleic%20Acids%20united-RESidue%202-Point%20model%20(NARES-2P)%20with%20a%20maximum-likelihood%20approach&amp;author=Y%20He&amp;author=A%20Liwo&amp;author=HA%20Scheraga&amp;volume=143&amp;issue=24&amp;publication_year=2015&amp;pages=243111&amp;pmid=26723596&amp;doi=10.1063/1.4932082&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref043">
<span class="label">43.</span><cite>Cragnolini T, Laurin Y, Derreumaux P, Pasquali S. Coarse-Grained HiRE-RNA Model for ab Initio RNA Folding beyond Simple Molecules, Including Noncanonical and Multiple Base Pairings. J Chem Theory Comput. 2015;11(7):3510–22. doi: 10.1021/acs.jctc.5b00200

</cite> [<a href="https://doi.org/10.1021/acs.jctc.5b00200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26575783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Theory%20Comput&amp;title=Coarse-Grained%20HiRE-RNA%20Model%20for%20ab%20Initio%20RNA%20Folding%20beyond%20Simple%20Molecules,%20Including%20Noncanonical%20and%20Multiple%20Base%20Pairings&amp;author=T%20Cragnolini&amp;author=Y%20Laurin&amp;author=P%20Derreumaux&amp;author=S%20Pasquali&amp;volume=11&amp;issue=7&amp;publication_year=2015&amp;pages=3510-22&amp;pmid=26575783&amp;doi=10.1021/acs.jctc.5b00200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref044">
<span class="label">44.</span><cite>Zhang Y, Zhou H, Ou-Yang ZC. Stretching single-stranded DNA: interplay of electrostatic, base-pairing, and base-pair stacking interactions. Biophys J. 2001;81(2):1133–43. doi: 10.1016/S0006-3495(01)75770-0

</cite> [<a href="https://doi.org/10.1016/S0006-3495(01)75770-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1301582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11463654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Stretching%20single-stranded%20DNA:%20interplay%20of%20electrostatic,%20base-pairing,%20and%20base-pair%20stacking%20interactions&amp;author=Y%20Zhang&amp;author=H%20Zhou&amp;author=ZC%20Ou-Yang&amp;volume=81&amp;issue=2&amp;publication_year=2001&amp;pages=1133-43&amp;pmid=11463654&amp;doi=10.1016/S0006-3495(01)75770-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref045">
<span class="label">45.</span><cite>Cruz-León S, Vanderlinden W, Müller P, Forster T, Staudt G, Lin Y-Y, et al. Twisting DNA by salt. Nucleic Acids Res. 2022;50(10):5726–38. doi: 10.1093/nar/gkac445

</cite> [<a href="https://doi.org/10.1093/nar/gkac445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9177979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35640616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Twisting%20DNA%20by%20salt&amp;author=S%20Cruz-Le%C3%B3n&amp;author=W%20Vanderlinden&amp;author=P%20M%C3%BCller&amp;author=T%20Forster&amp;author=G%20Staudt&amp;volume=50&amp;issue=10&amp;publication_year=2022&amp;pages=5726-38&amp;pmid=35640616&amp;doi=10.1093/nar/gkac445&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref046">
<span class="label">46.</span><cite>Lipfert J, Doniach S, Das R, Herschlag D. Understanding nucleic acid-ion interactions. Annu Rev Biochem. 2014;83:813–41. doi: 10.1146/annurev-biochem-060409-092720

</cite> [<a href="https://doi.org/10.1146/annurev-biochem-060409-092720" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4384882/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24606136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Biochem&amp;title=Understanding%20nucleic%20acid-ion%20interactions&amp;author=J%20Lipfert&amp;author=S%20Doniach&amp;author=R%20Das&amp;author=D%20Herschlag&amp;volume=83&amp;publication_year=2014&amp;pages=813-41&amp;pmid=24606136&amp;doi=10.1146/annurev-biochem-060409-092720&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref047">
<span class="label">47.</span><cite>Maity A, Singh A, Singh N. Differential stability of DNA based on salt concentration. Eur Biophys J. 2017;46(1):33–40. doi: 10.1007/s00249-016-1132-3

</cite> [<a href="https://doi.org/10.1007/s00249-016-1132-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27165706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20Biophys%20J&amp;title=Differential%20stability%20of%20DNA%20based%20on%20salt%20concentration&amp;author=A%20Maity&amp;author=A%20Singh&amp;author=N%20Singh&amp;volume=46&amp;issue=1&amp;publication_year=2017&amp;pages=33-40&amp;pmid=27165706&amp;doi=10.1007/s00249-016-1132-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref048">
<span class="label">48.</span><cite>Tan Z-J, Chen S-J. Electrostatic free energy landscapes for nucleic acid helix assembly. Nucleic Acids Res. 2006;34(22):6629–39. doi: 10.1093/nar/gkl810

</cite> [<a href="https://doi.org/10.1093/nar/gkl810" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1751542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17145719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Electrostatic%20free%20energy%20landscapes%20for%20nucleic%20acid%20helix%20assembly&amp;author=Z-J%20Tan&amp;author=S-J%20Chen&amp;volume=34&amp;issue=22&amp;publication_year=2006&amp;pages=6629-39&amp;pmid=17145719&amp;doi=10.1093/nar/gkl810&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref049">
<span class="label">49.</span><cite>Hinckley DM, Freeman GS, Whitmer JK, de Pablo JJ. An experimentally-informed coarse-grained 3-Site-Per-Nucleotide model of DNA: structure, thermodynamics, and dynamics of hybridization. J Chem Phys. 2013;139(14):144903. doi: 10.1063/1.4822042

</cite> [<a href="https://doi.org/10.1063/1.4822042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3808442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24116642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Phys&amp;title=An%20experimentally-informed%20coarse-grained%203-Site-Per-Nucleotide%20model%20of%20DNA:%20structure,%20thermodynamics,%20and%20dynamics%20of%20hybridization&amp;author=DM%20Hinckley&amp;author=GS%20Freeman&amp;author=JK%20Whitmer&amp;author=JJ%20de%20Pablo&amp;volume=139&amp;issue=14&amp;publication_year=2013&amp;pages=144903&amp;pmid=24116642&amp;doi=10.1063/1.4822042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref050">
<span class="label">50.</span><cite>Knotts TA 4th, Rathore N, Schwartz DC, de Pablo JJ. A coarse grain model for DNA. J Chem Phys. 2007;126(8):084901. doi: 10.1063/1.2431804

</cite> [<a href="https://doi.org/10.1063/1.2431804" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17343470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Phys&amp;title=A%20coarse%20grain%20model%20for%20DNA&amp;author=TA%20Knotts&amp;author=N%20Rathore&amp;author=DC%20Schwartz&amp;author=JJ%20de%20Pablo&amp;volume=126&amp;issue=8&amp;publication_year=2007&amp;pages=084901&amp;pmid=17343470&amp;doi=10.1063/1.2431804&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref051">
<span class="label">51.</span><cite>Freeman GS, Hinckley DM, Lequieu JP, Whitmer JK, De Pablo JJ. Coarse-grained modeling of DNA curvature. J Chem Phys. 2014;141:165103.
</cite> [<a href="https://doi.org/10.1063/1.4897649" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25362344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Phys&amp;title=Coarse-grained%20modeling%20of%20DNA%20curvature&amp;author=GS%20Freeman&amp;author=DM%20Hinckley&amp;author=JP%20Lequieu&amp;author=JK%20Whitmer&amp;author=JJ%20De%20Pablo&amp;volume=141&amp;publication_year=2014&amp;pages=165103&amp;pmid=25362344&amp;doi=10.1063/1.4897649&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref052">
<span class="label">52.</span><cite>Ouldridge TE, Louis AA, Doye JPK. Structural, mechanical, and thermodynamic properties of a coarse-grained DNA model. J Chem Phys. 2011;134(8):085101. doi: 10.1063/1.3552946

</cite> [<a href="https://doi.org/10.1063/1.3552946" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21361556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Phys&amp;title=Structural,%20mechanical,%20and%20thermodynamic%20properties%20of%20a%20coarse-grained%20DNA%20model&amp;author=TE%20Ouldridge&amp;author=AA%20Louis&amp;author=JPK%20Doye&amp;volume=134&amp;issue=8&amp;publication_year=2011&amp;pages=085101&amp;pmid=21361556&amp;doi=10.1063/1.3552946&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref053">
<span class="label">53.</span><cite>Hong F, Schreck JS, Šulc P. Understanding DNA interactions in crowded environments with a coarse-grained model. Nucleic Acids Res. 2020;48(19):10726–38. doi: 10.1093/nar/gkaa854

</cite> [<a href="https://doi.org/10.1093/nar/gkaa854" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7641764/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33045749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Understanding%20DNA%20interactions%20in%20crowded%20environments%20with%20a%20coarse-grained%20model&amp;author=F%20Hong&amp;author=JS%20Schreck&amp;author=P%20%C5%A0ulc&amp;volume=48&amp;issue=19&amp;publication_year=2020&amp;pages=10726-38&amp;pmid=33045749&amp;doi=10.1093/nar/gkaa854&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref054">
<span class="label">54.</span><cite>Hayes RL, Noel JK, Mandic A, Whitford PC, Sanbonmatsu KY, Mohanty U, et al. Generalized Manning Condensation Model Captures the RNA Ion Atmosphere. Phys Rev Lett. 2015;114(25):258105. doi: 10.1103/PhysRevLett.114.258105

</cite> [<a href="https://doi.org/10.1103/PhysRevLett.114.258105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4833092/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26197147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phys%20Rev%20Lett&amp;title=Generalized%20Manning%20Condensation%20Model%20Captures%20the%20RNA%20Ion%20Atmosphere&amp;author=RL%20Hayes&amp;author=JK%20Noel&amp;author=A%20Mandic&amp;author=PC%20Whitford&amp;author=KY%20Sanbonmatsu&amp;volume=114&amp;issue=25&amp;publication_year=2015&amp;pages=258105&amp;pmid=26197147&amp;doi=10.1103/PhysRevLett.114.258105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref055">
<span class="label">55.</span><cite>Nguyen HT, Hori N, Thirumalai D. Theory and simulations for RNA folding in mixtures of monovalent and divalent cations. Proc Natl Acad Sci U S A. 2019;116(42):21022–30. doi: 10.1073/pnas.1911632116

</cite> [<a href="https://doi.org/10.1073/pnas.1911632116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6800359/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31570624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Theory%20and%20simulations%20for%20RNA%20folding%20in%20mixtures%20of%20monovalent%20and%20divalent%20cations&amp;author=HT%20Nguyen&amp;author=N%20Hori&amp;author=D%20Thirumalai&amp;volume=116&amp;issue=42&amp;publication_year=2019&amp;pages=21022-30&amp;pmid=31570624&amp;doi=10.1073/pnas.1911632116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref056">
<span class="label">56.</span><cite>Shi Y-Z, Jin L, Wang F-H, Zhu X-L, Tan Z-J. Predicting 3D Structure, Flexibility, and Stability of RNA Hairpins in Monovalent and Divalent Ion Solutions. Biophys J. 2015;109(12):2654–65. doi: 10.1016/j.bpj.2015.11.006

</cite> [<a href="https://doi.org/10.1016/j.bpj.2015.11.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4701004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26682822/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Predicting%203D%20Structure,%20Flexibility,%20and%20Stability%20of%20RNA%20Hairpins%20in%20Monovalent%20and%20Divalent%20Ion%20Solutions&amp;author=Y-Z%20Shi&amp;author=L%20Jin&amp;author=F-H%20Wang&amp;author=X-L%20Zhu&amp;author=Z-J%20Tan&amp;volume=109&amp;issue=12&amp;publication_year=2015&amp;pages=2654-65&amp;pmid=26682822&amp;doi=10.1016/j.bpj.2015.11.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref057">
<span class="label">57.</span><cite>Wang X, Tan Y-L, Yu S, Shi Y-Z, Tan Z-J. Predicting 3D structures and stabilities for complex RNA pseudoknots in ion solutions. Biophys J. 2023;122(8):1503–16. doi: 10.1016/j.bpj.2023.03.017

</cite> [<a href="https://doi.org/10.1016/j.bpj.2023.03.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10147842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36924021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Predicting%203D%20structures%20and%20stabilities%20for%20complex%20RNA%20pseudoknots%20in%20ion%20solutions&amp;author=X%20Wang&amp;author=Y-L%20Tan&amp;author=S%20Yu&amp;author=Y-Z%20Shi&amp;author=Z-J%20Tan&amp;volume=122&amp;issue=8&amp;publication_year=2023&amp;pages=1503-16&amp;pmid=36924021&amp;doi=10.1016/j.bpj.2023.03.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref058">
<span class="label">58.</span><cite>Manning GS. The molecular theory of polyelectrolyte solutions with applications to the electrostatic properties of polynucleotides. Q Rev Biophys. 1978;11(2):179–246. doi: 10.1017/s0033583500002031

</cite> [<a href="https://doi.org/10.1017/s0033583500002031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/353876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Q%20Rev%20Biophys&amp;title=The%20molecular%20theory%20of%20polyelectrolyte%20solutions%20with%20applications%20to%20the%20electrostatic%20properties%20of%20polynucleotides&amp;author=GS%20Manning&amp;volume=11&amp;issue=2&amp;publication_year=1978&amp;pages=179-246&amp;pmid=353876&amp;doi=10.1017/s0033583500002031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref059">
<span class="label">59.</span><cite>Tan Z-J, Chen S-J. Nucleic acid helix stability: effects of salt concentration, cation valence and size, and chain length. Biophys J. 2006;90(4):1175–90. doi: 10.1529/biophysj.105.070904

</cite> [<a href="https://doi.org/10.1529/biophysj.105.070904" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1367269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16299077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Nucleic%20acid%20helix%20stability:%20effects%20of%20salt%20concentration,%20cation%20valence%20and%20size,%20and%20chain%20length&amp;author=Z-J%20Tan&amp;author=S-J%20Chen&amp;volume=90&amp;issue=4&amp;publication_year=2006&amp;pages=1175-90&amp;pmid=16299077&amp;doi=10.1529/biophysj.105.070904&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref060">
<span class="label">60.</span><cite>Shi Y-Z, Wang F-H, Wu Y-Y, Tan Z-J. A coarse-grained model with implicit salt for RNAs: predicting 3D structure, stability and salt effect. J Chem Phys. 2014;141(10):105102. doi: 10.1063/1.4894752

</cite> [<a href="https://doi.org/10.1063/1.4894752" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25217954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Phys&amp;title=A%20coarse-grained%20model%20with%20implicit%20salt%20for%20RNAs:%20predicting%203D%20structure,%20stability%20and%20salt%20effect&amp;author=Y-Z%20Shi&amp;author=F-H%20Wang&amp;author=Y-Y%20Wu&amp;author=Z-J%20Tan&amp;volume=141&amp;issue=10&amp;publication_year=2014&amp;pages=105102&amp;pmid=25217954&amp;doi=10.1063/1.4894752&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref061">
<span class="label">61.</span><cite>Hukushima K, Nemoto K. Exchange Monte Carlo method and application to spin glass simulations. J Phys Soc Japan. 1996;65:1604–8.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Phys%20Soc%20Japan&amp;title=Exchange%20Monte%20Carlo%20method%20and%20application%20to%20spin%20glass%20simulations&amp;author=K%20Hukushima&amp;author=K%20Nemoto&amp;volume=65&amp;publication_year=1996&amp;pages=1604-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref062">
<span class="label">62.</span><cite>Okamoto Y. Generalized-ensemble algorithms: enhanced sampling techniques for Monte Carlo and molecular dynamics simulations. J Mol Graph Model. 2004;22:425–39.
</cite> [<a href="https://doi.org/10.1016/j.jmgm.2003.12.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15099838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Graph%20Model&amp;title=Generalized-ensemble%20algorithms:%20enhanced%20sampling%20techniques%20for%20Monte%20Carlo%20and%20molecular%20dynamics%20simulations&amp;author=Y%20Okamoto&amp;volume=22&amp;publication_year=2004&amp;pages=425-39&amp;pmid=15099838&amp;doi=10.1016/j.jmgm.2003.12.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref063">
<span class="label">63.</span><cite>Fukunishi H, Watanabe O, Takada S. On the Hamiltonian replica exchange method for efficient sampling of biomolecular systems: application to protein structure prediction. J Chem Phys. 2002;116:9058–67.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Phys&amp;title=On%20the%20Hamiltonian%20replica%20exchange%20method%20for%20efficient%20sampling%20of%20biomolecular%20systems:%20application%20to%20protein%20structure%20prediction&amp;author=H%20Fukunishi&amp;author=O%20Watanabe&amp;author=S%20Takada&amp;volume=116&amp;publication_year=2002&amp;pages=9058-67&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref064">
<span class="label">64.</span><cite>Boniecki MJ, Lach G, Dawson WK, Tomala K, Lukasz P, Soltysinski T, et al. SimRNA: a coarse-grained method for RNA folding simulations and 3D structure prediction. Nucleic Acids Res. 2016;44(7):e63. doi: 10.1093/nar/gkv1479

</cite> [<a href="https://doi.org/10.1093/nar/gkv1479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4838351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26687716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=SimRNA:%20a%20coarse-grained%20method%20for%20RNA%20folding%20simulations%20and%203D%20structure%20prediction&amp;author=MJ%20Boniecki&amp;author=G%20Lach&amp;author=WK%20Dawson&amp;author=K%20Tomala&amp;author=P%20Lukasz&amp;volume=44&amp;issue=7&amp;publication_year=2016&amp;pmid=26687716&amp;doi=10.1093/nar/gkv1479&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref065">
<span class="label">65.</span><cite>Stasiewicz J, Mukherjee S, Nithin C, Bujnicki JM. QRNAS: software tool for refinement of nucleic acid structures. BMC Struct Biol. 2019;19(1):5. doi: 10.1186/s12900-019-0103-1

</cite> [<a href="https://doi.org/10.1186/s12900-019-0103-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6429776/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30898165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Struct%20Biol&amp;title=QRNAS:%20software%20tool%20for%20refinement%20of%20nucleic%20acid%20structures&amp;author=J%20Stasiewicz&amp;author=S%20Mukherjee&amp;author=C%20Nithin&amp;author=JM%20Bujnicki&amp;volume=19&amp;issue=1&amp;publication_year=2019&amp;pages=5&amp;pmid=30898165&amp;doi=10.1186/s12900-019-0103-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref066">
<span class="label">66.</span><cite>Chodera JD, Swope WC, Pitera JW, Seok C, Dill KA. Use of the Weighted Histogram Analysis Method for the Analysis of Simulated and Parallel Tempering Simulations. J Chem Theory Comput. 2007;3(1):26–41. doi: 10.1021/ct0502864

</cite> [<a href="https://doi.org/10.1021/ct0502864" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26627148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Theory%20Comput&amp;title=Use%20of%20the%20Weighted%20Histogram%20Analysis%20Method%20for%20the%20Analysis%20of%20Simulated%20and%20Parallel%20Tempering%20Simulations&amp;author=JD%20Chodera&amp;author=WC%20Swope&amp;author=JW%20Pitera&amp;author=C%20Seok&amp;author=KA%20Dill&amp;volume=3&amp;issue=1&amp;publication_year=2007&amp;pages=26-41&amp;pmid=26627148&amp;doi=10.1021/ct0502864&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref067">
<span class="label">67.</span><cite>Xia T, SantaLucia J Jr, Burkard ME, Kierzek R, Schroeder SJ, Jiao X, et al. Thermodynamic parameters for an expanded nearest-neighbor model for formation of RNA duplexes with Watson− Crick base pairs. Biochemistry. 1998;37:14719–35.
</cite> [<a href="https://doi.org/10.1021/bi9809425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9778347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Thermodynamic%20parameters%20for%20an%20expanded%20nearest-neighbor%20model%20for%20formation%20of%20RNA%20duplexes%20with%20Watson%E2%88%92%20Crick%20base%20pairs&amp;author=T%20Xia&amp;author=J%20SantaLucia&amp;author=ME%20Burkard&amp;author=R%20Kierzek&amp;author=SJ%20Schroeder&amp;volume=37&amp;publication_year=1998&amp;pages=14719-35&amp;pmid=9778347&amp;doi=10.1021/bi9809425&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref068">
<span class="label">68.</span><cite>Kanungo T, Mount DM, Netanyahu NS, Piatko C, Silverman R, Wu AY. Proceedings of the sixteenth annual symposium on Computational geometry. 2000. p. 100–9.</cite>
</li>
<li id="pcbi.1013346.ref069">
<span class="label">69.</span><cite>Shi Y-Z, Jin L, Feng C-J, Tan Y-L, Tan Z-J. Predicting 3D structure and stability of RNA pseudoknots in monovalent and divalent ion solutions. PLoS Comput Biol. 2018;14(6):e1006222. doi: 10.1371/journal.pcbi.1006222

</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1006222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6007934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29879103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=Predicting%203D%20structure%20and%20stability%20of%20RNA%20pseudoknots%20in%20monovalent%20and%20divalent%20ion%20solutions&amp;author=Y-Z%20Shi&amp;author=L%20Jin&amp;author=C-J%20Feng&amp;author=Y-L%20Tan&amp;author=Z-J%20Tan&amp;volume=14&amp;issue=6&amp;publication_year=2018&amp;pmid=29879103&amp;doi=10.1371/journal.pcbi.1006222&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref070">
<span class="label">70.</span><cite>Zhao Y, Huang Y, Gong Z, Wang Y, Man J, Xiao Y. Automated and fast building of three-dimensional RNA structures. Sci Rep. 2012;2:734.
</cite> [<a href="https://doi.org/10.1038/srep00734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3471093/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23071898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Automated%20and%20fast%20building%20of%20three-dimensional%20RNA%20structures&amp;author=Y%20Zhao&amp;author=Y%20Huang&amp;author=Z%20Gong&amp;author=Y%20Wang&amp;author=J%20Man&amp;volume=2&amp;publication_year=2012&amp;pages=734&amp;pmid=23071898&amp;doi=10.1038/srep00734&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref071">
<span class="label">71.</span><cite>Zhang D, Chen S-J. IsRNA: An Iterative Simulated Reference State Approach to Modeling Correlated Interactions in RNA Folding. J Chem Theory Comput. 2018;14(4):2230–9. doi: 10.1021/acs.jctc.7b01228

</cite> [<a href="https://doi.org/10.1021/acs.jctc.7b01228" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6311103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29499114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Theory%20Comput&amp;title=IsRNA:%20An%20Iterative%20Simulated%20Reference%20State%20Approach%20to%20Modeling%20Correlated%20Interactions%20in%20RNA%20Folding&amp;author=D%20Zhang&amp;author=S-J%20Chen&amp;volume=14&amp;issue=4&amp;publication_year=2018&amp;pages=2230-9&amp;pmid=29499114&amp;doi=10.1021/acs.jctc.7b01228&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref072">
<span class="label">72.</span><cite>Singh J, Hanson J, Paliwal K, Zhou Y. RNA secondary structure prediction using an ensemble of two-dimensional deep neural networks and transfer learning. Nat Commun. 2019;10(1):5407. doi: 10.1038/s41467-019-13395-9

</cite> [<a href="https://doi.org/10.1038/s41467-019-13395-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6881452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31776342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=RNA%20secondary%20structure%20prediction%20using%20an%20ensemble%20of%20two-dimensional%20deep%20neural%20networks%20and%20transfer%20learning&amp;author=J%20Singh&amp;author=J%20Hanson&amp;author=K%20Paliwal&amp;author=Y%20Zhou&amp;volume=10&amp;issue=1&amp;publication_year=2019&amp;pages=5407&amp;pmid=31776342&amp;doi=10.1038/s41467-019-13395-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref073">
<span class="label">73.</span><cite>Cruz JA, Blanchet M-F, Boniecki M, Bujnicki JM, Chen S-J, Cao S, et al. RNA-Puzzles: a CASP-like evaluation of RNA three-dimensional structure prediction. RNA. 2012;18(4):610–25. doi: 10.1261/rna.031054.111

</cite> [<a href="https://doi.org/10.1261/rna.031054.111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3312550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22361291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=RNA-Puzzles:%20a%20CASP-like%20evaluation%20of%20RNA%20three-dimensional%20structure%20prediction&amp;author=JA%20Cruz&amp;author=M-F%20Blanchet&amp;author=M%20Boniecki&amp;author=JM%20Bujnicki&amp;author=S-J%20Chen&amp;volume=18&amp;issue=4&amp;publication_year=2012&amp;pages=610-25&amp;pmid=22361291&amp;doi=10.1261/rna.031054.111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref074">
<span class="label">74.</span><cite>Wu B, Girard F, van Buuren B, Schleucher J, Tessari M, Wijmenga S. Global structure of a DNA three-way junction by solution NMR: towards prediction of 3H fold. Nucleic Acids Res. 2004;32(10):3228–39. doi: 10.1093/nar/gkh645

</cite> [<a href="https://doi.org/10.1093/nar/gkh645" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC434450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15199171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Global%20structure%20of%20a%20DNA%20three-way%20junction%20by%20solution%20NMR:%20towards%20prediction%20of%203H%20fold&amp;author=B%20Wu&amp;author=F%20Girard&amp;author=B%20van%20Buuren&amp;author=J%20Schleucher&amp;author=M%20Tessari&amp;volume=32&amp;issue=10&amp;publication_year=2004&amp;pages=3228-39&amp;pmid=15199171&amp;doi=10.1093/nar/gkh645&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref075">
<span class="label">75.</span><cite>Huang R-H, Fremont DH, Diener JL, Schaub RG, Sadler JE. A structural explanation for the antithrombotic activity of ARC1172, a DNA aptamer that binds von Willebrand factor domain A1. Structure. 2009;17(11):1476–84. doi: 10.1016/j.str.2009.09.011

</cite> [<a href="https://doi.org/10.1016/j.str.2009.09.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3845234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19913482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=A%20structural%20explanation%20for%20the%20antithrombotic%20activity%20of%20ARC1172,%20a%20DNA%20aptamer%20that%20binds%20von%20Willebrand%20factor%20domain%20A1&amp;author=R-H%20Huang&amp;author=DH%20Fremont&amp;author=JL%20Diener&amp;author=RG%20Schaub&amp;author=JE%20Sadler&amp;volume=17&amp;issue=11&amp;publication_year=2009&amp;pages=1476-84&amp;pmid=19913482&amp;doi=10.1016/j.str.2009.09.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref076">
<span class="label">76.</span><cite>Andrałojć W, Wieruszewska J, Pasternak K, Gdaniec Z. Solution Structure of a Lanthanide-binding DNA Aptamer Determined Using High Quality pseudocontact shift restraints. Chemistry. 2022;28(66):e202202114. doi: 10.1002/chem.202202114

</cite> [<a href="https://doi.org/10.1002/chem.202202114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9828363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36043489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemistry&amp;title=Solution%20Structure%20of%20a%20Lanthanide-binding%20DNA%20Aptamer%20Determined%20Using%20High%20Quality%20pseudocontact%20shift%20restraints&amp;author=W%20Andra%C5%82oj%C4%87&amp;author=J%20Wieruszewska&amp;author=K%20Pasternak&amp;author=Z%20Gdaniec&amp;volume=28&amp;issue=66&amp;publication_year=2022&amp;pmid=36043489&amp;doi=10.1002/chem.202202114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref077">
<span class="label">77.</span><cite>Zhang Y, Xiong Y, Yang C, Xiao Y. 3dRNA/DNA: 3D Structure Prediction from RNA to DNA. J Mol Biol. 2024;436(17):168742. doi: 10.1016/j.jmb.2024.168742

</cite> [<a href="https://doi.org/10.1016/j.jmb.2024.168742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39237199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=3dRNA/DNA:%203D%20Structure%20Prediction%20from%20RNA%20to%20DNA&amp;author=Y%20Zhang&amp;author=Y%20Xiong&amp;author=C%20Yang&amp;author=Y%20Xiao&amp;volume=436&amp;issue=17&amp;publication_year=2024&amp;pages=168742&amp;pmid=39237199&amp;doi=10.1016/j.jmb.2024.168742&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref078">
<span class="label">78.</span><cite>Carr CE, Marky LA. Effect of GCAA stabilizing loops on three- and four-way intramolecular junctions. Phys Chem Chem Phys. 2018;20(7):5046–56. doi: 10.1039/c7cp08329g

</cite> [<a href="https://doi.org/10.1039/c7cp08329g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29388988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phys%20Chem%20Chem%20Phys&amp;title=Effect%20of%20GCAA%20stabilizing%20loops%20on%20three-%20and%20four-way%20intramolecular%20junctions&amp;author=CE%20Carr&amp;author=LA%20Marky&amp;volume=20&amp;issue=7&amp;publication_year=2018&amp;pages=5046-56&amp;pmid=29388988&amp;doi=10.1039/c7cp08329g&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref079">
<span class="label">79.</span><cite>Fu H, Zhang C, Qiang X-W, Yang Y-J, Dai L, Tan Z-J, et al. Opposite Effects of High-Valent Cations on the Elasticities of DNA and RNA Duplexes Revealed by Magnetic Tweezers. Phys Rev Lett. 2020;124(5):058101. doi: 10.1103/PhysRevLett.124.058101

</cite> [<a href="https://doi.org/10.1103/PhysRevLett.124.058101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32083903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phys%20Rev%20Lett&amp;title=Opposite%20Effects%20of%20High-Valent%20Cations%20on%20the%20Elasticities%20of%20DNA%20and%20RNA%20Duplexes%20Revealed%20by%20Magnetic%20Tweezers&amp;author=H%20Fu&amp;author=C%20Zhang&amp;author=X-W%20Qiang&amp;author=Y-J%20Yang&amp;author=L%20Dai&amp;volume=124&amp;issue=5&amp;publication_year=2020&amp;pages=058101&amp;pmid=32083903&amp;doi=10.1103/PhysRevLett.124.058101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref080">
<span class="label">80.</span><cite>Zhang C, Tian FJ, Zuo HW, Qiu QY, Zhang JH, Wei W, et al. Counterintuitive DNA destabilization by monovalent salt at high concentrations due to overcharging. Nat Commun. 2025;16:113.
</cite> [<a href="https://doi.org/10.1038/s41467-024-55404-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11697420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39747043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Counterintuitive%20DNA%20destabilization%20by%20monovalent%20salt%20at%20high%20concentrations%20due%20to%20overcharging&amp;author=C%20Zhang&amp;author=FJ%20Tian&amp;author=HW%20Zuo&amp;author=QY%20Qiu&amp;author=JH%20Zhang&amp;volume=16&amp;publication_year=2025&amp;pages=113&amp;pmid=39747043&amp;doi=10.1038/s41467-024-55404-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref081">
<span class="label">81.</span><cite>Sardana D, Alam P, Yadav K, Clovis NS, Kumar P, Sen S. Unusual similarity of DNA solvation dynamics in high-salinity crowding with divalent cations of varying concentrations. Phys Chem Chem Phys. 2023;25:27744–55.
</cite> [<a href="https://doi.org/10.1039/d3cp02606j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37814577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phys%20Chem%20Chem%20Phys&amp;title=Unusual%20similarity%20of%20DNA%20solvation%20dynamics%20in%20high-salinity%20crowding%20with%20divalent%20cations%20of%20varying%20concentrations&amp;author=D%20Sardana&amp;author=P%20Alam&amp;author=K%20Yadav&amp;author=NS%20Clovis&amp;author=P%20Kumar&amp;volume=25&amp;publication_year=2023&amp;pages=27744-55&amp;pmid=37814577&amp;doi=10.1039/d3cp02606j&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref082">
<span class="label">82.</span><cite>Zuker M. Mfold web server for nucleic acid folding and hybridization prediction. Nucleic Acids Res. 2003;31(13):3406–15. doi: 10.1093/nar/gkg595

</cite> [<a href="https://doi.org/10.1093/nar/gkg595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC169194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12824337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Mfold%20web%20server%20for%20nucleic%20acid%20folding%20and%20hybridization%20prediction&amp;author=M%20Zuker&amp;volume=31&amp;issue=13&amp;publication_year=2003&amp;pages=3406-15&amp;pmid=12824337&amp;doi=10.1093/nar/gkg595&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref083">
<span class="label">83.</span><cite>Das J, Mukherjee S, Mitra A, Bhattacharyya D. Non-canonical base pairs and higher order structures in nucleic acids: crystal structure database analysis. J Biomol Struct Dyn. 2006;24(2):149–61. doi: 10.1080/07391102.2006.10507108

</cite> [<a href="https://doi.org/10.1080/07391102.2006.10507108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16928138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biomol%20Struct%20Dyn&amp;title=Non-canonical%20base%20pairs%20and%20higher%20order%20structures%20in%20nucleic%20acids:%20crystal%20structure%20database%20analysis&amp;author=J%20Das&amp;author=S%20Mukherjee&amp;author=A%20Mitra&amp;author=D%20Bhattacharyya&amp;volume=24&amp;issue=2&amp;publication_year=2006&amp;pages=149-61&amp;pmid=16928138&amp;doi=10.1080/07391102.2006.10507108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref084">
<span class="label">84.</span><cite>Mukherjee S, Bansal M, Bhattacharyya D. Conformational specificity of non-canonical base pairs and higher order structures in nucleic acids: crystal structure database analysis. J Comput Aided Mol Des. 2006;20(10–11):629–45. doi: 10.1007/s10822-006-9083-x

</cite> [<a href="https://doi.org/10.1007/s10822-006-9083-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17124630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Aided%20Mol%20Des&amp;title=Conformational%20specificity%20of%20non-canonical%20base%20pairs%20and%20higher%20order%20structures%20in%20nucleic%20acids:%20crystal%20structure%20database%20analysis&amp;author=S%20Mukherjee&amp;author=M%20Bansal&amp;author=D%20Bhattacharyya&amp;volume=20&amp;publication_year=2006&amp;pages=629-45&amp;pmid=17124630&amp;doi=10.1007/s10822-006-9083-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref085">
<span class="label">85.</span><cite>van Dongen MJ, Doreleijers JF, van der Marel GA, van Boom JH, Hilbers CW, Wijmenga SS. Structure and mechanism of formation of the H-y5 isomer of an intramolecular DNA triple helix. Nat Struct Biol. 1999;6:854–9.
</cite> [<a href="https://doi.org/10.1038/12313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10467098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Struct%20Biol&amp;title=Structure%20and%20mechanism%20of%20formation%20of%20the%20H-y5%20isomer%20of%20an%20intramolecular%20DNA%20triple%20helix&amp;author=MJ%20van%20Dongen&amp;author=JF%20Doreleijers&amp;author=GA%20van%20der%20Marel&amp;author=JH%20van%20Boom&amp;author=CW%20Hilbers&amp;volume=6&amp;publication_year=1999&amp;pages=854-9&amp;pmid=10467098&amp;doi=10.1038/12313&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref086">
<span class="label">86.</span><cite>Koshlap KM, Schultze P, Brunar H, Dervan PB, Feigon J. Solution structure of an intramolecular DNA triplex containing an N7-glycosylated guanine which mimics a protonated cytosine. Biochemistry. 1997;36(9):2659–68. doi: 10.1021/bi962438a

</cite> [<a href="https://doi.org/10.1021/bi962438a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9054573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Solution%20structure%20of%20an%20intramolecular%20DNA%20triplex%20containing%20an%20N7-glycosylated%20guanine%20which%20mimics%20a%20protonated%20cytosine&amp;author=KM%20Koshlap&amp;author=P%20Schultze&amp;author=H%20Brunar&amp;author=PB%20Dervan&amp;author=J%20Feigon&amp;volume=36&amp;issue=9&amp;publication_year=1997&amp;pages=2659-68&amp;pmid=9054573&amp;doi=10.1021/bi962438a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref087">
<span class="label">87.</span><cite>Lam EYN, Beraldi D, Tannahill D, Balasubramanian S. G-quadruplex structures are stable and detectable in human genomic DNA. Nat Commun. 2013;4:1796.
</cite> [<a href="https://doi.org/10.1038/ncomms2792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3736099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23653208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=G-quadruplex%20structures%20are%20stable%20and%20detectable%20in%20human%20genomic%20DNA&amp;author=EYN%20Lam&amp;author=D%20Beraldi&amp;author=D%20Tannahill&amp;author=S%20Balasubramanian&amp;volume=4&amp;publication_year=2013&amp;pages=1796&amp;pmid=23653208&amp;doi=10.1038/ncomms2792&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref088">
<span class="label">88.</span><cite>Mao S-Q, Ghanbarian AT, Spiegel J, Martínez Cuesta S, Beraldi D, Di Antonio M, et al. DNA G-quadruplex structures mold the DNA methylome. Nat Struct Mol Biol. 2018;25(10):951–7. doi: 10.1038/s41594-018-0131-8

</cite> [<a href="https://doi.org/10.1038/s41594-018-0131-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6173298/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30275516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Struct%20Mol%20Biol&amp;title=DNA%20G-quadruplex%20structures%20mold%20the%20DNA%20methylome&amp;author=S-Q%20Mao&amp;author=AT%20Ghanbarian&amp;author=J%20Spiegel&amp;author=S%20Mart%C3%ADnez%20Cuesta&amp;author=D%20Beraldi&amp;volume=25&amp;issue=10&amp;publication_year=2018&amp;pages=951-7&amp;pmid=30275516&amp;doi=10.1038/s41594-018-0131-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref089">
<span class="label">89.</span><cite>Tan YL, Wang X, Shi YZ, Zhang W, Tan ZJ. rsRNASP: A residue-separation-based statistical potential for RNA 3D structure evaluation. Biophys J. 2022;121:142–56.
</cite> [<a href="https://doi.org/10.1016/j.bpj.2021.11.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8758408/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34798137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=rsRNASP:%20A%20residue-separation-based%20statistical%20potential%20for%20RNA%203D%20structure%20evaluation&amp;author=YL%20Tan&amp;author=X%20Wang&amp;author=YZ%20Shi&amp;author=W%20Zhang&amp;author=ZJ%20Tan&amp;volume=121&amp;publication_year=2022&amp;pages=142-56&amp;pmid=34798137&amp;doi=10.1016/j.bpj.2021.11.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref090">
<span class="label">90.</span><cite>Singh A, Singh N. DNA melting in the presence of molecular crowders. Phys Chem Chem Phys. 2017;19(29):19452–60. doi: 10.1039/c7cp03624h

</cite> [<a href="https://doi.org/10.1039/c7cp03624h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28718468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phys%20Chem%20Chem%20Phys&amp;title=DNA%20melting%20in%20the%20presence%20of%20molecular%20crowders&amp;author=A%20Singh&amp;author=N%20Singh&amp;volume=19&amp;issue=29&amp;publication_year=2017&amp;pages=19452-60&amp;pmid=28718468&amp;doi=10.1039/c7cp03624h&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref091">
<span class="label">91.</span><cite>Mathur N, Singh A, Singh N. Force-induced unzipping of DNA in the presence of solvent molecules. Biophys Chem. 2024;307:107175. doi: 10.1016/j.bpc.2024.107175

</cite> [<a href="https://doi.org/10.1016/j.bpc.2024.107175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38244296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20Chem&amp;title=Force-induced%20unzipping%20of%20DNA%20in%20the%20presence%20of%20solvent%20molecules&amp;author=N%20Mathur&amp;author=A%20Singh&amp;author=N%20Singh&amp;volume=307&amp;publication_year=2024&amp;pages=107175&amp;pmid=38244296&amp;doi=10.1016/j.bpc.2024.107175&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref092">
<span class="label">92.</span><cite>Darty K, Denise A, Ponty Y. VARNA: Interactive drawing and editing of the RNA secondary structure. Bioinformatics. 2009;25:1974.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2712331/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19398448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=VARNA:%20Interactive%20drawing%20and%20editing%20of%20the%20RNA%20secondary%20structure&amp;author=K%20Darty&amp;author=A%20Denise&amp;author=Y%20Ponty&amp;volume=25&amp;publication_year=2009&amp;pages=1974&amp;pmid=19398448&amp;doi=10.1093/bioinformatics/btp250&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013346.ref093">
<span class="label">93.</span><cite>DeLano WL. Pymol: An open-source molecular graphics tool. CCP4 Newsl Protein Crystallogr. 2002;40:82–92.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=CCP4%20Newsl%20Protein%20Crystallogr&amp;title=Pymol:%20An%20open-source%20molecular%20graphics%20tool&amp;author=WL%20DeLano&amp;volume=40&amp;publication_year=2002&amp;pages=82-92&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Fig</span><p><strong>(A) The schematic diagram for the formation of one base stacking between base pairs (</strong><strong><em>i, j</em></strong><strong>) and (</strong><strong><em>i</em></strong><strong> + 1,</strong>
<strong><em>j</em></strong><strong>-1)</strong><strong>; (B) The conformational entropy changes ΔS</strong><sub><strong>c</strong></sub>
<strong>for the formation of base-pairs stacking at different location <em>i</em> (symbols), and the average value of ΔS</strong><sub><strong>c</strong></sub>
<strong>(line).</strong></p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s001.tif" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s001.tif</a><sup> (508.6KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S2 Fig. The schematic diagram of the rebuilding of coarse-grained DNA structures into all-atom ones in our present model.</span><p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s002.tif" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s002.tif</a><sup> (785.8KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S3 Fig. The eight structural states of 3WJ, including a folded state, six intermediate states, and a unfolded state.</span><p>Here, F state: Stems 1, 2, and 3 retained, I1: Stem 1 resolved, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stem 3 resolved, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stem 2 resolved, I2: Stems 1 and 3 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 1 and 2 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 2 and 3 resolved, U: All stems resolved.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s003.tif" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s003.tif</a><sup> (450KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S4 Fig. The sequence and secondary structure of 6 DNAs with multi-way junctions employed in this paper.</span><p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s004.tif" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s004.tif</a><sup> (480.6KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>S5 Fig. The average radius of gyration (Rg) and ion neutralization fraction of different structural states for 3WJ and 4WJ.</span><p>(A,B) The average Rg (A) and ion neutralization fraction (B) of folded state (F), intermediate states I1*/I2* (two stems retained/one stem retained), unfolded state (zero stem retained) for 3WJ. (C,D) The average Rg (A) and ion neutralization fraction (B) of folded state (F), intermediate states I1*/I3* (three stems retained/one stem retained), unfolded state (zero stem retained) for 4WJ.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s005.tif" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s005.tif</a><sup> (1MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>S6 Fig. The sixteen structural states of 4Wj, including a folded state, fourteen intermediate states, a unfolded state.</span><p>Here F: All stem retained, I1: Stem 3 resolved, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stem 1 resolved, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stem 4 resolved, I1<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stem 2 resolved, I2: Stems 1 and 3 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 1 and 4 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 2 and 3 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 1 and 2 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 2 and 3 resolved, I2<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 2 and 4 resolved, I3: Stems 1,3, and 4 resolved, I3<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 1, 2, and 3 resolved, I3<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 1, 2, and 4 resolved, I3<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow><mrow><mi>′</mi></mrow></mrow></mrow></math></span>: Stems 2, 3, and 4 resolved, U: All stems resolved.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s006.tif" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s006.tif</a><sup> (496.4KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>S1 Table. The PDB codes of 138 DNAs used in our statistical analysis for CG force field.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s007.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s007.xlsx</a><sup> (10.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p">
<span>S2 Table. The parameters of bonded potentials of CG force field.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s008.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s008.xlsx</a><sup> (31.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="caption p">
<span>S3 Table. The parameters for the energy functions of base pairing and base stacking.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s009.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s009.xlsx</a><sup> (21KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="caption p">
<span>S4 Table. The predicted RMSD and F1-socre by three prediction models for 4 DNAs.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s010.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s010.xlsx</a><sup> (10.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="caption p">
<span>S5 Table. The predicted temperature of 3WJ at extensive ion concentration by our present model.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s011.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s011.xlsx</a><sup> (14.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="caption p">
<span>S6 Table. The predicted temperature of 4WJ at extensive ion concentration by our present model.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s012.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s012.xlsx</a><sup> (14.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="caption p">
<span>S1 Text. Detailed description of the coarse-grained force field of the present model and the Weighted Histogram Analysis Method.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373291/bin/pcbi.1013346.s013.docx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013346.s013.docx</a><sup> (44.6KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All relevant computational data are within the paper and its Supporting information files. Experimental data are publicly available from published papers (DOI: <a href="https://doi.org/10.1039/c7cp08329g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1039/c7cp08329g</a>) cited in the work. The package of the present model is freely available at <a href="https://github.com/RNA-folding-lab/DNAfold2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/RNA-folding-lab/DNAfold2</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS Computational Biology are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pcbi.1013346"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pcbi.1013346.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373291/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373291/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373291%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373291/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373291/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373291/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40825053/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373291/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40825053/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373291/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373291/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="zus5F1eB8Kqu4BxlJRxY9OXDZwvThAkTTJHq1dyQQMGgeVQVgJJRwckD9jnZBx7s">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
