
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            MRD4U: A path to development for personalized liquid biopsy for children with central nervous system tumors - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E487118AEF55230587110010D8311B.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmccanc">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374395/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Cancer">
<meta name="citation_title" content="MRD4U: A path to development for personalized liquid biopsy for children with central nervous system tumors">
<meta name="citation_author" content="Anthony R Miller">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Tithi Shah">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Corinne N Strawser">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Adithe Rivaldi">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Sarah A Wilson">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Huachun Zhong">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Jocelyn M Lucyshyn">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Elizabeth AR Garfinkle">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Makenna McCoy">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Seth Cummins">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Alejandro Otero Bravo">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Peter Chang">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Gregory L Wheeler">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="David M Gordon">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Benjamin J Kelly">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Caitlyn Evanovich">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Samara L Potter">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Elizabeth A Varga">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Diana P Rodriguez">
<meta name="citation_author_institution" content="Department of Radiology, Nationwide Children’s Hospital, The Ohio State University College of Medicine, Columbus, OH USA">
<meta name="citation_author" content="Richard K Wilson">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author_institution" content="Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA">
<meta name="citation_author" content="Catherine E Cottrell">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author_institution" content="Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA">
<meta name="citation_author_institution" content="Department of Pathology, The Ohio State University College of Medicine, Columbus, OH USA">
<meta name="citation_author" content="Ralph Salloum">
<meta name="citation_author_institution" content="Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA">
<meta name="citation_author_institution" content="Pediatric Brain Tumor Program, Division of Hematology, Oncology, and Bone Marrow Transplant, Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Sara Khan">
<meta name="citation_author_institution" content="Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA">
<meta name="citation_author_institution" content="Pediatric Brain Tumor Program, Division of Hematology, Oncology, and Bone Marrow Transplant, Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Maryam Fouladi">
<meta name="citation_author_institution" content="Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA">
<meta name="citation_author_institution" content="Pediatric Brain Tumor Program, Division of Hematology, Oncology, and Bone Marrow Transplant, Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Elaine R Mardis">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author_institution" content="Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA">
<meta name="citation_author_institution" content="Department of Neurosurgery, The Ohio State University College of Medicine, Columbus, OH USA">
<meta name="citation_author" content="Margot A Lazow">
<meta name="citation_author_institution" content="Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA">
<meta name="citation_author_institution" content="Pediatric Brain Tumor Program, Division of Hematology, Oncology, and Bone Marrow Transplant, Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author" content="Katherine E Miller">
<meta name="citation_author_institution" content="Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA">
<meta name="citation_author_institution" content="Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA">
<meta name="citation_publication_date" content="2025 Aug 23">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="1365">
<meta name="citation_doi" content="10.1186/s12885-025-14711-x">
<meta name="citation_pmid" content="40849662">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374395/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374395/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374395/pdf/12885_2025_Article_14711.pdf">
<meta name="description" content="Liquid biopsy assays using cerebrospinal fluid (CSF) can revolutionize care for children with central nervous system (CNS) tumors by enabling precise monitoring of therapeutic responses and detecting recurrence or measurable residual disease (MRD). ...">
<meta name="og:title" content="MRD4U: A path to development for personalized liquid biopsy for children with central nervous system tumors">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Liquid biopsy assays using cerebrospinal fluid (CSF) can revolutionize care for children with central nervous system (CNS) tumors by enabling precise monitoring of therapeutic responses and detecting recurrence or measurable residual disease (MRD). ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374395/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12374395">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12885-025-14711-x"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12885_2025_Article_14711.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374395%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12374395/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12374395/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374395/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmccanc.png" alt="BMC Cancer logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Cancer" title="Link to BMC Cancer" shape="default" href="https://doi.org/10.1186/s12885-025-14711-x" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Cancer</button></div>. 2025 Aug 23;25:1365. doi: <a href="https://doi.org/10.1186/s12885-025-14711-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12885-025-14711-x</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Cancer%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Cancer%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Cancer%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Cancer%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>MRD4U: A path to development for personalized liquid biopsy for children with central nervous system tumors</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Miller%20AR%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Anthony R Miller</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Anthony R Miller</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Miller%20AR%22%5BAuthor%5D" class="usa-link"><span class="name western">Anthony R Miller</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shah%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Tithi Shah</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Tithi Shah</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shah%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tithi Shah</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Strawser%20CN%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Corinne N Strawser</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Corinne N Strawser</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Strawser%20CN%22%5BAuthor%5D" class="usa-link"><span class="name western">Corinne N Strawser</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rivaldi%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Adithe Rivaldi</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Adithe Rivaldi</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rivaldi%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Adithe Rivaldi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilson%20SA%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Sarah A Wilson</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Sarah A Wilson</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilson%20SA%22%5BAuthor%5D" class="usa-link"><span class="name western">Sarah A Wilson</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhong%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Huachun Zhong</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Huachun Zhong</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhong%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huachun Zhong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lucyshyn%20JM%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jocelyn M Lucyshyn</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jocelyn M Lucyshyn</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lucyshyn%20JM%22%5BAuthor%5D" class="usa-link"><span class="name western">Jocelyn M Lucyshyn</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garfinkle%20EAR%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Elizabeth AR Garfinkle</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Elizabeth AR Garfinkle</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garfinkle%20EAR%22%5BAuthor%5D" class="usa-link"><span class="name western">Elizabeth AR Garfinkle</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McCoy%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Makenna McCoy</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Makenna McCoy</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McCoy%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Makenna McCoy</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cummins%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Seth Cummins</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Seth Cummins</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cummins%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Seth Cummins</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bravo%20AO%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Alejandro Otero Bravo</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Alejandro Otero Bravo</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bravo%20AO%22%5BAuthor%5D" class="usa-link"><span class="name western">Alejandro Otero Bravo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chang%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Peter Chang</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Peter Chang</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chang%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter Chang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wheeler%20GL%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Gregory L Wheeler</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Gregory L Wheeler</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wheeler%20GL%22%5BAuthor%5D" class="usa-link"><span class="name western">Gregory L Wheeler</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gordon%20DM%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">David M Gordon</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">David M Gordon</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gordon%20DM%22%5BAuthor%5D" class="usa-link"><span class="name western">David M Gordon</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kelly%20BJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Benjamin J Kelly</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Benjamin J Kelly</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kelly%20BJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Benjamin J Kelly</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Evanovich%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Caitlyn Evanovich</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Caitlyn Evanovich</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Evanovich%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Caitlyn Evanovich</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Potter%20SL%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Samara L Potter</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Samara L Potter</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Potter%20SL%22%5BAuthor%5D" class="usa-link"><span class="name western">Samara L Potter</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Varga%20EA%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Elizabeth A Varga</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Elizabeth A Varga</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Varga%20EA%22%5BAuthor%5D" class="usa-link"><span class="name western">Elizabeth A Varga</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20DP%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Diana P Rodriguez</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Diana P Rodriguez</span></h3>
<div class="p">
<sup>2</sup>Department of Radiology, Nationwide Children’s Hospital, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20DP%22%5BAuthor%5D" class="usa-link"><span class="name western">Diana P Rodriguez</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilson%20RK%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Richard K Wilson</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Richard K Wilson</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">
<sup>3</sup>Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilson%20RK%22%5BAuthor%5D" class="usa-link"><span class="name western">Richard K Wilson</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cottrell%20CE%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Catherine E Cottrell</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Catherine E Cottrell</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">
<sup>3</sup>Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div class="p">
<sup>4</sup>Department of Pathology, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cottrell%20CE%22%5BAuthor%5D" class="usa-link"><span class="name western">Catherine E Cottrell</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Salloum%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Ralph Salloum</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Ralph Salloum</span></h3>
<div class="p">
<sup>3</sup>Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div class="p">
<sup>5</sup>Pediatric Brain Tumor Program, Division of Hematology, Oncology, and Bone Marrow Transplant, Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Salloum%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ralph Salloum</span></a>
</div>
</div>
<sup>3,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khan%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id23"><span class="name western">Sara Khan</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Sara Khan</span></h3>
<div class="p">
<sup>3</sup>Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div class="p">
<sup>5</sup>Pediatric Brain Tumor Program, Division of Hematology, Oncology, and Bone Marrow Transplant, Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khan%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sara Khan</span></a>
</div>
</div>
<sup>3,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fouladi%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id24"><span class="name western">Maryam Fouladi</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Maryam Fouladi</span></h3>
<div class="p">
<sup>3</sup>Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div class="p">
<sup>5</sup>Pediatric Brain Tumor Program, Division of Hematology, Oncology, and Bone Marrow Transplant, Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fouladi%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maryam Fouladi</span></a>
</div>
</div>
<sup>3,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mardis%20ER%22%5BAuthor%5D" class="usa-link" aria-describedby="id25"><span class="name western">Elaine R Mardis</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Elaine R Mardis</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">
<sup>3</sup>Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div class="p">
<sup>6</sup>Department of Neurosurgery, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mardis%20ER%22%5BAuthor%5D" class="usa-link"><span class="name western">Elaine R Mardis</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lazow%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id26"><span class="name western">Margot A Lazow</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Margot A Lazow</span></h3>
<div class="p">
<sup>3</sup>Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div class="p">
<sup>5</sup>Pediatric Brain Tumor Program, Division of Hematology, Oncology, and Bone Marrow Transplant, Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lazow%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Margot A Lazow</span></a>
</div>
</div>
<sup>3,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Miller%20KE%22%5BAuthor%5D" class="usa-link" aria-describedby="id27"><span class="name western">Katherine E Miller</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Katherine E Miller</span></h3>
<div class="p">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div class="p">
<sup>3</sup>Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Miller%20KE%22%5BAuthor%5D" class="usa-link"><span class="name western">Katherine E Miller</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Steve and Cindy Rasmussen Institute for Genomic Medicine, Abigail Wexner Research Institute at Nationwide Children’s Hospital, Columbus, OH USA </div>
<div id="Aff2">
<sup>2</sup>Department of Radiology, Nationwide Children’s Hospital, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div id="Aff3">
<sup>3</sup>Department of Pediatrics, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div id="Aff4">
<sup>4</sup>Department of Pathology, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div id="Aff5">
<sup>5</sup>Pediatric Brain Tumor Program, Division of Hematology, Oncology, and Bone Marrow Transplant, Nationwide Children’s Hospital, Columbus, OH USA </div>
<div id="Aff6">
<sup>6</sup>Department of Neurosurgery, The Ohio State University College of Medicine, Columbus, OH USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 23; Accepted 2025 Jul 18; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12374395  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40849662/" class="usa-link">40849662</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Liquid biopsy assays using cerebrospinal fluid (CSF) can revolutionize care for children with central nervous system (CNS) tumors by enabling precise monitoring of therapeutic responses and detecting recurrence or measurable residual disease (MRD). These assays can detect cell-free, circulating tumor DNA (ctDNA) via somatic alterations, though accurately measuring low-abundance ctDNA in CSF is challenging.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">Our research focused on the optimization of next-generation sequencing library preparation from cell-free DNA (cfDNA), evaluating four commercial kits to address the low nucleic acid yield in CSF-derived cfDNA. The selected kit minimized false positives and detected somatic variants at 5% variant allele frequency using 0.1 ng input of synthetic cfDNA, suitable for low-volume CSF samples.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">We then applied our optimized workflow to six children with CNS tumors using a personalized hybrid-capture sequencing strategy (“MRD4U”), in which individualized panels were designed based on each patient’s tumor sequencing. Using MRD4U, we identified ctDNA in two samples, even though neither patient had radiographic or clinical evidence of disease at the time of liquid biopsy. Notably, one ctDNA-positive patient developed radiographic recurrence four months later, demonstrating the assay’s potential to detect molecular relapse ahead of conventional clinical measures.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par4">These findings demonstrate applicability of our personalized MRD4U assay in early detection of disease recurrence. Unlike non-targeted or tumor-agnostic CSF liquid biopsy approaches, MRD4U leverages patient-specific genomic information to enable sensitive, tumor-informed monitoring that can be deployed across a wide range of pediatric CNS tumors. Our approach is broadly applicable to any tumor type with existing genomic data, enabling ctDNA detection across diverse diagnoses. Ultimately, this strategy may inform clinical decision-making and enable earlier therapeutic intervention.</p></section><section id="sec5"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12885-025-14711-x.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Pediatric central nervous system tumors, Liquid biopsy, Cell-free DNA, Personalized medicine, Cancer genomics</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par32">In recent years liquid biopsy-based testing, particularly utilizing next-generation sequencing (NGS) of cell-free DNA (cfDNA) from plasma to identify circulating tumor DNA (ctDNA), has gained traction in research studies and clinical trials [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. cfDNA consists of DNA fragments circulating freely in bodily fluids like blood, urine, or cerebrospinal fluid (CSF), released through processes such as cell death [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. A specific subset of cfDNA originating from tumor cells is termed ctDNA, the detection of which allows for non-invasive monitoring of disease status. The appeal of liquid biopsy lies in its advantages over traditional tissue biopsies: it is less invasive, more cost-effective, and more sensitive than traditional imaging techniques [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Initially, the use of liquid biopsy focused on treatment decisions for various cancers including non-small cell lung, prostate, ovarian, and breast cancer, enabling tailored treatments based on specific genetic alterations detected via ctDNA analysis [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. Historically, most liquid biopsy studies have focused on common diagnoses in adults and have used blood as the sole source of biofluid [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Furthermore, in adult studies, liquid biopsy assays are designed to identify ctDNA by leveraging the fact that many types of adult tumors share commonly mutated genes. For example, in colorectal cancer ~ 80% of tumors have a mutation in the gene <em>APC</em> and in acute myeloid leukemia, &gt; 75% of tumors have mutations in one of three genes: <em>FLT3</em>,<em> DNMT3A</em>, or <em>NPM1</em> [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Therefore, assay designs for detecting mutations in ctDNA in common adult cancers are usually straightforward, employing targeted sequencing panels capable of identifying mutations across specific recurring gene subsets.</p>
<p id="Par33">While liquid biopsy has revolutionized cancer screening, diagnosis, therapy selection, and monitoring response to treatment via detection of minimal or measurable residual disease (MRD) in certain adult tumors, its application specifically in pediatric central nervous system (CNS) tumors presents intriguing challenges. For example, children with CNS tumors typically present with an intact blood brain barrier which prevents tumor shedding into the periphery, precluding the use of blood-based liquid biopsies [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Rather, CSF liquid biopsies are a more appealing route since many tumors anatomically abut ventricles which are reservoirs to produce CSF, a biofluid which can contain cfDNA depending on disease status [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. CSF presents additional challenges for this type of testing; most notably, the yield of extracted cfDNA is relatively low, making NGS library preparation challenging [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. In addition, pediatric CNS tumors exhibit a diverse genomic landscape, with few common gene mutations shared across the many disease entities [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. While some liquid biopsy studies in pediatric brain tumors have explored general approaches such as genome sequencing of CSF cfDNA to detect copy number alterations, these methods have limited sensitivity in detecting low levels of MRD [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Other brain tumor liquid biopsy studies have used CSF cfDNA to perform targeted sequencing with either a static panel or exome approach and have successfully demonstrated feasibility and clinical utility [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. However, static panels often lack generalizability due to the diverse mutational landscapes across different brain tumor types, while exome sequencing, although broadly applicable, demands extremely high sequencing depth and is thus costly. Our personalized approach, as described below, balances these factors by utilizing smaller, patient-specific panels, enabling sensitive ctDNA detection from CSF in a more targeted, cost-effective manner.</p>
<p id="Par34">Here, we describe the development of a liquid biopsy assay for pediatric CNS tumors by first identifying the most suitable methodology for maximizing assay sensitivity with low input cfDNA. In light of the recent recommendations set forth by the Association for Molecular Pathology and College of American Pathologists Liquid Biopsy Working Group for cfDNA assay validations, we report on several key pre-analytical considerations and assay performance metrics including a thorough evaluation of various NGS library preparation kits, the use of appropriate controls, assessment of decreasing levels of input cfDNA, and determination of sensitivity and limit of detection for our assay [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Our progression towards implementing this type of workflow in the pediatric CNS tumor patient population began with foundational laboratory tasks aimed at investigating cfDNA extraction from 44 CSF samples and assessing feasibility of detecting ctDNA in the cell-free extracts using a simple targeted amplicon sequencing approach. These preliminary investigations laid the groundwork for our subsequent endeavors. Focusing then on a unified library preparation protocol, we adopted a two-pronged approach to detect ctDNA in CSF in real-time from pediatric patients with CNS tumors as a surrogate for measuring MRD. First, we used a tumor-informed approach to design personalized hybridization capture panels (MRD4U; “MRD for you”) with known sequence variants identified previously through tumor exome sequencing. Second, low-pass genome sequencing (lpGS) allowed us to identify specific copy number variations (CNVs) known from prior tumor sequencing. Applying our approach to a cohort of six children with embryonal central nervous system tumors, we detected tumor-derived DNA in the CSF (“ctDNA+”) from two patients. Both ctDNA + children had negative findings on magnetic resonance imaging (MRI), a reported absence of malignant cells based on clinical cytological analysis at the time of our liquid biopsy assay, and notably– one of them developed radiographic disease recurrence months later, thus highlighting the increased sensitivity and early detection capabilities of our assay.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Specimen collection and processing</h3>
<p id="Par35">Written informed consent was obtained from the patient and/or legal guardians under a research study approved by the Institutional Review Board (IRB) at Nationwide Children’s Hospital (STUDY00003080). All research presented in this study was performed in accordance with relevant guidelines and regulations as set forth by the IRB at Nationwide Children’s Hospital. The volume of cerebrospinal fluid (CSF) collected was determined by the physician based on factors such as patient age, weight, and the quantity required for standard-of-care clinical laboratory analyses. CSF was collected into sterile vials or syringes via one of the following procedures: lumbar puncture, external ventricular drain (EVD), or ventriculoperitoneal (VP) shunt placement (Supplementary Table <a href="#MOESM2" class="usa-link">S2</a>). The CSF samples were first transported to a clinical laboratory for standard-of-care assessment including cytological analysis. Residual CSF samples were then transferred to appropriately sized conical tubes compatible with an Eppendorf 5810R centrifuge and centrifuged at 2000x g for 5 min at 4 °C. Following centrifugation, the CSF supernatant (CSF-SN) was carefully aspirated using a sterile serological pipette without disturbing any cell pellet present, transferred to 2.0 mL cryovials, and stored at −80 °C for varying lengths of time until needed for extraction (Table <a href="#Tab1" class="usa-link">1</a>). Prior to cell-free DNA (cfDNA) extraction from CSF-SN, the samples were thawed and transferred to a 1.5 mL Eppendorf tube and centrifuged at &gt; 12,000 g for 15 min. The CSF-SN was aspirated for cfDNA extraction. In addition, we also requested a blood sample to be taken at the same time as CSF collection, but only when there was a clinical need for a blood draw. For these samples, blood was collected in a purple top EDTA tube. Blood was combined into an appropriate-sized conical tube compatible with an Eppendorf 5810R and centrifuged at 1,200 x g for 10 min at 15–20℃. Plasma was separated from red cells using a sterile serological pipette and transferred to a labeled 15 mL or 50 mL conical tube. The conical tube containing plasma was re-centrifuged at maximum speed for 15 min at 4℃. After centrifugation, any visible fat layer on the surface of the plasma was removed using a sterile wooden stick or serological pipette. The plasma was aliquoted into a 2.0 mL cryovial, with 0.5-1 mL of plasma, and stored at −80℃. Both CSF-SN and plasma were used for cfDNA extraction using Quick-cfDNA/cfRNA Serum and Plasma kit (Zymo Research; catalog No. R1072) following the manufacturer’s protocol with the following modification: for maximum recovery, the initial cfDNA eluate was dispensed back onto the same spin column filter followed by incubation for 2 min and a second centrifugation at 12,000 x g for 30 s. For the extraction process, 1 mL of AcroMetrix TM Multi-Analyte ctDNA Plasma Control (AcroMetrix, catalog No. 957563) was utilized as a positive control to ensure the effectiveness of the DNA extraction. cfDNA quantity and quality were evaluated with the Qubit 1X dsDNA HS assay (Invitrogen; catalog No. <a href="https://www.ncbi.nlm.nih.gov/protein/Q33231" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q33231</a>) and Bioanalyzer High Sensitivity DNA kit (Agilent; catalog No. 5067 − 4626). Based on the Qubit dsDNA HS assay specifications, the lower limit of detection is 0.1 ng. Samples with DNA concentrations below this threshold were recorded as 0.0 ng when calculating yield per mL of extracted CSF.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>The status of treatment during biofluid collection, the collection methodology employed, the volume of fluid subjected to extraction, and the resulting yield</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Biofluid characteristics</th>
<th align="left" colspan="1" rowspan="1">Biofluid sample count</th>
<th align="left" colspan="1" rowspan="1">Average no. of days between collection and extractions</th>
<th align="left" colspan="1" rowspan="1">Average input volume (mL) into extractions</th>
<th align="left" colspan="1" rowspan="1">Average of dsDNA yield (ng/mL)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>CSF supernatant</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>44</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>107</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>1.2</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>5.7</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> <em>Biofluid collected at diagnosis</em>
</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">191</td>
<td align="left" colspan="1" rowspan="1">1.2</td>
<td align="left" colspan="1" rowspan="1">24.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">   External ventricular drain</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">289</td>
<td align="center" colspan="1" rowspan="1">1.5</td>
<td align="center" colspan="1" rowspan="1">55.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">   Lumbar puncture</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">158</td>
<td align="center" colspan="1" rowspan="1">0.7</td>
<td align="center" colspan="1" rowspan="1">6.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">   Ventriculoperitoneal shunt placement</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">28</td>
<td align="center" colspan="1" rowspan="1">2.0</td>
<td align="center" colspan="1" rowspan="1">2.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> <em>Biofluid collected at end of therapy</em>
</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="center" colspan="1" rowspan="1">1.8</td>
<td align="center" colspan="1" rowspan="1">0.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">   Lumbar puncture</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="center" colspan="1" rowspan="1">1.8</td>
<td align="center" colspan="1" rowspan="1">0.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> <em>Biofluid collected at mid-treatment: mid-chemo</em>
</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="center" colspan="1" rowspan="1">2.1</td>
<td align="center" colspan="1" rowspan="1">0.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">   Lumbar puncture</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="center" colspan="1" rowspan="1">2.1</td>
<td align="center" colspan="1" rowspan="1">0.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> <em>Biofluid collected at mid-treatment mid-chemo/radiation</em>:</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">102</td>
<td align="center" colspan="1" rowspan="1">1.0</td>
<td align="center" colspan="1" rowspan="1">1.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    Lumbar puncture</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">102</td>
<td align="center" colspan="1" rowspan="1">1.0</td>
<td align="center" colspan="1" rowspan="1">1.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> <em>Biofluid collected at post chemo</em>
</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="center" colspan="1" rowspan="1">1.2</td>
<td align="center" colspan="1" rowspan="1">1.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">   Lumbar puncture</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="center" colspan="1" rowspan="1">1.2</td>
<td align="center" colspan="1" rowspan="1">1.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> Biofluid collected at post radiation</em>
</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">44</td>
<td align="center" colspan="1" rowspan="1">1.2</td>
<td align="center" colspan="1" rowspan="1">1.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">   Lumbar puncture</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">44</td>
<td align="center" colspan="1" rowspan="1">1.2</td>
<td align="center" colspan="1" rowspan="1">1.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> <em>Biofluid collected at relapse</em>
</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">224</td>
<td align="center" colspan="1" rowspan="1">1.0</td>
<td align="center" colspan="1" rowspan="1">3.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    Lumbar puncture</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">224</td>
<td align="center" colspan="1" rowspan="1">1.0</td>
<td align="center" colspan="1" rowspan="1">3.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> <em>Treatment status not disclosed at collection</em>
</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">13</td>
<td align="center" colspan="1" rowspan="1">1.1</td>
<td align="center" colspan="1" rowspan="1">0.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">   Lumbar puncture</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">13</td>
<td align="center" colspan="1" rowspan="1">1.2</td>
<td align="center" colspan="1" rowspan="1">0.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">   Not specified</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">Not specified</td>
<td align="center" colspan="1" rowspan="1">1.0</td>
<td align="center" colspan="1" rowspan="1">0.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Plasma</strong>
</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="center" colspan="1" rowspan="1">4.2</td>
<td align="center" colspan="1" rowspan="1">5.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> <em> Biofluid collected at mid-treatment: mid-chemoBiofluid collected at mid-treatment: mid-chemo</em>
</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="center" colspan="1" rowspan="1">4.2</td>
<td align="center" colspan="1" rowspan="1">5.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">   Blood Draw, arm</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="center" colspan="1" rowspan="1">4.2</td>
<td align="center" colspan="1" rowspan="1">5.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">
<em>N</em> = 45</td>
<td align="left" colspan="1" rowspan="1">105</td>
<td align="center" colspan="1" rowspan="1">1.3</td>
<td align="center" colspan="1" rowspan="1">5.7</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p12"><p>Abbreviations: CSF - cerebrospinal fluid, dsDNA - double-stranded DNA, mL - milliliters, ng - nanograms</p></div></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Targeted DNA sequencing</h3>
<p id="Par37">Targeted amplicon deep sequencing (“Libricon”) followed our previously described protocol optimized for genomic DNA [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. To evaluate the effectiveness of this amplicon approach using cfDNA as input material, we conducted serial dilutions of the Pan-Cancer cfDNA reference standards v1 (Twist BioScience; catalog No. 104549). These dilutions varied the input amounts from 10 ng, 5 ng, 1 ng, 0.5 ng, and 0.25 ng for each of the reference standard variant allele frequencies (VAFs) controls: 5%, 2%, 1%, 0.5%, 0.25%, 0.1%, and 0%. Our initial assessment used primers designed to target the <em>BRAF</em> p.V600E variant (Supplementary Table <a href="#MOESM2" class="usa-link">S5</a>) known to be present in the Pan-Cancer cfDNA reference standards at the aforementioned VAFs. For deep targeted sequencing of patient-specific variants in CSF cfDNA we also, when available, used the patient’s tumor DNA as a positive control for detecting the targeted tumor variant. Additionally, we included human reference DNA GM24143 (Horizon Discovery; catalog No. GM24143) as a negative control. All controls used 10 ng of input, if available from extractions, into the targeted amplicon workflow. To identify patient-specific somatic and germline variants, we leveraged the patient’s prior tumor and comparator blood exome sequencing data. We selected a patient germline target that was a known pathogenic variant and/or absent from the gnomAD database [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]; for tumor variants, we chose the highest VAF coding or splicing variant based on the prior exome sequencing data and designed primers to amplify this region (Supplementary Table <a href="#MOESM2" class="usa-link">S5</a>). The entire CSF cfDNA extract was used as input into the Libricon workflow, splitting into two separate PCR reactions, one targeting the somatic tumor variant and the other targeting the germline variant. The final libraries were sequenced on an Illumina iSeq100 platform using paired-end 150 bp chemistry aiming for &gt; 10,000x coverage. The resulting data were analyzed following the procedure described in our ‘Variant Calling’ method section. Outliers for the targeted <em>BRAF</em> assessment were identified using the Grubbs’ test for outliers and subsequently removed. For the patient Libricon data displayed in Fig. <a href="#Fig2" class="usa-link">2</a>, only samples in which we successfully identified the patient germline variant in CSF cfDNA at &gt; 25% VAF were included for plotting and analysis, and all other results were considered “inconclusive.”</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374395_12885_2025_14711_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/03d2/12374395/683bf4e542e6/12885_2025_14711_Fig2_HTML.jpg" loading="lazy" id="MO2" height="442" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comprehensive analysis of ctDNA detection using Libricon sequencing. (A) Sensitivity and precision of the Libricon approach in detecting the <em>BRAF</em> p.V600E variant using Twist PanCancer cfDNA controls at VAFs of 0%, 0.10%, 0.25%, 0.50%, 1.00%, 2.00%, and 5.00%. Libricons were created with DNA inputs ranging from 0.25 ng to 10 ng, illustrating the detection method’s performance at various levels, and visually representing the observed VAFs. (B) Comparison of the VAF of the targeted variant from exome sequencing (displayed as a blue bar) with the VAF derived from tumor genomic DNA analyzed using the Libricon approach (purple bar). (C) Targeted variant VAF as determined by Libricon on tumor DNA and in CSF cfDNA, with ctDNA + in CSF cfDNA marked by an orange circle. (D) Displays longitudinal Libricon results for three patients, charting disease state at CSF collection against the observed VAF, with ctDNA + indicated by orange bars. Abbreviations: VAF - variant allele frequency, ES - exome sequencing, CSF - cerebrospinal fluid, cfDNA - cell-free DNA, ctDNA+ - positive circulating tumor DNA</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Comparative NGS assay development: workflow and custom hybridization probe panel design</h3>
<section id="Sec6"><h4 class="pmc_sec_title">Custom probe panel design for assay development using reference standards</h4>
<p id="Par38">A custom hybridization probe panel was designed using the Integrated DNA Technologies (IDT) xGen™ MRD Hybridization Panel portal. The panel was designed to target known variants (25 SNV and 24 indel variants representing 39 unique genes; (Supplementary Table <a href="#MOESM2" class="usa-link">S6</a>) present in the cfDNA Pan-Cancer Reference Standards. The panel design encompassed a per-probe length of 120 bp and employed a 3x tiling approach across the target regions. The resulting probe panel was used to evaluate the sensitivity of detecting known variants in the Twist cfDNA control using probe hybridization capture enrichment in four distinct library preparation workflows as described below.</p></section><section id="Sec7"><h4 class="pmc_sec_title">New England biolabs NEBNext<sup>®</sup> ultra™ II library Prep kit for Illumina<sup>®</sup>
</h4>
<p id="Par39">The NEBNext<sup>®</sup> Ultra™ II Library Prep Kit for Illumina<sup>®</sup> (catalog No. E7645L) was used to create pre-capture libraries using 10 ng of Twist cfDNA Pan-Cancer Reference Standards: 5% (Twist-5), 1% (Twist-1), and 0% (Twist-0) VAF. The pre-capture library construction process adhered to the recommended protocol, except for the following modifications: during ligation, the IDT xGen™ UDI-UMI adapter (catalog No. 10005903) was utilized at a concentration of 1.5 µM. Subsequently, the ligation products underwent purification using a 1.2X ratio of SPRI Select beads (Beckman Coulter; catalog No. <a href="https://www.ncbi.nlm.nih.gov/nuccore/B23319" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">B23319</a>) and were eluted in 20 µl of nuclease-free water. Nine cycles of PCR were performed, using 4 µl of ligation product in duplicate 50 µl total volume reactions for each sample. The PCR reactions used the NEBNext<sup>®</sup> Ultra™ II Q5 Master Mix and universal primers specific to the Illumina adapter sequences P5 and P7, following the recommended cycling conditions. After cycling, the duplicate PCR reactions were combined, purified with SPRI Select beads at a 1.2X ratio, and eluted in 22 µl of nuclease-free water. DNA quantity and quality were assessed using the Qubit 1X dsDNA HS assay (Invitrogen; catalog No. <a href="https://www.ncbi.nlm.nih.gov/protein/Q33230" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q33230</a>) and the Bioanalyzer High Sensitivity DNA kit (Agilent; catalog No. 5067 − 4626). Subsequently, approximately 300 ng of each library were pooled together for capture enrichment using the custom IDT xGen™ MRD panel (described in methods ‘custom probe panel design’) and the IDT xGen™ Hybridization and Wash Kit (catalog No. 1080577) with overnight probe hybridization. Post-capture PCR involved 14 cycles of amplification, utilizing 5 µl of post-capture product in duplicate 50 µl total volume reactions for each sample. The PCR reactions used the NEBNext<sup>®</sup> Ultra™ II Q5 Master Mix, following the recommended cycling conditions. After cycling, the duplicate PCR reactions were combined, purified with SPRI Select beads at a 1.2X ratio, and eluted in 22 µl of nuclease-free water. DNA quantity and quality were assessed using the Qubit 1X dsDNA HS assay and the Bioanalyzer High Sensitivity DNA kit. The resulting libraries were sequenced on an Illumina NovaSeq6000, generating paired-end 100-bp reads. The resulting fastq files were randomly downsampled to 45 M paired-end reads for each sample, with subsequent read alignment to human reference build GRCh38 (<a href="https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000001405.26/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000001405.26/</a>), and marked duplicate reads computed from BAM files using Picard.</p></section><section id="Sec8"><h4 class="pmc_sec_title">TwinStrand biosciences duplex library Prep kit</h4>
<p id="Par40">Next, the TwinStrand Biosciences Duplex Library Prep Kit (manual version 1.1, November 2021) was used to generate pre-captured libraries from 10 ng of Twist-0, Twist-1, and Twist-5 controls. The library construction process followed the manufacturer’s guidelines, with a few specific modifications. First, half of the pre-capture libraries, ranging from 400 to 500 ng each, were utilized as input for hybridization and capture. These captures were performed as single-reaction processes, employing the panel described in methods ‘custom probe panel design,’ using the TwinStrand hybridization and wash buffers. Following the recommended cycling conditions, the first post-capture PCR used 16 cycles of amplification with all the resulting PCR product then used as input for the second round of hybridization and capture. In this second step, the post-capture PCR was conducted with 6 cycles, again adhering to the kit’s recommended cycling parameters. To assess the quality and quantity of the final libraries, we used the Qubit 1X dsDNA HS assay (Invitrogen; catalog No. <a href="https://www.ncbi.nlm.nih.gov/protein/Q33230" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q33230</a>) and the Bioanalyzer High Sensitivity DNA kit (Agilent; catalog No. 5067 − 4626). The final libraries were sequenced on an Illumina NovaSeq6000, generating paired-end 100-bp reads. Analysis involved the use of the DNANexus platform, which required fastq files and a target bed file to perform the TwinStrand DuplexSeq FASTQ to VCF Parallel App (4.3.0) application. This application performs error corrected alignment and variant calls across multiple samples. The fastq files for each sample were randomly downsampled to 45 M paired-end reads prior to uploading to DNANexus. The bed file provided included the coordinates for the targeted Twist variants used in the design of the ‘custom probe panel.’ The output files include aligned BAM files to the human reference build GRCh38 (<a href="https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000001405.26/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000001405.26/</a>).</p></section><section id="Sec9"><h4 class="pmc_sec_title">Claret bioscience SRSLY<sup>®</sup> PicoPlus NGS library Prep kit</h4>
<p id="Par41">The single-stranded DNA library prep kit from Claret Bioscience, SRSLY<sup>®</sup> PicoPlus Kit (catalog No. CBS-K250B-96) with UDI-UMI primers (catalog No. CBS-UMI-96) was used to create pre-captured libraries from varying input amounts (10 ng, 5 ng, 1 ng, 0.5 ng, and 0.1 ng) of Twist-0, Twist-1, and Twist-5 controls. The library construction process adhered to the manufacturer’s guidelines, with a few specific modifications: during ligation, the 10 ng input samples used the Claret Biosciences NanoPlus SRSLY NGS Adapters (catalog No. CBS-K155B-96), whereas all other inputs were prepared using the standard PicoPlus SRSLY NGS Adapters. The number of PCR cycles performed pre- and post-capture varied based on the initial sample input amounts. Specifically, pre-capture libraries generated from 10 ng, 5 ng, and 1 ng of starting input material underwent 10 cycles of PCR. In contrast, pre-capture libraries generated from 0.5 ng starting input material underwent 12 cycles of amplification, and the 0.1 ng starting material underwent 14 cycles of amplification. PCR conditions followed the recommended cycling parameters. After amplification, the PCR reactions were purified using the specific single-sided SPRI Select workflow and eluted in 22 µl of nuclease-free water. The DNA quantity and quality were assessed using the Qubit 1X dsDNA HS assay (Invitrogen; catalog No. <a href="https://www.ncbi.nlm.nih.gov/protein/Q33230" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q33230</a>) and the Bioanalyzer High Sensitivity DNA kit (Agilent; catalog No. 5067 − 4626). For samples that had 10 ng of starting input material, approximately 500 ng of each pre-capture library were combined into a single pool for capture enrichment. In a separate reaction, 185 ng of each pre-captured library derived from samples starting with 5 ng and 1 ng of Twist cfDNA were pooled together. In a third reaction, 140 ng of each pre-captured library constructed from samples starting with 0.5 ng and 0.1 ng of Twist cfDNA were combined for capture enrichment. The capture enrichment process used the panel described in methods ‘custom probe panel design’ with the IDT xGen™ Hybridization and Wash Kit (catalog No. 1080577). Probe hybridization was performed overnight. The post-capture PCR was conducted with 14 cycles of amplification for the Twist cfDNA samples with starting inputs of 10 ng, 5 ng, and 1 ng of cfDNA and 13 cycles of amplification for the 0.5 ng and 0.1 ng inputs. Each sample underwent duplicate PCR reactions, with 5 µl of post-capture product in a total volume of 50 µl per reaction. The PCR reactions used the NEBNext<sup>®</sup> Ultra™ II Q5 Master Mix, following the recommended cycling conditions. For purification, the duplicate PCR reactions were combined and treated with SPRI Select at a 1.2X ratio. The elution was performed using 22 µl of nuclease-free water. To assess DNA quantity and quality, we employed the Qubit 1X dsDNA HS assay (Invitrogen) and the Bioanalyzer High Sensitivity DNA kit (Agilent). The resulting libraries were sequenced on an Illumina NovaSeq6000, generating paired-end 100-bp reads. The resulting fastq files were randomly downsampled to 45 M paired-end reads for each sample, with subsequent read alignment to human reference build GRCh38 (<a href="https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000001405.26/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000001405.26/</a>), and marked duplicate reads computed from BAM files using Picard.</p></section><section id="Sec10"><h4 class="pmc_sec_title">Integrated DNA technologies (IDT) xGen™ CfDNA &amp; FFPE DNA library Prep kit</h4>
<p id="Par42">The xGen™ cfDNA &amp; FFPE DNA Library Prep Kit (catalog No. 10010207) library prep kit from Integrated DNA Technologies (IDT) was used to create pre-capture libraries from varying input amounts (10 ng, 5 ng, 1 ng, 0.5 ng, and 0.1 ng) of the Twist-0, Twist-1, and Twist-5 controls. The library prep process adhered to the manufacturer’s guidelines, with a few specific modifications: the number of PCR cycles performed pre- and post-capture varied based on the initial sample input amounts. The number of pre-capture PCR cycles performed varied from 10 cycles (Twist cfDNA sample inputs of 5 ng and 1 ng), 11 cycles (Twist cfDNA sample input of 10 ng), and 14 cycles (Twist cfDNA sample input of 0.5 and 0.1 ng). The resulting purified PCR products were assessed using the Qubit 1X dsDNA HS assay (Invitrogen; catalog No. <a href="https://www.ncbi.nlm.nih.gov/protein/Q33230" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q33230</a>) and the Bioanalyzer High Sensitivity DNA kit (Agilent; catalog No. 5067 − 4626). For samples that had 10 ng of starting input material, approximately 500 ng of each pre-capture library were combined into a single pool for capture enrichment. In a separate reaction, 185 ng of each pre-captured library derived from samples starting with 5 ng and 1 ng of Twist cfDNA were pooled together. In a third reaction, 140 ng of each pre-captured library constructed from samples starting with 0.5 ng and 0.1 ng of Twist cfDNA were combined for capture enrichment. The capture enrichment process used the panel described in methods ‘custom probe panel design’ with the IDT xGen™ Hybridization and Wash Kit (catalog No. 1080577). Probe hybridization was performed overnight. The post-capture PCR was conducted with 14 cycles of amplification for the Twist cfDNA samples with starting inputs of 10 ng, 5 ng, and 1 ng of cfDNA and 11 cycles of amplification for the 0.5 ng and 0.1 ng inputs. Each sample underwent duplicate PCR reactions, with 5 µl of post-capture product in a total volume of 50 µl per reaction. The PCR reactions used the NEBNext<sup>®</sup> Ultra™ II Q5 Master Mix, following the recommended cycling conditions. For purification, the duplicate PCR reactions were combined and treated with SPRI Select at a 1.2X ratio. The elution was performed using 22 µl of nuclease-free water. To assess DNA quantity and quality, we employed the Qubit 1X dsDNA HS assay (Invitrogen) and the Bioanalyzer High Sensitivity DNA kit (Agilent). The resulting libraries were sequenced on an Illumina NovaSeq6000, generating paired-end 100-bp reads. All samples were downsampled to 45 M paired-end reads. Strand consensus reads using fixed in-line UMIs were produced and aligned to the human reference genome GRCh38(<a href="https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000001405.26/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000001405.26/</a>) using the fgbio best practices fastq to consensus pipeline (<a href="https://github.com/fulcrumgenomics/fgbio/blob/main/docs/best-practice-consensus-pipeline.md" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/fulcrumgenomics/fgbio/blob/main/docs/best-practice-consensus-pipeline.md</a>). Briefly, this used bwa mem with alt-aware alignment, then grouped reads by UMI, followed by consensus read extraction using fgbio (2.1.0, using --error-rate-post-umi = 30 --min-reads = 3). Consensus reads were then aligned again to the reference with alt-aware bwa mem, filtered, and sorted.</p></section></section><section id="Sec11"><h3 class="pmc_sec_title">Pilot study in pediatric brain tumor cohort</h3>
<p id="Par43">We enrolled six pediatric patients for MRD4U, disclosed here as; LB002 was an adult and therefore excluded from this manuscript. Our pilot study utilized a dual-pronged approach from patient CSF-cfDNA: (1) patient-specific hybridization capture panel approach (MRD4U; “MRD for you”) targeting multiple somatic and germline variants identified via prior tumor-normal testing and (2) low-pass genome sequencing (lpGS) to detect tumor-specific copy number variation (CNV). Patients enrolled on our study all had either clinical or research exome sequencing performed on their tumor previously at our institution, and those data was available to us as part of our liquid biopsy IRB-approved study. We have previously described our exome sequencing method along with variant calling and CNV detection workflows [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. CSF-cfDNA libraries were prepared (using the entirety of the extracted cfDNA) following the IDT workflow described above. The resulting library was then split, and &gt; = 125 ng was used for MRD4U capture and &gt; 10 ng for lpGS. <em>MRD4U</em>: Per patient, we created xGen™ MRD hybridization panels designed to include up to 500 probes per panel at low cost and a rapid turnaround time of five business days from when the order is placed (Integrated DNA Technologies). Each patient had a custom panel that included 10 germline targets (private to the patient and absent from gnomAD) and 20–100 personalized somatic variants, which were chosen based on the highest VAF if more than 100 variants were present in the tumor exome data. For all panels, we also included a static list of 26 targets either present in the Twist Pan-Cancer cfDNA reference standard controls and/or genes or variants reported to be commonly altered in pediatric brain tumors (Supplementary Table <a href="#MOESM2" class="usa-link">S8</a>) [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>, <a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. A tiling approach was used to design probes for all somatic and germline targets, with each target having three different probes covering the area of interest, to yield up to 500 total probes per panel (Supplementary Fig. <a href="#MOESM1" class="usa-link">S1</a>). A minimum of 125 ng of the amplified CSF-cfDNA IDT library was used for MRD4U capture along with (when available) 1–10 ng of patient plasma cfDNA and the following controls: Twist-5, Structural Multiplex cfDNA Reference Standard HD786 (Horizon Discovery; catalog No. HD786), and human reference DNA GM24143 (ref-gDNA). Pooling and hybridization capture was performed according to the manufacturer’s protocol for xGen Hybridization and Wash (Integrated DNA Technologies; catalog No. 1080584) (Supplementary Table <a href="#MOESM2" class="usa-link">S9</a>). Final libraries were sequenced on Illumina NovaSeq 6000 using a run recipe of 2 × 100 (Read1 = 109; i7 = 8; i5 = 8; Read2 = 109). Variant calling was performed as described below. <em>lpGS</em>: At least 10 ng of the CSF-cfDNA final IDT library (pre-capture) was used for lpGS and sequenced on NovaSeq 6000 using a run recipe of 2 × 100 (Read1 = 109; i7 = 8; i5 = 8; Read2 = 109) aiming for 5x coverage. CNVs were called and plotted by comparing to a panel of normals (PON) using our previously described approach [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. We generated the PON using plasma cfDNA from ten anonymous blood donors, using 1 ng of cfDNA prepared with the IDT xGen cfDNA &amp; FFPE DNA Library Prep Kit as described above. Tumor-specific CNVs, known from the patient’s prior exome sequencing, were then searched for in the CSF-cfDNA lpGS data.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Variant calling and copy number variation detection</h3>
<p id="Par44">Variant calling and VAF calculations for both the Libricon workflow and the hybridization capture workflows were generated utilizing SAMtools mpileup and mpileup2cns VarScan (version 2.3.4) commands to generate read counts for the variants of interest from raw, UMI-deduplicated BAM files. VarScan2 and GATK were used to assess CNVs and loss of heterozygosity across all chromosomes [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. In the Libricon study, only samples in which we successfully identified the patient germline variant in CSF cfDNA at &gt; 25% VAF were considered for positive or negative ctDNA detection; all others were considered to be “inconclusive” based on our assay. In the MRD4U pilot study, variant calling was refined using processed controls concurrently run with patient samples. These controls included the Twist Pan-Cancer cfDNA at a 5% VAF (Twist-5), the Horizon Discovery Structural Multiplex Reference Standard (HD786), and the commercial genomic DNA reference standard GM24143 (ref-gDNA). These controls served to identify background noise and potential sequencing or alignment artifacts. Variants identified in any of the control samples were excluded from the patient data to ensure that only non-artifactual patient-specific alterations were considered. To classify a sample as ctDNA+, it must have exhibited alternative read support of at least three consensus reads for at least one variant, a cutoff based on the low input testing with 0.1 ng of the Twist-0 standard in the IDT workflow described above.</p></section></section><section id="Sec13"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Pediatric and AYA patient sample collection</h3>
<p id="Par45">Our research focused on analyzing 45 biofluid samples (44 CSF and 1 plasma) from 25 pediatric, adolescent and young adult (AYA) patients, collected during distinct phases of their clinical treatment for CNS tumors (Supplementary Table <a href="#MOESM3" class="usa-link">S3</a>). The samples served diverse purposes within our study: some were for refining extraction methods, while others were instrumental in optimizing molecular workflows to detect ctDNA through targeted amplicon sequencing (referred to as “Libricon”) and deep sequencing techniques; referred to as “Vignette_#” throughout (Supplementary Table <a href="#MOESM4" class="usa-link">S4</a><strong>)</strong>. Additionally, a select group of samples (referred to as “LB00#”) were incorporated into a pilot study to detect ctDNA as a surrogate for measuring the presence of MRD, called MRD4U (“MRD for you”) (Table <a href="#Tab2" class="usa-link">2</a>). This innovative approach combined tumor-informed targeted capture enrichment with lpGS to enhance the measurement of ctDNA, offering a comprehensive, multimodal assessment of MRD (Fig. <a href="#Fig1" class="usa-link">1</a>A-B).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Matrix plots for library kit bakeoff</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374395_12885_2025_14711_Tab2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/03d2/12374395/59a4a4efe5b0/12885_2025_14711_Tab2_HTML.jpg" loading="lazy" id="MO3" height="223" width="784" alt="graphic file with name 12885_2025_14711_Tab2_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p24"><p>Abbreviations: a closed black square (■) represents capability, while an open square (□) denotes absence, UMIs - unique molecular identifiers, gDNA - genomic DNA, ng - nanogram, NGS - next-generation sequencing.</p></div></div></section><figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374395_12885_2025_14711_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/03d2/12374395/a59a5bea61d2/12885_2025_14711_Fig1_HTML.jpg" loading="lazy" id="MO1" height="1016" width="757" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Illustration of MRD4U workflow for detecting circulating tumor DNA and liquid biopsy sample utilization. (A) This diagram illustrates the analysis of CSF for ctDNA in pediatric and AYA patients as part of the MRD4U pilot study. Initially, the patient’s tumor is profiled using exome sequencing to identify tumor-specific variants. These variants guide the creation of a personalized probe-based capture panel, named MRD4U, used to analyze cfDNA extracted from the patient’s CSF-SN. Concurrently, part of the CSF-derived cfDNA library is subjected to lpGS to detect CNVs. (B) Sankey plot displaying how biofluid samples from pediatric and AYA patients were allocated and analyzed across different workflows. Relationships between the blocks are represented by the flowing lines, with their width corresponding to the number of patients. Abbreviations: CSF - cerebrospinal fluid, ctDNA - circulating tumor DNA, AYA - adolescent and young adult, cfDNA - cell-free DNA, CSF-SN - CSF supernatant, lpGS– low-pass genome sequencing, CNVs - copy number variations</p></figcaption></figure><p id="Par48">The diagnoses at the time of sample collection included medulloblastoma (<em>n</em> = 34 biofluid samples obtained), ependymoma (<em>n</em> = 4), atypical teratoid rhabdoid tumor (ATRT; <em>n</em> = 2), atypical choroid plexus papilloma (<em>n</em> = 1), brainstem lesion (<em>n</em> = 1), mature teratoma (<em>n</em> = 1), and pineoblastoma (<em>n</em> = 1) (Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>). Samples were obtained at diagnosis– 10 to 14 days post initial surgery– (<em>n</em> = 8), at relapse (<em>n</em> = 8), during treatment (<em>n</em> = 8), or post radiation (CSI; <em>n</em> = 8), with additional samples collected at end of therapy (<em>n</em> = 5) or post-chemotherapy (<em>n</em> = 3). A limited number of biofluid samples were without disclosed treatment status during time of collection (<em>n</em> = 5). Most patients had only a single biofluid sample collected (<em>n</em> = 17), with three patients providing two samples and five patients having three or more samples for assessment. Out of the 45 liquid biopsy samples, 44 were CSF, primarily collected through lumbar puncture (<em>n</em> = 38) (Table <a href="#Tab1" class="usa-link">1</a>). The remaining CSF samples were obtained through an external ventricular drain (<em>n</em> = 3), during ventriculoperitoneal shunt replacement (<em>n</em> = 1), or through an unspecified collection method (<em>n</em> = 2).</p>
<p id="Par49">In most cases, CSF was first sent clinically for cytology, cell count, and/or culture analyses; remaining additional CSF was used for ctDNA analyses. The volume of CSF extracted for ctDNA analyses varied between 0.3 and 3.4 ml (average = 1.2 ml), with most samples (<em>n</em> = 29) being ≤ 1.0 ml (Table <a href="#Tab1" class="usa-link">1</a> and Supplementary Table <a href="#MOESM2" class="usa-link">S2</a>). The CSF collected during ventriculoperitoneal shunt replacement had the largest volume at 2 mL, followed by CSF obtained via external ventricular drain (average = 1.5 mL), lumbar puncture (average = 1.2 mL), and CSF collected through unspecified means (average = 1.0 mL). The collected CSF was stored at −80 °C until extraction, with the duration between collection and extraction averaging 107 days and varying from a minimum of 6 days to a maximum of 806 days. In contrast, the single plasma sample isolated from blood exhibited a shorter duration of 16 days between plasma separation and extraction.</p>
<p id="Par50">The extracted DNA yield varied, with 20 of 44 CSF samples falling below our limit of detection (see methods). Among quantifiable samples, the highest DNA concentration was observed in CSF collected via external ventricular drain (average 55.9 ng/mL), followed by shunt replacement (2.7 ng/mL), lumbar puncture (2.0 ng/mL), and unspecified methods (0.9 ng/mL) (Table <a href="#Tab1" class="usa-link">1</a> and Supplementary Table <a href="#MOESM2" class="usa-link">S2</a>). We examined whether DNA yield was associated with treatment status at the time of CSF collection. We specifically focused on CSF collected through lumbar puncture, as this method of collection was consistent across all defined treatment statuses, and lumbar punctures are routinely performed for standard of care disease evaluations for many CNS tumors, especially CNS embryonal tumors. CSF collected at diagnosis yielded the highest average DNA concentration (6.8 ng/mL), followed by samples at relapse (3.5 ng/mL), post-radiation (1.3 ng/mL), mid-treatment (1.2 ng/mL), end of therapy (0.4 ng/mL), and mid-chemotherapy (0.0 ng/mL) (Supplementary Fig. <a href="#MOESM2" class="usa-link">S2</a> and Table <a href="#Tab1" class="usa-link">1</a>).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Targeted amplicon sequencing to detect ctdna in CSF</h3>
<p id="Par51">There are inherent challenges associated with evaluating MRD using circulating cfDNA, including degradation and low yield of the cfDNA extract. Furthermore, ctDNA is often at low levels warranting a deep and/or targeted sequencing approach to provide higher coverage at the regions of interest. As such, our initial strategy involved generating targeted amplicons (“Libricons”) followed by deep sequencing to assess for feasibility of detecting ctDNA from CSF. We have used this method previously to successfully confirm low-level somatic mosaic variants (as low as 0.1% variant allele frequencies (VAF)) in resected brain tissue from children with epilepsy [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. The method’s prior utility, however, relied on 10 ng of genomic DNA as input. To assess the Libricon approach effectiveness with cfDNA, we conducted an evaluation using serial dilutions of the Twist Pan-Cancer cfDNA reference standards, a control containing known sequence variants at specific VAFs. Specifically, we designed primers to target the <em>BRAF</em> p. V600E single nucleotide variant (SNV) within the Twist control, varying input amounts from 0.25 to 10 ng and exploring a range of VAFs from 0 to 5%. This approach allowed us to define the lower limit of detection (LOD) and sensitivity for our Libricon approach. Average coverage for the <em>BRAF</em> Libricons was ~ 27,500x (Range: 282x − 141,600x). Our findings indicate a strong concordance between the expected and observed <em>BRAF</em> p.V600E VAFs for inputs of 10 and 5 ng, demonstrated by high R [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>] values of 0.96 and 0.93, respectively (Fig. <a href="#Fig2" class="usa-link">2</a>A). When the input was reduced to 1 and 0.5 ng, we noted a decrease in concordance, marked by increased variation between expected and actual VAFs, with an R [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>] value of 0.65. At the lowest input level of 0.25 ng, the discrepancy between expected and observed VAFs further increased, especially at the lower expected VAFs, evidenced by an R [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>] value of 0.59. When using the Twist Pan Cancer cfDNA control with a 5% VAF, despite the observed VAF ranging between 2 and 6%, the <em>BRAF</em> p.V600E was detected across all input levels (0.25 ng − 10 ng), including replicates. Below 5%, the LOD for 1 ng input was 2% VAF and for inputs of 5 and 10 ng, the LOD was 0.25% VAF.</p>
<p id="Par52">Next, we applied our optimized cfDNA Libricon method to a subset of our patient cohort (<em>n</em> = 24 samples), for cases from whom residual tumor genomic DNA (gDNA) was also available to generate Libricons in parallel <strong>(</strong>Fig. <a href="#Fig1" class="usa-link">1</a>B and Supplementary Table <a href="#MOESM4" class="usa-link">S4</a>). First, we demonstrated that Libricon sequencing is concordant with exome sequencing in detecting similar VAFs, by initially identifying the somatic variant with the highest VAF from prior exome sequencing of tumor DNA (refer to Methods section) and subsequently generating Libricons, targeting this specific variant from tumor gDNA. We achieved sequencing coverage exceeding 10,000x for these Libricons, and our analysis demonstrated that the VAFs detected by the Libricon method closely matched those identified by exome sequencing (Fig. <a href="#Fig2" class="usa-link">2</a>B-C). Using this same subset of patients, we then targeted the same somatic variant in the CSF cfDNA extract. The range of tumor gDNA VAFs for targeted variants was 35–90% and of the 24 CSF cfDNA samples evaluated, 5 were found to be positive for ctDNA, with a lower LOD of 0.37% VAF detected in cfDNA (Figs. <a href="#Fig1" class="usa-link">1</a>B, <a href="#Fig2" class="usa-link">2</a>C and Supplementary Table <a href="#MOESM4" class="usa-link">S4</a>). Our initial patient cohort included longitudinal collection of CSF for three children, all of whom were diagnosed with medulloblastoma. We performed Libricon sequencing and subsequently plotted longitudinal analyses for these patients to track any changes in ctDNA/MRD status throughout their course of treatment <strong>(</strong>Fig. <a href="#Fig2" class="usa-link">2</a>D). For Vignette_14, ctDNA was not detected at any of the sampled timepoints (negative circulating tumor DNA; ctDNA-). In contrast, Vignette_17 was positive for circulating tumor DNA (ctDNA+) at the time of diagnosis but subsequently ctDNA- following radiation treatment, suggesting a potentially favorable response to therapy. Vignette_18 presented a more complex pattern, initially testing ctDNA + post-radiation, then negative at the later time of clinical relapse, indicating fluctuating levels of ctDNA that may either reflect dynamic changes in tumor behavior or treatment response or, rather, a limited sensitivity of the Libricon approach given that only one somatic variant is targeted. While our Libricon approach has proven effective for detecting circulating tumor DNA from low yields of CSF cfDNA, its scalability to target multiple loci is inherently limited. This limitation arises from the need to optimize multiplexed primer pairs, a process further complicated by the necessity to tailor primer pairs on a per-patient basis based on unique genomic profiles. To enhance the comprehensiveness of our approach, we next sought to optimize a sample preparation and hybridization capture panel strategy to profile a broader array of known tumor variants in our cohort.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Comparative analysis of Commercially-Available NGS sample Preparation kits for CfDNA</h3>
<p id="Par53">This study aimed to compare the effectiveness of four commercial NGS library preparation kits designed for either low input or highly fragmented DNA—key traits of cfDNA—focusing on their capabilities for single-stranded DNA processing and error correction via unique molecular identifiers (UMIs) (Table <a href="#Tab3" class="usa-link">3</a>). Using Twist Pan-Cancer cfDNA reference standards, we evaluated kits from New England Biolabs (NEBNext Ultra II, abbreviated as NEB), Claret Bioscience (SRSLY PicoPlus, abbreviated as Claret), TwinStrand Biosciences (Duplex Library Prep, abbreviated as TwinStrand), and Integrated DNA Technologies (cfDNA &amp; FFPE DNA Library Prep, abbreviated as IDT) based on the quality of sequencing data. These quality metrics included the precision of identifying targeted variants and the accuracy of VAFs from libraries prepared with 10 ng of Twist cfDNA control samples at VAFs of 5% (Twist-5), 1% (Twist-1), and 0% (Twist-0), targeting 25 single nucleotide variants (SNVs) and 24 insertion/deletion (INDEL) events across 39 genes using a probe-based capture enrichment strategy.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>MRD4U summary of findings</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Sample<br>ID</th>
<th align="left" rowspan="2" colspan="1">Sex</th>
<th align="left" rowspan="2" colspan="1">Diagnosis</th>
<th align="left" colspan="2" rowspan="1">Age (years)</th>
<th align="left" rowspan="2" colspan="1">Timepoint of CSF analyzed</th>
<th align="left" colspan="6" rowspan="1">MRD4U</th>
<th align="left" rowspan="2" colspan="1">Clinical Findings* at time of MRD4U</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">At Diagnosis</th>
<th align="left" colspan="1" rowspan="1">At CSF Collection</th>
<th align="left" colspan="1" rowspan="1">Somatic Variants Detected<br>Number<br>(Average VAF)</th>
<th align="left" colspan="1" rowspan="1">Somatic Target Duplex Depth<br>(Average)</th>
<th align="left" colspan="1" rowspan="1">Germline Variants Detected<br>Number (Average VAF)</th>
<th align="left" colspan="1" rowspan="1">Germline Target Duplex Depth<br>(Average)</th>
<th align="left" colspan="1" rowspan="1">Tumor CNVs Detected in CSF cfDNA lpGS?</th>
<th align="left" colspan="1" rowspan="1">Finding</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">LB001</td>
<td align="left" colspan="1" rowspan="1">M</td>
<td align="left" colspan="1" rowspan="1">Medulloblastoma, WNT</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="center" colspan="1" rowspan="1">17.6</td>
<td align="left" colspan="1" rowspan="1">Mid-way through chemo</td>
<td align="left" colspan="1" rowspan="1">0/100 (N/A)</td>
<td align="left" colspan="1" rowspan="1">34.0x</td>
<td align="center" colspan="1" rowspan="1">10/10 (53.0%)</td>
<td align="left" colspan="1" rowspan="1">33.7x</td>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">ctDNA-</td>
<td align="left" colspan="1" rowspan="1">No evidence of disease</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LB003</td>
<td align="left" colspan="1" rowspan="1">F</td>
<td align="left" colspan="1" rowspan="1">Medulloblastoma, SHH; Gorlin syndrome</td>
<td align="left" colspan="1" rowspan="1">0.8</td>
<td align="center" colspan="1" rowspan="1">1.8</td>
<td align="left" colspan="1" rowspan="1">End of therapy</td>
<td align="left" colspan="1" rowspan="1">0/100 (N/A)</td>
<td align="left" colspan="1" rowspan="1">13.5x</td>
<td align="center" colspan="1" rowspan="1">8/10 (42.3%)</td>
<td align="left" colspan="1" rowspan="1">37.6x</td>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">ctDNA-</td>
<td align="left" colspan="1" rowspan="1">No evidence of disease</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LB004</td>
<td align="left" colspan="1" rowspan="1">F</td>
<td align="left" colspan="1" rowspan="1">Medulloblastoma, SHH activated and TP53 wildtype</td>
<td align="left" colspan="1" rowspan="1">1.5</td>
<td align="center" colspan="1" rowspan="1">1.8</td>
<td align="left" colspan="1" rowspan="1">Mid-way through chemo</td>
<td align="left" colspan="1" rowspan="1">0/20 (N/A)</td>
<td align="left" colspan="1" rowspan="1">4.9x</td>
<td align="center" colspan="1" rowspan="1">3/10 (77.0%)</td>
<td align="left" colspan="1" rowspan="1">4.8x</td>
<td align="left" colspan="1" rowspan="1">Inconclusive</td>
<td align="left" colspan="1" rowspan="1">ctDNA-</td>
<td align="left" colspan="1" rowspan="1">No evidence of disease</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LB005</td>
<td align="left" colspan="1" rowspan="1">F</td>
<td align="left" colspan="1" rowspan="1">Medulloblastoma, non-WNT/non-SHH (Group 4)</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="center" colspan="1" rowspan="1">14.4</td>
<td align="left" colspan="1" rowspan="1">End of therapy</td>
<td align="left" colspan="1" rowspan="1">11/50 (37.2%)</td>
<td align="left" colspan="1" rowspan="1">43.3x</td>
<td align="center" colspan="1" rowspan="1">9/10 (51.2%)</td>
<td align="left" colspan="1" rowspan="1">63.7x</td>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">ctDNA+</td>
<td align="left" colspan="1" rowspan="1">No evidence of disease</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LB006</td>
<td align="left" colspan="1" rowspan="1">M</td>
<td align="left" colspan="1" rowspan="1">Medulloblastoma, SHH and TP53 mutant; relapsed</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">7.2</td>
<td align="left" colspan="1" rowspan="1">End of therapy</td>
<td align="left" colspan="1" rowspan="1">0/100 (N/A)</td>
<td align="left" colspan="1" rowspan="1">7.3x</td>
<td align="center" colspan="1" rowspan="1">7/10 (53.6%)</td>
<td align="left" colspan="1" rowspan="1">14.7x</td>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">ctDNA-</td>
<td align="left" colspan="1" rowspan="1">No evidence of disease</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LB007</td>
<td align="left" colspan="1" rowspan="1">F</td>
<td align="left" colspan="1" rowspan="1">ATRT</td>
<td align="left" colspan="1" rowspan="1">19.3</td>
<td align="center" colspan="1" rowspan="1">19.6</td>
<td align="left" colspan="1" rowspan="1">Post radiation</td>
<td align="left" colspan="1" rowspan="1">52/100 (10.9%)</td>
<td align="left" colspan="1" rowspan="1">173.1x</td>
<td align="center" colspan="1" rowspan="1">9/10 (43.9%)</td>
<td align="left" colspan="1" rowspan="1">122.7x</td>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">ctDNA+</td>
<td align="left" colspan="1" rowspan="1">No evidence of disease</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p36"><p>Abbreviations: M - male, F - female, cfDNA– cell-free DNA, CSF - cerebrospinal fluid, lpGS– low pass genome sequencing, VAF - variant allele frequency, N/A - not applicable, ctDNA+ - positive circulating tumor DNA, ctDNA- - negative circulating tumor DNA. *Clinical findings as reported via MRI and CSF cytology</p></div></div></section><p id="Par55">Bioinformatically, we randomly downsampled the total number of sequenced paired-end reads to be 45 million for each sample and observed variation in both coverage and the number of variants detected across the different kits. Ideally, at 0% VAF, representing the control scenario with no expected variants, the detection of any variants indicates false positives. Here, a variant is considered detected if it is supported by more than one read or more than one consensus read. Using the Twist-0 control and preparing it with four different NGS kits, we observed 39 of the 49 variants detected with the Claret workflow, 22 in the NEB-prepared sample, 12 from the IDT preparation, and 6 in the TwinStrand sample (Fig. <a href="#Fig3" class="usa-link">3</a>A; Supplementary Tables <a href="#MOESM2" class="usa-link">S6</a> and <a href="#MOESM2" class="usa-link">S7</a>). Notably, both the IDT and TwinStrand kits incorporate fixed inline UMIs that can be used to form a consensus read during data analysis. This strand consensus process could potentially account for the fewer variants detected in Twist-0, as it more stringently filters out sequencing errors and potential artifacts, thus reducing the likelihood of false positives. Consequently, the observed low VAFs in Twist-0 for TwinStrand (average 0.13%, range of 0–2.48%) and IDT (average 0.17%, range 0–4.68%) are consistent with the expected outcomes of the thorough consensus filtering. In comparison, the variants detected in Twist-0 processed with NEB and Claret exhibited higher average VAFs of 0.35% (range 0–8.58%) and 0.37% (range 0–8.42%), respectively. Notably, performing strand consensus impacts overall coverage at the variant of interest loci, leading to TwinStrand having the lowest average duplex consensus coverage for the Twist cfDNA controls (Twist-0: 375x; Twist-1: 309x; Twist-5: 342x), while Claret reported the highest average coverage (Twist-0: 1,851x; Twist-1: 1,851x; Twist-5: 1,508x) in the normalized downsampled data. While there were distinct differences in coverage between the kits, several variants detected within the Twist-0 control were supported by only a few reads. The Claret sample had 24 detected variants supported by one or two reads, the NEB sample had 19, the TwinStrand sample had three variants with read support of one or two consensus reads, and IDT had eight variants with read support of one or two consensus reads. These results suggest the need for custom filtered analysis to mitigate workflow-specific artifacts while maintaining a balance between coverage and read support thresholds, highlighting the challenges in specificity and revealing the diverse background noise levels across the kits.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374395_12885_2025_14711_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/03d2/12374395/57142a90b702/12885_2025_14711_Fig3_HTML.jpg" loading="lazy" id="MO4" height="545" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance evaluation of NGS kits for cfDNA variant detection by input and VAF. (A) The plot displays the variant detection capabilities of four NGS kits—NEBNext Ultra II (NEB, green), SRSLY PicoPlus (Claret, blue), Duplex Library Prep (TwinStrand, orange), and cfDNA &amp; FFPE DNA Library Prep (IDT, purple)—using cfDNA samples. Variants are shown as triangles for SNVs and circles for INDELs, mapped against observed VAFs. Colors represent different kits, demonstrating their precision and accuracy in identifying variants at controlled VAFs (5%, 1%, 0%) with 10 ng of Twist Pan-Cancer cfDNA. (B) This plot evaluates the performance of Claret (blue) and IDT (purple) kits under varying inputs from 0.1 to 10 ng, illustrating their efficacy in variant detection across different Twist controls (−0, −1, −5), plotted against observed VAFs. Abbreviations: NGS - next-generation sequencing, NEB - New England Biolabs, IDT - Integrated DNA Technologies, cfDNA - cell-free DNA, SNVs - single nucleotide variants, INDELs - insertions/deletions, VAF - variant allele frequencies</p></figcaption></figure><p id="Par57">Next, we evaluated variant detection within the Twist-1 and Twist-5 controls which demonstrated differences in sensitivity and precision among the kits at these VAF thresholds. For Twist-1 and Twist-5, NEB identified 43 variants in each sample, achieving average VAFs of 0.83% for Twist-1 and 2.57% for Twist-5. Claret detected 48 variants in both controls, with average VAFs reaching 0.86% in Twist-1 and 3.09% in Twist-5. TwinStrand showed a lower detection rate, with 34 variants in Twist-1 (average VAF 0.52%) and 42 in Twist-5 (average VAF 2.02%). Lastly, the IDT workflow resulted in 40 variants detected in Twist-1 at an average VAF of 0.67% and 45 variants detected in Twist-5, with an average VAF of 2.83%.</p>
<p id="Par58">Given these collective results, along with Claret’s ability to construct sequenceable libraries from single-stranded DNA, thus improving coverage, and IDT’s use of strand consensus to minimize background noise in variant detection, both kits were selected for further evaluation. This assessment used the Twist − 0, −1, and − 5 controls with inputs varying from 0.1 to 5 ng, aiming to evaluate kit efficacy under a wide spectrum of challenging input conditions. The same probe panel targeting 25 SNVs and 24 INDELs, as previously described, was used to determine kit efficacy in variant detection at these varying input amounts (Fig. <a href="#Fig3" class="usa-link">3</a>B; Supplementary Tables <a href="#MOESM2" class="usa-link">S6</a> and <a href="#MOESM2" class="usa-link">S7</a>). For both Claret and IDT kits at the minimal 0.1ng input, the number of variants detected in the Twist-0 control (Claret: 12 variants detected; IDT: 4) closely matched those found in the Twist-1 control (Claret: 9; IDT: 3), and for Claret, the Twist-5 control (13 variants detected). However, more than twice as many variants were found in the IDT prepared Twist-5 control (10 variants detected) than the IDT prepared Twist-0 control (4 variants detected). This deviation highlights that, despite the overall reduced sensitivity at such low DNA input, IDT’s strand consensus methodology may still discern true variants, at a VAF greater than 5%, with effectiveness at a 0.1 ng input. At the 0.5 ng input level, the Claret workflow resulted in eight detected variants in Twist-0 with an average VAF of 0.51% (range: 0–7.35%), 16 variants in Twist-1 with an average VAF of 0.93% (range: 0–9.78%), and 28 variants in Twist-5, showcasing an average VAF of 2.07% (range: 0–13.33%). Conversely, IDT’s performance at the same input level revealed four variants detected in Twist-0 with an average VAF of 0.19% (range: 0–4.23%), 11 variants in Twist-1 with an average VAF of 0.44% (range: 0–3.31%), and an increase to 34 variants in Twist-5, where the average VAF was 2.85% (range: 0–13.51%). At 1 ng and 5 ng inputs, Claret and IDT kits demonstrated improved variant detection and observed VAF, with Claret detecting up to 43 variants and IDT up to 45 variants in the Twist-5 control. Considering the IDT kit’s effective variant detection with the Twist Pan-Cancer cfDNA controls at minimal inputs, this kit was subsequently selected for evaluating patient samples for ctDNA.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Assessment of liquid biopsy for detecting MRD in children with brain tumors</h3>
<p id="Par59">In this pilot study, liquid biopsy (LB) was employed as a novel approach to detect ctDNA as a surrogate for MRD in patients with central nervous system tumors after various stages of treatment. We collected CSF samples from six patients (Supplementary Table <a href="#MOESM2" class="usa-link">S2</a>) to assess MRD at various timepoints including mid-chemotherapy (LB001 and LB004), post-radiation therapy (LB007), and completion of their treatment regimen (LB003, LB005 and LB006). This study employed a multimodal approach combining our optimized hybridization capture of cfDNA plus lpGS (Fig. <a href="#Fig1" class="usa-link">1</a>A). A distinct feature of our study was the development of personalized “MRD4U” panels for each patient. All patients undergoing MRD4U had prior tumor sequencing in either the research or clinical setting and from those data, we chose up to 100 patient-specific known somatic variants to be targeted in our assay (Supplementary Figure <a href="#MOESM1" class="usa-link">S1</a>). In addition, we included private, heterozygous germline variants and a “static backbone” comprising probes against common pediatric CNS tumor alterations and against the Twist Pan-Cancer cfDNA variants for positive processing controls (Supplementary Table <a href="#MOESM2" class="usa-link">S8</a>).</p>
<p id="Par60">In this LB cohort, the extracted cfDNA yields from CSF were below the limit of detection for our quantification method, except for a single sample (LB007) which yielded 1.1 ng of cfDNA (Supplementary Table <a href="#MOESM2" class="usa-link">S2</a>). Despite most yields being classified as “undetectable” with our current method, we successfully targeted and detected patient-specific germline variants in all cfDNA samples, indicating that patient-specific cfDNA was present in each extract (Table <a href="#Tab2" class="usa-link">2</a> Supplementary Table <a href="#MOESM2" class="usa-link">S10</a>). The entirety of each CSF cfDNA extract was used for IDT library preparation, and the resulting amplified product was then split: &gt;= 125 ng was used for MRD4U capture and &gt; 10 ng for lpGS Supplementary Table <a href="#MOESM2" class="usa-link">S9</a>). Leveraging the strand consensus capability of the highlighted IDT library kit enabled us to achieve an average targeted variant loci consensus coverage of 46.0x (range = 4.9x-173.1x) across the six CSF-derived cfDNA samples (Supplementary Table <a href="#MOESM2" class="usa-link">S10</a>).</p>
<p id="Par61">We identified two ctDNA + in our LB cohort: LB005 (end of therapy) and LB007 (post-radiation) (Table <a href="#Tab2" class="usa-link">2</a>). For LB005, the personalized MRD4U panel targeted 50 tumor-informed somatic variant targets, of which 11 were detected in the patient’s CSF-derived cfDNA with a mean VAF of 37.2% (range: from 17.4 to 70.8%) (Table <a href="#Tab2" class="usa-link">2</a> and Fig. <a href="#Fig4" class="usa-link">4</a>A-B). The MRD4U panel also targeted 10 patient-specific germline variants, which in LB005 CSF cfDNA achieved an average consensus coverage of 63.7x and showed an average VAF of 51.2%. In patient LB007, the personalized MRD4U panel, designed with 100 tumor-informed variant targets, detected 52 of those variants in the CSF-derived cfDNA with an average VAF of 10.9%, ranging from 1.3 to 35.6%. For the patient’s targeted 10 germline variants, we observed an average consensus coverage of 122.7x with an average VAF of 43.9% (Supplementary Table <a href="#MOESM2" class="usa-link">S10</a>). In LB007, we observed a significant correlation between tumor VAF and CSF cfDNA VAF among variants detected in both samples, with a wide range of allele frequencies in the CSF; across both LB005 and LB007, ctDNA was consistently detectable in the CSF for variants with tumor VAFs greater than 5%, with detection rates increasing in the 10–20% and 20–50% bins (Supplementary Fig <a href="#MOESM3" class="usa-link">S3</a>). Neither LB005 nor LB007 had evidence of disease based on radiographic or clinical findings at the time of MRD4U; clinical cytological analysis was negative for malignant cells, and both brain and spine imaging (MRI) was negative for evidence of recurrent or metastatic disease, respectively. Subsequent disease imaging, however, in LB005 confirmed disease recurrence and is described in greater detail in the Case Study section below. LB007 unfortunately passed away due to disease; longitudinal follow-up of patients characterized as ctDNA- (LB001, LB003, LB004, and LB006) confirmed that these patients have remained in remission without any reoccurrence or relapse of cancer 10–13 months following CSF sample collection.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374395_12885_2025_14711_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/03d2/12374395/cc3d3725ee00/12885_2025_14711_Fig4_HTML.jpg" loading="lazy" id="MO5" height="473" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comprehensive analysis of genetic variants and copy number variations in patient LB005 from the MRD4U study. Panels A and B illustrate the VAF of variants detected in patient LB005 from the MRD4U study. Panel A presents SNVs, with somatic variants depicted as closed circles and germline variants as closed triangles, plotted against VAFs on the y-axis and various samples on the x-axis: ref-gDNA (green), Twist-5 control (blue), HD786 control (orange), and CSF-SN (purple), alongside original tumor genomic DNA VAFs (pink) for context. Panel B mirrors this for INDELs. Panels C and D detail CNVs in LB005, with Panel C showing CNV changes in tumor genomic DNA from exome profiling, and Panel D displaying CNVs in CSF cfDNA identified via lpGS. Abbreviations: VAF - variant allele frequencies, SNV - single nucleotide variant, ref-gDNA - reference genomic DNA, CSF - cerebrospinal fluid, CSF-SN - CSF supernatant, INDEL - insertion/deletion, CNV - copy number variation, lpGS– low-pass genome sequencing, cfDNA - cell-free DNA</p></figcaption></figure><p id="Par63">Prior to MRD4U hybridization capture, a portion of each patient’s CSF cfDNA library was saved for lpGS to assess for known tumor copy number variations (CNVs). First, as a positive control for our lpGS assay we sequenced the Horizon Structural Multiplex cfDNA reference standard HD786, which contains known focal amplifications. To evaluate efficiency of CNV detection using low coverage, the control was prepared with the same IDT kit with varying inputs (1 ng and 5 ng) and sequenced to different depths. The 5 ng input reached an average coverage of 22.9x, while the 1 ng inputs were sequenced to 18.7x and a lower depth of 4.0x. Known focal amplifications were successfully detected in the 1 ng input with low coverage (Supplementary Fig <a href="#MOESM4" class="usa-link">S4</a>A). To establish a comparator dataset for somatic CNV calling in our CSF cfDNA samples, we generated a “panel of normals” using cfDNA isolated from ten anonymous blood donors, prepared using the same IDT kit with 1 ng input (refer to Methods section). Average coverage for lpGS in all LB samples was 2.8x (range = 1.0x-5.5x) (Supplementary Table <a href="#MOESM2" class="usa-link">S11</a>). We found that except for LB004 (1.0x coverage), all lpGS results were interpretable at coverages &gt; 1.5x.</p>
<p id="Par64">Known tumor CNVs were detected only in LB005 and LB007, supporting the ctDNA + finding noted in our MRD4U hybridization capture assay (Supplementary Table <a href="#MOESM2" class="usa-link">S11</a> and Supplementary Fig. <a href="#MOESM4" class="usa-link">S4</a>B). All others were reported to have no CNVs is the CSF cfDNA, except LB004, whose sample had too low coverage to make a conclusive finding. Notably for LB005, prior tumor exome sequencing indicated a gain of chromosome arm 7q, losses of chromosome arms 11p and 20p, and an isochromosome 17q [(i)17q] (Fig. <a href="#Fig4" class="usa-link">4</a>C), all of which were positively detected and visualized in the CSF cfDNA lpGS at 2.6x coverage (Fig. <a href="#Fig4" class="usa-link">4</a>D).</p></section><section id="Sec18"><h3 class="pmc_sec_title">A case study highlighting early detection capabilities</h3>
<p id="Par65">LB005 is a female diagnosed at 10 years of age with localized (M0) non-WNT/non-SHH (Group 4) medulloblastoma. She underwent gross total resection of posterior fossa disease, followed by adjuvant proton craniospinal irradiation and maintenance chemotherapy as per the Children’s Oncology Group (COG) ACNS0331 regimen, though was only able to complete six of nine planned chemotherapy cycles due to significant infectious and gastrointestinal toxicity. Approximately two years following completion of chemotherapy, imaging demonstrated relapsed disease with distant spinal metastases (two bulky masses in cervical and thoracic spine, without intracranial disease), for which she underwent resection of both masses (confirming group 4 medulloblastoma), followed by craniospinal proton re-irradiation. She was then treated with maintenance chemotherapy as per COG ACNS0821 regimen with temozolomide, irinotecan, and bevacizumab, though was only able to complete three cycles due to substantial hematologic toxicity leading to prolonged delays, despite several dose reductions. The patient subsequently underwent close surveillance with regular MRI, at which time she was enrolled on our research liquid biopsy study as “LB005.” A lumbar puncture performed approximately 6 months following chemotherapy completion was negative for malignant cells by cytology, though results of our MRD4U assay were positive for presence of ctDNA in CSF, indicative of residual or recurrent disease, as described above. Brain MRI at this time remained stable from recent prior scans, without radiographic evidence of recurrent disease (Fig. <a href="#Fig5" class="usa-link">5</a>A-B). Repeat MRI approximately three months after the CSF MRD4U ctDNA + research finding demonstrated new/increased small enhancing, diffusion restricting nodules around the posterior fossa surgical cavity, worrisome for relapsed intracranial disease (Fig. <a href="#Fig5" class="usa-link">5</a>C-D). Short interval follow-up MRI then one month later showed further increase in size and number of these enhancing, diffusion-restricting nodules, now convincing for disease recurrence (Fig. <a href="#Fig5" class="usa-link">5</a>E-F).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374395_12885_2025_14711_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/03d2/12374395/a2e08bed4801/12885_2025_14711_Fig5_HTML.jpg" loading="lazy" id="MO6" height="577" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>LB005 MRI from three longitudinal timepoints. Imaging at the time of MRD4U liquid biopsy in (A&amp;B): Axial T1-weighted (T1-W) post contrast image in (A) shows no definite enhancing foci along the fourth ventricular margin while axial diffusion weighted imaging (DWI) in (B) shows no evidence of foci or restricted diffusion along the fourth ventricular margin. Imaging three months post MRD4U liquid biopsy in (C&amp;D): Axial T1-W post contrast image in (C) shows an enhancing nodule along the fourth ventricular margin (arrow) while Axial DWI in (D) shows restricted diffusion of this nodule (arrow). Imaging four months post MRD4U liquid biopsy in (E&amp;F): Axial T1-W post contrast image in (E) shows enlarging nodule along the fourth ventricular margin (arrow) while axial DWI in (F) shows restricted diffusion of this nodule (arrow)</p></figcaption></figure></section></section><section id="Sec19"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par67">Despite significant advancements in liquid biopsy research for pediatric brain tumors, challenges in standardizing methodologies persist, particularly in adapting to low input, highly fragmented cfDNA and establishing robust analytical pipelines for potential transition into clinical application [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Our study addresses the challenge of working with limited CSF volumes from children with CNS tumors, in which we obtained an average of 1.2 mL, yielding 2.02 ng/mL of cfDNA on average (range: 0-16.6 ng/mL) in our study from lumbar puncture-derived CSF samples; other similar studies have shown wide ranges of extraction yields and similar average as ours [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Our comparative analysis of four commercially available NGS library preparation kits evaluated their performance with these low input amounts, their ability to process the single-stranded DNA commonly found in cfDNA, and their error correction capabilities using unique molecular identifiers (UMIs) and strand consensus variant calling. The most effective kit demonstrated the best capacity to minimize false positives, a crucial attribute when analyzing low variant allele frequencies (VAFs) in circulating cfDNA. This was particularly evident when we sequenced a negative control, Twist-0, where no somatic variants were expected. Additionally, the selected preparation method excelled in detecting somatic variants at extremely low DNA inputs, down to 0.1 ng inputs efficiently in our study using synthetic controls. Similarly, samples in our LB pilot study, with inputs at or below the 0.1 ng threshold, still yielded positive findings. This demonstrates the flexibility of our assay, which can work with significantly lower DNA inputs compared to the 2–5 ng lower limit commonly reported in other studies [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Assays amenable to low input are especially important for pediatric CSF applications.</p>
<p id="Par68">Upon optimizing this workflow, we implemented a multi-modal liquid biopsy approach that combined lpGS for detecting CNVs with our personalized, tumor-informed targeted sequencing strategy MRD4U for somatic variants. The MRD4U workflow targets mutations identified from each patient’s prior tumor sequencing data. However, a limitation of MRD4U is its reliance on diagnostic tumor profiling, which may miss spatial or temporal heterogeneity including relapse-specific alterations or secondary malignancies [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. Integrating lpGS as a complementary assay helps address this gap by enabling detection of evolving genomic changes without prior knowledge. Our decision to adopt a personalized panel design, instead of a broader static panel, is based on several factors. Primarily, pediatric CNS entities represent individualized diseases and while recurrent hotspot variants are observed in some cases, the mutational profile is diverse even amid genes within recurrently altered signaling pathways. We opted for a method (hybridization capture) to target a diverse array of mutations. This approach contrasts with our demonstrated Libricon technique or digital droplet PCR methods, both of which require extensive optimization to scale the number of targets they can assess. Additionally, unlike gene-specific panels that apply only to certain tumor types - such as targeting <em>BRAF</em> in low-grade gliomas or <em>H3K27M</em> in diffuse midline gliomas - our chosen method offers broader applicability [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>–<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. Moreover, while many “pan-cancer” panels exist, these are generally designed around adult cancers and are often too extensive to achieve deep coverage practically, particularly when using strand consensus for variant calling [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. The development of personalized panels for targeted sequencing in our study relies on prior tumor sequencing, which is becoming more routine in pediatrics, as seen in the Children’s Oncology Group Molecular Characterization Initiative [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. Additionally, all tumors, including fusion-driven cancers, harbor some somatic variants at relatively high VAFs; even in low-mutation burden tumors, passenger mutations are typically present and detectable using high-sensitivity sequencing approaches such as deep exome [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. In our study, tumor variants present at &gt; 5% VAF were detectable in matched CSF using our current MRD4U workflow, supporting its applicability even in tumors with low mutational burden or without recurrent driver mutations. In our LB pilot study, the minimum number of somatic variants to be targeted was 20 (LB004), with tumor VAFs ranging 2–70%. Our workflow evaluation with synthetic cfDNA controls successfully demonstrated circulating VAF detection as low as 5% even with low input starting material.</p>
<p id="Par69">Tailoring patient-specific panels ensures that sequencing coverage is directed toward known tumor variants for a given individual, which is crucial when the available coverage is expected to be relatively low. In our study, the average somatic target consensus depth in MRD4U ranged from 4.9x to 173.1x and for lpGS, it ranged from 1.0x to 5.5x. The highest targeted coverage, observed in LB ctDNA + patients (43.3x and 173.1x), highlights the potential limitations in detecting ctDNA/MRD in patients with lower coverage, likely due to minimal amounts of cfDNA initially available for library preparation, although further studies with a larger number of patients are needed to definitively determine the optimal coverage depth. Our selected method for library preparation used known fixed UMI sequences, added to initial DNA molecules, to allow for strand consensus calling. This strategy decreases base calling errors which is especially important in liquid biopsy samples which often have either low input or low VAF. In our pipeline, we require a minimum of three consensus reads to positively identify a somatic variant, reducing the chance of false positive calls.</p>
<p id="Par70">In our study we applied our optimized MRD4U workflow to a small, exploratory cohort of six children and successfully identified two as ctDNA + using CSF. Interestingly, in one of these cases, the presence of ctDNA was detected three months prior to radiographic evidence of recurrence. Due to the experimental nature of our research assay and in adherence with consent and protocol guidelines, these liquid biopsy results were not shared with either the clinicians or the patients. While our workflow was designed with clinical implementation in mind and emphasizes speed, cost-efficiency, and feasibility with panel design and probe synthesis achievable within five days, it is important to note limitations. Our cohort is heavily weighted towards medulloblastoma cases, reflecting real-world accessibility as CSF is more readily collected in this patient population due to routine lumbar punctures for disease staging and surveillance. This imbalance, however, limits ability to draw broader conclusions about associations between ctDNA detectability and tumor types. Although our MRD4U approach is theoretically applicable to any patient with prior tumor sequencing, broader implementation may be constrained by institutional infrastructure for custom assay development. Larger, diverse cohorts and further real-world testing are needed to fully evaluate the clinical utility and scalability of this personalized strategy across pediatric CNS tumors.</p></section><section id="Sec20"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par71">The capacity to detect ctDNA in CSF holds great promise in monitoring pediatric brain tumors, particularly in scenarios where traditional imaging and cytological methods may fail to identify residual or recurrent disease, as demonstrated in our exploratory pilot study and many other previous studies [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. This capability could support earlier prediction of relapse and more precise measurement of responses to therapeutic interventions, thereby informing treatment decisions and monitoring strategies. Additionally, since lumbar punctures are frequently part of standard care for some patients, CSF remains readily accessible for these assays. Our personalized “MRD4U” approach, which tailors capture panels to each patient’s tumor profile, offers a scalable and cost-conscious alternative to static or genome-wide methods, and may improve sensitivity across diverse pediatric CNS tumor types.</p></section><section id="Sec21"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12374395/bin/12885_2025_14711_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (24.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12374395/bin/12885_2025_14711_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (459.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12374395/bin/12885_2025_14711_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (63.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12374395/bin/12885_2025_14711_MOESM4_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (158.1KB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12374395/bin/12885_2025_14711_MOESM5_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (695.2KB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12374395/bin/12885_2025_14711_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (1.8MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We are grateful for the patients and families who participated in this study. The schematics in Figure 1 A was generated using biorender.com. We thank Daniel C Koboldt, MS for providing code for data analysis.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>AYA</dt>
<dd><p id="Par5">Adolescent and Young Adult</p></dd>
<dt>cfDNA</dt>
<dd><p id="Par6">Cell-Free DNA</p></dd>
<dt>CNS</dt>
<dd><p id="Par7">Central Nervous System</p></dd>
<dt>CNVs</dt>
<dd><p id="Par8">Copy Number Variations</p></dd>
<dt>CSI</dt>
<dd><p id="Par9">Craniospinal Irradiation</p></dd>
<dt>CSF</dt>
<dd><p id="Par10">Cerebrospinal Fluid</p></dd>
<dt>CSF-SN</dt>
<dd><p id="Par11">CSF Supernatant</p></dd>
<dt>ctDNA</dt>
<dd><p id="Par12">Circulating Tumor DNA</p></dd>
<dt>ctDNA-</dt>
<dd><p id="Par13">Negative for Circulating Tumor DNA</p></dd>
<dt>ctDNA+</dt>
<dd><p id="Par14">Positive for Circulating Tumor DNA</p></dd>
<dt>dsDNA</dt>
<dd><p id="Par15">Double-Stranded DNA</p></dd>
<dt>ES</dt>
<dd><p id="Par16">Exome Sequencing</p></dd>
<dt>EVD</dt>
<dd><p id="Par17">External Ventricular Drain</p></dd>
<dt>INDEL</dt>
<dd><p id="Par18">Insertion/Deletion</p></dd>
<dt>IRB</dt>
<dd><p id="Par19">Institutional Review Board</p></dd>
<dt>LOD</dt>
<dd><p id="Par20">Limit of Detection</p></dd>
<dt>lpGS</dt>
<dd><p id="Par21">low-pass Genome Sequencing</p></dd>
<dt>MRI</dt>
<dd><p id="Par22">Magnetic Resonance Imaging</p></dd>
<dt>MRD</dt>
<dd><p id="Par23">Measurable Residual Disease</p></dd>
<dt>MRD4U</dt>
<dd><p id="Par24">MRD for You</p></dd>
<dt>ng</dt>
<dd><p id="Par25">Nanogram</p></dd>
<dt>NGS</dt>
<dd><p id="Par26">Next-Generation Sequencing</p></dd>
<dt>PON</dt>
<dd><p id="Par27">Panel of Normals</p></dd>
<dt>SNV</dt>
<dd><p id="Par28">Single Nucleotide Variant</p></dd>
<dt>UMIs</dt>
<dd><p id="Par29">Unique Molecular Identifiers</p></dd>
<dt>VAF</dt>
<dd><p id="Par30">Variant Allele Frequency</p></dd>
<dt>VP</dt>
<dd><p id="Par31">Ventriculoperitoneal</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: ARM, ERM, KEM. Methodology: ARM, SAW, HZ. Software: CNS, EARG, AOB, GW, DMG, PC, BJK, KEM. Formal analysis: ARM, TS, CNS, AR, SAW, AOB, GW, DMG, PC, BJK, KEM. Investigation: TS, AR, SAW, HZ, JML, MM, SC. Resources: DPR, RKW, CEC, SK, MF, ERM, MAL. Data curation: ARM, TS, CNS, AR, SAW, EARG, SLP, KEM. Visualization: ARM, TS, CNS, EARG, GW, DMG, KEM. Writing– Original draft: ARM, TS, CNS, AR, AOB, MAL, KEM. Writing– Review &amp; editing: All authors. Supervision: RKW, ERM. Project administration: ARM, CMC, EAV, KEM. Funding acquisition: RKW, ERM, MAL, KEM.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the Nationwide Foundation Pediatric Innovation Fund and by a Clinical and Translational Intramural Funding Award (IFPAWRI092022) from the Abigail Wexner Research Institute at Nationwide Children’s Hospital.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All sequencing BAM files have been deposited in NCBI’s Sequence Read Archive which is accessible through BioProject ID PRJNA1115058. All computational code is provided as supplementary data files.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests </h3>
<p id="Par72">The authors declare no competing interests. </p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par73">Consent for publication was obtained from the patients and parent(s).</p></section><section id="FPar3"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par74">Written informed consent to participate was obtained from the patients (or parents for cases younger than the age of 16) under a research protocol (STUDY00003080) approved by the Institutional Review Board at Nationwide Children’s Hospital. The research and methods performed related to human use were performed in accordance with relevant guidelines, institutional policies and in accordance with the tenets of the Helsinki Declaration.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Wan JCM, et al. Liquid biopsies come of age: towards implementation of Circulating tumour DNA. Nat Rev Cancer. 2017;17:223–38. 10.1038/nrc.2017.7.
</cite> [<a href="https://doi.org/10.1038/nrc.2017.7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28233803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wan%20JCM,%20et%20al.%20Liquid%20biopsies%20come%20of%20age:%20towards%20implementation%20of%20Circulating%20tumour%20DNA.%20Nat%20Rev%20Cancer.%202017;17:223%E2%80%9338.%2010.1038/nrc.2017.7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Siravegna G, Marsoni S, Siena S, Bardelli A. Integrating liquid biopsies into the management of cancer. Nat Rev Clin Oncol. 2017;14:531–48. 10.1038/nrclinonc.2017.14.
</cite> [<a href="https://doi.org/10.1038/nrclinonc.2017.14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28252003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Siravegna%20G,%20Marsoni%20S,%20Siena%20S,%20Bardelli%20A.%20Integrating%20liquid%20biopsies%20into%20the%20management%20of%20cancer.%20Nat%20Rev%20Clin%20Oncol.%202017;14:531%E2%80%9348.%2010.1038/nrclinonc.2017.14." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Jahr S, et al. DNA fragments in the blood plasma of cancer patients: quantitations and evidence for their origin from apoptotic and necrotic cells. Cancer Res. 2001;61:1659–65.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/11245480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jahr%20S,%20et%20al.%20DNA%20fragments%20in%20the%20blood%20plasma%20of%20cancer%20patients:%20quantitations%20and%20evidence%20for%20their%20origin%20from%20apoptotic%20and%20necrotic%20cells.%20Cancer%20Res.%202001;61:1659%E2%80%9365." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Diaz LA Jr., Bardelli A. Liquid biopsies: genotyping Circulating tumor DNA. J Clin Oncol. 2014;32:579–86. 10.1200/JCO.2012.45.2011.
</cite> [<a href="https://doi.org/10.1200/JCO.2012.45.2011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4820760/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24449238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diaz%20LA%20Jr.,%20Bardelli%20A.%20Liquid%20biopsies:%20genotyping%20Circulating%20tumor%20DNA.%20J%20Clin%20Oncol.%202014;32:579%E2%80%9386.%2010.1200/JCO.2012.45.2011." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Parikh AR, et al. Liquid versus tissue biopsy for detecting acquired resistance and tumor heterogeneity in Gastrointestinal cancers. Nat Med. 2019;25:1415–21. 10.1038/s41591-019-0561-9.
</cite> [<a href="https://doi.org/10.1038/s41591-019-0561-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6741444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31501609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Parikh%20AR,%20et%20al.%20Liquid%20versus%20tissue%20biopsy%20for%20detecting%20acquired%20resistance%20and%20tumor%20heterogeneity%20in%20Gastrointestinal%20cancers.%20Nat%20Med.%202019;25:1415%E2%80%9321.%2010.1038/s41591-019-0561-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Oxnard GR, et al. Association between plasma genotyping and outcomes of treatment with osimertinib (AZD9291) in advanced Non-Small-Cell lung cancer. J Clin Oncol. 2016;34:3375–82. 10.1200/JCO.2016.66.7162.
</cite> [<a href="https://doi.org/10.1200/JCO.2016.66.7162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5035123/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27354477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oxnard%20GR,%20et%20al.%20Association%20between%20plasma%20genotyping%20and%20outcomes%20of%20treatment%20with%20osimertinib%20(AZD9291)%20in%20advanced%20Non-Small-Cell%20lung%20cancer.%20J%20Clin%20Oncol.%202016;34:3375%E2%80%9382.%2010.1200/JCO.2016.66.7162." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Dagogo-Jack I, et al. Treatment with Next-Generation ALK inhibitors fuels plasma ALK mutation diversity. Clin Cancer Res. 2019;25:6662–70. 10.1158/1078-0432.CCR-19-1436.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-19-1436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6858956/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31358542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dagogo-Jack%20I,%20et%20al.%20Treatment%20with%20Next-Generation%20ALK%20inhibitors%20fuels%20plasma%20ALK%20mutation%20diversity.%20Clin%20Cancer%20Res.%202019;25:6662%E2%80%9370.%2010.1158/1078-0432.CCR-19-1436." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Thierry AR, et al. Clinical validation of the detection of KRAS and BRAF mutations from Circulating tumor DNA. Nat Med. 2014;20:430–5. 10.1038/nm.3511.
</cite> [<a href="https://doi.org/10.1038/nm.3511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24658074/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thierry%20AR,%20et%20al.%20Clinical%20validation%20of%20the%20detection%20of%20KRAS%20and%20BRAF%20mutations%20from%20Circulating%20tumor%20DNA.%20Nat%20Med.%202014;20:430%E2%80%935.%2010.1038/nm.3511." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Nakamura Y, et al. Clinical utility of Circulating tumor DNA sequencing in advanced Gastrointestinal cancer: SCRUM-Japan GI-SCREEN and GOZILA studies. Nat Med. 2020;26:1859–64. 10.1038/s41591-020-1063-5.
</cite> [<a href="https://doi.org/10.1038/s41591-020-1063-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33020649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakamura%20Y,%20et%20al.%20Clinical%20utility%20of%20Circulating%20tumor%20DNA%20sequencing%20in%20advanced%20Gastrointestinal%20cancer:%20SCRUM-Japan%20GI-SCREEN%20and%20GOZILA%20studies.%20Nat%20Med.%202020;26:1859%E2%80%9364.%2010.1038/s41591-020-1063-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Bidard FC, et al. Circulating tumor cells and Circulating tumor DNA detection in potentially resectable metastatic colorectal cancer: A prospective ancillary study to the unicancer Prodige-14 trial. Cells. 2019;8. 10.3390/cells8060516.</cite> [<a href="https://doi.org/10.3390/cells8060516" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6627974/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31142037/" class="usa-link">PubMed</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Pritchett MA, et al. Prospective clinical validation of the InVisionFirst-Lung Circulating tumor DNA assay for molecular profiling of patients with advanced nonsquamous Non-Small-Cell lung cancer. JCO Precis Oncol. 2019;3. 10.1200/PO.18.00299.</cite> [<a href="https://doi.org/10.1200/PO.18.00299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7450945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32914040/" class="usa-link">PubMed</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Jenkins S, et al. Plasma ctdna analysis for detection of the EGFR T790M mutation in patients with advanced Non-Small cell lung cancer. J Thorac Oncol. 2017;12:1061–70. 10.1016/j.jtho.2017.04.003.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2017.04.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28428148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jenkins%20S,%20et%20al.%20Plasma%20ctdna%20analysis%20for%20detection%20of%20the%20EGFR%20T790M%20mutation%20in%20patients%20with%20advanced%20Non-Small%20cell%20lung%20cancer.%20J%20Thorac%20Oncol.%202017;12:1061%E2%80%9370.%2010.1016/j.jtho.2017.04.003." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Bachet JB, et al. RAS mutation analysis in Circulating tumor DNA from patients with metastatic colorectal cancer: the AGEO RASANC prospective multicenter study. Ann Oncol. 2018;29:1211–9. 10.1093/annonc/mdy061.
</cite> [<a href="https://doi.org/10.1093/annonc/mdy061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29438522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bachet%20JB,%20et%20al.%20RAS%20mutation%20analysis%20in%20Circulating%20tumor%20DNA%20from%20patients%20with%20metastatic%20colorectal%20cancer:%20the%20AGEO%20RASANC%20prospective%20multicenter%20study.%20Ann%20Oncol.%202018;29:1211%E2%80%939.%2010.1093/annonc/mdy061." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Heitzer E, et al. Tumor-associated copy number changes in the circulation of patients with prostate cancer identified through whole-genome sequencing. Genome Med. 2013;5:30. 10.1186/gm434.
</cite> [<a href="https://doi.org/10.1186/gm434" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3707016/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23561577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heitzer%20E,%20et%20al.%20Tumor-associated%20copy%20number%20changes%20in%20the%20circulation%20of%20patients%20with%20prostate%20cancer%20identified%20through%20whole-genome%20sequencing.%20Genome%20Med.%202013;5:30.%2010.1186/gm434." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Dawson SJ, et al. Analysis of Circulating tumor DNA to monitor metastatic breast cancer. N Engl J Med. 2013;368:1199–209. 10.1056/NEJMoa1213261.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1213261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23484797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dawson%20SJ,%20et%20al.%20Analysis%20of%20Circulating%20tumor%20DNA%20to%20monitor%20metastatic%20breast%20cancer.%20N%20Engl%20J%20Med.%202013;368:1199%E2%80%93209.%2010.1056/NEJMoa1213261." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Tyner JW, et al. Functional genomic landscape of acute myeloid leukaemia. Nature. 2018;562:526–31. 10.1038/s41586-018-0623-z.
</cite> [<a href="https://doi.org/10.1038/s41586-018-0623-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6280667/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30333627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tyner%20JW,%20et%20al.%20Functional%20genomic%20landscape%20of%20acute%20myeloid%20leukaemia.%20Nature.%202018;562:526%E2%80%9331.%2010.1038/s41586-018-0623-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Cancer Genome Atlas, N. Comprehensive molecular characterization of human colon and rectal cancer. Nature. 2012;487:330–7. 10.1038/nature11252.
</cite> [<a href="https://doi.org/10.1038/nature11252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3401966/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22810696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cancer%20Genome%20Atlas,%20N.%20Comprehensive%20molecular%20characterization%20of%20human%20colon%20and%20rectal%20cancer.%20Nature.%202012;487:330%E2%80%937.%2010.1038/nature11252." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>De Mattos-Arruda L, et al. Cerebrospinal fluid-derived Circulating tumour DNA better represents the genomic alterations of brain tumours than plasma. Nat Commun. 2015;6:8839. 10.1038/ncomms9839.
</cite> [<a href="https://doi.org/10.1038/ncomms9839" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5426516/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26554728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Mattos-Arruda%20L,%20et%20al.%20Cerebrospinal%20fluid-derived%20Circulating%20tumour%20DNA%20better%20represents%20the%20genomic%20alterations%20of%20brain%20tumours%20than%20plasma.%20Nat%20Commun.%202015;6:8839.%2010.1038/ncomms9839." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Miller AM, et al. Tracking tumour evolution in glioma through liquid biopsies of cerebrospinal fluid. Nature. 2019;565:654–8. 10.1038/s41586-019-0882-3.
</cite> [<a href="https://doi.org/10.1038/s41586-019-0882-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6457907/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30675060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miller%20AM,%20et%20al.%20Tracking%20tumour%20evolution%20in%20glioma%20through%20liquid%20biopsies%20of%20cerebrospinal%20fluid.%20Nature.%202019;565:654%E2%80%938.%2010.1038/s41586-019-0882-3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Wang Y, et al. Detection of tumor-derived DNA in cerebrospinal fluid of patients with primary tumors of the brain and spinal cord. Proc Natl Acad Sci U S A. 2015;112:9704–9. 10.1073/pnas.1511694112.
</cite> [<a href="https://doi.org/10.1073/pnas.1511694112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4534284/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26195750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Y,%20et%20al.%20Detection%20of%20tumor-derived%20DNA%20in%20cerebrospinal%20fluid%20of%20patients%20with%20primary%20tumors%20of%20the%20brain%20and%20spinal%20cord.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202015;112:9704%E2%80%939.%2010.1073/pnas.1511694112." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Escudero L, Martinez-Ricarte F, Seoane J. ctDNA-Based liquid biopsy of cerebrospinal fluid in brain cancer. Cancers (Basel). 2021;13. 10.3390/cancers13091989.</cite> [<a href="https://doi.org/10.3390/cancers13091989" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8122255/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33919036/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Buccilli B, et al. Liquid biopsies for monitoring medulloblastoma: Circulating tumor DNA as a biomarker for disease progression and treatment response. Cureus. 2024;16:e51712. 10.7759/cureus.51712.
</cite> [<a href="https://doi.org/10.7759/cureus.51712" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10838584/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38313884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buccilli%20B,%20et%20al.%20Liquid%20biopsies%20for%20monitoring%20medulloblastoma:%20Circulating%20tumor%20DNA%20as%20a%20biomarker%20for%20disease%20progression%20and%20treatment%20response.%20Cureus.%202024;16:e51712.%2010.7759/cureus.51712." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Liu APY et al. Serial assessment of measurable residual disease in medulloblastoma liquid biopsies. <em>Cancer Cell</em> 39, 1519–1530 e1514. 10.1016/j.ccell.2021.09.012 (2021).</cite> [<a href="https://doi.org/10.1016/j.ccell.2021.09.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9620970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34678152/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Ma X, et al. Pan-cancer genome and transcriptome analyses of 1,699 paediatric leukaemias and solid tumours. Nature. 2018;555:371–6. 10.1038/nature25795.
</cite> [<a href="https://doi.org/10.1038/nature25795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5854542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29489755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma%20X,%20et%20al.%20Pan-cancer%20genome%20and%20transcriptome%20analyses%20of%201,699%20paediatric%20leukaemias%20and%20solid%20tumours.%20Nature.%202018;555:371%E2%80%936.%2010.1038/nature25795." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Grobner SN, et al. The landscape of genomic alterations across childhood cancers. Nature. 2018;555:321–7. 10.1038/nature25480.
</cite> [<a href="https://doi.org/10.1038/nature25480" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29489754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grobner%20SN,%20et%20al.%20The%20landscape%20of%20genomic%20alterations%20across%20childhood%20cancers.%20Nature.%202018;555:321%E2%80%937.%2010.1038/nature25480." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Li J, et al. Reliable tumor detection by whole-genome methylation sequencing of cell-free DNA in cerebrospinal fluid of pediatric Medulloblastoma. Sci Adv. 2020;6. 10.1126/sciadv.abb5427.</cite> [<a href="https://doi.org/10.1126/sciadv.abb5427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7567591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33067228/" class="usa-link">PubMed</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>O’Halloran K, et al. Low-pass whole-genome and targeted sequencing of cell-free DNA from cerebrospinal fluid in pediatric patients with central nervous system tumors. Neurooncol Adv. 2023;5:vdad077. 10.1093/noajnl/vdad077.
</cite> [<a href="https://doi.org/10.1093/noajnl/vdad077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10349915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37461402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?O%E2%80%99Halloran%20K,%20et%20al.%20Low-pass%20whole-genome%20and%20targeted%20sequencing%20of%20cell-free%20DNA%20from%20cerebrospinal%20fluid%20in%20pediatric%20patients%20with%20central%20nervous%20system%20tumors.%20Neurooncol%20Adv.%202023;5:vdad077.%2010.1093/noajnl/vdad077." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Crotty EE, et al. Cerebrospinal fluid liquid biopsy by low-pass whole genome sequencing for clinical disease monitoring in pediatric embryonal tumors. Neurooncol Adv. 2024;6:vdae126. 10.1093/noajnl/vdae126.
</cite> [<a href="https://doi.org/10.1093/noajnl/vdae126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11407906/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39290875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crotty%20EE,%20et%20al.%20Cerebrospinal%20fluid%20liquid%20biopsy%20by%20low-pass%20whole%20genome%20sequencing%20for%20clinical%20disease%20monitoring%20in%20pediatric%20embryonal%20tumors.%20Neurooncol%20Adv.%202024;6:vdae126.%2010.1093/noajnl/vdae126." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Escudero L, et al. Circulating tumour DNA from the cerebrospinal fluid allows the characterisation and monitoring of Medulloblastoma. Nat Commun. 2020;11:5376. 10.1038/s41467-020-19175-0.
</cite> [<a href="https://doi.org/10.1038/s41467-020-19175-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7591522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33110059/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Escudero%20L,%20et%20al.%20Circulating%20tumour%20DNA%20from%20the%20cerebrospinal%20fluid%20allows%20the%20characterisation%20and%20monitoring%20of%20Medulloblastoma.%20Nat%20Commun.%202020;11:5376.%2010.1038/s41467-020-19175-0." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Miller AM, et al. Next-generation sequencing of cerebrospinal fluid for clinical molecular diagnostics in pediatric, adolescent and young adult brain tumor patients. Neuro Oncol. 2022;24:1763–72. 10.1093/neuonc/noac035.
</cite> [<a href="https://doi.org/10.1093/neuonc/noac035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9527510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35148412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miller%20AM,%20et%20al.%20Next-generation%20sequencing%20of%20cerebrospinal%20fluid%20for%20clinical%20molecular%20diagnostics%20in%20pediatric,%20adolescent%20and%20young%20adult%20brain%20tumor%20patients.%20Neuro%20Oncol.%202022;24:1763%E2%80%9372.%2010.1093/neuonc/noac035." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Ronsley R, et al. Detection of tumor-derived cell-free DNA in cerebrospinal fluid using a clinically validated targeted sequencing panel for pediatric brain tumors. J Neurooncol. 2024;168:215–24. 10.1007/s11060-024-04645-y.
</cite> [<a href="https://doi.org/10.1007/s11060-024-04645-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38755519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ronsley%20R,%20et%20al.%20Detection%20of%20tumor-derived%20cell-free%20DNA%20in%20cerebrospinal%20fluid%20using%20a%20clinically%20validated%20targeted%20sequencing%20panel%20for%20pediatric%20brain%20tumors.%20J%20Neurooncol.%202024;168:215%E2%80%9324.%2010.1007/s11060-024-04645-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Lockwood CM, et al. Recommendations for Cell-Free DNA assay validations: A joint consensus recommendation of the association for molecular pathology and college of American pathologists. J Mol Diagn. 2023;25:876–97. 10.1016/j.jmoldx.2023.09.004.
</cite> [<a href="https://doi.org/10.1016/j.jmoldx.2023.09.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37806433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lockwood%20CM,%20et%20al.%20Recommendations%20for%20Cell-Free%20DNA%20assay%20validations:%20A%20joint%20consensus%20recommendation%20of%20the%20association%20for%20molecular%20pathology%20and%20college%20of%20American%20pathologists.%20J%20Mol%20Diagn.%202023;25:876%E2%80%9397.%2010.1016/j.jmoldx.2023.09.004." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Miller KE, et al. Somatic SLC35A2 mosaicism correlates with clinical findings in epilepsy brain tissue. Neurol Genet. 2020;6:e460. 10.1212/NXG.0000000000000460.
</cite> [<a href="https://doi.org/10.1212/NXG.0000000000000460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7323482/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32637635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miller%20KE,%20et%20al.%20Somatic%20SLC35A2%20mosaicism%20correlates%20with%20clinical%20findings%20in%20epilepsy%20brain%20tissue.%20Neurol%20Genet.%202020;6:e460.%2010.1212/NXG.0000000000000460." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Karczewski KJ, et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature. 2020;581:434–43. 10.1038/s41586-020-2308-7.
</cite> [<a href="https://doi.org/10.1038/s41586-020-2308-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7334197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karczewski%20KJ,%20et%20al.%20The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141,456%20humans.%20Nature.%202020;581:434%E2%80%9343.%2010.1038/s41586-020-2308-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Miller KE, et al. Molecular heterogeneity in pediatric malignant rhabdoid tumors in patients with Multi-Organ involvement. Front Oncol. 2022;12:932337. 10.3389/fonc.2022.932337.
</cite> [<a href="https://doi.org/10.3389/fonc.2022.932337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9326117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35912263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miller%20KE,%20et%20al.%20Molecular%20heterogeneity%20in%20pediatric%20malignant%20rhabdoid%20tumors%20in%20patients%20with%20Multi-Organ%20involvement.%20Front%20Oncol.%202022;12:932337.%2010.3389/fonc.2022.932337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Petralia F et al. Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer. <em>Cell</em> 183, 1962–1985 e1931, 10.1016/j.cell.2020.10.044 (2020).</cite> [<a href="https://doi.org/10.1016/j.cell.2020.10.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8143193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33242424/" class="usa-link">PubMed</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Northcott PA, et al. The whole-genome landscape of Medulloblastoma subtypes. Nature. 2017;547:311–7. 10.1038/nature22973.
</cite> [<a href="https://doi.org/10.1038/nature22973" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5905700/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28726821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Northcott%20PA,%20et%20al.%20The%20whole-genome%20landscape%20of%20Medulloblastoma%20subtypes.%20Nature.%202017;547:311%E2%80%937.%2010.1038/nature22973." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Koboldt DC, et al. VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing. Genome Res. 2012;22:568–76. 10.1101/gr.129684.111.
</cite> [<a href="https://doi.org/10.1101/gr.129684.111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3290792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22300766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koboldt%20DC,%20et%20al.%20VarScan%202:%20somatic%20mutation%20and%20copy%20number%20alteration%20discovery%20in%20cancer%20by%20exome%20sequencing.%20Genome%20Res.%202012;22:568%E2%80%9376.%2010.1101/gr.129684.111." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Bedrosian TA, et al. Detection of brain somatic variation in epilepsy-associated developmental lesions. Epilepsia. 2022;63:1981–97. 10.1111/epi.17323.
</cite> [<a href="https://doi.org/10.1111/epi.17323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35687047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bedrosian%20TA,%20et%20al.%20Detection%20of%20brain%20somatic%20variation%20in%20epilepsy-associated%20developmental%20lesions.%20Epilepsia.%202022;63:1981%E2%80%9397.%2010.1111/epi.17323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Morrissy AS, et al. Spatial heterogeneity in Medulloblastoma. Nat Genet. 2017;49:780–8. 10.1038/ng.3838.
</cite> [<a href="https://doi.org/10.1038/ng.3838" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5553617/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28394352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morrissy%20AS,%20et%20al.%20Spatial%20heterogeneity%20in%20Medulloblastoma.%20Nat%20Genet.%202017;49:780%E2%80%938.%2010.1038/ng.3838." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Kojic M, et al. Efficient detection and monitoring of pediatric brain malignancies with liquid biopsy based on patient-specific somatic mutation screening. Neuro Oncol. 2023;25:1507–17. 10.1093/neuonc/noad032.
</cite> [<a href="https://doi.org/10.1093/neuonc/noad032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10398816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36757207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kojic%20M,%20et%20al.%20Efficient%20detection%20and%20monitoring%20of%20pediatric%20brain%20malignancies%20with%20liquid%20biopsy%20based%20on%20patient-specific%20somatic%20mutation%20screening.%20Neuro%20Oncol.%202023;25:1507%E2%80%9317.%2010.1093/neuonc/noad032." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Krynina O, et al. The potential of liquid biopsy for detection of the KIAA1549-BRAF fusion in Circulating tumor DNA from children with pilocytic Astrocytoma. Neurooncol Adv. 2024;6:vdae008. 10.1093/noajnl/vdae008.
</cite> [<a href="https://doi.org/10.1093/noajnl/vdae008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10874216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38371226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krynina%20O,%20et%20al.%20The%20potential%20of%20liquid%20biopsy%20for%20detection%20of%20the%20KIAA1549-BRAF%20fusion%20in%20Circulating%20tumor%20DNA%20from%20children%20with%20pilocytic%20Astrocytoma.%20Neurooncol%20Adv.%202024;6:vdae008.%2010.1093/noajnl/vdae008." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Huang TY, et al. Detection of histone H3 mutations in cerebrospinal fluid-derived tumor DNA from children with diffuse midline glioma. Acta Neuropathol Commun. 2017;5:28. 10.1186/s40478-017-0436-6.
</cite> [<a href="https://doi.org/10.1186/s40478-017-0436-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5392913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28416018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20TY,%20et%20al.%20Detection%20of%20histone%20H3%20mutations%20in%20cerebrospinal%20fluid-derived%20tumor%20DNA%20from%20children%20with%20diffuse%20midline%20glioma.%20Acta%20Neuropathol%20Commun.%202017;5:28.%2010.1186/s40478-017-0436-6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Saratsis AM, et al. Insights into pediatric diffuse intrinsic Pontine glioma through proteomic analysis of cerebrospinal fluid. Neuro Oncol. 2012;14:547–60. 10.1093/neuonc/nos067.
</cite> [<a href="https://doi.org/10.1093/neuonc/nos067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3337313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22492959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saratsis%20AM,%20et%20al.%20Insights%20into%20pediatric%20diffuse%20intrinsic%20Pontine%20glioma%20through%20proteomic%20analysis%20of%20cerebrospinal%20fluid.%20Neuro%20Oncol.%202012;14:547%E2%80%9360.%2010.1093/neuonc/nos067." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Rose Brannon A, et al. Enhanced specificity of clinical high-sensitivity tumor mutation profiling in cell-free DNA via paired normal sequencing using MSK-ACCESS. Nat Commun. 2021;12:3770. 10.1038/s41467-021-24109-5.
</cite> [<a href="https://doi.org/10.1038/s41467-021-24109-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8213710/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34145282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rose%20Brannon%20A,%20et%20al.%20Enhanced%20specificity%20of%20clinical%20high-sensitivity%20tumor%20mutation%20profiling%20in%20cell-free%20DNA%20via%20paired%20normal%20sequencing%20using%20MSK-ACCESS.%20Nat%20Commun.%202021;12:3770.%2010.1038/s41467-021-24109-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>NCI Launches Pediatric Molecular Characterization Initiative. Cancer Discov. 2022;12:1402–3. 10.1158/2159-8290.CD-NB2022-0031.</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-NB2022-0031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35446390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?NCI%20Launches%20Pediatric%20Molecular%20Characterization%20Initiative.%20Cancer%20Discov.%202022;12:1402%E2%80%933.%2010.1158/2159-8290.CD-NB2022-0031." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374395/bin/12885_2025_14711_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (24.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374395/bin/12885_2025_14711_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (459.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374395/bin/12885_2025_14711_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (63.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374395/bin/12885_2025_14711_MOESM4_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (158.1KB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374395/bin/12885_2025_14711_MOESM5_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (695.2KB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374395/bin/12885_2025_14711_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (1.8MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All sequencing BAM files have been deposited in NCBI’s Sequence Read Archive which is accessible through BioProject ID PRJNA1115058. All computational code is provided as supplementary data files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Cancer are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12885-025-14711-x"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12885_2025_Article_14711.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12374395/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12374395/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374395%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374395/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12374395/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12374395/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40849662/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12374395/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40849662/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12374395/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12374395/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="PRlUg5i2TlvvDIE5iS04Z0xkg3p82Rh4imfGwCZLD7aH48ZC0SA0MRgRkCeAZCLg">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
