
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Investigating the role of FOX gene family in development and stress response in Labeo rohita: A multi-faceted analysis of phylogeny and genome characterization - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A0C38AF1CA9305A0C3005EE143BE.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosone">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370200/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS One">
<meta name="citation_title" content="Investigating the role of FOX gene family in development and stress response in Labeo rohita: A multi-faceted analysis of phylogeny and genome characterization">
<meta name="citation_author" content="Saima Naz">
<meta name="citation_author_institution" content="Department of Zoology, The Government Sadiq College Women University, Bahawalpur, Pakistan">
<meta name="citation_author" content="Urwah Ishaque">
<meta name="citation_author_institution" content="Department of Zoology, The Government Sadiq College Women University, Bahawalpur, Pakistan">
<meta name="citation_author" content="Ahmad Manan Mustafa Chatha">
<meta name="citation_author_institution" content="Department of Entomology, Faculty of Agriculture and Environment, The Islamia University of Bahawalpur, Punjab, Pakistan">
<meta name="citation_author" content="Muhammad Farooq">
<meta name="citation_author_institution" content="Department of Soil, Plant and Food science, University of Bari, Aldo Moro, Italy">
<meta name="citation_author" content="Qudrat Ullah">
<meta name="citation_author_institution" content="Department of Theriogenology, Cholistan University of Veterinary and Animal Sciences, Bahawalpur, Pakistan">
<meta name="citation_author" content="Shabana Naz">
<meta name="citation_author_institution" content="Department of Zoology, Government College University Faisalabad, Pakistan">
<meta name="citation_author" content="Marco Ragni">
<meta name="citation_author_institution" content="Department of Soil, Plant and Food science, University of Bari, Aldo Moro, Italy">
<meta name="citation_author" content="Ibrahim Alhidary">
<meta name="citation_author_institution" content="Department of Animal Produciton, College of Food and Agriculture Science, King Saud University, Riyadh, Saudi Arabia">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="20">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0323740">
<meta name="citation_doi" content="10.1371/journal.pone.0323740">
<meta name="citation_pmid" content="40839711">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370200/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370200/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370200/pdf/pone.0323740.pdf">
<meta name="description" content="The forkhead box (FOX) gene family of transcription factors regulates muscle development, immune responses, and metabolic processes across species. Despite extensive studies on FOX genes in other organisms, their evolutionary and functional roles in ...">
<meta name="og:title" content="Investigating the role of FOX gene family in development and stress response in Labeo rohita: A multi-faceted analysis of phylogeny and genome characterization">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The forkhead box (FOX) gene family of transcription factors regulates muscle development, immune responses, and metabolic processes across species. Despite extensive studies on FOX genes in other organisms, their evolutionary and functional roles in ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370200/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370200">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pone.0323740"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pone.0323740.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370200%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370200/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370200/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370200/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosone.png" alt="PLOS One logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS One" title="Link to PLOS One" shape="default" href="https://doi.org/10.1371/journal.pone.0323740" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS One</button></div>. 2025 Aug 21;20(8):e0323740. doi: <a href="https://doi.org/10.1371/journal.pone.0323740" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0323740</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20One%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20One%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Investigating the role of FOX gene family in development and stress response in <em>Labeo rohita</em>: A multi-faceted analysis of phylogeny and genome characterization</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Naz%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Saima Naz</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Saima Naz</span></h3>
<div class="p">
<sup>1</sup>Department of Zoology, The Government Sadiq College Women University, Bahawalpur, Pakistan</div>
<div>Conceptualization, Data curation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Naz%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Saima Naz</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ishaque%20U%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Urwah Ishaque</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Urwah Ishaque</span></h3>
<div class="p">
<sup>1</sup>Department of Zoology, The Government Sadiq College Women University, Bahawalpur, Pakistan</div>
<div>Investigation, Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ishaque%20U%22%5BAuthor%5D" class="usa-link"><span class="name western">Urwah Ishaque</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chatha%20AMM%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Ahmad Manan Mustafa Chatha</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Ahmad Manan Mustafa Chatha</span></h3>
<div class="p">
<sup>2</sup>Department of Entomology, Faculty of Agriculture and Environment, The Islamia University of Bahawalpur, Punjab, Pakistan</div>
<div>Investigation, Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chatha%20AMM%22%5BAuthor%5D" class="usa-link"><span class="name western">Ahmad Manan Mustafa Chatha</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Farooq%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Muhammad Farooq</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Muhammad Farooq</span></h3>
<div class="p">
<sup>3</sup>Department of Soil, Plant and Food science, University of Bari, Aldo Moro, Italy</div>
<div>Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Farooq%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Muhammad Farooq</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ullah%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Qudrat Ullah</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Qudrat Ullah</span></h3>
<div class="p">
<sup>4</sup>Department of Theriogenology, Cholistan University of Veterinary and Animal Sciences, Bahawalpur, Pakistan</div>
<div>Validation, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ullah%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qudrat Ullah</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Naz%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Shabana Naz</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Shabana Naz</span></h3>
<div class="p">
<sup>5</sup>Department of Zoology, Government College University Faisalabad, Pakistan</div>
<div>Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Naz%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shabana Naz</span></a>
</div>
</div>
<sup>5,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ragni%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Marco Ragni</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Marco Ragni</span></h3>
<div class="p">
<sup>3</sup>Department of Soil, Plant and Food science, University of Bari, Aldo Moro, Italy</div>
<div>Data curation, Formal analysis, Methodology, Software</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ragni%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marco Ragni</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alhidary%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Ibrahim Alhidary</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Ibrahim Alhidary</span></h3>
<div class="p">
<sup>6</sup>Department of Animal Produciton, College of Food and Agriculture Science, King Saud University, Riyadh, Saudi Arabia</div>
<div>Resources, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alhidary%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ibrahim Alhidary</span></a>
</div>
</div>
<sup>6</sup>
</div>
<div class="cg p">Editor: <span class="name western">Bijay Kumar Behera</span><sup>7</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Department of Zoology, The Government Sadiq College Women University, Bahawalpur, Pakistan</div>
<div id="aff002">
<sup>2</sup>Department of Entomology, Faculty of Agriculture and Environment, The Islamia University of Bahawalpur, Punjab, Pakistan</div>
<div id="aff003">
<sup>3</sup>Department of Soil, Plant and Food science, University of Bari, Aldo Moro, Italy</div>
<div id="aff004">
<sup>4</sup>Department of Theriogenology, Cholistan University of Veterinary and Animal Sciences, Bahawalpur, Pakistan</div>
<div id="aff005">
<sup>5</sup>Department of Zoology, Government College University Faisalabad, Pakistan</div>
<div id="aff006">
<sup>6</sup>Department of Animal Produciton, College of Food and Agriculture Science, King Saud University, Riyadh, Saudi Arabia</div>
<div id="edit1">
<sup>7</sup>Rani Lakshmi Bai Central Agricultural University, INDIA</div>
<div class="author-notes p">
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>drshabananaz@gcuf.edu.pk</span></p>
</div>
<div class="fn" id="coi001"><p><strong>Competing Interests: </strong>The authors have declared that no competing interests exist.</p></div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Saima Naz</span></strong>: <span class="role">Conceptualization, Data curation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Urwah Ishaque</span></strong>: <span class="role">Investigation, Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ahmad Manan Mustafa Chatha</span></strong>: <span class="role">Investigation, Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Muhammad Farooq</span></strong>: <span class="role">Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Qudrat Ullah</span></strong>: <span class="role">Validation, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Shabana Naz</span></strong>: <span class="role">Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Marco Ragni</span></strong>: <span class="role">Data curation, Formal analysis, Methodology, Software</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ibrahim Alhidary</span></strong>: <span class="role">Resources, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Bijay Kumar Behera</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 30; Accepted 2025 Apr 14; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Naz et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370200  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40839711/" class="usa-link">40839711</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The forkhead box (FOX) gene family of transcription factors regulates muscle development, immune responses, and metabolic processes across species. Despite extensive studies on FOX genes in other organisms, their evolutionary and functional roles in <em>Labeo rohita</em>, an economically and ecologically important freshwater fish, remain unclear. Owing to its unique physiological and ecological traits, <em>L. rohita</em> is an ideal model for exploring these roles. Here, we present the first computational analysis of the FOX gene family in <em>L. rohita</em>, identifying 21 FOX genes. Physicochemical analysis revealed that most FOX proteins have a basic nature except for FOX A3, D3, I2, O1, O3, O4, P1, and P2. Instability index analysis indicated that all FOX proteins are unstable (values &gt; 40), while hydrophobicity assessment showed that except FOX O1, all proteins are hydrophobic. Phylogenetic analysis grouped FOX homologs into 11 major clades with other vertebrates. All proteins exhibited structural homogeneity by sharing the Forkhead Box domain. Gene structure comparisons revealed seven duplicated pairs, and Circos analysis demonstrated organization into 20 clusters. This study highlights the critical roles of FOX genes and fills a significant knowledge gap, providing a foundation for future functional and phylogenomic studies with implications for aquaculture and evolutionary biology.</p></section><section id="sec001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>The Forkhead box (FOX) gene family is known for its versatility and evolutionary conservation among transcription factors, distinguished by a unique forkhead or winged-helix domain [<a href="#pone.0323740.ref001" class="usa-link" aria-describedby="pone.0323740.ref001">1</a>]. Since its identification in <em>Drosophila melanogaster</em> in the early 1990s, interest in FOX genes has surged, highlighting their essential regulatory functions across various species, including invertebrates, vertebrates, and humans [<a href="#pone.0323740.ref002" class="usa-link" aria-describedby="pone.0323740.ref002">2</a>]. These genes are crucial for regulating gene expression related to key biological processes such as cell proliferation, differentiation, apoptosis, and metabolism [<a href="#pone.0323740.ref003" class="usa-link" aria-describedby="pone.0323740.ref003">3</a>]. In metazoans, the FOX family plays a significant role in early embryonic development, tissue specification, and organogenesis, underscoring its importance in both vertebrate and invertebrate systems [<a href="#pone.0323740.ref004" class="usa-link" aria-describedby="pone.0323740.ref004">4</a>].</p>
<p>FOX genes are characterized by their distinct helix-turn-helix DNA binding domain, enabling them to attach to specific DNA sequences and regulate gene networks crucial for development and maintaining homeostasis [<a href="#pone.0323740.ref005" class="usa-link" aria-describedby="pone.0323740.ref005">5</a>]. The FOX family is categorized into subclasses labeled with letters (A to S), determined by sequence similarities found within the forkhead domain [<a href="#pone.0323740.ref006" class="usa-link" aria-describedby="pone.0323740.ref006">6</a>]. In the last twenty years, significant research has concentrated on the evolutionary divergence of FOX genes and their functions in developmental biology. For instance, investigations in humans and model organisms like mice and zebrafish have shown that FOX genes play crucial roles in numerous processes, including neural development, organogenesis, and the regulation of immune responses. These results indicate that the FOX gene family acts as a master regulator of various developmental and physiological pathways [<a href="#pone.0323740.ref007" class="usa-link" aria-describedby="pone.0323740.ref007">7</a>].</p>
<p>In zebrafish (<em>Danio rerio</em>), a key model organism for investigating vertebrate development, the FOX family includes 64 isoforms spanning several subfamilies, such as FOX A, FOX B, FOX C, FOX D, and FOX P. Research involving zebrafish has offered valuable insights into the varied functions of these genes during early development and neurogenesis. For instance, FOX D3 is crucial for the differentiation of neural crest cells, while FOX P2 is vital for speech and language development in humans and the regulation of neural circuits in vertebrates [<a href="#pone.0323740.ref008" class="usa-link" aria-describedby="pone.0323740.ref008">8</a>]. The role of FOX genes in stress response, especially concerning oxidative stress and environmental adaptations, has garnered increased interest in recent years. For example, Zhang et al. (2023) performed a thorough analysis of the Corsac FOX (<em>Vulpes corsac</em>) genome, uncovering genes linked to stress response mechanisms [<a href="#pone.0323740.ref009" class="usa-link" aria-describedby="pone.0323740.ref009">9</a>]. Despite these advancements, the exact molecular mechanisms and target genes regulated by FOX proteins are still not well understood, especially in non-model species like teleost fish.</p>
<p><em>Labeo rohita</em>, commonly referred to as rohu, is an essential freshwater species within the Indian major carps (IMCs) group and plays a crucial role in aquaculture throughout South Asia. As a widely cultivated species, <em>L. rohita</em> significantly contributes to food security and the livelihoods of millions in the region [<a href="#pone.0323740.ref010" class="usa-link" aria-describedby="pone.0323740.ref010">10</a>]. However, similar to other fish species, <em>L. rohita</em> is vulnerable to environmental stressors like temperature fluctuations, hypoxia, and water pollution, which can negatively impact its growth, survival, and reproduction [<a href="#pone.0323740.ref011" class="usa-link" aria-describedby="pone.0323740.ref011">11</a>]. Gaining insights into the genetic and molecular foundations of stress response and adaptation in <em>L. rohita</em> is essential for creating strategies to improve its resilience and productivity in the face of changing environmental conditions. However, there is still a considerable gap in understanding the regulatory networks that govern stress responses in this species.</p>
<p>Considering the crucial roles of FOX genes in regulating development, stress responses, and immune functions in other vertebrates, it is vital to explore their functions in <em>L. rohita</em>. The FOX gene family in <em>L. rohita</em>, similar to other teleost species, likely plays a key role in facilitating developmental processes and responses to environmental challenges. Research in teleost fish has shown that members of the FOX gene family, such as FOX O and FOX A, are involved in pathways that regulate cellular responses to oxidative stress and promote longevity [<a href="#pone.0323740.ref012" class="usa-link" aria-describedby="pone.0323740.ref012">12</a>]. However, comprehensive computational studies of the FOX gene family in <em>L. rohita</em> have yet to be performed, creating a significant gap in our understanding of the genetic regulation of essential physiological processes in this species.</p>
<p>The present study focuses on the comprehensive computational analysis of the FOX gene family in <em>L. rohita.</em> By utilizing bioinformatics tools and comparative genomics methods, we aim to identify and characterize the FOX genes in <em>L. rohita</em>, shedding light on their evolutionary relationships, molecular structures, and potential regulatory functions. The main objectives of this research include: (1) identifying the complete set of FOX genes in the <em>L. rohita</em> genome, (2) conducting phylogenetic analysis to explore their evolutionary relationships with FOX genes in other vertebrates, (3) performing motif and domain analysis to uncover structural conservation and divergence, and (4) predicting the transcription factor binding sites that regulate the expression of FOX genes under various physiological conditions.</p>
<p>This study will also investigate the role of FOX genes in stress responses, emphasizing their involvement in pathways related to oxidative stress, immune regulation, and apoptosis. By integrating gene expression data with in-silico analyses, we aim to explore the transcriptional regulation of FOX genes in <em>L. rohita</em> under stress conditions such as hypoxia and temperature fluctuations. Understanding the molecular mechanisms through which FOX genes mediate stress responses will provide insights into how <em>L. rohita</em> adapts to environmental stressors and could inform the development of selective breeding programs designed to enhance stress resilience in aquaculture.</p>
<p>Moreover, the comparative genomics aspect of this study will enable us to identify both evolutionary conservation and species-specific adaptations of FOX genes in <em>L. rohita</em>. This will contribute to a broader understanding of the functional diversification of FOX genes in teleosts and their role in vertebrate evolution. The findings from this research will not only enhance our knowledge of the FOX gene family in <em>L. rohita</em> but will also establish a framework for future studies on the molecular mechanisms governing development and stress responses in other economically significant fish species.</p>
<p>This comprehensive computational analysis of the FOX gene family in <em>L. rohita</em> aims to illuminate their evolutionary history, molecular functions, and regulatory roles. By investigating the FOX genes in this species, we seek to fill the knowledge gap regarding the genetic factors influencing development, stress response, and immune function in <em>L. rohita</em>. The results of this research have the potential to guide strategies for improving the growth, health, and resilience of <em>L. rohita</em> in aquaculture, contributing to more sustainable and efficient fish farming practices in South Asia. Furthermore, computational analysis of the FOX gene family in <em>L. rohita</em> offers a valuable opportunity to explore the genetic basis of development and environmental adaptation in this important aquaculture species. By elucidating the molecular mechanisms through which FOX genes regulate key physiological processes, this study will deepen our understanding of the developmental biology and stress physiology of <em>L. rohita</em> and other teleost fish. The insights gained from this research will have broader implications for vertebrate genomics and the application of molecular genetics in aquaculture and fisheries management.</p></section><section id="sec002"><h2 class="pmc_sec_title">2. Results</h2>
<section id="sec003"><h3 class="pmc_sec_title">2.1. Analysis of the Phylogenetic Relationships in the FOX Gene Superfamily</h3>
<p>To explore the evolutionary history of the FOX gene superfamily, a molecular phylogenetic analysis was performed. In total, 94 amino acid sequences from <em>Labeo rohita</em> (Rohu), <em>Oreochromis niloticus</em> (Nile tilapia), <em>Ctenopharyngodon idella</em> (Grass carp), <em>Danio rerio</em> (Zebrafish), and <em>Homo sapiens</em> (Human) were analyzed. These sequences were organized into 11 major clades: FOX H, FOX M, FOX P, FOX O, FOX F, FOX L, FOX G, FOX D, FOX C, FOX I, and FOX A, based on the percentage of homologous gene sequences (<a href="#pone.0323740.g001" class="usa-link">Figure 1</a>). The results indicated that the FOX gene superfamily in <em>L. rohita</em> showed a higher degree of sequence similarity with <em>D. rerio</em> and <em>C. idella</em> compared to other vertebrate species (<a href="#pone.0323740.g001" class="usa-link">Figure 1</a>).</p>
<figure class="fig xbox font-sm" id="pone.0323740.g001"><h4 class="obj_head">Fig 1. Phylogenetic tree of the FOX gene superfamily based on 94 amino acid sequences from five species (<em>L. rohita</em>, <em>O. niloticus</em>, <em>C. idella</em>, <em>D. rerio</em>, and <em>H. sapiens</em>), clustering into 11 clades.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370200_pone.0323740.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e71/12370200/c0bfc5879766/pone.0323740.g001.jpg" loading="lazy" height="720" width="731" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0323740.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec004"><h3 class="pmc_sec_title">2.2. Characterization of the Structure of FOX Genes Superfamily in <em>L. rohita</em>
</h3>
<p>To investigate the structural characteristics of the FOX gene superfamily in <em>L. rohita</em>, we conducted an analysis of the evolutionary relationships within the FOX gene superfamily, as well as an examination of motifs, conserved regions, and gene structures (<a href="#pone.0323740.g002" class="usa-link">Figure 2</a>). We identified ten conserved motifs in the FOX genes of <em>L. rohita</em> through MEME analysis, with MEME-6 containing the highest number of amino acids (50). This motif was classified as the Forkhead Box protein domain using InterPro (<a href="#pone.0323740.g002" class="usa-link">Figure 2B</a>; <a href="#pone.0323740.t001" class="usa-link">Table 1</a>). Our findings were further supported by a comparison with the NCBI CDD database (<a href="#pone.0323740.g002" class="usa-link">Figure 2C</a>). Additionally, the Forkhead Box protein domain was present in all homologs of the FOX gene superfamily. The gene structure analysis revealed significant variations in the arrangement of exons, introns, and both downstream and upstream untranslated regions (UTRs) (<a href="#pone.0323740.g002" class="usa-link">Figure 2D</a>).</p>
<figure class="fig xbox font-sm" id="pone.0323740.g002"><h4 class="obj_head">Fig 2. Structural Analysis of the FOX gene superfamily: (A) Phylogenetic tree, (B) Motifs, (C) Domains, and (D) Gene structure in <em>L. rohita.</em>
</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370200_pone.0323740.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e71/12370200/b09f6e0f53c1/pone.0323740.g002.jpg" loading="lazy" height="495" width="778" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0323740.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><section class="tw xbox font-sm" id="pone.0323740.t001"><h4 class="obj_head">Table 1. Ten highly conserved motifs identified within the FOX gene family of <em>L. rohita.</em>
</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">MEME Motif</th>
<th align="left" rowspan="1" colspan="1">Amino Acid Sequence</th>
<th align="left" rowspan="1" colspan="1">Width</th>
<th align="left" rowspan="1" colspan="1">InterPro Domain</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">PYSYIALITMAIQNSPEKRLTLSEIYQWIMDRFPYYRDNK</td>
<td align="left" rowspan="1" colspan="1">40</td>
<td align="left" rowspan="1" colspan="1">Forkhead Box Protein</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">DCFVKVPREPGKPGKGNYWTLDPNSEBMFENGSFRRRRKRF</td>
<td align="left" rowspan="1" colspan="1">41</td>
<td align="left" rowspan="1" colspan="1">Forkhead Box Protein</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">WQNSIRHNLSL</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">Forkhead Box Protein</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">RVQNEGTGKSSWWMLNPEGGKSGKAPRRRAASMDNSSKLJK</td>
<td align="left" rowspan="1" colspan="1">41</td>
<td align="left" rowspan="1" colspan="1">Forkhead Box Protein</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">HKCFVRVENVKGAVWTVDELEFQKRRPQKI</td>
<td align="left" rowspan="1" colspan="1">30</td>
<td align="left" rowspan="1" colspan="1">Forkhead Box Protein</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">VFEDFGQFLKHLNNEHALDDRSTAQCRVQMQVVQQLEJQLSKERERLQAM</td>
<td align="left" rowspan="1" colspan="1">50</td>
<td align="left" rowspan="1" colspan="1">Forkhead Box Protein</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">IDPDFEPQSRPRSCTWPLPRPE</td>
<td align="left" rowspan="1" colspan="1">22</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">DDFDAWTAFRPRTSSNASTLSGRLSPI</td>
<td align="left" rowspan="1" colspan="1">27</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">DPHYSFNHPFSINNLM</td>
<td align="left" rowspan="1" colspan="1">16</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">FDCDVEAIJHBDLMDGEGLDFNFD</td>
<td align="left" rowspan="1" colspan="1">24</td>
<td align="left" rowspan="1" colspan="1">Forkhead Box Protein</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0323740.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec005"><h3 class="pmc_sec_title">2.3. Physicochemical Analysis of FOX Genes in <em>L. rohita</em>
</h3>
<p>The physicochemical properties of the FOX gene family in <em>L. rohita</em> were assessed, including chromosome number, amino acid count, exon number for each peptide, isoelectric point (pI), molecular weight (MW), instability index (II), grand average of hydropathicity index (GRAVY), and aliphatic index (AI), as detailed in <a href="#pone.0323740.t002" class="usa-link">Table 2</a>. The molecular weights of the FOX proteins in <em>L. rohita</em> ranged from 32705.96 to 119782.93 Da, while the pI values varied from 4.81 to 9.50. The aliphatic index values exceeded 41.83, indicating that all FOX genes in <em>L. rohita</em> possess thermostable characteristics. Most proteins exhibited basic properties, except for FOX A3, FOX D3, FOX I2, FOX O1, FOX O3, FOX O4, FOX P1, and FOX P2. Based on the instability index, all proteins were deemed unstable, with values exceeding 40 (<a href="#pone.0323740.t002" class="usa-link">Table 2</a>). Additionally, except for FOX O1, all FOX proteins were hydrophobic, reflected by their lower GRAVY values (<a href="#pone.0323740.t002" class="usa-link">Table 2</a>). The genomic properties of FOX genes were also evaluated such as gene length (nt), chromosome length (bp), gene location on chromosome (start), gene location on chromosome (end), and the direction of strand.</p>
<section class="tw xbox font-sm" id="pone.0323740.t002"><h4 class="obj_head">Table 2. Physicochemical properties of the FOX gene superfamily in <em>L. rohita.</em>
</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Gene name</th>
<th align="left" rowspan="1" colspan="1">Chr. No</th>
<th align="left" rowspan="1" colspan="1">Exon count</th>
<th align="left" rowspan="1" colspan="1">Molecular Weight (D)</th>
<th align="left" rowspan="1" colspan="1">Amino acid</th>
<th align="left" rowspan="1" colspan="1">Isoelectric point</th>
<th align="left" rowspan="1" colspan="1">Instability index</th>
<th align="left" rowspan="1" colspan="1">Aliphatic index</th>
<th align="left" rowspan="1" colspan="1">GRAVY</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">FOXA1</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">46614.79</td>
<td align="left" rowspan="1" colspan="1">428</td>
<td align="left" rowspan="1" colspan="1">9.09</td>
<td align="left" rowspan="1" colspan="1">65.14</td>
<td align="left" rowspan="1" colspan="1">46.36</td>
<td align="left" rowspan="1" colspan="1">0.691</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXA2</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">45057.48</td>
<td align="left" rowspan="1" colspan="1">409</td>
<td align="left" rowspan="1" colspan="1">8.8</td>
<td align="left" rowspan="1" colspan="1">67.78</td>
<td align="left" rowspan="1" colspan="1">41.83</td>
<td align="left" rowspan="1" colspan="1">0.711</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXA3</td>
<td align="left" rowspan="1" colspan="1">18</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">47543.6</td>
<td align="left" rowspan="1" colspan="1">435</td>
<td align="left" rowspan="1" colspan="1">6.99</td>
<td align="left" rowspan="1" colspan="1">69.94</td>
<td align="left" rowspan="1" colspan="1">55.2</td>
<td align="left" rowspan="1" colspan="1">0.695</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXC1</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">54597.25</td>
<td align="left" rowspan="1" colspan="1">495</td>
<td align="left" rowspan="1" colspan="1">8.97</td>
<td align="left" rowspan="1" colspan="1">61.54</td>
<td align="left" rowspan="1" colspan="1">51.07</td>
<td align="left" rowspan="1" colspan="1">0.842</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXD1</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">37852.56</td>
<td align="left" rowspan="1" colspan="1">347</td>
<td align="left" rowspan="1" colspan="1">7.23</td>
<td align="left" rowspan="1" colspan="1">68.89</td>
<td align="left" rowspan="1" colspan="1">70.61</td>
<td align="left" rowspan="1" colspan="1">0.452</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXD3</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">40412.44</td>
<td align="left" rowspan="1" colspan="1">327</td>
<td align="left" rowspan="1" colspan="1">5.28</td>
<td align="left" rowspan="1" colspan="1">64.36</td>
<td align="left" rowspan="1" colspan="1">73.47</td>
<td align="left" rowspan="1" colspan="1">0.409</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXF1</td>
<td align="left" rowspan="1" colspan="1">18</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">41556.67</td>
<td align="left" rowspan="1" colspan="1">381</td>
<td align="left" rowspan="1" colspan="1">9.16</td>
<td align="left" rowspan="1" colspan="1">65.46</td>
<td align="left" rowspan="1" colspan="1">50.47</td>
<td align="left" rowspan="1" colspan="1">0.604</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXF2</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">119782.93</td>
<td align="left" rowspan="1" colspan="1">1087</td>
<td align="left" rowspan="1" colspan="1">9.33</td>
<td align="left" rowspan="1" colspan="1">60.16</td>
<td align="left" rowspan="1" colspan="1">64.29</td>
<td align="left" rowspan="1" colspan="1">0.43</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXG1</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">36535.29</td>
<td align="left" rowspan="1" colspan="1">335</td>
<td align="left" rowspan="1" colspan="1">9.48</td>
<td align="left" rowspan="1" colspan="1">62.54</td>
<td align="left" rowspan="1" colspan="1">66.69</td>
<td align="left" rowspan="1" colspan="1">0.577</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXH1</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">51534.91</td>
<td align="left" rowspan="1" colspan="1">472</td>
<td align="left" rowspan="1" colspan="1">8.59</td>
<td align="left" rowspan="1" colspan="1">52.64</td>
<td align="left" rowspan="1" colspan="1">61.19</td>
<td align="left" rowspan="1" colspan="1">0.636</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXI1</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">42180.51</td>
<td align="left" rowspan="1" colspan="1">381</td>
<td align="left" rowspan="1" colspan="1">7.3</td>
<td align="left" rowspan="1" colspan="1">52.06</td>
<td align="left" rowspan="1" colspan="1">56.14</td>
<td align="left" rowspan="1" colspan="1">0.745</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXI2</td>
<td align="left" rowspan="1" colspan="1">13</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">41648.74</td>
<td align="left" rowspan="1" colspan="1">383</td>
<td align="left" rowspan="1" colspan="1">6.83</td>
<td align="left" rowspan="1" colspan="1">50.39</td>
<td align="left" rowspan="1" colspan="1">60.23</td>
<td align="left" rowspan="1" colspan="1">0.646</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXL1</td>
<td align="left" rowspan="1" colspan="1">18</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">40102.14</td>
<td align="left" rowspan="1" colspan="1">359</td>
<td align="left" rowspan="1" colspan="1">9.41</td>
<td align="left" rowspan="1" colspan="1">41.98</td>
<td align="left" rowspan="1" colspan="1">60.14</td>
<td align="left" rowspan="1" colspan="1">0.811</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXL2</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">32705.96</td>
<td align="left" rowspan="1" colspan="1">289</td>
<td align="left" rowspan="1" colspan="1">9.09</td>
<td align="left" rowspan="1" colspan="1">61.44</td>
<td align="left" rowspan="1" colspan="1">50.07</td>
<td align="left" rowspan="1" colspan="1">0.805</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXM1</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">69522.1</td>
<td align="left" rowspan="1" colspan="1">628</td>
<td align="left" rowspan="1" colspan="1">8.76</td>
<td align="left" rowspan="1" colspan="1">65.49</td>
<td align="left" rowspan="1" colspan="1">68.46</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXO1</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">69520.16</td>
<td align="left" rowspan="1" colspan="1">653</td>
<td align="left" rowspan="1" colspan="1">6.13</td>
<td align="left" rowspan="1" colspan="1">57.93</td>
<td align="left" rowspan="1" colspan="1">56.08</td>
<td align="left" rowspan="1" colspan="1">−0.614</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXO3</td>
<td align="left" rowspan="1" colspan="1">20</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">68891.7</td>
<td align="left" rowspan="1" colspan="1">646</td>
<td align="left" rowspan="1" colspan="1">4.81</td>
<td align="left" rowspan="1" colspan="1">67.03</td>
<td align="left" rowspan="1" colspan="1">66.19</td>
<td align="left" rowspan="1" colspan="1">0.563</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXO4</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">65256.46</td>
<td align="left" rowspan="1" colspan="1">626</td>
<td align="left" rowspan="1" colspan="1">5.18</td>
<td align="left" rowspan="1" colspan="1">66.08</td>
<td align="left" rowspan="1" colspan="1">59.42</td>
<td align="left" rowspan="1" colspan="1">0.566</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXP1</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">25</td>
<td align="left" rowspan="1" colspan="1">73828.85</td>
<td align="left" rowspan="1" colspan="1">659</td>
<td align="left" rowspan="1" colspan="1">6.64</td>
<td align="left" rowspan="1" colspan="1">62.42</td>
<td align="left" rowspan="1" colspan="1">74.82</td>
<td align="left" rowspan="1" colspan="1">0.69</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXP2</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">21</td>
<td align="left" rowspan="1" colspan="1">67076.64</td>
<td align="left" rowspan="1" colspan="1">604</td>
<td align="left" rowspan="1" colspan="1">6.17</td>
<td align="left" rowspan="1" colspan="1">66.11</td>
<td align="left" rowspan="1" colspan="1">74.11</td>
<td align="left" rowspan="1" colspan="1">0.638</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXP3</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">13</td>
<td align="left" rowspan="1" colspan="1">50113.89</td>
<td align="left" rowspan="1" colspan="1">439</td>
<td align="left" rowspan="1" colspan="1">9.5</td>
<td align="left" rowspan="1" colspan="1">46.53</td>
<td align="left" rowspan="1" colspan="1">65.72</td>
<td align="left" rowspan="1" colspan="1">0.639</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0323740.t002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec006"><h3 class="pmc_sec_title">2.4. Collinearity analysis of FOX genes in <em>L. rohita</em> and <em>H. sapiens</em>
</h3>
<p>The collinearity analysis revealed that the majority of FOX genes in <em>L. rohita</em> are distributed randomly across chromosomes 1, 2, 10, 18, and 23, while in <em>H. sapiens</em>, they are located on chromosomes 2, 7, 11, 14, 20, and Y. Most FOX genes are situated at the terminal ends of the chromosomes, with the exception of those on chromosomes 10 and 23 in <em>L. rohita</em> and on chromosomes 7, 11, and 20 in <em>H. sapiens</em> (<a href="#pone.0323740.g003" class="usa-link">Figure 3</a>). The gray lines in the background represent collinear blocks within the genomes of <em>L. rohita</em> and <em>H. sapiens</em>, while the red lines highlight the syntenic FOX gene pairs.</p>
<figure class="fig xbox font-sm" id="pone.0323740.g003"><h4 class="obj_head">Fig 3. Syntenic relationships between FOX genes in <em>L. rohita</em> and <em>H. sapiens.</em>
</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370200_pone.0323740.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e71/12370200/2a3574bd88ff/pone.0323740.g003.jpg" loading="lazy" height="121" width="727" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0323740.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The gray lines in the background depict collinear blocks in the genomes of <em>L. rohita</em> and <em>H. sapiens</em>, while the red lines emphasize the syntenic pairs of FOX genes.</p></figcaption></figure></section><section id="sec007"><h3 class="pmc_sec_title">2.5. Duplication and localization of FOX genes in <em>L. rohita</em>
</h3>
<p>To gain a clearer understanding of the evolutionary history of the FOX gene family in <em>L. rohita</em>, we investigated duplication events (<a href="#pone.0323740.t003" class="usa-link">Table 3</a>). A total of seven duplicated gene pairs were identified, comprising segmental duplications (FOX I1/ FOX I2 and FOX D1/ FOX D3), tandem duplications (FOX A1/ FOX A2), and whole-genome duplications (FOX H1/ FOX M1, FOX F1/ FOX F2, FOX O1/ FOX O3, and FOX P1/ FOX P2) (<a href="#pone.0323740.t003" class="usa-link">Table 3</a>). Additionally, these gene pairs underwent a Ka/ Ks ratio analysis to compare non-synonymous and synonymous divergence rates, which helps estimate the protein-coding potential of genomic regions. The Ka/ Ks ratio results showed that none of the values exceeded 1, suggesting that these proteins are under positive selection. Furthermore, all the genes were found to be influenced by purifying selection. The gene pairs with the longest estimated duplication times include FOX D1- FOX D3 (70.19 million years ago), FOX A1- FOX A2 (49.85 million years ago), FOX I1- FOX I2 (42.39 million years ago), and FOX P1- FOX P2 (42.00 million years ago), while more recent duplications include FOX F1- FOX F2 (33.34 million years ago), FOX M1- FOX H1 (29.63 million years ago), and FOX O1- FOX O3 (29.53 million years ago) (<a href="#pone.0323740.t003" class="usa-link">Table 3</a>). Each gene is specifically located on chromosomes (<a href="#pone.0323740.g004" class="usa-link">Fig 4A</a>), highlighting the relationships between FOX gene pairs that dictate the size, location, and orientation of connected genomic components. The localization and duplication events of FOX genes are illustrated in <a href="#pone.0323740.g004" class="usa-link">Figs 4A</a> and <a href="#pone.0323740.g004" class="usa-link">4B</a>.</p>
<section class="tw xbox font-sm" id="pone.0323740.t003"><h4 class="obj_head">Table 3. Ka/ Ks ratio analysis for each gene pair within the FOX gene family of <em>L. rohita.</em>
</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Gene Pairs</th>
<th align="left" rowspan="1" colspan="1">Chromosomes</th>
<th align="left" rowspan="1" colspan="1">Duplication</th>
<th align="left" rowspan="1" colspan="1">Ka</th>
<th align="left" rowspan="1" colspan="1">Ks</th>
<th align="left" rowspan="1" colspan="1">Ka/Ks</th>
<th align="left" rowspan="1" colspan="1">Selection</th>
<th align="left" rowspan="1" colspan="1">Time (MYA)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">FOXA1-FOXA2</td>
<td align="left" rowspan="1" colspan="1">17/17</td>
<td align="left" rowspan="1" colspan="1">TD</td>
<td align="left" rowspan="1" colspan="1">0.3672</td>
<td align="left" rowspan="1" colspan="1">1.0966</td>
<td align="left" rowspan="1" colspan="1">0.334853183</td>
<td align="left" rowspan="1" colspan="1">Purifying Selection</td>
<td align="left" rowspan="1" colspan="1">49.85</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXI1-FOXI2</td>
<td align="left" rowspan="1" colspan="1">12/13</td>
<td align="left" rowspan="1" colspan="1">SD</td>
<td align="left" rowspan="1" colspan="1">0.2606</td>
<td align="left" rowspan="1" colspan="1">0.9325</td>
<td align="left" rowspan="1" colspan="1">0.279463807</td>
<td align="left" rowspan="1" colspan="1">Purifying Selection</td>
<td align="left" rowspan="1" colspan="1">42.39</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXD1-FOXD3</td>
<td align="left" rowspan="1" colspan="1">5/6</td>
<td align="left" rowspan="1" colspan="1">SD</td>
<td align="left" rowspan="1" colspan="1">1.1288</td>
<td align="left" rowspan="1" colspan="1">1.5442</td>
<td align="left" rowspan="1" colspan="1">0.730993395</td>
<td align="left" rowspan="1" colspan="1">Purifying Selection</td>
<td align="left" rowspan="1" colspan="1">70.19</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXH1-FOXM1</td>
<td align="left" rowspan="1" colspan="1">12/4</td>
<td align="left" rowspan="1" colspan="1">WGD</td>
<td align="left" rowspan="1" colspan="1">0.2762</td>
<td align="left" rowspan="1" colspan="1">0.6519</td>
<td align="left" rowspan="1" colspan="1">0.423684614</td>
<td align="left" rowspan="1" colspan="1">Purifying Selection</td>
<td align="left" rowspan="1" colspan="1">29.63</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXF1-FOXF2</td>
<td align="left" rowspan="1" colspan="1">18/2</td>
<td align="left" rowspan="1" colspan="1">WGD</td>
<td align="left" rowspan="1" colspan="1">0.2855</td>
<td align="left" rowspan="1" colspan="1">0.7335</td>
<td align="left" rowspan="1" colspan="1">0.389229721</td>
<td align="left" rowspan="1" colspan="1">Purifying Selection</td>
<td align="left" rowspan="1" colspan="1">33.34</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXO1-FOXO3</td>
<td align="left" rowspan="1" colspan="1">15/20</td>
<td align="left" rowspan="1" colspan="1">WGD</td>
<td align="left" rowspan="1" colspan="1">0.2691</td>
<td align="left" rowspan="1" colspan="1">0.6497</td>
<td align="left" rowspan="1" colspan="1">0.414191165</td>
<td align="left" rowspan="1" colspan="1">Purifying Selection</td>
<td align="left" rowspan="1" colspan="1">29.53</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXP1-FOXP2</td>
<td align="left" rowspan="1" colspan="1">6/4</td>
<td align="left" rowspan="1" colspan="1">WGD</td>
<td align="left" rowspan="1" colspan="1">0.272</td>
<td align="left" rowspan="1" colspan="1">0.9241</td>
<td align="left" rowspan="1" colspan="1">0.294340439</td>
<td align="left" rowspan="1" colspan="1">Purifying Selection</td>
<td align="left" rowspan="1" colspan="1">42.00</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0323740.t003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t003fn001"><p>TD: Tandem Duplication; SD: Segmental duplication; WGD: Whole genome duplication; Ka: non-Synonymous substitutions; Ks: synonymous substitutions; MYA: Millions of years ago.</p></div></div></section><figure class="fig xbox font-sm" id="pone.0323740.g004"><h4 class="obj_head">Fig 4. Illustration of FOX superfamily gene localization and duplication events.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370200_pone.0323740.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e71/12370200/bc160ee3f366/pone.0323740.g004.jpg" loading="lazy" height="833" width="733" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0323740.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) The positions of FOX genes across all 12 chromosomes of <em>L. rohita</em> are shown, with mega-bases (Mb) used as the scale on the left. The corresponding chromosomal numbers are indicated at the center of each bar. (B) A Circos plot depicts the FOX genes in the <em>L. rohita</em> genome, highlighting gene duplication events. Duplicated regions are shown in proximity due to the arrangement of the chromosomes, with segmentally duplicated genes connected by a direct communication pathway.</p></figcaption></figure></section><section id="sec008"><h3 class="pmc_sec_title">2.6. Prediction of Secondary and Tertiary Protein Structures</h3>
<p>The prediction of secondary and three-dimensional structures was performed for all FOX proteins identified in the <em>L. rohita</em> genome. PSIPRED analysis revealed that the secondary structure of FOX proteins comprises random coils, beta twists, helices, and extended strands (S1 Fig). The secondary structural features were further assessed using the Phyre2 online tool, as presented in <a href="#pone.0323740.t004" class="usa-link">Table 4</a>. The analysis indicated that the FOX gene structure has alpha helices constituting about 5–9%, beta sheets ranging from 0 to 6%. It was also revealed that the 2D structure of FOX proteins is also predominantly disordered (66–83%). Three-dimensional models for the FOX protein sequences were generated using Phyre2 (<a href="#pone.0323740.g005" class="usa-link">Fig 5</a>) and subsequently validated with the SWISS MODEL online tool (S2 Fig). Both methods consistently confirmed the accuracy and robustness of our 3D structural predictions.</p>
<section class="tw xbox font-sm" id="pone.0323740.t004"><h4 class="obj_head">Table 4. Predicted secondary structure characteristics of the FOX gene superfamily using Phyre2.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Sr. No</th>
<th align="left" rowspan="1" colspan="1">Name of Protein</th>
<th align="left" rowspan="1" colspan="1">Alpha Helix</th>
<th align="left" rowspan="1" colspan="1">Beta strand</th>
<th align="left" rowspan="1" colspan="1">Disorder</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">FOXA1</td>
<td align="left" rowspan="1" colspan="1">5%</td>
<td align="left" rowspan="1" colspan="1">1%</td>
<td align="left" rowspan="1" colspan="1">78%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">FOXA2</td>
<td align="left" rowspan="1" colspan="1">5%</td>
<td align="left" rowspan="1" colspan="1">1%</td>
<td align="left" rowspan="1" colspan="1">80%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">FOXA3</td>
<td align="left" rowspan="1" colspan="1">5%</td>
<td align="left" rowspan="1" colspan="1">1%</td>
<td align="left" rowspan="1" colspan="1">81%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">FOXC1</td>
<td align="left" rowspan="1" colspan="1">5%</td>
<td align="left" rowspan="1" colspan="1">1%</td>
<td align="left" rowspan="1" colspan="1">83%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">FOXD1</td>
<td align="left" rowspan="1" colspan="1">9%</td>
<td align="left" rowspan="1" colspan="1">2%</td>
<td align="left" rowspan="1" colspan="1">78%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">FOXD3</td>
<td align="left" rowspan="1" colspan="1">8%</td>
<td align="left" rowspan="1" colspan="1">1%</td>
<td align="left" rowspan="1" colspan="1">79%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">FOXF1</td>
<td align="left" rowspan="1" colspan="1">6%</td>
<td align="left" rowspan="1" colspan="1">2%</td>
<td align="left" rowspan="1" colspan="1">80%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">FOXF2</td>
<td align="left" rowspan="1" colspan="1">9%</td>
<td align="left" rowspan="1" colspan="1">6%</td>
<td align="left" rowspan="1" colspan="1">66%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">FOXG1</td>
<td align="left" rowspan="1" colspan="1">7%</td>
<td align="left" rowspan="1" colspan="1">2%</td>
<td align="left" rowspan="1" colspan="1">77%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">FOXH1</td>
<td align="left" rowspan="1" colspan="1">6%</td>
<td align="left" rowspan="1" colspan="1">2%</td>
<td align="left" rowspan="1" colspan="1">82%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">FOXI1</td>
<td align="left" rowspan="1" colspan="1">9%</td>
<td align="left" rowspan="1" colspan="1">1%</td>
<td align="left" rowspan="1" colspan="1">80%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">FOXI2</td>
<td align="left" rowspan="1" colspan="1">9%</td>
<td align="left" rowspan="1" colspan="1">0%</td>
<td align="left" rowspan="1" colspan="1">81%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">13</td>
<td align="left" rowspan="1" colspan="1">FOXL1</td>
<td align="left" rowspan="1" colspan="1">6%</td>
<td align="left" rowspan="1" colspan="1">2%</td>
<td align="left" rowspan="1" colspan="1">74%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">FOXL2</td>
<td align="left" rowspan="1" colspan="1">9%</td>
<td align="left" rowspan="1" colspan="1">2%</td>
<td align="left" rowspan="1" colspan="1">74%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">FOXM1</td>
<td align="left" rowspan="1" colspan="1">5%</td>
<td align="left" rowspan="1" colspan="1">2%</td>
<td align="left" rowspan="1" colspan="1">75%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">16</td>
<td align="left" rowspan="1" colspan="1">FOXO1</td>
<td align="left" rowspan="1" colspan="1">6%</td>
<td align="left" rowspan="1" colspan="1">2%</td>
<td align="left" rowspan="1" colspan="1">82%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">FOXO3</td>
<td align="left" rowspan="1" colspan="1">6%</td>
<td align="left" rowspan="1" colspan="1">2%</td>
<td align="left" rowspan="1" colspan="1">82%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">18</td>
<td align="left" rowspan="1" colspan="1">FOXO4</td>
<td align="left" rowspan="1" colspan="1">7%</td>
<td align="left" rowspan="1" colspan="1">1%</td>
<td align="left" rowspan="1" colspan="1">82%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">19</td>
<td align="left" rowspan="1" colspan="1">FOXP1</td>
<td align="left" rowspan="1" colspan="1">27%</td>
<td align="left" rowspan="1" colspan="1">1%</td>
<td align="left" rowspan="1" colspan="1">82%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">20</td>
<td align="left" rowspan="1" colspan="1">FOXP2</td>
<td align="left" rowspan="1" colspan="1">28%</td>
<td align="left" rowspan="1" colspan="1">1%</td>
<td align="left" rowspan="1" colspan="1">83%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">21</td>
<td align="left" rowspan="1" colspan="1">FOXP3</td>
<td align="left" rowspan="1" colspan="1">20%</td>
<td align="left" rowspan="1" colspan="1">8%</td>
<td align="left" rowspan="1" colspan="1">66%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0323740.t004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="pone.0323740.g005"><h4 class="obj_head">Fig 5. Three-dimensional structure of the FOX gene superfamily in <em>L. rohita</em> generated using Phyre2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370200_pone.0323740.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e71/12370200/edd1d34fbf5f/pone.0323740.g005.jpg" loading="lazy" height="1294" width="786" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0323740.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec009"><h3 class="pmc_sec_title">2.7. Scan PROSITE analysis</h3>
<p>PROSITE was utilized to analyze FOX proteins for functional and structural residues associated with ProRule and the PROSITE signature. This tool helps identify intradomain features, such as active sites, binding sites, and disulfide bridges. The accuracy of functional predictions was enhanced by integrating the sensitivity of profiles with the specificity of motif recognition. <a href="#pone.0323740.g006" class="usa-link">Fig 6</a> illustrates the graphical representation of matches found in FOX proteins and the predicted features within those matches. The domain analysis of FOX D1, FOX G1, FOX P2, and FOX P3 proteins from the PROSITE database revealed various profiles alongside the Fork_head domain. In contrast, the FOX F2 protein displayed an additional range known as ZF-THAP. Additionally, FOX L1, FOX O1, and FOX O3 exhibited different profiles associated with the Fork_head domain. Among the other homologs in the FOX superfamily, only a single Fork_head domain was identified at varying amino acid residues.</p>
<figure class="fig xbox font-sm" id="pone.0323740.g006"><h4 class="obj_head">Fig 6. Scan PROSITE identified intra-domain predictions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370200_pone.0323740.g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e71/12370200/7270f4b274ef/pone.0323740.g006.jpg" loading="lazy" height="699" width="759" alt="Fig 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0323740.g006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>This image represents all detected features and matches of the specified type. Pattern hits are displayed as thin colored bars without text, while hit profiles are represented as colored shapes accompanied by their corresponding PROSITE names.</p></figcaption></figure></section><section id="sec010"><h3 class="pmc_sec_title">2.8. Transcription factor binding sites (TFBSs) analysis</h3>
<p>Transcriptional regulation within cells relies on transcription factors binding to specific genomic sites. The FOX gene superfamily in <em>L. rohita</em> was analyzed for its transcription factor binding sites (TFBSs) based on four transcription factors. Three promoter sites—GATA-1, AP-1, and p53—and one suppressor site, YY1, were selected for this study. <a href="#pone.0323740.t005" class="usa-link">Table 5</a> illustrates the distribution of TFBSs within the FOX gene superfamily in <em>L. rohita</em>. The analysis revealed a complex regulatory environment, with the following total numbers of binding sites for key transcription factors: AP-1 had 58, GATA-1 had 86, p53 had 85, and YY1 had 172 binding sites. These results highlight the ability of FOX genes to engage with a variety of transcription factors, indicating their important roles in gene regulation and cellular functions.</p>
<section class="tw xbox font-sm" id="pone.0323740.t005"><h4 class="obj_head">Table 5. Transcription Factor Binding Sites (TFBSs) of FOX genes in <em>L. rohita</em>, highlighting the 4 TFBSs under study, total sites present, associated amino acid sequences and their localization in each gene.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Gene Name</th>
<th align="left" rowspan="1" colspan="1">TFBSs</th>
<th align="left" rowspan="1" colspan="1">Total Sites</th>
<th align="left" rowspan="1" colspan="1">Amino Acid Sequence</th>
<th align="left" rowspan="1" colspan="1">Amino Acid Localization</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">FOXA1</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">TAAAGTCA, TGACTAAT</td>
<td align="left" rowspan="1" colspan="1">520-527, 537-544</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">GGAGATAA,TTATCTGA, TTATCAAA, TTATCGAT</td>
<td align="left" rowspan="1" colspan="1">355-362, 436-443, 500-507, 778-785</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">GGGCTTT, GGGCCAT, TAGGCCC, GGGCATT, GGGCTGT</td>
<td align="left" rowspan="1" colspan="1">260-266, 294-300, 930-936, 1383-1389, 1464-1470</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">ATGGTT, AACCAT, ATGGGG, ATGGTC, GGCCAT, ATGGTG, TTCCAT</td>
<td align="left" rowspan="1" colspan="1">54-59, 115-120, 256-261, 270-275, 295-300, 337-342, 549-554</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXA2</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">GATAGTCA</td>
<td align="left" rowspan="1" colspan="1">865-872</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">TTATCTGG</td>
<td align="left" rowspan="1" colspan="1">1408-1415</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">GGGCGCA, GCAGCCC</td>
<td align="left" rowspan="1" colspan="1">670-676, 1151-1157</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">TACCAT, ACCCAT, ATGGAT, ATGGAT, ATGGAT</td>
<td align="left" rowspan="1" colspan="1">583-588, 631-636, 1198-1203, 1257-1262, 1269-1274</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXA3</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">TGACTGAT</td>
<td align="left" rowspan="1" colspan="1">937-944</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">GACGATAA, TTATCCGT</td>
<td align="left" rowspan="1" colspan="1">945-952, 1299-1306</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">GGGCTTT, CATGCCC, TAAGCCC, GGGCTTT, ACAGCCC, GGGCAAA, GGGCCTG, GGGCGGT</td>
<td align="left" rowspan="1" colspan="1">1-7, 702-708, 873-879, 1187-1193, 1285-1291, 1398-1404, 1437-1443, 1446-1452</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">ATGGCA, CTCCAT, AGCCAT, CGCCAT, TTCCAT, ATGGAC</td>
<td align="left" rowspan="1" colspan="1">160-165, 256-261, 267-272, 1250-1255, 1311-1316, 1361-1366</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXC1</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">TTATCATT, AATGATAA, GCAGATAA</td>
<td align="left" rowspan="1" colspan="1">158-165, 479-486, 527-534</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">GGGCACA, CATGCCC, GGGCGGA, TAGGCCC, GGGCTTT, TCTGCCC, GGGCTTT, GGGCTGG, GGGCTGG, ACGGCCC</td>
<td align="left" rowspan="1" colspan="1">25-31, 42-48, 242-248, 313-319, 712-718, 805-811, 820-826, 1035-1041, 1335-1341, 1397-1403</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">GTCCAT, ATGGTC, ATGGCA, AACCAT, TGCCAT, ATGGCA, ATGGTG, ACCCAT, ATGGGC, TCCCAT, ATGGAC, AGCCAT, ATGGTT, TCCCAT</td>
<td align="left" rowspan="1" colspan="1">86-91, 90-95, 524-529, 572-577, 605-610, 640-645, 667-672, 696-701, 818-823, 934-939, 978-983, 998-1003, 1293-1298, 1408-1413</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXD1</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">TGACTATT, TGACTTAT, TGACTGAT, TGACTTTT, TCGAGTCA, TGACTGTT</td>
<td align="left" rowspan="1" colspan="1">24-31, 844-851, 972-979, 993-1000, 1269-1276, 1353-1360</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">AATGATAAT, TCTGATAAT, TTTATCATT, TTTATCATG, TTTATCTCT</td>
<td align="left" rowspan="1" colspan="1">128-136, 194-202, 723-731, 1177-1185, 1416-1424</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">TAGGCCC, TTGGCCC, GGGCAAG, TGTGCCC, TCCGCCC</td>
<td align="left" rowspan="1" colspan="1">54-60, 680-686, 913-919, 926-932, 1490-1496</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">CTCCAT, ATGGTT, TTCCAT, ATGGCT, ATGGAA, ATGGGT</td>
<td align="left" rowspan="1" colspan="1">432-437, 479-484, 1002-1007, 1013-1018, 1077-1082, 1183-1188</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXD3</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">GCAAGTCA</td>
<td align="left" rowspan="1" colspan="1">1015-1022</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">AGAGATAAG, ATCGATAAG</td>
<td align="left" rowspan="1" colspan="1">349-357, 527-535</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">GGGCAGA, GGGCGAG</td>
<td align="left" rowspan="1" colspan="1">505-511, 1227-1233</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">ATGGTA, ATGGTG, TCCCAT, ATGGCT, TTCCAT, ATGGTC, ATCCAT</td>
<td align="left" rowspan="1" colspan="1">35-40, 444-449, 708-713, 837-842, 945-950, 1303-1308, 1358-1363</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXF1</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">GAAAGTCA, TGACTAAA, GTCAGTCA, TGACTTCA, TGACTTTC</td>
<td align="left" rowspan="1" colspan="1">187-194, 375-382, 613-620, 628-635, 826-833</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">GCTGATAAA, ACTGATAAC, AGAGATAAA, ACAGATAAG</td>
<td align="left" rowspan="1" colspan="1">3-11, 106-114, 342-350, 1480-1488</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">GGGCGCT, CTAGCCC, GGGCTTT, GGGCGGT</td>
<td align="left" rowspan="1" colspan="1">18-24, 551-557, 983-989, 1420-1426</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">AACCAT, GCCCAT, TTCCAT, ATGGGC, ATGGTG, GTCCAT, ATGGAA</td>
<td align="left" rowspan="1" colspan="1">78-83, 554-559, 731-736, 981-986, 996-1001, 1425-1430, 1474-1479</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXF2</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">TTTAGTCA, TGACTGCA</td>
<td align="left" rowspan="1" colspan="1">259-266, 912-919</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">ATTATCCCC, TTTATCTCC, TTTGATAAA, AATGATAAT, GTTATCCAT, CAAGATAAT, TTTATCCAG</td>
<td align="left" rowspan="1" colspan="1">321-329, 498-506, 523-531, 762-770, 854-862, 941-949, 1153-1161</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">GGGCTTT, GGGCGAT, GGGCTGA, ATTGCCC, ACAGCCC, TTCGCCC, TTCGCCC</td>
<td align="left" rowspan="1" colspan="1">39-45, 454-460, 490-496, 576-582, 1163-1169, 1207-1213, 1450-1456</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">ATCCAT, ATGGCC, ATGGTA, ATGGAA, CCCCAT, CGCCAT, CGCCAT</td>
<td align="left" rowspan="1" colspan="1">857-862, 996-1001, 1064-1069, 1118-1123,1216-1221, 1339-1344, 1421-1426</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXG1</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">TTCAGTCA, TGACTGGG</td>
<td align="left" rowspan="1" colspan="1">343-350, 1492-1499</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">TTTATCTGA, CTTATCTAA, GTTATCTCT, TTTATCACG, CTTATCAAA, ATTATCAAA</td>
<td align="left" rowspan="1" colspan="1">175-183, 562-570, 617-625, 801-809, 912-920, 1411-1419</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">CAGGCCC, GGGCCTT</td>
<td align="left" rowspan="1" colspan="1">214-220, 558-564</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">TTCCAT, CTCCAT, ATGGTG, ATGGTA, ATGGAT, AGCCAT</td>
<td align="left" rowspan="1" colspan="1">395-400, 646-651, 955-960, 1154-1159, 1384-1389, 1483-1488</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXH1</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">AAGAGTCA, ACTAGTCA, TCGAGTCA,ACTAGTCA, TGACTGTA, TGACTTTT</td>
<td align="left" rowspan="1" colspan="1">42-49, 55-62, 571-578, 983-990, 1036-1043,1293-1300</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">TTTGATAAC, TTTATCACA, ATTATCACA, GTTATCATT</td>
<td align="left" rowspan="1" colspan="1">123-131, 608-616, 616-624, 1475-1483</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">CTCCAT, ATGGTA, ATGGCT, TTCCAT, TGCCAT</td>
<td align="left" rowspan="1" colspan="1">467-472, 648-653, 1089-1094, 1110-1115, 1313-1318</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXI1</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">TTGAGTCA, TTGAGTCA, AAAAGTCA, TGTAGTCA</td>
<td align="left" rowspan="1" colspan="1">653-660, 665-672, 682-689, 1115-1122</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">AAAGATAAA, GATGATAAT, ATTGATAAA, CTTATCGCA</td>
<td align="left" rowspan="1" colspan="1">13-21, 501-509, 544-552, 1303-1311</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">ACTGCCC, GGGCTAT, CATGCCC, GCGGCCC, GGGCACT</td>
<td align="left" rowspan="1" colspan="1">356-362, 631-637, 1020-1026, 1413-1419, 1455-1461</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">AACCAT, ATGGGC, ATCCAT</td>
<td align="left" rowspan="1" colspan="1">392-397, 629-634, 1246-1251</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXI2</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">TGACTTAA, TGACTGAT, TGACTTTA</td>
<td align="left" rowspan="1" colspan="1">40-47, 299-306, 1350-1357</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">TTTATCCTC, TTTATCGTA, TCGGATAAT</td>
<td align="left" rowspan="1" colspan="1">6-14, 56-64, 859-867</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">TTAGCCC</td>
<td align="left" rowspan="1" colspan="1">1293-1299</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">ATCCAT, ATGGAA, ATGGTT, ATGGGA, ATGGCT, AACCAT, TGCCAT, TACCAT, ATGGTT</td>
<td align="left" rowspan="1" colspan="1">96-101, 259-264, 275-280, 524-529, 728-733, 852-857, 895-900, 936-941, 1164-1169</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXL1</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">TAGAGTCA</td>
<td align="left" rowspan="1" colspan="1">525-532</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">TTTATCCAG, ATCGATAAG, ATTGATAAA, GTGGATAAG</td>
<td align="left" rowspan="1" colspan="1">248-256, 399-407, 418-426, 1090-1098</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">GGTGCCC</td>
<td align="left" rowspan="1" colspan="1">1468-1474</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">TTCCAT,ACCCAT, ATGGAT, TACCAT, ATGGAT, ATGGAT, ATGGCA, ATGGGT, ATGGTA, ATGGAA, TACCAT,ATGGTT, ACCCAT, ATGGAT</td>
<td align="left" rowspan="1" colspan="1">212-217, 438-443, 452-457, 490-495, 589-594, 754-759, 792-797,831-836, 908-913, 928-933, 958-963, 998-1003, 1067-1072, 1257-1262</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXL2</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">GGCAGTCA, TGACTCTA</td>
<td align="left" rowspan="1" colspan="1">400-407, 732-739</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">TGTGATAAT, ACTGATAAG, TTTGATAAT, CTAGATAAT, TTTATCATG, TAAGATAAT, ATTATCATA, ATTATCCAC, AGAGATAAC</td>
<td align="left" rowspan="1" colspan="1">225-233, 239-247, 516-524, 737-745, 755-763, 995-1003, 1040-1048, 1202-1210, 1454-1462</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">ACTGCCC, AATGCCC, GGGCAAT, GGGCCTA, GGGCAGT, TTAGCCC, GGGCTGA</td>
<td align="left" rowspan="1" colspan="1">150-156, 171-177, 195-201, 368-374, 773-779, 1224-1230, 1316-1322</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">ATGGTG, CACCAT, GACCAT, ATGGAA, ATGGTT, ATCCAT, ATGGGA, ATGGAA, ATGGGT, ATGGCC, ATGGAC</td>
<td align="left" rowspan="1" colspan="1">82-87, 334-339, 476-481, 625-630, 640-645,690-695, 761-766, 809-814, 831-836, 899-904, 1079-1084</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXM1</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">TGACTGTC, TGACTATT, ACCAGTCA</td>
<td align="left" rowspan="1" colspan="1">63-70, 654-661, 1230-1237</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">TGGGATAAA, TTTATCGTA, GTGGATAAA</td>
<td align="left" rowspan="1" colspan="1">418-426, 728-736, 794-802</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">GGGCGCC, GGGCATT, GGGCGGG, GGGCTTG, ATGGCCC, ATCGCCC, TCCGCCC, GGGCTAG</td>
<td align="left" rowspan="1" colspan="1">473-479, 530-536, 1021-1027, 1026-1032, 1033-1039, 1358-1364,1425-1431, 1448-1454</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">18</td>
<td align="left" rowspan="1" colspan="1">AGCCAT, AGCCAT, ATGGTG, ATGGCA, GTCCAT, GTCCAT, ATGGGC, TGCCAT, ATGGTG, ATGGCG, GACCAT, CCCCAT, ATGGAA, ATGGCC, ATGGGT, CACCAT, GCCCAT, TGCCAT</td>
<td align="left" rowspan="1" colspan="1">154-159, 215-220, 376-381, 426-431, 445-450, 463-468, 471-476, 614-619, 777-782, 807-812, 886-891, 939-944, 1006-1011, 1033-1038, 1067-1072, 1389-1394, 1428-1433, 1441-1446</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXO1</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">TGACTCAA, CGTAGTCA, TGACTTCG, TGACTATT, TGCAGTCA</td>
<td align="left" rowspan="1" colspan="1">680-687, 827-834, 1236-1243, 1245-1252, 1330-1337</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">GTCGATAAA</td>
<td align="left" rowspan="1" colspan="1">1315-1323</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">ATTGCCC, ATAGCCC, CCCGCCC, GGGCAAA, GGGCTTC, GGGCGGT</td>
<td align="left" rowspan="1" colspan="1">74-80, 88-94, 371-377, 577-583, 607-613, 1473-1479</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">TACCAT,ATGGTT,GACCAT, ATGGCA, ATGGTC, AGCCAT</td>
<td align="left" rowspan="1" colspan="1">36-41, 40-45, 243-248, 776-781, 870-875, 1325-1330</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXO3</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">ACCAGTCA, TGACTGGT, TGACTTGT</td>
<td align="left" rowspan="1" colspan="1">443-450, 814-821, 1392-1399</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">ATTATCATC, ATCGATAAT, CAAGATAAA, CTTATCAAG, CCAGATAAT, CCAGATAAT, TCCGATAAA</td>
<td align="left" rowspan="1" colspan="1">300-308, 764-772, 994-1002, 1171-1179, 1282-1290, 1295-1303, 1357-1365</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">GGGCTCT, GGGCGCA, GTGGCCC, GGGCAGG</td>
<td align="left" rowspan="1" colspan="1">1373-1379, 1435-1441, 1442-1448, 1481-1487</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">AGCCAT, ATGGGT, ATGGTT, ATGGGT, ATGGTT, ATGGAA, ATCCAT, ATGGGC, ATGGGC</td>
<td align="left" rowspan="1" colspan="1">29-34, 108-113, 331-336, 452-457, 523-528,715-720, 1030-1035, 1371-1376, 1433-1438</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXO4</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">TGACTGAT, TGACTTTC, TGACTGTA, TGACTTTA, TGACTGAC, TGACTGCA</td>
<td align="left" rowspan="1" colspan="1">132-139, 795-802, 1158-1165, 1167-1174, 1474-1481, 1478-1485</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">CTTATCAAA, TTTATCCAT, CGTGATAAA, TCAGATAAC, TCAGATAAG, TCAGATAAA</td>
<td align="left" rowspan="1" colspan="1">17-25, 61-69, 635-643, 762-770, 1027-1035, 1137-1145</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">GGGCAAG, ATCGCCC, CTAGCCC</td>
<td align="left" rowspan="1" colspan="1">161-167, 459-465, 477-483</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">13</td>
<td align="left" rowspan="1" colspan="1">ATCCAT,ATGGTT,ATGGTA, CACCAT, ATGGGG, TTCCAT, ATGGTT,ATGGGG, CCCCAT, ACCCAT, GACCAT, ACCCAT, TCCCAT</td>
<td align="left" rowspan="1" colspan="1">64-69, 99-104, 178-183, 224-229, 405-410, 414-419, 427-432, 556-561, 627-632, 1122-1127, 1245-1250, 1464-1469, 1488-1493</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXP1</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">GACAGTCA</td>
<td align="left" rowspan="1" colspan="1">549-556</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">GGGCTCC, TGGGCCC, GGGCCCG</td>
<td align="left" rowspan="1" colspan="1">103-109, 486-492, 487-493</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">AACCAT, AACCAT, TTCCAT, ATGGGT</td>
<td align="left" rowspan="1" colspan="1">349-354, 605-610, 917-922, 1438-1443</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXP2</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">CCAAGTCA</td>
<td align="left" rowspan="1" colspan="1">388-395</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">TGTGATAAA, GTTATCAGA, ATTATCAAT, AAAGATAAA, CGTGATAAA, CTCGATAAA</td>
<td align="left" rowspan="1" colspan="1">200-208, 348-356, 619-627, 989-997, 1221-1229, 1341-1349</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">ATCCAT,GACCAT, ATCCAT, ATGGGG, ATGGCA</td>
<td align="left" rowspan="1" colspan="1">165-170, 487-492, 491-496, 875-880, 1250-1255</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FOXP3</td>
<td align="left" rowspan="1" colspan="1">AP-1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">TGACTCAG, AGAAGTCA, TAGAGTCA</td>
<td align="left" rowspan="1" colspan="1">93-100, 319-326, 363-370</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">GATA-1</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">GTTATCACC, TTGGATAAA, CACGATAAA, TTTATCCAG, AGTGATAAA</td>
<td align="left" rowspan="1" colspan="1">223-231, 473-481, 729-737, 1112-1120, 1343-1351</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p53</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">GGGCATT, TTTGCCC</td>
<td align="left" rowspan="1" colspan="1">385-391, 1101-1107</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">YY1</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">TACCAT,ACCCAT, CTCCAT, ATGGTT, CTCCAT, ATGGCT, AGCCAT, GCCCAT, ATGGTC, TGCCAT</td>
<td align="left" rowspan="1" colspan="1">15-20, 73-78, 148-153, 372-377, 398-403, 590-595, 996-1001, 1104-1109, 1337-1342, 1404-1409</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0323740.t005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section></section><section id="sec011"><h2 class="pmc_sec_title">3. Discussion</h2>
<p>In this study, we conducted a comprehensive analysis of the FOX gene family in <em>L. rohita</em>, focusing on their roles in development and stress response. To the best of our knowledge, this is the first systematic investigation of the FOX gene family in <em>L. rohita</em> that integrates phylogenetic analysis and computational genome characterization. Our findings provide significant insights into the diversity, structure, and functional significance of these genes. The identified differentially expressed FOX genes under various stress conditions offer valuable genetic resources for future studies on the adaptation and resilience of <em>L. rohita</em> to environmental challenges. Additionally, our multi-faceted approach, including the characterization of gene family members and their evolutionary dynamics, has broadened our understanding of the FOX genes’ involvement in both developmental processes and stress responses across different physiological states of the species. These insights lay a foundation for further research on the molecular mechanisms driving development and stress tolerance in <em>L. rohita</em>.</p>
<p>The phylogenetic analysis of the FOX gene superfamily in <em>L. rohita</em> reveals valuable insights into the evolutionary trajectory of these genes. A maximum likelihood method was used to construct a phylogenetic tree based on 94 amino acid sequences from five species: <em>L. rohita</em>, <em>O. niloticus</em>, <em>C. idella</em>, <em>D. rerio</em>, and <em>H. sapiens</em>. This approach is consistent with previous research, which highlights the conservation of specific motifs within the FOX gene subclasses across species, pointing to a shared ancestral lineage for these genes [<a href="#pone.0323740.ref013" class="usa-link" aria-describedby="pone.0323740.ref013">13</a>]. The clustering of the FOX gene superfamily in <em>L. rohita</em> with <em>D. rerio</em> and <em>C. idella</em> suggests a common evolutionary origin, supporting earlier studies on the phylogenetic relationships among forkhead transcription factors across various taxa [<a href="#pone.0323740.ref014" class="usa-link" aria-describedby="pone.0323740.ref014">14</a>]. The identification of the 11 major clades—FOXH, FOXM, FOXP, FOXO, FOXF, FOXL, FOXG, FOXD, FOXC, FOXI, and FOXA—further underscores the conserved nature of this gene family, with previous investigations noting that specific FOX genes retain evolutionarily conserved functions across species, strengthening the idea of a unified evolutionary pathway [<a href="#pone.0323740.ref015" class="usa-link" aria-describedby="pone.0323740.ref015">15</a>]. The genetic similarity between <em>L. rohita</em> and other teleost species, like <em>D. rerio</em>, implies parallel evolution of FOX genes under similar selective pressures, aligning with findings on the conservation of gene regulatory elements across vertebrates [<a href="#pone.0323740.ref016" class="usa-link" aria-describedby="pone.0323740.ref016">16</a>]. Additionally, the high degree of sequence similarity between <em>L. rohita</em> and other teleosts reflects the evolutionary preservation of these genes, as demonstrated in subsequent analyses of conserved motifs and gene structures [<a href="#pone.0323740.ref017" class="usa-link" aria-describedby="pone.0323740.ref017">17</a>]. Previous studies have explored the evolutionary dynamics of FOX genes, emphasizing their roles in development, and further highlight the importance of comparing functional roles across species. This comparative analysis is crucial for understanding the stress response and developmental processes in <em>L. rohita</em>, similar to the functions of FOX genes observed in other biological contexts [<a href="#pone.0323740.ref001" class="usa-link" aria-describedby="pone.0323740.ref001">1</a>].</p>
<p>The characterization of the FOX gene family in <em>L. rohita</em> has unveiled significant structural features with potential implications for gene function. Through motif analysis using MEME, ten conserved motifs were identified, with the most notable being MEME-6, which contains the Forkhead Box (FOX) protein domain. This domain plays a key role in DNA binding and transcriptional regulation, a feature that is well-established as characteristic of FOX family members due to their highly conserved DNA-binding forkhead domain, essential for various biological functions [<a href="#pone.0323740.ref018" class="usa-link" aria-describedby="pone.0323740.ref018">18</a>]. The conservation of the Forkhead Box domain across all FOX family members in <em>L. rohita</em> highlights its critical role in regulating processes such as cell differentiation and metabolism [<a href="#pone.0323740.ref019" class="usa-link" aria-describedby="pone.0323740.ref019">19</a>]. This conserved motif suggests that FOX genes in <em>L. rohita</em> likely perform roles similar to those seen in other vertebrates. Prior studies have confirmed that FOX family genes are involved in diverse biological processes across species, reinforcing the idea of a conserved functional role [<a href="#pone.0323740.ref020" class="usa-link" aria-describedby="pone.0323740.ref020">20</a>].</p>
<p>Gene structure analysis further revealed notable variations in exon-intron arrangements among the FOX genes in <em>L. rohita</em>, as well as differences in the untranslated regions (UTRs) upstream and downstream of coding sequences. These variations point to potential functional diversity within the FOX genes, possibly influenced by the unique environmental pressures faced by <em>L. rohita</em> in its aquatic habitat [<a href="#pone.0323740.ref019" class="usa-link" aria-describedby="pone.0323740.ref019">19</a>]. The presence of multiple exons and introns suggests opportunities for alternative splicing, a mechanism critical for enhancing the functional complexity of genes. This idea aligns with previous research that highlights the involvement of FOX proteins in alternative splicing regulation through interactions with spliceosomal components [<a href="#pone.0323740.ref021" class="usa-link" aria-describedby="pone.0323740.ref021">21</a>]. Furthermore, variations in UTRs may contribute to the differential regulation of gene expression, with earlier studies showing that intron presence can influence tissue-specific splicing and gene expression patterns [<a href="#pone.0323740.ref022" class="usa-link" aria-describedby="pone.0323740.ref022">22</a>]. Understanding these structural elements is essential for predicting how FOX genes may respond to environmental cues, such as stressors in aquaculture environments, where differential regulation could be crucial for organismal adaptation [<a href="#pone.0323740.ref019" class="usa-link" aria-describedby="pone.0323740.ref019">19</a>].</p>
<p>The physicochemical analysis of FOX proteins in <em>L. rohita</em> revealed a broad range of molecular weights, from 32705.96 to 119782.93 Da, reflecting the diversity typical of FOX gene family proteins. This range is consistent with findings from previous studies on teleost species, which also highlight the evolutionary diversity of these proteins in terms of molecular size [<a href="#pone.0323740.ref023" class="usa-link" aria-describedby="pone.0323740.ref023">23</a>]. The pI values of these proteins, ranging from 4.81 to 9.50, suggest that <em>L. rohita</em> FOX proteins possess varying charge properties, potentially influencing their interactions with DNA and other cellular components. The importance of charge variability in protein interactions, particularly within aquatic species, has been well-documented [<a href="#pone.0323740.ref024" class="usa-link" aria-describedby="pone.0323740.ref024">24</a>].</p>
<p>Most of the FOX proteins exhibited aliphatic index values above 41.83, indicating a high degree of thermostability, which is crucial for maintaining functionality across diverse environmental conditions. This attribute is particularly beneficial in aquaculture, where stressors like temperature fluctuations and hypoxia can challenge protein function. The role of thermostability in enhancing protein resilience to thermal stress is supported by earlier research showing a correlation between high aliphatic index values and increased protein stability under stress [<a href="#pone.0323740.ref025" class="usa-link" aria-describedby="pone.0323740.ref025">25</a>]. Despite this, instability index values for the majority of FOX proteins exceeded 40, suggesting that these proteins may experience rapid turnover or conformational changes in response to environmental signals—a common characteristic of transcription factors involved in stress response and development [<a href="#pone.0323740.ref026" class="usa-link" aria-describedby="pone.0323740.ref026">26</a>].</p>
<p>Additionally, the analysis showed that most FOX proteins in <em>L. rohita</em> are hydrophobic, as indicated by their low GRAVY (Grand Average of Hydropathy) values, implying their potential involvement in membrane-associated activities or protein-protein interactions requiring hydrophobic interfaces. This aligns with previous research, which emphasizes the critical role of hydrophobic interactions in the functionality of membrane-associated proteins in aquatic organisms [<a href="#pone.0323740.ref027" class="usa-link" aria-describedby="pone.0323740.ref027">27</a>]. Understanding these physicochemical properties is key to predicting how FOX proteins might respond to environmental stressors in aquaculture environments, where rapid adaptation is essential.</p>
<p>The duplication and localization analysis of FOX genes in <em>L. rohita</em> has shed light on the evolutionary development of this gene family. Seven duplicated gene pairs were identified in this study, including segmental, tandem, and whole-genome duplications, which contribute to the expansion and functional diversification of gene families, allowing them to play specialized roles in various physiological processes. The Ka/Ks ratio analysis indicated that these duplicated genes are under purifying selection, suggesting that they have preserved essential functions throughout evolutionary time. Previous studies have similarly shown that duplicated genes often retain their original functions or develop complementary roles over time, supporting the notion of functional conservation [<a href="#pone.0323740.ref028" class="usa-link" aria-describedby="pone.0323740.ref028">28</a>].</p>
<p>Gene localization analysis revealed that FOX genes in <em>L. rohita</em> are distributed across various chromosomes, with a notable clustering of genes at the terminal ends of chromosomes. This distribution is consistent with patterns observed in other species, such as <em>H. sapiens</em>, where syntenic gene pairs are conserved across species. This suggests that FOX genes may have similar functional roles across different organisms, an idea supported by research on FOX gene conservation and synteny [<a href="#pone.0323740.ref001" class="usa-link" aria-describedby="pone.0323740.ref001">1</a>]. Additionally, conserved FOX gene clusters have been observed in bilaterians, reinforcing the concept of evolutionary conservation and functional similarity among these genes across various species [<a href="#pone.0323740.ref029" class="usa-link" aria-describedby="pone.0323740.ref029">29</a>].</p>
<p>The evolutionary history of the FOX gene family in <em>L. rohita</em> appears to have been influenced by species-specific chromosomal rearrangements, likely driven by serial gene duplications. This observation aligns with earlier findings that FOX gene clusters have evolved through such mechanisms, highlighting the dynamic nature of gene family evolution [<a href="#pone.0323740.ref015" class="usa-link" aria-describedby="pone.0323740.ref015">15</a>]. The insights gained from this study provide a valuable framework for future research on the functional roles of FOX genes in <em>L. rohita</em>, particularly in relation to environmental stressors and other physiological challenges.</p>
<p>The transcription factor binding site (TFBS) analysis of FOX genes in <em>L. rohita</em> uncovered a complex regulatory framework. We identified binding sites for several key transcription factors, including AP-1, GATA-1, p53, and YY1. Notably, the analysis revealed 58 binding sites for AP-1, 86 for GATA-1, 85 for p53, and 172 for YY1, indicating that FOX genes may regulate a broad spectrum of cellular processes, such as cell cycle regulation, apoptosis, and stress response. This is consistent with earlier studies showing the role of AP-1 in modulating gene expression in response to diverse stimuli, including stress [<a href="#pone.0323740.ref030" class="usa-link" aria-describedby="pone.0323740.ref030">30</a>]. Similarly, GATA-1 is widely recognized for its involvement in hematopoiesis and stress regulation [<a href="#pone.0323740.ref031" class="usa-link" aria-describedby="pone.0323740.ref031">31</a>]. The abundance of binding sites for these transcription factors underscores the critical role of FOX genes in mediating gene expression, particularly under stress conditions.</p>
<p>This regulatory complexity suggests that FOX genes may serve as integrators of multiple signaling pathways, enabling responses to environmental stressors like oxidative stress, hypoxia, and temperature changes. The capacity of FOX genes to engage with several transcription factors hints at their pivotal role in stress-response mechanisms. Previous research also highlights how transcription factors like p53 and YY1 influence cellular stress responses through cis- and trans-regulatory elements, coordinating responses to changing environmental conditions [<a href="#pone.0323740.ref032" class="usa-link" aria-describedby="pone.0323740.ref032">32</a>]. The involvement of p53, a well-established tumor suppressor that governs cell cycle regulation and apoptosis, further emphasizes the importance of FOX genes in maintaining cellular integrity under stress [<a href="#pone.0323740.ref033" class="usa-link" aria-describedby="pone.0323740.ref033">33</a>]. YY1, functioning as both a transcriptional repressor and activator, adds another layer of regulatory complexity, particularly through histone modifications essential for stress adaptation [<a href="#pone.0323740.ref033" class="usa-link" aria-describedby="pone.0323740.ref033">33</a>]. Understanding these regulatory interactions is crucial for deciphering the role of FOX genes in <em>L. rohita</em>, especially in relation to stress-response pathways. The findings indicate that FOX genes contribute not only to fundamental cellular processes but also to the organism’s adaptive responses to fluctuating environmental conditions. This intricate regulatory network highlights the evolutionary significance of FOX genes in teleosts, as they navigate diverse and often challenging habitats.</p>
<p>The strong correlation between phylogenetic clustering, conserved motifs, gene duplication, and transcriptional regulation highlights the evolutionary pressures shaping FOX genes in <em>L. rohita</em>, optimizing their roles in development, stress adaptation, and cellular homeostasis. The identification of conserved motifs within duplicated gene clusters suggests that these expansions have not led to functional redundancy but rather contributed to regulatory diversification, allowing fine-tuned gene expression in response to environmental fluctuations. This is further reinforced by the presence of multiple stress-responsive transcription factor binding sites, such as AP-1, p53, and GATA-1, which are crucial for oxidative stress regulation, immune response, and developmental processes. Additionally, the physicochemical properties of FOX proteins, including their broad molecular weight range, thermostability, and predominantly hydrophilic nature, align with their involvement in signal transduction, transcriptional regulation, and protein-protein interactions necessary for maintaining cellular function under stress. The clustering of FOX genes in chromosomal regions known to facilitate regulatory interactions further underscores their evolutionary significance. Collectively, these findings suggest that FOX genes in <em>L. rohita</em> have maintained structural and functional integrity through strong evolutionary conservation while simultaneously acquiring regulatory flexibility via gene duplication and transcription factor interactions. This interplay between conservation and adaptability likely provides <em>L. rohita</em> with enhanced resilience to environmental challenges, reinforcing the critical role of FOX genes in both developmental stability and adaptive potential.</p>
<p>While this study provides valuable insights, the limitations of this research should be acknowledged. The in-silico analysis of the FOX gene superfamily in <em>L. rohita</em> offers important perspectives into their genomic landscape, evolutionary significance, and roles in development and stress response. However, several limitations must be considered. Potential inaccuracies in gene annotation could impact the identification and characterization of these genes, leading to incomplete or misleading findings. Furthermore, the absence of experimental validation for the predicted gene functions restricts a deeper understanding of the biological relevance and specific roles of these FOX genes. Without laboratory-based confirmation, the predictions remain theoretical. Additionally, this study does not consider environmental influences, which may have a significant effect on gene expression and regulation under natural conditions. Finally, technical limitations inherent in bioinformatics tools, such as algorithmic biases, may introduce errors into the analysis.</p>
<p>To fill these gaps, future research on the FOX gene family in <em>L. rohita</em> should focus on exploring the functional roles of these genes in various biological processes, with a particular emphasis on their response to environmental stressors such as temperature fluctuations, hypoxia, and pollution. To validate these roles, we could use cutting-edge techniques like CRISPR-Cas9 or RNA interference to either knock down or overexpress specific genes. This would provide clearer evidence of how FOX genes help the fish adapt to stress and support normal development. It would also be helpful to explore how FOX genes interact with other important signaling pathways, such as those involved in the body’s response to oxidative stress or immune challenges. This could give us a more complete picture of how FOX genes maintain balance within cells. To dig deeper, we could look at transcriptomic and proteomic data under various stress conditions, which would help us identify specific targets regulated by FOX genes. Moreover, it’s worth considering the potential of using FOX gene variants in selective breeding programs to create stronger, more resilient fish strains. This could help improve both the productivity and stress resistance of <em>L. rohita</em> and other fish species. Lastly, conducting comparative genomics across different fish species would provide valuable insights into how the FOX gene family has evolved, shedding light on its role in helping these species adapt to different environments.</p></section><section id="sec012"><h2 class="pmc_sec_title">4. Materials and Methods</h2>
<p>This study has been approved by the ethical committee on animal rights and welfare, Department of Zoology, Government Sadiq College Women University, Bahawalpur 63100, Punjab, Pakistan (No. 12/DZ/GSCWUB/2023)</p>
<section id="sec013"><h3 class="pmc_sec_title">4.1. Identification and Characterization of FOX Genes in the <em>L. rohita</em> Genome</h3>
<p>To obtain the complete genome sequence files for the study species <em>L. rohita</em>, <em>O. niloticus</em>, <em>C. idella</em>, <em>D. rerio</em>, and <em>H. sapiens</em>, the National Center for Biotechnology Information (NCBI) Genome database (<a href="https://www.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/</a>) was consulted. Human sequences served as the query to retrieve non-redundant genomic, proteomic, and coding sequence data related to the <em>L. rohita</em> FOX gene superfamily, with the corresponding identification numbers (IDs) listed in <a href="#pone.0323740.s001" class="usa-link">S1 Table</a>. This retrieval was performed using the Basic Local Alignment Search Tool (BLAST) and Hidden Markov Model (HMM) searches, applying an E value threshold of ≤1.0 × e-5 [<a href="#pone.0323740.ref034" class="usa-link" aria-describedby="pone.0323740.ref034">34</a>]. Additionally, the Forkhead box protein domain was identified through the InterPro online tool (<a href="https://www.ebi.ac.uk/interpro/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/interpro/</a>) by utilizing various supplementary programs [<a href="#pone.0323740.ref035" class="usa-link" aria-describedby="pone.0323740.ref035">35</a>]. After gathering the relevant protein sequences, duplicates were eliminated to avoid confusion [<a href="#pone.0323740.ref036" class="usa-link" aria-describedby="pone.0323740.ref036">36</a>]. The resulting non-redundant sequences were analyzed with the Simple Modular Architecture Research Tool (SMART) (<a href="http://smart.embl-heidelberg.de/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://smart.embl-heidelberg.de/</a>), and the NCBI Conserved Domain Database (CDD) (<a href="https://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml</a>) was employed to search for conserved FOX protein domains in <em>L. rohita</em> proteins [<a href="#pone.0323740.ref037" class="usa-link" aria-describedby="pone.0323740.ref037">37</a>].</p>
<p>The physicochemical properties, such as isoelectric point (pI), grand average of hydropathy (GRAVY), molecular weight (MW), aliphatic index (AI), and instability index (II), were determined using Expasy ProtParam tool (<a href="https://web.expasy.org/protparam/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://web.expasy.org/protparam/</a>) [<a href="#pone.0323740.ref038" class="usa-link" aria-describedby="pone.0323740.ref038">38</a>]. Chromosome numbers were identified through genomic sequence data using the NCBI database [<a href="#pone.0323740.ref039" class="usa-link" aria-describedby="pone.0323740.ref039">39</a>].</p></section><section id="sec014"><h3 class="pmc_sec_title">4.2. Phylogenetic Analysis of FOX genes superfamily</h3>
<p>The 94 amino acid sequences were aligned using the MUSCLE algorithm in MEGA11. A phylogenetic tree was constructed using the maximum likelihood method with the Jones-Taylor-Thornton (JTT) model and 1,000 bootstrap replicates to ensure robustness. The resulting Newick file was exported and visualized using the online tool iTOL (<a href="https://itol.embl.de/login.cgi" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://itol.embl.de/login.cgi</a>) [<a href="#pone.0323740.ref040" class="usa-link" aria-describedby="pone.0323740.ref040">40</a>]. This analysis elucidated the evolutionary relationships among <em>L. rohita</em> and the reference species <em>O. niloticus</em>, <em>C. idella</em>, <em>D. rerio</em>, and <em>H. sapiens</em> based on their physical and genetic similarities and differences.</p></section><section id="sec015"><h3 class="pmc_sec_title">4.3. Syntax gene duplication and chromosomal distribution of FOX superfamily</h3>
<p>The occurrence of duplications in the FOX gene family of <em>L. rohita</em> was assessed through pairwise alignment of the FOX genes, utilizing the MUSCLE algorithms available in MEGA11 v.11.0. [<a href="#pone.0323740.ref041" class="usa-link" aria-describedby="pone.0323740.ref041">41</a>]. Additionally, the pairwise synonymous substitutions per synonymous site (Ks) and non-synonymous substitutions per non-synonymous site (Ka) [<a href="#pone.0323740.ref042" class="usa-link" aria-describedby="pone.0323740.ref042">42</a>], adjusted for multiple hits, were calculated using the DnaSP v10.0 software (<a href="http://www.ub.edu/dnasp/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.ub.edu/dnasp/</a>) [<a href="#pone.0323740.ref043" class="usa-link" aria-describedby="pone.0323740.ref043">43</a>]. Furthermore, the age of all duplicated genes was estimated using the formula t = Ks/2r where r is 1.1 × 10−8 [<a href="#pone.0323740.ref044" class="usa-link" aria-describedby="pone.0323740.ref044">44</a>]. The FOX genes of both <em>L. rohita</em> and <em>H. sapiens</em> were also aligned to create a dual synteny plot [<a href="#pone.0323740.ref045" class="usa-link" aria-describedby="pone.0323740.ref045">45</a>]. Genomic resources and the General Feature Format (GFF) file [<a href="#pone.0323740.ref046" class="usa-link" aria-describedby="pone.0323740.ref046">46</a>] were utilized to identify the chromosomal gene locations within the <em>L. rohita</em> genome using the TBtools program (<a href="https://github.com/CJ-Chen/TBtools" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/CJ-Chen/TBtools</a>) as a display server. Chromosomal and genomic data served as input for plotting Circos using the same TBtools program [<a href="#pone.0323740.ref047" class="usa-link" aria-describedby="pone.0323740.ref047">47</a>]. Circos illustrated the variations in genomic organization and the overall positional relationships among the genomic intervals of <em>L. rohita</em> FOX genes [<a href="#pone.0323740.ref048" class="usa-link" aria-describedby="pone.0323740.ref048">48</a>].</p></section><section id="sec016"><h3 class="pmc_sec_title">4.4. Characterization of structural features</h3>
<p>The conserved motifs were analyzed using the MEME Suite (<a href="https://meme-suite.org/meme/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://meme-suite.org/meme/</a>), with all <em>L. rohita</em> FOX protein sequences submitted as queries in FASTA format. The MEME Suite was also employed to identify 10 conserved motif patterns within the FOX gene [<a href="#pone.0323740.ref049" class="usa-link" aria-describedby="pone.0323740.ref049">49</a>]. A hit data file obtained from the Conserved Domain online tool (<a href="https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi</a>) was used for domain analysis in TBtools. The GTF file facilitated gene structure visualization within TBtools. Ultimately, TBtools produced a comprehensive graph that integrated these three analyses with a phylogenetic tree, offering a holistic view of the conserved motifs, exon-intron structures, and functional domains of the FOX genes [<a href="#pone.0323740.ref050" class="usa-link" aria-describedby="pone.0323740.ref050">50</a>,<a href="#pone.0323740.ref051" class="usa-link" aria-describedby="pone.0323740.ref051">51</a>].</p></section><section id="sec017"><h3 class="pmc_sec_title">4.5. Secondary structure and 3D models of FOX superfamily proteins</h3>
<p>We began by predicting protein secondary structures using the PSIPRED online server (<a href="http://bioinf.cs.ucl.ac.uk/psipred/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bioinf.cs.ucl.ac.uk/psipred/</a>) renowned for its accuracy [<a href="#pone.0323740.ref052" class="usa-link" aria-describedby="pone.0323740.ref052">52</a>,<a href="#pone.0323740.ref053" class="usa-link" aria-describedby="pone.0323740.ref053">53</a>]. These 2D predictions were further validated with Pyre2 (<a href="https://www.sbg.bio.ic.ac.uk/phyre2/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sbg.bio.ic.ac.uk/phyre2/</a>) [<a href="#pone.0323740.ref054" class="usa-link" aria-describedby="pone.0323740.ref054">54</a>], which confirmed the observed structural elements. For three-dimensional modeling, we employed Phyre2, a robust web-based homology modeling platform [<a href="#pone.0323740.ref054" class="usa-link" aria-describedby="pone.0323740.ref054">54</a>,<a href="#pone.0323740.ref055" class="usa-link" aria-describedby="pone.0323740.ref055">55</a>]. To ensure the consistency and accuracy of our 3D model, we subsequently validated it using the Swiss-Model online tool (<a href="https://swissmodel.expasy.org/interactive" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://swissmodel.expasy.org/interactive</a>) [<a href="#pone.0323740.ref056" class="usa-link" aria-describedby="pone.0323740.ref056">56</a>].</p></section><section id="sec018"><h3 class="pmc_sec_title">4.6. Scan Prosite analysis</h3>
<p>The online tool Scan Prosite (<a href="https://prosite.expasy.org/scanprosite/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://prosite.expasy.org/scanprosite/</a>) was employed to identify functional and structural intra-domains. The protein sequence was submitted to query the Prosite motif collection for analysis [<a href="#pone.0323740.ref057" class="usa-link" aria-describedby="pone.0323740.ref057">57</a>].</p></section><section id="sec019"><h3 class="pmc_sec_title">4.7. Retrieval and identification of transcription factor binding sites (TFBSs)</h3>
<p>Genomic files containing only the promoter sequences of 1500 bp for each gene were submitted to the PROMO software (<a href="https://alggen.lsi.upc.es/cgi-bin/promov3/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://alggen.lsi.upc.es/cgi-bin/promov3/</a>), which utilizes version 6.4 of TRANSFAC to identify potential binding sites in the sequences [<a href="#pone.0323740.ref058" class="usa-link" aria-describedby="pone.0323740.ref058">58</a>]. This analysis focused on three transcriptional promoter binding sites: GATA-1, AP-1, and p53, as well as one repressor site, YY1. The GATA-1 transcription factor is crucial for development, as it interacts with FOX genes to regulate gene expression during the development and functioning of erythroid cells (red blood cells) and other hematopoietic lineages [<a href="#pone.0323740.ref059" class="usa-link" aria-describedby="pone.0323740.ref059">59</a>,<a href="#pone.0323740.ref060" class="usa-link" aria-describedby="pone.0323740.ref060">60</a>]. AP-1 is responsible for regulating gene expression in response to various stimuli, including stress. It can modulate the activity of FOXO and FOXA proteins in reaction to oxidative stress, promoting either cell survival or apoptosis [<a href="#pone.0323740.ref061" class="usa-link" aria-describedby="pone.0323740.ref061">61</a>,<a href="#pone.0323740.ref062" class="usa-link" aria-describedby="pone.0323740.ref062">62</a>]. The p53 protein is activated under stress conditions, such as DNA damage or oxidative stress, and it can regulate FOXO1 and FOXO3, thereby enhancing their roles in apoptosis and cell cycle arrest [<a href="#pone.0323740.ref063" class="usa-link" aria-describedby="pone.0323740.ref063">63</a>]. YY1 is known to repress the expression of FOX O genes, particularly during stress responses like oxidative stress [<a href="#pone.0323740.ref064" class="usa-link" aria-describedby="pone.0323740.ref064">64</a>]. Additionally, YY1 can regulate developmental processes by inhibiting genes that are involved in cell differentiation [<a href="#pone.0323740.ref065" class="usa-link" aria-describedby="pone.0323740.ref065">65</a>].</p></section></section><section id="sec020"><h2 class="pmc_sec_title">5. Conclusions</h2>
<p>To sum up, this study presents a comprehensive computational analysis of the FOX gene family in <em>L. rohita</em>, shedding light on their crucial roles in development and response to stress. By exploring the diversity, structure, and functions of these genes, we have uncovered important insights into how they have evolved, driven by genetic variation, duplication, and selective forces. This comprehensive approach, combining phylogenetic analysis with genome characterization, highlights the FOX genes’ key role in helping <em>L. rohita</em> adapt to environmental challenges. The discovery of these genes not only deepens our understanding of their evolutionary importance but also provides valuable genetic resources for genetic enhancement and sustainable management of <em>L. rohita</em>. These findings create a solid foundation for future work in selective breeding and aquaculture, while also paving the way for deeper exploration into the molecular mechanisms that govern the species’ development, stress response, and resilience.</p></section><section id="sec021"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pone.0323740.s001"><div class="caption p">
<span>S1 Table. Genomic properties of the FOX gene superfamily in <em>L. rohita.</em></span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370200/bin/pone.0323740.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0323740.s001.docx</a><sup> (16.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0323740.s002"><div class="caption p">
<span>S1 Fig</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370200/bin/pone.0323740.s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0323740.s002.pdf</a><sup> (3.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0323740.s003"><div class="caption p">
<span>S2 Fig</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370200/bin/pone.0323740.s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0323740.s003.pdf</a><sup> (1.8MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors are thankful to the Ongoing Research Funding (ORF-2025-833), King Saud University, Riyadh, Saudi Arabia</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>The data are all contained within the manuscript.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>The author(s) received no specific funding for this work.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pone.0323740.ref001">
<span class="label">1.</span><cite>Jackson BC, Carpenter C, Nebert DW, Vasiliou V. Update of human and mouse forkhead box (FOX) gene families. Hum Genomics. 2010;4(5):345–52. doi: 10.1186/1479-7364-4-5-345

</cite> [<a href="https://doi.org/10.1186/1479-7364-4-5-345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3500164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20650821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hum%20Genomics&amp;title=Update%20of%20human%20and%20mouse%20forkhead%20box%20(FOX)%20gene%20families&amp;author=BC%20Jackson&amp;author=C%20Carpenter&amp;author=DW%20Nebert&amp;author=V%20Vasiliou&amp;volume=4&amp;issue=5&amp;publication_year=2010&amp;pages=345-52&amp;pmid=20650821&amp;doi=10.1186/1479-7364-4-5-345&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref002">
<span class="label">2.</span><cite>Pascual-Carreras E, Herrera-Úbeda C, Rosselló M, Coronel-Córdoba P, Garcia-Fernàndez J, Saló E, et al. Analysis of Fox genes in Schmidtea mediterranea reveals new families and a conserved role of Smed-foxO in controlling cell death. Sci Rep. 2021;11(1):2947. doi: 10.1038/s41598-020-80627-0

</cite> [<a href="https://doi.org/10.1038/s41598-020-80627-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7859237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33536473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Analysis%20of%20Fox%20genes%20in%20Schmidtea%20mediterranea%20reveals%20new%20families%20and%20a%20conserved%20role%20of%20Smed-foxO%20in%20controlling%20cell%20death&amp;author=E%20Pascual-Carreras&amp;author=C%20Herrera-%C3%9Abeda&amp;author=M%20Rossell%C3%B3&amp;author=P%20Coronel-C%C3%B3rdoba&amp;author=J%20Garcia-Fern%C3%A0ndez&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=2947&amp;pmid=33536473&amp;doi=10.1038/s41598-020-80627-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref003">
<span class="label">3.</span><cite>Hu W, Li M, Wang Y, Zhong C, Si X, Shi X, et al. Comprehensive bioinformatics analysis reveals the significance of forkhead box family members in pancreatic adenocarcinoma. Aging (Albany NY). 2023;15(1):92–107. doi: 10.18632/aging.204455

</cite> [<a href="https://doi.org/10.18632/aging.204455" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9876641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36622275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&amp;title=Comprehensive%20bioinformatics%20analysis%20reveals%20the%20significance%20of%20forkhead%20box%20family%20members%20in%20pancreatic%20adenocarcinoma&amp;author=W%20Hu&amp;author=M%20Li&amp;author=Y%20Wang&amp;author=C%20Zhong&amp;author=X%20Si&amp;volume=15&amp;issue=1&amp;publication_year=2023&amp;pages=92-107&amp;pmid=36622275&amp;doi=10.18632/aging.204455&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref004">
<span class="label">4.</span><cite>Magie CR, Pang K, Martindale MQ. Genomic inventory and expression of Sox and Fox genes in the cnidarian Nematostella vectensis. Dev Genes Evol. 2005;215(12):618–30. doi: 10.1007/s00427-005-0022-y

</cite> [<a href="https://doi.org/10.1007/s00427-005-0022-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16193320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev%20Genes%20Evol&amp;title=Genomic%20inventory%20and%20expression%20of%20Sox%20and%20Fox%20genes%20in%20the%20cnidarian%20Nematostella%20vectensis&amp;author=CR%20Magie&amp;author=K%20Pang&amp;author=MQ%20Martindale&amp;volume=215&amp;issue=12&amp;publication_year=2005&amp;pages=618-30&amp;pmid=16193320&amp;doi=10.1007/s00427-005-0022-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref005">
<span class="label">5.</span><cite>Bach D-H, Long NP, Luu T-T-T, Anh NH, Kwon SW, Lee SK. The Dominant Role of Forkhead Box Proteins in Cancer. Int J Mol Sci. 2018;19(10):3279. doi: 10.3390/ijms19103279

</cite> [<a href="https://doi.org/10.3390/ijms19103279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6213973/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30360388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=The%20Dominant%20Role%20of%20Forkhead%20Box%20Proteins%20in%20Cancer&amp;author=D-H%20Bach&amp;author=NP%20Long&amp;author=T-T-T%20Luu&amp;author=NH%20Anh&amp;author=SW%20Kwon&amp;volume=19&amp;issue=10&amp;publication_year=2018&amp;pages=3279&amp;pmid=30360388&amp;doi=10.3390/ijms19103279&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref006">
<span class="label">6.</span><cite>Nakagawa T, Nakayama K. Protein monoubiquitylation: targets and diverse functions. Genes Cells. 2015;20(7):543–62. doi: 10.1111/gtc.12250

</cite> [<a href="https://doi.org/10.1111/gtc.12250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4744734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26085183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Cells&amp;title=Protein%20monoubiquitylation:%20targets%20and%20diverse%20functions&amp;author=T%20Nakagawa&amp;author=K%20Nakayama&amp;volume=20&amp;issue=7&amp;publication_year=2015&amp;pages=543-62&amp;pmid=26085183&amp;doi=10.1111/gtc.12250&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref007">
<span class="label">7.</span><cite>Hannenhalli S, Kaestner KH. The evolution of Fox genes and their role in development and disease. Nat Rev Genet. 2009;10(4):233–40. doi: 10.1038/nrg2523

</cite> [<a href="https://doi.org/10.1038/nrg2523" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2733165/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19274050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&amp;title=The%20evolution%20of%20Fox%20genes%20and%20their%20role%20in%20development%20and%20disease&amp;author=S%20Hannenhalli&amp;author=KH%20Kaestner&amp;volume=10&amp;issue=4&amp;publication_year=2009&amp;pages=233-40&amp;pmid=19274050&amp;doi=10.1038/nrg2523&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref008">
<span class="label">8.</span><cite>Sollis E, Deriziotis P, Saitsu H, Miyake N, Matsumoto N, Hoffer MJV, et al. Equivalent missense variant in the FOXP2 and FOXP1 transcription factors causes distinct neurodevelopmental disorders. Hum Mutat. 2017;38(11):1542–54. doi: 10.1002/humu.23303

</cite> [<a href="https://doi.org/10.1002/humu.23303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28741757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&amp;title=Equivalent%20missense%20variant%20in%20the%20FOXP2%20and%20FOXP1%20transcription%20factors%20causes%20distinct%20neurodevelopmental%20disorders&amp;author=E%20Sollis&amp;author=P%20Deriziotis&amp;author=H%20Saitsu&amp;author=N%20Miyake&amp;author=N%20Matsumoto&amp;volume=38&amp;issue=11&amp;publication_year=2017&amp;pages=1542-54&amp;pmid=28741757&amp;doi=10.1002/humu.23303&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref009">
<span class="label">9.</span><cite>Zhang Z, Xia T, Zhou S, Yang X, Lyu T, Wang L, et al. High-Quality Chromosome-Level Genome Assembly of the Corsac Fox (Vulpes corsac) Reveals Adaptation to Semiarid and Harsh Environments. Int J Mol Sci. 2023;24(11):9599. doi: 10.3390/ijms24119599

</cite> [<a href="https://doi.org/10.3390/ijms24119599" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10253726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37298549/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=High-Quality%20Chromosome-Level%20Genome%20Assembly%20of%20the%20Corsac%20Fox%20(Vulpes%20corsac)%20Reveals%20Adaptation%20to%20Semiarid%20and%20Harsh%20Environments&amp;author=Z%20Zhang&amp;author=T%20Xia&amp;author=S%20Zhou&amp;author=X%20Yang&amp;author=T%20Lyu&amp;volume=24&amp;issue=11&amp;publication_year=2023&amp;pages=9599&amp;pmid=37298549&amp;doi=10.3390/ijms24119599&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref010">
<span class="label">10.</span><cite>Jayasankar P. Present status of freshwater aquaculture in India - A review. Indian J Fish. 2018;65(4). doi: 10.21077/ijf.2018.65.4.81300-20</cite> [<a href="https://doi.org/10.21077/ijf.2018.65.4.81300-20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20J%20Fish&amp;title=Present%20status%20of%20freshwater%20aquaculture%20in%20India%20-%20A%20review&amp;author=P%20Jayasankar&amp;volume=65&amp;issue=4&amp;publication_year=2018&amp;doi=10.21077/ijf.2018.65.4.81300-20&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref011">
<span class="label">11.</span><cite>Shahbaz Azhar M, Zubair Anjum M, Akhter S, Bibi S, Qayash Khan M, Rasool F. Effect of Temperature Fluctuation on the Indian Major Carp Labeo rohita and Cirrhinus mrigala Larvae under a Controlled Environment. Jammu Kashmir J Agric. 2022;2(3):9–17. doi: 10.56810/jkjagri.002.03.0044</cite> [<a href="https://doi.org/10.56810/jkjagri.002.03.0044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Jammu%20Kashmir%20J%20Agric&amp;title=Effect%20of%20Temperature%20Fluctuation%20on%20the%20Indian%20Major%20Carp%20Labeo%20rohita%20and%20Cirrhinus%20mrigala%20Larvae%20under%20a%20Controlled%20Environment&amp;author=M%20Shahbaz%20Azhar&amp;author=M%20Zubair%20Anjum&amp;author=S%20Akhter&amp;author=S%20Bibi&amp;author=M%20Qayash%20Khan&amp;volume=2&amp;issue=3&amp;publication_year=2022&amp;pages=9-17&amp;doi=10.56810/jkjagri.002.03.0044&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref012">
<span class="label">12.</span><cite>Golson ML, Kaestner KH. Fox transcription factors: from development to disease. Development. 2016;143(24):4558–70. doi: 10.1242/dev.112672

</cite> [<a href="https://doi.org/10.1242/dev.112672" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5201025/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27965437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Fox%20transcription%20factors:%20from%20development%20to%20disease&amp;author=ML%20Golson&amp;author=KH%20Kaestner&amp;volume=143&amp;issue=24&amp;publication_year=2016&amp;pages=4558-70&amp;pmid=27965437&amp;doi=10.1242/dev.112672&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref013">
<span class="label">13.</span><cite>Yaklichkin S, Vekker A, Stayrook S, Lewis M, Kessler DS. Prevalence of the EH1 Groucho interaction motif in the metazoan Fox family of transcriptional regulators. BMC Genomics. 2007;8:201. doi: 10.1186/1471-2164-8-201

</cite> [<a href="https://doi.org/10.1186/1471-2164-8-201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1939712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17598915/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Prevalence%20of%20the%20EH1%20Groucho%20interaction%20motif%20in%20the%20metazoan%20Fox%20family%20of%20transcriptional%20regulators&amp;author=S%20Yaklichkin&amp;author=A%20Vekker&amp;author=S%20Stayrook&amp;author=M%20Lewis&amp;author=DS%20Kessler&amp;volume=8&amp;publication_year=2007&amp;pages=201&amp;pmid=17598915&amp;doi=10.1186/1471-2164-8-201&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref014">
<span class="label">14.</span><cite>Schomburg C, Janssen R, Prpic N-M. Phylogenetic analysis of forkhead transcription factors in the Panarthropoda. Dev Genes Evol. 2022;232(1):39–48. doi: 10.1007/s00427-022-00686-3

</cite> [<a href="https://doi.org/10.1007/s00427-022-00686-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8918179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35230523/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev%20Genes%20Evol&amp;title=Phylogenetic%20analysis%20of%20forkhead%20transcription%20factors%20in%20the%20Panarthropoda&amp;author=C%20Schomburg&amp;author=R%20Janssen&amp;author=N-M%20Prpic&amp;volume=232&amp;issue=1&amp;publication_year=2022&amp;pages=39-48&amp;pmid=35230523&amp;doi=10.1007/s00427-022-00686-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref015">
<span class="label">15.</span><cite>Fritzenwanker JH, Gerhart J, Freeman RM Jr, Lowe CJ. The Fox/Forkhead transcription factor family of the hemichordate Saccoglossus kowalevskii. Evodevo. 2014;5:17. doi: 10.1186/2041-9139-5-17

</cite> [<a href="https://doi.org/10.1186/2041-9139-5-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4077281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24987514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evodevo&amp;title=The%20Fox/Forkhead%20transcription%20factor%20family%20of%20the%20hemichordate%20Saccoglossus%20kowalevskii&amp;author=JH%20Fritzenwanker&amp;author=J%20Gerhart&amp;author=Jr%20Freeman%20RM&amp;author=CJ%20Lowe&amp;volume=5&amp;publication_year=2014&amp;pages=17&amp;pmid=24987514&amp;doi=10.1186/2041-9139-5-17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref016">
<span class="label">16.</span><cite>Long HK, Sims D, Heger A, Blackledge NP, Kutter C, Wright ML, et al. Epigenetic conservation at gene regulatory elements revealed by non-methylated DNA profiling in seven vertebrates. Elife. 2013;2:e00348. doi: 10.7554/eLife.00348

</cite> [<a href="https://doi.org/10.7554/eLife.00348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3583005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23467541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Epigenetic%20conservation%20at%20gene%20regulatory%20elements%20revealed%20by%20non-methylated%20DNA%20profiling%20in%20seven%20vertebrates&amp;author=HK%20Long&amp;author=D%20Sims&amp;author=A%20Heger&amp;author=NP%20Blackledge&amp;author=C%20Kutter&amp;volume=2&amp;publication_year=2013&amp;pmid=23467541&amp;doi=10.7554/eLife.00348&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref017">
<span class="label">17.</span><cite>Vij S, Rink JC, Ho HK, Babu D, Eitel M, Narasimhan V, et al. Evolutionarily ancient association of the FoxJ1 transcription factor with the motile ciliogenic program. PLoS Genet. 2012;8(11):e1003019. doi: 10.1371/journal.pgen.1003019

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1003019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3493443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23144623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&amp;title=Evolutionarily%20ancient%20association%20of%20the%20FoxJ1%20transcription%20factor%20with%20the%20motile%20ciliogenic%20program&amp;author=S%20Vij&amp;author=JC%20Rink&amp;author=HK%20Ho&amp;author=D%20Babu&amp;author=M%20Eitel&amp;volume=8&amp;issue=11&amp;publication_year=2012&amp;pmid=23144623&amp;doi=10.1371/journal.pgen.1003019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref018">
<span class="label">18.</span><cite>Minoretti P, Arra M, Emanuele E, Olivieri V, Aldeghi A, Politi P, et al. A W148R mutation in the human FOXD4 gene segregating with dilated cardiomyopathy, obsessive-compulsive disorder, and suicidality. Int J Mol Med. 2007. doi: 10.3892/ijmm.19.3.369</cite> [<a href="https://doi.org/10.3892/ijmm.19.3.369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17273782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Med&amp;title=A%20W148R%20mutation%20in%20the%20human%20FOXD4%20gene%20segregating%20with%20dilated%20cardiomyopathy,%20obsessive-compulsive%20disorder,%20and%20suicidality&amp;author=P%20Minoretti&amp;author=M%20Arra&amp;author=E%20Emanuele&amp;author=V%20Olivieri&amp;author=A%20Aldeghi&amp;publication_year=2007&amp;pmid=17273782&amp;doi=10.3892/ijmm.19.3.369&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref019">
<span class="label">19.</span><cite>Jia Z, Wan F, Zhu Y, Shi G, Zhang H, Dai B, et al. Forkhead-box series expression network is associated with outcome of clear-cell renal cell carcinoma. Oncol Lett. 2018;15(6):8669–80. doi: 10.3892/ol.2018.8405

</cite> [<a href="https://doi.org/10.3892/ol.2018.8405" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5950509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29805604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncol%20Lett&amp;title=Forkhead-box%20series%20expression%20network%20is%20associated%20with%20outcome%20of%20clear-cell%20renal%20cell%20carcinoma&amp;author=Z%20Jia&amp;author=F%20Wan&amp;author=Y%20Zhu&amp;author=G%20Shi&amp;author=H%20Zhang&amp;volume=15&amp;issue=6&amp;publication_year=2018&amp;pages=8669-80&amp;pmid=29805604&amp;doi=10.3892/ol.2018.8405&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref020">
<span class="label">20.</span><cite>Yang W, Chen H, Ma L, Dong J, Wei M, Xue X, et al. A comprehensive analysis of the FOX family for predicting kidney renal clear cell carcinoma prognosis and the oncogenic role of FOXG1. Aging (Albany NY). 2022;14(24):10107–24. doi: 10.18632/aging.204448

</cite> [<a href="https://doi.org/10.18632/aging.204448" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9831721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36585925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&amp;title=A%20comprehensive%20analysis%20of%20the%20FOX%20family%20for%20predicting%20kidney%20renal%20clear%20cell%20carcinoma%20prognosis%20and%20the%20oncogenic%20role%20of%20FOXG1&amp;author=W%20Yang&amp;author=H%20Chen&amp;author=L%20Ma&amp;author=J%20Dong&amp;author=M%20Wei&amp;volume=14&amp;issue=24&amp;publication_year=2022&amp;pages=10107-24&amp;pmid=36585925&amp;doi=10.18632/aging.204448&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref021">
<span class="label">21.</span><cite>Zhou H-L, Lou H. Repression of prespliceosome complex formation at two distinct steps by Fox-1/Fox-2 proteins. Mol Cell Biol. 2008;28(17):5507–16. doi: 10.1128/MCB.00530-08

</cite> [<a href="https://doi.org/10.1128/MCB.00530-08" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2519742/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18573872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&amp;title=Repression%20of%20prespliceosome%20complex%20formation%20at%20two%20distinct%20steps%20by%20Fox-1/Fox-2%20proteins&amp;author=H-L%20Zhou&amp;author=H%20Lou&amp;volume=28&amp;issue=17&amp;publication_year=2008&amp;pages=5507-16&amp;pmid=18573872&amp;doi=10.1128/MCB.00530-08&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref022">
<span class="label">22.</span><cite>Jin Y, Suzuki H, Maegawa S, Endo H, Sugano S, Hashimoto K, et al. A vertebrate RNA-binding protein Fox-1 regulates tissue-specific splicing via the pentanucleotide GCAUG. EMBO J. 2003;22(4):905–12. doi: 10.1093/emboj/cdg089

</cite> [<a href="https://doi.org/10.1093/emboj/cdg089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC145449/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12574126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J&amp;title=A%20vertebrate%20RNA-binding%20protein%20Fox-1%20regulates%20tissue-specific%20splicing%20via%20the%20pentanucleotide%20GCAUG&amp;author=Y%20Jin&amp;author=H%20Suzuki&amp;author=S%20Maegawa&amp;author=H%20Endo&amp;author=S%20Sugano&amp;volume=22&amp;issue=4&amp;publication_year=2003&amp;pages=905-12&amp;pmid=12574126&amp;doi=10.1093/emboj/cdg089&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref023">
<span class="label">23.</span><cite>Zhao Y, Liu H, Cong F, Wu W, Zhao R, Kong X. Phosphoprotein Contributes to the Thermostability of Newcastle Disease Virus. Biomed Res Int. 2018;2018:8917476. doi: 10.1155/2018/8917476

</cite> [<a href="https://doi.org/10.1155/2018/8917476" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6241354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30519589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomed%20Res%20Int&amp;title=Phosphoprotein%20Contributes%20to%20the%20Thermostability%20of%20Newcastle%20Disease%20Virus&amp;author=Y%20Zhao&amp;author=H%20Liu&amp;author=F%20Cong&amp;author=W%20Wu&amp;author=R%20Zhao&amp;volume=2018&amp;publication_year=2018&amp;pages=8917476&amp;pmid=30519589&amp;doi=10.1155/2018/8917476&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref024">
<span class="label">24.</span><cite>Mahboob S, Farooq M, Mahmood S, Nasir N, Sultana S, Chaudhry AS, et al. Phylogenetic Relationship of Cultured and WildLabeo rohitaandCirrhinus mrigalaBased on Muscles Proteins Profile in Different Weight Groups: A New Tool in Phylogenetic Analysis. Int J Food Properties. 2012;15(5):949–60. doi: 10.1080/10942912.2010.511752</cite> [<a href="https://doi.org/10.1080/10942912.2010.511752" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Food%20Properties&amp;title=Phylogenetic%20Relationship%20of%20Cultured%20and%20WildLabeo%20rohitaandCirrhinus%20mrigalaBased%20on%20Muscles%20Proteins%20Profile%20in%20Different%20Weight%20Groups:%20A%20New%20Tool%20in%20Phylogenetic%20Analysis&amp;author=S%20Mahboob&amp;author=M%20Farooq&amp;author=S%20Mahmood&amp;author=N%20Nasir&amp;author=S%20Sultana&amp;volume=15&amp;issue=5&amp;publication_year=2012&amp;pages=949-60&amp;doi=10.1080/10942912.2010.511752&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref025">
<span class="label">25.</span><cite>Kumar P, Pal AK, Sahu NP, Christina L, Jha AK. Biochemical and haematological responses to thermal stress in Labeo rohita (Hamilton, 1822) fingerlings. Indian J Fish. 2019;66(2). doi: 10.21077/ijf.2019.66.2.78397-10</cite> [<a href="https://doi.org/10.21077/ijf.2019.66.2.78397-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20J%20Fish&amp;title=Biochemical%20and%20haematological%20responses%20to%20thermal%20stress%20in%20Labeo%20rohita%20(Hamilton,%201822)%20fingerlings&amp;author=P%20Kumar&amp;author=AK%20Pal&amp;author=NP%20Sahu&amp;author=L%20Christina&amp;author=AK%20Jha&amp;volume=66&amp;issue=2&amp;publication_year=2019&amp;doi=10.21077/ijf.2019.66.2.78397-10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref026">
<span class="label">26.</span><cite>Olude OO, Sahu NP, Nuzaiba PM, Krishna G. Feeding sundried cassava leaf meal as a replacement for de‐oiled rice bran in the diets of rohu, L abeo rohita , fingerlings: effect on growth, enzyme activities and gene expression of igf‐i and igf‐bpi. Aquacult Nutr. 2021;27(3):817–28. doi: 10.1111/anu.13226</cite> [<a href="https://doi.org/10.1111/anu.13226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aquacult%20Nutr&amp;title=Feeding%20sundried%20cassava%20leaf%20meal%20as%20a%20replacement%20for%20de%E2%80%90oiled%20rice%20bran%20in%20the%20diets%20of%20rohu,%20L%20abeo%20rohita%20,%20fingerlings:%20effect%20on%20growth,%20enzyme%20activities%20and%20gene%20expression%20of%20igf%E2%80%90i%20and%20igf%E2%80%90bpi&amp;author=OO%20Olude&amp;author=NP%20Sahu&amp;author=PM%20Nuzaiba&amp;author=G%20Krishna&amp;volume=27&amp;issue=3&amp;publication_year=2021&amp;pages=817-28&amp;doi=10.1111/anu.13226&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref027">
<span class="label">27.</span><cite>Harshini V, Shukla N, Raval I, Kumar S, Shrivastava V, Chaudhari A, et al. Interplay of gene expression and regulators under salinity stress in gill of Labeo rohita. BMC Genomics. 2023;24(1):336. doi: 10.1186/s12864-023-09426-x

</cite> [<a href="https://doi.org/10.1186/s12864-023-09426-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10278310/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37337199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Interplay%20of%20gene%20expression%20and%20regulators%20under%20salinity%20stress%20in%20gill%20of%20Labeo%20rohita&amp;author=V%20Harshini&amp;author=N%20Shukla&amp;author=I%20Raval&amp;author=S%20Kumar&amp;author=V%20Shrivastava&amp;volume=24&amp;issue=1&amp;publication_year=2023&amp;pages=336&amp;pmid=37337199&amp;doi=10.1186/s12864-023-09426-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref028">
<span class="label">28.</span><cite>Moore RC, Purugganan MD. The early stages of duplicate gene evolution. Proc Natl Acad Sci U S A. 2003;100(26):15682–7. doi: 10.1073/pnas.2535513100

</cite> [<a href="https://doi.org/10.1073/pnas.2535513100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC307628/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14671323/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=The%20early%20stages%20of%20duplicate%20gene%20evolution&amp;author=RC%20Moore&amp;author=MD%20Purugganan&amp;volume=100&amp;issue=26&amp;publication_year=2003&amp;pages=15682-7&amp;pmid=14671323&amp;doi=10.1073/pnas.2535513100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref029">
<span class="label">29.</span><cite>Mazet F, Amemiya CT, Shimeld SM. An ancient Fox gene cluster in bilaterian animals. Curr Biol. 2006;16(9):R314-6. doi: 10.1016/j.cub.2006.03.088

</cite> [<a href="https://doi.org/10.1016/j.cub.2006.03.088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16682334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&amp;title=An%20ancient%20Fox%20gene%20cluster%20in%20bilaterian%20animals&amp;author=F%20Mazet&amp;author=CT%20Amemiya&amp;author=SM%20Shimeld&amp;volume=16&amp;issue=9&amp;publication_year=2006&amp;pmid=16682334&amp;doi=10.1016/j.cub.2006.03.088&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref030">
<span class="label">30.</span><cite>Koivisto E, Kaikkonen L, Tokola H, Pikkarainen S, Aro J, Pennanen H, et al. Distinct regulation of B-type natriuretic peptide transcription by p38 MAPK isoforms. Mol Cell Endocrinol. 2011;338(1–2):18–27. doi: 10.1016/j.mce.2011.02.015

</cite> [<a href="https://doi.org/10.1016/j.mce.2011.02.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21354263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Endocrinol&amp;title=Distinct%20regulation%20of%20B-type%20natriuretic%20peptide%20transcription%20by%20p38%20MAPK%20isoforms&amp;author=E%20Koivisto&amp;author=L%20Kaikkonen&amp;author=H%20Tokola&amp;author=S%20Pikkarainen&amp;author=J%20Aro&amp;volume=338&amp;publication_year=2011&amp;pages=18-27&amp;pmid=21354263&amp;doi=10.1016/j.mce.2011.02.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref031">
<span class="label">31.</span><cite>Kishore A, Mukesh M, Sobti RC, Mishra BP, Sodhi M. Variations in the Regulatory Region of Alpha S1-Casein Milk Protein Gene among Tropically Adapted Indian Native (Bos Indicus) Cattle. ISRN Biotechnol. 2013;2013:926025. doi: 10.5402/2013/926025

</cite> [<a href="https://doi.org/10.5402/2013/926025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4393073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25937984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISRN%20Biotechnol&amp;title=Variations%20in%20the%20Regulatory%20Region%20of%20Alpha%20S1-Casein%20Milk%20Protein%20Gene%20among%20Tropically%20Adapted%20Indian%20Native%20(Bos%20Indicus)%20Cattle&amp;author=A%20Kishore&amp;author=M%20Mukesh&amp;author=RC%20Sobti&amp;author=BP%20Mishra&amp;author=M%20Sodhi&amp;volume=2013&amp;publication_year=2013&amp;pages=926025&amp;pmid=25937984&amp;doi=10.5402/2013/926025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref032">
<span class="label">32.</span><cite>Wong ES, Schmitt BM, Kazachenka A, Thybert D, Redmond A, Connor F, et al. Interplay of cis and trans mechanisms driving transcription factor binding and gene expression evolution. Nat Commun. 2017;8(1):1092. doi: 10.1038/s41467-017-01037-x

</cite> [<a href="https://doi.org/10.1038/s41467-017-01037-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5653656/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29061983/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Interplay%20of%20cis%20and%20trans%20mechanisms%20driving%20transcription%20factor%20binding%20and%20gene%20expression%20evolution&amp;author=ES%20Wong&amp;author=BM%20Schmitt&amp;author=A%20Kazachenka&amp;author=D%20Thybert&amp;author=A%20Redmond&amp;volume=8&amp;issue=1&amp;publication_year=2017&amp;pages=1092&amp;pmid=29061983&amp;doi=10.1038/s41467-017-01037-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref033">
<span class="label">33.</span><cite>de Souza Carrocini GC, Venancio LPR, Bonini-Domingos CR. Screening of Transcription Factors Involved in Fetal Hemoglobin Regulation Using Phylogenetic Footprinting. Evol Bioinform Online. 2015;11:239–44. doi: 10.4137/EBO.S15364

</cite> [<a href="https://doi.org/10.4137/EBO.S15364" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4624090/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26543346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evol%20Bioinform%20Online&amp;title=Screening%20of%20Transcription%20Factors%20Involved%20in%20Fetal%20Hemoglobin%20Regulation%20Using%20Phylogenetic%20Footprinting&amp;author=GC%20de%20Souza%20Carrocini&amp;author=LPR%20Venancio&amp;author=CR%20Bonini-Domingos&amp;volume=11&amp;publication_year=2015&amp;pages=239-44&amp;pmid=26543346&amp;doi=10.4137/EBO.S15364&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref034">
<span class="label">34.</span><cite>Wang G, Xu Y, Guan SL, Zhang J, Jia Z, Hu L, et al. Comprehensive genomic analysis of CiPawPYL-PP2C-SnRK family genes in pecan (Carya illinoinensis) and functional characterization of CiPawSnRK2.1 under salt stress responses. Int J Biol Macromol. 2024;279(Pt 3):135366. doi: 10.1016/j.ijbiomac.2024.135366

</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2024.135366" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39244129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Biol%20Macromol&amp;title=Comprehensive%20genomic%20analysis%20of%20CiPawPYL-PP2C-SnRK%20family%20genes%20in%20pecan%20(Carya%20illinoinensis)%20and%20functional%20characterization%20of%20CiPawSnRK2.1%20under%20salt%20stress%20responses&amp;author=G%20Wang&amp;author=Y%20Xu&amp;author=SL%20Guan&amp;author=J%20Zhang&amp;author=Z%20Jia&amp;volume=279&amp;publication_year=2024&amp;pages=135366&amp;pmid=39244129&amp;doi=10.1016/j.ijbiomac.2024.135366&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref035">
<span class="label">35.</span><cite>Arizmendi-Izazaga A, Navarro-Tito N, Jiménez-Wences H, Evaristo-Priego A, Priego-Hernández VD, Dircio-Maldonado R, et al. Bioinformatics Analysis Reveals E6 and E7 of HPV 16 Regulate Metabolic Reprogramming in Cervical Cancer, Head and Neck Cancer, and Colorectal Cancer through the PHD2-VHL-CUL2-ELOC-HIF-1α Axis. Curr Issues Mol Biol. 2024;46(6):6199–222. doi: 10.3390/cimb46060370

</cite> [<a href="https://doi.org/10.3390/cimb46060370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11202971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38921041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Issues%20Mol%20Biol&amp;title=Bioinformatics%20Analysis%20Reveals%20E6%20and%20E7%20of%20HPV%2016%20Regulate%20Metabolic%20Reprogramming%20in%20Cervical%20Cancer,%20Head%20and%20Neck%20Cancer,%20and%20Colorectal%20Cancer%20through%20the%20PHD2-VHL-CUL2-ELOC-HIF-1%CE%B1%20Axis&amp;author=A%20Arizmendi-Izazaga&amp;author=N%20Navarro-Tito&amp;author=H%20Jim%C3%A9nez-Wences&amp;author=A%20Evaristo-Priego&amp;author=VD%20Priego-Hern%C3%A1ndez&amp;volume=46&amp;issue=6&amp;publication_year=2024&amp;pages=6199-222&amp;pmid=38921041&amp;doi=10.3390/cimb46060370&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref036">
<span class="label">36.</span><cite>Whiting JR, Booker TR, Rougeux C, Lind BM, Singh P, Lu M, et al. The genetic architecture of repeated local adaptation to climate in distantly related plants. Nat Ecol Evol. 2024;8(10):1933–47. doi: 10.1038/s41559-024-02514-5

</cite> [<a href="https://doi.org/10.1038/s41559-024-02514-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11461274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39187610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Ecol%20Evol&amp;title=The%20genetic%20architecture%20of%20repeated%20local%20adaptation%20to%20climate%20in%20distantly%20related%20plants&amp;author=JR%20Whiting&amp;author=TR%20Booker&amp;author=C%20Rougeux&amp;author=BM%20Lind&amp;author=P%20Singh&amp;volume=8&amp;issue=10&amp;publication_year=2024&amp;pages=1933-47&amp;pmid=39187610&amp;doi=10.1038/s41559-024-02514-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref037">
<span class="label">37.</span><cite>Possenti A, Di Cristina M, Nicastro C, Lunghi M, Messina V, Piro F, et al. Functional Characterization of the Thrombospondin-Related Paralogous Proteins Rhoptry Discharge Factors 1 and 2 Unveils Phenotypic Plasticity in Toxoplasma gondii Rhoptry Exocytosis. Front Microbiol. 2022;13:899243. doi: 10.3389/fmicb.2022.899243

</cite> [<a href="https://doi.org/10.3389/fmicb.2022.899243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9218915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35756016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Functional%20Characterization%20of%20the%20Thrombospondin-Related%20Paralogous%20Proteins%20Rhoptry%20Discharge%20Factors%201%20and%202%20Unveils%20Phenotypic%20Plasticity%20in%20Toxoplasma%20gondii%20Rhoptry%20Exocytosis&amp;author=A%20Possenti&amp;author=M%20Di%20Cristina&amp;author=C%20Nicastro&amp;author=M%20Lunghi&amp;author=V%20Messina&amp;volume=13&amp;publication_year=2022&amp;pages=899243&amp;pmid=35756016&amp;doi=10.3389/fmicb.2022.899243&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref038">
<span class="label">38.</span><cite>Bhattacharya M, Hota A, Kar A, Sankar Chini D, Chandra Malick R, Chandra Patra B, et al. In silico structural and functional modelling of Antifreeze protein (AFP) sequences of Ocean pout (Zoarces americanus, Bloch &amp; Schneider 1801). J Genet Eng Biotechnol. 2018;16(2):721–30. doi: 10.1016/j.jgeb.2018.08.004

</cite> [<a href="https://doi.org/10.1016/j.jgeb.2018.08.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6353770/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30733793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Genet%20Eng%20Biotechnol&amp;title=In%20silico%20structural%20and%20functional%20modelling%20of%20Antifreeze%20protein%20(AFP)%20sequences%20of%20Ocean%20pout%20(Zoarces%20americanus,%20Bloch%20&amp;%20Schneider%201801)&amp;author=M%20Bhattacharya&amp;author=A%20Hota&amp;author=A%20Kar&amp;author=D%20Sankar%20Chini&amp;author=R%20Chandra%20Malick&amp;volume=16&amp;issue=2&amp;publication_year=2018&amp;pages=721-30&amp;pmid=30733793&amp;doi=10.1016/j.jgeb.2018.08.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref039">
<span class="label">39.</span><cite>Gasteiger E, Hoogland C, Gattiker A, Duvaud S, Wilkins MR.; Appel RD, Bairoch A.
Protein Identification and Analysis Tools on the ExPASy Server. In <em>The Proteomics Protocols Handbook</em>. Walker JM, editor. Humana Press: Totowa, NJ, 2005; pp. 571–607 ISBN 978-1-58829-343-5.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Protein%20Identification%20and%20Analysis%20Tools%20on%20the%20ExPASy%20Server.%20In%20The%20Proteomics%20Protocols%20Handbook&amp;author=E%20Gasteiger&amp;author=C%20Hoogland&amp;author=A%20Gattiker&amp;author=S%20Duvaud&amp;author=MR.%20Wilkins&amp;publication_year=2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref040">
<span class="label">40.</span><cite>Letunic I, Bork P. Interactive Tree Of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. Nucleic Acids Res. 2021;49(W1):W293–6. doi: 10.1093/nar/gkab301

</cite> [<a href="https://doi.org/10.1093/nar/gkab301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8265157/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33885785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Interactive%20Tree%20Of%20Life%20(iTOL)%20v5:%20an%20online%20tool%20for%20phylogenetic%20tree%20display%20and%20annotation&amp;author=I%20Letunic&amp;author=P%20Bork&amp;volume=49&amp;publication_year=2021&amp;pmid=33885785&amp;doi=10.1093/nar/gkab301&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref041">
<span class="label">41.</span><cite>Kumar S, Stecher G, Tamura K. MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets. Mol Biol Evol. 2016;33(7):1870–4. doi: 10.1093/molbev/msw054

</cite> [<a href="https://doi.org/10.1093/molbev/msw054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8210823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27004904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=MEGA7:%20Molecular%20Evolutionary%20Genetics%20Analysis%20Version%207.0%20for%20Bigger%20Datasets&amp;author=S%20Kumar&amp;author=G%20Stecher&amp;author=K%20Tamura&amp;volume=33&amp;issue=7&amp;publication_year=2016&amp;pages=1870-4&amp;pmid=27004904&amp;doi=10.1093/molbev/msw054&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref042">
<span class="label">42.</span><cite>Sghaier H, Ghedira K, Benkahla A, Barkallah I. Basal DNA repair machinery is subject to positive selection in ionizing-radiation-resistant bacteria. BMC Genomics. 2008;9:297. doi: 10.1186/1471-2164-9-297

</cite> [<a href="https://doi.org/10.1186/1471-2164-9-297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2441631/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18570673/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Basal%20DNA%20repair%20machinery%20is%20subject%20to%20positive%20selection%20in%20ionizing-radiation-resistant%20bacteria&amp;author=H%20Sghaier&amp;author=K%20Ghedira&amp;author=A%20Benkahla&amp;author=I%20Barkallah&amp;volume=9&amp;publication_year=2008&amp;pages=297&amp;pmid=18570673&amp;doi=10.1186/1471-2164-9-297&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref043">
<span class="label">43.</span><cite>Olivieri DN, Mirete-Bachiller S, Gambón-Deza F. Insights into the evolution of IG genes in Amphibians and reptiles. Dev Comp Immunol. 2021;114:103868. doi: 10.1016/j.dci.2020.103868

</cite> [<a href="https://doi.org/10.1016/j.dci.2020.103868" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32949685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev%20Comp%20Immunol&amp;title=Insights%20into%20the%20evolution%20of%20IG%20genes%20in%20Amphibians%20and%20reptiles&amp;author=DN%20Olivieri&amp;author=S%20Mirete-Bachiller&amp;author=F%20Gamb%C3%B3n-Deza&amp;volume=114&amp;publication_year=2021&amp;pages=103868&amp;pmid=32949685&amp;doi=10.1016/j.dci.2020.103868&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref044">
<span class="label">44.</span><cite>Roach JC, Glusman G, Smit AFA, Huff CD, Hubley R, Shannon PT, et al. Analysis of genetic inheritance in a family quartet by whole-genome sequencing. Science. 2010;328(5978):636–9. doi: 10.1126/science.1186802

</cite> [<a href="https://doi.org/10.1126/science.1186802" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3037280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20220176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Analysis%20of%20genetic%20inheritance%20in%20a%20family%20quartet%20by%20whole-genome%20sequencing&amp;author=JC%20Roach&amp;author=G%20Glusman&amp;author=AFA%20Smit&amp;author=CD%20Huff&amp;author=R%20Hubley&amp;volume=328&amp;issue=5978&amp;publication_year=2010&amp;pages=636-9&amp;pmid=20220176&amp;doi=10.1126/science.1186802&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref045">
<span class="label">45.</span><cite>Aldea D, Hanna P, Munoz D, Espinoza J, Torrejon M, Sachs L, et al. Evolution of the vertebrate bone matrix: an expression analysis of the network forming collagen paralogues in amphibian osteoblasts. J Exp Zool B Mol Dev Evol. 2013;320(6):375–84. doi: 10.1002/jez.b.22511

</cite> [<a href="https://doi.org/10.1002/jez.b.22511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23677533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Zool%20B%20Mol%20Dev%20Evol&amp;title=Evolution%20of%20the%20vertebrate%20bone%20matrix:%20an%20expression%20analysis%20of%20the%20network%20forming%20collagen%20paralogues%20in%20amphibian%20osteoblasts&amp;author=D%20Aldea&amp;author=P%20Hanna&amp;author=D%20Munoz&amp;author=J%20Espinoza&amp;author=M%20Torrejon&amp;volume=320&amp;issue=6&amp;publication_year=2013&amp;pages=375-84&amp;pmid=23677533&amp;doi=10.1002/jez.b.22511&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref046">
<span class="label">46.</span><cite>Wang Y, Tang H, Debarry JD, Tan X, Li J, Wang X, et al. MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity. Nucleic Acids Res. 2012;40(7):e49. doi: 10.1093/nar/gkr1293

</cite> [<a href="https://doi.org/10.1093/nar/gkr1293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3326336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22217600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=MCScanX:%20a%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20collinearity&amp;author=Y%20Wang&amp;author=H%20Tang&amp;author=JD%20Debarry&amp;author=X%20Tan&amp;author=J%20Li&amp;volume=40&amp;issue=7&amp;publication_year=2012&amp;pmid=22217600&amp;doi=10.1093/nar/gkr1293&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref047">
<span class="label">47.</span><cite>Yang S, Zhang X, Li X, Yin X, Teng L, Ji G, et al. Evolutionary and Expression Analysis of MOV10 and MOV10L1 Reveals Their Origin, Duplication and Divergence. Int J Mol Sci. 2022;23(14):7523. doi: 10.3390/ijms23147523

</cite> [<a href="https://doi.org/10.3390/ijms23147523" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9319325/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35886872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Evolutionary%20and%20Expression%20Analysis%20of%20MOV10%20and%20MOV10L1%20Reveals%20Their%20Origin,%20Duplication%20and%20Divergence&amp;author=S%20Yang&amp;author=X%20Zhang&amp;author=X%20Li&amp;author=X%20Yin&amp;author=L%20Teng&amp;volume=23&amp;issue=14&amp;publication_year=2022&amp;pages=7523&amp;pmid=35886872&amp;doi=10.3390/ijms23147523&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref048">
<span class="label">48.</span><cite>Rasche H, Hiltemann S. Galactic Circos: User-friendly Circos plots within the Galaxy platform. Gigascience. 2020;9(6):giaa065. doi: 10.1093/gigascience/giaa065

</cite> [<a href="https://doi.org/10.1093/gigascience/giaa065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7291503/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32530465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Galactic%20Circos:%20User-friendly%20Circos%20plots%20within%20the%20Galaxy%20platform&amp;author=H%20Rasche&amp;author=S%20Hiltemann&amp;volume=9&amp;issue=6&amp;publication_year=2020&amp;pmid=32530465&amp;doi=10.1093/gigascience/giaa065&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref049">
<span class="label">49.</span><cite>Bailey TL, Boden M, Buske FA, Frith M, Grant CE, Clementi L, et al. MEME SUITE: tools for motif discovery and searching. Nucleic Acids Res. 2009;37(Web Server issue):W202-8. doi: 10.1093/nar/gkp335

</cite> [<a href="https://doi.org/10.1093/nar/gkp335" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2703892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19458158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=MEME%20SUITE:%20tools%20for%20motif%20discovery%20and%20searching&amp;author=TL%20Bailey&amp;author=M%20Boden&amp;author=FA%20Buske&amp;author=M%20Frith&amp;author=CE%20Grant&amp;volume=37&amp;publication_year=2009&amp;pmid=19458158&amp;doi=10.1093/nar/gkp335&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref050">
<span class="label">50.</span><cite>Chen C, Chen H, Zhang Y, Thomas HR, Frank MH, He Y, et al. TBtools: An Integrative Toolkit Developed for Interactive Analyses of Big Biological Data. Mol Plant. 2020;13(8):1194–202. doi: 10.1016/j.molp.2020.06.009

</cite> [<a href="https://doi.org/10.1016/j.molp.2020.06.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32585190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Plant&amp;title=TBtools:%20An%20Integrative%20Toolkit%20Developed%20for%20Interactive%20Analyses%20of%20Big%20Biological%20Data&amp;author=C%20Chen&amp;author=H%20Chen&amp;author=Y%20Zhang&amp;author=HR%20Thomas&amp;author=MH%20Frank&amp;volume=13&amp;issue=8&amp;publication_year=2020&amp;pages=1194-202&amp;pmid=32585190&amp;doi=10.1016/j.molp.2020.06.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref051">
<span class="label">51.</span><cite>Roelants K, Fry BG, Ye L, Stijlemans B, Brys L, Kok P, et al. Origin and functional diversification of an amphibian defense peptide arsenal. PLoS Genet. 2013;9(8):e1003662. doi: 10.1371/journal.pgen.1003662

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1003662" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3731216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23935531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&amp;title=Origin%20and%20functional%20diversification%20of%20an%20amphibian%20defense%20peptide%20arsenal&amp;author=K%20Roelants&amp;author=BG%20Fry&amp;author=L%20Ye&amp;author=B%20Stijlemans&amp;author=L%20Brys&amp;volume=9&amp;issue=8&amp;publication_year=2013&amp;pmid=23935531&amp;doi=10.1371/journal.pgen.1003662&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref052">
<span class="label">52.</span><cite>McGuffin LJ, Bryson K, Jones DT. The PSIPRED protein structure prediction server. Bioinformatics. 2000;16(4):404–5. doi: 10.1093/bioinformatics/16.4.404

</cite> [<a href="https://doi.org/10.1093/bioinformatics/16.4.404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10869041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The%20PSIPRED%20protein%20structure%20prediction%20server&amp;author=LJ%20McGuffin&amp;author=K%20Bryson&amp;author=DT%20Jones&amp;volume=16&amp;issue=4&amp;publication_year=2000&amp;pages=404-5&amp;pmid=10869041&amp;doi=10.1093/bioinformatics/16.4.404&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref053">
<span class="label">53.</span><cite>Bhat R, Chakraborty M, Mian IS, Newman SA. Structural divergence in vertebrate phylogeny of a duplicated prototype galectin. Genome Biol Evol. 2014;6(10):2721–30. doi: 10.1093/gbe/evu215

</cite> [<a href="https://doi.org/10.1093/gbe/evu215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4224342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25260584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol%20Evol&amp;title=Structural%20divergence%20in%20vertebrate%20phylogeny%20of%20a%20duplicated%20prototype%20galectin&amp;author=R%20Bhat&amp;author=M%20Chakraborty&amp;author=IS%20Mian&amp;author=SA%20Newman&amp;volume=6&amp;issue=10&amp;publication_year=2014&amp;pages=2721-30&amp;pmid=25260584&amp;doi=10.1093/gbe/evu215&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref054">
<span class="label">54.</span><cite>Powell HR, Islam SA, David A, Sternberg MJE. Phyre2.2: A Community Resource for Template-based Protein Structure Prediction. J Mol Biol. 2025;437(15):168960. doi: 10.1016/j.jmb.2025.168960

</cite> [<a href="https://doi.org/10.1016/j.jmb.2025.168960" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7617537/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40133783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Phyre2.2:%20A%20Community%20Resource%20for%20Template-based%20Protein%20Structure%20Prediction&amp;author=HR%20Powell&amp;author=SA%20Islam&amp;author=A%20David&amp;author=MJE%20Sternberg&amp;volume=437&amp;issue=15&amp;publication_year=2025&amp;pages=168960&amp;pmid=40133783&amp;doi=10.1016/j.jmb.2025.168960&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref055">
<span class="label">55.</span><cite>Chen J, Zhang C-Y, Chen J-Y, Seah RWX, Zhang L, Ma L, et al. Host defence peptide LEAP2 contributes to antimicrobial activity in a mustache toad (Leptobrachium liui). BMC Vet Res. 2023;19(1):47. doi: 10.1186/s12917-023-03606-3

</cite> [<a href="https://doi.org/10.1186/s12917-023-03606-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9921027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36765333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Vet%20Res&amp;title=Host%20defence%20peptide%20LEAP2%20contributes%20to%20antimicrobial%20activity%20in%20a%20mustache%20toad%20(Leptobrachium%20liui)&amp;author=J%20Chen&amp;author=C-Y%20Zhang&amp;author=J-Y%20Chen&amp;author=RWX%20Seah&amp;author=L%20Zhang&amp;volume=19&amp;issue=1&amp;publication_year=2023&amp;pages=47&amp;pmid=36765333&amp;doi=10.1186/s12917-023-03606-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref056">
<span class="label">56.</span><cite>Waterhouse AM, Studer G, Robin X, Bienert S, Tauriello G, Schwede T. The structure assessment web server: for proteins, complexes and more. Nucleic Acids Res. 2024;52(W1):W318–23. doi: 10.1093/nar/gkae270

</cite> [<a href="https://doi.org/10.1093/nar/gkae270" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223858/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38634802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20structure%20assessment%20web%20server:%20for%20proteins,%20complexes%20and%20more&amp;author=AM%20Waterhouse&amp;author=G%20Studer&amp;author=X%20Robin&amp;author=S%20Bienert&amp;author=G%20Tauriello&amp;volume=52&amp;publication_year=2024&amp;pmid=38634802&amp;doi=10.1093/nar/gkae270&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref057">
<span class="label">57.</span><cite>Rashmi R, Nandi C, Majumdar S. Evolutionary analysis of THAP9 transposase: conserved regions, novel motifs. 2021.</cite> [<a href="https://doi.org/10.1016/j.crstbi.2023.100113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10824691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38292821/" class="usa-link">PubMed</a>]</li>
<li id="pone.0323740.ref058">
<span class="label">58.</span><cite>Messeguer X, Escudero R, Farré D, Núñez O, Martínez J, Albà MM. PROMO: detection of known transcription regulatory elements using species-tailored searches. Bioinformatics. 2002;18(2):333–4. doi: 10.1093/bioinformatics/18.2.333

</cite> [<a href="https://doi.org/10.1093/bioinformatics/18.2.333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11847087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PROMO:%20detection%20of%20known%20transcription%20regulatory%20elements%20using%20species-tailored%20searches&amp;author=X%20Messeguer&amp;author=R%20Escudero&amp;author=D%20Farr%C3%A9&amp;author=O%20N%C3%BA%C3%B1ez&amp;author=J%20Mart%C3%ADnez&amp;volume=18&amp;issue=2&amp;publication_year=2002&amp;pages=333-4&amp;pmid=11847087&amp;doi=10.1093/bioinformatics/18.2.333&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref059">
<span class="label">59.</span><cite>Dai T-Y, Lan J-J, Gao R-L, Zhao Y-N, Yu X-L, Liang S-X, et al. Panaxdiol saponins component promotes hematopoiesis by regulating GATA transcription factors of intracellular signaling pathway in mouse bone marrow. Ann Transl Med. 2022;10(2):38. doi: 10.21037/atm-21-4800

</cite> [<a href="https://doi.org/10.21037/atm-21-4800" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8848385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35282082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann%20Transl%20Med&amp;title=Panaxdiol%20saponins%20component%20promotes%20hematopoiesis%20by%20regulating%20GATA%20transcription%20factors%20of%20intracellular%20signaling%20pathway%20in%20mouse%20bone%20marrow&amp;author=T-Y%20Dai&amp;author=J-J%20Lan&amp;author=R-L%20Gao&amp;author=Y-N%20Zhao&amp;author=X-L%20Yu&amp;volume=10&amp;issue=2&amp;publication_year=2022&amp;pages=38&amp;pmid=35282082&amp;doi=10.21037/atm-21-4800&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref060">
<span class="label">60.</span><cite>Morceau F, Schnekenburger M, Dicato M, Diederich M. GATA-1: friends, brothers, and coworkers. Ann N Y Acad Sci. 2004;1030:537–54. doi: 10.1196/annals.1329.064

</cite> [<a href="https://doi.org/10.1196/annals.1329.064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15659837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&amp;title=GATA-1:%20friends,%20brothers,%20and%20coworkers&amp;author=F%20Morceau&amp;author=M%20Schnekenburger&amp;author=M%20Dicato&amp;author=M%20Diederich&amp;volume=1030&amp;publication_year=2004&amp;pages=537-54&amp;pmid=15659837&amp;doi=10.1196/annals.1329.064&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref061">
<span class="label">61.</span><cite>Mukherjee JJ, Gupta SK, Kumar S. Inhibition of benzopyrene diol epoxide-induced apoptosis by cadmium(II) is AP-1-independent: role of extracelluler signal related kinase. Chem Biol Interact. 2008;172(1):72–80. doi: 10.1016/j.cbi.2007.11.002

</cite> [<a href="https://doi.org/10.1016/j.cbi.2007.11.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2972655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18093576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20Biol%20Interact&amp;title=Inhibition%20of%20benzopyrene%20diol%20epoxide-induced%20apoptosis%20by%20cadmium(II)%20is%20AP-1-independent:%20role%20of%20extracelluler%20signal%20related%20kinase&amp;author=JJ%20Mukherjee&amp;author=SK%20Gupta&amp;author=S%20Kumar&amp;volume=172&amp;issue=1&amp;publication_year=2008&amp;pages=72-80&amp;pmid=18093576&amp;doi=10.1016/j.cbi.2007.11.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref062">
<span class="label">62.</span><cite>Lehtinen MK, Yuan Z, Boag PR, Yang Y, Villén J, Becker EBE, et al. A conserved MST-FOXO signaling pathway mediates oxidative-stress responses and extends life span. Cell. 2006;125(5):987–1001. doi: 10.1016/j.cell.2006.03.046

</cite> [<a href="https://doi.org/10.1016/j.cell.2006.03.046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16751106/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20conserved%20MST-FOXO%20signaling%20pathway%20mediates%20oxidative-stress%20responses%20and%20extends%20life%20span&amp;author=MK%20Lehtinen&amp;author=Z%20Yuan&amp;author=PR%20Boag&amp;author=Y%20Yang&amp;author=J%20Vill%C3%A9n&amp;volume=125&amp;issue=5&amp;publication_year=2006&amp;pages=987-1001&amp;pmid=16751106&amp;doi=10.1016/j.cell.2006.03.046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref063">
<span class="label">63.</span><cite>Renault VM, Thekkat PU, Hoang KL, White JL, Brady CA, Kenzelmann Broz D, et al. The pro-longevity gene FoxO3 is a direct target of the p53 tumor suppressor. Oncogene. 2011;30(29):3207–21. doi: 10.1038/onc.2011.35

</cite> [<a href="https://doi.org/10.1038/onc.2011.35" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3136551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21423206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=The%20pro-longevity%20gene%20FoxO3%20is%20a%20direct%20target%20of%20the%20p53%20tumor%20suppressor&amp;author=VM%20Renault&amp;author=PU%20Thekkat&amp;author=KL%20Hoang&amp;author=JL%20White&amp;author=CA%20Brady&amp;volume=30&amp;issue=29&amp;publication_year=2011&amp;pages=3207-21&amp;pmid=21423206&amp;doi=10.1038/onc.2011.35&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref064">
<span class="label">64.</span><cite>Grönroos E, Terentiev AA, Punga T, Ericsson J. YY1 inhibits the activation of the p53 tumor suppressor in response to genotoxic stress. Proc Natl Acad Sci U S A. 2004;101(33):12165–70. doi: 10.1073/pnas.0402283101

</cite> [<a href="https://doi.org/10.1073/pnas.0402283101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC514451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15295102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=YY1%20inhibits%20the%20activation%20of%20the%20p53%20tumor%20suppressor%20in%20response%20to%20genotoxic%20stress&amp;author=E%20Gr%C3%B6nroos&amp;author=AA%20Terentiev&amp;author=T%20Punga&amp;author=J%20Ericsson&amp;volume=101&amp;issue=33&amp;publication_year=2004&amp;pages=12165-70&amp;pmid=15295102&amp;doi=10.1073/pnas.0402283101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0323740.ref065">
<span class="label">65.</span><cite>Kumar N, Srivillibhuthur M, Joshi S, Walton KD, Zhou A, Faller WJ, et al. A YY1-dependent increase in aerobic metabolism is indispensable for intestinal organogenesis. Development. 2016;143(20):3711–22. doi: 10.1242/dev.137992

</cite> [<a href="https://doi.org/10.1242/dev.137992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5087649/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27802136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=A%20YY1-dependent%20increase%20in%20aerobic%20metabolism%20is%20indispensable%20for%20intestinal%20organogenesis&amp;author=N%20Kumar&amp;author=M%20Srivillibhuthur&amp;author=S%20Joshi&amp;author=KD%20Walton&amp;author=A%20Zhou&amp;volume=143&amp;issue=20&amp;publication_year=2016&amp;pages=3711-22&amp;pmid=27802136&amp;doi=10.1242/dev.137992&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section></section><article class="sub-article" id="pone.0323740.r001"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0323740.r001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0323740.r001</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 0</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Behera%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Bijay Behera</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Bijay Behera</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Behera%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bijay Behera</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.c" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.c" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.c" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Bijay Behera</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.c" class="d-panel p" style="display: none">
<div>© 2025 Bijay Behera</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>11 Feb 2025</em>
</p>
<p>Dear Dr. Naz,</p>
<p>Thank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE’s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.</p>
<p>Please submit your revised manuscript by Mar 28 2025 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at <span>plosone@plos.org</span> . When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pone/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file.</p>
<ul class="list" style="list-style-type:disc">
<li><p>A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'.</p></li>
<li><p>A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.</p></li>
<li><p>An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.</p></li>
</ul>
<p>If you would like to make changes to your financial disclosure, please include your updated statement in your cover letter. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter.</p>
<p>If applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see: <a href="https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols</a> . Additionally, PLOS ONE offers an option for publishing peer-reviewed Lab Protocol articles, which describe protocols hosted on protocols.io. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a> .</p>
<p>We look forward to receiving your revised manuscript.</p>
<p>Kind regards,</p>
<p>Bijay Kumar Behera, Ph.D.</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Journal Requirements:</p>
<p>When submitting your revision, we need you to address these additional requirements.</p>
<p>1. Please ensure that your manuscript meets PLOS ONE's style requirements, including those for file naming. The PLOS ONE style templates can be found at </p>
<p><a href="https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf</a> and </p>
<p>
<a href="https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf</a>
</p>
<p>2. Your ethics statement should only appear in the Methods section of your manuscript. If your ethics statement is written in any section besides the Methods, please move it to the Methods section and delete it from any other section. Please ensure that your ethics statement is included in your manuscript, as the ethics statement entered into the online submission form will not be published alongside your manuscript. </p>
<p>Additional Editor Comments:</p>
<p>The recent manuscript has its own merit. According to the two reviewers recommendations, my decision is to Major revision of the manuscript. Kindly see the two reviewers comment.</p>
<p>[Note: HTML markup is below. Please do not edit.]</p>
<p>Reviewers' comments:</p>
<p>Reviewer's Responses to Questions</p>
<p>
<strong>Comments to the Author</strong>
</p>
<p>1. Is the manuscript technically sound, and do the data support the conclusions?</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>2. Has the statistical analysis been performed appropriately and rigorously? --&gt;?&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>3. Have the authors made all data underlying the findings in their manuscript fully available??&gt;</p>
<p>The <a href="http://www.plosone.org/static/policies.action#sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a></p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>4. Is the manuscript presented in an intelligible fashion and written in standard English??&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: No</p>
<p>**********</p>
<p>Reviewer #1: The article “Investigating the Role of FOX Gene Family in Development and Stress Response in Labeo rohita: A Multi-faceted Analysis of Phylogeny and Genome Characterization” is well written.</p>
<p>The authors need to consider a few points mentioned below before publication.</p>
<p>•The Abstract section is not justified.</p>
<p>•L499: If possible, for secondary structure, authors are advised to use more than one program for their validation with suitable references</p>
<p>•For using iTol, the file type used for designing the phylogeny should be mentioned.</p>
<p>•The study only highlights the available sequences of the FOX gene in the NCBI database. The authors haven’t confirmed the 5’-UTR and 3’-Poly A tail by using RACE PCR. Using the terms in conclusion, like detailed genome-wide analysis, might not be true. If possible, avoid using terms like that.</p>
<p>• The discussion section is well written. The authors advised to revise accordingly.</p>
<p>Reviewer #2: Comments about the manuscript:</p>
<p>The manuscript entitled “Investigating the role of FOX gene family in development and stress response in Labeo rohita: a multi-faceted analysis of phylogeny and genome characterization” aims to explore the evolutionary and molecular roles of FOX genes in L. rohita using computational analysis. The introduction is streamlined and clear but the other section should be improved. The manuscript may be published with a revision as detailed below.</p>
<p>•In Abstract, authors should specifically state the need of this research.</p>
<p>•Line No 134-135: Bootstrap consensus values were calculated for each node in the analysis……………. In Figure 1, no bootstrap values are visible.</p>
<p>•Figure caption should be improved with details.</p>
<p>•Discussion section should be revised by the authors. Try to correlate the results obtained with each other.</p>
<p>•Line No 499: Authors are advised to use multiple programs for validation with appropriate references.</p>
<p>•4.2. Phylogenetic Analysis of FOX genes superfamily: Authors should write their methodology in details.</p>
<p>•Line No 528: Avoid the term ‘detailed genome-wide analysis of the FOX gene family’, because authors only used computational tools for this study.</p>
<p>**********</p>
<p><a href="https://journals.plos.org/plosone/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a></p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: No</p>
<p>**********</p>
<p>[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files.]</p>
<p>While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, <a href="https://pacev2.apexcovantage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pacev2.apexcovantage.com/</a> . PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at <span>figures@plos.org</span> . Please note that Supporting Information files do not need this step.</p></section></article><article class="sub-article" id="pone.0323740.r002"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. 2025 Aug 21;20(8):e0323740. doi: <a href="https://doi.org/10.1371/journal.pone.0323740.r002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0323740.r002</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 1</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.d" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.d" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.d" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub2" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.d" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>27 Mar 2025</em>
</p>
<p>Dear Editor and Reviewers,</p>
<p>We would like to express our sincere gratitude for the time and effort invested in reviewing our manuscript entitled “Investigating the Role of FOX Gene Family in Development and Stress Response in Labeo rohita: A Multi-faceted Analysis of Phylogeny and Genome Characterization.” Your insightful comments have been invaluable in enhancing the quality and clarity of our work. Below, we provide a detailed, point-by-point response to all comments raised.</p>
<p>Response to Reviewer #1</p>
<p>Comment: “The Abstract section is not justified.”</p>
<p>Response: We have revised the abstract to ensure full justification in accordance with the journal’s formatting guidelines. The updated abstract now meets the required presentation style.</p>
<p>Comment: “If possible, for secondary structure, authors are advised to use more than one program for their validation with suitable references.”</p>
<p>Response: We thank the reviewer for the suggestion. Our manuscript now specifies that FOX protein secondary structure was predicted using PSIPRED and independently validated with Pyre2, with both methods yielding consistent results. All relevant references are now provided in the manuscript.</p>
<p>Comment: “For using iTol, the file type used for designing the phylogeny should be mentioned.”</p>
<p>Response: We have now specified in the Methods section that the phylogenetic tree was generated using the Newick format file generated by MEGA11.</p>
<p>Comment: “The study only highlights the available sequences of the FOX gene in the NCBI database. The authors haven’t confirmed the 5’-UTR and 3’-Poly A tail by using RACE PCR. Using the terms in conclusion, like detailed genome-wide analysis, might not be true. If possible, avoid using terms like that.”</p>
<p>Response: We acknowledge this concern and have revised the manuscript accordingly. The text now clearly states that the study is based on computational analysis of sequences available in the NCBI database. The term “detailed genome-wide analysis” has been replaced with “comprehensive computational analysis” to more accurately reflect the scope of our work.</p>
<p>Comment: “The discussion section is well written. The authors advised to revise accordingly.”</p>
<p>Response: We appreciate the positive feedback. Minor revisions have been made in the discussion to enhance clarity and to improve the integration and correlation of the results.</p>
<p>Response to Reviewer #2</p>
<p>Comment: “In Abstract, authors should specifically state the need of this research.”</p>
<p>Response: The abstract has been updated to explicitly state the motivation behind our study, highlighting the significance of understanding the role of FOX genes in the developmental and stress response mechanisms of Labeo rohita.</p>
<p>Comment: “Bootstrap consensus values were calculated for each node in the analysis……………. In Figure 1, no bootstrap values are visible.”</p>
<p>Response: We have revised Figure 1 to include bootstrap values for each node. The figure legend has been updated to describe these values and the method of calculation in detail.</p>
<p>Comment: “Figure caption should be improved with details.”</p>
<p>Response: The caption for Figure 1 has been expanded to include additional details.</p>
<p>Comment: “Discussion section should be revised by the authors. Try to correlate the results obtained with each other.”</p>
<p>Response: We have restructured the discussion to provide a more cohesive interpretation of our findings. The revised version more clearly correlates the outcomes.</p>
<p>Comment: “Authors are advised to use multiple programs for validation with appropriate references.”</p>
<p>Response: In our study, FOX protein secondary structure was accurately predicted using PSIPRED and validated with Pyre2 for 2D analysis. We have now incorporated SWISS-MODEL for 3D structure validation, and relevant references have been added to the manuscript.</p>
<p>Comment: “4.2. Phylogenetic Analysis of FOX genes superfamily: Authors should write their methodology in details.”</p>
<p>Response: We have expanded Section 4.2 to include a detailed description of our phylogenetic analysis methodology to enhance reproducibility.</p>
<p>Comment: “Avoid the term ‘detailed genome-wide analysis of the FOX gene family’, because authors only used computational tools for this study.”</p>
<p>Response: In line with the feedback, we have replaced the term “detailed genome-wide analysis” with “comprehensive computational analysis” to more accurately represent our study’s scope.</p>
<p>We believe that these revisions have significantly improved the manuscript and have addressed all the concerns raised. We appreciate the reviewers’ constructive feedback and are hopeful that the revised manuscript meets the journal’s standards for publication.</p>
<p>Thank you for considering our revised manuscript. We look forward to your positive response.</p>
<section class="sm xbox font-sm" id="pone.0323740.s004"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to Reviewers.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370200/bin/pone.0323740.s004.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0323740.s004.docx</a><sup> (16.5KB, docx) </sup>
</div></div></section></section></article><article class="sub-article" id="pone.0323740.r003"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0323740.r003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0323740.r003</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 1</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Behera%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Bijay Behera</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Bijay Behera</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Behera%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bijay Behera</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.e" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.e" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.e" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Bijay Behera</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.e" class="d-panel p" style="display: none">
<div>© 2025 Bijay Behera</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>8 Apr 2025</em>
</p>
<p>Dear Dr. Naz,</p>
<p>Thank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE’s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.</p>
<p>Please submit your revised manuscript by May 23 2025 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at <span>plosone@plos.org</span> . When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pone/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file.</p>
<ul class="list" style="list-style-type:disc">
<li><p>A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'.</p></li>
<li><p>A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.</p></li>
<li><p>An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.</p></li>
</ul>
<p>If applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see: <a href="https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols</a> . Additionally, PLOS ONE offers an option for publishing peer-reviewed Lab Protocol articles, which describe protocols hosted on protocols.io. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a> .</p>
<p>We look forward to receiving your revised manuscript.</p>
<p>Kind regards,</p>
<p>Bijay Kumar Behera, Ph.D.</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Journal Requirements:</p>
<p>Please review your reference list to ensure that it is complete and correct. If you have cited papers that have been retracted, please include the rationale for doing so in the manuscript text, or remove these references and replace them with relevant current references. Any changes to the reference list should be mentioned in the rebuttal letter that accompanies your revised manuscript. If you need to cite a retracted article, indicate the article’s retracted status in the References list and also include a citation and full reference for the retraction notice.</p>
<p>Additional Editor Comments:</p>
<p>Dear Author,</p>
<p>Now the manuscript has been revised. One minor comment is in the manuscript. Kindly see the reviewers comments and revise accordingly.</p>
<p>[Note: HTML markup is below. Please do not edit.]</p>
<p>Reviewers' comments:</p>
<p>Reviewer's Responses to Questions</p>
<p>
<strong>Comments to the Author</strong>
</p>
<p>Reviewer #1: All comments have been addressed</p>
<p>Reviewer #2: All comments have been addressed</p>
<p>**********</p>
<p>2. Is the manuscript technically sound, and do the data support the conclusions??&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>3. Has the statistical analysis been performed appropriately and rigorously? --&gt;?&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>4. Have the authors made all data underlying the findings in their manuscript fully available??&gt;</p>
<p>The <a href="http://www.plosone.org/static/policies.action#sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a></p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>5. Is the manuscript presented in an intelligible fashion and written in standard English??&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>Reviewer #1: (No Response)</p>
<p>Reviewer #2: The authors have addressed all the comments satisfactorily. Only one issue I have found in the Revised manuscript which is mentioned below:</p>
<p>Line no 137-138: These sequences were organized into 12 major clades ……….. But there are 11 clades. In Figure 1, they have also clustered the phylogenetic tree into 11 clades. Please rectify it.</p>
<p>The manuscript may be accepted after fixing the issue.</p>
<p>**********</p>
<p><a href="https://journals.plos.org/plosone/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a></p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: No</p>
<p>**********</p>
<p>[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files.]</p>
<p>While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, <a href="https://pacev2.apexcovantage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pacev2.apexcovantage.com/</a> . PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at <span>figures@plos.org</span></p>
<section class="sm xbox font-sm" id="pone.0323740.s005"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Review comments 2.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370200/bin/pone.0323740.s005.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0323740.s005.docx</a><sup> (14.7KB, docx) </sup>
</div></div></section></section></article><article class="sub-article" id="pone.0323740.r004"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. 2025 Aug 21;20(8):e0323740. doi: <a href="https://doi.org/10.1371/journal.pone.0323740.r004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0323740.r004</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 2</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.f" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.f" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.f" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub4" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.f" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>11 Apr 2025</em>
</p>
<p>Dear Editor and Reviewers,</p>
<p>We would like to express our sincere gratitude for the time and effort you have dedicated to reviewing our manuscript entitled “Investigating the Role of FOX Gene Family in Development and Stress Response in Labeo rohita: A Multi-faceted Analysis of Phylogeny and Genome Characterization.” Your thoughtful and constructive feedback has been instrumental in improving the overall quality and clarity of our work.</p>
<p>Following the recent round of minor revision, we have carefully addressed the remaining comment provided by Reviewer #2, as detailed below:</p>
<p>Response to Reviewer #2</p>
<p>Comment:</p>
<p>Line no. 137–138: “These sequences were organized into 12 major clades...” But there are 11 clades. In Figure 1, they have also clustered the phylogenetic tree into 11 clades. Please rectify it.</p>
<p>Response:</p>
<p>The mentioned line has been corrected in the revised manuscript, changing the number of clades from 12 to 11 to accurately reflect the phylogenetic analysis and Figure 1. We have reviewed the manuscript thoroughly to ensure consistency throughout the text.</p>
<p>We appreciate your continued support and are hopeful that the manuscript is now suitable for acceptance. Thank you once again for your guidance and constructive suggestions throughout the review process.</p>
<p>Sincerely,</p>
<p>Shabana Naz</p>
<p>Department of Zoology, Government Sadiq College Women University, Bahawalpur 63100, Punjab, Pakistan.</p>
<p>Email: drshabananaz@gcuf.edu.pk</p>
<section class="sm xbox font-sm" id="pone.0323740.s006"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response_to_Reviewers_auresp_2.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370200/bin/pone.0323740.s006.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0323740.s006.docx</a><sup> (14.7KB, docx) </sup>
</div></div></section></section></article><article class="sub-article" id="pone.0323740.r005"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0323740.r005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0323740.r005</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 2</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Behera%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Bijay Behera</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Bijay Behera</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Behera%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bijay Behera</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.g" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.g" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.g" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Bijay Behera</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.g" class="d-panel p" style="display: none">
<div>© 2025 Bijay Behera</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>15 Apr 2025</em>
</p>
<p>Investigating the Role of FOX Gene Family in Development and Stress Response in Labeo rohita: A Multi-faceted Analysis of Phylogeny and Genome Characterization</p>
<p>PONE-D-24-60377R2</p>
<p>Dear Dr. Naz,</p>
<p>We’re pleased to inform you that your manuscript has been judged scientifically suitable for publication and will be formally accepted for publication once it meets all outstanding technical requirements.</p>
<p>Within one week, you’ll receive an e-mail detailing the required amendments. When these have been addressed, you’ll receive a formal acceptance letter and your manuscript will be scheduled for publication.</p>
<p>An invoice will be generated when your article is formally accepted. Please note, if your institution has a publishing partnership with PLOS and your article meets the relevant criteria, all or part of your publication costs will be covered. Please make sure your user information is up-to-date by logging into Editorial Manager at <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Editorial Manager®</a>  and clicking the ‘Update My Information' link at the top of the page. If you have any questions relating to publication charges, please contact our Author Billing department directly at authorbilling@plos.org.</p>
<p>If your institution or institutions have a press office, please notify them about your upcoming paper to help maximize its impact. If they’ll be preparing press materials, please inform our press team as soon as possible -- no later than 48 hours after receiving the formal acceptance. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact onepress@plos.org.</p>
<p>Kind regards,</p>
<p>Bijay Kumar Behera, Ph.D.</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Additional Editor Comments (optional):</p>
<p>The manuscript entitled "Investigating the Role of FOX Gene Family in Development and Stress Response in Labeo rohita: A Multi-faceted Analysis of Phylogeny and Genome Characterization" has been accepted for publication.</p>
<p>Reviewers' comments:</p></section></article><article class="sub-article" id="pone.0323740.r006"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0323740.r006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0323740.r006</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Acceptance letter</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Behera%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Bijay Behera</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Bijay Behera</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Behera%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bijay Behera</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.h" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.h" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.h" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Bijay Behera</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.h" class="d-panel p" style="display: none">
<div>© 2025 Bijay Behera</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>PONE-D-24-60377R2</p>
<p>PLOS ONE</p>
<p>Dear Dr. Naz,</p>
<p>I'm pleased to inform you that your manuscript has been deemed suitable for publication in PLOS ONE. Congratulations! Your manuscript is now being handed over to our production team.</p>
<p>At this stage, our production department will prepare your paper for publication. This includes ensuring the following:</p>
<p>* All references, tables, and figures are properly cited</p>
<p>* All relevant supporting information is included in the manuscript submission,</p>
<p>* There are no issues that prevent the paper from being properly typeset</p>
<p>You will receive further instructions from the production team, including instructions on how to review your proof when it is ready. Please keep in mind that we are working through a large volume of accepted articles, so please give us a few days to review your paper and let you know the next and final steps.</p>
<p>Lastly, if your institution or institutions have a press office, please let them know about your upcoming paper now to help maximize its impact. If they'll be preparing press materials, please inform our press team within the next 48 hours. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact onepress@plos.org.</p>
<p>You will receive an invoice from PLOS for your publication fee after your manuscript has reached the completed accept phase. If you receive an email requesting payment before acceptance or for any other service, this may be a phishing scheme. Learn how to identify phishing emails and protect your accounts at <a href="https://explore.plos.org/phishing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://explore.plos.org/phishing</a>.</p>
<p>If we can help with anything else, please email us at customercare@plos.org.</p>
<p>Thank you for submitting your work to PLOS ONE and supporting open access.</p>
<p>Kind regards,</p>
<p>PLOS ONE Editorial Office Staff</p>
<p>on behalf of</p>
<p>Dr. Bijay Kumar Behera</p>
<p>Academic Editor</p>
<p>PLOS ONE</p></section></article><article class="sub-article" id="_ad93_"><section class="pmc-layout__citation font-secondary font-xs"><div></div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Associated Data</h1></hgroup><ul class="d-buttons inline-list"></ul>
<div class="d-panels font-secondary-light"></div>
<div></div>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
</div></section><section class="body sub-article-body"><section id="_adsm93_" lang="en" class="supplementary-materials"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Table. Genomic properties of the FOX gene superfamily in <em>L. rohita.</em></span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370200/bin/pone.0323740.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0323740.s001.docx</a><sup> (16.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S1 Fig</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370200/bin/pone.0323740.s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0323740.s002.pdf</a><sup> (3.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S2 Fig</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370200/bin/pone.0323740.s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0323740.s003.pdf</a><sup> (1.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to Reviewers.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370200/bin/pone.0323740.s004.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0323740.s004.docx</a><sup> (16.5KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Review comments 2.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370200/bin/pone.0323740.s005.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0323740.s005.docx</a><sup> (14.7KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response_to_Reviewers_auresp_2.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370200/bin/pone.0323740.s006.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0323740.s006.docx</a><sup> (14.7KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The data are all contained within the manuscript.</p></section></section></article><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS One are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pone.0323740"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pone.0323740.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370200/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370200/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370200%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370200/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370200/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370200/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40839711/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370200/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40839711/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370200/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370200/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="WTUll32gqalscFcFEp02oblvxDHyFQPHByUAIBW8AoUC28BcaJMk8VGzFi4OBI0p">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
