
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Synthesis of novel N1 functionalized diazepinone analogues via a base-mediated C–N cross coupling reaction as reverse transcriptase inhibitors: theoretical and experimental investigations - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4BC618AF4380305BC61000EBDD249.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="rscadv">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376879/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="RSC Advances">
<meta name="citation_title" content="Synthesis of novel N1 functionalized diazepinone analogues via a base-mediated C–N cross coupling reaction as reverse transcriptase inhibitors: theoretical and experimental investigations">
<meta name="citation_author" content="Shivangi Jaiswal">
<meta name="citation_author_institution" content="Department of Chemistry, Banasthali Vidyapith, Banasthali, Rajasthan, India, Email: jayadwivedi@yahoo.co.in">
<meta name="citation_author" content="Smita Jain">
<meta name="citation_author_institution" content="Department of Pharmacy, School of Chemical Sciences and Pharmacy, Central University of Rajasthan, Kishangarh, Rajasthan, India">
<meta name="citation_author" content="Achal Mukhija">
<meta name="citation_author_institution" content="Department of Chemistry, Banasthali Vidyapith, Banasthali, Rajasthan, India, Email: jayadwivedi@yahoo.co.in">
<meta name="citation_author" content="Kanika Verma">
<meta name="citation_author_institution" content="Department of Internal Medicine, Division of Cardiology, LSU Health Sciences Center–Shreveport, Louisiana, USA">
<meta name="citation_author" content="Sonika Jain">
<meta name="citation_author_institution" content="Department of Chemistry, Banasthali Vidyapith, Banasthali, Rajasthan, India, Email: jayadwivedi@yahoo.co.in">
<meta name="citation_author" content="Dharma Kishore">
<meta name="citation_author_institution" content="Department of Chemistry, Banasthali Vidyapith, Banasthali, Rajasthan, India, Email: jayadwivedi@yahoo.co.in">
<meta name="citation_author" content="Jaya Dwivedi">
<meta name="citation_author_institution" content="Department of Chemistry, Banasthali Vidyapith, Banasthali, Rajasthan, India, Email: jayadwivedi@yahoo.co.in">
<meta name="citation_author" content="Swapnil Sharma">
<meta name="citation_author_institution" content="Department of Pharmacy, Banasthali Vidyapith, Banasthali, Rajasthan, India">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="15">
<meta name="citation_issue" content="36">
<meta name="citation_firstpage" content="29160">
<meta name="citation_doi" content="10.1039/d5ra03504j">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376879/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376879/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376879/pdf/RA-015-D5RA03504J.pdf">
<meta name="description" content="The sodium hydrogen orthophosphate (Na2HPO4) base was utilized in a stereospecific C–N coupling reaction to synthesize a novel series of nevirapine analogues in two-step reactions. This base is moisture tolerant, commercially available and makes the ...">
<meta name="og:title" content="Synthesis of novel N1 functionalized diazepinone analogues via a base-mediated C–N cross coupling reaction as reverse transcriptase inhibitors: theoretical and experimental investigations">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The sodium hydrogen orthophosphate (Na2HPO4) base was utilized in a stereospecific C–N coupling reaction to synthesize a novel series of nevirapine analogues in two-step reactions. This base is moisture tolerant, commercially available and makes the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376879/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12376879">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1039/d5ra03504j"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/RA-015-D5RA03504J.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12376879%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12376879/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12376879/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376879/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-rscadv.png" alt="RSC Advances logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to RSC Advances" title="Link to RSC Advances" shape="default" href="https://www.rsc.org/journals-books-databases/about-journals/rsc-advances/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">RSC Adv</button></div>. 2025 Aug 18;15(36):29160–29175. doi: <a href="https://doi.org/10.1039/d5ra03504j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1039/d5ra03504j</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22RSC%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22RSC%20Adv%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22RSC%20Adv%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22RSC%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Synthesis of novel N1 functionalized diazepinone analogues <em>via</em> a base-mediated C–N cross coupling reaction as reverse transcriptase inhibitors: theoretical and experimental investigations</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaiswal%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Shivangi Jaiswal</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Shivangi Jaiswal</span></h3>
<div class="p">
<sup>a</sup>Department of Chemistry, Banasthali Vidyapith, Banasthali, Rajasthan, India, Email: jayadwivedi@yahoo.co.in</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaiswal%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shivangi Jaiswal</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jain%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Smita Jain</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Smita Jain</span></h3>
<div class="p">
<sup>b</sup>Department of Pharmacy, School of Chemical Sciences and Pharmacy, Central University of Rajasthan, Kishangarh, Rajasthan, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jain%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Smita Jain</span></a>
</div>
</div>
<sup>b</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mukhija%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Achal Mukhija</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Achal Mukhija</span></h3>
<div class="p">
<sup>a</sup>Department of Chemistry, Banasthali Vidyapith, Banasthali, Rajasthan, India, Email: jayadwivedi@yahoo.co.in</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mukhija%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Achal Mukhija</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Verma%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Kanika Verma</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kanika Verma</span></h3>
<div class="p">
<sup>c</sup>Department of Internal Medicine, Division of Cardiology, LSU Health Sciences Center–Shreveport, Louisiana, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Verma%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kanika Verma</span></a>
</div>
</div>
<sup>c</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jain%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Sonika Jain</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Sonika Jain</span></h3>
<div class="p">
<sup>a</sup>Department of Chemistry, Banasthali Vidyapith, Banasthali, Rajasthan, India, Email: jayadwivedi@yahoo.co.in</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jain%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sonika Jain</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kishore%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Dharma Kishore</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Dharma Kishore</span></h3>
<div class="p">
<sup>a</sup>Department of Chemistry, Banasthali Vidyapith, Banasthali, Rajasthan, India, Email: jayadwivedi@yahoo.co.in</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kishore%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dharma Kishore</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dwivedi%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jaya Dwivedi</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jaya Dwivedi</span></h3>
<div class="p">
<sup>a</sup>Department of Chemistry, Banasthali Vidyapith, Banasthali, Rajasthan, India, Email: jayadwivedi@yahoo.co.in</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dwivedi%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jaya Dwivedi</span></a>
</div>
</div>
<sup>a,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Swapnil Sharma</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Swapnil Sharma</span></h3>
<div class="p">
<sup>d</sup>Department of Pharmacy, Banasthali Vidyapith, Banasthali, Rajasthan, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Swapnil Sharma</span></a>
</div>
</div>
<sup>d</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="affa">
<sup>a</sup>Department of Chemistry, Banasthali Vidyapith, Banasthali, Rajasthan, India, Email: jayadwivedi@yahoo.co.in</div>
<div id="affb">
<sup>b</sup>Department of Pharmacy, School of Chemical Sciences and Pharmacy, Central University of Rajasthan, Kishangarh, Rajasthan, India</div>
<div id="affc">
<sup>c</sup>Department of Internal Medicine, Division of Cardiology, LSU Health Sciences Center–Shreveport, Louisiana, USA</div>
<div id="affd">
<sup>d</sup>Department of Pharmacy, Banasthali Vidyapith, Banasthali, Rajasthan, India</div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 18; Accepted 2025 Jul 30; Collection date 2025 Aug 18.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>This journal is © The Royal Society of Chemistry</div>
<p>This article is licensed under a <a href="https://creativecommons.org/licenses/by-nc/3.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution-Non Commercial 3.0 Unported Licence</a>. You can use material from this article in other publications without requesting further permissions from the RSC, provided that the correct acknowledgement is given and it is not used for commercial purposes.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12376879</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The sodium hydrogen orthophosphate (Na<sub>2</sub>HPO<sub>4</sub>) base was utilized in a stereospecific C–N coupling reaction to synthesize a novel series of nevirapine analogues in two-step reactions. This base is moisture tolerant, commercially available and makes the protocol cheap and energy efficient, with broad substrate tolerance, leading to the formation of cyclopropyl, cyclobutyl, cyclopentyl and propane-engrafted dipyridodiazepinone derivatives in good yield with a higher atom economy &gt;70%. All the synthesized analogues were examined for reverse transcriptase inhibitory activity and compared with the reference drug nevirapine. Further <em>in silico</em> analysis <em>via</em> molecular docking, molecular simulation, and ADMET studies revealed that compounds 5a and 5b showed prominent inhibitory activity against reverse transcriptase. Additionally, isothermal titration calorimetry experiments were performed to determine the thermodynamic parameters of the interaction between nevirapine analogues and human serum albumin. The binding affinity of 5b in the order of 10<sup>2</sup> indicates that the synthesized analogues can be easily carried out into the bloodstream. These findings demonstrate that nevirapine analogous are promising reverse transcriptase inhibitors for the therapeutic treatment of HIV infection, offering a new avenue for the less toxic and more effective development of anti-retroviral drugs.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>The sodium hydrogen orthophosphate base-mediated C–N coupling reaction was described for the synthesis of a novel series of dipyridodiazepinone derivatives as reverse transcriptase inhibitors.<a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-ga.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/3de3a1092eae/d5ra03504j-ga.jpg" loading="lazy" id="ga" height="370" width="756" alt="graphic file with name d5ra03504j-ga.jpg"></a></p></section><section id="sec1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Seven-membered nitrogen-containing diazepine framework is a core heterocyclic skeleton in various bioactive agents, including compounds with antidepressant, antibacterial, antiviral, antifungal and antitumor activities.<sup><a href="#cit1" class="usa-link" aria-describedby="cit1">1–4</a></sup> Many reports have revealed that benzodiazepine and pyridodiazepine in zapizolam, clozapine and olanzapine possess good anti-anxiety, anticonvulsant, sleep-inducing and muscle relaxant properties, while the dipyridodiazepinone skeleton in nevirapine plays a pivotal role in inhibiting the reverse transcriptase enzyme (<a href="#fig1" class="usa-link">Fig. 1</a>).<sup><a href="#cit5" class="usa-link" aria-describedby="cit5">5,6</a></sup> This enzyme is responsible for converting RNA into DNA and promoting the replication of retroviruses in host cells.<sup><a href="#cit7" class="usa-link" aria-describedby="cit7">7</a></sup> Most importantly, diazepine motifs have received increasing interest in clinical practice and drug discovery. Hence, new strategies for constructing dipyridodiazepinone scaffolds have received considerable attention, and this skeleton is suitable for the design and development of more effective compounds for the treatment of HIV infections and related issues.</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">Fig. 1. Representative bioactive diazepine derivatives.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/add4fe545026/d5ra03504j-f1.jpg" loading="lazy" height="212" width="756" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>In recent years, the C–N coupling reaction has been very powerful for the construction of complex molecules from simple synthons.<sup><a href="#cit8" class="usa-link" aria-describedby="cit8">8–10</a></sup> Among coupling reactions, the Buchwald–Hartwig cross coupling reaction has attracted great attention for intermolecular and intramolecular coupling, resulting in the formation of a cyclic ring, <em>i.e</em>. a seven-membered diazepine ring. Previously, this reaction was assisted by caesium carbonate and a metal catalyst, mainly palladium catalysts, with additional ancillary ligands. Currently, metal-free and highly atom-economical methods are being explored worldwide for the synthesis of nitrogen heterocycles <em>via</em> coupling reactions.</p>
<p>The principal objective of the present study is the cost-effective synthesis of novel dipyridodiazepinones, which are analogues of nevirapine. The present study reports the design of dipyridodiazepinone analogues using pharmacophore modelling and synthesizes them <em>via</em> base-promoted C–N coupling and cyclocondensation reactions (<a href="#fig2" class="usa-link">Fig. 2</a>).</p>
<figure class="fig xbox font-sm" id="fig2"><h3 class="obj_head">Fig. 2. Preparation of designed compounds in a two-step synthetic process <em>via</em> C–N coupling.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/4cd6e482a240/d5ra03504j-f2.jpg" loading="lazy" height="391" width="738" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The synthesized analogues were examined for an inhibitory effect against the HIV-1 RT enzyme. Furthermore, a spectrum of computational studies including MD simulation, ADMET/pharmacokinetics analysis, physicochemical analysis, <em>in silico</em> drug likeness, and molecular docking were performed. Additionally, the thermodynamic parameter of the binding interaction of synthesized compounds with human serum albumin (HSA) was investigated using isothermal titration calorimetry experiments. It is imperative to achieve the thermodynamic and binding parameters of the synthesized analogues to the macromolecules. Understanding thermodynamics and binding parameters is crucial for physicochemical mechanisms, such as the pharmacokinetics and pharmacodynamics of the synthesized dipyridodiazepinone.<sup><a href="#cit11" class="usa-link" aria-describedby="cit11">11</a></sup> Serum albumins are the most abundant protein in blood plasma responsible for the transportation of drugs in the circulatory system. The physiological role of the serum protein is to transport numerous ligands, such as amino acids, fatty acids, steroids, drugs, and metal ions, in the bloodstream to the targeted organs. In the circulatory system, serum albumin plays a major role in binding synthesized compounds or drugs and further transporting them to the target site. Thus, the binding of synthesized compounds to serum albumin is a pharmacokinetic criterion. The availability of the free drug is inversely proportional to the binding constant of serum albumin to the ligand. It means that when the binding is stronger, the availability of free drug is lower in the blood. Similarly, if the binding is weaker, the amount of free drug in the circulation is greater.<sup><a href="#cit12" class="usa-link" aria-describedby="cit12">12</a></sup> To understand the pharmacokinetic and ADMET mechanisms, knowledge of the binding interaction between the drug and protein is important. From this perspective, particular attention has been given to HSA, the most important plasma protein and the main carrier for transporting metabolites and drugs into the bloodstream.</p></section><section id="sec2"><h2 class="pmc_sec_title">2. Results and discussion</h2>
<section id="sec3"><h3 class="pmc_sec_title">2.1. Feature mapping analysis using a structure-based pharmacophore</h3>
<p>The computer-aided drug design process, such as structure-based pharmacophore modeling, is useful for identifying possible lead compounds that inhibit the HIV reverse transcriptase enzyme. Discovery Studio 2.0 is used to develop a six-feature pharmacophore model consisting of two hydrogen bond acceptors (HBA), two hydrophobic features (HY), and two hydrogen bond donors (HBD), as shown in <a href="#fig3" class="usa-link">Fig. 3</a>. Feature mapping of the synthesized compounds 5(a–h) on the developed HIV RT enzyme's structure-based pharmacophore and observed that 5a and 5b showed four feature mappings with 2.42008 and 2.59238 fit values. The PharmPrint of the synthesized analogues is shown in SI Table S1.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">Fig. 3. Feature mapping analysis of 5(a–h) using structure-based pharmacophore modelling (green colour: HBA, cyan colour: HY, and pink colour: HBD).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/77020eb927dd/d5ra03504j-f3.jpg" loading="lazy" height="322" width="738" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec4"><h3 class="pmc_sec_title">2.2. Chemistry</h3>
<p>The exploration of the concept of the C–N coupling reaction in the synthesis of new nevirapine analogues is formulated in <a href="#sch1" class="usa-link">Scheme 1</a>. The synthesis of carboxamide 3(a–b) was achieved by the intermolecular C–N coupling reaction of 1(a–b) with 2-chloronicotinyol chloride 2 in the presence of base sodium hydrogen orthophosphate (Na<sub>2</sub>HPO<sub>4</sub>). The formation of compounds 3(a–b) was established by their <sup>1</sup>H NMR spectra, which showed a sharp singlet at <em>δ</em> 8.8 ppm for one proton attached to nitrogen.</p>
<figure class="fig xbox font-sm" id="sch1"><h4 class="obj_head">Scheme 1. Synthesis of carboxamide 3(a–b).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-s1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/44c1b2991748/d5ra03504j-s1.jpg" loading="lazy" height="377" width="774" alt="Scheme 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/sch1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Compound 3(a–b) underwent an intermolecular coupling on its treatment with cyclopropylamine, cyclobutylamine, cyclopentylamine and propylamine in Na<sub>2</sub>HPO<sub>4</sub>; then, intramolecular coupling resulted in cyclization to form 1,4-dipyridiodiazepinone 5(a–h) (<a href="#sch2" class="usa-link">Scheme 2</a>). The <sup>1</sup>H-NMR, <sup>13</sup>C-NMR, and mass spectral analysis were used to confirm the structures and purity of the synthesized nevirapine analogues. The result of the HSQC NMR experiment confirms the assignment of the NH signal at 8.87 ppm, as it shows no direct correlation to any carbon signal, consistent with the presence of a non-carbon-bound proton. This observation validates the presence of an NH group in the structure of compound 3a. Compared to starting material 2-chloro-3-aminopyridine 1(a–b), the most relevant variation in the NMR spectra arises owing to the amide and amine coupling. The spectra of nevirapine analogues were in accordance with the structure of synthesized analogues shown in SI S1–S42.</p>
<figure class="fig xbox font-sm" id="sch2"><h4 class="obj_head">Scheme 2. Substrate scope for the synthesis of novel dipyridodiazepinone derivatives.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-s2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/40a1a88401da/d5ra03504j-s2.jpg" loading="lazy" height="515" width="738" alt="Scheme 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/sch2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Initially, 3-amino-2-chloro-5-methyl pyridine 1a and 2-chloronicotinyol chloride 2 were selected as the model substrates to optimize the reaction conditions (<a href="#tab1" class="usa-link">Table 1</a>). This reaction was carried out in potassium carbonate K<sub>2</sub>CO<sub>3</sub> and 1,4-dioxane at 60 °C for 6 h, and only a 20% yield was obtained. The change in solvent from 1,4-dioxane to THF results in an increase in yield from 20% to 40% (entry 2). Next, the caesium carbonate base is tested with solvent 1,4-dioxane; again, a decrease in the yield of the product was observed (entry 3). The same base tested with solvent THF showed an increase in yield (entry 4). We hypothesized that a change in the solvent system would affect the yield of the reaction owing to the solubility of the starting reactant and ion pairing. Further, to increase the yield from 50%, the solvent effect was further explored using a mixture of THF–H<sub>2</sub>O (entry 5). Encouraged by the result, Na<sub>2</sub>HPO<sub>4</sub> base utilized in THF–H<sub>2</sub>O (5 : 2) solvent showed the best conversion with a maximum yield of 91% (entry 6). In further exploration, an increase in the ratio of water provided 80% and 75% yields of 3a (entry 7–8). Finally, a reaction was performed on triethylamine, and a trace yield of the product was achieved (entry 9). The amide bond difference in products 3a and 3a′ was confirmed by FTIR spectroscopy, as evidenced by distinct variations in the carbonyl stretching frequencies of 1650 cm<sup>−1</sup> and 1725.04 cm<sup>−1</sup>, respectively, as shown in SI S2 and S3. The spot of the TLC plate indicates a favorable C–N coupling reaction between the amine of compound 1a and the carbonyl chloride of compound 2 in base Na<sub>2</sub>HPO<sub>4</sub> to form 3a in good yield.</p>
<section class="tw xbox font-sm" id="tab1"><h4 class="obj_head">Table 1. Optimization of reaction conditions<a href="#tab1fna" class="usa-link"><sup>a</sup></a>.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead>
<tr><th colspan="6" rowspan="1">
<img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/a5ef6b2a159d/d5ra03504j-u1.jpg" loading="lazy" id="ugr1" alt="graphic file with name d5ra03504j-u1.jpg">
</th></tr>
<tr>
<th rowspan="1" colspan="1">S. no.</th>
<th rowspan="1" colspan="1">Base</th>
<th rowspan="1" colspan="1">Solvent</th>
<th rowspan="1" colspan="1">Temp./time</th>
<th rowspan="1" colspan="1">Yield (3a)</th>
<th rowspan="1" colspan="1">Yield (3a′)</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">1</td>
<td rowspan="1" colspan="1">K<sub>2</sub>CO<sub>3</sub>
</td>
<td rowspan="1" colspan="1">1,4-Dioxane</td>
<td rowspan="1" colspan="1">60 °C, 6 h</td>
<td rowspan="1" colspan="1">20%</td>
<td rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td rowspan="1" colspan="1">2</td>
<td rowspan="1" colspan="1">K<sub>2</sub>CO<sub>3</sub>
</td>
<td rowspan="1" colspan="1">THF</td>
<td rowspan="1" colspan="1">60 °C, 6 h</td>
<td rowspan="1" colspan="1">40%</td>
<td rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td rowspan="1" colspan="1">3</td>
<td rowspan="1" colspan="1">Cs<sub>2</sub>CO<sub>3</sub>
</td>
<td rowspan="1" colspan="1">1,4-Dioxane</td>
<td rowspan="1" colspan="1">60 °C, 6 h</td>
<td rowspan="1" colspan="1">24%</td>
<td rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td rowspan="1" colspan="1">4</td>
<td rowspan="1" colspan="1">Cs<sub>2</sub>CO<sub>3</sub>
</td>
<td rowspan="1" colspan="1">THF</td>
<td rowspan="1" colspan="1">60 °C, 6 h</td>
<td rowspan="1" colspan="1">50%</td>
<td rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td rowspan="1" colspan="1">5</td>
<td rowspan="1" colspan="1">Cs<sub>2</sub>CO<sub>3</sub>
</td>
<td rowspan="1" colspan="1">THF–H<sub>2</sub>O (5 : 2)</td>
<td rowspan="1" colspan="1">60 °C, 6 h</td>
<td rowspan="1" colspan="1">60%</td>
<td rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td rowspan="1" colspan="1">6</td>
<td rowspan="1" colspan="1">Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td rowspan="1" colspan="1">THF</td>
<td rowspan="1" colspan="1">60 °C, 6 h</td>
<td rowspan="1" colspan="1">82%</td>
<td rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td rowspan="1" colspan="1">7</td>
<td rowspan="1" colspan="1">Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td rowspan="1" colspan="1">THF–H<sub>2</sub>O (5 : 2)</td>
<td rowspan="1" colspan="1">60 °C, 6 h</td>
<td rowspan="1" colspan="1">91%</td>
<td rowspan="1" colspan="1">1%</td>
</tr>
<tr>
<td rowspan="1" colspan="1">8</td>
<td rowspan="1" colspan="1">Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td rowspan="1" colspan="1">THF–H<sub>2</sub>O (4 : 3)</td>
<td rowspan="1" colspan="1">60 °C, 6 h</td>
<td rowspan="1" colspan="1">80%</td>
<td rowspan="1" colspan="1">2%</td>
</tr>
<tr>
<td rowspan="1" colspan="1">9</td>
<td rowspan="1" colspan="1">Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td rowspan="1" colspan="1">THF–H<sub>2</sub>O (3 : 4)</td>
<td rowspan="1" colspan="1">60 °C, 6 h</td>
<td rowspan="1" colspan="1">75%</td>
<td rowspan="1" colspan="1">Trace</td>
</tr>
<tr>
<td rowspan="1" colspan="1">10</td>
<td rowspan="1" colspan="1">TEA</td>
<td rowspan="1" colspan="1">THF</td>
<td rowspan="1" colspan="1">60 °C, 6 h</td>
<td rowspan="1" colspan="1">Trace</td>
<td rowspan="1" colspan="1">Trace</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tab1fna">
<sup>a</sup><p class="display-inline">Reaction conditions: 1a (0.0187 mmol), 2 (0.0187 mmol), Na<sub>2</sub>HPO<sub>4</sub> (0.02 mmol), 6 h, and temperature 60 °C in sealed tube.</p>
</div></div></section><p>Under the best optimized conditions (<a href="#tab1" class="usa-link">Table 1</a>, entry 6), the substrate scope of various cyclic and acyclic amines, such as cyclopropyl amine, cyclobutyl amine, cyclopentyl amine and propyl amine, was investigated in C–N coupling reaction and cyclization (<a href="#sch2" class="usa-link">Scheme 2</a>). It is noteworthy that the size of the ring influences the yield of the product. The smaller cyclopropylamine has a higher yield than cyclopentyl amine due to exhibiting steric hindrance in C–N cross coupling reactions. A good to high yield was obtained, where R is substituted with a methyl or hydrogen group. Noteworthy, 5a and 5b were obtained in higher yields, followed by substituents with cyclobutyl amine (5c–5d), cyclopentyl amine (5e–5f) and propyl amine (5g–5h). The larger substituents might cause steric hindrances, which further prevent intermolecular cyclocondensation from forming the diazepine ring.</p>
<p>Further series of control experiments were conducted to gain insight into this reaction, as outlined in <a href="#sch1" class="usa-link">Scheme 1</a>. The initial radical inhibition experiment was performed using TEMPO. The result showed that a product yield 77% was obtained and the reaction was not inhibited, which clearly stated that the radical mechanism was not involved (<a href="#sch3" class="usa-link">Scheme 3a</a>). According to previous reports, we hypothesized that the reaction involves amidation, amination and cyclocondensation reaction.<sup><a href="#cit13" class="usa-link" aria-describedby="cit13">13</a></sup> For further investigation, we synthesized intermediates 3A and 3B at 60 °C, giving 85% and 91% yields in <a href="#sch3" class="usa-link">Scheme 3b and c</a>, respectively. Both intermediates under standard conditions produced 5a in 50% and 88%, respectively. These outcomes suggest that the formed intermediate might be responsible for accelerating the reaction. The mechanism is proposed based on the control experiment results, as outlined in SI Scheme S1. Further sustainability of the reaction pathway was examined by evaluating the atom economy. The synthesized compounds have an atom economy greater than 70%, indicating that the process is sustainable with minimum waste generation. The atom economy of the synthesized compounds is in the following order: 5a (71%), 5b (70%), 5c (72%), 5d (71%), 5e (73%), 5f (72%), 5g (71%), and 5h (70%).</p>
<figure class="fig xbox font-sm" id="sch3"><h4 class="obj_head">Scheme 3. (a–c) Control experiments.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-s3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/cd1634409d9d/d5ra03504j-s3.jpg" loading="lazy" height="464" width="792" alt="Scheme 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/sch3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec5"><h3 class="pmc_sec_title">2.3. HIV-1 reverse transcriptase inhibitory activity</h3>
<p>The synthesized dipyridodiazepinone analogue 5(a–h) was screened <em>in vitro</em> for reverse transcriptase inhibitory effect using the HIV-1 RT ELISA kit. Among all, analogue 5b (IC<sub>50</sub> = 0.86 nM) was the most potent, followed by 5a (IC<sub>50</sub> = 1.07 nM), when compared with standard nevirapine (IC<sub>50</sub> = 3.05 nM). The results of the HIV-1 RT assay for all synthesized analogues are shown in <a href="#fig4" class="usa-link">Fig. 4</a>. The activity data indicate that cyclopropyl substituents in 5a and 5b showed prominent inhibitory activity compared to cyclobutyl, cyclopentyl and alkyl substituted dipyridodiazepinone derivatives.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">Fig. 4. Inhibitory effect of synthesized analogues on the HIV-RT enzyme.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/b45d88a6f5e9/d5ra03504j-f4.jpg" loading="lazy" height="458" width="771" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec6"><h3 class="pmc_sec_title">2.4. Molecular docking</h3>
<p>A molecular docking study is a crucial tool for evaluating the interaction between the target protein and drug. In this current <em>in silico</em> study, dipyridodiazepinone analogues (5a, 5b, 5c, 5d, 5e, 5f, 5g and 5h) were docked against HIV-1 reverse transcriptase using the Auto-Dock Vina module. The docking scores of all synthesized compounds are shown in <a href="#tab2" class="usa-link">Table 2</a>. The binding site of the HIV-1 reverse transcriptase is surrounded by the 10 amino acid residues <em>viz.</em> Lys101, Lys103, Val179, Val106, Leu100, Tyr318, Tyr188, Tyr181, Pro95 and Trp229. We found that among all synthesized compounds, nevirapine analogues 5a and 5b had higher binding energies of −9.5 kcal mol<sup>−1</sup> and −9.8 kcal mol<sup>−1</sup> when compared to nevirapine −9.2 kcal mol<sup>−1</sup>, as shown in <a href="#tab2" class="usa-link">Table 2</a>. This prompted the initiative to visualize their binding pose and interaction patterns with the target proteins relative to the reference drug (nevirapine) using PyMol and Discovery Studio Visualizer. The analogues were observed to fit into the same cavity of the binding sites. Interestingly, in 5a, the amino group of the diazepin-6-one ring was involved in the one hydrogen bond formation within the binding cavities of macromolecules, such as Lys101 at a bond length of 2.10 Å. A pi–sigma bond with 3.82 Å was observed between the diazepin-6-one ring and Leu100 amino acid residues, while 5a compound was also involved in the formation of alkyl and pi–alkyl bond interactions with the Val106 (3.89, 4.68 Å), Leu234 (4.65 Å), Tyr188 (4.66 Å), Tyr181 (5.27 Å), Val179 (4.23 Å), and Lys103 (5.20 Å) amino acid residues of the macromolecules. In the 5b compound, the diazepin-6-one ring was involved in hydrogen bond formation with Lys101 amino acid residues at a 2.05 Å bond length. Similarly, the diazepin-6-one ring was involved in the formation of a pi–sigma bond with the Leu100 (4.54 Å) amino acid residue of the macromolecule, while the 5b compound was also involved in the formation of alkyl and pi–alkyl bond interactions with the Lys103 (5.29 Å), Tyr181 (4.83 Å), Val106 (3.86, 4.90 Å), Leu234 (4.51 Å), and Tyr188 (4.83 Å). The reference drug, nevirapine, used in this study showed a comparable binding energy of −9.2 kcal mol<sup>−1</sup>. The amide group of the diazepinone ring was able to form one hydrogen bond with Lys 101 at a distance of 2.54 Å. Meanwhile, Leu 103 established a carbon–hydrogen bond at a distance of 4.60 Å. Tyr188 (4.51 Å), Val 106 (5.49, 4.36 A), Tyr 318 (4.59 A), Tyr 188 (4.51 A), Tyr 181 (5.24 A), Val 179 (5.04 A), and Leu 100 (4.60, 3.93, 3.74 A) were associated with the reference drug <em>via</em> alkyl and pi–alkyl bonds, as shown in <a href="#fig5" class="usa-link">Fig. 5</a>. The binding interactions of 5c, 5d, 5e, 5f, 5g and 5h are shown in SI (Fig. S1–S6). The results from molecular docking suggest that amino acids, such as leucine, valine, and tyrosine, are vital for binding ligands to the HIV-1 RT active site. In conclusion, the docking results were in line with the <em>in vitro</em> assay and indicated that the synthesized analogues might be potential inhibitors of HIV-1 RT.</p>
<section class="tw xbox font-sm" id="tab2"><h4 class="obj_head">Table 2. Binding energies of synthesized analogues docked against the HIV-I reverse transcriptase enzyme<a href="#tab2fna" class="usa-link"><sup>a</sup></a>.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead><tr>
<th rowspan="1" colspan="1">Compounds</th>
<th rowspan="1" colspan="1">Binding energy (kcal mol<sup>−1</sup>)</th>
<th rowspan="1" colspan="1">Amino acid residues with distance (Å)</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">5a</td>
<td align="center" rowspan="1" colspan="1">−9.5</td>
<td rowspan="1" colspan="1">H-Lys101 (2.10), C-Lys103 (5.20), C-Val106 (3.89, 4.68), C-Tyr188 (4.66), C-Tyr181 (5.27), C-Val179 (4.23), C-Leu100 (3.82)</td>
</tr>
<tr>
<td rowspan="1" colspan="1">5b</td>
<td align="center" rowspan="1" colspan="1">−9.8</td>
<td rowspan="1" colspan="1">H-Lys101 (2.18), C-Leu100 (3.57), C-Lys103 (5.29), C-Leu234 (4.58), C-Tyr181 (4.83), C-Tyr188 (4.83), C-Val106 (3.86, 4.90)</td>
</tr>
<tr>
<td rowspan="1" colspan="1">5c</td>
<td align="center" rowspan="1" colspan="1">−8.3</td>
<td rowspan="1" colspan="1">H-Lys101 (1.98), C-Leu234 (4.82), C-Tyr318 (5.47), C-Tyr188 (4.38), C-Tyr181 (5.40), C-Val106 (3.84, 4.54)</td>
</tr>
<tr>
<td rowspan="1" colspan="1">5d</td>
<td align="center" rowspan="1" colspan="1">−8.4</td>
<td rowspan="1" colspan="1">H-Lys101 (1.97), C-Leu234 (4.75), C-Tyr318 (5.45), C-Val106 (3.87, 4.56), C-Tyr188 (4.45), C-Tyr181 (5.38), C-Lys103 (5.22, 5.17), C-Lys101 (4.55), C-Val179 (4.61)</td>
</tr>
<tr>
<td rowspan="1" colspan="1">5e</td>
<td align="center" rowspan="1" colspan="1">−8.5</td>
<td rowspan="1" colspan="1">H-Lys101 (2.55), C-Tyr188 (3.75, 4.63), C-Val106 (4.22), C-Lys103 (5.13), C-Tyr318 (4.89), C-Trp229 (3.87, 3.63), C-Tyr318 (4.89)</td>
</tr>
<tr>
<td rowspan="1" colspan="1">5f</td>
<td align="center" rowspan="1" colspan="1">−8.9</td>
<td rowspan="1" colspan="1">H-Lys101 (2.13), O-Lys101 (3.21), C-Val106 (4.10), C-Lys103 (5.26), C-Tyr188 (4.34), C-Tyr181 (5.31), C-Trp229 (4.87, 4.89), C-Leu100 (4.87, 3.57, 3.81, 4.82)</td>
</tr>
<tr>
<td rowspan="1" colspan="1">5g</td>
<td align="center" rowspan="1" colspan="1">−8.7</td>
<td rowspan="1" colspan="1">H-Lys101 (2.05), C-Tyr188 (3.90), C-Tyr181 (4.91), C-Leu100 (4.28), C-Leu100 (3.74, 4.31), C-Lys101 (4.26), C-Val179 (4.28), C-leu234 (4.55), C-Val106 (3.89)</td>
</tr>
<tr>
<td rowspan="1" colspan="1">5h</td>
<td align="center" rowspan="1" colspan="1">−9.0</td>
<td rowspan="1" colspan="1">H-Lys101 (2.07), C-Tyr181 (4.87), C-Tyr188 (3.84), C-Leu100 (4.32), C-Leu100 (3.72, 4.24), C-Val106 (3.85, 4.81), C-Leu234 (4.59)</td>
</tr>
<tr>
<td rowspan="1" colspan="1">NVP</td>
<td align="center" rowspan="1" colspan="1">−9.2</td>
<td rowspan="1" colspan="1">H-Lys101 (2.18), C-Leu100 (3.57), C-Tyr188 (4.53), C-Leu234 (4.58), C-Phe227 (5.28), C-Lys103 (4.45), C-Tyr318 (4.40), C-Val106 (4.60)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tab2fna">
<sup>a</sup><p class="display-inline">NVP: nevirapine (standard drug).</p>
</div></div></section><figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">Fig. 5. Binding interactions of 5a, 5b and nevirapine with the active binding pockets of HIV-I reverse transcriptase enzyme.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-f5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/344cf07b49f5/d5ra03504j-f5.jpg" loading="lazy" height="532" width="778" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec7"><h3 class="pmc_sec_title">2.5. ADMET</h3>
<p>The findings of the drug-likeness and physicochemical studies provide supporting evidence for the drug-like properties of the synthesized analogues (<a href="#tab3" class="usa-link">Table 3</a>). According to the results, nevirapine and its analogues 5a, 5b had log <em>P</em> (<em>C</em> log <em>P</em>) &lt; 5, molecular weight &lt; 500, fewer than 10<em>H</em>-bond acceptors, and 5<em>H</em>-bond donors. These findings indicate that the synthesized analogues follows Lipinski's rule of 5 and exhibit drug-like properties.</p>
<section class="tw xbox font-sm" id="tab3"><h4 class="obj_head">Table 3. Drug-like, physicochemical, and ADMET properties.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead><tr>
<th rowspan="1" colspan="1">Class</th>
<th rowspan="1" colspan="1">Properties</th>
<th rowspan="1" colspan="1">5a</th>
<th rowspan="1" colspan="1">5b</th>
<th rowspan="1" colspan="1">NVP</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="11" colspan="1">Physicochemical properties</td>
<td rowspan="1" colspan="1">Molecular weight (g mol<sup>−1</sup>)</td>
<td rowspan="1" colspan="1">266</td>
<td rowspan="1" colspan="1">252</td>
<td rowspan="1" colspan="1">266</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Lipophilicity (<em>c</em> log <em>P</em>)</td>
<td rowspan="1" colspan="1">1.96</td>
<td rowspan="1" colspan="1">1.62</td>
<td rowspan="1" colspan="1">1.96</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Solubility</td>
<td rowspan="1" colspan="1">−4.67</td>
<td rowspan="1" colspan="1">−4.32</td>
<td rowspan="1" colspan="1">−4.67</td>
</tr>
<tr>
<td rowspan="1" colspan="1">TPSA (Å)</td>
<td rowspan="1" colspan="1">58.12</td>
<td rowspan="1" colspan="1">58.12</td>
<td rowspan="1" colspan="1">58.12</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Drug likeness</td>
<td rowspan="1" colspan="1">3.02</td>
<td rowspan="1" colspan="1">3.09</td>
<td rowspan="1" colspan="1">3.02</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Drug score</td>
<td rowspan="1" colspan="1">0.73</td>
<td rowspan="1" colspan="1">0.78</td>
<td rowspan="1" colspan="1">0.73</td>
</tr>
<tr>
<td rowspan="1" colspan="1">H bond acceptor</td>
<td rowspan="1" colspan="1">4</td>
<td rowspan="1" colspan="1">4</td>
<td rowspan="1" colspan="1">4</td>
</tr>
<tr>
<td rowspan="1" colspan="1">H bond donor</td>
<td rowspan="1" colspan="1">1</td>
<td rowspan="1" colspan="1">1</td>
<td rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Rotatable bond</td>
<td rowspan="1" colspan="1">1</td>
<td rowspan="1" colspan="1">1</td>
<td rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Violation</td>
<td rowspan="1" colspan="1">0</td>
<td rowspan="1" colspan="1">0</td>
<td rowspan="1" colspan="1">0</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Bioavailability score</td>
<td rowspan="1" colspan="1">0.55</td>
<td rowspan="1" colspan="1">0.55</td>
<td rowspan="1" colspan="1">0.55</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Absorption</td>
<td rowspan="1" colspan="1">GI absorption</td>
<td rowspan="1" colspan="1">High</td>
<td rowspan="1" colspan="1">High</td>
<td rowspan="1" colspan="1">High</td>
</tr>
<tr>
<td rowspan="7" colspan="1">Metabolism</td>
<td rowspan="1" colspan="1">CYP3A4 substrate</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✓</td>
</tr>
<tr>
<td rowspan="1" colspan="1">CYP2D6 substrate</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
</tr>
<tr>
<td rowspan="1" colspan="1">CYP1A2 inhibitor</td>
<td rowspan="1" colspan="1">✓</td>
<td rowspan="1" colspan="1">✓</td>
<td rowspan="1" colspan="1">✓</td>
</tr>
<tr>
<td rowspan="1" colspan="1">CYP2C9 inhibitor</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
</tr>
<tr>
<td rowspan="1" colspan="1">CYP2C19 inhibitor</td>
<td rowspan="1" colspan="1">✓</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
</tr>
<tr>
<td rowspan="1" colspan="1">CYP3A4 inhibitor</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✓</td>
<td rowspan="1" colspan="1">✗</td>
</tr>
<tr>
<td rowspan="1" colspan="1">CYP2D6 inhibitor</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
</tr>
<tr>
<td rowspan="2" colspan="1">Distribution</td>
<td rowspan="1" colspan="1">BBB</td>
<td rowspan="1" colspan="1">High</td>
<td rowspan="1" colspan="1">High</td>
<td rowspan="1" colspan="1">High</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Plasma protein binding (PPB)</td>
<td rowspan="1" colspan="1">✓</td>
<td rowspan="1" colspan="1">✓</td>
<td rowspan="1" colspan="1">✓</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Excretion</td>
<td rowspan="1" colspan="1">Total clearance (log ml min<sup>−1</sup> kg<sup>−1</sup>)</td>
<td rowspan="1" colspan="1">0.005</td>
<td rowspan="1" colspan="1">0.02</td>
<td rowspan="1" colspan="1">0.02</td>
</tr>
<tr>
<td rowspan="5" colspan="1">Toxicity</td>
<td rowspan="1" colspan="1">AMES mutagenicity</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Oral rat acute toxicity (LD<sub>50</sub>) (mol kg<sup>−1</sup>)</td>
<td rowspan="1" colspan="1">2.494</td>
<td rowspan="1" colspan="1">2.529</td>
<td rowspan="1" colspan="1">2.644</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Hepatotoxicity</td>
<td rowspan="1" colspan="1">✓</td>
<td rowspan="1" colspan="1">✓</td>
<td rowspan="1" colspan="1">✓</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Skin sensitisation</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Carcinogenicity</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
<td rowspan="1" colspan="1">✗</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>ADMET analysis was conducted to define the pharmacological profile of nevirapine and its synthesized analogues, which paves the way for predicting it as a drug candidate. Drug candidates demonstrated inadequate absorption when their topological polar surface area (TPSA) exceeds the threshold of 140 Å<sup>2</sup>, which serves as a baseline for approved pharmaceuticals. There exists a positive association between TPSA and mass, wherein molecules with a mass above 500 g mol<sup>−1</sup> were reported to exhibit TPSA values outside the range of 0–140. The TPSA values and Gastrointestinal Absorption (GIA) of the retrieved analogues were deemed satisfactory.<sup><a href="#cit14" class="usa-link" aria-describedby="cit14">14</a></sup> A bioavailability score larger than zero, <em>i.e.</em> 0.55, suggests that the analogues exhibit at least 10% oral bioavailability or measurable Caco-2 permeability, contributing to their drug-like property. The permeability characteristics of a medication, namely its GIA and Blood–Brain Barrier (BBB) permeability, have significant importance in the context of developing a treatment intended for extensive use.</p>
<p>Many previous studies have been conducted to enhance the gastrointestinal penetration of molecules for the oral delivery of poorly absorbed medications.<sup><a href="#cit15" class="usa-link" aria-describedby="cit15">15</a></sup> All three compounds demonstrated satisfactory growth inhibitory activity and were able to penetrate the blood–brain barrier. Cytochrome P450s (CYPs) are an imperative group of enzymes, including heme, and play a substantial role in metabolising pharmaceutical compounds and eliminating foreign substances. CYP activity might be affected by pharmaceutical agents, resulting in noteworthy drug–drug interactions with potential clinical implications. These interactions may give rise to therapeutic inefficacy or unexpected side effects. It is noteworthy that 5a, 5b, and nevirapine do not inhibit enzymes CYP2C9, CYP2D6, and CYP3A4, indicating that they have increased chances of being eliminated and metabolized through the metabolic biotransformation process at a substantial rate. The acute toxicity prediction assays, including the AMES test, acute oral toxicity, skin sensitization, and hepatotoxicity, revealed 5a, 5b, and nevirapine to be non-toxic, non-carcinogenic, non-mutagenic, and non-irritant. Although 5a, 5b, and nevirapine were predicted to be hepatotoxic, these effects might be produced at a higher dose range. Notably, according to the Globally Harmonized System (300 &lt; category 4 ≤ 2000 mg per kg per day), LD<sub>50</sub> values found within the range of 1266–2175 mg per kg per day were considered in category 4, indicating that 5a and 5b are non-toxic.<sup><a href="#cit16" class="usa-link" aria-describedby="cit16">16</a></sup> Consequently, based on preliminary drug-likeness, physicochemical, and ADMET prediction studies, 5a and 5b may be promising candidates for further investigations.</p></section><section id="sec8"><h3 class="pmc_sec_title">2.6. Molecular simulation</h3>
<p>The synthesized analogues showed inspiring results <em>in silico</em> and drug-likeness studies; the analogues were screened for MD simulation studies.<sup><a href="#cit17" class="usa-link" aria-describedby="cit17">17,18</a></sup> MD simulations on the analogues were conducted using the Desmond package with significant predicted binding energies (&gt;−70.00 kcal mol<sup>−1</sup>). The binding energies were calculated between the HIV–I reverse transcriptase protein and synthesized analogues. The 5a and 5b exhibited −73.53 kcal mol<sup>−1</sup> and −86.44 kcal mol<sup>−1</sup> binding energies, respectively (<a href="#fig6" class="usa-link">Fig. 6</a> and <a href="#fig7" class="usa-link">7</a>). In the current investigation, the stability of novel synthesized nevirapine analogues (5a and 5b) and enzyme (1tkx) complexes were identified using HIV-I reverse transcriptase and 100 ns MD simulations.</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">Fig. 6. (a) RMSD analysis of 5a, (b) RMSF analysis of 5a, (c) binding interaction analysis of 5a with the HIV-I reverse transcriptase, and (d) 2D-contact information of 5a with the HIV-I reverse transcriptase enzyme.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-f6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/faed9079d102/d5ra03504j-f6.jpg" loading="lazy" height="509" width="787" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><figure class="fig xbox font-sm" id="fig7"><h4 class="obj_head">Fig. 7. (a) RMSD analysis of 5b, (b) RMSF analysis of 5b, (c) binding interaction analysis of 5b with the HIV-I reverse transcriptase, and (d) 2D-contact information of 5b with the HIV-I reverse transcriptase enzyme.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-f7.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/f0a7395db284/d5ra03504j-f7.jpg" loading="lazy" height="475" width="734" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The root mean square deviation (RMSD) calculates the average distance caused by atom displacement for a given time frame. RMSD analysis can be used to establish the occurrence of equilibrated simulation. Notably, fluctuations between the range of 1–3 Å are considered perfectly acceptable within a reference protein structure. A significant conformational change in the structure of a protein indicates an unstable system. During a 100 ns simulation experiment, analogues 5a and 5b exhibited an extended variation of 4.53 Å and 5.57 Å, respectively, as presented in SI (Table 2). According to the results, HIV-1 reverse transcriptase undergoes a protein conformational change as a result of binding with analogues. Analogue 5a showed minimal structural modifications within the range of 32–46 Å, suggesting a rather stable alignment throughout the binding region (SI (Fig. S7a)). Similarly, 5b was involved in the hydrogen bond formation with Lys101 and Tyr318 amino acid residues; hydrophobic interactions with Pro95, Pro97, Tyr181, Val179, Val106, Leu100, Tyr183, Tyr188, Phe227, Trp229, and Leu234 and water bridge interactions with Ile94, Pro95, Lys101, Lys103, Ile180, His235 and Pro236 amino acid residues of HIV-I RT enzyme. Similarly, 5b showed minimal structural modifications ranging from 28 to 38 Å, suggesting a rather stable alignment within the binding site (SI (Fig. S7c)). The molecular docking and molecular dynamics simulation results showed that compounds 5a and 5b exhibited hydrogen bond interaction (hydrophilic interaction) with the Lys101 amino acid residue of the HIV-1 reverse transcriptase enzyme. Similarly, both compounds 5a and 5b formed hydrophobic interactions with various Leu234, Val106, Lys103, Leu100, Tyr188, Tyr181 and Val179 amino acid residues of HIV-1 reverse transcriptase enzyme. These amino acid residues represent the allosteric binding sites of the HIV-1 RT enzyme.<sup><a href="#cit19" class="usa-link" aria-describedby="cit19">19</a></sup> The literature review demonstrates that an allosteric binding site of the HIV-1 RT enzyme, also named the NNRTIBP (non-nucleoside reverse transcriptase inhibitors binding pockets), is located within the primer grip region near the polymerase's active site, which lies primarily in the p66 region of the polymerase active site.<sup><a href="#cit20" class="usa-link" aria-describedby="cit20">20,21</a></sup> It comprises several aromatic (Tyr181, Tyr188, Phe227, Trp229 and Tyr232), hydrophilic (Lys101, Lys103, Ser105, Asp132, and Glu224), and hydrophobic residues (Pro59, Leu100, Val106, Val179, and Leu236) and two more amino acids (Ile135 and Glu138) from p51 subunits. Similarly, NNRTIs, including nevirapine, rilpivirine and efavirenz, showed binding interactions with the allosteric site of the enzyme and acted as non-competitive inhibitors.<sup><a href="#cit22" class="usa-link" aria-describedby="cit22">22,23</a></sup> These analyses hypothesized that compounds 5a and 5b showed similar binding interactions with the amino acid residues of the allosteric sites, so both compounds might act as non-competitive inhibitors of the HIV-1 reverse transcriptase enzyme (<a href="#fig8" class="usa-link">Fig. 8</a>).</p>
<figure class="fig xbox font-sm" id="fig8"><h4 class="obj_head">Fig. 8. Mechanism for the inhibition of reverse transcriptase <em>via</em> synthesized analogues.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-f8.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/bc556703f837/d5ra03504j-f8.jpg" loading="lazy" height="215" width="774" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec9"><h3 class="pmc_sec_title">2.7. Isothermal titration calorimetry (ITC)</h3>
<p>ITC studies on the interaction of nevirapine and its analogues (5a and 5b) with HSA protein were performed by injecting the ligand solution into the sample cell loaded with 60 μM HSA protein. The concentration of nevirapine and its analogues in the syringe was optimized (0.34 mM) to confirm the saturation of the HSA protein with the derivatives within 19 additional injections. <a href="#fig9" class="usa-link">Fig. 9</a> displays a representative ITC profile for the interaction of nevirapine and its analogues with HSA protein at 298 K. <a href="#fig9" class="usa-link">Fig. 9</a>, upper panel, presents raw data as power <em>versus</em> time upon the addition of nineteen sequential injections. However, the lower panel shows the heat profile as a function of analogues to the HSA protein molar ratio.</p>
<figure class="fig xbox font-sm" id="fig9"><h4 class="obj_head">Fig. 9. ITC profiles for nevirapine and its analogues (5a and 5b) with HSA at 298 K.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-f9.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/a5e76fa36844/d5ra03504j-f9.jpg" loading="lazy" height="562" width="738" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>A single set of binding site models fit the experimental results well, thereby providing the values of the stoichiometry of binding (<em>n</em>), Gibbs free energy change (Δ<em>G</em><sub>m</sub>), standard molar enthalpy (Δ<em>H</em><sub>m</sub>), and binding constant (<em>K</em><sub>b</sub>). <a href="#fig9" class="usa-link">Fig. 9</a> shows the obtained values of these thermodynamic parameters. The associated thermodynamic parameters of interactions of nevirapine with HSA are as <em>K</em><sub>b</sub> = 1.0 × 10<sup>5</sup> M<sup>−1</sup>, enthalpy change Δ<em>H</em><sub>m</sub> = −10.6 kJ mol<sup>−1</sup>, Δ<em>G</em><sub>m</sub> = −28.6 kJ mol<sup>−1</sup>, and <em>n</em> = (0.9). The obtained value of <em>K</em><sub>b</sub> with order 10<sup>5</sup> M<sup>−1</sup> suggests the presence of strong binding between nevirapine and the HSA protein. The binding was observed to be associated with exothermic heat effects, indicating the involvement of hydrophilic interactions in the binding process since hydrophilic interactions are accompanied by exothermic heat effects. In addition, the negative Δ<em>G</em><sub>m</sub> value directly relates to the binding affinity for spontaneous binding between nevirapine and the biomolecule HSA. The nevirapine stoichiometry value (0.9) implies the binding of one molecule of nevirapine to the HSA protein. These findings of ITC investigations imply that the binding interactions of nevirapine with HSA macromolecules are thermodynamically favourable.</p>
<p>Next, the observed quantitative characteristics of interactions of synthesized analogues 5a and HSA protein, including binding constant, enthalpy change, Gibbs free energy change, and stoichiometry, are 2.7 × 10<sup>3</sup> M<sup>−1</sup>, −8.1 kJ mol<sup>−1</sup>, −19.5 kJ mol<sup>−1</sup>, and 1.3, respectively.</p>
<p>The results suggested that the binding constant value was observed to be of the order of 10<sup>3</sup> M<sup>−1</sup>, indicating that the drug molecule with CH<sub>3</sub> substitution from γ to β position binds to HSA with lesser affinity. The change in exothermic enthalpy value indicates that the association process is largely driven by electrostatic interactions and the negative Δ<em>G</em><sub>m</sub> value directly relates to binding affinity for spontaneous binding. The stoichiometry value implies the binding of one molecule of compound 5a to the HSA protein. Furthermore, the thermodynamic characteristics of interactions of compound 5b with HSA protein in terms of a binding constant, enthalpy change, Gibbs free energy change, and stoichiometry are 1.1 × 10<sup>2</sup> M<sup>−1</sup>, −4.2 kJ mol<sup>−1</sup>, −11.6 kJ mol<sup>−1</sup>, and 2.7, respectively. The effect of the substitution of CH<sub>3</sub> for H was observed on the binding constant values of the interaction with the HSA protein. The binding was observed to be least for compound 5b, and with an exothermic enthalpy change, the binding was again observed to be spontaneous. All the analogues exhibited Δ<em>H</em> &lt; 0, which in turn revealed a favourable entropy contribution Δ<em>S</em> &gt; 0, as estimated in the calorimetric experiments. The positive and negative entropy signs confirm the ionic and electrostatic interactions between the target compound and HSA. According to the ITC results, nevirapine analogues 5b bind weakly to HSA. This finding demonstrates that HSA is a carrier for synthesized nevirapine analogues, which are easily released in the body. This study gives accurate and thorough information about synthesized analogues binding to HSA, which can be used as a reference in anti-HIV drug binding behaviour. These findings are in agreement with the results of the study reported by Pîrnău <em>et al.</em>, 2018.<sup><a href="#cit12" class="usa-link" aria-describedby="cit12">12</a></sup></p></section><section id="sec10"><h3 class="pmc_sec_title">2.8. Fluorescence quenching studies</h3>
<p>Fluorescence quenching studies were performed to understand the binding of dipyridodiazepinone derivatives with HSA and to complement the ITC results. Fluorescence quenching studies were also performed with nevirapine to compare the effect of the methyl group position change on the binding interaction with HSA. For this, HSA was taken in a cuvette, and aliquots of nevirapine and its analogues were added to this solution. The concentrations of nevirapine and its derivative solution were increased from 0 to 0.17 mM to achieve quenching of HSA fluorescence.</p>
<p>To evaluate the exact quenching mechanism, the Stern–Volmer equation was applied to determine the binding constant values. The findings of the modified Stern–Volmer equation are provided in <a href="#eqn1" class="usa-link">eqn (1)</a>:</p>
<table class="disp-formula p" id="eqn1"><tr>
<td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML">ln[(<em>F</em><sub>0</sub> − <em>F</em>)/<em>F</em>] = ln <em>K</em><sub>b</sub> + <em>n</em> ln[<em>Q</em>]</span></td>
<td class="label">1</td>
</tr></table>
<p>where <em>F</em><sub>0</sub> and <em>F</em> are the fluorescence intensity of HSA in the absence and presence of the quencher, and <em>K</em><sub>b</sub> is the binding constant. The quenching spectra are shown in the upper panel of <a href="#fig10" class="usa-link">Fig. 10</a>, and Stern–Volmer plots are shown in the lower panel. Quenching spectra show the strong quenching of HSA fluorescence in the presence of nevirapine. Compound 5a shows comparatively less quenching than nevirapine, while compound 5b shows the least quenching of HSA fluorescence. Binding constant values obtained by plotting ln[(<em>F</em><sub>0</sub> − <em>F</em>)/<em>F</em>] as a function of ln[<em>Q</em>] were 5.7 × 10<sup>5</sup> M<sup>−1</sup>, 2.5 × 10<sup>3</sup> M<sup>−1</sup>, and 1.1 × 10<sup>2</sup> M<sup>−1</sup> for nevirapine, compound 5a, and compound 5b, respectively. Fluorescence intensities <em>F</em> and <em>F</em><sub>0</sub> of HSA were observed at 338 nm (emission) and 280 nm (excitation), respectively. No significant shift was observed in wavelength with maximum absorbance after the addition of the quencher. Binding constant values suggest the maximum binding of nevirapine with HSA. However, changing the position of the CH<sub>3</sub> group from γ to β position in compound 5a decreased the binding constant value with HSA. Compound 5b, where the CH<sub>3</sub> group is being replaced by H, showed the least binding with HSA and complemented the ITC results. Fluorescence quenching analysis also confirmed that 5b possessed poor binding affinity with HSA and was released easily in the body, which significantly influenced the ADMET mechanism.</p>
<figure class="fig xbox font-sm" id="fig10"><h4 class="obj_head">Fig. 10. Fluorescence quenching titration spectra of HSA with increasing concentrations of nevirapine (a), compound 5a (b), and compound 5b (c). Lower panel shows the Stern–Volmer plots of the respective compounds.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376879_d5ra03504j-f10.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/0bbe3b31abe6/d5ra03504j-f10.jpg" loading="lazy" height="417" width="767" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><section id="sec11"><h4 class="pmc_sec_title">2.8.1. Evaluation of binding forces using thermodynamic parameters</h4>
<p>Thermodynamic parameters were analyzed to determine the type of forces involved in the macromolecule–ligand interaction. The thermodynamic parameters, including free energy change (Δ<em>G</em>), enthalpy change (Δ<em>H</em>), and entropy change (Δ<em>S</em>), are crucial in determining the acting force between synthesized compounds and macromolecules, mainly including hydrophobic forces, hydrogen bonds, van der Waals forces, and electrostatic interactions. The ITC and fluorescence readings were exploited to calculate the binding affinity, <em>K</em><sub>b</sub> and the number of binding sites for the synthesized compound-HSA complex. The following van't-Hoff and thermodynamic equations (<a href="#eqn2" class="usa-link">eqn (2)</a> and <a href="#eqn3" class="usa-link">(3)</a>, respectively) were used to analyse the thermodynamic parameters in the binding phenomenon between the synthesized compound and HSA:</p>
<table class="disp-formula p" id="eqn2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f71/12376879/6f1ead8f8580/d5ra03504j-t1.jpg" loading="lazy" id="ugt1" height="139" width="600" alt="2.8.1."></td>
<td class="label">2</td>
</tr></table>
<table class="disp-formula p" id="eqn3"><tr>
<td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML">Δ<em>G</em>° = Δ<em>H</em>° − <em>T</em>Δ<em>S</em>°</span></td>
<td class="label">3</td>
</tr></table>
<p>Here, <em>T</em> is the experimental temperature, <em>R</em> is the molar gas constant and <em>K</em> is the binding constant. The nature of the acting forces in the interaction process is as follows: (i) electrostatic/ionic interactions: Δ<em>H</em> &lt; 0 and Δ<em>S</em> &gt; 0; (ii) hydrophobic force: Δ<em>H</em> &gt; 0 and Δ<em>S</em> &gt; 0; and (iii) hydrogen and van der Waal's force: Δ<em>H</em> &lt; 0 and Δ<em>S</em> &lt; 0. It is evident from the ITC results that the calculated Δ<em>G</em> value is negative, which signifies that the binding of the synthesized compound with HSA is spontaneous. The entropies of the nevirapine, 5a and 5b were found to be +60.4, +38.26 and +25.5 J K<sup>−1</sup> mol<sup>−1</sup>, respectively. The negative Δ<em>H</em> and positive Δ<em>S</em> support the electrostatic/ionic interactions with HSA and are responsible for the intrinsic fluorescence property.</p></section></section></section><section id="sec12"><h2 class="pmc_sec_title">3. Experimental section</h2>
<section id="sec13"><h3 class="pmc_sec_title">3.1. Materials and instruments</h3>
<p>All solvents, reagents, and HSA were commercially purchased from Sigma-Aldrich and used without purification. The structures of all the synthesized analogues were confirmed by applying the NMR technique. <sup>13</sup>C NMR and <sup>1</sup>H spectra were obtained using a Bruker spectrometer (400 MHz) with tetramethylsilane as an internal reference. A mass spectrum was recorded using a PE-SCIEX API-300 with a Turboion spray mass spectrometer. Elemental analysis was carried out using the EURO EA3000 elemental analyser instrument. The open capillary tube method was used to determine the melting points on the Buchi melting point apparatus.</p></section><section id="sec14"><h3 class="pmc_sec_title">3.2. Feature mapping analysis using a structure-based pharmacophore</h3>
<p>The precise location of catalytic interactions between a protein and its co-crystallized ligand can be determined using a structure-based pharmacophore model. By analysing the data from this model, it is possible to identify the key pharmacophoric features that inhibit protein activity. In this case, a structure-based pharmacophore was developed based on the 3D structure of HIV-1 reverse transcriptase (PDB id: 1tkx) and its co-crystallized modulator, NAD. Using Discovery Studio 2.0, six essential pharmacophoric features were generated: hydrogen bond acceptors (HBAs), hydrophobic regions (HY), hydrogen bond donors (HBDs), and aromatic rings.<sup><a href="#cit24" class="usa-link" aria-describedby="cit24">24</a></sup></p></section><section id="sec15"><h3 class="pmc_sec_title">3.3. Synthesis of nevirapine analogues</h3>
<section id="sec16"><h4 class="pmc_sec_title">3.3.1. Preparation of 3(a–b)</h4>
<p>A mixture of 3-amino-2-chloro pyridine derivatives (2.4 g, 0.0187 mmol) and sodium hydrogen orthophosphate (6.25 g, 0.02 mmol) in THF-H<sub>2</sub>O was stirred at room temperature for 30 min. To this, 2-chloronicotinyol chloride (3.29 g, 0.0187 mmol) was added with continued stirring for 8 h at 60 °C to obtain a pinkish colour. Further, the reaction mixture was cooled to room temperature, extracted with ethyl acetate, and dried over anhydrous sodium sulphate, and the solvent was removed by rotatory evaporation to give crude carboxamide. It was further purified by column chromatography (silica gel 60–120 mesh size; 15–18% ethyl acetate in hexane) to give pure carboxamide.</p>
<p>3a: yield 91%, mp: 258.2–258.4 °C, IR (ATR, <em>ν</em>/cm<sup>−1</sup>): 3440.7 (NH), 3061.09 (C–H Ar), 2917.06 (Me), 1650.92 (CO), 1560.15 (C<span aria-roledescription="private character" aria-label="DOUBLE BOND, LENGTH AS M-DASH" aria-description="double bond, length as m-dash">

<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="13.200000pt" height="16.000000pt" viewBox="0 0 13.200000 16.000000" preserveAspectRatio="xMidYMid meet"><metadata>
Created by potrace 1.16, written by Peter Selinger 2001-2019
</metadata><g transform="translate(1.000000,15.000000) scale(0.017500,-0.017500)" fill="currentColor" stroke="none"><path d="M0 440 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z M0 280 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z"/></g></svg>

</span>N). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): <em>δ</em> 8.88 (s, 1H), 8.75 (s, 1H), 8.59 (d, 1H, <em>J</em> = 6.9 Hz), 8.30 (d, 1H, <em>J</em> = 7.64 Hz), 8.04 (s, 1H), 7.49 (t, 1H, <em>J</em> = 7.64 Hz), 2.41 (s, 3H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): <em>δ</em> 162.85, 151.96, 147.06, 144.90, 140.55, 137.72, 133.76, 130.87, 130.32, 130.02, 123.13, 18.08. MS-ESI <em>m</em>/<em>z</em> (calcd/found): 281.0012/281.0123. Elemental analysis for C<sub>12</sub>H<sub>9</sub>C<sub>12</sub>N<sub>3</sub>O (Calcd/Found): C(51.09/51.05), H(3.22/3.20), N(14.89/14.85).</p>
<p>3b: yield 90%, mp: 267.5–268.6 °C, IR (ATR, <em>ν</em>/cm<sup>−1</sup>): 3438.5 (NH), 3061 (C–H Ar), 1652.28 (CO), 1496 (C<span aria-roledescription="private character" aria-label="DOUBLE BOND, LENGTH AS M-DASH" aria-description="double bond, length as m-dash">

<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="13.200000pt" height="16.000000pt" viewBox="0 0 13.200000 16.000000" preserveAspectRatio="xMidYMid meet"><metadata>
Created by potrace 1.16, written by Peter Selinger 2001-2019
</metadata><g transform="translate(1.000000,15.000000) scale(0.017500,-0.017500)" fill="currentColor" stroke="none"><path d="M0 440 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z M0 280 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z"/></g></svg>

</span>N). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): <em>δ</em> 8.99 (s, 1H), 8.90 (d, 1H, <em>J</em> = 7.96 Hz), 8.57 (d, 1H, <em>J</em> = 4.36 Hz), 8.30 (d, 1H, <em>J</em> = 7.52 Hz), 8.21 (d, 1H, <em>J</em> = 4.48 Hz), 7.48 (t, 1H, <em>J</em> = 2.1 Hz), 7.37 (t, 1H, <em>J</em> = 2.4 Hz). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): <em>δ</em> 162.85, 151.96, 147.06, 144.90, 140.55, 137.72, 133.76, 130.87, 130.32, 130.02, 123.13. MS-ESI <em>m</em>/<em>z</em> (calcd/found): 266.9937/266.9966. Elemental analysis for C<sub>11</sub>H<sub>7</sub>C<sub>12</sub>N<sub>3</sub>O (calcd/found): C(49.28/49.25), H(2.63/2.60), N(15.67/15.60).</p></section><section id="sec17"><h4 class="pmc_sec_title">3.3.2. Preparation of 5(a–h)</h4>
<p>Sodium hydride (0.565 mmol) was washed with dry hexane. Then, substituted amine (0.3 mmol) and 10 ml dry THF were added to it at room temperature in a two-neck round-bottom flask. The reaction was stirred for 30 min, and appropriate carboxamide (1.7 mmol) was added. The reaction mixture was allowed to stir for 12 h at 60 °C. After the completion of the reaction, the reaction mixture was further cooled to room temperature and acidified with diluted HCl and extracted with ethyl acetate (2 × 10 ml). All organic layers were combined, dried over anhydrous sodium sulphate, and concentrated under vacuum to produce crude dipyridodiazepinone. The crude compound was purified from column chromatography (silica gel 60–120 mesh size, 10–14% ethyl acetate in hexane) to give pure dipyridodiazepine 5(a–h).</p>
<p>5a: yield 88%, mp: 244.4–244.8 °C, IR (ATR, <em>ν</em>/cm<sup>−1</sup>): 3440.7 (NH), 3061.09(C–H, Ar), 2917.06 (CH<sub>3</sub>), 1650.92 (CO). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): <em>δ</em> 8.62 (s, 1H), 8.29 (d, 1H, <em>J</em> = 4.2 Hz), 8.12 (d, 1H, <em>J</em> = 7.12 Hz), 7.72 (s, 1H), 7.37 (s, 1H), 6.92 (t, 1H, <em>J</em> = 6 Hz), 2.44 (s, 3H), 2.25 (m, 1H), 0.84 (m, 2H), 0.69 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): <em>δ</em> 162.82, 151.99, 147.03, 144.69, 140.53, 138.68, 130.20, 129.43, 123.38, 123.11, 122.39, 23.86, 18.96 7.24, 7.16. MS-ESI <em>m</em>/<em>z</em> (calcd/found): 266.3040/266.1168. Elemental analysis for C<sub>15</sub>H<sub>14</sub>N<sub>4</sub>O (calcd/found): C(67.65/67.60), H(5.30/5.25), N(21.04/21.02).</p>
<p>5b: yield 84%, mp: 247.2–257.9 °C, IR (ATR, <em>ν</em>/cm<sup>−1</sup>): 3438.5 (NH), 3061.09 (CH–Ar), 1652.28 (CO). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): <em>δ</em> 8.86 (s, 1H), 8.31 (d, 1H, <em>J</em> = 3.6 Hz), 8.05 (dd, 2H, <em>J</em> = 2.48 Hz), 7.59 (d, 1H, <em>J</em> = 2.48 Hz), 7.19 (t, 1H, <em>J</em> = 5.8 Hz), 6.94 (t, 1H, <em>J</em> = 8 Hz), 2.22 (m, 1H), 0.79 (m, 2H), 0.58 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): <em>δ</em> 162.82, 151.99, 147.03, 144.69, 140.65, 138.68, 130.20, 129.43, 123.38, 123.11, 122.39, 23.86, 7.24, 7.16. MS-ESI <em>m</em>/<em>z</em> (calcd/found): 252.2770/252.1011. Elemental analysis for C<sub>14</sub>H<sub>12</sub>N<sub>4</sub>O (calcd/found): C(66.65/62.55), H(4.79/4.75), N(22.21/22.20).</p>
<p>5c: yield 83%, mp: 243.5–243.8 °C, IR (ATR, <em>ν</em>/cm<sup>−1</sup>): 3440.7 (NH), 3061.09 (CH Ar), 2917.06 (Me), 1650.92 (CO). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): <em>δ</em> 8.91 (s, 1H), 8.31 (d, 1H, <em>J</em> = 3.6 Hz), 8.25 (d, 1H, <em>J</em> = 6.2 Hz), 7.99 (s, 1H), 7.58 (s, 1H), 6.95 (t, 1H, <em>J</em> = 4.5 Hz), 3.47 (m, 1H), 2.35 (m, 2H), 2.30 (s, 3H), 2.07 (m, 4H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): 169.26, 166.75, 147.83, 147.27, 141.74, 139.06, 136.82, 127.65, 125.14, 120.22, 114.49, 56.34, 30.08, 18.76. MS-ESI <em>m</em>/<em>z</em> (calcd/found): 280.1324/280.1321. Elemental analysis for C<sub>16</sub>H<sub>16</sub>N<sub>4</sub>O (calcd/found): C (68.55/68.50), H (5.75/5.71), N(19.99/19.95).</p>
<p>5d: yield: 82%, mp: 225.1–225.2 °C, IR (ATR, <em>ν</em>/cm<sup>−1</sup>): 3438.5 (NH), 3061.09 (C–H Ar), 1652.28 (CO), 1496.44 (C<span aria-roledescription="private character" aria-label="DOUBLE BOND, LENGTH AS M-DASH" aria-description="double bond, length as m-dash">

<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="13.200000pt" height="16.000000pt" viewBox="0 0 13.200000 16.000000" preserveAspectRatio="xMidYMid meet"><metadata>
Created by potrace 1.16, written by Peter Selinger 2001-2019
</metadata><g transform="translate(1.000000,15.000000) scale(0.017500,-0.017500)" fill="currentColor" stroke="none"><path d="M0 440 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z M0 280 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z"/></g></svg>

</span>N). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): <em>δ</em> 9.11 (s, 1H), 8.47 (d, 1H, <em>J</em> = 3.64 Hz), 8.18 (d, 1H, <em>J</em> = 3.52 Hz), 8.01 (d, 1H, <em>J</em> = 3.52 Hz), 7.62 (d, 1H, <em>J</em> = 2.4 Hz), 6.99 (t, 1H, <em>J</em> = 6.04 Hz), 6.88 (t, 1H, <em>J</em> = 5.96 Hz), 3.47 (m, 1H), 2.36 (m, 2H), 2.12 (m, 2H), 1.94 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): 169.26, 166.75, 147.83, 147.27, 141.74, 139.06, 136.77, 127.65, 125.14, 120.22, 114.49, 50.15, 18.76, 13.77. MS-ESI <em>m</em>/<em>z</em> (calcd/found): 266.1168/266.1165. Elemental analysis for C<sub>15</sub>H<sub>14</sub>N<sub>4</sub>O (calcd/found): C(67.65/67.65), H(5.30/5.25), N(21.04/21.01).</p>
<p>5e: yield 80%, mp: 225.1–225.2 °C, IR (ATR, <em>ν</em>/cm<sup>−1</sup>): 3397 (NH), 2926.1 (CH Ar), 1650.5 (CO), 1496.44 (C<span aria-roledescription="private character" aria-label="DOUBLE BOND, LENGTH AS M-DASH" aria-description="double bond, length as m-dash">

<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="13.200000pt" height="16.000000pt" viewBox="0 0 13.200000 16.000000" preserveAspectRatio="xMidYMid meet"><metadata>
Created by potrace 1.16, written by Peter Selinger 2001-2019
</metadata><g transform="translate(1.000000,15.000000) scale(0.017500,-0.017500)" fill="currentColor" stroke="none"><path d="M0 440 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z M0 280 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z"/></g></svg>

</span>N). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): <em>δ</em> 8.31 (s, 1H), 8.15 (d, 1H, <em>J</em> = 5.4 Hz), 7.89 (d, 1H, <em>J</em> = 6.2 Hz), 7.66 (s, 1H), 7.47 (s, 1H), 6.75 (t, 1H, <em>J</em> = 4.9 Hz), 3.02 (m, 1H), 2.23 (s, 3H), 1.86 (m, 2H), 1.56 (m, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): 169.26, 166.75, 147.83, 147.27, 141.74, 139.06, 136.77, 127.65, 125.14, 120.22, 114.49, 61.82, 34.70, 25.66, 18.76. MS-ESI <em>m</em>/<em>z</em> (calcd/found): 294.1481/294.1475. Elemental analysis for C<sub>17</sub>H<sub>18</sub>N<sub>4</sub>O (calcd/found): C(69.37/69.35), H(6.16/6.14), N(19.03/19.01).</p>
<p>5f: yield 81%, mp: 222.1–222.4 °C, IR (ATR, <em>ν</em>/cm<sup>−1</sup>): 3438.5 (NH), 3061.9 (C–H Ar), 1652.28 (CO), 1496.44 (C<span aria-roledescription="private character" aria-label="DOUBLE BOND, LENGTH AS M-DASH" aria-description="double bond, length as m-dash">

<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="13.200000pt" height="16.000000pt" viewBox="0 0 13.200000 16.000000" preserveAspectRatio="xMidYMid meet"><metadata>
Created by potrace 1.16, written by Peter Selinger 2001-2019
</metadata><g transform="translate(1.000000,15.000000) scale(0.017500,-0.017500)" fill="currentColor" stroke="none"><path d="M0 440 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z M0 280 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z"/></g></svg>

</span>N). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): <em>δ</em> 8.80 (s, 1H), 8.22 (d, 1H, <em>J</em> = 2.36 Hz), 7.98 (d, 1H, <em>J</em> = 6.9 Hz), 7.90 (d, 1H, <em>J</em> = 2.36 Hz), 7.52 (d, 1H, <em>J</em> = 3.56 Hz), 6.90 (t, 1H, <em>J</em> = 5.88 Hz), 6.77 (t, 1H, <em>J</em> = 6.16 Hz), 3.02 (m, 1H), 1.85 (m, 2H), 1.58 (m, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): 169.26, 166.75, 149.04, 147.83, 142.78, 139.06, 134.91, 129.21, 120.22, 120.13, 114.49, 61.82, 34.70, 25.66. MS-ESI <em>m</em>/<em>z</em> (calcd/found): 280.1324/280.1322. Elemental analysis for C<sub>16</sub>H<sub>16</sub>N<sub>4</sub>O (calcd/found): C(68.55/68.51), H(5.75/5.73), N(19.99/19.95).</p>
<p>5g: yield 83%, mp: 220.4–220.5 °C, IR (ATR, <em>ν</em>/cm<sup>−1</sup>): 3324.06 (NH), 3142.34 (CH–Ar), 2920.77(Me), 2851.32 (alkyl), 1666.73 (CO). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): <em>δ</em> 8.50 (s, 1H), 8.14 (d, 1H, <em>J</em> = 3.6 Hz), 8.04 (d, 1H, <em>J</em> = 3.56 Hz), 7.59 (s, 1H), 7.30 (s, 1H), 6.86 (t, 1H, <em>J</em> = 5.92 Hz), 4.25 (dt, 2H, <em>J</em> = 4.2 Hz), 2.29 (s, 3H), 1.61 (m, 2H), 1.04 (t, 3H, <em>J</em> = 5.32 Hz). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): 169.26, 167.48, 148.38, 146.64, 142.76, 139.24, 135.76, 127.96, 127.48, 121.76, 113.95, 46.25, 21.13, 18.76, 11.99. MS-ESI <em>m</em>/<em>z</em> (calcd/found): 268.1324/268.1321. Elemental analysis for C<sub>15</sub>H<sub>16</sub>N<sub>4</sub>O (calcd/found): C(67.15/67.12), H(6.01/6.04), N(20.88/20.80).</p>
<p>5h: yield 82%, mp: 221.2–221.4 °C, IR (ATR, <em>ν</em>/cm<sup>−1</sup>): 3314.95 (NH), 3060.03 (CH–Ar), 2924.85 (Me), 1670.12 (CO), 1495.13 (C<span aria-roledescription="private character" aria-label="DOUBLE BOND, LENGTH AS M-DASH" aria-description="double bond, length as m-dash">

<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="13.200000pt" height="16.000000pt" viewBox="0 0 13.200000 16.000000" preserveAspectRatio="xMidYMid meet"><metadata>
Created by potrace 1.16, written by Peter Selinger 2001-2019
</metadata><g transform="translate(1.000000,15.000000) scale(0.017500,-0.017500)" fill="currentColor" stroke="none"><path d="M0 440 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z M0 280 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z"/></g></svg>

</span>N). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): <em>δ</em> 8.84 (s, 1H), 8.17 (d, 1H, <em>J</em> = 5.8 Hz), 8.16 (dd, 2H, <em>J</em> = 7.5 Hz), 7.91 (d, 1H, <em>J</em> = 6.4 Hz), 6.87 (t, 1H, <em>J</em> = 4.5 Hz), 6.74 (t, 1H, <em>J</em> = 5.4 Hz), 4.04 (dt, 2H, <em>J</em> = 4.6 Hz), 1.75 (m, 2H), 1.06 (t, 3H, <em>J</em> = 5.32 Hz). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): 169.26, 167.48, 148.38, 148.08, 144.04, 139.24, 133.86, 129.73, 122.09, 121.76, 113.95, 46.25, 21.13, 11.99. MS-ESI <em>m</em>/<em>z</em> (calcd/found): 254.1168/254.1164. Elemental analysis for C<sub>14</sub>H<sub>14</sub>N<sub>4</sub>O (calcd/found): C(66.13/66.12), H(5.55/5.51), N(22.03/22.01).</p></section></section><section id="sec18"><h3 class="pmc_sec_title">3.4. HIV-1 reverse transcriptase inhibitory study</h3>
<p>The effect of synthesized analogues was determined using the RT assay colorimetric kit (MyBioSource, Cat MBS412605) as per the instructions of the manufacturer. Briefly, reaction buffer and HIV RT reverse transcriptase enzyme solution were added to microfuge tubes containing different concentrations (1, 10, 50, and 100 nM) of nevirapine and its analogues. Lysis buffer, reaction buffer, and HIV reverse transcriptase were used without treatment as negative controls. After 20 min incubation at 37 °C, 100 μl of each reaction mixture was transferred to a streptavidin-coated microwell and again incubated for 20 min at 37 °C. Before adding 100 μl of HRP Anti Digoxigenin Conjugate, the ELISA plate was completely washed 5 times with wash buffer. A further 45 min incubation was followed by washing again 5 times using wash buffer at 37 °C. 100 μl of ABTS substrate solution was added to the wells and incubated for 30 min at 37 °C until the development of colour was sufficient to detect using an ELISA plate reader at 405 nm. Otherwise, the addition of 25 μl of stop solution can also be used to terminate the reaction and then read the absorbance at 405 nm.<sup><a href="#cit25" class="usa-link" aria-describedby="cit25">25</a></sup></p></section><section id="sec19"><h3 class="pmc_sec_title">3.5. Molecular docking</h3>
<p>The computational technique of molecular docking is utilized to examine the interaction between ligands and macromolecules. The three-dimensional (3D) structure of HIV-1 reverse transcriptase (PDB id: 1tkx) was retrieved from the <a href="https://www.rcsb.org/protein" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org/protein</a> data bank site. Further, the macromolecule's 3D structure was opened in Discovery Studio Visualizer, hydrogen atoms were added, water molecules were removed, and valency was satisfied and saved in the PDBQT format.<sup><a href="#cit26" class="usa-link" aria-describedby="cit26">26,27</a></sup> Subsequently, both ligands (5a, 5b, and nevirapine) and macromolecules were subjected to a PyRx window to predict the binding interaction using the AutoDock Vina module.<sup><a href="#cit28" class="usa-link" aria-describedby="cit28">28</a></sup> Other parameters remained at their default values, except the grid box modified following the active sites of individual macromolecules. Additionally, the ligand's bonds were permitted to rotate unrestrictedly, resulting in a rigid receptor. After the molecular docking study was completed and 10 configurations for each drug–protein complex were constructed for all synthesized analogues, text files with scoring results were generated for manual comparison. The docking poses with weaker binding energy were believed to be the best. Using Discovery Studio Visualizer and PyMol, the interaction between ligands and macromolecules was analyzed and visualized.</p></section><section id="sec20"><h3 class="pmc_sec_title">3.6. Drug-like, physicochemical, and ADMET properties</h3>
<p>Lipinski's rule of five criteria was used to evaluate the potential drug-likeness of each ligand. The ADMETSar web tool was used to validate Lipinski's rule of five parameters. To evaluate the pharmacodynamics of a potential drug candidate, it is necessary to consider the ADMET mechanism of the molecule.<sup><a href="#cit29" class="usa-link" aria-describedby="cit29">29</a></sup> After molecular docking, ADMETSar and SWISSADME servers were used to predict the ADMET mechanism of the nevirapine analogues. The SMILES format of the ligand structure was used in the web server.</p></section><section id="sec21"><h3 class="pmc_sec_title">3.7. Molecular dynamics simulations</h3>
<p>MD simulations were conducted with the GROMACS 2020 software to evaluate the credibility of the docking results.<sup><a href="#cit30" class="usa-link" aria-describedby="cit30">30,31</a></sup> The simulations used the CHARMM36 force field, TIP3P water model, and CHARMM36-Feb 2021 update version.<sup><a href="#cit13" class="usa-link" aria-describedby="cit13">13</a></sup> Proteins, namely, 5dyw and 4ey7, were placed together with their respective complexes and positioned in the geometric center of a dodecahedron-shaped container. The distance between the protein complexes and the edges of the container was set at 10 Å. To achieve system neutralization at this step, negative ions (Cl<sup>−</sup>) and positive ions (Na<sup>+</sup>) were introduced parallelly by displacing the solvent molecules as required. Then energy minimization procedure was conducted using the Verlet cutoff methodology to mitigate steric hindrances inside the system. Further, the energy minimum was conducted using the steepest descent method with a maximum step size of 50 000 to ensure the absence of atomic collisions. Furthermore, equilibrium states were achieved by 100 ns NPT and NVT simulations at a pressure of 1 ATM and at a temperature of 300 kelvin. Subsequently, a molecular simulation lasting 10 ns was conducted. Following MD simulation, an investigation of several properties, such as Radius of Gyration (Rg), Root Mean Square Deviation (RMSD), and Root Mean Square Fluctuation (RMSF), was performed on the trajectory. These properties were obtained using the built-in capabilities provided by Desmond software.</p></section><section id="sec22"><h3 class="pmc_sec_title">3.8. Isothermal titration calorimetry (ITC)</h3>
<p>To analyse the energetics of the interaction of nevirapine within the solution, ITC is recognized as the gold standard experimental device. ITC offers accurate, quick, and easy label-free monitoring of the thermodynamics of molecular interactions by determining the heat gained or released during interactions.<sup><a href="#cit32" class="usa-link" aria-describedby="cit32">32</a></sup> Therefore, MICRO CAL PEAQ-ITC (Malvern) is used to determine the thermodynamics of the interactions of ligands with the macromolecule.<sup><a href="#cit33" class="usa-link" aria-describedby="cit33">33,34</a></sup> The calibration of ITC was performed before running the experiments with standard EDTA and Ca<sup>2+</sup> solutions, and binding parameters matched well with the standard thermodynamic parameters of Ca–EDTA binding. The exact concentration of macromolecule (HSA) and synthesized compounds required for ITC experiments was determined using a UV-Visible spectrophotometer from SHIMADZU at 280 nm (<em>A</em><sub>1cm</sub><sup>1%</sup> = 5.3). All the solutions were prepared in 5% DMSO solution in 20 mM phosphate buffer (pH = 7.4) for microcalorimetric experimental analysis. The same solution was also used in the reference cell to evaluate the thermodynamic properties associated with the interactions. The macromolecule solution in the sample cell was titrated with the ligand solution filled in a computer-controlled syringe. A syringe (40 μl) was used to inject the ligand solution into the sample cell that contained 200 μl of macromolecules. Dilution experiments were conducted by titrating the syringe solution (5a, 5b and nevirapine) against a similar buffer. The final heat reported in the ITC profiles was calculated by subtracting the heat produced in the dilution experiment from the primary titration profile. At the end of titration, the concentration in the syringe was adjusted to achieve saturated heat for each experiment. The experiment was programmed for 19 consecutive injections, with the first injection 0.4 μl, followed by 18 subsequent injections (2 μl), at 150-second time intervals. For all titrations throughout the run, a stirring speed of 750 rpm was fixed for the syringe.</p>
<section id="sec23"><h4 class="pmc_sec_title">3.8.1. Calculation</h4>
<p>To analyse the data, the Micro CAL PEAQ ITC analysis software provided with the instrument was used. The association constant (<em>K</em><sub>a</sub>) and binding stoichiometry (<em>n</em>), as well as the entropy and enthalpy changes (Δ<em>S</em> and Δ<em>H</em>), during drug macromolecule interaction were obtained directly from the ITC instrument. The changes in the free energy value Δ<em>G</em> were subsequently calculated using the following equation:</p>
<table class="disp-formula p" id="eqn4"><tr>
<td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML">Δ<em>G</em> = Δ<em>H</em> − <em>T</em>Δ<em>S</em></span></td>
<td class="label">4</td>
</tr></table></section></section><section id="sec24"><h3 class="pmc_sec_title">3.9. Fluorescence spectroscopy</h3>
<p>A PerkinElmer model LS45 spectrofluorometer was used to measure the intrinsic fluorescence of HSA in the presence of nevirapine analogs in a quartz cell of 3 ml capacity and path length of 1 cm. The protein concentration in all experiments was kept constant. The excitation and emission slits were fixed at 10 nm. The wavelength of excitation was fixed at 280 nm, and the emission spectra were recorded in the wavelength range of 300–500 nm at a scan rate of 600 nm min<sup>−1</sup>. For this, HSA and drug solutions were made in 5% DMSO solution in 20 mM phosphate buffer to achieve the required solubility of nevirapine and its analogous for quenching studies.<sup><a href="#cit35" class="usa-link" aria-describedby="cit35">35</a></sup></p></section></section><section id="sec25"><h2 class="pmc_sec_title">4. Conclusion</h2>
<p>In the present study, novel dipyridodiazepinone (nevirapine) analogue 5(a–h) was synthesized using a simple and efficient base-promoted C–N coupling reaction assisted by amination amidation and cyclocondensation reactions. Synthesized analogues were analyzed by employing feature mapping analysis using structure-based pharmacophore design, molecular docking and <em>in vitro</em> HIV-1 reverse transcriptase inhibitory activity, MD simulation and <em>in silico</em> pharmacokinetic properties utilizing the ADMETSar web tool. Further, thermodynamic and spectroscopic analyses of the synthesized analogues were conducted using an ITC experiment to understand their binding interactions with HSA.</p>
<p>The feature mapping analysis and molecular docking studies indicated that compounds 5a and 5b possess good fit value and affinity towards HIV-1 reverse transcriptase and showed considerable interaction with key residues such as hydrogen residue (Lys101), pi–sigma residue (Leu100), alkyl and pi–alkyl residues (Lys103, Tyr181, Val106, Leu234, and Tyr188) available at the catalytic site of enzyme. Furthermore, based on ADMET and physicochemical analysis, the results indicate that compound 5b had favourable pharmacokinetic characteristics and drug-like properties. Most notably, both compounds exhibited considerable safety in toxicity assessments. The findings from ITC revealed key interactions between the synthesized analogues and HSA protein, which in turn indicated the possibility of their expedited transportation and associated therapeutic effect in the recipient. In conclusion, these findings suggest that these compounds have significant HIV-1 reverse transcriptase inhibitory activities and possess drug-like properties and thus need immediate in-depth exploration for their utility as therapeutic agents in the management of HIV-1-mediated diseases.</p></section><section id="sec26"><h2 class="pmc_sec_title">Author contributions</h2>
<p>SJ: investigation and writing original draft; SJ: molecular docking studies; AM: fluorescence and ITC studies; KV: <em>in vitro</em> studies; SoJ: data validation; DK: project administration; JD: supervision, conceptualization and manuscript editing; SS: supervision and manuscript editing.</p></section><section id="sec27"><h2 class="pmc_sec_title">Conflicts of interest</h2>
<p>The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.</p></section><section id="sec28"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="RA-015-D5RA03504J-s001"><div class="caption p"><span>RA-015-D5RA03504J-s001</span></div>
<div class="media p" id="d67e2131"><div class="caption">
<a href="/articles/instance/12376879/bin/RA-015-D5RA03504J-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">RA-015-D5RA03504J-s001.pdf</a><sup> (4MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors acknowledge the Department of Science and Technology (DST-FIST, India, Order Number SR/FST/2022/252) for providing all the necessary facilities. The author would also like to thank Prof. Ina Aditya Shastri, Vice-Chancellor of Banasthali Vidyapith, Rajasthan, India, for the successful accomplishment of the present work.</p></section><section id="sec29"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data supporting this article have been included as part of the SI. IR, HSQC, <sup>1</sup>H NMR, <sup>13</sup>C NMR, mass. See DOI: <a href="https://doi.org/10.1039/d5ra03504j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1039/d5ra03504j</a>.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="cit1">
<cite>Leśniewska A. Przybylski P. Eur. J. Med. Chem. 2024;275:116556. doi: 10.1016/j.ejmech.2024.116556.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2024.116556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38879971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;author=A.%20Le%C5%9Bniewska&amp;author=P.%20Przybylski&amp;volume=275&amp;publication_year=2024&amp;pages=116556&amp;pmid=38879971&amp;doi=10.1016/j.ejmech.2024.116556&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit2">
<cite>Malki Y. Martinez J. Masurier N. Med. Res. Rev. 2021;41:2247–2315. doi: 10.1002/med.21795.</cite> [<a href="https://doi.org/10.1002/med.21795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33645848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Med.%20Res.%20Rev.&amp;author=Y.%20Malki&amp;author=J.%20Martinez&amp;author=N.%20Masurier&amp;volume=41&amp;publication_year=2021&amp;pages=2247-2315&amp;pmid=33645848&amp;doi=10.1002/med.21795&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit3">
<cite>Jaiswal S. Kishore D. Bhardwaj A. Bhardwaj K. Richa S. Jain S. Dwivedi J. Sharma S. Org. Biomol. Chem. 2024;22:6520–6531. doi: 10.1039/d4ob00828f.</cite> [<a href="https://doi.org/10.1039/d4ob00828f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39072429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Org.%20Biomol.%20Chem.&amp;author=S.%20Jaiswal&amp;author=D.%20Kishore&amp;author=A.%20Bhardwaj&amp;author=K.%20Bhardwaj&amp;author=S.%20Richa&amp;volume=22&amp;publication_year=2024&amp;pages=6520-6531&amp;pmid=39072429&amp;doi=10.1039/d4ob00828f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit4">
<cite>Fader L. D. Landry S. Morin S. Kawai S. H. Bousquet Y. Hucke O. Goudreau N. Lemke C. T. Bonneau P. Titolo S. Mason S. Bioorg. Med. Chem. Lett. 2013;23:3396–3400. doi: 10.1016/j.bmcl.2013.03.073.</cite> [<a href="https://doi.org/10.1016/j.bmcl.2013.03.073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23583513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Med.%20Chem.%20Lett.&amp;author=L.%20D.%20Fader&amp;author=S.%20Landry&amp;author=S.%20Morin&amp;author=S.%20H.%20Kawai&amp;author=Y.%20Bousquet&amp;volume=23&amp;publication_year=2013&amp;pages=3396-3400&amp;pmid=23583513&amp;doi=10.1016/j.bmcl.2013.03.073&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit5">
<cite>Bellarosa D. Antonelli G. Bambacioni F. Giannotti D. Viti G. Nannicini R. Giachetti A. Dianzani F. Witvrouw M. Pauwels R. Desmyter J. Antiviral Res. 1996;30:109–124. doi: 10.1016/0166-3542(95)00906-x.</cite> [<a href="https://doi.org/10.1016/0166-3542(95)00906-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8783803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antiviral%20Res.&amp;author=D.%20Bellarosa&amp;author=G.%20Antonelli&amp;author=F.%20Bambacioni&amp;author=D.%20Giannotti&amp;author=G.%20Viti&amp;volume=30&amp;publication_year=1996&amp;pages=109-124&amp;pmid=8783803&amp;doi=10.1016/0166-3542(95)00906-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit6">
<cite>Fader L. D. Bethell R. Bonneau P. Bös M. Bousquet Y. Cordingley M. G. Coulombe R. Deroy P. Faucher A. M. Gagnon A. Goudreau N. Bioorg. Med. Chem. Lett. 2011;21:398–404. doi: 10.1016/j.bmcl.2010.10.131.</cite> [<a href="https://doi.org/10.1016/j.bmcl.2010.10.131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21087861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Med.%20Chem.%20Lett.&amp;author=L.%20D.%20Fader&amp;author=R.%20Bethell&amp;author=P.%20Bonneau&amp;author=M.%20B%C3%B6s&amp;author=Y.%20Bousquet&amp;volume=21&amp;publication_year=2011&amp;pages=398-404&amp;pmid=21087861&amp;doi=10.1016/j.bmcl.2010.10.131&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit7">
<cite>Guo W. Zhao M. Tan W. Zheng L. Tao K. Fan X. Org. Chem. Front. 2019;6:2120–2141.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Org.%20Chem.%20Front.&amp;author=W.%20Guo&amp;author=M.%20Zhao&amp;author=W.%20Tan&amp;author=L.%20Zheng&amp;author=K.%20Tao&amp;volume=6&amp;publication_year=2019&amp;pages=2120-2141&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit8">
<cite>Bhardwaj K. Jaiswal S. Bhardwaj A. Arora D. Jain S. Kumar G. Dwivedi J. Sharma S. J. Mol. Struct. 2025;1336:142105.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Struct.&amp;author=K.%20Bhardwaj&amp;author=S.%20Jaiswal&amp;author=A.%20Bhardwaj&amp;author=D.%20Arora&amp;author=S.%20Jain&amp;volume=1336&amp;publication_year=2025&amp;pages=142105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit9">
<cite>Yaduvanshi N. Tewari S. Jaiswal S. Devi M. Shukla S. Dwivedi J. Sharma S. Inorg. Chem. Commun. 2024;161:111927.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Inorg.%20Chem.%20Commun.&amp;author=N.%20Yaduvanshi&amp;author=S.%20Tewari&amp;author=S.%20Jaiswal&amp;author=M.%20Devi&amp;author=S.%20Shukla&amp;volume=161&amp;publication_year=2024&amp;pages=111927&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit10">
<cite>Bhardwaj A. Jaiswal S. Verma K. Bhardwaj K. Sharma M. Jain S. Dwivedi J. Sharma S. Chem. Rec. 2025;25:202400156. doi: 10.1002/tcr.202400156.</cite> [<a href="https://doi.org/10.1002/tcr.202400156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39803800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Rec.&amp;author=A.%20Bhardwaj&amp;author=S.%20Jaiswal&amp;author=K.%20Verma&amp;author=K.%20Bhardwaj&amp;author=M.%20Sharma&amp;volume=25&amp;publication_year=2025&amp;pages=202400156&amp;pmid=39803800&amp;doi=10.1002/tcr.202400156&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit11">
<cite>Bec G. Meyer B. Gerard M. A. Steger J. Fauster K. Wolff P. Burnouf D. Micura R. Dumas P. Ennifar E. J. Am. Chem. Soc. 2013;135:9743–9752. doi: 10.1021/ja4018418.</cite> [<a href="https://doi.org/10.1021/ja4018418" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23742167/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;author=G.%20Bec&amp;author=B.%20Meyer&amp;author=M.%20A.%20Gerard&amp;author=J.%20Steger&amp;author=K.%20Fauster&amp;volume=135&amp;publication_year=2013&amp;pages=9743-9752&amp;pmid=23742167&amp;doi=10.1021/ja4018418&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit12">
<cite>Pîrnău A. Mic M. Neamţu S. Floare C. G. Bogdan M. Spectrochim. Acta, Part A. 2018;191:226–232. doi: 10.1016/j.saa.2017.10.032.</cite> [<a href="https://doi.org/10.1016/j.saa.2017.10.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29040928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Spectrochim.%20Acta,%20Part%20A&amp;author=A.%20P%C3%AErn%C4%83u&amp;author=M.%20Mic&amp;author=S.%20Neam%C5%A3u&amp;author=C.%20G.%20Floare&amp;author=M.%20Bogdan&amp;volume=191&amp;publication_year=2018&amp;pages=226-232&amp;pmid=29040928&amp;doi=10.1016/j.saa.2017.10.032&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit13">
<cite>Jaiswal S. Devi M. Yaduvanshi N. Jain S. Dwivedi J. Kishore D. Kuznetsov A. E. Sharma S. J. Mol. Struct. 2024;1314:138734.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Struct.&amp;author=S.%20Jaiswal&amp;author=M.%20Devi&amp;author=N.%20Yaduvanshi&amp;author=S.%20Jain&amp;author=J.%20Dwivedi&amp;volume=1314&amp;publication_year=2024&amp;pages=138734&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit14">
<cite>Guan L. Yang H. Cai Y. Sun L. Di P. Li W. Liu G. Tang Y. Medchemcomm. 2019;10:148–157. doi: 10.1039/c8md00472b.</cite> [<a href="https://doi.org/10.1039/c8md00472b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6350845/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30774861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Medchemcomm&amp;author=L.%20Guan&amp;author=H.%20Yang&amp;author=Y.%20Cai&amp;author=L.%20Sun&amp;author=P.%20Di&amp;volume=10&amp;publication_year=2019&amp;pages=148-157&amp;pmid=30774861&amp;doi=10.1039/c8md00472b&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit15">
<cite>Kaur A. Kaur K. Banipal P. K. Banipal T. S. J. Chem. Thermodyn. 2021;152:106269.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Thermodyn.&amp;author=A.%20Kaur&amp;author=K.%20Kaur&amp;author=P.%20K.%20Banipal&amp;author=T.%20S.%20Banipal&amp;volume=152&amp;publication_year=2021&amp;pages=106269&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit16">
<cite>Alshaikh N. E. Zaki M. Sharfalddin A. A. Al-Radadi N. S. Hussien M. A. Hassan W. M. Arabian J. Chem. 2023;16:104845.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Arabian%20J.%20Chem.&amp;author=N.%20E.%20Alshaikh&amp;author=M.%20Zaki&amp;author=A.%20A.%20Sharfalddin&amp;author=N.%20S.%20Al-Radadi&amp;author=M.%20A.%20Hussien&amp;volume=16&amp;publication_year=2023&amp;pages=104845&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit17">
<cite>Jorgensen W. L. Bollini M. Thakur V. V. Domaoal R. A. Spasov K. A. Anderson K. S. J. Am. Chem. Soc. 2011;133:15686–15696. doi: 10.1021/ja2058583.</cite> [<a href="https://doi.org/10.1021/ja2058583" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3183387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21853995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;author=W.%20L.%20Jorgensen&amp;author=M.%20Bollini&amp;author=V.%20V.%20Thakur&amp;author=R.%20A.%20Domaoal&amp;author=K.%20A.%20Spasov&amp;volume=133&amp;publication_year=2011&amp;pages=15686-15696&amp;pmid=21853995&amp;doi=10.1021/ja2058583&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit18">
<cite>Molla M. H. R. Asseri A. H. Islam M. S. Egypt. J. Med. Hum. Genet. 2023;24:51.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Egypt.%20J.%20Med.%20Hum.%20Genet.&amp;author=M.%20H.%20R.%20Molla&amp;author=A.%20H.%20Asseri&amp;author=M.%20S.%20Islam&amp;volume=24&amp;publication_year=2023&amp;pages=51&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit19">
<cite>Mahboubi-Rabbani M. Abbasi M. Hajimahdi Z. Zarghi A. Iran. J. Pharm. Res. 2021;20:333–369. doi: 10.22037/ijpr.2021.115446.15370.</cite> [<a href="https://doi.org/10.22037/ijpr.2021.115446.15370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8457747/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34567166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Iran.%20J.%20Pharm.%20Res.&amp;author=M.%20Mahboubi-Rabbani&amp;author=M.%20Abbasi&amp;author=Z.%20Hajimahdi&amp;author=A.%20Zarghi&amp;volume=20&amp;publication_year=2021&amp;pages=333-369&amp;pmid=34567166&amp;doi=10.22037/ijpr.2021.115446.15370&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit20">
<cite>Seckler J. M. Barkley M. D. Wintrode P. L. Biophys. J. 2011;100:144. doi: 10.1016/j.bpj.2010.11.004.</cite> [<a href="https://doi.org/10.1016/j.bpj.2010.11.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3010017/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21190666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.%20J.&amp;author=J.%20M.%20Seckler&amp;author=M.%20D.%20Barkley&amp;author=P.%20L.%20Wintrode&amp;volume=100&amp;publication_year=2011&amp;pages=144&amp;pmid=21190666&amp;doi=10.1016/j.bpj.2010.11.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit21">
<cite>Chandra P. Ganguly S. Karmakar S. Chem. Proc. 2021;3:33.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Proc.&amp;author=P.%20Chandra&amp;author=S.%20Ganguly&amp;author=S.%20Karmakar&amp;volume=3&amp;publication_year=2021&amp;pages=33&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit22">
<cite>Kim M. J. Yu K. L. Han R. Lee Y. Oh K. You J. C. ACS Infect. Dis. 2023;9:1582–1592. doi: 10.1021/acsinfecdis.3c00166.</cite> [<a href="https://doi.org/10.1021/acsinfecdis.3c00166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37415514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Infect.%20Dis.&amp;author=M.%20J.%20Kim&amp;author=K.%20L.%20Yu&amp;author=R.%20Han&amp;author=Y.%20Lee&amp;author=K.%20Oh&amp;volume=9&amp;publication_year=2023&amp;pages=1582-1592&amp;pmid=37415514&amp;doi=10.1021/acsinfecdis.3c00166&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit23">
<cite>Singh A. K. Kumar A. Arora S. Kumar R. Verma A. Khalilullah H. Jaremko M. Emwas A. H. Kumar P. Chem. Biol. Drug Des. 2024;103:e14372. doi: 10.1111/cbdd.14372.</cite> [<a href="https://doi.org/10.1111/cbdd.14372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37817296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.%20Drug%20Des.&amp;author=A.%20K.%20Singh&amp;author=A.%20Kumar&amp;author=S.%20Arora&amp;author=R.%20Kumar&amp;author=A.%20Verma&amp;volume=103&amp;publication_year=2024&amp;pages=e14372&amp;pmid=37817296&amp;doi=10.1111/cbdd.14372&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit24">
<cite>Jain S. Sharma S. Paliwal A. Dwivedi J. Paliwal S. Paliwal V. Paliwal S. Sharma J. Med. Chem. Res. 2023;33:1–15.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Med.%20Chem.%20Res.&amp;author=S.%20Jain&amp;author=S.%20Sharma&amp;author=A.%20Paliwal&amp;author=J.%20Dwivedi&amp;author=S.%20Paliwal&amp;volume=33&amp;publication_year=2023&amp;pages=1-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit25">
<cite>Devi M. Jaiswal S. Yaduvanshi N. Kaur N. Kishore D. Dwivedi J. Sharma S. ChemistrySelect. 2023;8:202204710.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=ChemistrySelect&amp;author=M.%20Devi&amp;author=S.%20Jaiswal&amp;author=N.%20Yaduvanshi&amp;author=N.%20Kaur&amp;author=D.%20Kishore&amp;volume=8&amp;publication_year=2023&amp;pages=202204710&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit26">
<cite>Sain S. Jaiswal S. Jain S. Misra N. Srivastava A. Jendra R. Kishore D. Dwivedi J. Wabaidur S. M. Islam M. A. Sharma S. Chem. Biodivers. 2022;19:202200540. doi: 10.1002/cbdv.202200540.</cite> [<a href="https://doi.org/10.1002/cbdv.202200540" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36310125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Biodivers.&amp;author=S.%20Sain&amp;author=S.%20Jaiswal&amp;author=S.%20Jain&amp;author=N.%20Misra&amp;author=A.%20Srivastava&amp;volume=19&amp;publication_year=2022&amp;pages=202200540&amp;pmid=36310125&amp;doi=10.1002/cbdv.202200540&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit27">
<cite>Devi M. Jaiswal S. Yaduvanshi N. Jain S. Jain S. Verma K. Verma R. Kishore D. Dwivedi J. Sharma S. J. Mol. Struct. 2023;1286:135571.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Struct.&amp;author=M.%20Devi&amp;author=S.%20Jaiswal&amp;author=N.%20Yaduvanshi&amp;author=S.%20Jain&amp;author=S.%20Jain&amp;volume=1286&amp;publication_year=2023&amp;pages=135571&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit28">
<cite>Verma K. Makwana S. Paliwal S. Paliwal V. Jain S. Paliwal S. Sharma S. Curr. Res. Pharmacol. Drug Discovery. 2022;3:100088. doi: 10.1016/j.crphar.2022.100088.</cite> [<a href="https://doi.org/10.1016/j.crphar.2022.100088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8818901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35146420/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Res.%20Pharmacol.%20Drug%20Discovery&amp;author=K.%20Verma&amp;author=S.%20Makwana&amp;author=S.%20Paliwal&amp;author=V.%20Paliwal&amp;author=S.%20Jain&amp;volume=3&amp;publication_year=2022&amp;pages=100088&amp;pmid=35146420&amp;doi=10.1016/j.crphar.2022.100088&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit29">
<cite>Wensing A. M. Calvez V. Ceccherini-Silberstein F. Charpentier C. Günthard H. F. Paredes R. Shafer R. W. Richman D. D. Antiviral Med. 2022;30:559.</cite> [<a href="/articles/PMC9681141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36375130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antiviral%20Med.&amp;author=A.%20M.%20Wensing&amp;author=V.%20Calvez&amp;author=F.%20Ceccherini-Silberstein&amp;author=C.%20Charpentier&amp;author=H.%20F.%20G%C3%BCnthard&amp;volume=30&amp;publication_year=2022&amp;pages=559&amp;pmid=36375130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit30">
<cite>Vijayan R. S. K. Arnold E. Das K. Proteins: Struct., Funct., Bioinf. 2014;82:815–829. doi: 10.1002/prot.24460.</cite> [<a href="https://doi.org/10.1002/prot.24460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4502926/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24174331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins:%20Struct.,%20Funct.,%20Bioinf.&amp;author=R.%20S.%20K.%20Vijayan&amp;author=E.%20Arnold&amp;author=K.%20Das&amp;volume=82&amp;publication_year=2014&amp;pages=815-829&amp;pmid=24174331&amp;doi=10.1002/prot.24460&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit31">
<cite>Jaiswal S. Verma K. Dwivedi J. Sharma S. Eur. J. Med. Chem. 2024;271:116388. doi: 10.1016/j.ejmech.2024.116388.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2024.116388" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38614062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;author=S.%20Jaiswal&amp;author=K.%20Verma&amp;author=J.%20Dwivedi&amp;author=S.%20Sharma&amp;volume=271&amp;publication_year=2024&amp;pages=116388&amp;pmid=38614062&amp;doi=10.1016/j.ejmech.2024.116388&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit32">
<cite>Zhou X. Yang Q. Xie X. Hu Q. Qi F. Rahman Z. U. Chen X. Dyes Pigm. 2012;92:1100–1107.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Dyes%20Pigm.&amp;author=X.%20Zhou&amp;author=Q.%20Yang&amp;author=X.%20Xie&amp;author=Q.%20Hu&amp;author=F.%20Qi&amp;volume=92&amp;publication_year=2012&amp;pages=1100-1107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit33">
<cite>Gharai P. K. Khan J. Pradhan K. Mallesh R. Garg S. Arshi M. U. Barman S. Ghosh S. ACS Chem. Neurosci. 2024;15(13):2470–2483. doi: 10.1021/acschemneuro.3c00777.</cite> [<a href="https://doi.org/10.1021/acschemneuro.3c00777" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38874606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Chem.%20Neurosci.&amp;author=P.%20K.%20Gharai&amp;author=J.%20Khan&amp;author=K.%20Pradhan&amp;author=R.%20Mallesh&amp;author=S.%20Garg&amp;volume=15&amp;issue=13&amp;publication_year=2024&amp;pages=2470-2483&amp;pmid=38874606&amp;doi=10.1021/acschemneuro.3c00777&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit34">
<cite>Gharai P. K. Khan J. Mallesh R. Garg S. Gupta S. Jaisankar P. Ghosh S. ACS Chem. Neurosci. 2025;16:2224–2236. doi: 10.1021/acschemneuro.5c00067.</cite> [<a href="https://doi.org/10.1021/acschemneuro.5c00067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40445128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Chem.%20Neurosci.&amp;author=P.%20K.%20Gharai&amp;author=J.%20Khan&amp;author=R.%20Mallesh&amp;author=S.%20Garg&amp;author=S.%20Gupta&amp;volume=16&amp;publication_year=2025&amp;pages=2224-2236&amp;pmid=40445128&amp;doi=10.1021/acschemneuro.5c00067&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit35">
<cite>Kurmi M. Sahu A. Balhara A. Singh I. P. Kulkarni S. Singh N. K. Garg P. Singh S. J. Pharm. Biomed. Anal. 2020;178:112911. doi: 10.1016/j.jpba.2019.112911.</cite> [<a href="https://doi.org/10.1016/j.jpba.2019.112911" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31627078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Biomed.%20Anal.&amp;author=M.%20Kurmi&amp;author=A.%20Sahu&amp;author=A.%20Balhara&amp;author=I.%20P.%20Singh&amp;author=S.%20Kulkarni&amp;volume=178&amp;publication_year=2020&amp;pages=112911&amp;pmid=31627078&amp;doi=10.1016/j.jpba.2019.112911&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>RA-015-D5RA03504J-s001</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12376879/bin/RA-015-D5RA03504J-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">RA-015-D5RA03504J-s001.pdf</a><sup> (4MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data supporting this article have been included as part of the SI. IR, HSQC, <sup>1</sup>H NMR, <sup>13</sup>C NMR, mass. See DOI: <a href="https://doi.org/10.1039/d5ra03504j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1039/d5ra03504j</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from RSC Advances are provided here courtesy of <strong>Royal Society of Chemistry</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1039/d5ra03504j"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/RA-015-D5RA03504J.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12376879/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12376879/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12376879%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376879/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12376879/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12376879/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12376879/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12376879/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12376879/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="iJKYeSNyVgfG9vKqDxUZLYlJYkM9D2LDG7AtysTq5rhFPnnNriIuax1Avy02Mj1O">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
