
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Study on obtaining bacterial cellulose by Komagataeibacter xylinus in co-culture with lactic acid bacteria in whey - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15253CDB8AF26303053CDB0008552B2D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="springeropen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370864/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Applied Microbiology and Biotechnology">
<meta name="citation_title" content="Study on obtaining bacterial cellulose by Komagataeibacter xylinus in co-culture with lactic acid bacteria in whey">
<meta name="citation_author" content="Justyna Płoska">
<meta name="citation_author_institution" content="Department of Food Technology and Assessment, Institute of Food Sciences, Warsaw University of Life Sciences - SGGW, Nowoursynowska Street 166, 02-787 Warsaw, Poland">
<meta name="citation_author" content="Monika Garbowska">
<meta name="citation_author_institution" content="Department of Food Technology and Assessment, Institute of Food Sciences, Warsaw University of Life Sciences - SGGW, Nowoursynowska Street 166, 02-787 Warsaw, Poland">
<meta name="citation_author" content="Iwona Ścibisz">
<meta name="citation_author_institution" content="Department of Food Technology and Assessment, Institute of Food Sciences, Warsaw University of Life Sciences - SGGW, Nowoursynowska Street 166, 02-787 Warsaw, Poland">
<meta name="citation_author" content="Lidia Stasiak-Różańska">
<meta name="citation_author_institution" content="Department of Food Technology and Assessment, Institute of Food Sciences, Warsaw University of Life Sciences - SGGW, Nowoursynowska Street 166, 02-787 Warsaw, Poland">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="109">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="191">
<meta name="citation_doi" content="10.1007/s00253-025-13582-3">
<meta name="citation_pmid" content="40839214">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370864/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370864/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370864/pdf/253_2025_Article_13582.pdf">
<meta name="description" content="The use of acid whey as a medium is an innovative approach to bacterial cellulose (BC) biosynthesis in co-cultures of acetic acid bacteria with lactic acid bacteria. The aim of this study was to evaluate the possibility of obtaining BC in acid whey ...">
<meta name="og:title" content="Study on obtaining bacterial cellulose by Komagataeibacter xylinus in co-culture with lactic acid bacteria in whey">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The use of acid whey as a medium is an innovative approach to bacterial cellulose (BC) biosynthesis in co-cultures of acetic acid bacteria with lactic acid bacteria. The aim of this study was to evaluate the possibility of obtaining BC in acid whey ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370864/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370864">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1007/s00253-025-13582-3"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/253_2025_Article_13582.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370864%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370864/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370864/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370864/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-springeropen.png" alt="Springer logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Springer" title="Link to Springer" shape="default" href="https://doi.org/10.1007/s00253-025-13582-3" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Appl Microbiol Biotechnol</button></div>. 2025 Aug 21;109(1):191. doi: <a href="https://doi.org/10.1007/s00253-025-13582-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1007/s00253-025-13582-3</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Appl%20Microbiol%20Biotechnol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Appl%20Microbiol%20Biotechnol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Appl%20Microbiol%20Biotechnol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Appl%20Microbiol%20Biotechnol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Study on obtaining bacterial cellulose by <em>Komagataeibacter xylinus</em> in co-culture with lactic acid bacteria in whey</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22P%C5%82oska%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Justyna Płoska</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Justyna Płoska</span></h3>
<div class="p">
<sup>1</sup>Department of Food Technology and Assessment, Institute of Food Sciences, Warsaw University of Life Sciences - SGGW, Nowoursynowska Street 166, 02-787 Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22P%C5%82oska%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Justyna Płoska</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garbowska%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Monika Garbowska</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Monika Garbowska</span></h3>
<div class="p">
<sup>1</sup>Department of Food Technology and Assessment, Institute of Food Sciences, Warsaw University of Life Sciences - SGGW, Nowoursynowska Street 166, 02-787 Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garbowska%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Monika Garbowska</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C5%9Acibisz%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Iwona Ścibisz</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Iwona Ścibisz</span></h3>
<div class="p">
<sup>1</sup>Department of Food Technology and Assessment, Institute of Food Sciences, Warsaw University of Life Sciences - SGGW, Nowoursynowska Street 166, 02-787 Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C5%9Acibisz%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Iwona Ścibisz</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stasiak-R%C3%B3%C5%BCa%C5%84ska%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Lidia Stasiak-Różańska</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Lidia Stasiak-Różańska</span></h3>
<div class="p">
<sup>1</sup>Department of Food Technology and Assessment, Institute of Food Sciences, Warsaw University of Life Sciences - SGGW, Nowoursynowska Street 166, 02-787 Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stasiak-R%C3%B3%C5%BCa%C5%84ska%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lidia Stasiak-Różańska</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Food Technology and Assessment, Institute of Food Sciences, Warsaw University of Life Sciences - SGGW, Nowoursynowska Street 166, 02-787 Warsaw, Poland </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 3; Revised 2025 Jul 30; Accepted 2025 Aug 10; Issue date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370864  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40839214/" class="usa-link">40839214</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Abstract</h3>
<p id="Par5">The use of acid whey as a medium is an innovative approach to bacterial cellulose (BC) biosynthesis in co-cultures of acetic acid bacteria with lactic acid bacteria. The aim of this study was to evaluate the possibility of obtaining BC in acid whey by co-culturing <em>K. xylinus</em> with selected strains of lactic acid bacteria and comparing the properties of this biopolymer with BC obtained in <em>K. xylinus</em> monoculture. The <em>K. xylinus</em> + <em>Lb. acidophilus</em> co-culture yielded 2.19 g·L<sup>−1</sup> of BC, which was 125% more than the <em>K. xylinus</em> monoculture. Additionally, <em>K. xylinus</em> in co-culture with <em>Lb. acidophilus</em> increased the degradation temperature of BC to 361 °C compared to 303 °C for BC obtained in monoculture. The BC obtained in the co-cultures showed better mechanical properties. BC obtained in co-culture with <em>Lb. delbrueckii</em> showed more than twice the Young’s modulus than BC from monoculture. Moreover, strain at break BC from co-culture with <em>Lb. acidophilus</em> and stress at break BC from co-culture with <em>Lb. helveticus</em> were 72% and 54% higher, respectively, than BC obtained from monoculture <em>K. xylinus.</em> In this study, it was shown that conducting acetic-lactic co-cultures increased the efficiency of BC biosynthesis and improved its properties. Moreover, this study has shown that acid whey is a sufficient and complete substrate for obtaining BC. Results presented in this paper indicate new possibilities for the management of this side product.</p></section><section id="sec2"><h3 class="pmc_sec_title">Key points</h3>
<p id="Par6">• <em>The K. xylinus + Lb. acidophilus co-culture produced 125% more cellulose than the monoculture.</em></p>
<p id="Par7">• <em>High lactic acid content and low pH of acid whey enhance cellulose biosynthesis.</em></p>
<p id="Par8">• <em>Acetic acid-lactic acid co-cultures improved the mechanical properties of cellulose.</em></p></section><section id="sec3"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1007/s00253-025-13582-3.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Bacterial cellulose, <em>Komagataeibacter xylinus</em>, Lactic acid bacteria, <em>Lactobacillus</em>, Lactic acid, Acid whey, Co-culture</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par9">Acid whey (AW) is a by-product of the dairy industry, obtained during acidic cheese production. It contains nutrients such as lactose, whey proteins, organic acids (lactic acid, citric acid), macro- and micronutrients, and B vitamins (B<sub>1</sub>, B<sub>2</sub>, B<sub>5</sub>) (Panesar et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2007</a>). Non-protein nitrogen compounds (urea, uric acid) and phosphorus compounds are also present in AW, making a risk of eutrophication if AW enters the environment. The presence of organic compounds makes AW a by-product difficult to manage and harmful for the natural environment (Kazimierowicz et al. <a href="#CR29" class="usa-link" aria-describedby="CR29">2022</a>). Therefore, it is important to find a solution to manage AW and reduce its adverse impact on the surrounding environment. In turn, the diversity of nutrients in AW means that this by-product might be a medium for the cultivation of microorganisms. The use of AW for the biosynthesis of new compounds fits into the framework of a circular economy (Lappa et al. <a href="#CR35" class="usa-link" aria-describedby="CR35">2021</a>). In addition, some microbial metabolites can add value and enrich food products, thanks to which they can be considered an element of fulfillment of Sustainable Development Goal 12 (Goal 12: Ensure sustainable consumption and production patterns). So far, AW has been used for the microbial biosynthesis of lactic acid (LA), bioethanol, fatty acids, and polyhydroxyalkanoates, among others (Murari et al. <a href="#CR46" class="usa-link" aria-describedby="CR46">2019</a>; Sampaio et al. <a href="#CR59" class="usa-link" aria-describedby="CR59">2020</a>; Lappa et al. <a href="#CR35" class="usa-link" aria-describedby="CR35">2021</a>; Bosco et al. <a href="#CR8" class="usa-link" aria-describedby="CR8">2021</a>; Uysal and Hamamcı <a href="#CR64" class="usa-link" aria-describedby="CR64">2021</a>).
</p>
<p id="Par10">An interesting biopolymer of microbial origin that can be made from AW is bacterial cellulose (BC). BC is a natural exopolysaccharide made of linear β−1,4-glucan chains linked by hydrogen bonds, van der Waals forces, and hydrophobic interactions (Płoska et al. <a href="#CR51" class="usa-link" aria-describedby="CR51">2023b</a>). Selected species of acetic acid bacteria (AAB), in particular <em>Komagataeibacter xylinus</em>, are considered to be the main producers of BC (Blanco Parte et al. <a href="#CR6" class="usa-link" aria-describedby="CR6">2020</a>). BC is characterized by biodegradability, high purity, high mechanical strength, liquid and gas permeability, ability to adsorb various compounds, and biocompatibility with living tissues (Faria et al. <a href="#CR19" class="usa-link" aria-describedby="CR19">2022</a>; Kumaravel et al. <a href="#CR34" class="usa-link" aria-describedby="CR34">2025</a>). In the food industry, BC can be used as a stabilizer (e.g., Pickering emulsion gels as fat replacers in ice cream), emulsifier (nanoemulsion of cinnamon oil), fat substitute (e.g., in ice cream), and in food packaging (e.g., bacterial cellulose/tomato puree edible films in multicomponent foods, BC-based material for cheese packaging) (Guo et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2018</a>; Razavi et al. <a href="#CR57" class="usa-link" aria-describedby="CR57">2020</a>; Freitas et al. <a href="#CR21" class="usa-link" aria-describedby="CR21">2022</a>; Gao et al. <a href="#CR22" class="usa-link" aria-describedby="CR22">2023</a>; Płoska et al. <a href="#CR52" class="usa-link" aria-describedby="CR52">2024</a>). Despite the unique properties of BC and a number of potential applications, this polymer is still not produced on a large scale. Obtaining BC on an industrial scale is expensive, mainly due to the high prices of media required for AAB growth. Therefore, it is reasonable to look for cheaper alternatives, e.g., replacing typical culture media with inexpensive industrial waste and by-products.
</p>
<p id="Par11">In our previous work, we obtained in AW approx. 3 g·L<sup>−1</sup> BC, which was approx. 2 g·L<sup>−1</sup> less compared to the classic medium (Płoska et al. <a href="#CR50" class="usa-link" aria-describedby="CR50">2023a</a>). The research was continued because an interesting dependency was observed. The highest BC mass was obtained in AW, in which the content of organic acids (expressed as LA) was the lowest after 14 days of culture. AW contains mainly LA and citric acid (CA) (Menchik et al. <a href="#CR44" class="usa-link" aria-describedby="CR44">2019</a>). The positive effect of LA on AAB growth has been documented several times (Matsuoka et al. <a href="#CR43" class="usa-link" aria-describedby="CR43">1996</a>; Cielecka et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2021a</a>; Płoska et al. <a href="#CR50" class="usa-link" aria-describedby="CR50">2023a</a>). According to these reports, LA acts as an accelerator of tricarboxylic acid (TCA) cycle, stimulating the growth of AAB cells, which in turn translates into increased efficiency of BC production. Moreover, it was found that LA, when converted into pyruvate, generates additional energy, which positively affects the BC biosynthesis (Cielecka et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2021a</a>). Lactic acid bacteria (LAB) are usually responsible for converting lactose to LA (Ayivi et al. <a href="#CR3" class="usa-link" aria-describedby="CR3">2020</a>). Co-cultivation of <em>K. xylinus</em> with LAB for BC biosynthesis may provide many benefits; for example, due to the diverse metabolic pathways of microorganisms, the product of the metabolism of one strain can also be a substrate in the metabolism of another (Antolak et al. <a href="#CR2" class="usa-link" aria-describedby="CR2">2021</a>). LAB assimilate lactose present in AW, which is difficult to absorb by most AAB, and convert it into LA. Another potential benefit of co-cultures may be the fact that some LAB produce exopolysaccharides (EPS) that can positively affect the chemical-physical and mechanical properties of BC (Liu and Catchmark <a href="#CR40" class="usa-link" aria-describedby="CR40">2019</a>). Recently, scientific reports have begun to appear raising the possibility of using co-cultures as a way to increase the efficiency of obtaining BC and/or improving its properties. For example, Brugnoli et al. (<a href="#CR9" class="usa-link" aria-describedby="CR9">2023a</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">b</a>) carried out a study where a co-culture system of two AAB strains and two LAB strains was developed to produce BC and hyaluronic acid composites. As a result, the co-cultures yielded more BC than the AAB monocultures, and the obtained composites had different properties with application potential in cosmetics and pharmaceuticals. However, there are still many unknowns in this field, and there is much to be explored. Thus, there is a need to fill the gap concerning, among other things, the selection of strains and composition of the medium (Huang et al. <a href="#CR27" class="usa-link" aria-describedby="CR27">2024</a>; Qiao et al. <a href="#CR54" class="usa-link" aria-describedby="CR54">2024</a>; Mollica et al. <a href="#CR45" class="usa-link" aria-describedby="CR45">2024</a>).</p>
<p id="Par12">The novel approach of this work is that instead of typical microbiological media (used in most of the works in this field, e.g., Hestrin-Schramm), we used only AW—a side product from the dairy industry as a medium for co-cultures and BC synthesis. Nowadays, AW is a particularly concerning environmental problem. The management of this side product is also the subject of many research. There is a lack of information in the scientific literature on the use of AW as a medium for BC production in acetic-lactic co-culture. Therefore, our research focuses on this aspect.</p>
<p id="Par13">In this study, we assumed that LA produced by LAB promotes the metabolic pathway of BC biosynthesis in AAB. Moreover, we hypothesized that co-culture of <em>K. xylinus</em> with LAB will increase BC efficiency because LAB, by breaking down lactose, will increase the availability of easily digestible simple sugar (glucose), and <em>K. xylinus</em> will be able to synthesize BC more efficiently. The use of LAB species can help explore the different interactions occurring between bacteria in co-culture, as well as the potential mechanisms of BC biosynthesis intensification and how this affects its properties. The aim of the research was to test the possibility of increasing the efficiency of BC production in acetic-lactic co-culture in AW. In addition, the impact of the type of culture (monoculture <em>vs</em> co-culture) on the chemical, thermal, and mechanical properties of BC was examined.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Microbial strains</h3>
<p id="Par14">In this study, one strain of AAB was used: <em>Komagataeibacter xylinus</em> UMCC 2756 (Unimore Microbial Culture Collection<em>,</em> University of Modena and Reggio Emilia, Italy) and three strains of LAB were used: <em>Lactobacillus acidophilus</em> ATCC 4356 (American Type Culture Collection, USA), <em>Lactobacillus helveticus</em> LhB01 (Chr. Hansen, Ltd., Czosnów, Poland), and <em>Lactobacillus delbrueckii</em> subsp. <em>lactis</em> ATCC 4797 (American Type Culture Collection, USA).</p></section><section id="Sec4"><h3 class="pmc_sec_title">Co-culture preparation</h3>
<p id="Par15">Preinoculum of <em>K. xylinus</em> was multiplied in YPM medium (composition: 5 g·L<sup>−1</sup> yeast extract, 3 g·L<sup>−1</sup> peptone, 25 g·L<sup>−1</sup> mannitol, pH 5.75) for 96 h at 30 °C with shaking (120 rpm), then seeded on YPM-agar slants and cultured for 96 h at 30 °C. Preinoculum of <em>Lb. acidophilus</em>, <em>Lb</em>. <em>delbrueckii</em>, and <em>Lb. helveticus</em> was multiplied in De Man, Rogosa, and Sharpe liquid medium (MRS, Merck, Poland) for 96 h at 30 °C under stationary conditions, then seeded on MRS slants and cultured for 96 h at 30 °C.</p>
<p id="Par16"><em>K. xylinus</em> inoculum was prepared by taking colonies from agar slants and suspending them in saline to obtain a turbidity comparable to McFarland standard turbidity of 2.0 (approximately 8.78 log CFU mL<sup>−1</sup>). LAB inoculum was prepared similarly. Acid whey (AW) (Laktopol Dairy Plant, Łosice, Poland), as a BC production medium, was reconstituted from powder: 70 g·L<sup>−1</sup>, pH 4.8, and then autoclaved at 118 °C for 15 min. Sterile AW was inoculated with 10% v/v inoculum (5% <em>K. xylinus</em> + 5% LAB strain) (Płoska et al. <a href="#CR52" class="usa-link" aria-describedby="CR52">2024</a>). The ratio of <em>K. xylinus</em> to LAB bacteria in co-culture was 1:1 (Jiang et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2023</a>).</p>
<p id="Par17">Three variants of acetic-lactic co-cultures were made:</p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par18"><em>K. xylinus</em> + <em>Lb. acidophilus</em> ATCC 4356</p></li>
<li><p id="Par19"><em>K. xylinus</em> + <em>Lb. delbrueckii</em> subsp. <em>lactis</em> ATCC 4797</p></li>
<li><p id="Par20"><em>K. xylinus</em> + <em>Lb. helveticus</em> LhB01</p></li>
</ol>
<p id="Par21">In addition, single cultures (monocultures) of all LAB and <em>K. xylinus</em> strains were conducted in AW for 14 days at 30 °C.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Obtaining BC in mono- and co-cultures and preparing it for further analyses</h3>
<p id="Par22">During culture, a network of polymer fibers was formed on the surface of the AW observed as a shiny, flexible membrane. The amount of BC produced in <em>K. xylinus</em> monoculture and co-cultures was evaluated at the beginning of culture (T0) and after 3, 6, 9, 12, and 14 days. Whey pH was measured at the same time points. BC obtained in <em>K. xylinus</em> monoculture and all co-culture variants was purified from bacterial cells and residual medium by incubation in 0.1 M NaOH at 95 °C for 5 h. BC was neutralized with 6% acetic acid, rinsed with distilled water, and sterilized at 121 °C for 20 min. Next, all BC sheets were placed on the sterile paper and dried at 45 °C for around 5 h ± 30 min until to establish a constant mass (Płoska et al. <a href="#CR50" class="usa-link" aria-describedby="CR50">2023a</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Determination of sugars and organic acids</h3>
<p id="Par23">Lactose, galactose, and glucose, as well as acetic acid (AA), CA, gluconic acid (GA), and LA, were determined in the AW by high-performance liquid chromatography (HPLC). Samples were centrifuged at 10,000 rpm for 10 min (MPW-350R; MPW Med. Instruments, Warsaw, Poland) and the supernatant was collected. The appropriately diluted supernatant was filtered through a 0.45-μm nylon syringe filter. The analysis of sugars in media followed a modified procedure based on the method described by Usenik et al. (<a href="#CR63" class="usa-link" aria-describedby="CR63">2008</a>) and Elghali et al. (<a href="#CR16" class="usa-link" aria-describedby="CR16">2005</a>). Chromatographic analyses were carried out using a Shimadzu Prominence HPLC system (Kyoto, Japan), comprising an LC-20AD pump, autosampler (SIL-20A HT), refractive index detector (RID-10A), column oven (CTO-10ASVP), degasser (DGU-20A5R), and LABSolution data collection software. Separation was executed on a RezexTM RCM-Monosaccharide Ca<sup>+</sup> column (300 × 7.8 mm) (Phenomenex, USA) at 65 °C. The mobile phase used was Milli-Q water eluted with a flow rate of 0.7 mL·min<sup>−1</sup>.</p>
<p id="Par24">The analysis of organic acid in media was performed according to the procedure described by Fernandez-Garcia and McGregor (<a href="#CR20" class="usa-link" aria-describedby="CR20">1994</a>) with minor modifications. Organic acid analysis was performed using the aforementioned Shimadzu HPLC system, equipped with a diode array detector (SPD M20A). Cosmosil 5C18-PAQ (4.6 mm × 150 mm) column (Waters, Etten-Leur, The Netherlands) was used for the separation of organic acid. The column temperature was maintained at 20 °C, and the mobile phase consisted of 20 mmol phosphoric acid with a flow rate of 1.0 mL·min<sup>−1</sup>. A 20 µL injection volume was used, and organic acid peaks were detected at 210 nm. The organic acids were identified by comparing their retention times and UV–Vis spectral characteristics to those of commercial standards. Lactose, glucose, galactose, and GA were obtained from Sigma-Aldrich with purities of 97–99%. CA, AA, and L(+)-LA were purchased from Merck KGAA (Darmstadt, Germany).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Assessment of exopolysaccharides production capacity</h3>
<p id="Par25">The ability of LAB to produce EPS was evaluated in MRS medium and in AW. Inoculum was prepared by suspending bacterial colonies in saline to obtain a turbidity comparable to McFarland standard turbidity of 2.0 (approximately 8.78 log CFU mL<sup>−1</sup>). In co-cultures, the ratio of AAB to LAB cells was 1:1. The analysis of EPS production was performed according to the procedure described by Liu and Catchmark (<a href="#CR40" class="usa-link" aria-describedby="CR40">2019</a>) and Ma’unatin et al. (<a href="#CR42" class="usa-link" aria-describedby="CR42">2020</a>) with minor modification. After finishing the cultivation, the medium was heated to 100 °C for 15 min to denature EPS-degrading enzymes. After cooling, the samples were centrifuged (10,000 rpm, 4 °C, 15 min). Then, EPS present in the supernatants was precipitated by adding 3 volumes of cold ethanol (96%) and left overnight at 4 °C, then centrifuged again. The resulting EPS precipitate was freeze-dried and weighed on an analytical balance (Liu and Catchmark <a href="#CR40" class="usa-link" aria-describedby="CR40">2019</a>; Ma’unatin et al. <a href="#CR42" class="usa-link" aria-describedby="CR42">2020</a>).</p></section><section id="Sec8"><h3 class="pmc_sec_title">Characterization of bacterial cellulose</h3>
<section id="Sec9"><h4 class="pmc_sec_title">Scanning electron microscope analysis</h4>
<p id="Par26">BC surface morphology and cross section were analyzed using the Jeol ISM-IT500HR INTOUCHSCOPE (JEOL Europe SAS) scanning electron microscope. BC films were attached to carbon tape and coated with gold. The surface structure was analyzed at a magnification of × 15,000 and the cross section was analyzed at a magnification of × 3000– × 3500 at an accelerating voltage of 15 kV and 5 kV for the surface and cross section, respectively. The average diameter of BC fibers was measured using ImageJ software (National Institute of Health, Bethesda, MD, USA) on 40 randomly selected fibers on SEM images.</p></section></section><section id="Sec10"><h3 class="pmc_sec_title">Attenuated total reflectance-Fourier transform infrared characterization</h3>
<p id="Par27">ATR-FTIR spectra were acquired by means of a Nicolet iS20 spectrophotometer (Thermo Scientific, Waltham, MA, USA). The spectra were collected in the range of 600–4000 cm<sup>−1</sup>, at a resolution of 4 cm<sup>−1</sup>, 32 scans. The ATR-FTIR air background spectrum was collected every 30 min. Spectra were normalized to 1.0 at 1030 cm<sup>−1</sup> (C-O–H stretching vibration). Baseline corrections were obtained on Omnic Software (Thermo Scientific).</p></section><section id="Sec11"><h3 class="pmc_sec_title">Thermogravimetric analysis</h3>
<p id="Par28">To study the thermal properties of BC, thermogravimetric analysis (TGA) and derivative thermogravimetric analysis (DTG) were performed on a Mettler Toledo™ TGA/DSC1 apparatus. The analysis was carried out from room temperature (25 °C) to 600 °C, with a heating rate of 10 °C·min<sup>−1</sup> and in a nitrogen atmosphere with a flow rate of 50 ml·min<sup>−1</sup>.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Mechanical properties</h3>
<p id="Par29">The tensile properties of BC were performed using Instron 3365 (Instron, USA) machine at room temperature (25 °C). Specimens of 30 mm length and 5 mm width were stretched at a rate of 1.5 mm·min<sup>−1</sup> until fracture. The thickness of the nanocomposites was measured using an electronic MI21 micrometer (MTS, Slovakia). Strain at break (ε) was calculated as ΔL/L<sub>0</sub> where L<sub>0</sub> is the initial length of a sample between clamps and ΔL is the increased length from L0. Stress at break (δ) was calculated by F/A, where F is the force applied in Newtons and A is the area of the cross-section of the sample. The apparent Young’s modulus (E) was defined by the slope of the linear region of the strain–stress curve during the stretching stage. All measurements were performed in ten repeats, and the average value was recorded.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Statistical analyses</h3>
<p id="Par30">All experiments were conducted in triplicate (unless otherwise noted) and results are presented as mean ± standard deviation (SD). One-way analysis of variance (ANOVA) was performed using Statistica 13.1 software (Dell Sp. z o.o., Warsaw, Poland). The Tukey test was applied to compare the significance of differences between mean values at a significance level of <em>α</em> = 0.05.</p></section></section><section id="Sec14"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec15"><h3 class="pmc_sec_title">The efficiency of BC biosynthesis in mono- and co-cultures</h3>
<p id="Par31">The dry weight of BC obtained in the <em>K. xylinus</em> monoculture and co-cultures is shown in Fig. <a href="#Fig1" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370864_253_2025_13582_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b56f/12370864/059892898fa0/253_2025_13582_Fig1_HTML.jpg" loading="lazy" id="MO1" height="459" width="751" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>BC dry matter obtained during monoculture of <em>K. xylinus</em> and acetic-lactic co-cultures; <sup>a–d</sup>Means marked with the same lowercase letters are not statistically significantly different at α = 0.05 within time. <sup>a–e</sup>Means marked with the same uppercase letters are not statistically significantly different at α = 0.05 within one culturing variant conducted during 14 days</p></figcaption></figure><p id="Par32">BC was observed after just 1 day of culture in the co-cultures, whereas in the monoculture <em>K. xylinus</em>, BC appeared only after 3 days. Over time, the BC mass increased in all cultures, except for the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture. Statistically significant differences (<em>p</em> &lt; 0.05) in BC dry matter (dm) were observed even after 3 and 6 days, with the highest BC production occurring in the <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture (0.35 g·L<sup>−1</sup> and 0.67 g·L<sup>−1</sup>, respectively). After 12 days of cultivation, the dm of BC in the <em>K. xylinus</em> + <em>Lb. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-cultures was comparable to the mass of BC from the <em>K. xylinus</em> monoculture and amounted to 0.95 g·L<sup>−1</sup>, 0.91 g·L<sup>−1</sup>, and 0.82 g·L<sup>−1</sup>, respectively. However, after 14 days, the highest BC yield was obtained in the <em>K. xylinus</em> + <em>Lb. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-cultures (2.19 g·L<sup>−1</sup> and 1.5 g·L<sup>−1</sup>, respectively) compared to the monoculture, where approximately 1 g·L<sup>−1</sup> was produced. This constituted 125% and 54% more BC, respectively, than in the <em>K. xylinus</em> monoculture. The increase in the BC mass in two of the three studied acetic-lactic co-cultures confirms the conclusions of other researchers that different strain combinations in co-cultures have varying effects on the BC biosynthesis (Jiang et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2023</a>).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Changes in sugar content in AW in mono- and co-cultures</h3>
<p id="Par33">Chromatographic analysis showed no glucose in the whey during the 14 days of culture. The content of lactose and galactose in AW with the different bacterial variants is shown in Fig. <a href="#Fig2" class="usa-link">2</a>.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370864_253_2025_13582_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b56f/12370864/1e6245e8fa11/253_2025_13582_Fig2_HTML.jpg" loading="lazy" id="MO2" height="711" width="678" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Lactose (<strong>A</strong>, <strong>C</strong>, <strong>E</strong>) and galactose (<strong>B</strong>, <strong>D</strong>, <strong>F</strong>) content in <em>K. xylinus</em> monoculture and co-cultures</p></figcaption></figure><p id="Par34">The lactose content in AW before the start of the cultivation (T<sub>0</sub>) was similar across most variants amounting to 4.2 g·100 mL<sup>−1</sup>. A similar lactose content in the whey medium was noted in previous studies by Revin et al. (<a href="#CR58" class="usa-link" aria-describedby="CR58">2018</a>). The lactose content in the medium decreased over time. The largest utilization of lactose after 14 days of cultivation occurred in the <em>K. xylinus</em> + <em>Lb. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-cultures (2.2 g·100 mL<sup>−1</sup> and 2.1 g·100 mL<sup>−1</sup>, respectively) (Fig. <a href="#Fig2" class="usa-link">2</a>A and <a href="#Fig2" class="usa-link">E</a>). This was most likely due to the highest BC biosynthesis in these co-cultures (Fig. <a href="#Fig1" class="usa-link">1</a>), high LA production (Table <a href="#Tab2" class="usa-link">2</a>), and the consumption of a carbon source to ensure the basic metabolic functions of the cells. In the <em>K. xylinus</em> monoculture, lactose consumption was 0.9 g·100 mL<sup>−1</sup>, which was reflected in the dm of BC, which after 14 days of cultivation was significantly (<em>p</em> &lt; 0.05) lower than in the <em>K. xylinus</em> + <em>Lb. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-cultures (Fig. <a href="#Fig1" class="usa-link">1</a>)<em>.</em></p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Organic acid content during BC biosynthesis in monocultures and co-cultures (g·100 mL<sup>−1</sup>)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Strain</th>
<th align="left" rowspan="2" colspan="1">Organic acid</th>
<th align="left" colspan="6" rowspan="1">Time (d)</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">0</th>
<th align="left" colspan="1" rowspan="1">3</th>
<th align="left" colspan="1" rowspan="1">6</th>
<th align="left" colspan="1" rowspan="1">9</th>
<th align="left" colspan="1" rowspan="1">12</th>
<th align="left" colspan="1" rowspan="1">14</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="2" rowspan="1"></td>
<td align="left" colspan="6" rowspan="1">(g·100 mL<sup>−1</sup>)</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1"><em>K. xylinus</em></td>
<td align="left" colspan="1" rowspan="1">AA</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CA</td>
<td align="center" colspan="1" rowspan="1">0.07 ± 0.017</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.007</td>
<td align="center" colspan="1" rowspan="1">0.04 ± 0.031</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.009</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.003</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.010</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GA</td>
<td align="center" colspan="1" rowspan="1">0.08 ± 0.029</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.009</td>
<td align="center" colspan="1" rowspan="1">0.03 ± 0.019</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.026</td>
<td align="center" colspan="1" rowspan="1">0.03 ± 0.024</td>
<td align="center" colspan="1" rowspan="1">0.07 ± 0.039</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LA</td>
<td align="center" colspan="1" rowspan="1">0.53 ± 0.100</td>
<td align="center" colspan="1" rowspan="1">0.44 ± 0.051</td>
<td align="center" colspan="1" rowspan="1">0.64 ± 0.029</td>
<td align="center" colspan="1" rowspan="1">0.78 ± 0.083</td>
<td align="center" colspan="1" rowspan="1">0.89 ± 0.129</td>
<td align="center" colspan="1" rowspan="1">1.04 ± 0.039</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1"><em>Lb. acidophilus</em></td>
<td align="left" colspan="1" rowspan="1">AA</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CA</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.009</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.005</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.011</td>
<td align="center" colspan="1" rowspan="1">0.04 ± 0.003</td>
<td align="center" colspan="1" rowspan="1">0.04 ± 0.014</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.007</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GA</td>
<td align="center" colspan="1" rowspan="1">0.07 ± 0.009</td>
<td align="center" colspan="1" rowspan="1">0.03 ± 0.018</td>
<td align="center" colspan="1" rowspan="1">0.04 ± 0.020</td>
<td align="center" colspan="1" rowspan="1">0.03 ± 0.018</td>
<td align="center" colspan="1" rowspan="1">0.03 ± 0.017</td>
<td align="center" colspan="1" rowspan="1">0.04 ± 0.023</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LA</td>
<td align="center" colspan="1" rowspan="1">0.50 ± 0.057</td>
<td align="center" colspan="1" rowspan="1">0.61 ± 0.097</td>
<td align="center" colspan="1" rowspan="1">0.91 ± 0.146</td>
<td align="center" colspan="1" rowspan="1">0.88 ± 0.045</td>
<td align="center" colspan="1" rowspan="1">1.07 ± 0.218</td>
<td align="center" colspan="1" rowspan="1">1.25 ± 0.159</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1"><em>Lb. delbrueckii</em></td>
<td align="left" colspan="1" rowspan="1">AA</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CA</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.008</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.003</td>
<td align="center" colspan="1" rowspan="1">0.08 ± 0.006</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.008</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.005</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.013</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GA</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.025</td>
<td align="center" colspan="1" rowspan="1">0.03 ± 0.005</td>
<td align="center" colspan="1" rowspan="1">0.03 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.03 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.03 ± 0.001</td>
<td align="center" colspan="1" rowspan="1">0.03 ± 0.008</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LA</td>
<td align="center" colspan="1" rowspan="1">0.50 ± 0.039</td>
<td align="center" colspan="1" rowspan="1">0.55 ± 0.067</td>
<td align="center" colspan="1" rowspan="1">0.74 ± 0.051</td>
<td align="center" colspan="1" rowspan="1">0.54 ± 0.057</td>
<td align="center" colspan="1" rowspan="1">0.60 ± 0.036</td>
<td align="center" colspan="1" rowspan="1">0.52 ± 0.107</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1"><em>Lb. helveticus</em></td>
<td align="left" colspan="1" rowspan="1">AA</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CA</td>
<td align="center" colspan="1" rowspan="1">0.08 ± 0.018</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.003</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.003</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.010</td>
<td align="center" colspan="1" rowspan="1">0.07 ± 0.017</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.004</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GA</td>
<td align="center" colspan="1" rowspan="1">0.08 ± 0.011</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.007</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.014</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.009</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.013</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.004</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LA</td>
<td align="center" colspan="1" rowspan="1">0.47 ± 0.072</td>
<td align="center" colspan="1" rowspan="1">0.55 ± 0.069</td>
<td align="center" colspan="1" rowspan="1">0.71 ± 0.030</td>
<td align="center" colspan="1" rowspan="1">0.84 ± 0.119</td>
<td align="center" colspan="1" rowspan="1">1.02 ± 0.134</td>
<td align="center" colspan="1" rowspan="1">0.94 ± 0.066</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1">
<em>K. xylinus</em> + <em>Lb. acidophilus</em>
</td>
<td align="left" colspan="1" rowspan="1">AA</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CA</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.009</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.014</td>
<td align="center" colspan="1" rowspan="1">0.04 ± 0.012</td>
<td align="center" colspan="1" rowspan="1">0.04 ± 0.009</td>
<td align="center" colspan="1" rowspan="1">0.04 ± 0.008</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GA</td>
<td align="center" colspan="1" rowspan="1">0.07 ± 0.012</td>
<td align="center" colspan="1" rowspan="1">0.03 ± 0.004</td>
<td align="center" colspan="1" rowspan="1">0.04 ± 0.008</td>
<td align="center" colspan="1" rowspan="1">0.04 ± 0.016</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.015</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.006</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LA</td>
<td align="center" colspan="1" rowspan="1">0.47 ± 0.017</td>
<td align="center" colspan="1" rowspan="1">0.69 ± 0.114</td>
<td align="center" colspan="1" rowspan="1">0.99 ± 0.191</td>
<td align="center" colspan="1" rowspan="1">0.96 ± 0.115</td>
<td align="center" colspan="1" rowspan="1">1.06 ± 0.188</td>
<td align="center" colspan="1" rowspan="1">0.85 ± 0.142</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>K. xylinus</em> + <em>Lb. delbrueckii</em>
</td>
<td align="left" colspan="1" rowspan="1">AA</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.024</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.006</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.002</td>
<td align="center" colspan="1" rowspan="1">0.07 ± 0.001</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1"></td>
<td align="left" colspan="1" rowspan="1">CA</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.003</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.009</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GA</td>
<td align="center" colspan="1" rowspan="1">0.08 ± 0.006</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.044</td>
<td align="center" colspan="1" rowspan="1">0.04 ± 0.009</td>
<td align="center" colspan="1" rowspan="1">0.04 ± 0.003</td>
<td align="center" colspan="1" rowspan="1">0.04 ± 0.008</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.016</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LA</td>
<td align="center" colspan="1" rowspan="1">0.49 ± 0.010</td>
<td align="center" colspan="1" rowspan="1">0.52 ± 0.071</td>
<td align="center" colspan="1" rowspan="1">0.41 ± 0.048</td>
<td align="center" colspan="1" rowspan="1">0.31 ± 0.049</td>
<td align="center" colspan="1" rowspan="1">0.30 ± 0.035</td>
<td align="center" colspan="1" rowspan="1">0.28 ± 0.038</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1">
<em>K. xylinus</em> + <em>Lb. helveticus</em>
</td>
<td align="left" colspan="1" rowspan="1">AA</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.00 ± 0.000</td>
<td align="center" colspan="1" rowspan="1">0.02 ± 0.004</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CA</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.005</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.002</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.037</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.014</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.013</td>
<td align="center" colspan="1" rowspan="1">0.03 ± 0.012</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GA</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.007</td>
<td align="center" colspan="1" rowspan="1">0.09 ± 0.012</td>
<td align="center" colspan="1" rowspan="1">0.08 ± 0.025</td>
<td align="center" colspan="1" rowspan="1">0.12 ± 0.051</td>
<td align="center" colspan="1" rowspan="1">0.08 ± 0.021</td>
<td align="center" colspan="1" rowspan="1">0.08 ± 0.017</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LA</td>
<td align="center" colspan="1" rowspan="1">0.45 ± 0.046</td>
<td align="center" colspan="1" rowspan="1">0.60 ± 0.069</td>
<td align="center" colspan="1" rowspan="1">0.93 ± 0.140</td>
<td align="center" colspan="1" rowspan="1">1.21 ± 0.126</td>
<td align="center" colspan="1" rowspan="1">1.36 ± 0.139</td>
<td align="center" colspan="1" rowspan="1">1.47 ± 0.155</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p39"><p><em>AA</em> acetic acid, <em>CA</em> citric acid, <em>GA</em> gluconic acid, <em>LA</em> lactic acid, mean ± standard deviation.</p></div></div></section><section id="Sec17"><h4 class="pmc_sec_title">Changes in organic acid content in AW in mono- and co-cultures</h4>
<p id="Par35">Table <a href="#Tab1" class="usa-link">1</a> shows the pH of whey during monoculture and co-culture.
</p>
<section class="tw xbox font-sm" id="Tab1"><h5 class="obj_head">Table 1.</h5>
<div class="caption p"><p>Changes of pH of media during BC biosynthesis in monocultures and co-cultures</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Strains</th>
<th align="left" colspan="6" rowspan="1">Time (d)</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">0</th>
<th align="left" colspan="1" rowspan="1">3</th>
<th align="left" colspan="1" rowspan="1">6</th>
<th align="left" colspan="1" rowspan="1">9</th>
<th align="left" colspan="1" rowspan="1">12</th>
<th align="left" colspan="1" rowspan="1">14</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1"><em>K. xylinus</em></td>
<td align="left" colspan="1" rowspan="1">4.80 ± 0.02<sup>c</sup>
</td>
<td align="left" colspan="1" rowspan="1">5.28 ± 0.06<sup>d</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.26 ± 0.04<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.06 ± 0.14<sup>ab</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.90 ± 0.16<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.84 ± 0.19<sup>a</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Lb. acidophilus</em></td>
<td align="left" colspan="1" rowspan="1">4.82 ± 0.01<sup>e</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.51 ± 0.06<sup>d</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.09 ± 0.02<sup>c</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.90 ± 0.01<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.75 ± 0.02<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.73 ± 0.01<sup>a</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Lb. delbrueckii</em></td>
<td align="left" colspan="1" rowspan="1">4.82 ± 0.01<sup>c</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.65 ± 0.03<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.53 ± 0.03<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.54 ± 0.01<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.57 ± 0.02<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.58 ± 0.03<sup>a</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>Lb. helveticus</em></td>
<td align="left" colspan="1" rowspan="1">4.82 ± 0.01<sup>f</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.43 ± 0.01<sup>e</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.11 ± 0.01<sup>d</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.04 ± 0.02<sup>c</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.91 ± 0.03<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.85 ± 0.02<sup>a</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>K. xylinus</em> + <em>Lb. acidophilus</em>
</td>
<td align="left" colspan="1" rowspan="1">4.80 ± 0.01<sup>c</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.21 ± 0.03<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.95 ± 0.10<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.81 ± 0.12<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.81 ± 0.08<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.83 ± 0.02<sup>a</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>K. xylinus</em> + <em>Lb. delbrueckii</em>
</td>
<td align="left" colspan="1" rowspan="1">4.79 ± 0.02<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.96 ± 0.01<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">5.85 ± 0.08<sup>c</sup>
</td>
<td align="left" colspan="1" rowspan="1">5.92 ± 0.03<sup>c</sup>
</td>
<td align="left" colspan="1" rowspan="1">5.95 ± 0.01<sup>c</sup>
</td>
<td align="left" colspan="1" rowspan="1">5.96 ± 0.02<sup>c</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>K. xylinus</em> + <em>Lb. helveticus</em>
</td>
<td align="left" colspan="1" rowspan="1">4.82 ± 0.06<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.50 ± 0.10<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.96 ± 0.11<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.77 ± 0.22<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.71 ± 0.11<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.59 ± 0.16<sup>a</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p36"><p><sup>a–f</sup>Means marked with the same superscript letters in the same row do not differ statistically significantly at <em>α</em> = 0.05</p></div></div></section><p id="Par36">At the beginning of bacterial growth (T<sub>0</sub>), the pH of the monocultures and acetic-lactic co-cultures media were comparable, averaging 4.82. During the culture incubation, the progressive fermentation and BC biosynthesis led to changes in the active acidity of the media. In the <em>K. xylinus</em> monoculture, the pH increased after 3 days of cultivation to 5.28, and then systematically decreased to 3.84 after 14 days of cultivation. Such pH changes could be due to the assimilation of LA contained in AW at the start of culture (Table <a href="#Tab2" class="usa-link">2</a>, LA concentration at T0 = 0.53 g·100 mL<sup>−1</sup>, LA concentration at T3 = 0.44 g·100 mL<sup>−1</sup>), which resulted in an increase in pH (day 3) followed by the production of acidic, pH-lowering metabolites in the subsequent days of culture. In the case of the <em>K. xylinus</em> + <em>Lb. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-cultures, a systematic pH decrease was observed during the culture, from pH 4.80 and pH 4.82 to pH 3.83 and pH 3.59, respectively. This progressive acidification of the medium occurred over a 14-day cultivation period and was accompanied by the highest BC production observed in this study (Fig. <a href="#Fig1" class="usa-link">1</a>). We assume that the ability of <em>Komagataeibacter</em> spp. and <em>Lactobacillus</em> spp. to tolerate low pH conditions enabled the cultures to proceed efficiently and produced BC even as the pH approached 3.6. To sum up, as the cultivation progressed, the acidity of the medium increased, while the yield of BC also increased. The increase in acidity in these co-cultures was likely caused by the increased production of LA by <em>Lb. acidophilus</em> and <em>Lb. helveticus</em> (Table <a href="#Tab2" class="usa-link">2</a>). This is supported by the significant decrease in the pH of whey in the monocultures of these bacteria. It was also observed that in the co-cultures, the pH stabilized and did not change significantly after 6 days of cultivation. In the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture, an increase in pH was observed during cultivation. After 3 days, the pH was higher than at the beginning of cultivation, and after 9 days, the pH reached 5.92 and remained at a similar level until the end of incubation. One possible explanation for this effect is the slight decrease in pH from 4.82 to 4.58 observed in the <em>Lb. delbrueckii</em> monoculture, suggesting a lower capacity for LA production by <em>Lb. delbrueckii</em> compared to other LAB (Table <a href="#Tab2" class="usa-link">2</a>) (Giraffa et al. <a href="#CR23" class="usa-link" aria-describedby="CR23">2004</a>). The content of individual organic acids in AW during monoculture and co-culture is shown in Table <a href="#Tab2" class="usa-link">2</a>.
</p>
<p id="Par37">AAB produce acetic acid through a two-step oxidation reaction of ethanol to acetaldehyde, which is then converted to acetic acid (He et al. <a href="#CR26" class="usa-link" aria-describedby="CR26">2022</a>). Since there is no ethanol in AW, AA was not detected in <em>K. xylinus</em> monoculture or the <em>K. xylinus</em> + <em>Lb. acidophilus</em> co-culture as expected. However, in the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture, AA was observed after 6 days of culture, and its concentration remained constant at 0.06–0.07 g·100 mL<sup>−1</sup>. In the <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture, AA was observed after 14 days of culture at a concentration of 0.02 g·100 mL<sup>−1</sup>. The production of AA in this case can be linked to citrate metabolism carried out by LAB. After 14 days of culture, the CA content decreased from 0.06 g·100 mL<sup>−1</sup> to 0.00 g·100 mL<sup>−1</sup> and 0.03 g·100 mL<sup>−1</sup> for the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-cultures, respectively (Table <a href="#Tab2" class="usa-link">2</a>). CA naturally occurs in AW; therefore, in our study, its content did not change significantly in all monocultures and the <em>K. xylinus</em> + <em>Lb. acidophilus</em> co-culture, remaining at approximately 0.05–0.07 g·100 mL<sup>−1</sup>.</p>
<p id="Par38">The content of GA in all mono- and co-cultures at the start of culture ranged from 0.05 to 0.08 g·100 mL<sup>−1</sup>. The presence of GA in AW was likely due to the addition of calcium gluconate to milk during the production of acidic cheeses, with its residues entering the AW (Pawlos et al. <a href="#CR49" class="usa-link" aria-describedby="CR49">2023</a>). The initial LA content in all cultures ranged from 0.45 g·100 mL<sup>−1</sup> to 0.53 g·100 mL<sup>−1</sup>. After 14 days, the lowest LA content was observed in the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture (0.28 g·100 mL<sup>−1</sup>), and the highest was in the <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture (1.47 g·100 mL<sup>−1</sup>)<em>.</em> In the <em>K. xylinus</em> monoculture, after 3 days of cultivation, the LA content in the medium decreased to 0.44 g·100 mL<sup>−1</sup>, which could be the result of LA uptake by bacteria in the early growth phase, as reported previously by Matsuoka et al. (<a href="#CR43" class="usa-link" aria-describedby="CR43">1996</a>). It is also worth noting that a 3-day delay in BC biosynthesis (compared to co-cultures) was observed in the <em>K. xylinus</em> monoculture, which may indicate a significant link between LA uptake, energy production, and BC biosynthesis. However, the LA content increased over time, and after 14 days, it amounted to 1.04 g·100 mL<sup>−1</sup>. An increase in LA content during BC biosynthesis in the <em>K. xylinus</em> monoculture was also observed in the study by Jiang et al. (2023), where 2.41 g·100 mL<sup>−1</sup> was obtained after 5 days of culture. This result was considered positive by the authors because the appropriate LA content may promote BC synthesis by accelerating the TCA cycle and gluconeogenesis, as well as directly affect the activity of enzymes involved in BC biosynthesis (Jiang et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2023</a>).</p></section><section id="Sec18"><h4 class="pmc_sec_title">Assessment of exopolysaccharides production capacity by LAB</h4>
<p id="Par39">The EPS production capacity of LAB was assessed in two media: MRS medium dedicated to the growth of lactobacilli and in AW. In both media, it was noted that the <em>Lb. delbrueckii</em> strain produced the most EPS, while the <em>Lb. acidophilus</em> and <em>Lb. helveticus</em> strains were characterized by similar EPS production. In the MRS medium, the <em>Lb. delbrueckii</em> strain produced five times more EPS than in AW (1.04 g·L<sup>−1</sup> and 0.24 g·L<sup>−1</sup>, respectively), while <em>Lb. acidophilus</em> and <em>Lb. helveticus</em> produced two times more EPS in the MRS medium (0.4 g·L<sup>−1</sup> and 0.48 g·L<sup>−1</sup>, respectively) than in AW (0.20 g·L<sup>−1</sup> and 0.21 g·L<sup>−1</sup>, respectively) (Table <a href="#Tab3" class="usa-link">3</a>). In co-cultures, the level of EPS produced by LAB was comparable to that from LAB monocultures (except for the co-culture with <em>Lb. acidophilus</em>). This may suggest that co-culture with <em>K. xylinus</em> had no effect on the EPS production capacity of LAB (except <em>Lb. acidophilus</em>). Statistically significantly (<em>p</em> &lt; 0.05), the highest yield of EPS was observed in the co-cultures of <em>K. xylinus</em> + <em>Lb. delbrueckii</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em>. These differences in EPS content in the medium correspond to the average fiber diameter (Fig. <a href="#Fig3" class="usa-link">3</a>) and the stress at break (Fig. <a href="#Fig6" class="usa-link">6</a>) of BC obtained in the corresponding co-cultures. However, it should be noted that all LAB strains have been proven to produce EPS, which could affect the biosynthesis of BC, its extracellular release, and the formation of 1,4-β-glucan chains (Liu and Catchmark <a href="#CR39" class="usa-link" aria-describedby="CR39">2018</a>).
</p>
<section class="tw xbox font-sm" id="Tab3"><h5 class="obj_head">Table 3.</h5>
<div class="caption p"><p>Exopolysaccharides produced by LAB in monocultures and co-cultures (g·L<sup>−1</sup>)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Strain</th>
<th align="left" colspan="1" rowspan="1">MRS medium (g·L<sup>−1</sup>)</th>
<th align="left" colspan="1" rowspan="1">AW medium (g·L<sup>−1</sup>)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Lb. acidophillus</em> ATCC 4356</td>
<td align="left" colspan="1" rowspan="1">0.40 ± 0.11<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.20 ± 0.01<sup>a</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Lb. delbrueckii</em> subs. <em>lactis</em> ATCC 4797</td>
<td align="left" colspan="1" rowspan="1">1.04 ± 0.20<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.24 ± 0.01<sup>a</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Lb. helveticus</em> LhB01</td>
<td align="left" colspan="1" rowspan="1">0.48 ± 0.18<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.21 ± 0.08<sup>a</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>K. xylinus</em> + <em>Lb. acidophilus</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">0.09 ± 0.02<sup>a</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>K. xylinus</em> + <em>Lb. delbrueckii</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">0.21 ± 0.02<sup>b</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>K. xylinus</em> + <em>Lb. helveticus</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">0.17 ± 0.03<sup>b</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p44"><p><sup>a–b</sup>Means marked with the same superscript letters in the same column are not statistically significantly different at <em>α</em> = 0.05</p></div></div></section><figure class="fig xbox font-sm" id="Fig3"><h5 class="obj_head">Fig. 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370864_253_2025_13582_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b56f/12370864/36415e3ae5b5/253_2025_13582_Fig3_HTML.jpg" loading="lazy" id="MO3" height="369" width="739" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SEM images of surface and cross section of BC samples from chosen mono- and co-cultures. <sup>a–c</sup>Means marked with the same superscript letters are not statistically significantly different at <em>α</em> = 0.05. High-resolution SEM images have been included in the <a href="#MOESM1" class="usa-link">supplementary materials</a></p></figcaption></figure><figure class="fig xbox font-sm" id="Fig6"><h5 class="obj_head">Fig. 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b56f/12370864/1d90767d0498/253_2025_13582_Fig6_HTML.jpg" loading="lazy" id="MO4" height="1143" width="644" alt="Fig. 6"></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mechanical properties of BC obtained in mono- and co-cultures, strain at break (<strong>A</strong>), stress at break (<strong>B</strong>), Young’s modulus (<strong>C</strong>); <sup>﻿a–c</sup>Means marked with the same superscript letters are not statistically significantly different at α = 0.05﻿</p></figcaption></figure></section><section id="Sec19"><h4 class="pmc_sec_title">Surface characteristics and fibre arrangement in BC</h4>
<p id="Par40">The surface morphology, average fiber diameters<strong>,</strong> and fiber structure of cellulose obtained in <em>K. xylinus</em> monoculture and co-cultures are shown in Fig. <a href="#Fig3" class="usa-link">3</a>.</p>
<p id="Par41">Based on the SEM images, it was observed that the BC surface from all culture variants showed a reticular structure consisting of densely spaced, ultra-thin cellulose fibrils, forming a three-dimensional web. BC from the <em>K. xylinus</em> monoculture and from the <em>K. xylinus</em> + <em>Lb. acidophilus</em> co-culture were characterized by similar surface morphology with a small number of compact spaces between the fibers. Numerous spaces were observed on the surface of BC from the <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture, which were a result of loosely arranged BC fibers. In other studies, it was proven that a higher concentration of ascorbic acid in the medium resulted in lower content and greater porosity of BC (Raiszadeh-Jahromi et al. <a href="#CR56" class="usa-link" aria-describedby="CR56">2020</a>). Raiszadeh-Jahromi et al. (<a href="#CR56" class="usa-link" aria-describedby="CR56">2020</a>) explained that organic acids can disturb the BC matrix, causing cracks and destructions in the homogeneity of its structure. In our study, the presence of LA and GA likely influenced the loosening of the BC fibre structure obtained in the <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture, as the medium from this co-culture was characterized by the highest content of these acids compared to other cultures (Table <a href="#Tab2" class="usa-link">2</a>). In the SEM images of the cross section of cellulose (Fig. <a href="#Fig3" class="usa-link">3</a>), it can be seen that the structure of BC from the <em>K. xylinus</em> monoculture was more compact, with an irregular arrangement of fibers and the space between them. In contrast, a layered arrangement of fibers was visible in the structure BC from co-cultures.</p>
<p id="Par42">The average diameters of the cellulose fibers obtained in these studies were similar to the diameter of BC fibers (from 35 to 55 nm) characterized by other authors (Wang et al. <a href="#CR65" class="usa-link" aria-describedby="CR65">2018</a>; Raiszadeh-Jahromi et al. <a href="#CR56" class="usa-link" aria-describedby="CR56">2020</a>; Brugnoli et al. <a href="#CR9" class="usa-link" aria-describedby="CR9">2023a</a>). The BC fibers from the <em>K. xylinus</em> monoculture (37.22 nm) were statistically significantly (<em>p</em> &lt; 0.05) thinner than BC fibers from <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture (43.75 nm) and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture (47.68 nm), which were connected and branched. No differences were observed in the diameter of BC fibers between BCs from <em>K. xylinus</em> monoculture and <em>K. xylinus</em> + <em>Lb. acidophilus</em> co-culture (34.86 nm), which were the thinnest. Statistically significantly, the thickest fiber diameter was characterized by BC from the <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture. Statistically significant differences in fiber diameter coincide with differences in stress at break (Fig. <a href="#Fig6" class="usa-link">6</a>B). A similar result was obtained in the study by Liu and Catchmark (<a href="#CR40" class="usa-link" aria-describedby="CR40">2019</a>), in which the BC fibers obtained in the <em>Ga. hansenii</em> co-culture from <em>E. coli</em> were thicker than those from the <em>Ga. hansenii</em> monoculture and were locally connected into larger ribbons. It was hypothesized that EPS produced by <em>E. coli</em> could coat the surface of the co-crystallized BC microfibrils, which promoted their merging.</p></section><section id="Sec20"><h4 class="pmc_sec_title">Chemical characterisation of bacterial cellulose</h4>
<p id="Par43">FTIR spectra of BC obtained in <em>K. xylinus</em> monoculture and co-cultures are shown in Fig. <a href="#Fig4" class="usa-link">4</a>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h5 class="obj_head">Fig. 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370864_253_2025_13582_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b56f/12370864/e5fc0bdd4cd1/253_2025_13582_Fig4_HTML.jpg" loading="lazy" id="MO5" height="390" width="678" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>FTIR spectra of BC obtained in mono- and co-cultures</p></figcaption></figure><p id="Par44">Characteristic peaks corresponding to the –OH stretching vibrations and H–O–H bending vibrations of absorbed water were observed around wavenumbers 3340 cm<sup>−1</sup> and 1650 cm<sup>−1</sup>, respectively (Fig. <a href="#Fig4" class="usa-link">4</a>). The peaks around 2890 cm<sup>−1</sup> were related to C–H stretching vibrations in the CH<sub>2</sub> and CH<sub>3</sub> groups (Jiang et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2023</a>). The signals at 1425 cm<sup>−1</sup>, 1160 cm<sup>−1</sup>, 1108 cm<sup>−1</sup>, 1055 cm<sup>−1</sup>, and 1030 cm<sup>−1</sup> were assigned to the CH<sub>2</sub> symmetric bending, C–O or C–C stretching vibrations, and C–O–C pyranose ring skeletal vibrations, respectively. Differences were observed in the intensities of some peaks, e.g., around wavenumber 1550 cm<sup>−1</sup>. A peak at ~ 1550 cm<sup>−1</sup> likely corresponded to carboxylate ions. In the spectra of BC from the <em>K. xylinus</em> monoculture and the <em>K. xylinus</em> + <em>Lb. acidophilus</em> co-culture, this peak was significantly higher than in the spectra of BC from the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-cultures.</p></section></section><section id="Sec21"><h3 class="pmc_sec_title">Thermal properties of BC</h3>
<p id="Par45">The TGA and DTG curves (Fig. <a href="#Fig5" class="usa-link">5</a>) showed differences in the thermal properties of BC produced in the <em>K. xylinus</em> monoculture and co-cultures. Three steps of BC degradation under the influence of temperature were identified. In the first phase (25–120 °C), there was a mass loss due to water evaporation. Up to 250 °C, the samples were thermally stable. The greatest degradation of BC was observed in the second stage (temp. 120 to 400 °C). Specifically, 55% of BC from the <em>K. xylinus</em> monoculture, 55% of BC from the <em>K. xylinus</em> + <em>Lb. acidophilus</em> co-culture, 67% of BC from the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture, and 49% of BC from the <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture were degraded at this stage. The decrease in the BC mass in this phase was caused by depolymerization and pyrolytic decomposition of glucose (Bagewadi et al. <a href="#CR4" class="usa-link" aria-describedby="CR4">2023</a>). Kumar et al. (<a href="#CR33" class="usa-link" aria-describedby="CR33">2021</a>) found that 43% of BC from AW degraded at 250–415 °C, which is similar to our results. After the third stage of degradation (temp. 600 °C), 66% of BC from the <em>K. xylinus</em> monoculture degraded. BC from the <em>K. xylinus</em> + <em>Lb. helveticus</em> and <em>K. xylinus</em> + <em>Lb. acidophilus</em> co-cultures degraded to a similar degree, with 61% and 62% of the compound broken down, respectively. In the case of the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture, 74% of BC degraded.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370864_253_2025_13582_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b56f/12370864/9323b39ff244/253_2025_13582_Fig5_HTML.jpg" loading="lazy" id="MO6" height="255" width="779" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>TGA (<strong>A</strong>) and dTG (<strong>B</strong>) curves of BC obtained in mono- and co-cultures</p></figcaption></figure><p id="Par46">The maximum degradation temperature determining the maximum rate of degradation temperature (MRDT) of BC from the <em>K. xylinus</em> monoculture was about 303 °C, from the <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture about 307 °C, from the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture about 342 °C, and from the <em>K. xylinus</em> + <em>Lb. acidophilus</em> co-culture about 361 °C (Fig. <a href="#Fig5" class="usa-link">5</a>B).</p></section><section id="Sec22"><h3 class="pmc_sec_title">Mechanical properties</h3>
<p id="Par47">In this study, the mechanical properties of BC obtained in monoculture and co-culture including strain at break, stress at break, and Young’s modulus were characterized (Fig. <a href="#Fig6" class="usa-link">6</a>). To validate the obtained results and minimize their randomness, the mechanical properties of BC were tested in ten repetitions. Although the standard deviations for the averages obtained are high, they still remain typical for the analysis of BC mechanical properties (Liu and Catchmark <a href="#CR40" class="usa-link" aria-describedby="CR40">2019</a>; Li et al. <a href="#CR36" class="usa-link" aria-describedby="CR36">2020</a>).</p>
<p id="Par48">This study showed that cellulose obtained from <em>K. xylinus</em> + <em>Lb. acidophilus</em> co-culture had the highest strain at break (Fig. <a href="#Fig6" class="usa-link">6</a>A), which was statistically significantly different (<em>p</em> &lt; 0.05) from BC obtained from <em>K. xylinus</em> monoculture and BC obtained from <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture. Stress at break BC obtained from <em>K. xylinus</em> + <em>Lb. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-cultures was similar to stress at break BC from <em>K. xylinus</em> monoculture. The highest value of this parameter was achieved by BC produced in the co-culture of <em>K. xylinus</em> + <em>Lb. helveticus</em>. BC from the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture had the highest Young’s modulus, which was 1.48 GPa and was statistically significantly higher than the other BC samples.</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par49">In this study, the most BC was produced in <em>K. xylinus</em> + <em>Lb. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-cultures. The sudden increase in BC weight between days 12 and 14 in the <em>K. xylinus</em> + <em>Lb. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-cultures may be attributed to increased lactose consumption at this time (Fig. <a href="#Fig2" class="usa-link">2</a>A and <a href="#Fig2" class="usa-link">E</a>). The stabilization of lactose levels in <em>Lb. acidophilus</em> and <em>Lb. helveticus</em> monocultures after 12 days of culture (Fig. <a href="#Fig2" class="usa-link">2</a>A and <a href="#Fig2" class="usa-link">E</a>) suggests that these bacteria no longer consume lactose from the medium and convert it to LA after this time. This is confirmed by the constant pH and LA content in these monocultures after 12 days (Tables <a href="#Tab1" class="usa-link">1</a> and <a href="#Tab2" class="usa-link">2</a>). In contrast, in the <em>K. xylinus</em> monoculture, lactose is still consumed between days 12 and 14 (Fig. <a href="#Fig2" class="usa-link">2</a>A, <a href="#Fig2" class="usa-link">C</a>, <a href="#Fig2" class="usa-link">E</a>). Thus, it can be assumed that <em>K. xylinus</em> is responsible for lactose consumption after 12 days in co-cultures of <em>K. xylinus</em> + L<em>b. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em>. Based on this, it can be hypothesized that <em>Lb. acidophilus</em> and <em>Lb. helveticus</em> in co-cultures consume lactose until day 12, after which, due to the disappearance of competition for the carbon source, <em>K. xylinus</em> becomes more active and utilizes it for increased BC production.</p>
<p id="Par50">The positive effect of co-culture on BC biosynthesis was noted in studies by Seto et al. (<a href="#CR60" class="usa-link" aria-describedby="CR60">2006</a>), in which <em>Gluconacetobacter xylinus</em> was cultivated with <em>Lactobacilus mali</em>. Specifically, 4.2 g·L<sup>−1</sup> of BC was obtained compared to 1.4 g·L<sup>−1</sup> of BC produced in a <em>Ga. xylinus</em> monoculture. As a possible explanation for this phenomenon, the authors suggested an increased activity of enzymes responsible for sucrose metabolism and EPS production by LAB, though this was not proved experimentally. Additionally, attention was drawn to the fact that it is not just EPS, but EPS-producing LAB that promote cellulose production, facilitating physical interaction between cells. Cell coaggregation may support the assembly of BC fibers into matrices and facilitate interactions between cells at the quorum sensing (QS) level (Seto et al. <a href="#CR60" class="usa-link" aria-describedby="CR60">2006</a>). QS is a phenomenon of chemical communication between bacterial cells, consisting of the synthesis and secretion of signaling molecules into the environment, which participate in regulating various physiological processes (Liu et al. <a href="#CR41" class="usa-link" aria-describedby="CR41">2018</a>). The effect of QS on increased BC biosynthesis may be associated with the transcriptional regulatory protein LuxR, the increased expression of which has been noted in the co-culture of <em>K. nataicola</em> Q2 and <em>Lb. fermentum</em> SR. QS can also regulate the expression of <em>bcs</em> genes (encoding cellulose synthase) by regulating the level of cyclic diguanylate, which affects the formation of β−1,4glucan chains and the crystallization of microfibrils (Liu et al. <a href="#CR41" class="usa-link" aria-describedby="CR41">2018</a>; Jiang et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2023</a>). The <em>K. xylinus</em> UMCC 2756 strain contains the <em>luxR</em> gene; however, further research is required to confirm this mechanism (Gullo et al. <a href="#CR24" class="usa-link" aria-describedby="CR24">2019</a>).</p>
<p id="Par51">Increased BC biosynthesis was also observed in co-cultures of AAB with other microorganisms, e.g., with <em>Lactobacilus fermentum</em>, <em>Bacilus cereus</em>, and <em>Escherichia coli</em> (Liu and Catchmark <a href="#CR40" class="usa-link" aria-describedby="CR40">2019</a>; Jiang et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2023</a>; Li et al. <a href="#CR37" class="usa-link" aria-describedby="CR37">2023</a>). The mechanisms promoting BC production in different co-cultures may vary. In one scheme, metabolites of one strain increase the activity of enzymes involved in ATP synthesis in AAB, thereby increasing energy levels and promoting cell growth (Li et al. <a href="#CR37" class="usa-link" aria-describedby="CR37">2023</a>). The second mechanism presumably involves one strain assimilating a compound or metabolite that inhibits a certain metabolic pathway in the second strain. Reducing the content of the inhibitor causes this pathway to be activated. For example, in an <em>E. coli</em> co-culture, it assimilates GA, preventing GA from inhibiting the pathway responsible for BC biosynthesis in acetic bacteria (Liu and Catchmark <a href="#CR40" class="usa-link" aria-describedby="CR40">2019</a>).</p>
<p id="Par52">Compared to the monoculture, the consumption of lactose in <em>K. xylinus</em> + <em>Lb. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-cultures was almost twice as high. Nevertheless, when analyzing these two co-cultures, a certain correlation can be observed. In both the <em>K. xylinus</em> + <em>Lb. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-cultures, there was a significant (mild in the case of the co-culture with <em>Lb. helveticus</em>) decrease in the lactose content of the medium after 6–9 days of culture, but this did not reflect a significant increase in BC mass (Fig. <a href="#Fig1" class="usa-link">1</a>) during this time. A potential explanation for this phenomenon is that lactose consumption could be related to the dominance of <em>Lb. acidophilus</em> and <em>Lb. helveticus</em>, which assimilated lactose and used it for their own metabolism, e.g., LA or EPS production. This may be confirmed by the observation that in the <em>K. xylinus</em> monoculture, the lactose consumption on days 6–9 is lower than the lactose consumption in the <em>Lb. acidophilus</em> and <em>Lb. helveticus</em> monocultures at this time. Moreover, lactose consumption by <em>Lb. acidophilus</em> and <em>Lb. helveticus</em> was observed until the 12th day of culture, after which time the lactose content of the medium with <em>Lb. acidophilus</em> and <em>Lb. helveticus</em> monocultures did not decrease (Fig. <a href="#Fig2" class="usa-link">2</a>A). Assuming that the metabolism of lactose by <em>Lb. acidophilus</em> and <em>Lb. helveticus</em> proceeds similarly in co-cultures with <em>K. xylinus</em>, it can be assumed that after 12 days of culture the remaining lactose in the medium becomes easily available to <em>K. xylinus</em>, since they no longer have to compete with LAB for this sugar. Therefore, <em>K. xylinus</em> may have increased BC production only after 12 days of culture. Low lactose consumption (0.3 g·100 mL<sup>−1</sup>) in the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture may suggest that allelopathic interactions occurred between <em>K. xylinus</em> and <em>Lb. delbrueckii</em>, meaning the growth and activity of one strain may have been limited by metabolites produced by the other strain. One possible explanation for this effect is that LAB produce compounds with high antioxidant activity (Xing et al. <a href="#CR67" class="usa-link" aria-describedby="CR67">2015</a>; Zhang et al. <a href="#CR68" class="usa-link" aria-describedby="CR68">2017</a>; Antolak et al. <a href="#CR2" class="usa-link" aria-describedby="CR2">2021</a>). Antioxidants could negatively affect aerobic <em>K. xylinus</em> bacteria, in which the oxidative phosphorylation of sugars is one of the primary mechanisms for obtaining energy (He et al. <a href="#CR26" class="usa-link" aria-describedby="CR26">2022</a>). Thus, <em>K. xylinus</em>, stimulated to obtain energy through alternative pathways, activated the substrate phosphorylation from non-sugar carbon sources by TCA (Zhong et al. <a href="#CR69" class="usa-link" aria-describedby="CR69">2013</a>), which would explain the low consumption of lactose, the decreasing content of LA and CA, and the production of AA (Table <a href="#Tab2" class="usa-link">2</a>). Another explanation suggests the inhibitory effect of metabolites (aldehydes, ketones, organic acids) of <em>K. xylinus</em> on the growth of <em>Lb. delbrueckii</em>, as evidenced by the lower content of LA, the main metabolite of LAB, compared to the <em>Lb. delbrueckii</em> monoculture. The impact of mutual interactions in the AAB co-culture with LAB was also noted by Xia et al. (<a href="#CR66" class="usa-link" aria-describedby="CR66">2022</a>). In the co-culture of <em>Acetobacter pasteurianus</em> with <em>Lb. helveticus</em>, inhibited growth of <em>Lb. helveticus</em> was observed compared to the monoculture, while the growth of <em>A. pasteurianus</em> remained unchanged. The authors argued that this was related to a higher content of AA in the medium, which could inhibit the growth of <em>Lb. helveticus</em>.</p>
<p id="Par53">One of our goals in this study was to check the possibility of producing BC in AW where the main carbon source is lactose. According to literature data, one of the challenges associated with obtaining BC from AW is the low ability of AAB to assimilate whey proteins and uptake lactose. This is due to the absence of the <em>lacZ</em> gene in the genomes of most AAB, the expression of which determines the synthesis of β-galactosidase, the enzyme responsible for breaking down lactose into glucose and galactose (Kolesovs and Semjonovs <a href="#CR32" class="usa-link" aria-describedby="CR32">2020</a>). Brugnoli et al. (<a href="#CR9" class="usa-link" aria-describedby="CR9">2023a</a>) used cheese whey as a medium for BC biosynthesis using, among others, <em>K. xylinus</em> UMCC 2756 and obtained more BC than in Hestrin-Schramm medium. However, before culturing, the whey was treated with β-galactosidase from <em>Aspergillus oryzae</em>, which confirms that <em>K. xylinus</em> UMCC 2756 is not able to degrade lactose into glucose and galactose (Brugnoli et al. <a href="#CR9" class="usa-link" aria-describedby="CR9">2023a</a>). However, our research has shown that in the <em>K. xylinus</em> UMCC 2756 monoculture, the lactose content decreased, suggesting that probably it is included in some metabolic pathway of <em>K. xylinus</em> UMCC 2756, which enables BC production in AW. It is worth noting that the use of lactose and BC biosynthesis increased in the presence of LA in the medium and at pH 3.5–3.8. This observation is consistent with reports by Li et al. (<a href="#CR37" class="usa-link" aria-describedby="CR37">2023</a>), which showed that glucose consumption increased after medium supplementation with acetoin. Therefore, to increase BC production in pure AW, it is necessary to use other stimulants like LA, as its biosynthesis in AW is not very efficient. Another hypothetical explanation for the lactose consumption by <em>K. xylinus</em> is its oxidation to lactobionic acid (4-<em>O</em>-β-galactopyranosyl-D-gluconic acid, LBA) (Kiryu et al. <a href="#CR31" class="usa-link" aria-describedby="CR31">2019</a>). LBA is formed as a result of enzymatic oxidation of lactose by the periplasmic membrane–bound quinoprotein glucose dehydrogenase (m-GDH). The presence of m-GDH and its oxidative activity has been proven in 48 AAB strains, with the highest activity found in <em>Komagataeibacte</em>r (formerly <em>Gluconacetobacter</em>) <em>medellinensis</em> NBRC3288 (Kiryu et al. <a href="#CR30" class="usa-link" aria-describedby="CR30">2015</a>). Interestingly, the presence of the <em>gcd</em> gene was found in the genome of <em>K. xylinus</em> UMCC 2756 (GeneBank no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/QQBI00000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">QQBI00000000</a>) (Gullo et al. <a href="#CR24" class="usa-link" aria-describedby="CR24">2019</a>), which encodes the production of m-GDH. However, these studies did not examine the content of LBA, so further research is needed to confirm the ability of <em>K. xylinus</em> UMCC 2756 to oxidize lactose.</p>
<p id="Par54">In recent years, the possibility of obtaining BC in media with lactose as the primary carbon source has been studied, but the efficiency was unsatisfactory (Cielecka et al. <a href="#CR14" class="usa-link" aria-describedby="CR14">2021b</a>). In studies on the cultivation of the <em>Komagataeibacter</em> sp. W1 strain by Wang et al. (<a href="#CR65" class="usa-link" aria-describedby="CR65">2018</a>), only 0.04 g·L<sup>−1</sup> of BC was obtained, and during the cultivation of the <em>G. hansenii</em> GH1/2008 strain by Bolgova et al. (<a href="#CR7" class="usa-link" aria-describedby="CR7">2023</a>), only 0.1 g·L<sup>−1</sup> was produced. In contrast, Singhsa et al. (<a href="#CR62" class="usa-link" aria-describedby="CR62">2018</a>) examined five strains of <em>K. xylinus</em> and found that the amount of BC produced from lactose may differ not only due to the strain used but also due to the cultivation method (stationary or mixed). Under stationary conditions, the highest BC production (approx. 1.4 g·L<sup>−1</sup>) from lactose was achieved by the <em>K. xylinus</em> strain, whose efficiency in shaken cultures did not change. In the stationary cultivation of the <em>K. xylinus</em> K1011 and <em>K. xylinus</em> K975 strains, 0.9 g·L<sup>−1</sup> and 1.3 g·L<sup>−1</sup> of BC was obtained, respectively, while in the mixed cultivation, the efficiency increased to approximately 4.7 g·L<sup>−1</sup> and 2.4 g·L<sup>−1</sup>, respectively (Singhsa et al. <a href="#CR62" class="usa-link" aria-describedby="CR62">2018</a>). Another example of one of the few studies showing the natural ability of AAB to assimilate lactose (and thus increase BC biosynthesis) is the research by Revin et al. (<a href="#CR58" class="usa-link" aria-describedby="CR58">2018</a>) which demonstrated that the <em>Gluconobacter sucrofermentans</em> B-11267 strain could assimilate lactose from whey and produced 5.45 g·L<sup>−1</sup> of BC. A summary of research showing the possibilities of producing BC from AW as a standalone medium or an additive to other media is described in our previous publication (Płoska et al. <a href="#CR51" class="usa-link" aria-describedby="CR51">2023b</a>).</p>
<p id="Par55">A characteristic feature of microbial co-cultures is that the metabolites of one microorganism can be substrates entering the metabolic pathways of another one. Metabolism of citrate to acetate is typical of LAB (Oliveira et al. <a href="#CR15" class="usa-link" aria-describedby="CR15">2012</a>). In homofermentative LAB, the production of acetate and oxaloacetate from citrate is catalyzed by citrate lyase (Fig. <a href="#Fig7" class="usa-link">7</a>). The acetate is excreted outside the cell, and the oxaloacetate is decarboxylated to pyruvate by oxaloacetate decarboxylase. In subsequent steps, pyruvate may be further converted to lactate, acetate, formate, ethanol, or four-carbon aromatic compounds such as acetoin and 2,3-butanediol (Quintans, et al. <a href="#CR55" class="usa-link" aria-describedby="CR55">2008</a>; Pudlik and Lolkema <a href="#CR53" class="usa-link" aria-describedby="CR53">2011</a>; Bintsis <a href="#CR5" class="usa-link" aria-describedby="CR5">2018</a>). The latter two metabolites may be relevant in the context of the AAB co-culture. According to Li et al. (<a href="#CR37" class="usa-link" aria-describedby="CR37">2023</a>), acetoin and 2,3-butanediol can positively affect the growth and activity of AAB by increasing the activity of acetoin dehydrogenase, which catalyzes the conversion of acetoin to acetyl-coenzyme A (CoA) and acetaldehyde. Additionally, acetaldehyde may be converted to acetate and then acetyl-CoA by acetaldehyde dehydrogenase and acetyl-CoA synthetase, respectively. An increase in acetyl-CoA concentration leads to the generation of more ATP molecules, which in turn provides the energy necessary for cell growth and BC biosynthesis (Fig. <a href="#Fig7" class="usa-link">7</a>). This mechanism was confirmed in the co-culture of <em>K. xylinus</em> with <em>Bacillus cereus</em>, where acetoin and 2,3-butanediol were metabolites of <em>B. cereus</em>. Notably, 4.4 g·L<sup>−1</sup> of BC was obtained from the co-culture, compared to 1.2 g·L<sup>−1</sup> from the <em>K. xylinus</em> monoculture (Li et al. <a href="#CR37" class="usa-link" aria-describedby="CR37">2023</a>). A hypothetical scheme illustrating the effect of LAB metabolites on AAB metabolism using citrate and acetoin metabolism as example is shown in Fig. <a href="#Fig7" class="usa-link">7</a>.</p>
<figure class="fig xbox font-sm" id="Fig7"><h3 class="obj_head">Fig. 7.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370864_253_2025_13582_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b56f/12370864/5671b36c9d1f/253_2025_13582_Fig7_HTML.jpg" loading="lazy" id="MO7" height="521" width="749" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Hypothetical scheme of metabolite flow in acetic-lactic co-culture using citrate and acetoin metabolism as an example</p></figcaption></figure><p id="Par56">In AAB, gluconic acid and its derivatives are formed as a result of aerobic glucose metabolism, where they participate in one of two pathways for energy production in the cell. When glucose is the carbon source in the medium, it is phosphorylated by glucokinase to glucose-6-phosphate, which is then converted to 6-phosphogluconolactone. This process is associated with the reduction of nicotinamide adenine dinucleotide phosphate (NADP<sup>+</sup>) to NADPH, generating energy and transferring electrons to electron acceptors (Liu et al. <a href="#CR41" class="usa-link" aria-describedby="CR41">2018</a>). In contrast, when glucose is absent from the medium and another carbon source, such as mannitol, lactose, or fructose, is present, GA is produced in very low concentrations or not produced at all (Anguluri et al. <a href="#CR1" class="usa-link" aria-describedby="CR1">2022</a>). In our study, the primary carbon source in AW was lactose, which justifies the low content of GA. In almost all cultures, the GA content decreased over time, with the exception of the <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture, where the GA content increased. The decrease in GA content in the medium can be explained by the hypothesis put forward by Liu et al. (<a href="#CR41" class="usa-link" aria-describedby="CR41">2018</a>), who argued that in the absence of glucose in the medium, GA can be converted to glucose-6-phosphate and further used as a carbon source for cell growth and BC production via the pentose phosphate pathway. Interestingly, the increase in GA content in the <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture may be correlated with the increase in galactose content. This supports the hypothesis about the oxidation of lactose to LBA by <em>K. xylinus</em>. It is possible that <em>Lb. helveticus</em> produces enzymes that enable the hydrolysis of LBA to galactose and GA, thus explaining the observed increase in their content in the medium (Fig. <a href="#Fig2" class="usa-link">2</a>F).</p>
<p id="Par57">The FTIR spectra of BC obtained in the <em>K. xylinus</em> monoculture and acetic-lactic co-cultures were similar (Fig. <a href="#Fig4" class="usa-link">4</a>). In all samples, peaks corresponding to bonds and functional groups characteristic of BC were observed, and no new peaks were observed in spectra of BC from co-cultures. In studies by Liu and Catchmark (<a href="#CR40" class="usa-link" aria-describedby="CR40">2019</a>), the spectrum of BC from the <em>Ga. hansenii</em> and <em>E. coli</em> co-culture showed the presence of a peak corresponding to the C–N bonds in glucosamine, which was derived from EPS produced by <em>E. coli</em>. In our study, despite the proven ability of LAB to produce EPS (Table <a href="#Tab3" class="usa-link">3</a>), no extra peaks indicating the occurrence of C–N bonds were observed. Nevertheless, from the wavenumber of 1400 to 600 cm<sup>−1</sup> (Fig. <a href="#Fig4" class="usa-link">4</a>), the peak intensity in the spectra of BC from all co-cultures was higher compared to that in the spectra of BC from the <em>K. xylinus</em> monoculture. Peak intensity is assumed to be directly proportional to the number (per unit volume) of functional groups and bonds occurring in the material (Farhadi and Sohbatzadeh <a href="#CR18" class="usa-link" aria-describedby="CR18">2023</a>). Therefore, considering the similarity of functional groups and bonds in the chemical structure of BC and EPS, potential interactions between BC and EPS produced by LAB can be assumed. A significant difference in peak intensity in the FTIR spectra was observed in BC from the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture. In this case, the intensity of the peak corresponding to the hydroxyl group (3336 cm<sup>−1</sup>) was lower than that seen in the spectra of other BC, which may indicate a lower number of hydroxyl groups and hydrogen bonds. According to Sharma et al. (<a href="#CR61" class="usa-link" aria-describedby="CR61">2023</a>), the intensity of the peak at wavenumber 3336 cm<sup>−1</sup> indicates the strength of the intramolecular H3(O)H…O(5) bond, which can be increased in the presence of LA in the medium. This explains the lower intensity of this peak in the spectrum of BC from the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture, where the lowest LA content was observed (Table <a href="#Tab2" class="usa-link">2</a>) compared to BC from other co-cultures and monoculture. It can be assumed that minor variations in the FTIR spectra of BC may have been influenced by different combinations of strains and differences in metabolites in the co-culture system, as mentioned by other researchers (Jiang et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2023</a>).</p>
<p id="Par58">Based on the MRDT values, it was found that the co-cultures <em>K. xylinus</em> + <em>Lb. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> positively affected the thermal properties of BC, with the highest thermostability exhibited by BC obtained from the co-culture <em>K. xylinus</em> + <em>Lb. acidophilus</em>. The apparent shift in the maximum temperature of BC degradation from the <em>K. xylinus</em> monoculture and the <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture toward a lower temperature may be related to the low crystallinity of these BC variants and a higher share of the amorphous fraction (unpublished results) compared to BC from the other cultures. According to Cichosz and Masek (<a href="#CR12" class="usa-link" aria-describedby="CR12">2020</a>), among others, the degree of crystallinity and the ratio of crystalline to amorphous areas determine the thermal stability of cellulose. The more amorphous areas in the structure (lower crystallinity), the more easily it degrades. According to Lin et al. (<a href="#CR38" class="usa-link" aria-describedby="CR38">2023</a>), changes in BC crystallinity, and therefore thermostability, can be caused by various interactions in the culture medium during crystallization, such as increasing the viscosity of the medium. Additionally, based on the conducted research, we cannot definitively state that the co-culture product is pure BC. We can suspect that some part of the EPS produced by the co-cultured LAB may have built into the BC structure, but we do not have definite evidence for this. On the basis of our results (including unpublished ones) and based on literature data, we assume that the differences in, among others, the thermal stability of the polymer obtained are due to specific interactions between <em>K. xylinus</em> and the particular strain of LAB.</p>
<p id="Par59">The mechanical properties of BC depend on many factors such as culture conditions, medium composition, fiber concentration, fiber arrangement, and bacterial strain used to obtain BC (Cielecka et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2021a</a>; Brugnoli et al. <a href="#CR11" class="usa-link" aria-describedby="CR11">2025</a>). Cellulose obtained from the <em>K. xylinus</em> + <em>Lb. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-ultures was characterized by the highest strain at break (Fig. <a href="#Fig6" class="usa-link">6</a>A), which could be related to a higher LA content in the media and a lower pH in these culture variants. As already mentioned, the presence of LA in the medium may increase the strength of the intramolecular H3(O)H…O(5) bonds within the glucopyranose ring (Sharma et al. <a href="#CR61" class="usa-link" aria-describedby="CR61">2023</a>). In turn, the created intra- and intermolecular bonds in cellulose affect not only the physical properties of the polymer, including solubility, hydroxyl reactivity, and crystallinity, but also play an important role in its mechanical properties (Fan et al. <a href="#CR17" class="usa-link" aria-describedby="CR17">2012</a>). According to Sharma et al. (<a href="#CR61" class="usa-link" aria-describedby="CR61">2023</a>), the addition of LA to the medium significantly increased the strain at break of the produced BC compared to the BC obtained in the medium without LA. These results also confirm the reports by Cielecka et al. (<a href="#CR14" class="usa-link" aria-describedby="CR14">2021b</a>) where LA medium supplementation positively influenced the strain at break. Moreover, it was shown that the strain at break of BC obtained in media with glucose and at different pH varied significantly, which would indicate the influence of the pH of the medium on the mechanical properties of BC. In our research, this hypothesis can be confirmed by the difference between the high strain at break of BC from the <em>K. xylinus</em> + <em>Lb. acidophilus</em> and <em>K. xylinus</em> + <em>Lb. helveticus</em> co-cultures, where the final pH was 3.8 and 3.6, respectively (Table <a href="#Tab1" class="usa-link">1</a>), and the low strain at break of BC obtained in the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture, where the final pH was approx. 6.0. The effect of the pH of the medium can also be observed in the case of Young’s modulus (Fig. <a href="#Fig6" class="usa-link">6</a>C); however, in this instance, a reverse trend was noted. The highest Young’s modulus was characterized by BC obtained in the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture, and it was significantly (<em>p</em> &lt; 0.05) higher than the values for other BC samples. Nevertheless, it is also worth noting that in the case of this co-culture, the high value of Young’s modulus could have been influenced by EPS produced by <em>Lb. delbrueckii</em> (Table <a href="#Tab3" class="usa-link">3</a>). Liu and Catchmark (<a href="#CR39" class="usa-link" aria-describedby="CR39">2018</a>) conducted studies which demonstrated that the addition of EPS to the culture medium had a positive effect on the Young’s modulus of BC compared to BC obtained from the Hestrin-Schramm medium without EPS. Moreover, it was observed that depending on the concentration, the addition of EPS may have a different effect on the strain at break and the stress at break. In our study, the highest stress at break value (Fig. <a href="#Fig6" class="usa-link">6</a>B) was observed for cellulose obtained in the <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture (17.18 MPa), and a slightly lower value was recorded for BC produced in the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture (14.43 MPa)<em>.</em> These results can be linked to the arrangement and the largest diameter of BC fibers from these co-culture variants. It is possible that more force was required to tear apart thicker BC fibers, resulting in increased tensile stress (Cielecka et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2021a</a>). Similar conclusions were formulated by Liu and Catchmark (<a href="#CR40" class="usa-link" aria-describedby="CR40">2019</a>) who, in research on the <em>Ga. hansenii</em> and <em>E. coli</em> co-culture, showed that the BC obtained from the co-culture was characterized by higher density and thicker fibers compared to BC from the <em>Ga. hansenii</em> monoculture, which affected its mechanical properties. On the other hand, Brugnoli et al. (<a href="#CR11" class="usa-link" aria-describedby="CR11">2025</a>) increased Young’s modulus and stress at break of the BC-hyaluronic acid composite obtained in co-culture of AAB with LAB, explained by high BC content and high crystallinity. Nevertheless, the authors noted the diverse effects of different strains on the mechanical properties of BC, which implies the necessity of proper selection of strains for co-culture.</p></section><section id="Sec24"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par60">In this study, it was shown that AW, a side product of the dairy industry, can be a valuable and efficient substrate for the biotechnological production of BC both in AAB monoculture and in AAB-LAB co-culture. The efficiency of biosynthesis and its properties varied depending on the used variant of co-culture of <em>K. xylinus</em> and LAB. It was found that co-cultures of AAB with LAB exhibiting higher acidification capabilities, i.e., <em>Lb. acidophilus</em> and <em>Lb. helveticus</em>, promoted BC production more effectively than the co-culture with <em>Lb. delbrueckii</em> or the <em>K. xylinus</em> monoculture. This fact confirms the first part of our hypothesis, suggesting that in the AAB co-culture with LAB, BC biosynthesis in AW can be increased. The production of BC was influenced by the presence of LA in the medium, which enhanced the activity of enzymes with TCA, resulting in the production of more energy, which probably translated into the biosynthesis of BC. It can be assumed that in the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture, negative allelopathy occurred, consisting in the growth and activity of one strain being limited by metabolites produced by the other strain. However, further research is necessary to learn the detailed mechanism of this relationship. It was observed that the <em>K. xylinus</em> UMCC 2756 strain used lactose derived from AW; however, the BC production took place only when LA was present in the medium and at a lower pH (3.5–3.8). This suggests that efficient BC biosynthesis from AW should be supported by additional LA supplementation.</p>
<p id="Par61">The consumption of lactose from whey by <em>K. xylinus</em> UMCC 2756 despite the lack of the <em>lacZ</em> gene, encoding β-galactosidase, is a mechanism worth investigating at the molecular biology level. Here, it would be of particular interest to find and study a potential metabolic pathway in AABs in which lactose is incorporated, which would explain how AABs acquire a carbon source to build BC subunits. It can be assumed that lactose is oxidized to LBA. In this case, conducting molecular and bioinformatics analyses may be of particular interest because the <em>gcd</em> gene, which encodes m-GDH responsible for oxidizing lactose to LBA, was found in <em>K. xylinus</em>.</p>
<p id="Par62">Our research confirmed the ability of LAB to produce EPS in AW. Notably, in AW, the EPS production was two and four times lower than in the MRS medium for <em>Lb. acidophilus</em>, <em>Lb. helveticus</em>, and <em>Lb. delbrueckii</em>, respectively. It was also demonstrated that LAB produce EPS in co-cultures, which could have influenced, among other factors, the properties of BC. By analyzing the FTIR spectra and basing on the similarity of functional groups and bonds in the chemical structures of BC and EPS, potential interactions between BC and EPS produced by LAB can be assumed. EPS presumably influenced the thickness and structure of BC fibers as well as their spatial arrangement. Increased thermostability was noted in BC obtained from the <em>K. xylinus</em> + <em>Lb. acidophilus</em> co-ulture compared to other BC samples. Cellulose obtained in co-cultures was characterized by better mechanical properties. The <em>K. xylinus</em> + <em>Lb. helveticus</em> co-culture positively influenced the stress at break of BC. BC from the <em>K. xylinus</em> + <em>Lb. delbrueckii</em> co-culture exhibited the highest Young’s modulus. In turn, BC obtained in the <em>K. xylinus</em> + <em>Lb. acidophilus</em> co-culture was characterized by the highest strain at break. This confirms the second part of our hypothesis that BC produced in the co-culture of AAB with LAB in AW shows improved chemical-hysical and mechanical properties.</p>
<p id="Par63">Future study of the relationships discovered in this work may contribute to more effective biosynthesis of BC in AW by acetic-lactic co-cultures. It has been demonstrated that the use of microorganisms with different metabolic abilities in co-cultures may lead to the production of metabolites stimulating BC biosynthesis as well as increase the availability of nutrients present in the media. In turn, cultivating co-cultures in inexpensive by-product media, such as AW, can not only reduce the costs of producing BC but also decrease the negative impact of this by-product on the environment.</p></section><section id="Sec26"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370864/bin/253_2025_13582_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link"><span>MOESM 1</span></a><sup> (10.5MB, docx) </sup><p>(DOCX 10,789 KB)</p>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author contribution
</h2>
<p>JP: Conceptualization, formal analysis, investigation, methodology; visualization; writing—original draft, writing—review &amp; editing; MG: Writing—review &amp; editing; IŚ: Investigation, methodology, formal analysis; LSR: Conceptualization, supervision, methodology, writing—review &amp; editing.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Part of the research was conducted during a Justyna Płoska scholarship funded by the Polish National Agency of Academic Exchange (NAWA) in the Exchange program for students and scientists as part of bilateral cooperation (No. BPN/BIL/2021/1/00171/U/00001).</p>
<p>Research was partially funded by the “Research Network of Life Science Universities for the development of the Polish dairy sector – research project” SUP-RIM, financed by the Ministry of Science and Higher Education, (No. MEiN/2023/DPI/2866).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability
</h2>
<p>The datasets and materials from this study can be available upon request to interested researchers.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethical approval</h3>
<p id="Par64">This article does not contain any studies with human participants or animals performed by any of the authors.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Competing interests
</h3>
<p id="Par65">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References
</h2>
<section id="Bib1_sec2"><ol class="ref-list font-sm">
<li id="CR1">
<cite>Anguluri K, La China S, Brugnoli M, Cassanelli S, Gullo M (2022) Better under stress: improving bacterial cellulose production by <em>Komagataeibacter xylinus</em> K2G30 (UMCC 2756) using adaptive laboratory evolution. Front Microbiol 13:994097. 10.3389/FMICB.2022.994097
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.994097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9605694/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36312960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Anguluri%20K,%20La%20China%20S,%20Brugnoli%20M,%20Cassanelli%20S,%20Gullo%20M%20(2022)%20Better%20under%20stress:%20improving%20bacterial%20cellulose%20production%20by%20Komagataeibacter%20xylinus%20K2G30%20(UMCC%202756)%20using%20adaptive%20laboratory%20evolution.%20Front%20Microbiol%2013:994097.%2010.3389/FMICB.2022.994097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<cite>Antolak H, Piechota D, Kucharska A (2021) Kombucha tea—a double power of bioactive compounds from tea and symbiotic culture of bacteria and yeasts (SCOBY). Antioxidants 10:1541. 10.3390/ANTIOX10101541
</cite> [<a href="https://doi.org/10.3390/antiox10101541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8532973/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34679676/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Antolak%20H,%20Piechota%20D,%20Kucharska%20A%20(2021)%20Kombucha%20tea%E2%80%94a%20double%20power%20of%20bioactive%20compounds%20from%20tea%20and%20symbiotic%20culture%20of%20bacteria%20and%20yeasts%20(SCOBY).%20Antioxidants%2010:1541.%2010.3390/ANTIOX10101541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<cite>Ayivi RD, Gyawali R, Krastanov A, Aljaloud SO, Worku M, Tahergorabi R, da Silva RC, Ibrahim SA (2020) Lactic acid bacteria: food safety and human health applications. Dairy 1:202–232. 10.3390/DAIRY1030015</cite> [<a href="https://scholar.google.com/scholar_lookup?Ayivi%20RD,%20Gyawali%20R,%20Krastanov%20A,%20Aljaloud%20SO,%20Worku%20M,%20Tahergorabi%20R,%20da%20Silva%20RC,%20Ibrahim%20SA%20(2020)%20Lactic%20acid%20bacteria:%20food%20safety%20and%20human%20health%20applications.%20Dairy%201:202%E2%80%93232.%2010.3390/DAIRY1030015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<cite>Bagewadi ZK, Dsouza V, Yaraguppi DA, Mulla SI, Deshpande SH, Shettar SS (2023) Low cost production of bacterial cellulose through statistical optimization and developing its composites for multipurpose applications. Process Biochem 125:47–60. 10.1016/j.procbio.2022.12.001</cite> [<a href="https://scholar.google.com/scholar_lookup?Bagewadi%20ZK,%20Dsouza%20V,%20Yaraguppi%20DA,%20Mulla%20SI,%20Deshpande%20SH,%20Shettar%20SS%20(2023)%20Low%20cost%20production%20of%20bacterial%20cellulose%20through%20statistical%20optimization%20and%20developing%20its%20composites%20for%20multipurpose%20applications.%20Process%20Biochem%20125:47%E2%80%9360.%2010.1016/j.procbio.2022.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<cite>Bintsis T (2018) Lactic acid bacteria as starter cultures: an update in their metabolism and genetics. AIMS Microbiol 4:665. 10.3934/MICROBIOL.2018.4.665
</cite> [<a href="https://doi.org/10.3934/microbiol.2018.4.665" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6613329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31294241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bintsis%20T%20(2018)%20Lactic%20acid%20bacteria%20as%20starter%20cultures:%20an%20update%20in%20their%20metabolism%20and%20genetics.%20AIMS%20Microbiol%204:665.%2010.3934/MICROBIOL.2018.4.665" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<cite>Blanco Parte FG, Santoso SP, Chou CC, Verma V, Wang HT, Ismadji S, Cheng KC (2020) Current progress on the production, modification, and applications of bacterial cellulose. Crit Rev Biotechnol 40:397–414. 10.1080/07388551.2020.1713721
</cite> [<a href="https://doi.org/10.1080/07388551.2020.1713721" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31937141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blanco%20Parte%20FG,%20Santoso%20SP,%20Chou%20CC,%20Verma%20V,%20Wang%20HT,%20Ismadji%20S,%20Cheng%20KC%20(2020)%20Current%20progress%20on%20the%20production,%20modification,%20and%20applications%20of%20bacterial%20cellulose.%20Crit%20Rev%20Biotechnol%2040:397%E2%80%93414.%2010.1080/07388551.2020.1713721" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<cite>Bolgova AL, Shevtsov AV, Arkharova NA, Karimov DN, Makarov IS, Gromovykh TI, Klechkovskaya VV (2023) Microstructure of gel films of bacterial cellulose synthesized under static conditions of cultivation of the <em>Gluconacetobacter hansenii</em> GH-1/2008 strain on nutrient media with different carbon sources. Crystallogr Rep 68:607–614. 10.1134/S1063774523600369</cite> [<a href="https://scholar.google.com/scholar_lookup?Bolgova%20AL,%20Shevtsov%20AV,%20Arkharova%20NA,%20Karimov%20DN,%20Makarov%20IS,%20Gromovykh%20TI,%20Klechkovskaya%20VV%20(2023)%20Microstructure%20of%20gel%20films%20of%20bacterial%20cellulose%20synthesized%20under%20static%20conditions%20of%20cultivation%20of%20the%20Gluconacetobacter%20hansenii%20GH-1/2008%20strain%20on%20nutrient%20media%20with%20different%20carbon%20sources.%20Crystallogr%20Rep%2068:607%E2%80%93614.%2010.1134/S1063774523600369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<cite>Bosco F, Cirrincione S, Carletto R, Marmo L, Chiesa F, Mazzoli R, Pessione E (2021) PHA production from cheese whey and “scotta”: Comparison between a consortium and a pure culture of <em>Leuconostoc mesenteroides</em>. Microorganisms 9:2426. 10.3390/MICROORGANISMS9122426
</cite> [<a href="https://doi.org/10.3390/microorganisms9122426" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8704080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34946028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bosco%20F,%20Cirrincione%20S,%20Carletto%20R,%20Marmo%20L,%20Chiesa%20F,%20Mazzoli%20R,%20Pessione%20E%20(2021)%20PHA%20production%20from%20cheese%20whey%20and%20%E2%80%9Cscotta%E2%80%9D:%20Comparison%20between%20a%20consortium%20and%20a%20pure%20culture%20of%20Leuconostoc%20mesenteroides.%20Microorganisms%209:2426.%2010.3390/MICROORGANISMS9122426" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<cite>Brugnoli M, La China S, Lasagni F, Romeo FV, Pulvirenti A, Gullo M (2023) Acetic acid bacteria in agro-wastes: from cheese whey and olive mill wastewater to cellulose. Appl Microbiol Biotechnol 107:3729–3744. 10.1007/S00253-023-12539-8
</cite> [<a href="https://doi.org/10.1007/s00253-023-12539-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37115254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brugnoli%20M,%20La%20China%20S,%20Lasagni%20F,%20Romeo%20FV,%20Pulvirenti%20A,%20Gullo%20M%20(2023)%20Acetic%20acid%20bacteria%20in%20agro-wastes:%20from%20cheese%20whey%20and%20olive%20mill%20wastewater%20to%20cellulose.%20Appl%20Microbiol%20Biotechnol%20107:3729%E2%80%933744.%2010.1007/S00253-023-12539-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<cite>Brugnoli M, Mazzini I, La China S, De Vero L, Gullo M (2023) A microbial co-culturing system for producing cellulose-hyaluronic acid composites. Microorganisms 11(6):1504. 10.3390/MICROORGANISMS11061504
</cite> [<a href="https://doi.org/10.3390/microorganisms11061504" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10305311/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37375006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brugnoli%20M,%20Mazzini%20I,%20La%20China%20S,%20De%20Vero%20L,%20Gullo%20M%20(2023)%20A%20microbial%20co-culturing%20system%20for%20producing%20cellulose-hyaluronic%20acid%20composites.%20Microorganisms%2011(6):1504.%2010.3390/MICROORGANISMS11061504" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<cite>Brugnoli M, Carvalho JPF, Arena MP, Oliveira H, Vilela C, Freire CSR, Gullo M (2025) Co-cultivation of <em>Komagataeibacter</em> sp. and <em>Lacticaseibacillus</em> sp. strains to produce bacterial nanocellulose-hyaluronic acid nanocomposite membranes for skin wound healing applications. Int J Biol Macromol 299:140208. 10.1016/J.IJBIOMAC.2025.140208
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2025.140208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39848389/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brugnoli%20M,%20Carvalho%20JPF,%20Arena%20MP,%20Oliveira%20H,%20Vilela%20C,%20Freire%20CSR,%20Gullo%20M%20(2025)%20Co-cultivation%20of%20Komagataeibacter%20sp.%20and%20Lacticaseibacillus%20sp.%20strains%20to%20produce%20bacterial%20nanocellulose-hyaluronic%20acid%20nanocomposite%20membranes%20for%20skin%20wound%20healing%20applications.%20Int%20J%20Biol%20Macromol%20299:140208.%2010.1016/J.IJBIOMAC.2025.140208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<cite>Cichosz S, Masek A (2020) Thermal behavior of green cellulose-filled thermoplastic elastomer polymer blends. Molecules. 10.3390/MOLECULES25061279
</cite> [<a href="https://doi.org/10.3390/molecules25061279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7143982/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32178229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cichosz%20S,%20Masek%20A%20(2020)%20Thermal%20behavior%20of%20green%20cellulose-filled%20thermoplastic%20elastomer%20polymer%20blends.%20Molecules.%2010.3390/MOLECULES25061279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<cite>Cielecka I, Ryngajłło M, Maniukiewicz W, Bielecki S (2021) Response surface methodology-based improvement of the yield and differentiation of properties of bacterial cellulose by metabolic enhancers. Int J Biol Macromol 187:584–593. 10.1016/J.IJBIOMAC.2021.07.147
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2021.07.147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34324907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cielecka%20I,%20Ryngaj%C5%82%C5%82o%20M,%20Maniukiewicz%20W,%20Bielecki%20S%20(2021)%20Response%20surface%20methodology-based%20improvement%20of%20the%20yield%20and%20differentiation%20of%20properties%20of%20bacterial%20cellulose%20by%20metabolic%20enhancers.%20Int%20J%20Biol%20Macromol%20187:584%E2%80%93593.%2010.1016/J.IJBIOMAC.2021.07.147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<cite>Cielecka I, Ryngajłło M, Maniukiewicz W, Bielecki S (2021) Highly stretchable bacterial cellulose produced by <em>Komagataeibacter hansenii</em> SI1. Polymers 13:4455. 10.3390/POLYM13244455
</cite> [<a href="https://doi.org/10.3390/polym13244455" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8707637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34961006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cielecka%20I,%20Ryngaj%C5%82%C5%82o%20M,%20Maniukiewicz%20W,%20Bielecki%20S%20(2021)%20Highly%20stretchable%20bacterial%20cellulose%20produced%20by%20Komagataeibacter%20hansenii%20SI1.%20Polymers%2013:4455.%2010.3390/POLYM13244455" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15"><cite> Oliveira RP de S, Torres BR, Perego P, Oliveira MN de, Converti A (2012) Co-metabolic models of <em>Streptococcus thermophilus</em> in co-culture with <em>Lactobacillus bulgaricus</em> or <em>Lactobacillus acidophilus</em>. Biochem Eng J 62:62–69. 10.1016/J.BEJ.2012.01.004</cite></li>
<li id="CR16">
<cite>Elghali S, Mustafa S, Amid M, Manap MYA, Ismail A, Abas F (2005) Variations on soymilk components during fermentation by <em>Lactobacillus</em> and <em>Bifidobacterium</em> strains. J Food Agric Environ 12:1–5</cite> [<a href="https://scholar.google.com/scholar_lookup?Elghali%20S,%20Mustafa%20S,%20Amid%20M,%20Manap%20MYA,%20Ismail%20A,%20Abas%20F%20(2005)%20Variations%20on%20soymilk%20components%20during%20fermentation%20by%20Lactobacillus%20and%20Bifidobacterium%20strains.%20J%20Food%20Agric%20Environ%2012:1%E2%80%935" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17"><cite>Fan M, Dai D, Huang B (2012) Fourier transform infrared spectroscopy for natural fibres. In: Salih S (ed) Fourier transform - Materials analysis, 1st edn. Intechopen, Rijeka, pp 45–69</cite></li>
<li id="CR18">
<cite>Farhadi M, Sohbatzadeh F (2023) Influence of a transient spark plasma discharge on producing high molecular masses of chemical products from l-cysteine. Sci Rep 13:1–20. 10.1038/s41598-023-28736-4
</cite> [<a href="https://doi.org/10.1038/s41598-023-28736-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9899256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36739465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Farhadi%20M,%20Sohbatzadeh%20F%20(2023)%20Influence%20of%20a%20transient%20spark%20plasma%20discharge%20on%20producing%20high%20molecular%20masses%20of%20chemical%20products%20from%20l-cysteine.%20Sci%20Rep%2013:1%E2%80%9320.%2010.1038/s41598-023-28736-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<cite>Faria M, Mohammadkazemi F, Aguiar R, Cordeiro N (2022) Agro-industrial byproducts as modification enhancers of the bacterial cellulose biofilm surface properties: an inverse chromatography approach. Ind Crops Prod 177:114447. 10.1016/J.INDCROP.2021.114447</cite> [<a href="https://scholar.google.com/scholar_lookup?Faria%20M,%20Mohammadkazemi%20F,%20Aguiar%20R,%20Cordeiro%20N%20(2022)%20Agro-industrial%20byproducts%20as%20modification%20enhancers%20of%20the%20bacterial%20cellulose%20biofilm%20surface%20properties:%20an%20inverse%20chromatography%20approach.%20Ind%20Crops%20Prod%20177:114447.%2010.1016/J.INDCROP.2021.114447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<cite>Fernandez-Garcia E, McGregor JU (1994) Determination of organic acids during the fermentation and cold storage of yogurt. J Dairy Sci 77:2934–2939. 10.3168/JDS.S0022-0302(94)77234-9
</cite> [<a href="https://doi.org/10.3168/jds.S0022-0302(94)77234-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7836580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fernandez-Garcia%20E,%20McGregor%20JU%20(1994)%20Determination%20of%20organic%20acids%20during%20the%20fermentation%20and%20cold%20storage%20of%20yogurt.%20J%20Dairy%20Sci%2077:2934%E2%80%932939.%2010.3168/JDS.S0022-0302(94)77234-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<cite>Freitas JAM, Mendonça GMN, Santos LB, Alonso JD, Mendes JF, Barud HS, Azeredo HMC (2022) Bacterial cellulose/tomato puree edible films as moisture barrier structures in multicomponent foods. Foods. 10.3390/foods11152336
</cite> [<a href="https://doi.org/10.3390/foods11152336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9368048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35954102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Freitas%20JAM,%20Mendon%C3%A7a%20GMN,%20Santos%20LB,%20Alonso%20JD,%20Mendes%20JF,%20Barud%20HS,%20Azeredo%20HMC%20(2022)%20Bacterial%20cellulose/tomato%20puree%20edible%20films%20as%20moisture%20barrier%20structures%20in%20multicomponent%20foods.%20Foods.%2010.3390/foods11152336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<cite>Gao Y, Lin D, Peng H, Zhang R, Zhang B, Yang X (2023) Low oil Pickering emulsion gels stabilized by bacterial cellulose nanofiber/soybean protein isolate: an excellent fat replacer for ice cream. Int J Biol Macromol 247:125623. 10.1016/J.IJBIOMAC.2023.125623
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2023.125623" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37392915/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao%20Y,%20Lin%20D,%20Peng%20H,%20Zhang%20R,%20Zhang%20B,%20Yang%20X%20(2023)%20Low%20oil%20Pickering%20emulsion%20gels%20stabilized%20by%20bacterial%20cellulose%20nanofiber/soybean%20protein%20isolate:%20an%20excellent%20fat%20replacer%20for%20ice%20cream.%20Int%20J%20Biol%20Macromol%20247:125623.%2010.1016/J.IJBIOMAC.2023.125623" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<cite>Giraffa G, Andrighetto C, Antonello C, Gatti M, Lazzi C, Marcazzan G, Lombardi A, Neviani E (2004) Genotypic and phenotypic diversity of <em>Lactobacillus delbrueckii</em> subsp. <em>lactis</em> strains of dairy origin. Int J Food Microbiol 91(2):129–139. 10.1016/S0168-1605(03)00368-4
</cite> [<a href="https://doi.org/10.1016/S0168-1605(03)00368-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14996456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Giraffa%20G,%20Andrighetto%20C,%20Antonello%20C,%20Gatti%20M,%20Lazzi%20C,%20Marcazzan%20G,%20Lombardi%20A,%20Neviani%20E%20(2004)%20Genotypic%20and%20phenotypic%20diversity%20of%20Lactobacillus%20delbrueckii%20subsp.%20lactis%20strains%20of%20dairy%20origin.%20Int%20J%20Food%20Microbiol%2091(2):129%E2%80%93139.%2010.1016/S0168-1605(03)00368-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<cite>Gullo M, La China S, Petroni G, Di Gregorio S, Giudici P (2019) Exploring K2G30 genome: a high bacterial cellulose producing strain in glucose and mannitol based media. Front Microbiol 10:58. 10.3389/fmicb.2019.00058
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.00058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6363697/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30761107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gullo%20M,%20La%20China%20S,%20Petroni%20G,%20Di%20Gregorio%20S,%20Giudici%20P%20(2019)%20Exploring%20K2G30%20genome:%20a%20high%20bacterial%20cellulose%20producing%20strain%20in%20glucose%20and%20mannitol%20based%20media.%20Front%20Microbiol%2010:58.%2010.3389/fmicb.2019.00058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<cite>Guo Y, Zhang X, Hao W, Xie Y, Chen L, Li Z, Zhu B, Feng X (2018) Nano-bacterial cellulose/soy protein isolate complex gel as fat substitutes in ice cream model. Carbohydr Polym 198:620–630. 10.1016/j.carbpol.2018.06.078
</cite> [<a href="https://doi.org/10.1016/j.carbpol.2018.06.078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30093042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20Y,%20Zhang%20X,%20Hao%20W,%20Xie%20Y,%20Chen%20L,%20Li%20Z,%20Zhu%20B,%20Feng%20X%20(2018)%20Nano-bacterial%20cellulose/soy%20protein%20isolate%20complex%20gel%20as%20fat%20substitutes%20in%20ice%20cream%20model.%20Carbohydr%20Polym%20198:620%E2%80%93630.%2010.1016/j.carbpol.2018.06.078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<cite>He Y, Xie Z, Zhang H, Liebl W, Toyama H, Chen F (2022) Oxidative fermentation of acetic acid bacteria and its products. Front Microbiol 13:879246. 10.3389/FMICB.2022.879246
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.879246" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9171043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35685922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20Y,%20Xie%20Z,%20Zhang%20H,%20Liebl%20W,%20Toyama%20H,%20Chen%20F%20(2022)%20Oxidative%20fermentation%20of%20acetic%20acid%20bacteria%20and%20its%20products.%20Front%20Microbiol%2013:879246.%2010.3389/FMICB.2022.879246" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<cite>Huang YC, Khumsupan D, Lin SP, Santoso SP, Hsu HY, Cheng KC (2024) Production of bacterial cellulose (BC)/nisin composite with enhanced antibacterial and mechanical properties through co-cultivation of <em>Komagataeibacter xylinum</em> and <em>Lactococcus lactis</em> subsp. <em>lactis</em>. Int J Biol Macromol 258:128977. 10.1016/J.IJBIOMAC.2023.128977
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2023.128977" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38154722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20YC,%20Khumsupan%20D,%20Lin%20SP,%20Santoso%20SP,%20Hsu%20HY,%20Cheng%20KC%20(2024)%20Production%20of%20bacterial%20cellulose%20(BC)/nisin%20composite%20with%20enhanced%20antibacterial%20and%20mechanical%20properties%20through%20co-cultivation%20of%20Komagataeibacter%20xylinum%20and%20Lactococcus%20lactis%20subsp.%20lactis.%20Int%20J%20Biol%20Macromol%20258:128977.%2010.1016/J.IJBIOMAC.2023.128977" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<cite>Jiang H, Song Z, Hao Y, Hu X, Lin X, Liu S, Li C (2023) Effect of co-culture of <em>Komagataeibacter nataicola</em> and selected <em>Lactobacillus fermentum</em> on the production and characterization of bacterial cellulose. LWT 173:114224. 10.1016/J.LWT.2022.114224</cite> [<a href="https://scholar.google.com/scholar_lookup?Jiang%20H,%20Song%20Z,%20Hao%20Y,%20Hu%20X,%20Lin%20X,%20Liu%20S,%20Li%20C%20(2023)%20Effect%20of%20co-culture%20of%20Komagataeibacter%20nataicola%20and%20selected%20Lactobacillus%20fermentum%20on%20the%20production%20and%20characterization%20of%20bacterial%20cellulose.%20LWT%20173:114224.%2010.1016/J.LWT.2022.114224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<cite>Kazimierowicz J, Zieliński M, Bartkowska I, Dębowski M (2022) Effect of acid whey pretreatment using ultrasonic disintegration on the removal of organic compounds and anaerobic digestion efficiency. Int J Environ Res Public Health 19:11362. 10.3390/IJERPH191811362
</cite> [<a href="https://doi.org/10.3390/ijerph191811362" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9517444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36141639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kazimierowicz%20J,%20Zieli%C5%84ski%20M,%20Bartkowska%20I,%20D%C4%99bowski%20M%20(2022)%20Effect%20of%20acid%20whey%20pretreatment%20using%20ultrasonic%20disintegration%20on%20the%20removal%20of%20organic%20compounds%20and%20anaerobic%20digestion%20efficiency.%20Int%20J%20Environ%20Res%20Public%20Health%2019:11362.%2010.3390/IJERPH191811362" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<cite>Kiryu T, Kiso T, Nakano H, Murakami H (2015) Lactobionic and cellobionic acid production profiles of the resting cells of acetic acid bacteria. Biosci Biotechnol Biochem 79:1712–1718. 10.1080/09168451.2015.1038214
</cite> [<a href="https://doi.org/10.1080/09168451.2015.1038214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25965080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kiryu%20T,%20Kiso%20T,%20Nakano%20H,%20Murakami%20H%20(2015)%20Lactobionic%20and%20cellobionic%20acid%20production%20profiles%20of%20the%20resting%20cells%20of%20acetic%20acid%20bacteria.%20Biosci%20Biotechnol%20Biochem%2079:1712%E2%80%931718.%2010.1080/09168451.2015.1038214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<cite>Kiryu T, Kiso T, Koma D, Tanaka S, Murakami H (2019) Identifying membrane-bound quinoprotein glucose dehydrogenase from acetic acid bacteria that produce lactobionic and cellobionic acids. Biosci Biotechnol Biochem 83:1171–1179. 10.1080/09168451.2019.1580136
</cite> [<a href="https://doi.org/10.1080/09168451.2019.1580136" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30777491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kiryu%20T,%20Kiso%20T,%20Koma%20D,%20Tanaka%20S,%20Murakami%20H%20(2019)%20Identifying%20membrane-bound%20quinoprotein%20glucose%20dehydrogenase%20from%20acetic%20acid%20bacteria%20that%20produce%20lactobionic%20and%20cellobionic%20acids.%20Biosci%20Biotechnol%20Biochem%2083:1171%E2%80%931179.%2010.1080/09168451.2019.1580136" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<cite>Kolesovs S, Semjonovs P (2020) Production of bacterial cellulose from whey—current state and prospects. Appl Microbiol Biotechnol 104:7723–7730. 10.1007/s00253-020-10803-9
</cite> [<a href="https://doi.org/10.1007/s00253-020-10803-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32761463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kolesovs%20S,%20Semjonovs%20P%20(2020)%20Production%20of%20bacterial%20cellulose%20from%20whey%E2%80%94current%20state%20and%20prospects.%20Appl%20Microbiol%20Biotechnol%20104:7723%E2%80%937730.%2010.1007/s00253-020-10803-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<cite>Kumar V, Sharma DK, Sandhu PP, Jadaun J, Sangwan RS, Yadav SK (2021) Sustainable process for the production of cellulose by an <em>Acetobacter pasteurianus</em> RSV-4 (MTCC 25117) on whey medium. Cellulose 28:103–116. 10.1007/S10570-020-03519-6</cite> [<a href="https://scholar.google.com/scholar_lookup?Kumar%20V,%20Sharma%20DK,%20Sandhu%20PP,%20Jadaun%20J,%20Sangwan%20RS,%20Yadav%20SK%20(2021)%20Sustainable%20process%20for%20the%20production%20of%20cellulose%20by%20an%20Acetobacter%20pasteurianus%20RSV-4%20(MTCC%2025117)%20on%20whey%20medium.%20Cellulose%2028:103%E2%80%93116.%2010.1007/S10570-020-03519-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<cite>Kumaravel A, Lin SP, Santoso SP, Shanmugasundaram S, Hsu HY, Hsieh CW, Chou YC, Lin HW, Cheng KC (2025) Unlocking the potential of bacterial cellulose: synthesis, functionalization, and industrial impact. Int J Biol Macromol 311:143951. 10.1016/J.IJBIOMAC.2025.143951
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2025.143951" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40339858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumaravel%20A,%20Lin%20SP,%20Santoso%20SP,%20Shanmugasundaram%20S,%20Hsu%20HY,%20Hsieh%20CW,%20Chou%20YC,%20Lin%20HW,%20Cheng%20KC%20(2025)%20Unlocking%20the%20potential%20of%20bacterial%20cellulose:%20synthesis,%20functionalization,%20and%20industrial%20impact.%20Int%20J%20Biol%20Macromol%20311:143951.%2010.1016/J.IJBIOMAC.2025.143951" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<cite>Lappa IK, Kachrimanidou V, Papadaki A, Stamatiou A, Ladakis D, Eriotou E, Kopsahelis N (2021) A comprehensive bioprocessing approach to foster cheese whey valorization: on-site β-galactosidase secretion for lactose hydrolysis and sequential bacterial cellulose production. Fermentation 7:184. 10.3390/fermentation7030184</cite> [<a href="https://scholar.google.com/scholar_lookup?Lappa%20IK,%20Kachrimanidou%20V,%20Papadaki%20A,%20Stamatiou%20A,%20Ladakis%20D,%20Eriotou%20E,%20Kopsahelis%20N%20(2021)%20A%20comprehensive%20bioprocessing%20approach%20to%20foster%20cheese%20whey%20valorization:%20on-site%20%CE%B2-galactosidase%20secretion%20for%20lactose%20hydrolysis%20and%20sequential%20bacterial%20cellulose%20production.%20Fermentation%207:184.%2010.3390/fermentation7030184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<cite>Li Q, Gao R, Wang L, Xu M, Yuan Y, Ma L, Wan Z, Yang X (2020) Nanocomposites of bacterial cellulose nanofibrils and zein nanoparticles for food packaging. ACS Appl Nano Mater 3:2899–2910. 10.1021/ACSANM.0C00159</cite> [<a href="https://scholar.google.com/scholar_lookup?Li%20Q,%20Gao%20R,%20Wang%20L,%20Xu%20M,%20Yuan%20Y,%20Ma%20L,%20Wan%20Z,%20Yang%20X%20(2020)%20Nanocomposites%20of%20bacterial%20cellulose%20nanofibrils%20and%20zein%20nanoparticles%20for%20food%20packaging.%20ACS%20Appl%20Nano%20Mater%203:2899%E2%80%932910.%2010.1021/ACSANM.0C00159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<cite>Li W, Huang X, Liu H, Lian H, Xu B, Zhang W, Sun X, Wang W, Jia S, Zhong C (2023) Improvement in bacterial cellulose production by co-culturing <em>Bacillus cereus</em> and <em>Komagataeibacter xylinus</em>. Carbohydr Polym 313:120892. 10.1016/J.CARBPOL.2023.120892
</cite> [<a href="https://doi.org/10.1016/j.carbpol.2023.120892" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37182977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20W,%20Huang%20X,%20Liu%20H,%20Lian%20H,%20Xu%20B,%20Zhang%20W,%20Sun%20X,%20Wang%20W,%20Jia%20S,%20Zhong%20C%20(2023)%20Improvement%20in%20bacterial%20cellulose%20production%20by%20co-culturing%20Bacillus%20cereus%20and%20Komagataeibacter%20xylinus.%20Carbohydr%20Polym%20313:120892.%2010.1016/J.CARBPOL.2023.120892" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<cite>Lin SP, Singajaya S, Lo TY, Santoso SP, Hsu HY, Cheng KC (2023) Evaluation of porous bacterial cellulose produced from foam templating with different additives and its application in 3D cell culture. Int J Biol Macromol 234:123680. 10.1016/J.IJBIOMAC.2023.123680
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2023.123680" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36801225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20SP,%20Singajaya%20S,%20Lo%20TY,%20Santoso%20SP,%20Hsu%20HY,%20Cheng%20KC%20(2023)%20Evaluation%20of%20porous%20bacterial%20cellulose%20produced%20from%20foam%20templating%20with%20different%20additives%20and%20its%20application%20in%203D%20cell%20culture.%20Int%20J%20Biol%20Macromol%20234:123680.%2010.1016/J.IJBIOMAC.2023.123680" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<cite>Liu K, Catchmark JM (2018) Effects of exopolysaccharides from <em>Escherichia coli</em> ATCC 35860 on the mechanical properties of bacterial cellulose nanocomposites. Cellulose 25:2273–2287. 10.1007/S10570-018-1709-3</cite> [<a href="https://scholar.google.com/scholar_lookup?Liu%20K,%20Catchmark%20JM%20(2018)%20Effects%20of%20exopolysaccharides%20from%20Escherichia%20coli%20ATCC%2035860%20on%20the%20mechanical%20properties%20of%20bacterial%20cellulose%20nanocomposites.%20Cellulose%2025:2273%E2%80%932287.%2010.1007/S10570-018-1709-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<cite>Liu K, Catchmark JM (2019) Enhanced mechanical properties of bacterial cellulose nanocomposites produced by co-culturing <em>Gluconacetobacter hansenii</em> and <em>Escherichia coli</em> under static conditions. Carbohydr Polym 219:12–20. 10.1016/J.CARBPOL.2019.04.071
</cite> [<a href="https://doi.org/10.1016/j.carbpol.2019.04.071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31151508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20K,%20Catchmark%20JM%20(2019)%20Enhanced%20mechanical%20properties%20of%20bacterial%20cellulose%20nanocomposites%20produced%20by%20co-culturing%20Gluconacetobacter%20hansenii%20and%20Escherichia%20coli%20under%20static%20conditions.%20Carbohydr%20Polym%20219:12%E2%80%9320.%2010.1016/J.CARBPOL.2019.04.071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<cite>Liu M, Liu L, Jia S, Li S, Zou Y, Zhong C (2018) Complete genome analysis of <em>Gluconacetobacter xylinus</em> CGMCC 2955 for elucidating bacterial cellulose biosynthesis and metabolic regulation. Sci Rep 8:1–10. 10.1038/s41598-018-24559-w
</cite> [<a href="https://doi.org/10.1038/s41598-018-24559-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5908849/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29674724/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20M,%20Liu%20L,%20Jia%20S,%20Li%20S,%20Zou%20Y,%20Zhong%20C%20(2018)%20Complete%20genome%20analysis%20of%20Gluconacetobacter%20xylinus%20CGMCC%202955%20for%20elucidating%20bacterial%20cellulose%20biosynthesis%20and%20metabolic%20regulation.%20Sci%20Rep%208:1%E2%80%9310.%2010.1038/s41598-018-24559-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<cite> Ma’unatin A, Harijono H, Zubaidah E, Rifa’i M (2020) The isolation of exopolysaccharide-producing lactic acid bacteria from lontar (<em>Borassus flabellifer</em> L.) sap. Iran J Microbiol 12:437. 10.18502/IJM.V12I5.4605</cite> [<a href="https://doi.org/10.18502/ijm.v12i5.4605" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7867694/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33603999/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<cite>Matsuoka M, Tsuchida T, Matsushita K, Adachi O, Yoshinaga F (1996) A synthetic medium for bacterial cellulose production by <em>Acetobacter xylinum</em> subsp. <em>sucrofermentans</em>. Biosci Biotechnol Biochem 60:575–579. 10.1271/BBB.60.575</cite> [<a href="https://scholar.google.com/scholar_lookup?Matsuoka%20M,%20Tsuchida%20T,%20Matsushita%20K,%20Adachi%20O,%20Yoshinaga%20F%20(1996)%20A%20synthetic%20medium%20for%20bacterial%20cellulose%20production%20by%20Acetobacter%20xylinum%20subsp.%20sucrofermentans.%20Biosci%20Biotechnol%20Biochem%2060:575%E2%80%93579.%2010.1271/BBB.60.575" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<cite>Menchik P, Zuber T, Zuber A, Moraru CI (2019) Short communication: composition of coproduct streams from dairy processing: acid whey and milk permeate. J Dairy Sci 102:3978–3984. 10.3168/JDS.2018-15951
</cite> [<a href="https://doi.org/10.3168/jds.2018-15951" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30879808/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Menchik%20P,%20Zuber%20T,%20Zuber%20A,%20Moraru%20CI%20(2019)%20Short%20communication:%20composition%20of%20coproduct%20streams%20from%20dairy%20processing:%20acid%20whey%20and%20milk%20permeate.%20J%20Dairy%20Sci%20102:3978%E2%80%933984.%2010.3168/JDS.2018-15951" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<cite>Mollica D, Thanthithum C, Chaves Lopez C, Prakitchaiwattana C (2024) Bioconversion of mangosteen pericarp extract juice with bacterial cellulose production and probiotic integration: a promising approach for functional food supplement development. Food Biosci 61:104549. 10.1016/J.FBIO.2024.104549</cite> [<a href="https://scholar.google.com/scholar_lookup?Mollica%20D,%20Thanthithum%20C,%20Chaves%20Lopez%20C,%20Prakitchaiwattana%20C%20(2024)%20Bioconversion%20of%20mangosteen%20pericarp%20extract%20juice%20with%20bacterial%20cellulose%20production%20and%20probiotic%20integration:%20a%20promising%20approach%20for%20functional%20food%20supplement%20development.%20Food%20Biosci%2061:104549.%2010.1016/J.FBIO.2024.104549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<cite>Murari CS, da Silva DCMN, Schuina GL, Mosinahti EF, Del Bianchi VL (2019) Bioethanol production from dairy industrial coproducts. Bioenergy Res 12:112–122. 10.1007/S12155-018-9949-5</cite> [<a href="https://scholar.google.com/scholar_lookup?Murari%20CS,%20da%20Silva%20DCMN,%20Schuina%20GL,%20Mosinahti%20EF,%20Del%20Bianchi%20VL%20(2019)%20Bioethanol%20production%20from%20dairy%20industrial%20coproducts.%20Bioenergy%20Res%2012:112%E2%80%93122.%2010.1007/S12155-018-9949-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<cite>Panesar PS, Kennedy JF, Gandhi DN, Bunko K (2007) Bioutilisation of whey for lactic acid production. Food Chem 105:1–14. 10.1016/J.FOODCHEM.2007.03.035</cite> [<a href="https://scholar.google.com/scholar_lookup?Panesar%20PS,%20Kennedy%20JF,%20Gandhi%20DN,%20Bunko%20K%20(2007)%20Bioutilisation%20of%20whey%20for%20lactic%20acid%20production.%20Food%20Chem%20105:1%E2%80%9314.%2010.1016/J.FOODCHEM.2007.03.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<cite>Pawlos M, Znamirowska-Piotrowska A, Kowalczyk M, Zaguła G, Szajnar K (2023) Possibility of using different calcium compounds for the manufacture of fresh acid rennet cheese from goat’s milk. Foods 12:3703. 10.3390/FOODS12193703
</cite> [<a href="https://doi.org/10.3390/foods12193703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10572428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37835357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pawlos%20M,%20Znamirowska-Piotrowska%20A,%20Kowalczyk%20M,%20Zagu%C5%82a%20G,%20Szajnar%20K%20(2023)%20Possibility%20of%20using%20different%20calcium%20compounds%20for%20the%20manufacture%20of%20fresh%20acid%20rennet%20cheese%20from%20goat%E2%80%99s%20milk.%20Foods%2012:3703.%2010.3390/FOODS12193703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<cite>Płoska J, Garbowska M, Klempová S, Stasiak-Różańska L (2023) Obtaining bacterial cellulose through selected strains of acetic acid bacteria in classical and waste media. Appl Sci. 10.3390/app13116429</cite> [<a href="https://scholar.google.com/scholar_lookup?P%C5%82oska%20J,%20Garbowska%20M,%20Klempov%C3%A1%20S,%20Stasiak-R%C3%B3%C5%BCa%C5%84ska%20L%20(2023)%20Obtaining%20bacterial%20cellulose%20through%20selected%20strains%20of%20acetic%20acid%20bacteria%20in%20classical%20and%20waste%20media.%20Appl%20Sci.%2010.3390/app13116429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<cite>Płoska J, Garbowska M, Pluta A, Stasiak-Różańska L (2023) Bacterial cellulose – innovative biopolymer and possibilities of its applications in dairy industry. Int Dairy J 140:105586. 10.1016/J.IDAIRYJ.2023.105586</cite> [<a href="https://scholar.google.com/scholar_lookup?P%C5%82oska%20J,%20Garbowska%20M,%20Pluta%20A,%20Stasiak-R%C3%B3%C5%BCa%C5%84ska%20L%20(2023)%20Bacterial%20cellulose%20%E2%80%93%20innovative%20biopolymer%20and%20possibilities%20of%20its%20applications%20in%20dairy%20industry.%20Int%20Dairy%20J%20140:105586.%2010.1016/J.IDAIRYJ.2023.105586" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<cite>Płoska J, Garbowska M, Rybak K, Berthold-Pluta A, Stasiak-Różańska L (2024) Study on application of biocellulose-based material for cheese packaging. Int J Biol Macromol 264:130433. 10.1016/J.IJBIOMAC.2024.130433
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2024.130433" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38408577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?P%C5%82oska%20J,%20Garbowska%20M,%20Rybak%20K,%20Berthold-Pluta%20A,%20Stasiak-R%C3%B3%C5%BCa%C5%84ska%20L%20(2024)%20Study%20on%20application%20of%20biocellulose-based%20material%20for%20cheese%20packaging.%20Int%20J%20Biol%20Macromol%20264:130433.%2010.1016/J.IJBIOMAC.2024.130433" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<cite>Pudlik AM, Lolkema JS (2011) Mechanism of citrate metabolism by an oxaloacetate decarboxylase-deficient mutant of <em>Lactococcus lactis</em> IL1403. J Bacteriol 193:4049–4056
</cite> [<a href="https://doi.org/10.1128/JB.05012-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3147662/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21665973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pudlik%20AM,%20Lolkema%20JS%20(2011)%20Mechanism%20of%20citrate%20metabolism%20by%20an%20oxaloacetate%20decarboxylase-deficient%20mutant%20of%20Lactococcus%20lactis%20IL1403.%20J%20Bacteriol%20193:4049%E2%80%934056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<cite>Qiao W, Jia C, Yang J, Gao G, Guo D, Xu X, Wu Z, Saris PEJ, Xu H, Qiao M (2024) Production of bacterial cellulose-based peptidopolysaccharide BC-L with anti-listerial properties using a co-cultivation strategy. Int J Biol Macromol 274:133047. 10.1016/J.IJBIOMAC.2024.133047
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2024.133047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38857722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qiao%20W,%20Jia%20C,%20Yang%20J,%20Gao%20G,%20Guo%20D,%20Xu%20X,%20Wu%20Z,%20Saris%20PEJ,%20Xu%20H,%20Qiao%20M%20(2024)%20Production%20of%20bacterial%20cellulose-based%20peptidopolysaccharide%20BC-L%20with%20anti-listerial%20properties%20using%20a%20co-cultivation%20strategy.%20Int%20J%20Biol%20Macromol%20274:133047.%2010.1016/J.IJBIOMAC.2024.133047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55"><cite>Quintans NG, Blancato V, Repizo GC, Magni C, Lopez P (2008) Citrate metabolism and aroma compound production in lactic acid bacteria. In: Mayo B, López P, Pérez-Martínez G (eds.) Molecular aspects of lactic acid bacteria for traditional and new applications, 1st edn. Research Signpost, Trivandrum, 65–88</cite></li>
<li id="CR56">
<cite>Raiszadeh-Jahromi Y, Rezazadeh-Bari M, Almasi H, Amiri S (2020) Optimization of bacterial cellulose production by <em>Komagataeibacter xylinus</em> PTCC 1734 in a low-cost medium using optimal combined design. J Food Sci Technol 57:2524–2533. 10.1007/S13197-020-04289-6
</cite> [<a href="https://doi.org/10.1007/s13197-020-04289-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7271089/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32549603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raiszadeh-Jahromi%20Y,%20Rezazadeh-Bari%20M,%20Almasi%20H,%20Amiri%20S%20(2020)%20Optimization%20of%20bacterial%20cellulose%20production%20by%20Komagataeibacter%20xylinus%20PTCC%201734%20in%20a%20low-cost%20medium%20using%20optimal%20combined%20design.%20J%20Food%20Sci%20Technol%2057:2524%E2%80%932533.%2010.1007/S13197-020-04289-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<cite>Razavi MS, Golmohammadi A, Nematollahzadeh A, Fiori F, Rovera C, Farris S (2020) Preparation of cinnamon essential oil emulsion by bacterial cellulose nanocrystals and fish gelatin. Food Hydrocolloids 109:106111. 10.1016/J.FOODHYD.2020.106111</cite> [<a href="https://scholar.google.com/scholar_lookup?Razavi%20MS,%20Golmohammadi%20A,%20Nematollahzadeh%20A,%20Fiori%20F,%20Rovera%20C,%20Farris%20S%20(2020)%20Preparation%20of%20cinnamon%20essential%20oil%20emulsion%20by%20bacterial%20cellulose%20nanocrystals%20and%20fish%20gelatin.%20Food%20Hydrocolloids%20109:106111.%2010.1016/J.FOODHYD.2020.106111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<cite>Revin V, Liyaskina E, Nazarkina M, Bogatyreva A, Shchankin M (2018) Cost-effective production of bacterial cellulose using acidic food industry by-products. Braz J Microbiol 49:151–159. 10.1016/j.bjm.2017.12.012
</cite> [<a href="https://doi.org/10.1016/j.bjm.2017.12.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6328854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29703527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Revin%20V,%20Liyaskina%20E,%20Nazarkina%20M,%20Bogatyreva%20A,%20Shchankin%20M%20(2018)%20Cost-effective%20production%20of%20bacterial%20cellulose%20using%20acidic%20food%20industry%20by-products.%20Braz%20J%20Microbiol%2049:151%E2%80%93159.%2010.1016/j.bjm.2017.12.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<cite>Sampaio FC, de Faria JT, da Silva MF, de Souza Oliveira RP, Converti A (2020) Cheese whey permeate fermentation by <em>Kluyveromyces lactis</em>: a combined approach to wastewater treatment and bioethanol production. Environ Technol 41:3210–3218. 10.1080/09593330.2019.1604813
</cite> [<a href="https://doi.org/10.1080/09593330.2019.1604813" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30955482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sampaio%20FC,%20de%20Faria%20JT,%20da%20Silva%20MF,%20de%20Souza%20Oliveira%20RP,%20Converti%20A%20(2020)%20Cheese%20whey%20permeate%20fermentation%20by%20Kluyveromyces%20lactis:%20a%20combined%20approach%20to%20wastewater%20treatment%20and%20bioethanol%20production.%20Environ%20Technol%2041:3210%E2%80%933218.%2010.1080/09593330.2019.1604813" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<cite>Seto A, Saito Y, Matsushige M, Kobayashi H, Sasaki Y, Tonouchi N, Tsuchida T, Yoshinaga F, Ueda K, Beppu T (2006) Effective cellulose production by a coculture of <em>Gluconacetobacter xylinus</em>/ and <em>Lactobacillus mali</em>. Appl Microbiol Biotechnol 73:915–921. 10.1007/S00253-006-0515-2
</cite> [<a href="https://doi.org/10.1007/s00253-006-0515-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17093989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seto%20A,%20Saito%20Y,%20Matsushige%20M,%20Kobayashi%20H,%20Sasaki%20Y,%20Tonouchi%20N,%20Tsuchida%20T,%20Yoshinaga%20F,%20Ueda%20K,%20Beppu%20T%20(2006)%20Effective%20cellulose%20production%20by%20a%20coculture%20of%20Gluconacetobacter%20xylinus/%20and%20Lactobacillus%20mali.%20Appl%20Microbiol%20Biotechnol%2073:915%E2%80%93921.%2010.1007/S00253-006-0515-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<cite>Sharma P, Sharma R, Ahuja S, Yadav A, Arora S, Aggarwal NK (2023) Enhancement of bacterial cellulose production by ethanol and lactic acid by using <em>Gluconacetobacter kombuchae</em>. Prep Biochem Biotechnol. 10.1080/10826068.2023.2276188
</cite> [<a href="https://doi.org/10.1080/10826068.2023.2276188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37937534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharma%20P,%20Sharma%20R,%20Ahuja%20S,%20Yadav%20A,%20Arora%20S,%20Aggarwal%20NK%20(2023)%20Enhancement%20of%20bacterial%20cellulose%20production%20by%20ethanol%20and%20lactic%20acid%20by%20using%20Gluconacetobacter%20kombuchae.%20Prep%20Biochem%20Biotechnol.%2010.1080/10826068.2023.2276188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<cite>Singhsa P, Narain R, Manuspiya H (2018) Physical structure variations of bacterial cellulose produced by different <em>Komagataeibacter xylinus</em> strains and carbon sources in static and agitated conditions. Cellulose 25:1571–1581. 10.1007/S10570-018-1699-1</cite> [<a href="https://scholar.google.com/scholar_lookup?Singhsa%20P,%20Narain%20R,%20Manuspiya%20H%20(2018)%20Physical%20structure%20variations%20of%20bacterial%20cellulose%20produced%20by%20different%20Komagataeibacter%20xylinus%20strains%20and%20carbon%20sources%20in%20static%20and%20agitated%20conditions.%20Cellulose%2025:1571%E2%80%931581.%2010.1007/S10570-018-1699-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<cite>Usenik V, Fabčič J, Štampar F (2008) Sugars, organic acids, phenolic composition and antioxidant activity of sweet cherry (<em>Prunus avium</em> L.). Food Chem 107:185–192. 10.1016/J.FOODCHEM.2007.08.004</cite> [<a href="https://scholar.google.com/scholar_lookup?Usenik%20V,%20Fab%C4%8Di%C4%8D%20J,%20%C5%A0tampar%20F%20(2008)%20Sugars,%20organic%20acids,%20phenolic%20composition%20and%20antioxidant%20activity%20of%20sweet%20cherry%20(Prunus%20avium%20L.).%20Food%20Chem%20107:185%E2%80%93192.%2010.1016/J.FOODCHEM.2007.08.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<cite>Uysal U, Hamamcı H (2021) Succinic acid production from cheese whey via fermentation by using alginate immobilized <em>Actinobacillus succinogenes</em>. Bioresour Technol Rep 16:100829. 10.1016/J.BITEB.2021.100829</cite> [<a href="https://scholar.google.com/scholar_lookup?Uysal%20U,%20Hamamc%C4%B1%20H%20(2021)%20Succinic%20acid%20production%20from%20cheese%20whey%20via%20fermentation%20by%20using%20alginate%20immobilized%20Actinobacillus%20succinogenes.%20Bioresour%20Technol%20Rep%2016:100829.%2010.1016/J.BITEB.2021.100829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<cite>Wang SS, Han YH, Chen JL, Zhang DC, Shi XX, Ye YX, Chen DL, Li M (2018) Insights into bacterial cellulose biosynthesis from different carbon sources and the associated biochemical transformation pathways in <em>Komagataeibacter</em> sp. W1. Polymers 10:963. 10.3390/POLYM10090963
</cite> [<a href="https://doi.org/10.3390/polym10090963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6403882/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30960888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20SS,%20Han%20YH,%20Chen%20JL,%20Zhang%20DC,%20Shi%20XX,%20Ye%20YX,%20Chen%20DL,%20Li%20M%20(2018)%20Insights%20into%20bacterial%20cellulose%20biosynthesis%20from%20different%20carbon%20sources%20and%20the%20associated%20biochemical%20transformation%20pathways%20in%20Komagataeibacter%20sp.%20W1.%20Polymers%2010:963.%2010.3390/POLYM10090963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<cite>Xia M, Zhang X, Xiao Y, Sheng Q, Tu L, Chen F, Yan Y, Zheng Y, Wang M (2022) Interaction of acetic acid bacteria and lactic acid bacteria in multispecies solid-state fermentation of traditional Chinese cereal vinegar. Front Microbiol 13:964855. 10.3389/FMICB.2022.964855
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.964855" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9557190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36246224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia%20M,%20Zhang%20X,%20Xiao%20Y,%20Sheng%20Q,%20Tu%20L,%20Chen%20F,%20Yan%20Y,%20Zheng%20Y,%20Wang%20M%20(2022)%20Interaction%20of%20acetic%20acid%20bacteria%20and%20lactic%20acid%20bacteria%20in%20multispecies%20solid-state%20fermentation%20of%20traditional%20Chinese%20cereal%20vinegar.%20Front%20Microbiol%2013:964855.%2010.3389/FMICB.2022.964855" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<cite>Xing J, Wang G, Zhang Q, Liu X, Gu Z, Zhang H, Chen YQ, Chen W (2015) Determining antioxidant activities of lactobacilli cell-free supernatants by cellular antioxidant assay: a comparison with traditional methods. PLoS One 10:1–16. 10.1371/journal.pone.0119058</cite> [<a href="https://doi.org/10.1371/journal.pone.0119058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4366247/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25789875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xing%20J,%20Wang%20G,%20Zhang%20Q,%20Liu%20X,%20Gu%20Z,%20Zhang%20H,%20Chen%20YQ,%20Chen%20W%20(2015)%20Determining%20antioxidant%20activities%20of%20lactobacilli%20cell-free%20supernatants%20by%20cellular%20antioxidant%20assay:%20a%20comparison%20with%20traditional%20methods.%20PLoS%20One%2010:1%E2%80%9316.%2010.1371/journal.pone.0119058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<cite>Zhang Y, Hu P, Lou L, Zhan J, Fan M, Li D, Liao Q (2017) Antioxidant activities of lactic acid bacteria for quality improvement of fermented sausage. J Food Sci 82:2960–2967. 10.1111/1750-3841.13975
</cite> [<a href="https://doi.org/10.1111/1750-3841.13975" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29131334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Y,%20Hu%20P,%20Lou%20L,%20Zhan%20J,%20Fan%20M,%20Li%20D,%20Liao%20Q%20(2017)%20Antioxidant%20activities%20of%20lactic%20acid%20bacteria%20for%20quality%20improvement%20of%20fermented%20sausage.%20J%20Food%20Sci%2082:2960%E2%80%932967.%2010.1111/1750-3841.13975" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<cite>Zhong C, Zhang GC, Liu M, Zheng XT, Han PP, Jia SR (2013) Metabolic flux analysis of <em>Gluconacetobacter xylinus</em> for bacterial cellulose production. Appl Microbiol Biotechnol 97:6189–6199. 10.1007/S00253-013-4908-8
</cite> [<a href="https://doi.org/10.1007/s00253-013-4908-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23640364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhong%20C,%20Zhang%20GC,%20Liu%20M,%20Zheng%20XT,%20Han%20PP,%20Jia%20SR%20(2013)%20Metabolic%20flux%20analysis%20of%20Gluconacetobacter%20xylinus%20for%20bacterial%20cellulose%20production.%20Appl%20Microbiol%20Biotechnol%2097:6189%E2%80%936199.%2010.1007/S00253-013-4908-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370864/bin/253_2025_13582_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link"><span>MOESM 1</span></a><sup> (10.5MB, docx) </sup><p>(DOCX 10,789 KB)</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets and materials from this study can be available upon request to interested researchers.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Applied Microbiology and Biotechnology are provided here courtesy of <strong>Springer</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1007/s00253-025-13582-3"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/253_2025_Article_13582.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370864/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370864/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370864%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370864/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370864/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370864/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40839214/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370864/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40839214/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370864/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370864/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="FNJ7ALwRq8vI3lm1yNZtA0DnLjDChQMP3WcNnHPDPpq7x8fgTjYi3aD7j83jemQx">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
