
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            The role of cyclic di-GMP in biomaterial-associated infections caused by commensal Escherichia coli - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152547D98AF352D30547D90014F430EE.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosone">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367115/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS One">
<meta name="citation_title" content="The role of cyclic di-GMP in biomaterial-associated infections caused by commensal Escherichia coli">
<meta name="citation_author" content="Shanshan Li">
<meta name="citation_author_institution" content="Department of Anesthesiology, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China">
<meta name="citation_author_institution" content="Department of Thoracic Surgery I, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China">
<meta name="citation_author" content="Chengwei Bi">
<meta name="citation_author_institution" content="Department of Thoracic Surgery I, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China">
<meta name="citation_author_institution" content="Department of Urinary Surgery, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China">
<meta name="citation_author" content="Bingquan Xiang">
<meta name="citation_author_institution" content="Intensity Care Unit, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China">
<meta name="citation_author" content="Zhonghui Wang">
<meta name="citation_author_institution" content="Department of Anesthesiology, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China">
<meta name="citation_author" content="Hui Yang">
<meta name="citation_author_institution" content="Department of Anesthesiology, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China">
<meta name="citation_author" content="Chaojiang Fu">
<meta name="citation_author_institution" content="Department of Emergency Medicine, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China">
<meta name="citation_author" content="Lianpu Chen">
<meta name="citation_author_institution" content="Department of Anesthesiology, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China">
<meta name="citation_author" content="Ying Chen">
<meta name="citation_author_institution" content="Department of Thoracic Surgery I, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="20">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0330229">
<meta name="citation_doi" content="10.1371/journal.pone.0330229">
<meta name="citation_pmid" content="40833985">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367115/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367115/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367115/pdf/pone.0330229.pdf">
<meta name="description" content="Biofilms are protective structures that bacteria use to evade the immune system and resist antibiotics, leading to complications in medical treatments, especially with implanted devices. The molecule cyclic di-GMP (c-di-GMP) is crucial for biofilm ...">
<meta name="og:title" content="The role of cyclic di-GMP in biomaterial-associated infections caused by commensal Escherichia coli">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Biofilms are protective structures that bacteria use to evade the immune system and resist antibiotics, leading to complications in medical treatments, especially with implanted devices. The molecule cyclic di-GMP (c-di-GMP) is crucial for biofilm ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367115/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12367115">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pone.0330229"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pone.0330229.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367115%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12367115/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12367115/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367115/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosone.png" alt="PLOS One logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS One" title="Link to PLOS One" shape="default" href="https://doi.org/10.1371/journal.pone.0330229" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS One</button></div>. 2025 Aug 20;20(8):e0330229. doi: <a href="https://doi.org/10.1371/journal.pone.0330229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330229</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20One%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20One%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>The role of cyclic di-GMP in biomaterial-associated infections caused by commensal <em>Escherichia coli</em>
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Shanshan Li</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Shanshan Li</span></h3>
<div class="p">
<sup>1</sup>Department of Anesthesiology, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div class="p">
<sup>2</sup>Department of Thoracic Surgery I, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div>Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shanshan Li</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bi%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Chengwei Bi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Chengwei Bi</span></h3>
<div class="p">
<sup>2</sup>Department of Thoracic Surgery I, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div class="p">
<sup>3</sup>Department of Urinary Surgery, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div>Conceptualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bi%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chengwei Bi</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiang%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Bingquan Xiang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Bingquan Xiang</span></h3>
<div class="p">
<sup>4</sup>Intensity Care Unit, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div>Data curation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiang%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bingquan Xiang</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Zhonghui Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Zhonghui Wang</span></h3>
<div class="p">
<sup>1</sup>Department of Anesthesiology, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div>Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhonghui Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Hui Yang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Hui Yang</span></h3>
<div class="p">
<sup>1</sup>Department of Anesthesiology, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hui Yang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Chaojiang Fu</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Chaojiang Fu</span></h3>
<div class="p">
<sup>5</sup>Department of Emergency Medicine, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div>Methodology</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chaojiang Fu</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Lianpu Chen</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Lianpu Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Anesthesiology, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div>Project administration</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lianpu Chen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Ying Chen</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Ying Chen</span></h3>
<div class="p">
<sup>2</sup>Department of Thoracic Surgery I, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div>Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Ying Chen</span></a>
</div>
</div>
<sup>2,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Geelsu Hwang</span><sup>6</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Department of Anesthesiology, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div id="aff002">
<sup>2</sup>Department of Thoracic Surgery I, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div id="aff003">
<sup>3</sup>Department of Urinary Surgery, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div id="aff004">
<sup>4</sup>Intensity Care Unit, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div id="aff005">
<sup>5</sup>Department of Emergency Medicine, The Third Affiliated Hospital of Kunming Medical University, Yunnan Cancer Hospital, Peking University Cancer Hospital Yunnan, Kunming, China</div>
<div id="edit1">
<sup>6</sup>University of Pennsylvania, UNITED STATES OF AMERICA</div>
<div class="author-notes p">
<div class="fn" id="coi001"><p><strong>Competing Interests: </strong>The authors have declared that no competing interests exist.</p></div>
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>36410008@qq.com</span></p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Shanshan Li</span></strong>: <span class="role">Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Chengwei Bi</span></strong>: <span class="role">Conceptualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Bingquan Xiang</span></strong>: <span class="role">Data curation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Zhonghui Wang</span></strong>: <span class="role">Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Hui Yang</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Chaojiang Fu</span></strong>: <span class="role">Methodology</span>
</div>
<div>
<strong class="contrib"><span class="name western">Lianpu Chen</span></strong>: <span class="role">Project administration</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ying Chen</span></strong>: <span class="role">Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Geelsu Hwang</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Nov 6; Accepted 2025 Jul 28; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Li et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12367115  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40833985/" class="usa-link">40833985</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Biofilms are protective structures that bacteria use to evade the immune system and resist antibiotics, leading to complications in medical treatments, especially with implanted devices. The molecule cyclic di-GMP (c-di-GMP) is crucial for biofilm formation in <em>Escherichia coli</em> (<em>E. coli</em>). To understand its role in biomaterial-associated infections (BAIs), we created four <em>E. coli</em> strains with varying c-di-GMP levels: a knockout strain (Δ<em>dgcQ</em>), an overexpression strain (O<em>dgcQ</em>), a complemented strain (CΔ<em>dgcQ</em>), and a wild-type mutant strain (WT). By employing <em>in vitro</em> BAI models and techniques such as crystal violet (CV) staining, XTT assay, confocal laser scanning microscopy (CLSM), and scanning electron microscopy (SEM), we observed that the Δ<em>dgcQ</em> strain, with low c-di-GMP levels, adhered more readily to biomaterial surfaces at the initial stage of biofilm formation, yet faced difficulties in sustaining mature biofilms. In contrast, O<em>dgcQ</em> and CΔ<em>dgcQ</em> with higher c-di-GMP were able to generate more mature biofilms on biomaterial surfaces. Additionally, c-di-GMP was found to negatively regulate bacterial swimming motility and enhance the ability to cope with environmental stresses. The results also reiterate the canonical function of c-di-GMP, which is to reduce the motility of bacteria. Concurrently, gene expression analysis confirmed these findings, revealing that genes related to motility (<em>flhC</em>, <em>flhD</em>, <em>motA</em>, <em>motB</em>, <em>ycgR</em>), extracellular polymeric substances (EPS) synthesis (<em>csgA</em>, <em>csgD</em>, <em>bcsA</em>, <em>ynfM</em>), and stress resistance (<em>sodA</em>, <em>katE</em>, <em>rstA</em>, <em>ibpA</em>, <em>ibpB</em>, <em>hdeA</em>, <em>hdeD</em>, <em>gadA</em>, <em>gadB</em>) were consistently up-regulated in O<em>dgcQ</em> with high c-di-GMP levels. Importantly, Δ<em>dgcQ</em> considerably promoted the adhesion to and invasion of host cells and elicited a stronger host immune response, whereas O<em>dgcQ</em> impaired the ability to interact with host cells, as evidenced by decreased adhesion/invasion and inhibited release of inflammatory cytokines (IL-1β, IFN-β, IP-10, and NF-κB). Collectively, our findings shed light on the c-di-GMP signaling pathway's role in BAIs and propose that modulating this pathway could be a promising strategy for combating <em>E. coli</em>-induced BAIs.</p></section><section id="sec001"><h2 class="pmc_sec_title">1 Introduction</h2>
<p>Commensal <em>Escherichia coli</em> (<em>E. coli</em>) is not pathogenic under normal intestinal conditions. However, once it is transported across a compromised intestinal barrier (e.g., shock, trauma, or chronic disease [<a href="#pone.0330229.ref001" class="usa-link" aria-describedby="pone.0330229.ref001">1</a>–<a href="#pone.0330229.ref003" class="usa-link" aria-describedby="pone.0330229.ref003">3</a>]), it becomes a clinically important opportunistic pathogen, particularly in immunocompromised patients with implanted biomaterials. In the clinical environment, biomaterial-associated infections (BAIs) are one of the most common and severe complications [<a href="#pone.0330229.ref004" class="usa-link" aria-describedby="pone.0330229.ref004">4</a>,<a href="#pone.0330229.ref005" class="usa-link" aria-describedby="pone.0330229.ref005">5</a>]. In the USA, BAIs account for 25.6% of all hospital-associated infections [<a href="#pone.0330229.ref006" class="usa-link" aria-describedby="pone.0330229.ref006">6</a>]. These infections are particularly difficult to eradicate because biofilms on medical devices provide protection against host immune clearance and confer resistance to antibiotic therapy [<a href="#pone.0330229.ref007" class="usa-link" aria-describedby="pone.0330229.ref007">7</a>]. Notably, nosocomial infections caused by <em>E. coli</em> have shown a concerning increase from 10% in 2007 to 15% in 2014 [<a href="#pone.0330229.ref008" class="usa-link" aria-describedby="pone.0330229.ref008">8</a>,<a href="#pone.0330229.ref009" class="usa-link" aria-describedby="pone.0330229.ref009">9</a>]. This pathogen ranks among the most frequently isolated microorganisms in surgical implant-related infections, including catheter-associated urinary tract infections [<a href="#pone.0330229.ref010" class="usa-link" aria-describedby="pone.0330229.ref010">10</a>], central line-associated bloodstream infections [<a href="#pone.0330229.ref011" class="usa-link" aria-describedby="pone.0330229.ref011">11</a>], and ventilator-associated pneumonia [<a href="#pone.0330229.ref012" class="usa-link" aria-describedby="pone.0330229.ref012">12</a>].</p>
<p>The pathogenesis of BAIs centers on two critical processes: bacterial adhesion and biofilm formation [<a href="#pone.0330229.ref013" class="usa-link" aria-describedby="pone.0330229.ref013">13</a>]. Biomaterials act as substrates for initial bacterial adhesion, where <em>E. coli</em> transitions from reversible surface attachment (0–12 h) to irreversible colonization. Subsequent biofilm maturation (24–48 h) involves microcolony organization, extracellular polymeric substance (EPS) secretion, and 3D architectural assembly, forming protective microbial communities on implant surfaces [<a href="#pone.0330229.ref014" class="usa-link" aria-describedby="pone.0330229.ref014">14</a>]. Cyclic di-GMP (c-di-GMP) is a ubiquitous second messenger in Gram-negative bacteria, including <em>Pseudomonas aeruginosa</em>, <em>Pseudomonas putida</em>, <em>Vibrio vulnificus</em> and <em>E. coli</em>. This molecule is synthesized from two GTP molecules by diguanylate cyclases (DGCs, containing GGDEF domains) and degraded by phosphodiesterases (PDEs, featuring either EAL or HD-GYP domains) [<a href="#pone.0330229.ref015" class="usa-link" aria-describedby="pone.0330229.ref015">15</a>]. Substantial evidence indicates that c-di-GMP serves as the primary regulator of bacterial lifestyle switching from planktonic to sessile biofilm states [<a href="#pone.0330229.ref016" class="usa-link" aria-describedby="pone.0330229.ref016">16</a>,<a href="#pone.0330229.ref017" class="usa-link" aria-describedby="pone.0330229.ref017">17</a>]. In <em>E. coli</em> K-12, 29 genes encode proteins containing GGDEF and/or EAL domains, comprising 12 functional DGCs, 13 PDEs, and four enzymatically inactive “degenerate” proteins [<a href="#pone.0330229.ref018" class="usa-link" aria-describedby="pone.0330229.ref018">18</a>]. The <em>E. coli</em> ATCC25922 strain maintains an identical genetic repertoire for c-di-GMP metabolism. Among these components, <em>dgcQ</em> encodes a diguanylate cyclase (DgcQ) featuring both GGDEF and CHASE domains. Interestingly, inactivation of <em>dgcQ</em> in <em>E. coli</em> has been observed to enhance bacterial adhesion by increasing swimming motility [<a href="#pone.0330229.ref019" class="usa-link" aria-describedby="pone.0330229.ref019">19</a>]. In contrast, studies in other species (<em>Comamonas testosteroni</em> [<a href="#pone.0330229.ref020" class="usa-link" aria-describedby="pone.0330229.ref020">20</a>], <em>Pseudomonas fluorescens</em> [<a href="#pone.0330229.ref021" class="usa-link" aria-describedby="pone.0330229.ref021">21</a>], <em>Pseudomonas putida</em> [<a href="#pone.0330229.ref022" class="usa-link" aria-describedby="pone.0330229.ref022">22</a>], and <em>Pseudomonas syringa</em> [<a href="#pone.0330229.ref023" class="usa-link" aria-describedby="pone.0330229.ref023">23</a>]) demonstrate that heterologous expression of <em>dgcQ</em> elevates c-di-GMP levels and promotes mature biofilm formation. However, in these studies, the <em>dgcQ</em> gene was not expressed from its native promoter, and the experiments did not involve <em>E. coli</em>. Thus, we cannot precisely determine the phenotype and function it regulates in <em>E. coli</em>. These perplexing findings suggest that the modular enzymes involved in c-di-GMP regulation are more complex than anticipated. Moreover, we cannot simply equate BAIs with bacterial biofilms, as BAIs arise from intricate interactions among pathogens, biomaterials, and host immune responses [<a href="#pone.0330229.ref024" class="usa-link" aria-describedby="pone.0330229.ref024">24</a>].</p>
<p>Polyvinyl chloride (PVC) remains widely used in medical catheters due to its favorable material properties, availability, and low cost [<a href="#pone.0330229.ref025" class="usa-link" aria-describedby="pone.0330229.ref025">25</a>]. Unfortunately, PVC-associated infections pose significant clinical challenges. Comparative studies reveal that PVC endotracheal tubes exhibit higher susceptibility to bacterial adhesion than silicone surfaces, which correlates with increased ventilator-associated pneumonia risk in intubated patients [<a href="#pone.0330229.ref026" class="usa-link" aria-describedby="pone.0330229.ref026">26</a>]. However, the effects of c-di-GMP in PVC-associated infections caused by <em>E. coli</em> remains underexplored. Consequently, in-depth research on the role of c-di-GMP in PVC BAIs caused by <em>E. coli</em> not only aids in understanding the mechanism of bacterial biofilm formation but also offers theoretical and practical guidance for developing more effective antibacterial materials.</p>
<p>In the present study, we constructed four mutant strains of <em>E. coli</em> ATCC25922 with varying levels of c-di-GMP by knocking out or overexpressing <em>dgcQ</em> to elucidate the contribution of c-di-GMP to PVC BAIs. We systematically investigated the effects of c-di-GMP on biological functions including growth, motility, and environmental stress resistance in planktonic bacteria. Using PVC substrates, we dynamically monitored biofilm formation processes and analyzed their morphology and ultrastructure. Subsequently, we investigated the changes in gene expression in biofilm bacteria. We further examined the effects of c-di-GMP on <em>E. coli</em>'s adhesion and invasion capabilities, and on host-pathogen interactions. Collectively, our findings advance the understanding of molecular mechanisms underlying <em>E. coli</em>-mediated medical device infections and suggest that targeting c-di-GMP could be a promising strategy for preventing biofilm formation and reducing the risk of BAIs.</p></section><section id="sec002"><h2 class="pmc_sec_title">2 Materials and methods</h2>
<section id="sec003"><h3 class="pmc_sec_title">2.1 Bacterial strains, plasmids, and growth conditions</h3>
<p>The strains and plasmids used in the present study are listed in <a href="#pone.0330229.t001" class="usa-link">Table 1</a>. <em>E. coli</em> ATCC25922 was obtained from the Institute of Microbiology, Chinese Academy of Sciences. Plasmids pCas, pTargetF, and pBAD30 were purchased from Haijihaoge Biotech Co., Ltd. (Shanghai, China). All strains were routinely cultured at 37°C in Luria-Bertani (LB) broth. When required, the following supplements were added: ampicillin (100 μg/mL), kanamycin (50 μg/mL), spectinomycin (50 μg/mL), and L-arabinose.</p>
<section class="tw xbox font-sm" id="pone.0330229.t001"><h4 class="obj_head">Table 1. Bacterial strains and plasmids used in this study.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Strains and plasmids</th>
<th align="left" rowspan="1" colspan="1">Description</th>
<th align="left" rowspan="1" colspan="1">Source</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>E. coli</em> ATCC25922</td>
<td align="left" rowspan="1" colspan="1">Wild-type</td>
<td align="left" rowspan="1" colspan="1">CGMCC</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>E. coli</em> ATCC25922Δ<em>dgcQ</em>
</td>
<td align="left" rowspan="1" colspan="1">In frame deletion of <em>dgcQ</em> derived from ATCC25922</td>
<td align="left" rowspan="1" colspan="1">This study</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">WT</td>
<td align="left" rowspan="1" colspan="1">
<em>E. coli</em> ATCC25922 containing an expression vector pBAD30, Amp<sup>r</sup>
</td>
<td align="left" rowspan="1" colspan="1">This study</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Δ<em>dgcQ</em>
</td>
<td align="left" rowspan="1" colspan="1">
<em>E. coli</em> ATCC25922Δ<em>dgcQ</em> containing an expression vector pBAD30, Amp<sup>r</sup>
</td>
<td align="left" rowspan="1" colspan="1">This study</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CΔ<em>dgcQ</em>
</td>
<td align="left" rowspan="1" colspan="1">
<em>E. coli</em> ATCC25922Δ<em>dgcQ</em> containing an expression vector pBAD-<em>dgcQ</em>, Amp<sup>r</sup>
</td>
<td align="left" rowspan="1" colspan="1">This study</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">O<em>dgcQ</em>
</td>
<td align="left" rowspan="1" colspan="1">
<em>E. coli</em> ATCC25922 containing an expression vector pBAD-<em>dgcQ</em>, Amp<sup>r</sup>
</td>
<td align="left" rowspan="1" colspan="1">This study</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>E. coli</em> DH5a</td>
<td align="left" rowspan="1" colspan="1">
<em>E. coli</em> for cloning</td>
<td align="left" rowspan="1" colspan="1">Tsingke</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">pCAS</td>
<td align="left" rowspan="1" colspan="1">
<em>repA101</em>(Ts) <em>kan Pcas-cas9 ParaB-Red lacIq Ptrc-sgRNA-pMB1</em>
</td>
<td align="left" rowspan="1" colspan="1">Haijihaoge</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">pTargetF</td>
<td align="left" rowspan="1" colspan="1">
<em>pMB1 aadA</em> sgRNA</td>
<td align="left" rowspan="1" colspan="1">Haijihaoge</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">pTargetF-<em>dgcQ</em>
</td>
<td align="left" rowspan="1" colspan="1">
<em>pMB1 aadA</em> sgRNA<em>-dgcQ</em>, sgRNA<em>-dgcQ</em>
</td>
<td align="left" rowspan="1" colspan="1">This study</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">pBAD30</td>
<td align="left" rowspan="1" colspan="1">Expression vector</td>
<td align="left" rowspan="1" colspan="1">Haijihaoge</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">pBAD-<em>dgcQ</em>
</td>
<td align="left" rowspan="1" colspan="1">pBAD30 with 1695 bp <em>dgcQ</em> gene</td>
<td align="left" rowspan="1" colspan="1">This study</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330229.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec004"><h3 class="pmc_sec_title">2.2 Construction of <em>E. coli</em> mutant strains</h3>
<p>The <em>dgcQ</em> gene knockout mutant ATCC25922Δ<em>dgcQ</em> was constructed using the pCas/pTargetF CRISPR-Cas9 system. Specifically, donor DNA fragments were amplified from the <em>E. coli</em> ATCC25922 genome: the upstream fragment was amplified using primers P1/P2, and the downstream fragment was amplified using primers P3/P4. Following purification, these fragments were fused by overlapping PCR to generate donor DNA. Two sgRNAs targeting <em>dgcQ</em> were subsequently cloned into pTargetF to generate pTargetF-<em>dgcQ.</em> Both pTarget-<em>dgcQ</em> and donor DNA were electroporated into ATCC25922 cells harboring pCas plasmid. Positive clones were selected on LB plates supplemented with kanamycin (50 μg/mL) and spectinomycin (50 μg/mL). To eliminate plasmids, selected colonies were streaked onto LB agar with kanamycin (50 μg/mL) and incubated at 30°C overnight to remove the pTargetF plasmid. Subsequently, colonies were incubated at 42°C overnight to remove the temperature-sensitive pCas plasmid.</p>
<p>For the <em>dgcQ</em>-overexpression mutant (O<em>dgcQ</em>), the <em>dgcQ</em> coding sequence was amplified with primers P1/P4, while pBAD30 vector was linearized by inverse PCR using primers P5/P6. These fragments were assembled using the ClonExpress® II One Step Cloning Kit (Vazyme Biotech Co., Ltd., Nanjing, China) to generate pBAD-<em>dgcQ.</em></p>
<p>The pBAD30 plasmid was transformed into competent cells of ATCC25922 and ATCC25922Δ<em>dgcQ</em> to produce strains ATCC25922/pBAD30 (WT) and ATCC25922Δ<em>dgcQ</em>/pBAD30 (Δ<em>dgcQ</em>). The pBAD-<em>dgcQ</em> plasmid was transformed into competent cells of ATCC25922Δ<em>dgcQ</em> and ATCC25922 to generate complemented strain ATCC25922Δ<em>dgcQ</em>/pBAD-<em>dgcQ</em> (CΔ<em>dgcQ</em>) and overexpression strain ATCC25922/pBAD-<em>dgcQ</em> (O<em>dgcQ</em>). All mutant strains were verified through PCR amplification using 2 × Phanta® Max Master Mix (Dye Plus; Vazyme Biotech Co., Ltd., Nanjing, China), 1% agarose gel electrophoresis, and Sanger sequencing. Primer sequences are provided in <a href="#pone.0330229.s001" class="usa-link">S1 Table</a>.</p></section><section id="sec005"><h3 class="pmc_sec_title">2.3 Expression of <em>dgcQ</em> in the four mutant strains</h3>
<p>To investigate <em>dgcQ</em> expression in WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains, overnight cultures of each strain were sub-cultured (1:100) in 10 mL LB broth supplemented with 100 μg/mL ampicillin, followed by incubation at 37°C with 220 rpm shaking. When the OD<sub>600</sub> reached 0.4 ~ 0.6, 0.2% L-arabinose was added to induce gene expression, with parallel control cultures maintained without inducer. All cultures were further incubated for 16 h under identical conditions. Cells were harvested by centrifugation (4,000 rpm, 10 min, 4°C) and washed twice with ice-cold PBS. Total RNA was extracted using TRIzol<sup>TM</sup> reagent (Life Technologies, USA) according to the previously described method [<a href="#pone.0330229.ref027" class="usa-link" aria-describedby="pone.0330229.ref027">27</a>]. RNA integrity was verified by 1% agarose gel electrophoresis. First-strand cDNA synthesis was performed using the HiScript II Q RT SuperMix for qPCR (Vazyme Biotech Co., Ltd., Nanjing, China). Quantitative real-time PCR (qRT-PCR) was conducted on an Applied Biosystems 7500 system with SYBR Green Master Mix (Vazyme Biotech Co., Ltd., Nanjing, China). The <em>dgcQ</em> expression levels were normalized to <em>16S rRNA</em> using the 2<sup>-∆∆Ct</sup> method.</p></section><section id="sec006"><h3 class="pmc_sec_title">2.4 DgcQ expression in the four mutant strains</h3>
<p>To determine the optimal induction concentration of pBAD-<em>dgcQ</em> plasmid, O<em>dgcQ</em> cultures were subcultured (1:100) as previously described. When the OD<sub>600</sub> reached 0.4 ~ 0.6, gradient concentrations of L-arabinose (0.05%, 0.1%, 0.2%, 0.4%) were added to each O<em>dgcQ</em> culture, which was then incubated for 16 h at 37°C with 220 rpm shaking. For comparative analysis of DgcQ expression in WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains, cultures were prepared as described above. When the OD<sub>600</sub> reached 0.4 ~ 0.6, 0.2% L-arabinose was added and the cultures were incubated for 16 h at 37°C with 220 rpm shaking. After incubation, all cultures were harvested, washed twice with ice-cold PBS, and resuspended in 1 mL lysis solution (lysis buffer: 988 μL; protease inhibitor mixture: 1 μL; 1M DDT: 1 μL; 100 mM PMSF: 10 μL) at 4°C for 10 min. Ultrasonication (300 W, 10 s on/10 s off cycles, 20 min total) was performed to disrupt cells until lysate clarification. The lysate was centrifuged (12,000 rpm, 10 min, 4°C) to collect total protein. Total protein concentration was determined using a BCA protein assay kit. 30 μg protein was loaded and resolved on 10% SDS-PAGE, followed by electrophoretic transfer to PVDF membranes. After blocking in 5% skim milk for 1 h, membranes were incubated with anti-DgcQ antibody (1:2,000) at 4°C for 16 h, washed three times with TBST. Subsequently, membranes were incubated with HRP-conjugated secondary antibody (1:4,000) at room temperature for 1 h, washed three times with TBST. Immunoreactive bands were visualized using an ECL system. Anti-GAPDH (1:5,000) served as loading control. All antibodies were provided by Cusabio Biotech Co., Ltd. (Wuhan, China). Parallel analyses were performed for ATCC25922 and ATCC25922Δ<em>dgcQ</em> using identical protocols.</p></section><section id="sec007"><h3 class="pmc_sec_title">2.5 Quantitation of cellular c-di-GMP in the four strains</h3>
<p>To assess the regulatory role of <em>dgcQ</em> on c-di-GMP homeostasis, intracellular c-di-GMP levels were quantified in the WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains. Cultures were grown under standard conditions (37°C, 220 rpm) in LB broth supplemented with 100 μg/mL ampicillin. When the OD<sub>600</sub> reached 0.4 ~ 0.6, 0.2% L-arabinose was supplemented to induce gene expression, followed by 16 h incubation. After incubation, the cultures were harvested by centrifugation, washed twice with ice-cold PBS, and subjected to c-di-GMP extraction as previously described [<a href="#pone.0330229.ref028" class="usa-link" aria-describedby="pone.0330229.ref028">28</a>]. Quantification was performed using a commercial c-di-GMP ELISA kit (Meimian Biotech Co., Ltd., Jiangsu, China) according to the manufacturer's protocol. Cellular c-di-GMP levels were normalized to total bacterial protein and expressed as pmol/g bacterial protein. Three independent biological replicates were analyzed. Parallel measurements were conducted for ATCC25922 and ATCC25922Δ<em>dgcQ</em> using identical protocols.</p></section><section id="sec008"><h3 class="pmc_sec_title">2.6 Planktonic growth curve</h3>
<p>Bacterial growth curves were generated as previously described [<a href="#pone.0330229.ref029" class="usa-link" aria-describedby="pone.0330229.ref029">29</a>]. Briefly, the WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains were cultured in 100 mL LB broth supplemented with 100 μg/mL ampicillin with or without 0.2% L-arabinose and incubated at 37°C for 48 h with shaking at 220 rpm. The OD<sub>600</sub> values of the four strains were measured periodically using a spectrophotometer. The experiments were conducted three times independently.</p></section><section id="sec009"><h3 class="pmc_sec_title">2.7 Biofilm biomass on PVC biomaterials</h3>
<p>The crystal violet (CV) staining assay was performed to evaluate biofilm biomass on PVC substrates following a previously described method with minor modifications[30]. Briefly, overnight cultures of WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> were normalized to an initial OD<sub>600</sub> of 0.4 using fresh LB broth containing 100 μg/mL ampicillin supplemented with 0.2% L-arabinose. A sterile 5 × 5 mm PVC piece (Kewei Co., Ltd., Guangzhou, China) and 100 μL bacterial suspension were added to each well of a 96-well plate, followed by static incubation at 37°C for 12, 24, 48, and 72 h. Four to six technical replicates per strain were included in each plate. After incubation, the PVC pieces were gently washed twice with PBS to remove non-adherent bacteria, stained with 0.1% crystal violet solution for 20 min at room temperature, and subsequently washed three times with PBS. Bound dye was solubilized in 95% ethanol, and absorbance at 570 nm was quantified using a microplate reader. The experiments were conducted three times independently. All experiments included PVC substrates pre-treated with human plasma (healthy donor-derived, 37°C, 4 h) after sterilization.</p></section><section id="sec010"><h3 class="pmc_sec_title">2.8 Viability of bacteria in biofilms on PVC biomaterials</h3>
<p>The tetrazolium salt (XTT) reduction assay was performed to evaluate the bacteria viability in biofilms using the XTT Bacteria Proliferation and Cytotoxicity Kit (KeyGEN Biotech Co., Ltd., Nanjing, China). Biofilms were established in 96-well plates containing sterile 5 × 5 mm PVC pieces following the protocol described in Section 2.7. Four to six wells per strain were inoculated in each plate. After incubation for 12–72 h at 37°C, PVC pieces were gently washed with PBS to remove non-adherent bacteria. Each well was supplemented with 100 μL of fresh LB broth and 20 μL XTT solution, followed by incubation at 37°C in the dark for 2 h. Absorbance at 450 nm was measured using a microplate reader. The experiments were conducted three times independently.</p></section><section id="sec011"><h3 class="pmc_sec_title">2.9 Swimming and swarming motility</h3>
<p>WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains were cultured in LB broth at 37°C with 220 rpm shaking for 16 h. Cultures were harvested by centrifugation (4,000 rpm, 10 min, 4°C) and resuspended in sterile PBS to normalize the OD<sub>600</sub> to 1.0. Swimming motility plates (0.3% agar containing 1% tryptone, 0.25% NaCl, 0.01% ampicillin, 0.2% L-arabinose) and swarming motility plates (0.4% agar with identical components) were prepared. Bacterial suspensions (2 μL) were spot-inoculated at the center of each plate using a sterile pipette tip. Plates were incubated at 37°C for 6–24 h, and motility diameters were measured with a digital caliper. The experiments were conducted three times independently.</p></section><section id="sec012"><h3 class="pmc_sec_title">2.10 Morphology of biofilms on PVC biomaterials</h3>
<p>The morphology of WT, Δ<em>dgcQ</em>, and O<em>dgcQ</em> biofilms on PVC was analyzed using confocal laser scanning microscopy (CLSM). Briefly, overnight cultures of WT, Δ<em>dgcQ</em>, and O<em>dgcQ</em> were adjusted to an initial OD<sub>600</sub> of 0.4 using fresh LB broth containing 100 μg/mL ampicillin supplemented with 0.2% L-arabinose. Bacterial suspensions (2 mL) and 10 × 10 mm PVC pieces were co-cultured in 24-well plates at 37°C for 12–72 h. After incubation, PVC pieces were gently washed with PBS to remove non-adherent bacteria, stained with a SYTO9/propidium iodide (PI) mixture (Live/Dead® BacLight<sup>TM</sup> Bacterial Viability Kit, Life Technologies, USA) for 20 min at 25°C in the dark, and rinsed with normal saline to remove unbound dye. CLSM imaging was performed on a Leica TCS SP2 system (Leica Microsystems, Germany). Five random fields per sample were acquired through Z-axis stack scanning. Biofilm thickness was measured by optical sectioning from the substratum to the biofilm surface, while live cell proportions were calculated based on SYTO9 (green)/PI (red) fluorescence intensity ratios using ImageJ v1.53 with the Biovolume plugin.</p></section><section id="sec013"><h3 class="pmc_sec_title">2.11 Ultrastructure of biofilms on PVC biomaterials</h3>
<p>The ultrastructure of WT, Δ<em>dgcQ</em>, and O<em>dgcQ</em> biofilms on PVC was analyzed by scanning electron microscopy (SEM) as described in our previous study [<a href="#pone.0330229.ref030" class="usa-link" aria-describedby="pone.0330229.ref030">30</a>]. Briefly, biofilm samples were prepared following the method described in CLSM protocol. After removing non-adherent bacteria, PVC pieces were fixed in 2% glutaraldehyde at 4°C for 16 h. The samples were sequentially subjected to primary fixation, post-fixation 1% osmium tetroxide, dehydration with gradient ethanol, and permeation with tert-butanol. Following critical point drying with CO<sub>2</sub> and ion sputter coating with golden brown, the samples were imaged using an S-3000N scanning electron microscope (Hitachi, Japan) at 20 kV and 20,000 × magnification. Three biological replicates were imaged per strain.</p></section><section id="sec014"><h3 class="pmc_sec_title">2.12 Bacterial adaptation to environmental stress</h3>
<p>Bacterial adaptation to environmental stress was examined as previously described [<a href="#pone.0330229.ref031" class="usa-link" aria-describedby="pone.0330229.ref031">31</a>,<a href="#pone.0330229.ref032" class="usa-link" aria-describedby="pone.0330229.ref032">32</a>]. For oxidative/acid stress assays, overnight cultures of WT, Δ<em>dgcQ</em>, and O<em>dgcQ</em> were harvested and adjusted to an OD<sub>600</sub> of 1.0. Bacterial suspensions were exposed to LB broth containing 5 mM H<sub>2</sub>O<sub>2</sub> or LB broth acidified to pH 4.0/3.0 with HCl at 37°C for 30 min. For heat stress, bacterial suspensions were incubated in a 48°C water bath for 20 min. After treatments, viable cells were quantified by plating on LB agar plates at serial dilutions. After gradient dilution, viable cells were counted by colony-forming units (CFU) on LB agar plates after 24 h incubation at 37°C. Survival rates were calculated as: (CFU post-treatment/ CFU pre-treatment) × 100%. The experiments were conducted three times independently.</p></section><section id="sec015"><h3 class="pmc_sec_title">2.13 Gene expression in biofilms on PVC biomaterials</h3>
<p>Overnight cultures of WT, Δ<em>dgcQ</em>, and O<em>dgcQ</em> were prepared as described above. Bacterial suspensions (20 mL) and sterile 6 × 6 cm PVC pieces were co-cultured in 90 mm cell culture dishes at 37°C for 48 h. After co-culture, PVC pieces were gently washed with PBS to remove non-adherent bacteria. Biofilms on PVC pieces were scraped using sterile cell scrapers and collected in 2 mL microcentrifuge tubes. Total RNA was extracted and reverse-transcribed into cDNA. Flagellar motility genes (<em>flhC</em>, <em>flhD</em>, <em>motA</em>, <em>motB</em>, <em>ycgR</em>), extracellular polymeric substance (EPS) synthesis genes (<em>csgA</em>, <em>csgD</em>, <em>bcsA</em>, <em>ynfM</em>), and stress response genes (<em>sodA</em>, <em>katE</em>, <em>rstA</em>, <em>ibpA</em>, <em>ibpB</em>, <em>hdeA</em>, <em>hdeD</em>, <em>gadA</em>, <em>gadB</em>) were quantified using qRT-PCR. <em>16S rRNA</em> served as the endogenous control. WT biofilm samples were designated as calibrators. The 2<sup>−∆∆Ct</sup> method was used to analyze qRT-PCR data. Primer sequences are listed in <a href="#pone.0330229.s002" class="usa-link">S2 Table</a>. The experiments were conducted three times independently.</p></section><section id="sec016"><h3 class="pmc_sec_title">2.14 Host-pathogen interaction</h3>
<section id="sec017"><h4 class="pmc_sec_title">2.14.1 Co-culture and infection.</h4>
<p>WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains were cultured in LB broth at 37°C with 220 rpm shaking for 18 h. Bacterial suspensions were adjusted to an OD<sub>600</sub> of 1.0 and stained with SYTO9<sup>®</sup>Green Fluorescent Dye for 15 min in the dark. The HCT116 human intestinal epithelial cell line was kindly donated by Prof. Li (Department of Colorectal Surgery, Yunnan Cancer Hospital, Yunnan, China). HCT116 cells were cultivated in RPMI-1640 medium supplemented with 10% fetal bovine serum (FBS; Gibco) under 5% CO<sub>2</sub> at 37°C.</p>
<p>For bacterial infection, sterile 20 × 20 mm PVC pieces were placed in 6-well plates, and HCT116 cells were seeded at a density of 1 × 10<sup>6</sup> cells/well. When cells reached 80% confluency, each well was washed twice with ice-cold PBS, and fresh RPMI-1640 medium containing 100 μg/mL ampicillin and 2 mg/mL L-arabinose was added. Subsequently, HCT116 cells were challenged with WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, or O<em>dgcQ</em> at a multiplicity of infection (MOI) of 100 (bacteria:cells = 100:1) at 37°C under 5% CO<sub>2</sub>. After 4 h of infection, the 6-well plates were observed directly using a fluorescence microscope. After 16 h of infection, the plates were observed following PBS washing to remove non-adherent bacteria.</p></section><section id="sec018"><h4 class="pmc_sec_title">2.14.2 Adhesion and invasion to host cells.</h4>
<p>For adhesion assays, HCT116 cells (1 × 10<sup>6</sup> cells/well) along with sterile PVC pieces were seeded and cultured in 6-well plates as previously described. Cells were infected with WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, or O<em>dgcQ</em> in RPMI-1640 containing 100 μg/mL ampicillin and 2 mg/mL L-arabinose at a MOI of 100. After incubation for 2 h, non-adherent bacteria were removed by gently washing three times with ice-cold PBS. HCT116 cells were lysed with 0.2% Triton X-100 for 10 min. After gradient dilution, lysates were plated onto LB agar plates, and CFUs were counted after 18 h incubation at 37°C. The experiments were conducted seven times independently.</p>
<p>For invasion assays, following the 2 h infection protocol, extracellular bacteria were eliminated by treatment with 100 μg/mL gentamicin (1 h, 37°C). Cells were washed with ice-cold PBS three times to remove gentamicin, lysed, and processed as above. Intracellular bacterial load was quantified via CFU enumeration. The experiments were conducted seven times independently.</p></section><section id="sec019"><h4 class="pmc_sec_title">2.14.3 Quantification of cytokines in the co-culture system.</h4>
<p>HCT116 cells (1 × 10<sup>6</sup> cells/well) along with sterile PVC pieces were seeded and cultured in 6-well plates as previously described. Following co-culture with WT, Δ<em>dgcQ</em>, and O<em>dgcQ</em> at an MOI of 100 for 3 h, cytokines IL-1β and IFN-β in the culture supernatants, as well as NF-κB and IP-10 in HCT116 cell lysates, were quantified by ELISA. Uninfected HCT116 cells served as negative controls. HCT116 cells infected with WT strain served as the positive controls. Human IL-1β, IFN-β, NF-κB, and IP-10 ELISA kits (Cusabio Biotech Co., Ltd., Wuhan, China) were used according to the manufacturer's protocol. The experiments were conducted three times independently.</p></section></section><section id="sec020"><h3 class="pmc_sec_title">2.15 Statistical analysis</h3>
<p>Statistical analyses were performed using the IBM SPSS Statistics 23.0. All graphs were generated using GraphPad Prism 8.0.1 and Adobe Illustrator CC 2019. The experimental data that followed the normal distribution or that did so after conversion are expressed as the mean ± standard deviation. One-way analysis of variance (ANOVA) with LSD or Tamhane’s T2 or Tukeyʼs HSD multiple comparison tests were used for intergroup comparisons. The threshold for statistical significance was set at <em><em>P</em> </em>&lt; 0.05.</p></section></section><section id="sec021"><h2 class="pmc_sec_title">3 Results</h2>
<section id="sec022"><h3 class="pmc_sec_title">3.1 Verification of WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains</h3>
<p>Four mutant strains were identified by PCR and agarose gel electrophoresis. The results demonstrated that WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains were successfully generated using the pCas/pTargetF and pBAD/<em>araC</em> expression systems (<a href="#pone.0330229.g001" class="usa-link">Fig 1A</a>). <em>dgcQ</em> expression in these strains was confirmed by qRT-PCR (<a href="#pone.0330229.g001" class="usa-link">Fig 1B</a> and <a href="#pone.0330229.g001" class="usa-link">1C</a>). Specifically, <em>dgcQ</em> expression was restored in CΔ<em>dgcQ</em> and upregulated in O<em>dgcQ</em> (<a href="#pone.0330229.g001" class="usa-link">Fig 1B</a>). Following 0.2% L-arabinose induction, <em>dgcQ</em> expression levels in CΔ<em>dgcQ</em> and O<em>dgcQ</em> were markedly enhanced, with CΔ<em>dgcQ</em> exhibiting 4-fold higher expression compared to WT, and O<em>dgcQ</em> showing a 180-fold increase (<a href="#pone.0330229.g001" class="usa-link">Fig 1C</a>). In contrast, <em>dgcQ</em> expression remained low in Δ<em>dgcQ</em> (<a href="#pone.0330229.g001" class="usa-link">Fig 1B</a> and <a href="#pone.0330229.g001" class="usa-link">1C</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330229.g001"><h4 class="obj_head">Fig 1. Construction and identification of WT, Δ<em>dgcQ</em>, C<em>dgcQ</em>, and O<em>dgcQ</em> strains.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367115_pone.0330229.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ff96/12367115/d37df54b729f/pone.0330229.g001.jpg" loading="lazy" height="633" width="743" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330229.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) PCR identification of the WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains. Lane1, 3, 5, 8 amplified by primers P7/8 (expected sizes: 378 bp, 378 bp, 1974 bp, 1974 bp). Lane2, 4, 6, 9 amplified by primers P1/4 (expected sizes: 1695 bp, 1425 bp, 1695/1425 bp, 1695 bp). Lane7 amplified by primers P1/9 (expected sizes: 1445 bp). M: 5000 bp ladder. (B) qRT-PCR identification of the <em>dgcQ</em> mRNA of WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains (n = 3). (C) qRT-PCR identification of the <em>dgcQ</em> mRNA of WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains induced by 0.2% L-arabinose (n = 3). (D) The DgcQ expression of O<em>dgcQ</em> strain induced by various concentrations of L-arabinose (n = 3). (E) The DgcQ expression of WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains induced by 0.2% L-arabinose (n = 3). (F) The c-di-GMP levels of WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains (n = 3). Statistical analysis was performed by One-way ANOVA with LSD (B, C, D, and E) or Tamhane’s T2 (F) multiple comparison test. The values obtained in the WT strain were used as control in qRT-PCR assays (B, C). The DgcQ protein levels were shown relative to GAPDH (D, E). <sup>*</sup><em><em>P</em> </em>&lt; 0.05; <sup>**</sup><em><em>P</em> </em>&lt; 0.01; <sup>***</sup><em><em>P</em> </em>&lt; 0.001; <sup>****</sup><em><em>P</em> </em>&lt; 0.0001. Error bars indicated standard deviations.</p></figcaption></figure><p>Western blot analysis revealed that DgcQ expression in O<em>dgcQ</em> increased dose-dependently with L-arabinose concentrations up to 0.2%, but declined at 0.4% (<a href="#pone.0330229.g001" class="usa-link">Fig 1D</a>). These data indicate that 0.2% L-arabinose is the optimal induction concentration. After induction, DgcQ was undetectable in Δ<em>dgcQ</em>, while CΔ<em>dgcQ</em> displayed intermediate expression levels between WT and O<em>dgcQ</em> (<a href="#pone.0330229.g001" class="usa-link">Fig 1E</a>).</p>
<p>As shown in <a href="#pone.0330229.g001" class="usa-link">Fig 1F</a>, the intracellular c-di-GMP concentration in O<em>dgcQ</em> was 273.3 pmol/g, higher than that in CΔ<em>dgcQ</em> (54.2 pmol/g). In contrast, c-di-GMP levels in Δ<em>dgcQ</em> were 27.3 pmol/g, representing a 17% decrease compared to WT (33.1 pmol/g). These findings conclusively establish DgcQ as a functional diguanylate cyclase regulating c-di-GMP homeostasis.</p>
<p>Identification of the ATCC25922 and ATCC25922Δ<em>dgcQ</em> strains is shown in <a href="#pone.0330229.s003" class="usa-link">S1 Fig</a>. Results demonstrated that neither DgcQ protein levels nor c-di-GMP concentrations in <em>E. coli</em> were significantly affected by the presence or absence of the pBAD vector. This confirms that the phenotypic differences observed in our original experiments (conducted with the vector) resulted directly from <em>dgcQ</em> genetic manipulation and were not due to vector effects.</p></section><section id="sec023"><h3 class="pmc_sec_title">3.2 c-di-GMP affects biofilm formation on PVC biomaterials</h3>
<p>Growth curve analysis revealed comparable growth patterns among the four mutant <em>E. coli</em> strains in standard LB broth (<a href="#pone.0330229.g002" class="usa-link">Fig 2A</a>). The addition of 0.2% L-arabinose to LB broth to induce differential DgcQ expression similarly showed no significant growth variations among the strains (<a href="#pone.0330229.g002" class="usa-link">Fig 2B</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330229.g002"><h4 class="obj_head">Fig 2. Biological characteristics analysis of the four mutant strains.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367115_pone.0330229.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ff96/12367115/99c47ddb93ab/pone.0330229.g002.jpg" loading="lazy" height="459" width="741" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330229.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Growth curve of WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains as determined by optical density (OD 600 nm) (n = 3). (B) Growth curve of WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains induced by 0.2% L-arabinose as determined by optical density (OD 600 nm) (n = 3). (C) The biofilm biomass of WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> strains on PVC (n = 3). (D) The bacterial viability of WT, Δ<em>dgcQ</em>, CΔ<em>dgcQ</em>, and O<em>dgcQ</em> biofilms on PVC (n = 3). Statistical analysis was performed by One-way ANOVA with Tukey’s HSD multiple comparison test for all the assays. <sup>**</sup>
<em><em>P</em> </em>&lt; 0.01; <sup>***</sup>
<em><em>P</em> </em>&lt; 0.001; <sup>****</sup>
<em><em>P</em> </em>&lt; 0.0001. Error bars indicated standard deviations.</p></figcaption></figure><p>Biofilm biomass analysis demonstrated temporal differences in PVC colonization, while <em>dgcQ</em> deletion (Δ<em>dgcQ</em>) transiently enhanced bacterial adhesion to PVC at 12 h (initial stage of biofilm formation). subsequent biofilm development from 24–72 h exhibited distinct patterns: O<em>dgcQ</em> biofilms accumulated the highest biomass, CΔ<em>dgcQ</em> showed restoration of WT characteristics, and Δ<em>dgcQ</em> consistently displayed the lowest colonization capacity (<a href="#pone.0330229.g002" class="usa-link">Fig 2C</a>).</p>
<p>XTT assays revealed that dynamic viability profiles within biofilms. At 12 h, Δ<em>dgcQ</em> exhibited a modest but statistically significant viability increase compared to CΔ<em>dgcQ</em> (<em><em>P</em> </em>= 0.0081) and O<em>dgcQ</em> (<em>P</em> = 0.0013). From 24–72 h, the XTT results demonstrated a hierarchical viability: O<em>dgcQ</em> &gt; CΔ<em><em>dgcQ</em> </em>= WT &gt; Δ<em>dgcQ</em> (<a href="#pone.0330229.g002" class="usa-link">Fig 2D</a>).</p></section><section id="sec024"><h3 class="pmc_sec_title">3.3 DgcQ represses the swimming ability of <em>E. coli</em> but not swarming</h3>
<p>Motility assays revealed distinct swimming phenotypes among the strains (<a href="#pone.0330229.g003" class="usa-link">Fig 3A</a> and <a href="#pone.0330229.g003" class="usa-link">3B</a>). The Δ<em>dgcQ</em> exhibited significantly expanded motility zones compared to other strains at both 6 h (<em>P</em> &lt; 0.001) and 12 h (<em>P</em> &lt; 0.0001), with diameter measurements following this descending order: Δ<em><em>dgcQ</em> </em>&gt; WT/CΔ<em><em>dgcQ</em> </em>&gt;<em> </em>O<em>dgcQ</em>. These findings demonstrate that <em>dgcQ</em> deletion enhances flagellar-driven motility, whereas <em>dgcQ</em> overexpression substantially impairs bacterial swimming capacity. In contrast, swarming assays showed no statistically significant differences between strains at either 12 h or 24 h under identical experimental conditions (<a href="#pone.0330229.g003" class="usa-link">Fig 3C</a> and <a href="#pone.0330229.g003" class="usa-link">3D</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330229.g003"><h4 class="obj_head">Fig 3. Comparison of the swimming and swarming ability among the four mutant strains.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367115_pone.0330229.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ff96/12367115/cf7abe777860/pone.0330229.g003.jpg" loading="lazy" height="237" width="740" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330229.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Swimming ability at 6 h (n = 3). (B) Swimming ability at 12 h (n = 3). (C) Swarming ability at 12 h (n = 3). (D) Swarming ability at 24 h (n = 3). Statistical analysis was performed by One-way ANOVA with LSD multiple comparison test for all the assays. <sup>***</sup><em><em>P</em> </em>&lt; 0.001; <sup>****</sup><em><em>P</em> </em>&lt; 0.0001. n.s, no significance. Error bars indicated standard deviations.</p></figcaption></figure></section><section id="sec025"><h3 class="pmc_sec_title">3.4 Morphology of biofilms on PVC biomaterials</h3>
<p>Biofilm morphology was characterized through fluorescence-based viability staining following PVC co-culture periods (12, 24, 48, and 72 h). Live and dead bacteria in biofilms on PVC were differentially visualized using green (SYTO9) and red (PI) fluorescence markers, respectively (<a href="#pone.0330229.g004" class="usa-link">Fig 4A</a>–<a href="#pone.0330229.g004" class="usa-link">4D</a>). Quantitative morphometric analysis revealed temporal dynamics in biofilm development: Δ<em>dgcQ</em> exhibited the thickest biofilm at 12 h, but its biofilm formation rate slowed significantly from 24–72 h. In contrast, O<em>dgcQ</em> demonstrated accelerated biofilm maturation, maintaining superior thickness compared to WT and Δ<em>dgcQ</em> throughout 24–72 h (<a href="#pone.0330229.g004" class="usa-link">Fig 4E</a>). Meanwhile, live/dead cell proportion analysis showed that the proportion of live bacteria in Δ<em>dgcQ</em> was consistently low during the experimental period, whereas WT and O<em>dgcQ</em> maintained higher proportions of live bacteria (<a href="#pone.0330229.g004" class="usa-link">Fig 4F</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330229.g004"><h4 class="obj_head">Fig 4. Thickness and live cell proportions of biofilms on PVC observed by CLSM.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367115_pone.0330229.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ff96/12367115/f3e193f92aeb/pone.0330229.g004.jpg" loading="lazy" height="716" width="736" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330229.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Observation at 12 h. (B) Observation at 24 h. (C) Observation at 48 h. (D) Observation at 72 h. (E) The thickness of biofilms at 12, 24, 48, 72 h (n = 5). (F) The live cell proportions of biofilms at 12, 24, 48, 72 h (n = 5). Green represented live cell, red represented dead cell. Scale bar: 200 μm. Statistical analysis was performed by One-way ANOVA with LSD multiple comparison test for all the assays. <sup>**</sup><em><em>P</em> </em>&lt; 0.01; <sup>****</sup><em><em>P</em> </em>&lt; 0.0001. Error bars indicated standard deviations.</p></figcaption></figure></section><section id="sec026"><h3 class="pmc_sec_title">3.5 Ultrastructure of biofilms on PVC biomaterials</h3>
<p>Biofilm ultrastructure was analyzed by SEM following PVC co-culture. As demonstrated in <a href="#pone.0330229.g005" class="usa-link">Fig 5</a>, biofilm architecture progressively increased in complexity, density, and three-dimensional organization with extended co-cultivation time across all <em>E. coli</em> strains. At the initial attachment phase (12 h), biofilms exhibited simple two-dimensional arrangements with minimal EPS deposition between cells. Bacterial aggregates were visible on PVC surfaces in all strains, though Δ<em>dgcQ</em> formed notably denser stacked aggregates compared to WT and O<em>dgcQ</em>. During maturation phases (24–72 h), Δ<em>dgcQ</em> biofilms remained structurally compromised, exhibiting sparse cellular networks with only grid-like intercellular connections and persistently low EPS accumulation. WT biofilms developed increasingly intricate three-dimensional matrices with moderate EPS deposition. O<em>dgcQ</em> biofilms displayed exceptional structural consolidation, forming densely packed architectures where bacterial cells were fully embedded within abundant EPS. These complexes developed into highly organized clusters with sophisticated three-dimensional topography.</p>
<figure class="fig xbox font-sm" id="pone.0330229.g005"><h4 class="obj_head">Fig 5. Microstructure of biofilms on PVC observed by SEM at 12, 24, 48, and 72 h.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367115_pone.0330229.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ff96/12367115/475ac1ccedd7/pone.0330229.g005.jpg" loading="lazy" height="412" width="730" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330229.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Scale bar: 2.0 μm.</p></figcaption></figure></section><section id="sec027"><h3 class="pmc_sec_title">3.6 DgcQ-mediated c-di-GMP signalling promotes <em>E. coli</em> adaptation to environmental stresses and alters gene expression in biofilms</h3>
<p>The effects of c-di-GMP on the adaptation of <em>E. coli</em> to environmental stresses, including oxidative stress (<a href="#pone.0330229.g006" class="usa-link">Fig 6A</a>), heat stress (<a href="#pone.0330229.g006" class="usa-link">Fig 6B</a>), and acid stress (<a href="#pone.0330229.g006" class="usa-link">Fig 6C</a>), were systematically evaluated. No significant differences were observed in the initial bacterial counts of WT, Δ<em>dgcQ</em>, and O<em>dgcQ</em> strains prior to stress exposure. Survival assays demonstrated significantly enhanced stress tolerance in O<em>dgcQ</em>, whereas Δ<em>dgcQ</em> exhibited compromised survival rates compared to WT under all tested conditions.</p>
<figure class="fig xbox font-sm" id="pone.0330229.g006"><h4 class="obj_head">Fig 6. The adaptation to environmental stresses and related gene expression.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367115_pone.0330229.g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ff96/12367115/d7116542a116/pone.0330229.g006.jpg" loading="lazy" height="710" width="741" alt="Fig 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330229.g006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) The adaptation to oxidative stress (n = 3). (B) The adaptation to heat stress (n = 3). (C) The adaptation to acid stress (n = 3). (D) Expression of genes related to motility (n = 3). (E) Expression of genes related to EPS (n = 3). (F) Expression of genes related to oxidative and heat stress (n = 3). (G) Expression of genes related to acid stress (n = 3). Statistical analysis was performed by One-way ANOVA with LSD multiple comparison test for all the assays. <sup>*</sup><em><em>P</em> </em>&lt; 0.05; <sup>**</sup><em><em>P</em> </em>&lt; 0.01; <sup>***</sup><em><em>P</em> </em>&lt; 0.001; <sup>****</sup><em><em>P</em> </em>&lt; 0.0001. The values obtained in the WT strain were used as control in qRT-PCR assays (D-G). Error bars indicated standard deviations.</p></figcaption></figure><p>qRT-PCR was performed to identify the expression of flagellar motility genes (<em>flhC</em>, <em>flhD</em>, <em>motA</em>, <em>motB</em>, <em>ycgR</em>), EPS synthesis genes (<em>csgA</em>, <em>csgD</em>, <em>bcsA</em>, <em>ynfM</em>), oxidative stress response genes (<em>sodA</em>, <em>katE</em>, <em>rstA</em>), heat stress response genes (<em>ibpA</em>, <em>ibpB</em>), and acid stress response genes (<em>hdeA</em>, <em>hdeD</em>, <em>gadA</em>, <em>gadB</em>) in the WT, Δ<em>dgcQ</em>, and O<em>dgcQ</em> biofilms. Results showed that all of these genes were significantly upregulated in O<em>dgcQ</em>. In contrast, some genes in Δ<em>dgcQ</em> biofilms (including <em>ycgR</em>, <em>ynfM</em>, <em>sodA</em>, <em>hdeD</em>, <em>gadA</em>, and <em>gadB</em>) were significantly downregulated, while <em>motB</em> expression was increased (<a href="#pone.0330229.g006" class="usa-link">Fig 6D</a>–<a href="#pone.0330229.g006" class="usa-link">6G</a>).</p></section><section id="sec028"><h3 class="pmc_sec_title">3.7 c-di-GMP modulates <em>E. coli</em>-host interactions and suppresses inflammatory responses</h3>
<p>The modulatory effects of c-di-GMP on <em>E. coli</em> pathogenesis were evaluated through host-pathogen interaction assays. As demonstrated in <a href="#pone.0330229.g007" class="usa-link">Fig 7A</a>–<a href="#pone.0330229.g007" class="usa-link">7C</a>, Δ<em>dgcQ</em> exhibited significantly enhanced adhesion and invasion capacity toward HCT116 cells compared to WT, whereas O<em>dgcQ</em> showed attenuated infectivity, confirming an inverse relationship between c-di-GMP levels and bacterial virulence.</p>
<figure class="fig xbox font-sm" id="pone.0330229.g007"><h4 class="obj_head">Fig 7. The adherence and invasion of the mutant strains to HCT116 and inflammatory response.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367115_pone.0330229.g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ff96/12367115/0e70a207fe63/pone.0330229.g007.jpg" loading="lazy" height="784" width="735" alt="Fig 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330229.g007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Observation of the co-culture system of the four mutant strains and HCT116 cells by fluorescence microscope. Green represented <em>E. coli</em>. Scale bar: 200 μm. (B) The adherence of <em>E. coli</em> to HCT116 (n = 7). (C) The invasion of <em>E. coli</em> to HCT116 (n = 7). (D) ELISA of IL-1β (n = 3). (E) ELISA of IFN-β (n = 3). (F) ELISA of NF-κb (n = 3). (G) ELISA of IP-10 (n = 3). Statistical analysis was performed by One-way ANOVA with LSD (D, E, and G) or Tamhane’s T2 (F) multiple comparison test. <sup>**</sup><em><em>P</em> </em>&lt; 0.01; <sup>***</sup><em><em>P</em> </em>&lt; 0.001; <sup>****</sup><em><em>P</em> </em>&lt; 0.0001. Error bars indicated standard deviations.</p></figcaption></figure><p>Host inflammatory responses were quantified via ELISA. The results (<a href="#pone.0330229.g007" class="usa-link">Fig 7D</a>–<a href="#pone.0330229.g007" class="usa-link">7G</a>) revealed that HCT116 cells infected with Δ<em>dgcQ</em> displayed the highest upregulation of IL-1β, IFN-β, NF-κB, and IP-10 levels. In contrast, cells infected with O<em>dgcQ</em> exhibited attenuated cytokine profiles, with all measured markers showing significantly lower levels. These findings suggest that c-di-GMP may mediate immunosuppression.</p></section></section><section id="sec029"><h2 class="pmc_sec_title">4 Discussion</h2>
<p>Nosocomial infections persist as a major clinical challenge, with bacterial biofilms implicated in 60–70% of cases [<a href="#pone.0330229.ref033" class="usa-link" aria-describedby="pone.0330229.ref033">33</a>]. Our study establishes <em>dgcQ</em>-mediated c-di-GMP signaling as a mater regulator of <em>E. coli</em> biofilm dynamics on PVC biomaterials. We demonstrate that c-di-GMP functions as a precise “biofilm chronometer”, orchestrating a critical balance between early colonization and mature biofilm resilience. Crucially, this pathway dictates host-pathogen interactions by modulating both bacterial virulence and immune evasion. These data offers novel insights for targeted anti-infective strategies.</p>
<p>While Δ<em>dgcQ</em> (low c-di-GMP) exhibited enhanced initial adhesion to PVC surfaces (12 h), this aligns with its elevated swimming motility (<a href="#pone.0330229.g003" class="usa-link">Fig 3A</a> and <a href="#pone.0330229.g003" class="usa-link">3B</a>). The results of initial adhesion and swimming ability seemed contradictory to each other. In fact, the paradoxical phenotypes can be explained by the dual role of low c-di-GMP in promoting motility-driven surface exploration while simultaneously increasing extracellular DNA (eDNA) production for attachment stabilization. These synergistic effects likely drove the transient biofilm advantage of Δ<em>dgcQ</em> during initial colonization (12 h), evidenced by thicker biofilm biomass (<a href="#pone.0330229.g002" class="usa-link">Fig 2C</a>) and higher viability (<a href="#pone.0330229.g002" class="usa-link">Fig 2D</a>). However, beyond this initial stage, O<em>dgcQ</em> (high c-di-GMP) formed structurally complex biofilms with dense EPS matrices at 24–72 h (<a href="#pone.0330229.g004" class="usa-link">Figs 4</a> and <a href="#pone.0330229.g005" class="usa-link">5</a>), supported by upregulation of EPS synthesis genes such as <em>csgD</em>, <em>csgA</em>, <em>bcsA</em>, and <em>ynfM</em> (<a href="#pone.0330229.g006" class="usa-link">Fig 6E</a>). <em>csgD</em> acts as a transcription factor that binds to the promoter of the <em>csgBAC</em> operon, which encodes the major curli subunit <em>csgA</em> and provides positive feedback for c-di-GMP, promoting <em>bcsA</em> expression in some strains. <em>csgA</em> and <em>bcsA</em> are major biofilm regulators that respectively control the expression of curli and cellulose [<a href="#pone.0330229.ref034" class="usa-link" aria-describedby="pone.0330229.ref034">34</a>]. Concurrently, O<em>dgcQ</em> biofilms demonstrated significant upregulation of flagellar genes (<em>flhC/D</em>, <em>motA/B</em>) despite the established role of c-di-GMP in motility suppression through YcgR-mediated flagellar braking [<a href="#pone.0330229.ref035" class="usa-link" aria-describedby="pone.0330229.ref035">35</a>,<a href="#pone.0330229.ref036" class="usa-link" aria-describedby="pone.0330229.ref036">36</a>]. This implies a compensatory regulatory mechanism by which bacterial cells counteract motility inhibition through flagellar gene overexpression, revealing the complexity of c-di-GMP signaling networks.</p>
<p>XTT/CLSM analysis revealed that the viability of Δ<em>dgcQ</em> biofilms was superior to that of WT and O<em>dgcQ</em> biofilms at 12 h (<a href="#pone.0330229.g002" class="usa-link">Fig 2D</a>), but the proportion of live bacteria was still lower (<a href="#pone.0330229.g004" class="usa-link">Fig 4A</a>). This seemingly contradictory result of higher viability alongside a lower proportion of live bacteria in Δ<em>dgcQ</em> biofilms can be explained by the higher biofilm biomass at 12 h, as shown in <a href="#pone.0330229.g002" class="usa-link">Fig 2C</a>. In contrast, the higher viability and live bacteria proportions of O<em>dgcQ</em> biofilms on PVC at the mature stages (24–72 h) indicated the long-term survival ability was enhanced in O<em>dgcQ</em> biofilms, correlating with significantly upregulated stress-response genes in O<em>dgcQ</em> biofilms (<a href="#pone.0330229.g006" class="usa-link">Fig 6F</a> and <a href="#pone.0330229.g006" class="usa-link">6G</a>). To better elucidate the relation between <em>E. coli</em> adaptation to stresses and c-di-GMP levels, we further observed that O<em>dgcQ</em> with elevated c-di-GMP exhibited stronger tolerances to oxidative, heat, and acid stresses in the planktonic state<em>.</em> These findings further indicate that <em>E. coli</em> with elevated c-di-GMP are more prone to long-term survival under extreme environmental conditions. Previous studies showed that <em>csgD</em> plays an important role in enhancing the long-term survival and persistence of <em>Salmonella</em> in biofilms under stressful conditions [<a href="#pone.0330229.ref037" class="usa-link" aria-describedby="pone.0330229.ref037">37</a>]. Additionally, it has been found that c-di-GMP can downregulate bacterial metabolism by competing with RpoS/RpoD for promoter binding sites and can also directly reprogram bacterial metabolism in an RpoS-independent manner [<a href="#pone.0330229.ref038" class="usa-link" aria-describedby="pone.0330229.ref038">38</a>,<a href="#pone.0330229.ref039" class="usa-link" aria-describedby="pone.0330229.ref039">39</a>]. The low metabolic state of bacteria in mature biofilms is a consequence of a c-di-GMP-regulated survival strategy [<a href="#pone.0330229.ref040" class="usa-link" aria-describedby="pone.0330229.ref040">40</a>]. Therefore, we propose that elevated c-di-GMP levels not only utilize the biofilm's physical barrier but also enhance bacterial stress resistance in both biofilm and planktonic states by upregulating <em>csgD</em> and stress-response genes, and by inducing low-metabolism survival strategies.</p>
<p>The relationship between c-di-GMP and virulence remains controversial [<a href="#pone.0330229.ref041" class="usa-link" aria-describedby="pone.0330229.ref041">41</a>]. This controversy stems from the generalized definition of virulence factors. Another reason for this controversy is the interspecies variation in the role of c-di-GMP. While some studies associate biofilms with increased virulence gene expression, others suggest that biofilm-embedded bacteria adopt a “stealth” phenotype to evade immune detection. Wang <em>et al.</em> demonstrated that bacterial pathogenicity is enhanced by phosphodiesterase-mediated degradation of c-di-GMP in <em>Salmonella</em> [<a href="#pone.0330229.ref042" class="usa-link" aria-describedby="pone.0330229.ref042">42</a>]. A subsequent study proposed that c-di-GMP competes with bacterial siderophores for binding to human lipocalin-2, thereby suppressing host antibacterial activity [<a href="#pone.0330229.ref043" class="usa-link" aria-describedby="pone.0330229.ref043">43</a>]. In contrast, other studies indicated that c-di-GMP binds to host immune proteins STING and DDX41, activating immune responses such as IFN-β and IP-10 release [<a href="#pone.0330229.ref044" class="usa-link" aria-describedby="pone.0330229.ref044">44</a>]. Our co-culture experiments demonstrated that Δ<em>dgcQ</em> (low c-di-GMP) enhanced <em>E. coli</em> adhesion to and invasion of HCT116 cells, while triggering robust proinflammatory responses (IL-1β, IFN-β, IP-10, and NF-κB) (<a href="#pone.0330229.g007" class="usa-link">Fig 7A</a>–<a href="#pone.0330229.g007" class="usa-link">7G</a>). Conversely, O<em>dgcQ</em> (high c-di-GMP) attenuated virulence factors, which in turn reduces immune system stimulation. Similar c-di-GMP-mediated modulation of host responses has been observed in the Gram-negative bacterium <em>Brucella</em> [<a href="#pone.0330229.ref045" class="usa-link" aria-describedby="pone.0330229.ref045">45</a>]. This dichotomy may reflect c-di-GMPʼs ability to modulate pathogen-associated molecular patterns (PAMPs): high c-di-GMP levels reduce flagellin and LPS exposure via EPS encapsulation and motility suppression, thereby limiting activation of immune receptors such as TLR5 and STING [<a href="#pone.0330229.ref046" class="usa-link" aria-describedby="pone.0330229.ref046">46</a>,<a href="#pone.0330229.ref047" class="usa-link" aria-describedby="pone.0330229.ref047">47</a>]. Emerging evidence suggests that virulence phenotypes predominantly drive acute infections by eliciting immune responses [<a href="#pone.0330229.ref048" class="usa-link" aria-describedby="pone.0330229.ref048">48</a>], while low-invasiveness bacteria evade immune recognition and clearance, facilitating chronic infections [<a href="#pone.0330229.ref045" class="usa-link" aria-describedby="pone.0330229.ref045">45</a>,<a href="#pone.0330229.ref049" class="usa-link" aria-describedby="pone.0330229.ref049">49</a>]. Therefore, we speculate that <em>E. coli</em> with high c-di-GMP levels may be more prone to immune escape by inhibiting host inflammatory responses, which is a strategy critical for persistent infections, particularly in biomaterial-associated contexts.</p>
<p>In summary, our study indicates that <em>dgcQ</em> serves as a central regulator of c-di-GMP levels in <em>E. coli</em>, shaping biofilm dynamics, stress resistance, and host-pathogen interactions. The results elucidate stage-specific roles of c-di-GMP in BAIs, suggesting that therapeutic strategies targeting this pathway require temporally precision. Early interventions could inhibit c-di-GMP degradation to suppress motility-driven colonization, while established biofilm require disruption c-di-GMP-mediated stress resistance or EPS synthesis to sensitize bacteria. The immunomodulatory effects of c-di-GMP further highlight its dual targeting potential—both in pathogens and host immune pathways. However, this study has limitations. First, the use of a single <em>E. coli</em> strain (ATCC25922) limits generalizability; future work should validate these findings in clinical isolates and other opportunistic pathogens. Second, focusing solely on PVC surfaces necessitates comparative studies across biomedical materials (e.g., silicone, polyurethane) to assess c-di-GMP's role in device-specific infections.</p></section><section id="sec030"><h2 class="pmc_sec_title">5 Future perspectives</h2>
<p>Our work provides a foundation for developing anti-biofilm therapies tailored to disrupt this critical signaling pathway in BAIs. Targeting c-di-GMP signaling presents a promising strategy to mitigate BAIs. For instance, inhibiting DgcQ activity or promoting PDE expression could destabilize biofilms, rendering bacteria more susceptible to antibiotics and immune clearance. Conversely, engineered materials coated with c-di-GMP-modulating agents such as PDE mimics or DGC inhibitors might prevent bacterial adhesion or promote biofilm dispersal. Future <em>in vitro</em>, <em>in vivo</em>, and translational studies are required to validate the therapeutic potential of these findings against BAIs.</p></section><section id="sec031"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pone.0330229.s001"><div class="caption p">
<span>S1 Table. The sequence of primers for constructing and identifying <em>dgcQ</em> mutant strains.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367115/bin/pone.0330229.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330229.s001.docx</a><sup> (12.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330229.s002"><div class="caption p">
<span>S2 Table. The sequence of primers for qRT-PCR.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367115/bin/pone.0330229.s002.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330229.s002.docx</a><sup> (13.7KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330229.s003"><div class="caption p">
<span>S1 Fig. Identification of ATCC25922 and ATCC25922Δ<em>dgcQ</em> strains.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367115/bin/pone.0330229.s003.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330229.s003.docx</a><sup> (279.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330229.s004"><div class="caption p">
<span>S2. RAW images.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367115/bin/pone.0330229.s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0330229.s004.pdf</a><sup> (356.4KB, pdf) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>All relevant data are within the manuscript and its <a href="#sec031" class="usa-link">Supporting Information</a> files.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This research was funded by Grants from the National Natural Science Foundation of China (No. 81960335); Yunnan Fundamental Research Kunming Medical University Projects (No. 202401AY070001-153); The Pioneer Incubator Fund of Chinese Anti-Cancer Association (No. GWRX-2023-4); Yunnan Health Training Project of High Level Talents (No. H-2024036); Yunnan Fundamental Research Projects (No. 202501AT070133); Yunnan Province Education Department Scientific Research Fund Project (No. 2023Y0565).</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pone.0330229.ref001">
<span class="label">1.</span><cite>Mori T, Yamamoto H, Tabata T, Shimizu T, Endo Y, Hanasawa K, et al. A free radical scavenger, edaravone (MCI-186), diminishes intestinal neutrophil lipid peroxidation and bacterial translocation in a rat hemorrhagic shock model. Crit Care Med. 2005;33(5):1064–9. doi: 10.1097/01.ccm.0000162952.14590.ec

</cite> [<a href="https://doi.org/10.1097/01.ccm.0000162952.14590.ec" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15891337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crit%20Care%20Med&amp;title=A%20free%20radical%20scavenger,%20edaravone%20(MCI-186),%20diminishes%20intestinal%20neutrophil%20lipid%20peroxidation%20and%20bacterial%20translocation%20in%20a%20rat%20hemorrhagic%20shock%20model&amp;author=T%20Mori&amp;author=H%20Yamamoto&amp;author=T%20Tabata&amp;author=T%20Shimizu&amp;author=Y%20Endo&amp;volume=33&amp;issue=5&amp;publication_year=2005&amp;pages=1064-9&amp;pmid=15891337&amp;doi=10.1097/01.ccm.0000162952.14590.ec&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref002">
<span class="label">2.</span><cite>Earley ZM, Akhtar S, Green SJ, Naqib A, Khan O, Cannon AR, et al. Burn Injury Alters the Intestinal Microbiome and Increases Gut Permeability and Bacterial Translocation. PLoS One. 2015;10(7):e0129996. doi: 10.1371/journal.pone.0129996

</cite> [<a href="https://doi.org/10.1371/journal.pone.0129996" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4496078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26154283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Burn%20Injury%20Alters%20the%20Intestinal%20Microbiome%20and%20Increases%20Gut%20Permeability%20and%20Bacterial%20Translocation&amp;author=ZM%20Earley&amp;author=S%20Akhtar&amp;author=SJ%20Green&amp;author=A%20Naqib&amp;author=O%20Khan&amp;volume=10&amp;issue=7&amp;publication_year=2015&amp;pmid=26154283&amp;doi=10.1371/journal.pone.0129996&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref003">
<span class="label">3.</span><cite>Ponziani FR, Zocco MA, Cerrito L, Gasbarrini A, Pompili M. Bacterial translocation in patients with liver cirrhosis: physiology, clinical consequences, and practical implications. Expert Rev Gastroenterol Hepatol. 2018;12(7):641–56. doi: 10.1080/17474124.2018.1481747

</cite> [<a href="https://doi.org/10.1080/17474124.2018.1481747" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29806487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Rev%20Gastroenterol%20Hepatol&amp;title=Bacterial%20translocation%20in%20patients%20with%20liver%20cirrhosis:%20physiology,%20clinical%20consequences,%20and%20practical%20implications&amp;author=FR%20Ponziani&amp;author=MA%20Zocco&amp;author=L%20Cerrito&amp;author=A%20Gasbarrini&amp;author=M%20Pompili&amp;volume=12&amp;issue=7&amp;publication_year=2018&amp;pages=641-56&amp;pmid=29806487&amp;doi=10.1080/17474124.2018.1481747&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref004">
<span class="label">4.</span><cite>Hodges NA, Sussman EM, Stegemann JP. Aseptic and septic prosthetic joint loosening: Impact of biomaterial wear on immune cell function, inflammation, and infection. Biomaterials. 2021;278:121127. doi: 10.1016/j.biomaterials.2021.121127

</cite> [<a href="https://doi.org/10.1016/j.biomaterials.2021.121127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34564034/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomaterials&amp;title=Aseptic%20and%20septic%20prosthetic%20joint%20loosening:%20Impact%20of%20biomaterial%20wear%20on%20immune%20cell%20function,%20inflammation,%20and%20infection&amp;author=NA%20Hodges&amp;author=EM%20Sussman&amp;author=JP%20Stegemann&amp;volume=278&amp;publication_year=2021&amp;pages=121127&amp;pmid=34564034&amp;doi=10.1016/j.biomaterials.2021.121127&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref005">
<span class="label">5.</span><cite>Buetti N, Souweine B, Mermel L, Mimoz O, Ruckly S, Loiodice A, et al. Obesity and risk of catheter-related infections in the ICU. A post hoc analysis of four large randomized controlled trials. Intensive Care Med. 2021;47(4):435–43. doi: 10.1007/s00134-020-06336-4

</cite> [<a href="https://doi.org/10.1007/s00134-020-06336-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33521871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Intensive%20Care%20Med&amp;title=Obesity%20and%20risk%20of%20catheter-related%20infections%20in%20the%20ICU.%20A%20post%20hoc%20analysis%20of%20four%20large%20randomized%20controlled%20trials&amp;author=N%20Buetti&amp;author=B%20Souweine&amp;author=L%20Mermel&amp;author=O%20Mimoz&amp;author=S%20Ruckly&amp;volume=47&amp;issue=4&amp;publication_year=2021&amp;pages=435-43&amp;pmid=33521871&amp;doi=10.1007/s00134-020-06336-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref006">
<span class="label">6.</span><cite>Magill SS, Edwards JR, Bamberg W, Beldavs ZG, Dumyati G, Kainer MA, et al. Multistate point-prevalence survey of health care-associated infections. N Engl J Med. 2014;370(13):1198–208. doi: 10.1056/NEJMoa1306801

</cite> [<a href="https://doi.org/10.1056/NEJMoa1306801" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4648343/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24670166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&amp;title=Multistate%20point-prevalence%20survey%20of%20health%20care-associated%20infections&amp;author=SS%20Magill&amp;author=JR%20Edwards&amp;author=W%20Bamberg&amp;author=ZG%20Beldavs&amp;author=G%20Dumyati&amp;volume=370&amp;issue=13&amp;publication_year=2014&amp;pages=1198-208&amp;pmid=24670166&amp;doi=10.1056/NEJMoa1306801&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref007">
<span class="label">7.</span><cite>Park S-C, Lee M-Y, Kim J-Y, Kim H, Jung M, Shin M-K, et al. Anti-Biofilm Effects of Synthetic Antimicrobial Peptides Against Drug-Resistant <em>Pseudomonas aeruginosa</em> and <em>Staphylococcus aureus</em> Planktonic Cells and Biofilm. Molecules. 2019;24(24):4560. doi: 10.3390/molecules24244560

</cite> [<a href="https://doi.org/10.3390/molecules24244560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6943720/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31842508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Anti-Biofilm%20Effects%20of%20Synthetic%20Antimicrobial%20Peptides%20Against%20Drug-Resistant%20Pseudomonas%20aeruginosa%20and%20Staphylococcus%20aureus%20Planktonic%20Cells%20and%20Biofilm&amp;author=S-C%20Park&amp;author=M-Y%20Lee&amp;author=J-Y%20Kim&amp;author=H%20Kim&amp;author=M%20Jung&amp;volume=24&amp;issue=24&amp;publication_year=2019&amp;pages=4560&amp;pmid=31842508&amp;doi=10.3390/molecules24244560&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref008">
<span class="label">8.</span><cite>Hidron AI, Edwards JR, Patel J, Horan TC, Sievert DM, Pollock DA, et al. NHSN annual update: antimicrobial-resistant pathogens associated with healthcare-associated infections: annual summary of data reported to the National Healthcare Safety Network at the Centers for Disease Control and Prevention, 2006-2007. Infect Control Hosp Epidemiol. 2008;29(11):996–1011. doi: 10.1086/591861

</cite> [<a href="https://doi.org/10.1086/591861" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18947320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Control%20Hosp%20Epidemiol&amp;title=NHSN%20annual%20update:%20antimicrobial-resistant%20pathogens%20associated%20with%20healthcare-associated%20infections:%20annual%20summary%20of%20data%20reported%20to%20the%20National%20Healthcare%20Safety%20Network%20at%20the%20Centers%20for%20Disease%20Control%20and%20Prevention,%202006-2007&amp;author=AI%20Hidron&amp;author=JR%20Edwards&amp;author=J%20Patel&amp;author=TC%20Horan&amp;author=DM%20Sievert&amp;volume=29&amp;issue=11&amp;publication_year=2008&amp;pages=996-1011&amp;pmid=18947320&amp;doi=10.1086/591861&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref009">
<span class="label">9.</span><cite>Weiner LM, Webb AK, Limbago B, Dudeck MA, Patel J, Kallen AJ, et al. Antimicrobial-Resistant Pathogens Associated With Healthcare-Associated Infections: Summary of Data Reported to the National Healthcare Safety Network at the Centers for Disease Control and Prevention, 2011-2014. Infect Control Hosp Epidemiol. 2016;37(11):1288–301. doi: 10.1017/ice.2016.174

</cite> [<a href="https://doi.org/10.1017/ice.2016.174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6857725/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27573805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Control%20Hosp%20Epidemiol&amp;title=Antimicrobial-Resistant%20Pathogens%20Associated%20With%20Healthcare-Associated%20Infections:%20Summary%20of%20Data%20Reported%20to%20the%20National%20Healthcare%20Safety%20Network%20at%20the%20Centers%20for%20Disease%20Control%20and%20Prevention,%202011-2014&amp;author=LM%20Weiner&amp;author=AK%20Webb&amp;author=B%20Limbago&amp;author=MA%20Dudeck&amp;author=J%20Patel&amp;volume=37&amp;issue=11&amp;publication_year=2016&amp;pages=1288-301&amp;pmid=27573805&amp;doi=10.1017/ice.2016.174&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref010">
<span class="label">10.</span><cite>Klingeberg A, Noll I, Willrich N, Feig M, Emrich D, Zill E, et al. Antibiotic-Resistant <em>E. coli</em> in Uncomplicated Community-Acquired Urinary Tract Infection. Dtsch Arztebl Int. 2018;115(29–30):494–500. doi: 10.3238/arztebl.2018.0494

</cite> [<a href="https://doi.org/10.3238/arztebl.2018.0494" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6121086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30135009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dtsch%20Arztebl%20Int&amp;title=Antibiotic-Resistant%20E.%20coli%20in%20Uncomplicated%20Community-Acquired%20Urinary%20Tract%20Infection&amp;author=A%20Klingeberg&amp;author=I%20Noll&amp;author=N%20Willrich&amp;author=M%20Feig&amp;author=D%20Emrich&amp;volume=115&amp;publication_year=2018&amp;pages=494-500&amp;pmid=30135009&amp;doi=10.3238/arztebl.2018.0494&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref011">
<span class="label">11.</span><cite>See I, Freifeld AG, Magill SS. Causative Organisms and Associated Antimicrobial Resistance in Healthcare-Associated, Central Line-Associated Bloodstream Infections From Oncology Settings, 2009-2012. Clin Infect Dis. 2016;62(10):1203–9. doi: 10.1093/cid/ciw113

</cite> [<a href="https://doi.org/10.1093/cid/ciw113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4894695/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26936664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&amp;title=Causative%20Organisms%20and%20Associated%20Antimicrobial%20Resistance%20in%20Healthcare-Associated,%20Central%20Line-Associated%20Bloodstream%20Infections%20From%20Oncology%20Settings,%202009-2012&amp;author=I%20See&amp;author=AG%20Freifeld&amp;author=SS%20Magill&amp;volume=62&amp;issue=10&amp;publication_year=2016&amp;pages=1203-9&amp;pmid=26936664&amp;doi=10.1093/cid/ciw113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref012">
<span class="label">12.</span><cite>Messika J, Magdoud F, Clermont O, Margetis D, Gaudry S, Roux D, et al. Pathophysiology of <em>Escherichia coli</em> ventilator-associated pneumonia: implication of highly virulent extraintestinal pathogenic strains. Intensive Care Med. 2012;38(12):2007–16. doi: 10.1007/s00134-012-2699-5

</cite> [<a href="https://doi.org/10.1007/s00134-012-2699-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23052953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Intensive%20Care%20Med&amp;title=Pathophysiology%20of%20Escherichia%20coli%20ventilator-associated%20pneumonia:%20implication%20of%20highly%20virulent%20extraintestinal%20pathogenic%20strains&amp;author=J%20Messika&amp;author=F%20Magdoud&amp;author=O%20Clermont&amp;author=D%20Margetis&amp;author=S%20Gaudry&amp;volume=38&amp;issue=12&amp;publication_year=2012&amp;pages=2007-16&amp;pmid=23052953&amp;doi=10.1007/s00134-012-2699-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref013">
<span class="label">13.</span><cite>Arciola CR, Campoccia D, Montanaro L. Implant infections: adhesion, biofilm formation and immune evasion. Nat Rev Microbiol. 2018;16(7):397–409. doi: 10.1038/s41579-018-0019-y

</cite> [<a href="https://doi.org/10.1038/s41579-018-0019-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29720707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Microbiol&amp;title=Implant%20infections:%20adhesion,%20biofilm%20formation%20and%20immune%20evasion&amp;author=CR%20Arciola&amp;author=D%20Campoccia&amp;author=L%20Montanaro&amp;volume=16&amp;issue=7&amp;publication_year=2018&amp;pages=397-409&amp;pmid=29720707&amp;doi=10.1038/s41579-018-0019-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref014">
<span class="label">14.</span><cite>Öztürk FY, Darcan C, Kariptaş E. The Determination, Monitoring, Molecular Mechanisms and Formation of Biofilm in <em>E. coli</em>. Braz J Microbiol. 2023;54(1):259–77. doi: 10.1007/s42770-022-00895-y

</cite> [<a href="https://doi.org/10.1007/s42770-022-00895-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9943865/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36577889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Braz%20J%20Microbiol&amp;title=The%20Determination,%20Monitoring,%20Molecular%20Mechanisms%20and%20Formation%20of%20Biofilm%20in%20E.%20coli&amp;author=FY%20%C3%96zt%C3%BCrk&amp;author=C%20Darcan&amp;author=E%20Karipta%C5%9F&amp;volume=54&amp;issue=1&amp;publication_year=2023&amp;pages=259-77&amp;pmid=36577889&amp;doi=10.1007/s42770-022-00895-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref015">
<span class="label">15.</span><cite>Mantoni F, Paiardini A, Brunotti P, D’Angelo C, Cervoni L, Paone A, et al. Insights into the GTP-dependent allosteric control of c-di-GMP hydrolysis from the crystal structure of PA0575 protein from <em>Pseudomonas aeruginosa</em>. Febs j. 2018;285(20):3815–34. doi: 10.1111/febs.14634

</cite> [<a href="https://doi.org/10.1111/febs.14634" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30106221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Febs%20j&amp;title=Insights%20into%20the%20GTP-dependent%20allosteric%20control%20of%20c-di-GMP%20hydrolysis%20from%20the%20crystal%20structure%20of%20PA0575%20protein%20from%20Pseudomonas%20aeruginosa&amp;author=F%20Mantoni&amp;author=A%20Paiardini&amp;author=P%20Brunotti&amp;author=C%20D%E2%80%99Angelo&amp;author=L%20Cervoni&amp;volume=285&amp;issue=20&amp;publication_year=2018&amp;pages=3815-34&amp;pmid=30106221&amp;doi=10.1111/febs.14634&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref016">
<span class="label">16.</span><cite>Zamorano-Sánchez D, Xian W, Lee CK, Salinas M, Thongsomboon W, Cegelski L, et al. Functional Specialization in <em>Vibrio cholerae</em> Diguanylate Cyclases: Distinct Modes of Motility Suppression and c-di-GMP Production. mBio. 2019;10(2):e00670-19. doi: 10.1128/mBio.00670-19

</cite> [<a href="https://doi.org/10.1128/mBio.00670-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6479008/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31015332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Functional%20Specialization%20in%20Vibrio%20cholerae%20Diguanylate%20Cyclases:%20Distinct%20Modes%20of%20Motility%20Suppression%20and%20c-di-GMP%20Production&amp;author=D%20Zamorano-S%C3%A1nchez&amp;author=W%20Xian&amp;author=CK%20Lee&amp;author=M%20Salinas&amp;author=W%20Thongsomboon&amp;volume=10&amp;issue=2&amp;publication_year=2019&amp;pmid=31015332&amp;doi=10.1128/mBio.00670-19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref017">
<span class="label">17.</span><cite>Wang Z, Xie X, Shang D, Xie L, Hua Y, Song L, et al. A c-di-GMP Signaling Cascade Controls Motility, Biofilm Formation, and Virulence in <em>Burkholderia thailandensis</em>. Appl Environ Microbiol. 2022;88(7):e0252921. doi: 10.1128/aem.02529-21

</cite> [<a href="https://doi.org/10.1128/aem.02529-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9004398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35323023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=A%20c-di-GMP%20Signaling%20Cascade%20Controls%20Motility,%20Biofilm%20Formation,%20and%20Virulence%20in%20Burkholderia%20thailandensis&amp;author=Z%20Wang&amp;author=X%20Xie&amp;author=D%20Shang&amp;author=L%20Xie&amp;author=Y%20Hua&amp;volume=88&amp;issue=7&amp;publication_year=2022&amp;pmid=35323023&amp;doi=10.1128/aem.02529-21&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref018">
<span class="label">18.</span><cite>Povolotsky TL, Hengge R. Genome-Based Comparison of Cyclic Di-GMP Signaling in Pathogenic and Commensal <em>Escherichia coli</em> Strains. J Bacteriol. 2015;198(1):111–26. doi: 10.1128/JB.00520-15

</cite> [<a href="https://doi.org/10.1128/JB.00520-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4686209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26303830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Genome-Based%20Comparison%20of%20Cyclic%20Di-GMP%20Signaling%20in%20Pathogenic%20and%20Commensal%20Escherichia%20coli%20Strains&amp;author=TL%20Povolotsky&amp;author=R%20Hengge&amp;volume=198&amp;issue=1&amp;publication_year=2015&amp;pages=111-26&amp;pmid=26303830&amp;doi=10.1128/JB.00520-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref019">
<span class="label">19.</span><cite>Sanchez-Torres V, Hu H, Wood TK. GGDEF proteins YeaI, YedQ, and YfiN reduce early biofilm formation and swimming motility in <em>Escherichia coli</em>. Appl Microbiol Biotechnol. 2011;90(2):651–8. doi: 10.1007/s00253-010-3074-5

</cite> [<a href="https://doi.org/10.1007/s00253-010-3074-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3158426/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21181144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&amp;title=GGDEF%20proteins%20YeaI,%20YedQ,%20and%20YfiN%20reduce%20early%20biofilm%20formation%20and%20swimming%20motility%20in%20Escherichia%20coli&amp;author=V%20Sanchez-Torres&amp;author=H%20Hu&amp;author=TK%20Wood&amp;volume=90&amp;issue=2&amp;publication_year=2011&amp;pages=651-8&amp;pmid=21181144&amp;doi=10.1007/s00253-010-3074-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref020">
<span class="label">20.</span><cite>Yang S, Wu Y, Qu C, Fein JB, He Y, Huang Q, et al. Quantitative analysis of the surficial and adhesion properties of the Gram-negative bacterial species <em>Comamonas testosteroni</em> modulated by c-di-GMP. Colloids Surf B Biointerfaces. 2021;198:111497. doi: 10.1016/j.colsurfb.2020.111497

</cite> [<a href="https://doi.org/10.1016/j.colsurfb.2020.111497" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33296824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Colloids%20Surf%20B%20Biointerfaces&amp;title=Quantitative%20analysis%20of%20the%20surficial%20and%20adhesion%20properties%20of%20the%20Gram-negative%20bacterial%20species%20Comamonas%20testosteroni%20modulated%20by%20c-di-GMP&amp;author=S%20Yang&amp;author=Y%20Wu&amp;author=C%20Qu&amp;author=JB%20Fein&amp;author=Y%20He&amp;volume=198&amp;publication_year=2021&amp;pages=111497&amp;pmid=33296824&amp;doi=10.1016/j.colsurfb.2020.111497&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref021">
<span class="label">21.</span><cite>Liang F, Zhang B, Yang Q, Zhang Y, Zheng D, Zhang L-Q, et al. Cyclic-di-GMP Regulates the Quorum-Sensing System and Biocontrol Activity of <em>Pseudomonas fluorescens</em> 2P24 through the RsmA and RsmE Proteins. Appl Environ Microbiol. 2020;86(24):e02016-20. doi: 10.1128/AEM.02016-20

</cite> [<a href="https://doi.org/10.1128/AEM.02016-20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7688223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33036989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Cyclic-di-GMP%20Regulates%20the%20Quorum-Sensing%20System%20and%20Biocontrol%20Activity%20of%20Pseudomonas%20fluorescens%202P24%20through%20the%20RsmA%20and%20RsmE%20Proteins&amp;author=F%20Liang&amp;author=B%20Zhang&amp;author=Q%20Yang&amp;author=Y%20Zhang&amp;author=D%20Zheng&amp;volume=86&amp;issue=24&amp;publication_year=2020&amp;pmid=33036989&amp;doi=10.1128/AEM.02016-20&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref022">
<span class="label">22.</span><cite>Gjermansen M, Ragas P, Tolker-Nielsen T. Proteins with GGDEF and EAL domains regulate <em>Pseudomonas putida</em> biofilm formation and dispersal. FEMS Microbiol Lett. 2006;265(2):215–24. doi: 10.1111/j.1574-6968.2006.00493.x

</cite> [<a href="https://doi.org/10.1111/j.1574-6968.2006.00493.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17054717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol%20Lett&amp;title=Proteins%20with%20GGDEF%20and%20EAL%20domains%20regulate%20Pseudomonas%20putida%20biofilm%20formation%20and%20dispersal&amp;author=M%20Gjermansen&amp;author=P%20Ragas&amp;author=T%20Tolker-Nielsen&amp;volume=265&amp;issue=2&amp;publication_year=2006&amp;pages=215-24&amp;pmid=17054717&amp;doi=10.1111/j.1574-6968.2006.00493.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref023">
<span class="label">23.</span><cite>Wang T, Cai Z, Shao X, Zhang W, Xie Y, Zhang Y, et al. Pleiotropic Effects of c-di-GMP Content in <em>Pseudomonas syringae</em>. Appl Environ Microbiol. 2019;85(10):e00152-19. doi: 10.1128/AEM.00152-19

</cite> [<a href="https://doi.org/10.1128/AEM.00152-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6498148/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30850427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Pleiotropic%20Effects%20of%20c-di-GMP%20Content%20in%20Pseudomonas%20syringae&amp;author=T%20Wang&amp;author=Z%20Cai&amp;author=X%20Shao&amp;author=W%20Zhang&amp;author=Y%20Xie&amp;volume=85&amp;issue=10&amp;publication_year=2019&amp;pmid=30850427&amp;doi=10.1128/AEM.00152-19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref024">
<span class="label">24.</span><cite>Amin Yavari S, Castenmiller SM, van Strijp JAG, Croes M. Combating Implant Infections: Shifting Focus from Bacteria to Host. Adv Mater. 2020;32(43):e2002962. doi: 10.1002/adma.202002962

</cite> [<a href="https://doi.org/10.1002/adma.202002962" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32914481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Mater&amp;title=Combating%20Implant%20Infections:%20Shifting%20Focus%20from%20Bacteria%20to%20Host&amp;author=S%20Amin%20Yavari&amp;author=SM%20Castenmiller&amp;author=JAG%20van%20Strijp&amp;author=M%20Croes&amp;volume=32&amp;issue=43&amp;publication_year=2020&amp;pmid=32914481&amp;doi=10.1002/adma.202002962&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref025">
<span class="label">25.</span><cite>Tarabal VS, Abud YKD, da Silva FG, da Cruz LF, Fontes GN, da Silva JA, et al. Effect of DMPEI coating against biofilm formation on PVC catheter surface. World J Microbiol Biotechnol. 2023;40(1):6. doi: 10.1007/s11274-023-03799-7

</cite> [<a href="https://doi.org/10.1007/s11274-023-03799-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37932532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=World%20J%20Microbiol%20Biotechnol&amp;title=Effect%20of%20DMPEI%20coating%20against%20biofilm%20formation%20on%20PVC%20catheter%20surface&amp;author=VS%20Tarabal&amp;author=YKD%20Abud&amp;author=FG%20da%20Silva&amp;author=LF%20da%20Cruz&amp;author=GN%20Fontes&amp;volume=40&amp;issue=1&amp;publication_year=2023&amp;pages=6&amp;pmid=37932532&amp;doi=10.1007/s11274-023-03799-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref026">
<span class="label">26.</span><cite>Thorarinsdottir HR, Kander T, Holmberg A, Petronis S, Klarin B. Biofilm formation on three different endotracheal tubes: a prospective clinical trial. Crit Care. 2020;24(1):382. doi: 10.1186/s13054-020-03092-1

</cite> [<a href="https://doi.org/10.1186/s13054-020-03092-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7322705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32600373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crit%20Care&amp;title=Biofilm%20formation%20on%20three%20different%20endotracheal%20tubes:%20a%20prospective%20clinical%20trial&amp;author=HR%20Thorarinsdottir&amp;author=T%20Kander&amp;author=A%20Holmberg&amp;author=S%20Petronis&amp;author=B%20Klarin&amp;volume=24&amp;issue=1&amp;publication_year=2020&amp;pages=382&amp;pmid=32600373&amp;doi=10.1186/s13054-020-03092-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref027">
<span class="label">27.</span><cite>Jahn CE, Charkowski AO, Willis DK. Evaluation of isolation methods and RNA integrity for bacterial RNA quantitation. J Microbiol Methods. 2008;75(2):318–24. doi: 10.1016/j.mimet.2008.07.004

</cite> [<a href="https://doi.org/10.1016/j.mimet.2008.07.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18674572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Microbiol%20Methods&amp;title=Evaluation%20of%20isolation%20methods%20and%20RNA%20integrity%20for%20bacterial%20RNA%20quantitation&amp;author=CE%20Jahn&amp;author=AO%20Charkowski&amp;author=DK%20Willis&amp;volume=75&amp;issue=2&amp;publication_year=2008&amp;pages=318-24&amp;pmid=18674572&amp;doi=10.1016/j.mimet.2008.07.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref028">
<span class="label">28.</span><cite>Wen Y, Wang Y, Chen S, Zhou X, Zhang Y, Yang D, et al. Dysregulation of Cytosolic c-di-GMP in <em>Edwardsiella piscicida</em> Promotes Cellular Non-Canonical Ferroptosis. Front Cell Infect Microbiol. 2022;12:825824. doi: 10.3389/fcimb.2022.825824

</cite> [<a href="https://doi.org/10.3389/fcimb.2022.825824" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8855483/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35186798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Infect%20Microbiol&amp;title=Dysregulation%20of%20Cytosolic%20c-di-GMP%20in%20Edwardsiella%20piscicida%20Promotes%20Cellular%20Non-Canonical%20Ferroptosis&amp;author=Y%20Wen&amp;author=Y%20Wang&amp;author=S%20Chen&amp;author=X%20Zhou&amp;author=Y%20Zhang&amp;volume=12&amp;publication_year=2022&amp;pages=825824&amp;pmid=35186798&amp;doi=10.3389/fcimb.2022.825824&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref029">
<span class="label">29.</span><cite>Sánchez-Clemente R, Guijo MI, Nogales J, Blasco R. Carbon Source Influence on Extracellular pH Changes along Bacterial Cell-Growth. Genes (Basel). 2020;11(11):1292. doi: 10.3390/genes11111292

</cite> [<a href="https://doi.org/10.3390/genes11111292" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7693008/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33142974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20(Basel)&amp;title=Carbon%20Source%20Influence%20on%20Extracellular%20pH%20Changes%20along%20Bacterial%20Cell-Growth&amp;author=R%20S%C3%A1nchez-Clemente&amp;author=MI%20Guijo&amp;author=J%20Nogales&amp;author=R%20Blasco&amp;volume=11&amp;issue=11&amp;publication_year=2020&amp;pages=1292&amp;pmid=33142974&amp;doi=10.3390/genes11111292&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref030">
<span class="label">30.</span><cite>Li S, Duan W, Lei Y, Wang Z, Fu C, He L, et al. Effects of lipid emulsions on the formation of <em>Escherichia coli-Candida albicans</em> mixed-species biofilms on PVC. Sci Rep. 2021;11(1):16929. doi: 10.1038/s41598-021-96385-6

</cite> [<a href="https://doi.org/10.1038/s41598-021-96385-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8376934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34413406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Effects%20of%20lipid%20emulsions%20on%20the%20formation%20of%20Escherichia%20coli-Candida%20albicans%20mixed-species%20biofilms%20on%20PVC&amp;author=S%20Li&amp;author=W%20Duan&amp;author=Y%20Lei&amp;author=Z%20Wang&amp;author=C%20Fu&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=16929&amp;pmid=34413406&amp;doi=10.1038/s41598-021-96385-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref031">
<span class="label">31.</span><cite>Gao Q, Xia L, Wang X, Ye Z, Liu J, Gao S. SodA Contributes to the Virulence of Avian Pathogenic <em>Escherichia coli</em> O2 Strain E058 in Experimentally Infected Chickens. J Bacteriol. 2019;201(6):e00625-18. doi: 10.1128/JB.00625-18

</cite> [<a href="https://doi.org/10.1128/JB.00625-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6398276/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30602490/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=SodA%20Contributes%20to%20the%20Virulence%20of%20Avian%20Pathogenic%20Escherichia%20coli%20O2%20Strain%20E058%20in%20Experimentally%20Infected%20Chickens&amp;author=Q%20Gao&amp;author=L%20Xia&amp;author=X%20Wang&amp;author=Z%20Ye&amp;author=J%20Liu&amp;volume=201&amp;issue=6&amp;publication_year=2019&amp;pmid=30602490&amp;doi=10.1128/JB.00625-18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref032">
<span class="label">32.</span><cite>Wang X, Lin X, Wan Z, Zuo J, Wang Z, Xu Y, et al. Regulation of <em>ydiV</em>-induced biological characteristics permits <em>Escherichia coli</em> evasion of the host STING inflammatory response. Vet Microbiol. 2021;261:109207. doi: 10.1016/j.vetmic.2021.109207

</cite> [<a href="https://doi.org/10.1016/j.vetmic.2021.109207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34419774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vet%20Microbiol&amp;title=Regulation%20of%20ydiV-induced%20biological%20characteristics%20permits%20Escherichia%20coli%20evasion%20of%20the%20host%20STING%20inflammatory%20response&amp;author=X%20Wang&amp;author=X%20Lin&amp;author=Z%20Wan&amp;author=J%20Zuo&amp;author=Z%20Wang&amp;volume=261&amp;publication_year=2021&amp;pages=109207&amp;pmid=34419774&amp;doi=10.1016/j.vetmic.2021.109207&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref033">
<span class="label">33.</span><cite>Caldara M, Belgiovine C, Secchi E, Rusconi R. Environmental, Microbiological, and Immunological Features of Bacterial Biofilms Associated with Implanted Medical Devices. Clin Microbiol Rev. 2022;35(2):e0022120. doi: 10.1128/cmr.00221-20

</cite> [<a href="https://doi.org/10.1128/cmr.00221-20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8768833/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35044203/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Microbiol%20Rev&amp;title=Environmental,%20Microbiological,%20and%20Immunological%20Features%20of%20Bacterial%20Biofilms%20Associated%20with%20Implanted%20Medical%20Devices&amp;author=M%20Caldara&amp;author=C%20Belgiovine&amp;author=E%20Secchi&amp;author=R%20Rusconi&amp;volume=35&amp;issue=2&amp;publication_year=2022&amp;pmid=35044203&amp;doi=10.1128/cmr.00221-20&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref034">
<span class="label">34.</span><cite>Yan C-H, Chen F-H, Yang Y-L, Zhan Y-F, Herman RA, Gong L-C, et al. The Transcription Factor CsgD Contributes to Engineered <em>Escherichia coli</em> Resistance by Regulating Biofilm Formation and Stress Responses. Int J Mol Sci. 2023;24(18):13681. doi: 10.3390/ijms241813681

</cite> [<a href="https://doi.org/10.3390/ijms241813681" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10530992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37761984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=The%20Transcription%20Factor%20CsgD%20Contributes%20to%20Engineered%20Escherichia%20coli%20Resistance%20by%20Regulating%20Biofilm%20Formation%20and%20Stress%20Responses&amp;author=C-H%20Yan&amp;author=F-H%20Chen&amp;author=Y-L%20Yang&amp;author=Y-F%20Zhan&amp;author=RA%20Herman&amp;volume=24&amp;issue=18&amp;publication_year=2023&amp;pages=13681&amp;pmid=37761984&amp;doi=10.3390/ijms241813681&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref035">
<span class="label">35.</span><cite>Hou Y-J, Yang W-S, Hong Y, Zhang Y, Wang D-C, Li D-F. Structural insights into the mechanism of c-di-GMP-bound YcgR regulating flagellar motility in <em>Escherichia coli</em>. J Biol Chem. 2020;295(3):808–21. doi: 10.1074/jbc.RA119.009739

</cite> [<a href="https://doi.org/10.1074/jbc.RA119.009739" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6970932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31836667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Structural%20insights%20into%20the%20mechanism%20of%20c-di-GMP-bound%20YcgR%20regulating%20flagellar%20motility%20in%20Escherichia%20coli&amp;author=Y-J%20Hou&amp;author=W-S%20Yang&amp;author=Y%20Hong&amp;author=Y%20Zhang&amp;author=D-C%20Wang&amp;volume=295&amp;issue=3&amp;publication_year=2020&amp;pages=808-21&amp;pmid=31836667&amp;doi=10.1074/jbc.RA119.009739&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref036">
<span class="label">36.</span><cite>Nieto V, Partridge JD, Severin GB, Lai R-Z, Waters CM, Parkinson JS, et al. Under Elevated c-di-GMP in <em>Escherichia coli</em>, YcgR Alters Flagellar Motor Bias and Speed Sequentially, with Additional Negative Control of the Flagellar Regulon via the Adaptor Protein RssB. J Bacteriol. 2019;202(1):e00578-19. doi: 10.1128/JB.00578-19

</cite> [<a href="https://doi.org/10.1128/JB.00578-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6932234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31611290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Under%20Elevated%20c-di-GMP%20in%20Escherichia%20coli,%20YcgR%20Alters%20Flagellar%20Motor%20Bias%20and%20Speed%20Sequentially,%20with%20Additional%20Negative%20Control%20of%20the%20Flagellar%20Regulon%20via%20the%20Adaptor%20Protein%20RssB&amp;author=V%20Nieto&amp;author=JD%20Partridge&amp;author=GB%20Severin&amp;author=R-Z%20Lai&amp;author=CM%20Waters&amp;volume=202&amp;issue=1&amp;publication_year=2019&amp;pmid=31611290&amp;doi=10.1128/JB.00578-19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref037">
<span class="label">37.</span><cite>Desai SK, Padmanabhan A, Harshe S, Zaidel-Bar R, Kenney LJ. <em>Salmonella</em> biofilms program innate immunity for persistence in Caenorhabditis elegans. Proc Natl Acad Sci U S A. 2019;116(25):12462–7. doi: 10.1073/pnas.1822018116

</cite> [<a href="https://doi.org/10.1073/pnas.1822018116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6589656/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31160462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Salmonella%20biofilms%20program%20innate%20immunity%20for%20persistence%20in%20Caenorhabditis%20elegans&amp;author=SK%20Desai&amp;author=A%20Padmanabhan&amp;author=S%20Harshe&amp;author=R%20Zaidel-Bar&amp;author=LJ%20Kenney&amp;volume=116&amp;issue=25&amp;publication_year=2019&amp;pages=12462-7&amp;pmid=31160462&amp;doi=10.1073/pnas.1822018116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref038">
<span class="label">38.</span><cite>Zlatkov N, Uhlin BE. Absence of Global Stress Regulation in <em>Escherichia coli</em> Promotes Pathoadaptation and Novel c-di-GMP-dependent Metabolic Capability. Sci Rep. 2019;9(1):2600. doi: 10.1038/s41598-019-39580-w

</cite> [<a href="https://doi.org/10.1038/s41598-019-39580-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6385356/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30796316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Absence%20of%20Global%20Stress%20Regulation%20in%20Escherichia%20coli%20Promotes%20Pathoadaptation%20and%20Novel%20c-di-GMP-dependent%20Metabolic%20Capability&amp;author=N%20Zlatkov&amp;author=BE%20Uhlin&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=2600&amp;pmid=30796316&amp;doi=10.1038/s41598-019-39580-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref039">
<span class="label">39.</span><cite>Zlatkov N, Näsman MEC, Uhlin BE. Metabolic and Morphotypic Trade-Offs within the Eco-Evolutionary Dynamics of <em>Escherichia coli</em>. Microbiol Spectr. 2022;10(5):e0067822. doi: 10.1128/spectrum.00678-22

</cite> [<a href="https://doi.org/10.1128/spectrum.00678-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9602443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36169422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol%20Spectr&amp;title=Metabolic%20and%20Morphotypic%20Trade-Offs%20within%20the%20Eco-Evolutionary%20Dynamics%20of%20Escherichia%20coli&amp;author=N%20Zlatkov&amp;author=MEC%20N%C3%A4sman&amp;author=BE%20Uhlin&amp;volume=10&amp;issue=5&amp;publication_year=2022&amp;pmid=36169422&amp;doi=10.1128/spectrum.00678-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref040">
<span class="label">40.</span><cite>Lichtenberg M, Kragh KN, Fritz B, Kirkegaard JB, Tolker-Nielsen T, Bjarnsholt T. Cyclic-di-GMP signaling controls metabolic activity in <em>Pseudomonas aeruginosa</em>. Cell Rep. 2022;41(3):111515. doi: 10.1016/j.celrep.2022.111515

</cite> [<a href="https://doi.org/10.1016/j.celrep.2022.111515" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36260996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Cyclic-di-GMP%20signaling%20controls%20metabolic%20activity%20in%20Pseudomonas%20aeruginosa&amp;author=M%20Lichtenberg&amp;author=KN%20Kragh&amp;author=B%20Fritz&amp;author=JB%20Kirkegaard&amp;author=T%20Tolker-Nielsen&amp;volume=41&amp;issue=3&amp;publication_year=2022&amp;pages=111515&amp;pmid=36260996&amp;doi=10.1016/j.celrep.2022.111515&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref041">
<span class="label">41.</span><cite>Hall CL, Lee VT. Cyclic-di-GMP regulation of virulence in bacterial pathogens. Wiley Interdiscip Rev RNA. 2018;9(1):10.1002/wrna.1454. doi: 10.1002/wrna.1454

</cite> [<a href="https://doi.org/10.1002/wrna.1454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5739959/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28990312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Wiley%20Interdiscip%20Rev%20RNA&amp;title=Cyclic-di-GMP%20regulation%20of%20virulence%20in%20bacterial%20pathogens&amp;author=CL%20Hall&amp;author=VT%20Lee&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pmid=28990312&amp;doi=10.1002/wrna.1454&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref042">
<span class="label">42.</span><cite>Dai Y, Liu R, Yue Y, Song N, Jia H, Ma Z, et al. A c-di-GMP binding effector STM0435 modulates flagellar motility and pathogenicity in <em>Salmonella</em>. Virulence. 2024;15(1):2331265. doi: 10.1080/21505594.2024.2331265

</cite> [<a href="https://doi.org/10.1080/21505594.2024.2331265" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10978029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38532247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virulence&amp;title=A%20c-di-GMP%20binding%20effector%20STM0435%20modulates%20flagellar%20motility%20and%20pathogenicity%20in%20Salmonella&amp;author=Y%20Dai&amp;author=R%20Liu&amp;author=Y%20Yue&amp;author=N%20Song&amp;author=H%20Jia&amp;volume=15&amp;issue=1&amp;publication_year=2024&amp;pages=2331265&amp;pmid=38532247&amp;doi=10.1080/21505594.2024.2331265&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref043">
<span class="label">43.</span><cite>Li W, Cui T, Hu L, Wang Z, Li Z, He Z-G. Cyclic diguanylate monophosphate directly binds to human siderocalin and inhibits its antibacterial activity. Nat Commun. 2015;6:8330. doi: 10.1038/ncomms9330

</cite> [<a href="https://doi.org/10.1038/ncomms9330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4595737/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26390966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Cyclic%20diguanylate%20monophosphate%20directly%20binds%20to%20human%20siderocalin%20and%20inhibits%20its%20antibacterial%20activity&amp;author=W%20Li&amp;author=T%20Cui&amp;author=L%20Hu&amp;author=Z%20Wang&amp;author=Z%20Li&amp;volume=6&amp;publication_year=2015&amp;pages=8330&amp;pmid=26390966&amp;doi=10.1038/ncomms9330&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref044">
<span class="label">44.</span><cite>Cui T, Cang H, Yang B, He Z-G. Cyclic Dimeric Guanosine Monophosphate: Activation and Inhibition of Innate Immune Response. J Innate Immun. 2019;11(3):242–8. doi: 10.1159/000492679

</cite> [<a href="https://doi.org/10.1159/000492679" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6738221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30253386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Innate%20Immun&amp;title=Cyclic%20Dimeric%20Guanosine%20Monophosphate:%20Activation%20and%20Inhibition%20of%20Innate%20Immune%20Response&amp;author=T%20Cui&amp;author=H%20Cang&amp;author=B%20Yang&amp;author=Z-G%20He&amp;volume=11&amp;issue=3&amp;publication_year=2019&amp;pages=242-8&amp;pmid=30253386&amp;doi=10.1159/000492679&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref045">
<span class="label">45.</span><cite>Khan M, Harms JS, Marim FM, Armon L, Hall CL, Liu Y-P, et al. The Bacterial Second Messenger Cyclic di-GMP Regulates <em>Brucella</em> Pathogenesis and Leads to Altered Host Immune Response. Infect Immun. 2016;84(12):3458–70. doi: 10.1128/IAI.00531-16

</cite> [<a href="https://doi.org/10.1128/IAI.00531-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5116723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27672085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&amp;title=The%20Bacterial%20Second%20Messenger%20Cyclic%20di-GMP%20Regulates%20Brucella%20Pathogenesis%20and%20Leads%20to%20Altered%20Host%20Immune%20Response&amp;author=M%20Khan&amp;author=JS%20Harms&amp;author=FM%20Marim&amp;author=L%20Armon&amp;author=CL%20Hall&amp;volume=84&amp;issue=12&amp;publication_year=2016&amp;pages=3458-70&amp;pmid=27672085&amp;doi=10.1128/IAI.00531-16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref046">
<span class="label">46.</span><cite>Li M-Z, Wen X-Y, Liu X-Q, Wang Y-Q, Yan L. LPS-Induced Activation of the cGAS-STING Pathway is Regulated by Mitochondrial Dysfunction and Mitochondrial DNA Leakage in Endometritis. J Inflamm Res. 2022;15:5707–20. doi: 10.2147/JIR.S374318

</cite> [<a href="https://doi.org/10.2147/JIR.S374318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9550576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36238763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Inflamm%20Res&amp;title=LPS-Induced%20Activation%20of%20the%20cGAS-STING%20Pathway%20is%20Regulated%20by%20Mitochondrial%20Dysfunction%20and%20Mitochondrial%20DNA%20Leakage%20in%20Endometritis&amp;author=M-Z%20Li&amp;author=X-Y%20Wen&amp;author=X-Q%20Liu&amp;author=Y-Q%20Wang&amp;author=L%20Yan&amp;volume=15&amp;publication_year=2022&amp;pages=5707-20&amp;pmid=36238763&amp;doi=10.2147/JIR.S374318&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref047">
<span class="label">47.</span><cite>Benedikz EK, Bailey D, Cook CNL, Gonçalves-Carneiro D, Buckner MMC, Blair JMA, et al. Bacterial flagellin promotes viral entry via an NF-kB and Toll Like Receptor 5 dependent pathway. Sci Rep. 2019;9(1):7903. doi: 10.1038/s41598-019-44263-7

</cite> [<a href="https://doi.org/10.1038/s41598-019-44263-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6536546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31133714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Bacterial%20flagellin%20promotes%20viral%20entry%20via%20an%20NF-kB%20and%20Toll%20Like%20Receptor%205%20dependent%20pathway&amp;author=EK%20Benedikz&amp;author=D%20Bailey&amp;author=CNL%20Cook&amp;author=D%20Gon%C3%A7alves-Carneiro&amp;author=MMC%20Buckner&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=7903&amp;pmid=31133714&amp;doi=10.1038/s41598-019-44263-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref048">
<span class="label">48.</span><cite>Lamprokostopoulou A, Monteiro C, Rhen M, Römling U. Cyclic di-GMP signalling controls virulence properties of Salmonella enterica serovar Typhimurium at the mucosal lining. Environ Microbiol. 2010;12(1):40–53. doi: 10.1111/j.1462-2920.2009.02032.x

</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2009.02032.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19691499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&amp;title=Cyclic%20di-GMP%20signalling%20controls%20virulence%20properties%20of%20Salmonella%20enterica%20serovar%20Typhimurium%20at%20the%20mucosal%20lining&amp;author=A%20Lamprokostopoulou&amp;author=C%20Monteiro&amp;author=M%20Rhen&amp;author=U%20R%C3%B6mling&amp;volume=12&amp;issue=1&amp;publication_year=2010&amp;pages=40-53&amp;pmid=19691499&amp;doi=10.1111/j.1462-2920.2009.02032.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330229.ref049">
<span class="label">49.</span><cite>She P, Liu Y, Luo Z, Chen L, Zhou L, Hussain Z, et al. <em>PA2146</em> Gene Knockout Is Associated With <em>Pseudomonas aeruginosa</em> Pathogenicity in Macrophage and Host Immune Response. Front Cell Infect Microbiol. 2020;10:559803. doi: 10.3389/fcimb.2020.559803

</cite> [<a href="https://doi.org/10.3389/fcimb.2020.559803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7579411/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33134189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Infect%20Microbiol&amp;title=PA2146%20Gene%20Knockout%20Is%20Associated%20With%20Pseudomonas%20aeruginosa%20Pathogenicity%20in%20Macrophage%20and%20Host%20Immune%20Response&amp;author=P%20She&amp;author=Y%20Liu&amp;author=Z%20Luo&amp;author=L%20Chen&amp;author=L%20Zhou&amp;volume=10&amp;publication_year=2020&amp;pages=559803&amp;pmid=33134189&amp;doi=10.3389/fcimb.2020.559803&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Table. The sequence of primers for constructing and identifying <em>dgcQ</em> mutant strains.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367115/bin/pone.0330229.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330229.s001.docx</a><sup> (12.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S2 Table. The sequence of primers for qRT-PCR.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367115/bin/pone.0330229.s002.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330229.s002.docx</a><sup> (13.7KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S1 Fig. Identification of ATCC25922 and ATCC25922Δ<em>dgcQ</em> strains.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367115/bin/pone.0330229.s003.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330229.s003.docx</a><sup> (279.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S2. RAW images.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367115/bin/pone.0330229.s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0330229.s004.pdf</a><sup> (356.4KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All relevant data are within the manuscript and its <a href="#sec031" class="usa-link">Supporting Information</a> files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS One are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pone.0330229"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pone.0330229.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12367115/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12367115/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367115%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367115/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12367115/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12367115/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40833985/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12367115/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40833985/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12367115/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12367115/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="rBx5MfF9Qj8xIZBt04hVcVg64Vciw14Vi8ltSi7JeD4IZ9tTlh71vNxKEM56HgbV">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
