
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Computational structure prediction of lanthipeptides with NMR data reveals underappreciated peptide flexibility - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E480548AEEB363058054001EFFD91F.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="prosci">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359204/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Protein Science : A Publication of the Protein Society">
<meta name="citation_title" content="Computational structure prediction of lanthipeptides with NMR data reveals underappreciated peptide flexibility">
<meta name="citation_author" content="Claiborne W Tydings">
<meta name="citation_author_institution" content="Department of Chemistry, Center for Structural Biology, Institute of Chemical Biology, Vanderbilt University, Nashville, Tennessee, USA">
<meta name="citation_author" content="Jens Meiler">
<meta name="citation_author_institution" content="Department of Chemistry, Center for Structural Biology, Institute of Chemical Biology, Vanderbilt University, Nashville, Tennessee, USA">
<meta name="citation_author_institution" content="Department of Pharmacology, Center for Structural Biology, Institute of Chemical Biology, Center for Applied Artificial Intelligence in Protein Dynamics, Vanderbilt University, Nashville, Tennessee, USA">
<meta name="citation_author_institution" content="Institute for Drug Discovery, Faculty of Medicine, Faculty of Mathematics and Informatics, Faculty of Chemistry and Mineralogy, Leipzig University Medical School, Leipzig, Germany">
<meta name="citation_author_institution" content="Center for Scalable Data Analytics and Artificial Intelligence ScaDS.AI and School of Embedded Composite Artificial Intelligence SECAI, Dresden, Germany">
<meta name="citation_author" content="Allison S Walker">
<meta name="citation_author_institution" content="Department of Chemistry, Center for Structural Biology, Institute of Chemical Biology, Vanderbilt University, Nashville, Tennessee, USA">
<meta name="citation_author_institution" content="Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee, USA">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="34">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70252">
<meta name="citation_doi" content="10.1002/pro.70252">
<meta name="citation_pmid" content="40823923">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359204/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359204/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359204/pdf/PRO-34-e70252.pdf">
<meta name="description" content="Lanthipeptides are a class of thioether‐containing ribosomally synthesized and post‐translationally modified peptides, which often have antibiotic activity. As a potential starting point for therapeutics, interest in engineering lanthipeptides is ...">
<meta name="og:title" content="Computational structure prediction of lanthipeptides with NMR data reveals underappreciated peptide flexibility">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Lanthipeptides are a class of thioether‐containing ribosomally synthesized and post‐translationally modified peptides, which often have antibiotic activity. As a potential starting point for therapeutics, interest in engineering lanthipeptides is ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359204/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12359204">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/pro.70252"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/PRO-34-e70252.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359204%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12359204/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12359204/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359204/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-prosci.gif" alt="Protein Science : A Publication of the Protein Society logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Protein Science : A Publication of the Protein Society" title="Link to Protein Science : A Publication of the Protein Society" shape="default" href="http://www.proteinscience.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Protein Sci</button></div>. 2025 Aug 18;34(9):e70252. doi: <a href="https://doi.org/10.1002/pro.70252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/pro.70252</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Protein%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Protein%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Computational structure prediction of lanthipeptides with NMR data reveals underappreciated peptide flexibility</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tydings%20CW%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Claiborne W Tydings</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Claiborne W Tydings</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, Center for Structural Biology, Institute of Chemical Biology, Vanderbilt University, Nashville, Tennessee, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tydings%20CW%22%5BAuthor%5D" class="usa-link"><span class="name western">Claiborne W Tydings</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meiler%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jens Meiler</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jens Meiler</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, Center for Structural Biology, Institute of Chemical Biology, Vanderbilt University, Nashville, Tennessee, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Pharmacology, Center for Structural Biology, Institute of Chemical Biology, Center for Applied Artificial Intelligence in Protein Dynamics, Vanderbilt University, Nashville, Tennessee, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Institute for Drug Discovery, Faculty of Medicine, Faculty of Mathematics and Informatics, Faculty of Chemistry and Mineralogy, Leipzig University Medical School, Leipzig, Germany</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Center for Scalable Data Analytics and Artificial Intelligence ScaDS.AI and School of Embedded Composite Artificial Intelligence SECAI, Dresden, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meiler%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jens Meiler</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Walker%20AS%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Allison S Walker</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Allison S Walker</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, Center for Structural Biology, Institute of Chemical Biology, Vanderbilt University, Nashville, Tennessee, USA</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Walker%20AS%22%5BAuthor%5D" class="usa-link"><span class="name western">Allison S Walker</span></a>
</div>
</div>
<sup>1,</sup><sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="pro70252-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, Center for Structural Biology, Institute of Chemical Biology, Vanderbilt University, Nashville, Tennessee, USA</div>
<div id="pro70252-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Pharmacology, Center for Structural Biology, Institute of Chemical Biology, Center for Applied Artificial Intelligence in Protein Dynamics, Vanderbilt University, Nashville, Tennessee, USA</div>
<div id="pro70252-aff-0003">
<sup>
<sup>3</sup>
</sup>Institute for Drug Discovery, Faculty of Medicine, Faculty of Mathematics and Informatics, Faculty of Chemistry and Mineralogy, Leipzig University Medical School, Leipzig, Germany</div>
<div id="pro70252-aff-0004">
<sup>
<sup>4</sup>
</sup>Center for Scalable Data Analytics and Artificial Intelligence ScaDS.AI and School of Embedded Composite Artificial Intelligence SECAI, Dresden, Germany</div>
<div id="pro70252-aff-0005">
<sup>
<sup>5</sup>
</sup>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, 
Allison S. Walker, Vanderbilt University, 1234 Stevenson Center Lane, Nashville, TN, 37240‐0001, USA. 
Email: <span>allison.s.walker@vanderbilt.edu</span>
, 
Jens Meiler, Vanderbilt University, 465 21st Ave South, BIOSCI/MRBIII, Room 5144B, Nashville, TN, 37232‐8725, USA. 
Email: <span>jens.meiler@vanderbilt.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 27; Received 2025 Mar 11; Accepted 2025 Jul 20; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Protein Science</em> published by Wiley Periodicals LLC on behalf of The Protein Society.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12359204  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40823923/" class="usa-link">40823923</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Lanthipeptides are a class of thioether‐containing ribosomally synthesized and post‐translationally modified peptides, which often have antibiotic activity. As a potential starting point for therapeutics, interest in engineering lanthipeptides is growing. Our inability to computationally model and design lanthipeptides in molecular modeling and design software such as Rosetta limits our ability to rationally design lanthipeptides for drug discovery campaigns. We propose that implementing support for the lanthionine rings and dehydrated amino acids found in lanthipeptides will enable accurate lanthipeptide modeling with Rosetta. We find that when compared to the ensembles of lanthipeptides with NMR‐determined structures in the PDB, lanthipeptide ensembles generated with Rosetta have similar experimental agreement, lower Rosetta energy scores, and greater flexibility. Our use of ensemble‐averaged NOE distances instead of requiring individual structures to satisfy all NOE restraints was key for revealing the flexibility of these peptides. Our Rosetta lanthipeptide ensembles show increased flexibility in non‐cyclized peptide regions as well as increased lanthionine ring flexibility when internal hydrogen bonds are absent and glycine residues are present. Support for lanthipeptides in Rosetta enables the design and modeling of lanthipeptides in Rosetta for therapeutic development.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> conformation analysis, conjugation, cyclic peptide, ensemble, flexible, lanthipeptide, NMR, non‐canonical amino acid, Rosetta, sidechain, structure prediction</p></section></section><section id="pro70252-sec-0001"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Antibiotic resistance is a global health problem that contributes to an increase in the number and mortality of bacterial infections (Martens &amp; Demain, <a href="#pro70252-bib-0017" class="usa-link" aria-describedby="pro70252-bib-0017">2017</a>). To combat bacterial infections, natural products have long been the source of new antibiotics. For example, nisin is a lanthionine ring‐containing ribosomally synthesized and post‐translationally modified peptide (RiPP) with antibiotic properties that has been used in the food industry since 1953 (Shin et al., <a href="#pro70252-bib-0028" class="usa-link" aria-describedby="pro70252-bib-0028">2016</a>). Nisin and other RiPPs with lanthionine rings are called lanthipeptides. The properties of lanthipeptides are tuned by their enzymatically introduced thioether sidechain conjugations, called lanthionine rings, that conformationally constrain the lanthipeptides and confer proteolytic resistance (Hegemann et al., <a href="#pro70252-bib-0011" class="usa-link" aria-describedby="pro70252-bib-0011">2019</a>). These rings are formed via dehydration of a serine or threonine residue and subsequent nucleophilic attack of the dehydrated amino acid in a Michael addition reaction. Both the lanthionine rings and the dehydrated amino acids can appear in the final peptide product (Arnison et al., <a href="#pro70252-bib-0003" class="usa-link" aria-describedby="pro70252-bib-0003">2013</a>) (Figure <a href="#pro70252-fig-0001" class="usa-link">1</a>). In attempts to gain insights into the mechanisms of lanthipeptide biological activity, the structures of several lanthipeptides in solution have been determined, primarily with NMR spectroscopy (Acedo et al., <a href="#pro70252-bib-0001" class="usa-link" aria-describedby="pro70252-bib-0001">2019</a>; Bobeica et al., <a href="#pro70252-bib-0006" class="usa-link" aria-describedby="pro70252-bib-0006"><sup>2020</sup></a>; Pei et al., <a href="#pro70252-bib-0020" class="usa-link" aria-describedby="pro70252-bib-0020"><sup>2022</sup></a>; Shenkarev et al., <a href="#pro70252-bib-0027" class="usa-link" aria-describedby="pro70252-bib-0027"><sup>2010</sup></a>; Zimmermann &amp; Jung, <a href="#pro70252-bib-0036" class="usa-link" aria-describedby="pro70252-bib-0036"><sup>1997</sup></a>). These structures revealed that lanthipeptides have diverse conformations with either large alpha helical segments or a lack of traditional secondary structure. Additionally, lanthipeptides have been applied in bioengineering, scaffolding, and library screening (Hegemann et al., <a href="#pro70252-bib-0011" class="usa-link" aria-describedby="pro70252-bib-0011">2019</a>; Hetrick et al., <a href="#pro70252-bib-0012" class="usa-link" aria-describedby="pro70252-bib-0012"><sup>2018</sup></a>; Schmitt et al., <a href="#pro70252-bib-0024" class="usa-link" aria-describedby="pro70252-bib-0024"><sup>2019</sup></a>; Urban et al., <a href="#pro70252-bib-0031" class="usa-link" aria-describedby="pro70252-bib-0031"><sup>2017</sup></a>). However, these approaches have not utilized computational modeling of lanthipeptides to inform their engineering strategies.</p>
<figure class="fig xbox font-sm" id="pro70252-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/639d/12359204/e39ce8f30c2b/PRO-34-e70252-g008.jpg" loading="lazy" id="jats-graphic-1" height="303" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/pro70252-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Lanthipeptides are ribosomally synthesized and post‐translationally modified peptides. (a) Lanthipeptides contain a leader peptide for enzymatic recognition and a core peptide that is modified. Serine and threonine residues are dehydrated in lanthipeptides and serve as Michael acceptors for nucleophilic attack by cysteine residues to form lanthionine rings. (b) For lanthionine rings, the important parameters for modeling the cyclization are the new bond distance and the two angles and three dihedrals involving this bond. (c) For methyllanthionine rings, the addition of bond angle terms and an improper dihedral term involving the methyl groups helps to maintain the proper geometry. (d) The dehydrated amino acids are often seen in the final lanthipeptide product and are important to model. Shown here are dehydroalanine (top) and dehydrobutyrine (bottom). Created in BioRender. Tydings, C. (2025) <a href="https://BioRender.com/uoltx29" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/uoltx29</a>
</p></figcaption></figure><p>Currently, many molecular modeling and design software applications such as Rosetta (Leman et al., <a href="#pro70252-bib-0016" class="usa-link" aria-describedby="pro70252-bib-0016">2020</a>) cannot model lanthipeptides. Implementation of lanthipeptide modeling in Rosetta would enhance our abilities to both predict the structures of lanthipeptides and perform structure‐based design of lanthipeptides. To enable lanthipeptide modeling in Rosetta, we implement lanthipeptide parameters for lanthionine rings and dehydrated amino acids in the Rosetta codebase. We benchmark Rosetta lanthipeptide structure prediction on 10 lanthipeptides with NMR‐determined structures in the PDB. We find that when compared to lanthipeptide ensembles in the PDB, Rosetta lanthipeptide ensembles refined with NMR data have similar agreement with experimental data, lower Rosetta energies, and higher flexibility. The higher lanthipeptide flexibility is a result of using ensemble‐averaged NOE distances when fitting a lanthipeptide ensemble instead of requiring individual structures to satisfy all NOE restraints. The Rosetta lanthipeptide ensembles show increased flexibility in non‐cyclized peptide regions as well as increased lanthionine ring flexibility when internal hydrogen bonds are absent, and glycine residues are present. The lower Rosetta energy scores indicate that the Rosetta‐predicted ensembles likely have lower free energy than ensembles deposited in the PDB and that lanthipeptides are more flexible than existing ensembles suggest.</p></section><section id="pro70252-sec-0002"><h2 class="pmc_sec_title">2. RESULTS</h2>
<section id="pro70252-sec-0003"><h3 class="pmc_sec_title">2.1. Lanthipeptide parameterization</h3>
<p>To accurately model the lanthionine ring sidechain conjugations and the dehydrated amino acids found in lanthipeptides, the introduction of new force field terms is required. For sidechain conjugations such as a lanthionine ring, these terms include the distance of the new C–S bond, two angles adjacent to the bond, and the three dihedrals defining the bond's conformation (Alford et al., <a href="#pro70252-bib-0002" class="usa-link" aria-describedby="pro70252-bib-0002">2017</a>). Parameters were taken from the Amber (Tian et al., <a href="#pro70252-bib-0030" class="usa-link" aria-describedby="pro70252-bib-0030">2020</a>) protein sidechain potentials for the torsion angles and Rosetta cartesian terms (Alford et al., <a href="#pro70252-bib-0002" class="usa-link" aria-describedby="pro70252-bib-0002">2017</a>) for the bond distances and angles. For methyllanthionine, an improper dihedral term and additional angle terms involving the methyl group aided in the accurate modeling of its geometry (Figure <a href="#pro70252-fig-0001" class="usa-link">1</a>). To model the dehydrated amino acids found in lanthipeptides, new Ramachandran tables were generated. The M05‐2X/6‐311G(d,p) functional was used to perform a 2D phi and psi angle scan for dehydroalanine and dehydrobutyrine (Figure <a href="#pro70252-fig-0002" class="usa-link">2</a>) because that functional has previously been shown to accurately capture the torsional potentials of protein backbones (Tian et al., <a href="#pro70252-bib-0030" class="usa-link" aria-describedby="pro70252-bib-0030">2020</a>). The nature of the sp<sup>2</sup> alpha carbon for dehydroalanine and dehydrobutyrine amino acids alters their Ramachandran potentials compared to the canonical amino acids. Additionally, the Ramachandran potential of dehydrobutyrine is further restrained to avoid steric clashes between the gamma carbon and the backbone. After implementing these parameters in Rosetta, we benchmarked lanthipeptide structure prediction with Rosetta for 10 lanthipeptides with NMR‐determined structures in the PDB.</p>
<figure class="fig xbox font-sm" id="pro70252-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/639d/12359204/268419a866e0/PRO-34-e70252-g003.jpg" loading="lazy" id="jats-graphic-3" height="291" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/pro70252-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Dehydrated amino acids Ramachandran plots were generated in a 2D scan with the M05‐2X/6‐311g(d,p) functional in the context of neighboring alanine residues. (a) The Ramachandran plot for dehydroalanine. (b) The Ramachandran plot for dehydrobutyrine.</p></figcaption></figure></section><section id="pro70252-sec-0004"><h3 class="pmc_sec_title">2.2. Lanthipeptide ensemble generation with Rosetta</h3>
<p>To benchmark the performance of lanthipeptide structure prediction with Rosetta, we first used Rosetta to predict the structures of 10 lanthipeptides with NMR‐determined structures in the PDB. Helical and non‐helical lanthipeptides have quite different structures. Thus, we use different structure prediction protocols for these differently structured peptides. For helical lanthipeptides, lanthionine rings commonly stabilize helices at the ends of helical segments (Figure <a href="#pro70252-fig-0003" class="usa-link">3</a>). Specific peptide regions between, on the edges, or within helical segments are flexible. These flexible regions often contain helix breakers such as glycine and proline. The non‐helical peptides have various ring topologies such as adjacent, overlapping, and nested rings and differing numbers and sizes of these rings (Figure <a href="#pro70252-fig-0003" class="usa-link">3</a>). The presence of unstructured regions and complex ring topologies in non‐helical lanthipeptides requires far more sampling to explore the larger potential conformational space of the lanthionine rings and the flexible, non‐cyclized peptide regions. Regardless of the structure prediction algorithm, the process of determining an ensemble that satisfies the experimental data is the same and mirrors previously published Monte Carlo protocols to traverse the ensemble landscape guided by experimental data. This protocol is summarized in the methods section (Vortmeier et al., <a href="#pro70252-bib-0032" class="usa-link" aria-describedby="pro70252-bib-0032">2015</a>).</p>
<figure class="fig xbox font-sm" id="pro70252-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/639d/12359204/74b15eee5d6e/PRO-34-e70252-g004.jpg" loading="lazy" id="jats-graphic-5" height="460" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/pro70252-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Lanthipeptides can adopt various topologies. (a) Lanthipeptide topologies include adjacent rings, overlapping rings, and nested rings. (b) Some lanthipeptides include large helical segments.</p></figcaption></figure><p>After using Rosetta to predict lanthipeptide structures, we selected an ensemble of lanthipeptide structures based on agreement with experimental NOE data. Instead of requiring each member of the ensemble to satisfy all experimental data, we optimized the lanthipeptide ensemble to satisfy ensemble averaged distance restraints by using an ensemble averaged NOE distance (Equation <a href="#pro70252-disp-0001" class="usa-link">1</a>) (Ashkinadze et al., <a href="#pro70252-bib-0004" class="usa-link" aria-describedby="pro70252-bib-0004">2022</a>). Compared to the ensembles in the PDB, the Rosetta‐generated ensembles have similar agreement with experimental data, higher flexibility, and lower Rosetta energies (Figure <a href="#pro70252-fig-0004" class="usa-link">4</a>). The responsible score term for the lower Rosetta energies can vary. The score terms that are commonly, but not always, lower in energy include fa_atr, fa_rep, fa_sol, fa_elec, and rama_prepro (Figure <a href="#pro70252-supitem-0001" class="usa-link">S1</a>). Additionally, as a control, we relaxed Rosetta‐generated lanthipeptide conformations with experimental NOE constraints. From the resulting lanthipeptide conformations, we selected 20 structures that agreed well with the NOE constraints and had low Rosetta energy scores as our control ensemble. For this ensemble, we see similar agreement with experimental data, higher flexibility, and lower Rosetta energies compared to the PDB ensemble. However, optimization to the ensemble averaged distance constraints resulted in lower Rosetta energies (Figure <a href="#pro70252-fig-0004" class="usa-link">4</a>). Overall, these data indicate that by optimizing an ensemble to satisfy experimental data instead of requiring all structures to individually satisfy those data, we can identify multiple low energy conformations that altogether satisfy the experimental data.</p>
<table class="disp-formula p" id="pro70252-disp-0001"><tr>
<td class="formula"><math id="jats-math-1" display="block" overflow="linebreak"><mrow><mrow><msubsup><mi>d</mi><mi>αβ</mi><mo>*</mo></msubsup><mo linebreak="goodbreak">=</mo><msup><mfenced open="(" close=")"><mrow><mfrac><mn>1</mn><mi>N</mi></mfrac><munderover><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></munderover><msup><mfenced open="(" close=")"><msubsup><mi>d</mi><mi>αβ</mi><mi>i</mi></msubsup></mfenced><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mfenced><mrow><mo>−</mo><mfrac><mn>1</mn><mn>6</mn></mfrac></mrow></msup></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline" overflow="linebreak"><mrow><mrow><msubsup><mi mathvariant="normal">d</mi><mi>αβ</mi><mo>*</mo></msubsup></mrow></mrow></math></span> is the ensemble averaged distance where <em>N</em> is the number of equally populated protein states and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline" overflow="linebreak"><mrow><mrow><msubsup><mi mathvariant="normal">d</mi><mi>αβ</mi><mi mathvariant="normal">i</mi></msubsup></mrow></mrow></math></span> is the distance between atoms α and β in state <em>i</em>.</p>
<figure class="fig xbox font-sm" id="pro70252-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/639d/12359204/ea54bd95bfa1/PRO-34-e70252-g009.jpg" loading="lazy" id="jats-graphic-7" height="470" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/pro70252-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of global ensemble features between PDB and Rosetta lanthipeptide ensembles. (a) The minimum Rosetta energy score for PDB and Rosetta ensemble members. (b) The ensemble NOE agreement of PDB and Rosetta ensembles. (c) The backbone RMSD to the lowest scoring ensemble member for PDB and Rosetta ensembles.</p></figcaption></figure></section><section id="pro70252-sec-0005"><h3 class="pmc_sec_title">2.3. Structure prediction of helical lanthipeptides</h3>
<p>We used Rosetta to predict the structure of four lanthipeptides with large helical portions. We found that canonicalized AlphaFold2 (Jumper et al., <a href="#pro70252-bib-0014" class="usa-link" aria-describedby="pro70252-bib-0014">2021</a>) models of these peptides provided excellent starting points for modeling these peptides, as AlphaFold can successfully identify the helical nature of these peptides. We then used Rosetta to form the lanthionine rings, add the dehydrated amino acids, and sample different lanthipeptide conformations. For these four helical lanthipeptides, sampled lanthionine ring conformations primarily deviated one angstrom or less from the conformation of the lanthionine ring in the PDB. For the global structures of these lanthipeptides, peptide conformations that deviated by up to 6 Å or more from the conformation in the PDB (Figure <a href="#pro70252-fig-0005" class="usa-link">5</a>). In contrast, the structures within the PDB ensemble deviate by 2 Å or more within the ensemble, and relaxing these PDBs in Rosetta moves their conformation away from the PDB conformation (Figure <a href="#pro70252-fig-0004" class="usa-link">4</a> and <a href="#pro70252-supitem-0001" class="usa-link">S2</a>). The flexibility of these peptides can be seen by comparing the lanthipeptide ensembles in the PDB to the ensembles generated (Figure <a href="#pro70252-fig-0005" class="usa-link">5</a>) with Rosetta. Overall, these data show that for helical lanthipeptides, the individual lanthionine rings are rigid due to their small size and their stabilization within alpha helical structures. When predicted with Rosetta, these helical lanthionine rings generally adopt conformations that differ by 1 Å or less compared to the PDB conformation. The global flexibility in these peptides primarily comes from flexible tails or flexible regions between alpha helical segments.</p>
<figure class="fig xbox font-sm" id="pro70252-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/639d/12359204/5ca2b80fe8b6/PRO-34-e70252-g005.jpg" loading="lazy" id="jats-graphic-9" height="630" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/pro70252-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of PDB and Rosetta ensembles for four helical lanthipeptides. (a) In green is the PDB ensemble and in blue is the ensemble generated with Rosetta. (b) For all Rosetta sampled peptide conformations, the average lanthionine ring RMSD compared to the PDB conformation is plotted on the <em>x</em> axis and the Rosetta energy of the rings is plotted on the <em>y</em> axis. (c) For all Rosetta sampled peptide conformations, the backbone RMSD compared to the PDB conformation is plotted on the <em>x</em> axis and the total Rosetta energy is plotted on the <em>y</em> axis.</p></figcaption></figure><p>Helical lanthipeptides have lanthionine rings that stabilize certain helical or non‐helical segments as well as flexible regions that contribute to global flexibility. For example, in peptides 6VGT and 6VE9, several glycine residues introduce flexibility into helical structures. For 6VE9, three glycine residues in the center of the peptide introduce flexibility. In the PDB ensemble, two glycines are helical and one is at the edge of a loop region. This loop is part of a helix bend that allows PHE6 and LEU15 to interact as is seen in the NOE data. However, this data conflicts with backbone chemical shift data and TALOS‐N (Shen &amp; Bax, <a href="#pro70252-bib-0026" class="usa-link" aria-describedby="pro70252-bib-0026">2013</a>) predictions which suggest the full peptide is mostly helical (Table <a href="#pro70252-tbl-0001" class="usa-link">1</a>). In contrast to the PDB ensemble, the Rosetta ensemble has two main conformations with non‐helical regions between residues 7 and 10 or residues 10 and 13 (Figure <a href="#pro70252-fig-0006" class="usa-link">6</a>). Though, LEU10 is sometimes helical. Altogether, these data indicate that allowing diverse conformation in the ensemble for 6VE9 enables it to satisfy both the NOE data and helical nature of the peptide. For 6VGT, a segment of three adjacent glycine residues and a fourth glycine another three residues away enable flexibility between two large helical segments in both the PDB and Rosetta ensembles (Figure <a href="#pro70252-fig-0006" class="usa-link">6</a>). For PDB 8CWX, a lanthionine ring links the two helical regions and changes in the conformation of that ring, primarily mediated by multiple residues, alter the interactions between those two helical segments in the Rosetta ensemble (Figure <a href="#pro70252-fig-0006" class="usa-link">6</a>). For 2KTO, the PDB and Rosetta ensembles are quite similar, but the Rosetta ensemble is more flexible primarily due to perturbations in the linkers between adjacent lanthionine rings at peptide C terminus (Figure <a href="#pro70252-fig-0006" class="usa-link">6</a>).</p>
<section class="tw xbox font-sm" id="pro70252-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Secondary structure prediction for peptide 6VE9 with TALOS‐N.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<tbody valign="top">
<tr><td align="left" valign="top" rowspan="1" colspan="1">XBPACFBIGLGVGALFJAKFC</td></tr>
<tr><td align="left" valign="top" rowspan="1" colspan="1">‐‐‐cH‐‐‐HHHHHHH‐‐‐HHL</td></tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pro70252-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="pro70252-note-0002"><p>
<em>Note</em>: The sequence of 6VE9 is shown with the secondary structure prediction for each amino acid below. B, J, and X are used to represent dehydrobutyrine, <span class="font-variant-small-caps">d</span>‐alanine, L‐2‐aminobutanoic acid (methyllanthionine), respectively. The predictions for non‐canonical amino acids and their neighbors are hidden, as these predictions are expected to be inaccurate. “H” is for helical prediction. “L” is for unstructured positions based on chemical shift. “c” is for unstructured positions based on sequence.</p></div></div></section><figure class="fig xbox font-sm" id="pro70252-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/639d/12359204/6a4066041814/PRO-34-e70252-g006.jpg" loading="lazy" id="jats-graphic-11" height="606" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/pro70252-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Key components of helical lanthipeptides contribute to their flexibility. (a) Peptide 6VE9 has a segment composed of GLGV. The Rosetta conformations in blue show how the glycine residues in this segment could adopt different conformations than the PDB conformation in green. (b) Peptide 2KTO has two adjacent lanthionine rings at the C terminus. The Rosetta conformation in blue shows how the orientation of these rings could differ from the PDB conformation in green. (c) For peptide 8CWX, the PDB conformation in green and Rosetta conformations in purple are quite similar; however, conformational changes at between residues 14 to 18 in the Rosetta conformation enable large changes in the orientation of the N and C termini compared to one another. (d) For peptide 6VGT, both the Rosetta conformation in blue and the PDB conformation in green show how a segment of three glycine residues and their neighbors introduce flexibility into this peptide.</p></figcaption></figure></section><section id="pro70252-sec-0006"><h3 class="pmc_sec_title">2.4. Structure prediction of non‐helical lanthipeptides</h3>
<p>We used Rosetta to predict the structure of six lanthipeptides without traditional secondary structure. The prediction protocol involved randomizing the backbone of each peptide based on the Ramachandran map, iteratively closing each lanthionine ring, and relaxing the peptide structure. For these peptides, while lanthionine rings can add conformational restrictions, certain regions of the peptide, such as uncyclized termini or the regions between adjacent rings, introduce large amounts of flexibility into these peptides. For the analyzed peptides, the average sampled backbone ring RMSD deviates between 1 and 3 Å, while the PDB conformation and the sampled global backbone RMSD generally vary from 2 to 8 Å compared to the PDB conformation. In contrast, the structures within the PDB ensemble deviate by 1 to 4 Å or more within the ensemble, and relaxing these PDBs in Rosetta moves their conformation away from the PDB conformation (Figure <a href="#pro70252-fig-0004" class="usa-link">4</a> and <a href="#pro70252-supitem-0001" class="usa-link">S3</a>). These data imply that the lanthionine rings themselves have some conformational rigidity, but the orientations of the rings to one another and non‐cyclized peptide regions introduce flexibility.</p>
<p>For example, the peptides 6VHJ and 6VLJ have two adjacent rings connected by a linker of three or more amino acids. In the PDB ensembles, the orientations of these two rings are relatively consistent; however, the Rosetta ensembles indicate that because no bond holds this orientation in place, these peptides can adopt a range of alternative conformations (Figure <a href="#pro70252-fig-0007" class="usa-link">7</a>). In proteins, large structures are held together by well‐packed hydrophobic cores, but peptides do not have these hydrophobic cores. For example, 6VLJ has a sandwich‐like structure in the PDB but no strong interactions hold it in this conformation (Figure <a href="#pro70252-fig-0007" class="usa-link">7</a>). Other peptides, such as 1AJ1 and 7JU9, contain non‐conjugated lanthionine rings as well. Like other peptides, the Rosetta ensembles show the orientations of these rings to the rest of the peptide to be more flexible than the PDB ensemble suggests. In contrast, the regions of these peptides that are conformationally constrained by overlapping rings are more conformationally rigid (Figure <a href="#pro70252-fig-0007" class="usa-link">7</a>). However, conjugated rings do not automatically make lanthipeptides highly rigid. For example, 7JVF has two overlapping rings and has the greatest sampled average RMSD for its rings (Figure <a href="#pro70252-fig-0007" class="usa-link">7</a>). Finally, despite only having connected rings, three overlapping and one nested, 6PQG is more flexible in the Rosetta ensemble than the PDB ensemble (Figure <a href="#pro70252-fig-0007" class="usa-link">7</a>). This data indicates that cyclizing a peptide with lanthionine rings reduces peptide flexibility, but cyclization alone fails to rigidify the peptide's conformation.</p>
<figure class="fig xbox font-sm" id="pro70252-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/639d/12359204/fc4d0a797d9c/PRO-34-e70252-g007.jpg" loading="lazy" id="jats-graphic-13" height="985" width="709" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/pro70252-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of PDB and Rosetta ensembles for six non‐helical lanthipeptides. (a) In green is the PDB ensemble and in blue is the ensemble generated with Rosetta. (b) For all Rosetta sampled peptide conformations, the average lanthionine ring RMSD compared to the PDB conformation is plotted on the x axis and the Rosetta energy of the rings is plotted on the y axis. (c) For all Rosetta sampled peptide conformations, the backbone RMSD compared to the PDB conformation is plotted on the x axis and the total Rosetta energy is plotted on the y axis.</p></figcaption></figure><p>The amino acid composition of lanthionine rings plays a key role in determining the conformational dynamics of those rings. Work to computationally design stable macrocyclic peptides has shown that macrocyclic peptides can be stabilized by proline residues and significant internal hydrogen bonds. Additionally, the conformational preferences of specific amino acids are needed to conformationally stabilize specific positions (Hosseinzadeh et al., <a href="#pro70252-bib-0013" class="usa-link" aria-describedby="pro70252-bib-0013">2017</a>; Mulligan et al., <a href="#pro70252-bib-0019" class="usa-link" aria-describedby="pro70252-bib-0019"><sup>2021</sup></a>). The flexible lanthionine rings lack stabilizing internal hydrogen bonds and often contain glycine residues that enable the flexibility of these rings. For example, 6VHJ has a five‐membered ring example with a glycine residue. In this ring, there are no observable hydrogen bonds, and an overlay of the PDB conformation with a Rosetta conformation shows how the glycine residue enables conformational flexibility in this ring (Figure <a href="#pro70252-fig-0008" class="usa-link">8</a>). 7JVF, with the most flexible rings in this set of peptides, has no internal hydrogen bonds and multiple glycine residues in its largest ring. The lack of stabilizing elements in this large lanthionine ring enables that ring to adopt different conformations in the structures predicted by Rosetta (Figure <a href="#pro70252-fig-0008" class="usa-link">8</a>).</p>
<figure class="fig xbox font-sm" id="pro70252-fig-0008"><h4 class="obj_head">FIGURE 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/639d/12359204/1023f4d228ca/PRO-34-e70252-g001.jpg" loading="lazy" id="jats-graphic-15" height="403" width="709" alt="FIGURE 8"></p>
<div class="p text-right font-secondary"><a href="figure/pro70252-fig-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Key components of non‐helical lanthipeptides contribute to their flexibility. (a) Peptide 6VLJ has two lanthionine rings and lacks a stabilizing, well‐packed hydrophobic core. (b) One ring in peptide 6VLJ contains a glycine residue that enables conformational flexibility in structures predicted with Rosetta in blue when compared to the PDB conformation in green. This ring lacks any stabilizing internal hydrogen bonds. (c) A ring in peptide 7JVF has two glycine residues and lacks internal hydrogen bonds. A Rosetta conformation in blue shows conformational changes compared to the PDB conformation in green.</p></figcaption></figure></section><section id="pro70252-sec-0007"><h3 class="pmc_sec_title">2.5. Filtering of lanthipeptide atropisomers</h3>
<p>For some unstructured lanthipeptides with overlapping rings, multiple potential atropisomers that cannot interconvert between one another due to steric constraints were observed. These peptides are kinetically trapped similarly to natural products such as lasso peptides and tryptorubin A (Reisberg et al., <a href="#pro70252-bib-0021" class="usa-link" aria-describedby="pro70252-bib-0021">2020</a>). For the analyzed lanthipeptides, different atropisomers would sometimes appear in the final ensemble together. The atropisomers were most prevalent for 1AJ1 and 7JU9 (Figure <a href="#pro70252-fig-0009" class="usa-link">9</a>). For these peptides, the non‐contributing atropisomers were filtered from the ensemble candidate pool prior to selecting the ensemble. In both cases, the atropisomer observed in the PDB was the majority isomer of the pre‐filtered lanthipeptide structures and the non‐filtered ensembles for those peptides.</p>
<figure class="fig xbox font-sm" id="pro70252-fig-0009"><h4 class="obj_head">FIGURE 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/639d/12359204/0b121025f2a9/PRO-34-e70252-g002.jpg" loading="lazy" id="jats-graphic-17" height="350" width="709" alt="FIGURE 9"></p>
<div class="p text-right font-secondary"><a href="figure/pro70252-fig-0009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Lanthipeptides with overlapping rings may have non‐interconvertible, atropisomeric conformation. (a) Peptide 1AJ1 has an atropisomer that flips the orientation of the bottom ring. The conformation can be seen by observing how the positions of ILE16 and ALA18 flip with respect to the rest of the peptide. (b) Peptide 6PQG has an atropisomer that flips the orientations of SER15 and ALA17 with respect to the rest of the peptide.</p></figcaption></figure></section></section><section id="pro70252-sec-0008"><h2 class="pmc_sec_title">3. DISCUSSION</h2>
<p>Here, we implement parameters for modeling lanthipeptides in Rosetta and use this implementation to predict the structures of 10 lanthipeptides in the PDB with NMR‐determined structures. We find that, compared to the PDB ensembles, using ensemble‐averaged NOE distances to model lanthipeptides in Rosetta leads to ensembles that have similar agreement to the experimental data, lower Rosetta energy scores, and higher flexibility. Several peptide features contribute to their flexibility. Uncyclized regions, either at the termini or between rings, can to adopt a variety of conformations. Within lanthionine rings, the presence of glycine residues and a lack of internal hydrogen bonds enhance the flexibility of those rings. Overall, this data suggests that lanthipeptides are more flexible than PDB ensembles indicate and that peptide conformational dynamics should be considered when determining the structures of lanthipeptides.</p>
<p>Previous studies support our hypothesis that lanthipeptides should be considered in a dynamic structural context. For example, previous study found that cyclization does not automatically rigidify a peptide backbone. Rather, it needs to support the conformational preferences of the cyclized region to result in a rigid macrocycle (Wang et al., <a href="#pro70252-bib-0033" class="usa-link" aria-describedby="pro70252-bib-0033">2015</a>). Additionally, for nisin, one of many lanthipeptides that bind lipid II (Repka et al., <a href="#pro70252-bib-0022" class="usa-link" aria-describedby="pro70252-bib-0022">2017</a>), the lanthionine rings are not entirely rigid, and the regions connecting lanthionine rings are flexible. Nisin's flexibility enables it to adopt distinct conformations when binding to lipid II and forming membrane pores than it adopts in solution (Dickman et al., <a href="#pro70252-bib-0007" class="usa-link" aria-describedby="pro70252-bib-0007">2019</a>; Garcia‐Ruiz et al., <a href="#pro70252-bib-0008" class="usa-link" aria-describedby="pro70252-bib-0008"><sup>2012</sup></a>). So, in lanthipeptides, flexibility can be important for solubility in water and adopting an active conformation when encountering a target such as a lipid bilayer.</p>
<p>To further improve the modeling of lanthipeptides with experimental data, it would be beneficial to have methods that could accurately and rapidly predict carbon, nitrogen, and hydrogen chemical shifts for non‐canonical amino acids or could predict backbone dihedral angles from experimental chemical shifts. Current methods of chemical shift and backbone dihedral angle prediction such as SHIFTX2 (Han et al., <a href="#pro70252-bib-0010" class="usa-link" aria-describedby="pro70252-bib-0010">2011</a>) and TALOS‐N (Shen &amp; Bax, <a href="#pro70252-bib-0026" class="usa-link" aria-describedby="pro70252-bib-0026">2013</a>) cannot incorporate non‐canonical amino acids, limiting their utility for large segments of lanthipeptides and the types of experimental data that can be used for lanthipeptide structure determination. As backbone chemical shifts are influenced by the identity of the amino acid and its immediate neighbors (Schwarzinger et al., <a href="#pro70252-bib-0025" class="usa-link" aria-describedby="pro70252-bib-0025">2001</a>), predictions for both non‐canonical amino acids and their neighbors are unreliable. Furthermore, current lanthipeptide structure prediction pipelines require prior knowledge of the lanthipeptide ring topology and chiral center configurations.</p>
<p>Our inability to predict the ring topologies of lanthipeptides and the chirality of their lanthionine rings from sequence data alone is a major barrier to de novo lanthipeptide structure prediction. Prediction of these features is a particularly challenging task because the enzyme or the substrate sequence can determine the ring topology and chirality of lanthipeptide substrates (Garg et al., <a href="#pro70252-bib-0009" class="usa-link" aria-describedby="pro70252-bib-0009">2016</a>; Le et al., <a href="#pro70252-bib-0015" class="usa-link" aria-describedby="pro70252-bib-0015"><sup>2021</sup></a>; Sarksian et al., <a href="#pro70252-bib-0023" class="usa-link" aria-describedby="pro70252-bib-0023"><sup>2022</sup></a>; Tang et al., <a href="#pro70252-bib-0029" class="usa-link" aria-describedby="pro70252-bib-0029"><sup>2015</sup></a>). For lanthipeptides with nested rings, the structure prediction problem can be compounded by the need to filter atropisomers that do not agree with experimental data.</p>
<p>For peptides that are known to be helical, starting AlphaFold models can reduce the complexity of the structure prediction problem. However, AlphaFold is not a conclusive predictor of peptide helicity (McDonald et al., <a href="#pro70252-bib-0018" class="usa-link" aria-describedby="pro70252-bib-0018">2023</a>). For the helical lanthipeptides we examined, we saw certain patterns that supported their helicity. The helical rings in these peptides had a cysteine residue that was three or four residues after the sidechain conjugation partner. For non‐helical lanthipeptides, their ring topologies were often incompatible with helices. Prior to predicting the structure of a lanthipeptide with undetermined structures, we would recommend checking if the ring pattern could support a helical lanthipeptide before using the helical structure prediction pipeline. For small helical peptides, the non‐helical lanthipeptide structure prediction pipeline has potential to accurately predict their structure. A combination of initial random sampling with Monte Carlo‐based fine tuning has previously been used for the structure prediction of cyclic peptides up to 24 amino acids in length without secondary structure constraints or fragments (Zhu et al., <a href="#pro70252-bib-0035" class="usa-link" aria-describedby="pro70252-bib-0035">2025</a>). Despite potential for improvement, the methodology for modeling lanthipeptides presented in this study has great potential for identifying new insights into the conformations of these peptides.</p>
<p>In addition to providing a useful method for modeling the conformations of natural lanthipeptides, this study can be applied to engineering new therapeutic lanthipeptides. Current bioengineering approaches involving lanthipeptides rely on screening large libraries of lanthipeptides to find hit compounds (Urban et al., <a href="#pro70252-bib-0031" class="usa-link" aria-describedby="pro70252-bib-0031">2017</a>; Yang et al., <a href="#pro70252-bib-0034" class="usa-link" aria-describedby="pro70252-bib-0034"><sup>2018</sup></a>). Computational modeling and design of lanthipeptides with Rosetta could be used to inform the screening libraries for specific targets or for optimization of hit compounds from these screens.</p></section><section id="pro70252-sec-0009"><h2 class="pmc_sec_title">4. CONCLUSION</h2>
<p>We implemented the parameters for modeling lanthipeptides in the Rosetta macromolecular modeling software to enable modeling and design of lanthipeptides in Rosetta. We used this implementation to predict the structures of 10 lanthipeptides with NMR‐determined structures in the PDB and show that Rosetta‐generated lanthipeptide ensembles are more flexible than the lanthipeptides in the PDB. These data suggest that lanthipeptides should be considered as dynamic molecules and that lanthionine rings can increase the local rigidity of a peptide but will not necessarily make peptides globally rigid. Methods that can rapidly predict the chemical shifts of non‐canonical amino acids could improve this modeling by refining the ensemble selection process. However, lanthipeptide modeling in Rosetta still provides valuable utility for generating lanthipeptide structural hypotheses, which can deepen our understanding of the mechanism of action of natural lanthipeptides, and as a tool for computationally designing and modeling lanthipeptides with therapeutic potential. These benefits include informing experimental library screens of lanthipeptides and optimizing hit compounds to accelerate the process of lanthipeptide therapeutic development.</p></section><section id="pro70252-sec-0010"><h2 class="pmc_sec_title">5. METHODS</h2>
<section id="pro70252-sec-0011"><h3 class="pmc_sec_title">5.1. Preparation of PDB lanthipeptide ensembles</h3>
<p>Prior to energy measurement of lanthipeptide from the PDB ensemble, all structures were relaxed 20 times with backbone restraints in Rosetta. The lanthipeptide Rosetta energy scores were then determined without the backbone constraint term. For RMSD calculations, the backbone RMSD of the unrelaxed PDB structures was computed with respect to the first structure in the PDB ensemble. We used this same structure for RMSD calculations comparing Rosetta‐generated lanthipeptide models to the native structure. For computing model distances of experimentally measured NOE distances, we used the unrelaxed PDB structures.</p></section><section id="pro70252-sec-0012"><h3 class="pmc_sec_title">5.2. Helical lanthipeptide prediction pipeline</h3>
<p>To predict the structures of lanthipeptides with alpha helical structures, first the lanthipeptide sequences are canonicalized by converting D amino acids to glycine and others to alanine. Additionally, some cysteine residues were mutated to alanine to prevent the formation of disulfide bonds. Using these canonicalized sequences, AlphaFold2 with no templates is used to generate starting models for these peptides. Then, Rosetta is used to mutate the starting models into the true lanthipeptide sequence and cyclize the lanthipeptide. After relaxing the peptides with Rosetta, Rosetta is used to sample lanthipeptide conformations using Monte Carlo‐based fragment insertion, backrub moves, shear moves, and sidechain moves with the XML interface. The lanthipeptide structures are refined with Rosetta relax, and a total of 5000 models are generated for each peptide. We then perform a second round of structure refinement. For each of the 5000 models, we start 20 Monte Carlo trajectories that make small, shear, and sidechain moves. The trajectory involves an initial 1000 burn‐in steps, after which the final pose is passed to another 9000 steps, and the lowest energy structure is recovered and relaxed. The top 1000 of these structures, based on Rosetta energy scores, are used to build the final ensemble.</p></section><section id="pro70252-sec-0013"><h3 class="pmc_sec_title">5.3. Unstructured lanthipeptide prediction pipeline</h3>
<p>To predict the structures of lanthipeptides without traditional secondary structure, the backbone of these peptides is first randomized based on the Ramachandran map. Then, in the XML interface, generalized kinematic loop closure is used to sample the structure of each lanthionine ring in every possible ring permutation. The lanthipeptide structures are refined with Rosetta relax. 150,000–200,000 peptide structures are generated with Rosetta, and the top 2.5% of these structures, based on Rosetta energy scores, are passed to a second round of structure refinement. We use the same protocol for the second round of refinement as described for the helical lanthipeptide structure prediction. Briefly, this refinement involves 1000 steps of Monte Carlo burn‐in steps and another 9000 steps, after which the lowest energy structure is recovered and relaxed. The top 2.5% of lanthipeptide structures from the second round of refinement, based on Rosetta energy scores, are used to build the final ensemble.</p></section><section id="pro70252-sec-0014"><h3 class="pmc_sec_title">5.4. Non canonical amino acid parameterization</h3>
<p>Dehydroalanine and dehydrobutyrine were parameterized for Rosetta with the molfile_to_params_polymer.py script included with Rosetta (Bell et al., <a href="#pro70252-bib-0005" class="usa-link" aria-describedby="pro70252-bib-0005">2025</a>). To generate the phi and psi Ramachandran potential, a 2D scan of these torsions is performed in Gaussian with the M05‐2X/6‐311g(d,p) functional previously shown to best reproduce the backbone conformational energies for an alanine dipeptide (Tian et al., <a href="#pro70252-bib-0030" class="usa-link" aria-describedby="pro70252-bib-0030">2020</a>). The scan of the dehydrated amino acids was performed in the context of adjacent alanine residues. All three residues with n‐methyl and acetyl caps formed a pentapeptide. The lanthionine ring parameters were taken from the Amber (Tian et al., <a href="#pro70252-bib-0030" class="usa-link" aria-describedby="pro70252-bib-0030">2020</a>) force field for torsions and the Rosetta cartesian force field for bond distances and angles.</p></section><section id="pro70252-sec-0015"><h3 class="pmc_sec_title">5.5. Ensemble generation</h3>
<p>The peptide ensemble generation is based on a previously published Monte Carlo protocol for selecting an ensemble that satisfies experimental data (Vortmeier et al., <a href="#pro70252-bib-0032" class="usa-link" aria-describedby="pro70252-bib-0032">2015</a>). Briefly, a random set of 20 peptide conformations is selected. This set is perturbed by adding a member, removing a member, or swapping a member with a non‐member so that the ensemble size stays between 10 and 30 members. The sum of the square of the difference between calculated ensemble averaged NOE values and the observed NOE values is used to score each ensemble (Ashkinadze et al., <a href="#pro70252-bib-0004" class="usa-link" aria-describedby="pro70252-bib-0004">2022</a>). Moves are accepted or rejected based on a Boltzmann criterion. After 500,000 steps, the lowest scoring ensemble is taken as the final ensemble. If chemical shift data is used, the difference between the SHIFTX2 (Han et al., <a href="#pro70252-bib-0010" class="usa-link" aria-describedby="pro70252-bib-0010">2011</a>) predicted chemical shift and the observed chemical shift is added to this score. To ensure that the chemical shift predictions are reliable, chemical shifts are only included for amino acids that are canonical and have canonical neighbors. To compare peptide structures to the PDB ensemble, RMSD calculations were done to the first structure in the PDB ensemble. For the Rosetta ensembles, RMSD calculations were done to the lowest energy structure.</p></section><section id="pro70252-sec-0016"><h3 class="pmc_sec_title">5.6. Ensemble generation from models relaxed with NOE restraints</h3>
<p>To generate peptide models that are directly informed by experimental NOE measurements, we relaxed all 5000 helical peptide models and the top 2.5% of the non‐helical peptide models for the initial structure prediction round, which is approximately 5000 models, 20 times with NOE measurements as Rosetta constraints. For each of the initial models, we selected the top scoring model, including distance constraints, of the 20 derivative models for the ensemble selection process. To select a final ensemble, we first filtered to the top 200 structures based on agreement with the experimental NOE measurements. We then selected the top 20 models based on Rosetta energy scores as the final ensemble. To compare peptide structures to the PDB ensemble, RMSD calculations were done to the first structure in the PDB ensemble. For the Rosetta ensembles, RMSD calculations were done to the lowest energy structure.</p></section><section id="pro70252-sec-0017"><h3 class="pmc_sec_title">5.7. Atropisomer identification</h3>
<p>Atropisomers are differentiated by strategically selecting sets of four residue alpha carbons. Three of these carbons are on nearby residues in a loop of interest and form a plane. The fourth carbon is selected to be on a separate loop or region such that it can be determined if this fourth carbon is on the positive or negative side of the vector normal to the plane formed by the other three carbons. Enough groups of four carbons are selected to distinguish all of the potential atropisomers from the atropisomer supported by the NOE data.</p></section><section id="pro70252-sec-0018"><h3 class="pmc_sec_title">5.8. Rosetta codebase changes</h3>
<p>The changes necessary to model lanthionine rings were added to the crosslinker mover in Rosetta. Additionally, patches were added to Rosetta for the modeling of lanthionine rings and dehydrated amino acids. To enable modeling of NOE distance with non‐canonical amino acids, the AmbiguousNMRDistance constraint in Rosetta was modified to be generalizable. These changes are available in the current version of Rosetta on github at <a href="https://github.com/RosettaCommons/rosetta" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/RosettaCommons/rosetta</a>. Rosetta is free to use for academic users and requires a license for commercial users. The code and starting models for the lanthipeptide analysis will be made available at <a href="https://github.com/tydingcw/lanthi_folding_protocol_capture" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/tydingcw/lanthi_folding_protocol_capture</a>.</p></section></section><section id="pro70252-sec-0022"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Claiborne W. Tydings:</strong> Conceptualization; investigation; methodology; writing – original draft; validation; visualization; writing – review and editing; software; formal analysis; data curation. <strong>Jens Meiler:</strong> Supervision; writing – review and editing; funding acquisition; conceptualization; resources. <strong>Allison S. Walker:</strong> Supervision; writing – review and editing; funding acquisition; conceptualization; resources.</p></section><section id="sec20"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pro70252-supitem-0001"><div class="caption p">
<p>
<strong>FIGURE S1.</strong> For each lanthipeptide, the minimum Rosetta energy conformation for the PDB ensemble and the ensemble generated with Rosetta and the Monte Carlo selection protocol were chosen. Here, we calculate the difference in Rosetta energy for various Rosetta score terms. Negative energy values indicate that the Rosetta conformer had a lower energy for that score term.</p>
<p>
<strong>FIGURE S2.</strong> RMSD vs. energy plots for the helical lanthipeptides. The conformations sampled with the Monte Carlo protocol are shown in small dots. The PDB structures that were relaxed with backbone coordinate constraints are shown in black dots. The PDB structures that were relaxed without any coordinate constraints are shown in red dots.</p>
<p>
<strong>FIGURE S3.</strong> RMSD vs. energy plots for the non‐helical lanthipeptides. The conformations sampled with the Monte Carlo protocol are shown in small dots. The PDB structures that were relaxed with backbone coordinate constraints are shown in black dots. The PDB structures that were relaxed without any coordinate constraints are shown in red dots.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12359204/bin/PRO-34-e70252-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70252-s001.docx</a><sup> (2MB, docx) </sup>
</div></div></section></section><section id="pro70252-sec-0019" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>Thank you to Rocco Moretti for code review as well as Oanh Vu and Benjamin Brown for helpful discussions. Allison Walker's work on this project was supported by the National Institute of General Medical Sciences of the National Institutes of Health under grant number R35GM146987. Jens Meiler is supported by a Humboldt Professorship of the Alexander von Humboldt Foundation. Work in the Meiler laboratory is supported through NIH R01 CA227833, R01 DA046138, R01 LM013434. Claiborne Tydings was supported by “Integrated Training in Engineering and Diabetes”, grant number T32 DK101003. Figure <a href="#pro70252-fig-0001" class="usa-link">1a</a> was created with the help of <a href="http://biorender.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">biorender.com</a>.</p></section><section id="notes1"><p>

Tydings CW, Meiler J, Walker AS. Computational structure prediction of lanthipeptides with NMR data reveals underappreciated peptide flexibility. Protein Science. 2025;34(9):e70252. 10.1002/pro.70252

</p>
<section id="pro70252-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="pro70252-note-0001"><p>
<strong>Review Editor:</strong> Nir Ben‐Tal</p></div></div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Jens Meiler, Email: jens.meiler@vanderbilt.edu.</p>
<p>Allison S. Walker, Email: allison.s.walker@vanderbilt.edu.</p></section><section id="pro70252-sec-0021"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section><section id="pro70252-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="pro70252-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="pro70252-bib-0001">
<cite>
Acedo JZ, Bothwell IR, An L, Trouth A, Frazier C, van der Donk WA. O‐methyltransferase‐mediated incorporation of a β‐amino acid in lanthipeptides. J Am Chem Soc. 2019;141(42):16790–16801.
</cite> [<a href="https://doi.org/10.1021/jacs.9b07396" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6812601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31568727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=O%E2%80%90methyltransferase%E2%80%90mediated%20incorporation%20of%20a%20%CE%B2%E2%80%90amino%20acid%20in%20lanthipeptides&amp;author=JZ%20Acedo&amp;author=IR%20Bothwell&amp;author=L%20An&amp;author=A%20Trouth&amp;author=C%20Frazier&amp;volume=141&amp;issue=42&amp;publication_year=2019&amp;pages=16790-16801&amp;pmid=31568727&amp;doi=10.1021/jacs.9b07396&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0002">
<cite>
Alford RF, Leaver‐Fay A, Jeliazkov JR, O'Meara MJ, DiMaio FP, Park H, et al. The rosetta all‐atom energy function for macromolecular modeling and design. J Chem Theory Comput. 2017;13(6):3031–3048.
</cite> [<a href="https://doi.org/10.1021/acs.jctc.7b00125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5717763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28430426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Theory%20Comput&amp;title=The%20rosetta%20all%E2%80%90atom%20energy%20function%20for%20macromolecular%20modeling%20and%20design&amp;author=RF%20Alford&amp;author=A%20Leaver%E2%80%90Fay&amp;author=JR%20Jeliazkov&amp;author=MJ%20O'Meara&amp;author=FP%20DiMaio&amp;volume=13&amp;issue=6&amp;publication_year=2017&amp;pages=3031-3048&amp;pmid=28430426&amp;doi=10.1021/acs.jctc.7b00125&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0003">
<cite>
Arnison PG, Bibb MJ, Bierbaum G, Bowers AA, Bugni TS, Bulaj G, et al. Ribosomally synthesized and post‐translationally modified peptide natural products: overview and recommendations for a universal nomenclature. Nat Prod Rep. 2013;30(1):108–160.
</cite> [<a href="https://doi.org/10.1039/c2np20085f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3954855/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23165928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Prod%20Rep&amp;title=Ribosomally%20synthesized%20and%20post%E2%80%90translationally%20modified%20peptide%20natural%20products:%20overview%20and%20recommendations%20for%20a%20universal%20nomenclature&amp;author=PG%20Arnison&amp;author=MJ%20Bibb&amp;author=G%20Bierbaum&amp;author=AA%20Bowers&amp;author=TS%20Bugni&amp;volume=30&amp;issue=1&amp;publication_year=2013&amp;pages=108-160&amp;pmid=23165928&amp;doi=10.1039/c2np20085f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0004">
<cite>
Ashkinadze D, Kadavath H, Riek R, Güntert P. Optimization and validation of multi‐state nmr protein structures using structural correlations. J Biomol NMR. 2022;76(1):39–47.
</cite> [<a href="https://doi.org/10.1007/s10858-022-00392-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9018667/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35305195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biomol%20NMR&amp;title=Optimization%20and%20validation%20of%20multi%E2%80%90state%20nmr%20protein%20structures%20using%20structural%20correlations&amp;author=D%20Ashkinadze&amp;author=H%20Kadavath&amp;author=R%20Riek&amp;author=P%20G%C3%BCntert&amp;volume=76&amp;issue=1&amp;publication_year=2022&amp;pages=39-47&amp;pmid=35305195&amp;doi=10.1007/s10858-022-00392-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0005">
<cite>
Bell EW, Brown BP, Meiler J. Fakerotlib: Expedient non‐canonical amino acid parameterization in rosetta. bioRxiv. 2025;2025.02.27.640629.</cite> [<a href="https://doi.org/10.1021/acs.jcim.5c01030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40789114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Fakerotlib:%20Expedient%20non%E2%80%90canonical%20amino%20acid%20parameterization%20in%20rosetta&amp;author=EW%20Bell&amp;author=BP%20Brown&amp;author=J%20Meiler&amp;publication_year=2025&amp;pages=2025.02.27.640629&amp;pmid=40789114&amp;doi=10.1021/acs.jcim.5c01030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0006">
<cite>
Bobeica SC, Zhu LY, Acedo JZ, Tang WX, van der Donk WA. Structural determinants of macrocyclization in substrate‐controlled lanthipeptide biosynthetic pathways. Chem Sci. 2020;11(47):12854–12870.
</cite> [<a href="https://doi.org/10.1039/d0sc01651a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8163290/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34094481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20Sci&amp;title=Structural%20determinants%20of%20macrocyclization%20in%20substrate%E2%80%90controlled%20lanthipeptide%20biosynthetic%20pathways&amp;author=SC%20Bobeica&amp;author=LY%20Zhu&amp;author=JZ%20Acedo&amp;author=WX%20Tang&amp;author=WA%20van%20der%20Donk&amp;volume=11&amp;issue=47&amp;publication_year=2020&amp;pages=12854-12870&amp;pmid=34094481&amp;doi=10.1039/d0sc01651a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0007">
<cite>
Dickman R, Danelius E, Mitchell SA, Hansen DF, Erdélyi M, Tabor AB. A chemical biology approach to understanding molecular recognition of lipid II by nisin(1–12): synthesis and nmr ensemble analysis of nisin(1–12) and analogues. Chem A Eur J. 2019;25(64):14572–14582.</cite> [<a href="https://doi.org/10.1002/chem.201902814" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6899958/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31599485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20A%20Eur%20J&amp;title=A%20chemical%20biology%20approach%20to%20understanding%20molecular%20recognition%20of%20lipid%20II%20by%20nisin(1%E2%80%9312):%20synthesis%20and%20nmr%20ensemble%20analysis%20of%20nisin(1%E2%80%9312)%20and%20analogues&amp;author=R%20Dickman&amp;author=E%20Danelius&amp;author=SA%20Mitchell&amp;author=DF%20Hansen&amp;author=M%20Erd%C3%A9lyi&amp;volume=25&amp;issue=64&amp;publication_year=2019&amp;pages=14572-14582&amp;pmid=31599485&amp;doi=10.1002/chem.201902814&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0008">
<cite>
Garcia‐Ruiz E, Gonzalez‐Perez D, Ruiz‐Duenas FJ, Martinez AT, Alcalde M. Directed evolution of a temperature‐, peroxide‐ and alkaline ph‐tolerant versatile peroxidase. Biochem J. 2012;441:487–498.
</cite> [<a href="https://doi.org/10.1042/BJ20111199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21980920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20J&amp;title=Directed%20evolution%20of%20a%20temperature%E2%80%90,%20peroxide%E2%80%90%20and%20alkaline%20ph%E2%80%90tolerant%20versatile%20peroxidase&amp;author=E%20Garcia%E2%80%90Ruiz&amp;author=D%20Gonzalez%E2%80%90Perez&amp;author=FJ%20Ruiz%E2%80%90Duenas&amp;author=AT%20Martinez&amp;author=M%20Alcalde&amp;volume=441&amp;publication_year=2012&amp;pages=487-498&amp;pmid=21980920&amp;doi=10.1042/BJ20111199&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0009">
<cite>
Garg N, Goto Y, Chen T, van der Donk WA. Characterization of the stereochemical configuration of lanthionines formed by the lanthipeptide synthetase geom. Pept Sci. 2016;106(6):834–842.</cite> [<a href="https://doi.org/10.1002/bip.22876" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5108700/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27178086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pept%20Sci&amp;title=Characterization%20of%20the%20stereochemical%20configuration%20of%20lanthionines%20formed%20by%20the%20lanthipeptide%20synthetase%20geom&amp;author=N%20Garg&amp;author=Y%20Goto&amp;author=T%20Chen&amp;author=WA%20van%20der%20Donk&amp;volume=106&amp;issue=6&amp;publication_year=2016&amp;pages=834-842&amp;pmid=27178086&amp;doi=10.1002/bip.22876&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0010">
<cite>
Han B, Liu Y, Ginzinger SW, Wishart DS. Shiftx2: significantly improved protein chemical shift prediction. J Biomol NMR. 2011;50(1):43–57.
</cite> [<a href="https://doi.org/10.1007/s10858-011-9478-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3085061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21448735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biomol%20NMR&amp;title=Shiftx2:%20significantly%20improved%20protein%20chemical%20shift%20prediction&amp;author=B%20Han&amp;author=Y%20Liu&amp;author=SW%20Ginzinger&amp;author=DS%20Wishart&amp;volume=50&amp;issue=1&amp;publication_year=2011&amp;pages=43-57&amp;pmid=21448735&amp;doi=10.1007/s10858-011-9478-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0011">
<cite>
Hegemann JD, Bobeica SC, Walker MC, Bothwell IR, van der Donk WA. Assessing the flexibility of the prochlorosin 2.8 scaffold for bioengineering applications. ACS Synth Biol. 2019;8(5):1204–1214.
</cite> [<a href="https://doi.org/10.1021/acssynbio.9b00080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6525029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31042373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Synth%20Biol&amp;title=Assessing%20the%20flexibility%20of%20the%20prochlorosin%202.8%20scaffold%20for%20bioengineering%20applications&amp;author=JD%20Hegemann&amp;author=SC%20Bobeica&amp;author=MC%20Walker&amp;author=IR%20Bothwell&amp;author=WA%20van%20der%20Donk&amp;volume=8&amp;issue=5&amp;publication_year=2019&amp;pages=1204-1214&amp;pmid=31042373&amp;doi=10.1021/acssynbio.9b00080&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0012">
<cite>
Hetrick KJ, Walker MC, van der Donk WA. Development and application of yeast and phage display of diverse lanthipeptides. ACS Central Sci. 2018;4(4):458–467.</cite> [<a href="https://doi.org/10.1021/acscentsci.7b00581" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5920614/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29721528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Central%20Sci&amp;title=Development%20and%20application%20of%20yeast%20and%20phage%20display%20of%20diverse%20lanthipeptides&amp;author=KJ%20Hetrick&amp;author=MC%20Walker&amp;author=WA%20van%20der%20Donk&amp;volume=4&amp;issue=4&amp;publication_year=2018&amp;pages=458-467&amp;pmid=29721528&amp;doi=10.1021/acscentsci.7b00581&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0013">
<cite>
Hosseinzadeh P, Bhardwaj G, Mulligan VK, Shortridge MD, Craven TW, Pardo‐Avila F, et al. Comprehensive computational design of ordered peptide macrocycles. Science. 2017;358(6369):1461–1466.
</cite> [<a href="https://doi.org/10.1126/science.aap7577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5860875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Comprehensive%20computational%20design%20of%20ordered%20peptide%20macrocycles&amp;author=P%20Hosseinzadeh&amp;author=G%20Bhardwaj&amp;author=VK%20Mulligan&amp;author=MD%20Shortridge&amp;author=TW%20Craven&amp;volume=358&amp;issue=6369&amp;publication_year=2017&amp;pages=1461-1466&amp;pmid=29242347&amp;doi=10.1126/science.aap7577&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0014">
<cite>
Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, et al. Highly accurate protein structure prediction with alphafold. Nature. 2021;596(7873):583–589.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20alphafold&amp;author=J%20Jumper&amp;author=R%20Evans&amp;author=A%20Pritzel&amp;author=T%20Green&amp;author=M%20Figurnov&amp;volume=596&amp;issue=7873&amp;publication_year=2021&amp;pages=583-589&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0015">
<cite>
Le T, Fouque KJD, Santos‐Fernandez M, Navo CD, Jimenez‐Oses G, Sarksian R, et al. Substrate sequence controls regioselectivity of lanthionine formation by procm. J Am Chem Soc. 2021;143(44):18733–18743.
</cite> [<a href="https://doi.org/10.1021/jacs.1c09370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8942616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34724611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Substrate%20sequence%20controls%20regioselectivity%20of%20lanthionine%20formation%20by%20procm&amp;author=T%20Le&amp;author=KJD%20Fouque&amp;author=M%20Santos%E2%80%90Fernandez&amp;author=CD%20Navo&amp;author=G%20Jimenez%E2%80%90Oses&amp;volume=143&amp;issue=44&amp;publication_year=2021&amp;pages=18733-18743&amp;pmid=34724611&amp;doi=10.1021/jacs.1c09370&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0016">
<cite>
Leman JK, Weitzner BD, Lewis SM, Adolf‐Bryfogle J, Alam N, Alford RF, et al. Macromolecular modeling and design in rosetta: recent methods and frameworks. Nat Methods. 2020;17(7):665–680.
</cite> [<a href="https://doi.org/10.1038/s41592-020-0848-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7603796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32483333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Macromolecular%20modeling%20and%20design%20in%20rosetta:%20recent%20methods%20and%20frameworks&amp;author=JK%20Leman&amp;author=BD%20Weitzner&amp;author=SM%20Lewis&amp;author=J%20Adolf%E2%80%90Bryfogle&amp;author=N%20Alam&amp;volume=17&amp;issue=7&amp;publication_year=2020&amp;pages=665-680&amp;pmid=32483333&amp;doi=10.1038/s41592-020-0848-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0017">
<cite>
Martens E, Demain AL. The antibiotic resistance crisis, with a focus on the United States. J Antibiot. 2017;70(5):520–526.</cite> [<a href="https://doi.org/10.1038/ja.2017.30" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28246379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Antibiot&amp;title=The%20antibiotic%20resistance%20crisis,%20with%20a%20focus%20on%20the%20United%20States&amp;author=E%20Martens&amp;author=AL%20Demain&amp;volume=70&amp;issue=5&amp;publication_year=2017&amp;pages=520-526&amp;pmid=28246379&amp;doi=10.1038/ja.2017.30&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0018">
<cite>
McDonald EF, Jones T, Plate L, Meiler J, Gulsevin A. Benchmarking alphafold2 on peptide structure prediction. Structure. 2023;31(1): 111–119.e2.
</cite> [<a href="https://doi.org/10.1016/j.str.2022.11.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9883802/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36525975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Benchmarking%20alphafold2%20on%20peptide%20structure%20prediction&amp;author=EF%20McDonald&amp;author=T%20Jones&amp;author=L%20Plate&amp;author=J%20Meiler&amp;author=A%20Gulsevin&amp;volume=31&amp;issue=1&amp;publication_year=2023&amp;pages=111-119.e2&amp;pmid=36525975&amp;doi=10.1016/j.str.2022.11.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0019">
<cite>
Mulligan VK, Workman S, Sun T, Rettie S, Li X, Worrall LJ, et al. Computationally designed peptide macrocycle inhibitors of new delhi metallo‐β‐lactamase 1. Proc Natl Acad Sci. 2021;118(12):e2012800118.
</cite> [<a href="https://doi.org/10.1073/pnas.2012800118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8000195/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33723038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&amp;title=Computationally%20designed%20peptide%20macrocycle%20inhibitors%20of%20new%20delhi%20metallo%E2%80%90%CE%B2%E2%80%90lactamase%201&amp;author=VK%20Mulligan&amp;author=S%20Workman&amp;author=T%20Sun&amp;author=S%20Rettie&amp;author=X%20Li&amp;volume=118&amp;issue=12&amp;publication_year=2021&amp;pages=e2012800118&amp;pmid=33723038&amp;doi=10.1073/pnas.2012800118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0020">
<cite>
Pei Z‐F, Zhu L, Sarksian R, van der Donk WA, Nair SK. Class v lanthipeptide cyclase directs the biosynthesis of a stapled peptide natural product. J Am Chem Soc. 2022;144(38):17549–17557.
</cite> [<a href="https://doi.org/10.1021/jacs.2c06808" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9621591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36107785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Class%20v%20lanthipeptide%20cyclase%20directs%20the%20biosynthesis%20of%20a%20stapled%20peptide%20natural%20product&amp;author=Z%E2%80%90F%20Pei&amp;author=L%20Zhu&amp;author=R%20Sarksian&amp;author=WA%20van%20der%20Donk&amp;author=SK%20Nair&amp;volume=144&amp;issue=38&amp;publication_year=2022&amp;pages=17549-17557&amp;pmid=36107785&amp;doi=10.1021/jacs.2c06808&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0021">
<cite>
Reisberg SH, Gao Y, Walker AS, Helfrich EJN, Clardy J, Baran PS. Total synthesis reveals atypical atropisomerism in a small‐molecule natural product, tryptorubin a. Science. 2020;367(6476):458–463.
</cite> [<a href="https://doi.org/10.1126/science.aay9981" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6996792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31896661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Total%20synthesis%20reveals%20atypical%20atropisomerism%20in%20a%20small%E2%80%90molecule%20natural%20product,%20tryptorubin%20a&amp;author=SH%20Reisberg&amp;author=Y%20Gao&amp;author=AS%20Walker&amp;author=EJN%20Helfrich&amp;author=J%20Clardy&amp;volume=367&amp;issue=6476&amp;publication_year=2020&amp;pages=458-463&amp;pmid=31896661&amp;doi=10.1126/science.aay9981&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0022">
<cite>
Repka LM, Chekan JR, Nair SK, van der Donk WA. Mechanistic understanding of lanthipeptide biosynthetic enzymes. Chem Rev. 2017;117:5457–5520.
</cite> [<a href="https://doi.org/10.1021/acs.chemrev.6b00591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5408752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28135077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20Rev&amp;title=Mechanistic%20understanding%20of%20lanthipeptide%20biosynthetic%20enzymes&amp;author=LM%20Repka&amp;author=JR%20Chekan&amp;author=SK%20Nair&amp;author=WA%20van%20der%20Donk&amp;volume=117&amp;publication_year=2017&amp;pages=5457-5520&amp;pmid=28135077&amp;doi=10.1021/acs.chemrev.6b00591&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0023">
<cite>
Sarksian R, Hegemann JD, Simon MA, Acedo JZ, van der Donk WA. Unexpected methyllanthionine stereochemistry in the morphogenetic lanthipeptide sapt. J Am Chem Soc. 2022;144(14):6373–6382.
</cite> [<a href="https://doi.org/10.1021/jacs.2c00517" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9011353/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35352944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Unexpected%20methyllanthionine%20stereochemistry%20in%20the%20morphogenetic%20lanthipeptide%20sapt&amp;author=R%20Sarksian&amp;author=JD%20Hegemann&amp;author=MA%20Simon&amp;author=JZ%20Acedo&amp;author=WA%20van%20der%20Donk&amp;volume=144&amp;issue=14&amp;publication_year=2022&amp;pages=6373-6382&amp;pmid=35352944&amp;doi=10.1021/jacs.2c00517&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0024">
<cite>
Schmitt S, Montalbán‐López M, Peterhoff D, Deng J, Wagner R, Held M, et al. Analysis of modular bioengineered antimicrobial lanthipeptides at nanoliter scale. Nat Chem Biol. 2019;15:437–443.
</cite> [<a href="https://doi.org/10.1038/s41589-019-0250-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30936500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Chem%20Biol&amp;title=Analysis%20of%20modular%20bioengineered%20antimicrobial%20lanthipeptides%20at%20nanoliter%20scale&amp;author=S%20Schmitt&amp;author=M%20Montalb%C3%A1n%E2%80%90L%C3%B3pez&amp;author=D%20Peterhoff&amp;author=J%20Deng&amp;author=R%20Wagner&amp;volume=15&amp;publication_year=2019&amp;pages=437-443&amp;pmid=30936500&amp;doi=10.1038/s41589-019-0250-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0025">
<cite>
Schwarzinger S, Kroon GJA, Foss TR, Chung J, Wright PE, Dyson HJ. Sequence‐dependent correction of random coil nmr chemical shifts. J Am Chem Soc. 2001;123(13):2970–2978.
</cite> [<a href="https://doi.org/10.1021/ja003760i" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11457007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Sequence%E2%80%90dependent%20correction%20of%20random%20coil%20nmr%20chemical%20shifts&amp;author=S%20Schwarzinger&amp;author=GJA%20Kroon&amp;author=TR%20Foss&amp;author=J%20Chung&amp;author=PE%20Wright&amp;volume=123&amp;issue=13&amp;publication_year=2001&amp;pages=2970-2978&amp;pmid=11457007&amp;doi=10.1021/ja003760i&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0026">
<cite>
Shen Y, Bax A. Protein backbone and sidechain torsion angles predicted from nmr chemical shifts using artificial neural networks. J Biomol NMR. 2013;56(3):227–241.
</cite> [<a href="https://doi.org/10.1007/s10858-013-9741-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3701756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23728592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biomol%20NMR&amp;title=Protein%20backbone%20and%20sidechain%20torsion%20angles%20predicted%20from%20nmr%20chemical%20shifts%20using%20artificial%20neural%20networks&amp;author=Y%20Shen&amp;author=A%20Bax&amp;volume=56&amp;issue=3&amp;publication_year=2013&amp;pages=227-241&amp;pmid=23728592&amp;doi=10.1007/s10858-013-9741-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0027">
<cite>
Shenkarev ZO, Finkina EI, Nurmukhamedova EK, Balandin SV, Mineev KS, Nadezhdin KD, et al. Isolation, structure elucidation, and synergistic antibacterial activity of a novel two‐component lantibiotic lichenicidin from bacillus licheniformis vk21. Biochemistry. 2010;49(30):6462–6472.
</cite> [<a href="https://doi.org/10.1021/bi100871b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20578714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Isolation,%20structure%20elucidation,%20and%20synergistic%20antibacterial%20activity%20of%20a%20novel%20two%E2%80%90component%20lantibiotic%20lichenicidin%20from%20bacillus%20licheniformis%20vk21&amp;author=ZO%20Shenkarev&amp;author=EI%20Finkina&amp;author=EK%20Nurmukhamedova&amp;author=SV%20Balandin&amp;author=KS%20Mineev&amp;volume=49&amp;issue=30&amp;publication_year=2010&amp;pages=6462-6472&amp;pmid=20578714&amp;doi=10.1021/bi100871b&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0028">
<cite>
Shin JM, Gwak JW, Kamarajan P, Fenno JC, Rickard AH, Kapila YL. Biomedical applications of nisin. J Appl Microbiol. 2016;120(6):1449–1465.
</cite> [<a href="https://doi.org/10.1111/jam.13033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4866897/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26678028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Microbiol&amp;title=Biomedical%20applications%20of%20nisin&amp;author=JM%20Shin&amp;author=JW%20Gwak&amp;author=P%20Kamarajan&amp;author=JC%20Fenno&amp;author=AH%20Rickard&amp;volume=120&amp;issue=6&amp;publication_year=2016&amp;pages=1449-1465&amp;pmid=26678028&amp;doi=10.1111/jam.13033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0029">
<cite>
Tang W, Jiménez‐Osés G, Houk KN, van der Donk WA. Substrate control in stereoselective lanthionine biosynthesis. Nat Chem. 2015;7(1):57–64.
</cite> [<a href="https://doi.org/10.1038/nchem.2113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4270103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25515891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Chem&amp;title=Substrate%20control%20in%20stereoselective%20lanthionine%20biosynthesis&amp;author=W%20Tang&amp;author=G%20Jim%C3%A9nez%E2%80%90Os%C3%A9s&amp;author=KN%20Houk&amp;author=WA%20van%20der%20Donk&amp;volume=7&amp;issue=1&amp;publication_year=2015&amp;pages=57-64&amp;pmid=25515891&amp;doi=10.1038/nchem.2113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0030">
<cite>
Tian C, Kasavajhala K, Belfon KAA, Raguette L, Huang H, Migues AN, et al. Ff19sb: amino‐acid‐specific protein backbone parameters trained against quantum mechanics energy surfaces in solution. J Chem Theory Comput. 2020;16(1):528–552.
</cite> [<a href="https://doi.org/10.1021/acs.jctc.9b00591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31714766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Theory%20Comput&amp;title=Ff19sb:%20amino%E2%80%90acid%E2%80%90specific%20protein%20backbone%20parameters%20trained%20against%20quantum%20mechanics%20energy%20surfaces%20in%20solution&amp;author=C%20Tian&amp;author=K%20Kasavajhala&amp;author=KAA%20Belfon&amp;author=L%20Raguette&amp;author=H%20Huang&amp;volume=16&amp;issue=1&amp;publication_year=2020&amp;pages=528-552&amp;pmid=31714766&amp;doi=10.1021/acs.jctc.9b00591&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0031">
<cite>
Urban JH, Moosmeier MA, Aumüller T, Thein M, Bosma T, Rink R, et al. Phage display and selection of lanthipeptides on the carboxy‐terminus of the gene‐3 minor coat protein. Nat Commun. 2017;8(1):1500.
</cite> [<a href="https://doi.org/10.1038/s41467-017-01413-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5686179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29138389/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Phage%20display%20and%20selection%20of%20lanthipeptides%20on%20the%20carboxy%E2%80%90terminus%20of%20the%20gene%E2%80%903%20minor%20coat%20protein&amp;author=JH%20Urban&amp;author=MA%20Moosmeier&amp;author=T%20Aum%C3%BCller&amp;author=M%20Thein&amp;author=T%20Bosma&amp;volume=8&amp;issue=1&amp;publication_year=2017&amp;pages=1500&amp;pmid=29138389&amp;doi=10.1038/s41467-017-01413-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0032">
<cite>
Vortmeier G, DeLuca SH, Els‐Heindl S, Chollet C, Scheidt HA, Beck‐Sickinger AG, et al. Integrating solid‐state NMR and computational modeling to investigate the structure and dynamics of membrane‐associated ghrelin. PLoS One. 2015;10(3):e0122444.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0122444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4372444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25803439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Integrating%20solid%E2%80%90state%20NMR%20and%20computational%20modeling%20to%20investigate%20the%20structure%20and%20dynamics%20of%20membrane%E2%80%90associated%20ghrelin&amp;author=G%20Vortmeier&amp;author=SH%20DeLuca&amp;author=S%20Els%E2%80%90Heindl&amp;author=C%20Chollet&amp;author=HA%20Scheidt&amp;volume=10&amp;issue=3&amp;publication_year=2015&amp;pages=e0122444&amp;pmid=25803439&amp;doi=10.1371/journal.pone.0122444&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0033">
<cite>
Wang CK, Swedberg JE, Northfield SE, Craik DJ. Effects of cyclization on peptide backbone dynamics. J Phys Chem B. 2015;119(52):15821–15830.
</cite> [<a href="https://doi.org/10.1021/acs.jpcb.5b11085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26633076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Phys%20Chem%20B&amp;title=Effects%20of%20cyclization%20on%20peptide%20backbone%20dynamics&amp;author=CK%20Wang&amp;author=JE%20Swedberg&amp;author=SE%20Northfield&amp;author=DJ%20Craik&amp;volume=119&amp;issue=52&amp;publication_year=2015&amp;pages=15821-15830&amp;pmid=26633076&amp;doi=10.1021/acs.jpcb.5b11085&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0034">
<cite>
Yang X, Lennard KR, He C, Walker MC, Ball AT, Doigneaux C, et al. A lanthipeptide library used to identify a protein–protein interaction inhibitor. Nat Chem Biol. 2018;14(4):375–380.
</cite> [<a href="https://doi.org/10.1038/s41589-018-0008-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5866752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29507389/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Chem%20Biol&amp;title=A%20lanthipeptide%20library%20used%20to%20identify%20a%20protein%E2%80%93protein%20interaction%20inhibitor&amp;author=X%20Yang&amp;author=KR%20Lennard&amp;author=C%20He&amp;author=MC%20Walker&amp;author=AT%20Ball&amp;volume=14&amp;issue=4&amp;publication_year=2018&amp;pages=375-380&amp;pmid=29507389&amp;doi=10.1038/s41589-018-0008-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0035">
<cite>
Zhu Q, Mulligan VK, Shasha D. Heuristic energy‐based cyclic peptide design. PLoS Comput Biol. 2025;21(4):e1012290.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1012290" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12043242/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40305587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=Heuristic%20energy%E2%80%90based%20cyclic%20peptide%20design&amp;author=Q%20Zhu&amp;author=VK%20Mulligan&amp;author=D%20Shasha&amp;volume=21&amp;issue=4&amp;publication_year=2025&amp;pages=e1012290&amp;pmid=40305587&amp;doi=10.1371/journal.pcbi.1012290&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70252-bib-0036">
<cite>
Zimmermann N, Jung G. The three‐dimensional solution structure of the lantibiotic murein‐biosynthesis‐inhibitor actagardine determined by nmr. Eur J Biochem. 1997;246(3):809–819.
</cite> [<a href="https://doi.org/10.1111/j.1432-1033.1997.00809.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9219543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Biochem&amp;title=The%20three%E2%80%90dimensional%20solution%20structure%20of%20the%20lantibiotic%20murein%E2%80%90biosynthesis%E2%80%90inhibitor%20actagardine%20determined%20by%20nmr&amp;author=N%20Zimmermann&amp;author=G%20Jung&amp;volume=246&amp;issue=3&amp;publication_year=1997&amp;pages=809-819&amp;pmid=9219543&amp;doi=10.1111/j.1432-1033.1997.00809.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>
<strong>FIGURE S1.</strong> For each lanthipeptide, the minimum Rosetta energy conformation for the PDB ensemble and the ensemble generated with Rosetta and the Monte Carlo selection protocol were chosen. Here, we calculate the difference in Rosetta energy for various Rosetta score terms. Negative energy values indicate that the Rosetta conformer had a lower energy for that score term.</p>
<p>
<strong>FIGURE S2.</strong> RMSD vs. energy plots for the helical lanthipeptides. The conformations sampled with the Monte Carlo protocol are shown in small dots. The PDB structures that were relaxed with backbone coordinate constraints are shown in black dots. The PDB structures that were relaxed without any coordinate constraints are shown in red dots.</p>
<p>
<strong>FIGURE S3.</strong> RMSD vs. energy plots for the non‐helical lanthipeptides. The conformations sampled with the Monte Carlo protocol are shown in small dots. The PDB structures that were relaxed with backbone coordinate constraints are shown in black dots. The PDB structures that were relaxed without any coordinate constraints are shown in red dots.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12359204/bin/PRO-34-e70252-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70252-s001.docx</a><sup> (2MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Protein Science : A Publication of the Protein Society are provided here courtesy of <strong>The Protein Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/pro.70252"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/PRO-34-e70252.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12359204/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12359204/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359204%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359204/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12359204/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12359204/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40823923/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12359204/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40823923/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12359204/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12359204/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="DNmQIw5eHVl6kL7vSbvpFqkZJkW0BP4c5p4IDQqPAzQ64rpIY7Jgc50QyuNHFJSB">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
