
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Visualizing acyl carrier protein interactions within a crosslinked type I polyketide synthase - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53312E8AF25D6305312E0015BA2340.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370991/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Visualizing acyl carrier protein interactions within a crosslinked type I polyketide synthase">
<meta name="citation_author" content="Ziran Jiang">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA USA">
<meta name="citation_author" content="Graham W Heberlig">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA USA">
<meta name="citation_author" content="Jeffrey A Chen">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA USA">
<meta name="citation_author" content="Jennifer Huynh">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA USA">
<meta name="citation_author" content="James J La Clair">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA USA">
<meta name="citation_author" content="Michael D Burkart">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA USA">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7798">
<meta name="citation_doi" content="10.1038/s41467-025-63024-x">
<meta name="citation_pmid" content="40841798">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370991/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370991/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370991/pdf/41467_2025_Article_63024.pdf">
<meta name="description" content="Using a combination of dual covalent crosslinking and cryo-EM analyses, we elucidate the structure of mycocerosic acid synthase from Mycobacterium tuberculosis trapped in two distinct catalytic states during its iterative cycle. These structures ...">
<meta name="og:title" content="Visualizing acyl carrier protein interactions within a crosslinked type I polyketide synthase">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Using a combination of dual covalent crosslinking and cryo-EM analyses, we elucidate the structure of mycocerosic acid synthase from Mycobacterium tuberculosis trapped in two distinct catalytic states during its iterative cycle. These structures ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370991/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370991">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63024-x"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63024.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370991%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370991/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370991/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370991/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7798. doi: <a href="https://doi.org/10.1038/s41467-025-63024-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63024-x</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Visualizing acyl carrier protein interactions within a crosslinked type I polyketide synthase</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Ziran Jiang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Ziran Jiang</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Ziran Jiang</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heberlig%20GW%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Graham W Heberlig</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Graham W Heberlig</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heberlig%20GW%22%5BAuthor%5D" class="usa-link"><span class="name western">Graham W Heberlig</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20JA%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jeffrey A Chen</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jeffrey A Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20JA%22%5BAuthor%5D" class="usa-link"><span class="name western">Jeffrey A Chen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huynh%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jennifer Huynh</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jennifer Huynh</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huynh%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jennifer Huynh</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22La%20Clair%20JJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">James J La Clair</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">James J La Clair</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22La%20Clair%20JJ%22%5BAuthor%5D" class="usa-link"><span class="name western">James J La Clair</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Burkart%20MD%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Michael D Burkart</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Michael D Burkart</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Burkart%20MD%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael D Burkart</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA USA </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Nov 6; Accepted 2025 Aug 1; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370991  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841798/" class="usa-link">40841798</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Using a combination of dual covalent crosslinking and cryo-EM analyses, we elucidate the structure of mycocerosic acid synthase from <em>Mycobacterium tuberculosis</em> trapped in two distinct catalytic states during its iterative cycle. These structures reveal domain architecture of the acyl carrier protein mediating condensation and dehydration through dual site-selective crosslinking of the acyl carrier protein with the ketosynthase and dehydratase domains. Map density was sufficient to visualize full domain architecture with active site-bound probes and elucidate key interactions of four distinct crosslinked species. Here, iterative vectorial polyketide biosynthesis arises through an overall twisting and tilting architecture, enabling positioning and entry of the cognate substrate at each enzymatic domain. These structures present valuable details for future therapeutic design against mycocerosic acid biosynthesis in <em>M. tuberculosis</em>.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Cryoelectron microscopy, Biochemistry, Natural products, Biophysics, Structural biology</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Bacterial Type I polyketide synthases are responsible for producing both lifesaving medicines and virulence factors, yet their stepwise mechanism remains elusive. Here, Burkart et al. characterize acyl carrier protein bound states of mycocerosic acid synthase from <em>Mycobacterium tuberculosis</em> through crosslinking and cryo-EM.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Approximately 25% of the global population is infected by <em>Mycobacterium tuberculosis</em> (Mtb), responsible for 1.5 million annual deaths worldwide<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup> and a leading cause of child mortality<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. This has been aggravated by multidrug-resistant strains, which now infect over half a million patients, and new therapies will be crucial for future control<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. As exemplified by isoniazid, which targets the enoylreductase in mycolic acid biosynthesis, mycocerosic acid synthase (MAS) offers a potential drug target (Figs. <a href="#Fig1" class="usa-link">1</a>a and <a href="#Fig2" class="usa-link">2b</a>) by producing part of phthiocerol dimycocerosate (PDIM) (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>), a key component of the cell envelope<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. While PDIM makes up 46% of extractable Mtb lipids, it plays a major role in Mtb virulence, immunosuppression, and evasion<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. MAS, a fully reducing iterative type I polyketide synthase (PKS), is composed of six domains (Fig. <a href="#Fig2" class="usa-link">2a, b</a>) that together catalyze the elongation of <em>n</em>-C6-C20 fatty acids to prepare mycocerosic acid<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Within this 448 kDa obligate homodimer, five monofunctional enzymatic domains sequentially catalyze discrete chemical steps upon a growing metabolite that is covalently tethered to the acyl carrier protein (ACP)<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup> (Fig. <a href="#Fig1" class="usa-link">1a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. Dual site-selective crosslinking of the mycocerosic acid synthase (MAS).</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370991_41467_2025_63024_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a2fa/12370991/bc64adadb682/41467_2025_63024_Fig1_HTML.jpg" loading="lazy" id="d33e302" height="736" width="800" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Depiction of ACP processivity in MAS biosynthesis. ACP must interact with each catalytic domain, acyltransferase (AT, step 5 extension unit loading), ketosynthase (KS, step 1 condensation. Substrate loading or reloading step is not shown), ketoreductase (KR, step 2 β-ketoreduction), dehydratase (DH, step 3 dehydration), and enoylreductase (ER, step 4 α,β-enoylreduction), through discrete protein-protein and protein-substrate interactions during each catalytic cycle. MAS can be divided into two compartments, one that serves to load and elongate acyl chains (condensation compartment, steps 1 and 5); and a second that serves to modify and fully reduce the condensation product (modification compartment, steps 2–4). MAS exists as a homodimer, where protomer 1 (indicated by <sup>A</sup> after each abbreviation) is depicted with domain abbreviations, and protomer 2 (indicated by <sup>B</sup> or by prime’ after each abbreviation). <strong>b</strong> Mechanism of ACP = KS (=indicates crosslink) crosslinking reaction. <strong>c</strong> Mechanism of ACP = DH crosslinking reaction. <strong>d</strong> This process is achieved through chemoenzymatic loading of <strong>1</strong> onto the <em>apo-</em>ACP to deliver <em>crypto-</em>ACP. Once loaded, the active site Cys of KS or His of DH react with the α-Br-pantetheinamide warhead, yielding a crosslinked complex.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig2"><h3 class="obj_head">Fig. 2. Architecture of complex A displaying crosslinking of ACP = KS domains.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370991_41467_2025_63024_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a2fa/12370991/e7ad7e1252a4/41467_2025_63024_Fig2_HTML.jpg" loading="lazy" id="d33e351" height="476" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Structure of complex A at 3.9 Å with a contour level of 0.088. Distances of key residues in select domains are shown to compare the conformation of each protomer. The homodimeric architecture provides two catalytic chambers housing the catalytic process. ACPs engage their catalytic domains in chambers 1 and 2. <strong>b</strong> Linear organization of each domain in respective MAS protomers, where L1-L5 are linkers 1 to 5. Two protomers comprise the structure in (<strong>a</strong>). <strong>c</strong> Coomassie brilliant blue stained 6% SDS-PAGE gel (Supplementary Fig. <a href="#MOESM1" class="usa-link">45</a>) depicting the outcome of crosslinking with <strong>1</strong>. <strong>d</strong> Selected particle stacks from 2D classification. <strong>e</strong> FSC curve for complex A density map (Supplementary Figs. <a href="#MOESM1" class="usa-link">S8</a>–<a href="#MOESM1" class="usa-link">S9</a>). <strong>f</strong> Demonstration of ACPs crosslinked at KSs, revealing dual site-selective crosslinking with <strong>1</strong> (Fig. <a href="#Fig1" class="usa-link">1b</a>).</p></figcaption></figure><p id="Par4">Type I PKSs, including MAS, have garnered significant attention as potential therapeutic targets and for the rational design of metabolites<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. However, progress toward their understanding has been hindered by the lack of fundamental structural knowledge attributed to their inherently large size and complex structural dynamics. Early type I PKS structural studies focused on excised catalytic domains, didomains, or pruned modules<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, where the ACP is either missing or poorly resolved. As single particle cryo-EM has advanced, modular PKSs have been captured utilizing antigen-binding fragments for structural stabilization, enabling some ACP visualization in complex with select domains, but the map density of substrates and ACP structure has remained missing or fragmentary<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p>
<p id="Par5">To address these challenges, we developed dual site-selective crosslinking probes as tools to conformationally constrain PKSs in their functional states for structural analysis<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Tethering a covalent crosslinker onto the ACP can trap the interactions mediating catalysis when crosslinked with catalytic domains, offering unambiguous structural information of the protein-protein interactions (PPIs) and protein-substrate interactions<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. We aim to visualize catalytic domains in complex with ACP and chose MAS as a type I PKS model by which to develop and demonstrate these tools (Fig. <a href="#Fig1" class="usa-link">1b–d</a>). Here, we elucidate the complete structure of MAS with well-defined ACP interactions through the application of dual site-selective crosslinking (Fig. <a href="#Fig2" class="usa-link">2a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>) and single particle cryo-EM analysis, resolving key protein-protein interactions between the ACP with enzymatic domains and linker regions that to date have remained unresolved. We find that dual covalent crosslinkers combined with cryo-EM single particle analysis can offer access to multiple ACP bound states and provide a dynamic framework from which to visualize complex ACP interactions in multidomain synthases.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Domain-selective crosslinking of MAS</h3>
<p id="Par6">We have previously demonstrated that dual covalent crosslinkers (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>) efficiently trap the ACP with a variety of partner domains, including ketosynthases (KS) (Fig. <a href="#Fig1" class="usa-link">1b</a>)<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, dehydratases (DH) (Fig. <a href="#Fig1" class="usa-link">1c</a>)<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>, and other partner proteins<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> in type II fatty acid synthase (FAS) and PKS systems. Here, a panel of crosslinkers (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>) can be loaded onto an ACP through a chemoenzymatic one-pot reaction using four enzymes (CoaA, CoaD, CoaE, and Sfp) to access <em>crypto</em>-ACP (Fig. <a href="#Fig1" class="usa-link">1d</a>)<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. In type II systems the <em>crypto</em>-ACP can be generated separately, purified, and incubated in any desired molar ratio with the target partner protein in trans (intermolecularly between two proteins). However, when applying this technique to type I PKSs, where multiple domains are found within a single megasynthase protein, the free reactive crosslinker has the potential to react with an enzymatic domain, thereby inactivating it and creating shunt domain products (Supplementary Fig. <a href="#MOESM1" class="usa-link">4a</a>).</p>
<p id="Par7">To avoid such shunt products, we first attempted to crosslink (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>) the excised <em>crypto</em>-ACP domain with excised KS, excised KS-LD-AT (including linker domain LD) of MAS, or <em>E. coli</em> FabB (type II FAS KS) in trans with a panel of crosslinkers (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). This experiment was designed to also investigate the PPIs between excised MAS ACP and excised MAS KS through structural analysis. However, modest or no crosslinking yield was observed between the <em>crypto</em>-ACP and KS or KS-LD-AT (Supplementary Figs. <a href="#MOESM1" class="usa-link">5a, b</a>). This is likely due to inactivation of the KS, as it has been reported that the excised MAS-like PKS KS-LD-AT tridomain construct exists as a monomer-dimer equilibrium in solution<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. This interpretation is supported by published observation of KSs in both type I and type II systems, confirming that KS domains function as obligate homodimers. Surprisingly, slight crosslinking between FabB and excised MAS-ACP was observed, indicating that MAS-ACP is functional even though excised. The low crosslinking yield was most likely due to the difference in ACP-KS PPIs between <em>E. coli</em> type II FAS and <em>M. tuberculosis</em> type I PKS systems. We examined crosslinking between the excised MAS ACP and intact full-length MAS. With the intact MAS, <em>crypto</em>-ACPs loaded with a panel of probes crosslinked efficiently (left, Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>), confirming that the KS activity requires a stable dimerization interface provided by other domains such as DH and ER. In comparison, crosslinking between <em>E. coli</em> AcpP (right, Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>) and intact MAS showed only very limited crosslinking with the same panel, demonstrating that PPIs between ACP and KS domains are distinct in type I and type II systems. These studies of crosslinking in trans demonstrate the importance of strict PPIs and domain boundaries as exemplified in previous PKS engineering works<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Crosslinking enables cryo-EM structure elucidation of multiple complexes</h3>
<p id="Par8">Unproductive crosslinking studies with excised MAS domains in trans led us to explore the reaction <em>in cis</em> (intramolecularly within the same protein) under the hypothesis that crosslinking between the ACP and KS could enhance resolution in cryo-EM by rigidifying these dynamic multidomain proteins (Fig. <a href="#Fig1" class="usa-link">1</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). To evaluate MAS with the ACP crosslinked to KS (Fig. <a href="#Fig1" class="usa-link">1b</a>), we revisited our methodology to improve crosslinking probe loading efficiency. We first converted the probes into the primed CoA analogs, followed by addition of <em>apo</em>-MAS to maximize the yield of crosslinked MAS (Supplementary Fig. <a href="#MOESM1" class="usa-link">4b</a>). In this way, this optimized reaction sequence minimized interaction time between the free probe and the <em>apo</em>-MAS, limiting shunt product formation.</p>
<p id="Par9">We then turned our attention to selecting an optimal probe for crosslinking MAS <em>in cis</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). With in trans crosslinking in <em>E. coli</em> ACP as a model (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>), we learned that α-bromoamide probes can target both KS and DH enzymes. Here, the use of a probe that targets more than one domain could provide a useful tool to trap the dynamic multistate movement in MAS (Fig. <a href="#Fig1" class="usa-link">1a</a>). We selected C16-α-bromoamide <strong>1</strong> (Figs. <a href="#Fig1" class="usa-link">1</a>b–<a href="#Fig1" class="usa-link">d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>), where a long lipid tail mimicked the growing mycocerosate chain, and the α-bromoamide moiety offered excellent crosslinking efficiency (Fig. <a href="#Fig2" class="usa-link">2c</a>). Size exclusion chromatography afforded crosslinked MAS products at &gt;45% crosslinking yield by SDS-PAGE (right, Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). Aliquots of a 1 mg/mL sample were applied to holey carbon grids, blotted, and plunge-frozen in liquid ethane on a Vitrobot Mark IV. Micrographs were collected at 130,000× nominal magnification with a pixel size of 0.889 Å and exposure dose of 55 e<sup>-</sup>/Å<sup>2</sup> on a Titan Krios G4 electron microscope. The resulting movies were processed within Cryo-SPARC and curated to remove micrographs with poor CTF fit, large in-frame motion, and images with overly thick ice. After unsupervised particle picking and filtering (Supplementary Figs. <a href="#MOESM1" class="usa-link">8</a>–<a href="#MOESM1" class="usa-link">9</a>), an initial stack of ~1,050,000 particles was obtained and further classified, filtered, and refined to generate four reconstructed maps for complexes A-D from a single data set. In complex A (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>), all domains KS, LD, AT (acyl transferase), DH, ΨKR (nonfunctional ketoreductase as structural support), ER (enoylreductase), KR (ketoreductase), and ACP are visualized. Both ACPs crosslinked at KSs with probe <strong>1</strong>. Based on the SDS-PAGE results of crosslinking, cross-protomer crosslinking was observed, but the data does not exclude crosslinking within the same protomer (Supplementary Figs. <a href="#MOESM1" class="usa-link">7</a> and <a href="#MOESM1" class="usa-link">45</a>). Complex B (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>) revealed the homodimeric condensation compartment (KS-LD-AT) with one ACP crosslinked at KS via <strong>1</strong>, while the other KS was inhibited by <strong>1</strong> in small molecule form (without ACP). Complex C (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>) demonstrated the modifying compartment (DH-ΨKR-ER-KR), with ACP crosslinked with DH through <strong>1</strong>. Complex D (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>) revealed all domains, with both ACPs crosslinked at DH and KS, respectively.</p>
<p id="Par10">Evaluating these four crosslinked complexes offered several important structural insights. First, dual site-selective crosslinking enabled high-resolution visualization of multidomain synthases detailing MAS ACP interactions with crosslinked domains, adjacent domains, and linker peptides. Second, ACP crosslinking can rigidify MAS, providing higher local resolution at the crosslinked sites. Finally, dual site-selective crosslinking greatly enhanced the homogeneity of 4 trapped states within 6 catalytic states in each catalytic chamber.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Architecture of the ACPs = KSs dual crosslinked complex A</h3>
<p id="Par11">We obtained a density map with 3.87 Å resolution (Fig. <a href="#Fig2" class="usa-link">2</a>) from samples of the ACPs = KSs ( = indicates crosslink) dual crosslinked protein, complex A, including all domains KS, AT, DH, ΨKR, ER, KR, and ACP (Fig. <a href="#Fig2" class="usa-link">2a</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">10</a>–<a href="#MOESM1" class="usa-link">12</a>). Both ACPs are crosslinked at the homodimeric KS domains, linking Ser2060 in ACP with Cys177 in KS with the C16 acyl tail bound within the canonical KS substrate pocket (Fig. <a href="#Fig2" class="usa-link">2f</a>). Our SDS-PAGE results (Supplementary Figs. <a href="#MOESM1" class="usa-link">7</a> and <a href="#MOESM1" class="usa-link">45</a>) demonstrate cross-protomer crosslinking, but they do not exclude crosslinking within the same protomer. The full MAS structure adopts a homodimeric architecture (Fig. <a href="#Fig2" class="usa-link">2a</a>), comprising two catalytic chambers (chambers 1 and 2, Fig. <a href="#Fig2" class="usa-link">2a</a>). The dimerization interface is comprised of the KS • KS’ (2566 Å<sup>2</sup>), DH<sup>A</sup> • DH<sup>B</sup> (429 Å<sup>2</sup>), and ER<sup>A</sup> • ER<sup>B</sup> (1165 Å<sup>2</sup>) domains. The condensation compartment consists of a KS • KS’ homodimer core followed by two AT domains on opposite sides spaced out by LD domains (Supplementary Fig. <a href="#MOESM1" class="usa-link">13a</a>). While the LD domain acts as a spacer between the KS and AT domains, it has limited contact with the KS. Intriguingly, the 62 residue post-AT linker (L1) brings the KS and AT into proximity via a polar interface (Supplementary Fig. <a href="#MOESM1" class="usa-link">13a</a>). L1 wraps around the LD, forming an LD/L1 interface (1627 Å<sup>2</sup>), which offers extensive L1/KS (970 Å<sup>2</sup>) and L1/AT (320 Å<sup>2</sup>) interactions. The LD has been previously reported to be involved in a turnstile mechanism, where in a closed conformation it forms extensive interactions with the AT to block access to the KS<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. While such a turnstile-closed state ensures vectorial biosynthesis in the DEBS type I modular PKS system<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, it is not observed in complex A, which is an iterative type I PKS (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>).</p>
<p id="Par12">From the AT, L1 leads to the DH<sup>A</sup> • DH<sup>B</sup> of the modifying compartment which resides just above the KS • KS’ core (Supplementary Figs. <a href="#MOESM1" class="usa-link">13a</a>, <a href="#MOESM1" class="usa-link">52a</a>, and 52b). Residues 882-893 residing in C-terminal of linker 1 were not modeled in due to weak density. Due to this, the connection between the condensation and modifying compartments cannot be defined within the same protomer. As a result, two separate labeling systems of KS-LD-AT/KS’-LD’-AT’ for the condensation compartment and DH<sup>A</sup>-ΨKR<sup>A</sup>-ER<sup>A</sup>-KR<sup>A</sup>-ACP<sup>A</sup>/DH<sup>B</sup>-ΨKR<sup>B</sup>-ER<sup>B</sup>-KR<sup>B</sup>-ACP<sup>B</sup> for the modifying compartment are used. DH<sup>A</sup> • DH<sup>B</sup> is a dimer of two double-hotdog folds, forming a “handshake fold” and creating a 429 Å<sup>2</sup> interface<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. The catalytically inactive pseudo-KR domain (ΨKR) is linked to the DH through a 31 residue DH-ΨKR linker (L2) (Supplementary Figs. <a href="#MOESM1" class="usa-link">13b</a> and <a href="#MOESM1" class="usa-link">52a</a>, b). Next, a 27 residue linker 3 (L3) connects the ΨKR to the ER, which is a homodimer that forms a 1165 Å<sup>2</sup> interface, followed by a 21 residue linker 4 (L4) connecting ER to KR. Altogether, L2, L3, and L4 form two sets of antiparallel β-strands, providing extensive interfaces with the KR (2365 Å<sup>2</sup>) and ER (686 Å<sup>2</sup>) domains. Supported by partial density, a 14 residue flexible KR-ACP linker (L5) allows the ACP to travel between the domains of each of the catalytic chambers of MAS (Supplementary Figs. <a href="#MOESM1" class="usa-link">13b</a> and <a href="#MOESM1" class="usa-link">52a</a>, b). The linker-based network creates two functional catalytic chambers and ensures flexibility for the ACP to reach all domains akin to porcine FAS (Supplementary Figs. <a href="#MOESM1" class="usa-link">48</a> and <a href="#MOESM1" class="usa-link">49</a>). Due to the flexibility offered by the linker-based architecture, complex A is not perfectly symmetrical. Distances between the active site residues Tyr1912 (KR) and Ser623 (AT), as well as Tyr1912 (KR) and Ser2060 (ACP), are 78.1 Å and 71.9 Å, respectively, in protomer 1, compared with 72.4 Å and 69.6 Å inter-residue distances in protomer 2, revealing a significant in-plane tilting movement of the modifying compartment that pivots at the KS • KS’ core (Fig. <a href="#Fig2" class="usa-link">2a</a>). The double crosslinking between ACPs and KSs was supported by unambiguous map density. The crosslinkers tethered on the ACP<sup>A</sup> and ACP<sup>B</sup> through a 4’-phosphopantetheine (PPant) arm are well resolved within the KS • KS’ binding tunnels, each making a covalent linkage at the active site Cys residues (Cys177 in KS and KS’) (Fig. <a href="#Fig2" class="usa-link">2f</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Intramolecular Crosslinking Between ACP and KS in Complex B</h3>
<p id="Par13">We also obtained a density map for complex B (Fig. <a href="#Fig3" class="usa-link">3</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>) from the same dataset with a higher overall resolution (3.22 Å) that further elucidates details of the condensation compartment (KS-LD-AT), with only one ACP crosslinked at KS (Fig. <a href="#Fig3" class="usa-link">3a</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">15</a>–<a href="#MOESM1" class="usa-link">17</a>). Due to the structural rigidification of covalent ACP crosslinking, the local resolution around the ACP crosslinked region is higher than the non-crosslinked region (Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>). This effect is particularly pronounced in AT’ in complex B. Such a rigidification effect through ACP crosslinking binding was also observed in the recent DEBS structures<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Along with crosslinker 1, ACP was unambiguously resolved. Covalent bond S-C between the KS active site Cys177 and the reactive α-position of the crosslinker (Fig. <a href="#Fig3" class="usa-link">3e</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>), as well as the O-P covalent bond between the ACP active residue Ser2060 and the PPant arm (Fig. <a href="#Fig3" class="usa-link">3e</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>), were determined by continuous map density. In the other protomer, continuous density was observed only for crosslinker <strong>1</strong> reacted with Cys177 of KS but not ACP due to the shunt reaction of <strong>1</strong> (Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Structure of complex B with a single ACP = KS crosslink.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370991_41467_2025_63024_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a2fa/12370991/4473790bd830/41467_2025_63024_Fig3_HTML.jpg" loading="lazy" id="d33e868" height="396" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Map of one ACP crosslinked at KS’ within the condensation compartment at 3.2 Å, with a contour level of 0.12. Colored circles represent regions of expansion in (<strong>b</strong>–<strong>e</strong>). Panel (<strong>f</strong>) further distinguishes the KS and KS’. <strong>b</strong> Crosslinking of ACP and KS traps PPIs between ACP and KS. A single hydrogen bond between Glu79 and Arg2045 mediates the ACP interaction with KS. <strong>c</strong> More extensive residue interactions are seen between ACP and AT. <strong>d</strong> The KS gating network (gating loops 1 and 2) residing at the entrance of the KS substrate binding tunnel demonstrates the inter-loop interactions (Asn272, Asp274, and Asn421) and its interactions with the PPant arm of <strong>1</strong>. This gate is visualized in both KSs in complex A. <strong>e</strong> Crosslinker <strong>1</strong> links ACP to KS by trapping Cys177 (Fig. <a href="#Fig1" class="usa-link">1b</a>), highlighting active site residues in the substrate loading and reloading states (Fig. <a href="#Fig1" class="usa-link">1a</a>). <strong>f</strong> Demonstration of the deep KS binding tunnel formed by the KS • KS’ dimerization interface. In complex B, one ACP is crosslinked at KS, while KS is shunted by <strong>1</strong> without ACP.</p></figcaption></figure><p id="Par14">In MAS, the ACP approaches the KS domain with a head-on orientation (Fig. <a href="#Fig3" class="usa-link">3a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">20A</a>). This contrasts with type II KS-ACP interactions, typified by AcpP = FabB (PDB ID: 7L4L), where the AcpP α-helical bundle is tilted to allow the KS to make more contact with helices 1, 2, and 3 (Supplementary Fig. <a href="#MOESM1" class="usa-link">20d</a>) and loop 1 of ACP. Type II ACPs are also typically negatively charged and utilize extensive electrostatic interactions for KS recognition. In contrast, the coulombic potentials are evenly distributed across the MAS ACP (Supplementary Fig. <a href="#MOESM1" class="usa-link">17</a>). This lack of electrostatic interactions is compensated by the multidomain architecture of MAS, where the ACP is located on the same polypeptide as other domains<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Interactions between ACP and KS are limited to a single hydrogen bond between Arg2045 on loop 1 of the ACP and Glu79 of KS (Fig. <a href="#Fig3" class="usa-link">3b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">21a</a>) along with weak hydrophobic and electrostatic interactions (Supplementary Fig. <a href="#MOESM1" class="usa-link">50</a>). Complex B reveals that the docking of ACP on KS is mainly facilitated by interactions with the neighboring AT domain via hydrogen bonding between Arg2032 and Arg2033 on helix 1 and Arg2090 on helix 4 with Asp819 on the AT domain (Fig. <a href="#Fig3" class="usa-link">3C</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">21b</a>).</p>
<p id="Par15">We adopted a quantitative crosslinking assay to evaluate the MAS mutants in this study, as the correlation between crosslinking yield and enzymatic activity has been validated<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. ACP-AT interactions were verified to be critical for the ACP catalytic efficiency with KS by evaluating comparative crosslinking of point mutants Glu79Lys, Asp819Ala, Asp819Arg, Arg2032Asp, and Arg2090Asp (Supplementary Figs. <a href="#MOESM1" class="usa-link">22</a>–<a href="#MOESM1" class="usa-link">27</a>). In comparison, ACPs in modular type I PKSs DEBS-M1 (PDB ID: 7M7F, Supplementary Fig. <a href="#MOESM1" class="usa-link">20b</a>) and Lsd14 (PDB ID: 7S6C, Supplementary Fig. <a href="#MOESM1" class="usa-link">20c</a>) adopt slightly different binding modes at the condensation state, with more interactions between helix 1 and 2 of ACP and KS, but no interactions with the AT (Supplementary Fig. <a href="#MOESM1" class="usa-link">20a</a>)<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. However, the observed ACP-AT interactions are in agreement with the recent complex between DEBS-M3 and ACP2 structure at the transacylation state<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. It is worth noting that a six residue helix of KS near the ACP docking interface mediates interactions with ACP in Lsd14 (Pro118-Ser123, PDB ID: 7S6C, Supplementary Fig. <a href="#MOESM1" class="usa-link">20c</a>), DEBS-M1 (Pro113-Ala118, PDB ID: 7M7F, Supplementary Fig. <a href="#MOESM1" class="usa-link">20b</a>), as well as type II AcpP=KS (Ser63-Arg68, PDB ID:7L4L, Supplementary Fig. <a href="#MOESM1" class="usa-link">20d</a>); but this structural feature is not seen in MAS complex B (Supplementary Figs. <a href="#MOESM1" class="usa-link">20a</a>, <a href="#MOESM1" class="usa-link">e</a>).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Non-turnstile ketosynthase substrate regulation</h3>
<p id="Par16">The KS from MAS shares similarities in KS substrate binding with type I and II PKSs by using the same canonical KS binding tunnel as demonstrated in <em>E. coli</em> FabF (PDB ID: 7L4L), Lsd14 (PDB ID: 7S6C), and DEBS-M1 (PDB ID: 7M7F) (Supplementary Fig. <a href="#MOESM1" class="usa-link">20e</a>). The AcpP = FabF structure revealed an unprecedented gating motif for KSs that assists in the substrate binding sequence of the ping-pong mechanism<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Here, Phe400 of gating loop 1 moves 7 Å upon the interaction of gating loop 2 with <em>acyl</em>-AcpP in preparation for the initial transacylation step; this is followed by Phe400 repositioning near the active site for malonyl-AcpP to undergo decarboxylative Claisen condensation (Supplementary Figs. <a href="#MOESM1" class="usa-link">28b</a>, <a href="#MOESM1" class="usa-link">c</a>). In MAS complex B (Supplementary Fig. <a href="#MOESM1" class="usa-link">28a</a>), the crosslinker, tethered through a phosphodiester linkage to Ser2060 of ACP, has the PPant arm forming hydrogen bonds with residues Thr279 and Thr281 in KS (Fig. <a href="#Fig3" class="usa-link">3d, e</a>). These reside on a loop analogous to gating loop 2 of FabF, which orchestrates with gating loop 1 to govern the recognition and processing of ACP substrates (Supplementary Figs. <a href="#MOESM1" class="usa-link">28b, c</a>)<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. In <em>E. coli</em> FabF, PPant interactive residues His268 and Thr270 in gating loop 2 are responsible for governing the gate-closed and gate-open conformation by interacting with Asp35 of AcpP. In KS, Thr279 and Thr281, respectively, reflect similar positions (Supplementary Fig. <a href="#MOESM1" class="usa-link">28</a>).</p>
<p id="Par17">Through the crosslinking assay (Supplementary Fig. <a href="#MOESM1" class="usa-link">22</a>), mutant Thr279Ala (Supplementary Figs. <a href="#MOESM1" class="usa-link">22</a>–<a href="#MOESM1" class="usa-link">23</a>) revealed a 2-fold decrease, and Thr281Ala (Supplementary Figs. <a href="#MOESM1" class="usa-link">22</a>–<a href="#MOESM1" class="usa-link">23</a>) showed complete abolishment of crosslinking efficiency. Based on this result, Thr279 and Thr281 in gating loop 2 are strong candidates for regulating the gating mechanism of KS. Consistent with FabF, the coordination of gating loops 1 and 2 is directed by the highly conserved residues Asp274 and Asn421 across type I and type II PKS and FAS KSs (also conserved in DEBS-M1 and LSD14)<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Surprisingly, crosslinking was completely eliminated in MAS mutant Asn421Ala, suggesting its potential role in orchestrating the KS gating system (Supplementary Fig. <a href="#MOESM1" class="usa-link">22</a>). Although the gating loops are in the closed conformation upon crosslinking in both MAS complexes A and B, Asp2059 of ACP appears to be a candidate residue to trigger opening of the gating network by interaction with loop 1 of KS (Supplementary Fig. <a href="#MOESM1" class="usa-link">28a</a>). Interestingly, the main KS gating residue (gating loop 1) is Met417 instead of Phe, as found in type II PKS and FAS systems<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. When mutated to Phe (Met417Phe), a 35% decrease in crosslinking efficiency was observed. This may be due to the use of methylmalonyl-CoA as the extension unit in MAS instead of malonyl-CoA, as Met is more flexible and less bulky. Although more detailed investigation is necessary, key residues Thr279, Thr281, Met417, and Asn421 are most probably involved in regulating the gate-open and gate-closed conformations in KS as a part of the substrate regulation.</p>
<p id="Par18">Within the KS active site, the covalent bond between the α carbon of <strong>1</strong> and Cys177 of KS is well visualized (Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>). The warhead carbonyl of crosslinker <strong>1</strong> coordinates with two highly conserved residues His312 and His348 (Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>), trapping the transacylation (substrate loading or reloading) state of KS in its gate-closed form (Supplementary Fig. <a href="#MOESM1" class="usa-link">28a</a>)<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Supported by continuous map density, the C16 acyl tail of <strong>1</strong> is buried deep within the hydrophobic tunnel formed by the KS • KS’ dimeric interface (Supplementary Figs. <a href="#MOESM1" class="usa-link">18</a> and <a href="#MOESM1" class="usa-link">29</a>). The end of the binding tunnel lacks a clear boundary with the binding tunnel of the other KS monomer, providing space for the final long, branched mycocerosic acid product (Supplementary Fig. <a href="#MOESM1" class="usa-link">29</a>). In comparison to KS from DEBS-M1 (Supplementary Fig. <a href="#MOESM1" class="usa-link">29c</a>) and Lsd14 (Supplementary Fig. <a href="#MOESM1" class="usa-link">29d</a>), the binding tunnels of MAS and FabB KSs (Supplementary Fig. <a href="#MOESM1" class="usa-link">29b</a>) are narrower. A published MAS-like PKS KS-LD-AT tridomain structure (PDB ID: 5BP1) was crystalized in the monomeric form, and the KS domain was reported to be catalytically inactive<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Compared with the intact MAS complex B, active site Cys178 is shifted by 10.2 Å. The dimerization interface in complex B (Supplementary Fig. <a href="#MOESM1" class="usa-link">30</a>) plays a pivotal role in maintaining the stability and positioning of the active site. In the other protomer’s KS domain, <strong>1</strong> can be found alone (a shunt product without ACP) with continuous density. We presume that the highly potent warhead reacted with the KS active site as a small molecule in the absence of ACP<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>.</p>
<p id="Par19">The AT domain in complex A is in the <em>apo</em> form (without substrate in the active site), consisting of an α/β-hydrolase and ferredoxin subdomains, sharing a similar fold with other type I and type II systems. The “turnstile closed” conformation shown in the reported DEBS-M1 is not observed in either complex A or B, leaving the KS domain accessible to ACPs<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. The turnstile model in type I modular PKS proposed by Khosla et al. emphasizes the reloading of extended or processed substrate is prohibited by the “turnstile closed” conformation (AT blocking ACP access to the KS) to ensure the vectorial synthesis of modular polyketides<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. The absence of observed “turnstile closed” conformations in MAS complexes A and B implies that iterative type I PKSs utilize a different mechanism to prevent premature reloading of the KS domain.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Intramolecular crosslinking between DH and ACP in Complex C</h3>
<p id="Par20">We also obtained a map of the modifying compartment DH-ΨKR-ER-KR-ACP (3.74 Å) from the same dataset. Here, one ACP is covalently crosslinked to the DH domain (Fig. <a href="#Fig4" class="usa-link">4a</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">31</a>–<a href="#MOESM1" class="usa-link">33</a>), which consists of a double-hotdog fold in type I PKSs. In this complex, <strong>1</strong> covalently crosslinks the ACP to the active site His929 of the DH (Fig. <a href="#Fig4" class="usa-link">4e</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">36</a>). Like the observations found with complexes A (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>) and B (Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>), local resolution on DH<sup>A</sup> is higher compared to that of the uncrosslinked DH<sup>B</sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">31</a>). Even though DH is composed of a double-hotdog fold, it only contains one set of reactive residues His797 and Asp1095 (Supplementary Fig. <a href="#MOESM1" class="usa-link">34a</a>, <a href="#MOESM1" class="usa-link">c</a>). The other hotdog fold is responsible for forming the DH<sup>A</sup> • DH<sup>B</sup> dimerization interface through a “handshake fold”<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. The two hotdog folds are connected through a 20-residue loop (Supplementary Fig. <a href="#MOESM1" class="usa-link">34b</a>). The ACP = DH crosslinking observed in complex C is bound in an opposite orientation when compared to that for the <em>E. coli</em> AcpP=FabA (PDB ID: 4KEH) (Supplementary Fig. <a href="#MOESM1" class="usa-link">34a</a>)<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. In contrast to the extensive interactions between the highly negatively charged AcpP (helices 2 and 3) and a positively charged patch of FabA, the ACP in MAS shows only limited interactions with the DH domain between Thr2072 of helix 2 of ACP and Asn933 of DH (Fig. <a href="#Fig4" class="usa-link">4b</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">33</a> and <a href="#MOESM1" class="usa-link">35a</a>). Akin to the binding between ACP and KS in complexes A and B, the binding between ACP and DH is mainly facilitated by surrounding structural elements. Arg2045, Thr2046, and Tyr2056 (ACP loop 1) interact with Ser1187 and Glu2000 of inter-domain linkers (L2 and L5) (Fig. <a href="#Fig4" class="usa-link">4c</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">35b</a>). Mutation of Ser1187Ala and Thr2046Val validated these interactions as important for activity, resulting in a loss of 22% and 20% crosslinking, respectively. These findings emphasize the outsized role that inter-domain linkers play in PKS catalysis.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Structure of single ACP = DH crosslinked complex C.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370991_41467_2025_63024_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a2fa/12370991/3c0cb9142799/41467_2025_63024_Fig4_HTML.jpg" loading="lazy" id="d33e1243" height="389" width="800" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Cryo-EM map at 3.7 Å and a contour level of 0.19, of ACP crosslinked to DH of the modifying compartment. Colored circles represent regions of expansion in (<strong>b</strong>–<strong>e</strong>). <strong>b)</strong> Upon ACP = DH crosslinking, interactions are observed between ACP with DH. A single hydrogen bond is observed between DH Asn933 and ACP Thr2072. <strong>c</strong> In addition, more extensive interactions are seen between ACP and L2 as well as L5. <strong>d</strong> Comparison amongst <em>apo</em>-DH of MAS-like PKS (purple) (PDB ID: 5BP4), <em>apo</em>-DH of MAS complex A (orange) (PDB ID: 9D2Y), and crosslinked ACP = DH of complex C (PDB ID: 9D30) demonstrating DH reactive residues His and Asp as well as the significant domain translation and rotation (Supplementary Fig. <a href="#MOESM1" class="usa-link">39</a>). DH domains of MAS-like PKS and Complex A were aligned to the crosslinked DH of complex C (RMSD = 1.266 Å). Residues shown are the reactive His and Asp residing at the active site of the DH domain. Lower left cartoon demonstrates the crosslinked DH substrate binding tunnel entrance (arrow) translates and rotates away from KR to be accessible for MAS ACP crosslinking in complex C. <strong>e</strong> ACP = DH crosslinking forged a covalent bond between active site His929 and <strong>1</strong>. The continuous map density for the crosslinking at His929 and Ser2060 can be found in Supplementary Fig. <a href="#MOESM1" class="usa-link">36</a>. <strong>f</strong> Depiction of the hydrophobic tail of <strong>1</strong> within the binding pocket of DH (Supplementary Fig. <a href="#MOESM1" class="usa-link">37</a>).</p></figcaption></figure><p id="Par21">Although crosslinker <strong>1</strong> has historically been used to target KS, complex C only contains crosslinking between ACP and DH. Unlike the shunt product in complex B (Fig. <a href="#Fig3" class="usa-link">3f</a>), there is no probe observed in the uncrosslinked DH of the opposite protomer, indicating that the reactivity between crosslinker <strong>1</strong> and DH is facilitated by ACP binding. To validate the ability of α-bromoamide warheads to crosslink domains other than KS, we performed crosslinking experiments of <em>E. coli crypto</em>-AcpP loaded with <strong>2,</strong>
<strong>3,</strong>
<strong>5</strong>, or <strong>6</strong> with a panel of partner proteins including FabF (KS), Cys163Ala FabF (KS*), FabG (KR), FabA (DH), FabI (ER), and FabD (AT) from <em>E. coli</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>). Similar to our observation in MAS complex C, where ACP is crosslinked with DH, <em>crypto</em>-AcpPs loaded with <strong>2</strong> and <strong>3</strong> were able to crosslink both FabA and FabF, with no crosslinking seen with the other enzymes.</p>
<p id="Par22">Crosslinker <strong>1</strong> enters the canonical DH substrate binding tunnel and is crosslinked to His929 (Fig. <a href="#Fig4" class="usa-link">4e</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">37A</a>), similar to the type II <em>E. coli</em> AcpP=FabA (PDB ID: 4KEH, Supplementary Fig. <a href="#MOESM1" class="usa-link">37b</a>) and AcpP=FabZ (PDB ID: 6N3P) structures<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Complete map density is observed from Ser2060 to C4 of the acyl tail of <strong>1</strong>, although no other significant substrate-protein polar interactions are observed (Supplementary Fig. <a href="#MOESM1" class="usa-link">36</a>). Unlike the narrow and well-defined FabA active site, MAS-DH features a deep binding tunnel to accommodate the long, branched-chain acyl intermediates (Fig. <a href="#Fig4" class="usa-link">4f</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">37</a>). The analogous <em>E. coli</em> crosslinked structure (AcpP = FabA) revealed a DH gating mechanism utilizing residues Phe165 and Phe171 to enforce substrate specificity (Supplementary Fig. <a href="#MOESM1" class="usa-link">38b</a>). In MAS-DH, these gating residues are replaced by the DH-ΨKR interdomain linker L2, leaving the DH entrance exposed (Supplementary Fig. <a href="#MOESM1" class="usa-link">38a, c</a>). Surprisingly, while the <em>E. coli</em> type II DH regulates the substrate specificity via gating residues, we observe a major domain rotation of the crosslinked DH in comparison to the <em>apo</em>-DHs in MAS complex A and MAS-like PKS modifying compartment (Fig. <a href="#Fig4" class="usa-link">4d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">39</a>). In the DH <em>apo</em> form the active site entrance resides underneath the ΨKR • KR domains, providing no access for ACP binding. In complex C, the DH translates and rotates away from the ΨKR • KR to enable access to the ACP (Supplementary Fig. <a href="#MOESM1" class="usa-link">39</a>). The limited space between <em>apo</em>-DH and ΨKR • KR hinders ACP docking and likely plays a role in shielding the active site of the DH in certain catalytic states.</p></section><section id="Sec9"><h3 class="pmc_sec_title">ACP = DH and ACP = KS Dual Crosslinked Complex D</h3>
<p id="Par23">Within the same data set, a 6.18 Å density map was obtained visualizing both ACPs trapped at DH and KS, respectively, yielding complex D (Fig. <a href="#Fig5" class="usa-link">5</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">40a</a>). While this reconstruction was not detailed enough to allow atomic modeling, it reveals the remarkable flexibility of MAS processivity. This map shows an asymmetric pose of MAS, and all domains are visualized, including both ACPs. Anchoring at the KS • KS’ core, the modifying compartment DH<sup>B</sup>-ΨKR<sup>B</sup>-ER<sup>B</sup>-KR<sup>B</sup> demonstrates an astonishing higher-order dynamic (tilted and twisted) pose directed toward KS • KS’, where ACP<sup>B</sup> is trapped at KS (Supplementary Fig. <a href="#MOESM1" class="usa-link">40b</a>). On the other side, DH<sup>A</sup>-ΨKR<sup>A</sup>-ER<sup>A</sup>-KR<sup>A</sup> is tilted and twisted away from the KS • KS’ core, with ACP<sup>A</sup> trapped at DH. In contrast to complex A, where both ACPs are crosslinked at KSs, showing only a slight tilt, the conformational state of complex D clearly revealed the capability of catalytic chambers 1 and 2 (Fig. <a href="#Fig2" class="usa-link">2a</a>) to process substrates asymmetrically. The fact that both ACPs are trapped at different domains in two different catalytic chambers in complex D, together with complex A, in which both ACPs are crosslinked at KSs, reveals that both chambers can process substrates synchronously, in contrast to DEBS-M1 and Lsd14 type I modular systems<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. At maximum extension, the length between active site residues Ser2060 (ACP<sup>A</sup>) and Ser1996 (L5) is 84 Å (Supplementary Fig. <a href="#MOESM1" class="usa-link">41</a>). The distance between Ser1995 (KR<sup>A</sup>) to the phosphodiester motif of the PPant in KS (chamber 1, Fig. <a href="#Fig2" class="usa-link">2a</a>) is approximately 66 Å. When DH-ΨKR-ER-KR is tilted and twisted away from the KS • KS’ core, ACP<sup>A</sup> active residue Ser2060 can still reach the KS • KS’ core through the length of L5; however, the modifying domains KR<sup>A</sup>, DH<sup>A</sup>, and ER<sup>A</sup> are preferred. In the comparison amongst the linker conformations of complexes A, B, and C, the tilting and twisting movements stem from linker 1 (unmodeled residues 882-893 at the C-terminal). Linkers 2, 3, 4, and most of linker 1 adopt similar conformations amongst complexes A, B, and C. Linker 5, which directly governs the flexibility of ACP, on the other hand, points in different directions between complexes A and C. (Supplementary Fig. <a href="#MOESM1" class="usa-link">52</a>) The tilting and twisting conformation demonstrates that type I PKSs can make use of conformational flexibility to localize reactivity between the distinct elongation and modification compartments, much like what has been indicated with type I FAS<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Structural insight into MAS conformationally-guided processivity.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370991_41467_2025_63024_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a2fa/12370991/d5a78c8455d7/41467_2025_63024_Fig5_HTML.jpg" loading="lazy" id="d33e1435" height="514" width="800" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Following KS (at KS and KS’), ACP•KS substrate reloading occurs through transacylation from ACP. Next, ACP accepts a methylmalonate monomer from AT. ACP shuttles the extension unit to its partner KS to induce condensation. Complex B informs this step. MAS flexibility enables in plane tilting and out-of-plane twisting. During this process, ACP is distal to the condensation compartment, therefore, interactions with the modifying compartment are favored, and the ACP binds sequentially to KR, DH, and ER. Dehydration processing is exemplified in complexes C and D, where complex D is trapped through crosslinking of ACPs with DH (chamber 1) and KS (chamber 2), respectively, showing this conformational movement. A full iteration is completed through substrate reloading. After 4 iterations, PapA5 transfers the mature mycocerosates to phthiocerol, forming phthiocerol dimycocerosate (PDIM).</p></figcaption></figure></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par24">The combination of developed dual covalent crosslinkers and the ability of cryo-EM to capture multiple protein complexes within a single dataset has enabled the visualization of multiple trapped catalytic states in the type I iterative PKS MAS from <em>M. tuberculosis</em>. Crosslinking between ACP and catalytic domains led to an overall rigidified protein architecture, provided resolution improvement, and yielded four uniquely trapped states, complexes A–D. Upon crosslinking, the ACP = KS and ACP = DH binding interfaces were revealed, highlighting interactions between these domains as well as with neighboring structural elements (Supplementary Fig. <a href="#MOESM1" class="usa-link">42</a>). In comparison to type II systems, the ACP in MAS adopted unique docking orientations at both KS and DH. Importantly, the expected PPIs between ACP and each of these domains were altered and largely replaced by PPIs with adjacent domains and linker peptides. In addition, crosslinker <strong>1</strong> interrogated the substrate-protein interactions in both the KS and DH domain active sites. The inability to crosslink truncated KS or KS-AT with ACP emphasizes the critical role of domain architecture and linker regions in stabilizing reactive interactions. Importantly, complexes A and B are the validation of cross-protomer processivity in type I PKSs based on the SDS-PAGE results (Supplementary Figs. <a href="#MOESM1" class="usa-link">7</a> and <a href="#MOESM1" class="usa-link">45</a>), while crosslinking within the same protomer is likely due to high out-of-plane twisting. Complex D shows that while ACP interacts with DH in chamber 1, the other ACP interacts with the KS in chamber 2 (Fig. <a href="#Fig5" class="usa-link">5</a>), corroborating the obligate dimerization of MAS. The cross-protomer processivity plays an important role in guiding synthetic efficiency in type I PKSs, and likely FASs by allowing the ACP<sup>A</sup> to interact with KS while ACP<sup>B</sup> interacts with KS’ or vice versa.</p>
<p id="Par25">In complex B, MAS potentially adopts KS gating loops similar to <em>E. coli</em> type II KSs, and the presence of key gating residues is essential for activity. In complex C, MAS-DH does not demonstrate gating control like that found in <em>E. coli</em> FabA and <em>H. pylori</em> FabZ<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>–<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Instead, DH rotation upon ACP binding presents a strong candidate for DH gating regulation. Based on the structural features of MAS observed in all complexes, an iterative assembly line mechanism can be suggested (Fig. <a href="#Fig5" class="usa-link">5</a>). Starting with the substrate loading step, <em>acyl-</em>ACP<sup>A</sup> interacts with KS in catalytic chamber 1 to deliver a long fatty acyl thioester to Cys177. Based on the higher-order dynamics in this step<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, if in-plane tilt and out-of-plane twist are moderate, ACP<sup>B</sup> can interact with the KS in the other chamber, exemplified by complex A. If the modifying compartment (DH<sup>B</sup>-ΨKR<sup>B</sup>-ER<sup>B</sup>-KR<sup>B</sup>) in chamber 2 tilts and twists away from the KS domain, ACP<sup>B</sup> kinetically prefers to interact with the modifying compartment, supported by complexes B and D. This high degree of twist and tilt may be the reason that the KS domain is inhibited by shunt product in complex B.</p>
<p id="Par26">Subsequently, <em>holo-</em>ACP visits the AT to load a methylmalonyl extender unit, followed by a return to the KS for chain elongation in chamber 1 (Fig. <a href="#Fig5" class="usa-link">5</a>). These three steps, substrate loading, transacylation, and condensation, require ACP to shuttle within the condensation compartment in chamber 1. As the modifying compartment tilts and twists away from the condensation compartment in chamber 1, the interactions of ACP<sup>A</sup> with modifying domains in chamber 1 are more favorable. In the reduction steps, the elongated substrate on ACP is fully reduced in the modifying compartment in chamber 1 (Fig. <a href="#Fig5" class="usa-link">5</a>). The dehydration step is structurally represented by complexes C and D. Complex D demonstrates ACP trapped at KS in one chamber, while ACP is trapped at DH in the other. After the substrate on ACP is fully reduced, it is reloaded onto KS for another iteration. Such a higher-order dynamics-driven vectorial processivity agrees with the “see-saw” model suggested by EcPKS1 and EcPKS2 from <em>Elysia chlorotica</em> by the Hill and Schmidt groups<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. In the EcPKS2 structure, the linker between KR and ACP alternates between disordered and ordered conformation (interacting with nonfunctional methyltransferase) to control the flexibility of ACP and, consequently, change the domain sampling profiles. Although the alternation between ordered and disordered conformations of linker 5 was not found in the MAS complexes, our model does not exclude this function.</p>
<p id="Par27">In modular PKS Lsd14, the post-ACP dimerization element or pre-KR dimerization element keeps both ACPs in proximity<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Due to this, the catalytic chambers function asynchronously in their proposed pendulum model. The absence of post-ACP dimerization element or pre-KR dimerization element allows MAS to function synchronously and asymmetrically (Supplementary Fig. <a href="#MOESM1" class="usa-link">51</a>). In DEBS-M1, to strictly enforce the vectorial biosynthesis, the protein adopts a turnstile-closed conformation after chain elongation, where the AT interacts with the LD to sterically occlude the ACP from binding KS. These structural features in modular PKS systems prevent substrate reloading onto KS to avoid unwanted products. However, in iterative type I PKS systems such as MAS, substrate reloading is essential for processivity. The higher-order dynamics can guide the ACP to interact preferentially with the condensation or modifying compartments, but it does not completely prohibit ACP from revisiting domains. Therefore, a substrate regulation mechanism is needed in iterative type I PKS system. In the MAS KS, while gating loops 1 and 2 may play a role in substrate specificity, recent studies have shown the substrate gatekeeping capability of KS in type I system<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Mutagenesis and analysis depicted that KS dimer interface loops containing conserved TNGQ or VMYH motifs are commonly found in KS families that preferentially accept D-β-hydroxyacyl or α/β-enoylacyl intermediates, respectively. Such a dimer interface loop is not conserved in KSs that accept β-methylene intermediates. MAS KS, which does not contain the TNGQ or VMYH motifs, instead possesses LTHE (residues 127-130 in MAS complex B), hinting that D-β-hydroxyacyl or α/β-enoylacyl intermediates are not the preferred substrates. KS substrate specificity in a type I system was also observed in our recent development of cerulenin crosslinker <strong>8</strong> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>)<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Due to the oxidized <em>α</em>, <em>β</em>, and <em>γ</em> positions in the cerulenin crosslinker, it had minimal crosslinking between MAS ACP and MAS KS. The fact that crosslinking between MAS ACP and MAS KS can be achieved by fully reduced crosslinker <strong>1</strong> also indicates the fully reduced substrate preference of MAS-KS. In terms of substrate chain length, it is noteworthy that the two substrate tunnels formed by the KS • KS’ dimer in complex B are connected (Fig. <a href="#Fig2" class="usa-link">2f</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">29a</a>), contributing to a potential substrate tunnel sharing to accommodate C22-C28 mycocerosic acids products. In contrast, FabB does not have shared tunnels. Its substrate chain length is controlled by a “back gate” consisting of two sets of Glu200 and Gln 113. The coordination of the “back gate” controls the size of the tunnels asymmetrically<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. This negative cooperativity contributes to the acceptance of long substrates in one tunnel, while the other tunnel is limited to accept short substrates (Supplementary Fig. <a href="#MOESM1" class="usa-link">43</a>). After the condensation step at the KS domain, the elongated substrate does not visit the AT domain, as the AT, although proximal, strictly accepts methylmalonyl-CoA<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. This substrate-based control is different from type II ACPs, which sequester acyl substrates, thereby altering the ACP binding interface and achieving preferential binding with specific partner proteins through allostery<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Since type I ACPs do not sequester their acyl substrates, the interactions with different domains are more likely guided by the loaded acyl substrates, as proposed for the pikromycin PKS<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>.</p>
<p id="Par28">Here we show how a dual covalent crosslinker in concert with cryo-EM can reveal previously inaccessible structural features of type I PKSs. The crosslinked species can be reliably resolved at high resolution due to constraints of dynamic movement in crosslinked megasynthase compartments. The ability for a single crosslinker to interact with more than one domain allows the collection of multiple complexes within a single data set that can be differentiated with particle sorting and reconstruction. The versatility of this method parallels prior findings in type II synthases solved by X-ray crystallography, hereby revealing the elegance of the type I PKS organization and control. Compared with non-specific crosslinkers, dual covalent crosslinkers offer not only the visualization of PPIs, but also substrate-protein interactions. Most importantly, covalent crosslinkers facilitate structural rigidification, which gives rise to higher resolution density maps. Most importantly, the dual site-selective crosslinking approach used herein reveals structures of states actively trapped during their enzymatic process. Recent studies, including those on human FAS<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> show that one can apply cryo-EM with high data content to refine structures resting at specific positions within the catalytic process, including reduction at the ER, dehydration at the DH, condensation at the KS, and acyl transfer at the AT. The fact that these states are resting means that they do not necessarily provide accurate representations of substrate undergoing enzymatic processing, but dual site-selective state trapping, as well as other approaches to enzymatically lock states during their reactions do.</p>
<p id="Par29">Using this technology, we reveal four informative MAS complexes from a single data set, visualizing both ACPs from the MAS homodimer in action. We show the ACP binding interfaces with KS and DH, interactions with adjacent domains and linkers, and substrate-protein interactions. While the KS of MAS potentially adopts a gating mechanism similar to type II KSs, DH rotation upon ACP crosslinking was identified as a potential DH gating mechanism. Complexes A and D suggest that both catalytic chambers can not only be functional synchronously but also asymmetrically, constructing a model that allows the ACP trajectory to follow the sequential catalytic loop (transacylation, condensation, reduction, and substrate reloading) controlled by substrate selectivity. The cross-protomer activity of ACP-KS interactions provides a model for multidomain catalysis. Altogether, this study provides fundamental concepts and structural details of a mycocerosic acid-producing machinery that offers potential as a therapeutic target.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">General experimental methods</h3>
<p id="Par30">For protein expression, LB media (RPI Research Products), agar (RPI Research Products), IPTG (Bio Pioneer), ampicillin (Fisher Scientific), and kanamycin (Apex BioResearch Products) were used. For buffer preparation, Tris base (Fisher Scientific), HCl (Fisher Scientific), NaCl (RPI Research Products), glycerol (Honeywell Research Chemicals), imidazole (MP Biomedicals), maltose (Sigma Aldrich), MgCl<sub>2</sub> (BDH Chemicals), Na<sub>2</sub>HPO<sub>4</sub> (Fisher Scientific), KH<sub>2</sub>PO<sub>4</sub> (Fisher Scientific), KCl (Fisher Scientific), ATP (Thermos Scientific), DTT (Fisher Scientific), and DMSO (Fisher Scientific) were used. For protein purification, we used a sonicator (Ultrasonic Processor FS-600N), Ni-NTA columns (Thermo Scientific), amylose columns (New England BioLabs), 10 K, 30 K, 50 K, and 100 K MWCO spin concentrators (Amicon Millipore), 3 K and 10 K MWCO Snake Skin dialysis tubing (Thermo Fisher), Hiload Superdex 200 pg (16 × 600) (GE Biosciences), HiLoad Superdex 75 pg (16 × 600) (GE Biosciences) size exclusion columns, and an FPLC (ÄKTA Pure). In reactions that contain crosslinkers 1–12 and fluorescent probe <strong>13</strong> (dissolved in DMSO), the DMSO concentration was kept below 1% in the final reaction. The preparation of <strong>1</strong>-<strong>13</strong> was prepared using previously published protocols<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Materials were used with a purity &gt;98% by NMR. Typically, stock solutions in DMSO were prepared fresh and used as is. For gel preparation or evaluation, SDS (Sigma Aldrich), 40% acrylamide/bis-Acrylamide 29:1 (Fisher Scientific), TEMED (Sigma Aldrich), APS (Fisher Scientific), Wide Range Gel Preparation Buffer (4×) (Nacalai Tesque), Mini-Protean TGX gel (4–15%) (Bio-Rad), and 4–12% bis-Tris NuPage (Invitrogen) were used. For gel imaging, UV transilluminator at 365 nm was used prior to Coomassie Brilliant Blue staining. For gel-based crosslinking yield quantification, Perfection V19 scanner (Epson) and ImageJ<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> were used. When possible raw images were used to make the figures. Unedited gel images were used or provided in Supplementary Fig. <a href="#MOESM1" class="usa-link">46</a>.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Cloning of M. tuberculosis MAS and variants</h3>
<p id="Par31">Primers for cloning were ordered from Integrated DNA Technologies. Restriction enzymes, Gibson master mix, and T4 ligase were ordered from New England Biolabs. Sanger sequencing was performed by Azenta Life Sciences. Wild-type MAS from <em>M. tuberculosis</em> H37Rv (UniProt ID: I6Y231) in a pET21 vector (pET23) was provided by H. Gramajo<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Excised KS from <em>M. tuberculosis</em> MAS was synthesized in pET28 as a codon-optimized construct by Twist Bioscience. The boundaries of <em>M. tuberculosis</em> KS-LD-AT were determined through sequence alignment with MAS-like type I PKS from <em>M. smegmatis</em> (UniProt ID: <a href="https://www.ncbi.nlm.nih.gov/protein/A0R1E8" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">A0R1E8</a>). Excised KS-LD-AT tridomain was subcloned from pET23 into pET21 using Gibson cloning (NEB HiFi assembly) and directly transformed into DH5α cells. Resulting colonies were grown overnight in LB broth (5 mL) with ampicillin, miniprepped, and sent for Sanger sequencing to confirm proper insertion. The ACP of <em>M. tuberculosis</em> MAS was synthesized by Twist Bioscience in a pET21 expression vector.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Expression and purification of M. tuberculosis MAS</h3>
<p id="Par32">C-terminal His<sub>6</sub>-tagged recombinant MAS was expressed in <em>E. coli</em> BL21(DE3) similar to that previously described in refs. <sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. A single colony with pET23 was cultured in LB broth (5 mL) with 100 mg/L of ampicillin for 12 h at 37 °C. This culture was added into LB broth (1 L) with 100 mg/L of ampicillin and incubated with shaking at 37 °C until the OD<sub>600</sub> reached 0.6–0.8. The culture was then cooled on ice followed by induction with 0.5 mM IPTG (addition of 1 mL of 500 mM IPTG) at 37 °C and left shaking for 18 h. Cell pellets were obtained through centrifugation (4816 × <em>g</em> for 30 min). The cell pellets were resuspended in 25 mL cold lysis buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 10% glycerol), submerged in ice, and lysed by sonication (1 s on and 3 s off cycle for 9 min). The lysate was then centrifuged at 4816 × <em>g</em> for 1 h. The resulting supernatant was loaded onto Ni-NTA column (Thermo Fisher Scientific) and washed with 10 CV (column volume) of the lysis buffer followed by lysis buffer containing 10 mM imidazole (2 × 10 mL). Lysis buffer containing 250 mM imidazole was used to elute the protein. Dialysis with 10 K MWCO SnakeSkin tubing (Thermo Fisher Scientific) was performed with lysis buffer (1 L) for 18 h at 4 °C. The dialyzed protein was then subjected to HiLoad Superdex 200 pg (16 × 600) (GE Biosciences) size exclusion column purified with lysis buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 10% glycerol). Fractions containing pure MAS were collected and concentrated to 32 mg/mL using a 100k Amicon Ultra spin concentrator (Millipore) and flash frozen in aliquots for storage at −80 °C.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Expression and purification of excised KS and excised KS-LD-AT</h3>
<p id="Par33">The expression and purification of excised KS and excised KS-LD-AT follow a common protocol. C-terminal His<sub>6</sub>-tagged recombinant excised KS and excised KS-LD-AT<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> were expressed in BL21(DE3) and cultured in LB broth (5 mL) with 100 mg/L of ampicillin for 12 h at 37 °C. The culture was added into LB broth (1 L) with 100 mg/L of ampicillin and incubated at 37 °C until the OD<sub>600</sub> reached 0.6–0.8. The culture was then cooled on ice, followed by induction with 0.5 mM IPTG (addition of 1 mL of 500 mM IPTG) at 37 °C and left shaking for 18 h. Cell pellets were obtained through centrifugation (4816 × <em>g</em> for 30 min). The cell pellets were resuspended in 25 mL of cold lysis buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 10% glycerol), submerged in ice, and lysed by sonication (1 s on and 3 s off cycle for 9 min). The lysate was then centrifuged at 4816 × <em>g</em> for 1 h. The resulting supernatant was loaded onto a Ni-NTA column (0.5 mL/L of culture) (Thermo Fisher Scientific) and washed with 10 CV of the lysis buffer, followed by lysis buffer containing 10 mM imidazole (2 × 10 mL). Lysis buffer containing 250 mM imidazole was used to elute the protein. Dialysis with 10 K MWCO SnakeSkin tubing (Thermo Fisher Scientific) was performed with lysis buffer (1 L) for 18 h at 4 °C. The dialyzed protein was then subjected to HiLoad Superdex 200 pg (16 × 600) (GE Biosciences) size exclusion column and purified with lysis buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 10% glycerol). Fractions containing excised KS or excised KS-LD-AT were collected and concentrated to 32 mg/mL using a 10k Amicon Ultra spin concentrator (Millipore) and flash frozen in aliquots for storage at −80 °C.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Expression and purification of excised ACP from MAS</h3>
<p id="Par34">The expression and purification of excised ACP adopted a reported protocol<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. C-terminal His<sub>6</sub>-tagged recombinant excised ACP was expressed in BL21(DE3) and cultured in LB broth (5 mL) with 100 mg/L of ampicillin for 12 h at 37 °C. The culture was added into LB broth (1 L) with 100 mg/L of ampicillin and incubated with shaking at 37 °C until the OD<sub>600</sub> reached 0.6–0.8. The culture was then cooled on ice followed by induction with 0.5 mM IPTG (addition of 1 mL of 500 mM IPTG) at 37 °C and left shaking for 18 h. Cell pellets were obtained through centrifugation (4816 × <em>g</em> for 30 min). The cell pellets were resuspended in 25 mL of cool lysis buffer (8 mM Na<sub>2</sub>HPO<sub>4</sub>, 286 mM NaCl, 1.4 mM KH<sub>2</sub>PO<sub>4</sub>, 2.6 mM KCl, and 1% SDS (w/v) pH 7.4), submerged in ice, and lysed by sonication until the lysate turns clear (1 s on and 3 s off sonication cycle). The sonicated lysate was then chilled on ice for 1 h to cause SDS precipitation. To remove the SDS precipitates, the cold lysate was centrifuged at 4816 × <em>g</em> for 30 min at 4 °C. The resulting supernatant was loaded onto a Ni-NTA-column (0.5 mL/L of culture) (Thermo Fisher Scientific) and washed with 10 CV of the washing buffer (8 mM Na<sub>2</sub>HPO<sub>4</sub>, 286 mM NaCl, 1.4 mM KH<sub>2</sub>PO<sub>4</sub>, 2.6 mM KCl, 0.15% tween 20 (w/v), and 15 mM imidazole, pH 7.4) followed by elution buffer (1 mL × 7) (8 mM Na<sub>2</sub>HPO<sub>4</sub>, 286 mM NaCl, 1.4 mM KH<sub>2</sub>PO<sub>4</sub>, 2.6 mM KCl, 0.15% tween 20 (w/v), and 500 mM imidazole, pH 7.4). Dialysis with 3 K MWCO SnakeSkin tubing (Thermo Fisher Scientific) was performed with dialysis buffer (1 L) (8 mM Na<sub>2</sub>HPO<sub>4</sub>, 286 mM NaCl, 1.4 mM KH<sub>2</sub>PO<sub>4</sub>, 2.6 mM KCl, and 10% glycerol) for 18 h at 4 °C. The dialyzed protein was collected and concentrated to 3 mg/mL using a 3k Amicon Ultra spin concentrator (Millipore) and flash frozen in aliquots for storage at −80 °C.</p></section><section id="Sec17"><h3 class="pmc_sec_title">MAS mutant site-directed mutagenesis</h3>
<p id="Par35">MAS mutants were made using Quikchange Mutagenesis. Primers were ordered from Integrated DNA Technologies. Q5 Hot Start High Fidelity polymerase and restriction enzyme DpnI were ordered from New England Biolabs. Whole-plasmid sequencing was performed by Primordium Labs. Briefly, PCR was performed for 35 cycles, followed by template digestion with DpnI at 37 °C for 3 h and inactivation at 80 °C for 20 min. Digested samples were directly transformed into 10-beta chemically competent cells and plated at 37 °C overnight with the corresponding selection antibiotics. Colonies were screened the following day by whole-plasmid sequencing to validate the mutation.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Expression and purification of MAS mutants</h3>
<p id="Par36">MAS mutants Asp819Arg, Asp2050Arg, Arg2090Asp, Glu79Lys, Asp819Ala, Arg2032Asp, Thr281Ala, Thr279Ala, Met417Phe, Asn421Ala, Ser1187Aal, Thr2046Val, Cys177Ala, and His929Ala were prepared by a common protocol. C-terminal His<sub>6</sub>-tagged recombinant MAS mutants<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> were expressed in BL21(DE3) cells and cultured in LB broth (5 mL) with 100 mg/L of ampicillin for 12 h at 37 °C. The culture was added into LB broth (1 L) with 100 mg/L of ampicillin and incubated with shaking at 37 °C until the OD<sub>600</sub> reached 0.6–0.8. The culture was then cooled on ice, followed by induction with 0.5 mM IPTG (addition of 1 mL of 500 mM IPTG) at 18 °C and left shaking for 18 h. Cell pellets were obtained through centrifugation (4816 × <em>g</em> for 30 min). The cell pellets were resuspended in 25 mL of cold lysis buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 10% glycerol), submerged in ice, and lysed by sonication (1 s on and 3 s off cycle for 9 min). The lysate was then centrifuged at 4816 × <em>g</em> for 1 h. The resulting supernatant was loaded onto a Ni-NTA column (0.5 mL/L of culture) (Thermo Fisher Scientific) and washed with 10 CV of the lysis buffer, followed by lysis buffer containing 10 mM imidazole (2 × 10 mL). Lysis buffer containing 250 mM imidazole was used to elute the protein. Dialysis with 10 K MWCO SnakeSkin tubing (Thermo Fisher Scientific) was performed with lysis buffer (1 L) for 18 h at 4 °C. The dialyzed protein was then subjected to HiLoad Superdex 200 pg (16 × 600) (GE Biosciences) size exclusion column and purified with lysis buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 5% glycerol). Fractions containing pure MAS were collected and concentrated to 3 mg/mL using a 50k Amicon Ultra spin concentrator (Millipore) and flash frozen in aliquots for storage at −80 °C.</p></section><section id="Sec19"><h3 class="pmc_sec_title">MAS crosslinker loading and crosslinking</h3>
<p id="Par37">Crosslinkers <strong>1,</strong>
<strong>2</strong>, or <strong>3</strong> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>) were loaded onto the ACP of <em>apo</em>-MAS by a one-pot chemoenzymatic method<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. The optimized reaction included 94.7 μL buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 10% glycerol) 5 μL CoaA (1 mg/mL), 5 μL CoaD (1 mg/mL), 5 μL CoaE (1 mg/mL), 2.9 μL Sfp (6.9 mg/mL), 25 μL MgCl<sub>2</sub> (250 mM), 2.5 μL DTT (100 mM),53.3 μL ATP (75 mM), and 0.88 μL <strong>1,</strong>
<strong>2</strong>, or <strong>3</strong> (from a 10 mM DMSO stock). In this step, crosslinkers are converted into CoA analogs of <strong>1,</strong>
<strong>2</strong>, or <strong>3</strong> (primed crosslinkers) by CoaA, CoaD, and CoaE as described in Supplementary Fig. <a href="#MOESM1" class="usa-link">4B</a>. The reaction was incubated at 23 °C for 2 h. Sequentially, 55.7 μL <em>apo</em>-MAS (31 μM) was added into the reaction mixture to generate a solution with 0.02 mg/mL CoaA, 0.02 mg/mL CoaD, 0.02 mg/mL CoaE, 0.08 mg/mL Sfp, 25 mM MgCl<sub>2</sub>, 1 mM DTT, 16 mM ATP, and 35 μM <strong>1,</strong>
<strong>2</strong>, or <strong>3</strong>, 7 μM MAS, and buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 10% glycerol). The reaction was then incubated at 37 °C for 1.5 h, followed by incubation at 23 °C for 18 h. The reaction mixture was then subjected to purification by HiLoad Superdex 200 pg (16 × 600) (GE Biosciences) size exclusion column with buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl). Fractions containing the crosslinked complexes were collected, diluted to 4.4 μM, and used immediately for cryo-EM sample preparation.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Grid preparation for cryo-EM data collection</h3>
<p id="Par38">Holey carbon grids (Quantifoil, R1.2/1.3, Cu, 300 mesh) were glow discharged, and 3.2 µL of 1 mg/mL crosslinked MAS sample was applied to the grid in a Vitrobot Mk IV with the chamber at 4 °C and 95% humidity. The grids were blotted, plunge frozen in liquid ethane, and stored in liquid nitrogen until needed.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Single particle cryo-EM data collection</h3>
<p id="Par39">Cryo-EM data were collected using a Titan Krios G4 operating at 200 kV and equipped with a Selectris X energy filter and a Falcon 4 direct electron detector. A total of 3135 movies were collected at a nominal magnification of 130,000×, nominal pixel size of 0.889 Å/px, and dose of 55 e<sup>-</sup>/Å<sup>2</sup> using the EPU software on a set of manually selected grid squares.</p></section><section id="Sec22"><h3 class="pmc_sec_title">General cryo-EM data processing</h3>
<p id="Par40">Movie data was imported into cryo-SPARC, and movie frame alignment was performed using patch motion correction, and patch CTF was used for CTF estimation. Exposures were curated to remove images with poor CTF fits, large in-frame motion, or extremely thick ice. This set of micrographs was used for all four maps reported in this study. All particle processing was carried out in cryo-SPARC, and final maps were sharpened in Phenix using <em>phenix.local_aniso_sharpen</em>. The initial stack of particles was picked using reference-free “blob” picking and subsequent 2D-classification to remove junk particles generated the initial stack of particles used in all subsequent operations. The resolution of all maps was estimated based on the gold standard Fourier shell correlation (FSC) using a cutoff of 0.143<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>,<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Complex A cryo-EM data processing</h3>
<p id="Par41">Initial classes representing side views where the full MAS particle was visible were selected and used as templates to select a new stack of particles. These were subjected to 2D classification, where junk particles were removed. These particles were used to ab initio reconstruct 4 classes, and these classes were subsequently heterogeneously refined (~113k particles). The particles from the 3D class representing the full MAS dimer were 3D classified twice, first to remove remaining junk particles and second to select particles belonging to the doubly ACPs = KSs population (35,170 particles). This was non-uniformly refined, enforcing C2 symmetry with pose marginalization to generate a 3.87 Å reconstructed map for complex A<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Complex B cryo-EM data processing</h3>
<p id="Par42">The full initial stack of cleaned particles (~184k particles) Fourier cropped to 1.78 Å/px were used in an ab initio 3D-reconstruction job with 3 classes, 1 class contained particles corresponding to the KS-LD-AT domains (complex B), 1 class contained particles corresponding to the DH-ΨKR-ER-KR domains (complex C), and 1 class captured additional junk particles. The class corresponding to the KS-LD-AT tridomain, when refined, showed strong map density for the first crosslink and some weaker density for a second crosslink on the other protomer. The particles were then extracted at full resolution and passed through 3D classification, and those corresponding to the double crosslink and those without any crosslinking were removed. The remaining particles (34,615) were non-uniformly refined in C1 symmetry, which generated a map for complex B at 3.22 Å.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Complex C cryo-EM data processing</h3>
<p id="Par43">The ab initio class containing particles corresponding to the DH-ΨKR-ER-KR domains was reextracted at full resolution and passed through 3D classification to remove particles that did not have any crosslinking. The remaining particles (35,969) were non-uniformly refined in C1 symmetry, which generated a map for complex C at 3.74 Å. Density map demonstrated in all Complex C figures was generated by DeepEMhancer<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup> with training models included in the package and default tight masking through the half maps was generated from refinement as the input volumes.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Complex D cryo-EM data processing</h3>
<p id="Par44">While processing the data for complex A, we observed a 3D class that corresponded to a single ACP = KS crosslink with a single ACP = DH crosslink. This map was used to generate templates, and particles were selected and extracted. The particles were passed through 2D classification to remove junk particles. We reasoned that the particle stacks for complexes B and C likely also contained some particles that corresponded to this crosslinking state, so those particle stacks were combined as well, duplicates were removed, and the full stack was used with the initial model from 3D classification in a homogeneous refinement job. Subsequent rounds of 3D classification were used to remove particles not corresponding to the ACP = KS/ACP = DH state. The final stack of particles (19,622) was non-uniformly refined in C1 symmetry to generate a 6.15 Å map for complex D.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Structure modeling</h3>
<p id="Par45">The AlphaFold predicted structure of MAS from <em>M. bovis</em> (AF-<a href="https://www.ncbi.nlm.nih.gov/protein/Q02251" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q02251</a>) was used as the starting model for complexes A–C, the condensation and modifying compartments (except the ACPs) were fit individually into the maps using UCSF ChimeraX. The ACP domain model, also from AF-<a href="https://www.ncbi.nlm.nih.gov/protein/Q02251" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q02251</a>, was fit into the maps separately. The models were rejoined and manually remodeled in Coot. Density was observed for crosslinker <strong>1</strong> in all KS domains, including the shunt KS in complex B. Ligand density within the DH domain of complex C was limited. After modeling of <strong>1</strong>, the structures were refined using real-space refinement in Phenix. All map-to-model validation was also carried out in Phenix. AlphaFold structure of MAS from <em>M. bovis</em> (UniProt ID: <a href="https://www.ncbi.nlm.nih.gov/protein/Q02251" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q02251</a>) contains one amino acid difference compared with our biological sample, MAS from <em>M. tuberculosis</em> H37Rv (UniProt ID: I6Y231). Therefore, Cys213 from the AlphaFold model (UniProt ID: <a href="https://www.ncbi.nlm.nih.gov/protein/Q02251" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q02251</a>) was changed into Ser213 (UniProt ID: I6Y231) in the final model.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Evaluating MAS mutation through screening crosslinking efficiency</h3>
<p id="Par46">We began by preparing a master mix of the coenzyme A analogue of <strong>1</strong> (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). This was conducted using 9.7 μL buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 10% glycerol), 0.5 μL CoaA (1 mg/mL), 0.5 μL CoaD (1 mg/mL), 0.5 μL CoaE (1 mg/mL), 0.07 μL Sfp (6.9 mg/mL), 2.5 μL MgCl<sub>2</sub> (250 mM), 0.5 μL DTT (100 mM), 2.7 μL ATP (75 mM), and 0.5 μL <strong>1</strong> (from a 1 mM DMSO stock). The reaction was incubated at 37 °C for 2 h. We then used this master mix for each reaction. Reactions were conducted by addition of 7.5 μL <em>apo</em>-MAS (13 µM) or 7.5 μL <em>apo</em>-MAS mutant (13 μM), stored at 23 °C for 2 h. These reactions generated mixtures containing 0.02 mg/mL CoaA, 0.02 mg/mL CoaD, 0.02 mg/mL CoaE, 0.02 mg/mL Sfp, 25 mM MgCl<sub>2</sub>, 2 mM DTT, 8 mM ATP, and 20 μM <strong>1</strong>, 4 μM MAS or 4 μM MAS mutant, and buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 10% glycerol) at their final concentrations. The reaction was then incubated at 23 °C for an additional 2 h. The experiments were conducted in triplicate and evaluated by 6% SDS-PAGE. Using densitometry calculations by ImageJ<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>, the crosslinking yield was determined by dividing crosslinked (&gt;290 kDa) band intensity by the total of crosslinked (&gt;290 kDa) and uncrosslinked (226 kDa) band intensity.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Crosslinking excised ACP from MAS in trans</h3>
<p id="Par47">Crosslinking in trans between ACP and excised KS, excised KS-LD-AT, full MAS, or FabB follows a similar protocol. Crosslinkers were loaded onto excised <em>apo</em>-ACP by a one-pot chemoenzymatic method<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. The reaction included 10.6 μL buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 10% glycerol), 29.4 μL excised ACP (0.17 mM), 0.5 μL CoaA (1 mg/mL), 0.5 μL CoaD (1 mg/mL), 0.5 μL CoaE (1 mg/mL), 1.4 μL Sfp (1.4 mg/mL), 2.5 μL MgCl<sub>2</sub> (250 mM), 0.5 μL DTT (100 mM), 2 μL ATP (200 mM), and 2 μL <strong>1,</strong>
<strong>2,</strong>
<strong>3,</strong>
<strong>4,</strong>
<strong>5,</strong>
<strong>6,</strong>
<strong>7</strong>, or <strong>8</strong> (from a 10 mM DMSO stock) (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>). The mixture contains the following components with final concentrations of 0.1 mM excised ACP, 1 mM DTT, 0.01 mg/mL CoaA, 0.01 mg/mL CoaD, 0.01 mg/mL CoaE, 0.04 mg/mL Sfp, 12.5 mM MgCl<sub>2</sub> 0.4 mM crosslinker, and 8 mM ATP. The reaction was incubated at 37 °C for 2 h. Excess probes were removed by 3 K MWCO Amicon Ultra spin concentrator (Millipore), resulting in a crude <em>crypto-</em>ACP with a concentration of 0.1 mM. Sequentially, 10 μL of this <em>crypto-</em>ACP (0.1 mM) mixture was added into 0.74 μL of excised KS (0.27 mM), excised KS-LD-AT (0.27 mM), full MAS (0.27 mM), or FabB (0.27 mM) in 9.2 μL buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 10% glycerol). The resulting reaction mixture containing 0.05 mM excised ACP, 0.01 mM KS containing protein (excised KS, excised KS-LD-AT, full MAS, or FabB) was incubated at 23 °C for 16 h. The in trans crosslinking was evaluated by 6% or 12% SDS-PAGE gel.</p></section><section id="Sec30"><h3 class="pmc_sec_title">
<em>E. coli</em> AcpP crosslinking with MAS and <em>E. coli</em> partner proteins</h3>
<p id="Par48">Crosslinking between <em>E. coli</em> AcpP and full MAS, or <em>E. coli</em> type II partner protein FabF (KS), FabF Cys163Ala (KS active site residue mutant), FabG (KR), FabA (DH), FabI (ER), FabD (AT), excised KS, and full MAS follows a similar protocol. The expression, purification, and preparation of <em>E. coli</em> type II <em>apo</em>-AcpP, <em>crypto</em>-AcpP, and partner proteins adopt the same protocol as reported<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Crosslinkers were loaded onto AcpP by a one-pot chemoenzymatic method<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. This reaction included 0.1 mM AcpP, 1 mM DTT, 0.01 mg/mL CoaA, 0.01 mg/mL CoaD, 0.01 mg/mL CoaE, 0.04 mg/mL Sfp, 0.2 mM crosslinker <strong>2,</strong>
<strong>3,</strong>
<strong>5,</strong>
<strong>6</strong>, or <strong>8</strong>, and 8 mM ATP in buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 10% glycerol). These reactions were incubated at 23 °C for 21 h. The reaction mixture was then purified by a HiLoad Superdex 75 pg (16 × 600) (16 × 600) (GE Biosciences) size exclusion column through FPLC (ÄKTA Pure) using a buffer containing 50 mM Tris • HCl pH 7.4, 150 mM NaCl, 10% glycerol to yield pure <em>crypto-</em>AcpP loaded with crosslinker <strong>2,</strong>
<strong>3,</strong>
<strong>5,</strong>
<strong>6</strong>, or <strong>8</strong> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). <em>Crypto-</em>AcpPs were concentrated to 0.1 mM by a 3 K MWCO Amicon Ultra spin concentrator (Millipore), flashed frozen, and stored at −80 °C. A crosslinking reaction contains the following components with their final concentrations of 0.05 mM <em>crypto-</em>AcpP with 0.01 mM FabF, 0.01 mM FabF Cys163Ala, 0.01 mM FabG, 0.01 mM FabA, 0.01 mM FabI, 0.01 mM FabD, 0.01 mM excised KS, or 0.01 mM full MAS. The crosslinking was evaluated by 6% or 12% SDS-PAGE gel (Supplementary Figs. <a href="#MOESM1" class="usa-link">5</a>–<a href="#MOESM1" class="usa-link">6</a>).</p></section><section id="Sec31"><h3 class="pmc_sec_title">Evaluation for the integrity of MAS and MAS mutants</h3>
<p id="Par49">Potential protein misfolding in MAS and MAS mutants were evaluated by labeling with fluorescent probe <strong>13</strong> (Fig. <a href="#Fig3" class="usa-link">3</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">44</a>). The labeling reaction contains a final concentration of 4 μM of MAS, 4 μM of MAS mutants, or 4 μM of denatured MAS with 0.02 mM <strong>13</strong>, 20 μM Sfp, 8 mM ATP, and 25 mM MgCl<sub>2</sub> in buffer (50 mM Tris • HCl pH 7.4, 150 mM NaCl, 5% glycerol). After 3 h of incubation at 25 °C, samples were evaluated with native 4–15% Mini-PROTEAN RGX (BioRad) gels at 150 V for 1 h. The resulting gels were imaged with a UV transilluminator at 365 nm (fluorescence) prior to Coomassie Brilliant Blue staining (total protein) (Supplementary Fig. <a href="#MOESM1" class="usa-link">44</a>).</p></section><section id="Sec32"><h3 class="pmc_sec_title">SDS PAGE and native PAGE gel analyses</h3>
<p id="Par50">SDS PAGE analyses were conducted with in-house prepared 6% gels or commercially prepared gradient SDS PAGE. SDS PAGE gels (Supplementary Fig. <a href="#MOESM1" class="usa-link">45</a>) were developed to identify the crosslinked products. Full gel images are used for all figures unless otherwise noted, and copies of these gels are in Supplementary Fig. <a href="#MOESM1" class="usa-link">46</a>. Conventional running buffers were used unless noted otherwise.</p></section><section id="Sec33"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par51">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec34"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370991/bin/41467_2025_63024_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (126.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370991/bin/41467_2025_63024_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370991/bin/41467_2025_63024_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (3.8MB, pdf) </sup>
</div></div></section></section><section id="Sec35"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370991/bin/41467_2025_63024_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (17.2KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors acknowledge the facilities of the UC San Diego cryo-EM facility, part of the Goeddel Family Technology Sandbox, along with the scientific and technical assistance of facility staff members Dr. Mariusz Matyszewski and Dr. Inga Kuschnerus. We also acknowledge J. Noel (Salk Institute), D. Lyumkis (Salk Institute), S. Konno (UC San Diego), D.O. Passos (Salk Institute), and A. Patel (UC San Diego) for initial studies on D.H. crosslinking<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> of MAS. This study was supported by NIH R01GM095970 to MDB and facilitated by a cryo-EM Seed Grant from the UC San Diego Department of Chemistry and Biochemistry. G.W.H. was supported by K12HL141956.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Z.J., J.J.L., and M.D.B. designed the research. Z.J., G.W.H., and J.A.C. produced the samples. Z.J., J.A.C., and J.H. conducted crosslinking and mutational studies. Z.J. and G.W.H. prepared the samples for cryoEM. G.W.H. and Z.J. carried out the cryoEM data collection. Z.J., G.W.H., J.J.L., and M.D.B. analyzed the cryoEM data. J.J.L. and M.D.B. guided the project. M.D.B. supervised the project. Z.J., G.W.H., J.J.L., and M.D.B. wrote the manuscript with input from all authors.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par52"><em>Nature Communications</em> thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Single-particle cryo-EM maps for complexes A–D have been deposited in the Electron Microscopy Data Bank (EMDB) and the associated atomic model coordinates for complexes A-C in the Protein Data Bank (PDB) under accession codes <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-46504" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMDB-46504</a> and <a href="https://doi.org/10.2210/pdb9D2Y/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9D2Y</a> (complex A); <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-46505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMDB-46505</a> and <a href="https://doi.org/10.2210/pdb9D2Z/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9D2Z</a> (complex B); <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-46506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMDB-46506</a> and <a href="https://www.wwpdb.org/pdb?id=pdb_00009d30" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9D30</a> (complex C); and <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-46507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMDB-46507</a> (complex D). Summary data for these structures are presented in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>. <a href="#Sec35" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par53">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Ziran Jiang, Graham W. Heberlig.</p></div>
</div></section><section id="sec36"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63024-x.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Furin, J., Cox, H. &amp; Pai, M. Tuberculosis. <em>Lancet</em><strong>393</strong>, 1642–1656 (2019).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(19)30308-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30904262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Furin,%20J.,%20Cox,%20H.%20&amp;%20Pai,%20M.%20Tuberculosis.%20Lancet393,%201642%E2%80%931656%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Jaganath, D., Beaudry, J. &amp; Salazar–Austin, N. Tuberculosis in children. <em>Infect. Dis. Clin. North. Am.</em><strong>36</strong>, 49–71 (2022).
</cite> [<a href="https://doi.org/10.1016/j.idc.2021.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8867883/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35168714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jaganath,%20D.,%20Beaudry,%20J.%20&amp;%20Salazar%E2%80%93Austin,%20N.%20Tuberculosis%20in%20children.%20Infect.%20Dis.%20Clin.%20North.%20Am.36,%2049%E2%80%9371%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Wulandari, D. A., Hartati, Y. W., Ibrahim, A. U., Pitaloka, D. A. E. &amp; Irkham Multidrug–resistant tuberculosis. <em>Clin. Chim. Acta</em><strong>59</strong>, 119701 (2024).</cite> [<a href="https://doi.org/10.1016/j.cca.2024.119701" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38697459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wulandari,%20D.%20A.,%20Hartati,%20Y.%20W.,%20Ibrahim,%20A.%20U.,%20Pitaloka,%20D.%20A.%20E.%20&amp;%20Irkham%20Multidrug%E2%80%93resistant%20tuberculosis.%20Clin.%20Chim.%20Acta59,%20119701%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Khawbung, J. L., Nath, D. &amp; Chakraborty, S. Drug resistant tuberculosis: a review. <em>Comp. Immunol. Microbiol. Infect. Dis.</em><strong>74</strong>, 101574 (2012).</cite> [<a href="https://doi.org/10.1016/j.cimid.2020.101574" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33249329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khawbung,%20J.%20L.,%20Nath,%20D.%20&amp;%20Chakraborty,%20S.%20Drug%20resistant%20tuberculosis:%20a%20review.%20Comp.%20Immunol.%20Microbiol.%20Infect.%20Dis.74,%20101574%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Herbst, D. A., Jakob, R. P., Zähringer, F. &amp; Maier, T. Mycocerosic acid synthase exemplifies the architecture of reducing polyketide synthases. <em>Nature</em><strong>531</strong>, 533–537 (2016).
</cite> [<a href="https://doi.org/10.1038/nature16993" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26976449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herbst,%20D.%20A.,%20Jakob,%20R.%20P.,%20Z%C3%A4hringer,%20F.%20&amp;%20Maier,%20T.%20Mycocerosic%20acid%20synthase%20exemplifies%20the%20architecture%20of%20reducing%20polyketide%20synthases.%20Nature531,%20533%E2%80%93537%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Mittal, E. et al. Mycobacterium tuberculosis virulence lipid PDIM inhibits autophagy in mice. <em>Nat. Microbiol</em>. 10.1038/s41564-024-01797-5. (2024).</cite> [<a href="https://doi.org/10.1038/s41564-024-01797-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12097147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39242815/" class="usa-link">PubMed</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Finin, P., Khan, R. M. N., Oh, S., Boshoff, H. I. M. &amp; Barry, C. E. 3rd Chemical approaches to unraveling the biology of mycobacteria. <em>Cell Chem. Biol.</em><strong>18</strong>, 420–435 (2023).</cite> [<a href="https://doi.org/10.1016/j.chembiol.2023.04.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10201459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37207631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Finin,%20P.,%20Khan,%20R.%20M.%20N.,%20Oh,%20S.,%20Boshoff,%20H.%20I.%20M.%20&amp;%20Barry,%20C.%20E.%203rd%20Chemical%20approaches%20to%20unraveling%20the%20biology%20of%20mycobacteria.%20Cell%20Chem.%20Biol.18,%20420%E2%80%93435%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Chandra, P., Grigsby, S. J. &amp; Philips, J. A. Immune evasion and provocation by mycobacterium tuberculosis. <em>Nat. Rev. Microbiol.</em><strong>20</strong>, 750–766 (2022).
</cite> [<a href="https://doi.org/10.1038/s41579-022-00763-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9310001/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35879556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chandra,%20P.,%20Grigsby,%20S.%20J.%20&amp;%20Philips,%20J.%20A.%20Immune%20evasion%20and%20provocation%20by%20mycobacterium%20tuberculosis.%20Nat.%20Rev.%20Microbiol.20,%20750%E2%80%93766%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Rainwater, D. L. &amp; Kolattukudy, P. E. Fatty acid biosynthesis in <em>Mycobacterium tuberculosis</em> var. bovis Bacillus Calmette–Guérin. Purification and characterization of a novel fatty acid synthase, mycocerosic acid synthase, which elongates n–fatty acyl–CoA with methylmalonyl–CoA. <em>J. Biol. Chem.</em><strong>260</strong>, 616–623 (1985).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/3880746/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rainwater,%20D.%20L.%20&amp;%20Kolattukudy,%20P.%20E.%20Fatty%20acid%20biosynthesis%20in%20Mycobacterium%20tuberculosis%20var.%20bovis%20Bacillus%20Calmette%E2%80%93Gu%C3%A9rin.%20Purification%20and%20characterization%20of%20a%20novel%20fatty%20acid%20synthase,%20mycocerosic%20acid%20synthase,%20which%20elongates%20n%E2%80%93fatty%20acyl%E2%80%93CoA%20with%20methylmalonyl%E2%80%93CoA.%20J.%20Biol.%20Chem.260,%20616%E2%80%93623%20(1985)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Chopra, T. &amp; Gokhale, R. S. Polyketide versatility in the biosynthesis of complex mycobacterial cell wall lipids. <em>Methods Enzymol.</em><strong>459</strong>, 259–229 (2009).
</cite> [<a href="https://doi.org/10.1016/S0076-6879(09)04612-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19362644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chopra,%20T.%20&amp;%20Gokhale,%20R.%20S.%20Polyketide%20versatility%20in%20the%20biosynthesis%20of%20complex%20mycobacterial%20cell%20wall%20lipids.%20Methods%20Enzymol.459,%20259%E2%80%93229%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Herbst, D. A., Townsend, C. A. &amp; Maier, T. The architectures of iterative type I PKS and FAS. <em>Nat. Prod. Rep.</em><strong>35</strong>, 1046–1069 (2018).
</cite> [<a href="https://doi.org/10.1039/c8np00039e" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6192843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30137093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herbst,%20D.%20A.,%20Townsend,%20C.%20A.%20&amp;%20Maier,%20T.%20The%20architectures%20of%20iterative%20type%20I%20PKS%20and%20FAS.%20Nat.%20Prod.%20Rep.35,%201046%E2%80%931069%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Zheng, Z., Gay, D. C., Demeler, B., White, M. A. &amp; Keatinge–Clay, A. T. Divergence of multimodular polyketide synthases revealed by a didomain structure. <em>Nat. Chem. Biol.</em><strong>8</strong>, 615–621 (2012).
</cite> [<a href="https://doi.org/10.1038/nchembio.964" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3477503/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22634636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20Z.,%20Gay,%20D.%20C.,%20Demeler,%20B.,%20White,%20M.%20A.%20&amp;%20Keatinge%E2%80%93Clay,%20A.%20T.%20Divergence%20of%20multimodular%20polyketide%20synthases%20revealed%20by%20a%20didomain%20structure.%20Nat.%20Chem.%20Biol.8,%20615%E2%80%93621%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Cogan, D. P. et al. Mapping the catalytic conformations of an assembly–line polyketide synthase module. <em>Science</em><strong>374</strong>, 729–734 (2021).
</cite> [<a href="https://doi.org/10.1126/science.abi8358" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8842837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34735239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cogan,%20D.%20P.%20et%20al.%20Mapping%20the%20catalytic%20conformations%20of%20an%20assembly%E2%80%93line%20polyketide%20synthase%20module.%20Science374,%20729%E2%80%93734%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Bagde, S. R., Mathews, I. I., Fromme, J. C. &amp; Kim, C. Y. Modular polyketide synthase contains two reaction chambers that operate asynchronously. <em>Science</em><strong>374</strong>, 723–729 (2021).
</cite> [<a href="https://doi.org/10.1126/science.abi8532" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8689591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34735234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bagde,%20S.%20R.,%20Mathews,%20I.%20I.,%20Fromme,%20J.%20C.%20&amp;%20Kim,%20C.%20Y.%20Modular%20polyketide%20synthase%20contains%20two%20reaction%20chambers%20that%20operate%20asynchronously.%20Science374,%20723%E2%80%93729%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Kim, S. K. et al. Structure and dynamics of the essential endogenous mycobacterial polyketide synthase Pks13. <em>Nat. Struct. Mol. Biol.</em><strong>30</strong>, 296–308 (2023).
</cite> [<a href="https://doi.org/10.1038/s41594-022-00918-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10312659/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36782050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20S.%20K.%20et%20al.%20Structure%20and%20dynamics%20of%20the%20essential%20endogenous%20mycobacterial%20polyketide%20synthase%20Pks13.%20Nat.%20Struct.%20Mol.%20Biol.30,%20296%E2%80%93308%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Tittes, Y. U. et al. The structure of a polyketide synthase bimodule core. <em>Sci. Adv.</em><strong>8</strong>, eabo6918 (2022).
</cite> [<a href="https://doi.org/10.1126/sciadv.abo6918" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9491710/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36129979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tittes,%20Y.%20U.%20et%20al.%20The%20structure%20of%20a%20polyketide%20synthase%20bimodule%20core.%20Sci.%20Adv.8,%20eabo6918%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Wang, J. et al. Structural basis for the biosynthesis of lovastatin. <em>Nat. Commun.</em><strong>12</strong>, 867 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-21174-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7870829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33558520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20J.%20et%20al.%20Structural%20basis%20for%20the%20biosynthesis%20of%20lovastatin.%20Nat.%20Commun.12,%20867%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Finzel, K., Lee, D. J. &amp; Burkart, M. D. Using modern tools to probe the structure–function relationship K.of fatty acid synthases. <em>Chembiochem</em><strong>16</strong>, 528–547 (2015).
</cite> [<a href="https://doi.org/10.1002/cbic.201402578" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4545599/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25676190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Finzel,%20K.,%20Lee,%20D.%20J.%20&amp;%20Burkart,%20M.%20D.%20Using%20modern%20tools%20to%20probe%20the%20structure%E2%80%93function%20relationship%20K.of%20fatty%20acid%20synthases.%20Chembiochem16,%20528%E2%80%93547%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Chen, A., Jiang, Z. &amp; Burkart, M. D. Enzymology of standalone elongating ketosynthases. <em>Chem. Sci.</em><strong>13</strong>, 4225–4238 (2022).
</cite> [<a href="https://doi.org/10.1039/d1sc07256k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9006962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35509474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20A.,%20Jiang,%20Z.%20&amp;%20Burkart,%20M.%20D.%20Enzymology%20of%20standalone%20elongating%20ketosynthases.%20Chem.%20Sci.13,%204225%E2%80%934238%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Jiang, Z. et al. Masked cerulenin enables a dual–site selective protein crosslink. <em>Chem. Sci.</em><strong>14</strong>, 10925–11093 (2023).
</cite> [<a href="https://doi.org/10.1039/d3sc02864j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10566503/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37829009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang,%20Z.%20et%20al.%20Masked%20cerulenin%20enables%20a%20dual%E2%80%93site%20selective%20protein%20crosslink.%20Chem.%20Sci.14,%2010925%E2%80%9311093%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Konno, S., La Clair, J. J. &amp; Burkart, M. D. Trapping the complex molecular machinery of polyketide and fatty acid synthases with tunable silylcyanohydrin crosslinkers. <em>Angew. Chem. Int. Ed. Engl.</em><strong>57</strong>, 17009–17013 (2018).
</cite> [<a href="https://doi.org/10.1002/anie.201806865" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6407627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30379389/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Konno,%20S.,%20La%20Clair,%20J.%20J.%20&amp;%20Burkart,%20M.%20D.%20Trapping%20the%20complex%20molecular%20machinery%20of%20polyketide%20and%20fatty%20acid%20synthases%20with%20tunable%20silylcyanohydrin%20crosslinkers.%20Angew.%20Chem.%20Int.%20Ed.%20Engl.57,%2017009%E2%80%9317013%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Ishikawa, F., Haushalter, R. W., Lee, D. J., Finzel, K. &amp; Burkart, M. D. Sulfonyl 3–alkynyl pantetheinamides as mechanism–based cross–linkers of acyl carrier protein dehydratase. <em>J. Am. Chem. Soc.</em><strong>135</strong>, 8846–8849 (2013).
</cite> [<a href="https://doi.org/10.1021/ja4042059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3713789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23718183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ishikawa,%20F.,%20Haushalter,%20R.%20W.,%20Lee,%20D.%20J.,%20Finzel,%20K.%20&amp;%20Burkart,%20M.%20D.%20Sulfonyl%203%E2%80%93alkynyl%20pantetheinamides%20as%20mechanism%E2%80%93based%20cross%E2%80%93linkers%20of%20acyl%20carrier%20protein%20dehydratase.%20J.%20Am.%20Chem.%20Soc.135,%208846%E2%80%938849%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Tallorin, L. et al. Trapping of the enoyl–acyl carrier protein reductase–acyl carrier protein interaction. <em>J. Am. Chem. Soc.</em><strong>138</strong>, 3962–3965 (2016).
</cite> [<a href="https://doi.org/10.1021/jacs.5b13456" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4988885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26938266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tallorin,%20L.%20et%20al.%20Trapping%20of%20the%20enoyl%E2%80%93acyl%20carrier%20protein%20reductase%E2%80%93acyl%20carrier%20protein%20interaction.%20J.%20Am.%20Chem.%20Soc.138,%203962%E2%80%933965%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Chen, A. et al. Visualizing the interface of biotin and fatty acid biosynthesis through SuFEx probes. <em>J. Am. Chem. Soc.</em><strong>146</strong>, 1388–1395 (2024).
</cite> [<a href="https://doi.org/10.1021/jacs.3c10181" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12332981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38176024/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20A.%20et%20al.%20Visualizing%20the%20interface%20of%20biotin%20and%20fatty%20acid%20biosynthesis%20through%20SuFEx%20probes.%20J.%20Am.%20Chem.%20Soc.146,%201388%E2%80%931395%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Worthington, A. S. &amp; Burkart, M. D. One–pot chemo–enzymatic synthesis of reporter–modified proteins. <em>Org. Biomol. Chem.</em><strong>4</strong>, 44–46 (2006).
</cite> [<a href="https://doi.org/10.1039/b512735a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16357994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Worthington,%20A.%20S.%20&amp;%20Burkart,%20M.%20D.%20One%E2%80%93pot%20chemo%E2%80%93enzymatic%20synthesis%20of%20reporter%E2%80%93modified%20proteins.%20Org.%20Biomol.%20Chem.4,%2044%E2%80%9346%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>van Wyk, M. &amp; Strauss, E. One-pot preparation of coenzyme A analogues via an improved chemo-enzymatic synthesis of pre-CoA thioester synthons. <em>Chem. Commun.</em><strong>4</strong>, 398–400 (2008).</cite> [<a href="https://doi.org/10.1039/b613527g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17220983/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20Wyk,%20M.%20&amp;%20Strauss,%20E.%20One-pot%20preparation%20of%20coenzyme%20A%20analogues%20via%20an%20improved%20chemo-enzymatic%20synthesis%20of%20pre-CoA%20thioester%20synthons.%20Chem.%20Commun.4,%20398%E2%80%93400%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Nava, A. A., Roberts, J., Haushalter, R. W., Wang, Z. &amp; Keasling, J. D. Module-based polyketide synthase engineering for de novo polyketide biosynthesis. <em>ACS Synth. Biol.</em><strong>12</strong>, 3148–3155 (2023).
</cite> [<a href="https://doi.org/10.1021/acssynbio.3c00282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10661043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37871264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nava,%20A.%20A.,%20Roberts,%20J.,%20Haushalter,%20R.%20W.,%20Wang,%20Z.%20&amp;%20Keasling,%20J.%20D.%20Module-based%20polyketide%20synthase%20engineering%20for%20de%20novo%20polyketide%20biosynthesis.%20ACS%20Synth.%20Biol.12,%203148%E2%80%933155%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Su, L. et al. Engineering the stambomycin modular polyketide synthase yields 37-membered mini-stambomycins. <em>Nat. Comm.</em><strong>13</strong>, 515 (2022).</cite> [<a href="https://doi.org/10.1038/s41467-022-27955-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8792006/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35082289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su,%20L.%20et%20al.%20Engineering%20the%20stambomycin%20modular%20polyketide%20synthase%20yields%2037-membered%20mini-stambomycins.%20Nat.%20Comm.13,%20515%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Soohoo, A. M., Cogan, D. P., Brodsky, K. L. &amp; Khosla, C. Structure and mechanisms of assembly-line polyketide synthases. <em>Annu. Rev. Biochem.</em><strong>93</strong>, 471–498 (2024).
</cite> [<a href="https://doi.org/10.1146/annurev-biochem-080923-043654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11907408/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38663033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Soohoo,%20A.%20M.,%20Cogan,%20D.%20P.,%20Brodsky,%20K.%20L.%20&amp;%20Khosla,%20C.%20Structure%20and%20mechanisms%20of%20assembly-line%20polyketide%20synthases.%20Annu.%20Rev.%20Biochem.93,%20471%E2%80%93498%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Cogan, D. P. et al. Structural basis for intermodular communication in assembly-line polyketide biosynthesis. <em>Nat. Chem. Biol</em>. 10.1038/s41589-024-01709-y. (2024).</cite> [<a href="https://doi.org/10.1038/s41589-024-01709-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11909739/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39179672/" class="usa-link">PubMed</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Buyachuihan, L., Stegemann, F. &amp; Grininger, M. How acyl carrier proteins (ACPs) direct fatty acid and polyketide biosynthesis. <em>Angew. Chem. Int. Ed. Engl.</em><strong>63</strong>, e202312476 (2024).
</cite> [<a href="https://doi.org/10.1002/anie.202312476" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37856285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buyachuihan,%20L.,%20Stegemann,%20F.%20&amp;%20Grininger,%20M.%20How%20acyl%20carrier%20proteins%20(ACPs)%20direct%20fatty%20acid%20and%20polyketide%20biosynthesis.%20Angew.%20Chem.%20Int.%20Ed.%20Engl.63,%20e202312476%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Worthington, A. S., Porter, D. F. &amp; Burkart, M. D. Mechanism–based crosslinking as a gauge for functional interaction of modular synthases. <em>Org. Biomol. Chem.</em><strong>8</strong>, 1769–1772 (2010).
</cite> [<a href="https://doi.org/10.1039/b925966j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20449476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Worthington,%20A.%20S.,%20Porter,%20D.%20F.%20&amp;%20Burkart,%20M.%20D.%20Mechanism%E2%80%93based%20crosslinking%20as%20a%20gauge%20for%20functional%20interaction%20of%20modular%20synthases.%20Org.%20Biomol.%20Chem.8,%201769%E2%80%931772%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Mindrebo, J. T. et al. Gating mechanism of elongating β–ketoacyl–ACP synthases. <em>Nat. Commun.</em><strong>11</strong>, 1727 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-15455-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7138838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32265440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mindrebo,%20J.%20T.%20et%20al.%20Gating%20mechanism%20of%20elongating%20%CE%B2%E2%80%93ketoacyl%E2%80%93ACP%20synthases.%20Nat.%20Commun.11,%201727%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Mindrebo, J. T. et al. Structure and mechanistic analyses of the gating mechanism of elongating ketosynthases. <em>ACS Catal.</em><strong>11</strong>, 6787–6799 (2021).
</cite> [<a href="https://doi.org/10.1021/acscatal.1c00745" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9524369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36187225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mindrebo,%20J.%20T.%20et%20al.%20Structure%20and%20mechanistic%20analyses%20of%20the%20gating%20mechanism%20of%20elongating%20ketosynthases.%20ACS%20Catal.11,%206787%E2%80%936799%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Zanon, P. R. A. et al. Profiling the proteome-wide selectivity of diverse electrophiles <em>ChemRxiv</em>. 10.26434/chemrxiv-2021-w7rss-v2 (2021).</cite>
</li>
<li id="CR36">
<span class="label">36.</span><cite>Lowry, B., Li, X., Robbins, T., Cane, D. E. &amp; Khosla, C. A. Turnstile mechanism for the controlled growth of biosynthetic intermediates on assembly line polyketide synthases. <em>ACS Cent. Sci.</em><strong>2</strong>, 14–20 (2016).
</cite> [<a href="https://doi.org/10.1021/acscentsci.5b00321" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4731828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26878060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lowry,%20B.,%20Li,%20X.,%20Robbins,%20T.,%20Cane,%20D.%20E.%20&amp;%20Khosla,%20C.%20A.%20Turnstile%20mechanism%20for%20the%20controlled%20growth%20of%20biosynthetic%20intermediates%20on%20assembly%20line%20polyketide%20synthases.%20ACS%20Cent.%20Sci.2,%2014%E2%80%9320%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Grininger, M. Enzymology of assembly line synthesis by modular polyketide synthases. <em>Nat. Chem. Biol.</em><strong>19</strong>, 401–415 (2023).
</cite> [<a href="https://doi.org/10.1038/s41589-023-01277-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36914860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grininger,%20M.%20Enzymology%20of%20assembly%20line%20synthesis%20by%20modular%20polyketide%20synthases.%20Nat.%20Chem.%20Biol.19,%20401%E2%80%93415%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Nguyen, C. et al. Trapping the dynamic acyl carrier protein in fatty acid biosynthesis. <em>Nature</em><strong>505</strong>, 427–431 (2014).
</cite> [<a href="https://doi.org/10.1038/nature12810" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4437705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24362570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen,%20C.%20et%20al.%20Trapping%20the%20dynamic%20acyl%20carrier%20protein%20in%20fatty%20acid%20biosynthesis.%20Nature505,%20427%E2%80%93431%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Shen, S. et al. A back-door Phenylalanine coordinates the stepwise hexameric loading of acyl carrier protein by the fatty acid biosynthesis enzyme β-hydroxyacyl-acyl carrier protein dehydratase (FabZ). <em>Int. J. Biol. Macromol.</em><strong>128</strong>, 5–11 (2019).
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2019.01.094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30677439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen,%20S.%20et%20al.%20A%20back-door%20Phenylalanine%20coordinates%20the%20stepwise%20hexameric%20loading%20of%20acyl%20carrier%20protein%20by%20the%20fatty%20acid%20biosynthesis%20enzyme%20%CE%B2-hydroxyacyl-acyl%20carrier%20protein%20dehydratase%20(FabZ).%20Int.%20J.%20Biol.%20Macromol.128,%205%E2%80%9311%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Brignole, E. J., Smith, S. &amp; Asturias, F. J. Conformational flexibility of metazoan fatty acid synthase enables catalysis. <em>Nat. Struct. Mol. Biol.</em><strong>16</strong>, 190–197 (2009).
</cite> [<a href="https://doi.org/10.1038/nsmb.1532" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2653270/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19151726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brignole,%20E.%20J.,%20Smith,%20S.%20&amp;%20Asturias,%20F.%20J.%20Conformational%20flexibility%20of%20metazoan%20fatty%20acid%20synthase%20enables%20catalysis.%20Nat.%20Struct.%20Mol.%20Biol.16,%20190%E2%80%93197%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Dodge, G. J. et al. Structural and dynamical rationale for fatty acid unsaturation in <em>Escherichia coli</em>. <em>Proc. Natl. Acad. Sci. USA</em><strong>116</strong>, 6775–6783 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1818686116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6452729/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30872475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dodge,%20G.%20J.%20et%20al.%20Structural%20and%20dynamical%20rationale%20for%20fatty%20acid%20unsaturation%20in%20Escherichia%20coli.%20Proc.%20Natl.%20Acad.%20Sci.%20USA116,%206775%E2%80%936783%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Leesong, M., Henderson, B. S., Gillig, J. R., Schwab, J. M. &amp; Smith, J. L. Structure of a dehydratase-isomerase from the bacterial pathway for biosynthesis of unsaturated fatty acids: two catalytic activities in one active site. <em>Structure</em><strong>4</strong>, 253–264 (1996).
</cite> [<a href="https://doi.org/10.1016/s0969-2126(96)00030-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8805534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leesong,%20M.,%20Henderson,%20B.%20S.,%20Gillig,%20J.%20R.,%20Schwab,%20J.%20M.%20&amp;%20Smith,%20J.%20L.%20Structure%20of%20a%20dehydratase-isomerase%20from%20the%20bacterial%20pathway%20for%20biosynthesis%20of%20unsaturated%20fatty%20acids:%20two%20catalytic%20activities%20in%20one%20active%20site.%20Structure4,%20253%E2%80%93264%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Zhang, L. et al. Crystal structure of FabZ-ACP complex reveals a dynamic seesaw-like catalytic mechanism of dehydratase in fatty acid biosynthesis. <em>Cell. Res.</em><strong>26</strong>, 1330–1344 (2016).
</cite> [<a href="https://doi.org/10.1038/cr.2016.136" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5143422/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27874013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20L.%20et%20al.%20Crystal%20structure%20of%20FabZ-ACP%20complex%20reveals%20a%20dynamic%20seesaw-like%20catalytic%20mechanism%20of%20dehydratase%20in%20fatty%20acid%20biosynthesis.%20Cell.%20Res.26,%201330%E2%80%931344%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Schubert, H. L., Li, F., Hill, C. P. &amp; Schmidt, E. W. The structure of full-length AFPK supports the ACP linker in a role that regulates iterative polyketide and fatty acid assembly. <em>Proc. Natl. Acad. Sci. USA</em><strong>122</strong>, e2419884122 (2025).
</cite> [<a href="https://doi.org/10.1073/pnas.2419884122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11831141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39913209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schubert,%20H.%20L.,%20Li,%20F.,%20Hill,%20C.%20P.%20&amp;%20Schmidt,%20E.%20W.%20The%20structure%20of%20full-length%20AFPK%20supports%20the%20ACP%20linker%20in%20a%20role%20that%20regulates%20iterative%20polyketide%20and%20fatty%20acid%20assembly.%20Proc.%20Natl.%20Acad.%20Sci.%20USA122,%20e2419884122%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Weissman, K. J. Uncovering the structures of modular polyketide synthases. <em>Nat. Prod. Rep.</em><strong>32</strong>, 436–453 (2015).
</cite> [<a href="https://doi.org/10.1039/c4np00098f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25310997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weissman,%20K.%20J.%20Uncovering%20the%20structures%20of%20modular%20polyketide%20synthases.%20Nat.%20Prod.%20Rep.32,%20436%E2%80%93453%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Zheng, J., Fage, C. D., Demeler, B., Hoffman, D. W. &amp; Keatinge-Clay, A. T. The missing linker: a dimerization motif located within polyketide synthase modules. <em>ACS Chem. Biol.</em><strong>8</strong>, 1263–1270 (2013).
</cite> [<a href="https://doi.org/10.1021/cb400047s" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4365927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23489133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20J.,%20Fage,%20C.%20D.,%20Demeler,%20B.,%20Hoffman,%20D.%20W.%20&amp;%20Keatinge-Clay,%20A.%20T.%20The%20missing%20linker:%20a%20dimerization%20motif%20located%20within%20polyketide%20synthase%20modules.%20ACS%20Chem.%20Biol.8,%201263%E2%80%931270%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Hirsch, M., Fitzgerald, B. J. &amp; Keatinge-Clay, A. T. How cis-Acyltransferase Assembly-Line Ketosynthases Gatekeep for Processed Polyketide Intermediates. <em>ACS Chem. Biol.</em><strong>16</strong>, 2515–2526 (2021).
</cite> [<a href="https://doi.org/10.1021/acschembio.1c00598" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9879353/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34590822/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hirsch,%20M.,%20Fitzgerald,%20B.%20J.%20&amp;%20Keatinge-Clay,%20A.%20T.%20How%20cis-Acyltransferase%20Assembly-Line%20Ketosynthases%20Gatekeep%20for%20Processed%20Polyketide%20Intermediates.%20ACS%20Chem.%20Biol.16,%202515%E2%80%932526%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Ray, K. A., Saif, N. &amp; Keatinge-Clay, A. T. Modular polyketide synthase ketosynthases collaborate with upstream dehydratases to install double bonds. <em>Chem. Commun.</em><strong>60</strong>, 8712–8715 (2024).</cite> [<a href="https://doi.org/10.1039/d4cc03034f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11321453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39056119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ray,%20K.%20A.,%20Saif,%20N.%20&amp;%20Keatinge-Clay,%20A.%20T.%20Modular%20polyketide%20synthase%20ketosynthases%20collaborate%20with%20upstream%20dehydratases%20to%20install%20double%20bonds.%20Chem.%20Commun.60,%208712%E2%80%938715%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Chen, A. et al. Mechanism-based cross-linking probes capture the <em>Escherichia coli</em> ketosynthase FabB in conformationally distinct catalytic states. <em>Acta Crystallogr. D. Struct. Biol.</em><strong>78</strong>, 1171–1179 (2022).
</cite> [<a href="https://doi.org/10.1107/S2059798322007434" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9435599/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36048156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20A.%20et%20al.%20Mechanism-based%20cross-linking%20probes%20capture%20the%20Escherichia%20coli%20ketosynthase%20FabB%20in%20conformationally%20distinct%20catalytic%20states.%20Acta%20Crystallogr.%20D.%20Struct.%20Biol.78,%201171%E2%80%931179%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Rainwater, L. &amp; Kolattukudy, P. E. Synthesis of mycocerosic acids from methylmalonyl coenzyme A by cell-free extracts of <em>Mycobacterium tuberculosis</em> var. bovis BCG. <em>J. Biol. Chem.</em><strong>258</strong>, 2979–2985 (1983).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/6402506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rainwater,%20L.%20&amp;%20Kolattukudy,%20P.%20E.%20Synthesis%20of%20mycocerosic%20acids%20from%20methylmalonyl%20coenzyme%20A%20by%20cell-free%20extracts%20of%20Mycobacterium%20tuberculosis%20var.%20bovis%20BCG.%20J.%20Biol.%20Chem.258,%202979%E2%80%932985%20(1983)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Sztain, T. et al. Decoding allosteric regulation by the acyl carrier protein. <em>Proc. Natl. Acad. Sci. USA</em><strong>18</strong>, e2025597118 (2021).</cite> [<a href="https://doi.org/10.1073/pnas.2025597118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8072227/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33846262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sztain,%20T.%20et%20al.%20Decoding%20allosteric%20regulation%20by%20the%20acyl%20carrier%20protein.%20Proc.%20Natl.%20Acad.%20Sci.%20USA18,%20e2025597118%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Whicher, J. R. et al. Structural rearrangements of a polyketide synthase module during its catalytic cycle. <em>Nature</em><strong>510</strong>, 560–564 (2014).
</cite> [<a href="https://doi.org/10.1038/nature13409" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4074775/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24965656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whicher,%20J.%20R.%20et%20al.%20Structural%20rearrangements%20of%20a%20polyketide%20synthase%20module%20during%20its%20catalytic%20cycle.%20Nature510,%20560%E2%80%93564%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Choi, W. et al. Structural dynamics of human fatty acid synthase in the condensing cycle. <em>Nature</em>10.1038/s41586-025-08782-w (2012).</cite> [<a href="https://doi.org/10.1038/s41586-025-08782-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12058526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39978408/" class="usa-link">PubMed</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Schultz, K. et al. Snapshots of acyl carrier protein shuttling in human fatty acid synthase. <em>Nature</em>10.1038/s41586-025-08587-x (2025).</cite> [<a href="https://doi.org/10.1038/s41586-025-08587-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12058525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39979457/" class="usa-link">PubMed</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Ishikawa, F., Haushalter, R. W. &amp; Burkart, M. D. Dehydratase-specific probes for fatty acid and polyketide synthases. <em>J. Am. Chem. Soc.</em><strong>134</strong>, 769–772 (2012).
</cite> [<a href="https://doi.org/10.1021/ja2082334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3269962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22188524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ishikawa,%20F.,%20Haushalter,%20R.%20W.%20&amp;%20Burkart,%20M.%20D.%20Dehydratase-specific%20probes%20for%20fatty%20acid%20and%20polyketide%20synthases.%20J.%20Am.%20Chem.%20Soc.134,%20769%E2%80%93772%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Worthington, A. S., Rivera, H., Torpey, J. W., Alexander, M. D. &amp; Burkart, M. D. Mechanism-based protein cross-linking probes to investigate carrier protein-mediated biosynthesis. <em>ACS Chem. Biol.</em><strong>1</strong>, 687–691 (2006).
</cite> [<a href="https://doi.org/10.1021/cb6003965" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17184132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Worthington,%20A.%20S.,%20Rivera,%20H.,%20Torpey,%20J.%20W.,%20Alexander,%20M.%20D.%20&amp;%20Burkart,%20M.%20D.%20Mechanism-based%20protein%20cross-linking%20probes%20to%20investigate%20carrier%20protein-mediated%20biosynthesis.%20ACS%20Chem.%20Biol.1,%20687%E2%80%93691%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Schneider, A., Rasband, W. S. &amp; Eliceiri, K. W. NIH Image to ImageJ: 25 years of image analysis. <em>Nat. Methods</em><strong>9</strong>, 671–675 (2012).
</cite> [<a href="https://doi.org/10.1038/nmeth.2089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5554542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schneider,%20A.,%20Rasband,%20W.%20S.%20&amp;%20Eliceiri,%20K.%20W.%20NIH%20Image%20to%20ImageJ:%2025%20years%20of%20image%20analysis.%20Nat.%20Methods9,%20671%E2%80%93675%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Menendez-Bravo, S., Comba, S., Sabatini, M., Arabolaza, A. &amp; Gramajo, H. Expanding the chemical diversity of natural esters by engineering a polyketide-derived pathway into <em>Escherichia coli</em>. <em>Metab. Eng.</em><strong>24</strong>, 97–106 (2014).
</cite> [<a href="https://doi.org/10.1016/j.ymben.2014.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24831705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Menendez-Bravo,%20S.,%20Comba,%20S.,%20Sabatini,%20M.,%20Arabolaza,%20A.%20&amp;%20Gramajo,%20H.%20Expanding%20the%20chemical%20diversity%20of%20natural%20esters%20by%20engineering%20a%20polyketide-derived%20pathway%20into%20Escherichia%20coli.%20Metab.%20Eng.24,%2097%E2%80%93106%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Schlager, B., Straessle, A. &amp; Hafen, E. Use of anionic denaturing detergents to purify insoluble proteins after overexpression. <em>BMC Biotechnol.</em><strong>12</strong>, 95 (2012).
</cite> [<a href="https://doi.org/10.1186/1472-6750-12-95" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3536628/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23231964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schlager,%20B.,%20Straessle,%20A.%20&amp;%20Hafen,%20E.%20Use%20of%20anionic%20denaturing%20detergents%20to%20purify%20insoluble%20proteins%20after%20overexpression.%20BMC%20Biotechnol.12,%2095%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Finzel, K., Beld, J., Burkart, M. D. &amp; Charkoudian, L. K. Utilizing mechanistic cross-linking technology to study protein-protein interactions: an experiment designed for an undergraduate biochemistry lab. <em>J. Chem. Educ.</em><strong>94</strong>, 375–379 (2017).
</cite> [<a href="https://doi.org/10.1021/acs.jchemed.6b00453" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5731787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29255327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Finzel,%20K.,%20Beld,%20J.,%20Burkart,%20M.%20D.%20&amp;%20Charkoudian,%20L.%20K.%20Utilizing%20mechanistic%20cross-linking%20technology%20to%20study%20protein-protein%20interactions:%20an%20experiment%20designed%20for%20an%20undergraduate%20biochemistry%20lab.%20J.%20Chem.%20Educ.94,%20375%E2%80%93379%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Punjani, A., Rubinstein, J. L., Fleet, D. J. &amp; Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. <em>Nat. Methods</em><strong>14</strong>, 290–296 (2017).
</cite> [<a href="https://doi.org/10.1038/nmeth.4169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28165473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Punjani,%20A.,%20Rubinstein,%20J.%20L.,%20Fleet,%20D.%20J.%20&amp;%20Brubaker,%20M.%20A.%20cryoSPARC:%20algorithms%20for%20rapid%20unsupervised%20cryo-EM%20structure%20determination.%20Nat.%20Methods14,%20290%E2%80%93296%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Rubinstein, J. L. &amp; Brubaker, M. A. Alignment of cryo-EM movies of individual particles by optimization of image translations. <em>J. Struct. Biol.</em><strong>192</strong>, 188–195 (2015).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2015.08.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26296328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rubinstein,%20J.%20L.%20&amp;%20Brubaker,%20M.%20A.%20Alignment%20of%20cryo-EM%20movies%20of%20individual%20particles%20by%20optimization%20of%20image%20translations.%20J.%20Struct.%20Biol.192,%20188%E2%80%93195%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Punjani, A., Zhang, H. &amp; Fleet, D. J. Non-uniform refinement: adaptive regularization improves single-particle cryo-EM reconstruction. <em>Nat. Methods</em><strong>17</strong>, 1214–1221 (2020).
</cite> [<a href="https://doi.org/10.1038/s41592-020-00990-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33257830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Punjani,%20A.,%20Zhang,%20H.%20&amp;%20Fleet,%20D.%20J.%20Non-uniform%20refinement:%20adaptive%20regularization%20improves%20single-particle%20cryo-EM%20reconstruction.%20Nat.%20Methods17,%201214%E2%80%931221%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Sanchez-Garcia, R. et al. DeepEMhancer: a deep learning solution for cryo-EM volume post-processing. <em>Commun. Biol.</em><strong>4</strong>, 874 (2021).
</cite> [<a href="https://doi.org/10.1038/s42003-021-02399-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8282847/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34267316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanchez-Garcia,%20R.%20et%20al.%20DeepEMhancer:%20a%20deep%20learning%20solution%20for%20cryo-EM%20volume%20post-processing.%20Commun.%20Biol.4,%20874%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370991/bin/41467_2025_63024_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (126.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370991/bin/41467_2025_63024_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370991/bin/41467_2025_63024_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (3.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370991/bin/41467_2025_63024_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (17.2KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Single-particle cryo-EM maps for complexes A–D have been deposited in the Electron Microscopy Data Bank (EMDB) and the associated atomic model coordinates for complexes A-C in the Protein Data Bank (PDB) under accession codes <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-46504" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMDB-46504</a> and <a href="https://doi.org/10.2210/pdb9D2Y/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9D2Y</a> (complex A); <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-46505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMDB-46505</a> and <a href="https://doi.org/10.2210/pdb9D2Z/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9D2Z</a> (complex B); <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-46506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMDB-46506</a> and <a href="https://www.wwpdb.org/pdb?id=pdb_00009d30" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9D30</a> (complex C); and <a href="https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-46507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMDB-46507</a> (complex D). Summary data for these structures are presented in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>. <a href="#Sec35" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63024-x"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63024.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370991/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370991/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370991%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370991/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370991/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370991/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841798/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370991/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841798/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370991/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370991/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="P1Wn6N9byZqIOtELc1N7ZN6YJFNmSYgJedx0MXArotqHf398QDx30SyuhuRn6wy1">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
