
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Comprehensive bioinformatics analysis of EXOSC family genes in head and neck squamous cell carcinoma - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4BA1A8AF40ED305BA1A0014CB673E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365086/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Comprehensive bioinformatics analysis of EXOSC family genes in head and neck squamous cell carcinoma">
<meta name="citation_author" content="Xuezhong Zhang">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, Zibo Central Hospital, Zibo, Shandong China">
<meta name="citation_author" content="Mengmeng Zhao">
<meta name="citation_author_institution" content="Department of Stomatology, Binzhou People’s Hospital, Binzhou, Shandong China">
<meta name="citation_author" content="Tingting Chu">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, Zibo Central Hospital, Zibo, Shandong China">
<meta name="citation_author" content="Jiasha Wei">
<meta name="citation_author_institution" content="Department of Otolaryngology Head and Neck Surgery, Zibo Central Hospital, Zibo, Shandong China">
<meta name="citation_author" content="Qingmei Jia">
<meta name="citation_author_institution" content="School of Health Management, Binzhou Medical University, Yantai, Shandong China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30361">
<meta name="citation_doi" content="10.1038/s41598-025-15758-3">
<meta name="citation_pmid" content="40830179">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365086/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365086/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365086/pdf/41598_2025_Article_15758.pdf">
<meta name="description" content="Head and neck squamous cell carcinoma (HNSC) is a highly aggressive malignancy with poor prognosis, necessitating the identification of novel biomarkers for improved diagnosis and treatment. The exosome complex (EXOSC) family plays a crucial role in ...">
<meta name="og:title" content="Comprehensive bioinformatics analysis of EXOSC family genes in head and neck squamous cell carcinoma">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Head and neck squamous cell carcinoma (HNSC) is a highly aggressive malignancy with poor prognosis, necessitating the identification of novel biomarkers for improved diagnosis and treatment. The exosome complex (EXOSC) family plays a crucial role in ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365086/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365086">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-15758-3"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_15758.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365086%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365086/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365086/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365086/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 19;15:30361. doi: <a href="https://doi.org/10.1038/s41598-025-15758-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-15758-3</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Comprehensive bioinformatics analysis of EXOSC family genes in head and neck squamous cell carcinoma</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Xuezhong Zhang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xuezhong Zhang</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, Zibo Central Hospital, Zibo, Shandong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xuezhong Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Mengmeng Zhao</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Mengmeng Zhao</span></h3>
<div class="p">
<sup>2</sup>Department of Stomatology, Binzhou People’s Hospital, Binzhou, Shandong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mengmeng Zhao</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chu%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Tingting Chu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Tingting Chu</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, Zibo Central Hospital, Zibo, Shandong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chu%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tingting Chu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jiasha Wei</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jiasha Wei</span></h3>
<div class="p">
<sup>3</sup>Department of Otolaryngology Head and Neck Surgery, Zibo Central Hospital, Zibo, Shandong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiasha Wei</span></a>
</div>
</div>
<sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jia%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Qingmei Jia</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Qingmei Jia</span></h3>
<div class="p">
<sup>4</sup>School of Health Management, Binzhou Medical University, Yantai, Shandong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jia%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qingmei Jia</span></a>
</div>
</div>
<sup>4,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Laboratory Medicine, Zibo Central Hospital, Zibo, Shandong China </div>
<div id="Aff2">
<sup>2</sup>Department of Stomatology, Binzhou People’s Hospital, Binzhou, Shandong China </div>
<div id="Aff3">
<sup>3</sup>Department of Otolaryngology Head and Neck Surgery, Zibo Central Hospital, Zibo, Shandong China </div>
<div id="Aff4">
<sup>4</sup>School of Health Management, Binzhou Medical University, Yantai, Shandong China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 11; Accepted 2025 Aug 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365086  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830179/" class="usa-link">40830179</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Head and neck squamous cell carcinoma (HNSC) is a highly aggressive malignancy with poor prognosis, necessitating the identification of novel biomarkers for improved diagnosis and treatment. The exosome complex (EXOSC) family plays a crucial role in RNA metabolism, but its significance in HNSC remains poorly understood. We performed a comprehensive multi-omics analysis integrating data from TCGA, GEO, CPTAC, and the Human Protein Atlas to investigate the expression, prognostic value, and immune relevance of EXOSC genes in HNSC. We conducted differential expression analysis, survival analysis (OS, DSS, PFI), ROC curve evaluation, and clinicopathological correlation studies. Genetic alterations were examined using cBioPortal. Gene co-expression and enrichment analyses were used to elucidate potential molecular functions, and a protein–protein interaction (PPI) network was constructed via GeneMANIA. Immune infiltration, immune checkpoint correlations, and RNA modification associations were assessed using ssGSEA, Spearman correlation, and RNA modification databases. Experimental validation was performed by qRT-PCR in HNSC and normal cell lines. All EXOSC family members were significantly upregulated in HNSC tissues and cell lines. ROC analysis demonstrated favorable diagnostic potential, particularly for EXOSC2 (AUC = 0.910). Elevated expression of EXOSC2, EXOSC3, EXOSC8, and EXOSC9 was significantly associated with poor OS, DSS, and PFI. High expression of EXOSC2, EXOSC4, EXOSC5, and EXOSC9 correlated with advanced clinical stage, lymphovascular invasion, and poor therapeutic outcomes. cBioPortal analysis revealed EXOSC4 had the highest genetic alteration frequency (8%), primarily due to amplification. Immune infiltration analysis showed EXOSC gene expression was significantly correlated with various immune cell populations and immune checkpoint molecules, especially EXOSC3, EXOSC9, and EXOSC10. Functional enrichment and PPI network analyses indicated that EXOSC family genes participate in RNA metabolism, exoribonuclease activity, and immune-related pathways. A prognostic risk model based on EXOSC co-expressed genes demonstrated strong predictive performance for patient survival. Our study reveals that EXOSC family genes are significantly dysregulated in HNSC and are associated with tumor progression, prognosis, immune microenvironment modulation, and RNA modification. These findings highlight the potential of EXOSC members as novel diagnostic and prognostic biomarkers and suggest their relevance as therapeutic targets in HNSC.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-15758-3.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> EXOSC family, Head and neck squamous cell carcinoma (HNSC), Prognosis, Immune infiltration, Bioinformatics, Tumor microenvironment</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Head and neck cancer, Databases</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Head and neck squamous cell carcinoma (HNSC) is one of the most common malignant tumors globally, originating primarily from the mucosal epithelium of the oral cavity, pharynx, and larynx<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Its high incidence and poor prognosis make it a significant public health concern<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. It is a highly heterogeneous malignancy, ranking among the top causes of cancer-related mortality globally. Despite advancements in surgical interventions, radiotherapy, and systemic therapies, the overall survival rate of HNSC patients remains unsatisfactory, largely due to late-stage diagnosis, therapeutic resistance, and tumor recurrence<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. The molecular heterogeneity of HNSC underscores the need for a deeper understanding of its genetic and epigenetic landscape to identify novel biomarkers and therapeutic targets. In this context, bioinformatics-driven analyses of cancer-associated gene families have emerged as a powerful approach to uncovering potential oncogenes or tumor suppressors that contribute to HNSC pathogenesis.</p>
<p id="Par3">Emerging evidence suggests that RNA degradation is a critical mechanism of post-transcriptional gene regulation, influencing cancer progression by modulating the stability and turnover of oncogenic and tumor-suppressive transcripts<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Among the key players in RNA metabolism, the exosome complex (EXOSC) is central to RNA degradation and processing, functioning in both the nucleus and cytoplasm<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. The EXOSC family, consisting of ten highly conserved subunits (EXOSC1-10), forms a multi-protein machinery responsible for the degradation of diverse RNA species, including aberrant and regulatory RNAs<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Beyond its canonical RNA degradation function, accumulating evidence suggests that EXOSC components contribute to oncogenesis. Several EXOSC subunits have been implicated in tumor development and progression across various malignancies<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. For instance, EXOSC1 has been shown to modulate DNA stability and enhance the sensitivity of renal carcinoma cells to poly(ADP-ribose) polymerase (PARP) inhibitors<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Similarly, EXOSC2 plays a crucial role in breast cancer metastasis<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, while EXOSC3 and EXOSC4 have been associated with colorectal tumorigenesis<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Additional studies indicate that EXOSC5<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, EXOSC8<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, EXOSC9<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, and EXOSC10<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> participate in cancer progression, but their specific contributions to HNSC remain largely unexplored. Given the functional significance of EXOSC genes in multiple cancer types, elucidating their role in HNSC may provide novel insights into tumor biology and potential therapeutic targets.</p>
<p id="Par4">In this study, we conducted a comprehensive bioinformatics analysis to investigate the expression profiles, prognostic value, and potential molecular functions of EXOSC family genes in HNSC. Using publicly available datasets, we explored the correlation between EXOSC expression and patient survival, tumor immune microenvironment characteristics, and key oncogenic pathways. Our findings provide new insights into the role of EXOSC genes in HNSC progression and highlight their potential as prognostic biomarkers and therapeutic targets for precision oncology.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Expression profile analysis, survival analysis, and clinicopathological features of EXOSC family genes in HNSC</h3>
<p id="Par5">Expression data for EXOSC1[ENSG00000171311.13], EXOSC2[ENSG00000130713.16], EXOSC3[ENSG00000107371.14], EXOSC4[ENSG00000178896.9], EXOSC5[ENSG00000077348.9], EXOSC6[ENSG00000223496.3], EXOSC7[ENSG00000075914.13], EXOSC8[ENSG00000120699.13], EXOSC9[ENSG00000123737.13] and EXOSC10[ENSG00000171824.14] were retrieved from the TCGA-HNSC (head and neck squamous cell carcinoma) dataset in the TCGA database (<a href="https://portal.gdc.cancer.gov" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.gdc.cancer.gov</a>). The RNA-seq data, processed using the STAR pipeline, were extracted in TPM (transcripts per million) format and subsequently transformed using log2(value + 1) for normalization. The visualization of EXOSC gene expression data was carried out using the “ggplot2” package in R. Additionally, the correlation between the mRNA expression levels of EXOSC family genes and clinicopathological characteristics in HNSC was examined through R’s “ggplot2” package. A Kaplan-Meier survival analysis was conducted to establish a Cox proportional hazards regression model, which was used to assess overall survival (OS), disease-specific survival (DSS), and progression-free interval (PFI) based on EXOSC gene expression levels<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. The “Survival” package in R software was used for this purpose, with hazard ratios (HR) and p-values calculated and documented. Results were visualized through forest plots, which were created using ggplot2. To evaluate the effectiveness of binary classification models, receiver operating characteristic (ROC) curve analysis was performed<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. The ROC curve is a graphical representation that compares sensitivity and specificity, while the area under the curve (AUC) serves as a measure of classification accuracy, with values approaching 1 indicating higher predictive power. The “pROC” package in R was utilized to conduct the ROC curve analysis. Protein expression data were obtained from Clinical Proteomic Tumor Analysis Consortium (CPTAC) database, and subsequently analyzed using the UALCAN platform. And, representative immunohistochemistry (IHC) images of EXOSC family protein distribution in human HNSC tumor specimens and matched normal tissues were obtained from the publicly available Human Protein Atlas database (HPA).</p></section><section id="Sec4"><h3 class="pmc_sec_title">EXOSC family genes expressions vlidation by GEO database</h3>
<p id="Par6">To further confirm the expression pattern of EXOSC family genes in HNSC, we employed datasets <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142083" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE142083</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178537" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE178537</a> obtained from the GEO database (<a href="https://www.ncbi.nlm.nih.gov/geo" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo</a>). The dataset encompass genes expression profiles derived from both normal and HNSC tissues.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Cell culture</h3>
<p id="Par7">The human oral keratinocyte (HOK) cell line and HN4 cell line were procured from the Shanghai Institute of Life Science Cell Bank Center (Shanghai, China). Both cell lines were maintained in Dulbecco’s Modified Eagle Medium (DMEM) supplemented with 10% fetal bovine serum (FBS) and 1% penicillin-streptomycin (100 U/mL penicillin and 100 µg/mL streptomycin) at 37 °C in a humidified atmosphere containing 5% CO₂. Cells in the exponential growth phase were harvested for subsequent experiments.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Quantitative real-time polymerase chain reaction (qRT-PCR)</h3>
<p id="Par8">The primer mix was validated on an ABI 7900HT system using the following protocol: initial enzyme activation at 50 °C for 2 min, followed by polymerase activation at 95 °C for 10 min. Amplification was carried out over 40 cycles, each consisting of denaturation at 95 °C for 15 s and annealing at 60 °C for 1 min. Finally, a melting curve analysis was performed with sequential steps at 95 °C for 15 s, 60 °C for 15 s, and 95 °C for 15 s to verify reaction specificity. EXOSC1-10 expression were quantified using the 2<sup>−ΔΔCt</sup> method. Primer sequences were: EXOSC1 forward: 5’-CACGGCTACATCTTTTCGTCGC-3’, reverse: 5’-CTCCCACATCTGGCAGTAACTG-3’; EXOSC2 forward: 5’-CTCGCTGGTAACTCAGAGGATG-3’, reverse: 5’-CTCCTGTTCCAAAAGCCTCTGG-3’; EXOSC3 forward: 5’-GGTGTCATTGGACAGGATGGTC-3’, reverse: 5’-TCTCCAGTGGATAGAGTTTTCCC-3’; EXOSC4 forward: 5’-GGACCGTAAGTCCTGTGAGATG-3’, reverse: 5’-AGGTCCCACCATCTGCCTGTAG-3’; EXOSC5 forward: 5’-GAACGGAAGCTGCTGATGTCCA-3’, reverse: 5’-GGTAGAAACGGAAGACGTGTTGC-3’; EXOSC6 forward: 5’-AGGGCTGCCAGCGCCTCTAC-3’, reverse: 5’-CTTGCGTCCGTAGTTGCTCAGG-3’; EXOSC7 forward: 5’-GTGAGGACTACCGATGTGTCGA-3’, reverse: 5’-CTGCTTTCACTCCCACCAAGAT-3’; EXOSC8 forward: 5’-GAACTGCCGTCCTGATGGAAGA-3’, reverse: 5’-GGCATCTGTTGATGGTGCTGCA-3’; EXOSC9 forward: 5’-CTGCCAGCATTGCTGCAATCGT-3’, reverse: 5’-GGTACAGGATCACGCTCTTCAG-3’; EXOSC10 forward: 5’-CTCTTTGGACCTCACGACTGCT-3’, reverse: 5’-AAGAGGCTCGCCTGCTTCTGAA-3’.</p></section><section id="Sec7"><h3 class="pmc_sec_title">cBioPortal analysis of EXOSC family genes in HNSC</h3>
<p id="Par9">Genetic variations in EXOSC family genes within the TCGA dataset were investigated using cBioPortal (<a href="http://www.cbioportal.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.cbioportal.org/</a>)<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. This analysis incorporated all EXOSC family members (EXOSC1-EXOSC10) from the TCGA pan-cancer atlas cohort for HNSC. By entering the names of EXOSC genes into the platform’s search tool, mutation data were retrieved from both the “OncoPrint” and “Cancer Types Summary” modules within cBioPortal.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Protein–Protein interaction (PPI) network construction of EXOSC family genes in HNSC</h3>
<p id="Par10">To explore the functional interactions of EXOSC family proteins, we utilized the GeneMANIA platform (<a href="http://genemania.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://genemania.org</a>), a comprehensive bioinformatics tool that integrates diverse datasets covering 166,691 genes and approximately 660 million documented interactions across nine model organisms. GeneMANIA facilitates the prediction of gene function based on co-expression, co-localization, physical interactions, shared protein domains, and other association data. In this study, we used GeneMANIA to predict potential protein interaction partners of the EXOSC family in Homo sapiens and constructed a PPI network to elucidate their biological associations.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Immune-related analysis of EXOSC family genes in HNSC</h3>
<p id="Par11">To assess immune cell infiltration across 24 immune cell types in HNSC tissue samples, the ssGSEA algorithm was applied<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Correlation analyses were performed to examine associations between EXOSC gene expression levels, immune cells<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, and immune checkpoint genes<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> using Spearman correlation analysis. The relationships were illustrated using the “ggplot2” and “pheatmap” packages. Additionally, to compare immune cell infiltration in glioma samples with varying levels of EXOSC family genes expression, the Wilcoxon rank-sum test was employed.</p></section><section id="Sec10"><h3 class="pmc_sec_title">RNA modification analysis of EXOSC family genes in HNSC</h3>
<p id="Par12">Expression data for EXOSC family genes and 44 marker genes associated with three major RNA modifications—m1A, m5C, and m6A—were retrieved from UCSC XENA<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Spearman correlation analysis was used to identify potential associations between EXOSC family genes expression and RNA modification-related genes using R (v4.0) and the “corrplot” package (v0.90). Correlation matrices and heatmaps were generated using “ggplot2” for visual interpretation.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Gene set enrichment analysis (GSEA) of EXOSC family genes in HNSC</h3>
<p id="Par13">GSEA was conducted using R software (version 3.6.3) to explore signaling pathways potentially associated with the expression of EXOSC family genes in HNSC. Patients were stratified into high and low expression groups based on the median expression value. The expression levels of EXOSC family genes were used as the phenotype label. Each analysis involved 1,000 permutations. The “c2.cp.v7.2.symbols.gmt” gene set from the Molecular Signatures Database (MSigDB) served as the reference. Pathways were considered significantly enriched if they met the criteria of false discovery rate (FDR) &lt; 0.25, P-value &lt; 0.05, and a normalized enrichment score (NES) &gt; 1. Pathways with the highest NES values were prioritized for downstream interpretation.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Functional and pathway analysis</h3>
<p id="Par14">HNSC patient gene expression data in TPM format were extracted from the TCGA database. Co-expressed genes were identified using Spearman correlation coefficients (<em>P</em> &lt; 0.01, |R| &gt; 0.3). A Venn diagram was generated to depict the overlapping relationships among EXOSC1-EXOSC10. To investigate the biological pathways and functional roles associated with EXOSC family genes, the “clusterProfiler” package was utilized for Kyoto Encyclopedia of Genes and Genomes (KEGG)<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> and Gene Ontology (GO)<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> enrichment analyses. GO analysis was further divided into three categories: biological process (BP), molecular function (MF), and cellular component (CC).</p></section><section id="Sec13"><h3 class="pmc_sec_title">Identification of prognostic genes</h3>
<p id="Par15">To systematically assess the prognostic relevance of genes co-expressed with EXOSC family members in head and neck squamous cell carcinoma (HNSC), we performed univariate Cox proportional hazards regression analysis using the “survival” R package, with a significance threshold set at <em>P</em> &lt; 0.05. To account for potential multicollinearity among variables and improve model interpretability, we employed the least absolute shrinkage and selection operator (LASSO)-penalized Cox regression using the “glmnet” (version 4.1.7) R package. Subsequently, a nomogram was constructed with the “rms” R package to predict individual survival probabilities.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Construction of a prognostic model based on EXOSC family co-expression genes</h3>
<p id="Par16">A prognostic risk score model was established using centralized and standardized mRNA expression data from the HNSC training cohort. The risk score was defined as:</p>
<p id="Par17">Risk score = Risk score=<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/e4f1390b82e6/d33e408.gif" loading="lazy" id="d33e408" alt="Inline graphic"></span>, where X represents the LASSO-derived coefficient for each gene and Y denotes the expression level of the corresponding EXOSC family co-expressed gene. Patients were stratified into high-risk and low-risk groups based on the median risk score. Overall survival (OS) between the two groups was compared to evaluate prognostic stratification. The model’s predictive performance was assessed using time-dependent receiver operating characteristic (ROC) curves generated with the “timeROC” R package. Clinical covariates included in the analysis comprised age, sex, histological grade, clinical stage, and TNM classification.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par18">For data following a normal distribution, results were represented as means ± standard deviation. Comparisons of EXOSC family genes expression levels between normal and HNSC tissues were conducted using Wilcoxon rank-sum and Wilcoxon signed-rank tests in SPSS 23.0. Associations between EXOSC family genes expression and clinicopathological features were examined using the chi-square test. Statistical significance was defined as *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001.</p></section></section><section id="Sec16"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec17"><h3 class="pmc_sec_title">Expressions and ROC of EXOSC family genes in HNSC</h3>
<p id="Par19">Analysis of data from the UCSC XENA database revealed differential expression patterns of EXOSC family members in head and neck squamous cell carcinoma (HNSC) based on 44 normal and 504 tumor samples (sample sizes detailed in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). As shown in Fig. <a href="#Fig1" class="usa-link">1</a>A, all EXOSC family genes (EXOSC1-10) were significantly upregulated in HNSC tissues compared to normal tissues (<em>P</em> &lt; 0.001). Figure <a href="#Fig1" class="usa-link">1</a>B shows that EXOSC1-10 expression levels were significantly higher in 43 HNSC tumors than in paired normal samples (TCGA data). To evaluate their prognostic potential, receiver operating characteristic (ROC) curve analysis was conducted (sample sizes detailed in Supplementary Table <a href="#MOESM2" class="usa-link">2</a>). The results indicated high predictive accuracy for HNSC, with area under the curve (AUC) values of 0.862 for EXOSC1, 0.910 for EXOSC2, 0.873 for EXOSC3, 0.882 for EXOSC4, 0.798 for EXOSC5, 0.762 for EXOSC6, 0.695 for EXOSC7, 0.809 for EXOSC8, 0.870 for EXOSC9, and 0.794 for EXOSC10 (Fig. <a href="#Fig1" class="usa-link">1</a>C). As shown in Fig. <a href="#Fig1" class="usa-link">1</a>D-E, to further validate our findings, we analyzed the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142083" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE142083</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178537" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE178537</a> datasets from the GEO database, which revealed that the expression levels of EXOSC1-10 were significantly elevated in HNSC tissues compared to normal controls (all <em>P</em> &lt; 0.05). As shown in Fig. <a href="#Fig1" class="usa-link">1</a>F, we used qRT-PCR to explore the gene levels of EXOSC family, we found that the expression levels of EXOSC1-10 were significantly higher in HNSC cell (HN4) than in normal cell (HOK) (all <em>P</em> &lt; 0.05). Using data from the CPTAC, we compared the protein expression levels of EXOSC1-10 between normal tissues and primary tumor tissues in HNSC. Our analysis revealed significantly elevated total protein levels of EXOSC3-10 in primary HNSC tissues compared to normal controls (<em>P</em> &lt; 0.001). In contrast, the expression levels of EXOSC1 and EXOSC2 showed no statistically significant differences between the two groups (Fig. <a href="#Fig1" class="usa-link">1</a>G). Furthermore, immunohistochemistry (IHC) data obtained from HPA database revealed the expression profiles of the EXOSC family genes in clinical samples, as illustrated in Fig. <a href="#Fig2" class="usa-link">2</a>A-I. Notably, the protein levels of EXOSC1, EXOSC2, EXOSC3, EXOSC4, EXOSC5, EXOSC7, EXOSC8, EXOSC9, and EXOSC10 were significantly upregulated in HNSC tissues compared to normal tissues.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365086_41598_2025_15758_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/4910c89db131/41598_2025_15758_Fig1_HTML.jpg" loading="lazy" id="d33e514" height="494" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Expressions and receiver operating characteristic (ROC) curve analysis of EXOSC family genes in Head and Neck Squamous Cell Carcinoma (HNSC). (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001). (<strong>A</strong>) Differential mRNA expression levels of EXOSC1-10 between HNSC tumors and normal tissues. (<strong>B</strong>) EXOSC1-10 expression levels between 43 HNSC tumors and paired normal samples. (<strong>C</strong>) Receiver operating characteristic (ROC) curves assessing the diagnostic performance of EXOSC family genes in distinguishing HNSC from normal tissue. <strong>(D</strong>) Validation of elevated EXOSC family gene expression in HNSC tissues compared to normal controls using the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142083" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE142083</a> dataset. (<strong>E</strong>) Validation of elevated EXOSC family gene expression in HNSC tissues using the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178537" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE178537</a> dataset. (<strong>F</strong>) Validation of elevated EXOSC family gene expression in HOK and HN4 cells by qRT-PCR. (<strong>G</strong>) Using data from the CPTAC, we compared the protein expression levels of EXOSC1-10 between normal tissues and primary tumor tissues in HNSC.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365086_41598_2025_15758_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/6f50bc43b654/41598_2025_15758_Fig2_HTML.jpg" loading="lazy" id="d33e553" height="388" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Validation of EXOSC family protein expressions. Immunohistochemical (IHC) analysis of EXOSC family expressions in HNSC and normal tissues was performed using data from the HPA data. (<strong>A</strong>) Representative IHC images from the HPA demonstrate EXOSC1 expression in HNSC primary tumor and matched normal tissue. (Antibody HPA038370, and 10X). (<strong>B</strong>) Representative IHC images from the HPA demonstrate EXOSC2 expression in HNSC primary tumor and matched normal tissue. (Antibody HPA021790, and 10X). (<strong>C</strong>) Representative IHC images from the HPA demonstrate EXOSC3 expression in HNSC primary tumor and matched normal tissue. (Antibody HPA020485, and 10X). (<strong>D</strong>) Representative IHC images from the HPA demonstrate EXOSC4 expression in HNSC primary tumor and matched normal tissue. (Antibody HPA024792, and 10X). (<strong>E</strong>) Representative IHC images from the HPA demonstrate EXOSC5 expression in HNSC primary tumor and matched normal tissue. (Antibody HPA053150, and 10X). (<strong>F</strong>) Representative IHC images from the HPA demonstrate EXOSC7 expression in HNSC primary tumor and matched normal tissue. (Antibody HPA057980, and 10X). <strong>(G</strong>) Representative IHC images from the HPA demonstrate EXOSC8 expression in HNSC primary tumor and matched normal tissue. (Antibody CAB034240, and 10X). <strong>(H</strong>) Representative IHC images from the HPA demonstrate EXOSC9 expression in HNSC primary tumor and matched normal tissue. (Antibody HPA048257, and 10X). (<strong>I</strong>) Representative IHC images from the HPA demonstrate EXOSC10 expression in HNSC primary tumor and matched normal tissue. (Antibody HPA028470, and 10X).</p></figcaption></figure></section><section id="Sec18"><h3 class="pmc_sec_title">Survival analysis of EXOSC family genes in HNSC</h3>
<p id="Par22">Kaplan-Meier survival analysis using data from The Cancer Genome Atlas (TCGA) was performed to assess the association between EXOSC family gene expression and patient survival outcomes. The study evaluated overall survival (OS), disease-specific survival (DSS), and progression-free interval (PFI) in HNSC patients (Fig. <a href="#Fig3" class="usa-link">3</a>). As shown in Fig. <a href="#Fig3" class="usa-link">3</a>A, high expression of EXOSC2 (HR = 1.33, <em>P</em> = 0.036) and EXOSC3 (HR = 1.32, <em>P</em> = 0.040) was significantly associated with increased mortality risk, suggesting that elevated levels of these genes may predict poorer OS. Although other EXOSC family members (EXOSC1, EXOSC4–EXOSC10) exhibited a trend toward increased mortality risk, the associations were not statistically significant (<em>P</em> &gt; 0.05). In the DSS analysis (Fig. <a href="#Fig3" class="usa-link">3</a>B), high expression of EXOSC2 (HR = 1.42, <em>P</em> = 0.049), EXOSC3 (HR = 1.56, <em>P</em> = 0.014), EXOSC8 (HR = 1.75, <em>P</em> = 0.002), and EXOSC9 (HR = 1.49, <em>P</em> = 0.025) was significantly correlated with increased disease-specific mortality, indicating their potential as prognostic markers for poorer DSS. Other EXOSC genes (EXOSC1, EXOSC4–EXOSC7, and EXOSC10) showed a similar trend, but the associations were not statistically significant (<em>P</em> &gt; 0.05). For PFI (Fig. <a href="#Fig3" class="usa-link">3</a>C), high expression of EXOSC2 (HR = 1.41, <em>P</em> = 0.019), EXOSC4 (HR = 1.41, <em>P</em> = 0.017), EXOSC8 (HR = 1.71, <em>P</em> &lt; 0.001), and EXOSC9 (HR = 1.41, <em>P</em> = 0.018) was significantly associated with an increased risk of disease progression, suggesting that elevated expression of these genes may predict shorter PFI. While other EXOSC family members (EXOSC1, EXOSC3, EXOSC5–EXOSC7, and EXOSC10) also exhibited a trend toward increased progression risk, these associations were not statistically significant (<em>P</em> &gt; 0.05).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365086_41598_2025_15758_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/9ea28c6b3931/41598_2025_15758_Fig3_HTML.jpg" loading="lazy" id="d33e633" height="126" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Prognostic significance of EXOSC family genes in HNSC. Survival analyses for EXOSC family genes in HNSC patients. Forest plots (<strong>A-C</strong>) showing overall survival (OS), disease specific survival (DSS) and progress free interval (PFI) stratified by high and low gene expression. (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001).</p></figcaption></figure></section><section id="Sec19"><h3 class="pmc_sec_title">Correlation between expressions of EXOSC family and clinicopathological parameters of HNSC</h3>
<p id="Par24">The relationship between EXOSC family gene expression and clinicopathological factors in HNSC was systematically analyzed in a cohort of 504 patients, with 252 individuals in both the high- and low-expression groups for each gene (Supplementary Table <a href="#MOESM38" class="usa-link">9</a>). EXOSC2 expression was significantly associated with multiple pathological and clinical parameters. A higher proportion of patients in the high-expression group presented with advanced pathological T stage (T4; 23.4% vs. 15%, <em>P</em> = 0.00056), pathological stage IV (33.3% vs. 26.8%, <em>P</em> = 0.0054), and clinical N stage (N2/N3; 19.7% vs. 13.9%, <em>P</em> = 0.012). Additionally, its expression correlated with clinical stage IV (30.6% vs. 25.1%, <em>P</em> = 0.0469), male gender (39.7% vs. 33.7%, <em>P</em> = 0.0025), higher histologic grade (G3/G4; 14.5% vs. 10.5%, <em>P</em> = 0.000148), and lymphovascular invasion (21.3% vs. 14.3%, <em>P</em> = 0.00675). Similarly, EXOSC5 was significantly linked to pathological stage IV (31.7% vs. 28.4%, <em>P</em> = 0.014), clinical stage IV (30.8% vs. 24.9%, <em>P</em> = 0.0186), advanced clinical N stage (N2/N3; 19.5% vs. 14.1%, <em>P</em> = 0.011), male gender (40.5% vs. 32.9%, <em>P</em> = 0.00013), higher histologic grade (17.4% vs. 7.6%, <em>P</em> = 5.998e-07), lymphovascular invasion (21.1% vs. 14.6%, <em>P</em> = 0.0202), and prior radiation therapy (35.7% vs. 29.2%, <em>P</em> = 0.0248). EXOSC4 and EXOSC9 were associated with primary therapy outcomes, where high-expression groups exhibited a significantly greater proportion of patients with stable disease, partial response, or progressive disease (EXOSC4: 8.8% vs. 3.8%, <em>P</em> = 0.0024; EXOSC9: 8.6% vs. 4.1%, <em>P</em> = 0.0162). Additionally, EXOSC4 was linked to advanced pathological N stage (N2/N3; 24.8% vs. 17.5%, <em>P</em> = 0.0045) and lymph node dissection (43.9% vs. 38.1%, <em>P</em> = 0.00201). Other notable associations included EXOSC1, EXOSC7, and EXOSC8, which showed significant correlations with male gender (EXOSC1: 38.9% vs. 34.5%, <em>P</em> = 0.0265; EXOSC7: 39.1% vs. 34.3%, <em>P</em> = 0.0155; EXOSC8: 19.7% vs. 13.9%, <em>P</em> = 0.0285), histologic grade (EXOSC1: 16.3% vs. 8.7%, <em>P</em> = 2.11e-05; EXOSC7: 15.1% vs. 9.9%, <em>P</em> = 0.00127; EXOSC8: 15.5% vs. 9.5%, <em>P</em> = 0.00366), and lymphovascular invasion (EXOSC1: 20.2% vs. 15.5%, <em>P</em> = 0.0491; EXOSC7: 22.2% vs. 13.5%, <em>P</em> = 0.00218). Furthermore, EXOSC7 was associated with prior radiation therapy (35.7% vs. 29.2%, <em>P</em> = 0.0123).Conversely, no significant correlations were observed between EXOSC family gene expression and clinical T stage, M stage, race, alcohol history, smoking status, or age.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Mutation and correlation analysis of EXOSC family genes in HNSC</h3>
<p id="Par25">Epigenetic modifications play a critical role in the early stages of malignancy. Using the cBioPortal online tool, we analyzed genetic alterations in EXOSC family genes in HNSC, including mutations and copy number variations. The specific genetic alterations and their respective frequencies are illustrated in Fig. <a href="#Fig4" class="usa-link">4</a>A. Among the EXOSC family members, EXOSC4 exhibited the highest alteration frequency (8%), with amplifications being the predominant alteration, suggesting that this gene is the most frequently altered in the analyzed samples. This was followed by EXOSC3 (3%), which displayed amplifications, deep deletions, and missense mutations. Other genes exhibited lower alteration frequencies: EXOSC2 (1.2%), EXOSC10 (1%), EXOSC5 (0.8%), EXOSC9 (0.8%), EXOSC1 (0.6%), EXOSC7 (0.4%), and EXOSC8 (0.4%), while EXOSC6 had the lowest alteration frequency (0.2%). These findings suggest that alterations in EXOSC4 may have a more significant impact on tumor initiation and progression. The broad distribution of EXOSC4 alterations further indicates their prevalence across multiple samples. In contrast, other genes, such as EXOSC1, EXOSC2, and EXOSC3, exhibited sparse alteration distributions, suggesting that these changes occur in only a small subset of cases. Overall, genetic alterations in EXOSC family genes were observed in 15.32% of the 496 HNSC patients, including structural variants, amplifications, and deep deletions. These alterations may significantly influence tumor initiation and progression, particularly as amplifications and deep deletions can lead to oncogene overexpression and loss of tumor suppressor gene function, respectively (Fig. <a href="#Fig4" class="usa-link">4</a>B). To further investigate the relationships among EXOSC family members in HNSC, Pearson correlation analysis was performed. As shown in Fig. <a href="#Fig4" class="usa-link">4</a>C, significant positive correlations were observed among all EXOSC family members (<em>P</em> &lt; 0.001), except for EXOSC4 and EXOSC10 and EXOSC7 and EXOSC10, which did not show statistically significant associations.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365086_41598_2025_15758_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/ab3796b5b5f9/41598_2025_15758_Fig4_HTML.jpg" loading="lazy" id="d33e771" height="738" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genetic alterations and expression correlations of EXOSC family genes in HNSC. (<strong>A-B</strong>) Genomic alteration profiles of EXOSC family genes in HNSC. (<strong>C</strong>) Co-expression analysis of <strong>EXOSC</strong> family genes in HNSC tissues. (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001).</p></figcaption></figure></section><section id="Sec21"><h3 class="pmc_sec_title">Protein–protein interaction (PPI) network of EXOSC family genes in HNSC</h3>
<p id="Par27">To systematically identify potential functional partners of the EXOSC family in humans, we constructed a PPI network using GeneMANIA. The analysis identified 20 significant interacting proteins, including MPHOSPH6, DIS3, C1D, DIS3L, PNPT1, XRN1, KHSRP, ZFP36L1, ZFP36, SUPT6H, MTREX, PARN, AURKAIP1, TNPO1, HBS1L, DCP1A, DCP2, EXD3, EXD1, and DDX47. Among these, MPHOSPH6 and DIS3 exhibited the highest interaction scores (Fig. <a href="#Fig5" class="usa-link">5</a>). Functional enrichment analysis revealed that the EXOSC family proteins and their interactors are primarily involved in biological processes and molecular functions such as the exoribonuclease complex, exosome (RNase complex), exonuclease activity, ncRNA catabolic process, regulation of mRNA catabolic process, RNA stability, and nuclease activity. These findings suggest that EXOSC family members may play a critical role in RNA degradation and post-transcriptional regulation within the tumor microenvironment.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365086_41598_2025_15758_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/c37bd3542d2d/41598_2025_15758_Fig5_HTML.jpg" loading="lazy" id="d33e788" height="1181" width="794" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Protein-protein interaction (PPI) network of the EXOSC family genes by GeneMANIA.</p></figcaption></figure></section><section id="Sec22"><h3 class="pmc_sec_title">Correlation between EXOSC family genes expressions and immune cell infiltration in HNSC</h3>
<p id="Par29">As shown in Fig. <a href="#Fig6" class="usa-link">6</a>, the relationship between the relative abundance of 24 immune cell types and EXOSC family gene expression in HNSC was assessed using the ssGSEA algorithm (sample sizes provided in Supplementary Table 3.1–3.4). EXOSC1 expression exhibited a significant positive correlation with NK CD56bright cells, NK cells, Th2 cells, and Tgd, while showing a negative correlation with TFH, Tcm, DC, iDC, B cells, neutrophils, Th17 cells, and mast cells (Fig. <a href="#Fig6" class="usa-link">6</a>A). EXOSC2 expression was positively correlated with Th2 cells and NK CD56bright cells but negatively correlated with NK cells, CD8 + T cells, cytotoxic cells, NK CD56dim cells, Th1 cells, pDC, T cells, eosinophils, macrophages, B cells, Treg, TFH, DC, neutrophils, Th17 cells, iDC, and mast cells (Fig. <a href="#Fig6" class="usa-link">6</a>B). EXOSC3 expression showed a significant positive correlation with aDC, Th2 cells, T helper cells, and cytotoxic cells, whereas it was negatively correlated with B cells, Th17 cells, Tem, pDC, NK cells, macrophages, eosinophils, DC, neutrophils, iDC, and mast cells (Fig. <a href="#Fig6" class="usa-link">6</a>C).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365086_41598_2025_15758_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/05d8dfa7c0c6/41598_2025_15758_Fig6_HTML.jpg" loading="lazy" id="d33e824" height="238" width="669" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Associations between EXOSC family genes expressions and tumor immune infiltration in HNSC. Bar plots showing correlations between EXOSC1-10 (A-J) expression and the infiltration of 24 immune cell types in HNSC. (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001).</p></figcaption></figure><p id="Par31">EXOSC4 expression was positively correlated with NK CD56bright cells but negatively correlated with NK CD56dim cells, Th1 cells, cytotoxic cells, CD8 + T cells, aDC, iDC, DC, Th2 cells, TFH, neutrophils, eosinophils, T cells, Tem, Treg, Th17 cells, B cells, mast cells, T helper cells, and Tcm (Fig. <a href="#Fig6" class="usa-link">6</a>D). EXOSC5 expression was positively associated with NK CD56bright cells and cytotoxic cells but negatively correlated with macrophages, Th17 cells, Th1 cells, Tcm, DC, Tgd, mast cells, and neutrophils (Fig. <a href="#Fig6" class="usa-link">6</a>E). EXOSC6 expression was positively correlated with NK CD56bright cells but negatively associated with T helper cells, Th1 cells, iDC, mast cells, neutrophils, CD8 + T cells, pDC, TFH, DC, aDC, NK CD56dim cells, Treg, cytotoxic cells, B cells, Th17 cells, and T cells (Fig. <a href="#Fig6" class="usa-link">6</a>F). EXOSC7 expression showed a significant positive correlation with NK CD56bright cells but was negatively correlated with T helper cells, cytotoxic cells, NK cells, Th2 cells, Tem, macrophages, B cells, T cells, aDC, NK CD56dim cells, Th1 cells, Tcm, TFH, Treg, DC, Th17 cells, eosinophils, iDC, neutrophils, and mast cells (Fig. <a href="#Fig6" class="usa-link">6</a>G). EXOSC8 expression was positively associated with Th2 cells and T helper cells but negatively correlated with NK cells, pDC, macrophages, Tem, B cells, TFH, eosinophils, Th17 cells, DC, iDC, neutrophils, and mast cells (Fig. <a href="#Fig6" class="usa-link">6</a>H). EXOSC9 expression was significantly positively correlated with Th2 cells, T helper cells, NK CD56bright cells, and CD8 + T cells, while showing a negative correlation with B cells, macrophages, TFH, eosinophils, DC, Th17 cells, neutrophils, iDC, and mast cells (Fig. <a href="#Fig6" class="usa-link">6</a>I). EXOSC10 expression exhibited a significant positive correlation with Th2 cells, T helper cells, Tcm, aDC, Th1 cells, Treg, NK cells, Tem, and Tgd, while showing a negative correlation with B cells, NK CD56bright cells, DC, mast cells, and pDC (Fig. <a href="#Fig6" class="usa-link">6</a>J).</p></section><section id="Sec23"><h3 class="pmc_sec_title">Association between EXOSC family genes expressions and immune checkpoint genes in HNSC</h3>
<p id="Par32">To further investigate the association between EXOSC family genes and immune regulation in HNSC, we conducted a comprehensive correlation analysis between EXOSC gene expression levels and immune checkpoint genes. The results of this analysis are presented in Supplementary Table <a href="#MOESM13" class="usa-link">4</a>. As shown in Fig. <a href="#Fig7" class="usa-link">7</a>, EXOSC1 expression was significantly positively correlated with CTLA4, ENTPD1, HAVCR2, IDO1, LAG3, LGALS9, TGFB1, TNFRSF18, TNFRSF4, and TNFRSF14, while it was negatively correlated with CD244 and KDR. EXOSC2 exhibited a significant positive correlation with ADORA2A, LGALS9, TGFB1, TNFRSF18, and TNFRSF14, whereas it showed a negative correlation with CD244. EXOSC3 demonstrated a consistent positive correlation with 14 immune checkpoint genes, including CD274, CD96, CTLA4, HAVCR2, ICOS, IDO1, KIR2DL1, KIR2DL3, LAG3, LGALS9, PDCD1, TGFBR1, TIGIT, and TNFRSF14, while showing a negative correlation with KDR and VSIR. EXOSC4 exhibited a significant negative correlation with 21 checkpoint genes, including ADORA2A, BTLA, CD160, CD244, CD27, CD274, CD96, CSF1R, ENTPD1, HAVCR2, ICOS, IDO1, IL10, KDR, KIR2DL3, PDCD1, SIGLEC15, TGFBR1, TIGIT, TNFRSF9, and VSIR. In contrast, it showed a positive correlation with TGFB1 and TNFRSF18. EXOSC5 was positively correlated with 13 checkpoint genes, including ADORA2A, CD96, CTLA4, ENTPD1, HAVCR2, IDO1, KIR2DL1, LAG3, LGALS9, PDCD1, TNFRSF18, TNFRSF4, and TNFRSF14, while showing a negative correlation with CD274, IL10, KDR, TGFB1, and TGFBR1. EXOSC6 displayed a negative correlation with 16 checkpoint genes, including BTLA, CD244, CD27, CD274, CD96, CSF1R, CTLA4, ICOS, IDO1, KIR2DL1, KIR2DL3, LAG3, PDCD1, TIGIT, TNFRSF9, and VSIR, while exhibiting a positive correlation with SIGLEC15, TGFB1, TGFBR1, and TNFRSF18. EXOSC7 was negatively correlated with 13 checkpoint genes, including BTLA, CD160, CD244, CD27, CSF1R, ICOS, IL10, KDR, TGFB1, TGFBR1, TIGIT, TNFRSF9, and VSIR, whereas it showed a positive correlation with LGALS9, TNFRSF18, and TNFRSF14. EXOSC8 exhibited a positive correlation with 10 checkpoint genes, including CD160, CD274, CTLA4, HAVCR2, IDO1, LAG3, LGALS9, PDCD1, TGFBR1, and TNFRSF14, while also showing a positive correlation with KDR. EXOSC9 was positively correlated with 20 checkpoint genes, including ADORA2A, CD160, CD274, CD96, CTLA4, HAVCR2, ENTPD1, ICOS, IDO1, KIR2DL1, KIR2DL3, LAG3, LGALS9, PDCD1, TIGIT, TNFRSF18, TNFRSF4, TNFRSF9, and TNFRSF14. EXOSC10 exhibited a strong positive correlation with 25 immune checkpoint genes, including ADORA2A, BTLA, CD160, CD27, CD274, CD96, CSF1R, CTLA4, HAVCR2, ENTPD1, ICOS, IDO1, IL10, KDR, KIR2DL3, LAG3, LGALS9, PDCD1, SIGLEC15, TGFB1, TGFBR1, TIGIT, TNFRSF9, TNFRSF14, and VSIR.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365086_41598_2025_15758_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/e7fee2aff64b/41598_2025_15758_Fig7_HTML.jpg" loading="lazy" id="d33e878" height="266" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Correlations between EXOSC family genes expressions and immune checkpoint molecules in HNSC. (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001).</p></figcaption></figure></section><section id="Sec24"><h3 class="pmc_sec_title">Association between EXOSC family genes expression and RNA modification in HNSC</h3>
<p id="Par34">RNA modifications play a crucial role in cancer progression, including angiogenesis. In thymoma, the EXOSC family genes exhibits significant associations with RNA modification-related pathways. As detailed in Supplementary Table 5.1–5.3, correlation coefficients and p-values indicate strong positive associations between EXOSC family genes and key regulators of m1A, m5C, and m6A modifications. As shown in Fig. <a href="#Fig8" class="usa-link">8</a>A, EXOSC family genes exhibit significant positive correlations with most m1A modification-related genes, including TRMT61A, TRMT10C, TRMT6, YTHDF1, ALKBH1 and ALKBH3. However, EXOSC4 and EXOSC7 are negatively associated with YTHDC1 and YTHDF3, respectively. Similarly, in m5C modification pathways (Fig. <a href="#Fig8" class="usa-link">8</a>B), EXOSC family genes show significant positive correlations with NSUN5, NSUN2, and ALYREF, while EXOSC1, EXOSC4, and EXOSC7 are negatively associated with TET2, EXOSC4 is negatively correlated with NSUN7, DNMT1, DNMT3A and NSUN3, and EXOSC5 is negatively related to DNMT3B. Additionally, as depicted in Fig. <a href="#Fig8" class="usa-link">8</a>C, EXOSC family genes demonstrate strong positive correlations with most m6A modification-related genes. while EXOSC3 is negatively associated with FTO, EXOSC4 is negatively correlated with ZC3H13, CBLL1, FTO, YTHDC2 and FMR1, and EXOSC7 is negatively related to FTO and YTHDF3. These findings suggest that RNA modifications may directly or indirectly regulate EXOSC family genes expression, highlighting a potential mechanistic link between EXOSC family genes and RNA modification pathways in HNSC.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365086_41598_2025_15758_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/7584c85e1de5/41598_2025_15758_Fig8_HTML.jpg" loading="lazy" id="d33e911" height="157" width="669" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Associations between EXOSC family genes expressions and RNA modification regulators in HNSC. Correlation analyses for RNA modification patterns: m1A (A), m5C (B), and m6A (C). (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001).</p></figcaption></figure></section><section id="Sec25"><h3 class="pmc_sec_title">GSEA of EXOSC family genes in HNSC</h3>
<p id="Par36">To explore the functional roles of the EXOSC family genes in HNSC, we conducted a comprehensive GSEA. Based on the median expression levels, the HNSC cohort was stratified into high and low expression groups for each EXOSC family gene. Enrichment analysis revealed that high expression of EXOSC family genes was significantly associated with biological pathways such as keratinization, formation of the cornified envelope, and developmental biology, suggesting their potential involvement in epithelial differentiation and tumor development (Fig. <a href="#Fig9" class="usa-link">9</a>A-J). These pathways were considered significantly enriched based on the thresholds of false discovery rate (FDR) &lt; 0.25, P-value &lt; 0.05, and normalized enrichment score (NES) &gt; 1. Notably, pathways with the highest NES values were prioritized for downstream interpretation, providing insights into the biological relevance of EXOSC gene expression in HNSC.</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365086_41598_2025_15758_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/414c7ea6ee83/41598_2025_15758_Fig9_HTML.jpg" loading="lazy" id="d33e928" height="266" width="669" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gene set enrichment analysis (GSEA) of the EXOSC family genes in HNSC. Significantly enriched pathways were identified using the following thresholds: false discovery rate (FDR) &lt; 0.25, P-value &lt; 0.05, and normalized enrichment score (NES) &gt; 1. Reactome pathway enrichment analysis for EXOSC1-10 (A-J).</p></figcaption></figure></section><section id="Sec26"><h3 class="pmc_sec_title">Genes co-expressed with EXOSC family genes in HNSC and enrichment analysis of EXOSC family co-expression genes in HNSC</h3>
<p id="Par38">Analysis of TCGA transcriptome data revealed that the number of genes positively correlated with EXOSC1, EXOSC2, EXOSC3, EXOSC4, EXOSC5, EXOSC6, EXOSC7, EXOSC8, EXOSC9, and EXOSC10 was 8,902; 11,397; 6,755; 3,928; 10,182; 11,163; 8,841; 8,796; 11,678; and 11,657, respectively. In contrast, the number of negatively correlated genes for these EXOSC family members was 1,276; 801; 1,447; 5,496; 2,072; 625; 2,307; 1,099; 856; and 471 (sample sizes provided in Supplementary Table 6.1–6.10). The top 10 positively and negatively correlated genes for each EXOSC family member are depicted in Fig. <a href="#Fig10" class="usa-link">10</a>A–J (sample sizes provided in Supplementary Table 7.1–7.10). Additionally, a Venn diagram (Fig. <a href="#Fig11" class="usa-link">11</a>A) highlights 40 intersecting genes co-expressed with the EXOSC family (|R| &gt; 0.3, <em>P</em> &lt; 0.001) (sample sizes provided in Supplementary Table <a href="#MOESM37" class="usa-link">8</a>). To further explore the biological roles of EXOSC family genes in HNSC, we performed Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analyses on their co-expressed genes (Fig. <a href="#Fig11" class="usa-link">11</a>B). GO analysis categorized these genes into three functional domains: molecular functions (MF), cellular components (CC), and biological processes (BP). The most enriched BPs included ribonucleoprotein complex biogenesis, ribosome biogenesis, and ribonucleoprotein complex assembly. The primary CCs associated with EXOSC family co-expressed genes included the mitochondrial inner membrane, mitochondrial ribosome, and organellar ribosome. Key MFs influenced by these genes involved the structural constituent of ribosome, rRNA binding, and ATP-dependent activity acting on DNA. KEGG pathway analysis further indicated that EXOSC family co-expressed genes were primarily involved in the ribosome, spliceosome, and mRNA surveillance pathways.</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365086_41598_2025_15758_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/46ba6496587c/41598_2025_15758_Fig10_HTML.jpg" loading="lazy" id="d33e967" height="267" width="669" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Top 10 positively and top 10 negatively correlated genes with EXOSC family members in HNSC. Correlation networks for EXOSC1-10 (A-J). (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001)</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig11"><h4 class="obj_head">Fig. 11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365086_41598_2025_15758_Fig11_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/916a970ade12/41598_2025_15758_Fig11_HTML.jpg" loading="lazy" id="d33e997" height="331" width="669" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Integrative analysis of co-expression networks and functional enrichment of EXOSC family genes in HNSC. (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001). (<strong>A</strong>) Venn diagram identifying 40 consensus co-expressed genes associated with EXOSC1-10 (|R| &gt; 0.3, <em>P</em> &lt; 0.001). (<strong>B</strong>) Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analyses of the 40 intersecting genes.</p></figcaption></figure></section><section id="Sec27"><h3 class="pmc_sec_title">Development of a prognostic model and nomogram based on EXOSC family co-expression genes in HNSC</h3>
<p id="Par41">To identify prognostic biomarkers among EXOSC family co-expressed genes in HNSC, we first conducted univariate Cox regression analysis, which identified 12 candidate genes significantly associated with overall survival: AIMP1, BCCIP, CCT3, CCT4, ERAL1, EXOSC2, MRPL18, MTX1, PRPF19, RSL1D1, RUVBL1, and UTP6 (Fig. <a href="#Fig12" class="usa-link">12</a>A). Subsequent LASSO Cox regression analysis further refined this list to five key prognostic genes: AIMP1, CCT3, CCT4, ERAL1, and RSL1D1 (Fig. <a href="#Fig12" class="usa-link">12</a>B–C). Based on these five genes, we constructed a prognostic risk model with the following formula: Risk score = (AIMP1 × 0.2520) + (CCT3 × 0.0330) + (CCT4 × 0.0820) + (ERAL1 × 0.0131) + (RSL1D1 × 0.1855). To rigorously assess the predictive power of this EXOSC family co-expression signature, we applied the model to the TCGA HNSC cohort (<em>n</em> = 504). Patients were stratified into high-risk (<em>n</em> = 252) and low-risk (<em>n</em> = 252) groups based on the median risk score. The prognostic value of the model was supported by three main findings. First, clinical outcome analysis revealed that patients in the high-risk group exhibited significantly poorer overall survival and higher mortality rates compared to those in the low-risk group (Fig. <a href="#Fig12" class="usa-link">12</a>D). Second, Kaplan–Meier survival analysis demonstrated a clear separation between the two groups, with high-risk patients showing significantly worse survival outcomes (<em>P</em> = 0.008; Fig. <a href="#Fig12" class="usa-link">12</a>E). Third, time-dependent ROC curve analysis confirmed the model’s moderate predictive accuracy, with AUCs of 0.581, 0.633, and 0.618 for 1-, 3-, and 5-year overall survival, respectively (Fig. <a href="#Fig12" class="usa-link">12</a>F). To further enhance the clinical utility of this model, we developed a nomogram that integrates both the risk score and relevant clinical variables (Fig. <a href="#Fig13" class="usa-link">13</a>). Notably, the LASSO-derived risk score was among the most significant predictors of prognosis in multivariate analysis, underscoring its potential value in clinical decision-making.</p>
<figure class="fig xbox font-sm" id="Fig12"><h4 class="obj_head">Fig. 12.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365086_41598_2025_15758_Fig12_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/7adf1bacba4a/41598_2025_15758_Fig12_HTML.jpg" loading="lazy" id="d33e1072" height="703" width="669" alt="Fig. 12"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Development of a prognostic risk model based on EXOSC family co-expression genes in HNSC using TCGA cohort data. (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001). (<strong>A</strong>) Univariate Cox regression analysis of the 12 prognostic EXOSC family co-expression genes, presented as forest plots of hazard ratios (HRs) with 95% confidence intervals. (<strong>B</strong>) Feature coefficients across varying lambda values in LASSO regression. (<strong>C</strong>) Partial likelihood deviance versus log(λ) in the LASSO Cox regression model. (<strong>D</strong>) Risk stratification analysis as follows: Top: Patient distribution into high-risk (red) and low-risk (blue) groups based on median risk score. Middle: Survival status (alive = blue; deceased = red). Bottom: Expression heatmap of the 5 prognostic genes across risk groups. (<strong>E</strong>) Kaplan-Meier analysis demonstrating significant survival difference between risk groups (log-rank test). (<strong>F</strong>) Time-dependent receiver operating characteristic (ROC) curves assessing model predictive performance.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig13"><h4 class="obj_head">Fig. 13.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365086_41598_2025_15758_Fig13_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed78/12365086/31d59e974ff4/41598_2025_15758_Fig13_HTML.jpg" loading="lazy" id="d33e1082" height="945" width="723" alt="Fig. 13"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Construction of a clinical nomogram for survival prediction in HNSC patients. Integrative prognostic nomogram incorporating molecular risk signatures and clinicopathological features for individualized survival prediction in HNSC.</p></figcaption></figure></section></section><section id="Sec28"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par44">The findings presented in this study provide a comprehensive analysis of the EXOSC family’s role in HNSC, highlighting their differential expression, prognostic significance, association with clinicopathological parameters, genetic alterations, immune cell infiltration, and immune checkpoint regulation. These results not only underscore the potential of EXOSC family genes as biomarkers for HNSC but also shed light on their functional roles in tumor biology and immune modulation. Our study revealed significant upregulation of all EXOSC family genes (EXOSC1–EXOSC10) in HNSC tissues compared to normal tissues, suggesting their potential involvement in tumorigenesis. These findings are consistent with previous studies demonstrating the oncogenic roles of EXOSC family members in various malignancies, including breast and colorectal cancers<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Notably, the diagnostic potential of EXOSC family genes was evaluated using ROC curve analysis, where EXOSC2 exhibited the highest AUC value (0.910), indicating superior predictive accuracy for distinguishing HNSC from normal tissues. The robust diagnostic performance of the EXOSC family in our study suggests that these genes could serve as promising biomarkers for early HNSC detection, potential prognostic indicator for risk stratification, and novel therapeutic target. Survival analysis highlighted the prognostic significance of EXOSC2, EXOSC3, EXOSC8, and EXOSC9 in HNSC. Patients with high expression of these genes exhibited significantly worse OS, DSS, and PFI, emphasizing their role in tumor aggressiveness and progression. Importantly, the diagnostic value of the EXOSC family in HNSC is notably high, with several members demonstrating exceptional AUC values (e.g., EXOSC2 AUC = 0.910), reinforcing their potential as reliable biomarkers. This contrasts with findings in other cancers, where EXOSC gene upregulation may be present but often lacks comparable diagnostic discrimination. Furthermore, the strong association of EXOSC2, EXOSC3, EXOSC8, and EXOSC9 with unfavorable survival outcomes-OS, DSS, and PFI-highlights their clinical relevance in prognostication. These associations are markedly stronger in HNSC compared to prior reports in malignancies such as colorectal carcinoma and liver carcinoma, where similar survival correlations were either absent or limited to a subset of EXOSC genes (e.g., EXOSC4 or EXOSC10)<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Our study provides strong evidence that EXOSC family genes, particularly EXOSC2, EXOSC3, EXOSC8, and EXOSC9, are significantly associated with poor survival outcomes in HNSC patients. These findings highlight their potential as prognostic biomarkers and therapeutic targets, paving the way for further research into their roles in tumor progression, immune regulation, and treatment resistance.</p>
<p id="Par45">The correlation between EXOSC family gene expression and advanced clinicopathological features, such as higher pathological T stage, clinical stage IV, and lymphovascular invasion, underscores their potential role in tumor aggressiveness. For instance, EXOSC2 and EXOSC5 were significantly associated with advanced disease stages and higher histologic grades, suggesting their involvement in promoting tumor invasion and metastasis. These findings are supported by studies showing that exosome complex components facilitate the expression of genes involved in epithelial-mesenchymal transition (EMT) and extracellular matrix remodeling<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. The association of EXOSC4 and EXOSC9 with primary therapy outcomes, particularly in patients with stable or progressive disease, highlights their potential as predictors of treatment resistance. This is consistent with the role of exosome complex genes in modulating DNA damage response and repair pathways, which are critical for radiation and chemotherapy resistance<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. EXOSC1, EXOSC7, and EXOSC8 showed significant associations with male gender, higher histologic grades, and lymphovascular invasion, suggesting a role in tumor differentiation and aggressive phenotypes. These findings indicate that certain EXOSC genes may contribute to disease progression and resistance to therapy, providing valuable insights for risk stratification and personalized treatment approaches in HNSC patients.</p>
<p id="Par46">Malignant progression unfolds through well-defined histopathological stages, a process fundamentally driven by the sequential accumulation of genetic mutations and their consequent impact on transcriptional programs<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Our study revealed that genetic alterations in EXOSC family genes are relatively common in HNSC, with a total alteration frequency of 15.32% across 496 patients. The analysis of genetic alterations revealed that EXOSC4 exhibited the highest alteration frequency (8%), predominantly through amplifications. The high frequency of EXOSC4 amplifications suggests that this gene may function as an oncogenic driver in HNSC. Gene amplifications often lead to increased mRNA and protein expression, which can enhance cell proliferation, invasion, and survival. Previous studies have shown that amplifications of ribonucleoprotein complex genes can disrupt RNA processing, promote genomic instability, and facilitate tumor growth<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Our findings support this hypothesis, as EXOSC4 alterations were widespread and frequently involved copy number gains, reinforcing its potential role in tumor progression. This suggests that EXOSC4 may play a pivotal role in HNSC pathogenesis, potentially through the overexpression of oncogenic transcripts.</p>
<p id="Par47">We also constructed a PPI network to investigate the potential molecular partners and functional associations of the EXOSC family genes in HNSC. The EXOSC family constitutes a core component of the RNA exosome complex, which is essential for RNA processing and degradation, particularly in the regulation of non-coding RNAs and mRNA surveillance pathways. Through GeneMANIA analysis, we identified 20 significant interacting proteins, including MPHOSPH6, DIS3, C1D, and PNPT1, with MPHOSPH6 and DIS3 showing the strongest interaction scores. Notably, DIS3 encodes a catalytic subunit of the exosome complex and possesses 3´ to 5´ exoribonuclease and endoribonuclease activities<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. It plays a central role in RNA quality control and turnover, and mutations in DIS3 have been implicated in multiple cancers, including multiple myeloma<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Its interaction with multiple EXOSC components in our analysis underscores its central role in exosome-mediated RNA decay in HNSC. MPHOSPH6, another high-confidence interactor, is known to associate with the nuclear exosome targeting complex and assist in rRNA processing<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Dysregulation of rRNA metabolism is frequently observed in cancers, where it supports enhanced protein synthesis and tumor progression<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Therefore, the interaction between EXOSC family members and MPHOSPH6 may reflect an oncogenic role of altered ribonucleolytic activity in HNSC. Functional enrichment analysis revealed that EXOSC family proteins and their interactors are significantly involved in pathways related to exonuclease activity, RNA degradation, ncRNA catabolic processes, and regulation of RNA stability. These biological processes are consistent with the known functions of the exosome complex and reinforce the hypothesis that EXOSC dysregulation contributes to oncogenesis by disturbing RNA homeostasis. Recent studies have shown that aberrant RNA processing and degradation can promote tumorigenesis through the accumulation of non-functional or deleterious RNA species, altered splicing, and disrupted post-transcriptional gene regulation<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Moreover, the implication of regulation of mRNA catabolic process in our analysis suggests that EXOSC family genes may influence cancer-relevant transcripts, including those involved in immune modulation, proliferation, and apoptosis.</p>
<p id="Par48">The tumor microenvironment (TME) plays a pivotal role in HNSC progression, influencing tumor immune evasion, treatment response, and overall prognosis<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Using ssGSEA analysis, we identified significant correlations between EXOSC family gene expression and the relative abundance of various immune cell types, highlighting their potential role in immune regulation within the TME. Our findings suggest that EXOSC family genes are widely involved in immune modulation, with some members exhibiting strong positive associations with immunosuppressive cell types (e.g., Th2 cells, Treg, NK CD56bright cells), while others show negative correlations with cytotoxic and antigen-presenting cells (e.g., CD8 + T cells, dendritic cells (DC), neutrophils, and NK cells). This suggests that alterations in EXOSC gene expression may shape the immunosuppressive landscape of HNSC tumors, contributing to immune evasion mechanisms. EXOSC1, EXOSC2, and EXOSC4 are Positively Correlated with Th2 Cells and NK CD56bright Cells. Th2 cells and NK CD56bright cells are known to be involved in pro-tumorigenic immune responses, suppressing cytotoxic T lymphocyte (CTL) activity and promoting an immunosuppressive TME<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. EXOSC1, EXOSC2, and EXOSC4 expression positively correlated with Th2 cells and NK CD56bright cells, suggesting that these genes may contribute to a tumor-permissive immune environment. Notably, EXOSC2 exhibited a strong negative correlation with cytotoxic immune cells, including CD8 + T cells, NK CD56dim cells, and Th1 cells, reinforcing its potential role in immune evasion. These findings suggest that targeting EXOSC1, EXOSC2, or EXOSC4 could reverse immunosuppressive signaling and enhance anti-tumor immunity. T helper cells play a dual role in tumor immunity, depending on the balance between Th1 (anti-tumor) and Th2 (pro-tumor) responses<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. EXOSC3, EXOSC8, and EXOSC10 expression positively correlated with T helper cells, but their impact on Th1 vs. Th2 differentiation needs further exploration. EXOSC10 exhibited a strong positive correlation with Treg and Th2 cells, which are known for their immunosuppressive roles. Tregs suppress anti-tumor immune responses, facilitating tumor immune escape and promoting a tolerogenic microenvironment<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Given these associations, EXOSC10 may be a key modulator of immunosuppression in HNSC and could serve as a potential therapeutic target for immunomodulation. CD8 + T cells and cytotoxic NK cells are crucial for tumor clearance, mediating direct tumor cell killing via perforin and granzyme pathways<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. The negative correlation between EXOSC4, EXOSC5, and EXOSC6 expression and CD8 + T cells/cytotoxic cells suggests that these genes may be involved in suppressing immune-mediated tumor clearance, leading to a weaker anti-tumor response. This could explain why high expression of EXOSC4 is linked to poor therapy response, as tumors with lower CD8 + T cell infiltration tend to be more resistant to immunotherapy. DCs are key antigen-presenting cells that activate T cell responses against tumors. The observed negative correlation between EXOSC1, EXOSC3, EXOSC6, and EXOSC7 expression and DC infiltration suggests that these genes might contribute to immune escape by impairing antigen presentation. Similar trends have been reported in lung and pancreatic cancers, where decreased DC infiltration was linked to worse prognosis and immune evasion mechanisms<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p>
<p id="Par49">Immune checkpoints play a critical role in regulating immune responses by preventing excessive immune activation<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. However, tumors can exploit these pathways to evade immune surveillance, leading to immune escape and progression. The correlation analysis between EXOSC family gene expression and immune checkpoint genes provides new insights into how these genes might contribute to immune modulation in HNSC. Our findings reveal that specific EXOSC family genes are significantly correlated with key immune checkpoint molecules, suggesting a potential role in immune escape mechanisms. Notably, EXOSC1, EXOSC3, EXOSC5, EXOSC9, and EXOSC10 exhibited strong positive correlations with multiple immune checkpoint genes, including PDCD1 (PD-1), CD274 (PD-L1), CTLA4, HAVCR2 (TIM-3), IDO1, and LAG3, which are known mediators of T cell exhaustion and tumor immune evasion<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Conversely, EXOSC4, EXOSC6, and EXOSC7 showed significant negative correlations with several immune checkpoint genes, suggesting a potential role in maintaining an anti-tumor immune response. EXOSC1, EXOSC3, and EXOSC10 were positively correlated with multiple immune checkpoint genes (e.g., PDCD1, CTLA4, LAG3, IDO1, and HAVCR2). These immune checkpoints are key mediators of T cell exhaustion, suppressing T cell proliferation and cytotoxic activity<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. The strong positive correlation between EXOSC3 and 14 immune checkpoints, including PDCD1, CTLA4, and LAG3, suggests that EXOSC3 may be involved in promoting an immunosuppressive microenvironment, potentially leading to reduced anti-tumor immunity and resistance to checkpoint blockade therapy. Similarly, EXOSC10 exhibited the most extensive positive correlations (25 immune checkpoints), including PDCD1, CD274, CTLA4, and TIGIT, further suggesting a role in immune suppression and tumor progression. Given these associations, targeting EXOSC3 or EXOSC10 may enhance immune checkpoint inhibitor (ICI) efficacy in HNSC patients by reducing immunosuppressive signaling. Unlike EXOSC1, EXOSC3, and EXOSC10, EXOSC4 and EXOSC6 showed significant negative correlations with multiple immune checkpoints, including PDCD1, CTLA4, CD274, and CSF1R. EXOSC4 was negatively correlated with 21 checkpoint genes, while EXOSC6 exhibited negative correlations with 16 checkpoint genes, suggesting a potential role in maintaining anti-tumor immune responses. This suggests that low EXOSC4 or EXOSC6 expression may be associated with better immune activation and improved response to immunotherapy. EXOSC5, EXOSC8, and EXOSC9 demonstrated diverse associations with immune checkpoint genes, showing both positive and negative correlations depending on the checkpoint molecule. For example, EXOSC5 was positively correlated with CTLA4 and PDCD1 but negatively correlated with IL10 and TGFB1, suggesting a dual role in immune regulation. EXOSC8 was positively associated with CD274 (PD-L1), IDO1, and CTLA4, supporting a role in immune suppression, while its positive correlation with KDR might indicate involvement in tumor angiogenesis. EXOSC9 had a strong positive correlation with 20 immune checkpoint genes, including PDCD1, CTLA4, LAG3, and TIGIT, suggesting that it may contribute to an immunosuppressive tumor microenvironment. This suggests their potential involvement in immune escape mechanisms and highlights their possible value as biomarkers for immunotherapeutic responsiveness. Notably, while previous studies have mainly linked EXOSC genes to RNA processing and tumor cell proliferation in other cancers, our findings uniquely emphasize their immune-related functions in HNSC. This suggests a context-dependent role of EXOSC genes, potentially shaped by the tissue-specific immune landscape. Among the EXOSC members, EXOSC10 showed strong positive correlations with a wide range of immune checkpoints, positioning it as a potential immunotherapeutic target, either alone or in combination with checkpoint inhibitors. Its distinct association with both effector and regulatory immune components suggests a dual role in modulating immune surveillance and immune suppression. Overall, our findings underscore the innovative potential of EXOSC genes as immune-related biomarkers and therapeutic targets in HNSC, expanding their known functional repertoire. Future experimental work is warranted to elucidate the mechanistic basis of these interactions and to explore whether modulation of EXOSC expression can enhance the efficacy of existing immunotherapies.</p>
<p id="Par50">Emerging evidence indicates that RNA modifications such as m6A, m5C, and m1A play pivotal roles in post-transcriptional gene regulation and are closely implicated in cancer progression, including HNSC<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>–<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Our study provides novel insights into the regulatory interplay between the EXOSC family of RNA exosome complex components and RNA modification regulators in HNSC, suggesting a mechanistic framework through which RNA modifications may contribute to tumor biology via modulation of EXOSC gene expression. In the study, we observed widespread and significant correlations between EXOSC family genes expressions and core regulators of m1A, m5C, and m6A modifications. Notably, most EXOSC genes exhibited strong positive associations with m1A-related genes, including TRMT61A, TRMT10C, TRMT6, and YTHDF1, implicating a coordinated expression pattern that may reflect shared regulatory mechanisms or functional convergence in RNA metabolism. Prior studies have shown that m1A writers such as TRMT61A are involved in maintaining translation fidelity and cellular stress responses<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. The strong correlation with YTHDF1, an m1A/m6A reader, further suggests that EXOSC genes may be embedded within RNA modification-dependent regulatory circuits that fine-tune mRNA stability and translation in HNSC cells. Interestingly, selective negative correlations were also observed; for instance, EXOSC4 and EXOSC7 displayed inverse associations with YTHDC1 and YTHDF3, respectively. These exceptions may indicate subunit-specific roles within the EXOSC complex or feedback mechanisms aimed at buffering aberrant modification-induced transcriptomic shifts. Similar heterogeneity has been reported in other cancers, where specific RNA-modifying enzymes exert context-dependent effects on gene networks<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Within the m5C regulatory network, EXOSC family genes demonstrated positive correlations with NSUN2, NSUN5, and the reader ALYREF. NSUN2, in particular, has been implicated in enhanced proliferation and metastasis in various cancers through m5C-dependent regulation of mRNA stability<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. These findings suggest that m5C modifications may modulate EXOSC gene function or expression, contributing to oncogenic phenotypes. Conversely, EXOSC1, EXOSC4, and EXOSC7 were negatively associated with the DNA demethylase TET2, a finding of particular interest given the dual role of TET enzymes in both DNA and RNA methylation landscapes. The negative correlation between EXOSC4 and multiple m5C regulators, including NSUN3, NSUN7, DNMT1, and DNMT3A, hints at a possible suppressive interplay, potentially reflective of EXOSC4-mediated degradation of RNA species enriched in these modifications. The m6A landscape similarly revealed robust positive correlations between EXOSC genes and key m6A writers/readers, reinforcing the notion that RNA modification and degradation machineries are co-regulated in HNSC. For instance, positive correlations with METTL3, WTAP, and YTHDF1 were prevalent across the family, which may support enhanced RNA turnover or selectivity in target mRNA surveillance. Nevertheless, EXOSC4 and EXOSC7 consistently showed negative associations with the eraser FTO and reader YTHDF3, as well as additional components like ZC3H13 and CBLL1, indicating a potentially unique regulatory axis for these subunits. The functional implications of these inverse relationships merit further mechanistic investigation but may represent a compensatory balance between RNA stabilization and degradation pathways. Collectively, these results highlight the tight interconnection between RNA modification machinery and the RNA exosome complex, positioning EXOSC genes as potential integrators of epitranscriptomic regulation in HNSC. Previous studies have demonstrated that alterations in RNA modification pathways can reprogram cancer transcriptomes, affecting splicing, export, translation, and decay<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Our findings suggest that EXOSC genes may not only respond to these regulatory cues but also participate actively in shaping RNA homeostasis in an RNA modification-dependent manner. Given the emerging role of RNA modifications as therapeutic targets, particularly m6A and m5C regulators<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, the EXOSC family may represent a novel class of co-targetable factors that modulate tumor cell plasticity and response to treatment.</p>
<p id="Par51">To further elucidate the biological functions of EXOSC family genes in HNSC, we conducted GSEA, which revealed significant enrichment of pathways related to keratinization, formation of the cornified envelope, and developmental biology. These findings provide critical insights into how EXOSC family members may contribute to tumor progression through modulation of epithelial differentiation and cellular developmental programs. Keratinization, a hallmark of squamous epithelial cells, is a tightly regulated process involving the terminal differentiation of keratinocytes<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. In the context of HNSC, aberrant keratinization has been widely associated with tumor differentiation status and clinical behavior. Poorly differentiated HNSC tumors often exhibit reduced keratinization, while well-differentiated tumors tend to retain keratin pearl formation<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Dysregulation of genes involved in keratinocyte differentiation pathways may facilitate a dedifferentiated, more aggressive tumor phenotype<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. The association between EXOSC gene expression and keratinization-related pathways in our analysis suggests that EXOSC family members may play a role in maintaining epithelial identity or, conversely, promoting epithelial dedifferentiation when dysregulated. The enrichment of the Formation of the Cornified Envelope pathway further supports this hypothesis. The cornified envelope is an insoluble protein layer formed during the late stages of keratinocyte differentiation, contributing to the physical barrier of stratified squamous epithelium<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Its formation involves cross-linking of involucrin, loricrin, and small proline-rich proteins by transglutaminases. Recent studies have implicated disruption of cornified envelope components in carcinogenesis<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. The involvement of EXOSC family genes in this pathway implies that they may regulate post-transcriptional processing of mRNAs encoding key structural proteins, thereby influencing epithelial barrier integrity and tumor invasiveness. Moreover, the enrichment of Developmental Biology pathways underscores the role of EXOSC family genes in regulating fundamental cellular programs often hijacked during tumorigenesis. Cancer is frequently described as a disease of altered development, where embryonic gene expression programs are reactivated to support proliferation, migration, and immune evasion<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. In HNSC, activation of developmental pathways such as Wnt, Notch, and Hedgehog signaling has been linked to epithelial–mesenchymal transition (EMT), tumor heterogeneity, and therapy resistance<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Given that the RNA exosome complex is known to modulate the stability and maturation of diverse classes of RNAs, the EXOSC family may influence these developmental processes by regulating non-coding RNAs, pre-mRNAs, or transcription factor transcripts essential for oncogenic reprogramming. Collectively, these results highlight a novel connection between EXOSC family gene expression and key epithelial and developmental processes in HNSC. This suggests a dual role for EXOSC genes in both maintaining normal epithelial differentiation and facilitating malignant transformation when dysregulated. Our findings are consistent with prior research demonstrating the oncogenic potential of RNA exosome components through disruption of RNA surveillance and transcriptome stability.</p>
<p id="Par52">The EXOSC family, which forms the core of the RNA exosome complex, plays a critical role in RNA processing, degradation, and surveillance<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Our TCGA transcriptome data analysis identified a substantial number of genes co-expressed with EXOSC family members, revealing both positively and negatively correlated genes. Notably, the number of positively correlated genes varied across EXOSC members, with EXOSC9 (11,678) and EXOSC10 (11,657) showing the highest co-expression, while EXOSC4 (3,928) exhibited the lowest number of positively correlated genes. These differences suggest distinct functional roles and regulatory mechanisms among EXOSC genes in HNSC pathogenesis. Interestingly, we identified 40 intersecting genes that were co-expressed with multiple EXOSC family members (|R| &gt; 0.3, <em>P</em> &lt; 0.001). This overlapping gene set may represent a core regulatory network influenced by EXOSC genes in HNSC, further emphasizing their potential role in tumor progression and RNA metabolism. Gene Ontology (GO) enrichment analysis revealed that co-expressed genes of the EXOSC family are highly enriched in ribonucleoprotein complex biogenesis, ribosome biogenesis, and ribonucleoprotein complex assembly. These findings are consistent with previous studies showing that dysregulated ribosome biogenesis is a hallmark of cancer, facilitating increased protein synthesis to sustain rapid tumor growth<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. EXOSC1, EXOSC2, and EXOSC10 showed strong correlations with genes involved in ribosome biogenesis, reinforcing their pro-tumorigenic role through the enhancement of translational capacity. EXOSC4 had the fewest positively correlated genes, suggesting that its role in ribosome biogenesis may be more restricted, or potentially inhibitory. The involvement of EXOSC genes in ribonucleoprotein complex assembly further supports their function in mRNA processing and decay, processes that are critical for maintaining the stability and turnover of oncogenic transcripts in cancer cells. GO analysis of cellular component (CC) enrichment revealed that EXOSC family co-expressed genes are significantly associated with mitochondrial inner membranes, mitochondrial ribosomes, and organellar ribosomes. Given that mitochondrial function is closely linked to cancer metabolism and chemoresistance<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>, this suggests that the EXOSC family may contribute to mitochondrial biogenesis and energy metabolism in HNSC. EXOSC9 and EXOSC10 showed the strongest enrichment in mitochondrial components, suggesting a role in mitochondria-related RNA processing or translation. Since mitochondrial ribosomes regulate oxidative phosphorylation (OXPHOS), the correlation between EXOSC family genes and mitochondrial ribosome components raises the possibility that EXOSC genes modulate metabolic reprogramming in HNSC, potentially influencing tumor progression and therapy resistance. The molecular function (MF) enrichment analysis revealed that EXOSC family co-expressed genes are significantly involved in: Structural constituent of ribosomes, rRNA binding and ATP-dependent activity acting on DNA. These findings align with the known functions of EXOSC genes as key regulators of RNA degradation<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. The strong enrichment in rRNA binding and ribosome structural functions suggests that EXOSC genes may contribute to ribosomal RNA processing and stability, thereby supporting cancer cell proliferation by enhancing protein translation. KEGG pathway analysis demonstrated that EXOSC family co-expressed genes are primarily involved in the ribosome, spliceosome, and mRNA surveillance pathways. These pathways play pivotal roles in post-transcriptional gene regulation, and their dysregulation is a common feature in cancer<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. EXOSC1, EXOSC3, and EXOSC10 were enriched in ribosome-related pathways, reinforcing their role in enhancing protein synthesis, a key feature of aggressive tumors. Given that tumor cells exploit ribosomal activity to support unchecked proliferation, targeting ribosome biogenesis has emerged as a potential therapeutic strategy in HNSC<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Our findings suggest that dysregulated EXOSC family expression could serve as a potential target for therapies aimed at disrupting ribosome function in HNSC. The spliceosome pathway was significantly enriched in EXOSC family co-expressed genes, particularly in EXOSC2, EXOSC5, and EXOSC6. Aberrant alternative splicing is a known driver of cancer progression, leading to the production of oncogenic protein isoforms that promote survival, invasion, and therapy resistance<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. These results suggest that EXOSC genes might influence splicing factor regulation, affecting the expression of tumor-promoting splice variants in HNSC. The enrichment of EXOSC-related genes in the mRNA surveillance pathway highlights their potential role in post-transcriptional regulation of tumor suppressor genes. The EXOSC complex is involved in the degradation of aberrant mRNAs, preventing the accumulation of defective transcripts. However, in the context of HNSC, it is possible that EXOSC overexpression enhances the degradation of tumor-suppressive mRNAs, leading to uncontrolled tumor growth.</p>
<p id="Par53">We then systematically explored the prognostic potential of EXOSC family co-expression genes in HNSC and successfully constructed a robust risk model and nomogram to predict patient outcomes. Our analysis identified five key co-expression genes-AIMP1, CCT3, CCT4, ERAL1, and RSL1D1-which collectively demonstrated significant predictive value for overall survival. The EXOSC family genes are involved in RNA degradation and processing, critical functions in post-transcriptional regulation that are often dysregulated in cancer<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Our findings suggest that dysregulation of EXOSC family co-expression partners may reflect broader transcriptomic instability in HNSC. For instance, RSL1D1 has been shown to mediate p53-dependent cell cycle regulation, which may account for its prognostic impact in HNSC<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Furthermore, CCT3 and CCT4, members of the chaperonin-containing TCP-1 complex, are known to assist in the folding of key oncogenic proteins, including cyclins and p53. Overexpression of CCT family members have been reported in various malignancies and is associated with enhanced tumor growth, metastasis, and chemoresistance<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Similarly, ERAL1, a mitochondrial RNA chaperone, may reflect mitochondrial dysfunction and altered metabolic reprogramming, hallmarks of many aggressive tumors<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. Our prognostic model demonstrated significant discriminative power, with high-risk patients exhibiting substantially poorer overall survival. The time-dependent ROC analysis further validated the model’s utility, yielding AUC values above 0.6 across 1-, 3-, and 5-year survival predictions. Although the AUCs were moderate, they were comparable to or better than those reported for other RNA-based prognostic signatures in HNSC. Importantly, we also developed a nomogram that incorporated both molecular (risk score) and clinical parameters, enhancing its translational potential. The LASSO-derived risk score emerged as one of the strongest independent predictors of survival, underscoring its robustness. Notably, the EXOSC family and its associated co-expressed genes may also offer insights into therapeutic vulnerabilities. Previous studies have shown that inhibition of RNA processing pathways can sensitize tumors to chemotherapy or immunotherapy<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Thus, components of the EXOSC-associated risk model may not only serve as prognostic biomarkers but also as potential therapeutic targets.</p>
<p id="Par54">The findings from this study have significant clinical implications, particularly in the development of personalized treatment strategies for HNSC. The strong prognostic value of EXOSC2, EXOSC3, EXOSC8, and EXOSC9 suggests that they could serve as novel biomarkers for patient risk stratification. Additionally, given their roles in immune modulation, EXOSC family genes may represent potential targets for combination therapies involving immune checkpoint inhibitors. Future research should focus on functional validation studies to elucidate the precise molecular mechanisms underlying EXOSC gene-mediated tumor progression and immune regulation in HNSC. Additionally, prospective clinical trials assessing the utility of EXOSC-based biomarkers in guiding therapeutic decisions would be valuable.</p>
<p id="Par55">In conclusion, this study provides a comprehensive bioinformatics analysis of EXOSC family genes in HNSC, revealing their significant upregulation in tumor tissues and strong prognostic implications. High expression levels of EXOSC2, EXOSC3, EXOSC4, EXOSC8, and EXOSC9 were associated with poorer survival outcomes, while genetic alterations, particularly in EXOSC4, emerged as critical events in tumor progression. Immune infiltration analysis further suggested that EXOSC genes may influence the tumor microenvironment by modulating immune cell populations. Functional enrichment analyses linked EXOSC genes to key oncogenic pathways, emphasizing their role in tumorigenesis. Collectively, these findings highlight the potential of EXOSC family members as prognostic biomarkers and therapeutic targets in HNSC, paving the way for further research into their mechanistic roles and clinical applications.</p></section><section id="Sec29"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (156.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (156.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (157.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 9</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 10</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 11</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM12"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 12</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM13"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM13_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 13</a><sup> (275.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM14"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM14_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 14</a><sup> (129KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM15"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM15_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 15</a><sup> (148.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM16"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM16_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 16</a><sup> (188.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM17"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM17_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 17</a><sup> (6.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM18"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM18_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 18</a><sup> (6.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM19"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM19_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 19</a><sup> (6.7MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM20"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM20_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 20</a><sup> (6.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM21"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM21_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 21</a><sup> (6.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM22"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM22_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 22</a><sup> (6.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM23"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM23_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 23</a><sup> (6.9MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM24"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM24_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 24</a><sup> (6.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM25"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM25_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 25</a><sup> (6.9MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM26"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM26_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 26</a><sup> (6.9MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM27"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM27_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 27</a><sup> (137.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM28"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM28_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 28</a><sup> (137.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM29"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM29_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 29</a><sup> (137.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM30"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM30_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 30</a><sup> (136.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM31"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM31_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 31</a><sup> (137.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM32"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM32_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 32</a><sup> (136.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM33"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM33_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 33</a><sup> (137.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM34"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM34_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 34</a><sup> (136.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM35"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM35_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 35</a><sup> (137.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM36"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM36_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 36</a><sup> (134.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM37"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM37_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 37</a><sup> (306.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM38"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM38_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 38</a><sup> (22KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM39"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM39_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 39</a><sup> (12.3KB, docx) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Xuezhong Zhang and Qingmei Jia proposed the study idea. Xuezhong Zhang, Mengmeng Zhao, Tingting Chu, Jiashawei and Qingmei Jia collected and analyzed the data. Xuezhong Zhang drafted the manuscript. Xuezhong Zhang and Qingmei Jia critically revised the manuscript. All authors contributed to the article and approved the submitted version.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>All supporting data are provided in the supplementary files.</p></section><section id="notes3"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par57">The authors declare no competing interests.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Ethical approval</h3>
<p id="Par58">The datasets analyzed in this study were sourced from publicly available repositories.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Consent to participate</h3>
<p id="Par59">As this research utilized anonymized, publicly accessible data, obtaining individual participant consent was not required.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Jiasha Wei, Email: weijiasha_421@163.com.</p>
<p>Qingmei Jia, Email: jqmswjs@163.com.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Johnson, D. E. et al. Head and neck squamous cell carcinoma [J]. <em>Nat. Rev. Dis. Primers</em>. <strong>6</strong> (1), 92 (2020).
</cite> [<a href="https://doi.org/10.1038/s41572-020-00224-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7944998/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33243986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johnson,%20D.%20E.%20et%20al.%20Head%20and%20neck%20squamous%20cell%20carcinoma%20%5BJ%5D.%20Nat.%20Rev.%20Dis.%20Primers.%206%20(1),%2092%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Nguyen, E. S. et al. Prognostic factors and outcomes of de Novo sinonasal squamous cell carcinoma: A systematic review and Meta-analysis [J]. <em>Otolaryngol. Head Neck Surg.</em><strong>166</strong> (3), 434–443 (2022).
</cite> [<a href="https://doi.org/10.1177/01945998211021023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34253092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen,%20E.%20S.%20et%20al.%20Prognostic%20factors%20and%20outcomes%20of%20de%20Novo%20sinonasal%20squamous%20cell%20carcinoma:%20A%20systematic%20review%20and%20Meta-analysis%20%5BJ%5D.%20Otolaryngol.%20Head%20Neck%20Surg.166%20(3),%20434%E2%80%93443%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Hartl, D. M. et al. Review of outcomes after salvage surgery for recurrent squamous cell carcinoma of the head and neck. <em>Cancers (Basel)</em>, <strong>15</strong>(19), (2023).</cite> [<a href="https://doi.org/10.3390/cancers15194692" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10571840/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37835386/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Kolapalli, S. P., Nielsen, T. M. &amp; Frankel, L. B. Post-transcriptional dynamics and RNA homeostasis in autophagy and cancer [J]. <em>Cell. Death Differ.</em><strong>32</strong> (1), 27–36 (2025).
</cite> [<a href="https://doi.org/10.1038/s41418-023-01201-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11742036/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37558732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kolapalli,%20S.%20P.,%20Nielsen,%20T.%20M.%20&amp;%20Frankel,%20L.%20B.%20Post-transcriptional%20dynamics%20and%20RNA%20homeostasis%20in%20autophagy%20and%20cancer%20%5BJ%5D.%20Cell.%20Death%20Differ.32%20(1),%2027%E2%80%9336%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Ogami, K. &amp; Suzuki, H. I. Nuclear RNA exosome and pervasive transcription: dual sculptors of genome function [J]. <em>Int. J. Mol. Sci.</em>, <strong>22</strong>(24), (2021).</cite> [<a href="https://doi.org/10.3390/ijms222413401" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8707817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34948199/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Fasken, M. B. et al. The RNA exosome and human disease [J]. <em>Methods Mol. Biol.</em><strong>2062</strong>, 3–33 (2020).
</cite> [<a href="https://doi.org/10.1007/978-1-4939-9822-7_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31768969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fasken,%20M.%20B.%20et%20al.%20The%20RNA%20exosome%20and%20human%20disease%20%5BJ%5D.%20Methods%20Mol.%20Biol.2062,%203%E2%80%9333%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Zhang, Y. et al. Integrated bioinformatic investigation of exoscs in hepatocellular carcinoma followed by the preliminary validation of EXOSC5 in cell proliferation [J]. <em>Int. J. Mol. Sci.</em>, <strong>23</strong>(20), (2022).</cite> [<a href="https://doi.org/10.3390/ijms232012161" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9603681/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36293016/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Liu, Q. et al. Exosome component 1 cleaves single-stranded DNA and sensitizes human kidney renal clear cell carcinoma cells to poly(ADP-ribose) polymerase inhibitor. <em>Elife</em><strong>10</strong>, (2021).</cite> [<a href="https://doi.org/10.7554/eLife.69454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8260222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34159897/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Lv, C. G. et al. EXOSC2 mediates the Pro-tumor role of WTAP in breast cancer cells via activating the Wnt/β-Catenin signal [J]. <em>Mol. Biotechnol.</em><strong>66</strong> (9), 2569–2582 (2024).
</cite> [<a href="https://doi.org/10.1007/s12033-023-00834-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37856011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lv,%20C.%20G.%20et%20al.%20EXOSC2%20mediates%20the%20Pro-tumor%20role%20of%20WTAP%20in%20breast%20cancer%20cells%20via%20activating%20the%20Wnt/%CE%B2-Catenin%20signal%20%5BJ%5D.%20Mol.%20Biotechnol.66%20(9),%202569%E2%80%932582%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Tsuda, M. et al. Aberrant expression of MYD88 via RNA-controlling CNOT4 and EXOSC3 in colonic mucosa impacts generation of colonic cancer [J]. <em>Cancer Sci.</em><strong>112</strong> (12), 5100–5113 (2021).
</cite> [<a href="https://doi.org/10.1111/cas.15157" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8645755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34626022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsuda,%20M.%20et%20al.%20Aberrant%20expression%20of%20MYD88%20via%20RNA-controlling%20CNOT4%20and%20EXOSC3%20in%20colonic%20mucosa%20impacts%20generation%20of%20colonic%20cancer%20%5BJ%5D.%20Cancer%20Sci.112%20(12),%205100%E2%80%935113%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Pan, Y. et al. EXOSC4 functions as a potential oncogene in development and progression of colorectal cancer [J]. <em>Mol. Carcinog.</em><strong>57</strong> (12), 1780–1791 (2018).
</cite> [<a href="https://doi.org/10.1002/mc.22896" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30155936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan,%20Y.%20et%20al.%20EXOSC4%20functions%20as%20a%20potential%20oncogene%20in%20development%20and%20progression%20of%20colorectal%20cancer%20%5BJ%5D.%20Mol.%20Carcinog.57%20(12),%201780%E2%80%931791%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Chen, X. et al. EXOSC5 promotes proliferation of gastric cancer through regulating AKT/STAT3 signaling pathways [J]. <em>J. Cancer</em>. <strong>13</strong> (5), 1456–1467 (2022).
</cite> [<a href="https://doi.org/10.7150/jca.69166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8965127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35371329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20X.%20et%20al.%20EXOSC5%20promotes%20proliferation%20of%20gastric%20cancer%20through%20regulating%20AKT/STAT3%20signaling%20pathways%20%5BJ%5D.%20J.%20Cancer.%2013%20(5),%201456%E2%80%931467%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Cui, K. et al. EXOSC8 promotes colorectal cancer tumorigenesis via regulating ribosome biogenesis-related processes [J]. <em>Oncogene</em><strong>41</strong> (50), 5397–5410 (2022).
</cite> [<a href="https://doi.org/10.1038/s41388-022-02530-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36348012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cui,%20K.%20et%20al.%20EXOSC8%20promotes%20colorectal%20cancer%20tumorigenesis%20via%20regulating%20ribosome%20biogenesis-related%20processes%20%5BJ%5D.%20Oncogene41%20(50),%205397%E2%80%935410%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Yoshino, S. et al. EXOSC9 depletion attenuates P-body formation, stress resistance, and tumorigenicity of cancer cells [J]. <em>Sci. Rep.</em><strong>10</strong> (1), 9275 (2020).
</cite> [<a href="https://doi.org/10.1038/s41598-020-66455-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7283315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32518284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshino,%20S.%20et%20al.%20EXOSC9%20depletion%20attenuates%20P-body%20formation,%20stress%20resistance,%20and%20tumorigenicity%20of%20cancer%20cells%20%5BJ%5D.%20Sci.%20Rep.10%20(1),%209275%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Meng, Z. Y. et al. EXOSC10 is a novel hepatocellular carcinoma prognostic biomarker: a comprehensive bioinformatics analysis and experiment verification [J]. <em>PeerJ</em>, 11e15860. (2023).</cite> [<a href="https://doi.org/10.7717/peerj.15860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10494838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37701829/" class="usa-link">PubMed</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Liu, J. et al. An Integrated TCGA Pan-Cancer Clinical Data Resource to Drive High-Quality Survival Outcome Analytics [J]. <em>Cell</em>, <strong>173</strong>(2), 400 – 16.e11 (2018).</cite> [<a href="https://doi.org/10.1016/j.cell.2018.02.052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6066282/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29625055/" class="usa-link">PubMed</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>De Hond, A. A. H., Steyerberg, E. W. &amp; Van Calster, B. Interpreting area under the receiver operating characteristic curve [J]. <em>Lancet Digit. Health</em>. <strong>4</strong> (12), e853–e5 (2022).
</cite> [<a href="https://doi.org/10.1016/S2589-7500(22)00188-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36270955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Hond,%20A.%20A.%20H.,%20Steyerberg,%20E.%20W.%20&amp;%20Van%20Calster,%20B.%20Interpreting%20area%20under%20the%20receiver%20operating%20characteristic%20curve%20%5BJ%5D.%20Lancet%20Digit.%20Health.%204%20(12),%20e853%E2%80%93e5%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Cerami, E. et al. The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data [J]. <em>Cancer Discov</em>. <strong>2</strong> (5), 401–404 (2012).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-12-0095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3956037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22588877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cerami,%20E.%20et%20al.%20The%20cBio%20cancer%20genomics%20portal:%20an%20open%20platform%20for%20exploring%20multidimensional%20cancer%20genomics%20data%20%5BJ%5D.%20Cancer%20Discov.%202%20(5),%20401%E2%80%93404%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Bindea, G. et al. Spatiotemporal dynamics of intratumoral immune cells reveal the immune landscape in human cancer [J]. <em>Immunity</em><strong>39</strong> (4), 782–795 (2013).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2013.10.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24138885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bindea,%20G.%20et%20al.%20Spatiotemporal%20dynamics%20of%20intratumoral%20immune%20cells%20reveal%20the%20immune%20landscape%20in%20human%20cancer%20%5BJ%5D.%20Immunity39%20(4),%20782%E2%80%93795%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Hu, F. F. et al. Expression profile of immune checkpoint genes and their roles in predicting immunotherapy response. <em>Brief. Bioinform</em>, <strong>22</strong>(3), (2021).</cite> [<a href="https://doi.org/10.1093/bib/bbaa176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32814346/" class="usa-link">PubMed</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Haruehanroengra, P. et al. RNA modifications and cancer [J]. <em>RNA Biol.</em><strong>17</strong> (11), 1560–1575 (2020).
</cite> [<a href="https://doi.org/10.1080/15476286.2020.1722449" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7567502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31994439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haruehanroengra,%20P.%20et%20al.%20RNA%20modifications%20and%20cancer%20%5BJ%5D.%20RNA%20Biol.17%20(11),%201560%E2%80%931575%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Wang, S. et al. UCSCXenaShiny: an R/CRAN package for interactive analysis of UCSC Xena data [J]. <em>Bioinformatics</em><strong>38</strong> (2), 527–529 (2022).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btab561" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8723150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34323947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20S.%20et%20al.%20UCSCXenaShiny:%20an%20R/CRAN%20package%20for%20interactive%20analysis%20of%20UCSC%20Xena%20data%20%5BJ%5D.%20Bioinformatics38%20(2),%20527%E2%80%93529%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Kanehisa, M. et al. KEGG for taxonomy-based analysis of pathways and genomes [J]. <em>Nucleic Acids Res.</em><strong>51</strong> (D1), D587–d92 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkac963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825424/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36300620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa,%20M.%20et%20al.%20KEGG%20for%20taxonomy-based%20analysis%20of%20pathways%20and%20genomes%20%5BJ%5D.%20Nucleic%20Acids%20Res.51%20(D1),%20D587%E2%80%93d92%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>The Gene Ontology Resource. 20 years and still going strong [J]. <em>Nucleic Acids Res.</em><strong>47</strong> (D1), D330–d8 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gky1055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30395331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?The%20Gene%20Ontology%20Resource.%2020%20years%20and%20still%20going%20strong%20%5BJ%5D.%20Nucleic%20Acids%20Res.47%20(D1),%20D330%E2%80%93d8%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Huang, Y. H. et al. EXOSC5 maintains cancer stem cell activity in endometrial cancer by regulating the NTN4/integrin β1 signalling axis [J]. <em>Int. J. Biol. Sci.</em><strong>20</strong> (1), 265–279 (2024).
</cite> [<a href="https://doi.org/10.7150/ijbs.86275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10750274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38164180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20Y.%20H.%20et%20al.%20EXOSC5%20maintains%20cancer%20stem%20cell%20activity%20in%20endometrial%20cancer%20by%20regulating%20the%20NTN4/integrin%20%CE%B21%20signalling%20axis%20%5BJ%5D.%20Int.%20J.%20Biol.%20Sci.20%20(1),%20265%E2%80%93279%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Yang, E. et al. Exosome-mediated metabolic reprogramming: the emerging role in tumor microenvironment remodeling and its influence on cancer progression [J]. <em>Signal. Transduct. Target. Ther.</em><strong>5</strong> (1), 242 (2020).
</cite> [<a href="https://doi.org/10.1038/s41392-020-00359-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7572387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33077737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20E.%20et%20al.%20Exosome-mediated%20metabolic%20reprogramming:%20the%20emerging%20role%20in%20tumor%20microenvironment%20remodeling%20and%20its%20influence%20on%20cancer%20progression%20%5BJ%5D.%20Signal.%20Transduct.%20Target.%20Ther.5%20(1),%20242%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Meng, X., Yang, S. &amp; Camp, V. J. A. The Interplay Between the DNA Damage Response, RNA Processing and Extracellular Vesicles. <em>Front. Oncol.</em><strong>9</strong>, 1538. (2019).</cite> [<a href="https://doi.org/10.3389/fonc.2019.01538" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6978769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32010626/" class="usa-link">PubMed</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Garnis, C., Buys, T. P. &amp; Lam, W. L. Genetic alteration and gene expression modulation during cancer progression. <em>Mol Cancer,</em><strong>3</strong>, 9. (2004).</cite> [<a href="https://doi.org/10.1186/1476-4598-3-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC408463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15035667/" class="usa-link">PubMed</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Wickramasinghe, V. O. &amp; Venkitaraman, A. R. RNA processing and genome stability: cause and consequence [J]. <em>Mol. Cell.</em><strong>61</strong> (4), 496–505 (2016).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2016.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5905668/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26895423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wickramasinghe,%20V.%20O.%20&amp;%20Venkitaraman,%20A.%20R.%20RNA%20processing%20and%20genome%20stability:%20cause%20and%20consequence%20%5BJ%5D.%20Mol.%20Cell.61%20(4),%20496%E2%80%93505%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Ohguchi, Y. &amp; Ohguchi, H. DIS3: the enigmatic gene in multiple myeloma. <em>Int. J. Mol. Sci.</em>, <strong>24</strong>(4), (2023).</cite> [<a href="https://doi.org/10.3390/ijms24044079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9958658/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36835493/" class="usa-link">PubMed</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Peng, R. R. et al. The 5.8S pre-rRNA maturation factor, M-phase phosphoprotein 6, is a female fertility factor required for oocyte quality and meiosis [J]. <em>Cell. Prolif.</em><strong>53</strong> (3), e12769 (2020).
</cite> [<a href="https://doi.org/10.1111/cpr.12769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7106954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32003502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peng,%20R.%20R.%20et%20al.%20The%205.8S%20pre-rRNA%20maturation%20factor,%20M-phase%20phosphoprotein%206,%20is%20a%20female%20fertility%20factor%20required%20for%20oocyte%20quality%20and%20meiosis%20%5BJ%5D.%20Cell.%20Prolif.53%20(3),%20e12769%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Barbieri, I. &amp; Kouzarides, T. Role of RNA modifications in cancer [J]. <em>Nat. Rev. Cancer</em>. <strong>20</strong> (6), 303–322 (2020).
</cite> [<a href="https://doi.org/10.1038/s41568-020-0253-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32300195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barbieri,%20I.%20&amp;%20Kouzarides,%20T.%20Role%20of%20RNA%20modifications%20in%20cancer%20%5BJ%5D.%20Nat.%20Rev.%20Cancer.%2020%20(6),%20303%E2%80%93322%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Elmusrati, A., Wang, J. &amp; Wang, C. Y. Tumor microenvironment and immune evasion in head and neck squamous cell carcinoma [J]. <em>Int. J. Oral Sci.</em><strong>13</strong> (1), 24 (2021).
</cite> [<a href="https://doi.org/10.1038/s41368-021-00131-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8329257/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34341329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elmusrati,%20A.,%20Wang,%20J.%20&amp;%20Wang,%20C.%20Y.%20Tumor%20microenvironment%20and%20immune%20evasion%20in%20head%20and%20neck%20squamous%20cell%20carcinoma%20%5BJ%5D.%20Int.%20J.%20Oral%20Sci.13%20(1),%2024%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Wang, Z. et al. T cells, particularly activated CD4(+) cells, maintain anti-CD20-mediated NK cell viability and antibody dependent cellular cytotoxicity [J]. <em>Cancer Immunol. Immunother</em>. <strong>71</strong> (2), 237–249 (2022).
</cite> [<a href="https://doi.org/10.1007/s00262-021-02976-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8783893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34110453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Z.%20et%20al.%20T%20cells,%20particularly%20activated%20CD4(+)%20cells,%20maintain%20anti-CD20-mediated%20NK%20cell%20viability%20and%20antibody%20dependent%20cellular%20cytotoxicity%20%5BJ%5D.%20Cancer%20Immunol.%20Immunother.%2071%20(2),%20237%E2%80%93249%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Silva, R., Lopes, M. F. &amp; Travassos, L. H. Distinct T helper cell-mediated antitumor immunity: T helper 2 cells in focus [J]. <em>Cancer Pathog Ther.</em><strong>1</strong> (1), 76–86 (2023).
</cite> [<a href="https://doi.org/10.1016/j.cpt.2022.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10846313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38328613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Silva,%20R.,%20Lopes,%20M.%20F.%20&amp;%20Travassos,%20L.%20H.%20Distinct%20T%20helper%20cell-mediated%20antitumor%20immunity:%20T%20helper%202%20cells%20in%20focus%20%5BJ%5D.%20Cancer%20Pathog%20Ther.1%20(1),%2076%E2%80%9386%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Li, C. et al. Regulatory T cells in tumor microenvironment: new mechanisms, potential therapeutic strategies and future prospects [J]. <em>Mol. Cancer</em>. <strong>19</strong> (1), 116 (2020).
</cite> [<a href="https://doi.org/10.1186/s12943-020-01234-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7367382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32680511/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20C.%20et%20al.%20Regulatory%20T%20cells%20in%20tumor%20microenvironment:%20new%20mechanisms,%20potential%20therapeutic%20strategies%20and%20future%20prospects%20%5BJ%5D.%20Mol.%20Cancer.%2019%20(1),%20116%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Prager, I. &amp; Watzl, C. Mechanisms of natural killer cell-mediated cellular cytotoxicity [J]. <em>J. Leukoc. Biol.</em><strong>105</strong> (6), 1319–1329 (2019).
</cite> [<a href="https://doi.org/10.1002/JLB.MR0718-269R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31107565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prager,%20I.%20&amp;%20Watzl,%20C.%20Mechanisms%20of%20natural%20killer%20cell-mediated%20cellular%20cytotoxicity%20%5BJ%5D.%20J.%20Leukoc.%20Biol.105%20(6),%201319%E2%80%931329%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Hegde, S. et al. Dendritic cell paucity leads to dysfunctional immune surveillance in pancreatic cancer [J]. <em>Cancer Cell.</em><strong>37</strong> (3), 289–307e9 (2020).
</cite> [<a href="https://doi.org/10.1016/j.ccell.2020.02.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7181337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32183949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hegde,%20S.%20et%20al.%20Dendritic%20cell%20paucity%20leads%20to%20dysfunctional%20immune%20surveillance%20in%20pancreatic%20cancer%20%5BJ%5D.%20Cancer%20Cell.37%20(3),%20289%E2%80%93307e9%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Francis, D. M. et al. Blockade of immune checkpoints in lymph nodes through locoregional delivery augments cancer immunotherapy. <em>Sci. Transl Med.</em>, <strong>12</strong>(563), (2020).</cite> [<a href="https://doi.org/10.1126/scitranslmed.aay3575" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8377700/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32998971/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Andrews, L. P. et al. LAG-3 and PD-1 synergize on CD8(+) T cells to drive T cell exhaustion and hinder autocrine IFN-γ-dependent anti-tumor immunity [J]. <em>Cell</em><strong>187</strong> (16), 4355–72e22 (2024).
</cite> [<a href="https://doi.org/10.1016/j.cell.2024.07.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11323044/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39121848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andrews,%20L.%20P.%20et%20al.%20LAG-3%20and%20PD-1%20synergize%20on%20CD8(+)%20T%20cells%20to%20drive%20T%20cell%20exhaustion%20and%20hinder%20autocrine%20IFN-%CE%B3-dependent%20anti-tumor%20immunity%20%5BJ%5D.%20Cell187%20(16),%204355%E2%80%9372e22%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Baessler, A. &amp; Vignali, D. A. A. T cell exhaustion [J]. <em>Annu. Rev. Immunol.</em><strong>42</strong> (1), 179–206 (2024).
</cite> [<a href="https://doi.org/10.1146/annurev-immunol-090222-110914" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38166256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baessler,%20A.%20&amp;%20Vignali,%20D.%20A.%20A.%20T%20cell%20exhaustion%20%5BJ%5D.%20Annu.%20Rev.%20Immunol.42%20(1),%20179%E2%80%93206%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Deng, X. et al. The roles and implications of RNA m(6)A modification in cancer [J]. <em>Nat. Rev. Clin. Oncol.</em><strong>20</strong> (8), 507–526 (2023).
</cite> [<a href="https://doi.org/10.1038/s41571-023-00774-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37221357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deng,%20X.%20et%20al.%20The%20roles%20and%20implications%20of%20RNA%20m(6)A%20modification%20in%20cancer%20%5BJ%5D.%20Nat.%20Rev.%20Clin.%20Oncol.20%20(8),%20507%E2%80%93526%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Li, M. et al. 5-methylcytosine RNA methyltransferases and their potential roles in cancer [J]. <em>J. Transl Med.</em><strong>20</strong> (1), 214 (2022).
</cite> [<a href="https://doi.org/10.1186/s12967-022-03427-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9102922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35562754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20M.%20et%20al.%205-methylcytosine%20RNA%20methyltransferases%20and%20their%20potential%20roles%20in%20cancer%20%5BJ%5D.%20J.%20Transl%20Med.20%20(1),%20214%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Liu, Y. et al. Research progress of N1-methyladenosine RNA modification in cancer [J]. <em>Cell. Commun. Signal.</em><strong>22</strong> (1), 79 (2024).
</cite> [<a href="https://doi.org/10.1186/s12964-023-01401-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10826226/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38291517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20et%20al.%20Research%20progress%20of%20N1-methyladenosine%20RNA%20modification%20in%20cancer%20%5BJ%5D.%20Cell.%20Commun.%20Signal.22%20(1),%2079%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Monshaugen, I. et al. Depletion of the m1A writer TRMT6/TRMT61A reduces proliferation and resistance against cellular stress in bladder cancer. <em>Front. Oncol.</em><strong>13</strong>, 1334112. (2023).</cite> [<a href="https://doi.org/10.3389/fonc.2023.1334112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10830773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38304034/" class="usa-link">PubMed</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Dalhat, M. H. et al. Dissecting the oncogenic properties of essential RNA-modifying enzymes: a focus on NAT10 [J]. <em>Oncogene</em><strong>43</strong> (15), 1077–1086 (2024).
</cite> [<a href="https://doi.org/10.1038/s41388-024-02975-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11092965/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38409550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dalhat,%20M.%20H.%20et%20al.%20Dissecting%20the%20oncogenic%20properties%20of%20essential%20RNA-modifying%20enzymes:%20a%20focus%20on%20NAT10%20%5BJ%5D.%20Oncogene43%20(15),%201077%E2%80%931086%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Yang, Y. et al. NOP2 facilitates EZH2-mediated epithelial-mesenchymal transition by enhancing EZH2 mRNA stability via m5C methylation in lung cancer progression [J]. <em>Cell. Death Dis.</em><strong>15</strong> (7), 506 (2024).
</cite> [<a href="https://doi.org/10.1038/s41419-024-06899-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11252406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39013911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20Y.%20et%20al.%20NOP2%20facilitates%20EZH2-mediated%20epithelial-mesenchymal%20transition%20by%20enhancing%20EZH2%20mRNA%20stability%20via%20m5C%20methylation%20in%20lung%20cancer%20progression%20%5BJ%5D.%20Cell.%20Death%20Dis.15%20(7),%20506%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Bártová, E. &amp; Stixová, L. Svobodová Kovaříková A. N4-acetylcytidine and other RNA modifications in epitranscriptome: insight into DNA repair and cancer development [J]. <em>Epigenomics</em><strong>17</strong> (6), 411–422 (2025).
</cite> [<a href="https://doi.org/10.1080/17501911.2025.2473308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11980489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40040517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?B%C3%A1rtov%C3%A1,%20E.%20&amp;%20Stixov%C3%A1,%20L.%20Svobodov%C3%A1%20Kova%C5%99%C3%ADkov%C3%A1%20A.%20N4-acetylcytidine%20and%20other%20RNA%20modifications%20in%20epitranscriptome:%20insight%20into%20DNA%20repair%20and%20cancer%20development%20%5BJ%5D.%20Epigenomics17%20(6),%20411%E2%80%93422%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Dai, Y. et al. RNA methylation and breast cancer: insights into m6A, m7G and m5C [J]. <em>Mol. Biol. Rep.</em><strong>52</strong> (1), 27 (2024).
</cite> [<a href="https://doi.org/10.1007/s11033-024-10138-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39611867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dai,%20Y.%20et%20al.%20RNA%20methylation%20and%20breast%20cancer:%20insights%20into%20m6A,%20m7G%20and%20m5C%20%5BJ%5D.%20Mol.%20Biol.%20Rep.52%20(1),%2027%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Deo, P. N. &amp; Deshmukh, R. Pathophysiology of keratinization [J]. <em>J. Oral Maxillofac. Pathol.</em><strong>22</strong> (1), 86–91 (2018).
</cite> [<a href="https://doi.org/10.4103/jomfp.JOMFP_195_16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5917548/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29731562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deo,%20P.%20N.%20&amp;%20Deshmukh,%20R.%20Pathophysiology%20of%20keratinization%20%5BJ%5D.%20J.%20Oral%20Maxillofac.%20Pathol.22%20(1),%2086%E2%80%9391%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Lin, C. et al. Innovative organ-on-a-chip platforms for exploring tumorigenesis and therapy in head and neck cancer [J]. <em>J. Transl Med.</em><strong>23</strong> (1), 798 (2025).
</cite> [<a href="https://doi.org/10.1186/s12967-025-06824-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12269115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40671128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20C.%20et%20al.%20Innovative%20organ-on-a-chip%20platforms%20for%20exploring%20tumorigenesis%20and%20therapy%20in%20head%20and%20neck%20cancer%20%5BJ%5D.%20J.%20Transl%20Med.23%20(1),%20798%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Liu, S. et al. 5-Hydroxymethylation highlights the heterogeneity in keratinization and cell junctions in head and neck cancers [J]. <em>Clin. Epigenetics</em>. <strong>12</strong> (1), 175 (2020).
</cite> [<a href="https://doi.org/10.1186/s13148-020-00965-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7672859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33203436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20S.%20et%20al.%205-Hydroxymethylation%20highlights%20the%20heterogeneity%20in%20keratinization%20and%20cell%20junctions%20in%20head%20and%20neck%20cancers%20%5BJ%5D.%20Clin.%20Epigenetics.%2012%20(1),%20175%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Elias, P. M. et al. Formation and functions of the corneocyte lipid envelope (CLE) [J]. <em>Biochim. Biophys. Acta</em>. <strong>1841</strong> (3), 314–318 (2014).
</cite> [<a href="https://doi.org/10.1016/j.bbalip.2013.09.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3943821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24076475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elias,%20P.%20M.%20et%20al.%20Formation%20and%20functions%20of%20the%20corneocyte%20lipid%20envelope%20(CLE)%20%5BJ%5D.%20Biochim.%20Biophys.%20Acta.%201841%20(3),%20314%E2%80%93318%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Candi, E., Schmidt, R. &amp; Melino, G. The cornified envelope: a model of cell death in the skin [J]. <em>Nat. Rev. Mol. Cell. Biol.</em><strong>6</strong> (4), 328–340 (2005).
</cite> [<a href="https://doi.org/10.1038/nrm1619" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15803139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Candi,%20E.,%20Schmidt,%20R.%20&amp;%20Melino,%20G.%20The%20cornified%20envelope:%20a%20model%20of%20cell%20death%20in%20the%20skin%20%5BJ%5D.%20Nat.%20Rev.%20Mol.%20Cell.%20Biol.6%20(4),%20328%E2%80%93340%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Sell, S. et al. Cancer: A problem of developmental biology; scientific evidence for reprogramming and differentiation therapy [J]. <em>Curr. Drug Targets</em>. <strong>17</strong> (10), 1103–1110 (2016).
</cite> [<a href="https://doi.org/10.2174/1389450116666150907102717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26343109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sell,%20S.%20et%20al.%20Cancer:%20A%20problem%20of%20developmental%20biology;%20scientific%20evidence%20for%20reprogramming%20and%20differentiation%20therapy%20%5BJ%5D.%20Curr.%20Drug%20Targets.%2017%20(10),%201103%E2%80%931110%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Nakaya, Y. &amp; Sheng, G. EMT in developmental morphogenesis [J]. <em>Cancer Lett.</em><strong>341</strong> (1), 9–15 (2013).
</cite> [<a href="https://doi.org/10.1016/j.canlet.2013.02.037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23462225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakaya,%20Y.%20&amp;%20Sheng,%20G.%20EMT%20in%20developmental%20morphogenesis%20%5BJ%5D.%20Cancer%20Lett.341%20(1),%209%E2%80%9315%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Elhamamsy, A. R. et al. Ribosome biogenesis: A central player in cancer metastasis and therapeutic resistance [J]. <em>Cancer Res.</em><strong>82</strong> (13), 2344–2353 (2022).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-21-4087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9256764/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35303060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elhamamsy,%20A.%20R.%20et%20al.%20Ribosome%20biogenesis:%20A%20central%20player%20in%20cancer%20metastasis%20and%20therapeutic%20resistance%20%5BJ%5D.%20Cancer%20Res.82%20(13),%202344%E2%80%932353%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Genovese, I. et al. Mitochondria: insights into crucial features to overcome cancer chemoresistance [J]. <em>Int. J. Mol. Sci.</em>, <strong>22</strong>(9), (2021).</cite> [<a href="https://doi.org/10.3390/ijms22094770" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8124268/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33946271/" class="usa-link">PubMed</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Morton, D. J. et al. The RNA exosome and RNA exosome-linked disease [J]. <em>Rna</em><strong>24</strong> (2), 127–142 (2018).
</cite> [<a href="https://doi.org/10.1261/rna.064626.117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5769741/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29093021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morton,%20D.%20J.%20et%20al.%20The%20RNA%20exosome%20and%20RNA%20exosome-linked%20disease%20%5BJ%5D.%20Rna24%20(2),%20127%E2%80%93142%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Bradley, R. K. &amp; Anczuków, O. RNA splicing dysregulation and the hallmarks of cancer [J]. <em>Nat. Rev. Cancer</em>. <strong>23</strong> (3), 135–155 (2023).
</cite> [<a href="https://doi.org/10.1038/s41568-022-00541-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10132032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36627445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bradley,%20R.%20K.%20&amp;%20Anczuk%C3%B3w,%20O.%20RNA%20splicing%20dysregulation%20and%20the%20hallmarks%20of%20cancer%20%5BJ%5D.%20Nat.%20Rev.%20Cancer.%2023%20(3),%20135%E2%80%93155%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Jiao, L. et al. Ribosome biogenesis in disease: new players and therapeutic targets [J]. <em>Signal. Transduct. Target. Ther.</em><strong>8</strong> (1), 15 (2023).
</cite> [<a href="https://doi.org/10.1038/s41392-022-01285-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9826790/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36617563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiao,%20L.%20et%20al.%20Ribosome%20biogenesis%20in%20disease:%20new%20players%20and%20therapeutic%20targets%20%5BJ%5D.%20Signal.%20Transduct.%20Target.%20Ther.8%20(1),%2015%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Zhang, Y. et al. Alternative splicing and cancer: a systematic review [J]. <em>Signal. Transduct. Target. Ther.</em><strong>6</strong> (1), 78 (2021).
</cite> [<a href="https://doi.org/10.1038/s41392-021-00486-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7902610/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33623018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20Alternative%20splicing%20and%20cancer:%20a%20systematic%20review%20%5BJ%5D.%20Signal.%20Transduct.%20Target.%20Ther.6%20(1),%2078%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Ding, L. et al. Mutations in DNA binding domain of p53 impede RSL1D1-p53 interaction to escape from degradation in human colorectal cancer cells [J]. <em>Exp. Cell. Res.</em><strong>417</strong> (1), 113211 (2022).
</cite> [<a href="https://doi.org/10.1016/j.yexcr.2022.113211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35597299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ding,%20L.%20et%20al.%20Mutations%20in%20DNA%20binding%20domain%20of%20p53%20impede%20RSL1D1-p53%20interaction%20to%20escape%20from%20degradation%20in%20human%20colorectal%20cancer%20cells%20%5BJ%5D.%20Exp.%20Cell.%20Res.417%20(1),%20113211%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Dong, Y. et al. CCTs as new biomarkers for the prognosis of head and neck squamous cancer [J]. <em>Open. Med. (Wars)</em>. <strong>15</strong> (1), 672–688 (2020).
</cite> [<a href="https://doi.org/10.1515/med-2020-0114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7706129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33313411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong,%20Y.%20et%20al.%20CCTs%20as%20new%20biomarkers%20for%20the%20prognosis%20of%20head%20and%20neck%20squamous%20cancer%20%5BJ%5D.%20Open.%20Med.%20(Wars).%2015%20(1),%20672%E2%80%93688%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Britton, R. A. et al. Isolation and preliminary characterization of the human and mouse homologues of the bacterial cell cycle gene era [J]. <em>Genomics</em><strong>67</strong> (1), 78–82 (2000).
</cite> [<a href="https://doi.org/10.1006/geno.2000.6243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10945472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Britton,%20R.%20A.%20et%20al.%20Isolation%20and%20preliminary%20characterization%20of%20the%20human%20and%20mouse%20homologues%20of%20the%20bacterial%20cell%20cycle%20gene%20era%20%5BJ%5D.%20Genomics67%20(1),%2078%E2%80%9382%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (156.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (156.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (157.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 9</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 10</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 11</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 12</a><sup> (157.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM13_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 13</a><sup> (275.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material14_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM14_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 14</a><sup> (129KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material15_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM15_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 15</a><sup> (148.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material16_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM16_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 16</a><sup> (188.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material17_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM17_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 17</a><sup> (6.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material18_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM18_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 18</a><sup> (6.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material19_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM19_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 19</a><sup> (6.7MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material20_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM20_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 20</a><sup> (6.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material21_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM21_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 21</a><sup> (6.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material22_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM22_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 22</a><sup> (6.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material23_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM23_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 23</a><sup> (6.9MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material24_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM24_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 24</a><sup> (6.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material25_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM25_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 25</a><sup> (6.9MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material26_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM26_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 26</a><sup> (6.9MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material27_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM27_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 27</a><sup> (137.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material28_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM28_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 28</a><sup> (137.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material29_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM29_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 29</a><sup> (137.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material30_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM30_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 30</a><sup> (136.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material31_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM31_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 31</a><sup> (137.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material32_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM32_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 32</a><sup> (136.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material33_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM33_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 33</a><sup> (137.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material34_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM34_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 34</a><sup> (136.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material35_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM35_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 35</a><sup> (137.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material36_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM36_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 36</a><sup> (134.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material37_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM37_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 37</a><sup> (306.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material38_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM38_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 38</a><sup> (22KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material39_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365086/bin/41598_2025_15758_MOESM39_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 39</a><sup> (12.3KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All supporting data are provided in the supplementary files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-15758-3"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_15758.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (10.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365086/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365086/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365086%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365086/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365086/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365086/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830179/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365086/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830179/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365086/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365086/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="KiS2Jhi0ysHTPe806Rw1ZT5oO5PnVajIz93MQfeGFtisGAAXt5DS2xCwyQiQi8se">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
