
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Characterization and transcriptomic analysis provide novel insights into the sexual differentiation of the dioecious plant Bama Huoma (Cannabis sativa) - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152532258AF15213053225005973052E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcps">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369072/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Plant Biology">
<meta name="citation_title" content="Characterization and transcriptomic analysis provide novel insights into the sexual differentiation of the dioecious plant Bama Huoma (Cannabis sativa)">
<meta name="citation_author" content="Xiaofang Liao">
<meta name="citation_author_institution" content="Cash Crop Institute of Guangxi Academy of Agricultural Sciences, Guangxi Crop Genetic Improvement and Biotechnology Laboratory, Nanning, Guangxi 530007 China">
<meta name="citation_author" content="Jie Zheng">
<meta name="citation_author_institution" content="National Key Laboratory of Cotton Biobreeding and Integrated Utilization, Institute of Cotton Research, Chinese Academy of Agricultural Sciences (ICR, CAAS), Anyang, Henan 455000 China">
<meta name="citation_author" content="Wenhuan Hou">
<meta name="citation_author_institution" content="Cash Crop Institute of Guangxi Academy of Agricultural Sciences, Guangxi Crop Genetic Improvement and Biotechnology Laboratory, Nanning, Guangxi 530007 China">
<meta name="citation_author" content="Xingfu Tang">
<meta name="citation_author_institution" content="Cash Crop Institute of Guangxi Academy of Agricultural Sciences, Guangxi Crop Genetic Improvement and Biotechnology Laboratory, Nanning, Guangxi 530007 China">
<meta name="citation_author" content="Bangbang Huang">
<meta name="citation_author_institution" content="College of Agriculture, Guangxi University, Nanning, Guangxi 530001 China">
<meta name="citation_author" content="Jie Tan">
<meta name="citation_author_institution" content="Bama Yao Autonomous County Agriculture and Rural Bureau, Hechi, Guangxi 547500 China">
<meta name="citation_author" content="Yanhong Zhao">
<meta name="citation_author_institution" content="Cash Crop Institute of Guangxi Academy of Agricultural Sciences, Guangxi Crop Genetic Improvement and Biotechnology Laboratory, Nanning, Guangxi 530007 China">
<meta name="citation_author" content="Kailiang Teng">
<meta name="citation_author_institution" content="College of Artificial Intelligence, Guangxi University for Nationalities, Nanning, Guangxi 530007 China">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="1108">
<meta name="citation_doi" content="10.1186/s12870-025-07096-3">
<meta name="citation_pmid" content="40841596">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369072/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369072/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369072/pdf/12870_2025_Article_7096.pdf">
<meta name="description" content="Bama Huoma is a high-value cash crop because its seeds are known as the “holy seeds of longevity”. However, its seed yield is restricted by dioecism, and information about flower sex differentiation in this plant is lacking. In this study, ...">
<meta name="og:title" content="Characterization and transcriptomic analysis provide novel insights into the sexual differentiation of the dioecious plant Bama Huoma (Cannabis sativa)">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Bama Huoma is a high-value cash crop because its seeds are known as the “holy seeds of longevity”. However, its seed yield is restricted by dioecism, and information about flower sex differentiation in this plant is lacking. In this study, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369072/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12369072">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12870-025-07096-3"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12870_2025_Article_7096.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369072%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12369072/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12369072/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369072/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcps.png" alt="BMC Plant Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Plant Biology" title="Link to BMC Plant Biology" shape="default" href="https://bmcplantbiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Plant Biol</button></div>. 2025 Aug 21;25:1108. doi: <a href="https://doi.org/10.1186/s12870-025-07096-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12870-025-07096-3</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Plant%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Characterization and transcriptomic analysis provide novel insights into the sexual differentiation of the dioecious plant Bama Huoma (Cannabis sativa)</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Xiaofang Liao</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xiaofang Liao</span></h3>
<div class="p">
<sup>1</sup>Cash Crop Institute of Guangxi Academy of Agricultural Sciences, Guangxi Crop Genetic Improvement and Biotechnology Laboratory, Nanning, Guangxi 530007 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaofang Liao</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jie Zheng</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jie Zheng</span></h3>
<div class="p">
<sup>4</sup>National Key Laboratory of Cotton Biobreeding and Integrated Utilization, Institute of Cotton Research, Chinese Academy of Agricultural Sciences (ICR, CAAS), Anyang, Henan 455000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jie Zheng</span></a>
</div>
</div>
<sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hou%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Wenhuan Hou</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Wenhuan Hou</span></h3>
<div class="p">
<sup>1</sup>Cash Crop Institute of Guangxi Academy of Agricultural Sciences, Guangxi Crop Genetic Improvement and Biotechnology Laboratory, Nanning, Guangxi 530007 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hou%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenhuan Hou</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Xingfu Tang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Xingfu Tang</span></h3>
<div class="p">
<sup>1</sup>Cash Crop Institute of Guangxi Academy of Agricultural Sciences, Guangxi Crop Genetic Improvement and Biotechnology Laboratory, Nanning, Guangxi 530007 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xingfu Tang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Bangbang Huang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Bangbang Huang</span></h3>
<div class="p">
<sup>3</sup>College of Agriculture, Guangxi University, Nanning, Guangxi 530001 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bangbang Huang</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tan%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jie Tan</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jie Tan</span></h3>
<div class="p">
<sup>5</sup>Bama Yao Autonomous County Agriculture and Rural Bureau, Hechi, Guangxi 547500 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tan%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jie Tan</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Yanhong Zhao</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Yanhong Zhao</span></h3>
<div class="p">
<sup>1</sup>Cash Crop Institute of Guangxi Academy of Agricultural Sciences, Guangxi Crop Genetic Improvement and Biotechnology Laboratory, Nanning, Guangxi 530007 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yanhong Zhao</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Teng%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Kailiang Teng</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Kailiang Teng</span></h3>
<div class="p">
<sup>2</sup>College of Artificial Intelligence, Guangxi University for Nationalities, Nanning, Guangxi 530007 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Teng%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kailiang Teng</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Cash Crop Institute of Guangxi Academy of Agricultural Sciences, Guangxi Crop Genetic Improvement and Biotechnology Laboratory, Nanning, Guangxi 530007 China </div>
<div id="Aff2">
<sup>2</sup>College of Artificial Intelligence, Guangxi University for Nationalities, Nanning, Guangxi 530007 China </div>
<div id="Aff3">
<sup>3</sup>College of Agriculture, Guangxi University, Nanning, Guangxi 530001 China </div>
<div id="Aff4">
<sup>4</sup>National Key Laboratory of Cotton Biobreeding and Integrated Utilization, Institute of Cotton Research, Chinese Academy of Agricultural Sciences (ICR, CAAS), Anyang, Henan 455000 China </div>
<div id="Aff5">
<sup>5</sup>Bama Yao Autonomous County Agriculture and Rural Bureau, Hechi, Guangxi 547500 China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jan 17; Accepted 2025 Jul 18; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12369072  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841596/" class="usa-link">40841596</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Bama Huoma is a high-value cash crop because its seeds are known as the “holy seeds of longevity”. However, its seed yield is restricted by dioecism, and information about flower sex differentiation in this plant is lacking.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">In this study, anatomical analysis and dynamic transcriptome profiling were performed to explore the mechanism of sex differentiation in Bama Huoma. The results revealed that female and male flowers undergo independent developmental processes and that sex differentiation occurs at an early stage before flower development. Illumina sequencing revealed that MADS-box genes play crucial roles in pistil and stamen development. qRT‒PCR and RNA in situ hybridization were subsequently performed to determine the spatiotemporal expression patterns of genes related to sex determination. The results revealed that eight MADS-box genes and two MYB genes were expressed in a sex-biased and tissue-specific manner in male or female flowers. Among these genes, MYB35, PMADS2, TM6 and AGL104 were specifically expressed in male flowers and found to regulate callose metabolism, pollen sac formation, tapetum development, and microspore maturation. In contrast, AGL11, FBP24, AGL16 and two CAL were restricted to female flowers, where they regulate pistil formation, ovary development, and stigma differentiation. These genes play distinct and essential roles in the sex differentiation of Bama Huoma, as well as in the development of floral organs.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par3">Our findings provide a comprehensive gene expression profile of flower sex differentiation in Bama Huoma as well as new clues and information for further study in this field.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12870-025-07096-3.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Bama Huoma (Cannabis sativa subsp. sativa), Dioecism, Sex differentiation, MADS-box genes, Transcriptional regulation</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par4">Plant dioecism is a phenomenon in which female and male flowers grow on different individuals. This phenomenon is widespread in plants, occurring in approximately 10% of flowering plants [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Dioecism provides not only favorable conditions for promoting cross-pollination and heterosis utilization in crops but also ideal materials for investigating the mechanisms involved in sex expression and the origin and evolution of sex-determining regions (SDRs) and sex chromosomes. SDRs and the evolution of dioecy have been studied in many species [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>], and the mechanisms of sex determination are complex and diverse in dioecious plants. In diploid persimmon, dioecism is determined by two genes, MeGI and OGI. The Y-encoded candidate sex determinant OGI displays male-specific conservation among Diospyros species and encodes a small RNA that targets the autosomal MeGI gene, resulting in separate male and female individuals [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. However, in the hexaploid persimmon Diospyros kaki, developing male buds exhibit higher methylation levels across the MeGI promoter than developing female flowers do, and a DNA methylation inhibitor induced developing male buds to form feminized flowers [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Cytokinin is an essential plant hormone and is known to be involved in numerous plant growth and developmental processes [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Previous studies have revealed that the cytokinin signaling pathway is important for sex determination in several species [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. The Y-encoded sex-determining candidate gene SyGI in kiwifruit is a type-C cytokinin response regulator and suppresses carpel development [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. The sex-determining candidate genes LOG in date palm trees and APRT3 in grapes are also involved in the inactivation of cytokinin [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Additionally, many species have sex determination systems that are regulated by epigenetic mechanisms, including the modification of genomic loci in males and females at the transcriptional level, by histone modification or DNA modification, or at the post-transcriptional level by RNA-mediated regulation [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. In melon, the integration of transcriptional control between the sex-specific genes ACS7, ACS11, and WIP1 through genetic interactions as well as the recruitment of H3K9ac and H3K27me3 epigenetically determines sex expression overall [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. In maize, sex determination is regulated by noncoding small RNAs through DNA methylation to restrict the function of the pistil-protecting factor silkless1 [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>], and sex expression in cucumber is regulated by methylation under different light and temperature conditions [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>].</p>
<p id="Par5">Nevertheless, the sex determination mechanisms of dioecious species are complex and diverse, and the genes involved in sex determination in dioecious species are still being debated. In recent years, RNA sequencing (RNA-seq) has emerged as an effective method for studying flower development and sex determination, and SDRs and genes have been identified in numerous plant species [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Previous studies have shown that transcription factors (TFs), such as MYB-type, MADS-box, NAC, WRKY, and bHLH TFs, play crucial roles in sex determination in dioecious plants [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. For example, MADS-box family TFs are critical for sex determination in plants: ZaMADS70 interacts with ZaMADS42 and ZaMADS48 to facilitate the loss of petals in Zanthoxylum armatum, and the ZaMADS92/ZaMADS40 heterodimer could be responsible for accelerating flowering [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. In litchi, the MADS-box gene LcMADS51 may positively regulate carpel formation, whereas six MADS-box genes, including LcMADS42/46/47/75/93/100, may play roles in stamen development [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. In cucumber, two MADS-box genes, AGAMOUS and CAULIFLOWER A, are regulated by promoter methylation to mediate the photoperiod-dependent regulation of sex expression [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. In Solanum lycopersicum, the TM8 gene, which shares homology with CAL (LOC115704295) in hemp, is hypothesized to function as a negative regulator of anther development [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. CsTM8 (LOC115704295) is considered a potential target gene downstream of genes involved in female flower development in hemp [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. In Asparagus officinalis, the male-specific gene MYB35 is tightly linked with the Y chromosome and is indispensable for tapetal cell development; this gene is specifically expressed in males and has lost its function on the X chromosome [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>].</p>
<p id="Par6">Bama Huoma (Cannabis sativa subsp. sativa), belonging to the Cannabis genus of the Cannabaceae family, is an annual herbaceous plant. This plant is widely grown in Bama Yao Autonomous County, Guangxi Province, China, a famous longevity-associated region worldwide. The seeds of this plant are commonly used as a traditional medicine and food and are considered potential contributors to the longevity and health of indigenous people. The dehulled seeds of Bama Huoma are not only rich in vegetable protein, phosphatidylinositol, and essential trace elements such as selenium, zinc, manganese, and germanium, but also contain α-linolenic acid-often referred to as “plant brain gold” ‒which has been shown to inhibit allergic responses and exhibit antioxidant, anti-inflammatory, and antibacterial properties [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. These seeds present major market prospects for applications in medicine and food. As a typical dioecious plant, sex determination in Bama Huoma is controlled by the X and Y chromosomes, and the female and male karyotypes are XX and XY, respectively [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Male plants are unable to produce seeds, resulting in a dramatic decline in the yield of Bama Huoma seeds. For this reason, elucidating the molecular genetic mechanism of sex differentiation in Bama Huoma will provide insights for targeted breeding of all-female varieties and increasing seed yield. However, little is known about the molecular basis and regulatory mechanisms underlying sex differentiation in Bama Huoma.</p>
<p id="Par7">In this study, for elucidation of the flower development process, the anatomical structures of male and female flowers were observed at different developmental stages. Whole-transcriptome strand-specific RNA-seq analysis was employed to identify the candidate genes involved in sex determination in Bama Huoma. Moreover, for confirmation of the reliability of the sequencing results, several differentially expressed genes (DEGs) were selected for quantitative real-time PCR (qRT‒PCR) verification, and tissue-specific expression profiles of candidate sex determination genes in female and male plants were investigated. The results lay a foundation for further research on the genetic mechanisms of sex differentiation and provide a theoretical basis for increasing Bama Huoma seed yield.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Plant materials</h3>
<p id="Par8">Bama Huoma (C. sativa subsp. sativa) is a locally cultivated species in Bama Yao Autonomous County, Guangxi, China. It is a typical dioecious plant, and its sex determination is controlled by the X and Y chromosomes [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. The flower buds undergo differentiation into male and female flowers during the flowering process (Fig. S1). In the present study, Bama Huoma seeds were sown in a field at the Guangxi Academy of Agricultural Sciences in Nanning, China (22°50′N, 108°14′E). Each plot covered an area of 48 m2 (32 m × 1.5 m), with a row spacing of 45 cm and plant spacing of 15 cm, and was randomly replicated three times. The total number of individual male and female plants in each replicate was counted after flowering (Table <a href="#Tab1" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Separation ratios of female and male Bama Huoma plants in each duplicate</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">Female plants</th>
<th align="left" colspan="1" rowspan="1">Male plants</th>
<th align="left" colspan="1" rowspan="1">Sex ratio</th>
<th align="left" colspan="1" rowspan="1">Theoretical ratio</th>
<th align="left" colspan="1" rowspan="1">
<em>χ</em><sup><em>2</em></sup>
</th>
<th align="left" colspan="1" rowspan="1">
<em>P </em>value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Duplicate 1</td>
<td align="left" colspan="1" rowspan="1">410</td>
<td align="left" colspan="1" rowspan="1">361</td>
<td align="left" colspan="1" rowspan="1">1.14</td>
<td align="left" colspan="1" rowspan="1">1:1</td>
<td align="left" colspan="1" rowspan="1">2.99</td>
<td align="left" colspan="1" rowspan="1">0.078</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Duplicate 2</td>
<td align="left" colspan="1" rowspan="1">318</td>
<td align="left" colspan="1" rowspan="1">277</td>
<td align="left" colspan="1" rowspan="1">1.15</td>
<td align="left" colspan="1" rowspan="1">1:1</td>
<td align="left" colspan="1" rowspan="1">2.69</td>
<td align="left" colspan="1" rowspan="1">0.093</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Duplicate 3</td>
<td align="left" colspan="1" rowspan="1">360</td>
<td align="left" colspan="1" rowspan="1">330</td>
<td align="left" colspan="1" rowspan="1">1.09</td>
<td align="left" colspan="1" rowspan="1">1:1</td>
<td align="left" colspan="1" rowspan="1">1.31</td>
<td align="left" colspan="1" rowspan="1">0.253</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Morphological and cytological observations</h3>
<p id="Par9">Bama Huoma is characterized by an infinite inflorescence. The morphology of the female and male flower buds of Bama Huoma at various developmental stages was observed under a stereomicroscope (Olympus, Tokyo, Japan) during the full-bloom stage. Images of the pistillate inflorescence and staminate inflorescence were captured with a digital camera (Canon, Tokyo, Japan). For characterization of the developmental processes of flower buds, floral buds of varying sizes were separately collected from female and male plants. The female floral buds were collected at stages with immature stigmas, protruding stigmas, and two unequally elongated stigmata. The male floral buds were collected and sorted according to their diameter: &lt; 0.5 mm, 0.5–1 mm, 1–1.5 mm, 1.5–2 mm, 2–3 mm, 3–4 mm and &gt; 4 mm. All of the buds were fixed with Carnoy’s fixation solution (ethanol: acetic acid = 3:1) for 24 h at 4 ℃. The fixed floral buds were further dehydrated with an ethanol gradient (75%, 80%, 85%, 90%, 95%, and 100%), with 1 h per wash. The dehydrated floral buds were embedded in paraffin wax. After one month, serial sections with a thickness of 10 μm were prepared with a Leica microtome. The sectioned floral buds were then mounted on glass slides, stained with 1% safranin, and observed and photographed with a Carl Zeiss Imager M2 microscope (Zeiss, Germany).</p>
<p id="Par10">Female and male floral buds at different developmental stages were also collected for electron microscopy. The samples were fixed in 2.5% glutaraldehyde for 12 h at 4 °C, washed with 0.1 M PBS (pH 7.4) 3 times and then postfixed in the dark with a 1% (w/v) aqueous solution of OsO4 in 0.1 M PB (pH 7.4) for 2 h at room temperature. After being washed three times with PBS (pH 7.4), the samples were dehydrated in an ethanol gradient (30%, 50%, 70%, 80%, 95%, and 100%), with 20 min per wash, and embedded in EMbed 812 resin (90,529–77–4, SPI, USA). The floral buds were cut into 60-nm thin sections on an ultramicrotome, collected on 150 mesh copper grids, stained with a 2% uranium acetate saturated alcohol solution in the dark for 8 min, rinsed in 70% ethanol three times and then rinsed in ultrapure water three times. The sections were subsequently stained with 2.6% lead citrate for 8 min and rinsed with ultrapure water three times. After drying with filter paper, the copper grids were placed onto the grid board and dried overnight at room temperature. The copper grids were observed and imaged with a Hitachi HT7800 transmission electron microscope (Hitachi, Tokyo, Japan). The female and male flower buds at various developmental stages were prepared for scanning electron microscopy (SEM) following the techniques of Ashfaq et al. [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. The acetolyzed pollen grains were suspended in 90% ethanol and then placed on stubs. The samples were sputter-coated with gold and then viewed and photographed using a Hitachi SU8100 scanning electron microscope (Hitachi, Tokyo, Japan) by the Wuhan Servicebio Technology Company.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Sample collection, RNA isolation, cDNA library construction, and RNA-seq</h3>
<p id="Par11">According to cytological observations, the development of female flowers can be divided into three stages: initial differentiation of the flower bud (T1 buds of female flowers, T1F), formation and development of bract and pistil primordia (T2 buds of female flowers, T2F), and formation and development of ovule primordia (T3 buds of female flowers, T3F). The development of male flowers can be divided into three stages: initial differentiation of the male flower bud (T1 buds of male flowers, T1M), formation and development of brace primordia and stamens (T2 buds of male flowers, T2M), and formation of anthers and pollen grains (T3 buds of male flowers, T3M). For determination of the expression levels of genes, flower buds at developmental stages T1 and T3 from females and males of Bama Huoma were collected individually, followed by rapid freezing in liquid nitrogen. The samples were subsequently stored at −80 ℃ until total RNA isolation for RNA-seq analysis. Three replicates of each sample were obtained.</p>
<p id="Par12">Total RNA from each sample was isolated and purified using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer’s instructions. The RNA content and purity of each sample were quantified using a NanoDrop ND-1000 spectrophotometer (NanoDrop, Wilmington, DE, USA). RNA integrity was assessed with an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, California, USA) on the basis of an RNA integrity number (RIN) &gt; 7.0. DNase I was used to degrade double-stranded and single-stranded DNA contaminants within the RNA samples. The RNA samples were ultimately divided into two groups for library construction and RNA-seq validation. Approximately 5 μg of total RNA was subjected to ribosomal RNA depletion according to the manufacturer’s instructions for the Ribo-Zero™ rRNA Removal Kit (Illumina, San Diego, USA). After ribosomal RNA was removed, the remaining RNA was fragmented into small pieces using divalent cations at high temperature. Then, the cleaved RNA fragments were reverse-transcribed to cDNA with reverse transcription reagent (Invitrogen, Carlsbad, CA, USA), which was subsequently used to synthesize U-labeled second-strand DNA with Escherichia coli DNA polymerase I, RNase H and dUTP. An A-base was then added to the blunt ends of each strand, preparing them for ligation to the indexed adapters. Each adapter contained a T-base overhang for ligation to the A-tailed fragmented DNA. Single- or dual-index adapters were ligated to the fragments, and size selection was performed with AMPure XP beads. After heat-labile UDG enzyme treatment of the U-labeled second-strand DNA, the ligated products were amplified by PCR under the following conditions: initial denaturation at 95 °C for 3 min; eight cycles of denaturation at 98 °C for 15 s, annealing at 60 °C for 15 s, and extension at 72 °C for 30 s; and a final extension at 72 °C for 5 min. The average insert size for the final cDNA library was 300 bp (± 50 bp). Finally, we performed paired-end sequencing on the Illumina NovaSeq™ 6000 platform (LC Sciences, Hangzhou, China) following the vendor’s recommended protocol.</p></section><section id="Sec6"><h3 class="pmc_sec_title">De novo assembly and functional annotation</h3>
<p id="Par13">For clean reads, the raw reads that contained adapter contaminants, low-quality bases and undetermined bases were removed according to the Cutadapt method [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. The sequence quality was subsequently verified using FastQC (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>). In addition, the percentages of reads with Q-scores of Q20 and Q30 and the GC contents and sequence duplication levels of the clean reads were also estimated. The clean paired-end RNA-seq reads (150 nucleotides) were then aligned to the C. sativa reference genomes obtained from the NCBI database (<a href="https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_029168945.1/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_029168945.1/</a>) using Bowtie2 [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] and HISAT2 [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. The mapped reads of each sample were assembled using StringTie [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. All the transcriptomes of the Bama Huoma samples were subsequently merged to reconstruct a comprehensive transcriptome using Perl scripts. The annotation of gene functions among the assembled transcripts was then performed on the basis of searches of multiple databases, including the Gene Ontology (GO) (<a href="http://www.geneontology.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.geneontology.org</a>), KEGG (<a href="http://www.genome.jp/kegg/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.genome.jp/kegg/</a>), and KO (<a href="https://www.kegg.jp/kegg/ko" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.kegg.jp/kegg/ko</a>.) databases, and EC numbers were obtained from the BRENDA enzyme database (<a href="http://www.brenda-enzymes.info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.brenda-enzymes.info</a>).</p></section><section id="Sec7"><h3 class="pmc_sec_title">mRNA differential expression analysis and GO enrichment analyses</h3>
<p id="Par14">After the final transcriptome was generated, StringTie [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] was used to determine the expression levels of mRNAs by calculating fragments per kilobase per million reads (FPKM) values [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. The differentially expressed mRNAs were selected on the basis of a log2 [fold change (FC)] &gt; 1 or log2 (FC) &lt; −1 and statistical significance (<em>p</em> value &lt; 0.05) using the R package edgeR [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Genes showing specific expression only in female or male flower buds were identified by creating a Venn diagram of all DEGs at the two developmental stages for female and male flowers of Bama Huoma, and the corresponding GO enrichment analysis of the DEGs was performed using OmicStudio tools at <a href="https://www.omicstudio.cn/tool" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.omicstudio.cn/tool</a>. By comparison with the genome background, gene numbers for every GO term were calculated, and a hypergeometric test was then performed to identify significantly enriched GO terms.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Phylogenetic tree, conserved motif, and protein domain structure analysis</h3>
<p id="Par15">The open reading frames (ORFs) and amino acids associated with sex differentiation in this study were predicted via the online ORF Finder software (<a href="https://www.ncbi.nlm.nih.gov/orffinder/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/orffinder/</a>). The orthologous protein information was obtained through pairwise sequence alignment using the BLASTp tool (<a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://blast.ncbi.nlm.nih.gov/Blast.cgi</a>) with default parameters. A phylogenetic tree was constructed via the maximum likelihood method with MEGA X software on the basis of the JTT matrix-based model [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. The conserved motifs were analyzed using the online program Multiple Em for Motif Elicitation (MEME), which is available at <a href="https://meme-suite.org/meme/tools/meme" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://meme-suite.org/meme/tools/meme</a>. The parameters were set as follows: a maximum of 10 motifs, a minimum width of 6, and a maximum width of 50. The conserved domains were analyzed using the NCBI Conserved Domain Database (CDD), which is available at <a href="https://www.ncbi.nlm.nih.gov/Structure/bwrpsb/bwrpsb.cgi" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/Structure/bwrpsb/bwrpsb.cgi</a>. Finally, the phylogenetic tree, protein domain, structure, and conserved motifs were visualized using TBtools software [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>].</p></section><section id="Sec9"><h3 class="pmc_sec_title">Synthesis of cDNA and qRT‒PCR analysis</h3>
<p id="Par16">Total RNA from the same samples employed for the RNA-seq analysis and from different tissues and development periods of flower buds was extracted using a modified CTAB method [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. One microgram of total RNA was subjected to reverse transcription with the HiScript III 1 st Strand cDNA Synthesis Kit (+ gDNA wiper) (Vazyme, Nanjing, China) to obtain 20 μL of cDNA solution. The validation of the RNA-seq data was performed by randomly selecting thirty genes on the basis of their FPKM values, with 21 genes being upregulated and 9 genes being downregulated. The gene-specific primers for qRT‒PCR were designed using Primer Premier 5.0 software. The actin housekeeping gene of Bama Huoma served as the internal reference. The qRT‒PCR system consisted of 1 × TB Green® Premix Ex Taq™ (Tli RNaseH Plus) (TaKaRa, Dalian, China), forward/reverse primers at a concentration of 0.4 μM, and 20 ng of cDNA template. An LC480 Real Time PCR Detection System (Roche, Germany) was used to perform qRT‒PCR, and the PCR cycling conditions were as follows: denaturation at 95 °C for 3 min and 40 cycles of 95 °C for 10 s, 60 °C for 20 s and 72 °C for 30 s. For each sample, three biological and technical replicates were performed. Relative gene expression levels were calculated according to the 2 − ΔΔCt method [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. The primers used for qRT‒PCR are shown in Supplementary Table S1.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par17">The relative gene expression data from the female and male Bama Huoma plants were statistically analyzed according to the appropriate procedure for one-way analysis of variance using SPSS 17.0 software. Additionally, Duncan’s test was employed to determine significantly distinct groups.</p></section><section id="Sec11"><h3 class="pmc_sec_title">RNA in situ hybridization analyses</h3>
<p id="Par18">Probes were designed via Primer Premier 5.0 (Table S1). The probes were subsequently labeled with CY3 and synthesized by Servicebio, a biotechnology company based in Wuhan, China. RNA in situ hybridization was performed as described previously [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>, <a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. Fresh flower buds at different developmental stages were fixed in plant-specific in situ hybridization fixative (Servicebio, Wuhan, China) for a minimum of 12 h. The tissues were dehydrated through a graded ethanol series prior to paraffin embedding. Sections of 6 μm thickness were prepared using a Leica RM2016 microtome (Leica Microsystems, Germany) and baked at 62 °C for 2 h. The sections were dewaxed in transparent liquid (Servicebio, Wuhan, China) for 15 min, which was repeated once, followed by rehydration through a descending ethanol series (100%, 85%, and 75% ethanol, 5 min each). After the samples were rinsed with DEPC-treated water, protease K (20 μg/mL) was added dropwise for digestion at 37 °C for 10 min, followed by washing with distilled water and three PBS washes (5 min each). Prehybridization solution was added, and the mixture was incubated at 37 °C for 1 h. Hybridization was performed overnight at 40 °C using probe-containing hybridization buffer. The hybridization mixture was washed with 2 × SSC at 37 °C for 10 min, followed by two subsequent washes with 1 × SSC at 37 °C for 5 min each, and finally, the mixture was rinsed with 0.5 × SSC at room temperature for 10 min. Blocking was carried out with rabbit serum at room temperature for 30 min, followed by incubation with an anti-DIG-AP antibody at 37 °C for 40 min. The samples were then washed four times with TBS (5 min each). BCIP/NBT color development solution was added, and positive signals were visualized under a microscope. Finally, the samples were rinsed with distilled water and imaged using an optical microscope (Nikon Eclipse CI, Tokyo, Japan) equipped with an imaging system (Nikon DS-U3, Tokyo, Japan).</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec13"><h3 class="pmc_sec_title">Analysis of the separation ratio of females and males in Bama Huoma</h3>
<p id="Par19">The number of male and female plants in each duplicate was counted after the flowering of Bama Huoma on the basis of the morphology of the floral organs. As shown in Table <a href="#Tab1" class="usa-link">1</a>, the population of female plants was larger than that of male plants in each duplicate. However, the separation ratio between males and females was not significantly different according to the chi-square test (<em>p</em> &gt; 0.05), indicating that the proportions of males and females fit a 1:1 separation ratio.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Phenotypic and cytological characterization of Bama Huoma flowers</h3>
<p id="Par20">Bama Huoma is a dioecious species in which individuals producing female or male flowers occur at a 1:1 ratio and are randomly distributed in the population. However, little is known about its floral development. In this study, we observed the development of female and male flowers in Bama Huoma on the basis of phenotypic features, histological analysis, SEM and transmission electron microscopy (TEM). The female inflorescence is composed of compacted branchlets, resulting in the formation of a compound raceme (Fig. S1a). As shown in Fig. <a href="#Fig1" class="usa-link">1</a>, the inflorescence meristem differentiated from the shoot apical meristem in the reproductive growth stage of female flowers. The inflorescence subsequently underwent further development to transform into the female inflorescence. During early flower differentiation, the female flower consists of a carpel and bract that surrounds the carpel (Fig. <a href="#Fig1" class="usa-link">1</a>a, k-n). As the flower matured, two uneven stigmata elongated and emerged from the bract, while the tepals underwent complete degradation (Fig. <a href="#Fig1" class="usa-link">1</a>b, c, o). At a later stage, the stigma extended further, and papilla cells developed and covered the stigma from the tip to the basal parts (Fig. <a href="#Fig1" class="usa-link">1</a>d, e, p; Fig. S2a-c), which could be favorable for pollination. During flower development, numerous non-glandular trichome hairs and glandular trichomes developed on the bract that enveloped the ovary (Fig. <a href="#Fig1" class="usa-link">1</a>a-e, k-p; Fig. S2a). The male flowers were clustered at the leaf axils, and each flower was pedicellate, with individual stalks (Fig. S1b). In the early stages of staminate inflorescence development, the male flower buds were fan shaped and had pilose surfaces with many non-glandular trichome hairs (Fig. <a href="#Fig1" class="usa-link">1</a>f, g, q, r). During stamen development, the anther expanded and extended into an oval shape. The outer bract was also elongated, and the non-glandular trichome hairs that covered the outer surface gradually degenerated (Fig. <a href="#Fig1" class="usa-link">1</a>g-i, s-u). The anthers underwent further expansion and elongation as the pollen grains matured, accompanied by the outward spread of the perigona bract, exposing five stamens each attached to a filament that dangled the anther (Fig. <a href="#Fig1" class="usa-link">1</a>j). Many mature pollen grains were released through the longitudinal groove of the anther (Fig. <a href="#Fig1" class="usa-link">1</a>j, v; Fig. S2d-f).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369072_12870_2025_7096_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/50f1/12369072/a6bb7ba8f90e/12870_2025_7096_Fig1_HTML.jpg" loading="lazy" id="MO1" height="992" width="701" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phenotypic characterization of Bama Huoma flowers. Stereomicroscopy (<strong>a</strong>-<strong>j</strong>) and SEM (<strong>k</strong>-<strong>v</strong>) were used to observe the developmental process of female and male flower buds. <strong>a</strong>-<strong>e</strong> and <strong>k</strong>-<strong>p</strong>, Developmental process of female flower buds. <strong>f</strong>-<strong>j</strong> and <strong>q</strong>–<strong>v</strong>, Developmental process of male flower buds. An, anther; Bp, bract primordium; Br, bract; Fa, floral apex; Gt, glandular trichomes; Gp, gynoecium primordium; nGt, non-glandular trichomes; Op, ovule primordia; Pb, perigonal bract; Pg, pollen grain; Pi, pistillate inflorescence; Si, staminate inflorescence; Sti, stigma. Scale bars: 2 mm, indicating the size of the female and male buds of Bama Huoma in panels (<strong>a</strong>)-(<strong>j</strong>)</p></figcaption></figure><p id="Par21">To further elucidate the developmental characteristics of male and female flowers, paraffin sectioning and TEM were performed to analyze morphological differentiation and cytological changes in Bama Huoma. The development of female flowers can be divided into three stages according to key morphological and histocytological changes. The stages included the initial differentiation of the flower bud (Fig. <a href="#Fig2" class="usa-link">2</a>a), the formation and development of the pistil and bract primordia (Fig. <a href="#Fig2" class="usa-link">2</a>b, c), and the formation and development of ovule primordia (Fig. <a href="#Fig2" class="usa-link">2</a>d-h). At the initial stage of female flower bud differentiation, the apex of the flower bud primordium had a cone-like shape (Fig. <a href="#Fig2" class="usa-link">2</a>a). The apical growth point became wider and flatter following the anticlinal division and periclinal division of the meristem cells. During the formation and development of the gynoecium and bract primordia, apex meristem cells underwent continuous division, resulting in a growth pyramid with semicircular protuberance. This process led to increases in both the width and height of the floral apex, which subsequently differentiated into gynoecium primordia. Additionally, the gynoecium primordium margin slightly bulged and gradually developed into the bract primordium, which underwent continuous differentiation on both sides to give rise to the perigonal bract (Fig. <a href="#Fig2" class="usa-link">2</a>b, c). The gynoecium primordium continued to develop to form ovule primordia, which were characterized by a small size, compact arrangement and dense cytoplasm (Fig. <a href="#Fig2" class="usa-link">2</a>d). The upper part of the ovule primordia developed into the stigma primordium, which continued to elongate and develop into the stigma, and the protrusions on both sides differentiated to form the perianth (Fig. <a href="#Fig2" class="usa-link">2</a>e), whereas the lower part developed into the ovary, which surrounded the ovule and then expanded. At this point, the morphological differentiation of the female flower buds was complete, while the perianth gradually degenerated during ovary development (Fig. <a href="#Fig2" class="usa-link">2</a>f-h). The TEM results revealed that all the cells of the female flowers were highly vacuolized, and the nucleus was pushed to the edge. The nucleolus was enlarged, and the chromatin disintegrated during the differentiation of female flowers, which indicated that the cells were in the active stage of division at all stages of the formation of female flowers. In addition, except for a few proteasomes found in vacuoles and a few cytoplasmic organelles, such as mitochondria and the endoplasmic reticulum, most of the organelles were dissolved, which could provide energy and nutrients for the development of female flowers (Fig. S3 a-i).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369072_12870_2025_7096_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/50f1/12369072/7cf0f992b68b/12870_2025_7096_Fig2_HTML.jpg" loading="lazy" id="MO2" height="574" width="778" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cytological observations of consecutive stages of differentiation of female Bama Huoma flowers. <strong>a</strong> Initial differentiation of female flower buds. <strong>b</strong> and <strong>c</strong> Formation and development of pistil and bract primordia. <strong>d</strong>-<strong>h</strong> Formation and development of ovule primordia. Bp, bract primordium; Fa, floral apex; Gp, growth pyramid; Op, ovule primordia; O, ovule; Ov, ovary wall; P, perianth; Pb, perigonal bract; Sb, stigma branch; Sp, stigma primordium</p></figcaption></figure><p id="Par22">The staminate flowers exhibited anatomical distinctions from the pistillate flowers and were divided into three developmental stages: the initial differentiation of the male flower bud (Fig. <a href="#Fig3" class="usa-link">3</a>a), the formation and development of brace primordia and stamens (Fig. <a href="#Fig3" class="usa-link">3</a>b), and the formation of anthers and pollen grains (Fig. <a href="#Fig3" class="usa-link">3</a>c-h). During the reproductive growth stage, the flower bud primordia at the top began to differentiate into stamen primordia, which continued to differentiate and form brace primordia and stamens via anticlinal division (Fig. <a href="#Fig3" class="usa-link">3</a>a, b). The young anther was oblong and consisted of homogeneous meristem cells that divided continuously to form archesporial cells (Fig. <a href="#Fig3" class="usa-link">3</a>c). TEM revealed that the archesporial cells contained many starch grains and proteasomes, providing important nutrients for microspore development (Fig. S4 a-d). The archesporial cells differentiated into primary parietal cells and primary sporogenous cells. The primary parietal cells differentiated into the endothecium, epidermis and middle layers, whereas the primary sporogenous cells further developed into microspore mother cells (Fig. <a href="#Fig3" class="usa-link">3</a>d). The epidermal cells became enlarged and vacuolized with the development of the anther, and the middle layer cells began to compress and disintegrated gradually to be absorbed. The tapetum cells were large, square shaped and neatly arranged, with condensed chromatin and prominent nucleoli, while the nucleus of the microspore mother cell was enlarged, and the nucleolus disintegrated (Fig. S4 e, g). Many organelles, such as mitochondria, the endoplasmic reticulum, starch grains and the proteasome, which provide necessary substances and energy for pollen development, were found in tapetum cells and microspore mother cells (Fig. S4 f, h). Callose deposition was observed after the formation of microspore mother cells, and microspore mother cells then underwent meiosis to form tetrads (Fig. <a href="#Fig3" class="usa-link">3</a>e). The epidermal cells continued to undergo vacuolation, and the intracellular material was dissolved by lysosomes. Tapetum cells were also vacuolized, expanded laterally and radially and increased in size, and the intracellular substances further concentrated and then gradually degraded. Various substances were secreted through the tangential plane of the tapetal cell to provide nutrients for microspore development (Fig. S4 i-l). Callose dissociated and released the tetrads to form free mononuclear microspores following the completion of meiosis (Fig. <a href="#Fig3" class="usa-link">3</a>f). At the mononuclear stage, the microspore cell nuclei were gradually squeezed to the cell edge by the large central vacuole, and the tapetum degraded further (Fig. <a href="#Fig3" class="usa-link">3</a>g). The microspores underwent two rounds of mitosis to form mature pollen, and the remaining tapetum degraded and was absorbed to provide nutrients for pollen development (Fig. S4 m‒p). At the later stage of pollen development, epidermal cells developed into flat fibrous layer cells until they were dry. The endothecium developed into fiber bundles and left behind a long, narrow strip outside the junction of the two pollen sacs. When the anthers matured, they broke open and dispersed their pollen (Fig. <a href="#Fig3" class="usa-link">3</a>h).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369072_12870_2025_7096_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/50f1/12369072/6414d198e2f0/12870_2025_7096_Fig3_HTML.jpg" loading="lazy" id="MO3" height="564" width="778" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cytological characterization of male Bama Huoma flowers. <strong>a</strong> Initial differentiation of the male flower bud. <strong>b</strong> Formation and development of brace primordia and stamens. <strong>c</strong>-<strong>h</strong> Formation of anthers and pollen grains. Ar, archesporial cell; Bp, brace primordia; En, endothecium; Ep, epidermis; Ml, middle layer; Mmc, microspore mother cell; Mm, mononuclear microspore; Pg, pollen grain; Sp, stamen primordia; St, stamen; T, tapetal layer; Tet, tetrad; Vm, vacuolated microspore</p></figcaption></figure></section><section id="Sec15"><h3 class="pmc_sec_title">Transcriptome sequencing, mapping and global expression analyses of Bama Huoma</h3>
<p id="Par23">For further elucidation of the potential regulatory mechanism of sex determination in Bama Huoma, whole-transcriptome strand-specific RNA-seq of female and male Bama Huoma plants at two flower developmental stages [initial differentiation of female/male flower buds (T1F or T1M), formation and development of ovule primordia in female flowers (T3F) and formation of anthers and pollen grains in male flowers (T3M)] was performed on the Illumina HiSeq 4000 platform with three biological replicates. A total of 165.34 Gb of raw data were obtained from twelve libraries by high-throughput sequencing. After raw read quality filtering, a total of 10,54,505,254 clean reads were generated, with each sample producing at least 76,917,286 clean reads. The average percentage of valid reads was ≥ 95.65%. The percentages of Q20 bases and Q30 bases were greater than 99.98% and 98.40%, respectively, in our sequencing results, with a GC content of at least 49% (Table S2). The sequencing results revealed good quality of the original data for subsequent analyses. The clean reads were subsequently mapped to the C. sativa reference genome using Bowtie2 [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] and HISAT2 [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. On average, more than 86.9% of the valid reads were mapped to C. sativa reference genomes via this sequencing procedure. A total of 748,692 transcripts and 392,316 genes were identified from the twelve cDNA libraries, and 40,439 genes were mapped to the reference genome (Table S3). In this study, the total number of expressed genes in the mature stage of flowering was greater than that in the initial stage. For example, approximately 33 thousand genes were expressed in the stage of ovule primordium formation in female flowers (T3F), whereas ~ 30,000 were expressed in the stage of initial flower bud differentiation (T1F) (Table S3). We also found that the total number of expressed genes in the anther formation stage was greater than that in the initial differentiation stage of the male flower bud (Table S3). Detailed information on the total mapped, uniquely mapped, multimapped, nonspliced, and spliced reads for each library is presented in Table S4. The Spearman correlation coefficient test for biological repetitions in the twelve libraries revealed a strong correlation between samples with values above 92% (Fig. S5). In different flower organs, the similarity value between T1F and T3F was greater than that between T1M and T3M. Moreover, the values of T1F and T1M were greater than those of T3F and T3M at different floral development stages (Fig. S5).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Analysis of DEGs in female and male flowers of Bama Huoma</h3>
<p id="Par24">To analyze and interpret the DEGs in the transcriptome effectively, we used FPKM values to measure the different mRNA expression levels in each sample. The DEGs between samples were selected according to a p value &lt; 0.05 and log2 (FC) &gt; 1 or log2 (FC) &lt; − 1, and the overall distribution of the DEGs was visualized via pairwise comparisons (Fig. <a href="#Fig4" class="usa-link">4</a>A). The number of DEGs showed very high variance among the different developmental stages. The greatest number of DEGs was found between T3M and T1M, and more than 7072 DEGs were identified: 2905 upregulated genes and 4167 downregulated genes. Similarly, 6816 DEGs were observed between T3M and T3F: 2564 upregulated and 4252 downregulated genes (Fig. <a href="#Fig4" class="usa-link">4</a>A). However, the T1M vs. T1F and T3F vs. T1F libraries presented 3936 and 1728 DEGs, respectively: 1818 and 816 upregulated DEGs and 2128 and 912 downregulated DEGs, respectively (Fig. <a href="#Fig4" class="usa-link">4</a>A). The number of downregulated DEGs was greater than that of upregulated DEGs during the development of both female and male flower buds, as revealed by pairwise comparisons (Fig. <a href="#Fig4" class="usa-link">4</a>A). The number of DEGs in the T3M stage was greater than that in the other stages, suggesting that the T3M stage may be a key stage for pollen development.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369072_12870_2025_7096_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/50f1/12369072/b51547d5d3e2/12870_2025_7096_Fig4_HTML.jpg" loading="lazy" id="MO4" height="569" width="778" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Total DEGs and their distribution in female and male Bama Huoma flowers. <strong>A</strong> shows the total number of DEGs in female and male Bama Huoma flowers. <strong>B</strong> represents the distribution of unique and common DEGs in the different developmental stages of Bama Huoma flower buds. <strong>C</strong> shows images of the different profiles of the common DEGs between T1M vs. T1F and T3M vs. T3F. <strong>D</strong> shows the GO terms of the common DEGs between T1M vs. T1F and T3M vs. T3F</p></figcaption></figure></section><section id="Sec17"><h3 class="pmc_sec_title">GO enrichment analysis of DEGs</h3>
<p id="Par25">To determine the main transcriptional dynamics related to sex differentiation in the dioecious plant Bama Huoma, we used a Venn diagram to analyze the numbers of DEGs in each pairwise comparison. Among the DEGs, a total of 253 common DEGs were identified through pairwise comparisons among these four sequenced samples, indicating their involvement in the growth and development processes of floral organs in Bama Huoma (Fig. <a href="#Fig4" class="usa-link">4</a>B, Table S5). GO enrichment analysis based on three GO categories, namely, biological process, molecular function, and cellular component, was performed for the DEGs. The GO enrichment analysis results revealed that most of these DEGs were enriched in the regulation of transcription (DNA dependent), except for some genes involved in the extracellular region and chitinase activity (Fig. S6 A, B). Heatmap analysis further revealed that the expression of most DEGs enriched in the regulation of transcription decreased in both female flowers and male flowers of Bama Huoma during floral development, suggesting that these genes associated with transcriptional regulation assume negative regulatory functions in the development of Bama Huoma floral organs (Fig. S6C). However, the expression of DEGs related to the extracellular region varied during the development of female and male flowers. Some of these genes were upregulated, whereas others were downregulated, but the expression trend was the same in both female and male flowers during flower development (Fig. S6D). These results suggested that these DEGs have the same functions in regulating the development of female and male flowers.</p>
<p id="Par26">Additionally, the common DEGs at the T1 and T3 developmental stages between female and male flowers were further analyzed to identify genes associated with sex differentiation in Bama Huoma. Upon comparative analysis, 981 common DEGs were identified between the T1M vs. T1F and T3M vs. T3F comparison groups, among which 326 genes were upregulated and 655 were downregulated (Fig. <a href="#Fig4" class="usa-link">4</a>B). The expression patterns of the common DEGs between T1M vs. T1F and T3M vs. T3F were predicted using STEM software. We established 15 model profiles, among which five were identified as significant (Fig. <a href="#Fig4" class="usa-link">4</a>C). Genes of the same type were included in the same cluster, where the shaded part is significant. We found that Profile 2 was the most significantly enriched of the four samples and had the smallest p value (Fig. <a href="#Fig4" class="usa-link">4</a>C, Table S6). In addition, for further analysis of the functions of common DEGs between T1M vs. T1F and T3M vs. T3F, the GO database was used to annotate DEGs according to three major categories: biological process, cellular component, and molecular function (Fig. <a href="#Fig4" class="usa-link">4</a>D). The GO classification analysis revealed that the genes involved in biological processes were enriched mainly in the categories of regulation of transcription (accounting for 81.61% of the total genes), transcription, and DNA-templated (accounting for 67.15% of the total genes) (Fig. <a href="#Fig4" class="usa-link">4</a>D). The genes related to cellular components were enriched in the nucleus, plasma membrane, and cytoplasm. The enriched molecular functions were protein binding, DNA-binding TF activity and DNA binding. The GO functional enrichment scatter plot revealed that many DEGs were related to DNA-binding TF activity (GO: 0003700), cell differentiation (GO: 0030154), transcription regulation (GO: 0006355), and transcription, DNA-templated (GO: 0006351) (Fig. <a href="#Fig5" class="usa-link">5</a>A). A comparative analysis of the DEGs through GO functional enrichment and STEM analysis revealed that genes associated with transcriptional regulation were predominantly enriched in the top four gene clusters identified by STEM, especially in Profile 2 (which accounted for 64.65% of all the pathways related to transcriptional regulation) (Table S6). These findings suggest the high significance of these genes in the sexual differentiation of Bama Huoma, which warrants further analysis.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369072_12870_2025_7096_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/50f1/12369072/a5c1989f7803/12870_2025_7096_Fig5_HTML.jpg" loading="lazy" id="MO5" height="814" width="778" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>GO functional enrichment analyses of the common DEGs between T1M vs. T1F and T3M vs. T3F and a heatmap of the genes related to sex differentiation in Bama Huoma. <strong>A</strong> GO functional enrichment analyses of the common DEGs. <strong>B</strong> Annotation of the TFs of the specific DEGs related to sex differentiation in Bama Huoma. The pie chart shows the relative proportions of TFs. <strong>C</strong> Heatmap of the expression levels of TFs in female and male Bama Huoma flowers at different developmental stages. <strong>D</strong> Heatmap analysis of MYB TFs in female and male Bama Huoma flowers at different developmental stages. <strong>E</strong> Heatmap analysis of MADS-box genes in female and male Bama Huoma flowers at different developmental stages. The expression levels of the genes are indicated by different colors. Red represents high expression, and green represents low expression. The red boxes in A indicate GO enrichment related to transcriptional regulation, and the red boxes in <strong>C</strong> and <strong>D</strong> indicate genes specifically expressed in female or male flowers</p></figcaption></figure></section><section id="Sec18"><h3 class="pmc_sec_title">Candidate genes involved in sexual differentiation in Bama Huoma</h3>
<p id="Par27">For analysis of the expression intensity of these DEGs related to the regulation of transcription, a heatmap was generated to illustrate the expression levels of female and male flowers in Bama Huoma at different developmental stages. The expression profiles of enriched pathway categories in two pairwise comparisons (T1M vs. T1F and T3M vs. T3F) were also examined. Most DEGs in the enriched categories were downregulated in male flowers compared with female flowers (Table S7). TFs are thought to be key regulators of transcription and have been previously reported to play important roles in sex differentiation in dioecious plants [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. In the present research, a total of 55 TFs belonging to 18 TF families were identified from these specific DEGs, including MYB, MADS-box, GATA, AP2/ERF, and WRKY TFs (Fig. <a href="#Fig5" class="usa-link">5</a>B, Table S7). Notably, MYB TFs accounted for 25% of the total number of TFs. Heatmap analysis and hierarchical clustering revealed that the expression patterns of these 55 TFs could be divided into four clusters (C1, C2, C3 and C4) between female and male flowers (Fig. <a href="#Fig5" class="usa-link">5</a>C). The genes in the C1 cluster included three MYB, two ARF and three MADS genes whose expression was in female flowers, especially the INAPERTURATE POLLEN1 gene (LOC115716968), whose expression peaked at T3M (Fig. <a href="#Fig5" class="usa-link">5</a>C). The expression of the TFs in the C2 and C4 clusters was upregulated in the female flower of Bama Huoma. However, the expression level of the C2 cluster in T3F was greater than that in T1F, particularly with respect to the preferential expression of MYB29 (LOC115708954) during the T3F stage. Conversely, the expression level of the C3 cluster in T1F was greater than that in T3F, with preferential expression observed for TCP13 (LOC115701300) and CAULIFLOWER (LOC115704295) during the T1F stage (Fig. <a href="#Fig5" class="usa-link">5</a>C, Table S7).</p>
<p id="Par28">MYB-type TFs, which belong to one of the largest families in plant biology, play pivotal roles not only in plant growth and development but also in floral development. In the present study, we observed that the expression of MYB TFs in Bama Huoma was associated with a sex preference. A majority of the MYB TFs presented higher expression levels in female flowers than in male flowers (Fig. <a href="#Fig5" class="usa-link">5</a>D). Furthermore, the expression of these genes was greater in the T3 development stage than in the T1 stage, suggesting that MYB plays a critical role in floral development in Bama Huoma (Fig. <a href="#Fig5" class="usa-link">5</a>D). Interestingly, we also observed specific expression of the TF MYB35 (LOC115711316 and LOC115700367) in male flowers, whereas its expression was minimal in female flowers (Fig. <a href="#Fig5" class="usa-link">5</a>D, Table S7). These findings strongly suggest that MYB35 plays a pivotal role in the development of male flowers.</p>
<p id="Par29">In plants, MADS-box TFs are involved in various developmental processes, such as floral development and flowering control [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. In this study, the expression of the MADS-box TF in Bama Huoma exhibited a sex bias (Fig. <a href="#Fig5" class="usa-link">5</a>E). We subsequently performed phylogenetic evolution, conserved motif, and domain organization analyses to identify closely related members of the MADS-box gene family associated with sex differentiation in Bama Huoma (Fig. S7). Three MADS-box TFs, namely, TM6 (LOC115714657, AP3-like), PMADS2 (LOC115699531, homologous to PI), and AGL104 (LOC115713471), were found to be specific to male flowers during various stages of development, whereas little expression was detected in female flowers (Fig. <a href="#Fig5" class="usa-link">5</a>E). However, the MADS-box TFs CAULIFLOWER A (LOC115698549, homologous to AP1), AGL16 (LOC115709034), CAL (LOC115704295, CAULIFLOWER A-like in other plants) (Fig. S7), FBP24 (LOC115700907), and AGL11 (LOC133031811) exhibited specific expression in female flowers but showed minimal expression in male flowers at various developmental stages (Fig. <a href="#Fig5" class="usa-link">5</a>E, Table S7).</p>
<p id="Par30">In addition to these TFs, several crucial functional and structural domain proteins, such as ERF, WRKY, NAC domain proteins, TFIIH C1-like domain proteins, and zinc-finger proteins, may be involved in flower organ development in the dioecious plant Bama Huoma. Heatmap analysis and hierarchical clustering revealed that a majority of these functional and structural domain proteins were upregulated in female flowers compared with male flowers (Fig. S8 A-C). The results of our study indicate that flower development is a complex process in plants that is controlled by intricate gene regulatory networks.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Validation of the RNA-seq data by qRT‒PCR</h3>
<p id="Par31">To evaluate the reliability and reproducibility of the transcriptome data, we used qRT‒PCR analysis was used to further confirm the RNA-seq data. A total of 20 genes, eight upregulated genes and twelve downregulated genes, were randomly selected for qRT‒PCR analysis. The FCs in gene expression between T1 buds and T3 buds of both female and male flowers of Bama Huoma, as determined by qRT‒PCR, were compared to those obtained through RNA-seq analysis. Our results demonstrated congruity between the expression profiles of these genes and those depicted by RNA-seq data, with relative R2 values of 0.8765 and 0.9278 in T1M vs. T1F and T3M vs. T3F, respectively, confirming the reliability of our RNA-seq results (Fig. S9).</p></section><section id="Sec20"><h3 class="pmc_sec_title">Expression of MYB and MADS-box genes involved in sex differentiation in Bama Huoma</h3>
<p id="Par32">TFs such as MYB and MADS-box proteins play a critical role in the growth and development of plants. On the basis of transcriptomic data, we found that numerous TFs were expressed in a sex-biased manner in male or female flowers of Bama Huoma. Therefore, we conducted qRT‒PCR analysis to examine the expression patterns of these sex-biased genes, including two MYB genes and eight MADS-box genes, in different tissues of female and male Bama Huoma plants. The relative expression level in the T1 buds of female flowers was used as a control. Three MADS-box genes (PMADS2, TM6, AGL104) and two MYB35 genes were specifically expressed in male flowers (Fig. <a href="#Fig6" class="usa-link">6</a>A, D, G, J, M), whereas five MADS-box TFs (AGL11, FBP24, AGL16, and two CAL genes) were specifically expressed in female flowers (Fig. <a href="#Fig7" class="usa-link">7</a>A-E). Among them, the expression of three genes, PMADS2, TM6 and AGL104, increased during male flower development, with a high expression level (≥ 103) in the T3 bud development stage, indicating the key roles of these genes in male flower formation in Bama Huoma (Fig. <a href="#Fig6" class="usa-link">6</a>G, J, M). However, the expression of two genes, AGL11 and FBP24, increased with female flower development, suggesting their involvement (Fig. <a href="#Fig7" class="usa-link">7</a>A, C), and the expression of three other genes, AGL16 and two CAL genes, increased in the early stages of female flower development, suggesting that these genes regulate the differentiation of female floral organs (Fig. <a href="#Fig7" class="usa-link">7</a>B, D, E). Furthermore, MYB35 genes (LOC115711316 and LOC115711316) exhibited much greater expression in male flowers, especially LOC115711316, which was specifically expressed in male flowers (Fig. <a href="#Fig6" class="usa-link">6</a>A, D). In conclusion, the development of dioecious flower organs is a complex process, and genes involved in the determination of floral organ identity may act in a combinatorial manner during sex differentiation in Bama Huoma.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369072_12870_2025_7096_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/50f1/12369072/78f68192ed8b/12870_2025_7096_Fig6_HTML.jpg" loading="lazy" id="MO6" height="991" width="680" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Expression profiles of candidate genes associated with male flower development in Bama Huoma were analyzed across various tissue types, organs, and distinct developmental stages of the male flower. The values on the bar graph correspond to the means ± SDs of three biological replicates (<em>n</em> = 3). Ma, mature anther; Pg, pollen grain; Ps, pollen sac; Pb, perianth bract; Ac, stage of archesporial cells; Pmc, pollen mother cell stage; Tet, tetrad stage; Un, uninucleate stage; Bi, binucleate stage; Mp, stage of mature pollen. Different capital letters indicate extremely significant differences between groups (<em>p</em> &lt; 0.01), whereas different lower letters indicate significant differences between groups (<em>p</em> &lt; 0.05). The same letter indicates no significant difference</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369072_12870_2025_7096_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/50f1/12369072/6cdefd7cdcab/12870_2025_7096_Fig7_HTML.jpg" loading="lazy" id="MO7" height="867" width="778" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Expression profiles of candidate genes related to sex differentiation of female flowers in Bama Huoma from different tissue sites and development stages. St, stigma; Ov, ovary; Pb, perigonal bract. The values on the bar graph correspond to the means ± SDs of three biological replicates (<em>n</em> = 3). Different capital letters indicate extremely significant differences between groups (<em>p</em> &lt; 0.01), whereas different lower letters indicate significant differences between groups (<em>p</em> &lt; 0.05). The same letter indicates no significant difference</p></figcaption></figure></section><section id="Sec21"><h3 class="pmc_sec_title">Analysis of the expression patterns of genes associated with sex differentiation in Bama Huoma</h3>
<p id="Par33">To determine the spatiotemporal expression patterns of genes exhibiting sex-biased expression across various organs and developmental stages of Bama Huoma flowers, we employed qRT‒PCR and RNA in situ hybridization. Previous studies have shown that MYB35 is involved mainly in anther development and tapetum cell formation [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>, <a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>], and the MADS-box TFs TM6 and PMADS2 are involved in the pathway specification of floral organ identity [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. In the present study, the expression levels of MYB35 were significantly greater in flower buds than in other tissues. In contrast, relatively lower expression levels were detected in pollen grains, mature anthers, pollen sacs, and perianth bracts (Fig. <a href="#Fig6" class="usa-link">6</a>B, E). Spatiotemporal expression analysis revealed distinct patterns for the two MYB35 genes during anther development. Specifically, LOC115700367 exhibited peak expression at the tetrad stage, followed by a progressive decline. In contrast, LOC115711316 expression gradually increased, peaking at the binucleate stage, and subsequently sharply decreased at the mature pollen grain stage (Fig. <a href="#Fig6" class="usa-link">6</a>C, F). In accordance with the qRT‒PCR results, RNA in situ hybridization confirmed that MYB35 was expressed ubiquitously across various developmental stages of the male flower, except the mature pollen grain stage (Fig. <a href="#Fig8" class="usa-link">8</a>A‒F). The highest expression level was detected in the tapetal layer during the anther tetrad stage (Fig. <a href="#Fig8" class="usa-link">8</a>C).</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369072_12870_2025_7096_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/50f1/12369072/2acb9b80b0f8/12870_2025_7096_Fig8_HTML.jpg" loading="lazy" id="MO8" height="615" width="778" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>RNA in situ hybridization was used to identify the expression of MYB35, TM6, PMADS2, and AGL104 across different developmental stages of male flowers. <strong>A</strong>-<strong>F</strong> MYB35 expression. <strong>G</strong>-<strong>L</strong> TM6 expression. <strong>M</strong>‒<strong>R</strong> PMADS2 expression. <strong>S</strong>-<strong>X</strong> AGL104 expression. Brp, brace primordia; Ep, epidermis; Mmc, microspore mother cell; Mm, mononuclear microspore; Bm, binuclear microspore; Pg, pollen grain; Sp, stamen primordia; T, tapetal layer. Ac, stage of archesporial cells; Pmc, pollen mother cell stage; Tet, tetrad stage; Un, uninucleate stage; Bi, binucleate stage; Mp, stage of mature pollen</p></figcaption></figure><p id="Par34">The spatiotemporal expression patterns of the other three MADS-box genes, PMADS2, TM6, and AGL104, exhibited variability across different developmental stages of male flowers and among various floral organs. The expression levels of TM6 and PMADS2 progressively increased during anther development, reaching a peak at the uninucleate stage, and subsequently declined to their lowest levels at the mature pollen grain stage (Fig. <a href="#Fig6" class="usa-link">6</a>I, L). In contrast, the expression of AGL104 consistently increased throughout anther development, peaking at the mature pollen grain stage (Fig. <a href="#Fig6" class="usa-link">6</a>O). In different floral organs, TM6 and PMADS2 presented significantly greater expression in pollen sacs than in mature anthers, perianth bracts, and pollen grains (Fig. <a href="#Fig6" class="usa-link">6</a>H, K). Conversely, AGL104 was highly expressed in pollen grains, pollen sacs, and anther bracteoles (Fig. <a href="#Fig6" class="usa-link">6</a>N), suggesting that it plays a key regulatory role in anther maturation through constitutive expression. Consistent with the qRT‒PCR results, RNA in situ hybridization analysis revealed that TM6 and PMADS2 displayed discernible hybridization signals throughout various stages of anther development, except mature pollen grains (Fig. <a href="#Fig8" class="usa-link">8</a>G‒R). Notably, TM6 presented a particularly strong hybridization signal in the pollen sac (Fig. <a href="#Fig8" class="usa-link">8</a>J), indicating its potential importance in pollen sac formation. In addition, the hybridization signal of AGL104 consistently increased throughout all stages of anther development (Fig. <a href="#Fig8" class="usa-link">8</a>S-X). In contrast to other genes, AGL104 also presented strong hybridization signals in mature pollen grains, highlighting its essential role in regulating pollen grain maturation.</p>
<p id="Par35">Furthermore, the spatiotemporal expression patterns of the other five MADS-box TFs specifically expressed in female flowers revealed that the expression levels of AGL11 and FBP24 progressively increased during female flower development (Fig. <a href="#Fig7" class="usa-link">7</a>F, H). Notably, compared with those of other tissues, the expression of FBP24 was significantly greater in the ovary, whereas it was barely expressed in the stigma (Fig. <a href="#Fig7" class="usa-link">7</a>H). Conversely, AGL11 exhibited markedly elevated expression in both the stigma and ovary relative to the perigonal bract (Fig. <a href="#Fig7" class="usa-link">7</a>F). RNA in situ hybridization analysis revealed a progressive intensification of hybridization signals for AGL11 and FBP24 as female floral organs developed, which is consistent with the results obtained from qRT‒PCR analysis (Fig. <a href="#Fig9" class="usa-link">9</a>A-H). Notably, hybridization signals were detected exclusively for the AGL11 gene within the stigma tissue, whereas no such signals were detected for the other genes examined in this tissue (Fig. <a href="#Fig9" class="usa-link">9</a>). These results demonstrate that AGL11 not only plays a regulatory role in the development of female flowers but is also a crucial factor in stigma formation. Additionally, both qRT‒PCR and RNA in situ hybridization revealed that the expression levels of AGL16 and two CAL genes gradually decreased throughout female flower development (Fig. <a href="#Fig7" class="usa-link">7</a>G, D, E; Fig. <a href="#Fig9" class="usa-link">9</a>I-T). Specifically, CALA expression was significantly elevated in the ovary relative to other tissues (Fig. <a href="#Fig9" class="usa-link">9</a>I-T). In contrast, the AGL16 and CAL genes presented relatively low expression levels in the stigma, ovary, and perigonal bract. Additionally, during the T1F and T2F stages, AGL16 and CAL presented markedly greater hybridization signals than did the T3F period (Fig. <a href="#Fig9" class="usa-link">9</a>M-T). The differential expression patterns of the five MADS-box genes in female flowers suggest their distinct roles in the development of this organ. Thus, we hypothesized that AGL16 and CAL may play critical roles in the differentiation of female floral organs, whereas AGL11, FBP24, and CALA predominantly regulate female flower development and ovary formation. In conclusion, the development of dioecious flower organs is a complex process, and genes involved in the determination of floral organ identity may act in a combinatorial manner during sex differentiation in Bama Huoma.</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369072_12870_2025_7096_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/50f1/12369072/2ee5f4abe961/12870_2025_7096_Fig9_HTML.jpg" loading="lazy" id="MO9" height="992" width="706" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>RNA in situ hybridization was used to identify the expression of AGL11, FBP24, AGL16, and two CALs across different developmental stages of female flowers. <strong>A</strong>-<strong>D</strong> AGL11 expression. <strong>E</strong>–<strong>H</strong> FBP24 expression. <strong>I</strong>-<strong>L</strong> CALA expression. <strong>M</strong>-<strong>P</strong> CAL expression. <strong>Q</strong>-<strong>T</strong> AGL16 expression. Br: bract; Bp, bract primordium; Fa, floral apex; Op, ovule primordia; O, ovule; Ov, ovary wall; P, perianth; Pb, perigonal bract; Sb, stigma branch; Sp, stigma primordium. T1F, initial differentiation of the female flower bud; T2F, formation and development of bract and pistil primordia; T3F, formation and development of ovule primordia</p></figcaption></figure></section></section><section id="Sec22"><h2 class="pmc_sec_title">Discussion</h2>
<section id="Sec23"><h3 class="pmc_sec_title">Characteristics of flower organ development in Bama Huoma</h3>
<p id="Par36">Unisexual flowers in monoecious and dioecious species have evolved from hermaphroditic ancestors [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. The conversion of hermaphrodite flowers into unisexual flowers is always achieved by suppressing the development of one of the sex organs (carpels or stamens) [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. In most dioecious angiosperm species, including Diospyros lotus [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>], Silene latifolia [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>] and A. officinalis [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>], flowering is initiated with hermaphroditic primordia, followed by abortion or arrest of sex organ development. The present study provides a comprehensive illustration of the anatomical features of female and male flowers in Bama Huoma. The observations clearly demonstrated distinct and independent developmental processes between female and male flowers, indicating that sexual differentiation occurs from flower initiation and that morphological sex differentiation occurs at an early stage prior to the establishment of floral organ identity. This finding aligns with previous research suggesting that the sex determination pathway may be activated during the vegetative stages in hemp [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>].</p>
<p id="Par37">In the present study, during the initial stage of female flower bud differentiation, the apical meristem was completely enveloped by bracts (Fig. <a href="#Fig1" class="usa-link">1</a>k-m); however, in the medicinal cultivar Cannabis NB140, the apical meristem was exposed and not covered by bracts, allowing clear visualization of the flower primordium, leaf primordium, and carpellary primordium under SEM [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. In male plants, the anthers are produced in clusters of staminate inflorescence that develop at the leaf axils. Substantial quantities of pollen grains are released through the longitudinal groove or stomium located on the back of the anther upon flower maturation. This finding is consistent with previous findings that male flowers in marijuana and Mercurialis annua produce significantly greater quantities of pollen than do hermaphroditic flowers [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. Each individual male flower has the capacity to generate up to 350,000 pollen grains [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. These findings suggest that male flowers in dioecious plants exhibit greater pollen activity than hermaphroditic flowers do, indicating an adaptive evolutionary response from monoecy to dioecy in plants.</p></section><section id="Sec24"><h3 class="pmc_sec_title">TFs involved in the regulation of sex determination in Bama Huoma</h3>
<p id="Par38">Flower development is the core of higher plant ontogenesis and is controlled by complex gene regulatory networks. In this study, in addition to TFs, several important proteins, such as NAC domain proteins, B3 DNA-binding domain proteins, and zinc-finger proteins, were shown to be involved in flower organ development in dioecious Bama Huoma. Previous studies have indicated that NAC domain proteins are involved in plant growth and development and play a regulatory role in flower development [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. In Arabidopsis, TAPNAC is specifically expressed in the tapetum and directs tapetal gene expression [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. CUC1 and CUC2, which are important members of the NAC family, act as negative regulatory factors in carpel fusion in the apical gynoecium [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>, <a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. CUC2a and miR164a synergistically regulate the development of flower organs and inflorescences in strawberry [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. In addition, previous studies have shown that NAC domain proteins may work together with the MYB, DYT1, TDF1 and AMS proteins to regulate plant pollen development [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>, <a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. In this study, the NAC domain protein level in female flowers was greater than that in male flowers, suggesting that NAC positively regulates the development of female flowers.</p>
<p id="Par39">Zinc-finger proteins, which include C2H2-type, CCHC-type, Dof-type and WIP2-like proteins, constitute one of the largest TF families and are highly involved in the transcriptional regulation of flowering induction, floral organ morphogenesis, and pollen and pistil maturation [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>–<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. In Arabidopsis, the transcriptional repressor SUP, which contains a C2H2-ZF DNA-binding domain and an ERF-associated repression (EAR) domain, can repress the expression of APETALA 3 (AP3) and PISTILLATA (PI, PMADS2-like) [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. This study revealed that all the DEGs encoding ZFP domain-containing proteins were upregulated in female Bama Huoma flowers, whereas TM6 (LOC115714657, AP3-like) was not expressed in female flowers, which suggests that ZFP domain-containing proteins could repress the expression of AP3 in female Bama Huoma flowers.</p>
<p id="Par40">The B3 DNA-binding domain is an important region in the AP2/ERF and auxin response factor (ARF) genes. These genes are transcriptional regulators with a variety of functions involved in reproductive, developmental and physiological processes in plants [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>–<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. In Arabidopsis thaliana, the reproductive meristem gene family encodes TFs belonging to the B3 DNA-binding domain superfamily, and its members are preferentially expressed during flower, ovule, and seed development [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>–<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. The auxin response factor ARF3 is expressed in various parts of hazel plants, including the ovary primordium and structures such as the funiculus, integument, endosperm, and cotyledon. This protein is believed to play a role in regulating genes induced by auxin and MADS, AP2/ERF, TCP, and LFY TFs [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. In this study, the expression of AP2/ERF TFs and B3 DNA-binding domain proteins in female flowers was significantly greater than that in male flowers, suggesting that B3 DNA-binding domain proteins play a key role in regulating the development of female flowers. However, in rice, the miR167d-ARF module regulates the elongation of stamen filaments and stigma size. The overexpression of miR167d or mutation of ARF results in failed elongation of stamen filaments and increased stigma size. The expression level of the ARF gene (LOC115706063) in male flowers was significantly greater than that in female flowers, suggesting that ARF is an important regulator of stamen filament formation in Bama Huoma. Taken together, these results indicate that hormone signal transduction and floral homeotic TFs act in a combinatorial manner during sex differentiation in Bama Huoma.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Floral organ identity is regulated by MADS-box genes in a combinatorial manner in Bama Huoma</h3>
<p id="Par41">Floral organ identity genes play important roles in floral sex differentiation. The combined control of organ identities by MADS-box genes is central to the ABCDE model of flower formation [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. In Arabidopsis, the A function, which specifies sepals and petals, is mainly mediated by AP1 and is involved in floral meristem specification. The B function is exerted via AP3 and PI and is responsible for petal and stamen identity, whereas the C function is exerted by AGAMOUS (AG) and controls stamen and carpel identity. Additionally, class D functions, related to ovule identity, are mediated by SEEDSTICK (STK/AGL11), SHATTERPROOF1 (SHP1/AGL1), and SHATTERPROOF2 (SHP2/AGL5). The SEPALLATA1 (SEP1), SEP2, SEP3 and SEP4 genes, which mediate class E function, are required for the formation of all four types of floral organs. In floral organ determination, class A + E genes specify sepals, A + B + E specify petals, B + C + E specify stamens, C + E specify carpels and C + D + E specify ovules; these functions are mediated by protein interactions with each other and the formation of tetrameric complexes [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>, <a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. The heterodimeric protein complexes formed by AP3 and PI play crucial roles in regulating petal and stamen development [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>, <a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. In Quercus suber trees, the interaction between QsPI and QsAP3 specifically promotes stamen development [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. The formation of dimeric protein complexes involving AP3-like and AP1-like proteins is essential for petal development in Z. armatum [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. These findings indicate that the heterodimerization of AP3 with flower organ homologous genes represents a conserved mechanism during plant male organ development. In the present study, the expression of the MADS-box genes AGL11, CAL, and AGL16 (homologous to AGL1 in Morus notabilis, Fig. S7) and FBP24 was found to be exclusively limited to female flowers but was completely absent in male flowers, which may be tightly interconnected with the lack of male organs in these flowers. Although TM6 (AP3-like) and PI exhibit male-specific expression, CAL (homologous to AP1, Fig. S7) appears to be suppressed in male flowers, suggesting that the inability to form heterodimeric proteins between TM6 and CAL leads to a loss of petals in the male flower of Bama Huoma.</p>
<p id="Par42">Previous studies have reported that MYB35 is specifically expressed in males of A. officinalis and has a loss of function on the X chromosome [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. MYB35 plays a vital role in tapetal differentiation and function. In Arabidopsis, MYB35 is a key component involved in controlling callose dissolution and acts downstream of DYT1 and upstream of AMS and AtMYB103 in the transcriptional regulatory networks that regulate tapetal development [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. In the present study, MYB35 was highly expressed in male flowers and might interact with MADS-box genes to regulate male flower development in Bama Huoma. Our findings suggest that the identities of different types of floral organs are specified by homeotic MADS elements that interact in a combinatorial manner, and the regulatory mechanisms involved in sex determination are being further investigated. In summary, we propose a simple model for the regulation of flower organ identity in the dioecious species Bama Huoma by the modulation of MADS-box and MYB TF functions (Fig. <a href="#Fig10" class="usa-link">10</a>). Elucidating the mechanism of sex determination in Bama Huoma is economically important for improving the seed yield of Bama Huoma and provides a theoretical basis for understanding the mechanism of sex determination in other dioecious plants.</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369072_12870_2025_7096_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/50f1/12369072/30c87362f519/12870_2025_7096_Fig10_HTML.jpg" loading="lazy" id="MO10" height="349" width="777" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Model of the putative roles of sex-related genes in floral organ identity in the dioecious plant Bama Huoma. <strong>A</strong> The ABCDE model is regulated by MADS-box genes during the development of floral organs. The expression of genes related to stamen development is suppressed in female Bama Huoma flowers, whereas that of genes related to pistil development is inhibited in male flowers. <strong>B</strong> The AGL11/AGL16/CAL/FBP24 complex regulates female flower development, whereas the MYB35/TM6/AGL104/PMADS2 complex regulates male development. According to this hypothesis, sex differentiation in Bama Huoma could be determined by MADS genes and MYB35. However, further investigations are needed to elucidate the detailed mechanism involved. FF, female factor; MF, male factor. The dashed line indicates that gene expression was repressed</p></figcaption></figure></section></section><section id="Sec26"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par43">Bama Huoma is a typical dioecious plant whose sex determination is controlled by the X and Y chromosomes; the female and male karyotypes are XX and XY, respectively. In the present study, we observed the development of female and male flowers by optical microscopy, SEM and TEM, and whole-transcriptome sequencing analysis was employed to identify the candidate genes involved in sex differentiation in Bama Huoma. The results revealed that the developmental processes of the female and male flowers were independent, implying that sex differentiation occurred before flower development. Transcriptome analysis and qRT‒PCR revealed that the MADS-box and MYB genes were expressed in a sex-biased and tissue-specific manner in female or male flowers. Among these genes, AGL11, CAL, FBP24 and AGL16 were specifically expressed in female flowers, whereas MYB35, PMADS2, TM6 and AGL104 were specifically expressed in male flowers, suggesting that the identities of different types of floral organs act in a combinatorial manner to regulate sex differentiation in Bama Huoma. Taken together, our findings provide a comprehensive gene expression profile of floral organ sex differentiation in Bama Huoma and a theoretical foundation for further research on the genetic mechanisms of sex determination in dioecious plants.</p></section><section id="Sec27"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (7MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (12.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (12.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4.</a><sup> (11.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5.</a><sup> (13.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6.</a><sup> (6.7MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7.</a><sup> (3.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8.</a><sup> (13.5KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Not applicable.</p></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>X.L. conducted the experiments and drafted the manuscript. J.Z. assisted in figure drawing. W.H., X.T., B. H. and J.T. assisted with the experiments. K.T. performed bioinformatic analysis and edited the manuscript. Y.Z. revised the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the National Natural Science Foundation of China (32260521 and 32401817) and the Basic Business Expenses Project of Guangxi Academy of Agricultural Sciences (Guinongke 2024YP054, Guinongke2021YT061, GuinongkeJZ2025KJ001, and GuiTS2022019).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Sequence data that support the findings of this study have been deposited in the Sequence Read Archive (SRA) of the National Center for Biotechnology Information (NCBI) with the primary accession code PRJNA1025044.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Ethics approval and consent to participate</h2>
<p id="Par44">Not applicable.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Consent for publication</h2>
<p id="Par45">Not applicable.</p></section><section id="FPar3"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par46">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Xiaofang Liao and Jie Zheng contributed equally to this work.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Yanhong Zhao, Email: zhaoyanhong402@163.com.</p>
<p>Kailiang Teng, Email: Teng_Thd@163.com.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Ming R, Bendahmane A, Renner SS. Sex chromosomes in land plants. Annu Rev Plant Biol. 2011;62:485–514.
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-042110-103914" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21526970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ming%20R,%20Bendahmane%20A,%20Renner%20SS.%20Sex%20chromosomes%20in%20land%20plants.%20Annu%20Rev%20Plant%20Biol.%202011;62:485%E2%80%93514." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Akagi T, Pilkington SM, Varkonyi-Gasic E, Henry IM, Sugano SS, Sonoda M, Firl A, McNeilage MA, Douglas MJ, Wang T, et al. Two Y-chromosome-encoded genes determine sex in kiwifruit. Nat Plants. 2019;5(8):801–9.
</cite> [<a href="https://doi.org/10.1038/s41477-019-0489-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31383971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akagi%20T,%20Pilkington%20SM,%20Varkonyi-Gasic%20E,%20Henry%20IM,%20Sugano%20SS,%20Sonoda%20M,%20Firl%20A,%20McNeilage%20MA,%20Douglas%20MJ,%20Wang%20T,%20et%20al.%20Two%20Y-chromosome-encoded%20genes%20determine%20sex%20in%20kiwifruit.%20Nat%20Plants.%202019;5(8):801%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Picq S, Santoni S, Lacombe T, Latreille M, Weber A, Ardisson M, Ivorra S, Maghradze D, Arroyo-Garcia R, Chatelet P, et al. A small XY chromosomal region explains sex determination in wild dioecious <em>V. vinifera</em> and the reversal to hermaphroditism in domesticated grapevines. BMC Plant Biol. 2014;14: 229.
</cite> [<a href="https://doi.org/10.1186/s12870-014-0229-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4167142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25179565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Picq%20S,%20Santoni%20S,%20Lacombe%20T,%20Latreille%20M,%20Weber%20A,%20Ardisson%20M,%20Ivorra%20S,%20Maghradze%20D,%20Arroyo-Garcia%20R,%20Chatelet%20P,%20et%20al.%20A%20small%20XY%20chromosomal%20region%20explains%20sex%20determination%20in%20wild%20dioecious%20V.%20vinifera%20and%20the%20reversal%20to%20hermaphroditism%20in%20domesticated%20grapevines.%20BMC%20Plant%20Biol.%202014;14:%20229." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Scharmann M, Grafe TU, Metali F, Widmer A. Sex is determined by XY chromosomes across the radiation of dioecious <em>Nepenthes</em> pitcher plants. Evol Lett. 2019;3(6):586–97.
</cite> [<a href="https://doi.org/10.1002/evl3.142" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6906984/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31867120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scharmann%20M,%20Grafe%20TU,%20Metali%20F,%20Widmer%20A.%20Sex%20is%20determined%20by%20XY%20chromosomes%20across%20the%20radiation%20of%20dioecious%20Nepenthes%20pitcher%20plants.%20Evol%20Lett.%202019;3(6):586%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Akagi T, Henry IM, Tao R, Comai L. Plant genetics. A Y-chromosome-encoded small RNA acts as a sex determinant in persimmons. Science. 2014; 346(6209): 646–50.</cite> [<a href="https://doi.org/10.1126/science.1257225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25359977/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Akagi T, Henry IM, Kawai T, Comai L, Tao R. Epigenetic regulation of the sex determination gene MeGI in polyploid persimmon. Plant Cell. 2016;28(12):2905–15.
</cite> [<a href="https://doi.org/10.1105/tpc.16.00532" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5240738/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27956470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akagi%20T,%20Henry%20IM,%20Kawai%20T,%20Comai%20L,%20Tao%20R.%20Epigenetic%20regulation%20of%20the%20sex%20determination%20gene%20MeGI%20in%20polyploid%20persimmon.%20Plant%20Cell.%202016;28(12):2905%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Marsch-Martinez N, Ramos-Cruz D, Irepan RJ, Lozano-Sotomayor P, Zuniga-Mayo VM, De Folter S. The role of cytokinin during Arabidopsis gynoecia and fruit morphogenesis and patterning. Plant J. 2012;72(2):222–34.
</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2012.05062.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22640521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marsch-Martinez%20N,%20Ramos-Cruz%20D,%20Irepan%20RJ,%20Lozano-Sotomayor%20P,%20Zuniga-Mayo%20VM,%20De%20Folter%20S.%20The%20role%20of%20cytokinin%20during%20Arabidopsis%20gynoecia%20and%20fruit%20morphogenesis%20and%20patterning.%20Plant%20J.%202012;72(2):222%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Hazzouri KM, Gros-Balthazard M, Flowers JM, Copetti D, Lemansour A, Lebrun M, Masmoudi K, Ferrand S, Dhar MI, Fresquez ZA, et al. Genome-wide association mapping of date palm fruit traits. Nat Commun. 2019;10(1):4680.
</cite> [<a href="https://doi.org/10.1038/s41467-019-12604-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6794320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31615981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hazzouri%20KM,%20Gros-Balthazard%20M,%20Flowers%20JM,%20Copetti%20D,%20Lemansour%20A,%20Lebrun%20M,%20Masmoudi%20K,%20Ferrand%20S,%20Dhar%20MI,%20Fresquez%20ZA,%20et%20al.%20Genome-wide%20association%20mapping%20of%20date%20palm%20fruit%20traits.%20Nat%20Commun.%202019;10(1):4680." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Liao Q, Du R, Gou J, Guo L, Shen H, Liu H, Nguyen JK, Ming R, Yin T, Huang S, et al. The genomic architecture of the sex-determining region and sex-related metabolic variation in <em>Ginkgo biloba</em>. Plant J. 2020;104(5):1399–409.
</cite> [<a href="https://doi.org/10.1111/tpj.15009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33015884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liao%20Q,%20Du%20R,%20Gou%20J,%20Guo%20L,%20Shen%20H,%20Liu%20H,%20Nguyen%20JK,%20Ming%20R,%20Yin%20T,%20Huang%20S,%20et%20al.%20The%20genomic%20architecture%20of%20the%20sex-determining%20region%20and%20sex-related%20metabolic%20variation%20in%20Ginkgo%20biloba.%20Plant%20J.%202020;104(5):1399%E2%80%93409." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Yang HW, Akagi T, Kawakatsu T, Tao R. Gene networks orchestrated by MeGI: a single-factor mechanism underlying sex determination in persimmon. Plant J. 2019;98(1):97–111.
</cite> [<a href="https://doi.org/10.1111/tpj.14202" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6850717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30556936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20HW,%20Akagi%20T,%20Kawakatsu%20T,%20Tao%20R.%20Gene%20networks%20orchestrated%20by%20MeGI:%20a%20single-factor%20mechanism%20underlying%20sex%20determination%20in%20persimmon.%20Plant%20J.%202019;98(1):97%E2%80%93111." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Akagi T, Henry IM, Ohtani H, Morimoto T, Beppu K, Kataoka I, Tao R. A y-encoded suppressor of feminization arose via lineage-specific duplication of a cytokinin response regulator in kiwifruit. Plant Cell. 2018;30(4):780–95.
</cite> [<a href="https://doi.org/10.1105/tpc.17.00787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5969274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29626069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akagi%20T,%20Henry%20IM,%20Ohtani%20H,%20Morimoto%20T,%20Beppu%20K,%20Kataoka%20I,%20Tao%20R.%20A%20y-encoded%20suppressor%20of%20feminization%20arose%20via%20lineage-specific%20duplication%20of%20a%20cytokinin%20response%20regulator%20in%20kiwifruit.%20Plant%20Cell.%202018;30(4):780%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Badouin H, Velt A, Gindraud F, Flutre T, Dumas V, Vautrin S, Marande W, Corbi J, Sallet E, Ganofsky J, et al. The wild grape genome sequence provides insights into the transition from dioecy to hermaphroditism during grape domestication. Genome Biol. 2020;21(1):223.
</cite> [<a href="https://doi.org/10.1186/s13059-020-02131-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7487632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32892750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Badouin%20H,%20Velt%20A,%20Gindraud%20F,%20Flutre%20T,%20Dumas%20V,%20Vautrin%20S,%20Marande%20W,%20Corbi%20J,%20Sallet%20E,%20Ganofsky%20J,%20et%20al.%20The%20wild%20grape%20genome%20sequence%20provides%20insights%20into%20the%20transition%20from%20dioecy%20to%20hermaphroditism%20during%20grape%20domestication.%20Genome%20Biol.%202020;21(1):223." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Torres MF, Mathew LS, Ahmed I, Al-Azwani IK, Krueger R, Rivera-Nunez D, Mohamoud YA, Clark AG, Suhre K, Malek JA. Genus-wide sequencing supports a two-locus model for sex-determination in <em>Phoenix</em>. Nat Commun. 2018;9(1): 3969.
</cite> [<a href="https://doi.org/10.1038/s41467-018-06375-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6162277/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30266991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Torres%20MF,%20Mathew%20LS,%20Ahmed%20I,%20Al-Azwani%20IK,%20Krueger%20R,%20Rivera-Nunez%20D,%20Mohamoud%20YA,%20Clark%20AG,%20Suhre%20K,%20Malek%20JA.%20Genus-wide%20sequencing%20supports%20a%20two-locus%20model%20for%20sex-determination%20in%20Phoenix.%20Nat%20Commun.%202018;9(1):%203969." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Jaligot E, Adler S, Debladis E, Beule T, Richaud F, Ilbert P, Finnegan EJ, Rival A. Epigenetic imbalance and the floral developmental abnormality of the in vitro-regenerated oil palm <em>Elaeis guineensis</em>. Ann Bot. 2011;108(8):1453–62.
</cite> [<a href="https://doi.org/10.1093/aob/mcq266" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3219487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21224269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jaligot%20E,%20Adler%20S,%20Debladis%20E,%20Beule%20T,%20Richaud%20F,%20Ilbert%20P,%20Finnegan%20EJ,%20Rival%20A.%20Epigenetic%20imbalance%20and%20the%20floral%20developmental%20abnormality%20of%20the%20in%20vitro-regenerated%20oil%20palm%20Elaeis%20guineensis.%20Ann%20Bot.%202011;108(8):1453%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Martin A, Troadec C, Boualem A, Rajab M, Fernandez R, Morin H, Pitrat M, Dogimont C, Bendahmane A. A transposon-induced epigenetic change leads to sex determination in melon. Nature. 2009;461(7267):1135–8.
</cite> [<a href="https://doi.org/10.1038/nature08498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19847267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martin%20A,%20Troadec%20C,%20Boualem%20A,%20Rajab%20M,%20Fernandez%20R,%20Morin%20H,%20Pitrat%20M,%20Dogimont%20C,%20Bendahmane%20A.%20A%20transposon-induced%20epigenetic%20change%20leads%20to%20sex%20determination%20in%20melon.%20Nature.%202009;461(7267):1135%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Aamir M, Karmakar P, Singh VK, Kashyap SP, Pandey S, Singh BK, Singh PM, Singh J. A novel insight into transcriptional and epigenetic regulation underlying sex expression and flower development in melon (<em>Cucumis melo</em> L.). Physiol Plant. 2021;173(4):1729–64.
</cite> [<a href="https://doi.org/10.1111/ppl.13357" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33547804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aamir%20M,%20Karmakar%20P,%20Singh%20VK,%20Kashyap%20SP,%20Pandey%20S,%20Singh%20BK,%20Singh%20PM,%20Singh%20J.%20A%20novel%20insight%20into%20transcriptional%20and%20epigenetic%20regulation%20underlying%20sex%20expression%20and%20flower%20development%20in%20melon%20(Cucumis%20melo%20L.).%20Physiol%20Plant.%202021;173(4):1729%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Parkinson SE, Gross SM, Hollick JB. Maize sex determination and abaxial leaf fates are canalized by a factor that maintains repressed epigenetic states. Dev Biol. 2007;308(2):462–73.
</cite> [<a href="https://doi.org/10.1016/j.ydbio.2007.06.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17612519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Parkinson%20SE,%20Gross%20SM,%20Hollick%20JB.%20Maize%20sex%20determination%20and%20abaxial%20leaf%20fates%20are%20canalized%20by%20a%20factor%20that%20maintains%20repressed%20epigenetic%20states.%20Dev%20Biol.%202007;308(2):462%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Lai YS, Shen D, Zhang W, Zhang X, Qiu Y, Wang H, Dou X, Li S, Wu Y, Song J, et al. Temperature and photoperiod changes affect cucumber sex expression by different epigenetic regulations. BMC Plant Biol. 2018;18(1):268.
</cite> [<a href="https://doi.org/10.1186/s12870-018-1490-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6220452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30400867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lai%20YS,%20Shen%20D,%20Zhang%20W,%20Zhang%20X,%20Qiu%20Y,%20Wang%20H,%20Dou%20X,%20Li%20S,%20Wu%20Y,%20Song%20J,%20et%20al.%20Temperature%20and%20photoperiod%20changes%20affect%20cucumber%20sex%20expression%20by%20different%20epigenetic%20regulations.%20BMC%20Plant%20Biol.%202018;18(1):268." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Xu G, Huang J, Yang Y, Yao YA. Transcriptome analysis of flower sex differentiation in <em>Jatropha curcas</em> L. using RNA sequencing. PLoS One. 2016;11(2): e145613.</cite> [<a href="https://doi.org/10.1371/journal.pone.0145613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4746058/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26848843/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20G,%20Huang%20J,%20Yang%20Y,%20Yao%20YA.%20Transcriptome%20analysis%20of%20flower%20sex%20differentiation%20in%20Jatropha%20curcas%20L.%20using%20RNA%20sequencing.%20PLoS%20One.%202016;11(2):%20e145613." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Liu Z, Wang H, Xu Z, Zhang H, Li G, Wang X, Qian W. Transcriptome profiling of differentially expressed genes of male and female inflorescences in spinach (<em>Spinacia oleracea</em> L.). Genome. 2021;64(8):777–88.
</cite> [<a href="https://doi.org/10.1139/gen-2020-0122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33539259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20Z,%20Wang%20H,%20Xu%20Z,%20Zhang%20H,%20Li%20G,%20Wang%20X,%20Qian%20W.%20Transcriptome%20profiling%20of%20differentially%20expressed%20genes%20of%20male%20and%20female%20inflorescences%20in%20spinach%20(Spinacia%20oleracea%20L.).%20Genome.%202021;64(8):777%E2%80%9388." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Hisanaga T, Okahashi K, Yamaoka S, Kajiwara T, Nishihama R, Shimamura M, Yamato KT, Bowman JL, Kohchi T, Nakajima K. A cis-acting bidirectional transcription switch controls sexual dimorphism in the liverwort. EMBO J. 2019;38(6): e100240.
</cite> [<a href="https://doi.org/10.15252/embj.2018100240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6418429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30609993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hisanaga%20T,%20Okahashi%20K,%20Yamaoka%20S,%20Kajiwara%20T,%20Nishihama%20R,%20Shimamura%20M,%20Yamato%20KT,%20Bowman%20JL,%20Kohchi%20T,%20Nakajima%20K.%20A%20cis-acting%20bidirectional%20transcription%20switch%20controls%20sexual%20dimorphism%20in%20the%20liverwort.%20EMBO%20J.%202019;38(6):%20e100240." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Huang X, Yue Y, Sun J, Peng H, Yang Z, Bao M, Hu H. Characterization of a fertility-related SANT/MYB gene (PhRL) from petunia. Sci Hortic. 2015;183:152–9.</cite> [<a href="https://scholar.google.com/scholar_lookup?Huang%20X,%20Yue%20Y,%20Sun%20J,%20Peng%20H,%20Yang%20Z,%20Bao%20M,%20Hu%20H.%20Characterization%20of%20a%20fertility-related%20SANT/MYB%20gene%20(PhRL)%20from%20petunia.%20Sci%20Hortic.%202015;183:152%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Tang N, Cao Z, Wu P, Zhang X, Lou J, Liu Y, Wang Q, Hu Y, Si S, Sun X, et al. Genome-wide identification, interaction of the MADS-box proteins in <em>Zanthoxylum armatum</em> and functional characterization of ZaMADS80 in floral development. Front Plant Sci. 2022;13:1038828.
</cite> [<a href="https://doi.org/10.3389/fpls.2022.1038828" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9732391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36507394/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang%20N,%20Cao%20Z,%20Wu%20P,%20Zhang%20X,%20Lou%20J,%20Liu%20Y,%20Wang%20Q,%20Hu%20Y,%20Si%20S,%20Sun%20X,%20et%20al.%20Genome-wide%20identification,%20interaction%20of%20the%20MADS-box%20proteins%20in%20Zanthoxylum%20armatum%20and%20functional%20characterization%20of%20ZaMADS80%20in%20floral%20development.%20Front%20Plant%20Sci.%202022;13:1038828." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Guan H, Wang H, Huang J, Liu M, Chen T, Shan X, Chen H, Shen J. Genome-wide identification and expression analysis of MADS-box family genes in litchi (<em>Litchi chinensis</em> Sonn.) and their involvement in floral sex determination. Plants. 2021;10(10): 2142.
</cite> [<a href="https://doi.org/10.3390/plants10102142" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8540616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34685951/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guan%20H,%20Wang%20H,%20Huang%20J,%20Liu%20M,%20Chen%20T,%20Shan%20X,%20Chen%20H,%20Shen%20J.%20Genome-wide%20identification%20and%20expression%20analysis%20of%20MADS-box%20family%20genes%20in%20litchi%20(Litchi%20chinensis%20Sonn.)%20and%20their%20involvement%20in%20floral%20sex%20determination.%20Plants.%202021;10(10):%202142." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Daminato M, Masiero S, Resentini F, Lovisetto A, Casadoro G. Characterization of TM8, a MADS-box gene expressed in tomato flowers. BMC Plant Biol. 2014;14:319.
</cite> [<a href="https://doi.org/10.1186/s12870-014-0319-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4258831/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25433802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Daminato%20M,%20Masiero%20S,%20Resentini%20F,%20Lovisetto%20A,%20Casadoro%20G.%20Characterization%20of%20TM8,%20a%20MADS-box%20gene%20expressed%20in%20tomato%20flowers.%20BMC%20Plant%20Biol.%202014;14:319." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Shi J, Toscani M, Dowling CA, Schilling S, Melzer R. Identification of genes associated with sex expression and sex determination in hemp (<em>Cannabis sativa</em> L.). J Exp Bot. 2025;76(1):175–90.
</cite> [<a href="https://doi.org/10.1093/jxb/erae429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11659178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39468733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi%20J,%20Toscani%20M,%20Dowling%20CA,%20Schilling%20S,%20Melzer%20R.%20Identification%20of%20genes%20associated%20with%20sex%20expression%20and%20sex%20determination%20in%20hemp%20(Cannabis%20sativa%20L.).%20J%20Exp%20Bot.%202025;76(1):175%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Murase K, Shigenobu S, Fujii S, Ueda K, Murata T, Sakamoto A, Wada Y, Yamaguchi K, Osakabe Y, Osakabe K, et al. <em>MYB</em> transcription factor gene involved in sex determination in <em>Asparagus officinalis</em>. Genes Cells. 2017;22(1):115–23.
</cite> [<a href="https://doi.org/10.1111/gtc.12453" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27869347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murase%20K,%20Shigenobu%20S,%20Fujii%20S,%20Ueda%20K,%20Murata%20T,%20Sakamoto%20A,%20Wada%20Y,%20Yamaguchi%20K,%20Osakabe%20Y,%20Osakabe%20K,%20et%20al.%20MYB%20transcription%20factor%20gene%20involved%20in%20sex%20determination%20in%20Asparagus%20officinalis.%20Genes%20Cells.%202017;22(1):115%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Tsugama D, Matsuyama K, Ide M, Hayashi M, Fujino K, Masuda K. A putative MYB35 ortholog is a candidate for the sex-determining genes in <em>Asparagus officinalis</em>. Sci Rep. 2017;7: 41497.
</cite> [<a href="https://doi.org/10.1038/srep41497" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5296871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28176806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsugama%20D,%20Matsuyama%20K,%20Ide%20M,%20Hayashi%20M,%20Fujino%20K,%20Masuda%20K.%20A%20putative%20MYB35%20ortholog%20is%20a%20candidate%20for%20the%20sex-determining%20genes%20in%20Asparagus%20officinalis.%20Sci%20Rep.%202017;7:%2041497." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Chen T, He J, Zhang J, Zhang H, Qian P, Hao J, Li L. Analytical characterization of hempseed (seed of <em>Cannabis sativa</em> L.) oil from eight regions in China. J Diet Suppl. 2010;7(2):117–29.
</cite> [<a href="https://doi.org/10.3109/19390211003781669" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22435611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20T,%20He%20J,%20Zhang%20J,%20Zhang%20H,%20Qian%20P,%20Hao%20J,%20Li%20L.%20Analytical%20characterization%20of%20hempseed%20(seed%20of%20Cannabis%20sativa%20L.)%20oil%20from%20eight%20regions%20in%20China.%20J%20Diet%20Suppl.%202010;7(2):117%E2%80%9329." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Xue R, Du M, Zhou TY, Ai WZ, Zhang ZS, Xiang XW, Zhou YF, Wen ZS. Polysaccharides from hemp seed protect against cyclophosphamide-induced intestinal oxidative damage via Nrf2-Keap1 signaling pathway in mice. Oxid Med Cell Longev. 2020;2020:1813798.
</cite> [<a href="https://doi.org/10.1155/2020/1813798" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7468657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32908623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xue%20R,%20Du%20M,%20Zhou%20TY,%20Ai%20WZ,%20Zhang%20ZS,%20Xiang%20XW,%20Zhou%20YF,%20Wen%20ZS.%20Polysaccharides%20from%20hemp%20seed%20protect%20against%20cyclophosphamide-induced%20intestinal%20oxidative%20damage%20via%20Nrf2-Keap1%20signaling%20pathway%20in%20mice.%20Oxid%20Med%20Cell%20Longev.%202020;2020:1813798." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Divashuk MG, Alexandrov OS, Razumova OV, Kirov IV, Karlov GI, Marais GAB. Molecular cytogenetic characterization of the dioecious <em>Cannabis sativa</em> with an XY chromosome sex determination system. PLoS One. 2014;9(1): e85118.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0085118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3897423/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24465491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Divashuk%20MG,%20Alexandrov%20OS,%20Razumova%20OV,%20Kirov%20IV,%20Karlov%20GI,%20Marais%20GAB.%20Molecular%20cytogenetic%20characterization%20of%20the%20dioecious%20Cannabis%20sativa%20with%20an%20XY%20chromosome%20sex%20determination%20system.%20PLoS%20One.%202014;9(1):%20e85118." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Ashfaq S, Zafar M, Ahmad M, Sultana S, Bahadur S, Khan A, Shah A. Microscopic investigations of palynological features of convolvulaceous species from arid zone of Pakistan. Microsc Res Tech. 2018;81(2):228–39.
</cite> [<a href="https://doi.org/10.1002/jemt.22971" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29160011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ashfaq%20S,%20Zafar%20M,%20Ahmad%20M,%20Sultana%20S,%20Bahadur%20S,%20Khan%20A,%20Shah%20A.%20Microscopic%20investigations%20of%20palynological%20features%20of%20convolvulaceous%20species%20from%20arid%20zone%20of%20Pakistan.%20Microsc%20Res%20Tech.%202018;81(2):228%E2%80%9339." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Martin M. Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet J. 2011;17(1):10.</cite> [<a href="https://scholar.google.com/scholar_lookup?Martin%20M.%20Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads.%20EMBnet%20J.%202011;17(1):10." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Langmead B, Salzberg SL. Fast gapped-read alignment with bowtie 2. Nat Methods. 2012;9(4):357–9.
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead%20B,%20Salzberg%20SL.%20Fast%20gapped-read%20alignment%20with%20bowtie%202.%20Nat%20Methods.%202012;9(4):357%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Kim D, Langmead B, Salzberg SL. Hisat: a fast spliced aligner with low memory requirements. Nat Methods. 2015;12(4):357–60.
</cite> [<a href="https://doi.org/10.1038/nmeth.3317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4655817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25751142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20D,%20Langmead%20B,%20Salzberg%20SL.%20Hisat:%20a%20fast%20spliced%20aligner%20with%20low%20memory%20requirements.%20Nat%20Methods.%202015;12(4):357%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Pertea M, Pertea GM, Antonescu CM, Chang TC, Mendell JT, Salzberg SL. Stringtie enables improved reconstruction of a transcriptome from RNA-seq reads. Nat Biotechnol. 2015;33(3):290–5.
</cite> [<a href="https://doi.org/10.1038/nbt.3122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4643835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25690850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pertea%20M,%20Pertea%20GM,%20Antonescu%20CM,%20Chang%20TC,%20Mendell%20JT,%20Salzberg%20SL.%20Stringtie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads.%20Nat%20Biotechnol.%202015;33(3):290%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Trapnell C, Williams BA, Pertea G, Mortazavi A, Kwan G, van Baren MJ, Salzberg SL, Wold BJ, Pachter L. Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. Nat Biotechnol. 2010;28(5):511–5.
</cite> [<a href="https://doi.org/10.1038/nbt.1621" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3146043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20436464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trapnell%20C,%20Williams%20BA,%20Pertea%20G,%20Mortazavi%20A,%20Kwan%20G,%20van%20Baren%20MJ,%20Salzberg%20SL,%20Wold%20BJ,%20Pachter%20L.%20Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation.%20Nat%20Biotechnol.%202010;28(5):511%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Robinson MD, McCarthy DJ, Smyth GK. edgeR: a bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics. 2010;26(1):139–40.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2796818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Robinson%20MD,%20McCarthy%20DJ,%20Smyth%20GK.%20edgeR:%20a%20bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data.%20Bioinformatics.%202010;26(1):139%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Jones DT, Taylor WR, Thornton JM. The rapid generation of mutation data matrices from protein sequences. Comput Appl Biosci. 1992;8(3):275–82.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/8.3.275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1633570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jones%20DT,%20Taylor%20WR,%20Thornton%20JM.%20The%20rapid%20generation%20of%20mutation%20data%20matrices%20from%20protein%20sequences.%20Comput%20Appl%20Biosci.%201992;8(3):275%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Chen C, Wu Y, Li J, Wang X, Zeng Z, Xu J, Liu Y, Feng J, Chen H, He Y, R X. TBtools-II: A “one for all, all for one” bioinformatics platform for biological big-data mining. Mol Plant. 2023; 16(11): 1733–1742.</cite> [<a href="https://doi.org/10.1016/j.molp.2023.09.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37740491/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Rosas-Cardenas FF, Duran-Figueroa N, Vielle-Calzada JP, Cruz-Hernandez A, Marsch-Martinez N, De Folter S. A simple and efficient method for isolating small RNAs from different plant species. Plant Methods. 2011;7:4.
</cite> [<a href="https://doi.org/10.1186/1746-4811-7-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3056851/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21349188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosas-Cardenas%20FF,%20Duran-Figueroa%20N,%20Vielle-Calzada%20JP,%20Cruz-Hernandez%20A,%20Marsch-Martinez%20N,%20De%20Folter%20S.%20A%20simple%20and%20efficient%20method%20for%20isolating%20small%20RNAs%20from%20different%20plant%20species.%20Plant%20Methods.%202011;7:4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Liao X, Wei M, Khan A, Zhao Y, Kong X, Zhou B, Li M, Peng S, Munsif F, Ullah A, et al. Comparative analysis of mitochondrial genome and expression variation between UG93A and UG93B reveals a candidate gene related to cytoplasmic male sterility in kenaf. Ind Crop Prod. 2020;152: 112502.</cite> [<a href="https://scholar.google.com/scholar_lookup?Liao%20X,%20Wei%20M,%20Khan%20A,%20Zhao%20Y,%20Kong%20X,%20Zhou%20B,%20Li%20M,%20Peng%20S,%20Munsif%20F,%20Ullah%20A,%20et%20al.%20Comparative%20analysis%20of%20mitochondrial%20genome%20and%20expression%20variation%20between%20UG93A%20and%20UG93B%20reveals%20a%20candidate%20gene%20related%20to%20cytoplasmic%20male%20sterility%20in%20kenaf.%20Ind%20Crop%20Prod.%202020;152:%20112502." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Chen X, Chen B, Shang X, Fang S. RNA <em>in situ</em> hybridization and expression of related genes regulating the accumulation of triterpenoids in <em>Cyclocarya paliurus</em>. Tree Physiol. 2021;41(11):2189–97.
</cite> [<a href="https://doi.org/10.1093/treephys/tpab067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33960380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20X,%20Chen%20B,%20Shang%20X,%20Fang%20S.%20RNA%20in%20situ%20hybridization%20and%20expression%20of%20related%20genes%20regulating%20the%20accumulation%20of%20triterpenoids%20in%20Cyclocarya%20paliurus.%20Tree%20Physiol.%202021;41(11):2189%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Li F, Wang Y, Gao H, Zhang X, Zhuang N. Comparative transcriptome analysis reveals differential gene expression in sterile and fertile rubber tree varieties during flower bud differentiation. J Plant Physiol. 2021;265: 153506.
</cite> [<a href="https://doi.org/10.1016/j.jplph.2021.153506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34492526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20F,%20Wang%20Y,%20Gao%20H,%20Zhang%20X,%20Zhuang%20N.%20Comparative%20transcriptome%20analysis%20reveals%20differential%20gene%20expression%20in%20sterile%20and%20fertile%20rubber%20tree%20varieties%20during%20flower%20bud%20differentiation.%20J%20Plant%20Physiol.%202021;265:%20153506." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Liang Y, Tan ZM, Zhu L, Niu QK, Zhou JJ, Li M, Chen LQ, Zhang XQ, Ye D. <em>MYB</em>97, <em>MYB</em>101 and <em>MYB</em>120 function as male factors that control pollen tube-synergid interaction in <em>Arabidopsis thaliana</em> fertilization. PLoS Genet. 2013;9(11): e1003933.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1003933" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3836714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24278028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liang%20Y,%20Tan%20ZM,%20Zhu%20L,%20Niu%20QK,%20Zhou%20JJ,%20Li%20M,%20Chen%20LQ,%20Zhang%20XQ,%20Ye%20D.%20MYB97,%20MYB101%20and%20MYB120%20function%20as%20male%20factors%20that%20control%20pollen%20tube-synergid%20interaction%20in%20Arabidopsis%20thaliana%20fertilization.%20PLoS%20Genet.%202013;9(11):%20e1003933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Becker A, Theissen G. The major clades of MADS-box genes and their role in the development and evolution of flowering plants. Mol Phylogenet Evol. 2003;29(3):464–89.
</cite> [<a href="https://doi.org/10.1016/s1055-7903(03)00207-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14615187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Becker%20A,%20Theissen%20G.%20The%20major%20clades%20of%20MADS-box%20genes%20and%20their%20role%20in%20the%20development%20and%20evolution%20of%20flowering%20plants.%20Mol%20Phylogenet%20Evol.%202003;29(3):464%E2%80%9389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Zhu J, Chen H, Li H, Gao JF, Jiang H, Wang C, Guan YF, Yang ZN. Defective in tapetal development and function 1 is essential for anther development and tapetal function for microspore maturation in Arabidopsis. Plant J. 2008;55(2):266–77.
</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2008.03500.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18397379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20J,%20Chen%20H,%20Li%20H,%20Gao%20JF,%20Jiang%20H,%20Wang%20C,%20Guan%20YF,%20Yang%20ZN.%20Defective%20in%20tapetal%20development%20and%20function%201%20is%20essential%20for%20anther%20development%20and%20tapetal%20function%20for%20microspore%20maturation%20in%20Arabidopsis.%20Plant%20J.%202008;55(2):266%E2%80%9377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Zhu J, Lou Y, Xu X, Yang Z. A genetic pathway for tapetum development and function in <em>Arabidopsis</em>. J Integr Plant Biol. 2011;53(11):892–900.
</cite> [<a href="https://doi.org/10.1111/j.1744-7909.2011.01078.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21957980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20J,%20Lou%20Y,%20Xu%20X,%20Yang%20Z.%20A%20genetic%20pathway%20for%20tapetum%20development%20and%20function%20in%20Arabidopsis.%20J%20Integr%20Plant%20Biol.%202011;53(11):892%E2%80%93900." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Yan W, Chen D, Kaufmann K. Molecular mechanisms of floral organ specification by MADS domain proteins. Curr Opin Plant Biol. 2016;29:154–62.
</cite> [<a href="https://doi.org/10.1016/j.pbi.2015.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26802807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan%20W,%20Chen%20D,%20Kaufmann%20K.%20Molecular%20mechanisms%20of%20floral%20organ%20specification%20by%20MADS%20domain%20proteins.%20Curr%20Opin%20Plant%20Biol.%202016;29:154%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Puterova J, Kubat Z, Kejnovsky E, Jesionek W, Cizkova J, Vyskot B, Hobza R. The slowdown of Y chromosome expansion in dioecious <em>Silene latifolia</em> due to DNA loss and male-specific silencing of retrotransposons. BMC Genomics. 2018;19(1):153.
</cite> [<a href="https://doi.org/10.1186/s12864-018-4547-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5819184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29458354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Puterova%20J,%20Kubat%20Z,%20Kejnovsky%20E,%20Jesionek%20W,%20Cizkova%20J,%20Vyskot%20B,%20Hobza%20R.%20The%20slowdown%20of%20Y%20chromosome%20expansion%20in%20dioecious%20Silene%20latifolia%20due%20to%20DNA%20loss%20and%20male-specific%20silencing%20of%20retrotransposons.%20BMC%20Genomics.%202018;19(1):153." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Harkess A, Huang K, van der Hulst R, Tissen B, Caplan JL, Koppula A, Batish M, Meyers BC, Leebens-Mack J. Sex determination by two Y-linked genes in garden Asparagus. Plant Cell. 2020;32(6):1790–6.
</cite> [<a href="https://doi.org/10.1105/tpc.19.00859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7268802/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32220850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harkess%20A,%20Huang%20K,%20van%20der%20Hulst%20R,%20Tissen%20B,%20Caplan%20JL,%20Koppula%20A,%20Batish%20M,%20Meyers%20BC,%20Leebens-Mack%20J.%20Sex%20determination%20by%20two%20Y-linked%20genes%20in%20garden%20Asparagus.%20Plant%20Cell.%202020;32(6):1790%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Spitzer-Rimon B, Duchin S, Bernstein N, Kamenetsky R. Architecture and florogenesis in female <em>Cannabis sativa</em> plants. Front Plant Sci. 2019;10:350.
</cite> [<a href="https://doi.org/10.3389/fpls.2019.00350" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6454139/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31001293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spitzer-Rimon%20B,%20Duchin%20S,%20Bernstein%20N,%20Kamenetsky%20R.%20Architecture%20and%20florogenesis%20in%20female%20Cannabis%20sativa%20plants.%20Front%20Plant%20Sci.%202019;10:350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Punja ZK, Holmes JE. Hermaphroditism in marijuana (<em>Cannabis sativa</em> L.) inflorescences- impact on floral morphology, seed formation, progeny sex ratios, and genetic variation. Front Plant Sci. 2020;11: 718.
</cite> [<a href="https://doi.org/10.3389/fpls.2020.00718" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7329997/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32670310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Punja%20ZK,%20Holmes%20JE.%20Hermaphroditism%20in%20marijuana%20(Cannabis%20sativa%20L.)%20inflorescences-%20impact%20on%20floral%20morphology,%20seed%20formation,%20progeny%20sex%20ratios,%20and%20genetic%20variation.%20Front%20Plant%20Sci.%202020;11:%20718." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Perry LE, Pannell JR, Dorken ME. Two’s company, three’s a crowd: experimental evaluation of the evolutionary maintenance of trioecy in <em>Mercurialis annua</em> (Euphorbiaceae). PLoS One. 2012;7(4): e35597.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0035597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3330815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22532862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perry%20LE,%20Pannell%20JR,%20Dorken%20ME.%20Two%E2%80%99s%20company,%20three%E2%80%99s%20a%20crowd:%20experimental%20evaluation%20of%20the%20evolutionary%20maintenance%20of%20trioecy%20in%20Mercurialis%20annua%20(Euphorbiaceae).%20PLoS%20One.%202012;7(4):%20e35597." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>DeDecker, J. Weighing the Risk of Cannabis Cross-Pollination. Michigan: Michigan State University. July 12, 2019, <a href="https://www.canr.msu.edu/news/weighing-the-risk-of-cannabiscross-pollination" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.canr.msu.edu/news/weighing-the-risk-of-cannabiscross-pollination</a> [<a href="https://perma.cc/D85H-S6JK" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://perma.cc/D85H-S6JK</a>].</cite>
</li>
<li id="CR56">
<span class="label">56.</span><cite>Cheng XQ, Zhang XY, Xue F, Zhu SH, Li YJ, Zhu QH, Liu F, Sun J. Characterization and transcriptome analysis of a dominant genic male sterile cotton mutant. BMC Plant Biol. 2020;20(1):312.
</cite> [<a href="https://doi.org/10.1186/s12870-020-02522-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7333317/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32620078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng%20XQ,%20Zhang%20XY,%20Xue%20F,%20Zhu%20SH,%20Li%20YJ,%20Zhu%20QH,%20Liu%20F,%20Sun%20J.%20Characterization%20and%20transcriptome%20analysis%20of%20a%20dominant%20genic%20male%20sterile%20cotton%20mutant.%20BMC%20Plant%20Biol.%202020;20(1):312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Mitsuda N, Seki M, Shinozaki K, Ohme-Takagi M. The NAC transcription factors NST1 and NST2 of Arabidopsis regulate secondary wall thickenings and are required for anther dehiscence. Plant Cell. 2005;17(11):2993–3006.
</cite> [<a href="https://doi.org/10.1105/tpc.105.036004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1276025/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16214898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mitsuda%20N,%20Seki%20M,%20Shinozaki%20K,%20Ohme-Takagi%20M.%20The%20NAC%20transcription%20factors%20NST1%20and%20NST2%20of%20Arabidopsis%20regulate%20secondary%20wall%20thickenings%20and%20are%20required%20for%20anther%20dehiscence.%20Plant%20Cell.%202005;17(11):2993%E2%80%933006." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Alvarado VY, Tag A, Thomas TL. A cis regulatory element in the TAPNAC promoter directs tapetal gene expression. Plant Mol Biol. 2011;75(1–2):129–39.
</cite> [<a href="https://doi.org/10.1007/s11103-010-9713-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21107887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alvarado%20VY,%20Tag%20A,%20Thomas%20TL.%20A%20cis%20regulatory%20element%20in%20the%20TAPNAC%20promoter%20directs%20tapetal%20gene%20expression.%20Plant%20Mol%20Biol.%202011;75(1%E2%80%932):129%E2%80%9339." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Li XG, Su YH, Zhao XY, Li W, Gao XQ, Zhang XS. Cytokinin overproduction-caused alteration of flower development is partially mediated by <em>CUC2</em> and <em>CUC3</em> in Arabidopsis. Gene. 2010;450(1–2):109–20.
</cite> [<a href="https://doi.org/10.1016/j.gene.2009.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19913077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20XG,%20Su%20YH,%20Zhao%20XY,%20Li%20W,%20Gao%20XQ,%20Zhang%20XS.%20Cytokinin%20overproduction-caused%20alteration%20of%20flower%20development%20is%20partially%20mediated%20by%20CUC2%20and%20CUC3%20in%20Arabidopsis.%20Gene.%202010;450(1%E2%80%932):109%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Nahar MA, Ishida T, Smyth DR, Tasaka M, Aida M. Interactions of cup-shaped cotyledon and spatula genes control carpel margin development in <em>Arabidopsis thaliana</em>. Plant Cell Physiol. 2012;53(6):1134–43.
</cite> [<a href="https://doi.org/10.1093/pcp/pcs057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3367164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22514090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nahar%20MA,%20Ishida%20T,%20Smyth%20DR,%20Tasaka%20M,%20Aida%20M.%20Interactions%20of%20cup-shaped%20cotyledon%20and%20spatula%20genes%20control%20carpel%20margin%20development%20in%20Arabidopsis%20thaliana.%20Plant%20Cell%20Physiol.%202012;53(6):1134%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Zheng G, Wei W, Li Y, Kan L, Wang F, Zhang X, Li F, Liu Z, Kang C. Conserved and novel roles of miR164-CUC2 regulatory module in specifying leaf and floral organ morphology in strawberry. New Phytol. 2019;224(1):480–92.
</cite> [<a href="https://doi.org/10.1111/nph.15982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31179543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng%20G,%20Wei%20W,%20Li%20Y,%20Kan%20L,%20Wang%20F,%20Zhang%20X,%20Li%20F,%20Liu%20Z,%20Kang%20C.%20Conserved%20and%20novel%20roles%20of%20miR164-CUC2%20regulatory%20module%20in%20specifying%20leaf%20and%20floral%20organ%20morphology%20in%20strawberry.%20New%20Phytol.%202019;224(1):480%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Ito T, Nagata N, Yoshiba Y, Ohme-Takagi M, Ma H, Shinozaki K. <em>Arabidopsis</em> male sterility1 encodes a PHD-type transcription factor and regulates pollen and tapetum development. Plant Cell. 2007;19(11):3549–62.
</cite> [<a href="https://doi.org/10.1105/tpc.107.054536" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2174881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18032630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ito%20T,%20Nagata%20N,%20Yoshiba%20Y,%20Ohme-Takagi%20M,%20Ma%20H,%20Shinozaki%20K.%20Arabidopsis%20male%20sterility1%20encodes%20a%20PHD-type%20transcription%20factor%20and%20regulates%20pollen%20and%20tapetum%20development.%20Plant%20Cell.%202007;19(11):3549%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Zhang Z, Zhu J, Gao J, Wang C, Li H, Li H, Zhang H, Zhang S, Wang D, Wang Q, Huang H, Xia H, Yang Z. Transcription factor AtMYB103 is required for anther development by regulating tapetum development, callose dissolution and exine formation in Arabidopsis. Plant J. 2007;52(3):528–38.
</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2007.03254.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17727613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Z,%20Zhu%20J,%20Gao%20J,%20Wang%20C,%20Li%20H,%20Li%20H,%20Zhang%20H,%20Zhang%20S,%20Wang%20D,%20Wang%20Q,%20Huang%20H,%20Xia%20H,%20Yang%20Z.%20Transcription%20factor%20AtMYB103%20is%20required%20for%20anther%20development%20by%20regulating%20tapetum%20development,%20callose%20dissolution%20and%20exine%20formation%20in%20Arabidopsis.%20Plant%20J.%202007;52(3):528%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Lyu T, Cao J. Cys(2)/His(2) zinc-finger proteins in transcriptional regulation of flower development. Int J Mol Sci. 2018;19(9): 2589.
</cite> [<a href="https://doi.org/10.3390/ijms19092589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6164605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30200325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lyu%20T,%20Cao%20J.%20Cys(2)/His(2)%20zinc-finger%20proteins%20in%20transcriptional%20regulation%20of%20flower%20development.%20Int%20J%20Mol%20Sci.%202018;19(9):%202589." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Liu H, Xiao S, Sui S, Huang R, Wang X, Wu H, Liu X. A tandem CCCH type zinc finger protein gene <em>CpC3H3</em> from <em>Chimonanthus praecox</em> promotes flowering and enhances drought tolerance in Arabidopsis. BMC Plant Biol. 2022;22(1):506.
</cite> [<a href="https://doi.org/10.1186/s12870-022-03877-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9617390/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36309643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20H,%20Xiao%20S,%20Sui%20S,%20Huang%20R,%20Wang%20X,%20Wu%20H,%20Liu%20X.%20A%20tandem%20CCCH%20type%20zinc%20finger%20protein%20gene%20CpC3H3%20from%20Chimonanthus%20praecox%20promotes%20flowering%20and%20enhances%20drought%20tolerance%20in%20Arabidopsis.%20BMC%20Plant%20Biol.%202022;22(1):506." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Han YY, Zhou HY, Xu LA, Liu XY, Fan SX, Cao JS. The zinc-finger transcription factor BcMF20 and its orthologs in Cruciferae which are required for pollen development. Biochem Biophys Res Commun. 2018;503(2):998–1003.
</cite> [<a href="https://doi.org/10.1016/j.bbrc.2018.06.108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29936180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han%20YY,%20Zhou%20HY,%20Xu%20LA,%20Liu%20XY,%20Fan%20SX,%20Cao%20JS.%20The%20zinc-finger%20transcription%20factor%20BcMF20%20and%20its%20orthologs%20in%20Cruciferae%20which%20are%20required%20for%20pollen%20development.%20Biochem%20Biophys%20Res%20Commun.%202018;503(2):998%E2%80%931003." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Nibau C, Di Stilio VS, Wu HM, Cheung AY. <em>Arabidopsis</em> and tobacco superman regulate hormone signalling and mediate cell proliferation and differentiation. J Exp Bot. 2011;62(3):949–61.
</cite> [<a href="https://doi.org/10.1093/jxb/erq325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3022392/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20980362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nibau%20C,%20Di%20Stilio%20VS,%20Wu%20HM,%20Cheung%20AY.%20Arabidopsis%20and%20tobacco%20superman%20regulate%20hormone%20signalling%20and%20mediate%20cell%20proliferation%20and%20differentiation.%20J%20Exp%20Bot.%202011;62(3):949%E2%80%9361." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Carbonero P, Iglesias-Fernandez R, Vicente-Carbajosa J. The AFL subfamily of B3 transcription factors: evolution and function in angiosperm seeds. J Exp Bot. 2017;68(4):871–80.
</cite> [<a href="https://doi.org/10.1093/jxb/erw458" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28007955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carbonero%20P,%20Iglesias-Fernandez%20R,%20Vicente-Carbajosa%20J.%20The%20AFL%20subfamily%20of%20B3%20transcription%20factors:%20evolution%20and%20function%20in%20angiosperm%20seeds.%20J%20Exp%20Bot.%202017;68(4):871%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Song S, Hao L, Zhao P, Xu Y, Zhong N, Zhang H, Liu N. Genome-wide identification, expression profiling and evolutionary analysis of auxin response factor gene family in potato (<em>Solanum tuberosum</em> Group Phureja). Sci Rep. 2019;9(1): 1755.
</cite> [<a href="https://doi.org/10.1038/s41598-018-37923-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6370904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30742001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song%20S,%20Hao%20L,%20Zhao%20P,%20Xu%20Y,%20Zhong%20N,%20Zhang%20H,%20Liu%20N.%20Genome-wide%20identification,%20expression%20profiling%20and%20evolutionary%20analysis%20of%20auxin%20response%20factor%20gene%20family%20in%20potato%20(Solanum%20tuberosum%20Group%20Phureja).%20Sci%20Rep.%202019;9(1):%201755." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Li Y, Han S, Qi Y. Advances in structure and function of auxin response factor in plants. J Integr Plant Biol. 2023;65(3):617–32.
</cite> [<a href="https://doi.org/10.1111/jipb.13392" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36263892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Y,%20Han%20S,%20Qi%20Y.%20Advances%20in%20structure%20and%20function%20of%20auxin%20response%20factor%20in%20plants.%20J%20Integr%20Plant%20Biol.%202023;65(3):617%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Zhuang J, Sun CC, Zhou XR, Xiong AS, Zhang J. Isolation and characterization of an AP2/ERF-RAV transcription factor BnaRAV-1-HY15 in <em>Brassica </em><em>napus</em> L. HuYou15. Mol Biol Rep. 2011;38(6):3921–8.
</cite> [<a href="https://doi.org/10.1007/s11033-010-0508-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21116861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhuang%20J,%20Sun%20CC,%20Zhou%20XR,%20Xiong%20AS,%20Zhang%20J.%20Isolation%20and%20characterization%20of%20an%20AP2/ERF-RAV%20transcription%20factor%20BnaRAV-1-HY15%20in%20Brassica%20napus%20L.%20HuYou15.%20Mol%20Biol%20Rep.%202011;38(6):3921%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Caselli F, Beretta VM, Mantegazza O, Petrella R, Leo G, Guazzotti A, Herrera-Ubaldo H, de Folter S, Mendes MA, Kater MM, et al. REM34 and REM35 control female and male gametophyte development in <em>Arabidopsis thaliana</em>. Front Plant Sci. 2019;10:1351.
</cite> [<a href="https://doi.org/10.3389/fpls.2019.01351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6821719/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31708954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Caselli%20F,%20Beretta%20VM,%20Mantegazza%20O,%20Petrella%20R,%20Leo%20G,%20Guazzotti%20A,%20Herrera-Ubaldo%20H,%20de%20Folter%20S,%20Mendes%20MA,%20Kater%20MM,%20et%20al.%20REM34%20and%20REM35%20control%20female%20and%20male%20gametophyte%20development%20in%20Arabidopsis%20thaliana.%20Front%20Plant%20Sci.%202019;10:1351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Mantegazza O, Gregis V, Mendes MA, Morandini P, Alves-Ferreira M, Patreze CM, Nardeli SM, Kater MM, Colombo L. Analysis of the arabidopsis REM gene family predicts functions during flower development. Ann Bot. 2014;114(7):1507–15.
</cite> [<a href="https://doi.org/10.1093/aob/mcu124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4204784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25002525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mantegazza%20O,%20Gregis%20V,%20Mendes%20MA,%20Morandini%20P,%20Alves-Ferreira%20M,%20Patreze%20CM,%20Nardeli%20SM,%20Kater%20MM,%20Colombo%20L.%20Analysis%20of%20the%20arabidopsis%20REM%20gene%20family%20predicts%20functions%20during%20flower%20development.%20Ann%20Bot.%202014;114(7):1507%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Zhao ZX, Yin XX, Li S, Peng YT, Yan XL, Chen C, Hassan B, Zhou SX, Pu M, Zhao JH, et al. miR167d-ARFs module regulates flower opening and stigma size in rice. Rice. 2022;15(1):40.
</cite> [<a href="https://doi.org/10.1186/s12284-022-00587-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9314575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35876915/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20ZX,%20Yin%20XX,%20Li%20S,%20Peng%20YT,%20Yan%20XL,%20Chen%20C,%20Hassan%20B,%20Zhou%20SX,%20Pu%20M,%20Zhao%20JH,%20et%20al.%20miR167d-ARFs%20module%20regulates%20flower%20opening%20and%20stigma%20size%20in%20rice.%20Rice.%202022;15(1):40." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Wei H, Cheng Y, Sun Y, Zhang X, He H, Liu J. Genome-wide identification of the ARF gene family and ARF3 target genes regulating ovary initiation in Hazel via ChIP sequencing. Front Plant Sci. 2021;12: 715820.
</cite> [<a href="https://doi.org/10.3389/fpls.2021.715820" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8382943/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34447403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei%20H,%20Cheng%20Y,%20Sun%20Y,%20Zhang%20X,%20He%20H,%20Liu%20J.%20Genome-wide%20identification%20of%20the%20ARF%20gene%20family%20and%20ARF3%20target%20genes%20regulating%20ovary%20initiation%20in%20Hazel%20via%20ChIP%20sequencing.%20Front%20Plant%20Sci.%202021;12:%20715820." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Sobral R, Costa M. Role of floral organ identity genes in the development of unisexual flowers of <em>Quercus suber</em> L. Sci Rep. 2017;7(1): 10368.
</cite> [<a href="https://doi.org/10.1038/s41598-017-10732-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5583232/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28871195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sobral%20R,%20Costa%20M.%20Role%20of%20floral%20organ%20identity%20genes%20in%20the%20development%20of%20unisexual%20flowers%20of%20Quercus%20suber%20L.%20Sci%20Rep.%202017;7(1):%2010368." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Ruelens P, Zhang Z, van Mourik H, Maere S, Kaufmann K, Geuten K. The origin of floral organ identity quartets. Plant Cell. 2017;29(2):229–42.
</cite> [<a href="https://doi.org/10.1105/tpc.16.00366" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5354184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28100708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruelens%20P,%20Zhang%20Z,%20van%20Mourik%20H,%20Maere%20S,%20Kaufmann%20K,%20Geuten%20K.%20The%20origin%20of%20floral%20organ%20identity%20quartets.%20Plant%20Cell.%202017;29(2):229%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Lenser T, Theissen G, Dittrich P. Developmental robustness by obligate interaction of class B floral homeotic genes and proteins. PLoS Comput Biol. 2009;5(1): e1000264.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1000264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2612583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19148269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lenser%20T,%20Theissen%20G,%20Dittrich%20P.%20Developmental%20robustness%20by%20obligate%20interaction%20of%20class%20B%20floral%20homeotic%20genes%20and%20proteins.%20PLoS%20Comput%20Biol.%202009;5(1):%20e1000264." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Wang YQ, Melzer R, Theissen G. Molecular interactions of orthologues of floral homeotic proteins from the gymnosperm Gnetum gnemon provide a clue to the evolutionary origin of ‘floral quartets.’ Plant J. 2010;64(2):177–90.
</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2010.04325.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21070403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20YQ,%20Melzer%20R,%20Theissen%20G.%20Molecular%20interactions%20of%20orthologues%20of%20floral%20homeotic%20proteins%20from%20the%20gymnosperm%20Gnetum%20gnemon%20provide%20a%20clue%20to%20the%20evolutionary%20origin%20of%20%E2%80%98floral%20quartets.%E2%80%99%20Plant%20J.%202010;64(2):177%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (7MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (12.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (12.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4.</a><sup> (11.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5.</a><sup> (13.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6.</a><sup> (6.7MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7.</a><sup> (3.8MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369072/bin/12870_2025_7096_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8.</a><sup> (13.5KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Sequence data that support the findings of this study have been deposited in the Sequence Read Archive (SRA) of the National Center for Biotechnology Information (NCBI) with the primary accession code PRJNA1025044.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Plant Biology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12870-025-07096-3"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12870_2025_Article_7096.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12369072/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12369072/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369072%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369072/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12369072/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12369072/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841596/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12369072/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841596/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12369072/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12369072/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="sDvVxwVEYwdsH4enr4oGC3jwSddVJ94PhiyOnfFAO5OWRVc8S54XBrdQFJHey2Zo">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
