
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            RnaXtract, a tool for extracting gene expression, variants, and cell-type composition from bulk RNA sequencing - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5313C08AEF41130513C00022A5E334.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375748/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="RnaXtract, a tool for extracting gene expression, variants, and cell-type composition from bulk RNA sequencing">
<meta name="citation_author" content="Sophiane G Bouirdene">
<meta name="citation_author_institution" content="Doctoral Program in Molecular Medicine, Faculté de Médecine, Université Laval, Québec, Québec G1V 0A6 Canada">
<meta name="citation_author_institution" content="Endocrinology and Nephrology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada">
<meta name="citation_author_institution" content="Computational Biology Laboratory, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada">
<meta name="citation_author" content="Simon Gotty">
<meta name="citation_author_institution" content="Endocrinology and Nephrology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada">
<meta name="citation_author_institution" content="Computational Biology Laboratory, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada">
<meta name="citation_author" content="Mickaël Leclercq">
<meta name="citation_author_institution" content="Endocrinology and Nephrology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada">
<meta name="citation_author_institution" content="Computational Biology Laboratory, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada">
<meta name="citation_author" content="Charles Joly-Beauparlant">
<meta name="citation_author_institution" content="Endocrinology and Nephrology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada">
<meta name="citation_author_institution" content="Computational Biology Laboratory, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada">
<meta name="citation_author" content="Emeric Texeraud">
<meta name="citation_author_institution" content="Endocrinology and Nephrology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada">
<meta name="citation_author_institution" content="Computational Biology Laboratory, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada">
<meta name="citation_author" content="Steve Bilodeau">
<meta name="citation_author_institution" content="Department of Molecular Biology, Medical Biochemistry and Pathology, Faculté de Médecine, Université Laval, Québec, Québec G1V 0A6 Canada">
<meta name="citation_author_institution" content="Oncology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1J 1Z4 Canada">
<meta name="citation_author_institution" content="Centre de Recherche sur le Cancer de l’Université Laval, Québec, Québec G1R 3S3 Canada">
<meta name="citation_author_institution" content="Centre de Recherche en Données Massives de l’Université Laval, Québec, Québec G1V 0A6 Canada">
<meta name="citation_author" content="Arnaud Droit">
<meta name="citation_author_institution" content="Endocrinology and Nephrology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada">
<meta name="citation_author_institution" content="Computational Biology Laboratory, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada">
<meta name="citation_author_institution" content="Centre de Recherche sur le Cancer de l’Université Laval, Québec, Québec G1R 3S3 Canada">
<meta name="citation_author_institution" content="Centre de Recherche en Données Massives de l’Université Laval, Québec, Québec G1V 0A6 Canada">
<meta name="citation_author_institution" content="Department of Molecular Medicine, Faculté de Médecine, Université Laval, Québec, Québec G1V 0A6 Canada">
<meta name="citation_publication_date" content="2025 Aug 24">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="31100">
<meta name="citation_doi" content="10.1038/s41598-025-16875-9">
<meta name="citation_pmid" content="40850998">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375748/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375748/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375748/pdf/41598_2025_Article_16875.pdf">
<meta name="description" content="RNA sequencing (RNA-seq) is a widely used method in transcriptomics research, offering insights into gene expression, variant discovery, and, when deconvoluted, the cellular composition of complex tissues. However, existing RNA-seq pipelines ...">
<meta name="og:title" content="RnaXtract, a tool for extracting gene expression, variants, and cell-type composition from bulk RNA sequencing">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="RNA sequencing (RNA-seq) is a widely used method in transcriptomics research, offering insights into gene expression, variant discovery, and, when deconvoluted, the cellular composition of complex tissues. However, existing RNA-seq pipelines ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375748/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375748">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-16875-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_16875.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375748%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375748/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375748/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375748/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 24;15:31100. doi: <a href="https://doi.org/10.1038/s41598-025-16875-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-16875-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>RnaXtract, a tool for extracting gene expression, variants, and cell-type composition from bulk RNA sequencing</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bouirdene%20SG%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Sophiane G Bouirdene</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sophiane G Bouirdene</span></h3>
<div class="p">
<sup>1</sup>Doctoral Program in Molecular Medicine, Faculté de Médecine, Université Laval, Québec, Québec G1V 0A6 Canada </div>
<div class="p">
<sup>2</sup>Endocrinology and Nephrology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div class="p">
<sup>3</sup>Computational Biology Laboratory, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bouirdene%20SG%22%5BAuthor%5D" class="usa-link"><span class="name western">Sophiane G Bouirdene</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gotty%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Simon Gotty</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Simon Gotty</span></h3>
<div class="p">
<sup>2</sup>Endocrinology and Nephrology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div class="p">
<sup>3</sup>Computational Biology Laboratory, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gotty%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Simon Gotty</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Leclercq%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Mickaël Leclercq</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Mickaël Leclercq</span></h3>
<div class="p">
<sup>2</sup>Endocrinology and Nephrology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div class="p">
<sup>3</sup>Computational Biology Laboratory, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Leclercq%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mickaël Leclercq</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joly-Beauparlant%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Charles Joly-Beauparlant</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Charles Joly-Beauparlant</span></h3>
<div class="p">
<sup>2</sup>Endocrinology and Nephrology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div class="p">
<sup>3</sup>Computational Biology Laboratory, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joly-Beauparlant%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Charles Joly-Beauparlant</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Texeraud%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Emeric Texeraud</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Emeric Texeraud</span></h3>
<div class="p">
<sup>2</sup>Endocrinology and Nephrology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div class="p">
<sup>3</sup>Computational Biology Laboratory, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Texeraud%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Emeric Texeraud</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bilodeau%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Steve Bilodeau</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Steve Bilodeau</span></h3>
<div class="p">
<sup>4</sup>Department of Molecular Biology, Medical Biochemistry and Pathology, Faculté de Médecine, Université Laval, Québec, Québec G1V 0A6 Canada </div>
<div class="p">
<sup>5</sup>Oncology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1J 1Z4 Canada </div>
<div class="p">
<sup>6</sup>Centre de Recherche sur le Cancer de l’Université Laval, Québec, Québec G1R 3S3 Canada </div>
<div class="p">
<sup>7</sup>Centre de Recherche en Données Massives de l’Université Laval, Québec, Québec G1V 0A6 Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bilodeau%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Steve Bilodeau</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>7,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Droit%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Arnaud Droit</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Arnaud Droit</span></h3>
<div class="p">
<sup>2</sup>Endocrinology and Nephrology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div class="p">
<sup>3</sup>Computational Biology Laboratory, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div class="p">
<sup>6</sup>Centre de Recherche sur le Cancer de l’Université Laval, Québec, Québec G1R 3S3 Canada </div>
<div class="p">
<sup>7</sup>Centre de Recherche en Données Massives de l’Université Laval, Québec, Québec G1V 0A6 Canada </div>
<div class="p">
<sup>8</sup>Department of Molecular Medicine, Faculté de Médecine, Université Laval, Québec, Québec G1V 0A6 Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Droit%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Arnaud Droit</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>6,</sup><sup>7,</sup><sup>8,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Doctoral Program in Molecular Medicine, Faculté de Médecine, Université Laval, Québec, Québec G1V 0A6 Canada </div>
<div id="Aff2">
<sup>2</sup>Endocrinology and Nephrology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div id="Aff3">
<sup>3</sup>Computational Biology Laboratory, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1V 4G2 Canada </div>
<div id="Aff4">
<sup>4</sup>Department of Molecular Biology, Medical Biochemistry and Pathology, Faculté de Médecine, Université Laval, Québec, Québec G1V 0A6 Canada </div>
<div id="Aff5">
<sup>5</sup>Oncology Axis, Centre de recherche du CHU de Québec—Université Laval, Québec, Québec G1J 1Z4 Canada </div>
<div id="Aff6">
<sup>6</sup>Centre de Recherche sur le Cancer de l’Université Laval, Québec, Québec G1R 3S3 Canada </div>
<div id="Aff7">
<sup>7</sup>Centre de Recherche en Données Massives de l’Université Laval, Québec, Québec G1V 0A6 Canada </div>
<div id="Aff8">
<sup>8</sup>Department of Molecular Medicine, Faculté de Médecine, Université Laval, Québec, Québec G1V 0A6 Canada </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 16; Accepted 2025 Aug 20; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375748  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40850998/" class="usa-link">40850998</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">RNA sequencing (RNA-seq) is a widely used method in transcriptomics research, offering insights into gene expression, variant discovery, and, when deconvoluted, the cellular composition of complex tissues. However, existing RNA-seq pipelines frequently emphasize gene expression analysis and often lack cell deconvolution and variant calling. To address these limitations, we present RnaXtract, a comprehensive and user-friendly pipeline designed to maximize extraction of valuable information from bulk RNA-seq data. RnaXtract automates an entire workflow, encompassing quality control, gene expression quantification, variant calling, and the cell-type deconvolution. Built on the Snakemake framework, RnaXtract ensures robust reproducibility, efficient resource management, and flexibility to adapt to diverse research needs. The pipeline integrates state-of-the-art tools, from quality control to the new updates on variant calling and cell-type deconvolution tools such as EcoTyper and CIBERSORTx, enabling researchers to extract biological insights with precision. By providing an end-to-end solution for bulk RNA-seq, RnaXtract addresses critical gaps in existing workflows, empowering researchers to explore gene expression, genetic variation, and cellular heterogeneity within a single cohesive framework.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Bulk RNA-seq, Transcriptomics, Snakemake pipeline, Gene expression, Variant calling, Cell deconvolution, Computational biology, RNA sequencing analysis</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Biomarkers, Molecular medicine, Computational biology and bioinformatics, Data processing, Software</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Despite the emergence of new technologies, such as single-cell and spatial transcriptomics, bulk RNA sequencing (RNA-seq) remains the most widely employed due to its cost-effectiveness in providing a comprehensive tissue overview<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Consequently, short-read sequencing technologies, which typically produce tens of millions of reads per sample, are the predominant methods for gene expression profiling<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. The widespread adoption of bulk RNA sequencing has led to the development of standardized workflows for data analysis. These workflows typically begin with quality control, assessing the quality and accuracy of bases in each read, followed by a trimming step to remove low-quality reads<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. The subsequent step involves mapping the reads to a reference genome or transcriptome, which is used for gene expression quantification by counting the reads mapped to each gene<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. This alignment also facilitates variant calling, allowing for the identification of individual-specific mutations<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. The extensive use and standardization of these methods have resulted in numerous analytical pipelines<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>.</p>
<p id="Par3">The advent of single-cell sequencing has provided new insights into cellular composition and characterization, opening the opportunity for analyzing tissue composition and its differentiation under various conditions<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Single-cell transcriptomics is usable to deconvolute bulk RNA-seq to estimate cellular composition of the tissue and their specific states<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. However, these methods are relatively new, and not included in existing bulk RNA-seq analysis pipelines. Additionally, many of these pipelines are optimized for gene expression analysis and do not align with current gold-standard methods for variant calling (Table <a href="#Tab1" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="Tab1"><h3 class="obj_head">Table 1.</h3>
<div class="caption p"><p>Comparative analysis of RnaXtract and other RNA-Seq processing pipelines.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Tool</th>
<th align="left" colspan="1" rowspan="1">Gene expression</th>
<th align="left" colspan="1" rowspan="1">Variant calling (joint genotyping)</th>
<th align="left" colspan="1" rowspan="1">Cell deconvolution</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">MIGNON</td>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">Yes (no)</td>
<td align="left" colspan="1" rowspan="1">No</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Latch</td>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">No</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RNAflow</td>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">No</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SPEAQeasy</td>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">Yes (no)</td>
<td align="left" colspan="1" rowspan="1">No</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">sequana_rnaseq</td>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">No</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GenPipes RNA</td>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">No</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RnaXtract</td>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">Yes (yes)</td>
<td align="left" colspan="1" rowspan="1">Yes</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p5"><p>Table placing RnaXtract in the various RNA-seq analysis tools in performing three key tasks: gene expression, variant calling, and cell deconvolution.</p></div></div></section><p id="Par4">With the quantity of data generated from bulk RNA sequencing, machine learning (ML) methods are used for biomarker discovery, enabling the identification of molecular signatures that correlate with specific conditions or therapeutic responses<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Consequently, its use is becoming prevalent in biological analyses to identify significant genes, pathways and disease mechanisms<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. However, many current pipelines are not optimized for ML integration. This often necessitates data transformation by the user, which can be burdensome and may deter biologists from using these types of tools.</p>
<p id="Par5">In this study, we present RnaXtract, a complete workflow designed to maximize extraction of the information contained in RNA-Seq data. RnaXtract not only generates the conventional normalized gene expression matrix but also performs state-of-the-art variant calling analysis and cellular deconvolution using CIBERSORTx and its extension, EcoTyper<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. This cellular composition analysis provides an additional layer of feature extraction beyond genomics and gene expression.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Workflow implementation</h3>
<p id="Par6">The complete pipeline was developed using Snakemake, a Python-based workflow definition language, due to its efficiency, scalability, and ease of use<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Snakemake facilitates seamless workflow management and reproducibility in bioinformatics analyses. To ensure long-term reproducibility, all tools and environments were containerized using Singularity images<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. A script was created to simplify the generation of a config.yml file to manage user interactions. Running RnaXtract requires four dependencies: R (v3.6.0)<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, Snakemake<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, Python3<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, and a container runtime engine: Singularity. The list of Docker containers utilized by RnaXtract is provided in Supplementary Table.S1.</p>
<p id="Par7">RnaXtract is structured into four modules: preprocessing with quality control, gene expression analysis, cell deconvolution and variant calling with an optional recalibration step for variant calling (Fig. <a href="#Fig1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375748_41598_2025_16875_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/145a/12375748/13d3a52113c1/41598_2025_16875_Fig1_HTML.jpg" loading="lazy" id="MO1" height="1168" width="749" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>RnaXtract workflow. Overview of the RNA-seq analysis pipeline, including quality control, pre-processing, variant calling, gene expression quantification, and cell deconvolution. The workflow begins with quality control using FastQC and MultiQC, generating a quality report. Pre-processing includes trimming and preparing FASTQ files. Variant calling involves alignment (STAR), deduplication (Picard), read splitting, and read group addition (GATK), followed by base quality score recalibration (BQSR) and variant identification using HaplotypeCaller and GenotypeGVCFs. Gene expression analysis is performed using Kallisto for quantification and subsequent formatting with the r-rnaseq package, generating gene count and TPM tables. Cell deconvolution is conducted with EcoTyper for ecotype analysis and CIBERSORTx for cell type abundance estimation. Final outputs include tables summarizing gene expression, variant calls, and inferred cell type compositions.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Quality control and alignment</h3>
<p id="Par8">Initially, raw sequences are inputted into the workflow, and fastp (v0.20.0)<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup> is employed to perform quality trimming and filtering of reads, utilizing default parameters for window size, mean quality, and length. Subsequently, FastQC (v0.11.5) generates a quality report for each pre-processed read file, and MultiQC (v0.11.5) compiles a comprehensive report across all samples. Following quality control, alignment is conducted using STAR (v2.7.2b)<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup> against a reference genome (or transcriptome), which is essential for variant calling. For the gene expression quantification Kallisto (v0.48.0)<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> was chosen as it is the quickest tool for this step while having equivalent quality to its concurrents<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Variant calling and transformation</h3>
<p id="Par9">Genomic variants can be inferred from RNA data through variant calling. Due to the number of intermediate steps carried out during this process, it was encapsulated in an independent sub-workflow run at sample level. The input data consists of the alignments generated with STAR and the output is a list of variants in the variant call format (VCF) and a table format file. The whole process is performed using the Genome Analysis ToolKit (v4.1.3.0)<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, and it was designed following the GATK best practices for the variant calling from RNA-Seq data which is the gold standard in the field. The variant calling is done on all the samples together instead of them separately using the “HaplotypeCaller” and “GenotypeGVCFs” functions following GATK and genotyping recommendations<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. This reduces the direct dependency of the depth and enhances the quality of the mutation identification.</p>
<p id="Par10">Variants are separated between single nucleotide polymorphism (SNP) and insertion/deletion (INDEL) to be filtered using a hard filter. The filters differ between the two types of variants and were chosen following GATK guidelines. The filtering differences exist because SNPs and INDELs have distinct characteristics, error profiles, and detection challenges. Filtering criteria were tailored to maximize sensitivity and specificity for each type of variant while minimizing false positives from sequencing or alignment artifacts.</p>
<p id="Par11">A primary challenge in machine learning applications is overfitting, where models capture dataset-specific features rather than generalizable biological signals. To reduce the risk of overfitting when using RnaXtract for downstream machine learning approaches, we implemented a user-defined script to filter variants based on their frequency within the dataset. The output VCFs are then transformed into a table format. This table has as lines the sample identification and as columns the variant which name is composed of: chromosome:position_reference &gt; mutation. The value of the column is represented by the two numbers associated with mutation present at this position separated by an underscore with 0 representing the absence of mutation. Mutations with absence of information for at least one sample at the position of the mutation were removed. Also, a possibility to exclude variants based on a threshold of presence in the cohort was added.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Normalization</h3>
<p id="Par12">The use of Transcripts per Million (TPM) was chosen as a normalization method for gene expression data. TPM accounts for both sequencing depth and gene length, ensuring comparability across samples by normalizing gene expression values relative to the total transcript count. This normalization method is essential for building machine learning models, as it enhances the model’s ability to generalize and reduces its susceptibility to batch effects. In addition, the cell deconvolution step in the pipeline required TPM, as it is one of the two input formats supported by EcoTyper and CIBERSORTx.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Cell deconvolution</h3>
<p id="Par13">The cell deconvolution is achieved through the utilization of EcoTyper, a tool specifically designed to decode cellular heterogeneity within bulk RNA-seq data. EcoTyper is a pre-trained deep learning tool capable of estimating the cell composition and states which compose the ecotype of the bulk sample. Moreover, in the case that the user possesses their own single-cell RNA-seq datasets as reference, another step using CIBERSORTx was added. CIBERSORTx enables the creation of custom signature matrices from single-cell data, thereby improving the accuracy of cell type quantification and functional analysis. CIBERSORTx has been shown to be one of the best tools for deconvolution<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> and its extension offers a pre-trained model for cancer research which enhances its granularity by going into cell ecotypes. The combined use of EcoTyper and CIBERSORTx provides insight on the cellular landscapes in bulk RNA-seq datasets.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Organisms</h3>
<p id="Par14">RnaXtract is mostly oriented for the analysis of human transcriptomic data. However, it can be used for other species that possess both a reference genome and transcriptome, with only minor adjustments, primarily the selection of appropriate reference annotations and, when relevant, species-specific single-cell reference datasets. For cell deconvolution, CIBERSORTx has demonstrated utility across multiple organisms<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. There are certain limitations in the use of EcoTyper, as its models were trained on human single-cell data. However, an option was included for users to apply EcoTyper if they supply a conversion table mapping their organism’s cell types to corresponding human counterparts.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Results</h2>
<p id="Par15">To evaluate RnaXtract’s ability to extract relevant data, we analyzed human breast cancer samples collected pre-chemotherapy to identify biomarkers predictive of treatment failure using machine learning. We selected 24 tumor samples from the European Nucleotide Archive project PRJNA1004593<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> (Supplementary Table S2). These samples included 12 patients who succeeded and 12 who failed the post-neoadjuvant chemotherapy. A random selection of 2,000 cells from a breast cancer single-cell dataset, available at CELLxGENE<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, were used to create the reference for CIBERSORTx. For the variant analysis, a threshold of 0.1 was applied, retaining only those variants present in at least 10% of the cohort.</p>
<p id="Par16">A gene expression table in TPM format, SNP and INDEL tables, and custom-generated cell composition and ecotype data were generated. These outputs were used to develop individual models with BioDiscML<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, a machine learning tool capable of identifying biomarkers. The most relevant features were combined to create a final model combining accuracy and optimal feature usage.</p>
<p id="Par17">Multiple models predicting chemotherapy success were derived with varying model performance. A broad range of Matthews Correlation Coefficient (MCC) scores were observed with the lowest for the EcoTyper deconvolution model (MCC = 0.029, 4 features) and the highest for the Gene Expression model (MCC = 0.762, 75 features). Integrating all four data types allowed us to reduce the number of features while building a high quality model. This integrated model achieved an MCC of 0.737—comparable to the Gene Expression model—but significantly lowering the standard deviation (STD) from 0.287 to 0.042 and requiring only 2 features (Fig. <a href="#Fig2" class="usa-link">2</a>): a mutation within the 3’UTR of Syntaxin-16 (<em>STX16</em>) and the RNA level of ELMO Domain Containing 1 (<em>ELMOD1</em>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h3 class="obj_head">Fig. 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375748_41598_2025_16875_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/145a/12375748/5b69e69b50d4/41598_2025_16875_Fig2_HTML.jpg" loading="lazy" id="MO2" height="327" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Quality of machine learning model. (<strong>a</strong>) Comparison of the number of selected features and (<strong>b</strong>) model performance measured by the Matthews Correlation Coefficient (MCC) across different feature sets. The “Merged” model, which integrates all data types, demonstrates a reduced number of features compared to the “Expression” model while maintaining a high MCC. Additionally, the STD (error bars) in MCC is lower in the “Merged” model compared to individual models, indicating increased stability and robustness. This suggests that integrating multiple data sources improves classification performance while reducing feature redundancy and variability.</p></figcaption></figure><p id="Par18"><em>STX16</em> is primarily involved in intracellular transport, particularly within the endosomal and Golgi apparatus pathways. It has been reported to be overexpressed in cervical cancer<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>, and data from The Cancer Genome Atlas Breast Invasive Carcinoma suggest that patients with <em>STX16</em> amplification exhibit poorer disease-free survival (Supplementary Fig. 1). Mutations affecting the 3’UTR could impact mRNA stability, potentially enhancing cancer cell survival. In our cohort, although no consistent pattern was observed between the number of adenine repeats in the STX16-associated variant and its expression level, patients who experienced treatment failure consistently displayed higher STX16 expression compared to those who responded well (Fig. <a href="#Fig3" class="usa-link">3</a>a). This observation aligns with our initial hypothesis that elevated STX16 expression may be linked to poor outcomes.</p>
<figure class="fig xbox font-sm" id="Fig3"><h3 class="obj_head">Fig. 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375748_41598_2025_16875_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/145a/12375748/22cd04422ce4/41598_2025_16875_Fig3_HTML.jpg" loading="lazy" id="MO3" height="327" width="668" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Interpretation of features from the machine learning model. (<strong>a</strong>) Mean STX16 expression (TPM) grouped by haplotype, with bars colored according to treatment outcome (green: success; red: failure). (<strong>b</strong>) ELMOD1 expression levels (TPM) ordered by individual patients and stratified by treatment outcome. Each bar represents a single patient.</p></figcaption></figure><p id="Par19">Similarly, <em>ELMOD1</em> is a GTPase-activating protein for the Arf family of small G proteins<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, playing a key role in cellular signaling regulation. Recent studies indicate that <em>ELMOD1</em> and <em>ELMOD2</em> are upregulated in breast cancer cell lines and are essential for cancer cell migration<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Low <em>ELMOD1</em> RNA expression may suggest reduced cancer cell migration and an improved likelihood of treatment success as seen in the cohort (Fig. <a href="#Fig3" class="usa-link">3</a>b).</p>
<p id="Par20">This example highlights RnaXtract’s effectiveness in multi-layered biomarker analysis for cancer research, offering a comprehensive and automated tool for RNA-seq data extraction with applications in machine learning. However, it is important to note that the chemotherapy response prediction model presented here is based on a limited sample size (n = 24). Further validation on larger, independent datasets will be necessary to confirm the predictive value. It has to be noted that it’s the same patient which has results indicating a failure but had the treatment succeeding, showing the heterogeneity in data and the need to be careful with these models.</p>
<section id="Sec10"><h3 class="pmc_sec_title">Workflow performance evaluation</h3>
<p id="Par21">To assess the performance and resource utilization of the RnaXtract pipeline, we conducted analyses on multiple human datasets with diverse characteristics, encompassing a total of 30 samples derived from two distinct datasets (24 paired-end and 6 single-end)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. The pipeline was executed on an AMD EPYC 7601 32-Core Processor<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, utilizing the default core configuration specified by the pipeline. Initial evaluations revealed that the variant calling step was the most computationally intensive, requiring an average of 29 h of CPU time (Fig. <a href="#Fig4" class="usa-link">4</a>a). Subsequent steps, including dataset integration, genotyping, and variant filtration, each consumed approximately 10 h of CPU time and demonstrated potential for parallelization. In contrast, EcoTyper and CIBERSORTx exhibited significantly lower computational demands, completing within 20 min of CPU time, with EcoTyper further benefiting from multi-threaded execution. The RNA-seq function, which processes Kallisto output into tabular format, was the least resource-intensive, completing in under 2 min.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375748_41598_2025_16875_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/145a/12375748/c11bd686291c/41598_2025_16875_Fig4_HTML.jpg" loading="lazy" id="MO4" height="332" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>RnaXtract performance results. (<strong>a</strong>) The barplot illustrates the comparison of data processing steps for paired-end and single-end sequencing data. The y-axis represents the CPU-times in minutes while the x-axis denotes the different steps involved in multi-samples RnaXtract’s steps. Two distinct data sets were used: the breast cancer SRA cohort, paired-end (dark grey) and a cohort using single-end sequencing (white). The bars for each step show the average performance with the standard deviation. (<strong>b</strong>) Maximum memory usage (in MB), measured as peak Resident Set Size (RSS), for pipeline executions with 40 and 100 CPU cores across datasets of 10, 24, and 50 samples.</p></figcaption></figure><p id="Par22">To characterize memory usage during the most demanding step of the pipeline, we benchmarked RnaXtract on the PRJNA1004593 dataset of increasing the number of samples (10, 25, and 50), using either 40 or 100 CPU cores with smem tool. A peak memory consumption was measured using Resident Set Size (RSS), which reflects the actual physical memory held in RAM during execution (Fig. <a href="#Fig4" class="usa-link">4</a>b). Memory usage scaled with the number of samples and was consistently higher with 100 cores, due to greater parallelism and concurrent memory allocation. At the largest scale (50 samples), maximum RSS exceeded 125 GB with 100 cores, compared to under 100 GB with 40 cores. These results show a linear correlation between the number of CPUs and samples for the memory usage. The step which consumed the most memory was HaplotypeCaller.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Functional overview of existing bulk RNA-seq workflows</h3>
<p id="Par23">In order to evaluate the place of RnaXtract within the field of bulk RNA-seq data processing, we compiled a list of pipelines published after 2020 (Table <a href="#Tab1" class="usa-link">1</a>). Six pipelines were selected as they were specialized for bulk RNA-seq and were taking as input raw FASTQ files: MIGNON<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, Latch<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, RNAflow<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, SPEAQeasy<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, sequana_rnaseq<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, and GenPipes RNA<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. A key observation is that most of these pipelines primarily focus on differential gene expression analysis, with only two (MIGNON and SPEAQeasy) supporting variant calling. This highlights that RnaXtract will operate in a distinct niche compared to existing tools. The primary objective of RnaXtract is to maximize the extraction of raw information from bulk RNA sequencing, offering outputs such as gene expression, variant calling, and novel cell deconvolution in a format that is easily accessible for machine learning applications. Notably, the other tools performing variant calling present variants in VCF format, which is less suited for downstream machine learning applications.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Interest of cell deconvolution</h3>
<p id="Par24">Cell deconvolution has proven its utility in machine learning frameworks by enhancing predictive accuracy and uncovering biologically meaningful features from bulk RNA-seq data. The integration of immune cell fractions, particularly tumor-associated neutrophils, significantly improved chemotherapy response prediction in gastric cancer, with deconvolution-derived features emerging among the most important variables in the model<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Similarly, large-scale deconvolution of AML cohorts using single-cell references has reconstructed differentiation trajectories and predicted drug resistance with high fidelity <sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Together, these studies demonstrate how cell deconvolution transforms bulk transcriptomic data into a powerful resource bridging tissue-level insights with single-cell resolution.</p></section></section><section id="Sec13"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par25">RnaXtract provides an automated RNA-seq data analysis workflow, transforming raw sequencing reads into structured, machine learning-compatible datasets. It offers a robust framework for extracting genomic, transcriptomic, and cellular composition data efficiently. Leveraging state-of-the-art methods for read processing, RnaXtract integrates the CIBERSORTx/EcoTyper deep learning model to deliver detailed cellular and ecotype profiling of samples. This makes it highly applicable for advancing personalized medicine, particularly in predicting cancer trajectories, due to its machine learning-compatible output format. Furthermore, RnaXtract can be deployed seamlessly across diverse computational environments, ensuring optimized resource utilization. Its modular architecture facilitates straightforward upgrades and maintenance, enhancing adaptability for evolving research needs.</p></section><section id="Sec14"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12375748/bin/41598_2025_16875_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 1.</a><sup> (64.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12375748/bin/41598_2025_16875_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 2.</a><sup> (22.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12375748/bin/41598_2025_16875_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 3.</a><sup> (5.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12375748/bin/41598_2025_16875_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 4.</a><sup> (6.1KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank all lab members for many interesting discussions. This work was supported by the Fonds de recherche du Québec (FRQ) through the research centre grant for the CHU de Québec-Université Laval Research Center (reference: 30641).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>SGB software, conceptualization, formal analysis, methodology, writing—original draft. SG software, conceptualization. ML supervision, conceptualization, writing—review &amp; editing CJB software, conceptualization ET conceptualization, methodology SB supervision, fund acquisition, writing—review &amp; editing, supporting conceptualization AD supervision, resources and fund acquisition, writing—review &amp; editing.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was funded by Natural Sciences and Engineering Research Council of Canada (#2019-06490, <a href="https://www.ncbi.nlm.nih.gov/nuccore/CG118883" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CG118883</a>) and Canadian Institutes of Health Research (#451568).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The RnaXtract pipeline is publicly available at <a href="https://gitlab-pub.compbio.ulaval.ca/sophiane/rnaxtract" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gitlab-pub.compbio.ulaval.ca/sophiane/rnaxtract</a> under the GNU-GPL 3 licence and the documentation is accessible within the repository. The samples used to test RnaXtract accession number of this manuscript are available in the supplementary data file.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par26">The authors declare no competing interests.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par27">The datasets used in this study are available under the accession number PRJNA1004593 and PRJEB34009 in the ENA and were published by Derouane et al.<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> (Breast Cancer Res 26, 29) and Kosinsky et al.<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> (Cell Death Dis 10, 911) respectively.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Steve Bilodeau, Email: steve.bilodeau@crchudequebec.ulaval.ca.</p>
<p>Arnaud Droit, Email: arnaud.droit@crchudequebec.ulaval.ca.</p></section><section id="sec15"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>The online version contains supplementary material available at 10.1038/s41598-025-16875-9.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Ximin Li, C.-Y. W. From bulk, single-cell to spatial RNA sequencing. <em>Int. J. Oral Sci</em>. <strong>13</strong>, (2021).</cite> [<a href="https://doi.org/10.1038/s41368-021-00146-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8593179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34782601/" class="usa-link">PubMed</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Mangul, S. et al. ROP: Dumpster diving in RNA-sequencing to find the source of 1 trillion reads across diverse adult human tissues. <em>Genome Biol.</em><strong>19</strong>, 36 (2018).
</cite> [<a href="https://doi.org/10.1186/s13059-018-1403-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5857127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29548336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mangul,%20S.%20et%20al.%20ROP:%20Dumpster%20diving%20in%20RNA-sequencing%20to%20find%20the%20source%20of%201%20trillion%20reads%20across%20diverse%20adult%20human%20tissues.%20Genome%20Biol.19,%2036%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: A flexible trimmer for Illumina sequence data. <em>Bioinformatics</em><strong>30</strong>, 2114–2120 (2014).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bolger,%20A.%20M.,%20Lohse,%20M.%20&amp;%20Usadel,%20B.%20Trimmomatic:%20A%20flexible%20trimmer%20for%20Illumina%20sequence%20data.%20Bioinformatics30,%202114%E2%80%932120%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Chen, S. Ultrafast one-pass FASTQ data preprocessing, quality control, and deduplication using fastp. <em>Imeta</em><strong>2</strong>, e107 (2023).
</cite> [<a href="https://doi.org/10.1002/imt2.107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10989850/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38868435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20S.%20Ultrafast%20one-pass%20FASTQ%20data%20preprocessing,%20quality%20control,%20and%20deduplication%20using%20fastp.%20Imeta2,%20e107%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Shibaguchi, H. &amp; Yasutaka, Y. HiSAT: A novel method for the rapid diagnosis of allergy. <em>Bio. Protoc.</em><strong>12</strong>, e4309 (2022).
</cite> [<a href="https://doi.org/10.21769/BioProtoc.4309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8855079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35284606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shibaguchi,%20H.%20&amp;%20Yasutaka,%20Y.%20HiSAT:%20A%20novel%20method%20for%20the%20rapid%20diagnosis%20of%20allergy.%20Bio.%20Protoc.12,%20e4309%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Dobin, A. et al. STAR: Ultrafast universal RNA-seq aligner. <em>Bioinformatics</em><strong>29</strong>, 15–21 (2013).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3530905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dobin,%20A.%20et%20al.%20STAR:%20Ultrafast%20universal%20RNA-seq%20aligner.%20Bioinformatics29,%2015%E2%80%9321%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>McKenna, A. et al. The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data. <em>Genome Res.</em><strong>20</strong>, 1297–1303 (2010).
</cite> [<a href="https://doi.org/10.1101/gr.107524.110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2928508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20644199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McKenna,%20A.%20et%20al.%20The%20Genome%20Analysis%20Toolkit:%20A%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data.%20Genome%20Res.20,%201297%E2%80%931303%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Kim, S. et al. Strelka2: Fast and accurate calling of germline and somatic variants. <em>Nat. Methods</em><strong>15</strong>, 591–594 (2018).
</cite> [<a href="https://doi.org/10.1038/s41592-018-0051-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30013048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20S.%20et%20al.%20Strelka2:%20Fast%20and%20accurate%20calling%20of%20germline%20and%20somatic%20variants.%20Nat.%20Methods15,%20591%E2%80%93594%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Garrido-Rodriguez, M. et al. A versatile workflow to integrate RNA-seq genomic and transcriptomic data into mechanistic models of signaling pathways. <em>PLoS Comput. Biol.</em><strong>17</strong>, e1008748 (2021).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1008748" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7904194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33571195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garrido-Rodriguez,%20M.%20et%20al.%20A%20versatile%20workflow%20to%20integrate%20RNA-seq%20genomic%20and%20transcriptomic%20data%20into%20mechanistic%20models%20of%20signaling%20pathways.%20PLoS%20Comput.%20Biol.17,%20e1008748%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Zhi-Xiong, C. Single-Cell RNA sequencing in ovarian cancer: Current progress and future prospects. <em>Prog. Biophys. Mol. Biol.</em>10.1016/j.pbiomolbio.2025.01.002 (2025).
</cite> [<a href="https://doi.org/10.1016/j.pbiomolbio.2025.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39778630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhi-Xiong,%20C.%20Single-Cell%20RNA%20sequencing%20in%20ovarian%20cancer:%20Current%20progress%20and%20future%20prospects.%20Prog.%20Biophys.%20Mol.%20Biol.10.1016/j.pbiomolbio.2025.01.002%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Newman, A. M. et al. Determining cell type abundance and expression from bulk tissues with digital cytometry. <em>Nat. Biotechnol.</em><strong>37</strong>, 773–782 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0114-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6610714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31061481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Newman,%20A.%20M.%20et%20al.%20Determining%20cell%20type%20abundance%20and%20expression%20from%20bulk%20tissues%20with%20digital%20cytometry.%20Nat.%20Biotechnol.37,%20773%E2%80%93782%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Fan, J. <em>et al.</em> MuSiC2: Cell-type deconvolution for multi-condition bulk RNA-seq data. <em>Brief. Bioinform</em>. <strong>23</strong>, (2022).</cite> [<a href="https://doi.org/10.1093/bib/bbac430" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9677503/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36208175/" class="usa-link">PubMed</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Zeng, T., Huang, T. &amp; Lu, C. <em>Machine Learning Advanced Dynamic Omics Data Analysis for Precision Medicine</em>. (Frontiers Media SA, 2020).</cite> [<a href="https://doi.org/10.3389/fgene.2019.01343" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7010801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32117409/" class="usa-link">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>El Naqa, I., Li, R. &amp; Murphy, M. J. <em>Machine Learning in Radiation Oncology: Theory and Applications</em>. (Springer, 2015).</cite>
</li>
<li id="CR15">
<span class="label">15.</span><cite>Saghaleyni, R., Sheikh Muhammad, A., Bangalore, P., Nielsen, J. &amp; Robinson, J. L. Machine learning-based investigation of the cancer protein secretory pathway. <em>PLoS Comput. Biol.</em><strong>17</strong>, e1008898 (2021).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1008898" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8049480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33819271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saghaleyni,%20R.,%20Sheikh%20Muhammad,%20A.,%20Bangalore,%20P.,%20Nielsen,%20J.%20&amp;%20Robinson,%20J.%20L.%20Machine%20learning-based%20investigation%20of%20the%20cancer%20protein%20secretory%20pathway.%20PLoS%20Comput.%20Biol.17,%20e1008898%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Luca, B. A. et al. Atlas of clinically distinct cell states and ecosystems across human solid tumors. <em>Cell</em><strong>184</strong>, 5482-5496.e28 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.09.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8526411/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34597583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luca,%20B.%20A.%20et%20al.%20Atlas%20of%20clinically%20distinct%20cell%20states%20and%20ecosystems%20across%20human%20solid%20tumors.%20Cell184,%205482-5496.e28%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Mölder, F. <em>et al.</em> Sustainable data analysis with Snakemake. <em>F1000Res</em><strong>10</strong>, 33 (2021).</cite> [<a href="https://doi.org/10.12688/f1000research.29032.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8114187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34035898/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Kurtzer, G. M., Sochat, V. &amp; Bauer, M. W. Singularity: Scientific containers for mobility of compute. <em>PLoS ONE</em><strong>12</strong>, e0177459 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0177459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5426675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28494014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kurtzer,%20G.%20M.,%20Sochat,%20V.%20&amp;%20Bauer,%20M.%20W.%20Singularity:%20Scientific%20containers%20for%20mobility%20of%20compute.%20PLoS%20ONE12,%20e0177459%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>The R Project for Statistical Computing. <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R-project.org/</a>.</cite>
</li>
<li id="CR20">
<span class="label">20.</span><cite>Van Rossum, G. &amp; Drake, F. L., Jr. <em>The Python Language Reference Manual</em>. (Network Theory, 2011).</cite>
</li>
<li id="CR21">
<span class="label">21.</span><cite>Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic RNA-seq quantification. <em>Nat. Biotechnol.</em><strong>34</strong>, 525–527 (2016).
</cite> [<a href="https://doi.org/10.1038/nbt.3519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27043002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bray,%20N.%20L.,%20Pimentel,%20H.,%20Melsted,%20P.%20&amp;%20Pachter,%20L.%20Near-optimal%20probabilistic%20RNA-seq%20quantification.%20Nat.%20Biotechnol.34,%20525%E2%80%93527%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Everaert, C. et al. Benchmarking of RNA-sequencing analysis workflows using whole-transcriptome RT-qPCR expression data. <em>Sci. Rep.</em><strong>7</strong>, 1559 (2017).
</cite> [<a href="https://doi.org/10.1038/s41598-017-01617-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5431503/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28484260/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Everaert,%20C.%20et%20al.%20Benchmarking%20of%20RNA-sequencing%20analysis%20workflows%20using%20whole-transcriptome%20RT-qPCR%20expression%20data.%20Sci.%20Rep.7,%201559%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Heldenbrand, J. R. et al. Recommendations for performance optimizations when using GATK3.8 and GATK4. <em>BMC Bioinform.</em><strong>20</strong>, 557 (2019).</cite> [<a href="https://doi.org/10.1186/s12859-019-3169-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6842142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31703611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heldenbrand,%20J.%20R.%20et%20al.%20Recommendations%20for%20performance%20optimizations%20when%20using%20GATK3.8%20and%20GATK4.%20BMC%20Bioinform.20,%20557%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Brouard, J.-S., Schenkel, F., Marete, A. &amp; Bissonnette, N. The GATK joint genotyping workflow is appropriate for calling variants in RNA-seq experiments. <em>J. Anim. Sci. Biotechnol.</em><strong>10</strong>, 44 (2019).
</cite> [<a href="https://doi.org/10.1186/s40104-019-0359-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6587293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31249686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brouard,%20J.-S.,%20Schenkel,%20F.,%20Marete,%20A.%20&amp;%20Bissonnette,%20N.%20The%20GATK%20joint%20genotyping%20workflow%20is%20appropriate%20for%20calling%20variants%20in%20RNA-seq%20experiments.%20J.%20Anim.%20Sci.%20Biotechnol.10,%2044%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Avila Cobos, F., Alquicira-Hernandez, J., Powell, J. E., Mestdagh, P. &amp; De Preter, K. Benchmarking of cell type deconvolution pipelines for transcriptomics data. <em>Nat. Commun</em>. <strong>11</strong>, 1–14 (2020).</cite> [<a href="https://doi.org/10.1038/s41467-020-19015-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7648640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33159064/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Ren, Y. et al. Deconvolution reveals cell-type-specific transcriptomic changes in the aging mouse brain. <em>Sci. Rep.</em><strong>13</strong>, 16855 (2023).
</cite> [<a href="https://doi.org/10.1038/s41598-023-44183-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10558435/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37803069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ren,%20Y.%20et%20al.%20Deconvolution%20reveals%20cell-type-specific%20transcriptomic%20changes%20in%20the%20aging%20mouse%20brain.%20Sci.%20Rep.13,%2016855%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Bouzeraa, L. et al. Decoding epigenetic markers: Implications of traits and genes through DNA methylation in resilience and susceptibility to mastitis in dairy cows. <em>Epigenetics</em><strong>19</strong>, 2391602 (2024).
</cite> [<a href="https://doi.org/10.1080/15592294.2024.2391602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11332640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39151128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bouzeraa,%20L.%20et%20al.%20Decoding%20epigenetic%20markers:%20Implications%20of%20traits%20and%20genes%20through%20DNA%20methylation%20in%20resilience%20and%20susceptibility%20to%20mastitis%20in%20dairy%20cows.%20Epigenetics19,%202391602%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Leitão, A. B. et al. Recognition of nonself is necessary to activate Drosophila’s immune response against an insect parasite. <em>BMC Biol.</em><strong>22</strong>, 89 (2024).
</cite> [<a href="https://doi.org/10.1186/s12915-024-01886-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11034056/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38644510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leit%C3%A3o,%20A.%20B.%20et%20al.%20Recognition%20of%20nonself%20is%20necessary%20to%20activate%20Drosophila%E2%80%99s%20immune%20response%20against%20an%20insect%20parasite.%20BMC%20Biol.22,%2089%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Derouane, F. et al. Metabolic adaptation towards glycolysis supports resistance to neoadjuvant chemotherapy in early triple negative breast cancers. <em>Breast Cancer Res.</em><strong>26</strong>, 29 (2024).
</cite> [<a href="https://doi.org/10.1186/s13058-024-01788-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10875828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38374113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Derouane,%20F.%20et%20al.%20Metabolic%20adaptation%20towards%20glycolysis%20supports%20resistance%20to%20neoadjuvant%20chemotherapy%20in%20early%20triple%20negative%20breast%20cancers.%20Breast%20Cancer%20Res.26,%2029%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>CZI Cell Science Program <em>et al.</em> CZ CELLxGENE Discover: A single-cell data platform for scalable exploration, analysis and modeling of aggregated data. <em>Nucleic Acids Res</em>. <strong>53</strong>, D886–D900 (2025).</cite> [<a href="https://doi.org/10.1093/nar/gkae1142" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11701654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39607691/" class="usa-link">PubMed</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Wu, S. Z. et al. A single-cell and spatially resolved atlas of human breast cancers. <em>Nat. Genet.</em><strong>53</strong>, 1334–1347 (2021).
</cite> [<a href="https://doi.org/10.1038/s41588-021-00911-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9044823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34493872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20S.%20Z.%20et%20al.%20A%20single-cell%20and%20spatially%20resolved%20atlas%20of%20human%20breast%20cancers.%20Nat.%20Genet.53,%201334%E2%80%931347%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Leclercq, M. et al. Large-scale automatic feature selection for biomarker discovery in high-dimensional OMICs data. <em>Front. Genet.</em><strong>10</strong>, 452 (2019).
</cite> [<a href="https://doi.org/10.3389/fgene.2019.00452" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6532608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31156708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leclercq,%20M.%20et%20al.%20Large-scale%20automatic%20feature%20selection%20for%20biomarker%20discovery%20in%20high-dimensional%20OMICs%20data.%20Front.%20Genet.10,%20452%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Scotto, L. et al. Identification of copy number gain and overexpressed genes on chromosome arm 20q by an integrative genomic approach in cervical cancer: Potential role in progression. <em>Genes Chromosomes Cancer</em><strong>47</strong>, 755–765 (2008).
</cite> [<a href="https://doi.org/10.1002/gcc.20577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18506748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scotto,%20L.%20et%20al.%20Identification%20of%20copy%20number%20gain%20and%20overexpressed%20genes%20on%20chromosome%20arm%2020q%20by%20an%20integrative%20genomic%20approach%20in%20cervical%20cancer:%20Potential%20role%20in%20progression.%20Genes%20Chromosomes%20Cancer47,%20755%E2%80%93765%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Turn, R. E. <em>et al.</em> The ARF GAPs ELMOD1 and ELMOD3 act at the Golgi and cilia to regulate ciliogenesis and ciliary protein traffic. <em>Mol. Biol. Cell</em>. <strong>33</strong>, cor1 (2022).</cite> [<a href="https://doi.org/10.1091/mbc.E21-09-0443_corr" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9561853/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35612986/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Ferreira, A., Castanheira, P., Escrevente, C., Barral, D. C. &amp; Barona, T. Membrane trafficking alterations in breast cancer progression. <em>Front. Cell Dev. Biol.</em><strong>12</strong>, 1350097 (2024).
</cite> [<a href="https://doi.org/10.3389/fcell.2024.1350097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10963426/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38533085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ferreira,%20A.,%20Castanheira,%20P.,%20Escrevente,%20C.,%20Barral,%20D.%20C.%20&amp;%20Barona,%20T.%20Membrane%20trafficking%20alterations%20in%20breast%20cancer%20progression.%20Front.%20Cell%20Dev.%20Biol.12,%201350097%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Kosinsky, R. L. et al. USP22-dependent HSP90AB1 expression promotes resistance to HSP90 inhibition in mammary and colorectal cancer. <em>Cell Death Dis.</em><strong>10</strong>, 911 (2019).
</cite> [<a href="https://doi.org/10.1038/s41419-019-2141-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6892875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31801945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kosinsky,%20R.%20L.%20et%20al.%20USP22-dependent%20HSP90AB1%20expression%20promotes%20resistance%20to%20HSP90%20inhibition%20in%20mammary%20and%20colorectal%20cancer.%20Cell%20Death%20Dis.10,%20911%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Website. <a href="https://www.techpowerup.com/cpu-specs/epyc-7601.c1920" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.techpowerup.com/cpu-specs/epyc-7601.c1920</a>.</cite>
</li>
<li id="CR38">
<span class="label">38.</span><cite>Le Hannah G. B., H. <em>et al.</em> Latch Verified Bulk-RNA Seq toolkit: a cloud-based suite of workflows for bulk RNA-seq quality control, analysis, and functional enrichment. <em>bioRxiv</em> (2022). 10.1101/2022.11.10.516016.</cite>
</li>
<li id="CR39">
<span class="label">39.</span><cite>Lataretu, M. &amp; Hölzer, M. RNAflow: An effective and simple RNA-Seq differential gene expression pipeline using Nextflow. <em>Genes (Basel)</em>. <strong>11</strong>, (2020).</cite> [<a href="https://doi.org/10.3390/genes11121487" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7763471/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33322033/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Eagles, N. J. et al. Correction to: SPEAQeasy: A scalable pipeline for expression analysis and quantification for R/bioconductor-powered RNA-seq analyses. <em>BMC Bioinform.</em><strong>22</strong>, 381 (2021).</cite> [<a href="https://doi.org/10.1186/s12859-021-04142-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8088074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33932985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eagles,%20N.%20J.%20et%20al.%20Correction%20to:%20SPEAQeasy:%20A%20scalable%20pipeline%20for%20expression%20analysis%20and%20quantification%20for%20R/bioconductor-powered%20RNA-seq%20analyses.%20BMC%20Bioinform.22,%20381%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Cokelaer, T., Desvillechabrol, D., Legendre, R. &amp; Cardon, M. Sequana’: A set of Snakemake NGS pipelines. <em>J. Open Source Softw.</em><strong>2</strong>, 352 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Cokelaer,%20T.,%20Desvillechabrol,%20D.,%20Legendre,%20R.%20&amp;%20Cardon,%20M.%20Sequana%E2%80%99:%20A%20set%20of%20Snakemake%20NGS%20pipelines.%20J.%20Open%20Source%20Softw.2,%20352%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Bourgey, M. <em>et al.</em> GenPipes: An open-source framework for distributed and scalable genomic analyses. <em>Gigascience</em>. <strong>8</strong>, (2019).</cite> [<a href="https://doi.org/10.1093/gigascience/giz037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6559338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31185495/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Sasagawa, S. et al. Predicting chemotherapy responsiveness in gastric cancer through machine learning analysis of genome, immune, and neutrophil signatures. <em>Gastr. Cancer</em><strong>28</strong>, 228–244 (2025).</cite> [<a href="https://doi.org/10.1007/s10120-024-01569-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11842519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39621213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sasagawa,%20S.%20et%20al.%20Predicting%20chemotherapy%20responsiveness%20in%20gastric%20cancer%20through%20machine%20learning%20analysis%20of%20genome,%20immune,%20and%20neutrophil%20signatures.%20Gastr.%20Cancer28,%20228%E2%80%93244%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Karakaslar, E. O. et al. A transcriptomic based deconvolution framework for assessing differentiation stages and drug responses of AML. <em>NPJ Precis. Oncol.</em><strong>8</strong>, 105 (2024).
</cite> [<a href="https://doi.org/10.1038/s41698-024-00596-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11102519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38762545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karakaslar,%20E.%20O.%20et%20al.%20A%20transcriptomic%20based%20deconvolution%20framework%20for%20assessing%20differentiation%20stages%20and%20drug%20responses%20of%20AML.%20NPJ%20Precis.%20Oncol.8,%20105%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375748/bin/41598_2025_16875_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 1.</a><sup> (64.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375748/bin/41598_2025_16875_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 2.</a><sup> (22.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375748/bin/41598_2025_16875_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 3.</a><sup> (5.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375748/bin/41598_2025_16875_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 4.</a><sup> (6.1KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The RnaXtract pipeline is publicly available at <a href="https://gitlab-pub.compbio.ulaval.ca/sophiane/rnaxtract" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gitlab-pub.compbio.ulaval.ca/sophiane/rnaxtract</a> under the GNU-GPL 3 licence and the documentation is accessible within the repository. The samples used to test RnaXtract accession number of this manuscript are available in the supplementary data file.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-16875-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_16875.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375748/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375748/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375748%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375748/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375748/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375748/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40850998/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375748/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40850998/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375748/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375748/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ttVTbkhl9drpZPZUwOAp4XRixI0Hv9HM881fGmzRMPIEi0vN2OaIR0g3NfmNoXWq">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
