
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Unveiling the Future of Infective Endocarditis Diagnosis: The Transformative Role of Metagenomic Next-Generation Sequencing in Culture-Negative Cases - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15251F5E8AEF58A3051F5E0025D64F64.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="jegh">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373624/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Journal of Epidemiology and Global Health">
<meta name="citation_title" content="Unveiling the Future of Infective Endocarditis Diagnosis: The Transformative Role of Metagenomic Next-Generation Sequencing in Culture-Negative Cases">
<meta name="citation_author" content="Shafiu A Umar Shinge">
<meta name="citation_author_institution" content="Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China">
<meta name="citation_author" content="Binbin Zhang">
<meta name="citation_author_institution" content="Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China">
<meta name="citation_author" content="Baixin Zheng">
<meta name="citation_author_institution" content="Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China">
<meta name="citation_author" content="Yongjia Qiang">
<meta name="citation_author_institution" content="Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China">
<meta name="citation_author" content="Hussein M Ali">
<meta name="citation_author_institution" content="Department of Pediatric, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, China">
<meta name="citation_author" content="Youmbi T Vanessa Melchiade">
<meta name="citation_author_institution" content="Department of Cardiovascular Surgery, Sun Yat-sen Memorial Hospital, Sun Yat-sen University, Guangzhou, China">
<meta name="citation_author" content="Lu Zhang">
<meta name="citation_author_institution" content="Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China">
<meta name="citation_author" content="Minnan Gao">
<meta name="citation_author_institution" content="Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China">
<meta name="citation_author" content="Guibin Feng">
<meta name="citation_author_institution" content="Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China">
<meta name="citation_author" content="Kuan Zeng">
<meta name="citation_author_institution" content="Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China">
<meta name="citation_author" content="Yanqi Yang">
<meta name="citation_author_institution" content="Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China">
<meta name="citation_author_institution" content="Department of Cardiothoracic and Vascular Surgery, University Hospital, Linköping University, Linköping, 58183 Sweden">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="15">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="108">
<meta name="citation_doi" content="10.1007/s44197-025-00455-1">
<meta name="citation_pmid" content="40847264">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373624/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373624/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373624/pdf/44197_2025_Article_455.pdf">
<meta name="description" content="Culture-negative infective endocarditis (CNE) remains a significant diagnostic challenge in cardiology and infectious disease, often leading to delayed or empirical treatment. Metagenomic next-generation sequencing (mNGS) has emerged as a ...">
<meta name="og:title" content="Unveiling the Future of Infective Endocarditis Diagnosis: The Transformative Role of Metagenomic Next-Generation Sequencing in Culture-Negative Cases">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Culture-negative infective endocarditis (CNE) remains a significant diagnostic challenge in cardiology and infectious disease, often leading to delayed or empirical treatment. Metagenomic next-generation sequencing (mNGS) has emerged as a ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373624/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373624">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1007/s44197-025-00455-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/44197_2025_Article_455.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373624%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373624/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373624/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373624/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-jegh.png" alt="Journal of Epidemiology and Global Health logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Journal of Epidemiology and Global Health" title="Link to Journal of Epidemiology and Global Health" shape="default" href="https://www.atlantis-press.com/journals/jegh" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">J Epidemiol Glob Health</button></div>. 2025 Aug 22;15(1):108. doi: <a href="https://doi.org/10.1007/s44197-025-00455-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1007/s44197-025-00455-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Epidemiol%20Glob%20Health%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Epidemiol%20Glob%20Health%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Epidemiol%20Glob%20Health%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Epidemiol%20Glob%20Health%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Unveiling the Future of Infective Endocarditis Diagnosis: The Transformative Role of Metagenomic Next-Generation Sequencing in Culture-Negative Cases</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shinge%20SAU%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Shafiu A Umar Shinge</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Shafiu A Umar Shinge</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shinge%20SAU%22%5BAuthor%5D" class="usa-link"><span class="name western">Shafiu A Umar Shinge</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Binbin Zhang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Binbin Zhang</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Binbin Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Baixin Zheng</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Baixin Zheng</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Baixin Zheng</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qiang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yongjia Qiang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yongjia Qiang</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qiang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yongjia Qiang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ali%20HM%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Hussein M Ali</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Hussein M Ali</span></h3>
<div class="p">
<sup>4</sup>Department of Pediatric, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ali%20HM%22%5BAuthor%5D" class="usa-link"><span class="name western">Hussein M Ali</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Melchiade%20YTV%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Youmbi T Vanessa Melchiade</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Youmbi T Vanessa Melchiade</span></h3>
<div class="p">
<sup>3</sup>Department of Cardiovascular Surgery, Sun Yat-sen Memorial Hospital, Sun Yat-sen University, Guangzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Melchiade%20YTV%22%5BAuthor%5D" class="usa-link"><span class="name western">Youmbi T Vanessa Melchiade</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Lu Zhang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Lu Zhang</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lu Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Minnan Gao</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Minnan Gao</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Minnan Gao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feng%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Guibin Feng</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Guibin Feng</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feng%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guibin Feng</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeng%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Kuan Zeng</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Kuan Zeng</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeng%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kuan Zeng</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Yanqi Yang</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Yanqi Yang</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China </div>
<div class="p">
<sup>2</sup>Department of Cardiothoracic and Vascular Surgery, University Hospital, Linköping University, Linköping, 58183 Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yanqi Yang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Cardiovascular Surgery, The Eighth Affiliated Hospital of Sun Yat-sen University, Futian, Shenzhen, 518033 China </div>
<div id="Aff2">
<sup>2</sup>Department of Cardiothoracic and Vascular Surgery, University Hospital, Linköping University, Linköping, 58183 Sweden </div>
<div id="Aff3">
<sup>3</sup>Department of Cardiovascular Surgery, Sun Yat-sen Memorial Hospital, Sun Yat-sen University, Guangzhou, China </div>
<div id="Aff4">
<sup>4</sup>Department of Pediatric, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 22; Accepted 2025 Aug 10; Collection date 2025 Dec.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373624  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40847264/" class="usa-link">40847264</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Culture-negative infective endocarditis (CNE) remains a significant diagnostic challenge in cardiology and infectious disease, often leading to delayed or empirical treatment. Metagenomic next-generation sequencing (mNGS) has emerged as a complementary diagnostic tool capable of identifying fastidious, unexpected, or novel pathogens without prior assumptions. This narrative review synthesizes evidence from 152 studies (2015–2024), evaluating mNGS within existing diagnostic frameworks for culture-negative IE. Compared to conventional diagnostics (blood cultures, PCR, 16 S rRNA sequencing), mNGS demonstrates enhanced detection capabilities for polymicrobial infections and rare pathogens, though methodological heterogeneity across studies precludes definitive performance comparisons. Performance varies substantially based on sample type, sequencing platform, and bioinformatic pipelines. Real-world applications reveal persistent challenges, including cost barriers, interpretive complexities in low-biomass samples, and contamination risks. Integration with host-response biomarkers and AI-driven interpretation platforms shows promise for advancing clinical utility. For mNGS to be effectively integrated into routine CNE care, standardization, regulatory clarity, and equitable implementation will be essential.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Metagenomic next-generation sequencing (mNGS), Culture-negative infective endocarditis, Heart valve, Infectious disease diagnostics, Antibiotic stewardship, Precision medicine</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par18">Infective endocarditis (IE) is a severe infection of the heart endocardial layer, predominantly affecting heart valves. Firstly, explained by Sir William Osler in 1885, IE has historically posed significant diagnostic and therapeutic challenges. Despite advancements in medical science, the prognosis for IE remains grim, with in-hospital or three-month mortality rates ranging from 18 to 25%, a figure that has remained largely unchanged over the years [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. The burden of IE on healthcare systems is substantial, given its associated high morbidity and long-term complications, which significantly impair patients’ quality of life. Over recent decades, changes in the epidemiology of IE have been observed. The increase in elderly populations, along with the increasing prevalences of degenerative valvular diseases and prosthetic heart valve usage, has shifted the context of this condition. Staphylococci, particularly Staphylococcus aureus, have surpassed Streptococci as the predominant causative pathogens. Furthermore, emerging and re-emerging pathogens, including rare and fastidious organisms, have complicated the clinical picture [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. For diagnostic challenges, culture-negative infective endocarditis (CNE) presents a unique set of challenges. CNE occurs when standard blood cultures fail to identify the causative organism, which happens in up to 30% of IE cases. The causes of CNE are multifactorial, including prior antibiotic therapy, which could suppresses bacterial growth in cultures; the presence of fastidious organisms that require specific growth conditions not met by routine culture techniques; and intracellular pathogens that evade detection [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. The clinical implications of CNE are profound. Diagnostic delays are common, as traditional methods often yield inconclusive results, leading to prolonged empirical therapy with broad-spectrum antibiotics. This approach can contribute to antibiotic resistance and adverse drug reactions, further complicating patient management. Inappropriate or delayed treatment significantly impacts patient outcomes, underlining the necessity for more effective diagnostic strategies [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Comparatively, molecular diagnostic techniques have revolutionized the approach to diagnosing infectious diseases, including IE. These methods offer several advantages over traditional culture-based techniques, particularly in their ability to detect a wide range of pathogens with high sensitivity and specificity (Table <a href="#Tab1" class="usa-link">1</a>). One of these methods, polymerase chain reaction (PCR) and 16 S rRNA sequencing have provided substantial improvements but are limited by the requirement for prior knowledge of the target pathogen and the scope of detectable organisms [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Metagenomic next-generation sequencing (mNGS) represents a significant advancement in molecular diagnostics. This technique enables for the extensive analysis of genetic material from clinical samples, identifying all present pathogens without prior knowledge. mNGS works by sequencing entire sample on nucleic acids, followed by bioinformatics analysis to identify the genetic sequences of potential pathogens [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Unlike PCR and targeted sequencing, mNGS offers a broader detection spectrum, including viruses, fungi, bacteria, and parasites. Our current work aims to critically evaluate the mNGS application in the early and accurate diagnoses of CNE. This work is not a formal meta-analysis, but it synthesizes recent advancements and the best available resources, offering new insights into the diagnostic potential of mNGS. We compare mNGS with traditional diagnostic methods, discuss its clinical utility, and examine its impact on patient outcomes. Additionally, we identify current trends, research gaps, and propose future directions to optimize the mNGS utilization in clinical practice. The review provides a extensive exploration on the evolution of diagnostic techniques for IE (Table <a href="#Tab2" class="usa-link">2</a>), with a focus on mNGS. It could serve as important resources for healthcare professionals, academics, and administrators about the potential of mNGS in improving the IE diagnoses and management. Through this synthesis, we hope to highlight the critical advancements and future possibilities that mNGS brings to the aspect of infectious diseases diagnostics.</p>
<section class="tw xbox font-sm" id="Tab1"><h3 class="obj_head">Table 1.</h3>
<div class="caption p"><p>Comparative sensitivity &amp; specificity on mNGS vs. traditional diagnostic Methods</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Diagnostic Method</th>
<th align="left" colspan="1" rowspan="1">Sensitivity</th>
<th align="left" colspan="1" rowspan="1">Specificity</th>
<th align="left" colspan="1" rowspan="1">Detection Time</th>
<th align="left" colspan="1" rowspan="1">Pathogen Coverage</th>
<th align="left" colspan="1" rowspan="1">Reference</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Traditional Blood Culture</td>
<td align="left" colspan="1" rowspan="1">40–60%</td>
<td align="left" colspan="1" rowspan="1">High</td>
<td align="left" colspan="1" rowspan="1">Days to Weeks</td>
<td align="left" colspan="1" rowspan="1">Bacteria, Limited</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PCR</td>
<td align="left" colspan="1" rowspan="1">70–80%</td>
<td align="left" colspan="1" rowspan="1">High</td>
<td align="left" colspan="1" rowspan="1">Hours to Days</td>
<td align="left" colspan="1" rowspan="1">Targeted Pathogens</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>, <a href="#CR103" class="usa-link" aria-describedby="CR103">103</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mNGS</td>
<td align="left" colspan="1" rowspan="1">90–95%</td>
<td align="left" colspan="1" rowspan="1">High</td>
<td align="left" colspan="1" rowspan="1">24–48 Hours</td>
<td align="left" colspan="1" rowspan="1">Comprehensive</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>]</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab2"><h3 class="obj_head">Table 2.</h3>
<div class="caption p"><p>Evolution of diagnostic techniques for infective endocarditis</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Diagnostic Technique</th>
<th align="left" colspan="1" rowspan="1">Introduction Period</th>
<th align="left" colspan="1" rowspan="1">Key Advantages</th>
<th align="left" colspan="1" rowspan="1">Limitations</th>
<th align="left" colspan="1" rowspan="1">Examples of Pathogens Detected</th>
<th align="left" colspan="1" rowspan="1">References</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Blood Cultures</td>
<td align="left" colspan="1" rowspan="1">Early 1900s</td>
<td align="left" colspan="1" rowspan="1">Gold standard; can identify a wide range of bacteria in IE; easy to implement.</td>
<td align="left" colspan="1" rowspan="1">Fails in 30% of cases, especially post-antibiotics; slow to yield results (days).</td>
<td align="left" colspan="1" rowspan="1">Staphylococcus aureus, Streptococcus viridans</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Serology</td>
<td align="left" colspan="1" rowspan="1">1950s-1960s</td>
<td align="left" colspan="1" rowspan="1">Non-invasive; useful for fastidious organisms like Coxiella burnetii.</td>
<td align="left" colspan="1" rowspan="1">Requires specific tests for different organisms; lacks broad applicability.</td>
<td align="left" colspan="1" rowspan="1">Coxiella burnetii (Q fever)</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Histopathology</td>
<td align="left" colspan="1" rowspan="1">1900s (Routine use began later)</td>
<td align="left" colspan="1" rowspan="1">Direct visualization of valvular tissue; provides definitive diagnosis.</td>
<td align="left" colspan="1" rowspan="1">Invasive; requires heart valve biopsy; not always feasible.</td>
<td align="left" colspan="1" rowspan="1">Any organism causing valve damage (e.g., bacterial, fungal)</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Polymerase Chain Reaction (PCR)</td>
<td align="left" colspan="1" rowspan="1">1980s-1990s</td>
<td align="left" colspan="1" rowspan="1">High sensitivity; detects DNA directly from clinical samples.</td>
<td align="left" colspan="1" rowspan="1">Relatively costly and limited to known pathogens with available primers.</td>
<td align="left" colspan="1" rowspan="1">Bartonella, Tropheryma whipplei</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">16S rRNA Sequencing</td>
<td align="left" colspan="1" rowspan="1">Late 1990s</td>
<td align="left" colspan="1" rowspan="1">Broadens bacterial identification range; effective for rare and slow-growing bacteria.</td>
<td align="left" colspan="1" rowspan="1">Limited to bacterial pathogens; cannot detect viruses or fungi.</td>
<td align="left" colspan="1" rowspan="1">Tropheryma whipplei, Bartonella species</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Metagenomic Next-Generation Sequencing (mNGS)</td>
<td align="left" colspan="1" rowspan="1">2010s-Present</td>
<td align="left" colspan="1" rowspan="1">Rapid, comprehensive detection of bacteria, fungi, and viruses.</td>
<td align="left" colspan="1" rowspan="1">High cost, complex data interpretation, lack of standardized clinical protocols.</td>
<td align="left" colspan="1" rowspan="1">Broad range: bacterial, viral, fungal (including rare and fastidious organisms)</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec2"><h2 class="pmc_sec_title">Methodology</h2>
<p id="Par21">A comprehensive literature search was conducted in accordance with PRISMA guidelines across PubMed, Scopus, and Web of Science to assess the role of metagenomic next-generation sequencing (mNGS) in diagnosing culture-negative endocarditis (CNE). The search spanned January 2015 to February 2024 using keywords such as ‘mNGS,’ ‘infective endocarditis,’ and ‘culture-negative endocarditis.’ Only human, English-language, peer-reviewed studies were included. Additional records were identified through manual screening of reference lists from guidelines (ESC, ACC/AHA), reviews, and meta-analyses. Eligible studies included original research and case series evaluating mNGS for CNE diagnosis, particularly those comparing it to conventional diagnostics (e.g., blood cultures, PCR, 16 S rRNA) or reporting on clinical outcomes, diagnostic performance, or cost-effectiveness. Exclusion criteria included non-human studies, non-English publications, and isolated case reports without outcome or comparative data. From 1,247 initial records, two independent reviewers screened titles/abstracts and performed full-text reviews, yielding 152 studies meeting inclusion criteria. Data were extracted using a standardized template capturing study design, patient population, diagnostic methods, performance metrics, clinical impact, and cost data. Subgroup analyses were performed based on sample type, sequencing platform, and bioinformatics approach. Study quality and risk of bias were assessed using the QUADAS-2 tool. Data synthesis included narrative review and pooled diagnostic estimates from high-quality studies, emphasizing comparisons between mNGS and conventional methods on their clinical utility. Heterogeneity was critically appraised with attention to methodological and cohort-level differences.</p></section><section id="Sec3"><h2 class="pmc_sec_title">Evolution of Diagnostic Techniques for Infective Endocarditis</h2>
<section id="Sec4"><h3 class="pmc_sec_title">Early Diagnostic Methods</h3>
<p id="Par22">Blood cultures have long been considered the ideal method in diagnosing infective endocarditis (IE), Tables <a href="#Tab2" class="usa-link">2</a> and <a href="#Tab3" class="usa-link">3</a>. This technique, while foundational, suffers from significant limitations. For example, studies indicate that blood cultures fail to identify pathogens in up to 30% of IE cases, particularly in those who receive prior antibiotics [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. This high rate of culture-negative results underscores the inadequacy of blood cultures in a subset of patients. Furthermore, blood cultures can take many days to yield results, delaying crucial treatment necessitating advanced diagnostic strategies. Comparatively, modern molecular techniques promise faster with highly precise diagnoses, emphasizing the needs for advancement beyond traditional methods (Fig. <a href="#Fig1" class="usa-link">1</a>). For the role of serology and histopathology in IE diagnosis, serological tests and histopathological examinations have supplemented blood cultures in diagnosing IE. Serology is highly significant in detecting antibodies against fastidious microbes, like Coxiella burnetii, responsible for Q fever endocarditis [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Histopathology provides direct visual confirmation of infection through examination of excised heart valves. While these methods offer critical diagnostic insights, their utility is restricted by the availability of specific tests and the invasive nature of tissue sampling. The need for less invasive yet highly accurate diagnostic tools remain pressing. Historical case studies intensely demonstrate the difficulties posed by early diagnostic methods. For instance, before the advent of advanced diagnostics, many cases of IE were only definitively diagnosed post-mortem, revealing the diagnostic limitations of the time [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. These cases often involved extensive valvular damage by the time of diagnosis, emphasizing the critical importance of earlier detection methods. The contrast with current capabilities underscores the significant progress made and highlights ongoing gaps that require further innovation. However, continuous research effort led to the presentation of polymerase chain reaction (PCR) and 16 S rRNA sequencing represented a leap forward in molecular diagnostics for IE summarized in Table <a href="#Tab2" class="usa-link">2</a>. PCR permits the specific sequences amplification, thus detecting pathogens directly from clinical sample with higher sensitivity. 16 S rRNA sequencing, which targets a conserved region of bacterial ribosomal RNA, broadens the detection scope to include a variety of bacteria [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Studies demonstrated that these methods significantly improve diagnostic accuracy in cases where traditional cultures fail, such as identifying Tropheryma whipplei and Bartonella species [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>].</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Summary of current IE diagnosis and treatment strategies by major global cardiovascular organizations</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Organization</th>
<th align="left" colspan="1" rowspan="1">Diagnostic Criteria</th>
<th align="left" colspan="1" rowspan="1">Diagnostic Methods</th>
<th align="left" colspan="1" rowspan="1">Treatment Strategies</th>
<th align="left" colspan="1" rowspan="1">Key Guidelines &amp; Updates</th>
<th align="left" colspan="1" rowspan="1">Reference</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">American Heart Association (AHA)</td>
<td align="left" colspan="1" rowspan="1">Modified Duke Criteria</td>
<td align="left" colspan="1" rowspan="1">Blood cultures, Echocardiography (TTE/TEE), Serology</td>
<td align="left" colspan="1" rowspan="1">Empirical broad-spectrum antibiotics, Valve surgery in severe cases</td>
<td align="left" colspan="1" rowspan="1">2020 Update: Emphasis on early surgery for certain high-risk patients</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">European Society of Cardiology (ESC)</td>
<td align="left" colspan="1" rowspan="1">Modified Duke Criteria with emphasis on imaging</td>
<td align="left" colspan="1" rowspan="1">Blood cultures, Echocardiography, 18F-FDG PET/CT for prosthetic valve IE</td>
<td align="left" colspan="1" rowspan="1">Targeted antibiotic therapy based on culture results, Early surgery for complications</td>
<td align="left" colspan="1" rowspan="1">2015 Guidelines: Inclusion of 18F-FDG PET/CT in diagnostic criteria</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">British Society for Antimicrobial Chemotherapy (BSAC)</td>
<td align="left" colspan="1" rowspan="1">Modified Duke Criteria</td>
<td align="left" colspan="1" rowspan="1">Blood cultures, Echocardiography, Molecular methods (PCR)</td>
<td align="left" colspan="1" rowspan="1">Combination antibiotic therapy, Consideration for surgery</td>
<td align="left" colspan="1" rowspan="1">2020 Guidelines: Updated antibiotic regimens, More emphasis on molecular diagnostics</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">World Health Organization (WHO)</td>
<td align="left" colspan="1" rowspan="1">Adaptation of Duke Criteria</td>
<td align="left" colspan="1" rowspan="1">Blood cultures, Echocardiography, Emerging molecular diagnostics (mNGS)</td>
<td align="left" colspan="1" rowspan="1">Standardized antibiotic protocols, Surgical intervention in severe cases</td>
<td align="left" colspan="1" rowspan="1">2018 REASSURED: Promotion of molecular diagnostics in resource-limited settings</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Infectious Diseases Society of America (IDSA)</td>
<td align="left" colspan="1" rowspan="1">Modified Duke Criteria</td>
<td align="left" colspan="1" rowspan="1">Blood cultures, Echocardiography, Histopathology</td>
<td align="left" colspan="1" rowspan="1">Empirical antibiotics, Custom therapy post-identification, Valve replacement</td>
<td align="left" colspan="1" rowspan="1">2015 Guidelines: Integration of newer diagnostic techniques, Aggressive surgical approach in high-risk patients</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373624_44197_2025_455_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc20/12373624/04b815ee91d7/44197_2025_455_Fig1_HTML.jpg" loading="lazy" id="d33e899" height="405" width="685" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Principles and clinical application of mNGS in culture negative IE. Figure 1 <em>illustrates the mNGS workflow for diagnosing CNE</em>,<em> from tissue sample processing and sequencing to bioinformatic analysis. Highlighting pathogen identification</em>,<em> phylogenetic analysis</em>,<em> and the clinical application of results for improving diagnosis</em>,<em> guiding therapy</em>,<em> and enhancing patient outcomes</em></p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">mNGS in CNE Diagnosis</h3>
<p id="Par24">The persistent challenge of CNE, accounting for approximately 30% of IE cases [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>], representing a critical diagnostic frontier where conventional methods reach their limitations. Prior antibiotic exposure, fastidious microorganisms, and intracellular pathogens collectively create a perfect diagnostic storm, leaving clinicians without actionable microbiological guidance. While PCR and 16 S rRNA sequencing offered incremental improvements, their targeted nature inherently restricts detection to predefined pathogens [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Against this backdrop, mNGS emerges as a complement to traditional approaches, enabling comprehensive, unbiased pathogen detection directly from clinical specimens without prior cultivation [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. This approach could enhance diagnostic possibilities for CNE by simultaneously interrogating bacterial, viral, fungal, and parasitic genomes within a single assay.</p>
<section id="Sec6"><h4 class="pmc_sec_title">Technical Principles and Diagnostic Advantages</h4>
<p id="Par25">The core innovation of mNGS lies in its capacity to sequence all nucleic acids from host and microbial extracts in clinical samples. This hypothesis-free approach is particularly powerful for IE diagnostics, where causative agents range from common bacteria to exotic fastidious pathogens [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. The analytical process integrates two critical phases: (i) Wet laboratory processing begins with optimal specimen selection. For IE, excised valvular tissue offers the highest diagnostic yield, as it harbors embedded pathogens protected from immune clearance. Blood samples, though less sensitive, provide a non-invasive alternative. Nucleic acid extraction must preserve both DNA and RNA to capture the full pathogen spectrum. Subsequent library preparation fragments nucleic acids and adds platform-specific adapters. A critical decision point involves selecting DNA-only sequencing (for bacteria, fungi, DNA viruses, parasites), RNA-only (for RNA viruses), or co-testing when the etiological category remains unknown, a flexibility indispensable for IE diverse microbiology [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. (ii) Bioinformatic analysis employs sophisticated computational pipelines to manage massive sequence datasets. After quality control, human-derived sequences (constituting &gt; 99% of data in blood/tissue samples) are subtracted using reference genomes. The remaining sequences undergo alignment against comprehensive microbial databases, enabling pathogen identification at species or even strain levels. Advanced algorithms further characterize antimicrobial resistance genes and virulence factors, providing clinically actionable insights beyond mere detection [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. This integrated workflow typically delivers results within 24–48 h, dramatically accelerating therapeutic decisions compared to weeks required for serological conversion or specialized cultures [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Figure <a href="#Fig1" class="usa-link">1</a> illustrates the mNGS technical principles. For deeper discussion on wet-lab workflows and bioinformatic innovations, readers should refer to recent advances in portable on-site sequencing [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>], fully automated clinical systems [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>], and comparative analyses of mNGS versus targeted NGS platforms [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>].</p>
<p id="Par26">The diagnostic superiority of mNGS manifests in several IE-specific applications, including (i) Detection of fastidious pathogens: mNGS excels in identifying <em>Bartonella</em> spp., <em>Coxiella burnetii</em>, and <em>Tropheryma whipplei</em>, organisms notoriously challenging to cultivate [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. Recent studies demonstrate mNGS identifying <em>Bartonella henselae</em> directly from excised valves in patients with months of unexplained febrile illness, where blood cultures and serology yielded equivocal results [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. (ii) Unbiased pathogen agnosticism: Unlike targeted methods, mNGS diagnosed a case of <em>Mycoplasma hominis</em> prosthetic valve endocarditis an organism not considered in initial differentials, highlighting its capacity for unexpected discoveries [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. (iii) Polymicrobial infection resolution: mNGS revealed mixed infections including <em>Enterococcus faecalis</em> and Granulicatella adiacens infections in a case of culture-negative IE post-cardiac device implantation, fundamentally altering antimicrobial strategy [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. (iv) Simultaneous DNA/RNA Interrogation: Co-testing allows detection of diverse pathogen types in a single assay, crucial for cases with broad differential diagnoses [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. (v) Potential for Rapid Turnaround: While current clinical pipelines vary, optimized mNGS workflows can potentially deliver results within 24–48 h of sample receipt, significantly faster than prolonged culture incubation or serological convalescent titers [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. (vi) Direct Analysis from Complex Samples: mNGS can be applied directly to surgically excised valve tissue (the gold-standard specimen), blood, or emboli, bypassing the need for viable organisms [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. However, mNGS sensitivity is constrained by low pathogen biomass and high host DNA background, necessitating expert interpretation to differentiate contamination from true infection; semi-quantitative results may obscure minor co-pathogens, and resistance gene detection cannot confirm functional expression or replace phenotypic susceptibility testing [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. For comprehensive details on mNGS methodologies, laboratory validation [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>], diagnostic advantages and limitations, see recent studies by Zhao 2024 [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>], Miller 2019 [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>], and comparative performance assessments Yi 2025 [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>].</p></section><section id="Sec7"><h4 class="pmc_sec_title">Sequencing Platform Evolution and Technical Trade-Offs for IE Applications</h4>
<p id="Par28">The diagnostic performance of mNGS hinges on selecting appropriate sequencing platforms, each with distinct advantages and limitations for IE applications: (i) Short-read platforms (Illumina/BGI) dominate clinical laboratories due to unparalleled accuracy (error rates &lt; 0.1%) and high throughput. The NovaSeq 6000 platform enables deep sequencing of low-biomass samples like blood, enhancing sensitivity for bloodstream infections. However, their read lengths (150–300 bp) struggle with repetitive genomic elements common in resistance islands and virulence cassettes [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. Recent enhancements in molecular tagging have improved error correction, making these platforms ideal for species-level identification and SNP-based resistance detection 56. (ii) Long-read technologies (Oxford Nanopore) generate reads exceeding 10 kb, enabling complete assembly of resistance operons and virulence gene clusters [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>–<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. This proves invaluable when predicting phenotypic resistance patterns in Enterobacteriaceae or Pseudomonas causing prosthetic valve infections. The MinION device portability facilitates rapid on-site sequencing [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>], with studies reporting IE diagnosis within 6 h of sample receipt [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>, <a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. Despite higher per-base error rates (5–15%), adaptive base-calling algorithms have significantly improved accuracy for clinical isolates [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. Crucially, Nanopore real-time sequencing permits aborting runs once pathogen-specific signatures are detected, conserving resources a feature leveraged in urgent IE diagnostics. The 2025 ABRF NGS Phase II and other studies provides critical platform comparisons: Illumina platforms achieved 99.4% sensitivity for single-pathogen infections, while Nanopore demonstrated superior performance in resolving mixed infections and structural variants. For IE diagnostics, where both precision and comprehensive resistance profiling matter, a hybrid approach using Illumina for primary detection and Nanopore for resistance/virulence characterization offers a powerful synergistic solution [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>, <a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>–<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. While nanopore sequencing exhibits higher error rates (∼5–15%) compared to Illumina platforms (&lt; 0.1%), its clinical impact varies by use case. For endocarditis diagnostics where the primary goal is pathogen identification to genus/species level rather than fine-scale strain typing, some report suggests accuracy may be sufficient [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>].</p></section></section><section id="Sec8"><h3 class="pmc_sec_title">Diagnostic Interpretation Challenges and Signal Vs. Noise in mNGS Results</h3>
<p id="Par31">mNGS is not a standalone diagnostic solution but a complementary tool within the Duke Criteria framework recommended by major cardiovascular organizations globally (Table <a href="#Tab3" class="usa-link">3</a>). Its value lies in enabling pathogen-directed therapy for previously undiagnosed CNE. One of mNGS strengths, its sensitivity (Table <a href="#Tab1" class="usa-link">1</a>), is also its major interpretive challenge. Not every sequence read equates to infection. Background flora, environmental DNA, and lab contaminants can all show up in results, especially in low-biomass samples like blood. Salter et al. (2014) showed that even trace contamination from reagents can skew microbiome analyses [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. In clinical mNGS, the stakes are higher: a false-positive hit can lead to unnecessary treatments or missed diagnoses. That is why modern mNGS protocols rely heavily on contamination controls, blank runs, batch indexing, and UMIs [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. Despite UMIs and batch controls, reagent-derived contamination persists in 15–30% of blood samples, requiring integration with host biomarkers for reliable interpretation. But beyond technical safeguards, clinical context is paramount. The most powerful mNGS result is one that aligns with patient presentation, imaging, and disease course. In an individual with prosthetic valve dysfunction and persistent fever, the detection of Bartonella henselae (previously undetected by PCR) may be the key to unlocking effective treatment [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. In a different patient with vague symptoms and low pathogen load, the same result might represent irrelevant noise. Here, multidisciplinary interpretation becomes essential. The best outcomes occur when clinicians, microbiologists, and bioinformaticians interpret results together, triangulating between data, disease, and decision-making [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. Emerging probabilistic scoring frameworks now weight pathogen likelihood using clinical parameters (e.g., fever duration, epidemiological exposure) and sequence abundance thresholds, reducing false positives in polymicrobial samples. For instance, algorithms like directed network flow entropy (DNFE) integrate host-response biomarkers (e.g., IL-6, procalcitonin) to distinguish pathogens from commensals in real time [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. Multidisciplinary review teams including microbiologists, infectious disease specialists, and bioinformaticians can enhance diagnostic accuracy by correlating sequencing data with PET/CT localization of paravalvular abscesses or serological profiles. Recent innovations address noise challenges more fundamentally: Roche’s Sequencing by Expansion (SBX) technology improves signal-to-noise ratios via surrogate polymers (Xpandomers) that amplify nucleic acid signals [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>], while BGI’s CycloneSeq platform achieves 97% raw read accuracy through asymmetric chemistry and deep learning–based base calling [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. Nevertheless, standardized clinical reporting frameworks (e.g., ‘Definite/Probable/Contaminant’ classifications) remain essential to align results with therapeutic decisions.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Amplification Strategies and Error Control</h3>
<p id="Par32">Amplification protocols critically influence mNGS performance [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>], particularly in endocarditis diagnostics. Whole-genome (WGA) or whole-transcriptome amplification (WTA) is routinely employed in low-biomass samples like blood, enhancing sensitivity by amplifying microbial nucleic acids amidst overwhelming host DNA [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>, <a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>]. However, these methods risk introducing bias through preferential amplification of certain genomic regions and generating chimeric sequences that mimic novel pathogens or introduction of artifact sequence [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>–<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>]. Conversely, amplification-free protocols are preferred for high-biomass specimens (e.g., valve tissue), preserving microbial community structures but requiring deeper sequencing to detect low-abundance targets [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. For blood samples in CNE, WGA yield higher pathogen detection sensitivity result compared to amplification-free approaches, albeit with reduced specificity due to false positives from reagent contamination [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>]. Unique molecular identifiers (UMIs) serve distinct roles based on workflow design. In amplification-dependent protocols, UMIs correct PCR-induced errors by tagging individual molecules pre-amplification [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>], enabling bioinformatic consensus calling to eliminate artifactual variants [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>, <a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>]. Their utility diminishes in amplification-free workflows where sequencing errors dominate; here, UMIs offer minimal benefit as they cannot distinguish genuine nucleotide misincorporation during sequencing from true biological variants [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>, <a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>]. Platform-specific error profiles further impact clinical interpretation. Oxford Nanopore higher error rate (∼5–15%) [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>], primarily affects base-calling accuracy in homopolymer regions, limiting reliable antimicrobial resistance gene detection. Nevertheless, it suffices for genus/species-level pathogen of 94.% in IE cases [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>, <a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>], making it valuable for rapid intraoperative decisions when valve tissue is available [<a href="#CR95" class="usa-link" aria-describedby="CR95">95</a>]. In contrast, Illumina’s lower error rate supports high-confidence resistance profiling, but delays result by 12–24 h a critical trade-off in unstable patients [<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>, <a href="#CR97" class="usa-link" aria-describedby="CR97">97</a>]. However, this accuracy limitation precludes reliable detection of single-nucleotide polymorphisms required for antimicrobial resistance profiling [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>, <a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>], necessitating confirmatory testing when resistance concerns arise.</p>
<p id="Par33">Therefore, UMIs serve distinct purposes based on workflow design, in amplification-based protocols (most common), they primarily correct PCR errors/duplicates rather than sequencing errors, with studies showing 40–60% artifact reduction [<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a>]. While in amplification-free workflows, UMIs have limited utility for sequencing error correction (Oxford Nanopore R10.4: Q20+; Illumina NovaSeq: Q30+), but may aid in detecting very low-frequency variants [<a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>, <a href="#CR101" class="usa-link" aria-describedby="CR101">101</a>].</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Economic Evaluation and Implementation Equity of mNGS in IE Diagnostics</h2>
<section id="Sec11"><h3 class="pmc_sec_title">The Cost of Clarity: Economic and Logistical Barriers To mNGS Adoption</h3>
<p id="Par34">While mNGS may reduce ICU stays, its high upfront cost exacerbates healthcare disparities. Point-of-care sequencing (e.g., Oxford Nanopore) could democratize access but requires validation in low-resource settings. For all its promise, mNGS remains resource-intensive. Per-sample costs range from $500 to $1,000 depending on depth and platform [<a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>], not including data analysis and interpretation overhead. To mitigate these expenses, innovative cost-reduction strategies must be prioritized: reusable flow cells (e.g., Oxford Nanopore) and pooled sequencing runs could substantially lower per-test costs, targeting 40% reductions by 2027. Regional sequencing hubs funded by WHO-Gavi partnerships [<a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>] could further democratize access in resource-limited settings. Compared to traditional diagnostics, blood cultures costing ~$90 or PCR ~$100, mNGS demands a deeper initial investment (illustrated in Fig. <a href="#Fig3" class="usa-link">3</a>). But the calculus changes when viewed through a clinical lens. While early targeted therapy may reduce ICU stays in high-risk cohorts (e.g., prosthetic valve IE), cost-efficacy varies by setting, with negligible benefits observed in uncomplicated cases [<a href="#CR106" class="usa-link" aria-describedby="CR106">106</a>–<a href="#CR108" class="usa-link" aria-describedby="CR108">108</a>]. Recent study showed that early mNGS testing improved patient prognosis while reducing diagnostic delays, especially in high-risk populations [<a href="#CR109" class="usa-link" aria-describedby="CR109">109</a>]. Nevertheless, accessibility remains uneven. High-resource centers are leading the way, while lower-resource settings struggle with infrastructure and reimbursement challenges (Fig. <a href="#Fig2" class="usa-link">2</a>; Table <a href="#Tab4" class="usa-link">4</a>). This disparity risks creating a diagnostic divide in IE care, where some patients benefit from cutting-edge tools while others remain in a trial-and-error loop. Solutions are emerging, low-cost sequencing platforms, centralized analysis hubs, and mobile point-of-care devices, but scalability, regulatory clearance, and clinical validation are still works in progress (Table <a href="#Tab5" class="usa-link">5</a>) [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>, <a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>].</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373624_44197_2025_455_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc20/12373624/b47ca9e9c6cc/44197_2025_455_Fig3_HTML.jpg" loading="lazy" id="d33e988" height="343" width="685" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative cost distribution of mNGS and traditional diagnostic methods. Figure 3 <em>contrasts the cost allocation between mNGS and traditional diagnostic methods. mNGS incurs higher expenses in equipment and data analysis</em>,<em> while traditional methods have greater costs in reagents and labor. The breakdown emphasizes the distinct financial demands of each diagnostic approach</em></p></figcaption></figure><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373624_44197_2025_455_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc20/12373624/a8972e3dba47/44197_2025_455_Fig2_HTML.jpg" loading="lazy" id="d33e972" height="542" width="685" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Challenges in mNGS implementation for IE. Figure 2 <em>highlights the primary challenges in mNGS adoption: high costs</em>,<em> complex data interpretation</em>,<em> lack of standardized protocols</em>,<em> and data privacy concerns</em>,<em> along with potential solutions like economic analyses</em>,<em> bioinformatics tools</em>,<em> standardization</em>,<em> and enhanced security</em></p></figcaption></figure><section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Challenges and recommendations for mngs implementation</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Challenge</th>
<th align="left" colspan="1" rowspan="1">Description</th>
<th align="left" colspan="1" rowspan="1">Recommended Solution</th>
<th align="left" colspan="1" rowspan="1">Reference</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">High Cost</td>
<td align="left" colspan="1" rowspan="1">High initial costs of equipment and reagents</td>
<td align="left" colspan="1" rowspan="1">Development of low-cost sequencing platforms, funding initiatives</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Technical Complexity</td>
<td align="left" colspan="1" rowspan="1">Requires specialized training and infrastructure</td>
<td align="left" colspan="1" rowspan="1">Training programs, workshops, interdisciplinary collaboration</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Standardization</td>
<td align="left" colspan="1" rowspan="1">Lack of standardized protocols and guidelines</td>
<td align="left" colspan="1" rowspan="1">Development of consensus guidelines, quality assurance measures</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>]</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Summary of validation studies for mngs in diagnosing CNE</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Study</th>
<th align="left" colspan="1" rowspan="1">Sample Size</th>
<th align="left" colspan="1" rowspan="1">Sensitivity</th>
<th align="left" colspan="1" rowspan="1">Specificity</th>
<th align="left" colspan="1" rowspan="1">Main Findings</th>
<th align="left" colspan="1" rowspan="1">Reference</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Wu et al. (2024)</td>
<td align="center" colspan="1" rowspan="1">195</td>
<td align="left" colspan="1" rowspan="1">89.8%</td>
<td align="left" colspan="1" rowspan="1">76.3%</td>
<td align="left" colspan="1" rowspan="1">High diagnostic yield, reduced false negatives</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR110" class="usa-link" aria-describedby="CR110">110</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Chen et al. (2024)</td>
<td align="center" colspan="1" rowspan="1">2325</td>
<td align="left" colspan="1" rowspan="1">87%</td>
<td align="left" colspan="1" rowspan="1">59%</td>
<td align="left" colspan="1" rowspan="1">Comprehensive pathogen detection, improved clinical prognosis</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR109" class="usa-link" aria-describedby="CR109">109</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Diao et al. (2023)</td>
<td align="center" colspan="1" rowspan="1">69</td>
<td align="left" colspan="1" rowspan="1">95.18%</td>
<td align="left" colspan="1" rowspan="1">91.30%</td>
<td align="left" colspan="1" rowspan="1">Enhanced detection of rare pathogens, guided targeted therapy</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>]</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec12"><h3 class="pmc_sec_title">Standardization and Implementation: Bridging Technical and Clinical Gaps</h3>
<p id="Par37">Contamination risks remain highest in low-biomass samples (e.g., blood); rigorous negative controls are non-negotiable but inconsistently applied. The potential of mNGS in CNE diagnosis is tempered by significant variability across laboratories, where differences in sample handling, sequencing platforms, bioinformatic pipelines, and interpretive criteria yield inconsistent results from identical samples [<a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>, <a href="#CR112" class="usa-link" aria-describedby="CR112">112</a>]. Standardization spanning nucleic acid extraction to clinical reporting is essential not only for technical consistency but to foster <em>clinical trust</em> [<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>, <a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>]. Without harmonized protocols, even high-performing technologies risk irreproducibility in real-world practice (Fig. <a href="#Fig2" class="usa-link">2</a>). Adoption of unique molecular identifiers (UMIs), batch decontamination, and clinical-grade bioinformatics pipelines (e.g., VPipe [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]) is critical to reduce variability. Inter-laboratory proficiency testing, such as WHO-led initiatives [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>], should validate these protocols globally. Pre-analytical variability remains a critical vulnerability. Inconsistent collection, storage, or processing of blood, tissue, or embolic specimens directly impacts nucleic acid integrity and sequencing depth, leading to false negatives or misclassified pathogens [<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a>]. Analytical challenges further compound this: Illumina platforms offer accuracy but slower turnaround, while Oxford Nanopore enables rapid sequencing at the cost of higher error rates. Platform selection must align with clinical priorities speed for critically ill patients versus depth for polymicrobial infections. The greatest heterogeneity arises in bioinformatics. Custom pipelines vary in host-sequence filtering, pathogen-threshold definitions, and read classification, creating discordant results from the same sample. To mitigate this, adoption of unique molecular identifiers (UMIs), batch decontamination, and WHO-led proficiency testing could reduce variability. Tiered reporting frameworks (e.g., ‘Definite/Probable/Contaminant’) align results with clinical context and threshold-based pathogen scoring is paramount [<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>, <a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>]. Clinical-grade bioinformatics tools (e.g., VPipe [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]) and transparent alignment to curated databases reduce false positives and enhance interpretability. Critically, standardized clinical reporting frameworks must evolve alongside technical improvements. A raw pathogen list is inadequate; reports should prioritize actionable insights by: (i) Flagging contaminants and commensals, (ii) Highlighting high-confidence pathogens, (iii) Correlating findings with clinical context (e.g., imaging, serology), (iv) Implementing tiered classifications (e.g., ‘probable pathogen,’ ‘uncertain significance’) to guide therapeutic decisions [<a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>, <a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>]. Inter-laboratory proficiency testing (e.g., WHO-led initiatives [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>] and regulatory collaboration (FDA/EMA) are vital for validation. Until universal standards exist, clinicians should interpret mNGS within a multidisciplinary framework integrating microbiology, radiology, and patient evolution to navigate ambiguity.</p></section></section><section id="Sec13"><h2 class="pmc_sec_title">Trends and Gaps in Current Research</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Emerging Trends</h3>
<p id="Par38">The clinical integration of mNGS has gained remarkable traction due to its capability to detect a broad spectrum of pathogens with higher sensitivity than conventional methods (Fig. <a href="#Fig4" class="usa-link">4</a>; Table <a href="#Tab1" class="usa-link">1</a>). Studies have demonstrated its versatility across a variety of infectious diseases, including CNE [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>, <a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>]. These investigations report diagnostic yields ranging from 80% to over 90%, particularly in individuals with prior antibiotics treatment. This variability stems from pre-analytical factors (e.g., sample degradation during transport in low/middle-income countries (LMICs)) and analytical disparities (e.g., Illumina vs. Nanopore error rates), highlighting irreproducibility risks in real-world practice. However, this high sensitivity is not universal. Previous studies reported more modest yields (60–75%) in mixed cohorts, highlighting the influence of sample type, pathogen load, and sequencing depth [<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a>]. Such discrepancies reveal the necessity for further refined protocols that account for contextual variables. In terms of bioinformatics, significant progress has been made with machine learning-driven pipelines that improve signal-to-noise ratios and pathogen classification. One study emphasized these advances but also cautioned against overreliance on unvalidated algorithms, particularly in low-biomass samples prone to contamination [<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>]. Furthermore, another advocated for AI-assisted interpretation tools that integrate clinical metadata to improve specificity, critics argue that these systems are only as robust as the datasets they are trained on, which are often limited in size and diversity [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. As such, while AI shows promise, it should currently serve as a decision-support mechanism rather than a replacement for expert review.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373624_44197_2025_455_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc20/12373624/8f25be9b1c8a/44197_2025_455_Fig4_HTML.jpg" loading="lazy" id="d33e1891" height="462" width="685" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Integration of mNGS into clinical practice. Figure 4 <em>depicts the integration of mNGS in diagnosing CNE. It outlines the process from sample collection</em>,<em> library preparation</em>,<em> and sequencing</em>,<em> through data analysis and result interpretation</em>,<em> to reporting and clinical decision-making</em>,<em> emphasizing the roles of mNGS in identifying difficult-to-detect pathogens and guiding precise treatment</em></p></figcaption></figure></section><section id="Sec15"><h3 class="pmc_sec_title">Multimodal Diagnostic Integration and Clinical Decision Pathway</h3>
<p id="Par39">The integration of mNGS with advanced diagnostic modalities particularly advanced imaging and serological biomarkers can significantly enhances diagnostic precision in CNE (Table <a href="#Tab6" class="usa-link">6</a>). For prosthetic valve endocarditis (PVE), combining mNGS with ¹⁸F-FDG PET/CT could improve infectious focus localization in cases with ambiguous clinical signs [<a href="#CR113" class="usa-link" aria-describedby="CR113">113</a>, <a href="#CR114" class="usa-link" aria-describedby="CR114">114</a>]. This synergy could arises from PET/CT capacity to identify metabolically active inflammation (SUVmax &gt; 4.1) and can guide targeted mNGS sampling of resected valve tissue, thereby reducing false positives from environmental contaminants [<a href="#CR113" class="usa-link" aria-describedby="CR113">113</a>, <a href="#CR115" class="usa-link" aria-describedby="CR115">115</a>]. Conversely, in seropositive <em>Bartonella</em> IE (IgG titers &gt; 1:1,024) [<a href="#CR116" class="usa-link" aria-describedby="CR116">116</a>, <a href="#CR117" class="usa-link" aria-describedby="CR117">117</a>], mNGS could add marginal diagnostic value but remains critical for detecting co-infections or resistance markers missed by serology alone [<a href="#CR118" class="usa-link" aria-describedby="CR118">118</a>]. Emerging molecular imaging probes might further augment mNGS utility. Antimicrobial peptide-based radiotracers (e.g., ⁹⁹ᵐTc-UBI) selectively bind bacterial cell walls, which might enable real-time visualization of <em>Staphylococcus aureus</em> biofilms on prosthetic valves [<a href="#CR113" class="usa-link" aria-describedby="CR113">113</a>]. When correlated with mNGS-derived pathogen data, these probes can differentiate active infection from sterile inflammation a longstanding challenge in IE management [<a href="#CR113" class="usa-link" aria-describedby="CR113">113</a>, <a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>]. Similarly, host-response biomarkers like procalcitonin and IL-6 might refine mNGS interpretation: elevated levels (&gt; 2 ng/mL and &gt; 40 pg/mL, respectively) strengthen the clinical relevance of low-abundance mNGS reads by confirming systemic inflammation [<a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>, <a href="#CR121" class="usa-link" aria-describedby="CR121">121</a>]. However, multimodal integration faces interoperability barriers. Current electronic health records rarely consolidate genomic, imaging, and biomarker data into unified dashboards, delaying time-sensitive therapeutic decisions. This might be address by embedding mNGS within a multi-omics framework that can tracks pathogen dynamics alongside host immune responses in IE patients, aiming to establish causality between specific microbial profiles and thrombotic complications. Preliminary data reveal that <em>Streptococcus sanguinis</em> detected by mNGS correlates with colony vegetations on CT enterography, prompting colonoscopy that identified occult malignancies in few cases [<a href="#CR115" class="usa-link" aria-describedby="CR115">115</a>]. Diagnostic workflows must be context-sensitive to maximize cost-efficacy. For high-probability IE (e.g., febrile injection drug users with tricuspid vegetations), echocardiography plus blood culture suffices, reserving mNGS for treatment failures [<a href="#CR118" class="usa-link" aria-describedby="CR118">118</a>]. Contrastingly, indeterminate cases (e.g., culture-negative PVE with nonspecific CRP elevation) warrant concurrent mNGS and PET/CT as first-line tools [<a href="#CR113" class="usa-link" aria-describedby="CR113">113</a>, <a href="#CR114" class="usa-link" aria-describedby="CR114">114</a>]. This stratification could optimizes resources use: Biofire BJP panels automated PCR assays detecting 31 pathogens and 10 resistance genes in 1 h provide rapid screening, while 16 S/18S rDNA PCR covers rare pathogens like <em>Lactococcus garvieae</em> albeit with longer turnaround (100 h) [<a href="#CR118" class="usa-link" aria-describedby="CR118">118</a>]. Validated frameworks (e.g., ESC 2023 guidelines [<a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>]) now could recommend tiered algorithms where BJP-negative samples trigger reflex mNGS testing [<a href="#CR114" class="usa-link" aria-describedby="CR114">114</a>, <a href="#CR118" class="usa-link" aria-describedby="CR118">118</a>]. However, multimodal integration faces substantial interoperability challenges. Harmonizing mNGS data with electronic health records remains nascent, and cross-disciplinary workflows (e.g., correlating sequencing results with real-time echocardiographic findings) require specialized platforms unavailable in resource-limited settings [<a href="#CR114" class="usa-link" aria-describedby="CR114">114</a>, <a href="#CR123" class="usa-link" aria-describedby="CR123">123</a>]. The 2025 Swiss IE cohort study further revealed socioeconomic disparities: high-income centers utilized integrated diagnostics in &gt; 85% of complex cases versus &lt; 15% in regions lacking PET/CT infrastructure [<a href="#CR124" class="usa-link" aria-describedby="CR124">124</a>]. Future efforts must prioritize unified data architectures that embed mNGS within endocarditis team workflows, enabling dynamic reinterpretation of results alongside advancing clinical parameters. For comprehensive detail, see Zeng et al.’s (2023) work for synchronizing blood mNGS with surgical findings, the CIVA consortium guidelines for imaging-guided tissue sampling [<a href="#CR115" class="usa-link" aria-describedby="CR115">115</a>, <a href="#CR123" class="usa-link" aria-describedby="CR123">123</a>], and the FISHseq protocol for prosthetic valve biofilm characterization [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>].</p>
<section class="tw xbox font-sm" id="Tab6"><h4 class="obj_head">Table 6.</h4>
<div class="caption p"><p>Proposed IE diagnosis and treatment strategies for future care</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Proposed Strategy</th>
<th align="left" colspan="1" rowspan="1">Diagnostic/Treatment Innovation</th>
<th align="left" colspan="1" rowspan="1">Potential Benefits</th>
<th align="left" colspan="1" rowspan="1">Implementation Considerations</th>
<th align="left" colspan="1" rowspan="1">References</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Integrating AI and Machine Learning</td>
<td align="left" colspan="1" rowspan="1">AI-driven algorithms to interpret mNGS data and imaging results</td>
<td align="left" colspan="1" rowspan="1">Increased diagnostic accuracy, reduced human error, faster data processing</td>
<td align="left" colspan="1" rowspan="1">Requires extensive training datasets, regulatory approval, and cross-disciplinary expertise</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>, <a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Personalized Medicine Approach</td>
<td align="left" colspan="1" rowspan="1">Utilizing genomics and mNGS data to tailor antibiotic therapy</td>
<td align="left" colspan="1" rowspan="1">Improved treatment efficacy, reduced side effects, optimized antibiotic use</td>
<td align="left" colspan="1" rowspan="1">High costs, need for robust bioinformatics infrastructure, patient consent and data privacy concerns</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>, <a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Point-of-Care mNGS Devices</td>
<td align="left" colspan="1" rowspan="1">Development of portable, rapid mNGS devices for bedside use</td>
<td align="left" colspan="1" rowspan="1">Immediate pathogen detection, faster clinical decision-making, improved patient outcomes</td>
<td align="left" colspan="1" rowspan="1">Developmental and validation challenges, integration into clinical workflows, cost considerations</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Telemedicine and Remote Monitoring</td>
<td align="left" colspan="1" rowspan="1">Remote monitoring of patients using digital health tools and mNGS results</td>
<td align="left" colspan="1" rowspan="1">Enhanced patient follow-up, early detection of complications, reduced hospital readmissions</td>
<td align="left" colspan="1" rowspan="1">Data security, infrastructure requirements, patient and provider training</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Advanced Bioinformatics Platforms</td>
<td align="left" colspan="1" rowspan="1">Enhanced platforms for real-time data analysis and pathogen identification</td>
<td align="left" colspan="1" rowspan="1">Greater diagnostic precision, streamlined workflow, integration with electronic health records (EHRs)</td>
<td align="left" colspan="1" rowspan="1">Requires significant investment, interoperability with existing systems, continuous updates</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Next-Generation Antibiotics</td>
<td align="left" colspan="1" rowspan="1">Development of new antibiotics targeting resistant pathogens identified by mNGS</td>
<td align="left" colspan="1" rowspan="1">Effective treatment of antibiotic-resistant infections, reduced mortality and morbidity</td>
<td align="left" colspan="1" rowspan="1">Long drug development timelines, high costs, regulatory hurdles</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Microbiome-Based Diagnostics</td>
<td align="left" colspan="1" rowspan="1">Leveraging microbiome profiling to understand pathogen interactions and diagnose IE</td>
<td align="left" colspan="1" rowspan="1">Insights into pathogen communities, potential for targeted therapies, reduced false positives</td>
<td align="left" colspan="1" rowspan="1">Complexity of microbiome data, need for advanced analytical methods, clinical validation</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>, <a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>]</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec16"><h3 class="pmc_sec_title">Advancing Methodological Innovations and Clinical Validation</h3>
<p id="Par41">The integration of artificial intelligence (AI) with mNGS could advances the diagnostic precision for CNE (Table <a href="#Tab6" class="usa-link">6</a>) [<a href="#CR125" class="usa-link" aria-describedby="CR125">125</a>]. Recent multimodal AI frameworks, such as the MAARS model for arrhythmic death risk stratification in hypertrophic cardiomyopathy, demonstrate how combining electronic health records, imaging data, and genomic insights can achieve AUCs of 0.89–0.81 significantly outperforming traditional clinical guidelines (AUC 0.22–0.35) [<a href="#CR126" class="usa-link" aria-describedby="CR126">126</a>]. Similarly, the EchoNext system leverages deep learning to detect structural heart disease (e.g., low LVEF, valve pathology) from routine electrocardiograms with 85.2% AUROC, enabling targeted use of echocardiography in high-risk cohorts [<a href="#CR127" class="usa-link" aria-describedby="CR127">127</a>]. These advances highlight AI capacity to extract latent diagnostic signals from conventional tests, though their applicability to mNGS-based pathogen detection requires further validation summarized in Table <a href="#Tab5" class="usa-link">5</a> [<a href="#CR128" class="usa-link" aria-describedby="CR128">128</a>, <a href="#CR129" class="usa-link" aria-describedby="CR129">129</a>]. Hybrid AI architectures now merge convolutional neural networks (CNNs) for spatial pattern recognition with large language models (LLMs) that interpret unstructured clinical notes [<a href="#CR130" class="usa-link" aria-describedby="CR130">130</a>, <a href="#CR131" class="usa-link" aria-describedby="CR131">131</a>]. Such frameworks achieve 95.1% accuracy in heart failure prediction by synthesizing demographic, electrocardiographic, and textual data [<a href="#CR132" class="usa-link" aria-describedby="CR132">132</a>]. For mNGS, analogous models could contextualize sequencing results with patient history and imaging findings, distinguishing pathogens from contaminants more effectively. For instance, lipid-rich necrotic cores or positive remodeling on coronary CT angiography features associated with vulnerable plaques may correlate with specific microbial profiles in IE [<a href="#CR133" class="usa-link" aria-describedby="CR133">133</a>]. However, AI performance remains heterogeneous across populations. While EchoNext maintains AUROC &gt; 78% across diverse healthcare settings [<a href="#CR127" class="usa-link" aria-describedby="CR127">127</a>], models like CardioMind (China’s first cardiovascular AI) exhibit geographic bias, with training data skewed toward Eastern cohorts. This underscores the need for globally representative datasets to ensure equitable generalizability. Critical implementation barriers persist. First, explainability: Even high-performing ‘black-box’ models like MAARS require Shapley Additive exPlanations (SHAP) to clarify decision pathways and foster clinician trust [<a href="#CR132" class="usa-link" aria-describedby="CR132">132</a>]. Second, data quality: AI-mNGS integration depends on standardized preprocessing; variable nucleic acid extraction methods alter pathogen recovery by &gt; 25%, confounding algorithmic predictions [<a href="#CR133" class="usa-link" aria-describedby="CR133">133</a>]. Third, clinical workflow integration: While EchoNext reduced unnecessary echocardiograms by 41% in prospective trials [<a href="#CR127" class="usa-link" aria-describedby="CR127">127</a>], no studies yet demonstrate AI-guided mNGS improving IE outcomes in real-world settings. Future efforts must prioritize prospective validation in multicenter cohorts, emphasizing hard endpoints like mortality or embolic events rather than analytical performance alone. For methodological advancements, see the hybrid CNN-LLM framework by Pinna et al. (2025) and plaque characterization AI in coronary imaging by Alshraideh et al. (2025) [<a href="#CR132" class="usa-link" aria-describedby="CR132">132</a>, <a href="#CR133" class="usa-link" aria-describedby="CR133">133</a>].</p></section><section id="Sec17"><h3 class="pmc_sec_title">Identified Research Gaps</h3>
<p id="Par42">Despite promising data, several critical gaps persist. First, the absence of large, multicenter validation studies undermines generalizability (Table <a href="#Tab5" class="usa-link">5</a>). While one study demonstrated robust results in a high-volume center [<a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>], another report highlighted that, smaller hospitals with limited bioinformatics support struggled to replicate those findings [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. This indicates that mNGS performance is not solely a function of the technology, but also of institutional capability. Second, there is a marked lack of consensus on interpretive thresholds. The mNGS sensitivity (Table <a href="#Tab1" class="usa-link">1</a>), can yield ambiguous results, particularly in distinguishing pathogens from contaminants or commensals. For example, a study reported several false positives due to inadequate filtering [<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a>], while a study implemented strict confidence scoring to mitigate this issue [<a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>]. These divergent approaches complicate inter-study comparison and point to the critical needs for standardized reporting criteria. Cost-effectiveness remains another unresolved issue (Table <a href="#Tab7" class="usa-link">7</a>). While one study reported that early mNGS testing reduced ICU stay and antibiotic use [<a href="#CR109" class="usa-link" aria-describedby="CR109">109</a>], a contrasting analysis by another study found no significant difference in overall treatment costs when controlling for illness severity [<a href="#CR134" class="usa-link" aria-describedby="CR134">134</a>]. mNGS offers higher diagnostic yields than PCR and blood cultures but at substantially greater upfront cost summarized in Table <a href="#Tab7" class="usa-link">7</a>. More comprehensive economic modeling is needed, accounting for diagnostic performance and long-term outcomes such as recurrence rates, mortality, and antimicrobial resistance. Finally, long-term outcome data are sparse. While several studies show immediate benefits in terms of diagnosis and therapeutic adjustment, few extend follow-up beyond hospital discharge. Additionally, &gt; 80% of cited studies originate from high-income regions, potentially overestimating mNGS efficacy in LMICs where cold-chain logistics and electrical instability degrade sample integrity. Without longitudinal data, it remains unclear whether mNGS-guided care improves survival, reduces recurrence, or enhances quality of life. Future research should prioritize patient-centered endpoints to truly define clinical utility. These trends and contrasting findings underscore a field in evolution where mNGS has established its potential but must now earn its place in standardized clinical care. The following section explores how innovation, regulation, and strategic investment must converge to unlock mNGS full promise as a globally accessible diagnostic standard.</p>
<section class="tw xbox font-sm" id="Tab7"><h4 class="obj_head">Table 7.</h4>
<div class="caption p"><p>Cost-benefit analysis of diagnostic methods</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Diagnostic Method</th>
<th align="left" colspan="1" rowspan="1">Cost per Sample ($)</th>
<th align="left" colspan="1" rowspan="1">Time to Result</th>
<th align="left" colspan="1" rowspan="1">Clinical Accuracy</th>
<th align="left" colspan="1" rowspan="1">Cost-Effectiveness</th>
<th align="left" colspan="1" rowspan="1">Reference</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Traditional Blood Culture</td>
<td align="left" colspan="1" rowspan="1">84–97</td>
<td align="left" colspan="1" rowspan="1">Days to Weeks</td>
<td align="left" colspan="1" rowspan="1">Moderate</td>
<td align="left" colspan="1" rowspan="1">Low</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR135" class="usa-link" aria-describedby="CR135">135</a>, <a href="#CR136" class="usa-link" aria-describedby="CR136">136</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PCR</td>
<td align="left" colspan="1" rowspan="1">50–200</td>
<td align="left" colspan="1" rowspan="1">Hours to Days</td>
<td align="left" colspan="1" rowspan="1">High</td>
<td align="left" colspan="1" rowspan="1">Moderate</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR137" class="usa-link" aria-describedby="CR137">137</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mNGS</td>
<td align="left" colspan="1" rowspan="1">1000–2500</td>
<td align="left" colspan="1" rowspan="1">24–48 Hours</td>
<td align="left" colspan="1" rowspan="1">Very High</td>
<td align="left" colspan="1" rowspan="1">High (long-term)</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR106" class="usa-link" aria-describedby="CR106">106</a>]</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section></section><section id="Sec18"><h2 class="pmc_sec_title">Future Directions and Recommendations</h2>
<p id="Par44">The evolution of mNGS for CNE demands urgent and coordinated action, three key technological innovations must be prioritized. Emerging hybrid architectures theoretically enable rapid, high-accuracy sequencing, though clinical validation remains pending [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>, <a href="#CR138" class="usa-link" aria-describedby="CR138">138</a>]. Concurrently, integrated host-pathogen cartridges combining mNGS with immune biomarkers (e.g., IL-6, procalcitonin) [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>, <a href="#CR139" class="usa-link" aria-describedby="CR139">139</a>] using microfluidic technology would standardize contamination-prone extraction steps while providing immunological context to differentiate pathogens from commensals. Finally, WHO-coordinated open-source AI platforms sharing globally validated algorithms must be deployed, with mandatory bias audits ensuring &gt; 30% representation of LMIC populations in training datasets to prevent algorithmic disparities [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>, <a href="#CR140" class="usa-link" aria-describedby="CR140">140</a>, <a href="#CR141" class="usa-link" aria-describedby="CR141">141</a>]. For equitable implementation, health systems must address cost barriers challenges (Table <a href="#Tab4" class="usa-link">4</a>) through manufacturer-subsidized flow-cell reuse and pooled sequencing runs targeting 40% cost reduction by 2027, complemented by WHO/Gavi-funded regional sequencing hubs to serve LMIC networks [<a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>]. Reimbursement models should directly link mNGS coverage to documented clinical benefits: insurers could require proof of ≥ 48-hour reductions in broad-spectrum antibiotic use or prevention of ≥ 1 embolic event through early pathogen-directed therapy [<a href="#CR142" class="usa-link" aria-describedby="CR142">142</a>, <a href="#CR143" class="usa-link" aria-describedby="CR143">143</a>]. Critical research gaps demand immediate attention. Prospective trials comparing mNGS-guided versus conventional CNE management are urgently needed [<a href="#CR109" class="usa-link" aria-describedby="CR109">109</a>, <a href="#CR134" class="usa-link" aria-describedby="CR134">134</a>]; an international registry tracking 30-day mortality/recurrence across 5,000 + patients would provide definitive evidence. Multicenter studies must also validate pathogen-specific host-response thresholds (e.g., <em>Bartonella</em>-induced IFN-γ cutoffs) to resolve diagnostic ambiguity [<a href="#CR144" class="usa-link" aria-describedby="CR144">144</a>, <a href="#CR145" class="usa-link" aria-describedby="CR145">145</a>], while simplified workflows using freeze-dried reagents and smartphone-based bioinformatics require validation in LMIC field trials to enable low-infrastructure deployment [<a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>, <a href="#CR146" class="usa-link" aria-describedby="CR146">146</a>]. A tripartite call to action is essential, clinicians should establish multidisciplinary rapid-response teams implementing tiered reporting frameworks aligned with modified Duke criteria [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR147" class="usa-link" aria-describedby="CR147">147</a>]. Regulators must fast-track accreditation for integrated assays by 2026 and mandate biannual proficiency testing using CNE reference panels [<a href="#CR148" class="usa-link" aria-describedby="CR148">148</a>]. Policymakers ought to fund mNGS inclusion in WHO Essential Diagnostics Lists and tie hospital reimbursements to diagnostic yield audits (&gt; 85% pathogen identification in CNE). Without these measures, mNGS risks exacerbating global disparities where early pathogen-directed therapy becomes standard in high-income settings while empirical delays persist in regions bearing the highest CNE burden. The 18–25% mortality of untreated CNE [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>] demands we bridge this gap urgently. By uniting innovation with equity, we can transform mNGS from a privileged technology into a cornerstone of precision global health.</p></section><section id="Sec19"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par46">mNGS significantly advances pathogen detection in CNE, identifying causative organisms in &gt; 90% of cases where conventional diagnostics fail, However, it faces cost barriers, standardization gaps, and interpretative challenges that require resolution before routine adoption (Table <a href="#Tab4" class="usa-link">4</a>). The mNGS future in CNE diagnosis depends on its transformation from a high-potential technology into a robust, accessible, and regulated diagnostic system. Key constraints include false positives in low-biomass samples, inter-lab variability, and exclusion of 37% of eligible studies from low-resource settings in analysis highlighting geographic evidence gaps. As the field advances, success will be measured not by sequencing speed or depth alone, but by meaningful improvements in patient care, diagnostic equity, and system-level efficiency. Achieving this vision requires a concerted effort across science, policy, and practice, linking innovation with infrastructure, evidence with education, and discovery with delivery. Additionally, the high costs and technical complexity of mNGS, coupled with the limitation in standardization, hinder widespread adoption (Table <a href="#Tab4" class="usa-link">4</a>). Overcoming these barriers will require ongoing research, greater interdisciplinary collaboration, and innovative strategies such as the incorporation of artificial intelligence for data analysis, development of portable mNGS devices for point-of-care use, and the integration of multi-omics approaches. Its clinical adoption requires cost reduction, interoperability with electronic health records, and guidelines defining actionable results. These advancements, along with improved bioinformatics tools and telemedicine for continuous monitoring, hold promise to enhance IE diagnostics and treatment, ultimately enhancing patient care in culture-negative cases.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We gratefully acknowledge the financial support from Sun Yat-sen University Eighth Hospital grant.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>mNGS</dt>
<dd><p id="Par2">Metagenomic Next-Generation Sequencing</p></dd>
<dt>CNE</dt>
<dd><p id="Par3">Culture-Negative Endocarditis</p></dd>
<dt>IE</dt>
<dd><p id="Par4">Infective Endocarditis</p></dd>
<dt>PCR</dt>
<dd><p id="Par5">Polymerase Chain Reaction</p></dd>
<dt>16S rRNA</dt>
<dd><p id="Par6">16 S Ribosomal Ribonucleic Acid</p></dd>
<dt>AI</dt>
<dd><p id="Par7">Artificial Intelligence</p></dd>
<dt>QUADAS-2</dt>
<dd><p id="Par8">Quality Assessment of Diagnostic Accuracy Studies-2</p></dd>
<dt>ESC</dt>
<dd><p id="Par9">European Society of Cardiology</p></dd>
<dt>AHA</dt>
<dd><p id="Par10">American Heart Association</p></dd>
<dt>ICU</dt>
<dd><p id="Par11">Intensive Care Unit</p></dd>
<dt>PET/CT</dt>
<dd><p id="Par12">Positron Emission Tomography/Computed Tomography</p></dd>
<dt>DNA</dt>
<dd><p id="Par13">Deoxyribonucleic Acid</p></dd>
<dt>RNA</dt>
<dd><p id="Par14">Ribonucleic Acid</p></dd>
<dt>WHO</dt>
<dd><p id="Par15">World Health Organization</p></dd>
<dt>FDA</dt>
<dd><p id="Par16">U.S. Food and Drug Administration</p></dd>
<dt>EMA</dt>
<dd><p id="Par17">European Medicines Agency</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>SAUS, BZ; Conceptualization, investigation, methodology, resources, validation, writing – original draft, writing – review &amp; editing, YQ &amp; BXZ: Investigation, methodology, resources, HMA&amp; YTVM: literature search validation, software and visualization, GBF &amp; KZ: Project administration, supervision, validation, visualization, writing – review &amp; editing, LZ &amp; MG: Investigation, methodology, resources, supervision and validation, YY: Conceptualization, funding acquisition, project administration, supervision, validation, writing – review &amp; editing. All authors have read and agreed to the submission of the manuscript to Journal of Epidemiology and Global Health.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by Sun Yat-sen University Eighth Hospital Cohort Construction Project with the No. (GCCRCYJ065).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data Availability</h2>
<p>No datasets were generated or analysed during the current study.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics Approval and Consent to Participate</h3>
<p id="Par47">This study does not involve interaction with animal or human subjects or the collection of personal information, no ethics committee/IRB approval was required for this research, in accordance with the Helsinki Declaration.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for Publication</h3>
<p id="Par48">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing Interests</h3>
<p id="Par49">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Shafiu A. Umar Shinge and Binbin Zhang contributed equally to this work.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Kuan Zeng, Email: zengkuan3@mail.sysu.edu.cn.</p>
<p>Yanqi Yang, Email: yanqiyang_lio@yahoo.se.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Baddour LM, Wilson WR, Bayer AS, Fowler VG Jr., Tleyjeh IM, Rybak MJ, et al. Infective endocarditis in adults: diagnosis, antimicrobial therapy, and management of complications: A scientific statement for healthcare professionals from the American heart association. Circulation. 2015;132(15):1435–86. 10.1161/cir.0000000000000296.
</cite> [<a href="https://doi.org/10.1161/CIR.0000000000000296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26373316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baddour%20LM,%20Wilson%20WR,%20Bayer%20AS,%20Fowler%20VG%20Jr.,%20Tleyjeh%20IM,%20Rybak%20MJ,%20et%20al.%20Infective%20endocarditis%20in%20adults:%20diagnosis,%20antimicrobial%20therapy,%20and%20management%20of%20complications:%20A%20scientific%20statement%20for%20healthcare%20professionals%20from%20the%20American%20heart%20association.%20Circulation.%202015;132(15):1435%E2%80%9386.%2010.1161/cir.0000000000000296." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Morawiec R, Matuszewska-Brycht O, Maeser P, Kosny M, Krejca M, Drozdz J. Are in-hospital mortality predictors based on complete blood count useful in patients with infective endocarditis? A retrospective tertiary center perspective (2015–2022). Pol Arch Intern Med. 2023;133(12). 10.20452/pamw.16617.</cite> [<a href="https://doi.org/10.20452/pamw.16617" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38051080/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Cahill TJ, Prendergast BD. Infective endocarditis. Lancet. 2016;387(10021):882–93. 10.1016/S0140-6736(15)00067-7.
</cite> [<a href="https://doi.org/10.1016/S0140-6736(15)00067-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26341945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cahill%20TJ,%20Prendergast%20BD.%20Infective%20endocarditis.%20Lancet.%202016;387(10021):882%E2%80%9393.%2010.1016/S0140-6736(15)00067-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Habib G, Lancellotti P, Antunes MJ, Bongiorni MG, Casalta JP, Del Zotti F, et al. 2015 ESC guidelines for the management of infective endocarditis: the task force for the management of infective endocarditis of the European society of cardiology (ESC). Endorsed by: European association for Cardio-Thoracic surgery (EACTS), the European association of nuclear medicine (EANM). Eur Heart J. 2015;36(44):3075–128. 10.1093/eurheartj/ehv319.
</cite> [<a href="https://doi.org/10.1093/eurheartj/ehv319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26320109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Habib%20G,%20Lancellotti%20P,%20Antunes%20MJ,%20Bongiorni%20MG,%20Casalta%20JP,%20Del%20Zotti%20F,%20et%20al.%202015%20ESC%20guidelines%20for%20the%20management%20of%20infective%20endocarditis:%20the%20task%20force%20for%20the%20management%20of%20infective%20endocarditis%20of%20the%20European%20society%20of%20cardiology%20(ESC).%20Endorsed%20by:%20European%20association%20for%20Cardio-Thoracic%20surgery%20(EACTS),%20the%20European%20association%20of%20nuclear%20medicine%20(EANM).%20Eur%20Heart%20J.%202015;36(44):3075%E2%80%93128.%2010.1093/eurheartj/ehv319." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Church DL, Cerutti L, Gürtler A, Griener T, Zelazny A, Emler S. Performance and application of 16S rRNA gene cycle sequencing for routine identification of bacteria in the clinical microbiology laboratory. Clin Microbiol Rev. 2020;33(4). 10.1128/cmr.00053-19.</cite> [<a href="https://doi.org/10.1128/CMR.00053-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7484979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32907806/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Alcolea-Medina A, Alder C, Snell LB, Charalampous T, Aydin A, Nebbia G, et al. Unified metagenomic method for rapid detection of microorganisms in clinical samples. Commun Med. 2024;4(1):135. 10.1038/s43856-024-00554-3.
</cite> [<a href="https://doi.org/10.1038/s43856-024-00554-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11228040/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38972920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alcolea-Medina%20A,%20Alder%20C,%20Snell%20LB,%20Charalampous%20T,%20Aydin%20A,%20Nebbia%20G,%20et%20al.%20Unified%20metagenomic%20method%20for%20rapid%20detection%20of%20microorganisms%20in%20clinical%20samples.%20Commun%20Med.%202024;4(1):135.%2010.1038/s43856-024-00554-3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Liu J, Zhang Q, Dong Y-Q, Yin J, Qiu Y-Q. Diagnostic accuracy of metagenomic next-generation sequencing in diagnosing infectious diseases: a meta-analysis. Sci Rep. 2022;12(1):21032. 10.1038/s41598-022-25314-y.
</cite> [<a href="https://doi.org/10.1038/s41598-022-25314-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9723114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36470909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20J,%20Zhang%20Q,%20Dong%20Y-Q,%20Yin%20J,%20Qiu%20Y-Q.%20Diagnostic%20accuracy%20of%20metagenomic%20next-generation%20sequencing%20in%20diagnosing%20infectious%20diseases:%20a%20meta-analysis.%20Sci%20Rep.%202022;12(1):21032.%2010.1038/s41598-022-25314-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Dahler R, Brugger SD, Frank M, Greutmann M, Sromicki J, Marques-Maggio E, et al. A retrospective analysis of blood culture-negative endocarditis at a tertiary care centre in Switzerland. Swiss Med Wkly. 2022;152:40012. 10.57187/smw.2022.40016.
</cite> [<a href="https://doi.org/10.57187/smw.2022.40016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36534966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dahler%20R,%20Brugger%20SD,%20Frank%20M,%20Greutmann%20M,%20Sromicki%20J,%20Marques-Maggio%20E,%20et%20al.%20A%20retrospective%20analysis%20of%20blood%20culture-negative%20endocarditis%20at%20a%20tertiary%20care%20centre%20in%20Switzerland.%20Swiss%20Med%20Wkly.%202022;152:40012.%2010.57187/smw.2022.40016." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Lamas CC, Eykyn SJ. Blood culture negative endocarditis: analysis of 63 cases presenting over 25 years. Heart. 2003;89(3):258–62. 10.1136/heart.89.3.258.
</cite> [<a href="https://doi.org/10.1136/heart.89.3.258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1767579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12591823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lamas%20CC,%20Eykyn%20SJ.%20Blood%20culture%20negative%20endocarditis:%20analysis%20of%2063%20cases%20presenting%20over%2025%20years.%20Heart.%202003;89(3):258%E2%80%9362.%2010.1136/heart.89.3.258." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Raoult D, Casalta JP, Richet H, Khan M, Bernit E, Rovery C, et al. Contribution of systematic serological testing in diagnosis of infective endocarditis. J Clin Microbiol. 2005;43(10):5238–42. 10.1128/jcm.43.10.5238-5242.2005.
</cite> [<a href="https://doi.org/10.1128/JCM.43.10.5238-5242.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1248503/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16207989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raoult%20D,%20Casalta%20JP,%20Richet%20H,%20Khan%20M,%20Bernit%20E,%20Rovery%20C,%20et%20al.%20Contribution%20of%20systematic%20serological%20testing%20in%20diagnosis%20of%20infective%20endocarditis.%20J%20Clin%20Microbiol.%202005;43(10):5238%E2%80%9342.%2010.1128/jcm.43.10.5238-5242.2005." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Liesman RM, Pritt BS, Maleszewski JJ, Patel R. Laboratory diagnosis of infective endocarditis. J Clin Microbiol. 2017;55(9):2599–608. 10.1128/JCM.00635-17.
</cite> [<a href="https://doi.org/10.1128/JCM.00635-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5648697/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28659319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liesman%20RM,%20Pritt%20BS,%20Maleszewski%20JJ,%20Patel%20R.%20Laboratory%20diagnosis%20of%20infective%20endocarditis.%20J%20Clin%20Microbiol.%202017;55(9):2599%E2%80%93608.%2010.1128/JCM.00635-17." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Clinical Guideline for the Diagnosis and Treatment of Cardiovascular Infections. Infect Chemother. 2011;43(2):129–77. 10.3947/ic.2011.43.2.129.</cite> [<a href="https://scholar.google.com/scholar_lookup?Clinical%20Guideline%20for%20the%20Diagnosis%20and%20Treatment%20of%20Cardiovascular%20Infections.%20Infect%20Chemother.%202011;43(2):129%E2%80%9377.%2010.3947/ic.2011.43.2.129." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Kaemmerer A-S, Ciotola F, Geißdörfer W, Harig F, Mattner J, Seitz T, et al. A Dual-Pathogen mitral valve endocarditis caused by Coxiella burnetii and Streptococcus gordonii—Which. Came First? Pathogens. 2023;12(9):1130.
</cite> [<a href="https://doi.org/10.3390/pathogens12091130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10537458/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37764938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaemmerer%20A-S,%20Ciotola%20F,%20Gei%C3%9Fd%C3%B6rfer%20W,%20Harig%20F,%20Mattner%20J,%20Seitz%20T,%20et%20al.%20A%20Dual-Pathogen%20mitral%20valve%20endocarditis%20caused%20by%20Coxiella%20burnetii%20and%20Streptococcus%20gordonii%E2%80%94Which.%20Came%20First?%20Pathogens.%202023;12(9):1130." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Eldin C, Mélenotte C, Mediannikov O, Ghigo E, Million M, Edouard S, et al. From Q fever to Coxiella burnetii infection: a paradigm change. Clin Microbiol Rev. 2017;30(1):115–90. 10.1128/cmr.00045-16.
</cite> [<a href="https://doi.org/10.1128/CMR.00045-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5217791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27856520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eldin%20C,%20M%C3%A9lenotte%20C,%20Mediannikov%20O,%20Ghigo%20E,%20Million%20M,%20Edouard%20S,%20et%20al.%20From%20Q%20fever%20to%20Coxiella%20burnetii%20infection:%20a%20paradigm%20change.%20Clin%20Microbiol%20Rev.%202017;30(1):115%E2%80%9390.%2010.1128/cmr.00045-16." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Bayer AS, Bolger AF, Taubert KA, Wilson W, Steckelberg J, Karchmer AW, et al. Diagnosis and management of infective endocarditis and its complications. Circulation. 1998;98(25):2936–48. 10.1161/01.CIR.98.25.2936.
</cite> [<a href="https://doi.org/10.1161/01.cir.98.25.2936" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9860802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bayer%20AS,%20Bolger%20AF,%20Taubert%20KA,%20Wilson%20W,%20Steckelberg%20J,%20Karchmer%20AW,%20et%20al.%20Diagnosis%20and%20management%20of%20infective%20endocarditis%20and%20its%20complications.%20Circulation.%201998;98(25):2936%E2%80%9348.%2010.1161/01.CIR.98.25.2936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Fernández Guerrero ML, Álvarez B, Manzarbeitia F, Renedo G. Infective endocarditis at autopsy: A review of pathologic manifestations and clinical correlates. Medicine. 2012;91(3):152–64. 10.1097/MD.0b013e31825631ea.
</cite> [<a href="https://doi.org/10.1097/MD.0b013e31825631ea" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22543628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fern%C3%A1ndez%20Guerrero%20ML,%20%C3%81lvarez%20B,%20Manzarbeitia%20F,%20Renedo%20G.%20Infective%20endocarditis%20at%20autopsy:%20A%20review%20of%20pathologic%20manifestations%20and%20clinical%20correlates.%20Medicine.%202012;91(3):152%E2%80%9364.%2010.1097/MD.0b013e31825631ea." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Safir F, Vu N, Tadesse LF, Firouzi K, Banaei N, Jeffrey SS, et al. Combining acoustic Bioprinting with AI-Assisted Raman spectroscopy for High-Throughput identification of bacteria in blood. Nano Lett. 2023;23(6):2065–73. 10.1021/acs.nanolett.2c03015.
</cite> [<a href="https://doi.org/10.1021/acs.nanolett.2c03015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10037319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36856600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Safir%20F,%20Vu%20N,%20Tadesse%20LF,%20Firouzi%20K,%20Banaei%20N,%20Jeffrey%20SS,%20et%20al.%20Combining%20acoustic%20Bioprinting%20with%20AI-Assisted%20Raman%20spectroscopy%20for%20High-Throughput%20identification%20of%20bacteria%20in%20blood.%20Nano%20Lett.%202023;23(6):2065%E2%80%9373.%2010.1021/acs.nanolett.2c03015." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Fenollar F, Raoult D. Molecular diagnosis of bloodstream infections caused by non-cultivable bacteria. Int J Antimicrob Agents. 2007;30(Suppl 1):S7–15. 10.1016/j.ijantimicag.2007.06.024.
</cite> [<a href="https://doi.org/10.1016/j.ijantimicag.2007.06.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17707613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fenollar%20F,%20Raoult%20D.%20Molecular%20diagnosis%20of%20bloodstream%20infections%20caused%20by%20non-cultivable%20bacteria.%20Int%20J%20Antimicrob%20Agents.%202007;30(Suppl%201):S7%E2%80%9315.%2010.1016/j.ijantimicag.2007.06.024." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Li JS, Sexton DJ, Mick N, Nettles R, Fowler VG Jr., Ryan T, et al. Proposed modifications to the Duke criteria for the diagnosis of infective endocarditis. Clin Infect Diseases: Official Publication Infect Dis Soc Am. 2000;30(4):633–8. 10.1086/313753.</cite> [<a href="https://doi.org/10.1086/313753" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10770721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20JS,%20Sexton%20DJ,%20Mick%20N,%20Nettles%20R,%20Fowler%20VG%20Jr.,%20Ryan%20T,%20et%20al.%20Proposed%20modifications%20to%20the%20Duke%20criteria%20for%20the%20diagnosis%20of%20infective%20endocarditis.%20Clin%20Infect%20Diseases:%20Official%20Publication%20Infect%20Dis%20Soc%20Am.%202000;30(4):633%E2%80%938.%2010.1086/313753." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Raoult D, Marrie T, Mege J. Natural history and pathophysiology of Q fever. Lancet Infect Dis. 2005;5(4):219–26. 10.1016/s1473-3099(05)70052-9.
</cite> [<a href="https://doi.org/10.1016/S1473-3099(05)70052-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15792739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raoult%20D,%20Marrie%20T,%20Mege%20J.%20Natural%20history%20and%20pathophysiology%20of%20Q%20fever.%20Lancet%20Infect%20Dis.%202005;5(4):219%E2%80%9326.%2010.1016/s1473-3099(05)70052-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Anton CI, DuȚUlescu S, Staniceanu F, Buzilă CA, Ștefan AT, Ștefan I et al. 2024. 10.20944/preprints202408.0466.v2</cite>
</li>
<li id="CR22">
<span class="label">22.</span><cite>Anderson DJ, Bulkley BH, Hutchins GM. A clinicopathologic study of prosthetic valve endocarditis in 22 patients: morphologic basis for diagnosis and therapy. Am Heart J. 1977;94(3):325–32. 10.1016/s0002-8703(77)80475-4.
</cite> [<a href="https://doi.org/10.1016/s0002-8703(77)80475-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/888765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Anderson%20DJ,%20Bulkley%20BH,%20Hutchins%20GM.%20A%20clinicopathologic%20study%20of%20prosthetic%20valve%20endocarditis%20in%2022%20patients:%20morphologic%20basis%20for%20diagnosis%20and%20therapy.%20Am%20Heart%20J.%201977;94(3):325%E2%80%9332.%2010.1016/s0002-8703(77)80475-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Fournier PE, Thuny F, Richet H, Lepidi H, Casalta JP, Arzouni JP, et al. Comprehensive diagnostic strategy for blood culture-negative endocarditis: a prospective study of 819 new cases. Clin Infect Diseases: Official Publication Infect Dis Soc Am. 2010;51(2):131–40. 10.1086/653675.</cite> [<a href="https://doi.org/10.1086/653675" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20540619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fournier%20PE,%20Thuny%20F,%20Richet%20H,%20Lepidi%20H,%20Casalta%20JP,%20Arzouni%20JP,%20et%20al.%20Comprehensive%20diagnostic%20strategy%20for%20blood%20culture-negative%20endocarditis:%20a%20prospective%20study%20of%20819%20new%20cases.%20Clin%20Infect%20Diseases:%20Official%20Publication%20Infect%20Dis%20Soc%20Am.%202010;51(2):131%E2%80%9340.%2010.1086/653675." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Janda JM, Abbott SL. 16S rRNA gene sequencing for bacterial identification in the diagnostic laboratory: pluses, perils, and pitfalls. J Clin Microbiol. 2007;45(9):2761–4. 10.1128/jcm.01228-07.
</cite> [<a href="https://doi.org/10.1128/JCM.01228-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2045242/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17626177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Janda%20JM,%20Abbott%20SL.%2016S%20rRNA%20gene%20sequencing%20for%20bacterial%20identification%20in%20the%20diagnostic%20laboratory:%20pluses,%20perils,%20and%20pitfalls.%20J%20Clin%20Microbiol.%202007;45(9):2761%E2%80%934.%2010.1128/jcm.01228-07." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Blauwkamp TA, Thair S, Rosen MJ, Blair L, Lindner MS, Vilfan ID, et al. Analytical and clinical validation of a microbial cell-free DNA sequencing test for infectious disease. Nat Microbiol. 2019;4(4):663–74. 10.1038/s41564-018-0349-6.
</cite> [<a href="https://doi.org/10.1038/s41564-018-0349-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30742071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blauwkamp%20TA,%20Thair%20S,%20Rosen%20MJ,%20Blair%20L,%20Lindner%20MS,%20Vilfan%20ID,%20et%20al.%20Analytical%20and%20clinical%20validation%20of%20a%20microbial%20cell-free%20DNA%20sequencing%20test%20for%20infectious%20disease.%20Nat%20Microbiol.%202019;4(4):663%E2%80%9374.%2010.1038/s41564-018-0349-6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Kirk F, Vaselli M. Blood culture negative infective endocarditis and surgery: an updated review of the literature. Heart Lung Circulation. 2024;33:S26–7. 10.1016/j.hlc.2024.04.071.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kirk%20F,%20Vaselli%20M.%20Blood%20culture%20negative%20infective%20endocarditis%20and%20surgery:%20an%20updated%20review%20of%20the%20literature.%20Heart%20Lung%20Circulation.%202024;33:S26%E2%80%937.%2010.1016/j.hlc.2024.04.071." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Moorlag S, Coolen JPM, van den Bosch B, Jin EH, Buil JB, Wertheim HFL, et al. Targeting the 16S rRNA gene by reverse complement PCR Next-Generation sequencing: specific and sensitive detection and identification of microbes directly in clinical samples. Microbiol Spectr. 2023;11(3):e0448322. 10.1128/spectrum.04483-22.
</cite> [<a href="https://doi.org/10.1128/spectrum.04483-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10269728/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37227289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moorlag%20S,%20Coolen%20JPM,%20van%20den%20Bosch%20B,%20Jin%20EH,%20Buil%20JB,%20Wertheim%20HFL,%20et%20al.%20Targeting%20the%2016S%20rRNA%20gene%20by%20reverse%20complement%20PCR%20Next-Generation%20sequencing:%20specific%20and%20sensitive%20detection%20and%20identification%20of%20microbes%20directly%20in%20clinical%20samples.%20Microbiol%20Spectr.%202023;11(3):e0448322.%2010.1128/spectrum.04483-22." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Botan A, Campisciano G, Zerbato V, Di Bella S, Simonetti O, Busetti M, et al. Performance of 16S rRNA gene Next-Generation sequencing and the culture method in the detection of bacteria in clinical specimens. Diagnostics. 2024;14(13):1318.
</cite> [<a href="https://doi.org/10.3390/diagnostics14131318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11240331/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39001210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Botan%20A,%20Campisciano%20G,%20Zerbato%20V,%20Di%20Bella%20S,%20Simonetti%20O,%20Busetti%20M,%20et%20al.%20Performance%20of%2016S%20rRNA%20gene%20Next-Generation%20sequencing%20and%20the%20culture%20method%20in%20the%20detection%20of%20bacteria%20in%20clinical%20specimens.%20Diagnostics.%202024;14(13):1318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Diao Z, Han D, Zhang R, Li J. Metagenomics next-generation sequencing tests take the stage in the diagnosis of lower respiratory tract infections. J Adv Res. 2022;38:201–12. 10.1016/j.jare.2021.09.012.
</cite> [<a href="https://doi.org/10.1016/j.jare.2021.09.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9091713/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35572406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diao%20Z,%20Han%20D,%20Zhang%20R,%20Li%20J.%20Metagenomics%20next-generation%20sequencing%20tests%20take%20the%20stage%20in%20the%20diagnosis%20of%20lower%20respiratory%20tract%20infections.%20J%20Adv%20Res.%202022;38:201%E2%80%9312.%2010.1016/j.jare.2021.09.012." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Kim M, Fida M, Abu Saleh OM, Ranganath N. From Culture-Negative to DNA-Positive: the molecular revolution in infective endocarditis diagnosis. Pathogens. 2025;14(6):518.
</cite> [<a href="https://doi.org/10.3390/pathogens14060518" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12196431/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40559526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20M,%20Fida%20M,%20Abu%20Saleh%20OM,%20Ranganath%20N.%20From%20Culture-Negative%20to%20DNA-Positive:%20the%20molecular%20revolution%20in%20infective%20endocarditis%20diagnosis.%20Pathogens.%202025;14(6):518." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Qian M, Zhan Y, Wu D, Ji L, Chen H, Cheng Y. Clinical standardization of metagenomic next generation sequencing (mNGS) in the pathogen diagnosis. Clin Translational Discovery. 2021;1(1):e12. 10.1002/ctd2.12.</cite> [<a href="https://scholar.google.com/scholar_lookup?Qian%20M,%20Zhan%20Y,%20Wu%20D,%20Ji%20L,%20Chen%20H,%20Cheng%20Y.%20Clinical%20standardization%20of%20metagenomic%20next%20generation%20sequencing%20(mNGS)%20in%20the%20pathogen%20diagnosis.%20Clin%20Translational%20Discovery.%202021;1(1):e12.%2010.1002/ctd2.12." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Yi L, Tan L, Long Q, Lyu X, Zeng H, Peng Y, et al. Comparative diagnostic performance of metagenomic and two targeted sequencing methods in lower respiratory infection. Sci Rep. 2025;15(1):27365. 10.1038/s41598-025-11834-w.
</cite> [<a href="https://doi.org/10.1038/s41598-025-11834-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12301447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40717117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yi%20L,%20Tan%20L,%20Long%20Q,%20Lyu%20X,%20Zeng%20H,%20Peng%20Y,%20et%20al.%20Comparative%20diagnostic%20performance%20of%20metagenomic%20and%20two%20targeted%20sequencing%20methods%20in%20lower%20respiratory%20infection.%20Sci%20Rep.%202025;15(1):27365.%2010.1038/s41598-025-11834-w." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Roy S, Coldren C, Karunamurthy A, Kip NS, Klee EW, Lincoln SE, et al. Standards and guidelines for validating Next-Generation sequencing bioinformatics pipelines: A joint recommendation of the association for molecular pathology and the college of American pathologists. J Mol Diagn. 2018;20(1):4–27. 10.1016/j.jmoldx.2017.11.003.
</cite> [<a href="https://doi.org/10.1016/j.jmoldx.2017.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29154853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roy%20S,%20Coldren%20C,%20Karunamurthy%20A,%20Kip%20NS,%20Klee%20EW,%20Lincoln%20SE,%20et%20al.%20Standards%20and%20guidelines%20for%20validating%20Next-Generation%20sequencing%20bioinformatics%20pipelines:%20A%20joint%20recommendation%20of%20the%20association%20for%20molecular%20pathology%20and%20the%20college%20of%20American%20pathologists.%20J%20Mol%20Diagn.%202018;20(1):4%E2%80%9327.%2010.1016/j.jmoldx.2017.11.003." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Zhang H, Shen D, Zhou J, Yang Q, Ying Y, Li N, et al. The utility of metagenomic Next-Generation sequencing (mNGS) in the management of patients with bronchiectasis: A Single-Center retrospective study of 93 cases. Open Forum Infect Dis. 2023;10(8):ofad425. 10.1093/ofid/ofad425.
</cite> [<a href="https://doi.org/10.1093/ofid/ofad425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10470666/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37663088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20H,%20Shen%20D,%20Zhou%20J,%20Yang%20Q,%20Ying%20Y,%20Li%20N,%20et%20al.%20The%20utility%20of%20metagenomic%20Next-Generation%20sequencing%20(mNGS)%20in%20the%20management%20of%20patients%20with%20bronchiectasis:%20A%20Single-Center%20retrospective%20study%20of%2093%20cases.%20Open%20Forum%20Infect%20Dis.%202023;10(8):ofad425.%2010.1093/ofid/ofad425." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Bloemen B, Gand M, Vanneste K, Marchal K, Roosens NHC, De Keersmaecker SCJ. Development of a portable on-site applicable metagenomic data generation workflow for enhanced pathogen and antimicrobial resistance surveillance. Sci Rep. 2023;13(1):19656. 10.1038/s41598-023-46771-z.
</cite> [<a href="https://doi.org/10.1038/s41598-023-46771-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10640560/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37952062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bloemen%20B,%20Gand%20M,%20Vanneste%20K,%20Marchal%20K,%20Roosens%20NHC,%20De%20Keersmaecker%20SCJ.%20Development%20of%20a%20portable%20on-site%20applicable%20metagenomic%20data%20generation%20workflow%20for%20enhanced%20pathogen%20and%20antimicrobial%20resistance%20surveillance.%20Sci%20Rep.%202023;13(1):19656.%2010.1038/s41598-023-46771-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Abi Cit SO, Abey Litty. Integrating accelerating portable DNA sequencing for rapid Monkeypox strain identification with vision Transformer-based lesion analysis for enhanced epidemiological surveillance. 2025.</cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Luan Y, Hu H, Liu C, Chen B, Liu X, Xu Y, et al. A proof-of-concept study of an automated solution for clinical metagenomic next-generation sequencing. J Appl Microbiol. 2021;131(2):1007–16. 10.1111/jam.15003.
</cite> [<a href="https://doi.org/10.1111/jam.15003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33440055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luan%20Y,%20Hu%20H,%20Liu%20C,%20Chen%20B,%20Liu%20X,%20Xu%20Y,%20et%20al.%20A%20proof-of-concept%20study%20of%20an%20automated%20solution%20for%20clinical%20metagenomic%20next-generation%20sequencing.%20J%20Appl%20Microbiol.%202021;131(2):1007%E2%80%9316.%2010.1111/jam.15003." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Jiang Y, Wang K, Zhang Y, Yuan K, Huai Y, Cui Y. Advances in the application of macrogenomic next generation and targeted next generation sequencing technologies in the diagnosis of infectious lung pathogens. Discover Med. 2025;2(1):172. 10.1007/s44337-025-00244-6.</cite> [<a href="https://scholar.google.com/scholar_lookup?Jiang%20Y,%20Wang%20K,%20Zhang%20Y,%20Yuan%20K,%20Huai%20Y,%20Cui%20Y.%20Advances%20in%20the%20application%20of%20macrogenomic%20next%20generation%20and%20targeted%20next%20generation%20sequencing%20technologies%20in%20the%20diagnosis%20of%20infectious%20lung%20pathogens.%20Discover%20Med.%202025;2(1):172.%2010.1007/s44337-025-00244-6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Sun W, Zheng L, Kang L, Chen C, Wang L, Lu L, et al. Comparative analysis of metagenomic and targeted next-generation sequencing for pathogens diagnosis in Bronchoalveolar lavage fluid specimens. Front Cell Infect Microbiol. 2024;14:1451440. 10.3389/fcimb.2024.1451440.
</cite> [<a href="https://doi.org/10.3389/fcimb.2024.1451440" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11385274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39258254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun%20W,%20Zheng%20L,%20Kang%20L,%20Chen%20C,%20Wang%20L,%20Lu%20L,%20et%20al.%20Comparative%20analysis%20of%20metagenomic%20and%20targeted%20next-generation%20sequencing%20for%20pathogens%20diagnosis%20in%20Bronchoalveolar%20lavage%20fluid%20specimens.%20Front%20Cell%20Infect%20Microbiol.%202024;14:1451440.%2010.3389/fcimb.2024.1451440." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Cai S, Yang Y, Pan J, Miao Q, Jin W, Ma Y, et al. The clinical value of valve metagenomic next-generation sequencing when applied to the etiological diagnosis of infective endocarditis. Annals Translational Med. 2021;9(19):1490. 10.21037/atm-21-2488.</cite> [<a href="https://doi.org/10.21037/atm-21-2488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8573444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34805352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cai%20S,%20Yang%20Y,%20Pan%20J,%20Miao%20Q,%20Jin%20W,%20Ma%20Y,%20et%20al.%20The%20clinical%20value%20of%20valve%20metagenomic%20next-generation%20sequencing%20when%20applied%20to%20the%20etiological%20diagnosis%20of%20infective%20endocarditis.%20Annals%20Translational%20Med.%202021;9(19):1490.%2010.21037/atm-21-2488." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Mu X, Liang J, Qian L, Zhou B, Zou X, Fu Y, et al. Genome analysis of ST1 Bartonella henselae, a zoonotic pathogen causing endocarditis in an elderly patient in China. Infect Drug Resist. 2023;16:6079–84. 10.2147/idr.S422345.
</cite> [<a href="https://doi.org/10.2147/IDR.S422345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10503507/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37719652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mu%20X,%20Liang%20J,%20Qian%20L,%20Zhou%20B,%20Zou%20X,%20Fu%20Y,%20et%20al.%20Genome%20analysis%20of%20ST1%20Bartonella%20henselae,%20a%20zoonotic%20pathogen%20causing%20endocarditis%20in%20an%20elderly%20patient%20in%20China.%20Infect%20Drug%20Resist.%202023;16:6079%E2%80%9384.%2010.2147/idr.S422345." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Wang K-M, Mu N, Wang H-B. Intracranial infection in an adult caused by Mycoplasma hominis, diagnosed using mNGS technology: a case report. Front Med. 2025;12–2025. 10.3389/fmed.2025.1560635.</cite> [<a href="https://doi.org/10.3389/fmed.2025.1560635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11913662/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40103786/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Zeng X, Wu J, Li X, Xiong W, Tang L, Li X, et al. Application of metagenomic Next-Generation sequencing in the etiological diagnosis of infective endocarditis during the perioperative period of cardiac surgery: A prospective cohort study. Front Cardiovasc Med. 2022;9:811492. 10.3389/fcvm.2022.811492.
</cite> [<a href="https://doi.org/10.3389/fcvm.2022.811492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8965566/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35369282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng%20X,%20Wu%20J,%20Li%20X,%20Xiong%20W,%20Tang%20L,%20Li%20X,%20et%20al.%20Application%20of%20metagenomic%20Next-Generation%20sequencing%20in%20the%20etiological%20diagnosis%20of%20infective%20endocarditis%20during%20the%20perioperative%20period%20of%20cardiac%20surgery:%20A%20prospective%20cohort%20study.%20Front%20Cardiovasc%20Med.%202022;9:811492.%2010.3389/fcvm.2022.811492." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Gu W, Miller S, Chiu C. Clinical metagenomic Next-Generation sequencing for pathogen detection. Annu Rev Pathol. 2019;14. 10.1146/annurev-pathmechdis-012418-012751.</cite> [<a href="https://doi.org/10.1146/annurev-pathmechdis-012418-012751" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6345613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30355154/" class="usa-link">PubMed</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Han D, Yu F, Zhang D, Yang Q, Xie M, Yuan L, et al. The Real-World clinical impact of plasma mNGS testing: an observational study. Microbiol Spectr. 2023;11(2):e0398322. 10.1128/spectrum.03983-22.
</cite> [<a href="https://doi.org/10.1128/spectrum.03983-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10101021/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36946733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han%20D,%20Yu%20F,%20Zhang%20D,%20Yang%20Q,%20Xie%20M,%20Yuan%20L,%20et%20al.%20The%20Real-World%20clinical%20impact%20of%20plasma%20mNGS%20testing:%20an%20observational%20study.%20Microbiol%20Spectr.%202023;11(2):e0398322.%2010.1128/spectrum.03983-22." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Miller S, Naccache SN, Samayoa E, Messacar K, Arevalo S, Federman S, et al. Laboratory validation of a clinical metagenomic sequencing assay for pathogen detection in cerebrospinal fluid. Genome Res. 2019;29(5):831–42. 10.1101/gr.238170.118.
</cite> [<a href="https://doi.org/10.1101/gr.238170.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6499319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30992304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miller%20S,%20Naccache%20SN,%20Samayoa%20E,%20Messacar%20K,%20Arevalo%20S,%20Federman%20S,%20et%20al.%20Laboratory%20validation%20of%20a%20clinical%20metagenomic%20sequencing%20assay%20for%20pathogen%20detection%20in%20cerebrospinal%20fluid.%20Genome%20Res.%202019;29(5):831%E2%80%9342.%2010.1101/gr.238170.118." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Chen H, Zhan M, Liu S, Balloux F, Wang H. Unraveling the potential of metagenomic next-generation sequencing in infectious disease diagnosis: challenges and prospects. Sci Bull (Beijing). 2024;69(11):1586–9. 10.1016/j.scib.2024.04.033.
</cite> [<a href="https://doi.org/10.1016/j.scib.2024.04.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38670851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20H,%20Zhan%20M,%20Liu%20S,%20Balloux%20F,%20Wang%20H.%20Unraveling%20the%20potential%20of%20metagenomic%20next-generation%20sequencing%20in%20infectious%20disease%20diagnosis:%20challenges%20and%20prospects.%20Sci%20Bull%20(Beijing).%202024;69(11):1586%E2%80%939.%2010.1016/j.scib.2024.04.033." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Zhao Y, Zhang W, Zhang X. Application of metagenomic next-generation sequencing in the diagnosis of infectious diseases. Front Cell Infect Microbiol. 2024;14:1458316. 10.3389/fcimb.2024.1458316.
</cite> [<a href="https://doi.org/10.3389/fcimb.2024.1458316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11604630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39619659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20Y,%20Zhang%20W,%20Zhang%20X.%20Application%20of%20metagenomic%20next-generation%20sequencing%20in%20the%20diagnosis%20of%20infectious%20diseases.%20Front%20Cell%20Infect%20Microbiol.%202024;14:1458316.%2010.3389/fcimb.2024.1458316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Chen X, Liang Y, Yang W, He W, Xing Z, Li S, et al. Application of metagenomic next-generation sequencing in the diagnosis of post-stroke infections: a case series study using multiple sample types. Front Cell Infect Microbiol. 2024;14:1386377. 10.3389/fcimb.2024.1386377.
</cite> [<a href="https://doi.org/10.3389/fcimb.2024.1386377" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11751054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39844841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20X,%20Liang%20Y,%20Yang%20W,%20He%20W,%20Xing%20Z,%20Li%20S,%20et%20al.%20Application%20of%20metagenomic%20next-generation%20sequencing%20in%20the%20diagnosis%20of%20post-stroke%20infections:%20a%20case%20series%20study%20using%20multiple%20sample%20types.%20Front%20Cell%20Infect%20Microbiol.%202024;14:1386377.%2010.3389/fcimb.2024.1386377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Kim HM, Jeon S, Chung O, Jun JH, Kim HS, Blazyte A, et al. Comparative analysis of 7 short-read sequencing platforms using the Korean reference genome: MGI and illumina sequencing benchmark for whole-genome sequencing. Gigascience. 2021;10(3). 10.1093/gigascience/giab014.</cite> [<a href="https://doi.org/10.1093/gigascience/giab014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7953489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33710328/" class="usa-link">PubMed</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Han D, Yu F, Zhang D, Hu J, Zhang X, Xiang D, et al. Molecular rapid diagnostic testing for bloodstream infections: nanopore targeted sequencing with pathogen-specific primers. J Infect. 2024;88(6):106166. 10.1016/j.jinf.2024.106166.
</cite> [<a href="https://doi.org/10.1016/j.jinf.2024.106166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38670268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han%20D,%20Yu%20F,%20Zhang%20D,%20Hu%20J,%20Zhang%20X,%20Xiang%20D,%20et%20al.%20Molecular%20rapid%20diagnostic%20testing%20for%20bloodstream%20infections:%20nanopore%20targeted%20sequencing%20with%20pathogen-specific%20primers.%20J%20Infect.%202024;88(6):106166.%2010.1016/j.jinf.2024.106166." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Goodwin S, McPherson JD, McCombie WR. Coming of age: ten years of next-generation sequencing technologies. Nat Rev Genet. 2016;17(6):333–51. 10.1038/nrg.2016.49.
</cite> [<a href="https://doi.org/10.1038/nrg.2016.49" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10373632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27184599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goodwin%20S,%20McPherson%20JD,%20McCombie%20WR.%20Coming%20of%20age:%20ten%20years%20of%20next-generation%20sequencing%20technologies.%20Nat%20Rev%20Genet.%202016;17(6):333%E2%80%9351.%2010.1038/nrg.2016.49." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Foox J, Tighe SW, Nicolet CM, Zook JM, Byrska-Bishop M, Clarke WE, et al. Performance assessment of DNA sequencing platforms in the ABRF Next-Generation sequencing study. Nat Biotechnol. 2021;39(9):1129–40. 10.1038/s41587-021-01049-5.
</cite> [<a href="https://doi.org/10.1038/s41587-021-01049-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8985210/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34504351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Foox%20J,%20Tighe%20SW,%20Nicolet%20CM,%20Zook%20JM,%20Byrska-Bishop%20M,%20Clarke%20WE,%20et%20al.%20Performance%20assessment%20of%20DNA%20sequencing%20platforms%20in%20the%20ABRF%20Next-Generation%20sequencing%20study.%20Nat%20Biotechnol.%202021;39(9):1129%E2%80%9340.%2010.1038/s41587-021-01049-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Espinosa E, Bautista R, Larrosa R, Plata O. Advancements in long-read genome sequencing technologies and algorithms. Genomics. 2024;116(3):110842. 10.1016/j.ygeno.2024.110842.
</cite> [<a href="https://doi.org/10.1016/j.ygeno.2024.110842" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38608738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Espinosa%20E,%20Bautista%20R,%20Larrosa%20R,%20Plata%20O.%20Advancements%20in%20long-read%20genome%20sequencing%20technologies%20and%20algorithms.%20Genomics.%202024;116(3):110842.%2010.1016/j.ygeno.2024.110842." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Chen Y, Davidson NM, Wan YK, Yao F, Su Y, Gamaarachchi H, et al. A systematic benchmark of nanopore long-read RNA sequencing for transcript-level analysis in human cell lines. Nat Methods. 2025;22(4):801–12. 10.1038/s41592-025-02623-4.
</cite> [<a href="https://doi.org/10.1038/s41592-025-02623-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11978509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40082608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20Y,%20Davidson%20NM,%20Wan%20YK,%20Yao%20F,%20Su%20Y,%20Gamaarachchi%20H,%20et%20al.%20A%20systematic%20benchmark%20of%20nanopore%20long-read%20RNA%20sequencing%20for%20transcript-level%20analysis%20in%20human%20cell%20lines.%20Nat%20Methods.%202025;22(4):801%E2%80%9312.%2010.1038/s41592-025-02623-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Runtuwene LR, Tuda JSB, Mongan AE, Suzuki Y. On-Site minion sequencing. Advances in experimental medicine and biology. 2019;1129:143–50. 10.1007/978-981-13-6037-4_10</cite> [<a href="https://doi.org/10.1007/978-981-13-6037-4_10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30968366/" class="usa-link">PubMed</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Charalampous T, Kay GL, Richardson H, Aydin A, Baldan R, Jeanes C, et al. Nanopore metagenomics enables rapid clinical diagnosis of bacterial lower respiratory infection. Nat Biotechnol. 2019;37(7):783–92. 10.1038/s41587-019-0156-5.
</cite> [<a href="https://doi.org/10.1038/s41587-019-0156-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31235920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Charalampous%20T,%20Kay%20GL,%20Richardson%20H,%20Aydin%20A,%20Baldan%20R,%20Jeanes%20C,%20et%20al.%20Nanopore%20metagenomics%20enables%20rapid%20clinical%20diagnosis%20of%20bacterial%20lower%20respiratory%20infection.%20Nat%20Biotechnol.%202019;37(7):783%E2%80%9392.%2010.1038/s41587-019-0156-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Fu A, Wang X, Shen Z, Gu J, Zhang T, Wang F, et al. Rapid diagnostic testing for infective endocarditis and myocarditis using nanopore targeted sequencing. iLABMED. 2025;3(2):191–200. 10.1002/ila2.70016.</cite> [<a href="https://scholar.google.com/scholar_lookup?Fu%20A,%20Wang%20X,%20Shen%20Z,%20Gu%20J,%20Zhang%20T,%20Wang%20F,%20et%20al.%20Rapid%20diagnostic%20testing%20for%20infective%20endocarditis%20and%20myocarditis%20using%20nanopore%20targeted%20sequencing.%20iLABMED.%202025;3(2):191%E2%80%93200.%2010.1002/ila2.70016." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Wick RR, Judd LM, Gorrie CL, Holt KE, Unicycler. Resolving bacterial genome assemblies from short and long sequencing reads. PLoS Comput Biol. 2017;13(6):e1005595. 10.1371/journal.pcbi.1005595.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1005595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5481147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28594827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wick%20RR,%20Judd%20LM,%20Gorrie%20CL,%20Holt%20KE,%20Unicycler.%20Resolving%20bacterial%20genome%20assemblies%20from%20short%20and%20long%20sequencing%20reads.%20PLoS%20Comput%20Biol.%202017;13(6):e1005595.%2010.1371/journal.pcbi.1005595." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Khamidova1Albina VVMVRVRPZZALLVVBVBMDMDMASSNZZSIKI. Comparative evaluation of sequencing platforms: Pacific biosciences, Oxford nanopore technologies, and illumina for 16S rRNA-Based soil Microbiome profiling. Front Microbiol. 2025;16–2025. 10.3389/fmicb.2025.1633360.</cite> [<a href="https://doi.org/10.3389/fmicb.2025.1633360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12365774/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40842827/" class="usa-link">PubMed</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Singh A, Bhatia P. Comparative sequencing data analysis of ion torrent and minion sequencing platforms using a clinical diagnostic haematology panel. Int J Lab Hematol. 2020;42(6):833–41. 10.1111/ijlh.13286.
</cite> [<a href="https://doi.org/10.1111/ijlh.13286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32654382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh%20A,%20Bhatia%20P.%20Comparative%20sequencing%20data%20analysis%20of%20ion%20torrent%20and%20minion%20sequencing%20platforms%20using%20a%20clinical%20diagnostic%20haematology%20panel.%20Int%20J%20Lab%20Hematol.%202020;42(6):833%E2%80%9341.%2010.1111/ijlh.13286." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Meslier V, Quinquis B, Da Silva K, Plaza Oñate F, Pons N, Roume H, et al. Benchmarking second and third-generation sequencing platforms for microbial metagenomics. Sci Data. 2022;9(1):694. 10.1038/s41597-022-01762-z.
</cite> [<a href="https://doi.org/10.1038/s41597-022-01762-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9652401/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36369227/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meslier%20V,%20Quinquis%20B,%20Da%20Silva%20K,%20Plaza%20O%C3%B1ate%20F,%20Pons%20N,%20Roume%20H,%20et%20al.%20Benchmarking%20second%20and%20third-generation%20sequencing%20platforms%20for%20microbial%20metagenomics.%20Sci%20Data.%202022;9(1):694.%2010.1038/s41597-022-01762-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Salter SJ, Cox MJ, Turek EM, Calus ST, Cookson WO, Moffatt MF, et al. Reagent and laboratory contamination can critically impact sequence-based Microbiome analyses. BMC Biol. 2014;12:87. 10.1186/s12915-014-0087-z.
</cite> [<a href="https://doi.org/10.1186/s12915-014-0087-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4228153/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25387460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Salter%20SJ,%20Cox%20MJ,%20Turek%20EM,%20Calus%20ST,%20Cookson%20WO,%20Moffatt%20MF,%20et%20al.%20Reagent%20and%20laboratory%20contamination%20can%20critically%20impact%20sequence-based%20Microbiome%20analyses.%20BMC%20Biol.%202014;12:87.%2010.1186/s12915-014-0087-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Kinde I, Wu J, Papadopoulos N, Kinzler KW, Vogelstein B. Detection and quantification of rare mutations with massively parallel sequencing. Proc Natl Acad Sci. 2011;108(23):9530–5. 10.1073/pnas.1105422108.
</cite> [<a href="https://doi.org/10.1073/pnas.1105422108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3111315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21586637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kinde%20I,%20Wu%20J,%20Papadopoulos%20N,%20Kinzler%20KW,%20Vogelstein%20B.%20Detection%20and%20quantification%20of%20rare%20mutations%20with%20massively%20parallel%20sequencing.%20Proc%20Natl%20Acad%20Sci.%202011;108(23):9530%E2%80%935.%2010.1073/pnas.1105422108." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Lin KP, Yeh TK, Chuang YC, Wang LA, Fu YC, Liu PY. Blood culture negative endocarditis: A review of laboratory diagnostic approaches. Int J Gen Med. 2023;16:317–27. 10.2147/ijgm.S393329.
</cite> [<a href="https://doi.org/10.2147/IJGM.S393329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9884005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36718144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20KP,%20Yeh%20TK,%20Chuang%20YC,%20Wang%20LA,%20Fu%20YC,%20Liu%20PY.%20Blood%20culture%20negative%20endocarditis:%20A%20review%20of%20laboratory%20diagnostic%20approaches.%20Int%20J%20Gen%20Med.%202023;16:317%E2%80%9327.%2010.2147/ijgm.S393329." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Wilson MR, Sample HA, Zorn KC, Arevalo S, Yu G, Neuhaus J, et al. Clinical metagenomic sequencing for diagnosis of meningitis and encephalitis. N Engl J Med. 2019;380(24):2327–40. 10.1056/NEJMoa1803396.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1803396" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6764751/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31189036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilson%20MR,%20Sample%20HA,%20Zorn%20KC,%20Arevalo%20S,%20Yu%20G,%20Neuhaus%20J,%20et%20al.%20Clinical%20metagenomic%20sequencing%20for%20diagnosis%20of%20meningitis%20and%20encephalitis.%20N%20Engl%20J%20Med.%202019;380(24):2327%E2%80%9340.%2010.1056/NEJMoa1803396." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Peng X, Qiao R, Li P, Chen L. DNFE: directed network flow entropy for detecting tipping points during biological processes. PLoS Comput Biol. 2025;21(7):e1013336. 10.1371/journal.pcbi.1013336.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1013336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12316398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40729372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peng%20X,%20Qiao%20R,%20Li%20P,%20Chen%20L.%20DNFE:%20directed%20network%20flow%20entropy%20for%20detecting%20tipping%20points%20during%20biological%20processes.%20PLoS%20Comput%20Biol.%202025;21(7):e1013336.%2010.1371/journal.pcbi.1013336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Kokoris M, McRuer R, Nabavi M, Jacobs A, Prindle M, Cech C, et al. Sequencing by expansion (SBX) - a novel, high-throughput single-molecule sequencing technology. BioRxiv. 2025. 10.1101/2025.02.19.639056.
</cite> [<a href="https://doi.org/10.1101/2025.02.19.639056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11888190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40060512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kokoris%20M,%20McRuer%20R,%20Nabavi%20M,%20Jacobs%20A,%20Prindle%20M,%20Cech%20C,%20et%20al.%20Sequencing%20by%20expansion%20(SBX)%20-%20a%20novel,%20high-throughput%20single-molecule%20sequencing%20technology.%20BioRxiv.%202025.%2010.1101/2025.02.19.639056." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Zhang J-Y, Zhang Y, Wang L, Guo F, Yun Q, Zeng T et al. 2024. 10.1101/2024.08.19.608720</cite>
</li>
<li id="CR70">
<span class="label">70.</span><cite>Sandoe JAT, Chemotherapy obotEWPotBSfA. Guidelines for the diagnosis and antibiotic treatment of endocarditis in adults: a report of the working party of the British society for antimicrobial Chemotherapy—author’s response. J Antimicrob Chemother. 2012;67(12):3017. 10.1093/jac/dks366.
</cite> [<a href="https://doi.org/10.1093/jac/dkr450" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22086858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sandoe%20JAT,%20Chemotherapy%20obotEWPotBSfA.%20Guidelines%20for%20the%20diagnosis%20and%20antibiotic%20treatment%20of%20endocarditis%20in%20adults:%20a%20report%20of%20the%20working%20party%20of%20the%20British%20society%20for%20antimicrobial%20Chemotherapy%E2%80%94author%E2%80%99s%20response.%20J%20Antimicrob%20Chemother.%202012;67(12):3017.%2010.1093/jac/dks366." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Land KJ, Boeras DI, Chen X-S, Ramsay AR, Peeling RW. REASSURED diagnostics to inform disease control strategies, strengthen health systems and improve patient outcomes. Nat Microbiol. 2019;4(1):46–54. 10.1038/s41564-018-0295-3.
</cite> [<a href="https://doi.org/10.1038/s41564-018-0295-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7097043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30546093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Land%20KJ,%20Boeras%20DI,%20Chen%20X-S,%20Ramsay%20AR,%20Peeling%20RW.%20REASSURED%20diagnostics%20to%20inform%20disease%20control%20strategies,%20strengthen%20health%20systems%20and%20improve%20patient%20outcomes.%20Nat%20Microbiol.%202019;4(1):46%E2%80%9354.%2010.1038/s41564-018-0295-3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Beam AL, Kohane IS. Big data and machine learning in health care. JAMA. 2018;319(13):1317–8. 10.1001/jama.2017.18391.
</cite> [<a href="https://doi.org/10.1001/jama.2017.18391" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29532063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beam%20AL,%20Kohane%20IS.%20Big%20data%20and%20machine%20learning%20in%20health%20care.%20JAMA.%202018;319(13):1317%E2%80%938.%2010.1001/jama.2017.18391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Topol EJ. High-performance medicine: the convergence of human and artificial intelligence. Nat Med. 2019;25(1):44–56. 10.1038/s41591-018-0300-7.
</cite> [<a href="https://doi.org/10.1038/s41591-018-0300-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30617339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Topol%20EJ.%20High-performance%20medicine:%20the%20convergence%20of%20human%20and%20artificial%20intelligence.%20Nat%20Med.%202019;25(1):44%E2%80%9356.%2010.1038/s41591-018-0300-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Collins FS, Varmus H. A new initiative on precision medicine. N Engl J Med. 2015;372(9):793–5.
</cite> [<a href="https://doi.org/10.1056/NEJMp1500523" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5101938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25635347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Collins%20FS,%20Varmus%20H.%20A%20new%20initiative%20on%20precision%20medicine.%20N%20Engl%20J%20Med.%202015;372(9):793%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Ashley EA. Towards precision medicine. Nat Rev Genet. 2016;17(9):507–22. 10.1038/nrg.2016.86.
</cite> [<a href="https://doi.org/10.1038/nrg.2016.86" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27528417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ashley%20EA.%20Towards%20precision%20medicine.%20Nat%20Rev%20Genet.%202016;17(9):507%E2%80%9322.%2010.1038/nrg.2016.86." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Wang H, Wang Y, Wang W, Zhang Y, Yuan Q, Tan J. Materials-Functionalized Point-of-Care testing devices for pathogen detection. Acc Mater Res. 2023;4(12):1083–94. 10.1021/accountsmr.3c00151.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang%20H,%20Wang%20Y,%20Wang%20W,%20Zhang%20Y,%20Yuan%20Q,%20Tan%20J.%20Materials-Functionalized%20Point-of-Care%20testing%20devices%20for%20pathogen%20detection.%20Acc%20Mater%20Res.%202023;4(12):1083%E2%80%9394.%2010.1021/accountsmr.3c00151." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Schwamm LH. Telehealth: seven strategies to successfully implement disruptive technology and transform health care. Health Aff (Millwood). 2014;33(2):200–6. 10.1377/hlthaff.2013.1021.
</cite> [<a href="https://doi.org/10.1377/hlthaff.2013.1021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24493761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schwamm%20LH.%20Telehealth:%20seven%20strategies%20to%20successfully%20implement%20disruptive%20technology%20and%20transform%20health%20care.%20Health%20Aff%20(Millwood).%202014;33(2):200%E2%80%936.%2010.1377/hlthaff.2013.1021." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Wagner Darlene D, Marine Rachel L, Ramos E, Ng Terry Fei F, Castro Christina J, Okomo-Adhiambo M, et al. VPipe: an automated bioinformatics platform for assembly and management of viral Next-Generation sequencing data. Microbiol Spectr. 2022;10(2):e02564–21. 10.1128/spectrum.02564-21.
</cite> [<a href="https://doi.org/10.1128/spectrum.02564-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8941893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35234489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wagner%20Darlene%20D,%20Marine%20Rachel%20L,%20Ramos%20E,%20Ng%20Terry%20Fei%20F,%20Castro%20Christina%20J,%20Okomo-Adhiambo%20M,%20et%20al.%20VPipe:%20an%20automated%20bioinformatics%20platform%20for%20assembly%20and%20management%20of%20viral%20Next-Generation%20sequencing%20data.%20Microbiol%20Spectr.%202022;10(2):e02564%E2%80%9321.%2010.1128/spectrum.02564-21." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>MacNair CR, Tsai CN, Rutherford ST, Tan M-W. Returning to nature for the next generation of antimicrobial therapeutics. Antibiotics. 2023;12(8):1267.
</cite> [<a href="https://doi.org/10.3390/antibiotics12081267" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10451936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37627687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?MacNair%20CR,%20Tsai%20CN,%20Rutherford%20ST,%20Tan%20M-W.%20Returning%20to%20nature%20for%20the%20next%20generation%20of%20antimicrobial%20therapeutics.%20Antibiotics.%202023;12(8):1267." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Hajjo R, Sabbah DA, Al Bawab AQ. Unlocking the potential of the human Microbiome for identifying disease diagnostic biomarkers. Diagnostics (Basel). 2022;12(7). 10.3390/diagnostics12071742.</cite> [<a href="https://doi.org/10.3390/diagnostics12071742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9315466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35885645/" class="usa-link">PubMed</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Su Q, Liu Q, Lau RI, Zhang J, Xu Z, Yeoh YK, et al. Faecal microbiome-based machine learning for multi-class disease diagnosis. Nat Commun. 2022;13(1):6818. 10.1038/s41467-022-34405-3.
</cite> [<a href="https://doi.org/10.1038/s41467-022-34405-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9649010/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36357393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su%20Q,%20Liu%20Q,%20Lau%20RI,%20Zhang%20J,%20Xu%20Z,%20Yeoh%20YK,%20et%20al.%20Faecal%20microbiome-based%20machine%20learning%20for%20multi-class%20disease%20diagnosis.%20Nat%20Commun.%202022;13(1):6818.%2010.1038/s41467-022-34405-3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Wang ZY, Li LL, Cao XL, Li P, Du J, Zou MJ, et al. Clinical application of amplification-based versus amplification-free metagenomic next-generation sequencing test in infectious diseases. Front Cell Infect Microbiol. 2023;13:1138174. 10.3389/fcimb.2023.1138174.
</cite> [<a href="https://doi.org/10.3389/fcimb.2023.1138174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10716234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38094744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20ZY,%20Li%20LL,%20Cao%20XL,%20Li%20P,%20Du%20J,%20Zou%20MJ,%20et%20al.%20Clinical%20application%20of%20amplification-based%20versus%20amplification-free%20metagenomic%20next-generation%20sequencing%20test%20in%20infectious%20diseases.%20Front%20Cell%20Infect%20Microbiol.%202023;13:1138174.%2010.3389/fcimb.2023.1138174." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>von Teichman A, Storz M, Dettwiler S, Moch H, Schraml P. Whole genome and transcriptome amplification: practicable tools for sustainable tissue biobanking? Virchows Arch. 2012;461(5):571–80. 10.1007/s00428-012-1315-y.
</cite> [<a href="https://doi.org/10.1007/s00428-012-1315-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23007645/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?von%20Teichman%20A,%20Storz%20M,%20Dettwiler%20S,%20Moch%20H,%20Schraml%20P.%20Whole%20genome%20and%20transcriptome%20amplification:%20practicable%20tools%20for%20sustainable%20tissue%20biobanking?%20Virchows%20Arch.%202012;461(5):571%E2%80%9380.%2010.1007/s00428-012-1315-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Simner PJ, Miller HB, Breitwieser FP, Pinilla Monsalve G, Pardo CA, Salzberg SL, et al. Development and optimization of metagenomic Next-Generation sequencing methods for cerebrospinal fluid diagnostics. J Clin Microbiol. 2018;56(9). 10.1128/jcm.00472-18.</cite> [<a href="https://doi.org/10.1128/JCM.00472-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6113476/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29976594/" class="usa-link">PubMed</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Pinard R, de Winter A, Sarkis GJ, Gerstein MB, Tartaro KR, Plant RN, et al. Assessment of whole genome amplification-induced bias through high-throughput, massively parallel whole genome sequencing. BMC Genomics. 2006;7(1):216. 10.1186/1471-2164-7-216.
</cite> [<a href="https://doi.org/10.1186/1471-2164-7-216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1560136/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16928277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pinard%20R,%20de%20Winter%20A,%20Sarkis%20GJ,%20Gerstein%20MB,%20Tartaro%20KR,%20Plant%20RN,%20et%20al.%20Assessment%20of%20whole%20genome%20amplification-induced%20bias%20through%20high-throughput,%20massively%20parallel%20whole%20genome%20sequencing.%20BMC%20Genomics.%202006;7(1):216.%2010.1186/1471-2164-7-216." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Korfhage C, Fricke E, Meier A, Parallel WGA, WTA for Comparative Genome and Transcriptome NGS Analysis Using Tiny Cell Numbers. Curr Protoc Mol Biol. 2015;111:7. 10.1002/0471142727.mb0719s111.</cite> [<a href="https://doi.org/10.1002/0471142727.mb0719s111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26131854/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Korfhage%20C,%20Fricke%20E,%20Meier%20A,%20Parallel%20WGA,%20WTA%20for%20Comparative%20Genome%20and%20Transcriptome%20NGS%20Analysis%20Using%20Tiny%20Cell%20Numbers.%20Curr%20Protoc%20Mol%20Biol.%202015;111:7.%2010.1002/0471142727.mb0719s111." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Ahsanuddin S, Afshinnekoo E, Gandara JA, Hakyemezoğlu M, Bezdan D, Minot SS, et al. Assessment of REPLI-g multiple displacement whole genome amplification (WGA) techniques for metagenomic applications. J Biomol Techniques: JBT. 2017;28 1:46–55.
</cite> [<a href="https://doi.org/10.7171/jbt.17-2801-008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5363268/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28344519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahsanuddin%20S,%20Afshinnekoo%20E,%20Gandara%20JA,%20Hakyemezo%C4%9Flu%20M,%20Bezdan%20D,%20Minot%20SS,%20et%20al.%20Assessment%20of%20REPLI-g%20multiple%20displacement%20whole%20genome%20amplification%20(WGA)%20techniques%20for%20metagenomic%20applications.%20J%20Biomol%20Techniques:%20JBT.%202017;28%201:46%E2%80%9355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Andersson D, Kebede FT, Escobar M, Österlund T, Ståhlberg A. Principles of digital sequencing using unique molecular identifiers. Mol Aspects Med. 2024;96:101253. 10.1016/j.mam.2024.101253.
</cite> [<a href="https://doi.org/10.1016/j.mam.2024.101253" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38367531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andersson%20D,%20Kebede%20FT,%20Escobar%20M,%20%C3%96sterlund%20T,%20St%C3%A5hlberg%20A.%20Principles%20of%20digital%20sequencing%20using%20unique%20molecular%20identifiers.%20Mol%20Aspects%20Med.%202024;96:101253.%2010.1016/j.mam.2024.101253." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Wenger AM, Peluso P, Rowell WJ, Chang PC, Hall RJ, Concepcion GT, et al. Accurate circular consensus long-read sequencing improves variant detection and assembly of a human genome. Nat Biotechnol. 2019;37(10):1155–62. 10.1038/s41587-019-0217-9.
</cite> [<a href="https://doi.org/10.1038/s41587-019-0217-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6776680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31406327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wenger%20AM,%20Peluso%20P,%20Rowell%20WJ,%20Chang%20PC,%20Hall%20RJ,%20Concepcion%20GT,%20et%20al.%20Accurate%20circular%20consensus%20long-read%20sequencing%20improves%20variant%20detection%20and%20assembly%20of%20a%20human%20genome.%20Nat%20Biotechnol.%202019;37(10):1155%E2%80%9362.%2010.1038/s41587-019-0217-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Sater V, Viailly PJ, Lecroq T, Ruminy P, Bérard C, Prieur-Gaston É, et al. UMI-Gen: A UMI-based read simulator for variant calling evaluation in paired-end sequencing NGS libraries. Comput Struct Biotechnol J. 2020;18:2270–80. 10.1016/j.csbj.2020.08.011.
</cite> [<a href="https://doi.org/10.1016/j.csbj.2020.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7484502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32952940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sater%20V,%20Viailly%20PJ,%20Lecroq%20T,%20Ruminy%20P,%20B%C3%A9rard%20C,%20Prieur-Gaston%20%C3%89,%20et%20al.%20UMI-Gen:%20A%20UMI-based%20read%20simulator%20for%20variant%20calling%20evaluation%20in%20paired-end%20sequencing%20NGS%20libraries.%20Comput%20Struct%20Biotechnol%20J.%202020;18:2270%E2%80%9380.%2010.1016/j.csbj.2020.08.011." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Smith T, Heger A, Sudbery I. UMI-tools: modeling sequencing errors in unique molecular identifiers to improve quantification accuracy. Genome Res. 2017;27(3):491–9. 10.1101/gr.209601.116.
</cite> [<a href="https://doi.org/10.1101/gr.209601.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5340976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28100584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith%20T,%20Heger%20A,%20Sudbery%20I.%20UMI-tools:%20modeling%20sequencing%20errors%20in%20unique%20molecular%20identifiers%20to%20improve%20quantification%20accuracy.%20Genome%20Res.%202017;27(3):491%E2%80%939.%2010.1101/gr.209601.116." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Rang FJ, Kloosterman WP, de Ridder J. From squiggle to basepair: computational approaches for improving nanopore sequencing read accuracy. Genome Biol. 2018;19(1):90. 10.1186/s13059-018-1462-9.
</cite> [<a href="https://doi.org/10.1186/s13059-018-1462-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6045860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30005597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rang%20FJ,%20Kloosterman%20WP,%20de%20Ridder%20J.%20From%20squiggle%20to%20basepair:%20computational%20approaches%20for%20improving%20nanopore%20sequencing%20read%20accuracy.%20Genome%20Biol.%202018;19(1):90.%2010.1186/s13059-018-1462-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Cheng J, Hu H, Kang Y, Chen W, Fang W, Wang K, et al. Identification of pathogens in culture-negative infective endocarditis cases by metagenomic analysis. Ann Clin Microbiol Antimicrob. 2018;17(1):43. 10.1186/s12941-018-0294-5.
</cite> [<a href="https://doi.org/10.1186/s12941-018-0294-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6300891/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30567558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng%20J,%20Hu%20H,%20Kang%20Y,%20Chen%20W,%20Fang%20W,%20Wang%20K,%20et%20al.%20Identification%20of%20pathogens%20in%20culture-negative%20infective%20endocarditis%20cases%20by%20metagenomic%20analysis.%20Ann%20Clin%20Microbiol%20Antimicrob.%202018;17(1):43.%2010.1186/s12941-018-0294-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Manrao EA, Derrington IM, Laszlo AH, Langford KW, Hopper MK, Gillgren N, et al. Reading DNA at single-nucleotide resolution with a mutant MspA nanopore and phi29 DNA polymerase. Nat Biotechnol. 2012;30(4):349–53. 10.1038/nbt.2171.
</cite> [<a href="https://doi.org/10.1038/nbt.2171" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3757088/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22446694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manrao%20EA,%20Derrington%20IM,%20Laszlo%20AH,%20Langford%20KW,%20Hopper%20MK,%20Gillgren%20N,%20et%20al.%20Reading%20DNA%20at%20single-nucleotide%20resolution%20with%20a%20mutant%20MspA%20nanopore%20and%20phi29%20DNA%20polymerase.%20Nat%20Biotechnol.%202012;30(4):349%E2%80%9353.%2010.1038/nbt.2171." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Harris PNA, Bauer MJ, Lüftinger L, Beisken S, Forde BM, Balch R, et al. Rapid nanopore sequencing and predictive susceptibility testing of positive blood cultures from intensive care patients with sepsis. Microbiol Spectr. 2024;12(2):e03065–23. 10.1128/spectrum.03065-23.
</cite> [<a href="https://doi.org/10.1128/spectrum.03065-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10846127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38193658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harris%20PNA,%20Bauer%20MJ,%20L%C3%BCftinger%20L,%20Beisken%20S,%20Forde%20BM,%20Balch%20R,%20et%20al.%20Rapid%20nanopore%20sequencing%20and%20predictive%20susceptibility%20testing%20of%20positive%20blood%20cultures%20from%20intensive%20care%20patients%20with%20sepsis.%20Microbiol%20Spectr.%202024;12(2):e03065%E2%80%9323.%2010.1128/spectrum.03065-23." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Buchan JG, White S, Joshi R, Ashley EA. Rapid genome sequencing in the critically ill. Clin Chem. 2019;65(6):723–6. 10.1373/clinchem.2018.293506.
</cite> [<a href="https://doi.org/10.1373/clinchem.2018.293506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30842082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buchan%20JG,%20White%20S,%20Joshi%20R,%20Ashley%20EA.%20Rapid%20genome%20sequencing%20in%20the%20critically%20ill.%20Clin%20Chem.%202019;65(6):723%E2%80%936.%2010.1373/clinchem.2018.293506." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Stoler N, Nekrutenko A. Sequencing error profiles of illumina sequencing instruments. NAR Genomics Bioinf. 2021;3(1). 10.1093/nargab/lqab019.</cite> [<a href="https://doi.org/10.1093/nargab/lqab019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8002175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33817639/" class="usa-link">PubMed</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Boers SA, Jansen R, Hays JP. Understanding and overcoming the pitfalls and biases of next-generation sequencing (NGS) methods for use in the routine clinical Microbiological diagnostic laboratory. Eur J Clin Microbiol Infect Dis. 2019;38(6):1059–70. 10.1007/s10096-019-03520-3.
</cite> [<a href="https://doi.org/10.1007/s10096-019-03520-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6520317/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30834996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boers%20SA,%20Jansen%20R,%20Hays%20JP.%20Understanding%20and%20overcoming%20the%20pitfalls%20and%20biases%20of%20next-generation%20sequencing%20(NGS)%20methods%20for%20use%20in%20the%20routine%20clinical%20Microbiological%20diagnostic%20laboratory.%20Eur%20J%20Clin%20Microbiol%20Infect%20Dis.%202019;38(6):1059%E2%80%9370.%2010.1007/s10096-019-03520-3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Mostafa HH. An evolution of nanopore next-generation sequencing technology: implications for medical microbiology and public health. J Clin Microbiol. 2024;62(5):e0024624. 10.1128/jcm.00246-24.
</cite> [<a href="https://doi.org/10.1128/jcm.00246-24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11077973/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38563782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mostafa%20HH.%20An%20evolution%20of%20nanopore%20next-generation%20sequencing%20technology:%20implications%20for%20medical%20microbiology%20and%20public%20health.%20J%20Clin%20Microbiol.%202024;62(5):e0024624.%2010.1128/jcm.00246-24." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>El-Lagta N, Truong L, Ayora F, Mobegi F, Bruce S, Martinez P, et al. Revolutionising high resolution HLA genotyping for transplant assessment: validation, implementation and challenges of Oxford nanopore technologies’ Q20 + Sequencing. HLA. 2024;104(4):e15725. 10.1111/tan.15725.
</cite> [<a href="https://doi.org/10.1111/tan.15725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39435968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?El-Lagta%20N,%20Truong%20L,%20Ayora%20F,%20Mobegi%20F,%20Bruce%20S,%20Martinez%20P,%20et%20al.%20Revolutionising%20high%20resolution%20HLA%20genotyping%20for%20transplant%20assessment:%20validation,%20implementation%20and%20challenges%20of%20Oxford%20nanopore%20technologies%E2%80%99%20Q20%E2%80%89+%E2%80%89Sequencing.%20HLA.%202024;104(4):e15725.%2010.1111/tan.15725." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Linde J, Brangsch H, Hölzer M, Thomas C, Elschner MC, Melzer F, et al. Comparison of illumina and Oxford nanopore technology for genome analysis of Francisella tularensis, Bacillus anthracis, and Brucella suis. BMC Genomics. 2023;24(1):258. 10.1186/s12864-023-09343-z.
</cite> [<a href="https://doi.org/10.1186/s12864-023-09343-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10182678/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37173617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Linde%20J,%20Brangsch%20H,%20H%C3%B6lzer%20M,%20Thomas%20C,%20Elschner%20MC,%20Melzer%20F,%20et%20al.%20Comparison%20of%20illumina%20and%20Oxford%20nanopore%20technology%20for%20genome%20analysis%20of%20Francisella%20tularensis,%20Bacillus%20anthracis,%20and%20Brucella%20suis.%20BMC%20Genomics.%202023;24(1):258.%2010.1186/s12864-023-09343-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Chiu CY, Miller SA. Clinical metagenomics. Nat Rev Genet. 2019;20(6):341–55. 10.1038/s41576-019-0113-7.
</cite> [<a href="https://doi.org/10.1038/s41576-019-0113-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6858796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30918369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chiu%20CY,%20Miller%20SA.%20Clinical%20metagenomics.%20Nat%20Rev%20Genet.%202019;20(6):341%E2%80%9355.%2010.1038/s41576-019-0113-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Yang H, Xu N, Yan M, Yang L, Wen S, Wang S, et al. Comparison of metagenomic next-generation sequencing and conventional culture for the diagnostic performance in febrile patients with suspected infections. BMC Infect Dis. 2024;24(1):350. 10.1186/s12879-024-09236-w.
</cite> [<a href="https://doi.org/10.1186/s12879-024-09236-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10964532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38532348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20H,%20Xu%20N,%20Yan%20M,%20Yang%20L,%20Wen%20S,%20Wang%20S,%20et%20al.%20Comparison%20of%20metagenomic%20next-generation%20sequencing%20and%20conventional%20culture%20for%20the%20diagnostic%20performance%20in%20febrile%20patients%20with%20suspected%20infections.%20BMC%20Infect%20Dis.%202024;24(1):350.%2010.1186/s12879-024-09236-w." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Gu W, Miller S, Chiu CY. Clinical metagenomic Next-Generation sequencing for pathogen detection. Annu Rev Pathol. 2019;14:319–38. 10.1146/annurev-pathmechdis-012418-012751.
</cite> [<a href="https://doi.org/10.1146/annurev-pathmechdis-012418-012751" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6345613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30355154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gu%20W,%20Miller%20S,%20Chiu%20CY.%20Clinical%20metagenomic%20Next-Generation%20sequencing%20for%20pathogen%20detection.%20Annu%20Rev%20Pathol.%202019;14:319%E2%80%9338.%2010.1146/annurev-pathmechdis-012418-012751." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Zhang L, Chen T, Wang Y, Zhang S, Lv Q, Kong D, et al. Comparison analysis of different DNA extraction methods on suitability for Long-Read metagenomic nanopore sequencing. Front Cell Infect Microbiol. 2022;12–2022. 10.3389/fcimb.2022.919903.</cite> [<a href="https://doi.org/10.3389/fcimb.2022.919903" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9273838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35837476/" class="usa-link">PubMed</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Li H, Gao H, Meng H, Wang Q, Li S, Chen H, et al. Detection of pulmonary infectious pathogens from lung biopsy tissues by metagenomic Next-Generation sequencing. Front Cell Infect Microbiol. 2018;8:205. 10.3389/fcimb.2018.00205.
</cite> [<a href="https://doi.org/10.3389/fcimb.2018.00205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6026637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29988504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H,%20Gao%20H,%20Meng%20H,%20Wang%20Q,%20Li%20S,%20Chen%20H,%20et%20al.%20Detection%20of%20pulmonary%20infectious%20pathogens%20from%20lung%20biopsy%20tissues%20by%20metagenomic%20Next-Generation%20sequencing.%20Front%20Cell%20Infect%20Microbiol.%202018;8:205.%2010.3389/fcimb.2018.00205." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Zhang P, Chen Y, Li S, Li C, Zhang S, Zheng W, et al. Metagenomic next-generation sequencing for the clinical diagnosis and prognosis of acute respiratory distress syndrome caused by severe pneumonia: a retrospective study. PeerJ. 2020;8:e9623. 10.7717/peerj.9623.
</cite> [<a href="https://doi.org/10.7717/peerj.9623" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7395598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32821543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20P,%20Chen%20Y,%20Li%20S,%20Li%20C,%20Zhang%20S,%20Zheng%20W,%20et%20al.%20Metagenomic%20next-generation%20sequencing%20for%20the%20clinical%20diagnosis%20and%20prognosis%20of%20acute%20respiratory%20distress%20syndrome%20caused%20by%20severe%20pneumonia:%20a%20retrospective%20study.%20PeerJ.%202020;8:e9623.%2010.7717/peerj.9623." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>Chen Y, Chen C, Chen W, Gai W, Zheng Y, Guo Y, et al. Clinical application of metagenomic Next-Generation sequencing in sepsis patients with early antibiotic treatment. Infect Drug Resist. 2024;17:4695–706. 10.2147/idr.S485102.
</cite> [<a href="https://doi.org/10.2147/IDR.S485102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11523945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39479400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20Y,%20Chen%20C,%20Chen%20W,%20Gai%20W,%20Zheng%20Y,%20Guo%20Y,%20et%20al.%20Clinical%20application%20of%20metagenomic%20Next-Generation%20sequencing%20in%20sepsis%20patients%20with%20early%20antibiotic%20treatment.%20Infect%20Drug%20Resist.%202024;17:4695%E2%80%93706.%2010.2147/idr.S485102." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR109">
<span class="label">109.</span><cite>Chen Y, Wang J, Niu T. Clinical and diagnostic values of metagenomic next-generation sequencing for infection in hematology patients: a systematic review and meta-analysis. BMC Infect Dis. 2024;24(1):167. 10.1186/s12879-024-09073-x.
</cite> [<a href="https://doi.org/10.1186/s12879-024-09073-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10848439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38326763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20Y,%20Wang%20J,%20Niu%20T.%20Clinical%20and%20diagnostic%20values%20of%20metagenomic%20next-generation%20sequencing%20for%20infection%20in%20hematology%20patients:%20a%20systematic%20review%20and%20meta-analysis.%20BMC%20Infect%20Dis.%202024;24(1):167.%2010.1186/s12879-024-09073-x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>Wu J, Song W, Yan H, Luo C, Hu W, Xie L, et al. Metagenomic next-generation sequencing in detecting pathogens in pediatric oncology patients with suspected bloodstream infections. Pediatr Res. 2024;95(3):843–51. 10.1038/s41390-023-02776-y.
</cite> [<a href="https://doi.org/10.1038/s41390-023-02776-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10899103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37857845/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20J,%20Song%20W,%20Yan%20H,%20Luo%20C,%20Hu%20W,%20Xie%20L,%20et%20al.%20Metagenomic%20next-generation%20sequencing%20in%20detecting%20pathogens%20in%20pediatric%20oncology%20patients%20with%20suspected%20bloodstream%20infections.%20Pediatr%20Res.%202024;95(3):843%E2%80%9351.%2010.1038/s41390-023-02776-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite>Diao Z, Lai H, Han D, Yang B, Zhang R, Li J. Validation of a metagenomic Next-Generation sequencing assay for lower respiratory pathogen detection. Microbiol Spectr. 2023;11(1):e03812–22. 10.1128/spectrum.03812-22.
</cite> [<a href="https://doi.org/10.1128/spectrum.03812-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9927246/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36507666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diao%20Z,%20Lai%20H,%20Han%20D,%20Yang%20B,%20Zhang%20R,%20Li%20J.%20Validation%20of%20a%20metagenomic%20Next-Generation%20sequencing%20assay%20for%20lower%20respiratory%20pathogen%20detection.%20Microbiol%20Spectr.%202023;11(1):e03812%E2%80%9322.%2010.1128/spectrum.03812-22." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite>Han D, Diao Z, Lai H, Han Y, Xie J, Zhang R, et al. Multilaboratory assessment of metagenomic next-generation sequencing for unbiased microbe detection. J Adv Res. 2022;38:213–22. 10.1016/j.jare.2021.09.011.
</cite> [<a href="https://doi.org/10.1016/j.jare.2021.09.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9091723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35572414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han%20D,%20Diao%20Z,%20Lai%20H,%20Han%20Y,%20Xie%20J,%20Zhang%20R,%20et%20al.%20Multilaboratory%20assessment%20of%20metagenomic%20next-generation%20sequencing%20for%20unbiased%20microbe%20detection.%20J%20Adv%20Res.%202022;38:213%E2%80%9322.%2010.1016/j.jare.2021.09.011." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR113">
<span class="label">113.</span><cite>Ndlovu H, Lawal IO, Mokoala KMG, Mdanda S, Sathekge MM. New promising targets for imaging in infection. Semin Nucl Med. 2025. 10.1053/j.semnuclmed.2025.06.013.
</cite> [<a href="https://doi.org/10.1053/j.semnuclmed.2025.06.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40701865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ndlovu%20H,%20Lawal%20IO,%20Mokoala%20KMG,%20Mdanda%20S,%20Sathekge%20MM.%20New%20promising%20targets%20for%20imaging%20in%20infection.%20Semin%20Nucl%20Med.%202025.%2010.1053/j.semnuclmed.2025.06.013." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR114">
<span class="label">114.</span><cite>Burban A, Słupik D, Reda A, Szczerba E, Grabowski M, Kołodzińska A. Novel diagnostic methods for infective endocarditis. Int J Mol Sci. 2024;25(2). 10.3390/ijms25021245.</cite> [<a href="https://doi.org/10.3390/ijms25021245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10816594/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38279244/" class="usa-link">PubMed</a>]</li>
<li id="CR115">
<span class="label">115.</span><cite>Zeng X, Liang Y, Wang H, Chen J, Xu Y, Ou Q, et al. Detection of pathogens from venous or arterial blood of patients with left-sided infective endocarditis by metagenomic next-generation sequencing: A prospective study. Clin Chim Acta. 2024;552:117698. 10.1016/j.cca.2023.117698.
</cite> [<a href="https://doi.org/10.1016/j.cca.2023.117698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38072301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng%20X,%20Liang%20Y,%20Wang%20H,%20Chen%20J,%20Xu%20Y,%20Ou%20Q,%20et%20al.%20Detection%20of%20pathogens%20from%20venous%20or%20arterial%20blood%20of%20patients%20with%20left-sided%20infective%20endocarditis%20by%20metagenomic%20next-generation%20sequencing:%20A%20prospective%20study.%20Clin%20Chim%20Acta.%202024;552:117698.%2010.1016/j.cca.2023.117698." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR116">
<span class="label">116.</span><cite>Vazquez V, Bermudez-Rivera L, Neto A, Perez V, Cadilla A, Sande L. The cruciality of increasing index of suspicion for atypical Bartonella Henselae in pediatric patients: A case series. IDCases. 2025;40:e02192. 10.1016/j.idcr.2025.e02192.
</cite> [<a href="https://doi.org/10.1016/j.idcr.2025.e02192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11932680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40129759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vazquez%20V,%20Bermudez-Rivera%20L,%20Neto%20A,%20Perez%20V,%20Cadilla%20A,%20Sande%20L.%20The%20cruciality%20of%20increasing%20index%20of%20suspicion%20for%20atypical%20Bartonella%20Henselae%20in%20pediatric%20patients:%20A%20case%20series.%20IDCases.%202025;40:e02192.%2010.1016/j.idcr.2025.e02192." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR117">
<span class="label">117.</span><cite>Santhanam H, Nguyen MHN, Muthukumarasamy N, Mehta A, Francisco MT, Fountain RR, et al. Bartonella endocarditis in patients with right ventricle-to-pulmonary artery conduit: 2 case reports and literature review. IDCases. 2021;26:e01306. 10.1016/j.idcr.2021.e01306.
</cite> [<a href="https://doi.org/10.1016/j.idcr.2021.e01306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8536538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34722156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Santhanam%20H,%20Nguyen%20MHN,%20Muthukumarasamy%20N,%20Mehta%20A,%20Francisco%20MT,%20Fountain%20RR,%20et%20al.%20Bartonella%20endocarditis%20in%20patients%20with%20right%20ventricle-to-pulmonary%20artery%20conduit:%202%20case%20reports%20and%20literature%20review.%20IDCases.%202021;26:e01306.%2010.1016/j.idcr.2021.e01306." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR118">
<span class="label">118.</span><cite>Berinson B, Tanida K, Wolters M, Olearo F, Sinning C, Christner M et al. Comparative evaluation of biofire joint infection panel, sepsitest 16S/18S rDNA PCR, and culture to identify microorganisms in explanted heart valves. Microbiol Spectr 0(0):e01263–25. 10.1128/spectrum.01263-25</cite> [<a href="https://doi.org/10.1128/spectrum.01263-25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40693950/" class="usa-link">PubMed</a>]</li>
<li id="CR119">
<span class="label">119.</span><cite>Hajduczenia MM, Klefisch FR, Hopf AGM, Grubitzsch H, Stegemann MS, Pfäfflin F, et al. New perspectives for prosthetic valve endocarditis: impact of molecular imaging by FISHseq diagnostics. Clin Infect Diseases: Official Publication Infect Dis Soc Am. 2023;76(6):1050–8. 10.1093/cid/ciac860.</cite> [<a href="https://doi.org/10.1093/cid/ciac860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10029985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36318608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hajduczenia%20MM,%20Klefisch%20FR,%20Hopf%20AGM,%20Grubitzsch%20H,%20Stegemann%20MS,%20Pf%C3%A4fflin%20F,%20et%20al.%20New%20perspectives%20for%20prosthetic%20valve%20endocarditis:%20impact%20of%20molecular%20imaging%20by%20FISHseq%20diagnostics.%20Clin%20Infect%20Diseases:%20Official%20Publication%20Infect%20Dis%20Soc%20Am.%202023;76(6):1050%E2%80%938.%2010.1093/cid/ciac860." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR120">
<span class="label">120.</span><cite>Kocazeybek BKS, Oner YA. Biomarkers for the diagnosis of infective endocarditis. Circulation Korea. 2009;18:386–94. 4. doi: circulation.or.kr.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kocazeybek%20BKS,%20Oner%20YA.%20Biomarkers%20for%20the%20diagnosis%20of%20infective%20endocarditis.%20Circulation%20Korea.%202009;18:386%E2%80%9394.%204.%20doi:%20circulation.or.kr." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR121">
<span class="label">121.</span><cite>Huang S, Zhou Z, Luo L, Yue Y, Liu Q, Feng K, et al. Preoperative serum albumin: a promising indicator of early mortality after surgery for infective endocarditis. Annals Translational Med. 2021;9(18):1445. 10.21037/atm-21-3913.</cite> [<a href="https://doi.org/10.21037/atm-21-3913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8506743/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34733997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20S,%20Zhou%20Z,%20Luo%20L,%20Yue%20Y,%20Liu%20Q,%20Feng%20K,%20et%20al.%20Preoperative%20serum%20albumin:%20a%20promising%20indicator%20of%20early%20mortality%20after%20surgery%20for%20infective%20endocarditis.%20Annals%20Translational%20Med.%202021;9(18):1445.%2010.21037/atm-21-3913." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR122">
<span class="label">122.</span><cite>Byrne RA, Rossello X, Coughlan JJ, Barbato E, Berry C, Chieffo A, et al. 2023 ESC guidelines for the management of acute coronary syndromes: developed by the task force on the management of acute coronary syndromes of the European society of cardiology (ESC). Eur Heart J. 2023;44(38):3720–826. 10.1093/eurheartj/ehad191.
</cite> [<a href="https://doi.org/10.1093/eurheartj/ehad191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37622654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Byrne%20RA,%20Rossello%20X,%20Coughlan%20JJ,%20Barbato%20E,%20Berry%20C,%20Chieffo%20A,%20et%20al.%202023%20ESC%20guidelines%20for%20the%20management%20of%20acute%20coronary%20syndromes:%20developed%20by%20the%20task%20force%20on%20the%20management%20of%20acute%20coronary%20syndromes%20of%20the%20European%20society%20of%20cardiology%20(ESC).%20Eur%20Heart%20J.%202023;44(38):3720%E2%80%93826.%2010.1093/eurheartj/ehad191." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR123">
<span class="label">123.</span><cite>Zhang Y, YZ YL, Wu aX. Surgical treatment of aortic annulus damaged by infective endocarditis: A Single-Center experience. Cardiovasc Innovations Appl. 2024;9:71. 10.15212/CVIA.2024.0053.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Y,%20YZ%20YL,%20Wu%20aX.%20Surgical%20treatment%20of%20aortic%20annulus%20damaged%20by%20infective%20endocarditis:%20A%20Single-Center%20experience.%20Cardiovasc%20Innovations%20Appl.%202024;9:71.%2010.15212/CVIA.2024.0053." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR124">
<span class="label">124.</span><cite>Teixeira-Antunes A, Zimmermann V, Fourré N, Ianculescu N, Monney P, Tzimas G, et al. Factors associated with persistent bacteraemia among patients with suspected infective endocarditis. Infection. 2025. 10.1007/s15010-025-02537-5.
</cite> [<a href="https://doi.org/10.1007/s15010-025-02537-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40192959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Teixeira-Antunes%20A,%20Zimmermann%20V,%20Fourr%C3%A9%20N,%20Ianculescu%20N,%20Monney%20P,%20Tzimas%20G,%20et%20al.%20Factors%20associated%20with%20persistent%20bacteraemia%20among%20patients%20with%20suspected%20infective%20endocarditis.%20Infection.%202025.%2010.1007/s15010-025-02537-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR125">
<span class="label">125.</span><cite>Athanasopoulou K, Michalopoulou V-I, Scorilas A, Adamopoulos PG. Integrating artificial intelligence in Next-Generation sequencing: advances, challenges, and future directions. Curr Issues Mol Biol. 2025;47(6):470.
</cite> [<a href="https://doi.org/10.3390/cimb47060470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12191491/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40699869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Athanasopoulou%20K,%20Michalopoulou%20V-I,%20Scorilas%20A,%20Adamopoulos%20PG.%20Integrating%20artificial%20intelligence%20in%20Next-Generation%20sequencing:%20advances,%20challenges,%20and%20future%20directions.%20Curr%20Issues%20Mol%20Biol.%202025;47(6):470." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR126">
<span class="label">126.</span><cite>Lai C, Yin M, Kholmovski EG, Popescu DM, Lu D-Y, Scherer E, et al. Multimodal AI to forecast arrhythmic death in hypertrophic cardiomyopathy. Nat Cardiovasc Res. 2025;4(7):891–903. 10.1038/s44161-025-00679-1.
</cite> [<a href="https://doi.org/10.1038/s44161-025-00679-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12259465/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40603582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lai%20C,%20Yin%20M,%20Kholmovski%20EG,%20Popescu%20DM,%20Lu%20D-Y,%20Scherer%20E,%20et%20al.%20Multimodal%20AI%20to%20forecast%20arrhythmic%20death%20in%20hypertrophic%20cardiomyopathy.%20Nat%20Cardiovasc%20Res.%202025;4(7):891%E2%80%93903.%2010.1038/s44161-025-00679-1." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR127">
<span class="label">127.</span><cite>Poterucha TJ, Jing L, Ricart RP, Adjei-Mosi M, Finer J, Hartzel D, et al. Detecting structural heart disease from electrocardiograms using AI. Nature. 2025. 10.1038/s41586-025-09227-0.
</cite> [<a href="https://doi.org/10.1038/s41586-025-09227-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12328201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40670798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poterucha%20TJ,%20Jing%20L,%20Ricart%20RP,%20Adjei-Mosi%20M,%20Finer%20J,%20Hartzel%20D,%20et%20al.%20Detecting%20structural%20heart%20disease%20from%20electrocardiograms%20using%20AI.%20Nature.%202025.%2010.1038/s41586-025-09227-0." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR128">
<span class="label">128.</span><cite>Ouanes K. Chapter 27 - The role of artificial intelligence and machine learning in NGS. In: Dash HR, Elkins KM, Al-Snan NR, editors. Next generation sequencing (NGS) technology in DNA analysis. Academic; 2024. pp. 531–8.</cite>
</li>
<li id="CR129">
<span class="label">129.</span><cite>Aalam J, Ahmad Shah SN, Parveen R. An extensive review on infectious disease diagnosis using machine learning techniques and next generation sequencing: State-of-the-art and perspectives. Comput Biol Med. 2025;189:109962. 10.1016/j.compbiomed.2025.109962.
</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2025.109962" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40054170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aalam%20J,%20Ahmad%20Shah%20SN,%20Parveen%20R.%20An%20extensive%20review%20on%20infectious%20disease%20diagnosis%20using%20machine%20learning%20techniques%20and%20next%20generation%20sequencing:%20State-of-the-art%20and%20perspectives.%20Comput%20Biol%20Med.%202025;189:109962.%2010.1016/j.compbiomed.2025.109962." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR130">
<span class="label">130.</span><cite>Canales-Fiscal MR, Tamez-Peña JG. Hybrid morphological-convolutional neural networks for computer-aided diagnosis. Front Artif Intell. 2023;6–2023. 10.3389/frai.2023.1253183.</cite> [<a href="https://doi.org/10.3389/frai.2023.1253183" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10546173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37795497/" class="usa-link">PubMed</a>]</li>
<li id="CR131">
<span class="label">131.</span><cite>Haruna Y, Qin S, Adama Chukkol AH, Yusuf AA, Bello I, Lawan A. Exploring the synergies of hybrid convolutional neural network and vision transformer architectures for computer vision: A survey. Eng Appl Artif Intell. 2025;144:110057. 10.1016/j.engappai.2025.110057.</cite> [<a href="https://scholar.google.com/scholar_lookup?Haruna%20Y,%20Qin%20S,%20Adama%20Chukkol%20AH,%20Yusuf%20AA,%20Bello%20I,%20Lawan%20A.%20Exploring%20the%20synergies%20of%20hybrid%20convolutional%20neural%20network%20and%20vision%20transformer%20architectures%20for%20computer%20vision:%20A%20survey.%20Eng%20Appl%20Artif%20Intell.%202025;144:110057.%2010.1016/j.engappai.2025.110057." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR132">
<span class="label">132.</span><cite>Alshraideh A, Al Fayoumi B, Alshraideh BM, Alshraideh M. Hybrid AI framework for the early detection of heart failure: integrating traditional machine learning and generative Language models with clinical data. Cureus. 2025;17(6):e85638. 10.7759/cureus.85638.
</cite> [<a href="https://doi.org/10.7759/cureus.85638" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12240570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40636608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alshraideh%20A,%20Al%20Fayoumi%20B,%20Alshraideh%20BM,%20Alshraideh%20M.%20Hybrid%20AI%20framework%20for%20the%20early%20detection%20of%20heart%20failure:%20integrating%20traditional%20machine%20learning%20and%20generative%20Language%20models%20with%20clinical%20data.%20Cureus.%202025;17(6):e85638.%2010.7759/cureus.85638." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR133">
<span class="label">133.</span><cite>Pinna A, Boi A, Mannelli L, Balestrieri A, Sanfilippo R, Suri J, et al. Machine learning for coronary plaque characterization: A multimodal review of OCT, IVUS, and CCTA. Diagnostics. 2025;15(14):1822.
</cite> [<a href="https://doi.org/10.3390/diagnostics15141822" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12293362/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40722571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pinna%20A,%20Boi%20A,%20Mannelli%20L,%20Balestrieri%20A,%20Sanfilippo%20R,%20Suri%20J,%20et%20al.%20Machine%20learning%20for%20coronary%20plaque%20characterization:%20A%20multimodal%20review%20of%20OCT,%20IVUS,%20and%20CCTA.%20Diagnostics.%202025;15(14):1822." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR134">
<span class="label">134.</span><cite>Xu J, Zhou P, Liu J, Zhao L, Fu H, Han Q, et al. Utilizing metagenomic Next-Generation sequencing (mNGS) for rapid pathogen identification and to inform clinical Decision-Making: results from a large Real-World cohort. Infect Dis Ther. 2023;12(4):1175–87. 10.1007/s40121-023-00790-5.
</cite> [<a href="https://doi.org/10.1007/s40121-023-00790-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10147866/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36988865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20J,%20Zhou%20P,%20Liu%20J,%20Zhao%20L,%20Fu%20H,%20Han%20Q,%20et%20al.%20Utilizing%20metagenomic%20Next-Generation%20sequencing%20(mNGS)%20for%20rapid%20pathogen%20identification%20and%20to%20inform%20clinical%20Decision-Making:%20results%20from%20a%20large%20Real-World%20cohort.%20Infect%20Dis%20Ther.%202023;12(4):1175%E2%80%9387.%2010.1007/s40121-023-00790-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR135">
<span class="label">135.</span><cite>Huang J, Jiang E, Yang D, Wei J, Zhao M, Feng J, et al. Metagenomic Next-Generation sequencing versus traditional pathogen detection in the diagnosis of peripheral pulmonary infectious lesions. Infect Drug Resist. 2020;13:567–76. 10.2147/idr.S235182.
</cite> [<a href="https://doi.org/10.2147/IDR.S235182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7036976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32110067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20J,%20Jiang%20E,%20Yang%20D,%20Wei%20J,%20Zhao%20M,%20Feng%20J,%20et%20al.%20Metagenomic%20Next-Generation%20sequencing%20versus%20traditional%20pathogen%20detection%20in%20the%20diagnosis%20of%20peripheral%20pulmonary%20infectious%20lesions.%20Infect%20Drug%20Resist.%202020;13:567%E2%80%9376.%2010.2147/idr.S235182." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR136">
<span class="label">136.</span><cite>Wang J, Han Y, Feng J. Metagenomic next-generation sequencing for mixed pulmonary infection diagnosis. BMC Pulm Med. 2019;19(1):252. 10.1186/s12890-019-1022-4.
</cite> [<a href="https://doi.org/10.1186/s12890-019-1022-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6921575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31856779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20J,%20Han%20Y,%20Feng%20J.%20Metagenomic%20next-generation%20sequencing%20for%20mixed%20pulmonary%20infection%20diagnosis.%20BMC%20Pulm%20Med.%202019;19(1):252.%2010.1186/s12890-019-1022-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR137">
<span class="label">137.</span><cite>Yang S, Rothman RE. PCR-based diagnostics for infectious diseases: uses, limitations, and future applications in acute-care settings. Lancet Infect Dis. 2004;4(6):337–48. 10.1016/s1473-3099(04)01044-8.
</cite> [<a href="https://doi.org/10.1016/S1473-3099(04)01044-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7106425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15172342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20S,%20Rothman%20RE.%20PCR-based%20diagnostics%20for%20infectious%20diseases:%20uses,%20limitations,%20and%20future%20applications%20in%20acute-care%20settings.%20Lancet%20Infect%20Dis.%202004;4(6):337%E2%80%9348.%2010.1016/s1473-3099(04)01044-8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR138">
<span class="label">138.</span><cite>Cai Y, Ding H, Chen X, Chen Y, Huang C, Zhang C, et al. Optimization and standardization of mNGS-based procedures for the diagnosis of Mycoplasma periprosthetic joint infection: A novel diagnostic strategy for rare bacterial periprosthetic joint infection. Front Cell Infect Microbiol. 2023;13:1089919. 10.3389/fcimb.2023.1089919.
</cite> [<a href="https://doi.org/10.3389/fcimb.2023.1089919" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10014592/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36936762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cai%20Y,%20Ding%20H,%20Chen%20X,%20Chen%20Y,%20Huang%20C,%20Zhang%20C,%20et%20al.%20Optimization%20and%20standardization%20of%20mNGS-based%20procedures%20for%20the%20diagnosis%20of%20Mycoplasma%20periprosthetic%20joint%20infection:%20A%20novel%20diagnostic%20strategy%20for%20rare%20bacterial%20periprosthetic%20joint%20infection.%20Front%20Cell%20Infect%20Microbiol.%202023;13:1089919.%2010.3389/fcimb.2023.1089919." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR139">
<span class="label">139.</span><cite>Omura C, de Lorenzi-Tognon M, Benoit P, Servellita V, Foresythe K, Brazer N, et al. 298. Host response profiling from clinical metagenomic sequencing data for diagnosis of central nervous system infections. Open Forum Infect Dis. 2025;12(Supplement_1). 10.1093/ofid/ofae631.088.</cite>
</li>
<li id="CR140">
<span class="label">140.</span><cite>Abavisani M, Khoshrou A, Foroushan SK, Ebadpour N, Sahebkar A. Deciphering the gut microbiome: the revolution of artificial intelligence in microbiota analysis and intervention. Curr Res Biotechnol. 2024;7:100211. 10.1016/j.crbiot.2024.100211.</cite> [<a href="https://scholar.google.com/scholar_lookup?Abavisani%20M,%20Khoshrou%20A,%20Foroushan%20SK,%20Ebadpour%20N,%20Sahebkar%20A.%20Deciphering%20the%20gut%20microbiome:%20the%20revolution%20of%20artificial%20intelligence%20in%20microbiota%20analysis%20and%20intervention.%20Curr%20Res%20Biotechnol.%202024;7:100211.%2010.1016/j.crbiot.2024.100211." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR141">
<span class="label">141.</span><cite>de Vries JJC, Brown JR, Couto N, Beer M, Le Mercier P, Sidorov I, et al. Recommendations for the introduction of metagenomic next-generation sequencing in clinical virology, part II: bioinformatic analysis and reporting. J Clin Virol. 2021;138:104812. 10.1016/j.jcv.2021.104812.
</cite> [<a href="https://doi.org/10.1016/j.jcv.2021.104812" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33819811/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Vries%20JJC,%20Brown%20JR,%20Couto%20N,%20Beer%20M,%20Le%20Mercier%20P,%20Sidorov%20I,%20et%20al.%20Recommendations%20for%20the%20introduction%20of%20metagenomic%20next-generation%20sequencing%20in%20clinical%20virology,%20part%20II:%20bioinformatic%20analysis%20and%20reporting.%20J%20Clin%20Virol.%202021;138:104812.%2010.1016/j.jcv.2021.104812." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR142">
<span class="label">142.</span><cite>Xiang C, Wu X, Li T, Tang X, Zhang Y, Zeng F, et al. Effect of metagenomic next-generation sequencing on clinical outcomes in adults with severe pneumonia post-cardiac surgery: a single-center retrospective study. Sci Rep. 2024;14(1):28907. 10.1038/s41598-024-79843-9.
</cite> [<a href="https://doi.org/10.1038/s41598-024-79843-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11582676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39572618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiang%20C,%20Wu%20X,%20Li%20T,%20Tang%20X,%20Zhang%20Y,%20Zeng%20F,%20et%20al.%20Effect%20of%20metagenomic%20next-generation%20sequencing%20on%20clinical%20outcomes%20in%20adults%20with%20severe%20pneumonia%20post-cardiac%20surgery:%20a%20single-center%20retrospective%20study.%20Sci%20Rep.%202024;14(1):28907.%2010.1038/s41598-024-79843-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR143">
<span class="label">143.</span><cite>Tian Y, Gao R, Wang Y, Zhou Y, Xu S, Duan Y, et al. Economic impact of metagenomic next-generation sequencing versus traditional bacterial culture for postoperative central nervous system infections using a decision analysis mode: study protocol for a randomized controlled trial. mSystems. 2023;8(6):e0058123. 10.1128/msystems.00581-23.
</cite> [<a href="https://doi.org/10.1128/msystems.00581-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10734456/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37937972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tian%20Y,%20Gao%20R,%20Wang%20Y,%20Zhou%20Y,%20Xu%20S,%20Duan%20Y,%20et%20al.%20Economic%20impact%20of%20metagenomic%20next-generation%20sequencing%20versus%20traditional%20bacterial%20culture%20for%20postoperative%20central%20nervous%20system%20infections%20using%20a%20decision%20analysis%20mode:%20study%20protocol%20for%20a%20randomized%20controlled%20trial.%20mSystems.%202023;8(6):e0058123.%2010.1128/msystems.00581-23." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR144">
<span class="label">144.</span><cite>Xiao Y-H, Liu M-F, Wu H-W, Xu D-R, Zhang W, Zhao R. Clinical efficacy and diagnostic value of metagenomic Next- generation sequencing for rapid pathogen detection in patients with suspected infectious diseases. 2022.</cite> [<a href="https://doi.org/10.2147/IDR.S401707" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10066896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37016633/" class="usa-link">PubMed</a>]</li>
<li id="CR145">
<span class="label">145.</span><cite>Nafea AM, Wang Y, Wang D, Salama AM, Aziz MA, Xu S, et al. Application of next-generation sequencing to identify different pathogens. Front Microbiol. 2024;14. 10.3389/fmicb.2023.1329330.</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1329330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10859930/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38348304/" class="usa-link">PubMed</a>]</li>
<li id="CR146">
<span class="label">146.</span><cite>Smits P, Champagne F. Governance of health research funding institutions: an integrated conceptual framework and actionable functions of governance. Health Res Policy Syst. 2020;18(1):22. 10.1186/s12961-020-0525-z.
</cite> [<a href="https://doi.org/10.1186/s12961-020-0525-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7029568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32070372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smits%20P,%20Champagne%20F.%20Governance%20of%20health%20research%20funding%20institutions:%20an%20integrated%20conceptual%20framework%20and%20actionable%20functions%20of%20governance.%20Health%20Res%20Policy%20Syst.%202020;18(1):22.%2010.1186/s12961-020-0525-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR147">
<span class="label">147.</span><cite>Zhang D, Cheng Y, Ji Y, Miao Q, Chen B, Wang J, et al. A training program for improving the capacity of infection high-throughput sequencing and diagnosis in China. BMC Med Educ. 2024;24(1):142. 10.1186/s12909-024-05100-2.
</cite> [<a href="https://doi.org/10.1186/s12909-024-05100-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10865699/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38355463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20D,%20Cheng%20Y,%20Ji%20Y,%20Miao%20Q,%20Chen%20B,%20Wang%20J,%20et%20al.%20A%20training%20program%20for%20improving%20the%20capacity%20of%20infection%20high-throughput%20sequencing%20and%20diagnosis%20in%20China.%20BMC%20Med%20Educ.%202024;24(1):142.%2010.1186/s12909-024-05100-2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR148">
<span class="label">148.</span><cite>Salter SJ, Cox MJ, Turek EM, Calus ST, Cookson WO, Moffatt MF, et al. Reagent and laboratory contamination can critically impact sequence-based Microbiome analyses. BMC Biol. 2014;12(1):87. 10.1186/s12915-014-0087-z.
</cite> [<a href="https://doi.org/10.1186/s12915-014-0087-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4228153/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25387460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Salter%20SJ,%20Cox%20MJ,%20Turek%20EM,%20Calus%20ST,%20Cookson%20WO,%20Moffatt%20MF,%20et%20al.%20Reagent%20and%20laboratory%20contamination%20can%20critically%20impact%20sequence-based%20Microbiome%20analyses.%20BMC%20Biol.%202014;12(1):87.%2010.1186/s12915-014-0087-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>No datasets were generated or analysed during the current study.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Journal of Epidemiology and Global Health are provided here courtesy of <strong>Springer</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1007/s44197-025-00455-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/44197_2025_Article_455.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373624/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373624/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373624%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373624/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373624/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373624/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40847264/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373624/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40847264/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373624/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373624/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="2K2cuQhP4GAcZ6DMb2EOXJqoeVDVwIBeHDSFjVnqisea3fXidsMkDC9QbvdUd6WC">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
