
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            LABind: identifying protein binding ligand-aware sites via learning interactions between ligand and protein - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53436C8AF401B305436C0022DA6250.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365077/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="LABind: identifying protein binding ligand-aware sites via learning interactions between ligand and protein">
<meta name="citation_author" content="Zhijun Zhang">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Soochow University, Suzhou, China">
<meta name="citation_author" content="Lijun Quan">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Soochow University, Suzhou, China">
<meta name="citation_author_institution" content="Collaborative Innovation Center of Novel Software Technology and Industrialization, Nanjing, China">
<meta name="citation_author_institution" content="Jiangsu Key Laboratory of Drug Discovery and Translational Research for Brain Diseases, Soochow University, Suzhou, China">
<meta name="citation_author" content="Junkai Wang">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Soochow University, Suzhou, China">
<meta name="citation_author" content="Liangchen Peng">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Soochow University, Suzhou, China">
<meta name="citation_author" content="Qiufeng Chen">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Soochow University, Suzhou, China">
<meta name="citation_author" content="Bei Zhang">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Soochow University, Suzhou, China">
<meta name="citation_author" content="Lexin Cao">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Soochow University, Suzhou, China">
<meta name="citation_author" content="Yelu Jiang">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Soochow University, Suzhou, China">
<meta name="citation_author" content="Geng Li">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Soochow University, Suzhou, China">
<meta name="citation_author" content="Liangpeng Nie">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Soochow University, Suzhou, China">
<meta name="citation_author" content="Tingfang Wu">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Soochow University, Suzhou, China">
<meta name="citation_author_institution" content="Collaborative Innovation Center of Novel Software Technology and Industrialization, Nanjing, China">
<meta name="citation_author_institution" content="Jiangsu Key Laboratory of Drug Discovery and Translational Research for Brain Diseases, Soochow University, Suzhou, China">
<meta name="citation_author" content="Qiang Lyu">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Soochow University, Suzhou, China">
<meta name="citation_author_institution" content="Collaborative Innovation Center of Novel Software Technology and Industrialization, Nanjing, China">
<meta name="citation_author_institution" content="Jiangsu Key Laboratory of Drug Discovery and Translational Research for Brain Diseases, Soochow University, Suzhou, China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7712">
<meta name="citation_doi" content="10.1038/s41467-025-62899-0">
<meta name="citation_pmid" content="40830361">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365077/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365077/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365077/pdf/41467_2025_Article_62899.pdf">
<meta name="description" content="Protein-ligand interactions are crucial for understanding various biological processes and drug discovery and design. However, experimental methods are costly; single-ligand-oriented methods are tailored to specific ligands; multi-ligand-oriented ...">
<meta name="og:title" content="LABind: identifying protein binding ligand-aware sites via learning interactions between ligand and protein">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Protein-ligand interactions are crucial for understanding various biological processes and drug discovery and design. However, experimental methods are costly; single-ligand-oriented methods are tailored to specific ligands; multi-ligand-oriented ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365077/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365077">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62899-0"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62899.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365077%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365077/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365077/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365077/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 19;16:7712. doi: <a href="https://doi.org/10.1038/s41467-025-62899-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62899-0</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>LABind: identifying protein binding ligand-aware sites via learning interactions between ligand and protein</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zhijun Zhang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zhijun Zhang</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Soochow University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhijun Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Quan%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Lijun Quan</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Lijun Quan</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Soochow University, Suzhou, China </div>
<div class="p">
<sup>2</sup>Collaborative Innovation Center of Novel Software Technology and Industrialization, Nanjing, China </div>
<div class="p">
<sup>3</sup>Jiangsu Key Laboratory of Drug Discovery and Translational Research for Brain Diseases, Soochow University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Quan%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lijun Quan</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Junkai Wang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Junkai Wang</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Soochow University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junkai Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Liangchen Peng</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Liangchen Peng</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Soochow University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Liangchen Peng</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Qiufeng Chen</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Qiufeng Chen</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Soochow University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qiufeng Chen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Bei Zhang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Bei Zhang</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Soochow University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bei Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Lexin Cao</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Lexin Cao</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Soochow University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lexin Cao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Yelu Jiang</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Yelu Jiang</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Soochow University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yelu Jiang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Geng Li</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Geng Li</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Soochow University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Geng Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nie%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Liangpeng Nie</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Liangpeng Nie</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Soochow University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nie%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Liangpeng Nie</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Tingfang Wu</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Tingfang Wu</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Soochow University, Suzhou, China </div>
<div class="p">
<sup>2</sup>Collaborative Innovation Center of Novel Software Technology and Industrialization, Nanjing, China </div>
<div class="p">
<sup>3</sup>Jiangsu Key Laboratory of Drug Discovery and Translational Research for Brain Diseases, Soochow University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tingfang Wu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lyu%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Qiang Lyu</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Qiang Lyu</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Technology, Soochow University, Suzhou, China </div>
<div class="p">
<sup>2</sup>Collaborative Innovation Center of Novel Software Technology and Industrialization, Nanjing, China </div>
<div class="p">
<sup>3</sup>Jiangsu Key Laboratory of Drug Discovery and Translational Research for Brain Diseases, Soochow University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lyu%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qiang Lyu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Computer Science and Technology, Soochow University, Suzhou, China </div>
<div id="Aff2">
<sup>2</sup>Collaborative Innovation Center of Novel Software Technology and Industrialization, Nanjing, China </div>
<div id="Aff3">
<sup>3</sup>Jiangsu Key Laboratory of Drug Discovery and Translational Research for Brain Diseases, Soochow University, Suzhou, China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Nov 6; Accepted 2025 Aug 1; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365077  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830361/" class="usa-link">40830361</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Protein-ligand interactions are crucial for understanding various biological processes and drug discovery and design. However, experimental methods are costly; single-ligand-oriented methods are tailored to specific ligands; multi-ligand-oriented methods are constrained by the lack of ligand encoding. In this study, we propose a structure-based method called LABind, designed to predict binding sites for small molecules and ions in a ligand-aware manner. LABind utilizes a graph transformer to capture binding patterns within the local spatial context of proteins, and incorporates a cross-attention mechanism to learn the distinct binding characteristics between proteins and ligands. Experimental results on three benchmark datasets demonstrate both the effectiveness of LABind and its ability to generalize to unseen ligands. Further analysis validates that LABind can effectively integrate ligand information to predict binding sites. Additionally, the application of LABind is extended to binding site center localization, sequence-based methods, and molecular docking tasks.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Computational biology and bioinformatics, Computational models, Protein function predictions</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Here, authors present LABind, a ligand-aware deep learning framework for predicting binding sites of small molecules and ions, with the capacity to generalize to unseen ligands.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Protein-ligand interactions play a fundamental role in many biological processes<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>, such as enzyme catalysis, signal transduction, and molecular recognition<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. These interactions are not only critical for understanding cellular functions but are also pivotal in drug discovery and design<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Although techniques such as X-ray crystallography, nuclear magnetic resonance, and cryo-electron microscopy can precisely determine binding sites, their extensive resource consumption limits widespread application. Given the rapid growth in the number of known protein structures and small molecules<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>, developing a computational method to accurately and efficiently identify protein-ligand binding sites is of importance.</p>
<p id="Par4">In literature<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>, computational methods for predicting protein-ligand binding sites started from valuable small datasets derived from biological experiments. Some methods train individual models on single datasets targeting specific ligands. Consequently, predicting binding sites necessitates selecting the corresponding model for the specific target ligand. We refer to these methods as single-ligand-oriented methods. Furthermore, single-ligand-oriented methods can be categorized into template-based methods, sequence-based methods, and structure-based methods, depending on the types of protein information used. Template-based methods such as IonCom<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>, MIB<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, and GASS-Metal<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup> employ alignment algorithms to match known ligand binding sites from similar proteins to the query protein. However, without high-quality known protein templates, these methods often fail to achieve the desired outcomes. Sequence-based methods identify binding sites solely based on protein sequence information. TargetS<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup> employs a sliding-window strategy to extract features from protein sequences, and uses a support vector machine to predict binding sites. However, the lack of experimental spatial structure information limits the effectiveness of sequence-based methods, leading to a growing interest in structure-based methods. DELIA<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> utilizes fixed-size convolution kernels to learn protein-ligand interaction patterns from 2D distance matrices of proteins and uses bidirectional long short-term memory networks to extract features from protein sequences. GraphBind<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> encodes protein structural context as graph features and then uses hierarchical graph neural networks to predict binding sites from protein structure graphs. LigBind<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> undergoes pre-training on a broad set of ligands, then fine-tunes for over 1000 specific ligands. GeoBind<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> combines surface point clouds with graph networks to predict protein-nucleic acid binding sites. While single-ligand-oriented methods often achieve better results for specific ligands, their inherent specialization limits their effectiveness when applied to unseen ligands.</p>
<p id="Par5">With advances in biological experiments, more binding datasets are available. It is now possible to combine several datasets containing multiple ligands to train a single computational method. We refer to these methods as multi-ligand-oriented methods. LMetalSite<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> and GPSite<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> utilize multi-task learning to combine multiple ligand datasets and train a single model for multiple specific ligands. However, LMetalSite and GPSite are still limited to specific ligands. Many multi-ligand-oriented studies (e.g., P2Rank<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, DeepSurf<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, and DeepPocket<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>) represent protein structures as corresponding features, such as the solvent accessible surface. They rely directly on protein structures without considering specific ligands, thereby underscoring the importance of the protein structures themselves. However, these methods overlook the differences in binding pattern among different ligands. Ligand-oriented models share the same inability to predict protein binding sites for unseen ligands, as they lack an explicit encoding of ligand properties during the training stage. In summary, it is both necessary and challenging to develop a unified model that incorporates ligand information for accurately addressing a wide range of different ligand scenarios.</p>
<p id="Par6">In this study, we propose a method capable of predicting protein binding sites for ligands that are not present in the training set. A binding site is defined here as a set of protein residues located within a specific distance from the ligand<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Consequently, the classification task in LABind is per-residue prediction, aiming to determine whether each residue in a protein is part of a binding site. During the training stage, our method explicitly models ions and small molecules as well as proteins. Thus, it can also identify protein binding sites for ligands not seen during the training stage. As our model explicitly learns the representations of ligands, we name our model ligand-aware binding site prediction (LABind for short). It is worth noting that LigBind also considers the characteristics of ligands and can identify binding sites for unseen ligands using the pre-training model. Nevertheless, the effectiveness of its pre-training is limited, and it still requires fine-tuning with specific ligands to achieve more accurate prediction results. Additionally, as partners in protein binding, ligands can be either biomacromolecules (e.g., proteins, RNAs) or other small molecules (including ions). We assume that protein-biomacromolecules binding may exhibit many different properties compared with protein-small molecule binding<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. In this study, we focus on predicting protein binding sites for small molecules and ions. LABind utilizes a recently published protein pre-trained language model (Ankh<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>) to obtain sequence representations of proteins, and uses a molecular pre-trained language model (MolFormer<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>) to represent molecular properties based on ligand Simplified Molecular Input Line Entry System (SMILES) sequences. It encodes proteins into graphs and uses a graph transformer to capture potential binding patterns in the local spatial context of proteins. By employing a cross-attention mechanism, it learns the distinct binding characteristics between proteins and ligands. Although LABind is a structure-based approach, we also develop a sequence-based program that leverages the structures predicted by ESMFold<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> to implement a sequence-based prediction strategy. Ultimately, LABind comprehensively considers the impact of ligand information on binding sites and encompasses all small molecules and ions through a unified model. The modeling of ligands enhances the understanding of interactions between proteins and ligands, which contributes to the formation of binding sites. This multi-ligand capability enables the model to learn representations shared between different ligand binding sites, while also learning representations specific to each ligand binding site. Comprehensive experiments demonstrate that LABind shows marked advantages over other multi-ligand-oriented and single-ligand-oriented methods. First, LABind outperforms other advanced methods in various benchmark datasets (DS1, DS2, and DS3, see Benchmark datasets for dataset details). Importantly, as a multi-ligand-oriented method that effectively incorporates ligand information into its model architecture, LABind demonstrably improves the accuracy of predicting binding sites for a variety of ligands, including small molecules, ions, and unseen ligands. The integration of ligand characteristics not only enhances the model’s performance but also underscores the pivotal role of these big data-pretrained features in identifying binding sites. Second, LABind outperforms competing methods in predicting binding site centers through clustering of predicted binding residues. The robustness of our method is validated by applying it to proteins without experimentally determined structures, even when using predicted structures from ESMFold and OmegaFold<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. LABind consistently demonstrates resilience and reliability. Furthermore, LABind shows a strong ability to effectively distinguish between different ligands and substantially enhance the accuracy of molecular docking tasks. Additionally, LABind showcases its potential and reliability in practical applications by successfully predicting the binding sites of the SARS-CoV-2 NSP3 macrodomain with unseen ligands. Through ablation studies, we demonstrate the importance of each protein feature source in LABind, with the protein representation proving to be crucial and the positive impact of ligand features also being evident. Additionally, the visualization of residue representations derived from the attention-based learning interaction shows how LABind effectively captures information about protein-ligand interactions, thereby learning the differences between binding and non-binding sites and enhancing the accuracy of predictions.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<p id="Par7">In our comparative analysis of LABind against other methodologies across three benchmark datasets (DS1, DS2, and DS3), LABind exhibited superior performance. LABind also outperformed competing methods in the prediction of binding site centers. We evaluated the influence of predicted protein structures on the identification of binding sites, focusing on how variations in structural accuracy affect the reliability of binding site prediction. To further evaluate the effectiveness of LABind in distinguishing between different ligands, we visualized LABind’s predictions of binding sites in two proteins associated with distinct ligands. Moreover, the binding sites predicted by LABind were utilized to improve the accuracy of docking poses generated by Smina<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. We conducted a case study on the binding sites of the SARS-CoV-2 NSP3 macrodomain. The insights gained from this case further validate LABind’s applicability in real-world scenarios. Additionally, we performed ablation studies to explore how various input features influence model’s predictive outcomes. Lastly, we visualized the residue representations obtained from the attention-based learning interaction to demonstrate their effectiveness in capturing crucial information about protein-ligand interactions.</p>
<section id="Sec3"><h3 class="pmc_sec_title">Overview of LABind</h3>
<p id="Par8">The overall architecture of LABind is illustrated in Fig. <a href="#Fig1" class="usa-link">1</a>a. First, the SMILES sequence of the ligand is input into the MolFormer pre-trained model to obtain the ligand representation. The sequence and structure of the protein receptor are input into Ankh and DSSP<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, respectively, to obtain the protein embedding and DSSP features, which are then concatenated to form the protein-DSSP embedding. The protein structure is converted into a protein graph through the graph converter module, where the node spatial features are angles, distances, and directions derived from atomic coordinates, and the edge spatial features include directions, rotations, and distances between residues. Then, the protein-DSSP embedding obtained previously is added to the node spatial features of the protein graph, resulting in the final protein representation (including residue representations and residue-residue interactions). Next, the ligand representation and the protein representation are processed through the attention-based learning interaction to learn the interactions between them. Finally, a multi-layer perceptron (MLP) classifier is used to predict the binding sites of the protein with the ligand.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. The overall architecture of LABind.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365077_41467_2025_62899_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc34/12365077/d9c0aaead1da/41467_2025_62899_Fig1_HTML.jpg" loading="lazy" id="d33e378" height="799" width="799" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The model architecture of LABind. The protein sequence and ligand SMILES are input into the Ankh and MolFormer pre-trained language models, respectively, to obtain protein embeddings and ligand representations. Node spatial features and edge spatial features are extracted from the protein structure by the graph converter module. After being combined with protein-DSSP embeddings to form protein representations, the binding sites are predicted by the attention-based learning interaction and an MLP classifier. <strong>b</strong> The architecture of the graph converter module. For node spatial features, R and S represent the centroid of the heavy side chain atoms and the position of the nearest atom on the surface, respectively. For edge spatial features, the C<sub><em>α</em></sub> is used to calculate the directions and torsion angles. The distances from all neighboring atoms to the target C<sub><em>α</em></sub> are calculated as part of the edge spatial features. <strong>c</strong> The architecture of the attention-based learning interaction. Local contextual information of the protein representations is learned through a neighbor-attention, and a cross-attention is used to facilitate the interactions between the ligand representation and residue representations. Residue-residue interactions are updated using the updated residue representations and original residue-residue interactions, and the ligand representation is updated by performing both max-pooling and averaging on residue representations. <strong>d</strong> The architecture of the multi-head cross-attention. Attention scores are computed using residue representations as queries and the ligand representation as both keys and values.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Evaluation metrics</h3>
<p id="Par9">To evaluate the performance of LABind on protein and ligand binding site prediction, we used recall (Rec), precision (Pre), F1 score (F1), Matthews correlation coefficient (MCC), area under the receiver operating characteristic curve (AUC) and area under the precision-recall curve (AUPR) as evaluation metrics. In addition, we used DCC (distance between the predicted binding site center and the true binding site center) and DCA (distance between the predicted binding site center and the closest ligand atom) to evaluate the performance in localizing binding site centers. The calculation methods for all metrics are provided in Supplementary Note <a href="#MOESM1" class="usa-link">1</a>. Due to the highly imbalanced distribution and number of binding sites and non-binding sites, MCC and AUPR are more reflective of the performance of a model in imbalanced two-class classification tasks<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. In addition to AUC and AUPR, which are not affected by threshold selection, other metrics are calculated using a threshold to convert the predicted binding probabilities to binary predictions. This threshold is determined by maximizing the MCC value. Therefore, we adopted AUPR as the criterion for hyperparameter optimization and ablation studies.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Baseline methods</h3>
<p id="Par10">To illustrate the improvements achieved by LABind, we compared it with five multi-ligand-oriented methods (P2Rank, DeepSurf, DeepPocket, LMetalSite, and GPSite) and four single-ligand-oriented methods (IonCom, DELIA, GraphBind, and LigBind). Among the compared methods, P2Rank, DeepSurf, and DeepPocket are structure-based approaches that predict the spatial center of a binding site rather than individual binding residues. The introductions of these methods are provided in Supplementary Note <a href="#MOESM1" class="usa-link">2</a>.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Prediction accuracy of LABind on benchmark</h3>
<p id="Par11">We compared LABind with literature single-ligand-oriented methods on the DS1 dataset. For a fair comparison, LABind, like LigBind, was trained on the DS1 training set and tested on the DS1 test set. The results of these 19 ligands on the DS1 test set are presented in Table <a href="#Tab1" class="usa-link">1</a>. Compared to the best-performing benchmark method, LigBind, the MCCs of LABind are 0.049-0.221 higher across all ligands except NO<sub>2</sub><sup>-</sup> and HEM. Although LABind underperforms LigBind by 0.029 and 0.015 in MCC on NO<sub>2</sub><sup>-</sup> and HEM, respectively, the average MCC of LABind is 21.8% higher than that of LigBind. It is worth noting that the metrics of LigBind are still targeted at models for individual ligands. Here, we compared LABind with only four methods on the DS1 test set. Since the DS1 test set overlaps with the DS2 training set, we excluded the metrics for GPSite and LMetalSite that were trained on the DS2 dataset. All metrics of all methods are shown in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>. When ligand information is ignored, with ligand representations set to an all-one embedding for all ligands, LABind-NL achieves an average MCC of 0.499 among these 19 ligands. The average MCC of LABind-NL is comparable to that of LigBind, but it is 0.110 lower than that of LABind. This result demonstrates that some protein-ligand binding samples in the DS1 training set are insensitive to the ligand. This is attributed to the fact that protein-ligand binding sites are generally located in pockets (clefts, grooves) on the surface of proteins<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, which have the capacity to interact with multiple ligands. Consequently, similar proteins with similar ligands can exhibit analogous binding sites<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. LABind-NL can capture the commonality of how protein structure influences binding sites in protein-ligand binding patterns.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>MCC comparison of LABind with four single-ligand-oriented methods for 19 ligands derived from the DS1 test set</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Ligand</th>
<th colspan="1" rowspan="1">IonCom</th>
<th colspan="1" rowspan="1">DELIA</th>
<th colspan="1" rowspan="1">GraphBind</th>
<th colspan="1" rowspan="1">LigBind</th>
<th colspan="1" rowspan="1">LABind-NL</th>
<th colspan="1" rowspan="1">LABind</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">Ca<sup>2+</sup>
</td>
<td colspan="1" rowspan="1">0.220</td>
<td colspan="1" rowspan="1">0.296</td>
<td colspan="1" rowspan="1">0.465</td>
<td colspan="1" rowspan="1">0.465</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.494</span></td>
<td colspan="1" rowspan="1"><strong>0.664</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Mg<sup>2+</sup>
</td>
<td colspan="1" rowspan="1">0.246</td>
<td colspan="1" rowspan="1">0.293</td>
<td colspan="1" rowspan="1">0.319</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.354</span></td>
<td colspan="1" rowspan="1">0.308</td>
<td colspan="1" rowspan="1"><strong>0.461</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Mn<sup>2+</sup>
</td>
<td colspan="1" rowspan="1">0.291</td>
<td colspan="1" rowspan="1">0.558</td>
<td colspan="1" rowspan="1">0.561</td>
<td colspan="1" rowspan="1">0.608</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.663</span></td>
<td colspan="1" rowspan="1"><strong>0.710</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Zn<sup>2+</sup>
</td>
<td colspan="1" rowspan="1">0.299</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.750</span></td>
<td colspan="1" rowspan="1">0.748</td>
<td colspan="1" rowspan="1"><strong>0.843</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Fe<sup>3+</sup>
</td>
<td colspan="1" rowspan="1">0.377</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">0.679</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.740</span></td>
<td colspan="1" rowspan="1"><strong>0.768</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Fe<sup>2+</sup>
</td>
<td colspan="1" rowspan="1">0.498</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">0.678</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.761</span></td>
<td colspan="1" rowspan="1"><strong>0.848</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Cu<sup>2+</sup>
</td>
<td colspan="1" rowspan="1">0.416</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">0.552</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.658</span></td>
<td colspan="1" rowspan="1"><strong>0.771</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Na<sup>+</sup>
</td>
<td colspan="1" rowspan="1">0.066</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">0.166</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.212</span></td>
<td colspan="1" rowspan="1"><strong>0.387</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">K<sup>+</sup>
</td>
<td colspan="1" rowspan="1">0.114</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">0.232</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.253</span></td>
<td colspan="1" rowspan="1"><strong>0.444</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">CO<sub>3</sub><sup>2-</sup>
</td>
<td colspan="1" rowspan="1">0.129</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.213</span></td>
<td colspan="1" rowspan="1">0.201</td>
<td colspan="1" rowspan="1"><strong>0.357</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">NO<sub>2</sub><sup>-</sup>
</td>
<td colspan="1" rowspan="1">0.196</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1"><strong>0.369</strong></td>
<td colspan="1" rowspan="1">0.213</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.340</span></td>
</tr>
<tr>
<td colspan="1" rowspan="1">SO<sub>4</sub><sup>2-</sup>
</td>
<td colspan="1" rowspan="1">0.197</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.364</span></td>
<td colspan="1" rowspan="1">0.300</td>
<td colspan="1" rowspan="1"><strong>0.441</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">PO<sub>4</sub><sup>3-</sup>
</td>
<td colspan="1" rowspan="1">0.207</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.422</span></td>
<td colspan="1" rowspan="1">0.418</td>
<td colspan="1" rowspan="1"><strong>0.548</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">ADP</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.663</span></td>
<td colspan="1" rowspan="1">0.635</td>
<td colspan="1" rowspan="1"><strong>0.712</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">AMP</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.497</span></td>
<td colspan="1" rowspan="1">0.447</td>
<td colspan="1" rowspan="1"><strong>0.553</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">ATP</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">0.535</td>
<td colspan="1" rowspan="1">0.472</td>
<td colspan="1" rowspan="1">0.578</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.612</span></td>
<td colspan="1" rowspan="1"><strong>0.667</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">GDP</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.628</span></td>
<td colspan="1" rowspan="1">0.593</td>
<td colspan="1" rowspan="1"><strong>0.678</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">GTP</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">0.554</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.594</span></td>
<td colspan="1" rowspan="1"><strong>0.667</strong></td>
</tr>
<tr>
<td colspan="1" rowspan="1">HEM</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">0.596</td>
<td colspan="1" rowspan="1">0.633</td>
<td colspan="1" rowspan="1"><strong>0.728</strong></td>
<td colspan="1" rowspan="1">0.635</td>
<td colspan="1" rowspan="1"><span class="text-underline">0.713</span></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p14"><p>Boldface and underline indicate the best performance and the second-best performance among the compared methods for each ligand, respectively. The results of other methods were obtained from the report of LigBind. LABind-NL represents the performance of LABind without ligand representations.</p></div></div></section><p id="Par12">Moreover, we compared LABind with multi-ligand-oriented methods on the DS2 dataset. To compare with multi-ligand-oriented methods, we retrained LABind on the DS2 training set and tested on the DS2 test set. We compared LABind with the nine methods on six ligands, including ATP, HEM, Zn<sup>2+</sup>, Ca<sup>2+</sup>, Mg<sup>2+</sup>, Mn<sup>2+</sup>. The results on the DS2 dataset are given in Table <a href="#Tab2" class="usa-link">2</a>. The MCC of LABind is lower than that of GPSite only on the ATP test set, while it is higher than other prediction methods on other ligands. The average MCC of LABind shows a 4.7% improvement over that of GPSite on the DS2 test set. Due to the smaller size of metal ions, the interactions of the metal ions with proteins are often found significantly more versatile and flexible compared with those of larger size ligands<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. LABind demonstrates improved performance in identifying metal ion-binding sites. However, the performance advantage of LABind-NL is not as appreciable on the DS2 dataset. We believe this is due to the smaller number of proteins, which prevents the network from effectively learning the common pattern across different ligand-protein bindings. Notably, LABind demonstrates strong performance on both small molecule ligands and ion ligands.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>MCC comparison of LABind with other methods on the DS2 test set</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Method</th>
<th colspan="1" rowspan="1">ATP</th>
<th colspan="1" rowspan="1">HEM</th>
<th colspan="1" rowspan="1">Zn<sup>2+</sup>
</th>
<th colspan="1" rowspan="1">Ca<sup>2+</sup>
</th>
<th colspan="1" rowspan="1">Mg<sup>2+</sup>
</th>
<th colspan="1" rowspan="1">Mn<sup>2+</sup>
</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">IonCom</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">0.317</td>
<td colspan="1" rowspan="1">0.258</td>
<td colspan="1" rowspan="1">0.237</td>
<td colspan="1" rowspan="1">0.344</td>
</tr>
<tr>
<td colspan="1" rowspan="1">DELIA</td>
<td colspan="1" rowspan="1">0.548</td>
<td colspan="1" rowspan="1">0.614</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">0.325</td>
<td colspan="1" rowspan="1">0.287</td>
<td colspan="1" rowspan="1">0.574</td>
</tr>
<tr>
<td colspan="1" rowspan="1">GraphBind</td>
<td colspan="1" rowspan="1">0.483</td>
<td colspan="1" rowspan="1">0.578</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">0.475</td>
<td colspan="1" rowspan="1">0.331</td>
<td colspan="1" rowspan="1">0.545</td>
</tr>
<tr>
<td colspan="1" rowspan="1">LigBind</td>
<td colspan="1" rowspan="1">0.565</td>
<td colspan="1" rowspan="1">0.588</td>
<td colspan="1" rowspan="1">0.680</td>
<td colspan="1" rowspan="1">0.288</td>
<td colspan="1" rowspan="1">0.242</td>
<td colspan="1" rowspan="1">0.571</td>
</tr>
<tr>
<td colspan="1" rowspan="1">LMetalSite</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">–</td>
<td colspan="1" rowspan="1">0.761</td>
<td colspan="1" rowspan="1">0.542</td>
<td colspan="1" rowspan="1">0.419</td>
<td colspan="1" rowspan="1">0.661</td>
</tr>
<tr>
<td colspan="1" rowspan="1">GPSite</td>
<td colspan="1" rowspan="1"><strong>0.668</strong></td>
<td colspan="1" rowspan="1">0.722</td>
<td colspan="1" rowspan="1">0.797</td>
<td colspan="1" rowspan="1">0.593</td>
<td colspan="1" rowspan="1">0.438</td>
<td colspan="1" rowspan="1">0.701</td>
</tr>
<tr>
<td colspan="1" rowspan="1">LABind-NL</td>
<td colspan="1" rowspan="1">0.497</td>
<td colspan="1" rowspan="1">0.591</td>
<td colspan="1" rowspan="1">0.778</td>
<td colspan="1" rowspan="1">0.494</td>
<td colspan="1" rowspan="1">0.375</td>
<td colspan="1" rowspan="1">0.705</td>
</tr>
<tr>
<td colspan="1" rowspan="1">LABind</td>
<td colspan="1" rowspan="1">0.663</td>
<td colspan="1" rowspan="1"><strong>0.730</strong></td>
<td colspan="1" rowspan="1"><strong>0.837</strong></td>
<td colspan="1" rowspan="1"><strong>0.679</strong></td>
<td colspan="1" rowspan="1"><strong>0.472</strong></td>
<td colspan="1" rowspan="1"><strong>0.722</strong></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p17"><p>Boldface indicates the best performance among the compared methods for each ligand. The results of LigBind were obtained from the LigBind-G program. The results of other methods were obtained from the report of GPSite. LABind-NL represents the performance of LABind without ligand representations.</p></div></div></section><p id="Par13">Having established the prior performance of LABind on ligand-oriented methods, we trained LABind on a comprehensive dataset as a final predictor that is capable of identifying binding sites between proteins and ligands unseen in the DS3 training set. The hyperparameter settings for the DS3 training set were identical to those used for the DS1 and DS2 training sets. The DS3 picking set (see DS3: recent assembled dataset for ligand-aware model for details) was used solely to tune hyperparameters and determine the binary classification threshold based on the model’s outputs, ensuring that no information from the test set influenced the training process. As shown in Supplementary Note <a href="#MOESM1" class="usa-link">3</a>, the training sets used by competing methods share a higher sequence identity with the DS3 test set compared to the DS3 training set. This could potentially introduce some bias and inadvertently favor the competing methods during evaluation. Figure <a href="#Fig2" class="usa-link">2</a> illustrates that LABind still outperforms LigBind on the DS3 test set, demonstrating its improved generalization capability on unseen data. Note that IonCom, DELIA, GraphBind, LMetalSite, and GPSite are unable to predict unseen ligands. In addition, P2Rank, DeepSurf, and DeepPocket primarily predict binding site centers rather than individual binding residues. Therefore, only LigBind was retained for this comparison. The comparison of MCC and AUPR between LigBind and LABind across all ligands is shown in Fig. <a href="#Fig2" class="usa-link">2</a>a. Among the 474 unseen ligands, LABind outperforms LigBind in MCC on 307 unseen ligands, and achieves higher AUPR on 366 unseen ligands. As shown in Fig. <a href="#Fig2" class="usa-link">2</a>b, for the entire binding sample, LABind exceeds LigBind by 0.125 in F1, 0.131 in MCC, and 0.152 in AUPR. Overall, the results demonstrate that LABind achieves performance improvement on unseen ligands compared to LigBind.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Performance comparison of LABind with LigBind on the DS3 test set.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365077_41467_2025_62899_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc34/12365077/f4a155e72e21/41467_2025_62899_Fig2_HTML.jpg" loading="lazy" id="d33e1066" height="282" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> MCC and AUPR comparison between LABind and LigBind. The number of binding residues is represented by the color of the points. The X-axis and Y-axis represent the corresponding values of LigBind and LABind, respectively. <strong>b</strong> Overall performance metrics of LABind and LigBind on the DS3 dataset. ‘*’ represents that ESMFold-predicted structures are used as the structural input. The results of LigBind were obtained from the LigBind-G program.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">Performance of LABind in predicting binding site centers</h3>
<p id="Par14">We further evaluated and compared the performance of LABind with three baseline methods (P2Rank, DeepSurf, and DeepPocket) in predicting binding site centers on the benchmark datasets. Since LABind does not directly output predicted binding site centers, we applied the mean-shift clustering algorithm<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> to the predicted binding residues from LABind to determine the binding site centers. This mean-shift-based identification of binding site centers and the evaluation protocol are detailed in Supplementary Note <a href="#MOESM1" class="usa-link">4</a>. Table <a href="#Tab3" class="usa-link">3</a> summarizes the performance of different methods in localizing binding site centers for small-molecule ligands. For the DS1 and DS2 test sets, we retained only the data involving small-molecule ligands, as the compared methods were specifically trained for small-molecule binding prediction. LABind outperforms other methods for predicting binding site centers on the DS1 and DS2 test sets; however, these datasets have limited ligand diversity. In contrast, the DS3 test set includes a broader range of ligands. On the DS3 test set, LABind achieves a DCC success rate only 0.8% lower than P2Rank but surpasses all other methods in DCA performance. As the DS1, DS2, and DS3 datasets are not specifically curated for evaluating binding site center localization, we further selected 64 binding sites from the COACH420 benchmark as an additional test set. Following DeepPocket’s stringent data filtering criteria, we excluded protein-ligand complexes from COACH420 that share more than 50% sequence identity with proteins in the DS3 training set or exhibit both greater than 0.9 ligand similarity and more than 30% sequence identity. The resulting dataset, termed COACH64, is described in detail in Supplementary Note <a href="#MOESM1" class="usa-link">5</a>. On the COACH64 test set, LABind achieves the best overall performance, with DCC and DCA success rates 4.7% and 3.1% higher, respectively, than those of the second-best method. Although LABind is not specifically designed to predict binding site centers, these results demonstrate its strong potential in this regard.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Performance comparison of LABind and similar methods in binding site center localization on the four test sets</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th colspan="1" rowspan="1">Method</th>
<th colspan="2" rowspan="1">DS1</th>
<th colspan="2" rowspan="1">DS2</th>
<th colspan="2" rowspan="1">DS3</th>
<th colspan="2" rowspan="1">COACH64</th>
</tr>
<tr>
<th colspan="1" rowspan="1"></th>
<th colspan="1" rowspan="1">DCC</th>
<th colspan="1" rowspan="1">DCA</th>
<th colspan="1" rowspan="1">DCC</th>
<th colspan="1" rowspan="1">DCA</th>
<th colspan="1" rowspan="1">DCC</th>
<th colspan="1" rowspan="1">DCA</th>
<th colspan="1" rowspan="1">DCC</th>
<th colspan="1" rowspan="1">DCA</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">P2Rank</td>
<td colspan="1" rowspan="1">33.7</td>
<td colspan="1" rowspan="1">57.4</td>
<td colspan="1" rowspan="1"><span class="text-underline">44.8</span></td>
<td colspan="1" rowspan="1">60.0</td>
<td colspan="1" rowspan="1"><strong>13.8</strong></td>
<td colspan="1" rowspan="1"><span class="text-underline">28.8</span></td>
<td colspan="1" rowspan="1">46.9</td>
<td colspan="1" rowspan="1"><span class="text-underline">82.8</span></td>
</tr>
<tr>
<td colspan="1" rowspan="1">DeepSurf</td>
<td colspan="1" rowspan="1">39.1</td>
<td colspan="1" rowspan="1"><span class="text-underline">62.8</span></td>
<td colspan="1" rowspan="1">44.1</td>
<td colspan="1" rowspan="1"><span class="text-underline">65.5</span></td>
<td colspan="1" rowspan="1">12.6</td>
<td colspan="1" rowspan="1">26.9</td>
<td colspan="1" rowspan="1"><span class="text-underline">48.4</span></td>
<td colspan="1" rowspan="1"><span class="text-underline">82.8</span></td>
</tr>
<tr>
<td colspan="1" rowspan="1">DeepPocket</td>
<td colspan="1" rowspan="1"><span class="text-underline">39.5</span></td>
<td colspan="1" rowspan="1">60.0</td>
<td colspan="1" rowspan="1">42.1</td>
<td colspan="1" rowspan="1">62.8</td>
<td colspan="1" rowspan="1">12.3</td>
<td colspan="1" rowspan="1">28.0</td>
<td colspan="1" rowspan="1">39.1</td>
<td colspan="1" rowspan="1">67.2</td>
</tr>
<tr>
<td colspan="1" rowspan="1">LABind</td>
<td colspan="1" rowspan="1"><strong>53.3</strong></td>
<td colspan="1" rowspan="1"><strong>73.5</strong></td>
<td colspan="1" rowspan="1"><strong>56.6</strong></td>
<td colspan="1" rowspan="1"><strong>74.5</strong></td>
<td colspan="1" rowspan="1"><span class="text-underline">13.0</span></td>
<td colspan="1" rowspan="1"><strong>30.2</strong></td>
<td colspan="1" rowspan="1"><strong>53.1</strong></td>
<td colspan="1" rowspan="1"><strong>85.9</strong></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p22"><p>Boldface and underline indicate the best performance and the second-best performance among the compared methods, respectively. The success rates of DCC and DCA are both based on the top-<em>n</em> predicted binding site centers, where <em>n</em> denotes the number of identical cognate ligands.</p></div></div></section></section><section id="Sec8"><h3 class="pmc_sec_title">Validation of LABind on binding sites of proteins without experimentally determined structures</h3>
<p id="Par15">The preceding results are based on holo structures (ligand-bound protein structures). However, physicochemical interactions between binding sites and ligands can induce conformational changes in proteins, leading to transitions between apo structures (unbound protein structures) and holo structures<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Accurately predicting binding sites on apo or predicted structures poses a greater challenge and holds higher practical significance than predictions on holo structures. Furthermore, due to the limitations of some experimental techniques, many proteins currently lack experimentally determined tertiary structures. For proteins without accurate atomic positions, protein structure prediction methods were combined to achieve sequence-based ligand binding site predictions. Specifically, we tested the effects of combining ESMFold and OmegaFold with LABind separately. The primary reason we did not choose AlphaFold2<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> is its requirement for multiple sequence alignment, which is time-consuming. We assessed the quality of predicted structures by calculating the TM-score between the native and predicted structures using US-align<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Figure <a href="#Fig3" class="usa-link">3</a> illustrates how the performance of LABind depends on high-quality protein structures and identifies which protein structure predictor is more compatible with LABind. The distributions of TM-scores for ESMFold and OmegaFold on the DS1 test set are presented in Fig. <a href="#Fig3" class="usa-link">3</a>a. The average TM-score of ESMFold is higher than that of OmegaFold on the DS1 test set (0.861 vs 0.824). As shown in Fig. <a href="#Fig3" class="usa-link">3</a>b, when LABind was retrained on the DS1 dataset using predicted structures from ESMFold and OmegaFold, respectively, it is evident that LABind based on native structures outperforms those based on predicted structures. This result is expected, as native structures provide more accurate surface and spatial information. Specifically, the average AUPR achieved using ESMFold and OmegaFold is lower than that achieved using the native structure by 0.055 and 0.066, respectively. Although the performance using OmegaFold-predicted structures is suboptimal, the average AUPR remains 0.065 higher than that of LigBind. Additionally, as shown in Fig. <a href="#Fig3" class="usa-link">3</a>c, we selected predicted structures with a TM-score  &lt;0.5 on the DS1 test set to examine the robustness of LABind for low-quality predictions. On the DS1 test set, 252 OmegaFold-predicted structures and 152 ESMFold-predicted structures have a TM-score  &lt;0.5. As visualized in Fig. <a href="#Fig3" class="usa-link">3</a>c, due to the larger number of low-quality proteins in OmegaFold, the linear regression line for LABind based on ESMFold-predicted structures consistently lies above the line based on OmegaFold-predicted structures.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. The performance of LABind without experimentally determined structures.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365077_41467_2025_62899_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc34/12365077/531add588ff2/41467_2025_62899_Fig3_HTML.jpg" loading="lazy" id="d33e1300" height="219" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The distributions of the TM-scores between the native structures and the predicted structures by ESMFold and OmegaFold on the DS1 test set (<em>n</em> = 2626 protein chains). The violin plot displays the minima and maxima (lower and upper ends), median (the white dot in the center), interquartile range (lower and upper ends of the box), and 1.5 times the interquartile range (whiskers). <strong>b</strong> The AUPR performance of LABind across various predicted structures on the DS1 test set. <strong>c</strong> The relationship between proteins with TM-score  &lt;0.5 for ESMFold and OmegaFold in each ligand test set of DS1 dataset. The lines are fitted to the TM-score and AUPR original data using linear regression.</p></figcaption></figure><p id="Par16">For proteins without experimentally determined structures from wet-lab, we instead chose the predicted structures from dry-lab approaches like ESMFold. We compared the DS1 training and test sets under four different scenarios, as shown in Supplementary Table <a href="#MOESM1" class="usa-link">2</a>. As observed, when the structure types in the DS1 training and test sets differ, the average AUPR remains nearly the same. Although the average AUPR using predicted structures for both training and testing is 0.003 higher than when using native structures for training and predicted structures for testing, we still opted to use the LABind trained on native structures to make predictions on predicted structures. To further validate the robustness of LABind on ESMFold-predicted structures, we used the predicted structures from ESMFold as structural input to predict binding sites on the DS3 test set. As illustrated in Fig. <a href="#Fig2" class="usa-link">2</a>b, the performance of LABind using ESMFold-predicted structures still outperforms LigBind evaluated on holo structures by 0.082 in MCC and 0.094 in AUPR. This demonstrates the robustness of LABind when using predicted structural inputs and its potential generalizability to sequence-based binding site prediction. To facilitate better application in scientific experiments, we developed a prediction pipeline based on both sequence and structure information. The detailed processing workflow is illustrated in Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>. For proteins with experimentally determined structures, these structures are directly used by LABind. For proteins with only amino acid sequences, predictions are made using the structures generated by ESMFold. Thus, LABind can be extended to predict the binding sites of protein-small molecule ligand interactions under various conditions.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Validation of LABind on binding sites of proteins with different ligands</h3>
<p id="Par17">To verify whether LABind, as a unified model, can accurately distinguish the binding sites of the same protein with different ligands, we selected two proteins from the DS1 test set that interact with multiple ligands. We selected the alkaline phosphatase from Sphingomonas in complex with inorganic phosphate (PDB ID: 5XWK, chain: A, 5XWK_A) with Zn<sup>2+</sup>, Ca<sup>2+</sup> and PO<sub>4</sub><sup>3-</sup>, as well as the crystal structure of human MORC2 with spinal muscular atrophy mutation S87L protein (PDB ID: 5OFB, chain: B, 5OFB_B) with Zn<sup>2+</sup> and ATP. Figure <a href="#Fig4" class="usa-link">4</a>a shows the binding scores predicted by LABind for two proteins with different ligands, where only residues with prediction scores greater than 0.3 are considered. For 5XWK_A, the predicted scores for Zn<sup>2+</sup> and PO<sub>4</sub><sup>3–</sup> overlap significantly, while the predicted scores for Ca<sup>2+</sup> are primarily located toward the rear of the protein. There is a small overlap between the high-scoring regions for Ca<sup>2+</sup> and Zn<sup>2+</sup>, but the scores for Ca<sup>2+</sup> are relatively lower compared to those for Zn<sup>2+</sup>. For 5OFB_B, the high-scoring region for ATP is concentrated at the front of the protein sequence, while that for Zn<sup>2+</sup> is mainly located toward the rear of the protein sequence. As shown in Fig. <a href="#Fig4" class="usa-link">4</a>b, it is evident that the positions of Zn<sup>2+</sup> and PO<sub>4</sub><sup>3-</sup> are in close proximity, which leads to a high degree of overlap in the binding sites. This aligns with the binding scores predicted by LABind. On the other hand, the Ca<sup>2+</sup> ion occupies a distinct region, and LABind also captures the corresponding binding sites. Similarly, as shown in Fig. <a href="#Fig4" class="usa-link">4</a>c, Zn<sup>2+</sup> and ATP are spatially distant in 5OFB_B, so their binding sites do not overlap. The binding scores predicted by LABind clearly reflect this difference and identify the corresponding binding sites. This indicates that LABind can also distinguish between the interactions of small molecules and ions. Furthermore, the scores predicted by LABind for Zn<sup>2+</sup> binding sites are generally high. Zn<sup>2+</sup> binding sites are relatively easier to identify, whereas Ca<sup>2+</sup> binding sites are more difficult to recognize, which is consistent with previous studies<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Collectively, these results suggest that LABind demonstrates a clear ability to distinguish between different ligands, which forms the basis for developing a unified model.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Visualization of LABind predictions for the binding sites of two proteins with different ligands.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc34/12365077/92fdb69ae02a/41467_2025_62899_Fig4_HTML.jpg" loading="lazy" id="d33e1407" height="1129" width="665" alt="Fig. 4"></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The prediction scores of LABind for the binding sites of two proteins with different ligands. <strong>b</strong> Visualization of the complex structure of the 5XWK_A protein with Zn<sup>2+</sup>, Ca<sup>2+</sup>, and PO<sub>4</sub><sup>3–</sup> ligands. <strong>c</strong> Visualization of the complex structure of the 5OFB_B protein with Zn<sup>2+</sup> and ATP ligands. Cyan and yellow spheres represent Zn<sup>2+</sup> and Ca<sup>2+</sup>, respectively. PO<sub>4</sub><sup>3-</sup> is represented by four red spheres (oxygen) and one orange sphere (phosphorus), while ATP is represented by a stick model. Additional visualization of the LABind prediction results for the binding regions is provided, and the binding regions are rotated by a certain angle to facilitate observation. True positives, false positives, and false negatives represent binding residues correctly predicted, non-binding residues incorrectly predicted as binding, and binding residues incorrectly predicted as non-binding, respectively.</p></figcaption></figure></section><section id="Sec10"><h3 class="pmc_sec_title">Assisting in molecular docking tasks</h3>
<p id="Par18">To further investigate the potential application of binding sites, we used LABind-predicted binding sites to assist in molecular docking tasks, to both examine how helpful these predictions are for Smina blind docking and explore the potential for further improvement of LABind. We selected CASF-2016 dataset<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> to evaluate performance on molecular docking tasks and used Smina as the docking program. Subsequently, LABind failed to identify binding sites on 16 complexes, resulting in 269 protein-ligand complexes for further analysis. Since all complexes in the CASF-2016 dataset consist of a single protein and a single ligand, we utilized the average position and spatial extent of the binding residues predicted by LABind to determine the center and radius of the search box for the specific ligand in each complex (binding site-guided docking). For comparison, we employed both blind docking and cognate ligand-guided docking. Blind docking was performed using the geometric center of the protein and a box that encompasses the whole structure. Cognate ligand-guided docking was performed based on the position of the native ligand using the –-autobox_ligand option in Smina. It is noted that this option denotes using almost known binding sites. Supplementary Note <a href="#MOESM1" class="usa-link">6</a> provides a description of three experimental details and procedures. We assessed the accuracy of ligand poses, defining a successful docking outcome by a strict RMSD threshold of  &lt; 2 Å. The success rate of the top 20 structures was used to compare the effectiveness of docking with and without LABind-predicted binding sites. As shown in Fig. <a href="#Fig5" class="usa-link">5</a>, docking using LABind-predicted binding sites improves the average top-20 success rate by 19.1% over blind docking. However, it still underperforms by 21.9% compared to docking with the cognate ligand, indicating room for further improvement. These results demonstrate the potential of LABind in enhancing blind docking scenarios and highlight directions for future enhancement.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. The top-n success rate under three different experimental conditions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc34/12365077/d8fcbfc2ef5b/41467_2025_62899_Fig5_HTML.jpg" loading="lazy" id="d33e1432" height="548" width="667" alt="Fig. 5"></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>‘Blind’, ‘Binding sites’, and ‘Cognate ligands’ represent the success rates of blind docking, binding site-guided docking, and cognate ligand-guided docking, respectively.</p></figcaption></figure></section><section id="Sec11"><h3 class="pmc_sec_title">Validation of LABind on binding sites of the SARS-CoV-2 NSP3 macrodomain</h3>
<p id="Par19">NSP3 macrodomain is an essential component of the replication and transcription complex. It participates in the replication and infection processes of coronaviruses through various mechanisms and is one of the important factors in the interaction between the virus and the host<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. We selected the protein chain from SARS-CoV-2 NSP3 macrodomain (PDB ID: 5SQQ, chain: A)<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> and the unseen ligand (Chemical ID: QU3), as an example for demonstration. The ground-truth binding sites were derived from the BioLiP2 database. We selected LABind trained on the DS3 dataset to predict the binding site. We utilized LABind and similar methods to predict the ligand binding site of the example protein. Supplementary Fig. <a href="#MOESM1" class="usa-link">2a</a> presents the structure of the protein-ligand complex. As depicted in Supplementary Fig. <a href="#MOESM1" class="usa-link">2b</a>, LABind accurately predicted all binding residues without falsely identifying any additional non-binding residues. The prediction result of LigBind is presented in Supplementary Fig. <a href="#MOESM1" class="usa-link">2c</a>. LigBind predicted 13 binding residues, including 5 true positives and 8 false positives. One binding residue was a false negative, and the incorrectly predicted residues exhibiting a more dispersed distribution on the binding residues. The visualizations of P2Rank, DeepSurf and DeepPocket for this case are also shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">2d–f</a>, respectively. Methods targeting the prediction of binding site centers exhibit a tendency to misidentify non-binding residues. This indicates that LABind not only enables the joint prediction of binding residues and binding site centers, but also offers certain assistance in practical applications.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Learned representations of protein and ligand</h3>
<p id="Par20">We wish to justify that the results we presented in the previous sections are not coincidental but result from the intentional design and training of LABind. Therefore, we presented investigations into the protein and ligand features of LABind. We found that the protein representation is the best comprehensive representation of the protein sequence and structure. We also discovered that incorporating ligand attributes significantly enhances the performance of LABind.</p>
<p id="Par21">Ablation studies were conducted to verify the contribution of each protein feature source of LABind. As illustrated in Fig. <a href="#Fig6" class="usa-link">6</a>a, we compared the impact of removing four main features—protein embeddings (Ankh features), residue-residue interactions of the protein representations, protein-DSSP embeddings (Ankh features + DSSP features), and protein graphs on the performance of binding site identification on the DS1 test set with six different ligands for demonstration. We selected metal ions (Zn<sup>2+</sup> and Ca<sup>2+</sup>), anions (PO<sub>4</sub><sup>3-</sup>), and small molecules (ATP, HEM, and GDP) and reported the average AUPR on the DS1 test set. Removing features meant that regardless of the input, we replaced the features with a matrix of all ones. When removing the protein embeddings from LABind, the average AUPR decreases by 0.043 across all 19 ligands (average AUPR of 0.573). This confirms that the absence of protein sequence information leads to a decrease in LABind’s performance. The removal of residue-residue interactions (retention of solely the positional residue-residue indices) results in a 0.132 decrease in average AUPR. Neglecting the relative positions of residues results in inadequate performance of LABind (average AUPR of 0.484). The removal of protein-DSSP features within LABind leads to a decrease of 0.048 in average AUPR. When protein graphs (node spatial features and edge spatial features) are removed, an action that randomizes residue-residue interactions as well as the spatial information of residue atoms, LABind experiences the most significant decrease in AUPR. This ablated model underperforms LABind by 0.132, 0.284, 0.135, 0.200, 0.200, and 0.224 in AUPR on Zn<sup>2+</sup>, Ca<sup>2+</sup>, PO<sub>4</sub><sup>3-</sup>, ATP, GDP, and HEM test sets. Given the complete lack of protein graph information, the model’s performance declines significantly (average AUPR from 0.616 to 0.433). Hence, it is evident that protein graphs are of paramount importance within the LABind architecture.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. The effects of protein features and model designs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365077_41467_2025_62899_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc34/12365077/eeadecd65653/41467_2025_62899_Fig6_HTML.jpg" loading="lazy" id="d33e1506" height="498" width="800" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Ablation studies on the six ligands in the DS1 test set. ‘w/o embedding’, ‘w/o edge’, ‘w/o dssp’ and ‘w/o graph’ denote removing protein embedding, removing residue-residue interactions, removing protein-DSSP embedding and using only the protein-DSSP embedding as input, respectively. AVG represents the average AUPR across all ligands. <strong>b</strong> Performance improvement of LABind over LABind without ligand information on the six ligand test sets in the DS1 dataset. AVG represents average values across all ligands. MCC and AUPR are colored in blue and yellow, respectively. Light colors indicate results obtained without using ligand information, while textured dark colors indicate performance improvements achieved with ligand information. The values indicate the improvement achieved by incorporating ligand information. <strong>c</strong> Performance comparison of LABind and LABind trained in a grouped manner. LABind denotes training on all ligands, and Group denotes training with categories based on ions and small molecules. <strong>d</strong> Visualization of the distributions of residue representations on six ligands before (left) and after (right) the attention-based learning interaction. Non-binding residues refer to amino acids that are not part of the binding sites for these six ligands and are represented in gray. Colored points represent binding residues for their respective ligands (e.g., points labeled Zn<sup>2+</sup> are Zn<sup>2+</sup> binding residues).</p></figcaption></figure><p id="Par22">Ligand features provide pivotal information for identifying binding sites. The MCCs and AUPRs of LABind without ligand information and LABind are illustrated in Fig. <a href="#Fig6" class="usa-link">6</a>b. The results show that LABind achieved improvements of 9.0–34.4% in MCC and 15.8–57.5% in AUPR on the six test sets over LABind without ligand information. When ligand information is removed, the average MCC and AUPR decreases by 0.110 and 0.152, respectively. The results clearly demonstrate that the model without ligand representations performs significantly worse than LABind. This indicates that cross-attention can effectively integrate features of both ligands and proteins. As presented in Table <a href="#Tab2" class="usa-link">2</a>, multi-ligand-oriented methods show a distinct difference in recognizing binding sites for ions and small molecules. Therefore, we categorized the DS1 dataset into small molecules and ions for separate training, and compared the performance of these two models on the test set. As shown in Fig. <a href="#Fig6" class="usa-link">6</a>c and Supplementary Table <a href="#MOESM1" class="usa-link">3</a>, the average MCC of LABind trained on the entire DS1 training set is 3.6% higher than that of LABind trained in a grouped manner. We believe this is because a unified model can learn from a larger and more diverse set of ligands, which contributes to better overall performance, whereas segregated training limits the data available to each model.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Learned interactions between protein and ligand</h3>
<p id="Par23">In addition to the importance of protein and ligand features for binding site prediction tasks, the integration of features from both ligands and receptors can also enhance prediction accuracy. However, effectively integrating these features remains a challenge. The deep learning architecture of LABind helps to quantitatively solve this integration task. To validate the roles of cross-attention and the graph network, we visualized the distributions of residue representations on the DS1 test set of Zn<sup>2+</sup>, Ca<sup>2+</sup>, PO<sub>4</sub><sup>3-</sup>, ATP, GDP and HEM using t-SNE<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. We performed non-linear dimensionality reduction visualization on the residue representations to demonstrate the positive impact of the attention-based learning interaction on the prediction task. Due to the large number of residues in the test sets, we randomly selected 50 binding proteins for each ligand. As shown in Fig. <a href="#Fig6" class="usa-link">6</a>d, the representations of binding and non-binding residues before the attention-based learning interaction exhibit highly overlapping distributions, making them difficult to distinguish. In contrast, the residue representations after the attention-based learning interaction considerably improve the discrimination between binding residues and non-binding residues. Additionally, a noticeable gap is observed between the main distribution areas of binding residues and non-binding residues, indicating that LABind effectively captures the differences between them. LABind clusters the binding residues of different ligands into the same regions because the optimization objective during model training is to distinguish whether a residue is part of a binding site.</p></section></section><section id="Sec14"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par24">In this study, we proposed LABind, a structure-based method to predict the binding sites of proteins for all ions and small molecules in a ligand-aware manner. LABind utilizes a cross-attention mechanism to facilitate interactions between ligand representations and residue representations, learning the intrinsic relationships of protein-ligand binding. Additionally, LABind integrates the atomic positions of proteins with embeddings from pre-trained language models, achieving a robust representation of proteins. By utilizing graph networks, it effectively learns the domain information of protein graphs. LABind is not only capable of accurately predicting unseen ligands but also outperforms competing methods in specific ligand recognition. Although LABind is a structure-based method, we also developed a program that enables a sequence-based approach by utilizing structures predicted by ESMFold.</p>
<p id="Par25">To comprehensively analyze the performance of LABind, we conducted multiple experiments: comparative experiments on three benchmark datasets and one additional dataset; demonstrations of the same protein binding with different ligands; auxiliary molecular docking experiments; predictions of binding sites for the SARS-CoV-2 NSP3 macrodomain protein with an unseen ligand; ablation studies; and visualizations of hidden residue representations. We first benchmarked LABind against other single-ligand-oriented methods using the DS1 dataset, and LABind achieved an improvement of 0.109 in the average MCC compared to LigBind. We further trained and tested LABind on a multi-ligand-oriented dataset DS2, and the predictive performance of LABind also surpassed existing multi-ligand-oriented methods, demonstrating its competitiveness in predicting known ligands. To test the effectiveness of LABind on unseen ligands, we additionally established a dataset DS3 for training and testing unseen ligands. Despite the increased difficulty, compared with LigBind, LABind demonstrated an average improvement of 0.131 in MCC and 0.125 in F1, showing both high precision and robustness. Furthermore, LABind outperformed competing methods in binding site center localization across three benchmark datasets and the COACH64 dataset. Specifically, LABind achieved an average increase of at least 8.0% in DCC success rate and 6.5% in DCA success rate across all evaluated datasets. Through the application of LABind in predicting binding sites for the same protein with various ligands, we demonstrated that LABind can accurately differentiate the binding sites among distinct ligands. Moreover, when combining the binding sites identified by LABind with Smina, we found that these binding sites can effectively assist in molecular docking tasks. Additionally, a case study on SARS-CoV-2 NSP3 macrodomain-related proteins highlighted LABind’s competitiveness in practical applications. To uncover the roles of input features and computational components in LABind, we conducted ablation studies and visualized residue representations derived from the attention-based learning interaction. Our exploration of the computational characteristics of LABind yielded several key insights: (1) The combination of Ankh, DSSP, and spatial information into a graph is the most effective representation for LABind; (2) The involvement of ligand representations is key to LABind; (3) Training all ligands collectively can be beneficial for LABind. By visually analyzing the evolution of protein features before and after the attention-based learning interaction, it is evident that LABind can effectively capture binding site information through protein-ligand interactions.</p>
<p id="Par26">The features, design, and training of LABind result in consistent performance improvements across datasets. This achievement can be attributed to several key factors: First, the rich feature representation reduces the difficulty of fitting. Second, cross-attention effectively integrates ligand features with protein features. Third, LABind is a unified model capable of handling various types of ligands, which avoids the need to train a separate model for each ligand. This multi-ligand capable model captures both shared representations across different ligand binding sites and distinct representations specific to each binding site. However, LABind currently lacks the capability to identify protein-macromolecule binding sites. In summary, we believe that LABind provides certain assistance in the work related to protein binding sites, and it can offer help in understanding protein functions and in the field of drug design. It can also help narrow down the search range for other tasks related to binding sites, such as molecular docking. In the future, we will further expand our method to predict binding sites for other macromolecules and other functional sites, such as phosphorylation sites, peptide binding sites, and protein-protein interaction sites, to provide more comprehensive and effective assistance in biological research.</p></section><section id="Sec15"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec16"><h3 class="pmc_sec_title">Benchmark datasets</h3>
<section id="Sec17"><h4 class="pmc_sec_title">DS1: dataset for literature single-ligand-oriented methods</h4>
<p id="Par27">The dataset created by LigBind contains ligand-specific sets of 1159 ligands, pre-training sets consisting of 1301 ligands, and unseen sets of 16 ligands. We selected 19 ligands from LigBind’s ligand-specific dataset to construct the DS1 benchmark dataset. The details of the DS1 dataset are described in Supplementary Note <a href="#MOESM1" class="usa-link">7</a>. The proteins in these datasets were derived from the BioLiP<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> database, released in February 2021. To ensure low sequence similarity between the training and test sets, redundant proteins sharing more than 30% sequence identity with any protein in the test set were removed using CD-HIT<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Proteins were divided into training and test sets based on their release date relative to January 2017, with those released before this date assigned to the training set and those released after assigned to the test set. In this study, we selected a subset of ions and small molecules from the LigBind dataset to test and evaluate the performance of LABind. Notably, the ligands in the test set appear in the corresponding training sets.</p></section><section id="Sec18"><h4 class="pmc_sec_title">DS2: dataset for multi-ligand-oriented methods</h4>
<p id="Par28">The dataset was composed of 10 ligands including biomacromolecules (DNA, RNA, peptide and protein), small molecules (ATP and HEM) and metal ions (Zn<sup>2+</sup>, Ca<sup>2+</sup>, Mg<sup>2+</sup>, and Mn<sup>2+</sup>). We focus on the identification of binding sites for proteins with small molecules and ions. Therefore, we excluded biomacromolecules to construct the DS2 dataset, which ultimately includes six ligands (ATP, HEM, Zn<sup>2+</sup>, Ca<sup>2+</sup>, Mg<sup>2+</sup>, and Mn<sup>2+</sup>). The small molecule dataset and the metal ion dataset were constructed from the BioLiP database released on 29 March 2023 and 29 December 2021, respectively. For small molecule datasets, proteins released before 1 January 2021 were assigned to the training set, while the remaining proteins were assigned to the test set. For metal ions datasets, proteins released before 1 January 2020 were assigned to the training set, while the remaining proteins were assigned to the test set. Supplementary Note <a href="#MOESM1" class="usa-link">8</a> shows the details of the DS2 dataset. Consistent with DS1, the ligands in the test set also appear in the corresponding training set.</p></section><section id="Sec19"><h4 class="pmc_sec_title">DS3: recent assembled dataset for ligand-aware model</h4>
<p id="Par29">To ensure the predictive accuracy of the model on unseen ligands, we constructed a dataset (DS3) from the BioLiP2 database<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> released on April 3, 2024. The DS3 dataset is a comprehensive dataset for predicting protein binding sites with small molecules and ions. Figure <a href="#Fig7" class="usa-link">7</a> illustrates the construction process of the DS3 dataset. First, we removed protein chains with resolutions of  &gt;3.0 Å and lengths of  &gt;1500 residues. Similar to previous studies<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, we used MMseqs2<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> to remove redundancy from receptor proteins. Proteins with protein sequence identity below 30% and alignment coverage greater than 30% were retained. Next, we used protein data released before January 1, 2021, for the training set. Data released after this date that contained ligands present in the training set were used as the picking set, whereas those containing ligands not observed in the training set were used as the test set. Finally, we removed protein sequences from the training and picking sets that shared more than 30% sequence identity with the protein sequences in the test set. This approach ensures that the ligands in the DS3 test set were not seen in the training set and that the protein structures are sufficiently distinct, thereby enabling a better assessment of the model’s generalization capability on unseen data. We removed ligands with fewer than two binding protein chains in the training, picking, and test sets to reduce potential bias introduced by individual binding samples. We used the training set to train the model’s weights, the picking set to determine the threshold for binary classification, and the test set to evaluate the model’s generalization performance on unseen ligands. Due to anomalies encountered with certain ligands and proteins when utilizing the LigBind online server, which rendered them unprocessable for prediction, these ligands were consequently excluded from the DS3 test set. A description of anomalies with the LigBind web server is provided in Supplementary Note <a href="#MOESM1" class="usa-link">9</a>. Specifically, the training set contains 21651 protein chains and 3651 ligands; the picking set contains 4024 protein chains and 1229 ligands; and the unseen test set contains 1049 protein chains and 474 ligands. Supplementary Note <a href="#MOESM1" class="usa-link">10</a> and Supplementary Table <a href="#MOESM1" class="usa-link">4</a> present MMseqs2 command operations and the Chemical IDs of the unseen ligands in the test set, respectively. Here, a residue is defined as a binding residue if the minimum atomic pair distance between the residue’s non-hydrogen atoms and the ligand’s non-hydrogen atoms is less than the sum of their Van der Waals radii plus 0.5 Å.</p>
<figure class="fig xbox font-sm" id="Fig7"><h5 class="obj_head">Fig. 7. The construction of training set, picking set and test set for DS3 dataset.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365077_41467_2025_62899_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc34/12365077/e560c526f985/41467_2025_62899_Fig7_HTML.jpg" loading="lazy" id="d33e1635" height="451" width="800" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The screening process from the BioLiP2 database. <strong>b</strong> The process of splitting DS3 dataset.</p></figcaption></figure></section><section id="Sec20"><h4 class="pmc_sec_title">Data splitting during training</h4>
<p id="Par30">Consistent with previous studies<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, DS1 and DS2 datasets were divided into training/test sets according to stringent sequence diversity thresholds (maximum 30% sequence identity between test and training sequences). Each training set was partitioned into five folds. For the same set of hyperparameters, five models were trained on four folds and validated on the remaining fold. Notably, the validation was only performed on the fold against those other four training folds based on the training set, without using the held-out test set. The final trained models with the optimized hyperparameters were used in the prediction task on the test set. The final predictions on the test set were obtained by averaging the outputs of the five converged models. The DS3 dataset was also divided into training/picking/test sets following sequence diversity thresholds. Additionally, none of the ligands in the test set appear in the training set. The picking set of the DS3 dataset is only used to determine the cutoff thresholds for binary classification on DS3 test set when required by certain evaluation metrics (e.g., MCC and F1). We emphasize that the picking set and the training set are both sufficiently distinct from the test set.</p></section></section><section id="Sec21"><h3 class="pmc_sec_title">Feature representation</h3>
<section id="Sec22"><h4 class="pmc_sec_title">Protein receptors</h4>
<p id="Par31">LABind utilizes the protein pre-trained language model Ankh to obtain protein sequence embeddings without multiple sequence alignment. Ankh is an encoder-decoder transformer<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> with ConvBERT<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> for generating amino acid-level embeddings. Ankh is an optimized protein language model focused on improving performance through protein-specific optimization rather than just scaling up the model size. We use the 1536-dimensional embeddings generated by Ankh as protein embeddings. Compared with ESM-2 (3B), Ankh occupies less GPU memory, enabling it to process proteins with longer sequences. LABind utilizes some structural properties predicted by DSSP, including relative solvent accessibility (RSA) and secondary structure types. We integrate protein embeddings from Ankh with DSSP features to construct protein-DSSP embeddings, which are subsequently combined with spatially-based protein graphs to generate the final protein representations.</p></section><section id="Sec23"><h4 class="pmc_sec_title">Small molecule ligands and ion ligands</h4>
<p id="Par32">MolFormer can learn molecular structures solely from SMILES sequences without precise 3D data. Its framework primarily utilizes rotary positional embeddings and linear attention mechanisms, making it both scalable and computationally efficient. Therefore, for a ligand, LABind leverages MolFormer to obtain a 768-dimensional embedding of the ligand through its SMILES sequence. The ligand representations are directly derived from MolFormer’s outputs.</p></section><section id="Sec24"><h4 class="pmc_sec_title">Min-max normalization</h4>
<p id="Par33">To accelerate model convergence, we apply min-max normalization to the protein embeddings, DSSP features, and ligand representations as follows:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e1675" display="block"><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi mathvariant="normal">norm</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi mathvariant="bold">x</mi><mo>−</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>min</mi></mrow></msub></mrow><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>min</mi></mrow></msub></mrow></mfrac><mo>.</mo></math></td>
<td class="label">1</td>
</tr></table>
<p>Here, <strong>x</strong> is the original feature vector, <strong>x</strong><sub>norm</sub> is the normalized feature that is finally input into the model, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e1724"><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>min</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e1738"><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>max</mi></mrow></msub></math></span> are the minimum and maximum values of the feature type on the training dataset, respectively.</p></section><section id="Sec25"><h4 class="pmc_sec_title">Graph converter module</h4>
<p id="Par34">Ingraham et al.<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup> constructed a protein representation graph based on the protein backbone and 3D coordinates through relative spatial encodings and relative positional encodings. We adopt a similar graph construction method to construct the protein graph. For each residue <em>i</em>, we construct a residue coordinate system based on the positions of the N, C<sub><em>α</em></sub>, and C atoms. The specific definition is as follows:</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e1768" display="block"><mfenced open="{"><mrow><mtable><mtr><mtd columnalign="center"><msub><mrow><mi mathvariant="bold">Q</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><mrow><mo>[</mo><mrow><msub><mrow><mi mathvariant="bold">b</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">n</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">b</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>×</mo><msub><mrow><mi mathvariant="bold">n</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo>]</mo></mrow></mtd></mtr><mtr><mtd columnalign="center"><msub><mrow><mi mathvariant="bold">b</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mrow><mi mathvariant="bold">u</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mo>∥</mo><msub><mrow><mi mathvariant="bold">u</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>∥</mo></mrow></mfrac><mo>,</mo><msub><mrow><mi mathvariant="bold">n</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mrow><mi mathvariant="bold">u</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>×</mo><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mo>∥</mo><msub><mrow><mi mathvariant="bold">u</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>×</mo><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>∥</mo></mrow></mfrac></mtd></mtr><mtr><mtd columnalign="center"><msub><mrow><mi mathvariant="bold">u</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">N</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi>α</mi><mi>i</mi></mrow></msub></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi>α</mi><mi>i</mi></mrow></msub></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub></mtd></mtr></mtable></mrow></mfenced><mo>,</mo></math></td>
<td class="label">2</td>
</tr></table>
<p>where <strong>Q</strong><sub><em>i</em></sub> represents the residue coordinate system, <strong>b</strong><sub><em>i</em></sub> is the negative bisector of the angle formed by the N<sub><em>i</em></sub>, C<sub><em>α</em><em>i</em></sub>, and C<sub><em>i</em></sub> atoms, and <strong>n</strong><sub><em>i</em></sub> is a unit vector normal to this plane. <strong>u</strong><sub><em>i</em></sub> and <strong>v</strong><sub><em>i</em></sub> are computed from the backbone atomic coordinates, and <strong>x</strong> represents the position of the corresponding atom. Fig. <a href="#Fig1" class="usa-link">1</a>b shows the architecture of the graph converter module.</p>
<p id="Par35">(i) Node spatial features. Using the residue coordinate system, with C<sub><em>α</em></sub> as the origin, we calculate the distance and direction of each atom within each residue to supplement the pre-trained embeddings. Given the clear correlation between protein-ligand interactions and surface features<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, we calculate the protein surface using MSMS<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> and compute the distance from each residue to the nearest atom S on the surface. Thus, a residue <em>i</em> is composed of an atomic set {N<sub><em>i</em></sub>, C<sub><em>α</em><em>i</em></sub>, C<sub><em>i</em></sub>, O<sub><em>i</em></sub>, R<sub><em>i</em></sub>, S<sub><em>i</em></sub>} (R<sub><em>i</em></sub> represents the centroid of all heavy atoms in the side chain). We calculate the internal distance and directional features based on the residue coordinate system. We use a Gaussian radial basis function RBF( ⋅ ) to encode pairwise atomic distances <em>D</em>, and then concatenate the resulting distance embeddings accordingly. For the directional feature, the directions from the C<sub><em>α</em></sub> atom to other atoms are calculated, then transformed into the residue coordinate system, and finally flattened into a vector. The directional feature is composed of components from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e2072"><mrow><mo>[</mo><mrow><msub><mrow><mi mathvariant="bold">d</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">N</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">d</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">d</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">O</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">d</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">R</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">d</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">S</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub></mrow><mo>]</mo></mrow></math></span>, for example, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e2143"><msub><mrow><mi mathvariant="bold">d</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">R</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub><mo>=</mo><msubsup><mrow><mi mathvariant="bold">Q</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">T</mi></mrow></msubsup><mfrac><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">R</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi>α</mi><mi>i</mi></mrow></msub></mrow></msub></mrow><mrow><mo>∥</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">R</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi>α</mi><mi>i</mi></mrow></msub></mrow></msub><mo>∥</mo></mrow></mfrac></math></span>. Additionally, since the protein backbone consists of the atoms N, C<sub><em>α</em></sub> and C, we calculate the bond angles and dihedral angles for the atoms on the backbone.</p>
<p id="Par36">(ii) Edge spatial features. Based on the residue coordinate system, the relation between the neighboring coordinate systems is mainly determined by C<sub><em>α</em></sub>. Similar to node spatial features, edge spatial features include directions, distances, and orientations. For edge <strong>e</strong><sub><em>i</em><em>j</em></sub>, we calculate the distances from all atoms ({N<sub><em>j</em></sub>, C<sub><em>α</em><em>j</em></sub>, C<sub><em>j</em></sub>, O<sub><em>j</em></sub>, R<sub><em>j</em></sub>, S<sub><em>j</em></sub>}) in residue <em>j</em> to the C<sub><em>α</em></sub> atom of residue <em>i</em>, as well as the directional information based on the coordinate system of residue <em>i</em>. Namely, the directional feature is <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e2284"><msubsup><mrow><mi mathvariant="bold">Q</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">T</mi></mrow></msubsup><mfrac><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">A</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi>α</mi><mi>i</mi></mrow></msub></mrow></msub></mrow><mrow><mo>∥</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">A</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi>α</mi><mi>i</mi></mrow></msub></mrow></msub><mo>∥</mo></mrow></mfrac></math></span>, and the distance feature is <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e2354"><mi mathvariant="normal">RBF</mi><mrow><mo>(</mo><mrow><mo>∣</mo><mo>∣</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">A</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi>α</mi><mi>i</mi></mrow></msub></mrow></msub><mo>∣</mo><mo>∣</mo></mrow><mo>)</mo></mrow></math></span>, where A<sub><em>j</em></sub> represents all atoms in residue <em>j</em>. The spatial relationship between different residue coordinate systems is represented by orientation feature <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e2403"><mi mathvariant="normal">q</mi><mrow><mo>(</mo><mrow><msubsup><mrow><mi mathvariant="bold">Q</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">T</mi></mrow></msubsup><msub><mrow><mi mathvariant="bold">Q</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow><mo>)</mo></mrow></math></span>, where an orientation encoding function q( ⋅ ) is employed to encode the quaternion representation of the spatial rotation matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e2433"><msubsup><mrow><mi mathvariant="bold">Q</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">T</mi></mrow></msubsup><msub><mrow><mi mathvariant="bold">Q</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span><sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>.</p></section><section id="Sec26"><h4 class="pmc_sec_title">Attention-based learning interaction</h4>
<p id="Par37">The attention-based learning interaction is the most crucial component in LABind, with its specific framework shown in Fig. <a href="#Fig1" class="usa-link">1</a>c. Multi-head neighbor-attention and multi-head cross-attention are two key modules for learning protein representations and interacting ligand information.</p>
<p id="Par38">(i) Multi-head neighbor-attention for protein representations. We process the residue representations using neighbor-attention by calculating the attention scores of the neighboring residues relative to the target residue, and then aggregating these scores to update the target residue. We update the residue representations via the neighbor-attention module as follows:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e2469" display="block"><msubsup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mi>i</mi></mrow><mrow><mo>′</mo><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mo>=</mo><msubsup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mi>i</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mo>+</mo><msubsup><mrow><mi mathvariant="bold">W</mi></mrow><mrow><mi mathvariant="normal">O</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mfenced close=")" open="("><mrow><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>j</mi><mo>∈</mo><mi>N</mi><mrow><mo>(</mo><mrow><mi>i</mi></mrow><mo>)</mo></mrow><mo>∪</mo><mi>i</mi></mrow></msub><msubsup><mrow><mi>α</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><msubsup><mrow><mi mathvariant="bold">W</mi></mrow><mrow><mi mathvariant="normal">V</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mrow><mo>(</mo><mrow><msubsup><mrow><mi mathvariant="bold">e</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mo>∥</mo><msubsup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup></mrow><mo>)</mo></mrow></mrow></mfenced><mo>,</mo></math></td>
<td class="label">3</td>
</tr></table>
<p>where the attention coefficients are calculated according to the following formula:</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e2591" display="block"><mfenced open="{"><mrow><mtable><mtr><mtd columnalign="left"><msubsup><mrow><mi>α</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mo>=</mo><mfrac><mrow><mi>exp</mi><mrow><mo>(</mo><mrow><msubsup><mrow><mi>w</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup></mrow><mo>)</mo></mrow></mrow><mrow><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>j</mi><mo>∈</mo><mi>N</mi><mrow><mo>(</mo><mrow><mi>i</mi></mrow><mo>)</mo></mrow><mo>∪</mo><mi>i</mi></mrow></msub><mi>exp</mi><mrow><mo>(</mo><mrow><msubsup><mrow><mi>w</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup></mrow><mo>)</mo></mrow></mrow></mfrac></mtd></mtr><mtr><mtd columnalign="left"><msubsup><mrow><mi>w</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mo>=</mo><mfrac><mrow><msup><mrow><mfenced close=")" open="("><mrow><msubsup><mrow><mi mathvariant="bold">W</mi></mrow><mrow><mi mathvariant="normal">Q</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><msubsup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mi>i</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup></mrow></mfenced></mrow><mrow><mi mathvariant="normal">T</mi></mrow></msup><mfenced close=")" open="("><mrow><msubsup><mrow><mi mathvariant="bold">W</mi></mrow><mrow><mi mathvariant="normal">K</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mrow><mo>(</mo><mrow><msubsup><mrow><mi mathvariant="bold">e</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mo>∥</mo><msubsup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup></mrow><mo>)</mo></mrow></mrow></mfenced></mrow><mrow><msqrt><mrow><mi>d</mi></mrow></msqrt></mrow></mfrac></mtd></mtr></mtable></mrow></mfenced><mo>.</mo></math></td>
<td class="label">4</td>
</tr></table>
<p>Here, <strong>h</strong><sub><em>i</em></sub>, <strong>h</strong><sub><em>j</em></sub>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e2781"><msubsup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mi>i</mi></mrow><mrow><mo>′</mo></mrow></msubsup></math></span> are the representations corresponding to the residues in the protein at the layer <em>l</em>. <strong>e</strong><sub><em>i</em><em>j</em></sub> represents the interaction between residue <em>i</em> and residue <em>j</em>. <strong>W</strong><sub>Q</sub>, <strong>W</strong><sub>K</sub>, and <strong>W</strong><sub>V</sub> are the learnable weight matrices. <em>d</em> is the dimension of the hidden layer, and ∥ denotes vector concatenation. Additionally, we apply multi-head attention to transform residue representations and residue-residue interactions multiple times to capture diverse information. The outputs are directly concatenated and linearly projected using a learned matrix <strong>W</strong><sub>O</sub>.</p>
<p id="Par39">(ii) Multi-head cross-attention in protein-ligand interactions. We utilize cross-attention to achieve information interaction between ligand representations and protein representations. The cross-attention mechanism is implemented in Transformer<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> to allow each position in the decoder to attend to all positions within the input sequence. Previous work has demonstrated that the cross-attention mechanism can effectively combine different types of information<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. The architecture and calculation of multi-head cross-attention are shown in Fig. <a href="#Fig1" class="usa-link">1</a>d. We use the residue representations as queries and multiply them by the matrix <strong>W</strong><sub>q</sub>. The ligand representation is used as both keys and values, which are projected using matrices <strong>W</strong><sub>k</sub> and <strong>W</strong><sub>v</sub>, respectively. We then compute the dot product of the queries and keys to get the attention coefficients, which are transformed into a probability distribution using the softmax function. Lastly, the attention coefficients are applied to the ligand representation to focus on the relevant information, and then are integrated into the residue representations through a learnable matrix <strong>W</strong><sub>o</sub>. Formally, the definition of the cross-attention mechanism between proteins and ligands is as follows:</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><math id="d33e2866" display="block"><mfenced open="{"><mrow><mtable><mtr><mtd columnalign="left"><msup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mo>″</mo><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msup><mo>=</mo><msup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mo>′</mo><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msup><mo>+</mo><msubsup><mrow><mi mathvariant="bold">W</mi></mrow><mrow><mi mathvariant="normal">o</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mfenced close=")" open="("><mrow><msubsup><mrow><mi>α</mi></mrow><mrow><mi mathvariant="normal">m</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><msubsup><mrow><mi mathvariant="bold">W</mi></mrow><mrow><mi mathvariant="normal">v</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><msup><mrow><mi mathvariant="bold">m</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msup></mrow></mfenced></mtd></mtr><mtr><mtd columnalign="left"><msubsup><mrow><mi>α</mi></mrow><mrow><mi mathvariant="normal">m</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mo>=</mo><mi mathvariant="normal">softmax</mi><mfenced close=")" open="("><mrow><mfrac><mrow><mfenced close=")" open="("><mrow><msubsup><mrow><mi mathvariant="bold">W</mi></mrow><mrow><mi mathvariant="normal">q</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><msup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mo>′</mo><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msup></mrow></mfenced><msup><mrow><mfenced close=")" open="("><mrow><msubsup><mrow><mi mathvariant="bold">W</mi></mrow><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><msup><mrow><mi mathvariant="bold">m</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msup></mrow></mfenced></mrow><mrow><mi mathvariant="normal">T</mi></mrow></msup></mrow><mrow><msqrt><mrow><mi>d</mi></mrow></msqrt></mrow></mfrac></mrow></mfenced></mtd></mtr></mtable></mrow></mfenced><mo>.</mo></math></td>
<td class="label">5</td>
</tr></table>
<p id="Par40">Here, <strong>m</strong> represents the ligand representation, <strong>h</strong><sup><em>″</em></sup> represents the updated residue representations, <em>α</em><sub>m</sub> represents the attention coefficients computed between residue representations and the ligand representation, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e3049"><mi mathvariant="normal">softmax</mi></math></span> refers to the softmax function. In this work, we employ the multi-head cross-attention mechanism, and the outputs from these heads are concatenated.</p>
<p id="Par41">(iii) Updating residue-residue interactions and the ligand representation. We not only update the residue representations but also leverage updated residue representations to update the ligand representation and residue-residue interactions. We first perform a linear transformation on <strong>h</strong><sup><em>″</em>(<em>l</em>)</sup> through a feed-forward layer to obtain <strong>h</strong><sup>(<em>l</em>+1)</sup>, as follows:</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><math id="d33e3080" display="block"><msup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow></msup><mo>=</mo><msup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mo>″</mo><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msup><mo>+</mo><mi mathvariant="normal">FFN</mi><mrow><mo>(</mo><mrow><msup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mo>″</mo><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msup></mrow><mo>)</mo></mrow><mo>.</mo></math></td>
<td class="label">6</td>
</tr></table>
<p id="Par42">For the ligand representation, we average and maximize the protein residue representations, concatenate them with the ligand representation, and then pass the result through an MLP to serve as the updated ligand representation. This design aims to reduce model complexity while preserving its predictive performance. The method for updating ligand representation is as follows:</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><math id="d33e3133" display="block"><msup><mrow><mi mathvariant="bold">m</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow></msup><mo>=</mo><msup><mrow><mi mathvariant="bold">m</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msup><mo>+</mo><mi mathvariant="normal">MLP</mi><mfenced close=")" open="("><mrow><mi mathvariant="normal">Mean</mi><mrow><mo>(</mo><mrow><msup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow></msup></mrow><mo>)</mo></mrow><mo>∥</mo><mi mathvariant="normal">Max</mi><mrow><mo>(</mo><mrow><msup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow></msup></mrow><mo>)</mo></mrow><mo>∥</mo><msup><mrow><mi mathvariant="bold">m</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msup></mrow></mfenced><mo>.</mo></math></td>
<td class="label">7</td>
</tr></table>
<p>We update the residue-residue interactions by concatenating the representations of the start residue and end residue. A shared MLP is applied to model all residue-residue interactions, as follows:</p>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><math id="d33e3219" display="block"><msubsup><mrow><mi mathvariant="bold">e</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow></msubsup><mo>=</mo><msubsup><mrow><mi mathvariant="bold">e</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mo>+</mo><mi mathvariant="normal">MLP</mi><mfenced close=")" open="("><mrow><msubsup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mi>i</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow></msubsup><mo>∥</mo><msubsup><mrow><mi mathvariant="bold">e</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mo>∥</mo><msubsup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow></msubsup></mrow></mfenced><mo>.</mo></math></td>
<td class="label">8</td>
</tr></table></section><section id="Sec27"><h4 class="pmc_sec_title">MLP classifier</h4>
<p id="Par43">Finally, the outputs of the last interaction layer are passed through an MLP classifier and a sigmoid function to obtain binding probabilities of each residue for the ligand:</p>
<table class="disp-formula p" id="Equ9"><tr>
<td class="formula"><math id="d33e3309" display="block"><mi>P</mi><mo>=</mo><mi mathvariant="normal">sigmoid</mi><mfenced close=")" open="("><mrow><mi mathvariant="normal">MLP</mi><mrow><mo>(</mo><mrow><msup><mrow><mi mathvariant="bold">h</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>L</mi></mrow><mo>)</mo></mrow></mrow></msup></mrow><mo>)</mo></mrow></mrow></mfenced><mo>,</mo></math></td>
<td class="label">9</td>
</tr></table>
<p>where <strong>h</strong><sup>(<em>L</em>)</sup> represents the residue representations of the last interaction layer, and <em>P</em> represents binding probabilities of each residue for the ligand. LABind utilizes numerous MLPs for learning features, which are defined as follows:</p>
<table class="disp-formula p" id="Equ10"><tr>
<td class="formula"><math id="d33e3352" display="block"><mi mathvariant="normal">MLP</mi><mrow><mo>(</mo><mrow><mi mathvariant="bold">x</mi></mrow><mo>)</mo></mrow><mo>=</mo><mi mathvariant="normal">SiLU</mi><mfenced close=")" open="("><mrow><msub><mrow><mi mathvariant="bold">W</mi></mrow><mrow><mn>2</mn></mrow></msub><mi mathvariant="normal">SiLU</mi><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">W</mi></mrow><mrow><mn>1</mn></mrow></msub><mi mathvariant="bold">x</mi><mo>+</mo><msub><mrow><mi>b</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>+</mo><msub><mrow><mi>b</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></mfenced><mo>.</mo></math></td>
<td class="label">10</td>
</tr></table>
<p id="Par44">Here, <strong>x</strong> represents the input, and the learnable parameters include weights (<strong>W</strong><sub>1</sub>, <strong>W</strong><sub>2</sub>) and biases (<em>b</em><sub>1</sub>, <em>b</em><sub>2</sub>). SiLU<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> represents the sigmoid-weighted linear unit activation function. Additional model algorithms are provided in Supplementary Note <a href="#MOESM1" class="usa-link">11</a>.</p></section></section><section id="Sec28"><h3 class="pmc_sec_title">Implementation details</h3>
<section id="Sec29"><h4 class="pmc_sec_title">Hyperparameter optimization</h4>
<p id="Par45">We used grid search to optimize the hyperparameters of the network. Given that the DS2 training set is the smallest, we optimized the hyperparameters using this dataset. The AUPRs corresponding to different hyperparameters on the DS2 dataset are provided in Supplementary Table <a href="#MOESM1" class="usa-link">5</a>. Specifically, we employed a four-layer graph transformer framework with cross-attention modules, four attention heads, and 256 hidden units. The hyperparameters were as follows: <em>ϵ</em> =0.1, Top<sub>k</sub> = 30, N<sub>rbf</sub> = 8. Here, <em>ϵ</em>, Top<sub>k</sub>, and N<sub>rbf</sub> represent the ratio of Gaussian noise added during training, the number of neighboring residues in the protein representation, and the output dimension of the radial basis function, respectively. Dropout rate and attention dropout rate were set to 0.3 and 0.1 to avoid overfitting, respectively. For model optimization, we employed Adam optimizer<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> with the NoamOpt strategy<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. The model was trained using the binary cross-entropy loss, with a peak learning rate of 4 × 10<sup>−4</sup> and a warmup period of 11 epochs. We trained the model for 70 epochs and employed early stopping if there was no improvement in validation performance for 10 epochs.</p></section><section id="Sec30"><h4 class="pmc_sec_title">Runtime environment</h4>
<p id="Par46">We implemented LABind based on Python v3.8 and Pytorch v2.2.1<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. The versions of other software used are provided in Supplementary Note <a href="#MOESM1" class="usa-link">12</a>. We trained LABind for the DS1 dataset on two Nvidia A100 GPUs, and for the DS2 and DS3 datasets on five Nvidia GeForce RTX 2080Ti GPUs. The choice of hardware depended on the longest protein length in each dataset.</p></section></section><section id="Sec31"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par47">Further information on research design is available in the <a href="#MOESM4" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec32"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365077/bin/41467_2025_62899_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365077/bin/41467_2025_62899_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62899_MOESM2_ESM.pdf</a><sup> (71KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365077/bin/41467_2025_62899_MOESM3_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (3.9MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12365077/bin/41467_2025_62899_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (161.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12365077/bin/41467_2025_62899_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (961.7KB, pdf) </sup>
</div></div></section></section><section id="Sec33"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12365077/bin/41467_2025_62899_MOESM6_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (11.7MB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported in part by the National Natural Science Foundation of China Fund under Grants (62272335 to L.Q. and 62002251 to T.W.), in part by Jiangsu Colleges and Universities QingLan project (L.Q.), in part by University-Industry Collaborative Education Program (L.Q.), in part by Project Funded by the Priority Academic Program Development of Jiangsu Higher Education Institutions, and in part by the Collaborative Innovation Center of Novel Software Technology and Industrialization (L.Q., T.W., and Q.L.).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>L.Q. and Z.Z. proposed the original idea. Z.Z. designed the model and processed the data. L.Q., J.W., T.W. and Q.L. diagnosed and improved the experimental design. Z.Z. analyzed the data and results. Y.J., G.L. and L.N. provided consultation and technical support. Z.Z., L.Q. and Q.L. wrote the paper. All authors, including L.P., Q.C., B.Z. and L.C. read and commented on the manuscript. L.Q. and Q.L. led the project.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par48"><em>Nature Communications</em> thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Unless otherwise stated, all data supporting the results of this study can be found in the article, <a href="#MOESM1" class="usa-link">supplementary</a>, and <a href="#MOESM3" class="usa-link">source data</a> files. Datasets used for all analysis are deposited via Zenodo [10.5281/zenodo.15875006]. For each dataset, the provided resources include sequence data, protein structure data, and annotated binding sites. Other sources include the DS1 dataset [<a href="http://www.csbio.sjtu.edu.cn/bioinf/LigBind/datasets.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.csbio.sjtu.edu.cn/bioinf/LigBind/datasets.html</a>] and the DS2 dataset<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. All protein structure data used in this study are also available from the BioLiP2 database [<a href="https://zhanggroup.org/BioLiP/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zhanggroup.org/BioLiP/</a>]. All ligand SMILES strings used in this study can also be downloaded from the official RCSB website<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> [<a href="https://www.rcsb.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org</a>]. The LABind-predicted binding sites in this study are provided in Supplementary Data <a href="#MOESM3" class="usa-link">1</a>. <a href="#Sec33" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>Our code is available on the GitHub repository [<a href="https://github.com/ljquanlab/LABind" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ljquanlab/LABind</a>] or the Zenodo repository<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par49">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Zhijun Zhang, Lijun Quan.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Lijun Quan, Email: ljquan@suda.edu.cn.</p>
<p>Tingfang Wu, Email: tfwu@suda.edu.cn.</p>
<p>Qiang Lyu, Email: qiang@suda.edu.cn.</p></section><section id="sec34"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62899-0.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Hirota, K. et al. Stepwise chromatin remodelling by a cascade of transcription initiation of non-coding RNAs. <em>Nature</em><strong>456</strong>, 130–134 (2008).
</cite> [<a href="https://doi.org/10.1038/nature07348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18820678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hirota,%20K.%20et%20al.%20Stepwise%20chromatin%20remodelling%20by%20a%20cascade%20of%20transcription%20initiation%20of%20non-coding%20RNAs.%20Nature456,%20130%E2%80%93134%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Chen, K. &amp; Arnold, F. H. Engineering new catalytic activities in enzymes. <em>Nat. Catal.</em><strong>3</strong>, 203–213 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Chen,%20K.%20&amp;%20Arnold,%20F.%20H.%20Engineering%20new%20catalytic%20activities%20in%20enzymes.%20Nat.%20Catal.3,%20203%E2%80%93213%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Brivanlou, A. H. &amp; Darnell, J. E. Signal transduction and the control of gene expression. <em>Science</em><strong>295</strong>, 813–818 (2002).
</cite> [<a href="https://doi.org/10.1126/science.1066355" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11823631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brivanlou,%20A.%20H.%20&amp;%20Darnell,%20J.%20E.%20Signal%20transduction%20and%20the%20control%20of%20gene%20expression.%20Science295,%20813%E2%80%93818%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Hayashi, R., Tashiro, S., Asakura, M., Mitsui, S. &amp; Shionoya, M. Effector-dependent structural transformation of a crystalline framework with allosteric effects on molecular recognition ability. <em>Nat. Commun.</em><strong>14</strong>, 4490 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-40091-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10415384/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37563107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hayashi,%20R.,%20Tashiro,%20S.,%20Asakura,%20M.,%20Mitsui,%20S.%20&amp;%20Shionoya,%20M.%20Effector-dependent%20structural%20transformation%20of%20a%20crystalline%20framework%20with%20allosteric%20effects%20on%20molecular%20recognition%20ability.%20Nat.%20Commun.14,%204490%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Wells, J. A. &amp; McClendon, C. L. Reaching for high-hanging fruit in drug discovery at protein–protein interfaces. <em>Nature</em><strong>450</strong>, 1001–1009 (2007).
</cite> [<a href="https://doi.org/10.1038/nature06526" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18075579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wells,%20J.%20A.%20&amp;%20McClendon,%20C.%20L.%20Reaching%20for%20high-hanging%20fruit%20in%20drug%20discovery%20at%20protein%E2%80%93protein%20interfaces.%20Nature450,%201001%E2%80%931009%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>The UniProt Consortium. UniProt: the universal protein knowledgebase in 2023. <em>Nucleic Acids Res</em>. <strong>51</strong>, D523–D531 (2023).</cite> [<a href="https://doi.org/10.1093/nar/gkac1052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36408920/" class="usa-link">PubMed</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Kim, S. et al. PubChem 2023 update. <em>Nucleic Acids Res.</em><strong>51</strong>, D1373–D1380 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkac956" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36305812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20S.%20et%20al.%20PubChem%202023%20update.%20Nucleic%20Acids%20Res.51,%20D1373%E2%80%93D1380%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Hu, X., Dong, Q., Yang, J. &amp; Zhang, Y. Recognizing metal and acid radical ion-binding sites by integrating ab initio modeling with template-based transferals. <em>Bioinformatics</em><strong>32</strong>, 3260–3269 (2016).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw396" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5079472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27378301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu,%20X.,%20Dong,%20Q.,%20Yang,%20J.%20&amp;%20Zhang,%20Y.%20Recognizing%20metal%20and%20acid%20radical%20ion-binding%20sites%20by%20integrating%20ab%20initio%20modeling%20with%20template-based%20transferals.%20Bioinformatics32,%203260%E2%80%933269%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Lin, Y. F. et al. MIB: metal ion-binding site prediction and docking server. <em>J. Chem. Inf. Model.</em><strong>56</strong>, 2287–2291 (2016).
</cite> [<a href="https://doi.org/10.1021/acs.jcim.6b00407" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27976886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20Y.%20F.%20et%20al.%20MIB:%20metal%20ion-binding%20site%20prediction%20and%20docking%20server.%20J.%20Chem.%20Inf.%20Model.56,%202287%E2%80%932291%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Paiva, V. A. et al. GASS-Metal: identifying metal-binding sites on protein structures using genetic algorithms. <em>Brief. Bioinform.</em><strong>23</strong>, bbac178 (2022).
</cite> [<a href="https://doi.org/10.1093/bib/bbac178" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35595534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paiva,%20V.%20A.%20et%20al.%20GASS-Metal:%20identifying%20metal-binding%20sites%20on%20protein%20structures%20using%20genetic%20algorithms.%20Brief.%20Bioinform.23,%20bbac178%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Yu, D. J. et al. Designing template-free predictor for targeting protein-ligand binding sites with classifier ensemble and spatial clustering. <em>IEEE/ACM Trans. Comput. Biol. Bioinform.</em><strong>10</strong>, 994–1008 (2013).
</cite> [<a href="https://doi.org/10.1109/TCBB.2013.104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24334392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20D.%20J.%20et%20al.%20Designing%20template-free%20predictor%20for%20targeting%20protein-ligand%20binding%20sites%20with%20classifier%20ensemble%20and%20spatial%20clustering.%20IEEE/ACM%20Trans.%20Comput.%20Biol.%20Bioinform.10,%20994%E2%80%931008%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Xia, C. Q., Pan, X. &amp; Shen, H. B. Protein–ligand binding residue prediction enhancement through hybrid deep heterogeneous learning of sequence and structure data. <em>Bioinformatics</em><strong>36</strong>, 3018–3027 (2020).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32091580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia,%20C.%20Q.,%20Pan,%20X.%20&amp;%20Shen,%20H.%20B.%20Protein%E2%80%93ligand%20binding%20residue%20prediction%20enhancement%20through%20hybrid%20deep%20heterogeneous%20learning%20of%20sequence%20and%20structure%20data.%20Bioinformatics36,%203018%E2%80%933027%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Xia, Y., Xia, C. Q., Pan, X. &amp; Shen, H. B. GraphBind: protein structural context embedded rules learned by hierarchical graph neural networks for recognizing nucleic-acid-binding residues. <em>Nucleic Acids Res.</em><strong>49</strong>, e51 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8136796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33577689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia,%20Y.,%20Xia,%20C.%20Q.,%20Pan,%20X.%20&amp;%20Shen,%20H.%20B.%20GraphBind:%20protein%20structural%20context%20embedded%20rules%20learned%20by%20hierarchical%20graph%20neural%20networks%20for%20recognizing%20nucleic-acid-binding%20residues.%20Nucleic%20Acids%20Res.49,%20e51%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Xia, Y., Pan, X. &amp; Shen, H. B. LigBind: identifying binding residues for over 1000 ligands with relation-aware graph neural networks. <em>J. Mol. Biol.</em><strong>435</strong>, 168091 (2023).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2023.168091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37054909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia,%20Y.,%20Pan,%20X.%20&amp;%20Shen,%20H.%20B.%20LigBind:%20identifying%20binding%20residues%20for%20over%201000%20ligands%20with%20relation-aware%20graph%20neural%20networks.%20J.%20Mol.%20Biol.435,%20168091%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Li, P. &amp; Liu, Z. P. GeoBind: segmentation of nucleic acid binding interface on protein surface with geometric deep learning. <em>Nucleic Acids Res.</em><strong>51</strong>, e60 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkad288" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10250245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37070217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20P.%20&amp;%20Liu,%20Z.%20P.%20GeoBind:%20segmentation%20of%20nucleic%20acid%20binding%20interface%20on%20protein%20surface%20with%20geometric%20deep%20learning.%20Nucleic%20Acids%20Res.51,%20e60%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Yuan, Q., Chen, S., Wang, Y., Zhao, H. &amp; Yang, Y. Alignment-free metal ion-binding site prediction from protein sequence through pretrained language model and multi-task learning. <em>Brief. Bioinform.</em><strong>23</strong>, bbac444 (2022).
</cite> [<a href="https://doi.org/10.1093/bib/bbac444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36274238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan,%20Q.,%20Chen,%20S.,%20Wang,%20Y.,%20Zhao,%20H.%20&amp;%20Yang,%20Y.%20Alignment-free%20metal%20ion-binding%20site%20prediction%20from%20protein%20sequence%20through%20pretrained%20language%20model%20and%20multi-task%20learning.%20Brief.%20Bioinform.23,%20bbac444%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Yuan, Q., Tian, C. &amp; Yang, Y. Genome-scale annotation of protein binding sites via language model and geometric deep learning. <em>eLife</em><strong>13</strong>, RP93695 (2024).
</cite> [<a href="https://doi.org/10.7554/eLife.93695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11023698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38630609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan,%20Q.,%20Tian,%20C.%20&amp;%20Yang,%20Y.%20Genome-scale%20annotation%20of%20protein%20binding%20sites%20via%20language%20model%20and%20geometric%20deep%20learning.%20eLife13,%20RP93695%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Krivák, R. &amp; Hoksza, D. P2Rank: machine learning based tool for rapid and accurate prediction of ligand binding sites from protein structure. <em>J. Cheminform.</em><strong>10</strong>, 39 (2018).
</cite> [<a href="https://doi.org/10.1186/s13321-018-0285-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6091426/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30109435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kriv%C3%A1k,%20R.%20&amp;%20Hoksza,%20D.%20P2Rank:%20machine%20learning%20based%20tool%20for%20rapid%20and%20accurate%20prediction%20of%20ligand%20binding%20sites%20from%20protein%20structure.%20J.%20Cheminform.10,%2039%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Mylonas, S. K., Axenopoulos, A. &amp; Daras, P. DeepSurf: a surface-based deep learning approach for the prediction of ligand binding sites on proteins. <em>Bioinformatics</em><strong>37</strong>, 1681–1690 (2021).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btab009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33471069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mylonas,%20S.%20K.,%20Axenopoulos,%20A.%20&amp;%20Daras,%20P.%20DeepSurf:%20a%20surface-based%20deep%20learning%20approach%20for%20the%20prediction%20of%20ligand%20binding%20sites%20on%20proteins.%20Bioinformatics37,%201681%E2%80%931690%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Aggarwal, R., Gupta, A., Chelur, V., Jawahar, C. V. &amp; Priyakumar, U. D. DeepPocket: ligand binding site detection and segmentation using 3D convolutional neural networks. <em>J. Chem. Inf. Model.</em><strong>62</strong>, 5069–5079 (2022).
</cite> [<a href="https://doi.org/10.1021/acs.jcim.1c00799" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34374539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aggarwal,%20R.,%20Gupta,%20A.,%20Chelur,%20V.,%20Jawahar,%20C.%20V.%20&amp;%20Priyakumar,%20U.%20D.%20DeepPocket:%20ligand%20binding%20site%20detection%20and%20segmentation%20using%203D%20convolutional%20neural%20networks.%20J.%20Chem.%20Inf.%20Model.62,%205069%E2%80%935079%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Zhang, C., Zhang, X., Freddolino, P. L. &amp; Zhang, Y. BioLiP2: An updated structure database for biologically relevant ligand–protein interactions. <em>Nucleic Acids Res.</em><strong>52</strong>, D404–D412 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkad630" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37522378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20C.,%20Zhang,%20X.,%20Freddolino,%20P.%20L.%20&amp;%20Zhang,%20Y.%20BioLiP2:%20An%20updated%20structure%20database%20for%20biologically%20relevant%20ligand%E2%80%93protein%20interactions.%20Nucleic%20Acids%20Res.52,%20D404%E2%80%93D412%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Zhang, W. B. &amp; Cheng, S. Z. D. Giant is different: size effects and the nature of macromolecules. <em>Giant</em><strong>1</strong>, 100011 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20W.%20B.%20&amp;%20Cheng,%20S.%20Z.%20D.%20Giant%20is%20different:%20size%20effects%20and%20the%20nature%20of%20macromolecules.%20Giant1,%20100011%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Elnaggar, A. et al. Ankh: optimized protein language model unlocks general-purpose modelling. Preprint at <a href="https://arxiv.org/abs/2301.06568" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/2301.06568</a> (2023).</cite>
</li>
<li id="CR24">
<span class="label">24.</span><cite>Ross, J. et al. Large-scale chemical language representations capture molecular structure and properties. <em>Nat. Mach. Intell.</em><strong>4</strong>, 1256–1264 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ross,%20J.%20et%20al.%20Large-scale%20chemical%20language%20representations%20capture%20molecular%20structure%20and%20properties.%20Nat.%20Mach.%20Intell.4,%201256%E2%80%931264%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Lin, Z. et al. Evolutionary-scale prediction of atomic-level protein structure with a language model. <em>Science</em><strong>379</strong>, 1123–1130 (2023).
</cite> [<a href="https://doi.org/10.1126/science.ade2574" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36927031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20Z.%20et%20al.%20Evolutionary-scale%20prediction%20of%20atomic-level%20protein%20structure%20with%20a%20language%20model.%20Science379,%201123%E2%80%931130%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Wu, R. et al. High-resolution de novo structure prediction from primary sequence. Preprint at 10.1101/2022.07.21.500999 (2022).</cite>
</li>
<li id="CR27">
<span class="label">27.</span><cite>Koes, D. R., Baumgartner, M. P. &amp; Camacho, C. J. Lessons learned in empirical scoring with smina from the CSAR 2011 benchmarking exercise. <em>J. Chem. Inf. Model.</em><strong>53</strong>, 1893–1904 (2013).
</cite> [<a href="https://doi.org/10.1021/ci300604z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3726561/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23379370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koes,%20D.%20R.,%20Baumgartner,%20M.%20P.%20&amp;%20Camacho,%20C.%20J.%20Lessons%20learned%20in%20empirical%20scoring%20with%20smina%20from%20the%20CSAR%202011%20benchmarking%20exercise.%20J.%20Chem.%20Inf.%20Model.53,%201893%E2%80%931904%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Joosten, R. P. et al. A series of PDB related databases for everyday needs. <em>Nucleic Acids Res.</em><strong>39</strong>, D411–D419 (2011).
</cite> [<a href="https://doi.org/10.1093/nar/gkq1105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3013697/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21071423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Joosten,%20R.%20P.%20et%20al.%20A%20series%20of%20PDB%20related%20databases%20for%20everyday%20needs.%20Nucleic%20Acids%20Res.39,%20D411%E2%80%93D419%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Chicco, D. &amp; Jurman, G. The advantages of the matthews correlation coefficient (MCC) over F1 score and accuracy in binary classification evaluation. <em>BMC Genomics</em><strong>21</strong>, 6 (2020).
</cite> [<a href="https://doi.org/10.1186/s12864-019-6413-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6941312/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31898477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chicco,%20D.%20&amp;%20Jurman,%20G.%20The%20advantages%20of%20the%20matthews%20correlation%20coefficient%20(MCC)%20over%20F1%20score%20and%20accuracy%20in%20binary%20classification%20evaluation.%20BMC%20Genomics21,%206%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Saito, T. &amp; Rehmsmeier, M. The precision-recall plot is more informative than the ROC plot when evaluating binary classifiers on imbalanced datasets. <em>PLoS ONE</em><strong>10</strong>, e0118432 (2015).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0118432" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4349800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25738806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saito,%20T.%20&amp;%20Rehmsmeier,%20M.%20The%20precision-recall%20plot%20is%20more%20informative%20than%20the%20ROC%20plot%20when%20evaluating%20binary%20classifiers%20on%20imbalanced%20datasets.%20PLoS%20ONE10,%20e0118432%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Hendlich, M., Rippmann, F. &amp; Barnickel, G. LIGSITE: automatic and efficient detection of potential small molecule-binding sites in proteins. <em>J. Mol. Graph. Model.</em><strong>15</strong>, 359–363 (1997).
</cite> [<a href="https://doi.org/10.1016/s1093-3263(98)00002-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9704298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hendlich,%20M.,%20Rippmann,%20F.%20&amp;%20Barnickel,%20G.%20LIGSITE:%20automatic%20and%20efficient%20detection%20of%20potential%20small%20molecule-binding%20sites%20in%20proteins.%20J.%20Mol.%20Graph.%20Model.15,%20359%E2%80%93363%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Brylinski, M. &amp; Skolnick, J. A threading-based method (FINDSITE) for ligand-binding site prediction and functional annotation. <em>Proc. Natl Acad. Sci. USA</em><strong>105</strong>, 129–134 (2008).
</cite> [<a href="https://doi.org/10.1073/pnas.0707684105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2224172/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18165317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brylinski,%20M.%20&amp;%20Skolnick,%20J.%20A%20threading-based%20method%20(FINDSITE)%20for%20ligand-binding%20site%20prediction%20and%20functional%20annotation.%20Proc.%20Natl%20Acad.%20Sci.%20USA105,%20129%E2%80%93134%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Yamashita, M. M., Wesson, L., Eisenman, G. &amp; Eisenberg, D. Where metal ions bind in proteins. <em>Proc. Natl Acad. Sci. USA</em><strong>87</strong>, 5648–5652 (1990).
</cite> [<a href="https://doi.org/10.1073/pnas.87.15.5648" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC54384/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2377604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamashita,%20M.%20M.,%20Wesson,%20L.,%20Eisenman,%20G.%20&amp;%20Eisenberg,%20D.%20Where%20metal%20ions%20bind%20in%20proteins.%20Proc.%20Natl%20Acad.%20Sci.%20USA87,%205648%E2%80%935652%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Comaniciu, D. &amp; Meer, P. Mean shift: a robust approach toward feature space analysis. <em>IEEE Trans. Pattern Anal. Mach. Intell.</em><strong>24</strong>, 603–619 (2002).</cite> [<a href="https://scholar.google.com/scholar_lookup?Comaniciu,%20D.%20&amp;%20Meer,%20P.%20Mean%20shift:%20a%20robust%20approach%20toward%20feature%20space%20analysis.%20IEEE%20Trans.%20Pattern%20Anal.%20Mach.%20Intell.24,%20603%E2%80%93619%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Clark, J. J., Benson, M. L., Smith, R. D. &amp; Carlson, H. A. Inherent versus induced protein flexibility: comparisons within and between apo and holo structures. <em>PLoS Comput. Biol.</em><strong>15</strong>, e1006705 (2019).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1006705" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6370239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30699115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clark,%20J.%20J.,%20Benson,%20M.%20L.,%20Smith,%20R.%20D.%20&amp;%20Carlson,%20H.%20A.%20Inherent%20versus%20induced%20protein%20flexibility:%20comparisons%20within%20and%20between%20apo%20and%20holo%20structures.%20PLoS%20Comput.%20Biol.15,%20e1006705%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <em>Nature</em><strong>596</strong>, 583–589 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jumper,%20J.%20et%20al.%20Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold.%20Nature596,%20583%E2%80%93589%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Zhang, C., Shine, M., Pyle, A. M. &amp; Zhang, Y. US-align: universal structure alignments of proteins, nucleic acids, and macromolecular complexes. <em>Nat. Methods</em><strong>19</strong>, 1109–1115 (2022).
</cite> [<a href="https://doi.org/10.1038/s41592-022-01585-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36038728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20C.,%20Shine,%20M.,%20Pyle,%20A.%20M.%20&amp;%20Zhang,%20Y.%20US-align:%20universal%20structure%20alignments%20of%20proteins,%20nucleic%20acids,%20and%20macromolecular%20complexes.%20Nat.%20Methods19,%201109%E2%80%931115%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Su, M. et al. Comparative assessment of scoring functions: the CASF-2016 update. <em>J. Chem. Inf. Model.</em><strong>59</strong>, 895–913 (2019).
</cite> [<a href="https://doi.org/10.1021/acs.jcim.8b00545" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30481020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su,%20M.%20et%20al.%20Comparative%20assessment%20of%20scoring%20functions:%20the%20CASF-2016%20update.%20J.%20Chem.%20Inf.%20Model.59,%20895%E2%80%93913%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Lei, J., Kusov, Y. &amp; Hilgenfeld, R. Nsp3 of coronaviruses: structures and functions of a large multi-domain protein. <em>Antivir. Res.</em><strong>149</strong>, 58–74 (2018).
</cite> [<a href="https://doi.org/10.1016/j.antiviral.2017.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7113668/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29128390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lei,%20J.,%20Kusov,%20Y.%20&amp;%20Hilgenfeld,%20R.%20Nsp3%20of%20coronaviruses:%20structures%20and%20functions%20of%20a%20large%20multi-domain%20protein.%20Antivir.%20Res.149,%2058%E2%80%9374%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Gahbauer, S. et al. Iterative computational design and crystallographic screening identifies potent inhibitors targeting the Nsp3 macrodomain of SARS-CoV-2. <em>Proc. Natl Acad. Sci. USA</em><strong>120</strong>, e2212931120 (2023).
</cite> [<a href="https://doi.org/10.1073/pnas.2212931120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9926234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36598939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gahbauer,%20S.%20et%20al.%20Iterative%20computational%20design%20and%20crystallographic%20screening%20identifies%20potent%20inhibitors%20targeting%20the%20Nsp3%20macrodomain%20of%20SARS-CoV-2.%20Proc.%20Natl%20Acad.%20Sci.%20USA120,%20e2212931120%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>van der Maaten, L. &amp; Hinton, G. Visualizing data using t-SNE. <em>J. Mach. Learn. Res.</em><strong>9</strong>, 2579–2605 (2008).</cite> [<a href="https://scholar.google.com/scholar_lookup?van%20der%20Maaten,%20L.%20&amp;%20Hinton,%20G.%20Visualizing%20data%20using%20t-SNE.%20J.%20Mach.%20Learn.%20Res.9,%202579%E2%80%932605%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Yang, J., Roy, A. &amp; Zhang, Y. BioLiP: a semi-manually curated database for biologically relevant ligand–protein interactions. <em>Nucleic Acids Res.</em><strong>41</strong>, D1096–D1103 (2012).
</cite> [<a href="https://doi.org/10.1093/nar/gks966" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23087378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20J.,%20Roy,%20A.%20&amp;%20Zhang,%20Y.%20BioLiP:%20a%20semi-manually%20curated%20database%20for%20biologically%20relevant%20ligand%E2%80%93protein%20interactions.%20Nucleic%20Acids%20Res.41,%20D1096%E2%80%93D1103%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Huang, Y., Niu, B., Gao, Y., Fu, L. &amp; Li, W. CD-HIT suite: a web server for clustering and comparing biological sequences. <em>Bioinformatics</em><strong>26</strong>, 680–682 (2010).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2828112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20053844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20Y.,%20Niu,%20B.,%20Gao,%20Y.,%20Fu,%20L.%20&amp;%20Li,%20W.%20CD-HIT%20suite:%20a%20web%20server%20for%20clustering%20and%20comparing%20biological%20sequences.%20Bioinformatics26,%20680%E2%80%93682%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Steinegger, M. &amp; Söding, J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets. <em>Nat. Biotechnol.</em><strong>35</strong>, 1026–1028 (2017).
</cite> [<a href="https://doi.org/10.1038/nbt.3988" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29035372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Steinegger,%20M.%20&amp;%20S%C3%B6ding,%20J.%20MMseqs2%20enables%20sensitive%20protein%20sequence%20searching%20for%20the%20analysis%20of%20massive%20data%20sets.%20Nat.%20Biotechnol.35,%201026%E2%80%931028%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Vaswani, A. et al. Attention is all you need. In <em>Proc. 31st International Conference on Neural Information Processing Systems</em> 6000–6010 (Curran Associates, 2017).</cite>
</li>
<li id="CR46">
<span class="label">46.</span><cite>Jiang, Z. et al. ConvBERT: improving BERT with span-based dynamic convolution. In <em>Proc. 34th International Conference on Neural Information Processing Systems</em> 12837–12848 (Curran Associates, 2020).</cite>
</li>
<li id="CR47">
<span class="label">47.</span><cite>Ingraham, J., Garg, V. K., Barzilay, R. &amp; Jaakkola, T. Generative models for graph-based protein design. In <em>Proc. 33rd International Conference on Neural Information Processing Systems</em> 15820–15831 (Curran Associates, 2019).</cite>
</li>
<li id="CR48">
<span class="label">48.</span><cite>Gainza, P. et al. Deciphering interaction fingerprints from protein molecular surfaces using geometric deep learning. <em>Nat. Methods</em><strong>17</strong>, 184–192 (2020).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0666-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31819266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gainza,%20P.%20et%20al.%20Deciphering%20interaction%20fingerprints%20from%20protein%20molecular%20surfaces%20using%20geometric%20deep%20learning.%20Nat.%20Methods17,%20184%E2%80%93192%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Sanner, M. F., Olson, A. J. &amp; Spehner, J. C. Reduced surface: an efficient way to compute molecular surfaces. <em>Biopolymers</em><strong>38</strong>, 305–320 (1996).
</cite> [<a href="https://doi.org/10.1002/(SICI)1097-0282(199603)38:3%3C305::AID-BIP4%3E3.0.CO;2-Y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8906967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanner,%20M.%20F.,%20Olson,%20A.%20J.%20&amp;%20Spehner,%20J.%20C.%20Reduced%20surface:%20an%20efficient%20way%20to%20compute%20molecular%20surfaces.%20Biopolymers38,%20305%E2%80%93320%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Huynh, D. Q. Metrics for 3D rotations: comparison and analysis. <em>J. Math. Imaging Vis.</em><strong>35</strong>, 155–164 (2009).</cite> [<a href="https://scholar.google.com/scholar_lookup?Huynh,%20D.%20Q.%20Metrics%20for%203D%20rotations:%20comparison%20and%20analysis.%20J.%20Math.%20Imaging%20Vis.35,%20155%E2%80%93164%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Gheini, M., Ren, X. &amp; May, J. Cross-attention is all you need: adapting pretrained transformers for machine translation. In <em>Proc. Proceedings of the 2021 Conference on Empirical Methods in Natural Language Processing</em> 1754–1765 (EMNLP, 2021).</cite>
</li>
<li id="CR52">
<span class="label">52.</span><cite>Elfwing, S., Uchibe, E. &amp; Doya, K. Sigmoid-weighted linear units for neural network function approximation in reinforcement learning. <em>Neural Netw.</em><strong>107</strong>, 3–11 (2018).
</cite> [<a href="https://doi.org/10.1016/j.neunet.2017.12.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29395652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elfwing,%20S.,%20Uchibe,%20E.%20&amp;%20Doya,%20K.%20Sigmoid-weighted%20linear%20units%20for%20neural%20network%20function%20approximation%20in%20reinforcement%20learning.%20Neural%20Netw.107,%203%E2%80%9311%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Kingma, D. P. &amp; Ba, J. Adam: a method for stochastic optimization. In <em>Proc. International Conference on Learning Representations</em> (ICLR, 2015).</cite>
</li>
<li id="CR54">
<span class="label">54.</span><cite>Paszke, A. et al. PyTorch: an imperative style, high-performance deep learning library. In <em>Proc. Advances in Neural Information Processing Systems</em> 8026–8037 (Curran Associates, 2019).</cite>
</li>
<li id="CR55">
<span class="label">55.</span><cite>Yuan, Q., Tian, C. &amp; Yang, Y. Data from: genome-scale annotation of protein binding sites via language model and geometric deep learning. <em>Zenodo</em>10.5281/zenodo.10845361 (2024).</cite> [<a href="https://doi.org/10.7554/eLife.93695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11023698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38630609/" class="usa-link">PubMed</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Berman, H. M. et al. The protein data bank. <em>Nucleic Acids Res.</em><strong>28</strong>, 235–242 (2000).
</cite> [<a href="https://doi.org/10.1093/nar/28.1.235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berman,%20H.%20M.%20et%20al.%20The%20protein%20data%20bank.%20Nucleic%20Acids%20Res.28,%20235%E2%80%93242%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Zhang, Z. et al. LABind: Identifying protein binding ligand-aware sites via learning interactions between ligand and protein. <em>Zenodo</em>10.5281/zenodo.15871908 (2025).</cite> [<a href="https://doi.org/10.1038/s41467-025-62899-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12365077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40830361/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365077/bin/41467_2025_62899_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365077/bin/41467_2025_62899_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62899_MOESM2_ESM.pdf</a><sup> (71KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365077/bin/41467_2025_62899_MOESM3_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (3.9MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365077/bin/41467_2025_62899_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (161.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365077/bin/41467_2025_62899_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (961.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365077/bin/41467_2025_62899_MOESM6_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (11.7MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Unless otherwise stated, all data supporting the results of this study can be found in the article, <a href="#MOESM1" class="usa-link">supplementary</a>, and <a href="#MOESM3" class="usa-link">source data</a> files. Datasets used for all analysis are deposited via Zenodo [10.5281/zenodo.15875006]. For each dataset, the provided resources include sequence data, protein structure data, and annotated binding sites. Other sources include the DS1 dataset [<a href="http://www.csbio.sjtu.edu.cn/bioinf/LigBind/datasets.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.csbio.sjtu.edu.cn/bioinf/LigBind/datasets.html</a>] and the DS2 dataset<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. All protein structure data used in this study are also available from the BioLiP2 database [<a href="https://zhanggroup.org/BioLiP/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zhanggroup.org/BioLiP/</a>]. All ligand SMILES strings used in this study can also be downloaded from the official RCSB website<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> [<a href="https://www.rcsb.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org</a>]. The LABind-predicted binding sites in this study are provided in Supplementary Data <a href="#MOESM3" class="usa-link">1</a>. <a href="#Sec33" class="usa-link">Source data</a> are provided with this paper.</p>
<p>Our code is available on the GitHub repository [<a href="https://github.com/ljquanlab/LABind" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ljquanlab/LABind</a>] or the Zenodo repository<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62899-0"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62899.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365077/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365077/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365077%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365077/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365077/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365077/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830361/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365077/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830361/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365077/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365077/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="dq2BeD5YKzVsgVF4jFwCRARZjQ9AWnqJRRNWzngW8A9BdzokFbHkKkYDESqPb3kF">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
