
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Analysis of genomic heterogeneity and the mutational landscape in cutaneous squamous cell carcinoma through multi-patient-targeted single-cell DNA sequencing - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5313868AEF3D23051386002A08131E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmccanc">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374374/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Cancer">
<meta name="citation_title" content="Analysis of genomic heterogeneity and the mutational landscape in cutaneous squamous cell carcinoma through multi-patient-targeted single-cell DNA sequencing">
<meta name="citation_author" content="Wen Chen">
<meta name="citation_author_institution" content="Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China">
<meta name="citation_author" content="Jiawei Xu">
<meta name="citation_author_institution" content="Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China">
<meta name="citation_author" content="Chengdong Yu">
<meta name="citation_author_institution" content="Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China">
<meta name="citation_author" content="Meng Zhou">
<meta name="citation_author_institution" content="Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China">
<meta name="citation_author" content="Yong Ai">
<meta name="citation_author_institution" content="Department of Dermatologic Surgery, Dermatology Hospital of Jiangxi Province, Nanchang, Jiangxi Province China">
<meta name="citation_author" content="Wenbing Rao">
<meta name="citation_author_institution" content="Department of Dermatologic Surgery, Dermatology Hospital of Jiangxi Province, Nanchang, Jiangxi Province China">
<meta name="citation_author" content="Qingchuan Wang">
<meta name="citation_author_institution" content="Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China">
<meta name="citation_author" content="Siyi Xu">
<meta name="citation_author_institution" content="Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China">
<meta name="citation_author" content="Lei Tang">
<meta name="citation_author_institution" content="Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China">
<meta name="citation_author" content="Si Gong">
<meta name="citation_author_institution" content="Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Jiangxi Clinical Research Center for Cancer, 519 East Beijing Road, Nanchang, Jiangxi Province 330029 China">
<meta name="citation_author" content="Jun Rao">
<meta name="citation_author_institution" content="Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Jiangxi Clinical Research Center for Cancer, 519 East Beijing Road, Nanchang, Jiangxi Province 330029 China">
<meta name="citation_publication_date" content="2025 Aug 23">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="1362">
<meta name="citation_doi" content="10.1186/s12885-025-14585-z">
<meta name="citation_pmid" content="40847288">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374374/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374374/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374374/pdf/12885_2025_Article_14585.pdf">
<meta name="description" content="Cutaneous squamous cell carcinoma (CSCC) is a prevalent skin cancer with aggressive progression that poses significant challenges, especially in metastatic cases. Single-cell DNA sequencing (scDNA-seq) has become an advanced technology for ...">
<meta name="og:title" content="Analysis of genomic heterogeneity and the mutational landscape in cutaneous squamous cell carcinoma through multi-patient-targeted single-cell DNA sequencing">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Cutaneous squamous cell carcinoma (CSCC) is a prevalent skin cancer with aggressive progression that poses significant challenges, especially in metastatic cases. Single-cell DNA sequencing (scDNA-seq) has become an advanced technology for ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374374/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12374374">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12885-025-14585-z"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12885_2025_Article_14585.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374374%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12374374/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12374374/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374374/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmccanc.png" alt="BMC Cancer logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Cancer" title="Link to BMC Cancer" shape="default" href="https://doi.org/10.1186/s12885-025-14585-z" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Cancer</button></div>. 2025 Aug 23;25:1362. doi: <a href="https://doi.org/10.1186/s12885-025-14585-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12885-025-14585-z</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Cancer%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Cancer%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Cancer%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Cancer%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Analysis of genomic heterogeneity and the mutational landscape in cutaneous squamous cell carcinoma through multi-patient-targeted single-cell DNA sequencing</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Wen Chen</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Wen Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wen Chen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jiawei Xu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jiawei Xu</span></h3>
<div class="p">
<sup>1</sup>Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiawei Xu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Chengdong Yu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Chengdong Yu</span></h3>
<div class="p">
<sup>1</sup>Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chengdong Yu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Meng Zhou</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Meng Zhou</span></h3>
<div class="p">
<sup>1</sup>Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Meng Zhou</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ai%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yong Ai</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yong Ai</span></h3>
<div class="p">
<sup>2</sup>Department of Dermatologic Surgery, Dermatology Hospital of Jiangxi Province, Nanchang, Jiangxi Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ai%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yong Ai</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rao%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Wenbing Rao</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Wenbing Rao</span></h3>
<div class="p">
<sup>2</sup>Department of Dermatologic Surgery, Dermatology Hospital of Jiangxi Province, Nanchang, Jiangxi Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rao%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenbing Rao</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Qingchuan Wang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Qingchuan Wang</span></h3>
<div class="p">
<sup>1</sup>Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qingchuan Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Siyi Xu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Siyi Xu</span></h3>
<div class="p">
<sup>1</sup>Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Siyi Xu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Lei Tang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Lei Tang</span></h3>
<div class="p">
<sup>1</sup>Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lei Tang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gong%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Si Gong</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Si Gong</span></h3>
<div class="p">
<sup>3</sup>Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Jiangxi Clinical Research Center for Cancer, 519 East Beijing Road, Nanchang, Jiangxi Province 330029 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gong%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Si Gong</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rao%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Jun Rao</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Jun Rao</span></h3>
<div class="p">
<sup>3</sup>Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Jiangxi Clinical Research Center for Cancer, 519 East Beijing Road, Nanchang, Jiangxi Province 330029 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rao%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jun Rao</span></a>
</div>
</div>
<sup>3,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Breast Surgery, Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Nanchang, Jiangxi Province China </div>
<div id="Aff2">
<sup>2</sup>Department of Dermatologic Surgery, Dermatology Hospital of Jiangxi Province, Nanchang, Jiangxi Province China </div>
<div id="Aff3">
<sup>3</sup>Jiangxi Cancer Hospital, The Second Affiliated Hospital of Nanchang Medical College, Jiangxi Cancer Institute, Jiangxi Clinical Research Center for Cancer, 519 East Beijing Road, Nanchang, Jiangxi Province 330029 China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Sep 8; Accepted 2025 Jun 25; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12374374Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40847288/" class="usa-link">40847288</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Cutaneous squamous cell carcinoma (CSCC) is a prevalent skin cancer with aggressive progression that poses significant challenges, especially in metastatic cases. Single-cell DNA sequencing (scDNA-seq) has become an advanced technology for elucidating tumor heterogeneity and clonal evolution. However, comprehensive scDNA-seq studies and tailored mutation panels for CSCC are lacking.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">We analyzed the genomic landscape of Chinese CSCC patients via a Multi-Patient-Targeted (MPT) scDNA-seq approach. This method combined bulk exome sequencing with Tapestri scDNA-seq.Â Mutations identified through bulk sequencing were used to design a targeted panel for scDNA-seq.Â Comparative analysis was conducted to explore the associations between specific gene mutations and clinical characteristics such as tumor stage and patient sex. Clonal evolution analysis was performed to understand the evolutionary trajectories of the tumors.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">Bulk sequencing revealed a diverse spectrum of somatic mutations in CSCC tumors, with missense mutations being predominant. The top tumor mutations, such as those in NOTCH1, TP53, NOTCH2, TTN, MUC16, RYR2, PRUNE2, DMD, HRAS, and CDKN2A, presented similar frequencies to those reported in studies in Korean and Caucasian populations. However, the mutation frequencies of HRAS, TTN, MUC16 and MUC4 were significantly different from the Korean and Caucasian populations. Comparative analysis revealed associations between specific gene mutations and clinical characteristics such as tumor stage and patient sex. Clonal evolution analysis via scDNA-seq revealed distinct evolutionary trajectories and their potential correlation with tumor development and patient prognosis. Furthermore, scDNA-seq identified two low-frequency mutation clones, NLRP5 and HMMR, which play important roles in the clonal evolution of CSCC.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par4">Our study introduced a novel MPT sequencing approach for CSCC, providing insights into tumor heterogeneity and clonal evolution. We identified novel mutations and their potential associations with patient survival and tumor characteristics. Overall, our study layed the groundwork for personalized treatment strategies and provides a reference for future applications of the MPT panel in CSCC patients.</p></section><section id="sec5"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12885-025-14585-z.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Cutaneous squamous cell carcinoma (CSCC), Genomic heterogeneity, Mutational landscape, Evolutionary dynamics</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par18">Cutaneous squamous cell carcinoma (CSCC) is a prevalent form of cancer that accounts for approximately 20% of all skin tumors [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Originating from keratinocytes, it typically arises on sun-exposed skin and progresses aggressively [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. While early-stage CSCC can often be managed surgically, metastatic CSCC poses a significant challenge due to limited treatment options and poor long-term survival rates [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Therefore, there is an urgent need to identify prognostic markers for early-stage CSCC and elucidate the molecular mechanisms underlying its progression to explore novel therapeutic approaches for patients across all stages of the disease. Major risk factors for CSCC include UV radiation exposure, aging, and fair skin [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Additionally, immunosuppression, autoimmune diseases, and chronic lymphocytic leukemia are known to increase the risk [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>].</p>
<p id="Par19">Recent advancements in high-throughput omics technologies have greatly enhanced our understanding of CSCC. For example, genome-wide association studies have identified several genetic variants linked to CSCC, affecting pathways related to skin pigmentation, immunity, cellular growth, and apoptosis [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Moreover, RNA sequencing studies have revealed significant transcriptomic differences between CSCC and healthy skin samples, highlighting alterations in genes related to cell cycle regulation, epidermal function, inflammation, and cell death pathways and providing valuable insights into the molecular mechanisms of this disease [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Whole-exome sequencing, also known as bulk sequencing, has been pivotal in revealing genetic alterations in CSCC [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. However, bulk sequencing can detect only average allele frequencies without clear reference to specific cells or zygosity, so it is unable to study clonal evolution and tumor heterogeneity. Furthermore, the variant allele frequency (VAF), which represents the proportion of the mutant allele among all alleles analyzed, often reaches approximately 50%, posing challenges in assessing the normal (nonleukemia) component in a sample.</p>
<p id="Par20">Single-cell DNA sequencing (scDNA-seq) has emerged as a powerful tool for dissecting intratumoral heterogeneity and understanding clonal evolution in cancer. Unlike traditional bulk sequencing methods, which provide an averaged view of genomic alterations across a tumor population, scDNA-seq allows for the examination of individual cells, enabling the identification of rare subclones and the reconstruction of evolutionary trajectories within tumors [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Some studies have investigated the utility of Tapestri scDNA-seq in exploring clonal evolution and the diversity of tumors in patients with myeloproliferative disorders [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. While scDNA-seq has been widely applied in various cancer types, its utility in cutaneous squamous cell carcinoma (CSCC) has rarely been reported. In addition, scDNA-seq has several inherent limitations. Unlike bulk sequencing methods, single-cell sequencing often suffers from higher levels of technical noise and amplification biases, leading to incomplete coverage and increased false positives [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. Moreover, the economic feasibility of single-cell sequencing hinges on the necessity of targeting specific genomic regions (gene panels) for sequencing, which might be insufficient for certain research questions that require unbiased study of the cancer exome/genome [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. As a result, the design of targeted panels becomes paramount in maximizing the cost-effectiveness and accuracy of single-cell sequencing assays, ensuring the comprehensive capture of relevant genetic information while minimizing sequencing costs and technical artifacts [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>].</p>
<p id="Par21">In this study, we conducted a thorough investigation into the genomic landscape of CSCC via multi-patient-targeted (MPT) scDNA-seq, which integrates bulk exome sequencing with Tapestri scDNA-seq. By pooling somatic mutations that were identified in the bulk sequencing analysis, we designed an optimal gene panel for scDNA-seq. Our research explored the clonal relationships, pinpointed crucial driver mutations, and elucidated the evolutionary dynamics of CSCC across multiple patients. This detailed analysis not only enhanced our understanding of the genomic structure of CSCC but also holded promise for advancing personalized and precisely targeted therapeutic strategies.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Patient cohort</h3>
<p id="Par22">The current investigation included twelve individuals who were diagnosed with CSCC, the fundamental details of which were outlined in Table <a href="#Tab1" class="usa-link">1</a>. These participants exhibited normal kidney and liver functionality and did not take any medicine prior to sampling to prevent metabolic interference. The CSCC samples along with their corresponding noncancerous tissue samples (locatedâ€‰&gt;â€‰2 cm away from the tumor periphery) were collected from 12 patients, promptly frozen in liquid nitrogen, transferred into Eppendorf tubes, and subsequently preserved at âˆ’80 Â°C until further analysis. The age range of the patients was between 56 and 97 years, with 5 males and 7 females. Approval for the study was obtained from the ethics committee of Jiangxi Cancer Hospital, and the subjects provided appropriate informed consent. Bulk exome sequencing was performed on the tumor and normal tissues from all 12 patients. The single cell sequencing was performed on Patient 2, Patient 8, Patient 9, Patient 12.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Summary of human samples in the present study</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Subject</th>
<th align="left" colspan="1" rowspan="1">Age</th>
<th align="left" colspan="1" rowspan="1">Gender</th>
<th align="left" colspan="1" rowspan="1">TNM stage</th>
<th align="left" colspan="1" rowspan="1">Tumor size (cm)</th>
<th align="left" colspan="1" rowspan="1">Progression-free Survival (months)</th>
<th align="left" colspan="1" rowspan="1">Survival status</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Patient 1</td>
<td align="left" colspan="1" rowspan="1">92</td>
<td align="left" colspan="1" rowspan="1">Female</td>
<td align="left" colspan="1" rowspan="1">T3N0M0</td>
<td align="left" colspan="1" rowspan="1">4*4*2.5</td>
<td align="left" colspan="1" rowspan="1">124</td>
<td align="left" colspan="1" rowspan="1">Live</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Patient 2</td>
<td align="left" colspan="1" rowspan="1">87</td>
<td align="left" colspan="1" rowspan="1">Female</td>
<td align="left" colspan="1" rowspan="1">T2N0M0</td>
<td align="left" colspan="1" rowspan="1">4*3</td>
<td align="left" colspan="1" rowspan="1">45</td>
<td align="left" colspan="1" rowspan="1">Live</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Patient 3</td>
<td align="left" colspan="1" rowspan="1">65</td>
<td align="left" colspan="1" rowspan="1">Female</td>
<td align="left" colspan="1" rowspan="1">Unknown</td>
<td align="left" colspan="1" rowspan="1">2*2</td>
<td align="left" colspan="1" rowspan="1">39</td>
<td align="left" colspan="1" rowspan="1">Live</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Patient 4</td>
<td align="left" colspan="1" rowspan="1">91</td>
<td align="left" colspan="1" rowspan="1">Female</td>
<td align="left" colspan="1" rowspan="1">Unknown</td>
<td align="left" colspan="1" rowspan="1">Unknown</td>
<td align="left" colspan="1" rowspan="1">19</td>
<td align="left" colspan="1" rowspan="1">Dead</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Patient 5</td>
<td align="left" colspan="1" rowspan="1">97</td>
<td align="left" colspan="1" rowspan="1">Male</td>
<td align="left" colspan="1" rowspan="1">T1N0M0</td>
<td align="left" colspan="1" rowspan="1">Unknown</td>
<td align="left" colspan="1" rowspan="1">38</td>
<td align="left" colspan="1" rowspan="1">Dead</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Patient 6</td>
<td align="left" colspan="1" rowspan="1">81</td>
<td align="left" colspan="1" rowspan="1">Male</td>
<td align="left" colspan="1" rowspan="1">Unknown</td>
<td align="left" colspan="1" rowspan="1">3*3.5</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">Live</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Patient 7</td>
<td align="left" colspan="1" rowspan="1">80</td>
<td align="left" colspan="1" rowspan="1">Male</td>
<td align="left" colspan="1" rowspan="1">T1N0M0</td>
<td align="left" colspan="1" rowspan="1">4.5*5.5</td>
<td align="left" colspan="1" rowspan="1">36</td>
<td align="left" colspan="1" rowspan="1">Live</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Patient 8</td>
<td align="left" colspan="1" rowspan="1">56</td>
<td align="left" colspan="1" rowspan="1">Female</td>
<td align="left" colspan="1" rowspan="1">T1N0M0</td>
<td align="left" colspan="1" rowspan="1">4.5*3.0</td>
<td align="left" colspan="1" rowspan="1">29</td>
<td align="left" colspan="1" rowspan="1">Live</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Patient 9</td>
<td align="left" colspan="1" rowspan="1">81</td>
<td align="left" colspan="1" rowspan="1">Male</td>
<td align="left" colspan="1" rowspan="1">T2N0M0</td>
<td align="left" colspan="1" rowspan="1">2*2</td>
<td align="left" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1">Live</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Patient 10</td>
<td align="left" colspan="1" rowspan="1">75</td>
<td align="left" colspan="1" rowspan="1">Female</td>
<td align="left" colspan="1" rowspan="1">T2N0M0</td>
<td align="left" colspan="1" rowspan="1">2*1</td>
<td align="left" colspan="1" rowspan="1">Unknown</td>
<td align="left" colspan="1" rowspan="1">Unknown</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Patient 11</td>
<td align="left" colspan="1" rowspan="1">96</td>
<td align="left" colspan="1" rowspan="1">Female</td>
<td align="left" colspan="1" rowspan="1">T2NxMx</td>
<td align="left" colspan="1" rowspan="1">Unknown</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">Live</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Patient 12</td>
<td align="left" colspan="1" rowspan="1">58</td>
<td align="left" colspan="1" rowspan="1">Male</td>
<td align="left" colspan="1" rowspan="1">T3N2M1</td>
<td align="left" colspan="1" rowspan="1">5*3.5</td>
<td align="left" colspan="1" rowspan="1">31</td>
<td align="left" colspan="1" rowspan="1">Live</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Enrichment of cell Lysis and nuclear suspensions</h3>
<p id="Par24">The frozen tumor tissues were sectioned via a surgical blade and then lysed in NST solution (146 mM NaCl, 10 mM Tris base at pH 7.8, 1 mM CaCl<sub>2</sub>, 0.05% BSA, 0.2% Nonidet P-40, and 21 mM MgCl<sub>2</sub>). The cell nuclear was stained with DAPI. The acquired cell nuclear were filtered and transferred into a 1.5 ml EP tube. Then the cell nuclear was enriched via a Cytomics FC500 cytometer (Beckman Coulter, FL, USA), with DAPI used as a label.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Bulk exome sequencing</h3>
<p id="Par25">The genomic DNA was then fragmented to an average size of 200â€“300 bp via a Covaris S220 focused-ultrasonicator (Covaris, USA). Following fragmentation, the DNA was subjected to end-repair, A-tailing, and adapter ligation via the KAPA HyperPrep Kit (Kapa Biosystems, USA). Adapter-ligated fragments were purified with AMPure XP beads (Beckman Coulter, USA), amplified via PCR, and subsequently purified. Exome capture was conducted via the SureSelect Human All Exon V7 Kit (Agilent Technologies, USA), where the hybridization of the library with the exome capture probes was carried out overnight. The captured DNA was then enriched via PCR and purified. The exome libraries were quantified, assessed for fragment size distribution, and pooled in equimolar amounts for sequencing. Sequencing was performed on an Illumina NovaSeq 6000 (Illumina, USA) via paired-end 150 bp reads. The bulk exome sequencing was performed to pool the tumor mutations from all 12 patients and design a targeted panel for scDNA-seq.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Data processing and analysis</h3>
<p id="Par26">Raw sequencing reads from bulk exome sequencing of tumor and matched normal tissues were processed using the GATK Best Practices workflow. Initially, the reads underwent quality trimming and adapter removal via Trimmomatic, followed by alignment to the reference human genome (GRCh38) via BWA-MEM. Post-alignment processing included duplicate removal, base quality score recalibration, and variant calling via the GATK toolkit. The identified variants were annotated with ANNOVAR and filtered to retain high-confidence exonic variants. Further analysis of these variants was conducted to assess their potential functional impact via SnpEff or VEP. Comparative analysis of the variant data against matched normal samples enabled the identification of somatic mutations, with key variants validated through Sanger sequencing or targeted re-sequencing.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Single-cell DNA sequencing and mutation detection via mission bio</h3>
<p id="Par27">The tumor mutations identified by bulk sequencing of tumor tissues from all 12 patients were sorted by allele frequencies ranging from 0.1 to 100%. On the basis of the mutation frequencies and previous literature, we selected 66 genes as a tumor-targeted panel for single-cell DNA sequencing. The 66 genes in the MPT panel were selected using a multi-faceted approach: (1) â€‹â€‹Mutation frequencyâ€‹â€‹: Genes with high mutation rates in our bulk sequencing data (e.g., NOTCH1, TP53, TTN) were prioritized. (2) â€‹â€‹Literature validationâ€‹â€‹: Genes with established roles in CSCC pathogenesis (e.g., TP53, HRAS, and CDKN2A) were included based on prior studies in cutaneous and head/neck squamous cell carcinomas [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>â€“<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. (3) Functional impactâ€‹â€‹: Mutations predicted as deleterious by computational tools (CADDâ€‰&gt;â€‰20, PolyPhen2â€‰&gt;â€‰0.8) were prioritized. (4) â€‹â€‹Noveltyâ€‹â€‹: Rare mutations (e.g., NLRP5, HMMR) identified in our cohort but not widely reported in CSCC were included to explore their potential roles in clonal evolution. For example, HMMR was selected for its pancancer association with poor prognosis [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>â€“<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. The Mission Bio Tapestri<sup>Â®</sup> platform was employed for targeted single-cell DNA sequencing using the following workflow: (1) â€‹â€‹Single-cell isolationâ€‹â€‹: Nuclei from frozen tumors were enriched via fluorescence-activated cell sorting (FACS) and loaded onto a microfluidic cartridge for encapsulation. (2) â€‹â€‹Multiplex PCR amplificationâ€‹â€‹: A custom MPT panel targeting 66 genes amplified mutation sites identified in bulk sequencing, with unique molecular identifiers (UMIs) incorporated to mitigate amplification bias. (3) â€‹â€‹Library preparation and sequencingâ€‹â€‹: Amplified DNA was indexed, purified, and sequenced on an Illumina HiSeq4000 (150 bp paired-end reads). (4) â€‹â€‹Data analysisâ€‹â€‹: Mission Bioâ€™s Tapestri Insights<sup>Â®</sup> software integrated read depth normalization, UMI correction, and hierarchical clustering to resolve clonal substructure. This workflow enabled high-resolution detection of somatic mutations at single-cell resolution, validated against bulk sequencing data.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Estimation of mutation impact</h3>
<p id="Par28">To assess the functional impact of detected mutations through single-cell DNA sequencing, a comprehensive approach combining multiple tools was employed. Initially, the Combined Annotation Dependent Depletion (CADD) scoring system [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>] which integrates over 60 genomic features to generate a comprehensive mutation score, was used. To further validate the functional impact of the mutations, POLYPHEN2 [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>] and SIFT [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] tools were utilized. Specifically, mutations were initially screened by selecting those with POLYPHEN2 scores greater than 0.8 or (1-SIFT) scores greater than 0.8. The CADD scores were subsequently used to rank the initially screened mutations, and the top 30 most deleterious genes in each sample were selected on the basis of the highest CADD scores, with CADDâ€‰&gt;â€‰10 indicating a mutation in the top 10%, CADDâ€‰&gt;â€‰20 indicating the top 1%, and CADDâ€‰&gt;â€‰30 indicating the top 0.1%.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Construction of a comprehensive phylogenetic tree</h3>
<p id="Par29">To uncover the evolutionary relationships between cell populations from single-cell mutation data, a neighbor-joining (NJ) phylogenetic tree was constructed via the Ape package [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Initially, single nucleotide variant (SNV) information was extracted from the sequencing data of scDNA-seq, and the NJ tree was constructed on the basis of these variant data. Next, cell cluster annotations defined by uniform manifold approximation and projection (UMAP) and hierarchical clustering methods were mapped onto the generated NJ tree. This step aimed to clarify the distribution of each cell cluster within the phylogenetic tree. Finally, mutations within each sampleâ€™s identified clusters were annotated, and harmful genes showing significant copy number variations (CNVs) in each cluster were further analyzed.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Copy number Estimation from mission bio data</h3>
<p id="Par30">In this study, we conducted single-cell CNV analysis using the Python-based Mosaic toolkit. The workflow included stringent quality control with the sample.cnv.filter_amplicons() function to retain high-quality amplicon data detected in â‰¥â€‰50% of cells and with read depthâ€‰&gt;â€‰0. Subsequently, dual normalization was performed using the sample.cnv.normalize_reads() function, based on total read depth per cell and median read depth of amplicons to correct for biases. Absolute copy numbers were calculated with the sample.cnv.compute_ploidy() function, referencing wild-type diploid cells (WT). Finally, clone-specific CNV variation patterns were visualized using the sample.cnv.heatmap() function.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec12"><h3 class="pmc_sec_title">MPT sequencing</h3>
<p id="Par31">A significant obstacle in targeted single-cell DNA sequencing of tumor mutations is the limited profiling of somatic mutations per patient via a small unbiased panel, resulting in high costs and hindering parallel cell profiling. To address this, we introduced a method called MPT sequencing (Fig. <a href="#Fig1" class="usa-link">1</a>). Initially, bulk deep-exome sequencing was conducted on a cohort of patients to identify somatic mutations (Table <a href="#Tab1" class="usa-link">1</a>). A custom targeted panel was subsequently developed to cover all identified mutation sites across the patient cohort. This panel was synthesized commercially and utilized for single-cell DNA sequencing of each tumor, with a focus on mutation sites of interest. This approach significantly reduced sequencing costs by avoiding noninformative genome profiling, enabling high-throughput analysis of thousands of single cells per patient at a low cost.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374374_12885_2025_14585_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/43fa/12374374/fbc01e83a1e1/12885_2025_14585_Fig1_HTML.jpg" loading="lazy" id="d33e710" height="555" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic representation of the Multi-Patient-Targeted (MPT) sequencing workflow. <strong>A</strong> Initial bulk exome sequencing conducted on a group of human tumors. <strong>B</strong> Collective mutations from all patients combined to create a customized MPT targeted panel and scDNA-seq. <strong>C</strong> Utilization of the MPT panel for single-cell DNA sequencing on individual patients, enabling parallel profiling of thousands of cells via a microfluidics platform</p></figcaption></figure></section><section id="Sec13"><h3 class="pmc_sec_title">Clinicopathological characteristics of patients</h3>
<p id="Par33">A diverse cohort of 12 patients diagnosed with CSCC participated in this extensive investigation. The cohort exhibited a spectrum of demographic attributes, including age, sex, and ethnic diversity. A detailed demographic breakdown was presented in Table <a href="#Tab1" class="usa-link">1</a>, illustrating the heterogeneous nature of the patient population. The average age of all patients was 80 years. Histopathological examination revealed characteristic features inherent to CSCC. The participants included 3 patients with T1 stage disease, 4 patients with T2 stage disease, 2 patients with T3 stage disease and 3 patients with uncertain T stage disease. Patients had better overall survival, with an average progression-free survival of approximately 37 months.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Bulk exome sequencing of CSCC tumors</h3>
<p id="Par34">We collected frozen tumor samples from 4 untreated CSCC patients for bulk exome sequencing and MPT profiling via a microfluidics system (Tapestri, Mission Bio). To enhance tumor cell populations, we isolated single-nucleus suspensions and sorted them via fluorescence-activated cell sorting (FACS) within a ploidy range of 2.65â€“3.7 (N). Aneuploid tumor cell fractions were enriched via this method. Subsequently, sequencing libraries were prepared from aneuploid cells, and both whole-genome sequencing (at low depth for copy number estimation) and exome capture (using Roche, Nimblegen V2) were conducted to detect point mutations. Exome sequencing of the aneuploid-sorted nuclei, along with matched normal tissues, revealed a variety of somatic mutations in the tumors (Supplemental file 1). The VAF of each variant in every sequenced sample was provided in Supplemental file 2. The mutation frequencies of the top 44 genes, along with the tumor mutation burden (TMB) for each patient, are shown in Fig. <a href="#Fig2" class="usa-link">2</a>A. On the basis of the integrated analysis of all mutations, we found that the mutations were mainly missense mutations, nonsense mutations, and splice site mutations. Among these, missense mutations occurred with the highest frequency (Fig. <a href="#Fig2" class="usa-link">2</a>B). Further comparison of the three mutation types, single nucleotide polymorphisms (SNPs), insertions, and deletions, revealed that SNPs were the most frequent mutation type (Fig. <a href="#Fig2" class="usa-link">2</a>C), with Câ€‰&gt;â€‰T mutations being the predominant mutation subtype (Fig. <a href="#Fig2" class="usa-link">2</a>D). The tumor mutational burden (TMB) values exhibited significant heterogeneity among patients, ranging from 15 to 1106 (Fig. <a href="#Fig2" class="usa-link">2</a>E). The top 10 mutated genes included NOTCH1 (50%), TP53 (50%), NOTCH2 (50%), TTN (42%), MUC16 (42%), MUC4 (42%), RYR2 (42%), PRUNE2 (42%), DMD (42%), and CDKN2A (42%) (Fig. <a href="#Fig2" class="usa-link">2</a>G). These findings underscored the complex mutational landscape and heterogeneity present within the patient cohort.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374374_12885_2025_14585_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/43fa/12374374/a8b40869624f/12885_2025_14585_Fig2_HTML.jpg" loading="lazy" id="d33e770" height="349" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mutation landscape and tumor mutational burden (TMB) analysis of CSCC tumors. <strong>A</strong> Mutation frequencies of top 44 genes and TMB values for each patient. <strong>B</strong> Distribution of mutation types, with missense mutations being the most prevalent. <strong>C</strong> Comparison of variant types, including single nucleotide polymorphisms (SNPs), deletion mutations and insert mutations. <strong>D</strong> Predominant mutation subtype, highlighting Câ€‰&gt;â€‰T mutations. <strong>E</strong> Heterogeneity in TMB values among patients. <strong>G</strong> Top 10 mutated genes in CSCC tumors</p></figcaption></figure><p id="Par36">Next, we conducted a comparative analysis of mutations with respect to tumor clinical stage, identifying 20 genes mutated exclusively in T1-stage patients (<em>n</em>â€‰=â€‰3) (Fig. <a href="#Fig3" class="usa-link">3</a>A). Further analysis of patient sex revealed that two mutated genes, OBSCN and USH2A, were correlated with female sex, and five mutated genes, OTOGL, ANK3, FAM205A, LOXHD1, and PCDH15, were correlated with male sex (Fig. <a href="#Fig3" class="usa-link">3</a>B). The values of TMB and mutant-allele tumor heterogeneity (MATH) across different sexes were shown in Fig. <a href="#Fig4" class="usa-link">4</a>, suggesting that there were no significant differences between males and females. These results provide valuable references for exploring certain gene mutations that are associated with CSCC clinical progression and patient sex.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374374_12885_2025_14585_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/43fa/12374374/edd1f28c12da/12885_2025_14585_Fig3_HTML.jpg" loading="lazy" id="d33e810" height="258" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative analysis of mutations with tumor clinical staging and gender in CSCC patients. <strong>A</strong> Identification of genes mutated exclusively in T1 stage patients. <strong>B</strong> Occurrence frequency of mutations in T1 stage patients. <strong>C</strong> and <strong>D</strong> Mutated genes correlated with males and females. OR represented Odds Ratio while the <em>P</em>-value meaned the significant differences of gene expression between females and males</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374374_12885_2025_14585_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/43fa/12374374/b0d1c38f7cac/12885_2025_14585_Fig4_HTML.jpg" loading="lazy" id="d33e820" height="247" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of tumor mutational burden (TMB) and mutant-allele tumor heterogeneity (MATH) across different genders</p></figcaption></figure></section><section id="Sec15"><h3 class="pmc_sec_title">Mutational substructure of CSCC tumors</h3>
<p id="Par39">After bulk exome sequencing for patients with CSCC, we selected 4 patients to investigate their tumor clonal substructure via scDNA-seq analysis. First, we combined all of the somatic mutations identified across the 4 CSCC patients to design a custom MPT panel (Mission Bio), which included 66 genes (Supplemental file 3). A total of 29,128 cells (ranging from 2,789 to 10,079 cells per tumor) were sequenced via the Tapestri microdroplet scDNA-seq platform on the basis of the above MPT panel. The resulting dataset revealed 330 targeted sites with an average coverage depth of 164x across the five tumors. Somatic mutations were identified in individual cells through a filtering process using matched germline bulk exome sequencing data. This approach provides comprehensive insights into the clonal architecture of CSCC tumors, leveraging the MPT panel and scDNA-seq technology.</p>
<p id="Par40">For Patient 2 with T2-stage middle skin squamous cell carcinoma, an in-depth analysis of 5,590 single cells spanning 66 genes revealed two predominant mutated genes, namely, NLRP5 and GAS2L2 (Fig. <a href="#Fig5" class="usa-link">5</a>A). Copy number assessments derived from read depth analysis revealed chromosomal gains and losses, with PCLO, NOTCH1, and KIAA0100 exhibiting amplifications, and WNK2, CNM1, and STK11 demonstrating copy number losses (Fig. <a href="#Fig5" class="usa-link">5</a>B and C). Subclones are defined from scDNA-seq data by clustering cells with shared mutation profiles (single nucleotide variants, SNVs and CNVs). Evolutionary relationships are inferred via phylogenetic trees, distinguishing truncal (shared) and subclonal (branch-specific) mutations. The temporal sequence of single nucleotide variants (SNVs) involved in tumor evolution was elucidated through a constructed clonal evolutionary tree, and WT cells were branched into two clones, NLRP5 p.T891M and GAS2L2 p.R441* (Fig. <a href="#Fig5" class="usa-link">5</a>D). Notably, Patient 2 had a PFS of 45 months, which was the longest among all four patients.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374374_12885_2025_14585_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/43fa/12374374/616e695a4ad6/12885_2025_14585_Fig5_HTML.jpg" loading="lazy" id="d33e858" height="826" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clonal substructure and phylogenetic reconstruction of Patient 2 with a T2 middle CSCC. The MPT panel covering 66 genes was utilized for scDNA-seq analysis of CSCC tumors. <strong>A</strong> Heatmaps of hierarchical clustered mutations in 5490 CSCC cell nuclei. The x-axis represents tumor mutations, and the y-axis represents tumor subclones. <strong>B</strong> and <strong>C</strong> Copy number variations (CNV) of tumor mutations in targeted panel. The x-axis represents mutated genes, and the y-axis represents tumor subclones. <strong>D</strong> Clonal evolution of CSCC tumor</p></figcaption></figure><p id="Par42">For Patient 8, who had T1-stage low skin squamous cell carcinoma, an extensive analysis of 6,984 single cells across 66 genes revealed predominant mutations in PCLO, NLRP5, and HMMR (Fig. <a href="#Fig6" class="usa-link">6</a>A). Copy number estimates revealed chromosomal gains and losses, with DNAH5 and EGFR-AS1/EGFR exhibiting amplification (Fig. <a href="#Fig6" class="usa-link">6</a>B and C). A clonal evolutionary tree delineated NLRP5 (p.M424I)/PCLO (p.D4152V)/HMMR (p.L403F and p.L403S) as the major evolutionary mutations (Fig. <a href="#Fig6" class="usa-link">6</a>D).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374374_12885_2025_14585_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/43fa/12374374/3c0d86e1a1ab/12885_2025_14585_Fig6_HTML.jpg" loading="lazy" id="d33e892" height="1016" width="730" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clonal substructure and phylogenetic reconstruction of Patient 8 with a T1 low CSCC. <strong>A</strong> Heatmaps of hierarchical clustered mutations in 6984 CSCC cell nuclei. The x-axis represents tumor mutations, and the y-axis represents tumor subclones. <strong>B</strong> and <strong>C</strong> Copy number variations (CNV) of tumor mutations in targeted panel. The x-axis represents mutated genes, and the y-axis represents tumor subclones. <strong>D</strong> Clonal evolution of CSCC tumor</p></figcaption></figure><p id="Par44">For Patient 9, who had T2-stage middle skin squamous cell carcinoma, an analysis of 10,079 single cells across 66 genes revealed predominant mutations in TP53 (Fig. <a href="#Fig7" class="usa-link">7</a>A). Copy number estimates revealed chromosomal gains, with ZDHHC11 exhibiting amplification (Fig. <a href="#Fig7" class="usa-link">7</a>B and C). The clonal evolution of the tumor followed a linear trajectory, evolving as WT-TP53 (p.P239L)-TP53 (p.C137S) (Fig. <a href="#Fig7" class="usa-link">7</a>D).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374374_12885_2025_14585_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/43fa/12374374/bf897d84c679/12885_2025_14585_Fig7_HTML.jpg" loading="lazy" id="d33e926" height="1015" width="772" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clonal substructure and phylogenetic reconstruction of Patient 9 with a T2 middle CSCC. <strong>A</strong> Heatmaps of hierarchical clustered mutations in 10,079 CSCC cell nuclei. The x-axis represents tumor mutations, and the y-axis represents tumor subclones. <strong>B</strong> and <strong>C</strong> Copy number variations (CNV) of tumor mutations in targeted panel. The x-axis represents mutated genes, and the y-axis represents tumor subclones. <strong>D</strong> Clonal evolution of CSCC tumor</p></figcaption></figure><p id="Par46">For Patient 12, who had T3-stage high-skin squamous cancer, an analysis of 2,789 single cells across 66 genes revealed predominant mutations in TP53, TTN, and HRAS (Fig. <a href="#Fig8" class="usa-link">8</a>A). Copy number estimates revealed chromosomal gains, with IGFN, OBSCN, RYR2, ZDHHC11, and HRAS exhibiting amplification (Fig. <a href="#Fig8" class="usa-link">8</a>B and C). The clonal evolutionary tree of Patient 12 exhibited the greatest degree of clonal evolution among all patients, evolving from WT-HRAS (p.G13D) to TP53 (p.R303*) and subsequently to TTN (p.V6842I) (Fig. <a href="#Fig8" class="usa-link">8</a>D). Furthermore, we also found that Patient 12 presented with the largest tumor size and highest tumor stage, indicating that tumor growth and progression might arise from the evolution of mutation clones within the tumor.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374374_12885_2025_14585_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/43fa/12374374/bc5c15b1184c/12885_2025_14585_Fig8_HTML.jpg" loading="lazy" id="d33e960" height="1016" width="779" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clonal substructure and phylogenetic reconstruction of Patient 12 with a T3 high CSCC. <strong>A</strong> Heatmaps of hierarchical clustered mutations in 2789 CSCC cell nuclei. The x-axis represents tumor mutations, and the y-axis represents tumor subclones. <strong>B</strong> and <strong>C</strong> Copy number variations (CNV) of tumor mutations in targeted panel. The x-axis represents mutated genes, and the y-axis represents tumor subclones. <strong>D</strong> Clonal evolution of CSCC tumor</p></figcaption></figure></section></section><section id="Sec16"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par48">In this study, we employed a novel MPT sequencing approach to elucidate the mutation landscape and clonal evolution lineage in Chinese CSCC patients. Initially, bulk deep-exome sequencing was employed to delineate the tumor gene landscape in 12 patients, followed by the construction of an MPT panel for single-cell DNA sequencing. Mutation signature analysis revealed a predominance of missense mutations, followed by nonsense and splice site mutations. Among the mutation types, SNPs were the most common, with Câ€‰&gt;â€‰T mutations being the predominant subtype. Comparative analysis with tumor clinical staging revealed that 20 genes were exclusively mutated in T1-stage patients. Gender-based analysis revealed specific gene mutations associated with males (OBSCN and USH2A) and females (OTOG, ANK3, FAM205A, LOXHD1, and PCDH15). Notably, there were no significant differences in TMB or MATH values between sexes. Previous studies have investigated the tumor mutation landscape of CSCC in various populations including Korean, British, American, and other Caucasian cohorts [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>â€“<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Compared with these studies, our identified top mutations presented similar frequencies, including those of NOTCH1, TP53, NOTCH2, TTN, MUC16, RYR2, PRUNE2, DMD, HRAS, and CDKN2A. For example, the frequency of TP53 mutation in Chinese patients is 50%, which is 79% in Korean patients and 66.3%, 70% and 94.9% in Caucasians [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>â€“<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. TP53 is recognized as a key tumor suppressor gene crucial for maintaining genomic stability by regulating cellular processes such as the cell cycle, DNA repair, and apoptosis. Mutations in TP53 are commonly associated with various cancers, including CSCC [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. NOTCH1 and NOTCH2, encoding single-pass transmembrane receptors for multiple EGF-like repeats, are considered important drivers in CSCC [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Mutations in these two genes usually lead to impaired cell differentiation, increased stem cell activity, and disease progression in CSCC [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. The mutation frequencies of NOTCH1 and NOTCH2 are comparable across Chinese (50%), Korean (68%, 58%), and Caucasian (ranging from 36.1 to 75%) populations [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>â€“<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>].</p>
<p id="Par49">HRAS showed similar mutation frequencies in Chinese and Korean patients (33% and 32%), which was higher than that of Caucasians (ranging from 9.6 to 22.5%). HRAS is a tumor oncogene. Gain-of-function mutation or amplification of this gene promotes CSCC occurrence and development [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Interestingly, HRASs are all missense mutations in Chinese patients, whereas in Korean patients, deletion mutations are predominant (82%). The mutation frequencies of TTN and MUC16 in Chinese patients (42%) were significantly lower than those in Korean patients (79%) and Caucasians (ranging from 70 to 90% and 39â€“87%, respectively) [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>â€“<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. The function of the TTN in the CSCC has been relatively rarely studied. However, TTN/TP53 co-mutation is considered a significant driver in colon adenocarcinoma and lung squamous cell carcinoma [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. MUC16, a transmembrane member of the mucin family, is involved in epithelial cell barrier formation and pathogen defense [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Notably, MUC4 exhibited a high frequency distribution (42%) in our study, in contrast with the lack of significant mutations in Korean, American, and British patients [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>â€“<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. However, in a study of Australian patients, MUC4 exhibited a high frequency of mutation in patients with head and neck squamous cell carcinoma (HNSC) [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. MUC4 was also found to be highly expressed in the tumor tissues of cutaneous condyloma acuminatum and squamous cell carcinoma and modulated by tissue inflammation and adjacent cutaneous malignancy [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. These findings suggest a potential pivotal role for MUC4 in the tumorigenesis of CSCC.</p>
<p id="Par50">Although bulk sequencing provides extensive information about the tumor mutation landscape, particularly regarding driver genes, it has several limitations. For example, bulk sequencing homogenizes DNA content from thousands to millions of cells, limiting its ability to identify potentially rare and genetically heterogeneous neoplastic cell populations, including subclones that may significantly impact tumor evolution and the development of therapeutic resistance [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>â€“<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Additionally, bulk sequencing often fails to detect rare alleles and cannot definitively determine whether mutations co-occur within the same cell. Single-cell sequencing has the potential to address these key issues. This technique offers unprecedented resolution by enabling the analysis of individual cells within a heterogeneous population, providing insights into intratumoral diversity and evolutionary trajectories [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>, <a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. In addition, scDNA-seq has been employed to investigate the mechanisms of therapy resistance, tumor metastasis and relapse [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. In the present study, by using a microdroplet-based scDNA-seq platform (Tapestri), we conducted an extensive analysis of thousands of individual cells from 4 CSCC patients. By focusing on regions exhibiting somatic mutations, as identified beforehand through bulk exome sequencing, we successfully characterized 66 mutations across thousands of individual cells via MPT. Investigation of the mutational substructure of CSCC tumors via scDNA-seq elucidated the clonal architecture and evolutionary trajectories of individual tumors. For Patient 2 with a T2-stage CSCC, a constructed clonal evolutionary tree was established, in which WT cells were branched into two clones, NLRP5 p.T891M and GAS2L2 p.R441*. For Patient 8 (T1-stage), WT cells were branched into three clones, including NLRP5 (p.M424I), PCLO (p.D4152V) and HMMR (p.L403F and p.L403S). For Patient 9 (T2 stage), the tumorâ€™s clonal evolution followed a linear trajectory, evolving as WT-TP53 (p.P239L)-TP53 (p.C137S). For Patient 12 (T3 stage), the clonal evolutionary tree depicted a linear trajectory, evolving from WT-HRAS (p.G13D) to TP53 (p.R303*) and subsequently to TTN (p.V6842I). scDNA-seq not only identified high-frequency mutations such as GAS2L2, PCLO, HRAS, TTN, and TP53, which were previously detected via bulk sequencing, but also revealed low-frequency tumor mutations, such as those in NLRP5 and HMMR. NLRP5 gene mutations were observed in 2 tumors. NLRP5 belongs to the Nod-like receptor (NLR) family and is implicated in inflammation and innate immunity. Although its role in skin cancer is less understood, emerging research indicates its potential involvement in malignancies [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>, <a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. The functional role of the HMMR in tumors is currently unknown. However, in a pancancer study, HMMR was found to be highly expressed in various cancer types, including HNSC, and was negatively correlated with patient prognosis [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. These findings suggest that mutations in HMMR, particularly gain-of-function mutations, may play a role in the development of CSCC. Furthermore, we observed significant differences in tumor stage, size, and survival status among patients with different clonal mutations and evolutionary structures. Patient 2 exhibited a progression-free survival (PFS) of 45 months, which was significantly greater than that of the other three patients (approximately 30 months), suggesting a potential correlation between shallow-depth simple mutation evolution and better prognosis. In contrast, Patient 12, with a WT-HRAS (p.G13D)-TP53 (p.R303*)-TTN (p.V6842I) profile, presented with the largest tumor size and highest tumor stage, indicating a potential correlation between clonal evolution depth and tumor development.</p>
<p id="Par51">However, it is important to acknowledge several limitations in our study. First, the sample size was limited to 4 CSCC patients as the primary focus was on developing the MPT approach. Therefore, generalizing our biological conclusions regarding tumor evolution to a larger population of CSCC patients may not be appropriate. Moreover, the custom amplicon panel utilized in our study targeted a restricted number of mutation sites (<em>N</em>â€‰=â€‰66). Future advancements in microdroplet technology may enable the expansion of amplicons to thousands of sites, increasing genomic coverage.</p></section><section id="Sec17"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par52">In conclusion, our study introduced a MPT sequencing approach to analyze the mutation landscape and clonal evolution of CSCC tumors. Through bulk exome sequencing and scDNA-seq, we identified diverse somatic mutations, highlighting the prevalence of missense mutations in key genes such as NLRP5, TP53, and TTN. Comparative analysis revealed associations between specific gene mutations and disease characteristics. Using single-cell sequencing, we elucidated the mutational substructure and evolutionary trajectories of CSCC tumors, revealing distinct patterns associated with tumor stage and size. Our findings provide insights into tumor heterogeneity and evolution, laying the groundwork for future research aimed at personalized treatment strategies and improved patient outcomes in CSCC and other cancers.</p></section><section id="Sec190"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12374374/bin/12885_2025_14585_MOESM1_ESM.xls" data-ga-action="click_feat_suppl" class="usa-link">12885_2025_14585_MOESM1_ESM.xls</a><sup> (5.2MB, xls) </sup><p>Supplemental file 1. The somatic mutations of 12 CSCC patients identified by bulk exome sequencing</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12374374/bin/12885_2025_14585_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">12885_2025_14585_MOESM2_ESM.pdf</a><sup> (421KB, pdf) </sup><p>Supplemental file 2. The variant allele fraction (VAF) of each variant in every sequenced sample. The images in the PDF file are arranged in order from Patient 1 to Patient 12</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12374374/bin/12885_2025_14585_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12885_2025_14585_MOESM3_ESM.xlsx</a><sup> (11.9KB, xlsx) </sup><p>Supplemental file 3. The MPT panel including 66 genes that were designed from somatic mutations identified by bulk sequencing in 12 CSCC patients</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Not applicable.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>scDNA-seq</dt>
<dd><p id="Par5">Cutaneous squamous cell carcinoma</p></dd>
<dt>scDNA-seq</dt>
<dd><p id="Par6">Single-cell DNA sequencing</p></dd>
<dt>MPT</dt>
<dd><p id="Par7">Multi-Patient-Targeted</p></dd>
<dt>VAF</dt>
<dd><p id="Par8">Variant allele frequency</p></dd>
<dt>CADD</dt>
<dd><p id="Par9">Combined Annotation Dependent Depletion</p></dd>
<dt>CNV</dt>
<dd><p id="Par10">Copy number variation</p></dd>
<dt>FACS</dt>
<dd><p id="Par11">Fluorescence-activated cell sorting</p></dd>
<dt>TMB</dt>
<dd><p id="Par12">Tumor mutation burden</p></dd>
<dt>SNPs</dt>
<dd><p id="Par13">Single nucleotide polymorphisms</p></dd>
<dt>MATH</dt>
<dd><p id="Par14">Mutant-allele tumor heterogeneity</p></dd>
<dt>SNVs</dt>
<dd><p id="Par15">Single nucleotide variants</p></dd>
<dt>HNSC</dt>
<dd><p id="Par16">Head and neck squamous cell carcinoma</p></dd>
<dt>PFS</dt>
<dd><p id="Par17">Progression-free survival</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authorsâ€™ contributions</h2>
<p>Wen Chen, Jiawei Xu, and Chengdong Yu contributed to study conceptualization and project administration. Yong Ai, Wenbing Rao and Qingchuan Wang contributed to data curation and formal analysis. Wen Chen, Jiawei Xu, Chengdong Yu and Jun Rao conceived and designed the experiments; Siyi Xu and Lei Tang contributed to validation and visualization; Wen Chen, Jiawei Xu, and Jun Rao contributed to the writing of the original draft. Si Gong, Meng Zhou and Chengdong Yu contributed to the investigation and methodology. All the authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the â€œFive-level Progressiveâ€ talent cultivation project of Jiangxi Cancer Hospital &amp; Institute (WCDJ2024JQ02), Jiangxi Key Laboratory of Oncology (2024SSY06041) and the Top Program (20224BAB206080) from Natural Science Foundation of Jiangxi Province, Jiangxi Provincial Administration of Traditional Chinese Medicine Projects (2024B0278), the General Science and Technology Program (202210056 and 202211009) from Jiangxi Provincial Health Commission, and the Jiangxi Cancer Hospital â€œExcellent Young Talent Programâ€ (2021EYS05).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets generated during the current study have been deposited in the GenBase [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] in the National Genomics Data Center [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>], China National Center for Bioinformation/Beijing Institute of Genomics, Chinese Academy of Sciences (GSA-Human: HRA008813 and HRA008816) that are publicly accessible at <a href="https://ngdc.cncb.ac.cn/gsa-human" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa-human</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par53">The study was performed in accordance with the principles of the Declaration of Helsinki. The study was approved by the Ethics Committee of Jiangxi Cancer Hospital (Grant Number: 2022ky306). All the subjects provided appropriate informed consent.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par54">The informed consent for publication was signed by each subject.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par55">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisherâ€™s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Das Mahapatra K, Pasquali L, Sondergaard JN, Lapins J, Nemeth IB, BaltÃ¡s E, KemÃ©ny L, Homey B, Moldovan LI, Kjems J, et al. A comprehensive analysis of coding and non-coding transcriptomic changes in cutaneous squamous cell carcinoma. Sci Rep. 2020;10(1):3637.</cite> [<a href="https://doi.org/10.1038/s41598-020-59660-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7046790/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32108138/" class="usa-link">PubMed</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Alam M, Ratner D. Primary care: cutaneous squamous-cell carcinoma. N Engl J Med. 2001;344(13):975â€“83.
</cite> [<a href="https://doi.org/10.1056/NEJM200103293441306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11274625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alam%20M,%20Ratner%20D.%20Primary%20care:%20cutaneous%20squamous-cell%20carcinoma.%20N%20Engl%20J%20Med.%202001;344(13):975%E2%80%9383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Gobillot TA, Greer M, Parvathaneni U, Liao JJ, Laramore GE, Goff P, Wallner K, Rodriguez CP, Houlton JJ, Barber BR, et al. Radiation therapy for low- and high-risk perineural invasion in head and neck cutaneous squamous cell carcinoma: clinical outcomes and patterns of failure. Head Neck. 2023;45(9):2323â€“34.
</cite> [<a href="https://doi.org/10.1002/hed.27458" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37448346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gobillot%20TA,%20Greer%20M,%20Parvathaneni%20U,%20Liao%20JJ,%20Laramore%20GE,%20Goff%20P,%20Wallner%20K,%20Rodriguez%20CP,%20Houlton%20JJ,%20Barber%20BR,%20et%20al.%20Radiation%20therapy%20for%20low-%20and%20high-risk%20perineural%20invasion%20in%20head%20and%20neck%20cutaneous%20squamous%20cell%20carcinoma:%20clinical%20outcomes%20and%20patterns%20of%20failure.%20Head%20Neck.%202023;45(9):2323%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Di Nardo L, Pellegrini C, Di Stefani A, Del Regno L, Sollena P, Piccerillo A, Longo C, Garbe C, Fargnoli MC, Peris K. Molecular genetics of cutaneous squamous cell carcinoma: perspective for treatment strategies. J Eur Acad Dermatol Venereol. 2020;34(5):932â€“41.
</cite> [<a href="https://doi.org/10.1111/jdv.16098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31747091/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Di%20Nardo%20L,%20Pellegrini%20C,%20Di%20Stefani%20A,%20Del%20Regno%20L,%20Sollena%20P,%20Piccerillo%20A,%20Longo%20C,%20Garbe%20C,%20Fargnoli%20MC,%20Peris%20K.%20Molecular%20genetics%20of%20cutaneous%20squamous%20cell%20carcinoma:%20perspective%20for%20treatment%20strategies.%20J%20Eur%20Acad%20Dermatol%20Venereol.%202020;34(5):932%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Sarin KY, Lin Y, Daneshjou R, Ziyatdinov A, Thorleifsson G, Rubin A, Pardo LM, Wu WT, Khavari PA, Uitterlinden A, et al. Genome-wide meta-analysis identifies eight new susceptibility loci for cutaneous squamous cell carcinoma. Nat Commun. 2020. 10.1038/s41467-020-14594-5.
</cite> [<a href="https://doi.org/10.1038/s41467-020-14594-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7010741/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32041948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sarin%20KY,%20Lin%20Y,%20Daneshjou%20R,%20Ziyatdinov%20A,%20Thorleifsson%20G,%20Rubin%20A,%20Pardo%20LM,%20Wu%20WT,%20Khavari%20PA,%20Uitterlinden%20A,%20et%20al.%20Genome-wide%20meta-analysis%20identifies%20eight%20new%20susceptibility%20loci%20for%20cutaneous%20squamous%20cell%20carcinoma.%20Nat%20Commun.%202020.%2010.1038/s41467-020-14594-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Lee SY, Lee M, Yu DS, Lee YB. Identification of genetic mutations of cutaneous squamous cell carcinoma using whole exome sequencing in non-Caucasian population. J Dermatol Sci. 2022;106(2):70â€“7.
</cite> [<a href="https://doi.org/10.1016/j.jdermsci.2022.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35365379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20SY,%20Lee%20M,%20Yu%20DS,%20Lee%20YB.%20Identification%20of%20genetic%20mutations%20of%20cutaneous%20squamous%20cell%20carcinoma%20using%20whole%20exome%20sequencing%20in%20non-Caucasian%20population.%20J%20Dermatol%20Sci.%202022;106(2):70%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Tsoucas D, Yuan G-C. Recent progress in single-cell cancer genomics. Curr Opin Genet Dev. 2017;42:22â€“32.
</cite> [<a href="https://doi.org/10.1016/j.gde.2017.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5446798/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28126650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsoucas%20D,%20Yuan%20G-C.%20Recent%20progress%20in%20single-cell%20cancer%20genomics.%20Curr%20Opin%20Genet%20Dev.%202017;42:22%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Morita K, Wang F, Jahn K, Hu T, Tanaka T, Sasaki Y, Kuipers J, Loghavi S, Wang SA, Yan Y, et al. Clonal evolution of acute myeloid leukemia revealed by high-throughput single-cell genomics. Nat Commun. 2020. 10.1038/s41467-020-19119-8.
</cite> [<a href="https://doi.org/10.1038/s41467-020-19119-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7577981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33087716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morita%20K,%20Wang%20F,%20Jahn%20K,%20Hu%20T,%20Tanaka%20T,%20Sasaki%20Y,%20Kuipers%20J,%20Loghavi%20S,%20Wang%20SA,%20Yan%20Y,%20et%20al.%20Clonal%20evolution%20of%20acute%20myeloid%20leukemia%20revealed%20by%20high-throughput%20single-cell%20genomics.%20Nat%20Commun.%202020.%2010.1038/s41467-020-19119-8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Sun W, Jin C, Gelfond JA, Chen M-H, Ibrahim JG. Joint analysis of single-cell and bulk tissue sequencing data to infer intratumor heterogeneity. Biometrics. 2020;76(3):983â€“94.
</cite> [<a href="https://doi.org/10.1111/biom.13198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7275921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31813161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun%20W,%20Jin%20C,%20Gelfond%20JA,%20Chen%20M-H,%20Ibrahim%20JG.%20Joint%20analysis%20of%20single-cell%20and%20bulk%20tissue%20sequencing%20data%20to%20infer%20intratumor%20heterogeneity.%20Biometrics.%202020;76(3):983%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Navin N, Kendall J, Troge J, Andrews P, Rodgers L, McIndoo J, Cook K, Stepansky A, Levy D, Esposito D, et al. Tumour evolution inferred by single-cell sequencing. Nature. 2011;472(7341):90â€“U119.
</cite> [<a href="https://doi.org/10.1038/nature09807" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4504184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21399628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Navin%20N,%20Kendall%20J,%20Troge%20J,%20Andrews%20P,%20Rodgers%20L,%20McIndoo%20J,%20Cook%20K,%20Stepansky%20A,%20Levy%20D,%20Esposito%20D,%20et%20al.%20Tumour%20evolution%20inferred%20by%20single-cell%20sequencing.%20Nature.%202011;472(7341):90%E2%80%93U119." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Wang Y, Waters J, Leung ML, Unruh A, Roh W, Shi X, Chen K, Scheet P, Vattathil S, Liang H, et al. Clonal evolution in breast cancer revealed by single nucleus genome sequencing. Nature. 2014;512(7513):155â€“.
</cite> [<a href="https://doi.org/10.1038/nature13600" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4158312/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25079324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Y,%20Waters%20J,%20Leung%20ML,%20Unruh%20A,%20Roh%20W,%20Shi%20X,%20Chen%20K,%20Scheet%20P,%20Vattathil%20S,%20Liang%20H,%20et%20al.%20Clonal%20evolution%20in%20breast%20cancer%20revealed%20by%20single%20nucleus%20genome%20sequencing.%20Nature.%202014;512(7513):155%E2%80%93." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Zahn H, Steif A, Laks E, Eirew P, VanInsberghe M, Shah SP, Aparicio S, Hansen CL. Scalable whole-genome single-cell library preparation without preamplification. Nat Methods. 2017;14(2):167â€“73.
</cite> [<a href="https://doi.org/10.1038/nmeth.4140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28068316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zahn%20H,%20Steif%20A,%20Laks%20E,%20Eirew%20P,%20VanInsberghe%20M,%20Shah%20SP,%20Aparicio%20S,%20Hansen%20CL.%20Scalable%20whole-genome%20single-cell%20library%20preparation%20without%20preamplification.%20Nat%20Methods.%202017;14(2):167%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Laks E, McPherson A, Zahn H, Lai D, Steif A, Brimhall J, Biele J, Wang B, Masud T, Ting J, et al. Clonal decomposition and DNA replication states defined by scaled single-cell genome sequencing. Cell. 2019;179(5):1207.
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.10.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6912164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31730858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laks%20E,%20McPherson%20A,%20Zahn%20H,%20Lai%20D,%20Steif%20A,%20Brimhall%20J,%20Biele%20J,%20Wang%20B,%20Masud%20T,%20Ting%20J,%20et%20al.%20Clonal%20decomposition%20and%20DNA%20replication%20states%20defined%20by%20scaled%20single-cell%20genome%20sequencing.%20Cell.%202019;179(5):1207." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Leighton J, Hu M, Sei E, Meric-Bernstam F, Navin NE. Reconstructing mutational lineages in breast cancer by multi-patient-targeted single-cell DNA sequencing. Cell Genom. 2023;3(1):100215â€“100215.
</cite> [<a href="https://doi.org/10.1016/j.xgen.2022.100215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9903705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36777188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leighton%20J,%20Hu%20M,%20Sei%20E,%20Meric-Bernstam%20F,%20Navin%20NE.%20Reconstructing%20mutational%20lineages%20in%20breast%20cancer%20by%20multi-patient-targeted%20single-cell%20DNA%20sequencing.%20Cell%20Genom.%202023;3(1):100215%E2%80%93100215." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Evrony GD, Hinch AG, Luo C. Applications of Single-Cell DNA sequencing. Annu Rev Genomics Hum Genet. 2021;22:171â€“97.
</cite> [<a href="https://doi.org/10.1146/annurev-genom-111320-090436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8410678/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33722077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Evrony%20GD,%20Hinch%20AG,%20Luo%20C.%20Applications%20of%20Single-Cell%20DNA%20sequencing.%20Annu%20Rev%20Genomics%20Hum%20Genet.%202021;22:171%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Su F, Viros A, Milagre C, Trunzer K, Bollag G, Spleiss O, Reis-Filho JS, Kong X, Koya RC, Flaherty KT, et al. RAS mutations in cutaneous squamous-cell carcinomas in patients treated with BRAF inhibitors. N Engl J Med. 2012;366(3):207â€“15.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1105358" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3724537/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22256804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su%20F,%20Viros%20A,%20Milagre%20C,%20Trunzer%20K,%20Bollag%20G,%20Spleiss%20O,%20Reis-Filho%20JS,%20Kong%20X,%20Koya%20RC,%20Flaherty%20KT,%20et%20al.%20RAS%20mutations%20in%20cutaneous%20squamous-cell%20carcinomas%20in%20patients%20treated%20with%20BRAF%20inhibitors.%20N%20Engl%20J%20Med.%202012;366(3):207%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Pickering CR, Zhou JH, Lee JJ, Drummond JA, Peng SA, Saade RE, Tsai KY, Curry JL, Tetzlaff MT, Lai SY, et al. Mutational landscape of aggressive cutaneous squamous cell carcinoma. Clin Cancer Res. 2014;20(24):6582â€“92.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-14-1768" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4367811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25303977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pickering%20CR,%20Zhou%20JH,%20Lee%20JJ,%20Drummond%20JA,%20Peng%20SA,%20Saade%20RE,%20Tsai%20KY,%20Curry%20JL,%20Tetzlaff%20MT,%20Lai%20SY,%20et%20al.%20Mutational%20landscape%20of%20aggressive%20cutaneous%20squamous%20cell%20carcinoma.%20Clin%20Cancer%20Res.%202014;20(24):6582%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Oberholzer PA, Kee D, Dziunycz P, Sucker A, Kamsukom N, Jones R, Roden C, Chalk CJ, Ardlie K, Palescandolo E, et al. &lt;iâ€‰&gt;â€‰RASâ€‰Mutations are associated with the development of cutaneous squamous cell tumors in patients treated with RAF inhibitors&lt;/iâ€‰&gt;. J Clin Oncol. 2012;30(3):316â€“21.
</cite> [<a href="https://doi.org/10.1200/JCO.2011.36.7680" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3269955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22067401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oberholzer%20PA,%20Kee%20D,%20Dziunycz%20P,%20Sucker%20A,%20Kamsukom%20N,%20Jones%20R,%20Roden%20C,%20Chalk%20CJ,%20Ardlie%20K,%20Palescandolo%20E,%20et%20al.%20&lt;i%E2%80%89&gt;%E2%80%89RAS%E2%80%89Mutations%20are%20associated%20with%20the%20development%20of%20cutaneous%20squamous%20cell%20tumors%20in%20patients%20treated%20with%20RAF%20inhibitors&lt;/i%E2%80%89&gt;.%20J%20Clin%20Oncol.%202012;30(3):316%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Tang H, Zhou F, Hu W, Zhang C, Tao J, Xing F, Zhang Z, Gao Y, Zhou F, Hu W. Identification of BIRC5 and HMMR as prognostic biomarkers for immune infiltration in prostate cancer. Transl Androl Urol. 2024. 10.21037/tau-24-359.
</cite> [<a href="https://doi.org/10.21037/tau-24-359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11650346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39698582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang%20H,%20Zhou%20F,%20Hu%20W,%20Zhang%20C,%20Tao%20J,%20Xing%20F,%20Zhang%20Z,%20Gao%20Y,%20Zhou%20F,%20Hu%20W.%20Identification%20of%20BIRC5%20and%20HMMR%20as%20prognostic%20biomarkers%20for%20immune%20infiltration%20in%20prostate%20cancer.%20Transl%20Androl%20Urol.%202024.%2010.21037/tau-24-359." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Ma X, Xie M, Xue Z, Yao J, Wang Y, Xue X, Wang J. HMMR associates with immune infiltrates and acts as a prognostic biomaker in lung adenocarcinoma. Comput Biol Med. 2022;151:106213.</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2022.106213" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36306573/" class="usa-link">PubMed</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Li Z, Zhu T, Yao F, Shen X, Xu M, Fu L, Wu Y, Ding J, Zhang J, Zhao J, et al. LINC00665 promotes Glycolysis in lung adenocarcinoma cells via the let-7c-5p/HMMR axis. J Bioenerg Biomembr. 2024;56(2):181â€“91.
</cite> [<a href="https://doi.org/10.1007/s10863-024-10004-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38411863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Z,%20Zhu%20T,%20Yao%20F,%20Shen%20X,%20Xu%20M,%20Fu%20L,%20Wu%20Y,%20Ding%20J,%20Zhang%20J,%20Zhao%20J,%20et%20al.%20LINC00665%20promotes%20Glycolysis%20in%20lung%20adenocarcinoma%20cells%20via%20the%20let-7c-5p/HMMR%20axis.%20J%20Bioenerg%20Biomembr.%202024;56(2):181%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Mather C, Salipante SJ, Scroggins S, Turner EH, Shirts BH, Wu D, Pritchard CC. Performance characteristics of combined annotation-dependent depletion (CADD) scores in clinical next-generation sequencing cancer panels. J Mol Diagn. 2014;16(6):746â€“746.</cite> [<a href="https://scholar.google.com/scholar_lookup?Mather%20C,%20Salipante%20SJ,%20Scroggins%20S,%20Turner%20EH,%20Shirts%20BH,%20Wu%20D,%20Pritchard%20CC.%20Performance%20characteristics%20of%20combined%20annotation-dependent%20depletion%20(CADD)%20scores%20in%20clinical%20next-generation%20sequencing%20cancer%20panels.%20J%20Mol%20Diagn.%202014;16(6):746%E2%80%93746." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Adzhubei IA, Schmidt S, Peshkin L, Ramensky VE, Gerasimova A, Bork P, Kondrashov AS, Sunyaev SR. A method and server for predicting damaging missense mutations. Nat Methods. 2010;7(4):248â€“9.
</cite> [<a href="https://doi.org/10.1038/nmeth0410-248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2855889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20354512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adzhubei%20IA,%20Schmidt%20S,%20Peshkin%20L,%20Ramensky%20VE,%20Gerasimova%20A,%20Bork%20P,%20Kondrashov%20AS,%20Sunyaev%20SR.%20A%20method%20and%20server%20for%20predicting%20damaging%20missense%20mutations.%20Nat%20Methods.%202010;7(4):248%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Kumar P, Henikoff S, Ng PC. Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm. Nat Protoc. 2009;4(7):1073â€“82.
</cite> [<a href="https://doi.org/10.1038/nprot.2009.86" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19561590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar%20P,%20Henikoff%20S,%20Ng%20PC.%20Predicting%20the%20effects%20of%20coding%20non-synonymous%20variants%20on%20protein%20function%20using%20the%20SIFT%20algorithm.%20Nat%20Protoc.%202009;4(7):1073%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Paradis E, Claude J, Strimmer K. APE: analyses of phylogenetics and evolution in R language. Bioinformatics. 2004;20(2):289â€“90.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btg412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14734327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paradis%20E,%20Claude%20J,%20Strimmer%20K.%20APE:%20analyses%20of%20phylogenetics%20and%20evolution%20in%20R%20language.%20Bioinformatics.%202004;20(2):289%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Chang D, Shain AH. The landscape of driver mutations in cutaneous squamous cell carcinoma. NPJ Genom Med. 2021;6(1): 61.
</cite> [<a href="https://doi.org/10.1038/s41525-021-00226-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8285521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34272401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chang%20D,%20Shain%20AH.%20The%20landscape%20of%20driver%20mutations%20in%20cutaneous%20squamous%20cell%20carcinoma.%20NPJ%20Genom%20Med.%202021;6(1):%2061." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Bibee KP, Kulkarni A, Lee S, Ho J, Osmanbeyoglu HU, Ferris RL, Zandberg DP. Genomic and transcriptomic analysis of cutaneous squamous cell carcinoma arising in immunocompetent and immunosuppressed patients. Oral Oncol. 2024;148:106582.</cite> [<a href="https://doi.org/10.1016/j.oraloncology.2023.106582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10917075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38039877/" class="usa-link">PubMed</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Inman GJ, Wang J, Nagano A, Alexandrov LB, Purdie KJ, Taylor RG, Sherwood V, Thomson J, Hogan S, Spender LC, et al. The genomic landscape of cutaneous SCC reveals drivers and a novel azathioprine associated mutational signature. Nat Commun. 2018. 10.1038/s41467-018-06027-1.
</cite> [<a href="https://doi.org/10.1038/s41467-018-06027-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6131170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30202019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Inman%20GJ,%20Wang%20J,%20Nagano%20A,%20Alexandrov%20LB,%20Purdie%20KJ,%20Taylor%20RG,%20Sherwood%20V,%20Thomson%20J,%20Hogan%20S,%20Spender%20LC,%20et%20al.%20The%20genomic%20landscape%20of%20cutaneous%20SCC%20reveals%20drivers%20and%20a%20novel%20azathioprine%20associated%20mutational%20signature.%20Nat%20Commun.%202018.%2010.1038/s41467-018-06027-1." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kim Y-S, Jung S-H, Park YM, Park GS, Kim HS, Maeng L-S, Chung Y-J. Targeted deep sequencing reveals genomic alterations of actinic keratosis/cutaneous squamous cell carcinoma in situ and cutaneous squamous cell carcinoma. Exp Dermatol. 2023;32(4):447â€“56.
</cite> [<a href="https://doi.org/10.1111/exd.14730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36533870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20Y-S,%20Jung%20S-H,%20Park%20YM,%20Park%20GS,%20Kim%20HS,%20Maeng%20L-S,%20Chung%20Y-J.%20Targeted%20deep%20sequencing%20reveals%20genomic%20alterations%20of%20actinic%20keratosis/cutaneous%20squamous%20cell%20carcinoma%20in%20situ%20and%20cutaneous%20squamous%20cell%20carcinoma.%20Exp%20Dermatol.%202023;32(4):447%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Harwood CA, Proby CM, Inman GJ, Leigh IM. The promise of genomics and the development of targeted therapies for cutaneous squamous cell carcinoma. Acta Derm Venereol. 2016;96(1):3â€“16.
</cite> [<a href="https://doi.org/10.2340/00015555-2181" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26084328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harwood%20CA,%20Proby%20CM,%20Inman%20GJ,%20Leigh%20IM.%20The%20promise%20of%20genomics%20and%20the%20development%20of%20targeted%20therapies%20for%20cutaneous%20squamous%20cell%20carcinoma.%20Acta%20Derm%20Venereol.%202016;96(1):3%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Watt FM, Estrach S, Ambler CA. Epidermal Notch signalling: differentiation, cancer and adhesion. Curr Opin Cell Biol. 2008;20(2):171â€“9.
</cite> [<a href="https://doi.org/10.1016/j.ceb.2008.01.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2324124/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18342499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Watt%20FM,%20Estrach%20S,%20Ambler%20CA.%20Epidermal%20Notch%20signalling:%20differentiation,%20cancer%20and%20adhesion.%20Curr%20Opin%20Cell%20Biol.%202008;20(2):171%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Kim N, Hong Y, Kwon D, Yoon S. Somatic mutaome profile in human cancer tissues. Genomics Inform. 2013;11(4):239â€“44.
</cite> [<a href="https://doi.org/10.5808/GI.2013.11.4.239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3897852/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24465236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20N,%20Hong%20Y,%20Kwon%20D,%20Yoon%20S.%20Somatic%20mutaome%20profile%20in%20human%20cancer%20tissues.%20Genomics%20Inform.%202013;11(4):239%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Bhatia R, Gautam SK, Cannon A, Thompson C, Hall BR, Aithal A, Banerjee K, Jain M, Solheim JC, Kumar S, et al. Cancer-associated mucins: role in immune modulation and metastasis. Cancer Metastasis Rev. 2019;38(1â€“2):223â€“36.
</cite> [<a href="https://doi.org/10.1007/s10555-018-09775-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6614013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30618016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bhatia%20R,%20Gautam%20SK,%20Cannon%20A,%20Thompson%20C,%20Hall%20BR,%20Aithal%20A,%20Banerjee%20K,%20Jain%20M,%20Solheim%20JC,%20Kumar%20S,%20et%20al.%20Cancer-associated%20mucins:%20role%20in%20immune%20modulation%20and%20metastasis.%20Cancer%20Metastasis%20Rev.%202019;38(1%E2%80%932):223%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Gupta R, Strbenac D, Satgunaseelan L, Cheung VK-Y, Narayanappa H, Ashford B, Mitchell J, Thind A, Palme CE, Châ€™ng S et al. Comparing genomic landscapes of oral and cutaneous squamous cell carcinoma of the head and neck: quest for novel diagnostic markers. Mod Pathol. 2023;36(8):100190.</cite> [<a href="https://doi.org/10.1016/j.modpat.2023.100190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37080394/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Chakraborty S, Swanson BJ, Bonthu N, Batra SK. Aberrant upregulation of MUC4 mucin expression in cutaneous condyloma acuminatum and squamous cell carcinoma suggests a potential role in the diagnosis and therapy of skin diseases. J Clin Pathol. 2010;63(7):579â€“84.
</cite> [<a href="https://doi.org/10.1136/jcp.2010.076125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2920126/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20591909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chakraborty%20S,%20Swanson%20BJ,%20Bonthu%20N,%20Batra%20SK.%20Aberrant%20upregulation%20of%20MUC4%20mucin%20expression%20in%20cutaneous%20condyloma%20acuminatum%20and%20squamous%20cell%20carcinoma%20suggests%20a%20potential%20role%20in%20the%20diagnosis%20and%20therapy%20of%20skin%20diseases.%20J%20Clin%20Pathol.%202010;63(7):579%E2%80%9384." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Welch JS, Ley TJ, Link DC, Miller CA, Larson DE, Koboldt DC, Wartman LD, Lamprecht TL, Liu F, Xia J, et al. The origin and evolution of mutations in acute myeloid leukemia. Cell. 2012;150(2):264â€“78.
</cite> [<a href="https://doi.org/10.1016/j.cell.2012.06.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3407563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22817890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Welch%20JS,%20Ley%20TJ,%20Link%20DC,%20Miller%20CA,%20Larson%20DE,%20Koboldt%20DC,%20Wartman%20LD,%20Lamprecht%20TL,%20Liu%20F,%20Xia%20J,%20et%20al.%20The%20origin%20and%20evolution%20of%20mutations%20in%20acute%20myeloid%20leukemia.%20Cell.%202012;150(2):264%E2%80%9378." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Saito Y, Mochizuki Y, Ogahara I, Watanabe T, Hogdal L, Takagi S, Sato K, Kaneko A, Kajita H, Uchida N, et al. Overcoming mutational complexity in acute myeloid leukemia by inhibition of critical pathways. Sci Transl Med. 2017. 10.1126/scitranslmed.aao1214.
</cite> [<a href="https://doi.org/10.1126/scitranslmed.aao1214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6377281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29070697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saito%20Y,%20Mochizuki%20Y,%20Ogahara%20I,%20Watanabe%20T,%20Hogdal%20L,%20Takagi%20S,%20Sato%20K,%20Kaneko%20A,%20Kajita%20H,%20Uchida%20N,%20et%20al.%20Overcoming%20mutational%20complexity%20in%20acute%20myeloid%20leukemia%20by%20inhibition%20of%20critical%20pathways.%20Sci%20Transl%20Med.%202017.%2010.1126/scitranslmed.aao1214." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Marin-Bejar O, Rogiers A, Dewaele M, Femel J, Karras P, Pozniak J, Bervoets G, Van Raemdonck N, Pedri D, Swings T, et al. Evolutionary predictability of genetic versus nongenetic resistance to anticancer drugs in melanoma. Cancer Cell. 2021;39(8):1135â€“e11491138.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2021.05.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34143978/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marin-Bejar%20O,%20Rogiers%20A,%20Dewaele%20M,%20Femel%20J,%20Karras%20P,%20Pozniak%20J,%20Bervoets%20G,%20Van%20Raemdonck%20N,%20Pedri%20D,%20Swings%20T,%20et%20al.%20Evolutionary%20predictability%20of%20genetic%20versus%20nongenetic%20resistance%20to%20anticancer%20drugs%20in%20melanoma.%20Cancer%20Cell.%202021;39(8):1135%E2%80%93e11491138." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Xu J, Liao K, Yang X, Wu C, Wu W, Han S. Using single-cell sequencing technology to detect Circulating tumor cells in solid tumors. Mol Cancer. 2021;20(1):104.</cite> [<a href="https://doi.org/10.1186/s12943-021-01392-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8375060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34412644/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Duan Q, Tang C, Ma Z, Chen C, Shang X, Yue J, Jiang H, Gao Y, Xu B. Genomic heterogeneity and clonal evolution in gastroesophageal junction cancer revealed by single cell DNA sequencing. Front Oncol. 2021. 10.3389/fonc.2021.672020.
</cite> [<a href="https://doi.org/10.3389/fonc.2021.672020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8144650/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34046362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duan%20Q,%20Tang%20C,%20Ma%20Z,%20Chen%20C,%20Shang%20X,%20Yue%20J,%20Jiang%20H,%20Gao%20Y,%20Xu%20B.%20Genomic%20heterogeneity%20and%20clonal%20evolution%20in%20gastroesophageal%20junction%20cancer%20revealed%20by%20single%20cell%20DNA%20sequencing.%20Front%20Oncol.%202021.%2010.3389/fonc.2021.672020." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Kim C, Gao R, Sei E, Brandt R, Hartman J, Hatschek T, Crosetto N, Foukakis T, Navin NE. Chemoresistance evolution in triple-negative breast cancer delineated by single-cell sequencing. Cell. 2018;173(4):879.
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.03.041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6132060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29681456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20C,%20Gao%20R,%20Sei%20E,%20Brandt%20R,%20Hartman%20J,%20Hatschek%20T,%20Crosetto%20N,%20Foukakis%20T,%20Navin%20NE.%20Chemoresistance%20evolution%20in%20triple-negative%20breast%20cancer%20delineated%20by%20single-cell%20sequencing.%20Cell.%202018;173(4):879." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Dai Z, Gu X-y, Xiang S-y, Gong D-d, Man C-f, Fan Y. Research and application of single-cell sequencing in tumor heterogeneity and drug resistance of circulating tumor cells. Biomark Res. 2020. 10.1186/s40364-020-00240-1.
</cite> [<a href="https://doi.org/10.1186/s40364-020-00240-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7653877/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33292625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dai%20Z,%20Gu%20X-y,%20Xiang%20S-y,%20Gong%20D-d,%20Man%20C-f,%20Fan%20Y.%20Research%20and%20application%20of%20single-cell%20sequencing%20in%20tumor%20heterogeneity%20and%20drug%20resistance%20of%20circulating%20tumor%20cells.%20Biomark%20Res.%202020.%2010.1186/s40364-020-00240-1." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Shi L, Shang X, Nie K, Lin Z, Zheng M, Wang M, Yuan H, Zhu Z. Identification of potential crucial genes associated with the pathogenesis and prognosis of liver hepatocellular carcinoma. J Clin Pathol. 2021;74(8):504â€“12.
</cite> [<a href="https://doi.org/10.1136/jclinpath-2020-206979" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33004423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi%20L,%20Shang%20X,%20Nie%20K,%20Lin%20Z,%20Zheng%20M,%20Wang%20M,%20Yuan%20H,%20Zhu%20Z.%20Identification%20of%20potential%20crucial%20genes%20associated%20with%20the%20pathogenesis%20and%20prognosis%20of%20liver%20hepatocellular%20carcinoma.%20J%20Clin%20Pathol.%202021;74(8):504%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Saadeh FS, Morsi RZ, El-Kurdi A, Nemer G, Mahfouz R, Charafeddine M, Khoury J, Najjar MW, Khoueiry P, Assi HI. Correlation of genetic alterations by whole-exome sequencing with clinical outcomes of glioblastoma patients from the Lebanese population. PLoS One. 2020. 10.1371/journal.pone.0242793.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0242793" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7688136/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33237934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saadeh%20FS,%20Morsi%20RZ,%20El-Kurdi%20A,%20Nemer%20G,%20Mahfouz%20R,%20Charafeddine%20M,%20Khoury%20J,%20Najjar%20MW,%20Khoueiry%20P,%20Assi%20HI.%20Correlation%20of%20genetic%20alterations%20by%20whole-exome%20sequencing%20with%20clinical%20outcomes%20of%20glioblastoma%20patients%20from%20the%20Lebanese%20population.%20PLoS%20One.%202020.%2010.1371/journal.pone.0242793." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Shang J, Zhang X, Hou G, Qi Y. HMMR potential as a diagnostic and prognostic biomarker of cancer-speculation based on a pan-cancer analysis. Front Surg. 2023. 10.3389/fsurg.2022.998598.
</cite> [<a href="https://doi.org/10.3389/fsurg.2022.998598" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9873350/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36704516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shang%20J,%20Zhang%20X,%20Hou%20G,%20Qi%20Y.%20HMMR%20potential%20as%20a%20diagnostic%20and%20prognostic%20biomarker%20of%20cancer-speculation%20based%20on%20a%20pan-cancer%20analysis.%20Front%20Surg.%202023.%2010.3389/fsurg.2022.998598." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Bu C, Zheng X, Zhao X, Xu T, Bai X, Jia Y, Chen M, Hao L, Xiao J, Zhang Z. Genbase: a nucleotide sequence database. Genomics Proteomics Bioinformatics. 2024. 10.1093/gpbjnl/qzae047.
</cite> [<a href="https://doi.org/10.1093/gpbjnl/qzae047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11434157/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38913867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bu%20C,%20Zheng%20X,%20Zhao%20X,%20Xu%20T,%20Bai%20X,%20Jia%20Y,%20Chen%20M,%20Hao%20L,%20Xiao%20J,%20Zhang%20Z.%20Genbase:%20a%20nucleotide%20sequence%20database.%20Genomics%20Proteomics%20Bioinformatics.%202024.%2010.1093/gpbjnl/qzae047." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Database resources of the national genomics data center. China National center for bioinformation in 2024. Nucleic Acids Res. 2024;52(D1):D18â€“32.
</cite> [<a href="https://doi.org/10.1093/nar/gkad1078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38018256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Database%20resources%20of%20the%20national%20genomics%20data%20center.%20China%20National%20center%20for%20bioinformation%20in%202024.%20Nucleic%20Acids%20Res.%202024;52(D1):D18%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374374/bin/12885_2025_14585_MOESM1_ESM.xls" data-ga-action="click_feat_suppl" class="usa-link">12885_2025_14585_MOESM1_ESM.xls</a><sup> (5.2MB, xls) </sup><p>Supplemental file 1. The somatic mutations of 12 CSCC patients identified by bulk exome sequencing</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374374/bin/12885_2025_14585_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">12885_2025_14585_MOESM2_ESM.pdf</a><sup> (421KB, pdf) </sup><p>Supplemental file 2. The variant allele fraction (VAF) of each variant in every sequenced sample. The images in the PDF file are arranged in order from Patient 1 to Patient 12</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374374/bin/12885_2025_14585_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12885_2025_14585_MOESM3_ESM.xlsx</a><sup> (11.9KB, xlsx) </sup><p>Supplemental file 3. The MPT panel including 66 genes that were designed from somatic mutations identified by bulk sequencing in 12 CSCC patients</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated during the current study have been deposited in the GenBase [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] in the National Genomics Data Center [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>], China National Center for Bioinformation/Beijing Institute of Genomics, Chinese Academy of Sciences (GSA-Human: HRA008813 and HRA008816) that are publicly accessible at <a href="https://ngdc.cncb.ac.cn/gsa-human" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa-human</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Cancer are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12885-025-14585-z"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12885_2025_Article_14585.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.6Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12374374/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12374374/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374374%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374374/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12374374/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12374374/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40847288/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12374374/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40847288/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12374374/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12374374/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Edsnn28XZarrsXKGa9c5TuzwyiO4xOMcdK9HfL3DbjqUmBnBw3EMHJSFWJYbRnT1">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
