
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Accurate modeling and simulation of the effect of bacterial growth on the pH of culture media using artificial intelligence approaches - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AFF68AF3269305AFF6000A04E4C2.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368003/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Accurate modeling and simulation of the effect of bacterial growth on the pH of culture media using artificial intelligence approaches">
<meta name="citation_author" content="Suleiman Ibrahim Mohammad">
<meta name="citation_author_institution" content="Electronic Marketing and Social Media, Economic and Administrative Sciences, Zarqa University, Zarqa, Jordan">
<meta name="citation_author_institution" content="INTI International University, Nilai, Negeri Sembilan 71800 Malaysia">
<meta name="citation_author" content="Hamza Abu Owida">
<meta name="citation_author_institution" content="Medical Engineering Department, Faculty of Engineering, Al-Ahliyya Amman University, Amman, 19328 Jordan">
<meta name="citation_author" content="Asokan Vasudevan">
<meta name="citation_author_institution" content="Faculty of Business and Communications, INTI International University, Nilai, 71800 Negeri Sembilan Malaysia">
<meta name="citation_author_institution" content="Shinawatra University, 99 Moo 10, Bangtoey, Samkhok, 12160 Pathum Thani Thailand">
<meta name="citation_author" content="Suhas Ballal">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, School of Sciences, JAIN (Deemed to be University), Bangalore, Karnataka India">
<meta name="citation_author" content="Shaker Al-Hasnaawei">
<meta name="citation_author_institution" content="College of Pharmacy, The Islamic University, Najaf, Iraq">
<meta name="citation_author_institution" content="Department of Medical Analysis, Medical Laboratory Technique College, The Islamic University of Al Diwaniyah, Al Diwaniyah, Iraq">
<meta name="citation_author" content="Subhashree Ray">
<meta name="citation_author_institution" content="Department of Biochemistry, IMS and SUM Hospital, Siksha ’O’ Anusandhan (Deemed to be University), Bhubaneswar, 751003 Odisha India">
<meta name="citation_author" content="Naveen Chandra Talniya">
<meta name="citation_author_institution" content="Department of Chemistry, Graphic Era Hill University, Dehradun, India">
<meta name="citation_author_institution" content="Centre for Promotion of Research, Graphic Era Deemed to be University, Dehradun, Uttarakhand, 248002 India">
<meta name="citation_author" content="Aashna Sinha">
<meta name="citation_author_institution" content="School of Applied and Life Sciences, Division of Research and Innovation, Uttaranchal University Dehradun, Dehradun, Uttarakhand India">
<meta name="citation_author" content="Vatsal Jain">
<meta name="citation_author_institution" content="Centre for Research Impact &amp; Outcome, Chitkara University Institute of Engineering and Technology, Chitkara University, Rajpura, 140401 Punjab India">
<meta name="citation_author" content="Ahmad Abumalek">
<meta name="citation_author_institution" content="Faculty of Engineering, Balkh University, Balkh, Afghanistan">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30569">
<meta name="citation_doi" content="10.1038/s41598-025-16150-x">
<meta name="citation_pmid" content="40835707">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368003/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368003/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368003/pdf/41598_2025_Article_16150.pdf">
<meta name="description" content="This research investigates the impact of bacterial growth on the pH of culture media, emphasizing its significance in microbiological and biotechnological applications. A range of sophisticated artificial intelligence methods, including ...">
<meta name="og:title" content="Accurate modeling and simulation of the effect of bacterial growth on the pH of culture media using artificial intelligence approaches">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="This research investigates the impact of bacterial growth on the pH of culture media, emphasizing its significance in microbiological and biotechnological applications. A range of sophisticated artificial intelligence methods, including ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368003/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368003">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-16150-x"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_16150.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368003%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368003/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368003/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368003/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 20;15:30569. doi: <a href="https://doi.org/10.1038/s41598-025-16150-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-16150-x</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Accurate modeling and simulation of the effect of bacterial growth on the pH of culture media using artificial intelligence approaches</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mohammad%20SI%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Suleiman Ibrahim Mohammad</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Suleiman Ibrahim Mohammad</span></h3>
<div class="p">
<sup>1</sup>Electronic Marketing and Social Media, Economic and Administrative Sciences, Zarqa University, Zarqa, Jordan </div>
<div class="p">
<sup>2</sup>INTI International University, Nilai, Negeri Sembilan 71800 Malaysia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mohammad%20SI%22%5BAuthor%5D" class="usa-link"><span class="name western">Suleiman Ibrahim Mohammad</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Owida%20HA%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Hamza Abu Owida</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Hamza Abu Owida</span></h3>
<div class="p">
<sup>3</sup>Medical Engineering Department, Faculty of Engineering, Al-Ahliyya Amman University, Amman, 19328 Jordan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Owida%20HA%22%5BAuthor%5D" class="usa-link"><span class="name western">Hamza Abu Owida</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vasudevan%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Asokan Vasudevan</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Asokan Vasudevan</span></h3>
<div class="p">
<sup>4</sup>Faculty of Business and Communications, INTI International University, Nilai, 71800 Negeri Sembilan Malaysia </div>
<div class="p">
<sup>5</sup>Shinawatra University, 99 Moo 10, Bangtoey, Samkhok, 12160 Pathum Thani Thailand </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vasudevan%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Asokan Vasudevan</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ballal%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Suhas Ballal</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Suhas Ballal</span></h3>
<div class="p">
<sup>6</sup>Department of Chemistry and Biochemistry, School of Sciences, JAIN (Deemed to be University), Bangalore, Karnataka India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ballal%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Suhas Ballal</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Al-Hasnaawei%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Shaker Al-Hasnaawei</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Shaker Al-Hasnaawei</span></h3>
<div class="p">
<sup>7</sup>College of Pharmacy, The Islamic University, Najaf, Iraq </div>
<div class="p">
<sup>8</sup>Department of Medical Analysis, Medical Laboratory Technique College, The Islamic University of Al Diwaniyah, Al Diwaniyah, Iraq </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Al-Hasnaawei%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shaker Al-Hasnaawei</span></a>
</div>
</div>
<sup>7,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ray%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Subhashree Ray</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Subhashree Ray</span></h3>
<div class="p">
<sup>9</sup>Department of Biochemistry, IMS and SUM Hospital, Siksha ’O’ Anusandhan (Deemed to be University), Bhubaneswar, 751003 Odisha India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ray%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Subhashree Ray</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Talniya%20NC%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Naveen Chandra Talniya</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Naveen Chandra Talniya</span></h3>
<div class="p">
<sup>10</sup>Department of Chemistry, Graphic Era Hill University, Dehradun, India </div>
<div class="p">
<sup>11</sup>Centre for Promotion of Research, Graphic Era Deemed to be University, Dehradun, Uttarakhand, 248002 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Talniya%20NC%22%5BAuthor%5D" class="usa-link"><span class="name western">Naveen Chandra Talniya</span></a>
</div>
</div>
<sup>10,</sup><sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sinha%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Aashna Sinha</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Aashna Sinha</span></h3>
<div class="p">
<sup>12</sup>School of Applied and Life Sciences, Division of Research and Innovation, Uttaranchal University Dehradun, Dehradun, Uttarakhand India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sinha%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Aashna Sinha</span></a>
</div>
</div>
<sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jain%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Vatsal Jain</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Vatsal Jain</span></h3>
<div class="p">
<sup>13</sup>Centre for Research Impact &amp; Outcome, Chitkara University Institute of Engineering and Technology, Chitkara University, Rajpura, 140401 Punjab India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jain%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Vatsal Jain</span></a>
</div>
</div>
<sup>13</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abumalek%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Ahmad Abumalek</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Ahmad Abumalek</span></h3>
<div class="p">
<sup>14</sup>Faculty of Engineering, Balkh University, Balkh, Afghanistan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abumalek%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Ahmad Abumalek</span></a>
</div>
</div>
<sup>14,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Electronic Marketing and Social Media, Economic and Administrative Sciences, Zarqa University, Zarqa, Jordan </div>
<div id="Aff2">
<sup>2</sup>INTI International University, Nilai, Negeri Sembilan 71800 Malaysia </div>
<div id="Aff3">
<sup>3</sup>Medical Engineering Department, Faculty of Engineering, Al-Ahliyya Amman University, Amman, 19328 Jordan </div>
<div id="Aff4">
<sup>4</sup>Faculty of Business and Communications, INTI International University, Nilai, 71800 Negeri Sembilan Malaysia </div>
<div id="Aff5">
<sup>5</sup>Shinawatra University, 99 Moo 10, Bangtoey, Samkhok, 12160 Pathum Thani Thailand </div>
<div id="Aff6">
<sup>6</sup>Department of Chemistry and Biochemistry, School of Sciences, JAIN (Deemed to be University), Bangalore, Karnataka India </div>
<div id="Aff7">
<sup>7</sup>College of Pharmacy, The Islamic University, Najaf, Iraq </div>
<div id="Aff8">
<sup>8</sup>Department of Medical Analysis, Medical Laboratory Technique College, The Islamic University of Al Diwaniyah, Al Diwaniyah, Iraq </div>
<div id="Aff9">
<sup>9</sup>Department of Biochemistry, IMS and SUM Hospital, Siksha ’O’ Anusandhan (Deemed to be University), Bhubaneswar, 751003 Odisha India </div>
<div id="Aff10">
<sup>10</sup>Department of Chemistry, Graphic Era Hill University, Dehradun, India </div>
<div id="Aff11">
<sup>11</sup>Centre for Promotion of Research, Graphic Era Deemed to be University, Dehradun, Uttarakhand, 248002 India </div>
<div id="Aff12">
<sup>12</sup>School of Applied and Life Sciences, Division of Research and Innovation, Uttaranchal University Dehradun, Dehradun, Uttarakhand India </div>
<div id="Aff13">
<sup>13</sup>Centre for Research Impact &amp; Outcome, Chitkara University Institute of Engineering and Technology, Chitkara University, Rajpura, 140401 Punjab India </div>
<div id="Aff14">
<sup>14</sup>Faculty of Engineering, Balkh University, Balkh, Afghanistan </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jun 7; Accepted 2025 Aug 13; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368003  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835707/" class="usa-link">40835707</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par17">This research investigates the impact of bacterial growth on the pH of culture media, emphasizing its significance in microbiological and biotechnological applications. A range of sophisticated artificial intelligence methods, including One-Dimensional Convolutional Neural Network (1D-CNN), Artificial Neural Networks (ANN), Decision Tree (DT), Ensemble Learning (EL), Adaptive Boosting (AdaBoost), Random Forest (RF), and Least Squares Support Vector Machine (LSSVM), were utilized to model and predict pH variations with high accuracy. The Coupled Simulated Annealing (CSA) algorithm was employed to optimize the hyperparameters of these models, enhancing their predictive performance. A robust dataset comprising 379 experimental data points was compiled, of which 80% (303 points) were used for training and 20% (76 points) for testing. The study focuses on three bacterial strains including Pseudomonas pseudoalcaligenes CECT 5344, Pseudomonas putida KT2440, and Escherichia coli ATCC 25,922 cultured in Luria Bertani (LB) and M63 media, across varying initial pH levels, time intervals, and bacterial cell concentrations (OD600). Key input variables for the models included bacterial type, culture medium type, initial pH, time (hours), and bacterial cell concentration, all critical to pH dynamics. Sensitivity analysis using Monte Carlo simulations revealed bacterial cell concentration as the most influential factor, followed by time, culture medium type, initial pH, and bacterial type. The dataset was rigorously validated before training to ensure its suitability for predictive modeling. Evaluation of model performance demonstrated that the 1D-CNN model exhibited enhanced predictive precision, attaining the minimal RMSE and the maximum R² values and MAPE percentages in both training and testing phases. These findings underscore the efficacy of artificial intelligence techniques, particularly 1D-CNN, in precisely predicting pH changes in culture media due to bacterial growth. This methodology provides a reliable, cost-effective, and efficient alternative to traditional experimental approaches, enabling researchers to forecast pH behavior with greater confidence and reduced experimental effort.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-16150-x.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Bacterial growth, pH variation modeling, Artificial intelligence techniques, Optimization, Culture media, Sensitivity analysis, pH prediction</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Chemistry, Engineering, Materials science, Mathematics and computing</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par18">Maintaining pH homeostasis plays a crucial role in cellular metabolism, and its significance becomes evident through multiple biochemical and physiological lenses<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. One key aspect is that the structure and function of biomacromolecules particularly proteins and enzymes are highly sensitive to pH, as even minor deviations can lead to conformational changes that impair biological activity<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Furthermore, pH directly influences the kinetics and thermodynamics of many metabolic reactions, especially those involving proton transfer, thus governing the directionality and efficiency of crucial biochemical pathways<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Notably, pH fluctuations can substantially affect energy metabolism, as the proton motive force is central to ATP synthesis via oxidative phosphorylation<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. In the context of bacterial growth, understanding how microbial metabolism alters the pH of culture media is vital not only for optimizing growth conditions in industrial and clinical microbiology but also for elucidating microbial interactions, biofilm formation, and pathogenesis. Modeling these dynamics provides critical insights into microbial ecology and supports the development of pH-controlled bioprocesses and therapeutic strategies<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>.</p>
<p id="Par19">In eukaryotic cells, molecular mechanisms are in place to regulate intracellular pH within a specific range across different subcellular compartments. For instance, mitochondria and chloroplasts are surrounded by cytoplasm, which tightly controls pH homeostasis. In contrast, bacteria thrive in a wide variety of environments, and the pH of their surroundings shapes their lifestyle, forming the basis for categorizing them as acidophiles (pH 1–3), alkaliphiles (pH 10–13), or neutrophiles (pH 5.5–9). Nevertheless, like eukaryotes, bacteria generally maintain a near-neutral intracellular pH to support metabolic activity and preserve cellular integrity. Their ability to regulate pH involves a range of mechanisms to sense and adapt to extracellular pH fluctuations<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Several factors can influence environmental pH changes. The initial pH and composition of the growth medium play a fundamental role, followed by the bacterial growth phase and the organism’s physiology and optimal pH range. In addition, microbial metabolism itself can alter extracellular pH. As a result, the growth of one bacterial strain may impact the proliferation of neighboring strains in a shared ecosystem, potentially shaping the fate of entire microbial populations<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>.</p>
<p id="Par20">Understanding pH homeostasis in this context can offer practical applications in fields such as bioremediation or the study of pathogenic bacterial behavior. While many studies have explored the effects of initial pH on bacterial growth or specific metabolite production, fewer have focused on how pH evolves during microbial growth. The aim of the present study was to describe and elucidate pH changes throughout bacterial growth and to identify the factors driving these changes. As a proof of concept, we demonstrated that in silico predictions of pH shifts using a well-curated genome-scale metabolic model of <em>Pseudomonas putida</em> KT2440 are consistent with experimentally observed pH dynamics, at least in media containing glucose, glycerol, or citrate as carbon sources<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Sánchez-Clemente and team studied pH effects on growth and extracellular pH changes in Escherichia coli ATCC 25,922, Pseudomonas putida KT2440, and Pseudomonas pseudoalcaligenes CECT 5344, selected for cyanide assimilation under alkaline conditions. They tested initial pH (6, 7, 8 for E. coli and P. putida; 7.5, 8.25, 9 for P. pseudoalcaligenes) in LB-medium, noting pH convergence to strain-specific values by stationary phase<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. In glucose minimal medium, Pseudomonadaceae pH remained stable, but E. coli’s pH dropped to 4.5 at initial pH 6, stopping growth, though higher pH allowed recovery. Carbon sources affected pH: glucose and glycerol stabilized it, while citrate caused alkalinization, matching P. putida KT2440 model predictions.</p>
<p id="Par21">The observed pH fluctuations during bacterial growth can be attributed to the metabolic activities of the microorganisms, which modulate their surrounding environment through the consumption of nutrients and the excretion of metabolites. For instance, species like <em>Lactobacillus plantarum</em> produce lactic acid as a metabolic by-product, thereby reducing the medium pH, while others such as <em>Corynebacterium ammoniagenes</em> produce ammonia via urease activity, which increases the pH of the medium<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. These opposing mechanisms contribute to a diverse range of pH shifts depending on the bacterial strain and the availability of carbon and nitrogen sources such as glucose and urea.</p>
<p id="Par22">Furthermore, the impact of these metabolic products on bacterial viability can manifest as beneficial (positive feedback) or detrimental (negative feedback) effects. For example, the accumulation of alkaline metabolites by <em>Pseudomonas veronii</em> despite its preference for acidic environments can lead to a phenomenon known as ecological suicide, where the population inadvertently alters the pH beyond its tolerance range, resulting in its own extinction<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. These feedback mechanisms illustrate how metabolic pathways intricately influence pH dynamics, supporting the predictive capacity of data-driven models like CNN in capturing the nonlinear relationship between bacterial growth and environmental pH. Incorporating such metabolic insights aligns the modeling outcomes with known biochemical behaviors and strengthens the interpretability of the predictions.</p>
<p id="Par23">Collectively, experimental research has generated a robust dataset elucidating the pH dynamics of culture media influenced by bacterial growth under diverse conditions. These data serve as vital inputs for microbiological modeling and process optimization, while also facilitating data-driven predictive approaches, such as artificial intelligence, to enhance the efficiency of pH prediction with reduced experimental demands.</p>
<p id="Par24">While prior studies have examined the pH changes in culture media due to bacterial activity, comprehensive datasets encompassing various bacterial strains and media types remain limited. Consequently, there is a pressing need to systematically gather extensive pH data across different bacterial types, culture media, initial pH levels, time intervals, and bacterial cell concentrations. Given the critical role of pH in microbiological processes and biotechnological applications, accurately predicting pH variations in culture media presents a complex challenge due to the interplay of factors such as bacterial type, culture medium, initial pH, time, and cell concentration. The pH of culture media is a key parameter in optimizing microbial growth conditions, bioreactor design, and bioprocess development, highlighting the necessity for precise pH predictions in practical applications. Traditional experimental methods for measuring pH, although dependable, are often resource-intensive and time-consuming, necessitating advanced computational techniques capable of providing rapid and reliable predictions.</p>
<p id="Par25">This study bridges this gap by employing a suite of artificial intelligence models, including One-Dimensional Convolutional Neural Network (1D-CNN), Artificial Neural Networks (ANN), Decision Tree (DT), Adaptive Boosting (AdaBoost), Random Forest (RF), Ensemble Learning (EL), and Least Squares Support Vector Machine (LSSVM), to simulate and predict pH variations in Luria Bertani (LB) and M63 media influenced by three bacterial strains: Escherichia coli ATCC 25,922, Pseudomonas putida KT2440, and Pseudomonas pseudoalcaligenes CECT 5344. To optimize the performance of these models, their hyperparameters were fine-tuned using the Coupled Simulated Annealing (CSA) algorithm. The models were developed and validated using a comprehensive dataset comprising 379 experimental data points, with 303 allocated for training (80%) and 76 for testing (20%). Thorough dataset validation ensured its suitability for predictive modeling, while sensitivity analysis via Monte Carlo simulations assessed the impact of each input parameter on pH outcomes. Model performance was meticulously evaluated using statistical metrics and visual representations, identifying the 1D-CNN model as the most accurate in predicting pH changes. The overall research methodology is depicted in Fig. <a href="#Fig1" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368003_41598_2025_16150_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/7791f4fdf484/41598_2025_16150_Fig1_HTML.jpg" loading="lazy" id="d33e468" height="240" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Procedure for determining the top-performing data-driven model.</p></figcaption></figure><p id="Par27">Although various artificial intelligence (AI) algorithms have been successfully applied in microbial systems, the modeling of pH dynamics in bacterial culture media remains largely unexplored. To the best of our knowledge, no comprehensive or generalizable predictive model exists for forecasting pH variations across diverse bacterial strains and culture conditions. A rare exception is the phenomenological model introduced by Ratzke and Gore, which utilized two differential equations to simulate microbial growth and environmental pH modification<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Their model captures basic pH-induced feedback loops within mono- and co-culture systems and demonstrated emergent behaviors such as ecological suicide, Allee effects, and bistability. However, the model remains qualitative and limited to synthetic examples with fixed parameters, without being trained or validated against experimental data at a large scale. In contrast, the present study offers a data-driven framework based on advanced machine learning models to quantitatively predict pH trajectories based on real experimental datasets. Therefore, our approach represents a novel and scalable contribution to microbial pH modeling.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Overview of machine learning and thermodynamic techniques</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Machine learning methods</h3>
<p id="Par28">Artificial intelligence techniques have emerged as vital tools in biotechnological research for predictive modeling, particularly in scenarios involving complex variable interactions such as pH variation forecasting. In this study, seven advanced artificial intelligence algorithms including 1D-CNN, ANN, DT, RF, AdaBoost, EL, and LSSVM are utilized to model the pH dynamics of culture media influenced by bacterial growth. These methods excel at detecting intricate, non-linear patterns within experimental datasets, offering a robust and efficient alternative to conventional experimental approaches due to their computational efficiency, adaptability, and high predictive precision.</p>
<section id="Sec4"><h4 class="pmc_sec_title">Artificial neural Network-based models</h4>
<section id="Sec5"><h5 class="pmc_sec_title">Convolutional neural networks (1D-CNNs)</h5>
<p id="Par29">1D-CNNs are specialized deep learning architectures designed to handle structured data, such as images. They employ convolutional layers to detect localized features like edges, patterns, and forms within the input, enabling robust identification of intricate structures and relationships<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Mathematically, the convolution operation in a 1D-CNN is expressed as Eq. (<a href="#Equ1" class="usa-link">1</a>):</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/261875edd543/d33e495.gif" loading="lazy" id="d33e495" alt="graphic file with name d33e495.gif"></td>
<td class="label">1</td>
</tr></table>
<p id="Par30">Here, f denotes the input image or feature map, g represents the convolutional kernel, and (x, y) indicate the spatial coordinates of the output feature map.</p>
<p id="Par31">1D-CNNs are composed of multiple convolutional layers, which progressively extract more complex and abstract features<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. A key advantage of 1D-CNNs is their ability to process high-dimensional datasets, making them well-suited for tasks such as image classification and object recognition. Pooling layers are often incorporated to reduce data dimensionality and prevent overfitting. Due to their powerful capabilities, 1D-CNNs have revolutionized computer vision. By stacking several convolutional layers, 1D-CNNs can learn hierarchical feature representations, capturing critical details at varying levels of abstraction. This multi-layered approach enables 1D-CNNs to effectively interpret complex visual data, driving significant advancements in numerous computer vision applications<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. 1D-CNN was used because of its strong ability to model spatial patterns and hierarchical representations, which are useful when capturing complex, nonlinear interactions among input features. Its use in a regression context for biological datasets is relatively novel.</p></section><section id="Sec6"><h5 class="pmc_sec_title">Artificial neural network (ANN)</h5>
<p id="Par32">In this research, ANN used is a Multi-Layer Perceptron (MLP) architecture, which consists of an input layer, one or more hidden layers, and an output layer. Typically, an ANN comprises an input layer, one or more hidden layers, and an output layer, each composed of interconnected artificial neurons<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Within the ANN algorithm, input data is first introduced to the input layer, where neurons in each subsequent hidden layer process the outputs from the preceding layer, generating new outputs. This layer-by-layer data transformation persists until the information reaches the output layer, yielding the final prediction or outcome<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. The result of a neuron in the ANN is mathematically represented as Eq. (<a href="#Equ2" class="usa-link">2</a>):</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/7f0c8442b599/d33e531.gif" loading="lazy" id="d33e531" alt="graphic file with name d33e531.gif"></td>
<td class="label">2</td>
</tr></table>
<p id="Par33">where n is the number of input features, w<sub>i</sub> and x<sub>i</sub> are the i-th weight and input value respectively, and b is the bias term.</p>
<p id="Par34">The training process of an ANN involves optimizing the weights and biases to minimize the error between predicted and actual outputs, typically using a backpropagation algorithm combined with gradient descent<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. During backpropagation, the error is propagated backward from the output layer to the input layer, adjusting the weights and biases based on the calculated gradients. This iterative process continues until the model converges to an acceptable error level or a predefined number of epochs is reached. Activation functions, such as sigmoid, Rectified Linear Unit (ReLU) are applied to the neurons’ outputs to introduce non-linearity, enabling the ANN to solve complex, non-linear problems effectively<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. The flexibility and robustness of ANNs make them suitable for applications ranging from image recognition to natural language processing, although they may require careful tuning and sufficient computational resources to achieve optimal performance. ANN was chosen due to its flexibility in capturing nonlinear relationships in multivariate biological data, especially when the input variables interact in complex ways. It serves as a baseline deep learning model for comparison.</p></section></section><section id="Sec7"><h4 class="pmc_sec_title">Tree-based models</h4>
<section id="Sec8"><h5 class="pmc_sec_title">Decision tree (DT)</h5>
<p id="Par35">DT are a flexible supervised learning approach suitable for both classification and regression tasks, producing interpretable models through iterative binary partitioning. This method relies on impurity measures, such as the Gini index, information gain, or variance reduction, to determine optimal splits at each node. Internal nodes define decision rules based on features, while leaf nodes provide the final predictions. This process is illustrated in Eqs. (<a href="#Equ3" class="usa-link">3</a>) and (<a href="#Equ4" class="usa-link">4</a>) below:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/9b8a51b077e1/d33e565.gif" loading="lazy" id="d33e565" alt="graphic file with name d33e565.gif"></td>
<td class="label">3</td>
</tr></table>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/179c226c9c6b/d33e571.gif" loading="lazy" id="d33e571" alt="graphic file with name d33e571.gif"></td>
<td class="label">4</td>
</tr></table>
<p id="Par36">The splitting criteria assess feature significance by analyzing probabilistic class distributions. This structured breakdown facilitates clear visualization of decision pathways and supports both numerical and categorical data with minimal preprocessing<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>.</p>
<p id="Par37">Valued for their transparency and ability to capture nonlinear patterns, DT require careful regularization to avoid overfitting, employing techniques such as pruning, restricting tree depth, or integration with ensemble methods. Their key strengths built-in feature selection, resilience to outliers, and minimal data assumptions make them especially useful in regulated domains like medical diagnostics and financial risk assessment<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. DT was applied for its interpretability and fast training. It provides insights into decision rules and feature importance, offering a transparent model that serves as a reference for more complex tree-based methods.</p></section><section id="Sec9"><h5 class="pmc_sec_title">Random forest (RF)</h5>
<p id="Par38">RF method is a powerful supervised learning method widely applied in regression and classification tasks. It operates by constructing a multitude of decision trees, each trained on randomly sampled subsets of the data and features through techniques like bagging (bootstrap aggregation) and random feature selection. This approach fosters diversity by training each tree on a unique random subset, thereby improving predictive accuracy and generalization. For regression jobs, the algorithm computes the average of all tree estimations, while for classification, it selects the predominant class through majority voting. The ensemble of trees mitigates the overfitting often associated with single DT and maintains strong performance even with noisy data or imbalanced classes. Owing to its reliability, versatility, and consistently high performance, RF is extensively used in fields such as medical diagnostics, financial modeling, and bioinformatics<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. The regression output is typically derived as the average of individual tree predictions, formulated as Eq. (<a href="#Equ5" class="usa-link">5</a>):</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/7ea112cb7f17/d33e609.gif" loading="lazy" id="d33e609" alt="graphic file with name d33e609.gif"></td>
<td class="label">5</td>
</tr></table>
<p id="Par39">A significant strength of RF lies in its capacity to assess the relative importance of features, providing critical insights into the effect of every variable on the model’s forecasts. Compared to more intricate models like deep neural networks, RF requires less extensive hyperparameter optimization while delivering robust performance on large, high-dimensional datasets. However, its predictive accuracy may occasionally fall short of advanced methods like gradient boosting, and it can demand substantial computational resources for very large datasets. Nevertheless, RF remains a cornerstone of machine learning due to its combination of interpretability, resilience, and dependable performance, making it a preferred choice for a wide range of analytical applications<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. RF was used to reduce overfitting compared to a single decision tree. By aggregating multiple trees, it enhances stability and is effective for datasets with noise or variable interactions.</p></section><section id="Sec10"><h5 class="pmc_sec_title">Adaptive boosting (AdaBoost)</h5>
<p id="Par40">AdaBoost is a prominent ensemble learning technique that improves predictive accuracy by iteratively training a sequence of weak classifiers, typically basic decision stumps. At each step, the algorithm prioritizes data points misclassified in previous iterations, guiding subsequent learners to focus on these challenging cases. Each weak learner is assigned a weight reflecting its predictive performance, and the final prediction is derived as a weighted combination of all learners’ outputs. This iterative approach enables AdaBoost to perform exceptionally well in binary classification and regression tasks with complex or noisy datasets<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. The process for updating the weight of each training sample is expressed as Eq. (<a href="#Equ6" class="usa-link">6</a>):</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/0a28468a622a/d33e638.gif" loading="lazy" id="d33e638" alt="graphic file with name d33e638.gif"></td>
<td class="label">6</td>
</tr></table>
<p id="Par41">Here, w<sub>i</sub>(t) represents the weight of the i-th sample in the t-th iteration, α<sub>t</sub> denotes the weight or influence of the weak classifier h<sub>t</sub> at that stage, y<sub>i</sub> is the true label of the i-th data point, and h<sub>t</sub>(x<sub>i</sub>) indicates the prediction made by the weak learner h<sub>t</sub> for that instance. The parameter ​ α<sub>t</sub>, which quantifies the contribution of the weak learner to the final model, is computed as Eq. (<a href="#Equ7" class="usa-link">7</a>):</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/b3b4024d3cf1/d33e666.gif" loading="lazy" id="d33e666" alt="graphic file with name d33e666.gif"></td>
<td class="label">7</td>
</tr></table>
<p id="Par42">In this formula, Error<sub>t</sub> signifies the weighted classification error of the weak learner h<sub>t</sub> during the t-th iteration. In this study, AdaBoost was employed to boost prediction precision by sequentially training multiple weak models, with increased focus on difficult-to-predict instances. Cross-validation was used to optimize the learning rate and the number of boosting iterations, ensuring a balance between reducing bias and preventing overfitting for robust generalization. This method was selected for its effectiveness in handling diverse datasets and enhancing the performance of simple classifiers. However, its sensitivity to noisy data and outliers necessitated thorough preprocessing to mitigate these challenges<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. AdaBoost focuses on correcting errors by emphasizing misclassified points, making it valuable for capturing difficult patterns in pH behavior. Its iterative learning strategy improves accuracy over weak base learners.</p></section></section><section id="Sec11"><h4 class="pmc_sec_title">Ensemble/Hybrid models</h4>
<p id="Par43">Ensemble Learning (EL) is not a single model but a general strategy that combines multiple base learners to improve predictive performance. In this study, a heterogeneous ensemble model was developed by integrating three distinct algorithms: Support Vector Machine (SVM) with an RBF kernel, Decision Tree (DT), and K-Nearest Neighbors (KNN) with Euclidean distance. The final prediction was made using a weighted voting mechanism based on the individual performance of each base learner. This EL model differs from Random Forest (RF), which is a homogeneous ensemble method composed exclusively of decision trees trained on random subsets of data and features. The inclusion of diverse model types in EL allows leveraging their complementary strengths, offering improved generalization compared to single-model or single-family ensembles like RF. These approaches are highly effective in reducing overfitting and enhancing generalization, making them well-suited for complex, diverse datasets, such as those employed in modeling pH variations in culture media<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. For instance, in a weighted voting framework, the final prediction ŷ ​is determined as Eq. (<a href="#Equ8" class="usa-link">8</a>):</p>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/46799c595c55/d33e699.gif" loading="lazy" id="d33e699" alt="graphic file with name d33e699.gif"></td>
<td class="label">8</td>
</tr></table>
<p id="Par44">Here, T represents the total number of models in the ensemble, w<sub>t</sub> denotes the weight assigned to the t-th model, h<sub>t</sub>(x) signifies the prediction of the t-th model for input x, I(⋅) is an indicator function returning 1 if the condition holds and 0 otherwise, and c indicates the possible class labels. A custom ensemble was created by combining heterogeneous models (SVM, DT, KNN) to leverage complementary strengths and improve generalization. This hybrid ensemble provides robustness in prediction across varied bacterial and media conditions.</p></section><section id="Sec12"><h4 class="pmc_sec_title">Kernel-based model</h4>
<p id="Par45">Support Vector Machine (SVM) is a supervised learning method commonly applied to both classification and regression problems. Its fundamental principle is to determine the hyperplane that best separates data points from different classes while maximizing the margin between them. For nonlinear problems, SVM employs kernel functions to transform data into higher-dimensional space, allowing linear separation. Despite its robustness and accuracy, standard SVM involves solving a quadratic programming problem, which can be computationally intensive for large datasets. Least Squares Support Vector Machine (LSSVM) modifies the standard SVM by replacing the quadratic optimization with a least squares cost function, transforming the problem into a system of linear equations.</p>
<p id="Par46">Unlike traditional SVM, which relies on quadratic programming, LSSVM utilizes a least squares loss function, simplifying the optimization process into solving a system of linear equations. This reformulation substantially lowers computational demands, making LSSVM highly effective for handling large datasets<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>.</p>
<p id="Par47">The goal of LSSVM is to identify a decision function that reduces prediction errors while preserving the model’s ability to generalize. The prediction function in LSSVM is typically formulated as Eq. (<a href="#Equ9" class="usa-link">9</a>):</p>
<table class="disp-formula p" id="Equ9"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/c8ff9e216d99/d33e726.gif" loading="lazy" id="d33e726" alt="graphic file with name d33e726.gif"></td>
<td class="label">9</td>
</tr></table>
<p id="Par48">where w represents the weight vector, ϕ(x) denotes the transformation of input data x into a higher-dimensional space via a kernel function, and b is the bias term. Frequently employed kernel functions, such as radial basis function (RBF), polynomial, and linear kernels, enable LSSVM to model non-linear patterns in the input data<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>.</p>
<p id="Par49">In this research, LSSVM was selected for its robust performance in managing complex, high-dimensional datasets with minimal computational overhead. Hyperparameters, including the regularization parameter and kernel-specific settings, were fine-tuned through cross-validation to optimize predictive accuracy. Additionally, feature scaling was implemented before training to ensure equitable contributions of all features to the kernel-based similarity computations. LSSVM was selected for its efficiency and effectiveness in modeling nonlinear relations with limited data. Its use of least squares formulation simplifies optimization and is computationally efficient.</p></section></section><section id="Sec13"><h3 class="pmc_sec_title">Coupled simulated annealing (CSA) optimization algorithm</h3>
<p id="Par50">Simulated Annealing (SA) is a probabilistic optimization algorithm inspired by the annealing process in metallurgy, where a material is heated and then slowly cooled to remove defects and reach a stable state. In the context of optimization, Simulated Annealing (SA) replicates this behavior by randomly searching the solution space and occasionally accepting inferior solutions with a defined probability, allowing the algorithm to avoid becoming trapped in local minima. Over time, the algorithm reduces its “temperature” parameter, gradually focusing on more promising areas of the solution space. This makes SA particularly useful for solving complex, multimodal, and non-convex optimization problems. CSA algorithm represents an evolved version of the conventional Simulated Annealing (SA) technique. Unlike standard SA, which navigates the solution space through a single trajectory and employs probabilistic acceptance of suboptimal solutions to escape local minima, CSA advances this method by executing multiple concurrent search processes<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. These processes are interconnected via a shared acceptance criterion, fostering solution diversity and markedly enhancing convergence speed and reliability. By incorporating this coupling mechanism, CSA reduces the chances of early convergence and boosts the probability of identifying the global optimum<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p>
<p id="Par51">The primary advantage of CSA lies in its effective balance of exploration and exploitation. Each independent search generates new potential solutions, but their acceptance is orchestrated globally through an entropy-driven control mechanism. This coordination maintains an optimal level of randomness or “temperature” throughout the optimization process. Consequently, CSA excels in tackling high-dimensional, intricate, and multimodal optimization challenges where conventional approaches may falter. Its applications extend to fields such as engineering design, machine learning hyperparameter optimization, and modeling complex systems<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>.</p></section></section><section id="Sec14"><h2 class="pmc_sec_title">Data gathering and evaluation indices</h2>
<section id="Sec15"><h3 class="pmc_sec_title">Data collection description</h3>
<p id="Par52">The dataset utilized for constructing the artificial intelligence models in this study was compiled from experimental investigations focused on assessing the pH variations in culture media influenced by bacterial growth across diverse conditions. A total of 379 experimental data points were gathered, incorporating critical input variables such as bacterial type, culture medium type, initial pH, time (hours), and bacterial cell concentration (OD600). The bacterial strains examined include Pseudomonas putida KT2440, Escherichia coli ATCC 25,922, and Pseudomonas pseudoalcaligenes CECT 5344, cultured in LB and M63 media. The bacterial strains examined include Pseudomonas putida KT2440, Escherichia coli ATCC 25,922, and Pseudomonas pseudoalcaligenes CECT 5344, cultured in Luria Bertani (LB) and M63 media. The bacterial strains selected for this study including Escherichia coli ATCC 25,922, Pseudomonas putida KT2440, and Pseudomonas pseudoalcaligenes CECT 5344 were chosen based on their relevance to microbiological and biotechnological research. E. coli ATCC 25,922 serves as a standard model organism frequently used in laboratory studies due to its well-characterized physiology. P. putida KT2440 is widely recognized for its metabolic versatility and its role in bioremediation and synthetic biology. P. pseudoalcaligenes CECT 5344, on the other hand, is an alkaliphilic bacterium known for its ability to degrade cyanide under basic conditions. Together, these strains represent a broad spectrum of physiological traits and environmental adaptability, making them suitable candidates for evaluating pH variation under diverse growth conditions. This comprehensive dataset establishes a strong foundation for developing and evaluating predictive models capable of simulating pH dynamics under a variety of microbial and environmental conditions<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. In order to introduce the bacterial strains in the developed models, one-hot encoding that can transform each category into a vector was used. It is essential since many algorithms cannot directly process categorical data. For this purpose, a vector in which all elements are 0 except for one position was used. This position was set to 1 showing the presence of that type of bacterial strain.</p>
<p id="Par53">Table <a href="#Tab1" class="usa-link">1</a> provides a detailed overview of the experimental systems analyzed in this study, summarizing their key characteristics and experimental parameters for modeling pH behavior. The dataset, encompassing 379 observations, covers initial pH values ranging from 6 to 9, time intervals from 0 to 68.27 h, OD600 between 0 and 1.7184, and system pH values spanning 4.59 to 9.03. This extensive dataset, capturing diverse bacterial strains, media types, and experimental conditions, provides a solid platform for building highly precise artificial intelligence models capable of accurately predicting pH variations across different microbial growth scenarios and environmental settings.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Summary of experimental data for bacterial growth in LB and M63 media, detailing the number of data points, time range, bacterial cell concentration (OD600), and system pH range across different bacterial strains and initial pH conditions.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Type of Bacteria</th>
<th align="left" colspan="1" rowspan="1">Culture Medium</th>
<th align="left" colspan="1" rowspan="1">Initial pH</th>
<th align="left" colspan="1" rowspan="1">Number of Data Points</th>
<th align="left" colspan="1" rowspan="1">Time Range (hr)</th>
<th align="left" colspan="1" rowspan="1">Bacterial Cell Concentration Range (OD600)</th>
<th align="left" colspan="1" rowspan="1">System pH Range</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli ATCC 25,922</td>
<td align="left" colspan="1" rowspan="1">LB</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="center" colspan="1" rowspan="1">0–39.11</td>
<td align="center" colspan="1" rowspan="1">0.0028–1.5078</td>
<td align="center" colspan="1" rowspan="1">6.02–8.69</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli ATCC 25,922</td>
<td align="left" colspan="1" rowspan="1">LB</td>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="center" colspan="1" rowspan="1">0–39.11</td>
<td align="center" colspan="1" rowspan="1">0.0028–1.5104</td>
<td align="center" colspan="1" rowspan="1">6.48–8.86</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli ATCC 25,922</td>
<td align="left" colspan="1" rowspan="1">LB</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="center" colspan="1" rowspan="1">0–39.11</td>
<td align="center" colspan="1" rowspan="1">0.0028–1.5194</td>
<td align="center" colspan="1" rowspan="1">6.98–9.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli ATCC 25,922</td>
<td align="left" colspan="1" rowspan="1">M63</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="center" colspan="1" rowspan="1">0–46.13</td>
<td align="center" colspan="1" rowspan="1">0–1.1589</td>
<td align="center" colspan="1" rowspan="1">4.59–6.27</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli ATCC 25,922</td>
<td align="left" colspan="1" rowspan="1">M63</td>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="center" colspan="1" rowspan="1">0.05–46.14</td>
<td align="center" colspan="1" rowspan="1">0–1.3569</td>
<td align="center" colspan="1" rowspan="1">6.02–7.08</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli ATCC 25,922</td>
<td align="left" colspan="1" rowspan="1">M63</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="center" colspan="1" rowspan="1">0.05–46.20</td>
<td align="center" colspan="1" rowspan="1">0–1.4893</td>
<td align="center" colspan="1" rowspan="1">6.58–7.76</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pseudomonas putida KT2440</td>
<td align="left" colspan="1" rowspan="1">LB</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">18</td>
<td align="center" colspan="1" rowspan="1">0–42.12</td>
<td align="center" colspan="1" rowspan="1">0.0028–1.7128</td>
<td align="center" colspan="1" rowspan="1">6.25–8.75</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pseudomonas putida KT2440</td>
<td align="left" colspan="1" rowspan="1">LB</td>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="center" colspan="1" rowspan="1">18</td>
<td align="center" colspan="1" rowspan="1">0–42.12</td>
<td align="center" colspan="1" rowspan="1">0.0028–1.7156</td>
<td align="center" colspan="1" rowspan="1">6.99–8.77</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pseudomonas putida KT2440</td>
<td align="left" colspan="1" rowspan="1">LB</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">18</td>
<td align="center" colspan="1" rowspan="1">0–42.12</td>
<td align="center" colspan="1" rowspan="1">0.0028–1.7184</td>
<td align="center" colspan="1" rowspan="1">7.76–8.77</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pseudomonas putida KT2440</td>
<td align="left" colspan="1" rowspan="1">M63</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">19</td>
<td align="center" colspan="1" rowspan="1">0–48.13</td>
<td align="center" colspan="1" rowspan="1">0–1.4569</td>
<td align="center" colspan="1" rowspan="1">5.92–6.32</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pseudomonas putida KT2440</td>
<td align="left" colspan="1" rowspan="1">M63</td>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="center" colspan="1" rowspan="1">19</td>
<td align="center" colspan="1" rowspan="1">0–48.13</td>
<td align="center" colspan="1" rowspan="1">0–1.4919</td>
<td align="center" colspan="1" rowspan="1">6.80–7.12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pseudomonas putida KT2440</td>
<td align="left" colspan="1" rowspan="1">M63</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="center" colspan="1" rowspan="1">0–48.13</td>
<td align="center" colspan="1" rowspan="1">0–1.5141</td>
<td align="center" colspan="1" rowspan="1">7.26–7.87</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pseudomonas pseudoalcaligenes CECT 5344</td>
<td align="left" colspan="1" rowspan="1">LB</td>
<td align="left" colspan="1" rowspan="1">7.5</td>
<td align="center" colspan="1" rowspan="1">25</td>
<td align="center" colspan="1" rowspan="1">0.05–57.19</td>
<td align="center" colspan="1" rowspan="1">0–1.2817</td>
<td align="center" colspan="1" rowspan="1">7.33–8.55</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pseudomonas pseudoalcaligenes CECT 5344</td>
<td align="left" colspan="1" rowspan="1">LB</td>
<td align="left" colspan="1" rowspan="1">8.25</td>
<td align="center" colspan="1" rowspan="1">25</td>
<td align="center" colspan="1" rowspan="1">0–57.08</td>
<td align="center" colspan="1" rowspan="1">0–1.2898</td>
<td align="center" colspan="1" rowspan="1">7.66–8.64</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pseudomonas pseudoalcaligenes CECT 5344</td>
<td align="left" colspan="1" rowspan="1">LB</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="center" colspan="1" rowspan="1">25</td>
<td align="center" colspan="1" rowspan="1">0–57.40</td>
<td align="center" colspan="1" rowspan="1">0–1.4235</td>
<td align="center" colspan="1" rowspan="1">7.84–8.69</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pseudomonas pseudoalcaligenes CECT 5344</td>
<td align="left" colspan="1" rowspan="1">M63</td>
<td align="left" colspan="1" rowspan="1">7.5</td>
<td align="center" colspan="1" rowspan="1">24</td>
<td align="center" colspan="1" rowspan="1">0.07–68.20</td>
<td align="center" colspan="1" rowspan="1">0–1.5816</td>
<td align="center" colspan="1" rowspan="1">7.10–7.45</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pseudomonas pseudoalcaligenes CECT 5344</td>
<td align="left" colspan="1" rowspan="1">M63</td>
<td align="left" colspan="1" rowspan="1">8.25</td>
<td align="center" colspan="1" rowspan="1">24</td>
<td align="center" colspan="1" rowspan="1">0.07–68.27</td>
<td align="center" colspan="1" rowspan="1">0–1.4925</td>
<td align="center" colspan="1" rowspan="1">7.24–7.87</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pseudomonas pseudoalcaligenes CECT 5344</td>
<td align="left" colspan="1" rowspan="1">M63</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="center" colspan="1" rowspan="1">24</td>
<td align="center" colspan="1" rowspan="1">0.07–68.26</td>
<td align="center" colspan="1" rowspan="1">0–1.4925</td>
<td align="center" colspan="1" rowspan="1">7.39–8.11</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec16"><h3 class="pmc_sec_title">Model evaluation indices</h3>
<p id="Par55">To assess and compare the predictive capabilities of the developed models, a range of essential performance metrics were calculated for each modeling technique. The models evaluated in this study encompass artificial intelligence methods, including 1D-CNN, ANN, RF, DT, EL, AdaBoost, LSSVM. These algorithms were applied to predict the pH variations in culture media influenced by bacterial growth, using input variables such as bacterial type, culture medium type, initial pH, time, and bacterial cell concentration (OD600). To measure the precision and reliability of each model across both training and testing phases, key evaluation metrics, including the coefficient of determination (R²), mean absolute percentage error (MAPE%) and root mean square error (RMSE), were determined<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. These are defined in Eqs. (<a href="#Equ10" class="usa-link">10</a>)–(<a href="#Equ13" class="usa-link">13</a>):</p>
<p id="Par56">Relative error percent:</p>
<table class="disp-formula p" id="Equ10"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/aa7a00682cdd/d33e1102.gif" loading="lazy" id="d33e1102" alt="graphic file with name d33e1102.gif"></td>
<td class="label">10</td>
</tr></table>
<p id="Par57">Mean absolute percentage error:</p>
<table class="disp-formula p" id="Equ11"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/d1c020067092/d33e1110.gif" loading="lazy" id="d33e1110" alt="graphic file with name d33e1110.gif"></td>
<td class="label">11</td>
</tr></table>
<p id="Par58">Mean square error:</p>
<table class="disp-formula p" id="Equ12"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/509677b33a9d/d33e1118.gif" loading="lazy" id="d33e1118" alt="graphic file with name d33e1118.gif"></td>
<td class="label">12</td>
</tr></table>
<p id="Par59">Determination coefficient:</p>
<table class="disp-formula p" id="Equ13"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/f5fba23444ee/d33e1126.gif" loading="lazy" id="d33e1126" alt="graphic file with name d33e1126.gif"></td>
<td class="label">13</td>
</tr></table>
<p id="Par60">In the given equations, the subscript ‘i’ represents the identifier for each specific points. The terms ‘pred’ and ‘exp’ represent estimated and experimentally observed pH values of the culture media, respectively<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. The variable ‘N’ indicates the total number of data points used in this study, comprising 379 experimental measurements of pH variations influenced by bacterial growth in culture media<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>.</p>
<p id="Par61">The models were developed using input variables including bacterial type, culture medium type, initial pH, time, and bacterial cell concentration (OD600), with the pH of the culture media as the output variable. To ensure effective model training and reliable performance evaluation, the dataset was randomly split into training and testing subsets, with 80% of the data (303 points) allocated for training and the remaining 20% (76 points) reserved for testing.</p>
<p id="Par62">To address the varying numerical scales of the input and output variables, a modified min-max normalization technique was implemented before model construction, scaling all features to a uniform range of [− 1, 1]. This preprocessing step improves model stability and convergence by reducing the influence of features with larger numerical values, thereby enhancing the predictive consistency of the artificial intelligence models. The normalization approach applied is outlined in Eq. (<a href="#Equ14" class="usa-link">14</a>):</p>
<table class="disp-formula p" id="Equ14"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/fdf55fdaafd9/d33e1152.gif" loading="lazy" id="d33e1152" alt="graphic file with name d33e1152.gif"></td>
<td class="label">14</td>
</tr></table>
<p id="Par63">In the given equation, x<sub>N</sub> signifies the normalized value, x denotes the original, unscaled data point, and X<sub>max</sub> and X<sub>min</sub> represent the maximum and minimum values within the dataset, respectively. By transforming the feature values to a standardized range of [− 1, 1], this preprocessing step ensures data consistency, thereby enhancing the accuracy and stability of the artificial intelligence models developed in this study.</p>
<p id="Par64">In the normalization formula provided, x<sub>N</sub> indicates the normalized value, x corresponds to the raw, unnormalized data, and X<sub>max</sub> and X<sub>min</sub> refer to the highest and lowest values in the dataset, respectively. This customized normalization technique was applied to the 379 experimental measurements of pH variations in culture media influenced by bacterial growth. By scaling the feature values to the [− 1, 1] range, this preprocessing method fosters uniformity across the dataset, significantly improving the precision and reliability of the artificial intelligence models constructed in this research.</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Results and analysis</h2>
<section id="Sec18"><h3 class="pmc_sec_title">Outlier detection</h3>
<p id="Par65">The leverage method is a diagnostic tool used to identify influential data points in regression analysis. It evaluates how far the values of a data point’s input variables deviate from the mean of all input values. High leverage points can exert strong influence on the model’s predictions. In combination with standardized residuals, leverage values are often used in Williams plots to detect outliers or influential observations. A data point is typically considered influential if its leverage value exceeds a specific threshold, and its standardized residual is large. The initial equation expresses the difference Di as follows (Eq. <a href="#Equ15" class="usa-link">15</a>)<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>–<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>:</p>
<table class="disp-formula p" id="Equ15"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/64b727691230/d33e1196.gif" loading="lazy" id="d33e1196" alt="graphic file with name d33e1196.gif"></td>
<td class="label">15</td>
</tr></table>
<p id="Par66">In this context, Di represents the residual for i-th point, calculated as the difference between the predicted pH value (X<sub>Pred, i</sub>) and the experimentally observed pH (X<sub>Exp, i</sub>) in the culture media affected by bacterial growth. This residual quantifies the prediction error for each data point within the dataset. The standardized residual (SD<sub>i</sub>) is defined by the subsequent Eq. (<a href="#Equ16" class="usa-link">16</a>):</p>
<table class="disp-formula p" id="Equ16"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/4e59689fa7b5/d33e1213.gif" loading="lazy" id="d33e1213" alt="graphic file with name d33e1213.gif"></td>
<td class="label">16</td>
</tr></table>
<p id="Par67">The standardized residual (SD<sub>i</sub>) is obtained by dividing the raw residual (the difference between predicted and experimental values) by the standard deviation of all residuals. This normalization allows for consistent detection of outliers by comparing how far each point deviates from the model’s general trend. Figure <a href="#Fig2" class="usa-link">2</a> illustrates the identification of outliers in the pH dataset for bacterial growth using the leverage method, presented via a Williams plot. The denominator normalizes the residual by the standard deviation of residuals, adjusted by the leverage hi, which measures the impact of every data point on the model’s alignment. The leverage threshold h<sup>∗</sup> is calculated as h<sup>∗</sup>=3(m + 1)/N, where m is the number of input features (5 in this study: bacterial type, culture medium type, initial pH, time, and bacterial cell concentration) and N is the dataset size. For this dataset, h<sup>∗</sup>=3(5 + 1)/379 = 0.0475. Data points with h<sub>i</sub>&gt; h<sup>∗</sup> are deemed high-leverage points, suggesting potential influence, while standardized residuals exceeding typical thresholds (commonly ∣SD<sub>i</sub>∣&gt;3) are flagged as outliers. This analysis confirms the dataset’s suitability for pH prediction modeling and provides valuable insights into the dynamics of bacterial growth and pH interactions in culture media.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368003_41598_2025_16150_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/97f1850a55c3/41598_2025_16150_Fig2_HTML.jpg" loading="lazy" id="d33e1247" height="459" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Identification of outliers using the Leverage approach.</p></figcaption></figure></section><section id="Sec19"><h3 class="pmc_sec_title">Hyperparameters optimization and models evaluation</h3>
<p id="Par69">To optimize the performance of each machine learning model, key hyperparameters were tuned using trial-and-error methods guided by performance metrics such as R² and MSE. While detailed optimization procedures were initially illustrated through individual figures for each model, these plots and their corresponding analyses have been moved to the Supporting Material (Figures <a href="#MOESM1" class="usa-link">S1</a>–S4) to enhance clarity and reduce visual overload in the main text. Hyperparameters for all models were meticulously optimized using CSA algorithm to ensure peak performance and avoid overfitting. A summary of the optimal hyperparameter configurations for each model is presented in Table <a href="#Tab2" class="usa-link">2</a>, which was found to be sufficient for conveying the necessary results without excessive redundancy. This adjustment was made in accordance with reviewer recommendations, ensuring that the main body of the manuscript remains concise and reader-friendly.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Optimal hyperparameter values for each machine learning model.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Model</th>
<th align="left" colspan="1" rowspan="1">Key Hyperparameters</th>
<th align="left" colspan="1" rowspan="1">Optimal Values</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="2" colspan="1">
<strong>AdaBoost</strong>
</td>
<td align="left" colspan="1" rowspan="1">Number of base estimators</td>
<td align="left" colspan="1" rowspan="1">15</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Learning rate</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">
<strong>RF</strong>
</td>
<td align="left" colspan="1" rowspan="1">Max depth</td>
<td align="left" colspan="1" rowspan="1">21</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Number of trees (n_estimators)</td>
<td align="left" colspan="1" rowspan="1">100 (default)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Max features</td>
<td align="left" colspan="1" rowspan="1">sqrt (default)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>DT</strong>
</td>
<td align="left" colspan="1" rowspan="1">Max depth</td>
<td align="left" colspan="1" rowspan="1">3.34</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">
<strong>ANN</strong>
</td>
<td align="left" colspan="1" rowspan="1">First hidden layer neurons</td>
<td align="left" colspan="1" rowspan="1">25</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Second hidden layer neurons</td>
<td align="left" colspan="1" rowspan="1">17</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Transfer functions</td>
<td align="left" colspan="1" rowspan="1">Hyperbolic Tangent Sigmoid (tansig-hidden layers), Linear activation function (purelin- output layer)</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">
<strong>LSSVM</strong>
</td>
<td align="left" colspan="1" rowspan="1">Kernel</td>
<td align="left" colspan="1" rowspan="1">RBF</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Regularization parameter (γ)</td>
<td align="left" colspan="1" rowspan="1">1213</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Kernel width (σ²)</td>
<td align="left" colspan="1" rowspan="1">0.77</td>
</tr>
<tr>
<td align="left" rowspan="7" colspan="1">
<strong>1D-CNN</strong>
</td>
<td align="left" colspan="1" rowspan="1">Convolutional layers</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pooling layers</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Fully connected layers</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Activation function</td>
<td align="left" colspan="1" rowspan="1">Rectified Linear Unit (ReLU)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Learning rate</td>
<td align="left" colspan="1" rowspan="1">0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Epochs</td>
<td align="left" colspan="1" rowspan="1">100 (default)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Batch size</td>
<td align="left" colspan="1" rowspan="1">16 (assumed)</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1">
<strong>EL</strong>
</td>
<td align="left" colspan="1" rowspan="1">Combined models</td>
<td align="left" colspan="1" rowspan="1">SVM, DT, KNN with C = 120, gamma = 0.02, epsilon = 0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SVM settings</td>
<td align="left" colspan="1" rowspan="1">RBF used as the Kernel function, distance = Euclidean</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KNN settings</td>
<td align="left" colspan="1" rowspan="1">k = 9, distance = Euclidean</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Combination method</td>
<td align="left" colspan="1" rowspan="1">Averaging</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par71">Table <a href="#Tab3" class="usa-link">3</a> offers a detailed overview of the performance metrics for seven artificial intelligence models, including DT, RF, EL, AdaBoost, ANN, 1D-CNN, and LSSVM, developed to predict pH variations in culture media influenced by bacterial growth. The assessment utilized three primary metrics: R², RMSE, and MAPE%, computed for the training (303 points), testing (76 points), and total datasets.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>The obtained values of evaluation indices for all created models with respect to training, testing and total segments.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Model</th>
<th align="left" colspan="3" rowspan="1">R2</th>
<th align="left" colspan="3" rowspan="1">RMSE</th>
<th align="left" colspan="3" rowspan="1">MAPE%</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Training</th>
<th align="left" colspan="1" rowspan="1">Test</th>
<th align="left" colspan="1" rowspan="1">Total</th>
<th align="left" colspan="1" rowspan="1">Training</th>
<th align="left" colspan="1" rowspan="1">Test</th>
<th align="left" colspan="1" rowspan="1">Total</th>
<th align="left" colspan="1" rowspan="1">Training</th>
<th align="left" colspan="1" rowspan="1">Test</th>
<th align="left" colspan="1" rowspan="1">Total</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">EL</td>
<td align="center" colspan="1" rowspan="1">0.9990</td>
<td align="center" colspan="1" rowspan="1">0.9960</td>
<td align="center" colspan="1" rowspan="1">0.9982</td>
<td align="center" colspan="1" rowspan="1">0.0382</td>
<td align="center" colspan="1" rowspan="1">0.0913</td>
<td align="center" colspan="1" rowspan="1">0.0533</td>
<td align="center" colspan="1" rowspan="1">0.2683</td>
<td align="center" colspan="1" rowspan="1">0.8237</td>
<td align="center" colspan="1" rowspan="1">0.3797</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AdaBoost</td>
<td align="center" colspan="1" rowspan="1">0.9982</td>
<td align="center" colspan="1" rowspan="1">0.9892</td>
<td align="center" colspan="1" rowspan="1">0.9960</td>
<td align="center" colspan="1" rowspan="1">0.0523</td>
<td align="center" colspan="1" rowspan="1">0.1438</td>
<td align="center" colspan="1" rowspan="1">0.0796</td>
<td align="center" colspan="1" rowspan="1">0.4801</td>
<td align="center" colspan="1" rowspan="1">1.0664</td>
<td align="center" colspan="1" rowspan="1">0.5977</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ANN</td>
<td align="center" colspan="1" rowspan="1">0.9986</td>
<td align="center" colspan="1" rowspan="1">0.9931</td>
<td align="center" colspan="1" rowspan="1">0.9972</td>
<td align="center" colspan="1" rowspan="1">0.0452</td>
<td align="center" colspan="1" rowspan="1">0.1163</td>
<td align="center" colspan="1" rowspan="1">0.0659</td>
<td align="center" colspan="1" rowspan="1">0.4005</td>
<td align="center" colspan="1" rowspan="1">0.9104</td>
<td align="center" colspan="1" rowspan="1">0.5027</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1D-CNN</td>
<td align="center" colspan="1" rowspan="1">0.9994</td>
<td align="center" colspan="1" rowspan="1">0.9951</td>
<td align="center" colspan="1" rowspan="1">0.9983</td>
<td align="center" colspan="1" rowspan="1">0.0298</td>
<td align="center" colspan="1" rowspan="1">0.0996</td>
<td align="center" colspan="1" rowspan="1">0.0519</td>
<td align="center" colspan="1" rowspan="1">0.2728</td>
<td align="center" colspan="1" rowspan="1">0.7602</td>
<td align="center" colspan="1" rowspan="1">0.3706</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LSSVM</td>
<td align="center" colspan="1" rowspan="1">0.9988</td>
<td align="center" colspan="1" rowspan="1">0.9961</td>
<td align="center" colspan="1" rowspan="1">0.9981</td>
<td align="center" colspan="1" rowspan="1">0.0422</td>
<td align="center" colspan="1" rowspan="1">0.0884</td>
<td align="center" colspan="1" rowspan="1">0.0547</td>
<td align="center" colspan="1" rowspan="1">0.4291</td>
<td align="center" colspan="1" rowspan="1">0.9071</td>
<td align="center" colspan="1" rowspan="1">0.5249</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DT</td>
<td align="center" colspan="1" rowspan="1">0.9970</td>
<td align="center" colspan="1" rowspan="1">0.9885</td>
<td align="center" colspan="1" rowspan="1">0.9949</td>
<td align="center" colspan="1" rowspan="1">0.0660</td>
<td align="center" colspan="1" rowspan="1">0.1483</td>
<td align="center" colspan="1" rowspan="1">0.0888</td>
<td align="center" colspan="1" rowspan="1">0.6575</td>
<td align="center" colspan="1" rowspan="1">1.2069</td>
<td align="center" colspan="1" rowspan="1">0.7677</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RF</td>
<td align="center" colspan="1" rowspan="1">0.9978</td>
<td align="center" colspan="1" rowspan="1">0.9968</td>
<td align="center" colspan="1" rowspan="1">0.9975</td>
<td align="center" colspan="1" rowspan="1">0.0575</td>
<td align="center" colspan="1" rowspan="1">0.0787</td>
<td align="center" colspan="1" rowspan="1">0.0623</td>
<td align="center" colspan="1" rowspan="1">0.5424</td>
<td align="center" colspan="1" rowspan="1">0.8332</td>
<td align="center" colspan="1" rowspan="1">0.6007</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LR</td>
<td align="center" colspan="1" rowspan="1">0.9745</td>
<td align="center" colspan="1" rowspan="1">0.9698</td>
<td align="center" colspan="1" rowspan="1">0.9732</td>
<td align="center" colspan="1" rowspan="1">0.1026</td>
<td align="center" colspan="1" rowspan="1">0.1099</td>
<td align="center" colspan="1" rowspan="1">0.1064</td>
<td align="center" colspan="1" rowspan="1">1.0135</td>
<td align="center" colspan="1" rowspan="1">1.0206</td>
<td align="center" colspan="1" rowspan="1">1.0171</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par72">The results in Table <a href="#Tab3" class="usa-link">3</a> establish a distinct performance hierarchy among the models. The 1D-CNN demonstrated the highest predictive accuracy, achieving an R² of 0.998301691 for the total dataset, with the lowest RMSE (0.051935903) and MAPE% (0.370554549), showcasing its exceptional ability to model the complex relationships among input features, including bacterial type, culture medium type, initial pH (6–9), time (0–68.27 h), and bacterial cell concentration (0–1.7184 OD600). The EL model followed closely, exhibiting strong performance with an R² of 0.998222719 and low error rates, while the ANN and LSSVM models also showed robust predictive capabilities, maintaining R² values above 0.997 and MAPE% below 0.6. Conversely, the RF and AdaBoost models displayed slightly lower accuracy compared to 1D-CNN and EL but remained within acceptable predictive ranges.</p>
<p id="Par73">Figure <a href="#Fig3" class="usa-link">3</a> enhances the insights from Table <a href="#Tab3" class="usa-link">3</a> by providing a visual comparison of model performance during the testing phase, likely through bar charts comparing R², RMSE, and MAPE% values across all models. The figure clearly highlights 1D-CNN’s superior predictive precision, with the closest alignment between predicted and experimental pH values for the 76 test points. It also reveals relatively higher error margins for models like DT and AdaBoost compared to 1D-CNN and EL, underscoring the advantages of deep learning and ensemble methods in addressing complex pH dynamics.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368003_41598_2025_16150_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/d2fd84d5adea/41598_2025_16150_Fig3_HTML.jpg" loading="lazy" id="d33e1697" height="591" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MSE, R-squared and MAPE% for all created models in this paper (testing phase).</p></figcaption></figure><p id="Par74">Collectively, Table <a href="#Tab3" class="usa-link">3</a>; Fig. <a href="#Fig3" class="usa-link">3</a> emphasize the efficacy of advanced artificial intelligence techniques, particularly 1D-CNN, in precisely predicting pH variations in culture media due to bacterial growth. These findings highlight the potential of CSA optimized computational models as reliable, efficient alternatives to conventional experimental methods, providing valuable tools for optimizing biotechnological processes where pH control is critical. Moreover, a standard ordinary least squares linear regression (LR) model was utilized using the same training and test subsets of the ML models. The performance metrics (R², RMSE, MAPE) of the LR model were then compared with those of the ML models and the results are shown in Table <a href="#Tab3" class="usa-link">3</a>; Fig. <a href="#Fig3" class="usa-link">3</a>. The results demonstrated that while the LR model achieved reasonably good performance, the ML models still outperformed it consistently, particularly in capturing nonlinear patterns and interactions between input features.</p>
<p id="Par77">Figure <a href="#Fig4" class="usa-link">4</a> further complements these findings by illustrating the distribution of absolute relative errors for all machine learning models. The histograms show that most predictions for 1D-CNN and EL models are concentrated in the lowest error bins, confirming their superior accuracy and consistency across the dataset. In contrast, models such as DT and AdaBoost exhibit broader error distributions, reflecting slightly higher prediction variability. The MLP-ANN, LSSVM, and RF models fall between these extremes, maintaining moderate error concentrations. This comparative view highlights the advantages of deep learning and ensemble approaches in minimizing prediction errors and reinforces the performance hierarchy established by the quantitative metrics in Table <a href="#Tab3" class="usa-link">3</a>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368003_41598_2025_16150_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/de11880a49ca/41598_2025_16150_Fig4_HTML.jpg" loading="lazy" id="d33e1715" height="1182" width="794" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Data frequency versus absolute error intervals for different ML models.</p></figcaption></figure><p id="Par79">Figure <a href="#Fig5" class="usa-link">5</a> present crossplots comparing predicted versus experimental pH values for culture media influenced by bacterial growth, employing seven distinct artificial intelligence models. These models include 1D-CNN, EL, AdaBoost, DT, RF, ANN, and LSSVM. The analysis is based on a 379-point experimental dataset, split into 303 data points for training and 76 for testing. These crossplots serve as vital visual tools for assessing the predictive performance of each model, with optimal performance indicated by data points tightly clustered along the 45-degree line.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368003_41598_2025_16150_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/88221bd8f4fc/41598_2025_16150_Fig5_HTML.jpg" loading="lazy" id="d33e1738" height="944" width="707" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Crossplots of estimated pH versus actual values for all machine learning models.</p></figcaption></figure><p id="Par80">The dataset encompasses three bacterial strains including Pseudomonas pseudoalcaligenes CECT 5344, Pseudomonas putida KT2440, Escherichia coli ATCC 25,922 and cultured in LB and M63 media, with initial pH values ranging from 6 to 9, time intervals from 0 to 68.27 h, OD600 from 0 to 1.7184, and system pH values spanning 4.59 to 9.03. Collectively, Fig. <a href="#Fig5" class="usa-link">5</a> provides a comprehensive visual assessment of the models’ predictive capabilities, with the 1D-CNN model exhibiting exceptional accuracy, demonstrated by the tight alignment of predicted and experimental pH values across all bacterial and media systems. EL and ANN also show robust performance, while models such as DT and AdaBoost display relatively wider scatter, suggesting slightly lower precision.</p>
<p id="Par81">These crossplots validate the reliability of the 379-point dataset and the effectiveness of the data normalization technique applied, which reduced disparities in input scales and supported stable, accurate model training. Overall, the figures underscore the 1D-CNN model’s superior capacity to capture complex, non-linear relationships between key input features and pH behavior, highlighting its potential for enhancing applications such as microbial process optimization, bioreactor design, and biotechnological advancements.</p>
<p id="Par83">Figure <a href="#Fig6" class="usa-link">6</a> illustrates the relative error percentages during the training and testing phases for seven artificial intelligence models developed to predict pH variations in culture media influenced by bacterial growth. Data points closer to the y = 0 line indicate superior predictive accuracy across the 379-point dataset (303 training points and 76 testing points). Among the models, 1D-CNN exhibits exceptional performance, with the tightest error distribution (MAPE% = 0.370554549, R² = 0.998301691), followed by EL (MAPE% = 0.37966203) and ANN (MAPE% = 0.502705344). In contrast, DT (MAPE% = 0.767710648) and AdaBoost models show wider error ranges. This visual analysis underscores 1D-CNN’s remarkable ability to precisely capture the complex effects of bacterial type, culture medium type, initial pH (6–9), time (0–68.27 h), and bacterial cell concentration (0–1.7184 OD600) on pH behavior, affirming its effectiveness for applications such as microbial process optimization, bioreactor design, and biotechnological advancements.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368003_41598_2025_16150_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/0bbb4232cc85/41598_2025_16150_Fig6_HTML.jpg" loading="lazy" id="d33e1753" height="945" width="688" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Relative error percent for training and testing segment for all the constructed models in this study.</p></figcaption></figure></section><section id="Sec20"><h3 class="pmc_sec_title">Sensitivity analysis</h3>
<p id="Par85">This section of the study explores the influence of key input variables include bacterial type, culture medium type, initial pH, time, and bacterial cell concentration (OD600) on the pH variations in culture media due to bacterial growth, while evaluating the relative significance of each factor. The importance of these input features is quantified through correlation coefficients, offering insights into their contributions to the pH prediction models developed using 379 experimental data points<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>.</p>
<p id="Par86">In this analysis, the Monte Carlo simulation approach, appreciated for its simplicity and clarity, is utilized to assess the relative impact of the input variables on the pH behavior of culture media. It is important to clarify that Monte Carlo simulation in this study is not used to directly predict pH values, but rather to assess the sensitivity of input variables within the trained machine learning models. While Monte Carlo methods can be employed in traditional simulation contexts, they require prior knowledge of system equations and are computationally intensive when applied directly to complex biological processes. In contrast, machine learning models offer a data-driven alternative that can learn from experimental data without requiring explicit mechanistic formulations, providing faster and often more accurate predictions. Therefore, the Monte Carlo method was used to explore the influence of input variables on the output of ML models, not to simulate pH itself. The ML models remain central to the prediction framework, while Monte Carlo simulation supports interpretability and sensitivity analysis. The best model, i.e. 1-D CNN model was coupled with the Monte-Carlo algorithm for determining the most sensitive factors.</p>
<p id="Par87">Monte Carlo simulation approach excels in managing uncertainties by systematically sampling a wide range of input values, allowing direct evaluation of output variability without dependence on proxy modeling. Within this framework, the model incorporates multiple input parameters, defined as Eq. (<a href="#Equ17" class="usa-link">17</a>)<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>:</p>
<table class="disp-formula p" id="Equ17"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/8813d0f020e5/d33e1774.gif" loading="lazy" id="d33e1774" alt="graphic file with name d33e1774.gif"></td>
<td class="label">17</td>
</tr></table>
<p id="Par88">The range and distribution properties of each input variable () are established and used to create a comprehensive sampling set as Eq. (<a href="#Equ18" class="usa-link">18</a>):</p>
<table class="disp-formula p" id="Equ18"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/0ca8ac4f6a2f/d33e1785.gif" loading="lazy" id="d33e1785" alt="graphic file with name d33e1785.gif"></td>
<td class="label">18</td>
</tr></table>
<p id="Par89">Here, k represents the total number of generated samples, and n denotes the number of input features considered. Various sampling techniques, such as random sampling, importance sampling, and Latin hypercube sampling (LHS), may be applied at this stage to construct the input dataset. The model is then executed for each set of sampled inputs to produce the corresponding output results as Eq. (<a href="#Equ19" class="usa-link">19</a>):</p>
<table class="disp-formula p" id="Equ19"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/63e6bd1751a1/d33e1796.gif" loading="lazy" id="d33e1796" alt="graphic file with name d33e1796.gif"></td>
<td class="label">19</td>
</tr></table>
<p id="Par90">In this context, k indicates the number of samples generated, and n represents the number of input variables. Multiple sampling strategies, including random sampling, importance sampling, and Latin hypercube sampling (LHS), can be employed during this phase. The model is applied to each set of sampled input variables to compute the corresponding output results, as defined in Eqs. (<a href="#Equ20" class="usa-link">20</a>) and (<a href="#Equ21" class="usa-link">21</a>):</p>
<table class="disp-formula p" id="Equ20"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/0bba8120abbe/d33e1810.gif" loading="lazy" id="d33e1810" alt="graphic file with name d33e1810.gif"></td>
<td class="label">20</td>
</tr></table>
<table class="disp-formula p" id="Equ21"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/71c834a82e86/d33e1816.gif" loading="lazy" id="d33e1816" alt="graphic file with name d33e1816.gif"></td>
<td class="label">21</td>
</tr></table>
<p id="Par91">Here, V and E denote the variance and expected value, respectively. Sensitivity analysis is then performed based on the input-to-output relationship described in Eq. (<a href="#Equ22" class="usa-link">22</a>). Among the various methods for visualizing this relationship, scatterplot generation is regarded as one of the most straightforward and effective techniques:</p>
<table class="disp-formula p" id="Equ22"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/a7bc43c6dbfd/d33e1827.gif" loading="lazy" id="d33e1827" alt="graphic file with name d33e1827.gif"></td>
<td class="label">22</td>
</tr></table>
<p id="Par92">Figure <a href="#Fig7" class="usa-link">7</a> presents a detailed sensitivity analysis assessing the relative influence of bacterial type, culture medium type, initial pH, time, and bacterial cell concentration on the pH behavior of culture media, employing a Monte Carlo simulation integrated with the artificial intelligence models developed in this study. The Monte Carlo method, recognized for its robustness in addressing uncertainties, systematically samples variations in the input parameters to evaluate their impact on pH predictions. The findings are displayed using a correlation matrix or comparable visualization, highlighting both the magnitude and orientation of the associations between each input variable and the resulting pH response.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368003_41598_2025_16150_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/4385dca954a2/41598_2025_16150_Fig7_HTML.jpg" loading="lazy" id="d33e1854" height="328" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Assessment of the factors affecting pH variations in culture media due to bacterial growth using the developed predictive models and Monte Carlo simulation.</p></figcaption></figure><p id="Par93">The analysis identifies bacterial cell concentration as the most influential factor, with a correlation coefficient of 4.9825, underscoring its pivotal role in driving pH variations across the studied systems. Time follows closely, with a correlation coefficient of 4.8312, indicating that the duration of bacterial growth significantly affects pH dynamics. The type of culture medium, with a correlation coefficient of 3.7482, also exerts a substantial influence, suggesting that the media composition plays a critical role in pH changes. Bacterial type and initial pH, with correlation coefficients of 2.4972 and 2.2483, respectively, demonstrate comparatively lesser but still notable impacts.</p>
<p id="Par94">Based on the analysis of 379 experimental data points, bacterial cell concentration and incubation time were identified as the most influential variables affecting pH changes in the culture media. Other features exhibited weaker correlations, suggesting that microbial metabolism and growth phase primarily drive acidification or alkalization dynamics. The use of Monte Carlo simulation enhances the reliability of this analysis by accounting for input uncertainties, providing a robust basis for understanding the underlying mechanisms. Figure <a href="#Fig7" class="usa-link">7</a> serves as a valuable resource for researchers, offering practical guidance for optimizing microbial processes where pH control is critical for biotechnological applications.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Temporal analysis of bacterial growth and pH dynamics in LB and M63 media using machine learning modeling</h3>
<p id="Par96">In the context of predictive modeling, a critical aspect is the model’s ability to accurately forecast not only individual data points but also the overall trend of the output variable. In this study, the top-performing model, 1D-CNN, was employed to predict the trend of pH variations in the growth medium during the bacterial growth process for each bacterial strain. The results demonstrate that the developed 1D-CNN model effectively captures this trend, reflecting its capability to simulate the dynamic pH behavior. Moreover, this approach serves as a form of process simulation through the application of intelligent models, providing a reliable tool for understanding and predicting microbial growth dynamics.</p>
<p id="Par97">Figure <a href="#Fig8" class="usa-link">8</a> presents a comprehensive overview of the 1D-CNN model’s predictive performance across six distinct bacterial strain–medium combinations. In each subplot (8a–8f), the red line represents the predicted pH trend, the green dots indicate experimentally measured pH values, and the blue line denotes bacterial growth over time (OD600). These experiments reflect different environmental settings and initial pH conditions, demonstrating the model’s ability to generalize across multiple growth scenarios.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368003_41598_2025_16150_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fcb4/12368003/7438558c969d/41598_2025_16150_Fig8_HTML.jpg" loading="lazy" id="d33e1897" height="758" width="669" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison between 1D-CNN-predicted and experimental pH values alongside bacterial growth trends for six different bacterial strain–medium combinations.</p></figcaption></figure><p id="Par98">Figure <a href="#Fig8" class="usa-link">8</a>a shows the results for Escherichia coli ATCC 25,922 cultivated in LB medium with an initial pH of 6. The bacterial growth exhibited a classical sigmoidal pattern, rising from 0.0028 to 1.46 OD600 over ~ 39 h. The predicted pH increased from 6.0 to 8.76, closely aligning with experimental data (6.15–8.69), confirming the model’s high accuracy in alkaline pH forecasting under nutrient-rich conditions. Figure <a href="#Fig8" class="usa-link">8</a>b corresponds to Pseudomonas putida KT2440 grown in LB medium (initial pH 6). Growth peaked at 1.62 OD600 in ~ 42 h, and the predicted pH ranged from 6.0 to 8.76. The predictions tightly followed the experimental range (6.25–8.75), demonstrating the model’s robustness in capturing pH elevation due to metabolic activity.</p>
<p id="Par99">Figure <a href="#Fig8" class="usa-link">8</a>c illustrates the growth of Pseudomonas pseudoalcaligenes CECT 5344 in LB medium with an initial pH of 7.5. Here, bacterial growth progressed steadily from 0 to 1.24 OD600 over ~ 57 h. The model accurately predicted a gradual pH increase from 7.43 to 8.55, matching experimental data (7.44–8.55), validating its effectiveness for neutral–alkaline starting conditions. Figure <a href="#Fig8" class="usa-link">8</a>d displays results for E. coli ATCC 25,922 cultured in M63 minimal medium starting at pH 7. The bacterial growth reached 1.35 OD600 after ~ 46 h. In contrast to the LB medium results, pH decreased over time, and the model captured this acidification trend, predicting a decline from 7.09 to 6.46, consistent with the measured range (7.05–6.49). This illustrates the 1D-CNN model’s capability to handle both increasing and decreasing pH dynamics. Figure <a href="#Fig8" class="usa-link">8</a>e reports the behavior of P. putida KT2440 in M63 medium with an initial pH of 8. The growth curve showed a peak at 1.42 OD600 after ~ 48 h. The model predicted a modest decline in pH from 7.82 to 7.42, accurately reproducing the experimental values (7.79–7.43), suggesting it can model moderate pH shifts under nutrient-limited conditions. Figure <a href="#Fig8" class="usa-link">8</a>f shows the dynamics for P. pseudoalcaligenes CECT 5344 in M63 medium with an initial pH of 9. Bacterial growth increased to 1.38 OD600 over ~ 68 h. The model forecasted a steady decrease in pH from 8.11 to 7.52, aligning well with the measured pH trend (8.11–7.56), reinforcing its performance in alkaline environments.</p>
<p id="Par100">Overall, the 1D-CNN model demonstrated consistent and accurate prediction across a wide spectrum of microbial growth conditions, pH trends (both rising and falling), and media compositions. These results validate the suitability of 1D-CNN for modeling pH dynamics in microbiological systems and reinforce its potential as a non-invasive, data-driven tool for predictive bioprocess monitoring and optimization.</p>
<p id="Par102">From a technological perspective, the artificial intelligence framework developed in this study, validated through sensitivity analyses and error distribution assessments, demonstrates promising capabilities for data-driven modeling of microbiological pH variations. While not claiming to establish a universal benchmark, the approach provides a practical example of how machine learning can capture key patterns in moderately complex systems using available experimental data. This work highlights potential applications in biotechnology and microbial process optimization, but further research with larger and more diverse datasets will be required to fully generalize these findings to broader and more intricate biological contexts.</p>
<p id="Par103">Moreover, the computational efficiency of this methodology supports more sustainable research practices by decreasing reliance on extensive experimental efforts, aligning with modern goals of operational efficiency and environmental responsibility in scientific and industrial settings. However, there are some limitations that should be acknowledged. Needing external validation and independent datasets, lack of interpretability for some of high-performing ML models like ANN and RF and the black box nature of the models, and high computational complexity of some models such as ANN and EL are the main limitations of the current study that need to be addressed in future works.</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par104">This research successfully developed and validated an extensive array of predictive models using advanced artificial intelligence techniques, including 1D-CNN, EL, AdaBoost, DT, RF, ANN, and LSSVM, to forecast pH variations in culture media influenced by bacterial growth. A comprehensive experimental dataset comprising 379 points was employed, incorporating critical input variables such as bacterial type, culture medium type, initial pH (6–9), time (0–68.27 h), and bacterial cell concentration (0–1.7184 OD600). The primary objective was to create precise predictive models for pH dynamics under diverse microbial conditions. Model performance was significantly improved through hyperparameter optimization using the CSA algorithm, ensuring exceptional predictive accuracy. Thorough evaluations, including correlation analysis, crossplots, and relative error assessments, identified 1D-CNN as the top-performing model, achieving an R² of 0.998301691, RMSE of 0.051935903, and MAPE% of 0.370554549, showcasing its superior ability to capture complex pH behavior. Sensitivity analysis via Monte Carlo simulations indicated that bacterial cell concentration had the greatest impact (correlation coefficient = 4.9825), followed by time (4.8312), culture medium type (3.7482), bacterial type (2.4972), and initial pH (2.2483), providing valuable insights into the key factors driving pH variations. These results underscore the significant benefits of employing artificial intelligence methods over conventional experimental approaches, offering a highly accurate, efficient, and cost-effective framework for predicting microbiological properties. This approach holds substantial potential for biotechnological applications, enabling the optimization of microbial processes and bioreactor design. Future studies could further refine model accuracy by incorporating additional biological or environmental factors, such as nutrient composition or oxygen levels, to broaden applicability across diverse microbial systems.</p></section><section id="Sec23"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368003/bin/41598_2025_16150_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (22KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12368003/bin/41598_2025_16150_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (88.9KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We acknowledge the support provided by Zarqa University.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>All authors contributed equally to this paper.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data used during this study are included in the published article and the provided supplementary file. All the codes and calculation files are also available from the corresponding author on reasonable request.</p></section><section id="notes3"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par105">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Aoi, W. &amp; Marunaka, Y. Importance of pH homeostasis in metabolic health and diseases: crucial role of membrane proton transport. <em>Biomed. Res. Int.</em><strong>2014</strong> (1), 598986 (2014).
</cite> [<a href="https://doi.org/10.1155/2014/598986" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4180894/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25302301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aoi,%20W.%20&amp;%20Marunaka,%20Y.%20Importance%20of%20pH%20homeostasis%20in%20metabolic%20health%20and%20diseases:%20crucial%20role%20of%20membrane%20proton%20transport.%20Biomed.%20Res.%20Int.2014%20(1),%20598986%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Batool, Z. et al. Regulation of physiological pH and consumption of potential food ingredients for maintaining homeostasis and metabolic function: an overview. <em>Food Reviews Int.</em><strong>39</strong> (8), 5087–5103 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Batool,%20Z.%20et%20al.%20Regulation%20of%20physiological%20pH%20and%20consumption%20of%20potential%20food%20ingredients%20for%20maintaining%20homeostasis%20and%20metabolic%20function:%20an%20overview.%20Food%20Reviews%20Int.39%20(8),%205087%E2%80%935103%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Matthew, J. B. et al. pH-dependent processes in protein. <em>Crit. Reviews Biochem.</em><strong>18</strong> (2), 91–197 (1985).</cite> [<a href="https://doi.org/10.3109/10409238509085133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3899508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matthew,%20J.%20B.%20et%20al.%20pH-dependent%20processes%20in%20protein.%20Crit.%20Reviews%20Biochem.18%20(2),%2091%E2%80%93197%20(1985)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Jin, Q. &amp; Bethke, C. M. The thermodynamics and kinetics of microbial metabolism. <em>Am. J. Sci.</em><strong>307</strong> (4), 643–677 (2007).</cite> [<a href="https://scholar.google.com/scholar_lookup?Jin,%20Q.%20&amp;%20Bethke,%20C.%20M.%20The%20thermodynamics%20and%20kinetics%20of%20microbial%20metabolism.%20Am.%20J.%20Sci.307%20(4),%20643%E2%80%93677%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Bibby, S. R. S. et al. Metabolism of the intervertebral disc: effects of low levels of oxygen, glucose, and pH on rates of energy metabolism of bovine nucleus pulposus cells. <em>Spine</em><strong>30</strong> (5), 487–496 (2005).
</cite> [<a href="https://doi.org/10.1097/01.brs.0000154619.38122.47" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15738779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bibby,%20S.%20R.%20S.%20et%20al.%20Metabolism%20of%20the%20intervertebral%20disc:%20effects%20of%20low%20levels%20of%20oxygen,%20glucose,%20and%20pH%20on%20rates%20of%20energy%20metabolism%20of%20bovine%20nucleus%20pulposus%20cells.%20Spine30%20(5),%20487%E2%80%93496%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Casey, J. R., Grinstein, S. &amp; Orlowski, J. Sensors and regulators of intracellular pH. <em>Nat. Rev. Mol. Cell Biol.</em><strong>11</strong> (1), 50–61 (2010).
</cite> [<a href="https://doi.org/10.1038/nrm2820" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19997129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Casey,%20J.%20R.,%20Grinstein,%20S.%20&amp;%20Orlowski,%20J.%20Sensors%20and%20regulators%20of%20intracellular%20pH.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.11%20(1),%2050%E2%80%9361%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Krulwich, T. A., Sachs, G. &amp; Padan, E. Molecular aspects of bacterial pH sensing and homeostasis. <em>Nat. Rev. Microbiol.</em><strong>9</strong> (5), 330–343 (2011).
</cite> [<a href="https://doi.org/10.1038/nrmicro2549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3247762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21464825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krulwich,%20T.%20A.,%20Sachs,%20G.%20&amp;%20Padan,%20E.%20Molecular%20aspects%20of%20bacterial%20pH%20sensing%20and%20homeostasis.%20Nat.%20Rev.%20Microbiol.9%20(5),%20330%E2%80%93343%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Ratzke, C. &amp; Gore, J. <em>Modifying and reacting to the environmental pH drives bacterial interactions.</em> BioRxiv, : p. 136838. (2017).</cite> [<a href="https://doi.org/10.1371/journal.pbio.2004248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5868856/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29538378/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Nogales, J. et al. <em>Expanding the computable reactome in Pseudomonas putida reveals metabolic cycles providing robustness.</em> BioRxiv, : p. 139121. (2017).</cite>
</li>
<li id="CR10">
<span class="label">10.</span><cite>Sánchez-Clemente, R. et al. <em>Study of pH Changes in Media during Bacterial Growth of Several Environmental Strains</em>. MDPI. (2018)</cite>
</li>
<li id="CR11">
<span class="label">11.</span><cite>Ratzke, C. &amp; Gore, J. Modifying and reacting to the environmental pH can drive bacterial interactions. <em>PLoS Biol.</em><strong>16</strong> (3), e2004248 (2018).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.2004248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5868856/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29538378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ratzke,%20C.%20&amp;%20Gore,%20J.%20Modifying%20and%20reacting%20to%20the%20environmental%20pH%20can%20drive%20bacterial%20interactions.%20PLoS%20Biol.16%20(3),%20e2004248%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Raja Sarobin, M. &amp; Panjanathan, R. V. and <em>Diabetic retinopathy classification using CNN and hybrid deep convolutional neural networks.</em> Symmetry, 14(9): p. 1932. (2022).</cite>
</li>
<li id="CR13">
<span class="label">13.</span><cite>Giusti, A. et al. <em>Fast Image Scanning with Deep max-pooling Convolutional Neural Networks</em>. IEEE. (2013).</cite>
</li>
<li id="CR14">
<span class="label">14.</span><cite>Yang, K. et al. Multi-criteria spare parts classification using the deep convolutional neural network method. <em>Appl. Sci.</em><strong>11</strong> (15), 7088 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yang,%20K.%20et%20al.%20Multi-criteria%20spare%20parts%20classification%20using%20the%20deep%20convolutional%20neural%20network%20method.%20Appl.%20Sci.11%20(15),%207088%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Valles, J. <em>Application of a Multilayer Perceptron Artificial Neural Network (MLP-ANN) in Hydrological Forecasting in El Salvador</em>p. 213–239 (Machine Learning and Optimization for Water Resources, 2024).</cite>
</li>
<li id="CR16">
<span class="label">16.</span><cite>Al-Mejibli, I. S., Alwan, J. K. &amp; Abd, D. H. The effect of gamma value on support vector machine performance with different kernels. <em>Int. J. Electr. Comput. Eng.</em><strong>10</strong> (5), 5497–5506 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Al-Mejibli,%20I.%20S.,%20Alwan,%20J.%20K.%20&amp;%20Abd,%20D.%20H.%20The%20effect%20of%20gamma%20value%20on%20support%20vector%20machine%20performance%20with%20different%20kernels.%20Int.%20J.%20Electr.%20Comput.%20Eng.10%20(5),%205497%E2%80%935506%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Paluang, P., Thavorntam, W. &amp; Phairuang, W. Application of multilayer perceptron artificial neural network (MLP-ANN) algorithm for PM2. 5 mass concentration Estimation during open biomass burning episodes in Thailand.<em>International Journal of Geoinformatics</em> (2024).</cite>
</li>
<li id="CR18">
<span class="label">18.</span><cite>Ighalo, J. O., Igwegbe, C. A. &amp; Adeniyi, A. G. Multi-layer perceptron artificial neural network (MLP-ANN) prediction of biomass higher heating value (HHV) using combined biomass proximate and ultimate analysis data. <em>Model. Earth Syst. Environ.</em><strong>8</strong> (3), 3177–3191 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ighalo,%20J.%20O.,%20Igwegbe,%20C.%20A.%20&amp;%20Adeniyi,%20A.%20G.%20Multi-layer%20perceptron%20artificial%20neural%20network%20(MLP-ANN)%20prediction%20of%20biomass%20higher%20heating%20value%20(HHV)%20using%20combined%20biomass%20proximate%20and%20ultimate%20analysis%20data.%20Model.%20Earth%20Syst.%20Environ.8%20(3),%203177%E2%80%933191%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Pattanayak, S. et al. Application of MLP-ANN models for estimating the higher heating value of bamboo biomass. <em>Biomass Convers. Biorefinery</em>. <strong>11</strong>, 2499–2508 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Pattanayak,%20S.%20et%20al.%20Application%20of%20MLP-ANN%20models%20for%20estimating%20the%20higher%20heating%20value%20of%20bamboo%20biomass.%20Biomass%20Convers.%20Biorefinery.%2011,%202499%E2%80%932508%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>De Ville, B. Decision trees. <em>Wiley Interdisciplinary Reviews: Comput. Stat.</em><strong>5</strong> (6), 448–455 (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?De%20Ville,%20B.%20Decision%20trees.%20Wiley%20Interdisciplinary%20Reviews:%20Comput.%20Stat.5%20(6),%20448%E2%80%93455%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Suthaharan, S. &amp; Suthaharan, S. <em>Decision tree learning.</em> Machine learning models and algorithms for big data classification: thinking with examples for effective learning, : pp. 237–269. (2016).</cite>
</li>
<li id="CR22">
<span class="label">22.</span><cite>Kingsford, C. &amp; Salzberg, S. L. What are decision trees? <em>Nat. Biotechnol.</em><strong>26</strong> (9), 1011–1013 (2008).
</cite> [<a href="https://doi.org/10.1038/nbt0908-1011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2701298/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18779814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kingsford,%20C.%20&amp;%20Salzberg,%20S.%20L.%20What%20are%20decision%20trees?%20Nat.%20Biotechnol.26%20(9),%201011%E2%80%931013%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Nowozin, S. et al. <em>Decision tree fields</em>. in <em>2011 International Conference on Computer Vision</em>. IEEE. (2011).</cite>
</li>
<li id="CR24">
<span class="label">24.</span><cite>Breiman, L. Random forests. <em>Mach. Learn.</em><strong>45</strong>, 5–32 (2001).</cite> [<a href="https://scholar.google.com/scholar_lookup?Breiman,%20L.%20Random%20forests.%20Mach.%20Learn.45,%205%E2%80%9332%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Everingham, Y. et al. Accurate prediction of sugarcane yield using a random forest algorithm. <em>Agron. Sustain. Dev.</em><strong>36</strong>, 1–9 (2016).</cite> [<a href="https://scholar.google.com/scholar_lookup?Everingham,%20Y.%20et%20al.%20Accurate%20prediction%20of%20sugarcane%20yield%20using%20a%20random%20forest%20algorithm.%20Agron.%20Sustain.%20Dev.36,%201%E2%80%939%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Hapfelmeier, A. &amp; Ulm, K. A new variable selection approach using random forests. <em>Comput. Stat. Data Anal.</em><strong>60</strong>, 50–69 (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hapfelmeier,%20A.%20&amp;%20Ulm,%20K.%20A%20new%20variable%20selection%20approach%20using%20random%20forests.%20Comput.%20Stat.%20Data%20Anal.60,%2050%E2%80%9369%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Genuer, R., Poggi, J. M. &amp; Tuleau-Malot, C. VSURF: an R package for variable selection using random forests. <em>R J.</em><strong>7</strong> (2), 19–33 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Genuer,%20R.,%20Poggi,%20J.%20M.%20&amp;%20Tuleau-Malot,%20C.%20VSURF:%20an%20R%20package%20for%20variable%20selection%20using%20random%20forests.%20R%20J.7%20(2),%2019%E2%80%9333%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Margineantu, D. D. &amp; Dietterich, T. G. <em>Pruning Adaptive Boosting. In ICML</em> (Citeseer, 1997).</cite>
</li>
<li id="CR29">
<span class="label">29.</span><cite>Zheng, Z. &amp; Yang, Y. Adaptive boosting for domain adaptation: toward robust predictions in scene segmentation. <em>IEEE Trans. Image Process.</em><strong>31</strong>, 5371–5382 (2022).
</cite> [<a href="https://doi.org/10.1109/TIP.2022.3195642" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35939457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20Z.%20&amp;%20Yang,%20Y.%20Adaptive%20boosting%20for%20domain%20adaptation:%20toward%20robust%20predictions%20in%20scene%20segmentation.%20IEEE%20Trans.%20Image%20Process.31,%205371%E2%80%935382%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Ferreira, A. J. &amp; Figueiredo, M. A. <em>Boosting algorithms: A review of methods, theory, and applications.</em> Ensemble machine learning: Methods and applications, : pp. 35–85. (2012).</cite>
</li>
<li id="CR31">
<span class="label">31.</span><cite>Lazarevic, A. &amp; Obradovic, Z. Adaptive boosting techniques in heterogeneous and Spatial databases. <em>Intell. Data Anal.</em><strong>5</strong> (4), 285–308 (2001).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lazarevic,%20A.%20&amp;%20Obradovic,%20Z.%20Adaptive%20boosting%20techniques%20in%20heterogeneous%20and%20Spatial%20databases.%20Intell.%20Data%20Anal.5%20(4),%20285%E2%80%93308%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Dong, X. et al. A survey on ensemble learning. <em>Front. Comput. Sci.</em><strong>14</strong>, 241–258 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Dong,%20X.%20et%20al.%20A%20survey%20on%20ensemble%20learning.%20Front.%20Comput.%20Sci.14,%20241%E2%80%93258%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>DietterichT.G. Ensemble learning. <em>Handb. Brain Theory Neural Networks</em>. <strong>2</strong> (1), 110–125 (2002).</cite> [<a href="https://scholar.google.com/scholar_lookup?DietterichT.G.%20Ensemble%20learning.%20Handb.%20Brain%20Theory%20Neural%20Networks.%202%20(1),%20110%E2%80%93125%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Bemani, A. et al. Estimation of adsorption capacity of CO2, CH4, and their binary mixtures in Quidam shale using LSSVM: application in CO2 enhanced shale gas recovery and CO2 storage. <em>J. Nat. Gas Sci. Eng.</em><strong>76</strong>, 103204 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bemani,%20A.%20et%20al.%20Estimation%20of%20adsorption%20capacity%20of%20CO2,%20CH4,%20and%20their%20binary%20mixtures%20in%20Quidam%20shale%20using%20LSSVM:%20application%20in%20CO2%20enhanced%20shale%20gas%20recovery%20and%20CO2%20storage.%20J.%20Nat.%20Gas%20Sci.%20Eng.76,%20103204%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Songolzadeh, R., Shahbazi, K. &amp; Madani, M. Modeling n-alkane solubility in supercritical CO 2 via intelligent methods. <em>J. Petroleum Explor. Prod.</em><strong>11</strong>, 279–287 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Songolzadeh,%20R.,%20Shahbazi,%20K.%20&amp;%20Madani,%20M.%20Modeling%20n-alkane%20solubility%20in%20supercritical%20CO%202%20via%20intelligent%20methods.%20J.%20Petroleum%20Explor.%20Prod.11,%20279%E2%80%93287%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Gonçalves-e-Silva, K. &amp; Aloise, D. Xavier-de-Souza, <em>Parallel synchronous and asynchronous coupled simulated annealing</em>. <em>J. Supercomputing</em>. <strong>74</strong>, 2841–2869 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gon%C3%A7alves-e-Silva,%20K.%20&amp;%20Aloise,%20D.%20Xavier-de-Souza,%20Parallel%20synchronous%20and%20asynchronous%20coupled%20simulated%20annealing.%20J.%20Supercomputing.%2074,%202841%E2%80%932869%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Yang, S. et al. A coupled simulated annealing and particle swarm optimization reliability-based design optimization strategy under hybrid uncertainties. <em>Mathematics</em><strong>11</strong> (23), 4790 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yang,%20S.%20et%20al.%20A%20coupled%20simulated%20annealing%20and%20particle%20swarm%20optimization%20reliability-based%20design%20optimization%20strategy%20under%20hybrid%20uncertainties.%20Mathematics11%20(23),%204790%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Xavier-de-Souza, S. et al. Coupled simulated annealing. <em>IEEE Trans. Syst. Man. Cybernetics Part. B (Cybernetics)</em>. <strong>40</strong> (2), 320–335 (2009).</cite> [<a href="https://doi.org/10.1109/TSMCB.2009.2020435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19651558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xavier-de-Souza,%20S.%20et%20al.%20Coupled%20simulated%20annealing.%20IEEE%20Trans.%20Syst.%20Man.%20Cybernetics%20Part.%20B%20(Cybernetics).%2040%20(2),%20320%E2%80%93335%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Suykens, J. A. K., Yalçin, M. E. &amp; Vandewalle, J. <em>Coupled Chaotic Simulated Annealing Processes</em>. IEEE. (2003).</cite>
</li>
<li id="CR40">
<span class="label">40.</span><cite>Bemani, A., Madani, M. &amp; Kazemi, A. Machine learning-based Estimation of nano-lubricants viscosity in different operating conditions. <em>Fuel</em><strong>352</strong>, 129102 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bemani,%20A.,%20Madani,%20M.%20&amp;%20Kazemi,%20A.%20Machine%20learning-based%20Estimation%20of%20nano-lubricants%20viscosity%20in%20different%20operating%20conditions.%20Fuel352,%20129102%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Madani, M. et al. Modeling of CO2-brine interfacial tension: application to enhanced oil recovery. <em>Pet. Sci. Technol.</em><strong>35</strong> (23), 2179–2186 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Madani,%20M.%20et%20al.%20Modeling%20of%20CO2-brine%20interfacial%20tension:%20application%20to%20enhanced%20oil%20recovery.%20Pet.%20Sci.%20Technol.35%20(23),%202179%E2%80%932186%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Daryasafar, A. et al. Connectionist approaches for solubility prediction of n-alkanes in supercritical carbon dioxide. <em>Neural Comput. Appl.</em><strong>29</strong>, 295–305 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Daryasafar,%20A.%20et%20al.%20Connectionist%20approaches%20for%20solubility%20prediction%20of%20n-alkanes%20in%20supercritical%20carbon%20dioxide.%20Neural%20Comput.%20Appl.29,%20295%E2%80%93305%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Yuan, H. et al. Microfluidic-assisted caenorhabditis elegans sorting: current status and future prospects. <em>Cyborg Bionic Syst.</em><strong>4</strong>, 0011 (2023).
</cite> [<a href="https://doi.org/10.34133/cbsystems.0011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10243201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37287459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan,%20H.%20et%20al.%20Microfluidic-assisted%20caenorhabditis%20elegans%20sorting:%20current%20status%20and%20future%20prospects.%20Cyborg%20Bionic%20Syst.4,%200011%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Zhang, Y. et al. Dual recombinase polymerase amplification system combined with lateral flow immunoassay for simultaneous detection of Staphylococcus aureus and vibrio parahaemolyticus. <em>J. Pharm. Biomed. Anal.</em><strong>255</strong>, 116621 (2025).
</cite> [<a href="https://doi.org/10.1016/j.jpba.2024.116621" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39644678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20Dual%20recombinase%20polymerase%20amplification%20system%20combined%20with%20lateral%20flow%20immunoassay%20for%20simultaneous%20detection%20of%20Staphylococcus%20aureus%20and%20vibrio%20parahaemolyticus.%20J.%20Pharm.%20Biomed.%20Anal.255,%20116621%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Izadmehr, M. et al. An exact analytical model for fluid flow through finite rock matrix block with special saturation function. <em>J. Hydrol.</em><strong>577</strong>, 123905 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Izadmehr,%20M.%20et%20al.%20An%20exact%20analytical%20model%20for%20fluid%20flow%20through%20finite%20rock%20matrix%20block%20with%20special%20saturation%20function.%20J.%20Hydrol.577,%20123905%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Bassir, S. M. &amp; Madani, M. A new model for predicting asphaltene precipitation of diluted crude oil by implementing LSSVM-CSA algorithm. <em>Pet. Sci. Technol.</em><strong>37</strong> (22), 2252–2259 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bassir,%20S.%20M.%20&amp;%20Madani,%20M.%20A%20new%20model%20for%20predicting%20asphaltene%20precipitation%20of%20diluted%20crude%20oil%20by%20implementing%20LSSVM-CSA%20algorithm.%20Pet.%20Sci.%20Technol.37%20(22),%202252%E2%80%932259%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Abbasi, P., Aghdam, S. K. &amp; Madani, M. Modeling subcritical multi-phase flow through surface chokes with new production parameters. <em>Flow Meas. Instrum.</em><strong>89</strong>, 102293 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Abbasi,%20P.,%20Aghdam,%20S.%20K.%20&amp;%20Madani,%20M.%20Modeling%20subcritical%20multi-phase%20flow%20through%20surface%20chokes%20with%20new%20production%20parameters.%20Flow%20Meas.%20Instrum.89,%20102293%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Madani, M., Moraveji, M. K. &amp; Sharifi, M. Modeling apparent viscosity of waxy crude oils doped with polymeric wax inhibitors. <em>J. Petrol. Sci. Eng.</em><strong>196</strong>, 108076 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Madani,%20M.,%20Moraveji,%20M.%20K.%20&amp;%20Sharifi,%20M.%20Modeling%20apparent%20viscosity%20of%20waxy%20crude%20oils%20doped%20with%20polymeric%20wax%20inhibitors.%20J.%20Petrol.%20Sci.%20Eng.196,%20108076%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Madani, M. &amp; Alipour, M. Gas-oil gravity drainage mechanism in fractured oil reservoirs: surrogate model development and sensitivity analysis. <em>Comput. GeoSci.</em><strong>26</strong> (5), 1323–1343 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Madani,%20M.%20&amp;%20Alipour,%20M.%20Gas-oil%20gravity%20drainage%20mechanism%20in%20fractured%20oil%20reservoirs:%20surrogate%20model%20development%20and%20sensitivity%20analysis.%20Comput.%20GeoSci.26%20(5),%201323%E2%80%931343%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368003/bin/41598_2025_16150_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (22KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368003/bin/41598_2025_16150_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (88.9KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data used during this study are included in the published article and the provided supplementary file. All the codes and calculation files are also available from the corresponding author on reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-16150-x"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_16150.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368003/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368003/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368003%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368003/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368003/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368003/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835707/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368003/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835707/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368003/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368003/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="qM44KrisbxYtnez1BZcaOffONsghDuLBub5jm8Jry10oiOfWEtgAILWkkVFGlrzI">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
