
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Small round cell sarcoma tumoroid biobank reveals CIC::DUX4 sarcoma vulnerability to MCL-1 inhibition - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152539AC8AF201630539AC006EE46F90.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371069/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Small round cell sarcoma tumoroid biobank reveals CIC::DUX4 sarcoma vulnerability to MCL-1 inhibition">
<meta name="citation_author" content="Femke C A S Ringnalda">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Oncode Institute, Utrecht, The Netherlands">
<meta name="citation_author" content="Gijs J F van Son">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Oncode Institute, Utrecht, The Netherlands">
<meta name="citation_author" content="Laurens H G Verweij">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Oncode Institute, Utrecht, The Netherlands">
<meta name="citation_author" content="Seok-Young Kim">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Oncode Institute, Utrecht, The Netherlands">
<meta name="citation_author" content="Vicky Amo-Addae">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author" content="Uta E Flucke">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Department of Pathology, Radboud University Medical Center, Nijmegen, The Netherlands">
<meta name="citation_author" content="Laura S Hiemcke–Jiwa">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Department of Pathology, University Medical Center Utrecht, Utrecht, The Netherlands">
<meta name="citation_author" content="Karin P S Langenberg">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author" content="Jos A M Bramer">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author" content="Lotte Heimans">
<meta name="citation_author_institution" content="Sarcomas and GIST Center, Netherlands Cancer Institute—Antoni van Leeuwenhoek (NKI-AVL), Amsterdam, The Netherlands">
<meta name="citation_author" content="Michiel A J van de Sande">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author" content="Winan J van Houdt">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Department of Surgical Oncology, Netherlands Cancer Institute—Antoni van Leeuwenhoek (NKI-AVL), Amsterdam, The Netherlands">
<meta name="citation_author" content="Max M van Noesel">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author" content="Hinri H D Kerstens">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author" content="Marcel Santoso">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author" content="Georg Seifert">
<meta name="citation_author_institution" content="Department of Pediatrics, Division of Oncology and Hematology, Charité—Universitätsmedizin Berlin, corporate member of Freie Universität Berlin and Humboldt-Universität zu Berlin, Berlin, Germany">
<meta name="citation_author" content="Olivier Delattre">
<meta name="citation_author_institution" content="Institut Curie Research Center, NSERM U1330, Diversity and Plasticity of Pediatric Sarcoma Lab, PSL Research University, SIREDO Oncology Center, Paris, France">
<meta name="citation_author" content="Katia Scotlandi">
<meta name="citation_author_institution" content="Laboratory of Experimental Oncology, IRCCS Istituto Ortopedico Rizzoli, Bologna, Italy">
<meta name="citation_author" content="Birgit Geoerger">
<meta name="citation_author_institution" content="Department of Pediatric and Adolescent Oncology, INSERM U1015, Gustave Roussy Cancer Campus, Université Paris-Saclay, Paris, France">
<meta name="citation_author" content="Johannes H M Merks">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Division of Imaging and Oncology, University Medical Center Utrecht, University of Utrecht, Utrecht, The Netherlands">
<meta name="citation_author" content="Jan J Molenaar">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author" content="Ruben van Boxtel">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Oncode Institute, Utrecht, The Netherlands">
<meta name="citation_author" content="Marc van de Wetering">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Oncode Institute, Utrecht, The Netherlands">
<meta name="citation_author" content="Karin Sanders">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Oncode Institute, Utrecht, The Netherlands">
<meta name="citation_author" content="Hans Clevers">
<meta name="citation_author_institution" content="Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Oncode Institute, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Hubrecht Institute, Royal Netherlands Academy of Arts and Sciences and University Medical Center, Utrecht, The Netherlands">
<meta name="citation_author_institution" content="Present Address: Pharma, Research and Early Development (pRED) of F. Hoffmann-La Roche Ltd, Basel, Switzerland">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7689">
<meta name="citation_doi" content="10.1038/s41467-025-62673-2">
<meta name="citation_pmid" content="40841360">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371069/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371069/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371069/pdf/41467_2025_Article_62673.pdf">
<meta name="description" content="Small round cell sarcomas (SRCS) are highly aggressive tumors in soft tissues and bone of mostly children and young adults. Despite being different in many aspects, including genetics, possible cell-of-origin, and pathology, patients with any of ...">
<meta name="og:title" content="Small round cell sarcoma tumoroid biobank reveals CIC::DUX4 sarcoma vulnerability to MCL-1 inhibition">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Small round cell sarcomas (SRCS) are highly aggressive tumors in soft tissues and bone of mostly children and young adults. Despite being different in many aspects, including genetics, possible cell-of-origin, and pathology, patients with any of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371069/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371069">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62673-2"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62673.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371069%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371069/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371069/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371069/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7689. doi: <a href="https://doi.org/10.1038/s41467-025-62673-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62673-2</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Small round cell sarcoma tumoroid biobank reveals CIC::DUX4 sarcoma vulnerability to MCL-1 inhibition</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ringnalda%20FCAS%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Femke C A S Ringnalda</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Femke C A S Ringnalda</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">
<sup>2</sup>Oncode Institute, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ringnalda%20FCAS%22%5BAuthor%5D" class="usa-link"><span class="name western">Femke C A S Ringnalda</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20Son%20GJF%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Gijs J F van Son</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Gijs J F van Son</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">
<sup>2</sup>Oncode Institute, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20Son%20GJF%22%5BAuthor%5D" class="usa-link"><span class="name western">Gijs J F van Son</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Verweij%20LHG%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Laurens H G Verweij</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Laurens H G Verweij</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">
<sup>2</sup>Oncode Institute, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Verweij%20LHG%22%5BAuthor%5D" class="usa-link"><span class="name western">Laurens H G Verweij</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SY%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Seok-Young Kim</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Seok-Young Kim</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">
<sup>2</sup>Oncode Institute, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SY%22%5BAuthor%5D" class="usa-link"><span class="name western">Seok-Young Kim</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Amo-Addae%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Vicky Amo-Addae</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Vicky Amo-Addae</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Amo-Addae%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Vicky Amo-Addae</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Flucke%20UE%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Uta E Flucke</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Uta E Flucke</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">
<sup>3</sup>Department of Pathology, Radboud University Medical Center, Nijmegen, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Flucke%20UE%22%5BAuthor%5D" class="usa-link"><span class="name western">Uta E Flucke</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hiemcke%E2%80%93Jiwa%20LS%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Laura S Hiemcke–Jiwa</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Laura S Hiemcke–Jiwa</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">
<sup>4</sup>Department of Pathology, University Medical Center Utrecht, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hiemcke%E2%80%93Jiwa%20LS%22%5BAuthor%5D" class="usa-link"><span class="name western">Laura S Hiemcke–Jiwa</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Langenberg%20KPS%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Karin P S Langenberg</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Karin P S Langenberg</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Langenberg%20KPS%22%5BAuthor%5D" class="usa-link"><span class="name western">Karin P S Langenberg</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bramer%20JAM%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Jos A M Bramer</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Jos A M Bramer</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bramer%20JAM%22%5BAuthor%5D" class="usa-link"><span class="name western">Jos A M Bramer</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heimans%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Lotte Heimans</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Lotte Heimans</span></h3>
<div class="p">
<sup>5</sup>Sarcomas and GIST Center, Netherlands Cancer Institute—Antoni van Leeuwenhoek (NKI-AVL), Amsterdam, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heimans%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lotte Heimans</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20de%20Sande%20MAJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Michiel A J van de Sande</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Michiel A J van de Sande</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20de%20Sande%20MAJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Michiel A J van de Sande</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20Houdt%20WJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Winan J van Houdt</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Winan J van Houdt</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">
<sup>6</sup>Department of Surgical Oncology, Netherlands Cancer Institute—Antoni van Leeuwenhoek (NKI-AVL), Amsterdam, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20Houdt%20WJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Winan J van Houdt</span></a>
</div>
</div>
<sup>1,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20Noesel%20MM%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Max M van Noesel</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Max M van Noesel</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20Noesel%20MM%22%5BAuthor%5D" class="usa-link"><span class="name western">Max M van Noesel</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kerstens%20HHD%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Hinri H D Kerstens</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Hinri H D Kerstens</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kerstens%20HHD%22%5BAuthor%5D" class="usa-link"><span class="name western">Hinri H D Kerstens</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Santoso%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Marcel Santoso</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Marcel Santoso</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Santoso%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marcel Santoso</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seifert%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Georg Seifert</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Georg Seifert</span></h3>
<div class="p">
<sup>7</sup>Department of Pediatrics, Division of Oncology and Hematology, Charité—Universitätsmedizin Berlin, corporate member of Freie Universität Berlin and Humboldt-Universität zu Berlin, Berlin, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seifert%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Georg Seifert</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Delattre%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Olivier Delattre</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Olivier Delattre</span></h3>
<div class="p">
<sup>8</sup>Institut Curie Research Center, NSERM U1330, Diversity and Plasticity of Pediatric Sarcoma Lab, PSL Research University, SIREDO Oncology Center, Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Delattre%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Olivier Delattre</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scotlandi%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Katia Scotlandi</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Katia Scotlandi</span></h3>
<div class="p">
<sup>9</sup>Laboratory of Experimental Oncology, IRCCS Istituto Ortopedico Rizzoli, Bologna, Italy </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scotlandi%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Katia Scotlandi</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Geoerger%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Birgit Geoerger</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Birgit Geoerger</span></h3>
<div class="p">
<sup>10</sup>Department of Pediatric and Adolescent Oncology, INSERM U1015, Gustave Roussy Cancer Campus, Université Paris-Saclay, Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Geoerger%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Birgit Geoerger</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Merks%20JHM%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Johannes H M Merks</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Johannes H M Merks</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">
<sup>11</sup>Division of Imaging and Oncology, University Medical Center Utrecht, University of Utrecht, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Merks%20JHM%22%5BAuthor%5D" class="usa-link"><span class="name western">Johannes H M Merks</span></a>
</div>
</div>
<sup>1,</sup><sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Molenaar%20JJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Jan J Molenaar</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Jan J Molenaar</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Molenaar%20JJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Jan J Molenaar</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20Boxtel%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Ruben van Boxtel</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Ruben van Boxtel</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">
<sup>2</sup>Oncode Institute, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20Boxtel%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ruben van Boxtel</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20de%20Wetering%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id23"><span class="name western">Marc van de Wetering</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Marc van de Wetering</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">
<sup>2</sup>Oncode Institute, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20de%20Wetering%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marc van de Wetering</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanders%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id24"><span class="name western">Karin Sanders</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Karin Sanders</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">
<sup>2</sup>Oncode Institute, Utrecht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanders%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Karin Sanders</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Clevers%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id25"><span class="name western">Hans Clevers</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Hans Clevers</span></h3>
<div class="p">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div class="p">
<sup>2</sup>Oncode Institute, Utrecht, The Netherlands </div>
<div class="p">
<sup>12</sup>Hubrecht Institute, Royal Netherlands Academy of Arts and Sciences and University Medical Center, Utrecht, The Netherlands </div>
<div class="p">
<sup>13</sup>Present Address: Pharma, Research and Early Development (pRED) of F. Hoffmann-La Roche Ltd, Basel, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Clevers%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hans Clevers</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>12,</sup><sup>13,</sup><sup>✉,</sup><sup>#</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands </div>
<div id="Aff2">
<sup>2</sup>Oncode Institute, Utrecht, The Netherlands </div>
<div id="Aff3">
<sup>3</sup>Department of Pathology, Radboud University Medical Center, Nijmegen, The Netherlands </div>
<div id="Aff4">
<sup>4</sup>Department of Pathology, University Medical Center Utrecht, Utrecht, The Netherlands </div>
<div id="Aff5">
<sup>5</sup>Sarcomas and GIST Center, Netherlands Cancer Institute—Antoni van Leeuwenhoek (NKI-AVL), Amsterdam, The Netherlands </div>
<div id="Aff6">
<sup>6</sup>Department of Surgical Oncology, Netherlands Cancer Institute—Antoni van Leeuwenhoek (NKI-AVL), Amsterdam, The Netherlands </div>
<div id="Aff7">
<sup>7</sup>Department of Pediatrics, Division of Oncology and Hematology, Charité—Universitätsmedizin Berlin, corporate member of Freie Universität Berlin and Humboldt-Universität zu Berlin, Berlin, Germany </div>
<div id="Aff8">
<sup>8</sup>Institut Curie Research Center, NSERM U1330, Diversity and Plasticity of Pediatric Sarcoma Lab, PSL Research University, SIREDO Oncology Center, Paris, France </div>
<div id="Aff9">
<sup>9</sup>Laboratory of Experimental Oncology, IRCCS Istituto Ortopedico Rizzoli, Bologna, Italy </div>
<div id="Aff10">
<sup>10</sup>Department of Pediatric and Adolescent Oncology, INSERM U1015, Gustave Roussy Cancer Campus, Université Paris-Saclay, Paris, France </div>
<div id="Aff11">
<sup>11</sup>Division of Imaging and Oncology, University Medical Center Utrecht, University of Utrecht, Utrecht, The Netherlands </div>
<div id="Aff12">
<sup>12</sup>Hubrecht Institute, Royal Netherlands Academy of Arts and Sciences and University Medical Center, Utrecht, The Netherlands </div>
<div id="Aff13">
<sup>13</sup>Present Address: Pharma, Research and Early Development (pRED) of F. Hoffmann-La Roche Ltd, Basel, Switzerland </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Sep 5; Accepted 2025 Jul 25; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371069  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841360/" class="usa-link">40841360</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Small round cell sarcomas (SRCS) are highly aggressive tumors in soft tissues and bone of mostly children and young adults. Despite being different in many aspects, including genetics, possible cell-of-origin, and pathology, patients with any of these entities all receive the same therapeutic regimen. Although several pre-clinical models of Ewing sarcoma have been established, such as cell lines and patient-derived tumor xenografts, few models exist for other SRCS. Here, we describe a pediatric SRCS tumor organoid (tumoroid) biobank containing long-term tumoroid cultures with different translocations, including EWSR1::FLI1, EWSR1::ERG, CIC::DUX4, and BCOR-rearrangements. Using histology, whole genome sequencing and RNA sequencing, we demonstrate that these tumoroids retain histological characteristics, known marker gene expression and chromosomal rearangements of their matching patient tumors. In addition, we compare mutation clusters in the tumoroids across patient-matched longitudinal samples, which shows that cellular heterogeneity is maintained. Drug screening on the tumoroid models unveils entity-specific drug sensitivity to various cytotoxic compounds and targeted compounds, including MCL-1 inhibitors for CIC::DUX4 sarcomas. Taken together, this newly established SRCS patient-derived tumoroid biobank represents a promising source of material for future basic cancer research and drug screening.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Sarcoma, Targeted therapies, Cancer models</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Patients with different small round cell sarcoma (SRCS) often receive the same treatment regimen but for some SRCS subtypes, response to chemotherapy is poor and targeted treatment options are limited. Here, the authors establish a biobank of paediatric patient-derived SRCS tumoroids and perform drug screening, identifying MCL inhibition as a potential therapeutic strategy in CIC::DUX4 sarcomas.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Small round cell sarcomas (SRCS) such as Ewing sarcoma (ES), CIC::DUX4 sarcoma (CDS) and BCOR-rearranged sarcoma (BRS) are highly aggressive tumors that mainly affect children and young adults<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. These tumors, which are composed of small round cells with a high nuclear-cytoplasmic ratio, are most often found in bone and soft tissues<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Genetically, these tumors are characterized by specific chromosomal translocations that result in the formation of a fusion gene. The hallmark translocation for ES generates a fusion of the Ewing Sarcoma Region 1 gene (<em>EWSR1</em>) with a member of the ETS family of transcription factors. The most common fusion is <em>EWSR1::FLI1</em> (85%), followed by <em>EWSR1::ERG</em> (10%). The resulting fusion proteins contain the N terminal domain of EWSR1, an RNA binding protein, and the DNA binding domain of the ETS factor. The EWSR1::FLI1 oncoprotein disrupts normal gene expression in multiple ways. It can bind to canonical ETS sites, whereby the chimeric oncoprotein suppresses expression of target genes, and it can bind to GGAA repeats which in turn are converted into de novo enhancers<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Other SRCS lack the molecular hallmark of ES, <em>EWSR1</em> fused with an ETS transcription factor. In CDS, the translocation results in a fusion between the <em>CIC</em> gene (19q13) and one of two <em>DUX4</em> retro-genes (4q35 or 10q26)<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Due to the rarity of this tumor entity, not much is known about its biology, while its clinical behavior is still poorly defined. Similar to ES, BRS also arise from bone and soft tissue but are characterized by genetic alterations that involve the <em>BCOR</em> gene, most commonly fused with <em>CCNB3</em>. Typical for these BCOR fusion-driven tumors is the expression of genes like <em>BCOR, TLE1</em> and <em>SATB2</em> and their high mitotic activity<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>.</p>
<p id="Par4">Despite being different in many aspects, including genetics, possible cell-of-origin, and pathology, patients with any of these entities receive the same therapeutic regimen. The ES treatment regimen is guided by standard protocols and includes systemic chemotherapy in addition to local therapy (surgery and/or radiotherapy)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. The prognosis for ES patients diagnosed with a primary tumor has improved in the last decades, while prognosis remains poor for patients that have refractory or relapsed disease<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. CDS are treated with the same treatment regimen as ES but show little to no response to chemotherapy. Previous studies have shown that CDS patients often progress under treatment<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Taken together, this underlines the need to develop new entity-specific therapeutic options.</p>
<p id="Par5">Although several pre-clinical models of ES have been established, such as cell lines and patient-derived tumor xenografts (PDX)<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, few models exist for other SRCS. It’s possible to use cell lines in long-term and high-throughput screening analyses, but they have several disadvantages, including clonal outgrowth and very low establishment success percentages<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. A recent study has described a screening platform for SRCS using short-term patient-derived in vitro cultures<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. For some pediatric tumor entities, including Wilms tumors, rhabdomyosarcomas, atypical teratoid/rhabdoid tumors and small cell carcinoma of the ovary, hypercalcemic type, long-term patient-derived tumor organoid (tumoroid) models have been reported. These tumoroid models allow for sustained in vitro expansion and retain key properties of the original tumors, including cellular heterogeneity<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Several studies have shown that tumoroids are valuable tools to detect entity- and patient-specific drug sensitivities with higher fidelity than regular cell lines<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Patient-derived tumoroids can also be used to dissect the transforming mechanism in detail or to study the consequences of oncogenic mutations and investigate tumor cell heterogeneity<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>.</p>
<p id="Par6">Here, we establish a pediatric tumoroid biobank of SRCSs containing different translocations, including EWSR1::FLI1, EWSR1::ERG, EWSR1::FEV, CIC::DUX4, and BCOR-rearrangements. We show that these long-term tumoroid cultures retain histological, genetic, and transcriptional features of their matching patient tumors, including cellular heterogeneity. Drug screening reveals subtype-specific responses, notably sensitivity to MCL-1 inhibition in CIC::DUX4 tumoroids. This biobank provides a robust platform for SRCS modeling and therapeutic discovery.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">A living biobank of small round cell sarcoma tumoroid models</h3>
<p id="Par7">For the establishment of SRCS tumoroid models, we obtained fresh tumor tissue from patients that underwent a diagnostic fine-needle aspiration biopsy (FNAB) or surgical resection (Fig.<a href="#Fig1" class="usa-link">1A</a>; Supplemantary Data <a href="#MOESM3" class="usa-link">1</a>). Fresh PDX tumor specimens were obtained from the Innovative Therapies for Children with Cancer (ITCC-P4) consortium (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). Since the FNAB samples were taken for diagnostic purposes, these samples are treatment-naïve, while almost all resection material was collected following chemotherapy. As a starting point, we used a basic organoid culture medium (Advanced DMEM/F12 supplemented with penicillin/streptomycin, HEPES, Glutamax, N-2, B27 without vitamin A and n-acetyl-cysteine). Using a trial and error-based approach, we empirically tested growth factors which eventually led to the development of three types of Sarcoma Culture Medium (SCM), all including EGF, FGF2 and IGF1, but in different concentrations (Fig. <a href="#Fig1" class="usa-link">1A</a>; Supplementary Data <a href="#MOESM4" class="usa-link">2</a>). We have thus been able to successfully culture and biobank a comprehensive collection of pediatric fusion-positive SRCS, including sarcomas with <em>EWSR1::FLI1, EWSR1::ERG, EWSR1::FEV, BCOR::CCNB3, KMT2D::BCOR</em> and <em>CIC::DUX4</em> gene fusions (Fig. <a href="#Fig1" class="usa-link">1B</a>). In 33 out of 37 (89%) samples, tumoroids were successfully established (Fig. <a href="#Fig1" class="usa-link">1B</a>, Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). The derivation efficiency was higher for treatment-naïve (10 out of 10, 100%) than for relapse and chemotherapy-treated patient samples (16 out of 20, 80%). Especially the ES chemotherapy-treated samples mainly consisted of necrotic tissue, which could explain the lower derivation efficiency. For PDX-derived tumoroids, we successfully established tumoroid lines in 7 out of 7 (100%) samples. Different media conditions were tested for all new samples, the media conditions that resulted in tumoroid growth per sample are shown in Supplementary Data <a href="#MOESM4" class="usa-link">2</a>. Depending on the amount of tumor tissue that was put in culture, a stable culture (defined as growth for at least 5 passages) was established within 5–9 weeks after receiving the tumor material. Upon establishment of a stable culture, cells were expanded for drug screen experiments, which were performed between passage 5 and 10. We collected DNA and RNA for further downstream analysis at the same timepoint. In general, tumoroid clusters appeared within 7–14 days after seeding on low attachment plates. The tumoroids could be split in a ratio between 1:2 and 1:6 every 8–14 days and could be passaged at least 20 times. Tumoroids were viably frozen down for long-term storage. We did not observe a clear difference in growth after a freeze-thaw cycle. Depending on the type of tumor, there was a modest difference in expansion rates between tumor entities. In particular, the tumoroids established from EWSR1::FLI1 tumors tended to expand somewhat faster (Fig. <a href="#Fig1" class="usa-link">1C</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">1A</a>). We noticed a difference in tumoroid morphology between the different fusion-driven entities. Amongst the ES tumoroids, EWSR1::ERG and EWSR1::FEV tumoroids contained more compact cell structures, indicated by a darker appearance in the center of the tumoroid. They also displayed a less uniform shape compared to the EWSR1::FLI1 tumoroids (Fig. <a href="#Fig1" class="usa-link">1C</a>). All ES tumoroids were quite loose in structure and easy to dissociate. The BCOR::CCNB3 cultures showed a mix of small to intermediate-sized tumoroids, which were exceptionally stiff and perfectly round, with some less dense pockets in the center of the tumoroid. H&amp;E staining showed that the core of the BRS tumoroids contained fewer cells compared to other fusion-positive sarcomas (Supplementary Fig. <a href="#MOESM1" class="usa-link">2B</a>). In contrast, CDS tumoroids were more difficult to dissociate and typically appeared as grape-like clumps of cells (Fig. <a href="#Fig1" class="usa-link">1C</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Biobank establishment of small round cell sarcoma cancer tumoroids.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371069_41467_2025_62673_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/39c1/12371069/642898070d3d/41467_2025_62673_Fig1_HTML.jpg" loading="lazy" id="d33e666" height="781" width="671" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Flow diagram with an overview of the study design. Tumoroids were established from patient tumor and PDX material and subsequently characterized using whole genome sequencing, RNA sequencing and histology. Drug screens were performed on a subset of ES, CDS and BRS. Lastly, primary tumors and matching tumoroid lines were viably frozen and stored in liquid nitrogen. Created in BioRender. Ringnalda, F. (2025) <a href="https://BioRender.com/ovfd63d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/ovfd63d</a>. <strong>B</strong> Donut chart (left) representing the different fusions and the number of patients per fusion that were included in the ES, CDS and BRS biobank. Bar chart (right) of the total number of established tumoroid lines from each fusion, subdivided in tumoroids derived from patient tumors and those derived from PDX material. <strong>C</strong> Representative brightfield microscopy images of EWSR1::FLI1, EWSR1::ERG, EWSR1::FEV, BCOR::CCNB3, KMT2D::BCOR and CIC::DUX4 tumoroids (<em>n</em> = 3, frames of individual area of interest). Scale bar: 200 µm, zoom-in 100 µm. ES Ewing sarcoma, CDS CIC::DUX4 sarcoma, BRS BCOR-rearranged sarcoma. Source data are provided as a <a href="#MOESM10" class="usa-link">Source Data</a> file.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Phenotypic characterization of small round cell sarcoma tumoroid models</h3>
<p id="Par8">Classification of bone and soft tissue sarcomas is based on pathological examination. To determine whether the SRCS tumoroids retained their phenotypic features in vitro, we first characterized the tumoroids models by histological analysis<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. This showed that the different tumoroid models recapitulated their parental tumor tissue hallmarks (Fig. <a href="#Fig2" class="usa-link">2</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">2A</a>). ES were characterized by strong CD99 staining and this was reflected in EWSR1::FLI1 and EWSR1::ERG tumoroids (Fig. <a href="#Fig2" class="usa-link">2B</a>)<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. EWSR1::FEV tumoroids showed reduced CD99 staining compared to their matching tumor tissue. A hallmark of CDS is the strong nuclear expression of Wilms tumor 1 (WT1) (Fig. <a href="#Fig2" class="usa-link">2C</a>) and ETS variant transcription factor 4 (ETV4) (Supplementary Fig. <a href="#MOESM1" class="usa-link">2A</a>). Indeed, our CIC::DUX4 tumoroids strongly expressed these proteins. While these protein markers are adapted as helpful tools to distinguish CIC-rearranged sarcoma from ES and other histologic mimics, diagnosing fusion-positive sarcomas relies on the detection of the oncogenic fusion transcript, either by fluorescence in situ hybridization (FISH) or reverse-transcription PCR (RT-PCR)<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Before submitting tumoroids for whole genome sequencing (WGS), the presence of the pertinent oncogenic fusion transcript -found in the original tumors- was confirmed by RT-PCR in all established tumoroid lines, primers are listed in Supplementary Data <a href="#MOESM5" class="usa-link">3</a>. An overview of genetic breakpoints is shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">2C</a>.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Histological characterization of small round cell sarcoma tumoroids.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371069_41467_2025_62673_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/39c1/12371069/10e70598d5b2/41467_2025_62673_Fig2_HTML.jpg" loading="lazy" id="d33e736" height="698" width="683" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Representative images of H&amp;E staining of tumor tissue (top) and matching tumoroids (bottom) derived from the indicated fusion-positive tumor types (<em>n</em> = 3, individual tumoroids). Scale bars: 100 µm, zoom-in 50 µm. <strong>B</strong> Representative images of CD99 staining of tumor tissue (top) and matching tumoroids (bottom) (<em>n</em> = 3, individual tumoroids). Scale bars: 100 µm, zoom-in 50 µm. <strong>C</strong> Representative images of WT1 staining of tumor tissue (top) and matching tumoroids (bottom) (<em>n</em> = 3, individual tumoroids). Scale bars: 100 µm, zoom-in 50 µm.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Genetic characterization of small round cell sarcoma tumoroid models</h3>
<p id="Par9">To determine whether SRCS tumoroids genetically recapitulate the tumors from which they were derived, we performed WGS on 21 tumoroid samples and compared these, when available, to WGS of their matching tumor tissue. We created mutation matrices and used these to calculate Jaccard’s distance between the samples. This revealed that the tumoroids genetically resemble their tumor counterparts (Fig. <a href="#Fig3" class="usa-link">3B</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">3B, C</a>). Some TT-TO paired samples were sequenced with different techniques, i.e., WXS versus WGS. These techniques cover different parts of the genome and share fewer variants, which explains why some pairs show relatively large Jaccard’s distances (e.g., ES-042 and ES-071). Also, several ES tumoroids showed CNAs on chromosomes 1, 8, 9 and <em>12</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3A, B</a>), which is consistent with previous data<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>–<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Furthermore, mutations frequently found in ES were detected, such as <em>STAG2</em> and <em>TP53</em> (Fig. <a href="#Fig3" class="usa-link">3A</a>) and these genetic characteristics were conserved in matching tumoroid models. In some samples (e.g., ES-027) additional mutations were detected in the tumoroid samples compared to their matching tumor sample. This could be an indication of de novo mutations or clonal selection. In a few cases, no recurrent somatic mutations other than the fusion gene could be detected. In CDS tumor and tumoroid samples, no additional oncogenic somatic mutations were found, except for an amplification of <em>MYC</em> in a relapse sample (CDS-045-TT)<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. In the tumoroid corresponding to this relapse sample (CDS-045-TO), this <em>MYC</em> amplification was not detected. Moreover, we detected <em>CDKN2A</em> homozygous deletions in various ES samples and one BCOR-rearranged sample, which have been shown to be associated with poor prognosis<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Genetic characterization of small round cell sarcoma tumoroid models.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371069_41467_2025_62673_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/39c1/12371069/cc7e284dc63e/41467_2025_62673_Fig3_HTML.jpg" loading="lazy" id="d33e820" height="1014" width="689" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Oncoplot depicting somatic mutations, copy number alterations (CNAs) and gene fusions identified in ES, CDS and BRS tumors and matching tumoroids. For samples in which the germline sample was not available (*), mutations in known tumor driver genes are indicated. For samples for which no sequencing data was available, the mutational information provided by the ITCC-P4 consortium is indicated (+). <strong>B</strong> Circos plots of ES-010 tumor and tumoroid. The circos plots show, from outside inwards: chromosomes, single-nucleotide variants (SNVs) (dark blue), copy number alterations (CNAs) as gained (red) or lost (light blue) regions and translocations (red line). <strong>C</strong> Circos plots of tumoroids derived from one EWSR1::FLI1 patient (top row; diagnostic [ES-016], first relapse [ES-055] and second relapse [ES-080] samples) and one CIC::DUX4 patient (bottom row; diagnostic [CDS-090], resection [CDS-032] and relapse [CDS-045] samples) at different timepoints during disease progression. The circos plots show, from outside inwards: chromosomes, single-nucleotide variants (SNVs) (dark blue), copy number alterations (CNAs) as gained (red) or lost (light blue) regions and translocations (red line). <strong>D</strong> River plot depicting differential clusters of mutations between time of diagnosis (ES-016), first relapse (ES-055) and second relapse (ES-080) tumor sample from one EWSR1::FLI1 patient. <strong>E</strong> The mean variant allele frequency of individual clusters of mutations in tumor and matching tumoroid samples in diagnostic (ES-016), first relapse (ES-055) and second relapse (ES-080) samples from one EWSR1::FLI1 patient. <strong>F</strong> Contribution of somatic mutational signatures per tumor and matching tumoroid model. SBS single base substitution, TT Tumor Tissue, TO TumorOid, ES Ewing sarcoma, CDS CIC::DUX4 sarcoma, BRS BCOR-rearranged sarcoma. Source data are provided as a <a href="#MOESM10" class="usa-link">Source Data</a> file.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Genetic landscape during tumor evolution</h3>
<p id="Par10">Tumor progression is generally accompanied by an increase in mutational burden<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. For tumoroids to be useful models to study disease progression, it is essential that they not only reflect the mutational landscape of primary tumors, but also of relapses and metastases. From 10 patients, we established tumoroid models at different time points during disease progression, namely primary tumor (diagnosis) and/or first and second relapse. We detected an increase in mutational load and CNAs, both in the relapsed tumors and in the tumoroids that were derived from them. However, this increase in mutational burden was different for ES, CDS and BRS tumors (Fig. <a href="#Fig3" class="usa-link">3A</a>). As shown in Fig. <a href="#Fig3" class="usa-link">3C</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">3A</a>, there was a striking increase in mutational burden and CNAs in tumoroids derived from patient M040AAB. The sample at diagnosis (ES-016) showed a few CNAs, gains on 12q, 14p and 15p and local losses on 1 and 4. The samples derived from first relapse and second relapse of the same patient showed a dramatic increase of CNAs (ES-055 and ES-080 respectively) (Fig. <a href="#Fig3" class="usa-link">3C</a>). Especially ES-055 shows CNAs on nearly all major chromosomes (1, 2, 4, 5, 6, 7, 8, 9, 11, 12, 14, 15, 16, 17, 18, 19 and 21). In one CDS patient (M616AAA), we obtained tumor samples at diagnosis, resection, and relapse, but the increase in CNAs and mutational burden in samples from later timepoints was rather modest (Fig. <a href="#Fig3" class="usa-link">3C</a>), in line with the more stable genome of CDS. We observed similar CNAs in tumors and tumoroids in both ES and CDS samples (Supplementary Fig. <a href="#MOESM1" class="usa-link">3D</a>). The genetic profiles of tumor progression were therefore reflected in our in vitro models.</p>
<p id="Par11">Next, we more deeply analyzed tumor tissue and matching tumoroid lines from the 3 different time points (ES-016, ES-055 and ES-080) of one ES patient to better understand the mutational composition at different timepoints. We clustered all mutations found over all samples based on similarity of variant allele frequency and allele-specific copy numbers. This resulted in 16 different clusters of mutations (Fig. <a href="#Fig3" class="usa-link">3D, E</a>). We noticed that not all clusters are present in all samples, indicating clonal evolution. In addition, we noticed that different groups of mutations could be either timepoint-specific or sample-specific. For instance, cluster 10 is present in all samples except for the first tumor sample (ES-016-TT). Possibly, in vitro, we selected for a clone already harboring this group of mutations, much like it seems to have happened in vivo. We also noticed clusters of mutations that only appeared in a single tumoroid sample, like cluster 12, 13, and 2. These mutations are only present in tumoroid samples and are likely a result of clonal selection or de novo mutations. In the second relapse sample (ES-080), an additional <em>STAG2</em> mutation could be identified (Fig. <a href="#Fig3" class="usa-link">3A</a>). We detected a <em>TP53</em> mutation in the first relapse sample (ES-055, tumor and tumoroid), but not in the second relapse sample (ES-080, tumor and tumoroids) (Fig. <a href="#Fig3" class="usa-link">3A</a>). This already indicates that there is variation in tumor development and clonal selection in vivo, as also indicated by the different clusters of mutations that are present in the tumor samples (Fig. <a href="#Fig3" class="usa-link">3D</a>). Subsequently, we compared the mutational distribution of the tumoroids to their matching tumor samples. Although not identical, most clusters of mutations that were present in the tumors, were also present in the matching tumoroids, and vice versa (Fig. <a href="#Fig3" class="usa-link">3E</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">3E</a>), with some exceptions that could be explained by either de novo mutations or clonal selection.</p>
<p id="Par12">To further confirm that tumoroids maintained the genetic characteristics of their matching tumors, we extracted mutational signatures from WGS/WXS data of the tumoroids and matching tumors (Fig. <a href="#Fig3" class="usa-link">3F</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">3F</a>). We compared these signatures to signatures reported in the Catalog of Somatic Mutations in Cancer (COSMIC) database<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. A substantial number of signatures were found in our ES, CDS and BRS tumors, which were maintained in the tumoroids (Fig. <a href="#Fig3" class="usa-link">3F</a>). ES samples harboring mutations in <em>CDKN2A, STAG2</em> or <em>TP53</em> showed a relatively high contribution of signatures SBS40 (etiology unknown), SBS6 (DNA mismatch repair), SBS12 (etiology unknown) and SBS3 (DNA-DSB repair by HR). ES samples that were treated with chemotherapy, mainly relapse samples, showed signatures that highly correlate to SBS18 (oxidative stress) and SBS25 (exposure to chemotherapy). Other signatures included SBS5 (cellular aging) and SBS12 (etiology unknown). The CDS tumors and tumoroids included signatures SBS6, SBS5 and SBS12, which stayed the same during disease progression and treatment. Overall, SRCS tumoroids displayed and maintained key features of the genomic landscape, such as cancer gene mutations, CNAs, and mutational signatures.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Gene expression profiles in small round cell sarcoma tumoroid models</h3>
<p id="Par13">To determine whether the tumoroids transcriptionally resembled the different SRCS entities, we performed paired-end RNA-sequencing on all samples. Firstly, STAR-Fusion confirmed the presence of the relevant fusion transcripts in all tumors and tumoroids<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">2C</a>). A principal component analysis (PCA) based on the expression of the top 5000 most variable genes showed that all tumors and tumoroids within one tumor entity (ES versus BRS versus CDS) clustered closely together in the first principal component. The second principal component separated samples mostly based on sample type (tumor versus PDX versus tumoroid). PDXs and tumoroids clustered more closely together than tumors and tumoroids (Fig. <a href="#Fig4" class="usa-link">4A</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4B</a>). Known marker genes for the specific tumor entities were indeed highly expressed in the relevant tumors and PDXs, which was reflected in their corresponding tumoroids (Fig. <a href="#Fig4" class="usa-link">4B</a>). Samples harboring <em>EWSR1::FLI1</em>, <em>EWSR1::ERG</em> and <em>EWSR1::FEV</em> fusions highly expressed <em>CD99</em>, a marker used to identify these tumors in a clinical setting. They also expressed other ES-specific markers such as <em>NKX2.2</em> and <em>PAX7</em>. Analysis of BRS samples revealed that <em>CCNB3</em> was highly expressed only in <em>BCOR::CCNB3</em> samples, while <em>NTRK3</em> was highly expressed only in <em>KMT2D::BCOR</em> samples. All BRS samples co-expressed <em>TLE1</em> and <em>SATB2</em>, a combination that is used to diagnose BRS in the clinic. Interestingly, while CD99 is used to diagnose CDS in the clinic, little to no mRNA expression of <em>CD99</em> could be observed in either tumor or tumoroid samples. Other markers such as <em>ETV1/4</em>, <em>WT1</em> and <em>DUX4</em> were highly expressed in both tumors and matching tumoroids<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>–<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Taken together, this confirmed the expression of most known marker genes in SRCS and showed conserved expression in their matching tumoroids.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Transcriptomic profiling of small round cell sarcoma tumoroid models.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371069_41467_2025_62673_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/39c1/12371069/1fe2448d1114/41467_2025_62673_Fig4_HTML.jpg" loading="lazy" id="d33e1003" height="827" width="682" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Principal component analysis (PCA) of top 5000 most variable genes, dots are colored based on tumor entity and shaped for tumoroid, tissue or PDX tissue. <strong>B</strong> Heatmap showing marker gene expression of the different tumor entities in tumor and tumoroid samples. <strong>C</strong> Heatmap showing the expression of the top 20 differentially expressed genes per tumor entity that have a CHIPseq binding peak of their characteristic (fusion-)protein in close proximity to the transcription start site. <strong>D</strong> Dot plot showing row-normalized mean expression values of genes that are targeted by our in-house drug library in the different tumor entities. Source data are provided as a <a href="#MOESM10" class="usa-link">Source Data</a> file.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Identification of fusion protein-driven transcription</h3>
<p id="Par14">Next, we aimed to determine whether genes that are specifically expressed in each tumor entity and can possibly be attributed to the activity of the fusion oncoprotein, are also expressed in the matching tumoroids. In addition to tumoroid characterization, these genes might contain interesting entity-specific drug targets. To achieve this, gene expression levels of SRCS tumors and tumoroids were compared yielding a gene set specific to each tumor entity. Because the fusion gene can control gene expression in SRCS by binding to DNA elements in close proximity to target genes, CHIP-seq datasets were explored to identify which differentially expressed genes (DEGs) are likely targeted by the fusion protein<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Fusion protein-specific binding sites were extracted from datasets generated by Rivera et al.<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup> for the EWSR1::FLI1 fusion protein, by J. Taipale et al.<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> for the EWSR1::ERG fusion protein, and by R.A. Okimoto et al.<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> for the CIC::DUX4 fusion protein. For BCOR-rearranged fusion proteins, no fusion-specific dataset was available. Therefore, the locations of binding sites for the native BCOR protein were used as an approximation for binding sites of BCOR-rearranged fusion proteins (ENCODE, identifier: ENCFF186JKG<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>). Taken together, this allowed us to study genes that were differentially expressed when comparing SRCS with different fusions and showed a binding peak of the fusion gene in close proximity to the transcription start site (Fig. <a href="#Fig4" class="usa-link">4C</a>). Most identified genes were known targets of the different tumors, such as <em>NKX2.2</em> and <em>ETS1</em> for <em>EWSR1::FLI1</em> and <em>CIC::DUX4</em>, respectively. For example, <em>MAPT</em> was expressed in both EWSR1::FLI1 and EWSR1::ERG sarcomas, but not in the others<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. This gene is a known target for Docetaxel and Paclitaxel, two anticancer drugs<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. <em>ETV5</em>, which is downstream of the target of Trametinib<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>, could be identified as a DEG that was regulated specifically by the CIC::DUX4 fusion protein. Lastly, we aimed to identify tumor-specific DEGs that could be used as drug targets. Because of low sample numbers for EWSR1::FEV and EWSR1::ERG, we chose to group these. Doing so, we plotted the expression of a selection of known drug targets that are covered by our in-house drug library (Fig. <a href="#Fig4" class="usa-link">4D</a>; Supplementary Data <a href="#MOESM6" class="usa-link">4</a>).</p>
<p id="Par15">To determine which signaling pathways were active in the different entities, we performed a gene set enrichment analysis on the tumors and tumoroids. Notably, the comparison between different tumor entities demonstrated enrichment of a variety of pathways. This further supported the notion that the different translocations lead to activation or suppression of specific pathways. For example, in CDS, we verified that mitogen activated protein kinase (MAPK) phosphatase activity was increased, which has also been previously reported by Lin et al.<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">4A</a>). In BRS samples, we identified increased SH3 domain binding (Supplementary Fig. <a href="#MOESM1" class="usa-link">4A</a>), which plays a role in tumor progression<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>.</p>
<p id="Par16">In summary, the transcriptomic analyses confirmed that ES, CDS and BRS tumoroids retain key traits of their original tumors, and we identified pathways that might be targeted therapeutically in ES, CDS and BRS.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Drug screening of ES, CDS and BRS tumoroids reveals entity-specific sensitivities</h3>
<p id="Par17">To assess how our tumoroid models respond to drugs that are used in current treatment regimens, as well as to identify potential novel treatment options, we performed drug screens. In total, we screened twenty-one tumoroid lines derived from fourteen patients, including ten EWSR1::FLI1 lines, six CIC::DUX4 lines, two BCOR-rearranged lines, two EWSR1::ERG lines and one EWSR1::FEV line. Tumoroids were established from both primary and metastatic disease timepoints. Tumoroids were subjected to an in-house-developed drug library containing ~34 standard of care chemotherapeutics, including currently used drugs for ES, BRS and CDS, and 150 early-phase clinical trial compounds targeting various pathways (Supplementary Data <a href="#MOESM6" class="usa-link">4</a>). Cell viability was measured at T0 and T120 to determine growth rate (control-no drug) (Supplementary Fig. <a href="#MOESM1" class="usa-link">1A, B</a>). The drugs were added in six different concentrations, allowing us to generate dose-response curves (Fig. <a href="#Fig5" class="usa-link">5A</a>; Supplementary Data <a href="#MOESM6" class="usa-link">4</a>−<a href="#MOESM7" class="usa-link">5</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Drug screening on small round cell sarcoma tumoroids.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371069_41467_2025_62673_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/39c1/12371069/ca01137500db/41467_2025_62673_Fig5_HTML.jpg" loading="lazy" id="d33e1185" height="1016" width="732" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic overview of the tumoroid drug screening workflow. Created in BioRender. Ringnalda, F. (2025) <a href="https://app.biorender.com/citation/68136ee9cceaa222dd9c568a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/mnd48e5</a>. <strong>B</strong> Clustered heatmap of AUC values of each tumoroid model (x axis) and compound (y axis), showing AUC values after 120 h drug treatment in a dose-range of 0.1 nM to 10 µM. Low AUC (red) indicates high drug sensitivity while high AUC (blue) indicates low drug sensitivity. Highlighted clusters include (1) chemotherapy, proteases and histone deacetylase inhibitors, (2) mTOR inhibitors, (3) E3 ubiquitin-protein ligases and (4) MCL-1 inhibitors. <strong>C</strong> Clustered heatmap showing AUC values after chemotherapy treatment, compounds that are routinely used in ES and ELS patients marked in red. <strong>D</strong> Box plots showing the mean AUCs values of mTOR inhibitors in the indicated ES (<em>n</em> = 13, tumoroid lines), CDS (<em>n</em> = 6, tumoroid lines) and BRS (<em>n</em> = 2, tumoroid lines) tumoroids entities. The boxes represent the interquartile range, the line inside the box marks the median, and the whiskers extend to the lowest and highest values. Error bars represent SEM of individual indicated ES, CDS and BRS tumoroid enitities. <em>P</em>-values were calculated using a one-way Anova, Dunnet multiple comparisons test:*&lt;0.05, **&lt;0.005, ns not significant. Exact adjusted P-values: AZD8055, EWSR1::FLI1 vs. EWSR1::ERG/FEV = 0.0033, EWSR1::FLI1 vs. BCOR::CCNB3/KMT2D = 0.0157, EWSR1::FLI1 vs. CIC::DUX4 = 0.0035. Sirolimus, EWSR1::FLI1 vs. EWSR1::ERG/FEV = 0.0147, EWSR1::FLI1 vs. BCOR::CCNB3/KMT2D = ns, EWSR1::FLI1 vs. CIC::DUX4 = 0.0281. Temsirolimus, EWSR1::FLI1 vs. EWSR1::ERG/FEV = ns, EWSR1::FLI1 vs. BCOR::CCNB3/KMT2D = ns, EWSR1::FLI1 vs. CIC::DUX4 = ns. <strong>E</strong> Box plots showing the mean AUCs values of P53 pathway activators in the indicated ES, CDS and BRS tumoroids entities. The boxes represent the interquartile range, the line inside the box marks the median, and the whiskers extend to the lowest and highest values. Error bars represent SEM of individual indicated <em>TP53</em> wildtype (p53<sup>WT</sup>) (<em>n</em> = 14) vs <em>TP53</em> mutant (p53<sup>MUT</sup>) (<em>n</em> = 7) tumoroids lines. <em>P</em>-values were calculated using a two-tailed Student’s <em>t</em> test: ****&lt;0.0001. Exact P-values: Idasanutlin ≤ 0.0001, Milademetan ≤ 0.0001, Siramadlin ≤ 0.0001. <strong>F</strong> Clustered heatmap of compounds targeting the apoptosis pathway, CDS tumoroids showing specific sensitivity towards MCL-1 inhibitors (in red). <strong>G</strong> Validation screen with MCL-1 inhibitors, dose-response curves of <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a>, MIK665 and AZD5991 for indicated ES and CDS tumoroid cultures. Error bars represent SD, each consisting of triplicate measurements. SEM standard error of mean, SD standard deviation, ES Ewing sarcoma, CDS CIC::DUX4 sarcoma, BRS BCOR-rearranged sarcoma. Source data are provided as a <a href="#MOESM10" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par18">We calculated the area under the curve (AUC) values of the individual drugs in all tumoroid models (Fig. <a href="#Fig5" class="usa-link">5B</a>). Members of four classes of drugs appeared to have tumor entity-specific effects: chemotherapeutics, mTOR inhibitors, p53 activators and MCL-1 inhibitors (Fig. <a href="#Fig5" class="usa-link">5B</a>). We first focused on chemotherapy sensitivity. Currently, patients with ES, CDS and BRS are treated with the same chemotherapy regimen, including either vincristine, ifosfamide, doxorubicin and etoposide (VIDE) (Ewing 2008 protocol) or vincristine, doxorubicin and cyclofosfamide, followed by ifosfamide and etoposide (VDC/IE)<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>,<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. The drugs used in these regimens and included in our drug library are marked in red in Fig. <a href="#Fig5" class="usa-link">5C</a>. Although the response varied between the different tumoroid lines, the CDS tumoroids overall responded less to the chemotherapeutics when compared to the other groups (Fig. <a href="#Fig5" class="usa-link">5C</a>). Previous retrospective studies have shown that CDS sarcomas are less chemo-sensitive than ES<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>–<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>, which was reflected in our tumoroids, indicating that tumoroid drug response can be predictive of in vivo response and highlighting the need for fusion gene-specific treatment options.</p>
<p id="Par19">We noticed that mTOR inhibitors (AZD8055, sirolimus and temsirolimus) were particularly effective in EWSR1::FLI1 tumoroids compared to the other groups (Fig. <a href="#Fig5" class="usa-link">5D</a>), in line with previous literature that showed that mTOR inhibition leads to reduced EWSR1::FLI1 protein expression<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. As the response to mTOR inhibition was independent of the <em>TP53</em> mutational status in ES tumoroids (Supplementary Fig. <a href="#MOESM1" class="usa-link">5A</a>), this response seems to be EWSR1::FLI1-specific. As expected, compounds that activate the p53 signaling pathway (idasanutlin, milademetan and siremadlin) were highly effective against <em>TP53</em> wildtype tumoroids compared to <em>TP53</em> mutant lines (Fig. <a href="#Fig5" class="usa-link">5E</a>). We also tested chemotherapeutics that are currently used in ES patients. All EWSR1::FLI1 tumoroids responded to currently used chemotherapy agents. We observed that EWSR1::FLI1 tumoroid lines derived from chemotherapy-treated tumors responded slightly less to chemotherapeutics than those derived from treatment-naïve samples. This difference was not significant, possibly due to the small sample size (Supplementary Fig. <a href="#MOESM1" class="usa-link">5B</a>). Looking at the MCL-1 inhibitors, we noticed that specifically CDS tumoroids were highly sensitive to <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a>, MIK665, AZD5991 and -to a lesser extent- to Tapotoclax (Fig. <a href="#Fig5" class="usa-link">5F, G</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">5D, E</a>). Looking back at the transcriptome analysis, we noticed that CDS indeed expressed higher levels of <em>MCL1</em>, the target of these drugs, compared to the other groups (Fig. <a href="#Fig4" class="usa-link">4D</a>). In contrast to the anti-apoptotic MCL1, NOXA is a pro-apoptotic protein that binds and neutralizes MCL1, and we found that <em>NOXA</em> mRNA expression is lower in CDS compared to ES (Supplementary Fig. <a href="#MOESM1" class="usa-link">6A</a>). In a Western blot analysis, we saw a similar trend in MCL1 and NOXA expression in two CDS compared to two ES tumoroid samples (Supplementary Fig. <a href="#MOESM1" class="usa-link">6B</a>). We also confirmed that CDS tumoroids have high MAPK phosphatase (MKPs) expression (Supplementary Fig. <a href="#MOESM1" class="usa-link">4A</a>), which may contribute to the MAPK/MEK inhibitor resistance that we observed in CDS tumoroids compared to ES tumoroids (Supplementary Fig. <a href="#MOESM1" class="usa-link">5C</a>)<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>.</p>
<p id="Par20">To confirm the specific vulnerability of CDS tumoroids to MCL-1 inhibitors, we performed validation screens on a selection of six ES (five EWSR1::FLI1 and one EWSR1::ERG line(s)) and six CIC::DUX4 tumoroid lines. The validations were performed with more drug concentrations (12) to accurately determine half-maximal inhibitory concentration (IC<sub>50</sub>) values for the different tumoroid lines. CDS tumoroids were markedly more sensitive to MCL-1 inhibitors compared to the ES tumoroids (Fig. <a href="#Fig5" class="usa-link">5G</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">5D, E</a>). MCL-1 inhibitors <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a> and MIK665 showed very similar dose-response curves in all tumoroid models. Since compound MIK665 is a derivative of <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a>, this confirmed the robustness of our tumoroid platform. The IC<sub>50</sub> values for <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a> and MIK665 in the CDS tumoroids ranged from 0.9 nM to 39 nM, while IC<sub>50</sub> values for the ES tumoroids could only be calculated from two lines, being 3235 nM and 4349 nM (Fig. <a href="#Fig5" class="usa-link">5G</a>). For the other ES lines, the full inhibitory plateau was not reached. The MCL-1 inhibitors AZD5991 and Tapotoclax gave a less strong response in all CIC::DUX4 models, with IC<sub>50</sub> values ranging from 20 to 110 nM and 160 to 550 nM, respectively (Fig. <a href="#Fig5" class="usa-link">5G</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">5D, E</a>). These results implied that MCL-1 plays an important role in the survival of CIC::DUX4 cells and that MCL-1 inhibition showed promise as a possible targeted therapy. To determine whether the MCL-1 inhibition reduces cell viability in CDS tumoroids by inducing apoptosis, we performed Annexin V and PI imaging experiments upon treatment with the MCL-1 inhibitor MIK665. This confirmed that MCL-1 inhibition induced apoptosis already after 1 h in CDS tumoroids but not in ES tumoroids (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>).</p>
<p id="Par21">Taken together, we show that ES, CDS and BRS tumoroids can be used for drug testing, which enabled us to find tumor entity-specific drug vulnerabilities. In particular, MCL-1 inhibitors stood out as potentially promising targeted drugs for CDS treatment.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par22">ES, CDS and BRS are highly aggressive tumors that mainly affect children and young adults. In the last decades, the survival rate has improved for patients that are diagnosed with a primary tumor, but remains poor for patients that have refractory or relapsed disease<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Despite being different in many aspects, including genetics and pathology, patients with any of these entities receive the same therapeutic regimen. In this study, we show that tumoroids of different ES, CDS and BRS entities reflect the genetic and transcriptomic profiles of their matching primary tumors and exhibit different responses to drug screens. We identify several classes of drugs with entity-specific responses, including MCL-1 inhibitors which specifically target CDS tumoroids.</p>
<p id="Par23">2D cell lines derived from SRCS have contributed tremendously to our understanding of the tumor biology. However, in most cases these were derived from late-stage cancers and thereby lack characteristics of early disease timepoints. Moreover, the currently available cell lines do not reflect the heterogeneous nature of these tumor entities, while the corresponding patient histories are typically not recorded<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>–<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. Patient-derived tumoroids have many advantages compared to 2D cell lines and are already applied in pre-clinical settings for multiple adult and pediatric cancers<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>–<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. PDX models are currently the gold standard preclinical models, but these are time-consuming, take rates are generally low and they are very costly to produce and maintain. Tumoroid models can be established in a matter of weeks with high take rates, and because we can efficiently expand them, they are suitable for large-scale high-drug screening platforms<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>–<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Several studies have shown that tumoroids of different cancer entities, including rectal cancer, colorectal cancer, lung cancer, head and neck cancer, pancreatic cancer, and ovarian cancer, can predict patient responses to radiation therapy, chemotherapy and targeted therapy<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p>
<p id="Par24">This study describes the establishment of a broad panel of ES, CDS and BRS patient-derived tumoroids which can be passaged and maintained long-term, including those with rare fusions such as EWSR1::FEV, EWSR1::ERG and CIC::DUX4. Importantly, we demonstrate that the tumoroids retain tumor heterogeneity in culture. By establishing tumoroids from different disease timepoints, including primary and relapsed tumors, we show that tumor evolution was reflected in the tumoroids. These tumoroids are therefore of value as representative pre-clinical models. A living tumoroid biobank can be used in future studies, not only for translational studies using drug screens as we did in this study, but also for more fundamental studies, e.g., by performing gene editing.</p>
<p id="Par25">Several studies have revealed that the genetic landscape of ES, CDS and BRS is heterogeneous<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. In ES, chromosomal aberrations are common, while recurrent somatic mutations other than the fusion gene are sparse but most commonly present in <em>TP53, STAG2</em> and <em>CDKN2A</em>. Mutations in genes such as <em>TP53</em> and <em>STAG2</em> often co-occur in a subset of patients with refractory ES<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. In our dataset, we noticed that several EWSR1::FLI1 sarcomas also harbored a loss of <em>CDKN2A/B</em>, either homo- or heterozygous. In CDS, loss of chromosome 1p and gain of chromosome 8 (including MYC amplification) has been reported previously; we detected this also in one CDS in our dataset. Additional recurrent driver mutations are scarce in CIC::DUX4 sarcomas<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. Similar to ES, BRS frequently show a loss of <em>CDKN2A/B</em>, which we also detected in our KMT2D::BCOR samples, both tumoroid and tumor<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>.</p>
<p id="Par26">Although the tumoroids closely resembled their matching tumors in many aspects, we also observed some differences. For instance, some somatic variants that were present in the tumor were not detected in the matching tumoroids, e.g., a <em>MYC</em> gene amplification in sample CDS-045. Vice versa, some tumoroids carried additional somatic variants compared to the matching tumor, e.g., in sample ES-041 (<em>TP53</em> mutant) and BRS-057. We reason that there are several explanations for this. First, the tumor tissue pieces used for diagnostic sequencing and for tumoroid derivation are different and there could be regional differences. In another study, we have analyzed clonal selection and tumor heterogeneity in three primary, previously untreated colon cancers<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. We found genetic differences even in tumor cells which are closely related within the same tumor. We assume the same could hold true for ES, CDS and BRS tumors. Second, discrepancies can result from clonal outgrowth of mutations in cells that were already present, but in too low numbers to be detected. Third, additional de novo mutations might have occurred while culturing. For some samples, we see relatively large Jaccard’s distances between TT-TO pairs, which is related to the different sequencing methods that were used (WXS versus WGS). Overall, we see more similarities than differences, meaning that these tumoroids can provide useful information, e.g., about tumor-specific drug sensitivities.</p>
<p id="Par27">In the REGOBONE study, regorafenib has been shown to modestly delay progression-free survival in ES patients with metastatic disease<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. However, in our ES tumoroids, we did not observe any response to regorafenib. In vivo, regorafenib is heavily metabolized by the liver. Therefore, not all active metabolites such as the M-2 and M-5 metabolites are present in our in vitro model, potentially explaining the lack of response to regorafenib in our ES tumoroids.</p>
<p id="Par28">Our data show that MCL-1 inhibitors, particularly <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a> and MIK665, induce strong and effective cell killing at low nM dose range. In line with this, the expression of MCL1 was increased in CDS samples, a feature that can be linked to drug resistance, in particular to chemotherapeutic agents<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>. Several MCL inhibitors are currently being evaluated in phase I clinical trials for acute myeloid leukemia (AML) and relapsed lymphoma as a single compound treatment or in combination with BLC2 inhibitors (clinicaltrials.gov: <a href="https://clinicaltrials.gov/ct2/show/NCT02992483" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT02992483</a>, <a href="https://clinicaltrials.gov/ct2/show/NCT02979366" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT02979366</a>, <a href="https://clinicaltrials.gov/ct2/show/NCT03672695" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT03672695</a>). One of the first trials with AMG-397 was put on hold because of possible concerns about on-target/off-tumor toxicity that caused cardiac side effects<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>. Because MCL-1 is implicated in normal cardiac myocyte functioning, side effects like these are probably directly related to the on-target activity of these compounds which may limit the therapeutic window of MCL-1 inhibitors<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>. Nonetheless, since the MCL-1 inhibitors already affected CDS tumoroids at low dose, these could have potential for future treatment of CDS patients, for whom effective treatment options currently do not exist. In future studies, it would be interesting to perform combination screens to identify drugs with synergistic effects, which could lower required dosing and thereby reduce MCL-1-related toxicity. For example, a recent study in zebrafish xenografts showed that dual targeting of MCL-1 and BCL-XL effectively targeted ES cells<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>.</p>
<p id="Par29">Here, we report that tumoroid models can be generated with high efficiency from ES, CDS and BRS, while preserving the molecular and phenotypic heterogeneity of their matching tumors. Furthermore, the tumoroid models confirm distinct entity-specific vulnerabilities towards chemotherapy and show specific sensitivities for targeted drugs, including ones that have not been described previously. In particular, MCL-1 inhibition appears to be a potentially useful therapeutic strategy for CDS. Future studies could focus on combination screens to identify drugs with synergistic effects to further decrease the required doses. This newly established ES, CDS and BRS patient-derived tumoroid biobank generates a unique source of material for future research.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Ethics statement tumor tissue</h3>
<p id="Par30">Tumor samples were obtained from the Netherlands Cancer Institute (Amsterdam, The Netherlands) and as part of the biobank initiative of the Princess Máxima Center for pediatric oncology, Utrecht, The Netherlands, and are listed in Supplementary Data <a href="#MOESM3" class="usa-link">1</a>. The biobank initiative was approved by the medical ethical committee NedMec, and the biobank and data access committee of the Máxima approved the use of tumor tissues for this project. For tumoroids derived from patient-derived xenografts (PDX) (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>), approval to collect patient samples for development of PDX was granted by the Innovative Therapies for Children with Cancer Paediatric Preclinical Proof-of-concept Platform (ITCC-P4) in Europe. All patients and/or their legal guardians signed informed consent for biobank usage of their tumor samples.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Patient material processing</h3>
<p id="Par31">Tumor material obtained via diagnostic FNAB or surgical resection was collected in Advanced Dulbecco’s Modified Eagle’s Medium with Nutrient Mixture Ham’s F-12 (Adv-DF) (#12634, Invitrogen), supplemented with 1% pen/strep (#15140-122, Invitrogen), 1% Glutamax (#35050-038, Invitrogen) and 1% Hepes (#15630-056, Invitrogen) (hereafter referred to as Adv-DF+++). Using scalpels, tumor tissues were minced into 1-3mm<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup> pieces, and was subdivided into three parts, one part was used for tumoroid establishment, one for downstream histopathological analysis and one part for cryopreservation. The part used for tumoroid establishment, was mechanically dissociated by repetitive pipetting with a P1000 pipet. After washing with ice-cold Adv-DF+++ medium, the cells were taken into culture (see: generation long-term tumoroid cultures).</p></section><section id="Sec14"><h3 class="pmc_sec_title">Patient-derived xenografts</h3>
<p id="Par32">The PDX samples were shipped on ice by courier. On arrival, tumor tissues were washed in Adv-DF+++ and cut into 1–3 mm<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup> pieces using scalpels. If tumor material contained fatty tissue, a pair of scissors and a tweezer were used to remove this prior to further handling. At this point, the tumor tissue was subdivided into four parts, one part was used for tumoroid establishment, one for downstream histopathological analysis, one for DNA/RNA isolation and one part for cryopreservation. When the sample contained too many red blood cells, red blood cell lysis buffer (#11814389001, Roche) was used according to the manufacturer’s guidelines. The part used for tumoroid establishment, was mechanically dissociated by repetitive pipetting with a P1000 pipet. After washing with ice-cold Adv-DF+++ medium, the cells were taken into culture (see: generation long-term tumoroid cultures).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Generation long-term tumoroid cultures</h3>
<p id="Par33">Cells were seeded in one of the three freshly prepared Sarcoma Culture Medium (SCM) (Supplementary Data <a href="#MOESM5" class="usa-link">3</a>). The culture medium contained the following components; Adv-DF+++ with 1x B27 supplement without Vitamin A (#12587-010, Gibco), 1× N2 (#17502-048, Gibco), 0.2% BME (#3533-010-02, Bio-Techne), 1.25 mM N-Acetylcystein (#A9165, Sigma Aldrich). For the different SCM media the following growth factors were added: SCM1: human epidermal growth factor (hEGF; 50 ng/mL, Peprotech), human fibroblast growth factor (hFGF2; 50 ng/mL, Peprotech) and human insulin-like growth factor-I (hIGF1; 100 ng/mL, Peprotech), SCM2; hEGF; 50 ng/mL, hFGF2; 10 ng/mL and hIGF1; 100 ng/mL and SCM3; hEGF; 15 ng/mL, hFGF2; 15 ng/mL and hIGF1; 100 ng/mL. Every 3–4 days, 50% of the medium was removed and fresh medium was added. Cultures were passaged every 1–3 weeks in a 1:2 to 1:6 ratio. For passaging, tumoroids were disrupted by mechanical dissociation by using a P1000 pipet. Following the addition of 5-10 mL ice-cold Adv-DF+++, cells were centrifuged at 300 × <em>G</em> for 5 min at 4 °C and subsequently reseeded in fresh STS medium. All tumoroid models were established in suspension plates (#665102, Greiner) and grown at 37 °C with 5% CO<sup>2</sup>. Tumoroids were regularly (every 6–8 weeks) checked for mycoplasma contamination using the MycoAlert Mycoplasma Detection Kit (#LT07-318, Lonza). Tumoroids were frozen down for long-term storage in CellBanker 2 (#11914, Amsbio).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Fusion detection by RT-qPCR</h3>
<p id="Par34">For the detection of fusion transcripts, snap-frozen cell pellets were collected after passaging the tumoroid cultures for at least five times. Total RNA was isolated from tumoroid lines using an Rneasy Mini Kit (#74104, Qiagen) according to the manufacturer’s instructions, including the recommended DNAse I treatment (#79254, Qiagen). A total of 500 ng RNA was used for cDNA synthesis, using an oligo-dT primed Superscript III Reverse Transcriptase (#18080044, Invitrogen) base reaction according to the manufacturer’s protocol. RT-PCR for the different fusion transcripts was performed with the obtained cDNA using iQ SYBR Green (Bio-Rad, #1708882). PCR reactions were run on 1.5% agarose gel. PCR products were cut out and purified with an QIAquick Gel Extraction Kit (#28704, Qiagen). Purified PCR products were sequenced with Sanger sequencing to confirm the identity and determine the exact fusion transcripts. Primers used in this study are listed in Supplementary Data <a href="#MOESM5" class="usa-link">3</a>.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Histology and immunohistochemistry</h3>
<p id="Par35">Tumor tissues and tumoroids were fixed in 4% paraformaldehyde, dehydrated, and subsequently embedded in paraffin. Subsequently, 4 µm thick sections were cut and used for H&amp;E and immunohistochemical staining according to standard protocols. The following primary antibodies were used: CD99 (#915604, Biolegend, 1:150), WT1 (#05-753, Sigma-Aldrich, 1:200), BCOR (#Z2334MP, Zeta Corporation, 1:100) and ETV4 (#LS-C800519_200, LS Bio, 1:200). Images were acquired with a Leica DMi8 microscope.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Whole-genome and whole-exome sequencing</h3>
<p id="Par36">Genomic DNA was extracted between passage 5–10 using QIAGEN blood and tissue DNA kit (QIAGEN, Hilden, Germany). DNA library preparation was done using the KAPA HyperPlus kit (Roche, Basel, Switzerland) according to the manufacturer’s instructions. Tumors (60–90× coverage), whole blood (30x coverage) and tumoroids (30× coverage) were sequenced on a NovaSeq 6000 (Illumina). Reads were mapped against the human reference genome (GRCh38) with the Burrows-Wheeler Aligner (v0.7.5a) mapping software with settings ‘bwa mem -c 100 -M’. Duplicate reads were marked with Sambamba (v0.6.8). Variants were annotated using the Genome Analysis Toolkit (GATK) (v3.8-1-0). A detailed description of the complete data analysis pipeline is available at: <a href="https://github.com/UMCUGenetics/IAP" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/UMCUGenetics/IAP</a> and <a href="https://github.com/Hubrecht-Clevers/NF-IAP_exome" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Hubrecht-Clevers/NF-IAP_exome</a> for WGS and WXS, respectively. Postprocessing of variants was done using the SMuRF pipeline with default settings. A detailed overview of the postprocessing pipeline can be found at <a href="https://github.com/ToolsVanBox/SMuRF" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ToolsVanBox/SMuRF</a>. Briefly, variants on all autosomal and sex chromosomes were used in this analysis. Variants were filtered on mapping quality &gt;40, base coverage of at least 10, a blacklist was used to filter out known false positives (available on request). Variants were also filtered to a matched normal sample when available. Furthermore, variants with a mean allele frequency of above 0.01 in the general population were filtered out unless these variants had a COSMIC identifier. For samples that did not have a matched normal, only known gene altering variants with a COSMIC identifier in a known oncogene (COSMIC cancer consensus genes) were taken into account. VCF-files from postprocessing were converted to MAF-files using a personalized script available at <a href="https://github.com/Hubrecht-Clevers/Convert_maf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Hubrecht-Clevers/Convert_maf</a>. Variants were further analyzed in Rstudio with R (v4.4.0), maftools (v2.19), ggplot2 (v3.5.1), data.table (v1.15.4), RColorBrewer (v1.1-3), NMF (v0.27), viridis (v0.6.5), gggenes (v0.5.1), Seurat (v5.0.3), ggrepel (v0.9.5) and readxl (v1.4.3). The full script is available at <a href="https://github.com/Hubrecht-Clevers/small_cell_sarcomas" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Hubrecht-Clevers/small_cell_sarcomas</a>. Briefly, variants were divided into coding silent mutations such as synonymous variants, intron variants and in-frame deletions or insertions and damaging mutations such as nonsynonymous variants, frameshifts and gene fusions. Visualization of mutations was further processed in R using maftools (v2.19.0). The detailed script is available at <a href="https://github.com/Hubrecht-Clevers/small_cell_sarcomas" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Hubrecht-Clevers/small_cell_sarcomas</a>.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Mutational signature analysis</h3>
<p id="Par37">All single nucleotide variants, both non-synonymous and synonymous were loaden into R. A trinucleotide matrix was constructed using the UCSC h38 genome. Mutational signatures were extracted using the extractSignatures function from Maftools. Signatures were compared with known SBS signatures form the COSMIC database. Cosine distance was computed and the best match for each signature was annotated to the extracted signatures. Ggplot2 was used for further visualization of the mutational signatures. The full script is available at <a href="https://github.com/Hubrecht-Clevers/small_cell_sarcomas" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Hubrecht-Clevers/small_cell_sarcomas</a>.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Clonal evolution on longitudinal tumor samples</h3>
<p id="Par38">A full description of the pipeline and variables is available at <a href="https://github.com/Hubrecht-Clevers/small_cell_sarcomas" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Hubrecht-Clevers/small_cell_sarcomas</a>. Briefly, allele specific copy number profiles were constructed using CNV-facets(v0.5.14) using standard settings<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. Doing so, for all variants (synonymous and non-synonymous) the minor and major copy number was computed. Next, subclones were identified using pyclone-vi<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup>. Pyclone fit function was run with the following settings “-c 40 -d beta-binomial -r 100”, followed by the write-results-file function. Visualization of subclones over the different samples was done in R using the ggalluvial package (v0.12.5).</p></section><section id="Sec21"><h3 class="pmc_sec_title">Bulk RNA sequencing</h3>
<p id="Par39">Total RNA was extracted from tumoroid lines using a Rneasy Mini Kit (#74104, Qiagen) following manufacturer’s instructions, including a recommended DNAse I treatment (#79254, Qiagen). In case of tumor tissue, total RNA was extracted using an AllPrep DNA/RNA/miRNA universal Kit (#80224, Qiagen) following manufacturer’s instructions. RNA sequencing of tumor tissues and tumoroids, unless specified, was performed as previously described<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup>. In short, cDNA libraries were generated using the KAPA RNA HyperPrep Kit with RiboErase (#08098140702, Roche) and sequenced on a NovaSeq 6000 (2 × 150 bp) (Illumina). Reads were mapped to the human<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> (GRCh38 and gencode version 32) by STAR(v2.7.2 d). Fusions were detected by STARfusion (v1.8.0) using standard settings. Raw read counts were obtained via featureCounts function of the subread package (v1.6.6). Fusion transcripts were compared between tumor and tumoroid to confirm matching tumor/tumoroid pairs. Readcounts were further analyzed in R using DESEQ2 (v1.44.0) for differential expression analysis. Pathway analyses were preformed using the Enrichr packages (v3.2). The full script used in this analysis and to generate figures is available at <a href="https://github.com/Hubrecht-Clevers/small_cell_sarcomas" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Hubrecht-Clevers/small_cell_sarcomas</a>.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Medium-throughput drug screens</h3>
<p id="Par40">Tumoroids were harvested, washed in ice-cold Adv-DF+++, filtered using a 70 μm nylon cell strainer (#542070, Greiner Bio-One) and resuspended in SCM. Subsequently, ~800 tumoroid clusters (one cluster consisting of ~6 cells) were plated in a volume of 40 µl per well in black 384-well plates (#3764, Corning) using a Multi-drop Combi Reagent Dispenser (#5840340, Thermo Scientific). Two days after plating, compounds were added (six different concentrations) using an acoustic liquid handler Echo550. Positive control samples were treated with staurosporine (final concentration 10 μM) and negative control samples were treated with DMSO (final concentration of 0.25%). All medium-throughput drug screens were carried out in duplicate. The drug screens were performed using a library containing 186 drugs. Drugs are listed in Supplementary Data <a href="#MOESM6" class="usa-link">4</a>. Readouts were performed at <em>T</em> = 0 (no drugs—control plate) and <em>T</em> = 120 (120 h after drugs added—readout) using CellTiter-Glo 3D reagent (#G9683, Promega) according to the manufacturer’s instructions on the Spectramax i3x luminescence microplate reader. The results were normalized to the DMSO vehicle (100% viability) (Supplementary Data <a href="#MOESM7" class="usa-link">5</a>). Data were analyzed with software GraphPad Prism v10.02. Forced group clustering was visualized using the R package ComplexHeatmap v2.2.0 with default values.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Validation drug screen</h3>
<p id="Par41">Validations drug screens were performed in the same workflow as mentioned above, with the exeption that compounds were added using the Tecan D300e Digital Dispenser. The following drugs were used: Bortezomib (#HY-10227), AZD5991 (#HY-101533), Tapotoclax (#HY-101565), <a href="https://www.ncbi.nlm.nih.gov/nuccore/S63845" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S63845</a> (#HY-10074), MIK665 (#HY-112218), and A − 1210477 (#HY-12468) were all purchased from MedChemExpress. For the validation screens, twelve concentrations (0.03 nM, 0.1 nM, 0.3 nM, 1 nM, 3 nM, 10 nM, 30 nM, 100 nM, 300 nM, 1 µM, 3 µM and 10 µM) were used. Drugs were added in triplicate for each compound per experiment. Validation experiments were performed at least twice. Bortezomib was taken along on every validation and was used as internal positive control. The results were normalized to the DMSO vehicle (100% viability). Data were analyzed with software GraphPad Prism v10.02.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Apoptosis assay</h3>
<p id="Par42">ES and CDS tumoroids were seeded in 96-well plates 2 days prior to treatment. The seeded tumoroids were incubated with single agent MIK665 (50 nM) for 1, 3 and 6 h at 37 °C. Individual wells with tumoroids were washed with PBS, and Annexin V binding buffer (#422201, Biolegend), followed by staining with Annexin V-FITC (#940906, Biolegend) for 15 min in the dark at room temperature. Then, propidium iodide (#421301, Biolegend) was added 5 min prior imaging. Images were acquired with a Leica STELLARIS8 FALCON microscope.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Western blotting assay</h3>
<p id="Par43">ES and CDS tumoroids were collected and washed 3× with PBS. Whole-cell lysates were prepared using RIPA lysis buffer (50 mM Tris, 150 mM NaCl, 0.1% sodium dodecyl sulfate, 0.5% sodium deoxycholate, 0.1% Triton X-100) to which protease/phosphatase inhibitor cocktails (#11873580001, Roche) were freshly added in a 1:100 ratio. Protein concentrations were determined using a BCA protein assay kit (#23227, Pierce™, Thermo Scientific™) per manufacturer’s instructions. Lysate samples containing 20 μg of protein were prepared with Laemmli loading buffer and separated on 12% SDS-polyacrylamide gels and transferred to 0.45 µM nitrocellulose membranes using wet transfer. Membranes were probed overnight at 4 °C with the following primary antibodies: MCL-1 (#CST#39224, Cell Signaling), NOXA (#MA5-32679, Invitrogen™) and Vinculin (CST#13901, Cell Signaling). Membranes were subsequently incubated with rabbit (#<a href="https://www.ncbi.nlm.nih.gov/nuccore/G21234" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G21234</a>, Invitrogen™) horseradish peroxidase-conjugated secondary antibody and protein bands were visualized using ECL Western Blotting Substrate (#32209, Pierce™, Thermo Scientific™). Images were captured using Bio-Rad ChemiDoc, and densitometric analysis was carried out using ImageJ (V2.0.0-rc-69/1.52i).</p></section><section id="Sec26"><h3 class="pmc_sec_title">Original western blots are available in the Source data file</h3>
<section id="Sec27"><h4 class="pmc_sec_title">Statistics analysis</h4>
<p id="Par44">AUC was calculated using Prism software version 10.0.2 with a 4PL regression curve. Values (AUC) were plotted on a matrix to evaluate sample to sample differences. Differences in AUC values of specific drugs were evaluated using a one-way ANOVA (more than 2 groups) or a students’ <em>T</em>-test (when only 2 groups) in Prism. Differential gene expression was calculated using the DEseq2 package using standard settings. nbinomLRT likelyhood test was used to identify DEGs and pvalues were adjusted with a false discovery rate by methods of Benjamini and Hochberg. Euclidian sample distance matrices were calculated based on RNA expression values. Samples were then clusterd using hiarchical clustering. For WGS, mutation matrices were generated in a binary format to allow a distance matrix calculaton in Jaccard distance. Samples were then again clustered using hiarchical clustering (R package stats v4.4.0).</p></section></section><section id="Sec28"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par45">Further information on research design is available in the <a href="#MOESM8" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec29"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (383.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62673_MOESM2_ESM.pdf</a><sup> (41.1KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 1</a><sup> (13.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 2</a><sup> (11.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 3</a><sup> (12.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 4</a><sup> (25.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 5</a><sup> (10.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (86.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (656.2KB, pdf) </sup>
</div></div></section></section><section id="Sec30"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (26.8MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank all children and their families for participating in our research as well as the clinical teams involved in approaching patients for consent and collecting tissue. The ITCC-P4 consortium has received funding from the Innovative Medicines Initiative 2 Joint Undertaking, which has been supported by the European Union’s Horizon 2020 research and innovation program and the European federation of Pharmaceutical Industries and Associations. Research reported in this publication was supported by the Children Cancer Free Foundation (KiKa) (M.v.d.W. and H.C.), by Oncode Institute (H.C.) and by Oncode Accelerator, a Dutch National Growth Fund Project under grant number NGFOP2201 (K.S. and M.v.d.W.).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: F.R., M.v.d.W., H.C.; Designed experiments: F.R., M.v.d.W.; Performed experiments and data analyses: F.R.; Performed Western blot experiments: S.Y.K; Data curation: F.R., L.V., V.A.A., H.K., M.S., J.J.M., R.v.B.; Performed histopathological evaluations: U.F., L.H.J.; Formal Analysis: F.R., G.v.S., L.V.; Performed bioinformatic analyses: G.v.S.; Project administration: F.R., M.v.d.W., K.S., H.C.; Resources: K.L., J.B., L.H., M.v.d.S., W.v.H., M.v.N., G.S., O.D., K.Sc., B.G., J.H.M.M. and J.J.M.; Provided clinical context and guidance: K.L., M.v.d.S., and J.H.M.M.; Developed pipelines for data analyses: R.v.B.; Supervision: M.v.d.W., K.S., H.C.; Writing—original draft: F.R.; Writing—review &amp; editing: F.R., G.v.S., M.v.d.W., K.S., H.C.; Funding acquisition: M.v.d.W., K.S., H.C.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par46"><em>Nature Communications</em> thanks Joseph Ludwig, Alice Soragni, and the other, anonymous, reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The transcriptomic and genomic data are deposited in the EGA study database under accession number <a href="https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fega-archive.org%2Fsearch%2FEGAS00001007941&amp;data=05%7C02%7Cf.c.a.ringnalda%40prinsesmaximacentrum.nl%7C7205b98651d24b80db2108ddca8e4164%7Ced9f3ca1097f493ea1295002e9c16d65%7C0%7C0%7C638889434506777356%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&amp;sdata=wWiht6B5IvmEtVanWoFZsXYgj3LyfaT%2B%2BzRil%2BRZCmo%3D&amp;reserved=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EGAS00001007941</a>, RNA-sequencing data is available under accession number <a href="https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fega-archive.org%2Fsearch%2FEGAD00001015419&amp;data=05%7C02%7Cf.c.a.ringnalda%40prinsesmaximacentrum.nl%7C7205b98651d24b80db2108ddca8e4164%7Ced9f3ca1097f493ea1295002e9c16d65%7C0%7C0%7C638889434506833488%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&amp;sdata=Cq9eKLlCpW7hS0EnLNir6e2Nz%2BDKxAqBAT06qzChAZg%3D&amp;reserved=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EGAD00001015419</a>, WGS data is available under accession number <a href="https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fega-archive.org%2Fsearch%2FEGAD00001015418&amp;data=05%7C02%7Cf.c.a.ringnalda%40prinsesmaximacentrum.nl%7C7205b98651d24b80db2108ddca8e4164%7Ced9f3ca1097f493ea1295002e9c16d65%7C0%7C0%7C638889434506874051%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&amp;sdata=EgQ7vy2R15%2FGwrGnkuDHshKNGRob%2BKyLrwVmCG2HlCE%3D&amp;reserved=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EGAD00001015418</a> and WXS data is available under accession number <a href="https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fega-archive.org%2Fsearch%2FEGAD00001015417&amp;data=05%7C02%7Cf.c.a.ringnalda%40prinsesmaximacentrum.nl%7C7205b98651d24b80db2108ddca8e4164%7Ced9f3ca1097f493ea1295002e9c16d65%7C0%7C0%7C638889434506900494%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&amp;sdata=OSqd4W44bgMikaJpaZq%2B1t1l5MYpDgwIHM1Md6QBOjY%3D&amp;reserved=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EGAD00001015417</a>. Datasets mentioned above are available under restricted access due to patient privacy and regulatory requirements. Access to the sequencing data is managed by the Biobank and Data Access Committee (BDAC) of the Princess Máxima Center. All researchers can request access by submitting a project proposal to the BDAC (biobank@prinsesmaximacentrum.nl). Requests are typically reviewed within ~2 weeks. The duration of access will be determined by the BDAC. The ChIP-seq publicly available dataset used in this study were accessed from ENCODE experiments <a href="https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.encodeproject.org%2Ffiles%2FENCFF186JKG%2F&amp;data=05%7C02%7Cf.c.a.ringnalda%40prinsesmaximacentrum.nl%7C7205b98651d24b80db2108ddca8e4164%7Ced9f3ca1097f493ea1295002e9c16d65%7C0%7C0%7C638889434506923626%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&amp;sdata=pYuH%2FhFffMEBtYEWp%2BVSkRK1YVxuJ53GfqgKK4Fy2WQ%3D&amp;reserved=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ENCFF186JKG</a> [<a href="https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.encodeproject.org%2Ffiles%2FENCFF186JKG%2F&amp;data=05%7C02%7Cf.c.a.ringnalda%40prinsesmaximacentrum.nl%7C7205b98651d24b80db2108ddca8e4164%7Ced9f3ca1097f493ea1295002e9c16d65%7C0%7C0%7C638889434506941686%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&amp;sdata=rO32szxEWFOzj2%2BvIkyHOzaBNB14Z6kuFUNHBfV3zks%3D&amp;reserved=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.encodeproject.org/files/ENCFF186JKG/</a>] for BCOR. Requests to use the tumoroid lines that were generated in this paper will be evaluated on a case-by-case basis by the BDAC of the Princess Máxima Center. When approved, specific material transfer agreements will be drawn up. To contact the BDAC, email biobank@prinsesmaximacentrum.nl. The remaining data are available within the Article, Supplementary Information or Source Data file. <a href="#Sec30" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>All code used to generate figures can be accessed on <a href="https://github.com/Hubrecht-Clevers/small_cell_sarcomas" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Hubrecht-Clevers/small_cell_sarcomas</a>.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par47">H.C. is an inventor on patents held by the Royal Netherlands Academy of Arts and Sciences that cover organoid technology. He is now head of Pharma Research and Early Development (pRED) at Roche, Basel, Switzerland. H.C.’s full disclosure is given at <a href="https://uu.nl/staff/JCClevers/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://uu.nl/staff/JCClevers/</a>. All other authors declare that they have no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Karin Sanders, Hans Clevers.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Karin Sanders, Email: k.sanders@prinsesmaximacentrum.nl.</p>
<p>Hans Clevers, Email: h.clevers@hubrecht.eu.</p></section><section id="sec31"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62673-2.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Dehner, C. A., Lazar, A. J. &amp; Chrisinger, J. S. A. Updates on WHO classification for small round cell tumors: Ewing sarcoma vs. everything else. <em>Hum. Pathol.</em><strong>147</strong>, 101–113 (2024).
</cite> [<a href="https://doi.org/10.1016/j.humpath.2024.01.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38280658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dehner,%20C.%20A.,%20Lazar,%20A.%20J.%20&amp;%20Chrisinger,%20J.%20S.%20A.%20Updates%20on%20WHO%20classification%20for%20small%20round%20cell%20tumors:%20Ewing%20sarcoma%20vs.%20everything%20else.%20Hum.%20Pathol.147,%20101%E2%80%93113%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Ross, K. A., Smyth, N. A., Murawski, C. D. &amp; Kennedy, J. G. The biology of Ewing sarcoma. <em>ISRN Oncol.</em><strong>2013</strong>, 759725 (2013).
</cite> [<a href="https://doi.org/10.1155/2013/759725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3549336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23346417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ross,%20K.%20A.,%20Smyth,%20N.%20A.,%20Murawski,%20C.%20D.%20&amp;%20Kennedy,%20J.%20G.%20The%20biology%20of%20Ewing%20sarcoma.%20ISRN%20Oncol.2013,%20759725%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Aurias, A., Rimbaut, C., Buffe, D., Zucker, J. M. &amp; Mazabraud, A. Translocation involving chromosome 22 in Ewing’s sarcoma. A cytogenetic study of four fresh tumors. <em>Cancer Genet. Cytogenet.</em><strong>12</strong>, 21–25 (1984).
</cite> [<a href="https://doi.org/10.1016/0165-4608(84)90003-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6713357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aurias,%20A.,%20Rimbaut,%20C.,%20Buffe,%20D.,%20Zucker,%20J.%20M.%20&amp;%20Mazabraud,%20A.%20Translocation%20involving%20chromosome%2022%20in%20Ewing%E2%80%99s%20sarcoma.%20A%20cytogenetic%20study%20of%20four%20fresh%20tumors.%20Cancer%20Genet.%20Cytogenet.12,%2021%E2%80%9325%20(1984)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Delattre, O. et al. Gene fusion with an ETS DNA-binding domain caused by chromosome translocation in human tumours. <em>Nature</em><strong>359</strong>, 162–165 (1992).
</cite> [<a href="https://doi.org/10.1038/359162a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1522903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Delattre,%20O.%20et%20al.%20Gene%20fusion%20with%20an%20ETS%20DNA-binding%20domain%20caused%20by%20chromosome%20translocation%20in%20human%20tumours.%20Nature359,%20162%E2%80%93165%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Lessnick, S. L., Braun, B. S., Denny, C. T. &amp; May, W. A. Multiple domains mediate transformation by the Ewing’s sarcoma EWS/FLI-1 fusion gene. <em>Oncogene</em><strong>10</strong>, 423–431 (1995).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/7845667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lessnick,%20S.%20L.,%20Braun,%20B.%20S.,%20Denny,%20C.%20T.%20&amp;%20May,%20W.%20A.%20Multiple%20domains%20mediate%20transformation%20by%20the%20Ewing%E2%80%99s%20sarcoma%20EWS/FLI-1%20fusion%20gene.%20Oncogene10,%20423%E2%80%93431%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>May, W. A. et al. Ewing sarcoma 11;22 translocation produces a chimeric transcription factor that requires the DNA-binding domain encoded by FLI1 for transformation. <em>Proc. Natl. Acad. Sci. USA</em><strong>90</strong>, 5752–5756 (1993).
</cite> [<a href="https://doi.org/10.1073/pnas.90.12.5752" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC46800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8516324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?May,%20W.%20A.%20et%20al.%20Ewing%20sarcoma%2011;22%20translocation%20produces%20a%20chimeric%20transcription%20factor%20that%20requires%20the%20DNA-binding%20domain%20encoded%20by%20FLI1%20for%20transformation.%20Proc.%20Natl.%20Acad.%20Sci.%20USA90,%205752%E2%80%935756%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Kawamura-Saito, M. et al. Fusion between CIC and DUX4 up-regulates PEA3 family genes in Ewing-like sarcomas with t(4;19)(q35;q13) translocation. <em>Hum. Mol. Genet.</em><strong>15</strong>, 2125–2137 (2006).
</cite> [<a href="https://doi.org/10.1093/hmg/ddl136" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16717057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kawamura-Saito,%20M.%20et%20al.%20Fusion%20between%20CIC%20and%20DUX4%20up-regulates%20PEA3%20family%20genes%20in%20Ewing-like%20sarcomas%20with%20t(4;19)(q35;q13)%20translocation.%20Hum.%20Mol.%20Genet.15,%202125%E2%80%932137%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Chebib, I. &amp; Jo, V. Y. Round cell sarcoma with CIC-DUX4 gene fusion: discussion of the distinctive cytomorphologic, immunohistochemical, and molecular features in the differential diagnosis of round cell tumors. <em>Cancer Cytopathol.</em><strong>124</strong>, 350–361 (2016).
</cite> [<a href="https://doi.org/10.1002/cncy.21685" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26800124/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chebib,%20I.%20&amp;%20Jo,%20V.%20Y.%20Round%20cell%20sarcoma%20with%20CIC-DUX4%20gene%20fusion:%20discussion%20of%20the%20distinctive%20cytomorphologic,%20immunohistochemical,%20and%20molecular%20features%20in%20the%20differential%20diagnosis%20of%20round%20cell%20tumors.%20Cancer%20Cytopathol.124,%20350%E2%80%93361%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Okimoto, R. A. et al. CIC-DUX4 oncoprotein drives sarcoma metastasis and tumorigenesis via distinct regulatory programs. <em>J. Clin. Invest</em><strong>129</strong>, 3401–3406 (2019).
</cite> [<a href="https://doi.org/10.1172/JCI126366" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6668665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31329165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Okimoto,%20R.%20A.%20et%20al.%20CIC-DUX4%20oncoprotein%20drives%20sarcoma%20metastasis%20and%20tumorigenesis%20via%20distinct%20regulatory%20programs.%20J.%20Clin.%20Invest129,%203401%E2%80%933406%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Peters, T. L. et al. BCOR-CCNB3 fusions are frequent in undifferentiated sarcomas of male children. <em>Mod. Pathol.</em><strong>28</strong>, 575–586 (2015).
</cite> [<a href="https://doi.org/10.1038/modpathol.2014.139" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4385430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25360585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peters,%20T.%20L.%20et%20al.%20BCOR-CCNB3%20fusions%20are%20frequent%20in%20undifferentiated%20sarcomas%20of%20male%20children.%20Mod.%20Pathol.28,%20575%E2%80%93586%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Kao, Y. C. et al. BCOR-CCNB3 fusion positive sarcomas: a clinicopathologic and molecular analysis of 36 cases with comparison to morphologic spectrum and clinical behavior of other round cell sarcomas. <em>Am. J. Surg. Pathol.</em><strong>42</strong>, 604–615 (2018).
</cite> [<a href="https://doi.org/10.1097/PAS.0000000000000965" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5893395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29300189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kao,%20Y.%20C.%20et%20al.%20BCOR-CCNB3%20fusion%20positive%20sarcomas:%20a%20clinicopathologic%20and%20molecular%20analysis%20of%2036%20cases%20with%20comparison%20to%20morphologic%20spectrum%20and%20clinical%20behavior%20of%20other%20round%20cell%20sarcomas.%20Am.%20J.%20Surg.%20Pathol.42,%20604%E2%80%93615%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Dirksen, U. et al. High-dose chemotherapy compared with standard chemotherapy and lung radiation in ewing sarcoma with pulmonary metastases: results of the European Ewing Tumour Working Initiative of National Groups, 99 Trial and EWING 2008. <em>J. Clin. Oncol.</em><strong>37</strong>, 3192–3202 (2019).
</cite> [<a href="https://doi.org/10.1200/JCO.19.00915" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6881099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31553693/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dirksen,%20U.%20et%20al.%20High-dose%20chemotherapy%20compared%20with%20standard%20chemotherapy%20and%20lung%20radiation%20in%20ewing%20sarcoma%20with%20pulmonary%20metastases:%20results%20of%20the%20European%20Ewing%20Tumour%20Working%20Initiative%20of%20National%20Groups,%2099%20Trial%20and%20EWING%202008.%20J.%20Clin.%20Oncol.37,%203192%E2%80%933202%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Kovar, H. et al. The first European interdisciplinary ewing sarcoma research summit. <em>Front. Oncol.</em><strong>2</strong>, 54 (2012).
</cite> [<a href="https://doi.org/10.3389/fonc.2012.00054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3361960/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22662320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kovar,%20H.%20et%20al.%20The%20first%20European%20interdisciplinary%20ewing%20sarcoma%20research%20summit.%20Front.%20Oncol.2,%2054%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Brahmi, M. et al. Patterns of care and outcome of CIC-rearranged sarcoma patients: a nationwide study of the French sarcoma group. <em>Cancer Med.</em><strong>12</strong>, 7801–7807 (2023).
</cite> [<a href="https://doi.org/10.1002/cam4.5539" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10134374/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36537582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brahmi,%20M.%20et%20al.%20Patterns%20of%20care%20and%20outcome%20of%20CIC-rearranged%20sarcoma%20patients:%20a%20nationwide%20study%20of%20the%20French%20sarcoma%20group.%20Cancer%20Med.12,%207801%E2%80%937807%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Kyriazoglou, A. &amp; Bagos, P. Meta-analysis of BCOR rearranged sarcomas: challenging the therapeutic approach. <em>Acta Oncol.</em><strong>60</strong>, 721–726 (2021).
</cite> [<a href="https://doi.org/10.1080/0284186X.2021.1890818" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33630701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kyriazoglou,%20A.%20&amp;%20Bagos,%20P.%20Meta-analysis%20of%20BCOR%20rearranged%20sarcomas:%20challenging%20the%20therapeutic%20approach.%20Acta%20Oncol.60,%20721%E2%80%93726%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Nanni, P. et al. Bone sarcoma patient-derived xenografts are faithful and stable preclinical models for molecular and therapeutic investigations. <em>Sci. Rep.</em><strong>9</strong>, 12174 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-48634-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6704066/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31434953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nanni,%20P.%20et%20al.%20Bone%20sarcoma%20patient-derived%20xenografts%20are%20faithful%20and%20stable%20preclinical%20models%20for%20molecular%20and%20therapeutic%20investigations.%20Sci.%20Rep.9,%2012174%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Orth, M. F. et al. Systematic multi-omics cell line profiling uncovers principles of Ewing sarcoma fusion oncogene-mediated gene regulation. <em>Cell Rep.</em><strong>41</strong>, 111761 (2022).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2022.111761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10333306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36476851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Orth,%20M.%20F.%20et%20al.%20Systematic%20multi-omics%20cell%20line%20profiling%20uncovers%20principles%20of%20Ewing%20sarcoma%20fusion%20oncogene-mediated%20gene%20regulation.%20Cell%20Rep.41,%20111761%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Sato, T. et al. Long-term expansion of epithelial organoids from human colon, adenoma, adenocarcinoma, and Barrett’s epithelium. <em>Gastroenterology</em><strong>141</strong>, 1762–1772 (2011).
</cite> [<a href="https://doi.org/10.1053/j.gastro.2011.07.050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21889923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sato,%20T.%20et%20al.%20Long-term%20expansion%20of%20epithelial%20organoids%20from%20human%20colon,%20adenoma,%20adenocarcinoma,%20and%20Barrett%E2%80%99s%20epithelium.%20Gastroenterology141,%201762%E2%80%931772%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Al Shihabi, A. et al. The landscape of drug sensitivity and resistance in sarcoma. <em>Cell Stem Cell</em><strong>31</strong>, 1524–1542.e1524 (2024).
</cite> [<a href="https://doi.org/10.1016/j.stem.2024.08.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12318355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39305899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Al%20Shihabi,%20A.%20et%20al.%20The%20landscape%20of%20drug%20sensitivity%20and%20resistance%20in%20sarcoma.%20Cell%20Stem%20Cell31,%201524%E2%80%931542.e1524%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Calandrini, C. et al. An organoid biobank for childhood kidney cancers that captures disease and tissue heterogeneity. <em>Nat. Commun.</em><strong>11</strong>, 1310 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-15155-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7066173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32161258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Calandrini,%20C.%20et%20al.%20An%20organoid%20biobank%20for%20childhood%20kidney%20cancers%20that%20captures%20disease%20and%20tissue%20heterogeneity.%20Nat.%20Commun.11,%201310%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Meister, M. T. et al. Mesenchymal tumor organoid models recapitulate rhabdomyosarcoma subtypes. <em>EMBO Mol. Med.</em><strong>14</strong>, e16001 (2022).
</cite> [<a href="https://doi.org/10.15252/emmm.202216001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9549731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35916583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meister,%20M.%20T.%20et%20al.%20Mesenchymal%20tumor%20organoid%20models%20recapitulate%20rhabdomyosarcoma%20subtypes.%20EMBO%20Mol.%20Med.14,%20e16001%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Kluiver, T. A. et al. Divergent WNT signaling and drug sensitivity profiles within hepatoblastoma tumors and organoids. <em>Nat. Commun.</em><strong>15</strong>, 8576 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-52757-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11579375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39567475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kluiver,%20T.%20A.%20et%20al.%20Divergent%20WNT%20signaling%20and%20drug%20sensitivity%20profiles%20within%20hepatoblastoma%20tumors%20and%20organoids.%20Nat.%20Commun.15,%208576%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Kim, S. Y. et al. Organoid drug profiling identifies methotrexate as a therapy for SCCOHT, a rare pediatric cancer. <em>Sci. Adv.</em><strong>11</strong>, eadq1724 (2025).
</cite> [<a href="https://doi.org/10.1126/sciadv.adq1724" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11864178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40009666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20S.%20Y.%20et%20al.%20Organoid%20drug%20profiling%20identifies%20methotrexate%20as%20a%20therapy%20for%20SCCOHT,%20a%20rare%20pediatric%20cancer.%20Sci.%20Adv.11,%20eadq1724%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Paassen, I. et al. Atypical teratoid/rhabdoid tumoroids reveal subgroup-specific drug vulnerabilities. <em>Oncogene</em><strong>42</strong>, 1661–1671 (2023).
</cite> [<a href="https://doi.org/10.1038/s41388-023-02681-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10181938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37020038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paassen,%20I.%20et%20al.%20Atypical%20teratoid/rhabdoid%20tumoroids%20reveal%20subgroup-specific%20drug%20vulnerabilities.%20Oncogene42,%201661%E2%80%931671%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Kim, S.-Y., van de Wetering, M., Clevers, H. &amp; Sanders, K. The future of tumor organoids in precision therapy. <em>Trends Cancer</em>. 10.1016/j.trecan.2025.03.005 (2025).</cite> [<a href="https://doi.org/10.1016/j.trecan.2025.03.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40185656/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Driehuis, E. et al. Pancreatic cancer organoids recapitulate disease and allow personalized drug screening. <em>Proc. Natl. Acad. Sci. USA</em><strong>116</strong>, 26580–26590 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1911273116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6936689/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31818951/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Driehuis,%20E.%20et%20al.%20Pancreatic%20cancer%20organoids%20recapitulate%20disease%20and%20allow%20personalized%20drug%20screening.%20Proc.%20Natl.%20Acad.%20Sci.%20USA116,%2026580%E2%80%9326590%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Vlachogiannis, G. et al. Patient-derived organoids model treatment response of metastatic gastrointestinal cancers. <em>Science</em><strong>359</strong>, 920–926 (2018).
</cite> [<a href="https://doi.org/10.1126/science.aao2774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6112415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29472484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vlachogiannis,%20G.%20et%20al.%20Patient-derived%20organoids%20model%20treatment%20response%20of%20metastatic%20gastrointestinal%20cancers.%20Science359,%20920%E2%80%93926%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Driehuis, E. et al. Oral mucosal organoids as a potential platform for personalized cancer therapy. <em>Cancer Discov.</em><strong>9</strong>, 852–871 (2019).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-18-1522" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31053628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Driehuis,%20E.%20et%20al.%20Oral%20mucosal%20organoids%20as%20a%20potential%20platform%20for%20personalized%20cancer%20therapy.%20Cancer%20Discov.9,%20852%E2%80%93871%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Ganesh, K. et al. A rectal cancer organoid platform to study individual responses to chemoradiation. <em>Nat. Med.</em><strong>25</strong>, 1607–1614 (2019).
</cite> [<a href="https://doi.org/10.1038/s41591-019-0584-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7385919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31591597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ganesh,%20K.%20et%20al.%20A%20rectal%20cancer%20organoid%20platform%20to%20study%20individual%20responses%20to%20chemoradiation.%20Nat.%20Med.25,%201607%E2%80%931614%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Ooft, S. N. et al. Patient-derived organoids can predict response to chemotherapy in metastatic colorectal cancer patients. <em>Sci. Transl. Med.</em><strong>11</strong>, eaay2574 (2019).
</cite> [<a href="https://doi.org/10.1126/scitranslmed.aay2574" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31597751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ooft,%20S.%20N.%20et%20al.%20Patient-derived%20organoids%20can%20predict%20response%20to%20chemotherapy%20in%20metastatic%20colorectal%20cancer%20patients.%20Sci.%20Transl.%20Med.11,%20eaay2574%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Tan, T. et al. Unified framework for patient-derived, tumor-organoid-based predictive testing of standard-of-care therapies in metastatic colorectal cancer. <em>Cell Rep. Med.</em><strong>4</strong>, 101335 (2023).
</cite> [<a href="https://doi.org/10.1016/j.xcrm.2023.101335" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10783557/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38118423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tan,%20T.%20et%20al.%20Unified%20framework%20for%20patient-derived,%20tumor-organoid-based%20predictive%20testing%20of%20standard-of-care%20therapies%20in%20metastatic%20colorectal%20cancer.%20Cell%20Rep.%20Med.4,%20101335%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Fujii, M. et al. A colorectal tumor organoid library demonstrates progressive loss of niche factor requirements during tumorigenesis. <em>Cell Stem Cell</em><strong>18</strong>, 827–838 (2016).
</cite> [<a href="https://doi.org/10.1016/j.stem.2016.04.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27212702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fujii,%20M.%20et%20al.%20A%20colorectal%20tumor%20organoid%20library%20demonstrates%20progressive%20loss%20of%20niche%20factor%20requirements%20during%20tumorigenesis.%20Cell%20Stem%20Cell18,%20827%E2%80%93838%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Liu, L., Yu, L., Li, Z., Li, W. &amp; Huang, W. Patient-derived organoid (PDO) platforms to facilitate clinical decision making. <em>J. Transl. Med.</em><strong>19</strong>, 40 (2021).
</cite> [<a href="https://doi.org/10.1186/s12967-020-02677-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7821720/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33478472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20L.,%20Yu,%20L.,%20Li,%20Z.,%20Li,%20W.%20&amp;%20Huang,%20W.%20Patient-derived%20organoid%20(PDO)%20platforms%20to%20facilitate%20clinical%20decision%20making.%20J.%20Transl.%20Med.19,%2040%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Hirt, C. K. et al. Drug screening and genome editing in human pancreatic cancer organoids identifies drug-gene interactions and candidates for off-label treatment. <em>Cell Genom.</em><strong>2</strong>, 100095 (2022).
</cite> [<a href="https://doi.org/10.1016/j.xgen.2022.100095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7612395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35187519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hirt,%20C.%20K.%20et%20al.%20Drug%20screening%20and%20genome%20editing%20in%20human%20pancreatic%20cancer%20organoids%20identifies%20drug-gene%20interactions%20and%20candidates%20for%20off-label%20treatment.%20Cell%20Genom.2,%20100095%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Yoshida, A. Ewing and Ewing-like sarcomas: a morphological guide through genetically-defined entities. <em>Pathol. Int</em><strong>73</strong>, 12–26 (2023).
</cite> [<a href="https://doi.org/10.1111/pin.13293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10107474/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36484765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshida,%20A.%20Ewing%20and%20Ewing-like%20sarcomas:%20a%20morphological%20guide%20through%20genetically-defined%20entities.%20Pathol.%20Int73,%2012%E2%80%9326%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Watson, S. et al. Transcriptomic definition of molecular subgroups of small round cell sarcomas. <em>J. Pathol.</em><strong>245</strong>, 29–40 (2018).
</cite> [<a href="https://doi.org/10.1002/path.5053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29431183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Watson,%20S.%20et%20al.%20Transcriptomic%20definition%20of%20molecular%20subgroups%20of%20small%20round%20cell%20sarcomas.%20J.%20Pathol.245,%2029%E2%80%9340%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Tsuda, Y. et al. Ewing sarcoma with FEV gene rearrangements is a rare subset with predilection for extraskeletal locations and aggressive behavior. <em>Genes Chromosomes Cancer</em><strong>59</strong>, 286–294 (2020).
</cite> [<a href="https://doi.org/10.1002/gcc.22828" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7135932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31756779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsuda,%20Y.%20et%20al.%20Ewing%20sarcoma%20with%20FEV%20gene%20rearrangements%20is%20a%20rare%20subset%20with%20predilection%20for%20extraskeletal%20locations%20and%20aggressive%20behavior.%20Genes%20Chromosomes%20Cancer59,%20286%E2%80%93294%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Hung, Y. P., Fletcher, C. D. &amp; Hornick, J. L. Evaluation of ETV4 and WT1 expression in CIC-rearranged sarcomas and histologic mimics. <em>Mod. Pathol.</em><strong>29</strong>, 1324–1334 (2016).
</cite> [<a href="https://doi.org/10.1038/modpathol.2016.140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27443513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hung,%20Y.%20P.,%20Fletcher,%20C.%20D.%20&amp;%20Hornick,%20J.%20L.%20Evaluation%20of%20ETV4%20and%20WT1%20expression%20in%20CIC-rearranged%20sarcomas%20and%20histologic%20mimics.%20Mod.%20Pathol.29,%201324%E2%80%931334%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Crompton, B. D. et al. The genomic landscape of pediatric Ewing sarcoma. <em>Cancer Discov.</em><strong>4</strong>, 1326–1341 (2014).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-13-1037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25186949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crompton,%20B.%20D.%20et%20al.%20The%20genomic%20landscape%20of%20pediatric%20Ewing%20sarcoma.%20Cancer%20Discov.4,%201326%E2%80%931341%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Brohl, A. S. et al. The genomic landscape of the Ewing Sarcoma family of tumors reveals recurrent STAG2 mutation. <em>PLoS Genet.</em><strong>10</strong>, e1004475 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4091782/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25010205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brohl,%20A.%20S.%20et%20al.%20The%20genomic%20landscape%20of%20the%20Ewing%20Sarcoma%20family%20of%20tumors%20reveals%20recurrent%20STAG2%20mutation.%20PLoS%20Genet.10,%20e1004475%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Grünewald, T. G. et al. Chimeric EWSR1-FLI1 regulates the Ewing sarcoma susceptibility gene EGR2 via a GGAA microsatellite. <em>Nat. Genet.</em><strong>47</strong>, 1073–1078 (2015).
</cite> [<a href="https://doi.org/10.1038/ng.3363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4591073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26214589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gr%C3%BCnewald,%20T.%20G.%20et%20al.%20Chimeric%20EWSR1-FLI1%20regulates%20the%20Ewing%20sarcoma%20susceptibility%20gene%20EGR2%20via%20a%20GGAA%20microsatellite.%20Nat.%20Genet.47,%201073%E2%80%931078%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Riskjell, A. I., Makinen, V. N., Sandfeld-Paulsen, B. &amp; Aggerholm-Pedersen, N. Targeted treatment of soft-tissue sarcoma. <em>J. Personal. Med.</em><strong>13</strong>, 730 (2023).</cite> [<a href="https://doi.org/10.3390/jpm13050730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10220720/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37240900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Riskjell,%20A.%20I.,%20Makinen,%20V.%20N.,%20Sandfeld-Paulsen,%20B.%20&amp;%20Aggerholm-Pedersen,%20N.%20Targeted%20treatment%20of%20soft-tissue%20sarcoma.%20J.%20Personal.%20Med.13,%20730%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Smith, S. C. et al. CIC-DUX sarcomas demonstrate frequent MYC amplification and ETS-family transcription factor expression. <em>Mod. Pathol.</em><strong>28</strong>, 57–68 (2015).
</cite> [<a href="https://doi.org/10.1038/modpathol.2014.83" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24947144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20S.%20C.%20et%20al.%20CIC-DUX%20sarcomas%20demonstrate%20frequent%20MYC%20amplification%20and%20ETS-family%20transcription%20factor%20expression.%20Mod.%20Pathol.28,%2057%E2%80%9368%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Bui, N. Q. et al. A clinico-genomic analysis of soft tissue sarcoma patients reveals CDKN2A deletion as a biomarker for poor prognosis. <em>Clin. Sarcoma Res.</em><strong>9</strong>, 12 (2019).
</cite> [<a href="https://doi.org/10.1186/s13569-019-0122-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6739971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31528332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bui,%20N.%20Q.%20et%20al.%20A%20clinico-genomic%20analysis%20of%20soft%20tissue%20sarcoma%20patients%20reveals%20CDKN2A%20deletion%20as%20a%20biomarker%20for%20poor%20prognosis.%20Clin.%20Sarcoma%20Res.9,%2012%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Lin, D. I. et al. Genomic profiling of BCOR-rearranged uterine sarcomas reveals novel gene fusion partners, frequent CDK4 amplification and CDKN2A loss. <em>Gynecol. Oncol.</em><strong>157</strong>, 357–366 (2020).
</cite> [<a href="https://doi.org/10.1016/j.ygyno.2020.02.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32156473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20D.%20I.%20et%20al.%20Genomic%20profiling%20of%20BCOR-rearranged%20uterine%20sarcomas%20reveals%20novel%20gene%20fusion%20partners,%20frequent%20CDK4%20amplification%20and%20CDKN2A%20loss.%20Gynecol.%20Oncol.157,%20357%E2%80%93366%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Liu, K. X. et al. Risk stratification by somatic mutation burden in Ewing sarcoma. <em>Cancer</em><strong>125</strong>, 1357–1364 (2019).
</cite> [<a href="https://doi.org/10.1002/cncr.31919" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30602061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20K.%20X.%20et%20al.%20Risk%20stratification%20by%20somatic%20mutation%20burden%20in%20Ewing%20sarcoma.%20Cancer125,%201357%E2%80%931364%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Sondka, Z. et al. COSMIC: a curated database of somatic variants and clinical data for cancer. <em>Nucleic Acids Res.</em><strong>52</strong>, D1210–D1217 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkad986" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38183204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sondka,%20Z.%20et%20al.%20COSMIC:%20a%20curated%20database%20of%20somatic%20variants%20and%20clinical%20data%20for%20cancer.%20Nucleic%20Acids%20Res.52,%20D1210%E2%80%93D1217%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Haas, B. J. et al. Accuracy assessment of fusion transcript detection via read-mapping and de novo fusion transcript assembly-based methods. <em>Genome Biol.</em><strong>20</strong>, 213 (2019).
</cite> [<a href="https://doi.org/10.1186/s13059-019-1842-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6802306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31639029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haas,%20B.%20J.%20et%20al.%20Accuracy%20assessment%20of%20fusion%20transcript%20detection%20via%20read-mapping%20and%20de%20novo%20fusion%20transcript%20assembly-based%20methods.%20Genome%20Biol.20,%20213%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Doll, S. et al. Proteomic characterization of undifferentiated small round cell sarcomas with EWSR1 and CIC::DUX4 translocations reveals diverging tumor biology and distinct diagnostic markers. <em>Mod. Pathol.</em><strong>37</strong>, 100511 (2024).
</cite> [<a href="https://doi.org/10.1016/j.modpat.2024.100511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38705279/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Doll,%20S.%20et%20al.%20Proteomic%20characterization%20of%20undifferentiated%20small%20round%20cell%20sarcomas%20with%20EWSR1%20and%20CIC::DUX4%20translocations%20reveals%20diverging%20tumor%20biology%20and%20distinct%20diagnostic%20markers.%20Mod.%20Pathol.37,%20100511%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Wei, G. H. et al. Genome-wide analysis of ETS-family DNA-binding in vitro and in vivo. <em>EMBO J.</em><strong>29</strong>, 2147–2160 (2010).
</cite> [<a href="https://doi.org/10.1038/emboj.2010.106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2905244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20517297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei,%20G.%20H.%20et%20al.%20Genome-wide%20analysis%20of%20ETS-family%20DNA-binding%20in%20vitro%20and%20in%20vivo.%20EMBO%20J.29,%202147%E2%80%932160%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Riggi, N. et al. EWS-FLI1 utilizes divergent chromatin remodeling mechanisms to directly activate or repress enhancer elements in Ewing sarcoma. <em>Cancer Cell</em><strong>26</strong>, 668–681 (2014).
</cite> [<a href="https://doi.org/10.1016/j.ccell.2014.10.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4492343/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25453903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Riggi,%20N.%20et%20al.%20EWS-FLI1%20utilizes%20divergent%20chromatin%20remodeling%20mechanisms%20to%20directly%20activate%20or%20repress%20enhancer%20elements%20in%20Ewing%20sarcoma.%20Cancer%20Cell26,%20668%E2%80%93681%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Lin, Y. K., Wu, W., Ponce, R. K., Kim, J. W. &amp; Okimoto, R. A. Negative MAPK-ERK regulation sustains CIC-DUX4 oncoprotein expression in undifferentiated sarcoma. <em>Proc. Natl. Acad. Sci. USA</em><strong>117</strong>, 20776–20784 (2020).
</cite> [<a href="https://doi.org/10.1073/pnas.2009137117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7456153/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32788348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20Y.%20K.,%20Wu,%20W.,%20Ponce,%20R.%20K.,%20Kim,%20J.%20W.%20&amp;%20Okimoto,%20R.%20A.%20Negative%20MAPK-ERK%20regulation%20sustains%20CIC-DUX4%20oncoprotein%20expression%20in%20undifferentiated%20sarcoma.%20Proc.%20Natl.%20Acad.%20Sci.%20USA117,%2020776%E2%80%9320784%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Luo, Y. et al. New developments on the Encyclopedia of DNA Elements (ENCODE) data portal. <em>Nucleic Acids Res.</em><strong>48</strong>, D882–d889 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkz1062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7061942/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31713622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luo,%20Y.%20et%20al.%20New%20developments%20on%20the%20Encyclopedia%20of%20DNA%20Elements%20(ENCODE)%20data%20portal.%20Nucleic%20Acids%20Res.48,%20D882%E2%80%93d889%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Hu-Lieskovan, S. et al. EWS-FLI1 fusion protein up-regulates critical genes in neural crest development and is responsible for the observed phenotype of Ewing’s family of tumors. <em>Cancer Res.</em><strong>65</strong>, 4633–4644 (2005).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-04-2857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15930281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu-Lieskovan,%20S.%20et%20al.%20EWS-FLI1%20fusion%20protein%20up-regulates%20critical%20genes%20in%20neural%20crest%20development%20and%20is%20responsible%20for%20the%20observed%20phenotype%20of%20Ewing%E2%80%99s%20family%20of%20tumors.%20Cancer%20Res.65,%204633%E2%80%934644%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>McGrogan, B. T., Gilmartin, B., Carney, D. N. &amp; McCann, A. Taxanes, microtubules and chemoresistant breast cancer. <em>Biochim. Biophys. Acta</em><strong>1785</strong>, 96–132 (2008).
</cite> [<a href="https://doi.org/10.1016/j.bbcan.2007.10.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18068131/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McGrogan,%20B.%20T.,%20Gilmartin,%20B.,%20Carney,%20D.%20N.%20&amp;%20McCann,%20A.%20Taxanes,%20microtubules%20and%20chemoresistant%20breast%20cancer.%20Biochim.%20Biophys.%20Acta1785,%2096%E2%80%93132%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Wang, B. et al. ATXN1L, CIC, and ETS transcription factors modulate sensitivity to MAPK pathway inhibition. <em>Cell Rep.</em><strong>18</strong>, 1543–1557 (2017).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.01.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5313047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28178529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20B.%20et%20al.%20ATXN1L,%20CIC,%20and%20ETS%20transcription%20factors%20modulate%20sensitivity%20to%20MAPK%20pathway%20inhibition.%20Cell%20Rep.18,%201543%E2%80%931557%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Hua, X.-Y., Bie, X.-X., Cheng, X. &amp; Zhang, S.-G. High expression of CIN85 promotes proliferation and invasion of human esophageal squamous cell carcinoma. <em>Mol. Med. Rep.</em><strong>23</strong>, 12 (2021).
</cite> [<a href="https://doi.org/10.3892/mmr.2020.11650" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7673327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33179079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hua,%20X.-Y.,%20Bie,%20X.-X.,%20Cheng,%20X.%20&amp;%20Zhang,%20S.-G.%20High%20expression%20of%20CIN85%20promotes%20proliferation%20and%20invasion%20of%20human%20esophageal%20squamous%20cell%20carcinoma.%20Mol.%20Med.%20Rep.23,%2012%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Gaspar, N. et al. Ewing sarcoma: current management and future approaches through collaboration. <em>J. Clin. Oncol.</em><strong>33</strong>, 3036–3046 (2015).
</cite> [<a href="https://doi.org/10.1200/JCO.2014.59.5256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26304893/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gaspar,%20N.%20et%20al.%20Ewing%20sarcoma:%20current%20management%20and%20future%20approaches%20through%20collaboration.%20J.%20Clin.%20Oncol.33,%203036%E2%80%933046%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Whelan, J. et al. Interval compressed vincristine, doxorubicin, cyclophosphamide alternating with ifosfamide, etoposide in patients with advanced Ewing’s and other small round cell sarcomas. <em>Clin. Sarcoma Res</em><strong>2</strong>, 12 (2012).
</cite> [<a href="https://doi.org/10.1186/2045-3329-2-12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3571911/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22998944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whelan,%20J.%20et%20al.%20Interval%20compressed%20vincristine,%20doxorubicin,%20cyclophosphamide%20alternating%20with%20ifosfamide,%20etoposide%20in%20patients%20with%20advanced%20Ewing%E2%80%99s%20and%20other%20small%20round%20cell%20sarcomas.%20Clin.%20Sarcoma%20Res2,%2012%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Connolly, E. A. et al. Systemic treatments and outcomes in CIC-rearranged Sarcoma: a national multi-centre clinicopathological series and literature review. <em>Cancer Med.</em><strong>11</strong>, 1805–1816 (2022).
</cite> [<a href="https://doi.org/10.1002/cam4.4580" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9041083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35178869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Connolly,%20E.%20A.%20et%20al.%20Systemic%20treatments%20and%20outcomes%20in%20CIC-rearranged%20Sarcoma:%20a%20national%20multi-centre%20clinicopathological%20series%20and%20literature%20review.%20Cancer%20Med.11,%201805%E2%80%931816%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Antonescu, C. R. et al. Sarcomas With CIC-rearrangements are a distinct pathologic entity with aggressive outcome: a clinicopathologic and molecular study of 115 cases. <em>Am. J. Surg. Pathol.</em><strong>41</strong>, 941–949 (2017).
</cite> [<a href="https://doi.org/10.1097/PAS.0000000000000846" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5468475/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28346326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Antonescu,%20C.%20R.%20et%20al.%20Sarcomas%20With%20CIC-rearrangements%20are%20a%20distinct%20pathologic%20entity%20with%20aggressive%20outcome:%20a%20clinicopathologic%20and%20molecular%20study%20of%20115%20cases.%20Am.%20J.%20Surg.%20Pathol.41,%20941%E2%80%93949%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Murphy, J. et al. Clinical outcomes of patients with CIC-rearranged sarcoma: a single institution retrospective analysis. <em>J. Cancer Res. Clin. Oncol.</em><strong>150</strong>, 112 (2024).
</cite> [<a href="https://doi.org/10.1007/s00432-024-05631-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10912249/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38436779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murphy,%20J.%20et%20al.%20Clinical%20outcomes%20of%20patients%20with%20CIC-rearranged%20sarcoma:%20a%20single%20institution%20retrospective%20analysis.%20J.%20Cancer%20Res.%20Clin.%20Oncol.150,%20112%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Prieur, A., Tirode, F., Cohen, P. &amp; Delattre, O. EWS/FLI-1 silencing and gene profiling of ewing cells reveal downstream oncogenic pathways and a crucial role for repression of insulin-like growth factor binding protein 3. <em>Mol. Cell. Biol.</em><strong>24</strong>, 7275–7283 (2004).
</cite> [<a href="https://doi.org/10.1128/MCB.24.16.7275-7283.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC479730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15282325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prieur,%20A.,%20Tirode,%20F.,%20Cohen,%20P.%20&amp;%20Delattre,%20O.%20EWS/FLI-1%20silencing%20and%20gene%20profiling%20of%20ewing%20cells%20reveal%20downstream%20oncogenic%20pathways%20and%20a%20crucial%20role%20for%20repression%20of%20insulin-like%20growth%20factor%20binding%20protein%203.%20Mol.%20Cell.%20Biol.24,%207275%E2%80%937283%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Mateo-Lozano, S., Tirado, O. M. &amp; Notario, V. Rapamycin induces the fusion-type independent downregulation of the EWS/FLI-1 proteins and inhibits Ewing’s sarcoma cell proliferation. <em>Oncogene</em><strong>22</strong>, 9282–9287 (2003).
</cite> [<a href="https://doi.org/10.1038/sj.onc.1207081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14681687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mateo-Lozano,%20S.,%20Tirado,%20O.%20M.%20&amp;%20Notario,%20V.%20Rapamycin%20induces%20the%20fusion-type%20independent%20downregulation%20of%20the%20EWS/FLI-1%20proteins%20and%20inhibits%20Ewing%E2%80%99s%20sarcoma%20cell%20proliferation.%20Oncogene22,%209282%E2%80%939287%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Oyama, R. et al. Generation of novel patient-derived CIC-DUX4 sarcoma xenografts and cell lines. <em>Sci. Rep.</em><strong>7</strong>, 4712 (2017).
</cite> [<a href="https://doi.org/10.1038/s41598-017-04967-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5498486/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28680140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oyama,%20R.%20et%20al.%20Generation%20of%20novel%20patient-derived%20CIC-DUX4%20sarcoma%20xenografts%20and%20cell%20lines.%20Sci.%20Rep.7,%204712%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Nakai, S. et al. Establishment of a novel human CIC-DUX(4) sarcoma cell line, Kitra-SRS, with autocrine IGF-1R activation and metastatic potential to the lungs. <em>Sci. Rep.</em><strong>9</strong>, 15812 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-52143-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6825133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31676869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakai,%20S.%20et%20al.%20Establishment%20of%20a%20novel%20human%20CIC-DUX(4)%20sarcoma%20cell%20line,%20Kitra-SRS,%20with%20autocrine%20IGF-1R%20activation%20and%20metastatic%20potential%20to%20the%20lungs.%20Sci.%20Rep.9,%2015812%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Fevre, R. et al. Combinatorial drug screening on 3D Ewing sarcoma spheroids using droplet-based microfluidics. <em>iScience</em><strong>26</strong>, 106651 (2023).
</cite> [<a href="https://doi.org/10.1016/j.isci.2023.106651" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10165258/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37168549/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fevre,%20R.%20et%20al.%20Combinatorial%20drug%20screening%20on%203D%20Ewing%20sarcoma%20spheroids%20using%20droplet-based%20microfluidics.%20iScience26,%20106651%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Nitsch, A. et al. Effective combination of cold physical plasma and chemotherapy against Ewing sarcoma cells in vitro. <em>Sci. Rep.</em><strong>14</strong>, 6505 (2024).
</cite> [<a href="https://doi.org/10.1038/s41598-024-56985-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10948386/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38499701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nitsch,%20A.%20et%20al.%20Effective%20combination%20of%20cold%20physical%20plasma%20and%20chemotherapy%20against%20Ewing%20sarcoma%20cells%20in%20vitro.%20Sci.%20Rep.14,%206505%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Guillen, K. P. et al. A human breast cancer-derived xenograft and organoid platform for drug discovery and precision oncology. <em>Nat. Cancer</em><strong>3</strong>, 232–250 (2022).
</cite> [<a href="https://doi.org/10.1038/s43018-022-00337-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8882468/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35221336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guillen,%20K.%20P.%20et%20al.%20A%20human%20breast%20cancer-derived%20xenograft%20and%20organoid%20platform%20for%20drug%20discovery%20and%20precision%20oncology.%20Nat.%20Cancer3,%20232%E2%80%93250%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Karkampouna, S. et al. Patient-derived xenografts and organoids model therapy response in prostate cancer. <em>Nat. Commun.</em><strong>12</strong>, 1117 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-21300-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7892572/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33602919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karkampouna,%20S.%20et%20al.%20Patient-derived%20xenografts%20and%20organoids%20model%20therapy%20response%20in%20prostate%20cancer.%20Nat.%20Commun.12,%201117%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Gao, J. et al. Promising preclinical patient-derived organoid (PDO) and xenograft (PDX) models in upper gastrointestinal cancers: progress and challenges. <em>BMC Cancer</em><strong>23</strong>, 1205 (2023).
</cite> [<a href="https://doi.org/10.1186/s12885-023-11434-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10702130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38062430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20J.%20et%20al.%20Promising%20preclinical%20patient-derived%20organoid%20(PDO)%20and%20xenograft%20(PDX)%20models%20in%20upper%20gastrointestinal%20cancers:%20progress%20and%20challenges.%20BMC%20Cancer23,%201205%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Kim, J., Koo, B.-K. &amp; Knoblich, J. A. Human organoids: model systems for human biology and medicine. <em>Nat. Rev. Mol. Cell Biol.</em><strong>21</strong>, 571–584 (2020).
</cite> [<a href="https://doi.org/10.1038/s41580-020-0259-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7339799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32636524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20J.,%20Koo,%20B.-K.%20&amp;%20Knoblich,%20J.%20A.%20Human%20organoids:%20model%20systems%20for%20human%20biology%20and%20medicine.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.21,%20571%E2%80%93584%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Ohali, A. et al. High frequency of genomic instability in Ewing family of tumors. <em>Cancer Genet. Cytogenet.</em><strong>150</strong>, 50–56 (2004).
</cite> [<a href="https://doi.org/10.1016/j.cancergencyto.2003.08.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15041223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ohali,%20A.%20et%20al.%20High%20frequency%20of%20genomic%20instability%20in%20Ewing%20family%20of%20tumors.%20Cancer%20Genet.%20Cytogenet.150,%2050%E2%80%9356%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Casey, D. L. et al. TP53 mutations increase radioresistance in rhabdomyosarcoma and Ewing sarcoma. <em>Br. J. Cancer</em><strong>125</strong>, 576–581 (2021).
</cite> [<a href="https://doi.org/10.1038/s41416-021-01438-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8368014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34017087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Casey,%20D.%20L.%20et%20al.%20TP53%20mutations%20increase%20radioresistance%20in%20rhabdomyosarcoma%20and%20Ewing%20sarcoma.%20Br.%20J.%20Cancer125,%20576%E2%80%93581%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Lazo de la Vega, L. et al. Targeted next-generation sequencing of CIC-DUX4 soft tissue sarcomas demonstrates low mutational burden and recurrent chromosome 1p loss. <em>Hum. Pathol.</em><strong>58</strong>, 161–170 (2016).
</cite> [<a href="https://doi.org/10.1016/j.humpath.2016.09.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5125884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27664537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lazo%20de%20la%20Vega,%20L.%20et%20al.%20Targeted%20next-generation%20sequencing%20of%20CIC-DUX4%20soft%20tissue%20sarcomas%20demonstrates%20low%20mutational%20burden%20and%20recurrent%20chromosome%201p%20loss.%20Hum.%20Pathol.58,%20161%E2%80%93170%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Roerink, S. F. et al. Intra-tumour diversification in colorectal cancer at the single-cell level. <em>Nature</em><strong>556</strong>, 457–462 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0024-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29643510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roerink,%20S.%20F.%20et%20al.%20Intra-tumour%20diversification%20in%20colorectal%20cancer%20at%20the%20single-cell%20level.%20Nature556,%20457%E2%80%93462%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Duffaud, F. et al. Regorafenib in patients with advanced Ewing sarcoma: results of a non-comparative, randomised, double-blind, placebo-controlled, multicentre Phase II study. <em>Br. J. Cancer</em><strong>129</strong>, 1940–1948 (2023).
</cite> [<a href="https://doi.org/10.1038/s41416-023-02413-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10703915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37914801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duffaud,%20F.%20et%20al.%20Regorafenib%20in%20patients%20with%20advanced%20Ewing%20sarcoma:%20results%20of%20a%20non-comparative,%20randomised,%20double-blind,%20placebo-controlled,%20multicentre%20Phase%20II%20study.%20Br.%20J.%20Cancer129,%201940%E2%80%931948%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Lee, K. M. et al. MYC and MCL1 cooperatively promote chemotherapy-resistant breast cancer stem cells via regulation of mitochondrial oxidative phosphorylation. <em>Cell Metab.</em><strong>26</strong>, 633–647.e637 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cmet.2017.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5650077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28978427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20K.%20M.%20et%20al.%20MYC%20and%20MCL1%20cooperatively%20promote%20chemotherapy-resistant%20breast%20cancer%20stem%20cells%20via%20regulation%20of%20mitochondrial%20oxidative%20phosphorylation.%20Cell%20Metab.26,%20633%E2%80%93647.e637%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Yuda, J. et al. Selective MCL-1 inhibitor ABBV-467 is efficacious in tumor models but is associated with cardiac troponin increases in patients. <em>Commun. Med.</em><strong>3</strong>, 154 (2023).
</cite> [<a href="https://doi.org/10.1038/s43856-023-00380-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10600239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37880389/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuda,%20J.%20et%20al.%20Selective%20MCL-1%20inhibitor%20ABBV-467%20is%20efficacious%20in%20tumor%20models%20but%20is%20associated%20with%20cardiac%20troponin%20increases%20in%20patients.%20Commun.%20Med.3,%20154%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Wang, X. et al. Deletion of MCL-1 causes lethal cardiac failure and mitochondrial dysfunction. <em>Genes Dev.</em><strong>27</strong>, 1351–1364 (2013).
</cite> [<a href="https://doi.org/10.1101/gad.215855.113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3701191/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23788622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20X.%20et%20al.%20Deletion%20of%20MCL-1%20causes%20lethal%20cardiac%20failure%20and%20mitochondrial%20dysfunction.%20Genes%20Dev.27,%201351%E2%80%931364%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Grissenberger, S. et al. High-content drug screening in zebrafish xenografts reveals high efficacy of dual MCL-1/BCL-XL inhibition against Ewing sarcoma. <em>Cancer Lett.</em><strong>554</strong>, 216028 (2023).
</cite> [<a href="https://doi.org/10.1016/j.canlet.2022.216028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36462556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grissenberger,%20S.%20et%20al.%20High-content%20drug%20screening%20in%20zebrafish%20xenografts%20reveals%20high%20efficacy%20of%20dual%20MCL-1/BCL-XL%20inhibition%20against%20Ewing%20sarcoma.%20Cancer%20Lett.554,%20216028%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Shen, R. &amp; Seshan, V. E. FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing. <em>Nucleic Acids Res.</em><strong>44</strong>, e131 (2016).
</cite> [<a href="https://doi.org/10.1093/nar/gkw520" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5027494/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27270079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen,%20R.%20&amp;%20Seshan,%20V.%20E.%20FACETS:%20allele-specific%20copy%20number%20and%20clonal%20heterogeneity%20analysis%20tool%20for%20high-throughput%20DNA%20sequencing.%20Nucleic%20Acids%20Res.44,%20e131%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Gillis, S. &amp; Roth, A. PyClone-VI: scalable inference of clonal population structures using whole genome data. <em>BMC Bioinform.</em><strong>21</strong>, 571 (2020).</cite> [<a href="https://doi.org/10.1186/s12859-020-03919-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7730797/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33302872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gillis,%20S.%20&amp;%20Roth,%20A.%20PyClone-VI:%20scalable%20inference%20of%20clonal%20population%20structures%20using%20whole%20genome%20data.%20BMC%20Bioinform.21,%20571%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Hehir-Kwa, J. Y. et al. Improved gene fusion detection in childhood cancer diagnostics using RNA sequencing. <em>JCO Precis Oncol.</em><strong>6</strong>, e2000504 (2022).
</cite> [<a href="https://doi.org/10.1200/PO.20.00504" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8830514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35085008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hehir-Kwa,%20J.%20Y.%20et%20al.%20Improved%20gene%20fusion%20detection%20in%20childhood%20cancer%20diagnostics%20using%20RNA%20sequencing.%20JCO%20Precis%20Oncol.6,%20e2000504%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (383.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62673_MOESM2_ESM.pdf</a><sup> (41.1KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 1</a><sup> (13.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 2</a><sup> (11.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 3</a><sup> (12.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 4</a><sup> (25.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Dataset 5</a><sup> (10.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (86.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (656.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371069/bin/41467_2025_62673_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (26.8MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The transcriptomic and genomic data are deposited in the EGA study database under accession number <a href="https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fega-archive.org%2Fsearch%2FEGAS00001007941&amp;data=05%7C02%7Cf.c.a.ringnalda%40prinsesmaximacentrum.nl%7C7205b98651d24b80db2108ddca8e4164%7Ced9f3ca1097f493ea1295002e9c16d65%7C0%7C0%7C638889434506777356%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&amp;sdata=wWiht6B5IvmEtVanWoFZsXYgj3LyfaT%2B%2BzRil%2BRZCmo%3D&amp;reserved=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EGAS00001007941</a>, RNA-sequencing data is available under accession number <a href="https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fega-archive.org%2Fsearch%2FEGAD00001015419&amp;data=05%7C02%7Cf.c.a.ringnalda%40prinsesmaximacentrum.nl%7C7205b98651d24b80db2108ddca8e4164%7Ced9f3ca1097f493ea1295002e9c16d65%7C0%7C0%7C638889434506833488%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&amp;sdata=Cq9eKLlCpW7hS0EnLNir6e2Nz%2BDKxAqBAT06qzChAZg%3D&amp;reserved=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EGAD00001015419</a>, WGS data is available under accession number <a href="https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fega-archive.org%2Fsearch%2FEGAD00001015418&amp;data=05%7C02%7Cf.c.a.ringnalda%40prinsesmaximacentrum.nl%7C7205b98651d24b80db2108ddca8e4164%7Ced9f3ca1097f493ea1295002e9c16d65%7C0%7C0%7C638889434506874051%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&amp;sdata=EgQ7vy2R15%2FGwrGnkuDHshKNGRob%2BKyLrwVmCG2HlCE%3D&amp;reserved=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EGAD00001015418</a> and WXS data is available under accession number <a href="https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fega-archive.org%2Fsearch%2FEGAD00001015417&amp;data=05%7C02%7Cf.c.a.ringnalda%40prinsesmaximacentrum.nl%7C7205b98651d24b80db2108ddca8e4164%7Ced9f3ca1097f493ea1295002e9c16d65%7C0%7C0%7C638889434506900494%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&amp;sdata=OSqd4W44bgMikaJpaZq%2B1t1l5MYpDgwIHM1Md6QBOjY%3D&amp;reserved=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EGAD00001015417</a>. Datasets mentioned above are available under restricted access due to patient privacy and regulatory requirements. Access to the sequencing data is managed by the Biobank and Data Access Committee (BDAC) of the Princess Máxima Center. All researchers can request access by submitting a project proposal to the BDAC (biobank@prinsesmaximacentrum.nl). Requests are typically reviewed within ~2 weeks. The duration of access will be determined by the BDAC. The ChIP-seq publicly available dataset used in this study were accessed from ENCODE experiments <a href="https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.encodeproject.org%2Ffiles%2FENCFF186JKG%2F&amp;data=05%7C02%7Cf.c.a.ringnalda%40prinsesmaximacentrum.nl%7C7205b98651d24b80db2108ddca8e4164%7Ced9f3ca1097f493ea1295002e9c16d65%7C0%7C0%7C638889434506923626%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&amp;sdata=pYuH%2FhFffMEBtYEWp%2BVSkRK1YVxuJ53GfqgKK4Fy2WQ%3D&amp;reserved=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ENCFF186JKG</a> [<a href="https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.encodeproject.org%2Ffiles%2FENCFF186JKG%2F&amp;data=05%7C02%7Cf.c.a.ringnalda%40prinsesmaximacentrum.nl%7C7205b98651d24b80db2108ddca8e4164%7Ced9f3ca1097f493ea1295002e9c16d65%7C0%7C0%7C638889434506941686%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&amp;sdata=rO32szxEWFOzj2%2BvIkyHOzaBNB14Z6kuFUNHBfV3zks%3D&amp;reserved=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.encodeproject.org/files/ENCFF186JKG/</a>] for BCOR. Requests to use the tumoroid lines that were generated in this paper will be evaluated on a case-by-case basis by the BDAC of the Princess Máxima Center. When approved, specific material transfer agreements will be drawn up. To contact the BDAC, email biobank@prinsesmaximacentrum.nl. The remaining data are available within the Article, Supplementary Information or Source Data file. <a href="#Sec30" class="usa-link">Source data</a> are provided with this paper.</p>
<p>All code used to generate figures can be accessed on <a href="https://github.com/Hubrecht-Clevers/small_cell_sarcomas" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Hubrecht-Clevers/small_cell_sarcomas</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62673-2"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62673.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371069/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371069/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371069%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371069/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371069/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371069/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841360/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371069/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841360/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371069/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371069/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="mxxfUBiYOU6I64EokdKEEjLS0qB89ZJRonCXzbIOt34oOhKZCat81EybDGIqnZKJ">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
