
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Application of Rational Design and Molecular Metadynamics for the Estimation of Changes in Trans-Splicing Efficiency during the Mutagenesis of Ssp DnaE Intein - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4838B8AEEF72305838B001DF1CDC8.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsbmcau">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371501/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Bio &amp; Med Chem Au">
<meta name="citation_title" content="Application of Rational Design and Molecular Metadynamics for the Estimation of Changes in Trans-Splicing Efficiency during the Mutagenesis of Ssp DnaE Intein">
<meta name="citation_author" content="Matvei O Sabantsev">
<meta name="citation_author_institution" content="Gene Therapy Department, Research Center for Translational Medicine, Sirius University of Science and Technology, Sirius Federal Territory, Krasnodar Region 354349, Russia">
<meta name="citation_author" content="Andrew N Brovin">
<meta name="citation_author_institution" content="Gene Therapy Department, Research Center for Translational Medicine, Sirius University of Science and Technology, Sirius Federal Territory, Krasnodar Region 354349, Russia">
<meta name="citation_author" content="Maxim A Gureev">
<meta name="citation_author_institution" content="Laboratory of Bio- and Chemoinformatics, HSE University, Soyuza Pechatnikov Str.16, Saint-Petersburg 190121, Russia">
<meta name="citation_author" content="Yuri B Porozov">
<meta name="citation_author_institution" content="Laboratory of Bio- and Chemoinformatics, HSE University, Soyuza Pechatnikov Str.16, Saint-Petersburg 190121, Russia">
<meta name="citation_author" content="Sergey A Chuvpilo">
<meta name="citation_author_institution" content="Gene Therapy Department, Research Center for Translational Medicine, Sirius University of Science and Technology, Sirius Federal Territory, Krasnodar Region 354349, Russia">
<meta name="citation_author" content="Alexander V Karabelsky">
<meta name="citation_author_institution" content="Gene Therapy Department, Research Center for Translational Medicine, Sirius University of Science and Technology, Sirius Federal Territory, Krasnodar Region 354349, Russia">
<meta name="citation_publication_date" content="2025 Aug 8">
<meta name="citation_volume" content="5">
<meta name="citation_issue" content="4">
<meta name="citation_firstpage" content="738">
<meta name="citation_doi" content="10.1021/acsbiomedchemau.5c00091">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371501/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371501/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371501/pdf/bg5c00091.pdf">
<meta name="description" content="Currently, inteins are some of the most popular multifunctional tools in the fields of molecular biology and biotechnology. In this study, we used the surface analysis method to identify the sites of intermolecular interactions between the N and ...">
<meta name="og:title" content="Application of Rational Design and Molecular Metadynamics for the Estimation of Changes in Trans-Splicing Efficiency during the Mutagenesis of Ssp DnaE Intein">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Currently, inteins are some of the most popular multifunctional tools in the fields of molecular biology and biotechnology. In this study, we used the surface analysis method to identify the sites of intermolecular interactions between the N and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371501/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371501">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsbiomedchemau.5c00091"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/bg5c00091.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371501%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371501/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371501/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371501/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsbmcau.png" alt="ACS Bio &amp; Med Chem Au logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Bio &amp; Med Chem Au" title="Link to ACS Bio &amp; Med Chem Au" shape="default" href="https://pubs.acs.org/journal/abmcb8" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Bio Med Chem Au</button></div>. 2025 Aug 8;5(4):738–752. doi: <a href="https://doi.org/10.1021/acsbiomedchemau.5c00091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsbiomedchemau.5c00091</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Application
of Rational Design and Molecular Metadynamics
for the Estimation of Changes in Trans-Splicing Efficiency during
the Mutagenesis of Ssp DnaE Intein</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sabantsev%20MO%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Matvei O Sabantsev</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Matvei O Sabantsev</span></h3>
<div class="p">
<sup>†</sup>
Gene
Therapy Department, Research Center for Translational Medicine, Sirius University of Science and Technology, Sirius Federal Territory, Krasnodar Region
354349, Russia
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sabantsev%20MO%22%5BAuthor%5D" class="usa-link"><span class="name western">Matvei O Sabantsev</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brovin%20AN%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Andrew N Brovin</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Andrew N Brovin</span></h3>
<div class="p">
<sup>†</sup>
Gene
Therapy Department, Research Center for Translational Medicine, Sirius University of Science and Technology, Sirius Federal Territory, Krasnodar Region
354349, Russia
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brovin%20AN%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew N Brovin</span></a>
</div>
</div>
<sup>†,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gureev%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Maxim A Gureev</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Maxim A Gureev</span></h3>
<div class="p">
<sup>‡</sup>
Laboratory
of Bio- and Chemoinformatics, HSE University, Soyuza Pechatnikov Str.16, Saint-Petersburg
190121, Russia
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gureev%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Maxim A Gureev</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Porozov%20YB%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yuri B Porozov</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yuri B Porozov</span></h3>
<div class="p">
<sup>‡</sup>
Laboratory
of Bio- and Chemoinformatics, HSE University, Soyuza Pechatnikov Str.16, Saint-Petersburg
190121, Russia
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Porozov%20YB%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuri B Porozov</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chuvpilo%20SA%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Sergey A Chuvpilo</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Sergey A Chuvpilo</span></h3>
<div class="p">
<sup>†</sup>
Gene
Therapy Department, Research Center for Translational Medicine, Sirius University of Science and Technology, Sirius Federal Territory, Krasnodar Region
354349, Russia
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chuvpilo%20SA%22%5BAuthor%5D" class="usa-link"><span class="name western">Sergey A Chuvpilo</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Karabelsky%20AV%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Alexander V Karabelsky</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Alexander V Karabelsky</span></h3>
<div class="p">
<sup>†</sup>
Gene
Therapy Department, Research Center for Translational Medicine, Sirius University of Science and Technology, Sirius Federal Territory, Krasnodar Region
354349, Russia
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Karabelsky%20AV%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexander V Karabelsky</span></a>
</div>
</div>
<sup>†</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Gene
Therapy Department, Research Center for Translational Medicine, Sirius University of Science and Technology, Sirius Federal Territory, Krasnodar Region
354349, Russia
</div>
<div id="aff2">
<sup>‡</sup>
Laboratory
of Bio- and Chemoinformatics, HSE University, Soyuza Pechatnikov Str.16, Saint-Petersburg
190121, Russia
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>brovin.an@talantiuspeh.ru</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 3; Accepted 2025 Jul 18; Revised 2025 Jul 18; Collection date 2025 Aug 20.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371501</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Currently, inteins
are some of the most popular multifunctional
tools in the fields of molecular biology and biotechnology. In this
study, we used the surface analysis method to identify the sites of
intermolecular interactions between the N and C-parts of the Ssp DnaE
intein. The obtained results were used to determine the key amino
acids that define the binding energy and type of contact between intein
subunits. <em>In silico</em> substitution of five neutral
amino acids in the C-part of Ssp DnaE with methionine was validated
by using oligomutagenesis of a previously assembled plasmid, which
was then used for <em>in vitro</em> tests with HEK293 cells.
GFP reconstruction assays were used to estimate changes in trans-splicing
efficiency using quantitative metrics such as the number of GFP+ cells
and median fluorescence intensity as well as qualitative metrics such
as microphotography and fluorescence curve analysis using live-cell
microscopy. The results of the <em>in vitro</em> tests revealed
significantly decreased splicing efficiency in four out of six mutant
variants, with no significant differences in the other two cases.
Additionally, we performed metadynamics modeling to explain how these
mutations affect the molecular mechanisms of intein-intein interactions.
Finally, we found a positive correlation between the structural and
free energy changes in the local minima distribution and the decrease
in splicing efficiency in the I151M and A162M+A165M cases. The resulting
method was used with control mutations that had an experimentally
confirmed positive (A168H) or negative (T198A) effect on the splicing
reaction. In summary, we propose a method of free energy surface analysis
in collective variables for quick and visual evaluation of mutation
effects. This approach could be applied for the development of new
biotechnological and gene therapy products to overcome AAV capacity
limitations.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> inteins, protein trans-splicing, rational mutagenesis, protein−protein docking, molecular metadynamics
simulation, gene therapy</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9067/12371501/18adf94b254a/bg5c00091_0011.jpg" loading="lazy" height="371" width="628" alt="graphic file with name bg5c00091_0011.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9067/12371501/96b898c57485/bg5c00091_0009.jpg" loading="lazy" id="tgr1" height="371" width="628" alt="graphic file with name bg5c00091_0009.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Inteins have become a popular and multifunctional
tool in modern
molecular biology, facilitating various modifications in protein
engineering. This technology has been
primarily applied in high-efficiency protein purification and the
assembly of large protein complexes. Split
inteins represent a class of supporting proteins that perform trans-splicing
reactions with external proteins both <em>in vitro</em> and <em>in vivo</em>. The trans-splicing
mechanism enables the joining of cleaved protein segments, which are
fused with intein domains and encoded by separate DNA sequences, into
a single functional protein.</p>
<p>This separate expression resolves
several important biotechnological
problems, such as assembling of toxic proteins for cell producents
in the form of two inactive prespliced parts, separate delivery of genetic material to overcome the packaging
capacity of viral particles,
<em>in
vivo</em> modification of receptors, antibody assembly, etc.</p>
<p>In translational
medicine, the application of split inteins is
justified by the perspective of creating new enhanced treatments based
on a system of double recombinant adeno-associated viruses (AAV),
which can increase the packaging capacity of gene therapy drugs. This approach has significantly expanded the
treatment options for effective compensation of mutations in genes
encoding dystrophin protein (cDNA of <em>DMD</em> gene -14
kb), Usherin (cDNA of <em>USH2A</em> gene -18.9 kb), factor
VIII of blood coagulation (cDNA of <em>F8</em> gene 7 kb),
and approximately a thousand other mutations related to large human
genes. In addition to the separate delivery
of genetic material for the replacement therapy of large genes, inteins have been used in gene editing, particularly in the assembly of adenine<em>
−


</em> or cytosine<em>
−


</em> base editors (ABE, CBE) and prime editors<em>
−

</em> (PE). Current
trends in the application of inteins in gene therapy are presented
schematically in <a href="#fig1" class="usa-link">Figure 
</a>.</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371501_bg5c00091_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9067/12371501/fd547dec3971/bg5c00091_0001.jpg" loading="lazy" id="gr1" height="732" width="635" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Illustration of intein trans-splicing technology for treating inherited
diseases caused by mutations in large genes. Target large genes (<em>MYO7A, ABCA4, EYS</em>, etc.) are highlighted in italics with
the approximate size of protein-coding DNA.</p></figcaption></figure><p>To enhance the efficiency of protein assembly in
the above-mentioned
applications, researchers are exploring approaches to increase the
yield of protein trans-splicing reactions by modifying natural intein
sequences.<em>
,
</em> Key trends in this field include
the minimization of intein sequence, detailed
analysis of the structure and properties of the entire molecule and mutagenesis of individual amino acids that
directly affect the trans-splicing reaction.
</p>
<p>The process of identifying and incorporating mutations has
improved
significantly since the first intein structure was determined using
nuclear magnetic resonance (NMR). For instance, the resolution structures
of the Npu DnaE intein were obtained both before and after the splicing
reaction. In the same study, the authors
proposed a strategy of “rational” mutagenesis to optimize
the amino acid sequence of inteins to improve trans-splicing efficiency.
This approach improved the affinity and stability of the intein complexes,
which were accompanied by changes in the reaction kinetics between
the N and C parts. Further studies on intein structures have identified
the key role of the initial cysteine of the C-intein, which forms
a cysteidyl-peptide thioester that subsequently undergoes an acyl
shift. The investigation of intein interactions
has been limited to resource-intensive experimental methods. For instance,
the application of an expensive peptide synthesizer was a common approach
to determine the impact of each amino acid at the N and C inteins
ends and their immediate surroundings.<em>
,
</em>
</p>
<p>Advances
in structural biology methods, along with the development
of molecular modeling techniques, have enabled significantly more
comprehensive and detailed investigations into the structural and
functional properties of biomolecules. An outstanding example of such
research is the study by Stevens and colleagues, in which a synthetic consensus intein (Cfa) was engineered.
Artificially constructed Cfa has 3-fold higher kinetics of the splicing
reaction than the natural Npu DnaE intein. This enhanced activity
was attributed to its <em>de novo</em> design, which incorporated
“fast” functional domains derived from high-performance
natural inteins of the DnaE group. A similar approach involving point
mutations around the active center was used in the work by Eryilmaz
et al.
</p>
<p>The most extensively studied
split intein Ssp DnaE was selected
for the analysis. This intein catalyzes the trans-splicing of α-subunit
of DNA polymerase III across bacterial, archaeal and eukaryotic systems. Members of the DnaE intein group exhibit rapid
trans-splicing kinetics, efficiently reconstructing protein complexes
within seconds to minutes. These features make them useful tools
for protein engineering. However, Ssp DnaE has been identified as
one of the slowest inteins from the DnaE group, with a half-reaction
time of 35 min in comparison to 63 s for Npu DnaE. Nevertheless, the number of experimentally evaluated structures
for Ssp DnaE (RCSB PDB ID: <a href="https://www.rcsb.org/structure/1ZD7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1ZD7</a>, 1ZDE, 4GIG, and 3NZM) is considerably higher than
that for other proteins in this group. This fact establishes a substantial
background for the application of molecular modeling methods to predict
the properties of modified inteins. The combination of these factors
enables the application of rational design to identify mutagenesis
points and incorporate mutations that enhance the efficiency of the
trans-splicing reaction in comparison with the original Ssp DnaE pair.</p>
<p>The development and validation of reliable methods that would enable
the modification of inteins to increase trans-splicing efficiency
were set as the primary goals for this research. Ssp DnaE was used
as an accessible example for modeling and experimental testing. The
molecular surface of the intein Ssp DnaE was analyzed with the surface
accessible to the solvent (SASA) method to identify potential mutation
sites, which were incorporated into the intein nucleotide sequence.
The preliminary effects of these mutations were estimated by calculating
the free energy changes in affinity and stability. Following experimental
validation, a retrospective analysis of molecular dynamics simulations
was completed in order to explain the structural and energetic basis
of the observed outcomes. The approach developed in this work can
be used to modify any other intein for specific purposes.</p></section><section id="sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">Model Preparation</h3>
<p>For computational
modeling of protein–protein
interactions, the resolved structures of the intein Ssp DnaE (PDB
ID: <a href="https://www.rcsb.org/structure/1ZDE" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1ZDE</a>) and
the GFP protein (PDB ID: <a href="https://www.rcsb.org/structure/5B61" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">5B61</a>, chain D) were extracted from the PDB. Before the calculations, each model was prepared
using the Protein PrepWizard module of the Schrödinger Suite
2022-4 software package. During the preparation
process, water molecules were removed from the model, and the X-ray
structure analysis errors were corrected using the following steps:
bonding types between atoms were restored, missing hydrogen atoms
and amino acids were added, charges were corrected, and protonation
was adjusted. After preparation, the geometry of the protein structures
was optimized using restrained minimization (Prime, Schrödinger
Suite 2022-4). The following minimization
settings were used: solvation model, VSGB; force field model, OPLS4; optimization method, LBFGS, with five iterations
and 65 steps within one iteration. Quality control of the protein
structure was performed using a Ramachandran plot.
</p></section><section id="sec2.2"><h3 class="pmc_sec_title">Molecular Dynamics</h3>
<p>Molecular dynamics
simulations were
performed with the Desmond program to evaluate protein stability in
the solvent. The OPLS4 force field was employed for all calculations,
with the following system parameters: an orthorhombic simulation box,
a 15 Å buffer zone from the protein surface, and the TIP3P water
model as the solvent. The aqueous solvent was designed to mimic physiological
conditions (0.15 M NaCl). The system charge was neutralized by adding
Na<sup>+</sup> or Cl<sup>–</sup> counterions. Simulation parameters:
preliminary relaxation of the system was performed using a hybrid
steepest descent method (15 steps) and LBFGS algorithm (2000 steps)
with constraints on the dissolved component of the system, followed
by energy minimization without constraints for another 2000 steps.
Then, a series of short molecular dynamics simulations were performed
for 12 ps in the NVT ensemble at 10 K and in the NPT ensemble at 10
K, with and without constraints on non-hydrogen atoms. A subsequent
series of short molecular dynamics simulations of the system (24 ps)
at 300 K in the NPT ensemble with and without constraints on non-hydrogen
atoms. Main simulation: 100 ns, system integration every 2 ps, frame
recording every 10 ps, temperature: 310 K, NPT ensemble. Monitored
parameters: RMSD and elements of the protein’s secondary structure.
Stability violations of the protein were assessed by analyzing regions
with disrupted folding and abrupt changes in the RMSD magnitude.</p></section><section id="sec2.3"><h3 class="pmc_sec_title">Protein–Protein Docking</h3>
<p>The reconstructed and
stable model of the original intein and its complex with fused GFP
was used for protein–protein docking with the PIPER algorithm.
</p>
<p>A smaller protein, C-part of Ssp DnaE,
was used as an interactant to reduce computational complexity during
docking without specific constraints, such as repulsive or attractive
potentials of amino acid groups in the presence of specific protein–protein
recognition. In the docking results, 70,000 orientations were obtained,
and the top 70 energetically favorable solutions were selected by
clustering analysis.</p>
<p>Key selection criteria included pose score
(an analogue of binding
energy in molecular docking, considering the interaction energy of
contacting amino acids, kcal/mol) and pose energy (the energy of the
entire protein–protein complex, accounting for potential strain
that may arise outside the interface of the protein–protein
interaction, kcal/mol).</p></section><section id="sec2.4"><h3 class="pmc_sec_title">Protein Surface Analysis</h3>
<p>Analysis
of the molecular
surface of intein was used to identify the primary and secondary amino
acids that form the protein–protein interface. For surface
analysis, the intein with parts of GFP was split into two separate
chains. The “heavier” fragment with the N-part of the
intein was used in the calculations. The method was based on calculating
the molecular surface accessible to the solvent (SASA) and projecting
the potentials of free energy (Δ<em>G</em>) of non-bonded
interactions between atoms. The lipophilicity characteristics and
electrostatic potentials were calculated, and the reactive amino acids
were identified based on the solvent accessibility of their side chains.</p></section><section id="sec2.5"><h3 class="pmc_sec_title">Molecular Metadynamics</h3>
<p>The resulting molecular structures
were used to investigate the effects of mutagenesis on the conformation
of the studied proteins. The method of molecular metadynamics was
used to evaluate changes in the free energy of the system (Δ<em>G</em>) and to associate its values with conformational rearrangements
in the coordinates of the selected collective variables (CVs).</p>
<p>The following parameters were used for the metadynamic calculations:
all investigated protein models were placed in an orthorhombic region
with a buffer distance of 15 Å from the protein surface. A molecular
system simulating a physiological solution (0.15 M NaCl) based on
the TIP3P water model was used as the solvent. The charge of the proteins
was neutralized by the addition of appropriate counterions (Na<sup>+</sup> and Cl<sup>–</sup>). The OPLS4 force field was used
for the calculations.</p>
<p>The following collective variables were
used to calculate the free
energy surface</p>
<table class="disp-formula p" id="ueq1"><tr><td class="formula"><math id="d1e398" display="block"><mi>CV</mi><mn>1</mn><mo>:</mo><mi mathvariant="normal">B</mi><mo>:</mo><mi mathvariant="normal">D</mi><mn>150</mn><mo>−</mo><mi mathvariant="normal">C</mi><mn>170</mn><mo>↔</mo><mi mathvariant="normal">A</mi><mo>:</mo><mi mathvariant="normal">V</mi><mn>184</mn><mo>−</mo><mi mathvariant="normal">L</mi><mn>224</mn></math></td></tr></table>
<table class="disp-formula p" id="ueq2"><tr><td class="formula"><math id="d1e427" display="block"><mi>CV</mi><mn>2</mn><mo>:</mo><mi mathvariant="normal">B</mi><mo>:</mo><mi mathvariant="normal">D</mi><mn>150</mn><mo>−</mo><mi mathvariant="normal">C</mi><mn>170</mn><mo>↔</mo><mi mathvariant="normal">A</mi><mo>:</mo><mi mathvariant="normal">G</mi><mn>134</mn><mo>−</mo><mi mathvariant="normal">V</mi><mn>184</mn></math></td></tr></table>
<p>The centroids of the amino acid groups
were chosen to represent
the center of mass of the most stable elements of the protein structure.
The sequence between centroids is labile and undergoes conformational
changes during the splicing process. Therefore, the analysis of free
energy surfaces revealed changes in the geometry of intein during
the splicing reaction.</p>
<p>The total simulation time for metadynamics
was 200 ns with frame
recording at 10 ps and energy recording at 2 ps. The temperature was
310 K, pressure was 1 atm (1.013 bar), biasing potential was 0.03
kcal/mol, and potential input interval was 0.09 ps. Ensemble - NPT,
with mandatory preliminary relaxation of the entire system, was used.</p></section><section id="sec2.6"><h3 class="pmc_sec_title">Plasmid Cloning</h3>
<p>Basic plasmids for this study were
obtained during our previous work. Mutagenesis
of Ssp DnaE C was performed during the PCR with the Tersus Plus PCR
kit (PK221, Evrogen, Russia) and primers containing intein nucleotide
sequences with single or double mutations (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>). A typical PCR contained 4 μL of 5× Tersus Red
buffer, 0.5 μL of 10 mM dNTP mix, 0.5 μL of 50× Tersus
polymerase mix, 2 μL of 10 pM oligonucleotide mix, 12 μL
of dH<sub>2</sub>O, and 1 μL of DNA template. The following
amplification protocol was used for the PCR on a T100 thermal cycler
(1861096, BioRad, United States): initial denaturation at 95 °C
for 5 min, followed by 30 cycles of denaturation at 95 °C for
10 s, annealing at 60 °C for 20 s, and extension at 72 °C
for 30–60 s, depending on the size of the PCR product. The
amplified PCR products were purified with the CleanUp Mini Kit (BC023L,
Evrogen, Russia) according to the manufacturer’s protocol.
Purified constructs were cloned via <em>BamH</em>I and <em>Hind</em>III sites into the pAAV-CMV-MCS expression vector from
the AAV5 CMV Expression System Kit (VPK-410-SER5, Cell Biolabs, United
States). Positive bacterial clones were verified by PCR and Sanger
sequencing (ABI 3500, Thermo Fisher Scientific Inc., United States).</p></section><section id="sec2.7"><h3 class="pmc_sec_title">Transfection of HEK293</h3>
<p>The adherent HEK293 cell line
(85120602, ECACC, 2024) and cationic polymer polyethylenimine PEI-MAX
(24765, Polysciences Inc., United States) were used for transfection.
The transfection mixture was prepared according to the following protocol:
2.5 μg PEI (PEI/DNA molar ratio of 5:1) was dissolved in high
glucose DMEM (D5648, Sigma-Aldrich, Germany) and mixed with 0.25 μg
of each plasmid of inten’s pair (0.5 μg of total pDNA
per well). After 12 min of incubation for the generation of the PEI–DNA
complex, the transfection mixture was added to 2×10<sup>5</sup> trypsinized adherent cells in DMEM supplemented with 10% FBS (S181H-500,
Biowest, France). The suspension was gently mixed, and the transfected
cells were seeded in a 48-well plate in three biological replicates
per sample. After 24 h, the medium containing the transfection reagents
was replaced with 0.4 mL of fresh DMEM containing 5% FBS for a further
48-h incubation.</p></section><section id="sec2.8"><h3 class="pmc_sec_title">Live-Cell Microscopy</h3>
<p>Fluorescence
microscopy scanning
and analysis were performed with IncuCyte S3 (Sartorius AG, Germany).
The instrument was programmed with a scanning schedule of every 2
h in two optical channels (brightfield and green). The instrument’s
built-in software was used for qualitative and quantitative analysis
of fluorescence for each scan (number of GFP+ objects per image) and
in the dynamics of fluorescence accumulation.</p></section><section id="sec2.9"><h3 class="pmc_sec_title">Flow Cytometry</h3>
<p>After 48 h of incubation, cells were
trypsinized for flow cytometry analysis. For sample preparation, the
cells were washed twice with 500 μL of PBS and then resuspended
in 250 μL chilled FACS buffer (1× PBS, 5% FBS, 1 mM EDTA).
Data were collected using a CytoFlex B2-R2-V0 flow cytometer (A00-1-1102,
Beckman Coulter, United States) and analyzed with CytExpert v1.2 software
to detect GFP+ live cells. Live cells were gated based on the population
distribution by FSC-A and SSC-A. Single cells were gated based on
population distribution using the FSC-A/FSC-H plot, and the percentage
of GFP+ cells was determined by comparison with autofluorescence of
negative control cells without plasmid transfection. Further statistical
analyses for normality of sample distribution and confidence intervals
for rejection of the null hypothesis were performed using GraphPad
Prism 9.3.1 (one-way ANOVA with Sidak’s test for multiple comparisons).
Results are presented as the mean ± standard deviation of 3 biological
replicates. Confidence intervals are as follows: significant (*) <em>p</em>-value &lt;0.05, (**) <em>p</em>-value &lt;0.01,
(***) <em>p</em>-value &lt;0.001, (****) <em>p</em>-value &lt;0.0001; nonsignificant (ns) <em>p</em>-value &gt;0.05.</p></section></section><section id="sec3"><h2 class="pmc_sec_title">Results</h2>
<section id="sec3.1"><h3 class="pmc_sec_title">Model Preparation for Calculations</h3>
<p>The consensus model
of intein annealed with GFP split at 69 amino acids was constructed
from resolved structures of Ssp DnaE intein precursor (1ZDE) and GFP
(chain D in model 5B61). The selection of the trans-splicing site
for GFP disruption was previously experimentally confirmed and discussed
in our previous study.
</p>
<p>The stability
of the resulting intein model with GFP was checked by using molecular
dynamics. The calculations have confirmed the asymptotic nature of
the RMSD curve (<a href="#fig2" class="usa-link">Figure 
</a>), indicating that the protein was stable and did not undergo any
significant structural changes.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371501_bg5c00091_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9067/12371501/1d44df82e06f/bg5c00091_0002.jpg" loading="lazy" id="gr2" height="410" width="749" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Graph of root-mean-square deviation (RMSD)
changes during 200 ns
of molecular dynamics simulation with the GFP-intein resulting model.
The asymptotic curve of the RMSD has confirmed the stability of the
resulting intein model.</p></figcaption></figure><p>Protein–protein
docking was used to verify the ability of
the obtained composite structure to restore the original intein complex.
Initially, the Ssp DnaE intein model (PDB ID <a href="https://www.rcsb.org/structure/1ZDE" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1ZDE</a>) was manually divided
into N and C-terminal parts based on the amino acid sequence from
the Intein Database.
</p>
<p>Subsequently,
limited minimization of the protein structure was
performed to relax the side chains from conformational changes induced
by the interactant and normalize the hydrogen bonding network. The
separated parts of the intein were subjected to protein–protein
docking (<a href="#fig3" class="usa-link">Figure 
</a>).
The evaluation function metrics and clustering indicated that the
most energetically favorable configuration corresponds to the original
protein–protein complex (cluster size = 183, pose score = −773.41
kcal/mol, pose energy = −2887.77 kcal/mol). Thus, protein–protein
docking confirmed that the constructed Ssp DnaE intein was restored
to the original protein–protein complex (<a href="#fig3" class="usa-link">Figure 
</a> A, B).</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371501_bg5c00091_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9067/12371501/81e62af747ab/bg5c00091_0003.jpg" loading="lazy" id="gr3" height="732" width="627" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Reconstructed protein models: (A) original
intein complex from
1ZDE structure; (B) the result of protein–protein docking evaluation
of N- and C-terminal parts of Ssp DnaE intein; (C) complex of intein
fused with GFP split at 69/70 amino acid; (D) the resulting consensus
model after protein–protein docking of N and C-terminal parts
of intein Ssp DnaE fused with GFP.</p></figcaption></figure><p>Similarly, protein–protein docking was performed
with an
identical protocol to obtain a model of the intein fused with GFP
split into two parts. The results of the calculations are listed in <a href="#tbl1" class="usa-link">Table 
</a>. Variant 2 was selected
as the best solution in the protein–protein docking result,
with the highest similarity to the original protein–protein
complex (<a href="#fig3" class="usa-link">Figure 
</a>C,D).
Other variants also reproduce the original complex, but with lower
clusterization resolution.</p>
<section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">1. Results of the Top
5 Solutions after
Protein–Protein Docking<sup></sup>
.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9067/12371501/ce935ae276e6/bg5c00091_0007.jpg" loading="lazy" id="fx1" height="388" width="661" alt="graphic file with name bg5c00091_0007.jpg"></p>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t1fn1">
<sup>a</sup><p class="display-inline">The best
solution is highlighted
in green.</p>
</div></div></section><p>The calculations
at this stage confirmed the thermodynamic stability
of the obtained construct. Models with errors would either be energetically
unfavorable or produce incorrect conformations.</p></section><section id="sec3.2"><h3 class="pmc_sec_title">Rational Design
Mutagenesis of Ssp DnaE</h3>
<p>In the next
stage, an analysis of the active surface in the intein structure was
performed to identify the major and minor amino acids involved in
the formation of the protein–protein complex. Initially, the
N/C intein fused with GFP was separated into two distinct chains.
A larger fragment containing the N-part of the intein was used for
further molecular surface analysis.</p>
<p>For the selected structure,
the molecular surface was calculated and mapped for potential non-bonded
interactions, which were divided into three general types: lipophilic,
positive electrostatic, and negative electrostatic. The maximum possible
intermolecular interaction potential (analogous to the Gibbs free
energy in kcal/mol) was calculated for each amino acid forming the
surface segment. This calculation considers the available molecular
surface area of the amino acid side chain and classifies it as lipophilic,
polar, or charged.</p>
<p>Mutations were selected by evaluating the
magnitudes of intermolecular
interaction potentials and the types of amino acids from the C-part
of the intein that interact with the mapped surface of the N-intein.
These amino acids and their intermolecular interaction potentials
are listed in <a href="#tbl2" class="usa-link">Table 
</a>
</p>
<section class="tw xbox font-sm" id="tbl2"><h4 class="obj_head">2. Binding Potentials of the Interacting
Amino Acids in the Intein Structure<sup></sup>
.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371501_bg5c00091_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9067/12371501/c7d913837d07/bg5c00091_0008.jpg" loading="lazy" id="fx2" height="990" width="773" alt="graphic file with name bg5c00091_0008.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/tbl2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t2fn1">
<sup>a</sup><p class="display-inline">Contact
type: rednegative
electrostatic potential; bluepositive electrostatic potential;
and greenlipophilic.</p>
</div></div></section><p>The pool of possible point mutations was determined
by the values
of the potential intermolecular interactions. For example, A162 was
estimated to be optimal for the replacement of alanine (a lipophilic
amino acid) with a polar or more accessible side-chain amino acid
with a negative charge localization. The complete panel of possible
point mutations is listed in <a href="#tbl3" class="usa-link">Table 
</a>.</p>
<section class="tw xbox font-sm" id="tbl3"><h4 class="obj_head">3. Variety of Mutations in the C-Part
of Ssp DnaE Intein after Evaluation of the Binding Interaction Potentials.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">residue of the intein C-part</th>
<th align="center" colspan="1" rowspan="1">point mutation variants</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">B/I148</td>
<td align="left" colspan="1" rowspan="1">Y, N, Q, H, S, T</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B/I151</td>
<td align="left" colspan="1" rowspan="1">A, C, G, L, M, F, P, W, V</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B/Q155</td>
<td align="left" colspan="1" rowspan="1">R, I, F, A, N,
D, C, E, G, L, K, M, P, S, T, W, Y, V</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B/F159</td>
<td align="left" colspan="1" rowspan="1">M, A, C, G, I, L, P, W, V</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B/A162</td>
<td align="left" colspan="1" rowspan="1">F, C, G, I, L, M, P, W, V</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B/A165</td>
<td align="left" colspan="1" rowspan="1">V, W, P, F, M, L, I, G, C</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The number of mutations
was limited to only single and double simultaneous
mutations by exponentially increasing the complexity of the calculations.
Initially, the mutation effect was predicted by changing the Gibbs
free energy in affinity (ΔΔ<em>G</em>) and stability
(ΔStability) parameters within the protein–protein complex.
In addition, conformational rearrangements of amino acids induced
by the introduced mutation were calculated within 7 Å around
the mutated amino acids. Mutant models were selected by optimal combination
of ΔΔ<em>G</em> and Δstability parameters
and selected variants are listed in <a href="#tbl4" class="usa-link">Table 
</a>.</p>
<section class="tw xbox font-sm" id="tbl4"><h4 class="obj_head">4. Evaluation of Ssp
DnaE C Part Mutations
Influence on the Trans-Splicing Efficiency<sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
</colgroup>
<thead>
<tr>
<th align="center" colspan="1" rowspan="1"> </th>
<th colspan="2" align="center" rowspan="1">calculated
free energy changes<hr>
</th>
<th colspan="3" align="center" rowspan="1">experimental
data from 48-h post-transfected HEK
293<hr>
</th>
</tr>
<tr>
<th align="center" colspan="1" rowspan="1">samples</th>
<th align="center" colspan="1" rowspan="1">affinity ΔΔ<em>G</em> dG(kcal/mol)</th>
<th align="center" colspan="1" rowspan="1">stability Δstability dG(kcal/mol)</th>
<th align="center" colspan="1" rowspan="1">mean %GFP+</th>
<th align="center" colspan="1" rowspan="1">mean MFI</th>
<th align="center" colspan="1" rowspan="1">mean number of GFP+ objects per image</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">pGFP</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">93,7</td>
<td align="center" colspan="1" rowspan="1">236959</td>
<td align="left" colspan="1" rowspan="1">5711.78</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A165M</td>
<td align="left" colspan="1" rowspan="1">–18.92</td>
<td align="left" colspan="1" rowspan="1">0.74</td>
<td align="left" colspan="1" rowspan="1">64,0</td>
<td align="center" colspan="1" rowspan="1">11132</td>
<td align="left" colspan="1" rowspan="1">1099</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SSP N+C</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">62,3</td>
<td align="center" colspan="1" rowspan="1">9947</td>
<td align="left" colspan="1" rowspan="1">1775</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A162M</td>
<td align="left" colspan="1" rowspan="1">–3.25</td>
<td align="left" colspan="1" rowspan="1">0.92</td>
<td align="left" colspan="1" rowspan="1">59,4</td>
<td align="center" colspan="1" rowspan="1">6250</td>
<td align="left" colspan="1" rowspan="1">992</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">F159M</td>
<td align="left" colspan="1" rowspan="1">–4.61</td>
<td align="left" colspan="1" rowspan="1">–2.28</td>
<td align="left" colspan="1" rowspan="1">43,4</td>
<td align="center" colspan="1" rowspan="1">1609</td>
<td align="left" colspan="1" rowspan="1">50</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A162F+A165M</td>
<td align="left" colspan="1" rowspan="1">–31.73</td>
<td align="left" colspan="1" rowspan="1">–26.29</td>
<td align="left" colspan="1" rowspan="1">41,6</td>
<td align="center" colspan="1" rowspan="1">1416</td>
<td align="left" colspan="1" rowspan="1">129</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A162M+A165M</td>
<td align="left" colspan="1" rowspan="1">–22.92</td>
<td align="left" colspan="1" rowspan="1">–14.04</td>
<td align="left" colspan="1" rowspan="1">26,2</td>
<td align="center" colspan="1" rowspan="1">1042</td>
<td align="left" colspan="1" rowspan="1">22</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">I151M</td>
<td align="left" colspan="1" rowspan="1">–14.00</td>
<td align="left" colspan="1" rowspan="1">–10.28</td>
<td align="left" colspan="1" rowspan="1">16,4</td>
<td align="center" colspan="1" rowspan="1">671</td>
<td align="left" colspan="1" rowspan="1">1.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HEK293</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">0,3</td>
<td align="center" colspan="1" rowspan="1">450</td>
<td align="left" colspan="1" rowspan="1">0.1</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t4fn1">
<sup>a</sup><p class="display-inline">Experimental results
after transfection
of HEK293 were compared with the calculations of free energy changes
in affinity and stability.</p>
</div></div></section></section><section id="sec3.3"><h3 class="pmc_sec_title">Experimental
Verification of the Effects of Mutations on the
Trans-Splicing Efficiency in HEK293 Cells</h3>
<p>Six selected mutations
(<a href="#tbl4" class="usa-link">Table 
</a>) with the
highest changes in ΔΔ<em>G</em> and Δstability
parameters were used for the gene-engineering plasmid modification.
Original plasmids for estimation of changes in the trans-splicing
efficiency by yield of the intein-mediated GFP reconstruction were
obtained in a previous article. Mutagenesis
primers (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>) were used for overlapping
PCR to modify the nucleotide sequences corresponding to the selected
amino acids.</p>
<p>Six mutant Ssp DnaE C fragments were obtained by
PCR and used for cloning. Non-modified pairs of the Ssp DnaE-N/C-GFP
plasmids were used to compare the effects of Ssp DnaE-C modifications.
pGFP plasmid was used to compare the direct expression of full-length
GFP with intein-mediated reconstruction of GFP divided at 69/70 split
site. Untransfected HEK293 cells were used as a negative control for
cytometry gating of GFP+ cells by values of measured autofluorescence.
The resulting experimental data are presented in <a href="#tbl4" class="usa-link">Table 
</a> and <a href="#fig4" class="usa-link">Figure 
</a>.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371501_bg5c00091_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9067/12371501/0d109c7e6ea5/bg5c00091_0004.jpg" loading="lazy" id="gr4" height="1059" width="703" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Experimental validation of the predicted mutagenesis
effect on
trans-splicing efficiency in combination of SspN-GFP and versions
of SspC-GFP (short name on plot SSP N+C): (A) fluorescence microphotographs
of 48 h post-transfected HEK293 cells captured by the IncuCyte S3
with parameters: six images per well with 10× magnification and
300 ms of GFP exposure. (B) Flow cytometry analysis of GFP+ cells
population, (C) flow cytometry analysis of MFI values normalized to
the fluorescence of the Ssp N+C sample in 10-base logarithmic scale,
(D) enlarged version of MFI values normalized to the fluorescence
of the Ssp N+C sample, (E) kinetic curves of green objects per image
accumulation from cells during 24 h incubation in IncuCyte S3. (*) <em>p</em>-value &lt;0.05, (**) <em>p</em>-value &lt;0.01,
(***) <em>p</em>-value &lt;0.001, and­(****) <em>p</em>-value &lt;0.0001.</p></figcaption></figure><p>The samples were arranged
by the percentage of GFP+ cells and MFI
(<a href="#tbl4" class="usa-link">Table 
</a>). The A165M
sample with simultaneous positive changes in stability (Δstability)
and negative changes in affinity (ΔΔ<em>G</em>) had slightly higher %GFP+ and MFI values compared to the original
pair Ssp N+C. Otherwise, simultaneous changes in affinity and stability
resulted in a 20% decrease in efficiency for the double mutation A162F
+ A165M and a 35% decrease for A162M + A165M. The difference between
the experimental results and calculations based on energy changes
with the SASA method indicated that a different approach should be
used to predict the effect of mutations at the calculation stage.
Fluorescence analysis of micrographs (<a href="#fig4" class="usa-link">Figure 
</a>A) demonstrated that the single mutation
I151M and double mutation A162M + A165M resulted in a significant
decrease in trans-splicing efficiency. Notably, single mutations at
the same positions, A162M and A165M, led to the opposite effect, which
is clearly illustrated in <a href="#fig4" class="usa-link">Figure 
</a>B–C. Kinetic curve analysis (<a href="#fig4" class="usa-link">Figure 
</a>E) was used to cluster samples
into two groups with amplitude similar to that of the original pair
(SspN+SspC, A162M, and A165M) and decreased amplitude in all other
samples.</p>
<p>The population analysis of GFP+ cells (<a href="#fig4" class="usa-link">Figure 
</a>B) was used to cluster the
samples more precisely
into three groups: the group with insignificant efficiency changes
in samples A162M and A165M, the group with significant efficiency
decrease up to 20% in samples A162F + A165M and F159M, and the group
with efficiency decrease of more than 30% in A162M + A165M and I151M.
MFI analysis with fluorescence normalized to SspN+SspC was plotted
on a 10-base logarithmic scale (<a href="#fig4" class="usa-link">Figure 
</a>C). Intergroup differences between Ssp N+C, A162M,
and A165 M were not significant. Significant differences were estimated
in the samples A162F+A165M, F159M, and I151M within 10 fold change
of magnitude from the control. A higher level of significance was
obtained in samples A162F + A165M, and the intensity was the same
as that of the negative control. Additional subplot (<a href="#fig4" class="usa-link">Figure 
</a>D) with fluorescence normalized
to Ssp N+C was plotted without a logarithmic scale for a more detailed
estimation. Despite the absence of significant differences, a positive
effect of the A165M mutation and a negative effect of the A162M mutation
were observed.</p>
<p>Summary analysis of the experimental data revealed
a negative effect
for mutations in I151M, F159M, A162M + A165M, and A162F + A165M and
a neutral or slightly positive effect for A162M and A165M. These results
were used to analyze protein–protein interactions with molecular
metadynamic modeling.</p></section><section id="sec3.4"><h3 class="pmc_sec_title">Molecular Metadynamics Analysis of Mutated
Protein Structures</h3>
<p>The method of molecular metadynamics was
used to analyze and explain
the experimental results obtained in terms of molecular dynamics.
This method was used as an alternative to predict the effects of mutagenesis
by analyzing the free energy changes in the system in terms of selected
collective variables. The I151M and A165M mutations were used to explain
the role of single mutations at different structural points of intein.
Mutations A165M and A162M + A165M were used to explain the negative
effects of double mutations with methionines close to each other.
The selected mutations demonstrated a contrast effect on trans-splicing
efficiency and could be used for a better understanding of intein’s
structural features.</p>
<p>Free energy surfaces were evaluated for
selected mutations in the coordinates of the collective variables
(<a href="#fig5" class="usa-link">Figure 
</a>). The obtained
surfaces could explain mutations effects by the rearrangement of local
energy minima, which determine the lability and flexibility of intein
molecules and kinetics of the splicing process. For instance, the
partitioning of free energy into multiple oriented minima indicates
changes in molecular interactions (<a href="#fig5" class="usa-link">Figures 
</a>A,C). <em>In vitro</em> tests in
the case of A165M did not show a significant change in efficiency
with the cell assay; therefore, other methods should be employed for
further investigation. Otherwise, the free energy concentration at
a single major minimum was correlated with a decrease in splicing
efficiency (<a href="#fig5" class="usa-link">Figures 
</a>B,D). This common trend has introduced a new insight for the primary
prediction approach to estimate the mutagenesis effect on trans-splicing
by the analysis of local free energy distribution and estimation of
energy barrier height between them.</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371501_bg5c00091_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9067/12371501/44c39e9c198d/bg5c00091_0005.jpg" loading="lazy" id="gr5" height="568" width="696" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Free energy surfaces obtained from 200
ns metadynamic calculations
of Ssp DnaE intein fused with GFP: (A) without mutations, (B) double
A162M+A165M mutation, (C) single A165M mutation, and (D) single I151M
mutation. Local minima of free energy are numbered in circles, and
colors represent the energy value. Individual images with fitted axes
are placed on <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S1–S4</a>.</p></figcaption></figure><p>The obtained metadynamic analysis results (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2–S5)</a> were used to illustrate the
energy distribution
with surfaces (<a href="#fig5" class="usa-link">Figure 
</a>). Similar scales of collective variable coordinates enable the estimation
of the clockwise and anticlockwise directions of the free energy distribution.
A borderline was placed on plots between points (0;15) and (40;8)
in CV1–CV2 coordinates to separate minima in the left corner
with coordinates that describe the interaction of C-part centroids
with the N-terminal part centroids. This visual implementation enables
quick estimation of the most probable effect of mutations on splicing
efficiency changes. A detailed explanation of each mutation effect
will be discussed further.</p>
<p>Primarily, the A165M mutation with
a clockwise free energy distribution
has a positive effect on the splicing reaction. Additionally, A165M
was characterized by an increased number of local minima shifted to
the right by the CV1 coordinates with a decreased height of the energy
barrier compared to the original intein. The most evident rearrangement
occurred by shifting the barrier value from +18.75 kcal/mol between
1 and 2 points in the non-mutated intein to +10.93 kcal/mol between
3 and 4 points in A165M (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S3</a>). This
fact has made energy clusterization more distinguishable by coordinates
and allowed us to draw a line on plots that defines the direction
of the energy distribution. The decrease in the energy barrier, along
with the increasing value of the common free energy (12.71 kcal/mol),
has enabled the expansion of energetically permissible intein conformations.
The summary effect of the A165M mutation can be characterized as an
increase in conformational mobility with a higher number of energy
minima and lower energy barriers, resulting in reaction-competent
states with lower energy transfer costs.</p>
<p>Secondly, mutations
with negative effects, I151M and A162M + A165M,
were characterized by an anticlockwise energy distribution. In the
case of the double mutation A162M + A165M, the number of minima has
increased from four in the original structure to six. This rearrangement
of the minima increases the conformational mobility of the molecules.
However, the double mutations A162M + A165M and A165M had almost equal
numbers of minima, but most of them were placed on the left side of
the CV1 coordinate. The inhibitory effect of the second methionine
mutation was indicated by the minima concentration at the position
of the initial state, with the higher barrier between 1 and 2 points
in A165M with+4.24 kcal/mol (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S3</a>)
and A162M + A165M with +10.96 kcal/mol (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S4</a>).</p>
<p>This fact has increased cost of energy transfer,
and collapsed
intein's viable conformations. In addition, structural analysis
has
revealed evidence of changes in the profile of hydrophobic contacts
within the intein core around amino acids A/L189, A/V194, A/I195,
and A/P144 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S7</a>).</p>
<p>A similar
effect was observed for the I151M mutation. Additional
hydrophobic contacts became available with A/I137/147, A/V159, and
B/F159 due to the longer chain length (5.51Å M vs 3.87Å
I). Essentially, the I151M mutation has stimulated hydrophobic contacts
between chains A and B of the intein while maintaining contacts with
B/L153 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8</a>). This region has originally
formed a “leucine zipper” that was stabilized with an
I151M mutation. The conformational mobility remained original with
the same number of minima, mostly at the initial positions. Suddenly,
the mutation has caused a decrease in each energy barrier with a simultaneous
increase in the depth of each minimum.</p>
<p>In summary, the I151M
mutation has a dual effect. On the one hand,
energy barriers were significantly lower than those in other cases,
and conformations were available for expansions and transitions. The
concentration of free energy in the four deep minima and the formation
of additional hydrophobic contacts made all conformations energetically
unfavorable for the molecular system. The resulting overstabilization
of the hydrophobic core of the intein molecule makes trans-splicing
energetically unaffordable.</p>
<p>These examples have demonstrated
that free energy analysis could
indirectly highlight the structural anomalies caused by important
conformational changes, which should be additionally observed to accept
the final decision regarding the predicted mutation effect.</p></section><section id="sec3.5"><h3 class="pmc_sec_title">Molecular
Metadynamics of Control Mutations</h3>
<p>The results
from the previous section were applied to the metadynamic analysis
for two control mutations, whose effects had been previously experimentally
estimated.</p>
<p>Mutation T198A completely blocked the trans-splicing
reaction and mutation A168H increased
molecular stability and splicing efficiency.<em>
,
</em> These facts were used to incorporate these mutations into the obtained
model of GFP fused with intein Ssp DnaE. The results of the metadynamic
simulation for this model are illustrated (<a href="#fig6" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371501_bg5c00091_0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9067/12371501/b9e921d09bcd/bg5c00091_0006.jpg" loading="lazy" id="gr6" height="558" width="696" alt="6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Free energy surfaces obtained from 200 ns metadynamic
calculations
of DnaE-fused Ssp intein with GFP: (A) without mutations, (B) T198A
mutation, and (C) A168H mutation. Local minima of free energy are
numbered in circles, and colors represent energy value. Individual
images with fitted axes are placed on <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S5–S6</a>.</p></figcaption></figure><p>The simulation results
have supported the experimental data in
the context of the free energy and structural changes in the model.
The threonine-alanine replacement has disrupted the hydrogen bond
network between D200 and Q182, where T198 was acting as a bridge,
realizing transient contacts (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S9 and S10</a>). From the position of energy changes, the T198A mutation forms
deep energy minima 1 (<a href="#fig6" class="usa-link">Figure 
</a>) with a high energy barrier for transitions to local minima
2 and 3 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S6</a>). At the same time, the
free energy of the system is even lower than that of intein without
mutations (<a href="#fig6" class="usa-link">Figure 
</a>, <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>).</p>
<p>In the structure without
mutations, there are 4 energy minima,
while with the T198A mutation, the fourth minimum disappears from
the initial coordinates.</p>
<p>As a result, we observe the formation
of a deep energy minimum
at point 1, with a high energy barrier for the transition to local
minima 2 and 3.</p>
<p>The opposite effect was observed for the A168H
mutation. Surface
analysis (<a href="#fig6" class="usa-link">Figure 
</a>C)
revealed an increase in the number of energy minima from the original
four to nine with a shift of the CV1 coordinates, which improved the
mobility of the molecule. The energy barriers in this case were reduced
to almost zero between minima 4 and 5, and between minima 5 and 6
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S7</a>). In addition, an overall increase
of free energy (+11.13 kcal/mol) made the intein system more flexible
and labile. This positive effect should provide active sites for the
splicing reaction and increase its efficiency. Structural changes
after the A168H mutation with the formation of π–cation
contacts with A/R202 and hydrogen bonds with B/D171 and B/N 158. These
intermolecular interactions have stabilized the core with a powerful
source of intramolecular contacts during conformational rearrangement
along the splicing reaction (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S11</a>).</p>
<p>In summary, the results obtained with the metadynamic simulation
were validated by experiments in this study and additionally tested
with control mutations. The results in the previous section were confirmed,
and the combination of metadynamics and structural analysis could
be used as a predictive tool for further modification of the intein
structure using point mutations.</p></section></section><section id="sec4"><h2 class="pmc_sec_title">Discussion</h2>
<p>This
study was done as a continuation of our previous research. The development and validation of a site-directed
mutagenesis method were the primary goals of this study. The obtained
metadynamic method of analysis could be used for the estimation of
mutagenesis of any inteins with correction to structure analysis.</p>
<p>Particularly, modification of slow Ssp DnaE intein was performed
with an insignificant experimental increase of trans-splicing efficiency
in case A165M and a considerable decrease in other cases. However,
analysis of free energy changes has revealed trends in structural
changes that are essential for trans-splicing efficiency. Furthermore,
this trend was replicated with results from control mutations that
supported the findings from this and previous studies.<em>
−

</em>
</p>
<p>The implementation of a metadynamic analysis to estimate the
mutagenic
effect is a key feature of this study. Previously, this method has
been used to estimate the inhibitory effect of molecules and the impact of each amino acid in exteins. Free energy distribution analysis in collective
variables was performed in inteins to predict the possible initial
confirmation for the first time. In previous studies, RMSD trajectories,
covariance matrices of backbone Cα atom motion, and principal component analysis on Cα coordinates were used to describe the effect of mutations
implemented with the molecular dynamics method. The implemented method
was double-checked with new and control mutations, which made this
approach more reliable than the primary estimation of changes in stability
and affinity.</p>
<p>New mutation points were obtained from the active
surface analysis
of protein models, which has been widely used to analyze protein–ligand
or protein–protein interactions. This approach was also used for the first time in intein mutagenesis.
Geometric structure analysis with Ramachandran plots was alternatively
used to identify structural features that could significantly change
the splicing yield.
</p>
<p>Experimental
screening of amino acids with more favorable chemical
properties is the most common method of directed mutagenesis. The structural analysis of the static and dynamic
changes supported the experimental data with more detailed information
on intermolecular mechanics.<em>
,
</em> Notably, all listed
chemical methods of intein rational modification were able to perform
single precise mutations in the conservative and catalytic domains
to significantly change the trans-splicing efficiency.</p>
<p>“Irrational”
methods of intein modification have
also been used with computational methods in genetics. Consensus analysis
of the amino acid sequence with annotation of functional domains has
enabled simultaneous manipulations with multiple mutations. Cfa intein
was assembled with the fast domain of DnaE inteins after alignment
and annotation of 73 sequences. The resulting intein has demonstrated
3 times faster kinetics than NpuDnaE, which has the highest yield
and kinetics in the natural intein group. The major drawback of this modification method was revealed in the
experimental application of Cfa for trans-splicing reconstruction
of large protein complexes. Suddenly,
the yield of target protein splicing was significantly higher for
NpuDnaE intein than for Cfa and Ssp DnaE. This unexpected result was
obtained as a consequence of the uncovered features of the artificial
intein, which has individual structural features at the protein level.
However, structural analysis and targeted precise modifications of
the C-part of the Cfa intein have already been performed to increase
the yield in the case of protein purification.
</p>
<p>Implemented point modification of the C-part of Ssp
DnaE has mostly
decreased the yield of the trans-splicing reaction but has also revealed
structural features. A series of single-point mutations with changes
from neural amino acids to methionine in different domains of intein
could be used to better understand the role of individual residues
in the structure of intein. The general trend emerged from the observation
that trans-splicing efficiency decreases in the direction from the
peripheral catalytic part to the intein’s core with the mutations
A165M, A162M, F159M, and I151M. In the peripheral part at A165 and
A162, the efficiency changes slightly in the percentage of GFP+ cells.
This flexible domain was bound by F159 with a significant decrease
in dozens of percent of GFP+ cells. The core domain with I151 could
be identified with a drastic decrease in GFP+ cells, similar to the
negative control. This fact was additionally explained by the analysis
of the structural changes in each case. For instance, I151M has formed
an increased number of lipophilic contacts in the core with N- and
C-parts, which has increased the absolute energy minimum and overstabilized
the C-part molecule.</p>
<p>The same negative effect was obtained with
the control T198A mutation
in the case of destabilization of the N-part. This mutation was associated
with the most conserved tyrosine located in the catalytic and geometrical
centers of the N-part of the intein. Two mutations in this core position
with opposite effects were tested after extraction of intein structural
features based on Ramachandran plot analysis. T198S mutation demonstrated a neutral effect on splicing with insignificant
changes in geometry, whereas T198A destabilized the structure and
completely blocked the response. These experimental data were supported
by the calculations in this study. The docking results illustrated
that amino acid Y198 formed a bridging bond between D200 and Q182
in the initial state of intein. Replacement of tyrosine with alanine
has disrupted these hydrogen bonds and destabilized the molecule at
the core of the N-part of the intein. Free energy surface analysis
reflected these structural changes with a decreased number of energy
minima and an increased height of the energy barriers. Similar results
were obtained in the case of I151M. The conclusion regarding the negative
influence of increased energy barriers on the splicing reaction is
consistent with previous research.
</p>
<p>Unexpected results were obtained from the analysis of experimental
data with the double mutations A162F + A165M and A162M + A165M. Firstly,
these results have disproved the primary method of prediction of mutation
effects based on energy changes with affinity (−ΔΔ<em>G</em>) and stability (ΔStability) parameters. Secondly,
structural analysis replaced this stage after the discovery of a significant
decrease in splicing efficiency in the case of simulated double methionine
mutations in A162M + A165M. Notably, similar single mutations at similar
positions in the cases of A162M and A165M did not demonstrate such
effects. This fact suggests that double methionines have formed contact
with each other, but structural analysis has demonstrated only destabilization
of the C-part with the extensive form of each methionine molecule.
Moreover, surface analysis has allowed a visual comparison of A162M
+ A165M with A165M and has outlined the effect of A162M in changing
the number of lipophilic contacts with the same number of energy minima.
Additionally, the formation of lipophilic contacts has shifted the
coordinates of the minima to the CV2 axes and increased the height
of the energy barriers. In summary, the overall mobility of the molecule
has been increased, but structural changes have prevented reactions
from favorable splicing interactions.</p>
<p>In the case of the experimentally
neutral effect of the A165M mutation,
both metadynamics and structural analysis have investigated mostly
the positive effects of alanine to methionine substitution. The map
of lipophilic contacts between the N- and C-portions was modified
by the formation of lipophilic contacts with F203, I195, L223,/189
and disruption of a quartet between A/A197, A/I195/213, and B/L161.
The location of the mutation at the boundary between two structured
regions (β-sheets transitioning into loops) has affected the
conformational rearrangement due to steric constraints that change
the mechanics of the conformational transition. The immediate proximity
of the mutations to the peripheral catalytic center at the end of
the C-part of the intein has provided a positive effect in the case
of A168H. The substitution of alanine with histidine has created
powerful π-cationic interactions between arginine at position
202 and hydrogen bonds between histidine and aspartic acid at position
158 and aspartic acid at position 171. These interactions have stabilized
the intein core during conformational rearrangement for splicing.
Metadynamic calculations have revealed the formation of additional
energy minima with low energy barriers, which made the molecular system
more conformationally labile and promoted an efficient trans-splicing
reaction. In both cases A165M and A168H, the replacement of alanine
with structurally more extended amino acids has formed additional
contacts that have stabilized the catalytic C-part center and further
accelerated the cyclization of aspartate in the trans-splicing reaction.</p>
<p>Stabilization of inteins has been used “as the simplest
approach for rational engineering of proteins”. Other chemical approaches have been used for
the targeted deletion of several amino acids by NMR analysis to increase
the cleavage of the N-part to promote the trans-splicing reaction. Another modification method based on NMR analysis
was used to enable the splicing reaction in a 2 M NaCl solution by
replacing positively charged amino acids with neutral ones. Biological approaches have also been used to
find genetically variable domains by screening all 20 amino acids
with oligonucleotide-directed mutagenesis. As a result, dozens of DNA libraries must be tested experimentally
with random results, mostly without any chemical explanation.</p>
<p>The most effective biological approach is the discovery of new
shorter natural inteins from the results of genomic data analysis.
For instance, gp41 intein was discovered as a result of massive metagenomic
data analysis and currently has the faster kinetics than Npu DnaE
by 10 times. However, its resolved structure
was only recently obtained and its features
have not yet been fully studied. In particular, the discovery of new
inteins did not stop the search for modifications and mutations of
newly discovered molecules. This problem
and the method of solution have already been discussed in the study
of increasing reaction yields, which
is particularly important in protein purification. Therefore, the development of new approaches for rational
mutagenesis of inteins, optimized for each specific purposes, would
also be relevant for the discovery of new inteins. The method of metadynamic
simulation would significantly accelerate the process of hypothesis
testing.</p></section><section id="sec5"><h2 class="pmc_sec_title">Conclusions</h2>
<p>In this discussion, the most relevant
methods of intein modification
were listed. The main trend was outlined in the discovery of new short
inteins and the modification of their structures. The most common
modification was identified as stabilization of the C-terminal catalytic
site of inteins to accelerate cyclization in the splicing reaction.
Other findings were also obtained from the experimental data.</p>
<p>First, the application of active site analysis should be modified
with corrections for genetic analysis of conserved and variable positions.
Interactant estimation should be used for further selection of amino
acids, but the selection method should be modified with structural
analysis. Domain functional analysis should be used to estimate the
parameters that would be optimized for a particular purposes. For
instance, in the case of trans-splicing efficiency, optimization of
the C-terminal catalytic site should be stabilized, and analysis of
energy changes in Δstability was applicable. In other cases,
this method was not applicable due to the structural features.</p>
<p>Second, molecular metadynamics methods were used to predict the
mutation effects. The main feature of this method was revealed with
the analysis of free energy surfaces in collective variables, which
could indicate significant structural rearrangements by coordinates
and the type of minima allocation and grouping. The concentration
of minima in a cluster with high values of energy and barriers has
indicated structural changes that could reduce conformational lability.
Otherwise, the widespread distribution of an increased number of minima
with lower energy barriers indicated structural changes that could
increase the mobility and conformational lability of molecules.</p>
<p>Third, most of the methionine mutations have destabilized the structure
of inteins, with the exception of A165M. Any replacement in the core
structure of inteins could significantly change the structure and
completely block splicing in the case of I151 M and T198A. The dual
effect of the extended structure of methionine molecule was revealed
in the case of double mutations A162M + A165M with destabilization
of the intein C core structure, and in the case of single A165M mutation,
methionine has stabilized the C-part catalytic center. A similar effect
of histidine extension structure was obtained in the case of A168H.</p>
<p>These new insights into the modification of intein Ssp DnaE could
be further used for the structural analysis of intein N and C core
interactions. Additional experimental evidence for the modification
of the intein C-part to stabilize the catalytic center could be used
to obtain new optimized intein pairs. The combination of computational
methods for the analysis of the structure and dynamics of intein interactions
would significantly accelerate the development of improved versions
of intein-mediated trans-splicing systems.</p>
<p>This methodology
could be used in many biotechnology applications,
such as protein assembly and purification, as a basis for visualizing
protein–protein interactions, and as a technology for advanced
gene therapy. For instance, in our next research, we will use the
obtained results as improved solutions to overcome the limitations
of AAV packaging with optimized trans-splicing efficiency. These findings
could assist in the discovery of new approaches to create more efficient
transgene delivery systems and genome editing systems.</p></section><section id="sec20"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371501/bin/bg5c00091_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg5c00091_si_001.pdf</a><sup> (1.8MB, pdf) </sup>
</div></div></section></section><section id="ACK-d14e1018-autogenerated" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We gratefully
acknowledge the Oligosynthesis group of the
Laboratory Complex Resource Center of the Center for Genetics and
Life Sciences (Dmitry Yakshin, Ekaterina Cherkasova, Anna Cherentaeva)
for oligonucleotide synthesis, the Sequencing group of the Laboratory
Complex Resource Center of the Center for Genetics and Life Sciences
(Elena Kolosova, Elena Dolgova) for sequencing of all constructs,
and the Resource Center of Cell Technology of the Center for Genetics
and Life Sciences for assistance with cell culturing and manuscript
review &amp; editing.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Glossary</h2>
<h3>Abbreviations</h3>
<dl class="def-list" id="dl1">
<dt>GFP</dt>
<dd><p>green
fluorescent protein</p></dd>
<dt>HEK293</dt>
<dd><p>human embryonic kidney 293</p></dd>
<dt>DNA</dt>
<dd><p>DNA</p></dd>
<dt>PCR</dt>
<dd><p>polymerase chain reaction</p></dd>
<dt>AAV</dt>
<dd><p>adeno-associated
viruses</p></dd>
<dt>cDNA</dt>
<dd><p>coding
DNA</p></dd>
<dt>DMD</dt>
<dd><p>Duchenne
muscular dystrophy</p></dd>
<dt>DYSF</dt>
<dd><p>dysferlin</p></dd>
<dt>USH2A</dt>
<dd><p>usher syndrome type II A</p></dd>
<dt>ABCA</dt>
<dd><p>ATP-binding cassette 4</p></dd>
<dt>EYS</dt>
<dd><p>eyes shut homologue</p></dd>
<dt>F8/VIII</dt>
<dd><p>factor VIII of blood coagulation</p></dd>
<dt>ABE</dt>
<dd><p>adenine base editor</p></dd>
<dt>CBE</dt>
<dd><p>cytosine base editor</p></dd>
<dt>NMR</dt>
<dd><p>nuclear magnetic
resonance</p></dd>
<dt>SASA</dt>
<dd><p>solvent-accessible
surface area
method</p></dd>
<dt>VSGB</dt>
<dd><p>variable
surface generalized Born
model</p></dd>
<dt>OPLS4</dt>
<dd><p>optimized
potential for the liquid
simulation model</p></dd>
<dt>LBFGS</dt>
<dd><p>limited Broyden–Fletcher–Goldfarb–Shanno
algorithm</p></dd>
<dt>NVT</dt>
<dd><p>canonical
ensemble amount of substance
(N), volume (V), temperature (T)</p></dd>
<dt>NPT</dt>
<dd><p>canonical ensemble amount of substance
(N), pressure (P), temperature (T)</p></dd>
<dt>RMSD</dt>
<dd><p>root-mean squared deviation</p></dd>
<dt>FACS</dt>
<dd><p>fluorescence-activated cell sorting</p></dd>
<dt>FSC</dt>
<dd><p>forward scatter</p></dd>
<dt>SSC</dt>
<dd><p>side scatter</p></dd>
</dl></section><section id="notes-1"><p>The Supporting Information
is available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsbiomedchemau.5c00091?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsbiomedchemau.5c00091</a>.</p>
<ul id="silist" class="list" style="list-style-type:none"><li><p>Oligonucleotide
mutagenesis primers for the C-part of
Ssp DnaE (Table S1); free energy surfaces after molecular metadynamic
simulations with plot (Figure S1–S6) values of energy minima
(Table S2–S7), and structural changes (Figure S7–S11)
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00091/suppl_file/bg5c00091_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes-2"><h2>§.</h2>
<p>M.O.S., A.N.B., and M.A.G. contributed equally
to this work. M.O.S.‡:
data curation, formal analysis, investigation, software, visualization,
and writing–original draft. A.N.B.: conceptualization, data
curation, formal analysis, investigation, methodology, supervision,
and writing–original draft. M.A.G.‡: conceptualization,
data curation, formal analysis, investigation, methodology, software,
visualization, and writing–original draft. Y.B.P.: conceptualization,
formal analysis, project administration, supervision, and writing–review
and editing. S.A.C.: conceptualization, formal analysis, methodology,
and writing–review and editing. A.V.K.: conceptualization,
formal analysis, funding acquisition, project administration, resources,
supervision, and writing–review and editing. CRediT: <strong>Matvei O Sabantsev</strong> data curation, formal analysis, investigation,
software, visualization, writing - original draft; <strong>Andrew N
Brovin</strong> conceptualization, data curation, formal analysis, investigation,
methodology, supervision, writing - original draft; <strong>Maxim A
Gureev</strong> conceptualization, data curation, formal analysis, investigation,
methodology, software, visualization, writing - original draft; <strong>Yuri B Porozov</strong> conceptualization, formal analysis, project
administration, supervision, writing - review &amp; editing; <strong>Sergey A Chuvpilo</strong> conceptualization, formal analysis, methodology,
writing - review &amp; editing; <strong>Alexander V Karabelsky</strong> conceptualization, formal analysis, funding acquisition, project
administration, resources, supervision, writing - review &amp; editing.</p></section><section id="notes-3"><p>The
author(s)
declare that financial support was received for the research, authorship,
and/or publication of this article. This research was funded by the
Ministry of Science and Higher Education of the Russian Federation,
Agreement No 075-10-2025-017 from the 27.02.2025 project (GTH-RND-2011).</p></section><section id="notes-4"><p>The
authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Wang H., Wang L., Zhong B., Dai Z.. Protein Splicing of
Inteins: A Powerful Tool in Synthetic Biology. Front. Bioeng. Biotechnol. 2022;10:810180.  doi: 10.3389/fbioe.2022.810180.</cite> [<a href="https://doi.org/10.3389/fbioe.2022.810180" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8899391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35265596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Bioeng.%20Biotechnol.&amp;title=Protein%20Splicing%20of%20Inteins:%20A%20Powerful%20Tool%20in%20Synthetic%20Biology&amp;author=H.%20Wang&amp;author=L.%20Wang&amp;author=B.%20Zhong&amp;author=Z.%20Dai&amp;volume=10&amp;publication_year=2022&amp;pages=810180&amp;pmid=35265596&amp;doi=10.3389/fbioe.2022.810180&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>

Shah, N. H.
; 
Stevens, A. J.
. 
Identification,
Characterization, and Optimization of Split Inteins. In Methods in Molecular Biology; Springer Nature, 2020; Vol. 2133, pp 31–54.
</cite> [<a href="https://doi.org/10.1007/978-1-0716-0434-2_3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32144662/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Molecular%20Biology&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Topilina N. I., Mills K. V.. Recent Advances in in Vivo Applications
of Intein-Mediated
Protein Splicing. Mobile DNA. 2014;5(1):5.  doi: 10.1186/1759-8753-5-5.</cite> [<a href="https://doi.org/10.1186/1759-8753-5-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3922620/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24490831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mobile%20DNA&amp;title=Recent%20Advances%20in%20in%20Vivo%20Applications%20of%20Intein-Mediated%20Protein%20Splicing&amp;author=N.%20I.%20Topilina&amp;author=K.%20V.%20Mills&amp;volume=5&amp;issue=1&amp;publication_year=2014&amp;pages=5&amp;pmid=24490831&amp;doi=10.1186/1759-8753-5-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Shi C., Tarimala A., Meng Q., Wood D. W.. A General Purification
Platform for Toxic Proteins Based on Intein Trans-Splicing. Appl. Microbiol. Biotechnol. 2014;98(22):9425–9435. doi: 10.1007/s00253-014-6080-1.</cite> [<a href="https://doi.org/10.1007/s00253-014-6080-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25296714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&amp;title=A%20General%20Purification%20Platform%20for%20Toxic%20Proteins%20Based%20on%20Intein%20Trans-Splicing&amp;author=C.%20Shi&amp;author=A.%20Tarimala&amp;author=Q.%20Meng&amp;author=D.%20W.%20Wood&amp;volume=98&amp;issue=22&amp;publication_year=2014&amp;pages=9425-9435&amp;pmid=25296714&amp;doi=10.1007/s00253-014-6080-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Tornabene P., Trapani I.. Can Adeno-Associated
Viral Vectors Deliver Effectively
Large Genes? Human Gene Ther. 2020;31(3–4):128–140. doi: 10.1089/hum.2019.220.</cite> [<a href="https://doi.org/10.1089/hum.2019.220" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31916856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Gene%20Ther.&amp;title=Can%20Adeno-Associated%20Viral%20Vectors%20Deliver%20Effectively%20Large%20Genes?&amp;author=P.%20Tornabene&amp;author=I.%20Trapani&amp;volume=31&amp;issue=3%E2%80%934&amp;publication_year=2020&amp;pages=128-140&amp;pmid=31916856&amp;doi=10.1089/hum.2019.220&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Ray D. M., Flood J. R., David Y.. Harnessing
Split-Inteins as a Tool
for the Selective Modification of Surface Receptors in Live Cells. Chembiochem. 2023;24:e202200487.  doi: 10.1002/cbic.202200487.</cite> [<a href="https://doi.org/10.1002/cbic.202200487" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9977608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36178424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chembiochem&amp;title=Harnessing%20Split-Inteins%20as%20a%20Tool%20for%20the%20Selective%20Modification%20of%20Surface%20Receptors%20in%20Live%20Cells&amp;author=D.%20M.%20Ray&amp;author=J.%20R.%20Flood&amp;author=Y.%20David&amp;volume=24&amp;publication_year=2023&amp;pages=e202200487&amp;pmid=36178424&amp;doi=10.1002/cbic.202200487&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Zong H., Han L., Chen J., Pan Z., Wang L., Sun R., Ding K., Xie Y., Jiang H., Lu H., Gilly J., Zhang B., Zhu J.. Kinetics Study of the
Natural Split Npu DnaE Intein in the Generation of Bispecific IgG
Antibodies. Appl. Microbiol. Biotechnol. 2022;106(1):161–171. doi: 10.1007/s00253-021-11707-y.</cite> [<a href="https://doi.org/10.1007/s00253-021-11707-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34882254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&amp;title=Kinetics%20Study%20of%20the%20Natural%20Split%20Npu%20DnaE%20Intein%20in%20the%20Generation%20of%20Bispecific%20IgG%20Antibodies&amp;author=H.%20Zong&amp;author=L.%20Han&amp;author=J.%20Chen&amp;author=Z.%20Pan&amp;author=L.%20Wang&amp;volume=106&amp;issue=1&amp;publication_year=2022&amp;pages=161-171&amp;pmid=34882254&amp;doi=10.1007/s00253-021-11707-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Tornabene P., Trapani I., Minopoli R., Centrulo M., Lupo M., de Simone S., Tiberi P., Dell’Aquila F., Marrocco E., Iodice C., Iuliano A., Gesualdo C., Rossi S., Giaquinto L., Albert S., Hoyng C. B., Polishchuk E., Cremers F. P. M., Surace E. M., Simonelli F.. et al. Intein-Mediated Protein Trans-Splicing Expands Adeno-Associated Virus
Transfer Capacity in the Retina. Sci. Transl.
Med. 2019;11(492):eaav4523.  doi: 10.1126/scitranslmed.aav4523.</cite> [<a href="https://doi.org/10.1126/scitranslmed.aav4523" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6863751/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31092694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Transl.%20Med.&amp;title=Intein-Mediated%20Protein%20Trans-Splicing%20Expands%20Adeno-Associated%20Virus%20Transfer%20Capacity%20in%20the%20Retina&amp;author=P.%20Tornabene&amp;author=I.%20Trapani&amp;author=R.%20Minopoli&amp;author=M.%20Centrulo&amp;author=M.%20Lupo&amp;volume=11&amp;issue=492&amp;publication_year=2019&amp;pages=eaav4523&amp;pmid=31092694&amp;doi=10.1126/scitranslmed.aav4523&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Kolesnik V. V., Nurtdinov R. F., Oloruntimehin E. S., Karabelsky A. V., Malogolovkin A. S.. Optimization Strategies and Advances in the Research
and Development of AAV-Based Gene Therapy to Deliver Large Transgenes. Clin. Transl. Med. 2024;14(3):e1607.  doi: 10.1002/ctm2.1607.</cite> [<a href="https://doi.org/10.1002/ctm2.1607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10941601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38488469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Med.&amp;title=Optimization%20Strategies%20and%20Advances%20in%20the%20Research%20and%20Development%20of%20AAV-Based%20Gene%20Therapy%20to%20Deliver%20Large%20Transgenes&amp;author=V.%20V.%20Kolesnik&amp;author=R.%20F.%20Nurtdinov&amp;author=E.%20S.%20Oloruntimehin&amp;author=A.%20V.%20Karabelsky&amp;author=A.%20S.%20Malogolovkin&amp;volume=14&amp;issue=3&amp;publication_year=2024&amp;pages=e1607&amp;pmid=38488469&amp;doi=10.1002/ctm2.1607&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Truong D.-J. J., Kühner K., Kühn R., Werfel S., Engelhardt S., Wurst W., Ortiz O.. Development
of an Intein-Mediated Split–Cas9 System for Gene Therapy. Nucleic Acids Res. 2015;43(13):6450–6458. doi: 10.1093/nar/gkv601.</cite> [<a href="https://doi.org/10.1093/nar/gkv601" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4513872/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26082496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Development%20of%20an%20Intein-Mediated%20Split%E2%80%93Cas9%20System%20for%20Gene%20Therapy&amp;author=D.-J.%20J.%20Truong&amp;author=K.%20K%C3%BChner&amp;author=R.%20K%C3%BChn&amp;author=S.%20Werfel&amp;author=S.%20Engelhardt&amp;volume=43&amp;issue=13&amp;publication_year=2015&amp;pages=6450-6458&amp;pmid=26082496&amp;doi=10.1093/nar/gkv601&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Huang T. P., Newby G. A., Liu D. R.. Precision
Genome Editing Using Cytosine
and Adenine Base Editors in Mammalian Cells. Nat. Protoc. 2021;16(2):1089–1128. doi: 10.1038/s41596-020-00450-9.</cite> [<a href="https://doi.org/10.1038/s41596-020-00450-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33462442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Precision%20Genome%20Editing%20Using%20Cytosine%20and%20Adenine%20Base%20Editors%20in%20Mammalian%20Cells&amp;author=T.%20P.%20Huang&amp;author=G.%20A.%20Newby&amp;author=D.%20R.%20Liu&amp;volume=16&amp;issue=2&amp;publication_year=2021&amp;pages=1089-1128&amp;pmid=33462442&amp;doi=10.1038/s41596-020-00450-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Zeng H., Yuan Q., Peng F., Ma D., Lingineni A., Chee K., Gilberd P., Osikpa E. C., Sun Z., Gao X.. A Split and Inducible Adenine Base Editor for Precise
in Vivo Base
Editing. Nat. Commun. 2023;14(1):5573.  doi: 10.1038/s41467-023-41331-5.</cite> [<a href="https://doi.org/10.1038/s41467-023-41331-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10495389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37696818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=A%20Split%20and%20Inducible%20Adenine%20Base%20Editor%20for%20Precise%20in%20Vivo%20Base%20Editing&amp;author=H.%20Zeng&amp;author=Q.%20Yuan&amp;author=F.%20Peng&amp;author=D.%20Ma&amp;author=A.%20Lingineni&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=5573&amp;pmid=37696818&amp;doi=10.1038/s41467-023-41331-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Peters C. W., Hanlon K. S., Ivanchenko M. V., Zinn E., Linarte E. F., Li Y., Levy J. M., Liu D. R., Kleinstiver B. P., Indzhykulian A. A., Corey D. P.. Rescue of Hearing by Adenine Base
Editing in a Humanized Mouse Model of Usher Syndrome Type 1F. Mol. Ther. 2023;31(8):2439–2453. doi: 10.1016/j.ymthe.2023.06.007.</cite> [<a href="https://doi.org/10.1016/j.ymthe.2023.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10421997/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37312453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Ther.&amp;title=Rescue%20of%20Hearing%20by%20Adenine%20Base%20Editing%20in%20a%20Humanized%20Mouse%20Model%20of%20Usher%20Syndrome%20Type%201F&amp;author=C.%20W.%20Peters&amp;author=K.%20S.%20Hanlon&amp;author=M.%20V.%20Ivanchenko&amp;author=E.%20Zinn&amp;author=E.%20F.%20Linarte&amp;volume=31&amp;issue=8&amp;publication_year=2023&amp;pages=2439-2453&amp;pmid=37312453&amp;doi=10.1016/j.ymthe.2023.06.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Wu S.. et al. Base Editing Effectively
Prevents Early-Onset Severe Cardiomyopathy
in Mybpc3Mutant Mice. Cell Res. 2024:00930–00937. doi: 10.1038/s41422-024-00930-7.</cite> [<a href="https://doi.org/10.1038/s41422-024-00930-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10978934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38337022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Res.&amp;title=Base%20Editing%20Effectively%20Prevents%20Early-Onset%20Severe%20Cardiomyopathy%20in%20Mybpc3Mutant%20Mice&amp;author=S.%20Wu&amp;publication_year=2024&amp;pages=00930-00937&amp;pmid=38337022&amp;doi=10.1038/s41422-024-00930-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Villiger L., Grisch-Chan H. M., Lindsay H., Ringnalda F., Pogliano C. B., Allegri G., Fingerhut R., Häberle J., Matos J., Robinson M. D., Thöny B., Schwank G.. Treatment of a Metabolic Liver Disease by in Vivo Genome
Base Editing in Adult Mice. Nat. Med. 2018;24(10):1519–1525. doi: 10.1038/s41591-018-0209-1.</cite> [<a href="https://doi.org/10.1038/s41591-018-0209-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30297904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&amp;title=Treatment%20of%20a%20Metabolic%20Liver%20Disease%20by%20in%20Vivo%20Genome%20Base%20Editing%20in%20Adult%20Mice&amp;author=L.%20Villiger&amp;author=H.%20M.%20Grisch-Chan&amp;author=H.%20Lindsay&amp;author=F.%20Ringnalda&amp;author=C.%20B.%20Pogliano&amp;volume=24&amp;issue=10&amp;publication_year=2018&amp;pages=1519-1525&amp;pmid=30297904&amp;doi=10.1038/s41591-018-0209-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Villiger L., Rothgangl T., Witzigmann D., Oka R., Lin P. J. C., Qi W., Janjuha S., Berk C., Ringnalda F., Beattie M. B., Stoffel M., Thöny B., Hall J., Rehrauer H., van Boxtel R., Tam Y. K., Schwank G.. In Vivo Cytidine Base Editing of
Hepatocytes without Detectable Off-Target Mutations in RNA and DNA. Nature Biomed. Eng. 2021;5(2):179–189. doi: 10.1038/s41551-020-00671-z.</cite> [<a href="https://doi.org/10.1038/s41551-020-00671-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7610981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33495639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biomed.%20Eng.&amp;title=In%20Vivo%20Cytidine%20Base%20Editing%20of%20Hepatocytes%20without%20Detectable%20Off-Target%20Mutations%20in%20RNA%20and%20DNA&amp;author=L.%20Villiger&amp;author=T.%20Rothgangl&amp;author=D.%20Witzigmann&amp;author=R.%20Oka&amp;author=P.%20J.%20C.%20Lin&amp;volume=5&amp;issue=2&amp;publication_year=2021&amp;pages=179-189&amp;pmid=33495639&amp;doi=10.1038/s41551-020-00671-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Liu Q., Chen Y., Hu S., Liu W., Xie D., Yang X., Huang W., Liu S., Chen X., Liu H., Huang J.. Screening an Effective
Dual-Adeno-Associated Virus
Split-Cytosine Base Editor System for C-To-T Conversion in Vivo. Human Gene Ther. 2023;34(13–14):629–638. doi: 10.1089/hum.2023.055.</cite> [<a href="https://doi.org/10.1089/hum.2023.055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37279283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Gene%20Ther.&amp;title=Screening%20an%20Effective%20Dual-Adeno-Associated%20Virus%20Split-Cytosine%20Base%20Editor%20System%20for%20C-To-T%20Conversion%20in%20Vivo&amp;author=Q.%20Liu&amp;author=Y.%20Chen&amp;author=S.%20Hu&amp;author=W.%20Liu&amp;author=D.%20Xie&amp;volume=34&amp;issue=13%E2%80%9314&amp;publication_year=2023&amp;pages=629-638&amp;pmid=37279283&amp;doi=10.1089/hum.2023.055&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>

Yun, S.

et al. 
Engineered
CRISPR-Base Eds. as a Permanent Treatment for Familial Dysautonomia. In BioRxiv; Preprint, 2024.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=BioRxiv&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Zhi S., Chen Y., Wu G., Wen J., Wu J., Liu Q., Li Y., Kang R., Hu S., Wang J., Liang P., Huang J.. Dual-AAV Delivering
Split Prime Editor
System for in Vivo Genome Editing. Mol. Ther. 2022;30(1):283–294. doi: 10.1016/j.ymthe.2021.07.011.</cite> [<a href="https://doi.org/10.1016/j.ymthe.2021.07.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8753371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34298129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Ther.&amp;title=Dual-AAV%20Delivering%20Split%20Prime%20Editor%20System%20for%20in%20Vivo%20Genome%20Editing&amp;author=S.%20Zhi&amp;author=Y.%20Chen&amp;author=G.%20Wu&amp;author=J.%20Wen&amp;author=J.%20Wu&amp;volume=30&amp;issue=1&amp;publication_year=2022&amp;pages=283-294&amp;pmid=34298129&amp;doi=10.1016/j.ymthe.2021.07.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>She K., Liu Y., Zhao Q.. et al. Dual-AAV Split Prime Editor Corrects the Mutation
and Phenotype in Mice with Inherited Retinal Degeneration. Signal Transduction Targeted Ther. 2023;8:57.  doi: 10.1038/s41392-022-01234-1.</cite> [<a href="https://doi.org/10.1038/s41392-022-01234-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9899767/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36740702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduction%20Targeted%20Ther.&amp;title=Dual-AAV%20Split%20Prime%20Editor%20Corrects%20the%20Mutation%20and%20Phenotype%20in%20Mice%20with%20Inherited%20Retinal%20Degeneration&amp;author=K.%20She&amp;author=Y.%20Liu&amp;author=Q.%20Zhao&amp;volume=8&amp;publication_year=2023&amp;pages=57&amp;pmid=36740702&amp;doi=10.1038/s41392-022-01234-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Wei R., Yu Z., Ding L., Lu Z., Yao K., Zhang H., Huang B., He M., Ma L.. Improved Split Prime
Editors Enable Efficient in Vivo Genome Editing. Cell Rep. 2025;44(1):115144.  doi: 10.1016/j.celrep.2024.115144.</cite> [<a href="https://doi.org/10.1016/j.celrep.2024.115144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39745853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=Improved%20Split%20Prime%20Editors%20Enable%20Efficient%20in%20Vivo%20Genome%20Editing&amp;author=R.%20Wei&amp;author=Z.%20Yu&amp;author=L.%20Ding&amp;author=Z.%20Lu&amp;author=K.%20Yao&amp;volume=44&amp;issue=1&amp;publication_year=2025&amp;pages=115144&amp;pmid=39745853&amp;doi=10.1016/j.celrep.2024.115144&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Xia H.-F., Luo J.-P., Yu S.-R., Zhou T.-J.. Modification
of
C-Segment of Cfa DnaE Split Intein for Improving Clean-In-Place in
Chromatography Process. Biotechnology Prog. 2022;38(5):e3266.  doi: 10.1002/btpr.3266.</cite> [<a href="https://doi.org/10.1002/btpr.3266" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35488391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnology%20Prog.&amp;title=Modification%20of%20C-Segment%20of%20Cfa%20DnaE%20Split%20Intein%20for%20Improving%20Clean-In-Place%20in%20Chromatography%20Process&amp;author=H.-F.%20Xia&amp;author=J.-P.%20Luo&amp;author=S.-R.%20Yu&amp;author=T.-J.%20Zhou&amp;volume=38&amp;issue=5&amp;publication_year=2022&amp;pages=e3266&amp;pmid=35488391&amp;doi=10.1002/btpr.3266&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>

Humberg, C.

et al. 
A
Cysteine-Less and Ultra-Fast Split Intein Rationally Engineered from
Being Aggregation-Prone to Highly Efficient in Protein Trans-Splicing. In BioRxiv; Preprint, 2025.</cite> [<a href="https://doi.org/10.1038/s41467-025-57596-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11923092/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40108172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=BioRxiv&amp;publication_year=2025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Hiltunen M.
K., Beyer H. M., Iwaï H.. Mini-Intein Structures from Extremophiles
Suggest a Strategy for Finding Novel Robust Inteins. Microorganisms. 2021;9(6):1226. doi: 10.3390/microorganisms9061226.</cite> [<a href="https://doi.org/10.3390/microorganisms9061226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8229266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34198729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microorganisms&amp;title=Mini-Intein%20Structures%20from%20Extremophiles%20Suggest%20a%20Strategy%20for%20Finding%20Novel%20Robust%20Inteins&amp;author=M.%20K.%20Hiltunen&amp;author=H.%20M.%20Beyer&amp;author=H.%20Iwa%C3%AF&amp;volume=9&amp;issue=6&amp;publication_year=2021&amp;pages=1226&amp;pmid=34198729&amp;doi=10.3390/microorganisms9061226&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Boral S., Maiti S., Basak A. J., Lee W., De S.. Structural,
Dynamic, and Functional Characterization of a DnaX Mini-Intein Derived
from Spirulina Platensis Provides Important Insights into Intein-Mediated
Catalysis of Protein Splicing. Biochemistry. 2020;59(50):4711–4724. doi: 10.1021/acs.biochem.0c00828.</cite> [<a href="https://doi.org/10.1021/acs.biochem.0c00828" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12021019/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33289560/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Structural,%20Dynamic,%20and%20Functional%20Characterization%20of%20a%20DnaX%20Mini-Intein%20Derived%20from%20Spirulina%20Platensis%20Provides%20Important%20Insights%20into%20Intein-Mediated%20Catalysis%20of%20Protein%20Splicing&amp;author=S.%20Boral&amp;author=S.%20Maiti&amp;author=A.%20J.%20Basak&amp;author=W.%20Lee&amp;author=S.%20De&amp;volume=59&amp;issue=50&amp;publication_year=2020&amp;pages=4711-4724&amp;pmid=33289560&amp;doi=10.1021/acs.biochem.0c00828&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Heikkinen H. A., Aranko A. S., Iwaï H.. The NMR Structure of the Engineered
Halophilic DnaE Intein for Segmental Isotopic Labeling Using Conditional
Protein Splicing. J. Magn. Reson. 2022;338:107195.  doi: 10.1016/j.jmr.2022.107195.</cite> [<a href="https://doi.org/10.1016/j.jmr.2022.107195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35398651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Magn.%20Reson.&amp;title=The%20NMR%20Structure%20of%20the%20Engineered%20Halophilic%20DnaE%20Intein%20for%20Segmental%20Isotopic%20Labeling%20Using%20Conditional%20Protein%20Splicing&amp;author=H.%20A.%20Heikkinen&amp;author=A.%20S.%20Aranko&amp;author=H.%20Iwa%C3%AF&amp;volume=338&amp;publication_year=2022&amp;pages=107195&amp;pmid=35398651&amp;doi=10.1016/j.jmr.2022.107195&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Shah N. H., Eryilmaz E., Cowburn D., Muir T. W.. Naturally Split
Inteins Assemble through a “Capture and Collapse” Mechanism. J. Am. Chem. Soc. 2013;135:18673–18681. doi: 10.1021/ja4104364.</cite> [<a href="https://doi.org/10.1021/ja4104364" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3865799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24236406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Naturally%20Split%20Inteins%20Assemble%20through%20a%20%E2%80%9CCapture%20and%20Collapse%E2%80%9D%20Mechanism&amp;author=N.%20H.%20Shah&amp;author=E.%20Eryilmaz&amp;author=D.%20Cowburn&amp;author=T.%20W.%20Muir&amp;volume=135&amp;publication_year=2013&amp;pages=18673-18681&amp;pmid=24236406&amp;doi=10.1021/ja4104364&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Ludwig C., Schwarzer D., Mootz H. D.. Interaction Studies and Alanine Scanning
Analysis of a Semi-Synthetic Split Intein Reveal Thiazoline Ring Formation
from an Intermediate of the Protein Splicing Reaction. J. Biol. Chem. 2008;283:25264–25272. doi: 10.1074/jbc.M802972200.</cite> [<a href="https://doi.org/10.1074/jbc.M802972200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18625708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Interaction%20Studies%20and%20Alanine%20Scanning%20Analysis%20of%20a%20Semi-Synthetic%20Split%20Intein%20Reveal%20Thiazoline%20Ring%20Formation%20from%20an%20Intermediate%20of%20the%20Protein%20Splicing%20Reaction&amp;author=C.%20Ludwig&amp;author=D.%20Schwarzer&amp;author=H.%20D.%20Mootz&amp;volume=283&amp;publication_year=2008&amp;pages=25264-25272&amp;pmid=18625708&amp;doi=10.1074/jbc.M802972200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Wasmuth A., Ludwig C., Mootz H. D.. Structure-Activity Studies on the
Upstream Splice Junction of a Semisynthetic Intein. Bioorg. Med. Chem. 2013;21:3495–3503. doi: 10.1016/j.bmc.2013.03.065.</cite> [<a href="https://doi.org/10.1016/j.bmc.2013.03.065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23618706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Med.%20Chem.&amp;title=Structure-Activity%20Studies%20on%20the%20Upstream%20Splice%20Junction%20of%20a%20Semisynthetic%20Intein&amp;author=A.%20Wasmuth&amp;author=C.%20Ludwig&amp;author=H.%20D.%20Mootz&amp;volume=21&amp;publication_year=2013&amp;pages=3495-3503&amp;pmid=23618706&amp;doi=10.1016/j.bmc.2013.03.065&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Stevens A. J., Brown Z. Z., Shah N. H., Sekar G., Cowburn D., Muir T. W.. Design of a Split Intein with Exceptional
Protein Splicing
Activity. J. Am. Chem. Soc. 2016;138(7):2162–2165. doi: 10.1021/jacs.5b13528.</cite> [<a href="https://doi.org/10.1021/jacs.5b13528" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4894280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26854538/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Design%20of%20a%20Split%20Intein%20with%20Exceptional%20Protein%20Splicing%20Activity&amp;author=A.%20J.%20Stevens&amp;author=Z.%20Z.%20Brown&amp;author=N.%20H.%20Shah&amp;author=G.%20Sekar&amp;author=D.%20Cowburn&amp;volume=138&amp;issue=7&amp;publication_year=2016&amp;pages=2162-2165&amp;pmid=26854538&amp;doi=10.1021/jacs.5b13528&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Eryilmaz E., Shah N. H., Muir T. W., Cowburn D.. Structural and Dynamical
Features of Inteins and Implications on Protein Splicing. J. Biol. Chem. 2014;289(21):14506–14511. doi: 10.1074/jbc.R113.540302.</cite> [<a href="https://doi.org/10.1074/jbc.R113.540302" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4031508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Structural%20and%20Dynamical%20Features%20of%20Inteins%20and%20Implications%20on%20Protein%20Splicing&amp;author=E.%20Eryilmaz&amp;author=N.%20H.%20Shah&amp;author=T.%20W.%20Muir&amp;author=D.%20Cowburn&amp;volume=289&amp;issue=21&amp;publication_year=2014&amp;pages=14506-14511&amp;pmid=24695731&amp;doi=10.1074/jbc.R113.540302&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Pietrokovski S.. Intein Spread
and Extinction in Evolution. Trends Genet. 2001;17:465–472. doi: 10.1016/S0168-9525(01)02365-4.</cite> [<a href="https://doi.org/10.1016/S0168-9525(01)02365-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11485819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Genet.&amp;title=Intein%20Spread%20and%20Extinction%20in%20Evolution&amp;author=S.%20Pietrokovski&amp;volume=17&amp;publication_year=2001&amp;pages=465-472&amp;pmid=11485819&amp;doi=10.1016/S0168-9525(01)02365-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Shah N. H., Dann G. P., Vila-Perelló M., Liu Z., Muir T. W.. Ultrafast
Protein Splicing Is Common among Cyanobacterial Split Inteins: Implications
for Protein Engineering. J. Am. Chem. Soc. 2012;134(28):11338–11341. doi: 10.1021/ja303226x.</cite> [<a href="https://doi.org/10.1021/ja303226x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3535263/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22734434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Ultrafast%20Protein%20Splicing%20Is%20Common%20among%20Cyanobacterial%20Split%20Inteins:%20Implications%20for%20Protein%20Engineering&amp;author=N.%20H.%20Shah&amp;author=G.%20P.%20Dann&amp;author=M.%20Vila-Perell%C3%B3&amp;author=Z.%20Liu&amp;author=T.%20W.%20Muir&amp;volume=134&amp;issue=28&amp;publication_year=2012&amp;pages=11338-11341&amp;pmid=22734434&amp;doi=10.1021/ja303226x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34"><cite>
RCSB PDB: Homepage.
<a href="https://www.rcsb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org/</a> (accessed
June 01, 2024).</cite></li>
<li id="ref35">
<cite>Sastry G. M., Adzhigirey M., Day T., Annabhimoju R., Sherman W.. Protein and Ligand Preparation: Parameters,
Protocols,
and Influence on Virtual Screening Enrichments. J. Comput. Mol. Des. 2013;53:221–234. doi: 10.1021/ci4005145.</cite> [<a href="https://doi.org/10.1021/ci4005145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23579614/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Mol.%20Des.&amp;title=Protein%20and%20Ligand%20Preparation:%20Parameters,%20Protocols,%20and%20Influence%20on%20Virtual%20Screening%20Enrichments&amp;author=G.%20M.%20Sastry&amp;author=M.%20Adzhigirey&amp;author=T.%20Day&amp;author=R.%20Annabhimoju&amp;author=W.%20Sherman&amp;volume=53&amp;publication_year=2013&amp;pages=221-234&amp;pmid=23579614&amp;doi=10.1021/ci4005145&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Jacobson M. P., Friesner R. A., Xiang Z., Honig B.. On the Role of the
Crystal Environment in Determining Protein Side-Chain Conformations. J. Mol. Biol. 2002;320:597–608. doi: 10.1016/S0022-2836(02)00470-9.</cite> [<a href="https://doi.org/10.1016/S0022-2836(02)00470-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12096912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=On%20the%20Role%20of%20the%20Crystal%20Environment%20in%20Determining%20Protein%20Side-Chain%20Conformations&amp;author=M.%20P.%20Jacobson&amp;author=R.%20A.%20Friesner&amp;author=Z.%20Xiang&amp;author=B.%20Honig&amp;volume=320&amp;publication_year=2002&amp;pages=597-608&amp;pmid=12096912&amp;doi=10.1016/S0022-2836(02)00470-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Lu C., Wu C., Ghoreishi D., Chen W., Wang L., Damm W., Ross G. A., Dahlgren M. K., Russell E., Von Bargen C. D., Abel R., Friesner R. A., Harder E. D.. OPLS4:
Improving
Force Field Accuracy on Challenging Regimes of Chemical Space. J. Chem. Theory Comput. 2021;17:4291–4300. doi: 10.1021/acs.jctc.1c00302.</cite> [<a href="https://doi.org/10.1021/acs.jctc.1c00302" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34096718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=OPLS4:%20Improving%20Force%20Field%20Accuracy%20on%20Challenging%20Regimes%20of%20Chemical%20Space&amp;author=C.%20Lu&amp;author=C.%20Wu&amp;author=D.%20Ghoreishi&amp;author=W.%20Chen&amp;author=L.%20Wang&amp;volume=17&amp;publication_year=2021&amp;pages=4291-4300&amp;pmid=34096718&amp;doi=10.1021/acs.jctc.1c00302&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Ramachandran G. N., Ramakrishnan C., Sasisekharan V.. Stereochemistry
of Polypeptide Chain
Configurations. J. Mol. Biol. 1963;7:95–99. doi: 10.1016/S0022-2836(63)80023-6.</cite> [<a href="https://doi.org/10.1016/S0022-2836(63)80023-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/13990617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Stereochemistry%20of%20Polypeptide%20Chain%20Configurations&amp;author=G.%20N.%20Ramachandran&amp;author=C.%20Ramakrishnan&amp;author=V.%20Sasisekharan&amp;volume=7&amp;publication_year=1963&amp;pages=95-99&amp;pmid=13990617&amp;doi=10.1016/S0022-2836(63)80023-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Kozakov D., Brenke R., Comeau S. R., Vajda S.. PIPER: An FFT-Based
Protein Docking Program with Pairwise Potentials. Proteins. 2006;65:392–406. doi: 10.1002/prot.21117.</cite> [<a href="https://doi.org/10.1002/prot.21117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16933295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=PIPER:%20An%20FFT-Based%20Protein%20Docking%20Program%20with%20Pairwise%20Potentials&amp;author=D.%20Kozakov&amp;author=R.%20Brenke&amp;author=S.%20R.%20Comeau&amp;author=S.%20Vajda&amp;volume=65&amp;publication_year=2006&amp;pages=392-406&amp;pmid=16933295&amp;doi=10.1002/prot.21117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Brovin A., Minskaia E., Sabantsev M., Chuvpilo S., Karabelsky A.. Protein Trans-Splicing:
Optimization of Intein-Mediated GFP Assembly as a Model for the Development
of Gene Therapy. Front. Bioeng. Biotechnol. 2024;12:1488912.  doi: 10.3389/fbioe.2024.1488912.</cite> [<a href="https://doi.org/10.3389/fbioe.2024.1488912" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11614648/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39634100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Bioeng.%20Biotechnol.&amp;title=Protein%20Trans-Splicing:%20Optimization%20of%20Intein-Mediated%20GFP%20Assembly%20as%20a%20Model%20for%20the%20Development%20of%20Gene%20Therapy&amp;author=A.%20Brovin&amp;author=E.%20Minskaia&amp;author=M.%20Sabantsev&amp;author=S.%20Chuvpilo&amp;author=A.%20Karabelsky&amp;volume=12&amp;publication_year=2024&amp;pages=1488912&amp;pmid=39634100&amp;doi=10.3389/fbioe.2024.1488912&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Perler F. B.. InBase:
The Intein Database. Nucleic Acids Res. 2002;30(1):383–384. doi: 10.1093/nar/30.1.383.</cite> [<a href="https://doi.org/10.1093/nar/30.1.383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC99080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11752343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=InBase:%20The%20Intein%20Database&amp;author=F.%20B.%20Perler&amp;volume=30&amp;issue=1&amp;publication_year=2002&amp;pages=383-384&amp;pmid=11752343&amp;doi=10.1093/nar/30.1.383&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Dearden A. K., Nayak S. K.. Proteins: Ssp DnaE Intein. Multiscale
Model. Biomech. Mechanobiol. 2015:3–12. doi: 10.1007/978-1-4471-6599-6_1.</cite> [<a href="https://doi.org/10.1007/978-1-4471-6599-6_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Multiscale%20Model.%20Biomech.%20Mechanobiol.&amp;title=Proteins:%20Ssp%20DnaE%20Intein&amp;author=A.%20K.%20Dearden&amp;author=S.%20K.%20Nayak&amp;publication_year=2015&amp;pages=3-12&amp;doi=10.1007/978-1-4471-6599-6_1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Nichols N. M., Evans T. C.. Mutational Analysis of Protein Splicing, Cleavage,
and Self-Association Reactions Mediated by the Naturally Split Ssp
DnaE Intein. Biochemistry. 2004;43(31):10265–10276. doi: 10.1021/bi0494065.</cite> [<a href="https://doi.org/10.1021/bi0494065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15287754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Mutational%20Analysis%20of%20Protein%20Splicing,%20Cleavage,%20and%20Self-Association%20Reactions%20Mediated%20by%20the%20Naturally%20Split%20Ssp%20DnaE%20Intein&amp;author=N.%20M.%20Nichols&amp;author=T.%20C.%20Evans&amp;volume=43&amp;issue=31&amp;publication_year=2004&amp;pages=10265-10276&amp;pmid=15287754&amp;doi=10.1021/bi0494065&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Kick L. M., Harteis S., Koch M. F., Schneider S.. Mechanistic
Insights into Cyclic Peptide Generation by DnaE Split-Inteins through
Quantitative and Structural Investigation. ChemBioChem. 2017;18(22):2242–2246. doi: 10.1002/cbic.201700503.</cite> [<a href="https://doi.org/10.1002/cbic.201700503" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28914478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ChemBioChem&amp;title=Mechanistic%20Insights%20into%20Cyclic%20Peptide%20Generation%20by%20DnaE%20Split-Inteins%20through%20Quantitative%20and%20Structural%20Investigation&amp;author=L.%20M.%20Kick&amp;author=S.%20Harteis&amp;author=M.%20F.%20Koch&amp;author=S.%20Schneider&amp;volume=18&amp;issue=22&amp;publication_year=2017&amp;pages=2242-2246&amp;pmid=28914478&amp;doi=10.1002/cbic.201700503&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Panda S., Rout M., Mishra S., Turuk J., Pati S., Dehury B.. Molecular Docking and MD Simulations Reveal Protease
Inhibitors Block the Catalytic Residues in Prp8 Intein of Aspergillus
Fumigatus: a Potential Target for Antimycotics. J. Biomol. Struct. Dyn. 2023:3526–3541. doi: 10.1080/07391102.2023.2298735.</cite> [<a href="https://doi.org/10.1080/07391102.2023.2298735" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38149850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20Struct.%20Dyn.&amp;title=Molecular%20Docking%20and%20MD%20Simulations%20Reveal%20Protease%20Inhibitors%20Block%20the%20Catalytic%20Residues%20in%20Prp8%20Intein%20of%20Aspergillus%20Fumigatus:%20a%20Potential%20Target%20for%20Antimycotics&amp;author=S.%20Panda&amp;author=M.%20Rout&amp;author=S.%20Mishra&amp;author=J.%20Turuk&amp;author=S.%20Pati&amp;publication_year=2023&amp;pages=3526-3541&amp;pmid=38149850&amp;doi=10.1080/07391102.2023.2298735&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Cronin M., Coolbaugh M. J., Nellis D., Zhu J., Wood D. W., Nussinov R., Ma B.. Dynamics Differentiate between Active
and Inactive Inteins. European J. Med. Chem. 2015;91:51–62. doi: 10.1016/j.ejmech.2014.07.094.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2014.07.094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4308580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25087201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20J.%20Med.%20Chem.&amp;title=Dynamics%20Differentiate%20between%20Active%20and%20Inactive%20Inteins&amp;author=M.%20Cronin&amp;author=M.%20J.%20Coolbaugh&amp;author=D.%20Nellis&amp;author=J.%20Zhu&amp;author=D.%20W.%20Wood&amp;volume=91&amp;publication_year=2015&amp;pages=51-62&amp;pmid=25087201&amp;doi=10.1016/j.ejmech.2014.07.094&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>Karp J. M., Eryilmaz E., Cowburn D.. Correlation
of Chemical Shifts Predicted
by Molecular Dynamics Simulations for Partially Disordered Proteins. J. Biomol. NMR. 2015;61(1):35–45. doi: 10.1007/s10858-014-9879-2.</cite> [<a href="https://doi.org/10.1007/s10858-014-9879-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4715900/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25416617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20NMR&amp;title=Correlation%20of%20Chemical%20Shifts%20Predicted%20by%20Molecular%20Dynamics%20Simulations%20for%20Partially%20Disordered%20Proteins&amp;author=J.%20M.%20Karp&amp;author=E.%20Eryilmaz&amp;author=D.%20Cowburn&amp;volume=61&amp;issue=1&amp;publication_year=2015&amp;pages=35-45&amp;pmid=25416617&amp;doi=10.1007/s10858-014-9879-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Wetie A. G. N., Sokolowska I., Woods A. G., Roy U., Deinhardt K., Darie C. C.. Protein–Protein Interactions:
Switch from Classical
Methods to Proteomics and Bioinformatics-Based Approaches. Cell. Mol. Life Sci. 2014;71(2):205–228. doi: 10.1007/s00018-013-1333-1.</cite> [<a href="https://doi.org/10.1007/s00018-013-1333-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11113707/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23579629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell.%20Mol.%20Life%20Sci.&amp;title=Protein%E2%80%93Protein%20Interactions:%20Switch%20from%20Classical%20Methods%20to%20Proteomics%20and%20Bioinformatics-Based%20Approaches&amp;author=A.%20G.%20N.%20Wetie&amp;author=I.%20Sokolowska&amp;author=A.%20G.%20Woods&amp;author=U.%20Roy&amp;author=K.%20Deinhardt&amp;volume=71&amp;issue=2&amp;publication_year=2014&amp;pages=205-228&amp;pmid=23579629&amp;doi=10.1007/s00018-013-1333-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Sekar G., Stevens A. J., Mostafavi A. Z., Sashi P., Muir T. W., Cowburn D.. A Conserved Histidine
Residue Drives Extein Dependence
in an Enhanced Atypically Split Intein. J. Am.
Chem. Soc. 2022;144(41):19196–19203. doi: 10.1021/jacs.2c08985.</cite> [<a href="https://doi.org/10.1021/jacs.2c08985" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10241006/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36194550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=A%20Conserved%20Histidine%20Residue%20Drives%20Extein%20Dependence%20in%20an%20Enhanced%20Atypically%20Split%20Intein&amp;author=G.%20Sekar&amp;author=A.%20J.%20Stevens&amp;author=A.%20Z.%20Mostafavi&amp;author=P.%20Sashi&amp;author=T.%20W.%20Muir&amp;volume=144&amp;issue=41&amp;publication_year=2022&amp;pages=19196-19203&amp;pmid=36194550&amp;doi=10.1021/jacs.2c08985&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Li R., Jing Q., She K., Wang Q., Jin X., Zhao Q., Su J., Song L., Fu J., Wu X., Xu Q., Lu F., Wei Y., Yang Y.. Split AAV8Mediated
ABCA4 Expression for Gene Therapy of Mouse Stargardt Disease (STGD1) Human Gene Ther. 2023;34(13–14):616–628. doi: 10.1089/hum.2023.017.</cite> [<a href="https://doi.org/10.1089/hum.2023.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37227014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Gene%20Ther.&amp;title=Split%20AAV8Mediated%20ABCA4%20Expression%20for%20Gene%20Therapy%20of%20Mouse%20Stargardt%20Disease%20(STGD1)&amp;author=R.%20Li&amp;author=Q.%20Jing&amp;author=K.%20She&amp;author=Q.%20Wang&amp;author=X.%20Jin&amp;volume=34&amp;issue=13%E2%80%9314&amp;publication_year=2023&amp;pages=616-628&amp;pmid=37227014&amp;doi=10.1089/hum.2023.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Purkayastha A., Kang T. J.. Stabilization of
Proteins by Covalent Cyclization. Biotechnol.
Bioprocess. Eng. 2019;24(5):702–712. doi: 10.1007/s12257-019-0363-4.</cite> [<a href="https://doi.org/10.1007/s12257-019-0363-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol.%20Bioprocess.%20Eng.&amp;title=Stabilization%20of%20Proteins%20by%20Covalent%20Cyclization&amp;author=A.%20Purkayastha&amp;author=T.%20J.%20Kang&amp;volume=24&amp;issue=5&amp;publication_year=2019&amp;pages=702-712&amp;doi=10.1007/s12257-019-0363-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>Oeemig J. S., Zhou D., Kajander T., Wlodawer A., Iwaï H.. NMR and Crystal
Structures of the Pyrococcus Horikoshii RadA Intein Guide a Strategy
for Engineering a Highly Efficient and Promiscuous Intein. J. Mol. Biol. 2012;421(1):85–99. doi: 10.1016/j.jmb.2012.04.029.</cite> [<a href="https://doi.org/10.1016/j.jmb.2012.04.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3392434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22560994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=NMR%20and%20Crystal%20Structures%20of%20the%20Pyrococcus%20Horikoshii%20RadA%20Intein%20Guide%20a%20Strategy%20for%20Engineering%20a%20Highly%20Efficient%20and%20Promiscuous%20Intein&amp;author=J.%20S.%20Oeemig&amp;author=D.%20Zhou&amp;author=T.%20Kajander&amp;author=A.%20Wlodawer&amp;author=H.%20Iwa%C3%AF&amp;volume=421&amp;issue=1&amp;publication_year=2012&amp;pages=85-99&amp;pmid=22560994&amp;doi=10.1016/j.jmb.2012.04.029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>Oeemig J. S., Beyer H. M., Aranko A. S., Mutanen J., Iwaï H.. Substrate
Specificities of Inteins Investigated by QuickDrop-Cassette Mutagenesis. FEBS Lett. 2020;594(20):3338–3355. doi: 10.1002/1873-3468.13909.</cite> [<a href="https://doi.org/10.1002/1873-3468.13909" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32805768/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&amp;title=Substrate%20Specificities%20of%20Inteins%20Investigated%20by%20QuickDrop-Cassette%20Mutagenesis&amp;author=J.%20S.%20Oeemig&amp;author=H.%20M.%20Beyer&amp;author=A.%20S.%20Aranko&amp;author=J.%20Mutanen&amp;author=H.%20Iwa%C3%AF&amp;volume=594&amp;issue=20&amp;publication_year=2020&amp;pages=3338-3355&amp;pmid=32805768&amp;doi=10.1002/1873-3468.13909&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>Carvajal-Vallejos P., Pallissé R., Mootz H. D., Schmidt S. R.. Unprecedented Rates
and Efficiencies Revealed for New Natural Split Inteins from Metagenomic
Sources. J. Biol. Chem. 2012;287(34):28686–28696. doi: 10.1074/jbc.M112.372680.</cite> [<a href="https://doi.org/10.1074/jbc.M112.372680" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3436554/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22753413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Unprecedented%20Rates%20and%20Efficiencies%20Revealed%20for%20New%20Natural%20Split%20Inteins%20from%20Metagenomic%20Sources&amp;author=P.%20Carvajal-Vallejos&amp;author=R.%20Palliss%C3%A9&amp;author=H.%20D.%20Mootz&amp;author=S.%20R.%20Schmidt&amp;volume=287&amp;issue=34&amp;publication_year=2012&amp;pages=28686-28696&amp;pmid=22753413&amp;doi=10.1074/jbc.M112.372680&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>Beyer H. M., Mikula K. M., Li M., Wlodawer A., Iwaï H.. The Crystal
Structure of the Naturally Split Gp41–1 Intein Guides the Engineering
of Orthogonal Split Inteins Fromcis-Splicing Inteins. FEBS J. 2020;287(9):1886–1898. doi: 10.1111/febs.15113.</cite> [<a href="https://doi.org/10.1111/febs.15113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7190452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31665813/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20J&amp;title=The%20Crystal%20Structure%20of%20the%20Naturally%20Split%20Gp41%E2%80%931%20Intein%20Guides%20the%20Engineering%20of%20Orthogonal%20Split%20Inteins%20Fromcis-Splicing%20Inteins&amp;author=H.%20M.%20Beyer&amp;author=K.%20M.%20Mikula&amp;author=M.%20Li&amp;author=A.%20Wlodawer&amp;author=H.%20Iwa%C3%AF&amp;volume=287&amp;issue=9&amp;publication_year=2020&amp;pages=1886-1898&amp;pmid=31665813&amp;doi=10.1111/febs.15113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Mariano A., Di Cristofano S., Raimondo D., D’Abusco A. S.. Split Gp41–1
Intein Splicing as a Model to Evaluate the Cellular Location of the
Oncosuppressor Maspin in an in Vitro Model of Osteosarcoma. Cell Biochem. Funct. 2024;42(2):e3987.  doi: 10.1002/cbf.3987.</cite> [<a href="https://doi.org/10.1002/cbf.3987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38509770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Biochem.%20Funct.&amp;title=Split%20Gp41%E2%80%931%20Intein%20Splicing%20as%20a%20Model%20to%20Evaluate%20the%20Cellular%20Location%20of%20the%20Oncosuppressor%20Maspin%20in%20an%20in%20Vitro%20Model%20of%20Osteosarcoma&amp;author=A.%20Mariano&amp;author=S.%20Di%20Cristofano&amp;author=D.%20Raimondo&amp;author=A.%20S.%20D%E2%80%99Abusco&amp;volume=42&amp;issue=2&amp;publication_year=2024&amp;pages=e3987&amp;pmid=38509770&amp;doi=10.1002/cbf.3987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref57">
<cite>Kothawala A., Kilpatrick K., Novoa J. A., Segatori L.. Quantitative Analysis
of α-Synuclein Solubility in Living Cells Using Split GFP Complementation. PLoS One. 2012;7(8):e43505.  doi: 10.1371/journal.pone.0043505.</cite> [<a href="https://doi.org/10.1371/journal.pone.0043505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3425482/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22927976/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Quantitative%20Analysis%20of%20%CE%B1-Synuclein%20Solubility%20in%20Living%20Cells%20Using%20Split%20GFP%20Complementation&amp;author=A.%20Kothawala&amp;author=K.%20Kilpatrick&amp;author=J.%20A.%20Novoa&amp;author=L.%20Segatori&amp;volume=7&amp;issue=8&amp;publication_year=2012&amp;pages=e43505&amp;pmid=22927976&amp;doi=10.1371/journal.pone.0043505&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371501/bin/bg5c00091_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg5c00091_si_001.pdf</a><sup> (1.8MB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Bio &amp; Med Chem Au are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsbiomedchemau.5c00091"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/bg5c00091.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371501/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371501/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371501%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371501/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371501/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371501/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371501/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371501/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371501/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="989bc2oL7CzLLvaK5zp1T1JamM9yuzeTWDchce46ICVZ0e1j6E1QIKrQzS4UEfNg">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
