
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Diversity of cereal rye (Secale cereale) germplasm in the Southeast United States - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="3D96DAB38094B9F300DAB30038EF72BF.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11997422/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Diversity of cereal rye (Secale cereale) germplasm in the Southeast United States">
<meta name="citation_author" content="Lauren J Brzozowski">
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, University of Kentucky, Lexington, Kentucky, USA">
<meta name="citation_author" content="Timothy D Phillips">
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, University of Kentucky, Lexington, Kentucky, USA">
<meta name="citation_author" content="David A Van Sanford">
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, University of Kentucky, Lexington, Kentucky, USA">
<meta name="citation_publication_date" content="2025 Apr 14">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="2">
<meta name="citation_firstpage" content="e70008">
<meta name="citation_doi" content="10.1002/tpg2.70008">
<meta name="citation_pmid" content="40230018">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11997422/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11997422/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11997422/pdf/TPG2-18-e70008.pdf">
<meta name="description" content="Cereal rye (Secale cereale) is a grain, forage, and cover crop, with specific regional production practices. Maintaining regional varieties is challenged by rye reproductive biology, as wind pollination may dilute varietal distinction. Similarly, ...">
<meta name="og:title" content="Diversity of cereal rye (Secale cereale) germplasm in the Southeast United States">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Cereal rye (Secale cereale) is a grain, forage, and cover crop, with specific regional production practices. Maintaining regional varieties is challenged by rye reproductive biology, as wind pollination may dilute varietal distinction. Similarly, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11997422/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11997422">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70008"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70008.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11997422%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11997422/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11997422/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11997422/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70008" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Apr 14;18(2):e70008. doi: <a href="https://doi.org/10.1002/tpg2.70008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70008</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Diversity of cereal rye (<em>Secale cereale</em>) germplasm in the Southeast United States</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brzozowski%20LJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Lauren J Brzozowski</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Lauren J Brzozowski</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant and Soil Sciences, University of Kentucky, Lexington, Kentucky, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brzozowski%20LJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Lauren J Brzozowski</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Phillips%20TD%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Timothy D Phillips</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Timothy D Phillips</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant and Soil Sciences, University of Kentucky, Lexington, Kentucky, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Phillips%20TD%22%5BAuthor%5D" class="usa-link"><span class="name western">Timothy D Phillips</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20Sanford%20DA%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">David A Van Sanford</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">David A Van Sanford</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant and Soil Sciences, University of Kentucky, Lexington, Kentucky, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20Sanford%20DA%22%5BAuthor%5D" class="usa-link"><span class="name western">David A Van Sanford</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270008-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Plant and Soil Sciences, University of Kentucky, Lexington, Kentucky, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Lauren J. Brzozowski, Department of Plant and Soil Sciences, University of Kentucky, Lexington, KY 40546, USA. Email: <span>lauren.brzozowski@uky.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Feb 3; Received 2024 Nov 25; Accepted 2025 Feb 3; Issue date 2025 Jun.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11997422  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40230018/" class="usa-link">40230018</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Cereal rye (<em>Secale cereale</em>) is a grain, forage, and cover crop, with specific regional production practices. Maintaining regional varieties is challenged by rye reproductive biology, as wind pollination may dilute varietal distinction. Similarly, breeding new population varieties for regional needs lacks efficiency gains seen in other grains. We sought to address the challenges in maintaining and breeding rye population varieties in the Southeastern United States, where rye is a forage and cover crop, with emerging grain markets. Genetic and phenotypic variation within and between populations was characterized to determine varietal differentiation and test the efficacy of genomic selection for population improvement. Using 15 Southeastern rye accessions and eight breeding populations, we genotyped and phenotyped more than 500 individuals and found that most phenotypic and genetic variation is within rather than among accessions. Latitude of variety source, but not end use, contributed to intervarietal variation for heading date, but not traits associated with seed yield. Genomic prediction accuracy was moderately high (<em>r </em>&gt; 0.3) for most traits, but within‐population prediction accuracy was more variable, with only some populations having nonzero within‐population prediction accuracy. This work establishes the inter‐ and intravarietal differentiation in Southeastern rye accessions, and demonstrates viability of genomics‐enabled population improvement for regional varieties.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270008-list-10001" class="list" style="list-style-type:disc">
<li><p>Rye population breeding is important for forage, cover crop, and grain markets in the southeastern United States.</p></li>
<li><p>Most phenotypic and genetic variation resides within rye populations, rather than between distinct populations.</p></li>
<li><p>Latitude of population development is predictive of heading date and spike size, but no other traits.</p></li>
<li><p>Genomic prediction accuracy was lower within populations than for individuals across multiple accessions.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270008-dl-0001">
<dt>BLUPs</dt>
<dd><p>best linear unbiased predictors</p></dd>
<dt>GBLUP</dt>
<dd><p>genomic BLUP</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>MAF</dt>
<dd><p>minor allele frequency</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>TKW</dt>
<dd><p>1000‐kernel weight</p></dd>
</dl>
<section id="tpg270008-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Cereal rye (<em>Secale cereale</em>) is an important grain, forage, and cover crop with worldwide distribution and cultivation (Korzun et al., <a href="#tpg270008-bib-0036" class="usa-link" aria-describedby="tpg270008-bib-0036">2021</a>), in part due to its resilience to biotic and abiotic stresses (Rakoczy‐Trojanowska et al., <a href="#tpg270008-bib-0060" class="usa-link" aria-describedby="tpg270008-bib-0060">2021</a>) and unique quality attributes (Brzozowski et al., <a href="#tpg270008-bib-0013" class="usa-link" aria-describedby="tpg270008-bib-0013">2023</a>). Different regional needs shape rye cultivation practices and end uses. For instance, in more temperate regions, rye is more frequently cultivated for grain production, while in warmer climates, rye is an important pasture and forage crop (Miedaner &amp; Laidig, <a href="#tpg270008-bib-0048" class="usa-link" aria-describedby="tpg270008-bib-0048">2019</a>; Newell &amp; Butler, <a href="#tpg270008-bib-0052" class="usa-link" aria-describedby="tpg270008-bib-0052">2013</a>). Regional priorities for rye have resulted in the development of local landraces and population varieties (Adamo et al., <a href="#tpg270008-bib-0001" class="usa-link" aria-describedby="tpg270008-bib-0001">2021</a>; Larsson et al., <a href="#tpg270008-bib-0039" class="usa-link" aria-describedby="tpg270008-bib-0039">2019</a>; Monteiro et al., <a href="#tpg270008-bib-0049" class="usa-link" aria-describedby="tpg270008-bib-0049">2016</a>), and these populations are a reservoir of genetic diversity for plant breeding (Hawliczek et al., <a href="#tpg270008-bib-0029" class="usa-link" aria-describedby="tpg270008-bib-0029">2023</a>). However, methods to quantify variation in rye populations and strategies to leverage genetic variation to develop new locally adapted population varieties remain underdeveloped.</p>
<p>The degree to which historical regional landraces and population varieties harbor genetic variation has been assessed to inform rye conservation and breeding efforts. While regional varieties preserve traits essential to local production and agricultural practices, there is frequently a lack of strong genetic differentiation between accessions. In studies of regional and global rye, percentage of genetic variation attributed to within‐accessions rather than between populations exceeds 76% (Hagenblad et al., <a href="#tpg270008-bib-0026" class="usa-link" aria-describedby="tpg270008-bib-0026">2016</a>; Larsson et al., <a href="#tpg270008-bib-0039" class="usa-link" aria-describedby="tpg270008-bib-0039">2019</a>; Monteiro et al., <a href="#tpg270008-bib-0049" class="usa-link" aria-describedby="tpg270008-bib-0049">2016</a>; Parat et al., <a href="#tpg270008-bib-0053" class="usa-link" aria-describedby="tpg270008-bib-0053">2016</a>). While variation within populations often eclipses variation between populations, breeding and production practices at a continental scale have led to between‐population differentiation. For instance, the coupled forces of more intensive breeding for grain in northern Europe and less intensive selection in southern Europe for forage have contributed to population differentiation (Hagenblad et al., <a href="#tpg270008-bib-0026" class="usa-link" aria-describedby="tpg270008-bib-0026">2016</a>; Parat et al., <a href="#tpg270008-bib-0053" class="usa-link" aria-describedby="tpg270008-bib-0053">2016</a>). Similarly, specific cultivation practices in Fennoscandia have led to varietal distinction between local populations and other European rye (Larsson et al., <a href="#tpg270008-bib-0039" class="usa-link" aria-describedby="tpg270008-bib-0039">2019</a>). Generally, however, minimal geographic structuring has been reported (Bolibok‐Brągoszewska et al., <a href="#tpg270008-bib-0009" class="usa-link" aria-describedby="tpg270008-bib-0009">2014</a>; Schreiber et al., <a href="#tpg270008-bib-0064" class="usa-link" aria-describedby="tpg270008-bib-0064">2022</a>; Sidhu et al., <a href="#tpg270008-bib-0068" class="usa-link" aria-describedby="tpg270008-bib-0068">2019</a>), except for at the broad scale of continent of source (e.g., Asia vs. not Asia) (Maraci et al., <a href="#tpg270008-bib-0046" class="usa-link" aria-describedby="tpg270008-bib-0046">2018</a>; Schreiber et al., <a href="#tpg270008-bib-0065" class="usa-link" aria-describedby="tpg270008-bib-0065">2019</a>). In addition, while genetic diversity has been widely sampled, only rarely has within‐accession phenotypic variation been addressed. For example, phenotypes were measured on bulked rather than individual plants from 100 global accessions (Targońska‐Karasek et al., <a href="#tpg270008-bib-0070" class="usa-link" aria-describedby="tpg270008-bib-0070">2020</a>). Together, this shows that regional accessions reflect variation necessary for local production practices, and that more variation exists within these accessions, than exists between accessions. However, tests of genetic structuring at finer geographical scales coupled with phenotypic measurements are still lacking.</p>
<p>Assessing genetic variation and differentiation within rye populations is challenged by rye being a seed‐propagated, allogamous, wind‐pollinated crop. As is seen in landraces and regional populations, there is an evidently high degree of intermating between rye populations, and maintaining isolated, differentiated populations is not trivial. Even rye populations maintained at gene banks are susceptible to genetic drift. For instance, rye stored in a gene bank had different allele frequencies between the original seed sample (stored in herbarium) when compared with the most recent multiplication, including loss of some alleles (Chebotar et al., <a href="#tpg270008-bib-0014" class="usa-link" aria-describedby="tpg270008-bib-0014">2003</a>). Similarly, rye stored in a gene bank in Portugal was genetically distinct from on‐farm conserved rye populations (Monteiro et al., <a href="#tpg270008-bib-0049" class="usa-link" aria-describedby="tpg270008-bib-0049">2016</a>). Despite these challenges, rye populations from germplasm collections have led to advances in rye association mapping, as shown in a genome‐wide association studies (GWAS) for tan spot resistance (<em>Pyrenophora tritici‐repentis</em>) (Sidhu et al., <a href="#tpg270008-bib-0068" class="usa-link" aria-describedby="tpg270008-bib-0068">2019</a>) and winter hardiness (Båga et al., <a href="#tpg270008-bib-0003" class="usa-link" aria-describedby="tpg270008-bib-0003">2022</a>). In addition, evaluating individuals within populations has also proven useful in genetic mapping. Gruner et al. (<a href="#tpg270008-bib-0025" class="usa-link" aria-describedby="tpg270008-bib-0025">2021</a>) evaluated 100 individual plants each from five population varieties for stem rust (<em>Puccinia graminis</em> f. sp. <em>secalis</em>) resistance and found that individuals within the populations had varied responses to three different isolates, resulting in successful quantitative trait loci mapping within the populations (Gruner et al., <a href="#tpg270008-bib-0025" class="usa-link" aria-describedby="tpg270008-bib-0025">2021</a>). Together, this indicates that there can be substantial heritable phenotypic variation between and within rye population varieties, and that specifically surveying population germplasm for regional adaptation can be of value to plant breeders.</p>
<p>To accelerate rye population improvement, it will be critical to leverage new genomic resources and tools, including the rye genome (Li et al., <a href="#tpg270008-bib-0041" class="usa-link" aria-describedby="tpg270008-bib-0041">2021</a>; Rabanus‐Wallace et al., <a href="#tpg270008-bib-0059" class="usa-link" aria-describedby="tpg270008-bib-0059">2021</a>). In hybrid breeding programs, there has been successful implementation of GWAS (e.g., Madsen et al., <a href="#tpg270008-bib-0045" class="usa-link" aria-describedby="tpg270008-bib-0045">2024</a>; Siekmann et al., <a href="#tpg270008-bib-0069" class="usa-link" aria-describedby="tpg270008-bib-0069">2021</a>; Vendelbo et al., <a href="#tpg270008-bib-0073" class="usa-link" aria-describedby="tpg270008-bib-0073">2021</a>) and genomic selection (e.g., Auinger et al., <a href="#tpg270008-bib-0002" class="usa-link" aria-describedby="tpg270008-bib-0002">2016</a>; Bernal‐Vasquez et al., <a href="#tpg270008-bib-0008" class="usa-link" aria-describedby="tpg270008-bib-0008">2017</a>; Galán et al., <a href="#tpg270008-bib-0021" class="usa-link" aria-describedby="tpg270008-bib-0021">2020</a>). Most commercial rye breeding efforts are hybrid breeding (Geiger &amp; Miedaner, <a href="#tpg270008-bib-0023" class="usa-link" aria-describedby="tpg270008-bib-0023">2009</a>; Miedaner &amp; Laidig, <a href="#tpg270008-bib-0048" class="usa-link" aria-describedby="tpg270008-bib-0048">2019</a>), and the landscape for population development lags in research investment, especially in developing approaches for genomics‐enabled population improvement. These differences in population and hybrid investment are evidenced by relatively slower progression in yield increases in population varieties (Laidig et al., <a href="#tpg270008-bib-0038" class="usa-link" aria-describedby="tpg270008-bib-0038">2017</a>). Furthermore, modern genomic techniques, such as genomic selection, have not been applied to population improvement in rye, although genomic selection for population improvement has shown promise for many allogamous crops (Esfandyari et al., <a href="#tpg270008-bib-0019" class="usa-link" aria-describedby="tpg270008-bib-0019">2020</a>; Lin et al., <a href="#tpg270008-bib-0043" class="usa-link" aria-describedby="tpg270008-bib-0043">2016</a>; Schwantes et al., <a href="#tpg270008-bib-0067" class="usa-link" aria-describedby="tpg270008-bib-0067">2020</a>; Yabe et al., <a href="#tpg270008-bib-0077" class="usa-link" aria-describedby="tpg270008-bib-0077">2013</a>). As such, it is desirable to develop genomic approaches relevant for rye population development, especially for geographic regions or end uses in which hybrid rye is not economically viable (e.g., forage, cover crops).</p>
<p>We thus sought to survey population variation and test new approaches for winter rye population breeding, with a specific focus on rye in the southeastern United States. Cereal rye continues to be an important forage and grazing crop in the Southeast (Kim et al., <a href="#tpg270008-bib-0035" class="usa-link" aria-describedby="tpg270008-bib-0035">2016</a>; Newell &amp; Butler, <a href="#tpg270008-bib-0052" class="usa-link" aria-describedby="tpg270008-bib-0052">2013</a>), and production of grain for both specialty markets (e.g., distilling) and cover crop seed is also growing (Brzozowski et al., <a href="#tpg270008-bib-0013" class="usa-link" aria-describedby="tpg270008-bib-0013">2023</a>). Historically, winter rye breeding was supported at several universities and nonprofits in the region, leading to releases of winter rye varieties still grown today (Barnett et al., <a href="#tpg270008-bib-0006" class="usa-link" aria-describedby="tpg270008-bib-0006">2002</a>; Newell &amp; Butler, <a href="#tpg270008-bib-0052" class="usa-link" aria-describedby="tpg270008-bib-0052">2013</a>). Today, there are fewer public programs engaged in winter rye breeding in the Southeast, in part because rye lacks the infrastructure and support of other small grains, like wheat. Nonetheless, due to the rising demand for local rye, there is a need to characterize Southeastern rye germplasm and to assess how to integrate genomic tools for more efficient development of new, regionally adapted populations.</p>
<p>In this study, we evaluated genetic and phenotypic diversity of winter rye populations in the southeastern United States by developing a regional testing panel as well as a panel of populations from the University of Kentucky breeding program. Specifically, we genotyped individual plants and measured a range of developmental, yield, and quality traits. We then partitioned genetic variation of individual traits to variation within and between populations. Importantly, we sought to leverage these data to test approaches for genomics‐enabled rye population breeding. Genomic prediction accuracy of these traits across and within populations was also estimated. Overall, we have characterized germplasm diversity of important rye accessions for the Southeast and developed approaches for integrating new genomic resources in rye population improvement. Results from this work have implications for rye population breeding, emphasize the importance of maintaining regionally adapted populations, and could be extended to other outcrossing species.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270008-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270008-list-0001" class="list" style="list-style-type:disc">
<li><p>Rye population breeding is important for forage, cover crop, and grain markets in the southeastern United States.</p></li>
<li><p>Most phenotypic and genetic variation resides within rye populations, rather than between distinct populations.</p></li>
<li><p>Latitude of population development is predictive of heading date and spike size, but no other traits.</p></li>
<li><p>Genomic prediction accuracy was lower within populations than for individuals across multiple accessions.</p></li>
</ul></section></section></section><section id="tpg270008-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270008-sec-0040"><h3 class="pmc_sec_title">2.1. Germplasm</h3>
<p>We used 15 diploid open‐pollinated winter rye accessions in the Southeast Panel (Table <a href="#tpg270008-tbl-0001" class="usa-link">1</a>): <em>n </em>= 9 and <em>n </em>= 4 forage and grain accessions, respectively, developed in the southern United States, and <em>n </em>= 2 northern grain accessions as comparisons. For some analyses, we excluded the northern grain accessions and refer to this as the “Reduced Southeast Panel.” Primary end use, breeding program, and location were determined by Plant Variety Protection, Plant Registration, or university documentation. We were unable to find a definitive reference for Mississippi Abruzzi, and University of Kentucky breeding lines do not have references. All plants except the breeding lines were sourced from the USDA National Plant Germplasm System National Small Grains Collection. The accessions are briefly described here and in full in Supporting Information <a href="#tpg270008-supinfo-0001" class="usa-link">S1</a>.</p>
<section class="tw xbox font-sm" id="tpg270008-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Description of accessions used in the Southeast Panel. Accession name, abbreviation (abbr.), predominant end use (use), the breeding program that developed the accession, the source of seed (source), documentation of its release (note), and rationale for inclusion (rationale). The Reduced Southeast Panel are the genotypes below but excluding ND Dylan and Rymin.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Variety</th>
<th align="left" rowspan="1" colspan="1">Abbr.</th>
<th align="left" rowspan="1" colspan="1">Use</th>
<th align="left" rowspan="1" colspan="1">Breeding program</th>
<th align="left" rowspan="1" colspan="1">Source</th>
<th align="left" rowspan="1" colspan="1">Note</th>
<th align="left" rowspan="1" colspan="1">Rationale</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Athens Abruzzi</td>
<td align="left" rowspan="1" colspan="1">AAB</td>
<td align="left" rowspan="1" colspan="1">Forage</td>
<td align="left" rowspan="1" colspan="1">University of Georgia (Athens, GA)</td>
<td align="left" rowspan="1" colspan="1">
<p>NPGS,</p>
<p>PI 534936</p>
</td>
<td align="left" rowspan="1" colspan="1">(Brown, <a href="#tpg270008-bib-0011" class="usa-link" aria-describedby="tpg270008-bib-0011">1973</a>)</td>
<td align="left" rowspan="1" colspan="1">Regional Abruzzi selection</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Balbo</td>
<td align="left" rowspan="1" colspan="1">BAL</td>
<td align="left" rowspan="1" colspan="1">Grain</td>
<td align="left" rowspan="1" colspan="1">Tennessee Agricultural Experiment Station (Oak Ridge, TN)</td>
<td align="left" rowspan="1" colspan="1">
<p>NPGS,</p>
<p>PI 534939</p>
</td>
<td align="left" rowspan="1" colspan="1">(Qualset &amp; Hoskinson, <a href="#tpg270008-bib-0057" class="usa-link" aria-describedby="tpg270008-bib-0057">1967</a>)</td>
<td align="left" rowspan="1" colspan="1">Well‐known heirloom grain variety</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Bates</td>
<td align="left" rowspan="1" colspan="1">BAT</td>
<td align="left" rowspan="1" colspan="1">Forage</td>
<td align="left" rowspan="1" colspan="1">Noble Foundation (Ardmore, OK)</td>
<td align="left" rowspan="1" colspan="1">
<p>NPGS,</p>
<p>PI 591057</p>
</td>
<td align="left" rowspan="1" colspan="1">(Baker et al., <a href="#tpg270008-bib-0004" class="usa-link" aria-describedby="tpg270008-bib-0004">1996</a>)</td>
<td align="left" rowspan="1" colspan="1">Noble Foundation rye variety</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Elbon</td>
<td align="left" rowspan="1" colspan="1">EBO</td>
<td align="left" rowspan="1" colspan="1">Forage</td>
<td align="left" rowspan="1" colspan="1">Noble Foundation (Ardmore, OK)</td>
<td align="left" rowspan="1" colspan="1">
<p>NPGS,</p>
<p>PI 534961; Syn Cise 11</p>
</td>
<td align="left" rowspan="1" colspan="1">(Kim et al., <a href="#tpg270008-bib-0035" class="usa-link" aria-describedby="tpg270008-bib-0035">2016</a>)<a href="#tpg270008-tbl1-note-0002" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="left" rowspan="1" colspan="1">First Noble Foundation forage rye variety</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Explorer</td>
<td align="left" rowspan="1" colspan="1">EXP</td>
<td align="left" rowspan="1" colspan="1">Grain</td>
<td align="left" rowspan="1" colspan="1">Mississippi Agricultural Experiment Station (State College, MS)</td>
<td align="left" rowspan="1" colspan="1">
<p>NPGS,</p>
<p>PI 534962</p>
</td>
<td align="left" rowspan="1" colspan="1">(Barnes &amp; Wells, <a href="#tpg270008-bib-0005" class="usa-link" aria-describedby="tpg270008-bib-0005">1960</a>)</td>
<td align="left" rowspan="1" colspan="1">Regional Abruzzi selection</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Florida Black</td>
<td align="left" rowspan="1" colspan="1">FLB</td>
<td align="left" rowspan="1" colspan="1">Forage</td>
<td align="left" rowspan="1" colspan="1">University of Florida (Gainesville, FL)</td>
<td align="left" rowspan="1" colspan="1">
<p>NPGS,</p>
<p>PI 534965</p>
</td>
<td align="left" rowspan="1" colspan="1">(Pfahler et al., <a href="#tpg270008-bib-0056" class="usa-link" aria-describedby="tpg270008-bib-0056">1985</a>)<a href="#tpg270008-tbl1-note-0002" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="left" rowspan="1" colspan="1">Florida forage rye</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KYSC1503C1</td>
<td align="left" rowspan="1" colspan="1">KY1</td>
<td align="left" rowspan="1" colspan="1">Grain</td>
<td align="left" rowspan="1" colspan="1">University of Kentucky (Lexington, KY)</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Grain selection</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KYSC1707C2</td>
<td align="left" rowspan="1" colspan="1">KY2</td>
<td align="left" rowspan="1" colspan="1">Grain</td>
<td align="left" rowspan="1" colspan="1">University of Kentucky (Lexington, KY)</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Grain selection</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KYSC1710C1</td>
<td align="left" rowspan="1" colspan="1">KY3</td>
<td align="left" rowspan="1" colspan="1">Forage</td>
<td align="left" rowspan="1" colspan="1">University of Kentucky (Lexington, KY)</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Forage selection</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KYSC1803C0</td>
<td align="left" rowspan="1" colspan="1">KY4</td>
<td align="left" rowspan="1" colspan="1">Forage</td>
<td align="left" rowspan="1" colspan="1">University of Kentucky (Lexington, KY)</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Forage selection</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Mississippi Abruzzi</td>
<td align="left" rowspan="1" colspan="1">MAB</td>
<td align="left" rowspan="1" colspan="1">Forage</td>
<td align="left" rowspan="1" colspan="1">Mississippi Agricultural Experiment Station (State College, MS)</td>
<td align="left" rowspan="1" colspan="1">
<p>NPGS,</p>
<p>PI 535154</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">Regional Abruzzi selection</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ND Dylan</td>
<td align="left" rowspan="1" colspan="1">NDD</td>
<td align="left" rowspan="1" colspan="1">Grain</td>
<td align="left" rowspan="1" colspan="1">North Dakota Agricultural Experiment Station (Carrington, ND)</td>
<td align="left" rowspan="1" colspan="1"> Commercial</td>
<td align="left" rowspan="1" colspan="1">(NDSU Research Foundation, <a href="#tpg270008-bib-0051" class="usa-link" aria-describedby="tpg270008-bib-0051">2020</a>)</td>
<td align="left" rowspan="1" colspan="1">Major production area grain comparison</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Rymin</td>
<td align="left" rowspan="1" colspan="1">RYM</td>
<td align="left" rowspan="1" colspan="1">Grain</td>
<td align="left" rowspan="1" colspan="1">University of Minnesota Agricultural Experiment Station (St. Paul, MN)</td>
<td align="left" rowspan="1" colspan="1">
<p>NPGS,</p>
<p>Cise 176</p>
</td>
<td align="left" rowspan="1" colspan="1">(Robinson, <a href="#tpg270008-bib-0061" class="usa-link" aria-describedby="tpg270008-bib-0061">1973</a>)</td>
<td align="left" rowspan="1" colspan="1">Major production area grain comparison</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wrens Abruzzi</td>
<td align="left" rowspan="1" colspan="1">WAB</td>
<td align="left" rowspan="1" colspan="1">Forage</td>
<td align="left" rowspan="1" colspan="1">University of Georgia (Tifton, GA)</td>
<td align="left" rowspan="1" colspan="1">
<p>NPGS,</p>
<p>Cise 36</p>
</td>
<td align="left" rowspan="1" colspan="1">(Morey, <a href="#tpg270008-bib-0050" class="usa-link" aria-describedby="tpg270008-bib-0050">1970</a>)</td>
<td align="left" rowspan="1" colspan="1">Widely grown; UGA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wintergrazer‐70</td>
<td align="left" rowspan="1" colspan="1">WGR</td>
<td align="left" rowspan="1" colspan="1">Forage</td>
<td align="left" rowspan="1" colspan="1">
<p>Pennington Grain &amp; Seed, Inc.</p>
<p>(Madison, GA)</p>
</td>
<td align="left" rowspan="1" colspan="1">
<p>NPGS,</p>
<p>CIse 38</p>
</td>
<td align="left" rowspan="1" colspan="1">(Pennington Grain &amp; Seed, Inc., <a href="#tpg270008-bib-0054" class="usa-link" aria-describedby="tpg270008-bib-0054">1978</a>)</td>
<td align="left" rowspan="1" colspan="1">Well‐known heirloom forage variety</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270008-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270008-tbl1-note-0001"><p>
<em>Note</em>: “–” indicates that documentation is not available.</p></div>
<div class="fn" id="tpg270008-tbl1-note-0002">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Variety referenced within, not original release notice of variety.</p>
</div>
</div></section><p>This Southeast Panel includes five Abruzzi‐type ryes that have grain harvest potential. Abruzzi ryes were introduced to the United States from Italy in the early 1900s by the USDA (Morey, <a href="#tpg270008-bib-0050" class="usa-link" aria-describedby="tpg270008-bib-0050">1970</a>), and those on the Southeast Panel include Balbo rye, released by the Tennessee Agricultural Experiment Station in 1933 (Qualset &amp; Hoskinson, <a href="#tpg270008-bib-0057" class="usa-link" aria-describedby="tpg270008-bib-0057">1967</a>); Explorer rye, released by the Mississippi Agricultural Experiment Station in 1960 as a grazing variety with potential additional harvest of a grain cash crop (Barnes &amp; Wells, <a href="#tpg270008-bib-0005" class="usa-link" aria-describedby="tpg270008-bib-0005">1960</a>); Mississippi Abruzzi was developed by Mississippi State University; Wrens Abruzzi rye by the University of Georgia in 1953 (Morey, <a href="#tpg270008-bib-0050" class="usa-link" aria-describedby="tpg270008-bib-0050">1970</a>); Athens Abruzzi rye, selected for forage yield and kernel size also by the University of Georgia (Brown, <a href="#tpg270008-bib-0011" class="usa-link" aria-describedby="tpg270008-bib-0011">1973</a>). The Southeast Panel also includes four ryes selected for forage yield including Florida Black, a variety developed for winter forage (Pfahler et al., <a href="#tpg270008-bib-0056" class="usa-link" aria-describedby="tpg270008-bib-0056">1985</a>); Wintergrazer‐70, a variety with high tiller number and later maturity developed in Georgia (Pennington Grain &amp; Seed, Inc., <a href="#tpg270008-bib-0054" class="usa-link" aria-describedby="tpg270008-bib-0054">1978</a>); Elbon, the first rye variety released by the Samuel Roberts Noble Foundation in 1956 (Kim et al., <a href="#tpg270008-bib-0035" class="usa-link" aria-describedby="tpg270008-bib-0035">2016</a>); and Bates, a forage variety release by the Samuel Roberts Noble Foundation and the Texas Agricultural Experiment Station (Baker et al., <a href="#tpg270008-bib-0004" class="usa-link" aria-describedby="tpg270008-bib-0004">1996</a>). Finally, the Southeast Panel includes two grain accessions developed by breeding programs in the northern plains as comparisons. Rymin was released by the University of Minnesota Agricultural Experiment Station in 1973 (Robinson, <a href="#tpg270008-bib-0061" class="usa-link" aria-describedby="tpg270008-bib-0061">1973</a>). ND Dylan is an open‐pollinated grain rye recently developed by North Dakota State University (NDSU Research Foundation, <a href="#tpg270008-bib-0051" class="usa-link" aria-describedby="tpg270008-bib-0051">2020</a>).</p>
<p>We also grew eight other University of Kentucky breeding lines that were evaluated separately as the Kentucky Panel (Table <a href="#tpg270008-tbl-0002" class="usa-link">2</a>). All populations in this panel were sourced from the University of Kentucky rye breeding program.</p>
<section class="tw xbox font-sm" id="tpg270008-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Description of accessions used in the Kentucky Panel. Accession name, abbreviation (abbr.), predominant end use (use), and rationale for inclusion (rationale).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Variety</th>
<th align="left" rowspan="1" colspan="1">Abbr.</th>
<th align="left" rowspan="1" colspan="1">Use</th>
<th align="left" rowspan="1" colspan="1">Rationale</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">KYSC1804C0</td>
<td align="left" rowspan="1" colspan="1">KY5</td>
<td align="left" rowspan="1" colspan="1">Forage</td>
<td align="left" rowspan="1" colspan="1">15 parents, southern USA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KYSC1812C0</td>
<td align="left" rowspan="1" colspan="1">KY6</td>
<td align="left" rowspan="1" colspan="1">Grain</td>
<td align="left" rowspan="1" colspan="1">27 parents, diverse USA and Europe</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KYSC1812C2</td>
<td align="left" rowspan="1" colspan="1">KY7</td>
<td align="left" rowspan="1" colspan="1">Grain</td>
<td align="left" rowspan="1" colspan="1">2 cycles of selection</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KYSC1815C0</td>
<td align="left" rowspan="1" colspan="1">KY8</td>
<td align="left" rowspan="1" colspan="1">Grain</td>
<td align="left" rowspan="1" colspan="1">40 parents, mostly European</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KYSC1816C0</td>
<td align="left" rowspan="1" colspan="1">KY9</td>
<td align="left" rowspan="1" colspan="1">Grain</td>
<td align="left" rowspan="1" colspan="1">4 parents, 1 each of Pearl, Animo, Aventino, and Weszanowski</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KYSC1902C1</td>
<td align="left" rowspan="1" colspan="1">KY10</td>
<td align="left" rowspan="1" colspan="1">Forage</td>
<td align="left" rowspan="1" colspan="1">15 parents, mostly southern USA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KYSC1904C0</td>
<td align="left" rowspan="1" colspan="1">KY11</td>
<td align="left" rowspan="1" colspan="1">Grain</td>
<td align="left" rowspan="1" colspan="1">15 parents, mostly European</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KYSC1912C0</td>
<td align="left" rowspan="1" colspan="1">KY12</td>
<td align="left" rowspan="1" colspan="1">Grain</td>
<td align="left" rowspan="1" colspan="1">15 parents, 7 Tenn 4063, 8 Petroske</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270008-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The third set of germplasm grown was a small evaluation of Aroostook, a common cover crop in the United States developed in Maine (USDA‐Natural Resources Conservation Service, <a href="#tpg270008-bib-0071" class="usa-link" aria-describedby="tpg270008-bib-0071">2015</a>), from two different sources (Albert Lea Seed, “CCA”; NPGS PI 464583, “CCG”). With these two sources, we tested the degree to which identity was conserved between sources.</p></section><section id="tpg270008-sec-0050"><h3 class="pmc_sec_title">2.2. Plant growth conditions</h3>
<p>The Southeast Panel and the Aroostook were grown in a common garden in a University of Kentucky greenhouse. Seeds were sown into flats into Promix BX potting mix in late January 2023. Note that 2 weeks after sowing, the plants were moved into a 4°C vernalization room with a 12‐h photoperiod for 10–12 weeks. The plants were then transplanted into 3.8 L pots in a greenhouse with a 14‐h photoperiod and 22°C (72°F) day/16°C (60°F) night temperatures. The plants were fertilized with Osmocote Plus 15‐9‐12 3–4 month release shortly after transplanting and watered as needed. The plants were sprayed twice with an insecticidal soap to manage aphid infestations after heading.</p>
<p>For each accession in the Southeast Panel, <em>n </em>= 24 individuals per accession were grown in a randomized complete block design in the greenhouse where 3–4 individuals per accession were in a block (greenhouse bench). The two Aroostook sources were grown with eight individual plants per accession with one per block. The perimeter of each block (bench) was a rye border.</p>
<p>The Kentucky Panel was likewise sown in flats and vernalized. After vernalization, individual plants were hand‐transplanted as space plants in blocks by accession, with 25 m between each accession at the University of Kentucky North Farm in Spring 2023. Accessions were grouped together such that the seed was usable for seed multiplication (rather than intermating of all accessions). Spikes were harvested at maturity in Summer 2023.</p></section><section id="tpg270008-sec-0060"><h3 class="pmc_sec_title">2.3. Plant phenotyping</h3>
<p>We recorded phenotypes for individual plants in each accession. For the Southeast Panel and the Aroostook, we recorded the heading date (Zadoks 55) from May 9, 2023 until June 11, 2023. At harvest, the number of spikes (number of reproductive tillers) was recorded for individual plants. Mature spikes were cut from the plant and scanned using an Epson Perfection V800 Photo Color Scanner at 6400 dpi with a dual lens system. Images were processed in ImageJ to measure spike area (Schneider et al., <a href="#tpg270008-bib-0063" class="usa-link" aria-describedby="tpg270008-bib-0063">2012</a>). We used a custom ImageJ macro to remove pollen grains and reduce awn visibility to measure spike area (<a href="#tpg270008-supinfo-0001" class="usa-link">Method S1</a>). Mean spike area (mean spike size) was calculated as total spike area divided by spike count. All spikes from a plant were then threshed together. Seeds were counted and weighed, and then 1000‐kernel weight (TKW) was estimated.</p>
<p>For the Kentucky Panel, spike count and seed mass were measured, and TKW was estimated. For all samples with seed mass &gt;10 g, seed composition was evaluated by near infrared (NIR) spectroscopy with a Perten DA 7250 NIR Analyzer (PerkinElmer) in the small (10 g) evaluation cup. The composition traits of starch, protein, fiber, and ash were determined by the Honing's regression rye equation available in the software (updated Spring 2024). Each sample was scanned three times, and results were averaged.</p>
<p>In sum, heading date, reproductive tiller number, spike area (total and mean), seed mass, and TKW were phenotyped for 344 individuals in the Southeast Panel, with complete phenotypic data for 336 individuals. Each of the 15 accessions had complete data for 21–24 individuals per accession. Aroostook was phenotyped in the same way as the Southeast Panel with 7–8 individuals with complete phenotypes per accession. There were 192 plants in the Kentucky Panel that were transplanted into the field (eight breeding populations with 24 individuals each), of which 165 survived to maturity. Of these, 117 had &gt;10 g seed, and composition was evaluated by NIR. In sum, 16–23 per population had field phenotypes (mean 20.6), and 10–19 (mean 14.6) per population had both field and composition phenotypes. Analysis was conducted with all available phenotypic data.</p></section><section id="tpg270008-sec-0070"><h3 class="pmc_sec_title">2.4. Phenotypic diversity analysis</h3>
<p>We first assessed if there were differences between accessions in the Southeast Panel using the following model:
</p>
<table class="disp-formula p" id="tpg270008-disp-0001"><tr>
<td class="formula"><math id="jats-math-1" display="block"><mrow><mtable displaystyle="true"><mtr><mtd columnalign="right"><mrow><msub><mi>Y</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub><mo>∼</mo><msub><mi>A</mi><mi>i</mi></msub></mrow></mtd><mtd><mo>+</mo></mtd><mtd columnalign="left"><mrow><mi>Bloc</mi><msub><mi mathvariant="normal">k</mi><mi>j</mi></msub><mo linebreak="badbreak">+</mo><mrow><mi>Block</mi><mo>:</mo><mi>Ro</mi></mrow><msub><mi mathvariant="normal">w</mi><mrow><mi>j</mi><mi>k</mi></mrow></msub></mrow></mtd></mtr><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mo linebreak="badbreak">+</mo><mspace width="0.16em"></mspace><mrow><mi>Block</mi><mo>:</mo><mi>Colum</mi></mrow><msub><mi mathvariant="normal">n</mi><mrow><mi>j</mi><mi>l</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <em>A<sub>i</sub>
</em> is the fixed effect of accession, “Block<em>
<sub>j</sub>
</em>” is the random effect of block (bench), and “Block:Row<em>
<sub>jk</sub>
</em>” and “Block:Column<em>
<sub>jl</sub>
</em>” are the random effects of row and column nested within block, respectively, and <em>e<sub>ijkl</sub>
</em> is the residual error variance. The model was fitted first to identify outliers, which were defined as values with studentized residuals &gt;3, outliers were removed, and the model was fitted again. The model was implemented using the “lmer” function in the lme4 package in R (Bates et al., <a href="#tpg270008-bib-0007" class="usa-link" aria-describedby="tpg270008-bib-0007">2015</a>), and significance testing was done with the lmerTest package (Kuznetsova et al., <a href="#tpg270008-bib-0037" class="usa-link" aria-describedby="tpg270008-bib-0037">2017</a>). As there were not spatial covariates for the Kentucky Panel, we used a linear model with only the fixed effect of accession and implemented the model with the “lm” function in R. Outlier removal was conducted as for the Southeast Panel. For both the Southeast and Kentucky Panels, accession means and confidence intervals were calculated as the estimated marginal means from the described models with the emmeans package (Length, <a href="#tpg270008-bib-0040" class="usa-link" aria-describedby="tpg270008-bib-0040">2023</a>), and significant pairwise differences were estimated using a Bonferroni adjustment with the “cld” function in the multcomp R package (Hothorn et al., <a href="#tpg270008-bib-0030" class="usa-link" aria-describedby="tpg270008-bib-0030">2008</a>). To determine if there were significant phenotypic differences between the two Aroostook sources, which were only grown with one plant per bench (block), we used a Welch two‐sample <em>t</em>‐test implemented with the “<em>t</em>.test” function in R.</p>
<p>We then tested if phenotypic differences between accessions in the Southeast Panel were geographically structured by latitude of variety development using the following model:
</p>
<table class="disp-formula p" id="tpg270008-disp-0002"><tr>
<td class="formula"><math id="jats-math-2" display="block"><mrow><mtable displaystyle="true"><mtr><mtd columnalign="right"><mrow><msub><mi>Y</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub><mo>∼</mo><mi>L</mi></mrow></mtd><mtd><mo>+</mo></mtd><mtd columnalign="left"><mrow><msub><mi>A</mi><mi>i</mi></msub><mo linebreak="badbreak">+</mo><mi>Bloc</mi><msub><mi mathvariant="normal">k</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><mrow><mi>Block</mi><mo>:</mo><mi>Ro</mi></mrow><msub><mi mathvariant="normal">w</mi><mrow><mi>j</mi><mi>k</mi></mrow></msub></mrow></mtd></mtr><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mo linebreak="badbreak">+</mo><mspace width="0.16em"></mspace><mrow><mi>Block</mi><mo>:</mo><mi>Colum</mi></mrow><msub><mi mathvariant="normal">n</mi><mrow><mi>j</mi><mi>l</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>where <em>L</em> is the fixed effect of latitude (continuous variable), and accession was fitted as a random effect; other random effects, outlier removal, and model implementation remain the same as above. We fitted the model for both the full and Reduced Southeast Panel. Results were plotted after outlier removal.</p>
<p>We then calculated the degree to which trait variation was partitioned between versus within accessions and attributed to experimental parameters (block, row, column). We fit Equation (<a href="#tpg270008-disp-0001" class="usa-link">1</a>) using the “fitVCA” function in the VCA package (Schuetzenmeister &amp; Dufey, <a href="#tpg270008-bib-0066" class="usa-link" aria-describedby="tpg270008-bib-0066">2022</a>) using data with outliers removed as described above from the full and Reduced Southeast Panel, and the Kentucky Panel, all separately. We also fit Equation (<a href="#tpg270008-disp-0001" class="usa-link">1</a>) with the addition of end use as a fixed effect to assess variance attributed to end use. We then extracted variance components for all phenotypes.</p>
<p>Last, we calculated best linear unbiased predictors (BLUPs) for individual plants in the Southeast Panel. Because the phenotypes of heading date, seed mass, TKW, and mean spike area were right‐skewed, those phenotypes were square‐root transformed. BLUPs were estimated for the individuals, all unreplicated, using an augmented design, where the two Aroostook accessions were repeated checks within each block:
</p>
<table class="disp-formula p" id="tpg270008-disp-0003"><tr>
<td class="formula"><math id="jats-math-3" display="block"><mrow><mtable displaystyle="true"><mtr><mtd columnalign="right"><mrow><msub><mi>Y</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi><mi>m</mi></mrow></msub><mo>∼</mo><mi>Chec</mi><msub><mi mathvariant="normal">k</mi><mi>i</mi></msub></mrow></mtd><mtd><mo>+</mo></mtd><mtd columnalign="left"><mrow><mrow><mi>New</mi><mo>:</mo><mi mathvariant="normal">I</mi></mrow><msub><mi mathvariant="normal">D</mi><mi>j</mi></msub><mo linebreak="badbreak">+</mo><mi>Bloc</mi><msub><mi mathvariant="normal">k</mi><mi>k</mi></msub><mo linebreak="goodbreak">+</mo><mrow><mi>Block</mi><mo>:</mo><mi>Ro</mi></mrow><msub><mi mathvariant="normal">w</mi><mrow><mi>k</mi><mi>l</mi></mrow></msub></mrow></mtd></mtr><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mo linebreak="badbreak">+</mo><mspace width="0.16em"></mspace><mrow><mi>Block</mi><mo>:</mo><mi>Colum</mi></mrow><msub><mi mathvariant="normal">n</mi><mrow><mi>k</mi><mi>m</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi><mi>m</mi></mrow></msub></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>The identity of the Aroostook check was denoted by “Check,” and whether the individual plant was a check or entry was identified by “new,” where entries also had individual identifiers, “ID.” The mixed model was implemented, and outliers were identified as above. The BLUPs were deregressed (Garrick et al., <a href="#tpg270008-bib-0022" class="usa-link" aria-describedby="tpg270008-bib-0022">2009</a>), and the deregressed BLUPs, drBLUPs, were used in further analysis. It was not possible to calculate BLUPs for the Kentucky Panel, and so phenotypes after outlier removal (described above) were used.</p></section><section id="tpg270008-sec-0080"><h3 class="pmc_sec_title">2.5. Plant genotyping</h3>
<p>We sequenced the panels with the Rye 5K SNP array by TraitGenetics. Prior to vernalization, we sampled leaf tissue from 24 individuals per accession, except for KY12 (KYSC1912C0), in which 20 individuals were sampled. For Aroostook, eight individuals were sampled from each seed source. The tissue was freeze‐dried, and DNA extraction, quality control, and sequencing were conducted by TraitGenetics.</p>
<p>The genotyping resulted in 564 individuals genotyped with 4960 markers. After removing the 328 markers that were fully missing, we inspected markers with missing rates &gt;20%, and found that missing markers were spread across accessions, rather than some accessions missing the single nucleotide polymorphism (SNP), and thus removed the 69 SNPs missing in &gt;20% of individuals. We then removed 11 individuals with fully missing data. After filtering, mean marker missingness per individual was 0.65% (30 SNPs). The SNP coding was converted to (−1, 0, 1) coding using the “atcg1234” function in sommer (Covarrubias‐Pazaran, <a href="#tpg270008-bib-0015" class="usa-link" aria-describedby="tpg270008-bib-0015">2016</a>), and monomorphic SNPs were removed. This resulted in the full dataset including 553 individuals and 4528 markers. From the full data, we generated four data subsets where monomorphic SNPs were likewise removed: (1) The Southeast Panel, 351 individuals and 4524 markers; (2) The Reduced Southeast Panel with northern checks removed, 305 individuals and 4519 markers; (3) The Kentucky Panel, 278 individuals and 4523 markers; and (4) Aroostook, 15 individuals and 4264 markers.</p></section><section id="tpg270008-sec-0090"><h3 class="pmc_sec_title">2.6. Plant genetic diversity analysis</h3>
<p>We first visualized the degree of genetic differentiation among accessions by genomic principal components using a scaled and centered marker matrix with the “prcomp” function in R. We then quantified population subdivision of rye using Wright's F<sub>ST</sub> for the Southeast and Kentucky Panels. The data were prepared by generating a “genind” object using the “df2genind” function from the “adegenet” package (Jombart, <a href="#tpg270008-bib-0031" class="usa-link" aria-describedby="tpg270008-bib-0031">2008</a>). Then pairwise F<sub>ST</sub> was calculated between accessions using the “genet.dist” function with the “WC84” method in the “hierfstat” R package (Goudet, <a href="#tpg270008-bib-0024" class="usa-link" aria-describedby="tpg270008-bib-0024">2005</a>). Finally, we assessed concordance as the percentage of shared alleles between individuals within and between accessions; for each locus, individuals that were identical shared two alleles, and one allele was shared if one individual was heterozygous and one individual was homozygous.</p>
<p>We then sought to characterize factors that contributed to varietal differentiation in the Southeast and Kentucky Panels. We started by testing if particular alleles were private to one or a few accessions (minor allele frequency [MAF] = 0, in all but one or a few accessions). We then examined the inverse: if an allele was missing from a variety (MAF = 0), but at moderate to high frequency in most other accessions. We also calculated observed heterozygosity (<em>H<sub>O</sub>
</em>; proportion of heterozygous calls) of individuals within accessions. We calculated estimated marginal means for individual accessions and evaluated significant contrasts between them using the “emmeans” and “contrast” functions. We then tested if observed heterozygosity (<em>H<sub>O</sub>
</em>) differed by predominant end use (forage or grain) for the Southeast and Kentucky Panels separately, using a linear mixed model, where end use was a fixed effect, and accession nested within end use was a random effect.</p></section><section id="tpg270008-sec-0100"><h3 class="pmc_sec_title">2.7. Genomic analysis and prediction</h3>
<p>We conducted a genome‐wide association study (GWAS) to elucidate genomic regions associated with the traits as well as genomic prediction to assess the prospect of genomics‐enabled population improvement. For these applications, we used only individuals that had both complete phenotypes and genotypes, which were 336 and 291 individuals in the full and Reduced Southeast Panel, respectively. From the Kentucky Panel, this was 159 and 116 genotyped individuals with complete field and composition phenotypes, respectively.</p>
<p>We conducted GWAS for all traits in the Southeast Panel. Positions of SNPs in the rye genome (Rabanus‐Wallace et al., <a href="#tpg270008-bib-0059" class="usa-link" aria-describedby="tpg270008-bib-0059">2021</a>) were provided by TraitGenetics (Lauren Brzozowski, personal communication, September 2024). Of the 4524 SNPs that passed filtering for the Southeast Panel, 3803 had genomic positions, and only those SNPs were used. The GWAS was implemented with a genomic relationship matrix and no other covariates in the R package “statgenGWAS” (van Rossum &amp; Kruijer, <a href="#tpg270008-bib-0072" class="usa-link" aria-describedby="tpg270008-bib-0072">2022</a>) using the “runSingleTraitGwas” function.</p>
<p>We then assessed the degree to which genomic prediction could be beneficial for population improvement. We used all available polymorphic SNPs, not just those aligned to the genome, for genomic prediction (4524 for Southeast Panel; 4496 and 4493 for the Kentucky Panel with field and composition phenotypes, respectively). Using the drBLUPs, we tested two models: genomic BLUP (GBLUP) implemented in rrBLUP (Endelman, <a href="#tpg270008-bib-0018" class="usa-link" aria-describedby="tpg270008-bib-0018">2011</a>) and Bayes B implemented with 20,000 iterations and a burn‐in of 5000 iterations in BGLR (Pérez &amp; de los Campos, <a href="#tpg270008-bib-0055" class="usa-link" aria-describedby="tpg270008-bib-0055">2014</a>). First, we conducted fivefold cross‐validation by panels (Southeast, Reduced Southeast, and Kentucky with field and composition phenotypes separately) with 50 replications, and prediction accuracy was reported as the Pearson correlation between the predicted and observed value. Then, we assessed prediction accuracy within populations again by panel. For this analysis, we calculated marker effects using the training set of all populations in the panel except the focal population, and then predicted within that population. As an example, we assessed within‐population prediction accuracy in Explorer with the Southeast Panel using all accessions in the panel except Explorer in the training set. In both cross‐validation and within‐population prediction, we used paired <em>t</em>‐tests to test if there were significant differences between models.</p></section><section id="tpg270008-sec-0110"><h3 class="pmc_sec_title">2.8. Statistical analysis</h3>
<p>All statistical analyses are conducted in R (R Core Team, <a href="#tpg270008-bib-0058" class="usa-link" aria-describedby="tpg270008-bib-0058">2016</a>), with data visualization done in the ggplot2 (Wickham, <a href="#tpg270008-bib-0075" class="usa-link" aria-describedby="tpg270008-bib-0075">2016</a>) and ggpubr packages (Kassambara, <a href="#tpg270008-bib-0033" class="usa-link" aria-describedby="tpg270008-bib-0033">2020</a>).</p></section></section><section id="tpg270008-sec-0120"><h2 class="pmc_sec_title">3. RESULTS</h2>
<p>We assessed inter‐ and intra‐accession phenotypic and genetic diversity in accessions (Southeast Panel, Table <a href="#tpg270008-tbl-0001" class="usa-link">1</a>; Figure <a href="#tpg270008-fig-0001" class="usa-link">1</a>; Kentucky Panel, Table <a href="#tpg270008-tbl-0002" class="usa-link">2</a>), and between two seed sources of Aroostook.</p>
<figure class="fig xbox font-sm" id="tpg270008-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/41d4/11997422/f6d2c541fa1e/TPG2-18-e70008-g005.jpg" loading="lazy" id="jats-graphic-1" height="561" width="680" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270008-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Map of development locations for the accessions in the Southeast Panel. The Reduced Southeast Panel excludes the accessions from the two northern states. Accession name abbreviation is given in Table <a href="#tpg270008-tbl-0001" class="usa-link">1</a>.</p></figcaption></figure><section id="tpg270008-sec-0130"><h3 class="pmc_sec_title">3.1. Phenotypic variation</h3>
<p>There were significant pairwise differences between accessions in the Southeast Panel for heading date and, to a lesser extent, spike morphology traits, but rarely for seed mass or TKW (Table <a href="#tpg270008-tbl-0003" class="usa-link">3</a>; Figure <a href="#tpg270008-supinfo-0001" class="usa-link">S1</a>). For instance, there was a 22‐ and 15‐day spread in heading date in the full and Reduced Southeast Panel, respectively, and number of spikes ranged from an average of 10 to 17. Spike and seed traits were also not significantly different between varieties in the Kentucky Panel. However, there were significant differences in composition (e.g., percent protein ranged from 16.75% to 18.92% and percent fiber ranged from 2.97 to 3.64) in the Kentucky Panel (Figure <a href="#tpg270008-supinfo-0001" class="usa-link">S2</a>; Table <a href="#tpg270008-tbl-0004" class="usa-link">4</a>). Finally, Aroostook from two different seed sources had significant differences in heading date (7.5 days), and the commercial variety had 55% higher spike area and 50% higher TKW than the germplasm accession (Figure <a href="#tpg270008-supinfo-0001" class="usa-link">S3</a>).</p>
<section class="tw xbox font-sm" id="tpg270008-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Mean (<em>μ</em>) and standard error (SE) of phenotypes measured in the Southeast Panel, where accession abbreviation is given in Table <a href="#tpg270008-tbl-0001" class="usa-link">1</a>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<thead>
<tr>
<th rowspan="2" align="left" colspan="1">Name</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Heading date</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Seed mass (g)</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">1000‐Kernel weight (g)</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Spike count</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Total spike area (cm<sup>2</sup>)</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Mean spike area (cm<sup>2</sup>)</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">
<em>μ</em>
</th>
<th align="left" rowspan="1" colspan="1">SE</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">
<em>μ</em>
</th>
<th align="left" rowspan="1" colspan="1">SE</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">
<em>μ</em>
</th>
<th align="left" rowspan="1" colspan="1">SE</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">
<em>μ</em>
</th>
<th align="left" rowspan="1" colspan="1">SE</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">
<em>μ</em>
</th>
<th align="left" rowspan="1" colspan="1">SE</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">
<em>μ</em>
</th>
<th align="left" rowspan="1" colspan="1">SE</th>
<th align="left" rowspan="1" colspan="1"></th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">AAB</td>
<td align="center" rowspan="1" colspan="1">140.1</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">BCD</td>
<td align="center" rowspan="1" colspan="1">4.1</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
<td align="center" rowspan="1" colspan="1">23.2</td>
<td align="center" rowspan="1" colspan="1">1.1</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">17.2</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">E</td>
<td align="center" rowspan="1" colspan="1">108.1</td>
<td align="center" rowspan="1" colspan="1">7.7</td>
<td align="left" rowspan="1" colspan="1">BCDE</td>
<td align="center" rowspan="1" colspan="1">6.3</td>
<td align="center" rowspan="1" colspan="1">0.3</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">BAL</td>
<td align="center" rowspan="1" colspan="1">149.8</td>
<td align="center" rowspan="1" colspan="1">0.9</td>
<td align="left" rowspan="1" colspan="1">H</td>
<td align="center" rowspan="1" colspan="1">2.5</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">26.1</td>
<td align="center" rowspan="1" colspan="1">1.1</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">14.7</td>
<td align="center" rowspan="1" colspan="1">0.9</td>
<td align="left" rowspan="1" colspan="1">ABCDE</td>
<td align="center" rowspan="1" colspan="1">114.2</td>
<td align="center" rowspan="1" colspan="1">7.4</td>
<td align="left" rowspan="1" colspan="1">CDE</td>
<td align="center" rowspan="1" colspan="1">7.9</td>
<td align="center" rowspan="1" colspan="1">0.2</td>
<td align="left" rowspan="1" colspan="1">DEFG</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">BAT</td>
<td align="center" rowspan="1" colspan="1">139.1</td>
<td align="center" rowspan="1" colspan="1">0.9</td>
<td align="left" rowspan="1" colspan="1">BC</td>
<td align="center" rowspan="1" colspan="1">5.8</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">C</td>
<td align="center" rowspan="1" colspan="1">24.7</td>
<td align="center" rowspan="1" colspan="1">1.1</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">16.8</td>
<td align="center" rowspan="1" colspan="1">0.9</td>
<td align="left" rowspan="1" colspan="1">DE</td>
<td align="center" rowspan="1" colspan="1">120.9</td>
<td align="center" rowspan="1" colspan="1">7.4</td>
<td align="left" rowspan="1" colspan="1">DE</td>
<td align="center" rowspan="1" colspan="1">7.2</td>
<td align="center" rowspan="1" colspan="1">0.2</td>
<td align="left" rowspan="1" colspan="1">CDEF</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EBO</td>
<td align="center" rowspan="1" colspan="1">142.9</td>
<td align="center" rowspan="1" colspan="1">0.9</td>
<td align="left" rowspan="1" colspan="1">CDEF</td>
<td align="center" rowspan="1" colspan="1">5.6</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">BC</td>
<td align="center" rowspan="1" colspan="1">25.9</td>
<td align="center" rowspan="1" colspan="1">1.1</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">16.6</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">CDE</td>
<td align="center" rowspan="1" colspan="1">121.9</td>
<td align="center" rowspan="1" colspan="1">7.5</td>
<td align="left" rowspan="1" colspan="1">DE</td>
<td align="center" rowspan="1" colspan="1">7.4</td>
<td align="center" rowspan="1" colspan="1">0.2</td>
<td align="left" rowspan="1" colspan="1">CDEFG</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EXP</td>
<td align="center" rowspan="1" colspan="1">138.3</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">AB</td>
<td align="center" rowspan="1" colspan="1">3.0</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">24.8</td>
<td align="center" rowspan="1" colspan="1">1.1</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">12.1</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">AB</td>
<td align="center" rowspan="1" colspan="1">86.8</td>
<td align="center" rowspan="1" colspan="1">7.8</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
<td align="center" rowspan="1" colspan="1">7.2</td>
<td align="center" rowspan="1" colspan="1">0.3</td>
<td align="left" rowspan="1" colspan="1">BCDEFG</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FLB</td>
<td align="center" rowspan="1" colspan="1">134.0</td>
<td align="center" rowspan="1" colspan="1">0.9</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">2.4</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">21.3</td>
<td align="center" rowspan="1" colspan="1">1.1</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">12.7</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">ABCD</td>
<td align="center" rowspan="1" colspan="1">73.5</td>
<td align="center" rowspan="1" colspan="1">7.4</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">5.4</td>
<td align="center" rowspan="1" colspan="1">0.2</td>
<td align="left" rowspan="1" colspan="1">A</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KY1</td>
<td align="center" rowspan="1" colspan="1">139.7</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">BCD</td>
<td align="center" rowspan="1" colspan="1">5.2</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">BC</td>
<td align="center" rowspan="1" colspan="1">23.8</td>
<td align="center" rowspan="1" colspan="1">1.1</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">16.3</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">BCDE</td>
<td align="center" rowspan="1" colspan="1">119.0</td>
<td align="center" rowspan="1" colspan="1">7.8</td>
<td align="left" rowspan="1" colspan="1">CDE</td>
<td align="center" rowspan="1" colspan="1">7.2</td>
<td align="center" rowspan="1" colspan="1">0.3</td>
<td align="left" rowspan="1" colspan="1">BCDEFG</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KY2</td>
<td align="center" rowspan="1" colspan="1">144.1</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">DEFG</td>
<td align="center" rowspan="1" colspan="1">3.0</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">23.5</td>
<td align="center" rowspan="1" colspan="1">1.1</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">12.1</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">AB</td>
<td align="center" rowspan="1" colspan="1">95.3</td>
<td align="center" rowspan="1" colspan="1">7.6</td>
<td align="left" rowspan="1" colspan="1">ABCDE</td>
<td align="center" rowspan="1" colspan="1">7.9</td>
<td align="center" rowspan="1" colspan="1">0.3</td>
<td align="left" rowspan="1" colspan="1">DEFG</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KY3</td>
<td align="center" rowspan="1" colspan="1">148.1</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">GH</td>
<td align="center" rowspan="1" colspan="1">3.6</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">AB</td>
<td align="center" rowspan="1" colspan="1">25.8</td>
<td align="center" rowspan="1" colspan="1">1.2</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">10.5</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">87.7</td>
<td align="center" rowspan="1" colspan="1">7.8</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
<td align="center" rowspan="1" colspan="1">8.3</td>
<td align="center" rowspan="1" colspan="1">0.3</td>
<td align="left" rowspan="1" colspan="1">FG</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KY4</td>
<td align="center" rowspan="1" colspan="1">146.5</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">FGH</td>
<td align="center" rowspan="1" colspan="1">4.5</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
<td align="center" rowspan="1" colspan="1">25.8</td>
<td align="center" rowspan="1" colspan="1">1.1</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">16.2</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">BCDE</td>
<td align="center" rowspan="1" colspan="1">127.2</td>
<td align="center" rowspan="1" colspan="1">7.6</td>
<td align="left" rowspan="1" colspan="1">E</td>
<td align="center" rowspan="1" colspan="1">8.1</td>
<td align="center" rowspan="1" colspan="1">0.2</td>
<td align="left" rowspan="1" colspan="1">EFG</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">MAB</td>
<td align="center" rowspan="1" colspan="1">138.8</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">BC</td>
<td align="center" rowspan="1" colspan="1">2.6</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">21.6</td>
<td align="center" rowspan="1" colspan="1">1.2</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">12.2</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
<td align="center" rowspan="1" colspan="1">79.8</td>
<td align="center" rowspan="1" colspan="1">7.9</td>
<td align="left" rowspan="1" colspan="1">AB</td>
<td align="center" rowspan="1" colspan="1">6.2</td>
<td align="center" rowspan="1" colspan="1">0.3</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NDD</td>
<td align="center" rowspan="1" colspan="1">156.5</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">I</td>
<td align="center" rowspan="1" colspan="1">2.7</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">23.2</td>
<td align="center" rowspan="1" colspan="1">1.2</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">10.7</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">92.7</td>
<td align="center" rowspan="1" colspan="1">7.7</td>
<td align="left" rowspan="1" colspan="1">ABCD</td>
<td align="center" rowspan="1" colspan="1">8.4</td>
<td align="center" rowspan="1" colspan="1">0.3</td>
<td align="left" rowspan="1" colspan="1">G</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RYM</td>
<td align="center" rowspan="1" colspan="1">145.3</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">EFGH</td>
<td align="center" rowspan="1" colspan="1">2.4</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">22.5</td>
<td align="center" rowspan="1" colspan="1">1.1</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">15.0</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">ABCDE</td>
<td align="center" rowspan="1" colspan="1">106.3</td>
<td align="center" rowspan="1" colspan="1">7.5</td>
<td align="left" rowspan="1" colspan="1">BCDE</td>
<td align="center" rowspan="1" colspan="1">7.0</td>
<td align="center" rowspan="1" colspan="1">0.2</td>
<td align="left" rowspan="1" colspan="1">BCDE</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">WAB</td>
<td align="center" rowspan="1" colspan="1">137.0</td>
<td align="center" rowspan="1" colspan="1">0.9</td>
<td align="left" rowspan="1" colspan="1">AB</td>
<td align="center" rowspan="1" colspan="1">3.9</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
<td align="center" rowspan="1" colspan="1">24.4</td>
<td align="center" rowspan="1" colspan="1">1.2</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">17.4</td>
<td align="center" rowspan="1" colspan="1">0.9</td>
<td align="left" rowspan="1" colspan="1">E</td>
<td align="center" rowspan="1" colspan="1">105.9</td>
<td align="center" rowspan="1" colspan="1">7.4</td>
<td align="left" rowspan="1" colspan="1">BCDE</td>
<td align="center" rowspan="1" colspan="1">6.1</td>
<td align="center" rowspan="1" colspan="1">0.2</td>
<td align="left" rowspan="1" colspan="1">AB</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">WGR</td>
<td align="center" rowspan="1" colspan="1">141.5</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">BCDE</td>
<td align="center" rowspan="1" colspan="1">4.4</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
<td align="center" rowspan="1" colspan="1">24.7</td>
<td align="center" rowspan="1" colspan="1">1.2</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">15.3</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="left" rowspan="1" colspan="1">BCDE</td>
<td align="center" rowspan="1" colspan="1">102.4</td>
<td align="center" rowspan="1" colspan="1">7.9</td>
<td align="left" rowspan="1" colspan="1">ABCDE</td>
<td align="center" rowspan="1" colspan="1">6.9</td>
<td align="center" rowspan="1" colspan="1">0.3</td>
<td align="left" rowspan="1" colspan="1">BCD</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270008-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270008-tbl3-note-0001"><p>
<em>Note</em>: The estimated marginal mean and standard error are presented and the compact letter display indicates significance pairwise differences within phenotypes with <em>p</em>
<sub>BONF</sub> &lt; 0.05.</p></div></div></section><section class="tw xbox font-sm" id="tpg270008-tbl-0004"><h4 class="obj_head">TABLE 4.</h4>
<div class="caption p"><p>Mean (<em>μ</em>) and standard error (SE) of phenotypes measured in the Kentucky Panel, where accession abbreviation is given in Table <a href="#tpg270008-tbl-0002" class="usa-link">2</a>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<thead>
<tr>
<th rowspan="2" align="left" colspan="1">Name</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Seed mass (g)</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">1000‐Kernel weight (g)</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Spike count</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Starch (% DW)</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Protein (% DW)</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Fiber (% DW)</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Ash (% DW)</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">
<em>μ</em>
</th>
<th align="left" rowspan="1" colspan="1">SE</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">
<em>μ</em>
</th>
<th align="left" rowspan="1" colspan="1">SE</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">
<em>μ</em>
</th>
<th align="left" rowspan="1" colspan="1">SE</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">
<em>μ</em>
</th>
<th align="left" rowspan="1" colspan="1">SE</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">
<em>μ</em>
</th>
<th align="left" rowspan="1" colspan="1">SE</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">
<em>μ</em>
</th>
<th align="left" rowspan="1" colspan="1">SE</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">
<em>μ</em>
</th>
<th align="left" rowspan="1" colspan="1">SE</th>
<th align="left" rowspan="1" colspan="1"></th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">KY5</td>
<td align="center" rowspan="1" colspan="1">20.07</td>
<td align="center" rowspan="1" colspan="1">2.24</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">28.04</td>
<td align="center" rowspan="1" colspan="1">1.16</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">20.60</td>
<td align="center" rowspan="1" colspan="1">1.68</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">68.73</td>
<td align="center" rowspan="1" colspan="1">0.45</td>
<td align="left" rowspan="1" colspan="1">BC</td>
<td align="center" rowspan="1" colspan="1">18.92</td>
<td align="center" rowspan="1" colspan="1">0.45</td>
<td align="left" rowspan="1" colspan="1">B</td>
<td align="center" rowspan="1" colspan="1">3.13</td>
<td align="center" rowspan="1" colspan="1">0.09</td>
<td align="left" rowspan="1" colspan="1">AB</td>
<td align="center" rowspan="1" colspan="1">1.59</td>
<td align="center" rowspan="1" colspan="1">0.04</td>
<td align="left" rowspan="1" colspan="1">C</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KY6</td>
<td align="center" rowspan="1" colspan="1">16.17</td>
<td align="center" rowspan="1" colspan="1">2.09</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">28.81</td>
<td align="center" rowspan="1" colspan="1">1.11</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">19.30</td>
<td align="center" rowspan="1" colspan="1">1.57</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">70.29</td>
<td align="center" rowspan="1" colspan="1">0.44</td>
<td align="left" rowspan="1" colspan="1">C</td>
<td align="center" rowspan="1" colspan="1">18.76</td>
<td align="center" rowspan="1" colspan="1">0.44</td>
<td align="left" rowspan="1" colspan="1">B</td>
<td align="center" rowspan="1" colspan="1">2.97</td>
<td align="center" rowspan="1" colspan="1">0.09</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">1.46</td>
<td align="center" rowspan="1" colspan="1">0.03</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KY7</td>
<td align="center" rowspan="1" colspan="1">19.19</td>
<td align="center" rowspan="1" colspan="1">2.19</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">30.50</td>
<td align="center" rowspan="1" colspan="1">1.08</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">17.48</td>
<td align="center" rowspan="1" colspan="1">1.64</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">69.75</td>
<td align="center" rowspan="1" colspan="1">0.41</td>
<td align="left" rowspan="1" colspan="1">C</td>
<td align="center" rowspan="1" colspan="1">18.55</td>
<td align="center" rowspan="1" colspan="1">0.41</td>
<td align="left" rowspan="1" colspan="1">AB</td>
<td align="center" rowspan="1" colspan="1">3.05</td>
<td align="center" rowspan="1" colspan="1">0.09</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">1.34</td>
<td align="center" rowspan="1" colspan="1">0.03</td>
<td align="left" rowspan="1" colspan="1">A</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KY8</td>
<td align="center" rowspan="1" colspan="1">20.27</td>
<td align="center" rowspan="1" colspan="1">2.14</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">29.77</td>
<td align="center" rowspan="1" colspan="1">1.08</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">16.14</td>
<td align="center" rowspan="1" colspan="1">1.60</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">68.43</td>
<td align="center" rowspan="1" colspan="1">0.40</td>
<td align="left" rowspan="1" colspan="1">BC</td>
<td align="center" rowspan="1" colspan="1">17.90</td>
<td align="center" rowspan="1" colspan="1">0.40</td>
<td align="left" rowspan="1" colspan="1">AB</td>
<td align="center" rowspan="1" colspan="1">3.25</td>
<td align="center" rowspan="1" colspan="1">0.08</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
<td align="center" rowspan="1" colspan="1">1.56</td>
<td align="center" rowspan="1" colspan="1">0.03</td>
<td align="left" rowspan="1" colspan="1">C</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KY9</td>
<td align="center" rowspan="1" colspan="1">14.70</td>
<td align="center" rowspan="1" colspan="1">2.09</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">26.07</td>
<td align="center" rowspan="1" colspan="1">1.06</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">13.39</td>
<td align="center" rowspan="1" colspan="1">1.57</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">66.44</td>
<td align="center" rowspan="1" colspan="1">0.45</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">16.75</td>
<td align="center" rowspan="1" colspan="1">0.45</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">3.55</td>
<td align="center" rowspan="1" colspan="1">0.09</td>
<td align="left" rowspan="1" colspan="1">BC</td>
<td align="center" rowspan="1" colspan="1">1.37</td>
<td align="center" rowspan="1" colspan="1">0.03</td>
<td align="left" rowspan="1" colspan="1">AB</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KY10</td>
<td align="center" rowspan="1" colspan="1">14.88</td>
<td align="center" rowspan="1" colspan="1">2.51</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">27.65</td>
<td align="center" rowspan="1" colspan="1">1.27</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">15.38</td>
<td align="center" rowspan="1" colspan="1">1.88</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">68.91</td>
<td align="center" rowspan="1" colspan="1">0.56</td>
<td align="left" rowspan="1" colspan="1">BC</td>
<td align="center" rowspan="1" colspan="1">18.83</td>
<td align="center" rowspan="1" colspan="1">0.55</td>
<td align="left" rowspan="1" colspan="1">AB</td>
<td align="center" rowspan="1" colspan="1">3.43</td>
<td align="center" rowspan="1" colspan="1">0.12</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
<td align="center" rowspan="1" colspan="1">1.46</td>
<td align="center" rowspan="1" colspan="1">0.04</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KY11</td>
<td align="center" rowspan="1" colspan="1">14.46</td>
<td align="center" rowspan="1" colspan="1">2.36</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">26.79</td>
<td align="center" rowspan="1" colspan="1">1.20</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">17.11</td>
<td align="center" rowspan="1" colspan="1">1.77</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">67.17</td>
<td align="center" rowspan="1" colspan="1">0.56</td>
<td align="left" rowspan="1" colspan="1">AB</td>
<td align="center" rowspan="1" colspan="1">17.42</td>
<td align="center" rowspan="1" colspan="1">0.55</td>
<td align="left" rowspan="1" colspan="1">AB</td>
<td align="center" rowspan="1" colspan="1">3.64</td>
<td align="center" rowspan="1" colspan="1">0.12</td>
<td align="left" rowspan="1" colspan="1">C</td>
<td align="center" rowspan="1" colspan="1">1.62</td>
<td align="center" rowspan="1" colspan="1">0.04</td>
<td align="left" rowspan="1" colspan="1">C</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">KY12</td>
<td align="center" rowspan="1" colspan="1">12.57</td>
<td align="center" rowspan="1" colspan="1">2.19</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">26.34</td>
<td align="center" rowspan="1" colspan="1">1.11</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">13.90</td>
<td align="center" rowspan="1" colspan="1">1.64</td>
<td align="left" rowspan="1" colspan="1">A</td>
<td align="center" rowspan="1" colspan="1">68.67</td>
<td align="center" rowspan="1" colspan="1">0.47</td>
<td align="left" rowspan="1" colspan="1">BC</td>
<td align="center" rowspan="1" colspan="1">18.36</td>
<td align="center" rowspan="1" colspan="1">0.47</td>
<td align="left" rowspan="1" colspan="1">AB</td>
<td align="center" rowspan="1" colspan="1">3.34</td>
<td align="center" rowspan="1" colspan="1">0.10</td>
<td align="left" rowspan="1" colspan="1">ABC</td>
<td align="center" rowspan="1" colspan="1">1.50</td>
<td align="center" rowspan="1" colspan="1">0.04</td>
<td align="left" rowspan="1" colspan="1">BC</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270008-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270008-tbl4-note-0001"><p>
<em>Note</em>: The estimated marginal mean and standard error are presented and the compact letter display indicates significance pairwise differences within phenotypes with <em>p</em>
<sub>BONF</sub> &lt; 0.05. DW, dry weight.</p></div></div></section><p>Some of these traits exhibited geographic structuring by the origin latitude of the accession in the Southeast Panel (Figure <a href="#tpg270008-fig-0002" class="usa-link">2</a>; Tables <a href="#tpg270008-supinfo-0001" class="usa-link">S1</a> and <a href="#tpg270008-supinfo-0001" class="usa-link">S2</a>). Both heading date and mean spike area increased with latitude (heading date: <em>F</em>
<sub>1,13 </sub>= 25.65, <em>p </em>&lt; 0.001; mean spike area: <em>F</em>
<sub>1,13 </sub>= 10.32, <em>p </em>= 0.007), and this relationship held even in the reduced panel with the two northern accessions removed (heading date: <em>F</em>
<sub>1,11 </sub>= 14.67, <em>p </em>= 0.003; mean spike area: <em>F</em>
<sub>1,11 </sub>= 38.37, <em>p </em>&lt; 0.001). In addition, in the reduced panel only, there was a marginal positive relationship between latitude and TKW (<em>F</em>
<sub>1,11 </sub>= 4.61, <em>p </em>= 0.054) and total spike area (<em>F</em>
<sub>1,11 </sub>= 3.2, <em>p </em>= 0.102), largely driven by low values from Florida Black. There was no relationship between latitude and spike count or seed mass in either the full or reduced panel.</p>
<figure class="fig xbox font-sm" id="tpg270008-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/41d4/11997422/5904152f5166/TPG2-18-e70008-g004.jpg" loading="lazy" id="jats-graphic-3" height="866" width="800" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270008-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Relationship between traits and the latitude at which the accessions were developed for the Southeast Panel (left panel) and Reduced Southeast Panel (right panel). Each point represents an individual within an accession, with outlier points removed. Accession name abbreviation is given in Table <a href="#tpg270008-tbl-0001" class="usa-link">1</a>, and full statistical analysis is given in Tables <a href="#tpg270008-supinfo-0001" class="usa-link">S1</a> and <a href="#tpg270008-supinfo-0001" class="usa-link">S2</a>. TKW, 1000‐kernel weight.</p></figcaption></figure><p>Finally, there was a higher degree of within‐ than between‐accession variation for all phenotypes in all panels except for the heading date in the Southeast Panel (Table <a href="#tpg270008-tbl-0005" class="usa-link">5</a>). Percent variation attributed to end use classification was also less than attributed to either between or within accessions in any panel (Table <a href="#tpg270008-supinfo-0001" class="usa-link">S3</a>). In the Reduced Southeast Panel, with the two northern grain accessions excluded, only mean spike area had a nonzero variance component attributed to end use (5.59%, Table <a href="#tpg270008-supinfo-0001" class="usa-link">S3</a>).</p>
<section class="tw xbox font-sm" id="tpg270008-tbl-0005"><h4 class="obj_head">TABLE 5.</h4>
<div class="caption p"><p>Variance components for phenotypes observed in the Southeast Panel, Reduced Southeast Panel, and the Kentucky Panel from the model defined by Equation (<a href="#tpg270008-disp-0001" class="usa-link">1</a>). Variance components are described as percent (%) of total variance by trait, and the "experimental" variance component refers to variance ascribed to block, row, and column.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Panel</th>
<th align="left" rowspan="1" colspan="1">Phenotype</th>
<th align="left" rowspan="1" colspan="1">Between accessions (%)</th>
<th align="left" rowspan="1" colspan="1">Within accessions (%)</th>
<th align="left" rowspan="1" colspan="1">Experimental (%)</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="6" align="left" colspan="1">Southeast</td>
<td align="left" rowspan="1" colspan="1">Heading date</td>
<td align="left" rowspan="1" colspan="1">62.32</td>
<td align="left" rowspan="1" colspan="1">34.61</td>
<td align="left" rowspan="1" colspan="1">3.07</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Seed mass</td>
<td align="left" rowspan="1" colspan="1">19.50</td>
<td align="left" rowspan="1" colspan="1">62.20</td>
<td align="left" rowspan="1" colspan="1">18.30</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TKW</td>
<td align="left" rowspan="1" colspan="1">5.45</td>
<td align="left" rowspan="1" colspan="1">76.84</td>
<td align="left" rowspan="1" colspan="1">17.71</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Spike count</td>
<td align="left" rowspan="1" colspan="1">20.27</td>
<td align="left" rowspan="1" colspan="1">64.66</td>
<td align="left" rowspan="1" colspan="1">15.07</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total spike area</td>
<td align="left" rowspan="1" colspan="1">17.28</td>
<td align="left" rowspan="1" colspan="1">63.68</td>
<td align="left" rowspan="1" colspan="1">19.04</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Mean spike area</td>
<td align="left" rowspan="1" colspan="1">36.57</td>
<td align="left" rowspan="1" colspan="1">56.07</td>
<td align="left" rowspan="1" colspan="1">7.36</td>
</tr>
<tr>
<td rowspan="6" align="left" colspan="1">Reduced Southeast</td>
<td align="left" rowspan="1" colspan="1">Heading date</td>
<td align="left" rowspan="1" colspan="1">52.51</td>
<td align="left" rowspan="1" colspan="1">40.28</td>
<td align="left" rowspan="1" colspan="1">7.21</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Seed mass</td>
<td align="left" rowspan="1" colspan="1">18.85</td>
<td align="left" rowspan="1" colspan="1">62.61</td>
<td align="left" rowspan="1" colspan="1">18.54</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TKW</td>
<td align="left" rowspan="1" colspan="1">6.89</td>
<td align="left" rowspan="1" colspan="1">76.15</td>
<td align="left" rowspan="1" colspan="1">16.96</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Spike count</td>
<td align="left" rowspan="1" colspan="1">18.94</td>
<td align="left" rowspan="1" colspan="1">67.96</td>
<td align="left" rowspan="1" colspan="1">13.10</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total spike area</td>
<td align="left" rowspan="1" colspan="1">20.19</td>
<td align="left" rowspan="1" colspan="1">63.93</td>
<td align="left" rowspan="1" colspan="1">15.88</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Mean spike area</td>
<td align="left" rowspan="1" colspan="1">40.20</td>
<td align="left" rowspan="1" colspan="1">47.02</td>
<td align="left" rowspan="1" colspan="1">12.78</td>
</tr>
<tr>
<td rowspan="7" align="left" colspan="1">Kentucky</td>
<td align="left" rowspan="1" colspan="1">Seed mass</td>
<td align="left" rowspan="1" colspan="1">3.55</td>
<td align="left" rowspan="1" colspan="1">96.45</td>
<td align="left" rowspan="1" colspan="1">NA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TKW</td>
<td align="left" rowspan="1" colspan="1">5.51</td>
<td align="left" rowspan="1" colspan="1">94.49</td>
<td align="left" rowspan="1" colspan="1">NA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Spike count</td>
<td align="left" rowspan="1" colspan="1">6.17</td>
<td align="left" rowspan="1" colspan="1">93.83</td>
<td align="left" rowspan="1" colspan="1">NA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Starch</td>
<td align="left" rowspan="1" colspan="1">30.82</td>
<td align="left" rowspan="1" colspan="1">69.18</td>
<td align="left" rowspan="1" colspan="1">NA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Protein</td>
<td align="left" rowspan="1" colspan="1">10.81</td>
<td align="left" rowspan="1" colspan="1">89.19</td>
<td align="left" rowspan="1" colspan="1">NA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Fiber</td>
<td align="left" rowspan="1" colspan="1">25.03</td>
<td align="left" rowspan="1" colspan="1">74.97</td>
<td align="left" rowspan="1" colspan="1">NA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ash</td>
<td align="left" rowspan="1" colspan="1">33.69</td>
<td align="left" rowspan="1" colspan="1">66.31</td>
<td align="left" rowspan="1" colspan="1">NA</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270008-tbl-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270008-tbl5-note-0001"><p>Abbreviations: NA, not available; TKW, 1000‐kernel weight.</p></div></div></section></section><section id="tpg270008-sec-0140"><h3 class="pmc_sec_title">3.2. Genetic variation</h3>
<p>While many accessions of the Southeast Panel were not differentiated by principal component analysis (Figure <a href="#tpg270008-fig-0003" class="usa-link">3a,b</a>), Rymin, ND Dylan, Balbo, Florida Black, and Elbon were distinguishable. The overall pairwise F<sub>ST</sub> in the Southeast Panel ranged from 0.010 to 0.088, with a mean of 0.045 (Figure <a href="#tpg270008-supinfo-0001" class="usa-link">S4</a>), and the highest mean pairwise <em>F</em>
<sub>ST</sub> in accessions Rymin, Bablo, and ND Dylan (<em>μ</em> &gt; 0.060), and lowest mean pairwise F<sub>ST</sub> in accessions KY1, KY4, and Wrens Abruzzi (<em>μ</em> &lt; 0.032). Likewise, the Kentucky Panel had a few breeding populations differentiate by principal component analysis (Figure <a href="#tpg270008-fig-0004" class="usa-link">4a</a>) and had a similar range of pairwise F<sub>ST</sub> to the Southeast Panel (0.001 &lt; <em>F</em>
<sub>ST</sub> &lt; 0.11, <em>μ</em> = 0.048; Figure <a href="#tpg270008-supinfo-0001" class="usa-link">S5</a>). In neither panel was there a clear separation between “grain” and “forage” types (Figures <a href="#tpg270008-fig-0003" class="usa-link">3c,d</a> and <a href="#tpg270008-fig-0004" class="usa-link">4b</a>). The mean concordance of individuals within accessions ranged from 62.7% to 87.1% (<em>μ</em> = 72.7%) and mean concordance of individuals between accessions ranged from 63.2% to 84.4% (<em>μ</em> = 70.8%) for the Southeast Panel, and was similar in the Kentucky Panel (within: 62.7%–85.6%, <em>μ</em> = 72.3%; between: 63.1%–77.9%, <em>μ</em> = 70.3%). In almost all cases, concordance was highest within accessions (Figures <a href="#tpg270008-supinfo-0001" class="usa-link">S6</a> and <a href="#tpg270008-supinfo-0001" class="usa-link">S7</a>). Finally, the two sources of Aroostook demonstrated differentiation in the PC plot (Figure <a href="#tpg270008-supinfo-0001" class="usa-link">S8</a>) and concordance was higher within sources (74.25% and 74.63%, respectively) than between sources (71.33%).</p>
<figure class="fig xbox font-sm" id="tpg270008-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/41d4/11997422/416e96685bcc/TPG2-18-e70008-g003.jpg" loading="lazy" id="jats-graphic-5" height="715" width="800" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270008-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genomic principal components for individuals within accessions in the (a) full Southeast Panel and (b) the Reduced Southeast Panel (two northern accessions removed). Accession name abbreviation is given in Table <a href="#tpg270008-tbl-0001" class="usa-link">1</a>. Accessions colored by end use are shown for the (c) Southeast Panel (d) the Reduced Southeast Panel.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270008-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/41d4/11997422/32859bbdbd42/TPG2-18-e70008-g001.jpg" loading="lazy" id="jats-graphic-7" height="314" width="800" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270008-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genomic principal components for (a) individuals within Kentucky Panel accessions and (b) accessions colored by end use. Accession description is given in Table <a href="#tpg270008-tbl-0002" class="usa-link">2</a>.</p></figcaption></figure><p>We then sought to characterize factors that contribute to accession differentiation. There were no private alleles identified in either panel, but there were alleles that were absent in a variety (MAF = 0), but at high frequencies (MAF &gt; 0.5) in others. In the Southeast Panel, there were nine markers that met this criterion, and it was driven by Rymin, ND Dylan, Balbo, and Elbon (Figure <a href="#tpg270008-supinfo-0001" class="usa-link">S9</a>). There were 23 SNPs that met this criterion for the KY breeding lines (Figure <a href="#tpg270008-supinfo-0001" class="usa-link">S10</a>), none of which overlapped with those of the Southeast Panel. Finally, mean observed heterozygosity was 0.327 and ranged from 0.279 (KY2) to 0.358 (CCA) where significant pairwise contrasts between accessions were exclusively due to low observed heterozygosity in KY2 (Table <a href="#tpg270008-supinfo-0001" class="usa-link">S4</a>). While we predicted that forage accessions would have higher heterozygosity than grain accessions, there were no significant differences in observed heterozygosity between end uses in either panel (Southeast, <em>F</em>
<sub>1,13 </sub>= 0.32, <em>p </em>= 0.58; Kentucky, <em>F</em>
<sub>1,10 </sub>= 1.02, <em>p </em>= 0.34).</p></section><section id="tpg270008-sec-0150"><h3 class="pmc_sec_title">3.3. Genomic analysis and prediction</h3>
<p>First, no significant loci were detected by GWAS for any trait (Figure <a href="#tpg270008-supinfo-0001" class="usa-link">S11</a>). Then, genomic prediction accuracy for individual plants was assessed. In both the full and Reduced Southeast Panel, prediction accuracy was highest for heading date (<em>r </em>&gt; 0.632), followed by mean spike area (<em>r </em>&gt; 0.548), and prediction accuracy was highest for starch in the Kentucky Panel (<em>r </em>&gt; 0.531) (Figure <a href="#tpg270008-fig-0005" class="usa-link">5</a>; Table <a href="#tpg270008-supinfo-0001" class="usa-link">S5</a>). Prediction accuracy was the lowest for TKW in all panels (<em>r </em>&lt; 0.267). While Bayes B models outperformed GBLUP for most traits, GBLUP outperformed Bayes B for TKW in the full Southeast Panel (Figure <a href="#tpg270008-fig-0005" class="usa-link">5</a>; Table <a href="#tpg270008-supinfo-0001" class="usa-link">S5</a>).</p>
<figure class="fig xbox font-sm" id="tpg270008-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/41d4/11997422/fe7ef9c86c5a/TPG2-18-e70008-g006.jpg" loading="lazy" id="jats-graphic-9" height="779" width="800" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270008-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genomic prediction accuracy determined by fivefold cross‐validation for all measured traits with a genomic BLUP (GBLUP) and Bayes B model. Prediction accuracy is reported for prediction models implemented separately by panel, the (a) Southeast Panel, (b) Reduced Southeast Panel, and the (c) Kentucky Panel, and reported in Table <a href="#tpg270008-supinfo-0001" class="usa-link">S5</a>. Significant differences between models are indicated by the following symbols: *<em>p</em> &lt; 0.05; **<em>p</em> &lt; 0.01; ***<em>p</em> &lt; 0.001; ****<em>p</em> &lt; 0.0001. Nonsignificant comparisons do not have an indicator symbol. TKW, 1000‐kernel weight.</p></figcaption></figure><p>We then assessed genomic prediction accuracy within accessions using the other accessions as the training set. Within‐population prediction accuracy was lower than prediction accuracy assessed by cross‐validation in the entire panel. Across all accessions, mean prediction accuracy within accessions was highest (<em>r </em>&gt; 0.11) for spike count, mean spike area, and heading date in the Southeast Panel; in the Reduced Southeast Panel, within‐accession prediction accuracy dropped for heading date (<em>r </em>&lt; 0.067), but remained higher for spike count and mean spike area (<em>r </em>&gt; 0.13) (Figure <a href="#tpg270008-fig-0006" class="usa-link">6</a>; Table <a href="#tpg270008-supinfo-0001" class="usa-link">S6</a>). In the Kentucky Panel, there were no traits that resulted in <em>r </em>&gt; 0 for all families, although a few families were well predicted (<em>r </em>&gt; 0.4) for protein, fiber, and ash content (Table <a href="#tpg270008-supinfo-0001" class="usa-link">S7</a>). Within panel as source of the training population (Southeast Panel or the Reduced Southeast Panel), there was only one trait with significant differences between the GBLUP and Bayes B models, but within model there were no significant differences between the training populations of the full or Reduced Southeast Panel (Figure <a href="#tpg270008-fig-0006" class="usa-link">6</a>). However, there were differences between accessions; for instance, across traits, mean within‐accession prediction accuracy exceeded <em>r </em>&gt; 0.2 for Wrens Abruzzi and KYSC1710C1 (full and Reduced Southeast Panel) and Balbo and KYSC1707C2 (Reduced Southeast Panel only), but was <em>r </em>&lt; 0 for Explorer, ND Dylan, Athens Abruzzi, Bates (full and Reduced Southeast Panel), and Mississippi Abruzzi (Reduced Southeast Panel only) (Table <a href="#tpg270008-supinfo-0001" class="usa-link">S6</a>). In general, there were positive rank correlations of prediction accuracy of the Southeast Panel accessions within training population and model, but only one instance where the correlation was significant (Table <a href="#tpg270008-supinfo-0001" class="usa-link">S8</a>).</p>
<figure class="fig xbox font-sm" id="tpg270008-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/41d4/11997422/4b64bff12266/TPG2-18-e70008-g002.jpg" loading="lazy" id="jats-graphic-11" height="706" width="800" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270008-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Within‐population genomic prediction accuracy for each focal population within a panel with a genomic BLUP (GBLUP) and Bayes B model. Marker effect estimates and correlations were implemented separately by panel, the Southeast Panel (<em>n </em>= 15 focal populations) and the Reduced Southeast Panel (<em>n </em>= 13 focal populations). Prediction accuracies by focal population are reported in Table <a href="#tpg270008-supinfo-0001" class="usa-link">S6</a>. Comparisons within training population by model (top row) use all available populations. Comparisons of training population within model use only the <em>n </em>= 13 focal populations common between the Southeast and Reduced Southeast Panels. Significant differences between models are indicated by *<em>p</em> &lt; 0.05. Nonsignificant comparisons do not have an indicator symbol. TKW, 1000‐kernel weight.</p></figcaption></figure></section></section><section id="tpg270008-sec-0160"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>We surveyed phenotypic and genotypic variation in and among 25 accessions of cereal rye (<em>Secale cereale</em>) important for regional production in the Southeastern United States. Using more than 500 individuals across the accessions, we show that most phenotypic and genetic variation is within accession, with the exception of the latitudinally structured trait of heading date. Reproductive isolation afforded by extreme differences in heading date likely also drives genetic differentiation of some accessions. Yield‐related traits, such as seed mass and TKW, did not differentiate varieties, although some spike size traits and composition traits exhibited pairwise differences between varieties. We also tested the efficacy of genomic prediction for population improvement in cereal rye and found moderate to high prediction accuracies for prediction across multiple accessions, but low accuracy for prediction within accessions. We thus show that improvement of rye varieties can include selection upon variation within known population varieties, individual plant identity is critical in developing crossing blocks from populations, and selection of individual plants can be facilitated by genomic selection. In summary, this provides a framework for genomics‐enabled population improvement of rye, especially outside of major grain production regions.</p>
<p>We examined phenotypic variation within and between rye accessions by measuring traits at the individual plant level. Although there were some significant pairwise differences in traits between accessions, most phenotypic variation resided within accessions in both the Southeast and Kentucky Panel. The exception was heading date: in both the full and Reduced Southeast Panel, most phenotypic variation in heading date was defined between accessions. In the Kentucky Panel, there was a higher degree of inter‐accession variation for composition rather than yield component traits. A limitation of this work is that the Southeast Panel and Kentucky Panel were grown in different environments, and thus do not allow for direct comparison of accessions between panels. Nonetheless, these findings align with previous reports of Southeastern forage varieties with plot‐level measurements, where environment rather than variety accounted for the preponderance of variation in yield and quality traits (Kim et al., <a href="#tpg270008-bib-0035" class="usa-link" aria-describedby="tpg270008-bib-0035">2016</a>; Webb et al., <a href="#tpg270008-bib-0074" class="usa-link" aria-describedby="tpg270008-bib-0074">2013</a>). Further, even in a survey of agronomic and disease resistance traits measured in bulked samples from 100 global accessions, the authors found no significant effect of genotype on trait values (Targońska‐Karasek et al., <a href="#tpg270008-bib-0070" class="usa-link" aria-describedby="tpg270008-bib-0070">2020</a>). For developmental traits specifically, they did not find significant effects of genotype or geographic origin for grain fill period and days to maturity, but heading date was not reported (Targońska‐Karasek et al., <a href="#tpg270008-bib-0070" class="usa-link" aria-describedby="tpg270008-bib-0070">2020</a>). This highlights the challenges of working in open‐pollinated varieties and accessions, but our study of individual plant phenotypes shows that heading date and grain composition traits contribute to inter‐accession differentiation.</p>
<p>We also evaluated genetic variation within and between accessions. The two northern grain varieties, ND Dylan and Rymin, differentiated from the Southeast Panel, likely owing to more intensive selection for grain quality. In addition, Rymin has two unique and known parents (a University of Manitoba breeding line and cv. Von Lochow) (Robinson, <a href="#tpg270008-bib-0061" class="usa-link" aria-describedby="tpg270008-bib-0061">1973</a>), unlike many other accessions in this study that were selected out of populations with unknown parentage. Within the Southeast Panel, Balbo and Florida Black were strongly differentiated, while various Abruzzi accessions were not. Like the Southeast Panel, two Kentucky breeding lines, KYSC1816C0 and KYSC1912C0 showed substantial differentiation from the others. Interestingly, these populations were the result of narrow crosses with few individuals as parents, compared to other populations that had more individuals from more accessions as parents. We report a mean pairwise F<sub>ST</sub> between accessions in the Southeast Panel and Kentucky Panel of 0.045 and 0.048, respectively. As expected, this is lower than reported for pairwise F<sub>ST</sub> between global accessions, which have ranged from 0.07 to 0.35 (Maraci et al., <a href="#tpg270008-bib-0046" class="usa-link" aria-describedby="tpg270008-bib-0046">2018</a>; Parat et al., <a href="#tpg270008-bib-0053" class="usa-link" aria-describedby="tpg270008-bib-0053">2016</a>). Our mean observed heterozygosity was 0.327 and aligns with previous reports of observed heterozygosity of 0.316–0.322 within accessions (Hagenblad et al., <a href="#tpg270008-bib-0026" class="usa-link" aria-describedby="tpg270008-bib-0026">2016</a>), but is lower than 0.37–0.40 (Gruner et al., <a href="#tpg270008-bib-0025" class="usa-link" aria-describedby="tpg270008-bib-0025">2021</a>) and 0.29–0.55 (Parat et al., <a href="#tpg270008-bib-0053" class="usa-link" aria-describedby="tpg270008-bib-0053">2016</a>) reported in surveys of rye populations with more individuals. Finally, our work supports previous independent studies that the majority of genetic variance resides within accessions rather than between accessions (Hagenblad et al., <a href="#tpg270008-bib-0026" class="usa-link" aria-describedby="tpg270008-bib-0026">2016</a>; Larsson et al., <a href="#tpg270008-bib-0039" class="usa-link" aria-describedby="tpg270008-bib-0039">2019</a>; Monteiro et al., <a href="#tpg270008-bib-0049" class="usa-link" aria-describedby="tpg270008-bib-0049">2016</a>; Parat et al., <a href="#tpg270008-bib-0053" class="usa-link" aria-describedby="tpg270008-bib-0053">2016</a>). The exception to this body of work is that Adamo et al. (<a href="#tpg270008-bib-0001" class="usa-link" aria-describedby="tpg270008-bib-0001">2021</a>) reported more variation between rye landraces found in the Italian Alps, likely due to inbreeding. In sum, rye populations show a high degree of heterozygosity within, but low levels of differentiation between, populations.</p>
<p>Our goal was to couple the phenotypic and genetic evaluation to elucidate factors driving the instances of accession homogeneity and differentiation. It is well‐established that maintaining rye population varieties is challenging due to pollen flow between accessions. Even at the time of release of some varieties used in this study, researchers noted that Abruzzi rye strains from multiple locations in the eastern United States had similar performance (Morey, <a href="#tpg270008-bib-0050" class="usa-link" aria-describedby="tpg270008-bib-0050">1970</a>). Similarly, a high degree of relatedness has been reported between European rye varieties, owing to shared ancestry and intermating (Bolibok‐Brągoszewska et al., <a href="#tpg270008-bib-0009" class="usa-link" aria-describedby="tpg270008-bib-0009">2014</a>). At the same time, selection and genetic drift challenge variety maintenance, and one of the varieties we used, Balbo provides a historic example. At the time of registration, eight states were producing certified seed of Balbo (Qualset &amp; Hoskinson, <a href="#tpg270008-bib-0057" class="usa-link" aria-describedby="tpg270008-bib-0057">1967</a>), spanning from east of Tennessee to Illinois and Kansas (Hancock &amp; Overton, <a href="#tpg270008-bib-0027" class="usa-link" aria-describedby="tpg270008-bib-0027">1960</a>). Researchers noted later, however, that Balbo sourced from locations beyond Tennessee had a different growth habit than the Balbo maintained on an isolated Tennessee island and thus prompted plant registration and foundation stock seed development over 30 years after release (Qualset &amp; Hoskinson, <a href="#tpg270008-bib-0057" class="usa-link" aria-describedby="tpg270008-bib-0057">1967</a>). Similarly, we report here that the variety Aroostook from two different seed sources shows significant phenotypic differences in multiple traits as well as genetic differentiation. This result aligns with a survey of seven Aroostook seed lots that showed that lots varied by more than 10,000 seeds/kg (Lounsbury et al., <a href="#tpg270008-bib-0044" class="usa-link" aria-describedby="tpg270008-bib-0044">2022</a>). Variation in traits like seed weight also has an environmental component (Hansen et al., <a href="#tpg270008-bib-0028" class="usa-link" aria-describedby="tpg270008-bib-0028">2004</a>), but nonetheless suggests differentiation even within named, commercially sourced population varieties. It is clear that balancing the requirements of sufficient population sizes as well as reproductive isolation will continue to challenge maintenance of rye populations.</p>
<p>A specific factor that has been proposed to influence differentiation between rye accessions is the intersection of end use and geography (Hagenblad et al., <a href="#tpg270008-bib-0026" class="usa-link" aria-describedby="tpg270008-bib-0026">2016</a>; Parat et al., <a href="#tpg270008-bib-0053" class="usa-link" aria-describedby="tpg270008-bib-0053">2016</a>), even though geography broadly does not structure rye genetic diversity (Bolibok‐Brągoszewska et al., <a href="#tpg270008-bib-0009" class="usa-link" aria-describedby="tpg270008-bib-0009">2014</a>; Schreiber et al., <a href="#tpg270008-bib-0064" class="usa-link" aria-describedby="tpg270008-bib-0064">2022</a>; Sidhu et al., <a href="#tpg270008-bib-0068" class="usa-link" aria-describedby="tpg270008-bib-0068">2019</a>). In this study, accessions were not delineated by end use categorization. This likely reflects that most rye in the United States is used for forage and cover crops, and that the breeding program at the University of Kentucky only recently included a focus on grain rather than forage breeding. Instead, the degree of reproductive isolation is likely the driving force for accession differentiation. Bolibok‐Brągoszewska et al. (<a href="#tpg270008-bib-0009" class="usa-link" aria-describedby="tpg270008-bib-0009">2014</a>) postulated that precise control of reproduction in a specific breeding company caused those accessions in their study to be differentiated from others more so than other factors. Likewise, Gruner et al. (<a href="#tpg270008-bib-0025" class="usa-link" aria-describedby="tpg270008-bib-0025">2021</a>) showed differentiation in rye populations that have been maintained with highly controlled isolation. Finally, Adamo et al. (<a href="#tpg270008-bib-0001" class="usa-link" aria-describedby="tpg270008-bib-0001">2021</a>) report differentiation between landraces that are separated by mountainous geography that would impede pollen flow. In our study, Florida Black and Balbo have a higher degree of differentiation, and they also have the most extreme heading dates measured. And so, due to early and late heading dates, respectively, Florida Black and Balbo likely have a degree of reproductive isolation not afforded to other accessions, especially in increases in limited area. Similarly, a survey of southeastern maize landraces also shows that flowering date contributes to isolation between varieties (Woore et al., <a href="#tpg270008-bib-0076" class="usa-link" aria-describedby="tpg270008-bib-0076">2024</a>). Overall, reproductive isolation—either purposeful or due to reproductive timing—rather than other geographic factors, may be the most important contributor to rye population differentiation.</p>
<p>In addition to investigating variation and sources of variation in rye, we tested how rye population breeding could be improved by genomic tools. Most GWAS in rye have been conducted in inbred or testcross lines from hybrid programs, and have revealed genetic architecture for many traits, including heading date (Gaikpa et al., <a href="#tpg270008-bib-0020" class="usa-link" aria-describedby="tpg270008-bib-0020">2020</a>; Siekmann et al., <a href="#tpg270008-bib-0069" class="usa-link" aria-describedby="tpg270008-bib-0069">2021</a>). We did not identify any significant marker‐trait associations by GWAS using the 3803 SNPs in the accessions used in our study; this is likely due to the inability to replicate genetically distinct individuals and complex genetic basis of the traits measured. However, using 5708–6286 SNPs, Gruner et al. (<a href="#tpg270008-bib-0025" class="usa-link" aria-describedby="tpg270008-bib-0025">2021</a>) detected significant loci for resistance to stem rust (<em>Puccinia graminis</em> f. sp. <em>secalis</em>) in rye populations, and significant loci were likewise detected for resistance to tan spot (<em>Pyrenophora tritici‐repentis</em>) using 4037 SNPs in a rye germplasm panel (Sidhu et al., <a href="#tpg270008-bib-0068" class="usa-link" aria-describedby="tpg270008-bib-0068">2019</a>). While increased marker density may have improved our results, these findings also underscore the challenge of mapping quantitative morphological traits in populations relative to oligogenic resistance traits. More importantly for breeding, we also tested genomic prediction accuracy. Genomic prediction is widely used in hybrid rye breeding (Miedaner et al., <a href="#tpg270008-bib-0047" class="usa-link" aria-describedby="tpg270008-bib-0047">2019</a>), but has not been evaluated for population breeding. In general, prediction accuracies were moderate to high for most traits using either panel. We also found that Bayes B frequently outperformed GBLUP, but in most cases this difference was not substantial, nor was whether the training population was the full or Reduced Southeast Panel. Because of this high prediction accuracy, a promising strategy for rye population improvement would be, for instance, using genomic prediction to select individuals to intermate, as is currently used in crops like intermediate wheatgrass (<em>Thinopyrum intermedium</em>) (Crain et al., <a href="#tpg270008-bib-0016" class="usa-link" aria-describedby="tpg270008-bib-0016">2021</a>; Zhang et al., <a href="#tpg270008-bib-0078" class="usa-link" aria-describedby="tpg270008-bib-0078">2016</a>). Within‐population prediction accuracies, however, were low and varied significantly by population and trait, where only some populations exhibited acceptable within‐population prediction accuracy. Within‐population prediction accuracy was also limited by population size in the Kentucky Panel. Further work for within‐population prediction may build upon training population optimization strategies developed for within‐biparental family prediction in other grain (Juliana et al., <a href="#tpg270008-bib-0032" class="usa-link" aria-describedby="tpg270008-bib-0032">2022</a>; Lian et al., <a href="#tpg270008-bib-0042" class="usa-link" aria-describedby="tpg270008-bib-0042">2014</a>; Zhu et al., <a href="#tpg270008-bib-0079" class="usa-link" aria-describedby="tpg270008-bib-0079">2021</a>) and fruit crops (Brault et al., <a href="#tpg270008-bib-0010" class="usa-link" aria-describedby="tpg270008-bib-0010">2022</a>; da Silva et al., <a href="#tpg270008-bib-0017" class="usa-link" aria-describedby="tpg270008-bib-0017">2024</a>; Roth et al., <a href="#tpg270008-bib-0062" class="usa-link" aria-describedby="tpg270008-bib-0062">2020</a>). However, given that most accessions lacked meaningful genetic differentiation, rye breeders would likely have more success using genomic selection to identify individuals, regardless of accession identity, for crossing.</p>
<p>Overall, surveying rye genetic and phenotypic diversity, as well as identifying opportunities for genomics‐enabled rye population breeding supports regional rye breeding. In the southeastern United States, rye has continued importance as a forage crop (Bruckner &amp; Raymer, <a href="#tpg270008-bib-0012" class="usa-link" aria-describedby="tpg270008-bib-0012">1990</a>; Newell &amp; Butler, <a href="#tpg270008-bib-0052" class="usa-link" aria-describedby="tpg270008-bib-0052">2013</a>), which has likewise been the main driver of the University of Kentucky rye breeding program. Rye also serves regional needs as a cover crop, and there is growing interest in grain production for regional end uses (Brzozowski et al., <a href="#tpg270008-bib-0013" class="usa-link" aria-describedby="tpg270008-bib-0013">2023</a>). Breeding rye for specific regional needs (e.g., shorter daylengths during grain fill, warmer climates) is critically important as this region is not centered in the major rye grain production areas in the northern United States and Europe where most breeding efforts are focused. Globally, rye serves unique regional niches, and in situ preservation of these rye landraces is critical for regional production systems (Adamo et al., <a href="#tpg270008-bib-0001" class="usa-link" aria-describedby="tpg270008-bib-0001">2021</a>; Larsson et al., <a href="#tpg270008-bib-0039" class="usa-link" aria-describedby="tpg270008-bib-0039">2019</a>; Monteiro et al., <a href="#tpg270008-bib-0049" class="usa-link" aria-describedby="tpg270008-bib-0049">2016</a>). These regional landraces are also an important source of genetic variation for rye breeders (Bolibok‐Brągoszewska et al., <a href="#tpg270008-bib-0009" class="usa-link" aria-describedby="tpg270008-bib-0009">2014</a>; Maraci et al., <a href="#tpg270008-bib-0046" class="usa-link" aria-describedby="tpg270008-bib-0046">2018</a>), and reflect the broader role of regional landraces and varieties in mitigating the erosion of crop genetic diversity to preserve local agricultural practices (Khoury et al., <a href="#tpg270008-bib-0034" class="usa-link" aria-describedby="tpg270008-bib-0034">2022</a>). In summary, we show here that regional Southeastern rye varieties, sourced almost entirely from public rye breeding programs, harbor significant variation and are amenable to further selection, including genomics‐enabled population improvement, for the benefit of regional agriculture.</p></section><section id="tpg270008-sec-0170"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Lauren J. Brzozowski</strong>: Conceptualization; formal analysis; funding acquisition; investigation; writing—original draft; writing—review and editing. <strong>Timothy D. Phillips</strong>: Conceptualization; funding acquisition; investigation; writing—review and editing. <strong>David A. Van Sanford</strong>: Funding acquisition; writing—review and editing.</p></section><section id="tpg270008-sec-0190"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec19"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270008-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270008-supitem-0001"><div class="caption p"><p>Supporting information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11997422/bin/TPG2-18-e70008-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70008-s002.docx</a><sup> (15.3MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270008-supitem-0002"><div class="caption p"><p>Supporting data</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11997422/bin/TPG2-18-e70008-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70008-s001.xlsx</a><sup> (65.6KB, xlsx) </sup>
</div></div></section></section><section id="tpg270008-sec-0180" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank Dr. Anthony Clark for DNA tissue sampling and plant care, and undergraduates Makaslyn Dunham, Jacob Hollar, Makayla Koehler, Ignacio Sanguinetti, and Logan West for care of plants and assistance with data collection. We also thank Dr. Ann Blount sharing historic circulars about rye from the University of Florida. This work was funded in part by a Kentucky Small Grains Growers Association 2023 Research Grant to Lauren J. Brzozowski, Timothy D. Phillips, and David A. Van Sanford, a Dendrifund and Brown‐Forman grant to Lauren J. Brzozowski, Timothy D. Phillips, and David A. Van Sanford as well as Hatch funding (KY006145) and University of Kentucky start‐up funding to Lauren J. Brzozowski.</p></section><section id="notes1"><p>


Brzozowski, L. J.
, 
Phillips, T. D.
, &amp; 
Van Sanford, D. A.
 (2025). Diversity of cereal rye (<em>Secale cereale</em>) germplasm in the Southeast United States. The Plant Genome, 18, e70008. 10.1002/tpg2.70008

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270008-note-0001"><p>Assigned to Associate Editor Rajeev Gupta.</p></div></div></section></section><section id="tpg270008-sec-0210"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>Phenotypic data used in this study are available as Supplementary Data <a href="#tpg270008-supinfo-0001" class="usa-link">S1</a>.</p></section><section id="tpg270008-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270008-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270008-bib-0001">
<cite>

Adamo, M.
, 
Blandino, M.
, 
Capo, L.
, 
Ravetto Enri, S.
, 
Fusconi, A.
, 
Lonati, M.
, &amp; 
Mucciarelli, M.
 (2021). A ddRADseq survey of the genetic diversity of rye (<em>Secale cereale</em> L.) landraces from the western alps reveals the progressive reduction of the local gene pool. Plants, 10, 2415. 10.3390/plants10112415

</cite> [<a href="https://doi.org/10.3390/plants10112415" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8624659/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34834778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=A%20ddRADseq%20survey%20of%20the%20genetic%20diversity%20of%20rye%20(Secale%20cereale%20L.)%20landraces%20from%20the%20western%20alps%20reveals%20the%20progressive%20reduction%20of%20the%20local%20gene%20pool&amp;author=M.%20Adamo&amp;author=M.%20Blandino&amp;author=L.%20Capo&amp;author=S.%20Ravetto%20Enri&amp;author=A.%20Fusconi&amp;volume=10&amp;publication_year=2021&amp;pages=2415&amp;pmid=34834778&amp;doi=10.3390/plants10112415&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0002">
<cite>

Auinger, H.‐J.
, 
Schönleben, M.
, 
Lehermeier, C.
, 
Schmidt, M.
, 
Korzun, V.
, 
Geiger, H. H.
, 
Piepho, H.‐P.
, 
Gordillo, A.
, 
Wilde, P.
, 
Bauer, E.
, &amp; 
Schön, C.‐C.
 (2016). Model training across multiple breeding cycles significantly improves genomic prediction accuracy in rye (<em>Secale cereale</em> L.). Theoretical and Applied Genetics, 129, 2043–2053. 10.1007/s00122-016-2756-5

</cite> [<a href="https://doi.org/10.1007/s00122-016-2756-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5069347/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27480157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Model%20training%20across%20multiple%20breeding%20cycles%20significantly%20improves%20genomic%20prediction%20accuracy%20in%20rye%20(Secale%20cereale%20L.)&amp;author=H.%E2%80%90J.%20Auinger&amp;author=M.%20Sch%C3%B6nleben&amp;author=C.%20Lehermeier&amp;author=M.%20Schmidt&amp;author=V.%20Korzun&amp;volume=129&amp;publication_year=2016&amp;pages=2043-2053&amp;pmid=27480157&amp;doi=10.1007/s00122-016-2756-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0003">
<cite>

Båga, M.
, 
Bahrani, H.
, 
Larsen, J.
, 
Hackauf, B.
, 
Graf, R. J.
, 
Laroche, A.
, &amp; 
Chibbar, R. N.
 (2022). Association mapping of autumn‐seeded rye (<em>Secale cereale</em> L.) reveals genetic linkages between genes controlling winter hardiness and plant development. Scientific Reports, 12, Article 5793. 10.1038/s41598-022-09582-2

</cite> [<a href="https://doi.org/10.1038/s41598-022-09582-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8986816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35388069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Association%20mapping%20of%20autumn%E2%80%90seeded%20rye%20(Secale%20cereale%20L.)%20reveals%20genetic%20linkages%20between%20genes%20controlling%20winter%20hardiness%20and%20plant%20development&amp;author=M.%20B%C3%A5ga&amp;author=H.%20Bahrani&amp;author=J.%20Larsen&amp;author=B.%20Hackauf&amp;author=R.%20J.%20Graf&amp;volume=12&amp;publication_year=2022&amp;pages=5793&amp;pmid=35388069&amp;doi=10.1038/s41598-022-09582-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0004">
<cite>

Baker, J. L.
, 
Bates, R. P.
, &amp; 
Nelson, L. R.
 (1996). Registration of ‘Bates’ rye. Crop Science, 36, 810–810. 10.2135/cropsci1996.0011183X003600030058x
</cite> [<a href="https://doi.org/10.2135/cropsci1996.0011183X003600030058x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Registration%20of%20%E2%80%98Bates%E2%80%99%20rye&amp;author=J.%20L.%20Baker&amp;author=R.%20P.%20Bates&amp;author=L.%20R.%20Nelson&amp;volume=36&amp;publication_year=1996&amp;pages=810-810&amp;doi=10.2135/cropsci1996.0011183X003600030058x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0005"><cite>

Barnes, B. S.
, &amp; 
Wells, D. G.
 (1960). 
Explorer: A new rye for Mississippi
 (Research Bulletin 600). Mississippi Agricultural and Forestry Experiment Station. <a href="https://scholarsjunction.msstate.edu/cgi/viewcontent.cgi?article=1400&amp;context=mafes-bulletins" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholarsjunction.msstate.edu/cgi/viewcontent.cgi?article=1400&amp;context=mafes‐bulletins</a>
</cite></li>
<li id="tpg270008-bib-0006">
<cite>

Barnett, R. D.
, 
Blount, A. R.
, 
Pfahler, P. L.
, 
Johnson, J. W.
, 
Buntin, G. D.
, &amp; 
Cunfer, B. M.
 (2002). Rye and triticale breeding in the south. University of Florida IFAS Extension.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Rye%20and%20triticale%20breeding%20in%20the%20south&amp;author=R.%20D.%20Barnett&amp;author=A.%20R.%20Blount&amp;author=P.%20L.%20Pfahler&amp;author=J.%20W.%20Johnson&amp;author=G.%20D.%20Buntin&amp;publication_year=2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0007">
<cite>

Bates, D.
, 
Maechler, M.
, 
Bolker, B.
, &amp; 
Walker, S.
 (2015). Fitting linear mixed‐effects models using lme4. Journal of Statistical Software, 67, 1–48. 10.18637/jss.v067.i01
</cite> [<a href="https://doi.org/10.18637/jss.v067.i01" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=Fitting%20linear%20mixed%E2%80%90effects%20models%20using%20lme4&amp;author=D.%20Bates&amp;author=M.%20Maechler&amp;author=B.%20Bolker&amp;author=S.%20Walker&amp;volume=67&amp;publication_year=2015&amp;pages=1-48&amp;doi=10.18637/jss.v067.i01&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0008">
<cite>

Bernal‐Vasquez, A.‐M.
, 
Gordillo, A.
, 
Schmidt, M.
, &amp; 
Piepho, H.‐P.
 (2017). Genomic prediction in early selection stages using multi‐year data in a hybrid rye breeding program. BMC Genetics, 18, Article 51. 10.1186/s12863-017-0512-8

</cite> [<a href="https://doi.org/10.1186/s12863-017-0512-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5452640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28569139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genetics&amp;title=Genomic%20prediction%20in%20early%20selection%20stages%20using%20multi%E2%80%90year%20data%20in%20a%20hybrid%20rye%20breeding%20program&amp;author=A.%E2%80%90M.%20Bernal%E2%80%90Vasquez&amp;author=A.%20Gordillo&amp;author=M.%20Schmidt&amp;author=H.%E2%80%90P.%20Piepho&amp;volume=18&amp;publication_year=2017&amp;pages=51&amp;pmid=28569139&amp;doi=10.1186/s12863-017-0512-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0009">
<cite>

Bolibok‐Brągoszewska, H.
, 
Targońska, M.
, 
Bolibok, L.
, 
Kilian, A.
, &amp; 
Rakoczy‐Trojanowska, M.
 (2014). Genome‐wide characterization of genetic diversity and population structure in <em>Secale</em>
. BMC Plant Biology, 14, 184. 10.1186/1471-2229-14-184

</cite> [<a href="https://doi.org/10.1186/1471-2229-14-184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4236688/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25085433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Genome%E2%80%90wide%20characterization%20of%20genetic%20diversity%20and%20population%20structure%20in%20Secale&amp;author=H.%20Bolibok%E2%80%90Br%C4%85goszewska&amp;author=M.%20Targo%C5%84ska&amp;author=L.%20Bolibok&amp;author=A.%20Kilian&amp;author=M.%20Rakoczy%E2%80%90Trojanowska&amp;volume=14&amp;publication_year=2014&amp;pages=184&amp;pmid=25085433&amp;doi=10.1186/1471-2229-14-184&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0010">
<cite>

Brault, C.
, 
Segura, V.
, 
This, P.
, 
Le Cunff, L.
, 
Flutre, T.
, 
François, P.
, 
Pons, T.
, 
Péros, J.‐P.
, &amp; 
Doligez, A.
 (2022). Across‐population genomic prediction in grapevine opens up promising prospects for breeding. Horticulture Research, 9, uhac041. 10.1093/hr/uhac041

</cite> [<a href="https://doi.org/10.1093/hr/uhac041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9070645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35184162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=Across%E2%80%90population%20genomic%20prediction%20in%20grapevine%20opens%20up%20promising%20prospects%20for%20breeding&amp;author=C.%20Brault&amp;author=V.%20Segura&amp;author=P.%20This&amp;author=L.%20Le%20Cunff&amp;author=T.%20Flutre&amp;volume=9&amp;publication_year=2022&amp;pages=uhac041&amp;pmid=35184162&amp;doi=10.1093/hr/uhac041&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0011">
<cite>

Brown, A. R.
 (1973). Registration of Athens Abruzzi rye (reg. no. 5). Crop Science, 13, 397–398. 10.2135/cropsci1973.0011183X001300030037x
</cite> [<a href="https://doi.org/10.2135/cropsci1973.0011183X001300030037x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Registration%20of%20Athens%20Abruzzi%20rye%20(reg.%20no.%205)&amp;author=A.%20R.%20Brown&amp;volume=13&amp;publication_year=1973&amp;pages=397-398&amp;doi=10.2135/cropsci1973.0011183X001300030037x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0012">
<cite>

Bruckner, P. L.
, &amp; 
Raymer, P. L.
 (1990). Factors influencing species and cultivar choice of small grains for winter forage. Journal of Production Agriculture, 3, 349–355. 10.2134/jpa1990.0349
</cite> [<a href="https://doi.org/10.2134/jpa1990.0349" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Production%20Agriculture&amp;title=Factors%20influencing%20species%20and%20cultivar%20choice%20of%20small%20grains%20for%20winter%20forage&amp;author=P.%20L.%20Bruckner&amp;author=P.%20L.%20Raymer&amp;volume=3&amp;publication_year=1990&amp;pages=349-355&amp;doi=10.2134/jpa1990.0349&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0013">
<cite>

Brzozowski, L. J.
, 
Szuleta, E.
, 
Phillips, T. D.
, 
Van Sanford, D. A.
, &amp; 
Clark, A. J.
 (2023). Breeding cereal rye (<em>Secale cereale</em>) for quality traits. Crop Science, 63, 1964–1987. 10.1002/csc2.21022
</cite> [<a href="https://doi.org/10.1002/csc2.21022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Breeding%20cereal%20rye%20(Secale%20cereale)%20for%20quality%20traits&amp;author=L.%20J.%20Brzozowski&amp;author=E.%20Szuleta&amp;author=T.%20D.%20Phillips&amp;author=D.%20A.%20Van%20Sanford&amp;author=A.%20J.%20Clark&amp;volume=63&amp;publication_year=2023&amp;pages=1964-1987&amp;doi=10.1002/csc2.21022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0014">
<cite>

Chebotar, S.
, 
Röder, M. S.
, 
Korzun, V.
, 
Saal, B.
, 
Weber, W. E.
, &amp; 
Börner, A.
 (2003). Molecular studies on genetic integrity of open‐pollinating species rye (<em>Secale cereale</em> L.) after long‐term genebank maintenance. Theoretical and Applied Genetics, 107, 1469–1476. 10.1007/s00122-003-1366-1

</cite> [<a href="https://doi.org/10.1007/s00122-003-1366-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12898026/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Molecular%20studies%20on%20genetic%20integrity%20of%20open%E2%80%90pollinating%20species%20rye%20(Secale%20cereale%20L.)%20after%20long%E2%80%90term%20genebank%20maintenance&amp;author=S.%20Chebotar&amp;author=M.%20S.%20R%C3%B6der&amp;author=V.%20Korzun&amp;author=B.%20Saal&amp;author=W.%20E.%20Weber&amp;volume=107&amp;publication_year=2003&amp;pages=1469-1476&amp;pmid=12898026&amp;doi=10.1007/s00122-003-1366-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0015">
<cite>

Covarrubias‐Pazaran, G.
 (2016). Genome‐Assisted prediction of quantitative traits using the R package <em>sommer</em>
. PLoS One, 11, e0156744. 10.1371/journal.pone.0156744

</cite> [<a href="https://doi.org/10.1371/journal.pone.0156744" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4894563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27271781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Genome%E2%80%90Assisted%20prediction%20of%20quantitative%20traits%20using%20the%20R%20package%20sommer&amp;author=G.%20Covarrubias%E2%80%90Pazaran&amp;volume=11&amp;publication_year=2016&amp;pages=e0156744&amp;pmid=27271781&amp;doi=10.1371/journal.pone.0156744&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0016">
<cite>

Crain, J.
, 
DeHaan, L.
, &amp; 
Poland, J.
 (2021). Genomic prediction enables rapid selection of high‐performing genets in an intermediate wheatgrass breeding program. The Plant Genome, 14, e20080. 10.1002/tpg2.20080

</cite> [<a href="https://doi.org/10.1002/tpg2.20080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33660427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20prediction%20enables%20rapid%20selection%20of%20high%E2%80%90performing%20genets%20in%20an%20intermediate%20wheatgrass%20breeding%20program&amp;author=J.%20Crain&amp;author=L.%20DeHaan&amp;author=J.%20Poland&amp;volume=14&amp;publication_year=2021&amp;pages=e20080&amp;pmid=33660427&amp;doi=10.1002/tpg2.20080&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0017">
<cite>

da Silva, A. V.
, 
Prado, M.
, 
Campos, G. R.
, 
Borges, K. L. R.
, 
Yassue, R. M.
, 
Husein, G.
, 
Sposito, M. B.
, 
Amorim, L.
, 
Crossa, J.
, &amp; 
Fritsche‐Neto, R.
 (2024). Comparing strategies for genomic predictions in interspecific biparental populations: A case study with the <em>Rubus</em> genus. Euphytica, 220, Article 146. 10.1007/s10681-024-03406-2
</cite> [<a href="https://doi.org/10.1007/s10681-024-03406-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Comparing%20strategies%20for%20genomic%20predictions%20in%20interspecific%20biparental%20populations:%20A%20case%20study%20with%20the%20Rubus%20genus&amp;author=A.%20V.%20da%20Silva&amp;author=M.%20Prado&amp;author=G.%20R.%20Campos&amp;author=K.%20L.%20R.%20Borges&amp;author=R.%20M.%20Yassue&amp;volume=220&amp;publication_year=2024&amp;pages=146&amp;doi=10.1007/s10681-024-03406-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0018">
<cite>

Endelman, J. B.
 (2011). Ridge regression and other kernels for genomic selection with R package rrBLUP. The Plant Genome, 4, 250–255. 10.3835/plantgenome2011.08.0024
</cite> [<a href="https://doi.org/10.3835/plantgenome2011.08.0024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Ridge%20regression%20and%20other%20kernels%20for%20genomic%20selection%20with%20R%20package%20rrBLUP&amp;author=J.%20B.%20Endelman&amp;volume=4&amp;publication_year=2011&amp;pages=250-255&amp;doi=10.3835/plantgenome2011.08.0024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0019">
<cite>

Esfandyari, H.
, 
Fè, D.
, 
Tessema, B. B.
, 
Janss, L. L.
, &amp; 
Jensen, J.
 (2020). Effects of different strategies for exploiting genomic selection in perennial ryegrass breeding programs. G3 Genes|Genomes|Genetics, 10, 3783–3795. 10.1534/g3.120.401382

</cite> [<a href="https://doi.org/10.1534/g3.120.401382" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7534426/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32819970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Effects%20of%20different%20strategies%20for%20exploiting%20genomic%20selection%20in%20perennial%20ryegrass%20breeding%20programs&amp;author=H.%20Esfandyari&amp;author=D.%20F%C3%A8&amp;author=B.%20B.%20Tessema&amp;author=L.%20L.%20Janss&amp;author=J.%20Jensen&amp;volume=10&amp;publication_year=2020&amp;pages=3783-3795&amp;pmid=32819970&amp;doi=10.1534/g3.120.401382&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0020">
<cite>

Gaikpa, D. S.
, 
Koch, S.
, 
Fromme, F. J.
, 
Siekmann, D.
, 
Würschum, T.
, &amp; 
Miedaner, T.
 (2020). Genome‐wide association mapping and genomic prediction of Fusarium head blight resistance, heading stage and plant height in winter rye (<em>Secale cereale</em>). Plant Breeding, 139, 508–520. 10.1111/pbr.12810
</cite> [<a href="https://doi.org/10.1111/pbr.12810" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding&amp;title=Genome%E2%80%90wide%20association%20mapping%20and%20genomic%20prediction%20of%20Fusarium%20head%20blight%20resistance,%20heading%20stage%20and%20plant%20height%20in%20winter%20rye%20(Secale%20cereale)&amp;author=D.%20S.%20Gaikpa&amp;author=S.%20Koch&amp;author=F.%20J.%20Fromme&amp;author=D.%20Siekmann&amp;author=T.%20W%C3%BCrschum&amp;volume=139&amp;publication_year=2020&amp;pages=508-520&amp;doi=10.1111/pbr.12810&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0021">
<cite>

Galán, R. J.
, 
Bernal‐Vasquez, A.‐M.
, 
Jebsen, C.
, 
Piepho, H.‐P.
, 
Thorwarth, P.
, 
Steffan, P.
, 
Gordillo, A.
, &amp; 
Miedaner, T.
 (2020). Integration of genotypic, hyperspectral, and phenotypic data to improve biomass yield prediction in hybrid rye. Theoretical and Applied Genetics, 133, 3001–3015. 10.1007/s00122-020-03651-8

</cite> [<a href="https://doi.org/10.1007/s00122-020-03651-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7548001/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32681289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Integration%20of%20genotypic,%20hyperspectral,%20and%20phenotypic%20data%20to%20improve%20biomass%20yield%20prediction%20in%20hybrid%20rye&amp;author=R.%20J.%20Gal%C3%A1n&amp;author=A.%E2%80%90M.%20Bernal%E2%80%90Vasquez&amp;author=C.%20Jebsen&amp;author=H.%E2%80%90P.%20Piepho&amp;author=P.%20Thorwarth&amp;volume=133&amp;publication_year=2020&amp;pages=3001-3015&amp;pmid=32681289&amp;doi=10.1007/s00122-020-03651-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0022">
<cite>

Garrick, D. J.
, 
Taylor, J. F.
, &amp; 
Fernando, R. L.
 (2009). Deregressing estimated breeding values and weighting information for genomic regression analyses. Genetics Selection Evolution, 41, Article 55. 10.1186/1297-9686-41-55
</cite> [<a href="https://doi.org/10.1186/1297-9686-41-55" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2817680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20043827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Selection%20Evolution&amp;title=Deregressing%20estimated%20breeding%20values%20and%20weighting%20information%20for%20genomic%20regression%20analyses&amp;author=D.%20J.%20Garrick&amp;author=J.%20F.%20Taylor&amp;author=R.%20L.%20Fernando&amp;volume=41&amp;publication_year=2009&amp;pages=55&amp;pmid=20043827&amp;doi=10.1186/1297-9686-41-55&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0023">
<cite>

Geiger, H. H.
, &amp; 
Miedaner, T.
 (2009). Rye breeding. In Carena M. J. (Ed.), Cereals (pp. 157–181). Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Cereals&amp;author=H.%20H.%20Geiger&amp;author=T.%20Miedaner&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0024">
<cite>

Goudet, J.
 (2005). HIERFSTAT, a package for R to compute and test hierarchical F‐statistics. Molecular Ecology Notes, 5, 184–186. 10.1111/j.1471-8286.2004.00828.x
</cite> [<a href="https://doi.org/10.1111/j.1471-8286.2004.00828.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Notes&amp;title=HIERFSTAT,%20a%20package%20for%20R%20to%20compute%20and%20test%20hierarchical%20F%E2%80%90statistics&amp;author=J.%20Goudet&amp;volume=5&amp;publication_year=2005&amp;pages=184-186&amp;doi=10.1111/j.1471-8286.2004.00828.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0025">
<cite>

Gruner, P.
, 
Schmitt, A.‐K.
, 
Flath, K.
, 
Piepho, H.‐P.
, &amp; 
Miedaner, T.
 (2021). Mapping and validating stem rust resistance genes directly in self‐incompatible genetic resources of winter rye. Theoretical and Applied Genetics, 134, 1989–2003. 10.1007/s00122-021-03800-7

</cite> [<a href="https://doi.org/10.1007/s00122-021-03800-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8263455/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33688982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Mapping%20and%20validating%20stem%20rust%20resistance%20genes%20directly%20in%20self%E2%80%90incompatible%20genetic%20resources%20of%20winter%20rye&amp;author=P.%20Gruner&amp;author=A.%E2%80%90K.%20Schmitt&amp;author=K.%20Flath&amp;author=H.%E2%80%90P.%20Piepho&amp;author=T.%20Miedaner&amp;volume=134&amp;publication_year=2021&amp;pages=1989-2003&amp;pmid=33688982&amp;doi=10.1007/s00122-021-03800-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0026">
<cite>

Hagenblad, J.
, 
Oliveira, H. R.
, 
Forsberg, N. E. G.
, &amp; 
Leino, M. W.
 (2016). Geographical distribution of genetic diversity in <em>Secale</em> landrace and wild accessions. BMC Plant Biology, 16, Article 23. 10.1186/s12870-016-0710-y

</cite> [<a href="https://doi.org/10.1186/s12870-016-0710-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4719562/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26786820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Geographical%20distribution%20of%20genetic%20diversity%20in%20Secale%20landrace%20and%20wild%20accessions&amp;author=J.%20Hagenblad&amp;author=H.%20R.%20Oliveira&amp;author=N.%20E.%20G.%20Forsberg&amp;author=M.%20W.%20Leino&amp;volume=16&amp;publication_year=2016&amp;pages=23&amp;pmid=26786820&amp;doi=10.1186/s12870-016-0710-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0027">
<cite>

Hancock, N. I.
, &amp; 
Overton, J. R.
 (1960). 
Behavior and adaptation of Balbo and Tetra Petkus rye
 (AgResearch Bulletin no. 307). University of Tennessee Agricultural Experiment Station.</cite> [<a href="https://scholar.google.com/scholar_lookup?Hancock,%20N.%20I.%20,%20&amp;%20Overton,%20J.%20R.%20(1960).%20Behavior%20and%20adaptation%20of%20Balbo%20and%20Tetra%20Petkus%20rye%20(AgResearch%20Bulletin%20no.%20307).%20University%20of%20Tennessee%20Agricultural%20Experiment%20Station." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0028">
<cite>

Hansen, H. B.
, 
Møller, B.
, 
Andersen, S. B.
, 
Jørgensen, J. R.
, &amp; 
Hansen, Å.
 (2004). Grain characteristics, chemical composition, and functional properties of rye (<em>Secale cereale</em> L.) as influenced by genotype and harvest year. Journal of Agricultural and Food Chemistry, 52, 2282–2291. 10.1021/jf0307191

</cite> [<a href="https://doi.org/10.1021/jf0307191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15080634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agricultural%20and%20Food%20Chemistry&amp;title=Grain%20characteristics,%20chemical%20composition,%20and%20functional%20properties%20of%20rye%20(Secale%20cereale%20L.)%20as%20influenced%20by%20genotype%20and%20harvest%20year&amp;author=H.%20B.%20Hansen&amp;author=B.%20M%C3%B8ller&amp;author=S.%20B.%20Andersen&amp;author=J.%20R.%20J%C3%B8rgensen&amp;author=%C3%85.%20Hansen&amp;volume=52&amp;publication_year=2004&amp;pages=2282-2291&amp;pmid=15080634&amp;doi=10.1021/jf0307191&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0029">
<cite>

Hawliczek, A.
, 
Borzęcka, E.
, 
Tofil, K.
, 
Alachiotis, N.
, 
Bolibok, L.
, 
Gawroński, P.
, 
Siekmann, D.
, 
Hackauf, B.
, 
Dušinský, R.
, 
Švec, M.
, &amp; 
Bolibok‐Brągoszewska, H.
 (2023). Selective sweeps identification in distinct groups of cultivated rye (<em>Secale cereale</em> L.) germplasm provides potential candidate genes for crop improvement. BMC Plant Biology, 23, Article 323. 10.1186/s12870-023-04337-1

</cite> [<a href="https://doi.org/10.1186/s12870-023-04337-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10273710/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37328739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Selective%20sweeps%20identification%20in%20distinct%20groups%20of%20cultivated%20rye%20(Secale%20cereale%20L.)%20germplasm%20provides%20potential%20candidate%20genes%20for%20crop%20improvement&amp;author=A.%20Hawliczek&amp;author=E.%20Borz%C4%99cka&amp;author=K.%20Tofil&amp;author=N.%20Alachiotis&amp;author=L.%20Bolibok&amp;volume=23&amp;publication_year=2023&amp;pages=323&amp;pmid=37328739&amp;doi=10.1186/s12870-023-04337-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0030">
<cite>

Hothorn, T.
, 
Bretz, F.
, &amp; 
Westfall, P.
 (2008). Simultaneous inference in general parametric models. Biometrical Journal, 50, 346–363. 10.1002/bimj.200810425

</cite> [<a href="https://doi.org/10.1002/bimj.200810425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18481363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrical%20Journal&amp;title=Simultaneous%20inference%20in%20general%20parametric%20models&amp;author=T.%20Hothorn&amp;author=F.%20Bretz&amp;author=P.%20Westfall&amp;volume=50&amp;publication_year=2008&amp;pages=346-363&amp;pmid=18481363&amp;doi=10.1002/bimj.200810425&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0031">
<cite>

Jombart, T.
 (2008). 
<em>adegenet</em> : A R package for the multivariate analysis of genetic markers. Bioinformatics, 24, 1403–1405. 10.1093/bioinformatics/btn129

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btn129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18397895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=adegenet%C2%A0:%20A%20R%20package%20for%20the%20multivariate%20analysis%20of%20genetic%20markers&amp;author=T.%20Jombart&amp;volume=24&amp;publication_year=2008&amp;pages=1403-1405&amp;pmid=18397895&amp;doi=10.1093/bioinformatics/btn129&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0032">
<cite>

Juliana, P.
, 
He, X.
, 
Poland, J.
, 
Roy, K. K.
, 
Malaker, P. K.
, 
Mishra, V. K.
, 
Chand, R.
, 
Shrestha, S.
, 
Kumar, U.
, 
Roy, C.
, 
Gahtyari, N. C.
, 
Joshi, A. K.
, 
Singh, R. P.
, &amp; 
Singh, P. K.
 (2022). Genomic selection for spot blotch in bread wheat breeding panels, full‐sibs and half‐sibs and index‐based selection for spot blotch, heading and plant height. Theoretical and Applied Genetics, 135, 1965–1983. 10.1007/s00122-022-04087-y

</cite> [<a href="https://doi.org/10.1007/s00122-022-04087-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9205839/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35416483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genomic%20selection%20for%20spot%20blotch%20in%20bread%20wheat%20breeding%20panels,%20full%E2%80%90sibs%20and%20half%E2%80%90sibs%20and%20index%E2%80%90based%20selection%20for%20spot%20blotch,%20heading%20and%20plant%20height&amp;author=P.%20Juliana&amp;author=X.%20He&amp;author=J.%20Poland&amp;author=K.%20K.%20Roy&amp;author=P.%20K.%20Malaker&amp;volume=135&amp;publication_year=2022&amp;pages=1965-1983&amp;pmid=35416483&amp;doi=10.1007/s00122-022-04087-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0033"><cite>

Kassambara, A.
 (2020). 
ggpubr: “ggplot2” Based Publication Ready Plots
. <a href="https://CRAN.R-project.org/package=ggpubr" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R‐project.org/package=ggpubr</a>
</cite></li>
<li id="tpg270008-bib-0034">
<cite>

Khoury, C. K.
, 
Brush, S.
, 
Costich, D. E.
, 
Curry, H. A.
, 
Haan, S.
, 
Engels, J. M. M.
, 
Guarino, L.
, 
Hoban, S.
, 
Mercer, K. L.
, 
Miller, A. J.
, 
Nabhan, G. P.
, 
Perales, H. R.
, 
Richards, C.
, 
Riggins, C.
, &amp; 
Thormann, I.
 (2022). Crop genetic erosion: Understanding and responding to loss of crop diversity. New Phytologist, 233, 84–118. 10.1111/nph.17733

</cite> [<a href="https://doi.org/10.1111/nph.17733" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34515358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Crop%20genetic%20erosion:%20Understanding%20and%20responding%20to%20loss%20of%20crop%20diversity&amp;author=C.%20K.%20Khoury&amp;author=S.%20Brush&amp;author=D.%20E.%20Costich&amp;author=H.%20A.%20Curry&amp;author=S.%20Haan&amp;volume=233&amp;publication_year=2022&amp;pages=84-118&amp;pmid=34515358&amp;doi=10.1111/nph.17733&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0035">
<cite>

Kim, K.‐S.
, 
Anderson, J. D.
, 
Newell, M. A.
, &amp; 
Butler, T. J.
 (2016). Variations of forage yield and nutritive value in winter rye germplasm. Crop Science, 56, 1018–1024. 10.2135/cropsci2015.08.0487
</cite> [<a href="https://doi.org/10.2135/cropsci2015.08.0487" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Variations%20of%20forage%20yield%20and%20nutritive%20value%20in%20winter%20rye%20germplasm&amp;author=K.%E2%80%90S.%20Kim&amp;author=J.%20D.%20Anderson&amp;author=M.%20A.%20Newell&amp;author=T.%20J.%20Butler&amp;volume=56&amp;publication_year=2016&amp;pages=1018-1024&amp;doi=10.2135/cropsci2015.08.0487&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0036">
<cite>

Korzun, V.
, 
Ponomareva, M. L.
, &amp; 
Sorrells, M. E.
 (2021). Economic and academic importance of rye. In Rabanus‐Wallace M. T. &amp; Stein N. (Eds.), The rye genome (pp. 1–12). Springer International Publishing.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20rye%20genome&amp;author=V.%20Korzun&amp;author=M.%20L.%20Ponomareva&amp;author=M.%20E.%20Sorrells&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0037">
<cite>

Kuznetsova, A.
, 
Brockhoff, P. B.
, &amp; 
Christensen, R. H. B.
 (2017). lmerTest package: Tests in linear mixed effects models. Journal of Statistical Software, 82, 1–26. 10.18637/jss.v082.i13
</cite> [<a href="https://doi.org/10.18637/jss.v082.i13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=lmerTest%20package:%20Tests%20in%20linear%20mixed%20effects%20models&amp;author=A.%20Kuznetsova&amp;author=P.%20B.%20Brockhoff&amp;author=R.%20H.%20B.%20Christensen&amp;volume=82&amp;publication_year=2017&amp;pages=1-26&amp;doi=10.18637/jss.v082.i13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0038">
<cite>

Laidig, F.
, 
Piepho, H.‐P.
, 
Rentel, D.
, 
Drobek, T.
, 
Meyer, U.
, &amp; 
Huesken, A.
 (2017). Breeding progress, variation, and correlation of grain and quality traits in winter rye hybrid and population varieties and national on‐farm progress in Germany over 26 years. Theoretical and Applied Genetics, 130, 981–998. 10.1007/s00122-017-2865-9

</cite> [<a href="https://doi.org/10.1007/s00122-017-2865-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5395587/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28289803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Breeding%20progress,%20variation,%20and%20correlation%20of%20grain%20and%20quality%20traits%20in%20winter%20rye%20hybrid%20and%20population%20varieties%20and%20national%20on%E2%80%90farm%20progress%20in%20Germany%20over%2026%20years&amp;author=F.%20Laidig&amp;author=H.%E2%80%90P.%20Piepho&amp;author=D.%20Rentel&amp;author=T.%20Drobek&amp;author=U.%20Meyer&amp;volume=130&amp;publication_year=2017&amp;pages=981-998&amp;pmid=28289803&amp;doi=10.1007/s00122-017-2865-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0039">
<cite>

Larsson, P.
, 
Oliveira, H. R.
, 
Lundström, M.
, 
Hagenblad, J.
, 
Lagerås, P.
, &amp; 
Leino, M. W.
 (2019). Population genetic structure in Fennoscandian landrace rye (<em>Secale cereale</em> L.) spanning 350 years. Genetic Resources and Crop Evolution, 66, 1059–1071. 10.1007/s10722-019-00770-0
</cite> [<a href="https://doi.org/10.1007/s10722-019-00770-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetic%20Resources%20and%20Crop%20Evolution&amp;title=Population%20genetic%20structure%20in%20Fennoscandian%20landrace%20rye%20(Secale%20cereale%20L.)%20spanning%20350%20years&amp;author=P.%20Larsson&amp;author=H.%20R.%20Oliveira&amp;author=M.%20Lundstr%C3%B6m&amp;author=J.%20Hagenblad&amp;author=P.%20Lager%C3%A5s&amp;volume=66&amp;publication_year=2019&amp;pages=1059-1071&amp;doi=10.1007/s10722-019-00770-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0040"><cite>

Length, R. V.
 (2023). 
emmeans: Estimated Marginal Means, aka Least‐Squares Means
. <a href="https://CRAN.R-project.org/package=emmeans" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R‐project.org/package=emmeans</a>
</cite></li>
<li id="tpg270008-bib-0041">
<cite>

Li, G.
, 
Wang, L.
, 
Yang, J.
, 
He, H.
, 
Jin, H.
, 
Li, X.
, 
Ren, T.
, 
Ren, Z.
, 
Li, F.
, 
Han, X.
, 
Zhao, X.
, 
Dong, L.
, 
Li, Y.
, 
Song, Z.
, 
Yan, Z.
, 
Zheng, N.
, 
Shi, C.
, 
Wang, Z.
, 
Yang, S.
, … 
Wang, D.
 (2021). A high‐quality genome assembly highlights rye genomic characteristics and agronomically important genes. Nature Genetics, 53, 574–584. 10.1038/s41588-021-00808-z

</cite> [<a href="https://doi.org/10.1038/s41588-021-00808-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8035075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33737755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=A%20high%E2%80%90quality%20genome%20assembly%20highlights%20rye%20genomic%20characteristics%20and%20agronomically%20important%20genes&amp;author=G.%20Li&amp;author=L.%20Wang&amp;author=J.%20Yang&amp;author=H.%20He&amp;author=H.%20Jin&amp;volume=53&amp;publication_year=2021&amp;pages=574-584&amp;pmid=33737755&amp;doi=10.1038/s41588-021-00808-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0042">
<cite>

Lian, L.
, 
Jacobson, A.
, 
Zhong, S.
, &amp; 
Bernardo, R.
 (2014). Genomewide prediction accuracy within 969 maize biparental populations. Crop Science, 54, 1514–1522. 10.2135/cropsci2013.12.0856
</cite> [<a href="https://doi.org/10.2135/cropsci2013.12.0856" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genomewide%20prediction%20accuracy%20within%20969%20maize%20biparental%20populations&amp;author=L.%20Lian&amp;author=A.%20Jacobson&amp;author=S.%20Zhong&amp;author=R.%20Bernardo&amp;volume=54&amp;publication_year=2014&amp;pages=1514-1522&amp;doi=10.2135/cropsci2013.12.0856&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0043">
<cite>

Lin, Z.
, 
Cogan, N. O. I.
, 
Pembleton, L. W.
, 
Spangenberg, G. C.
, 
Forster, J. W.
, 
Hayes, B. J.
, &amp; 
Daetwyler, H. D.
 (2016). Genetic gain and inbreeding from genomic selection in a simulated commercial breeding program for perennial ryegrass. The Plant Genome, 9, plantgenome2015.06.0046. 10.3835/plantgenome2015.06.0046
</cite> [<a href="https://doi.org/10.3835/plantgenome2015.06.0046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27898764/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genetic%20gain%20and%20inbreeding%20from%20genomic%20selection%20in%20a%20simulated%20commercial%20breeding%20program%20for%20perennial%20ryegrass&amp;author=Z.%20Lin&amp;author=N.%20O.%20I.%20Cogan&amp;author=L.%20W.%20Pembleton&amp;author=G.%20C.%20Spangenberg&amp;author=J.%20W.%20Forster&amp;volume=9&amp;publication_year=2016&amp;pmid=27898764&amp;doi=10.3835/plantgenome2015.06.0046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0044">
<cite>

Lounsbury, N. P.
, 
Warren, N. D.
, 
Hobbie, J.
, 
Darby, H.
, 
Ryan, M. R.
, 
Mortensen, D. A.
, &amp; 
Smith, R. G.
 (2022). Seed size variability has implications for achieving cover cropping goals. Agricultural &amp; Environmental Letters, 7, e20080. 10.1002/ael2.20080
</cite> [<a href="https://doi.org/10.1002/ael2.20080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agricultural%20&amp;%20Environmental%20Letters&amp;title=Seed%20size%20variability%20has%20implications%20for%20achieving%20cover%20cropping%20goals&amp;author=N.%20P.%20Lounsbury&amp;author=N.%20D.%20Warren&amp;author=J.%20Hobbie&amp;author=H.%20Darby&amp;author=M.%20R.%20Ryan&amp;volume=7&amp;publication_year=2022&amp;pages=e20080&amp;doi=10.1002/ael2.20080&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0045">
<cite>

Madsen, M. D.
, 
Kristensen, P. S.
, 
Mahmood, K.
, 
Thach, T.
, 
Mohlfeld, M.
, 
Orabi, J.
, 
Sarup, P.
, 
Jahoor, A.
, 
Hovmøller, M. S.
, 
Rodriguez‐Algaba, J.
, &amp; 
Jensen, J.
 (2024). Scald resistance in hybrid rye (<em>Secale cereale</em>): Genomic prediction and GWAS. Frontiers in Plant Science, 15, 1306591.
</cite> [<a href="https://doi.org/10.3389/fpls.2024.1306591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10830712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38304738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Scald%20resistance%20in%20hybrid%20rye%20(Secale%20cereale):%20Genomic%20prediction%20and%20GWAS&amp;author=M.%20D.%20Madsen&amp;author=P.%20S.%20Kristensen&amp;author=K.%20Mahmood&amp;author=T.%20Thach&amp;author=M.%20Mohlfeld&amp;volume=15&amp;publication_year=2024&amp;pages=1306591&amp;pmid=38304738&amp;doi=10.3389/fpls.2024.1306591&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0046">
<cite>

Maraci, Ö.
, 
Özkan, H.
, &amp; 
Bilgin, R.
 (2018). Phylogeny and genetic structure in the genus <em>Secale</em>
. PLoS One, 13, e0200825. 10.1371/journal.pone.0200825

</cite> [<a href="https://doi.org/10.1371/journal.pone.0200825" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6053196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30024916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Phylogeny%20and%20genetic%20structure%20in%20the%20genus%20Secale&amp;author=%C3%96.%20Maraci&amp;author=H.%20%C3%96zkan&amp;author=R.%20Bilgin&amp;volume=13&amp;publication_year=2018&amp;pages=e0200825&amp;pmid=30024916&amp;doi=10.1371/journal.pone.0200825&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0047">
<cite>

Miedaner, T.
, 
Korzun, V.
, &amp; 
Bauer, E.
 (2019). Genomics‐based hybrid rye breeding. In  T.
Miedaner &amp;  V.
Korzun (Eds.), Applications of genetic and genomic research in cereals (pp. 329–348). Elsevier.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Applications%20of%20genetic%20and%20genomic%20research%20in%20cereals&amp;author=T.%20Miedaner&amp;author=V.%20Korzun&amp;author=E.%20Bauer&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0048">
<cite>

Miedaner, T.
, &amp; 
Laidig, F.
 (2019). Hybrid breeding in rye (<em>Secale cereale</em> L.). In Al‐Khayri J. M., Jain S. M., &amp; Johnson D. V. (Eds.), Advances in plant breeding strategies: Cereals (pp. 343–372). Springer International Publishing.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20plant%20breeding%20strategies:%20Cereals&amp;author=T.%20Miedaner&amp;author=F.%20Laidig&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0049">
<cite>

Monteiro, F.
, 
Vidigal, P.
, 
Barros, A. B.
, 
Monteiro, A.
, 
Oliveira, H. R.
, &amp; 
Viegas, W.
 (2016). Genetic distinctiveness of rye in situ accessions from Portugal unveils a new hotspot of unexplored genetic resources. Frontiers in Plant Science, 7, 1334. 10.3389/fpls.2016.01334

</cite> [<a href="https://doi.org/10.3389/fpls.2016.01334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5006150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27630658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genetic%20distinctiveness%20of%20rye%20in%20situ%20accessions%20from%20Portugal%20unveils%20a%20new%20hotspot%20of%20unexplored%20genetic%20resources&amp;author=F.%20Monteiro&amp;author=P.%20Vidigal&amp;author=A.%20B.%20Barros&amp;author=A.%20Monteiro&amp;author=H.%20R.%20Oliveira&amp;volume=7&amp;publication_year=2016&amp;pages=1334&amp;pmid=27630658&amp;doi=10.3389/fpls.2016.01334&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0050">
<cite>

Morey, D. D.
 (1970). Registration of Wrens Abruzzi rye (reg. no. 3). Crop Science, 10, 209–210. 10.2135/cropsci1970.0011183X001000020034x
</cite> [<a href="https://doi.org/10.2135/cropsci1970.0011183X001000020034x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Registration%20of%20Wrens%20Abruzzi%20rye%20(reg.%20no.%203)&amp;author=D.%20D.%20Morey&amp;volume=10&amp;publication_year=1970&amp;pages=209-210&amp;doi=10.2135/cropsci1970.0011183X001000020034x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0051"><cite>
NDSU Research Foundation
. (2020). 
ND Dylan winter rye (RFP‐282)
 (PVPO: 201800167). <a href="https://apps.ams.usda.gov/CMS/AdobeImages/201800167.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://apps.ams.usda.gov/CMS/AdobeImages/201800167.pdf</a>
</cite></li>
<li id="tpg270008-bib-0052">
<cite>

Newell, M. A.
, &amp; 
Butler, T. J.
 (2013). Forage rye improvement in the southern United States: A review. Crop Science, 53, 38–47. 10.2135/cropsci2012.05.0319
</cite> [<a href="https://doi.org/10.2135/cropsci2012.05.0319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Forage%20rye%20improvement%20in%20the%20southern%20United%20States:%20A%20review&amp;author=M.%20A.%20Newell&amp;author=T.%20J.%20Butler&amp;volume=53&amp;publication_year=2013&amp;pages=38-47&amp;doi=10.2135/cropsci2012.05.0319&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0053">
<cite>

Parat, F.
, 
Schwertfirm, G.
, 
Rudolph, U.
, 
Miedaner, T.
, 
Korzun, V.
, 
Bauer, E.
, 
Schön, C.‐C.
, &amp; 
Tellier, A.
 (2016). Geography and end use drive the diversification of worldwide winter rye populations. Molecular Ecology, 25, 500–514. 10.1111/mec.13495

</cite> [<a href="https://doi.org/10.1111/mec.13495" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26607414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Geography%20and%20end%20use%20drive%20the%20diversification%20of%20worldwide%20winter%20rye%20populations&amp;author=F.%20Parat&amp;author=G.%20Schwertfirm&amp;author=U.%20Rudolph&amp;author=T.%20Miedaner&amp;author=V.%20Korzun&amp;volume=25&amp;publication_year=2016&amp;pages=500-514&amp;pmid=26607414&amp;doi=10.1111/mec.13495&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0054"><cite>
Pennington Grain &amp; Seed, Inc
. (1978). 
<em>Wintergrazer‐70</em> (PVPO 72010). <a href="https://apps.ams.usda.gov/CMS/AdobeImages/007200010.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://apps.ams.usda.gov/CMS/AdobeImages/007200010.pdf</a>
</cite></li>
<li id="tpg270008-bib-0055">
<cite>

Pérez, P.
, &amp; 
de los Campos, G.
 (2014). Genome‐wide regression and prediction with the BGLR statistical package. Genetics, 198, 483–495. 10.1534/genetics.114.164442

</cite> [<a href="https://doi.org/10.1534/genetics.114.164442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4196607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25009151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genome%E2%80%90wide%20regression%20and%20prediction%20with%20the%20BGLR%20statistical%20package&amp;author=P.%20P%C3%A9rez&amp;author=G.%20de%20los%20Campos&amp;volume=198&amp;publication_year=2014&amp;pages=483-495&amp;pmid=25009151&amp;doi=10.1534/genetics.114.164442&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0056">
<cite>

Pfahler, P. L.
, 
Barnett, R. D.
, &amp; 
Luke, H. H.
 (1985). Florida 401 rye: An early forage type adapted to minimum tillage, multiple cropping systems. University of Florida Agriculture Experiment Stations.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Florida%20401%20rye:%20An%20early%20forage%20type%20adapted%20to%20minimum%20tillage,%20multiple%20cropping%20systems&amp;author=P.%20L.%20Pfahler&amp;author=R.%20D.%20Barnett&amp;author=H.%20H.%20Luke&amp;publication_year=1985&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0057">
<cite>

Qualset, C. O.
, &amp; 
Hoskinson, P. E.
 (1967). Registration of Balbo rye (reg. no. 2). Crop Science, 7, 81–81. 10.2135/cropsci1967.0011183X000700010035x
</cite> [<a href="https://doi.org/10.2135/cropsci1967.0011183X000700010035x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Registration%20of%20Balbo%20rye%20(reg.%20no.%202)&amp;author=C.%20O.%20Qualset&amp;author=P.%20E.%20Hoskinson&amp;volume=7&amp;publication_year=1967&amp;pages=81-81&amp;doi=10.2135/cropsci1967.0011183X000700010035x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0058">
<cite>
R Core Team
. (2016). 
R: A language and environment for statistical computing
. R Foundation for Statistical Computing.</cite> [<a href="https://scholar.google.com/scholar_lookup?R%20Core%20Team%20.%20(2016).%20R:%20A%20language%20and%20environment%20for%20statistical%20computing%20.%20R%20Foundation%20for%20Statistical%20Computing." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0059">
<cite>

Rabanus‐Wallace, M. T.
, 
Hackauf, B.
, 
Mascher, M.
, 
Lux, T.
, 
Wicker, T.
, 
Gundlach, H.
, 
Baez, M.
, 
Houben, A.
, 
Mayer, K. F. X.
, 
Guo, L.
, 
Poland, J.
, 
Pozniak, C. J.
, 
Walkowiak, S.
, 
Melonek, J.
, 
Praz, C. R.
, 
Schreiber, M.
, 
Budak, H.
, 
Heuberger, M.
, 
Steuernagel, B.
, … 
Stein, N.
 (2021). Chromosome‐scale genome assembly provides insights into rye biology, evolution and agronomic potential. Nature Genetics, 53, 564–573. 10.1038/s41588-021-00807-0

</cite> [<a href="https://doi.org/10.1038/s41588-021-00807-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8035072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33737754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Chromosome%E2%80%90scale%20genome%20assembly%20provides%20insights%20into%20rye%20biology,%20evolution%20and%20agronomic%20potential&amp;author=M.%20T.%20Rabanus%E2%80%90Wallace&amp;author=B.%20Hackauf&amp;author=M.%20Mascher&amp;author=T.%20Lux&amp;author=T.%20Wicker&amp;volume=53&amp;publication_year=2021&amp;pages=564-573&amp;pmid=33737754&amp;doi=10.1038/s41588-021-00807-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0060">
<cite>

Rakoczy‐Trojanowska, M.
, 
Bolibok‐Brągoszewska, H.
, 
Myśków, B.
, 
Dzięgielewska, M.
, 
Stojałowski, S.
, 
Grądzielewska, A.
, 
Boczkowska, M.
, &amp; 
Moskal, K.
 (2021). Genetics and genomics of stress tolerance. In Rabanus‐Wallace M. T. &amp; Stein N. (Eds.), The rye genome (pp. 213–251). Springer International Publishing.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20rye%20genome&amp;author=M.%20Rakoczy%E2%80%90Trojanowska&amp;author=H.%20Bolibok%E2%80%90Br%C4%85goszewska&amp;author=B.%20My%C5%9Bk%C3%B3w&amp;author=M.%20Dzi%C4%99gielewska&amp;author=S.%20Stoja%C5%82owski&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0061">
<cite>

Robinson, R. G.
 (1973). Registration of Rymin rye (reg. no. 6). Crop Science, 13, 775. 10.2135/cropsci1973.0011183X001300060075x
</cite> [<a href="https://doi.org/10.2135/cropsci1973.0011183X001300060075x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Registration%20of%20Rymin%20rye%20(reg.%20no.%206)&amp;author=R.%20G.%20Robinson&amp;volume=13&amp;publication_year=1973&amp;pages=775&amp;doi=10.2135/cropsci1973.0011183X001300060075x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0062">
<cite>

Roth, M.
, 
Muranty, H.
, 
Di Guardo, M.
, 
Guerra, W.
, 
Patocchi, A.
, &amp; 
Costa, F.
 (2020). Genomic prediction of fruit texture and training population optimization towards the application of genomic selection in apple. Horticulture Research, 7, 148. 10.1038/s41438-020-00370-5

</cite> [<a href="https://doi.org/10.1038/s41438-020-00370-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7459338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32922820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=Genomic%20prediction%20of%20fruit%20texture%20and%20training%20population%20optimization%20towards%20the%20application%20of%20genomic%20selection%20in%20apple&amp;author=M.%20Roth&amp;author=H.%20Muranty&amp;author=M.%20Di%20Guardo&amp;author=W.%20Guerra&amp;author=A.%20Patocchi&amp;volume=7&amp;publication_year=2020&amp;pages=148&amp;pmid=32922820&amp;doi=10.1038/s41438-020-00370-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0063">
<cite>

Schneider, C. A.
, 
Rasband, W. S.
, &amp; 
Eliceiri, K. W.
 (2012). NIH Image to ImageJ: 25 years of image analysis. Nature Methods, 9, 671–675. 10.1038/nmeth.2089

</cite> [<a href="https://doi.org/10.1038/nmeth.2089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5554542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=NIH%20Image%20to%20ImageJ:%2025%20years%20of%20image%20analysis&amp;author=C.%20A.%20Schneider&amp;author=W.%20S.%20Rasband&amp;author=K.%20W.%20Eliceiri&amp;volume=9&amp;publication_year=2012&amp;pages=671-675&amp;pmid=22930834&amp;doi=10.1038/nmeth.2089&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0064">
<cite>

Schreiber, M.
, 
Gao, Y.
, 
Koch, N.
, 
Fuchs, J.
, 
Heckmann, S.
, 
Himmelbach, A.
, 
Börner, A.
, 
Özkan, H.
, 
Maurer, A.
, 
Stein, N.
, 
Mascher, M.
, &amp; 
Dreissig, S.
 (2022). Recombination landscape divergence between populations is marked by larger low‐recombining regions in domesticated rye. Molecular Biology and Evolution, 39, msac131. 10.1093/molbev/msac131

</cite> [<a href="https://doi.org/10.1093/molbev/msac131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9218680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35687854/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=Recombination%20landscape%20divergence%20between%20populations%20is%20marked%20by%20larger%20low%E2%80%90recombining%20regions%20in%20domesticated%20rye&amp;author=M.%20Schreiber&amp;author=Y.%20Gao&amp;author=N.%20Koch&amp;author=J.%20Fuchs&amp;author=S.%20Heckmann&amp;volume=39&amp;publication_year=2022&amp;pages=msac131&amp;pmid=35687854&amp;doi=10.1093/molbev/msac131&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0065">
<cite>

Schreiber, M.
, 
Himmelbach, A.
, 
Börner, A.
, &amp; 
Mascher, M.
 (2019). Genetic diversity and relationship between domesticated rye and its wild relatives as revealed through genotyping‐by‐sequencing. Evolutionary Applications, 12, 66–77. 10.1111/eva.12624

</cite> [<a href="https://doi.org/10.1111/eva.12624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6304746/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30622636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolutionary%20Applications&amp;title=Genetic%20diversity%20and%20relationship%20between%20domesticated%20rye%20and%20its%20wild%20relatives%20as%20revealed%20through%20genotyping%E2%80%90by%E2%80%90sequencing&amp;author=M.%20Schreiber&amp;author=A.%20Himmelbach&amp;author=A.%20B%C3%B6rner&amp;author=M.%20Mascher&amp;volume=12&amp;publication_year=2019&amp;pages=66-77&amp;pmid=30622636&amp;doi=10.1111/eva.12624&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0066">
<cite>

Schuetzenmeister, A.
, &amp; 
Dufey, F.
 (2022). 
VCA: Variance component analysis
 (Version 1.4.5) [Computer software]. CRAN. <a href="https://CRAN.R-project.org/package=VCA" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R‐project.org/package=VCA</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Schuetzenmeister,%20A.%20,%20&amp;%20Dufey,%20F.%20(2022).%20VCA:%20Variance%20component%20analysis%20(Version%201.4.5)%20%5BComputer%20software%5D.%20CRAN.%20https://CRAN.R%E2%80%90project.org/package=VCA" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0067">
<cite>

Schwantes, I. A.
, 
do Amaral, A. T.
,  Jr., 
de Almeida Filho, J. E.
, 
Vivas, M.
, 
Silva Cabral, P. D.
, 
Gonçalves Guimarães, A.
, 
de Lima e Silva, F. H.
, 
Araújo Diniz Santos, P. H.
, 
Gonzaga Pereira, M.
, 
Pio Viana, A.
, 
Ferreira Pena, G.
, &amp; 
Alves Ferreira, F. R.
 (2020). Genomic selection helps accelerate popcorn population breeding. Crop Science, 60, 1373–1385. 10.1002/csc2.20112
</cite> [<a href="https://doi.org/10.1002/csc2.20112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genomic%20selection%20helps%20accelerate%20popcorn%20population%20breeding&amp;author=I.%20A.%20Schwantes&amp;author=A.%20T.%20do%20Amaral&amp;author=J.%20E.%20de%20Almeida%20Filho&amp;author=M.%20Vivas&amp;author=P.%20D.%20Silva%20Cabral&amp;volume=60&amp;publication_year=2020&amp;pages=1373-1385&amp;doi=10.1002/csc2.20112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0068">
<cite>

Sidhu, J. S.
, 
Ramakrishnan, S. M.
, 
Ali, S.
, 
Bernardo, A.
, 
Bai, G.
, 
Abdullah, S.
, 
Ayana, G.
, &amp; 
Sehgal, S. K.
 (2019). Assessing the genetic diversity and characterizing genomic regions conferring Tan Spot resistance in cultivated rye. PLoS One, 14, e0214519. 10.1371/journal.pone.0214519

</cite> [<a href="https://doi.org/10.1371/journal.pone.0214519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6438500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30921415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Assessing%20the%20genetic%20diversity%20and%20characterizing%20genomic%20regions%20conferring%20Tan%20Spot%20resistance%20in%20cultivated%20rye&amp;author=J.%20S.%20Sidhu&amp;author=S.%20M.%20Ramakrishnan&amp;author=S.%20Ali&amp;author=A.%20Bernardo&amp;author=G.%20Bai&amp;volume=14&amp;publication_year=2019&amp;pages=e0214519&amp;pmid=30921415&amp;doi=10.1371/journal.pone.0214519&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0069">
<cite>

Siekmann, D.
, 
Jansen, G.
, 
Zaar, A.
, 
Kilian, A.
, 
Fromme, F. J.
, &amp; 
Hackauf, B.
 (2021). A genome‐wide association study pinpoints quantitative trait genes for plant height, heading date, grain quality, and yield in rye (<em>Secale cereale</em> L.). Frontiers in Plant Science, 12, 23. 10.3389/fpls.2021.718081
</cite> [<a href="https://doi.org/10.3389/fpls.2021.718081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8586073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34777409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=A%20genome%E2%80%90wide%20association%20study%20pinpoints%20quantitative%20trait%20genes%20for%20plant%20height,%20heading%20date,%20grain%20quality,%20and%20yield%20in%20rye%20(Secale%20cereale%20L.)&amp;author=D.%20Siekmann&amp;author=G.%20Jansen&amp;author=A.%20Zaar&amp;author=A.%20Kilian&amp;author=F.%20J.%20Fromme&amp;volume=12&amp;publication_year=2021&amp;pages=23&amp;pmid=34777409&amp;doi=10.3389/fpls.2021.718081&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0070">
<cite>

Targońska‐Karasek, M.
, 
Boczkowska, M.
, 
Podyma, W.
, 
Pasnik, M.
, 
Niedzielski, M.
, 
Rucinska, A.
, 
Nowak‐Zyczynska, Z.
, &amp; 
Rakoczy‐Trojanowska, M.
 (2020). Investigation of obsolete diversity of rye (<em>Secale cereale</em> L.) using multiplexed SSR fingerprinting and evaluation of agronomic traits. Journal of Applied Genetics, 61, 513–529. 10.1007/s13353-020-00579-z

</cite> [<a href="https://doi.org/10.1007/s13353-020-00579-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7652744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32895738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Applied%20Genetics&amp;title=Investigation%20of%20obsolete%20diversity%20of%20rye%20(Secale%20cereale%20L.)%20using%20multiplexed%20SSR%20fingerprinting%20and%20evaluation%20of%20agronomic%20traits&amp;author=M.%20Targo%C5%84ska%E2%80%90Karasek&amp;author=M.%20Boczkowska&amp;author=W.%20Podyma&amp;author=M.%20Pasnik&amp;author=M.%20Niedzielski&amp;volume=61&amp;publication_year=2020&amp;pages=513-529&amp;pmid=32895738&amp;doi=10.1007/s13353-020-00579-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0071">
<cite>
USDA‐Natural Resources Conservation Service
. (2015). 
Release brochure for Aroostook cereal rye (Secale cereale L.)
. USDA, Big Flats Plant Materials Center.</cite> [<a href="https://scholar.google.com/scholar_lookup?USDA%E2%80%90Natural%20Resources%20Conservation%20Service%20.%20(2015).%20Release%20brochure%20for%20Aroostook%20cereal%20rye%20(Secale%20cereale%20L.)%20.%20USDA,%20Big%20Flats%20Plant%20Materials%20Center." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0072">
<cite>

van Rossum, B.‐J.
, &amp; 
Kruijer, W.
 (2022). 
Package ‘statgenGWAS’
 (Version 1) [Computer software]. CRAN. <a href="https://CRAN.R-project.org/package=statgenGWAS" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R‐project.org/package=statgenGWAS</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?van%20Rossum,%20B.%E2%80%90J.%20,%20&amp;%20Kruijer,%20W.%20(2022).%20Package%20%E2%80%98statgenGWAS%E2%80%99%20(Version%201)%20%5BComputer%20software%5D.%20CRAN.%20https://CRAN.R%E2%80%90project.org/package=statgenGWAS" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0073">
<cite>

Vendelbo, N. M.
, 
Mahmood, K.
, 
Sarup, P.
, 
Kristensen, P. S.
, 
Orabi, J.
, &amp; 
Jahoor, A.
 (2021). Discovery of a novel powdery mildew (<em>Blumeria graminis</em>) resistance locus in rye (<em>Secale cereale</em> L.). Scientific Reports, 11, Article 23057. 10.1038/s41598-021-02488-5

</cite> [<a href="https://doi.org/10.1038/s41598-021-02488-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8630102/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34845285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Discovery%20of%20a%20novel%20powdery%20mildew%20(Blumeria%20graminis)%20resistance%20locus%20in%20rye%20(Secale%20cereale%20L.)&amp;author=N.%20M.%20Vendelbo&amp;author=K.%20Mahmood&amp;author=P.%20Sarup&amp;author=P.%20S.%20Kristensen&amp;author=J.%20Orabi&amp;volume=11&amp;publication_year=2021&amp;pages=23057&amp;pmid=34845285&amp;doi=10.1038/s41598-021-02488-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0074">
<cite>

Webb, S. L.
, 
Mosali, J.
, 
Baker, J. L.
, &amp; 
Newell, M. A.
 (2013). Estimates of variance components and repeatability for total forage yield in rye: Implications for breeding. Plant Breeding, 132, 580–585. 10.1111/pbr.12115
</cite> [<a href="https://doi.org/10.1111/pbr.12115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding&amp;title=Estimates%20of%20variance%20components%20and%20repeatability%20for%20total%20forage%20yield%20in%20rye:%20Implications%20for%20breeding&amp;author=S.%20L.%20Webb&amp;author=J.%20Mosali&amp;author=J.%20L.%20Baker&amp;author=M.%20A.%20Newell&amp;volume=132&amp;publication_year=2013&amp;pages=580-585&amp;doi=10.1111/pbr.12115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0075">
<cite>

Wickham, H.
 (2016). Ggplot2: Elegant graphics for data analysis. Springer‐Verlag.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Ggplot2:%20Elegant%20graphics%20for%20data%20analysis&amp;author=H.%20Wickham&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0076">
<cite>

Woore, M. S.
, 
Flint‐Garcia, S. A.
, &amp; 
Holland, J. B.
 (2024). Phenotypic characterization of Southeastern United States open‐pollinated maize landraces. Crop Science, 64, 772–787. 10.1002/csc2.21198
</cite> [<a href="https://doi.org/10.1002/csc2.21198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Phenotypic%20characterization%20of%20Southeastern%20United%20States%20open%E2%80%90pollinated%20maize%20landraces&amp;author=M.%20S.%20Woore&amp;author=S.%20A.%20Flint%E2%80%90Garcia&amp;author=J.%20B.%20Holland&amp;volume=64&amp;publication_year=2024&amp;pages=772-787&amp;doi=10.1002/csc2.21198&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0077">
<cite>

Yabe, S.
, 
Ohsawa, R.
, &amp; 
Iwata, H.
 (2013). Potential of genomic selection for mass selection breeding in annual allogamous crops. Crop Science, 53, 95–105. 10.2135/cropsci2012.03.0167
</cite> [<a href="https://doi.org/10.2135/cropsci2012.03.0167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Potential%20of%20genomic%20selection%20for%20mass%20selection%20breeding%20in%20annual%20allogamous%20crops&amp;author=S.%20Yabe&amp;author=R.%20Ohsawa&amp;author=H.%20Iwata&amp;volume=53&amp;publication_year=2013&amp;pages=95-105&amp;doi=10.2135/cropsci2012.03.0167&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0078">
<cite>

Zhang, X.
, 
Sallam, A.
, 
Gao, L.
, 
Kantarski, T.
, 
Poland, J.
, 
DeHaan, L. R.
, 
Wyse, D. L.
, &amp; 
Anderson, J. A.
 (2016). Establishment and optimization of genomic selection to accelerate the domestication and improvement of intermediate wheatgrass. The Plant Genome, 9, plantgenome2015.07.0059. 10.3835/plantgenome2015.07.0059
</cite> [<a href="https://doi.org/10.3835/plantgenome2015.07.0059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27898759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Establishment%20and%20optimization%20of%20genomic%20selection%20to%20accelerate%20the%20domestication%20and%20improvement%20of%20intermediate%20wheatgrass&amp;author=X.%20Zhang&amp;author=A.%20Sallam&amp;author=L.%20Gao&amp;author=T.%20Kantarski&amp;author=J.%20Poland&amp;volume=9&amp;publication_year=2016&amp;pmid=27898759&amp;doi=10.3835/plantgenome2015.07.0059&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270008-bib-0079">
<cite>

Zhu, X.
, 
Leiser, W. L.
, 
Hahn, V.
, &amp; 
Würschum, T.
 (2021). Training set design in genomic prediction with multiple biparental families. The Plant Genome, 14(3), e20124. 10.1002/tpg2.20124

</cite> [<a href="https://doi.org/10.1002/tpg2.20124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34302722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Training%20set%20design%20in%20genomic%20prediction%20with%20multiple%20biparental%20families&amp;author=X.%20Zhu&amp;author=W.%20L.%20Leiser&amp;author=V.%20Hahn&amp;author=T.%20W%C3%BCrschum&amp;volume=14&amp;issue=3&amp;publication_year=2021&amp;pages=e20124&amp;pmid=34302722&amp;doi=10.1002/tpg2.20124&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Supporting information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11997422/bin/TPG2-18-e70008-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70008-s002.docx</a><sup> (15.3MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>Supporting data</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11997422/bin/TPG2-18-e70008-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70008-s001.xlsx</a><sup> (65.6KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Phenotypic data used in this study are available as Supplementary Data <a href="#tpg270008-supinfo-0001" class="usa-link">S1</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70008"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70008.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11997422/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11997422/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11997422%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11997422/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11997422/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11997422/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40230018/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11997422/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40230018/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11997422/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11997422/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="86CDUEmxku2VR0Hsxjbq1RkdwiEbMlzEBeK2rYgv6jHCmy713Uq4Ld0s3x3JcxVi">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
