
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            The epigenetics effects of transposable elements are genomic context dependent and not restricted to gene silencing in Drosophila - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53470C8AF4444305470C000D9D9E19.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="genbio">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360000/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Genome Biology">
<meta name="citation_title" content="The epigenetics effects of transposable elements are genomic context dependent and not restricted to gene silencing in Drosophila">
<meta name="citation_author" content="Marta Coronado-Zamora">
<meta name="citation_author_institution" content="Institut Botànic de Barcelona, CSIC, CMCNB, Barcelona, Spain">
<meta name="citation_author" content="Josefa González">
<meta name="citation_author_institution" content="Institut Botànic de Barcelona, CSIC, CMCNB, Barcelona, Spain">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="251">
<meta name="citation_doi" content="10.1186/s13059-025-03705-4">
<meta name="citation_pmid" content="40826096">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360000/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360000/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360000/pdf/13059_2025_Article_3705.pdf">
<meta name="description" content="Transposable elements (TEs) represent a threat to genome integrity due to their proliferative capacity. Eukaryotic cells silence TEs through different molecular mechanisms, including the deposition of repressive histone marks. Previous studies have ...">
<meta name="og:title" content="The epigenetics effects of transposable elements are genomic context dependent and not restricted to gene silencing in Drosophila">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Transposable elements (TEs) represent a threat to genome integrity due to their proliferative capacity. Eukaryotic cells silence TEs through different molecular mechanisms, including the deposition of repressive histone marks. Previous studies have ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360000/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12360000">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s13059-025-03705-4"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/13059_2025_Article_3705.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12360000%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12360000/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12360000/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360000/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-genbio.png" alt="Genome Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Genome Biology" title="Link to Genome Biology" shape="default" href="https://genomebiology.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Genome Biol</button></div>. 2025 Aug 18;26:251. doi: <a href="https://doi.org/10.1186/s13059-025-03705-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s13059-025-03705-4</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Genome%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Genome%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Genome%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Genome%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>The epigenetics effects of transposable elements are genomic context dependent and not restricted to gene silencing in <em>Drosophila</em>
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Coronado-Zamora%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Marta Coronado-Zamora</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Marta Coronado-Zamora</span></h3>
<div class="p">
<sup>1</sup>Institut Botànic de Barcelona, CSIC, CMCNB, Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Coronado-Zamora%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marta Coronado-Zamora</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gonz%C3%A1lez%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Josefa González</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Josefa González</span></h3>
<div class="p">
<sup>1</sup>Institut Botànic de Barcelona, CSIC, CMCNB, Barcelona, Spain </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gonz%C3%A1lez%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Josefa González</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Institut Botànic de Barcelona, CSIC, CMCNB, Barcelona, Spain </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jun 13; Accepted 2025 Jul 18; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12360000  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40826096/" class="usa-link">40826096</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par5">Transposable elements (TEs) represent a threat to genome integrity due to their proliferative capacity. Eukaryotic cells silence TEs through different molecular mechanisms, including the deposition of repressive histone marks. Previous studies have shown that TE repressive marks can spread to neighboring sequences. However, evidence for this spreading leading to nearby gene silencing remains limited. Similarly, whether TEs induce changes in the enrichment of active histone marks genome-wide, and the potential impact on gene expression have not been widely studied.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par6">In this work, we perform a comprehensive study of the epigenetic effects of 2235 TEs and their potential effects on nearby gene expression on <em>Drosophila melanogaster</em> head, gut, and ovary. While most TEs (816) induce the enrichment of the H3K9me3 repressive mark, with stronger epigenetic effects in the ovary, a substantial number (345 TEs) induce the enrichment of the H3K27ac active mark, particularly in the gut. We find that 70% of the H3K9me3 enriched TEs associated with expression changes downregulate the nearby gene, and 50% of the H3K27ac enriched TEs associated with expression changes lead to gene upregulation. These changes in expression affect specific regulatory networks only in the head. Furthermore, TE epigenetic effects on gene expression are genomic context dependent. Finally, we find that 221 TEs also affect gene expression by disrupting regions enriched for histone marks. </p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par7">Overall, our results show that TEs contribute to the generation of regulatory novelty through epigenetic changes, with these epigenetic effects not restricted to gene silencing and being genomic context dependent.
</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s13059-025-03705-4.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> H3K9me3, H3K27ac, Histone marks, <em>Drosophila</em>, Gene expression, Tissue specificity</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par8">The capacity of transposable elements (TEs) to move and expand within the genome is a threat to genome integrity [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. As a result, cells have evolved a range of molecular strategies, including epigenetic mechanisms, to repress and control TE activity [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. One epigenetic mechanism widespread among eukaryotes is the methylation of H3K9 (H3K9me3), a repressive mark associated with the formation of heterochromatin, that targets TEs to suppress their activity [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. While the enrichment of repressive epigenetic marks at TEs would limit the expression of TEs, which should be beneficial for the cell, several studies have shown that TE silencing could also influence nearby gene sequences across species, including plants [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>] and animals [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. A potential functional consequence of the spread of repressive marks is the downregulation of the expression of nearby genes. However, the evidence for this effect genome-wide is limited across species [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Several works have used <em>Drosophila</em> species to investigate the potential effect of TE repressive marks on nearby gene expression. In <em>D. albomicans</em>, TEs located nearby lowly expressed genes showed a higher H3K9me3 enrichment and spreading than those near highly expressed genes [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. However, the lack of systematic downregulation between genes with insertions and their orthologs, lacking the insertions, suggests that TEs preferentially accumulate near lowly expressed genes rather than affecting their expression. The few studies comparing the expression of homologous alleles with and without TE insertions with epigenetic effects found no clear association between enrichment of H3K9me3 and gene downregulation [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Lee and Karpen [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] found H3K9me3 enrichment around the flanking regions of TEs in <em>D. melanogaster</em>, but this enrichment was associated with both down and upregulation of nearby genes. Huang et al. [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>] extended the analysis to <em>D. simulans</em> and <em>D. yakuba</em> and concluded that TEs did not have a negative impact on nearby gene expression. Finally, Fablet et al. [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>] analyzed the association of TE insertion numbers with variability in gene expression and histone enrichment. They found limited evidence for an association between H3K9me3 enrichment and TE insertion numbers, and no negative impact of TEs on the expression of nearby genes. </p>
<p id="Par9">While TE enrichment in H3K9me3 has been widely studied, less is known about the enrichment of active histone marks in TE sequences. Some studies, mainly using human data, have found evidence of TEs inserted in active chromatin-state regions suggesting that these TEs could be acting as <em>cis</em>-regulatory elements [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. However, the majority of analyses available are based on single genomes and could not discern whether TEs induced the enrichment of active marks or were inserted into regions that were already enriched for those marks. Indeed, the few studies comparing alleles with and without TE insertions found that at least some TE families preferentially insert into genomic regions enriched for histone marks [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>–<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. In <em>D. melanogaster</em>, there is evidence for an individual TE insertion that, under oxidative stress conditions, recruits an active epigenetic mark, inducing changes in the expression of adjacent genes [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. So far, there is only one genome-wide study of TEs enriched for an active histone mark (H3K4me3) that found that the number of TE insertions was not associated with the level of this active mark in gene regions [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Therefore, our understanding of how TEs contribute to the genome-wide enrichment of active marks is still very limited.
</p>
<p id="Par10">Finally, whether TE insertions disrupt the epigenetic landscape of the regions where they insert has not been widely explored. There is evidence in <em>D. melanogaster</em> for the depletion of active marks in the regions flanking TE insertions [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Depletion of the repressive mark H3K9me2 was also found specifically for <em>roo</em> family insertions [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. However, whether this pattern is TE-induced, and whether it impacts gene expression has not been investigated yet.</p>
<p id="Par11">In this work, we performed a thorough study of the epigenetic effects of 2235 polymorphic TEs and their potential effects on nearby gene expression using ChIP-seq data, for a repressive (H3K9me3) and an active histone mark (the enhancer-related H3K27ac mark [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]), and transcriptomic data obtained from head, gut, and ovary of five <em>D. melanogaster</em> strains. The five strains, collected from diverse eco-geographical locations, have high-quality de novo reference genomes and TE annotations available, which allowed us to systematically characterize the epigenetic effect of TEs [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. We analyzed (i) whether TE insertions induced epigenetic changes: both enrichment and depletion of repressive and active histone marks; (ii) whether the epigenetic effect of these TEs was body part-specific; (iii) whether these effects were associated to particular TE families; and (iv) whether TEs with epigenetic effects are associated with gene expression changes and their potential functional implications.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">TEs induce the enrichment of active and repressive histone marks differentially across body parts</h3>
<p id="Par12">To gain insight into the epigenetic effects of TEs across adult body parts, we took advantage of the high-quality de novo TE annotations and ChIP-seq data, available for the repressive mark H3K9me3 and for the enhancer-related mark H3K27ac, from three biological replicates in the head, gut, and ovary of five <em>D. melanogaster</em> European strains [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Four of them are inbred strains that were collected in Finland (AKA-017), Denmark (JUT-011), Germany (MUN-016), and Spain (TOM-007), and the one collected in Serbia (SLA-001) is an isofemale strain (see Methods). We first analyzed the average epigenetic states of 4823 euchromatic TE flanking sequences (± 20 kb) across the five genomes and the three body parts (see Methods). In the adult head, we observed an average depletion of the repressive mark H3K9me3 (Fig. <a href="#Fig1" class="usa-link">1</a>A and Additional file 1: Table S1A). In the gut and ovary, there is a strong average enrichment of H3K9me3 in the TE flanking sequences, as previously described in <em>D. melanogaster</em> embryos [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] (Fig. <a href="#Fig1" class="usa-link">1</a>A). Regarding the active mark, all three body parts showed a depletion of H3K27ac, supporting previous observations that active histone modifications are depleted near TE insertions genome-wide [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] (Fig. <a href="#Fig1" class="usa-link">1</a>A and Additional file 1: Table S1A). However, by analyzing the average epigenetic effects of all TEs annotated in the genomes studied, we cannot discard that these patterns are due to the preferential insertion of TEs in regions enriched or depleted for active and repressive histone marks [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>–<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. </p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360000_13059_2025_3705_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e73/12360000/a19e3ae37194/13059_2025_3705_Fig1_HTML.jpg" loading="lazy" id="MO1" height="372" width="709" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Epigenetic states in the ± 20 kb TE flanking regions across body parts. <strong>A</strong> Median H3K9me3 and H3K27ac fold enrichment of all TEs annotated in the five genomes analyzed (<em>n</em> = 4823 TEs). Head (blue line), gut (orange line), ovary (red line). <strong>B</strong> Median H3K9me3 and H2K27ac fold enrichment of polymorphic TEs (<em>n</em> = 2325 TEs). The red line represents histone mark enrichment of genomes with a TE [TE(+)] and the gray line represents histone mark enrichment of genomes without a TE [TE(−)]. The median was calculated for H3K9me3 and H3K27ac fold enrichment across all sequences flanking the analyzed TEs in the five genomes. Plots were generated using LOESS smoothing (span = 10%)</p></figcaption></figure><p id="Par13">To test whether the observed enrichment and depletion of histone marks is TE-induced, we focused on polymorphic TEs, for which we can compare the epigenetic states of the TE flanking regions in genomes with and without the TE insertions. We performed this analysis for 2325 polymorphic TEs (see Methods). In the head, we did not observe differences between the genomes with and without TEs, suggesting that TEs are not driving the previously observed depletion in the H3K9me3 mark (Fig. <a href="#Fig1" class="usa-link">1</a>B and Additional file 1: Table S1B). On the other hand, TE insertions were associated with H3K9me3 enrichment in gut and ovary (6.4% and 16.6% enrichment in the flanking ± 1 kb window, respectively) (Fig. <a href="#Fig1" class="usa-link">1</a>B). We hypothesized that the lower levels of H3K9me3 observed in the head could be explained by the level of expression of methyltransferases (<em>Su(var)3–9</em> and <em>eggless</em>) and demethylases (<em>Kdm3</em> and <em>Kdm4B</em>) in this tissue. We found that the expression level of <em>Su(var)3–9</em>, a key heterochromatin regulator of H3K9me3, was lower in the head compared to gut and ovary (Wilcoxon test, <em>p</em> values &lt; 0.001 for both comparisons, Additional file 2: Fig. S1), while for <em>eggless</em>, a histone H3 N-methyltransferase, the expression was lower compared to ovary (Wilcoxon test, <em>p</em> value &lt; 0.001) but not to gut. Indeed, the expression level of Su(var) proteins has been previously associated with the magnitude of TE’s epigenetic effects in <em>Drosophila</em> [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Also consistent with our hypothesis, we found that the expression level of the <em>Kdm3</em> and <em>Kdm4B</em> demethylases were higher in the head compared to gut and ovary (Wilcoxon test, <em>p</em> values &lt; 0.05 for all comparisons, for <em>Kdm4B</em>, the comparison was only significant for head vs. gut a (Additional file 2: Fig. S1).</p>
<p id="Par14">Regarding the active mark H3K27ac, we observed a slight depletion in genomes with TEs in head (− 1.4%), an enrichment in gut (2.3%), and no enrichment in ovary (Fig. <a href="#Fig1" class="usa-link">1</a>B and Additional file 1: Table S1B). This contrasts with the previous observation of a general depletion of the active mark across body parts (Fig. <a href="#Fig1" class="usa-link">1</a>A). Note that the enrichment pattern observed in gut is the one expected for active promoters, where the histone mark enrichment flanks rather than directly overlap with the enhancer sequences [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>, <a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] (Fig. <a href="#Fig1" class="usa-link">1</a>B).</p>
<p id="Par15">Although the enrichment and depletion patterns described above (Fig. <a href="#Fig1" class="usa-link">1</a>B) were only found in the genomes that contain the TE insertions, thus suggesting that they are TE-induced, to discard that these patterns could be due to strain-specific epigenetic variation, we checked whether similar patterns were found in genomic regions that do not contain TE insertions (see Methods). We did not find consistent patterns of enrichment or depletion in the TE-free genomic regions analyzed (Additional file 2: Fig. S2) suggesting that the previously observed patterns (Fig. <a href="#Fig1" class="usa-link">1</a>B) are most likely TE-induced.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Individual TEs are mostly enrich for repressive marks in the ovary and for active marks in the gut and depleted similarly across body parts</h3>
<p id="Par16">While overall, polymorphic TEs induce the enrichment and depletion of repressive and active marks across body parts (Fig. <a href="#Fig1" class="usa-link">1</a>B), the analysis of individual TE insertions allows us to gain further insight into both the percentage of enrichment and depletion and the spread of these effects to nearby regions. We considered an individual TE to have an epigenetic effect if the fold enrichment of H3K9me3, H3K27ac, or both marks (bivalent) was significantly higher or lower within ± 1 kb of the TE insertion (see Methods; [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]). For these TEs, we also computed: (i) the percentage change in enrichment or depletion compared to strains without the TE within the ± 1 kb window and (ii) the extent of spread from the TE insertion (up to ± 20 kb) with consecutive significant changes in histone marks. We computed the same metrics for our dataset of TE-free regions to show that these effects are induced by TEs and not due to epigenetic variability present in the strains (Additional file 1: Table S2A).</p>
<p id="Par17">We found that most of the polymorphic TEs analyzed exert epigenetic effects in at least one of the body parts analyzed (71.5%, 1597/2235; Supplemental Data and Additional file 1: Table S2B, C). Among the TEs with epigenetic effects, 65.4% (1045/1597) induced the enrichment of one or both histone marks simultaneously (bivalent), 25% (400/1597) induced the depletion of one or both histone marks, and 9.6% (152/1597) induced the enrichment of histone marks in some body parts but the depletion in others. If we focus on the TEs that induce the enrichment of histone marks in at least one body part, we found that the most prevalent and strongest effect is the enrichment of H3K9me3: 816 out of 1197 TEs (Table <a href="#Tab1" class="usa-link">1</a> and Fig. <a href="#Fig2" class="usa-link">2</a>A). The majority of these TEs exert their effect on the ovary (576/816), where the average percentage of enrichment and the spread of the histone mark is higher than in other body parts: 119.83% average enrichment (± 100.39) and 3.76 kb spread (± 3.5) (Wilcoxon’s test, <em>p</em> values &lt; 0.001, Fig. <a href="#Fig2" class="usa-link">2</a>B, <a href="#Fig2" class="usa-link">C</a>, Table <a href="#Tab1" class="usa-link">1</a> and Additional file 1: Table S2D). Head, in contrast, is the body part with less TEs inducing the enrichment of H3K9me3 compared to gut and ovary (140 TEs, χ<sup>2</sup> test, <em>p</em> values &lt; 0.001). Although we observed an average enrichment of H3K27ac only in the gut (Fig. <a href="#Fig1" class="usa-link">1</a>B), we found that TEs also enrich for the active mark in other body parts (Fig. <a href="#Fig2" class="usa-link">2</a>A). From 345 TEs that induce the enrichment of the active mark, half of them exert their effect in the gut (183/345), which is a percentage significantly higher than the TEs found in the ovary (112 TEs, χ<sup>2</sup> test, <em>p</em> value &lt; 0.001) and the head (88 TEs χ<sup>2</sup> test, <em>p</em> value &lt; 0.001) (Fig. <a href="#Fig2" class="usa-link">2</a>A). The percentage of enrichment and spread is higher in the gut: 37.16% (± 21.97) and 2.92 kb (± 2.95), respectively (Wilcoxon’s test, <em>p</em> values &lt; 0.05, Fig. <a href="#Fig2" class="usa-link">2</a>B, <a href="#Fig2" class="usa-link">C</a>, Table <a href="#Tab1" class="usa-link">1</a> and Additional file 1: Table S2D). We found that the 183 TE insertions that enrich for H3K27ac in the gut were enriched for full-length copies (90 full-length vs. 93 fragments) compared to the TE insertions not enriched for H3K27ac in the gut (719 TEs: 264 full-length and 457 fragments; Fisher’s exact test, <em>p</em> value &lt; 0.01), suggesting that some of these insertions might be active. We also identified 304 TEs that induced the enrichment of both marks simultaneously (bivalent) (Fig. <a href="#Fig2" class="usa-link">2</a>A). This effect is observed in similar proportions in gut (147 TEs) and ovary (133 TEs) (χ<sup>2</sup> test, <em>p</em> value = 0.29) with stronger effects in the ovary than in the gut (Wilcoxon’s test, <em>p</em> values &lt; 0.05; Fig. <a href="#Fig2" class="usa-link">2</a>B, <a href="#Fig2" class="usa-link">C</a>, Table <a href="#Tab1" class="usa-link">1</a> and Additional file 1: Table S2D). </p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Epigenetic effects of the 1597 polymorphic TEs analyzed in this study</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Effect</th>
<th align="left" colspan="1" rowspan="1">Histone</th>
<th align="left" colspan="1" rowspan="1">Body part</th>
<th align="left" colspan="1" rowspan="1">Number of TEs</th>
<th align="left" colspan="1" rowspan="1">Average spread in kb (SD)</th>
<th align="left" colspan="1" rowspan="1">Average enrichment in % (SD)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="12" colspan="1">Enrichment</td>
<td align="left" colspan="1" rowspan="1">H3K9me3</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">816</td>
<td align="left" colspan="1" rowspan="1">3.25 (± 3.22)</td>
<td align="left" colspan="1" rowspan="1">86.01 (± 87.83)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K27ac</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">345</td>
<td align="left" colspan="1" rowspan="1">2.78 (± 2.95)</td>
<td align="left" colspan="1" rowspan="1">36.28 (± 99.14)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bivalent</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">304</td>
<td align="left" colspan="1" rowspan="1">3.12 (± 3.41)</td>
<td align="left" colspan="1" rowspan="1">54.56 (± 54.7)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K9me3</td>
<td align="left" colspan="1" rowspan="1">Gut</td>
<td align="left" colspan="1" rowspan="1">305</td>
<td align="left" colspan="1" rowspan="1">2.83 (± 2.94)</td>
<td align="left" colspan="1" rowspan="1">42.8 (± 24.97)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K9me3</td>
<td align="left" colspan="1" rowspan="1">Head</td>
<td align="left" colspan="1" rowspan="1">140</td>
<td align="left" colspan="1" rowspan="1">1.88 (± 1.48)</td>
<td align="left" colspan="1" rowspan="1">29.15 (± 16.87)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K9me3</td>
<td align="left" colspan="1" rowspan="1">Ovary</td>
<td align="left" colspan="1" rowspan="1">576</td>
<td align="left" colspan="1" rowspan="1">3.76 (± 3.5)</td>
<td align="left" colspan="1" rowspan="1">119.83 (± 100.39)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K27ac</td>
<td align="left" colspan="1" rowspan="1">Gut</td>
<td align="left" colspan="1" rowspan="1">183</td>
<td align="left" colspan="1" rowspan="1">2.92 (± 2.95)</td>
<td align="left" colspan="1" rowspan="1">37.19 (± 21.98)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K27ac</td>
<td align="left" colspan="1" rowspan="1">Head</td>
<td align="left" colspan="1" rowspan="1">88</td>
<td align="left" colspan="1" rowspan="1">2.34 (± 2.34)</td>
<td align="left" colspan="1" rowspan="1">36.74 (± 201.93)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K27ac</td>
<td align="left" colspan="1" rowspan="1">Ovary</td>
<td align="left" colspan="1" rowspan="1">112</td>
<td align="left" colspan="1" rowspan="1">2.87 (± 3.34)</td>
<td align="left" colspan="1" rowspan="1">34.42 (± 38.8)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bivalent</td>
<td align="left" colspan="1" rowspan="1">Gut</td>
<td align="left" colspan="1" rowspan="1">147</td>
<td align="left" colspan="1" rowspan="1">3.15 (± 3.19)</td>
<td align="left" colspan="1" rowspan="1">44.04 (± 34.95)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bivalent</td>
<td align="left" colspan="1" rowspan="1">Head</td>
<td align="left" colspan="1" rowspan="1">45</td>
<td align="left" colspan="1" rowspan="1">1.58 (± 1.22)</td>
<td align="left" colspan="1" rowspan="1">33.19 (± 18.56)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bivalent</td>
<td align="left" colspan="1" rowspan="1">Ovary</td>
<td align="left" colspan="1" rowspan="1">133</td>
<td align="left" colspan="1" rowspan="1">3.59 (± 3.94)</td>
<td align="left" colspan="1" rowspan="1">73.23 (± 72.17)</td>
</tr>
<tr>
<td align="left" rowspan="12" colspan="1">Depletion</td>
<td align="left" colspan="1" rowspan="1">H3K9me3</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">200</td>
<td align="left" colspan="1" rowspan="1">1.92 (± 1.83)</td>
<td align="left" colspan="1" rowspan="1"> − 24.06 (± 13.05)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K27ac</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">223</td>
<td align="left" colspan="1" rowspan="1">2.73 (± 3.38)</td>
<td align="left" colspan="1" rowspan="1"> − 24.21 (± 12.17)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bivalent</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">178</td>
<td align="left" colspan="1" rowspan="1">7.32 (± 6.91)</td>
<td align="left" colspan="1" rowspan="1"> − 64.55 (± 33.13)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K9me3</td>
<td align="left" colspan="1" rowspan="1">Gut</td>
<td align="left" colspan="1" rowspan="1">52</td>
<td align="left" colspan="1" rowspan="1">1.6 (± 1.24)</td>
<td align="left" colspan="1" rowspan="1"> − 24.28 (± 14.13)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K9me3</td>
<td align="left" colspan="1" rowspan="1">Head</td>
<td align="left" colspan="1" rowspan="1">87</td>
<td align="left" colspan="1" rowspan="1">1.62 (± 1.21)</td>
<td align="left" colspan="1" rowspan="1"> − 21.77 (± 11.01)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K9me3</td>
<td align="left" colspan="1" rowspan="1">Ovary</td>
<td align="left" colspan="1" rowspan="1">86</td>
<td align="left" colspan="1" rowspan="1">2.42 (± 2.44)</td>
<td align="left" colspan="1" rowspan="1"> − 26.67 (± 11.54)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K27ac</td>
<td align="left" colspan="1" rowspan="1">Gut</td>
<td align="left" colspan="1" rowspan="1">104</td>
<td align="left" colspan="1" rowspan="1">2.55 (± 2.92)</td>
<td align="left" colspan="1" rowspan="1"> − 25.34 (± 12.73)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K27ac</td>
<td align="left" colspan="1" rowspan="1">Head</td>
<td align="left" colspan="1" rowspan="1">73</td>
<td align="left" colspan="1" rowspan="1">2.13 (± 2.57)</td>
<td align="left" colspan="1" rowspan="1"> − 20.63 (± 11.84)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K27ac</td>
<td align="left" colspan="1" rowspan="1">Ovary</td>
<td align="left" colspan="1" rowspan="1">68</td>
<td align="left" colspan="1" rowspan="1">3.66 (± 4.44)</td>
<td align="left" colspan="1" rowspan="1"> − 25.66 (± 14.13)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bivalent</td>
<td align="left" colspan="1" rowspan="1">Gut</td>
<td align="left" colspan="1" rowspan="1">111</td>
<td align="left" colspan="1" rowspan="1">7.66 (± 6.97)</td>
<td align="left" colspan="1" rowspan="1"> − 71.04 (± 33.03)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bivalent</td>
<td align="left" colspan="1" rowspan="1">Head</td>
<td align="left" colspan="1" rowspan="1">111</td>
<td align="left" colspan="1" rowspan="1">7.95 (± 6.97)</td>
<td align="left" colspan="1" rowspan="1"> − 73.27 (± 32.27)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bivalent</td>
<td align="left" colspan="1" rowspan="1">Ovary</td>
<td align="left" colspan="1" rowspan="1">133</td>
<td align="left" colspan="1" rowspan="1">7.81 (± 6.85)</td>
<td align="left" colspan="1" rowspan="1"> − 63.87 (± 33.26)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360000_13059_2025_3705_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e73/12360000/faf4481d950b/13059_2025_3705_Fig2_HTML.jpg" loading="lazy" id="MO2" height="504" width="710" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Epigenetics effects of the 1597 polymorphic TEs analyzed across body parts. <strong>A</strong> 1197 TEs induce the enrichment of epigenetic marks. Dark colors indicate body part-specific TEs and light colors indicate TEs that are not body part-specific. <strong>B</strong> Mean percentage of increase in H3K9me3, H3K27ac, and bivalent enrichment of the TEs across body parts. Outliers (TEs with epigenetic effects &gt; 500%) are not plotted for clarity. <strong>C</strong> Mean extent of spread (kb) of H3K9me3, H3K27ac, and bivalent enrichment by body parts. The spread was calculated in windows of up to 20 kb adjacent to the TE insertion. <strong>D</strong> 552 TEs inducing the depletion of epigenetic marks. Dark colors indicate body part-specific TEs and light colors indicate TEs that are not body part-specific. <strong>E</strong> Mean percentage of decrease in H3K9me3, H3K27ac, and bivalent enrichment of the TEs by body parts. <strong>F</strong> Mean extent of spread (kb) of H3K9me3, H3K27ac, and bivalent depletion of the TEs by body parts. Boxplot colors in <strong>B</strong>, <strong>C</strong>, <strong>E</strong>, and <strong>F</strong> panels indicate the body part: ovary (red), gut (orange), and head (blue). Values and statistics related to this figure can be found in Table <a href="#Tab1" class="usa-link">1</a> and Additional file 1: Table S2</p></figcaption></figure><p id="Par19">In contrast to histone enrichment, which likely results from histone recruitment dependent on specific sequences within TEs, depletion is likely due to the insertional disruption of these sequences in the genome. Therefore, we hypothesized that TEs would similarly deplete H3K9me3 and H3K27ac across the three body parts. As expected, we found that TEs depleted H3K9me3 and H3K27ac in similar proportions (χ<sup>2</sup> test, <em>p</em> value = 0.17; Fig. <a href="#Fig2" class="usa-link">2</a>D, Table <a href="#Tab1" class="usa-link">1</a> and Additional file 1: Table S2B, C) and that histone mark depletion was similar across body parts (χ<sup>2</sup> tests, <em>p</em> values &gt; 0.1, Table <a href="#Tab1" class="usa-link">1</a> and Additional file 1: Table S2E). Notably, the bivalent depletion is the one with the highest effects and longest spread: 9.13 kb (± 7.05) and − 78.1% (± 31.23), respectively (Wilcoxon’s test, <em>p</em> values &lt; 0.001; Fig. <a href="#Fig2" class="usa-link">2</a>E, <a href="#Fig2" class="usa-link">F</a> and Additional file 1: Table S2D).</p>
<p id="Par20">Finally, we found that TEs inducing the enrichment and depletion of epigenetic marks are highly body part-specific: 757 out of the 1197 TEs that enrich and 376 out of 552 TEs that deplete exert their epigenetic effect in only one body part (Fig. <a href="#Fig2" class="usa-link">2</a>A and <a href="#Fig2" class="usa-link">D</a> and Additional file 1: Table S2B). Focusing on TEs that induce histone mark enrichment, the most body part-specific effects were the enrichment of H3K27ac and bivalent marks (308/345 and 285/304, respectively). In the case of TEs inducing depletion, the most body part-specific TEs are those depleting for H3K27ac and H3K9me3 (177/200 and 201/223, respectively) (Fig. <a href="#Fig2" class="usa-link">2</a>D and Additional file 1: Table S2B).</p>
<p id="Par21">Overall, the majority of individual TE insertions induced the enrichment of H3K9me3 in the ovary and the enrichment of H3K27ac in the gut, although we also found enrichment of these histone marks in other body parts. Moreover, the epigenetic effects of TEs are not restricted to the enrichment of histone marks, as we also identified 552 TEs that depleted histone marks.</p></section><section id="Sec5"><h3 class="pmc_sec_title">TE families differed in the proportion of copies enriched for H3K9me3 and H3K27ac</h3>
<p id="Par22">TE families vary in many properties, including transposition mechanisms, lengths, genome distribution, density, and the targeting by small RNAs [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>–<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Variation in the proportion of TEs with epigenetic effects, spread, and enrichment of H3K9me3 has already been reported across families [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. We thus tested whether we could find variation in the epigenetic effects of TEs depending on their family identity. We focused on families with &gt; 20 genome copies, and we found that TE families that showed a larger proportion of copies associated with the enrichment of H3K9me3 and with the enrichment of the two marks were all LTRs: <em>blood</em>, <em>mdg3</em>, <em>Blastopia</em>, and <em>412</em> (Fig. <a href="#Fig3" class="usa-link">3</a>A and <a href="#Fig3" class="usa-link">C</a> and Additional file 1: Table S3A). <em>Pogo</em> and <em>H</em> elements were the only families with significantly fewer proportion of insertions associated with H3K9me3 enrichment (Fig. <a href="#Fig3" class="usa-link">3</a>A). On the other hand, a LINE family, <em>Doc</em>, was the only family that showed a significantly larger proportion of copies enriched for H3K27ac (Fig. <a href="#Fig3" class="usa-link">3</a>B). In the case of TEs associated with histone mark depletion, we do not expect them to be enriched for particular families as the effect is not dependent on the TE itself but rather on whether the TE inserts in a sequence that recruits chromatin modifiers. Indeed, we only found one family with a larger proportion of copies depleted for H3K27ac: <em>FB4</em> (Additional file 1: Table S3B).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360000_13059_2025_3705_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e73/12360000/5c648ad3e999/13059_2025_3705_Fig3_HTML.jpg" loading="lazy" id="MO3" height="470" width="709" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Proportion of TEs per family enriched for epigenetic marks. <strong>A</strong> Proportion of copies per family that are enriched for H3K9me3 epigenetic mark. <strong>B</strong> Proportion of copies per family that are enriched for H3K27ac epigenetic mark. <strong>C</strong> Proportion of copies per family enriched for bivalent epigenetic histone marks. Red: larger proportion of copies associated with enrichment of histone mark than expected, blue: fewer proportion of copies associated with enrichment of histone mark than expected, gray: non-significant. Only families with a minimum of 8 copies with epigenetic effects and more than 20 copies in the genome were considered. Numbers in Additional file 1: Table S3</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Sixty percent of the TEs that induce epigenetic changes have an effect on the expression of adjacent genes</h3>
<p id="Par23">To assess whether the epigenetic effect of TEs has an impact on the expression of nearby genes, we analyzed matching RNA-seq data available for all the strains and body parts [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Out of the 1597 TEs that have epigenetic effects, 1168 TEs had an expressed gene located within the spread of the histone mark enrichment/depletion. We calculated the <em>z</em>-score of the gene expression change between strains with and without the TE insertion. A negative <em>z</em>-score indicates that the expression level of the gene adjacent to the TE is lower than the gene expression of the same gene when the TE is not present (see Methods). We found that 703 TEs (60%) have a significant impact on gene expression (728 genes): 389 TEs were associated with reduced gene expression, 231 with increased gene expression, and 83 were associated with increased gene expression in one body part and reduced expression in another (Table <a href="#Tab2" class="usa-link">2</a>). We found that 70% (238/340) of the TEs that are enriched for the repressive mark, and associated with gene expression changes, are indeed associated with the downregulation of the nearby gene (Fig. <a href="#Fig4" class="usa-link">4</a>A, <a href="#Fig4" class="usa-link">B</a>, Table <a href="#Tab2" class="usa-link">2</a> and Additional file 1: Table S4A). A permutation test showed that while for some strains and tissues, the association of a TE with the downregulation of the nearby gene is likely due to the presence of the repressive histone marks, in other cases we cannot discard that the effect on expression is due to the presence of the TE itself (Additional file 1: Table S4B). We further tested whether the effect on gene downregulation of TEs enriched for the repressive mark is significant compared with TE-free regions that are also enriched for H3K9me3. We found that this was the case for the majority of strains and body parts (Additional file 1: Table S4C).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>TEs with epigenetic effects associated with gene expression changes. FC: fold change, of the difference in expression of genes in strains with and without the TE insertion</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Effect</th>
<th align="left" colspan="1" rowspan="1">Histone mark</th>
<th align="left" colspan="1" rowspan="1">TEs</th>
<th align="left" colspan="1" rowspan="1">TEs inducing a significant change in expression</th>
<th align="left" colspan="1" rowspan="1">TEs that induce downregulation (range log<sub>2</sub>FC and mean)</th>
<th align="left" colspan="1" rowspan="1">TEs that induce upregulation (range log<sub>2</sub>FC and mean)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="3" colspan="1">Enrichment</td>
<td align="left" colspan="1" rowspan="1">H3K9me3</td>
<td align="left" colspan="1" rowspan="1">606</td>
<td align="left" colspan="1" rowspan="1">340</td>
<td align="left" colspan="1" rowspan="1">238 (− 0.08 to − 79.7, − 1.5)</td>
<td align="center" colspan="1" rowspan="1">135 (0.08–13.7, 1.5)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K27ac</td>
<td align="left" colspan="1" rowspan="1">244</td>
<td align="left" colspan="1" rowspan="1">102</td>
<td align="left" colspan="1" rowspan="1">55 (− 0.14 to − 79.72, − 2.46)</td>
<td align="center" colspan="1" rowspan="1">51 (0.12–5.7, 1.2)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bivalent</td>
<td align="left" colspan="1" rowspan="1">216</td>
<td align="left" colspan="1" rowspan="1">112</td>
<td align="left" colspan="1" rowspan="1">78 (− 0.14 to − 79.72, − 2.1)</td>
<td align="center" colspan="1" rowspan="1">43 (0.078–13.7, 1.6)</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">Depletion</td>
<td align="left" colspan="1" rowspan="1">H3K9me3</td>
<td align="left" colspan="1" rowspan="1">149</td>
<td align="left" colspan="1" rowspan="1">70</td>
<td align="left" colspan="1" rowspan="1">34 (− 0.12 to − 79.72, − 3.23)</td>
<td align="center" colspan="1" rowspan="1">37 (0.11–2.2, 0.79)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3K27ac</td>
<td align="left" colspan="1" rowspan="1">171</td>
<td align="left" colspan="1" rowspan="1">101</td>
<td align="left" colspan="1" rowspan="1">68 (− 0.081 to − 79.72, − 2.2)</td>
<td align="center" colspan="1" rowspan="1">41 (0.13–5.8, 1.1)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bivalent</td>
<td align="left" colspan="1" rowspan="1">106</td>
<td align="left" colspan="1" rowspan="1">59</td>
<td align="left" colspan="1" rowspan="1">44 (− 0.1 to − 79.72, − 2.1)</td>
<td align="center" colspan="1" rowspan="1">24 (0.18–2.9, 1.1)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360000_13059_2025_3705_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e73/12360000/a1313a9605a7/13059_2025_3705_Fig4_HTML.jpg" loading="lazy" id="MO4" height="937" width="709" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>TEs with epigenetic effects are associated with gene expression changes. <strong>A</strong> H3K9me3, H3K27ac, and bivalent enrichment and expression <em>z</em>-score for genes with and without nearby TE insertions. The <em>z</em>-score for gene (TE +) vs. gene (TE −) expression (x-axis) and mean percentage of increase of the histone mark enrichment (y-axis) is plotted for TE-gene pairs. Green numbers indicate TE-gene pairs with the expected association between the epigenetic mark and the expression level, red numbers indicate TEs when the association is the opposite, and gray when there are no expectations. <strong>B</strong> Snapshot from DrosOmics browser for a TE that is enriched for H3K9me3 and associated with gene downregulation [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. The <em>Myo31DF</em> gene region in two strains is shown: TOM-007 in the top (with a <em>Quasimodo</em> TE insertion in the promoter region of the gene, in purple) and JUT-11 in the bottom (without the TE insertion). There is an enrichment of H3K9me3 in the flanking regions of the insertion that is not observed when the TE is absent, and the level of expression of <em>Myo31DF</em> is lower in TOM-007 (with the insertion). The brown track represents the ChIP-seq fold-change signal of H3K9me3, and the turquoise track the RNA-seq coverage levels. <strong>C</strong> Snapshot from DrosOmics browser for a TE enriched in H3K27ac and associated with gene upregulation [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. The <em>Bin1</em> and <em>sra</em> gene regions in two strains are shown: TOM-007 in the top (with an <em>Invader4</em> TE insertion in the promoter region of <em>Bin1</em> and within the first intron of <em>sra</em>) and JUT-011 in the bottom (without the TE insertion). There is a H3K27ac peak flanking the TE that is not present when the TE is absent, and the level of expression of <em>Bin1</em> is higher in the genome of TOM-007 (with the insertion). As <strong>B</strong>, the brown track represents the ChIP-seq fold-change signal of H3K27ac and the turquoise track the RNA-seq coverage levels. Statistical analyses and expression data are included in Additional file 1: Table S4A</p></figcaption></figure><p id="Par25">On the other hand, 50% (51/102) of the TEs that induced the enrichment of the active mark are associated with gene upregulation (Fig. <a href="#Fig4" class="usa-link">4</a>A and <a href="#Fig4" class="usa-link">C</a>, Table <a href="#Tab2" class="usa-link">2</a> and Additional file 1: Table S4A). Overall, the effect of TEs enriched for active histone marks on gene upregulation is not significant compared with TE-free regions enriched for the same active histone mark (Additional file 1: Table S4C). Finally, the depletion of the repressive mark was associated with gene upregulation in 52% of the TEs (37/70) while the depletion of the active mark with gene downregulation in 67% of the TEs (68/101). Regarding TEs that either induce bivalent enrichment or depletion, they are mostly associated with gene downregulation (70% 78/112, and 75% 44/59, respectively).</p>
<p id="Par26">The position of the TE relative to the gene (upstream, downstream, UTRs, CDS, or intron) was not associated with the direction of the expression change observed for H3K9me3 or H3K27ac enrichment and depletion (Additional file 1: Table S4D). Head and ovary were the body parts with more genes being significantly up or downregulated due to the epigenetic effects of TEs compared to gut (head: 47% 228/482, gut: 41% 305/746, and ovary: 49% 427/879, χ<sup>2</sup> test <em>p</em> value = 0.031 and 0.002, respectively).</p>
<p id="Par27">Finally, we did not find differences on the histone mark spread, percentage of enrichment, TE location regarding the nearby gene (upstream, downstream, exonic, and intronic), TE length distribution, and TE frequencies, between TEs with epigenetic effects associated with significant changes in expression and TEs with epigenetic effects that do not affect expression (Additional file 1: Table S4E). This result suggests that the epigenetic effect of a TE, or lack thereof, is not explained only by the TE itself but also depends on the genomic context where it is inserted [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>].</p></section><section id="Sec7"><h3 class="pmc_sec_title">Twelve of the TEs affecting gene expression through epigenetic changes are likely to be adaptive</h3>
<p id="Par28">It has been hypothesized that the epigenetic silencing of TEs could have a deleterious effect through the downregulation of nearby genes [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Indeed, we found that TEs that are enriched for the repressive histone mark and are associated with gene downregulation are enriched for low frequent insertions, as expected if these TEs have deleterious effects [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] (Fisher’s exact test, <em>p</em> value &lt; 0.001, Additional file 1: Table S4F). Note that the low frequency of these insertions could also be indicative of recent TE activity [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. On the other hand, nine of these TEs are present at high frequencies (&gt; 10%) suggesting that they might be adaptive, with one of these TEs present at a higher frequency compared to other insertions from the same family and another two TEs previously identified as candidate adaptive TEs (Additional file 1: Table S4G). These nine TEs are located nearby 11 genes related to <em>transport/localization</em> (5/11), <em>cell organization/biogenesis</em> (4/11), and <em>cell cycle proliferation</em> (3/11) (Additional file 1: Table S4G). Additionally, one TE (<em>2R_14883304_14883308_BS2</em>) is associated with three cytochrome P450 genes (<em>Cyp6a20</em>, <em>Cyp6a21</em>, and <em>Cyp6a9</em>) which are associated to insecticide resistance in <em>Drosophila</em> [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>].</p>
<p id="Par29">Two of the 51 TEs that are enriched for the active histone mark and associated with gene upregulation are also present at high frequencies (&gt; 10%; Additional file 1: Table S4G). The three genes associated with gene upregulation, <em>Bin1</em>, <em>sra</em>, and <em>Svil</em>, are all related to <em>cell organization/biogenesis</em>. One TE (<em>FBti0019386</em>) affecting both <em>Bin1</em> and <em>sra</em> was already described to be associated with gene upregulation in cold-stress conditions and in response to infection in <em>D. melanogaster</em> (Fig. <a href="#Fig4" class="usa-link">4</a>C, [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]). The other TE, associated with <em>Svil</em> upregulation, has also been previously identified as a candidate adaptive TE (Additional file 1: Table S4G).</p>
<p id="Par30">If we focus on TEs inducing depletion of histone marks, for the 37 TEs depleting H3K9me3 that induce gene upregulation, seven are present at high frequencies (&gt; 10%), with one TE showing a significantly higher frequency compared with all other insertions in that family. Five of these seven TEs are located nearby genes enriched for <em>cell organization/biogenesis</em> processes and four have previous evidence for an adaptive role (Additional file 1: Table S4G). <em>FBti0019170</em>, affecting the <em>kuzbanian</em> gene, has already been associated with adaptation in response to zinc-stress tolerance [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>, <a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] (Additional file 1: Table S4G). While <em>FBti0019386</em> was associated with H3K27ac in the gut in two of the strains analyzed, we found that this TE is also associated with H3K9me3 depletion in the gut in another strain, further suggesting that the epigenetic effect of TEs is context dependent.</p>
<p id="Par31">Among the 68 TEs inducing the depletion of the active mark associated with gene downregulation, there are 6 TEs present at high frequencies (&gt; 10%), associated with only two genes with a known function: one gene is a small nuclear RNA and the other one is a rho GTPase activating protein (Additional file 1: Table S4G). Three of these TEs were previously identified as candidate adaptive TEs (Additional file 1: Table S4G).</p>
<p id="Par32">Overall, we found that 12 of the TEs with epigenetic effects and associated with gene expression changes are present at higher population frequencies compared to other insertions in the same family, suggesting that they have increased in frequency due to positive selection rather than genetic drift, or have previous evidence for an adaptive role (Additional file 1: Table S4G).</p></section><section id="Sec8"><h3 class="pmc_sec_title">Genes regulated by TEs with epigenetic effects in head have functions associated with catabolic process and signal transduction</h3>
<p id="Par33">To analyze whether TEs with epigenetic effects are regulating genes with similar functions, potentially indicative of a rewiring of specific gene regulatory networks, we performed a Gene Ontology (GO) enrichment analysis [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. We found that genes upregulated in the head were enriched for catabolic processes and signal transduction biological processes (e.g., <em>negative regulation of cell communication, response to stimulus, and signaling</em>; Additional file 1: Table S5). In the gut and in the ovary, the latter being the body part with more genes associated with TEs, genes located nearby TEs with epigenetic effects were not enriched for any GO terms (Additional file 1: Table S5).</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par34">In this work, we have assessed the epigenetic effects of 2235 polymorphic TEs across three <em>D. melanogaster</em> adult body parts (Fig. <a href="#Fig1" class="usa-link">1</a>B). The availability of high-quality TE genome annotations and ChIP-seq data for three body parts and five genomes has allowed us to determine that 1197 of the TEs analyzed induced the enrichment of repressive or active histone marks (or both) rather than being inserted in regions already enriched for these marks (Fig. <a href="#Fig2" class="usa-link">2</a> and Additional file 2: Fig. S2). Including RNA-seq data in the analysis has also allowed us to investigate the functional consequences of TE insertions on gene expression. In contrast to previous studies [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>], we have found evidence for an effect of TEs on neighboring gene expression. Overall, we observed that 70% (238/340) of the TEs that were enriched for H3K9me3, and associated with gene expression changes, lead to the downregulation of the genes located within the spread of this histone mark (Fig. <a href="#Fig4" class="usa-link">4</a>A and Table <a href="#Tab2" class="usa-link">2</a>). Previous evidence for an association between H3K9me3 enrichment and gene downregulation in <em>D. melanogaster</em> was limited [reviewed in 1,14]. These studies either restricted their analyses to reference TE insertions, annotated in a single genome [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>], or used RNA-seq and ChIP-seq data generated for whole animals reflecting the average epigenetic states of multiple cell types [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. In contrast, we have analyzed de novo TE annotations in five genomes, together with ChIP-seq and RNA-seq obtained from three adult body parts of the same individuals, which allows to establish a significant association between TEs enriched for repressive histone mark and downregulation of nearby genes. Our results highlight the importance of multi-omic integrative approaches analyzing several reference genomes to get a more comprehensive understanding of the functional consequences of TEs [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>].</p>
<p id="Par35">Although smaller, a substantial proportion of the TEs enriched for H3K27ac that affect nearby gene expression were associated with the expected gene expression change: gene upregulation (51/102). While the enrichment of H3K9me3 occurs through small-RNA mediated mechanisms [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>], the capacity of TEs to enrich for active marks has been suggested to be associated with the recruitment of chromatin modifiers by TEs [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>].</p>
<p id="Par36">Furthermore, we also showed that the epigenetic effects of TEs extend beyond histone mark enrichment as 552 of the analyzed TEs induce the depletion of histone marks in at least one body part. The percentage of TEs disrupting H3K9me3 and H3K27ac marks and the number of TEs depleting these histone marks across body parts was similar (Table <a href="#Tab1" class="usa-link">1</a>). This pattern is not unexpected as the disruption of histone marks by TEs is due to the insertion of these sequences and not to host control mechanisms (see below). TEs disrupting epigenetic marks were also associated with gene expression changes. We found that 67% of the TEs inducing the depletion of the active mark were associated with gene downregulation, and that 52% associated with the depletion of H3K9me3 were associated with gene upregulation. Overall, both for TEs that enrich and for TEs that deplete histone marks, the percentage of TEs associated with the expected change of expression was high (Fig. <a href="#Fig4" class="usa-link">4</a>A). While some TEs were associated with the opposite pattern of expression, it is known that the association between epigenetic marks and gene expression is complex (e.g., [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]). Indeed, previous evidence for a correlation between H3K27ac enrichment and gene upregulation is weak [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>, <a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>], which is consistent with results suggesting that besides H3K27ac enrichment enhancer activity requires additional acetylation events on other histone lysine residues [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>].</p>
<p id="Par37">By analyzing expression data on three distinct body parts—head, gut, and ovary—we found that the epigenetic effects of TEs are body part-dependent. Overall, we have observed that the most prevalent epigenetic effect is the enrichment of H3K9me3 in ovaries (576 TEs), driven by the piRNA pathway [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. This is not unexpected, as TE insertions in the female germ line can compromise the genome integrity and the fitness of the offspring [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>, <a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. On the contrary, the head body part exhibited fewer TEs enriched for repressive marks (140 TEs), which could be explained by the lower expression of <em>Su(var)3–9</em> gene, which encodes a key heterochromatin regulator of H3K9me3 (Additional file 2: Fig. S1). The lower enrichment of TEs for H3K9me3 could be linked to a potentially higher rate of TE mobilization in the brain, as has been observed for <em>Drosophila</em> neurons [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>] and in the human brain ([<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>–<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]; but see also [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]). Thus, while our data suggest a lower repression of TEs in the head, more research is needed to elucidate whether this leads to an increased mobilization of TEs in the brain. We have also found that in the gut, the prevalent epigenetic effect of TEs is the enrichment of the active mark H3K27ac. TEs have previously been reported to be enriched in transcriptionally active chromatin (rich in acetylated histone H3) in the <em>Drosophila</em> midgut [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. Indeed, 5 of the 41 families enriched for H3K27ac in the gut were also found to be active by Siudeja et al. [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. During viral infections, the reverse transcriptase of certain TEs is used to generate copies of the invading virus, amplifying the immune response against viral infections [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>, <a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. A higher expression of TEs in the gut, induced by H3K27ac, might thus be seen as a trade-off between TE activity and response to viral infections occurring in the gut [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>].</p></section><section id="Sec10"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par38">Overall, our study provides evidence that the epigenetic effect of TEs goes beyond gene silencing and shows that TEs can also affect gene expression by disrupting the epigenetic marks of the regions where they insert. In the ovary, TEs are primarily enriched for the repressive H3K9me3 mark, probably reflecting the need to protect the germ line, while in the head, TEs affect genes involved in specific biological functions, suggesting a potential role in rewiring gene regulatory networks. By comparing several TE-related metrics between TEs with epigenetic effects that do and do not affect gene expression (i.e., histone mark spread, percentage of enrichment, TE position, TE length distribution, and TE frequencies), we concluded that the epigenetic consequences of TE insertions are genomic context dependent. Furthermore, 12 TEs with epigenetic effects that were associated with a change in gene expression were present at significantly higher population frequencies or were previously linked to an adaptive phenotypic effect or reported in genome-wide screenings for adaptive signals (Additional file 1: Table S4G). Because the majority of TEs inducing epigenetic changes were found at low population frequencies, the analysis of more genomes, and thus of an even more complete set of TEs present in natural populations, should help elucidate the overall role of TEs in generating gene regulatory novelty by inducing or disrupting epigenetic states.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Strain genomes and TE annotations</h3>
<p id="Par39">We have used available de novo reference genomes of five European <em>D. melanogaster</em> natural strains: AKA-017, JUT-011, MUN-016, SLA-001, and TOM-007 [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. For these genomes, there are high-quality annotations of transposable elements [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. In brief, we analyzed the 4823 TEs annotated in euchromatic region that are non-nested (Additional file 1: Table S6). Genomes and TE annotations are available for visualization at the DrosOmics genome browser [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>].</p></section><section id="Sec13"><h3 class="pmc_sec_title">TE breakpoints calls</h3>
<p id="Par40">To determine the TE insertion location (breakpoint) in genomes lacking a TE insertion, we focused on polymorphic TEs, a total of 3878 insertions. We extracted the ± 500 bp flanking regions of each TE using <em>BEDOPS</em> (v2.4.39, [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]) and <em>BEDtools</em> (v2.30.0, [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]) and used <em>minimap2</em> (with parameters <em>-ax splice -G100k -B2 -t 4 -C5</em>) (v2.17, [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]) to map such flanking regions in the genome lacking the TE. We called a TE breakpoint location when the mapping of the upstream, downstream, and concatenated flanking region was unique, allowing a gap/overlap of ± 50 bp between the upstream and downstream regions. We could determine the insertion location for 2325 (60%) of the TEs analyzed. The ± 20 kb flanking region of each TE insertion was divided into 20 non-overlapping 1 kb windows respectively to perform ChIP-seq enrichment analyses.</p></section><section id="Sec14"><h3 class="pmc_sec_title">RNA-seq and ChIP-seq data for H3K9me3 and H3K27ac in three body parts</h3>
<p id="Par41">RNA-seq data and ChIP-seq data for H3K9me3 and H3K27ac in three body parts for five strains were obtained from Coronado-Zamora et al. [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>], where a full description of the protocols used to generate these data can be obtained. Briefly, head, gut, and ovary body parts of each strain were dissected at the same time from the same individual flies. For each body part and strain, three replicates of 30 females aged 4–6 days were processed. RNA-seq library preparation was performed using the TruSeq Stranded mRNA sample prep kit from Illumina and sequenced using Illumina 125 bp paired-end reads. ChIP-seq libraries were performed using TruSeq ChIP library preparation kit. Sequencing was carried out in an Illumina HiSeq 2500 platform, producing 50 bp single-end reads. Raw RNA-seq and ChIP-seq data are available in the NCBI Sequence Read Archive (SRA) database (BioProject PRJNA643665, [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]) and can be retrieved and visualized through DrosOmics genome browser [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>].</p></section><section id="Sec15"><h3 class="pmc_sec_title">ChIP-seq data processing</h3>
<p id="Par42">ChIP-seq reads were processed using <em>fastp</em> (v.0.20.1, [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]) to remove adaptors and low-quality sequences. Processed reads were mapped to the corresponding de novo reference genome using the <em>readAllocate</em> function (<em>chipThres</em> = <em>500</em>) of the <em>Perm-seq</em> R package (v0.3.0, [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]), with <em>Bowtie</em> (v1.2.2, [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]) as the aligner and the Csem program (v2.3, [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]) in order to try to define a single location to multi-mapping reads. In all cases, <em>Bowtie</em> was performed with default parameters selected by <em>Perm-seq</em>.</p>
<p id="Par43">Then, we used the ENCODE ChIP-Seq caper pipeline (v2, [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]) in histone mode, using <em>Bowtie 2</em> as the aligner [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>], disabling pseudo replicate generation and all related analyses <em>(chip.true_rep_only</em> = <em>TRUE</em>) and pooling controls (i.e., ChIP-seq input, <em>chip.always_use_pooled_ctl</em> = <em>TRUE</em>). The MACS2 peak caller was used with default settings [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. The signal value tracks produced by MACS2 were processed as follows. We used <em>bigWigToBedGraph</em> from UCSC (available at <a href="https://genome.ucsc.edu/goldenPath/help/bigWig.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://genome.ucsc.edu/goldenPath/help/bigWig.html</a>) to transform bigWig files to BED files. Then, for each replicate, we used <em>bedmap</em> with the <em>mean</em> option (v2.4.39, [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]) to obtain the average signal values in 10 bp windows. Finally, the three replicates were averaged using <em>bedmap mean</em> to obtain a single track value for each sample that was used to estimate the spread and enrichment of histone marks adjacent to TE insertions. The enrichment signal by MACS for the two histone marks in each body part and strain is shown in Additional file 2: Fig. S3. As expected, H3K9me3 is enriched in heterochromatic regions (near pericentromeric regions), while H3K27ac is enriched in euchromatic regions and depleted in pericentromeric regions. However, two strains (AKA-017 and JUT-011) showed a more noisy H3K9me3 coverage and slightly different H3K27ac coverage in the head body part. To ensure that enrichment patterns were not affected by these two strains, we repeated Fig. <a href="#Fig1" class="usa-link">1</a>A excluding these two strains. The overall patterns remained consistent (Additional file 2: Fig. S4). Finally, to assess heterogeneity across strains, the same figure (Fig. <a href="#Fig1" class="usa-link">1</a>A) was plotted for individual strains. Overall, the patterns are consistent across strains, with H3K27ac showing depletion across the three body parts and H3K9me3 showing enrichment in the gut and ovary, while both marks are depleted in the head (Additional file 2: Fig. S5).</p></section><section id="Sec16"><h3 class="pmc_sec_title">ChIP-seq data analysis</h3>
<p id="Par44">From the 2325 polymorphic TEs, we computed two statistics when comparing pairs of TE(+) and TE(−) genomes [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]: the <em>percentage of increase or depletion of H3K9me3/H3K27ac enrichment</em> and the <em>extent of H3K9me3/H3K27ac spread</em>. We used Wilcoxon’s test (with the <em>wilcox.test</em> function from the <em>stats</em> package in R, v3.6.3, [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]) to assess if the H3K9me3/H3K27ac enrichment or depletion in the <em>i</em>th upstream and downstream windows differs significantly between pairs of TE(+) vs. TE(−) genomes. The most distant windows considered are 20 kb from TE insertions. The <em>percentage increase or depletion of H3K9me3/H3K27ac enrichment</em> is the difference of median H3K9me3 enrichment between two genomes in the ± 1 kb windows immediately next to the TE insertion, divided by the enrichment level for the strain without TE. The <em>extent of H3K9me3/H3K27ac spread</em> is the farthest window in which the H3K9me3/H3K27ac enrichment or depletion is consecutively and significantly higher or lower in the genome with TE compared with the genome without the TE. When the farthest windows are different between the left and right sides of a TE insertion, we used the window closer to the TE for the <em>extent of H3K9me3/H3K27ac spread</em> (to be conservative). To consider a TE for further analyses, we allowed only one different comparison across pairs of TE(+)-TE(−) genomes (that could be due to inconsistencies in the epigenetic change, no statistical enrichment or NA data, Additional file 2: Fig. S6). We excluded 49 TEs that did not fulfill the criteria in any of the body parts. From the remaining 2276 TEs, 1597 have epigenetic effects in at least one body part and at least for one histone mark. Six hundred thirty-eight TEs do not have an effect (no enrichment) in any of the body parts and 41 have a “mix” effect (i.e., enrich for one mark and deplete for the other) and were not analyzed. Therefore, the final dataset analyzed was of 2235 TEs (Additional file 2: Fig. S7).</p></section><section id="Sec17"><h3 class="pmc_sec_title">Analysis of histone enrichment patterns in TE-free regions</h3>
<p id="Par45">For every strain-specific TE insertion (2171 TEs), we selected an euchromatic region without a TE insertion annotated within 1 kb. Note that some of the TE insertions have more than one body part-dependent effect or affect more than one gene, we thus selected as many TE-free regions as TE-associated regions affected. In total, we analyzed 5088 TE-free regions. For these 5088 TE-free regions, we plotted the median H3K9me3 and H3K27ac in ± 20 kb regions across body parts and genomes.</p></section><section id="Sec18"><h3 class="pmc_sec_title">RNA-seq data processing to obtain gene expression levels</h3>
<p id="Par46">To obtain gene expression levels from RNA-seq, we used a reference-guided transcriptome assembly obtained for each strain following Pertea et al. [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. A detailed description can be found in Coronado-Zamora et al. [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. To obtain the gene count estimations normalized for TMM (trimmed mean of the M-values), we first used the <em>prepDE.py</em> script from StringTie [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>] to obtain a matrix of gene counts, with parameter <em>-l 125</em>. Next, we used the <em>edgeR</em> R package (v3.28.0, [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]) to create a <em>DGEList</em> object from the table of counts using the <em>DGEList</em> function. Next, we used <em>calcNormFactors</em> function with the TMM method to calculate the normalization factors to scale the raw library sizes. Finally, we used the function <em>cpm</em> to obtain the normalized counts using the TMM normalization factors.</p></section><section id="Sec19"><h3 class="pmc_sec_title">
<em>Z</em>-score calculation</h3>
<p id="Par47">We assigned each of the TEs with epigenetic effect to the gene(s) overlapping or adjacent, taking into account the spread of the histone mark. For each pair of TE-gene, a <em>z</em>-score was calculated as the mean of gene expression of gene with TE (averaging the three available replicates) minus the mean expression of gene without the TE, divided by the standard error of both groups of genes. A negative <em>z</em>-score indicates that the gene associated with the TE has lower expression compared to the gene without a TE insertion, while a positive <em>z</em>-score indicates that the gene associated with the TE has a higher expression.</p></section><section id="Sec20"><h3 class="pmc_sec_title">TE positional effect vs. TE insertion with epigenetic effects on gene expression</h3>
<p id="Par48">To determine whether TEs have an impact on gene expression because of their epigenetic effects and not by their presence alone (positional effect), we analyzed all strain-specific TE insertions located within 1 kb of genes. Using a permutation test (sampling = 1000), we compared the observed effects of TEs with epigenetic effects and associated with gene downregulation to random sets of TEs of equal size drawn from all the insertions annotated in the genome and matched by their location relative to genes (intron, 5′/3′ UTRs, CDS, and upstream/downstream regions). This approach allowed us to assess whether expression changes linked to TEs with epigenetic effects are significantly greater than would be expected from the presence of the insertion alone (Additional file 1: Table S4B).</p></section><section id="Sec21"><h3 class="pmc_sec_title">Epigenetic effects on gene expression in TE-free vs. TE-associated regions</h3>
<p id="Par49">To evaluate whether TEs with epigenetic effects have a stronger influence on gene expression than TE-free random genomic regions, we first identified TE-free regions located nearby genes (within 1 kb) that did not overlap any annotated TE (with a minimum distance of 1 kb from any TE). These regions were selected to match the genomic contexts where TEs were previously found to exert epigenetic effects (introns, 5′ and 3′ UTRs, CDS, and upstream/downstream regions, Additional file 1: Table S4C). For each genome strain, we selected as many TE-free null regions as possible within each category. For each body part and histone mark analyzed, we focused on TE-free regions enriched for that mark in one strain relative to the others, and recorded the number of associated genes showing up or downregulation. We then conducted permutation tests (sampling = 1000) to determine whether the number of differentially expressed genes in these TE-free regions differed significantly from the number observed for TE-associated regions (Additional file 1: Table S4C).</p></section><section id="Sec22"><h3 class="pmc_sec_title">Gene ontology (GO) enrichment analysis</h3>
<p id="Par50">To perform the GO enrichment analysis, we used the software GOWINDA (v.1.12, [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]) using the “gene” mode and the “updownstream20000” gene-definition and only considering the genes associated with TEs with epigenetic effects (parameters: –simulations 100,000, –min-genes 2, –min-significance 1). We applied a FDR &lt; 0.05.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Analysis of the proportion of TE copies with epigenetic effects by family</h3>
<p id="Par51">For the TE family analysis, we considered TE families with a minimum of 20 copies in the genome and with at least 8 copies with epigenetic effects to perform the statistical analysis. We considered a TE family as significant when the number of copies exhibiting epigenetic effects was significantly higher than the respective average. Specifically, we used the averages of 36.5% (816/2235), 15.4% (345/2235), and 13.6% (304/2235) as the thresholds for H3K9me3, H3K27ac, and bivalent enrichment, respectively. To test the significance of the enrichment, we applied a χ<sup>2</sup> test (using the <em>chisq.test</em> function from the <em>stats</em> package in R (v3.6.3, [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]). </p></section></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12360000/bin/13059_2025_3705_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3705_MOESM1_ESM.zip</a><sup> (1.4MB, zip) </sup><p>Additional file 1: Supplementary tables.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12360000/bin/13059_2025_3705_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3705_MOESM2_ESM.docx</a><sup> (2.7MB, docx) </sup><p>Additional file 2: Supplementary table legends and supplementary figures.</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Grace Yuh Chwen Lee for discussions and suggestions on how to perform some of the analyses.</p></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contribution</h2>
<p>JG conceived the project; JG and MCZ designed the analyses; MCZ performed the analyses; JG and MCZ wrote and revised the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This project has received funding from the European Research Council (ERC) under the European Union’s Horizon 2020 research and innovation programme (H2020-ERC-2014-CoG-647900), from grant PID2020-115874GB-I00 funded by MCIN/AEI 10.13039/501100011033, and from grant 2021 SGR 00417 funded by Departament de Recerca i Universitats, Generalitat de Catalunya awarded to J.G.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data Availability</h2>
<p>All scripts and R Markdown documents to reproduce analyses and figures have been deposited in GitHub: <a href="https://github.com/GonzalezLab/epigenetic-effects-transposons-dmelanogaster" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/GonzalezLab/epigenetic-effects-transposons-dmelanogaster</a> [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>] and are also available in Zenodo (10.5281/zenodo.15876976). Raw RNA-seq and ChIP-seq data for five D. melanogaster strains (AKA-017, JUT-011, MUN-016, SLA-001 and TOM-007, [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]) are available in the NCBI Sequence Read Archive (SRA) database (BioProject PRJNA643665) [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>].</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par52">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par53">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par54">Tim Sands was the primary editor of this article and managed its editorial process and peer review in collaboration with the rest of the editorial team. The peer-review history is available in the online version of this article.</p></section><section id="FPar4"><h3 class="pmc_sec_title">Competing interests
</h3>
<p id="Par55">The authors declare no competing interests.
</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Kelleher ES, Barbash DA, Blumenstiel JP. Taming the turmoil within: new insights on the containment of transposable elements. Trends Genet. 2020;36:474–89.
</cite> [<a href="https://doi.org/10.1016/j.tig.2020.04.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7347376/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32473745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kelleher%20ES,%20Barbash%20DA,%20Blumenstiel%20JP.%20Taming%20the%20turmoil%20within:%20new%20insights%20on%20the%20containment%20of%20transposable%20elements.%20Trends%20Genet.%202020;36:474%E2%80%9389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Liu P, Cuerda-Gil D, Shahid S, Slotkin RK. The epigenetic control of the transposable element life cycle in plant genomes and beyond. Annu Rev Genet. 2022;56:63–87.
</cite> [<a href="https://doi.org/10.1146/annurev-genet-072920-015534" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36449356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20P,%20Cuerda-Gil%20D,%20Shahid%20S,%20Slotkin%20RK.%20The%20epigenetic%20control%20of%20the%20transposable%20element%20life%20cycle%20in%20plant%20genomes%20and%20beyond.%20Annu%20Rev%20Genet.%202022;56:63%E2%80%9387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Almeida MV, Vernaz G, Putman ALK, Miska EA. Taming transposable elements in vertebrates: from epigenetic silencing to domestication. Trends Genet. 2022;38:529–53.
</cite> [<a href="https://doi.org/10.1016/j.tig.2022.02.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35307201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Almeida%20MV,%20Vernaz%20G,%20Putman%20ALK,%20Miska%20EA.%20Taming%20transposable%20elements%20in%20vertebrates:%20from%20epigenetic%20silencing%20to%20domestication.%20Trends%20Genet.%202022;38:529%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Lawlor MA, Ellison CE. Evolutionary dynamics between transposable elements and their host genomes: mechanisms of suppression and escape. Curr Opin Genet Dev. 2023;82: 102092.
</cite> [<a href="https://doi.org/10.1016/j.gde.2023.102092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10530431/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37517354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lawlor%20MA,%20Ellison%20CE.%20Evolutionary%20dynamics%20between%20transposable%20elements%20and%20their%20host%20genomes:%20mechanisms%20of%20suppression%20and%20escape.%20Curr%20Opin%20Genet%20Dev.%202023;82:%20102092." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Yin H, Sweeney S, Raha D, Snyder M, Lin H. A high-resolution whole-genome map of key chromatin modifications in the adult Drosophila melanogaster. PLoS Genet. 2011;7: e1002380.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1002380" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3240582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22194694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yin%20H,%20Sweeney%20S,%20Raha%20D,%20Snyder%20M,%20Lin%20H.%20A%20high-resolution%20whole-genome%20map%20of%20key%20chromatin%20modifications%20in%20the%20adult%20Drosophila%20melanogaster.%20PLoS%20Genet.%202011;7:%20e1002380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Lee YCG. The role of piRNA-mediated epigenetic silencing in the population dynamics of transposable elements in Drosophila melanogaster. PLoS Genet. 2015;11: e1005269.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4456100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26042931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20YCG.%20The%20role%20of%20piRNA-mediated%20epigenetic%20silencing%20in%20the%20population%20dynamics%20of%20transposable%20elements%20in%20Drosophila%20melanogaster.%20PLoS%20Genet.%202015;11:%20e1005269." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Kabi M, Filion GJ. Heterochromatin: did H3K9 methylation evolve to tame transposons? Genome Biol. 2021;22:325.
</cite> [<a href="https://doi.org/10.1186/s13059-021-02550-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8641213/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34857038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kabi%20M,%20Filion%20GJ.%20Heterochromatin:%20did%20H3K9%20methylation%20evolve%20to%20tame%20transposons?%20Genome%20Biol.%202021;22:325." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Ninova M, Fejes Tóth K, Aravin AA. The control of gene expression and cell identity by H3K9 trimethylation. Development. 2019;146:dev181180.</cite> [<a href="https://doi.org/10.1242/dev.181180" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6803365/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31540910/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Choi JY, Purugganan MD. Evolutionary epigenomics of retrotransposon-mediated methylation spreading in rice. Mol Biol Evol. 2018;35:365–82.
</cite> [<a href="https://doi.org/10.1093/molbev/msx284" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5850837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29126199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choi%20JY,%20Purugganan%20MD.%20Evolutionary%20epigenomics%20of%20retrotransposon-mediated%20methylation%20spreading%20in%20rice.%20Mol%20Biol%20Evol.%202018;35:365%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Eichten SR, Ellis NA, Makarevitch I, Yeh C-T, Gent JI, Guo L, et al. Spreading of heterochromatin is limited to specific families of maize retrotransposons. PLoS Genet. 2012;8: e1003127.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1003127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3521669/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23271981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eichten%20SR,%20Ellis%20NA,%20Makarevitch%20I,%20Yeh%20C-T,%20Gent%20JI,%20Guo%20L,%20et%20al.%20Spreading%20of%20heterochromatin%20is%20limited%20to%20specific%20families%20of%20maize%20retrotransposons.%20PLoS%20Genet.%202012;8:%20e1003127." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Gent JI, Ellis NA, Guo L, Harkess AE, Yao Y, Zhang X, et al. CHH islands: de novo DNA methylation in near-gene chromatin regulation in maize. Genome Res. 2013;23:628–37.
</cite> [<a href="https://doi.org/10.1101/gr.146985.112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3613580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23269663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gent%20JI,%20Ellis%20NA,%20Guo%20L,%20Harkess%20AE,%20Yao%20Y,%20Zhang%20X,%20et%20al.%20CHH%20islands:%20de%20novo%20DNA%20methylation%20in%20near-gene%20chromatin%20regulation%20in%20maize.%20Genome%20Res.%202013;23:628%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Rebollo R, Karimi MM, Bilenky M, Gagnier L, Miceli-Royer K, Zhang Y, et al. Retrotransposon-induced heterochromatin spreading in the mouse revealed by insertional polymorphisms. PLOS Genet. 2011;7: e1002301.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1002301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3183085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21980304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rebollo%20R,%20Karimi%20MM,%20Bilenky%20M,%20Gagnier%20L,%20Miceli-Royer%20K,%20Zhang%20Y,%20et%20al.%20Retrotransposon-induced%20heterochromatin%20spreading%20in%20the%20mouse%20revealed%20by%20insertional%20polymorphisms.%20PLOS%20Genet.%202011;7:%20e1002301." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Sienski G, Dönertas D, Brennecke J. Transcriptional silencing of transposons by Piwi and maelstrom and its impact on chromatin state and gene expression. Cell. 2012;151:964–80.
</cite> [<a href="https://doi.org/10.1016/j.cell.2012.10.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3504300/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23159368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sienski%20G,%20D%C3%B6nertas%20D,%20Brennecke%20J.%20Transcriptional%20silencing%20of%20transposons%20by%20Piwi%20and%20maelstrom%20and%20its%20impact%20on%20chromatin%20state%20and%20gene%20expression.%20Cell.%202012;151:964%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Choi JY, Lee YCG. Double-edged sword: the evolutionary consequences of the epigenetic silencing of transposable elements. PLOS Genet. 2020;16: e1008872. 10.1371/journal.pgen.1008872.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1008872" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7365398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32673310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choi%20JY,%20Lee%20YCG.%20Double-edged%20sword:%20the%20evolutionary%20consequences%20of%20the%20epigenetic%20silencing%20of%20transposable%20elements.%20PLOS%20Genet.%202020;16:%20e1008872.%2010.1371/journal.pgen.1008872." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Wei KH-C, Mai D, Chatla K, Bachtrog D. Dynamics and impacts of transposable element proliferation in the Drosophila nasuta species group radiation. Mol Biol Evol. 2022;39:msac080.
</cite> [<a href="https://doi.org/10.1093/molbev/msac080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9075770/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35485457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei%20KH-C,%20Mai%20D,%20Chatla%20K,%20Bachtrog%20D.%20Dynamics%20and%20impacts%20of%20transposable%20element%20proliferation%20in%20the%20Drosophila%20nasuta%20species%20group%20radiation.%20Mol%20Biol%20Evol.%202022;39:msac080." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Lee YCG, Karpen GH. Pervasive epigenetic effects of Drosophila euchromatic transposable elements impact their evolution. eLife. 2017;6:e25762. 10.7554/eLife.25762.
</cite> [<a href="https://doi.org/10.7554/eLife.25762" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5505702/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28695823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20YCG,%20Karpen%20GH.%20Pervasive%20epigenetic%20effects%20of%20Drosophila%20euchromatic%20transposable%20elements%20impact%20their%20evolution.%20eLife.%202017;6:e25762.%2010.7554/eLife.25762." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Stuart T, Eichten SR, Cahn J, Karpievitch YV, Borevitz JO, Lister R. Population scale mapping of transposable element diversity reveals links to gene regulation and epigenomic variation. eLife. 2016;5:e20777.
</cite> [<a href="https://doi.org/10.7554/eLife.20777" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5167521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27911260/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stuart%20T,%20Eichten%20SR,%20Cahn%20J,%20Karpievitch%20YV,%20Borevitz%20JO,%20Lister%20R.%20Population%20scale%20mapping%20of%20transposable%20element%20diversity%20reveals%20links%20to%20gene%20regulation%20and%20epigenomic%20variation.%20eLife.%202016;5:e20777." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Huang Y, Shukla H, Lee YCG. Species-specific chromatin landscape determines how transposable elements shape genome evolution. Nordborg M, Przeworski M, editors. eLife. 2022;11:e81567.
</cite> [<a href="https://doi.org/10.7554/eLife.81567" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9398452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35997258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20Y,%20Shukla%20H,%20Lee%20YCG.%20Species-specific%20chromatin%20landscape%20determines%20how%20transposable%20elements%20shape%20genome%20evolution.%20Nordborg%20M,%20Przeworski%20M,%20editors.%20eLife.%202022;11:e81567." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Fablet M, Salces-Ortiz J, Jacquet A, Menezes BF, Dechaud C, Veber P, et al. A quantitative, genome-wide analysis in Drosophila reveals transposable elements’ influence on gene expression is species-specific. Genome Biol Evol. 2023;15:evad160.
</cite> [<a href="https://doi.org/10.1093/gbe/evad160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10492446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37652057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fablet%20M,%20Salces-Ortiz%20J,%20Jacquet%20A,%20Menezes%20BF,%20Dechaud%20C,%20Veber%20P,%20et%20al.%20A%20quantitative,%20genome-wide%20analysis%20in%20Drosophila%20reveals%20transposable%20elements%E2%80%99%20influence%20on%20gene%20expression%20is%20species-specific.%20Genome%20Biol%20Evol.%202023;15:evad160." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Cao Y, Chen G, Wu G, Zhang X, McDermott J, Chen X, et al. Widespread roles of enhancer-like transposable elements in cell identity and long-range genomic interactions. Genome Res. 2019;29:40–52.
</cite> [<a href="https://doi.org/10.1101/gr.235747.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6314169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30455182/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cao%20Y,%20Chen%20G,%20Wu%20G,%20Zhang%20X,%20McDermott%20J,%20Chen%20X,%20et%20al.%20Widespread%20roles%20of%20enhancer-like%20transposable%20elements%20in%20cell%20identity%20and%20long-range%20genomic%20interactions.%20Genome%20Res.%202019;29:40%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Sundaram V, Cheng Y, Ma Z, Li D, Xing X, Edge P, et al. Widespread contribution of transposable elements to the innovation of gene regulatory networks. Genome Res. 2014;24:1963–76.
</cite> [<a href="https://doi.org/10.1101/gr.168872.113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4248313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25319995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sundaram%20V,%20Cheng%20Y,%20Ma%20Z,%20Li%20D,%20Xing%20X,%20Edge%20P,%20et%20al.%20Widespread%20contribution%20of%20transposable%20elements%20to%20the%20innovation%20of%20gene%20regulatory%20networks.%20Genome%20Res.%202014;24:1963%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Trizzino M, Kapusta A, Brown CD. Transposable elements generate regulatory novelty in a tissue-specific fashion. BMC Genomics. 2018;19:468.
</cite> [<a href="https://doi.org/10.1186/s12864-018-4850-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6006921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29914366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trizzino%20M,%20Kapusta%20A,%20Brown%20CD.%20Transposable%20elements%20generate%20regulatory%20novelty%20in%20a%20tissue-specific%20fashion.%20BMC%20Genomics.%202018;19:468." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>He J, Fu X, Zhang M, He F, Li W, Abdul MM, et al. Transposable elements are regulated by context-specific patterns of chromatin marks in mouse embryonic stem cells. Nat Commun. 2019;10:34.
</cite> [<a href="https://doi.org/10.1038/s41467-018-08006-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6318327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30604769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20J,%20Fu%20X,%20Zhang%20M,%20He%20F,%20Li%20W,%20Abdul%20MM,%20et%20al.%20Transposable%20elements%20are%20regulated%20by%20context-specific%20patterns%20of%20chromatin%20marks%20in%20mouse%20embryonic%20stem%20cells.%20Nat%20Commun.%202019;10:34." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Roquis D, Robertson M, Yu L, Thieme M, Julkowska M, Bucher E. Genomic impact of stress-induced transposable element mobility in Arabidopsis. Nucleic Acids Res. 2021;49:10431–47.
</cite> [<a href="https://doi.org/10.1093/nar/gkab828" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8501995/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34551439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roquis%20D,%20Robertson%20M,%20Yu%20L,%20Thieme%20M,%20Julkowska%20M,%20Bucher%20E.%20Genomic%20impact%20of%20stress-induced%20transposable%20element%20mobility%20in%20Arabidopsis.%20Nucleic%20Acids%20Res.%202021;49:10431%E2%80%9347." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Sultana T, Zamborlini A, Cristofari G, Lesage P. Integration site selection by retroviruses and transposable elements in eukaryotes. Nat Rev Genet. 2017;18:292–308.
</cite> [<a href="https://doi.org/10.1038/nrg.2017.7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28286338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sultana%20T,%20Zamborlini%20A,%20Cristofari%20G,%20Lesage%20P.%20Integration%20site%20selection%20by%20retroviruses%20and%20transposable%20elements%20in%20eukaryotes.%20Nat%20Rev%20Genet.%202017;18:292%E2%80%93308." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Quadrana L, Etcheverry M, Gilly A, Caillieux E, Madoui M-A, Guy J, et al. Transposition favors the generation of large effect mutations that may facilitate rapid adaption. Nat Commun. 2019;10:3421.
</cite> [<a href="https://doi.org/10.1038/s41467-019-11385-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6668482/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31366887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quadrana%20L,%20Etcheverry%20M,%20Gilly%20A,%20Caillieux%20E,%20Madoui%20M-A,%20Guy%20J,%20et%20al.%20Transposition%20favors%20the%20generation%20of%20large%20effect%20mutations%20that%20may%20facilitate%20rapid%20adaption.%20Nat%20Commun.%202019;10:3421." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Guio L, Vieira C, González J. Stress affects the epigenetic marks added by natural transposable element insertions in Drosophila melanogaster. Sci Rep. 2018;8:12197. 10.1038/s41598-018-30491-w.
</cite> [<a href="https://doi.org/10.1038/s41598-018-30491-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6093896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30111890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guio%20L,%20Vieira%20C,%20Gonz%C3%A1lez%20J.%20Stress%20affects%20the%20epigenetic%20marks%20added%20by%20natural%20transposable%20element%20insertions%20in%20Drosophila%20melanogaster.%20Sci%20Rep.%202018;8:12197.%2010.1038/s41598-018-30491-w." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Rebollo R, Horard B, Begeot F, Delattre M, Gilson E, Vieira C. A snapshot of histone modifications within transposable elements in Drosophila wild type strains. PLoS ONE. 2012;7: e44253. 10.1371/journal.pone.0044253.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0044253" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3433462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22962605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rebollo%20R,%20Horard%20B,%20Begeot%20F,%20Delattre%20M,%20Gilson%20E,%20Vieira%20C.%20A%20snapshot%20of%20histone%20modifications%20within%20transposable%20elements%20in%20Drosophila%20wild%20type%20strains.%20PLoS%20ONE.%202012;7:%20e44253.%2010.1371/journal.pone.0044253." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Koenecke N, Johnston J, Gaertner B, Natarajan M, Zeitlinger J. Genome-wide identification of Drosophila dorso-ventral enhancers by differential histone acetylation analysis. Genome Biol. 2016;17:196. 10.1186/s13059-016-1057-2.
</cite> [<a href="https://doi.org/10.1186/s13059-016-1057-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5037609/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27678375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koenecke%20N,%20Johnston%20J,%20Gaertner%20B,%20Natarajan%20M,%20Zeitlinger%20J.%20Genome-wide%20identification%20of%20Drosophila%20dorso-ventral%20enhancers%20by%20differential%20histone%20acetylation%20analysis.%20Genome%20Biol.%202016;17:196.%2010.1186/s13059-016-1057-2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Coronado-Zamora M, Salces-Ortiz J, González J. DrosOmics: a browser to explore -omics variation across high-quality reference genomes from natural populations of Drosophila melanogaster. Mol Biol Evol. 2023;40: msad075. 10.1093/molbev/msad075.
</cite> [<a href="https://doi.org/10.1093/molbev/msad075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10089646/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36971243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coronado-Zamora%20M,%20Salces-Ortiz%20J,%20Gonz%C3%A1lez%20J.%20DrosOmics:%20a%20browser%20to%20explore%20-omics%20variation%20across%20high-quality%20reference%20genomes%20from%20natural%20populations%20of%20Drosophila%20melanogaster.%20Mol%20Biol%20Evol.%202023;40:%20msad075.%2010.1093/molbev/msad075." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Rech GE, Radío S, Guirao-Rico S, Aguilera L, Horvath V, Green L, et al. Population-scale long-read sequencing uncovers transposable elements associated with gene expression variation and adaptive signatures in Drosophila. Nat Commun. 2022;13:1–16.
</cite> [<a href="https://doi.org/10.1038/s41467-022-29518-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9005704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35413957/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rech%20GE,%20Rad%C3%ADo%20S,%20Guirao-Rico%20S,%20Aguilera%20L,%20Horvath%20V,%20Green%20L,%20et%20al.%20Population-scale%20long-read%20sequencing%20uncovers%20transposable%20elements%20associated%20with%20gene%20expression%20variation%20and%20adaptive%20signatures%20in%20Drosophila.%20Nat%20Commun.%202022;13:1%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Noshay JM, Springer NM. Stories that can’t be told by SNPs; DNA methylation variation in plant populations. Curr Opin Plant Biol. 2021;61: 101989.
</cite> [<a href="https://doi.org/10.1016/j.pbi.2020.101989" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33445144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Noshay%20JM,%20Springer%20NM.%20Stories%20that%20can%E2%80%99t%20be%20told%20by%20SNPs;%20DNA%20methylation%20variation%20in%20plant%20populations.%20Curr%20Opin%20Plant%20Biol.%202021;61:%20101989." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Buecker C, Wysocka J. Enhancers as information integration hubs in development: lessons from genomics. Trends Genet TIG. 2012;28:276–84.
</cite> [<a href="https://doi.org/10.1016/j.tig.2012.02.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5064438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22487374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buecker%20C,%20Wysocka%20J.%20Enhancers%20as%20information%20integration%20hubs%20in%20development:%20lessons%20from%20genomics.%20Trends%20Genet%20TIG.%202012;28:276%E2%80%9384." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Pundhir S, Poirazi P, Gorodkin J. Emerging applications of read profiles towards the functional annotation of the genome. Front Genet. 2015;6:188.
</cite> [<a href="https://doi.org/10.3389/fgene.2015.00188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4437211/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26042150/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pundhir%20S,%20Poirazi%20P,%20Gorodkin%20J.%20Emerging%20applications%20of%20read%20profiles%20towards%20the%20functional%20annotation%20of%20the%20genome.%20Front%20Genet.%202015;6:188." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Wicker T, Sabot F, Hua-Van A, Bennetzen JL, Capy P, Chalhoub B, et al. A unified classification system for eukaryotic transposable elements. Nat Rev Genet. 2007;8:973–82.
</cite> [<a href="https://doi.org/10.1038/nrg2165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17984973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wicker%20T,%20Sabot%20F,%20Hua-Van%20A,%20Bennetzen%20JL,%20Capy%20P,%20Chalhoub%20B,%20et%20al.%20A%20unified%20classification%20system%20for%20eukaryotic%20transposable%20elements.%20Nat%20Rev%20Genet.%202007;8:973%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Quesneville H, Bergman CM, Andrieu O, Autard D, Nouaud D, Ashburner M, et al. Combined evidence annotation of transposable elements in genome sequences. PLOS Comput Biol. 2005;1: e22.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.0010022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1185648/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16110336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quesneville%20H,%20Bergman%20CM,%20Andrieu%20O,%20Autard%20D,%20Nouaud%20D,%20Ashburner%20M,%20et%20al.%20Combined%20evidence%20annotation%20of%20transposable%20elements%20in%20genome%20sequences.%20PLOS%20Comput%20Biol.%202005;1:%20e22." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Brennecke J, Malone CD, Aravin AA, Sachidanandam R, Stark A, Hannon GJ. An epigenetic role for maternally inherited piRNAs in transposon silencing. Science. 2008;322:1387–92.
</cite> [<a href="https://doi.org/10.1126/science.1165171" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2805124/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19039138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brennecke%20J,%20Malone%20CD,%20Aravin%20AA,%20Sachidanandam%20R,%20Stark%20A,%20Hannon%20GJ.%20An%20epigenetic%20role%20for%20maternally%20inherited%20piRNAs%20in%20transposon%20silencing.%20Science.%202008;322:1387%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Kaminker JS, Bergman CM, Kronmiller B, Carlson J, Svirskas R, Patel S, et al. The transposable elements of the Drosophila melanogaster euchromatin: a genomics perspective. Genome Biol. 2002;3:research0084 .1.
</cite> [<a href="https://doi.org/10.1186/gb-2002-3-12-research0084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC151186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12537573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaminker%20JS,%20Bergman%20CM,%20Kronmiller%20B,%20Carlson%20J,%20Svirskas%20R,%20Patel%20S,%20et%20al.%20The%20transposable%20elements%20of%20the%20Drosophila%20melanogaster%20euchromatin:%20a%20genomics%20perspective.%20Genome%20Biol.%202002;3:research0084%20.1." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Blumenstiel JP, Chen X, He M, Bergman CM. An age-of-allele test of neutrality for transposable element insertions. Genetics. 2014;196:523–38.
</cite> [<a href="https://doi.org/10.1534/genetics.113.158147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3914624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24336751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blumenstiel%20JP,%20Chen%20X,%20He%20M,%20Bergman%20CM.%20An%20age-of-allele%20test%20of%20neutrality%20for%20transposable%20element%20insertions.%20Genetics.%202014;196:523%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Lange JD, Bastide H, Lack JB, Pool JE. A population genomic assessment of three decades of evolution in a natural Drosophila population. Mol Biol Evol. 2022;39:msab368.
</cite> [<a href="https://doi.org/10.1093/molbev/msab368" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8826484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34971382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lange%20JD,%20Bastide%20H,%20Lack%20JB,%20Pool%20JE.%20A%20population%20genomic%20assessment%20of%20three%20decades%20of%20evolution%20in%20a%20natural%20Drosophila%20population.%20Mol%20Biol%20Evol.%202022;39:msab368." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Ullastres A, Petit N, González J. Exploring the phenotypic space and the evolutionary history of a natural mutation in Drosophila melanogaster. Mol Biol Evol. 2015;32:1800–14.
</cite> [<a href="https://doi.org/10.1093/molbev/msv061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4476160/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25862139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ullastres%20A,%20Petit%20N,%20Gonz%C3%A1lez%20J.%20Exploring%20the%20phenotypic%20space%20and%20the%20evolutionary%20history%20of%20a%20natural%20mutation%20in%20Drosophila%20melanogaster.%20Mol%20Biol%20Evol.%202015;32:1800%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Ullastres A, Merenciano M, González J. Regulatory regions in natural transposable element insertions drive interindividual differences in response to immune challenges in Drosophila. Genome Biol. 2021;22:265.
</cite> [<a href="https://doi.org/10.1186/s13059-021-02471-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8439047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34521452/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ullastres%20A,%20Merenciano%20M,%20Gonz%C3%A1lez%20J.%20Regulatory%20regions%20in%20natural%20transposable%20element%20insertions%20drive%20interindividual%20differences%20in%20response%20to%20immune%20challenges%20in%20Drosophila.%20Genome%20Biol.%202021;22:265." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Le Manh H, Guio L, Merenciano M, Rovira Q, Barrón MG, González J. Natural and laboratory mutations in kuzbanian are associated with zinc stress phenotypes in Drosophila melanogaster. Sci Rep. 2017;7: 42663.
</cite> [<a href="https://doi.org/10.1038/srep42663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5316978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28218276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Le%20Manh%20H,%20Guio%20L,%20Merenciano%20M,%20Rovira%20Q,%20Barr%C3%B3n%20MG,%20Gonz%C3%A1lez%20J.%20Natural%20and%20laboratory%20mutations%20in%20kuzbanian%20are%20associated%20with%20zinc%20stress%20phenotypes%20in%20Drosophila%20melanogaster.%20Sci%20Rep.%202017;7:%2042663." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>González J, Lenkov K, Lipatov M, Macpherson JM, Petrov DA. High rate of recent transposable element–induced adaptation in Drosophila melanogaster. PLOS Biol. 2008;6: e251.
</cite> [<a href="https://doi.org/10.1371/journal.pbio.0060251" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2570423/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18942889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gonz%C3%A1lez%20J,%20Lenkov%20K,%20Lipatov%20M,%20Macpherson%20JM,%20Petrov%20DA.%20High%20rate%20of%20recent%20transposable%20element%E2%80%93induced%20adaptation%20in%20Drosophila%20melanogaster.%20PLOS%20Biol.%202008;6:%20e251." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Kofler R, Schlötterer C. Gowinda: unbiased analysis of gene set enrichment for genome-wide association studies. Bioinformatics. 2012;28:2084–5.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts315" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3400962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22635606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kofler%20R,%20Schl%C3%B6tterer%20C.%20Gowinda:%20unbiased%20analysis%20of%20gene%20set%20enrichment%20for%20genome-wide%20association%20studies.%20Bioinformatics.%202012;28:2084%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Meng D, Dubin M, Zhang P, Osborne EJ, Stegle O, Clark RM, et al. Limited contribution of DNA methylation variation to expression regulation in Arabidopsis thaliana. PLOS Genet. 2016;12: e1006141.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1006141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4939946/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27398721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meng%20D,%20Dubin%20M,%20Zhang%20P,%20Osborne%20EJ,%20Stegle%20O,%20Clark%20RM,%20et%20al.%20Limited%20contribution%20of%20DNA%20methylation%20variation%20to%20expression%20regulation%20in%20Arabidopsis%20thaliana.%20PLOS%20Genet.%202016;12:%20e1006141." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Riddle NC, Minoda A, Kharchenko PV, Alekseyenko AA, Schwartz YB, Tolstorukov MY, et al. Plasticity in patterns of histone modifications and chromosomal proteins in Drosophila heterochromatin. Genome Res. 2011;21:147–63.
</cite> [<a href="https://doi.org/10.1101/gr.110098.110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3032919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21177972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Riddle%20NC,%20Minoda%20A,%20Kharchenko%20PV,%20Alekseyenko%20AA,%20Schwartz%20YB,%20Tolstorukov%20MY,%20et%20al.%20Plasticity%20in%20patterns%20of%20histone%20modifications%20and%20chromosomal%20proteins%20in%20Drosophila%20heterochromatin.%20Genome%20Res.%202011;21:147%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Borghini L, Hibberd M, Davila S. Changes in H3K27ac following lipopolysaccharide stimulation of nasopharyngeal epithelial cells. BMC Genomics. 2018;19:969.
</cite> [<a href="https://doi.org/10.1186/s12864-018-5295-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6307289/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30587130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Borghini%20L,%20Hibberd%20M,%20Davila%20S.%20Changes%20in%20H3K27ac%20following%20lipopolysaccharide%20stimulation%20of%20nasopharyngeal%20epithelial%20cells.%20BMC%20Genomics.%202018;19:969." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Sun W, Poschmann J, Cruz-Herrera del Rosario R, Parikshak NN, Hajan HS, Kumar V, et al. Histone acetylome-wide association study of autism spectrum disorder. Cell. 2016;167:1385–1397.e11.</cite> [<a href="https://doi.org/10.1016/j.cell.2016.10.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27863250/" class="usa-link">PubMed</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Zhang T, Zhang Z, Dong Q, Xiong J, Zhu B. Histone H3K27 acetylation is dispensable for enhancer activity in mouse embryonic stem cells. Genome Biol. 2020;21:45.
</cite> [<a href="https://doi.org/10.1186/s13059-020-01957-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7035716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32085783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20T,%20Zhang%20Z,%20Dong%20Q,%20Xiong%20J,%20Zhu%20B.%20Histone%20H3K27%20acetylation%20is%20dispensable%20for%20enhancer%20activity%20in%20mouse%20embryonic%20stem%20cells.%20Genome%20Biol.%202020;21:45." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Aravin AA, Sachidanandam R, Girard A, Fejes-Toth K, Hannon GJ. Developmentally regulated piRNA clusters implicate MILI in transposon control. Science. 2007;316:744–7.
</cite> [<a href="https://doi.org/10.1126/science.1142612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17446352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aravin%20AA,%20Sachidanandam%20R,%20Girard%20A,%20Fejes-Toth%20K,%20Hannon%20GJ.%20Developmentally%20regulated%20piRNA%20clusters%20implicate%20MILI%20in%20transposon%20control.%20Science.%202007;316:744%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Wang SH, Elgin SCR. Drosophila Piwi functions downstream of piRNA production mediating a chromatin-based transposon silencing mechanism in female germ line. Proc Natl Acad Sci. 2011;108:21164–9.
</cite> [<a href="https://doi.org/10.1073/pnas.1107892109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3248523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22160707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20SH,%20Elgin%20SCR.%20Drosophila%20Piwi%20functions%20downstream%20of%20piRNA%20production%20mediating%20a%20chromatin-based%20transposon%20silencing%20mechanism%20in%20female%20germ%20line.%20Proc%20Natl%20Acad%20Sci.%202011;108:21164%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Akkouche A, Mugat B, Barckmann B, Varela-Chavez C, Li B, Raffel R, et al. Piwi is required during Drosophila embryogenesis to license dual-strand piRNA clusters for transposon repression in adult ovaries. Mol Cell. 2017;66:411-419.e4.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2017.03.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28457744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akkouche%20A,%20Mugat%20B,%20Barckmann%20B,%20Varela-Chavez%20C,%20Li%20B,%20Raffel%20R,%20et%20al.%20Piwi%20is%20required%20during%20Drosophila%20embryogenesis%20to%20license%20dual-strand%20piRNA%20clusters%20for%20transposon%20repression%20in%20adult%20ovaries.%20Mol%20Cell.%202017;66:411-419.e4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Perrat PN, DasGupta S, Wang J, Theurkauf W, Weng Z, Rosbash M, et al. Transposition-driven genomic heterogeneity in the Drosophila brain. Science. 2013;340:91–5.
</cite> [<a href="https://doi.org/10.1126/science.1231965" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3887341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23559253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perrat%20PN,%20DasGupta%20S,%20Wang%20J,%20Theurkauf%20W,%20Weng%20Z,%20Rosbash%20M,%20et%20al.%20Transposition-driven%20genomic%20heterogeneity%20in%20the%20Drosophila%20brain.%20Science.%202013;340:91%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Baillie JK, Barnett MW, Upton KR, Gerhardt DJ, Richmond TA, De Sapio F, et al. Somatic retrotransposition alters the genetic landscape of the human brain. Nature. 2011;479:534–7.
</cite> [<a href="https://doi.org/10.1038/nature10531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3224101/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22037309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baillie%20JK,%20Barnett%20MW,%20Upton%20KR,%20Gerhardt%20DJ,%20Richmond%20TA,%20De%20Sapio%20F,%20et%20al.%20Somatic%20retrotransposition%20alters%20the%20genetic%20landscape%20of%20the%20human%20brain.%20Nature.%202011;479:534%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Richardson SR, Morell S, Faulkner GJ. L1 retrotransposons and somatic mosaicism in the brain. Annu Rev Genet. 2014;48:1–27.
</cite> [<a href="https://doi.org/10.1146/annurev-genet-120213-092412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25036377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Richardson%20SR,%20Morell%20S,%20Faulkner%20GJ.%20L1%20retrotransposons%20and%20somatic%20mosaicism%20in%20the%20brain.%20Annu%20Rev%20Genet.%202014;48:1%E2%80%9327." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Sanchez-Luque FJ, Kempen MJHC, Gerdes P, Vargas-Landin DB, Richardson SR, Troskie RL, et al. LINE-1 evasion of epigenetic repression in humans. Mol Cell. 2019;75:590-604.e12.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2019.05.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31230816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanchez-Luque%20FJ,%20Kempen%20MJHC,%20Gerdes%20P,%20Vargas-Landin%20DB,%20Richardson%20SR,%20Troskie%20RL,%20et%20al.%20LINE-1%20evasion%20of%20epigenetic%20repression%20in%20humans.%20Mol%20Cell.%202019;75:590-604.e12." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Treiber CD, Waddell S. Resolving the prevalence of somatic transposition in Drosophila. eLife. 2017;6:e28297.
</cite> [<a href="https://doi.org/10.7554/eLife.28297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5553932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28742021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Treiber%20CD,%20Waddell%20S.%20Resolving%20the%20prevalence%20of%20somatic%20transposition%20in%20Drosophila.%20eLife.%202017;6:e28297." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Evrony GD, Lee E, Park PJ, Walsh CA. Resolving rates of mutation in the brain using single-neuron genomics. eLife. 2016;5:e12966.
</cite> [<a href="https://doi.org/10.7554/eLife.12966" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4805530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26901440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Evrony%20GD,%20Lee%20E,%20Park%20PJ,%20Walsh%20CA.%20Resolving%20rates%20of%20mutation%20in%20the%20brain%20using%20single-neuron%20genomics.%20eLife.%202016;5:e12966." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Siudeja K, Beek M, Riddiford N, Boumard B, Wurmser A, Stefanutti M, et al. Unraveling the features of somatic transposition in the Drosophila intestine. EMBO J. 2021;e106388.</cite> [<a href="https://doi.org/10.15252/embj.2020106388" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8090852/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33634906/" class="usa-link">PubMed</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Roy M, Viginier B, Saint-Michel É, Arnaud F, Ratinier M, Fablet M. Viral infection impacts transposable element transcript amounts in Drosophila. Proc Natl Acad Sci. 2020;117:12249–57.
</cite> [<a href="https://doi.org/10.1073/pnas.2006106117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7275722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32434916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roy%20M,%20Viginier%20B,%20Saint-Michel%20%C3%89,%20Arnaud%20F,%20Ratinier%20M,%20Fablet%20M.%20Viral%20infection%20impacts%20transposable%20element%20transcript%20amounts%20in%20Drosophila.%20Proc%20Natl%20Acad%20Sci.%202020;117:12249%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Mayeux CA, Larue A, Oliveira DS, Varoqui M, Henri H, Rebollo R, et al. Intricate interactions between antiviral immunity and transposable element control in Drosophila. bioRxiv. 2024. 10.1101/2024.03.18.585529.</cite>
</li>
<li id="CR63">
<span class="label">63.</span><cite>Neph S, Kuehn MS, Reynolds AP, Haugen E, Thurman RE, Johnson AK, et al. BEDOPS: high-performance genomic feature operations. Bioinformatics. 2012;28:1919–20.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3389768/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22576172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Neph%20S,%20Kuehn%20MS,%20Reynolds%20AP,%20Haugen%20E,%20Thurman%20RE,%20Johnson%20AK,%20et%20al.%20BEDOPS:%20high-performance%20genomic%20feature%20operations.%20Bioinformatics.%202012;28:1919%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Quinlan AR, Hall IM. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics. 2010;26:841–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2832824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20110278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quinlan%20AR,%20Hall%20IM.%20BEDTools:%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features.%20Bioinformatics.%202010;26:841%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Li H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics. 2018;34:3094–100.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6137996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29750242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H.%20Minimap2:%20pairwise%20alignment%20for%20nucleotide%20sequences.%20Bioinformatics.%202018;34:3094%E2%80%93100." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>BioProject PRJNA643665. Raw RNA-seq and ChIP-seq datasets. Available from: <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA643665" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA643665</a>.</cite>
</li>
<li id="CR67">
<span class="label">67.</span><cite>Chen S, Zhou Y, Chen Y, Gu J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics. 2018;34:i884–90.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20S,%20Zhou%20Y,%20Chen%20Y,%20Gu%20J.%20fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor.%20Bioinformatics.%202018;34:i884%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Zeng X, Li B, Welch R, Rojo C, Zheng Y, Dewey CN, et al. Perm-seq: mapping protein-DNA interactions in segmental duplication and highly repetitive regions of genomes with prior-enhanced read mapping. PLoS Comput Biol. 2015;11: e1004491.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1004491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4618727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26484757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng%20X,%20Li%20B,%20Welch%20R,%20Rojo%20C,%20Zheng%20Y,%20Dewey%20CN,%20et%20al.%20Perm-seq:%20mapping%20protein-DNA%20interactions%20in%20segmental%20duplication%20and%20highly%20repetitive%20regions%20of%20genomes%20with%20prior-enhanced%20read%20mapping.%20PLoS%20Comput%20Biol.%202015;11:%20e1004491." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9:357–9.
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead%20B,%20Salzberg%20SL.%20Fast%20gapped-read%20alignment%20with%20Bowtie%202.%20Nat%20Methods.%202012;9:357%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Chung D, Kuan PF, Li B, Sanalkumar R, Liang K, Bresnick EH, et al. Discovering transcription factor binding sites in highly repetitive regions of genomes with multi-read analysis of ChIP-Seq data. PLOS Comput Biol. 2011;7: e1002111.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1002111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3136429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21779159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chung%20D,%20Kuan%20PF,%20Li%20B,%20Sanalkumar%20R,%20Liang%20K,%20Bresnick%20EH,%20et%20al.%20Discovering%20transcription%20factor%20binding%20sites%20in%20highly%20repetitive%20regions%20of%20genomes%20with%20multi-read%20analysis%20of%20ChIP-Seq%20data.%20PLOS%20Comput%20Biol.%202011;7:%20e1002111." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Hitz BC, Jin-Wook L, Jolanki O, Kagda MS, Graham K, Sud P, et al. The ENCODE uniform analysis pipelines. bioRxiv. 2023;2023.04.04.535623.</cite>
</li>
<li id="CR72">
<span class="label">72.</span><cite>Gaspar JM. Improved peak-calling with MACS2. bioRxiv. 2018;496521v1.</cite>
</li>
<li id="CR73">
<span class="label">73.</span><cite>R Core Team. R: a language and environment for statistical computing. Vienna: R Foundation for Statistical Computing; 2021. Available from: <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R-project.org/</a>.</cite>
</li>
<li id="CR74">
<span class="label">74.</span><cite>Pertea M, Kim D, Pertea G, Leek JT, Salzberg SL. Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie, and Ballgown. Nat Protoc. 2016;11:1650–67.
</cite> [<a href="https://doi.org/10.1038/nprot.2016.095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5032908/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27560171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pertea%20M,%20Kim%20D,%20Pertea%20G,%20Leek%20JT,%20Salzberg%20SL.%20Transcript-level%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20HISAT,%20StringTie,%20and%20Ballgown.%20Nat%20Protoc.%202016;11:1650%E2%80%9367." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Pertea M, Pertea GM, Antonescu CM, Chang T-C, Mendell JT, Salzberg SL. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. Nat Biotechnol. 2015;33:290–5.
</cite> [<a href="https://doi.org/10.1038/nbt.3122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4643835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25690850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pertea%20M,%20Pertea%20GM,%20Antonescu%20CM,%20Chang%20T-C,%20Mendell%20JT,%20Salzberg%20SL.%20StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads.%20Nat%20Biotechnol.%202015;33:290%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Robinson MD, McCarthy DJ, Smyth GK. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics. 2010;26:139–40.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2796818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Robinson%20MD,%20McCarthy%20DJ,%20Smyth%20GK.%20edgeR:%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data.%20Bioinformatics.%202010;26:139%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Coronado-Zamora M. GonzalezLab/epigenetic-effects-transposons-dmelanogaster: scripts and data for the “The epigenetics effects of transposable elements are genomic context dependent and not restricted to gene silencing”. Code repository. Zenodo; 2025. Available from: <a href="https://zenodo.org/records/15876977" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/15876977</a>.</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12360000/bin/13059_2025_3705_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3705_MOESM1_ESM.zip</a><sup> (1.4MB, zip) </sup><p>Additional file 1: Supplementary tables.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12360000/bin/13059_2025_3705_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3705_MOESM2_ESM.docx</a><sup> (2.7MB, docx) </sup><p>Additional file 2: Supplementary table legends and supplementary figures.</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All scripts and R Markdown documents to reproduce analyses and figures have been deposited in GitHub: <a href="https://github.com/GonzalezLab/epigenetic-effects-transposons-dmelanogaster" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/GonzalezLab/epigenetic-effects-transposons-dmelanogaster</a> [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>] and are also available in Zenodo (10.5281/zenodo.15876976). Raw RNA-seq and ChIP-seq data for five D. melanogaster strains (AKA-017, JUT-011, MUN-016, SLA-001 and TOM-007, [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]) are available in the NCBI Sequence Read Archive (SRA) database (BioProject PRJNA643665) [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>].</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Genome Biology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s13059-025-03705-4"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/13059_2025_Article_3705.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12360000/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12360000/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12360000%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360000/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12360000/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12360000/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40826096/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12360000/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40826096/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12360000/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12360000/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="7FswI4odKVthkBKvKVkHOOwV9WaZhyxCp91cfyUDbHRjNArRWRtr28xuUKjsC8du">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
