
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Root-to-shoot mobile mRNA CmoKARI1 promotes JA-Ile biosynthesis to confer chilling tolerance in grafted cucumbers - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5326478AF147C30526470058455A0E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368190/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Root-to-shoot mobile mRNA CmoKARI1 promotes JA-Ile biosynthesis to confer chilling tolerance in grafted cucumbers">
<meta name="citation_author" content="Miao Zhang">
<meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China">
<meta name="citation_author" content="Wenqian Liu">
<meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China">
<meta name="citation_author" content="Cuicui Wang">
<meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China">
<meta name="citation_author" content="Shujin Lin">
<meta name="citation_author_institution" content="College of Biology Science and Engineering, Fuzhou University, Fuzhou, 350116 China">
<meta name="citation_author" content="Ying Chen">
<meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China">
<meta name="citation_author" content="Heran Cui">
<meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China">
<meta name="citation_author" content="Chenggang Xiang">
<meta name="citation_author_institution" content="College of Life Science and Technology, Honghe University, Mengzi, Yunnan 661100 China">
<meta name="citation_author" content="Yujia Ma">
<meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China">
<meta name="citation_author" content="Xiaojun Li">
<meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China">
<meta name="citation_author" content="Yongzhen Lu">
<meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China">
<meta name="citation_author" content="Xiao Han">
<meta name="citation_author_institution" content="College of Biology Science and Engineering, Fuzhou University, Fuzhou, 350116 China">
<meta name="citation_author" content="Yihan Dong">
<meta name="citation_author_institution" content="Institut de Biologie Moléculaire des Plantes, Centre National de la Recherche Scientifique, UPR 2357, Université de Strasbourg, 67000 Strasbourg, France">
<meta name="citation_author_institution" content="Present Address: Institut de Génétique et de Biologie Moléculaire et Cellulaire, 67404 Illkirch, France">
<meta name="citation_author" content="Lihong Gao">
<meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China">
<meta name="citation_author" content="Wenna Zhang">
<meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7782">
<meta name="citation_doi" content="10.1038/s41467-025-63228-1">
<meta name="citation_pmid" content="40835858">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368190/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368190/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368190/pdf/41467_2025_Article_63228.pdf">
<meta name="description" content="Pumpkin rootstock is commonly used to graft cucurbit crops, improving their ability to withstand stress. While the significance of systemic signals from rootstocks to scions is recognized, the role of root-to-shoot transported mRNAs remains ...">
<meta name="og:title" content="Root-to-shoot mobile mRNA CmoKARI1 promotes JA-Ile biosynthesis to confer chilling tolerance in grafted cucumbers">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Pumpkin rootstock is commonly used to graft cucurbit crops, improving their ability to withstand stress. While the significance of systemic signals from rootstocks to scions is recognized, the role of root-to-shoot transported mRNAs remains ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368190/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368190">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63228-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63228.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368190%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368190/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368190/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368190/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7782. doi: <a href="https://doi.org/10.1038/s41467-025-63228-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63228-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Root-to-shoot mobile mRNA <em>CmoKARI1</em> promotes JA-Ile biosynthesis to confer chilling tolerance in grafted cucumbers</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Miao Zhang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Miao Zhang</span></h3>
<div class="p">
<sup>1</sup>Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Miao Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Wenqian Liu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Wenqian Liu</span></h3>
<div class="p">
<sup>1</sup>Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenqian Liu</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Cuicui Wang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Cuicui Wang</span></h3>
<div class="p">
<sup>1</sup>Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Cuicui Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Shujin Lin</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Shujin Lin</span></h3>
<div class="p">
<sup>2</sup>College of Biology Science and Engineering, Fuzhou University, Fuzhou, 350116 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shujin Lin</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ying Chen</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ying Chen</span></h3>
<div class="p">
<sup>1</sup>Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Ying Chen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cui%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Heran Cui</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Heran Cui</span></h3>
<div class="p">
<sup>1</sup>Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cui%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Heran Cui</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiang%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Chenggang Xiang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Chenggang Xiang</span></h3>
<div class="p">
<sup>3</sup>College of Life Science and Technology, Honghe University, Mengzi, Yunnan 661100 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiang%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chenggang Xiang</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Yujia Ma</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Yujia Ma</span></h3>
<div class="p">
<sup>1</sup>Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yujia Ma</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Xiaojun Li</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Xiaojun Li</span></h3>
<div class="p">
<sup>1</sup>Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaojun Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Yongzhen Lu</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Yongzhen Lu</span></h3>
<div class="p">
<sup>1</sup>Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yongzhen Lu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Xiao Han</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Xiao Han</span></h3>
<div class="p">
<sup>2</sup>College of Biology Science and Engineering, Fuzhou University, Fuzhou, 350116 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiao Han</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Yihan Dong</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Yihan Dong</span></h3>
<div class="p">
<sup>4</sup>Institut de Biologie Moléculaire des Plantes, Centre National de la Recherche Scientifique, UPR 2357, Université de Strasbourg, 67000 Strasbourg, France </div>
<div class="p">
<sup>5</sup>Present Address: Institut de Génétique et de Biologie Moléculaire et Cellulaire, 67404 Illkirch, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yihan Dong</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Lihong Gao</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Lihong Gao</span></h3>
<div class="p">
<sup>1</sup>Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lihong Gao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Wenna Zhang</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Wenna Zhang</span></h3>
<div class="p">
<sup>1</sup>Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenna Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, China Agricultural University, Beijing, 100193 China </div>
<div id="Aff2">
<sup>2</sup>College of Biology Science and Engineering, Fuzhou University, Fuzhou, 350116 China </div>
<div id="Aff3">
<sup>3</sup>College of Life Science and Technology, Honghe University, Mengzi, Yunnan 661100 China </div>
<div id="Aff4">
<sup>4</sup>Institut de Biologie Moléculaire des Plantes, Centre National de la Recherche Scientifique, UPR 2357, Université de Strasbourg, 67000 Strasbourg, France </div>
<div id="Aff5">
<sup>5</sup>Present Address: Institut de Génétique et de Biologie Moléculaire et Cellulaire, 67404 Illkirch, France </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2023 Dec 3; Accepted 2025 Aug 13; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368190  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835858/" class="usa-link">40835858</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Pumpkin rootstock is commonly used to graft cucurbit crops, improving their ability to withstand stress. While the significance of systemic signals from rootstocks to scions is recognized, the role of root-to-shoot transported mRNAs remains understudied. Cucumber plants often face growth and productivity limitations due to low temperatures. To shed light on the enhancement of chilling tolerance in grafted cucumber scions by pumpkin-derived mRNAs, we revisit the metabolomic and transcriptomic dataset from the cucumber/pumpkin heterograft under chilling condition. We identify pumpkin <em>Ketol-acid reductoisomerase 1</em> (<em>CmoKARI1</em>) as the key mobile mRNA that specifically travels from pumpkin rootstock to cucumber scion upon early chilling stress. Overexpressing <em>CmoKARI1</em> results in increased isoleucine level and chilling tolerance in both cucumber and Arabidopsis. The increased isoleucine is further used to synthesize JA-Ile conjugates, activating JA-Ile signaling and enabling heterografts to weather low temperatures. This study represents the instance of a unidirectional mobile mRNA triggered by specific environmental cues.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Plant signalling, Abiotic, Plant hormones</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Pumpkin rootstock delivers CmoKARI1 mRNA to cucumber scion under chilling stress, enhancing tolerance by activating JA-Ile signaling. This mobile mRNA acts as a key chilling-responsive signal, redefining the understanding of rootstock–scion communication.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Climate change caused increasing and unpredictable chilling stress (&gt;0 °C). To cope with low temperatures, plants have developed intricate mechanisms<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Different from chilling stress, freezing stress, characterized by temperatures below zero, can lead to the formation of ice crystals that damage membrane systems. While some crops, like winter wheat, exhibit resilience to freezing stress, others, such as cucumber (<em>Cucumis sativus</em> L.), are susceptible to being fatally impacted by chilling stress (&lt;10 °C)<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Although the underlying mechanisms for these variations remain insufficiently understood, both chilling and freezing stress responses are governed by conserved genetic factors. One of the most extensively studied transcription factors is the C-repeat binding factor/dehydration-responsive element binding protein 1 (CBF/DREB1)<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. CBF/DREB1 specifically binds to the CRT/DRE sequence present in the promoter regions of cold-responsive genes (<em>COR</em>)<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. This regulatory interaction initiates the transcriptional activation of <em>COR</em> genes. CBF Expression 1 (ICE1), a MYC-type transcription factor, is the most-studied activator of <em>CBF</em> transcription. The stability of ICE is upheld by phosphorylated Open Stomata 1 (OST1) and MPK3, which transduce signals stemming from upstream reactive oxygen species (ROS) or calcium fluctuations induced by lowered temperatures<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>.</p>
<p id="Par4">To enhance cucumber’s chilling tolerance, grafting with pumpkin rootstocks such as figleaf gourd (<em>Cucurbita ficifolia</em> Bouché) and other pumpkin varieties (e.g., <em>Cucurbita moschata</em>) has become a widely adopted method during the chilling season for cultivation<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Different rootstock resources confer varying levels of chilling tolerance to the scion<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>, indicating the existence of long-distance signals from rootstocks to scions. Among these long-distance signals, e.g., nutrients and ROS, many influence crucial agricultural traits such as flavor and yield<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Therefore, uncovering the mechanism underlying these rootstock-to-scion signals in grafted cucumbers under chilling conditions holds significant importance. A recent study uncovered accumulation of jasmonates (JAs) in the scions to be crucial for chilling tolerance in heterografts of tomatoes and watermelons<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. JAs play crucial roles as phytohormones in promoting chilling tolerance in plants<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Chilling triggers a rapid increase in JA biosynthesis and activates JA signaling pathways. Application of JAs has been shown to enhance chilling tolerance in tomatoes and apples by positively regulating the ICE1-CBF pathway<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Particularly in cucumber, JA triggers degradation of JAZ5, thereby allowing direct ICE1 binding to the promoter of <em>CsCBF1</em><sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. In <em>planta</em>, JA-Ile, the most abundant active form of JA, is synthesized through the conjugation of JA with isoleucine, a reaction catalyzed by Jasmonic acid-amido synthetase 1 (JAR1). The highly induced endogenous JA-Ile is perceived by the JA receptor Coronatine Insensitive 1 (COI1), which interacts with Jasmonate Zim Domain proteins (JAZs), leading to their ubiquitination and subsequent degradation via the 26S proteasome pathway<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. This process activates downstream transcription factors and confers resistance to biotic and abiotic stresses in plants<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. However, the mechanism by which the rootstock leads to elevated levels of JAs in the grafted scions remains a subject of debate. Given that JAs cannot move from root to shoot<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, other components involved in JA biosynthesis are likely delivered from the pumpkin rootstock upon chilling stress.</p>
<p id="Par5">Increasing evidence highlights the critical role of mobile mRNAs as systemic signaling molecules between rootstock and scions<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, exemplified by various transcripts such as <em>CmNACP</em><sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>, <em>KNOTTED1</em><sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, <em>StBEL5</em><sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, <em>PFP-LET6</em><sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, <em>CmGAIP</em><sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup><em>, PEBP</em><sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup><em>, AtTCTP1</em><sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup><em>, SlPS</em><sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup><em>, HSP70</em><sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup><em>, PbWoxT1</em><sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, and <em>PbDRM</em><sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. These mobile mRNAs, akin to small RNAs<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, are predominantly transported from shoot to root via the phloem stream along with photosynthates<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, which can hardly explain the function of pumpkin as rootstocks. The specific role of mobile mRNAs in the exchange between pumpkin rootstocks and cucumber scions has been characterized on a large scale, revealing an association of mobile mRNAs with fatty acid and amino acid metabolism under chilling stress<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Nonetheless, the precise mechanisms by which fatty acid and amino acid metabolism contribute to chilling tolerance, as well as the potential key mRNAs provided by pumpkin rootstocks to enhance chilling tolerance in cucumber scions, are yet to be elucidated.</p>
<p id="Par6">In this study, we identified a specific mRNA <em>CmoKARI1</em>, which plays a crucial role in enhancing chilling stress tolerance in cucumber (Csa)/pumpkin (Cmo) heterografts. Interestingly, we found that pumpkin rootstock selectively transports <em>KARI1</em> to the cucumber scion in response to chilling stress, while cucumber is unable to transport its own <em>KARI1</em> upwards to the shoot. Through the overexpression of <em>CmoKARI1</em> in both cucumber and Arabidopsis, we unraveled the significance of this gene in conferring chilling tolerance. Notably, <em>KARI1</em> encodes an enzyme involved in the synthesis of branched-chain amino acids (BCAAs). Of the exogenously applied BCAAs, only isoleucine (Ile) was effective in enhancing chilling tolerance in cucumber. This led us to investigate the JA-Ile pathway. As a result of <em>CmoKARI1</em> overexpression, we observed a significant increase in both Ile and JA-Ile levels upon chilling stress, subsequently activating the JA-Ile signaling pathway. Furthermore, inhibition of JA-Ile synthesis compromised chilling tolerance in <em>CmoKARI1</em> overexpression lines. Our findings present the pioneering case of a root-to-shoot mobile mRNA triggered by specific environmental stimuli, offering valuable insights for future advancements in crop breeding and grafting techniques.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Integrated analysis of transcript and metabolite profiles reveals an important role of <em>CmoKARI1</em> in cucumber heterografts upon chilling stress</h3>
<p id="Par7">Pumpkin rootstock is widely employed to enhance stress tolerance in heterografts. To shed light on the essential mobile RNA signals transmitted from pumpkin (Cmo) rootstocks to cucumber (Csa) scions during chilling stress, we re-examined previously identified mobile transcript data from Cmo rootstock and Csa scion samples of reciprocally grafted seedlings subjected to chilling stress<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Venn analysis revealed that 3460 Csa transcripts and 99 Cmo transcripts were transported upon chilling stress (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>; Supplementary Dataset <a href="#MOESM3" class="usa-link">1</a>). Remarkably, among the 99 chilling-induced mobile Cmo transcripts, 80 transcripts were exclusively transported from Cmo rootstocks to Csa scions (Fig. <a href="#Fig1" class="usa-link">1a</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">1b, c</a>). KEGG enrichment analysis highlighted that these 80 transcripts were enriched in pathways related to the biosynthesis of branched-chain amino acids as well as other metabolic pathways (<em>p</em> = 0.092) (Fig. <a href="#Fig1" class="usa-link">1b</a>; Supplementary Dataset 2).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Analysis of mobile mRNAs between rootstock and scion under 6-hour chilling conditions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368190_41467_2025_63228_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74f8/12368190/40a376900d19/41467_2025_63228_Fig1_HTML.jpg" loading="lazy" id="d33e608" height="728" width="794" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Number and direction of mobile mRNAs exchanged between cucumber (Csa) scion and pumpkin (Cmo) rootstock under chilling stress. Move-down Csa mRNAs and move-up Cmo mRNAs are indicated by red letters. <strong>b</strong> KEGG enriched pathway analysis of 80 move-up Cmo mRNAs induced by cold stress. The red letters indicate the BCAA (Valine, leucine, and isoleucine) biosynthesis pathway, while the blue letters on the graphs represent the proportion of gene numbers and <em>p</em>-value. KEGG enrichment analyses were performed using Fisher’s exact test (<em>p</em> &lt; 0.05) and corrected for multiple comparisons by false discovery rate (FDR). <strong>c</strong> KEGG enriched pathway analysis of 174 CsaDIMs (differential intensity of metabolites) in the Csa scion of Csa/Cmo grafts under chilling stress compared to normal conditions. The red letters indicate the BCAA biosynthesis pathway, and the blue letters on the graphs represent the proportion of metabolite numbers and <em>P</em>-value. KEGG enrichment analyses were performed using Fisher’s exact test (<em>p</em> &lt; 0.05) and corrected for multiple comparisons by false discovery rate (FDR). <strong>d</strong> Normalized fold change of CsaDIMs in the Csa scion of Csa/Csa and Csa/Cmo under chilling stress compared to the same tissue under normal conditions. The square red-to-blue colored block indicates the normalized fold changes compared to Csa scion of Csa/Csa under normal conditions. <strong>e</strong> Networks of chilling-induced Cmo and Csa mobile mRNAs, Csa DEGs involved in the metabolites <span class="font-variant-small-caps">L</span><em>-</em>leucine, <span class="font-variant-small-caps">L</span>-valine, and <span class="font-variant-small-caps">L</span><em>-</em>isoleucine biosynthesis. Red circles indicate the four metabolites, grey ones indicate Csa DEGs, blue circles indicate Csa mobile mRNAs, and orange ones indicate Csa DEG-coordinated homologous Cmo mobile mRNAs. Arrows (red for positive and blue for negative) indicate the correlation between metabolites and Csa DEGs. The correlation analysis was conducted using Pearson correlation coefficients (|cor| ≥ 0.95, <em>P</em>-value &lt; 0.05, two-sided). Source data are provided as a Source Data file. <strong>f</strong> Schematic showing mRNA mobility in the BCAA biosynthesis pathway in Csa/Cmo grafts.</p></figcaption></figure><p id="Par8">To assess the metabolic consequences of the 80 mobile mRNAs, we examined the metabolomic data obtained from the same Csa tissues of heterografts<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. We identified 175 differentially induced metabolites (CsaDIMs) between Csa/Csa homograft and Csa/Cmo heterograft in response to chilling stress. 11 CsaDIMs were enriched in the BCAA biosynthesis pathway (ko00290) (<em>p</em> = 0.0000911) (Fig. <a href="#Fig1" class="usa-link">1c</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">2a, b</a>; Supplementary Dataset <a href="#MOESM3" class="usa-link">3</a> and <a href="#MOESM3" class="usa-link">4</a>). Notably, <span class="font-variant-small-caps">L</span>-Isoleucine, <span class="font-variant-small-caps">L</span>-Leucine, and (R)−2-Methylmalate exhibited substantial accumulation in the leaves of cucumber in Csa/Cmo heterografts under chilling conditions (Fig. <a href="#Fig1" class="usa-link">1d</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">2c</a>).</p>
<p id="Par9">To explore the relationships between BCAA-related CsaDIMs and cold-induced CsaDEGs, a correlation analysis was conducted. Remarkably, the 11 BCAA-related CsaDIMs in Csa leaves of Csa/Cmo grafts showed significant correlations with 361 cold-induced CsaDEGs (Person |cor| ≥ 0.95, <em>p</em>-value &lt; 0.05) (<a href="#MOESM3" class="usa-link">Supplementary Data 5</a>; <a href="#MOESM3" class="usa-link">Supplementary Data 6</a>). Utilizing Cytoscape based on the Pearson correlation coefficients, we observed a notable correlation between four key metabolites (<span class="font-variant-small-caps">L</span>-Isoleucine, <span class="font-variant-small-caps">L</span>-Leucine, <span class="font-variant-small-caps">L</span>-Valine, (R)−2,3-Dihydroxy-3-methylpentanoate) and three mobile Cmo mRNAs (<em>CmoCRLPK2</em>, CmoCh05G000400; <em>CmoEEIG1/EHBP1</em>, CmoCh06G005400; <em>CmoKARI1</em>, CmoCh19G010620) in cucumber leaves of the Csa/Cmo heterografts in response to chilling stress (Fig. <a href="#Fig1" class="usa-link">1e</a>; <a href="#MOESM3" class="usa-link">Supplementary Data 7</a>). Among them, <em>CmoKARI1</em> encodes an enzyme (ilvC, EC: 1.1.1.86) directly involved in the second step of parallel pathways leading to BCAA biosynthesis.</p>
<p id="Par10">Our findings suggest that BCAA biosynthesis is likely a key metabolic pathway enriched with cold-induced directionally mobile mRNAs. Under normal conditions in Csa/Cmo heterografts, several cucumber mRNAs, including <em>CsaKARI, CsaIIL1/IPMI1/2</em>, <em>CsaAHAS1/CSR1</em>, <em>CsaDHAD</em>, <em>CsaIMS1</em> and <em>CsaTD1</em>, move from the Csa scion to the Cmo rootstock. Under cold conditions in Csa/Cmo heterografts, <em>CsaKARI, CsaIIL1/IPMI1/2</em>, <em>CsaAHAS1/CSR1</em>, <em>CsaDHAD</em>, <em>CsaIMD</em>, and <em>CsaTD1</em> move from the Csa scion to the Cmo rootstock (Fig. <a href="#Fig1" class="usa-link">1f</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">3a, b</a>; <a href="#MOESM3" class="usa-link">Supplementary Data 8</a>). To better understand the role of pumpkin as a rootstock in enhancing stress tolerance across diverse scion species, it is essential to focus on signals that originate directly from the pumpkin root. A criterion for such a signal is that it must be both transported to the scion and, ideally, responsive to environmental factors. Among the components of the BCAA pathway, CmoKARI1 is the only factor that meets these criteria. In contrast, although the other two genes also showed correlation, there are currently no reports suggesting a direct functional role (Supplementary Fig. <a href="#MOESM1" class="usa-link">3b</a>). Therefore, we prioritized the study of <em>CmoKARI1</em> in pumpkin-mediated chilling tolerance.</p></section><section id="Sec4"><h3 class="pmc_sec_title">
<em>CmoKARI1</em> mRNA, but not <em>CsaKARI1</em>, is transported from root to shoot upon chilling</h3>
<p id="Par11">The mobility of these identified mRNAs was validated using an established RT-PCR identification method for mRNA mobility in heterologous grafts<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> (Fig. <a href="#Fig2" class="usa-link">2a</a>). To further validate the long-distance mobility of <em>CmoKARI1</em> in cucumbers, we cloned and constructed the full-length cDNA of <em>CmoKARI1</em> into an overexpression (OE) binary vector fused with GFP. These constructs were introduced into cucumber using the established hairy roots system<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. RT-PCR analysis of the newly emerged leaves confirmed that <em>CmoKARI1</em> triggered the movement of <em>GFP</em> mRNA from transgenic roots to untransgenic new leaves under low temperature (Fig. <a href="#Fig2" class="usa-link">2b</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. <em>CmoKARI1</em>, but not <em>CsaKARI</em>, exhibits upward movement and is translated by the cucumber scion in response to chilling stress.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368190_41467_2025_63228_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74f8/12368190/2eb1b07ad7b5/41467_2025_63228_Fig2_HTML.jpg" loading="lazy" id="d33e845" height="669" width="797" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> RT-PCR identification of Csa and Cmo mRNA mobility under chilling conditions. Asterisks indicate mobile mRNAs identified in heterologous tissues. Arrows indicate movement direction. Cucumber or pumpkin <em>ACTIN7</em> was used as an internal reference for mRNA abundance. Lower bands that appeared were primer dimers serving as sample loading controls. See uncropped images in the source file. The samples derive from the same experiment, and the gels were processed in parallel. <strong>b</strong> Identification of <em>CmoKARI1</em> mobility from transformed cucumber hairy root to new leaves. The hairy root was transformed with cultures of <em>Agrobacterium rhizogenes</em> (Rhizobium rhizogenes) strain K599 harboring an empty vector (pCambia1300-Super::GFP) or pCambia1300-Super::CmoKARI1-GFP. K599 alone was transformed into a mock control. Fourteen days after transformation, RT-PCR with primers recognizing GFP was used to verify the mobility of <em>GFP</em> or <em>CmoKARI1-GFP</em>. #1, #2, and #3 represent independent transgenic seedlings as biological replicates. Mobility rate was analyzed by mobility seedlings/total independent hairy root transgenic seedlings. <em>CsaACTIN7</em> was used as an internal reference for mRNA abundance. <em>NPTII</em> was used to exclude the <em>Agrobacterium rhizogenes</em> contamination or mobility. GFP fluorescence was identified using the LUYOR-3415RG fluorescent protein lamp. The samples derive from the same experiment, and the gels were processed in parallel. <strong>c</strong> Mobile <em>CmoKARI1</em> from transformed cucumber hairy root was translated into new true leaves, shown by GUS staining. The hairy root was transformed with <em>Agrobacterium rhizogenes</em> (Rhizobium rhizogenes) strain K599 harboring an empty vector (pCambia1305-35S::GUS) or pCambia1305-35S::CmoKARI1-GUS. R: Agrobacterium-transformed cucumber hairy root, L: The emerged new leaves at 14 days after transformation. <strong>d</strong> RT-PCR with primers recognizing GUS was used to verify the mobility of <em>GUS</em> or <em>CmoKARI1-GUS</em>. #1, #2, and #3 represent independent transgenic seedlings as biological replicates. Mobility rate was analyzed by mobility seedlings/total independent hairy root transgenic seedlings. <em>CsaACTIN7</em> was used as an internal reference for mRNA abundance. The hygromycin resistance (Hyg R) gene was used to exclude <em>Agrobacterium rhizogenes</em> contamination or mobility. The samples derive from the same experiment, and the gels were processed in parallel. <strong>e</strong> RT-PCR identification of <em>CsaKARI</em> from cucumber rootstock to pumpkin scion at 6 h, 12 h, and 24 h under chilling conditions. R rootstock, L true leaves. The samples derive from the same experiment, and the gels were processed in parallel. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par12">One critical question is whether, after the rootstock-to-scion transport of <em>CmoKARI1</em>, it can be translated in the destination tissues. Since the GUS protein exceeds the size of plasmadesmata and cannot be transported<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, we employed <em>CmoKARI1</em> fused with GUS reporter to exclude the possibility of protein transport. Therefore, GUS staining can be used as an indication of local translation of imported mRNAs. By performing root transformation, we introduced <em>CmoKARI1</em> fused to a GUS reporter into cucumber roots. GUS staining indicated that the transported <em>CmoKARI1</em> mRNA could be translated into a functional protein in the scions upon chilling (Fig. <a href="#Fig2" class="usa-link">2c, d</a>).</p>
<p id="Par13">Interestingly, in the pumpkin scions grafted onto cucumber rootstocks, <em>CsaKARI</em> mRNA was unable to move to the scions upon chilling (Fig. <a href="#Fig2" class="usa-link">2e</a>). We further characterized <em>KARI</em> genes in cucumber and pumpkin genomes. <em>CsaKARI</em> is a single-copy gene, and additional <em>CmoKARI2</em> (CmoCh11G019640) was identified (Supplementary Fig. <a href="#MOESM1" class="usa-link">4a, b</a>). They shared a high similarity of protein sequences, indicating their conserved catalytic activity. However, the transcript level of <em>CsaKARI</em> and <em>CmoKARI2</em> was not regulated by low temperature or heterografting (Supplementary Fig. <a href="#MOESM1" class="usa-link">3b</a>). In addition, <em>CmoKARI2</em> was not mobile in Csa/Cmo heterografts upon chilling (<a href="#MOESM3" class="usa-link">Supplementary Data 1</a>). These findings suggest that the root-to-shoot mobility of <em>CmoKARI1</em> mRNA contributes specifically to the chilling tolerance of pumpkin as rootstocks, and excludes the potential role of <em>CsaKARI</em> and <em>CmoKARI2</em> in improving chilling tolerance of Csa/Cmo heterografts.</p></section><section id="Sec5"><h3 class="pmc_sec_title">The root-to-shoot mobility of <em>CmoKARI1</em> is specifically associated with chilling-tolerant rootstocks derived from various pumpkin varieties</h3>
<p id="Par14">To investigate the impact of <em>CmoKARI1</em> long-distance movement on chilling responses and tolerance in grafted cucumbers, we used three different pumpkin varieties, including Figleaf gourd, ‘Qianglishi,’ and ‘Lizhiyuan,’ as rootstocks. All three pumpkin varieties as rootstocks significantly improved chilling tolerance in cucumber scions<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. In our experimental system, we conducted a comprehensive analysis to assess the role of these three pumpkin rootstocks under chilling stress. Our findings categorized Figleaf gourd and ‘Qianglishi’ as chilling-tolerant rootstocks, while ‘Lizhiyuan’ exhibited lower chilling tolerance (Fig. <a href="#Fig3" class="usa-link">3a</a>). Notably, the Csa/‘Lizhiyuan’ grafts displayed wilting and dehydration after 6 hours of chilling stress, whereas wilting was observed in Csa/‘Qianglishi’ and Csa/Figleaf gourd grafts after 3 days, respectively. Chilling Injury Index (CII), Relative Electrolyte Permeability (REP), and the expression of genes from the CBF-COR pathway confirmed comparable chilling tolerance in Csa/Figleaf gourd and Csa/‘Qianglishi’ grafts, with lower tolerance observed in Csa/‘Lizhiyuan’ grafts (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. <em>CmoKARI1</em> moves from pumpkin root to cucumber shoot in response to chilling stress.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368190_41467_2025_63228_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74f8/12368190/fc3d2f10390f/41467_2025_63228_Fig3_HTML.jpg" loading="lazy" id="d33e982" height="403" width="798" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Phenotypic changes and <strong>b, c</strong> relative expression of <em>CmoKARI1</em> and <em>CsaKARI</em> in the leaf and root of grafted Csa/Cmo (Figleaf gourd, ‘Qianglishi’, and ‘Lizhiyuan’) under different time-course cold stress by qPCR. Each treatment includes <em>n</em> = 3 biological replicates, and each replicate includes 5 individual plants (mean ± s.d., one-way ANOVA followed by Duncan’s test). Lowercase letters indicate significant differences (<em>p</em> &lt; 0.05). Cucumber <em>ACTIN7</em> was used as an internal reference. <strong>d</strong> RT-PCR identification of <em>CmoKARI1</em> in grafted Csa/Cmo with pumpkin rootstock ‘Lizhiyuan’, Figleaf gourd and ‘Qianglishi’ at 6 h, 12 h, and 24 h under chilling conditions. R pumpkin rootstock, C cucumber cotyledons, L cucumber true leaves. The normalized relative expression of RT-PCR bands was analyzed by comparing with the corresponding <em>Csa/CmoACTIN7</em> using ImageJ software. Each treatment includes <em>n</em> = 3 biological replicates, and each replicate includes 5 individual plants (mean ± s.d.). Asterisks indicate mobile <em>CmoKARI1</em> in cucumber cotyledons and true leaves. The samples derive from the same experiment, and the gels were processed in parallel. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par15">The expression analysis showed that the relative expression of <em>CmoKARI1</em> was induced in the chilling-tolerant pumpkin rootstocks, Figleaf gourd, and ‘Qianglishi,’ after 12 h and 1 day of chilling stress (one-way ANOVA followed by Duncan’s test, <em>p</em> &lt; 0.05). The mobility of <em>CmoKARI1</em> mRNA appears to be associated with the induced mRNA pool in the roots of chilling-tolerant pumpkin varieties. However, the relative expression of <em>CsaKARI</em> (CsGy7G004580) in cucumber scions was not influenced by different pumpkin rootstocks and only exhibited slight induction after 12 hours and 1 day of chilling stress (Fig. <a href="#Fig3" class="usa-link">3b, c</a>). Since KARI controls BCAA biosynthesis, silencing the single-copy <em>CsaKARI</em> resulted in dwarf, chlorotic, and stunted cucumber seedlings, unsuitable for any further grafting or stress challenge (Supplementary Fig. <a href="#MOESM1" class="usa-link">6a, d</a>). Both REP and MDA levels indicated a markedly higher sensitivity to chilling stress in the <em>CsaKARI</em>-silenced line (Supplementary Fig. <a href="#MOESM1" class="usa-link">6b, e</a>). Due to the strong leaf chlorosis and waterlogging phenotype, the reference gene may no longer be reliable. This could explain the unexpected accumulation of <em>CsaCOR</em> and <em>CsaKARI</em> mRNAs following chilling stress in the silenced line (Supplementary Fig. <a href="#MOESM1" class="usa-link">6c, f</a>). These findings suggest that the non-cold-inducible <em>CsaKARI</em> likely provides a housekeeping function, while the chilling-inducible <em>CmoKARI1</em> from the rootstock appears to be required for improved chilling tolerance in the heterograft.</p>
<p id="Par16">To confirm the movement of <em>CmoKARI1</em> mRNA to the cucumber scions, we conducted RT-PCR identification. We observed that <em>CmoKARI1</em> mRNA was transported from all rootstocks to the 1st–2nd true leaves and particularly the new leaves of cucumber scions after 6 h and 12 h of chilling stress. However, after 24 h of chilling stress, <em>CmoKARI1</em> mRNA was only detected in the scions of chilling-tolerant Figleaf gourd and ‘Qianglishi’ rootstocks. The chilling-sensitive ‘Lizhiyuan’ rootstock was no longer able to deliver <em>CmoKARI1</em> mRNA to the cucumber scions (Fig. <a href="#Fig3" class="usa-link">3d</a>). Considering <em>CsaKARI</em> as a housekeeping system and <em>CmoKARI1</em> as the inducible one, an interesting question arises: what is the relative amount of <em>CmoKARI1</em> mRNA transported to the cucumber scion during chilling stress compared to <em>CsaKARI</em> mRNA transcribed in the scion? We re-analyzed the original RNA-seq data and found that after 6 h of chilling stress, the amount of transported <em>CmoKARI1</em> mRNA reads in the scion was approximately 10% of the total <em>CsaKARI</em> transcripts (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>; <a href="#MOESM3" class="usa-link">Supplementary Data 9</a>). Importantly, transport of <em>CmoKARI1</em> mRNA increased and was even detectable after 36 h of chilling stress, the latest time point we could analyze before plant health declined (one-way ANOVA followed by Duncan’s test, <em>p</em> &lt; 0.05) (Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>). This suggests that <em>CmoKARI1</em> mRNA contributes significantly to the scion’s BCAA pool alongside the scion’s own <em>CsaKARI</em> expression.</p></section><section id="Sec6"><h3 class="pmc_sec_title">The overexpression of <em>CmoKARI1</em> improves chilling tolerance of transgenic cucumber and <em>Arabidopsis thaliana</em>
</h3>
<p id="Par17"><em>CmoKARI1</em> knockout pumpkin rootstock would have allowed us to conclusively establish the essential role of mobile <em>CmoKARI1</em> in conferring cold tolerance to Csa/Cmo heterografts. However, pumpkin transformation is a technically demanding process with worldwide obstacles. Consequently, we employed root-transformation to introduce CmoKARI1-GFP/GUS into cucumber roots (Fig. <a href="#Fig2" class="usa-link">2</a>), and evaluated its response to chilling stress. Notably, the overexpression of CmoKARI1–GFP/GUS in cucumber roots resulted in improved chilling tolerance after 12 h of chilling stress, as indicated by relative electrolyte permeability (REP) and malondialdehyde (MDA) content (Fig. <a href="#Fig4" class="usa-link">4a, b</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>). Particularly, CmoKARI1–GUS protein is not capable of moving long-distance due to the size limitation of plasmodesmata<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. These results further indicate that over-expressing <em>CmoKARI1</em> mRNA alone in the cucumber root is enough to confer chilling tolerance (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>), excluding the possible involvement of protein transport.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Move-up <em>CmoKARI1</em> improves cucumber chilling tolerance by inducing isoleucine biosynthesis.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368190_41467_2025_63228_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74f8/12368190/da95ac62e9dc/41467_2025_63228_Fig4_HTML.jpg" loading="lazy" id="d33e1220" height="763" width="799" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Phenotypes, and <strong>b</strong> relative electrolyte permeability (REP) and MDA content of <em>CmoKARI1-GFP</em> root-transformed cucumbers after 12 h of 4 °C cold stress. <strong>a</strong>, <strong>b</strong> Each treatment includes 5 individual plants as <em>n</em> = 5 biological replicates (mean ± s.d., one-way ANOVA followed by Duncan’s test). Lowercase letters indicate significant differences (<em>p</em> &lt; 0.05). <strong>c</strong> Phenotypes, and <strong>d</strong> REP and MDA in two-leaf-old cucumber seedlings treated with 300 μM <span class="font-variant-small-caps">L</span>-isoleucine before and after 12 hours of chilling conditions. The control group received H<sub>2</sub>O. <strong>c</strong>, <strong>d</strong> At least <em>n</em> = 3 replicate pools were included, with each pool containing 12 plants, at both 0-h and 12-h chilling conditions (mean ± s.d., one-way ANOVA followed by Duncan’s test). Lowercase letters on the graphs indicate significant differences (<em>p</em> &lt; 0.05). <strong>e</strong> Phenotypes, <strong>f</strong> REP and MDA content, <strong>g</strong> isoleucine content, and <strong>h</strong> Relative expression of <em>CsaCBF1</em> and <em>CsaCOR</em> in two-leaf-old Super::CmoKARI1-GFP transgenic (TG) lines (TG#2 and TG#4) and wild-type cucumber seedlings before and after 12 h of chilling conditions. Cucumber <em>ACTIN7</em> was used as an internal reference. The control group received H<sub>2</sub>O. Each treatment includes <em>n</em> = 3 (<strong>f</strong>, <strong>h</strong>), <em>n </em>= 5 (<strong>g</strong>) biological replicates, and each replicate includes 12 individual plants (mean ± s.d.), for each condition (one-way ANOVA followed by Duncan’s test for the 28 °C and 4 °C groups). Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par18"><em>KARI1</em> encodes an enzyme involved in the BCAA biosynthesis. To examine the significance of CmoKARI1-induced BCAA biosynthesis in the chilling tolerance of heterografts, we applied different concentrations of <span class="font-variant-small-caps">L</span>-Leucine, <span class="font-variant-small-caps">L</span>-Isoleucine, and <span class="font-variant-small-caps">L</span>-Valine to the leaves of two-true-leaf-old cucumber seedlings (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>). Following 12 h of cold treatment, it was observed that only 300 μM <span class="font-variant-small-caps">L</span>-Isoleucine enhanced the cold tolerance of cucumber seedlings, as indicated by REP and MDA content (one-way ANOVA followed by Duncan’s test, <em>p</em> &lt; 0.05) (Fig. <a href="#Fig4" class="usa-link">4c, d</a>). Interestingly, the application of <span class="font-variant-small-caps">L</span>-Valine increased susceptibility to chilling, which aligned with the specific elevation of leucine and isoleucine levels, but not <span class="font-variant-small-caps">L</span>-Valine, in the Csa/Cmo heterografts under chilling conditions (Supplementary Fig. <a href="#MOESM1" class="usa-link">2c</a>). These findings suggest the potential of <span class="font-variant-small-caps">L</span>-Isoleucine application to modulate early chilling tolerance in cucumber, warranting further investigation.</p>
<p id="Par19">To establish the relevance of <em>CmoKARI1</em> mobility and its impact on isoleucine biosynthesis in chilling tolerance, we introduced the constitutive expression of <em>CmoKARI1-GFP</em> into cucumber through stable transformation. Three independent lines with <em>CmoKARI1</em> transgenic lines (TG #1, #2 and #4) were selected based on genomic identification and transcription levels of <em>CmoKARI1</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>). After subjecting the cucumber plants to 12 h of 4 °C chilling stress, it was evident that overexpressed <em>CmoKARI1</em> significantly enhanced cucumber’s chilling tolerance, and the exogenous application of <span class="font-variant-small-caps">L</span>-isoleucine exhibited additive effects. The levels of MDA and REP in the <em>CmoKARI1</em> TG lines were lower than in the wild type, indicating improved chilling tolerance in both the <em>CmoKARI1</em> TG lines and through <span class="font-variant-small-caps">L</span>-isoleucine application (one-way ANOVA followed by Duncan’s test, <em>p</em> &lt; 0.05) (Fig. <a href="#Fig4" class="usa-link">4e, f</a>). Overexpression of <em>CmoKARI1</em> led to an increase in the endogenous isoleucine level, which correlated with the observed chilling tolerance phenotype (Fig. <a href="#Fig4" class="usa-link">4g</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">11a–i</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">12a–e</a>). Additionally, the expression levels of <em>CsaCBF1</em> and <em>CsaCOR</em>, two key genes involved in cold response, were higher in the <em>CmoKARI1</em> TG lines compared to the wild type after 6 h of chilling stress (one-way ANOVA followed by Duncan’s test, <em>p</em> &lt; 0.05) (Supplementary Fig. <a href="#MOESM1" class="usa-link">12f</a>). Treatment with <span class="font-variant-small-caps">L</span>-isoleucine further elevated the expression levels of <em>CsaCBF1</em> and <em>CsaCOR</em> in the <em>CmoKARI1</em> TG lines compared to the wild type and H<sub>2</sub>O-treated TG seedlings (Fig. <a href="#Fig4" class="usa-link">4h</a>). Furthermore, heterografs of WT and <em>CmoKARI1</em> TG cucumber lines exhibited stronger chilling tolerance compared to the self-rooted graft (Fig. <a href="#Fig5" class="usa-link">5a–c</a>), with <em>CmoKARI1</em> moving from <em>CmoKARI1</em> TG rootstock into WT scion (Fig. <a href="#Fig5" class="usa-link">5d</a>). These results collectively demonstrate that the overexpression of <em>CmoKARI1</em> in cucumber root and further its mobility to the shoot enhances cold tolerance in cucumber seedlings by promoting <span class="font-variant-small-caps">L</span>-isoleucine biosynthesis.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. The rootstock-to-scion mobility of <em>CmoKARI1</em> mRNA confers cucumber chilling tolerance.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368190_41467_2025_63228_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74f8/12368190/d68a22fef346/41467_2025_63228_Fig5_HTML.jpg" loading="lazy" id="d33e1409" height="601" width="796" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The two-leaf-old wild-type cucumber seedlings grafted onto Super::CmoKARI1-GFP transgenic (TG) lines (TG#2 and #4) and wild-type cucumber seedlings transgenic lines. The lower panel image showed the graft union. Each graft combination includes <em>n</em> = 9 individual grafted plants. <strong>b</strong> Phenotypes of the two-leaf-old wild-type cucumber seedlings grafted onto Super::CmoKARI1-GFP transgenic lines (TG) (TG #2 and #4) and wild-type cucumber rootstocks before and after 6 h of chilling conditions. <strong>c</strong> Relative electrolyte permeability (REP) and MDA content of wild-type grafted onto TG #2/#4 and wild-type cucumber rootstocks at 6 h of chilling conditions. Each treatment includes three biological replicates, with 3–4 individual plants per replicate (mean ± s.d., one-way ANOVA followed by Duncan’s test, <em>p</em> &lt; 0.05). Significance analysis of REP and MDA content was performed in separate groups at 28 °C and 4 °C. <strong>d</strong> RT-PCR identification of <em>CmoKARI1-GFP</em> in grafted WT/WT, WT/TG#2 and WT/TG#4 before and 6 h chilling conditions. R Rootstock, L Cucumber wild-type scion, Asterisks indicate movement fusion mRNA of <em>CmoKARI1-GFP</em>. <em>n</em> = 3 replicates were included, with each containing nine individual plants. Cucumber <em>ACTIN7</em> was used as an internal reference. The samples derive from the same experiment, and the gels were processed in parallel. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par20">To explore the broader effects of <em>CmoKARI1</em> and the potential application of <span class="font-variant-small-caps">L</span>-isoleucine in improving cold tolerance in other plants, we also introduced <em>CmoKARI1</em> into <em>Arabidopsis</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). Wild-type <em>Arabidopsis</em> Col-0 and two independent lines overexpressing <em>CmoKARI1</em> were treated with 50 μM and 100 μM <span class="font-variant-small-caps">L</span>-isoleucine (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). Since Arabidopsis is far more cold-tolerant than cucumber, here we challenged Arabidopsis with freezing conditions. Although chilling and freezing stress are rather different, it is believed that all freezing-tolerant plants are also chilling-tolerant<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. In the two independent transgenic <em>Arabidopsis</em> lines overexpressing <em>CmoKARI1</em> and supplied with <span class="font-variant-small-caps">L</span>-isoleucine (Supplementary Fig. <a href="#MOESM1" class="usa-link">13a–d</a>), REP analysis revealed increased tolerance compared to the mock control (one-way ANOVA followed by Duncan’s test, <em>p</em> &lt; 0.05) (Supplementary Fig. <a href="#MOESM1" class="usa-link">13e</a>). Moreover, overexpression of <em>CmoKARI1</em> increased the content of <span class="font-variant-small-caps">L</span>-isoleucine in transgenic Arabidopsis lines compared to the wild type after 4 days of recovery following freezing stress (Supplementary Fig. <a href="#MOESM1" class="usa-link">13f</a>). Additionally, elevated <em>JAZ1</em> expression indicated activation of the JA signaling pathway, consistent with the increased Ile levels (Supplementary Fig. <a href="#MOESM1" class="usa-link">13f</a>). Gene expression analysis in the Arabidopsis lines treated with 50 μM <span class="font-variant-small-caps">L</span>-isoleucine showed similar results to cucumber seedlings, with higher expression levels of <em>AtCBF1</em> and <em>AtCOR15b</em> in the <em>CmoKARI1</em>ox lines compared to the wild type after chilling stress. Furthermore, when <span class="font-variant-small-caps">L</span>-isoleucine was applied to the Arabidopsis lines, the expression levels of <em>AtCBF1</em> and <em>AtCOR15b</em> were even higher in the <em>CmoKARI1</em>ox lines compared to the wild type and H<sub>2</sub>O-treated <em>CmoKARI1</em>ox seedlings (one-way ANOVA followed by Duncan’s test, <em>p</em> &lt; 0.05) (Supplementary Fig. <a href="#MOESM1" class="usa-link">13g</a>). These findings suggest that the function of overexpressed <em>CmoKARI1</em> in regulating isoleucine biosynthesis and influencing cold tolerance is conserved in both cucumber and <em>Arabidopsis</em>.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Root-to-shoot mobile <em>CmoKARI1</em> activates JA-Ile signaling</h3>
<p id="Par21">Different from valine and leucine, isoleucine plays a crucial role in synthesizing JA-Ile conjugates. It has been shown that Ile enhances abiotic stress resistance via the JA signaling pathway<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Therefore, we hypothesize that the delivery of <em>CmoKARI1</em> from the pumpkin rootstock to the cucumber scions during chilling stress enhances chilling tolerance by promoting the biosynthesis of isoleucine and JA-Ile, thereby activating JA-Ile signaling. After subjecting the plants to 12 h of chilling stress, the H<sub>2</sub>O-treated <em>CmoKARI1</em> TG line exhibited greater cold tolerance than the wild type. Remarkably, the application of JA-Ile demonstrated an effect on chilling tolerance compared to JA or isoleucine treatment alone, as evidenced by both the phenotype and physiological parameters (one-way ANOVA followed by Duncan’s test, <em>p</em> &lt; 0.05) (Fig. <a href="#Fig6" class="usa-link">6a–c</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Move-up <em>CmoKARI1</em> improves cucumber chilling tolerance by specifically inducing the JA-Ile signaling pathway.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368190_41467_2025_63228_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74f8/12368190/7f9faaee54d0/41467_2025_63228_Fig6_HTML.jpg" loading="lazy" id="d33e1630" height="992" width="745" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Phenotypes, <strong>b</strong> Relative electrolyte permeability (REP), and <strong>c</strong> MDA content in two-leaf-old Super::CmoKARI1-GFP transgenic (TG) and wild-type cucumber seedlings treated with H<sub>2</sub>O control, 300 μM Ile, 100 μM JA, 200 μM MeJA, 100 μM JA-Ile, and 200 μM Jarin-1 before and after 12 h of chilling conditions. H<sub>2</sub>O was used as the control. <em>n</em> = 4 replicate pools were included, with each pool containing 12 plants, for each treatment (mean ± s.d., one-way ANOVA followed by Duncan’s multiple comparisons test within each genotype, <em>p</em> &lt; 0.05; <em>p</em>-values at the top indicating the statistical difference between genotypes within each treatment group determined by one-way ANOVA). <strong>d</strong> Relative gene expression levels of <em>CsaCBF1</em> and <em>CsaJAR1</em> in the first true leaf under different growth conditions. Each treatment includes <em>n</em> = 3 biological replicates, with 12 individual plants per replicate (mean ± s.d., one-way ANOVA followed by Duncan’s multiple comparisons test within each genotype, <em>p</em> &lt; 0.05; <em>p-</em>values at the top indicating the statistical difference between WT and TG within each treatment group determined by Student <em>t</em>-test). Cucumber <em>ACTIN7</em> was used as an internal reference. <strong>e</strong> Content of JA, MeJA, JA-Ile, and OPDA was determined in wild-type (WT) and Super::CmoKARI1-GFP transgenic (TG#4) under 28 °C and 4 °C conditions. Each treatment includes <em>n</em> = 3 biological replicates, with 4 individual plants per replicate (mean ± s.d., one-way ANOVA followed by Duncan’s test, <em>p</em> &lt; 0.05). <strong>f</strong> Schematic model depicting the improvement of cold tolerance in cucumber scion under cold stress through the pumpkin rootstock-derived mobile <em>CmoKARI1</em>. The long-distance movement of <em>CmoKARI1</em> accelerates isoleucine biosynthesis, providing substrates for JA-Ile biosynthesis controlled by <em>CsaJAR1</em>. The accumulation of JA-Ile activates the CBF-COR pathway, thereby enhancing chilling tolerance in cucumber. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par22">JA-Ile is synthesized by Jasmonoyl-<span class="font-variant-small-caps">L</span>-amino acid synthetase 1 (JAR1)<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. The JAR1 inhibitor Jarin-1 impaired chilling tolerance, particularly in the <em>CmoKARI1</em> TG lines (Fig. <a href="#Fig6" class="usa-link">6a–c</a>). The expression level of <em>CsaCBF1</em> showed the highest induction upon JA-Ile treatment in the <em>CmoKARI1</em> TG lines, and this induction was abolished by Jarin-1 treatment (Fig. <a href="#Fig6" class="usa-link">6d</a>). After 3-day chilling stress, JA-Ile application enhanced chilling tolerance in CmoKARI-GUS root transformed cucumber. In the presence of two JA-Ile biosynthetic substrates, JA and Ile, Jarin-1 impaired chilling tolerance and <em>CsaCBF1</em> expression (one-way ANOVA followed by Duncan’s test, <em>p</em> &lt; 0.05) (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>). Further genetic evidence by silencing <em>JAR1</em> in the <em>CmoKARI1</em> TG lines under 3-day chilling conditions further supported the biosynthesis of JA-Ile determined the <em>CmoKARI1</em>-mediated chilling tolerance (Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>). Notably, isoleucine treatment increased the expression of <em>CsaJAR1</em>, suggesting an enhanced JA-Ile synthesis in response to the availability of biosynthetic precursors (Fig. <a href="#Fig6" class="usa-link">6d</a>).</p>
<p id="Par23">While JA can be used to synthesize either JA-Ile or MeJA, our findings indicate that MeJA’s effect on chilling tolerance is much less significant compared to JA-Ile (Fig. <a href="#Fig6" class="usa-link">6e</a>). Intriguingly, the <em>CmoKARI1</em> TG line treated with MeJA exhibited increased sensitivity compared to the MeJA-treated wild type or H<sub>2</sub>O-treated <em>CmoKARI1</em> TG cucumber lines (Fig. <a href="#Fig6" class="usa-link">6a</a>). This result highlights the specific role of JA-Ile, rather than other forms of JAs, in cucumber’s chilling tolerance. Consistently, MeJA levels decreased in the wild type upon cold treatment, while JA-Ile levels, especially in the <em>CmoKARI1</em> TG line, increased (Fig. <a href="#Fig6" class="usa-link">6e</a>). We further confirmed elevated levels of both Ile and JA-Ile in Csa/Cmo heterografts compared to control self-rooted grafts (Supplementary Fig. <a href="#MOESM1" class="usa-link">16a</a>), and in <em>CmoKARI1</em> TG cucumber compared with WT (Supplementary Fig. <a href="#MOESM1" class="usa-link">12c, d</a>). Additionally, both heterografts and <em>CmoKARI1</em> TG lines exhibited increased transcript levels of the JA-signaling genes <em>JAZs</em>, <em>MYC2</em><sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>, and <em>CBF1</em><sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">12e, f</a>).</p>
<p id="Par24">In conclusion, our findings suggest that <em>CmoKARI1</em> mRNA is specifically delivered from pumpkin rootstocks to cucumber scions in response to low temperatures. Once translated, CmoKARI1 participates in isoleucine biosynthesis, which further promotes the biosynthesis of JA-Ile and activates JA-Ile signaling to cope with chilling stress (Fig. <a href="#Fig6" class="usa-link">6f</a>).</p>
<p id="Par25">Although mobile <em>CmoKARI1</em> significantly enhances chilling tolerance in heterografts, it is essential to distinguish the effects of CmoKARI1-mediated local Ile biosynthesis from potential Ile transport from pumpkin root to cucumber scions. To address this, we treated Csa/Cmo heterografts with FITC-labeled Ile and measured the relative signal intensities. Pumpkin root showed a greater uptake and transport capacity of Ile and Leu compared to cucumber root; however, the rate of Ile and Leu transport decreased to almost the background level under chilling stress (Fig. <a href="#Fig7" class="usa-link">7a–c</a>). These findings suggest that the increased Ile levels observed in <em>CmoKARI1</em> TG lines and Csa/Cmo heterografts likely result from CmoKARI1-mediated local Ile biosynthesis.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. Uptake and transport of FITC-Isoleucine (Ile) and FITC-Leucine (Leu) from pumpkin (Cmo) rootstock to cucumber (Csa) scion under chilling stress.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368190_41467_2025_63228_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74f8/12368190/66cce19aa7a0/41467_2025_63228_Fig7_HTML.jpg" loading="lazy" id="d33e1781" height="1250" width="731" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Bright-field and fluorescence images of grafted plants, roots, stems, and petioles of Csa/Csa and Csa/Cmo plants after root irrigation with 50 μM FITC-Ile and FITC-Leu under normal (28 °C) and cold stress (4 °C for 12 h) conditions. H<sub>2</sub>O was used as the control. BF bright-field images, FITC fluorescence images of FITC-Ile and FITC-Leu (channel RGB = 0:18:0) manually extracted using ImageJ, UV fluorescence detected using a LUYOR-3415RG fluorescent protein lamp. Fluorescence images of roots and vascular bundle sheath (VBS) cells in stem cross-sections and petioles were captured using a confocal laser scanning microscope (TCSSP8, Leica) with an excitation wavelength of 488 nm. Three individual plants were analyzed per treatment. <strong>b</strong>, <strong>c</strong> Statistical analysis of FITC fluorescence intensity per pixel, quantified using ImageJ software. Each treatment includes <em>n</em> = 3 biological replicates, with 5 individual plants per replicate (mean ± s.d.). Statistical significance was determined by one-way ANOVA followed by Duncan’s multiple comparisons test with the wild type as the control (<em>p</em> &lt; 0.05). Source data are provided as a Source Data file.</p></figcaption></figure></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par26">Pumpkin is extensively used as a rootstock for grafting cucurbit crops, with the aim of enhancing yield and stress tolerance in the scions. Pumpkin rootstocks induce various phenotypic and physiological changes in the scions, including the regulation of nutrient homeostasis, alteration of endogenous hormones, and maintenance of photosynthesis and antioxidant capacity<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. However, there is a lack of comprehensive research on the mobile root-borne signals, and the underlying mechanism remains largely unknown. In this study, we made an important discovery by identifying a specific mRNA molecule, <em>CmoKARI1</em>, which is transported exclusively from the pumpkin rootstock to the cucumber scion under chilling stress conditions (Figs. <a href="#Fig1" class="usa-link">1</a> and <a href="#Fig2" class="usa-link">2</a>). Particularly, when cucumber was used as a rootstock, the corresponding mRNA molecule in cucumber (<em>CsaKARI</em>) did not exhibit upward movement (Fig. <a href="#Fig2" class="usa-link">2e</a>). These findings establish a significant mechanistic connection between pumpkin rootstock and the stress tolerance of grafted cucurbit crops.</p>
<p id="Par27">The transportation of mobile RNAs is primarily known to occur from shoot to root through the phloem, alongside the movement of photosynthates<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. However, this mechanism alone does not fully explain the role of pumpkin as a rootstock. In this study, we made an intriguing discovery of <em>CmoKARI1</em> being transported from the pumpkin root to the scion, specifically under chilling stress conditions. This finding suggests that the xylem, which is responsible for water and nutrient transport, may play a crucial role in mRNA transport. While Buhtz et al. previously reported the absence of RNAs in the xylem of Brassica plants<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>, subsequent studies have identified various mRNAs and non-coding RNAs in micro-dissected xylem samples<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Additionally, RNA m<sup>5</sup>C modification has been extensively studied and found to regulate RNA mobility<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Interestingly, in our investigation using the Cucume database of RNA mobility and methylation, we observed that mobile <em>CmoKARI1</em> exhibited m5C modification only in the vascular samples, but not in the samples from the whole plants<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. The precise mechanism by which the chilling signal triggers the specific loading of <em>CmoKARI1</em> into the xylem for its long-distance transport remains unclear. However, the mobility of <em>CmoKARI1</em> demonstrated a positive correlation with the chilling tolerance of different pumpkin breeding varieties (Fig. <a href="#Fig3" class="usa-link">3a–d</a>). This correlation suggests that different pumpkin breeding varieties could serve as a valuable genetic tool for further exploration and identification of communication factors involved in transmitting root-to-shoot signals in response to environmental cues.</p>
<p id="Par28"><em>KARI1</em> encodes a critical enzyme involved in the synthesis of branched-chain amino acids (BCAAs). Studies have shown that mutating other genes involved in BCAA biosynthesis, such as <em>IIL1/IPMI1</em>, results in growth defects and increased sensitivity to cold temperatures<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. This suggests that BCAAs play an important role in plant cold tolerance. During low temperatures, energy production in plants is often limited. BCAAs are physically connected to the mitochondrial respiratory chain, which regulates energy metabolism<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. In eukaryotes, BCAAs are known to activate the target of rapamycin (TOR) kinase, a master energy sensor<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. TOR integrates various environmental signals and coordinates transcription and translation<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Under cold stress, TOR activity is rapidly diminished after 2 h but recovers after 24 h<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. In fact, silencing of TOR has been shown to lead to heightened sensitivity to cold conditions<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. The increased isoleucine levels, possibly resulting from Csa/Cmo heterografts, could potentially activate TOR kinase and benefit plants under chilling conditions. Recent research has also highlighted the importance of TOR in shoot-root communication during abiotic stress<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. A future challenge will be to investigate whether the mobile <em>CmoKARI1</em>-mediated isoleucine biosynthesis is required for TOR activation and how TOR enhances chilling tolerance in grafted crops. Understanding these mechanisms will contribute to a more comprehensive understanding of the role of BCAAs, TOR kinase, and root-to-shoot signaling in improving the cold tolerance of grafted crops.</p>
<p id="Par29">Intriguingly, when only isoleucine was sprayed, and not other BCAAs, the chilling tolerance of cucumber was enhanced (one-way ANOVA followed by Duncan’s test, <em>p</em> &lt; 0.05) (Fig. <a href="#Fig4" class="usa-link">4</a>). This finding suggests that isoleucine, among the BCAAs, plays a crucial role in promoting chilling tolerance. One notable distinction between BCAAs is that isoleucine is involved in the synthesis of JA-Ile conjugates. JA signaling is known to be relevant for plants in coping with low temperatures<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. A previous study showed that applying Ile enhances JA-Ile-dependent Botrytis resistance<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. JA signaling is known to be relevant for plants in coping with low temperatures<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. JA-Ile, JA, and the precursor OPDA have been shown to be mobile, facilitating shoot-to-root coordination in response to wounding. Importantly, it is worth noting that JA-Ile and its biosynthetic precursors are capable of moving downward, likely following the phloem stream along with photosynthates. However, they are unable to move upward in <em>Arabidopsis</em><sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. This excludes the possibility of pumpkin rootstocks delivering JA-related metabolites to the scion. Recently, we identified that <em>CmoCK1</em> mRNA is specifically transported from pumpkin rootstock to cucumber scion in response to chilling stress. CK1 plays a role in JA biosynthesis, which further supports the concept of local JA biosynthesis within the scion, rather than long-distance transport from the rootstock<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Consequently, our conclusion is further reinforced that <em>CmoKARI1</em> is the key communication molecule responsible for inducing local biosynthesis of JA-Ile in the grafted scion.</p>
<p id="Par30">A significant limitation of this study is the inability to definitively establish a causal relationship between <em>CmoKARI1</em> mRNA transport and chilling tolerance in the scion. The lack of loss-of-function mutants in pumpkin, due to technical constraints, prevents a more conclusive assessment. This limitation leaves open the possibility that other systemic signals, such as metabolites, proteins, or small RNAs, may also contribute to chilling tolerance in the scion. For instance, gamma-aminobutyric acid (GABA) has been implicated in heterograft chilling tolerance, as FITC-labeled GABA exhibited a higher transport rate from pumpkin rootstock to cucumber scion under chilling stress<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. In contrast, the transport of Ile and Leu was less pronounced under chilling conditions (Fig. <a href="#Fig7" class="usa-link">7a–c</a>), supporting the notion that increased Ile and Leu levels in the heterograft during chilling are primarily due to local synthesis. Notably, Ile, but not Leu, specifically enhances chilling tolerance via the JA-Ile pathway. Further research is needed to elucidate how GABA and JA-Ile coordinate responses to chilling stress. It is important to acknowledge that FITC labeling is not an entirely accurate quantitative method. The use of high concentrations of labeled metabolites may result in non-physiological transport behavior<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Thus, determining the mobility of metabolites under physiological conditions remains a technical challenge that requires further methodological advancements.</p>
<p id="Par31">Overall, our results unveil a notable case where <em>CmoKARI1</em> mRNA is transported from the pumpkin rootstock to the grafted scion, specifically under chilling stress conditions. While researchers generally believed that the enhanced stress tolerance conferred by pumpkin rootstock was primarily attributed to its robust root system, our findings suggest that pumpkin rootstock is capable of delivering specific mobile molecules in response to distinct environmental cues. This discovery not only identifies an important mobile mRNA target for future crop breeding but also provides unprecedented insights into grafting biology.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Plant materials, growth conditions, grafting and chilling treatment</h3>
<p id="Par32">Cucumber (<em>Cucumis sativus</em> L. ‘Xintai Mici’) (Csa) was used as the scion, while pumpkin (<em>Cucurbita moschata</em> ‘Qianglishi’, ‘Lizhiyuan’, Shouguang Hongliang Seed Co.) and pumpkin (<em>Cucurbita ficifolia</em> Bouché.) (Cmo) were used as rootstocks. The grafting combinations included Csa/Cmo, Cmo/Csa, Csa/Csa, and Cmo/Cmo. Grafted plants were initially kept in complete darkness for 3 days in a growth chamber maintained at 28 °C and 100% relative humidity. This was followed by another 4-day period under low light conditions (16 h light/8 h dark; light intensity: 60–100 μmol m<sup>−2</sup> s<sup>−1</sup>) in a growth chamber with a 28 °C/18 °C day/night temperature regime and 90% relative humidity. Seven days post-grafting, the formation of the graft union was evaluated after removing the grafting clip.</p>
<p id="Par33">For chillin treatment, either cucumber plants or 7-day post-grafted cucumbers were transferred to a growth chamber under chilling stress conditions (4 °C; 16 h light/8 h dark; light intensity: 60–100 μmol m<sup>−</sup><sup>2</sup> s<sup>−1</sup>; relative humidity: 70%). For metabolites application treatment, cucumber seedlings were cultured in a growth chamber (28 °C/18 °C 16 h/8 h day/night; light intensity: 200 μmol m<sup>−2</sup> s<sup>−1</sup>; relative humidity: 60%) in a mixed matrix (peat:vermiculite:perlite = 2:1:1, by vol.) until they were two-leaf old. Approximately 100–200 seedlings were subjected to treatment by spraying them twice per day with 25 ml of a water solution containing either 0, 100, 200, 300, or 400 μM <span class="font-variant-small-caps">L</span>-leucine, <span class="font-variant-small-caps">L</span>-isoleucine, and <span class="font-variant-small-caps">L</span>-valine (Sigma-Aldrich, Shanghai, China). Alternatively, seedlings were sprayed twice per day with a water solution containing 100 μM JA (TCI, Shanghai, China), 100 μM MeJA (Sigma-Aldrich, Shanghai, China), 100 μM JA-Ile (Macklin, Shanghai, China), or 100 μM Jarin-1 (Macklin, Shanghai, China). After three days, all seedlings were transferred to 4 °C with normal light and humidity. Physiological traits of the seedlings were assessed at two time points: 0 h (non-chilled control) and after 12 h of chilling treatment. For each condition, the leaves of 5–6 randomly selected individual plants were harvested and combined into one replicate pool. At least three replicate pools were collected for each condition to perform further analysis of physiological traits and quantitative RT-PCR (RT-qPCR) assays.</p>
<p id="Par34">Chilling treatment of <em>Arabidopsis thaliana</em> (Col-0) was performed, with modification. Surface sterilization of wild type (Col-0) and overexpressed (TG) seeds was performed using 70% ethanol solution and 5% NaClO solution, and grown on 1/2 MS medium. The seeds were then placed in a growth chamber at 22 °C for 8–10 days till seedling growth. Chilling treatment was performed at −10 °C freezing room for 1 h, then cultured under 22 °C growth chamber (12 h/12 h day/night; light intensity: 200 μmol m<sup>−2</sup> s<sup>−1</sup>; relative humidity: 60%) for 2 days and 4 days, and then the phenotypes, survival rate and cold tolerance were determined.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Cucumber and <em>Arabidopsis</em> transformation</h3>
<p id="Par35">A 1773-bp PCR fragment containing the complete coding sequence of <em>CmoKARI1</em> was obtained from the Cucurbit Genomics Database (<em>Cucurbita moschata</em> (Rifu) Genome). The fragment was amplified from pumpkin ‘Qianglishi’ cDNA using specific primers (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>) and subsequently cloned into the pGEM-T Easy (TaKaRa, Beijing, China) vector. The open reading frame of <em>CmoKARI1</em> was inserted into the Super promoter-derived GFP-fused binary vector <em>pCambia1300</em> using XbaI/SalI restriction digestion sites. It was also inserted into the 35S promoter-derived GUS-fused binary vector, a modified version of <em>pCambia1305.1</em>, using XbaI/BamHI restriction digestion sites. Either the recombined vector <em>pCambia1305-</em>p35S::CmoKARI1-GUS or <em>pCambia1300-</em>pSuper::CmoKARI1-GFP was transformed into cucumber hairy roots of the cultivar ‘Xintaimici’ through <em>Agrobacterium tumefaciens</em> (K599)-mediated cucumber hairy root transformation with modification<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. The seeds were immersed in 55 °C water for 30 min, and the seed coats were removed. After surface-sterilization with 70% ethanol for 30 s and 3% sodium hypochlorite for 8 min, they were sown onto MS medium and incubated at 28 °C in the dark for 4 d. A. rhizogenes strain K599 grown to an OD600 of 0.8–1.0 is pelleted by centrifugation and re-suspended in MS liquid medium supplemented with 200 µM Acetosyringone (CA1061, Beijing Coollab Technology Co., Ltd.) for infection. After inoculation, explants are co-cultivated in the dark at 25 °C for 3 d. Following co-cultivation, the hypocotyl of each explant is inserted into rooting medium and cultured under a 16 h light/8 h dark photoperiod to induce hairy-root formation. Transgenic roots are identified by GFP epifluorescence under Fluorescent Protein Lamp (LUYOR-3415RG, Shanghai Luyor Instrument Co., Ltd.). The <em>Agrobacterium tumefaciens</em> (GV3101) containing the recombined vector <em>pCambia1300-</em>pSuper::CmoKARI1-GFP fused binary vector was used to transfect <em>Arabidopsis thaliana</em> (Col-0) and cucumber cotyledon. Transgenic lines were screened in a selection medium and verified at the DNA and RNA levels through PCR and RT-qPCR. T2 transgenic lines were used for further study. The primers used for cloning, construction, and PCR are listed in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>.</p></section><section id="Sec12"><h3 class="pmc_sec_title">REP and MDA measurement</h3>
<p id="Par36">Relative electrolyte permeability (REP) and malondialdehyde (MDA) content were assessed following the method with some modifications<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. To measure REP, leaf discs were immersed in distilled water at room temperature. The initial electrical conductivity of the distilled water was measured and recorded as EC<sub>0</sub>. After shaking for 2.5 h, the electrical conductivity of the solution was measured again and designated as EC<sub>1</sub>. The samples were then incubated in a 95 °C water bath for 30 min, cooled to room temperature, and the final electrical conductivity was recorded as EC<sub>2</sub>. The relative electrolyte leakage was calculated using the formula: (EC<sub>1</sub> − EC<sub>0</sub>)/(EC<sub>2</sub> − EC<sub>0</sub>) × 100%. Malondialdehyde (MDA) content thiobarbituric acid (TBA) method. Fresh leaf tissue was homogenized in trichloroacetic acid (TCA) using a pre-chilled mortar and pestle. The absorbance of the supernatant was measured at 532 nm and corrected for nonspecific turbidity by subtracting the absorbance at 600 nm. The MDA concentration was calculated using an extinction coefficient of 155 mM<sup>−1</sup> cm<sup>−1</sup> and expressed as nmol MDA per gram of fresh weight (nmol g<sup>−1</sup> FW).</p>
<p id="Par37">For each graft combination, three to five individual cucumber seedlings and 7–10 leaf disks per leaf were harvested and pooled into three replicates at each chilling treatment time point for subsequent analysis. Similarly, for each transgenic <em>A. thaliana</em> line, three pools, each consisting of 10 individual seedlings with similar growth status, were harvested at each chilling treatment time point.</p></section><section id="Sec13"><h3 class="pmc_sec_title">RT-PCR and RT-quantitative PCR</h3>
<p id="Par38">Total RNA was extracted from the entire first leaf and root using a FreeZol Reagent kit (R711-01, Vazyme Biotech Co., Ltd., Nanjing, China). Reverse transcription reactions were performed using a HiScript IV All-in-One Ultra RT SuperMix kit (Vazyme Biotech Co., Ltd., Nanjing, China, Cat#R433-01) following the manufacturer’s instructions with the following steps: total RNA (~1 µg) treated with 5 µL 4× All-in-One Ultra qRT SuperMix. The reaction program was as follows: 50 °C for 5 min, followed by 85 °C for 5 s.</p>
<p id="Par39">Quantitative RT-PCR was performed according to the SYBR method in a 10-µL volume using 20 ng total RNA, 5 µL Taq Pro Universal SYBR qPCR Master Mix (Vazyme Biotech Co., Ltd., Nanjing, China, Cat#Q712-02) and 0.2 µl forward and reverse primers. Each sample comprised RNA isolated from 3 to 5 individual plants. At least three technical replicates were performed. An ABI System Sequence Detector (QuantStudio<sup>TM</sup> 6 Flex real-time PCR system, CA, United States) was used with the following thermal cycling conditions. Stage 1: 1 cycle at 95 °C for 30 s; Stage 2: 40 cycles at 95 °C for 10 s, 60 °C for 30 s; Dissociation stage: 15 s at 95 °C, 1 min at 60 °C, 15 s at 95 °C. After amplification, melting curve analysis was performed to verify the product. Measured Ct values were converted to relative copy numbers using the ∆∆Ct method. Each sample comprised RNA isolated from 3 to 5 individual plants. At least three technical replicates were performed. Values were normalized by using <em>CmoActin</em> or <em>CsaActin</em>. Primers are listed in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Identification of mobile mRNAs by RNA-seq and RT- PCR</h3>
<p id="Par40">Based on the nonsense consensus genomes of Csa (<em>‘Xintaimici’</em>) and Cmo (<em>‘Qianglishi’</em>), the detailed strategy of mobile mRNA calling was performed based on Single Nucleotide Polymorphism (SNP) comparison with consensus Cmo-modified and a Csa-modified genome using bcftools 1.8-1 software<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Lists of mobile mRNA datasets were generated for both no chilling and 6 h chilling conditions using the Tophat v2.2 and BLASTn programs. The verification of mRNA mobility by RT-PCR was performed using established technology<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. RT-PCR assays on RNA samples of all homo- and heterograft combinations using primers designed to recognize specific sequences of Csa/Cmo transcripts with 2xRapid Taq Master Mix (Vazyme Biotech Co., Ltd., Nanjing, China, Cat#P222-03). Stage 1:1 cycle at 95 °C for 3 min; Stage 2: 40 cycles at 95 °C for 15 s, 60 °C for 15 s, 72 °C for 15 s/kb; Stage 3: 5 min at 72 °C. The primer used in this study was listed in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Integrative analysis of metabolites and transcripts</h3>
<p id="Par41">KEGG pathway and Venn analyses of DIMs or DEGs were both performed with the OmicShare tools (<a href="http://www.omicshare.com/tools" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.omicshare.com/tools</a>). The corresponding numbers of DEGs from the FPKM values were performed as follows: (i) The difference in transcript abundance in the same tissue between heterologous and homologous grafts without chilling, i.e., fold difference of FPKM value (control Csa/Cmo leaf − control Csa/Csa leaf), was recorded as Δ1. (ii) The difference in transcript abundance in the same tissue between heterologous and homologous grafts in the 6 h chilling condition, e.g., fold difference of FPKM value (chilling Csa/Cmo leaf − chilling Csa/Csa leaf), was recorded as Δ2. (iii) The fold change in transcript abundance between chilling and control conditions, e.g., fold change of FPKM value (Δ2 − Δ1), was recorded as the Csa DEGs in the leaf. (iv) Genes with differences in transcript abundance with log<sub>2</sub>(fold change) &gt;2 or log<sub>2</sub>(fold change) &lt;−2 were identified as up- or downregulated DEGs, respectively.</p>
<p id="Par42">The corresponding differential intense metabolites (DIMs), based on the ion intensity was analyzed as follows: (i) The difference in ion intensity values between the same tissues of heterografts and homografts in the control condition, i.e. the fold difference in ion intensity (control Csa/Cmo leaf − control Csa/Csa leaf), was recorded as the Δ1. (ii) The difference in ion intensity values between the same tissues in heterografts and homografts in the 6 h chilling condition, e.g., the fold change in ion intensity (chilling Csa/Cmo leaf − chilling Csa/Csa leaf), was recorded as the Δ2. (iii) The final difference between the values from chilling and control conditions, e.g., the fold change in ion intensity in (Δ2 − Δ1), was recorded as the multiple fold change of heterologous grafts in the chilling condition. (iv) The value of log<sub>2</sub>(Δ2 − Δ1) was used to verify DIM status: compounds with log<sub>2</sub>(Δ2 − Δ1) &gt; 2 or log<sub>2</sub>(Δ2 − Δ1) &lt;−2 were identified as up- or downregulated DIMs, respectively. Gene ID lists of DEGs and mobile mRNAs in different tissues of cucumber and pumpkin heterografts (Csa/Cmo, Cmo/Csa) and homografts (Csa/Csa, Cmo/Cmo) under both normal and chilling conditions were used for KEGG analysis.</p>
<p id="Par43">Integrative analysis was performed as described by using Pearson correlation coefficient calculation for metabolome and transcriptome data integration. The mean of all biological replicates of each cultivar in the metabolome data and the mean value for the expression of each transcript in the transcriptome data were calculated. For example, the fold differences in Csa/Cmo leaves tissues were calculated from both the metabolome and transcriptome data and compared with those of the control Csa/Csa leaves under both control and 6 h chilling conditions. The coefficients were calculated from log<sub>2</sub>(fold change) of each metabolite and log<sub>2</sub>(fold change) of each transcript using Excel software. The relationship between metabolome, transcriptome and mobile mRNAs was performed based on KEGG map IDs using the Excel VLOOKUP function. We used Circos diagrams to analyze the correlation between the rootstock/scion of each heterologous grafting combination. Metabolome, transcriptome and mobile mRNA relationships were visualized using Cytoscape (version 3.8.1).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Determination of endogenous JAs and Isoleucine</h3>
<p id="Par44">The quantification of JAs was conducted at MetWare Biotechnology Co., Ltd. (Wuhan, China) using the AB Sciex QTRAP 6500 LC–MS/MS platform (<a href="https://sciex.com.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sciex.com.cn/</a>). Leaves from two-leaf-old transgenic cucumber and wild-type seedlings, weighing 50 mg, were harvested and frozen in liquid nitrogen. 1 μg/ml internal standards of JA (Jasmonic acid, Sigma, cat. no. J2500), JA-Ile (Jasmonoyl-<span class="font-variant-small-caps">L</span>-isoleucine, OlChemim, cat. no. 0146233), MeJA (Jasmonic acid methyl ester, OlChemim, cat. no. 0412963) and OPDA (13-epi-12-Oxo-phytodienoic acid, OlChemim, cat. no. 0146193) were used. The calibration was performed as described in Pan et al.<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. To enable quantification, 10 μL of an internal standard mixed solution (100 ng/mL) was added to the extract as internal standards (IS). They were then extracted by the solvent, 2-propanol/H<sub>2</sub>O/ concentrated HCl (2:1:0.002, vol/vol/vol) strictly following the extraction protocol<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. The samples were mixed with another 1 ml dichloromethane for 30 minutes at 4 °C, followed by centrifugation for 5 min at 13,000<em>g</em> and 4 °C. The solvent was transferred from the lower phase into a screw-cap vial and concentrated the solvent mixture (not completely dry) using a nitrogen evaporator with nitrogen flow. To avoid the loss of samples, Sep-Pak C18 solid-phase extraction (SPE) was not used in the extraction. The samples are redissolved in 0.1 mL of methanol. Fifty microlitres of the sample solution was injected into the reverse-phase C18 Gemini HPLC column. The triple quadrupole-linear ion trap mass spectrometer (QTRAP), QTRAP® 6500 + LC–MS/MS System, was equipped with an ESI Turbo Ion-Spray interface and operated in both positive and negative ion modes.</p>
<p id="Par45">The ESI source operation parameters were as follows: ion source, ESI+/−; source temperature 550 °C; ion spray voltage (IS) 5500 V, Positive, −4500 V (Negative); curtain gas (CUR) was set at 35 psi, respectively. Phytohormones were analyzed using scheduled multiple reaction monitoring (MRM). The analysis was controlled by Analyst 1.6.3 software (Sciex)<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Multiquant 3.0.3 software (Sciex) was used to quantify all metabolites. Mass spectrometer parameters, including the delustering potentials (DP) and collision energies (CE) for individual MRM transitions, were optimized with further DP and CE optimization. Three biological replicates were performed, and each replicate included 4–6 individual cucumber plants.</p>
<p id="Par46">For the measurement of isoleucine levels, the analysis was conducted at IGENECODE Co., Ltd. (Beijing, China). A UPLC system (ACQUITY UPLC; Waters, Milford, MA, USA) and a hybrid quadrupole time-of-flight (Q-TOF) tandem mass spectrometry (triple-TOF-MS/MS; Triple TOF 6545 system) were utilized following modifications<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Four biological replicates were performed, and each replicate included 4–6 individual cucumber plants. Statistical analysis was performed using IBM SPSS Statistics 27 v27.0.1. One-way ANOVA followed by Duncan’s test was used (<em>p</em> &lt; 0.05).</p></section><section id="Sec17"><h3 class="pmc_sec_title">Virus-induced gene silencing (VIGS)</h3>
<p id="Par47">Specific sequences from the CDS of <em>CsaKARI</em> or <em>CsaJAR1</em> were cloned into the pTRSV(Tobacco RingSpot Virus) vector and transformed into <em>Agrobacterium</em> GV3101. Cucumber seedlings at the cotyledon stage were injected with a mixture of <em>Agrobacterium</em> containing pTRSV1 and pTRSV2 recombinant plasmids. The inoculated cucumber seedlings were placed in a growth chamber for cultivation. Cucumber Phytoene Desaturase (CsaPDS) construct (TRSV1: TRSV2-PDS, 1:1) used for silencing control. TRSV2 and TRSV1 empty vector (TRSV1: TRSV2-00,1:1) were used as a negative control. TRSV1:TRSV2-<em>CsaKARI</em> and TRSV1:TRSV2-<em>CsaJAR1</em> were used to inoculate cucumber wild type and CmoKARI1-TG transgenic cucumbers, respectively. When the plants grew to the two-leaf stage, the presence of TRSV1 and TRSV2 vectors in the cucumbers was detected, and the silencing efficiency was verified using RT-qPCR. Plants with relatively high silencing efficiency (&gt;70%) were selected for subsequent low-temperature treatment experiments.</p></section><section id="Sec18"><h3 class="pmc_sec_title">DAB and NBT staining</h3>
<p id="Par48">DAB (3,3′-diaminobenzidine) and NBT (Nitrotetrazolium Blue chloride) staining were performed on the first true leaf of cucumber seedlings. DAB and NBT reagents were purchased from Beijing Coolaber Company. For DAB staining, 50 mg of DAB was added to a 50 ml centrifuge tube containing 45 ml of distilled water, followed by the addition of 25 μl Tween-20 and 2.5 ml of 200 mM Na<sub>2</sub>HPO<sub>4</sub> to the tube to prepare a 10 mM Na<sub>2</sub>HPO<sub>4</sub>-DAB solution. Three true leaves from each of three seedlings were used as one biological replicate, and three biological replicates were performed for each treatment. For NBT staining, 0.1 g of NBT was dissolved in 50 ml of 50 mM phosphate buffer to prepare a 0.2% NBT staining solution. The intensity of DAB and NBT staining was analyzed by using ImageJ. Three true leaves from each of three seedlings were used as one biological replicate, and three biological replicates were performed for each treatment.</p></section><section id="Sec19"><h3 class="pmc_sec_title">ELISA (enzyme-linked immunosorbent assay) for the determination of Ile and JA-Ile content</h3>
<p id="Par49">Ile and JA-Ile content in cucumber leaves were determined by using Plant ILE ELISA KIT (Shanghai Yiyan Biotechnology Co., Ltd., China) and Plant JA-Ile, Ile ELISA KIT (MyBioSource, SanDiego, USA; Shanghai Renjie) as described in the manual instructions. Cucumber leaves in different treatments were harvested from 5 to 6 individual plants randomly and combined into one replicate pool. At least three replicate pools were collected for each condition to analyze Ile and JA-Ile content. Each 0.15 g of ground sample mixed with PBS (pH 7.4) was centrifuged at 8000 rpm for 30 min at 4 °C, and the supernatant was used for Ile and JA-Ile content determination.</p>
<p id="Par50">First, the standard curve was created according to the manual instructions. In summary, 50 μl of various concentrations of standard samples were added to 40 μl of diluted sample supernatant and 10 μl of sample solution. After adding 100 μl of conjugate reagent, the samples were then incubated at 37 °C for 60 min. Following the incubation, the liquid was discarded, and the wells were washed with washing solution five times. Subsequently, 50 μl each of Chromogen solution A and Chromogen solution B provided by the kit were added to each sample and incubated for 15 min at 37 °C in the dark. Finally, the reaction was halted by adding 50 μl of stop solution, and the optical density (OD) value was measured at a wavelength of 450 nm. The polynomial quadratic regression equation of the standard curve was then calculated using these two formulas:</p>
<p id="Par51">Ile: C (content, pmol/L) = 459.23*(OD) − 73.249;</p>
<p id="Par52">JA-Ile: C (content, pmol/L) = 171.09*(OD) + 20.545.</p>
<p id="Par53">The formula to convert JA-Ile and Ile content in pmol/L into ng/g FW is as follows:</p>
<p id="Par54">Ile or JA-Ile content (ng/g FW) = <em>C</em> (pmol/<em>L</em>) * <em>M</em> * 10<sub>−</sub> * <em>V</em> * <em>X</em> * 10<sup>−3</sup>/<em>W</em></p>
<p id="Par55"><em>C</em> signifies the solution concentration (pmol/L), <em>M</em> represents the molecular weight, <em>V</em> stands for the sample volume of 10 μl, <em>X</em> indicates the extraction factor of 70, and <em>W</em> denotes the sample weight of 0.15 g.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Fluorescein isothiocyanate (FITC)-labeled amino acids application in grafted seedlings</h3>
<p id="Par56">FITC-Isoleucine and FITC-leucine were synthesized by Xi’an Ruixi Biological Technology Co., Ltd. (<a href="http://www.xarxbio.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.xarxbio.com</a>, Xi’an, China). Briefly, 4 g of FITC was dissolved in a solvent mixture of 50 mL N,N- Dimethylformamide (DMF) and water. <span class="font-variant-small-caps">L</span>-Isoleucine, <span class="font-variant-small-caps">L</span>-leucine (1.5 eq.) and disopropylethylamine (DIPEA) (3.0 eq.) were added until completely dissolved. The reaction was carried out at 25 °C for 0.5 h. The reaction mixture was concentrated under reduced pressure with a rotary vane vacuum pump (HR-21M, -30mmHg/1BAR (100 kpa)) at 50 °C for 20 min, and the crude product was eluted by dichloromethane: methanol = 10:1 at 25 °C for 2 h, and purified by silica gel chromatography column (200–300 mesh). The product was vacuum-dried at 25 °C for 2 days. The quality of synthesized FITC-Isoleucine and FITC-leucine was inspected by the detector (SPD-20A) and mass spectrometer (API3200) with chromatographic column (HPLC column Shim-pack GIST-HP C18, 3 μm, 150 mm × 2.1 mm), Liquid-mass conditions (Flow rate:0.2 ml/min, wavelength: 490 nm, Injection volume: 10 μl, Mobile phase: methanol, 20 mM ammonium acetate-ammonia buffer (pH 8.0), ESI+)</p>
<p id="Par57">The roots of grafted cucumber/pumpkin and cucumber/cucumber seedlings were pretreated with 50 μM FITC-Isoleucine, or leucine solution for 12 h. The distilled water was used as the negative control<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Then, grafted seedlings were transferred to a climate chamber at low (4/4 °C) and normal (28/18 °C) temperature with a 14 h/10 h light/dark photoperiod and PPFD of 300 μmol m<sup>−2</sup> s<sup>−1</sup>. After 6 h of low temperature treatment, the distribution of FITC-Isoleucine and FITC-leucine in roots, stems, petioles and leaves of grafted seedlings was examined by confocal laser scanning microscope (TCSSP8, Lecia). FITC fluorescence signal was quantified by ImageJ software.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par58">Statistical analysis was performed using IBM SPSS Statistics 27 v27.0.1. For time-series data where treatment is compared only with the control, one-way ANOVA followed by Dunnett’s multiple comparisons test was used (<em>p</em> &lt; 0.05). Conversely, in cases where pairwise comparisons are made between multiple data sets, one-way ANOVA followed by Duncan’s test was used (<em>p</em> &lt; 0.05).</p></section><section id="Sec22"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par59">Further information on research design is available in the <a href="#MOESM4" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368190/bin/41467_2025_63228_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (206.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12368190/bin/41467_2025_63228_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63228_MOESM2_ESM.pdf</a><sup> (87.3KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12368190/bin/41467_2025_63228_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1-9</a><sup> (363.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12368190/bin/41467_2025_63228_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (2.7MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12368190/bin/41467_2025_63228_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.2MB, pdf) </sup>
</div></div></section></section><section id="Sec24"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12368190/bin/41467_2025_63228_MOESM6_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (98MB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We would like to thank Dr. Thierry Heitz (IBMP-CNRS, Strasbourg, France) and Prof. Dr. Xuna Wu (Yunnan University, Kunming, China) for their critical comments on our manuscript. We greatly thank the ikann-editorial team for their scientific editing. This work was supported by the National Natural Science Foundation of China (32372792) for W. Zhang, the 2115 Talent Development Program of China Agricultural University to W. Zhang, and the Earmarked Fund for China Agriculture Research System (CAS-23) to L. Gao.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Y. Dong, L. Gao, and W. Zhang conceived and designed the experiments; M. Zhang, W. Liu, C. Wang, S. Lin, Y. Lu, and X. Li performed the experiments; Y. Dong and W. Zhang wrote the paper; Y. Dong, W. Zhang, and X. Han revised the paper; M. Zhang, C. Xiang, W. Liu, Y. Chen, H. Cui, Y. Ma, and W. Zhang analyzed the data; All authors read the final version of this paper and approved it for publication.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par60"><em>Nature Communications</em> thanks Tien-Shin Yu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data supporting the results of this article are included within the article and its additional files. Source Data are provided with this paper. The original RNA sequencing data used in this study are deposited in the NCBI Small Read Archive under accession numbers <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA552914/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA552914</a> and <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA673087/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA673087</a>. The accession code of all genes used in this study was obtained from the Cucurbit Genomics Database (Cucumber Gy14 v2 <a href="http://cucurbitgenomics.org/organism/16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://cucurbitgenomics.org/organism/16</a>, <em>Cucurbita moschata</em> (Rifu) genome <a href="http://cucurbitgenomics.org/organism/9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://cucurbitgenomics.org/organism/9</a>) or the <a href="https://www.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCBI</a> and are provided in the Supplementary Dataset 8. <a href="#Sec24" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par61">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Miao Zhang, Wenqian Liu, Cuicui Wang, Shujin Lin.</p></div>
</div></section><section id="sec25"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63228-1.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Ding, Y., Shi, Y. &amp; Yang, S. Advances and challenges in uncovering cold tolerance regulatory mechanisms in plants. <em>N. Phytol.</em><strong>222</strong>, 1690–1704 (2019).</cite> [<a href="https://doi.org/10.1111/nph.15696" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30664232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ding,%20Y.,%20Shi,%20Y.%20&amp;%20Yang,%20S.%20Advances%20and%20challenges%20in%20uncovering%20cold%20tolerance%20regulatory%20mechanisms%20in%20plants.%20N.%20Phytol.222,%201690%E2%80%931704%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Zhu, J. K. Abiotic Stress Signaling and Responses in Plants. <em>Cell</em><strong>167</strong>, 313–324 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.08.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5104190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27716505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20J.%20K.%20Abiotic%20Stress%20Signaling%20and%20Responses%20in%20Plants.%20Cell167,%20313%E2%80%93324%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Guan, W., Haseman, D. &amp; Nowaskie, D. Rootstock evaluation for grafted cucumbers grown in high tunnels: yield and plant growth. <em>HortScience</em><strong>55</strong>, 914–919 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Guan,%20W.,%20Haseman,%20D.%20&amp;%20Nowaskie,%20D.%20Rootstock%20evaluation%20for%20grafted%20cucumbers%20grown%20in%20high%20tunnels:%20yield%20and%20plant%20growth.%20HortScience55,%20914%E2%80%93919%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Stockinger, E. J., Gilmour, S. J. &amp; Thomashow, M. F. Arabidopsis thaliana CBF1 encodes an AP2 domain-containing transcriptional activator that binds to the C-repeat/DRE, a cis-acting DNA regulatory element that stimulates transcription in response to low temperature and water deficit. <em>Proc. Natl Acad. Sci. USA</em><strong>94</strong>, 1035–1040 (1997).
</cite> [<a href="https://doi.org/10.1073/pnas.94.3.1035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC19635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9023378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stockinger,%20E.%20J.,%20Gilmour,%20S.%20J.%20&amp;%20Thomashow,%20M.%20F.%20Arabidopsis%20thaliana%20CBF1%20encodes%20an%20AP2%20domain-containing%20transcriptional%20activator%20that%20binds%20to%20the%20C-repeat/DRE,%20a%20cis-acting%20DNA%20regulatory%20element%20that%20stimulates%20transcription%20in%20response%20to%20low%20temperature%20and%20water%20deficit.%20Proc.%20Natl%20Acad.%20Sci.%20USA94,%201035%E2%80%931040%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Liu, Q. et al. Two transcription factors, DREB1 and DREB2, with an EREBP/AP2 DNA binding domain separate two cellular signal transduction pathways in drought- and low-temperature-responsive gene expression, respectively, in Arabidopsis. <em>Plant Cell</em><strong>10</strong>, 1391–1406 (1998).
</cite> [<a href="https://doi.org/10.1105/tpc.10.8.1391" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC144379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9707537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Q.%20et%20al.%20Two%20transcription%20factors,%20DREB1%20and%20DREB2,%20with%20an%20EREBP/AP2%20DNA%20binding%20domain%20separate%20two%20cellular%20signal%20transduction%20pathways%20in%20drought-%20and%20low-temperature-responsive%20gene%20expression,%20respectively,%20in%20Arabidopsis.%20Plant%20Cell10,%201391%E2%80%931406%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Warschefsky, E. J. et al. Rootstocks: diversity, domestication, and impacts on shoot phenotypes. <em>Trends Plant Sci.</em><strong>21</strong>, 418–437 (2016).
</cite> [<a href="https://doi.org/10.1016/j.tplants.2015.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26698413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Warschefsky,%20E.%20J.%20et%20al.%20Rootstocks:%20diversity,%20domestication,%20and%20impacts%20on%20shoot%20phenotypes.%20Trends%20Plant%20Sci.21,%20418%E2%80%93437%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Liu, W. et al. Transcriptomic and physiological analysis reveal that alpha-linolenic acid biosynthesis responds to early chilling tolerance in pumpkin rootstock varieties. <em>Front. Plant Sci.</em><strong>12</strong>, 669565 (2021).
</cite> [<a href="https://doi.org/10.3389/fpls.2021.669565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8104029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33968120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20W.%20et%20al.%20Transcriptomic%20and%20physiological%20analysis%20reveal%20that%20alpha-linolenic%20acid%20biosynthesis%20responds%20to%20early%20chilling%20tolerance%20in%20pumpkin%20rootstock%20varieties.%20Front.%20Plant%20Sci.12,%20669565%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Guo, Y. et al. Abscisic acid mediates grafting-induced cold tolerance of watermelon via interaction with melatonin and methyl jasmonate. <em>Front. Plant Sci.</em><strong>12</strong>, 785317 (2021).
</cite> [<a href="https://doi.org/10.3389/fpls.2021.785317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8719526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34975972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20Y.%20et%20al.%20Abscisic%20acid%20mediates%20grafting-induced%20cold%20tolerance%20of%20watermelon%20via%20interaction%20with%20melatonin%20and%20methyl%20jasmonate.%20Front.%20Plant%20Sci.12,%20785317%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Li, H. et al. Methyl jasmonate mediates melatonin-induced cold tolerance of grafted watermelon plants. <em>Hortic. Res.</em><strong>8</strong>, 57 (2021).
</cite> [<a href="https://doi.org/10.1038/s41438-021-00496-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7943586/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33750773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.%20et%20al.%20Methyl%20jasmonate%20mediates%20melatonin-induced%20cold%20tolerance%20of%20grafted%20watermelon%20plants.%20Hortic.%20Res.8,%2057%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Wang, L. et al. The essential role of jasmonate signaling in Solanum habrochaites rootstock-mediated cold tolerance in tomato grafts. <em>Hortic. Res.</em><strong>10</strong>, uhac227 (2023).
</cite> [<a href="https://doi.org/10.1093/hr/uhac227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9832872/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36643752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20L.%20et%20al.%20The%20essential%20role%20of%20jasmonate%20signaling%20in%20Solanum%20habrochaites%20rootstock-mediated%20cold%20tolerance%20in%20tomato%20grafts.%20Hortic.%20Res.10,%20uhac227%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Huang, H., Liu, B., Liu, L. &amp; Song, S. Jasmonate action in plant growth and development. <em>J. Exp. Bot.</em><strong>68</strong>, 1349–1359 (2017).
</cite> [<a href="https://doi.org/10.1093/jxb/erw495" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28158849/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20H.,%20Liu,%20B.,%20Liu,%20L.%20&amp;%20Song,%20S.%20Jasmonate%20action%20in%20plant%20growth%20and%20development.%20J.%20Exp.%20Bot.68,%201349%E2%80%931359%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>An, J. P., Wang, X. F., Zhang, X. W., You, C. X. &amp; Hao, Y. J. Apple B-box protein BBX37 regulates jasmonic acid mediated cold tolerance through the JAZ-BBX37-ICE1-CBF pathway and undergoes MIEL1-mediated ubiquitination and degradation. <em>N. Phytol.</em><strong>229</strong>, 2707–2729 (2021).</cite> [<a href="https://doi.org/10.1111/nph.17050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33119890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?An,%20J.%20P.,%20Wang,%20X.%20F.,%20Zhang,%20X.%20W.,%20You,%20C.%20X.%20&amp;%20Hao,%20Y.%20J.%20Apple%20B-box%20protein%20BBX37%20regulates%20jasmonic%20acid%20mediated%20cold%20tolerance%20through%20the%20JAZ-BBX37-ICE1-CBF%20pathway%20and%20undergoes%20MIEL1-mediated%20ubiquitination%20and%20degradation.%20N.%20Phytol.229,%202707%E2%80%932729%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>An, J. P. et al. Abscisic acid insensitive 4 interacts with ICE1 and JAZ proteins to regulate ABA signaling-mediated cold tolerance in apple. <em>J. Exp. Bot.</em><strong>73</strong>, 980–997 (2022).
</cite> [<a href="https://doi.org/10.1093/jxb/erab433" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34555166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?An,%20J.%20P.%20et%20al.%20Abscisic%20acid%20insensitive%204%20interacts%20with%20ICE1%20and%20JAZ%20proteins%20to%20regulate%20ABA%20signaling-mediated%20cold%20tolerance%20in%20apple.%20J.%20Exp.%20Bot.73,%20980%E2%80%93997%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Qi, C. et al. Heat shock-induced cold acclimation in cucumber through CsHSFA1d-activated JA biosynthesis and signaling. <em>Plant J.</em><strong>111</strong>, 85–102 (2022).
</cite> [<a href="https://doi.org/10.1111/tpj.15780" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35436390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qi,%20C.%20et%20al.%20Heat%20shock-induced%20cold%20acclimation%20in%20cucumber%20through%20CsHSFA1d-activated%20JA%20biosynthesis%20and%20signaling.%20Plant%20J.111,%2085%E2%80%93102%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Chini, A. et al. The JAZ family of repressors is the missing link in jasmonate signalling. <em>Nature</em><strong>448</strong>, 666–671 (2007).
</cite> [<a href="https://doi.org/10.1038/nature06006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17637675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chini,%20A.%20et%20al.%20The%20JAZ%20family%20of%20repressors%20is%20the%20missing%20link%20in%20jasmonate%20signalling.%20Nature448,%20666%E2%80%93671%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Thines, B. et al. JAZ repressor proteins are targets of the SCF(COI1) complex during jasmonate signalling. <em>Nature</em><strong>448</strong>, 661–665 (2007).
</cite> [<a href="https://doi.org/10.1038/nature05960" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17637677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thines,%20B.%20et%20al.%20JAZ%20repressor%20proteins%20are%20targets%20of%20the%20SCF(COI1)%20complex%20during%20jasmonate%20signalling.%20Nature448,%20661%E2%80%93665%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Xie, D. X., Feys, B. F., James, S., Nieto-Rostro, M. &amp; Turner, J. G. COI1: an Arabidopsis gene required for jasmonate-regulated defense and fertility. <em>Science</em><strong>280</strong>, 1091–1094 (1998).
</cite> [<a href="https://doi.org/10.1126/science.280.5366.1091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9582125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie,%20D.%20X.,%20Feys,%20B.%20F.,%20James,%20S.,%20Nieto-Rostro,%20M.%20&amp;%20Turner,%20J.%20G.%20COI1:%20an%20Arabidopsis%20gene%20required%20for%20jasmonate-regulated%20defense%20and%20fertility.%20Science280,%201091%E2%80%931094%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Yan, J. et al. The Arabidopsis F-box protein CORONATINE INSENSITIVE1 is stabilized by SCFCOI1 and degraded via the 26S proteasome pathway. <em>Plant Cell</em><strong>25</strong>, 486–498 (2013).
</cite> [<a href="https://doi.org/10.1105/tpc.112.105486" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3608773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23386265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan,%20J.%20et%20al.%20The%20Arabidopsis%20F-box%20protein%20CORONATINE%20INSENSITIVE1%20is%20stabilized%20by%20SCFCOI1%20and%20degraded%20via%20the%2026S%20proteasome%20pathway.%20Plant%20Cell25,%20486%E2%80%93498%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Schulze, A. et al. Wound-induced shoot-to-root relocation of ja-ile precursors coordinates arabidopsis growth. <em>Mol. Plant</em><strong>12</strong>, 1383–1394 (2019).
</cite> [<a href="https://doi.org/10.1016/j.molp.2019.05.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31181337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schulze,%20A.%20et%20al.%20Wound-induced%20shoot-to-root%20relocation%20of%20ja-ile%20precursors%20coordinates%20arabidopsis%20growth.%20Mol.%20Plant12,%201383%E2%80%931394%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Kehr, J. &amp; Kragler, F. Long distance RNA movement. <em>N. Phytol.</em><strong>218</strong>, 29–40 (2018).</cite> [<a href="https://doi.org/10.1111/nph.15025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29418002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kehr,%20J.%20&amp;%20Kragler,%20F.%20Long%20distance%20RNA%20movement.%20N.%20Phytol.218,%2029%E2%80%9340%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Kondhare, K. R., Patil, N. S. &amp; Banerjee, A. K. A historical overview of long-distance signalling in plants. <em>J. Exp. Bot.</em><strong>72</strong>, 4218–4236 (2021).
</cite> [<a href="https://doi.org/10.1093/jxb/erab048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33682884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kondhare,%20K.%20R.,%20Patil,%20N.%20S.%20&amp;%20Banerjee,%20A.%20K.%20A%20historical%20overview%20of%20long-distance%20signalling%20in%20plants.%20J.%20Exp.%20Bot.72,%204218%E2%80%934236%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Ruiz-Medrano, R., Xoconostle-Cazares, B. &amp; Lucas, W. J. Phloem long-distance transport of CmNACP mRNA: implications for supracellular regulation in plants. <em>Development</em><strong>126</strong>, 4405–4419 (1999).
</cite> [<a href="https://doi.org/10.1242/dev.126.20.4405" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10498677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruiz-Medrano,%20R.,%20Xoconostle-Cazares,%20B.%20&amp;%20Lucas,%20W.%20J.%20Phloem%20long-distance%20transport%20of%20CmNACP%20mRNA:%20implications%20for%20supracellular%20regulation%20in%20plants.%20Development126,%204405%E2%80%934419%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Lucas, W. J. et al. Selective trafficking of KNOTTED1 homeodomain protein and its mRNA through plasmodesmata. <em>Science</em><strong>270</strong>, 1980–1983 (1995).
</cite> [<a href="https://doi.org/10.1126/science.270.5244.1980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8533088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lucas,%20W.%20J.%20et%20al.%20Selective%20trafficking%20of%20KNOTTED1%20homeodomain%20protein%20and%20its%20mRNA%20through%20plasmodesmata.%20Science270,%201980%E2%80%931983%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Banerjee, A. K. et al. Dynamics of a mobile RNA of potato involved in a long-distance signaling pathway. <em> Plant Cell</em><strong>18</strong>, 3443–3457 (2006).
</cite> [<a href="https://doi.org/10.1105/tpc.106.042473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1785412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17189340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banerjee,%20A.%20K.%20et%20al.%20Dynamics%20of%20a%20mobile%20RNA%20of%20potato%20involved%20in%20a%20long-distance%20signaling%20pathway.%20Plant%20Cell18,%203443%E2%80%933457%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Kim, M., Canio, W., Kessler, S. &amp; Sinha, N. Developmental changes due to long-distance movement of a homeobox fusion transcript in tomato. <em>Science</em><strong>293</strong>, 287–289 (2001).
</cite> [<a href="https://doi.org/10.1126/science.1059805" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11452121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20M.,%20Canio,%20W.,%20Kessler,%20S.%20&amp;%20Sinha,%20N.%20Developmental%20changes%20due%20to%20long-distance%20movement%20of%20a%20homeobox%20fusion%20transcript%20in%20tomato.%20Science293,%20287%E2%80%93289%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Haywood, V., Yu, T. S., Huang, N. C. &amp; Lucas, W. J. Phloem long-distance trafficking of gibberellic acid-insensitive RNA regulates leaf development. <em>Plant J.</em><strong>42</strong>, 49–68 (2005).
</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2005.02351.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15773853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haywood,%20V.,%20Yu,%20T.%20S.,%20Huang,%20N.%20C.%20&amp;%20Lucas,%20W.%20J.%20Phloem%20long-distance%20trafficking%20of%20gibberellic%20acid-insensitive%20RNA%20regulates%20leaf%20development.%20Plant%20J.42,%2049%E2%80%9368%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Huang, N. C., Luo, K. R. &amp; Yu, T. S. Mobility of antiflorigen and PEBP mRNAs in tomato-tobacco heterografts. <em>Plant Physiol.</em><strong>178</strong>, 783–794 (2018).
</cite> [<a href="https://doi.org/10.1104/pp.18.00725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6181055/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30150303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20N.%20C.,%20Luo,%20K.%20R.%20&amp;%20Yu,%20T.%20S.%20Mobility%20of%20antiflorigen%20and%20PEBP%20mRNAs%20in%20tomato-tobacco%20heterografts.%20Plant%20Physiol.178,%20783%E2%80%93794%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Yang, L. et al. m(5)C Methylation guides systemic transport of messenger RNA over graft junctions in plants. <em>Curr. Biol.</em><strong>29</strong>, 2465–2476 e2465 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cub.2019.06.042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31327714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20L.%20et%20al.%20m(5)C%20Methylation%20guides%20systemic%20transport%20of%20messenger%20RNA%20over%20graft%20junctions%20in%20plants.%20Curr.%20Biol.29,%202465%E2%80%932476%20e2465%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Zhang, H. et al. Expression of tomato prosystemin gene in Arabidopsis reveals systemic translocation of its mRNA and confers necrotrophic fungal resistance. <em>N. Phytol.</em><strong>217</strong>, 799–812 (2018).</cite> [<a href="https://doi.org/10.1111/nph.14858" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29105094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20H.%20et%20al.%20Expression%20of%20tomato%20prosystemin%20gene%20in%20Arabidopsis%20reveals%20systemic%20translocation%20of%20its%20mRNA%20and%20confers%20necrotrophic%20fungal%20resistance.%20N.%20Phytol.217,%20799%E2%80%93812%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Yang, L. et al. Noncell-autonomous HSC70.1 chaperone displays homeostatic feedback regulation by binding its own mRNA. <em>N. Phytol.</em><strong>237</strong>, 2404–2421 (2023).</cite> [<a href="https://doi.org/10.1111/nph.18703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36564968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20L.%20et%20al.%20Noncell-autonomous%20HSC70.1%20chaperone%20displays%20homeostatic%20feedback%20regulation%20by%20binding%20its%20own%20mRNA.%20N.%20Phytol.237,%202404%E2%80%932421%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Duan, X. et al. PbWoxT1 mRNA from pear (Pyrus betulaefolia) undergoes long-distance transport assisted by a polypyrimidine tract binding protein. <em>N. Phytol.</em><strong>210</strong>, 511–524 (2016).</cite> [<a href="https://doi.org/10.1111/nph.13793" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26661583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duan,%20X.%20et%20al.%20PbWoxT1%20mRNA%20from%20pear%20(Pyrus%20betulaefolia)%20undergoes%20long-distance%20transport%20assisted%20by%20a%20polypyrimidine%20tract%20binding%20protein.%20N.%20Phytol.210,%20511%E2%80%93524%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Wang, S. et al. A chaperonin containing T-complex polypeptide-1 facilitates the formation of the PbWoxT1-PbPTB3 ribonucleoprotein complex for long-distance RNA trafficking in Pyrus betulaefolia. <em>N. Phytol.</em><strong>238</strong>, 1115–1128 (2023).</cite> [<a href="https://doi.org/10.1111/nph.18789" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36751904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20S.%20et%20al.%20A%20chaperonin%20containing%20T-complex%20polypeptide-1%20facilitates%20the%20formation%20of%20the%20PbWoxT1-PbPTB3%20ribonucleoprotein%20complex%20for%20long-distance%20RNA%20trafficking%20in%20Pyrus%20betulaefolia.%20N.%20Phytol.238,%201115%E2%80%931128%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Hao, L. et al. A constitutive and drought-responsive mRNA undergoes long-distance transport in pear (<em>Pyrus betulaefolia</em>) phloem. <em>Plant Sci.</em><strong>293</strong>, 110419 (2020).
</cite> [<a href="https://doi.org/10.1016/j.plantsci.2020.110419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32081266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hao,%20L.%20et%20al.%20A%20constitutive%20and%20drought-responsive%20mRNA%20undergoes%20long-distance%20transport%20in%20pear%20(Pyrus%20betulaefolia)%20phloem.%20Plant%20Sci.293,%20110419%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Devers, E. A. et al. Movement and differential consumption of short interfering RNA duplexes underlie mobile RNA interference. <em>Nat. Plants</em><strong>6</strong>, 789–799 (2020).
</cite> [<a href="https://doi.org/10.1038/s41477-020-0687-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32632272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Devers,%20E.%20A.%20et%20al.%20Movement%20and%20differential%20consumption%20of%20short%20interfering%20RNA%20duplexes%20underlie%20mobile%20RNA%20interference.%20Nat.%20Plants6,%20789%E2%80%93799%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Thieme, C. J. et al. Endogenous Arabidopsis messenger RNAs transported to distant tissues. <em>Nat. Plants</em><strong>1</strong>, 15025 (2015).
</cite> [<a href="https://doi.org/10.1038/nplants.2015.25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27247031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thieme,%20C.%20J.%20et%20al.%20Endogenous%20Arabidopsis%20messenger%20RNAs%20transported%20to%20distant%20tissues.%20Nat.%20Plants1,%2015025%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Liu, W. et al. Rootstock-scion exchanging mRNAs participate in the pathways of amino acids and fatty acid metabolism in cucumber under early chilling stress. <em>Hortic. Res.</em>10.1093/hr/uhac031 (2022).</cite> [<a href="https://doi.org/10.1093/hr/uhac031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9039506/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35184197/" class="usa-link">PubMed</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Liu, W. et al. Identification of long-distance transmissible mRNA between scion and rootstock in cucurbit seedling heterografts. <em>Int. J. Mol. Sci.</em>10.3390/ijms21155253 (2020).</cite> [<a href="https://doi.org/10.3390/ijms21155253" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7432352/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32722102/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Zhang, W. et al. tRNA-related sequences trigger systemic mRNA transport in plants. <em>Plant Cell</em><strong>28</strong>, 1237–1249 (2016).
</cite> [<a href="https://doi.org/10.1105/tpc.15.01056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4944404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27268430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20W.%20et%20al.%20tRNA-related%20sequences%20trigger%20systemic%20mRNA%20transport%20in%20plants.%20Plant%20Cell28,%201237%E2%80%931249%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Wang, L., Sadeghnezhad, E. &amp; Nick, P. Upstream of gene expression: what is the role of microtubules in cold signalling?. <em>J. Exp. Bot.</em><strong>71</strong>, 36–48 (2020).
</cite> [<a href="https://doi.org/10.1093/jxb/erz419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31560041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20L.,%20Sadeghnezhad,%20E.%20&amp;%20Nick,%20P.%20Upstream%20of%20gene%20expression:%20what%20is%20the%20role%20of%20microtubules%20in%20cold%20signalling?.%20J.%20Exp.%20Bot.71,%2036%E2%80%9348%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Li, Y. et al. Isoleucine enhances plant resistance against <em>Botrytis cinerea</em> via jasmonate signaling pathway. <em>Front Plant Sci.</em><strong>12</strong>, 628328 (2021).
</cite> [<a href="https://doi.org/10.3389/fpls.2021.628328" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8416682/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34489985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Y.%20et%20al.%20Isoleucine%20enhances%20plant%20resistance%20against%20Botrytis%20cinerea%20via%20jasmonate%20signaling%20pathway.%20Front%20Plant%20Sci.12,%20628328%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Staswick, P. E. &amp; Tiryaki, I. The oxylipin signal jasmonic acid is activated by an enzyme that conjugates it to isoleucine in Arabidopsis. <em>Plant Cell</em><strong>16</strong>, 2117–2127 (2004).
</cite> [<a href="https://doi.org/10.1105/tpc.104.023549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC519202/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15258265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Staswick,%20P.%20E.%20&amp;%20Tiryaki,%20I.%20The%20oxylipin%20signal%20jasmonic%20acid%20is%20activated%20by%20an%20enzyme%20that%20conjugates%20it%20to%20isoleucine%20in%20Arabidopsis.%20Plant%20Cell16,%202117%E2%80%932127%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Van Moerkercke, A. et al. A MYC2/MYC3/MYC4-dependent transcription factor network regulates water spray-responsive gene expression and jasmonate levels. <em>Proc. Natl Acad. Sci. USA</em><strong>116</strong>, 23345–23356 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1911758116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6859355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31662474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Moerkercke,%20A.%20et%20al.%20A%20MYC2/MYC3/MYC4-dependent%20transcription%20factor%20network%20regulates%20water%20spray-responsive%20gene%20expression%20and%20jasmonate%20levels.%20Proc.%20Natl%20Acad.%20Sci.%20USA116,%2023345%E2%80%9323356%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Bao, H. et al. The developing xylem transcriptome and genome-wide analysis of alternative splicing in Populus trichocarpa (black cottonwood) populations. <em>BMC Genomics</em><strong>14</strong>, 359 (2013).
</cite> [<a href="https://doi.org/10.1186/1471-2164-14-359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3680236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23718132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bao,%20H.%20et%20al.%20The%20developing%20xylem%20transcriptome%20and%20genome-wide%20analysis%20of%20alternative%20splicing%20in%20Populus%20trichocarpa%20(black%20cottonwood)%20populations.%20BMC%20Genomics14,%20359%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Sureshkumar, S. et al. A genetic defect caused by a triplet repeat expansion in <em>Arabidopsis thaliana</em>. <em>Science</em><strong>323</strong>, 1060–1063 (2009).
</cite> [<a href="https://doi.org/10.1126/science.1164014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19150812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sureshkumar,%20S.%20et%20al.%20A%20genetic%20defect%20caused%20by%20a%20triplet%20repeat%20expansion%20in%20Arabidopsis%20thaliana.%20Science323,%201060%E2%80%931063%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Tabib, A. et al. A polynucleotide repeat expansion causing temperature-sensitivity persists in wild irish accessions of <em>Arabidopsis thaliana</em>. <em>Front. Plant Sci.</em><strong>7</strong>, 1311 (2016).
</cite> [<a href="https://doi.org/10.3389/fpls.2016.01311" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5006647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27630650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tabib,%20A.%20et%20al.%20A%20polynucleotide%20repeat%20expansion%20causing%20temperature-sensitivity%20persists%20in%20wild%20irish%20accessions%20of%20Arabidopsis%20thaliana.%20Front.%20Plant%20Sci.7,%201311%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Heinemann, B. &amp; Hildebrandt, T. M. The role of amino acid metabolism in signaling and metabolic adaptation to stress-induced energy deficiency in plants. <em>J. Exp. Bot.</em><strong>72</strong>, 4634–4645 (2021).
</cite> [<a href="https://doi.org/10.1093/jxb/erab182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33993299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heinemann,%20B.%20&amp;%20Hildebrandt,%20T.%20M.%20The%20role%20of%20amino%20acid%20metabolism%20in%20signaling%20and%20metabolic%20adaptation%20to%20stress-induced%20energy%20deficiency%20in%20plants.%20J.%20Exp.%20Bot.72,%204634%E2%80%934645%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Liu, Y. et al. Diverse nitrogen signals activate convergent ROP2-TOR signaling in Arabidopsis. <em>Dev. Cell</em><strong>56</strong>, 1283–1295 e1285 (2021).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2021.03.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33831352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20et%20al.%20Diverse%20nitrogen%20signals%20activate%20convergent%20ROP2-TOR%20signaling%20in%20Arabidopsis.%20Dev.%20Cell56,%201283%E2%80%931295%20e1285%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Cao, P. et al. Homeostasis of branched-chain amino acids is critical for the activity of TOR signaling in Arabidopsis. <em>Elife</em>10.7554/eLife.50747 (2019).</cite> [<a href="https://doi.org/10.7554/eLife.50747" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6937141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31808741/" class="usa-link">PubMed</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Liu, Y. &amp; Xiong, Y. Plant target of rapamycin signaling network: Complexes, conservations, and specificities. <em>J. Integr. Plant Biol.</em><strong>64</strong>, 342–370 (2022).
</cite> [<a href="https://doi.org/10.1111/jipb.13212" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34964268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20&amp;%20Xiong,%20Y.%20Plant%20target%20of%20rapamycin%20signaling%20network:%20Complexes,%20conservations,%20and%20specificities.%20J.%20Integr.%20Plant%20Biol.64,%20342%E2%80%93370%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Dong, Y. et al. TOR represses stress responses through global regulation of H3K27 trimethylation in plants. <em>J. Exp. Bot.</em>10.1093/jxb/erac486 (2022).</cite> [<a href="https://doi.org/10.1093/jxb/erac486" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36515098/" class="usa-link">PubMed</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Li, X. et al. m5C and m6A modifications regulate the mobility of pumpkin CHOLINE KINASE 1 mRNA under chilling stress. <em>Plant Physiol.</em>10.1093/plphys/kiae511 (2024).</cite> [<a href="https://doi.org/10.1093/plphys/kiae511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39325727/" class="usa-link">PubMed</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Dong, Y., Teleman, A., Jedmowski, C., Wirtz, M. &amp; Hell, R. The Arabidopsis THADA homolog modulates TOR activity and cold acclimation. <em>Plant Biol. (Stuttg)</em>10.1111/plb.12893 (2018).</cite> [<a href="https://doi.org/10.1111/plb.12893" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30098100/" class="usa-link">PubMed</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Geng, S. et al. An efficient root transformation system for CRISPR/Cas9-based analyses of shoot-root communication in cucurbit crops. <em>Hortic. Res.</em>10.1093/hr/uhab082 (2022).</cite> [<a href="https://doi.org/10.1093/hr/uhab082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9071382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35048110/" class="usa-link">PubMed</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Qin, Y. et al. γ-aminobutyric acid contributes to a novel long-distance signaling in figleaf gourd rootstock-induced cold tolerance of grafted cucumber seedlings. <em>Plant Physiol. Biochem.</em><strong>216</strong>, 109168 (2024).
</cite> [<a href="https://doi.org/10.1016/j.plaphy.2024.109168" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39366198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qin,%20Y.%20et%20al.%20%CE%B3-aminobutyric%20acid%20contributes%20to%20a%20novel%20long-distance%20signaling%20in%20figleaf%20gourd%20rootstock-induced%20cold%20tolerance%20of%20grafted%20cucumber%20seedlings.%20Plant%20Physiol.%20Biochem.216,%20109168%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Pan, X., Welti, R. &amp; Wang, X. Quantitative analysis of major plant hormones in crude plant extracts by high-performance liquid chromatography-mass spectrometry. <em>Nat. Protoc.</em><strong>5</strong>, 986–992 (2010).
</cite> [<a href="https://doi.org/10.1038/nprot.2010.37" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20448544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan,%20X.,%20Welti,%20R.%20&amp;%20Wang,%20X.%20Quantitative%20analysis%20of%20major%20plant%20hormones%20in%20crude%20plant%20extracts%20by%20high-performance%20liquid%20chromatography-mass%20spectrometry.%20Nat.%20Protoc.5,%20986%E2%80%93992%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Flokova, K. et al. UHPLC-MS/MS based target profiling of stress-induced phytohormones. <em>Phytochemistry</em><strong>105</strong>, 147–157 (2014).
</cite> [<a href="https://doi.org/10.1016/j.phytochem.2014.05.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24947339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Flokova,%20K.%20et%20al.%20UHPLC-MS/MS%20based%20target%20profiling%20of%20stress-induced%20phytohormones.%20Phytochemistry105,%20147%E2%80%93157%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Patti, G. J., Tautenhahn, R. &amp; Siuzdak, G. Meta-analysis of untargeted metabolomic data from multiple profiling experiments. <em>Nat. Protoc.</em><strong>7</strong>, 508–516 (2012).
</cite> [<a href="https://doi.org/10.1038/nprot.2011.454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3683249/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22343432/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patti,%20G.%20J.,%20Tautenhahn,%20R.%20&amp;%20Siuzdak,%20G.%20Meta-analysis%20of%20untargeted%20metabolomic%20data%20from%20multiple%20profiling%20experiments.%20Nat.%20Protoc.7,%20508%E2%80%93516%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368190/bin/41467_2025_63228_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (206.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368190/bin/41467_2025_63228_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63228_MOESM2_ESM.pdf</a><sup> (87.3KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368190/bin/41467_2025_63228_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1-9</a><sup> (363.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368190/bin/41467_2025_63228_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (2.7MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368190/bin/41467_2025_63228_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368190/bin/41467_2025_63228_MOESM6_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (98MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data supporting the results of this article are included within the article and its additional files. Source Data are provided with this paper. The original RNA sequencing data used in this study are deposited in the NCBI Small Read Archive under accession numbers <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA552914/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA552914</a> and <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA673087/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA673087</a>. The accession code of all genes used in this study was obtained from the Cucurbit Genomics Database (Cucumber Gy14 v2 <a href="http://cucurbitgenomics.org/organism/16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://cucurbitgenomics.org/organism/16</a>, <em>Cucurbita moschata</em> (Rifu) genome <a href="http://cucurbitgenomics.org/organism/9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://cucurbitgenomics.org/organism/9</a>) or the <a href="https://www.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCBI</a> and are provided in the Supplementary Dataset 8. <a href="#Sec24" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63228-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63228.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368190/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368190/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368190%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368190/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368190/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368190/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835858/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368190/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835858/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368190/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368190/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="hq4BsCJ2ZPgmtQRqmxrvrUGU0xqmi1ZKZkCQFRwMCL12dQhEZ24CFMagfIFuNBa9">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
