
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Inhibitor‐induced dimerization mediates lufotrelvir resistance in mutants of SARS‐CoV‐2 3C‐like protease - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15251BFE8AEEF3A3051BFE003F923314.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="prosci">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375981/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Protein Science : A Publication of the Protein Society">
<meta name="citation_title" content="Inhibitor‐induced dimerization mediates lufotrelvir resistance in mutants of SARS‐CoV‐2 3C‐like protease">
<meta name="citation_author" content="Guanyu Wang">
<meta name="citation_author_institution" content="Department of Chemistry, McGill University, Montreal, Quebec, Canada">
<meta name="citation_author" content="Felipe Venegas">
<meta name="citation_author_institution" content="Department of Chemistry, McGill University, Montreal, Quebec, Canada">
<meta name="citation_author_institution" content="Departamento de Ciencias Biomédicas, Facultad de Medicina, Universidad Católica del Norte, Coquimbo, Chile">
<meta name="citation_author_institution" content="Programa de Doctorado en Ciencias Biomédicas, Facultad de Medicina, Universidad Católica del Norte, Coquimbo, Chile">
<meta name="citation_author" content="Andres Rueda">
<meta name="citation_author_institution" content="Department of Chemistry, McGill University, Montreal, Quebec, Canada">
<meta name="citation_author" content="Osvaldo Yañez">
<meta name="citation_author_institution" content="Centro de Modelación Ambiental y Dinámica de Sistemas (CEMADIS), Facultad de Ingeniería y Negocios, Universidad de Las Américas, Santiago, Chile">
<meta name="citation_author" content="Manuel I Osorio">
<meta name="citation_author_institution" content="Facultad de Odontología, Universidad Andres Bello, Santiago, Chile">
<meta name="citation_author_institution" content="Facultad de Medicina, Universidad Diego Portales, Santiago, Chile">
<meta name="citation_author" content="Sibei Qin">
<meta name="citation_author_institution" content="Department of Chemistry, McGill University, Montreal, Quebec, Canada">
<meta name="citation_author" content="José Manuel Pérez‐Donoso">
<meta name="citation_author_institution" content="BioNanotechnology and Microbiology Laboratory, Center for Bioinformatics and Integrative Biology (CBIB), Facultad de Ciencias de la Vida, Universidad Andres Bello, Santiago, Chile">
<meta name="citation_author" content="Christopher J Thibodeaux">
<meta name="citation_author_institution" content="Department of Chemistry, McGill University, Montreal, Quebec, Canada">
<meta name="citation_author" content="Nicolas Moitessier">
<meta name="citation_author_institution" content="Department of Chemistry, McGill University, Montreal, Quebec, Canada">
<meta name="citation_author" content="Anthony K Mittermaier">
<meta name="citation_author_institution" content="Department of Chemistry, McGill University, Montreal, Quebec, Canada">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="34">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70275">
<meta name="citation_doi" content="10.1002/pro.70275">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375981/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375981/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375981/pdf/PRO-34-e70275.pdf">
<meta name="description" content="The emergence of SARS‐CoV‐2 and other lethal coronaviruses has prompted extensive research into targeted antiviral treatments, particularly focusing on the viral 3C‐like protease (3CLpro) due to its essential role for viral replication. However, the ...">
<meta name="og:title" content="Inhibitor‐induced dimerization mediates lufotrelvir resistance in mutants of SARS‐CoV‐2 3C‐like protease">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The emergence of SARS‐CoV‐2 and other lethal coronaviruses has prompted extensive research into targeted antiviral treatments, particularly focusing on the viral 3C‐like protease (3CLpro) due to its essential role for viral replication. However, the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375981/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375981">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/pro.70275"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/PRO-34-e70275.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375981%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375981/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375981/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375981/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-prosci.gif" alt="Protein Science : A Publication of the Protein Society logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Protein Science : A Publication of the Protein Society" title="Link to Protein Science : A Publication of the Protein Society" shape="default" href="http://www.proteinscience.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Protein Sci</button></div>. 2025 Aug 25;34(9):e70275. doi: <a href="https://doi.org/10.1002/pro.70275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/pro.70275</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Protein%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Protein%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Inhibitor‐induced dimerization mediates lufotrelvir resistance in mutants of SARS‐CoV‐2 3C‐like protease
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Guanyu Wang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Guanyu Wang</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, McGill University, Montreal, Quebec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guanyu Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venegas%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Felipe Venegas</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Felipe Venegas</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, McGill University, Montreal, Quebec, Canada</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Departamento de Ciencias Biomédicas, Facultad de Medicina, Universidad Católica del Norte, Coquimbo, Chile</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Programa de Doctorado en Ciencias Biomédicas, Facultad de Medicina, Universidad Católica del Norte, Coquimbo, Chile</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venegas%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Felipe Venegas</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rueda%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Andres Rueda</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Andres Rueda</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, McGill University, Montreal, Quebec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rueda%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andres Rueda</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ya%C3%B1ez%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Osvaldo Yañez</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Osvaldo Yañez</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Centro de Modelación Ambiental y Dinámica de Sistemas (CEMADIS), Facultad de Ingeniería y Negocios, Universidad de Las Américas, Santiago, Chile</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ya%C3%B1ez%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Osvaldo Yañez</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Osorio%20MI%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Manuel I Osorio</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Manuel I Osorio</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Facultad de Odontología, Universidad Andres Bello, Santiago, Chile</div>
<div class="p">
<sup>
<sup>6</sup>
</sup>Facultad de Medicina, Universidad Diego Portales, Santiago, Chile</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Osorio%20MI%22%5BAuthor%5D" class="usa-link"><span class="name western">Manuel I Osorio</span></a>
</div>
</div>
<sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qin%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Sibei Qin</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Sibei Qin</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, McGill University, Montreal, Quebec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qin%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sibei Qin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22P%C3%A9rez%E2%80%90Donoso%20JM%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">José Manuel Pérez‐Donoso</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">José Manuel Pérez‐Donoso</span></h3>
<div class="p">
<sup>
<sup>7</sup>
</sup>BioNanotechnology and Microbiology Laboratory, Center for Bioinformatics and Integrative Biology (CBIB), Facultad de Ciencias de la Vida, Universidad Andres Bello, Santiago, Chile</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22P%C3%A9rez%E2%80%90Donoso%20JM%22%5BAuthor%5D" class="usa-link"><span class="name western">José Manuel Pérez‐Donoso</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thibodeaux%20CJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Christopher J Thibodeaux</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Christopher J Thibodeaux</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, McGill University, Montreal, Quebec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thibodeaux%20CJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Christopher J Thibodeaux</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moitessier%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Nicolas Moitessier</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Nicolas Moitessier</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, McGill University, Montreal, Quebec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moitessier%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nicolas Moitessier</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mittermaier%20AK%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Anthony K Mittermaier</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Anthony K Mittermaier</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, McGill University, Montreal, Quebec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mittermaier%20AK%22%5BAuthor%5D" class="usa-link"><span class="name western">Anthony K Mittermaier</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="pro70275-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, McGill University, Montreal, Quebec, Canada</div>
<div id="pro70275-aff-0002">
<sup>
<sup>2</sup>
</sup>Departamento de Ciencias Biomédicas, Facultad de Medicina, Universidad Católica del Norte, Coquimbo, Chile</div>
<div id="pro70275-aff-0003">
<sup>
<sup>3</sup>
</sup>Programa de Doctorado en Ciencias Biomédicas, Facultad de Medicina, Universidad Católica del Norte, Coquimbo, Chile</div>
<div id="pro70275-aff-0004">
<sup>
<sup>4</sup>
</sup>Centro de Modelación Ambiental y Dinámica de Sistemas (CEMADIS), Facultad de Ingeniería y Negocios, Universidad de Las Américas, Santiago, Chile</div>
<div id="pro70275-aff-0005">
<sup>
<sup>5</sup>
</sup>Facultad de Odontología, Universidad Andres Bello, Santiago, Chile</div>
<div id="pro70275-aff-0006">
<sup>
<sup>6</sup>
</sup>Facultad de Medicina, Universidad Diego Portales, Santiago, Chile</div>
<div id="pro70275-aff-0007">
<sup>
<sup>7</sup>
</sup>BioNanotechnology and Microbiology Laboratory, Center for Bioinformatics and Integrative Biology (CBIB), Facultad de Ciencias de la Vida, Universidad Andres Bello, Santiago, Chile</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, 
Anthony K. Mittermaier, Department of Chemistry, McGill University, 801 Sherbrooke St W, Montreal, QC H3A 0B8, Canada. 
Email: <span>anthony.mittermaier@mcgill.ca</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jul 17; Received 2025 Apr 22; Accepted 2025 Aug 7; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Protein Science</em> published by Wiley Periodicals LLC on behalf of The Protein Society.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited and is not used for commercial purposes.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375981</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The emergence of SARS‐CoV‐2 and other lethal coronaviruses has prompted extensive research into targeted antiviral treatments, particularly focusing on the viral 3C‐like protease (3CL<sup>pro</sup>) due to its essential role for viral replication. However, the rise of drug resistance mutations poses threats to public health and underscores the need to predict resistance mutations and understand the mechanism of how these mutations confer resistance. The binding of inhibitor to 3CL<sup>pro</sup> drives it from the monomeric to the active dimeric form, which can counterintuitively lead to enzyme activation rather than inhibition. Furthermore, we find this allosteric coupling between binding and dimerization is sensitive to mutation, leading to a new mechanism for drug resistance. Understanding the relationship between inhibitor binding and dimerization is important for resistant strain surveillance and development of robust antivirals. Herein, we present a systematic study of drug resistance mediated by inhibitor‐induced dimerization of 3CL<sup>pro</sup>.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> 3CL<sup>pro</sup>
, allosteric coupling, coronavirus, drug resistance, ligand‐induced dimerization, SARS‐CoV‐2</p></section></section><section id="pro70275-sec-0001"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Coronaviruses have long been recognized as human pathogens (Kahn &amp; McIntosh, <a href="#pro70275-bib-0032" class="usa-link" aria-describedby="pro70275-bib-0032">2005</a>). However, they were not considered a major public health threat until relatively recently, as they were primarily associated with mild upper respiratory tract infections, similar to the common cold, and the significance of these viruses was largely underestimated (Kapikian, <a href="#pro70275-bib-0033" class="usa-link" aria-describedby="pro70275-bib-0033">1975</a>; Macnaughton, <a href="#pro70275-bib-0041" class="usa-link" aria-describedby="pro70275-bib-0041"><sup>1982</sup></a>; Myint, <a href="#pro70275-bib-0047" class="usa-link" aria-describedby="pro70275-bib-0047"><sup>1995</sup></a>; Peiris et al., <a href="#pro70275-bib-0053" class="usa-link" aria-describedby="pro70275-bib-0053"><sup>2003</sup></a>; Tyrrell &amp; Bynoe, <a href="#pro70275-bib-0065" class="usa-link" aria-describedby="pro70275-bib-0065"><sup>1965</sup></a>). The emergence of severe acute respiratory syndrome coronavirus 1 (SARS‐CoV‐1) in 2002 (Peiris et al., <a href="#pro70275-bib-0053" class="usa-link" aria-describedby="pro70275-bib-0053">2003</a>), followed by Middle East respiratory syndrome coronavirus (MERS‐CoV in 2012) (Zhang et al., <a href="#pro70275-bib-0078" class="usa-link" aria-describedby="pro70275-bib-0078">2021</a>), and severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) (the coronavirus disease 2019 (COVID‐19 pandemic) in 2019 (WHO, <a href="#pro70275-bib-0070" class="usa-link" aria-describedby="pro70275-bib-0070"><sup>2024</sup></a>), drastically changed the perception of coronaviruses, highlighting their potential to cause severe disease in humans and create global health emergencies. Since the beginning of the COVID‐19 pandemic, there have been numerous drug campaigns aiming to discover effective antivirals against SARS‐CoV‐2. These small molecule antivirals usually target various stages of the virus replication cycle.</p>
<p>3CL<sup>pro</sup> (also known as main protease, M<sup>pro</sup>) is regarded as an attractive drug target as it is responsible for proteolytic processing of the majority of viral polyprotein cleavage sites, releasing nonstructural proteins (nsps), including RNA‐dependent RNA polymerase (RdRp), which are essential for the viral lifecycle (Thiel et al., <a href="#pro70275-bib-0063" class="usa-link" aria-describedby="pro70275-bib-0063">2003</a>; V'kovski et al., <a href="#pro70275-bib-0067" class="usa-link" aria-describedby="pro70275-bib-0067"><sup>2021</sup></a>; Ziebuhr et al., <a href="#pro70275-bib-0082" class="usa-link" aria-describedby="pro70275-bib-0082"><sup>2000</sup></a>). 3CL<sup>pro</sup> is a highly conserved, dimeric cysteine protease (Graziano et al., <a href="#pro70275-bib-0015" class="usa-link" aria-describedby="pro70275-bib-0015">2006</a>; Hsu et al., <a href="#pro70275-bib-0023" class="usa-link" aria-describedby="pro70275-bib-0023"><sup>2005</sup></a>; Mody et al., <a href="#pro70275-bib-0043" class="usa-link" aria-describedby="pro70275-bib-0043"><sup>2021</sup></a>), which is only active in the dimeric form (Chen, Zhang, et al., <a href="#pro70275-bib-0005" class="usa-link" aria-describedby="pro70275-bib-0005">2008</a>; Ferreira et al., <a href="#pro70275-bib-0010" class="usa-link" aria-describedby="pro70275-bib-0010"><sup>2022</sup></a>; Graziano et al., <a href="#pro70275-bib-0015" class="usa-link" aria-describedby="pro70275-bib-0015"><sup>2006</sup></a>; Nashed et al., <a href="#pro70275-bib-0048" class="usa-link" aria-describedby="pro70275-bib-0048"><sup>2022</sup></a>; Zhang et al., <a href="#pro70275-bib-0080" class="usa-link" aria-describedby="pro70275-bib-0080"><sup>2010</sup></a>). Currently, the Food and Drug Administration (FDA) has approved one antiviral targeting the 3CL<sup>pro</sup> for the treatment of COVID‐19: the orally administered <em>Paxlovid</em> (ritonavir‐boosted nirmatrelvir) (U.S. Food &amp; Drug Administration, <a href="#pro70275-bib-0066" class="usa-link" aria-describedby="pro70275-bib-0066"><sup>2023</sup></a>). <em>Paxlovid</em> has an excellent 88% efficacy against hospitalization or death in unvaccinated high‐risk adults (Hammond et al., <a href="#pro70275-bib-0017" class="usa-link" aria-describedby="pro70275-bib-0017">2022</a>). In addition, <em>lufotrelvir</em> is another potent 3CL<sup>pro</sup> inhibitor developed by Pfizer as a potential treatment for COVID‐19 (Figure <a href="#pro70275-fig-0001" class="usa-link">1</a>) (Hoffman et al., <a href="#pro70275-bib-0022" class="usa-link" aria-describedby="pro70275-bib-0022">2020</a>), which also demonstrates broad‐spectrum effectiveness against various 3CL<sup>pro</sup> enzymes across the coronavirus genera (Boras et al., <a href="#pro70275-bib-0003" class="usa-link" aria-describedby="pro70275-bib-0003">2021</a>).</p>
<figure class="fig xbox font-sm" id="pro70275-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc3b/12375981/b3fb0989e7af/PRO-34-e70275-g001.jpg" loading="lazy" id="jats-graphic-1" height="317" width="669" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/pro70275-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Representative structures of 3CL<sup>pro</sup> inhibitors and their in vitro potency values (Rai et al., <a href="#pro70275-bib-0055" class="usa-link" aria-describedby="pro70275-bib-0055">2022</a>; Vries et al., <a href="#pro70275-bib-0068" class="usa-link" aria-describedby="pro70275-bib-0068"><sup>2021</sup></a>).</p></figcaption></figure><p>Drugs targeting viral replication run the risk of selecting for variants with mutations that allow viruses to evade the therapeutic effects, especially in the face of large virus populations and high mutation rates (Irwin et al., <a href="#pro70275-bib-0028" class="usa-link" aria-describedby="pro70275-bib-0028">2016</a>). Thus, the ability to avoid drug resistance mutations is essential for developing treatments with long‐term effectiveness. Several studies have already investigated 3CL<sup>pro</sup> mutations that confer resistance to nirmatrelvir (active ingredient of Paxlovid) using both computational methods (Havranek et al., <a href="#pro70275-bib-0019" class="usa-link" aria-describedby="pro70275-bib-0019">2023</a>; Sasi et al., <a href="#pro70275-bib-0058" class="usa-link" aria-describedby="pro70275-bib-0058"><sup>2022</sup></a>) and in vitro serial viral passage experiments (Heilmann et al., <a href="#pro70275-bib-0020" class="usa-link" aria-describedby="pro70275-bib-0020">2023</a>; Iketani et al., <a href="#pro70275-bib-0026" class="usa-link" aria-describedby="pro70275-bib-0026"><sup>2023</sup></a>; Jochmans et al., <a href="#pro70275-bib-0031" class="usa-link" aria-describedby="pro70275-bib-0031"><sup>2023</sup></a>; Zhou et al., <a href="#pro70275-bib-0081" class="usa-link" aria-describedby="pro70275-bib-0081"><sup>2022</sup></a>). Moreover, naturally occurring mutations that are resistant to nirmatrelvir have also been discovered (Hu et al., <a href="#pro70275-bib-0024" class="usa-link" aria-describedby="pro70275-bib-0024">2023</a>; Moghadasi et al., <a href="#pro70275-bib-0044" class="usa-link" aria-describedby="pro70275-bib-0044"><sup>2023</sup></a>; Noske et al., <a href="#pro70275-bib-0049" class="usa-link" aria-describedby="pro70275-bib-0049"><sup>2023</sup></a>). On the other hand, there have only been a small number of studies of 3CL<sup>pro</sup> resistance mutations against other potent inhibitors such as lufotrelvir (Hu et al., <a href="#pro70275-bib-0024" class="usa-link" aria-describedby="pro70275-bib-0024">2023</a>; Jiang et al., <a href="#pro70275-bib-0030" class="usa-link" aria-describedby="pro70275-bib-0030"><sup>2023</sup></a>). Understanding the link between mutation and resistance is important for managing the emergence of resistance and offers valuable insights for developing next‐generation antivirals.</p>
<p>3CL<sup>pro</sup> is only active as a homodimer, yet it naturally exists in equilibrium between its monomeric and dimeric forms in solution (Ferreira et al., <a href="#pro70275-bib-0010" class="usa-link" aria-describedby="pro70275-bib-0010">2022</a>). Interestingly, the monomer/dimer equilibrium is affected by ligand binding: 3CL<sup>pro</sup> from SARS‐CoV (Cheng et al., <a href="#pro70275-bib-0006" class="usa-link" aria-describedby="pro70275-bib-0006">2010</a>; Li et al., <a href="#pro70275-bib-0038" class="usa-link" aria-describedby="pro70275-bib-0038"><sup>2010</sup></a>), MERS‐CoV (Tomar et al., <a href="#pro70275-bib-0064" class="usa-link" aria-describedby="pro70275-bib-0064">2015</a>), and SARS‐CoV‐2 (Jochmans et al., <a href="#pro70275-bib-0031" class="usa-link" aria-describedby="pro70275-bib-0031">2023</a>; Nashed et al., <a href="#pro70275-bib-0048" class="usa-link" aria-describedby="pro70275-bib-0048"><sup>2022</sup></a>; Paciaroni et al., <a href="#pro70275-bib-0051" class="usa-link" aria-describedby="pro70275-bib-0051"><sup>2023</sup></a>) all exhibit ligand‐induced dimerization (LID), where the binding of a ligand to the monomer promotes dimerization and formation of the active dimer. It implies that allosteric communication occurs between the ligand binding site and the dimerization interface such that ligand binding increases the affinity of dimerization and vice versa. LID has been shown to be essential for the maturation of polyprotein and the production of mature SARS‐CoV 3CL<sup>pro</sup>, and is proposed as a mechanism to regulate enzyme activity during virus replication (Li et al., <a href="#pro70275-bib-0038" class="usa-link" aria-describedby="pro70275-bib-0038">2010</a>). Moreover, under conditions where the enzyme is primarily in the inactive monomeric form, LID can lead to the counter‐intuitive situation where enzyme activity initially increases with increasing concentrations of the inhibitor. Binding of inhibitor to one monomer promotes dimerization, with one subunit bound to the inhibitor and the other subunit unoccupied and active. As such, LID leads to distinctive dose–response curves, with increasing activity at low inhibitor concentrations, maximum activity at an intermediate inhibitor concentration, followed by complete inhibition at high inhibitor concentrations. Given that LID has been demonstrated as an enzyme activation mechanism that shifts inhibition to higher concentrations (Tomar et al., <a href="#pro70275-bib-0064" class="usa-link" aria-describedby="pro70275-bib-0064">2015</a>), we propose that LID can serve as a potential mechanism for drug resistance. However, to date, analyses of LID and its characteristic dose–response curves have only been qualitative, as no physical chemical model has been developed. Even though the effect of different inhibitors on the tendency of dimerization, defined by the dissociation constant, <em>K</em>
<sub>D</sub>, has been quantified as a function of the inhibitor concentration (Silvestrini et al., <a href="#pro70275-bib-0061" class="usa-link" aria-describedby="pro70275-bib-0061">2021</a>), a model that explains LID is still lacking and it still remains unclear whether LID actually attenuates inhibition as opposed to simply altering the shape of the dose–response curve. A quantitative, mechanistic description of LID is essential for understanding its potential role in drug resistance.</p>
<p>To better understand this mechanism, we focused on lufotrelvir and employed a combination of molecular dynamics (MD) simulations, site‐directed mutagenesis, and Förster resonance energy transfer (FRET) based activity assays. We developed a general, quantitative model of LID, which allowed us to separately extract the affinity of inhibitor binding, the strength of allosteric coupling between the ligand binding site and dimerization interface, and to calculate how modulating LID could mediate resistance. We demonstrated LID as a new and hitherto underappreciated mechanism that can be exploited by pathogens to evade inhibitor drugs.</p></section><section id="pro70275-sec-0002"><h2 class="pmc_sec_title">2. RESULTS</h2>
<section id="pro70275-sec-0003"><h3 class="pmc_sec_title">2.1. Computational predictions of lufotrelvir resistance mutations</h3>
<p>Possible sites of resistance mutations may be located by calculating the distances of enzyme/substrate and enzyme/inhibitor interactions on a per‐residue basis in crystal structures or simulations. Residues that contact the inhibitor substantially more than the substrate may be considered possible mutation sites, as mutations could preferentially disrupt interactions with the inhibitor while retaining catalytic activity (Friedman, <a href="#pro70275-bib-0012" class="usa-link" aria-describedby="pro70275-bib-0012">2022</a>). In this work, MD simulations were performed on 3CL<sup>pro</sup>/substrate (<em>TSAVLQSGFRK</em>) and 3CL<sup>pro</sup>/lufotrelvir complexes following previously published protocols (Osorio et al., <a href="#pro70275-bib-0050" class="usa-link" aria-describedby="pro70275-bib-0050">2022</a>; Yañez et al., <a href="#pro70275-bib-0076" class="usa-link" aria-describedby="pro70275-bib-0076"><sup>2021</sup></a>). The resulting trajectories were clustered into representative structures, and the strengths of interactions between 3CL<sup>pro</sup> residues and either the substrate or the inhibitor were quantified using non‐covalent interactions. These values were population‐averaged across the representative structures and are reported in parallel coordinate plots in Figure <a href="#pro70275-fig-0002" class="usa-link">2a</a>, comparing non‐covalent interaction values to the substrate and to the inhibitor. This comprehensive analysis revealed distinct interaction patterns between the substrate and inhibitor binding modes. By comparing these interaction profiles, as visualized, we identified specific residues that exhibited stronger interactions with the inhibitor compared to the substrate. This differential binding pattern was particularly significant because it suggested potential sites where mutations could selectively disrupt inhibitor binding while preserving substrate recognition. Four residues near the active site produced higher non‐covalent interaction values with the inhibitor than with the substrate, namely Y54, Q189, A191, and S1 (from the other subunit of the dimer). However, for Q189, the side chain was oriented away from the inhibitor and toward the solvent. Thus, mutations at this site were unlikely to affect inhibitor/enzyme interactions, and this residue was not investigated further. We therefore decided to focus our attention on S1, Y54, and A191, as shown in Figure <a href="#pro70275-fig-0002" class="usa-link">2b,c</a>.</p>
<figure class="fig xbox font-sm" id="pro70275-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc3b/12375981/d517029194b2/PRO-34-e70275-g006.jpg" loading="lazy" id="jats-graphic-3" height="512" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/pro70275-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Non‐covalent interactions of 3CL<sup>pro</sup> with the substrate and the inhibitor. (a) Interactions of the substrate compared with the inhibitor, where "*" marks residues with higher non‐covalent interaction values with the inhibitor than with the substrate. (b) Positions of the potential resistance‐inducing residues near the binding site of lufotrelvir (in magenta). (c) Positions of the potential resistance‐inducing residues in the overall dimeric structure of 3CL<sup>pro</sup> (Protein Data Bank [PDB] code: 6XHM).</p></figcaption></figure></section><section id="pro70275-sec-0004"><h3 class="pmc_sec_title">2.2. Experimental characterization of potential resistance mutations</h3>
<p>For each of the three residues identified in the MD analysis above, we generated three to four single site mutants that modified the physical properties of the targeted residue, for instance by changing the bulkiness, polarity, or charge, while restricting modifications to mono‐ or di‐nucleotide substitutions. For S1, S1A (polar to nonpolar), S1I (increased bulkiness), and S1D (neutral to negatively charged) mutations were selected. For Y54, Y54A (reduced bulkiness), Y54D (to negatively charged), Y54F (no OH), and Y54K (positively charged) were chosen. For A191, A191E (nonpolar to negatively charged), A191I (increased bulkiness), and A191R (nonpolar to positively charged) were selected.</p>
<p>All mutants were expressed with high yields in <em>Escherichia coli</em> and purified using previously published procedures (Stille et al., <a href="#pro70275-bib-0062" class="usa-link" aria-describedby="pro70275-bib-0062">2022</a>). Activity was initially tested at a fixed enzyme concentration of 15 nM in a FRET‐based enzyme assay (Figure <a href="#pro70275-fig-0003" class="usa-link">3a</a>) (Grum‐Tokars et al., <a href="#pro70275-bib-0016" class="usa-link" aria-describedby="pro70275-bib-0016">2008</a>). S1I, S1R, Y54F, and A191I mutants exhibited similar activities to the wild type (WT), while the S1A and A191E mutants were 1.5 and 1.9 times more active than the WT, respectively. On the other hand, Y54 mutants, except for Y54F, showed no enzyme activity. Y54 is a highly conserved residue among <em>Coronaviridae</em> and is important for maintaining the enzyme's catalytic activity and structural stability (Al Adem et al., <a href="#pro70275-bib-0001" class="usa-link" aria-describedby="pro70275-bib-0001">2023</a>). Mutation of this residue likely caused the binding site to collapse, resulting in inactive enzymes. Additionally, the S1D mutation was also detrimental to the enzyme, resulting in only 5% activity in comparison to the WT.</p>
<figure class="fig xbox font-sm" id="pro70275-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc3b/12375981/ffe82c70a35a/PRO-34-e70275-g005.jpg" loading="lazy" id="jats-graphic-5" height="280" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/pro70275-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Enzyme activities and dose–response curves of the wild‐type and mutant 3CL<sup>pro</sup> with lufotrelvir in FRET assay. (a) Relative enzyme activities of the wild‐type 3CL<sup>pro</sup> compared to the mutants, with the activity of wild‐type 3CL<sup>pro</sup> set to 100%. (b) Dose–response curves of the wild‐type 3CL<sup>pro</sup> with the mutants in logarithmic scale. All assays were performed at a fixed enzyme concentration of 15 nM and a fixed substrate concentration of 12 μM. The activity of the enzyme in the absence of inhibitor is set as 100% for each individual enzyme. All measurements are in triplicate. The error bars represent standard deviations of the data points.</p></figcaption></figure><p>Dose–response curves were measured with the WT and all the active mutants (Figure <a href="#pro70275-fig-0003" class="usa-link">3b</a>), and the inhibition constants (<em>K</em>
<sub>i</sub>) were determined for all of them except S1D (Table <a href="#pro70275-tbl-0001" class="usa-link">1</a>) by fitting to the Morrison equation with Equation (<a href="#pro70275-disp-0001" class="usa-link">1</a>) and (<a href="#pro70275-disp-0002" class="usa-link">2</a>), where [E], [I], and [S] refer to the molar concentrations of enzyme, inhibitor, and substrate, respectively, while <em>K</em>
<sub>m</sub> represents the Michaelis constant (Morrison, <a href="#pro70275-bib-0046" class="usa-link" aria-describedby="pro70275-bib-0046">1969</a>). The values of <em>K</em>
<sub>m</sub> are measured and presented in the later part of the paper. The <em>K</em>
<sub>i</sub> value of the WT (0.25 ± 0.04 nM) is in excellent agreement with the literature value (0.27 ± 0.01 nM) (Hoffman et al., <a href="#pro70275-bib-0022" class="usa-link" aria-describedby="pro70275-bib-0022">2020</a>). Most mutants had similar <em>K</em>
<sub>i</sub> values to that of the WT, except for the A191E and A191I mutations, which showed increased <em>K</em>
<sub>i</sub> values of 1.6 ± 0.1 and 1.2 ± 0.1 nM, respectively. We speculate that the close proximity of the 4‐methoxyindole moiety of lufotrelvir to A191 (Figure <a href="#pro70275-fig-0002" class="usa-link">2b</a>) led to some steric repulsion when the alanine was replaced with a bulkier residue.</p>
<table class="disp-formula p" id="pro70275-disp-0001"><tr>
<td class="formula"><math id="jats-math-1" display="block" overflow="linebreak"><mrow><mrow><mo>%</mo><mtext>Activity</mtext><mo linebreak="goodbreak">=</mo><mn>100</mn><mo linebreak="goodbreak">×</mo><mfenced><mrow><mn>1</mn><mo linebreak="goodbreak">−</mo><mfrac><mrow><mfenced><mrow><mfenced open="[" close="]"><mi mathvariant="normal">E</mi></mfenced><mo linebreak="goodbreak">+</mo><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mo linebreak="goodbreak">+</mo><msubsup><mi>K</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">app</mi></msubsup></mrow></mfenced><mo>−</mo><msqrt><mrow><msup><mfenced><mrow><mfenced open="[" close="]"><mi mathvariant="normal">E</mi></mfenced><mo linebreak="goodbreak">+</mo><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mo linebreak="goodbreak">+</mo><msubsup><mi>K</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">app</mi></msubsup></mrow></mfenced><mn>2</mn></msup><mo>−</mo><mn>4</mn><mfenced open="[" close="]"><mi mathvariant="normal">E</mi></mfenced><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced></mrow></msqrt></mrow><mrow><mn>2</mn><mfenced open="[" close="]"><mi mathvariant="normal">E</mi></mfenced></mrow></mfrac></mrow></mfenced><mo>.</mo></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<table class="disp-formula p" id="pro70275-disp-0002"><tr>
<td class="formula"><math id="jats-math-2" display="block" overflow="linebreak"><mrow><mrow><msubsup><mi>K</mi><mi mathvariant="normal">i</mi><mi>app</mi></msubsup><mo linebreak="goodbreak">=</mo><msub><mi>K</mi><mi mathvariant="normal">i</mi></msub><mfenced open="(" close=")"><mrow><mn>1</mn><mo linebreak="goodbreak">+</mo><mfrac><mfenced open="[" close="]"><mi mathvariant="normal">S</mi></mfenced><msub><mi>K</mi><mi mathvariant="normal">m</mi></msub></mfrac></mrow></mfenced><mo>.</mo></mrow></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<section class="tw xbox font-sm" id="pro70275-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Measured <em>K</em>
<sub>i</sub> values of lufotrelvir with the WT and mutant 3CL<sup>pro</sup>, with all values expressed as mean ± standard deviation (SD).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Enzyme</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>i</sub> (nM)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">WT</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.25 ± 0.04</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">S1A</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.41 ± 0.08</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">S1I</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.35 ± 0.09</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">S1R</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.60 ± 0.09</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Y54F</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.45 ± 0.09</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">A191E</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.6 ± 0.1</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">A191I</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.2 ± 0.1</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">A191R</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.9 ± 0.2</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pro70275-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="pro70275-sec-0005"><h3 class="pmc_sec_title">2.3. Characterization of the S1D mutant</h3>
<p>S1 is located in the 3CL<sup>pro</sup> dimer interface, where it forms a hydrogen bond with E166 of the other subunit and stabilizes the dimer (Figure <a href="#pro70275-fig-0002" class="usa-link">2b</a>) (Ferreira et al., <a href="#pro70275-bib-0010" class="usa-link" aria-describedby="pro70275-bib-0010">2022</a>). We assumed that the low enzyme activity of the S1D mutant was due to charge repulsion with E166 that weakened the dimer interface and shifted the monomer‐dimer equilibrium toward the non‐active monomer. To test this, we measured the disassociation constant, <em>K</em>
<sub>D</sub>
<sup>app</sup>, of the process D → 2M, for the mutant using a concentration‐dependant enzyme activity assay (Figure <a href="#pro70275-fig-0004" class="usa-link">4</a>), which is based on a previously published FRET‐based enzyme assay (see Section <a href="#pro70275-sec-0014" class="usa-link">5</a> for details) (Wang et al., <a href="#pro70275-bib-0069" class="usa-link" aria-describedby="pro70275-bib-0069">2024</a>). At very low enzyme concentrations, the enzyme was mostly monomeric and inactive. With increasing enzyme concentrations, the active dimer formed, and the activity increased, leading to upward concave curvature in a plot of the catalytic rate <em>v</em> as a function of enzyme concentration [E] (Figure <a href="#pro70275-fig-0004" class="usa-link">4</a>). The data were fit as described previously (Wang et al., <a href="#pro70275-bib-0069" class="usa-link" aria-describedby="pro70275-bib-0069">2024</a>), yielding <em>K</em>
<sub>D</sub>
<sup>app</sup> = 2000 ± 400 nM. This is significantly higher than the reported <em>K</em>
<sub>D</sub>
<sup>app</sup> values of WT SARS‐CoV‐2 3CL<sup>pro</sup> (ranging from 16 to 140 nM) (El‐Baba et al., <a href="#pro70275-bib-0008" class="usa-link" aria-describedby="pro70275-bib-0008">2020</a>) (Wralstad et al., <a href="#pro70275-bib-0071" class="usa-link" aria-describedby="pro70275-bib-0071">2023</a>) (Wang et al., <a href="#pro70275-bib-0069" class="usa-link" aria-describedby="pro70275-bib-0069">2024</a>), demonstrating that the S1D mutation indeed disrupted the dimer interface and resulted in a weakly binding dimer. Furthermore, the weakened dimer interface completely explains the decrease in activity observed for the S1D mutant, since it was only 2% in the active dimeric form under the conditions of the assay (15 nM), compared to the WT, which was 48% active.</p>
<figure class="fig xbox font-sm" id="pro70275-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc3b/12375981/18f20a727e9d/PRO-34-e70275-g002.jpg" loading="lazy" id="jats-graphic-7" height="537" width="669" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/pro70275-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Concentration‐dependent activity assay of S1D showing the upward concave shape of the curve. All experiments were performed in triplicate. The error bars represent standard deviations of the data points.</p></figcaption></figure><p>In addition, the dose–response curve of S1D exhibited pronounced downward concave curvature with a peak at approximately 12.5 nM inhibitor, where the enzyme activity was nearly 250% that of the negative control without inhibitor (Figure <a href="#pro70275-fig-0003" class="usa-link">3b</a>). The enzyme was only fully inhibited when the inhibitor concentration exceeded 1 μM. Such downward concave dose–response curves were previously observed for WT MERS‐CoV 3CL<sup>pro</sup> (Tomar et al., <a href="#pro70275-bib-0064" class="usa-link" aria-describedby="pro70275-bib-0064">2015</a>) and a mutant of SARS‐CoV‐2 3CL<sup>pro</sup> (Nashed et al., <a href="#pro70275-bib-0048" class="usa-link" aria-describedby="pro70275-bib-0048">2022</a>) and attributed to LID. However, to date, analyses on LID have only been qualitative, and the relationships between LID, inhibitor binding, and enzyme catalysis remain poorly understood. We have developed a quantitative model to describe and analyze this phenomenon, as illustrated in Scheme <a href="#pro70275-fig-0011" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="pro70275-fig-0011"><h4 class="obj_head">SCHEME 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc3b/12375981/afa4bfc137ce/PRO-34-e70275-g003.jpg" loading="lazy" id="jats-graphic-9" height="559" width="669" alt="SCHEME 1"></p>
<div class="p text-right font-secondary"><a href="figure/pro70275-fig-0011/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Proposed model for ligand‐induced dimerization. The monomer, M, can dimerize to form the active dimer, D, or bind to the inhibitor, I, to form the complex MI, which is penalized by the factor <em>f</em>
<sub>c</sub>. MI can bind to another monomer to form DI, which is controlled by the dissociation constant <em>K</em>
<sub>D</sub>. D can also bind to the inhibitor to form the singly occupied DI, which is controlled by the inhibition constant <em>K</em>
<sub>i</sub>. DI can also bind to I to form DI<sub>2</sub>, which is also governed by <em>K</em>
<sub>i</sub>. Both D and DI are assumed to be catalytically active, which catalyze the substrate, S, to the product, P.</p></figcaption></figure><p>LID occurs when there is positive allosteric coupling between ligand binding and dimerization. This manifests as weaker dimerization if both subunits are ligand‐free, compared to when one of the subunits is bound to ligand, as well as a weaker equilibrium constant for ligand binding in the monomeric state, compared to the dimeric state, as illustrated in Scheme <a href="#pro70275-fig-0011" class="usa-link">1</a>. In our model, the strength of the coupling is given by <em>f</em>
<sub>c</sub>, such that the dissociation constant for dimerization is <em>K</em>
<sub>D</sub>
<em>f</em>
<sub>c</sub> for D → 2M, <em>K</em>
<sub>D</sub> for DI → M + MI, and <em>K</em>
<sub>D</sub>/<em>f</em>
<sub>c</sub> for DI<sub>2</sub> → 2MI, where D, M, and I are the dimer, monomer, and inhibitor, respectively (Scheme <a href="#pro70275-fig-0011" class="usa-link">1</a>). Similarly, the dissociation constant for ligand binding is <em>K</em>
<sub>i</sub>
<em>f</em>
<sub>c</sub> for MI → M + I and <em>K</em>
<sub>i</sub> for DI → D + I. Note that both <em>K</em>
<sub>D</sub> and <em>K</em>
<sub>i</sub> must be modified by the same factor <em>f</em>
<sub>c</sub> in order for the dimerization/inhibitor‐binding thermodynamic cycle to be internally consistent. Values of <em>f</em>
<sub>c</sub> &gt;1 mean that ligand binding increases the strength of dimerization and vice versa. Analysis of LID dose–response data using Scheme <a href="#pro70275-fig-0011" class="usa-link">1</a> is complicated by the fact that the concentrations of enzyme and inhibitor are of the same order of magnitude as the <em>K</em>
<sub>i</sub> and <em>K</em>
<sub>D</sub> equilibrium constants, and we are not aware of any analytical solution giving the concentration of active enzyme (D and DI) as a function of inhibitor concentration. We therefore developed a numerical approach for solving these equations, which allowed us to calculate dose–response curves exactly in terms of the dissociation constants for dimerization and inhibitor binding, <em>K</em>
<sub>D</sub> and <em>K</em>
<sub>i</sub>, the total concentrations of enzyme and inhibitor, and the strength of allosteric coupling between inhibitor binding and dimerization, <em>f</em>
<sub>c</sub> (see Section <a href="#pro70275-sec-0014" class="usa-link">5</a>).</p>
<p>We applied this approach to the dose–response curves for the S1D mutant against lufotrelvir obtained at 15 and 60 nM enzyme concentrations, fitting both data sets simultaneously (Figure <a href="#pro70275-fig-0005" class="usa-link">5a</a>). The downward concave dose–response curve was much more prominent for the concentration at 15 nM compared to 60 nM, since the enzyme initially populated the monomeric state to a greater extent at the lower concentration. The fitted curves agreed well with the experimental data for the S1D mutant (Figure <a href="#pro70275-fig-0005" class="usa-link">5a</a>), which yielded values of <em>K</em>
<sub>i</sub> = 0.82 ± 0.04 nM, <em>K</em>
<sub>D</sub> = 290 ± 30 nM, and <em>f</em>
<sub>c</sub> = 21 ± 1 (Table <a href="#pro70275-tbl-0002" class="usa-link">2</a>). These results also indicated that the inhibition constant for the dimer (DI → D + I) of the S1D mutant (0.82 ± 0.04 nM) is about three‐fold higher than that of the WT (0.25 ± 0.04 nM). The dissociation constant for the fully ligand‐bound S1D dimer (DI<sub>2</sub> → 2MI, <em>K</em>
<sub>D</sub>/<em>f</em>
<sub>c</sub> = 13 ± 2 nM) was very close to that of the WT enzyme (17 ± 2 nM). However, the strong allosteric coupling meant that the inhibition constant for the monomer (MI → M + I) of the S1D mutant monomer (<em>K</em>
<sub>i</sub>
<em>f</em>
<sub>c</sub> = 17 ± 1 nM) is about 60‐fold higher than that of the WT (0.25 ± 0.04 nM) and that the dissociation constant for the ligand‐free dimer (2M → D) of the S1D mutant (<em>K</em>
<sub>D</sub>
<em>f</em>
<sub>c</sub> = 6000 ± 700 nM) is about 400‐fold higher than that of the ligand‐free WT (17 ± 2 nM). This dimerization constant was in good agreement with the value obtained from enzyme concentration‐dependent activity measurements described above (<em>K</em>
<sub>D</sub>
<sup>app</sup> = 2000 ± 400 nM). Note that fitting the model in Scheme <a href="#pro70275-fig-0011" class="usa-link">1</a> to WT data against lufotrelvir obtained at a 5 nM enzyme concentration gave a value of <em>f</em>
<sub>c</sub> = 1.7 (Figure <a href="#pro70275-fig-0009" class="usa-link">9a</a> and Table <a href="#pro70275-tbl-0003" class="usa-link">3</a>), indicating that the S1D mutation substantially enhances the allosteric coupling (represented by <em>f</em>
<sub>c</sub>) between ligand binding and dimerization.</p>
<figure class="fig xbox font-sm" id="pro70275-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc3b/12375981/6b252a1b836b/PRO-34-e70275-g009.jpg" loading="lazy" id="jats-graphic-11" height="279" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/pro70275-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Fitting of the dose–response curves of the S1D mutant with lufotrelvir. (a) Dose–response curves of lufotrelvir with S1D 3CL<sup>pro</sup> at 15 nM (green) and 60 nM (blue) enzyme concentration. The activity of the enzyme in the absence of inhibitor is set as 100% for each enzyme concentration. (b) Relative dose–response curves of lufotrelvir with S1D (green) 3CL<sup>pro</sup> compared to the WT at 15 nM (black) enzyme concentration. The pink region represents the inhibition concentration where the activity of S1D surpasses that of the WT. The activity of the WT enzyme in the absence of inhibitor is set as 100%, and the activity of S1D in the absence of inhibitor is 5% that of the WT. All measurements are in triplicate. The error bars represent standard deviations of the data points.</p></figcaption></figure><section class="tw xbox font-sm" id="pro70275-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Measured <em>K</em>
<sub>i</sub>, <em>K</em>
<sub>D</sub>, and <em>f</em>
<sub>c</sub> values of lufotrelvir with the S1D mutant and S1D + A191E mutant, by fitting to the ligand‐induced dimerization model, with all values expressed as mean ± SD.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Enzyme</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>i</sub> (nM)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>D</sub> (nM)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>f</em>
<sub>c</sub>
</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>i</sub>
<em>f</em>
<sub>c</sub> (nM)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>D</sub>
<em>f</em>
<sub>c</sub> (nM)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>D</sub>/<em>f</em>
<sub>c</sub> (nM)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">S1D</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.82 ± 0.04</td>
<td align="center" valign="top" rowspan="1" colspan="1">290 ± 30</td>
<td align="center" valign="top" rowspan="1" colspan="1">21 ± 1</td>
<td align="center" valign="top" rowspan="1" colspan="1">17 ± 1</td>
<td align="center" valign="top" rowspan="1" colspan="1">6000 ± 700</td>
<td align="center" valign="top" rowspan="1" colspan="1">13 ± 2</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">S1D + A191E</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.4 ± 0.2</td>
<td align="center" valign="top" rowspan="1" colspan="1">140 ± 10</td>
<td align="center" valign="top" rowspan="1" colspan="1">16 ± 1</td>
<td align="center" valign="top" rowspan="1" colspan="1">105 ± 7</td>
<td align="center" valign="top" rowspan="1" colspan="1">2300 ± 300</td>
<td align="center" valign="top" rowspan="1" colspan="1">9 ± 1</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pro70275-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="pro70275-fig-0009"><h4 class="obj_head">FIGURE 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc3b/12375981/8b319e2719a8/PRO-34-e70275-g011.jpg" loading="lazy" id="jats-graphic-19" height="278" width="709" alt="FIGURE 9"></p>
<div class="p text-right font-secondary"><a href="figure/pro70275-fig-0009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Dose–response curve of lufotrelvir with the WT and G11S mutant. (a) Dose–response curves of lufotrelvir with WT 3CL<sup>pro</sup> at 5 nM enzyme concentration. The solid blue line is from fitting our ligand‐induced dimerization (LID) model. (b) Dose–response curve of lufotrelvir with the G11S mutant at 15 nM enzyme concentration. The solid pink line is from fitting our LID model. The activity of the enzyme in the absence of inhibitor is set at 100%. All measurements are in triplicate. The error bars represent standard deviations of the data points.</p></figcaption></figure><section class="tw xbox font-sm" id="pro70275-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Measured <em>K</em>
<sub>i</sub>, <em>K</em>
<sub>D</sub>, and <em>f</em>
<sub>c</sub> values of lufotrelvir, the WT and G11S 3CL<sup>pro</sup>, with all values expressed as mean ± SD.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Enzyme</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>i</sub> (nM)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>f</em>
<sub>c</sub>
</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>D</sub>
<sup>app</sup> (nM)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">WT</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.3 ± 0.2<a href="#pro70275-note-0004" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.7 ± 0.7<a href="#pro70275-note-0004" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">17 ± 2<a href="#pro70275-note-0005" class="usa-link">
<sup>b</sup>
</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">G11S</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.2 ± 0.1<a href="#pro70275-note-0004" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.8 ± 0.3<a href="#pro70275-note-0004" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">10,000 ± 2000<a href="#pro70275-note-0005" class="usa-link">
<sup>b</sup>
</a>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pro70275-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="pro70275-note-0004">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Ligand‐induced dimerization model.</p>
</div>
<div class="fn" id="pro70275-note-0005">
<sup>
<sup>b</sup>
</sup><p class="display-inline">Previous study (Wang et al., <a href="#pro70275-bib-0069" class="usa-link" aria-describedby="pro70275-bib-0069">2024</a>).</p>
</div>
</div></section><p>An important question is whether the S1D mutation provides any benefit to the enzyme when challenged by lufotrelvir, given that the enhanced LID allostery results in destabilization of the active dimer and a roughly 95% reduction in activity relative to the WT at 15 nM. We compared the activities of the S1D and WT enzymes, with the data of S1D normalized to that of the WT (Figure <a href="#pro70275-fig-0005" class="usa-link">5b</a>). In the absence of inhibitor, the relative activity of the WT was 100%; while that of S1D was only 5%. However, at all inhibitor concentrations greater than about 10 nM, the activity of S1D was much greater than that of the WT, by a factor up to an order of magnitude. Therefore, the S1D mutation, while detrimental to the activity of the enzyme at low inhibitor concentrations, leads to increased enzyme activity in the presence of moderate concentrations of lufotrelvir, in principle conferring resistance to the drug.</p></section><section id="pro70275-sec-0006"><h3 class="pmc_sec_title">2.4. Testing of the S1D/A191E double mutant</h3>
<p>We further investigated the potential resistance mutations S1D and A191E by constructing the double mutant (DM). The DM was also produced with high yield and showed a two‐fold increase in activity compared to the S1D mutant. Dose–response curves of the DM were measured at both 15 and 60 nM enzyme concentrations (Figure <a href="#pro70275-fig-0006" class="usa-link">6a</a>) and were typical of LID with pronounced concave‐down shapes. Fitting the curves using Scheme <a href="#pro70275-fig-0011" class="usa-link">1</a> yielded <em>K</em>
<sub>i</sub> = 6.4 ± 0.2 nM, <em>K</em>
<sub>D</sub> = 140 ± 10 nM, and <em>f</em>
<sub>c</sub> = 16 ± 1 (Table <a href="#pro70275-tbl-0002" class="usa-link">2</a>). Thus, the affinity of lufotrelvir for the DM dimer (DI → D + I, 6.4 ± 0.2 nM) was diminished by 26‐fold compared to the WT (0.25 ± 0.04 nM), which is close to the product (21‐fold) of the 6.4‐fold reduction in affinity for A191E (1.6 ± 0.1 nM) and the 3.3‐fold reduction for S1D (0.82 ± 0.04 nM), indicative of nearly additive changes in binding energy for the two. Again, the dissociation constant of the fully inhibitor‐bound DM dimer (<em>K</em>
<sub>D</sub>/<em>f</em>
<sub>c</sub> ≈9 ± 1 nM) was close to that of the WT. However, the increase in the LID allosteric coupling (<em>f</em>
<sub>c</sub>) meant that the affinity of the inhibitor for the DM monomer (MI → M + I, <em>K</em>
<sub>i</sub>
<em>f</em>
<sub>c</sub> = 105 ± 7 nM) was reduced about 400‐fold compared to the WT, and the dimerization in the absence of inhibitor was reduced about 130‐fold (<em>K</em>
<sub>D</sub>
<em>f</em>
<sub>c</sub> = 2300 ± 300 nM). Comparing WT and DM dose–response curves produced a similar result to that of S1D; although the DM showed lower activity than the WT in the absence of inhibitor, it was much more active in the presence of ≥10 nM lufotrelvir.</p>
<figure class="fig xbox font-sm" id="pro70275-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc3b/12375981/6e121eb45d35/PRO-34-e70275-g007.jpg" loading="lazy" id="jats-graphic-13" height="281" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/pro70275-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Fitting of the dose–response curves of the double mutant (DM) with lufotrelvir. (a) Dose–response curves of lufotrelvir with DM 3CL<sup>pro</sup> at 15 nM (green) and 60 nM (blue) enzyme concentration. The activity of the enzyme in the absence of inhibitor is set at 100% for each enzyme concentration. (b) Relative dose–response curves of lufotrelvir with DM (pink) 3CL<sup>pro</sup> compared to the WT (black) at 15 nM enzyme concentration. The pink region represents the inhibition concentration where the activity of DM surpasses that of the WT. The activity of the WT enzyme in the absence of inhibitor is set at 100%; the activity of DM in the absence of inhibitor is 10% that of the WT. All measurements are in triplicate. The error bars represent standard deviations of the data points.</p></figcaption></figure></section><section id="pro70275-sec-0007"><h3 class="pmc_sec_title">2.5. Native nanoelectrospray ionization mass spectrometry</h3>
<p>We employed nanoelectrospray ionization mass spectrometry (nanoESI‐MS) to validate our observations of perturbed monomer/dimer equilibria and LID in S1D and the DM. As shown in the native mass spectra in Figure <a href="#pro70275-fig-0007" class="usa-link">7a</a>, the WT 3CL<sup>pro</sup> produced peaks corresponding exclusively to the dimer, consistent with tight dimerization. In contrast, both the S1D and the DM produced peaks corresponding to both monomers and dimers (Figure <a href="#pro70275-fig-0007" class="usa-link">7b,c</a>), consistent with weaker dimerization and the presence of both monomers and dimers in solution. The monomeric peaks were slightly weaker for the DM than for S1D, which agrees with the slightly tighter dissociation constant for the DM (<em>K</em>
<sub>D</sub>
<em>f</em>
<sub>c</sub> = 6000 ± 700 nM and 2300 ± 300 nM for S1D and DM, respectively). It is worth noting that, at a protein concentration of 2 μM, the two mutants are expected to exist as approximately 70% monomer (S1D) and 50% monomer (DM), based on Equation <a href="#pro70275-disp-0004" class="usa-link">4</a> and <em>K</em>
<sub>D</sub>
<em>f</em>
<sub>c</sub> values determined by fluorescence assay (Table <a href="#pro70275-tbl-0002" class="usa-link">2</a>). The monomer: dimer peak intensity ratios in the nanoESI‐MS spectra (Figure <a href="#pro70275-fig-0007" class="usa-link">7</a>) are substantially less than those values; however, comparisons between monomer and dimer peak intensities must be made with caution, as the transmission efficiencies of the two forms through the mass spectrometer may be highly unequal. Differences in protein conformation and solvation can also lead to dramatically different efficiencies of ionization and MS peak intensities, even for otherwise very similar proteins (Kuprowski et al., <a href="#pro70275-bib-0037" class="usa-link" aria-describedby="pro70275-bib-0037">2007</a>). Furthermore, there are multiple charged residues that are buried at the dimer interface (including R4, K12, E14, E166, and E290) but exposed in the monomer, which could contribute to these discrepancies (Ferreira et al., <a href="#pro70275-bib-0010" class="usa-link" aria-describedby="pro70275-bib-0010">2022</a>; Kitova et al., <a href="#pro70275-bib-0035" class="usa-link" aria-describedby="pro70275-bib-0035"><sup>2012</sup></a>; Zhang et al., <a href="#pro70275-bib-0079" class="usa-link" aria-describedby="pro70275-bib-0079"><sup>2003</sup></a>). Importantly, the addition of lufotrelvir eliminated the monomeric peaks for S1D and the DM and caused a shift to exclusively dimeric peaks, providing independent corroboration of ligand‐induced dimerization for these proteins.</p>
<figure class="fig xbox font-sm" id="pro70275-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc3b/12375981/51c1eb08cd26/PRO-34-e70275-g008.jpg" loading="lazy" id="jats-graphic-15" height="379" width="709" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/pro70275-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Native mass spectrometry analysis of wild‐type and mutant 3CL<sup>pro</sup> enzymes. 3CL<sup>pro</sup> enzymes were nanoelectrosprayed at 2 μM concentration either alone (top panels) or in the presence of 2 μM of lufotrelvir (bottom panels). The charge states and molecular weights (observed and calculated) are indicated. (a) The WT (purple). (b) S1D mutant (orange). (c) The double mutant (DM) (red). Note that due to the gentle desolvation provided by native ESI‐MS, the protein ions remain partially solvated, resulting in MW<sub>obs</sub> that are significantly larger than MW<sub>calc</sub>. The WT enzyme existed solely as a dimer at this concentration, whereas small amounts of monomer were detected for the S1D and DM enzymes. Upon lufotrelvir binding, all enzymes were detected only in the dimeric form. The insets show a closer view of the indicated ions to illustrate the binding of up to two equivalents of lufotrelvir.</p></figcaption></figure></section><section id="pro70275-sec-0008"><h3 class="pmc_sec_title">2.6. Testing the mutants with nirmatrelvir</h3>
<p>We subsequently measured the dose–response curves of nirmatrelvir, the active ingredient of Paxlovid, with the WT, S1D, A191E, and DM variants of 3CL<sup>pro</sup>, as shown in Figure <a href="#pro70275-fig-0008" class="usa-link">8a</a>. The data for the WT and A191E followed simple sigmoidal shapes and were fit to the Morrison Equation (Equations <a href="#pro70275-disp-0001" class="usa-link">1</a> and <a href="#pro70275-disp-0002" class="usa-link">2</a>). The <em>K</em>
<sub>i</sub> of the A191E mutant (2.3 ± 0.1 nM) was very similar to that of the WT (1.7 ± 0.2 nM). This may be rationalized by the observation that, unlike lufotrelvir, nirmatrelvir is not in close proximity to residue A191 in the x‐ray crystal structure (PDB code: 7SI9). Thus, substitutions of A191 would be expected to have less effect on the affinity of nirmatrelvir. The dose–response curves of the S1D mutant and DM both exhibited strong peaks at intermediate inhibitor concentrations, indicative of LID, similarly to what was seen for lufotrelvir. The analyses of S1D (<em>K</em>
<sub>i</sub> = 1.4 ± 0.1 nM, <em>K</em>
<sub>D</sub> = 220 ± 50 nM, and <em>f</em>
<sub>c</sub> = 37 ± 2) and DM (<em>K</em>
<sub>i</sub> = 6.3 ± 0.4 nM, <em>K</em>
<sub>D</sub> = 100 ± 20 nM, and <em>f</em>
<sub>c</sub> = 20 ± 5) also yielded <em>K</em>
<sub>i</sub> values close to that of WT, as well as <em>K</em>
<sub>D</sub> and <em>f</em>
<sub>c</sub> values similar to those obtained for lufotrelvir (Table <a href="#pro70275-tbl-0004" class="usa-link">4</a>). Thus, the S1D and DM dimers bind nirmatrelvir similarly to the WT, but the LID mechanism is responsible for the 20‐ to 40‐fold reductions in the affinities of the S1D and DM monomers for nirmatrelvir, compared to dimers or the WT. At the same time, the fully nirmatrelvir‐bound S1D and DM dimerize with affinities (<em>K</em>
<sub>D</sub>/<em>f</em>
<sub>c</sub>) of 5–6 nM, which is similar to that of the WT, but the dimerization of the unbound monomers is roughly 900‐fold weaker. As a result, both S1D and DM are far more monomeric and less active than the WT at low concentrations of nirmatrelvir, but above about 20 nM of the drug, both S1D and DM show higher absolute activities, indicating that these mutations confer a degree of resistance to nirmatrelvir via the LID mechanism.</p>
<figure class="fig xbox font-sm" id="pro70275-fig-0008"><h4 class="obj_head">FIGURE 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc3b/12375981/18c09668200e/PRO-34-e70275-g004.jpg" loading="lazy" id="jats-graphic-17" height="288" width="709" alt="FIGURE 8"></p>
<div class="p text-right font-secondary"><a href="figure/pro70275-fig-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Dose–response curves of nirmatrelvir with the WT, A191E, S1D and double mutant (DM). (a) Dose–response curves of nirmatrelvir with WT (black), A191E (pale blue), S1D (green), DM (pink) 3CL<sup>pro</sup> at 15 nM enzyme concentration. The activity of the enzyme in the absence of inhibitor is set as 100% for each individual enzyme. (b) Relative dose‐response curves of nirmatrelvir with S1D (green) and DM (pink) 3CL<sup>pro</sup> compared to the WT (black) at 15 nM enzyme concentration. The pink region represents the inhibition concentration where the activity of S1D surpasses that of the WT. The blue region represents the inhibition concentration where the activity of DM surpasses that of the WT. The activity of the WT enzyme in the absence of inhibitor is set as 100%; the activity of S1D and DM in the absence of inhibitor is 5% and 10% that of the WT, respectively. All measurements are in triplicate. The error bars represent standard deviations of the data points.</p></figcaption></figure><section class="tw xbox font-sm" id="pro70275-tbl-0004"><h4 class="obj_head">TABLE 4.</h4>
<div class="caption p"><p>Measured <em>K</em>
<sub>i</sub>, <em>K</em>
<sub>D</sub>, and <em>f</em>
<sub>c</sub> values of nirmatrelvir for the WT, S1D, A191E, and DM 3CL<sup>pro</sup>, with all values expressed as mean ± SD.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Enzyme</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>i</sub> (nM)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>D</sub> (nM)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>f</em>
<sub>c</sub>
</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>i</sub>
<em>f</em>
<sub>c</sub> (nM)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>D</sub>
<em>f</em>
<sub>c</sub> (nM)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>D</sub>/<em>f</em>
<sub>c</sub> (nM)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">WT</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.7 ± 0.2<a href="#pro70275-note-0002" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">‐</td>
<td align="left" valign="top" rowspan="1" colspan="1">‐</td>
<td align="left" valign="top" rowspan="1" colspan="1">‐</td>
<td align="left" valign="top" rowspan="1" colspan="1">‐</td>
<td align="left" valign="top" rowspan="1" colspan="1">‐</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">S1D</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.4 ± 0.1<a href="#pro70275-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">220 ± 50<a href="#pro70275-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">37 ± 2<a href="#pro70275-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">53 ± 3<a href="#pro70275-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">8000 ± 2000<a href="#pro70275-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">6 ± 1<a href="#pro70275-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">A191E</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.3 ± 0.1<a href="#pro70275-note-0002" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">‐</td>
<td align="left" valign="top" rowspan="1" colspan="1">‐</td>
<td align="left" valign="top" rowspan="1" colspan="1">‐</td>
<td align="left" valign="top" rowspan="1" colspan="1">‐</td>
<td align="left" valign="top" rowspan="1" colspan="1">‐</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">S1D + A191E</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.3 ± 0.4<a href="#pro70275-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">100 ± 20<a href="#pro70275-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">20 ± 5<a href="#pro70275-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">130 ± 40<a href="#pro70275-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">2000 ± 700<a href="#pro70275-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">5 ± 2<a href="#pro70275-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pro70275-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="pro70275-note-0002">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Morrison equation.</p>
</div>
<div class="fn" id="pro70275-note-0003">
<sup>
<sup>b</sup>
</sup><p class="display-inline">Ligand‐induced dimerization model.</p>
</div>
</div></section></section><section id="pro70275-sec-0009"><h3 class="pmc_sec_title">2.7. Effects of mutations on enzyme kinetics</h3>
<p>We characterized the catalytic parameters of the WT, S1D, A191E, and DM 3CL<sup>pro</sup> variants, measuring activity as a function of substrate concentration (Figure <a href="#pro70275-supitem-0001" class="usa-link">S5</a>). Data for both the WT and A191E had a hyperbolic shape and were well fit by the Michaelis–Menten equation. The <em>K</em>
<sub>m</sub> of A191E (224 ± 29 μM) was slightly lower than that of the WT (360 ± 30 μM), while the catalytic constant, <em>k</em>
<sub>cat</sub>, was slightly higher (4.0 ± 0.3 vs. 2.1 ± 0.1 s<sup>−s</sup>), leading to a roughly three‐fold increase in enzyme efficiency (<em>k</em>
<sub>cat</sub>/<em>K</em>
<sub>m</sub> = 1.8 ± 0.3 × 10<sup>4</sup> M<sup>−1</sup> s<sup>−1</sup> and 5.8 ± 0.6 × 10<sup>3</sup> M<sup>−1</sup> s<sup>−1</sup> for A191E and WT, respectively). On the other hand, both the S1D mutant and the DM could not be saturated even at the highest substrate concentration of 400 μM, implying a large increase in the apparent <em>K</em>
<sub>m</sub>. Interestingly, substrate‐induced dimerization, that is, LID, with a value of <em>f</em>
<sub>c</sub> on the order of what we observed experimentally for the inhibitors would produce a very similar increase in the apparent <em>K</em>
<sub>m</sub> as we observed here. The idea that the substrate induces dimerization of the mutants is supported by the observation that at close to 400 μM substrate, the S1D and DM variants are 50% as active as the WT, while the apparent dissociation constant of S1D, measured at 12 μM substrate, would predict that only less than 2% of the enzyme should be in the active, dimeric form. Nevertheless, we cannot rule out the possibility that these mutations also change the intrinsic substrate affinity of the dimeric forms. In either case, the LID experiments in this study were performed with substrate concentrations of only about 5% of the WT <em>K</em>
<sub>m</sub>, so that any substrate‐induced dimerization can be neglected in the analysis of inhibitor‐induced dimerization.</p></section><section id="pro70275-sec-0010"><h3 class="pmc_sec_title">2.8. 
G11S mutant with disrupted dimer interface does not exhibit LID
</h3>
<p>In order to better understand the relationship between disruption of the dimer interface and LID, we studied SARS‐CoV‐2 3CL<sup>pro</sup> G11S, a naturally occurring mutant with a highly destabilized dimer interface (<em>K</em>
<sub>D</sub>
<sup>app</sup> of 10,000 ± 2000 nM) (Wang et al., <a href="#pro70275-bib-0069" class="usa-link" aria-describedby="pro70275-bib-0069">2024</a>). G11S was largely monomeric (and inactive) at the enzyme concentrations used in this study, and any LID due to lufotrelvir would be expected to produce a large peak at an intermediate inhibitor concentration. However, dose–response curve for G11S did not exhibit LID, as shown in Figure <a href="#pro70275-fig-0009" class="usa-link">9b</a>, and fitting to Scheme <a href="#pro70275-fig-0011" class="usa-link">1</a> yielded a low value for <em>f</em>
<sub>c</sub> = 1.8, similar to that of the WT and an order of magnitude less than we measured for S1D and DM. Thus, destabilization of dimerization and enhancement of LID are separate effects in 3CL<sup>pro</sup>, such that a single mutation can disrupt the interface without greatly affecting coupling between the interface and the active site.</p></section><section id="pro70275-sec-0011"><h3 class="pmc_sec_title">2.9. Implications of LID on drug resistance</h3>
<p>Given that LID was demonstrated to be an enzyme activation mechanism (Tomar et al., <a href="#pro70275-bib-0064" class="usa-link" aria-describedby="pro70275-bib-0064">2015</a>), it could act as a mechanism of drug resistance. Our model of the process has allowed us to investigate quantitatively how modulating the strengths of dimerization, inhibitor binding, and allosteric coupling can enhance enzyme activity in the presence of inhibitors. Figure <a href="#pro70275-fig-0010" class="usa-link">10a</a> shows a series of curves simulated for a range of coupling strengths (<em>f</em>
<sub>c</sub>), while the inhibitor affinity of the dimer (<em>K</em>
<sub>i</sub>) and the strength of the M:MI interface (<em>K</em>
<sub>D</sub>) were held fixed. The simulated enzyme concentration was lower than the <em>K</em>
<sub>D</sub>, meaning that, in the absence of inhibitor, the enzyme was largely (≈85%) monomeric. With <em>f</em>
<sub>c</sub> = 1, there was no allosteric coupling between inhibitor binding and dimerization, and a standard sigmoidal dose–response curve was obtained. For increasing coupling strengths (increasing <em>f</em>
<sub>c</sub>), the simulated activity of the enzyme was decreasing, due to the increasing dissociation constant of the M:M interface (<em>K</em>
<sub>D</sub>
<em>f</em>
<sub>c</sub>). At the same time, the curves increasingly showed the peaked profiles associated with LID, along with higher activity at high inhibitor concentrations, that is, resistance. Figure <a href="#pro70275-fig-0010" class="usa-link">10b</a> shows a similar scenario, except in this case the <em>K</em>
<sub>D</sub> of the enzyme was 100‐fold tighter, meaning that the enzyme was initially only about 20% monomeric. In this case, increasing the value of <em>f</em>
<sub>c</sub> did not produce the peaked curves characteristic of LID, except at the highest values of <em>f</em>
<sub>c</sub>, and the activity of the enzyme was not enhanced at high inhibitor concentrations. In Figure <a href="#pro70275-fig-0010" class="usa-link">10c</a>, the value of <em>f</em>
<sub>c</sub> was held fixed, and the value of <em>K</em>
<sub>D</sub> was increased. The dose–response curves became more peaked and LID‐like at larger values of <em>K</em>
<sub>D</sub>, where the enzyme was more monomeric. However, in this case, increasing the value of <em>K</em>
<sub>D</sub> did not enhance the enzyme's activity at any inhibitor concentration. Finally, in Figure <a href="#pro70275-fig-0010" class="usa-link">10d</a>, the values of <em>f</em>
<sub>c</sub> and <em>
<span class="font-variant-small-caps">k</span>
</em>
<sub>
<span class="font-variant-small-caps">d</span>
</sub> were held fixed, and <em>K</em>
<sub>i</sub> was varied. Increasing the value of <em>K</em>
<sub>i</sub> did not change the shape of the dose–response curve but shifted it to higher inhibitor concentrations, as expected. In summary, increasing the LID coupling strength (<em>f</em>
<sub>c</sub>) alone is sufficient to produce inhibitor resistance. However, this is only true under conditions where the enzyme must dimerize to become active but is initially largely monomeric. No enhancement is seen when the enzyme is already mostly dimeric in the absence of inhibitor. Furthermore, simply destabilizing the dimer interface (increasing <em>K</em>
<sub>D</sub>) does not produce any inhibitor resistance. In contrast, reducing the inhibitor binding strength (increasing <em>K</em>
<sub>i</sub>) also produces resistance in a mechanism that is wholly distinct from altering the coupling strength.</p>
<figure class="fig xbox font-sm" id="pro70275-fig-0010"><h4 class="obj_head">FIGURE 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc3b/12375981/ae4f3249a6af/PRO-34-e70275-g010.jpg" loading="lazy" id="jats-graphic-21" height="579" width="709" alt="FIGURE 10"></p>
<div class="p text-right font-secondary"><a href="figure/pro70275-fig-0010/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Simulations to show the effects of ligand‐induced dimerization on the dose–response curves under various conditions. All simulations were performed with [<em>E</em>]<sub>t</sub> = 10 nM. The activity of the fully dimerized enzyme is set to 100%. (a) <em>K</em>
<sub>D</sub> = 100 nM, <em>K</em>
<sub>i</sub> = 50 nM, <em>f</em>
<sub>c</sub> starts from 1 (deep purple) and increases two‐fold till it reaches 128 (light purple). (b) <em>K</em>
<sub>D</sub> = 1 nM, <em>K</em>
<sub>i</sub> = 50 nM, <em>f</em>
<sub>c</sub> starts from 1 (deep purple) and increases two‐fold till it reaches 128 (light purple). (c) <em>K</em>
<sub>i</sub> = 50 nM, <em>f</em>
<sub>c</sub> = 32, <em>K</em>
<sub>D</sub> starts from 1 nM (deep blue) and increases two‐fold till it reaches 256 nM (light blue). (d) <em>K</em>
<sub>D</sub> = 100 nM, <em>f</em>
<sub>c</sub> = 32, <em>K</em>
<sub>i</sub> starts from 12.5 nM (deep red) and increases two‐fold till it reaches 800 nM (light red).</p></figcaption></figure></section></section><section id="pro70275-sec-0012"><h2 class="pmc_sec_title">3. DISCUSSION</h2>
<p>Enzyme activation via substrate‐ and inhibitor‐induced dimerization is a relatively common phenomenon (Atkinson et al., <a href="#pro70275-bib-0002" class="usa-link" aria-describedby="pro70275-bib-0002">2018</a>; Cheng et al., <a href="#pro70275-bib-0006" class="usa-link" aria-describedby="pro70275-bib-0006"><sup>2010</sup></a>; Datta et al., <a href="#pro70275-bib-0007" class="usa-link" aria-describedby="pro70275-bib-0007"><sup>2013</sup></a>; Ivanov et al., <a href="#pro70275-bib-0029" class="usa-link" aria-describedby="pro70275-bib-0029"><sup>2012</sup></a>; Louis et al., <a href="#pro70275-bib-0040" class="usa-link" aria-describedby="pro70275-bib-0040"><sup>2011</sup></a>; Patel et al., <a href="#pro70275-bib-0052" class="usa-link" aria-describedby="pro70275-bib-0052"><sup>2016</sup></a>; Tomar et al., <a href="#pro70275-bib-0064" class="usa-link" aria-describedby="pro70275-bib-0064"><sup>2015</sup></a>; Xie et al., <a href="#pro70275-bib-0073" class="usa-link" aria-describedby="pro70275-bib-0073"><sup>1999</sup></a>) that has been linked to functional regulation, including sensing enzyme concentration (Datta et al., <a href="#pro70275-bib-0007" class="usa-link" aria-describedby="pro70275-bib-0007">2013</a>). LID has been reported for 3CL<sup>pro</sup> from MERS (Tomar et al., <a href="#pro70275-bib-0064" class="usa-link" aria-describedby="pro70275-bib-0064">2015</a>) and SARS‐CoV‐2 (Paciaroni et al., <a href="#pro70275-bib-0051" class="usa-link" aria-describedby="pro70275-bib-0051">2023</a>) and 3CL<sup>pro</sup> mutants from SARS‐CoV (Cheng et al., <a href="#pro70275-bib-0006" class="usa-link" aria-describedby="pro70275-bib-0006">2010</a>). In the case of SARS‐CoV‐2 3CL<sup>pro</sup>, saturation with nirmatrelvir increased the strength of dimerization about seven‐fold (Paciaroni et al., <a href="#pro70275-bib-0051" class="usa-link" aria-describedby="pro70275-bib-0051">2023</a>). This ratio corresponds to <em>f</em>
<sub>c</sub>
<sup>2</sup> and is similar to the value of <em>f</em>
<sub>c</sub>
<sup>2</sup> we measured here for lufotrelvir and the WT enzyme (2.9‐ to 3.2‐fold). The <em>f</em>
<sub>c</sub>
<sup>2</sup> values for S1D and DM are more than 400‐fold greater. Thus, allosteric pathways linking the active site to the dimer interface likely have evolved in many enzymes, including the 3CL<sup>pro</sup> from several coronaviruses. Our findings suggest that simply shifting the energetic balance of this allosteric coupling can be enough to generate inhibitor‐resistant enzyme variants. It is worth noting that several inhibitor‐resistant variants of human immunodeficiency virus (HIV) protease exhibit pronounced LID (Louis et al., <a href="#pro70275-bib-0040" class="usa-link" aria-describedby="pro70275-bib-0040">2011</a>; Xie et al., <a href="#pro70275-bib-0073" class="usa-link" aria-describedby="pro70275-bib-0073"><sup>1999</sup></a>), raising the possibility that resistance has arisen at least partly via this mechanism.</p>
<p>In Scheme <a href="#pro70275-fig-0011" class="usa-link">1</a>, this shift is described by an increase in <em>f</em>
<sub>c</sub>; however, the model is deliberately general and does not speculate as to the physical nature of an increase in coupling strength. Two of the most studied allosteric paradigms used to explain coupling are those due to Monod–Wyman–Changeux (MWC) (Monod et al., <a href="#pro70275-bib-0045" class="usa-link" aria-describedby="pro70275-bib-0045">1965</a>) and Koshland–Nemethy–Filmer (KNF) (Koshland Jr. et al., <a href="#pro70275-bib-0036" class="usa-link" aria-describedby="pro70275-bib-0036">1966</a>). In the MWC model, conformational changes are synchronized such that monomers are either in an active form, capable of both dimerization and ligand binding, or in an inactive one, capable of neither. In this mechanism, <em>f</em>
<sub>c</sub> is the equilibrium constant between free, monomeric, inactive, and active forms, <em>f</em>
<sub>c</sub> = [inactive]/[active] (Figure <a href="#pro70275-supitem-0001" class="usa-link">S7</a>). Any mutation that shifts the conformational equilibrium toward the inactive form could potentially generate inhibitor resistance. In the KNF model, both free and bound monomers can dimerize, but there is an energetic penalty of RTln(<em>f</em>
<sub>c</sub>) for a monomer that is dimerized but not bound, and vice versa (Figure <a href="#pro70275-supitem-0001" class="usa-link">S8</a>). Likewise, any mutation that enhances this penalty could generate inhibitor resistance.</p>
<p>Structural studies of 3CL<sup>pro</sup> variants offer some clues into the nature of allosteric coupling in this enzyme. Several monomeric 3CL<sup>pro</sup> mutants have an active site that is collapsed due to the formation of a short 3<sub>10</sub>‐helix in the S139‐F140‐L141 loop, which may render the monomer incapable of binding substrate or inhibitor (Chen, Hu, et al., <a href="#pro70275-bib-0004" class="usa-link" aria-describedby="pro70275-bib-0004">2008</a>; Li et al., <a href="#pro70275-bib-0039" class="usa-link" aria-describedby="pro70275-bib-0039"><sup>2016</sup></a>; Shi et al., <a href="#pro70275-bib-0060" class="usa-link" aria-describedby="pro70275-bib-0060"><sup>2008</sup></a>). One of these enzymes is G11A 3CL<sup>pro</sup> from SARS‐CoV, which is exclusively monomeric and catalytically inactive (Chen, Hu, et al., <a href="#pro70275-bib-0004" class="usa-link" aria-describedby="pro70275-bib-0004">2008</a>). However, SARS‐CoV‐2 G11S 3CL<sup>pro</sup> does not exhibit LID, even though it is also predominantly monomeric (Wang et al., <a href="#pro70275-bib-0069" class="usa-link" aria-describedby="pro70275-bib-0069">2024</a>) and might be expected to show the same active site collapse as G11A. This suggests that allosteric coupling between the active site and dimer interface may be more complicated than simply forming this 3<sub>10</sub>‐helix in the monomeric state. In addition, several studies of R298A SARS‐CoV 3CL<sup>pro</sup> showed that, in the absence of bound ligand, domain III undergoes a 33° rotation that creates a steric clash between protomers and prohibits dimer formation (Shi et al., <a href="#pro70275-bib-0060" class="usa-link" aria-describedby="pro70275-bib-0060">2008</a>; Wu et al., <a href="#pro70275-bib-0072" class="usa-link" aria-describedby="pro70275-bib-0072"><sup>2013</sup></a>). In the presence of ligand, domain III readopts to the correct orientation and the enzyme dimerizes. We hypothesize that ligand‐induced reorientation of domain III may be the mechanism that links the active site and dimer interface and produces LID in SARS‐CoV‐2 3CL<sup>pro</sup> WT, S1D, and DM. Interestingly, G11A SARS‐CoV 3CL<sup>pro</sup> does not exhibit this rotation of domain III in the crystal structure, which might explain why the SARS‐CoV‐2 G11S 3CL<sup>pro</sup> does not exhibit LID (Chen, Hu, et al., <a href="#pro70275-bib-0004" class="usa-link" aria-describedby="pro70275-bib-0004">2008</a>).</p>
<p>In SARS‐CoV 3CL<sup>pro</sup>, E166 acts as a link between the substrate binding site and the dimer interface, and the E166A mutation eliminates the substrate‐induced dimerization of the R298A mutant (Cheng et al., <a href="#pro70275-bib-0006" class="usa-link" aria-describedby="pro70275-bib-0006">2010</a>). In MERS‐CoV 3CL<sup>pro</sup>, the homologous E169A mutation shifted the equilibrium toward the monomer, even in the presence of substrate (Ho et al., <a href="#pro70275-bib-0021" class="usa-link" aria-describedby="pro70275-bib-0021">2015</a>). In SARS‐CoV‐2 3CL<sup>pro</sup>, it was proposed that nirmatrelvir stabilized the dimer via a hydrogen bond network involving the inhibitor and E166 and F140 of one monomer and S1 of the other monomer (Paciaroni et al., <a href="#pro70275-bib-0051" class="usa-link" aria-describedby="pro70275-bib-0051">2023</a>). Similar interactions are observed between lufotrelvir and 3CL<sup>pro</sup> (Figure <a href="#pro70275-fig-0002" class="usa-link">2b</a>). This hydrogen‐binding network that includes S1 may help to explain why the S1D mutation has such a profound effect on the strength of LID in SARS‐CoV‐2 3CL<sup>pro</sup>.</p>
<p>Most resistance mutations on 3CL<sup>pro</sup> directly affect the binding of the inhibitor by changing <em>K</em>
<sub>i</sub>. These mutations occur near the binding pocket, including mutations on S144, M165, E166, H172, F170, Q192, among many others (FDA, <a href="#pro70275-bib-0009" class="usa-link" aria-describedby="pro70275-bib-0009"><sup>2023</sup></a>; Hu et al., <a href="#pro70275-bib-0024" class="usa-link" aria-describedby="pro70275-bib-0024"><sup>2023</sup></a>; Iketani et al., <a href="#pro70275-bib-0026" class="usa-link" aria-describedby="pro70275-bib-0026"><sup>2023</sup></a>; Ip et al., <a href="#pro70275-bib-0027" class="usa-link" aria-describedby="pro70275-bib-0027"><sup>2023</sup></a>). Most notably, the E166V mutation not only reduces binding affinity to nirmatrelvir but also increases the kinetic barrier required for forming the covalent complex, resulting in more than 7000‐fold increase in <em>K</em>
<sub>i</sub> and up to 255‐fold increase in EC<sub>50</sub> (FDA, <a href="#pro70275-bib-0009" class="usa-link" aria-describedby="pro70275-bib-0009"><sup>2023</sup></a>; Ramos‐Guzmán et al., <a href="#pro70275-bib-0056" class="usa-link" aria-describedby="pro70275-bib-0056"><sup>2023</sup></a>). Other resistance mutations such as T21I, L50F, P252L, and T304I are distal to the binding pocket. These mutations themselves do not confer high level of resistance alone. Instead, they are often part of a combination of mutations that lead to significant resistance. LID‐promoting mutations could potentially work in tandem with these and other mutations to produce substantial antiviral resistance. That said, the S1D and A191E mutations we identified in this study are not common in the global initiative on sharing all influenza data (GISAID) database of SARS‐CoV‐2 patient‐derived sequences (Khare et al., <a href="#pro70275-bib-0034" class="usa-link" aria-describedby="pro70275-bib-0034">2021</a>). The most common substitution at S1 is G, making up to 65% of all mutations at this position, while no record of S1D mutation has been reported to date. Since the S1D mutation requires two nucleotide substitutions, a possible pathway to achieve this mutation would be through the S1G mutation to G1D mutation. It should also be noted that even though S1D mutation is detrimental to the enzyme activity, it does not necessarily render the virus incapable of replication, as it has been shown that the virus carrying the G11S mutant is viable and detected in patient sequencing samples (Wang et al., <a href="#pro70275-bib-0069" class="usa-link" aria-describedby="pro70275-bib-0069">2024</a>), and that the WT MERS‐CoV 3CL<sup>pro</sup> is a weakly binding dimer (<em>K</em>
<sub>D</sub> = 7.8 μM) that mainly exists in the inactive monomer form (Tomar et al., <a href="#pro70275-bib-0064" class="usa-link" aria-describedby="pro70275-bib-0064">2015</a>). As for substitutions of A191, the most common mutation is A191V, making up to 90% of the mutations at this position. A191E, on the other hand, accounts for only 0.06% of mutations at this site.</p></section><section id="pro70275-sec-0013"><h2 class="pmc_sec_title">4. CONCLUSION</h2>
<p>Using a combination of MD simulations and enzymatic assays, we have identified two mutations in SARS‐CoV‐2 3CL<sup>pro</sup>, S1D and A191E, that decrease the binding affinity of lufotrelvir, one of the most potent 3CL<sup>pro</sup> inhibitors to date, and nirmatrelvir, the active component of Paxlovid. Combining these in an S1D/A191E DM led to roughly additive inhibitor resistance. Dose–response curves for S1D and DM showed peaks in activity at intermediate inhibitor concentrations, which are indicative of ligand‐induced dimerization. We developed a general, numerical model of LID that allowed us to quantify the effects of the mutations on this process and clarify how it relates to inhibitor resistance. We found that the S1D mutation increases the strength of coupling between binding in the active site and dimerization by roughly 12‐fold, compared to the WT, in addition to slightly increasing the value of <em>K</em>
<sub>i</sub>. In parallel, we also directly observed ligand‐induced dimerization in nanoESI‐MS, which demonstrated good agreement with our LID model predictions. In general, increasing the strength of LID leads to reduced inhibitor sensitivity at high concentrations, under conditions where the enzyme is initially mostly monomeric. Since LID is a relatively common enzyme regulatory mechanism, we postulate that LID‐enhancing mutations may be an important source of inhibitor resistance that is orthogonal to mutations that affect inhibitor binding directly.</p></section><section id="pro70275-sec-0014"><h2 class="pmc_sec_title">5. MATERIALS AND METHODS</h2>
<section id="pro70275-sec-0015"><h3 class="pmc_sec_title">5.1. In silico preparation of the enzyme ligand complexes</h3>
<p>The 3CL<sup>pro</sup> systems in complex with the inhibitor <em>PF‐00835231</em> (V2M) were prepared from the crystal structures 6XHM at 1.41 Å resolution. The 3CL<sup>pro</sup>/substrate complex was prepared using the 11‐residue substrate (<em>TSAVLQSGFRK</em>) coordinates obtained from the SARS‐CoV 3CL<sup>pro</sup>/substrate complex (2Q6G) and the SARS‐CoV2 3CL<sup>pro</sup> apoprotein coordinates (6YB7) (Xue et al., <a href="#pro70275-bib-0075" class="usa-link" aria-describedby="pro70275-bib-0075">2008</a>). Point mutations were generated with the tleap program, modifying the atom type in each case to match the parameter file. The protonation states of the ionizable residues at pH 7.0 were established with the H++ web interface based on calculations of the p<em>K</em> values of the ionizable groups (Gordon et al., <a href="#pro70275-bib-0014" class="usa-link" aria-describedby="pro70275-bib-0014">2005</a>). The polypeptide chains are parameterized with Amber's ff19SB force field. Inhibitors were parameterized with the generalized amber force field and the restrained electrostatic potential (REsP) charges were recalculated by quantum mechanics in the Gaussian16 program using a B3LYP functional and a 6‐31g* basis (Frisch et al., <a href="#pro70275-bib-0013" class="usa-link" aria-describedby="pro70275-bib-0013">2016</a>; Salomon‐Ferrer et al., <a href="#pro70275-bib-0057" class="usa-link" aria-describedby="pro70275-bib-0057"><sup>2013</sup></a>). Finally, the systems were hydrated in a TIP3P water and ion model box to reach charge neutrality.</p></section><section id="pro70275-sec-0016"><h3 class="pmc_sec_title">5.2. Molecular dynamics simulation</h3>
<p>MD simulations were performed with AMBER20‐GPU using the following MD protocol (Osorio et al., <a href="#pro70275-bib-0050" class="usa-link" aria-describedby="pro70275-bib-0050">2022</a>; Yañez et al., <a href="#pro70275-bib-0076" class="usa-link" aria-describedby="pro70275-bib-0076"><sup>2021</sup></a>): (i) minimization and structural relaxation of water molecules with 2000 minimization steps and MD simulation with an isothermal‐isobaric (NPT) assembly (300 K) for 1000 ps using harmonic constraints of 10 kcal molÅ<sup>−2</sup> for protein and ligand; (ii) full structure minimization considering 6500 steps of conjugate gradient minimization; (iii) the minimized systems were progressively heated up to 300 K, with harmonic constraints of 10 kcal molÅ<sup>−2</sup> for carbonate skeleton and ligand for 0.5 ns; (iv) the system was then equilibrated for 0.5 ns maintaining the constraints and then for 5 ns without constraints at 300 K in a canonical assembly (NVT); (v) finally, 200 ns simulations were performed for each system (Purawat et al., <a href="#pro70275-bib-0054" class="usa-link" aria-describedby="pro70275-bib-0054">2017</a>; Salomon‐Ferrer et al., <a href="#pro70275-bib-0057" class="usa-link" aria-describedby="pro70275-bib-0057"><sup>2013</sup></a>). During the MD simulations, the equations of motion were integrated with a time step of 2 fs in the NPT assembly at a pressure of 1 atm. The SHAKE algorithm was applied to all hydrogen atoms, and the van der Waals limit was set to 12 Å (Forester &amp; Smith, <a href="#pro70275-bib-0011" class="usa-link" aria-describedby="pro70275-bib-0011">1998</a>). The temperature was maintained at 300 K, employing the Langevin thermostat method with a relaxation time of 1 ps. The Berendsen barostat was used to control the pressure at 1 atm. Long‐range electrostatic forces were accounted for using the particle‐mesh Ewald (PME) approach (Harvey &amp; De Fabritiis, <a href="#pro70275-bib-0018" class="usa-link" aria-describedby="pro70275-bib-0018">2009</a>). Data were collected every 1 ps during MD tests. Molecular visualization of the systems and MD trajectory analysis were carried out with the visual molecular dynamics (VMD) software package (Humphrey et al., <a href="#pro70275-bib-0025" class="usa-link" aria-describedby="pro70275-bib-0025">1996</a>). To assess the reproducibility of the results, three replicates were performed using ig = 40, 100, and 1000, which defines the seed for random number generation and controls the initial assignment of velocities, as well as the thermostat and barostat settings in the simulation. Results are shown in Figures <a href="#pro70275-supitem-0001" class="usa-link">S1–S3</a>.</p>
<section id="pro70275-sec-0017"><h4 class="pmc_sec_title">5.2.1. 
Cluster analysis
</h4>
<p>This statistical methodology separates the data points into several groups that exhibit similar properties and differ from the other groups. To perform the clustering, the density‐based spatial method of applications with noise (DBSCAN) implemented in the CPPTRAJ tool was used (Schubert et al., <a href="#pro70275-bib-0059" class="usa-link" aria-describedby="pro70275-bib-0059">2017</a>). This algorithm performs the separation by considering a cluster in the data space as a contiguous region of high point density, separated from other similar clusters by contiguous regions of low point density. Each analysis was performed with a cutoff distance of 1.5 Å based on the root‐mean‐square (rmsd) of the ligand's distinct hydrogen atoms and 5 points as a minimum for each cluster. According to this calculation, the representative structures of each simulation were obtained, in relation to the position of the ligand in the protein.</p></section></section><section id="pro70275-sec-0018"><h3 class="pmc_sec_title">5.3. Enzyme production</h3>
<p>Plasmids carrying WT and SARS‐CoV‐2 3CL<sup>pro</sup> mutants were purchased from GenScript (NJ, USA). These proteases were produced in <em>E. coli</em> BL21 (DE3) according to previously published procedures (Stille et al., <a href="#pro70275-bib-0062" class="usa-link" aria-describedby="pro70275-bib-0062">2022</a>; Zhang et al., <a href="#pro70275-bib-0077" class="usa-link" aria-describedby="pro70275-bib-0077"><sup>2020</sup></a>). The protein sequence was preceded by an N‐terminal glutathione <em>S</em>‐transferase  sequence followed by a 3CL<sup>pro</sup> cleavage site (SAVLQ↓SGFRK). Autocleavage activity of 3CL<sup>pro</sup> leaves its authentic N‐terminus starting with serine. The C‐terminal glutamine was followed by a GPHHHHHH 6‐His purification tag (Xue et al., <a href="#pro70275-bib-0074" class="usa-link" aria-describedby="pro70275-bib-0074">2007</a>; Zhang et al., <a href="#pro70275-bib-0077" class="usa-link" aria-describedby="pro70275-bib-0077"><sup>2020</sup></a>). Proteins were purified using a 5 mL HisTrap FF column (GE Healthcare, IL, USA) on an ÄKTA Avant system (Cytiva), applying a gradient between buffer A (50 mM Tris–HCl pH 8.0, 150 mM NaCl, 20 mM imidazole) and buffer B (50 mM Tris–HCl pH 8.0, 150 mM NaCl, 500 mM imidazole). Eluted fractions were pooled, buffer exchanged (50 mM Tris–HCl pH 8.0, 150 mM NaCl), and concentrated using Amicon® Ultra centrifugal filters with a 10 kDa cutoff (Merck Millipore, MA, USA) to final concentrations of 1–5 mg mL<sup>−1</sup>. Proteins were stored at −80°C until use.</p></section><section id="pro70275-sec-0019"><h3 class="pmc_sec_title">5.4. Assay setup</h3>
<p>The enzymatic activity of 3CL<sup>pro</sup> was measured using the fluorescent substrate DABCYL‐KTSAVLQ↓SGFRKME‐EDANS from CanPeptide (Montréal, Québec, Canada) (Mesel‐Lemoine et al., <a href="#pro70275-bib-0042" class="usa-link" aria-describedby="pro70275-bib-0042">2012</a>). The rate of enzymatic activity was determined at 25°C by following the increase in fluorescence (excitation at 360 nm and emission at 460 nm) due to peptide hydrolysis as a function of time. Assays were conducted in Nunc™ F96 MicroWell™ black plates (Thermo Scientific, Waltham, MA, USA) in assay buffer (20 mM 4‐(2‐hydroxyethyl)‐1‐piperazineethanesulfonic acid, 10 mM NaCl, 1 mM ethylenediaminetetraacectic acid, 1 mM dithiothreitol, and 0.1% bovine serum albumin, pH 7.5) with a final reaction volume of 50 μL. The resulting fluorescence was monitored using a BioTek Synergy™ H4 Hybrid Multi‐Mode Microplate Reader (Winooski, VT, USA). The rate of the reaction in relative fluorescent units per second (RFU s<sup>−1</sup>) was determined by measuring the initial slope of the progress curves. All reactions were carried out in triplicate.</p></section><section id="pro70275-sec-0020"><h3 class="pmc_sec_title">5.5. 
<em>K</em>
<sub>i</sub> measurement</h3>
<p>Reactions were performed in 100 μL volume. The final concentration of the enzyme was 15 nM, and the final concentration of the substrate was 12 μM. Fifty‐six microliters of enzyme (diluted in the assay buffer) were incubated for 30 min at room temperature with 4 μL of compound (<em>PF‐00835231</em> or <em>PF‐07321332</em>, Selleck Chemicals, TX, USA) diluted in dimethyl sulfoxide (DMSO) (Millipore, MA, USA). To initiate the reaction, 40 μL of substrate was added, and the reaction was monitored by following the fluorescence as a function of time. Controls were (i) negative control: no inhibitor dissolved in DMSO, (ii) positive control: 4 μL of 500 mM GC376 (BPS Bioscience) in DMSO, and (iii) blank control: no enzyme was added in the 56 μL assay buffer, no inhibitor dissolved in DMSO. The reactions ran for 2 h, and the linear initial slopes of the progress curves were used to calculate the reaction initial velocity in RFU in time (RFU s<sup>−1</sup>). <em>K</em>
<sub>i</sub> values were fit to the Morrison equation using MATLAB (MathWorks, Inc., Natick, MA, USA).</p></section><section id="pro70275-sec-0022"><h3 class="pmc_sec_title">5.6. Measurement of disassociation constant</h3>
<p>The dependence of the enzymatic activity on the total enzyme concentration was determined using the FRET‐based assay described above with the previously described method (Wang et al., <a href="#pro70275-bib-0069" class="usa-link" aria-describedby="pro70275-bib-0069">2024</a>). The final enzyme concentrations were varied over a range from 20 nM to 5.12 μM for S1D 3CL<sup>pro</sup>. Reactions were initiated by the addition of the substrate (final concentration: 12 μM) to the wells containing enzyme in the assay buffer. Initial rates in RFU s<sup>−1</sup> were converted to μM s<sup>−1</sup> using the calibration curve. The rate of the reaction was plotted as a function of the enzyme concentration and fitted in MATLAB (MathWorks, Inc., Natick, MA, USA) to yield <em>K</em>
<sub>D</sub> using <em>fminsearch</em> as the fitting algorithm and Monte‐Carlo sampling for generating the standard deviations of the fitted parameters.</p></section><section id="pro70275-sec-0023"><h3 class="pmc_sec_title">5.7. Michaelis–Menten kinetics</h3>
<p>The rates of enzymatic activity were measured as a function of varying substrate concentrations at a fixed enzyme concentration. Reactions were initiated by adding the enzyme to the wells containing varying concentrations of substrate (final concentrations: from 3.125 to 200 μM). The final concentration of 3CL<sup>pro</sup> was 400 nM. The rate of the reaction was plotted as a function of the substrate concentration and fitted to the Michaelis–Menten equation in MATLAB using <em>fminsearch</em> as the fitting algorithm.</p></section><section id="pro70275-sec-0021"><h3 class="pmc_sec_title">5.8. Ligand‐induced dimerization</h3>
<p>Reactions were performed in 100 μL volume. The final concentration of the enzyme was 15 nM or 60 nM, and the final concentration of the substrate was 12 μM. Fifty‐eight microliters of enzyme (diluted in the assay buffer) were incubated for 30 min at room temperature with 2 μL of compound diluted in DMSO (MilliporeSigma, MA, USA). To initiate the reaction, 40 μL of substrate was added, and the reaction was monitored by following the fluorescence as a function of time. Controls were (i) negative control: no inhibitor dissolved in DMSO and (ii) blank control: no enzyme was added in the 28 μL assay buffer, no inhibitor dissolved in DMSO. The reactions ran for 2 h, and the linear initial slopes of the progress curves were used to calculate the reaction initial velocity in RFU in time (RFU s<sup>−1</sup>).</p>
<p>To fit the experimental data presented in Figure <a href="#pro70275-fig-0003" class="usa-link">3b</a>, the percentage activity measured in the FRET assay can be expressed as:</p>
<table class="disp-formula p" id="pro70275-disp-0003"><tr>
<td class="formula"><math id="jats-math-3" display="block" overflow="linebreak"><mrow><mrow><mo>%</mo><mtext>Activity</mtext><mo linebreak="goodbreak">=</mo><mn>100</mn><mfrac><mrow><mfenced open="[" close="]"><mi mathvariant="normal">D</mi></mfenced><mo>+</mo><mn>0.5</mn><mfenced open="[" close="]"><mi>DI</mi></mfenced></mrow><msub><mfenced open="[" close="]"><mi mathvariant="normal">D</mi></mfenced><mi>CTR</mi></msub></mfrac><mo>.</mo></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>The singly occupied dimer, DI, is only half as active as the free dimer, D, as DI only has one site available for catalysis and is statistically 50% less active than D. [D]<sub>CTR</sub> is the concentration of the dimer in the control experiment without inhibitor, which is expressed as (Wang et al., <a href="#pro70275-bib-0069" class="usa-link" aria-describedby="pro70275-bib-0069">2024</a>):</p>
<table class="disp-formula p" id="pro70275-disp-0004"><tr>
<td class="formula"><math id="jats-math-4" display="block" overflow="linebreak"><mrow><mrow><msub><mfenced open="[" close="]"><mi mathvariant="normal">D</mi></mfenced><mi>CTR</mi></msub><mo linebreak="goodbreak">=</mo><mfrac><mrow><mn>4</mn><msub><mfenced open="[" close="]"><mi mathvariant="normal">E</mi></mfenced><mi mathvariant="normal">t</mi></msub><mo>+</mo><msub><mi>K</mi><mi mathvariant="normal">D</mi></msub><msub><mi>f</mi><mi mathvariant="normal">c</mi></msub><mo>−</mo><msqrt><mrow><mn>8</mn><msub><mfenced open="[" close="]"><mi mathvariant="normal">E</mi></mfenced><mi mathvariant="normal">t</mi></msub><msub><mi>K</mi><mi mathvariant="normal">D</mi></msub><msub><mi>f</mi><mi mathvariant="normal">c</mi></msub><mo>+</mo><msup><mfenced open="(" close=")"><mrow><msub><mi>K</mi><mi mathvariant="normal">D</mi></msub><msub><mi>f</mi><mi mathvariant="normal">c</mi></msub></mrow></mfenced><mn>2</mn></msup></mrow></msqrt></mrow><mn>8</mn></mfrac><mo>.</mo></mrow></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>Dimerization of a monomer and a monomer‐inhibitor complex (M + MI → DI) is controlled by the dissociation constant <em>K</em>
<sub>D</sub>, where:</p>
<table class="disp-formula p" id="pro70275-disp-0005"><tr>
<td class="formula"><math id="jats-math-5" display="block" overflow="linebreak"><mrow><mrow><msub><mi>K</mi><mi mathvariant="normal">D</mi></msub><mo linebreak="goodbreak">=</mo><mfrac><mrow><mfenced open="[" close="]"><mi mathvariant="normal">M</mi></mfenced><mfenced open="[" close="]"><mi>MI</mi></mfenced></mrow><mfenced open="[" close="]"><mi mathvariant="normal">D</mi></mfenced></mfrac><mo>.</mo></mrow></mrow></math></td>
<td class="label">(5)</td>
</tr></table>
<p>And dimerization of two monomers (2M → D) is defined as:</p>
<table class="disp-formula p" id="pro70275-disp-0006"><tr>
<td class="formula"><math id="jats-math-6" display="block" overflow="linebreak"><mrow><mrow><msub><mi>K</mi><mi mathvariant="normal">D</mi></msub><msub><mi>f</mi><mi mathvariant="normal">c</mi></msub><mo linebreak="goodbreak">=</mo><mfrac><msup><mfenced open="[" close="]"><mi mathvariant="normal">M</mi></mfenced><mn>2</mn></msup><mfenced open="[" close="]"><mi mathvariant="normal">D</mi></mfenced></mfrac><mo>.</mo></mrow></mrow></math></td>
<td class="label">(6)</td>
</tr></table>
<p>Similarly, the inhibition constant, <em>K</em>
<sub>i</sub>, between the dimer and the ligand (D + I → DI) or between the singly occupied dimer and the ligand (DI + I → DI<sub>2</sub>) is defined as:</p>
<table class="disp-formula p" id="pro70275-disp-0007"><tr>
<td class="formula"><math id="jats-math-7" display="block" overflow="linebreak"><mrow><mrow><msub><mi>K</mi><mi mathvariant="normal">i</mi></msub><mo linebreak="goodbreak">=</mo><mfrac><mrow><mfenced open="[" close="]"><mi mathvariant="normal">D</mi></mfenced><msub><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">f</mi></msub></mrow><mfenced open="[" close="]"><mi>DI</mi></mfenced></mfrac><mo linebreak="goodbreak">=</mo><mfrac><mrow><mfenced open="[" close="]"><mi>DI</mi></mfenced><msub><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">f</mi></msub></mrow><mfenced open="[" close="]"><mrow><mi mathvariant="normal">D</mi><msub><mi mathvariant="normal">I</mi><mn>2</mn></msub></mrow></mfenced></mfrac><mo>.</mo></mrow></mrow></math></td>
<td class="label">(7)</td>
</tr></table>
<p>where [I]<sub>f</sub> is the concentration of the free inhibitor in the assay. Binding of the monomer to the inhibitor (M + I → MI) is controlled by:</p>
<table class="disp-formula p" id="pro70275-disp-0008"><tr>
<td class="formula"><math id="jats-math-8" display="block" overflow="linebreak"><mrow><mrow><msub><mi>K</mi><mi mathvariant="normal">i</mi></msub><msub><mi>f</mi><mi mathvariant="normal">c</mi></msub><mo linebreak="goodbreak">=</mo><mfrac><mrow><mfenced open="[" close="]"><mi mathvariant="normal">M</mi></mfenced><msub><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">f</mi></msub></mrow><mfenced open="[" close="]"><mi>MI</mi></mfenced></mfrac><mo>.</mo></mrow></mrow></math></td>
<td class="label">(8)</td>
</tr></table>
<p>Consequently, the total enzyme concentration can be calculated in terms of <em>K</em>
<sub>
<em>i</em>
</sub>, <em>K</em>
<sub>D</sub>, <em>f</em>
<sub>c</sub>, [M], and [I]<sub>f</sub> according to:</p>
<table class="disp-formula p" id="pro70275-disp-0009"><tr>
<td class="formula"><math id="jats-math-9" display="block" overflow="linebreak"><mrow><mrow><mtable columnalign="right center left" columnspacing="3pt" displaystyle="true"><mtr><mtd><msubsup><mfenced open="[" close="]"><mi mathvariant="normal">E</mi></mfenced><mi mathvariant="normal">t</mi><mtext>calc</mtext></msubsup></mtd><mtd><mo linebreak="goodbreak">=</mo></mtd><mtd><mfenced open="[" close="]"><mi mathvariant="normal">M</mi></mfenced><mo linebreak="goodbreak">+</mo><mfenced open="[" close="]"><mi mathvariant="normal">MI</mi></mfenced><mo linebreak="goodbreak">+</mo><mn>2</mn><mfenced><mrow><mfenced open="[" close="]"><mi mathvariant="normal">D</mi></mfenced><mo linebreak="goodbreak">+</mo><mfenced open="[" close="]"><mi mathvariant="normal">DI</mi></mfenced><mo linebreak="goodbreak">+</mo><mfenced open="[" close="]"><msub><mi mathvariant="normal">DI</mi><mn>2</mn></msub></mfenced></mrow></mfenced></mtd></mtr><mtr><mtd><mrow></mrow></mtd><mtd><mo linebreak="goodbreak">=</mo></mtd><mtd><mfenced open="[" close="]"><mi mathvariant="normal">M</mi></mfenced><mo linebreak="goodbreak">+</mo><mfrac><mrow><mfenced open="[" close="]"><mi mathvariant="normal">M</mi></mfenced><msub><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">f</mi></msub></mrow><mrow><msub><mi>K</mi><mi mathvariant="normal">i</mi></msub><msub><mi>f</mi><mi mathvariant="normal">c</mi></msub></mrow></mfrac><mo linebreak="goodbreak">+</mo><mn>2</mn><mfenced><mrow><mfrac><msup><mfenced open="[" close="]"><mi mathvariant="normal">M</mi></mfenced><mn>2</mn></msup><mrow><msub><mi>K</mi><mi mathvariant="normal">D</mi></msub><msub><mi>f</mi><mi mathvariant="normal">c</mi></msub></mrow></mfrac><mo linebreak="goodbreak">+</mo><mfrac><mrow><msup><mfenced open="[" close="]"><mi mathvariant="normal">M</mi></mfenced><mn>2</mn></msup><msub><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">f</mi></msub></mrow><mrow><msub><mi>K</mi><mi mathvariant="normal">D</mi></msub><msub><mi>K</mi><mi mathvariant="normal">i</mi></msub><msub><mi>f</mi><mi mathvariant="normal">c</mi></msub></mrow></mfrac><mo linebreak="goodbreak">+</mo><mfrac><mrow><msup><mfenced open="[" close="]"><mi mathvariant="normal">M</mi></mfenced><mn>2</mn></msup><msubsup><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">f</mi><mn>2</mn></msubsup></mrow><mrow><msub><mi>K</mi><mi mathvariant="normal">D</mi></msub><msubsup><mi>K</mi><mi mathvariant="normal">i</mi><mn>2</mn></msubsup><msub><mi>f</mi><mi mathvariant="normal">c</mi></msub></mrow></mfrac></mrow></mfenced><mo>.</mo></mtd></mtr></mtable></mrow></mrow></math></td>
<td class="label">(9)</td>
</tr></table>
<p>Similarly, the total inhibitor concentration can be expressed as:</p>
<table class="disp-formula p" id="pro70275-disp-0010"><tr>
<td class="formula"><math id="jats-math-10" display="block" overflow="linebreak"><mrow><mrow><msubsup><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">t</mi><mtext>calc</mtext></msubsup><mo linebreak="goodbreak">=</mo><mtable columnalign="left" displaystyle="true"><mtr><mtd><msub><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">f</mi></msub><mo linebreak="goodbreak">+</mo><mfenced open="[" close="]"><mi mathvariant="normal">MI</mi></mfenced><mo linebreak="goodbreak">+</mo><mfenced open="[" close="]"><mi mathvariant="normal">DI</mi></mfenced><mo linebreak="goodbreak">+</mo><mn>2</mn><mfenced open="[" close="]"><msub><mi mathvariant="normal">DI</mi><mn>2</mn></msub></mfenced><mo linebreak="goodbreak">=</mo><msub><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">f</mi></msub><mo linebreak="goodbreak">+</mo><mfrac><mrow><mfenced open="[" close="]"><mi mathvariant="normal">M</mi></mfenced><msub><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">f</mi></msub></mrow><mrow><msub><mi>K</mi><mi mathvariant="normal">i</mi></msub><msub><mi>f</mi><mi mathvariant="normal">c</mi></msub></mrow></mfrac></mtd></mtr><mtr><mtd><mo linebreak="goodbreak">+</mo><mspace width="2px"></mspace><mfrac><mrow><msup><mfenced open="[" close="]"><mi mathvariant="normal">M</mi></mfenced><mn>2</mn></msup><msub><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">f</mi></msub></mrow><mrow><msub><mi>K</mi><mi mathvariant="normal">D</mi></msub><msub><mi>K</mi><mi mathvariant="normal">i</mi></msub><msub><mi>f</mi><mi mathvariant="normal">c</mi></msub></mrow></mfrac><mo linebreak="goodbreak">+</mo><mn>2</mn><mfrac><mrow><msup><mfenced open="[" close="]"><mi mathvariant="normal">M</mi></mfenced><mn>2</mn></msup><msubsup><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">f</mi><mn>2</mn></msubsup></mrow><mrow><msub><mi>K</mi><mi mathvariant="normal">D</mi></msub><msubsup><mi>K</mi><mi mathvariant="normal">i</mi><mn>2</mn></msubsup><msub><mi>f</mi><mi mathvariant="normal">c</mi></msub></mrow></mfrac><mo>.</mo></mtd></mtr></mtable></mrow></mrow></math></td>
<td class="label">(10)</td>
</tr></table>
<p>For any given set of <em>K</em>
<sub>i</sub>, <em>K</em>
<sub>D</sub>, and <em>f</em>
<sub>c</sub> parameters, the values of [I]<sub>f</sub> and [M] were varied in order to minimize the target function:</p>
<table class="disp-formula p" id="pro70275-disp-0011"><tr>
<td class="formula"><math id="jats-math-11" display="block" overflow="linebreak"><mrow><mrow><mi>RSS</mi><mo linebreak="goodbreak">=</mo><msup><mfenced open="(" close=")"><mrow><msubsup><mfenced open="[" close="]"><mi mathvariant="normal">E</mi></mfenced><mi mathvariant="normal">t</mi><mtext>calc</mtext></msubsup><mo linebreak="goodbreak">−</mo><msubsup><mfenced open="[" close="]"><mi mathvariant="normal">E</mi></mfenced><mi mathvariant="normal">t</mi><mtext>true</mtext></msubsup></mrow></mfenced><mn>2</mn></msup><mo linebreak="goodbreak">+</mo><msup><mfenced open="(" close=")"><mrow><msubsup><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">t</mi><mtext>calc</mtext></msubsup><mo linebreak="goodbreak">−</mo><msubsup><mfenced open="[" close="]"><mi mathvariant="normal">I</mi></mfenced><mi mathvariant="normal">t</mi><mtext>true</mtext></msubsup></mrow></mfenced><mn>2</mn></msup></mrow></mrow></math></td>
<td class="label">(11)</td>
</tr></table>
<p>using the <em>fminsearch</em> function in MATLAB and standard deviations of the fitted parameters were generated by Monte‐Carlo sampling. The resulting values of [I]<sub>f</sub> and [M] were then used to calculate the concentrations of D and DI and overall enzyme activity. The values of <em>K</em>
<sub>i</sub>, <em>K</em>
<sub>D</sub>, and <em>f</em>
<sub>c</sub> were then varied until the calculated enzyme activities matched the experimental dose–response curves (for the details of this numerical solution, please see the MATLAB published on GitHub at: <a href="https://github.com/GuanyuWang-McGill/Ligand_Induced_Dimerization" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/GuanyuWang-McGill/Ligand_Induced_Dimerization</a>).</p></section><section id="pro70275-sec-0025"><h3 class="pmc_sec_title">5.9. 
nanoESI‐MS
</h3>
<p>Immediately prior to native MS analysis, 3CL<sup>pro</sup> proteins were buffer exchanged into 200 mM ammonium acetate using Micro Bio‐Spin 6 columns (Bio‐Rad, CA, USA) and the pH was adjusted to 7.5 using ammonium hydroxide. Following buffer exchange, the concentrations of the protein samples were determined using UV absorption and were diluted to 2 μM in 200 mM ammonium acetate. Protein samples (5–10 μL) were loaded into gold‐coated, nanospray emitters that were prepared in‐house from borosilicate capillaries (1.0 mm OD × 0.78 mm ID × 100 mm L, Harvard Apparatus). The capillaries were pulled to have a 1 μm orifice (estimated by light microscopy) using a Sutter Instruments Model P‐2000 tip puller (parameters: heat = 350, filament = 4, velocity = 60, delay = 255, pull = 0). The loaded emitter was briefly centrifuged to move the fluid to the end of the tip and was then mounted onto the nanospray ESI source of a Synapt G2‐Si mass spectrometer (Waters). Data were acquired in positive ion and sensitivity modes over an <em>m</em>/<em>z</em> range of 100–8000 using a capillary voltage = 1.5 kV, cone voltage = 50 V, source offset = 50 V, source temperature = 50°C, trap collision energy = 4 V, trap gas (argon) flow = 2 mL min<sup>−1</sup>, and trap direct current bias = 35 V. Mass spectra were smoothed (Savitsky‐Golay, smooth window = 10, number of smooths = 3) and deconvoluted (Maxent 1, resolution = 1.00, uniform Gaussian, width at half height = 25 Da) using Mass Lynx v4.2 software (Waters).</p></section></section><section id="pro70275-sec-0030"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Guanyu Wang:</strong> Conceptualization; writing – original draft; methodology; formal analysis; data curation; visualization; writing – review and editing; resources; investigation. <strong>Felipe Venegas:</strong> Resources; investigation; writing – review and editing. <strong>Andres Rueda:</strong> Investigation; resources; writing – review and editing. <strong>Osvaldo Yañez:</strong> Investigation; formal analysis; data curation; writing – review and editing; visualization. <strong>Manuel I. Osorio:</strong> Investigation; formal analysis; data curation; writing – review and editing; visualization. <strong>Sibei Qin:</strong> Data curation; writing – review and editing. <strong>José Manuel Pérez‐Donoso:</strong> Project administration; writing – review and editing. <strong>Christopher J. Thibodeaux:</strong> Formal analysis; data curation; writing – review and editing; visualization. <strong>Nicolas Moitessier:</strong> Project administration; writing – review and editing; funding acquisition. <strong>Anthony K. Mittermaier:</strong> Funding acquisition; writing – review and editing; project administration; supervision.</p></section><section id="pro70275-sec-0027"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflict of interest.</p></section><section id="sec27"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pro70275-supitem-0001"><div class="caption p"><p>
<strong>Data S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375981/bin/PRO-34-e70275-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70275-s001.docx</a><sup> (14.3MB, docx) </sup>
</div></div></section></section><section id="pro70275-sec-0026" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This research was supported by the Natural Sciences and Engineering Research Council (NSERC, Canada, Grant 327028‐09) and the Canadian Institutes of Health Research (CIHR, OV3‐170644 and MOP‐13694). Anthony Mittemaier is a member of the PROTEO Regroupement Stratégique, Fonds de recherche du Québec—Nature et Technologies (FRQNT). Anthony Mittermaier and Nicolas Moitessier are members of the Centre de Recherche en Biologie Structurale, Fonds de recherche du Québec—Santé (FRQS).</p></section><section id="notes1"><p>

Wang G, Venegas F, Rueda A, Yañez O, Osorio MI, Qin S, et al. Inhibitor‐induced dimerization mediates lufotrelvir resistance in mutants of SARS‐CoV‐2 3C‐like protease
. Protein Science. 2025;34(9):e70275. 10.1002/pro.70275

</p>
<section id="pro70275-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="pro70275-note-0001"><p>
<strong>Review Editor:</strong> Lynn Kamerlin</p></div></div></section></section><section id="pro70275-sec-0029"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The data that supports the findings of this study are available in the supplementary material of this article.</p></section><section id="pro70275-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="pro70275-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="pro70275-bib-0001">
<cite>
Al Adem K, Ferreira JC, Fadl S, Mustafa M, Rabeh WM. Key allosteric and active site residues of SARS‐CoV‐2 3CL<sup>pro</sup> are promising drug targets. Biochem J. 2023;480(11):791–813.</cite> [<a href="https://doi.org/10.1042/BCJ20230027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37254750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20J&amp;title=Key%20allosteric%20and%20active%20site%20residues%20of%20SARS%E2%80%90CoV%E2%80%902%203CLpro%20are%20promising%20drug%20targets&amp;author=K%20Al%20Adem&amp;author=JC%20Ferreira&amp;author=S%20Fadl&amp;author=M%20Mustafa&amp;author=WM%20Rabeh&amp;volume=480&amp;issue=11&amp;publication_year=2023&amp;pages=791-813&amp;pmid=37254750&amp;doi=10.1042/BCJ20230027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0002">
<cite>
Atkinson SC, Dogovski C, Wood K, Griffin MDW, Gorman MA, Hor L, et al. Substrate locking promotes dimer‐dimer docking of an enzyme antibiotic target. Structure. 2018;26(7):948–959.e945.
</cite> [<a href="https://doi.org/10.1016/j.str.2018.04.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29804823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Substrate%20locking%20promotes%20dimer%E2%80%90dimer%20docking%20of%20an%20enzyme%20antibiotic%20target&amp;author=SC%20Atkinson&amp;author=C%20Dogovski&amp;author=K%20Wood&amp;author=MDW%20Griffin&amp;author=MA%20Gorman&amp;volume=26&amp;issue=7&amp;publication_year=2018&amp;pages=948-959.e945&amp;pmid=29804823&amp;doi=10.1016/j.str.2018.04.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0003">
<cite>
Boras B, Jones RM, Anson BJ, Arenson D, Aschenbrenner L, Bakowski MA, et al. Preclinical characterization of an intravenous coronavirus 3CL protease inhibitor for the potential treatment of COVID19. Nat Commun. 2021;12(1):6055.
</cite> [<a href="https://doi.org/10.1038/s41467-021-26239-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8523698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34663813/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Preclinical%20characterization%20of%20an%20intravenous%20coronavirus%203CL%20protease%20inhibitor%20for%20the%20potential%20treatment%20of%20COVID19&amp;author=B%20Boras&amp;author=RM%20Jones&amp;author=BJ%20Anson&amp;author=D%20Arenson&amp;author=L%20Aschenbrenner&amp;volume=12&amp;issue=1&amp;publication_year=2021&amp;pages=6055&amp;pmid=34663813&amp;doi=10.1038/s41467-021-26239-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0004">
<cite>
Chen S, Hu T, Zhang J, Chen J, Chen K, Ding J, et al. Mutation of Gly‐11 on the dimer interface results in the complete crystallographic dimer dissociation of severe acute respiratory syndrome coronavirus 3C‐like protease: crystal structure with molecular dynamics simulations. J Biol Chem. 2008;283(1):554–564.
</cite> [<a href="https://doi.org/10.1074/jbc.M705240200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7982321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17977841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Mutation%20of%20Gly%E2%80%9011%20on%20the%20dimer%20interface%20results%20in%20the%20complete%20crystallographic%20dimer%20dissociation%20of%20severe%20acute%20respiratory%20syndrome%20coronavirus%203C%E2%80%90like%20protease:%20crystal%20structure%20with%20molecular%20dynamics%20simulations&amp;author=S%20Chen&amp;author=T%20Hu&amp;author=J%20Zhang&amp;author=J%20Chen&amp;author=K%20Chen&amp;volume=283&amp;issue=1&amp;publication_year=2008&amp;pages=554-564&amp;pmid=17977841&amp;doi=10.1074/jbc.M705240200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0005">
<cite>
Chen S, Zhang J, Hu T, Chen K, Jiang H, Shen X. Residues on the dimer interface of SARS coronavirus 3C‐like protease: dimer stability characterization and enzyme catalytic activity analysis. J Biochem. 2008;143(4):525–536.
</cite> [<a href="https://doi.org/10.1093/jb/mvm246" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7109808/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18182387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biochem&amp;title=Residues%20on%20the%20dimer%20interface%20of%20SARS%20coronavirus%203C%E2%80%90like%20protease:%20dimer%20stability%20characterization%20and%20enzyme%20catalytic%20activity%20analysis&amp;author=S%20Chen&amp;author=J%20Zhang&amp;author=T%20Hu&amp;author=K%20Chen&amp;author=H%20Jiang&amp;volume=143&amp;issue=4&amp;publication_year=2008&amp;pages=525-536&amp;pmid=18182387&amp;doi=10.1093/jb/mvm246&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0006">
<cite>
Cheng SC, Chang GG, Chou CY. Mutation of Glu‐166 blocks the substrate‐induced dimerization of SARS coronavirus main protease. Biophys J. 2010;98(7):1327–1336.
</cite> [<a href="https://doi.org/10.1016/j.bpj.2009.12.4272" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2849084/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20371333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Mutation%20of%20Glu%E2%80%90166%20blocks%20the%20substrate%E2%80%90induced%20dimerization%20of%20SARS%20coronavirus%20main%20protease&amp;author=SC%20Cheng&amp;author=GG%20Chang&amp;author=CY%20Chou&amp;volume=98&amp;issue=7&amp;publication_year=2010&amp;pages=1327-1336&amp;pmid=20371333&amp;doi=10.1016/j.bpj.2009.12.4272&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0007">
<cite>
Datta D, McClendon CL, Jacobson MP, Wells JA. Substrate and inhibitor‐induced dimerization and cooperativity in caspase‐1 but not caspase‐3. J Biol Chem. 2013;288(14):9971–9981.
</cite> [<a href="https://doi.org/10.1074/jbc.M112.426460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3617296/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23386603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Substrate%20and%20inhibitor%E2%80%90induced%20dimerization%20and%20cooperativity%20in%20caspase%E2%80%901%20but%20not%20caspase%E2%80%903&amp;author=D%20Datta&amp;author=CL%20McClendon&amp;author=MP%20Jacobson&amp;author=JA%20Wells&amp;volume=288&amp;issue=14&amp;publication_year=2013&amp;pages=9971-9981&amp;pmid=23386603&amp;doi=10.1074/jbc.M112.426460&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0008">
<cite>
El‐Baba TJ, Lutomski CA, Kantsadi AL, Malla TR, John T, Mikhailov V, et al. Allosteric inhibition of the SARS‐CoV‐2 main protease: insights from mass spectrometry based assays. Angew Chem Int Ed. 2020;59(52):23544–23548.</cite> [<a href="https://doi.org/10.1002/anie.202010316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7461284/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32841477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew%20Chem%20Int%20Ed&amp;title=Allosteric%20inhibition%20of%20the%20SARS%E2%80%90CoV%E2%80%902%20main%20protease:%20insights%20from%20mass%20spectrometry%20based%20assays&amp;author=TJ%20El%E2%80%90Baba&amp;author=CA%20Lutomski&amp;author=AL%20Kantsadi&amp;author=TR%20Malla&amp;author=T%20John&amp;volume=59&amp;issue=52&amp;publication_year=2020&amp;pages=23544-23548&amp;pmid=32841477&amp;doi=10.1002/anie.202010316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0009"><cite>
FDA
. Coronavirus (covid‐19)|drugs. 2023. <a href="https://www.fda.gov/drugs/emergency-preparedness-drugs/coronavirus-covid-19-drugs" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/drugs/emergency‐preparedness‐drugs/coronavirus‐covid‐19‐drugs</a>. Accessed 18 Dec 2023</cite></li>
<li id="pro70275-bib-0010">
<cite>
Ferreira JC, Fadl S, Rabeh WM. Key dimer interface residues impact the catalytic activity of 3CL<sup>pro</sup>, the main protease of SARS‐CoV‐2. J Biol Chem. 2022;298(6):102023.
</cite> [<a href="https://doi.org/10.1016/j.jbc.2022.102023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9091064/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35568197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Key%20dimer%20interface%20residues%20impact%20the%20catalytic%20activity%20of%203CLpro,%20the%20main%20protease%20of%20SARS%E2%80%90CoV%E2%80%902&amp;author=JC%20Ferreira&amp;author=S%20Fadl&amp;author=WM%20Rabeh&amp;volume=298&amp;issue=6&amp;publication_year=2022&amp;pages=102023&amp;pmid=35568197&amp;doi=10.1016/j.jbc.2022.102023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0011">
<cite>
Forester TR, Smith W. Shake, rattle, and roll: efficient constraint algorithms for linked rigid bodies. J Comput Chem. 1998;19(1):102–111.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Chem&amp;title=Shake,%20rattle,%20and%20roll:%20efficient%20constraint%20algorithms%20for%20linked%20rigid%20bodies&amp;author=TR%20Forester&amp;author=W%20Smith&amp;volume=19&amp;issue=1&amp;publication_year=1998&amp;pages=102-111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0012">
<cite>
Friedman R. Computational studies of protein–drug binding affinity changes upon mutations in the drug target. WIREs Comput Mol Sci. 2022;12(1):e1563.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=WIREs%20Comput%20Mol%20Sci&amp;title=Computational%20studies%20of%20protein%E2%80%93drug%20binding%20affinity%20changes%20upon%20mutations%20in%20the%20drug%20target&amp;author=R%20Friedman&amp;volume=12&amp;issue=1&amp;publication_year=2022&amp;pages=e1563&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0013"><cite>
Frisch MJ, Trucks GW, Schlegel HB, Scuseria GE, Robb MA, Cheeseman JR, et al. Gaussian 16 rev. C.01. Wallingford: Gaussian, Inc.
2016. <a href="https://gaussian.com/citation/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gaussian.com/citation/</a>.</cite></li>
<li id="pro70275-bib-0014">
<cite>
Gordon JC, Myers JB, Folta T, Shoja V, Heath LS, Onufriev A. H++: a server for estimating pKas and adding missing hydrogens to macromolecules. Nucleic Acids Res. 2005;33(Web Server Issue):W368–W371.
</cite> [<a href="https://doi.org/10.1093/nar/gki464" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1160225/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15980491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=H++:%20a%20server%20for%20estimating%20pKas%20and%20adding%20missing%20hydrogens%20to%20macromolecules&amp;author=JC%20Gordon&amp;author=JB%20Myers&amp;author=T%20Folta&amp;author=V%20Shoja&amp;author=LS%20Heath&amp;volume=33&amp;issue=Web%20Server%20Issue&amp;publication_year=2005&amp;pages=W368-W371&amp;pmid=15980491&amp;doi=10.1093/nar/gki464&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0015">
<cite>
Graziano V, McGrath WJ, DeGruccio AM, Dunn JJ, Mangel WF. Enzymatic activity of the SARS coronavirus main proteinase dimer. FEBS Lett. 2006;580(11):2577–2583.
</cite> [<a href="https://doi.org/10.1016/j.febslet.2006.04.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7094300/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16647061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&amp;title=Enzymatic%20activity%20of%20the%20SARS%20coronavirus%20main%20proteinase%20dimer&amp;author=V%20Graziano&amp;author=WJ%20McGrath&amp;author=AM%20DeGruccio&amp;author=JJ%20Dunn&amp;author=WF%20Mangel&amp;volume=580&amp;issue=11&amp;publication_year=2006&amp;pages=2577-2583&amp;pmid=16647061&amp;doi=10.1016/j.febslet.2006.04.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0016">
<cite>
Grum‐Tokars V, Ratia K, Begaye A, Baker SC, Mesecar AD. Evaluating the 3C‐like protease activity of SARS‐coronavirus: recommendations for standardized assays for drug discovery. Virus Res. 2008;133(1):63–73.
</cite> [<a href="https://doi.org/10.1016/j.virusres.2007.02.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4036818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17397958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virus%20Res&amp;title=Evaluating%20the%203C%E2%80%90like%20protease%20activity%20of%20SARS%E2%80%90coronavirus:%20recommendations%20for%20standardized%20assays%20for%20drug%20discovery&amp;author=V%20Grum%E2%80%90Tokars&amp;author=K%20Ratia&amp;author=A%20Begaye&amp;author=SC%20Baker&amp;author=AD%20Mesecar&amp;volume=133&amp;issue=1&amp;publication_year=2008&amp;pages=63-73&amp;pmid=17397958&amp;doi=10.1016/j.virusres.2007.02.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0017">
<cite>
Hammond J, Leister‐Tebbe H, Gardner A, Abreu P, Bao W, Wisemandle W, et al. Oral nirmatrelvir for high‐risk, nonhospitalized adults with COVID‐19. N Engl J Med. 2022;386(15):1397–1408.
</cite> [<a href="https://doi.org/10.1056/NEJMoa2118542" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8908851/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35172054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&amp;title=Oral%20nirmatrelvir%20for%20high%E2%80%90risk,%20nonhospitalized%20adults%20with%20COVID%E2%80%9019&amp;author=J%20Hammond&amp;author=H%20Leister%E2%80%90Tebbe&amp;author=A%20Gardner&amp;author=P%20Abreu&amp;author=W%20Bao&amp;volume=386&amp;issue=15&amp;publication_year=2022&amp;pages=1397-1408&amp;pmid=35172054&amp;doi=10.1056/NEJMoa2118542&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0018">
<cite>
Harvey MJ, De Fabritiis G. An implementation of the smooth particle mesh Ewald method on GPU hardware. J Chem Theory Comput. 2009;5(9):2371–2377.
</cite> [<a href="https://doi.org/10.1021/ct900275y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26616618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Theory%20Comput&amp;title=An%20implementation%20of%20the%20smooth%20particle%20mesh%20Ewald%20method%20on%20GPU%20hardware&amp;author=MJ%20Harvey&amp;author=G%20De%20Fabritiis&amp;volume=5&amp;issue=9&amp;publication_year=2009&amp;pages=2371-2377&amp;pmid=26616618&amp;doi=10.1021/ct900275y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0019">
<cite>
Havranek B, Demissie R, Lee H, Lan S, Zhang H, Sarafianos S, et al. Discovery of nirmatrelvir resistance mutations in SARS‐CoV‐2 3CL<sup>pro</sup>: a computational‐experimental approach. J Chem Inf Model. 2023;63:7180–7188.
</cite> [<a href="https://doi.org/10.1021/acs.jcim.3c01269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10976418/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37947496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Inf%20Model&amp;title=Discovery%20of%20nirmatrelvir%20resistance%20mutations%20in%20SARS%E2%80%90CoV%E2%80%902%203CLpro:%20a%20computational%E2%80%90experimental%20approach&amp;author=B%20Havranek&amp;author=R%20Demissie&amp;author=H%20Lee&amp;author=S%20Lan&amp;author=H%20Zhang&amp;volume=63&amp;publication_year=2023&amp;pages=7180-7188&amp;pmid=37947496&amp;doi=10.1021/acs.jcim.3c01269&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0020">
<cite>
Heilmann E, Costacurta F, Moghadasi SA, Ye C, Pavan M, Bassani D, et al. SARS‐CoV‐2 3CL<sup>pro</sup> mutations selected in a VSV‐based system confer resistance to nirmatrelvir, ensitrelvir, and GC376. Sci Transl Med. 2023;15(678):eabq7360.
</cite> [<a href="https://doi.org/10.1126/scitranslmed.abq7360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9765458/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36194133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&amp;title=SARS%E2%80%90CoV%E2%80%902%203CLpro%20mutations%20selected%20in%20a%20VSV%E2%80%90based%20system%20confer%20resistance%20to%20nirmatrelvir,%20ensitrelvir,%20and%20GC376&amp;author=E%20Heilmann&amp;author=F%20Costacurta&amp;author=SA%20Moghadasi&amp;author=C%20Ye&amp;author=M%20Pavan&amp;volume=15&amp;issue=678&amp;publication_year=2023&amp;pages=eabq7360&amp;pmid=36194133&amp;doi=10.1126/scitranslmed.abq7360&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0021">
<cite>
Ho B‐L, Cheng S‐C, Shi L, Wang T‐Y, Ho K‐I, Chou C‐Y. Critical assessment of the important residues involved in the dimerization and catalysis of MERS coronavirus main protease. PLoS One. 2015;10(12):e0144865.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0144865" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4682845/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26658006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Critical%20assessment%20of%20the%20important%20residues%20involved%20in%20the%20dimerization%20and%20catalysis%20of%20MERS%20coronavirus%20main%20protease&amp;author=B%E2%80%90L%20Ho&amp;author=S%E2%80%90C%20Cheng&amp;author=L%20Shi&amp;author=T%E2%80%90Y%20Wang&amp;author=K%E2%80%90I%20Ho&amp;volume=10&amp;issue=12&amp;publication_year=2015&amp;pages=e0144865&amp;pmid=26658006&amp;doi=10.1371/journal.pone.0144865&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0022">
<cite>
Hoffman RL, Kania RS, Brothers MA, Davies JF, Ferre RA, Gajiwala KS, et al. Discovery of ketone‐based covalent inhibitors of coronavirus 3CL proteases for the potential therapeutic treatment of COVID‐19. J Med Chem. 2020;63(21):12725–12747.
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.0c01063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33054210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Med%20Chem&amp;title=Discovery%20of%20ketone%E2%80%90based%20covalent%20inhibitors%20of%20coronavirus%203CL%20proteases%20for%20the%20potential%20therapeutic%20treatment%20of%20COVID%E2%80%9019&amp;author=RL%20Hoffman&amp;author=RS%20Kania&amp;author=MA%20Brothers&amp;author=JF%20Davies&amp;author=RA%20Ferre&amp;volume=63&amp;issue=21&amp;publication_year=2020&amp;pages=12725-12747&amp;pmid=33054210&amp;doi=10.1021/acs.jmedchem.0c01063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0023">
<cite>
Hsu M‐F, Kuo C‐J, Chang K‐T, Chang H‐C, Chou C‐C, Ko T‐P, et al. Mechanism of the maturation process of SARS‐CoV 3CL protease. J Biol Chem. 2005;280(35):31257–31266.
</cite> [<a href="https://doi.org/10.1074/jbc.M502577200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8062786/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15788388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Mechanism%20of%20the%20maturation%20process%20of%20SARS%E2%80%90CoV%203CL%20protease&amp;author=M%E2%80%90F%20Hsu&amp;author=C%E2%80%90J%20Kuo&amp;author=K%E2%80%90T%20Chang&amp;author=H%E2%80%90C%20Chang&amp;author=C%E2%80%90C%20Chou&amp;volume=280&amp;issue=35&amp;publication_year=2005&amp;pages=31257-31266&amp;pmid=15788388&amp;doi=10.1074/jbc.M502577200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0024">
<cite>
Hu Y, Lewandowski EM, Tan H, Zhang X, Morgan RT, Zhang X, et al. Naturally occurring mutations of SARS‐CoV‐2 main protease confer drug resistance to nirmatrelvir. ACS Cent Sci. 2023;9(8):1658–1669.
</cite> [<a href="https://doi.org/10.1021/acscentsci.3c00538" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10451032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37637734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Cent%20Sci&amp;title=Naturally%20occurring%20mutations%20of%20SARS%E2%80%90CoV%E2%80%902%20main%20protease%20confer%20drug%20resistance%20to%20nirmatrelvir&amp;author=Y%20Hu&amp;author=EM%20Lewandowski&amp;author=H%20Tan&amp;author=X%20Zhang&amp;author=RT%20Morgan&amp;volume=9&amp;issue=8&amp;publication_year=2023&amp;pages=1658-1669&amp;pmid=37637734&amp;doi=10.1021/acscentsci.3c00538&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0025">
<cite>
Humphrey W, Dalke A, Schulten K. VMD: visual molecular dynamics. J Mol Graph. 1996;14(1):33–38.
</cite> [<a href="https://doi.org/10.1016/0263-7855(96)00018-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8744570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Graph&amp;title=VMD:%20visual%20molecular%20dynamics&amp;author=W%20Humphrey&amp;author=A%20Dalke&amp;author=K%20Schulten&amp;volume=14&amp;issue=1&amp;publication_year=1996&amp;pages=33-38&amp;pmid=8744570&amp;doi=10.1016/0263-7855(96)00018-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0026">
<cite>
Iketani S, Mohri H, Culbertson B, Hong SJ, Duan Y, Luck MI, et al. Multiple pathways for SARS‐CoV‐2 resistance to nirmatrelvir. Nature. 2023;613(7944):558–564.
</cite> [<a href="https://doi.org/10.1038/s41586-022-05514-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9849135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36351451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Multiple%20pathways%20for%20SARS%E2%80%90CoV%E2%80%902%20resistance%20to%20nirmatrelvir&amp;author=S%20Iketani&amp;author=H%20Mohri&amp;author=B%20Culbertson&amp;author=SJ%20Hong&amp;author=Y%20Duan&amp;volume=613&amp;issue=7944&amp;publication_year=2023&amp;pages=558-564&amp;pmid=36351451&amp;doi=10.1038/s41586-022-05514-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0027">
<cite>
Ip JD, Wing‐Ho Chu A, Chan W‐M, Cheuk‐Ying Leung R, Umer Abdullah SM, Sun Y, et al. Global prevalence of SARS‐CoV‐2 3CL protease mutations associated with nirmatrelvir or ensitrelvir resistance. EBioMedicine. 2023;91:104559.
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2023.104559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10101811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37060743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EBioMedicine&amp;title=Global%20prevalence%20of%20SARS%E2%80%90CoV%E2%80%902%203CL%20protease%20mutations%20associated%20with%20nirmatrelvir%20or%20ensitrelvir%20resistance&amp;author=JD%20Ip&amp;author=A%20Wing%E2%80%90Ho%20Chu&amp;author=W%E2%80%90M%20Chan&amp;author=R%20Cheuk%E2%80%90Ying%20Leung&amp;author=SM%20Umer%20Abdullah&amp;volume=91&amp;publication_year=2023&amp;pages=104559&amp;pmid=37060743&amp;doi=10.1016/j.ebiom.2023.104559&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0028">
<cite>
Irwin KK, Renzette N, Kowalik TF, Jensen JD. Antiviral drug resistance as an adaptive process. Virus Evol. 2016;2(1):vew014.
</cite> [<a href="https://doi.org/10.1093/ve/vew014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5499642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28694997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virus%20Evol&amp;title=Antiviral%20drug%20resistance%20as%20an%20adaptive%20process&amp;author=KK%20Irwin&amp;author=N%20Renzette&amp;author=TF%20Kowalik&amp;author=JD%20Jensen&amp;volume=2&amp;issue=1&amp;publication_year=2016&amp;pages=vew014&amp;pmid=28694997&amp;doi=10.1093/ve/vew014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0029">
<cite>
Ivanov I, Shang W, Toledo L, Masgrau L, Svergun DI, Stehling S, et al. Ligand‐induced formation of transient dimers of mammalian 12/15‐lipoxygenase: a key to allosteric behavior of this class of enzymes?
Proteins. 2012;80(3):703–712.
</cite> [<a href="https://doi.org/10.1002/prot.23227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22189720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Ligand%E2%80%90induced%20formation%20of%20transient%20dimers%20of%20mammalian%2012/15%E2%80%90lipoxygenase:%20a%20key%20to%20allosteric%20behavior%20of%20this%20class%20of%20enzymes?&amp;author=I%20Ivanov&amp;author=W%20Shang&amp;author=L%20Toledo&amp;author=L%20Masgrau&amp;author=DI%20Svergun&amp;volume=80&amp;issue=3&amp;publication_year=2012&amp;pages=703-712&amp;pmid=22189720&amp;doi=10.1002/prot.23227&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0030">
<cite>
Jiang H, Zou X, Zeng P, Zeng X, Zhou X, Wang J, et al. Crystal structures of main protease (M<sup>pro</sup>) mutants of SARS‐CoV‐2 variants bound to PF‐07304814. Mol Biomed. 2023;4(1):23.
</cite> [<a href="https://doi.org/10.1186/s43556-023-00134-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10397169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37532968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biomed&amp;title=Crystal%20structures%20of%20main%20protease%20(Mpro)%20mutants%20of%20SARS%E2%80%90CoV%E2%80%902%20variants%20bound%20to%20PF%E2%80%9007304814&amp;author=H%20Jiang&amp;author=X%20Zou&amp;author=P%20Zeng&amp;author=X%20Zeng&amp;author=X%20Zhou&amp;volume=4&amp;issue=1&amp;publication_year=2023&amp;pages=23&amp;pmid=37532968&amp;doi=10.1186/s43556-023-00134-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0031">
<cite>
Jochmans D, Liu C, Donckers K, Stoycheva A, Boland S, Stevens SK, et al. The substitutions L50f, E166a, and L167f in SARS‐CoV‐2 3CL<sup>pro</sup> are selected by a protease inhibitor in vitro and confer resistance to nirmatrelvir. MBio. 2023;14(1):e02815–e02822.
</cite> [<a href="https://doi.org/10.1128/mbio.02815-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9973015/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36625640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MBio&amp;title=The%20substitutions%20L50f,%20E166a,%20and%20L167f%20in%20SARS%E2%80%90CoV%E2%80%902%203CLpro%20are%20selected%20by%20a%20protease%20inhibitor%20in%20vitro%20and%20confer%20resistance%20to%20nirmatrelvir&amp;author=D%20Jochmans&amp;author=C%20Liu&amp;author=K%20Donckers&amp;author=A%20Stoycheva&amp;author=S%20Boland&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=e02815-e02822&amp;pmid=36625640&amp;doi=10.1128/mbio.02815-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0032">
<cite>
Kahn JS, McIntosh K. History and recent advances in coronavirus discovery. Pediatr Infect Dis J. 2005;24(11):S223–S227.
</cite> [<a href="https://doi.org/10.1097/01.inf.0000188166.17324.60" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16378050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr%20Infect%20Dis%20J&amp;title=History%20and%20recent%20advances%20in%20coronavirus%20discovery&amp;author=JS%20Kahn&amp;author=K%20McIntosh&amp;volume=24&amp;issue=11&amp;publication_year=2005&amp;pages=S223-S227&amp;pmid=16378050&amp;doi=10.1097/01.inf.0000188166.17324.60&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0033">
<cite>
Kapikian AZ. The coronaviruses. Dev Biol Stand. 1975;28:42–64.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/1092577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev%20Biol%20Stand&amp;title=The%20coronaviruses&amp;author=AZ%20Kapikian&amp;volume=28&amp;publication_year=1975&amp;pages=42-64&amp;pmid=1092577&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0034">
<cite>
Khare S, Gurry C, Freitas L, Schultz MB, Bach G, Diallo A, et al. GISAID's role in pandemic response. China CDC Wkly. 2021;3(49):1049–1051.
</cite> [<a href="https://doi.org/10.46234/ccdcw2021.255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8668406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34934514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=China%20CDC%20Wkly&amp;title=GISAID's%20role%20in%20pandemic%20response&amp;author=S%20Khare&amp;author=C%20Gurry&amp;author=L%20Freitas&amp;author=MB%20Schultz&amp;author=G%20Bach&amp;volume=3&amp;issue=49&amp;publication_year=2021&amp;pages=1049-1051&amp;pmid=34934514&amp;doi=10.46234/ccdcw2021.255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0035">
<cite>
Kitova EN, El‐Hawiet A, Schnier PD, Klassen JS. Reliable determinations of protein–ligand interactions by direct ESI‐MS measurements. Are we there yet?
J Am Soc Mass Spectrom. 2012;23(3):431–441.
</cite> [<a href="https://doi.org/10.1007/s13361-011-0311-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22270873/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Soc%20Mass%20Spectrom&amp;title=Reliable%20determinations%20of%20protein%E2%80%93ligand%20interactions%20by%20direct%20ESI%E2%80%90MS%20measurements.%20Are%20we%20there%20yet?&amp;author=EN%20Kitova&amp;author=A%20El%E2%80%90Hawiet&amp;author=PD%20Schnier&amp;author=JS%20Klassen&amp;volume=23&amp;issue=3&amp;publication_year=2012&amp;pages=431-441&amp;pmid=22270873&amp;doi=10.1007/s13361-011-0311-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0036">
<cite>
Koshland DE Jr, Némethy G, Filmer D. Comparison of experimental binding data and theoretical models in proteins containing subunits. Biochemistry. 1966;5(1):365–385.
</cite> [<a href="https://doi.org/10.1021/bi00865a047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5938952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Comparison%20of%20experimental%20binding%20data%20and%20theoretical%20models%20in%20proteins%20containing%20subunits&amp;author=DE%20Koshland&amp;author=G%20N%C3%A9methy&amp;author=D%20Filmer&amp;volume=5&amp;issue=1&amp;publication_year=1966&amp;pages=365-385&amp;pmid=5938952&amp;doi=10.1021/bi00865a047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0037">
<cite>
Kuprowski MC, Boys BL, Konermann L. Analysis of protein mixtures by electrospray mass spectrometry: effects of conformation and desolvation behavior on the signal intensities of hemoglobin subunits. J Am Soc Mass Spectrom. 2007;18(7):1279–1285.
</cite> [<a href="https://doi.org/10.1016/j.jasms.2007.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17500007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Soc%20Mass%20Spectrom&amp;title=Analysis%20of%20protein%20mixtures%20by%20electrospray%20mass%20spectrometry:%20effects%20of%20conformation%20and%20desolvation%20behavior%20on%20the%20signal%20intensities%20of%20hemoglobin%20subunits&amp;author=MC%20Kuprowski&amp;author=BL%20Boys&amp;author=L%20Konermann&amp;volume=18&amp;issue=7&amp;publication_year=2007&amp;pages=1279-1285&amp;pmid=17500007&amp;doi=10.1016/j.jasms.2007.04.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0038">
<cite>
Li C, Qi Y, Teng X, Yang Z, Wei P, Zhang C, et al. Maturation mechanism of severe acute respiratory syndrome (SARS) coronavirus 3C‐like proteinase. J Biol Chem. 2010;285(36):28134–28140.
</cite> [<a href="https://doi.org/10.1074/jbc.M109.095851" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2934678/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20489209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Maturation%20mechanism%20of%20severe%20acute%20respiratory%20syndrome%20(SARS)%20coronavirus%203C%E2%80%90like%20proteinase&amp;author=C%20Li&amp;author=Y%20Qi&amp;author=X%20Teng&amp;author=Z%20Yang&amp;author=P%20Wei&amp;volume=285&amp;issue=36&amp;publication_year=2010&amp;pages=28134-28140&amp;pmid=20489209&amp;doi=10.1074/jbc.M109.095851&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0039">
<cite>
Li C, Teng X, Qi Y, Tang B, Shi H, Ma X, et al. Conformational flexibility of a short loop near the active site of the SARS‐3CL<sup>pro</sup> is essential to maintain catalytic activity. Sci Rep. 2016;6(1):20918.
</cite> [<a href="https://doi.org/10.1038/srep20918" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4754693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26879383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Conformational%20flexibility%20of%20a%20short%20loop%20near%20the%20active%20site%20of%20the%20SARS%E2%80%903CLpro%20is%20essential%20to%20maintain%20catalytic%20activity&amp;author=C%20Li&amp;author=X%20Teng&amp;author=Y%20Qi&amp;author=B%20Tang&amp;author=H%20Shi&amp;volume=6&amp;issue=1&amp;publication_year=2016&amp;pages=20918&amp;pmid=26879383&amp;doi=10.1038/srep20918&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0040">
<cite>
Louis JM, Zhang Y, Sayer JM, Wang Y‐F, Harrison RW, Weber IT. The L76V drug resistance mutation decreases the dimer stability and rate of autoprocessing of HIV‐1 protease by reducing internal hydrophobic contacts. Biochemistry. 2011;50(21):4786–4795.
</cite> [<a href="https://doi.org/10.1021/bi200033z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3101314/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21446746/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=The%20L76V%20drug%20resistance%20mutation%20decreases%20the%20dimer%20stability%20and%20rate%20of%20autoprocessing%20of%20HIV%E2%80%901%20protease%20by%20reducing%20internal%20hydrophobic%20contacts&amp;author=JM%20Louis&amp;author=Y%20Zhang&amp;author=JM%20Sayer&amp;author=Y%E2%80%90F%20Wang&amp;author=RW%20Harrison&amp;volume=50&amp;issue=21&amp;publication_year=2011&amp;pages=4786-4795&amp;pmid=21446746&amp;doi=10.1021/bi200033z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0041">
<cite>
Macnaughton MR. Occurrence and frequency of coronavirus infections in humans as determined by enzyme‐linked immunosorbent assay. Infect Immun. 1982;38(2):419–423.
</cite> [<a href="https://doi.org/10.1128/iai.38.2.419-423.1982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC347755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6292101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&amp;title=Occurrence%20and%20frequency%20of%20coronavirus%20infections%20in%20humans%20as%20determined%20by%20enzyme%E2%80%90linked%20immunosorbent%20assay&amp;author=MR%20Macnaughton&amp;volume=38&amp;issue=2&amp;publication_year=1982&amp;pages=419-423&amp;pmid=6292101&amp;doi=10.1128/iai.38.2.419-423.1982&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0042">
<cite>
Mesel‐Lemoine M, Millet J, Vidalain P‐O, Law H, Vabret A, Lorin V, et al. A human coronavirus responsible for the common cold massively kills dendritic cells but not monocytes. J Virol. 2012;86(14):7577–7587.
</cite> [<a href="https://doi.org/10.1128/JVI.00269-12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3416289/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22553325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Virol&amp;title=A%20human%20coronavirus%20responsible%20for%20the%20common%20cold%20massively%20kills%20dendritic%20cells%20but%20not%20monocytes&amp;author=M%20Mesel%E2%80%90Lemoine&amp;author=J%20Millet&amp;author=P%E2%80%90O%20Vidalain&amp;author=H%20Law&amp;author=A%20Vabret&amp;volume=86&amp;issue=14&amp;publication_year=2012&amp;pages=7577-7587&amp;pmid=22553325&amp;doi=10.1128/JVI.00269-12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0043">
<cite>
Mody V, Ho J, Wills S, Mawri A, Lawson L, Ebert MCCJC, et al. Identification of 3‐chymotrypsin like protease (3CL<sup>pro</sup>) inhibitors as potential anti‐SARS‐CoV‐2 agents. Commun Biol. 2021;4(1):93.
</cite> [<a href="https://doi.org/10.1038/s42003-020-01577-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7817688/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33473151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Commun%20Biol&amp;title=Identification%20of%203%E2%80%90chymotrypsin%20like%20protease%20(3CLpro)%20inhibitors%20as%20potential%20anti%E2%80%90SARS%E2%80%90CoV%E2%80%902%20agents&amp;author=V%20Mody&amp;author=J%20Ho&amp;author=S%20Wills&amp;author=A%20Mawri&amp;author=L%20Lawson&amp;volume=4&amp;issue=1&amp;publication_year=2021&amp;pages=93&amp;pmid=33473151&amp;doi=10.1038/s42003-020-01577-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0044">
<cite>
Moghadasi SA, Heilmann E, Khalil AM, Nnabuife C, Kearns FL, Ye C, et al. Transmissible SARS‐CoV‐2 variants with resistance to clinical protease inhibitors. Sci Adv. 2023;9(13):eade8778.
</cite> [<a href="https://doi.org/10.1126/sciadv.ade8778" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10058310/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36989354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Adv&amp;title=Transmissible%20SARS%E2%80%90CoV%E2%80%902%20variants%20with%20resistance%20to%20clinical%20protease%20inhibitors&amp;author=SA%20Moghadasi&amp;author=E%20Heilmann&amp;author=AM%20Khalil&amp;author=C%20Nnabuife&amp;author=FL%20Kearns&amp;volume=9&amp;issue=13&amp;publication_year=2023&amp;pages=eade8778&amp;pmid=36989354&amp;doi=10.1126/sciadv.ade8778&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0045">
<cite>
Monod J, Wyman J, Changeux J‐P. On the nature of allosteric transitions: a plausible model. J Mol Biol. 1965;12(1):88–118.
</cite> [<a href="https://doi.org/10.1016/s0022-2836(65)80285-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14343300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=On%20the%20nature%20of%20allosteric%20transitions:%20a%20plausible%20model&amp;author=J%20Monod&amp;author=J%20Wyman&amp;author=J%E2%80%90P%20Changeux&amp;volume=12&amp;issue=1&amp;publication_year=1965&amp;pages=88-118&amp;pmid=14343300&amp;doi=10.1016/s0022-2836(65)80285-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0046">
<cite>
Morrison JF. Kinetics of the reversible inhibition of enzyme‐catalysed reactions by tight‐binding inhibitors. Biochim Biophys Acta (BBA) – Enzymol. 1969;185(2):269–286.</cite> [<a href="https://doi.org/10.1016/0005-2744(69)90420-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4980133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta%20(BBA)%20%E2%80%93%20Enzymol&amp;title=Kinetics%20of%20the%20reversible%20inhibition%20of%20enzyme%E2%80%90catalysed%20reactions%20by%20tight%E2%80%90binding%20inhibitors&amp;author=JF%20Morrison&amp;volume=185&amp;issue=2&amp;publication_year=1969&amp;pages=269-286&amp;pmid=4980133&amp;doi=10.1016/0005-2744(69)90420-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0047">
<cite>
Myint SH. Human coronavirus infections. Boston, MA: Springer US; 1995. p. 389–401.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Human%20coronavirus%20infections&amp;author=SH%20Myint&amp;publication_year=1995&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0048">
<cite>
Nashed NT, Aniana A, Ghirlando R, Chiliveri SC, Louis JM. Modulation of the monomer‐dimer equilibrium and catalytic activity of SARS‐CoV‐2 main protease by a transition‐state analog inhibitor. Commun Biol. 2022;5(1):160.
</cite> [<a href="https://doi.org/10.1038/s42003-022-03084-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8888643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35233052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Commun%20Biol&amp;title=Modulation%20of%20the%20monomer%E2%80%90dimer%20equilibrium%20and%20catalytic%20activity%20of%20SARS%E2%80%90CoV%E2%80%902%20main%20protease%20by%20a%20transition%E2%80%90state%20analog%20inhibitor&amp;author=NT%20Nashed&amp;author=A%20Aniana&amp;author=R%20Ghirlando&amp;author=SC%20Chiliveri&amp;author=JM%20Louis&amp;volume=5&amp;issue=1&amp;publication_year=2022&amp;pages=160&amp;pmid=35233052&amp;doi=10.1038/s42003-022-03084-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0049">
<cite>
Noske GD, de Souza Silva E, Godoy MO, Dolci I, Fernandes RS, Guido RVC, et al. Structural basis of nirmatrelvir and ensitrelvir activity against naturally occurring polymorphisms of the SARS‐CoV‐2 main protease. J Biol Chem. 2023;299(3):103004.
</cite> [<a href="https://doi.org/10.1016/j.jbc.2023.103004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9916189/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36775130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Structural%20basis%20of%20nirmatrelvir%20and%20ensitrelvir%20activity%20against%20naturally%20occurring%20polymorphisms%20of%20the%20SARS%E2%80%90CoV%E2%80%902%20main%20protease&amp;author=GD%20Noske&amp;author=E%20de%20Souza%20Silva&amp;author=MO%20Godoy&amp;author=I%20Dolci&amp;author=RS%20Fernandes&amp;volume=299&amp;issue=3&amp;publication_year=2023&amp;pages=103004&amp;pmid=36775130&amp;doi=10.1016/j.jbc.2023.103004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0050">
<cite>
Osorio MI, Yáñez O, Gallardo M, Zuñiga‐Bustos M, Mulia‐Rodríguez J, López‐Rendón R, et al. Search for novel potent inhibitors of the SARS‐CoV‐2 papain‐like enzyme: a computational biochemistry approach. Pharmaceuticals. 2022;15(8):986.
</cite> [<a href="https://doi.org/10.3390/ph15080986" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9414997/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36015134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmaceuticals&amp;title=Search%20for%20novel%20potent%20inhibitors%20of%20the%20SARS%E2%80%90CoV%E2%80%902%20papain%E2%80%90like%20enzyme:%20a%20computational%20biochemistry%20approach&amp;author=MI%20Osorio&amp;author=O%20Y%C3%A1%C3%B1ez&amp;author=M%20Gallardo&amp;author=M%20Zu%C3%B1iga%E2%80%90Bustos&amp;author=J%20Mulia%E2%80%90Rodr%C3%ADguez&amp;volume=15&amp;issue=8&amp;publication_year=2022&amp;pages=986&amp;pmid=36015134&amp;doi=10.3390/ph15080986&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0051">
<cite>
Paciaroni A, Libera V, Ripanti F, Orecchini A, Petrillo C, Francisci D, et al. Stabilization of the dimeric state of SARS‐CoV‐2 main protease by GC376 and nirmatrelvir. Int J Mol Sci. 2023;24(7):6062.
</cite> [<a href="https://doi.org/10.3390/ijms24076062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10093836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37047038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Stabilization%20of%20the%20dimeric%20state%20of%20SARS%E2%80%90CoV%E2%80%902%20main%20protease%20by%20GC376%20and%20nirmatrelvir&amp;author=A%20Paciaroni&amp;author=V%20Libera&amp;author=F%20Ripanti&amp;author=A%20Orecchini&amp;author=C%20Petrillo&amp;volume=24&amp;issue=7&amp;publication_year=2023&amp;pages=6062&amp;pmid=37047038&amp;doi=10.3390/ijms24076062&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0052">
<cite>
Patel D, Kopec J, Fitzpatrick F, McCorvie TJ, Yue WW. Structural basis for ligand‐dependent dimerization of phenylalanine hydroxylase regulatory domain. Sci Rep. 2016;6(1):23748.
</cite> [<a href="https://doi.org/10.1038/srep23748" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4822156/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27049649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Structural%20basis%20for%20ligand%E2%80%90dependent%20dimerization%20of%20phenylalanine%20hydroxylase%20regulatory%20domain&amp;author=D%20Patel&amp;author=J%20Kopec&amp;author=F%20Fitzpatrick&amp;author=TJ%20McCorvie&amp;author=WW%20Yue&amp;volume=6&amp;issue=1&amp;publication_year=2016&amp;pages=23748&amp;pmid=27049649&amp;doi=10.1038/srep23748&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0053">
<cite>
Peiris J, Lai S, Poon L, Guan Y, Yam L, Lim W, et al. Coronavirus as a possible cause of severe acute respiratory syndrome. Lancet. 2003;361(9366):1319–1325.
</cite> [<a href="https://doi.org/10.1016/S0140-6736(03)13077-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7112372/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12711465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Coronavirus%20as%20a%20possible%20cause%20of%20severe%20acute%20respiratory%20syndrome&amp;author=J%20Peiris&amp;author=S%20Lai&amp;author=L%20Poon&amp;author=Y%20Guan&amp;author=L%20Yam&amp;volume=361&amp;issue=9366&amp;publication_year=2003&amp;pages=1319-1325&amp;pmid=12711465&amp;doi=10.1016/S0140-6736(03)13077-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0054">
<cite>
Purawat S, Ieong PU, Malmstrom RD, Chan GJ, Yeung AK, Walker RC, et al. A Kepler workflow tool for reproducible AMBER GPU molecular dynamics. Biophys J. 2017;112(12):2469–2474.
</cite> [<a href="https://doi.org/10.1016/j.bpj.2017.04.055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5479118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28636905/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=A%20Kepler%20workflow%20tool%20for%20reproducible%20AMBER%20GPU%20molecular%20dynamics&amp;author=S%20Purawat&amp;author=PU%20Ieong&amp;author=RD%20Malmstrom&amp;author=GJ%20Chan&amp;author=AK%20Yeung&amp;volume=112&amp;issue=12&amp;publication_year=2017&amp;pages=2469-2474&amp;pmid=28636905&amp;doi=10.1016/j.bpj.2017.04.055&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0055">
<cite>
Rai DK, Yurgelonis I, McMonagle P, Rothan HA, Hao L, Gribenko A, et al. Nirmatrelvir, an orally active Mpro inhibitor, is a potent inhibitor of SARS‐CoV‐2 variants of concern. bioRxiv [Preprint]. 2022. 10.1101/2022.01.17.476644.</cite> [<a href="https://doi.org/10.1101/2022.01.17.476644" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="pro70275-bib-0056">
<cite>
Ramos‐Guzmán CA, Andjelkovic M, Zinovjev K, Ruiz‐Pernía JJ, Tuñón I. The impact of SARS‐CoV‐2 3CL protease mutations on nirmatrelvir inhibitory efficiency. Computational insights into potential resistance mechanisms. Chem Sci. 2023;14(10):2686–2697.
</cite> [<a href="https://doi.org/10.1039/d2sc06584c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9993853/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36908962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20Sci&amp;title=The%20impact%20of%20SARS%E2%80%90CoV%E2%80%902%203CL%20protease%20mutations%20on%20nirmatrelvir%20inhibitory%20efficiency.%20Computational%20insights%20into%20potential%20resistance%20mechanisms&amp;author=CA%20Ramos%E2%80%90Guzm%C3%A1n&amp;author=M%20Andjelkovic&amp;author=K%20Zinovjev&amp;author=JJ%20Ruiz%E2%80%90Pern%C3%ADa&amp;author=I%20Tu%C3%B1%C3%B3n&amp;volume=14&amp;issue=10&amp;publication_year=2023&amp;pages=2686-2697&amp;pmid=36908962&amp;doi=10.1039/d2sc06584c&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0057">
<cite>
Salomon‐Ferrer R, Case DA, Walker RC. An overview of the Amber biomolecular simulation package. WIREs Comput Mol Sci. 2013;3(2):198–210.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=WIREs%20Comput%20Mol%20Sci&amp;title=An%20overview%20of%20the%20Amber%20biomolecular%20simulation%20package&amp;author=R%20Salomon%E2%80%90Ferrer&amp;author=DA%20Case&amp;author=RC%20Walker&amp;volume=3&amp;issue=2&amp;publication_year=2013&amp;pages=198-210&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0058">
<cite>
Sasi VM, Ullrich S, Ton J, Fry SE, Johansen‐Leete J, Payne RJ, et al. Predicting antiviral resistance mutations in SARS‐CoV‐2 main protease with computational and experimental screening. Biochemistry. 2022;61(22):2495–2505.
</cite> [<a href="https://doi.org/10.1021/acs.biochem.2c00489" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36326185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Predicting%20antiviral%20resistance%20mutations%20in%20SARS%E2%80%90CoV%E2%80%902%20main%20protease%20with%20computational%20and%20experimental%20screening&amp;author=VM%20Sasi&amp;author=S%20Ullrich&amp;author=J%20Ton&amp;author=SE%20Fry&amp;author=J%20Johansen%E2%80%90Leete&amp;volume=61&amp;issue=22&amp;publication_year=2022&amp;pages=2495-2505&amp;pmid=36326185&amp;doi=10.1021/acs.biochem.2c00489&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0059">
<cite>
Schubert E, Sander J, Ester M, Kriegel HP, Xu X. DBSCAN revisited, revisited: why and how you should (still) use DBSCAN. ACM Trans Database Syst. 2017;42(3):19.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=ACM%20Trans%20Database%20Syst&amp;title=DBSCAN%20revisited,%20revisited:%20why%20and%20how%20you%20should%20(still)%20use%20DBSCAN&amp;author=E%20Schubert&amp;author=J%20Sander&amp;author=M%20Ester&amp;author=HP%20Kriegel&amp;author=X%20Xu&amp;volume=42&amp;issue=3&amp;publication_year=2017&amp;pages=19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0060">
<cite>
Shi J, Sivaraman J, Song J. Mechanism for controlling the dimer‐monomer switch and coupling dimerization to catalysis of the severe acute respiratory syndrome coronavirus 3C‐like protease. J Virol. 2008;82(9):4620–4629.
</cite> [<a href="https://doi.org/10.1128/JVI.02680-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2293028/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18305031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Virol&amp;title=Mechanism%20for%20controlling%20the%20dimer%E2%80%90monomer%20switch%20and%20coupling%20dimerization%20to%20catalysis%20of%20the%20severe%20acute%20respiratory%20syndrome%20coronavirus%203C%E2%80%90like%20protease&amp;author=J%20Shi&amp;author=J%20Sivaraman&amp;author=J%20Song&amp;volume=82&amp;issue=9&amp;publication_year=2008&amp;pages=4620-4629&amp;pmid=18305031&amp;doi=10.1128/JVI.02680-07&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0061">
<cite>
Silvestrini L, Belhaj N, Comez L, Gerelli Y, Lauria A, Libera V, et al. The dimer‐monomer equilibrium of SARS‐CoV‐2 main protease is affected by small molecule inhibitors. Sci Rep. 2021;11(1):9283.
</cite> [<a href="https://doi.org/10.1038/s41598-021-88630-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8085067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33927258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=The%20dimer%E2%80%90monomer%20equilibrium%20of%20SARS%E2%80%90CoV%E2%80%902%20main%20protease%20is%20affected%20by%20small%20molecule%20inhibitors&amp;author=L%20Silvestrini&amp;author=N%20Belhaj&amp;author=L%20Comez&amp;author=Y%20Gerelli&amp;author=A%20Lauria&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=9283&amp;pmid=33927258&amp;doi=10.1038/s41598-021-88630-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0062">
<cite>
Stille JK, Tjutrins J, Wang G, Venegas FA, Hennecker C, Rueda AM, et al. Design, synthesis and in vitro evaluation of novel SARS‐CoV‐2 3CL<sup>pro</sup> covalent inhibitors. Eur J Med Chem. 2022;229:114046.
</cite> [<a href="https://doi.org/10.1016/j.ejmech.2021.114046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8665847/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34995923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Med%20Chem&amp;title=Design,%20synthesis%20and%20in%20vitro%20evaluation%20of%20novel%20SARS%E2%80%90CoV%E2%80%902%203CLpro%20covalent%20inhibitors&amp;author=JK%20Stille&amp;author=J%20Tjutrins&amp;author=G%20Wang&amp;author=FA%20Venegas&amp;author=C%20Hennecker&amp;volume=229&amp;publication_year=2022&amp;pages=114046&amp;pmid=34995923&amp;doi=10.1016/j.ejmech.2021.114046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0063">
<cite>
Thiel V, Ivanov KA, Putics Á, Hertzig T, Schelle B, Bayer S, et al. Mechanisms and enzymes involved in SARS coronavirus genome expression. J Gen Virol. 2003;84(9):2305–2315.
</cite> [<a href="https://doi.org/10.1099/vir.0.19424-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12917450/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Gen%20Virol&amp;title=Mechanisms%20and%20enzymes%20involved%20in%20SARS%20coronavirus%20genome%20expression&amp;author=V%20Thiel&amp;author=KA%20Ivanov&amp;author=%C3%81%20Putics&amp;author=T%20Hertzig&amp;author=B%20Schelle&amp;volume=84&amp;issue=9&amp;publication_year=2003&amp;pages=2305-2315&amp;pmid=12917450&amp;doi=10.1099/vir.0.19424-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0064">
<cite>
Tomar S, Johnston ML, St. John SE, Osswald HL, Nyalapatla PR, Paul LN, et al. Ligand‐induced dimerization of middle east respiratory syndrome (MERS) coronavirus nsp5 protease (3CL<sup>pro</sup>): implications for nsp5 regulation and the development of antivirals. J Biol Chem. 2015;290(32):19403–19422.
</cite> [<a href="https://doi.org/10.1074/jbc.M115.651463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4528106/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26055715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Ligand%E2%80%90induced%20dimerization%20of%20middle%20east%20respiratory%20syndrome%20(MERS)%20coronavirus%20nsp5%20protease%20(3CLpro):%20implications%20for%20nsp5%20regulation%20and%20the%20development%20of%20antivirals&amp;author=S%20Tomar&amp;author=ML%20Johnston&amp;author=SE%20St.%20John&amp;author=HL%20Osswald&amp;author=PR%20Nyalapatla&amp;volume=290&amp;issue=32&amp;publication_year=2015&amp;pages=19403-19422&amp;pmid=26055715&amp;doi=10.1074/jbc.M115.651463&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0065">
<cite>
Tyrrell DAJ, Bynoe ML. Cultivation of a novel type of common‐cold virus in organ cultures. Br Med J. 1965;1(5448):1467–1470.
</cite> [<a href="https://doi.org/10.1136/bmj.1.5448.1467" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2166670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14288084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br%20Med%20J&amp;title=Cultivation%20of%20a%20novel%20type%20of%20common%E2%80%90cold%20virus%20in%20organ%20cultures&amp;author=DAJ%20Tyrrell&amp;author=ML%20Bynoe&amp;volume=1&amp;issue=5448&amp;publication_year=1965&amp;pages=1467-1470&amp;pmid=14288084&amp;doi=10.1136/bmj.1.5448.1467&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0066"><cite>
U.S. Food &amp; Drug Administration
. Fact sheet for healthcare providers: Emergency use authorization for paxlovid. 2023. <a href="https://www.fda.gov/media/155050/download" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fda.gov/media/155050/download</a>. Accessed 29 Nov 2023.</cite></li>
<li id="pro70275-bib-0067">
<cite>
V'kovski P, Kratzel A, Steiner S, Stalder H, Thiel V. Coronavirus biology and replication: implications for SARS‐CoV‐2. Nat Rev Microbiol. 2021;19(3):155–170.
</cite> [<a href="https://doi.org/10.1038/s41579-020-00468-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7592455/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33116300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Microbiol&amp;title=Coronavirus%20biology%20and%20replication:%20implications%20for%20SARS%E2%80%90CoV%E2%80%902&amp;author=P%20V'kovski&amp;author=A%20Kratzel&amp;author=S%20Steiner&amp;author=H%20Stalder&amp;author=V%20Thiel&amp;volume=19&amp;issue=3&amp;publication_year=2021&amp;pages=155-170&amp;pmid=33116300&amp;doi=10.1038/s41579-020-00468-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0068">
<cite>
Vries M, Mohamed AS, Prescott RA, Valero‐Jimenez AM, Desvignes L, O'Connor R, et al. A comparative analysis of SARS‐CoV‐2 antivirals characterizes 3CL<sup>pro</sup> inhibitor pf‐00835231 as a potential new treatment for COVID‐19. J Virol. 2021;95(10):e01819.
</cite> [<a href="https://doi.org/10.1128/JVI.01819-20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8139662/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33622961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Virol&amp;title=A%20comparative%20analysis%20of%20SARS%E2%80%90CoV%E2%80%902%20antivirals%20characterizes%203CLpro%20inhibitor%20pf%E2%80%9000835231%20as%20a%20potential%20new%20treatment%20for%20COVID%E2%80%9019&amp;author=M%20Vries&amp;author=AS%20Mohamed&amp;author=RA%20Prescott&amp;author=AM%20Valero%E2%80%90Jimenez&amp;author=L%20Desvignes&amp;volume=95&amp;issue=10&amp;publication_year=2021&amp;pages=e01819&amp;pmid=33622961&amp;doi=10.1128/JVI.01819-20&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0069">
<cite>
Wang G, Venegas FA, Rueda AM, Weerasinghe NW, Uggowitzer KA, Thibodeaux CJ, et al. A naturally occurring G11S mutation in the 3C‐like protease from the SARS‐CoV‐2 virus dramatically weakens the dimer interface. Protein Sci. 2024;33(1):e4857.
</cite> [<a href="https://doi.org/10.1002/pro.4857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10731504/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38058248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=A%20naturally%20occurring%20G11S%20mutation%20in%20the%203C%E2%80%90like%20protease%20from%20the%20SARS%E2%80%90CoV%E2%80%902%20virus%20dramatically%20weakens%20the%20dimer%20interface&amp;author=G%20Wang&amp;author=FA%20Venegas&amp;author=AM%20Rueda&amp;author=NW%20Weerasinghe&amp;author=KA%20Uggowitzer&amp;volume=33&amp;issue=1&amp;publication_year=2024&amp;pages=e4857&amp;pmid=38058248&amp;doi=10.1002/pro.4857&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0070"><cite>
World Health Organization
. Who coronavirus (covid‐19) dashboard. 2024. <a href="https://covid19.who.int/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://covid19.who.int/</a>. Accessed 3 Sep 2024</cite></li>
<li id="pro70275-bib-0071">
<cite>
Wralstad EC, Sayers J, Raines RT. Bayesian inference elucidates the catalytic competency of the SARS‐CoV‐2 main protease 3CL<sup>pro</sup>
. Anal Chem. 2023;95(40):14981–14989.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.3c02459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10662973/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37750823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal%20Chem&amp;title=Bayesian%20inference%20elucidates%20the%20catalytic%20competency%20of%20the%20SARS%E2%80%90CoV%E2%80%902%20main%20protease%203CLpro&amp;author=EC%20Wralstad&amp;author=J%20Sayers&amp;author=RT%20Raines&amp;volume=95&amp;issue=40&amp;publication_year=2023&amp;pages=14981-14989&amp;pmid=37750823&amp;doi=10.1021/acs.analchem.3c02459&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0072">
<cite>
Wu CG, Cheng SC, Chen SC, Li JY, Fang YH, Chen YH, et al. Mechanism for controlling the monomer‐dimer conversion of SARS coronavirus main protease. Acta Crystallogr D Biol Crystallogr. 2013;69(Pt 5):747–755.
</cite> [<a href="https://doi.org/10.1107/S0907444913001315" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7161611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23633583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Biol%20Crystallogr&amp;title=Mechanism%20for%20controlling%20the%20monomer%E2%80%90dimer%20conversion%20of%20SARS%20coronavirus%20main%20protease&amp;author=CG%20Wu&amp;author=SC%20Cheng&amp;author=SC%20Chen&amp;author=JY%20Li&amp;author=YH%20Fang&amp;volume=69&amp;issue=Pt%205&amp;publication_year=2013&amp;pages=747-755&amp;pmid=23633583&amp;doi=10.1107/S0907444913001315&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0073">
<cite>
Xie D, Gulnik S, Gustchina E, Yu B, Shao W, Qoronfleh W, et al. Drug resistance mutations can affect dimer stability of HIV‐1 protease at neutral pH. Protein Sci. 1999;8(8):1702–1707.
</cite> [<a href="https://doi.org/10.1110/ps.8.8.1702" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2144419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10452615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Drug%20resistance%20mutations%20can%20affect%20dimer%20stability%20of%20HIV%E2%80%901%20protease%20at%20neutral%20pH&amp;author=D%20Xie&amp;author=S%20Gulnik&amp;author=E%20Gustchina&amp;author=B%20Yu&amp;author=W%20Shao&amp;volume=8&amp;issue=8&amp;publication_year=1999&amp;pages=1702-1707&amp;pmid=10452615&amp;doi=10.1110/ps.8.8.1702&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0074">
<cite>
Xue X, Yang H, Shen W, Zhao Q, Li J, Yang K, et al. Production of authentic SARS‐CoV M<sup>pro</sup> with enhanced activity: application as a novel tag‐cleavage endopeptidase for protein overproduction. J Mol Biol. 2007;366(3):965–975.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2006.11.073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7094453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17189639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Production%20of%20authentic%20SARS%E2%80%90CoV%20Mpro%20with%20enhanced%20activity:%20application%20as%20a%20novel%20tag%E2%80%90cleavage%20endopeptidase%20for%20protein%20overproduction&amp;author=X%20Xue&amp;author=H%20Yang&amp;author=W%20Shen&amp;author=Q%20Zhao&amp;author=J%20Li&amp;volume=366&amp;issue=3&amp;publication_year=2007&amp;pages=965-975&amp;pmid=17189639&amp;doi=10.1016/j.jmb.2006.11.073&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0075">
<cite>
Xue X, Yu H, Yang H, Xue F, Wu Z, Shen W, et al. Structures of two coronavirus main proteases: implications for substrate binding and antiviral drug design. J Virol. 2008;82(5):2515–2527.
</cite> [<a href="https://doi.org/10.1128/JVI.02114-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2258912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18094151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Virol&amp;title=Structures%20of%20two%20coronavirus%20main%20proteases:%20implications%20for%20substrate%20binding%20and%20antiviral%20drug%20design&amp;author=X%20Xue&amp;author=H%20Yu&amp;author=H%20Yang&amp;author=F%20Xue&amp;author=Z%20Wu&amp;volume=82&amp;issue=5&amp;publication_year=2008&amp;pages=2515-2527&amp;pmid=18094151&amp;doi=10.1128/JVI.02114-07&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0076">
<cite>
Yañez O, Osorio MI, Areche C, Vasquez‐Espinal A, Bravo J, Sandoval‐Aldana A, et al. 
<em>Theobroma cacao</em> l. compounds: theoretical study and molecular modeling as inhibitors of main SARS‐CoV‐2 protease. Biomed Pharmacother. 2021;140:111764.
</cite> [<a href="https://doi.org/10.1016/j.biopha.2021.111764" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8141698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34051617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomed%20Pharmacother&amp;title=Theobroma%20cacao%20l.%20compounds:%20theoretical%20study%20and%20molecular%20modeling%20as%20inhibitors%20of%20main%20SARS%E2%80%90CoV%E2%80%902%20protease&amp;author=O%20Ya%C3%B1ez&amp;author=MI%20Osorio&amp;author=C%20Areche&amp;author=A%20Vasquez%E2%80%90Espinal&amp;author=J%20Bravo&amp;volume=140&amp;publication_year=2021&amp;pages=111764&amp;pmid=34051617&amp;doi=10.1016/j.biopha.2021.111764&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0077">
<cite>
Zhang L, Lin D, Sun X, Curth U, Drosten C, Sauerhering L, et al. Crystal structure of SARS‐CoV‐2 main protease provides a basis for design of improved α‐ketoamide inhibitors. Science. 2020;368(6489):409–412.
</cite> [<a href="https://doi.org/10.1126/science.abb3405" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7164518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32198291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Crystal%20structure%20of%20SARS%E2%80%90CoV%E2%80%902%20main%20protease%20provides%20a%20basis%20for%20design%20of%20improved%20%CE%B1%E2%80%90ketoamide%20inhibitors&amp;author=L%20Zhang&amp;author=D%20Lin&amp;author=X%20Sun&amp;author=U%20Curth&amp;author=C%20Drosten&amp;volume=368&amp;issue=6489&amp;publication_year=2020&amp;pages=409-412&amp;pmid=32198291&amp;doi=10.1126/science.abb3405&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0078">
<cite>
Zhang A‐R, Shi W‐Q, Liu K, Li X‐L, Liu M‐J, Zhang W‐H, et al. Epidemiology and evolution of Middle East respiratory syndrome coronavirus, 2012–2020. Infect Dis Poverty. 2021;10(1):66.
</cite> [<a href="https://doi.org/10.1186/s40249-021-00853-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8105704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33964965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Dis%20Poverty&amp;title=Epidemiology%20and%20evolution%20of%20Middle%20East%20respiratory%20syndrome%20coronavirus,%202012%E2%80%932020&amp;author=A%E2%80%90R%20Zhang&amp;author=W%E2%80%90Q%20Shi&amp;author=K%20Liu&amp;author=X%E2%80%90L%20Li&amp;author=M%E2%80%90J%20Liu&amp;volume=10&amp;issue=1&amp;publication_year=2021&amp;pages=66&amp;pmid=33964965&amp;doi=10.1186/s40249-021-00853-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0079">
<cite>
Zhang S, Van Pelt CK, Wilson DB. Quantitative determination of noncovalent binding interactions using automated nanoelectrospray mass spectrometry. Anal Chem. 2003;75(13):3010–3018.
</cite> [<a href="https://doi.org/10.1021/ac034089d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12964745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal%20Chem&amp;title=Quantitative%20determination%20of%20noncovalent%20binding%20interactions%20using%20automated%20nanoelectrospray%20mass%20spectrometry&amp;author=S%20Zhang&amp;author=CK%20Van%20Pelt&amp;author=DB%20Wilson&amp;volume=75&amp;issue=13&amp;publication_year=2003&amp;pages=3010-3018&amp;pmid=12964745&amp;doi=10.1021/ac034089d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0080">
<cite>
Zhang S, Zhong N, Xue F, Kang X, Ren X, Chen J, et al. Three‐dimensional domain swapping as a mechanism to lock the active conformation in a super‐active octamer of SARS‐CoV main protease. Protein Cell. 2010;1(4):371–383.
</cite> [<a href="https://doi.org/10.1007/s13238-010-0044-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4875095/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21203949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Cell&amp;title=Three%E2%80%90dimensional%20domain%20swapping%20as%20a%20mechanism%20to%20lock%20the%20active%20conformation%20in%20a%20super%E2%80%90active%20octamer%20of%20SARS%E2%80%90CoV%20main%20protease&amp;author=S%20Zhang&amp;author=N%20Zhong&amp;author=F%20Xue&amp;author=X%20Kang&amp;author=X%20Ren&amp;volume=1&amp;issue=4&amp;publication_year=2010&amp;pages=371-383&amp;pmid=21203949&amp;doi=10.1007/s13238-010-0044-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0081">
<cite>
Zhou Y, Gammeltoft KA, Ryberg LA, Pham LV, Tjørnelund HD, Binderup A, et al. Nirmatrelvir‐resistant SARS‐CoV‐2 variants with high fitness in an infectious cell culture system. Sci Adv. 2022;8(51):eadd7197.
</cite> [<a href="https://doi.org/10.1126/sciadv.add7197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9770952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36542720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Adv&amp;title=Nirmatrelvir%E2%80%90resistant%20SARS%E2%80%90CoV%E2%80%902%20variants%20with%20high%20fitness%20in%20an%20infectious%20cell%20culture%20system&amp;author=Y%20Zhou&amp;author=KA%20Gammeltoft&amp;author=LA%20Ryberg&amp;author=LV%20Pham&amp;author=HD%20Tj%C3%B8rnelund&amp;volume=8&amp;issue=51&amp;publication_year=2022&amp;pages=eadd7197&amp;pmid=36542720&amp;doi=10.1126/sciadv.add7197&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70275-bib-0082">
<cite>
Ziebuhr J, Snijder EJ, Gorbalenya AE. Virus‐encoded proteinases and proteolytic processing in the nidovirales. J Gen Virol. 2000;81(4):853–879.
</cite> [<a href="https://doi.org/10.1099/0022-1317-81-4-853" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10725411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Gen%20Virol&amp;title=Virus%E2%80%90encoded%20proteinases%20and%20proteolytic%20processing%20in%20the%20nidovirales&amp;author=J%20Ziebuhr&amp;author=EJ%20Snijder&amp;author=AE%20Gorbalenya&amp;volume=81&amp;issue=4&amp;publication_year=2000&amp;pages=853-879&amp;pmid=10725411&amp;doi=10.1099/0022-1317-81-4-853&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Data S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375981/bin/PRO-34-e70275-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70275-s001.docx</a><sup> (14.3MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that supports the findings of this study are available in the supplementary material of this article.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Protein Science : A Publication of the Protein Society are provided here courtesy of <strong>The Protein Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/pro.70275"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/PRO-34-e70275.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375981/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375981/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375981%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375981/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375981/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375981/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375981/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375981/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375981/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="I0QZnrj4SIL7SxekfYm92OdeYy9Cf1Eq8HipHPj2tkvdfkTk4WkVA8Ed5H67kWfZ">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
