
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Structural basis for the recognition and ubiquitylation of type-2 N-degron substrate by PRT1 plant N-recognin - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A5778AF2305305A5770043F92BAF.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370974/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Structural basis for the recognition and ubiquitylation of type-2 N-degron substrate by PRT1 plant N-recognin">
<meta name="citation_author" content="Woo Seok Yang">
<meta name="citation_author_institution" content="Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea">
<meta name="citation_author" content="Seu Ha Kim">
<meta name="citation_author_institution" content="Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea">
<meta name="citation_author" content="Minsang Kim">
<meta name="citation_author_institution" content="Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea">
<meta name="citation_author" content="Hejeong Shin">
<meta name="citation_author_institution" content="Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea">
<meta name="citation_author" content="Juyeon Lee">
<meta name="citation_author_institution" content="Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea">
<meta name="citation_author" content="Alexander Sandmann">
<meta name="citation_author_institution" content="Department of Plant Physiology and Protein Metabolism Laboratory, University of Osnabruck, Osnabruck, Germany">
<meta name="citation_author" content="Ohkmae K Park">
<meta name="citation_author_institution" content="Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea">
<meta name="citation_author" content="Nico Dissmeyer">
<meta name="citation_author_institution" content="Department of Plant Physiology and Protein Metabolism Laboratory, University of Osnabruck, Osnabruck, Germany">
<meta name="citation_author" content="Hyun Kyu Song">
<meta name="citation_author_institution" content="Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7817">
<meta name="citation_doi" content="10.1038/s41467-025-63282-9">
<meta name="citation_pmid" content="40841552">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370974/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370974/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370974/pdf/41467_2025_Article_63282.pdf">
<meta name="description" content="PROTEOLYSIS1 (PRT1), an N-recognin of Arabidopsis thaliana, recognizes the N-terminal aromatic hydrophobic residue (Tyr/Phe/Trp) of its substrates and ubiquitylates them for degradation by the ubiquitin-proteasome system. Herein, we report the ...">
<meta name="og:title" content="Structural basis for the recognition and ubiquitylation of type-2 N-degron substrate by PRT1 plant N-recognin">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="PROTEOLYSIS1 (PRT1), an N-recognin of Arabidopsis thaliana, recognizes the N-terminal aromatic hydrophobic residue (Tyr/Phe/Trp) of its substrates and ubiquitylates them for degradation by the ubiquitin-proteasome system. Herein, we report the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370974/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370974">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63282-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63282.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370974%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370974/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370974/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370974/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7817. doi: <a href="https://doi.org/10.1038/s41467-025-63282-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63282-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Structural basis for the recognition and ubiquitylation of type-2 <em>N</em>-degron substrate by PRT1 plant <em>N</em>-recognin</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20WS%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Woo Seok Yang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Woo Seok Yang</span></h3>
<div class="p">
<sup>1</sup>Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20WS%22%5BAuthor%5D" class="usa-link"><span class="name western">Woo Seok Yang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SH%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Seu Ha Kim</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Seu Ha Kim</span></h3>
<div class="p">
<sup>1</sup>Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SH%22%5BAuthor%5D" class="usa-link"><span class="name western">Seu Ha Kim</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Minsang Kim</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Minsang Kim</span></h3>
<div class="p">
<sup>1</sup>Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Minsang Kim</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shin%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Hejeong Shin</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Hejeong Shin</span></h3>
<div class="p">
<sup>1</sup>Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shin%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hejeong Shin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Juyeon Lee</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Juyeon Lee</span></h3>
<div class="p">
<sup>1</sup>Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Juyeon Lee</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sandmann%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Alexander Sandmann</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Alexander Sandmann</span></h3>
<div class="p">
<sup>2</sup>Department of Plant Physiology and Protein Metabolism Laboratory, University of Osnabruck, Osnabruck, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sandmann%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexander Sandmann</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20OK%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Ohkmae K Park</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Ohkmae K Park</span></h3>
<div class="p">
<sup>1</sup>Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20OK%22%5BAuthor%5D" class="usa-link"><span class="name western">Ohkmae K Park</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dissmeyer%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Nico Dissmeyer</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Nico Dissmeyer</span></h3>
<div class="p">
<sup>2</sup>Department of Plant Physiology and Protein Metabolism Laboratory, University of Osnabruck, Osnabruck, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dissmeyer%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nico Dissmeyer</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20HK%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Hyun Kyu Song</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Hyun Kyu Song</span></h3>
<div class="p">
<sup>1</sup>Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20HK%22%5BAuthor%5D" class="usa-link"><span class="name western">Hyun Kyu Song</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Life Sciences, Korea University, Seongbuk-gu Seoul, South Korea </div>
<div id="Aff2">
<sup>2</sup>Department of Plant Physiology and Protein Metabolism Laboratory, University of Osnabruck, Osnabruck, Germany </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Sep 19; Accepted 2025 Aug 15; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370974  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841552/" class="usa-link">40841552</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">PROTEOLYSIS1 (PRT1), an <em>N</em>-recognin of <em>Arabidopsis thaliana</em>, recognizes the <em>N</em>-terminal aromatic hydrophobic residue (Tyr/Phe/Trp) of its substrates and ubiquitylates them for degradation by the ubiquitin-proteasome system. Herein, we report the structures of the ZZ domain of PRT1 (PRT1<sup>ZZ</sup>) in complex with bulky hydrophobic <em>N</em>-degron peptides. Unlike other ZZ domains, PRT1<sup>ZZ</sup> has an unusual binding site with two hydrophobic regions. The <em>N</em>-terminal aromatic residues of <em>N</em>-degrons interact with Ile333 and Phe352 in the flexible loops, which undergo a conformational change. Notably, we identify a third residue from the <em>N</em>-terminus of the substrate that participates in the hydrophobic network with PRT1<sup>ZZ</sup>. Moreover, AlphaFold prediction and biochemical assays revealed that the tandem RING1 and RING2 domains of PRT1 interact intramolecularly. The dimeric RING domains in a single protein represent a unique feature among the RING-type E3 ligases. The biochemical assays using the <em>N</em>-terminal tyrosine-exposed substrate, BIG BROTHER, show that the intramolecular RING dimer is essential for PRT1’s robust activity. Therefore, this study expands our knowledge of the structural repertoire in the <em>N</em>-degron pathway and provides insights into the regulation of E3 ligases containing tandem RING domains.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> X-ray crystallography, Plant molecular biology, Ubiquitin ligases</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">PRT1 is an E3 ligase that recognizes type-2 hydrophobic aromatic Arg/N-degrons. Here, the authors reveal the structure of the ZZ domain of Arabidopsis PRT1 bound to bulky hydrophobic type-2 N-degrons, showing a unique binding site formed by a substantial conformational change of two loops and uncovering an intramolecular tandem RING dimer essential for its ubiquitylation activity.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">The ubiquitin-proteasome system regulates protein homeostasis in all species, including plants. The polyubiquitin (poly-Ub) chain acts as a degradation marker by delivering ubiquitylated substrates to the proteasome<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. E3 Ub ligases generate a poly-Ub chain coupled with E1 Ub activating and E2 Ub conjugating enzymes<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Through this mechanism, the half-life of various substrates is determined by the characteristics of their <em>N</em>-terminal (Nt) residues, which are specifically recognized by a class of E3 Ub ligases known as “<em>N</em>-recognins”<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. In this case, the Nt residues function as a degradational signal known as “<em>N</em>-degrons”<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Since the first <em>N</em>-recognin Ubr1 was characterized<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>, many <em>N</em>-degron pathways such as those of Arg/<em>N</em>-degron, Ac/<em>N</em>-degron, Pro/<em>N</em>-degron, Gly/<em>N</em>-degron, and fMet/<em>N</em>-degron have been extensively studied<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. The Arg/<em>N</em>-degron pathway was the first to be characterized, targeting at least type-1 (positively charged Nt residues, Arg/Lys/His) and type-2 (hydrophobic Nt residues, Leu/Ile/Phe/Tyr/Trp) substrates<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. The first and best-characterized <em>N</em>-recognin is yeast Ubr1, which consists of a UBR box and a ClpS-homology domain for recognizing type-1 and type-2 substrates, respectively<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>–<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>.</p>
<p id="Par4">Yeast Ubr1 <em>N</em>-recognin shares sequence and functional homology with mammalian UBR1/2 (Ubiquitin Protein Ligase E3 Component <em>N</em>-Recognin 1/2), which also contain the UBR box and ClpS-homology domain<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. However, the closest homolog in plants is the PROTEOLYSIS6 (PRT6), which only has a UBR box<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. No other defined <em>N</em>-recognin ortholog containing the ClpS-homology domain has been identified in plants. However, an E3 ligase known as “PROTEOLYSIS1” (PRT1), which had been isolated from <em>Arabidopsis thaliana</em>, has been defined as an <em>N</em>-recognin recognizing type-2 hydrophobic aromatic Arg/<em>N</em>-degrons<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. BIG BROTHER (BB), a RING-type E3 Ub ligase that controls organ size in <em>A. thaliana</em>, has been identified as a substrate of PRT1<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Although BB controls plant cell proliferation, the downstream regulator remains unknown. BB is negatively regulated by the endoprotease DA1 from <em>A. thaliana</em> (DA1), which cleaves the peptide bond between the 60th Ala residue and the 61st Tyr residue of BB<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. As a consequence, the Tyr residue, a neo-Nt residue in BB, exposes the type-2 <em>N</em>-degron. Notably, PRT1 from <em>A. thaliana</em> (<em>At</em>PRT1) interacts with the cleaved BB (hereafter Tyr61-BB) through the bulky hydrophobic <em>N</em>-degrons and ubiquitylates it<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>.</p>
<p id="Par5">The recognition mechanism of bulky hydrophobic residues by PRT1 is not clearly understood because PRT1 consists of tandem RING domains, a pair of canonical (RING1) and non-canonical (RING2) RING domains, and a C-terminal ZZ domain, instead of the ClpS-homology domain typically associated with type-2 <em>N</em>-degron recognition (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>)<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. As noted above, PRT1 lacks conventional <em>N</em>-degron recognition domains. The ZZ domain was presumably predicted to be the recognition domain of bulky hydrophobic <em>N</em>-degrons, particularly after the ZZ domain of p62/SQSTM1 interacts with type-1 and -2 <em>N</em>-degrons<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Here, we specify the ZZ domain of PRT1 (PRT1<sup>ZZ</sup>) as an <em>N</em>-degron-binding region and describe the high-resolution structures of PRT1<sup>ZZ</sup> in complex with type-2 bulky hydrophobic <em>N</em>-degrons. Biochemical and mutational analyses confirm the key interaction determinants between <em>N</em>-recognin PRT1 and the <em>N</em>-degron substrate Tyr61-BB. We also compare the structure of PRT1<sup>ZZ</sup> with those of other ZZ domain proteins<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Furthermore, we find that the tandem RING organization in PRT1 is critical for E3 ubiquitylation activity, as verified using the <em>N</em>-degron substrate Tyr61-BB. These results reveal how the plant ZZ domain recognizes bulky hydrophobic <em>N</em>-degron substrates and regulates the ubiquitylation activity of the tandem RING structures in PRT1. Hence, this study expands our knowledge of the structural repertoire of the <em>N</em>-degron pathway, particularly within plant systems, and provides insights into the regulation of E3 Ub ligases with tandem RING domains.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">
<em>N</em>-degron recognition by the ZZ domain of <em>At</em>PRT1</h3>
<p id="Par6"><em>At</em>PRT1<sup>ZZ</sup> is responsible for recognizing type-2 bulky hydrophobic <em>N</em>-degrons<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>; however, the binding affinity between PRT1 and <em>N</em>-degrons has not been quantitatively analyzed. First, we measured the dissociation constant (<em>K</em><sub>D</sub>) between PRT1<sup>ZZ</sup> and a Tyr <em>N</em>-degron peptide (Tyr-Lys-Phe) using isothermal titration calorimetry (ITC), yielding a <em>K</em><sub>D</sub> value of 10.1 μM (Supplementary Fig. <a href="#MOESM1" class="usa-link">1b</a>). However, no binding affinity was detected between ZZ domain-deleted RING domains (RING1 + RING2) and the same <em>N</em>-degron peptide (Supplementary Fig. <a href="#MOESM1" class="usa-link">1c</a>). Thus, these results show that type-2 <em>N</em>-degrons are solely recognized by PRT1<sup>ZZ</sup>, with dissociation constants in the micromolecular range, depending on the Nt residue.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Crystal structure of PRT1<sup>ZZ</sup>
</h3>
<p id="Par7">The structure of PRT1<sup>ZZ</sup> (residues 303–366) was determined using the single-wavelength anomalous dispersion (SAD) method at the Zn absorption edge (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). PRT1<sup>ZZ</sup> has a canonical “Zig-Zag fold”<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>, comprising three β-strands and an α-helix (Fig. <a href="#Fig1" class="usa-link">1a–c</a>). The ZZ domain has two conserved Asp residues (Asp312 and Asp336) that belong to the negatively charged patch (Fig. <a href="#Fig1" class="usa-link">1b</a>), which recognizes the α-NH<sub>3</sub><sup>+</sup> group of <em>N</em>-degrons. The overall structure of PRT1<sup>ZZ</sup> has a highly conserved folding pattern that coordinates two Zn<sup>2+</sup>. The first Zn<sup>2+</sup> (Zn1) is coordinated by four Cys residues, and the second Zn<sup>+2</sup> (Zn2) by two Cys and two His residues (Fig. <a href="#Fig1" class="usa-link">1d</a>). In this arrangement, the Zn<sup>2+</sup> coordinating Cys residues form the CxxC motif (“x” represents any amino acid residue). However, the number of amino acid residues between Zn-coordinating His residues differs from that of other ZZ domains<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Notably, PRT1<sup>ZZ</sup> features two long loops, Loop 2 (L2) connecting β1 and β2 strands and Loop 3 (L3) connecting the α1 helix and β3 strand (Fig. <a href="#Fig1" class="usa-link">1a, c</a>), which are characteristic features compared with other ZZ domains. Because of the flexibility of these loops, building models at these positions is challenging. The three protruding residues (Tyr317, Ile333, and Phe352) around the negatively charged patch potentially participate in <em>N</em>-degron recognition. However, these hydrophobic residues are far from each other in the apo structure of the ZZ domain (Fig. <a href="#Fig1" class="usa-link">1a</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Structure of the apo-form of PRT1<sup>ZZ</sup>.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370974_41467_2025_63282_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/044e/12370974/ed6ee109ca00/41467_2025_63282_Fig1_HTML.jpg" loading="lazy" id="d33e677" height="507" width="675" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Structure of the apo-form of PRT1<sup>ZZ</sup> in a ribbon diagram. Three loops (1: red, 2: purple, and 3: green) are indicated and labeled. Key residues for <em>N</em>-degron recognition are drawn using a stick model with labels. N* and C* denote the first and last residues of the construct. Bound zinc atoms (Zn1 and Zn2) are shown as gray balls. The estimated position of F352 is represented by the transparent stick model (invisible phenyl side chain in apo structure). <strong>b</strong> Transparent molecular surface of apo-form ZZ domain representing electrostatic potential. Red and blue colors represent negatively and positively charged areas, respectively. The <em>N</em>-degron binding pocket is highlighted as a red circle. <strong>c</strong> Topology diagram of PRT1<sup>ZZ</sup>. The ZZ domain of PRT1 has a conserved ZZ domain topology. The construction comprises three β-sheets and one α-helix. The loops (L1–L3) containing the key residues are represented with the same colors as those in (<strong>a</strong>). <strong>d</strong> Schematic diagram showing Zn ion coordination of PRT1<sup>ZZ</sup>. Zn1 and Zn2 ions coordinate with C<sub>4</sub> and C<sub>2</sub>H<sub>2</sub> Zn-finger motifs, respectively.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">ZZ domain structure in complex with <em>N</em>-degron of Tyr61-BB</h3>
<p id="Par8">To determine the structure of the ZZ domain in complex, we fused the <em>N</em>-degron of Tyr61-BB (Y61-K62-F63-G64) to the <em>N</em>-terminus of the ZZ domain and then exposed the Tyr residue using the LC3B-fusion technique (Fig. <a href="#Fig2" class="usa-link">2a</a>)<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Utilizing the Tyr-exposed PRT1<sup>ZZ</sup> protein, we determined the structure of PRT1<sup>ZZ</sup> in complex with Tyr61-BB <em>N</em>-degron at a resolution of 1.74 Å (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). Each <em>N</em>-degron attached to the ZZ domain mutually interacted with a neighboring ZZ domain in the crystal, forming a dimer in the crystalline lattice (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). The binding site of the ZZ domain revealed interactions between the positively charged Nt NH<sub>3</sub><sup>+</sup> group of Tyr61 and the side chains of Asp312 and Asp336. The main chains of the <em>N</em>-degron peptide formed a hydrogen-bond network with Gly308, Gly310, and Ile333 (Supplementary Figs. <a href="#MOESM1" class="usa-link">3a-c</a>). The aromatic hydrophobic side chain of the Tyr61 residue nests in the hydrophobic pocket and is formed by three key residues: Ile333, Tyr317, and Phe352 (Fig. <a href="#Fig2" class="usa-link">2b, c</a>). In addition, it comprises two hydrophobic regions (sites 1 and 2). The Tyr317 residue is at the site 1 formed by L1, whereas the Ile333 and Phe352 residues are at the site 2 formed by L2 and L3. In the apo structure, the putative <em>N</em>-degron-binding region was wide open (Fig. <a href="#Fig1" class="usa-link">1b</a>) because these loops (L2 and L3) were dynamic and flexible, corresponding to a weak electron density. However, when PRT1<sup>ZZ</sup> recognized the <em>N</em>-degron substrate, the electron density required for recognizing the residues and loops became clear (Supplementary Figs. <a href="#MOESM1" class="usa-link">3d–f</a>). Nt Tyr61 and the third Phe63 of the <em>N</em>-degron residues are snugged into the narrow pocket, and the second Lys62 stretches outside (Fig. <a href="#Fig2" class="usa-link">2c</a>). A dramatic conformational change in which the two loops, L2 and L3, moved toward the incoming substrate as shown in the induced-fit model (Fig. <a href="#Fig2" class="usa-link">2d</a>). To evaluate a possible conformational change affected by the crystal contact, the individual asymmetric units within the crystalline lattice were analyzed (Supplementary Fig. <a href="#MOESM1" class="usa-link">4a</a>). The loops were not involved in the crystalline packing, and regardless of the packings, all different <em>N</em>-degron-bound PRT1<sup>ZZ</sup> structures were almost identical, strongly supporting the conformational change derived from the complex formation. When comparing the B-factor putty representation of the apo and the ligand-bound structures, the B-factors of key residues in PRT1<sup>ZZ</sup> were notably reduced in the ligand-bound form (Supplementary Fig. <a href="#MOESM1" class="usa-link">4b</a>). Unlike the other flexible loops (L2 and L3), L1, containing Tyr317, maintained the same position with or without the <em>N</em>-degron (Fig. <a href="#Fig2" class="usa-link">2e</a>). While Ile333 exhibited a relatively modest conformational change of 2.9 Å, Phe352 showed the largest conformational change of up to 9.5 Å (distance between the equivalent Cα atoms) (Fig. <a href="#Fig2" class="usa-link">2e</a>). Following this conformational change, hydrophobic site 2 established favorable interactions with Tyr61 of Tyr61-BB.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Complex structure of PRT1<sup>ZZ</sup> with <em>N</em>-degron.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370974_41467_2025_63282_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/044e/12370974/9b190c8fb5aa/41467_2025_63282_Fig2_HTML.jpg" loading="lazy" id="d33e846" height="494" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>
<em>A. thaliana</em> BB has a proteolytic site cleaved by DA1 protease. As a result, cleaved BB (Tyr61-BB) has an exposed tyrosine residue at the N-terminus, which is an <em>N</em>-degron. To mimic this <em>N</em>-degron, the YKFG amino acid sequence was added between LC3B and PRT1<sup>ZZ</sup> by mutagenesis. The <em>N</em>-degron residues (Tyr, Phe, and Trp) are exposed by cleaving LC3B using ATG4B protease. <strong>b</strong> Complex structure of PRT1<sup>ZZ</sup> with the <em>N</em>-degron of Tyr61-BB. Tyr317 at site 1 (red dotted box) and Ile333 and Phe352 at site 2 (blue dotted box) are key residues interacting with <em>N</em>-degron. <strong>c</strong> Close-up view of <em>N</em>-degron-binding region with a transparent electrostatic surface. <strong>d</strong> Structural superposition of <em>N</em>-degron-bound PRT1<sup>ZZ</sup> (cyan) and apo-form (orange). L2 and L3 loops are very flexible without bound <em>N</em>-degron. <strong>e</strong> Close-up view of the superposition of the <em>N</em>-degron-binding region. Compared with the apo-form ZZ domain, the flexible loops underwent a large conformational change upon complex formation. The movements of the residues upon complex formation are as follows: Phe352 − 9.5 Å and Ile333 − 2.9 Å.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Unique flexible loops of PRT1<sup>ZZ</sup>
</h3>
<p id="Par9">L3, containing Phe352, is a unique structure exclusively present in the ZZ domain of PRT1. This long loop is conserved among PRT1 proteins from various species (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>) but shares no similarity among the structurally determined ZZ domains (Fig. <a href="#Fig3" class="usa-link">3a</a>). According to our sequence alignment, ZZ domains can be classified depending on the number of residues between the two His residues that coordinate the second Zn<sup>2+</sup> (Fig. <a href="#Fig3" class="usa-link">3a</a>). The His-x-x-x-His motif (HxxxH) has a more curved turn for coordinating Zn<sup>2+</sup> compared with the His-x-x-His (HxxH) or His-x-His motif (HxH). When more residues exist between the two His residues coordinating Zn<sup>2+</sup>, a longer loop (L3) between the α1-helix and the last Zn-coordinating His residue (before the β3-strand) is required. This tendency is also observed in the reported ZZ domains from the PROSITE database<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>). This clearly shows the correlation between the length of the loops and the type of Zn coordination (Fig. <a href="#Fig3" class="usa-link">3a</a>). PRT1 has long loops with three more residues in L2 and an exceptionally long L3, which is five more residues among the HxxxH-type ZZ domains (Fig. <a href="#Fig3" class="usa-link">3b</a>). Considering that the key residues responsible for establishing hydrophobic networks are positioned in these loops, these are distinctive features of PRT1. Consequently, the Phe352 side chain can reach the binding pocket and interact with the Nt hydrophobic bulky residue of the substrate. We compared the structure of PRT1<sup>ZZ</sup> with that of the p62/SQSTM1 ZZ domain, which contains an HxxH motif and a short L3 (Fig. <a href="#Fig4" class="usa-link">4a</a>). p62/SQSTM1 has only a negatively charged patch on its shallow binding site and thus utilizes only limited interacting residues with type-1 Arg/N-degrons (R-BiP), such as Asn132 and Asp147 (Fig. <a href="#Fig4" class="usa-link">4b</a>). Unlike p62/SQSTM1, PRT1<sup>ZZ</sup> forms a narrow and deep hydrophobic binding pocket consisting of more residues, including Tyr317, Ile333, and Phe352, to recognize its substrates (Fig. <a href="#Fig4" class="usa-link">4</a>). PRT1 displayed a stronger affinity for its substrates than that of p62/SQSTM1<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> (Supplementary Figs. <a href="#MOESM1" class="usa-link">1b</a> and <a href="#MOESM1" class="usa-link">8a</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Correlation between loop lengths and types of Zn coordination.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370974_41467_2025_63282_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/044e/12370974/63c9a18c46a0/41467_2025_63282_Fig3_HTML.jpg" loading="lazy" id="d33e938" height="546" width="750" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Sequence alignment of structurally available ZZ domains. PDB IDs and types of Zn ion coordination are indicated on the left. Conserved zinc-coordinating cysteine and histidine residues are marked with red and green boxes, respectively. The key determinants of hydrophobic type-2 <em>N</em>-degron in the loops are circled and labeled in red. <strong>b</strong>, <strong>c</strong> Structure superposition of PRT1<sup>ZZ</sup> with other known ZZ domain structures. Superpositions with ZZ domains possessing HxxxH-type ZZ domains are shown in (<strong>b</strong>), and HxxH and HxH-type ZZ domains are presented in <strong>c</strong>. PDB IDs: 6e86 (ZZZ3; white), 2e5r (Dystrobrevin alpha; blue), 4xi6 (Mindbomb1; orange), 6ww3 (HERC2; green), 5yp7 (p62/SQSTM1; magenta), 2dip (SWIM; gray), 1tot (CBP; salmon), and 6ds6 (p300; yellow).</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Structural comparison of PRT1<sup>ZZ</sup> and p62/SQSTM1 ZZ domain.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370974_41467_2025_63282_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/044e/12370974/a68e253b9b3d/41467_2025_63282_Fig4_HTML.jpg" loading="lazy" id="d33e968" height="840" width="750" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Electrostatic surface of PRT1 and p62/SQSTM1 ZZ domains (PDB IDs: PRT1 [8zg9], p62/SQSTM1 [5yp7]). A negatively charged region of PRT1<sup>ZZ</sup> (left) is enclosed by L3, while that of the p62/SQSTM1 ZZ domain (right) is exposed. <strong>b</strong> Close-up view of structural comparison. The α-NH<sub>3</sub> group of both <em>N</em>-degrons (YKFG and R-BiP peptides) is recognized by two conserved Asp residues. The non-conserved key residues of the PRT1<sup>ZZ</sup> participate in the interaction with the hydrophobic bulky <em>N</em>-degron.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">Type-2 Arg/<em>N</em>-degron selectivity for PRT1</h3>
<p id="Par10">To further characterize the binding mode to type-2 Arg/<em>N</em>-degrons, we crystallized the ZZ domain fused to FKFG and WAAG. The binding mode of the Phe <em>N</em>-degron was similar to that of the Tyr <em>N</em>-degron, whereas that of the Trp <em>N</em>-degron partially differed (Supplementary Fig. <a href="#MOESM1" class="usa-link">2b</a>). Specifically, the Tyr317 residue shifts approximately 2.1 Å backward to accommodate the bulky side chain of Trp into the narrow binding pocket (Fig. <a href="#Fig5" class="usa-link">5a</a>). These bulky hydrophobic Nt residues reached the key PRT1<sup>ZZ</sup> residues—Tyr317, Ile333, and Phe352—within a distance of 4 Å (Fig. <a href="#Fig5" class="usa-link">5b</a>). Type-1 Arg/<em>N</em>-degrons (e.g., Arg) cannot interact with key residues because of their positive charge. Similarly, non-bulky hydrophobic residues (such as Leu and Ile) were too short to be in contact with these key residues (Supplementary Fig. <a href="#MOESM1" class="usa-link">7a</a>). Therefore, our inference from the structure was that only the type-2 bulky Arg/<em>N</em>-degrons fit PRT1<sup>ZZ</sup>. This coincides with the affinity measurements obtained by ITC experiments with <em>N</em>-degron peptides containing x<sub>1</sub>AA sequences (x<sub>1</sub>; Tyr, Phe, Trp, Leu, Ile, and Arg). The ZZ domain showed a similar binding affinity range to the exposed Nt hydrophobic bulky residues, regardless of the type (Fig. <a href="#Fig5" class="usa-link">5c</a>). Conversely, non-bulky hydrophobic residues and type-1 Arg/<em>N</em>-degrons showed no affinity for the ZZ domain (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). These non-bulky and polar residues may fail to induce conformational changes in Phe352 and L3 after docking the <em>N</em>-degron into the ZZ domain.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Comparison of the bound type-2 Arg/<em>N</em>-degrons.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370974_41467_2025_63282_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/044e/12370974/e224211477d1/41467_2025_63282_Fig5_HTML.jpg" loading="lazy" id="d33e1072" height="995" width="798" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Superposition of ZZ domain complexes with peptides, YKFG (colored cyan for protein and white for peptide), FKFG (magenta for protein and slate for peptide), and WAAG (green for protein and salmon for peptide). <strong>b</strong> Close-up view of the binding mode of the ZZ domain with Nt Tyr, Phe, and Trp residues. Type-2 bulky hydrophobic <em>N</em>-degrons are positioned near the key residues of the ZZ domain. The hydrophobic network is highlighted as a transparent brownish oval. The colors are consistent with those in (<strong>a</strong>). <strong>c</strong> Isothermal calorimetry results of the FL PRT1<sup>WT</sup> with type-2 bulky hydrophobic Arg/<em>N</em>-degrons. <em>K</em><sub>D</sub> values: YAA = 23.3 ± 0.958 μM; FAA = 5.45 ± 0.212 μM; WAA = 2.85 ± 0.164 μM. The detailed parameters are listed in Supplementary Table <a href="#MOESM1" class="usa-link">3</a>.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Hydrophobic network between the PRT1<sup>ZZ</sup> and <em>N</em>-degron</h3>
<p id="Par11">Tyr317 and Ile333 also established hydrophobic interactions with the third residue of Tyr61-BB, Phe63. Because of these interactions, the aromatic ring of the Phe63 residue covered the outside of the binding site, similar to a roof (Fig. <a href="#Fig5" class="usa-link">5a</a>). To investigate the contribution of each residue to the <em>N</em>-degron sequence, we measured the binding affinity between full-length (FL) PRT1 wild-type (WT) and various Tyr-degron peptides (YKF, YKA, YAF, and YAA) using the ITC approach (Supplementary Figs. <a href="#MOESM1" class="usa-link">8a–c</a> and Fig. <a href="#Fig5" class="usa-link">5c</a>). The hydrophobic interaction between the third <em>N</em>-degron residue and PRT1<sup>WT</sup> was not critical; however, the presence of a bulky aromatic ring was partially preferred. This preference agreed with the ITC data obtained using F-peptides. When comparing the binding affinities of FKF and FKA peptides, FKF showed a stronger affinity with FL PRT1 than FKA (Supplementary Figs. <a href="#MOESM1" class="usa-link">8d, e</a>). Unfortunately, due to its poor solubility in the buffer, the binding affinity of FAF peptide could not be measured.</p>
<p id="Par12">However, the third Phe of <em>N</em>-degron peptide significantly influenced the binding affinity with the FL PRT1 mutants (Y317A, I333A, and Y317A/I333A). According to the binding affinity measurement, the YKA peptide was not bound to the PRT1<sup>Y317A</sup> and PRT1<sup>I333A</sup> mutants (Fig. <a href="#Fig6" class="usa-link">6a</a>). In contrast, the YKF peptide maintained its binding affinity with the PRT1<sup>Y317A</sup> and PRT1<sup>I333A</sup> mutants at levels similar to those of PRT1<sup>WT</sup> (Fig. <a href="#Fig6" class="usa-link">6b</a>). Moreover, the PRT1<sup>Y317A/I333A</sup> double mutant of PRT1 showed a considerably low affinity for the YKF peptide (Fig. <a href="#Fig6" class="usa-link">6b</a>). These results show that the Tyr317 and Ile333 residues of PRT1 and Phe63 of Tyr61-BB form a hydrophobic network to stabilize the <em>N</em>-degron in the binding pocket of PRT1<sup>ZZ</sup>. As a result, the Tyr317 and Ile333 residues support positioning hydrophobic <em>N</em>-degrons and work in concert with Phe352, the primary residue for recognizing the type-2 bulky hydrophobic Nt residue. Next, we investigated whether the presence of a third Phe residue forms a hydrophobic network with non-bulky hydrophobic <em>N</em>-degrons, such as Leu or Ile. We measured the binding affinity of the LKF peptide and found no detectable affinity (Supplementary Fig. <a href="#MOESM1" class="usa-link">7b</a>). Non-bulky hydrophobic residues were insufficient to establish a hydrophobic network. These results indicate that Phe352 of PRT1 is the primary determinant of the interaction with the type-2 bulky hydrophobic Nt residue of <em>N</em>-degrons. Moreover, additional Tyr317 and Ile333 residues are essential for establishing the hydrophobic network, which prefers a bulky hydrophobic residue at the third position of the <em>N</em>-degron (Figs. <a href="#Fig5" class="usa-link">5</a> and <a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. The effects of key residue mutations of FL PRT1 and the third residue of <em>N</em>-degron.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370974_41467_2025_63282_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/044e/12370974/f791bbba7edb/41467_2025_63282_Fig6_HTML.jpg" loading="lazy" id="d33e1238" height="568" width="799" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Isothermal titration calorimetry (ITC) results of PRT1 mutants (PRT1<sup>Y317A</sup> and PRT1<sup>I333A</sup>) with the YKA peptide. No affinity or very weak affinity were determined. <strong>b</strong> ITC results of PRT1 mutants (PRT1<sup>Y317A</sup>, PRT1<sup>I333A</sup>, PRT1<sup>Y317A/I333A</sup>, and PRT1<sup>F352A</sup>) with the YKF peptide. <em>K</em><sub>D</sub> values: PRT1<sup>Y317A</sup> = 41.3 ± 0.799 μM; PRT1<sup>I333A</sup> = 13.7 ± 1.68 μM; PRT1<sup>Y317A/I333A</sup> = N.D. (not determined); PRT1<sup>F352A</sup> = N.D. The detailed parameters are listed in Supplementary Table <a href="#MOESM1" class="usa-link">3</a>. <strong>c</strong> Ubiquitylation activity of FL PRT1 depending on reaction time (each lane at 0, 0.5, 1, 3, 5, and 7 h). The poly-Ub pattern of Tyr61-BB<sup>V198E</sup> was detected in the presence (left) and absence (right) of PRT1. See Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a> for the details of the diminished auto-ubiquitylation activity of the Tyr61-BB<sup>V198E</sup>. <strong>d</strong> Ubiquitylation activity of the PRT1 mutants with an authentic Nt sequence-containing substrate, YKF-BB<sup>V198E</sup>. Ctrl: no PRT1 protein. <strong>e</strong> Ubiquitylation activity of the PRT1 mutants with an Nt permutated substrate, Y<span class="text-underline">AA</span>-BB<sup>V198E</sup> (alanine at the second and third position). The key residue mutants of PRT1 showed reduced or no ubiquitylation activity compared with that of the WT. These loss-of-function effects were more reinforced when the <em>N</em>-terminus of Tyr61-BB<sup>V198E</sup> was permutated to YAA. Ctrl: no PRT1 protein. The ubiquitylation reactions are quenched at 3 h. All images probed with anti-His Ab were detected under intensive conditions (2.5× loading volume and longer exposure time) compared to those probed with anti-FLAG or anti-Ub. The original western (same loading volume and exposure intensity) images with anti-His Ab are shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">10b</a>. These ubiquitylation assays were performed at least three times.</p></figcaption></figure></section><section id="Sec9"><h3 class="pmc_sec_title">A ubiquitylation-defective mutant of Tyr61-BB substrate</h3>
<p id="Par13">To assess the relationship between the binding and ubiquitylation of <em>N</em>-degron substrates by PRT1, we performed in vitro ubiquitylation assays using the Tyr61-BB protein. However, BB is also an E3 Ub ligase with a RING domain at its C-terminus and in vitro auto-ubiquitylation activity, resulting in robust poly-Ub chain formation (Supplementary Figs. <a href="#MOESM1" class="usa-link">9a, b</a>). Therefore, we designed a ubiquitylation-defective Tyr61-BB mutant by changing the two Val residues of the E2~Ub binding motif to Glu residues, where “~” denotes the covalent binding (V198E and V236E)<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. These Tyr61-BB mutants showed diminished auto-ubiquitylation activity originated from BB compared with that of the BB<sup>WT</sup>, and the BB<sup>V198E</sup> mutant was more effective in reducing this activity than BB<sup>V236E</sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">9a</a>). Although the V198E mutant of Tyr61-BB (Tyr61-BB<sup>V198E</sup>) showed low auto-ubiquitylation activity, it showed a poly-ubiquitylation pattern in the presence of PRT1<sup>WT</sup> (Fig. <a href="#Fig6" class="usa-link">6c</a><strong>)</strong>. Therefore, it is a useful substrate for evaluating the ubiquitylation activity of PRT1<sup>WT</sup> and its mutants.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Ubiquitylation activity of PRT1 and its mutants</h3>
<p id="Par14">Although PRT1 also exhibited minimal in vitro auto-ubiquitylation activity, regardless of the presence or absence of the substrate (Fig. <a href="#Fig6" class="usa-link">6c</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">10a</a>), the Tyr61-BB<sup>V198E</sup> mutant substrate was used to investigate the poly-ubiquitylation activity of the PRT1 E3 Ub ligase. The input amounts of the PRT1<sup>WT</sup> and its variants were relatively lower than those of other components, making the detection of their bands difficult under standard conditions (Supplementary Fig. <a href="#MOESM1" class="usa-link">10b</a>). Therefore, each corresponding sample was loaded at a 2.5-fold greater volume and detected under intensive exposure conditions to improve visibility (His-tag Ab detection). To confirm the relationship between in vitro binding affinity and the enzymatic activity of PRT1, we performed a ubiquitylation assay using PRT1<sup>WT</sup> and various mutants (Fig. <a href="#Fig6" class="usa-link">6d</a>). A similar tendency in the ubiquitylation pattern was observed, consistent with the binding affinity. The PRT1<sup>WT</sup> showed the most robust poly-Ub band, and the PRT1<sup>Y317A</sup> and PRT1<sup>I333A</sup> mutants and PRT1<sup>Y317A/I333A</sup> double mutant showed decreased ubiquitylation activity compared with that of the WT (Fig. <a href="#Fig6" class="usa-link">6d</a>). The Tyr317 residue of PRT1 is more crucial than Ile333 for binding to the substrate, which can be expected in the complex structure (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). This was further confirmed by the ubiquitylation activity of the mutants (Fig. <a href="#Fig6" class="usa-link">6d</a>). Ubiquitylation activity was completely abolished in the PRT1<sup>F352A</sup> mutant, confirming that Phe352 is the primary determinant for recognizing the hydrophobic Nt residue (Fig. <a href="#Fig6" class="usa-link">6d</a>). A similar pattern was also observed in ubiquitylation assay using the Tyr-CP8GFP model substrate<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>). Since PRT1<sup>WT</sup> and its variants were auto-ubiquitylated in vitro (Fig. <a href="#Fig6" class="usa-link">6c</a>), the western blotting was performed at initial time point to compare input amount of PRT1<sup>WT</sup> and its variants (Fig. <a href="#Fig6" class="usa-link">6d</a>, e and Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>).</p>
<p id="Par15">To verify the effect of the third <em>N</em>-degron residue, Phe63, we performed an in vitro ubiquitylation assay with Tyr61-BB<sup>V198E</sup>, in which the residue in the <em>N</em>-degron was changed to an Ala residue (YAA-BB<sup>V198E</sup>) using the same batch of PRT1<sup>WT</sup> and its mutants (Fig. <a href="#Fig6" class="usa-link">6e</a>). In all cases, the YAA-BB<sup>V198E</sup> substrate showed diminished ubiquitylation activity compared with that of Tyr61-BB<sup>V198E</sup> (YKF-BB<sup>V198E</sup>), which has the original <em>N</em>-degron (Fig. <a href="#Fig6" class="usa-link">6d, e</a>). Again, the PRT1<sup>Y317A</sup> mutant showed lower ubiquitylation activity than the PRT1<sup>I333A</sup> mutant. The disruption of the hydrophobic network by mutating the hydrophobic residues, Tyr317 and/or Ile333, dramatically decreased the ubiquitylation of YAA-BB<sup>V198E</sup> more than YKF-BB<sup>V198E</sup> (Fig. <a href="#Fig6" class="usa-link">6d, e</a>). These results indicate that the key hydrophobic residues in the binding pocket of PRT1 exert differential effects in constructing the hydrophobic network and fine-tuning the substrates, depending on the third position of the <em>N</em>-degron sequence.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Overall shape of PRT1 and characteristics of tandem RING domains</h3>
<p id="Par16">Although PRT1 has tandem RING domains (RING1 and RING2), RING1 is essential for the ubiquitylation of the substrate (Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>). To gain insights into the role of the RING2 domain, the FL PRT1 structure was predicted using AlphaFold (AF)<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. The AF model exhibited a compact structure with two RINGs and one ZZ domain (Supplementary Figs. <a href="#MOESM1" class="usa-link">13a, b</a>). However, the AF model contradicts the ‘L’ shaped molecular envelope of FL PRT1 of the small-angle X-ray scattering (SAXS) method (Fig. <a href="#Fig7" class="usa-link">7a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">13c</a>). As expected, FL PRT1 exhibited a monomeric state confirmed by both SEC-MALS (Supplementary Fig. <a href="#MOESM1" class="usa-link">14a</a>) and SEC-SAXS analyses (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). Therefore, we dissected the monomeric FL PRT1 into RING and ZZ domains, and investigated these interactions using separate domains. The C-terminal ZZ domain behaved as an independent domain in solution, based on individual migration in size-exclusion chromatography (Supplementary Fig. <a href="#MOESM1" class="usa-link">14b</a>), as expected from the SAXS envelope model (Fig. <a href="#Fig7" class="usa-link">7a</a>). Meanwhile, the AF model of RING1 + 2 showed an intramolecular heterodimer due to the hydrophobic network of their four Leu residues (Leu79 and Leu82 in RING1 and Leu246 and Leu249 in RING2). Similar dimer formation was observed in other intermolecular homo- and heterodimeric RING–RING structures (Fig. <a href="#Fig7" class="usa-link">7b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>)<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>–<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. As expected, the two RING domains co-migrated in size-exclusion chromatography, suggesting intramolecular heterodimeric formation between RING1 and RING2 (Fig. <a href="#Fig7" class="usa-link">7c, d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">14c</a>). RING dimerization reportedly is crucial in ubiquitylation activity<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Therefore, we verified whether the PRT1 mutant that disrupted the RING intramolecular heterodimer showed significantly reduced ubiquitylation activity (Fig. <a href="#Fig7" class="usa-link">7e</a>). The core hydrophobic leucine residues were replaced with hydrophilic serine residues (2 L/S—one of the RINGs: L79S/L82S in RING1 and L246S/L249S in RING2; 4 L/S—both of the RINGs: L79S/L82S/L246S/L249S) at the dimeric interface. Unlike the RING2<sup>WT</sup>, the RING2<sup>2L/S</sup> mutant did not co-migrate with RING1<sup>WT</sup> (Fig. <a href="#Fig7" class="usa-link">7c, d</a>). Moreover, RING2<sup>WT</sup> acted as a dimer in solution, whereas RING2<sup>2L/S</sup> behaved as a monomer (Fig. <a href="#Fig7" class="usa-link">7c</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">14c</a>). Therefore, it is clear that the RING1 and RING2 of PRT1 form an intramolecular dimer. As expected, the RING dimer-disrupted mutant, PRT1<sup>4L/S</sup> exhibited little ubiquitylation activity (Fig. <a href="#Fig7" class="usa-link">7e</a>). In addition, to assess the loss of function in the PRT1<sup>4L/S</sup> mutant, we performed FRET-based Ub discharge assay. As a result, the PRT1<sup>4L/S</sup> mutant showed 10-fold lower activity compared to PRT1<sup>WT</sup> (Fig. <a href="#Fig7" class="usa-link">7f</a>). These results indicate that while disruption of the intramolecular RING1+2 dimer organization substantially reduces Ub discharge activity, it does not completely abolish it. These findings support that the correct orientation of the dimeric RING organization is critical for full E3 ligase activity.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. Overall structure of FL PRT1 and RING dimerization.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370974_41467_2025_63282_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/044e/12370974/db70e7123d8c/41467_2025_63282_Fig7_HTML.jpg" loading="lazy" id="d33e1548" height="490" width="799" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> SEC-SAXS envelope model of PRT1. PRT1 has an L-shaped structure. The upper part of the model corresponds to the C-terminus, where the ZZ domain is located. The dimeric RING domain is positioned at the curved body. The ZZ domain is colored cyan, and the AlphaFold2 model of the dimeric RING domain is colored blue (RING1) and salmon (RING2) in the atomic models fitted into the SAXS molecular envelope. <strong>b</strong> RING1 and RING2 heterodimer formation. This model was obtained using AlphaFold, and the mutated residues (4 L/S: L79S/L82S in RING1 and L246S/L249S in RING2) for disrupting dimers are shown as yellow sticks. <strong>c</strong> Peak migration profile for characterizing tandem RING domains. RING2<sup>WT</sup> co-migrates with RING1<sup>WT</sup>, whereas RING2<sup>2L/S</sup> (L246S/L249S double mutant) behaves independently. Both species in peak 1 are too low a concentration to calculate experimental molecular weights. The detailed SEC-MALS analysis for peak 2 is shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">14c</a>. UV absorption was measured at a 280-nm wavelength. The results for the RING1<sup>WT</sup> + RING2<sup>WT</sup> mixture are represented by a yellow line, and those for RING1<sup>WT</sup> + RING2<sup>2L/S</sup> by a blue line. Schematic oligomeric states of RING domains are shown below the UV profile. <strong>d</strong> Corresponding gel images for the fractions in (<strong>c</strong>). RING1<sup>WT</sup> and RING2<sup>WT</sup> are shown at 20 and 10 kDa, respectively. We observed consistent results at least three times. <strong>e</strong> Ubiquitylation assay of PRT1<sup>WT</sup> and 4 L/S mutant. When the dimeric interface of RING domains was disrupted, the ubiquitylation activity of PRT1 was lost. Each of the samples was incubated at 30 °C for 3 h. Ctrl: No PRT1. The ubiquitylation assays were performed at least three times. <strong>f</strong> FRET-based Ub discharge assay. The E2~Ub discharge activity of the 4 L/S mutant was decreased by 10-fold compared to the WT. The normalized FRET data was analyzed by one-phase decay analysis with GraphPad Prism 5. The data are represented by mean ± SEM from three independent experiments.</p></figcaption></figure></section></section><section id="Sec12"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par17">Well-known <em>N</em>-recognins, such as those of the UBR family, p62/SQSTM1, and ClpS, can recognize corresponding <em>N</em>-degron substrates using their unique domains, such as the UBR box and ZZ and ClpS domains, to bind with <em>N</em>-degrons. In contrast to UBR proteins in yeast and mammals, no defined <em>N</em>-recognin ortholog containing a ClpS-homology domain for recognizing type-2 <em>N</em>-degron has been identified in plants<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. However, PRT1 is a well-known <em>N</em>-recognin that has been reported to ubiquitylate the F-DHFR model<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup> and Tyr61-BB physiological substrates<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. PRT1 has a ZZ domain that explicitly recognizes only type-2 Arg/<em>N</em>-degrons, particularly the bulky hydrophobic residues, whereas the ZZ domain of p62/SQSTM1 prefers positively charged type-1 Arg/<em>N</em>-degrons<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Therefore, the mechanism of type-2 <em>N</em>-degron recognition by PRT1<sup>ZZ</sup> is a long-standing mystery in plant Ub biology. Furthermore, the <em>N</em>-degron substrate specificity of the ZZ domain is notable. Our structural and biochemical studies characterized the unique features of PRT1<sup>ZZ</sup> in comparison to other ZZ domains and identified the key residues that participate in recognizing type-2 bulky hydrophobic Arg/<em>N</em>-degrons. In contrast to the <em>N</em>-degron recognition domains in the <em>N</em>-recognin E3 ligases, PRT1<sup>ZZ</sup> underwent dramatic conformational changes upon <em>N</em>-degron complex formation (Fig. <a href="#Fig8" class="usa-link">8a</a>). The key residues, Tyr317, Ile333, and Phe352, form hydrophobic networks not only with the first residue of the <em>N</em>-degrons but also with the third Phe. These hydrophobic residues in PRT1 are at the three loops (L1, L2, and L3). These dynamic loops, especially L2 and L3, are long enough to participate in the hydrophobic network, which is a unique feature of PRT1<sup>ZZ</sup> relative to the p62/SQSTM1 ZZ domain for recognizing basic type-1 <em>N</em>-degrons as well as other ZZ domains, as well as for binding to proteins involved in various cellular processes such as transcriptional regulation, chromatin remodeling, and ubiquitylation<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig8"><h3 class="obj_head">Fig. 8. Mode of action of PRT1 E3 Ub ligase.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/044e/12370974/d00011d28be7/41467_2025_63282_Fig8_HTML.jpg" loading="lazy" id="d33e1666" height="430" width="666" alt="Fig. 8"></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Conformational change of two PRT1<sup>ZZ</sup> loops, L2 and L3, upon complex formation with the <em>N</em>-degron substrate, while L1 maintains position as a rigid frame. <strong>b</strong> Active RING1-RING2 heterodimer interacts with the UBC8 E2 Ub-conjugating enzyme and Ub, which forms a competent ternary complex for ubiquitylation. The ZZ domain at the C-terminal region of PRT1 grabs the Tyr61-BB substrate through its bulky hydrophobic <em>N</em>-degron (Nt tyrosine). The flexibility between ZZ and RING domains enables the efficient delivery of the substrate to RING1, which interacts with the E2 enzyme and Ub. The red star represents the ubiquitylation reaction, and several rounds of reaction generate the poly-ubiquitylated Tyr61-BB.</p></figcaption></figure><p id="Par18">Bulky hydrophobic <em>N</em>-degrons are sufficient for maintaining the hydrophobic network at the binding site; however, small hydrophobic and charged <em>N</em>-degrons are unlikely to do so. The hydrophobic networks between PRT1<sup>ZZ</sup> and type-2 <em>N</em>-degrons significantly affected ubiquitylation activity. Next, we investigated how PRT1 ubiquitylates recognized type-2 <em>N</em>-degron substrates. Based on our SAXS data of PRT1, an overall ‘L’ shape was observed, where the heterodimeric RING domains are at the bottom and the ZZ domain is at the top (Fig. <a href="#Fig7" class="usa-link">7a</a>). RING1 + 2 in PRT1 appears to have an active RING dimer conformation, as seen in other RING-type E3 Ub ligases<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>–<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Various RING homodimerizations activate E3 ubiquitylation, and provide several examples of intermolecular RING heterodimerization (Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>). In the case of PRT1, intramolecular RING heterodimerization may occur based on AF2 prediction; more importantly, the disruption mutant for dimerization abolishes ubiquitylation activity. Therefore, we speculate that monomeric PRT1 becomes an active E3 ligase via its unique tandem RING domain structure. Notably, the functional RING domains and <em>N</em>-degron recognizing the ZZ domain were far from each other (Fig. <a href="#Fig8" class="usa-link">8b</a>). Although a dynamic structural change in PRT1 must precede the ubiquitylation reaction, the PRT1 substrate, Tyr61-BB, must have a stretched shape that allows it to interact with the ZZ domain and simultaneously reach the RING1 domain for ubiquitylation (Fig. <a href="#Fig8" class="usa-link">8b</a>). Indeed, Tyr61-BB has an unstructured region consisting of approximately 130 residues in front of the C-terminal RING domain, which contains many lysine residues for ubiquitylation.</p>
<p id="Par19">We propose the following two possibilities for the relevance of the intramolecular RING dimerization for the ubiquitylation activity of PRT1. First, the dimeric interface is critical for maintaining protein stability (Supplementary Fig. <a href="#MOESM1" class="usa-link">16</a>). The mixture of RING1<sup>WT</sup> and RING2<sup>WT</sup> formed a stable complex, whereas RING1<sup>WT</sup> in the presence of RING2<sup>2L/S</sup> exhibited a degradation pattern without complex formation (Fig. <a href="#Fig7" class="usa-link">7d</a>). These results suggest that the RING1 and RING2 dimeric organization is essential for the stability of PRT1. Second, the disruption of the intramolecular dimerization reduces the probability of forming a functional ubiquitylation complex involving RING1-RING2-E2~Ub-substrate (Supplementary Fig. <a href="#MOESM1" class="usa-link">16</a>). As mentioned above, the Ub discharge activity of 4 L/S mutant 10-fold weaker than that of the WT (Fig. <a href="#Fig7" class="usa-link">7f</a>). Considering that PRT1 contains an intrinsically disordered region consisting of approximately 80 amino acid residues between the two RING domains, PRT1<sup>4L/S</sup> mutant must be a spatially distributed contact site on both RING domains to interact with E2~Ub conjugate. Especially once the dimeric interface of PRT1 RING domains is disrupted, the functional RING1 domain exhibits increased conformational flexibility. In conclusion, even when the E2~Ub conjugate contacts the RING domain, the increased mobility of mutant RING1 impairs the precise alignment required for the Ub transfer to the substrates (Supplementary Fig. <a href="#MOESM1" class="usa-link">16</a>). Together, these results highlight the importance of intramolecular RING organization for maintaining the ubiquitylation activity of PRT1 (Fig. <a href="#Fig8" class="usa-link">8b</a>). Several tandemly duplicated RING proteins from various species, including <em>Arabidopsis</em>, have been identified<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>, and several tandemly repeated RING constructs have demonstrated enhanced ubiquitylation activity<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. PRT1 represents a single-chain tandem RING domain, an active and robust E3 ligase without intermolecular dimerization. Thus, our results represent a prototype case of tandem RING domain architecture, which might help interpret the activation mechanism. In the current study, essential structural inquiries concerning the Arg/<em>N</em>-degron pathway of plants were addressed, in addition to <em>At</em>ClpS1 and <em>At</em>PRT6<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>–<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>, expanding our understanding of the structural repertoire of <em>N</em>-degron recognition.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Cloning</h3>
<p id="Par20">The PRT1<sup>ZZ</sup> domain (residues 302–366) coding gene was amplified by PCR using codon-optimized gBlock as a synthesized gene template (Integrated DNA Technologies, Coralville, IA, USA). The amplified ZZ domain gene was inserted into a pET-His-LC3B vector, a modified pET-His vector containing the Kpn1 and Xho1 restriction enzyme sites<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. The resulting plasmids were transformed into <em>Escherichia coli</em> BL21(DE3) cells. The cloning process for FL PRT1 was the same, except that the coding gene was inserted into FP-His, a modified pET-His<sub>6</sub> vector. The <em>N</em>-degron sequence of BB (YKFG) and variations in its <em>N</em>-terminal sequence were introduced between the LC3B and ZZ domain to generate various <em>N</em>-degron-fused ZZ domains. Key mutations (Y317A, I333A, and F352A) were introduced into the FL PRT1 plasmid. These mutations were induced by PCR-based site-directed mutagenesis. PRT1 constructs (FL, 1–273, and 308–365) were cloned into the pET-bRIP (barley ribosome-activating protein) vector, a bRIP-tag-inserted version of the pET-His vector<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Protein expression and purification</h3>
<p id="Par21">LC3B-<em>N</em>-degron-fused PRT1<sup>ZZ</sup> and FL His<sub>6</sub>-PRT1 plasmid transformed <em>E. coli</em> BL21(DE3) cells were cultured at 37 °C in LB medium in a shaking incubator until OD<sub>600</sub> = 1.0. Thereafter, protein expression was induced by adding 0.5 mM isopropyl β-D-1-thiogalactopyranoside, and 400 μM of ZnSO<sub>4</sub> was also added to the culture medium simultaneously. The induced cells were further incubated at 18 °C for 20 h. The cultured cells were harvested by centrifugation (10,000 × <em>g</em> for 20 min) and resuspended in a lysis buffer containing 50 mM Tris-HCl pH 8.0, 200 mM NaCl, 0.5 mM Tris(2-carboxyethyl)phosphine hydrochloride (TCEP-HCl), and 10 mM β-mercaptoethanol. After cell lysis by ultrasonication, the soluble fraction was separated from the pellet by centrifugation (35,000 × <em>g</em> for 1 h). The LC3B-<em>N</em>-degron-fused PRT1<sup>ZZ</sup> was purified using Ni-NTA affinity chromatography by loading the supernatants onto a HisTrap<sup>TM</sup> HP column (17524802; Cytiva, Marlborough, MA, USA) and eluted by increasing the concentration of His elution buffer containing 50 mM Tris-HCl pH 8.0, 100 mM NaCl, and 0.5 mM TCEP. The His<sub>6</sub>-LC3B tag was cleaved by incubating with His<sub>6</sub>-tagged ATG4B protease (20 °C, overnight)<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>, after which the His<sub>6</sub>-ATG4B protease and His<sub>6</sub>-LC3B tag were removed using HisTrap<sup>TM</sup> HP and HiTrap<sup>TM</sup> Q HP (17115301; Cytiva) columns. As a final step, the <em>N</em>-degron-fused PRT1<sup>ZZ</sup> domain was loaded onto a HiLoad<sup>TM</sup> Superdex<sup>TM</sup> 75 16/600 pg column (28989333; Cytiva) pre-equilibrated with crystallization buffer comprising 25 mM Tris-HCl pH 8.0, 150 mM NaCl, and 0.5 mM TCEP. FL PRT1 and its mutants were purified following Ni-NTA affinity chromatography, His<sub>6</sub>-TEV protease incubation, a second Ni-NTA affinity chromatography, and size-exclusion chromatography with HiLoad<sup>TM</sup> Superdex<sup>TM</sup> 75 16/600 pg pre-equilibrated with the ITC buffer (50 mM Tris-HCl pH 7.5, 150 mM NaCl, and 1 mM TCEP) or Ub assay buffer (50 mM Tris-HCl pH 7.5 and 150 mM NaCl). The detailed processes for each purification step were the same as those described above. The bRIP-tagged PRT1 construct was expressed as described previously and purified through Ni-NTA and HiTrap<sup>TM</sup> Blue HP (Cytiva, 17041301) affinity chromatography and size-exclusion chromatography with HiLoad<sup>TM</sup> Superdex<sup>TM</sup> 75 16/600 pg pre-equilibrated with the ITC buffer.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Crystallization</h3>
<p id="Par22">The purified <em>N</em>-degron-fused PRT1<sup>ZZ</sup> was concentrated to approximately 20 mg/mL and crystallized at 20 °C using the sitting-drop vapor diffusion method with a Gryphon machine (Art Robbins Instruments, Sunnyvale, CA, USA). The <em>N</em>-degron-fused PRT1<sup>ZZ</sup> was mixed at a ratio of 1:1 with crystallization solutions (apo—4 M sodium formate [Crystal Screen, C9 condition: HR2-130; Hampton Research, Aliso Viejo, CA, USA]; Y(F)KFG-<em>At</em>PRT1<sup>ZZ</sup>—32.5–37.5% (v/v) precipitant mix 4, 0.1 M buffer system 2 pH 7.5, and 0.06 M divalent [Morpheus, A8 condition: MD1-46; Molecular Dimensions, Sheffield, UK]; WAAG-<em>At</em>PRT1<sup>ZZ</sup>—1.4 M lithium sulfate, 100 mM Tris-HCl pH 8.5, and 2% (w/v) PEG 400 [Wizard Classic 3, E8 condition: 1009533; Rigaku, Tokyo, Japan]). Next, 20% (v/v) glycerol and 20% (v/v) ethylene glycol were mixed with mother liquors as Y(F)KFG-<em>At</em>PRT1<sup>ZZ</sup> and WAAG-<em>At</em>PRT1<sup>ZZ</sup> crystal cryoprotectants, respectively. The crystals were then flash-frozen in liquid nitrogen.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Data collection and structure determination</h3>
<p id="Par23">The X-ray data of the <em>N</em>-degron-fused PRT1<sup>ZZ</sup> domains (apo and YKFG, 1.7-Å; FKFG, 2.1-Å; WAAG, 2.8-Å resolution) were obtained at Pohang Accelerator Laboratory (PAL) in South Korea and Spring-8 in Japan (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). A single-anomalous dispersion dataset of apo-crystal was obtained at the absorption edge of the zinc atom (λ = 1.28 Å) at beamlines 11 C (PAL)<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> and BL44XU (Spring-8)<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Zinc site determination, phasing, and model auto-building (Autosol and AutoBuild modules) were performed using Phenix software<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> with the apo PRT1<sup>ZZ</sup> dataset. After the initial model was generated using AutoBuild, the remaining parts of the ZZ domain model were manually built using Coot<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. After completing the apo PRT1<sup>ZZ</sup> model, the phases of the other crystal data were determined by molecular replacement using the apo-model. The models were rebuilt manually and refined in iterative cycles using Coot<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. The final structural models were validated using MolProbity<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Isothermal titration calorimetry (ITC)</h3>
<p id="Par24">The purified FL PRT1 and its mutations (Y317A, I333A, F352A, ΔRING1 + RING2, and ΔZZ domain) were concentrated to 45 ~ 55 μM in an ITC buffer. The <em>N</em>-degron peptides (synthesized by Peptron<sup>TM</sup>, Daejeon, South Korea) were dissolved in the same buffer and diluted to a concentration of ~550 μM. The ITC experiment was executed at 25 °C using a Microcal PEAQ-ITC (Malvern Panalytical, Malvern, UK). Each <em>N-</em>degron peptide was injected 19 times (2 μL each) into a 280-μL sample of each protein. ITC data were processed using a one-site binding model with the MicroCal PEAK-ITC analysis software. All ITC experiments were conducted at least thrice.</p></section><section id="Sec19"><h3 class="pmc_sec_title">In vitro ubiquitylation assay</h3>
<p id="Par25">To assess the auto-ubiquitylation activity of Tyr61-BB and its mutants, we used purified and concentrated components as follows: 0.1 μM <em>Hs</em>UBA1 (E1), 0.5 μM <em>At</em>UBC8 (E2), 60 μM Ub, 3 μM Tyr61-BB WT and its mutants, 10 μM ATP, and 10 μM MgCl<sub>2</sub> as the final concentrations in a 50-μL mixture. Each sample was incubated at 30 °C overnight and collected for activity monitoring every 1 h. To perform the Tyr61-BB ubiquitylation assay with PRT1 over different time courses, all components were mixed as follows: 0.1 μM <em>Hs</em>UBA1, 0.5 μM <em>At</em>UBC8, 1 μM PRT1 FL, 60 μM Ub, 3 μM Tyr61-BB V198E mutant, 10 μM ATP, and 10 μM MgCl<sub>2</sub> as the final concentrations in a 50-μL mixture. These samples were incubated at 30 °C. To compare the ubiquitylation activity of PRT1<sup>WT</sup> and its mutants, the same concentrations of all components except 0.5 μM PRT1<sup>WT</sup> and mutants in a 100-μL mixture were incubated for 3 h (or over 12 h) at 30 °C. Ubiquitylated Tyr61-BB was detected using an HRP-conjugated anti-FLAG tag (sc-166355; Santa Cruz Biotechnology, Dallas, TX, USA, 1:2000 ratio) and anti-Ub antibodies (sc-8017; Santa Cruz Biotechnology, 1:2000 ratio), and PRT1 was detected using an HRP-conjugated anti-His antibody (sc-8036; Santa Cruz Biotechnology, 1:1000 ratio). Chemiluminescent signals were captured by brief incubation with 2 mL ECL<sup>TM</sup> Prime Western Blotting Detection Reagent (RPN2232; Cytiva). The 3 μM YKF-CP8GFP was used as a model substrate, which contained no RING domains<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. The GFP was probed by anti-GFP antibodies (sc-9996; Santa Cruz Biotechnology 1:2000 ratio) and HRP-conjugated Goat anti-Mouse IgG antibodies (31430, Invitrogen, 1:5000 ratio). The anti-His chemiluminescence signal for input PRT1 was detected under stronger exposure conditions (high exposure mode, 50 s), while the others were acquired under standard conditions (high-resolution mode, 100 s). The other conditions are the same as mentioned above.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Size-exclusion chromatography coupled with small-angle X-ray scattering (SEC-SAXS)</h3>
<p id="Par26">For the SEC-SAXS experiment, FL PRT1 (10 mg/mL, 250 μL) was loaded onto an S200 increase 10/300 GL column (28990944; Cytiva) pre-equilibrated with a buffer containing 50 mM Tris-HCl pH 7.5, 150 mM NaCl, 1 mM TCEP, and 2% (w/v) glycerol. The scattering data from solutions of PRT1 in 50 mM Tris-HCl pH 7.5, 150 mM NaCl, 1 mM TCEP, and 2% (w/v) glycerol were collected at a wavelength of 1.5 Å and a distance of 3.0 m between the sample and detector at BL10C, Photon Factory, Japan (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). A PILATUS3 2 M (DECTRIS, Täfernhof II, Switzerland) was used as the X-ray detector at a sample-detector distance of 3.0 m and at a wavelength of λ = 0.15 nm (<em>I</em>(<em>s</em>) <em>vs</em> s, where <em>s</em> = 4πsinθ/λ, and 2θ is the scattering angle). The SEC parameters were as follows: A 250.00 μL sample at 10 mg/mL was injected at different flow rates in the protein peak (20 sec/image with flow at 0.05 mL/min) and the baseline (20 sec/image with flow at 0.5 mL/min) using a GE Superdex 200 Increase 10/300 column at 25 °C, and 332 successive 20-second frames were collected. Raw data were initially subtracted from the background signals using SAngler<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. Protein signal peaks and baselines were selected using Chromixs<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. Next, the data were analyzed using PRIMUS to obtain structural parameters and molecular weight deviations<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. GNOM<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> was used to calculate the distance distribution function [<em>P(r)</em>] and maximum particle dimension (<em>D</em><sub>max</sub>). Ab initio modeling was performed using DAMMIF<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup> with 10 calculations with <em>q</em> range (0.0157–0.1657) and DAMMIN<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup> in PRIMUS program suit. The DAMMIN model was represented using ChimeraX<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. The AlphaFold model fit was performed by CRYSOL<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. Detailed parameters and results are described in Supplementary Table <a href="#MOESM1" class="usa-link">2</a><sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>–<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Size-exclusion chromatography coupled with multi-angle laser scattering (SEC-MALS)</h3>
<p id="Par27">For SEC-MALS, RING1<sup>WT</sup> + RING2<sup>WT</sup>, and RING1<sup>WT</sup> + RING2<sup>2L/S</sup> mixtures (1:1 ratio of each component; a final concentration of 3 mg/mL) were loaded after incubation (4 °C, 1 h) onto an S75 Increase 10/300 GL column (29148721; Cytiva) pre-equilibrated with a buffer containing 50 mM Tris-HCl (pH 7.5), 150 mM NaCl, and 1 mM TCEP. FL PRT1 (3 mg/mL) was loaded onto an S200 increase 10/300 GL column (28990944; Cytiva). Light scattering data were collected using a miniDAWN instrument (Wyatt Technology, Santa Barbara, CA, USA). ASTRA software (Wyatt Technology) was used for data analysis with an appropriate extinction coefficient (RING1<sup>WT</sup>: 0.592 mL/mg·cm; RING2<sup>WT</sup> and RING2<sup>2L/S</sup>: 0.294 mL/mg·cm; PRT1<sup>WT</sup> FL: 0.596 mL/mg·cm).</p></section><section id="Sec22"><h3 class="pmc_sec_title">FRET-based ubiquitin discharge assay</h3>
<p id="Par28">The FITC-labeled <em>At</em>UBC8 was generated by sortase-catalyzed transpeptidation<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>. A FITC-labeled sortase recognition peptide (FITC-KLPETGG) was synthesized by Peptron<sup>TM</sup>. The transpeptidation reaction was conducted using 800 μM FITC-labeled peptide, 200 μM GST-sortase A, and 200 μM <em>At</em>UBC8 (the <em>N</em>-terminal Gly-Ser of <em>At</em>UBC8 remained after TEV cleavage). The mixture was incubated at 20 °C for 12 h. For AlexaFlor<sup>TM</sup> 594 labeling of Ub, the <em>N</em>-terminal Cys-Gly-Gly-Gly sequence was inserted by mutagenesis between the <em>N</em>-terminal His<sub>6</sub>-TEV and Ub sequences (C-UB). The <em>N</em>-terminal His<sub>6</sub>-tag was cleaved by TEV during purification. C-UB was mixed with AF594 C<sub>5</sub> maleimide (A10256; Invitrogen<sup>TM</sup>) at a 3:1 molar ratio and incubated at 20 °C for 12 h. Excess peptide, GST-sortase, and AF594 dye were removed by size-exclusion chromatography using a S75 Increase 10/300 GL column (29148721; Cytiva). The Fluorescence labeled E2~Ub conjugate was prepared by incubating 0.5 μM <em>h</em>UBA1, 5 μM FITC-<em>At</em>UBC8, 5.5 μM AF594-UB, 5 μM USP2 catalytic domain, 100 mM MgCl<sub>2</sub>, and 100 mM ATP in assay buffer (50 mM Tris-HCl pH 7.5 and 150 mM NaCl) for 30 min at 30 °C. The charging reaction was quenched after a 30-minute reaction time by ATP depletion. Discharge reactions were performed by incubating 125 nM E2~Ub mix, 10 μM unlabeled Ub, and 1 μM PRT1 variants (WT or 4 L/S mutant). Fluorescence signals were recorded in a 384-well black flat-bottom plate (3621; Corning<sup>TM</sup>, NY, US) using a SpectraMax iD5 microplate reader (Molecular Devices, CA, US) at 30 °C for 3 h, with 34-second intervals. The FRET data was obtained by exciting the donor fluorophore (FITC) at 475 nm and measuring emissions at 545 nm (<em>F</em><sub>D</sub>; donor intensity) and 600 nm (<em>F</em><sub>A</sub>; acceptor intensity). The raw FRET ratio (<em>F°</em>) of acceptor to donor intensity was calculated (1):</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e2160" display="block"><msup><mrow><mi>F</mi></mrow><mrow><mo>∘</mo></mrow></msup><mo>=</mo><mfrac><mrow><msub><mrow><mi>F</mi></mrow><mrow><mi>A</mi></mrow></msub></mrow><mrow><msub><mrow><mi>F</mi></mrow><mrow><mi>D</mi></mrow></msub></mrow></mfrac></math></td>
<td class="label">1</td>
</tr></table>
<p id="Par29">The decay in FRET signal was fitted to one-phase exponential decay Eq. (<a href="#Equ2" class="usa-link">2</a>):</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e2192" display="block"><msup><mrow><mi>F</mi></mrow><mrow><mo>∘</mo></mrow></msup><mo>=</mo><mrow><mo>(</mo><mrow><msub><mrow><msup><mrow><mi>F</mi></mrow><mrow><mo>∘</mo></mrow></msup></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi>P</mi><mi>l</mi><mi>a</mi><mi>t</mi><mi>e</mi><mi>a</mi><mi>u</mi></mrow><mo>)</mo></mrow><mo>×</mo><msup><mrow><mi>e</mi></mrow><mrow><mo>−</mo><mi>k</mi><mo>⋅</mo><mi>x</mi></mrow></msup></math></td>
<td class="label">2</td>
</tr></table>
<p>where <em>F</em>°<sub><em>0</em></sub> is the initial normalized FRET signal, Plateau represents the baseline signal after complete discharge or nonspecific decay, <em>k</em> is the decay rate constant, and <em>x</em> is time. To eliminate background signal reduction that occurs independently of PRT1 activity—such as spontaneous E2~Ub hydrolysis or photobleaching—we performed a normalization step using the no-PRT1 control. Fluorescence curves measured in the absence of PRT1 were used to define the baseline decay behavior. The following transformation was applied to correct for PRT1-independent decay and normalize the experimental curves (3):</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e2251" display="block"><msub><mrow><mi>F</mi></mrow><mrow><mi>n</mi></mrow></msub><mo>=</mo><mfrac><mrow><msup><mrow><mi>F</mi></mrow><mrow><mo>∘</mo></mrow></msup><mo>−</mo><mi>P</mi><mi>l</mi><mi>a</mi><mi>t</mi><mi>e</mi><mi>a</mi><mi>u</mi></mrow><mrow><msub><mrow><msup><mrow><mi>F</mi></mrow><mrow><mo>∘</mo></mrow></msup></mrow><mrow><mi>n</mi><mi>o</mi><mi>P</mi><mi>R</mi><mi>T</mi><mn>1</mn></mrow></msub><mo>−</mo><mi>P</mi><mi>l</mi><mi>a</mi><mi>t</mi><mi>e</mi><mi>a</mi><mi>u</mi></mrow></mfrac><mo>+</mo><mi>P</mi><mi>l</mi><mi>a</mi><mi>t</mi><mi>e</mi><mi>a</mi><mi>u</mi></math></td>
<td class="label">3</td>
</tr></table>
<p id="Par30">Here, <em>F°</em><sub><em>no PRT1</em></sub> refers to the FRET signal measured from the control reaction lacking PRT1 under identical conditions. This transformation rescales the data to PRT1-mediated Ub discharge, thereby enabling comparison between WT and mutant variants.</p>
<p id="Par31">The normalized FRET data (<em>F</em><sub><em>n</em></sub>) were refitted using the same exponential decay model (Eq. <a href="#Equ2" class="usa-link">2</a>), and the half-life (<em>t</em><sub><em>1/2</em></sub>) for each reaction condition was calculated using (4):</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e2336" display="block"><mfrac><mrow><mi>l</mi><mi>n</mi><mrow><mo>(</mo><mrow><mn>2</mn></mrow><mo>)</mo></mrow></mrow><mrow><mi>k</mi></mrow></mfrac></math></td>
<td class="label">4</td>
</tr></table>
<p id="Par32">The equations were modified based on methodologies established in prior experiments<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>. All reactions were performed in triplicate. Curve fitting and kinetic parameter extraction were conducted using nonlinear regression analysis in GraphPad Prism 5 software.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par33">Further information on research design is available in the <a href="#MOESM4" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370974/bin/41467_2025_63282_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (13MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370974/bin/41467_2025_63282_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63282_MOESM2_ESM.pdf</a><sup> (138.3KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370974/bin/41467_2025_63282_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (17KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370974/bin/41467_2025_63282_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (995.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12370974/bin/41467_2025_63282_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (865.5KB, pdf) </sup>
</div></div></section></section><section id="Sec25"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12370974/bin/41467_2025_63282_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (5.6MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the staff at beamlines 4 C, 5 C, and 11 C at the Pohang Accelerator Laboratory in South Korea, beamline BL44XU at SPring-8, and beamline BL-10C at the Photon Factory in Japan for their assistance with X-ray data collection. We appreciate Dr. Cy Jeffries at EMBL for his advice on SAXS data analysis. This study was partially supported by the International Collaborative Research Program of the Institute for Protein Research at Osaka University (ICR-22-05 and ICR-23-05). Diffraction data were collected at the Osaka University beamline BL44XU at SPring-8 (Harima, Japan) (proposal no. 2023A6873). This study was supported by the National Research Foundation of Korea (NRF) grants from the Korean government (Grant Nos. RS-2020-NR049540, RS-2021-NR056525, RS-2021-NR056577, RS-2022-NR067411, and RS-2025-00521009 to H.K.S.).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>W.S.Y., S.H.K., M.K., H.S., J.L., and A.S. performed biochemical experiments. W.S.Y. prepared the crystals, solved the structures, and conducted SAXS experiments. W. S. Y., O. K. P., N. D., and H. K. S. analyzed the data. W.S.Y. and H.K.S. designed the experiments and wrote the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par34"><em>Nature Communications</em> thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The atomic coordinates and structure factors were deposited in the Protein Data Bank under accession codes <a href="http://doi.org/10.2210/pdb8ZG8/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8ZG8</a> (apo), <a href="http://doi.org/10.2210/pdb8ZG9/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8ZG9</a> (Y-peptide complex), <a href="http://doi.org/10.2210/pdb8ZGA/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8ZGA</a> (F-peptide complex), and <a href="http://doi.org/10.2210/pdb8ZGB/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8ZGB</a> (W-peptide complex). We deposited our SEC-SAXS data at the Small Angle Scattering Biological Data Bank under accession code <a href="https://www.sasbdb.org/data/SASDWM2/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SASDWM2</a> (SAXS data for full-length PRT1). Data Source are provided with this paper. <a href="#Sec25" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par35">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec26"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63282-9.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Hershko, A. &amp; Ciechanover, A. The ubiquitin system. <em>Annu Rev. Biochem.</em><strong>67</strong>, 425–479 (1998).
</cite> [<a href="https://doi.org/10.1146/annurev.biochem.67.1.425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9759494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hershko,%20A.%20&amp;%20Ciechanover,%20A.%20The%20ubiquitin%20system.%20Annu%20Rev.%20Biochem.67,%20425%E2%80%93479%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Ciechanover, A. &amp; Schwartz, A. L. The ubiquitin-proteasome pathway: The complexity and myriad functions of proteins death. <em>Proc. Natl. Acad. Sci. USA</em><strong>95</strong>, 2727–2730 (1998).
</cite> [<a href="https://doi.org/10.1073/pnas.95.6.2727" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC34259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9501156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ciechanover,%20A.%20&amp;%20Schwartz,%20A.%20L.%20The%20ubiquitin-proteasome%20pathway:%20The%20complexity%20and%20myriad%20functions%20of%20proteins%20death.%20Proc.%20Natl.%20Acad.%20Sci.%20USA95,%202727%E2%80%932730%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Rechsteiner, M. Ubiquitin-mediated pathways for intracellular proteolysis. <em>Annu. Rev. Cell Biol.</em><strong>3</strong>, 1–30 (1987).
</cite> [<a href="https://doi.org/10.1146/annurev.cb.03.110187.000245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2825735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rechsteiner,%20M.%20Ubiquitin-mediated%20pathways%20for%20intracellular%20proteolysis.%20Annu.%20Rev.%20Cell%20Biol.3,%201%E2%80%9330%20(1987)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Coux, O., Tanaka, K. &amp; Goldberg, A. L. Structure and functions of the 20S and 26S proteasomes. <em>Annu. Rev. Biochem.</em><strong>65</strong>, 801–847 (1996).
</cite> [<a href="https://doi.org/10.1146/annurev.bi.65.070196.004101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8811196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coux,%20O.,%20Tanaka,%20K.%20&amp;%20Goldberg,%20A.%20L.%20Structure%20and%20functions%20of%20the%2020S%20and%2026S%20proteasomes.%20Annu.%20Rev.%20Biochem.65,%20801%E2%80%93847%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Bachmair, A., Finley, D. &amp; Varshavsky, A. In vivo half-life of a protein is a function of its amino-terminal residue. <em>Science</em><strong>234</strong>, 179–186 (1986).
</cite> [<a href="https://doi.org/10.1126/science.3018930" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3018930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bachmair,%20A.,%20Finley,%20D.%20&amp;%20Varshavsky,%20A.%20In%20vivo%20half-life%20of%20a%20protein%20is%20a%20function%20of%20its%20amino-terminal%20residue.%20Science234,%20179%E2%80%93186%20(1986)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Varshavsky, A. N-degron and C-degron pathways of protein degradation. <em>Proc. Natl. Acad. Sci. USA</em><strong>116</strong>, 358–366 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1816596116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6329975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30622213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varshavsky,%20A.%20N-degron%20and%20C-degron%20pathways%20of%20protein%20degradation.%20Proc.%20Natl.%20Acad.%20Sci.%20USA116,%20358%E2%80%93366%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Varshavsky, A. N-degron pathways. <em>Proc. Natl. Acad. Sci. USA</em><strong>121</strong>, e2408697121 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2408697121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11441550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39264755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varshavsky,%20A.%20N-degron%20pathways.%20Proc.%20Natl.%20Acad.%20Sci.%20USA121,%20e2408697121%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Bartel, B., Wünning, I. &amp; Varshavsky, A. The recognition component of the N-end rule pathway. <em> EMBO J.</em><strong>9</strong>, 3179–3189 (1990).
</cite> [<a href="https://doi.org/10.1002/j.1460-2075.1990.tb07516.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC552047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2209542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bartel,%20B.,%20W%C3%BCnning,%20I.%20&amp;%20Varshavsky,%20A.%20The%20recognition%20component%20of%20the%20N-end%20rule%20pathway.%20EMBO%20J.9,%203179%E2%80%933189%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Kim, B. H. et al. Crystal structure of the Ate1 arginyl-tRNA-protein transferase and arginylation of N-degron substrates. <em>Proc. Natl. Acad. Sci. USA</em><strong>119</strong>, e2209597119 (2022).
</cite> [<a href="https://doi.org/10.1073/pnas.2209597119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9351520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35878037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20B.%20H.%20et%20al.%20Crystal%20structure%20of%20the%20Ate1%20arginyl-tRNA-protein%20transferase%20and%20arginylation%20of%20N-degron%20substrates.%20Proc.%20Natl.%20Acad.%20Sci.%20USA119,%20e2209597119%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Kim, M. K., Oh, S. J., Lee, B. G. &amp; Song, H. K. Structural basis for dual specificity of yeast N-terminal amidase in the N-end rule pathway. <em>Proc. Natl. Acad. Sci. USA</em><strong>113</strong>, 12438–12443 (2016).
</cite> [<a href="https://doi.org/10.1073/pnas.1612620113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5098633/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27791147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20M.%20K.,%20Oh,%20S.%20J.,%20Lee,%20B.%20G.%20&amp;%20Song,%20H.%20K.%20Structural%20basis%20for%20dual%20specificity%20of%20yeast%20N-terminal%20amidase%20in%20the%20N-end%20rule%20pathway.%20Proc.%20Natl.%20Acad.%20Sci.%20USA113,%2012438%E2%80%9312443%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Oh, J. H., Hyun, J. Y., Chen, S. J. &amp; Varshavsky, A. Five enzymes of the Arg/N-degron pathway form a targeting complex: the concept of superchanneling. <em>Proc. Natl. Acad. Sci. USA</em><strong>117</strong>, 10778–10788 (2020).
</cite> [<a href="https://doi.org/10.1073/pnas.2003043117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7245096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32366662/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oh,%20J.%20H.,%20Hyun,%20J.%20Y.,%20Chen,%20S.%20J.%20&amp;%20Varshavsky,%20A.%20Five%20enzymes%20of%20the%20Arg/N-degron%20pathway%20form%20a%20targeting%20complex:%20the%20concept%20of%20superchanneling.%20Proc.%20Natl.%20Acad.%20Sci.%20USA117,%2010778%E2%80%9310788%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Hwang, C. S., Shemorry, A. &amp; Varshavsky, A. N-terminal acetylation of cellular proteins creates specific degradation signals. <em>Science</em><strong>327</strong>, 973–977 (2010).
</cite> [<a href="https://doi.org/10.1126/science.1183147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4259118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20110468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hwang,%20C.%20S.,%20Shemorry,%20A.%20&amp;%20Varshavsky,%20A.%20N-terminal%20acetylation%20of%20cellular%20proteins%20creates%20specific%20degradation%20signals.%20Science327,%20973%E2%80%93977%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Chen S. J., Wu X., Wadas B., Oh J. H. &amp; Varshavsky A. An N-end rule pathway that recognizes proline and destroys gluconeogenic enzymes. <em>Science</em><strong>355</strong>, eaal3655 (2017).</cite> [<a href="https://doi.org/10.1126/science.aal3655" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5457285/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28126757/" class="usa-link">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Chen S. J., Kim L., Song H. K. &amp; Varshavsky A. Aminopeptidases trim Xaa-Pro proteins, initiating their degradation by the Pro/N-degron pathway. <em>Proc. Natl. Acad. Sci. USA</em><strong>118</strong>, e2115430118 (2021).</cite> [<a href="https://doi.org/10.1073/pnas.2115430118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8639330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34663735/" class="usa-link">PubMed</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Timms R. T., et al. A glycine-specific N-degron pathway mediates the quality control of protein N-myristoylation. <em>Science</em><strong>365</strong>, eaaw4912 (2019).</cite> [<a href="https://doi.org/10.1126/science.aaw4912" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7090375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31273098/" class="usa-link">PubMed</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Kim J. M., et al. Formyl-methionine as an N-degron of a eukaryotic N-end rule pathway. <em>Science</em><strong>362</strong>, eaat0174 (2018).</cite> [<a href="https://doi.org/10.1126/science.aat0174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6551516/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30409808/" class="usa-link">PubMed</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Xia, Z. et al. Substrate-binding sites of UBR1, the ubiquitin ligase of the N-end rule pathway. <em>J. Biol. Chem.</em><strong>283</strong>, 24011–24028 (2008).
</cite> [<a href="https://doi.org/10.1074/jbc.M802583200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2527112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18566452/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia,%20Z.%20et%20al.%20Substrate-binding%20sites%20of%20UBR1,%20the%20ubiquitin%20ligase%20of%20the%20N-end%20rule%20pathway.%20J.%20Biol.%20Chem.283,%2024011%E2%80%9324028%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Varshavsky, A. The N-end rule pathway and regulation by proteolysis. <em>Protein Sci.</em><strong>20</strong>, 1298–1345 (2011).
</cite> [<a href="https://doi.org/10.1002/pro.666" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3189519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21633985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varshavsky,%20A.%20The%20N-end%20rule%20pathway%20and%20regulation%20by%20proteolysis.%20Protein%20Sci.20,%201298%E2%80%931345%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Choi, W. S. et al. Structural basis for the recognition of N-end rule substrates by the UBR box of ubiquitin ligases. <em>Nat. Struct. Mol. Biol.</em><strong>17</strong>, 1175–1181 (2010).
</cite> [<a href="https://doi.org/10.1038/nsmb.1907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20835240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choi,%20W.%20S.%20et%20al.%20Structural%20basis%20for%20the%20recognition%20of%20N-end%20rule%20substrates%20by%20the%20UBR%20box%20of%20ubiquitin%20ligases.%20Nat.%20Struct.%20Mol.%20Biol.17,%201175%E2%80%931181%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Matta-Camacho, E., Kozlov, G., Li, F. F. &amp; Gehring, K. Structural basis of substrate recognition and specificity in the N-end rule pathway. <em>Nat. Struct. Mol. Biol.</em><strong>17</strong>, 1182–1187 (2010).
</cite> [<a href="https://doi.org/10.1038/nsmb.1894" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20835242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matta-Camacho,%20E.,%20Kozlov,%20G.,%20Li,%20F.%20F.%20&amp;%20Gehring,%20K.%20Structural%20basis%20of%20substrate%20recognition%20and%20specificity%20in%20the%20N-end%20rule%20pathway.%20Nat.%20Struct.%20Mol.%20Biol.17,%201182%E2%80%931187%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Pan, M. et al. Structural insights into Ubr1-mediated N-degron polyubiquitination. <em>Nature</em><strong>600</strong>, 334–338 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-04097-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8798225/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34789879/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan,%20M.%20et%20al.%20Structural%20insights%20into%20Ubr1-mediated%20N-degron%20polyubiquitination.%20Nature600,%20334%E2%80%93338%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Tasaki, T., Sriram, S. M., Park, K. S. &amp; Kwon, Y. T. The N-end rule pathway. <em>Annu. Rev. Biochem.</em><strong>81</strong>, 261–289 (2012).
</cite> [<a href="https://doi.org/10.1146/annurev-biochem-051710-093308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3610525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22524314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tasaki,%20T.,%20Sriram,%20S.%20M.,%20Park,%20K.%20S.%20&amp;%20Kwon,%20Y.%20T.%20The%20N-end%20rule%20pathway.%20Annu.%20Rev.%20Biochem.81,%20261%E2%80%93289%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Garzón, M. et al. PRT6/At5g02310 encodes an Arabidopsis ubiquitin ligase of the N-end rule pathway with arginine specificity and is not the CER3 locus. <em>FEBS Lett.</em><strong>581</strong>, 3189–3196 (2007).
</cite> [<a href="https://doi.org/10.1016/j.febslet.2007.06.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17572409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garz%C3%B3n,%20M.%20et%20al.%20PRT6/At5g02310%20encodes%20an%20Arabidopsis%20ubiquitin%20ligase%20of%20the%20N-end%20rule%20pathway%20with%20arginine%20specificity%20and%20is%20not%20the%20CER3%20locus.%20FEBS%20Lett.581,%203189%E2%80%933196%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Potuschak, T. et al. PRT1 of Arabidopsis thaliana encodes a component of the plant N-end rule pathway. <em>Proc. Natl. Acad. Sci. USA</em><strong>95</strong>, 7904–7908 (1998).
</cite> [<a href="https://doi.org/10.1073/pnas.95.14.7904" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC20902/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9653113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Potuschak,%20T.%20et%20al.%20PRT1%20of%20Arabidopsis%20thaliana%20encodes%20a%20component%20of%20the%20plant%20N-end%20rule%20pathway.%20Proc.%20Natl.%20Acad.%20Sci.%20USA95,%207904%E2%80%937908%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Stary, S. et al. PRT1 of Arabidopsis is a ubiquitin protein ligase of the plant N-end rule pathway with specificity for aromatic amino-terminal residues. <em>Plant Physiol.</em><strong>133</strong>, 1360–1366 (2003).
</cite> [<a href="https://doi.org/10.1104/pp.103.029272" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC281630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14551326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stary,%20S.%20et%20al.%20PRT1%20of%20Arabidopsis%20is%20a%20ubiquitin%20protein%20ligase%20of%20the%20plant%20N-end%20rule%20pathway%20with%20specificity%20for%20aromatic%20amino-terminal%20residues.%20Plant%20Physiol.133,%201360%E2%80%931366%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Disch, S. et al. The E3 ubiquitin ligase BIG BROTHER controls Arabidopsis organ size in a dosage-dependent manner. <em>Curr. Biol.</em><strong>16</strong>, 272–279 (2006).
</cite> [<a href="https://doi.org/10.1016/j.cub.2005.12.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16461280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Disch,%20S.%20et%20al.%20The%20E3%20ubiquitin%20ligase%20BIG%20BROTHER%20controls%20Arabidopsis%20organ%20size%20in%20a%20dosage-dependent%20manner.%20Curr.%20Biol.16,%20272%E2%80%93279%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Dong, H. et al. Ubiquitylation activates a peptidase that promotes cleavage and destabilization of its activating E3 ligases and diverse growth regulatory proteins to limit cell proliferation in Arabidopsis. <em>Genes Dev.</em><strong>31</strong>, 197–208 (2017).
</cite> [<a href="https://doi.org/10.1101/gad.292235.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5322733/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28167503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong,%20H.%20et%20al.%20Ubiquitylation%20activates%20a%20peptidase%20that%20promotes%20cleavage%20and%20destabilization%20of%20its%20activating%20E3%20ligases%20and%20diverse%20growth%20regulatory%20proteins%20to%20limit%20cell%20proliferation%20in%20Arabidopsis.%20Genes%20Dev.31,%20197%E2%80%93208%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Mot, A. C. et al. Real-time detection of N-end rule-mediated ubiquitination via fluorescently labeled substrate probes. <em>N. Phytol.</em><strong>217</strong>, 613–624 (2018).</cite> [<a href="https://doi.org/10.1111/nph.14497" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5763331/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28277608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mot,%20A.%20C.%20et%20al.%20Real-time%20detection%20of%20N-end%20rule-mediated%20ubiquitination%20via%20fluorescently%20labeled%20substrate%20probes.%20N.%20Phytol.217,%20613%E2%80%93624%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kwon, D. H. et al. Insights into degradation mechanism of N-end rule substrates by p62/SQSTM1 autophagy adapter. <em>Nat. Commun.</em><strong>9</strong>, 3291 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-018-05825-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6098011/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30120248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kwon,%20D.%20H.%20et%20al.%20Insights%20into%20degradation%20mechanism%20of%20N-end%20rule%20substrates%20by%20p62/SQSTM1%20autophagy%20adapter.%20Nat.%20Commun.9,%203291%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Zhang, Y. et al. ZZ-dependent regulation of p62/SQSTM1 in autophagy. <em>Nat. Commun.</em><strong>9</strong>, 4373 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-018-06878-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6197226/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30349045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20ZZ-dependent%20regulation%20of%20p62/SQSTM1%20in%20autophagy.%20Nat.%20Commun.9,%204373%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Legge, G. B. et al. ZZ domain of CBP: an unusual zinc finger fold in a protein interaction module. <em>J. Mol. Biol.</em><strong>343</strong>, 1081–1093 (2004).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2004.08.087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15476823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Legge,%20G.%20B.%20et%20al.%20ZZ%20domain%20of%20CBP:%20an%20unusual%20zinc%20finger%20fold%20in%20a%20protein%20interaction%20module.%20J.%20Mol.%20Biol.343,%201081%E2%80%931093%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Mi, W. et al. The ZZ-type zinc finger of ZZZ3 modulates the ATAC complex-mediated histone acetylation and gene activation. <em>Nat. Commun.</em><strong>9</strong>, 3759 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-018-06247-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6138639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30217978/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mi,%20W.%20et%20al.%20The%20ZZ-type%20zinc%20finger%20of%20ZZZ3%20modulates%20the%20ATAC%20complex-mediated%20histone%20acetylation%20and%20gene%20activation.%20Nat.%20Commun.9,%203759%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>McMillan, B. rianJ. et al. A tail of two sites: a bipartite mechanism for recognition of notch ligands by mind bomb E3 ligases. <em>Mol. Cell</em><strong>57</strong>, 912–924 (2015).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2015.01.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4355479/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25747658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McMillan,%20B.%20rianJ.%20et%20al.%20A%20tail%20of%20two%20sites:%20a%20bipartite%20mechanism%20for%20recognition%20of%20notch%20ligands%20by%20mind%20bomb%20E3%20ligases.%20Mol.%20Cell57,%20912%E2%80%93924%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Liu, J. et al. Structural insight into binding of the ZZ domain of HERC2 to histone H3 and SUMO1. <em>Structure</em><strong>28</strong>, 1225–1230.e1223 (2020).
</cite> [<a href="https://doi.org/10.1016/j.str.2020.07.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11537069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32726574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20J.%20et%20al.%20Structural%20insight%20into%20binding%20of%20the%20ZZ%20domain%20of%20HERC2%20to%20histone%20H3%20and%20SUMO1.%20Structure28,%201225%E2%80%931230.e1223%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Zhang, Y. et al. The ZZ domain of p300 mediates specificity of the adjacent HAT domain for histone H3. <em>Nat. Struct. Mol. Biol.</em><strong>25</strong>, 841–849 (2018).
</cite> [<a href="https://doi.org/10.1038/s41594-018-0114-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6482957/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30150647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20The%20ZZ%20domain%20of%20p300%20mediates%20specificity%20of%20the%20adjacent%20HAT%20domain%20for%20histone%20H3.%20Nat.%20Struct.%20Mol.%20Biol.25,%20841%E2%80%93849%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Ponting, C. P., Blake, D. J., Davies, K. E., Kendrick-Jones, J. &amp; Winder, S. J. ZZ and TAZ: new putative zinc fingers in dystrophin and other proteins. <em>Trends Biochem. Sci.</em><strong>21</strong>, 11–13 (1996).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/8848831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ponting,%20C.%20P.,%20Blake,%20D.%20J.,%20Davies,%20K.%20E.,%20Kendrick-Jones,%20J.%20&amp;%20Winder,%20S.%20J.%20ZZ%20and%20TAZ:%20new%20putative%20zinc%20fingers%20in%20dystrophin%20and%20other%20proteins.%20Trends%20Biochem.%20Sci.21,%2011%E2%80%9313%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Kim, L., Kwon, D. H., Heo, J., Park, M. R. &amp; Song, H. K. Use of the LC3B-fusion technique for biochemical and structural studies of proteins involved in the N-degron pathway. <em>J. Biol. Chem.</em><strong>295</strong>, 2590–2600 (2020).
</cite> [<a href="https://doi.org/10.1074/jbc.RA119.010912" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7049975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31919097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20L.,%20Kwon,%20D.%20H.,%20Heo,%20J.,%20Park,%20M.%20R.%20&amp;%20Song,%20H.%20K.%20Use%20of%20the%20LC3B-fusion%20technique%20for%20biochemical%20and%20structural%20studies%20of%20proteins%20involved%20in%20the%20N-degron%20pathway.%20J.%20Biol.%20Chem.295,%202590%E2%80%932600%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Kim, B. H., Kwon, D. H., Kim, L., Yang, W. S. &amp; Song, H. K. In vitro production of N-degron fused proteins and its application. <em>Methods Enzymol.</em><strong>686</strong>, 99–123 (2023).
</cite> [<a href="https://doi.org/10.1016/bs.mie.2023.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37532410/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20B.%20H.,%20Kwon,%20D.%20H.,%20Kim,%20L.,%20Yang,%20W.%20S.%20&amp;%20Song,%20H.%20K.%20In%20vitro%20production%20of%20N-degron%20fused%20proteins%20and%20its%20application.%20Methods%20Enzymol.686,%2099%E2%80%93123%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Sigrist, C. J. et al. New and continuing developments at PROSITE. <em>Nucleic Acids Res.</em><strong>41</strong>, D344–D347 (2013).
</cite> [<a href="https://doi.org/10.1093/nar/gks1067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531220/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23161676/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sigrist,%20C.%20J.%20et%20al.%20New%20and%20continuing%20developments%20at%20PROSITE.%20Nucleic%20Acids%20Res.41,%20D344%E2%80%93D347%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Moududee, S. A. et al. Structural and functional characterization of hMEX-3C Ring finger domain as an E3 ubiquitin ligase. <em>Protein Sci.</em><strong>27</strong>, 1661–1669 (2018).
</cite> [<a href="https://doi.org/10.1002/pro.3473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6194269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30095198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moududee,%20S.%20A.%20et%20al.%20Structural%20and%20functional%20characterization%20of%20hMEX-3C%20Ring%20finger%20domain%20as%20an%20E3%20ubiquitin%20ligase.%20Protein%20Sci.27,%201661%E2%80%931669%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Khmelinskii, A. et al. Incomplete proteasomal degradation of green fluorescent proteins in the context of tandem fluorescent protein timers. <em>Mol. Biol. Cell</em><strong>27</strong>, 360–370 (2016).
</cite> [<a href="https://doi.org/10.1091/mbc.E15-07-0525" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4713137/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26609072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khmelinskii,%20A.%20et%20al.%20Incomplete%20proteasomal%20degradation%20of%20green%20fluorescent%20proteins%20in%20the%20context%20of%20tandem%20fluorescent%20protein%20timers.%20Mol.%20Biol.%20Cell27,%20360%E2%80%93370%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Varadi, M. et al. AlphaFold protein structure database: massively expanding the structural coverage of protein-sequence space with high-accuracy models. <em>Nucleic Acids Res.</em><strong>50</strong>, D439–D444 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkab1061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728224/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34791371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varadi,%20M.%20et%20al.%20AlphaFold%20protein%20structure%20database:%20massively%20expanding%20the%20structural%20coverage%20of%20protein-sequence%20space%20with%20high-accuracy%20models.%20Nucleic%20Acids%20Res.50,%20D439%E2%80%93D444%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <em>Nature</em><strong>596</strong>, 583–589 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jumper,%20J.%20et%20al.%20Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold.%20Nature596,%20583%E2%80%93589%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Sanchez, J. G. et al. Mechanism of TRIM25 catalytic activation in the antiviral RIG-I pathway. <em>Cell Rep.</em><strong>16</strong>, 1315–1325 (2016).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2016.06.070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5076470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27425606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanchez,%20J.%20G.%20et%20al.%20Mechanism%20of%20TRIM25%20catalytic%20activation%20in%20the%20antiviral%20RIG-I%20pathway.%20Cell%20Rep.16,%201315%E2%80%931325%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Anandapadamanaban, M. et al. E3 ubiquitin-protein ligase TRIM21-mediated lysine capture by UBE2E1 reveals substrate-targeting mode of a ubiquitin-conjugating E2. <em>J. Biol. Chem.</em><strong>294</strong>, 11404–11419 (2019).
</cite> [<a href="https://doi.org/10.1074/jbc.RA119.008485" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6663867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31160341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Anandapadamanaban,%20M.%20et%20al.%20E3%20ubiquitin-protein%20ligase%20TRIM21-mediated%20lysine%20capture%20by%20UBE2E1%20reveals%20substrate-targeting%20mode%20of%20a%20ubiquitin-conjugating%20E2.%20J.%20Biol.%20Chem.294,%2011404%E2%80%9311419%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Campbell, S. J. et al. Molecular insights into the function of RING finger (RNF)-containing proteins hRNF8 and hRNF168 in Ubc13/Mms2-dependent ubiquitylation. <em>J. Biol. Chem.</em><strong>287</strong>, 23900–23910 (2012).
</cite> [<a href="https://doi.org/10.1074/jbc.M112.359653" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3390666/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22589545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Campbell,%20S.%20J.%20et%20al.%20Molecular%20insights%20into%20the%20function%20of%20RING%20finger%20(RNF)-containing%20proteins%20hRNF8%20and%20hRNF168%20in%20Ubc13/Mms2-dependent%20ubiquitylation.%20J.%20Biol.%20Chem.287,%2023900%E2%80%9323910%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Yudina, Z. et al. RING dimerization links higher-order assembly of TRIM5alpha to synthesis of K63-linked polyubiquitin. <em>Cell Rep.</em><strong>12</strong>, 788–797 (2015).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2015.06.072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4526822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26212332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yudina,%20Z.%20et%20al.%20RING%20dimerization%20links%20higher-order%20assembly%20of%20TRIM5alpha%20to%20synthesis%20of%20K63-linked%20polyubiquitin.%20Cell%20Rep.12,%20788%E2%80%93797%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Keown, J. R., Yang, J., Black, M. M. &amp; Goldstone, D. C. The RING domain of TRIM69 promotes higher-order assembly. <em>Acta Crystallogr. D Struct. Biol.</em><strong>76</strong>, 954–961 (2020).
</cite> [<a href="https://doi.org/10.1107/S2059798320010499" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33021497/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keown,%20J.%20R.,%20Yang,%20J.,%20Black,%20M.%20M.%20&amp;%20Goldstone,%20D.%20C.%20The%20RING%20domain%20of%20TRIM69%20promotes%20higher-order%20assembly.%20Acta%20Crystallogr.%20D%20Struct.%20Biol.76,%20954%E2%80%93961%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Koliopoulos, M. G., Esposito, D., Christodoulou, E., Taylor, I. A. &amp; Rittinger, K. Functional role of TRIM E3 ligase oligomerization and regulation of catalytic activity. <em>EMBO J.</em><strong>35</strong>, 1204–1218 (2016).
</cite> [<a href="https://doi.org/10.15252/embj.201593741" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4864278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27154206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koliopoulos,%20M.%20G.,%20Esposito,%20D.,%20Christodoulou,%20E.,%20Taylor,%20I.%20A.%20&amp;%20Rittinger,%20K.%20Functional%20role%20of%20TRIM%20E3%20ligase%20oligomerization%20and%20regulation%20of%20catalytic%20activity.%20EMBO%20J.35,%201204%E2%80%931218%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Park, S. H. et al. Structure and activation of the RING E3 ubiquitin ligase TRIM72 on the membrane. <em>Nat. Struct. Mol. Biol.</em><strong>30</strong>, 1695–1706 (2023).
</cite> [<a href="https://doi.org/10.1038/s41594-023-01111-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10643145/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37770719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Park,%20S.%20H.%20et%20al.%20Structure%20and%20activation%20of%20the%20RING%20E3%20ubiquitin%20ligase%20TRIM72%20on%20the%20membrane.%20Nat.%20Struct.%20Mol.%20Biol.30,%201695%E2%80%931706%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Brzovic, P. S., Rajagopal, P., Hoyt, D. W., King, M. C. &amp; Klevit, R. E. Structure of a BRCA1-BARD1 heterodimeric RING-RING complex. <em>Nat. Struct. Biol.</em><strong>8</strong>, 833–837 (2001).
</cite> [<a href="https://doi.org/10.1038/nsb1001-833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11573085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brzovic,%20P.%20S.,%20Rajagopal,%20P.,%20Hoyt,%20D.%20W.,%20King,%20M.%20C.%20&amp;%20Klevit,%20R.%20E.%20Structure%20of%20a%20BRCA1-BARD1%20heterodimeric%20RING-RING%20complex.%20Nat.%20Struct.%20Biol.8,%20833%E2%80%93837%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Shukla, S. et al. Small-molecule inhibitors targeting polycomb repressive complex 1 RING domain. <em>Nat. Chem. Biol.</em><strong>17</strong>, 784–793 (2021).
</cite> [<a href="https://doi.org/10.1038/s41589-021-00815-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8238916/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34155404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shukla,%20S.%20et%20al.%20Small-molecule%20inhibitors%20targeting%20polycomb%20repressive%20complex%201%20RING%20domain.%20Nat.%20Chem.%20Biol.17,%20784%E2%80%93793%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Fiorentini, F., Esposito, D. &amp; Rittinger, K. Does it take two to tango? RING domain self-association and activity in TRIM E3 ubiquitin ligases. <em>Biochem. Soc. Trans.</em><strong>48</strong>, 2615–2624 (2020).
</cite> [<a href="https://doi.org/10.1042/BST20200383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7752041/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33170204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fiorentini,%20F.,%20Esposito,%20D.%20&amp;%20Rittinger,%20K.%20Does%20it%20take%20two%20to%20tango?%20RING%20domain%20self-association%20and%20activity%20in%20TRIM%20E3%20ubiquitin%20ligases.%20Biochem.%20Soc.%20Trans.48,%202615%E2%80%932624%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Kiss, L., Clift, D., Renner, N., Neuhaus, D. &amp; James, L. C. RING domains act as both substrate and enzyme in a catalytic arrangement to drive self-anchored ubiquitination. <em>Nat. Commun.</em><strong>12</strong>, 1220 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-21443-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7900206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33619271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kiss,%20L.,%20Clift,%20D.,%20Renner,%20N.,%20Neuhaus,%20D.%20&amp;%20James,%20L.%20C.%20RING%20domains%20act%20as%20both%20substrate%20and%20enzyme%20in%20a%20catalytic%20arrangement%20to%20drive%20self-anchored%20ubiquitination.%20Nat.%20Commun.12,%201220%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Cha-Molstad, H. et al. p62/SQSTM1/Sequestosome-1 is an N-recognin of the N-end rule pathway which modulates autophagosome biogenesis. <em>Nat. Commun.</em><strong>8</strong>, 102 (2017).
</cite> [<a href="https://doi.org/10.1038/s41467-017-00085-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5524641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28740232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cha-Molstad,%20H.%20et%20al.%20p62/SQSTM1/Sequestosome-1%20is%20an%20N-recognin%20of%20the%20N-end%20rule%20pathway%20which%20modulates%20autophagosome%20biogenesis.%20Nat.%20Commun.8,%20102%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Stone, S. L. et al. Functional analysis of the RING-type ubiquitin ligase family of Arabidopsis. <em>Plant Physiol.</em><strong>137</strong>, 13–30 (2005).
</cite> [<a href="https://doi.org/10.1104/pp.104.052423" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC548835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15644464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stone,%20S.%20L.%20et%20al.%20Functional%20analysis%20of%20the%20RING-type%20ubiquitin%20ligase%20family%20of%20Arabidopsis.%20Plant%20Physiol.137,%2013%E2%80%9330%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Fletcher, A. J. et al. Trivalent RING assembly on retroviral capsids activates TRIM5 ubiquitination and innate immune signaling. <em>Cell Host Microbe</em><strong>24</strong>, 761–775 e766 (2018).
</cite> [<a href="https://doi.org/10.1016/j.chom.2018.10.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6299210/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30503508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fletcher,%20A.%20J.%20et%20al.%20Trivalent%20RING%20assembly%20on%20retroviral%20capsids%20activates%20TRIM5%20ubiquitination%20and%20innate%20immune%20signaling.%20Cell%20Host%20Microbe24,%20761%E2%80%93775%20e766%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Kim, L. et al. Structural basis for the N-degron specificity of ClpS1 from Arabidopsis thaliana. <em>Protein Sci.</em><strong>30</strong>, 700–708 (2021).
</cite> [<a href="https://doi.org/10.1002/pro.4018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7888574/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33368743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20L.%20et%20al.%20Structural%20basis%20for%20the%20N-degron%20specificity%20of%20ClpS1%20from%20Arabidopsis%20thaliana.%20Protein%20Sci.30,%20700%E2%80%93708%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Kim L., et al. Structural analyses of the plant PRT6-UBR box in the Cys-Arg/N-degron pathway and insights into the plant submergence response. <em>bioRxiv</em>, 2022.2008.2019.504472 (2022).</cite>
</li>
<li id="CR60">
<span class="label">60.</span><cite>Rudi, O. et al. The UBR domain of plant Ubr1 homolog PRT6 accommodates basic and hydrophobic amino termini for substrate recognition. <em>J. Mol. Biol.</em><strong>437</strong>, 168939 (2025).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2025.168939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39799992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rudi,%20O.%20et%20al.%20The%20UBR%20domain%20of%20plant%20Ubr1%20homolog%20PRT6%20accommodates%20basic%20and%20hydrophobic%20amino%20termini%20for%20substrate%20recognition.%20J.%20Mol.%20Biol.437,%20168939%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Lee, B.-G., Kim, M. K., Kim, B.-W., Suh, S. W. &amp; Song, H. K. Structures of the ribosome-inactivating protein from barley seeds reveal a unique activation mechanism. <em>Acta Crystallogr. Sect. D</em><strong>68</strong>, 1488–1500 (2012).
</cite> [<a href="https://doi.org/10.1107/S0907444912037110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23090398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20B.-G.,%20Kim,%20M.%20K.,%20Kim,%20B.-W.,%20Suh,%20S.%20W.%20&amp;%20Song,%20H.%20K.%20Structures%20of%20the%20ribosome-inactivating%20protein%20from%20barley%20seeds%20reveal%20a%20unique%20activation%20mechanism.%20Acta%20Crystallogr.%20Sect.%20D68,%201488%E2%80%931500%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Kwon, D. H. et al. The 1:2 complex between RavZ and LC3 reveals a mechanism for deconjugation of LC3 on the phagophore membrane. <em>Autophagy</em><strong>13</strong>, 70–81 (2017).
</cite> [<a href="https://doi.org/10.1080/15548627.2016.1243199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5240826/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27791457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kwon,%20D.%20H.%20et%20al.%20The%201:2%20complex%20between%20RavZ%20and%20LC3%20reveals%20a%20mechanism%20for%20deconjugation%20of%20LC3%20on%20the%20phagophore%20membrane.%20Autophagy13,%2070%E2%80%9381%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Gu, D.-H. et al. BL-11C micro-MX: a high-flux microfocus macromolecular-crystallography beamline for micrometre-sized protein crystals at Pohang light source II. <em>J. Synchrotron Radiat.</em><strong>28</strong>, 1210–1215 (2021).
</cite> [<a href="https://doi.org/10.1107/S1600577521004355" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8284399/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34212886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gu,%20D.-H.%20et%20al.%20BL-11C%20micro-MX:%20a%20high-flux%20microfocus%20macromolecular-crystallography%20beamline%20for%20micrometre-sized%20protein%20crystals%20at%20Pohang%20light%20source%20II.%20J.%20Synchrotron%20Radiat.28,%201210%E2%80%931215%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Higashiura, A. et al. SPring-8 BL44XU, beamline designed for structure analysis of large biological macromolecular assemblies. <em>AIP Conf. Proc.</em><strong>1741</strong>, 030028 (2016).</cite>
</li>
<li id="CR65">
<span class="label">65.</span><cite>Liebschner, D. et al. Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix. <em>Acta Crystallogr. Sect. D</em><strong>75</strong>, 861–877 (2019).</cite> [<a href="https://doi.org/10.1107/S2059798319011471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6778852/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31588918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liebschner,%20D.%20et%20al.%20Macromolecular%20structure%20determination%20using%20X-rays,%20neutrons%20and%20electrons:%20recent%20developments%20in%20Phenix.%20Acta%20Crystallogr.%20Sect.%20D75,%20861%E2%80%93877%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Emsley, P., Lohkamp, B., Scott, W. G. &amp; Cowtan, K. Features and development of Coot. <em>Acta Crystallogr. D Biol. Crystallogr.</em><strong>66</strong>, 486–501 (2010).
</cite> [<a href="https://doi.org/10.1107/S0907444910007493" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2852313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20383002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Emsley,%20P.,%20Lohkamp,%20B.,%20Scott,%20W.%20G.%20&amp;%20Cowtan,%20K.%20Features%20and%20development%20of%20Coot.%20Acta%20Crystallogr.%20D%20Biol.%20Crystallogr.66,%20486%E2%80%93501%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Prisant, M. G., Williams, C. J., Chen, V. B., Richardson, J. S. &amp; Richardson, D. C. New tools in MolProbity validation: CaBLAM for CryoEM backbone, UnDowser to rethink “waters,” and NGL viewer to recapture online 3D graphics. <em>Protein Sci.</em><strong>29</strong>, 315–329 (2020).
</cite> [<a href="https://doi.org/10.1002/pro.3786" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6933861/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31724275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prisant,%20M.%20G.,%20Williams,%20C.%20J.,%20Chen,%20V.%20B.,%20Richardson,%20J.%20S.%20&amp;%20Richardson,%20D.%20C.%20New%20tools%20in%20MolProbity%20validation:%20CaBLAM%20for%20CryoEM%20backbone,%20UnDowser%20to%20rethink%20%E2%80%9Cwaters,%E2%80%9D%20and%20NGL%20viewer%20to%20recapture%20online%203D%20graphics.%20Protein%20Sci.29,%20315%E2%80%93329%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Shimizu N. et al. Software development for analysis of small-angle x-ray scattering data. <em>AIP Conf. Proc.</em><strong>1741</strong>, 050017 (2016).</cite>
</li>
<li id="CR69">
<span class="label">69.</span><cite>Panjkovich, A. &amp; Svergun, D. I. CHROMIXS: automatic and interactive analysis of chromatography-coupled small-angle X-ray scattering data. <em>Bioinformatics</em><strong>34</strong>, 1944–1946 (2017).</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx846" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5972624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29300836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Panjkovich,%20A.%20&amp;%20Svergun,%20D.%20I.%20CHROMIXS:%20automatic%20and%20interactive%20analysis%20of%20chromatography-coupled%20small-angle%20X-ray%20scattering%20data.%20Bioinformatics34,%201944%E2%80%931946%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Manalastas-Cantos, K. et al. ATSAS 3.0: expanded functionality and new tools for small-angle scattering data analysis. <em>J. Appl. Crystallogr.</em><strong>54</strong>, 343–355 (2021).
</cite> [<a href="https://doi.org/10.1107/S1600576720013412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7941305/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33833657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manalastas-Cantos,%20K.%20et%20al.%20ATSAS%203.0:%20expanded%20functionality%20and%20new%20tools%20for%20small-angle%20scattering%20data%20analysis.%20J.%20Appl.%20Crystallogr.54,%20343%E2%80%93355%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Svergun, D. Determination of the regularization parameter in indirect-transform methods using perceptual criteria. <em>J. Appl. Crystallogr.</em><strong>25</strong>, 495–503 (1992).</cite> [<a href="https://scholar.google.com/scholar_lookup?Svergun,%20D.%20Determination%20of%20the%20regularization%20parameter%20in%20indirect-transform%20methods%20using%20perceptual%20criteria.%20J.%20Appl.%20Crystallogr.25,%20495%E2%80%93503%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Franke, D. &amp; Svergun, D. I. DAMMIF, a program for rapid ab-initio shape determination in small-angle scattering. <em>J. Appl. Crystallogr.</em><strong>42</strong>, 342–346 (2009).
</cite> [<a href="https://doi.org/10.1107/S0021889809000338" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5023043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27630371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Franke,%20D.%20&amp;%20Svergun,%20D.%20I.%20DAMMIF,%20a%20program%20for%20rapid%20ab-initio%20shape%20determination%20in%20small-angle%20scattering.%20J.%20Appl.%20Crystallogr.42,%20342%E2%80%93346%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Svergun, D. I. Restoring low resolution structure of biological macromolecules from solution scattering using simulated annealing. <em>Biophys. J.</em><strong>76</strong>, 2879–2886 (1999).
</cite> [<a href="https://doi.org/10.1016/S0006-3495(99)77443-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1300260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10354416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Svergun,%20D.%20I.%20Restoring%20low%20resolution%20structure%20of%20biological%20macromolecules%20from%20solution%20scattering%20using%20simulated%20annealing.%20Biophys.%20J.76,%202879%E2%80%932886%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Goddard, T. D. et al. UCSF ChimeraX: meeting modern challenges in visualization and analysis. <em>Protein Sci.</em><strong>27</strong>, 14–25 (2018).
</cite> [<a href="https://doi.org/10.1002/pro.3235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5734306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28710774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goddard,%20T.%20D.%20et%20al.%20UCSF%20ChimeraX:%20meeting%20modern%20challenges%20in%20visualization%20and%20analysis.%20Protein%20Sci.27,%2014%E2%80%9325%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Franke, D. et al. ATSAS 2.8: a comprehensive data analysis suite for small-angle scattering from macromolecular solutions. <em>J. Appl. Crystallogr.</em><strong>50</strong>, 1212–1225 (2017).
</cite> [<a href="https://doi.org/10.1107/S1600576717007786" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5541357/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28808438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Franke,%20D.%20et%20al.%20ATSAS%202.8:%20a%20comprehensive%20data%20analysis%20suite%20for%20small-angle%20scattering%20from%20macromolecular%20solutions.%20J.%20Appl.%20Crystallogr.50,%201212%E2%80%931225%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Kikhney, A. G., Borges, C. R., Molodenskiy, D. S., Jeffries, C. M. &amp; Svergun, D. I. SASBDB: Towards an automatically curated and validated repository for biological scattering data. <em>Protein Sci.</em><strong>29</strong>, 66–75 (2020).
</cite> [<a href="https://doi.org/10.1002/pro.3731" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6933840/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31576635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kikhney,%20A.%20G.,%20Borges,%20C.%20R.,%20Molodenskiy,%20D.%20S.,%20Jeffries,%20C.%20M.%20&amp;%20Svergun,%20D.%20I.%20SASBDB:%20Towards%20an%20automatically%20curated%20and%20validated%20repository%20for%20biological%20scattering%20data.%20Protein%20Sci.29,%2066%E2%80%9375%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Trewhella, J. et al. 2017 publication guidelines for structural modelling of small-angle scattering data from biomolecules in solution: an update. <em>Acta Crystallogr. D. Struct. Biol.</em><strong>73</strong>, 710–728 (2017).
</cite> [<a href="https://doi.org/10.1107/S2059798317011597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5586245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28876235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trewhella,%20J.%20et%20al.%202017%20publication%20guidelines%20for%20structural%20modelling%20of%20small-angle%20scattering%20data%20from%20biomolecules%20in%20solution:%20an%20update.%20Acta%20Crystallogr.%20D.%20Struct.%20Biol.73,%20710%E2%80%93728%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Trewhella, J., Jeffries, C. M. &amp; Whitten, A. E. 2023 update of template tables for reporting biomolecular structural modelling of small-angle scattering data. <em>Acta Crystallogr. D. Struct. Biol.</em><strong>79</strong>, 122–132 (2023).
</cite> [<a href="https://doi.org/10.1107/S2059798322012141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9912924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36762858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trewhella,%20J.,%20Jeffries,%20C.%20M.%20&amp;%20Whitten,%20A.%20E.%202023%20update%20of%20template%20tables%20for%20reporting%20biomolecular%20structural%20modelling%20of%20small-angle%20scattering%20data.%20Acta%20Crystallogr.%20D.%20Struct.%20Biol.79,%20122%E2%80%93132%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Theile, C. S. et al. Site-specific N-terminal labeling of proteins using sortase-mediated reactions. <em>Nat. Protoc.</em><strong>8</strong>, 1800–1807 (2013).
</cite> [<a href="https://doi.org/10.1038/nprot.2013.102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3941705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23989674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Theile,%20C.%20S.%20et%20al.%20Site-specific%20N-terminal%20labeling%20of%20proteins%20using%20sortase-mediated%20reactions.%20Nat.%20Protoc.8,%201800%E2%80%931807%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Herkules, F. et al. Structural and functional asymmetry of RING trimerization controls priming and extension events in TRIM5alpha autoubiquitylation. <em>Nat. Commun.</em><strong>13</strong>, 7104 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-34920-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9675739/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36402777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herkules,%20F.%20et%20al.%20Structural%20and%20functional%20asymmetry%20of%20RING%20trimerization%20controls%20priming%20and%20extension%20events%20in%20TRIM5alpha%20autoubiquitylation.%20Nat.%20Commun.13,%207104%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370974/bin/41467_2025_63282_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (13MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370974/bin/41467_2025_63282_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63282_MOESM2_ESM.pdf</a><sup> (138.3KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370974/bin/41467_2025_63282_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (17KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370974/bin/41467_2025_63282_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (995.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370974/bin/41467_2025_63282_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (865.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370974/bin/41467_2025_63282_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (5.6MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The atomic coordinates and structure factors were deposited in the Protein Data Bank under accession codes <a href="http://doi.org/10.2210/pdb8ZG8/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8ZG8</a> (apo), <a href="http://doi.org/10.2210/pdb8ZG9/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8ZG9</a> (Y-peptide complex), <a href="http://doi.org/10.2210/pdb8ZGA/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8ZGA</a> (F-peptide complex), and <a href="http://doi.org/10.2210/pdb8ZGB/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8ZGB</a> (W-peptide complex). We deposited our SEC-SAXS data at the Small Angle Scattering Biological Data Bank under accession code <a href="https://www.sasbdb.org/data/SASDWM2/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SASDWM2</a> (SAXS data for full-length PRT1). Data Source are provided with this paper. <a href="#Sec25" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63282-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63282.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370974/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370974/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370974%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370974/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370974/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370974/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841552/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370974/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841552/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370974/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370974/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="rFHBrHmMiolXILQCMABARmxAerBK7YpepLnpxs8OjXrKZmFU8mAwOqopbw8OGIpg">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
