
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Characterization of the complete mitochondrial genome of Pennisetum giganteum A. Rich. (Poaceae) - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5312FD8AEF2EE30512FD003A204FA3.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="mdres">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377103/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Mitochondrial DNA. Part B, Resources">
<meta name="citation_title" content="Characterization of the complete mitochondrial genome of Pennisetum giganteum A. Rich. (Poaceae)">
<meta name="citation_author" content="Man Zhang">
<meta name="citation_author_institution" content="Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, China">
<meta name="citation_author_institution" content="State Key Laboratory of High-Efficiency Production of Wheat-Maize Double Cropping, Xinxiang, China">
<meta name="citation_author_institution" content="Henan Key Laboratory of Hybrid Wheat, Xinxiang, China">
<meta name="citation_author" content="Xiaojun Wu">
<meta name="citation_author_institution" content="Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, China">
<meta name="citation_author_institution" content="State Key Laboratory of High-Efficiency Production of Wheat-Maize Double Cropping, Xinxiang, China">
<meta name="citation_author_institution" content="Henan Key Laboratory of Hybrid Wheat, Xinxiang, China">
<meta name="citation_author" content="Kaiqiang Fang">
<meta name="citation_author_institution" content="Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, China">
<meta name="citation_author_institution" content="State Key Laboratory of High-Efficiency Production of Wheat-Maize Double Cropping, Xinxiang, China">
<meta name="citation_author_institution" content="Henan Key Laboratory of Hybrid Wheat, Xinxiang, China">
<meta name="citation_author" content="Haobo Wang">
<meta name="citation_author_institution" content="Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, China">
<meta name="citation_author_institution" content="State Key Laboratory of High-Efficiency Production of Wheat-Maize Double Cropping, Xinxiang, China">
<meta name="citation_author_institution" content="Henan Key Laboratory of Hybrid Wheat, Xinxiang, China">
<meta name="citation_author" content="Junchao You">
<meta name="citation_author_institution" content="Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, China">
<meta name="citation_author_institution" content="State Key Laboratory of High-Efficiency Production of Wheat-Maize Double Cropping, Xinxiang, China">
<meta name="citation_author_institution" content="Henan Key Laboratory of Hybrid Wheat, Xinxiang, China">
<meta name="citation_author" content="Xiangdong Chen">
<meta name="citation_author_institution" content="Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, China">
<meta name="citation_author_institution" content="State Key Laboratory of High-Efficiency Production of Wheat-Maize Double Cropping, Xinxiang, China">
<meta name="citation_author_institution" content="Henan Key Laboratory of Hybrid Wheat, Xinxiang, China">
<meta name="citation_publication_date" content="2025 Aug 23">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="868">
<meta name="citation_doi" content="10.1080/23802359.2025.2550607">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377103/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377103/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377103/pdf/TMDN_10_2550607.pdf">
<meta name="description" content="The complete mitochondrial genome of Pennisetum giganteum, an African species, was sequenced and characterized for the first time. The genome is 491,311 bp long and contains 53 unique genes. Phylogenetic analysis based on these genes using maximum ...">
<meta name="og:title" content="Characterization of the complete mitochondrial genome of Pennisetum giganteum A. Rich. (Poaceae)">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The complete mitochondrial genome of Pennisetum giganteum, an African species, was sequenced and characterized for the first time. The genome is 491,311 bp long and contains 53 unique genes. Phylogenetic analysis based on these genes using maximum ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377103/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12377103">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1080/23802359.2025.2550607"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TMDN_10_2550607.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377103%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12377103/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12377103/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377103/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-mdres.png" alt="Mitochondrial DNA. Part B, Resources logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Mitochondrial DNA. Part B, Resources" title="Link to Mitochondrial DNA. Part B, Resources" shape="default" href="https://doi.org/10.1080/23802359.2025.2550607" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Mitochondrial DNA B Resour</button></div>. 2025 Aug 23;10(9):868–873. doi: <a href="https://doi.org/10.1080/23802359.2025.2550607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1080/23802359.2025.2550607</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Mitochondrial%20DNA%20B%20Resour%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mitochondrial%20DNA%20B%20Resour%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Mitochondrial%20DNA%20B%20Resour%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Mitochondrial%20DNA%20B%20Resour%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Characterization of the complete mitochondrial genome of <em>Pennisetum giganteum</em> A. Rich. (Poaceae)</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Man Zhang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Man Zhang</span></h3>
<div class="p">
<sup><sup>a</sup></sup>Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, China</div>
<div class="p">
<sup><sup>b</sup></sup>State Key Laboratory of High-Efficiency Production of Wheat-Maize Double Cropping, Xinxiang, China</div>
<div class="p">
<sup><sup>c</sup></sup>Henan Key Laboratory of Hybrid Wheat, Xinxiang, China</div>
<div>Conceptualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Man Zhang</span></a>
</div>
</div>
<sup>a,</sup><sup>b,</sup><sup>c</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Xiaojun Wu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Xiaojun Wu</span></h3>
<div class="p">
<sup><sup>a</sup></sup>Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, China</div>
<div class="p">
<sup><sup>b</sup></sup>State Key Laboratory of High-Efficiency Production of Wheat-Maize Double Cropping, Xinxiang, China</div>
<div class="p">
<sup><sup>c</sup></sup>Henan Key Laboratory of Hybrid Wheat, Xinxiang, China</div>
<div>Conceptualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaojun Wu</span></a>
</div>
</div>
<sup>a,</sup><sup>b,</sup><sup>c</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fang%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Kaiqiang Fang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Kaiqiang Fang</span></h3>
<div class="p">
<sup><sup>a</sup></sup>Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, China</div>
<div class="p">
<sup><sup>b</sup></sup>State Key Laboratory of High-Efficiency Production of Wheat-Maize Double Cropping, Xinxiang, China</div>
<div class="p">
<sup><sup>c</sup></sup>Henan Key Laboratory of Hybrid Wheat, Xinxiang, China</div>
<div>Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fang%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kaiqiang Fang</span></a>
</div>
</div>
<sup>a,</sup><sup>b,</sup><sup>c</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Haobo Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Haobo Wang</span></h3>
<div class="p">
<sup><sup>a</sup></sup>Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, China</div>
<div class="p">
<sup><sup>b</sup></sup>State Key Laboratory of High-Efficiency Production of Wheat-Maize Double Cropping, Xinxiang, China</div>
<div class="p">
<sup><sup>c</sup></sup>Henan Key Laboratory of Hybrid Wheat, Xinxiang, China</div>
<div>Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haobo Wang</span></a>
</div>
</div>
<sup>a,</sup><sup>b,</sup><sup>c</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22You%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Junchao You</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Junchao You</span></h3>
<div class="p">
<sup><sup>a</sup></sup>Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, China</div>
<div class="p">
<sup><sup>b</sup></sup>State Key Laboratory of High-Efficiency Production of Wheat-Maize Double Cropping, Xinxiang, China</div>
<div class="p">
<sup><sup>c</sup></sup>Henan Key Laboratory of Hybrid Wheat, Xinxiang, China</div>
<div>Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22You%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junchao You</span></a>
</div>
</div>
<sup>a,</sup><sup>b,</sup><sup>c</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Xiangdong Chen</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Xiangdong Chen</span></h3>
<div class="p">
<sup><sup>a</sup></sup>Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, China</div>
<div class="p">
<sup><sup>b</sup></sup>State Key Laboratory of High-Efficiency Production of Wheat-Maize Double Cropping, Xinxiang, China</div>
<div class="p">
<sup><sup>c</sup></sup>Henan Key Laboratory of Hybrid Wheat, Xinxiang, China</div>
<div>Conceptualization, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiangdong Chen</span></a>
</div>
</div>
<sup>a,</sup><sup>b,</sup><sup>c,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="AF0001">
<sup><sup>a</sup></sup>Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, China</div>
<div id="AF0002">
<sup><sup>b</sup></sup>State Key Laboratory of High-Efficiency Production of Wheat-Maize Double Cropping, Xinxiang, China</div>
<div id="AF0003">
<sup><sup>c</sup></sup>Henan Key Laboratory of Hybrid Wheat, Xinxiang, China</div>
<div class="author-notes p">
<div class="fn" id="AUFN1"><p>Supplemental data for this article can be accessed online at <a href="https://doi.org/10.1080/23802359.2025.2550607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1080/23802359.2025.2550607</a>.</p></div>
<div class="fn" id="AN0001">
<sup>✉</sup><p class="display-inline">CONTACT Xiangdong Chen <span>xiangdc2020@126.com</span>
Wheat Research Center, Henan Institute of Science and Technology, Xinxiang, Henan 453000, China</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Man Zhang</span></strong>: <span class="role">Conceptualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xiaojun Wu</span></strong>: <span class="role">Conceptualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Kaiqiang Fang</span></strong>: <span class="role">Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Haobo Wang</span></strong>: <span class="role">Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Junchao You</span></strong>: <span class="role">Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xiangdong Chen</span></strong>: <span class="role">Conceptualization, Formal analysis</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 17; Accepted 2025 Aug 15; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). Published by Informa UK Limited, trading as Taylor &amp; Francis Group.</div>
<p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. The terms on which this article has been published allow the posting of the Accepted Manuscript in a repository by the author(s) or with their consent.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12377103</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The complete mitochondrial genome of <em>Pennisetum giganteum</em>, an African species, was sequenced and characterized for the first time. The genome is 491,311 bp long and contains 53 unique genes. Phylogenetic analysis based on these genes using maximum likelihood revealed that <em>P. giganteum</em> is most closely related to Saccharum officinarum and most distantly related to <em>Oryza sativa</em>. This study provides essential genomic data that enhances the understanding of the taxonomy and evolutionary relationships of this versatile and high-yielding species.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> Poaceae, C<sub>4</sub> plant, mitogenomics, mitochondrial genome assembly, phylogenomics, Panicoideae</p></section></section><section id="S0001"><h2 class="pmc_sec_title">Introduction</h2>
<p><em>Pennisetum giganteum</em> A. Rich. (Poaceae) is a perennial C<sub>4</sub> grass native to Africa, widely distributed in tropical to subtropical regions due to its adaptability to warm climates (Fulkerson et al. <a href="#CIT0013" class="usa-link" aria-describedby="CIT0013"><sup>2007</sup></a>; Xing et al. <a href="#CIT0035" class="usa-link" aria-describedby="CIT0035"><sup>2023</sup></a>). As a member of the genus <em>Pennisetum</em>, <em>P. giganteum</em> has significant ecological and agricultural value, noted for rapid growth, high biomass yield, and applications in sustainable agriculture (Samson et al. <a href="#CIT0026" class="usa-link" aria-describedby="CIT0026"><sup>2005</sup></a>; Hayat et al. <a href="#CIT0017" class="usa-link" aria-describedby="CIT0017"><sup>2020</sup></a>). However, genomic resources for <em>P. giganteum</em> remain scarce, particularly for organellar genomes. While chloroplast data exist for close relatives like <em>Pennisetum glaucum</em> (Xu et al. <a href="#CIT0036" class="usa-link" aria-describedby="CIT0036"><sup>2019</sup></a>), no mitochondrial genome has been reported for any <em>Pennisetum</em> species (Zheng et al. <a href="#CIT0040" class="usa-link" aria-describedby="CIT0040"><sup>2023</sup></a>).</p>
<p>Mitochondrial genomes are pivotal for resolving plant phylogenies due to structural complexity and variable evolutionary rates (Zardoya <a href="#CIT0038" class="usa-link" aria-describedby="CIT0038"><sup>2020</sup></a>). In <em>Pennisetum</em>, taxonomic relationships are complicated by hybridization and polyploidization (Dos Reis et al. <a href="#CIT0011" class="usa-link" aria-describedby="CIT0011"><sup>2014</sup></a>; Xing et al. <a href="#CIT0035" class="usa-link" aria-describedby="CIT0035"><sup>2023</sup></a>). Although nuclear and chloroplast markers have been used (Xu et al. <a href="#CIT0036" class="usa-link" aria-describedby="CIT0036"><sup>2019</sup></a>; Zheng et al. <a href="#CIT0040" class="usa-link" aria-describedby="CIT0040"><sup>2023</sup></a>), mitochondrial data are critical for evolutionary insights (Shearman et al. <a href="#CIT0028" class="usa-link" aria-describedby="CIT0028"><sup>2016</sup></a>; Wu et al. <a href="#CIT0033" class="usa-link" aria-describedby="CIT0033"><sup>2022</sup></a>). Here, we present the first complete mitochondrial genome of <em>P. giganteum</em>, assembled using Illumina and Nanopore sequencing. This work addresses a genomic gap and provides foundational data for studies on mitochondrial evolution, C<sub>4</sub> photosynthesis (Fan et al. <a href="#CIT0012" class="usa-link" aria-describedby="CIT0012"><sup>2022</sup></a>), and phylogenetics in this key genus.</p></section><section id="S0002"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="S0002-S2001"><h3 class="pmc_sec_title">Plant materials</h3>
<p>The <em>P. giganteum</em> was used in this study. The voucher specimen was identified by researchers and deposited at the Herbarium of Henan Institute of Science and Technology, Xinxiang, China, under voucher number XM-W145 (contact: Zhengang Ru; <span>rzgh5819@163.com</span>). Fresh roots of <em>P. giganteum</em> were collected from the Botanical Garden of the Wheat Research Center, Xinxiang City, Henan Province, China (113°52′E, 35°30′N) by Man Zhang (<a href="#F0001" class="usa-link">Figure 1</a>).</p>
<figure class="fig xbox font-sm" id="F0001"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377103_TMDN_A_2550607_F0001_C.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8bc/12377103/3f9df507d4de/TMDN_A_2550607_F0001_C.jpg" loading="lazy" height="1117" width="750" alt="Figure 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/F0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Morphology features of <em>Pennisetum giganteum</em>. The seedling stage of <em>Pennisetum giganteum</em> is characterized by erect plants and alternate leaves. The photos of <em>Pennisetum giganteum</em> were taken by Man Zhang at the Botanical Garden of the Center of Wheat Research of Henan Institute of Science and Technology, Xinxiang, Henan, China.</p></figcaption></figure></section><section id="S0002-S2002"><h3 class="pmc_sec_title">DNA sequencing, genome assembly, and annotation</h3>
<p>A genomic DNA sample was isolated from the roots with a Plant Tissue Mitochondrial DNA Extraction Kit (Genmed Scientific Inc., Arlington, MA, USA) according to the manufacturer’s instructions, and quality control was subsequently carried out on the purified DNA samples. The quality of DNA was assessed using Qubit3.0 and 1% agarose gel electrophoresis. Highly qualified DNA sample (OD260/280 = 1.8 ∼ 2.0, &gt;6ug) was utilized to construct the fragment library.</p>
<p>To obtain full-length mitochondrial genome sequences, we used both short-read (Illumina NovaSeq 6000 platform) and long-read sequencing (Nanopore sequencing platform) technologies in this study. The raw paired-end reads were trimmed and quality controlled by Trimmomatic with parameters (SLIDINGWINDOW:4:15 MINLEN:75) (Bolger et al. <a href="#CIT0001" class="usa-link" aria-describedby="CIT0001"><sup>2014</sup></a>). Clean data obtained through the above quality control processes was used for further analysis.</p>
<p>The Illumina NovaSeq 6000 short reads (6.3 Gb) were used to evaluate the complexity of the genome and correct the Nanopore long reads. First, we used ABySS 2.2.0 (Simpson et al. <a href="#CIT0029" class="usa-link" aria-describedby="CIT0029"><sup>2009</sup></a>) to perform genome assembly with multiple-kmer parameters and received the optimal results of the assembly. Secondly, blasR (Chaisson and Tesler <a href="#CIT0004" class="usa-link" aria-describedby="CIT0004"><sup>2012</sup></a>) was used to map the preliminary assembly results to the Nanopore long reads (6.3 Gb). Then, SPAdes-3.15.5 (Prjibelski et al. <a href="#CIT0025" class="usa-link" aria-describedby="CIT0025"><sup>2020</sup></a>) was used to assemble them (Nanopore and NGS data) together to construct contigs (Scaffolds). Finally, GapCloser v1.12 (Xu et al. <a href="#CIT0037" class="usa-link" aria-describedby="CIT0037"><sup>2020</sup></a>) software was applied to fill the remaining local inner gaps and correct the single-base polymorphism for the final assembly results.</p>
<p>The mitochondrial genes were annotated through homology alignments and de novo prediction. Gene prediction for tRNAs was performed using tRNAscan-SE 2.0.4 (Chan and Lowe <a href="#CIT0005" class="usa-link" aria-describedby="CIT0005"><sup>2019</sup></a>) with default parameters. For rRNA and protein-coding genes (PCGs), the assembled mitochondrial genome of <em>P. giganteum</em> (query) was aligned against the complete mitochondrial genome of <em>Saccharum officinarum</em> (reference database; GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/LC107874" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">LC107874</a>) <em>via</em> BLASTn (BLAST + 2.9.0; Camacho et al. <a href="#CIT0002" class="usa-link" aria-describedby="CIT0002"><sup>2009</sup></a>). Homologous regions meeting thresholds of ≥80% sequence identity and ≥90% coverage were retained, and gene boundaries were further refined using ORFfinder (v0.4.1; Sayers et al. <a href="#CIT0027" class="usa-link" aria-describedby="CIT0027"><sup>2024</sup></a>).</p>
<p>Finally, the circular mitochondrial genome assembly and split gene identification were performed and validated using the Plant Mitochondrial Genomes Map (PMGmap) v2.0 (Zhang et al. <a href="#CIT0039" class="usa-link" aria-describedby="CIT0039"><sup>2024</sup></a>), a tool specialized in structural annotation based on sequence conservation and break-join modeling.</p>
<p>The annotated mitogenome was submitted to the GenBank database under accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ768538" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ768538</a>. Then all gene models were analyzed by BLASTp (Camacho et al. <a href="#CIT0002" class="usa-link" aria-describedby="CIT0002"><sup>2009</sup></a>) against the non-redundant (NR) protein database (Sayers et al. <a href="#CIT0027" class="usa-link" aria-describedby="CIT0027"><sup>2024</sup></a>), SwissProt (The UniProt Consortium <a href="#CIT0032" class="usa-link" aria-describedby="CIT0032"><sup>2019</sup></a>), KEGG (Kanehisa and Goto <a href="#CIT0018" class="usa-link" aria-describedby="CIT0018"><sup>2000</sup></a>), Gene Ontology (GO) (The Gene Ontology Consortium <a href="#CIT0014" class="usa-link" aria-describedby="CIT0014"><sup>2000</sup></a>), and Clusters of Orthologous Groups (COG) (Tatusov et al. <a href="#CIT0030" class="usa-link" aria-describedby="CIT0030"><sup>2000</sup></a>) for functional annotation.</p>
<p>To validate the assembly completeness and uniformity, sequencing reads were mapped back to the mitochondrial genome using BWA-MEM v0.7.17 (Li <a href="#CIT0021" class="usa-link" aria-describedby="CIT0021"><sup>2013</sup></a>) with default parameters. The coverage depth across the genome was calculated using SAMtools v1.18 (Danecek et al. <a href="#CIT0008" class="usa-link" aria-describedby="CIT0008"><sup>2021</sup></a>) and visualized as a coverage plot (<a href="https://doi.org/10.1080/23802359.2025.2550607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1</a>) <em>via</em> the Integrative Genomics Viewer (IGV) v2.19.1 (Thorvaldsdóttir et al. <a href="#CIT0031" class="usa-link" aria-describedby="CIT0031"><sup>2013</sup></a>). The average coverage depth of 284.04× indicates sufficient sequencing depth to ensure high-confidence assembly and highlights uniform read distribution, which minimizes potential gaps or misassemblies (<a href="https://doi.org/10.1080/23802359.2025.2550607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1</a>). Coverage analysis also helps identify repetitive regions or structural variations that may challenge assembly algorithms.</p></section><section id="S0002-S2003"><h3 class="pmc_sec_title">Phylogenetic tree construction</h3>
<p>To determine the molecular position of <em>P. giganteum</em> with other closely related species, phylogenetic analyses were performed using 11 complete mitochondrial genome sequences (comprising 34 PCGs, 16 tRNAs, and three rRNAs). First, global alignment was conducted with MUMmer v3.23 (Delcher et al. <a href="#CIT0010" class="usa-link" aria-describedby="CIT0010"><sup>2002</sup></a>) to identify syntenic blocks. Local alignment refinement was performed using BLAT v36 (Kent <a href="#CIT0019" class="usa-link" aria-describedby="CIT0019"><sup>2002</sup></a>) with default parameters. To ensure alignment quality, ambiguous and divergent regions were filtered using Gblocks v0.91b (Castresana <a href="#CIT0003" class="usa-link" aria-describedby="CIT0003"><sup>2000</sup></a>) under strict parameters (-t = d -b4 = 5 -b5 = h), retaining only highly conserved blocks suitable for phylogenetic inference. The best-fit nucleotide substitution model (GTR+I + G) was selected using jModelTest 2.1.10 (Darriba et al. <a href="#CIT0009" class="usa-link" aria-describedby="CIT0009"><sup>2012</sup></a>), and maximum-likelihood trees were constructed with PhyML 3.0 (Guindon et al. <a href="#CIT0015" class="usa-link" aria-describedby="CIT0015"><sup>2010</sup></a>) using 1,000 bootstrap replicates. <em>Oryza sativa</em> Japonica Group (accession <a href="https://www.ncbi.nlm.nih.gov/nuccore/BA000029.3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">BA000029.3</a>) was selected as the outgroup, as its placement in the evolutionarily distinct Oryzoideae subfamily provides a robust phylogenetic contrast to the Panicoideae subfamily (containing <em>Pennisetum</em>), and its well-annotated mitochondrial genome is widely adopted in Poaceae studies.</p>
<p>The assembled mitochondrial genome (491,311 bp; GenBank ID: <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ768538" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ768538</a>) contains 34 protein-coding genes, 16 tRNA genes, and 3 rRNA genes. This circular map displays the following features from the innermost to the outermost ring: Relationships between dispersed repeats; Distribution of dispersed repeats along the genome (direct repeats shown in yellow, inverted repeats in green).</p>
<p>Distribution of tandem repeats; GC content across the genome; Scale axis; Genes encoded on the negative strand, genes encoded on the positive strand.</p></section></section><section id="S0003"><h2 class="pmc_sec_title">Results</h2>
<p>The <em>P. giganteum</em> mitochondrial genome (GenBank ID: <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ768538" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ768538</a>) was assembled into a single, circular sequence with a length of 491,311 bp and a GC content of 43.81%.</p>
<p>The mitochondrial genome of <em>P. giganteum</em> includes 34 PCGs, 16 tRNA genes, and three rRNA genes (<a href="#F0002" class="usa-link">Figure 2</a>, <a href="https://doi.org/10.1080/23802359.2025.2550607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>). Most of the PCGs have an ATG start codon, except for nad1 and nad4L, which have an ACG start codon. There are two types of termination codons in the PCGs: TAA and TAG, which correspond to 15 and 19 genes, respectively. Among the PCGs, eight were <em>cis</em>-spliced genes, six of which contained one intron (<em>ccmFc</em>, <em>cox2</em>, <em>rpl2</em>, <em>rps3</em>, <em>trnD-GUC</em>, <em>trnF-GAA</em>), one contained three introns (<em>nad4</em>), and one contained four introns (<em>nad7</em>). <em>nad1</em>, <em>nad2</em>, and <em>nad5</em> were <em>trans</em>-spliced genes, all of which contained four introns (<a href="https://doi.org/10.1080/23802359.2025.2550607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2</a>).</p>
<figure class="fig xbox font-sm" id="F0002"><h3 class="obj_head">Figure 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377103_TMDN_A_2550607_F0002_C.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8bc/12377103/ad605cee0241/TMDN_A_2550607_F0002_C.jpg" loading="lazy" height="863" width="750" alt="Figure 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/F0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mitochondrial genome map of <em>Pennisetum giganteum</em> constructed with PMGmap.</p></figcaption></figure><p>The 16 tRNAs ranged from 68 to 89 bp in length. <em>rrn5</em>, <em>rrnL</em>, and <em>rrnS</em> genes were 120 bp, 3535 bp, and 1967 bp in length, respectively. There were 17 gene duplications, including 9 PCGs, five tRNA genes, and three rRNA genes.</p>
<p>This study analyzed the phylogenetic relationships of 11 species using the complete mitochondrial genome sequence (<a href="#F0003" class="usa-link">Figure 3</a>). As designated a priori for phylogenetic contrast, Oryza sativa Japonica Group (<a href="https://www.ncbi.nlm.nih.gov/nuccore/BA000029.3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">BA000029.3</a>) formed a distinct basal branch serving as the outgroup. <em>P. giganteum</em> is most closely related to <em>S. officinarum</em> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/LC107874" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">LC107874</a>), followed by <em>Tripsacum dactyloides</em> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/DQ984517" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DQ984517</a>), <em>Zea mays</em> voucher SUMG 004 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/OP832499" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OP832499</a>), and <em>Zea mays</em> voucher SUMG 005 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/OP832500" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OP832500</a>). The above five species are grouped into one clade. Another large clade also contains five species: <em>Elymus magellanicus</em> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/OQ086977" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OQ086977</a>), <em>Thinopyrum obtusiflorum</em> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/OK120846" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OK120846</a>), <em>Aegilops speltoides</em> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/AP013107" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AP013107</a>), <em>Triticum aestivum</em> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/AP008982" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AP008982</a>), and <em>Hordeum vulgare</em> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/MN127974" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MN127974</a>). Results from distant species are presented as supplementary references, with core conclusions drawn from intra-Panicoideae relationships.</p>
<figure class="fig xbox font-sm" id="F0003"><h3 class="obj_head">Figure 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377103_TMDN_A_2550607_F0003_C.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8bc/12377103/e947aab3ee5d/TMDN_A_2550607_F0003_C.jpg" loading="lazy" height="302" width="750" alt="Figure 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/F0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The maximum likelihood (ML) phylogenetic tree was constructed using the complete mitochondrial genome of 11 species. The following sequences were used: <em>Pennisetum giganteum</em> (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ768538" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ768538</a>), <em>Oryza sativa</em> Japonica Group (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/BA000029.3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">BA000029.3</a>; Notsu et al. <a href="#CIT0023" class="usa-link" aria-describedby="CIT0023"><sup>2002</sup></a>), <em>Hordeum vulgare</em> (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/MN127974" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MN127974</a>), <em>Triticum aestivum</em> (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/AP008982" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AP008982</a>; Ogihara et al. <a href="#CIT0024" class="usa-link" aria-describedby="CIT0024"><sup>2005</sup></a>), <em>Aegilops speltoides</em> (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/AP013107" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AP013107</a>), <em>Thinopyrum obtusiflorum</em> (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/OK120846" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OK120846</a>; Wu et al. <a href="#CIT0033" class="usa-link" aria-describedby="CIT0033"><sup>2022</sup></a>), <em>Elymus magellanicus</em> (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/OQ086977" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OQ086977</a>; Chen et al. <a href="#CIT0006" class="usa-link" aria-describedby="CIT0006"><sup>2023</sup></a>), <em>Zea mays</em> voucher SUMG 004 (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/OP832499" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OP832499</a>), <em>Zea mays</em> voucher SUMG 005 (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/OP832500" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OP832500</a>), <em>Tripsacum dactyloides</em> (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/DQ984517" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DQ984517</a>), <em>Saccharum officinarum</em> (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/LC107874" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">LC107874</a>; Shearman et al. <a href="#CIT0028" class="usa-link" aria-describedby="CIT0028"><sup>2016</sup></a>). The phylogenetic tree was rooted with the outgroup (<em>Oryza</em> sativa), with branch lengths scaled to substitutions per site. Numbers next to each node indicate bootstrap support values.</p></figcaption></figure></section><section id="S0004"><h2 class="pmc_sec_title">Discussion and conclusion</h2>
<p>Compared to animal mitochondria, plant mitochondrial genomes exhibit greater structural complexity and variability. Consequently, research has predominantly focused on chloroplasts, leaving many plant mitochondrial genomes unresolved (Zardoya <a href="#CIT0038" class="usa-link" aria-describedby="CIT0038"><sup>2020</sup></a>). This study presents the first complete assembly and annotation of the mitochondrial genome of <em>P. giganteum</em>, marking the first reported mitochondrial genome for any species within the genus <em>Pennisetum</em>.</p>
<p>The assembly and annotation of the <em>P. giganteum</em> mitochondrial genome (491,311 bp) provide critical insights into its genomic architecture. Its moderate GC content (43.81%) aligns closely with mitochondrial genomes of other Poaceae species, such as <em>S. officinarum</em> (43.9%) and <em>Zea mays</em> (43.9%), suggesting conserved compositional stability among C<sub>4</sub> grasses (Clifton et al. <a href="#CIT0007" class="usa-link" aria-describedby="CIT0007"><sup>2004</sup></a>; Xing et al. <a href="#CIT0035" class="usa-link" aria-describedby="CIT0035"><sup>2023</sup></a>; Li et al. <a href="#CIT0022" class="usa-link" aria-describedby="CIT0022"><sup>2025</sup></a>). Notably, the presence of atypical start codons (e.g. ACG for <em>nad1</em>) mirrors findings in <em>Elymus magellanicus</em> (Chen et al. <a href="#CIT0006" class="usa-link" aria-describedby="CIT0006"><sup>2023</sup></a>) and <em>Saccharum</em> species (Zhou et al. <a href="#CIT0041" class="usa-link" aria-describedby="CIT0041"><sup>2022</sup></a>), highlighting potential evolutionary constraints in mitochondrial gene regulation.</p>
<p>Mitochondrial PCGs exhibit significantly slower evolutionary rates, particularly in synonymous substitutions (Wu et al. <a href="#CIT0034" class="usa-link" aria-describedby="CIT0034"><sup>2020</sup></a>). This high conservation makes them suitable for resolving deep-level phylogenetic relationships, contrasting with chloroplast genomes better suited for lower taxonomic ranks (Lan et al. <a href="#CIT0020" class="usa-link" aria-describedby="CIT0020"><sup>2024</sup></a>).</p>
<p>The coexistence of <em>cis</em>-spliced and <em>trans</em>-spliced genes underscores intricate RNA processing in <em>P. giganteum</em>, aligning with mitochondrial genome plasticity observed in other polyploid grasses (Guo et al. <a href="#CIT0016" class="usa-link" aria-describedby="CIT0016"><sup>2020</sup></a>). This characterization provides foundational insights into its potential role in supporting C<sub>4</sub> photosynthetic evolution.</p>
<p>Phylogenetic analysis revealed <em>P. giganteum</em>’s closest relationship with <em>S. officinarum</em> (a C<sub>4</sub> grass), consistent with morphological and ecological similarities, while showing marked divergence from C<sub>3</sub> species like <em>O. sativa</em>. This mitochondrial genome serves as a crucial reference for resolving taxonomic uncertainties, studying polyploid mitochondrial evolution, and developing molecular markers. These findings establish a foundation for future investigations into C<sub>4</sub> plant mitochondrial-chloroplast interactions and stress tolerance genetics. Integrating mitogenomic data with nuclear and chloroplast genomes will enhance understanding of adaptive evolution in this genus.</p></section><section id="sec8"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="SM6937"><div class="caption p"><span>Supplementary materials.docx</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377103/bin/TMDN_A_2550607_SM6937.docx" data-ga-action="click_feat_suppl" class="usa-link">TMDN_A_2550607_SM6937.docx</a><sup> (793.4KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>Conceptualization: MZ, XC, XW; Formal analysis: KF, HW, JY, XC; Writing – original draft: MZ, XW; Writing – review &amp; editing: MZ, XW; All authors read and approved of the final manuscript.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This work was supported by the Training Plan for Young Backbone Teachers in Colleges and Universities in Henan Province under Grant [No.2021GGJS122], National Natural Science Foundation of China [No.31701502], and Key Scientific and Technological Research Projects in Henan Province under Grant [No.252102111133]. This work was funded by the Project of Wheat Industry Technology system in Henan Province.</p></section><section id="S0008"><h2 class="pmc_sec_title">Ethical approval</h2>
<p>The plant samples used in this study were non-protected species. Sample collection was carried out in strict compliance with local and Chinese regulations. No ethical approval was required for this study.</p></section><section id="S0007"><h2 class="pmc_sec_title">Disclosure statement</h2>
<p>No potential conflict of interest was reported by the authors.</p></section><section id="S0006"><h2 class="pmc_sec_title">Data availability statement</h2>
<p>The genome sequence data that support the findings of this study are openly available in GenBank of NCBI at (<a href="https://www.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/</a>), reference number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ768538" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ768538</a>. The associated BioProject, Bio-Sample, and SRA numbers are PRJNA1196327, SAMN45421721, SRR31666584 (Nanopore) and SRR31666585 (NovaSeq), respectively.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="CIT0001">
<cite>Bolger AM, Lohse M, Usadel B.. 
2014. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics. 30(15):2114–2120. doi: 10.1093/bioinformatics/btu170.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Trimmomatic:%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;volume=30&amp;issue=15&amp;publication_year=2014&amp;pages=2114-2120&amp;pmid=24695404&amp;doi=10.1093/bioinformatics/btu170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0002">
<cite>Camacho C, Coulouris G, Avagyan V, Ma N, Papadopoulos J, Bealer K, Madden TL.. 
2009. BLAST+: architecture and applications. BMC Bioinform. 10(1):421. doi: 10.1186/1471-2105-10-421.</cite> [<a href="https://doi.org/10.1186/1471-2105-10-421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2803857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20003500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinform&amp;title=BLAST+:%20architecture%20and%20applications&amp;volume=10&amp;issue=1&amp;publication_year=2009&amp;pages=421&amp;pmid=20003500&amp;doi=10.1186/1471-2105-10-421&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0003">
<cite>Castresana J.
2000. Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 17(4):540–552. doi: 10.1093/oxfordjournals.molbev.a026334.
</cite> [<a href="https://doi.org/10.1093/oxfordjournals.molbev.a026334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10742046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=Selection%20of%20conserved%20blocks%20from%20multiple%20alignments%20for%20their%20use%20in%20phylogenetic%20analysis&amp;volume=17&amp;issue=4&amp;publication_year=2000&amp;pages=540-552&amp;pmid=10742046&amp;doi=10.1093/oxfordjournals.molbev.a026334&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0004">
<cite>Chaisson MJ, Tesler G.. 
2012. Mapping single molecule sequencing reads using basic local alignment with successive refinement (BLASR): application and theory. BMC Bioinformatics. 13(1):238. doi: 10.1186/1471-2105-13-238.
</cite> [<a href="https://doi.org/10.1186/1471-2105-13-238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3572422/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22988817/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Mapping%20single%20molecule%20sequencing%20reads%20using%20basic%20local%20alignment%20with%20successive%20refinement%20(BLASR):%20application%20and%20theory&amp;volume=13&amp;issue=1&amp;publication_year=2012&amp;pages=238&amp;pmid=22988817&amp;doi=10.1186/1471-2105-13-238&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0005">
<cite>Chan PP, Lowe TM.. 
2019. tRNAscan-SE: searching for tRNA genes in genomic sequences. Methods Mol Biol. 1962:1–14. doi: 10.1007/978-1-4939-9173-0_1.
</cite> [<a href="https://doi.org/10.1007/978-1-4939-9173-0_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6768409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31020551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&amp;title=tRNAscan-SE:%20searching%20for%20tRNA%20genes%20in%20genomic%20sequences&amp;volume=1962&amp;publication_year=2019&amp;pages=1-14&amp;pmid=31020551&amp;doi=10.1007/978-1-4939-9173-0_1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0006">
<cite>Chen XD, Wu XJ, Zhang JL, Zhang M, You JC, Ru ZG.. 
2023. Characterization of the complete mitochondrial genome of <em>Elymus magellanicus</em> (É. Desv.) Á. Löve (Poaceae, Pooideae). Mitochondrial DNA B Resour. 8(8):795–798. doi: 10.1080/23802359.2023.2238931.
</cite> [<a href="https://doi.org/10.1080/23802359.2023.2238931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10399486/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37545550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mitochondrial%20DNA%20B%20Resour&amp;title=Characterization%20of%20the%20complete%20mitochondrial%20genome%20of%20Elymus%20magellanicus%20(%C3%89.%20Desv.)%20%C3%81.%20L%C3%B6ve%20(Poaceae,%20Pooideae)&amp;volume=8&amp;issue=8&amp;publication_year=2023&amp;pages=795-798&amp;pmid=37545550&amp;doi=10.1080/23802359.2023.2238931&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0007">
<cite>Clifton SW, Minx P, Fauron CM, Gibson M, Allen JO, Sun H, Thompson M, Barbazuk WB, Kanuganti S, Tayloe C, et al. 
2004. Sequence and comparative analysis of the maize NB mitochondrial genome. Plant Physiol. 136(3):3486–3503. doi: 10.1104/pp.104.044602.
</cite> [<a href="https://doi.org/10.1104/pp.104.044602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC527149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15542500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Sequence%20and%20comparative%20analysis%20of%20the%20maize%20NB%20mitochondrial%20genome&amp;volume=136&amp;issue=3&amp;publication_year=2004&amp;pages=3486-3503&amp;pmid=15542500&amp;doi=10.1104/pp.104.044602&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0008">
<cite>Danecek P, Bonfield JK, Liddle J, Marshall J, Ohan V, Pollard MO, Whitwham A, Keane T, McCarthy SA, Davies RM, et al. 
2021. Twelve years of SAMtools and BCFtools. Gigascience. 10(2):giab008. doi: 10.1093/gigascience/giab008.
</cite> [<a href="https://doi.org/10.1093/gigascience/giab008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7931819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33590861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Twelve%20years%20of%20SAMtools%20and%20BCFtools&amp;volume=10&amp;issue=2&amp;publication_year=2021&amp;pages=giab008&amp;pmid=33590861&amp;doi=10.1093/gigascience/giab008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0009">
<cite>Darriba D, Taboada GL, Doallo R, Posada D.. 
2012. jModelTest 2: more models, new heuristics and parallel computing. Nat Methods. 9(8):772–772. doi: 10.1038/nmeth.2109.</cite> [<a href="https://doi.org/10.1038/nmeth.2109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4594756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22847109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=jModelTest%202:%20more%20models,%20new%20heuristics%20and%20parallel%20computing&amp;volume=9&amp;issue=8&amp;publication_year=2012&amp;pages=772-772&amp;pmid=22847109&amp;doi=10.1038/nmeth.2109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0010">
<cite>Delcher AL, Phillippy A, Carlton J, Salzberg SL.. 
2002. Fast algorithms for large-scale genome alignment and comparison. Nucleic Acids Res. 30(11):2478–2483. doi: 10.1093/nar/30.11.2478.
</cite> [<a href="https://doi.org/10.1093/nar/30.11.2478" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC117189/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12034836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Fast%20algorithms%20for%20large-scale%20genome%20alignment%20and%20comparison&amp;volume=30&amp;issue=11&amp;publication_year=2002&amp;pages=2478-2483&amp;pmid=12034836&amp;doi=10.1093/nar/30.11.2478&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0011">
<cite>Dos Reis GB, Mesquita AT, Torres GA, Andrade-Vieira LF, Pereira AV, Davide LC.. 
2014. Genomic homeology between <em>Pennisetum purpureum</em> and <em>Pennisetum glaucum</em> (Poaceae). Comp Cytogenet. 8(3):199–209. doi: 10.3897/CompCytogen.v8i3.7732.
</cite> [<a href="https://doi.org/10.3897/CompCytogen.v8i3.7732" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4205489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25349671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comp%20Cytogenet&amp;title=Genomic%20homeology%20between%20Pennisetum%20purpureum%20and%20Pennisetum%20glaucum%20(Poaceae)&amp;volume=8&amp;issue=3&amp;publication_year=2014&amp;pages=199-209&amp;pmid=25349671&amp;doi=10.3897/CompCytogen.v8i3.7732&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0012">
<cite>Fan YZ, Asao S, Furbank RT, Caemmerer SV, Day DA, Tcherkez G, Sage TL, Sage RF, Atkin OK.. 
2022. The crucial roles of mitochondria in supporting C4 photosynthesis. New Phytol. 233(3):1083–1096. doi: 10.1111/nph.17818.
</cite> [<a href="https://doi.org/10.1111/nph.17818" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34669188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol&amp;title=The%20crucial%20roles%20of%20mitochondria%20in%20supporting%20C4%20photosynthesis&amp;volume=233&amp;issue=3&amp;publication_year=2022&amp;pages=1083-1096&amp;pmid=34669188&amp;doi=10.1111/nph.17818&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0013">
<cite>Fulkerson WJ, Neal JS, Clark CF, Horadagoda A, Nandra KS, Barchia I.. 
2007. Nutritive value of forage species grown in the warm temperate climate of Australia for dairy cows: grasses and legumes. Livest Sci. 107(2-3):253–264. doi: 10.1016/j.livsci.2007.04.013.</cite> [<a href="https://doi.org/10.1016/j.livsci.2007.04.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Livest%20Sci&amp;title=Nutritive%20value%20of%20forage%20species%20grown%20in%20the%20warm%20temperate%20climate%20of%20Australia%20for%20dairy%20cows:%20grasses%20and%20legumes&amp;volume=107&amp;issue=2-3&amp;publication_year=2007&amp;pages=253-264&amp;doi=10.1016/j.livsci.2007.04.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0014">
<cite>Gene Ontology Consortium . 2000. Gene ontology: tool for the unification of biology. Nat Genet. 25(1):25–29. doi: 10.1038/75556.
</cite> [<a href="https://doi.org/10.1038/75556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3037419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10802651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Gene%20ontology:%20tool%20for%20the%20unification%20of%20biology&amp;volume=25&amp;issue=1&amp;publication_year=2000&amp;pages=25-29&amp;pmid=10802651&amp;doi=10.1038/75556&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0015">
<cite>Guindon S, Dufayard J-F, Lefort V, Anisimova M, Hordijk W, Gascuel O.. 
2010. New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0. Syst Biol. 59(3):307–321. doi: 10.1093/sysbio/syq010.
</cite> [<a href="https://doi.org/10.1093/sysbio/syq010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20525638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Syst%20Biol&amp;title=New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies:%20assessing%20the%20performance%20of%20PhyML%203.0&amp;volume=59&amp;issue=3&amp;publication_year=2010&amp;pages=307-321&amp;pmid=20525638&amp;doi=10.1093/sysbio/syq010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0016">
<cite>Guo W, Zhu A, Fan W, Adams RP, Mower JP.. 
2020. Extensive shifts from <em>cis</em>- to <em>trans</em>-splicing of gymnosperm mitochondrial introns. Mol Biol Evol. 37(6):1615–1620. doi: 10.1093/molbev/msaa029.
</cite> [<a href="https://doi.org/10.1093/molbev/msaa029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32027368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=Extensive%20shifts%20from%20cis-%20to%20trans-splicing%20of%20gymnosperm%20mitochondrial%20introns&amp;volume=37&amp;issue=6&amp;publication_year=2020&amp;pages=1615-1620&amp;pmid=32027368&amp;doi=10.1093/molbev/msaa029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0017">
<cite>Hayat K, Zhou YF, Menhas S, Bundschuh J, Hayat S, Ullah A, Wang J, Chen XF, Zhang D, Zhou P.. 
2020. <em>Pennisetum giganteum</em>: an emerging salt accumulating/tolerant non-conventional crop for sustainable saline agriculture and simultaneous phytoremediation. Environ Pollut. 265(Pt A):114876. doi: 10.1016/j.envpol.2020.114876.
</cite> [<a href="https://doi.org/10.1016/j.envpol.2020.114876" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32512425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environ%20Pollut&amp;title=Pennisetum%20giganteum:%20an%20emerging%20salt%20accumulating/tolerant%20non-conventional%20crop%20for%20sustainable%20saline%20agriculture%20and%20simultaneous%20phytoremediation&amp;volume=265&amp;issue=Pt%20A&amp;publication_year=2020&amp;pages=114876&amp;pmid=32512425&amp;doi=10.1016/j.envpol.2020.114876&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0018">
<cite>Kanehisa M, Goto S.. 
2000. KEGG: kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 28(1):27–30. doi: 10.1093/nar/28.1.27.
</cite> [<a href="https://doi.org/10.1093/nar/28.1.27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=KEGG:%20kyoto%20encyclopedia%20of%20genes%20and%20genomes&amp;volume=28&amp;issue=1&amp;publication_year=2000&amp;pages=27-30&amp;pmid=10592173&amp;doi=10.1093/nar/28.1.27&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0019">
<cite>Kent WJ.
2002. BLAT—the BLAST-like alignment tool. Genome Res. 12(4):656–664. doi: 10.1101/gr.229202.
</cite> [<a href="https://doi.org/10.1101/gr.229202" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC187518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11932250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=BLAT%E2%80%94the%20BLAST-like%20alignment%20tool&amp;volume=12&amp;issue=4&amp;publication_year=2002&amp;pages=656-664&amp;pmid=11932250&amp;doi=10.1101/gr.229202&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0020">
<cite>Lan WX, Mo Q, Jin MM, Wen YH, Yang MQ, Ma H, Huang HQ, Huang MJ.. 
2024. Exploring the phylogenetic framework and trait evolution of Impatiens through chloroplast genome analysis. BMC Plant Biol. 24(1):1218. doi: 10.1186/s12870-024-05964-y.
</cite> [<a href="https://doi.org/10.1186/s12870-024-05964-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11660898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39702025/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biol&amp;title=Exploring%20the%20phylogenetic%20framework%20and%20trait%20evolution%20of%20Impatiens%20through%20chloroplast%20genome%20analysis&amp;volume=24&amp;issue=1&amp;publication_year=2024&amp;pages=1218&amp;pmid=39702025&amp;doi=10.1186/s12870-024-05964-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0021">
<cite>Li H.
2013. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. <em>arXiv:1303.3997</em>. doi: 10.48550/arXiv.1303.3997.</cite> [<a href="https://doi.org/10.48550/arXiv.1303.3997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H.%202013.%20Aligning%20sequence%20reads,%20clone%20sequences%20and%20assembly%20contigs%20with%20BWA-MEM.%20arXiv:1303.3997.%20doi:%2010.48550/arXiv.1303.3997." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0022">
<cite>Li Y, Li S, Hua X, Wang M, Zhang Y, Chen R, Liu L, Zhang J, Mei Y, Wu J, et al. 
2025. Mitochondrial genome structural variants and candidate cytoplasmic male sterility-related gene in sugarcane. BMC Genomics. 26(1):16. doi: 10.1186/s12864-025-1234-9.
</cite> [<a href="https://doi.org/10.1186/s12864-025-1234-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11724576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39794692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Mitochondrial%20genome%20structural%20variants%20and%20candidate%20cytoplasmic%20male%20sterility-related%20gene%20in%20sugarcane&amp;volume=26&amp;issue=1&amp;publication_year=2025&amp;pages=16&amp;pmid=39794692&amp;doi=10.1186/s12864-025-1234-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0023">
<cite>Notsu Y, Masood S, Nishikawa T, Kubo N, Akiduki G, Nakazono M, Hirai A, Kadowaki K.. 
2002. The complete sequence of the rice (<em>Oryza sativa</em> L.) mitochondrial genome: frequent DNA sequence acquisition and loss during the evolution of flowering plants. Mol Genet Genomics. 268(4):434–445. doi: 10.1007/s00438-002-0767-1.
</cite> [<a href="https://doi.org/10.1007/s00438-002-0767-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12471441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Genet%20Genomics&amp;title=The%20complete%20sequence%20of%20the%20rice%20(Oryza%20sativa%20L.)%20mitochondrial%20genome:%20frequent%20DNA%20sequence%20acquisition%20and%20loss%20during%20the%20evolution%20of%20flowering%20plants&amp;volume=268&amp;issue=4&amp;publication_year=2002&amp;pages=434-445&amp;pmid=12471441&amp;doi=10.1007/s00438-002-0767-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0024">
<cite>Ogihara Y, Yamazaki Y, Murai K, Kanno A, Terachi T, Shiina T, Miyashita N, Nasuda S, Nakamura C, Mori N, et al. 
2005. Structural dynamics of cereal mitochondrial genomes as revealed by complete nucleotide sequencing of the wheat mitochondrial genome. Nucleic Acids Res. 33(19):6235–6250. doi: 10.1093/nar/gki925.
</cite> [<a href="https://doi.org/10.1093/nar/gki925" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1275586/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16260473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Structural%20dynamics%20of%20cereal%20mitochondrial%20genomes%20as%20revealed%20by%20complete%20nucleotide%20sequencing%20of%20the%20wheat%20mitochondrial%20genome&amp;volume=33&amp;issue=19&amp;publication_year=2005&amp;pages=6235-6250&amp;pmid=16260473&amp;doi=10.1093/nar/gki925&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0025">
<cite>Prjibelski AD, Vasilinetc I, Bankevich A, Gurevich A, Krivosheeva T, Nurk S, Antipov D.. 
2020. Using SPAdes de novo assembler. Curr Protoc Bioinformatics. 70(1):e102. doi: 10.1002/cpbi.102.
</cite> [<a href="https://doi.org/10.1002/cpbi.102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32559359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Protoc%20Bioinformatics&amp;title=Using%20SPAdes%20de%20novo%20assembler&amp;volume=70&amp;issue=1&amp;publication_year=2020&amp;pages=e102&amp;pmid=32559359&amp;doi=10.1002/cpbi.102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0026">
<cite>Samson R, Mani S, Boddey R, Sokhansanj S, Quesada D, Urquiaga S, Reis V, Lem CH.. 
2005. The potential of C<sub>4</sub> perennial grasses for developing a global BIOHEAT industry. Crit Rev Plant Sci. 24(5-6):461–495. doi: 10.1080/07352680500316508.</cite> [<a href="https://doi.org/10.1080/07352680500316508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Plant%20Sci&amp;title=The%20potential%20of%20C4%20perennial%20grasses%20for%20developing%20a%20global%20BIOHEAT%20industry&amp;volume=24&amp;issue=5-6&amp;publication_year=2005&amp;pages=461-495&amp;doi=10.1080/07352680500316508&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0027">
<cite>Sayers EW, Beck J, Bolton EE, Brister JR, Chan J, Comeau DC, Connor R, DiCuccio M, Farrell CM, Feldgarden M, et al. 
2024. Database resources of the National Center for Biotechnology Information. Nucleic Acids Res. 52(D1):D33–D43. doi: 10.1093/nar/gkad1044.
</cite> [<a href="https://doi.org/10.1093/nar/gkad1044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767890/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37994677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Database%20resources%20of%20the%20National%20Center%20for%20Biotechnology%20Information&amp;volume=52&amp;issue=D1&amp;publication_year=2024&amp;pages=D33-D43&amp;pmid=37994677&amp;doi=10.1093/nar/gkad1044&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0028">
<cite>Shearman JR, Sonthirod C, Naktang C, Pootakham W, Yoocha T, Sangsrakru D, Jomchai N, Tragoonrung S, Tangphatsornruang S.. 
2016. The two chromosomes of the mitochondrial genome of a sugarcane cultivar: assembly and recombination analysis using long PacBio reads. Sci Rep. 6(1):31533. doi: 10.1038/srep31533.
</cite> [<a href="https://doi.org/10.1038/srep31533" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4987617/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27530092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=The%20two%20chromosomes%20of%20the%20mitochondrial%20genome%20of%20a%20sugarcane%20cultivar:%20assembly%20and%20recombination%20analysis%20using%20long%20PacBio%20reads&amp;volume=6&amp;issue=1&amp;publication_year=2016&amp;pages=31533&amp;pmid=27530092&amp;doi=10.1038/srep31533&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0029">
<cite>Simpson JT, Wong K, Jackman SD, Schein JE, Jones SJ, Birol I.. 
2009. ABySS: a parallel assembler for short read sequence data. Genome Res. 19(6):1117–1123. doi: 10.1101/gr.089532.108.
</cite> [<a href="https://doi.org/10.1101/gr.089532.108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2694472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19251739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=ABySS:%20a%20parallel%20assembler%20for%20short%20read%20sequence%20data&amp;volume=19&amp;issue=6&amp;publication_year=2009&amp;pages=1117-1123&amp;pmid=19251739&amp;doi=10.1101/gr.089532.108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0030">
<cite>Tatusov RL, Galperin MY, Natale DA, Koonin EV.. 
2000. The COG database: a tool for genome-scale analysis of protein functions and evolution. Nucleic Acids Res. 28(1):33–36. doi: 10.1093/nar/28.1.33.
</cite> [<a href="https://doi.org/10.1093/nar/28.1.33" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592175/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20COG%20database:%20a%20tool%20for%20genome-scale%20analysis%20of%20protein%20functions%20and%20evolution&amp;volume=28&amp;issue=1&amp;publication_year=2000&amp;pages=33-36&amp;pmid=10592175&amp;doi=10.1093/nar/28.1.33&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0031">
<cite>Thorvaldsdóttir H, Robinson JT, Mesirov JP.. 
2013. Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. Brief Bioinform. 14(2):178–192. doi: 10.1093/bib/bbs017.
</cite> [<a href="https://doi.org/10.1093/bib/bbs017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3603213/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22517427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform&amp;title=Integrative%20Genomics%20Viewer%20(IGV):%20high-performance%20genomics%20data%20visualization%20and%20exploration&amp;volume=14&amp;issue=2&amp;publication_year=2013&amp;pages=178-192&amp;pmid=22517427&amp;doi=10.1093/bib/bbs017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0032">
<cite>UniProt Consortium . 2019. UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res. 47(D1):D506–D515. doi: 10.1093/nar/gky1049.
</cite> [<a href="https://doi.org/10.1093/nar/gky1049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30395287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=UniProt:%20a%20worldwide%20hub%20of%20protein%20knowledge&amp;volume=47&amp;issue=D1&amp;publication_year=2019&amp;pages=D506-D515&amp;pmid=30395287&amp;doi=10.1093/nar/gky1049&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0033">
<cite>Wu XJ, Hu XG, Chen XD, Zhang JL, Ren CC, Song LT, Fang F, Dong N, Hu TZ, Ru ZG.. 
2022. Sequencing and characterization of the complete mitochondrial genome of <em>Thinopyrum obtusiflorum</em> (DC.) Banfi, 2018 (Poaceae). Mitochondrial DNA B Resour. 7(3):539–540. doi: 10.1080/23802359.2022.2054378.
</cite> [<a href="https://doi.org/10.1080/23802359.2022.2054378" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8959512/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35356795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mitochondrial%20DNA%20B%20Resour&amp;title=Sequencing%20and%20characterization%20of%20the%20complete%20mitochondrial%20genome%20of%20Thinopyrum%20obtusiflorum%20(DC.)%20Banfi,%202018%20(Poaceae)&amp;volume=7&amp;issue=3&amp;publication_year=2022&amp;pages=539-540&amp;pmid=35356795&amp;doi=10.1080/23802359.2022.2054378&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0034">
<cite>Wu Z, Waneka G, Sloan DB.. 
2020. The tempo and mode of angiosperm mitochondrial genome divergence inferred from intraspecific variation in Arabidopsis thaliana. G3 (Bethesda). 10(3):1077–1086. doi: 10.1534/g3.119.401023.
</cite> [<a href="https://doi.org/10.1534/g3.119.401023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7056966/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31964685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20(Bethesda)&amp;title=The%20tempo%20and%20mode%20of%20angiosperm%20mitochondrial%20genome%20divergence%20inferred%20from%20intraspecific%20variation%20in%20Arabidopsis%20thaliana&amp;volume=10&amp;issue=3&amp;publication_year=2020&amp;pages=1077-1086&amp;pmid=31964685&amp;doi=10.1534/g3.119.401023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0035">
<cite>Xing LS, Wang MJ, He Q, Zhang HY, Liang HF, Zhou QH, Liu Y, Liu Z, Wang Y, Du CL, et al. 
2023. Differential subgenome expression underlies biomass accumulation in allotetraploid <em>Pennisetum giganteum</em>. BMC Biol. 21(1):161. doi: 10.1186/s12915-023-01643-w.
</cite> [<a href="https://doi.org/10.1186/s12915-023-01643-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10362693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37480118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Biol&amp;title=Differential%20subgenome%20expression%20underlies%20biomass%20accumulation%20in%20allotetraploid%20Pennisetum%20giganteum&amp;volume=21&amp;issue=1&amp;publication_year=2023&amp;pages=161&amp;pmid=37480118&amp;doi=10.1186/s12915-023-01643-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0036">
<cite>Xu J, Song Y, Jing XY, Li MF.. 
2019. Characterization of the complete chloroplast genome sequence of <em>Pennisetum glaucum</em> and its phylogenetic implications. Mitochondrial DNA B Resour. 4(2):3764–3765. doi: 10.1080/23802359.2019.1668312.
</cite> [<a href="https://doi.org/10.1080/23802359.2019.1668312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7707298/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33366180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mitochondrial%20DNA%20B%20Resour&amp;title=Characterization%20of%20the%20complete%20chloroplast%20genome%20sequence%20of%20Pennisetum%20glaucum%20and%20its%20phylogenetic%20implications&amp;volume=4&amp;issue=2&amp;publication_year=2019&amp;pages=3764-3765&amp;pmid=33366180&amp;doi=10.1080/23802359.2019.1668312&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0037">
<cite>Xu MY, Guo LD, Gu SQ, Wang O, Zhang R, Peters BA, Fan GY, Liu X, Xu X, Deng L, et al. 
2020. TGS-GapCloser: a fast and accurate gap closer for large genomes with low coverage of error-prone long reads. Gigascience. 9(9):giaa094. doi: 10.1093/gigascience/giaa094.
</cite> [<a href="https://doi.org/10.1093/gigascience/giaa094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7476103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32893860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=TGS-GapCloser:%20a%20fast%20and%20accurate%20gap%20closer%20for%20large%20genomes%20with%20low%20coverage%20of%20error-prone%20long%20reads&amp;volume=9&amp;issue=9&amp;publication_year=2020&amp;pages=giaa094&amp;pmid=32893860&amp;doi=10.1093/gigascience/giaa094&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0038">
<cite>Zardoya R.
2020. Recent advances in understanding mitochondrial genome diversity. F1000Res. 9:270. doi: 10.12688/f1000research.21490.1.</cite> [<a href="https://doi.org/10.12688/f1000research.21490.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7194472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32399193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=F1000Res&amp;title=Recent%20advances%20in%20understanding%20mitochondrial%20genome%20diversity&amp;volume=9&amp;publication_year=2020&amp;pages=270&amp;pmid=32399193&amp;doi=10.12688/f1000research.21490.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0039">
<cite>Zhang X, Chen H, Ni Y, Wu B, Li J, Burzyński A, Liu C.,. 2024. Plant mitochondrial genome map (PMGmap): a software tool for the comprehensive visualization of coding, noncoding and genome features of plant mitochondrial genomes. Mol Ecol Resour. 24(5):e13952. doi: 10.1111/1755-0998.13952.
</cite> [<a href="https://doi.org/10.1111/1755-0998.13952" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38523350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Ecol%20Resour&amp;title=Plant%20mitochondrial%20genome%20map%20(PMGmap):%20a%20software%20tool%20for%20the%20comprehensive%20visualization%20of%20coding,%20noncoding%20and%20genome%20features%20of%20plant%20mitochondrial%20genomes&amp;volume=24&amp;issue=5&amp;publication_year=2024&amp;pages=e13952&amp;pmid=38523350&amp;doi=10.1111/1755-0998.13952&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0040">
<cite>Zheng HK, Wang BY, Hua XT, Gao RT, Wang YH, Zhang ZX, Zhang YX, Mei J, Huang YJ, Huang YM, et al. 
2023. A near-complete genome assembly of the allotetrapolyploid <em>Cenchrus fungigraminus</em> (JUJUNCAO) provides insights into its evolution and C<sub>4</sub> photosynthesis. Plant Commun. 4(5):100633. doi: 10.1016/j.xplc.2023.100633.
</cite> [<a href="https://doi.org/10.1016/j.xplc.2023.100633" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10504591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37271992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Commun&amp;title=A%20near-complete%20genome%20assembly%20of%20the%20allotetrapolyploid%20Cenchrus%20fungigraminus%20(JUJUNCAO)%20provides%20insights%20into%20its%20evolution%20and%20C4%20photosynthesis&amp;volume=4&amp;issue=5&amp;publication_year=2023&amp;pages=100633&amp;pmid=37271992&amp;doi=10.1016/j.xplc.2023.100633&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CIT0041">
<cite>Zhou D, Liu Y, Yao J, Yin Z, Wang X, Xu L, Que Y, Mo P, Liu X.. 
2022. Characterization and phylogenetic analyses of the complete mitochondrial genome of sugarcane (<em>Saccharum</em> spp. Hybrids) line A1. Diversity. 14(5):333. doi: 10.3390/d14050333.</cite> [<a href="https://doi.org/10.3390/d14050333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diversity&amp;title=Characterization%20and%20phylogenetic%20analyses%20of%20the%20complete%20mitochondrial%20genome%20of%20sugarcane%20(Saccharum%20spp.%20Hybrids)%20line%20A1&amp;volume=14&amp;issue=5&amp;publication_year=2022&amp;pages=333&amp;doi=10.3390/d14050333&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Supplementary materials.docx</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377103/bin/TMDN_A_2550607_SM6937.docx" data-ga-action="click_feat_suppl" class="usa-link">TMDN_A_2550607_SM6937.docx</a><sup> (793.4KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The genome sequence data that support the findings of this study are openly available in GenBank of NCBI at (<a href="https://www.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/</a>), reference number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ768538" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ768538</a>. The associated BioProject, Bio-Sample, and SRA numbers are PRJNA1196327, SAMN45421721, SRR31666584 (Nanopore) and SRR31666585 (NovaSeq), respectively.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Mitochondrial DNA. Part B, Resources are provided here courtesy of <strong>Taylor & Francis</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1080/23802359.2025.2550607"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TMDN_10_2550607.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12377103/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12377103/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377103%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377103/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12377103/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12377103/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12377103/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12377103/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12377103/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="heznYhmMsbwUNUGy5XFIMfSAT17Byh7WftEvWqN0kdQ4sPvoteD93JubIbnKEnsG">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
