
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Absolute Quantitation of Phosphopeptides and Glycopeptides Using Coulometric Mass Spectrometry - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15251CC08AEEFC93051CC000199E165A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsmsau">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371591/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Measurement Science Au">
<meta name="citation_title" content="Absolute Quantitation of Phosphopeptides and Glycopeptides Using Coulometric Mass Spectrometry">
<meta name="citation_author" content="Md Tanim-Al Hassan">
<meta name="citation_author_institution" content="Department of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey 07102, United States">
<meta name="citation_author" content="Yongling Ai">
<meta name="citation_author_institution" content="Department of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey 07102, United States">
<meta name="citation_author" content="Bhavya Deshaboina">
<meta name="citation_author_institution" content="Department of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey 07102, United States">
<meta name="citation_author" content="Timothy Yaroshuk">
<meta name="citation_author_institution" content="Department of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey 07102, United States">
<meta name="citation_author" content="Arjun Sharma">
<meta name="citation_author_institution" content="Department of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey 07102, United States">
<meta name="citation_author" content="Quentin Young">
<meta name="citation_author_institution" content="Department of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey 07102, United States">
<meta name="citation_author" content="Howard D Dewald">
<meta name="citation_author_institution" content="Chemistry &amp; Biochemistry Department, Ohio University, Athens, Ohio 45701, United States">
<meta name="citation_author" content="Hao Chen">
<meta name="citation_author_institution" content="Department of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey 07102, United States">
<meta name="citation_publication_date" content="2025 Jul 18">
<meta name="citation_volume" content="5">
<meta name="citation_issue" content="4">
<meta name="citation_firstpage" content="559">
<meta name="citation_doi" content="10.1021/acsmeasuresciau.5c00047">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371591/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371591/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371591/pdf/tg5c00047.pdf">
<meta name="description" content="Phosphorylation and glycosylation are two important protein post-transitional modifications (PTMs). However, quantification of these PTMs is challenging due to the lack of protein or peptide standards. In this study, we introduced a novel approach ...">
<meta name="og:title" content="Absolute Quantitation of Phosphopeptides and Glycopeptides Using Coulometric Mass Spectrometry">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Phosphorylation and glycosylation are two important protein post-transitional modifications (PTMs). However, quantification of these PTMs is challenging due to the lack of protein or peptide standards. In this study, we introduced a novel approach ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371591/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371591">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsmeasuresciau.5c00047"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/tg5c00047.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371591%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371591/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371591/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371591/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsmsau.png" alt="ACS Measurement Science Au logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Measurement Science Au" title="Link to ACS Measurement Science Au" shape="default" href="https://pubs.acs.org/journal/amachv" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Meas Sci Au</button></div>. 2025 Jul 18;5(4):559–571. doi: <a href="https://doi.org/10.1021/acsmeasuresciau.5c00047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsmeasuresciau.5c00047</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Meas%20Sci%20Au%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Meas%20Sci%20Au%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Meas%20Sci%20Au%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Meas%20Sci%20Au%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Absolute
Quantitation of Phosphopeptides and Glycopeptides
Using Coulometric Mass Spectrometry</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tanim-Al%20Hassan%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Md Tanim-Al Hassan</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Md Tanim-Al Hassan</span></h3>
<div class="p">
<sup>†</sup>
Department
of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey
07102, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tanim-Al%20Hassan%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Md Tanim-Al Hassan</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ai%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yongling Ai</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yongling Ai</span></h3>
<div class="p">
<sup>†</sup>
Department
of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey
07102, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ai%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yongling Ai</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deshaboina%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Bhavya Deshaboina</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Bhavya Deshaboina</span></h3>
<div class="p">
<sup>†</sup>
Department
of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey
07102, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deshaboina%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bhavya Deshaboina</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yaroshuk%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Timothy Yaroshuk</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Timothy Yaroshuk</span></h3>
<div class="p">
<sup>†</sup>
Department
of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey
07102, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yaroshuk%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Timothy Yaroshuk</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Arjun Sharma</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Arjun Sharma</span></h3>
<div class="p">
<sup>†</sup>
Department
of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey
07102, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Arjun Sharma</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Young%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Quentin Young</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Quentin Young</span></h3>
<div class="p">
<sup>†</sup>
Department
of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey
07102, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Young%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Quentin Young</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dewald%20HD%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Howard D Dewald</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Howard D Dewald</span></h3>
<div class="p">
<sup>‡</sup>
Chemistry
&amp; Biochemistry Department, Ohio University, Athens, Ohio
45701, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dewald%20HD%22%5BAuthor%5D" class="usa-link"><span class="name western">Howard D Dewald</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Hao Chen</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Hao Chen</span></h3>
<div class="p">
<sup>†</sup>
Department
of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey
07102, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hao Chen</span></a>
</div>
</div>
<sup>†,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Department
of Chemistry and Environmental Science, New Jersey Institutes of Technology, University Heights, Newark, New Jersey
07102, United States
</div>
<div id="aff2">
<sup>‡</sup>
Chemistry
&amp; Biochemistry Department, Ohio University, Athens, Ohio
45701, United States
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>hao.chen.2@njit.edu</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 26; Accepted 2025 Jun 11; Revised 2025 Jun 10; Collection date 2025 Aug 20.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371591</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Phosphorylation and glycosylation are two important protein
post-transitional
modifications (PTMs). However, quantification of these PTMs is challenging
due to the lack of protein or peptide standards. In this study, we
introduced a novel approach using coulometric mass spectrometry (CMS)
for absolute quantitation of phosphopeptides and glycopeptides without
using standards. First, phosphorylated tyrosine peptides such as TSTEPQpYQPGENL
and RRLIEDAEpYAARG can be converted into electrochemically active
tyrosine peptides via enzymatic phosphate removal using alkaline phosphatase
prior to CMS quantitation. Accurate quantitation was obtained with
small quantitation errors (0.3–6.6%). Alternatively, for electrochemically
inactive phosphopeptides and glycopeptides, derivatization of their
N-termini with an NHS ester reagent, 2,5-dioxo-1-pyrrolidinyl 3,4-dihydroxybenzene
propanoate (DPDP), was conducted to introduce one electroactive catechol
tag, allowing the DPDP-derivatized peptides to be quantified by CMS.
This strategy was first validated using peptides RGD, GGYR, phosphopeptide
RRApSVA, and glycopeptide NYIVGQPSS­(β-GlcNAc)­TGNL–OH,
and successful quantification was achieved with quantification errors
less than 6%. Taking one step further, we applied this approach to
quantify glycopeptides generated from tryptic digestion of the NIST
monoclonal antibody (mAb). Through hydrophilic interaction liquid
chromatography column separation, five N297 glycopeptides were successfully
derivatized, separated, and quantified by CMS without the use of standards.
Due to the biological significance of PTMs, this study for quantifying
peptides carrying PTMs would have a high potential for quantitative
proteomics and biological research.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> phosphopeptide, glycopeptide, post-translational
modification, absolute quantitation, electrochemistry, mass spectrometry</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec7f/12371591/71fdab5a147a/tg5c00047_0010.jpg" loading="lazy" height="230" width="648" alt="graphic file with name tg5c00047_0010.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec7f/12371591/71fdab5a147a/tg5c00047_0008.jpg" loading="lazy" id="tgr1" height="230" width="648" alt="graphic file with name tg5c00047_0008.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Phosphorylation and glycosylation are
among the most common and
extensively studied post-translational modifications (PTMs).<em>
−





</em> About 30% of the human proteome is phosphorylated, and phosphoproteins
may exist as multiple phosphorylated forms. There are two main kinds of phosphorylation: <em>O</em>-phosphorylation,
which occurs on serine, threonine, and tyrosine residues, and <em>N</em>-phosphorylation, which takes places on histidine, arginine,
and lysine. Glycosylation plays various
roles in the structures and functions of proteins, and alterations
in glycosylation could lead to numerous diseases including cancer
and immune system deficiencies.<em>
−

</em>
<em>N</em>-glycosylation
occurs mainly at asparagine (Asn or N) residues, where the consensus
tripeptide sequence NXS/T is located (where <em>X</em> is
any amino acid except proline). At least one <em>N</em>-glycosylation
motif is present in two-thirds of all human proteins, and about 50%
of all human proteins are <em>N</em>-glycosylated. Other types
of glycosylation include <em>O</em>-glycosylation, which occurs
in the serine and threonine residues, and <em>C</em>-glycosylation,
which bonds on the indole rings of the tryptophan residues.<em>
,
</em>
</p>
<p>In proteomics research, quantitative information on peptides
and
proteins is critical for helping us understand biological processes.
Quantification in mass spectrometry (MS)-based proteomics research
relies on two fundamental strategies: relative and absolute quantitation.<em>
−



</em> Relative quantitation methods involve comparing the levels of peptides
or proteins among different samples. A
traditional label-free approach involves analyzing samples and comparing
their mass spectra to determine peptide abundances relative to another
sample.<em>
,
</em> Alternatively, isotopic labeling is commonly
used in relative quantitation, where peptides from two samples are
labeled with heavy and light isotopes, respectively.<em>
,
</em> The intensities of these labeled peptides are then compared to discern
changes in abundance between the samples.<em>
,−

</em> Elegant approaches like tandem mass tag, isotope-coded
affinity tags, stable isotope labeling by amino acids in cell culture
(SILAC), isobaric tags for relative and absolute quantitation, metal
element chelated tags, isotope-coded protein labeling, and <em>N</em>,<em>N</em>-dimethyl leucine (DiLeu) isobaric tag
methods fall under this category.<em>
−







</em> Likewise, absolute quantitation of proteins involves spiking a known
quantity of isotope-labeled target peptide or protein into an experimental
sample, followed by LC–MS or LC–MS/MS analysis.<em>
,,,−










</em> The three most well-known methods for absolute quantitation are
absolute quantification (AQUA), quantification conCATamer (QconCAT),
and protein standard absolute quantification. While these elegant techniques offer the advantage of determining
the absolute amounts of target peptides across different samples,
the necessity of using standard or isotope-labeled peptides poses
challenges. These standard peptides may not always be readily available,
and their synthesis can be expensive and time-consuming.<em>
,,
</em> Thus, the quest for a standard-free
absolute quantitation method remains an ideal goal in proteomics research.
This is particularly true for quantifying protein PTMs such as phosphorylation
and glycosylation, in which the standard phosphopeptides or glycopeptides
may be even more difficult to obtain.</p>
<p>Recently, we developed
coulometric mass spectrometry (CMS) for
absolute quantitation of electroactive analytes, using a liquid chromatography
(LC)/electrochemistry/MS (LC/EC/MS) apparatus (illustrated in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1</a>).<em>
−


,−


</em> The method is based on electrochemical oxidation/reduction of analytes,
followed by MS to measure the oxidation/reduction yield. Electrochemical
reactions result in an electric current response, which can be integrated
over time to calculate the electric charge <em>Q</em> involved
in the redox reaction. According to Faraday’s law, <em>Q</em> is proportional to the quantity of the oxidized/reduced
analyte: <em>Q</em> = <em>nzF</em>, where <em>n</em> is the mole of the oxidized/reduced analyte, <em>z</em> is the number of electrons transferred per molecule during the redox
reaction, and <em>F</em> is the Faraday’s constant
(9.65 × 10<sup>4</sup> C/mol). Therefore, the moles of the oxidized/reduced
analyte can be calculated as <em>n = Q</em>/<em>zF</em>. Meanwhile, upon oxidation or reduction, the target analyte shows
a reduced intensity in the acquired MS spectra, and the relative analyte
ion intensity change, Δ<em>i</em>, reflects the redox
conversion yield. Thus, the moles of the oxidized/reduced analyte,
in combination with the conversion yield, can be used to calculate
the total amount of the analyte. In other words</p>
<table class="disp-formula p" id="uneq1"><tr><td class="formula"><math id="d1e437" display="block" overflow="linebreak"><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">a</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">t</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">o</mi><mi mathvariant="normal">f</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">e</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">y</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">e</mi></mrow><mo>=</mo><mrow><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="normal">a</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">t</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">o</mi><mi mathvariant="normal">f</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">e</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">o</mi><mi mathvariant="normal">x</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">d</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">z</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">d</mi></mrow><mo>/</mo><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">d</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">d</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">y</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">e</mi></mrow><mo stretchy="false">)</mo></mrow><mo>/</mo><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">e</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">c</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">v</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">y</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">d</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="uneq2"><tr><td class="formula"><math id="d1e628" display="block" overflow="linebreak"><mo>=</mo><mrow><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="normal">a</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">t</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">o</mi><mi mathvariant="normal">f</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">e</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">o</mi><mi mathvariant="normal">x</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">d</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">z</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">d</mi></mrow><mo>/</mo><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">d</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">d</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">y</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">e</mi></mrow><mo stretchy="false">)</mo></mrow><mo>/</mo><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">e</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">c</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">v</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi></mrow><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">y</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">d</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="eq1"><tr>
<td class="formula"><math id="d1e765" display="block" overflow="linebreak"><mo>=</mo><mi>Q</mi><mo>/</mo><mrow><mo stretchy="false">(</mo><mi>z</mi><mi>F</mi><mi mathvariant="normal">Δ</mi><mi>i</mi><mo stretchy="false">)</mo></mrow></math></td>
<td class="label">1</td>
</tr></table>
<p>With this CMS method, we conducted
absolute quantitation of peptides
and proteins based on electrochemical oxidation of oxidizable cysteine,
tyrosine, and tryptophan residues.<em>
−


,−


</em> For peptides without oxidizable residues, they could be derivatized
to carry an electrochemical tag such as methylene blue, allowing them to be quantifiable by CMS after
derivatization. In this study, we extended the CMS quantitation approach
for peptides with PTMs such as phosphopeptides and glycopeptides.
For phosphorylated tyrosine peptides, their phosphates were removed
enzymatically to generate electroactive tyrosine peptides (<a href="#sch1" class="usa-link">Scheme 
</a>a), before CMS quantitation.
Alternatively, for electrochemically inactive phosphopeptides or glycopeptides,
peptide derivatization with DPDP (<a href="#sch1" class="usa-link">Scheme 
</a>b) was performed to introduce one electroactive
catechol tag, prior to CMS quantitation. Upon electrochemical oxidation
at a low oxidation potential (0.7 V vs Ag/AgCl), the catechol tag
tends to lose two protons and two electrons to form a quinone product
(<em>z</em> = 2, <a href="#sch1" class="usa-link">Scheme 
</a>b), facilitating CMS quantitation. Using these strategies,
several phosphopeptides, glycopeptides, as well as glycopeptides from
NIST mAb digest were successfully quantified, without using standards.</p>
<figure class="fig xbox font-sm" id="sch1"><h3 class="obj_head">1. Our Approaches for Absolute Quantitation of a PTM-Modified Peptide
by CMS (a) after Removing Phosphate Using Phosphatase and (b) after
Derivatization of Its N-Terminal with DPDP.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec7f/12371591/37cf3b5c64be/tg5c00047_0006.jpg" loading="lazy" id="gr1" height="1307" width="3501" alt="1"></p>
<div class="p text-right font-secondary"><a href="figure/sch1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec2"><h2 class="pmc_sec_title">Experimental Section</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">Chemicals</h3>
<p>N-terminal-protected amino acid BOC-Lys-OH,
Gly-Gly-Tyr-Arg (GGYR, HPLC grade), Arg-Gly-Asp (RGD, HPLC grade),
and 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES) buffer
were purchased from Sigma-Aldrich (St Louis, MO). Phosphorylated RRApSVA
(HPLC grade) was obtained from EMD Millipore (Temecula, CA). RRLIEDAEpYAARG,
TSTEPQpYQPGENL, alkaline phosphatase (1 KU) from bovine intestinal
mucosa, and NIST mAb antibody were bought from Sigma-Aldrich (St.
Louis, MO). NYIVGQPSS­(β-GlcNAc)­TGNL–OH was purchased
from Sussex Research (ON, Canada). 2,5-Dioxo-1-pyrrolidinyl 3,4-dihydroxybenzene
propanoate (DPDP) was bought from Aquila Pharmatech LLC (Waterville,
OH). Trypsin (sequencing grade) was purchased from Promega (Madison,
WI). Dithiothreitol (DTT), iodoacetamide (IAA), ammonium bicarbonate
(ABC), and sodium hydroxide were procured from Fisher Chemical (Fair
Lawn, NJ). Formic acid (FA) and acetonitrile were obtained from Fisher
Chemical (Fair Lawn, NJ), and deionized water used for sample preparation
was obtained by using a Millipore purification system (Burlington,
MA).</p></section><section id="sec2.2"><h3 class="pmc_sec_title">Instrument</h3>
<p>Our CMS system (apparatus shown in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1</a>) consisted of LC, electrochemistry
(EC), and MS. Basically, an ultraperformance LC (UPLC, Waters, Milford,
MA) was connected with an electrochemical thin-layer flow cell equipped
with either a glassy carbon electrode (GCD, 6 mm ID) or a boron-doped
diamond electrode (BDD, 8 mm ID) as the working electrode (WE). The
flow cell was further coupled to a high-resolution Orbitrap Q Exactive
mass spectrometer (Thermo Scientific, San Jose, CA). A Ag/AgCl (3
M NaCl) electrode (reference electrode (RE), catalog no. <em>M</em>
<sub>W</sub>-2021, BASi, West Lafayette, IN) was employed as a RE,
while the stainless-steel body of the cell functioned as the counter
electrode (CE).</p>
<p>Separation of peptides was achieved using either
an Agilent poroshell reversed-phase column (HPH–C18, 2.1 mm
× 100 mm, 1.9 μm, Agilent, Santa Clara, CA), a reversed-phase
column (BEH C18, 2.1 mm × 100 mm, 1.7 μm), or a hydrophilic
interaction liquid chromatography (HILIC) column (HALO Penta HILIC
column, 2.1 mm × 250 mm; 2.7 μm, used for the separation
of NIST mAb glycopeptides). The mobile phase flow rate was set at
0.2–0.3 mL/min. Mobile phase A used was water with 0.1% FA,
and mobile phase B was acetonitrile with 0.1% FA.</p>
<p>OriginPro
2018b was used to import and integrate the recorded electric
current peak to calculate the total electric charge of <em>Q</em>. The peptide flowing out of the electrochemical cell was online
analyzed using the Orbitrap mass spectrometer equipped with a heated
electrospray ionization (HESI) source with the following parameters:
sheath gas flow rate, 35; auxiliary gas flow rate, 10; spray voltage,
3.0 kV; sweep gas flow rate, 0; capillary temperature, 300 °C;
auxiliary gas heater temperature, 100 °C; and S-lens RF level,
50. Mass spectra were acquired by a Thermo Xcalibur (3.0.63). Before
each use, the glassy carbon WE underwent thorough cleaning by polishing
with an alumina slurry. In situ cleaning involved applying alternative
positive and negative potentials of + 2.0 V and – 2.0 V was
used for the BDD electrode.</p></section><section id="sec2.3"><h3 class="pmc_sec_title">Cyclic Voltammetry Experiment</h3>
<p>We first derivatized
the GGYR peptide using DPDP and isolated the derivatized peptide,
denoted as *GGYR (* represents the derivatized peptide), via fractionation
by Shimadzu LC (LC 40B XR). Using a homemade undivided cell, we utilized
20 mL of 200 μM intact GGYR or 30 μM of *GGYR for the
cyclic voltammetry (CV) experiments conducted in H<sub>2</sub>O:ACN:FA
(70:30:1). A 3 mm GCE electrode with an area of 0.07 cm<sup>2</sup> served as the WE, while a Ag/AgCl RE was positioned in close proximity
to the WE to regulate the potential. Additionally, a carbon cloth
of 5 cm<sup>2</sup> size was positioned opposite to the WE, serving
as the CE. Monitoring and recording of the oxidation current were
facilitated by an Epsilon EClipse potentiostat (BASi, West Lafayette,
IN, USA).</p></section><section id="sec2.4"><h3 class="pmc_sec_title">Alkaline Phosphatase Treatment</h3>
<p>Phosphatase was dissolved
in 1 mL of 10 mM tris–HCL (pH 8.0) containing 1 mM magnesium
chloride and then stored in the fridge (4 <sup>°</sup>C). 50
μL of RRLIEDAEpYAARG (1 mM) or 100 μL of TSTEPQpYQPGENL
(0.5 mM) was mixed with 20 μL of alkaline phosphatase (0.86
KU/mL) and incubated at 37 °C for 1 h. FA was added (final FA
concentration: 1% by volume) after the dephosphorylation reaction.
The sample was cooled to room temperature and diluted with 0.1% FA
to a final concentration of 1.0 μM for CMS analysis.</p></section><section id="sec2.5"><h3 class="pmc_sec_title">Protein Digestion</h3>
<p>A 40 μL portion of 10 μg/μL
(400 μg) of NIST mAb (ref: 8671) was denatured using 80 μL
of 8 M urea at room temperature and then vortexed for 15 min. Five
μL of 1 M DTT (freshly prepared) was added to the sample and
then shaken for 1 h at 37 °C in a water bath. A 20 μL portion
of 500 mM IAA was then added to the sample at room temperature in
the dark for 30 min. The sample mixture was then desalted and buffer-exchanged
with 50 mM ABC buffer by an Amicon 10 K molecular weight cutoff filter
(MWCOF) and then digested overnight with 20 μg of trypsin (protein/trypsin
ratio: 20:1).</p></section><section id="sec2.6"><h3 class="pmc_sec_title">Peptide Derivatization with DPDP</h3>
<p>For the proof-of-concept
experiment, we first derivatized the N-terminal-protected amino acid
BOC-Lys-OH with DPDP-NHS ester in a 1:10 molar ratio using H<sub>2</sub>O:ACN (50:50) as a solvent with pH 8.3 adjusted by NaOH. The sample
was derivatized for overnight reaction using mild vortexing in dark.
The same derivatization condition was applied for peptides RGD, RRA<sub>p</sub>SVA, and NYIVGQPSS­(β-GlcNAc) TGNL–OH. To maintain
a good buffering system for controlling pH and reduce reaction time,
we also derivatized the GGYR peptide using 100 mM HEPES buffer (pH
8.3) and using a 1:10 molar ratio (peptide/DPDP) for the derivatization
process.</p>
<p>For the derivatization of glycopeptides from mAb, the
mAb digest sample was vacuum-dried and reconstituted in 20 mM HEPES
buffer (pH 8.5) to prepare a 5.3 μM sample, which was then shaken
with DPDP (159 mM, 3000-fold in excess) overnight in the dark for
derivatization. The sample was dried out and reconstituted in ACN/H<sub>2</sub>O (65:35 v/v) to prepare a 5.3 μM derivatized sample.
An underivatized sample was also prepared to assist in the measurement
of the glycopeptide derivatization yields.</p></section><section id="sec2.7"><h3 class="pmc_sec_title">LC Separation for CMS Quantitation</h3>
<p>For CMS quantitation
of phosphopeptides RRLIEDAEpYAARG and TSTEPQpYQPGENL, after removal
of phosphate, 10 μL of 1.0 μM peptide solution was injected
by LC (10 pmol injection amount). The mobile phase flow rate was 200
μL/min. In a 10 min gradient elution, mobile phase B increased
from 2% to 70% in 3 min and reached 90% in 1 min. Then, mobile phase
B was kept at 90% for 3 min before returning to 2%.</p>
<p>For CMS
quantitation of derivatized peptides, the LC elution program was similar,
and the mobile phase flow rate was set 300 μL/min, and 5 μL
of 1 μM derivatized peptides (5 pmol injection amount) was injected
for analysis For the derivatized BOC-Lys-OH, the LC gradient elution
program started with 90% A for 1 min, 90% A to 40% A in 8 min, linear
gradient at 5% A for 2 min, and then 90% A for 2 min. For the derivatized
RGD and phosphopeptide RRA<sub>p</sub>SVA, the LC gradient elution
program started with 90% A for 1 min, 90% A to 80% A in 3 min, linear
gradient at 5% A for 1 min, and then 90% A for 1 min. For the derivatized
NYIVGQPSS­(β-GlcNAc) TGNL–OH LC, the gradient elution
program started with 95% A for 1 min, 95% A to 70% A in 10 min, linear
gradient at 5% A for 3 min, and then 95% A for 2 min. For the derivatized
GGYR, the mobile phase started with 95% A, 95% A to 91% A in 10 min,
linear gradient at 5% A for 2 min, and then 95% A for 2 min. To avoid
interference of the HEPES buffer in the GGYR sample, the first 3 min
of eluate was directed to waste.</p>
<p>For CMS quantitation of derivatized
mAb glycopeptides, the mobile
phase flow rate was set at the 200 μL/min A gradient elution
program starting with 25% A for 5 min (the first 5 min of eluate was
directed to waste), 25% A to 35% A in 30 min, linear gradient at 60%
A for 5 min, and then 25% A for 5 min. A 10 μL portion of the
derivatized mAb digest (5.3 μM, injection amount 53 pmol) was
injected for CMS quantitation.</p></section></section><section id="sec3"><h2 class="pmc_sec_title">Results and Discussion</h2>
<p>The integration of MS with electrochemistry
(EC), namely, EC/MS,
has drawn considerable attention and found many applications, including
drug metabolism study, protein structural analysis, and electrochemical
reaction mechanism elucidation.<em>
−




</em> The method can also be used to capture elusive reaction intermediates<em>
−








</em> and enable electrosynthetic reaction screening.<em>
−

</em> As a new application
of the EC/MS technique, the unprecedented advantage of CMS is the
capability of absolute quantitation without using any standards or
establishing a calibration curve. In our previous research, we successfully
demonstrated the capability of CMS to quantify peptides containing
electroactive residues like cysteine, tyrosine, and tryptophan.<em>
−


,−


</em> In this study, a similar CMS
approach was adopted to quantify peptides containing phosphorylated
tyrosine residues after enzymatic removal of phosphate to generate
electrochemically active tyrosine peptides. As the removal of phosphate
by phosphatase is quantitative, the measured quantity of the resulting
tyrosine peptide can reflect the amount of the phosphopeptide precursor.
On the other hand, in a more general approach, phosphopeptides or
glycopeptides can be selectively derivatized by DPDP to carry an electrochemical
tag of catechol (<a href="#sch1" class="usa-link">Scheme 
</a>b), prior to CMS quantitation. In comparison to our previously used
methylene blue tag, the catechol tag
is more polar and water-soluble, leading to better peptide separation
by LC and therefore benefits CMS quantitation.</p>
<section id="sec3.1"><h3 class="pmc_sec_title">CMS Quantitation of Phosphopeptides after Phosphate Removal</h3>
<p>To explore the feasibility of CMS quantitation of phosphorylated
tyrosine peptides, one standard peptide TSTEPQpYQPGENL was first tested.
As shown in <a href="#fig1" class="usa-link">Figure 
</a>a,b, after incubation with alkaline phosphatase, the TSTEPQpYQPGENL
was quantitatively<em>
−

</em> converted into TSTEPQYQPGENL (the original peak of
TSTEPQpYQPGENL at 3.2 min disappeared (<a href="#fig1" class="usa-link">Figure 
</a>a); instead, the product TSTEPQYQPGENL peak
appeared at 3.5 min in the TIC diagram (<a href="#fig1" class="usa-link">Figure 
</a>b). When the intact TSTEPQpYQPGENL was injected,
no electric current peak was observed with an applied voltage of +
1.05 V for oxidation (<a href="#fig1" class="usa-link">Figure 
</a>g), indicating that phosphorylated tyrosine is not electrochemically
active. In contrast, after the injection of the dephosphorylated peptide
TSTEPQYQPGENL sample, a sharp electric current peak was observed under
the same oxidation conditions (<a href="#fig1" class="usa-link">Figure 
</a>h), which is in line with our previous observation
of electrochemical oxidation of tyrosine-containing peptides.<em>
,
</em> Indeed, this was further confirmed by the corresponding MS spectra
recorded upon electrochemical oxidation (<a href="#fig1" class="usa-link">Figure 
</a>c,d). Compared to the MS spectrum (<a href="#fig1" class="usa-link">Figure 
</a>c) without electrolysis,
the +2 ion of TSTEPQYQPGENL was observed at <em>m</em>/<em>z</em> 731.84, whereas a new peak at <em>m</em>/<em>z</em> 730.84 (<a href="#fig1" class="usa-link">Figure 
</a>d), corresponding to the +2 ion of the oxidized peptide product
and resulting from 2 electron and 2 proton losses upon tyrosine oxidation,<em>
,,
</em> was detected, when +1.05 V was
applied to the cell for oxidation. The integrated extracted ion chromatogram
(EIC) peak area of <em>m</em>/<em>z</em> 731.84 shown
in <a href="#fig1" class="usa-link">Figure 
</a>f was reduced
by 18.4% compared to the same peak in <a href="#fig1" class="usa-link">Figure 
</a>e, suggesting that the oxidation yield for
TSTEPQYQPGENL was 18.4% (see the detailed data in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>). The amount of the oxidized TSTEPQYQPGENL was calculated
to be 10.64 pmol based on the integration of the current peak area
shown in <a href="#fig1" class="usa-link">Figure 
</a>h
and the oxidation yield using <a href="#eq1" class="usa-link">eq 
</a>. Therefore, the measured amount of TSTEPQYQPGENL was 10.64
pmol (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>). A triplicate measurement
gave the average amount of TSTEPQYQPGENL to be 10.66 pmol, which turned
out to be close to the theoretical amount of 10.0 pmol, with a measurement
error of 6.6% (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>).</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371591_tg5c00047_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec7f/12371591/6570382180c4/tg5c00047_0001.jpg" loading="lazy" id="gr2" height="350" width="693" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>TICs of TSTEPQpYQPGENL
(a) before and (b) after adding phosphatase.
MS spectra of TSTEPQYQPGENL (c) when the cell was off and (d) when
the cell was turned on (applied potential: +1.05 V). The oxidation
product of TSTEPQYQPGENL was detected at <em>m</em>/<em>z</em> 730.84. EICs of TSTEPQYQPGENL at <em>m</em>/<em>z</em> 731.84 were acquired (e) when the cell was off and (f)
when the cell was turned on (applied potential: +1.05 V). Electric
current diagrams were collected from (g) the oxidation of TSTEPQpYQPGENL
and (h) the oxidation of TSTEPQYQPGENL.</p></figcaption></figure><p>RRLIEDAEpYAARG, another phosphopeptide that contains
phosphotyrosine,
was also analyzed in the same way by CMS. RRLIEDAEpYAARG was also
quantitatively converted into RRLIEDAEYAARG by phosphatase (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2a,b</a>). No electric current was detected
for RRLIEDAEpYAARG (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2g</a>), while
a sharp current was observed for the dephosphorylated peptide RRLIEDAEYAARG
sample (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2h</a>). The oxidation product
was seen at <em>m</em>/<em>z</em> 506.27 after electrolysis
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2d</a>). The oxidation yield was measured
by the comparison of the EIC peak area of <em>m</em>/<em>z</em> 506.94 before and after oxidation (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2e,f</a>). In a triplicate measurement, the averaged quantity
of this peptide measured by CMS was 10.03 pmol, which is very close
to the initial amount of the expected value of 10.00 pmol with a measurement
error 0.3% (detailed data is shown in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>). The results shown above confirm that CMS can be used for absolute
quantitation of phosphorylated tyrosine peptides, in combination with
enzymatic phosphate removal.</p></section><section id="sec3.2"><h3 class="pmc_sec_title">CMS Quantitation after Derivatization</h3>
<section id="sec3.2.1"><h4 class="pmc_sec_title">Quantitation of Standard Amino Acid and Peptides</h4>
<p>In
this study, we also developed an alternative CMS approach for quantifying
phosphopeptides and glycopeptides, in combination with the DPDP derivatization
strategy. First, we tested some model peptides to validate the quantitation
method and then applied it to quantify phosphopeptides and glycopeptides.</p>
<p>We initially chose the N-terminal-protected amino acid BOC-Lys-OH
as a model compound for our investigation. We observed a 94% reaction
efficiency for this amino acid, as evidenced by a decrease in the
EIC peak area of the intact peptide ion at <em>m</em>/<em>z</em> 247.1 from 5.80 × 10<sup>6</sup> before DPDP derivatization
to 3.44 × 10<sup>5</sup> after DPDP derivatization (shown in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S3</a>). For CMS quantitation, we injected
5 μL of 1 μM BOC-Lys-OH after its derivatization (injection
amount: 5.0 pmol). Additionally, a minor potential of 0.7 V (vs Ag/AgCl)
was applied to the WE of the flow cell to facilitate the sample oxidation.
Prior to oxidation, the +1 ion of DPDP-derivatized BOC-Lys-OH was
detected at <em>m</em>/<em>z</em> 411.2 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4a</a>). Another peak at <em>m</em>/<em>z</em> 409.2 was also seen, probably due to in-source
oxidation during sample ionization (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4a</a>). Upon electrochemical oxidation, the intensity of the +1 ion of
the DPDP-derivatized BOC-Lys-OH decreased (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4,b</a>). Conversely, the intensity peak at <em>m</em>/<em>z</em> 409.2 increased (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4,b</a>) due to electrochemical oxidation. The EIC peak area of the DPDP-derivatized
BOC-Lys-OH at <em>m</em>/<em>z</em> 411.2 decreased
by 21% following oxidation (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4,d</a>) compared to its preoxidation EIC peak (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4,c</a>), indicating an oxidation yield of 21% (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S3</a>). The total charge involved in peptide oxidation
was determined to be 1.67 × 10<sup>–7</sup>C through integration
of the peptide oxidation current (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4,f</a>). According to <a href="#eq1" class="usa-link">eq 
</a> and <em>z</em> = 2 for the oxidation of the catechol moiety,
it was determined that the amount of BOC-Lys-OH by CMS was 4.4 pmol,
after considering 94% derivatization yield. In triplicate measurements,
the average value of the three runs was calculated to be 4.7 pmol
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S3</a>). The theoretical amount of BOC-Lys-OH
used for CMS quantification was 5.0 pmol. The quantitation error of
BOC-Lys-OH was −5.4% (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S3</a>),
suggesting the quantitation feasibility of this alternative CMS approach.</p>
<p>Peptide RGD was then chosen as an additional test sample for CMS
quantification method validation. A derivatization yield of 94% for
this peptide using DPDP was measured using LC/MS in a similar way
as described above for sample BOC-Lys-OH. <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S5,a</a> displays the +1 ion of the derivatized RGD (<em>m</em>/<em>z</em> 511.2) and its oxidized product (<em>m</em>/<em>z</em> 509.2, probably due to in-source oxidation).
The EIC peak area of <em>m</em>/<em>z</em> 511.2 decreased
by 10% following oxidation, indicating an oxidation yield of 10% (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S5c,d</a>). The total charge Q during oxidation
was recorded at 7.16 × 10<sup>–8</sup>C. A theoretical
5.0 pmol of the peptide was injected, with an actual measured amount
of 4.7 pmol (calculated based on <a href="#eq1" class="usa-link">eq 
</a> with further consideration of the 94% derivatization
yield, see details in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S4</a>), reflecting
a slight measurement error of −6%.</p>
<p>Another peptide GGRY
was also selected and derivatized by DPDP
for testing, for the reason to show that the catechol tag can be selectively
oxidized without oxidizing its tyrosine residue when the applied potential
is low. In order to gain deeper insights into the oxidation properties
of the DPDP-derivatized peptide, we conducted CV of intact GGYR and
the derivatized GGRY, *GGYR. *GGRY was obtained by the reaction of
GGYR with DPDP followed by LC purification. A CV of 20 mL of 200 μM
intact GGYR peptide (dissolved in H<sub>2</sub>O:ACN:FA, 70:30:1 by
volume) was conducted. Upon initiating the CV experiment with a positive
scan from 0 to 1400 mV (vs Ag/AgCl), a discernible peak (I) at a potential
of 1050 mV corresponding to the oxidization of the tyrosine residue
of GGYR was observed as shown in <a href="#fig2" class="usa-link">Figure 
</a>a (the observed onset oxidation potential
for tyrosine is 900 mV). The oxidation of tyrosine at 1050 mV is consistent
with our previous observation. In contrast,
CV of 20 mL of 30 μM *GGYR (dissolved in H<sub>2</sub>O:ACN:FA,
70:30:1 by volume) was performed, two peaks (I and II) were noticed,
corresponding to the oxidation of the tyrosine moiety of *GGYR at
1050 mV (peak I, <a href="#fig2" class="usa-link">Figure 
</a>b), and the oxidation of the catechol tag of *GGYR at 700 mV (the
observed onset oxidation potential for the catechol tag is 620 mV,
peak II, <a href="#fig2" class="usa-link">Figure 
</a>b).
A background CV in H<sub>2</sub>O:ACN:FA (70:30:1 by volume) marked
by dashed lines in <a href="#fig2" class="usa-link">Figure 
</a>a,b was also collected to separate faradaic current from its
nonfaradic counterpart. From the CV plots shown in <a href="#fig2" class="usa-link">Figure 
</a>a,b, it was clear that the
selective oxidation of the tag of the derivatized peptide without
oxidizing the tyrosine residue can be achieved at a potential range
between 620 and 900 mV. Therefore, in our experiment, 700 or 800 mV
was selected as the oxidation potential for our CMS experiments. It
can be seen that our newly employed derivatizing agent DPDP offers
a distinct advantage by undergoing oxidation at a lower potential
of 0.7 V vs Ag/AgCl during CMS quantitation. This characteristic shows
that it is less susceptible to the phenomenon of surface fouling on
the electrode. Throughout our experimental endeavors, we observed
a reduction in the frequency of electrode cleaning procedures.</p>
<figure class="fig xbox font-sm" id="fig2"><h5 class="obj_head">2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371591_tg5c00047_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec7f/12371591/bcb7a2a98e2c/tg5c00047_0002.jpg" loading="lazy" id="gr3" height="634" width="696" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cyclic voltammetric
behaviors of (a) GGYR and (b) derivatized GGYR.
Mass spectra of derivatized GGYR (c) before electrochemical oxidation
and (d) after electrochemical oxidation at 0.7 V; EIC peaks of derivatized
GGYR at <em>m</em>/<em>z</em> 616.2 (e) before oxidation
and (f) after oxidation; diagram showing the electrochemical oxidation
currents of (g) the blank solvent and (h) the derivatized GGYR sample.</p></figcaption></figure><p>In order to improve the efficiency of peptide derivatization
using
the DPDP reagent, 100 mM HEPES buffer (pH 8.3) was used as the buffer
for GGYR derivatization, and a 100% derivatization yield was achieved
(no intact GGYR was seen after derivatization for 4 h), thus avoiding
the necessity of measuring the derivatization yield. This process
also significantly shortened the reaction time compared to that of
the traditional overnight protocol. Subsequently, *GGRY was subjected
to CMS measurement. Before introducing the sample to the electrochemical
cell, the first 3 min were diverted to waste to eliminate the HEPES
buffer. When the cell was turned off, an ion of <em>m</em>/<em>z</em> 616.2 was observed (<a href="#fig2" class="usa-link">Figure 
</a>c), corresponding to the protonated *GGRY.
Another ion at <em>m</em>/<em>z</em> 614.2 was also
seen (<a href="#fig2" class="usa-link">Figure 
</a>c), probably
due to in-source oxidation. The integrated EIC peak area for <em>m</em>/<em>z</em> 616.2 decreased by 7.2% following oxidation
(<a href="#fig2" class="usa-link">Figure 
</a>e,f), indicating
a peptide oxidation yield of 7.2%. The total oxidation charge (<em>Q</em>) was calculated as 6.92 × 10<sup>–8</sup> C
(<a href="#fig2" class="usa-link">Figure 
</a>g,h). We injected
an expected 5.0 pmol of peptide with an actual measured amount of
4.7 pmol (triplicate measurements, <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S5</a>), yielding a small measurement error of −5.4%.</p>
<p>To verify
that the oxidation did occur to the catechol tag located
in the N-terminus of the *GGRY peptide, both the oxidized peptide
product ion (<em>m</em>/<em>z</em> 614.2) and the
intact peptide ion (<em>m</em>/<em>z</em> 616.2) were
subject to MS/MS analysis. For this analysis, the *GGYR sample was
introduced into the electrochemical cell for oxidation at 0.7 V using
a syringe pump. The oxidation product was collected for analysis by
nanoelectrospray ionization, resulting in the detection of the oxidized
G’GYR ion (’ indicates the oxidized catechol tag) at <em>m</em>/<em>z</em> 614.2. As depicted in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S6</a>, upon collision induced dissociation, <em>m</em>/<em>z</em> 614.2 produced a fragment ion b<sub>2</sub>-2H
(<em>m</em>/<em>z</em> 277.1), whereas the intact
*GGYR ion (<em>m</em>/<em>z</em> 616.2) generated
b<sub>2</sub> (<em>m</em>/<em>z</em> 279.1), which
is in line with the occurrence of oxidation to the N-terminal catechol
tag of *GGYR indeed.</p>
<p>Our method’s quantitation sensitivity
was also evaluated
by investigating CMS quantitation by injecting 6 μL of 25 nM
*GGYR sample (total amount: 150 fmol *GGYR). The integrated EIC peak
area for <em>m</em>/<em>z</em> 616.2 decreased by
9.5% following oxidation (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S7c,d</a>),
indicating a peptide oxidation yield of 9.5%. The total oxidation
charge (<em>Q</em>) was calculated as 2.54 × 10<sup>–9</sup> C (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S7f</a>). The actual measured amount
of 146 fmol by CMS (triplicate measurements, <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S6</a>) showed a small measurement error of −3.0% in comparison
with the theoretical injection amount of 150 fmol.</p></section><section id="sec3.2.2"><h4 class="pmc_sec_title">Quantification of Phosphopeptides/Glycopeptides</h4>
<p>After
quantifying standard peptides, we applied the method to quantify phosphopeptides
and glycopeptides, as a demonstration of its application. A phosphorylated
peptide, RRApSVA, was selected as a test sample that had no oxidizable
amino acid residue. The derivatization yield of this peptide was determined
to be 68% by comparing the EIC peak area of the intact peptide (<em>m</em>/<em>z</em> 739.4) signal before and after the
derivatization reaction (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S7</a>). Prior
to oxidation, the +1 ion of derivatized RRA<sub>P</sub>SVA was detected
at <em>m</em>/<em>z</em> 903.4 (<a href="#fig3" class="usa-link">Figure 
</a>a). An in-source oxidation peak was seen
at <em>m</em>/<em>z</em> 901.4 (<a href="#fig3" class="usa-link">Figure 
</a>a), and its signal intensity increased after
triggering the oxidation by applying 0.7 V (vs Ag/AgCl) to the WE
of the cell (<a href="#fig3" class="usa-link">Figure 
</a>b). In-source oxidation caused the measurement complexity due to
an overlap of <em>m</em>/<em>z</em> 903.4 with the
M + 2 isotope peak of <em>m</em>/<em>z</em> 901.4.
To avoid this complexity, we selected the peak at <em>m</em>/<em>z</em> 904.4 to calculate the oxidation yield. Upon
oxidation, the integrated EIC peak area for <em>m</em>/<em>z</em> 904.4 was reduced by 5.5% (<a href="#fig3" class="usa-link">Figure 
</a>d) compared to before oxidation (<a href="#fig3" class="usa-link">Figure 
</a>c), indicating an
oxidation yield of 5.5%. The total charge <em>Q</em> involved
in the oxidation in this case was determined to be 3.65 × 10<sup>–8</sup>C (<a href="#fig3" class="usa-link">Figure 
</a>e,f). This <em>Q</em> value, along with the oxidation
yield (i.e., <a href="#eq1" class="usa-link">eq 
</a>), was
utilized to calculate the quantity of the derivatized peptide to be
3.43 pmol (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S7</a>). Considering the 68%
derivatization yield, the total peptide amount was calculated to be
5.0 pmol. Triplicate measurements gave an average peptide amount of
5.2 pmol (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S7</a>). Once again, this value
aligns well with the theoretical peptide amount of 5 pmol with a small
quantitation error of 3.4% (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S7</a>).</p>
<figure class="fig xbox font-sm" id="fig3"><h5 class="obj_head">3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371591_tg5c00047_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec7f/12371591/4917bb9da45f/tg5c00047_0003.jpg" loading="lazy" id="gr4" height="382" width="758" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mass spectra
of the *RRApSVA (a) before electrochemical oxidation
and (b) after electrochemical oxidation at 0.7 V; EIC peaks of the
*RRApSVA ion at <em>m</em>/<em>z</em> 904.4 (c) before
oxidation and (d) after oxidation; diagrams showing the electrochemical
oxidation currents of (e) the blank solvent and (f) *RRApSVA.</p></figcaption></figure><p>To investigate the CMS quantification method for
glycopeptide analysis,
a glycosylated peptide, NYIVGQPSS­(β-GlcNAc)­TGNL–OH, was
selected for testing. The derivatization yield of this peptide was
determined to be 98% by comparing the EIC peak area of the intact
peptide (<em>m</em>/<em>z</em> 776.9) signal before
and after the derivatization reaction (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S8</a>). The derivatized peptide was then measured using CMS. Before oxidation,
the +1 ion of *NYIVGQPSS­(β-GlcNAc)­TGNL–OH was detected
at <em>m</em>/<em>z</em> 858.9 (<a href="#fig4" class="usa-link">Figure 
</a>a), and its isotope peak at <em>m</em>/<em>z</em> 859.9 was used for calculation of the oxidation
yield. Upon peptide oxidation, the intensity at <em>m</em>/<em>z</em> 859.9 decreased by 5.3% after oxidation (<a href="#fig4" class="usa-link">Figure 
</a>d) compared to no
oxidation (<a href="#fig4" class="usa-link">Figure 
</a>c), indicating a peptide oxidation yield of 5.3%. The total charge <em>Q</em> involved in the oxidation in this case was determined
to be 5.2 × 10<sup>–8</sup>C (<a href="#fig4" class="usa-link">Figure 
</a>e,f). This <em>Q</em> value, along
with the oxidation yield, was utilized to calculate the quantity of
the derivatized peptide, which was 5.13 pmol. Considering the 98%
derivatization yield, the total average peptide amount was calculated
to be 5.25 pmol (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S8</a>). Once again,
this value aligns well with the theoretical peptide amount of 5 pmol,
with a small quantitation error of 5.0%.</p>
<figure class="fig xbox font-sm" id="fig4"><h5 class="obj_head">4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371591_tg5c00047_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec7f/12371591/12c53cae1bb2/tg5c00047_0004.jpg" loading="lazy" id="gr5" height="319" width="693" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mass spectra of *NYIVGQPSS­(β-GlcNAc)­TGNL–OH
(a) before
electrochemical oxidation at 0.7 V and (b) after electrochemical oxidation;
M represents *NYIVGQPSS­(β-GlcNAc)­TGNL–OH. EIC peaks of
the *NYIVGQPSS­(β-GlcNAc)­TGNL–OH at <em>m</em>/<em>z</em> 859.9 (c) before oxidation and (d) after oxidation;
diagrams showing the electrochemical oxidation currents of (e) the
blank solvent and (f) *NYIVGQPSS­(β-GlcNAc)­TGNL–OH.</p></figcaption></figure><p>As shown above, several peptides including phosphopeptides
and
glycopeptides were tested in this study, and the quantitation errors
were good and within ±7%. The results are summarized in <a href="#tbl1" class="usa-link">Table 
</a>. The results suggest
the feasibility and accuracy of our CMS for quantitation.</p>
<section class="tw xbox font-sm" id="tbl1"><h5 class="obj_head">1. Summary of CMS Quantitation Results
for the Measured Peptides.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">peptides</th>
<th align="center" colspan="1" rowspan="1">injected
amount (pmol)</th>
<th align="center" colspan="1" rowspan="1">measured
amount (pmol)</th>
<th align="center" colspan="1" rowspan="1">quantification
error (%)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">BOC-Lys-OH</td>
<td align="center" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">4.73</td>
<td align="center" colspan="1" rowspan="1">–5.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RGD</td>
<td align="center" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">4.70</td>
<td align="center" colspan="1" rowspan="1">–6.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GGYR</td>
<td align="center" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">4.73</td>
<td align="center" colspan="1" rowspan="1">–5.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TSTEPQpYQPGENL</td>
<td align="center" colspan="1" rowspan="1">10</td>
<td align="center" colspan="1" rowspan="1">10.66</td>
<td align="center" colspan="1" rowspan="1">6.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RRLIEDAEpYAARG</td>
<td align="center" colspan="1" rowspan="1">10</td>
<td align="center" colspan="1" rowspan="1">10.03</td>
<td align="center" colspan="1" rowspan="1">0.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RRA<sub>p</sub>SVA</td>
<td align="center" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">5.20</td>
<td align="center" colspan="1" rowspan="1">3.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NYIVGQPSS(β-GlcNAc) TGNL–OH</td>
<td align="center" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">5.25</td>
<td align="center" colspan="1" rowspan="1">5.0</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec3.2.3"><h4 class="pmc_sec_title">Quantitation of Glycopeptides from IgG</h4>
<p>As a demonstration
of the application of our CMS approach, we applied it for quantifying
glycopeptides of the antibody IgG. MS techniques for antibody quantification
in biotherapeutics typically employ a “bottom-up” strategy,
in which proteins are enzymatically digested into peptides. However,
quantitation of <em>N</em>-glycosylation is a challenging
task. The lack of commercially available <em>N</em>-glycosylated
standards precludes absolute quantitation, and thus, relative quantitation
of glycosylation is performed by the multiple reaction monitoring
method where relative comparisons of glycan/glycopeptide signals are
taken into account. In our study, using
CMS after peptide derivatization, we enabled standard-free quantification
of several glycopeptides originating from the heavy chain Fc of the
IgG (located on residue N297, see the sequence information on NIST
monoclonal antibody (mAb) in the <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a>).</p>
<p>A HILIC column was used for the separation of these glycopeptides
due to the high affinity of the glycopeptides. Previously, we reported NIST mAb glycopeptide quantitation
by CMS, but the direct electrochemical oxidation of those glycopeptides
was complicated due to the presence of two oxidizable tyrosine residues
in them. As the oxidation of one or both
of the tyrosine residues occurred, the CMS quantitation required the
assumption that the oxidation product resulting from one tyrosine
oxidation has the same ionization efficiency as the oxidation product
resulting from the oxidation of two tyrosine residues, making the
calculation not straightforward. In this study, we took the protocol
to derivatize these glycopeptides with DPDP, so the oxidation at a
low oxidation potential of 0.8 V could selectively occur in the catechol
tags of the derivatized glycopeptides rather than the tyrosine residues,
thus simplifying the quantitation calculation.</p>
<p>In our study,
the NIST mAb was subjected to trypsin digestion,
from which five glycopeptides (each with glycosylation at the N297
residue of the Fc heavy chain), EEQYNSTYR-G0F, EEQYNSTYR-G1F, EEQYNSTYR-G2F,
EEQYNSTYR-G1F­(-GlcNAc), and EEQYNSTYR-G2F­(Hex) (structures shown in <a href="#tbl2" class="usa-link">Table 
</a>) were identified.
The glycopeptides were derivatized using DPDP, and the derivatization
yields were found to be 93–98% (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S9</a>). Subsequently, the derivatized glycopeptides were separated by
a HILIC column, and their EICs are shown in <a href="#fig5" class="usa-link">Figure 
</a>a. The major glycoforms detected were *EEQYNSTYR-G0F,
*EEQYNSTYR-G1F, and *EEQYNSTYR-G2F based on observed EIC peak intensities,
while minor glycoforms included *EEQYNSTYR-G1F­(-GlcNAc) and *EEQYNSTYR-G2F­(Hex).
It is noteworthy that *EEQYNSTYR-G1F exhibited a split peak at 22.4
min (the third panel of <a href="#fig5" class="usa-link">Figure 
</a>a) due to partial separation of glycan isomers resulting from
differences in galactose α-3/6 linkage. Additionally, the glycopeptide *EEQYNSTYR-G1F­(-GlcNAc) eluted at
20.4 min (second panel of <a href="#fig5" class="usa-link">Figure 
</a>a), with an extra peak observed at 22.4 min, most likely
due to in-source fragmentation of the *EEQYNSTYR-G1F ion. The electric
oxidation current peaks of these derivatized glycopeptides are shown
in <a href="#fig5" class="usa-link">Figure 
</a>b.</p>
<section class="tw xbox font-sm" id="tbl2"><h5 class="obj_head">2. NIST mAb Glycopeptide Quantities Measured
by CMS.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec7f/12371591/562285301a97/tg5c00047_0007.jpg" loading="lazy" id="gr7" height="2517" width="3552" alt="graphic file with name tg5c00047_0007.jpg"></p>
<div class="p text-right font-secondary"><a href="table/tbl2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t2fn1">
<sup>a</sup><p class="display-inline">Structures using the consortium
for functional glycomics notation. Symbol representations of glycans:
galactose = yellow circles, mannose = green circles, N-acetylglucosamine
= blues boxes, and fucose = red triangles.</p>
</div></div></section><figure class="fig xbox font-sm" id="fig5"><h5 class="obj_head">5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371591_tg5c00047_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec7f/12371591/605ba3e5d7f7/tg5c00047_0005.jpg" loading="lazy" id="gr6" height="745" width="666" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) EICs of +2 ions of 5 glycopeptides from tryptic digestion of
IgG after derivatization. (b) Electric current responses of the derivatized
glycopeptides from IgG digest upon electrochemical oxidation.</p></figcaption></figure><p>For CMS quantitation, prior to electrolysis, the
doubly charged
ion of *EEQYNSTYR-G0F was detected at <em>m</em>/<em>z</em> 1399.5 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8,a</a>) along with an in-source
oxidation product peak at <em>m</em>/<em>z</em> 1398.5,
whereas postelectrolysis analysis revealed the increased intensity
of the oxidation product at <em>m</em>/<em>z</em> 1398.5
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8,b</a>). To avoid interference of
the isotope peak of the oxidized product to the ion intensity of the
intact peptide ion, we chose the EIC area change from <em>m</em>/<em>z</em> 1401.0, the M + 3 peak of <em>m</em>/<em>z</em> 1399.5, for the oxidation yield calculation. The oxidation
yield for *EEQYNSTYR-G0F was determined to be 4.3% (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S9</a>) based on the EIC peak area drop of <em>m</em>/<em>z</em> 1401.0 upon oxidation (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8g,h</a>). The oxidation current was recorded as 2.86 × 10<sup>–7</sup>C. According to <a href="#eq1" class="usa-link">eq 
</a>, the measured amount of *EEQYNSTYR-G0F was 29.4 pmol
(triplicate measurements, <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S9</a>). Considering
the 94% derivatization yield for this glycopeptide (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S9</a>), the amount of EEQYNSTYR-G0F was found to be 31.3
pmol. Similar calculations were performed for *EEQYNSTYR-G1F, *EEQYNSTYR-G2F,
*EEQYNSTYR-G1F­(-GlcNAc), and *EEQYNSTYR-G2F­(Hex), yielding the measured
quantities of 33.2, 6.2, 0.9, and 1.3 pmol, respectively (see <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S9</a>, <a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S8 and S9</a>). These glycopeptide quantities were also calculated relative
to the theoretical total amount of IgG glycopeptides, which was 106
pmol (10 μL of 5.3 μM IgG, totaling 53.0 pmol which would
produce 106 pmol glycopeptides in total by considering that one IgG
has two symmetrical heavy chain totals) and listed in <a href="#tbl2" class="usa-link">Table 
</a>. The summative amount of the
five measured glycopeptides was 72.9 pmol, which is in reasonable
agreement with the theoretical value of 106 pmol, with a discrepancy
of approximately −31.3% (<a href="#tbl2" class="usa-link">Table 
</a>). This discrepancy likely arose from sample losses
during sample preparation such as enzymatic digestion or from the
presence of additional minor glycopeptides that were not quantified. Nevertheless, our CMS results thus provide an
effective estimation of the precursor glycoprotein quantity. Furthermore,
our CMS results were compared with data from previous NIST mAb glycosylation
studies by Zhao et al. which involved
relative quantification of glycosylated peptides based on MS peak
areas. As presented in <a href="#tbl2" class="usa-link">Table 
</a>, the relative abundances of the three major glycopeptides
(*EEQYNSTYR-G0F, *EEQYNSTYR-G1F, and*EEQYNSTYR-G2F) determined by
CMS were close to the value reported by Zhao et al., representing
(29.5% vs 38.4%), (31.3% vs 34.7%), and (5.9% vs 7.4%) of total antibody
abundance, respectively. While the relative abundances determined
by CMS and Zhao’s method are similar, our CMS method uniquely
provides absolute quantification for each glycopeptide. These findings
underscore the effectiveness and potential utility of CMS for precise
glycopeptide quantification.</p></section></section></section><section id="sec4"><h2 class="pmc_sec_title">Conclusions</h2>
<p>In this study, we demonstrated an absolute
quantitation strategy
for quantifying phosphopeptides and glycopeptides by using the CMS
approach in combination with an electrochemical labeling technique.
Standard peptides, phosphopeptides, and glycopeptides were successfully
quantified by CMS with good quantitation accuracy. We also quantified
glycopeptides from digested IgG. By this approach, we were able to
estimate the amount of glycoprotein based on the abundances of surrogate
glycopeptides. The highlight of CMS is that it does not require an
isotope-labeled standard to achieve absolute quantity, which has the
potential to break the bottleneck in many current methods. It can
be seen that our method could be also applicable for quantifying peptides
or proteins with other types of PTMs such as <em>S</em>-nitrosation, <em>N</em>-acetylation, methylation, or lipidation, which could contribute
to quantitative proteomics.</p></section><section id="sec17"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371591/bin/tg5c00047_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">tg5c00047_si_001.pdf</a><sup> (914.8KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>This work was supported by NSF (CHE-2203284), NIH
(1R21GM148874-01), and New Jersey Health Foundation grants (PC 23-23
and PC 38-25).</p></section><section id="notes1"><p>The
Supporting Information is
available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsmeasuresciau.5c00047?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsmeasuresciau.5c00047</a>.</p>
<ul id="silist" class="list" style="list-style-type:none"><li><p>Schematic diagram; electric current and MS spectra;
EICs; CID MS/MS spectra; and MS data as well as experimental setup
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsmeasuresciau.5c00047/suppl_file/tg5c00047_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes3"><p>MTAH, YA,
BD, TY, AS, and QY performed the investigation and analysis. HC conceived
the project idea and directed the research, and HDD guided us on the
electrochemistry experiments. MTAH, YA, and HC wrote the manuscript.</p></section><section id="notes2"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Urban J.. A review on
recent trends in the phosphoproteomics workflow. From sample preparation
to data analysis. Anal. Chim. Acta. 2022;1199:338857. doi: 10.1016/j.aca.2021.338857.</cite> [<a href="https://doi.org/10.1016/j.aca.2021.338857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35227377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chim.%20Acta&amp;title=A%20review%20on%20recent%20trends%20in%20the%20phosphoproteomics%20workflow.%20From%20sample%20preparation%20to%20data%20analysis&amp;author=J.%20Urban&amp;volume=1199&amp;publication_year=2022&amp;pages=338857&amp;pmid=35227377&amp;doi=10.1016/j.aca.2021.338857&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Zittlau K., Nashier P., Cavarischia-Rega C., Macek B., Spät P., Nalpas N.. Recent progress in
quantitative phosphoproteomics. Expert Rev.
Proteom. 2023;20(12):469–482. doi: 10.1080/14789450.2023.2295872.</cite> [<a href="https://doi.org/10.1080/14789450.2023.2295872" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38116637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Rev.%20Proteom&amp;title=Recent%20progress%20in%20quantitative%20phosphoproteomics&amp;author=K.%20Zittlau&amp;author=P.%20Nashier&amp;author=C.%20Cavarischia-Rega&amp;author=B.%20Macek&amp;author=P.%20Sp%C3%A4t&amp;volume=20&amp;issue=12&amp;publication_year=2023&amp;pages=469-482&amp;pmid=38116637&amp;doi=10.1080/14789450.2023.2295872&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Cao L., Lih T. M., Hu Y., Schnaubelt M., Chen S.-Y., Zhou Y., Guo C., Dong M., Yang W., Eguez R. V.. et al. Characterization
of
core fucosylation via sequential enzymatic treatments of intact glycopeptides
and mass spectrometry analysis. Nat. Commun. 2022;13(1):3910. doi: 10.1038/s41467-022-31472-4.</cite> [<a href="https://doi.org/10.1038/s41467-022-31472-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9262967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35798744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Characterization%20of%20core%20fucosylation%20via%20sequential%20enzymatic%20treatments%20of%20intact%20glycopeptides%20and%20mass%20spectrometry%20analysis&amp;author=L.%20Cao&amp;author=T.%20M.%20Lih&amp;author=Y.%20Hu&amp;author=M.%20Schnaubelt&amp;author=S.-Y.%20Chen&amp;volume=13&amp;issue=1&amp;publication_year=2022&amp;pages=3910&amp;pmid=35798744&amp;doi=10.1038/s41467-022-31472-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Chen S.-Y., Clark D. J., Zhang H.. High-Throughput
Analyses of Glycans,
Glycosites, and Intact Glycopeptides Using C4-and C18/MAX-Tips and
Liquid Handling System. Curr. Protoc. 2021;1(7):e186.  doi: 10.1002/cpz1.186.</cite> [<a href="https://doi.org/10.1002/cpz1.186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8485138/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34232571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc&amp;title=High-Throughput%20Analyses%20of%20Glycans,%20Glycosites,%20and%20Intact%20Glycopeptides%20Using%20C4-and%20C18/MAX-Tips%20and%20Liquid%20Handling%20System&amp;author=S.-Y.%20Chen&amp;author=D.%20J.%20Clark&amp;author=H.%20Zhang&amp;volume=1&amp;issue=7&amp;publication_year=2021&amp;pages=e186&amp;pmid=34232571&amp;doi=10.1002/cpz1.186&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Kawahara R., Chernykh A., Alagesan K., Bern M., Cao W., Chalkley R. J., Cheng K., Choo M. S., Edwards N., Goldman R.. et al. Community evaluation of glycoproteomics informatics
solutions reveals high-performance search strategies for serum glycopeptide
analysis. Nat. Methods. 2021;18(11):1304–1316. doi: 10.1038/s41592-021-01309-x.</cite> [<a href="https://doi.org/10.1038/s41592-021-01309-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8566223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34725484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Community%20evaluation%20of%20glycoproteomics%20informatics%20solutions%20reveals%20high-performance%20search%20strategies%20for%20serum%20glycopeptide%20analysis&amp;author=R.%20Kawahara&amp;author=A.%20Chernykh&amp;author=K.%20Alagesan&amp;author=M.%20Bern&amp;author=W.%20Cao&amp;volume=18&amp;issue=11&amp;publication_year=2021&amp;pages=1304-1316&amp;pmid=34725484&amp;doi=10.1038/s41592-021-01309-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Yang G., Zhang H., Yi W., Yan S., Cao L.. Editorial:
Protein Glycosylation–Advances in Identification, Characterization
and Biological Function Elucidation Using Mass Spectrometry. Front. Chem. 2022;10:847242. doi: 10.3389/fchem.2022.847242.</cite> [<a href="https://doi.org/10.3389/fchem.2022.847242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8896496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35252121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Chem&amp;title=Editorial:%20Protein%20Glycosylation%E2%80%93Advances%20in%20Identification,%20Characterization%20and%20Biological%20Function%20Elucidation%20Using%20Mass%20Spectrometry&amp;author=G.%20Yang&amp;author=H.%20Zhang&amp;author=W.%20Yi&amp;author=S.%20Yan&amp;author=L.%20Cao&amp;volume=10&amp;publication_year=2022&amp;pages=847242&amp;pmid=35252121&amp;doi=10.3389/fchem.2022.847242&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Xie Y., Liu X., Zhao C., Chen S., Wang S., Lin Z., Robison F. M., George B. M., Flynn R. A., Lebrilla C. B.. et al. Development
and application of GlycanDIA workflow for glycomic analysis. bioRxiv. 2024;2024:2003.2012.584702. doi: 10.1101/2024.03.12.584702.</cite> [<a href="https://doi.org/10.1101/2024.03.12.584702" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12317082/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40750788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Development%20and%20application%20of%20GlycanDIA%20workflow%20for%20glycomic%20analysis&amp;author=Y.%20Xie&amp;author=X.%20Liu&amp;author=C.%20Zhao&amp;author=S.%20Chen&amp;author=S.%20Wang&amp;volume=2024&amp;publication_year=2024&amp;pages=2003.2012.584702&amp;pmid=40750788&amp;doi=10.1101/2024.03.12.584702&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Seok S.-H.. Structural
Insights into Protein Regulation by Phosphorylation and Substrate
Recognition of Protein Kinases/Phosphatases. Life. 2021;11(9):957. doi: 10.3390/life11090957.</cite> [<a href="https://doi.org/10.3390/life11090957" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8467178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34575106/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Life&amp;title=Structural%20Insights%20into%20Protein%20Regulation%20by%20Phosphorylation%20and%20Substrate%20Recognition%20of%20Protein%20Kinases/Phosphatases&amp;author=S.-H.%20Seok&amp;volume=11&amp;issue=9&amp;publication_year=2021&amp;pages=957&amp;pmid=34575106&amp;doi=10.3390/life11090957&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Possemato A. P., Paulo J. A., Mulhern D., Guo A., Gygi S. P., Beausoleil S. A.. Multiplexed Phosphoproteomic Profiling
Using Titanium
Dioxide and Immunoaffinity Enrichments Reveals Complementary Phosphorylation
Events. J. Proteome Res. 2017;16(4):1506–1514. doi: 10.1021/acs.jproteome.6b00905.</cite> [<a href="https://doi.org/10.1021/acs.jproteome.6b00905" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5538569/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28171727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=Multiplexed%20Phosphoproteomic%20Profiling%20Using%20Titanium%20Dioxide%20and%20Immunoaffinity%20Enrichments%20Reveals%20Complementary%20Phosphorylation%20Events&amp;author=A.%20P.%20Possemato&amp;author=J.%20A.%20Paulo&amp;author=D.%20Mulhern&amp;author=A.%20Guo&amp;author=S.%20P.%20Gygi&amp;volume=16&amp;issue=4&amp;publication_year=2017&amp;pages=1506-1514&amp;pmid=28171727&amp;doi=10.1021/acs.jproteome.6b00905&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Imre T., Schlosser G., Pocsfalvi G., Siciliano R., Molnár-Szöllosi E., Kremmer T., Malorni A., Vékey K.. Glycosylation
site analysis of human alpha-1-acid glycoprotein
(AGP) by capillary liquid chromatography-electrospray mass spectrometry. J. Mass Spectrom. 2005;40(11):1472–1483. doi: 10.1002/jms.938.</cite> [<a href="https://doi.org/10.1002/jms.938" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16261636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mass%20Spectrom.&amp;title=Glycosylation%20site%20analysis%20of%20human%20alpha-1-acid%20glycoprotein%20(AGP)%20by%20capillary%20liquid%20chromatography-electrospray%20mass%20spectrometry&amp;author=T.%20Imre&amp;author=G.%20Schlosser&amp;author=G.%20Pocsfalvi&amp;author=R.%20Siciliano&amp;author=E.%20Moln%C3%A1r-Sz%C3%B6llosi&amp;volume=40&amp;issue=11&amp;publication_year=2005&amp;pages=1472-1483&amp;pmid=16261636&amp;doi=10.1002/jms.938&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Banazadeh A., Veillon L., Wooding K. M., Zabet-moghaddam M., Mechref Y.. Recent advances in mass spectrometric analysis of glycoproteins. Electrophoresis. 2017;38(1):162–189. doi: 10.1002/elps.201600357.</cite> [<a href="https://doi.org/10.1002/elps.201600357" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5221507/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27757981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Electrophoresis&amp;title=Recent%20advances%20in%20mass%20spectrometric%20analysis%20of%20glycoproteins&amp;author=A.%20Banazadeh&amp;author=L.%20Veillon&amp;author=K.%20M.%20Wooding&amp;author=M.%20Zabet-moghaddam&amp;author=Y.%20Mechref&amp;volume=38&amp;issue=1&amp;publication_year=2017&amp;pages=162-189&amp;pmid=27757981&amp;doi=10.1002/elps.201600357&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Gutierrez-Reyes C. D., Jiang P., Atashi M., Bennett A., Yu A., Peng W., Zhong J., Mechref Y.. Advances in mass spectrometry-based
glycoproteomics: An update covering the period 2017–2021. Electrophoresis. 2022;43(1–2):370–387. doi: 10.1002/elps.202100188.</cite> [<a href="https://doi.org/10.1002/elps.202100188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34614238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Electrophoresis&amp;title=Advances%20in%20mass%20spectrometry-based%20glycoproteomics:%20An%20update%20covering%20the%20period%202017%E2%80%932021&amp;author=C.%20D.%20Gutierrez-Reyes&amp;author=P.%20Jiang&amp;author=M.%20Atashi&amp;author=A.%20Bennett&amp;author=A.%20Yu&amp;volume=43&amp;issue=1%E2%80%932&amp;publication_year=2022&amp;pages=370-387&amp;pmid=34614238&amp;doi=10.1002/elps.202100188&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Lee J. Y., Lee H. K., Park G. W., Hwang H., Jeong H. K., Yun K. N., Ji E. S., Kim K. H., Kim J. S., Kim J. W.. et al. Characterization of Site-Specific N-Glycopeptide Isoforms
of α-1-Acid Glycoprotein from an Interlaboratory Study Using
LC–MS/MS. J. Proteome Res. 2016;15(12):4146–4164. doi: 10.1021/acs.jproteome.5b01159.</cite> [<a href="https://doi.org/10.1021/acs.jproteome.5b01159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27760464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=Characterization%20of%20Site-Specific%20N-Glycopeptide%20Isoforms%20of%20%CE%B1-1-Acid%20Glycoprotein%20from%20an%20Interlaboratory%20Study%20Using%20LC%E2%80%93MS/MS&amp;author=J.%20Y.%20Lee&amp;author=H.%20K.%20Lee&amp;author=G.%20W.%20Park&amp;author=H.%20Hwang&amp;author=H.%20K.%20Jeong&amp;volume=15&amp;issue=12&amp;publication_year=2016&amp;pages=4146-4164&amp;pmid=27760464&amp;doi=10.1021/acs.jproteome.5b01159&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Roberts D. S., Mann M., Melby J. A., Larson E. J., Zhu Y., Brasier A. R., Jin S., Ge Y.. Structural O-Glycoform
Heterogeneity of the SARS-CoV-2 Spike Protein Receptor-Binding Domain
Revealed by Top-Down Mass Spectrometry. J. Am.
Chem. Soc. 2021;143(31):12014–12024. doi: 10.1021/jacs.1c02713.</cite> [<a href="https://doi.org/10.1021/jacs.1c02713" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8353889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34328324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Structural%20O-Glycoform%20Heterogeneity%20of%20the%20SARS-CoV-2%20Spike%20Protein%20Receptor-Binding%20Domain%20Revealed%20by%20Top-Down%20Mass%20Spectrometry&amp;author=D.%20S.%20Roberts&amp;author=M.%20Mann&amp;author=J.%20A.%20Melby&amp;author=E.%20J.%20Larson&amp;author=Y.%20Zhu&amp;volume=143&amp;issue=31&amp;publication_year=2021&amp;pages=12014-12024&amp;pmid=34328324&amp;doi=10.1021/jacs.1c02713&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Ai Y., Gunawardena H. P., Li X., Kim Y. I., Dewald H. D., Chen H.. Standard-Free Absolute
Quantitation of Antibody Deamidation Degradation
and Host Cell Proteins by Coulometric Mass Spectrometry. Anal. Chem. 2022;94(36):12490–12499. doi: 10.1021/acs.analchem.2c02709.</cite> [<a href="https://doi.org/10.1021/acs.analchem.2c02709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10492508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36018377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Standard-Free%20Absolute%20Quantitation%20of%20Antibody%20Deamidation%20Degradation%20and%20Host%20Cell%20Proteins%20by%20Coulometric%20Mass%20Spectrometry&amp;author=Y.%20Ai&amp;author=H.%20P.%20Gunawardena&amp;author=X.%20Li&amp;author=Y.%20I.%20Kim&amp;author=H.%20D.%20Dewald&amp;volume=94&amp;issue=36&amp;publication_year=2022&amp;pages=12490-12499&amp;pmid=36018377&amp;doi=10.1021/acs.analchem.2c02709&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Ai Y., Zhao P., Fnu P. I. J., Chen H.. Absolute Quantitation
of Tryptophan-Containing Peptides and Amyloid β-Peptide Fragments
by Coulometric Mass Spectrometry. J. Am. Soc.
Mass Spectrom. 2021;32(7):1771–1779. doi: 10.1021/jasms.1c00121.</cite> [<a href="https://doi.org/10.1021/jasms.1c00121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8925997/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34101439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Soc.%20Mass%20Spectrom.&amp;title=Absolute%20Quantitation%20of%20Tryptophan-Containing%20Peptides%20and%20Amyloid%20%CE%B2-Peptide%20Fragments%20by%20Coulometric%20Mass%20Spectrometry&amp;author=Y.%20Ai&amp;author=P.%20Zhao&amp;author=P.%20I.%20J.%20Fnu&amp;author=H.%20Chen&amp;volume=32&amp;issue=7&amp;publication_year=2021&amp;pages=1771-1779&amp;pmid=34101439&amp;doi=10.1021/jasms.1c00121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Fnu P. I. J., Hassan M. T. A., Yaroshuk T., Ai Y., Chen H.. Absolute Quantitation
of Peptides and Proteins by Coulometric Mass Spectrometry After Derivatization. Int. J. Mass Spectrom. 2024;495:117153. doi: 10.1016/j.ijms.2023.117153.</cite> [<a href="https://doi.org/10.1016/j.ijms.2023.117153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10673616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38009161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mass%20Spectrom.&amp;title=Absolute%20Quantitation%20of%20Peptides%20and%20Proteins%20by%20Coulometric%20Mass%20Spectrometry%20After%20Derivatization&amp;author=P.%20I.%20J.%20Fnu&amp;author=M.%20T.%20A.%20Hassan&amp;author=T.%20Yaroshuk&amp;author=Y.%20Ai&amp;author=H.%20Chen&amp;volume=495&amp;publication_year=2024&amp;pages=117153&amp;pmid=38009161&amp;doi=10.1016/j.ijms.2023.117153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Zhao P., Wang Q., Kaur M., Kim Y.-I., Dewald H. D., Mozziconacci O., Liu Y., Chen H.. Absolute Quantitation
of Proteins by Coulometric Mass Spectrometry. Anal. Chem. 2020;92(11):7877–7883. doi: 10.1021/acs.analchem.0c01151.</cite> [<a href="https://doi.org/10.1021/acs.analchem.0c01151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32368902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Absolute%20Quantitation%20of%20Proteins%20by%20Coulometric%20Mass%20Spectrometry&amp;author=P.%20Zhao&amp;author=Q.%20Wang&amp;author=M.%20Kaur&amp;author=Y.-I.%20Kim&amp;author=H.%20D.%20Dewald&amp;volume=92&amp;issue=11&amp;publication_year=2020&amp;pages=7877-7883&amp;pmid=32368902&amp;doi=10.1021/acs.analchem.0c01151&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Zhao P., Zare R. N., Chen H.. Absolute Quantitation of Oxidizable
Peptides by Coulometric Mass Spectrometry. J.
Am. Soc. Mass Spectrom. 2019;30(11):2398–2407. doi: 10.1007/s13361-019-02299-z.</cite> [<a href="https://doi.org/10.1007/s13361-019-02299-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31429055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Soc.%20Mass%20Spectrom.&amp;title=Absolute%20Quantitation%20of%20Oxidizable%20Peptides%20by%20Coulometric%20Mass%20Spectrometry&amp;author=P.%20Zhao&amp;author=R.%20N.%20Zare&amp;author=H.%20Chen&amp;volume=30&amp;issue=11&amp;publication_year=2019&amp;pages=2398-2407&amp;pmid=31429055&amp;doi=10.1007/s13361-019-02299-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Hoofnagle A. N.. Quantitative
Clinical Proteomics by Liquid Chromatography–Tandem Mass Spectrometry:
Assessing the Platform. Clin. Chem. 2010;56(2):161–164. doi: 10.1373/clinchem.2009.134049.</cite> [<a href="https://doi.org/10.1373/clinchem.2009.134049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19959622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Chem&amp;title=Quantitative%20Clinical%20Proteomics%20by%20Liquid%20Chromatography%E2%80%93Tandem%20Mass%20Spectrometry:%20Assessing%20the%20Platform&amp;author=A.%20N.%20Hoofnagle&amp;volume=56&amp;issue=2&amp;publication_year=2010&amp;pages=161-164&amp;pmid=19959622&amp;doi=10.1373/clinchem.2009.134049&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Xie F., Liu T., Qian W.-J., Petyuk V. A., Smith R. D.. Liquid Chromatography-Mass
Spectrometry-based Quantitative Proteomics. J. Biol. Chem. 2011;286(29):25443–25449. doi: 10.1074/jbc.R110.199703.</cite> [<a href="https://doi.org/10.1074/jbc.R110.199703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3138289/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21632532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Liquid%20Chromatography-Mass%20Spectrometry-based%20Quantitative%20Proteomics&amp;author=F.%20Xie&amp;author=T.%20Liu&amp;author=W.-J.%20Qian&amp;author=V.%20A.%20Petyuk&amp;author=R.%20D.%20Smith&amp;volume=286&amp;issue=29&amp;publication_year=2011&amp;pages=25443-25449&amp;pmid=21632532&amp;doi=10.1074/jbc.R110.199703&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Wasinger V. C., Zeng M., Yau Y.. Current Status and Advances in Quantitative
Proteomic Mass Spectrometry. Int J Proteomics. 2013;2013:180605. doi: 10.1155/2013/180605.</cite> [<a href="https://doi.org/10.1155/2013/180605" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3606794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23533757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Proteomics&amp;title=Current%20Status%20and%20Advances%20in%20Quantitative%20Proteomic%20Mass%20Spectrometry&amp;author=V.%20C.%20Wasinger&amp;author=M.%20Zeng&amp;author=Y.%20Yau&amp;volume=2013&amp;publication_year=2013&amp;pages=180605&amp;pmid=23533757&amp;doi=10.1155/2013/180605&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Sohn C. H., Lee J. E., Sweredoski M. J., Graham R. L., Smith G. T., Hess S., Czerwieniec G., Loo J. A., Deshaies R. J., Beauchamp J. L.. Click chemistry facilitates formation of reporter ions
and simplified synthesis of amine-reactive multiplexed isobaric tags
for protein quantification. J. Am. Chem. Soc. 2012;134(5):2672–2680. doi: 10.1021/ja2099003.</cite> [<a href="https://doi.org/10.1021/ja2099003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3668889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22225568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Click%20chemistry%20facilitates%20formation%20of%20reporter%20ions%20and%20simplified%20synthesis%20of%20amine-reactive%20multiplexed%20isobaric%20tags%20for%20protein%20quantification&amp;author=C.%20H.%20Sohn&amp;author=J.%20E.%20Lee&amp;author=M.%20J.%20Sweredoski&amp;author=R.%20L.%20Graham&amp;author=G.%20T.%20Smith&amp;volume=134&amp;issue=5&amp;publication_year=2012&amp;pages=2672-2680&amp;pmid=22225568&amp;doi=10.1021/ja2099003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Tan H., Wu Z., Wang H., Bai B., Li Y., Wang X., Zhai B., Beach T. G., Peng J.. Refined phosphopeptide
enrichment by phosphate additive and the analysis of human brain phosphoproteome. Proteomics. 2015;15(2–3):500–507. doi: 10.1002/pmic.201400171.</cite> [<a href="https://doi.org/10.1002/pmic.201400171" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4598062/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25307156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Refined%20phosphopeptide%20enrichment%20by%20phosphate%20additive%20and%20the%20analysis%20of%20human%20brain%20phosphoproteome&amp;author=H.%20Tan&amp;author=Z.%20Wu&amp;author=H.%20Wang&amp;author=B.%20Bai&amp;author=Y.%20Li&amp;volume=15&amp;issue=2%E2%80%933&amp;publication_year=2015&amp;pages=500-507&amp;pmid=25307156&amp;doi=10.1002/pmic.201400171&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Gerber S. A., Rush J., Stemman O., Kirschner M. W., Gygi S. P.. Absolute quantification of proteins and phosphoproteins
from cell lysates by tandem MS. Proc. Natl.
Acad. Sci. U.S.A. 2003;100(12):6940–6945. doi: 10.1073/pnas.0832254100.</cite> [<a href="https://doi.org/10.1073/pnas.0832254100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC165809/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12771378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Absolute%20quantification%20of%20proteins%20and%20phosphoproteins%20from%20cell%20lysates%20by%20tandem%20MS&amp;author=S.%20A.%20Gerber&amp;author=J.%20Rush&amp;author=O.%20Stemman&amp;author=M.%20W.%20Kirschner&amp;author=S.%20P.%20Gygi&amp;volume=100&amp;issue=12&amp;publication_year=2003&amp;pages=6940-6945&amp;pmid=12771378&amp;doi=10.1073/pnas.0832254100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Calderón-Celis F., Diez-Fernández S., Costa-Fernández J. M., Encinar J. R., Calvete J. J., Sanz-Medel A.. Elemental
Mass Spectrometry for Absolute Intact Protein Quantification without
Protein-Specific Standards: Application to Snake Venomics. Anal. Chem. 2016;88(19):9699–9706. doi: 10.1021/acs.analchem.6b02585.</cite> [<a href="https://doi.org/10.1021/acs.analchem.6b02585" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27593495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Elemental%20Mass%20Spectrometry%20for%20Absolute%20Intact%20Protein%20Quantification%20without%20Protein-Specific%20Standards:%20Application%20to%20Snake%20Venomics&amp;author=F.%20Calder%C3%B3n-Celis&amp;author=S.%20Diez-Fern%C3%A1ndez&amp;author=J.%20M.%20Costa-Fern%C3%A1ndez&amp;author=J.%20R.%20Encinar&amp;author=J.%20J.%20Calvete&amp;volume=88&amp;issue=19&amp;publication_year=2016&amp;pages=9699-9706&amp;pmid=27593495&amp;doi=10.1021/acs.analchem.6b02585&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Narumi R., Shimizu Y., Ukai-Tadenuma M., Ode K. L., Kanda G. N., Shinohara Y., Sato A., Matsumoto K., Ueda H. R.. Mass spectrometry-based
absolute quantification reveals
rhythmic variation of mouse circadian clock proteins. Proc. Natl. Acad. Sci. U.S.A. 2016;113(24):E3461–E3467. doi: 10.1073/pnas.1603799113.</cite> [<a href="https://doi.org/10.1073/pnas.1603799113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4914154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27247408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Mass%20spectrometry-based%20absolute%20quantification%20reveals%20rhythmic%20variation%20of%20mouse%20circadian%20clock%20proteins&amp;author=R.%20Narumi&amp;author=Y.%20Shimizu&amp;author=M.%20Ukai-Tadenuma&amp;author=K.%20L.%20Ode&amp;author=G.%20N.%20Kanda&amp;volume=113&amp;issue=24&amp;publication_year=2016&amp;pages=E3461-E3467&amp;pmid=27247408&amp;doi=10.1073/pnas.1603799113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>

Zhang, L.
; 
Elias, J.
E.
. 
Relative Protein
Quantification Using Tandem Mass Tag Mass Spectrometry. In Proteomics: Methods and Protocols; 
Comai, L.
, 
Katz, J.
E.
, 
Mallick, P.
, Eds.; Springer: New York, 2017; pp 185–198.</cite> [<a href="https://doi.org/10.1007/978-1-4939-6747-6_14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28188531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Proteomics:%20Methods%20and%20Protocols&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>

Colangelo, C. M.
; 
Williams, K. R.
. 
Isotope-Coded
Affinity Tags for Protein Quantification. In New and Emerging Proteomic Techniques; 
Nedelkov, D.
, 
Nelson, R. W.
, Eds.; Humana
Press, 2006; pp 151–158.</cite> [<a href="https://doi.org/10.1385/1-59745-026-X:151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16785647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=New%20and%20Emerging%20Proteomic%20Techniques&amp;publication_year=2006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Ong S.-E., Blagoev B., Kratchmarova I., Kristensen D. B., Steen H., Pandey A., Mann M.. Stable Isotope
Labeling
by Amino Acids in Cell Culture, SILAC, as a Simple and Accurate Approach
to Expression Proteomics. Mol Cell Proteomics. 2002;1(5):376–386. doi: 10.1074/mcp.M200025-MCP200.</cite> [<a href="https://doi.org/10.1074/mcp.M200025-MCP200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12118079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Proteomics&amp;title=Stable%20Isotope%20Labeling%20by%20Amino%20Acids%20in%20Cell%20Culture,%20SILAC,%20as%20a%20Simple%20and%20Accurate%20Approach%20to%20Expression%20Proteomics&amp;author=S.-E.%20Ong&amp;author=B.%20Blagoev&amp;author=I.%20Kratchmarova&amp;author=D.%20B.%20Kristensen&amp;author=H.%20Steen&amp;volume=1&amp;issue=5&amp;publication_year=2002&amp;pages=376-386&amp;pmid=12118079&amp;doi=10.1074/mcp.M200025-MCP200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>

Vélez-Bermúdez, I. C.
; 
Wen, T.-N.
; 
Lan, P.
; 
Schmidt, W.
. 
Isobaric
Tag for Relative and Absolute Quantitation (iTRAQ)-Based Protein Profiling
in Plants. In Plant Proteostasis: Methods
and Protocols; 
Lois, L. M.
, 
Matthiesen, R.
, Eds.; Springer: New York, 2016; pp 213–221.</cite> [<a href="https://doi.org/10.1007/978-1-4939-3759-2_17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27424757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Plant%20Proteostasis:%20Methods%20and%20Protocols&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Liu H., Zhang Y., Wang J., Wang D., Zhou C., Cai Y., Qian X.. Method for Quantitative Proteomics Research by Using
Metal Element Chelated Tags Coupled with Mass Spectrometry. Anal. Chem. 2006;78(18):6614–6621. doi: 10.1021/ac060895j.</cite> [<a href="https://doi.org/10.1021/ac060895j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16970341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Method%20for%20Quantitative%20Proteomics%20Research%20by%20Using%20Metal%20Element%20Chelated%20Tags%20Coupled%20with%20Mass%20Spectrometry&amp;author=H.%20Liu&amp;author=Y.%20Zhang&amp;author=J.%20Wang&amp;author=D.%20Wang&amp;author=C.%20Zhou&amp;volume=78&amp;issue=18&amp;publication_year=2006&amp;pages=6614-6621&amp;pmid=16970341&amp;doi=10.1021/ac060895j&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Chahrour O., Cobice D., Malone J.. Stable isotope labelling methods
in mass spectrometry-based quantitative proteomics. J. Pharm. Biomed. Anal. 2015;113:2–20. doi: 10.1016/j.jpba.2015.04.013.</cite> [<a href="https://doi.org/10.1016/j.jpba.2015.04.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25956803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Biomed.%20Anal.&amp;title=Stable%20isotope%20labelling%20methods%20in%20mass%20spectrometry-based%20quantitative%20proteomics&amp;author=O.%20Chahrour&amp;author=D.%20Cobice&amp;author=J.%20Malone&amp;volume=113&amp;publication_year=2015&amp;pages=2-20&amp;pmid=25956803&amp;doi=10.1016/j.jpba.2015.04.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Xiang F., Ye H., Chen R., Fu Q., Li L. N.. N-dimethyl leucines
as novel isobaric tandem mass tags for quantitative proteomics and
peptidomics. Anal. Chem. 2010;82(7):2817–2825. doi: 10.1021/ac902778d.</cite> [<a href="https://doi.org/10.1021/ac902778d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2859709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20218596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=N-dimethyl%20leucines%20as%20novel%20isobaric%20tandem%20mass%20tags%20for%20quantitative%20proteomics%20and%20peptidomics&amp;author=F.%20Xiang&amp;author=H.%20Ye&amp;author=R.%20Chen&amp;author=Q.%20Fu&amp;author=L.%20N.%20Li&amp;volume=82&amp;issue=7&amp;publication_year=2010&amp;pages=2817-2825&amp;pmid=20218596&amp;doi=10.1021/ac902778d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Molden R. C., Goya J., Khan Z., Garcia B. A.. Stable Isotope Labeling
of Phosphoproteins for Large-scale Phosphorylation Rate Determination. Mol Cell Proteomics. 2014;13(4):1106–1118. doi: 10.1074/mcp.O113.036145.</cite> [<a href="https://doi.org/10.1074/mcp.O113.036145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3977188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24532841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Proteomics&amp;title=Stable%20Isotope%20Labeling%20of%20Phosphoproteins%20for%20Large-scale%20Phosphorylation%20Rate%20Determination&amp;author=R.%20C.%20Molden&amp;author=J.%20Goya&amp;author=Z.%20Khan&amp;author=B.%20A.%20Garcia&amp;volume=13&amp;issue=4&amp;publication_year=2014&amp;pages=1106-1118&amp;pmid=24532841&amp;doi=10.1074/mcp.O113.036145&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>

Yu, K.
; 
Wang, Z.
; 
Wu, Z.
; 
Tan, H.
; 
Mishra, A.
; 
Peng, J.
. 
High-Throughput Profiling of Proteome Modifications
Proteomes and Posttranslational Modifications post translational Modifications
by 16-Plex TMT Labeling and Mass Spectrometry Massspectrometry. In Quantitative Methods in Proteomics; 
Marcus, K.
, 
Eisenacher, M.
, 
Sitek, B.
, Eds.; Springer US, 2021; pp 205–224.</cite> [<a href="https://doi.org/10.1007/978-1-0716-1024-4_15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8458009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33950493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20Methods%20in%20Proteomics&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Ankney J. A., Muneer A., Chen X.. Relative and
absolute quantitation
in mass spectrometry–based proteomics. Annu. Rev. Anal. Chem. 2018;11(1):49–77. doi: 10.1146/annurev-anchem-061516-045357.</cite> [<a href="https://doi.org/10.1146/annurev-anchem-061516-045357" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29894226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Anal.%20Chem&amp;title=Relative%20and%20absolute%20quantitation%20in%20mass%20spectrometry%E2%80%93based%20proteomics&amp;author=J.%20A.%20Ankney&amp;author=A.%20Muneer&amp;author=X.%20Chen&amp;volume=11&amp;issue=1&amp;publication_year=2018&amp;pages=49-77&amp;pmid=29894226&amp;doi=10.1146/annurev-anchem-061516-045357&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Beynon R. J., Doherty M. K., Pratt J. M., Gaskell S. J.. Multiplexed absolute
quantification in proteomics using artificial QCAT proteins of concatenated
signature peptides. Nat. Methods. 2005;2(8):587–589. doi: 10.1038/nmeth774.</cite> [<a href="https://doi.org/10.1038/nmeth774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16094383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Multiplexed%20absolute%20quantification%20in%20proteomics%20using%20artificial%20QCAT%20proteins%20of%20concatenated%20signature%20peptides&amp;author=R.%20J.%20Beynon&amp;author=M.%20K.%20Doherty&amp;author=J.%20M.%20Pratt&amp;author=S.%20J.%20Gaskell&amp;volume=2&amp;issue=8&amp;publication_year=2005&amp;pages=587-589&amp;pmid=16094383&amp;doi=10.1038/nmeth774&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Kettenbach A. N., Rush J., Gerber S. A.. Absolute
quantification of protein
and post-translational modification abundance with stable isotope-labeled
synthetic peptides. Nat. Protoc. 2011;6(2):175–186. doi: 10.1038/nprot.2010.196.</cite> [<a href="https://doi.org/10.1038/nprot.2010.196" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3736726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21293459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Absolute%20quantification%20of%20protein%20and%20post-translational%20modification%20abundance%20with%20stable%20isotope-labeled%20synthetic%20peptides&amp;author=A.%20N.%20Kettenbach&amp;author=J.%20Rush&amp;author=S.%20A.%20Gerber&amp;volume=6&amp;issue=2&amp;publication_year=2011&amp;pages=175-186&amp;pmid=21293459&amp;doi=10.1038/nprot.2010.196&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Kito K., Ito T.. Mass spectrometry-based
approaches toward absolute quantitative proteomics. Curr. Genomics. 2008;9(4):263–274. doi: 10.2174/138920208784533647.</cite> [<a href="https://doi.org/10.2174/138920208784533647" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2682933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19452043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Genomics&amp;title=Mass%20spectrometry-based%20approaches%20toward%20absolute%20quantitative%20proteomics&amp;author=K.%20Kito&amp;author=T.%20Ito&amp;volume=9&amp;issue=4&amp;publication_year=2008&amp;pages=263-274&amp;pmid=19452043&amp;doi=10.2174/138920208784533647&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Ludwig C., Claassen M., Schmidt A., Aebersold R.. Estimation
of Absolute Protein Quantities of Unlabeled Samples by Selected Reaction
Monitoring Mass Spectrometry. Mol Cell Proteomics. 2012;11(3):M111.013987. doi: 10.1074/mcp.M111.013987.</cite> [<a href="https://doi.org/10.1074/mcp.M111.013987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3316728/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22101334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Proteomics&amp;title=Estimation%20of%20Absolute%20Protein%20Quantities%20of%20Unlabeled%20Samples%20by%20Selected%20Reaction%20Monitoring%20Mass%20Spectrometry&amp;author=C.%20Ludwig&amp;author=M.%20Claassen&amp;author=A.%20Schmidt&amp;author=R.%20Aebersold&amp;volume=11&amp;issue=3&amp;publication_year=2012&amp;pages=M111.013987&amp;pmid=22101334&amp;doi=10.1074/mcp.M111.013987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Narumi R., Shimizu Y., Ukai-Tadenuma M., Ode K. L., Kanda G. N., Shinohara Y., Sato A., Matsumoto K., Ueda H. R.. Mass spectrometry-based
absolute quantification reveals
rhythmic variation of mouse circadian clock proteins. Proc. Natl. Acad. Sci. U.S.A. 2016;113(24):E3461–E3467. doi: 10.1073/pnas.1603799113.</cite> [<a href="https://doi.org/10.1073/pnas.1603799113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4914154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27247408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Mass%20spectrometry-based%20absolute%20quantification%20reveals%20rhythmic%20variation%20of%20mouse%20circadian%20clock%20proteins&amp;author=R.%20Narumi&amp;author=Y.%20Shimizu&amp;author=M.%20Ukai-Tadenuma&amp;author=K.%20L.%20Ode&amp;author=G.%20N.%20Kanda&amp;volume=113&amp;issue=24&amp;publication_year=2016&amp;pages=E3461-E3467&amp;pmid=27247408&amp;doi=10.1073/pnas.1603799113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Nosti A. J., Barrio L. C., Calderón Celis F., Soldado A., Encinar J. R.. Absolute quantification of proteins
using element mass
spectrometry and generic standards. J. Proteomics. 2022;256:104499. doi: 10.1016/j.jprot.2022.104499.</cite> [<a href="https://doi.org/10.1016/j.jprot.2022.104499" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35092838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteomics&amp;title=Absolute%20quantification%20of%20proteins%20using%20element%20mass%20spectrometry%20and%20generic%20standards&amp;author=A.%20J.%20Nosti&amp;author=L.%20C.%20Barrio&amp;author=F.%20Calder%C3%B3n%20Celis&amp;author=A.%20Soldado&amp;author=J.%20R.%20Encinar&amp;volume=256&amp;publication_year=2022&amp;pages=104499&amp;pmid=35092838&amp;doi=10.1016/j.jprot.2022.104499&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Raghuraman B. K., Bogdanova A., Moon H., Rzagalinski I., Geertsma E. R., Hersemann L., Shevchenko A.. Median-Based
Absolute Quantification of Proteins Using Fully Unlabeled Generic
Internal Standard (FUGIS) J. Proteome Res. 2022;21(1):132–141. doi: 10.1021/acs.jproteome.1c00596.</cite> [<a href="https://doi.org/10.1021/acs.jproteome.1c00596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8749952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34807614/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=Median-Based%20Absolute%20Quantification%20of%20Proteins%20Using%20Fully%20Unlabeled%20Generic%20Internal%20Standard%20(FUGIS)&amp;author=B.%20K.%20Raghuraman&amp;author=A.%20Bogdanova&amp;author=H.%20Moon&amp;author=I.%20Rzagalinski&amp;author=E.%20R.%20Geertsma&amp;volume=21&amp;issue=1&amp;publication_year=2022&amp;pages=132-141&amp;pmid=34807614&amp;doi=10.1021/acs.jproteome.1c00596&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Rzagalinski I., Bogdanova A., Raghuraman B. K., Geertsma E. R., Hersemann L., Ziemssen T., Shevchenko A.. FastCAT Accelerates Absolute Quantification
of Proteins Using Multiple Short Nonpurified Chimeric Standards. J. Proteome Res. 2022;21(6):1408–1417. doi: 10.1021/acs.jproteome.2c00014.</cite> [<a href="https://doi.org/10.1021/acs.jproteome.2c00014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9171895/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35561006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=FastCAT%20Accelerates%20Absolute%20Quantification%20of%20Proteins%20Using%20Multiple%20Short%20Nonpurified%20Chimeric%20Standards&amp;author=I.%20Rzagalinski&amp;author=A.%20Bogdanova&amp;author=B.%20K.%20Raghuraman&amp;author=E.%20R.%20Geertsma&amp;author=L.%20Hersemann&amp;volume=21&amp;issue=6&amp;publication_year=2022&amp;pages=1408-1417&amp;pmid=35561006&amp;doi=10.1021/acs.jproteome.2c00014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Schmidt C., Urlaub H.. Absolute quantification of proteins
using standard
peptides and multiple reaction monitoring. Methods
Mol. Biol. 2012;893:249–265. doi: 10.1007/978-1-61779-885-6_17.</cite> [<a href="https://doi.org/10.1007/978-1-61779-885-6_17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22665306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&amp;title=Absolute%20quantification%20of%20proteins%20using%20standard%20peptides%20and%20multiple%20reaction%20monitoring&amp;author=C.%20Schmidt&amp;author=H.%20Urlaub&amp;volume=893&amp;publication_year=2012&amp;pages=249-265&amp;pmid=22665306&amp;doi=10.1007/978-1-61779-885-6_17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>Silva J. C., Gorenstein M. V., Li G.-Z., Vissers J. P. C., Geromanos S. J.. Absolute
Quantification of Proteins by LCMSE: A Virtue of Parallel ms Acquisition
*S. Mol Cell Proteomics. 2006;5(1):144–156. doi: 10.1074/mcp.M500230-MCP200.</cite> [<a href="https://doi.org/10.1074/mcp.M500230-MCP200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16219938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Proteomics&amp;title=Absolute%20Quantification%20of%20Proteins%20by%20LCMSE:%20A%20Virtue%20of%20Parallel%20ms%20Acquisition%20*S&amp;author=J.%20C.%20Silva&amp;author=M.%20V.%20Gorenstein&amp;author=G.-Z.%20Li&amp;author=J.%20P.%20C.%20Vissers&amp;author=S.%20J.%20Geromanos&amp;volume=5&amp;issue=1&amp;publication_year=2006&amp;pages=144-156&amp;pmid=16219938&amp;doi=10.1074/mcp.M500230-MCP200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Yu K., Wang Z., Wu Z., Tan H., Mishra A., Peng J.. High-Throughput Profiling of Proteome
and Posttranslational Modifications
by 16-Plex TMT Labeling and Mass Spectrometry. Methods Mol. Biol. 2021;2228:205–224. doi: 10.1007/978-1-0716-1024-4_15.</cite> [<a href="https://doi.org/10.1007/978-1-0716-1024-4_15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8458009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33950493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&amp;title=High-Throughput%20Profiling%20of%20Proteome%20and%20Posttranslational%20Modifications%20by%2016-Plex%20TMT%20Labeling%20and%20Mass%20Spectrometry&amp;author=K.%20Yu&amp;author=Z.%20Wang&amp;author=Z.%20Wu&amp;author=H.%20Tan&amp;author=A.%20Mishra&amp;volume=2228&amp;publication_year=2021&amp;pages=205-224&amp;pmid=33950493&amp;doi=10.1007/978-1-0716-1024-4_15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Wang Q., Liu Z., Liu Y., Chen H.. Absolute Quantitation of N-Nitrosamines
by Coulometric Mass Spectrometry without Using Standards. J. Am. Soc. Mass Spectrom. 2022;33(5):875–884. doi: 10.1021/jasms.2c00064.</cite> [<a href="https://doi.org/10.1021/jasms.2c00064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9119692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35446584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Soc.%20Mass%20Spectrom.&amp;title=Absolute%20Quantitation%20of%20N-Nitrosamines%20by%20Coulometric%20Mass%20Spectrometry%20without%20Using%20Standards&amp;author=Q.%20Wang&amp;author=Z.%20Liu&amp;author=Y.%20Liu&amp;author=H.%20Chen&amp;volume=33&amp;issue=5&amp;publication_year=2022&amp;pages=875-884&amp;pmid=35446584&amp;doi=10.1021/jasms.2c00064&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Xu C., Zheng Q., Zhao P., Paterson J., Chen H.. A New Quantification
Method Using Electrochemical Mass Spectrometry. J. Am. Soc. Mass Spectrom. 2019;30(4):685–693. doi: 10.1007/s13361-018-2116-6.</cite> [<a href="https://doi.org/10.1007/s13361-018-2116-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30604392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Soc.%20Mass%20Spectrom.&amp;title=A%20New%20Quantification%20Method%20Using%20Electrochemical%20Mass%20Spectrometry&amp;author=C.%20Xu&amp;author=Q.%20Zheng&amp;author=P.%20Zhao&amp;author=J.%20Paterson&amp;author=H.%20Chen&amp;volume=30&amp;issue=4&amp;publication_year=2019&amp;pages=685-693&amp;pmid=30604392&amp;doi=10.1007/s13361-018-2116-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Zhao P., Guo Y., Dewald H. D., Chen H.. Improvements
for absolute quantitation
using electrochemical mass spectrometry. Int.
J. Mass Spectrom. 2019;443:41–45. doi: 10.1016/j.ijms.2019.05.017.</cite> [<a href="https://doi.org/10.1016/j.ijms.2019.05.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mass%20Spectrom.&amp;title=Improvements%20for%20absolute%20quantitation%20using%20electrochemical%20mass%20spectrometry&amp;author=P.%20Zhao&amp;author=Y.%20Guo&amp;author=H.%20D.%20Dewald&amp;author=H.%20Chen&amp;volume=443&amp;publication_year=2019&amp;pages=41-45&amp;doi=10.1016/j.ijms.2019.05.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>Chiu K.-Y., Ai Y., Tanim-Ai
Hassan M., Li X., Gunawardena H. P., Chen H.. Standards-Free Absolute Quantitation of Oxidizable Glycopeptides
by Coulometric Mass Spectrometry. J. Am. Soc.
Mass Spectrom. 2024;35(7):1441–1450. doi: 10.1021/jasms.4c00052.</cite> [<a href="https://doi.org/10.1021/jasms.4c00052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38815255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Soc.%20Mass%20Spectrom.&amp;title=Standards-Free%20Absolute%20Quantitation%20of%20Oxidizable%20Glycopeptides%20by%20Coulometric%20Mass%20Spectrometry&amp;author=K.-Y.%20Chiu&amp;author=Y.%20Ai&amp;author=M.%20Tanim-Ai%20Hassan&amp;author=X.%20Li&amp;author=H.%20P.%20Gunawardena&amp;volume=35&amp;issue=7&amp;publication_year=2024&amp;pages=1441-1450&amp;pmid=38815255&amp;doi=10.1021/jasms.4c00052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>Diehl G., Karst U.. On-line electrochemistry – MS and related techniques. Anal. Bioanal. Chem. 2002;373(6):390–398. doi: 10.1007/s00216-002-1281-3.</cite> [<a href="https://doi.org/10.1007/s00216-002-1281-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12172673/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Bioanal.%20Chem.&amp;title=On-line%20electrochemistry%20%E2%80%93%20MS%20and%20related%20techniques&amp;author=G.%20Diehl&amp;author=U.%20Karst&amp;volume=373&amp;issue=6&amp;publication_year=2002&amp;pages=390-398&amp;pmid=12172673&amp;doi=10.1007/s00216-002-1281-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>Permentier H. P., Bruins A. P., Bischoff R.. Electrochemistry-mass
spectrometry
in drug metabolism and protein research. Mini
Rev. Med. Chem. 2008;8(1):46–56. doi: 10.2174/138955708783331586.</cite> [<a href="https://doi.org/10.2174/138955708783331586" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18220984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mini%20Rev.%20Med.%20Chem.&amp;title=Electrochemistry-mass%20spectrometry%20in%20drug%20metabolism%20and%20protein%20research&amp;author=H.%20P.%20Permentier&amp;author=A.%20P.%20Bruins&amp;author=R.%20Bischoff&amp;volume=8&amp;issue=1&amp;publication_year=2008&amp;pages=46-56&amp;pmid=18220984&amp;doi=10.2174/138955708783331586&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>Li J., Dewald H. D., Chen H.. Online Coupling of Electrochemical
Reactions with Liquid Sample Desorption Electrospray Ionization-Mass
Spectrometry. Anal. Chem. 2009;81(23):9716–9722. doi: 10.1021/ac901975j.</cite> [<a href="https://doi.org/10.1021/ac901975j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19873992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Online%20Coupling%20of%20Electrochemical%20Reactions%20with%20Liquid%20Sample%20Desorption%20Electrospray%20Ionization-Mass%20Spectrometry&amp;author=J.%20Li&amp;author=H.%20D.%20Dewald&amp;author=H.%20Chen&amp;volume=81&amp;issue=23&amp;publication_year=2009&amp;pages=9716-9722&amp;pmid=19873992&amp;doi=10.1021/ac901975j&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Chintalapudi K., Badu-Tawiah A. K.. An integrated electrocatalytic nESI-MS platform for
quantification of fatty acid isomers directly from untreated biofluids. Chem. Sci. 2020;11(36):9891–9897. doi: 10.1039/D0SC03403G.</cite> [<a href="https://doi.org/10.1039/D0SC03403G" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8162127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34094249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Sci.&amp;title=An%20integrated%20electrocatalytic%20nESI-MS%20platform%20for%20quantification%20of%20fatty%20acid%20isomers%20directly%20from%20untreated%20biofluids&amp;author=K.%20Chintalapudi&amp;author=A.%20K.%20Badu-Tawiah&amp;volume=11&amp;issue=36&amp;publication_year=2020&amp;pages=9891-9897&amp;pmid=34094249&amp;doi=10.1039/D0SC03403G&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref57">
<cite>Tang S., Fan L., Cheng H., Yan X.. Incorporating Electro-Epoxidation
into Electrospray Ionization Mass Spectrometry for Simultaneous Analysis
of Negatively and Positively Charged Unsaturated Glycerophospholipids. J. Am. Soc. Mass Spectrom. 2021;32(9):2288–2295. doi: 10.1021/jasms.0c00356.</cite> [<a href="https://doi.org/10.1021/jasms.0c00356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33232136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Soc.%20Mass%20Spectrom.&amp;title=Incorporating%20Electro-Epoxidation%20into%20Electrospray%20Ionization%20Mass%20Spectrometry%20for%20Simultaneous%20Analysis%20of%20Negatively%20and%20Positively%20Charged%20Unsaturated%20Glycerophospholipids&amp;author=S.%20Tang&amp;author=L.%20Fan&amp;author=H.%20Cheng&amp;author=X.%20Yan&amp;volume=32&amp;issue=9&amp;publication_year=2021&amp;pages=2288-2295&amp;pmid=33232136&amp;doi=10.1021/jasms.0c00356&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref58">
<cite>Zhang X., Zhan J., Yu Z., Deng J., Li M., Shao Y.. Recent Advances in Real-Time Analysis of Electrochemical Reactions
by Electrochemical Mass Spectrometry. Chin.
J. Chem. 2023;41(2):214–224. doi: 10.1002/cjoc.202200523.</cite> [<a href="https://doi.org/10.1002/cjoc.202200523" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chin.%20J.%20Chem.&amp;title=Recent%20Advances%20in%20Real-Time%20Analysis%20of%20Electrochemical%20Reactions%20by%20Electrochemical%20Mass%20Spectrometry&amp;author=X.%20Zhang&amp;author=J.%20Zhan&amp;author=Z.%20Yu&amp;author=J.%20Deng&amp;author=M.%20Li&amp;volume=41&amp;issue=2&amp;publication_year=2023&amp;pages=214-224&amp;doi=10.1002/cjoc.202200523&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref59">
<cite>Hu J., Zhang N., Zhang P.-K., Chen Y., Xia X.-H., Chen H.-Y., Xu J.-J.. Coupling
a Wireless Bipolar Ultramicroelectrode
with Nano-electrospray Ionization Mass Spectrometry: Insights into
the Ultrafast Initial Step of Electrochemical Reactions. Angew. Chem., Int. Ed. 2020;59(41):18244–18248. doi: 10.1002/anie.202008577.</cite> [<a href="https://doi.org/10.1002/anie.202008577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32652831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.&amp;title=Coupling%20a%20Wireless%20Bipolar%20Ultramicroelectrode%20with%20Nano-electrospray%20Ionization%20Mass%20Spectrometry:%20Insights%20into%20the%20Ultrafast%20Initial%20Step%20of%20Electrochemical%20Reactions&amp;author=J.%20Hu&amp;author=N.%20Zhang&amp;author=P.-K.%20Zhang&amp;author=Y.%20Chen&amp;author=X.-H.%20Xia&amp;volume=59&amp;issue=41&amp;publication_year=2020&amp;pages=18244-18248&amp;pmid=32652831&amp;doi=10.1002/anie.202008577&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref60">
<cite>Brown T. A., Chen H., Zare R. N.. Identification of fleeting electrochemical
reaction intermediates using desorption electrospray ionization mass
spectrometry. J. Am. Chem. Soc. 2015;137(23):7274–7277. doi: 10.1021/jacs.5b03862.</cite> [<a href="https://doi.org/10.1021/jacs.5b03862" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26030136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Identification%20of%20fleeting%20electrochemical%20reaction%20intermediates%20using%20desorption%20electrospray%20ionization%20mass%20spectrometry&amp;author=T.%20A.%20Brown&amp;author=H.%20Chen&amp;author=R.%20N.%20Zare&amp;volume=137&amp;issue=23&amp;publication_year=2015&amp;pages=7274-7277&amp;pmid=26030136&amp;doi=10.1021/jacs.5b03862&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref61">
<cite>Brown T. A., Chen H., Zare R. N.. Detection
of the short-lived radical
cation intermediate in the electrooxidation of N,N-dimethylaniline
by mass spectrometry. Angew. Chem., Int. Ed.
Engl. 2015;54(38):11183–11185. doi: 10.1002/anie.201506316.</cite> [<a href="https://doi.org/10.1002/anie.201506316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26352029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.%20Engl.&amp;title=Detection%20of%20the%20short-lived%20radical%20cation%20intermediate%20in%20the%20electrooxidation%20of%20N,N-dimethylaniline%20by%20mass%20spectrometry&amp;author=T.%20A.%20Brown&amp;author=H.%20Chen&amp;author=R.%20N.%20Zare&amp;volume=54&amp;issue=38&amp;publication_year=2015&amp;pages=11183-11185&amp;pmid=26352029&amp;doi=10.1002/anie.201506316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref62">
<cite>Qiu R., Zhang X., Luo H., Shao Y.. Mass spectrometric
snapshots for electrochemical reactions. Chem.
Sci. 2016;7(11):6684–6688. doi: 10.1039/C6SC01978A.</cite> [<a href="https://doi.org/10.1039/C6SC01978A" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5355862/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28451110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Sci.&amp;title=Mass%20spectrometric%20snapshots%20for%20electrochemical%20reactions&amp;author=R.%20Qiu&amp;author=X.%20Zhang&amp;author=H.%20Luo&amp;author=Y.%20Shao&amp;volume=7&amp;issue=11&amp;publication_year=2016&amp;pages=6684-6688&amp;pmid=28451110&amp;doi=10.1039/C6SC01978A&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref63">
<cite>Yu J., Zhou Y., Hua X., Liu S., Zhu Z., Yu X.-Y.. Capturing the transient species at
the electrode–electrolyte
interface by in situ dynamic molecular imaging. Chem. Commun. 2016;52(73):10952–10955. doi: 10.1039/C6CC02893D.</cite> [<a href="https://doi.org/10.1039/C6CC02893D" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27500288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Commun.&amp;title=Capturing%20the%20transient%20species%20at%20the%20electrode%E2%80%93electrolyte%20interface%20by%20in%20situ%20dynamic%20molecular%20imaging&amp;author=J.%20Yu&amp;author=Y.%20Zhou&amp;author=X.%20Hua&amp;author=S.%20Liu&amp;author=Z.%20Zhu&amp;volume=52&amp;issue=73&amp;publication_year=2016&amp;pages=10952-10955&amp;pmid=27500288&amp;doi=10.1039/C6CC02893D&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref64">
<cite>Wang Z., Zhang Y., Liu B., Wu K., Thevuthasan S., Baer D. R., Zhu Z., Yu X. Y., Wang F.. In Situ Mass
Spectrometric Monitoring of the Dynamic Electrochemical Process at
the Electrode-Electrolyte Interface: a SIMS Approach. Anal. Chem. 2017;89(1):960–965. doi: 10.1021/acs.analchem.6b04189.</cite> [<a href="https://doi.org/10.1021/acs.analchem.6b04189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27936704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=In%20Situ%20Mass%20Spectrometric%20Monitoring%20of%20the%20Dynamic%20Electrochemical%20Process%20at%20the%20Electrode-Electrolyte%20Interface:%20a%20SIMS%20Approach&amp;author=Z.%20Wang&amp;author=Y.%20Zhang&amp;author=B.%20Liu&amp;author=K.%20Wu&amp;author=S.%20Thevuthasan&amp;volume=89&amp;issue=1&amp;publication_year=2017&amp;pages=960-965&amp;pmid=27936704&amp;doi=10.1021/acs.analchem.6b04189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref65">
<cite>Khanipour P., Löffler M., Reichert A. M., Haase F. T., Mayrhofer K. J. J., Katsounaros I.. Electrochemical Real-Time Mass Spectrometry
(EC-RTMS):
Monitoring Electrochemical Reaction Products in Real Time. Angew. Chem., Int. Ed. Engl. 2019;58(22):7273–7277. doi: 10.1002/anie.201901923.</cite> [<a href="https://doi.org/10.1002/anie.201901923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30882984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.%20Engl.&amp;title=Electrochemical%20Real-Time%20Mass%20Spectrometry%20(EC-RTMS):%20Monitoring%20Electrochemical%20Reaction%20Products%20in%20Real%20Time&amp;author=P.%20Khanipour&amp;author=M.%20L%C3%B6ffler&amp;author=A.%20M.%20Reichert&amp;author=F.%20T.%20Haase&amp;author=K.%20J.%20J.%20Mayrhofer&amp;volume=58&amp;issue=22&amp;publication_year=2019&amp;pages=7273-7277&amp;pmid=30882984&amp;doi=10.1002/anie.201901923&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref66">
<cite>Hasa B., Jouny M., Ko B. H., Xu B., Jiao F.. Flow Electrolyzer
Mass Spectrometry with a Gas-Diffusion Electrode Design. Angew. Chem., Int. Ed. 2021;60(6):3277–3282. doi: 10.1002/anie.202013713.</cite> [<a href="https://doi.org/10.1002/anie.202013713" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33090694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.&amp;title=Flow%20Electrolyzer%20Mass%20Spectrometry%20with%20a%20Gas-Diffusion%20Electrode%20Design&amp;author=B.%20Hasa&amp;author=M.%20Jouny&amp;author=B.%20H.%20Ko&amp;author=B.%20Xu&amp;author=F.%20Jiao&amp;volume=60&amp;issue=6&amp;publication_year=2021&amp;pages=3277-3282&amp;pmid=33090694&amp;doi=10.1002/anie.202013713&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref67">
<cite>Liu J., Yu K., Zhang H., He J., Jiang J., Luo H.. Mass spectrometric
detection of fleeting neutral intermediates generated in electrochemical
reactions. Chem. Sci. 2021;12(27):9494–9499. doi: 10.1039/D1SC01385H.</cite> [<a href="https://doi.org/10.1039/D1SC01385H" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8278903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34349924/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Sci.&amp;title=Mass%20spectrometric%20detection%20of%20fleeting%20neutral%20intermediates%20generated%20in%20electrochemical%20reactions&amp;author=J.%20Liu&amp;author=K.%20Yu&amp;author=H.%20Zhang&amp;author=J.%20He&amp;author=J.%20Jiang&amp;volume=12&amp;issue=27&amp;publication_year=2021&amp;pages=9494-9499&amp;pmid=34349924&amp;doi=10.1039/D1SC01385H&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref68">
<cite>Hu J., Wang T., Zhang W. J., Hao H., Yu Q., Gao H., Zhang N., Chen Y., Xia X. H., Chen H. Y.. et al. Dissecting the Flash Chemistry of Electrogenerated Reactive Intermediates
by Microdroplet Fusion Mass Spectrometry. Angew.
Chem., Int. Ed. Engl. 2021;60(34):18494–18498. doi: 10.1002/anie.202106945.</cite> [<a href="https://doi.org/10.1002/anie.202106945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34129259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.%20Engl.&amp;title=Dissecting%20the%20Flash%20Chemistry%20of%20Electrogenerated%20Reactive%20Intermediates%20by%20Microdroplet%20Fusion%20Mass%20Spectrometry&amp;author=J.%20Hu&amp;author=T.%20Wang&amp;author=W.%20J.%20Zhang&amp;author=H.%20Hao&amp;author=Q.%20Yu&amp;volume=60&amp;issue=34&amp;publication_year=2021&amp;pages=18494-18498&amp;pmid=34129259&amp;doi=10.1002/anie.202106945&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref69">
<cite>Wan Q., Chen S., Badu-Tawiah A. K.. An integrated mass spectrometry platform
enables picomole-scale real-time electrosynthetic reaction screening
and discovery. Chem. Sci. 2018;9(26):5724–5729. doi: 10.1039/C8SC00251G.</cite> [<a href="https://doi.org/10.1039/C8SC00251G" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6050606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30079181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Sci.&amp;title=An%20integrated%20mass%20spectrometry%20platform%20enables%20picomole-scale%20real-time%20electrosynthetic%20reaction%20screening%20and%20discovery&amp;author=Q.%20Wan&amp;author=S.%20Chen&amp;author=A.%20K.%20Badu-Tawiah&amp;volume=9&amp;issue=26&amp;publication_year=2018&amp;pages=5724-5729&amp;pmid=30079181&amp;doi=10.1039/C8SC00251G&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref70">
<cite>Wang Q., Wang Q., Zhang Y., Mohamed Y. M., Pacheco C., Zheng N., Zare R. N., Chen H.. Electrocatalytic redox
neutral [3 + 2] annulation of N-cyclopropylanilines and alkenes. Chem. Sci. 2021;12(3):969–975. doi: 10.1039/D0SC05665K.</cite> [<a href="https://doi.org/10.1039/D0SC05665K" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8179209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34163863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Sci.&amp;title=Electrocatalytic%20redox%20neutral%20%5B3%20+%202%5D%20annulation%20of%20N-cyclopropylanilines%20and%20alkenes&amp;author=Q.%20Wang&amp;author=Q.%20Wang&amp;author=Y.%20Zhang&amp;author=Y.%20M.%20Mohamed&amp;author=C.%20Pacheco&amp;volume=12&amp;issue=3&amp;publication_year=2021&amp;pages=969-975&amp;pmid=34163863&amp;doi=10.1039/D0SC05665K&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref71">
<cite>Cheng H., Yang T., Edwards M., Tang S., Xu S., Yan X.. Picomole-Scale Transition Metal Electrocatalysis Screening Platform
for Discovery of Mild C-C Coupling and C-H Arylation through in Situ
Anodically Generated Cationic Pd. J. Am. Chem.
Soc. 2022;144(3):1306–1312. doi: 10.1021/jacs.1c11179.</cite> [<a href="https://doi.org/10.1021/jacs.1c11179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35015550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Picomole-Scale%20Transition%20Metal%20Electrocatalysis%20Screening%20Platform%20for%20Discovery%20of%20Mild%20C-C%20Coupling%20and%20C-H%20Arylation%20through%20in%20Situ%20Anodically%20Generated%20Cationic%20Pd&amp;author=H.%20Cheng&amp;author=T.%20Yang&amp;author=M.%20Edwards&amp;author=S.%20Tang&amp;author=S.%20Xu&amp;volume=144&amp;issue=3&amp;publication_year=2022&amp;pages=1306-1312&amp;pmid=35015550&amp;doi=10.1021/jacs.1c11179&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref72">
<cite>Hsu C.-C., Xue L., Arrington J. V., Wang P., Paez Paez J. S., Zhou Y., Zhu J.-K., Tao W. A.. Estimating the Efficiency
of Phosphopeptide Identification by Tandem Mass Spectrometry. J. Am. Soc. Mass Spectrom. 2017;28(6):1127–1135. doi: 10.1007/s13361-017-1603-5.</cite> [<a href="https://doi.org/10.1007/s13361-017-1603-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5438756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28283928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Soc.%20Mass%20Spectrom.&amp;title=Estimating%20the%20Efficiency%20of%20Phosphopeptide%20Identification%20by%20Tandem%20Mass%20Spectrometry&amp;author=C.-C.%20Hsu&amp;author=L.%20Xue&amp;author=J.%20V.%20Arrington&amp;author=P.%20Wang&amp;author=J.%20S.%20Paez%20Paez&amp;volume=28&amp;issue=6&amp;publication_year=2017&amp;pages=1127-1135&amp;pmid=28283928&amp;doi=10.1007/s13361-017-1603-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref73">
<cite>Ho H.-P., Rathod P., Louis M., Tada C. K., Rahaman S., Mark K. J., Leng J., Dana D., Kumar S., Lichterfeld M.. et al. Studies on quantitative
phosphopeptide analysis
by matrix-assisted laser desorption/ionization mass spectrometry without
label, chromatography or calibration curves. Rapid Commun. Mass Spectrom. 2014;28(24):2681–2689. doi: 10.1002/rcm.7063.</cite> [<a href="https://doi.org/10.1002/rcm.7063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4225566/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25380489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rapid%20Commun.%20Mass%20Spectrom.&amp;title=Studies%20on%20quantitative%20phosphopeptide%20analysis%20by%20matrix-assisted%20laser%20desorption/ionization%20mass%20spectrometry%20without%20label,%20chromatography%20or%20calibration%20curves&amp;author=H.-P.%20Ho&amp;author=P.%20Rathod&amp;author=M.%20Louis&amp;author=C.%20K.%20Tada&amp;author=S.%20Rahaman&amp;volume=28&amp;issue=24&amp;publication_year=2014&amp;pages=2681-2689&amp;pmid=25380489&amp;doi=10.1002/rcm.7063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref74">
<cite>Smith J. R., Olivier M., Greene A. S.. Relative quantification of peptide
phosphorylation in a complex mixture using 18O labeling. Physiol. Genom. 2007;31(2):357–363. doi: 10.1152/physiolgenomics.00096.2007.</cite> [<a href="https://doi.org/10.1152/physiolgenomics.00096.2007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2771636/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17684036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physiol.%20Genom&amp;title=Relative%20quantification%20of%20peptide%20phosphorylation%20in%20a%20complex%20mixture%20using%2018O%20labeling&amp;author=J.%20R.%20Smith&amp;author=M.%20Olivier&amp;author=A.%20S.%20Greene&amp;volume=31&amp;issue=2&amp;publication_year=2007&amp;pages=357-363&amp;pmid=17684036&amp;doi=10.1152/physiolgenomics.00096.2007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref75">
<cite>Yang N., Goonatilleke E., Park D., Song T., Fan G., Lebrilla C. B.. Quantitation
of Site-Specific Glycosylation in Manufactured
Recombinant Monoclonal Antibody Drugs. Anal.
Chem. 2016;88(14):7091–7100. doi: 10.1021/acs.analchem.6b00963.</cite> [<a href="https://doi.org/10.1021/acs.analchem.6b00963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4955800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27311011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Quantitation%20of%20Site-Specific%20Glycosylation%20in%20Manufactured%20Recombinant%20Monoclonal%20Antibody%20Drugs&amp;author=N.%20Yang&amp;author=E.%20Goonatilleke&amp;author=D.%20Park&amp;author=T.%20Song&amp;author=G.%20Fan&amp;volume=88&amp;issue=14&amp;publication_year=2016&amp;pages=7091-7100&amp;pmid=27311011&amp;doi=10.1021/acs.analchem.6b00963&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref76">
<cite>Huang Y., Nie Y., Boyes B., Orlando R.. Resolving Isomeric Glycopeptide Glycoforms
with Hydrophilic Interaction Chromatography (HILIC) J. Biomol Tech. 2016;27(3):98–104. doi: 10.7171/jbt.16-2703-003.</cite> [<a href="https://doi.org/10.7171/jbt.16-2703-003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4972402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27582638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol%20Tech&amp;title=Resolving%20Isomeric%20Glycopeptide%20Glycoforms%20with%20Hydrophilic%20Interaction%20Chromatography%20(HILIC)&amp;author=Y.%20Huang&amp;author=Y.%20Nie&amp;author=B.%20Boyes&amp;author=R.%20Orlando&amp;volume=27&amp;issue=3&amp;publication_year=2016&amp;pages=98-104&amp;pmid=27582638&amp;doi=10.7171/jbt.16-2703-003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref77">
<cite>Lowenthal M. S., Liang Y., Phinney K. W., Stein S. E.. Quantitative bottom-up
proteomics depends on digestion conditions. Anal. Chem. 2014;86(1):551–558. doi: 10.1021/ac4027274.</cite> [<a href="https://doi.org/10.1021/ac4027274" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24294946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Quantitative%20bottom-up%20proteomics%20depends%20on%20digestion%20conditions&amp;author=M.%20S.%20Lowenthal&amp;author=Y.%20Liang&amp;author=K.%20W.%20Phinney&amp;author=S.%20E.%20Stein&amp;volume=86&amp;issue=1&amp;publication_year=2014&amp;pages=551-558&amp;pmid=24294946&amp;doi=10.1021/ac4027274&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref78">
<cite>Zhao J., Peng W., Dong X., Mechref Y.. Analysis of NIST Monoclonal
Antibody Reference Material Glycosylation Using the LC-MS/MS-Based
Glycoproteomic Approach. J. Proteome Res. 2021;20(1):818–830. doi: 10.1021/acs.jproteome.0c00659.</cite> [<a href="https://doi.org/10.1021/acs.jproteome.0c00659" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33196194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=Analysis%20of%20NIST%20Monoclonal%20Antibody%20Reference%20Material%20Glycosylation%20Using%20the%20LC-MS/MS-Based%20Glycoproteomic%20Approach&amp;author=J.%20Zhao&amp;author=W.%20Peng&amp;author=X.%20Dong&amp;author=Y.%20Mechref&amp;volume=20&amp;issue=1&amp;publication_year=2021&amp;pages=818-830&amp;pmid=33196194&amp;doi=10.1021/acs.jproteome.0c00659&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371591/bin/tg5c00047_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">tg5c00047_si_001.pdf</a><sup> (914.8KB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Measurement Science Au are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsmeasuresciau.5c00047"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/tg5c00047.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371591/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371591/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371591%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371591/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371591/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371591/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371591/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371591/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371591/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Z9gSiFmhq6GG1J1iyP5hRoDwtxXykRz0ixkqaHEGpCx7JzrYREEc24SHFrcohFnU">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
