
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            A β‐1,2‐glucan‐associated glycoside hydrolase family 1 β‐glucosidase from Streptomyces griseus - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E484E18AEF1C030584E1000BF55BE9.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="prosci">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369401/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Protein Science : A Publication of the Protein Society">
<meta name="citation_title" content="A β‐1,2‐glucan‐associated glycoside hydrolase family 1 β‐glucosidase from Streptomyces griseus">
<meta name="citation_author" content="Haruto Kumakura">
<meta name="citation_author_institution" content="Department of Applied Biological Science, Faculty of Science and Technology, Tokyo University of Science, Noda, Japan">
<meta name="citation_author" content="Sei Motouchi">
<meta name="citation_author_institution" content="Department of Applied Biological Science, Faculty of Science and Technology, Tokyo University of Science, Noda, Japan">
<meta name="citation_author" content="Kaito Kobayashi">
<meta name="citation_author_institution" content="Artificial Intelligence Research Center, National Institute of Advanced Industrial Science and Technology (AIST), Tokyo, Japan">
<meta name="citation_author" content="Miyu Inoue">
<meta name="citation_author_institution" content="Department of Applied Biological Science, Faculty of Science and Technology, Tokyo University of Science, Noda, Japan">
<meta name="citation_author" content="Natsuki Kariuda">
<meta name="citation_author_institution" content="Department of Applied Biological Science, Faculty of Science and Technology, Tokyo University of Science, Noda, Japan">
<meta name="citation_author" content="Hiroyuki Nakai">
<meta name="citation_author_institution" content="Faculty of Agriculture, Niigata University, Niigata, Japan">
<meta name="citation_author" content="Masahiro Nakajima">
<meta name="citation_author_institution" content="Department of Applied Biological Science, Faculty of Science and Technology, Tokyo University of Science, Noda, Japan">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="34">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70255">
<meta name="citation_doi" content="10.1002/pro.70255">
<meta name="citation_pmid" content="40838837">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369401/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369401/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369401/pdf/PRO-34-e70255.pdf">
<meta name="description" content="β‐Glucosidases, major enzymes that release glucose from various natural compounds, are phylogenetically classified into glycoside hydrolase (GH) families. GH1 is the largest of these families. No β‐1,2‐glucan‐associated GH1 enzyme has been found, ...">
<meta name="og:title" content="A β‐1,2‐glucan‐associated glycoside hydrolase family 1 β‐glucosidase from Streptomyces griseus">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="β‐Glucosidases, major enzymes that release glucose from various natural compounds, are phylogenetically classified into glycoside hydrolase (GH) families. GH1 is the largest of these families. No β‐1,2‐glucan‐associated GH1 enzyme has been found, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369401/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12369401">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/pro.70255"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/PRO-34-e70255.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369401%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12369401/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12369401/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369401/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-prosci.gif" alt="Protein Science : A Publication of the Protein Society logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Protein Science : A Publication of the Protein Society" title="Link to Protein Science : A Publication of the Protein Society" shape="default" href="http://www.proteinscience.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Protein Sci</button></div>. 2025 Aug 21;34(9):e70255. doi: <a href="https://doi.org/10.1002/pro.70255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/pro.70255</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Protein%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Protein%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A β‐1,2‐glucan‐associated glycoside hydrolase family 1 β‐glucosidase from <em>Streptomyces griseus</em>
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumakura%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Haruto Kumakura</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Haruto Kumakura</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Applied Biological Science, Faculty of Science and Technology, Tokyo University of Science, Noda, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumakura%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haruto Kumakura</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Motouchi%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Sei Motouchi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Sei Motouchi</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Applied Biological Science, Faculty of Science and Technology, Tokyo University of Science, Noda, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Motouchi%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sei Motouchi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kobayashi%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Kaito Kobayashi</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Kaito Kobayashi</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Artificial Intelligence Research Center, National Institute of Advanced Industrial Science and Technology (AIST), Tokyo, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kobayashi%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kaito Kobayashi</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Inoue%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Miyu Inoue</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Miyu Inoue</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Applied Biological Science, Faculty of Science and Technology, Tokyo University of Science, Noda, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Inoue%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Miyu Inoue</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kariuda%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Natsuki Kariuda</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Natsuki Kariuda</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Applied Biological Science, Faculty of Science and Technology, Tokyo University of Science, Noda, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kariuda%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Natsuki Kariuda</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakai%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Hiroyuki Nakai</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Hiroyuki Nakai</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Faculty of Agriculture, Niigata University, Niigata, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakai%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hiroyuki Nakai</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakajima%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Masahiro Nakajima</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Masahiro Nakajima</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Applied Biological Science, Faculty of Science and Technology, Tokyo University of Science, Noda, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakajima%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Masahiro Nakajima</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="pro70255-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Applied Biological Science, Faculty of Science and Technology, Tokyo University of Science, Noda, Japan</div>
<div id="pro70255-aff-0002">
<sup>
<sup>2</sup>
</sup>Artificial Intelligence Research Center, National Institute of Advanced Industrial Science and Technology (AIST), Tokyo, Japan</div>
<div id="pro70255-aff-0003">
<sup>
<sup>3</sup>
</sup>Faculty of Agriculture, Niigata University, Niigata, Japan</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, 
Masahiro Nakajima, Department of Applied Biological Science, Faculty of Science and Technology, Tokyo University of Science, 2641 Yamazaki, Noda, Chiba 278‐8510, Japan. 
Email: <span>m-nakajima@rs.tus.ac.jp</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jul 11; Received 2025 May 2; Accepted 2025 Jul 20; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Protein Science</em> published by Wiley Periodicals LLC on behalf of The Protein Society.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12369401  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40838837/" class="usa-link">40838837</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>β‐Glucosidases, major enzymes that release glucose from various natural compounds, are phylogenetically classified into glycoside hydrolase (GH) families. GH1 is the largest of these families. No β‐1,2‐glucan‐associated GH1 enzyme has been found, even though β‐1,2‐glucans are natural carbohydrates that are important for interaction between organisms and environmental adaptation. In this study, functional and structural analyses of a GH1 enzyme from <em>Streptomyces griseus</em> (SGR_2426 protein) were performed. SGR_2426 showed the highest hydrolytic activity toward <em>p</em>‐nitrophenyl β‐glucopyranoside among <em>p</em>‐nitrophenyl sugars. This enzyme showed hydrolytic activity toward β‐1,2‐glucooligosaccharides specifically among β‐linked glucooligosaccharides. A structure of the enzyme in complex with sophorose (β‐1,2‐glucodisaccharide) was obtained as a Michaelis complex. The six‐membered ring of the glucose unit at the reducing end of sophorose is positioned in a hydrophobic environment between Trp291 and Met171, while only residue Gln229 forms a hydrogen bond directly. Trp291 and Gln229 are proposed as candidates for the residues important for substrate specificity based on comparison with structurally characterized GH1 homologs. Mutational analysis of Trp291 and Gln229 suggested that Trp291 is important for substrate recognition but not for substrate specificity and that Gln229 is involved in substrate specificity. SGR_2426 is the first identified β‐1,2‐glucan‐associated β‐glucosidase in the GH1 family.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> glycoside glycosidase; β‐1,2‐glucan; β‐1,2‐glucanase; β‐1,2‐glucooligosaccharide; β‐glucosidase</p></section></section><section id="pro70255-sec-0001"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Carbohydrates are important biomolecules that serve as energy sources and play crucial roles in cell wall components as cell skeletons, storage polymers such as starch, and interactions between organisms such as infection and symbiosis (Lenardon et al. <a href="#pro70255-bib-0040" class="usa-link" aria-describedby="pro70255-bib-0040">2010</a>; Yamada et al. <a href="#pro70255-bib-0080" class="usa-link" aria-describedby="pro70255-bib-0080"><sup>2017</sup></a>; Zeeman et al. <a href="#pro70255-bib-0081" class="usa-link" aria-describedby="pro70255-bib-0081"><sup>2010</sup></a>). Carbohydrates have complex and diverse structures, corresponding to the wide variety of enzymes responsible for their synthesis and degradation. These enzymes have been classified into four main categories in the Carbohydrate‐Active enZYmes database: glycoside hydrolases (GHs), glycosyltransferases, polysaccharide lyases, and carbohydrate esterases; redox enzymes were recently added to this classification as a group called “auxiliary activity” (Drula et al. <a href="#pro70255-bib-0015" class="usa-link" aria-describedby="pro70255-bib-0015">2022</a>; Henrissat and Davies <a href="#pro70255-bib-0025" class="usa-link" aria-describedby="pro70255-bib-0025"><sup>1997</sup></a>). The enzymes in this database are classified into families based on homology of their amino acid sequences. “GH family” is the most diverse category by number of families. As of Jul 2025, there are 194 GH families (including eight deleted families).</p>
<p>Glucose is one of the representative monosaccharides and forms glucans such as cellulose, laminarin, and starch. Numerous exo‐ and endo‐type enzymes that act on these glucans have been reported, including cellulases and cellobiohydrolases (β‐1,4‐linkages), β‐1,3‐glucanases, amylases, and glucoamylases (α‐1,4‐linkages), and their functions, structures, and molecular mechanisms have been studied extensively (Janeček et al. <a href="#pro70255-bib-0029" class="usa-link" aria-describedby="pro70255-bib-0029">2014</a>; Linton <a href="#pro70255-bib-0041" class="usa-link" aria-describedby="pro70255-bib-0041"><sup>2020</sup></a>). However, there is much less knowledge regarding enzymes that act on β‐1,2‐glucans compared with other enzyme groups.</p>
<p>β‐1,2‐Glucan was first discovered in the 1940s as a polymer secreted by <em>Agrobacterium</em>, a phytopathogen that causes crown gall on plant roots (McIntire et al. <a href="#pro70255-bib-0046" class="usa-link" aria-describedby="pro70255-bib-0046"><sup>1940</sup></a>; McIntire et al. <a href="#pro70255-bib-0047" class="usa-link" aria-describedby="pro70255-bib-0047"><sup>1942</sup></a>). The compound was later proved to be a cyclic glucan, and its physiological functions were reported (Putman et al. <a href="#pro70255-bib-0066" class="usa-link" aria-describedby="pro70255-bib-0066">1950</a>; Zevenhuizen and Scholten‐Koerselman <a href="#pro70255-bib-0082" class="usa-link" aria-describedby="pro70255-bib-0082"><sup>1979</sup></a>). Cyclic β‐1,2‐glucans in <em>Ensifer fredii</em> JCM20967 (formerly <em>Sinorhizobium</em> sp. strain NGR234) are important for symbiosis with plants. Cyclic β‐1,2‐glucans are also responsible for evasion of host defense systems during infection by <em>Brucella abortus</em> (Arellano‐Reynoso et al. <a href="#pro70255-bib-0007" class="usa-link" aria-describedby="pro70255-bib-0007"><sup>2005</sup></a>; Gay‐Fraret et al. <a href="#pro70255-bib-0019" class="usa-link" aria-describedby="pro70255-bib-0019"><sup>2012</sup></a>). Genes encoding cyclic β‐1,2‐glucan synthases were identified in <em>Rhizobium radiobacter</em> (<em>Agrobacterium tumefaciens</em>), <em>Ensifer meliloti</em> (<em>Sinorhizobium meliloti</em>), and <em>B. abortus</em> (Castro et al. <a href="#pro70255-bib-0009" class="usa-link" aria-describedby="pro70255-bib-0009">1996</a>; Iannino et al. <a href="#pro70255-bib-0027" class="usa-link" aria-describedby="pro70255-bib-0027"><sup>1998</sup></a>; Roset et al. <a href="#pro70255-bib-0069" class="usa-link" aria-describedby="pro70255-bib-0069"><sup>2004</sup></a>; Zorreguieta and Ugalde <a href="#pro70255-bib-0083" class="usa-link" aria-describedby="pro70255-bib-0083"><sup>1986</sup></a>). The detailed molecular mechanism of cyclic β‐1,2‐glucan synthesis has been elucidated (Ciocchini et al. <a href="#pro70255-bib-0011" class="usa-link" aria-describedby="pro70255-bib-0011">2007</a>; Sedzicki et al. <a href="#pro70255-bib-0072" class="usa-link" aria-describedby="pro70255-bib-0072"><sup>2024</sup></a>). However, prior to 2014, only a few studies reported enzymatic activities toward β‐1,2‐glucans originating from a bacterium and some filamentous fungi without gene identification (Kitahata and Edagawa <a href="#pro70255-bib-0035" class="usa-link" aria-describedby="pro70255-bib-0035">1987</a>; Mendoza and Amemura <a href="#pro70255-bib-0050" class="usa-link" aria-describedby="pro70255-bib-0050"><sup>1983</sup></a>; Nakajima <a href="#pro70255-bib-0056" class="usa-link" aria-describedby="pro70255-bib-0056"><sup>2023</sup></a>; Reese et al. <a href="#pro70255-bib-0067" class="usa-link" aria-describedby="pro70255-bib-0067"><sup>1961</sup></a>).</p>
<p>More recently, a novel glycoside phosphorylase that acts on β‐1,2‐glucooligosaccharides (Sop<sub>n</sub>s, where <em>n</em> is the degree of polymerization; “S” is an abbreviation for sophorooligosaccharides, an alternative name for β‐1,2‐glucooligosaccharides) was discovered from <em>Listeria innocua</em> (protein Lin1839 in GH94) (Figure <a href="#pro70255-fig-0001" class="usa-link">1</a>) (Nakajima et al. <a href="#pro70255-bib-0060" class="usa-link" aria-describedby="pro70255-bib-0060">2014</a>), which led to the development of a large‐scale production method of β‐1,2‐glucans (Abe et al. <a href="#pro70255-bib-0001" class="usa-link" aria-describedby="pro70255-bib-0001">2015</a>; Kobayashi et al. <a href="#pro70255-bib-0036" class="usa-link" aria-describedby="pro70255-bib-0036"><sup>2019</sup></a>; Nakajima et al. <a href="#pro70255-bib-0058" class="usa-link" aria-describedby="pro70255-bib-0058"><sup>2021</sup></a>). Identification of β‐1,2‐glucanases (SGLs) from a bacterium and a filamentous fungus by utilizing these polysaccharides led to the establishment of GH144 and GH162 families, respectively (Abe et al. <a href="#pro70255-bib-0002" class="usa-link" aria-describedby="pro70255-bib-0002">2017</a>; Tanaka et al. <a href="#pro70255-bib-0076" class="usa-link" aria-describedby="pro70255-bib-0076"><sup>2019</sup></a>). Recently, OpgG and OpgD from <em>Escherichia coli</em>, osmoregulated periplasmic glucan biosynthesis proteins with unknown biochemical functions, were found to be SGLs, which established a new GH family, GH186 (Motouchi et al. <a href="#pro70255-bib-0053" class="usa-link" aria-describedby="pro70255-bib-0053">2023</a>). In addition, an enzyme cyclizing a linear β‐1,2‐glucan to produce a cyclic β‐1,2‐glucan was found as a GH189‐establishing member. An enzyme that produces an α‐1,6‐cyclized β‐1,2‐glucan was also found from GH186 recently (Motouchi et al. <a href="#pro70255-bib-0054" class="usa-link" aria-describedby="pro70255-bib-0054">2024</a>; Tanaka et al. <a href="#pro70255-bib-0077" class="usa-link" aria-describedby="pro70255-bib-0077"><sup>2024</sup></a>).</p>
<figure class="fig xbox font-sm" id="pro70255-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/87b7/12369401/2f0595badc9a/PRO-34-e70255-g005.jpg" loading="lazy" id="jats-graphic-1" height="345" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/pro70255-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic representation of gene clusters related to the metabolism of Sop<sub>n</sub>s in <em>L. innocua</em> and <em>S. griseus</em> genomes. Locus tags without “Lin_” or “SGR_” are shown in arrowheads. The genes coding characterized proteins and the target protein in this study are shown in bold.</p></figcaption></figure><p>Along with the findings of new GH families, a novel exo‐type GH144 enzyme that releases a sophorose unit from Sop<sub>n</sub>s was discovered (Shimizu et al. <a href="#pro70255-bib-0075" class="usa-link" aria-describedby="pro70255-bib-0075">2018</a>), and a novel transglycosylase from GH35 that acts on β‐1,2‐glucosidic bonds was discovered by exploring an SGL gene cluster of a Gram‐negative bacterium (Kobayashi et al. <a href="#pro70255-bib-0037" class="usa-link" aria-describedby="pro70255-bib-0037">2022</a>). Furthermore, in the gene cluster of the <em>lin1839</em> gene in <em>L. innocua</em>, a sugar‐binding subunit of an ABC transporter that binds to Sop<sub>n</sub>s was found, and a GH3 enzyme was evidenced to be a β‐glucosidase (BGL) preferring Sop<sub>n</sub>s as substrates (proteins Lin1841 and Lin1840, respectively) (Abe et al. <a href="#pro70255-bib-0003" class="usa-link" aria-describedby="pro70255-bib-0003">2018</a>; Nakajima et al. <a href="#pro70255-bib-0060" class="usa-link" aria-describedby="pro70255-bib-0060"><sup>2014</sup></a>; Nakajima et al. <a href="#pro70255-bib-0061" class="usa-link" aria-describedby="pro70255-bib-0061"><sup>2016</sup></a>) (Figure <a href="#pro70255-fig-0001" class="usa-link">1</a>). These findings suggest that the products of the gene cluster can metabolize β‐1,2‐glucans and/or Sop<sub>n</sub>s. A GH3 homolog from <em>Bacteroides thetaiotaomicron</em> exhibited the same linkage position specificity as Lin1840 (Ishiguro et al. <a href="#pro70255-bib-0028" class="usa-link" aria-describedby="pro70255-bib-0028">2017</a>). Tertiary structures of these newly discovered enzymes are now available, and their structure–function relationships have been clearly explained (Abe et al. <a href="#pro70255-bib-0002" class="usa-link" aria-describedby="pro70255-bib-0002">2017</a>; Abe et al. <a href="#pro70255-bib-0003" class="usa-link" aria-describedby="pro70255-bib-0003"><sup>2018</sup></a>; Ishiguro et al. <a href="#pro70255-bib-0028" class="usa-link" aria-describedby="pro70255-bib-0028"><sup>2017</sup></a>; Kobayashi et al. <a href="#pro70255-bib-0037" class="usa-link" aria-describedby="pro70255-bib-0037"><sup>2022</sup></a>; Nakajima et al. <a href="#pro70255-bib-0061" class="usa-link" aria-describedby="pro70255-bib-0061"><sup>2016</sup></a>; Nakajima et al. <a href="#pro70255-bib-0057" class="usa-link" aria-describedby="pro70255-bib-0057"><sup>2017</sup></a>; Shimizu et al. <a href="#pro70255-bib-0075" class="usa-link" aria-describedby="pro70255-bib-0075"><sup>2018</sup></a>; Tanaka et al. <a href="#pro70255-bib-0076" class="usa-link" aria-describedby="pro70255-bib-0076"><sup>2019</sup></a>).</p>
<p>BGLs, which are essential enzymes for releasing glucose exolytically from the non‐reducing end, mainly belong to GH1 and GH3 families, and some BGLs are found in GH2, GH5, GH30, GH116, GH131, GH175, and GH180 (Broeker et al. <a href="#pro70255-bib-0008" class="usa-link" aria-describedby="pro70255-bib-0008">2018</a>; Ferrara et al. <a href="#pro70255-bib-0018" class="usa-link" aria-describedby="pro70255-bib-0018"><sup>2014</sup></a>; Helbert et al. <a href="#pro70255-bib-0024" class="usa-link" aria-describedby="pro70255-bib-0024"><sup>2019</sup></a>; Schröder et al. <a href="#pro70255-bib-0071" class="usa-link" aria-describedby="pro70255-bib-0071"><sup>2015</sup></a>; Shen et al. <a href="#pro70255-bib-0074" class="usa-link" aria-describedby="pro70255-bib-0074"><sup>2019</sup></a>). The GH1 family is so diverse in amino acid sequences that regions in substrate pockets recognizing moieties lacking a non‐reducing end glucose unit in a substrate show wide structural variety. However, no GH1 BGL involved in the metabolism of β‐1,2‐glucans and/or Sop<sub>n</sub>s has been reported. <em>Streptomyces griseus</em>, a bacterium famous for producing secondary metabolites (Lacey and Rutledge <a href="#pro70255-bib-0039" class="usa-link" aria-describedby="pro70255-bib-0039">2022</a>), has a homolog of the Sop<sub>n</sub>s‐binding subunit of the ABC transporter from <em>L. innocua</em> (Figure <a href="#pro70255-fig-0001" class="usa-link">1</a>). The gene cluster encoding this protein contains SGR_2426, a GH1 protein, and SGR_2427 protein belonging to GH193 (Figure <a href="#pro70255-fig-0001" class="usa-link">1</a>). Therefore, we hypothesized that SGR_2426 is involved in the metabolism of β‐1,2‐glucans and Sop<sub>n</sub>s. SGR_2426 has only been analyzed in screens for cellulose‐degrading enzymes, and its activity toward lactose, cellobiose, and xylobiose has been roughly investigated, though no proper quantitative data are available (Heins et al. <a href="#pro70255-bib-0023" class="usa-link" aria-describedby="pro70255-bib-0023">2014</a>). In this study, through functional and structural analyses, we reveal that SGR_2426 is a β‐1,2‐glucan‐associated BGL that acts preferentially on Sop<sub>n</sub>s as substrates.</p></section><section id="pro70255-sec-0002"><h2 class="pmc_sec_title">2. RESULTS</h2>
<section id="pro70255-sec-0003"><h3 class="pmc_sec_title">2.1. General properties of SGR_2426r</h3>
<p>SGR_2426 does not contain a predicted <em>N</em>‐terminal signal peptide. The recombinant SGR_2426 (SGR_2426r) His‐tagged at the <em>C</em>‐terminus was successfully expressed in <em>E. coli</em> and purified using a nickel affinity chromatography column (Figure <a href="#pro70255-supitem-0001" class="usa-link">S1a</a>, Supporting Information). The purified enzyme showed the highest hydrolytic activity toward <em>p</em>‐nitrophenyl‐β‐<span class="font-variant-small-caps">d</span>‐glucopyranoside (pNP‐β‐Glc) among pNP‐sugars tested. The enzyme showed approximately 31% and 27% relative activity toward pNP‐β‐<span class="font-variant-small-caps">d</span>‐fucopyranoside (pNP‐β‐<span class="font-variant-small-caps">d</span>‐Fuc) and pNP‐β‐<span class="font-variant-small-caps">d</span>‐galactopyranoside (pNP‐β‐Gal), respectively, in the presence of 5 mM substrate (Table <a href="#pro70255-tbl-0001" class="usa-link">1</a>). Thus, pNP‐β‐Glc was used as the substrate in investigations of pH and temperature profiles of enzyme activity (Figure <a href="#pro70255-supitem-0001" class="usa-link">S1b–e</a>). The enzyme showed high activity at pH 5.5–7.5. At pH 6.5, the enzyme showed similar activity in the presence of either MOPS or Bis‐Tris buffer, but MES buffer had some effect on the activity. Large differences in activities between buffers with the same pHs were also found at pH 5.5, 6.5, and 9.0, which may be due to buffer effect. SGR_2426r showed sufficient stability at pH 7.5–9.0, while the enzyme was not stable at pH 6.5. Thus, the enzyme was diluted using 50 mM MOPS (pH 7.5) and an assay of kinetics was performed at pH 6.5. Substrates of SGR_2426r may stabilize the enzyme at pH 5.5–6.5. SGR_2426r showed its highest activity at 30°C and was stable at up to 30°C, which is consistent with the optimum growth temperature of <em>S. griseus</em> (Gunjal and Bhagat <a href="#pro70255-bib-0022" class="usa-link" aria-describedby="pro70255-bib-0022">2021</a>). It should be noted that there is approximately 20% and 40% relative activity at 0 and 10°C, respectively, which implies that the enzyme is cold‐adaptive as in the case of Antarctic <em>Marinomonas</em> sp. ef1 (Gourlay et al. <a href="#pro70255-bib-0020" class="usa-link" aria-describedby="pro70255-bib-0020">2024</a>).</p>
<section class="tw xbox font-sm" id="pro70255-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Kinetic parameters of SGR_2426r.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1"></th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>V</em>
<sub>max</sub> (U/mg)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>m</sub> (mM)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>V</em>
<sub>max</sub>/<em>K</em>
<sub>m</sub> (U/mg/mM)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">Linear regression for <em>V</em>
<sub>max</sub>/<em>K</em>
<sub>m</sub>
<a href="#pro70255-note-0006" class="usa-link">
<sup>d</sup>
</a>
</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>V</em>
<sub>max2</sub> (U/mg)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>m2</sub> (mM)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>V</em>
<sub>max2</sub>/<em>K</em>
<sub>m2</sub> (U/mg/mM)</th>
</tr></thead>
<tbody valign="top">
<tr><td colspan="8" align="left" valign="top" rowspan="1">Wild‐type</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">pNP‐β‐Glc</td>
<td align="left" valign="top" rowspan="1" colspan="1">28 ± 1</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.0 ± 0.1</td>
<td align="left" valign="top" rowspan="1" colspan="1">28 ± 2</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">pNP‐β‐<span class="font-variant-small-caps">d</span>‐Fuc</td>
<td align="left" valign="top" rowspan="1" colspan="1">4.6 ± 0.3</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.16 ± 0.1</td>
<td align="left" valign="top" rowspan="1" colspan="1">29 ± 3.1</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1">21 ± 10</td>
<td align="left" valign="top" rowspan="1" colspan="1">27 ± 20</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.79 ± 0.26</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">pNP‐β‐Gal</td>
<td align="left" valign="top" rowspan="1" colspan="1">13 ± 1</td>
<td align="left" valign="top" rowspan="1" colspan="1">5.8 ± 0.3</td>
<td align="left" valign="top" rowspan="1" colspan="1">2.3 ± 0.1</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.9 ± 0.1</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sop<sub>2</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">36 ± 2</td>
<td align="left" valign="top" rowspan="1" colspan="1">17 ± 2</td>
<td align="left" valign="top" rowspan="1" colspan="1">2.0 ± 0.1</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.8 ± 0.1</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sop<sub>3</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">35 ± 2</td>
<td align="left" valign="top" rowspan="1" colspan="1">19 ± 2</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.9 ± 0.1</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.6 ± 0.1</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sop<sub>4</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">24 ± 2</td>
<td align="left" valign="top" rowspan="1" colspan="1">11 ± 2</td>
<td align="left" valign="top" rowspan="1" colspan="1">2.2 ± 0.2</td>
<td align="left" valign="top" rowspan="1" colspan="1">1.9 ± 0.2</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sop<sub>5</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">–<a href="#pro70255-note-0002" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">–<a href="#pro70255-note-0002" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">(3.4)</td>
<td align="left" valign="top" rowspan="1" colspan="1">3.8 ± 0.3</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Lam<sub>2</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">–<a href="#pro70255-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">–<a href="#pro70255-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.089 ± 0.006</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.079 ± 0.002</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Cel<sub>2</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">(&lt;0.01)<a href="#pro70255-note-0004" class="usa-link">
<sup>c</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Gen<sub>2</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">(&lt;0.01)<a href="#pro70255-note-0004" class="usa-link">
<sup>c</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Lam<sub>3–5</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">(&lt;0.01)<a href="#pro70255-note-0004" class="usa-link">
<sup>c</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr><td colspan="8" align="left" valign="top" rowspan="1">W291E</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sop<sub>2</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">4.7 ± 1.1</td>
<td align="left" valign="top" rowspan="1" colspan="1">36 ± 11</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.13 ± 0.01</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.11 ± 0.01</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Lam<sub>2</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">(0.0042)<a href="#pro70255-note-0004" class="usa-link">
<sup>c</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Cel<sub>2</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">(&lt;0.001)</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Gen<sub>2</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">(&lt;0.001)</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr><td colspan="8" align="left" valign="top" rowspan="1">Q229N</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sop<sub>2</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">2.1 ± 0.4</td>
<td align="left" valign="top" rowspan="1" colspan="1">33 ± 9</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.064 ± 0.005</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.057 ± 0.003</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Lam<sub>2</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">–<a href="#pro70255-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">–<a href="#pro70255-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.020 ± 0.002</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.017 ± 0.001</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Cel<sub>2</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">(&lt;0.001)</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Gen<sub>2</sub>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">(&lt;0.001)</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pro70255-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="pro70255-note-0002">
<sup>
<sup>a</sup>
</sup><p class="display-inline">The kinetic values were not determined, although the curve was fitted with the plots. The value in a parenthesis represents the slope when the plots (up to 2 mM) were applied to linear regression.</p>
</div>
<div class="fn" id="pro70255-note-0003">
<sup>
<sup>b</sup>
</sup><p class="display-inline">These values are not accurate because of the high <em>K</em>
<sub>m</sub> value. Thus, the calculated values are provided here for reference only. Lam<sub>2</sub> (WT), <em>V</em>
<sub>max</sub> = 8.4 ± 4.2, <em>K</em>
<sub>m</sub> = 95 ± 54; Lam<sub>2</sub> (Q229N), <em>V</em>
<sub>max</sub> = 1.8 ± 0.8, <em>K</em>
<sub>m</sub> = 92 ± 42.</p>
</div>
<div class="fn" id="pro70255-note-0004">
<sup>
<sup>c</sup>
</sup><p class="display-inline">Specific activity in the presence of 5 mM substrate.</p>
</div>
<div class="fn" id="pro70255-note-0006">
<sup>
<sup>d</sup>
</sup><p class="display-inline">Linear regression was performed using the data up to 1.5 mM for pNP‐Gal (WT), 2 mM for Sop5 (WT), 15 mM for Lam<sub>2</sub> (WT and Q229N), and 3 mM for the other samples as described in section <a href="#pro70255-sec-0011" class="usa-link">4</a>.</p>
</div>
</div></section></section><section id="pro70255-sec-0004"><h3 class="pmc_sec_title">2.2. Substrate specificity of pNP‐sugars</h3>
<p>SGR_2426r showed comparable hydrolytic activity toward pNP‐β‐Glc, pNP‐β‐<span class="font-variant-small-caps">d</span>‐Fuc, and pNP‐β‐Gal, and therefore kinetic analysis was performed using these substrates (Table <a href="#pro70255-tbl-0001" class="usa-link">1</a> and Figure <a href="#pro70255-fig-0002" class="usa-link">2a–c</a>). Data for pNP‐β‐Glc and pNP‐β‐Gal were regressed with the Michaelis–Menten equation. Contrarily, data for pNP‐β‐<span class="font-variant-small-caps">d</span>‐Fuc was regressed with a biphasic equation rather than the Michaelis–Menten equation, which suggests a second substrate molecule participates in the reaction. Thus, a transglycosylation reaction could occur on pNP‐β‐<span class="font-variant-small-caps">d</span>‐Fuc. The <em>V</em>
<sub>max</sub> value for pNP‐β‐Glc (28 U/mg) was approximately twice the value of pNP‐β‐Gal (13 U/mg). The <em>K</em>
<sub>m</sub> value for pNP‐β‐Glc (1.0 mM) was approximately 6 times lower than that for pNP‐β‐Gal (5.8 mM). Consequently, the <em>V</em>
<sub>max</sub>/<em>K</em>
<sub>m</sub> value for pNP‐β‐Glc was over 10 times higher than that for pNP‐β‐Gal. In the case of pNP‐β‐<span class="font-variant-small-caps">d</span>‐Fuc, although the <em>V</em>
<sub>max</sub> and <em>K</em>
<sub>m</sub> values were remarkably different from the <em>V</em>
<sub>max2</sub> and <em>K</em>
<sub>m2</sub> values, hydrolytic activity toward pNP‐β‐Glc was apparently higher than that for pNP‐β‐<span class="font-variant-small-caps">d</span>‐Fuc according to the plots. These results suggest that SGR_2426r is a β‐glycosidase with a preference for β‐glucosides.</p>
<figure class="fig xbox font-sm" id="pro70255-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/87b7/12369401/fa12e335f9d1/PRO-34-e70255-g001.jpg" loading="lazy" id="jats-graphic-3" height="884" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/pro70255-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Kinetic analysis of SGR_2426r. Medians of the triplicate experiments are used for plots. The other data are used for error bars. (a–h) The wild‐type, (i) W291E mutant, (j–k) Q229N mutant. (h, k) The fitting curves are depicted as dashed lines because the errors of <em>V</em>
<sub>max</sub> and <em>K</em>
<sub>m</sub> values are large.</p></figcaption></figure></section><section id="pro70255-sec-0005"><h3 class="pmc_sec_title">2.3. Substrate specificity of glucooligosaccharides</h3>
<p>Because SGR_2426r preferred a β‐glucoside among tested pNP‐sugars, its substrate specificity toward β‐linked glucooligosaccharides was investigated. In the presence of 5 mM glucodisaccharide, the specific activity was the highest toward Sop<sub>2</sub> (sophorose, β‐1,2‐glucodisaccharide). The enzyme exhibited approximately 5.2% of the specific activity toward Lam<sub>2</sub> (here, laminarioligosaccharides, β‐1,3‐glucooligosaccharides, are represented as Lam<sub>n</sub>, where <em>n</em> is the degree of polymerization) compared to Sop<sub>2</sub> (Table <a href="#pro70255-tbl-0001" class="usa-link">1</a> and Figure <a href="#pro70255-fig-0002" class="usa-link">2d–h</a>). The enzyme showed very low specific activities (&lt;0.001 U/mg) toward cellobiose (Cel<sub>2</sub>, β‐1,4‐glucodisaccharide) and gentiobiose (Gen<sub>2</sub>, β‐1,6‐glucodisaccharide). Thus, kinetic analysis toward Sop<sub>n</sub>s and Lam<sub>2</sub> was performed. <em>V</em>
<sub>max</sub> and <em>K</em>
<sub>m</sub> values and the resulting <em>V</em>
<sub>max</sub>/<em>K</em>
<sub>m</sub> values were comparable for Sop<sub>2–4</sub> (Table <a href="#pro70255-tbl-0001" class="usa-link">1</a> and Figure <a href="#pro70255-fig-0002" class="usa-link">2d–f</a>). While the <em>K</em>
<sub>m</sub> values for them were larger than a usual range of typical BGLs, the <em>V</em>
<sub>max</sub>/<em>K</em>
<sub>m</sub> values are within the range (Opassiri et al. <a href="#pro70255-bib-0063" class="usa-link" aria-describedby="pro70255-bib-0063">2004</a>), suggesting that Sop<sub>2–4</sub> are actual substrates of SGR_2426. Although kinetic parameters for Sop<sub>5</sub> were not determined by fitting the plots, unfortunately, the pattern of the plots could explain substrate inhibition or transglycosylation. Specific activity at low concentrations was a similar level to those of Sop<sub>2–4</sub> (Table <a href="#pro70255-tbl-0001" class="usa-link">1</a> and Figure <a href="#pro70255-fig-0002" class="usa-link">2g</a>). This result suggests that Sop<sub>5</sub> is also an actual substrate for SGR_2426 as well as Sop<sub>2–4</sub>.</p>
<p>Next, kinetic analysis was performed with Lam<sub>2</sub> as the substrate (Table <a href="#pro70255-tbl-0001" class="usa-link">1</a> and Figure <a href="#pro70255-fig-0002" class="usa-link">2h</a>). The <em>V</em>
<sub>max</sub> value was comparable to those for Sop<sub>2–4</sub>. The <em>V</em>
<sub>max</sub> and <em>K</em>
<sub>m</sub> values for Lam<sub>2</sub> were approximately 5 times lower and 5.5 times higher than those for Sop<sub>2</sub>, although their error values were somewhat high. The resulting <em>V</em>
<sub>max</sub>/<em>K</em>
<sub>m</sub> value was thus &gt;20 times smaller than that for Sop<sub>2</sub>, suggesting that Lam<sub>2</sub> is not a natural substrate of the enzyme. In addition, the enzyme showed low activity toward Lam<sub>3–5</sub> (&lt;0.001 U/mg). Overall, these results suggest that SGR_2426 is specific for the β‐1,2‐linkage in glucooligosaccharides.</p></section><section id="pro70255-sec-0006"><h3 class="pmc_sec_title">2.4. Structure of SGR_2426r</h3>
<p>A ligand‐free X‐ray crystal structure of SGR_2426r was determined at 2.20‐Å resolution (Table <a href="#pro70255-supitem-0001" class="usa-link">S1</a>). There are four chains in the asymmetric unit, each with almost the same configuration (root‐mean‐square deviation &lt;0.25 Å) (Figure <a href="#pro70255-fig-0003" class="usa-link">3a</a>). However, the biological assembly was judged to be a monomer according to the PISA server (<a href="https://www.ebi.ac.uk/msd-srv/prot_int/pistart.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/msd-srv/prot_int/pistart.html</a>) (Krissinel and Henrick <a href="#pro70255-bib-0038" class="usa-link" aria-describedby="pro70255-bib-0038">2007</a>). Similar to other structurally characterized GH1 enzymes, the enzyme possesses a (β/α)<sub>8</sub>‐barrel domain. There are, however, obvious differences in the shape of the substrate pocket compared with structurally characterized close homologs (Akiba et al. <a href="#pro70255-bib-0005" class="usa-link" aria-describedby="pro70255-bib-0005">2004</a>; Park et al. <a href="#pro70255-bib-0065" class="usa-link" aria-describedby="pro70255-bib-0065"><sup>2015</sup></a>; Seshadri et al. <a href="#pro70255-bib-0073" class="usa-link" aria-describedby="pro70255-bib-0073"><sup>2009</sup></a>), especially in the α‐helix region of residues 169–195 (Figure <a href="#pro70255-fig-0003" class="usa-link">3b–e</a>). This α‐helix (light‐brown in Figure <a href="#pro70255-fig-0003" class="usa-link">3b–e</a>) is inserted between the 3rd and 4th α‐helices of the TIM‐barrel (Figures <a href="#pro70255-fig-0003" class="usa-link">3</a> and <a href="#pro70255-supitem-0001" class="usa-link">S2</a>). Amino acid sequences and structures of this region are drastically diversified among GH1 enzymes, which are probably related to substrate specificity (as discussed later).</p>
<figure class="fig xbox font-sm" id="pro70255-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/87b7/12369401/92f8cf75733b/PRO-34-e70255-g006.jpg" loading="lazy" id="jats-graphic-5" height="410" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/pro70255-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overall structure of SGR_2426r. (a) The ligand‐free structure of SGR_2426r is shown as a rainbow cartoon. The PDB ID of the ligand‐free structure of SGR_2426r is 8KAP. The <em>N</em>‐terminal end is colored blue. α‐Helices in the (β/α)<sub>8</sub>‐barrel are labeled. The inserted α‐helix (α9) is labeled with a parenthesis. The numbers by α‐helices correspond to the numbers in the multiple sequence alignment shown in Figure <a href="#pro70255-supitem-0001" class="usa-link">S2</a>. (b–e) Superimposition of SGR_2426r with glycoside hydrolase family 1 (GH1) homologs. The superimposed proteins are the top four homologs retrieved from the KEGG database after a BLASTP search using SGR_2426 as the query. SGR_2426r is shown as a semi‐transparent green cartoon, except for residues 169–195, which are shown as a light‐brown cartoon. The corresponding regions in the superimposed proteins are shown as orange, magenta, yellow, and purple cartoons, respectively. The other parts are shown as semi‐transparent cyan, light‐cyan, blue–green, and light‐blue–green cartoons, respectively. PDB IDs are shown at the bottom‐right of the images (4R27, BGL167 from <em>Microbacterium</em> sp. Gsoil167; 6IER, a protein from an uncultured bacterium; 1VFF, β‐glycosidase from <em>Pyrococcus horikoshii</em>; 3SCU, BGlu1 from <em>Oryza sativa</em> Japonica Group). α‐Helices (α7 and α10) in SGR_2426r are labeled.</p></figcaption></figure></section><section id="pro70255-sec-0007"><h3 class="pmc_sec_title">2.5. Complex structure with Sop<sub>2</sub>
</h3>
<p>To obtain a complex structure with a substrate, residues Glu168 and Glu318, which are candidates for an acid/base and a nucleophile involved in the catalytic reaction, were replaced with glutamine and glycine residues, respectively. Both mutants showed drastically decreased hydrolytic activity toward pNP‐β‐Glc compared with the wild type (data not shown). Because soaking with usual concentrations of Sop<sub>2</sub> (approximately 10–20 mM) did not yield sufficient electron density of Sop<sub>2</sub>, crystals of the E318G mutant prepared without Sop<sub>2</sub> were soaked in a cryoprotected solution containing 500 mM Sop<sub>2</sub>. The overall structure of the complex is almost the same as that of the ligand‐free enzyme, although the space groups of the crystals were different between the ligand‐free and the complex structures. However, conformations of glucose (Glc) moieties at subsite −1 (subsite is a nomenclature for substrate binding sites, see <a href="https://www.cazypedia.org/index.php/Sub-site_nomenclature" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cazypedia.org/index.php/Sub-site_nomenclature</a> for details) (Davies et al. <a href="#pro70255-bib-0013" class="usa-link" aria-describedby="pro70255-bib-0013">1997</a>), electron density of Glc moieties at subsite +1, and B‐factors for parts of the substrate pocket vary between the four molecules (chains A–D) in the asymmetric unit (Figures <a href="#pro70255-fig-0004" class="usa-link">4a–d</a>, <a href="#pro70255-supitem-0001" class="usa-link">S3</a>, and <a href="#pro70255-supitem-0001" class="usa-link">S4</a>). In addition, two other Sop<sub>2</sub> molecules were observed at the interface of chains A and C clearly in the asymmetric unit (Figure <a href="#pro70255-supitem-0001" class="usa-link">S3</a>). Binding of these Sop<sub>2</sub> molecules might be an artifact because the enzyme was judged to be a monomer by PISA analysis.</p>
<figure class="fig xbox font-sm" id="pro70255-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/87b7/12369401/0aac6351be6e/PRO-34-e70255-g007.jpg" loading="lazy" id="jats-graphic-7" height="923" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/pro70255-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Conformations of substrates in the catalytic pockets of SGR_2426r. (a–d) Electron densities of Sop<sub>2</sub> (a–c) and Glc (d) in the catalytic pockets of the SGR_2426r–Sop<sub>2</sub> complex. Chains (a–d) of SGR_2426r (PDB ID, 9U98) are shown as green, cyan, magenta, and yellow cartoons, respectively. Sop<sub>2</sub> and Glc are shown as white sticks. <em>F</em>
<sub>o</sub> − <em>F</em>
<sub>c</sub> maps are shown at the 3σ contour level as blue meshes. The numbers indicate subsite positions. (a–c) The glucose moieties at the reducing end perhaps contain both anomers in solution, although only one anomer is fitted as a model in each subunit. (e) Six‐membered ring puckering of the Glc moieties at subsite −1 in the complex of SGR_2426r with Sop<sub>2</sub>. Calculation of Cremer–Pople parameters was performed using the site created by Dr. Fushinobu (<a href="http://enzyme13.bt.a.u-tokyo.ac.jp/CP/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://enzyme13.bt.a.u‐tokyo.ac.jp/CP/</a>). The panel is a figure that the results obtained from the four chains are integrated. Each of the plots is manually placed based on each result. Because the plots for Chain A and Chain C are located at almost the same position, the plots are represented as hemispheres.</p></figcaption></figure><p>To assess if the substrate binding mode is plausible for the reaction, six‐membered ring puckering of the Glc moieties at subsite −1 was analyzed using the Cremer–Pople model (Cremer and Pople <a href="#pro70255-bib-0012" class="usa-link" aria-describedby="pro70255-bib-0012">1975</a>). In chain B, the electron density of Sop<sub>2</sub> was clearly observed and the Glc moiety at subsite −1 adopted a chair conformation (<sup>4</sup>
<em>C</em>
<sub>1</sub>) (Figure <a href="#pro70255-fig-0004" class="usa-link">4b</a>). This conformation is a state immediately after substrate binding, which is not suitable for the reaction because distortion of the six‐membered ring is needed to reach the transition state. In chains A and C, the electron densities of the Glc moieties at subsite +1 are slightly less observed than that in chain B (Figure <a href="#pro70255-fig-0004" class="usa-link">4a–c</a>), and the Glc moieties at subsite −1 adopt distorted conformations around <sup>1</sup>
<em>S</em>
<sub>3</sub>, <sup>1,4</sup>
<em>B</em>, and <sup>4</sup>
<em>E</em> (Figure <a href="#pro70255-fig-0004" class="usa-link">4e</a>). In chain D, the Glc moiety at subsite −1 adopts a skewed conformation (<sup>1</sup>
<em>S</em>
<sub>3</sub>) (Figure <a href="#pro70255-fig-0004" class="usa-link">4e</a>), which is often observed in the Michaelis complex in GH enzymes (Nakajima et al. <a href="#pro70255-bib-0061" class="usa-link" aria-describedby="pro70255-bib-0061">2016</a>; Tanaka et al. <a href="#pro70255-bib-0076" class="usa-link" aria-describedby="pro70255-bib-0076"><sup>2019</sup></a>). Furthermore, the Glc moiety at subsite +1 is almost completely disordered (Figure <a href="#pro70255-fig-0004" class="usa-link">4d</a>). Based on the fact that both <sup>1</sup>
<em>S</em>
<sub>3</sub> and <sup>4</sup>
<em>C</em>
<sub>1</sub> conformations of substrates at subsite −1 are structurally available in GH1 BGLs, reaction itineraries are suggested to proceed via a <sup>1</sup>
<em>S</em>
<sub>3</sub> → <sup>4</sup>
<em>H</em>
<sub>3</sub> → <sup>4</sup>
<em>C</em>
<sub>1</sub> pathway (Sansenya et al. <a href="#pro70255-bib-0070" class="usa-link" aria-describedby="pro70255-bib-0070">2011</a>; Seshadri et al. <a href="#pro70255-bib-0073" class="usa-link" aria-describedby="pro70255-bib-0073"><sup>2009</sup></a>; Verdoucq et al. <a href="#pro70255-bib-0079" class="usa-link" aria-describedby="pro70255-bib-0079"><sup>2004</sup></a>). Thus, the three different conformations obtained in this study are consistent with the suggested itinerary.</p>
<p>The clarity of electron densities at subsite +1 decreases as the reaction proceeds according to this itinerary (Figure <a href="#pro70255-fig-0004" class="usa-link">4a–d</a>). To understand the reason for this difference in clarity, the B‐factor of each chain was visualized (Figure <a href="#pro70255-supitem-0001" class="usa-link">S4</a>). Chains A–C showed similar B‐factors (Table <a href="#pro70255-supitem-0002" class="usa-link">S1</a>). Nevertheless, regions with large B‐factors in Chain B are larger than those in Chains A and C (Figure <a href="#pro70255-supitem-0001" class="usa-link">S4a–c</a>). Unlike in these three chains, there are three disordered regions in chain D (residues 182–188, 232–243, and 271–282) (Figure <a href="#pro70255-supitem-0001" class="usa-link">S4d</a>). Superimposition of chains B and A, A and C, and C and D suggests that the extent of deviations between chains corresponds to the conformational changes of the Glc moieties at subsite −1 (Figure <a href="#pro70255-supitem-0001" class="usa-link">S5a–c</a>). Five deviated regions (designated as regions 1–5 in Figure <a href="#pro70255-supitem-0001" class="usa-link">S5a</a>) are found. Regions 2–4 are the same as the distorted regions in Chain D. Regions 1 and 5 are additionally found as deviated regions (Figures <a href="#pro70255-supitem-0001" class="usa-link">S4</a> and <a href="#pro70255-supitem-0001" class="usa-link">S5a–c</a>). Region 1 interacts with the Glc moiety at subsite −1 and region 5 forms the entrance of the catalytic pocket along with the regions (residues 182–188 and 271–282). The five regions are found to exhibit relatively larger fluctuations (root mean square fluctuation, RMSF) according to the molecular dynamics (MD) simulation of the ligand‐free structure (Figure <a href="#pro70255-supitem-0001" class="usa-link">S5d</a>). The fluctuation of region 5 reduced when the complex with Sop<sub>2</sub> was used for the simulation, which may be due to binding to the substrate.</p></section><section id="pro70255-sec-0008"><h3 class="pmc_sec_title">2.6. Substrate recognition in the Sop<sub>2</sub> complex</h3>
<p>The Glc moiety at subsite −1 is firmly recognized by multiple residues (Gln29, His123, Asn167, Tyr267, Gln363, Trp364) through hydrogen bonds. In contrast, only a single hydrogen bond is observed at subsite +1, involving Gln229 (Figure <a href="#pro70255-fig-0005" class="usa-link">5a</a>). The main chain of Asn228 indirectly recognizes the substrate through a water molecule. The six‐membered ring of the Glc moiety at subsite +1 is sandwiched between Met171 and Trp291 hydrophobically. These observations suggest that substrate recognition at subsite +1 is loose, being consistent with the disordered electron density at subsite +1 in Chain D. However, the anomeric hydroxy group at subsite +1 faces solvent, suggesting that Sop<sub>2</sub> binds correctly to the enzyme in the crystal.</p>
<figure class="fig xbox font-sm" id="pro70255-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/87b7/12369401/d0889ca0006d/PRO-34-e70255-g004.jpg" loading="lazy" id="jats-graphic-9" height="944" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/pro70255-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of binding modes of substrates between GH1 enzymes. Hydrogen bonds are shown as blue dashed lines. Distances up to 3.5 Å are regarded as hydrogen bonds. A water molecule is shown as a sphere. Main‐chain atoms that are not involved in substrate recognition are omitted for visuality. (a) Binding mode of Sop<sub>2</sub> in the SGR_2426r–Sop<sub>2</sub> complex. Chain A of the complex is used for visualization (PDB ID, 9U98). The main‐chain is shown as a green cartoon. The numbers beside the Glc moieties are subsite positions. Recognition residues for subsites −1 and +1 are shown as green and cyan sticks, respectively, and are labeled with black plain and red bold letters, respectively, for SGR_2426r. (b) Comparison of binding modes of ligands with BGL167 (left) and MeBglD2 (right). SGR_2426r (PDB ID, 9U98) is superimposed with BGL167 (from <em>Microbacterium</em> sp. Gsoil167) and MeBglD2 (from a metagenome). SGR_2426r is shown as thin sticks in the same color as (a). Residues in the other enzymes are shown as light pink sticks. The labels for residues in SGR_2426r are shown in blue. Sop<sub>2</sub> molecules are shown as thin white sticks. Parentheses indicate labels for SGR_2426r. (right) Cel<sub>2</sub> and Glc are shown as light‐brown sticks. Subsite positions are labeled with numbers. (c) Comparison of binding modes of Sop<sub>2</sub> between GH3 BGL (Lin1840, PDB ID 4ZO8, left) and SGR_2426r–Sop<sub>2</sub> complex (PDB ID, 9U98, right). (left) Residues recognizing the Glc moieties at the subsites +1 and Sop<sub>2</sub> molecule are shown as pink and yellow sticks, respectively, for Lin1840. Asterisks represent that the residues are derived from another subunit in the dimer. (right) SGR_2426r is shown in the same way as (b).</p></figcaption></figure></section><section id="pro70255-sec-0009"><h3 class="pmc_sec_title">2.7. Mutational analysis</h3>
<p>To understand the substrate preference of SGR_2426r in detail, W291E and Q229N mutants were investigated based on the observations described below. GH1 BGL from <em>Mycobacterium</em> sp. Gsoil167 is an enzyme that hydrolyzes glucosides derived from a triterpenoid saponin (Rb1) containing Sop<sub>2</sub> and Gen<sub>2</sub> (Park et al. <a href="#pro70255-bib-0065" class="usa-link" aria-describedby="pro70255-bib-0065">2015</a>). Non‐reducing end Glc moieties in Sop<sub>2</sub> and Gen<sub>2</sub> are hydrolyzed by this enzyme. An E316W mutant loses hydrolytic activity toward the β‐1,6‐glucosidic linkage. E316 corresponds to Trp291 in SGR_2426, implying the importance of this residue for specificity toward both β‐1,2‐ and β‐1,6‐linkages (Figure <a href="#pro70255-fig-0005" class="usa-link">5b</a>, left). Meanwhile, Asn253 and Ser254, corresponding to Gln229 in SGR_2426, seem not to be involved in substrate recognition because their side chains face the opposite side compared with Gln229 in SGR_2426. Thus, the W291E mutant was prepared to elucidate the roles of W291.</p>
<p>Trp291 in SGR_2426 is conserved in a GH1 BGL from a metagenome (Genbank accession number, <a href="https://www.ncbi.nlm.nih.gov/protein/BAV69317.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">BAV69317.1</a>); however, the latter enzyme shows similar activity toward β‐linked glucodisaccharides (Sop<sub>2</sub>, Lam<sub>2</sub>, Cel<sub>2</sub>, and Gen<sub>2</sub>), unlike SGR_2426 (Matsuzawa et al. <a href="#pro70255-bib-0045" class="usa-link" aria-describedby="pro70255-bib-0045">2022</a>). Gln229 in SGR_2426 is also conserved (as a chemically similar residue) in the enzyme from the metagenome (Figure <a href="#pro70255-fig-0005" class="usa-link">5b</a>, right). Thus, Q229N was also prepared to elucidate the roles of Q229.</p>
<p>Consequently, W291E showed a remarkable decrease in activity on Sop<sub>2</sub> and Lam<sub>2</sub> (Table <a href="#pro70255-tbl-0001" class="usa-link">1</a> and Figure <a href="#pro70255-fig-0002" class="usa-link">2i</a>). The <em>V</em>
<sub>max</sub> value decreased approximately 5 times and the <em>K</em>
<sub>m</sub> value increased approximately twice that of the wild‐type. No activity on Cel<sub>2</sub> nor Gen<sub>2</sub> was detected. Only a weak activity was detected on Lam<sub>2</sub>. The ratio of specific activity between Sop<sub>2</sub> and Lam<sub>2</sub> in the presence of 5 mM substrate was at a level similar to that of the wild‐type enzyme. This result suggests that W291 is important for substrate recognition but does not contribute to substrate preference. The hydrolytic activity of the Q229N mutant toward Sop<sub>2</sub> was also reduced remarkably, as well as that of the W291E mutant (Table <a href="#pro70255-tbl-0001" class="usa-link">1</a> and Figure <a href="#pro70255-fig-0002" class="usa-link">2j,k</a>). However, activity toward Lam<sub>2</sub> was retained apparently compared to the W291E mutant, suggesting that Q229 contributes to the substrate preference of SGR_2426r.</p></section></section><section id="pro70255-sec-0010"><h2 class="pmc_sec_title">3. DISCUSSION</h2>
<p>SGR_2426 was previously investigated as a possible cellulose‐degrading enzyme (Heins et al. <a href="#pro70255-bib-0023" class="usa-link" aria-describedby="pro70255-bib-0023">2014</a>). Among <em>Streptomyces</em> species, SCO1059 and SCO7558 from <em>Streptomyces coelicolor</em> A3(2), a protein from <em>Streptomyces rochei</em> A2 (accession number, <a href="https://www.ncbi.nlm.nih.gov/protein/CAA52344.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CAA52344.1</a>), SCAB_2391 and SCAB_57721 from <em>Streptomyces scabiei</em> 87.22, and BgluS from <em>Streptomyces</em> sp. RW2 (accession number, <a href="https://www.ncbi.nlm.nih.gov/protein/UUY85975.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">UUY85975.1</a>) were reported to be GH1 family enzymes. However, only Cel<sub>2</sub> and glycosides were investigated as substrates (Deflandre et al. <a href="#pro70255-bib-0014" class="usa-link" aria-describedby="pro70255-bib-0014">2020</a>; Gu et al. <a href="#pro70255-bib-0021" class="usa-link" aria-describedby="pro70255-bib-0021"><sup>2013</sup></a>; Heins et al. <a href="#pro70255-bib-0023" class="usa-link" aria-describedby="pro70255-bib-0023"><sup>2014</sup></a>; Jourdan et al. <a href="#pro70255-bib-0031" class="usa-link" aria-describedby="pro70255-bib-0031"><sup>2018</sup></a>; Mastromei et al. <a href="#pro70255-bib-0044" class="usa-link" aria-describedby="pro70255-bib-0044"><sup>1992</sup></a>; Mei et al. <a href="#pro70255-bib-0048" class="usa-link" aria-describedby="pro70255-bib-0048"><sup>2021</sup></a>); thus, natural substrates have not been fully investigated. These homologs, except the <em>S. rochei</em> A2 protein, are far from SGR_2426 phylogenetically (Figure <a href="#pro70255-supitem-0001" class="usa-link">S6</a>). Characteristics of close homologs of SGR_2426 other than the <em>S. rochei</em> A2 protein have not been investigated. A GH1 BGL from <em>Talaromyces amestolkiae</em> is annotated as β‐1,2‐glucosidase in the CAZy database (Drula et al. <a href="#pro70255-bib-0015" class="usa-link" aria-describedby="pro70255-bib-0015">2022</a>). Although the enzyme acts on a β‐1,2‐glucosidic linkage in glucosides, it also shows comparable hydrolytic activity on cellooligosaccharides (Méndez‐Líter et al. <a href="#pro70255-bib-0049" class="usa-link" aria-describedby="pro70255-bib-0049"><sup>2020</sup></a>). This fact suggests whether the BGL is β‐1,2‐glucan‐associated or not is controversial. Os3BGlu6 from rice (<em>Oryza sativa</em>) is reported to be a GH1 family BGL that prefers β‐1,2‐ and β‐1,3‐linked glucodisaccharides as substrates (Seshadri et al. <a href="#pro70255-bib-0073" class="usa-link" aria-describedby="pro70255-bib-0073">2009</a>). However, Os3BGlu6 preferentially acts on pNP‐β‐<span class="font-variant-small-caps">d</span>‐Fuc over pNP‐β‐Glc, indicating that the enzyme is actually a β‐<span class="font-variant-small-caps">d</span>‐fucosidase (Seshadri et al. <a href="#pro70255-bib-0073" class="usa-link" aria-describedby="pro70255-bib-0073">2009</a>).</p>
<p>This study revealed that SGR_2426r had comparable hydrolytic activity toward Sop<sub>2–5</sub> among the tested glucooligosaccharides in the presence of up to 2 mM substrate and that the enzyme showed low affinity (a high <em>K</em>
<sub>m</sub> value) for Lam<sub>2</sub> (β‐1,3) and very low activities toward Cel<sub>2</sub> (β‐1,4) and Gen<sub>2</sub> (β‐1,6) (Table <a href="#pro70255-tbl-0001" class="usa-link">1</a>). These results suggest that SGR_2426 is not an enzyme for cellobiose degradation, but is the first identified GH1 enzyme acting on Sop<sub>n</sub>s as natural substrates. It should be noted that SGR_2426 prefers Sop<sub>2</sub> over Lam<sub>2</sub> much more strongly than Lin1840 and BT_3567, GH3 β‐1,2‐glucan‐associated enzymes hydrolyzing Sop<sub>2</sub> and Lam<sub>2</sub> with comparable velocity (Ishiguro et al. <a href="#pro70255-bib-0028" class="usa-link" aria-describedby="pro70255-bib-0028">2017</a>; Nakajima et al. <a href="#pro70255-bib-0061" class="usa-link" aria-describedby="pro70255-bib-0061"><sup>2016</sup></a>).</p>
<p>The gene cluster that contains the SGR_2426‐encoding gene also encompasses ABC transporter homolog genes (<em>sgr_2423</em>–<em>2425</em>) (Figure <a href="#pro70255-fig-0006" class="usa-link">6a</a>). SGR_2425, annotated as a solute‐binding protein, is a close homolog of Lin1841 from <em>L. innocua</em>, a binding subunit for Sop<sub>3–5</sub> in an ABC transporter (Abe et al. <a href="#pro70255-bib-0003" class="usa-link" aria-describedby="pro70255-bib-0003">2018</a>); SGR_2425 is expected to have a similar substrate preference to Lin1841. SGR_2423–2426 can explain the metabolism of Sop<sub>n</sub>s (Figure <a href="#pro70255-fig-0006" class="usa-link">6b</a>). Although SGR_2427 is annotated as a hypothetical protein in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database, a homologous protein from <em>Sanguibacter keddieii</em> has been found to be an SGL recently, and this phylogenetic group was classified into GH193, a newly established GH family (Nakajima et al. <a href="#pro70255-bib-0059" class="usa-link" aria-describedby="pro70255-bib-0059">2025</a>). SGR_2422 is annotated as an acetyl‐esterase in the KEGG database, although this protein is not classified into any CE family (Lombard et al. <a href="#pro70255-bib-0042" class="usa-link" aria-describedby="pro70255-bib-0042">2010</a>). SGR_2422 and SGR_2426 have no <em>N</em>‐terminal signal peptide. However, considering that SGR_2424–2425 (transmembrane subunits of ABC transporter) do not have an <em>N</em>‐terminal signal peptide either, SGR_2422 and SGR_2426 could be located extracellularly.</p>
<figure class="fig xbox font-sm" id="pro70255-fig-0006"><h3 class="obj_head">FIGURE 6.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/87b7/12369401/4755262df45e/PRO-34-e70255-g002.jpg" loading="lazy" id="jats-graphic-11" height="656" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/pro70255-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic representation of gene clusters containing homologs of gene <em>SGR_2426</em> and β‐1,2‐glucan metabolism. (a) Gene clusters containing homologs of gene <em>sgr_2426</em>. Annotations of proteins are based on information in the KEGG database (<a href="https://www.kegg.jp/kegg/kegg2.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.kegg.jp/kegg/kegg2.html</a>). Annotation of GH193 is based on the previous study (Nakajima et al. <a href="#pro70255-bib-0059" class="usa-link" aria-describedby="pro70255-bib-0059">2025</a>). Parentheses indicate that the homologs are not in the gene clusters of the <em>sgr_2426</em> gene homologs. Directions of arrowheads correspond to directions of genes. (b) Speculated β‐1,2‐glucan metabolism in <em>S. griseus</em>. Glc units are depicted based on Neelamegham et al. (<a href="#pro70255-bib-0062" class="usa-link" aria-describedby="pro70255-bib-0062">2019</a>). (+) and (−) represent proteins with and without a predicted <em>N</em>‐terminal signal, respectively. A question represents that a degradation step is not supported by the results of prediction of <em>N</em>‐terminal signal peptides. (c) Comparison of residues participating in recognition at subsite +1 between the six GH1 enzymes. The residues are extracted from a multiple sequence alignment (Data <a href="#pro70255-supitem-0002" class="usa-link">S2</a>).</p></figcaption></figure><p>Genes encoding close homologs of SGR_2426 and SGR_2423 form gene clusters in several species, including some <em>Bifidobacterium</em> (Figure <a href="#pro70255-fig-0006" class="usa-link">6a</a>). Among residues recognizing the Glc unit at subsite +1, Trp291 and Gln229 are conserved, and Met171 is also conserved as a hydrophobic residue (Figure <a href="#pro70255-fig-0006" class="usa-link">6c</a>). Contrarily, Asn228 indirectly participating in recognition is not conserved. β‐1,2‐Glucans are produced in various microorganisms while the existence of Sop<sub>n</sub>s is reported mainly as sophorosides (Sop<sub>2</sub>‐linked compounds) (Kim et al. <a href="#pro70255-bib-0033" class="usa-link" aria-describedby="pro70255-bib-0033">2012</a>). Although several gene clusters lack GH193 genes, five GH families of SGLs (GH144, GH162, GH192–194) were found, and their homologs are widely distributed among bacteria and fungi (Abe et al. <a href="#pro70255-bib-0002" class="usa-link" aria-describedby="pro70255-bib-0002">2017</a>; Nakajima et al. <a href="#pro70255-bib-0059" class="usa-link" aria-describedby="pro70255-bib-0059"><sup>2025</sup></a>; Tanaka et al. <a href="#pro70255-bib-0076" class="usa-link" aria-describedby="pro70255-bib-0076"><sup>2019</sup></a>). Sop<sub>n</sub>s could be provided by these potential SGLs. Overall, SGR_2426 is a β‐glycosidase preferring β‐glucosides; more specifically, the enzyme is considered to be a β‐1,2‐glucosidase physiologically. Based on this, we speculate that <em>S. griseus</em> can metabolize (alkylated) Sop<sub>n</sub>s.</p>
<p>To understand structural determinants of the substrate specificity of the enzyme, subsite +1 in the SGR_2426r–Sop<sub>2</sub> complex was compared with that in GH3 Lin1840 (Figure <a href="#pro70255-fig-0005" class="usa-link">5c</a>). Lin1840 prefers Sop<sub>2</sub> as a substrate but also shows comparable activity toward Lam<sub>2</sub>. The enzyme is considered to be involved in metabolism of β‐1,2‐glucans and/or Sop<sub>n</sub>s (Ishiguro et al. <a href="#pro70255-bib-0028" class="usa-link" aria-describedby="pro70255-bib-0028">2017</a>; Nakajima et al. <a href="#pro70255-bib-0061" class="usa-link" aria-describedby="pro70255-bib-0061"><sup>2016</sup></a>). In the Lin1840–Sop<sub>2</sub> complex, the Glc moiety at subsite +1 is sandwiched hydrophobically by aromatic rings of Trp271 and Tyr583, and forms multiple hydrogen bonds between the 3‐ and 4‐OH groups and Arg572 (Figure <a href="#pro70255-fig-0005" class="usa-link">5c</a>, left). Meanwhile, in SGR_2426, Trp291 and Met181 are located perpendicularly to the six‐membered ring of the Glc moiety at the subsite +1, thereby forming a hydrophobic interaction (Figure <a href="#pro70255-fig-0005" class="usa-link">5c</a>, right). Furthermore, only the 4‐hydroxy group forms a direct hydrogen bond with residue Gln229. These observations suggest that the mechanism of substrate preference is different between Lin1840 and SGR_2426.</p>
<p>Substrate recognition residues for subsite −1 are well conserved among GH1 enzymes (Figures <a href="#pro70255-fig-0005" class="usa-link">5</a> and <a href="#pro70255-supitem-0001" class="usa-link">S2</a>). Thus, subsites +1 were compared using complexes with ligands binding both at subsites −1 and +1 among structurally characterized close homologs of SGR_2426 (Figures <a href="#pro70255-fig-0005" class="usa-link">5</a> and <a href="#pro70255-fig-0007" class="usa-link">7</a>). Comparison with BGL167 and MeBglD2 brought up Trp291 and Gln229 as candidates for contributing substrate preference of SGR_2426. W291E did not obtain hydrolytic activity toward Gen<sub>2</sub> (Table <a href="#pro70255-tbl-0001" class="usa-link">1</a>), although BGL167 shows hydrolytic activity toward Gen<sub>2</sub> moiety in a substrate. This suggests a possibility that the aglycone in the substrate is important to act on Gen<sub>2</sub> moiety for BGL167. It might also be possible that combination with structural changes at residues 228–229 in SGR_2426 is needed to obtain activity toward Gen<sub>2</sub> because a single mutation at Gln229 did not add substrate preference for Gen<sub>2</sub> (Table <a href="#pro70255-tbl-0001" class="usa-link">1</a>). Substitution of Gln229 to Asn decreased hydrolytic activity toward Sop<sub>2</sub> while not so much toward Lam<sub>2</sub> (Table <a href="#pro70255-tbl-0001" class="usa-link">1</a>). This suggests that Q229 highly contributes to substrate preference for Sop<sub>2</sub>.</p>
<figure class="fig xbox font-sm" id="pro70255-fig-0007"><h3 class="obj_head">FIGURE 7.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/87b7/12369401/9782f487a28a/PRO-34-e70255-g003.jpg" loading="lazy" id="jats-graphic-13" height="663" width="709" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/pro70255-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of binding modes of ligands with BGlu1 from <em>Oryza sativa</em>. (a–d) Superimposition of SGR_2426r–Sop<sub>2</sub> complex with GH1 BGlu1–Lam<sub>2</sub> (a) and BGlu1–Cel<sub>5</sub> (b–d) complexes. Residues in SGR_2426r (PDB ID, 9U98) and BGlu1 are shown as cyan and yellow sticks, respectively. Lam<sub>2</sub> and Cel<sub>5</sub> are shown as pale pink and orange sticks, respectively. Sop<sub>2</sub> in SGR_2426r is shown using thin white sticks. Parentheses used for residues and substrates represent labels for SGR_2426r. Hydrogen bonds are shown as blue dashed lines. Distances up to 3.5 Å are regarded as hydrogen bonds. (a) Residues indicated with bold letters are referred to in the main text. (b) SGR_2426r is shown as a cyan surface. Cel<sub>5</sub> is shown as a thin stick. (d) Superimposition of SGR_2426r–Sop<sub>2</sub> complex (cyan cartoon) with BGlu1–Cel<sub>5</sub> complex (white surface).</p></figcaption></figure><p>Then, SGR_2426r was compared with another structurally available enzyme. BGlu1 from <em>O. sativa</em> is a GH1 enzyme homologous to SGR_2426 that shows wide substrate specificity, acting on Cel<sub>n</sub>s, Lam<sub>n</sub>s, and Sop<sub>2</sub> (Opassiri et al. <a href="#pro70255-bib-0063" class="usa-link" aria-describedby="pro70255-bib-0063">2004</a>). The Glc moieties at subsite +1 in SGR_2426r–Sop<sub>2</sub> and BGlu1–Lam<sub>2</sub> complexes superimpose well (Figure <a href="#pro70255-fig-0007" class="usa-link">7a</a>). The 4‐OH group is recognized by Asn229, and the 3‐OH group faces the substrate pocket in the Sop<sub>2</sub> molecule in SGR_2426r. The Lam<sub>2</sub> molecule in BGlu1 also has hydroxy groups (1‐OH and 2‐OH) at similar positions to those of the Sop<sub>2</sub> molecule (Figure <a href="#pro70255-fig-0007" class="usa-link">7a,b</a>). Thus, it is unclear why SGR_2426 is highly specific for Sop<sub>2</sub>. Asn245 and Trp358, substrate recognition residues in BGlu1, are conserved in the primary structure of SGR_2426 (chemically for Asn245) (Figure <a href="#pro70255-supitem-0001" class="usa-link">S2</a>). However, the position of Asn245 and the orientation of Trp358 in BGlu1 are apparently different from the corresponding residues in SGR_2426, which may result in their different affinity for Lam<sub>2</sub> (Chuenchor et al. <a href="#pro70255-bib-0010" class="usa-link" aria-describedby="pro70255-bib-0010">2011</a>).</p>
<p>The low activity of SGR_2426 toward Cel<sub>2</sub> can be explained clearly, unlike in the case of Lam<sub>2</sub>. If the Glc moieties at subsite +1 in Cel<sub>2</sub> and Sop<sub>2</sub> are aligned in SGR_2426r, the 6‐hydroxy group in Cel<sub>2</sub> potentially collides with the surface of SGR_2426r (Figure <a href="#pro70255-fig-0007" class="usa-link">7b</a>) because there is insufficient space beyond 3‐OH of the Sop<sub>2</sub> molecule at subsite +1 to accommodate the 6‐OH of Cel<sub>2</sub>. In the case of BGlu1, the position of the Glc moiety at subsite +1 in Cel<sub>5</sub> deviates from the corresponding Glc moiety in SGR_2426r to form a hydrogen bond with Asn245, which enables the 6‐OH group in Cel<sub>5</sub> to avoid such steric hindrance (Figure <a href="#pro70255-fig-0007" class="usa-link">7b–d</a>). Residues 182–193 in SGR_2426r, narrowing the substrate pocket, are absent from BGlu1, which allows sufficient space for binding Cel<sub>5</sub> (Figures <a href="#pro70255-fig-0003" class="usa-link">3</a>, <a href="#pro70255-fig-0007" class="usa-link">7</a>, and <a href="#pro70255-supitem-0001" class="usa-link">S2</a>).</p>
<p>In the case of GH1 enzymes, when Sop<sub>2</sub> is placed at subsites −1 and +1, the 1‐OH group at subsite +1 faces the solvent side. This implies GH1 originally has the potential to be a β‐1,2‐glucan‐associated enzyme. Given the broad diversity of the GH1 family, we actually found a GH1 enzyme for the metabolism of β‐1,2‐glucans and/or Sop<sub>n</sub>s. Our findings also reinforce that β‐1,2‐glucan‐associated enzymes are distributed widely in microorganisms, enhancing our understanding of the importance of β‐1,2‐glucans in nature.</p></section><section id="pro70255-sec-0011"><h2 class="pmc_sec_title">4. MATERIALS AND METHODS</h2>
<section id="pro70255-sec-0012"><h3 class="pmc_sec_title">4.1. Materials</h3>
<p>pNP‐β‐Glc was purchased from FUJIFILM Wako Chemical Corporation (Japan), and pNP‐β‐<span class="font-variant-small-caps">d</span>‐Fuc and pNP‐β‐Gal from Nacalai Tesque, Inc. (Japan). Lam<sub>n</sub>s and Cel<sub>2</sub> were purchased from Neogen (MI, USA), and Gen<sub>2</sub> from Tokyo Chemical Industry (Japan). Sop<sub>n</sub>s were prepared according to the previous studies (Kobayashi et al. <a href="#pro70255-bib-0036" class="usa-link" aria-describedby="pro70255-bib-0036">2019</a>; Nakajima et al. <a href="#pro70255-bib-0060" class="usa-link" aria-describedby="pro70255-bib-0060"><sup>2014</sup></a>; Nakajima et al. <a href="#pro70255-bib-0058" class="usa-link" aria-describedby="pro70255-bib-0058"><sup>2021</sup></a>).</p></section><section id="pro70255-sec-0013"><h3 class="pmc_sec_title">4.2. Sequence analysis</h3>
<p>Multiple sequence alignment was performed using MUSCLE (<a href="https://www.ebi.ac.uk/Tools/msa/muscle/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/Tools/msa/muscle/</a>) (Edgar <a href="#pro70255-bib-0016" class="usa-link" aria-describedby="pro70255-bib-0016">2004</a>). A figure showing sequence alignment was prepared using ESPript 3.0 (<a href="https://espript.ibcp.fr/ESPript/cgi-bin/ESPript.cgi" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://espript.ibcp.fr/ESPript/cgi-bin/ESPript.cgi</a>) (Robert and Gouet <a href="#pro70255-bib-0068" class="usa-link" aria-describedby="pro70255-bib-0068">2014</a>).</p></section><section id="pro70255-sec-0014"><h3 class="pmc_sec_title">4.3. Cloning, expression, and purification of recombinant SGR_2426 protein</h3>
<p>The gene encoding SGR_2426 protein was amplified using <em>S. griseus</em> subsp. <em>griseus</em> (NBRC 13350) genomic DNA as the template. The genomic DNA was purchased from the National Institute of Technology and Evolution. Primer pair 5′–TTTAAGAAGGAGATATACATATGACACACACCCCTGCTTGGG–3′ and 5′–GTGGTGGTGGTGGTGGTGCTCGAGGGCTGCCGTGCGCGGGAGTTCG–3′ and PrimeSTAR Max DNA Polymerase (Takara, Japan) were used for PCR. The primer sequences were designed to fuse a His<sub>6</sub>‐tag at the <em>C</em>‐terminus of the protein. The PCR product was purified and then inserted into pET30a cut with restriction enzymes NdeI and XhoI (Takara) using the SLiCE method (Motohashi <a href="#pro70255-bib-0052" class="usa-link" aria-describedby="pro70255-bib-0052">2015</a>). After the sequence of the inserted gene in the resulting plasmid was checked, the plasmid was transformed into <em>E. coli</em> BL21 (DE3). The transformant was seeded into Luria‐Bertani (LB) medium containing 30 μg/mL kanamycin and was cultured at 37°C overnight. The culture medium was seeded into LB medium containing the same antibiotic for large‐scale production of SGR_2426. After the culture medium was agitated vigorously until OD<sub>600 nm</sub> reached 0.6–0.8, isopropyl β‐<span class="font-variant-small-caps">d</span>‐1‐thiogalactopyranoside was added to 0.1 mM (final concentration) to induce production of SGR_2426r. The culture medium was kept agitated at the same velocity at 30°C overnight. The cells were collected by centrifugation at 5640<em>g</em> for approximately 5 min, and then suspended in 50 mM MOPS (pH 7.0) buffer. After the cells were disrupted by sonication, the suspension was centrifuged at 27,000<em>g</em> for 10 min. The supernatant was loaded onto a HisTrap™ FF crude column (5 mL; Cytiva, MA, USA) equilibrated with 50 mM Tris–HCl (pH 7.0) containing 500 mM NaCl (buffer A). The column was washed with buffer A containing 10 mM imidazole until most unbound proteins were removed. SGR_2426r was eluted with buffer A containing a 10–300 mM linear gradient of imidazole. The purity of the eluate was investigated by 10% SDS‐PAGE. The enzyme migrated at almost the same molecular weight as the theoretical molecular weight. The SGR_2426r solution was exchanged into 5 mM MOPS buffer (pH 7.0) by ultrafiltration (Vivaspin MWCO 10000, Sartorius, Germany). The concentration of SGR_2426r was calculated from absorbance at 280 nm. The theoretical molecular weight and extinction coefficient of SGR_2426r at 280 nm are 45691.255 Da and 103,735 mol<sup>−1</sup> cm<sup>−1</sup>, respectively (Pace et al. <a href="#pro70255-bib-0064" class="usa-link" aria-describedby="pro70255-bib-0064">1995</a>). No <em>N</em>‐terminal signal peptide was detected by SignalP5.0 analysis (Almagro Armenteros et al. <a href="#pro70255-bib-0006" class="usa-link" aria-describedby="pro70255-bib-0006">2019</a>).</p></section><section id="pro70255-sec-0015"><h3 class="pmc_sec_title">4.4. Temperature and pH profiles</h3>
<p>To investigate the optimum pH for SGR_2426r activity, substrate solution (pNP‐β‐Glc, 95 μL) in various buffers was incubated at 30°C for 5 min, and then SGR_2426r solution (5 μL) at an appropriate concentration (1–3 μg) was added to start the reaction. The final concentrations of buffer and substrate were 50 mM and 5 mM, respectively. After the reaction mixtures were incubated at 30°C for 10 min, 20 μL sample was mixed with 180 μL of 0.2M Na<sub>2</sub>CO<sub>3</sub> to stop the reaction. The mixture (175 μL) was put into a 96‐well microplate (EIA/RIA plate, 96‐well half area, Corning, NY, USA) and the sample absorbance was measured at 400 nm. pNP was used as a standard.</p>
<p>To investigate the optimum reaction temperature for the enzyme, the reaction procedure described above was applied at various temperatures with 50 mM MOPS (pH 7.0) as the buffer. For investigating pH and temperature stabilities, SGR_2426r solutions (0.011 mg/mL) were incubated for 1 h in 50 mM each buffer at 30°C, or 50 mM MOPS (pH 6.5) at each temperature. Enzyme activity assays were performed in 50 mM MOPS (pH 7.0) at 30°C for 10 min with 5 mM substrate. Experiments were performed in triplicate.</p></section><section id="pro70255-sec-0016"><h3 class="pmc_sec_title">4.5. Kinetic analysis toward glucooligosaccharides</h3>
<p>After substrate solutions (95 μL) in MOPS (pH 6.5, 50 mM) were incubated at 30°C for 5 min, SGR_2426r solution of an appropriate concentration (5 μL, 0.03–0.14 mg/mL) was added to start the reaction. When investigating substrate specificity for Lam<sub>3–5</sub> as substrates, 45 μL of substrate solutions containing 5 mM substrate were used instead. Various substrate concentrations were adopted for the kinetic analysis. The reaction mixtures were incubated at 30°C for 10 min and then heated at 100°C for 5 min to terminate the reaction. The reaction mixtures (30 μL) were mixed with 170 μL of GOPOD solution (Neogen). After the samples were incubated at 45°C for 20 min, the absorbance of the samples was measured at 510 nm. <span class="font-variant-small-caps">d</span>‐Glucose was used as a standard. Medians were plotted and the plots were regressed with the Michaelis–Menten equation by R. For substrates except for pNP‐β‐<span class="font-variant-small-caps">d</span>‐Fuc and Sop<sub>5</sub>, the Michaelis–Menten equation <em>v</em> = <em>V</em>
<sub>max</sub> [S]/(<em>K</em>
<sub>m</sub> + [S]) was used. For pNP‐β‐<span class="font-variant-small-caps">d</span>‐Fuc, the equation <em>v</em> = {(<em>V</em>
<sub>max</sub>[S]/<em>K</em>
<sub>m</sub>) + <em>V</em>
<sub>max2</sub>[S]<sup>2</sup>/(<em>K</em>
<sub>m</sub>
<em>K</em>
<sub>m2</sub>)}/{1 + [S]/<em>K</em>
<sub>m</sub> + [S]<sup>2</sup>/(<em>K</em>
<sub>m</sub>
<em>K</em>
<sub>m2</sub>)} was used to adopt biphasic kinetics. For Sop<sub>5</sub>, the equation <em>v</em> = <em>V</em>
<sub>max</sub> [S]<sup>n</sup>/{<em>S</em>
<sub>0.5</sub>
<sup>n</sup> + (1 + [S]/<em>K</em>
<sub>i</sub>) [S]<sup>n</sup>} was used to take substrate inhibition into account. When calculating <em>V</em>
<sub>max</sub>/<em>K</em>
<sub>m</sub>, <em>V</em>
<sub>max2</sub>/<em>K</em>
<sub>m2</sub> and <em>V</em>
<sub>max</sub>/<em>S</em>
<sub>0.5</sub>, <em>v</em> = (<em>V</em>
<sub>max</sub>/<em>K</em>
<sub>m</sub>) [S]/(1 + [S]/<em>K</em>
<sub>m</sub>), <em>v</em> = {(<em>V</em>
<sub>max</sub>/<em>K</em>
<sub>m</sub>) [S] + (<em>V</em>
<sub>max2</sub>/<em>K</em>
<sub>m2</sub>) [S]<sup>2</sup>/<em>K</em>
<sub>m</sub>}/{1 + [S]/<em>K</em>
<sub>m</sub> + [S]<sup>2</sup>/(<em>K</em>
<sub>m</sub>
<em>K</em>
<sub>m2</sub>)} and <em>v</em> = (<em>V</em>
<sub>max</sub>/<em>S</em>
<sub>0.5</sub>)[S]<sup>n</sup>/[{<em>S</em>
<sub>0.5</sub>
<sup>n</sup> + (1 + [S]/<em>K</em>
<sub>i</sub>) [S]<sup>n</sup>} <em>S</em>
<sub>0.5</sub>], respectively, were used. The two values (<em>V</em>
<sub>max</sub>/<em>K</em>
<sub>m</sub>, <em>V</em>
<sub>max2</sub>/<em>K</em>
<sub>m2</sub> or <em>V</em>
<sub>max</sub>/<em>S</em>
<sub>0.5</sub>, and <em>K</em>
<sub>m</sub>, <em>K</em>
<sub>m2</sub> or <em>S</em>
<sub>0.5</sub>) were set as variables. Linear regression through the origin was also used to confirm the plausibility of the values. Experiments were performed in triplicate.</p></section><section id="pro70255-sec-0017"><h3 class="pmc_sec_title">4.6. Mutational analysis</h3>
<p>E168Q, E316G, Q229N, and W291E mutants of SGR_2426r were constructed using PrimeSTAR Max DNA polymerase according to the manufacturer's instructions. The primer pairs used were 5′–ATCAACcaaCCGAACATGATCGCCGTG–3′ and 5′–GTTCGGttgGTTGATCGTGCAGACGTG–3′, 5′–GTCACGgggAACGGCATCGCCACCGCC–3′ and 5′–GCCGTTcccCGTGACGATCAGCGGGAC–3′, 5′–GCCAACaatGTCTACCAGGCGCTCCCC–3′ and 5′–GTAGACattGTTGGCGATGGTCCAGCC–3′, and 5′–ACGACGgaaGAGTACTACCCGACGGCG–3′ and 5′–GTACTCttcCGTCGTCAGGGTCCGCTC–3′, respectively. Replaced nucleotides are represented in small letters. Mutant enzymes were expressed and purified in the same way as wild‐type SGR_2426r.</p></section><section id="pro70255-sec-0018"><h3 class="pmc_sec_title">4.7. Crystallography</h3>
<p>Initial screening of crystallization conditions for wild‐type SGR_2426r was performed using PACT primer™ HT‐96 MD1‐36 and JCSG‐plus™ HT‐96 MD1‐40 (Molecular Dimensions, UK). Purified SGR_2426r (7.5 mg/mL, 0.5 or 1.0 μL) and reservoir solution (1.0 μL) were mixed in 96‐well CrystalQuick plates (Greiner Bio‐One, Germany) for sitting drop vapor diffusion with 70 μL reservoir solutions. After the plates were sealed, they were incubated at 20°C for several days until crystals were observed. Crystallization conditions were optimized using the wild‐type enzyme and the E318G mutant by hanging drop vapor diffusion using VDX plates (Hampton Research, Germany). The wild‐type enzyme (8.0 mg/mL, 1 μL) was mixed with 1 μL of reservoir solution containing 0.1M Bis‐Tris (pH 5.5), 0.2M ammonium acetate, and 15% (w/v) PEG 3350 (FUJIFILM Wako Chemical Corporation) for crystallization. In the case of the E318G mutant, 0.5 μL of 7.0 mg/mL enzyme solution was mixed with 0.5 μL of reservoir solution containing 0.1M Bis‐Tris propane (pH 6.5), 0.3M sodium formate, and 18% PEG 3350. Crystals of wild‐type SGR_2426r were soaked in reservoir solution containing 25% PEG 400 as a cryoprotectant. For the E318G mutant, the crystals were soaked in reservoir solution containing 25% PEG 200 as a cryoprotectant and 500 mM Sop<sub>2</sub> for approximately 2 h. The crystal was cooled and then kept at 100 K in a nitrogen‐gas stream during data collection. X‐ray diffraction data were collected using a Pilatus detector at beamlines BL‐5A and NW‐12A at the Photon Factory, Tsukuba, Japan. The diffraction data set was processed by XDS (Kabsch <a href="#pro70255-bib-0032" class="usa-link" aria-describedby="pro70255-bib-0032">2010</a>). The structure of a GH1 enzyme (PDB ID: 4R27) was used as a search model for molecular replacement by MOLREP (Vagin and Teplyakov <a href="#pro70255-bib-0078" class="usa-link" aria-describedby="pro70255-bib-0078">2010</a>) to determine initial phases. Automated and manual model building were performed using ARP/wARP classic (Morris et al. <a href="#pro70255-bib-0051" class="usa-link" aria-describedby="pro70255-bib-0051">2002</a>) and Coot, respectively (Emsley and Cowtan <a href="#pro70255-bib-0017" class="usa-link" aria-describedby="pro70255-bib-0017">2004</a>). Refinement was performed using Refmac5 (Murshudov et al. <a href="#pro70255-bib-0055" class="usa-link" aria-describedby="pro70255-bib-0055">1997</a>). The quality of the structures was checked using the wwPDB validation server (<a href="https://validate-rcsb-1.wwpdb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://validate-rcsb-1.wwpdb.org/</a>). Figures were prepared using PyMOL (DeLano Scientific, CA, USA).</p></section><section id="pro70255-sec-0019"><h3 class="pmc_sec_title">4.8. 
MD simulation and RMSF analysis</h3>
<p>The Chain A of 8KAP and Chain B of 9U98 were used for MD simulation as a ligand‐free and a complex structure of SGR_2426, respectively. The proteins and Sop<sub>2</sub> were protonated and placed in a dodecahedral box. Box size was determined in order that all molecules were placed at least 1.5 nm from the box edges. The periodic boundary conditions were applied for all directions. The box was filled with water molecules. Sodium and chloride ions were added to each box to neutralize the total charge as well as to set the ion density to 10 mM. The AMBER ff14SB force field (Maier et al. <a href="#pro70255-bib-0043" class="usa-link" aria-describedby="pro70255-bib-0043">2015</a>) and GLYCAM06 (Kirschner et al. <a href="#pro70255-bib-0034" class="usa-link" aria-describedby="pro70255-bib-0034">2008</a>) were used to represent proteins and Sop<sub>2</sub>, respectively. The TIP3P model (Jorgensen et al. <a href="#pro70255-bib-0030" class="usa-link" aria-describedby="pro70255-bib-0030">1983</a>) was used for water molecules. After energy minimization, the constant‐pressure and constant‐temperature (NPT) MD simulations for equilibration were performed at 1 bar and 300 K for 200 ps. Position restraints were applied to Cα atoms of the proteins and all heavy atoms of the Sop<sub>2</sub> during equilibration. Three production runs for each system were performed for 500 ns at 300 K (total simulation time was 500 ns * 3 runs = 1.5 μs for the ligand‐free structure and the complex structure, respectively). The C‐rescale method and Parrinello‐Rahman method were used to maintain the pressure during the equilibrations and the production runs, respectively. The covalent bonds of hydrogen atoms were constrained using the LINCS algorithm (Hess et al. <a href="#pro70255-bib-0026" class="usa-link" aria-describedby="pro70255-bib-0026">1997</a>), and the integration time step was 2.0 fs. The coordinates of atoms were output every 100 ps, and the total number of outputs was 15,000 frames (5000 frames * 3 runs). RMSF values of Cα atoms of proteins were calculated using the gmx rmsf command of GROMACS. The MD simulation and RMSF analysis were performed by using GROMACS 2022.4 (Abraham et al. <a href="#pro70255-bib-0004" class="usa-link" aria-describedby="pro70255-bib-0004">2015</a>).</p></section></section><section id="pro70255-sec-0024"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Haruto Kumakura:</strong> Investigation; writing – review and editing; writing – original draft. <strong>Sei Motouchi:</strong> Investigation; writing – review and editing; funding acquisition. <strong>Kaito Kobayashi:</strong> Writing – review and editing; investigation. <strong>Miyu Inoue:</strong> Investigation; writing – review and editing. <strong>Natsuki Kariuda:</strong> Investigation; writing – review and editing. <strong>Hiroyuki Nakai:</strong> Conceptualization; resources; writing – review and editing. <strong>Masahiro Nakajima:</strong> Conceptualization; investigation; writing – original draft; writing – review and editing; funding acquisition; data curation.</p></section><section id="pro70255-sec-0021"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec22"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pro70255-supitem-0001"><div class="caption p"><p>
<strong>Data S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12369401/bin/PRO-34-e70255-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70255-s001.docx</a><sup> (9.6MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pro70255-supitem-0002"><div class="caption p"><p>
<strong>Data S2.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12369401/bin/PRO-34-e70255-s002.txt" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70255-s002.txt</a><sup> (3.9KB, txt) </sup>
</div></div></section></section><section id="pro70255-sec-0020" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank Edanz (<a href="https://jp.edanz.com/ac" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://jp.edanz.com/ac</a>) for editing a draft of this manuscript. In this research work we used the supercomputer of ACCMS, Kyoto University. We appreciate the help of all the staff at the Photon Factory in X‐ray data collection (Proposal Nos. 2020G527 and 2022G523). This work was supported in part by JSPS KAKENHI grant number 20K05830 and JST SPRING grant number JPMJSP2151.</p></section><section id="notes1"><p>

Kumakura H, Motouchi S, Kobayashi K, Inoue M, Kariuda N, Nakai H, et al. A β‐1,2‐glucan‐associated glycoside hydrolase family 1 β‐glucosidase from <em>Streptomyces griseus</em>
. Protein Science. 2025;34(9):e70255. 10.1002/pro.70255

</p>
<section id="pro70255-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="pro70255-note-0001"><p>
<strong>Review Editor:</strong> Aitziber L. Cortajarena</p></div></div></section></section><section id="pro70255-sec-0023"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The atomic coordinates and structure factors (codes 8KAP and 9U98) have been deposited in the PDB. The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section><section id="pro70255-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="pro70255-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="pro70255-bib-0001">
<cite>
Abe K, Nakajima M, Kitaoka M, Toyoizumi H, Takahashi Y, Sugimoto N, et al. Large‐scale preparation of 1,2‐β‐glucan using 1,2‐β‐oligoglucan phosphorylase. J Appl Glycosci. 2015;62:47–52.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Glycosci&amp;title=Large%E2%80%90scale%20preparation%20of%201,2%E2%80%90%CE%B2%E2%80%90glucan%20using%201,2%E2%80%90%CE%B2%E2%80%90oligoglucan%20phosphorylase&amp;author=K%20Abe&amp;author=M%20Nakajima&amp;author=M%20Kitaoka&amp;author=H%20Toyoizumi&amp;author=Y%20Takahashi&amp;volume=62&amp;publication_year=2015&amp;pages=47-52&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0002">
<cite>
Abe K, Nakajima M, Yamashita T, Matsunaga H, Kamisuki S, Nihira T, et al. Biochemical and structural analyses of a bacterial <em>endo</em>‐β‐1,2‐glucanase reveal a new glycoside hydrolase family. J Biol Chem. 2017;292:7487–7506.
</cite> [<a href="https://doi.org/10.1074/jbc.M116.762724" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5418048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28270506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Biochemical%20and%20structural%20analyses%20of%20a%20bacterial%20endo%E2%80%90%CE%B2%E2%80%901,2%E2%80%90glucanase%20reveal%20a%20new%20glycoside%20hydrolase%20family&amp;author=K%20Abe&amp;author=M%20Nakajima&amp;author=T%20Yamashita&amp;author=H%20Matsunaga&amp;author=S%20Kamisuki&amp;volume=292&amp;publication_year=2017&amp;pages=7487-7506&amp;pmid=28270506&amp;doi=10.1074/jbc.M116.762724&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0003">
<cite>
Abe K, Sunagawa N, Terada T, Takahashi Y, Arakawa T, Igarashi K, et al. Structural and thermodynamic insights into β‐1,2‐glucooligosaccharide capture by a solute‐binding protein in <em>Listeria innocua</em>
. J Biol Chem. 2018;293:8812–8828.
</cite> [<a href="https://doi.org/10.1074/jbc.RA117.001536" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5995521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29678880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Structural%20and%20thermodynamic%20insights%20into%20%CE%B2%E2%80%901,2%E2%80%90glucooligosaccharide%20capture%20by%20a%20solute%E2%80%90binding%20protein%20in%20Listeria%20innocua&amp;author=K%20Abe&amp;author=N%20Sunagawa&amp;author=T%20Terada&amp;author=Y%20Takahashi&amp;author=T%20Arakawa&amp;volume=293&amp;publication_year=2018&amp;pages=8812-8828&amp;pmid=29678880&amp;doi=10.1074/jbc.RA117.001536&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0004">
<cite>
Abraham MJ, Murtola T, Schulz R, Páll S, Smith JC, Hess B, et al. GROMACS: high performance molecular simulations through multi‐level parallelism from laptops to supercomputers. SoftwareX. 2015;1–2:19–25.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=SoftwareX&amp;title=GROMACS:%20high%20performance%20molecular%20simulations%20through%20multi%E2%80%90level%20parallelism%20from%20laptops%20to%20supercomputers&amp;author=MJ%20Abraham&amp;author=T%20Murtola&amp;author=R%20Schulz&amp;author=S%20P%C3%A1ll&amp;author=JC%20Smith&amp;volume=1%E2%80%932&amp;publication_year=2015&amp;pages=19-25&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0005">
<cite>
Akiba T, Nishio M, Matsui I, Harata K. X‐ray structure of a membrane‐bound β‐glycosidase from the hyperthermophilic archaeon <em>Pyrococcus horikoshii</em>
. Proteins: Structure, Function and Genetics. 2004;57:422–431.</cite> [<a href="https://doi.org/10.1002/prot.20203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15340929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins:%20Structure,%20Function%20and%20Genetics&amp;title=X%E2%80%90ray%20structure%20of%20a%20membrane%E2%80%90bound%20%CE%B2%E2%80%90glycosidase%20from%20the%20hyperthermophilic%20archaeon%20Pyrococcus%20horikoshii&amp;author=T%20Akiba&amp;author=M%20Nishio&amp;author=I%20Matsui&amp;author=K%20Harata&amp;volume=57&amp;publication_year=2004&amp;pages=422-431&amp;pmid=15340929&amp;doi=10.1002/prot.20203&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0006">
<cite>
Almagro Armenteros JJ, Tsirigos KD, Sønderby CK, Petersen TN, Winther O, Brunak S, et al. SignalP 5.0 improves signal peptide predictions using deep neural networks. Nat Biotechnol. 2019;37:420–423.
</cite> [<a href="https://doi.org/10.1038/s41587-019-0036-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30778233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=SignalP%205.0%20improves%20signal%20peptide%20predictions%20using%20deep%20neural%20networks&amp;author=JJ%20Almagro%20Armenteros&amp;author=KD%20Tsirigos&amp;author=CK%20S%C3%B8nderby&amp;author=TN%20Petersen&amp;author=O%20Winther&amp;volume=37&amp;publication_year=2019&amp;pages=420-423&amp;pmid=30778233&amp;doi=10.1038/s41587-019-0036-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0007">
<cite>
Arellano‐Reynoso B, Lapaque N, Salcedo S, Briones G, Ciocchini AE, Ugalde R, et al. Cyclic β‐1,2‐glucan is a brucella virulence factor required for intracellular survival. Nat Immunol. 2005;6:618–625.
</cite> [<a href="https://doi.org/10.1038/ni1202" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15880113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&amp;title=Cyclic%20%CE%B2%E2%80%901,2%E2%80%90glucan%20is%20a%20brucella%20virulence%20factor%20required%20for%20intracellular%20survival&amp;author=B%20Arellano%E2%80%90Reynoso&amp;author=N%20Lapaque&amp;author=S%20Salcedo&amp;author=G%20Briones&amp;author=AE%20Ciocchini&amp;volume=6&amp;publication_year=2005&amp;pages=618-625&amp;pmid=15880113&amp;doi=10.1038/ni1202&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0008">
<cite>
Broeker J, Mechelke M, Baudrexl M, Mennerich D, Hornburg D, Mann M, et al. The hemicellulose‐degrading enzyme system of the thermophilic bacterium <em>Clostridium stercorarium</em>: comparative characterisation and addition of new hemicellulolytic glycoside hydrolases. Biotechnol Biofuels. 2018;11:229.
</cite> [<a href="https://doi.org/10.1186/s13068-018-1228-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6106730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30159029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol%20Biofuels&amp;title=The%20hemicellulose%E2%80%90degrading%20enzyme%20system%20of%20the%20thermophilic%20bacterium%20Clostridium%20stercorarium:%20comparative%20characterisation%20and%20addition%20of%20new%20hemicellulolytic%20glycoside%20hydrolases&amp;author=J%20Broeker&amp;author=M%20Mechelke&amp;author=M%20Baudrexl&amp;author=D%20Mennerich&amp;author=D%20Hornburg&amp;volume=11&amp;publication_year=2018&amp;pages=229&amp;pmid=30159029&amp;doi=10.1186/s13068-018-1228-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0009">
<cite>
Castro OA, Zorreguieta A, Ielmini V, Vega G, Ielpi L. Cyclic β‐(1,2)‐glucan synthesis in <em>Rhizobiaceae</em>: roles of the 319‐kilodalton protein intermediate. J Bacteriol. 1996;178:6043–6048.
</cite> [<a href="https://doi.org/10.1128/jb.178.20.6043-6048.1996" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC178464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8830704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Cyclic%20%CE%B2%E2%80%90(1,2)%E2%80%90glucan%20synthesis%20in%20Rhizobiaceae:%20roles%20of%20the%20319%E2%80%90kilodalton%20protein%20intermediate&amp;author=OA%20Castro&amp;author=A%20Zorreguieta&amp;author=V%20Ielmini&amp;author=G%20Vega&amp;author=L%20Ielpi&amp;volume=178&amp;publication_year=1996&amp;pages=6043-6048&amp;pmid=8830704&amp;doi=10.1128/jb.178.20.6043-6048.1996&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0010">
<cite>
Chuenchor W, Pengthaisong S, Robinson RC, Yuvaniyama J, Svasti J, Cairns JRK. The structural basis of oligosaccharide binding by rice BGlu1 β‐glucosidase. J Struct Biol. 2011;173:169–179.
</cite> [<a href="https://doi.org/10.1016/j.jsb.2010.09.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20884352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Struct%20Biol&amp;title=The%20structural%20basis%20of%20oligosaccharide%20binding%20by%20rice%20BGlu1%20%CE%B2%E2%80%90glucosidase&amp;author=W%20Chuenchor&amp;author=S%20Pengthaisong&amp;author=RC%20Robinson&amp;author=J%20Yuvaniyama&amp;author=J%20Svasti&amp;volume=173&amp;publication_year=2011&amp;pages=169-179&amp;pmid=20884352&amp;doi=10.1016/j.jsb.2010.09.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0011">
<cite>
Ciocchini AE, Guidolin LS, Casabuono AC, Couto AS, de Iannino NI, Ugalde RA. A glycosyltransferase with a length‐controlling activity as a mechanism to regulate the size of polysaccharides. Proc Natl Acad Sci U S A. 2007;104:16492–16497.
</cite> [<a href="https://doi.org/10.1073/pnas.0708025104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2034269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17921247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=A%20glycosyltransferase%20with%20a%20length%E2%80%90controlling%20activity%20as%20a%20mechanism%20to%20regulate%20the%20size%20of%20polysaccharides&amp;author=AE%20Ciocchini&amp;author=LS%20Guidolin&amp;author=AC%20Casabuono&amp;author=AS%20Couto&amp;author=NI%20de%20Iannino&amp;volume=104&amp;publication_year=2007&amp;pages=16492-16497&amp;pmid=17921247&amp;doi=10.1073/pnas.0708025104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0012">
<cite>
Cremer D, Pople JA. A general definition of ring puckering coordinates. J Am Chem Soc. 1975;97:1354–1358.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=A%20general%20definition%20of%20ring%20puckering%20coordinates&amp;author=D%20Cremer&amp;author=JA%20Pople&amp;volume=97&amp;publication_year=1975&amp;pages=1354-1358&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0013">
<cite>
Davies GJ, Wilson KS, Henrissat B. Nomenclature for sugar‐binding subsites in glycosyl hydrolases. Biochem J. 1997;321:557–559.
</cite> [<a href="https://doi.org/10.1042/bj3210557" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1218105/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9020895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20J&amp;title=Nomenclature%20for%20sugar%E2%80%90binding%20subsites%20in%20glycosyl%20hydrolases&amp;author=GJ%20Davies&amp;author=KS%20Wilson&amp;author=B%20Henrissat&amp;volume=321&amp;publication_year=1997&amp;pages=557-559&amp;pmid=9020895&amp;doi=10.1042/bj3210557&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0014">
<cite>
Deflandre B, Thiébaut N, Planckaert S, Jourdan S, Anderssen S, Hanikenne M, et al. Deletion of <em>bglC</em> triggers a genetic compensation response by awakening the expression of alternative beta‐glucosidase. Biochim Biophys Acta Gene Regul Mech. 2020;1863:194615.
</cite> [<a href="https://doi.org/10.1016/j.bbagrm.2020.194615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32758700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta%20Gene%20Regul%20Mech&amp;title=Deletion%20of%20bglC%20triggers%20a%20genetic%20compensation%20response%20by%20awakening%20the%20expression%20of%20alternative%20beta%E2%80%90glucosidase&amp;author=B%20Deflandre&amp;author=N%20Thi%C3%A9baut&amp;author=S%20Planckaert&amp;author=S%20Jourdan&amp;author=S%20Anderssen&amp;volume=1863&amp;publication_year=2020&amp;pages=194615&amp;pmid=32758700&amp;doi=10.1016/j.bbagrm.2020.194615&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0015">
<cite>
Drula E, Garron ML, Dogan S, Lombard V, Henrissat B, Terrapon N. The carbohydrate‐active enzyme database: functions and literature. Nucleic Acids Res. 2022;50:D571–D577.
</cite> [<a href="https://doi.org/10.1093/nar/gkab1045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34850161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20carbohydrate%E2%80%90active%20enzyme%20database:%20functions%20and%20literature&amp;author=E%20Drula&amp;author=ML%20Garron&amp;author=S%20Dogan&amp;author=V%20Lombard&amp;author=B%20Henrissat&amp;volume=50&amp;publication_year=2022&amp;pages=D571-D577&amp;pmid=34850161&amp;doi=10.1093/nar/gkab1045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0016">
<cite>
Edgar RC. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 2004;32:1792–1797.
</cite> [<a href="https://doi.org/10.1093/nar/gkh340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC390337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15034147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=MUSCLE:%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=RC%20Edgar&amp;volume=32&amp;publication_year=2004&amp;pages=1792-1797&amp;pmid=15034147&amp;doi=10.1093/nar/gkh340&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0017">
<cite>
Emsley P, Cowtan K. Coot: model‐building tools for molecular graphics. Acta Crystallogr D Biol Crystallogr. 2004;60:2126–2132.
</cite> [<a href="https://doi.org/10.1107/S0907444904019158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15572765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Biol%20Crystallogr&amp;title=Coot:%20model%E2%80%90building%20tools%20for%20molecular%20graphics&amp;author=P%20Emsley&amp;author=K%20Cowtan&amp;volume=60&amp;publication_year=2004&amp;pages=2126-2132&amp;pmid=15572765&amp;doi=10.1107/S0907444904019158&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0018">
<cite>
Ferrara MC, Cobucci‐Ponzano B, Carpentieri A, Henrissat B, Rossi M, Amoresano A, et al. The identification and molecular characterization of the first archaeal bifunctional exo‐β‐glucosidase/N‐acetyl‐β‐glucosaminidase demonstrate that family GH116 is made of three functionally distinct subfamilies. Biochim Biophys Acta (BBA). 2014;1840:367–377.
</cite> [<a href="https://doi.org/10.1016/j.bbagen.2013.09.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24060745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta%20(BBA)&amp;title=The%20identification%20and%20molecular%20characterization%20of%20the%20first%20archaeal%20bifunctional%20exo%E2%80%90%CE%B2%E2%80%90glucosidase/N%E2%80%90acetyl%E2%80%90%CE%B2%E2%80%90glucosaminidase%20demonstrate%20that%20family%20GH116%20is%20made%20of%20three%20functionally%20distinct%20subfamilies&amp;author=MC%20Ferrara&amp;author=B%20Cobucci%E2%80%90Ponzano&amp;author=A%20Carpentieri&amp;author=B%20Henrissat&amp;author=M%20Rossi&amp;volume=1840&amp;publication_year=2014&amp;pages=367-377&amp;pmid=24060745&amp;doi=10.1016/j.bbagen.2013.09.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0019">
<cite>
Gay‐Fraret J, Ardissone S, Kambara K, Broughton WJ, Deakin WJ, Le Quéré A. Cyclic‐β‐glucans of <em>Rhizobium</em> (<em>Sinorhizobium</em>) sp. strain NGR234 are required for hypo‐osmotic adaptation, motility, and efficient symbiosis with host plants. FEMS Microbiol Lett. 2012;333:28–36.
</cite> [<a href="https://doi.org/10.1111/j.1574-6968.2012.02595.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22583376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol%20Lett&amp;title=Cyclic%E2%80%90%CE%B2%E2%80%90glucans%20of%20Rhizobium%20(Sinorhizobium)%20sp.%20strain%20NGR234%20are%20required%20for%20hypo%E2%80%90osmotic%20adaptation,%20motility,%20and%20efficient%20symbiosis%20with%20host%20plants&amp;author=J%20Gay%E2%80%90Fraret&amp;author=S%20Ardissone&amp;author=K%20Kambara&amp;author=WJ%20Broughton&amp;author=WJ%20Deakin&amp;volume=333&amp;publication_year=2012&amp;pages=28-36&amp;pmid=22583376&amp;doi=10.1111/j.1574-6968.2012.02595.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0020">
<cite>
Gourlay LJ, Mangiagalli M, Moroni E, Lotti M, Nardini M. Structural determinants of cold activity and glucose tolerance of a family 1 glycoside hydrolase (GH1) from Antarctic <em>Marinomonas</em> sp. ef1. FEBS J. 2024;291:2897–2917.
</cite> [<a href="https://doi.org/10.1111/febs.17096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38400529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20J&amp;title=Structural%20determinants%20of%20cold%20activity%20and%20glucose%20tolerance%20of%20a%20family%201%20glycoside%20hydrolase%20(GH1)%20from%20Antarctic%20Marinomonas%20sp.%20ef1&amp;author=LJ%20Gourlay&amp;author=M%20Mangiagalli&amp;author=E%20Moroni&amp;author=M%20Lotti&amp;author=M%20Nardini&amp;volume=291&amp;publication_year=2024&amp;pages=2897-2917&amp;pmid=38400529&amp;doi=10.1111/febs.17096&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0021">
<cite>
Gu MZ, Wang JC, Liu WB, Zhou Y, Ye BC. Expression and displaying of β‐glucosidase from <em>Streptomyces coelicolor</em> A3 in <em>Escherichia coli</em>
. Appl Biochem Biotechnol. 2013;170:1713–1723.
</cite> [<a href="https://doi.org/10.1007/s12010-013-0301-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23722947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Biochem%20Biotechnol&amp;title=Expression%20and%20displaying%20of%20%CE%B2%E2%80%90glucosidase%20from%20Streptomyces%20coelicolor%20A3%20in%20Escherichia%20coli&amp;author=MZ%20Gu&amp;author=JC%20Wang&amp;author=WB%20Liu&amp;author=Y%20Zhou&amp;author=BC%20Ye&amp;volume=170&amp;publication_year=2013&amp;pages=1713-1723&amp;pmid=23722947&amp;doi=10.1007/s12010-013-0301-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0022">
<cite>
Gunjal A, Bhagat DS. Diversity of actinomycetes in Western Ghats. Microbial diversity and ecology in hotspots. Massachusetts, USA: Academic Press, 2021. p. 117–133.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Microbial%20diversity%20and%20ecology%20in%20hotspots&amp;author=A%20Gunjal&amp;author=DS%20Bhagat&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0023">
<cite>
Heins RA, Cheng X, Nath S, Deng K, Bowen BP, Chivian DC, et al. Phylogenomically guided identification of industrially relevant GH1 β‐glucosidases through DNA synthesis and nanostructure‐initiator mass spectrometry. ACS Chem Biol. 2014;9:2082–2091.
</cite> [<a href="https://doi.org/10.1021/cb500244v" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4168791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24984213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Chem%20Biol&amp;title=Phylogenomically%20guided%20identification%20of%20industrially%20relevant%20GH1%20%CE%B2%E2%80%90glucosidases%20through%20DNA%20synthesis%20and%20nanostructure%E2%80%90initiator%20mass%20spectrometry&amp;author=RA%20Heins&amp;author=X%20Cheng&amp;author=S%20Nath&amp;author=K%20Deng&amp;author=BP%20Bowen&amp;volume=9&amp;publication_year=2014&amp;pages=2082-2091&amp;pmid=24984213&amp;doi=10.1021/cb500244v&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0024">
<cite>
Helbert W, Poulet L, Drouillard S, Mathieu S, Loiodice M, Couturier M, et al. Discovery of novel carbohydrate‐active enzymes through the rational exploration of the protein sequences space. Proc Natl Acad Sci U S A. 2019;116:6063–6068.
</cite> [<a href="https://doi.org/10.1073/pnas.1815791116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6442616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30850540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Discovery%20of%20novel%20carbohydrate%E2%80%90active%20enzymes%20through%20the%20rational%20exploration%20of%20the%20protein%20sequences%20space&amp;author=W%20Helbert&amp;author=L%20Poulet&amp;author=S%20Drouillard&amp;author=S%20Mathieu&amp;author=M%20Loiodice&amp;volume=116&amp;publication_year=2019&amp;pages=6063-6068&amp;pmid=30850540&amp;doi=10.1073/pnas.1815791116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0025">
<cite>
Henrissat B, Davies G. Structural and sequence‐based classification of glycoside hydrolases. Curr Opin Struct Biol. 1997;7:637–644.
</cite> [<a href="https://doi.org/10.1016/s0959-440x(97)80072-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9345621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Struct%20Biol&amp;title=Structural%20and%20sequence%E2%80%90based%20classification%20of%20glycoside%20hydrolases&amp;author=B%20Henrissat&amp;author=G%20Davies&amp;volume=7&amp;publication_year=1997&amp;pages=637-644&amp;pmid=9345621&amp;doi=10.1016/s0959-440x(97)80072-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0026">
<cite>
Hess B, Bekker H, Berendsen HJC, Fraaije JGEM. LINCS: a linear constraint solver for molecular simulations. J Comput Chem. 1997;18:1463–1472.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Chem&amp;title=LINCS:%20a%20linear%20constraint%20solver%20for%20molecular%20simulations&amp;author=B%20Hess&amp;author=H%20Bekker&amp;author=HJC%20Berendsen&amp;author=JGEM%20Fraaije&amp;volume=18&amp;publication_year=1997&amp;pages=1463-1472&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0027">
<cite>
Iannino N, Briones G, Tolmasky M, Ugalde R. Molecular cloning and characterization of <em>cgs</em>, the <em>Brucella abortus</em> cyclic β(1‐2) glucan synthetase gene: genetic complementation of <em>Rhizobium meliloti ndvB</em> and <em>Agrobacterium tumefaciens chvB</em> mutants. J Bacteriol. 1998;180:4392–4400.
</cite> [<a href="https://doi.org/10.1128/jb.180.17.4392-4400.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC107446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9721274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Molecular%20cloning%20and%20characterization%20of%20cgs,%20the%20Brucella%20abortus%20cyclic%20%CE%B2(1%E2%80%902)%20glucan%20synthetase%20gene:%20genetic%20complementation%20of%20Rhizobium%20meliloti%20ndvB%20and%20Agrobacterium%20tumefaciens%20chvB%20mutants&amp;author=N%20Iannino&amp;author=G%20Briones&amp;author=M%20Tolmasky&amp;author=R%20Ugalde&amp;volume=180&amp;publication_year=1998&amp;pages=4392-4400&amp;pmid=9721274&amp;doi=10.1128/jb.180.17.4392-4400.1998&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0028">
<cite>
Ishiguro R, Tanaka N, Abe K, Nakajima M, Maeda T, Miyanaga A, et al. Function and structure relationships of a β‐1,2‐glucooligosaccharide‐degrading β‐glucosidase. FEBS Lett. 2017;591:3926–3936.
</cite> [<a href="https://doi.org/10.1002/1873-3468.12911" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29131329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&amp;title=Function%20and%20structure%20relationships%20of%20a%20%CE%B2%E2%80%901,2%E2%80%90glucooligosaccharide%E2%80%90degrading%20%CE%B2%E2%80%90glucosidase&amp;author=R%20Ishiguro&amp;author=N%20Tanaka&amp;author=K%20Abe&amp;author=M%20Nakajima&amp;author=T%20Maeda&amp;volume=591&amp;publication_year=2017&amp;pages=3926-3936&amp;pmid=29131329&amp;doi=10.1002/1873-3468.12911&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0029">
<cite>
Janeček Š, Svensson B, MacGregor EA. α‐Amylase: an enzyme specificity found in various families of glycoside hydrolases. Cell and Mol Life Sci. 2014;71:1149–1170.
</cite> [<a href="https://doi.org/10.1007/s00018-013-1388-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11114072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23807207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20and%20Mol%20Life%20Sci&amp;title=%CE%B1%E2%80%90Amylase:%20an%20enzyme%20specificity%20found%20in%20various%20families%20of%20glycoside%20hydrolases&amp;author=%C5%A0%20Jane%C4%8Dek&amp;author=B%20Svensson&amp;author=EA%20MacGregor&amp;volume=71&amp;publication_year=2014&amp;pages=1149-1170&amp;pmid=23807207&amp;doi=10.1007/s00018-013-1388-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0030">
<cite>
Jorgensen WL, Chandrasekhar J, Madura JD, Impey RW, Klein ML. Comparison of simple potential functions for simulating liquid water. J Chem Phys. 1983;79:926–935.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Phys&amp;title=Comparison%20of%20simple%20potential%20functions%20for%20simulating%20liquid%20water&amp;author=WL%20Jorgensen&amp;author=J%20Chandrasekhar&amp;author=JD%20Madura&amp;author=RW%20Impey&amp;author=ML%20Klein&amp;volume=79&amp;publication_year=1983&amp;pages=926-935&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0031">
<cite>
Jourdan S, Francis IM, Deflandre B, Tenconi E, Riley J, Planckaert S, et al. Contribution of the β‐glucosidase BglC to the onset of the pathogenic lifestyle of <em>Streptomyces scabies</em>
. Mol Plant Pathol. 2018;19:1480–1490.
</cite> [<a href="https://doi.org/10.1111/mpp.12631" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6638027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29077242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Plant%20Pathol&amp;title=Contribution%20of%20the%20%CE%B2%E2%80%90glucosidase%20BglC%20to%20the%20onset%20of%20the%20pathogenic%20lifestyle%20of%20Streptomyces%20scabies&amp;author=S%20Jourdan&amp;author=IM%20Francis&amp;author=B%20Deflandre&amp;author=E%20Tenconi&amp;author=J%20Riley&amp;volume=19&amp;publication_year=2018&amp;pages=1480-1490&amp;pmid=29077242&amp;doi=10.1111/mpp.12631&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0032">
<cite>
Kabsch W. XDS. Acta Crystallogr D Biol Crystallogr. 2010;D66:125–132.</cite> [<a href="https://doi.org/10.1107/S0907444909047337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2815665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20124692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Biol%20Crystallogr&amp;title=XDS&amp;author=W%20Kabsch&amp;volume=D66&amp;publication_year=2010&amp;pages=125-132&amp;pmid=20124692&amp;doi=10.1107/S0907444909047337&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0033">
<cite>
Kim TH, Ku SK, Lee IC, Bae JS. Anti‐inflammatory effects of kaempferol‐3‐<em>O</em>‐sophoroside in human endothelial cells. Inflamm Res. 2012;61:217–224.
</cite> [<a href="https://doi.org/10.1007/s00011-011-0403-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22113342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Inflamm%20Res&amp;title=Anti%E2%80%90inflammatory%20effects%20of%20kaempferol%E2%80%903%E2%80%90O%E2%80%90sophoroside%20in%20human%20endothelial%20cells&amp;author=TH%20Kim&amp;author=SK%20Ku&amp;author=IC%20Lee&amp;author=JS%20Bae&amp;volume=61&amp;publication_year=2012&amp;pages=217-224&amp;pmid=22113342&amp;doi=10.1007/s00011-011-0403-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0034">
<cite>
Kirschner KN, Yongye AB, Tschampel SM, González‐Outeiriño J, Daniels CR, Foley BL, et al. GLYCAM06: a generalizable biomolecular force field. Carbohydrates. J Comput Chem. 2008;29:622–655.
</cite> [<a href="https://doi.org/10.1002/jcc.20820" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4423547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17849372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Chem&amp;title=GLYCAM06:%20a%20generalizable%20biomolecular%20force%20field.%20Carbohydrates&amp;author=KN%20Kirschner&amp;author=AB%20Yongye&amp;author=SM%20Tschampel&amp;author=J%20Gonz%C3%A1lez%E2%80%90Outeiri%C3%B1o&amp;author=CR%20Daniels&amp;volume=29&amp;publication_year=2008&amp;pages=622-655&amp;pmid=17849372&amp;doi=10.1002/jcc.20820&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0035">
<cite>
Kitahata S, Edagawa S. Cyclic (1→2)‐β‐d‐glucan‐hydrolyzing enzymes from <em>Acremonium</em> sp. 15: purification and some properties of <em>endo</em>‐(1→2)‐β‐d‐glucanase and β‐d‐glucosidase. Agric Biol Chem. 1987;51:2701–2708.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Agric%20Biol%20Chem&amp;title=Cyclic%20(1%E2%86%922)%E2%80%90%CE%B2%E2%80%90d%E2%80%90glucan%E2%80%90hydrolyzing%20enzymes%20from%20Acremonium%20sp.%2015:%20purification%20and%20some%20properties%20of%20endo%E2%80%90(1%E2%86%922)%E2%80%90%CE%B2%E2%80%90d%E2%80%90glucanase%20and%20%CE%B2%E2%80%90d%E2%80%90glucosidase&amp;author=S%20Kitahata&amp;author=S%20Edagawa&amp;volume=51&amp;publication_year=1987&amp;pages=2701-2708&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0036">
<cite>
Kobayashi K, Nakajima M, Aramasa H, Kimura S, Iwata T, Nakai H, et al. Large‐scale preparation of β‐1,2‐glucan using quite a small amount of sophorose. Biosci Biotechnol Biochem. 2019;83:1867–1874.
</cite> [<a href="https://doi.org/10.1080/09168451.2019.1630257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31189457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biosci%20Biotechnol%20Biochem&amp;title=Large%E2%80%90scale%20preparation%20of%20%CE%B2%E2%80%901,2%E2%80%90glucan%20using%20quite%20a%20small%20amount%20of%20sophorose&amp;author=K%20Kobayashi&amp;author=M%20Nakajima&amp;author=H%20Aramasa&amp;author=S%20Kimura&amp;author=T%20Iwata&amp;volume=83&amp;publication_year=2019&amp;pages=1867-1874&amp;pmid=31189457&amp;doi=10.1080/09168451.2019.1630257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0037">
<cite>
Kobayashi K, Shimizu H, Tanaka N, Kuramochi K, Nakai H, Nakajima M, et al. Characterization and structural analyses of a novel glycosyltransferase acting on the β‐1,2‐glucosidic linkages. J Biol Chem. 2022;298:101606.
</cite> [<a href="https://doi.org/10.1016/j.jbc.2022.101606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8861115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35065074/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Characterization%20and%20structural%20analyses%20of%20a%20novel%20glycosyltransferase%20acting%20on%20the%20%CE%B2%E2%80%901,2%E2%80%90glucosidic%20linkages&amp;author=K%20Kobayashi&amp;author=H%20Shimizu&amp;author=N%20Tanaka&amp;author=K%20Kuramochi&amp;author=H%20Nakai&amp;volume=298&amp;publication_year=2022&amp;pages=101606&amp;pmid=35065074&amp;doi=10.1016/j.jbc.2022.101606&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0038">
<cite>
Krissinel E, Henrick K. Inference of macromolecular assemblies from crystalline state. J Mol Biol. 2007;372:774–797.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2007.05.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17681537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Inference%20of%20macromolecular%20assemblies%20from%20crystalline%20state&amp;author=E%20Krissinel&amp;author=K%20Henrick&amp;volume=372&amp;publication_year=2007&amp;pages=774-797&amp;pmid=17681537&amp;doi=10.1016/j.jmb.2007.05.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0039">
<cite>
Lacey HJ, Rutledge PJ. Recently discovered secondary metabolites from <em>Streptomyces</em> species. Molecules. 2022;27:887.
</cite> [<a href="https://doi.org/10.3390/molecules27030887" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8838263/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35164153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Recently%20discovered%20secondary%20metabolites%20from%20Streptomyces%20species&amp;author=HJ%20Lacey&amp;author=PJ%20Rutledge&amp;volume=27&amp;publication_year=2022&amp;pages=887&amp;pmid=35164153&amp;doi=10.3390/molecules27030887&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0040">
<cite>
Lenardon MD, Munro CA, Gow NAR. Chitin synthesis and fungal pathogenesis. Curr Opin Microbiol. 2010;13:416–423.
</cite> [<a href="https://doi.org/10.1016/j.mib.2010.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2923753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20561815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Microbiol&amp;title=Chitin%20synthesis%20and%20fungal%20pathogenesis&amp;author=MD%20Lenardon&amp;author=CA%20Munro&amp;author=NAR%20Gow&amp;volume=13&amp;publication_year=2010&amp;pages=416-423&amp;pmid=20561815&amp;doi=10.1016/j.mib.2010.05.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0041">
<cite>
Linton SM. Review: the structure and function of cellulase (endo‐β‐1,4‐glucanase) and hemicellulase (β‐1,3‐glucanase and endo‐β‐1,4‐mannase) enzymes in invertebrates that consume materials ranging from microbes, algae to leaf litter. Comp Biochem Physiol B Biochem Mol Biol. 2020;240:110354.
</cite> [<a href="https://doi.org/10.1016/j.cbpb.2019.110354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31647988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comp%20Biochem%20Physiol%20B%20Biochem%20Mol%20Biol&amp;title=Review:%20the%20structure%20and%20function%20of%20cellulase%20(endo%E2%80%90%CE%B2%E2%80%901,4%E2%80%90glucanase)%20and%20hemicellulase%20(%CE%B2%E2%80%901,3%E2%80%90glucanase%20and%20endo%E2%80%90%CE%B2%E2%80%901,4%E2%80%90mannase)%20enzymes%20in%20invertebrates%20that%20consume%20materials%20ranging%20from%20microbes,%20algae%20to%20leaf%20litter&amp;author=SM%20Linton&amp;volume=240&amp;publication_year=2020&amp;pages=110354&amp;pmid=31647988&amp;doi=10.1016/j.cbpb.2019.110354&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0042">
<cite>
Lombard V, Bernard T, Rancurel C, Brumer H, Coutinho PM, Henrissat B. A hierarchical classification of polysaccharide lyases for glycogenomics. Biochem J. 2010;432:437–444.
</cite> [<a href="https://doi.org/10.1042/BJ20101185" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20925655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20J&amp;title=A%20hierarchical%20classification%20of%20polysaccharide%20lyases%20for%20glycogenomics&amp;author=V%20Lombard&amp;author=T%20Bernard&amp;author=C%20Rancurel&amp;author=H%20Brumer&amp;author=PM%20Coutinho&amp;volume=432&amp;publication_year=2010&amp;pages=437-444&amp;pmid=20925655&amp;doi=10.1042/BJ20101185&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0043">
<cite>
Maier JA, Martinez C, Kasavajhala K, Wickstrom L, Hauser KE, Simmerling C. ff14SB: improving the accuracy of protein side chain and backbone oarameters from ff99SB. J Chem Theory Comput. 2015;11:3696–3713.
</cite> [<a href="https://doi.org/10.1021/acs.jctc.5b00255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4821407/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26574453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Theory%20Comput&amp;title=ff14SB:%20improving%20the%20accuracy%20of%20protein%20side%20chain%20and%20backbone%20oarameters%20from%20ff99SB&amp;author=JA%20Maier&amp;author=C%20Martinez&amp;author=K%20Kasavajhala&amp;author=L%20Wickstrom&amp;author=KE%20Hauser&amp;volume=11&amp;publication_year=2015&amp;pages=3696-3713&amp;pmid=26574453&amp;doi=10.1021/acs.jctc.5b00255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0044">
<cite>
Mastromei G, Hanhart E, Perito B, Polsinelli M. Cloning and expression in <em>Escherichia coli</em> of a <em>Streptomyces</em> β‐glucosidase gene. J Biotechnol. 1992;24:149–157.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biotechnol&amp;title=Cloning%20and%20expression%20in%20Escherichia%20coli%20of%20a%20Streptomyces%20%CE%B2%E2%80%90glucosidase%20gene&amp;author=G%20Mastromei&amp;author=E%20Hanhart&amp;author=B%20Perito&amp;author=M%20Polsinelli&amp;volume=24&amp;publication_year=1992&amp;pages=149-157&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0045">
<cite>
Matsuzawa T, Watanabe M, Nakamichi Y, Akita H, Yaoi K. Crystal structure of metagenomic β‐glycosidase MeBglD2 in complex with various saccharides. Appl Microbiol Biotechnol. 2022;106:4539–4551.
</cite> [<a href="https://doi.org/10.1007/s00253-022-12018-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35723691/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&amp;title=Crystal%20structure%20of%20metagenomic%20%CE%B2%E2%80%90glycosidase%20MeBglD2%20in%20complex%20with%20various%20saccharides&amp;author=T%20Matsuzawa&amp;author=M%20Watanabe&amp;author=Y%20Nakamichi&amp;author=H%20Akita&amp;author=K%20Yaoi&amp;volume=106&amp;publication_year=2022&amp;pages=4539-4551&amp;pmid=35723691&amp;doi=10.1007/s00253-022-12018-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0046">
<cite>
McIntire FC, Peterson WH, Riker AJ. Factors influencing the carbon metabolism of the crown gall organism. J Agric Res. 1940;61:313–320.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Agric%20Res&amp;title=Factors%20influencing%20the%20carbon%20metabolism%20of%20the%20crown%20gall%20organism&amp;author=FC%20McIntire&amp;author=WH%20Peterson&amp;author=AJ%20Riker&amp;volume=61&amp;publication_year=1940&amp;pages=313-320&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0047">
<cite>
McIntire FC, Peterson WH, Riker AJ. A polysaccharide produced by the crown‐gall organism. J Biol Chem. 1942;143:491–496.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=A%20polysaccharide%20produced%20by%20the%20crown%E2%80%90gall%20organism&amp;author=FC%20McIntire&amp;author=WH%20Peterson&amp;author=AJ%20Riker&amp;volume=143&amp;publication_year=1942&amp;pages=491-496&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0048">
<cite>
Mei J, Wu X, Zheng S, Chen X, Huang Z, Wu Y. Improvement of cucurbitacin B content in cucumis melo pedicel extracts by biotransformation using recombinant β‐glucosidase. Separations. 2021;8:138.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Separations&amp;title=Improvement%20of%20cucurbitacin%20B%20content%20in%20cucumis%20melo%20pedicel%20extracts%20by%20biotransformation%20using%20recombinant%20%CE%B2%E2%80%90glucosidase&amp;author=J%20Mei&amp;author=X%20Wu&amp;author=S%20Zheng&amp;author=X%20Chen&amp;author=Z%20Huang&amp;volume=8&amp;publication_year=2021&amp;pages=138&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0049">
<cite>
Méndez‐Líter JA, Nieto‐Domínguez M, Fernández de Toro B, González Santana A, Prieto A, Asensio JL, et al. A glucotolerant β‐glucosidase from the fungus <em>Talaromyces amestolkiae</em> and its conversion into a glycosynthase for glycosylation of phenolic compounds. Microb Cell Fact. 2020;19:127.
</cite> [<a href="https://doi.org/10.1186/s12934-020-01386-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7288487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32522206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microb%20Cell%20Fact&amp;title=A%20glucotolerant%20%CE%B2%E2%80%90glucosidase%20from%20the%20fungus%20Talaromyces%20amestolkiae%20and%20its%20conversion%20into%20a%20glycosynthase%20for%20glycosylation%20of%20phenolic%20compounds&amp;author=JA%20M%C3%A9ndez%E2%80%90L%C3%ADter&amp;author=M%20Nieto%E2%80%90Dom%C3%ADnguez&amp;author=B%20Fern%C3%A1ndez%20de%20Toro&amp;author=A%20Gonz%C3%A1lez%20Santana&amp;author=A%20Prieto&amp;volume=19&amp;publication_year=2020&amp;pages=127&amp;pmid=32522206&amp;doi=10.1186/s12934-020-01386-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0050">
<cite>
Mendoza NS, Amemura A. (1,2)‐β‐d‐Glucan‐hydrolyzing enzymes in <em>Cytophaga arvensicola</em>: partial purification and some properties of <em>endo</em>‐(1,2)‐β‐glucanase and β‐d‐glucosidase specific for (1,2)‐ and (1,3)‐ linkages. J Ferment Technol. 1983;61:473–481.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Ferment%20Technol&amp;title=(1,2)%E2%80%90%CE%B2%E2%80%90d%E2%80%90Glucan%E2%80%90hydrolyzing%20enzymes%20in%20Cytophaga%20arvensicola:%20partial%20purification%20and%20some%20properties%20of%20endo%E2%80%90(1,2)%E2%80%90%CE%B2%E2%80%90glucanase%20and%20%CE%B2%E2%80%90d%E2%80%90glucosidase%20specific%20for%20(1,2)%E2%80%90%20and%20(1,3)%E2%80%90%20linkages&amp;author=NS%20Mendoza&amp;author=A%20Amemura&amp;volume=61&amp;publication_year=1983&amp;pages=473-481&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0051">
<cite>
Morris RJ, Perrakis A, Lamzin VS. ARP/wARP's model‐building algorithms. I. The main chain. Acta Crystallogr D Biol Crystallogr. 2002;58:968–975.
</cite> [<a href="https://doi.org/10.1107/s0907444902005462" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12037299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Biol%20Crystallogr&amp;title=ARP/wARP's%20model%E2%80%90building%20algorithms.%20I.%20The%20main%20chain&amp;author=RJ%20Morris&amp;author=A%20Perrakis&amp;author=VS%20Lamzin&amp;volume=58&amp;publication_year=2002&amp;pages=968-975&amp;pmid=12037299&amp;doi=10.1107/s0907444902005462&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0052">
<cite>
Motohashi K. A simple and efficient seamless DNA cloning method using SLiCE from <em>Escherichia coli</em> laboratory strains and its application to SLiP site‐directed mutagenesis. BMC Biotechnol. 2015;15:47.
</cite> [<a href="https://doi.org/10.1186/s12896-015-0162-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4453199/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26037246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Biotechnol&amp;title=A%20simple%20and%20efficient%20seamless%20DNA%20cloning%20method%20using%20SLiCE%20from%20Escherichia%20coli%20laboratory%20strains%20and%20its%20application%20to%20SLiP%20site%E2%80%90directed%20mutagenesis&amp;author=K%20Motohashi&amp;volume=15&amp;publication_year=2015&amp;pages=47&amp;pmid=26037246&amp;doi=10.1186/s12896-015-0162-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0053">
<cite>
Motouchi S, Kobayashi K, Nakai H, Nakajima M. Identification of enzymatic functions of osmo‐regulated periplasmic glucan biosynthesis proteins from <em>Escherichia coli</em> reveals a novel glycoside hydrolase family. Commun Biol. 2023;6:961.
</cite> [<a href="https://doi.org/10.1038/s42003-023-05336-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10514313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37735577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Commun%20Biol&amp;title=Identification%20of%20enzymatic%20functions%20of%20osmo%E2%80%90regulated%20periplasmic%20glucan%20biosynthesis%20proteins%20from%20Escherichia%20coli%20reveals%20a%20novel%20glycoside%20hydrolase%20family&amp;author=S%20Motouchi&amp;author=K%20Kobayashi&amp;author=H%20Nakai&amp;author=M%20Nakajima&amp;volume=6&amp;publication_year=2023&amp;pages=961&amp;pmid=37735577&amp;doi=10.1038/s42003-023-05336-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0054">
<cite>
Motouchi S, Komba S, Nakai H, Nakajima M. Discovery of anomer‐inverting transglycosylase: cyclic glucohexadecaose‐producing enzyme from <em>Xanthomonas</em>, a phytopathogen. J Am Chem Soc. 2024;146:17738–17746.
</cite> [<a href="https://doi.org/10.1021/jacs.4c02579" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11228985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38957137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Discovery%20of%20anomer%E2%80%90inverting%20transglycosylase:%20cyclic%20glucohexadecaose%E2%80%90producing%20enzyme%20from%20Xanthomonas,%20a%20phytopathogen&amp;author=S%20Motouchi&amp;author=S%20Komba&amp;author=H%20Nakai&amp;author=M%20Nakajima&amp;volume=146&amp;publication_year=2024&amp;pages=17738-17746&amp;pmid=38957137&amp;doi=10.1021/jacs.4c02579&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0055">
<cite>
Murshudov GN, Vagin AA, Dodson EJ. Refinement of macromolecular structures by the maximum‐likelihood method. Acta Crystallogr D Biol Crystallogr. 1997;53:240–255.
</cite> [<a href="https://doi.org/10.1107/S0907444996012255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15299926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Biol%20Crystallogr&amp;title=Refinement%20of%20macromolecular%20structures%20by%20the%20maximum%E2%80%90likelihood%20method&amp;author=GN%20Murshudov&amp;author=AA%20Vagin&amp;author=EJ%20Dodson&amp;volume=53&amp;publication_year=1997&amp;pages=240-255&amp;pmid=15299926&amp;doi=10.1107/S0907444996012255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0056">
<cite>
Nakajima M. β‐1,2‐Glucans and associated enzymes. Biologia (Bratisl). 2023;78:1741–1757.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Biologia%20(Bratisl)&amp;title=%CE%B2%E2%80%901,2%E2%80%90Glucans%20and%20associated%20enzymes&amp;author=M%20Nakajima&amp;volume=78&amp;publication_year=2023&amp;pages=1741-1757&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0057">
<cite>
Nakajima M, Tanaka N, Furukawa N, Nihira T, Kodutsumi Y, Takahashi Y, et al. Mechanistic insight into the substrate specificity of 1,2‐β‐oligoglucan phosphorylase from <em>Lachnoclostridium phytofermentans</em>
. Sci Rep. 2017;7:42671.
</cite> [<a href="https://doi.org/10.1038/srep42671" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5309861/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28198470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Mechanistic%20insight%20into%20the%20substrate%20specificity%20of%201,2%E2%80%90%CE%B2%E2%80%90oligoglucan%20phosphorylase%20from%20Lachnoclostridium%20phytofermentans&amp;author=M%20Nakajima&amp;author=N%20Tanaka&amp;author=N%20Furukawa&amp;author=T%20Nihira&amp;author=Y%20Kodutsumi&amp;volume=7&amp;publication_year=2017&amp;pages=42671&amp;pmid=28198470&amp;doi=10.1038/srep42671&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0058">
<cite>
Nakajima M, Tanaka N, Kobayashi K, Nakai H, Kimura S, Iwata T, et al. Enzymatic control and evaluation of degrees of polymerization of β‐(1→2)‐glucans. Anal Biochem. 2021;632:114366.
</cite> [<a href="https://doi.org/10.1016/j.ab.2021.114366" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34509443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&amp;title=Enzymatic%20control%20and%20evaluation%20of%20degrees%20of%20polymerization%20of%20%CE%B2%E2%80%90(1%E2%86%922)%E2%80%90glucans&amp;author=M%20Nakajima&amp;author=N%20Tanaka&amp;author=K%20Kobayashi&amp;author=H%20Nakai&amp;author=S%20Kimura&amp;volume=632&amp;publication_year=2021&amp;pages=114366&amp;pmid=34509443&amp;doi=10.1016/j.ab.2021.114366&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0059">
<cite>
Nakajima M, Tanaka N, Motouchi S, Kobayashi K, Shimizu H, Abe K, et al. New glycoside hydrolase families of β‐1,2‐glucanases. Protein Sci. 2025;34:e70147.
</cite> [<a href="https://doi.org/10.1002/pro.70147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12102758/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40411428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=New%20glycoside%20hydrolase%20families%20of%20%CE%B2%E2%80%901,2%E2%80%90glucanases&amp;author=M%20Nakajima&amp;author=N%20Tanaka&amp;author=S%20Motouchi&amp;author=K%20Kobayashi&amp;author=H%20Shimizu&amp;volume=34&amp;publication_year=2025&amp;pages=e70147&amp;pmid=40411428&amp;doi=10.1002/pro.70147&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0060">
<cite>
Nakajima M, Toyoizumi H, Abe K, Nakai H, Taguchi H, Kitaoka M. 1,2‐β‐oligoglucan phosphorylase from <em>Listeria innocua</em>
. PLoS One. 2014;9:e92353.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0092353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3960220/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24647662/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=1,2%E2%80%90%CE%B2%E2%80%90oligoglucan%20phosphorylase%20from%20Listeria%20innocua&amp;author=M%20Nakajima&amp;author=H%20Toyoizumi&amp;author=K%20Abe&amp;author=H%20Nakai&amp;author=H%20Taguchi&amp;volume=9&amp;publication_year=2014&amp;pages=e92353&amp;pmid=24647662&amp;doi=10.1371/journal.pone.0092353&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0061">
<cite>
Nakajima M, Yoshida R, Miyanaga A, Abe K, Takahashi Y, Sugimoto N, et al. Functional and structural analysis of a β‐glucosidase involved in β‐1,2‐glucan metabolism in <em>Listeria innocua</em>
. PLoS One. 2016;11:e0148870.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0148870" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4757417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26886583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Functional%20and%20structural%20analysis%20of%20a%20%CE%B2%E2%80%90glucosidase%20involved%20in%20%CE%B2%E2%80%901,2%E2%80%90glucan%20metabolism%20in%20Listeria%20innocua&amp;author=M%20Nakajima&amp;author=R%20Yoshida&amp;author=A%20Miyanaga&amp;author=K%20Abe&amp;author=Y%20Takahashi&amp;volume=11&amp;publication_year=2016&amp;pages=e0148870&amp;pmid=26886583&amp;doi=10.1371/journal.pone.0148870&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0062">
<cite>
Neelamegham S, Aoki‐Kinoshita K, Bolton E, Frank M, Lisacek F, Lütteke T, et al. Updates to the symbol nomenclature for Glycans guidelines. Glycobiology. 2019;29:620–624.
</cite> [<a href="https://doi.org/10.1093/glycob/cwz045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7335484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31184695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Glycobiology&amp;title=Updates%20to%20the%20symbol%20nomenclature%20for%20Glycans%20guidelines&amp;author=S%20Neelamegham&amp;author=K%20Aoki%E2%80%90Kinoshita&amp;author=E%20Bolton&amp;author=M%20Frank&amp;author=F%20Lisacek&amp;volume=29&amp;publication_year=2019&amp;pages=620-624&amp;pmid=31184695&amp;doi=10.1093/glycob/cwz045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0063">
<cite>
Opassiri R, Hua Y, Wara‐Aswapati O, Akiyama T, Svasti J, Esen A, et al. β‐Glucosidase, exo‐β‐glucanase and pyridoxine transglucosylase activities of rice BGlu1. Biochem J. 2004;379:125–131.
</cite> [<a href="https://doi.org/10.1042/BJ20031485" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1224054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14692878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20J&amp;title=%CE%B2%E2%80%90Glucosidase,%20exo%E2%80%90%CE%B2%E2%80%90glucanase%20and%20pyridoxine%20transglucosylase%20activities%20of%20rice%20BGlu1&amp;author=R%20Opassiri&amp;author=Y%20Hua&amp;author=O%20Wara%E2%80%90Aswapati&amp;author=T%20Akiyama&amp;author=J%20Svasti&amp;volume=379&amp;publication_year=2004&amp;pages=125-131&amp;pmid=14692878&amp;doi=10.1042/BJ20031485&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0064">
<cite>
Pace CN, Vajdos F, Fee L, Grimsley G, Gray T. How to measure and predict the molar absorption coefficient of a protein. Protein Sci. 1995;4:2411–2423.
</cite> [<a href="https://doi.org/10.1002/pro.5560041120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2143013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8563639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=How%20to%20measure%20and%20predict%20the%20molar%20absorption%20coefficient%20of%20a%20protein&amp;author=CN%20Pace&amp;author=F%20Vajdos&amp;author=L%20Fee&amp;author=G%20Grimsley&amp;author=T%20Gray&amp;volume=4&amp;publication_year=1995&amp;pages=2411-2423&amp;pmid=8563639&amp;doi=10.1002/pro.5560041120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0065">
<cite>
Park SJ, Choi JM, Kyeong HH, Kim SG, Kim HS. Rational design of a β‐glycosidase with high regiospecificity for triterpenoid tailoring. Chembiochem. 2015;16:854–860.
</cite> [<a href="https://doi.org/10.1002/cbic.201500004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25703680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chembiochem&amp;title=Rational%20design%20of%20a%20%CE%B2%E2%80%90glycosidase%20with%20high%20regiospecificity%20for%20triterpenoid%20tailoring&amp;author=SJ%20Park&amp;author=JM%20Choi&amp;author=HH%20Kyeong&amp;author=SG%20Kim&amp;author=HS%20Kim&amp;volume=16&amp;publication_year=2015&amp;pages=854-860&amp;pmid=25703680&amp;doi=10.1002/cbic.201500004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0066">
<cite>
Putman EW, Potter AL, Hodgson R, Hassid WZ. The structure of crown‐gall polysaccharide. I. J Am Chem Soc. 1950;72:5024–5026.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=The%20structure%20of%20crown%E2%80%90gall%20polysaccharide.%20I&amp;author=EW%20Putman&amp;author=AL%20Potter&amp;author=R%20Hodgson&amp;author=WZ%20Hassid&amp;volume=72&amp;publication_year=1950&amp;pages=5024-5026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0067">
<cite>
Reese ET, Parrish FW, Mandels M. β‐d‐1,2‐Glucanases in fungi. Can J Microbiol. 1961;7:309–317.
</cite> [<a href="https://doi.org/10.1139/m61-038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/13740314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Can%20J%20Microbiol&amp;title=%CE%B2%E2%80%90d%E2%80%901,2%E2%80%90Glucanases%20in%20fungi&amp;author=ET%20Reese&amp;author=FW%20Parrish&amp;author=M%20Mandels&amp;volume=7&amp;publication_year=1961&amp;pages=309-317&amp;pmid=13740314&amp;doi=10.1139/m61-038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0068">
<cite>
Robert X, Gouet P. Deciphering key features in protein structures with the new ENDscript server. Nucleic Acids Res. 2014;42:320–324.</cite> [<a href="https://doi.org/10.1093/nar/gku316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4086106/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24753421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Deciphering%20key%20features%20in%20protein%20structures%20with%20the%20new%20ENDscript%20server&amp;author=X%20Robert&amp;author=P%20Gouet&amp;volume=42&amp;publication_year=2014&amp;pages=320-324&amp;pmid=24753421&amp;doi=10.1093/nar/gku316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0069">
<cite>
Roset MS, Ciocchini AE, Ugalde RA, Iñón de Iannino N. Molecular cloning and characterization of <em>cgt</em>, the <em>Brucella abortus</em> cyclic β‐1,2‐glucan transporter gene, and its role in virulence. Infect Immun. 2004;72:2263–2271.
</cite> [<a href="https://doi.org/10.1128/IAI.72.4.2263-2271.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC375185/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15039351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&amp;title=Molecular%20cloning%20and%20characterization%20of%20cgt,%20the%20Brucella%20abortus%20cyclic%20%CE%B2%E2%80%901,2%E2%80%90glucan%20transporter%20gene,%20and%20its%20role%20in%20virulence&amp;author=MS%20Roset&amp;author=AE%20Ciocchini&amp;author=RA%20Ugalde&amp;author=N%20I%C3%B1%C3%B3n%20de%20Iannino&amp;volume=72&amp;publication_year=2004&amp;pages=2263-2271&amp;pmid=15039351&amp;doi=10.1128/IAI.72.4.2263-2271.2004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0070">
<cite>
Sansenya S, Opassiri R, Kuaprasert B, Chen CJ, Cairns JRK. The crystal structure of rice (<em>Oryza sativa</em> L.) Os4BGlu12, an oligosaccharide and tuberonic acid glucoside‐hydrolyzing β‐glucosidase with significant thioglucohydrolase activity. Arch Biochem Biophys. 2011;510:62–72.
</cite> [<a href="https://doi.org/10.1016/j.abb.2011.04.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21521631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arch%20Biochem%20Biophys&amp;title=The%20crystal%20structure%20of%20rice%20(Oryza%20sativa%20L.)%20Os4BGlu12,%20an%20oligosaccharide%20and%20tuberonic%20acid%20glucoside%E2%80%90hydrolyzing%20%CE%B2%E2%80%90glucosidase%20with%20significant%20thioglucohydrolase%20activity&amp;author=S%20Sansenya&amp;author=R%20Opassiri&amp;author=B%20Kuaprasert&amp;author=CJ%20Chen&amp;author=JRK%20Cairns&amp;volume=510&amp;publication_year=2011&amp;pages=62-72&amp;pmid=21521631&amp;doi=10.1016/j.abb.2011.04.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0071">
<cite>
Schröder C, Blank S, Antranikian G. First glycoside hydrolase family 2 enzymes from <em>Thermus antranikianii</em> and <em>Thermus brockianus</em> with ß‐glucosidase activity. Front Bioeng Biotechnol. 2015;3:76.
</cite> [<a href="https://doi.org/10.3389/fbioe.2015.00076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4453472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26090361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Bioeng%20Biotechnol&amp;title=First%20glycoside%20hydrolase%20family%202%20enzymes%20from%20Thermus%20antranikianii%20and%20Thermus%20brockianus%20with%20%C3%9F%E2%80%90glucosidase%20activity&amp;author=C%20Schr%C3%B6der&amp;author=S%20Blank&amp;author=G%20Antranikian&amp;volume=3&amp;publication_year=2015&amp;pages=76&amp;pmid=26090361&amp;doi=10.3389/fbioe.2015.00076&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0072">
<cite>
Sedzicki J, Ni D, Lehmann F, Stahlberg H, Dehio C. Structure‐function analysis of the cyclic β‐1,2‐glucan synthase from <em>Agrobacterium tumefaciens</em>
. Nat Commun. 2024;15:1844.
</cite> [<a href="https://doi.org/10.1038/s41467-024-45415-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10901819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38418509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Structure%E2%80%90function%20analysis%20of%20the%20cyclic%20%CE%B2%E2%80%901,2%E2%80%90glucan%20synthase%20from%20Agrobacterium%20tumefaciens&amp;author=J%20Sedzicki&amp;author=D%20Ni&amp;author=F%20Lehmann&amp;author=H%20Stahlberg&amp;author=C%20Dehio&amp;volume=15&amp;publication_year=2024&amp;pages=1844&amp;pmid=38418509&amp;doi=10.1038/s41467-024-45415-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0073">
<cite>
Seshadri S, Akiyama T, Opassiri R, Kuaprasert B, Cairns JK. Structural and enzymatic characterization of Os3BGlu6, a rice β‐glucosidase hydrolyzing hydrophobic glycosides and (1→3)‐ and (1→2)‐linked disaccharides. Plant Physiol. 2009;151:47–58.
</cite> [<a href="https://doi.org/10.1104/pp.109.139436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2735989/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19587102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Structural%20and%20enzymatic%20characterization%20of%20Os3BGlu6,%20a%20rice%20%CE%B2%E2%80%90glucosidase%20hydrolyzing%20hydrophobic%20glycosides%20and%20(1%E2%86%923)%E2%80%90%20and%20(1%E2%86%922)%E2%80%90linked%20disaccharides&amp;author=S%20Seshadri&amp;author=T%20Akiyama&amp;author=R%20Opassiri&amp;author=B%20Kuaprasert&amp;author=JK%20Cairns&amp;volume=151&amp;publication_year=2009&amp;pages=47-58&amp;pmid=19587102&amp;doi=10.1104/pp.109.139436&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0074">
<cite>
Shen Y, Li Z, Huo YY, Bao L, Gao B, Xiao P, et al. Structural and functional insights into CmGH1, a novel GH39 family β‐glucosidase from deep‐sea bacterium. Front Microbiol. 2019;10:2922.
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.02922" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6933502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31921083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Structural%20and%20functional%20insights%20into%20CmGH1,%20a%20novel%20GH39%20family%20%CE%B2%E2%80%90glucosidase%20from%20deep%E2%80%90sea%20bacterium&amp;author=Y%20Shen&amp;author=Z%20Li&amp;author=YY%20Huo&amp;author=L%20Bao&amp;author=B%20Gao&amp;volume=10&amp;publication_year=2019&amp;pages=2922&amp;pmid=31921083&amp;doi=10.3389/fmicb.2019.02922&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0075">
<cite>
Shimizu H, Nakajima M, Miyanaga A, Takahashi Y, Tanaka N, Kobayashi K, et al. Characterization and structural analysis of a novel <em>exo</em>‐type enzyme acting on β‐1,2‐glucooligosaccharides from <em>Parabacteroides distasonis</em>
. Biochemistry. 2018;57:3849–3860.
</cite> [<a href="https://doi.org/10.1021/acs.biochem.8b00385" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29763309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Characterization%20and%20structural%20analysis%20of%20a%20novel%20exo%E2%80%90type%20enzyme%20acting%20on%20%CE%B2%E2%80%901,2%E2%80%90glucooligosaccharides%20from%20Parabacteroides%20distasonis&amp;author=H%20Shimizu&amp;author=M%20Nakajima&amp;author=A%20Miyanaga&amp;author=Y%20Takahashi&amp;author=N%20Tanaka&amp;volume=57&amp;publication_year=2018&amp;pages=3849-3860&amp;pmid=29763309&amp;doi=10.1021/acs.biochem.8b00385&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0076">
<cite>
Tanaka N, Nakajima M, Narukawa‐Nara M, Matsunaga H, Kamisuki S, Aramasa H, et al. Identification, characterization, and structural analyses of a fungal <em>endo</em>‐β‐1,2‐glucanase reveal a new glycoside hydrolase family. J Biol Chem. 2019;294:7942–7965.
</cite> [<a href="https://doi.org/10.1074/jbc.RA118.007087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6514616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30926603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Identification,%20characterization,%20and%20structural%20analyses%20of%20a%20fungal%20endo%E2%80%90%CE%B2%E2%80%901,2%E2%80%90glucanase%20reveal%20a%20new%20glycoside%20hydrolase%20family&amp;author=N%20Tanaka&amp;author=M%20Nakajima&amp;author=M%20Narukawa%E2%80%90Nara&amp;author=H%20Matsunaga&amp;author=S%20Kamisuki&amp;volume=294&amp;publication_year=2019&amp;pages=7942-7965&amp;pmid=30926603&amp;doi=10.1074/jbc.RA118.007087&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0077">
<cite>
Tanaka N, Saito R, Kobayashi K, Nakai H, Kamo S, Kuramochi K, et al. Functional and structural analysis of a cyclization domain in a cyclic β‐1,2‐glucan synthase. Appl Microbiol Biotechnol. 2024;108:187.
</cite> [<a href="https://doi.org/10.1007/s00253-024-13013-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10834661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38300345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&amp;title=Functional%20and%20structural%20analysis%20of%20a%20cyclization%20domain%20in%20a%20cyclic%20%CE%B2%E2%80%901,2%E2%80%90glucan%20synthase&amp;author=N%20Tanaka&amp;author=R%20Saito&amp;author=K%20Kobayashi&amp;author=H%20Nakai&amp;author=S%20Kamo&amp;volume=108&amp;publication_year=2024&amp;pages=187&amp;pmid=38300345&amp;doi=10.1007/s00253-024-13013-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0078">
<cite>
Vagin A, Teplyakov A. Molecular replacement with MOLREP. Acta Crystallogr D Biol Crystallogr. 2010;66:22–25.
</cite> [<a href="https://doi.org/10.1107/S0907444909042589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20057045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr%20D%20Biol%20Crystallogr&amp;title=Molecular%20replacement%20with%20MOLREP&amp;author=A%20Vagin&amp;author=A%20Teplyakov&amp;volume=66&amp;publication_year=2010&amp;pages=22-25&amp;pmid=20057045&amp;doi=10.1107/S0907444909042589&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0079">
<cite>
Verdoucq L, Morinière J, Bevan DR, Esen A, Vasella A, Henrissat B, et al. Structural determinants of substrate specificity in family 1 β‐glucosidases: novel insights from the crystal structure of sorghum dhurrinase‐1, a plant β‐glucosidase with strict specificity, in complex with its natural substrate. J Biol Chem. 2004;279:31796–31803.
</cite> [<a href="https://doi.org/10.1074/jbc.M402918200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15148317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Structural%20determinants%20of%20substrate%20specificity%20in%20family%201%20%CE%B2%E2%80%90glucosidases:%20novel%20insights%20from%20the%20crystal%20structure%20of%20sorghum%20dhurrinase%E2%80%901,%20a%20plant%20%CE%B2%E2%80%90glucosidase%20with%20strict%20specificity,%20in%20complex%20with%20its%20natural%20substrate&amp;author=L%20Verdoucq&amp;author=J%20Morini%C3%A8re&amp;author=DR%20Bevan&amp;author=A%20Esen&amp;author=A%20Vasella&amp;volume=279&amp;publication_year=2004&amp;pages=31796-31803&amp;pmid=15148317&amp;doi=10.1074/jbc.M402918200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0080">
<cite>
Yamada C, Gotoh A, Sakanaka M, Hattie M, Stubbs KA, Katayama‐Ikegami A, et al. Molecular insight into evolution of symbiosis between breast‐fed infants and a member of the human gut microbiome <em>Bifidobacterium longum</em>
. Cell Chem Biol. 2017;24:515–524.e5.
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2017.03.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28392148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Chem%20Biol&amp;title=Molecular%20insight%20into%20evolution%20of%20symbiosis%20between%20breast%E2%80%90fed%20infants%20and%20a%20member%20of%20the%20human%20gut%20microbiome%20Bifidobacterium%20longum&amp;author=C%20Yamada&amp;author=A%20Gotoh&amp;author=M%20Sakanaka&amp;author=M%20Hattie&amp;author=KA%20Stubbs&amp;volume=24&amp;publication_year=2017&amp;pages=515-524.e5&amp;pmid=28392148&amp;doi=10.1016/j.chembiol.2017.03.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0081">
<cite>
Zeeman SC, Kossmann J, Smith AM. Starch: its metabolism, evolution, and biotechnological modification in plants. Annu Rev Plant Biol. 2010;61:209–234.
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-042809-112301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20192737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Plant%20Biol&amp;title=Starch:%20its%20metabolism,%20evolution,%20and%20biotechnological%20modification%20in%20plants&amp;author=SC%20Zeeman&amp;author=J%20Kossmann&amp;author=AM%20Smith&amp;volume=61&amp;publication_year=2010&amp;pages=209-234&amp;pmid=20192737&amp;doi=10.1146/annurev-arplant-042809-112301&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0082">
<cite>
Zevenhuizen LPTM, Scholten‐Koerselman HJ. Surface carbohydrates of <em>Rhizobium</em>
. Antonie Van Leeuwenhoek. 1979;45:165–175.
</cite> [<a href="https://doi.org/10.1007/BF00418581" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/582886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antonie%20Van%20Leeuwenhoek&amp;title=Surface%20carbohydrates%20of%20Rhizobium&amp;author=LPTM%20Zevenhuizen&amp;author=HJ%20Scholten%E2%80%90Koerselman&amp;volume=45&amp;publication_year=1979&amp;pages=165-175&amp;pmid=582886&amp;doi=10.1007/BF00418581&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70255-bib-0083">
<cite>
Zorreguieta A, Ugalde RA. Formation in <em>Rhizobium</em> and <em>Agrobacterium</em> spp. of a 235‐kilodalton protein intermediate in β‐D(1‐2) glucan synthesis. J Bacteriol. 1986;167:947–951.
</cite> [<a href="https://doi.org/10.1128/jb.167.3.947-951.1986" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC215963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3745125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Formation%20in%20Rhizobium%20and%20Agrobacterium%20spp.%20of%20a%20235%E2%80%90kilodalton%20protein%20intermediate%20in%20%CE%B2%E2%80%90D(1%E2%80%902)%20glucan%20synthesis&amp;author=A%20Zorreguieta&amp;author=RA%20Ugalde&amp;volume=167&amp;publication_year=1986&amp;pages=947-951&amp;pmid=3745125&amp;doi=10.1128/jb.167.3.947-951.1986&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Data S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12369401/bin/PRO-34-e70255-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70255-s001.docx</a><sup> (9.6MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
<strong>Data S2.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12369401/bin/PRO-34-e70255-s002.txt" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70255-s002.txt</a><sup> (3.9KB, txt) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The atomic coordinates and structure factors (codes 8KAP and 9U98) have been deposited in the PDB. The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Protein Science : A Publication of the Protein Society are provided here courtesy of <strong>The Protein Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/pro.70255"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/PRO-34-e70255.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12369401/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12369401/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369401%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369401/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12369401/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12369401/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40838837/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12369401/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40838837/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12369401/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12369401/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="y5uviziZYn5buMB2PaBCWle3tmlmS7OQKcd50vBrIyAI7kPfy5sZaaeug7t50PZM">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
