
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genetic determinants of pOXA-48 plasmid maintenance and propagation in Escherichia coli - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152548818AF35BC30548810024F20002.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365149/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Genetic determinants of pOXA-48 plasmid maintenance and propagation in Escherichia coli">
<meta name="citation_author" content="Yannick Baffert">
<meta name="citation_author_institution" content="Microbiologie Moléculaire et Biochimie Structurale (MMSB), Université Lyon 1, CNRS, Inserm, UMR5086, Lyon, France">
<meta name="citation_author" content="Nathan Fraikin">
<meta name="citation_author_institution" content="Microbiologie Moléculaire et Biochimie Structurale (MMSB), Université Lyon 1, CNRS, Inserm, UMR5086, Lyon, France">
<meta name="citation_author" content="Yasmine Makhloufi">
<meta name="citation_author_institution" content="Microbiologie Moléculaire et Biochimie Structurale (MMSB), Université Lyon 1, CNRS, Inserm, UMR5086, Lyon, France">
<meta name="citation_author" content="Julie Baltenneck">
<meta name="citation_author_institution" content="Microbiologie Adaptation et Pathogénie (MAP), Université Lyon 1, INSA de Lyon, CNRS, UMR 5240, Villeurbanne, France">
<meta name="citation_author" content="Marie-Eve Val">
<meta name="citation_author_institution" content="Institut Pasteur, Université Paris Cité, CNRS UMR3525, Unité Plasticité du Génome Bactérien, Département Génomes et Génétique, Paris, France">
<meta name="citation_author" content="Annick Dedieu-Berne">
<meta name="citation_author_institution" content="Microbiologie Moléculaire et Biochimie Structurale (MMSB), Université Lyon 1, CNRS, Inserm, UMR5086, Lyon, France">
<meta name="citation_author" content="Jonathan Degosserie">
<meta name="citation_author_institution" content="Namur Molecular Tech, UCLouvain, CHU UCL Namur, Yvoir, Belgium">
<meta name="citation_author" content="Bogdan I Iorga">
<meta name="citation_author_institution" content="Université Paris-Saclay, CNRS UPR 2301, Institut de Chimie des Substances Naturelles, Gif-sur-Yvette, France">
<meta name="citation_author" content="Pierre Bogaerts">
<meta name="citation_author_institution" content="National reference center for antimicrobial resistance in Gram negative, CHU UCL Namur, Yvoir, Belgium">
<meta name="citation_author" content="Erwan Gueguen">
<meta name="citation_author_institution" content="Microbiologie Adaptation et Pathogénie (MAP), Université Lyon 1, INSA de Lyon, CNRS, UMR 5240, Villeurbanne, France">
<meta name="citation_author" content="Christian Lesterlin">
<meta name="citation_author_institution" content="Microbiologie Moléculaire et Biochimie Structurale (MMSB), Université Lyon 1, CNRS, Inserm, UMR5086, Lyon, France">
<meta name="citation_author" content="Sarah Bigot">
<meta name="citation_author_institution" content="Microbiologie Moléculaire et Biochimie Structurale (MMSB), Université Lyon 1, CNRS, Inserm, UMR5086, Lyon, France">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7734">
<meta name="citation_doi" content="10.1038/s41467-025-62404-7">
<meta name="citation_pmid" content="40830342">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365149/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365149/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365149/pdf/41467_2025_Article_62404.pdf">
<meta name="description" content="Conjugative plasmids are the main drivers of antibiotic resistance dissemination contributing to the emergence and extensive spread of multidrug resistance clinical bacterial pathogens. pOXA-48 plasmids, belonging to the IncL group, emerge as the ...">
<meta name="og:title" content="Genetic determinants of pOXA-48 plasmid maintenance and propagation in Escherichia coli">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Conjugative plasmids are the main drivers of antibiotic resistance dissemination contributing to the emergence and extensive spread of multidrug resistance clinical bacterial pathogens. pOXA-48 plasmids, belonging to the IncL group, emerge as the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365149/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365149">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62404-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62404.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365149%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365149/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365149/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365149/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 19;16:7734. doi: <a href="https://doi.org/10.1038/s41467-025-62404-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62404-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genetic determinants of pOXA-48 plasmid maintenance and propagation in <em>Escherichia coli</em>
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baffert%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yannick Baffert</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yannick Baffert</span></h3>
<div class="p">
<sup>1</sup>Microbiologie Moléculaire et Biochimie Structurale (MMSB), Université Lyon 1, CNRS, Inserm, UMR5086, Lyon, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baffert%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yannick Baffert</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fraikin%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Nathan Fraikin</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Nathan Fraikin</span></h3>
<div class="p">
<sup>1</sup>Microbiologie Moléculaire et Biochimie Structurale (MMSB), Université Lyon 1, CNRS, Inserm, UMR5086, Lyon, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fraikin%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nathan Fraikin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Makhloufi%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yasmine Makhloufi</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yasmine Makhloufi</span></h3>
<div class="p">
<sup>1</sup>Microbiologie Moléculaire et Biochimie Structurale (MMSB), Université Lyon 1, CNRS, Inserm, UMR5086, Lyon, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Makhloufi%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yasmine Makhloufi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baltenneck%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Julie Baltenneck</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Julie Baltenneck</span></h3>
<div class="p">
<sup>2</sup>Microbiologie Adaptation et Pathogénie (MAP), Université Lyon 1, INSA de Lyon, CNRS, UMR 5240, Villeurbanne, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baltenneck%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Julie Baltenneck</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Val%20ME%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Marie-Eve Val</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Marie-Eve Val</span></h3>
<div class="p">
<sup>3</sup>Institut Pasteur, Université Paris Cité, CNRS UMR3525, Unité Plasticité du Génome Bactérien, Département Génomes et Génétique, Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Val%20ME%22%5BAuthor%5D" class="usa-link"><span class="name western">Marie-Eve Val</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dedieu-Berne%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Annick Dedieu-Berne</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Annick Dedieu-Berne</span></h3>
<div class="p">
<sup>1</sup>Microbiologie Moléculaire et Biochimie Structurale (MMSB), Université Lyon 1, CNRS, Inserm, UMR5086, Lyon, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dedieu-Berne%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Annick Dedieu-Berne</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Degosserie%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jonathan Degosserie</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jonathan Degosserie</span></h3>
<div class="p">
<sup>4</sup>Namur Molecular Tech, UCLouvain, CHU UCL Namur, Yvoir, Belgium </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Degosserie%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jonathan Degosserie</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iorga%20BI%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Bogdan I Iorga</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Bogdan I Iorga</span></h3>
<div class="p">
<sup>5</sup>Université Paris-Saclay, CNRS UPR 2301, Institut de Chimie des Substances Naturelles, Gif-sur-Yvette, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iorga%20BI%22%5BAuthor%5D" class="usa-link"><span class="name western">Bogdan I Iorga</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bogaerts%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Pierre Bogaerts</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Pierre Bogaerts</span></h3>
<div class="p">
<sup>6</sup>National reference center for antimicrobial resistance in Gram negative, CHU UCL Namur, Yvoir, Belgium </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bogaerts%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pierre Bogaerts</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gueguen%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Erwan Gueguen</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Erwan Gueguen</span></h3>
<div class="p">
<sup>2</sup>Microbiologie Adaptation et Pathogénie (MAP), Université Lyon 1, INSA de Lyon, CNRS, UMR 5240, Villeurbanne, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gueguen%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Erwan Gueguen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lesterlin%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Christian Lesterlin</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Christian Lesterlin</span></h3>
<div class="p">
<sup>1</sup>Microbiologie Moléculaire et Biochimie Structurale (MMSB), Université Lyon 1, CNRS, Inserm, UMR5086, Lyon, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lesterlin%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christian Lesterlin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bigot%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Sarah Bigot</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Sarah Bigot</span></h3>
<div class="p">
<sup>1</sup>Microbiologie Moléculaire et Biochimie Structurale (MMSB), Université Lyon 1, CNRS, Inserm, UMR5086, Lyon, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bigot%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sarah Bigot</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Microbiologie Moléculaire et Biochimie Structurale (MMSB), Université Lyon 1, CNRS, Inserm, UMR5086, Lyon, France </div>
<div id="Aff2">
<sup>2</sup>Microbiologie Adaptation et Pathogénie (MAP), Université Lyon 1, INSA de Lyon, CNRS, UMR 5240, Villeurbanne, France </div>
<div id="Aff3">
<sup>3</sup>Institut Pasteur, Université Paris Cité, CNRS UMR3525, Unité Plasticité du Génome Bactérien, Département Génomes et Génétique, Paris, France </div>
<div id="Aff4">
<sup>4</sup>Namur Molecular Tech, UCLouvain, CHU UCL Namur, Yvoir, Belgium </div>
<div id="Aff5">
<sup>5</sup>Université Paris-Saclay, CNRS UPR 2301, Institut de Chimie des Substances Naturelles, Gif-sur-Yvette, France </div>
<div id="Aff6">
<sup>6</sup>National reference center for antimicrobial resistance in Gram negative, CHU UCL Namur, Yvoir, Belgium </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 26; Accepted 2025 Jul 21; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365149  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830342/" class="usa-link">40830342</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Conjugative plasmids are the main drivers of antibiotic resistance dissemination contributing to the emergence and extensive spread of multidrug resistance clinical bacterial pathogens. pOXA-48 plasmids, belonging to the IncL group, emerge as the primary vehicle for carbapenem resistance in <em>Enterobacteriaceae</em>. Despite the problematic prevalence of pOXA-48, most research focus on epidemiology and genomics, leaving gaps in our understanding of the mechanisms behind its propagation. In this study, we use a transposon sequencing approach to identify genetic elements critical for plasmid stability, replication, and conjugative transfer. Our results identify a novel type I toxin-antitoxin system, uncharacterized essential maintenance factors, and components of the type IV secretion system and regulatory elements crucial for conjugation. This study advances our understanding of pOXA-48 biology, providing key insights into the genetic factors underlying its successful maintenance and spread in bacterial populations.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Antimicrobials, Bacterial genetics</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">pOXA-48 plasmids have emerged as key vectors of carbapenem resistance within Enterobacteriaceae. In this study, the authors use a transposon sequencing (Tn-seq) approach to identify genetic determinants critical for plasmid stability and conjugative transfer.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Conjugative plasmids are extrachromosomal DNA elements responsible for the spread of antimicrobial resistance (AMR) through the transfer of resistance genes across bacterial populations. Multidrug-resistant commensal, environmental, and pathogenic strains have revealed a wide range of conjugative plasmids carrying genes that confer resistance to nearly all antibiotics currently used in clinical treatments. Carbapenem resistance in <em>Enterobacteriaceae</em> has emerged as a critical public health concern, largely attributed to the IncL group pOXA-48 plasmids. These plasmids are the main carriers of the <em>bla</em><sub>OXA-48</sub> gene, which encodes the OXA-48 carbapenemase within the Tn<em>1999</em> transposon<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>.</p>
<p id="Par4">Despite the rapid global dissemination of pOXA-48 among enteric pathogens, previous studies have primarily focused on epidemiological tracking and genomic characterization, leaving the mechanistic aspects of its transfer and persistence poorly defined. Recent works have shed light on the plasmid dynamics of pOXA-48 in different bacterial hosts, revealing variability in fitness costs and conjugation across strains<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. This variability suggests that plasmid persistence and transfer are influenced by host diversity, evolutionary pressures, and complex genetic interactions<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Consistent with these findings, a recent study discovered the peculiarity of the <em>Escherichia coli</em> ST38 strain to integrate the <em>bla</em><sub>OXA-48</sub> gene into its chromosome, driven by the fitness cost associated with the pOXA-48 plasmid and its destabilization by a recent characterized antiplasmid system, ApsAB<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Altogether, these results suggest that understanding genetic interactions between plasmids and hosts is key to predicting successful plasmid maintenance and spread.</p>
<p id="Par5">However, the factors and molecular mechanisms that facilitate the successful adaptation, maintenance, and spread of the pOXA-48 plasmid remain poorly understood. Conjugative plasmids often rely on functional elements for replication, stability, and conjugative transfer, which include replication and partitioning systems for stable inheritance, toxin-antitoxin systems that kill cells that have lost the plasmid, and conjugative machinery for horizontal gene transfer. Moreover, plasmid transfer imposes a fitness cost due to the synthesis of the type IV secretion system (T4SS), which drives the transfer of plasmid DNA into recipient cells. Plasmid-encoded regulatory factors are known to control the transfer and mitigate such fitness costs. While such factors have been yet identified for many conjugative plasmids of the IncF, IncX, IncA/C, and IncHI groups<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, none have yet been identified for pOXA-48. Understanding these mechanisms is crucial to determining how pOXA-48 balances the cost of transfer while ensuring its persistence across diverse bacterial hosts. While several genetic modules of pOXA-48 can be predicted based on its similarity to the IncM plasmid pCTX-M3<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, experimental validation has been scarce. To date, only the type II toxin-antitoxin system PemIK has been experimentally shown to contribute to plasmid maintenance and eliminate plasmid-free cells through a post-segregational killing mechanism<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Other factors involved in replication, stability, and conjugation remain largely unidentified experimentally.</p>
<p id="Par6">In this study, we employed a transposon insertion sequencing (Tn-Seq) approach to systematically identify genes necessary for the maintenance, stability, and efficient transfer of the pOXA-48 plasmid in <em>E. coli</em>. By combining genetic, microscopy, bioinformatics, and structural modeling approaches, we have identified genetic determinants that ensure the success of pOXA-48 in bacterial populations.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">High-density transposon insertion library of the pOXA-48 plasmid</h3>
<p id="Par7">To validate the genetic elements required for pOXA-48 maintenance and transfer, we first sequenced the plasmid isolated from <em>Klebsiella pneumoniae</em><sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>, revealing a 63,580 bp conjugative plasmid. The plasmid exhibits 95.6% nucleotide identity with the IncM pCTX-M3 plasmid<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, which allowed us to extrapolate its genetic content. Among the 83 predicted genes, several were annotated based on sequence similarity, including the <em>tra</em>, <em>trb</em> and <em>rep</em> genes, which are likely involved in conjugative transfer and replication, respectively (Fig. <a href="#Fig1" class="usa-link">1</a> and Table <a href="#MOESM1" class="usa-link">S1</a>). Interestingly, 43% of the predicted genes remain uncharacterized.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Circular genetic map of the pOXA-48 plasmid.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365149_41467_2025_62404_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc23/12365149/a35020c12c59/41467_2025_62404_Fig1_HTML.jpg" loading="lazy" id="d33e385" height="514" width="674" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The outer and inner arrows represent the coding DNA sequences (CDS) on the forward and reverse strands, respectively. Each CDS is color-coded according to its predicted function, as indicated in the figure legend. The categories include transposon elements (Tn<em>1999</em>), insertion sequences (IS), genes involved in plasmid maintenance/establishment, replication, transfer, and others, as well as genes with unknown function.</p></figcaption></figure><p id="Par8">To explore these unknown factors, we constructed a high-density transposon-insertion mutant library of the pOXA-48 plasmid in the <em>E. coli</em> K12 MG1655 strain, referred to as the input library. The Himar1 transposition cassette inserts either into the chromosome of <em>E. coli</em> or into the pOXA-48a plasmid. Two experimental replicates were generated, and the reads were aligned to the pOXA-48 plasmid. A total of 916,455 reads aligning to a TA dinucleotide border of pOXA-48 were mapped. Given that pOXA-48a contains 2788 TAs, the average insertion density was 52.3% and 50.9% for replicates 1 and 2, respectively, while the average number of reads per TA was 323.5 and 176.4 (Supplementary Data <a href="#MOESM4" class="usa-link">1</a>). For the analysis, given that the Pearson correlation between the two replicates was 0.76, we decided to aggregate the results from the two experimental replicates. This analysis identified 1671 unique insertion sites, with 59.9% of the 2788 TA dinucleotide sites of pOXA-48 showing successful insertion with the Himar-1 transposon, without any detectable insertion bias. To determine which genes are involved in plasmid transfer, we subjected the input library to two independent conjugation assays in which the mutated plasmid pool was transferred to an <em>E. coli</em> MG1655 recipient strain. Sequencing of the resulting output library 1 and 2 of transconjugants yielded a total of 28,844,013 reads, all of which mapped to the pOXA-48 plasmid, with a Pearson correlation between the two replicates of 0.91. Only 44.1% of the plasmid’s TA sites showed insertions (Supplementary Data <a href="#MOESM4" class="usa-link">1</a>). These findings indicate that a substantial portion of the pOXA-48 plasmid plays a crucial role in the transfer process.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Functional mapping of replication, segregation, and stability genes in the pOXA-48 plasmid</h3>
<p id="Par9">To identify the key elements required for pOXA-48 maintenance, we first analyzed the input transposon library. By calculating the Log1p of mean read counts for each gene, we identified seven genes with either null read values (<em>repA</em>, <em>repB</em>, <em>korC</em>) or low transposon insertion frequencies (<em>parA</em>, <em>parB</em>, <em>orf19</em> and <em>orf20</em>), indicating their potential roles in the maintenance of the pOXA-48 plasmid (Fig. <a href="#Fig2" class="usa-link">2a</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Identification of genes required for replication and maintenance of pOXA-48 plasmid.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365149_41467_2025_62404_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc23/12365149/b70e06cdc331/41467_2025_62404_Fig2_HTML.jpg" loading="lazy" id="d33e513" height="994" width="786" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Tn-Seq analysis of transposon insertion abundance across the pOXA-48 plasmid represented as Log1p(reads/gene). The mean of reads (filled line) with SD (dotted lines) are represented. Genes are color-coded based on their predicted function. <strong>b</strong> Read coverage of transposon insertions around the <em>repCBA</em> locus. The schematic below shows the genomic context of these replication-related genes, highlighting regulatory elements, including the antisense RNA molecule RNAI and the origin of replication (<em>oriV</em>). <strong>c</strong> Tn-Seq read distribution for the <em>korC</em> gene and its neighboring region. The putative binding sites of KorC are shown below, along with the predicted promoter regions and the <em>orf24</em>’s codon start. <strong>d</strong> Read counts for the transposon insertions across the <em>resD</em>, <em>parA</em>, <em>parB</em>, <em>nuc</em>, <em>orf19</em>, <em>orf20</em>, and <em>orf21</em> genes. <strong>e</strong> Stability assay of pOXA-48 plasmid derivatives with deletions of <em>parA</em>, <em>parB</em>, and <em>nuc</em> over 40 generations. The percentage of cells retaining the plasmid was measured at different time points, and the mean and SD of three independent clones are represented. <strong>f</strong> Effect of the <em>orf19</em> deletion on cell growth was monitored by measuring the optical density at 600 nm (OD<sub>600</sub>). The mean and SD of three independent clones are shown. <strong>g</strong> Stability assay of pOXA-48 plasmid deleted of <em>orf19</em> at time 0 and after 40 generations. The percentage of cells retaining the plasmid is estimated from three independent clones (white dots).</p></figcaption></figure><p id="Par10">The replication of IncL/M plasmids relies on the RepA initiation protein, whose synthesis is translationally coupled to the expression of the leader peptide RepB. Plasmid copy number control occurs at two levels: transcriptionally, through the repressor protein RepC, which inhibits the activity of the P<sub><em>repA</em></sub> promoter, and post-transcriptionally, via the antisense RNA molecule RNAI, which binds to its complementary sequence in the leader region of the <em>repA</em> mRNA, thereby inhibiting RepA translation<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> (Fig. <a href="#Fig2" class="usa-link">2b</a>). The absence of transposon insertions in the region containing the predicted origin of replication (<em>oriV</em>) and the annotated replication initiation genes <em>repA</em>, <em>repB</em> underscores their essential role in the maintenance of the pOXA-48 plasmid (Fig. <a href="#Fig2" class="usa-link">2b</a>).</p>
<p id="Par11">Beyond the replication region, analysis of the average read counts showed a significant lack of transposon insertions in the <em>korC</em> gene, suggesting its potential role in plasmid maintenance (Fig. <a href="#Fig2" class="usa-link">2a</a>, <a href="#Fig2" class="usa-link">c</a>). This hypothesis was further supported by a parallel study, which demonstrated that silencing <em>korC</em> expression by CRISPR interference led to a strong defect in pOXA-48 stability<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. The <em>korC</em> gene encodes a putative transcriptional repressor that shares 39% homology with the KorC repressor of IncP RP4 plasmid, which is known to regulate the promoter regions upstream of <em>klcA</em>, <em>kleA</em>, and <em>kleC</em> genes, although its precise function in RP4 plasmid physiology remains undefined. A search for putative KorC binding sites from the IncL/M group identified a consensus sequence<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> which we found uniquely between the predicted <em>−35</em> and <em>−10</em> boxes of the <em>orf24-korC</em> operon promoter, suggesting potential autoregulation of <em>korC</em> expression (Fig. <a href="#Fig2" class="usa-link">2c</a>). The absence of transposon insertions in this sequence suggests that the <em>korC</em> promoter is located here and supports the essential role of KorC in pOXA-48 maintenance.</p>
<p id="Par12">The <em>parA</em> and <em>parB</em> genes, which showed lower insertion abundance (Fig. <a href="#Fig2" class="usa-link">2a</a>, <a href="#Fig2" class="usa-link">d</a>), share 98% and 99% nucleotide identity, respectively, with the <em>parA</em> and <em>parB</em> genes of the IncM pCTX-M3 plasmid, which encode a partitioning system<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> (Table <a href="#MOESM1" class="usa-link">S1</a>). Stability assays confirmed that the ParAB system of pOXA-48 plays a crucial role in the plasmid stable inheritance, as the deletion of either the <em>parA</em> or <em>parB</em> gene led to a rapid loss of the plasmid in the absence of selection pressure (Fig. <a href="#Fig2" class="usa-link">2e</a>). Consistently, this loss was associated with a growth defect under selective pressure (Fig. <a href="#MOESM1" class="usa-link">S1</a>). Interestingly, unlike the reported role of the putative nuclease Nuc in the partitioning of the pCTX-M3 plasmid<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, we found that Nuc of pOXA-48 does not contribute to plasmid partitioning, as its deletion did not affect plasmid stability (Fig. <a href="#Fig2" class="usa-link">2e</a>).</p>
<p id="Par13">We identified two genes of unknown function, <em>orf19</em> without any conserved domain, and <em>orf20</em> predicted to encode an XRE-like transcriptional regulator, both of which showed a lower abundance of transposon insertions in the pOXA-48 Tn-Seq library (Fig. <a href="#Fig2" class="usa-link">2a</a>, <a href="#Fig2" class="usa-link">d</a>). This observation suggested that <em>orf19</em> and <em>orf20</em> might play essential roles in plasmid maintenance. To explore their potential contributions, we first generated a deletion of <em>orf19</em> and determined the effect of its absence on host cells. Deletion of <em>orf19</em> had no noticeable impact as the growth of pOXA-48 mutant and wild-type strains was similar in the presence or absence of selective pressure (Fig. <a href="#Fig2" class="usa-link">2f</a>). Moreover, we evaluated the stability of the plasmid mutant and found that 100% of cells still contained the <em>orf19</em> plasmid mutant after 40 generations of growth (Fig. <a href="#Fig2" class="usa-link">2g</a>). These findings suggested that the reduced number of transposon insertions observed for <em>orf19</em> in the Tn-Seq analysis may be attributable to a polar effect on the expression of <em>orf20</em> suggesting that <em>orf19</em>-<em>orf20</em> form an operon. In contrast, generating a plasmid deleted for <em>orf20</em> proved to be highly challenging, as its deletion was only obtained when <em>orf20</em> was provided in <em>trans</em> on a complementation plasmid, suggesting its essential role in plasmid maintenance. This finding is corroborated by the study of Calvo-Villamañán et al., which demonstrated that silencing <em>orf20</em> (DNDJGHEP_0014) for 24 h led to a drastic plasmid loss in <em>E. coli</em><sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>.</p>
<p id="Par14">To further explore the impact of acquiring an <em>orf20</em>-deleted plasmid, we performed real-time imaging using microfluidic chambers in combination with the following fluorescence reporter system (Movie <a href="#MOESM3" class="usa-link">S1</a>). We introduced a P<sub>Tet</sub>-sfGFP fluorescent reporter, consisting of the <em>superfolder gfp</em> gene under the control of the P<sub>Tet</sub> promoter, into an intergenic region of the <em>orf20</em>-deleted pOXA-48 plasmid. The TetR-producing donor cells exhibited no fluorescence due to <em>sfgfp</em> gene repression by TetR and expressed <em>orf20</em> from the complementation plasmid pOrf20 to maintain the <em>orf20</em>-deleted plasmid. The TetR-free recipient cells carried the mCherry reporter system that conferred red fluorescence. Transconjugants acquiring the <em>orf20</em>-deleted plasmid relieve TetR repression and produced sfGFP, appearing yellow (red and green fluorescence) in overlay images (Movie <a href="#MOESM3" class="usa-link">S1</a>). In these experiments, we initially mixed donor and recipient cells on a filter for 4 h prior to real-time imaging of the resulting conjugation mix. Microfluidic experiments were then performed with a constant flow of ampicillin, which selected for plasmid-maintaining cells that produced the carbapenemase OXA-48, enabling growth in the presence of ampicillin. We observed that recipient cells expressing <em>orf20</em> in <em>trans</em> acquired the plasmid, and the resulting yellow transconjugants proliferated under selection pressure. In contrast, the Orf20-free recipient cells did not successfully maintain the plasmid. Only a few transconjugants were observed, and these cells were elongated, indicating halted division despite their ability to resist ampicillin. Additionally, some red recipient cells exhibited abnormal elongation and increased cell width before undergoing bulging and eventual lysis, a phenotype characteristic of ampicillin-induced stress. These results suggest that these recipient cells initially acquired the <em>orf20</em>-deleted plasmid and likely lost it rapidly. The temporary production of the carbapenemase OXA-48 may have delayed ampicillin-induced killing, but the plasmid loss finally ultimately resulted in filamentation and then lysis. This highlights the crucial role of <em>orf20</em> in plasmid stability, explaining why a viable <em>orf20</em> deletion mutant could not be obtained.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Identification and characterization of a novel type I toxin-antitoxin system</h3>
<p id="Par15">We identified an unannotated region between the <em>repA</em> and <em>trbC</em> genes (Fig. <a href="#Fig1" class="usa-link">1</a>), which prompted further investigation into potential coding sequences within this region. Detailed analysis revealed a likely type I toxin-antitoxin (TA) system of a DqlB (<em>dinQ</em>-like B) family that was never experimentally validated<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. This family consists of DqlB, a short helical hydrophobic toxic peptide, and the noncoding RNA <em>agrB trans</em>-acting antitoxin that should inhibit translation of <em>dqlB</em> by complementary base paring<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. We identified a 35-nt sequence in the 5’ untranslated region (5’ UTR) of the <em>dqlB</em> mRNA that is complementary to the <em>agrB</em> RNA (Fig. <a href="#Fig3" class="usa-link">3a</a> and Fig. <a href="#MOESM1" class="usa-link">S2a</a>), suggesting that base-pairing between these elements could inhibit the translation of <em>dqlB</em>. Tn-Seq analysis supported the essentiality of this predicted antitoxin, as transposon insertions were not recovered in the predicted <em>agrB</em> RNA or its potential promoter region, indicating its crucial role in plasmid maintenance (Fig. <a href="#Fig3" class="usa-link">3a</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Identification of a type I toxin-antitoxin encoded by the pOXA-48 plasmid.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365149_41467_2025_62404_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc23/12365149/de671785a99e/41467_2025_62404_Fig3_HTML.jpg" loading="lazy" id="d33e882" height="669" width="799" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Tn-Seq data showing read counts for transposon insertions across the <em>agrB</em>/<em>dqlB</em> region of the pOXA-48 plasmid. The schematic below illustrates the genomic organization of the predicted toxin-antitoxin system, with <em>agrB</em> in green encoding the antitoxin (A) and <em>dqlB</em> in gray the toxin (T). <strong>b</strong> Spot assays evaluating the toxicity of DqlB and the protective effect of <em>agrB</em>. Growth of <em>E. coli</em> (−/−; LY4078), <em>E. coli</em> producing <em>agrB</em> (LY4080), DqlB (LY4079) or <em>agrB</em> and DqlB (LY4081) is shown under conditions with either glucose or arabinose. <strong>c</strong> Schematic representation of the Targeted Antibacterial Plasmid (TAP) system used to investigate the role of <em>agrB</em> and DqlB. Donor cells carry a TAP-OXA degrading the pOXA-48 plasmid, leading to cell death or a TAP-OXA-A targeting the pOXA-48 plasmid but producing the antitoxin gene to counteract the toxin activity and rescue the transconjugant cell. The histograms show the number of viable transconjugants (CFU/mL) after acquisition of the different TAPs. Donors TAP-nsp (LY1369), TAP-OXA48 (LY1522), TAP-OXA48-PemI (LY1549), TAP-OXA48-agrB (LY4038); Recipients (MS388). Mean and SD are calculated from six independent experiments. <em>P-</em>value significance from Tukey’s multiple comparisons statistical analysis is indicated by ns: not significant, *** (<em>p</em> = 0.0002), **** (<em>p</em> &lt; 0.0001). <strong>d</strong> Histograms showing the proportion of Inc plasmids containing a <em>dqlB</em>-like homolog.</p></figcaption></figure><p id="Par16">To confirm the functionality of this TA system, the <em>dqlB</em> gene, including its 5’ UTR, was cloned into a low-copy vector under the P<sub>BAD</sub> arabinose-inducible promoter. Production of the DqlB peptide, induced by arabinose, was found to be toxic, as no growth was observed under inducing conditions (Fig. <a href="#Fig3" class="usa-link">3b</a>). Microscopic examination of DqlB-producing cells revealed the presence of ghost cells (Fig. <a href="#MOESM1" class="usa-link">S2b</a>), suggesting that DqlB targets the inner membrane and compromises its integrity through a mechanism similar to its DinQ homolog<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> or the Hok toxin from plasmid R1<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Furthermore, co-expression of <em>agrB</em> in <em>trans</em> provided protection against the toxic effects of DqlB, confirming that <em>agrB</em> functions as an antitoxin (Fig. <a href="#Fig3" class="usa-link">3b</a>). To determine whether repression of <em>dqlB</em> by <em>agrB</em> relies on a complementary base-pairing mechanism between the <em>agrB</em> RNA and the 5’ UTR of <em>dqlB</em>, we constructed an <em>agrB</em> mutant with a 4-nt mismatch in the predicted pairing region (Fig. <a href="#MOESM1" class="usa-link">S2a</a>). This mutant failed to repress the toxicity induced by wild-type DqlB (Fig. <a href="#MOESM1" class="usa-link">S2c</a>). We then generated a compensatory mutation into the 5’ UTR of <em>dqlB</em>, restoring complementarity to the mutated <em>agrB</em>. Reestablishing base pairing fully restored the ability of the <em>agrB</em> mutant to neutralize DqlB toxicity (Fig. <a href="#MOESM1" class="usa-link">S2c</a>). These results demonstrate that specific base-pairing interactions are critical for <em>agrB</em> to function as an effective antitoxin.</p>
<p id="Par17">To validate DqlB-<em>agrB</em> as an addiction module, <em>i.e</em>. a system that promotes post-segregational killing of plasmid-cured cells, we took advantage of an antibacterial strategy previously designed to cure the pOXA-48 plasmid<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. In this system, a mobilizable Targeted Antibacterial Plasmid (TAP) encoding a CRISPR system targeting the pOXA-48 plasmid (TAP-OXA48) is transferred by conjugation in pOXA-48-containing cells. CRISPR-mediated cleavage of pOXA-48 leads to its loss, which triggers post-segregational killing though antitoxin depletion, toxin activation and cell death (Fig. <a href="#Fig3" class="usa-link">3c</a>). Providing antitoxins by cloning them on TAP (TAP-OXA48-A) prevent this viability loss by preventing toxin activation (Fig. <a href="#Fig3" class="usa-link">3c</a>). This approach was previously used to validate the activity of the PemIK type II TA system encoded on the pOXA-48 plasmid. In that case, production of the PemI antitoxin by a TAP-OXA48-PemI resulted in partial rescue of transconjugant viability compared to transconjugants acquiring a non-specific TAP (TAP-nsp) which did not degrade the plasmid and thus did not activate any TA system<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> (Fig. <a href="#Fig3" class="usa-link">3c</a>). This partial rescue suggested that another yet unidentified TA system could be responsible for the remaining viability loss that could not be complemented through PemI production. We therefore assess the role of DqlB-<em>agrB</em> in this loss of viability by cloning <em>agrB</em> in TAP-OXA48. Remarkably, expression of <em>agrB</em> alone was sufficient to rescue killing by TAP-OXA48, confirming that DqlB-<em>agrB</em> induces cell death upon plasmid loss and therefore functions as an addiction module (Fig. <a href="#Fig3" class="usa-link">3c</a>).</p>
<p id="Par18">Given that DqlB-<em>agrB</em> is a major stability determinant of pOXA-48, we investigated the conservation of this system across various plasmids. To address this question, we mined the PLSDB database<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup> for <em>dqlB</em> homolog, using an Hmm model based on seven known <em>dqlB</em><sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> homologs and the pOXA-48-encoded <em>dqlB</em> nucleotide sequence as a query. A total of 3310 plasmids carrying a copy of this gene was found among a subset of 15775 plasmids for which the incompatibility group was well defined (Fig. <a href="#MOESM1" class="usa-link">S3a</a>). Homologs of <em>dqlB</em> are ubiquitous in the IncL/M taxonomic unit, with 99.3% of IncL and 95.5% of IncM plasmids encoding a copy of <em>dqlB</em>. The <em>dqlB</em> gene is also prevalent on IncB/O/K/Z (83.2%), IncI1 (67.4%) and IncX (53.3%) (Fig. <a href="#Fig3" class="usa-link">3d</a>). Homologs can also be found uncommonly among IncN (16.6%), IncHI (5.5%), IncF (4.2%) and IncP1 (2.9%), and sporadically on IncC (0.66%), IncR (0.4%) and IncY (0.37%) plasmids. Notably, the 3310 <em>dqlB</em> homologs identified during our screen were highly redundant. To study the diversity of these homologs, identical sequences were collapsed together and used to build a maximum likelihood tree. Each leave was ultimately decorated with the incompatibility group distribution of their plasmids (Fig. <a href="#MOESM1" class="usa-link">S3b</a>). Overall, <em>dqlB</em> homologs are distributed in clusters corresponding to the incompatibility group of their vector, with the exception of IncF-encoded homologs, which are found scattered across the tree, suggesting that IncF plasmids are prone to acquire this TA system by horizontal gene transfer from another plasmid. Interestingly, <em>dqlB</em> homologs from pOXA-48 and other IncL/M plasmids form a monophyletic group with IncN-encoded homologs (Fig. <a href="#MOESM1" class="usa-link">S3b</a>), which could therefore indicate a common origin for the acquisition of this toxin between IncL/M and IncN plasmids. Overall, these results show that the DqlB-<em>agrB</em> TA system can be commonly found among plasmids. In practice, however, its presence is currently likely overlooked as it is not always annotated accordingly due to its small size.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Genes required for efficient pOXA-48 plasmid transfer</h3>
<p id="Par19">The comparison between the initial transposon library and the post-transfer library provided insights into the genes required for efficient plasmid transfer. The analysis of the Log<sub>2</sub>FC (fold-change) indicated that 24 genes were underrepresented in the transconjugant population, suggesting their essential role in the conjugative transfer process (Fig. <a href="#Fig4" class="usa-link">4</a>). Specifically, disruptions in <em>tra</em> and <em>trb</em> genes encoding for the T4SS components (Table <a href="#MOESM1" class="usa-link">S1</a>) led to a marked reduction of transposon insertion reads, thus confirming their key role in the transfer of the pOXA-48 plasmid. Interestingly, we also identified two hypothetical proteins, Orf36.1 and Orf38, with unknown functions, whose disruption appeared to significantly impact transfer efficiency, suggesting that these proteins could represent novel factors involved in plasmid conjugation (Fig. <a href="#Fig4" class="usa-link">4</a>). To further investigate the roles of <em>orf36</em>.<em>1</em> and <em>orf38</em> in conjugation and validate the Tn-Seq findings, we assessed the impact of deleting these genes on conjugation efficiency (Fig. <a href="#Fig5" class="usa-link">5</a>). Deletion of <em>orf38</em> led to a 10<sup>6</sup>-fold decrease in conjugation efficiency, whereas deletion of <em>orf36.1</em> caused only a threefold reduction (Fig. <a href="#Fig5" class="usa-link">5</a>), indicating that Orf38 is more critical for conjugation than Orf36.1. The ectopic production of Orf36.1 from a complementation plasmid fully restored conjugation frequency to the wild-type levels, and production of Orf38 in the <em>orf38</em> mutant led to a 10<sup>5</sup>-fold increase in conjugation frequency. However, this complementation remained partial, as it did not fully restore the transfer frequency to that of the wild-type.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Identification of genes essential for conjugation and enriched in transposon insertion sites.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365149_41467_2025_62404_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc23/12365149/67c282a901fd/41467_2025_62404_Fig4_HTML.jpg" loading="lazy" id="d33e1120" height="190" width="798" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Log<sub>2</sub>(FC) analysis of transposon insertion abundance comparing the output library to the input library. Threshold of 2 and −2 are indicated by the dotted lines. Genes are color-coded based on their predicted function.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Impact of <em>orf38</em> and <em>orf36</em>.1 deletions on conjugation frequency.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc23/12365149/e5183523339a/41467_2025_62404_Fig5_HTML.jpg" loading="lazy" id="d33e1161" height="667" width="662" alt="Fig. 5"></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Histograms showing the frequency of transfer after 4 h of mating for the wild-type (wt), <em>orf38</em> and <em>orf36</em>.1 deletion mutants, as well as their complementation with pOrf38 and pOrf36.1 plasmids producing respectively Orf38 and Orf36.1. Data are represented as mean and SD from (n) independent experiments (white circles). <em>p</em>-value significance from Kruskal-Wallis test corrected with Dunn’s multiple comparisons test is indicated by ns (not significant), * (<em>p</em> = 0.023, pOrf38; <em>p</em> = 0.0461, <em>orf36.1</em>), ** (<em>p</em> = 0.0013). # indicates two replicates with a detection limit of transconjugants below 10<sup>−8</sup>. Donors LY1844, LY3303, LY3697, LY3739, LY3747; Recipient LY945.</p></figcaption></figure><p id="Par20">Orf36.1 is a small protein consisting of 65 amino acids and contains a conserved domain from the Hha superfamily (Table <a href="#MOESM1" class="usa-link">S1</a>), which is found in proteins involved in modulating of bacterial gene expression, and can influence processes such as bacterial virulence, horizontal gene transfer and cell physiology<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Although the precise role of Orf36.1 in the pOXA-48 transfer process remains unknown and further investigations are needed, we hypothesize that it may function as a transcriptional regulator, promoting plasmid transfer by modulating the expression of genes essential for the pOXA-48 conjugation.</p>
<p id="Par21">To validate the role of Orf38 in plasmid transfer, we conducted an in silico analysis to better understand its potential function. Orf38 is a 164 amino acid protein without any identifiable conserved domains (Table <a href="#MOESM1" class="usa-link">S1</a>). In silico analysis did not predict the presence of a signal peptide or transmembrane domain, suggesting that Orf38 is likely localized in the cytoplasm. To gain further insights into its function, we used AlphaFold 3.0 to generate a structural model of Orf38, which showed an average predicted local distance difference test (pLDDT) score of 79.6 ± 12.6%. A subsequent search for homologous structures in the Protein Data Bank using DALI<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> identified DotN, a component of the T4SS in <em>Legionella pneumophila</em><sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>, as the best and only matching structure spanning the entire length of Orf38 model. Our model of Orf38 shared a root mean square deviation (RMSD) of 5.4 with the crystal structure of DotN (PDB 5X1H), indicating significant structural homology (Fig. <a href="#MOESM1" class="usa-link">S4a</a>). Although the precise function of DotN remains uncertain, it is known to form part of a multimeric complex with the DotM and DotL T4SS components, forming the Type IV coupling complex (T4CC), which plays a role in recognizing secreted effectors<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. To further explore whether Orf38 might be part of a similar complex, we modeled its interaction with proteins homologous to DotL and DotM, which correspond to TrbC and TrbA, respectively, in the pOXA-48 plasmid (Table <a href="#MOESM1" class="usa-link">S1</a> and Fig. <a href="#MOESM1" class="usa-link">S4b</a>). The resulting structure showed strong similarity to the cryo-electron microscopy (cryo-EM) structure of the T4CC from <em>L. pneumophila</em> (PDB 6SZ9), with an RMSD of 2.8 Å, suggesting that the T4SS of pOXA-48 may contain a T4CC closely related to that of <em>L. pneumophila</em>. However, the cryo-EM structure did not include the N-terminal regions of DotM and DotL, which were reported as disordered transmembrane regions<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. In contrast, our AlphaFold model predicted these N-terminal regions for TrbA and TrbC, the homologs of DotM and DotL in pOXA-48. We subsequently generated a complete model of the DotN/DotM/DotL complex using AlphaFold, as shown in the right panel of Fig. <a href="#MOESM1" class="usa-link">S4b</a>, which includes these previously unstructured N-terminal regions. The presence of these regions in our model further supports the similarity between the Orf38-TrbA-TrbC and DotN-DotM-DotL complexes. While significant similarities exist between these complexes, key differences were also noted. Notably, TrbC appears to lack the swinging C-terminal tail of DotL after the <em>α</em>-helix16 region, which was shown to be essential for the secretion of a subset of effectors in <em>L. pneumophila</em><sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. This absence could reflect functional divergence between the T4SS of pOXA-48, which is primarily involved in DNA transfer, and the T4SS of <em>L. pneumophila</em>, which is dedicated to secrete over 300 effectors.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Analysis of genes with enrichment in transposon insertions</h3>
<p id="Par22">Transposon insertion can disrupt genes that function as repressors of conjugation, potentially enhancing plasmid transfer efficiency. In our Tn-Seq analysis, we observed an enrichment of transposon insertions in five genes: <em>orf32</em>, <em>ssb</em>, <em>orf33</em>, <em>excA</em>, and <em>repC</em>, with a Log<sub>2</sub>FC above the defined threshold (Fig. <a href="#Fig4" class="usa-link">4</a> and Fig. <a href="#MOESM1" class="usa-link">S5</a>). This suggested that disrupting these genes might increase conjugation efficiency. To test this hypothesis, we constructed deletion mutants for each of these genes and assessed their impact on conjugation efficiency compared to the wild-type plasmid. Surprisingly, conjugation experiments showed that only the deletion of <em>orf32</em> and <em>repC</em> significantly affected the frequency of plasmid transfer, with conjugation rates respectively 1.87-fold and 1.56-fold higher than those observed for the wild-type (Fig. <a href="#Fig6" class="usa-link">6a</a>). The ectopic production of RepC and Orf32 from complementation plasmids restore the efficiency of conjugation to the wild-type level.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Analysis of phenotypes associated with deletion of <em>orf32</em>, <em>ssb</em>, <em>orf33</em>, <em>excA</em>, or <em>repC</em> genes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365149_41467_2025_62404_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dc23/12365149/55f6afdbb6d9/41467_2025_62404_Fig6_HTML.jpg" loading="lazy" id="d33e1380" height="1242" width="781" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Conjugative transfer frequency after 150 min of mating for the wild-type (wt) (LY1844) and mutants, <em>orf32</em> (LY4148) and complemented strain with pOrf32 plasmid (LY4192), <em>ssb</em> (LY4149), <em>orf33</em> (LY4153), <em>excA</em> (LY3260), and <em>repC</em> (LY3259) and complemented strain with pRepC plasmid (LY3606). Recipient LY945. Data are represented as mean and SD from at least six independent experiments (gray circles). Comparisons test is indicated by ns (not significant), *(<em>p</em> = 0.0133), **(<em>p</em> = 0.0095). <strong>b</strong> Plasmid copy number (PCN) for each mutant and <em>repC</em> complemented strain (<em>repC</em> pRepC) from stationary phase culture. The ratio of the mutant PCN to the wild-type PCN is represented. Data are represented as mean and SD from (n) independent experiments (gray circles). Comparisons test is indicated by ns (not significant), *(<em>p</em> = 0.044) <strong>c</strong> Minimum inhibitory concentration (MIC) of imipenem inhibiting the growth of the wild-type (wt), <em>repC</em> mutant, and <em>repC</em> complementated strain (<em>repC</em> pRepC). Data are represented as mean and SD from three independent experiments. Comparisons test is indicated by ns (not significant), *(<em>p</em> = 0.0463). <strong>d</strong> Histogram showing the exclusion levels of wild-type pOXA-48 transfer from donor to the recipients cells carrying wild-type pOXA-48 (wt), <em>excA</em> mutant with or without pExcA complementation plasmid. Data are represented as the mean and SD from (n) independent experiments (white circles). Comparisons test is indicated by ns (not significant), **(<em>p</em> = 0.0081). <strong>e</strong> PCN for <em>excA</em> mutant and complemented strain (<em>excA</em> pExcA) derived from colonies scratched from LB agar plates (solid). The ratio of the mutant PCN to the wild-type PCN are represented. Data are represented as mean and SD from seven independent experiments (gray circles). Comparison test is indicated by ns (not significant), **(<em>p</em> = 0.0011). <strong>f</strong> Conjugative transfer frequency after 60 min of mating for the wild-type (wt) and <em>excA</em> mutant, and complemented strain with pExcA derived from colonies scratched from LB agar plates (solid). Data are represented as mean and SD from (n) independent experiments (gray circles). Comparisons test is indicated by ns (not significant), ****(<em>p</em> &lt; 0.0001). <strong>a</strong><strong>–e</strong>
<em>p-</em>value significance from the Kruskal-Wallis test corrected with Dunn’s multiple comparisons test.</p></figcaption></figure><p id="Par23">We hypothesized that the increased transposon insertions could also be linked to an alteration in plasmid copy number (PCN). To explore this possibility, we measured the PCN of each deletion mutant and compared it to the wild-type plasmid. Our analysis revealed that only the <em>repC</em> deletion mutant exhibited a significant increase in PCN, with a 1.9-fold higher copy number compared to the wild-type, and that ectopic production of RepC from a complementation plasmid restores the PCN to a wild-type level (Fig. <a href="#Fig6" class="usa-link">6b</a>). This increase in PCN was also associated with heightened resistance to imipenem, likely due to the higher number of copies of the <em>bla</em><sub>OXA-48</sub> gene, as previously reported<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup> (Fig. <a href="#Fig6" class="usa-link">6c</a>). These results confirm the predicted role of RepC as a regulator of PCN, certainly by repressing the promoter activity of <em>repA</em>, and associated with the increase transfer efficiency help explain the reason for the increased number of reads observed in the Tn-Seq results.</p>
<p id="Par24">For the remaining genes - <em>orf32</em>, <em>ssb</em>, <em>orf33</em>, and <em>excA</em> - the reason behind the enrichment in Tn-Seq reads is less clear. Bioinformatic analysis suggests that <em>ssb</em> and <em>orf33</em> form an operon with <em>orf32</em>, and their disruption should not affect the expression of the upstream <em>orf32</em> gene. Both <em>orf32</em> and <em>orf33</em> lack conserved domains, leaving their function unknown (Table <a href="#MOESM1" class="usa-link">S1</a>). The <em>ssb</em> gene encodes a single-stranded DNA-binding protein that binds single-stranded DNA. While the precise role of plasmid-encoded Ssb remains unclear, studies on the F plasmid suggest that Ssb facilitates the initial round of plasmid duplication in newly formed transconjugant cells<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Additional studies are required to elucidate the specific roles of these three genes in the process of pOXA-48 plasmid transfer. The <em>excA</em> gene is predicted to function as an exclusion gene, potentially preventing the acquisition of additional plasmids by host cells that already harbor the pOXA-48 plasmid. To validate this hypothesis, we assessed the efficiency of exclusion in an <em>excA</em> deletion mutant by calculating the exclusion index (EI)<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. The EI was determined by comparing the frequency of plasmid acquisition in a recipient strain containing the pOXA-48 plasmid to that of an isogenic plasmid-free strain (Fig. <a href="#Fig6" class="usa-link">6d</a>). For the wild-type pOXA-48 plasmid, the EI was approximately 490, indicating that <em>E. coli</em> cells harboring pOXA-48 are about 490 times less likely to acquire an additional plasmid compared to plasmid-free cells. In the <em>excA</em> deletion mutant, the EI dropped significantly to a value of 2, suggesting that the exclusion mechanism was effectively abolished in the absence of <em>excA</em> (Fig. <a href="#Fig6" class="usa-link">6d</a>). To confirm the role of <em>excA</em> in this exclusion activity, we introduced a complementation plasmid encoding <em>excA</em> into the deletion mutant. The ectopic production of ExcA restored the exclusion phenotype, demonstrating that ExcA is essential for the exclusion function of the pOXA-48 plasmid. These results established ExcA as a key component of the exclusion mechanism, though further investigation is needed to explain the increased number of reads for this gene in the Tn-Seq data, given the observation that approximately 50% of total reads aligned with the <em>excA</em> gene (Fig. <a href="#MOESM1" class="usa-link">S5</a>). The lack of PCN increase in <em>excA</em> mutants is surprising, given that clonal populations of these mutants constantly acquire additional plasmids, which should theoretically result in a higher plasmid number per cell. Like most conjugative plasmids, pOXA-48 conjugates at a very low frequency in liquid media but at a much higher frequency on solid surfaces. We hypothesized that <em>excA</em> mutants grown in liquid media conjugate at low frequency in this condition, leading to the observed low PCN estimated from stationary phase liquid cultures (Fig. <a href="#Fig6" class="usa-link">6b</a>), while colonies grown on solid agar plates would exhibit continual transfer of the <em>excA</em> mutant plasmid, resulting in a significantly higher PCN per cell compared to liquid culture. To test this, we measured the PCN of bacteria directly collected from agar plates. The results confirmed that <em>excA</em> mutants collected from plates had a PCN nearly 10-fold higher than that of the wild type (Fig. <a href="#Fig6" class="usa-link">6e</a>). Furthermore, ectopic production of ExcA restored the PCN to wild-type levels. To further investigate the biological significance of this PCN increase, we assessed the conjugation frequency of the <em>excA</em> mutant when donor cells were collected directly from colonies grown on agar plates. In this condition, the <em>excA</em> mutant displayed a 12-fold increase in conjugation frequency compared to the wild type, and this phenotype was reversed by ectopic expression of <em>excA</em>, restoring conjugation to wild-type levels (Fig. <a href="#Fig6" class="usa-link">6f</a>). These results demonstrate a direct correlation between elevated PCN and enhanced conjugation frequency, consistent with observations made for the <em>repC</em> mutant. Most importantly, they provide an explanation for the enrichment of transposon insertions in <em>excA</em> observed in the Tn-Seq output library. Since donors for conjugation in the Tn-Seq experiment were prepared from colonies grown on solid medium, the elevated PCN and conjugation rate of <em>excA</em> mutants likely conferred a strong selective advantage during the transfer process, leading to their overrepresentation in the transconjugant population.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par25">The transposon insertional analysis and functional assays presented in this study provide an integrated view of the genetic elements involved in the biology of the pOXA-48 plasmid. We identified multiple factors contributing to plasmid stability, including a novel toxin-antitoxin system, components of the replication machinery, elements of the type IV secretion system, and regulatory factors critical for conjugation.</p>
<p id="Par26">Although the IncM pCTX-M3 and IncL pOXA-48 plasmids share 95.6% nucleotide identity, they differ notably in their replication and exclusion systems, placing them in different incompatibility groups<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. The replication genes are highly conserved (Table <a href="#MOESM1" class="usa-link">S1</a>), but differentiation occurs primarily based on the identity of their RNAI sequences, which show significant divergence. The replicon of pOXA-48, composed of RepC, RNAI, RepB, and RepA, closely resembles the components of the R1 plasmid replicon from the IncFII group, which includes CopB, the antisense RNA <em>copA</em>, Tap and RepA<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Our data demonstrate that RepC regulates the plasmid copy number (PCN), as its disruption or deletion leads to increased PCN. Notably, this elevated PCN in <em>repC</em>-deleted mutants correlates with enhanced conjugation rates, resembling the observations made for the R1 plasmid, where increased PCN is also linked to higher conjugation rates<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Interestingly, a recent study on pOXA-48 described the isolation of an evolved plasmid variant from a clinical patient sample, which exhibited a higher PCN associated with increased antimicrobial resistance (AMR)<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. However, in that study, no corresponding increase in conjugation frequency was observed. This discrepancy may be explained by the location of the mutation, which was identified upstream of the <em>repA</em> gene and did not affect <em>repC</em>. These findings suggest that while mutations impacting <em>repC</em> can simultaneously influence PCN and conjugation frequency, mutations outside this region may not exert the same regulatory effect on transfer efficiency.</p>
<p id="Par27">Further differences between the pOXA-48 and pCTX-M3 plasmids are observed in their exclusion systems. The <em>traX</em>, <em>traY</em>, and <em>excA</em> genes of the pOXA-48 and pCTX-M3 plasmids are more divergent from one another than other genes, encoding proteins with significantly lower amino acid identity (Table <a href="#MOESM1" class="usa-link">S1</a>). While the exact origin of this divergence remains unknown, one hypothesis is that a recombination event occurred during the evolutionary history of these plasmids. It has been proposed that both pOXA-48 and pCTX-M3 were derived from the ancestral pEL60 plasmid<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Interestingly, pCTX-M3 harbors <em>traX</em>, <em>traY</em>, and <em>excA</em> genes that are nearly identical to those of pEL60, whereas the corresponding genes in pOXA-48 have substantially diverged. This suggests that a gene exchange event may have introduced a variant region in the pOXA-48 lineage, replacing the original gene cluster found in the ancestral plasmid. In this study, we identified ExcA as the entry exclusion protein for the pOXA-48 plasmid. Although the precise mechanism of plasmid exclusion remains unclear, exclusion in IncI plasmids is driven by the recognition of TraY by ExcA<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, similar to the TraG protein of plasmid F, which interacts with the exclusion protein TraS<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Interestingly, the region of TraY recognized by ExcA varies between plasmids; for instance, the central region is targeted in plasmid R64, whereas the C-terminal region is recognized in plasmid R621a<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. The divergence in the <em>excA</em> region between pCTX-M3 and pOXA-48 thus reflects the specificity of exclusion mechanisms, and these plasmids do not mutually exclude one another<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. The role of <em>traX</em> in this exclusion process remains unknown, but it is noteworthy that <em>traX</em> is part of the recombined region and appears genetically linked to <em>traY</em> and <em>excA</em>. While initial conjugation assays performed in liquid medium showed no impact of <em>excA</em> deletion on pOXA-48 transfer, experiments using donor cells from solid medium revealed a high increase in transfer frequency. This contrasts with the pCTX-M3 plasmid, where <em>excA</em> deletion has been shown to affect plasmid transfer<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p>
<p id="Par28">Importantly, our study reveals additional factors involved in plasmid maintenance and stability. Beyond the well-characterized Par partitioning system, which ensures stable inheritance of the plasmid, we identified two other genes, <em>korC</em> and <em>orf20</em>, whose deletion was detrimental to cell growth under selective pressure, as well as a novel type I toxin-antitoxin system. The regulatory role of KorC in pOXA-48 appears similar to that in the RP4 plasmid, where the absence of Kor functions leads to a host-lethal phenotype<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. In RP4, KorC represses the <em>klcA</em> gene, which encodes a putative anti-restriction protein, with a KorC binding site found in the <em>klcA</em> promoter. Interestingly, pOXA-48 contains a <em>klcA</em> homolog downstream of <em>korC</em>, near the origin of transfer (Fig. <a href="#Fig1" class="usa-link">1</a>). However, no KorC binding site was predicted upstream of this gene, and only one site was found, located in the predicted <em>korC</em> promoter upstream of <em>orf24</em>, which may be in the same operon as <em>korC</em> (Fig. <a href="#Fig2" class="usa-link">2c</a>). This suggests that KorC may autoregulate its own expression, as seen in other KorC systems. Further experiments are needed to determine the role of KorC in pOXA-48 and whether it regulates genes with toxic potential or modulates stress responses, thereby contributing to plasmid stability and host viability. Similarly, Orf20 also plays a critical role in plasmid maintenance. Our findings show that a plasmid deleted of <em>orf20</em> could be maintained when <em>orf20</em> was provided in <em>trans</em>, indicating an essential role in plasmid stability. This phenotype resembles that observed in the absence of KorC, which results in plasmid loss. Sequence analysis predicts that Orf20 is an XRE-like transcriptional regulator, suggesting a potential regulatory function similar to KorC. Further studies are required to elucidate the precise function of Orf20 in pOXA-48 physiology.</p>
<p id="Par29">The essentiality of <em>orf20</em> and <em>korC</em> was also reported by Calvo-Villamañán et al., who used a plasmid-wide CRISPRi screen to investigate pOXA-48 fitness effects in various clinical strains of enterobacteria<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. While our experiments were conducted in the laboratory strain <em>E. coli</em> K12 MG1655, their study included both clinical isolates and the laboratory strain <em>E. coli</em> J53. They showed that silencing <em>pemI</em>, <em>pri</em> and <em>H-NS</em> (<em>orf36</em>.2) in J53 had minimal effects, consistent with our findings in MG1655, where no significant decrease in transposon insertions was observed for these genes. In contrast, expression of these genes is essential in clinical isolates, highlighting the importance of plasmid-host interactions and supporting the idea that gene essentiality is context-dependent, influenced by both genetic and physiological characteristics of the host strain.</p>
<p id="Par30">In this study, we also identified several genes critical for pOXA-48 transfer, including components of the relaxosome and T4SS. However, the exact roles of some components, such as TraL, TraQ, TraR, TraW, TraX, and TrbB (Table <a href="#MOESM1" class="usa-link">S1</a>), remain to be fully elucidated. Interestingly, we identified Orf38, previously uncharacterized as a T4SS component, as an essential factor for plasmid transfer. Structural modeling suggests that Orf38 forms a type IV coupling complex (T4CC) with the T4CP TrbC and TrbA. Such a complex is absent in “minimized” T4SS-A systems, like those found in R388 or pKM101 plasmids, but is present in “expanded” T4SS-B systems, such as the one in <em>L. pneumophila</em><sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. In this bacteria, the T4CC acts as a platform for recruiting and secreting effectors. The role of such a T4CC in conjugative T4SS dedicated to DNA transfer remains unclear. While it is known that, in addition to DNA, proteins such as relaxase can be transferred through T4SS by covalent attachment to the DNA, other plasmid-encoded proteins may also be co-transferred in a DNA-dependent manner<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, or independently, as seen with the primase Sog of the IncI ColIb plasmid<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>.</p>
<p id="Par31">In recent years, increasing attention has been given to clinically relevant conjugative plasmids from groups such as IncI1<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, IncA/C<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, and more recently, IncX<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, which are implicated in the dissemination of resistance to expanded-spectrum beta-lactams, carbapenems, and colistin. These studies have uncovered numerous molecular factors and highlighted that the biology of conjugative plasmids remains an extensive field for exploration. Our work contributes significantly to advancing our understanding of the molecular mechanisms underlying pOXA-48 transfer, identifying critical genetic elements that warrant further investigation.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Bacterial strains, plasmids and growth</h3>
<p id="Par32">Bacterial strains are listed in Table <a href="#MOESM1" class="usa-link">S2</a>, plasmids in Table <a href="#MOESM1" class="usa-link">S3</a>, and oligonucleotides in Table <a href="#MOESM1" class="usa-link">S4</a>. Gene deletion on the pOXA-48 plasmid used λRed recombination<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Modified pOXA-48 plasmids were transferred to the background strain <em>E. coli</em> K12 MG1655 by conjugation. The kan or Cm resistance cassette were removed using the site-specific recombination of the Flp recombinase from pCP20 plasmid<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Plasmid cloning was performed by Gibson Assembly<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Strains and plasmids were verified by Sanger sequencing (Eurofins Genomics and Microsynth).</p>
<p id="Par33">Cells were grown at 37 °C in Luria-Bertani (LB) or Rich Defined Medium (RDM) broth. When appropriate, supplements were used in the following concentrations; Ampicillin (Ap) 100 µg/ml, Chloramphenicol (Cm) 20 µg/ml, Kanamycin (Kan) 50 µg/ml, Streptomycin (St) 20 µg/ml, Erythromycin (Erm) 100 µg/ml, Tetracycline (Tc) 10 µg/ml and diaminopimelate (DAP) 300 μM.</p></section><section id="Sec11"><h3 class="pmc_sec_title">pOXA-48 genome sequencing</h3>
<p id="Par34">Overnight cultures in LB medium of LY1844 strain was pelleted at 15000 g for 5 min. Supernatant was discarded, and pellet was resuspended in 525 µl of lysis buffer (CleanNA, Waddinxveen, The Netherlands) for bacterial DNA extraction. Semi-automated DNA extraction on a KingFisher platform (Thermo Fisher Scientific, Waltham, MA, USA) was performed with the Clean Pathogen kit (Clean NA) according to manufacturer’s recommendations. Briefly, a mix of 73 µl of proteinase K and proteinase K solution were added to each sample prior vortexing and incubation for 15 min at 70 °C. Samples were centrifuged at 17,500 g for 10 min and 300 µl of supernatant were added to 620 µl of beads mix in a deep well plate. Two plates of Wash Buffer 1, 1 plate of Wash Buffer 2 and 1 plate of elution buffer were prepared. All plates were loaded on the KingFisher device and the extraction program was started. Eluted DNA (100 µl) were stored at −20 °C until use or directly processed to sequencing.</p>
<p id="Par35">Bacterial DNA was sequenced using the Rapid Barcoding Kit SQK_RBK114.96, FLO-MIN114 flow cells, and a GridION (Oxford Nanopore Technologies, OX4 4DQ, UK) following the manufacturer’s procedure. Four hundred ng of QC bacterial DNA were added to nuclease-free water to reach a volume of 20 µl per sample. Two µl of individual barcode were added to each sample before incubation at 30 °C for 2 min and 80 °C for 2 min. The pooled DNA library was purified with AMPure beads at a 2:1 DNA: beads ratio. One µl of Rapid Adapter solution was added to 800 ng of library to reach a maximum volume of 12 µl before incubation at RT for 5 min. of Sequencing Buffer (37.5 µl) and of Library Beads (25.5 µl) were added to the DNA library, and the mix was loaded on the flow cell and sequenced in “super accurate” mode. Generated data were basecalled with Dorado 7.3.11 and analyzed with in house bioinformatics pipeline. Data are available at BioProject PRJEB88520.</p></section><section id="Sec12"><h3 class="pmc_sec_title">High-density himar-1 transposon mutagenesis</h3>
<p id="Par36">Overnight cultures in LB medium of donor <em>E. coli</em> MFD<em>pir</em>+ carrying pSAM_Ec and recipient <em>E. coli</em> MG1655 carrying pOXA-48 were mixed at a ratio 1:1 in presence of 300 mM DAP, and incubated on filters (MF Membrane Filters-0.45 μm HA-Ref HAWP02500 Millipore) placed on LB-agar plates 4 h at 37 °C. The mating mixture was then resuspended in LB medium and spread onto LB-agar plates supplemented with Kan and Ap to select clones carrying the Himar-1 transposon and the pOXA-48 plasmid, respectively. After overnight incubation at 37 °C, about 10<sup>6</sup> resistant colonies, designated as the input library, were collected, resuspended in LB medium and frozen in 40% glycerol at −80 °C. 1 ml of the frozen input library was washed and used as the donor in a mating of ratio 1:1 with an <em>E. coli</em> MG1655 recipient strain. The mating mix was spotted on filters, incubated for 4 h at 37 °C, then resuspended LB medium and entirely spread onto LB-agar plates supplemented with Kan and Cm to select transferred plasmids carrying transposon. After overnight incubation at 37 °C, approximately 2,000,000 transconjugant colonies were collected, providing an estimated ~715-fold theoretical coverage of the 2788 potential TA insertion sites in the pOXA-48 plasmid. This ensured robust representation of the mutant library in the output pool, despite the relatively low conjugation frequency (~0.3%). The resulting output library was resuspended in LB medium and frozen in 40% glycerol at −80 °C.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Preparation of Tn-Seq sequencing library</h3>
<p id="Par37">Library DNA preparation and sequencing procedures were adapted from Royet et al.<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Genomic DNA (gDNA) from the mutant library was extracted using the Promega Wizard Genomic DNA Purification kit. Then, 30 µg of gDNA was digested with 30 U of MmeI in a final volume of 1.2 ml at 37 °C for 1.5 h. Digestions were heat-inactivated for 20 min at 65 °C and purified with a ratio of 0.45X of the magnetic beads Mag-Bind Total Pure NGS (Omega Biotek), allowing the removal of fragments smaller than 1000 bp. 2 µg of digested and purified DNA was ligated to double-stranded barcoded adapters at 0.55 µM. The ligation was performed at 16 °C overnight using 800 U of T4 DNA ligase in a final volume of 40 µL. Ligated DNA were purified using a ratio of 0.6X of the magnetic beads Mag-Bind Total Pure NGS (Omega Biotek) to eliminate the fragments smaller than 500 bp. To amplify the DNA adjacent to the transposon, a total of six 18-cycle PCRs were conducted for each sample. Each of the PCR mixtures comprised 40 ng of DNA, 1 U of Q5 DNA polymerase (New England Biolabs), 1X Q5 buffer, 0.2 mM of dNTPs, and 0.4 µM of the L1204-I1 and L1204-I2 primers. The six PCR products per sample were then combined, concentrated to a volume of 50 µl using a vacuum concentrator and loaded onto a 2% (w/v) agarose gel. The 125 bp bands were excised and purified using a QIAquick Gel Extraction kit (Qiagen). All gel electrophoresis and gel extraction steps were carried out on a blue-light transilluminator to minimize DNA damage, with DNA stained using GelGreen (Ozyme). Finally, DNA samples were dialyzed for 4 h on 0.025 µm nitrocellulose membranes (Milipore). 50 nM of each DNA library was then sent to MGX (CNRS Sequencing Service) for quality control and high-throughput sequencing on an Illumina Novaseq 6000 instrument using a single lane of an SP flow cell in single-read mode, producing 100 nt reads. Reads were demultiplexed by MGX using cutadapt<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. This involved trimming the P7 adapter at the 3’ end of the reads, followed by demultiplexing based on the index located at the 3’ end and then based on the barcode at the 5’ end. The data were analyzed using the TnSeek pipeline (<a href="https://hal.science/hal-03827389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://hal.science/hal-03827389</a>), available online (<a href="https://bioinfo.cristal.univ-lille.fr/tnseek/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bioinfo.cristal.univ-lille.fr/tnseek/</a>) (supplementary Data <a href="#MOESM4" class="usa-link">1</a>; data are available at BioProject PRJNA1249410).</p></section><section id="Sec14"><h3 class="pmc_sec_title">Stability assay</h3>
<p id="Par38">Overnight cultures were normalized to an OD<sub>600</sub> of 2, then diluted 1:1000 in 5 ml of LB medium and grown at 37 °C with shaking at 160 rpm. After 8 h, the culture was diluted again 1:1000 in 5 ml of fresh LB medium and grown for an additional 16 h under the same conditions. This process was repeated for two more consecutive days. To monitor plasmid loss, samples were taken at each dilution step, and cells were plated on non-selective LB-agar plates. After overnight incubation at 37 °C, one hundred clones were streak on non-selective or Ap- supplemented LB-agar plates to determine the presence of the pOXA-48 plasmid. Plasmid stability was calculated as the percentage of colonies resistant to Ap relative to the total number of colonies tested.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Live-cell microscopy experiments</h3>
<p id="Par39">Overnight cultures of recipient and donor cells grown separately in LB at 37 °C were diluted to OD<sub>600</sub> ∼ 0.05 in RDM and grown further at 37 °C to OD<sub>600</sub>∼ 0.5. Then, 25 μl of donor and 75 μl recipient cultures were mixed and 50 µl dropped on a filter (MF Membrane Filters-0.45 μm HA-Ref HAWP02500 Millipore) placed on LB-agar medium. Conjugation mixes were incubated for 4 h, then resuspended in 1 ml RDM and loaded to a quasi-2D microfluidic chamber (model B04A, ONIX, CellASIC®) with pressure 5 psi for 1 min. The temperature of the microfluidic chamber was maintained at 37 °C and cells were imaged every 10 min for 3 h with the flow of fresh medium supplemented with Ap.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Conjugation assays</h3>
<section id="Sec17"><h4 class="pmc_sec_title">Liquid condition</h4>
<p id="Par40">Overnight cultures grown in LB medium of recipient and donor cells were diluted to an OD<sub>600</sub> of 0.05 and grown further at 160 rpm to reach an OD<sub>600</sub> 0.7–0.9. 25 µl of donor and 75 µl of recipient cultures were mixed into an Eppendorf tube and incubated for 90 min at 37 °C. 1 ml of LB was added gently, and the tubes were incubated again for 90 min at 37 °C. Conjugation mixes were vortexed, serial diluted, and plated on LB-agar supplemented with antibiotics to select recipients and transconjugants.</p></section><section id="Sec18"><h4 class="pmc_sec_title">Filter condition</h4>
<p id="Par41">Overnight cultures grown in LB medium of recipient and donor cells were normalized to an OD<sub>600</sub> = 2. 100 µl of donor and 100 µl recipient were mixed and 100 µl dropped on a filter (MF Membrane Filters-0.45 μm HA-Ref HAWP02500 Millipore) placed on LB-agar medium. Conjugation mix were incubated for 150 min at 37 °C, then resuspended into 1 ml of LB medium, serial diluted, and plated on LB-agar supplemented with antibiotics to select recipients and transconjugants.</p></section></section><section id="Sec19"><h3 class="pmc_sec_title">
<em>dqlB</em>-<em>agrB</em> killing and rescue assays</h3>
<p id="Par42">For killing/rescue spot assays, cells were grown at 37 °C to OD<sub>600</sub> ∼0.4–0.5 in LB medium supplemented with 0.4% glucose, Cm and Kan. Cultures were then serially diluted and spotted on LB-agar plates supplemented with the same antibiotics and with either 0.4% glucose or 0.2% arabinose, and incubated overnight at 30 °C. For imaging experiments, cells were grown at 37 °C in LB medium supplemented with Cm. At OD<sub>600</sub> ∼0.4–0.5, 0.2% arabinose was added, and cells were imaged after 240 min of induction using a Ti-E microscope (Nikon) equipped with a 100x phase contrast objective (Plan apo λ 1.45, Nikon). Conjugation of TAPs was performed as previously described in liquid condition<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Viable counts for transconjugants were determined by plating cells on LB-agar medium supplemented with St and Kan.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Estimation of plasmid copy number by digital PCR quantification (dPCR)</h3>
<p id="Par43">Quantifications of DNA loci: <em>terC</em> (<em>E. coli</em> chromosome) and <em>traN</em> (pOXA-48) were performed as described in<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> using multiplex dPCR (Stilla Technologies). dPCR was performed directly on cell lysate: 1 ml of overnight culture (liquid) or a scratch of colonies (solid) was washed twice with 1 ml PBS, pellets were then frozen at −20 °C and resuspended in 200 μl PBS, samples were then boiled for 10 min. Samples were centrifuged for 10 min, supernatant was recovered, and DNA content was measured using Qubit device (ThermoFisher). PCR reactions were performed with 0.1 ng of DNA using the naica® multiplex PCR MIX on a Sapphire or Ruby chip (Stilla Technologies). dPCR was conducted on a Naica Geode, and Image acquisition on the Naica Prism6 reader. Images were analyzed by Crystal Miner software (Stilla Technologies). The dPCR run was performed using the following steps: droplet partition (40 °C, atmospheric pressure AP to + 950 mbar, 12 min), initial denaturation (95 °C, +950 mbar, for 2 min), followed by 45 cycles at (95 °C for 10 s and 58 °C for 30 s), droplet release (down 25 °C, down to AP, 33 min).</p></section><section id="Sec21"><h3 class="pmc_sec_title">Exclusion index assays</h3>
<p id="Par44">Conjugation assays were performed between a donor strain carrying the wild-type pOXA-48 and a recipient strain without a plasmid or harboring the plasmid to be tested. The exclusion index was calculated by comparing the frequency of plasmid acquisition by a recipient strain to the frequency obtained with the plasmid-free recipient strain<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Bioinformatic analysis</h3>
<p id="Par45">A dataset of 15,984 plasmid sequences with defined incompatibility groups was subsampled from the PLSDB database<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Homologs of the <em>dqlB</em> nucleotide sequences were identified using the HMMER web server<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>, starting with a seed set of six known <em>dqlB</em><sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> and <em>dqlB</em> of pOXA-48. The distribution of <em>dqlB</em> homologs across plasmid incompatibility groups was determined using PLSDB metadata. Identical homologous sequences were clustered into 178 groups before alignment with the L-INS-I method of MAFFT<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Multiples sequence alignments were trimmed using BMGE<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>, and maximum likelihood phylogenetic trees were constructed with IQ-TREE<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. 1000 replicates ultrafast bootstrap alignments were performed. The resulting tree was visualized using TreeViewer<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>, with the distribution of incompatibility group indicated for each leaf.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Alphafold analysis</h3>
<p id="Par46">To model proteins, full-length amino acid sequences were uploaded to the Alphafold 3.0 web server<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. Based on the published structures of DotN<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> and the T4CC<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> of <em>L. pneumophila</em>, Orf38 was modeled in the presence of a Zn ion as co-factor. The protein structures were aligned and visualized using PyMol (v1.20).</p></section><section id="Sec24"><h3 class="pmc_sec_title">Statistics &amp; Reproducibility</h3>
<p id="Par47">No statistical method was used to predetermine sample size. No data were excluded from the analyses.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par48">Further information on research design is available in the <a href="#MOESM5" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec26"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62404_MOESM2_ESM.pdf</a><sup> (56KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM3_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (3.6MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (161.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (73.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (217KB, pdf) </sup>
</div></div></section></section><section id="Sec27"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (29.7KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by the ANR through DeCa-P [grant number ANR-22-CE35-0017 to S.B., E.G., Y.M. and J.B.] and ETA-2 [grant number ANR-22-CE12-0032 to C.L. and N.F.], the FRM (Foundation for Medical Research) [grant number EQU202103012587 to C.L.], ANR JCJC [grant number ANR-19CE12-0001 to M.E.V.] and the University of Lyon through funding to Y.B.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>S.B. Conceived, designed and supervised the execution of the study; Y.B., N.F., Y.M., J.B., M.E.V., A.D.B., S.B. and E.G. performed and analyzed the data. J.D., B.I., and P.B. sequenced the pOXA-48 plasmid. S.B., C.L., E.G. and M.E.V. provided funding.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par49"><em>Nature Communications</em> thanks Willem van Schaik, and the other, anonymous, reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data to understand and assess the conclusions of this research are available in the main text, Supplementary Information and Supplementary Data <a href="#MOESM4" class="usa-link">1</a>. Source data are provided with this paper in the Source Data file. <a href="#Sec27" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par50">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec28"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62404-7.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Poirel, L., Bonnin, R. A. &amp; Nordmann, P. Genetic features of the widespread plasmid coding for the carbapenemase OXA-48. <em>Antimicrob. Agents Chemother.</em><strong>56</strong>, 559–562 (2012).
</cite> [<a href="https://doi.org/10.1128/AAC.05289-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3256075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22083465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poirel,%20L.,%20Bonnin,%20R.%20A.%20&amp;%20Nordmann,%20P.%20Genetic%20features%20of%20the%20widespread%20plasmid%20coding%20for%20the%20carbapenemase%20OXA-48.%20Antimicrob.%20Agents%20Chemother.56,%20559%E2%80%93562%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Alonso-del Valle, A. et al. Variability of plasmid fitness effects contributes to plasmid persistence in bacterial communities. <em>Nat. Commun.</em><strong>12</strong>, 2653 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-22849-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8113577/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33976161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alonso-del%20Valle,%20A.%20et%20al.%20Variability%20of%20plasmid%20fitness%20effects%20contributes%20to%20plasmid%20persistence%20in%20bacterial%20communities.%20Nat.%20Commun.12,%202653%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Alonso-del Valle, A. et al. Antimicrobial resistance level and conjugation permissiveness shape plasmid distribution in clinical enterobacteria. <em>Proc. Natl Acad. Sci. USA</em><strong>120</strong>, e2314135120 (2023).
</cite> [<a href="https://doi.org/10.1073/pnas.2314135120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10741383/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38096417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alonso-del%20Valle,%20A.%20et%20al.%20Antimicrobial%20resistance%20level%20and%20conjugation%20permissiveness%20shape%20plasmid%20distribution%20in%20clinical%20enterobacteria.%20Proc.%20Natl%20Acad.%20Sci.%20USA120,%20e2314135120%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Fernández-Calvet, A. et al. The distribution of fitness effects of plasmid pOXA-48 in clinical enterobacteria: this article is part of the Microbial Evolution collection. Microbiology <strong>169</strong>, 001369 (2023).
</cite> [<a href="https://doi.org/10.1099/mic.0.001369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10433420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37505800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fern%C3%A1ndez-Calvet,%20A.%20et%20al.%20The%20distribution%20of%20fitness%20effects%20of%20plasmid%20pOXA-48%20in%20clinical%20enterobacteria:%20this%20article%20is%20part%20of%20the%20Microbial%20Evolution%20collection.%20Microbiology%20169,%20001369%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>DelaFuente, J. et al. Within-patient evolution of plasmid-mediated antimicrobial resistance. <em>Nat. Ecol. Evol.</em><strong>6</strong>, 1980–1991 (2022).
</cite> [<a href="https://doi.org/10.1038/s41559-022-01908-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7613874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36303001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?DelaFuente,%20J.%20et%20al.%20Within-patient%20evolution%20of%20plasmid-mediated%20antimicrobial%20resistance.%20Nat.%20Ecol.%20Evol.6,%201980%E2%80%931991%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>DelaFuente, J., Diaz-Colunga, J., Sanchez, A. &amp; San Millan, A. Global epistasis in plasmid-mediated antimicrobial resistance. <em>Mol. Syst. Biol.</em><strong>20</strong>, 311–320 (2024).
</cite> [<a href="https://doi.org/10.1038/s44320-024-00012-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10987494/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38409539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?DelaFuente,%20J.,%20Diaz-Colunga,%20J.,%20Sanchez,%20A.%20&amp;%20San%20Millan,%20A.%20Global%20epistasis%20in%20plasmid-mediated%20antimicrobial%20resistance.%20Mol.%20Syst.%20Biol.20,%20311%E2%80%93320%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Zongo, P. D. et al. An antiplasmid system drives antibiotic resistance gene integration in carbapenemase-producing Escherichia coli lineages. <em>Nat. Commun.</em><strong>15</strong>, 4093 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-48219-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11096173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38750030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zongo,%20P.%20D.%20et%20al.%20An%20antiplasmid%20system%20drives%20antibiotic%20resistance%20gene%20integration%20in%20carbapenemase-producing%20Escherichia%20coli%20lineages.%20Nat.%20Commun.15,%204093%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Virolle, C., Goldlust, K., Djermoun, S., Bigot, S. &amp; Lesterlin, C. Plasmid transfer by conjugation in gram-negative bacteria: from the cellular to the community level. <em>Genes</em><strong>11</strong>, E1239 (2020).</cite> [<a href="https://doi.org/10.3390/genes11111239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7690428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33105635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Virolle,%20C.,%20Goldlust,%20K.,%20Djermoun,%20S.,%20Bigot,%20S.%20&amp;%20Lesterlin,%20C.%20Plasmid%20transfer%20by%20conjugation%20in%20gram-negative%20bacteria:%20from%20the%20cellular%20to%20the%20community%20level.%20Genes11,%20E1239%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Gibert, M., Juárez, A., Zechner, E. L., Madrid, C. &amp; Balsalobre, C. TrhR, TrhY and HtdA, a novel regulatory circuit that modulates conjugation of the IncHI plasmids. <em>Mol. Microbiol.</em><strong>94</strong>, 1146–1161 (2014).</cite> [<a href="https://doi.org/10.1111/mmi.12823" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25302406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gibert,%20M.,%20Ju%C3%A1rez,%20A.,%20Zechner,%20E.%20L.,%20Madrid,%20C.%20&amp;%20Balsalobre,%20C.%20TrhR,%20TrhY%20and%20HtdA,%20a%20novel%20regulatory%20circuit%20that%20modulates%20conjugation%20of%20the%20IncHI%20plasmids.%20Mol.%20Microbiol.94,%201146%E2%80%931161%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Carraro, N., Matteau, D., Luo, P., Rodrigue, S. &amp; Burrus, V. The master activator of IncA/C conjugative plasmids stimulates genomic islands and multidrug resistance dissemination. <em>PLoS Genet.</em><strong>10</strong>, e1004714 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004714" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4207636/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25340549/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carraro,%20N.,%20Matteau,%20D.,%20Luo,%20P.,%20Rodrigue,%20S.%20&amp;%20Burrus,%20V.%20The%20master%20activator%20of%20IncA/C%20conjugative%20plasmids%20stimulates%20genomic%20islands%20and%20multidrug%20resistance%20dissemination.%20PLoS%20Genet.10,%20e1004714%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Hancock, S. J. et al. Comprehensive analysis of IncC plasmid conjugation identifies a crucial role for the transcriptional regulator AcaB. <em>Nat. Microbiol.</em><strong>5</strong>, 1340–1348 (2020).
</cite> [<a href="https://doi.org/10.1038/s41564-020-0775-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32807890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hancock,%20S.%20J.%20et%20al.%20Comprehensive%20analysis%20of%20IncC%20plasmid%20conjugation%20identifies%20a%20crucial%20role%20for%20the%20transcriptional%20regulator%20AcaB.%20Nat.%20Microbiol.5,%201340%E2%80%931348%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Yi, L. et al. PixR, a novel activator of conjugative transfer of IncX4 resistance plasmids, mitigates the fitness cost of mcr-1 carriage in Escherichia coli. mBio <strong>13</strong>, e0320921 (2022).
</cite> [<a href="https://doi.org/10.1128/mbio.03209-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8725589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35089067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yi,%20L.%20et%20al.%20PixR,%20a%20novel%20activator%20of%20conjugative%20transfer%20of%20IncX4%20resistance%20plasmids,%20mitigates%20the%20fitness%20cost%20of%20mcr-1%20carriage%20in%20Escherichia%20coli.%20mBio%2013,%20e0320921%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Ma, T. et al. VirBR, a transcription regulator, promotes IncX3 plasmid transmission and persistence of blaNDM-5 in zoonotic bacteria. <em>Nat. Commun.</em><strong>15</strong>, 5498 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-49800-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11214620/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38944647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20T.%20et%20al.%20VirBR,%20a%20transcription%20regulator,%20promotes%20IncX3%20plasmid%20transmission%20and%20persistence%20of%20blaNDM-5%20in%20zoonotic%20bacteria.%20Nat.%20Commun.15,%205498%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Liu, B. et al. Impact of plasmid-encoded H-NS–like protein on blaNDM-1-Bearing IncX3 Plasmid in Escherichia coli. <em> J. Infect. Dis.</em><strong>221</strong>, S229–S236 (2020).
</cite> [<a href="https://doi.org/10.1093/infdis/jiz567" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32176784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20B.%20et%20al.%20Impact%20of%20plasmid-encoded%20H-NS%E2%80%93like%20protein%20on%20blaNDM-1-Bearing%20IncX3%20Plasmid%20in%20Escherichia%20coli.%20J.%20Infect.%20Dis.221,%20S229%E2%80%93S236%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Gołębiewski, M. et al. Complete nucleotide sequence of the pCTX-M3 plasmid and its involvement in spread of the extended-spectrum β-lactamase gene <em>bla</em><sub>CTX-M-3</sub>. <em>Antimicrob. Agents Chemother.</em><strong>51</strong>, 3789–3795 (2007).
</cite> [<a href="https://doi.org/10.1128/AAC.00457-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2151408/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17698626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Go%C5%82%C4%99biewski,%20M.%20et%20al.%20Complete%20nucleotide%20sequence%20of%20the%20pCTX-M3%20plasmid%20and%20its%20involvement%20in%20spread%20of%20the%20extended-spectrum%20%CE%B2-lactamase%20gene%20blaCTX-M-3.%20Antimicrob.%20Agents%20Chemother.51,%203789%E2%80%933795%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Reuter, A. et al. Targeted-antibacterial-plasmids (TAPs) combining conjugation and CRISPR/Cas systems achieve strain-specific antibacterial activity. <em>Nucleic Acids Res.</em><strong>49</strong>, 3584–3598 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8034655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33660775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reuter,%20A.%20et%20al.%20Targeted-antibacterial-plasmids%20(TAPs)%20combining%20conjugation%20and%20CRISPR/Cas%20systems%20achieve%20strain-specific%20antibacterial%20activity.%20Nucleic%20Acids%20Res.49,%203584%E2%80%933598%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Athanasopoulos, V., Praszkier, J. &amp; Pittard, A. J. The replication of an IncL/M plasmid is subject to antisense control. <em>J. Bacteriol.</em><strong>177</strong>, 4730–4741 (1995).
</cite> [<a href="https://doi.org/10.1128/jb.177.16.4730-4741.1995" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC177239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7543895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Athanasopoulos,%20V.,%20Praszkier,%20J.%20&amp;%20Pittard,%20A.%20J.%20The%20replication%20of%20an%20IncL/M%20plasmid%20is%20subject%20to%20antisense%20control.%20J.%20Bacteriol.177,%204730%E2%80%934741%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Athanasopoulos, V., Praszkier, J. &amp; Pittard, A. J. Analysis of elements involved in pseudoknot-dependent expression and regulation of the repA gene of an IncL/M plasmid. <em>J. Bacteriol.</em><strong>181</strong>, 1811–1819 (1999).
</cite> [<a href="https://doi.org/10.1128/jb.181.6.1811-1819.1999" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC93579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10074073/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Athanasopoulos,%20V.,%20Praszkier,%20J.%20&amp;%20Pittard,%20A.%20J.%20Analysis%20of%20elements%20involved%20in%20pseudoknot-dependent%20expression%20and%20regulation%20of%20the%20repA%20gene%20of%20an%20IncL/M%20plasmid.%20J.%20Bacteriol.181,%201811%E2%80%931819%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Calvo-Villamanan, A., Sastre-Dominguez, J., Barrera-Martin, A., Costas, C. &amp; San Millan, A. Dissecting pOXA-48 fitness effects in clinical enterobacteria using plasmid-wide CRISPRi screens. Preprint at <em>bioRxiv</em>10.1101/2025.01.23.633114 (2025).</cite> [<a href="https://doi.org/10.1038/s41467-025-63082-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12361523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40825783/" class="usa-link">PubMed</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Ludwiczak, M. et al. Global transcriptional regulator KorC coordinates expression of three backbone modules of the broad-host-range RA3 plasmid from IncU incompatibility group. <em>Plasmid</em><strong>70</strong>, 131–145 (2013).
</cite> [<a href="https://doi.org/10.1016/j.plasmid.2013.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23583562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ludwiczak,%20M.%20et%20al.%20Global%20transcriptional%20regulator%20KorC%20coordinates%20expression%20of%20three%20backbone%20modules%20of%20the%20broad-host-range%20RA3%20plasmid%20from%20IncU%20incompatibility%20group.%20Plasmid70,%20131%E2%80%93145%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Mierzejewska, J., Kulińska, A. &amp; Jagura-Burdzy, G. Functional analysis of replication and stability regions of broad-host-range conjugative plasmid CTX-M3 from the IncL/M incompatibility group. <em>Plasmid</em><strong>57</strong>, 95–107 (2007).
</cite> [<a href="https://doi.org/10.1016/j.plasmid.2006.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17087993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mierzejewska,%20J.,%20Kuli%C5%84ska,%20A.%20&amp;%20Jagura-Burdzy,%20G.%20Functional%20analysis%20of%20replication%20and%20stability%20regions%20of%20broad-host-range%20conjugative%20plasmid%20CTX-M3%20from%20the%20IncL/M%20incompatibility%20group.%20Plasmid57,%2095%E2%80%93107%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Kristiansen, K. I., Weel-Sneve, R., Booth, J. A. &amp; Bjørås, M. Mutually exclusive RNA secondary structures regulate translation initiation of DinQ in <em>Escherichia coli</em>. <em>RNA</em><strong>22</strong>, 1739–1749 (2016).
</cite> [<a href="https://doi.org/10.1261/rna.058461.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5066626/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27651528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kristiansen,%20K.%20I.,%20Weel-Sneve,%20R.,%20Booth,%20J.%20A.%20&amp;%20Bj%C3%B8r%C3%A5s,%20M.%20Mutually%20exclusive%20RNA%20secondary%20structures%20regulate%20translation%20initiation%20of%20DinQ%20in%20Escherichia%20coli.%20RNA22,%201739%E2%80%931749%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Weel-Sneve, R. et al. Single transmembrane peptide DinQ modulates membrane-dependent activities. <em>PLoS Genet.</em><strong>9</strong>, e1003260 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1003260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3567139/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23408903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weel-Sneve,%20R.%20et%20al.%20Single%20transmembrane%20peptide%20DinQ%20modulates%20membrane-dependent%20activities.%20PLoS%20Genet.9,%20e1003260%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Fraikin, N. &amp; Van Melderen, L. Single-cell evidence for plasmid addiction mediated by toxin-antitoxin systems. <em>Nucleic Acids Res.</em><strong>52</strong>, 1847–1859 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkae018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10899753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38224456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fraikin,%20N.%20&amp;%20Van%20Melderen,%20L.%20Single-cell%20evidence%20for%20plasmid%20addiction%20mediated%20by%20toxin-antitoxin%20systems.%20Nucleic%20Acids%20Res.52,%201847%E2%80%931859%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Gerdes, K., Rasmussen, P. B. &amp; Molin, S. Unique type of plasmid maintenance function: postsegregational killing of plasmid-free cells. <em>Proc. Natl Acad. Sci. USA</em><strong>83</strong>, 3116–3120 (1986).
</cite> [<a href="https://doi.org/10.1073/pnas.83.10.3116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC323463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3517851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gerdes,%20K.,%20Rasmussen,%20P.%20B.%20&amp;%20Molin,%20S.%20Unique%20type%20of%20plasmid%20maintenance%20function:%20postsegregational%20killing%20of%20plasmid-free%20cells.%20Proc.%20Natl%20Acad.%20Sci.%20USA83,%203116%E2%80%933120%20(1986)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Schmartz, G. P. et al. PLSDB: advancing a comprehensive database of bacterial plasmids. <em>Nucleic Acids Res.</em><strong>50</strong>, D273–D278 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkab1111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34850116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schmartz,%20G.%20P.%20et%20al.%20PLSDB:%20advancing%20a%20comprehensive%20database%20of%20bacterial%20plasmids.%20Nucleic%20Acids%20Res.50,%20D273%E2%80%93D278%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Galata, V., Fehlmann, T., Backes, C. &amp; Keller, A. PLSDB: a resource of complete bacterial plasmids. <em>Nucleic Acids Res.</em><strong>47</strong>, D195–D202 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gky1050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30380090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Galata,%20V.,%20Fehlmann,%20T.,%20Backes,%20C.%20&amp;%20Keller,%20A.%20PLSDB:%20a%20resource%20of%20complete%20bacterial%20plasmids.%20Nucleic%20Acids%20Res.47,%20D195%E2%80%93D202%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Madrid, C., Balsalobre, C., García, J. &amp; Juárez, A. The novel Hha/YmoA family of nucleoid-associated proteins: use of structural mimicry to modulate the activity of the H-NS family of proteins. <em>Mol. Microbiol.</em><strong>63</strong>, 7–14 (2007).
</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2006.05497.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17116239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Madrid,%20C.,%20Balsalobre,%20C.,%20Garc%C3%ADa,%20J.%20&amp;%20Ju%C3%A1rez,%20A.%20The%20novel%20Hha/YmoA%20family%20of%20nucleoid-associated%20proteins:%20use%20of%20structural%20mimicry%20to%20modulate%20the%20activity%20of%20the%20H-NS%20family%20of%20proteins.%20Mol.%20Microbiol.63,%207%E2%80%9314%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Holm, L. &amp; Sander, C. Dali: a network tool for protein structure comparison. <em>Trends Biochem Sci.</em><strong>20</strong>, 478–480 (1995).
</cite> [<a href="https://doi.org/10.1016/s0968-0004(00)89105-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8578593/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Holm,%20L.%20&amp;%20Sander,%20C.%20Dali:%20a%20network%20tool%20for%20protein%20structure%20comparison.%20Trends%20Biochem%20Sci.20,%20478%E2%80%93480%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Kwak, M.-J. et al. Architecture of the type IV coupling protein complex of Legionella pneumophila. <em>Nat. Microbiol</em><strong>2</strong>, 17114 (2017).
</cite> [<a href="https://doi.org/10.1038/nmicrobiol.2017.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6497169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28714967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kwak,%20M.-J.%20et%20al.%20Architecture%20of%20the%20type%20IV%20coupling%20protein%20complex%20of%20Legionella%20pneumophila.%20Nat.%20Microbiol2,%2017114%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Meir, A. et al. Mechanism of effector capture and delivery by the type IV secretion system from Legionella pneumophila. <em>Nat. Commun.</em><strong>11</strong>, 2864 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-16681-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7280309/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32513920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meir,%20A.%20et%20al.%20Mechanism%20of%20effector%20capture%20and%20delivery%20by%20the%20type%20IV%20secretion%20system%20from%20Legionella%20pneumophila.%20Nat.%20Commun.11,%202864%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Couturier, A. et al. Real-time visualisation of the intracellular dynamics of conjugative plasmid transfer. <em>Nat. Commun.</em><strong>14</strong>, 294 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-35978-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9849209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36653393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Couturier,%20A.%20et%20al.%20Real-time%20visualisation%20of%20the%20intracellular%20dynamics%20of%20conjugative%20plasmid%20transfer.%20Nat.%20Commun.14,%20294%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Achtman, M., Kennedy, N. &amp; Skurray, R. Cell-cell interactions in conjugating Escherichia coli: role of traT protein in surface exclusion. <em>Proc. Natl Acad. Sci. Usa.</em><strong>74</strong>, 5104–5108 (1977).
</cite> [<a href="https://doi.org/10.1073/pnas.74.11.5104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC432108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/337311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Achtman,%20M.,%20Kennedy,%20N.%20&amp;%20Skurray,%20R.%20Cell-cell%20interactions%20in%20conjugating%20Escherichia%20coli:%20role%20of%20traT%20protein%20in%20surface%20exclusion.%20Proc.%20Natl%20Acad.%20Sci.%20Usa.74,%205104%E2%80%935108%20(1977)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Carattoli, A., Seiffert, S. N., Schwendener, S., Perreten, V. &amp; Endimiani, A. Differentiation of IncL and IncM plasmids associated with the spread of clinically relevant antimicrobial resistance. <em>PLoS ONE</em><strong>10</strong>, e0123063 (2015).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0123063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4416936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25933288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carattoli,%20A.,%20Seiffert,%20S.%20N.,%20Schwendener,%20S.,%20Perreten,%20V.%20&amp;%20Endimiani,%20A.%20Differentiation%20of%20IncL%20and%20IncM%20plasmids%20associated%20with%20the%20spread%20of%20clinically%20relevant%20antimicrobial%20resistance.%20PLoS%20ONE10,%20e0123063%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Nordström, K. Plasmid R1—Replication and its control. <em>Plasmid</em><strong>55</strong>, 1–26 (2006).
</cite> [<a href="https://doi.org/10.1016/j.plasmid.2005.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16199086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nordstr%C3%B6m,%20K.%20Plasmid%20R1%E2%80%94Replication%20and%20its%20control.%20Plasmid55,%201%E2%80%9326%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Dimitriu, T., Matthews, A. C. &amp; Buckling, A. Increased copy number couples the evolution of plasmid horizontal transmission and plasmid-encoded antibiotic resistance. <em>Proc. Natl Acad. Sci. USA</em><strong>118</strong>, e2107818118 (2021).
</cite> [<a href="https://doi.org/10.1073/pnas.2107818118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8346908/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34326267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dimitriu,%20T.,%20Matthews,%20A.%20C.%20&amp;%20Buckling,%20A.%20Increased%20copy%20number%20couples%20the%20evolution%20of%20plasmid%20horizontal%20transmission%20and%20plasmid-encoded%20antibiotic%20resistance.%20Proc.%20Natl%20Acad.%20Sci.%20USA118,%20e2107818118%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Bonnin, R. A., Nordmann, P., Carattoli, A. &amp; Poirel, L. Comparative genomics of IncL/M-type plasmids: evolution by acquisition of resistance genes and insertion sequences. <em>Antimicrob. Agents Chemother.</em><strong>57</strong>, 674–676 (2013).
</cite> [<a href="https://doi.org/10.1128/AAC.01086-12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3535931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23114767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bonnin,%20R.%20A.,%20Nordmann,%20P.,%20Carattoli,%20A.%20&amp;%20Poirel,%20L.%20Comparative%20genomics%20of%20IncL/M-type%20plasmids:%20evolution%20by%20acquisition%20of%20resistance%20genes%20and%20insertion%20sequences.%20Antimicrob.%20Agents%20Chemother.57,%20674%E2%80%93676%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Sakuma, T., Tazumi, S., Furuya, N. &amp; Komano, T. ExcA proteins of IncI1 plasmid R64 and IncIγ plasmid R621a recognize different segments of their cognate TraY proteins in entry exclusion. <em>Plasmid</em><strong>69</strong>, 138–145 (2013).
</cite> [<a href="https://doi.org/10.1016/j.plasmid.2012.11.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23201046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sakuma,%20T.,%20Tazumi,%20S.,%20Furuya,%20N.%20&amp;%20Komano,%20T.%20ExcA%20proteins%20of%20IncI1%20plasmid%20R64%20and%20IncI%CE%B3%20plasmid%20R621a%20recognize%20different%20segments%20of%20their%20cognate%20TraY%20proteins%20in%20entry%20exclusion.%20Plasmid69,%20138%E2%80%93145%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Audette, G. F., Manchak, J., Beatty, P., Klimke, W. A. &amp; Frost, L. S. Entry exclusion in F-like plasmids requires intact TraG in the donor that recognizes its cognate TraS in the recipient. <em>Microbiology</em><strong>153</strong>, 442–451 (2007).
</cite> [<a href="https://doi.org/10.1099/mic.0.2006/001917-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17259615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Audette,%20G.%20F.,%20Manchak,%20J.,%20Beatty,%20P.,%20Klimke,%20W.%20A.%20&amp;%20Frost,%20L.%20S.%20Entry%20exclusion%20in%20F-like%20plasmids%20requires%20intact%20TraG%20in%20the%20donor%20that%20recognizes%20its%20cognate%20TraS%20in%20the%20recipient.%20Microbiology153,%20442%E2%80%93451%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Dmowski, M., Gołębiewski, M. &amp; Kern-Zdanowicz, I. Characteristics of the conjugative transfer system of the IncM plasmid pCTX-M3 and identification of its putative regulators. J. Bacteriol. <strong>200</strong>, e00234–e00318 (2018).
</cite> [<a href="https://doi.org/10.1128/JB.00234-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6112013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29986941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dmowski,%20M.,%20Go%C5%82%C4%99biewski,%20M.%20&amp;%20Kern-Zdanowicz,%20I.%20Characteristics%20of%20the%20conjugative%20transfer%20system%20of%20the%20IncM%20plasmid%20pCTX-M3%20and%20identification%20of%20its%20putative%20regulators.%20J.%20Bacteriol.%20200,%20e00234%E2%80%93e00318%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Goncharoff, P., Saadi, S., Chang, C. H., Saltman, L. H. &amp; Figurski, D. H. Structural, molecular, and genetic analysis of the kilA operon of broad-host-range plasmid RK2. <em>J. Bacteriol.</em><strong>173</strong>, 3463–3477 (1991).
</cite> [<a href="https://doi.org/10.1128/jb.173.11.3463-3477.1991" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC207960/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2045366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goncharoff,%20P.,%20Saadi,%20S.,%20Chang,%20C.%20H.,%20Saltman,%20L.%20H.%20&amp;%20Figurski,%20D.%20H.%20Structural,%20molecular,%20and%20genetic%20analysis%20of%20the%20kilA%20operon%20of%20broad-host-range%20plasmid%20RK2.%20J.%20Bacteriol.173,%203463%E2%80%933477%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Costa, T. R. D., Patkowski, J. B., Macé, K., Christie, P. J. &amp; Waksman, G. Structural and functional diversity of type IV secretion systems. <em>Nat. Rev. Microbiol</em><strong>22</strong>, 170–185 (2024).
</cite> [<a href="https://doi.org/10.1038/s41579-023-00974-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11290344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37814112/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Costa,%20T.%20R.%20D.,%20Patkowski,%20J.%20B.,%20Mac%C3%A9,%20K.,%20Christie,%20P.%20J.%20&amp;%20Waksman,%20G.%20Structural%20and%20functional%20diversity%20of%20type%20IV%20secretion%20systems.%20Nat.%20Rev.%20Microbiol22,%20170%E2%80%93185%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Al Mamun, A. A. M., Kishida, K. &amp; Christie, P. J. Protein transfer through an f plasmid-encoded type iv secretion system suppresses the mating-induced SOS response. <em>mBio</em><strong>12</strong>, e0162921 (2021).
</cite> [<a href="https://doi.org/10.1128/mBio.01629-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8406263/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34253063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Al%20Mamun,%20A.%20A.%20M.,%20Kishida,%20K.%20&amp;%20Christie,%20P.%20J.%20Protein%20transfer%20through%20an%20f%20plasmid-encoded%20type%20iv%20secretion%20system%20suppresses%20the%20mating-induced%20SOS%20response.%20mBio12,%20e0162921%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Wilkins, B. M. &amp; Thomas, A. T. DNA-independent transport of plasmid primase protein between bacteria by the I1 conjugation system. <em>Mol. Microbiol.</em><strong>38</strong>, 650–657 (2000).
</cite> [<a href="https://doi.org/10.1046/j.1365-2958.2000.02164.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11069687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilkins,%20B.%20M.%20&amp;%20Thomas,%20A.%20T.%20DNA-independent%20transport%20of%20plasmid%20primase%20protein%20between%20bacteria%20by%20the%20I1%20conjugation%20system.%20Mol.%20Microbiol.38,%20650%E2%80%93657%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Yamaichi, Y. et al. High-resolution genetic analysis of the requirements for horizontal transmission of the ESBL plasmid from Escherichia coli O104:H4. <em>Nucleic Acids Res.</em><strong>43</strong>, 348–360 (2015).
</cite> [<a href="https://doi.org/10.1093/nar/gku1262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4288162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25477379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamaichi,%20Y.%20et%20al.%20High-resolution%20genetic%20analysis%20of%20the%20requirements%20for%20horizontal%20transmission%20of%20the%20ESBL%20plasmid%20from%20Escherichia%20coli%20O104:H4.%20Nucleic%20Acids%20Res.43,%20348%E2%80%93360%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Lian, Z. J. et al. Genetic basis of I-complex plasmid stability and conjugation. <em>PLoS Genet</em><strong>19</strong>, e1010773 (2023).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1010773" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10286972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37347771/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lian,%20Z.%20J.%20et%20al.%20Genetic%20basis%20of%20I-complex%20plasmid%20stability%20and%20conjugation.%20PLoS%20Genet19,%20e1010773%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Hancock, S. J. et al. Identification of IncA/C plasmid replication and maintenance genes and development of a plasmid multilocus sequence typing scheme. Antimicrob. Agents Chemother. <strong>61</strong>, e01740–e01816 (2017).
</cite> [<a href="https://doi.org/10.1128/AAC.01740-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5278728/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27872077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hancock,%20S.%20J.%20et%20al.%20Identification%20of%20IncA/C%20plasmid%20replication%20and%20maintenance%20genes%20and%20development%20of%20a%20plasmid%20multilocus%20sequence%20typing%20scheme.%20Antimicrob.%20Agents%20Chemother.%2061,%20e01740%E2%80%93e01816%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Yi, L. et al. <em>PixR, a Novel Activator of Conjugative Transfer of IncX4 Resistance Plasmids, Mitigates the Fitness Cost of</em> Mcr-1 <em>Carriage in</em> Escherichia Coli. 10.1101/2021.08.04.455183 (2021).</cite> [<a href="https://doi.org/10.1128/mbio.03209-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8725589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35089067/" class="usa-link">PubMed</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Datsenko, K. A. &amp; Wanner, B. L. One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. <em>Proc. Natl Acad. Sci. USA</em><strong>97</strong>, 6640–6645 (2000).
</cite> [<a href="https://doi.org/10.1073/pnas.120163297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC18686/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10829079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Datsenko,%20K.%20A.%20&amp;%20Wanner,%20B.%20L.%20One-step%20inactivation%20of%20chromosomal%20genes%20in%20Escherichia%20coli%20K-12%20using%20PCR%20products.%20Proc.%20Natl%20Acad.%20Sci.%20USA97,%206640%E2%80%936645%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Yu, D. et al. An efficient recombination system for chromosome engineering in Escherichia coli. <em>Proc. Natl Acad. Sci. USA</em><strong>97</strong>, 5978–5983 (2000).
</cite> [<a href="https://doi.org/10.1073/pnas.100127597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC18544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10811905/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20D.%20et%20al.%20An%20efficient%20recombination%20system%20for%20chromosome%20engineering%20in%20Escherichia%20coli.%20Proc.%20Natl%20Acad.%20Sci.%20USA97,%205978%E2%80%935983%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Gibson, D. G. et al. Enzymatic assembly of DNA molecules up to several hundred kilobases. <em>Nat. Methods</em><strong>6</strong>, 343–345 (2009).
</cite> [<a href="https://doi.org/10.1038/nmeth.1318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19363495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gibson,%20D.%20G.%20et%20al.%20Enzymatic%20assembly%20of%20DNA%20molecules%20up%20to%20several%20hundred%20kilobases.%20Nat.%20Methods6,%20343%E2%80%93345%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Royet, K., Parisot, N., Rodrigue, A., Gueguen, E. &amp; Condemine, G. Identification by Tn-seq of Dickeya dadantii genes required for survival in chicory plants. <em>Mol. Plant Pathol.</em><strong>20</strong>, 287–306 (2019).
</cite> [<a href="https://doi.org/10.1111/mpp.12754" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6637903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30267562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Royet,%20K.,%20Parisot,%20N.,%20Rodrigue,%20A.,%20Gueguen,%20E.%20&amp;%20Condemine,%20G.%20Identification%20by%20Tn-seq%20of%20Dickeya%20dadantii%20genes%20required%20for%20survival%20in%20chicory%20plants.%20Mol.%20Plant%20Pathol.20,%20287%E2%80%93306%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. <em>EMBnet J.</em><strong>17</strong>, 10 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Martin,%20M.%20Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads.%20EMBnet%20J.17,%2010%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Fournes, F. et al. The coordinated replication of <em>Vibrio cholerae</em>’s two chromosomes required the acquisition of a unique domain by the RctB initiator. <em>Nucleic Acids Res.</em><strong>49</strong>, 11119–11133 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab903" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8565311/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34643717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fournes,%20F.%20et%20al.%20The%20coordinated%20replication%20of%20Vibrio%20cholerae%E2%80%99s%20two%20chromosomes%20required%20the%20acquisition%20of%20a%20unique%20domain%20by%20the%20RctB%20initiator.%20Nucleic%20Acids%20Res.49,%2011119%E2%80%9311133%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Finn, R. D., Clements, J. &amp; Eddy, S. R. HMMER web server: interactive sequence similarity searching. <em>Nucleic Acids Res.</em><strong>39</strong>, W29–W37 (2011).
</cite> [<a href="https://doi.org/10.1093/nar/gkr367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3125773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21593126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Finn,%20R.%20D.,%20Clements,%20J.%20&amp;%20Eddy,%20S.%20R.%20HMMER%20web%20server:%20interactive%20sequence%20similarity%20searching.%20Nucleic%20Acids%20Res.39,%20W29%E2%80%93W37%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Katoh, K. &amp; Standley, D. M. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. <em>Mol. Biol. Evol.</em><strong>30</strong>, 772–780 (2013).
</cite> [<a href="https://doi.org/10.1093/molbev/mst010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3603318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23329690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katoh,%20K.%20&amp;%20Standley,%20D.%20M.%20MAFFT%20multiple%20sequence%20alignment%20software%20version%207:%20improvements%20in%20performance%20and%20usability.%20Mol.%20Biol.%20Evol.30,%20772%E2%80%93780%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Criscuolo, A. &amp; Gribaldo, S. BMGE (Block Mapping and Gathering with Entropy): a new software for selection of phylogenetic informative regions from multiple sequence alignments. <em>BMC Evol. Biol.</em><strong>10</strong>, 210 (2010).
</cite> [<a href="https://doi.org/10.1186/1471-2148-10-210" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3017758/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20626897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Criscuolo,%20A.%20&amp;%20Gribaldo,%20S.%20BMGE%20(Block%20Mapping%20and%20Gathering%20with%20Entropy):%20a%20new%20software%20for%20selection%20of%20phylogenetic%20informative%20regions%20from%20multiple%20sequence%20alignments.%20BMC%20Evol.%20Biol.10,%20210%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Trifinopoulos, J., Nguyen, L.-T., von Haeseler, A. &amp; Minh, B. Q. W-IQ-TREE: a fast online phylogenetic tool for maximum likelihood analysis. <em>Nucleic Acids Res</em>. <strong>44</strong>, W232–W235 (2016).
</cite> [<a href="https://doi.org/10.1093/nar/gkw256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4987875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27084950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trifinopoulos,%20J.,%20Nguyen,%20L.-T.,%20von%20Haeseler,%20A.%20&amp;%20Minh,%20B.%20Q.%20W-IQ-TREE:%20a%20fast%20online%20phylogenetic%20tool%20for%20maximum%20likelihood%20analysis.%20Nucleic%20Acids%20Res.%2044,%20W232%E2%80%93W235%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Bianchini, G. &amp; Sánchez-Baracaldo, P. TreeViewer: flexible, modular software to visualise and manipulate phylogenetic trees. <em>Ecol. Evol.</em><strong>14</strong>, e10873 (2024).
</cite> [<a href="https://doi.org/10.1002/ece3.10873" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10834882/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38314311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bianchini,%20G.%20&amp;%20S%C3%A1nchez-Baracaldo,%20P.%20TreeViewer:%20flexible,%20modular%20software%20to%20visualise%20and%20manipulate%20phylogenetic%20trees.%20Ecol.%20Evol.14,%20e10873%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Abramson, J. et al. Accurate structure prediction of biomolecular interactions with AlphaFold 3. <em>Nature</em><strong>630</strong>, 493–500 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-024-07487-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11168924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abramson,%20J.%20et%20al.%20Accurate%20structure%20prediction%20of%20biomolecular%20interactions%20with%20AlphaFold%203.%20Nature630,%20493%E2%80%93500%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62404_MOESM2_ESM.pdf</a><sup> (56KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM3_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (3.6MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (161.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (73.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (217KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365149/bin/41467_2025_62404_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (29.7KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data to understand and assess the conclusions of this research are available in the main text, Supplementary Information and Supplementary Data <a href="#MOESM4" class="usa-link">1</a>. Source data are provided with this paper in the Source Data file. <a href="#Sec27" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62404-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62404.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365149/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365149/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365149%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365149/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365149/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365149/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830342/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365149/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830342/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365149/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365149/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="IyIvhJ91ms9LsrQ3yrfopB9H01yuduLao1mM8qWdaGGsb75UmZmS2Q2mX3aSbxDo">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
