
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Quantifying collective interactions in biomolecular phase separation - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5340AC8AF3C3830540AC0018B08F68.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365266/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Quantifying collective interactions in biomolecular phase separation">
<meta name="citation_author" content="Hannes Ausserwöger">
<meta name="citation_author_institution" content="Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK">
<meta name="citation_author" content="Ella de Csilléry">
<meta name="citation_author_institution" content="Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK">
<meta name="citation_author" content="Daoyuan Qian">
<meta name="citation_author_institution" content="Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK">
<meta name="citation_author" content="Georg Krainer">
<meta name="citation_author_institution" content="Institute of Molecular Biosciences (IMB), University of Graz, Graz, Austria">
<meta name="citation_author" content="Timothy J Welsh">
<meta name="citation_author_institution" content="Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK">
<meta name="citation_author" content="Tomas Sneideris">
<meta name="citation_author_institution" content="Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK">
<meta name="citation_author" content="Titus M Franzmann">
<meta name="citation_author_institution" content="Biotechnology Center (BIOTEC), Center for Molecular and Cellular Bioengineering (CMCB), Technische Universität Dresden, Dresden, Germany">
<meta name="citation_author" content="Seema Qamar">
<meta name="citation_author_institution" content="Cambridge Institute for Medical Research, Department of Clinical Neurosciences, Clinical School, University of Cambridge, Cambridge, UK">
<meta name="citation_author" content="Nadia A Erkamp">
<meta name="citation_author_institution" content="Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK">
<meta name="citation_author" content="Jonathon Nixon-Abell">
<meta name="citation_author_institution" content="Cambridge Institute for Medical Research, Department of Clinical Neurosciences, Clinical School, University of Cambridge, Cambridge, UK">
<meta name="citation_author" content="Mrityunjoy Kar">
<meta name="citation_author_institution" content="Max Planck Institute of Cell Biology and Genetics (MPI-CBG), Dresden, Germany">
<meta name="citation_author" content="Peter St George-Hyslop">
<meta name="citation_author_institution" content="Department of Medicine (Division of Neurology), Temerty Faculty of Medicine, University Health Network, University of Toronto, Toronto, ON Canada">
<meta name="citation_author_institution" content="Department of Neurology, Columbia University, New York, NY USA">
<meta name="citation_author" content="Anthony A Hyman">
<meta name="citation_author_institution" content="Max Planck Institute of Cell Biology and Genetics (MPI-CBG), Dresden, Germany">
<meta name="citation_author" content="Simon Alberti">
<meta name="citation_author_institution" content="Biotechnology Center (BIOTEC), Center for Molecular and Cellular Bioengineering (CMCB), Technische Universität Dresden, Dresden, Germany">
<meta name="citation_author" content="Rohit V Pappu">
<meta name="citation_author_institution" content="Department of Biomedical Engineering and Center for Biomolecular Condensates, Washington University in St. Louis, St. Louis, MO USA">
<meta name="citation_author" content="Tuomas P J Knowles">
<meta name="citation_author_institution" content="Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7724">
<meta name="citation_doi" content="10.1038/s41467-025-62437-y">
<meta name="citation_pmid" content="40830100">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365266/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365266/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365266/pdf/41467_2025_Article_62437.pdf">
<meta name="description" content="Biomolecular phase separation is an emerging theme for protein assembly and cellular organisation. The collective forces driving such condensation, however, remain challenging to characterise. Here we show that tracking the dilute phase ...">
<meta name="og:title" content="Quantifying collective interactions in biomolecular phase separation">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Biomolecular phase separation is an emerging theme for protein assembly and cellular organisation. The collective forces driving such condensation, however, remain challenging to characterise. Here we show that tracking the dilute phase ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365266/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365266">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62437-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62437.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365266%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365266/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365266/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365266/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 19;16:7724. doi: <a href="https://doi.org/10.1038/s41467-025-62437-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62437-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Quantifying collective interactions in biomolecular phase separation</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ausserw%C3%B6ger%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Hannes Ausserwöger</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Hannes Ausserwöger</span></h3>
<div class="p">
<sup>1</sup>Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ausserw%C3%B6ger%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hannes Ausserwöger</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Csill%C3%A9ry%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ella de Csilléry</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ella de Csilléry</span></h3>
<div class="p">
<sup>1</sup>Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Csill%C3%A9ry%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Ella de Csilléry</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qian%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Daoyuan Qian</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Daoyuan Qian</span></h3>
<div class="p">
<sup>1</sup>Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qian%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daoyuan Qian</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Krainer%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Georg Krainer</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Georg Krainer</span></h3>
<div class="p">
<sup>2</sup>Institute of Molecular Biosciences (IMB), University of Graz, Graz, Austria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Krainer%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Georg Krainer</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Welsh%20TJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Timothy J Welsh</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Timothy J Welsh</span></h3>
<div class="p">
<sup>1</sup>Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Welsh%20TJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Timothy J Welsh</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sneideris%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Tomas Sneideris</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Tomas Sneideris</span></h3>
<div class="p">
<sup>1</sup>Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sneideris%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tomas Sneideris</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Franzmann%20TM%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Titus M Franzmann</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Titus M Franzmann</span></h3>
<div class="p">
<sup>3</sup>Biotechnology Center (BIOTEC), Center for Molecular and Cellular Bioengineering (CMCB), Technische Universität Dresden, Dresden, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Franzmann%20TM%22%5BAuthor%5D" class="usa-link"><span class="name western">Titus M Franzmann</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qamar%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Seema Qamar</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Seema Qamar</span></h3>
<div class="p">
<sup>4</sup>Cambridge Institute for Medical Research, Department of Clinical Neurosciences, Clinical School, University of Cambridge, Cambridge, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qamar%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Seema Qamar</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Erkamp%20NA%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Nadia A Erkamp</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Nadia A Erkamp</span></h3>
<div class="p">
<sup>1</sup>Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Erkamp%20NA%22%5BAuthor%5D" class="usa-link"><span class="name western">Nadia A Erkamp</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nixon-Abell%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Jonathon Nixon-Abell</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Jonathon Nixon-Abell</span></h3>
<div class="p">
<sup>4</sup>Cambridge Institute for Medical Research, Department of Clinical Neurosciences, Clinical School, University of Cambridge, Cambridge, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nixon-Abell%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jonathon Nixon-Abell</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kar%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Mrityunjoy Kar</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Mrityunjoy Kar</span></h3>
<div class="p">
<sup>5</sup>Max Planck Institute of Cell Biology and Genetics (MPI-CBG), Dresden, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kar%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mrityunjoy Kar</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22St%20George-Hyslop%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Peter St George-Hyslop</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Peter St George-Hyslop</span></h3>
<div class="p">
<sup>6</sup>Department of Medicine (Division of Neurology), Temerty Faculty of Medicine, University Health Network, University of Toronto, Toronto, ON Canada </div>
<div class="p">
<sup>7</sup>Department of Neurology, Columbia University, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22St%20George-Hyslop%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter St George-Hyslop</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hyman%20AA%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Anthony A Hyman</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Anthony A Hyman</span></h3>
<div class="p">
<sup>5</sup>Max Planck Institute of Cell Biology and Genetics (MPI-CBG), Dresden, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hyman%20AA%22%5BAuthor%5D" class="usa-link"><span class="name western">Anthony A Hyman</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alberti%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Simon Alberti</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Simon Alberti</span></h3>
<div class="p">
<sup>3</sup>Biotechnology Center (BIOTEC), Center for Molecular and Cellular Bioengineering (CMCB), Technische Universität Dresden, Dresden, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alberti%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Simon Alberti</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pappu%20RV%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Rohit V Pappu</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Rohit V Pappu</span></h3>
<div class="p">
<sup>8</sup>Department of Biomedical Engineering and Center for Biomolecular Condensates, Washington University in St. Louis, St. Louis, MO USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pappu%20RV%22%5BAuthor%5D" class="usa-link"><span class="name western">Rohit V Pappu</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Knowles%20TPJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Tuomas P J Knowles</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Tuomas P J Knowles</span></h3>
<div class="p">
<sup>1</sup>Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Knowles%20TPJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Tuomas P J Knowles</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Centre for Misfolding Diseases, Yusuf Hamied Department of Chemistry, University of Cambridge, Cambridge, UK </div>
<div id="Aff2">
<sup>2</sup>Institute of Molecular Biosciences (IMB), University of Graz, Graz, Austria </div>
<div id="Aff3">
<sup>3</sup>Biotechnology Center (BIOTEC), Center for Molecular and Cellular Bioengineering (CMCB), Technische Universität Dresden, Dresden, Germany </div>
<div id="Aff4">
<sup>4</sup>Cambridge Institute for Medical Research, Department of Clinical Neurosciences, Clinical School, University of Cambridge, Cambridge, UK </div>
<div id="Aff5">
<sup>5</sup>Max Planck Institute of Cell Biology and Genetics (MPI-CBG), Dresden, Germany </div>
<div id="Aff6">
<sup>6</sup>Department of Medicine (Division of Neurology), Temerty Faculty of Medicine, University Health Network, University of Toronto, Toronto, ON Canada </div>
<div id="Aff7">
<sup>7</sup>Department of Neurology, Columbia University, New York, NY USA </div>
<div id="Aff8">
<sup>8</sup>Department of Biomedical Engineering and Center for Biomolecular Condensates, Washington University in St. Louis, St. Louis, MO USA </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2023 Sep 19; Accepted 2025 Jul 21; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365266  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830100/" class="usa-link">40830100</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Biomolecular phase separation is an emerging theme for protein assembly and cellular organisation. The collective forces driving such condensation, however, remain challenging to characterise. Here we show that tracking the dilute phase concentration of only one component suffices to quantify composition and energetics of multicomponent condensates. Applying this assay to several disease- and stress-related proteins, we find that monovalent ions can either deplete from or enrich within the dense phase in a context-dependent manner. By analysing the effect of the widely used modulator 1,6-hexanediol, we find that the compound inhibits phase separation by acting as a solvation agent that expands polypeptide chains. Extending the strategy to <em>in cellulo</em> data, we even quantify the relative energetic contributions of individual proteins within complex condensates. Together, our approach provides a generic and broadly applicable tool for dissecting the forces governing biomolecular condensation and guiding the rational modulation of condensate behaviour.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Biophysics, Biophysical chemistry</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Biomolecular phase separation arises from collective molecular interactions and is emerging as a key theme for biological function. Here the authors propose a broadly applicable method to quantify these interactions based on compositional and energetic parameters.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">The assembly of proteins is a ubiquitous phenomenon in living systems that enables essential functions spanning from synthesis<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> to cellular signalling<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Recently, phase separation of proteins into biomolecular condensates has been discovered as a pathway for protein assembly<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Through phase separation, proteins spontaneously demix from a homogenous phase into a protein-rich, condensed phase and a protein-poor, dilute phase<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Such condensate formation has been attributed an important role in cellular organisation<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> and is associated with aberrant behaviour in pathology<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>.</p>
<p id="Par4">Biomolecular phase separation is driven by collective interactions, stemming from the collaborative association of large numbers of molecules. These collective interactions give rise to emergent behaviours such as differential partitioning of solutes between dense and dilute phases. Conversely, individual components also differ in their contributions to the overall decrease in free energy, acting as the driving force for phase separation. Common protein interaction assays, however, fail to inform of the collective properties underlying phase separation, given a focus on binary interactions. Even assays that map out the phase boundary, which refers to the set of critical concentrations<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, do not provide information on the dense phase composition and fail to quantify the energetics.</p>
<p id="Par5">Recent theoretical advances have introduced novel descriptors that enable the quantification of collective interactions<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. These descriptors are derived from the analysis of tie line gradients<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> and the quantification of component dominance<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, and provide means to physically characterise the mechanisms of condensate formation and modulation. Tie lines describe the demixing process between biomolecular dense and dilute phases, providing insights into condensate composition and the underlying interactions. Additionally, component dominance analysis offers the opportunity to discern individual components’ contributions to the decrease in free energy, thus shedding light on the underlying driving forces and energetics in a locally defined manner. Hence, translating these theoretical advances into commonly accessible experiments addresses the current limitations in characterising the collective interactions underlying protein phase separation.</p>
<p id="Par6">Here, we establish a general strategy for quantifying the collective interactions in biomolecular phase separation with a widely accessible experimental approach. By applying this framework, we shed light on condensate composition and the underlying energetics of a range of physiologically relevant phase separation-prone proteins. We find that even elemental ions commonly display concentration gradients between the dense and dilute phases, and that partitioning effects are context-dependent on the governing interactions. We further show that the common small-molecule hydrophobic disruptor 1,6-hexanediol acts by regulating polypeptide chain expansion and even highlight how our approach can be applied intracellularly.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Descriptors for quantifying collective interactions</h3>
<p id="Par7">Tie lines connect the equilibrium dilute and dense phase concentrations of constituent species formed after demixing, and so describe condensate composition including partitioning (see Fig. <a href="#Fig1" class="usa-link">1a</a>). In a two-component system, the tie line gradient can be determined by identifying a set of total concentration conditions that fulfil <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e406"><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span> = const. For higher-dimensional systems, as is generally the case in practical settings (additional components include ions, buffers, etc.), tie lines can be approximated by the same approach of identifying the contour <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e423"><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span> = const<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. This leads to a dimensionality reduction of tie lines to the A-B measurement plane of interest, by intersection of the higher-dimensional tie lines with said plane (Fig. <a href="#Fig1" class="usa-link">1b</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). Crucially, the reduced tie line gradient <em>K</em> preserves essential information on the partitioning of (co-)solutes of interest as it approximates the ratio of the higher-dimensional tie line gradient entries in A and B<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Specifically, for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e461"><mi>K</mi></math></span> &lt; 0, component A is enriched in the dense phase (i.e., included in condensates), while component B is decreased (i.e. excluded from condensates). By contrast, a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e470"><mi>K</mi></math></span> &gt; 0 means that both species are present at higher concentrations in the dense phase than in the dilute phase, indicating preferential partitioning into condensates.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Descriptors of collective interactions in biomolecular phase separation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365266_41467_2025_62437_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2e32/12365266/70051349ffd6/41467_2025_62437_Fig1_HTML.jpg" loading="lazy" id="d33e601" height="645" width="698" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic of a full phase diagram for a 2-D system between components A and B (abbreviated as Comp.). The coexistence region is shaded in blue. Tie lines describe the demixing of a total composition (<em>c</em><sub>tot</sub>) within the coexistence region into dilute (<em>c</em><sub>dil</sub>) and dense phase concentrations (<em>c</em><sub>den</sub>). The tie line gradient <em>K</em> is informative of component partitioning and can be obtained by determining conditions of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e504"><msubsup><mrow><mi>c</mi></mrow><mrow><mstyle><mtext>dil</mtext></mstyle></mrow><mrow><mstyle><mtext>A</mtext></mstyle></mrow></msubsup></math></span> = const. <strong>b</strong> In higher-dimensional systems, tie lines do not have to be constrained to the A-B measurement plane (blue). Enforcing the condition <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e524"><msubsup><mrow><mi>c</mi></mrow><mrow><mstyle><mtext>dil</mtext></mstyle></mrow><mrow><mstyle><mtext>A</mtext></mstyle></mrow></msubsup></math></span> = const., reduces the higher-dimensional tie lines to the A-B measurement plane, resulting in a so-called dilute phase contour. Dilute phase contours correspond to the intersection of the higher-dimensional tie lines originating from the B-C plane at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e541"><msubsup><mrow><mi>c</mi></mrow><mrow><mstyle><mtext>dil</mtext></mstyle></mrow><mrow><mstyle><mtext>A</mtext></mstyle></mrow></msubsup></math></span> = const (purple), with the A-B measurement plane of interest. <strong>c</strong> Reduction of tie lines to the A-B measurement plane gives rise to information on <em>K</em>. Furthermore, the local dependence of the saturation concentration on both component A and B is described by the phase boundary gradient <em>P</em>. <strong>d</strong> Together <em>K</em> and <em>P</em> inform on the relative free energy decrease for phase separation of component A as given by the dominance of component A (<em>D</em><sup>A</sup>). Both <em>K</em> and <em>P</em> can be determined from measurements of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e588"><msubsup><mrow><mi>c</mi></mrow><mrow><mstyle><mtext>dil</mtext></mstyle></mrow><mrow><mstyle><mtext>A</mtext></mstyle></mrow></msubsup></math></span> only.</p></figcaption></figure><p id="Par8">In addition to tie lines, the shape of the phase boundary plays an essential role in understanding collective interactions. The phase boundary describes the local dependence of the saturation concentration (Fig. <a href="#Fig1" class="usa-link">1c</a>) on the individual components of interest. This relationship can be quantified by the local phase boundary gradient <em>P</em>. In the extreme cases where <em>P</em> approaches 0 or ∞, the onset of phase separation becomes solely dependent on component B or A, respectively. Together, <em>P</em> and <em>K</em> inform on the energetics of the phase separation process according to (Fig. <a href="#Fig1" class="usa-link">1d</a>, see “Derivation of the dominance framework” section)<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e633" display="block"><msup><mrow><mi>D</mi></mrow><mrow><mi>A</mi></mrow></msup><mo>=</mo><mfrac><mrow><msup><mrow><mi mathvariant="normal">Δ</mi><mi>f</mi></mrow><mrow><mi>A</mi></mrow></msup></mrow><mrow><mi mathvariant="normal">Δ</mi><mi>f</mi></mrow></mfrac><mo>=</mo><mfrac><mrow><mi>K</mi></mrow><mrow><mi>K</mi><mo>−</mo><mi>P</mi></mrow></mfrac></math></td>
<td class="label">1</td>
</tr></table>
<p>where <em>D</em><sup>A</sup> denotes the dominance of component A and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e672"><mi mathvariant="normal">Δ</mi><msup><mrow><mi>f</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup></math></span> the free energy decrease of component A with regard to the overall free energy decrease Δ<em>f</em> <sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. <em>D</em><sup>A</sup> gives the relative energetic contribution of component A to the overall free energy decrease, which provides a measure of the system's energetic dependence on said component.</p>
<p id="Par9">Reduced tie line gradients <em>K</em>, phase boundary gradients <em>P</em> and dominance <em>D</em><sup>A</sup> can be extracted by measuring dilute phase concentrations of one component only, as shown in Fig. <a href="#Fig2" class="usa-link">2</a>. Such dilute phase concentration measurements can be performed using image-based confocal detection or even standard epifluorescence or absorbance-based measurements post-separation of dilute and dense phase. Here, we have chosen a microfluidic flow cell setup with a scanning confocal microscope (see Fig. <a href="#Fig2" class="usa-link">2a</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a> for confocal measurement setup) to optimise precision and throughput. The dilute phase concentrations are then extracted from the baseline signal in the fluorescence time traces (Fig. <a href="#Fig2" class="usa-link">2b, c</a>). Short bursts of intensity represent condensates but are not considered for further analysis.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Quantification of collective interactions using one-component dilute phase concentrations measurements.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365266_41467_2025_62437_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2e32/12365266/3139d5b26fb0/41467_2025_62437_Fig2_HTML.jpg" loading="lazy" id="d33e802" height="694" width="699" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Heterotypic line scans are performed by injecting samples with constant concentrations of component A but varying concentrations of component B (illustrated by yellow-purple colour gradient) into a microfluidic flow cell. Readout of fluorescence is done by confocal detection. <strong>b</strong> Representative intensity time traces of the component obtained from confocal detection measurements. <strong>c</strong> Intensity histograms obtained from time traces in (<strong>b</strong>), where the maximum is representative of the dilute phase concentration of component A. <strong>d</strong> Component A (FUS-EGFP … Fused in sarcoma protein with a enhanced green fluorescent protein tag), dilute phase concentration changes with varying component B (KCl … potassium chloride) concentrations along two separate line scans. The raw data for the line scan at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e746"><msubsup><mrow><mi>c</mi></mrow><mrow><mstyle><mtext>tot</mtext></mstyle></mrow><mrow><mstyle><mtext>FUS</mtext></mstyle></mrow></msubsup><mo>=</mo><mn>2</mn><mi mathvariant="normal">μ</mi><mstyle><mtext>M</mtext></mstyle></math></span> is shown in (<strong>b</strong>, <strong>c</strong>), and the corresponding data for the line scan at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e776"><msubsup><mrow><mi>c</mi></mrow><mrow><mstyle><mtext>tot</mtext></mstyle></mrow><mrow><mstyle><mtext>FUS</mtext></mstyle></mrow></msubsup><mo>=</mo><mn>1</mn><mi mathvariant="normal">μ</mi><mstyle><mtext>M</mtext></mstyle></math></span> is given in Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>. Evaluation of compositional and thermodynamic parameters follows from the dilute phase response gradient <em>R</em><sub>AB</sub> and the saturation concentrations. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par10">To extract <em>K</em>, <em>P</em> and <em>D</em><sup>A</sup>, the change in the dilute phase concentration of component A is quantified by conducting a series of experiments with constant total concentration of component A (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e820"><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span>) and varying total concentration of component B (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e837"><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow></msub><mo>,</mo><msup><mrow><mn>1</mn></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msup></math></span>) (see Fig. <a href="#Fig2" class="usa-link">2d</a>). These so-called heterotypic line scans are then performed at two different constant total concentrations of component A (i.e., <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e863"><msub><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow></msub><mo>,</mo><msup><mrow><mn>1</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e884"><msub><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow></msub><mo>,</mo><msup><mrow><mn>2</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup></math></span>). Measurements of additional heterotypic line scans at different concentrations, followed by averaging, can improve the accuracy, but are not necessary. At higher concentrations of component B, the heterotypic line scan displays a flat plateau region because no phase separation occurs in this range. Consequently, the dilute phase concentration is equal to the total concentration. However, at total concentrations of component B below the saturation concentration of component B (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e905"><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">sat</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span>), phase separation takes place, leading to a decrease in the dilute phase concentration due to the formation of condensates.</p>
<p id="Par11">The phase boundary gradient <em>P</em> is then determined from the change in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e927"><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">sat</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span> with changing <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e944"><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span>:</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e962" display="block"><mi>P</mi><mo>=</mo><mfrac><mrow><msub><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow></msub><mo>,</mo><msup><mrow><mn>2</mn></mrow><mrow><mi>A</mi></mrow></msup><mo>−</mo><msub><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow></msub><mo>,</mo><msup><mrow><mn>1</mn></mrow><mrow><mi>A</mi></mrow></msup></mrow><mrow><msub><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">sat</mi></mrow></msub><mo>,</mo><msup><mrow><mn>2</mn></mrow><mrow><mi>B</mi></mrow></msup><mo>−</mo><msub><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">sat</mi></mrow></msub><mo>,</mo><msup><mrow><mn>1</mn></mrow><mrow><mi>B</mi></mrow></msup></mrow></mfrac><mo>=</mo><mfrac><mrow><msubsup><mrow><mi mathvariant="normal">Δ</mi><mi>c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi>A</mi></mrow></msubsup></mrow><mrow><mi mathvariant="normal">Δ</mi><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">sat</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></mrow></mfrac></math></td>
<td class="label">2</td>
</tr></table>
<p id="Par12">In order to calculate <em>K</em>, the heterotypic dilute phase response gradient <em>R</em><sub>AB</sub> is determined from the initial slope of the decrease in dilute phase concentration (Fig. <a href="#Fig2" class="usa-link">2d</a>). <em>R</em><sub>AB</sub> quantifies the change in dilute phase concentration of component A with increasing concentrations of component B. From <em>P</em> and <em>R</em><sub>AB</sub>, the reduced tie line gradient <em>K</em> is calculated according to<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e1081" display="block"><mi>K</mi><mo>=</mo><mfrac><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">AB</mi></mrow></msub><mo>*</mo><mi>P</mi></mrow><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">AB</mi></mrow></msub><mo>−</mo><mi>P</mi></mrow></mfrac></math></td>
<td class="label">3</td>
</tr></table>
<p id="Par13">Finally, from <em>K</em> and <em>P</em>, the dominance <em>D</em> of component A is determined according to Eq. <a href="#Equ1" class="usa-link">1</a>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Ions are excluded from FUS condensates at low ionic strengths to counteract charge screening</h3>
<p id="Par14">The intrinsically disordered protein FUS is involved in physiologically relevant intracellular condensation events<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> and associated with the emergence of amyotrophic lateral sclerosis (ALS)<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. A hallmark of FUS is its tendency to undergo phase separation with decreasing ionic strengths, but key features such as ion partitioning or the impact of ions on the energetic driving force remain unaddressed.</p>
<p id="Par15">Hence, we set out to characterise the collective interactions of FUS with salt species using the tie line gradient and dominance analysis approach. We first determined the dilute phase concentrations of FUS at 1 and 2 µM total protein concentration and KCl concentrations ranging from 20 to 150 mM (Fig. <a href="#Fig3" class="usa-link">3a, b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). At KCl concentrations &gt;75 mM, the protein dilute phase concentration did not vary, as no phase separation was observed (see Fig. <a href="#Fig3" class="usa-link">3b</a>). Only with decreasing charge screening at lower KCl concentrations a drop-off in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e1158"><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">FUS</mi></mrow></msubsup></math></span> was observed, corresponding to the onset of phase separation. This enables determination of the dilute phase response gradient <em>R</em><sub>AB</sub> = 0.020 ± 0.003 µM/mM, which confirms that increasing KCl concentrations cause an increase in the FUS dilute phase concentration. The phase boundary gradient of <em>P</em> = 30.3 ± 8.1 µM/mM highlights that lower concentrations of KCl are required to induce phase separation at lower protein concentrations. This can be further illustrated by determining the FUS/KCl phase boundary in the experimentally probed concentration range (Fig. <a href="#Fig3" class="usa-link">3c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a> for phase boundary determination approach).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Ion exclusion facilitates homotypic protein phase separation at low salt conditions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365266_41467_2025_62437_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2e32/12365266/c02e4f0309ef/41467_2025_62437_Fig3_HTML.jpg" loading="lazy" id="d33e1226" height="403" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Phase separation of FUS-EGFP was studied in the presence of potassium chloride (KCl). <strong>b</strong> Dilute phase concentration changes of FUS at varying total KCl concentrations. <strong>c</strong> Determination of the FUS/KCl phase boundary from dilute phase concentration changes in (<strong>a</strong>). <strong>d</strong> Schematic representation of the dominance value of FUS. <strong>e</strong> Dense phase EGFP fluorescent lifetimes (<em>t</em><sub>den</sub>) at 2 µM FUS and varying KCl concentrations. Data are presented as mean ± SD from <em>n</em> &gt; 3 repeats. Dilute phase concentration changes of proteins TDP43 (<strong>f</strong>) and PGL3 (<strong>g</strong>) at varying total KCl concentrations. Grey dotted lines highlight the FUS saturation concentration for comparison. <strong>h</strong> Schematic illustration of ion behaviour for FUS phase separation at low KCl concentrations. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par16">We then determined the reduced tie line gradient as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e1234"><mi>K</mi></math></span> = –0.05 ± 0.04 µM/mM. A negative gradient indicates that the ion concentration is lower in the dense phase than in the dilute phase (Fig. <a href="#Fig3" class="usa-link">3c</a>). Here, considering the dissociation of the salt into K<sup>+</sup> and Cl<sup>-</sup>, the obtained gradient corresponds to the ratio of protein to a weighted sum of the cationic and anionic species, with the weights relating to their valency<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Hence, on average, the ions are preferentially excluded from FUS condensates, likely causing a decrease in electrostatic shielding and strengthening protein–protein interactions. Such ion exclusion from the dense phase is corroborated by simulation results for FUS condensates<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> and has also been observed for polyelectrolyte polymer phase separation<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, while recent evidence further highlights that such ion partitioning and associated charge neutralisation can even drive pH gradients<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>.</p>
<p id="Par17">KCl has further been shown to interact with a large fraction of available polypeptide side chains<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, forming a hydration shell around the dispersed protein, which inhibits intermolecular protein interactions. A release of ions from the hydration shell would therefore allow for FUS to engage in much more effective protein–protein interactions within condensates. Indeed, we determined a FUS dominance of <em>D</em><sup>FUS</sup> = 0.61 ± 0.24, meaning that only ~61% of the free energy decrease released by phase separation can be associated with FUS (Fig. <a href="#Fig3" class="usa-link">3e</a>). This highlights the potential importance of the ion partitioning to the overall energetic driving force.</p>
<p id="Par18">To investigate this further, we determined the fluorescence lifetime of the EGFP protein tag on FUS as a function of varying KCl concentrations. Here, a much shorter lifetime was observed in the dense phase (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>), as driven by a higher refractive index due to the locally increased protein concentration within condensates<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. The fluorescence lifetime of the condensed phase was determined to further decrease with decreasing KCl concentration (see Fig. <a href="#Fig3" class="usa-link">3e</a>). Hence, FUS condensates appear to become more densely packed, indicative of enhanced protein interactions through the removal of potassium chloride ions. This further supports our observations above, that ion exclusion from the dense phase and potential release from FUS polypeptide chains act as important contributors to driving phase separation.</p>
<p id="Par19">Condensate assembly at low ionic strength is also commonly observed for other phase separation-prone proteins, which might suggest that the underlying partitioning behaviour with ions may be similar. To probe the flexibility and simplicity of our approach, we then sought to investigate ion partitioning and energetics for two further physiologically relevant phase separation-prone proteins in TDP43 and PGL3<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Upon determining the protein dilute phase concentrations under varying KCl concentrations by performing heterotypic line scan measurements for TDP43 and PGL3 (Fig. <a href="#Fig3" class="usa-link">3f, g</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">6</a> and <a href="#MOESM1" class="usa-link">7</a> for time traces), we find condensate dissolution at increasing ion concentrations. Both TDP43 and PGL3 exhibited <em>K</em> &lt; 0, indicating a similar ion exclusion mechanism to FUS. Despite the conserved ion exclusion behaviour, the response gradients, critical concentrations, and dominance values varied between the protein systems, highlighting the impact of distinct sequence features. Here, PGL3, for example, presented the highest dominance fraction of 72%. This suggests an increased importance of protein-associated interactions, as corroborated by an increase in the critical concentration of KCl compared to FUS and TDP43. TDP43, on the other hand, displayed the highest relative ion partitioning, indicating that the KCl dense and dilute phase concentration difference is most significant in this system, therefore yielding the lowest dominance fraction of <em>D</em> = 28%.</p>
<p id="Par20">Together, we highlight the ability of this dilute phase concentration measurement approach to evaluate the collective interaction properties of protein condensates, revealing a conserved ion partitioning mechanism at low ionic strengths (Fig. <a href="#Fig3" class="usa-link">3h</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Ions partition into condensates at high salt concentrations to drive non-ionic interactions</h3>
<p id="Par21">FUS, like many other proteins, commonly displays a so-called reentrant regime, where phase separation is observed at very high salt concentrations<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> (Fig. <a href="#Fig4" class="usa-link">4a</a>). Prompted by the observation of differential ion partitioning at low ionic strengths, we set out to characterise collective interactions in the presence of high concentrations of lithium chloride (LiCl) and caesium chloride (CsCl). Both LiCl and CsCl have been previously shown to display large differences in the required saturation concentration<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>; however, the mechanistic basis of this behaviour has remained elusive.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Ions are enriched in FUS condensates under high salt reentrant conditions to foster non-ionic interactions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365266_41467_2025_62437_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2e32/12365266/59871404404f/41467_2025_62437_Fig4_HTML.jpg" loading="lazy" id="d33e1363" height="861" width="698" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> FUS displays reentrant phase separation at high salt concentrations. <strong>b</strong> FUS dilute phase line scans at varying total LiCl (green) and CsCl (blue) concentrations. <strong>c</strong> Phase boundaries show that phase separation occurs only above the respective critical salt and FUS concentrations. <strong>d</strong> Comparison of ion partition (1/<em>K</em>) of LiCl and CsCl into FUS condensates. Preferential partitioning at high ionic strength drives enhancement of non-ionic interactions. Data are presented as mean ± SD, parameter errors are estimated by repeated perturbation on fitting data and quantifying the spread of best-fit parameters. <strong>e</strong> High salt FUS phase separation indicates extreme cases where phase separation is driven by only the limiting component. This leads to dominance values of 0 or 1 depending on the position on the phase boundary. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par22">We first characterised the change in dilute phase concentration of FUS-EGFP under varying concentrations of LiCl upwards of 4 M (Fig. <a href="#Fig4" class="usa-link">4b</a>). A rapid drop-off in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e1374"><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">FUS</mi></mrow></msubsup></math></span> at the onset of reentrant phase separation was observed at around 5 M LiCl (see Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a> for time traces), consistent with previous observations<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. We then determined the ion partitioning of LiCl from the dilute phase concentration changes to find an enrichment of LiCl in condensates in this reentrant regime. The drastic increase in ionic strength drives non-ionic interactions such as π-stacking or non-polar contacts as well as hydrophobic interactions driven by interfacial water release<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. This likely causes the observed recruitment of ions into the condensed phase, which is expected to further strengthen these non-ionic interactions (Fig. <a href="#Fig4" class="usa-link">4c</a>).</p>
<p id="Par23">Next, we characterised changes in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e1408"><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">FUS</mi></mrow></msubsup></math></span> as a function of CsCl concentrations (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>) to elucidate the impact of changing the cation from Li<sup>+</sup> to Cs<sup>+</sup> (see Fig. <a href="#Fig4" class="usa-link">4d</a>). Phase separation was already observed at approx. 3.8 M CsCl compared to a saturation concentration of around 5 M for LiCl. Therefore, Cs<sup>+</sup> is more potent at triggering reentrant phase separation of FUS, as also highlighted by the steeper drop-off in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e1438"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">FUS</mi></mrow></msubsup></math></span> with increasing CsCl concentration compared to LiCl (see Fig. <a href="#Fig4" class="usa-link">4b</a>). CsCl further partitions less into FUS condensates, indicating that fewer ions are required in the dense phase to allow for sufficient non-ionic interactions to trigger phase separation. This is further highlighted by the fact that the tie line gradient appears to be closely linked to the solubility of the salts individually with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e1459"><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">sat</mi></mrow><mrow><mi mathvariant="normal">LiCl</mi></mrow></msubsup></math></span> being 13.4 M (1/<em>K</em> = 143 mM/µM) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e1479"><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">sat</mi></mrow><mrow><mi mathvariant="normal">CsCl</mi></mrow></msubsup></math></span> being 5.9 M (1/<em>K</em> = 64 mM/µM) (according to supplier specification) as well as following the expected trend of the Hofmeister series<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>.</p>
<p id="Par24">The reentrant phase transition highlights extreme cases with regard to the energetics. At both tested protein concentrations (3 and 5 µM), the critical salt concentrations are largely similar. This yields a phase boundary gradient that approaches <em>P</em> → ∞, with that <em>D</em><sup>FUS</sup> → 0, meaning little free energy release is associated with the protein. This is because the system is entirely limited by the ion concentration, highlighting a critical stoichiometry that is necessary to trigger the formation of non-ionic interactions (Fig. <a href="#Fig4" class="usa-link">4e</a>). In addition, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e1516"><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">FUS</mi></mrow></msubsup></math></span> decreased to almost 0 µM protein concentration at high salt concentrations, suggesting that in this ‘salting out’ regime, the protein is almost completely sequestered into the condensed phase. Here, the phase boundary gradient approaches <em>P</em> → 0 and with it <em>D</em><sup>FUS</sup> → 1, suggesting that the system moves from an entirely salt-limited state to a protein-limited state (Fig. <a href="#Fig4" class="usa-link">4e</a>). This can be rationalised by the fact that the salt solubility in the absence of protein is significantly larger.</p>
<p id="Par25">Together, our experiments reveal that even simple ion species can display differential condensate partitioning, which can vary from enrichment to exclusion in a context-dependent manner.</p></section><section id="Sec6"><h3 class="pmc_sec_title">1,6-hexanediol alters the solvation properties of biomolecules to destabilise condensates</h3>
<p id="Par26">1,6-hexanediol (1,6-HD) is commonly used as a generic inhibitor of hydrophobic interactions, and applying it has become a powerful strategy for studying phase separation. While 1,6-HD is used to investigate the nature of interactions underlying condensate formation or to assess the liquidity of condensates<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>–<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, it remains largely unclear how the protein physicochemical properties and collective interactions are altered as a result.</p>
<p id="Par27">To shed light on the impact of 1,6-HD on collective interactions, we investigated FUS phase separation triggered by the addition of polyethylene glycol (PEG, 10 kDa) (Fig. <a href="#Fig5" class="usa-link">5a</a>). We first characterised the system in the absence of 1,6-HD for which line scans were performed at a total FUS concentration of 1 and 2 µM where the total PEG concentration was varied from 0–8 % (w/v) (Fig. <a href="#Fig5" class="usa-link">5a</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">10</a> and <a href="#MOESM1" class="usa-link">11</a>). In the absence of 1,6-HD, phase separation of FUS was observed already at a PEG concentration of 2–3% (w/v), see Fig. <a href="#Fig5" class="usa-link">5b</a>. A positive <em>K</em> was determined between FUS and PEG, indicating that PEG co-partitions into condensates, rather than acting as a traditional crowder. This is in agreement with previous observations of PEG co-phase separating with protein condensates<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> and suggests potential favourable protein-PEG interactions.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. 1,6-hexanediol (1,6-HD) decreases the phase separation propensity by modulating solvent quality.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365266_41467_2025_62437_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2e32/12365266/e81f530756c6/41467_2025_62437_Fig5_HTML.jpg" loading="lazy" id="d33e1616" height="515" width="799" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> To characterise the mechanism of action of 1,6-hexanediol, phase separation of FUS with PEG was studied in the presence (+) and absence (−) of the compound. <strong>b</strong> FUS dilute phase concentration changes with varying concentrations of polyethylene glycol (PEG) in the presence (+, green) and absence (−, blue) of 1,6-HD (7% (w/v)). <strong>c</strong> Phase boundaries for FUS against PEG in the absence and presence of 1,6-HD were created from changes in dilute phase concentrations and tie line gradients. <strong>d</strong> (left) Shift in PEG partitioning (1/K) in the absence and presence of 1,6-HD, indicating a change in condensate stoichiometry by decreasing PEG/protein ratio. (right) FUS dominance in the presence and absence of 1,6-HD. Data are presented as mean ± SD, parameter errors are estimated by repeated perturbation on fitting data and quantifying the spread of best-fit parameters. <strong>e</strong> Changes in hydrodynamic radius <em>R</em><sub>h</sub> as a function of 1,6-HD concentration. Shown are data for monomeric FUS-EGFP (dark green) and EGFP protein alone (light green) at 1 µM protein concentration each. Data are presented as mean ± SD from <em>n</em> = 3 repeats. <strong>f</strong> Change in hydrodynamic radius for a set of proteins with increasing 1,6-HD concentration. Data are presented as mean ± SD from <em>n</em> = 3 repeats. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par28">Next, we repeated this experiment in the presence of 1,6-HD, shifting the phase boundary towards higher concentrations of PEG (Fig. <a href="#Fig5" class="usa-link">5c</a>). The reduced tie line gradient between FUS and PEG was observed to decrease upon addition of 1,6-HD compared to the FUS and PEG system alone (Fig. <a href="#Fig5" class="usa-link">5d</a>). From the partitioning behaviour, the dense-phase stoichiometry of constituent components can be inferred<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. The results suggest that the presence of 1,6-HD decreases the number of PEG copolymers per FUS molecule by approximately half within condensates. Such a decrease in copolymer partitioning points towards a decrease in the interaction strength of PEG within the condensed phase. Dominance analysis shows that the free energy decrease associated with FUS does not change significantly, indicating a decrease in FUS self-interaction. This could be explained by 1,6-HD functioning as a solvation agent of the protein by decreasing the solvent polarity. The compositional changes can be traced back to the fact that FUS can also engage in electrostatic interactions, while PEG, with only ether functional groups, will be less capable of doing so. FUS, however, is not capable of fully compensating for these effects with its electrostatic interactions, which leads to dissolution.</p>
<p id="Par29">An increase in solvent interaction of the intrinsically disordered FUS protein would be expected to lead to an expansion of the polypeptide chain. Using microfluidic diffusional sizing<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, we determined the hydrodynamic radius <em>R</em><sub>h</sub> of monomeric FUS-EGFP at different concentrations of 1,6-HD. We find a significant increase in the protein hydrodynamic radius with increasing 1,6-HD concentrations (Fig. <a href="#Fig5" class="usa-link">5f</a>). The hydrodynamic radius of the folded EGFP protein alone, in comparison, displayed no change. This suggests that 1,6-HD acts as a solvation agent for the FUS disordered polypeptide chain that inhibits the intramolecular contacts driving protein compaction to favour solvent interactions, leading to an expansion of the protein size. Interestingly, this solvation effect is not limited to just the disordered protein. A similar expansion was also observed for the PEG copolymer (see Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>). This highlights that 1,6-HD effectively modulates the solvent quality for biomolecules to dissolve condensates.</p>
<p id="Par30">We then set out to investigate if the observed 1,6-HD polypeptide chain expansion effect also translates to other protein systems. We tested for 1,6-HD mediated size expansion of GFP tagged variants of TDP43, PGL3 and SOX2, phase-separating proteins known to be sensitive to 1,6-HD dissolution<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. In all cases, we observed an increase in the hydrodynamic radius upon addition of 1,6-HD (Fig. <a href="#Fig5" class="usa-link">5f</a>), suggesting that it more generally acts as a condensate dissolver by regulating the solvent-IDR interactions. We further sought to contrast the behaviour of these homotypic phase separation-prone proteins<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> with G3BP1, which is known for its RNA-mediated condensation<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. G3BP1 did not display such polypeptide chain expansion with increasing 1,6-HD concentration (see Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). This is expected as G3BP1, in the absence of RNA, adopts a self-inhibiting, compacted conformation driven by electrostatic interactions between oppositely charged intrinsically disordered regions (IDRs)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>.</p>
<p id="Par31">Overall, our data highlights that measuring dilute phase concentrations only can enable the precise evaluation of the mechanism of action of condensate modulators. In the case of 1,6-hexanediol, this reveals a hydrophobic disruption effect driven by changes to the effective protein-solvent interaction.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Application to complex environments</h3>
<p id="Par32">Lastly, we set out to translate our approach for the quantification of collective interactions to complex environments, such as <em>in cellulo</em>. Specifically, we can amend our dilute phase response function measurement approach to focus only on the protein<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> of interest by performing so-called homotypic line scans. Here, the concentration of the protein (component A) is varied while the concentrations of the other components i ≠ A are kept constant (Fig. <a href="#Fig6" class="usa-link">6a</a>). This allows us to quantify the energetics of phase separation without having to vary the concentration of an additional species, such as salt or copolymers. By plotting the total protein concentration against the dilute phase protein concentration, the homotypic response gradient (<em>R</em><sub>AA</sub>) can be determined. At concentrations below <em>c</em><sub>sat,</sub> the dilute phase protein concentration increases linearly with the total protein concentration with R<sub>AA</sub> = 1. Above <em>c</em><sub>sat</sub>, the protein is recruited into the dense phase, giving <em>R</em><sub>AA</sub> &lt; 1, where the dominance can be determined as (see “Derivation of the dominance framework” section):</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e1711" display="block"><msub><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msub><mo>=</mo><mn>1</mn><mo>−</mo><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">AA</mi></mrow></msub></math></td>
<td class="label">4</td>
</tr></table>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Intracellular dominance determination.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365266_41467_2025_62437_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2e32/12365266/e76a3b369cda/41467_2025_62437_Fig6_HTML.jpg" loading="lazy" id="d33e1766" height="220" width="800" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Homotypic line scans are performed by quantifying dilute phase concentrations of component A while varying the total concentration of component A (illustrated by yellow-purple colour gradient). <strong>b</strong> Homotypic response function of G3BP1 <em>in cellulo</em> from ref. <sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. <strong>c</strong> Comparison of homotypic response functions of opto-protein (opto-FUS … green, opto-FUS mutant 5Y &gt; S … light green, opto-G3BP1 … blue) fusion constructs intracellularly from refs. <sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. <strong>d</strong> Dominance fraction comparison of different proteins in (<strong>c</strong>). Data are presented as mean ± SD, parameter errors are estimated by repeated perturbation on fitting data and quantifying the spread of best-fit parameters. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par33">We then reanalysed previously published data from Riback et al.<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> where the intracellular dilute phase concentration post phase separation was quantified for G3BP1 at increasing expression levels (Fig. <a href="#Fig6" class="usa-link">6b</a>). This shows a response gradient <em>R</em><sub>AA</sub> of 0.26 ± 0.04, indicating that when observing increased G3BP1 concentrations, the dilute phase also rises. Hence, the added G3BP1 molecules are not just incorporated into the dense phase. This gives a free energy dominance fraction of ~74% indicating that other components also contribute to the free energy decrease.</p>
<p id="Par34">We then  turned to previously published data on opto-protein systems involving FUS and a tyrosine/serine mutation variant of FUS (5Y &gt; S) as well as G3BP1<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. In these opto-systems, as described by Riback et al.<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> and Wei et al.<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, the protein of interest is fused with Cry2, which can oligomerize in a light-inducible manner to trigger phase separation. To evaluate the intracellular dominance, we again plotted the dilute phase protein concentration against the total protein concentration for all protein systems (Fig. <a href="#Fig6" class="usa-link">6c</a>). From the resulting response gradient, we find that opto-FUS displays a dominance of approximately one. Hence, despite the increased complexity <em>in cellulo</em>, opto-FUS is essentially the sole driver of phase separation. The introduction of the additional interaction mode through Cry2 appears to modulate the overall collective interactions to cause opto-FUS to behave effectively as a one-component system.</p>
<p id="Par35">We then set out to quantify the effect of mutational changes to the protein sequence by performing dominance quantification for the 5Y &gt; S mutant. Opto-FUS 5Y &gt; S displays an increase in the protein <em>c</em><sub>sat</sub> as expected from the removal of the interaction-prone tyrosine residues. The dominance fraction of FUS 5Y &gt; S, however, remained approximately equal to one. Hence, while higher protein concentrations are required to yield phase separation for the 5Y &gt; S mutant, the energetic driving force still purely stems from homotypic opto-FUS interactions. This is in stark contrast with opto-G3BP1, where a sloped response gradient is observed. This indicates that the opto-G3BP1 system displays a multicomponent character, as a fraction of the excess protein is not directly converted to the dense phase. Despite the addition of the Cry2 motif, G3BP1 therefore, is still reliant on heterotypic interactions with, e.g., RNA.</p>
<p id="Par36">As such, using our measurement approach, protein-dependent energetics of condensation can be assessed directly <em>in cellulo</em> by determination of the dilute phase concentration of a single component only.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par37">Here, we have formulated a generic approach for the quantification of collective interactions in biomolecular phase separation. Our strategy captures both compositional and energetic details of biomolecular phase separation and requires measurement of dilute phase concentrations of a single component only (Fig. <a href="#Fig7" class="usa-link">7a</a>). We have discovered that protein phase separation at low ionic strength conditions commonly drives preferential exclusion of ions from condensates to decrease charge screening (Fig. <a href="#Fig7" class="usa-link">7b</a>). At very high salt concentrations an opposing trend is observed, where ions preferentially partition into condensates to enable non-ionic interactions (Fig. <a href="#Fig7" class="usa-link">7c</a>). Our findings also demonstrate that 1,6-HD disrupts condensates by functioning as a solvation agent that drives polypeptide chain expansion (Fig. <a href="#Fig7" class="usa-link">7d</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h3 class="obj_head">Fig. 7. Quantification of collective interactions enables deciphering the physicochemical driving forces of phase separation.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365266_41467_2025_62437_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2e32/12365266/99d3293eff67/41467_2025_62437_Fig7_HTML.jpg" loading="lazy" id="d33e1849" height="176" width="800" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Dilute phase concentration measurements of one component only can enable assessment of the energetics of phase separation processes. <strong>b</strong> For phase separation triggered by a decrease in ionic strength, an ion exclusion mechanism from the dense phase was observed across multiple proteins. <strong>c</strong> Ion partitioning displays opposing behaviours in phase separation at high and low ionic strength conditions. <strong>d</strong> 1,6-HD dissolution effects were observed to correlate with an increase in polypeptide chain expansion.</p></figcaption></figure><p id="Par38">Our investigations of protein phase behaviour at different ionic strength regimes highlight that even simple molecules can display distinct partitioning and stabilisation behaviour. Specifically, in the low salt regime, KCl acts to prevent  phase separation as it inhibits protein interactions by charge screening. Therefore, phase separation occurs upon decreasing the salt concentration, and it becomes favourable to exclude ions from the dense phase. Conversely, in the high salt regime, the increase of LiCl or CsCl salt concentration acts as the trigger for phase separation by driving non-ionic interactions, favouring ion enrichment. More generally, this might indicate that the partitioning of molecular species is largely a consequence of their propensity to enhance or counteract collective interactions.</p>
<p id="Par39">1,6-Hexanediol is widely used as a modulator of protein phase separation to probe physicochemical driving forces, dissolution effects, or material properties<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>–<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. In this context, 1,6-HD is generally recognised as a disruptor of hydrophobic interaction<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Our analysis reveals that this effect stems from 1,6-HD acting as a solvating agent for biomolecules. Specifically, 1,6-HD decreases intra- and intermolecular interactions by favouring solvent interactions, as corroborated by an expansion of the polypeptide chain. Our mechanistic study of 1,6-HD suggests that affecting the biomolecule's native state and chain expansion is a powerful avenue for modulating phase behaviour. Hence, screening for molecules capable of inducing expansion or compaction of intrinsically disordered proteins might be a simple but effective drug discovery strategy.</p>
<p id="Par40">The presented approach allows for the study of the behaviour of species, such as salt ions or small-molecule compounds, which would otherwise be inaccessible by conventional labelling or detection strategies. This makes the assay especially suited to applications in more complex systems and in cellulo, as the characterisation of the underlying driving forces can be performed by measuring the response of a single component only. The dilute phase centred strategy presents a number of important advantages, such as drastically decreasing material requirements, and dilute phase readouts can also be generated easily using commonly available measurement assays.</p>
<p id="Par41">Taken together, the quantification of collective interactions with the presented approach promises to be a powerful tool for studying phase separation in a wide range of contexts. We envision that the approach could be applied broadly to gain mechanistic insights to rationally design condensate modulators; dissect the impact of potential drug candidates; study the effect of biologically relevant molecular species on in vivo phase transitions; and optimise designer phase separation systems for functional applications.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Fabrication of microfluidic devices</h3>
<p id="Par42">Microfluidic devices were designed using AutoCAD software (Autodesk), followed by printing on acetate transparency masks (Micro Lithography Services). The replica master was obtained via standard soft-lithography steps using spin-coating of SU-8 photoresists (MicroChem) onto polished silicon wafers<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Typically, SU-8 3050 was applied to achieve a device height of approximately 100 µm. After UV exposure, using a custom-built LED-based apparatus<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, the precise heights of the features were measured using a profilometer (Dektak, Bruker). Devices were then produced in polydimethylsiloxane (PDMS). PDMS (Dow Corning) was mixed in a 10:1 (w/w) ratio with curing agent (Sylgard 184, Dow Corning) and poured onto the master, followed by degassing and baking for 1.5 h at 65 °C. The PDMS was then removed from the master and punched using biopsy punches to generate inlet holes, after which the slab was bonded onto thin glass slides using oxygen plasma surface activation (Diener electronic, 40% power for 30 s).</p></section><section id="Sec11"><h3 class="pmc_sec_title">Reagents</h3>
<p id="Par43">Potassium chloride (KCl), Lithium Chloride (LiCl), Caesium Chloride, polyethylene glycol (PEG, 10 kDa), Tween 20, Tris(hydroxymethyl)aminomethane (TRIS), and 1,6-hexanediol (1,6-HD) were purchased from Sigma–Aldrich (United Kingdom).</p></section><section id="Sec12"><h3 class="pmc_sec_title">Protein expression</h3>
<p id="Par44">EGFP-tagged fused in sarcoma (FUS) was expressed as reported previously in an insect cell expression system<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. After purification, the protein was stored at −80 °C in 50 mM TRIS buffer (pH 7.4), 500 mM KCl and at a total protein concentration of 70 µM. PGL3-6xHis-mEGFP was obtained from a previously published purification protocol<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. The protein was stored at −80 °C followed by aliquoting and flash freezing in liquid nitrogen. EGFP-tagged TDP43 was expressed as described previously<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> from Sf9 insect cells using a baculovirus system. The purified protein was then stored at −80 °C in TRIS buffer (pH 7.4, 50 mM), 500 mM KCl, 5% (<em>w</em>/<em>v</em>) glycerol and 1 mM DTT. Similarly, His<sub>6</sub>-MBP-Sox2-GFP was also expressed in Sf9 insect cells. Following purification, the protein was stored at −80 °C in Bis-Tris-Propane (pH 7.5, 50 mM), as well as 500 mM KCl, 1 mM DTT and 5% glycerol. G3BP1-6xHis-mEGFP was produced using a previously published protocol<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> and post-purification stored in TRIS buffer (pH 7.4, 50 mM) including 300 mM KCl, 1 mM DTT and 5% glycerol.</p></section><section id="Sec13"><h3 class="pmc_sec_title">EGFP synthesis and purification</h3>
<p id="Par45">pET28a vector harbouring the gene encoding His-tagged enhanced green fluorescent protein (EGFP) was purchased from GenScript. 40 mL of lysogeny broth (LB) medium supplemented with 36 µg/mL of kanamycin was inoculated with one colony of <em>E. coli</em> BL21 DE3 cells (New England Biolabs) harbouring the vector encoding EGFP. The cells were grown overnight at 37 °C and 180 rpm. The next day, 40 mL of fresh LB-kanamycin media was inoculated with 1 mL of overnight culture and grown for 2–3 h, until the optical density reached ~0.5 optical units. Then, the cell culture was supplemented with 1 mM of isopropyl-beta-D-thiogalactopyranoside (IPTG, Sigma–Aldrich) and grown for an additional 3 h. The cells were harvested by centrifugation at 5000 × <em>g</em> and 4 °C for 10 min. Harvested cells were resuspended in lysis buffer (1 mg/mL lysozyme, 10 mM imidazole in PBS (pH 7.4)) and incubated on ice for 30 min. Subsequently, cells were sonicated on ice using 6 × 10 s on/pause cycles (at 42% power cycle). Sonicated cells were centrifuged for 30 min at 10.000 × <em>g</em> and 4 °C. The supernatant was collected and loaded onto 1 mL of equilibrated Ni-NTA slurry in a BioRad gravity flow column. The column was placed on a rocker and incubated in a cold room at 4 °C for 60 min. Subsequently, the column was placed in a ring stand. After the beads settled, the flow-through was allowed to run, then the column was washed with 4 × 1 mL of wash buffer solution (20 mM imidazole in PBS, pH 7.4). Subsequently, EGFP was eluted using 2 mL of elution buffer (250 mM imidazole in PBS, pH 7.4). The collected EGFP solution was buffer exchanged to 50 mM TRIS buffer (pH 7.4), 150 mM KCl using 10 kDa cut-off centrifugal filters (Millipore).</p></section><section id="Sec14"><h3 class="pmc_sec_title">Sample preparation and phase separation conditions</h3>
<p id="Par46">Phase separation was induced by gently mixing the individual components in Eppendorf tubes. Briefly, in a first step, buffer was added to the Eppendorf tube. The buffer was 50 mM TRIS (pH 7.4) supplemented with 0.01% (v/v) Tween 20 in all cases. For FUS/KCl experiments, KCl stock solution (150 mM KCl in 50 mM TRIS (pH 7.4)) was then added to achieve the desired total KCl concentration, also considering the amount of KCl introduced from the protein stock. This was followed by the addition of FUS from the protein stock (70 µM FUS-GFP in 50 mM TRIS buffer (pH 7.4), 500 mM KCl). For the FUS/KGlu experiments, the KCl concentration was kept constant at 40 mM, also considering the amount from protein addition later on. Then, KGlu stock solution (1000 mM KGlu in 50 mM TRIS, pH 7.4) was added to achieve the desired KGlu concentration prior to the addition of FUS stock solution. For FUS/1,6-hexanediol and GUG aptamer experiments, 150 mM KCl and 15% (w/v) PEG (in 50 mM TRIS (pH 7.4), 150 mM KCl) were added to the buffer. Then, either 1,6-hexanediol (50% (w/v) in 50 mM TRIS (pH 7.4), 150 mM KCl) or GUG aptamer (100 µM in 50 mM TRIS (pH 7.4), 150 mM KCl) were added, followed by the addition of pre-diluted FUS stock (5 µM FUS-GFP in 50 mM TRIS buffer (pH 7.4), 150 mM KCl). This yielded solutions containing 5% (w/v) PEG with 7% (w/v) 1,6-hexanediol in 50 mM TRIS buffer (pH 7.4), 150 mM KCl in the case of the 1,6-hexanediol experiment series or varying GUG concentrations between 1 and 1000 nM in GUG aptamer experiments. In all cases, typically 10 µL of sample was prepared and final FUS concentrations were 1 or 2 µM.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Dilute phase concentration measurements using confocal detection</h3>
<p id="Par47">Samples were prepared as described above by mixing first off-chip and incubating for 5 min prior to the experiment. Microfluidic chips were filled with sample buffer followed by flushing the sample for 5 min at 100 µL/h to ensure equilibration of the channel (see Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>). To operate flow control through the microfluidic channel, negative pressure created by a glass syringe (Hamilton, Switzerland) and syringe pump (neMESYS, Cetoni, Germany) was applied. Thereby, channel loading was controlled by an inlet reservoir containing buffer or sample that was connected to the channel inlet. Following equilibration, dilute phase concentrations were measured using a home-built confocal setup. Differences between devices utilised were minimal compared to the baseline noise level (see Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). The setup is equipped with picosecond-pulsed 485-nm and 640-nm laser sources, a 60× water-immersion objective (CFI Plan Apochromat WI 60×, NA 1.2, Nikon), and an automated sample stage onto which the microfluidic device can be mounted. An illustration of the setup can be found in Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>. Following laser excitation of the sample through the objective in a diffraction-limited spot and collection of fluorescence through the same objective, emitted photons were registered via avalanche photodiodes and recorded using a time-correlated single-photon counting unit (HydraHarp400, Picoquant). Laser synchronisation was done via a laser controller (Sepia PDL 828, Picoquant). The setup is further equipped with a lens-pinhole-lens system to remove out-of-focus light and with appropriate dichroic and bandpass filter sets to spectrally separate photons into blue- and red-emitting channels. The two lasers were run in pulsed-interleaved excitation (PIE) mode, driven by a synchronisation signal from the laser driver at a frequency of 25 MHz. Recording of data was done in T3 mode. Recorded macroscopic times <em>T</em> were binned into 1-ms time intervals to give an intensity readout in units of number of photons per second. Signal collection from samples was typically performed for 30 s with collection times as low as 0.1 s being sufficient to calculate tie line gradients (see Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>). The intensity time traces generated in this manner typically display a stable baseline (see Fig. <a href="#Fig2" class="usa-link">2b</a>), which corresponds to the dilute phase concentrations, as the volume fraction of the dense phase can be assumed to be significantly smaller than the dilute phase. The baseline value can then be extracted from the maximum in the intensity histogram (see Fig. <a href="#Fig2" class="usa-link">2c</a>). The standard deviation of the baseline signal can be determined from numerical fitting to the lower intensity branch of the histogram. The higher intensity branch will show condensed species, as condensates exhibit high protein concentrations. Concentration conversion of the baseline intensities was performed by calibration via quadratic fitting to the intensity recorded of samples at different, known FUS concentrations under non-phase-separated conditions.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Fluorescence lifetime measurements</h3>
<p id="Par48">Fluorescence lifetime decays were calculated by generating a decay time histogram from a large number of photons. This was achieved by calculating the time difference between the APD signal and the last synchronisation event for each recorded photon. To estimate the lifetime of GFP in the dense phase, photons that correspond to the GFP fluorophore in the condensates are selected by applying a high-pass filter to the intensity readout previously computed by binning the macroscopic time <em>T</em> in 1 ms intervals. More specifically, photons with a macroscopic time <em>T</em> that fall in the high-intensity time window are identified as signal from condensates, and the <em>t</em> values of all these photons are binned to give a single fluorescent decay histogram. To extract the lifetime, we truncated the histogram away from the instrument response function (IRF) and fitted an exponential decay function, whereas the inverse of the decay constant corresponds to the lifetime.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Microfluidic diffusional sizing</h3>
<p id="Par49">Microfluidic diffusional sizing was performed as described elsewhere<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Briefly, the Fluidity One-M instrument (Fluidic Analytics) was utilised for measurements. Auxiliary channels were first primed with 50 mM TRIS buffer (pH 7.4), 150 mM KCl. Subsequently, 3.5 µL of sample with 1 µM protein concentration and specified concentrations of 1,6-hexanediol were added. Each sample was measured three times. Detection was set for Alexa488 fluorescence detection, and size-range settings of <em>R</em><sub>H</sub> = 2–9 nm were applied on the Fluidity One-M instrument.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Derivation of the dominance framework</h3>
<p id="Par50">We assign <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e1996"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msubsup></math></span> as the total concentration of solutes i = A, B,…, N, while dense and dilute phase concentrations are given as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e2014"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">den</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msubsup></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e2032"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msubsup></math></span>, respectively. The phase boundary is then given as a dense and dilute (N-1)-dimensional surface provided by the set of concentrations of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e2050"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">den</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msubsup></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e2068"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msubsup></math></span>. Mass balance demands conservation of solutes, leading to the definition of the tie line vector:</p>
<table class="disp-formula p" id="Equb"><tr><td class="formula"><math id="d33e2087" display="block"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msubsup><mo>=</mo><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msubsup><mo>+</mo><mi mathvariant="normal">v</mi><mo>×</mo><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msup></math></td></tr></table>
<p>where <em>v</em> is the dense phase volume fraction and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e2130"><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msup></math></span> = <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e2145"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">den</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msubsup></math></span> - <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e2163"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msubsup></math></span>.</p>
<p id="Par52">Assuming a binary phase separation system, a fixed volume of particles and considering only small dense volume fractions, i.e. probing in close proximity to the dilute phase branch of the phase boundary, we can compute the energy difference between the homogeneous and phase-separated state Δ<em>f</em> as<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>:</p>
<table class="disp-formula p" id="Equc"><tr><td class="formula"><math id="d33e2190" display="block"><mi mathvariant="normal">Δ</mi><mi mathvariant="normal">f</mi><mo>=</mo><mo>−</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac><munderover accent="false" accentunder="false"><mrow><mo>∑</mo></mrow><mrow><mi mathvariant="normal">i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">N</mi></mrow></munderover><munderover accent="false" accentunder="false"><mrow><mo>∑</mo></mrow><mrow><mi mathvariant="normal">j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">N</mi></mrow></munderover><mi mathvariant="normal">v</mi><mo>×</mo><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msup><mi mathvariant="normal">v</mi><mo>×</mo><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">j</mi></mrow></msup><msub><mrow><mi>∂</mi></mrow><mrow><mi mathvariant="normal">j</mi></mrow></msub><msub><mrow><mi mathvariant="normal">μ</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msub><mfenced close=")" open="("><mrow><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow></msub></mrow></mfenced><mo>+</mo><mi mathvariant="normal">O</mi><mfenced close=")" open="("><mrow><msup><mrow><mi mathvariant="normal">v</mi></mrow><mrow><mn>3</mn></mrow></msup></mrow></mfenced></math></td></tr></table>
<p id="Par53">Here, μ<sub>i</sub> is the chemical potential as μ<sub>i</sub>(c<sub>tot</sub>) and as can be inferred from the definition of the tie line gradient <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e2292"><mi mathvariant="normal">v</mi><mo>×</mo><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msup></math></span> as the distance between the total and dilute phase composition. To access the solute species-specific difference Δ<em>f</em> <sup>i</sup> we consider the sum over <em>j</em> to find:</p>
<table class="disp-formula p" id="Equd"><tr><td class="formula"><math id="d33e2319" display="block"><mi mathvariant="normal">Δ</mi><msup><mrow><mi mathvariant="normal">f</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msup><mo>≡</mo><mo>−</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac><mi mathvariant="normal">v</mi><mo>×</mo><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msup><munderover accent="false" accentunder="false"><mrow><mo>∑</mo></mrow><mrow><mi mathvariant="normal">j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">N</mi></mrow></munderover><mi mathvariant="normal">v</mi><mo>×</mo><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">j</mi></mrow></msup><msub><mrow><mi>∂</mi></mrow><mrow><mi mathvariant="normal">j</mi></mrow></msub><msub><mrow><mi mathvariant="normal">μ</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msub><mfenced close=")" open="("><mrow><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow></msub></mrow></mfenced></math></td></tr></table>
<p id="Par54">At the onset of phase separation, i.e., v →,0, this becomes:</p>
<table class="disp-formula p" id="Eque"><tr><td class="formula"><math id="d33e2398" display="block"><msub><mrow><mi mathvariant="normal">lim</mi></mrow><mrow><mi mathvariant="normal">v</mi><mo>→</mo><mn>0</mn></mrow></msub><mfrac><mrow><mi mathvariant="normal">Δ</mi><msup><mrow><mi mathvariant="normal">f</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msup></mrow><mrow><mi mathvariant="normal">Δ</mi><mi mathvariant="normal">f</mi></mrow></mfrac><mo>=</mo><mfrac><mrow><msubsup><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi mathvariant="normal">j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">N</mi></mrow></msubsup><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msup><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">j</mi></mrow></msup><mrow><mo>(</mo><mrow><msub><mrow><mi>∂</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msub><msub><mrow><mi>∂</mi></mrow><mrow><mi mathvariant="normal">j</mi></mrow></msub><mi mathvariant="normal">f</mi></mrow><mo>)</mo></mrow><msub><mrow><mo>∣</mo></mrow><mrow><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow></msub></mrow></msub></mrow><mrow><msubsup><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi mathvariant="normal">l</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">N</mi></mrow></msubsup><msubsup><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi mathvariant="normal">n</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">N</mi></mrow></msubsup><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">l</mi></mrow></msup><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">n</mi></mrow></msup><mfenced close=")" open="("><mrow><msub><mrow><mi>∂</mi></mrow><mrow><mi mathvariant="normal">l</mi></mrow></msub><msub><mrow><mi>∂</mi></mrow><mrow><mi mathvariant="normal">n</mi></mrow></msub><mi mathvariant="normal">f</mi></mrow></mfenced><msub><mrow><mo>∣</mo></mrow><mrow><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow></msub></mrow></msub></mrow></mfrac><mo>≡</mo><msup><mrow><mi mathvariant="normal">D</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msup></math></td></tr></table>
<p>yielding the dominance of species i. To connect this to accessible quantities, we first introduce the dilute phase boundary normal vector: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e2563"><msubsup><mrow><mo>∑</mo></mrow><mrow><mi mathvariant="normal">i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">N</mi></mrow></msubsup><msub><mrow><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msub><mi mathvariant="normal">δ</mi><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msubsup><mo>=</mo><mn>0</mn></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e2603"><mi mathvariant="normal">δ</mi><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msubsup></math></span> a vector on the dilute phase boundary branch. Applying equilibrium perturbations to obtain <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e2623"><msub><mrow><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msub><mo>∝</mo><msubsup><mrow><mo>∑</mo></mrow><mrow><mi mathvariant="normal">j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">N</mi></mrow></msubsup><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">j</mi></mrow></msup><mrow><mo>(</mo><mrow><msub><mrow><mi>∂</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msub><msub><mrow><mi>∂</mi></mrow><mrow><mi mathvariant="normal">j</mi></mrow></msub><mi mathvariant="normal">f</mi></mrow><mo>)</mo></mrow><msub><mrow><mo>∣</mo></mrow><mrow><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow></msub></mrow></msub></math></span><sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, we can rewrite the dominance as:</p>
<table class="disp-formula p" id="Equf"><tr><td class="formula"><math id="d33e2689" display="block"><msup><mrow><mi mathvariant="normal">D</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msup><mo>=</mo><mfrac><mrow><msub><mrow><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msub><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msup></mrow><mrow><msub><mrow><msubsup><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi mathvariant="normal">l</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">N</mi></mrow></msubsup><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">l</mi></mrow></msub><msup><mrow><mi mathvariant="normal">k</mi></mrow><mrow><mi mathvariant="normal">l</mi></mrow></msup></mrow></mfrac></math></td></tr></table>
<p id="Par55">Further, quantification of all species concentrations is typically inaccessible. This leads us to focus on the quantification of the dilute phase concentration of a selected species, which can be quantified experimentally using simple separation of the dense phase from the dilute phase or application of confocal techniques<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Hence, we focus on the variation of the dilute phase concentration of an individual species i, upon applying perturbations of the total concentration of another species j to define the response function</p>
<table class="disp-formula p" id="Equg"><tr><td class="formula"><math id="d33e2756" display="block"><msub><mrow><mi mathvariant="normal">R</mi></mrow><mrow><mi mathvariant="normal">ij</mi></mrow></msub><mo>≡</mo><mfrac><mrow><mi>∂</mi><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msubsup></mrow><mrow><mi>∂</mi><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">j</mi></mrow></msubsup></mrow></mfrac></math></td></tr></table>
<p>which can be rewritten by applying mass balance to give<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>:</p>
<table class="disp-formula p" id="Equh"><tr><td class="formula"><math id="d33e2801" display="block"><msub><mrow><mi mathvariant="normal">lim</mi></mrow><mrow><mi mathvariant="normal">v</mi><mo>→</mo><mn>0</mn></mrow></msub><msub><mrow><mi mathvariant="normal">R</mi></mrow><mrow><mi mathvariant="normal">ij</mi></mrow></msub><mo>=</mo><msub><mrow><mi mathvariant="normal">δ</mi></mrow><mrow><mi mathvariant="normal">ij</mi></mrow></msub><mo>−</mo><mfrac><mrow><msub><mrow><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">j</mi></mrow></msub></mrow><mrow><msub><mrow><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msub></mrow></mfrac><msup><mrow><mi mathvariant="normal">D</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msup></math></td></tr></table>
<p>with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e2858"><msub><mrow><mi mathvariant="normal">δ</mi></mrow><mrow><mi mathvariant="normal">ij</mi></mrow></msub></math></span> the Kronecker-Delta. Now, in a specific example involving the primary species A and B, we can then further look to identify a set of dilute phase concentrations of component A that do not change <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e2874"><mfenced close=")" open="("><mrow><mi mathvariant="normal">δ</mi><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup><mo>=</mo><mn>0</mn></mrow></mfenced></math></span> while the total concentration of another solute species B is varied. Furthermore, we consider that the concentration of all other species is kept constant, i.e., <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e2898"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">j</mi></mrow></msubsup><mo>=</mo><mn>0</mn></math></span> for j other than A or B. Considering the definition of the response function, this becomes <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><math id="d33e2918"><mi mathvariant="normal">δ</mi><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup><mo>=</mo><msub><mrow><mi mathvariant="normal">R</mi></mrow><mrow><mi mathvariant="normal">AA</mi></mrow></msub><mo>×</mo><mi mathvariant="normal">δ</mi><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup><mo>+</mo><msub><mrow><mi mathvariant="normal">R</mi></mrow><mrow><mi mathvariant="normal">AB</mi></mrow></msub><mo>×</mo><mi mathvariant="normal">δ</mi><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span> since all other components <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><math id="d33e2981"><mi mathvariant="normal">δ</mi><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">j</mi></mrow></msubsup></math></span> = 0 for j ≠ A,B. This allows us to introduce the 2-D reduced tie line gradient:</p>
<table class="disp-formula p" id="Equi"><tr><td class="formula"><math id="d33e3001" display="block"><mi mathvariant="normal">K</mi><mo>≡</mo><msub><mrow><mfenced close=")" open="("><mrow><mfrac><mrow><mi>∂</mi><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></mrow><mrow><mi>∂</mi><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></mrow></mfrac></mrow></mfenced></mrow><mrow><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></mrow></msub><mo>=</mo><mo>−</mo><mfrac><mrow><msub><mrow><mi mathvariant="normal">R</mi></mrow><mrow><mi mathvariant="normal">AB</mi></mrow></msub></mrow><mrow><msub><mrow><mi mathvariant="normal">R</mi></mrow><mrow><mi mathvariant="normal">AA</mi></mrow></msub></mrow></mfrac></math></td></tr></table>
<p>which describes the line defined by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><math id="d33e3071"><mi mathvariant="normal">δ</mi><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup><mo>=</mo><mn>0</mn></math></span> in the A-B plane. Again, considering only investigations at the onset of phase separation, we can use the previous relation between <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><math id="d33e3094"><msub><mrow><mi mathvariant="normal">R</mi></mrow><mrow><mi mathvariant="normal">ij</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><math id="d33e3109"><msup><mrow><mi mathvariant="normal">D</mi></mrow><mrow><mi mathvariant="normal">i</mi></mrow></msup></math></span> to give:</p>
<table class="disp-formula p" id="Equj"><tr><td class="formula"><math id="d33e3124" display="block"><msub><mrow><mi mathvariant="normal">lim</mi></mrow><mrow><mi mathvariant="normal">v</mi><mo>→</mo><mn>0</mn></mrow></msub><mi mathvariant="normal">K</mi><mo>=</mo><mfrac><mrow><msub><mrow><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msub></mrow><mrow><msub><mrow><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msub></mrow></mfrac><mfrac><mrow><msup><mrow><mi mathvariant="normal">D</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup></mrow><mrow><mn>1</mn><mo>−</mo><msup><mrow><mi mathvariant="normal">D</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup></mrow></mfrac></math></td></tr></table>
<p id="Par56">Now defining P as the phase boundary gradient vector in the A-B plane, which is orthogonal to the normal vector <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><math id="d33e3181"><mfenced close=")" open="("><mrow><mi mathvariant="normal">P</mi><mo>=</mo><mo>−</mo><mfrac><mrow><msub><mrow><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msub></mrow><mrow><msub><mrow><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msub></mrow></mfrac></mrow></mfenced></math></span> and staying close to the phase boundary, we obtain maintext Eq. <a href="#Equ1" class="usa-link">1</a>:</p>
<table class="disp-formula p" id="Equk"><tr><td class="formula"><math id="d33e3215" display="block"><msup><mrow><mi mathvariant="normal">D</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup><mo>=</mo><mfrac><mrow><mi mathvariant="normal">Δ</mi><msup><mrow><mi mathvariant="normal">f</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup></mrow><mrow><mi mathvariant="normal">Δ</mi><mi mathvariant="normal">f</mi></mrow></mfrac><mo>=</mo><mfrac><mrow><mi mathvariant="normal">K</mi></mrow><mrow><mi mathvariant="normal">K</mi><mo>−</mo><mi mathvariant="normal">P</mi></mrow></mfrac></math></td></tr></table></section><section id="Sec19"><h3 class="pmc_sec_title">Experimental characterisation of response and phase boundary gradient</h3>
<p id="Par57">The dilute phase response gradient R<sub>AB</sub> is determined directly from dilute phase line scan data. Line scans correspond to the change in dilute phase concentration of component A <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><math id="d33e3263"><mrow><mo>(</mo><mrow><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></mrow><mo>)</mo></mrow></math></span> at constant total concentration of component A (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><math id="d33e3285"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span>= const.) but varying total concentration of component B (vary <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><math id="d33e3303"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span>). Here, the concentration of component B is varied from outside the two-phase coexistence region to well within the phase-separated regime. This is repeated at two separate total concentrations of component A <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><math id="d33e3321"><mrow><mo>(</mo><mrow><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi><mo>,</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup><mo>,</mo><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi><mo>,</mo><mn>2</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></mrow><mo>)</mo></mrow></math></span>) to give two line scans. Both line scans are then individually fitted in the experimentally probed concentration range to give a function <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><math id="d33e3359"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup><mo>=</mo><mi mathvariant="normal">f</mi><mrow><mo>(</mo><mrow><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></mrow><mo>)</mo></mrow></math></span>. In the case of FUS and KCl (A = FUS, B = KCl), for example, the data shows a linear response to changes in KCl; therefore, the following form was chosen for phenomenological fitting to both line scans at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><math id="d33e3394"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi><mo>,</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup><mi mathvariant="normal">and</mi><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi><mo>,</mo><mn>2</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span>.</p>
<table class="disp-formula p" id="Equl"><tr><td class="formula"><math id="d33e3428" display="block"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup><mo>=</mo><mfenced open="{"><mrow><mtable><mtr><mtd columnalign="center"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup><mo>,</mo><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup><mo>&gt;</mo><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">sat</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></mtd></mtr><mtr><mtd columnalign="center"><msubsup><mrow><mi mathvariant="normal">a</mi><mo>+</mo><mi mathvariant="normal">b</mi><mo>*</mo><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup><mo>,</mo><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup><mo>&lt;</mo><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">sat</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></mtd></mtr></mtable></mrow></mfenced></math></td></tr></table>
<p id="Par58">In doing so, a, b and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><math id="d33e3525"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">sat</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span> can be determined for both line scans. For this particular form of the dilute phase response, the response gradient <em>R</em><sub>AB</sub> is simply given as the slope of the line scan fits at the phase boundary (e.g., R<sub>AB</sub> = (b<sub>1</sub> + b<sub>2</sub>)/2, if the slope changes between line scans). The dilute phase boundary gradient P is then determined as follows:</p>
<table class="disp-formula p" id="Equm"><tr><td class="formula"><math id="d33e3554" display="block"><mi>P</mi><mo>=</mo><mfrac><mrow><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">tot</mi><mo>,</mo><mn>2</mn></mrow><mrow><mi>A</mi></mrow></msubsup><mo>−</mo><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">tot</mi><mo>,</mo><mn>1</mn></mrow><mrow><mi>A</mi></mrow></msubsup></mrow><mrow><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">sat</mi><mo>,</mo><mn>2</mn></mrow><mrow><mi>B</mi></mrow></msubsup><mo>−</mo><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">sat</mi><mo>,</mo><mn>1</mn></mrow><mrow><mi>B</mi></mrow></msubsup></mrow></mfrac><mo>=</mo><mfrac><mrow><mi mathvariant="normal">Δ</mi><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi>A</mi></mrow></msubsup></mrow><mrow><mi mathvariant="normal">Δ</mi><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">sat</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></mrow></mfrac></math></td></tr></table>
<p id="Par59">With R and P in hand, the reduced tie line gradient K, as well as the component A dominance D<sup>A</sup> is calculated:</p>
<table class="disp-formula p" id="Equn"><tr><td class="formula"><math id="d33e3637" display="block"><msup><mrow><mi mathvariant="normal">D</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup><mo>=</mo><mfrac><mrow><mi mathvariant="normal">Δ</mi><msup><mrow><mi mathvariant="normal">f</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup></mrow><mrow><mi mathvariant="normal">Δ</mi><mi mathvariant="normal">f</mi></mrow></mfrac><mo>=</mo><mfrac><mrow><mi mathvariant="normal">K</mi></mrow><mrow><mi mathvariant="normal">K</mi><mo>−</mo><mi mathvariant="normal">P</mi></mrow></mfrac><mi mathvariant="normal">and</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">K</mi><mo>=</mo><mfrac><mrow><msub><mrow><mi mathvariant="normal">R</mi></mrow><mrow><mi mathvariant="normal">AB</mi></mrow></msub><mo>*</mo><mi mathvariant="normal">P</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">R</mi></mrow><mrow><mi mathvariant="normal">AB</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">P</mi></mrow></mfrac></math></td></tr></table></section><section id="Sec20"><h3 class="pmc_sec_title">Phase diagram extrapolation</h3>
<p id="Par60">The phase boundary represents the set of conditions of dilute (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><math id="d33e3713"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><math id="d33e3731"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span>) and dense (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><math id="d33e3749"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">den</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><math id="d33e3767"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">den</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span>) phase concentrations formed as a result of the demixing of a set of total concentration conditions (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><math id="d33e3785"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><math id="d33e3804"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span>) within the two-phase coexistence region. Here, the dilute phase part of the phase boundary can be extrapolated from component A dilute phase response function data only, as reported previously<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Briefly, for a specific condition (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><math id="d33e3826"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi><mo>,</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><math id="d33e3846"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi><mo>,</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span>) the experimentally determined dilute phase line scans provide the corresponding dilute phase concentration <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><math id="d33e3866"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi><mo>,</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span>. The dilute phase concentration of component B <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><math id="d33e3886"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi><mo>,</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span> can then be determined by following the tie line (with reduced tie line gradient K) originating in (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><math id="d33e3907"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi><mo>,</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><math id="d33e3927"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi><mo>,</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span>) until <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><math id="d33e3947"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi><mo>,</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span> (see Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). Thereby, the point on the dilute phase branch of the phase boundary corresponding to the demixing of (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><math id="d33e3970"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi><mo>,</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><math id="d33e3990"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi><mo>,</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span>) can be simply written as:</p>
<table class="disp-formula p" id="Equo"><tr><td class="formula"><math id="d33e4011" display="block"><mrow><mo>(</mo><mrow><mi mathvariant="normal">x</mi><mo>,</mo><mi mathvariant="normal">y</mi></mrow><mo>)</mo></mrow><mo>=</mo><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow></msub><mo>,</mo><msup><mrow><mn>1</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup><mo>,</mo><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow></msub><mo>,</mo><msup><mrow><mn>1</mn></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msup></mrow><mo>)</mo></mrow><mo>=</mo><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow></msub><mo>,</mo><msup><mrow><mn>1</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup><mo>,</mo><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow></msub><mo>,</mo><msup><mrow><mn>1</mn></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msup><mo>−</mo><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow></msub><mo>,</mo><msup><mrow><mn>1</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup><mo>−</mo><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow></msub><mo>,</mo><msup><mrow><mn>1</mn></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup></mrow><mo>)</mo></mrow><mo>/</mo><mi mathvariant="normal">K</mi></mrow><mo>)</mo></mrow></math></td></tr></table>
<p id="Par61">By assuming a constant tie line gradient and employing previously mentioned phenomenological fitting for the line scans, this can be repeated for any (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><math id="d33e4134"><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow></msub><mo>,</mo><msup><mrow><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><math id="d33e4157"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">tot</mi><mo>,</mo><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span>) within the experimentally characterised concentration range to determine the corresponding phase boundary condition (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><math id="d33e4178"><msub><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi></mrow></msub><mo>,</mo><msup><mrow><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msup></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><math id="d33e4201"><msubsup><mrow><mi mathvariant="normal">c</mi></mrow><mrow><mi mathvariant="normal">dil</mi><mo>,</mo><mi mathvariant="normal">n</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msubsup></math></span>).</p></section><section id="Sec21"><h3 class="pmc_sec_title">2-D tie line reduction</h3>
<p id="Par62">A reduced tie line is by definition a collection of points on the 2-D experimental phase diagram that have the same dilute phase component A concentration (see Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). When the full phase space is 2-D, the phase boundary and tie lines all lie on a plane, and near the dilute phase boundary points on each tie line all have the same dilute phase A concentration. As such, in the 2-D scenario, the reduced tie line constructed using component A dilute phase concentration is equivalent to the higher-dimensional tie line, and is also equivalent to the reduced tie line constructed using dilute phase B concentration. On the other hand, when the full phase space is 3-D, the phase boundary itself becomes a 2-D surface. The collection of points with the same component A dilute phase concentration constitutes another surface, since 3 (dimensions) − 1 (constraint) = 2 (degrees of freedom). This surface is a flat plane perpendicular to the component A concentration axis in the mixed region, where the dilute phase concentration is equivalent to the total concentration. When propagating this surface into the phase-separated region, one has to bear in mind that many tie lines in the 3-D space share the same dilute phase A concentration, and these tie lines can be identified by the intersection between the trivial flat plane at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><math id="d33e4229"><msubsup><mrow><mi>c</mi></mrow><mrow><mi mathvariant="normal">tot</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msubsup></math></span> = const. and the 2-D phase boundary. We identify the tie lines of interest as the ones with one end on this intersection curve, and stitching them together produces a 2-D surface that divides the phase-separating region into two parts. This newly generated surface can be thought of as a tie plane with respect to component A, and it is, in general, curved. When line scans are performed in a 2-D experimental cross-section of the full 3-D space, this cross-section intersects the tie plane, and this intersection is the reduced tie line measured here. Hence, the reduced tie line depends on both tie lines and the phase boundary itself when more than 2 components are present.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par63">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365266/bin/41467_2025_62437_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365266/bin/41467_2025_62437_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (151.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365266/bin/41467_2025_62437_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.4MB, pdf) </sup>
</div></div></section></section><section id="Sec24"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12365266/bin/41467_2025_62437_MOESM4_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (10.5MB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The research leading to these results has received funding from Global Research Technologies, Novo Nordisk A/S (H.A., T.P.J.K.), the European Research Council under the European Union’s Horizon 2020 Framework Programme through the Marie Sklodowska-Curie grant MicroSPARK (agreement no. 841466; G.K.), the Herchel Smith Funds (G.K.), the Alzheimer's Association Zenith Award (P.S.G.-H.) and the Wolfson College Junior Research Fellowship (G.K.), and the Frances and Augustus Newman Foundation (E.d.C., T.S.). T.J.W. thanks the Harding Distinguished Postgraduate Scholar Programme.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>H.A., D.Q., G.K., T.J.W. and T.P.J.K. designed and conceptualised the study. H.A. and E.d.C. performed experiments. G.K., T.J.W., T.S., T.M.F., S.Q., N.A.E., S.A., A.A.H., P.S.G.-H. and T.P.J.K. provided materials and methods. H.A. and D.Q. analysed the data. H.A., D.Q., E.d.C., G.K., T.J.W., J.N.A., M.K., R.V.P. and T.P.J.K. interpreted data. H.A., G.K. and T.P.J.K. wrote the original draft of the paper. All authors discussed the results, commented on the manuscript, and contributed to the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par64"><em>Nature Communications</em> thanks Vijayaraghavan Rangachari and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Source data are provided with this paper, including the raw data dilute phase concentration data in the associated Supplementary Figs. <a href="#Sec24" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>Exemplary analysis code for the determination of tie line gradients is provided with the source data. Additional analysis steps have been described in detail in the “Methods” section.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par65">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Hannes Ausserwöger, Ella de Csilléry, Daoyuan Qian, Georg Krainer.</p></div>
</div></section><section id="sec25"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62437-y.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Gallie, D. R. Protein-protein interactions required during translation. <em>Plant Mol. Biol.</em><strong>50</strong>, 949–970 (2002).
</cite> [<a href="https://doi.org/10.1023/a:1021220910664" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12516864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gallie,%20D.%20R.%20Protein-protein%20interactions%20required%20during%20translation.%20Plant%20Mol.%20Biol.50,%20949%E2%80%93970%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Guarracino, D. A., Bullock, B. N. &amp; Arora, P. S. Mini review: protein-protein interactions in transcription: a fertile ground for helix mimetics. <em>Biopolymers</em><strong>95</strong>, 1–7 (2011).
</cite> [<a href="https://doi.org/10.1002/bip.21546" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2966508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20882600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guarracino,%20D.%20A.,%20Bullock,%20B.%20N.%20&amp;%20Arora,%20P.%20S.%20Mini%20review:%20protein-protein%20interactions%20in%20transcription:%20a%20fertile%20ground%20for%20helix%20mimetics.%20Biopolymers95,%201%E2%80%937%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Pawson, T. &amp; Nash, P. Assembly of cell regulatory systems through protein interaction domains. <em>Science</em><strong>300</strong>, 445–452 (2003).
</cite> [<a href="https://doi.org/10.1126/science.1083653" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12702867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pawson,%20T.%20&amp;%20Nash,%20P.%20Assembly%20of%20cell%20regulatory%20systems%20through%20protein%20interaction%20domains.%20Science300,%20445%E2%80%93452%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Pawson, T. &amp; Nash, P. Protein-protein interactions define specificity in signal transduction. <em>Genes Dev.</em><strong>14</strong>, 1027–1047 (2000).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/10809663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pawson,%20T.%20&amp;%20Nash,%20P.%20Protein-protein%20interactions%20define%20specificity%20in%20signal%20transduction.%20Genes%20Dev.14,%201027%E2%80%931047%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Banani, S. F., Lee, H. O., Hyman, A. A. &amp; Rosen, M. K. Biomolecular condensates: organizers of cellular biochemistry. <em>Nat. Rev. Mol. Cell Biol.</em><strong>18</strong>, 285–298 (2017).
</cite> [<a href="https://doi.org/10.1038/nrm.2017.7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7434221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28225081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banani,%20S.%20F.,%20Lee,%20H.%20O.,%20Hyman,%20A.%20A.%20&amp;%20Rosen,%20M.%20K.%20Biomolecular%20condensates:%20organizers%20of%20cellular%20biochemistry.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.18,%20285%E2%80%93298%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Shin, Y. &amp; Brangwynne, C. P. Liquid phase condensation in cell physiology and disease. <em>Science</em><strong>357</strong>, eaaf4382 (2017).
</cite> [<a href="https://doi.org/10.1126/science.aaf4382" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28935776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shin,%20Y.%20&amp;%20Brangwynne,%20C.%20P.%20Liquid%20phase%20condensation%20in%20cell%20physiology%20and%20disease.%20Science357,%20eaaf4382%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Hyman, A. A., Weber, C. A. &amp; Jülicher, F. Liquid-liquid phase separation in biology. <em>Annu. Rev. Cell Dev. Biol.</em><strong>30</strong>, 39–58 (2014).
</cite> [<a href="https://doi.org/10.1146/annurev-cellbio-100913-013325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25288112/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hyman,%20A.%20A.,%20Weber,%20C.%20A.%20&amp;%20J%C3%BClicher,%20F.%20Liquid-liquid%20phase%20separation%20in%20biology.%20Annu.%20Rev.%20Cell%20Dev.%20Biol.30,%2039%E2%80%9358%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Brangwynne, C. P., Tompa, P. &amp; Pappu, R. V. Polymer physics of intracellular phase transitions. <em>Nat. Phys.</em><strong>11</strong>, 899–904 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Brangwynne,%20C.%20P.,%20Tompa,%20P.%20&amp;%20Pappu,%20R.%20V.%20Polymer%20physics%20of%20intracellular%20phase%20transitions.%20Nat.%20Phys.11,%20899%E2%80%93904%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Alberti, S., Gladfelter, A. &amp; Mittag, T. Considerations and challenges in studying liquid-liquid phase separation and biomolecular condensates. <em>Cell</em><strong>176</strong>, 419–434 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.12.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6445271/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30682370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alberti,%20S.,%20Gladfelter,%20A.%20&amp;%20Mittag,%20T.%20Considerations%20and%20challenges%20in%20studying%20liquid-liquid%20phase%20separation%20and%20biomolecular%20condensates.%20Cell176,%20419%E2%80%93434%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Kedersha, N. &amp; Anderson, P. Mammalian stress granules and processing bodies. In <em>Methods in Enzymology</em> Vol. 431 61–81 (Academic Press, 2007).</cite> [<a href="https://doi.org/10.1016/S0076-6879(07)31005-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17923231/" class="usa-link">PubMed</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Wolozin, B. &amp; Ivanov, P. Stress granules and neurodegeneration. <em>Nat. Rev. Neurosci.</em><strong>20</strong>, 649–666 (2019).
</cite> [<a href="https://doi.org/10.1038/s41583-019-0222-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6986315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31582840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolozin,%20B.%20&amp;%20Ivanov,%20P.%20Stress%20granules%20and%20neurodegeneration.%20Nat.%20Rev.%20Neurosci.20,%20649%E2%80%93666%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Guillén-Boixet, J. et al. RNA-induced conformational switching and clustering of G3BP drive stress granule assembly by condensation. <em>Cell</em><strong>181</strong>, 346–361.e17 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.03.049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7181197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32302572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guill%C3%A9n-Boixet,%20J.%20et%20al.%20RNA-induced%20conformational%20switching%20and%20clustering%20of%20G3BP%20drive%20stress%20granule%20assembly%20by%20condensation.%20Cell181,%20346%E2%80%93361.e17%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Patel, A. et al. A liquid-to-solid phase transition of the ALS protein FUS accelerated by disease mutation. <em>Cell</em><strong>162</strong>, 1066–1077 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.07.047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26317470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patel,%20A.%20et%20al.%20A%20liquid-to-solid%20phase%20transition%20of%20the%20ALS%20protein%20FUS%20accelerated%20by%20disease%20mutation.%20Cell162,%201066%E2%80%931077%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Alberti, S. &amp; Dormann, D. Liquid-liquid phase separation in disease. <em>Annu. Rev. Genet.</em><strong>53</strong>, 171–194 (2019).
</cite> [<a href="https://doi.org/10.1146/annurev-genet-112618-043527" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31430179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alberti,%20S.%20&amp;%20Dormann,%20D.%20Liquid-liquid%20phase%20separation%20in%20disease.%20Annu.%20Rev.%20Genet.53,%20171%E2%80%93194%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Arter, W. E. et al. Biomolecular condensate phase diagrams with a combinatorial microdroplet platform. <em>Nat. Commun.</em><strong>13</strong>, 7845 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-35265-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9768726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36543777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arter,%20W.%20E.%20et%20al.%20Biomolecular%20condensate%20phase%20diagrams%20with%20a%20combinatorial%20microdroplet%20platform.%20Nat.%20Commun.13,%207845%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>McCarty, J., Delaney, K. T., Danielsen, S. P. O., Fredrickson, G. H. &amp; Shea, J.-E. Complete phase diagram for liquid–liquid phase separation of intrinsically disordered proteins. <em>J. Phys. Chem. Lett.</em><strong>10</strong>, 1644–1652 (2019).
</cite> [<a href="https://doi.org/10.1021/acs.jpclett.9b00099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7379843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30873835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McCarty,%20J.,%20Delaney,%20K.%20T.,%20Danielsen,%20S.%20P.%20O.,%20Fredrickson,%20G.%20H.%20&amp;%20Shea,%20J.-E.%20Complete%20phase%20diagram%20for%20liquid%E2%80%93liquid%20phase%20separation%20of%20intrinsically%20disordered%20proteins.%20J.%20Phys.%20Chem.%20Lett.10,%201644%E2%80%931652%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Villois, A. et al. Droplet microfluidics for the label-free extraction of complete phase diagrams and kinetics of liquid–liquid phase separation in finite volumes. <em>Small</em><strong>18</strong>, 2202606 (2022).</cite> [<a href="https://doi.org/10.1002/smll.202202606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36180409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Villois,%20A.%20et%20al.%20Droplet%20microfluidics%20for%20the%20label-free%20extraction%20of%20complete%20phase%20diagrams%20and%20kinetics%20of%20liquid%E2%80%93liquid%20phase%20separation%20in%20finite%20volumes.%20Small18,%202202606%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Qian, D. et al. Tie-line analysis reveals interactions driving heteromolecular condensate formation. <em>Phys. Rev. X</em><strong>12</strong>, 041038 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Qian,%20D.%20et%20al.%20Tie-line%20analysis%20reveals%20interactions%20driving%20heteromolecular%20condensate%20formation.%20Phys.%20Rev.%20X12,%20041038%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Qian, D. et al. Dominance analysis to assess solute contributions to multicomponent phase equilibria. <em>Proc. Natl. Acad. Sci. USA</em><strong>121</strong>, e2407453121 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2407453121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11331137/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39102550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qian,%20D.%20et%20al.%20Dominance%20analysis%20to%20assess%20solute%20contributions%20to%20multicomponent%20phase%20equilibria.%20Proc.%20Natl.%20Acad.%20Sci.%20USA121,%20e2407453121%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Murakami, T. et al. ALS/FTD mutation-induced phase transition of FUS liquid droplets and reversible hydrogels into irreversible hydrogels impairs RNP granule function. <em>Neuron</em><strong>88</strong>, 678–690 (2015).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2015.10.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4660210/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26526393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murakami,%20T.%20et%20al.%20ALS/FTD%20mutation-induced%20phase%20transition%20of%20FUS%20liquid%20droplets%20and%20reversible%20hydrogels%20into%20irreversible%20hydrogels%20impairs%20RNP%20granule%20function.%20Neuron88,%20678%E2%80%93690%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Maharana, S. et al. RNA buffers the phase separation behavior of prion-like RNA binding proteins. <em>Science</em><strong>360</strong>, 918–921 (2018).
</cite> [<a href="https://doi.org/10.1126/science.aar7366" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6091854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29650702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maharana,%20S.%20et%20al.%20RNA%20buffers%20the%20phase%20separation%20behavior%20of%20prion-like%20RNA%20binding%20proteins.%20Science360,%20918%E2%80%93921%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Qamar, S. et al. FUS phase separation is modulated by a molecular chaperone and methylation of arginine cation-π interactions. <em>Cell</em><strong>173</strong>, 720–734.e15 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.03.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5927716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29677515/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qamar,%20S.%20et%20al.%20FUS%20phase%20separation%20is%20modulated%20by%20a%20molecular%20chaperone%20and%20methylation%20of%20arginine%20cation-%CF%80%20interactions.%20Cell173,%20720%E2%80%93734.e15%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Dormann, D. et al. ALS-associated fused in sarcoma (FUS) mutations disrupt Transportin-mediated nuclear import. <em>EMBO J.</em><strong>29</strong>, 2841–2857 (2010).
</cite> [<a href="https://doi.org/10.1038/emboj.2010.143" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2924641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20606625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dormann,%20D.%20et%20al.%20ALS-associated%20fused%20in%20sarcoma%20(FUS)%20mutations%20disrupt%20Transportin-mediated%20nuclear%20import.%20EMBO%20J.29,%202841%E2%80%932857%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Dormann, D. &amp; Haass, C. Fused in sarcoma (FUS): an oncogene goes awry in neurodegeneration. <em>Mol. Cell. Neurosci.</em><strong>56</strong>, 475–486 (2013).
</cite> [<a href="https://doi.org/10.1016/j.mcn.2013.03.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23557964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dormann,%20D.%20&amp;%20Haass,%20C.%20Fused%20in%20sarcoma%20(FUS):%20an%20oncogene%20goes%20awry%20in%20neurodegeneration.%20Mol.%20Cell.%20Neurosci.56,%20475%E2%80%93486%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Welsh, T. J. et al. Surface electrostatics govern the emulsion stability of biomolecular condensates. <em>Nano Lett.</em><strong>22</strong>, 612–621 (2022).
</cite> [<a href="https://doi.org/10.1021/acs.nanolett.1c03138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35001622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Welsh,%20T.%20J.%20et%20al.%20Surface%20electrostatics%20govern%20the%20emulsion%20stability%20of%20biomolecular%20condensates.%20Nano%20Lett.22,%20612%E2%80%93621%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Li, L. et al. Phase behavior and salt partitioning in polyelectrolyte complex coacervates. <em>Macromolecules</em><strong>51</strong>, 2988–2995 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Li,%20L.%20et%20al.%20Phase%20behavior%20and%20salt%20partitioning%20in%20polyelectrolyte%20complex%20coacervates.%20Macromolecules51,%202988%E2%80%932995%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Friedowitz, S. et al. Looping-in complexation and ion partitioning in nonstoichiometric polyelectrolyte mixtures. <em>Sci. Adv.</em><strong>7</strong>, eabg8654 (2021).
</cite> [<a href="https://doi.org/10.1126/sciadv.abg8654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8324053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34330707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Friedowitz,%20S.%20et%20al.%20Looping-in%20complexation%20and%20ion%20partitioning%20in%20nonstoichiometric%20polyelectrolyte%20mixtures.%20Sci.%20Adv.7,%20eabg8654%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Ausserwöger, H. et al. Biomolecular condensates sustain pH gradients at equilibrium driven by charge neutralisation. Preprint at <em>bioRxiv</em>10.1101/2024.05.23.595321 (2024).</cite>
</li>
<li id="CR29">
<span class="label">29.</span><cite>Kozlov, A. G. et al. How glutamate promotes liquid-liquid phase separation and DNA binding cooperativity of E. coli SSB protein. <em>J. Mol. Biol.</em><strong>434</strong>, 167562 (2022).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2022.167562" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9400470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35351518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kozlov,%20A.%20G.%20et%20al.%20How%20glutamate%20promotes%20liquid-liquid%20phase%20separation%20and%20DNA%20binding%20cooperativity%20of%20E.%20coli%20SSB%20protein.%20J.%20Mol.%20Biol.434,%20167562%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Suhling, K. et al. Fluorescence lifetime imaging (FLIM): Basic concepts and some recent developments. <em>Med. Photonics</em><strong>27</strong>, 3–40 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Suhling,%20K.%20et%20al.%20Fluorescence%20lifetime%20imaging%20(FLIM):%20Basic%20concepts%20and%20some%20recent%20developments.%20Med.%20Photonics27,%203%E2%80%9340%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Suhling, K. et al. Imaging the environment of green fluorescent protein. <em>Biophys. J.</em><strong>83</strong>, 3589–3595 (2002).
</cite> [<a href="https://doi.org/10.1016/S0006-3495(02)75359-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1302434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12496126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suhling,%20K.%20et%20al.%20Imaging%20the%20environment%20of%20green%20fluorescent%20protein.%20Biophys.%20J.83,%203589%E2%80%933595%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Saha, S. et al. Polar positioning of phase-separated liquid compartments in cells regulated by an mRNA competition mechanism. <em>Cell</em><strong>166</strong>, 1572–1584.e16 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.08.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5034880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27594427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saha,%20S.%20et%20al.%20Polar%20positioning%20of%20phase-separated%20liquid%20compartments%20in%20cells%20regulated%20by%20an%20mRNA%20competition%20mechanism.%20Cell166,%201572%E2%80%931584.e16%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Yan, X. et al. Intra-condensate demixing of TDP-43 inside stress granules generates pathological aggregates. <em>Cell</em><strong>188</strong>, 4124–4140.e18 (2025).</cite> [<a href="https://doi.org/10.1016/j.cell.2025.04.039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12303766/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40412392/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Krainer, G. et al. Reentrant liquid condensate phase of proteins is stabilized by hydrophobic and non-ionic interactions. <em>Nat. Commun.</em><strong>12</strong>, 1085 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-21181-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7889641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33597515/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krainer,%20G.%20et%20al.%20Reentrant%20liquid%20condensate%20phase%20of%20proteins%20is%20stabilized%20by%20hydrophobic%20and%20non-ionic%20interactions.%20Nat.%20Commun.12,%201085%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>CACACE, M. G., LANDAU, E. M. &amp; RAMSDEN, J. J. The Hofmeister series: salt and solvent effects on interfacial phenomena. <em>Q. Rev. Biophys.</em><strong>30</strong>, 241–277 (1997).
</cite> [<a href="https://doi.org/10.1017/s0033583597003363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9394422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?CACACE,%20M.%20G.,%20LANDAU,%20E.%20M.%20&amp;%20RAMSDEN,%20J.%20J.%20The%20Hofmeister%20series:%20salt%20and%20solvent%20effects%20on%20interfacial%20phenomena.%20Q.%20Rev.%20Biophys.30,%20241%E2%80%93277%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Ambadipudi, S., Reddy, J. G., Biernat, J., Mandelkow, E. &amp; Zweckstetter, M. Residue-specific identification of phase separation hot spots of Alzheimer’s-related protein tau. <em>Chem. Sci.</em><strong>10</strong>, 6503–6507 (2019).
</cite> [<a href="https://doi.org/10.1039/c9sc00531e" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6610569/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31341602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ambadipudi,%20S.,%20Reddy,%20J.%20G.,%20Biernat,%20J.,%20Mandelkow,%20E.%20&amp;%20Zweckstetter,%20M.%20Residue-specific%20identification%20of%20phase%20separation%20hot%20spots%20of%20Alzheimer%E2%80%99s-related%20protein%20tau.%20Chem.%20Sci.10,%206503%E2%80%936507%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Yamazaki, T. et al. Functional domains of NEAT1 architectural lncRNA induce paraspeckle assembly through phase separation. <em>Mol. Cell</em><strong>70</strong>, 1038–1053.e7 (2018).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2018.05.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29932899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamazaki,%20T.%20et%20al.%20Functional%20domains%20of%20NEAT1%20architectural%20lncRNA%20induce%20paraspeckle%20assembly%20through%20phase%20separation.%20Mol.%20Cell70,%201038%E2%80%931053.e7%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Strom, A. R. et al. Phase separation drives heterochromatin domain formation. <em>Nature</em><strong>547</strong>, 241–245 (2017).
</cite> [<a href="https://doi.org/10.1038/nature22989" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6022742/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28636597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Strom,%20A.%20R.%20et%20al.%20Phase%20separation%20drives%20heterochromatin%20domain%20formation.%20Nature547,%20241%E2%80%93245%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Lu, H. et al. Phase-separation mechanism for C-terminal hyperphosphorylation of RNA polymerase II. <em>Nature</em><strong>558</strong>, 318–323 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0174-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6475116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29849146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20H.%20et%20al.%20Phase-separation%20mechanism%20for%20C-terminal%20hyperphosphorylation%20of%20RNA%20polymerase%20II.%20Nature558,%20318%E2%80%93323%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>André, A. A. M., Yewdall, N. A. &amp; Spruijt, E. Crowding-induced phase separation and gelling by co-condensation of PEG in NPM1-rRNA condensates. <em>Biophys. J.</em><strong>122</strong>, 397–407 (2023).
</cite> [<a href="https://doi.org/10.1016/j.bpj.2022.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9892608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36463407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andr%C3%A9,%20A.%20A.%20M.,%20Yewdall,%20N.%20A.%20&amp;%20Spruijt,%20E.%20Crowding-induced%20phase%20separation%20and%20gelling%20by%20co-condensation%20of%20PEG%20in%20NPM1-rRNA%20condensates.%20Biophys.%20J.122,%20397%E2%80%93407%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Arosio, P. et al. Microfluidic diffusion analysis of the sizes and interactions of proteins under native solution conditions. <em>ACS Nano</em><strong>10</strong>, 333–341 (2016).
</cite> [<a href="https://doi.org/10.1021/acsnano.5b04713" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26678709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arosio,%20P.%20et%20al.%20Microfluidic%20diffusion%20analysis%20of%20the%20sizes%20and%20interactions%20of%20proteins%20under%20native%20solution%20conditions.%20ACS%20Nano10,%20333%E2%80%93341%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Riback, J. A. et al. Composition-dependent thermodynamics of intracellular phase separation. <em>Nature</em><strong>581</strong>, 209–214 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2256-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7733533/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32405004/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Riback,%20J.%20A.%20et%20al.%20Composition-dependent%20thermodynamics%20of%20intracellular%20phase%20separation.%20Nature581,%20209%E2%80%93214%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Wei, M.-T. et al. Nucleated transcriptional condensates amplify gene expression. <em>Nat. Cell Biol.</em><strong>22</strong>, 1187–1196 (2020).
</cite> [<a href="https://doi.org/10.1038/s41556-020-00578-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32929202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei,%20M.-T.%20et%20al.%20Nucleated%20transcriptional%20condensates%20amplify%20gene%20expression.%20Nat.%20Cell%20Biol.22,%201187%E2%80%931196%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Duffy, D. C., McDonald, J. C., Schueller, O. J. A. &amp; Whitesides, G. M. Rapid prototyping of microfluidic systems in poly(dimethylsiloxane). <em>Anal. Chem.</em><strong>70</strong>, 4974–4984 (1998).
</cite> [<a href="https://doi.org/10.1021/ac980656z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21644679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duffy,%20D.%20C.,%20McDonald,%20J.%20C.,%20Schueller,%20O.%20J.%20A.%20&amp;%20Whitesides,%20G.%20M.%20Rapid%20prototyping%20of%20microfluidic%20systems%20in%20poly(dimethylsiloxane).%20Anal.%20Chem.70,%204974%E2%80%934984%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Challa, P. K., Kartanas, T., Charmet, J. &amp; Knowles, T. P. J. Microfluidic devices fabricated using fast wafer-scale LED-lithography patterning. <em>Biomicrofluidics</em><strong>11</strong>, 014113 (2017).
</cite> [<a href="https://doi.org/10.1063/1.4976690" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5315664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28289484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Challa,%20P.%20K.,%20Kartanas,%20T.,%20Charmet,%20J.%20&amp;%20Knowles,%20T.%20P.%20J.%20Microfluidic%20devices%20fabricated%20using%20fast%20wafer-scale%20LED-lithography%20patterning.%20Biomicrofluidics11,%20014113%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Qian, D. et al. Molecular mechanisms of condensate modulation from energy-dominance analysis. <em>Phys. Rev. Appl.</em><strong>23</strong>, 064017 (2025).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365266/bin/41467_2025_62437_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365266/bin/41467_2025_62437_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (151.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365266/bin/41467_2025_62437_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365266/bin/41467_2025_62437_MOESM4_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (10.5MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Source data are provided with this paper, including the raw data dilute phase concentration data in the associated Supplementary Figs. <a href="#Sec24" class="usa-link">Source data</a> are provided with this paper.</p>
<p>Exemplary analysis code for the determination of tie line gradients is provided with the source data. Additional analysis steps have been described in detail in the “Methods” section.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62437-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62437.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365266/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365266/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365266%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365266/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365266/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365266/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830100/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365266/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830100/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365266/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365266/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="teFb23WVJEt2C6IQ9bvBEOvcs3lNNGx8ifnqrKkzZ7I1hz5Kcs9VfEauIC1gdhnb">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
